

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/103/4562529","@type":"ScholarlyArticle","name":"Investigating transcription factor synergism in humans","datePublished":"2017-10-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx041","keywords":["transcription factors","gene regulation","biological process","computational biology"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/103/4562529","author":[{"name":"Cumbo, Fabio","affiliation":"Institute for Systems Analysis and Computer Science ‘Antonio Ruberti’, National Research Council of Italy, 00185 Rome, Italy  Department of Engineering, Third University of Rome, 00146 Rome, Italy  SYSBIO.IT–Centre of Systems Biology, 20126 Milan, Italy","@type":"Person"},{"name":"Vergni, Davide","affiliation":"Institute for Applied Mathematics ‘Mauro Picone’, National Research Council of Italy, 00185 Rome, Italy","@type":"Person"},{"name":"Santoni, Daniele","affiliation":"Institute for Systems Analysis and Computer Science ‘Antonio Ruberti’, National Research Council of Italy, 00185 Rome, Italy","@type":"Person"}],"description":"Abstract.  Proteins are the core and the engine of every process in cells thus the study of mechanisms that drive the regulation of protein expression, is essen","pageStart":"103","pageEnd":"112","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Investigating transcription factor synergism in humans","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Investigating transcription factor synergism in humans" />
    <meta property="og:description" content="Abstract.  Proteins are the core and the engine of every process in cells thus the study of mechanisms that drive the regulation of protein expression, is essen" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/103/4562529" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-10-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Cumbo, Fabio" /><meta name="citation_author_institution" content="Institute for Systems Analysis and Computer Science ‘Antonio Ruberti’, National Research Council of Italy, 00185 Rome, Italy" /><meta name="citation_author_institution" content="Department of Engineering, Third University of Rome, 00146 Rome, Italy" /><meta name="citation_author_institution" content="SYSBIO.IT–Centre of Systems Biology, 20126 Milan, Italy" /><meta name="citation_author" content="Vergni, Davide" /><meta name="citation_author_institution" content="Institute for Applied Mathematics ‘Mauro Picone’, National Research Council of Italy, 00185 Rome, Italy" /><meta name="citation_author" content="Santoni, Daniele" /><meta name="citation_author_institution" content="Institute for Systems Analysis and Computer Science ‘Antonio Ruberti’, National Research Council of Italy, 00185 Rome, Italy" /><meta name="citation_title" content="Investigating transcription factor synergism in humans" /><meta name="citation_firstpage" content="103" /><meta name="citation_lastpage" content="112" /><meta name="citation_doi" content="10.1093/dnares/dsx041" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Transcriptional regulation in mammalian cells by sequence-specific DNA binding proteins;  citation_journal_title=Science;  citation_year=1989;  citation_volume=245;  citation_pages=371-8; " /><meta name="citation_reference" content="citation_title=Transcription factors: an overview;  citation_journal_title=Int. J. Biochem. Cell Biol;  citation_year=1997;  citation_volume=29;  citation_pages=1305-12; " /><meta name="citation_reference" content="citation_title=Rapid search for specific sites on DNA through conformational switch of nonspecifically bound proteins;  citation_journal_title=Proc. Nat. Acad. Sci. U.S.A;  citation_year=2011;  citation_volume=108;  citation_pages=8651-6; " /><meta name="citation_reference" content="citation_title=Diffusion-driven mechanisms of protein translocation on nucleic acids. 1. Models and theory;  citation_journal_title=Biochemistry;  citation_year=1981;  citation_volume=20;  citation_pages=6929-48; " /><meta name="citation_reference" content="citation_title=Activators and target genes of Rel/NF-kB transcription factors;  citation_journal_title=Oncogene;  citation_year=1999;  citation_volume=18;  citation_pages=6853-66; " /><meta name="citation_reference" content="citation_title=TRANSFAC: an integrated system for gene expression regulation;  citation_journal_title=Nucleic Acids Res;  citation_year=2000;  citation_volume=28;  citation_pages=316-9; " /><meta name="citation_reference" content="citation_title=The TRANSFAC project as an example of framework technology that supports the analysis of genomic regulation;  citation_journal_title=Brief. Bioinform;  citation_year=2008;  citation_volume=9;  citation_pages=326-32; " /><meta name="citation_reference" content="citation_title=JASPAR: an open-access database for eukaryotic transcription factor binding profiles;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=D91-4; " /><meta name="citation_reference" content="citation_title=Assessing computational tools for the discovery of transcription factor binding sites;  citation_journal_title=Nat. Biotechnol;  citation_year=2005;  citation_volume=23;  citation_pages=137-44; " /><meta name="citation_reference" content="citation_title=Identifying transcriptional cis-regulatory modules in animal genomes;  citation_journal_title=Wires. Dev. Biol;  citation_year=2015;  citation_volume=4;  citation_pages=59-84; " /><meta name="citation_reference" content="citation_title=Genome-wide computational prediction of transcriptional regulatory modules reveals new insights into human gene expression. Genome.;  citation_journal_title=Res;  citation_year=2006;  citation_volume=16;  citation_pages=656-68; " /><meta name="citation_reference" content="citation_title=Close sequence comparisons are sufficient to identify human cis-regulatory elements;  citation_journal_title=Genome Res;  citation_year=2006;  citation_volume=16;  citation_pages=855-63; " /><meta name="citation_reference" content="citation_title=Prediction of synergistic transcription factors by function conservation;  citation_journal_title=Genome Biol;  citation_year=2007;  citation_volume=8;  citation_pages=R257" /><meta name="citation_reference" content="citation_title=Computational analysis of tissue-specific combinatorial gene regulation: predicting interaction between transcription factors in human tissues;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=4925-36; " /><meta name="citation_reference" content="citation_title=Identification of cis-regulatory modules in promoters of human genes exploiting mutual positioning of transcription factors;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=8822-41; " /><meta name="citation_reference" content="citation_title=Predicting transcription factor synergism;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=4278-84; " /><meta name="citation_reference" content="citation_title=The human genome browser at UCSC;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=996-1006; " /><meta name="citation_reference" content="citation_title=Bioconductor: open software development for computational biology and bioinformatics;  citation_journal_title=Genome Biol;  citation_year=2004;  citation_volume=5;  citation_pages=1" /><meta name="citation_reference" content="citation_title=The MAPPER database: a multi-genome catalog of putative transcription factor binding sites;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33 (suppl 1;  citation_pages=D91-7; " /><meta name="citation_reference" content="Pagès H., Aboyoun P., Gentleman R., DebRoy S.  2009 , String objects representing biological sequences, and matching algorithms. R package version 2.2. " /><meta name="citation_reference" content="citation_title=Divergence measures based on the shannon entropy;  citation_journal_title=IEEE Trans. Inform. Theory;  citation_year=1991;  citation_volume=37;  citation_pages=145-51; " /><meta name="citation_reference" content="citation_title=The STRING database in 2017: quality-controlled protein-protein association networks, made broadly accessible;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=D362-8; " /><meta name="citation_reference" content="citation_publisher=Wiley-Blackwell, Hoboken, New Jersey;  citation_title=Statistical Bioinformatics for Biomedical and Life Science Researchers;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=mSin3A corepressor regulates diverse transcriptional networks governing normal and neoplastic growth and survival;  citation_journal_title=Genes Dev;  citation_year=2005;  citation_volume=19;  citation_pages=1581-95; " /><meta name="citation_reference" content="citation_title=NuA4, an essential transcription adaptor/histone H4 acetyltransferase complex containing Esa1p and the ATM-related cofactor Tra1p;  citation_journal_title=Embo J;  citation_year=1999;  citation_volume=18;  citation_pages=5108-19; " /><meta name="citation_reference" content="citation_title=The chromatin structure of the dual c-myc promoter P1/P2 is regulated by separate elements;  citation_journal_title=J. Biol. Chem;  citation_year=2001;  citation_volume=276;  citation_pages=20482-90; " /><meta name="citation_reference" content="citation_title=Copy number variations in 6q14. 1 and 5q13. 2 are associated with alcohol dependence;  citation_journal_title=Alcohol. Clin. Exp. Res;  citation_year=2012;  citation_volume=36;  citation_pages=1512-8; " /><meta name="citation_reference" content="citation_title=Treatment of spinal muscular atrophy cells with drugs that upregulate SMN expression reveals inter-and intra-patient variability;  citation_journal_title=Eur. J. Hum. Genet;  citation_year=2011;  citation_volume=19;  citation_pages=1059-65; " /><meta name="citation_reference" content="citation_title=Molecular analysis of the human chromosome 5q13. 3 region in patients with hairy cell leukemia and identification of tumor suppressor gene candidates;  citation_journal_title=Genomics;  citation_year=1999;  citation_volume=60;  citation_pages=161-71; " /><meta name="citation_reference" content="citation_title=‘Mowat-Wilson’ syndrome with and without hirschsprung disease is a distinct, recognizable multiple congenital anomalies-mental retardation syndrome caused by mutations in the zinc finger homeo box 1B;  citation_journal_title=Am. J. Med. Genet;  citation_year=2002;  citation_volume=108;  citation_pages=177-81; " /><meta name="citation_reference" content="citation_title=Common regulatory elements in the polycystic kidney disease 1 and 2 promoter regions;  citation_journal_title=Eur. J. Hum. Genet;  citation_year=2005;  citation_volume=13;  citation_pages=649-59; " /><meta name="citation_reference" content="citation_title=Genome-wide approach to identify risk factors for therapy-related myeloid leukemia;  citation_journal_title=Leukemia;  citation_year=2006;  citation_volume=20;  citation_pages=239-46; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/103/24151658/dsx041.pdf" /><meta name="description" content="Abstract.  Proteins are the core and the engine of every process in cells thus the study of mechanisms that drive the regulation of protein expression, is essen" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/103/4562529" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/103/4562529" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/103/4562529");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx041';
    </script>
<title>
	Investigating transcription factor synergism in humans | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4562529" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeBPPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDdW1ibywgRmFiaW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgZm9yIFN5c3RlbXMgQW5hbHlzaXMgYW5kIENvbXB1dGVyIFNjaWVuY2Ug4oCYQW50b25pbyBSdWJlcnRp4oCZLCBOYXRpb25hbCBSZXNlYXJjaCBDb3VuY2lsIG9mIEl0YWx5LCAwMDE4NSBSb21lLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRW5naW5lZXJpbmcsIFRoaXJkIFVuaXZlcnNpdHkgb2YgUm9tZSwgMDAxNDYgUm9tZSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTWVNCSU8uSVTigJNDZW50cmUgb2YgU3lzdGVtcyBCaW9sb2d5LCAyMDEyNiBNaWxhbiwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWZXJnbmksIERhdmlkZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBmb3IgQXBwbGllZCBNYXRoZW1hdGljcyDigJhNYXVybyBQaWNvbmXigJksIE5hdGlvbmFsIFJlc2VhcmNoIENvdW5jaWwgb2YgSXRhbHksIDAwMTg1IFJvbWUsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FudG9uaSwgRGFuaWVsZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBmb3IgU3lzdGVtcyBBbmFseXNpcyBhbmQgQ29tcHV0ZXIgU2NpZW5jZSDigJhBbnRvbmlvIFJ1YmVydGnigJksIE5hdGlvbmFsIFJlc2VhcmNoIENvdW5jaWwgb2YgSXRhbHksIDAwMTg1IFJvbWUsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJbnZlc3RpZ2F0aW5nIHRyYW5zY3JpcHRpb24gZmFjdG9yIHN5bmVyZ2lzbSBpbiBodW1hbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjExMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDA0MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb24gaW4gbWFtbWFsaWFuIGNlbGxzIGJ5IHNlcXVlbmNlLXNwZWNpZmljIEROQSBiaW5kaW5nIHByb3RlaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTI0NTsgIGNpdGF0aW9uX3BhZ2VzPTM3MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIGZhY3RvcnM6IGFuIG92ZXJ2aWV3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIEJpb2NoZW0uIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTEzMDUtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIHNlYXJjaCBmb3Igc3BlY2lmaWMgc2l0ZXMgb24gRE5BIHRocm91Z2ggY29uZm9ybWF0aW9uYWwgc3dpdGNoIG9mIG5vbnNwZWNpZmljYWxseSBib3VuZCBwcm90ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0LiBBY2FkLiBTY2kuIFUuUy5BOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTg2NTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZnVzaW9uLWRyaXZlbiBtZWNoYW5pc21zIG9mIHByb3RlaW4gdHJhbnNsb2NhdGlvbiBvbiBudWNsZWljIGFjaWRzLiAxLiBNb2RlbHMgYW5kIHRoZW9yeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjkyOS00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWN0aXZhdG9ycyBhbmQgdGFyZ2V0IGdlbmVzIG9mIFJlbC9ORi1rQiB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU9uY29nZW5lOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9Njg1My02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VFJBTlNGQUM6IGFuIGludGVncmF0ZWQgc3lzdGVtIGZvciBnZW5lIGV4cHJlc3Npb24gcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0zMTYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFRSQU5TRkFDIHByb2plY3QgYXMgYW4gZXhhbXBsZSBvZiBmcmFtZXdvcmsgdGVjaG5vbG9neSB0aGF0IHN1cHBvcnRzIHRoZSBhbmFseXNpcyBvZiBnZW5vbWljIHJlZ3VsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyaWVmLiBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTMyNi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SkFTUEFSOiBhbiBvcGVuLWFjY2VzcyBkYXRhYmFzZSBmb3IgZXVrYXJ5b3RpYyB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIHByb2ZpbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPUQ5MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgY29tcHV0YXRpb25hbCB0b29scyBmb3IgdGhlIGRpc2NvdmVyeSBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIHNpdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZ5aW5nIHRyYW5zY3JpcHRpb25hbCBjaXMtcmVndWxhdG9yeSBtb2R1bGVzIGluIGFuaW1hbCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1XaXJlcy4gRGV2LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz01OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgY29tcHV0YXRpb25hbCBwcmVkaWN0aW9uIG9mIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0b3J5IG1vZHVsZXMgcmV2ZWFscyBuZXcgaW5zaWdodHMgaW50byBodW1hbiBnZW5lIGV4cHJlc3Npb24uIEdlbm9tZS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvc2Ugc2VxdWVuY2UgY29tcGFyaXNvbnMgYXJlIHN1ZmZpY2llbnQgdG8gaWRlbnRpZnkgaHVtYW4gY2lzLXJlZ3VsYXRvcnkgZWxlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz04NTUtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2Ygc3luZXJnaXN0aWMgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGJ5IGZ1bmN0aW9uIGNvbnNlcnZhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVIyNTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGFuYWx5c2lzIG9mIHRpc3N1ZS1zcGVjaWZpYyBjb21iaW5hdG9yaWFsIGdlbmUgcmVndWxhdGlvbjogcHJlZGljdGluZyBpbnRlcmFjdGlvbiBiZXR3ZWVuIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBpbiBodW1hbiB0aXNzdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTQ5MjUtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGNpcy1yZWd1bGF0b3J5IG1vZHVsZXMgaW4gcHJvbW90ZXJzIG9mIGh1bWFuIGdlbmVzIGV4cGxvaXRpbmcgbXV0dWFsIHBvc2l0aW9uaW5nIG9mIHRyYW5zY3JpcHRpb24gZmFjdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz04ODIyLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHRyYW5zY3JpcHRpb24gZmFjdG9yIHN5bmVyZ2lzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz00Mjc4LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaHVtYW4gZ2Vub21lIGJyb3dzZXIgYXQgVUNTQzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTk5Ni0xMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9jb25kdWN0b3I6IG9wZW4gc29mdHdhcmUgZGV2ZWxvcG1lbnQgZm9yIGNvbXB1dGF0aW9uYWwgYmlvbG9neSBhbmQgYmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1BUFBFUiBkYXRhYmFzZTogYSBtdWx0aS1nZW5vbWUgY2F0YWxvZyBvZiBwdXRhdGl2ZSB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIHNpdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMyAoc3VwcGwgMTsgIGNpdGF0aW9uX3BhZ2VzPUQ5MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJQYWfDqHMgSC4sIEFib3lvdW4gUC4sIEdlbnRsZW1hbiBSLiwgRGViUm95IFMuICAyMDA5ICwgU3RyaW5nIG9iamVjdHMgcmVwcmVzZW50aW5nIGJpb2xvZ2ljYWwgc2VxdWVuY2VzLCBhbmQgbWF0Y2hpbmcgYWxnb3JpdGhtcy4gUiBwYWNrYWdlIHZlcnNpb24gMi4yLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbmNlIG1lYXN1cmVzIGJhc2VkIG9uIHRoZSBzaGFubm9uIGVudHJvcHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlFRUUgVHJhbnMuIEluZm9ybS4gVGhlb3J5OyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTQ1LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU1RSSU5HIGRhdGFiYXNlIGluIDIwMTc6IHF1YWxpdHktY29udHJvbGxlZCBwcm90ZWluLXByb3RlaW4gYXNzb2NpYXRpb24gbmV0d29ya3MsIG1hZGUgYnJvYWRseSBhY2Nlc3NpYmxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPUQzNjItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVdpbGV5LUJsYWNrd2VsbCwgSG9ib2tlbiwgTmV3IEplcnNleTsgIGNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIEJpb2luZm9ybWF0aWNzIGZvciBCaW9tZWRpY2FsIGFuZCBMaWZlIFNjaWVuY2UgUmVzZWFyY2hlcnM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1TaW4zQSBjb3JlcHJlc3NvciByZWd1bGF0ZXMgZGl2ZXJzZSB0cmFuc2NyaXB0aW9uYWwgbmV0d29ya3MgZ292ZXJuaW5nIG5vcm1hbCBhbmQgbmVvcGxhc3RpYyBncm93dGggYW5kIHN1cnZpdmFsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lcyBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNTgxLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdUE0LCBhbiBlc3NlbnRpYWwgdHJhbnNjcmlwdGlvbiBhZGFwdG9yL2hpc3RvbmUgSDQgYWNldHlsdHJhbnNmZXJhc2UgY29tcGxleCBjb250YWluaW5nIEVzYTFwIGFuZCB0aGUgQVRNLXJlbGF0ZWQgY29mYWN0b3IgVHJhMXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVtYm8gSjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTUxMDgtMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjaHJvbWF0aW4gc3RydWN0dXJlIG9mIHRoZSBkdWFsIGMtbXljIHByb21vdGVyIFAxL1AyIGlzIHJlZ3VsYXRlZCBieSBzZXBhcmF0ZSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzY7ICBjaXRhdGlvbl9wYWdlcz0yMDQ4Mi05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29weSBudW1iZXIgdmFyaWF0aW9ucyBpbiA2cTE0LiAxIGFuZCA1cTEzLiAyIGFyZSBhc3NvY2lhdGVkIHdpdGggYWxjb2hvbCBkZXBlbmRlbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbGNvaG9sLiBDbGluLiBFeHAuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTE1MTItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJlYXRtZW50IG9mIHNwaW5hbCBtdXNjdWxhciBhdHJvcGh5IGNlbGxzIHdpdGggZHJ1Z3MgdGhhdCB1cHJlZ3VsYXRlIFNNTiBleHByZXNzaW9uIHJldmVhbHMgaW50ZXItYW5kIGludHJhLXBhdGllbnQgdmFyaWFiaWxpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1ci4gSi4gSHVtLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTEwNTktNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBhbmFseXNpcyBvZiB0aGUgaHVtYW4gY2hyb21vc29tZSA1cTEzLiAzIHJlZ2lvbiBpbiBwYXRpZW50cyB3aXRoIGhhaXJ5IGNlbGwgbGV1a2VtaWEgYW5kIGlkZW50aWZpY2F0aW9uIG9mIHR1bW9yIHN1cHByZXNzb3IgZ2VuZSBjYW5kaWRhdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTE2MS03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU94oCYTW93YXQtV2lsc29u4oCZIHN5bmRyb21lIHdpdGggYW5kIHdpdGhvdXQgaGlyc2Noc3BydW5nIGRpc2Vhc2UgaXMgYSBkaXN0aW5jdCwgcmVjb2duaXphYmxlIG11bHRpcGxlIGNvbmdlbml0YWwgYW5vbWFsaWVzLW1lbnRhbCByZXRhcmRhdGlvbiBzeW5kcm9tZSBjYXVzZWQgYnkgbXV0YXRpb25zIGluIHRoZSB6aW5jIGZpbmdlciBob21lbyBib3ggMUI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBNZWQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTE3Ny04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbW9uIHJlZ3VsYXRvcnkgZWxlbWVudHMgaW4gdGhlIHBvbHljeXN0aWMga2lkbmV5IGRpc2Vhc2UgMSBhbmQgMiBwcm9tb3RlciByZWdpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXIuIEouIEh1bS4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NDktNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFwcHJvYWNoIHRvIGlkZW50aWZ5IHJpc2sgZmFjdG9ycyBmb3IgdGhlcmFweS1yZWxhdGVkIG15ZWxvaWQgbGV1a2VtaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUxldWtlbWlhOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjM5LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvMS8xMDMvMjQxNTE2NTgvZHN4MDQxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQcm90ZWlucyBhcmUgdGhlIGNvcmUgYW5kIHRoZSBlbmdpbmUgb2YgZXZlcnkgcHJvY2VzcyBpbiBjZWxscyB0aHVzIHRoZSBzdHVkeSBvZiBtZWNoYW5pc21zIHRoYXQgZHJpdmUgdGhlIHJlZ3VsYXRpb24gb2YgcHJvdGVpbiBleHByZXNzaW9uLCBpcyBlc3NlbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDU2MjUyOSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQgnUpu6C/aCtMPAtk07OQE1exN+g==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f103%2f4562529" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f103%2f4562529" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112483370">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483370">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483372">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483372">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483374">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483374">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483398">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483398">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483410">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483410">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483415">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483415">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483417">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483417">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/87/4282306">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Investigating transcription factor synergism in humans
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fabio Cumbo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute for Systems Analysis and Computer Science ‘Antonio Ruberti’, National Research Council of Italy, 00185 Rome, Italy</div><div class="aff">Department of Engineering, Third University of Rome, 00146 Rome, Italy</div><div class="aff">SYSBIO.IT–Centre of Systems Biology, 20126 Milan, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fabio+Cumbo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cumbo F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cumbo F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fabio Cumbo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Davide Vergni</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute for Applied Mathematics ‘Mauro Picone’, National Research Council of Italy, 00185 Rome, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Davide+Vergni">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vergni D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vergni D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Davide Vergni</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daniele Santoni</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute for Systems Analysis and Computer Science ‘Antonio Ruberti’, National Research Council of Italy, 00185 Rome, Italy</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx041-cor1" data-open="dsx041-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +39 0649937119. Fax. +39 0649937137. Email: <a href="mailto:daniele.santoni@iasi.cnr.it" target="_blank">daniele.santoni@iasi.cnr.it</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daniele+Santoni">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Santoni D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Santoni D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daniele Santoni</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx041-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Mikita Suyama</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 103–112, <a href='https://doi.org/10.1093/dnares/dsx041'>https://doi.org/10.1093/dnares/dsx041</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 October 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 February 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 September 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4562529" href="/dnaresearch/article-pdf/25/1/103/24151658/dsx041.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Fabio Cumbo, Davide Vergni, Daniele Santoni;  Investigating transcription factor synergism in humans, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 103–112, <a href='https://doi.org/10.1093/dnares/dsx041'>https://doi.org/10.1093/dnares/dsx041</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4562529?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4562529?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4562529?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4562529?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4562529?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4562529?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4562529?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Investigating%20transcription%20factor%20synergism%20in%20humans&amp;publicationDate=2017-10-23&amp;volumeNum=25&amp;issueNum=1&amp;author=Cumbo%2C%20Fabio%3B%20Vergni%2C%20Davide&amp;startPage=103&amp;endPage=112&amp;contentId=10.1093%2Fdnares%2Fdsx041&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f103%2f4562529" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112483370 id="112483370" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Proteins are the core and the engine of every process in cells thus the study of mechanisms that drive the regulation of protein expression, is essential. Transcription factors play a central role in this extremely complex task and they synergically co-operate in order to provide a fine tuning of protein expressions. In the present study, we designed a mathematically well-founded procedure to investigate the mutual positioning of transcription factors binding sites related to a given couple of transcription factors in order to evaluate the possible association between them. We obtained a list of highly related transcription factors couples, whose binding site occurrences significantly group together for a given set of gene promoters, identifying the biological contexts in which the couples are involved in and the processes they should contribute to regulate.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factors&quot;">transcription factors</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene regulation&quot;">gene regulation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;biological process&quot;">biological process</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;computational biology&quot;">computational biology</a></div>                            <h2 scrollto-destination=112483372 id="112483372" class="section-title" >1. Introduction</h2>
<p>The regulation of gene expression is a fundamental mechanism driving biological processes. Transcriptional regulation rules the access of polymerase complex to the gene, activating or repressing the transcription process. Transcription Factors (TF) play a central role in this context; they are proteins able to bind specific DNA regions recognizing a short sequence of nucleotides called Transcription Factor Binding Sites (TFBS). There is a vast literature concerning TFs, starting from seminal and general papers<a reveal-id="dsx041-B1" data-open="dsx041-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx041-B2" data-open="dsx041-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> to the most specific ranging from the study of the mechanism that allows TFs to efficiently and rapidly find the target along the DNA helix,<a reveal-id="dsx041-B3" data-open="dsx041-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx041-B4" data-open="dsx041-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> to the study of the roles that specific TFs play in given biological tasks and how they influence the regulation of the transcription of their target genes.<a reveal-id="dsx041-B5" data-open="dsx041-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> TFBS occurrences along a DNA sequence can be statistically determined via a Position-specific Weight Matrix (PWM), that is essentially a matrix reporting the frequency of nucleotides for each position of the experimentally found binding sites. Several databases based on experimental evidences were designed collecting information about TFs and the corresponding TFBSs, TRANSFAC<a reveal-id="dsx041-B6" data-open="dsx041-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx041-B7" data-open="dsx041-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and JASPAR<a reveal-id="dsx041-B8" data-open="dsx041-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> are often used as a reference for Eukariotes. In the review<a reveal-id="dsx041-B9" data-open="dsx041-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> different algorithmic approaches to predict TFBSs, starting from PWMs, are presented and compared. Despite the large number of genes in higher animals (∼30,000 genes) the number of known TFs is one order of magnitude smaller, indicating a necessary interplay between different TFs in order to regulate the activity of all genes. TFBSs are often clustered in peculiar families and patterns made of several TFBSs can be found in the promoter region, i.e. thousands bases upstream the Transcription Starting Site (TSS), or also very far from the TSS in the Cis-regulatory modules (CRM). The interested reader can find in Ref. <a reveal-id="dsx041-B10" data-open="dsx041-B10" class="link link-ref link-reveal xref-bibr">10</a> (see also references therein) a comprehensive review of the update knowledge about CRM. In this work, we will focus on the synergism of different TFs in the promoter region, trying to identify those TFs couples able to co-operate, and to discover the context in which those couples are involved in. The synergism between TFs in human DNA has been investigated by several studies focused on the analysis of mutual positioning of TFBSs along the DNA regions. Some studies, taking into consideration cross-species sequence comparison trying to identify possible regulatory modules in human DNA.<a reveal-id="dsx041-B11 dsx041-B12 dsx041-B13" data-open="dsx041-B11 dsx041-B12 dsx041-B13" class="link link-ref link-reveal xref-bibr"><sup>11–13</sup></a> Other works focus on a specific class of TFs<a reveal-id="dsx041-B14" data-open="dsx041-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> or on a single TF<a reveal-id="dsx041-B15" data-open="dsx041-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> trying to identify the regulatory modules to which the considered TF belongs to. Finally in Ref. <a reveal-id="dsx041-B16" data-open="dsx041-B16" class="link link-ref link-reveal xref-bibr">16</a>, the whole human genome has been investigated in order to reveal the couples of TFs that synergically act in the transcription regulation. In the present work, we developed a new computational method able to predict the possible interaction between couples of TFs analysing the statistical properties of the distribution of couples of TFBSs in the promoter region of human DNA, and comparing them with a random distribution of TFBSs. We obtained a list of highly related TFs couples, whose TFBSs occurrences significantly group together for a given set of promoter genes, identifying the biological context in which the couples are involved in and the process they should contribute to regulate. </p>                            <h2 scrollto-destination=112483374 id="112483374" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=112483375 id="112483375" class="section-title" >2.1. Extracting TFBSs of human genes</h3>
<p>The list of all human genes was extracted from the Genome Browser (UCSC)<a reveal-id="dsx041-B17" data-open="dsx041-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and all the associated 104,178 transcript sequences (genome version hg38) were retrieved by a proper R library<a reveal-id="dsx041-B18" data-open="dsx041-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> querying the UCSC data repository. For each transcript a promoter region, made of a sequence of 2,000 bp upstream the TSS, was extracted. In order to remove redundant information, the set of all promoter sequences was processed filtering out redundant overlapped transcripts, reducing their number (and the related promoter sequences) to 36,830. In this study, we used a set of 194 most studied and experimentally validated Human Transcription Factors according to MAPPER.<a reveal-id="dsx041-B19" data-open="dsx041-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> We associated to each TF the corresponding Position-specific Weight Matrix (PWM), i.e. the matrix reporting the frequency of nucleotides for each position of the experimentally validated binding sites. It is worth noting that a TF can be associated to different PWMs and, on the other hand, a PWM can be associated to more than one TF. A total number of 192 PWMs was considered in this work and each of them was associated in a biunivocal relation to one TF with only a few exceptions. For the sake of simplicity and readability, we refer to PWM models as TFs and vice versa. The PWMs matrices were retrieved from an open access repository available at the following address <a class="link link-uri" href="http://cistrome.org/&amp;sim;jian/motif_collection/databases/Transfac/pwm/" target="_blank">http://cistrome.org/∼jian/motif_collection/databases/Transfac/pwm/</a> (12 October 2017, date last accessed). We applied the matchPWM() function, a sequence-based transcription factor binding site search algorithm, integrated into the Biostrings R library,<a reveal-id="dsx041-B20" data-open="dsx041-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> to extract and collect the positions list of all binding sites for both all considered transcripts and PWMs. For our purposes, we have considered an acceptance threshold probability of 0.9. The described procedure is clearly summarized in the workflow in <a reveal-id="dsx041-F1" data-open="dsx041-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. </p>                        <a id="112483377" scrollto-destination="112483377"></a>
<div id="dsx041-F1" data-id="dsx041-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f1.png?Expires=2147483647&Signature=Crk9L6Ijb4cfXmt0yqtm5hGpO7QUibrr2Vp0OeKxA5jb6maJR9BQVKwc4N~Vz2TIGH7Yemh6y-b4kHMXBOHxRlZb3gHeBpeNBqrefXtefAwxztBT~WyAUgr-qHgZk2jy8Sd~6YSD6k5nWziYjCCVwOnlLG4O6MbfbGjMQOHzOiLnY1UQR0qSO1Ody8WyQth7m7jUkLfhCu4j7zSBmm9lzM3sWqTLGQfnwgrzpvPkWxdVQ-qUw6jZEf28K4kyJxI~61KFRPpCLxZI7pGnlFyROYWT15EgwHPTqsUY9wmYjd9dfXNJCrSxMONSqvUk9WLSprKvNC1eIzPjbAZDDwYedw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow for the extraction of human TFBSs." path-from-xml="dsx041f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483377/dsx041f1.tif" path-from-xml="dsx041f1.tif" target="_blank">View large</a><a section="112483377" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f1.png?Expires=2147483647&Signature=C3FkhpdfiIcfUBFfjMshP9ks-oP8PSfp0~QMJcCP7405dwY8tSAgLpvtmeGPlvJ3LEFURPFybnI7ZXWJ0JDPmTOpfKgEkOxa61Eoy99zPBACq5f7oFL28NWebtOVRO0mmXbS~q2t8YBGkKfRCSzdX8WaAkFxhmyGbwsfhasgWzsGh4lQSgYKnP9e6TQLNHzRjHO8CvbozVV2C4BKg7s3Sa3HV8tf9F-B0n0vF4LTwuAmGbwDnGfZGWCvAe-vkduxmIsmZzSyvMTy75KHdK0UFQf4NXnDnXjM0WFxEOGz4DJjCNDVOicJDl~a6dB5Qx88YvOvOeFRdU1DALPmh9wldQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483377&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow for the extraction of human TFBSs.</p></div></div><div content-id="dsx041-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f1.png?Expires=2147483647&Signature=Crk9L6Ijb4cfXmt0yqtm5hGpO7QUibrr2Vp0OeKxA5jb6maJR9BQVKwc4N~Vz2TIGH7Yemh6y-b4kHMXBOHxRlZb3gHeBpeNBqrefXtefAwxztBT~WyAUgr-qHgZk2jy8Sd~6YSD6k5nWziYjCCVwOnlLG4O6MbfbGjMQOHzOiLnY1UQR0qSO1Ody8WyQth7m7jUkLfhCu4j7zSBmm9lzM3sWqTLGQfnwgrzpvPkWxdVQ-qUw6jZEf28K4kyJxI~61KFRPpCLxZI7pGnlFyROYWT15EgwHPTqsUY9wmYjd9dfXNJCrSxMONSqvUk9WLSprKvNC1eIzPjbAZDDwYedw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow for the extraction of human TFBSs." path-from-xml="dsx041f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483377/dsx041f1.tif" path-from-xml="dsx041f1.tif" target="_blank">View large</a><a section="112483377" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f1.png?Expires=2147483647&Signature=C3FkhpdfiIcfUBFfjMshP9ks-oP8PSfp0~QMJcCP7405dwY8tSAgLpvtmeGPlvJ3LEFURPFybnI7ZXWJ0JDPmTOpfKgEkOxa61Eoy99zPBACq5f7oFL28NWebtOVRO0mmXbS~q2t8YBGkKfRCSzdX8WaAkFxhmyGbwsfhasgWzsGh4lQSgYKnP9e6TQLNHzRjHO8CvbozVV2C4BKg7s3Sa3HV8tf9F-B0n0vF4LTwuAmGbwDnGfZGWCvAe-vkduxmIsmZzSyvMTy75KHdK0UFQf4NXnDnXjM0WFxEOGz4DJjCNDVOicJDl~a6dB5Qx88YvOvOeFRdU1DALPmh9wldQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483377&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow for the extraction of human TFBSs.</p></div></div>                            <h3 scrollto-destination=112483378 id="112483378" class="section-title" >2.2. Transcription factor co-localization and deviation from randomness</h3>
<div class="block-child-p">In this Subsection, we present the indicator we have developed in order to determine whether two different TFs are linked with respect to a given promoter. Let <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mo xmlns="">=</mo><mfenced open="{" close="|" separators="|" xmlns=""><mrow><msub><mrow><mi>x</mi></mrow><mrow><mn>1</mn></mrow></msub><msub><mrow><mi>x</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>…</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>L</mi></mrow></msub><mi> </mi></mrow></mfenced><msub xmlns=""><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub><mi xmlns=""> </mi><mo xmlns="">∈</mo><mfenced open="{" close="}" separators="|" xmlns=""><mrow><mi>A</mi><mo>,</mo><mi>C</mi><mo>,</mo><mi>G</mi><mo>,</mo><mi>T</mi></mrow></mfenced><mo xmlns="">}</mo></math></span> be the promoter region of a given transcript, i.e. the set of <em>L</em> = 2,000 bases upstream the TSS of that transcript. Using PWM<sub><em>i</em></sub> and PWM<sub><em>j</em></sub>, i.e. the matrices associated to two different transcription factors TF<sub><em>i</em></sub> and TF<sub><em>j</em></sub>, we select, in the promoter <em>P</em>, all the TFBSs for TF<sub><em>i</em></sub> and TF<sub><em>j</em></sub> (see the previous Subsection for details) and we collect them in two sets, {TFBS<sub><em>i</em></sub>} and {TFBS<sub><em>j</em></sub>}, respectively. The idea is to compare the number of couples of elements from {TFBS<sub><em>i</em></sub>} and {TFBS<sub><em>j</em></sub>} whose distance is closer than <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span>—where we set <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi><mo xmlns="">=</mo><mn xmlns="">80</mn><mi xmlns=""> </mi><mi mathvariant="normal" xmlns="">b</mi><mi mathvariant="normal" xmlns="">p</mi></math></span>—with the expected number of couples obtained by setting at random the occurrences of {TFBS<sub><em>i</em></sub>} and {TFBS<sub><em>j</em></sub>}. As first we consider the case of random distribution of elements in {TFBS<sub><em>i</em></sub>} and {TFBS<sub><em>j</em></sub>} and we compute the expected number of joined occurrences within a distance <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math>⁠</span>. If there is just one binding site for factors TF<sub><em>i</em></sub> and TF<sub><em>j</em></sub> (i.e. there is just an element <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">x</mi><mo xmlns="">∈</mo><mo xmlns="">{</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">}</mo></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">y</mi><mo xmlns="">∈</mo><mo xmlns="">{</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">}</mo></math>⁠</span>) put at random in a promoter of length <em>L</em>, the probability that their distance is exactly <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> is <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mfenced separators="|" xmlns=""><mrow><mfenced open="|" close="|" separators="|"><mrow><mi>x</mi><mo>-</mo><mi>y</mi></mrow></mfenced><mo>=</mo><mi>ℓ</mi></mrow></mfenced><mo xmlns="">=</mo><mn xmlns="">2</mn><mfrac xmlns=""><mrow><mi>L</mi><mo>-</mo><mi>ℓ</mi></mrow><mrow><mi>L</mi><mo>(</mo><mi>L</mi><mo>-</mo><mn>1</mn><mo>)</mo></mrow></mfrac></math></div></div><div class="label">(1)</div></div></div><div class="block-child-p">A simple computation gives the probability that their distance is less than or equal to <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span><a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mfenced separators="|" xmlns=""><mrow><mfenced open="|" close="|" separators="|"><mrow><mi>x</mi><mo>-</mo><mi>y</mi></mrow></mfenced><mo>≤</mo><mi>ℓ</mi></mrow></mfenced><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mfenced separators="|"><mrow><mn>2</mn><mi>L</mi><mo>-</mo><mn>1</mn></mrow></mfenced><mi>ℓ</mi><mo>-</mo><msup><mrow><mi>ℓ</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><mi>L</mi><mo>(</mo><mi>L</mi><mo>-</mo><mn>1</mn><mo>)</mo></mrow></mfrac></math></div></div><div class="label">(2)</div></div></div><div class="block-child-p">Going further, the probability that one element in {TFBS<sub><em>i</em></sub>} and several element {TFBS<sub><em>j</em></sub>}, say <em>n<sub>j</sub></em>, put at random in a promoter of length <em>L</em>, have a distance less than or equal to <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> is well-approximated by considering the probability of a single matching <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mfenced separators="|" xmlns=""><mrow><mfenced open="|" close="|" separators="|"><mrow><mi>x</mi><mo>-</mo><mi>y</mi></mrow></mfenced><mo>≤</mo><mi>ℓ</mi></mrow></mfenced><mo xmlns="">=</mo><mi xmlns="">p</mi></math></span> as a Bernoulli event, and the probability of having <em>k</em> elements of {TFBS<sub><em>j</em></sub>} in an interval of size <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> around the element <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">x</mi><mo xmlns="">∈</mo><mo xmlns="">{</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">}</mo></math></span> is <a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mfenced separators="|" xmlns=""><mrow><mi>k</mi><mo>;</mo><mfenced open="|" close="|" separators="|"><mrow><mi>x</mi><mo>-</mo><mi>y</mi></mrow></mfenced><mo>≤</mo><mi>ℓ</mi></mrow></mfenced><mo xmlns="">=</mo><mfenced separators="|" xmlns=""><mrow><mtable><mtr><mtd><msub><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub></mtd></mtr><mtr><mtd><mi>k</mi></mtd></mtr></mtable></mrow></mfenced><msup xmlns=""><mrow><mi>p</mi></mrow><mrow><mi>k</mi></mrow></msup><msup xmlns=""><mrow><mo>(</mo><mn>1</mn><mo>-</mo><mi>p</mi><mo>)</mo></mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>-</mo><mi>k</mi></mrow></msup></math></div></div><div class="label">(3)</div></div></div><div class="block-child-p">Finally, if there is more than one element in {TFBS<sub><em>i</em></sub>}, say <em>n<sub>i</sub></em>, the expected number of occurrences of couple of elements in {TFBS<sub><em>i</em></sub>} and {TFBS<sub><em>j</em></sub>} within a distance <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> can be approximated by <a id="E4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">n</mi><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo xmlns="">=</mo><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub><mrow xmlns=""><munderover><mo stretchy="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>0</mn></mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></munderover><mrow><mi>k</mi><mi>P</mi><mfenced separators="|"><mrow><mi>k</mi><mo>;</mo><mfenced open="|" close="|" separators="|"><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>-</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo>≤</mo><mi>ℓ</mi></mrow></mfenced></mrow></mrow><mo xmlns="">=</mo><mo xmlns="">=</mo><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub><mi xmlns="">p</mi><mo xmlns="">=</mo><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub><mfrac xmlns=""><mrow><mfenced separators="|"><mrow><mn>2</mn><mi>L</mi><mo>-</mo><mn>1</mn></mrow></mfenced><mi>ℓ</mi><mo>-</mo><msup><mrow><mi>ℓ</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><mi>L</mi><mo>(</mo><mi>L</mi><mo>-</mo><mn>1</mn><mo>)</mo></mrow></mfrac></math></div></div><div class="label">(4)</div></div> while its variance is <a id="E5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><msup xmlns=""><mrow><mi>σ</mi></mrow><mrow><mn>2</mn></mrow></msup><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo xmlns="">=</mo><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub><mfenced open="[" close="]" separators="|" xmlns=""><mrow><mrow><munderover><mo stretchy="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>0</mn></mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></munderover><mrow><msup><mrow><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msup><mi>P</mi><mfenced separators="|"><mrow><mi>k</mi><mo>;</mo><mfenced open="|" close="|" separators="|"><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>-</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo>≤</mo><mi>ℓ</mi></mrow></mfenced><mo>-</mo><mi>n</mi><msup><mrow><mfenced separators="|"><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></mrow></mrow></mfenced><mo xmlns="">=</mo><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub><mi xmlns="">p</mi><mfenced separators="|" xmlns=""><mrow><mn>1</mn><mo>-</mo><mi>p</mi></mrow></mfenced><mo xmlns="">=</mo><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>j</mi></mrow></msub><mfrac xmlns=""><mrow><mfenced open="[" close="]" separators="|"><mrow><mfenced separators="|"><mrow><mn>2</mn><mi>L</mi><mo>-</mo><mn>1</mn></mrow></mfenced><mi>ℓ</mi><mo>-</mo><msup><mrow><mi>ℓ</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfenced><mo>[</mo><msup><mrow><mfenced separators="|"><mrow><mi>L</mi><mo>-</mo><mi>ℓ</mi></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>-</mo><mfenced separators="|"><mrow><mi>L</mi><mo>-</mo><mi>ℓ</mi></mrow></mfenced><mo>]</mo></mrow><mrow><msup><mrow><mi>L</mi></mrow><mrow><mn>2</mn></mrow></msup><msup><mrow><mo>(</mo><mi>L</mi><mo>-</mo><mn>1</mn><mo>)</mo></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac></math></div></div><div class="label">(5)</div></div> The Bernoulli approximations in <a href="#E4" class="sectionLink xref-sec">Equations (4)</a> and <a href="#E5" class="sectionLink xref-sec">(5)</a> are valid when <em>n</em><sub>i</sub> and <em>n</em><sub>j</sub> are small with respect to <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> is small with respect to L. By computing the actual number of couples (x, y), say <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ν</mi><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">)</mo></math>⁠</span>, with <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">x</mi><mo xmlns="">∈</mo><mo xmlns="">{</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">}</mo></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">y</mi><mo xmlns="">∈</mo><mo xmlns="">{</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">}</mo></math></span> such that <em>x</em> and <em>y</em> are closer than <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math>⁠</span>, and using <a href="#E4" class="sectionLink xref-sec">Equations (4)</a> and <a href="#E5" class="sectionLink xref-sec">(5)</a> we build an indicator evaluating the deviation from randomness of the actual number of join occurrences <a id="E6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E6"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">z</mi><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mi>ν</mi><mfenced separators="|"><mrow><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo>-</mo><mi>n</mi><mo>(</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>)</mo></mrow><mrow><mi>σ</mi><mo>(</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>)</mo></mrow></mfrac></math></div></div><div class="label">(6)</div></div></div><p>The above <em>z</em>-score function, measuring how many standard deviations the actual number of couples is far from the expected number of couples in the case of a random distribution, provides a reasonable value of the association between TFBS<sub><em>i</em></sub> and TFBS<sub><em>j</em></sub>. Significantly, high values of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">z</mi><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced></math></span> suggest that the two considered TFs are associated, i.e. their TFBSs have a probability to occur as neighbours higher than the one expected for a random distribution. On the contrary when <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">z</mi><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced></math></span> is negative, with a high absolute value, elements in the two TFBSs occur as neighbours less frequently than expected in a random distribution, so that in this case the TFs are disassociated. In the following, in order to identify the synergy between transcription factors, we will focus on the case of high positive values of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">z</mi><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced></math>⁠</span>.</p>                            <h3 scrollto-destination=112483384 id="112483384" class="section-title" >2.3. Computing similarity score of PWMs</h3>
<p>The prediction algorithm of TFBSs is based on the PWMs obtained by experimental analysis. Since we are interested in identifying couples of TFs whose TFBSs occur together in a window of a given size, we have to be sure that the two corresponding models are enough different since if two models are very similar they are likely to share binding sites or a significant portion of them. We used two different approaches to evaluate models similarity. The former is a direct approach aiming to provide a similarity pairwise score of two models based on the comparison of nucleotide frequencies of the PWMs. The latter approach can be defined as ‘a posteriori’, since we evaluate the closeness of models by computing the number of overlapped binding sites with respect to the total number of couples whose distance is closer than <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math>⁠</span>. Using a linear combination of the above described similarity scores we determine if two models are ‘structurally’ far from each other and we can significantly evaluate the association between two truly different models according to the algorithm described in the previous section.</p>                            <h3 scrollto-destination=112483386 id="112483386" class="section-title" >2.4. PWMs distance based on Jensen-Shannon divergence</h3>
<div class="block-child-p">The Jensen-Shannon divergence (JS) is a symmetrized and smoothed version of the Kullback-Leibler divergence and it is often used to tell how two frequency distributions are close to each other.<a reveal-id="dsx041-B21" data-open="dsx041-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Here, we use Jensen-Shannon divergence to identify similar PWMs using the nucleotide frequencies as probability distribution. The Jensen-Shannon divergence of two frequency distributions, <em>P</em> and <em>Q</em>, can be computed using the Shannon entropy according to the following equation: <a id="E7" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E7"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">J</mi><mi mathvariant="normal" xmlns="">S</mi><mfenced separators="|" xmlns=""><mrow><mi>P</mi><mo>,</mo><mi>Q</mi></mrow></mfenced><mo xmlns="">=</mo><mi xmlns="">H</mi><mfenced separators="|" xmlns=""><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mi>P</mi><mo>+</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mi>Q</mi></mrow></mfenced><mo xmlns="">-</mo><mfrac xmlns=""><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mi xmlns="">H</mi><mfenced separators="|" xmlns=""><mrow><mi>P</mi></mrow></mfenced><mo xmlns="">-</mo><mfrac xmlns=""><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mi xmlns="">H</mi><mo xmlns="">(</mo><mi xmlns="">Q</mi><mo xmlns="">)</mo></math></div></div><div class="label">(7)</div></div></div><div class="block-child-p">where <em>H</em>(<em>X</em>) is the Shannon entropy of the distribution <em>X</em><a id="E8" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E8"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">H</mi><mfenced separators="|" xmlns=""><mrow><mi>X</mi></mrow></mfenced><mo xmlns="">=</mo><mo xmlns="">-</mo><mrow xmlns=""><munderover><mo stretchy="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>N</mi></mrow></munderover><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>k</mi></mrow></msub><mi mathvariant="normal">l</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">g</mi><mo>⁡</mo><mo>(</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>)</mo></mrow></mrow></math></div></div></div></div><div class="block-child-p">Given a couple of models, PWM<sub>1</sub> and PWM<sub>2</sub>, we indicate with <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mi>P</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>i</mi></mrow></msubsup></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mi>Q</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>j</mi></mrow></msubsup></math></span> the frequency of the nucleotide <em>k</em> (the index <em>k</em> ranges from 1 to 4 indicating the four nucleotides A, C, G, T) in the position i of the model PWM<sub>1</sub> (the index <em>i</em> ranges from 1 to <em>I</em>, where <em>I</em> is the length of the model PWM<sub>1</sub>) and position <em>j</em> of the model PWM<sub>2</sub> (the index <em>j</em> ranges from 1 to <em>J</em>, where <em>J</em> is the length of the model PWM<sub>2</sub>), respectively. For example, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mi>P</mi></mrow><mrow><mn>4</mn></mrow><mrow><mn>1</mn></mrow></msubsup></math></span> is the frequency of the nucleotide T in position 1 of the first model while <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mi>Q</mi></mrow><mrow><mn>3</mn></mrow><mrow><mn>2</mn></mrow></msubsup></math></span> is the frequency of the nucleotide G in position 2 of the second model. The Jensen-Shannon divergence of the nucleotide distributions at position <em>i</em> of the first model and at position <em>j</em> of the second model is defined as <a id="E9" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E9"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">J</mi><mi mathvariant="normal" xmlns="">S</mi><mfenced separators="|" xmlns=""><mrow><mi>P</mi><mo>,</mo><mi>Q</mi><mo>;</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></mfenced><mo xmlns="">=</mo><mo xmlns="">-</mo><mrow xmlns=""><munderover><mo stretchy="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>4</mn></mrow></munderover><mrow><mfrac><mrow><msubsup><mrow><mi>P</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mo>+</mo><msubsup><mrow><mi>Q</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>j</mi></mrow></msubsup></mrow><mrow><mn>2</mn></mrow></mfrac></mrow></mrow><mrow xmlns=""><mrow><mi mathvariant="normal">log</mi></mrow><mo>⁡</mo><mrow><mfrac><mrow><msubsup><mrow><mi>P</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mo>+</mo><msubsup><mrow><mi>Q</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>j</mi></mrow></msubsup></mrow><mrow><mn>2</mn></mrow></mfrac></mrow></mrow><mo xmlns="">+</mo><mfrac xmlns=""><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mrow xmlns=""><munderover><mo stretchy="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>4</mn></mrow></munderover><mrow><msubsup><mrow><mi>P</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mrow><mrow><mi mathvariant="normal">log</mi></mrow><mo>⁡</mo><mrow><msubsup><mrow><mi>P</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>i</mi></mrow></msubsup></mrow></mrow></mrow></mrow><mo xmlns="">+</mo><mrow xmlns=""><munderover><mo stretchy="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>4</mn></mrow></munderover><mrow><msubsup><mrow><mi>Q</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>j</mi></mrow></msubsup><mrow><mrow><mi mathvariant="normal">log</mi></mrow><mo>⁡</mo><mrow><msubsup><mrow><mi>Q</mi></mrow><mrow><mi>k</mi></mrow><mrow><mi>j</mi></mrow></msubsup></mrow></mrow></mrow></mrow></math></div></div><div class="label">(8)</div></div></div><div class="block-child-p">We measure the similarity of two models with respect to a given alignment by summing up the Jensen-Shannon divergences as in <a href="#E9" class="sectionLink xref-sec">Equation (8)</a>) of each couple of distributions corresponding to the aligned positions (empty boxes in the overlapping area, see <a reveal-id="dsx041-F2" data-open="dsx041-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>) (<em>i</em>, <em>i</em> + a), where a is the offset between the starting positions of model 1 and 2 in that alignment, plus a penalty score for the non overlapped positions (“p” marked boxes in the penalty area, see <a reveal-id="dsx041-F2" data-open="dsx041-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>): <a id="E10" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E10"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">J</mi><mi mathvariant="normal" xmlns="">S</mi><mfenced separators="|" xmlns=""><mrow><mi>P</mi><mo>,</mo><mi>Q</mi><mo>;</mo><mi>a</mi></mrow></mfenced><mo xmlns="">=</mo><mrow xmlns=""><munder><mo stretchy="false">∑</mo><mrow><mi>i</mi><mo>∈</mo><mi>I</mi></mrow></munder><mrow><mi mathvariant="normal">J</mi><mi mathvariant="normal">S</mi><mfenced separators="|"><mrow><mi>P</mi><mo>,</mo><mi>Q</mi><mo>;</mo><mi>i</mi><mo>,</mo><mi>i</mi><mo>+</mo><mi>a</mi></mrow></mfenced><mo>+</mo><mi>K</mi><msub><mrow><mi>n</mi></mrow><mrow><mi>o</mi></mrow></msub></mrow></mrow></math></div></div><div class="label">(9)</div></div> where <em>I</em> is the set of overlapped positions in the given alignment and <em>n</em><sub>o</sub> is the number of non overlapped positions of the shortest model. The penalty <em>K</em> for each of non overlapped positions is determined by computing the probability distribution of the Jensen-Shannon divergence in the case of two random nucleotide distributions. The value <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">K</mi><mo xmlns="">≃</mo><mn xmlns="">0.23</mn></math></span> is obtained by summing up 2 S.D. to the mean value of the random-nucleotide Jensen-Shannon divergence distributions. Finally, the model similarity score of two models JSD(<em>P</em>, <em>Q</em>) is defined as the smallest score JS(<em>P</em>, <em>Q</em>; <em>a</em>) out of all the possible alignments: <a id="E11" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E11"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">J</mi><mi mathvariant="normal" xmlns="">S</mi><mi mathvariant="normal" xmlns="">D</mi><mfenced separators="|" xmlns=""><mrow><mi>P</mi><mo>,</mo><mi>Q</mi></mrow></mfenced><mo xmlns="">=</mo><mrow xmlns=""><mrow><munder><mrow><mi mathvariant="normal">min</mi></mrow><mrow><mi>a</mi><mo>∈</mo><mi mathvariant="normal">AL</mi></mrow></munder></mrow><mo>⁡</mo><mrow><mo>{</mo><mi mathvariant="normal">J</mi><mi mathvariant="normal">S</mi><mo>(</mo><mi>P</mi><mo>,</mo><mi>Q</mi><mo>;</mo><mi>a</mi><mo>)</mo><mo>}</mo></mrow></mrow></math></div></div><div class="label">(10)</div></div> where AL is the set of all possible alignments with a non-empty superposition. The need of a ‘strong’' non-overlapping penalty comes out in order to select alignment with a significant overlapping between the two models (i.e. to avoid bias due to a good affinity between too small part of the two models). </div>                        <a id="112483391" scrollto-destination="112483391"></a>
<div id="dsx041-F2" data-id="dsx041-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f2.png?Expires=2147483647&Signature=qSSTxkOot41~JrPtI-lptBVWdDrqRfQwx5o2~TUq59M-uLkCBEyyWNjiGse6f44PCyv1E2ASuRDsbpctiJ9rcUegaIwOazRt-ZQOrFYcTHAezaA26YJJlMm2~On6Cm~pHD-5wF5YKgEwIVKgmKjBeeprbhr~HiK9~xo3zFqfdlJZVSxp~XGthl2p7F3-akeoDbbmhGBZomD0uNQnBnTBntA6a50SPT18EhtO0EAErqr7UlJNPMTG00n8xEL0HTVt5nSVcZBFKuUAeuOJJ6sZdvZrGwUE5xkoAUBBg-ROyaoZiKrHW17eYtAqPe6cuI9KylZ0CR4GXopWjz0OQLvccw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme of the algorithm for computing similarity score of models." path-from-xml="dsx041f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483391/dsx041f2.tif" path-from-xml="dsx041f2.tif" target="_blank">View large</a><a section="112483391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f2.png?Expires=2147483647&Signature=ICgHOnD8U~AvQgFwuXX618mfq8s~jQyLcB0gXQjbCUWFcgecqRmIkHCj2nTKI5~GkiumK9OHMhnAHXHWnbhT8lnoVPjP6xfEmYQ5MxjVF~tWJFvoLSNtqNZ4qF700tObsrwbjAyc09jwCFicxDQoZMf5M1iSBNQbPmZ-LQxZx-WgFoHsabBWbazGRoyoTmXmql~BMwFzcPgTCC4QtJ~610hT4Ubs9RxZQ4DQk3fAVw8gFbjCJIojHdvHaNfVPFLFAnC2A8XMxov7FhKextmt2KTjYjKFr5Kvpql3-jg7rUda8iTtDGk1RyKf0qLdTVYajkns2LyDsemtCJ3C7TiHaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483391&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scheme of the algorithm for computing similarity score of models.</p></div></div><div content-id="dsx041-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f2.png?Expires=2147483647&Signature=qSSTxkOot41~JrPtI-lptBVWdDrqRfQwx5o2~TUq59M-uLkCBEyyWNjiGse6f44PCyv1E2ASuRDsbpctiJ9rcUegaIwOazRt-ZQOrFYcTHAezaA26YJJlMm2~On6Cm~pHD-5wF5YKgEwIVKgmKjBeeprbhr~HiK9~xo3zFqfdlJZVSxp~XGthl2p7F3-akeoDbbmhGBZomD0uNQnBnTBntA6a50SPT18EhtO0EAErqr7UlJNPMTG00n8xEL0HTVt5nSVcZBFKuUAeuOJJ6sZdvZrGwUE5xkoAUBBg-ROyaoZiKrHW17eYtAqPe6cuI9KylZ0CR4GXopWjz0OQLvccw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme of the algorithm for computing similarity score of models." path-from-xml="dsx041f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483391/dsx041f2.tif" path-from-xml="dsx041f2.tif" target="_blank">View large</a><a section="112483391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f2.png?Expires=2147483647&Signature=ICgHOnD8U~AvQgFwuXX618mfq8s~jQyLcB0gXQjbCUWFcgecqRmIkHCj2nTKI5~GkiumK9OHMhnAHXHWnbhT8lnoVPjP6xfEmYQ5MxjVF~tWJFvoLSNtqNZ4qF700tObsrwbjAyc09jwCFicxDQoZMf5M1iSBNQbPmZ-LQxZx-WgFoHsabBWbazGRoyoTmXmql~BMwFzcPgTCC4QtJ~610hT4Ubs9RxZQ4DQk3fAVw8gFbjCJIojHdvHaNfVPFLFAnC2A8XMxov7FhKextmt2KTjYjKFr5Kvpql3-jg7rUda8iTtDGk1RyKf0qLdTVYajkns2LyDsemtCJ3C7TiHaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483391&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scheme of the algorithm for computing similarity score of models.</p></div></div>                            <h3 scrollto-destination=112483392 id="112483392" class="section-title" >2.5. PWMs distance based on TFBS overlapping</h3>
<div class="block-child-p">Here, we introduce a different similarity function between pairs of TFs based on the percentage of overlapped binding sites they share. First of all, we consider two TFBSs to be overlapped if their distance is smaller than the half of the size of the smaller PWM, i.e. <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfenced open="|" close="|" separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>-</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi><mi mathvariant="normal">B</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo xmlns="">&lt;</mo><mfrac xmlns=""><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mo>⁡</mo><mo>{</mo><msub><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">W</mi><mi mathvariant="normal">M</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><mi> </mi><mi mathvariant="normal"> </mi><msub><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">W</mi><mi mathvariant="normal">M</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>}</mo></mrow><mrow><mn>2</mn></mrow></mfrac></math>⁠</span>. If the TFBSs of two TSs are often overlapped it is likely that they result to be associated, according to our algorithm, showing a high <em>Z</em>-score. They are indeed often close because the TFs recognize the same binding sites (or a large portion of them). Given a couple of TFs, TF<sub><em>i</em></sub> and TF<sub><em>j</em></sub>, the percentage of all overlapping occurrences of TFBSs with respect to all the joined occurrences in a radius <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> in all the transcript promoter sequences, namely OVD(TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>), is defined as follows: <a id="E12" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E12"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">O</mi><mi mathvariant="normal" xmlns="">V</mi><mi mathvariant="normal" xmlns="">D</mi><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo xmlns="">=</mo><mfrac xmlns=""><mrow><msub><mrow><mi>μ</mi></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi></mrow></msub><mo>(</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>)</mo></mrow><mrow><msub><mrow><mi>ν</mi></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi></mrow></msub><mo>(</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>)</mo></mrow></mfrac></math></div></div><div class="label">(11)</div></div> where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>ν</mi></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi></mrow></msub><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">)</mo></math></span> is the total number of the TFBSs couples (one from TF<sub><em>i</em></sub> and the other from TF<sub><em>j</em></sub>) falling together in the considered radius of size <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>μ</mi></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi></mrow></msub><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">)</mo></math></span> is the total number of TFBSs overlapped couples (one from TF<sub><em>i</em></sub> and the other from TF<sub><em>j</em></sub>), for all the transcript promoter sequences. In the next section (as reported in <a reveal-id="dsx041-F3" data-open="dsx041-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), OVD scores are plotted against JSD scores in order to select significant couples. </div>                        <a id="112483394" scrollto-destination="112483394"></a>
<div id="dsx041-F3" data-id="dsx041-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f3.png?Expires=2147483647&Signature=L410ZqADe8y1rfUhOgoFv1jit3b9g2CX5EErh0KrsibMXUlwo8c1ev-fqFHj5lMgq5ryuS9P2iKcs24hiRhYdnvVvP8bA96KBFaqGF6RTTd-gXapgIdNLI9SxApOWi20v3HGpmjhe10ejUT~KrsniYLGwFo6U2c4cndTvBn2MV9-n~tt6C7RB44qeySCekXAiqDtp4kE4UTQmX~X9q6s5JJeKsaP3aPJaqCj5SVuvbylfNdzzUekJJ1XmTRLlyym3zV07TP5J4aPbBb03suU6NXPw842fTZ53P34nJ8P6D0COHFzmQZJzJa3guKI3U4lFH4MEkBit88wfCCgPN2lcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Each point represents a TF couple whose Z-scores are significant (Z &gt; 5) for at least 20 transcripts. The plane is divided into two regions: the upper-left sector, where Overlapping Score is higher than Similarity Score, provides selected couples. Lower-right sector, where Overlapping Score is lower than Similarity Score, provides Discarded couples." path-from-xml="dsx041f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483394/dsx041f3.tif" path-from-xml="dsx041f3.tif" target="_blank">View large</a><a section="112483394" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f3.png?Expires=2147483647&Signature=2USdnOSaIkpYUT30nuR~LJL3Q2Vg9yFaq2BXIiiqox6V~P8b8hxwL~rpdZj2Gh6iNbCIEmCeVXRpTe7xRY0GQpUjmVVCg4qc1XaA1IHeb2iWfwDI4qmYgNaYbVV28VAOl97dN8qwVzJQVjE5uS0ayzZhVtHp~faITqOg8hoMcMbhnQc-DsetVt~zKsZTOnBGCoCbHeB4H9iZe3eK9KyDtf9~ZiyLAvDyUqC4YsZ8Mq3kUVQBIpOaQ~Gt-x62scpyHymd6P6LBcC2WVw6yK6nR~fLJPw4-5o0i6GmLxKcCblFcACxV7ejNkT7NFdhDU~iM1uZFJw0lnftW2jRai5ELQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483394&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Each point represents a TF couple whose <em>Z</em>-scores are significant (<em>Z</em> &gt; 5) for at least 20 transcripts. The plane is divided into two regions: the upper-left sector, where Overlapping Score is higher than Similarity Score, provides selected couples. Lower-right sector, where Overlapping Score is lower than Similarity Score, provides Discarded couples.</p></div></div><div content-id="dsx041-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f3.png?Expires=2147483647&Signature=L410ZqADe8y1rfUhOgoFv1jit3b9g2CX5EErh0KrsibMXUlwo8c1ev-fqFHj5lMgq5ryuS9P2iKcs24hiRhYdnvVvP8bA96KBFaqGF6RTTd-gXapgIdNLI9SxApOWi20v3HGpmjhe10ejUT~KrsniYLGwFo6U2c4cndTvBn2MV9-n~tt6C7RB44qeySCekXAiqDtp4kE4UTQmX~X9q6s5JJeKsaP3aPJaqCj5SVuvbylfNdzzUekJJ1XmTRLlyym3zV07TP5J4aPbBb03suU6NXPw842fTZ53P34nJ8P6D0COHFzmQZJzJa3guKI3U4lFH4MEkBit88wfCCgPN2lcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Each point represents a TF couple whose Z-scores are significant (Z &gt; 5) for at least 20 transcripts. The plane is divided into two regions: the upper-left sector, where Overlapping Score is higher than Similarity Score, provides selected couples. Lower-right sector, where Overlapping Score is lower than Similarity Score, provides Discarded couples." path-from-xml="dsx041f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483394/dsx041f3.tif" path-from-xml="dsx041f3.tif" target="_blank">View large</a><a section="112483394" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f3.png?Expires=2147483647&Signature=2USdnOSaIkpYUT30nuR~LJL3Q2Vg9yFaq2BXIiiqox6V~P8b8hxwL~rpdZj2Gh6iNbCIEmCeVXRpTe7xRY0GQpUjmVVCg4qc1XaA1IHeb2iWfwDI4qmYgNaYbVV28VAOl97dN8qwVzJQVjE5uS0ayzZhVtHp~faITqOg8hoMcMbhnQc-DsetVt~zKsZTOnBGCoCbHeB4H9iZe3eK9KyDtf9~ZiyLAvDyUqC4YsZ8Mq3kUVQBIpOaQ~Gt-x62scpyHymd6P6LBcC2WVw6yK6nR~fLJPw4-5o0i6GmLxKcCblFcACxV7ejNkT7NFdhDU~iM1uZFJw0lnftW2jRai5ELQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483394&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Each point represents a TF couple whose <em>Z</em>-scores are significant (<em>Z</em> &gt; 5) for at least 20 transcripts. The plane is divided into two regions: the upper-left sector, where Overlapping Score is higher than Similarity Score, provides selected couples. Lower-right sector, where Overlapping Score is lower than Similarity Score, provides Discarded couples.</p></div></div>                            <h3 scrollto-destination=112483395 id="112483395" class="section-title" >2.6. Protein-protein interaction network analysis</h3>
<p>In order to validate the synergy of TFs couples identified by the algorithm here presented we analysed Protein-Protein Interaction network (PPI) to test the interaction of this couples in the network by using shortest path.</p><p>Protein-Protein Interaction network was downloaded from STRING database.<a reveal-id="dsx041-B22" data-open="dsx041-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> We considered all the possible interaction sources between proteins giving a score threshold of 0.7 (high confidence interaction). The obtained network is made of 719,288 edges (interactions) and 14,932 nodes (proteins). The iGraph R package (<a class="link link-uri" href="https://cran.r-project.org/web/packages/igraph/index.html" target="_blank">https://cran.r-project.org/web/packages/igraph/index.html</a> (12 October 2017, date last accessed)) was used to compute the shortest path from the adjacency matrix of the graph.</p>                            <h2 scrollto-destination=112483398 id="112483398" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=112483399 id="112483399" class="section-title" >3.1. Selecting significant TF couples</h3>
<p>In the previous Section, <a href="#E6" class="sectionLink xref-sec">Equation (6)</a> has been derived in order to provide a measure of association between each couple of transcription factors (TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>) with respect to a given transcript TR<sub><em>k</em></sub>. Our goal is to find out couples of TFs that are significantly related in gene-regulation mechanism. As previously mentioned, the higher the value of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">z</mi><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">;</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">R</mi></mrow><mrow><mi>k</mi></mrow></msub><mo xmlns="">)</mo></math>⁠</span>, the higher the association between TF<sub><em>i</em></sub> and TF<sub><em>j</em></sub> in the transcript promoter TR<sub><em>k</em></sub> sequence will be. Since for each transcript, we have to test a considerably large number of couples (TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>) and we test the association between couples in a large number of transcripts, we select only those couples for which the value of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">z</mi><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">;</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">R</mi></mrow><mrow><mi>k</mi></mrow></msub><mo xmlns="">)</mo></math></span> is higher than 5 (meaning that the number of occurrence of couples TF<sub><em>i</em></sub> TF<sub><em>j</em></sub> in a radius <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">ℓ</mi></math></span> is far from that expected by chance more than 5 S.D.) and we indicate with <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">Z</mi></mrow><mrow><mn>5</mn></mrow></msub><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">)</mo></math></span> the number of transcripts TR<sub><em>k</em></sub> for which the couple (TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>) has a <em>z</em>-score greater than 5. Given a couple (TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>) it is reasonable to expect that a large value of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">Z</mi></mrow><mrow><mn>5</mn></mrow></msub><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">)</mo></math></span> indicates that the two models TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub> are functionally associated. However, a high value of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">Z</mi></mrow><mrow><mn>5</mn></mrow></msub><mo xmlns="">(</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo xmlns="">,</mo><msub xmlns=""><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">)</mo></math></span> could refer to a couple of models with a high similarity score rather than a functional association between TF<sub><em>i</em></sub> and TF<sub><em>j</em></sub>. Therefore, to correctly identify relevant associated transcription factors, we will use the overlapping score OVD(TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>) and the similarity score JSD(TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>). In <a reveal-id="dsx041-F3" data-open="dsx041-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, OVD(TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>) and JSD(TF<sub><em>i</em></sub>, TF<sub><em>j</em></sub>) values are shown for all the TF couples which have <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">Z</mi></mrow><mrow><mn>5</mn></mrow></msub><mfenced separators="|" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">F</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo xmlns="">&gt;</mo><mn xmlns="">20</mn></math>⁠</span>. Since the higher the value of JSD the lower the similarity between two models is, while the higher the values of OVD the higher the similarity between binding sites is, as selection criterion to identify significant couples we used the inequality JSD &gt; OVD. In such a way, the plane (OVD, JSD) is divided into two regions: the upper-left sector, where models are not similar and the overlapping is low (selected couples), while in the lower-right sector, where models are similar and the overlapping is high (discarded couples). It is worth remarking that even if measures JSD and OVD could appear redundant, <a reveal-id="dsx041-F3" data-open="dsx041-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> clearly stated that the two measures are quite different: there are couples with similar JSD and very different OVD and vice versa. In fact JSD measure only depends on the related PWM while OVD measure depends on the considered dataset, e.g. two models can bind the same sequence even if they are different. Considering a combination of both measures we obtain a more robust selection criterion.</p><p>In <a reveal-id="dsx041-F4" data-open="dsx041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> (panel A) is sketched the procedure we use to select the 547 couples identifying pairs of transcription factors that are likely to synergistically regulate gene transcription. In the following sections, we will analyse the set of selected couples in order to associate to them a biological task and to find the most relevant associated couples according to their statistical features (see <a reveal-id="dsx041-F4" data-open="dsx041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> panel B). </p>                        <a id="112483402" scrollto-destination="112483402"></a>
<div id="dsx041-F4" data-id="dsx041-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f4.png?Expires=2147483647&Signature=4h3wH57a~BH5py86Qk9aq-V0YrL37TKN5~RLoghgzgoVJd1NN-1biYfg9SG-haNs2uPZ6c2CnoWwlV9O~WvvNNyZi1OWAnybNizVdo4c4ZrkHesFNGW6ikr9pfxYkm~X~1DNwERL-4DLuo94SWCFHWyYgabvO6EW~s9B~aupSF~Zw1u-Pf06W87KBWecoGReSlAEQ5VH5sFA1uSHgCq8fCgPDZgnKU1bUjPw5jZFVkC5VpMojDYeI4sQshtQm4nv5ixdr1th85W~gHrR9CtxW1Bdne7FuwtpODCm1c~rtClUfOxDKLoZf1we86QBrXt7dr-bpD2nOmOCD~i6BVMhxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow for the analysis of TF couples. In panel A, the procedure for the selection of relevant couples is depicted. In panel B, the analysis of significant couples is described both in terms of enrichment tests and sorting of best couples." path-from-xml="dsx041f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483402/dsx041f4.tif" path-from-xml="dsx041f4.tif" target="_blank">View large</a><a section="112483402" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f4.png?Expires=2147483647&Signature=k2Pf5dtZw3gXzxgG5Ra2fA06o9eTrwXglaW0tN8cKXw005CZwPn2KztJRMZiAk-7cr2YftojqeFSb7G9yXI-icO7tOdNgdR0wveKwzG3G15PkRU5WGbP97iB4pMpC5JuMhEQ-iFvvhnPXz6DBGXiAVwgua9QMM7bR-N7EGxcGvgKDeSOxbOdJlIKjrMmXt6jDMY0FbC2TatnrekoCctHyBqlhgSNnPx7OyUHMzf-~EQ1Yi~URBj8D90qvjn9wtYyiLKDg9uH1NFXxDzYwTDlymulk9qrp1ph9E6L~4PiUeyNk~wkH0odi~flE1E-kZRQ1vJJGnp6I68RL9dIqgkRyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483402&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow for the analysis of TF couples. In panel A, the procedure for the selection of relevant couples is depicted. In panel B, the analysis of significant couples is described both in terms of enrichment tests and sorting of best couples.</p></div></div><div content-id="dsx041-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f4.png?Expires=2147483647&Signature=4h3wH57a~BH5py86Qk9aq-V0YrL37TKN5~RLoghgzgoVJd1NN-1biYfg9SG-haNs2uPZ6c2CnoWwlV9O~WvvNNyZi1OWAnybNizVdo4c4ZrkHesFNGW6ikr9pfxYkm~X~1DNwERL-4DLuo94SWCFHWyYgabvO6EW~s9B~aupSF~Zw1u-Pf06W87KBWecoGReSlAEQ5VH5sFA1uSHgCq8fCgPDZgnKU1bUjPw5jZFVkC5VpMojDYeI4sQshtQm4nv5ixdr1th85W~gHrR9CtxW1Bdne7FuwtpODCm1c~rtClUfOxDKLoZf1we86QBrXt7dr-bpD2nOmOCD~i6BVMhxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow for the analysis of TF couples. In panel A, the procedure for the selection of relevant couples is depicted. In panel B, the analysis of significant couples is described both in terms of enrichment tests and sorting of best couples." path-from-xml="dsx041f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483402/dsx041f4.tif" path-from-xml="dsx041f4.tif" target="_blank">View large</a><a section="112483402" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f4.png?Expires=2147483647&Signature=k2Pf5dtZw3gXzxgG5Ra2fA06o9eTrwXglaW0tN8cKXw005CZwPn2KztJRMZiAk-7cr2YftojqeFSb7G9yXI-icO7tOdNgdR0wveKwzG3G15PkRU5WGbP97iB4pMpC5JuMhEQ-iFvvhnPXz6DBGXiAVwgua9QMM7bR-N7EGxcGvgKDeSOxbOdJlIKjrMmXt6jDMY0FbC2TatnrekoCctHyBqlhgSNnPx7OyUHMzf-~EQ1Yi~URBj8D90qvjn9wtYyiLKDg9uH1NFXxDzYwTDlymulk9qrp1ph9E6L~4PiUeyNk~wkH0odi~flE1E-kZRQ1vJJGnp6I68RL9dIqgkRyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483402&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow for the analysis of TF couples. In panel A, the procedure for the selection of relevant couples is depicted. In panel B, the analysis of significant couples is described both in terms of enrichment tests and sorting of best couples.</p></div></div>                            <h3 scrollto-destination=112483403 id="112483403" class="section-title" >3.2. Enrichment analysis and validation</h3>
<p>In order to evaluate the biological significance of the obtained TF couples, we performed hypergeometric enrichment tests.<a reveal-id="dsx041-B23" data-open="dsx041-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Given a couple of models, we want to test whether the set of genes potentially regulated by the synergy of the corresponding two TFs, is enriched in a particular category-term. We wondered whether those genes are linked in a given biological task in order to associate the TF couple to a given biological process. We selected a list of categories and related terms, reported in <a reveal-id="dsx041-T1" data-open="dsx041-T1" class="link link-reveal link-table xref-fig">Table 1</a>, that identify different biological tasks and contexts. We then performed for each identified TF couple hypergeometric enrichment tests related to all the considered terms. <div content-id="dsx041-T1" class="table-modal"><div class="table-wrap"><div id="dsx041-T1" data-id="dsx041-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Categories and terms for the enrichment analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Number of terms </th><th>Number of genes </th></tr></thead><tbody><tr><td>OMIM expanded </td><td>187 </td><td>2,178 </td></tr><tr><td>Tissue protein expression from proteomicsDB </td><td>207 </td><td>62,307 </td></tr><tr><td>KEGG 2015 </td><td>179 </td><td>3,800 </td></tr><tr><td>OMIM disease </td><td>90 </td><td>1,759 </td></tr><tr><td>GO molecular function </td><td>1,136 </td><td>12,753 </td></tr><tr><td>GO cellular component </td><td>641 </td><td>13,236 </td></tr><tr><td>GO biological process </td><td>5,192 </td><td>14,264 </td></tr><tr><td>Chromosome location </td><td>386 </td><td>32,740 </td></tr></tbody></table></div><div content-id="dsx041-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Number of terms </th><th>Number of genes </th></tr></thead><tbody><tr><td>OMIM expanded </td><td>187 </td><td>2,178 </td></tr><tr><td>Tissue protein expression from proteomicsDB </td><td>207 </td><td>62,307 </td></tr><tr><td>KEGG 2015 </td><td>179 </td><td>3,800 </td></tr><tr><td>OMIM disease </td><td>90 </td><td>1,759 </td></tr><tr><td>GO molecular function </td><td>1,136 </td><td>12,753 </td></tr><tr><td>GO cellular component </td><td>641 </td><td>13,236 </td></tr><tr><td>GO biological process </td><td>5,192 </td><td>14,264 </td></tr><tr><td>Chromosome location </td><td>386 </td><td>32,740 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx041-T1" data-id="dsx041-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Categories and terms for the enrichment analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Number of terms </th><th>Number of genes </th></tr></thead><tbody><tr><td>OMIM expanded </td><td>187 </td><td>2,178 </td></tr><tr><td>Tissue protein expression from proteomicsDB </td><td>207 </td><td>62,307 </td></tr><tr><td>KEGG 2015 </td><td>179 </td><td>3,800 </td></tr><tr><td>OMIM disease </td><td>90 </td><td>1,759 </td></tr><tr><td>GO molecular function </td><td>1,136 </td><td>12,753 </td></tr><tr><td>GO cellular component </td><td>641 </td><td>13,236 </td></tr><tr><td>GO biological process </td><td>5,192 </td><td>14,264 </td></tr><tr><td>Chromosome location </td><td>386 </td><td>32,740 </td></tr></tbody></table></div><div content-id="dsx041-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Number of terms </th><th>Number of genes </th></tr></thead><tbody><tr><td>OMIM expanded </td><td>187 </td><td>2,178 </td></tr><tr><td>Tissue protein expression from proteomicsDB </td><td>207 </td><td>62,307 </td></tr><tr><td>KEGG 2015 </td><td>179 </td><td>3,800 </td></tr><tr><td>OMIM disease </td><td>90 </td><td>1,759 </td></tr><tr><td>GO molecular function </td><td>1,136 </td><td>12,753 </td></tr><tr><td>GO cellular component </td><td>641 </td><td>13,236 </td></tr><tr><td>GO biological process </td><td>5,192 </td><td>14,264 </td></tr><tr><td>Chromosome location </td><td>386 </td><td>32,740 </td></tr></tbody></table></div></div></p><p>For each TF couple we obtained a list of <em>P</em>-values linked to the corresponding terms. The enrichment results were then compared with those of ten random sets of genes. Each set was generated preserving the same number of sample couples (i.e. 547) and the same number of genes for each couple. The results are reported in <a reveal-id="dsx041-F5" data-open="dsx041-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> where solid plot represents the enrichment of couples identified by our algorithm and dashed plots show enrichment of random sets. Each point of the plots shows in log-scale the number of TF couples (<em>y</em>-axis) with at least one term whose <em>P</em>-value falls in the corresponding <em>P</em>-value interval (<em>x</em>-axis). For example in the black plot, there is only one couple (M00106—M00967) having a term [ubiquitinyl hydrolase activity (GO: 0036459)] enriched with a <em>P</em>-value smaller than 10<sup>−18</sup> (3.63E–19), another couple (M00739—M00799) for <em>P</em>-value smaller than 10<sup>−14</sup> and three different couples for <em>P</em>-value smaller than 10<sup>−13</sup> and so on (see <a reveal-id="dsx041-T2" data-open="dsx041-T2" class="link link-reveal link-table xref-fig">Table 2</a> where all the enriched terms are ordered according to their <em>P</em>-values). It is worth noting that some couples have more than one term with a significant <em>P</em>-value, for example couple (M00106—M00967) shows several terms with a remarkable enrichment: GO Molecular function—ubiquitinyl hydrolase activity (GO: 0036459)—<em>P</em>-value 3.63E–19, Go Molecular function—cysteine-type peptidase activity (GO: 0008234)—<em>P</em>-value 4.56E–17, Go Biological Process—Ubiquitin-dependent protein catabolic process (GO: 0006511), and so on. The comparison of enrichment <em>P</em>-values, related to couples identified by our algorithm (black plot), with those related to random sets (ten coloured plots) clearly highlights the significance and consistency of our analysis, since none of the random sets shows <em>P</em>-values smaller than 10<sup>−7</sup>. Also analysing the plots for larger <em>P</em>-values intervals the number of enriched couples is significantly higher than all the corresponding numbers of random sets (taking into account the log-scale units in <em>y</em>-axis). <div content-id="dsx041-T2" class="table-modal"><div class="table-wrap"><div id="dsx041-T2" data-id="dsx041-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The PWM model IDs and the corresponding TF names are reported in columns 1–4 for each couple; the category and the related term the couple resulted enriched in are reported in columns 5–6; column 7 reports the related <em>P</em>-value of the hypergeometric test</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Category </th><th>Term </th><th>P-value </th></tr></thead><tbody><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>ubiquitinyl hydrolase activity (GO: 0036459) </td><td>3, 63E–19 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>cysteine-type peptidase activity (GO: 0008234) </td><td>4, 56E–17 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>6, 60E–15 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Ubiquitin-dependent protein catabolic process (GO: 0006511) </td><td>3, 59E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>4, 09E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent protein catabolic process (GO: 0019941) </td><td>4, 42E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent macromolecule catabolic process (GO: 0043632) </td><td>4, 90E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Proteolysis involved in cellular protein catabolic process (GO: 0051603) </td><td>8, 43E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 42E–13 </td></tr><tr><td>M00777 </td><td>STAT </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 54E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>4, 01E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>2, 81E–12 </td></tr><tr><td>M00457 </td><td>STAT5A </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>6, 40E–12 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>8, 69E–12 </td></tr><tr><td>M00223 </td><td>STATx </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>4, 82E–11 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00927 </td><td>AP-4 </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>1, 12E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>2, 71E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>4, 02E–10 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>1, 66E–09 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>2, 46E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>4, 46E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>FK506 binding (GO: 0005528) </td><td>7, 13E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>Macrolide binding (GO: 0005527) </td><td>7, 13E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>9, 90E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 44E–08 </td></tr><tr><td>M00655 </td><td>PEA3 </td><td>M00803 </td><td>E2F </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 50E–08 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00803 </td><td>E2F </td><td>GO Biological process </td><td>Protein autophosphorylation (GO: 0046777) </td><td>2, 05E–08 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>2, 73E–08 </td></tr><tr><td>M00059 </td><td>YY1 </td><td>M00148 </td><td>SRY </td><td>GO Molecular function </td><td>Sodium ion transmembrane transporter activity (GO: 0015081) </td><td>2, 96E–08 </td></tr><tr><td>M00415 </td><td>AREB6 </td><td>M00706 </td><td>TFII-I </td><td>KEGG </td><td>Valine leucine and isoleucine biosynthesis </td><td>4, 10E–08 </td></tr></tbody></table></div><div content-id="dsx041-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Category </th><th>Term </th><th>P-value </th></tr></thead><tbody><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>ubiquitinyl hydrolase activity (GO: 0036459) </td><td>3, 63E–19 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>cysteine-type peptidase activity (GO: 0008234) </td><td>4, 56E–17 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>6, 60E–15 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Ubiquitin-dependent protein catabolic process (GO: 0006511) </td><td>3, 59E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>4, 09E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent protein catabolic process (GO: 0019941) </td><td>4, 42E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent macromolecule catabolic process (GO: 0043632) </td><td>4, 90E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Proteolysis involved in cellular protein catabolic process (GO: 0051603) </td><td>8, 43E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 42E–13 </td></tr><tr><td>M00777 </td><td>STAT </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 54E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>4, 01E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>2, 81E–12 </td></tr><tr><td>M00457 </td><td>STAT5A </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>6, 40E–12 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>8, 69E–12 </td></tr><tr><td>M00223 </td><td>STATx </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>4, 82E–11 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00927 </td><td>AP-4 </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>1, 12E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>2, 71E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>4, 02E–10 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>1, 66E–09 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>2, 46E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>4, 46E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>FK506 binding (GO: 0005528) </td><td>7, 13E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>Macrolide binding (GO: 0005527) </td><td>7, 13E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>9, 90E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 44E–08 </td></tr><tr><td>M00655 </td><td>PEA3 </td><td>M00803 </td><td>E2F </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 50E–08 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00803 </td><td>E2F </td><td>GO Biological process </td><td>Protein autophosphorylation (GO: 0046777) </td><td>2, 05E–08 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>2, 73E–08 </td></tr><tr><td>M00059 </td><td>YY1 </td><td>M00148 </td><td>SRY </td><td>GO Molecular function </td><td>Sodium ion transmembrane transporter activity (GO: 0015081) </td><td>2, 96E–08 </td></tr><tr><td>M00415 </td><td>AREB6 </td><td>M00706 </td><td>TFII-I </td><td>KEGG </td><td>Valine leucine and isoleucine biosynthesis </td><td>4, 10E–08 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx041-T2" data-id="dsx041-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The PWM model IDs and the corresponding TF names are reported in columns 1–4 for each couple; the category and the related term the couple resulted enriched in are reported in columns 5–6; column 7 reports the related <em>P</em>-value of the hypergeometric test</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Category </th><th>Term </th><th>P-value </th></tr></thead><tbody><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>ubiquitinyl hydrolase activity (GO: 0036459) </td><td>3, 63E–19 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>cysteine-type peptidase activity (GO: 0008234) </td><td>4, 56E–17 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>6, 60E–15 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Ubiquitin-dependent protein catabolic process (GO: 0006511) </td><td>3, 59E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>4, 09E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent protein catabolic process (GO: 0019941) </td><td>4, 42E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent macromolecule catabolic process (GO: 0043632) </td><td>4, 90E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Proteolysis involved in cellular protein catabolic process (GO: 0051603) </td><td>8, 43E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 42E–13 </td></tr><tr><td>M00777 </td><td>STAT </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 54E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>4, 01E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>2, 81E–12 </td></tr><tr><td>M00457 </td><td>STAT5A </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>6, 40E–12 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>8, 69E–12 </td></tr><tr><td>M00223 </td><td>STATx </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>4, 82E–11 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00927 </td><td>AP-4 </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>1, 12E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>2, 71E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>4, 02E–10 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>1, 66E–09 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>2, 46E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>4, 46E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>FK506 binding (GO: 0005528) </td><td>7, 13E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>Macrolide binding (GO: 0005527) </td><td>7, 13E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>9, 90E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 44E–08 </td></tr><tr><td>M00655 </td><td>PEA3 </td><td>M00803 </td><td>E2F </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 50E–08 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00803 </td><td>E2F </td><td>GO Biological process </td><td>Protein autophosphorylation (GO: 0046777) </td><td>2, 05E–08 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>2, 73E–08 </td></tr><tr><td>M00059 </td><td>YY1 </td><td>M00148 </td><td>SRY </td><td>GO Molecular function </td><td>Sodium ion transmembrane transporter activity (GO: 0015081) </td><td>2, 96E–08 </td></tr><tr><td>M00415 </td><td>AREB6 </td><td>M00706 </td><td>TFII-I </td><td>KEGG </td><td>Valine leucine and isoleucine biosynthesis </td><td>4, 10E–08 </td></tr></tbody></table></div><div content-id="dsx041-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Category </th><th>Term </th><th>P-value </th></tr></thead><tbody><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>ubiquitinyl hydrolase activity (GO: 0036459) </td><td>3, 63E–19 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Molecular function </td><td>cysteine-type peptidase activity (GO: 0008234) </td><td>4, 56E–17 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>6, 60E–15 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Ubiquitin-dependent protein catabolic process (GO: 0006511) </td><td>3, 59E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>4, 09E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent protein catabolic process (GO: 0019941) </td><td>4, 42E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Modification-dependent macromolecule catabolic process (GO: 0043632) </td><td>4, 90E–14 </td></tr><tr><td>M00106 </td><td>CDP CR3+HD </td><td>M00967 </td><td>HNF4 COUP </td><td>GO Biological process </td><td>Proteolysis involved in cellular protein catabolic process (GO: 0051603) </td><td>8, 43E–14 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 42E–13 </td></tr><tr><td>M00777 </td><td>STAT </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 54E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>4, 01E–13 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>2, 81E–12 </td></tr><tr><td>M00457 </td><td>STAT5A </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>6, 40E–12 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>8, 69E–12 </td></tr><tr><td>M00223 </td><td>STATx </td><td>M00980 </td><td>TBP </td><td>Chromosome location </td><td>Chr5q13 </td><td>4, 82E–11 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00927 </td><td>AP-4 </td><td>GO Biological process </td><td>Gonadal mesoderm development (GO: 0007506) </td><td>1, 12E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>2, 71E–10 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>4, 02E–10 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesoderm development (GO: 0007498) </td><td>1, 66E–09 </td></tr><tr><td>M00736 </td><td>E2F-1: DP-1 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Mesenchyme development (GO: 0060485) </td><td>2, 46E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome assembly (GO: 0006334) </td><td>4, 46E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>FK506 binding (GO: 0005528) </td><td>7, 13E–09 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00921 </td><td>GR </td><td>GO Molecular function </td><td>Macrolide binding (GO: 0005527) </td><td>7, 13E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex assembly (GO: 0065004) </td><td>9, 90E–09 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Nucleosome organization (GO: 0034728) </td><td>1, 44E–08 </td></tr><tr><td>M00655 </td><td>PEA3 </td><td>M00803 </td><td>E2F </td><td>Chromosome location </td><td>Chr5q13 </td><td>1, 50E–08 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00803 </td><td>E2F </td><td>GO Biological process </td><td>Protein autophosphorylation (GO: 0046777) </td><td>2, 05E–08 </td></tr><tr><td>M00739 </td><td>E2F-4: DP-2 </td><td>M00799 </td><td>Myc </td><td>GO Biological process </td><td>Protein-DNA complex subunit organization (GO: 0071824) </td><td>2, 73E–08 </td></tr><tr><td>M00059 </td><td>YY1 </td><td>M00148 </td><td>SRY </td><td>GO Molecular function </td><td>Sodium ion transmembrane transporter activity (GO: 0015081) </td><td>2, 96E–08 </td></tr><tr><td>M00415 </td><td>AREB6 </td><td>M00706 </td><td>TFII-I </td><td>KEGG </td><td>Valine leucine and isoleucine biosynthesis </td><td>4, 10E–08 </td></tr></tbody></table></div></div></p>                        <a id="112483406" scrollto-destination="112483406"></a>
<div id="dsx041-F5" data-id="dsx041-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f5.png?Expires=2147483647&Signature=DzJTek4bYiKZzUtQMrt8wePMiVt4bfi-MDP339TD41WFA1zGKoMTTgWy2PWCiBTijOE2yFPNh~StOqz~caDJUzgWJTF6S4KFAwyTHPyIDgtUQ~8C8vO2fa5kHbJ6vGL6ik~h-qXwA4aus5kAnw0VvaysZpyvnEjkUnSxsew6AEGu8~ngjYvViDVhnGxS-NaT5YdsZr-0loGUL7iL1FsQmWf3tJ2JQVqgTexHZoJLxr0X0M5bPD8O211t9aCGXYYnjJvYwbH7qZCQFXnfwEUzxbxrI9zhIuJfoBXMRQhk3eL9yyEyIlddng~XYhoh3DzYrUX8g5lrLzdlXmCB9BH86g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of couples enriched in at least one term as a function of their smallest P-value. Solid plot shows the enrichment of the couples identified by our algorithm, dashed plots show the enrichment of random couples for comparison." path-from-xml="dsx041f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483406/dsx041f5.tif" path-from-xml="dsx041f5.tif" target="_blank">View large</a><a section="112483406" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f5.png?Expires=2147483647&Signature=lBWpYt~-2~z9QMUZROWGfBEAvfCXb6fiwLb8hK91kBTLFQh9gJT~E89o2vtA32D9ghurQYWLDs7-GbXjToVIVjX1DDXuuVgGETUcbAm6MO2k0~PlUVzPTVfuERI8Qxu5llR2tiNqdGncwV1CimnejZP9KNktSnsIJ-tz4z6HdhM-sYU3KEdgIRd18jZo1lIDqHMPoMY830SROneUOQ3XTCFK9z2CqJulO1QFr3ujv4qeV4lH4f4edh-iqeDE4buQB6hsUTPkRpkaTSdFYtv3ghI2djLDU8fWMwMwD4IeSaJvKLVIhClPcKdTc~1wFz-PoPMz~nGWdvywY6rhxqr0WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483406&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of couples enriched in at least one term as a function of their smallest <em>P</em>-value. Solid plot shows the enrichment of the couples identified by our algorithm, dashed plots show the enrichment of random couples for comparison.</p></div></div><div content-id="dsx041-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/m_dsx041f5.png?Expires=2147483647&Signature=DzJTek4bYiKZzUtQMrt8wePMiVt4bfi-MDP339TD41WFA1zGKoMTTgWy2PWCiBTijOE2yFPNh~StOqz~caDJUzgWJTF6S4KFAwyTHPyIDgtUQ~8C8vO2fa5kHbJ6vGL6ik~h-qXwA4aus5kAnw0VvaysZpyvnEjkUnSxsew6AEGu8~ngjYvViDVhnGxS-NaT5YdsZr-0loGUL7iL1FsQmWf3tJ2JQVqgTexHZoJLxr0X0M5bPD8O211t9aCGXYYnjJvYwbH7qZCQFXnfwEUzxbxrI9zhIuJfoBXMRQhk3eL9yyEyIlddng~XYhoh3DzYrUX8g5lrLzdlXmCB9BH86g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of couples enriched in at least one term as a function of their smallest P-value. Solid plot shows the enrichment of the couples identified by our algorithm, dashed plots show the enrichment of random couples for comparison." path-from-xml="dsx041f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483406/dsx041f5.tif" path-from-xml="dsx041f5.tif" target="_blank">View large</a><a section="112483406" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx041/3/dsx041f5.png?Expires=2147483647&Signature=lBWpYt~-2~z9QMUZROWGfBEAvfCXb6fiwLb8hK91kBTLFQh9gJT~E89o2vtA32D9ghurQYWLDs7-GbXjToVIVjX1DDXuuVgGETUcbAm6MO2k0~PlUVzPTVfuERI8Qxu5llR2tiNqdGncwV1CimnejZP9KNktSnsIJ-tz4z6HdhM-sYU3KEdgIRd18jZo1lIDqHMPoMY830SROneUOQ3XTCFK9z2CqJulO1QFr3ujv4qeV4lH4f4edh-iqeDE4buQB6hsUTPkRpkaTSdFYtv3ghI2djLDU8fWMwMwD4IeSaJvKLVIhClPcKdTc~1wFz-PoPMz~nGWdvywY6rhxqr0WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483406&ar=4562529&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx041f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of couples enriched in at least one term as a function of their smallest <em>P</em>-value. Solid plot shows the enrichment of the couples identified by our algorithm, dashed plots show the enrichment of random couples for comparison.</p></div></div>                            <h3 scrollto-destination=112483407 id="112483407" class="section-title" >3.3. Analysis of relevant transcription factor couples</h3>
<p>In this section, we analyse the most significant couples identified by our algorithm. Among the 547 selected couples, we consider as particularly relevant those couples with the smallest enrichment <em>P</em>-value or with the best associated parameters (highest number of transcripts, smallest OVD and highest JSD) as reported in <a reveal-id="dsx041-F4" data-open="dsx041-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> panel B. The list of couples with the smallest <em>P</em>-values (smaller than 10<sup>−7</sup>) is reported in <a reveal-id="dsx041-T2" data-open="dsx041-T2" class="link link-reveal link-table xref-fig">Table 2</a>. The analysis of the couples reported in the table suggests several observations, providing and/or confirming evidence of the synergism of several TF couples and the related biological contexts they are involved in. The smallest <em>P</em>-values are reported for the couple CDP (also known as Cutl1) (M00106)–HNF4 (M00967), as already mentioned above. The related enriched terms are mainly associated (4 out of 6) with catabolic processes while the other two terms are ubiquitinyl hydrolase activity and cysteine-type peptidase activity. Concerning the couple Myc (M00799)–E2F (M00803) (and its cluster E2F–1 and E2F–4) it can be observed that it appears several times in the table associated to different terms. These terms can be grouped into main activities/contexts regarding: (1) gonadal mesoderm development and mesenchyme development, (2) Nucleosome assembly and organization, Protein-DNA complex assembly and Protein-DNA Complex Subunit Organization and (3) Protein Autophosphorylation. It is worth noting that in literature there are many references associated to the couple Myc–E2F. In Ref. <a reveal-id="dsx041-B24" data-open="dsx041-B24" class="link link-ref link-reveal xref-bibr">24</a>, it has been shown how the two TFs are linked via mSin3A, a core component of a large multiprotein co-repressor complex with associated histone deacetylase (HDAC) enzymatic activity. As reported in Ref. <a reveal-id="dsx041-B25" data-open="dsx041-B25" class="link link-ref link-reveal xref-bibr">25</a>, it has been identified a subunit of the complex NuA4 as the product of TRA1, an ATM-related gene homologous to human TRRAP, an essential cofactor for c-Myc- and E2F-mediated oncogenic transformation. A further link between Myc and E2F was studied in Ref. <a reveal-id="dsx041-B26" data-open="dsx041-B26" class="link link-ref link-reveal xref-bibr">26</a> highlighting their relationships with chromatin structure and stability. It is remarkable and can be source of further investigations the association of the couple AREB (M00415)–TFII-I (M00706) (also known as GTF2I) with KEGG pathway Valine leucine and isoleucine biosynthesis (the couple showing a <em>P</em>-value of 4.10E–8). Interestingly, transcription factors of STAT family (STAT—M00777, STATx—M00223 and STAT5A–M00457) strongly interact with TBP (M00980) in the promoters of genes belonging to the region of chr5q13, that is known to be associated with various neurological disorders and pathologies such as Spinal Muscular Atrophy, Hairy Cell Leukemia and it is also connected to Alcohol Dependence.<a reveal-id="dsx041-B27 dsx041-B28 dsx041-B29" data-open="dsx041-B27 dsx041-B28 dsx041-B29" class="link link-ref link-reveal xref-bibr"><sup>27–29</sup></a> Finally, it could deserve attention the couple Areb6 (M00414)–E2F (M00803) associated to the term Mental retardation, even if not reported in <a reveal-id="dsx041-T2" data-open="dsx041-T2" class="link link-reveal link-table xref-fig">Table 2</a>, it shows a remarkable <em>P</em>-value (6·10<sup>−6</sup>). Areb6 belongs to the Zeb transcription factor family that has been shown to be involved in mental retardation syndromes.<a reveal-id="dsx041-B30" data-open="dsx041-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> In <a reveal-id="dsx041-T3" data-open="dsx041-T3" class="link link-reveal link-table xref-fig">Table 3</a>, the best couples, in terms of association, are reported ordered by the number of significant Transcripts. We selected those couples with a number of significant transcripts (<em>Z</em>-score higher than 5) higher than 100, with an Overlapping score (OVD) smaller than 0.2 and a Similarity score (JSD) higher than 0.2 (it is also reported in the fifth column the average <em>Z</em>-Score). The couple MZF1 (M00083)—MAZ (M00649) shows a <em>Z</em>-score higher than 5 in 1,014 transcript promoter sequences (with an average <em>z</em>-score of 7.77—more than 7 S.D. far from the expected average value). Myc-associated zinc finger protein (MAZ) and Myeloid zinc finger 1 (MZF1) are both transcription factors characterized by a zinc finger small protein structural motif. The associated PWMs show a similar common sub-motif characterized by a GGGA sequence. Nevertheless the PWMs have different length (6 and 8) and the correspondent similarity score, JSD, results to be 0.21 meaning that the two PWMs are similar but globally not so close to each other. Significantly, the OVD overlapping score is quite low 0.188 meaning that, on average, among all the identified TFBSs couple (both in a window of 80 bp) of the two TFs, less than 1 out of 5 couples are overlapped. This is why we included this couple in the selected set of TFs couples. The surprisingly high number of transcripts for which the two TFs co-occur should deserve deeper investigations. The couple MZF1 (M00083)–E2F (M00803) shows 456 transcripts with a <em>Z</em>-score higher than 5 (with an average <em>z</em>-score of 6, 77, OVD = 0.09 and JSD = 0.21). Interestingly they are involved in several diseases, in particular, as reported in Ref. <a reveal-id="dsx041-B31" data-open="dsx041-B31" class="link link-ref link-reveal xref-bibr">31</a>, they are both potential key regulators of PKD1 and PKD2 whose mutations are linked with autosomal dominant polycystic kidney disease (ADPKD). We found one transcript of PKD1 uc002cos.1 with a <em>Z</em>-score for the couple equal to 2 and two transcripts of PKD2 uc003hre.3 and uc011cdg.2 with <em>Z</em>-score equal to 5.80 and 3.42, respectively. It is interesting that a minor groove binding protein SRY (M00148) is associated to both the models of TBP (M00471 and M00980) with a number of significant transcripts equal to 239 and 203, respectively, a very small OVD close to 0 for both and JSD 0.21 and 0.23, respectively. It is worth noting that the couple Myc (M00799)–E2F (M00803) (also included in <a reveal-id="dsx041-T2" data-open="dsx041-T2" class="link link-reveal link-table xref-fig">Table 2</a>) shows a number of significant transcripts equal to 293, meaning that, besides a clear association with given biological contexts, as discussed in the previous section, there is a strong synergism between the two TFs confirmed by the values reported in the table. Concerning the couple Maz (M00649)–Pu.1 (M00658) (number of significant transcripts equal to 187) it has been shown that three transcription factors Maz, PU.1 and ARNT show significant recognition elements among similarly up or down-regulated genes involved in hematopoietic differentiation or leukemogenesis.<a reveal-id="dsx041-B32" data-open="dsx041-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> We note that also the couple Maz (M00649)–ARNT (M00976) is included in the table with a number of significant transcripts equal to 106. <div content-id="dsx041-T3" class="table-modal"><div class="table-wrap"><div id="dsx041-T3" data-id="dsx041-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>The PWM model IDs and the corresponding TF names are reported in columns 1–4 for each couple; Column 5 reports the number of transcripts whose <em>Z</em>-scores, related to the couple, are higher than 5; the average <em>Z</em>-score of all the significant (Z &gt; 5) transcripts is reported in column 6; Overlapping and Similarity Scores are reported in columns 7 and 8, respectively</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Number of transcripts </th><th>Average <em>Z</em>-score </th><th>Overlapping score </th><th>Similarity score </th></tr></thead><tbody><tr><td>M00083 </td><td>MZF1 </td><td>M00649 </td><td>MAZ </td><td>1,014 </td><td>7, 77 </td><td>0, 19 </td><td>0, 21 </td></tr><tr><td>M00083 </td><td>MZF1 </td><td>M00803 </td><td>E2F </td><td>456 </td><td>6, 77 </td><td>0, 09 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>338 </td><td>6, 63 </td><td>0, 03 </td><td>0, 21 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00971 </td><td>Ets </td><td>317 </td><td>7, 71 </td><td>0, 16 </td><td>0, 24 </td></tr><tr><td><strong>M00799</strong> </td><td><strong>Myc</strong> </td><td><strong>M00803</strong> </td><td><strong>E2F</strong> </td><td><strong>293</strong> </td><td><strong>7, 16</strong> </td><td><strong>0, 00</strong> </td><td><strong>0, 23</strong> </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00803 </td><td>E2F </td><td>275 </td><td>6, 61 </td><td>0, 00 </td><td>0, 27 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00747 </td><td>IRF-1 </td><td>254 </td><td>7, 77 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00471 </td><td>TBP </td><td>239 </td><td>6, 78 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00980 </td><td>TBP </td><td>203 </td><td>6, 45 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00698 </td><td>HEB </td><td>M00803 </td><td>E2F </td><td>196 </td><td>6, 35 </td><td>0, 02 </td><td>0, 29 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00658 </td><td>PU.1 </td><td>187 </td><td>7, 49 </td><td>0, 03 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00799 </td><td>Myc </td><td>184 </td><td>6, 85 </td><td>0, 00 </td><td>0, 33 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00933 </td><td>Sp1 </td><td>182 </td><td>7, 06 </td><td>0, 01 </td><td>0, 28 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00471 </td><td>TBP </td><td>182 </td><td>6, 47 </td><td>0, 08 </td><td>0, 21 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00931 </td><td>Sp1 </td><td>167 </td><td>6, 97 </td><td>0, 00 </td><td>0, 30 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00927 </td><td>AP-4 </td><td>160 </td><td>6, 29 </td><td>0, 01 </td><td>0, 24 </td></tr><tr><td>M00801 </td><td>CREB </td><td>M00803 </td><td>E2F </td><td>153 </td><td>6, 43 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00931 </td><td>Sp1 </td><td>141 </td><td>6, 58 </td><td>0, 04 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00971 </td><td>Ets </td><td>132 </td><td>7, 32 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00933 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>127 </td><td>6, 65 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00981 </td><td>CREB ATF </td><td>127 </td><td>6, 72 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00932 </td><td>Sp1 </td><td>121 </td><td>7, 17 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00706 </td><td>TFII-I </td><td>120 </td><td>8, 58 </td><td>0, 00 </td><td>0, 31 </td></tr><tr><td>M00931 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>117 </td><td>6, 47 </td><td>0, 01 </td><td>0, 20 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00917 </td><td>CREB </td><td>115 </td><td>6, 81 </td><td>0, 00 </td><td>0, 25 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00706 </td><td>TFII-I </td><td>114 </td><td>6, 36 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00791 </td><td>HNF3 </td><td>M00975 </td><td>RFX </td><td>107 </td><td>6, 58 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00471 </td><td>TBP </td><td>M00747 </td><td>IRF-1 </td><td>107 </td><td>6, 32 </td><td>0, 02 </td><td>0, 25 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>106 </td><td>6, 28 </td><td>0, 00 </td><td>0, 26 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00962 </td><td>AR </td><td>106 </td><td>6, 04 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00789 </td><td>GATA </td><td>106 </td><td>6, 08 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00975 </td><td>RFX </td><td>104 </td><td>6, 20 </td><td>0, 00 </td><td>0, 21 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00799 </td><td>Myc </td><td>102 </td><td>6, 77 </td><td>0, 00 </td><td>0, 29 </td></tr><tr><td>M00775 </td><td>NF-Y </td><td>M00803 </td><td>E2F </td><td>101 </td><td>6, 24 </td><td>0, 03 </td><td>0, 20 </td></tr></tbody></table></div><div content-id="dsx041-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Number of transcripts </th><th>Average <em>Z</em>-score </th><th>Overlapping score </th><th>Similarity score </th></tr></thead><tbody><tr><td>M00083 </td><td>MZF1 </td><td>M00649 </td><td>MAZ </td><td>1,014 </td><td>7, 77 </td><td>0, 19 </td><td>0, 21 </td></tr><tr><td>M00083 </td><td>MZF1 </td><td>M00803 </td><td>E2F </td><td>456 </td><td>6, 77 </td><td>0, 09 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>338 </td><td>6, 63 </td><td>0, 03 </td><td>0, 21 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00971 </td><td>Ets </td><td>317 </td><td>7, 71 </td><td>0, 16 </td><td>0, 24 </td></tr><tr><td><strong>M00799</strong> </td><td><strong>Myc</strong> </td><td><strong>M00803</strong> </td><td><strong>E2F</strong> </td><td><strong>293</strong> </td><td><strong>7, 16</strong> </td><td><strong>0, 00</strong> </td><td><strong>0, 23</strong> </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00803 </td><td>E2F </td><td>275 </td><td>6, 61 </td><td>0, 00 </td><td>0, 27 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00747 </td><td>IRF-1 </td><td>254 </td><td>7, 77 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00471 </td><td>TBP </td><td>239 </td><td>6, 78 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00980 </td><td>TBP </td><td>203 </td><td>6, 45 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00698 </td><td>HEB </td><td>M00803 </td><td>E2F </td><td>196 </td><td>6, 35 </td><td>0, 02 </td><td>0, 29 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00658 </td><td>PU.1 </td><td>187 </td><td>7, 49 </td><td>0, 03 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00799 </td><td>Myc </td><td>184 </td><td>6, 85 </td><td>0, 00 </td><td>0, 33 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00933 </td><td>Sp1 </td><td>182 </td><td>7, 06 </td><td>0, 01 </td><td>0, 28 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00471 </td><td>TBP </td><td>182 </td><td>6, 47 </td><td>0, 08 </td><td>0, 21 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00931 </td><td>Sp1 </td><td>167 </td><td>6, 97 </td><td>0, 00 </td><td>0, 30 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00927 </td><td>AP-4 </td><td>160 </td><td>6, 29 </td><td>0, 01 </td><td>0, 24 </td></tr><tr><td>M00801 </td><td>CREB </td><td>M00803 </td><td>E2F </td><td>153 </td><td>6, 43 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00931 </td><td>Sp1 </td><td>141 </td><td>6, 58 </td><td>0, 04 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00971 </td><td>Ets </td><td>132 </td><td>7, 32 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00933 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>127 </td><td>6, 65 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00981 </td><td>CREB ATF </td><td>127 </td><td>6, 72 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00932 </td><td>Sp1 </td><td>121 </td><td>7, 17 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00706 </td><td>TFII-I </td><td>120 </td><td>8, 58 </td><td>0, 00 </td><td>0, 31 </td></tr><tr><td>M00931 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>117 </td><td>6, 47 </td><td>0, 01 </td><td>0, 20 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00917 </td><td>CREB </td><td>115 </td><td>6, 81 </td><td>0, 00 </td><td>0, 25 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00706 </td><td>TFII-I </td><td>114 </td><td>6, 36 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00791 </td><td>HNF3 </td><td>M00975 </td><td>RFX </td><td>107 </td><td>6, 58 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00471 </td><td>TBP </td><td>M00747 </td><td>IRF-1 </td><td>107 </td><td>6, 32 </td><td>0, 02 </td><td>0, 25 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>106 </td><td>6, 28 </td><td>0, 00 </td><td>0, 26 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00962 </td><td>AR </td><td>106 </td><td>6, 04 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00789 </td><td>GATA </td><td>106 </td><td>6, 08 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00975 </td><td>RFX </td><td>104 </td><td>6, 20 </td><td>0, 00 </td><td>0, 21 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00799 </td><td>Myc </td><td>102 </td><td>6, 77 </td><td>0, 00 </td><td>0, 29 </td></tr><tr><td>M00775 </td><td>NF-Y </td><td>M00803 </td><td>E2F </td><td>101 </td><td>6, 24 </td><td>0, 03 </td><td>0, 20 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The couple <strong>Myc</strong>, <strong>E2F</strong> is reported in bold to highlight it is also included in <a reveal-id="dsx041-T2" data-open="dsx041-T2" class="link link-reveal link-table xref-fig">Table 2</a>.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx041-T3" data-id="dsx041-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>The PWM model IDs and the corresponding TF names are reported in columns 1–4 for each couple; Column 5 reports the number of transcripts whose <em>Z</em>-scores, related to the couple, are higher than 5; the average <em>Z</em>-score of all the significant (Z &gt; 5) transcripts is reported in column 6; Overlapping and Similarity Scores are reported in columns 7 and 8, respectively</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Number of transcripts </th><th>Average <em>Z</em>-score </th><th>Overlapping score </th><th>Similarity score </th></tr></thead><tbody><tr><td>M00083 </td><td>MZF1 </td><td>M00649 </td><td>MAZ </td><td>1,014 </td><td>7, 77 </td><td>0, 19 </td><td>0, 21 </td></tr><tr><td>M00083 </td><td>MZF1 </td><td>M00803 </td><td>E2F </td><td>456 </td><td>6, 77 </td><td>0, 09 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>338 </td><td>6, 63 </td><td>0, 03 </td><td>0, 21 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00971 </td><td>Ets </td><td>317 </td><td>7, 71 </td><td>0, 16 </td><td>0, 24 </td></tr><tr><td><strong>M00799</strong> </td><td><strong>Myc</strong> </td><td><strong>M00803</strong> </td><td><strong>E2F</strong> </td><td><strong>293</strong> </td><td><strong>7, 16</strong> </td><td><strong>0, 00</strong> </td><td><strong>0, 23</strong> </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00803 </td><td>E2F </td><td>275 </td><td>6, 61 </td><td>0, 00 </td><td>0, 27 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00747 </td><td>IRF-1 </td><td>254 </td><td>7, 77 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00471 </td><td>TBP </td><td>239 </td><td>6, 78 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00980 </td><td>TBP </td><td>203 </td><td>6, 45 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00698 </td><td>HEB </td><td>M00803 </td><td>E2F </td><td>196 </td><td>6, 35 </td><td>0, 02 </td><td>0, 29 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00658 </td><td>PU.1 </td><td>187 </td><td>7, 49 </td><td>0, 03 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00799 </td><td>Myc </td><td>184 </td><td>6, 85 </td><td>0, 00 </td><td>0, 33 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00933 </td><td>Sp1 </td><td>182 </td><td>7, 06 </td><td>0, 01 </td><td>0, 28 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00471 </td><td>TBP </td><td>182 </td><td>6, 47 </td><td>0, 08 </td><td>0, 21 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00931 </td><td>Sp1 </td><td>167 </td><td>6, 97 </td><td>0, 00 </td><td>0, 30 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00927 </td><td>AP-4 </td><td>160 </td><td>6, 29 </td><td>0, 01 </td><td>0, 24 </td></tr><tr><td>M00801 </td><td>CREB </td><td>M00803 </td><td>E2F </td><td>153 </td><td>6, 43 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00931 </td><td>Sp1 </td><td>141 </td><td>6, 58 </td><td>0, 04 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00971 </td><td>Ets </td><td>132 </td><td>7, 32 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00933 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>127 </td><td>6, 65 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00981 </td><td>CREB ATF </td><td>127 </td><td>6, 72 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00932 </td><td>Sp1 </td><td>121 </td><td>7, 17 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00706 </td><td>TFII-I </td><td>120 </td><td>8, 58 </td><td>0, 00 </td><td>0, 31 </td></tr><tr><td>M00931 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>117 </td><td>6, 47 </td><td>0, 01 </td><td>0, 20 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00917 </td><td>CREB </td><td>115 </td><td>6, 81 </td><td>0, 00 </td><td>0, 25 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00706 </td><td>TFII-I </td><td>114 </td><td>6, 36 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00791 </td><td>HNF3 </td><td>M00975 </td><td>RFX </td><td>107 </td><td>6, 58 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00471 </td><td>TBP </td><td>M00747 </td><td>IRF-1 </td><td>107 </td><td>6, 32 </td><td>0, 02 </td><td>0, 25 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>106 </td><td>6, 28 </td><td>0, 00 </td><td>0, 26 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00962 </td><td>AR </td><td>106 </td><td>6, 04 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00789 </td><td>GATA </td><td>106 </td><td>6, 08 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00975 </td><td>RFX </td><td>104 </td><td>6, 20 </td><td>0, 00 </td><td>0, 21 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00799 </td><td>Myc </td><td>102 </td><td>6, 77 </td><td>0, 00 </td><td>0, 29 </td></tr><tr><td>M00775 </td><td>NF-Y </td><td>M00803 </td><td>E2F </td><td>101 </td><td>6, 24 </td><td>0, 03 </td><td>0, 20 </td></tr></tbody></table></div><div content-id="dsx041-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Model 1 </th><th>TF 1 </th><th>Model 2 </th><th>TF 2 </th><th>Number of transcripts </th><th>Average <em>Z</em>-score </th><th>Overlapping score </th><th>Similarity score </th></tr></thead><tbody><tr><td>M00083 </td><td>MZF1 </td><td>M00649 </td><td>MAZ </td><td>1,014 </td><td>7, 77 </td><td>0, 19 </td><td>0, 21 </td></tr><tr><td>M00083 </td><td>MZF1 </td><td>M00803 </td><td>E2F </td><td>456 </td><td>6, 77 </td><td>0, 09 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>338 </td><td>6, 63 </td><td>0, 03 </td><td>0, 21 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00971 </td><td>Ets </td><td>317 </td><td>7, 71 </td><td>0, 16 </td><td>0, 24 </td></tr><tr><td><strong>M00799</strong> </td><td><strong>Myc</strong> </td><td><strong>M00803</strong> </td><td><strong>E2F</strong> </td><td><strong>293</strong> </td><td><strong>7, 16</strong> </td><td><strong>0, 00</strong> </td><td><strong>0, 23</strong> </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00803 </td><td>E2F </td><td>275 </td><td>6, 61 </td><td>0, 00 </td><td>0, 27 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00747 </td><td>IRF-1 </td><td>254 </td><td>7, 77 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00471 </td><td>TBP </td><td>239 </td><td>6, 78 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00980 </td><td>TBP </td><td>203 </td><td>6, 45 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00698 </td><td>HEB </td><td>M00803 </td><td>E2F </td><td>196 </td><td>6, 35 </td><td>0, 02 </td><td>0, 29 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00658 </td><td>PU.1 </td><td>187 </td><td>7, 49 </td><td>0, 03 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00799 </td><td>Myc </td><td>184 </td><td>6, 85 </td><td>0, 00 </td><td>0, 33 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00933 </td><td>Sp1 </td><td>182 </td><td>7, 06 </td><td>0, 01 </td><td>0, 28 </td></tr><tr><td>M00462 </td><td>GATA-6 </td><td>M00471 </td><td>TBP </td><td>182 </td><td>6, 47 </td><td>0, 08 </td><td>0, 21 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00931 </td><td>Sp1 </td><td>167 </td><td>6, 97 </td><td>0, 00 </td><td>0, 30 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00927 </td><td>AP-4 </td><td>160 </td><td>6, 29 </td><td>0, 01 </td><td>0, 24 </td></tr><tr><td>M00801 </td><td>CREB </td><td>M00803 </td><td>E2F </td><td>153 </td><td>6, 43 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00706 </td><td>TFII-I </td><td>M00931 </td><td>Sp1 </td><td>141 </td><td>6, 58 </td><td>0, 04 </td><td>0, 22 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00971 </td><td>Ets </td><td>132 </td><td>7, 32 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00933 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>127 </td><td>6, 65 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00981 </td><td>CREB ATF </td><td>127 </td><td>6, 72 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00799 </td><td>Myc </td><td>M00932 </td><td>Sp1 </td><td>121 </td><td>7, 17 </td><td>0, 00 </td><td>0, 28 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00706 </td><td>TFII-I </td><td>120 </td><td>8, 58 </td><td>0, 00 </td><td>0, 31 </td></tr><tr><td>M00931 </td><td>Sp1 </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>117 </td><td>6, 47 </td><td>0, 01 </td><td>0, 20 </td></tr><tr><td>M00803 </td><td>E2F </td><td>M00917 </td><td>CREB </td><td>115 </td><td>6, 81 </td><td>0, 00 </td><td>0, 25 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00706 </td><td>TFII-I </td><td>114 </td><td>6, 36 </td><td>0, 02 </td><td>0, 21 </td></tr><tr><td>M00791 </td><td>HNF3 </td><td>M00975 </td><td>RFX </td><td>107 </td><td>6, 58 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00471 </td><td>TBP </td><td>M00747 </td><td>IRF-1 </td><td>107 </td><td>6, 32 </td><td>0, 02 </td><td>0, 25 </td></tr><tr><td>M00649 </td><td>MAZ </td><td>M00976 </td><td>AhR Arnt HIF-1 </td><td>106 </td><td>6, 28 </td><td>0, 00 </td><td>0, 26 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00962 </td><td>AR </td><td>106 </td><td>6, 04 </td><td>0, 00 </td><td>0, 20 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00789 </td><td>GATA </td><td>106 </td><td>6, 08 </td><td>0, 00 </td><td>0, 23 </td></tr><tr><td>M00148 </td><td>SRY </td><td>M00975 </td><td>RFX </td><td>104 </td><td>6, 20 </td><td>0, 00 </td><td>0, 21 </td></tr><tr><td>M00008 </td><td>Sp1 </td><td>M00799 </td><td>Myc </td><td>102 </td><td>6, 77 </td><td>0, 00 </td><td>0, 29 </td></tr><tr><td>M00775 </td><td>NF-Y </td><td>M00803 </td><td>E2F </td><td>101 </td><td>6, 24 </td><td>0, 03 </td><td>0, 20 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The couple <strong>Myc</strong>, <strong>E2F</strong> is reported in bold to highlight it is also included in <a reveal-id="dsx041-T2" data-open="dsx041-T2" class="link link-reveal link-table xref-fig">Table 2</a>.</p></div></div></div></div><div content-id="dsx041-T4" class="table-modal"><div class="table-wrap"><div id="dsx041-T4" data-id="dsx041-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Frequencies of couples as a function of the shortest path (SP) distance for three classes of protein couples: (i) the selected 547 TF couples (namely BEST, first row), (ii) all the couples of TFs (namely ALL, second row) and iii) 10 random sample sets made of 547 randomly picked protein couples from the whole PPI (namely RANDOM, mean and standard deviation in the third and fourth row, respectively)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>SP 1 </th><th>SP 2 </th><th>SP 3 </th><th>SP 4 </th><th>SP 5 </th><th>SP 6 </th><th>SP 7 </th></tr></thead><tbody><tr><td>BEST </td><td>0.117318 </td><td>0.478585 </td><td>0.284916 </td><td>0.10987 </td><td>0.009311 </td><td>0 </td><td>0 </td></tr><tr><td>ALL </td><td>0.056789 </td><td>0.463694 </td><td>0.376242 </td><td>0.084080 </td><td>0.014167 </td><td>0.004661 </td><td>0.000368 </td></tr><tr><td>RANDOM (mean) </td><td>0.003291 </td><td>0.065601 </td><td>0.377048 </td><td>0.385820 </td><td>0.134512 </td><td>0.027972 </td><td>0.005346 </td></tr><tr><td>RANDOM (stdv) </td><td>0.000943 </td><td>0.006363 </td><td>0.016408 </td><td>0.008298 </td><td>0.009771 </td><td>0.003329 </td><td>0.002414 </td></tr></tbody></table></div><div content-id="dsx041-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>SP 1 </th><th>SP 2 </th><th>SP 3 </th><th>SP 4 </th><th>SP 5 </th><th>SP 6 </th><th>SP 7 </th></tr></thead><tbody><tr><td>BEST </td><td>0.117318 </td><td>0.478585 </td><td>0.284916 </td><td>0.10987 </td><td>0.009311 </td><td>0 </td><td>0 </td></tr><tr><td>ALL </td><td>0.056789 </td><td>0.463694 </td><td>0.376242 </td><td>0.084080 </td><td>0.014167 </td><td>0.004661 </td><td>0.000368 </td></tr><tr><td>RANDOM (mean) </td><td>0.003291 </td><td>0.065601 </td><td>0.377048 </td><td>0.385820 </td><td>0.134512 </td><td>0.027972 </td><td>0.005346 </td></tr><tr><td>RANDOM (stdv) </td><td>0.000943 </td><td>0.006363 </td><td>0.016408 </td><td>0.008298 </td><td>0.009771 </td><td>0.003329 </td><td>0.002414 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx041-T4" data-id="dsx041-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Frequencies of couples as a function of the shortest path (SP) distance for three classes of protein couples: (i) the selected 547 TF couples (namely BEST, first row), (ii) all the couples of TFs (namely ALL, second row) and iii) 10 random sample sets made of 547 randomly picked protein couples from the whole PPI (namely RANDOM, mean and standard deviation in the third and fourth row, respectively)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>SP 1 </th><th>SP 2 </th><th>SP 3 </th><th>SP 4 </th><th>SP 5 </th><th>SP 6 </th><th>SP 7 </th></tr></thead><tbody><tr><td>BEST </td><td>0.117318 </td><td>0.478585 </td><td>0.284916 </td><td>0.10987 </td><td>0.009311 </td><td>0 </td><td>0 </td></tr><tr><td>ALL </td><td>0.056789 </td><td>0.463694 </td><td>0.376242 </td><td>0.084080 </td><td>0.014167 </td><td>0.004661 </td><td>0.000368 </td></tr><tr><td>RANDOM (mean) </td><td>0.003291 </td><td>0.065601 </td><td>0.377048 </td><td>0.385820 </td><td>0.134512 </td><td>0.027972 </td><td>0.005346 </td></tr><tr><td>RANDOM (stdv) </td><td>0.000943 </td><td>0.006363 </td><td>0.016408 </td><td>0.008298 </td><td>0.009771 </td><td>0.003329 </td><td>0.002414 </td></tr></tbody></table></div><div content-id="dsx041-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>SP 1 </th><th>SP 2 </th><th>SP 3 </th><th>SP 4 </th><th>SP 5 </th><th>SP 6 </th><th>SP 7 </th></tr></thead><tbody><tr><td>BEST </td><td>0.117318 </td><td>0.478585 </td><td>0.284916 </td><td>0.10987 </td><td>0.009311 </td><td>0 </td><td>0 </td></tr><tr><td>ALL </td><td>0.056789 </td><td>0.463694 </td><td>0.376242 </td><td>0.084080 </td><td>0.014167 </td><td>0.004661 </td><td>0.000368 </td></tr><tr><td>RANDOM (mean) </td><td>0.003291 </td><td>0.065601 </td><td>0.377048 </td><td>0.385820 </td><td>0.134512 </td><td>0.027972 </td><td>0.005346 </td></tr><tr><td>RANDOM (stdv) </td><td>0.000943 </td><td>0.006363 </td><td>0.016408 </td><td>0.008298 </td><td>0.009771 </td><td>0.003329 </td><td>0.002414 </td></tr></tbody></table></div></div></p><p>In order to further validate and provide significance to obtained results, we computed the shortest paths between couples of TFs related to the Protein-Protein Interaction (PPI) network downloaded from String database.<a reveal-id="dsx041-B22" data-open="dsx041-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> We found a significant overall difference between shortest path distribution of the set of couples selected by our method and the set of all TFs couples. In <a reveal-id="dsx041-T4" data-open="dsx041-T4" class="link link-reveal link-table xref-fig">Table 4</a>, we report the percentages of couples as a function of the Shortest Path (SP) distances for three classes of protein couples: (i) the selected 547 TF couples (namely BEST, first row), (ii) all the couples of TFs (namely ALL, second row) and (iii) 10 random sample sets made of 547 randomly picked protein couples from the whole PPI (namely RANDOM, mean and standard deviation in the third and fourth row, respectively). The majority of couples (around 50%) related to TFs shows a SP equal to 2 both for couples obtained by our algorithm and for all TF couples, while only around 6% of protein random couples has a SP distance equal to 2. The most relevant result is in the difference of the frequency for SP = 1 (indicating a direct interaction) that is ∼0.12 for our best couples and ∼0.057 for all TF couples (i.e. the ratio is greater than 2), and also for SP = 3 (∼0.28 our best couples versus ∼0.37 for all TF couples). The distribution for the random protein data shows, not surprisingly, a complete different pattern (the most part of couples having SP equal to 3 and 4). Even considering the limitations of this kind of analysis—a PPI is a global representation of potential interactions between proteins (and consequently TFs) that not always (referring to time and space) is an actual interaction and, moreover, the results depend on the threshold chosen to select significant interactions—it reveals that TFs couples selected by our method show an overall stronger relationship than those in the set of all TFs couples. In particular, we chose a quite strict threshold of 0.7 (as reported in the Material and methods section), so the significance of results has to be found in the ratio between the number of our selected couples and all TFs couples at SP 1 (ratio equal to 2) rather than in the percentages (12% and 6%, approximately) that could significantly change since they depend on the threshold. </p>                            <h2 scrollto-destination=112483410 id="112483410" class="section-title" >4. Conclusions</h2>
<p>A better understanding of the mechanisms driving the regulation of protein expression is an essential requisite to shed light on the behaviour of cells. Transcription factors play a central role in this extremely complex task and it has been shown that they synergically co-operate in order to provide a fine tuning of protein expressions. Among the different methods able to detect and identify TFs interplay, a very important resource is the computational methods to which our work mainly refers. In this work, we present a mathematically well-founded procedure able to identify TFs couples that act together, inferring for several of those couples the biological context they are involved in. We introduced a new and robust statistical method based on the use of a good Bernoulli approximation and on a <em>z</em>-score measures able to discriminate between random and non-random co-occurrences of couple of TFs. We extended previous methods based on randomize sequences as in Refs. <a reveal-id="dsx041-B15" data-open="dsx041-B15" class="link link-ref link-reveal xref-bibr">15</a> and <a reveal-id="dsx041-B16" data-open="dsx041-B16" class="link link-ref link-reveal xref-bibr">16</a>. In order to avoid biases due to the structural similarity of different models Overlapping and Similarity scores were designed to select TF couples that are significant for the analysis. We used such a method to find pairs of associated transcription factors in the set of all human promoter sequences (selected with a careful analysis from all human transcripts) and we also performed enrichment analysis on the set of genes regulated by identified couples. This analysis provides consistence to our results but also provides a biological context to be associated to the couples. Moreover, we also performed network analysis showing that TFs couples identified by the algorithm here presented are closer than expected in the protein-protein interaction network in terms of shortest path.</p><p>To our knowledge this is the only work concerning synergy of Transcription Factors taking into account all those features. Some of the couples emerging in this study are already known to be linked in several biological contexts (such as Myc–E2F,<a reveal-id="dsx041-B24 dsx041-B25 dsx041-B26" data-open="dsx041-B24 dsx041-B25 dsx041-B26" class="link link-ref link-reveal xref-bibr"><sup>24–26</sup></a> while in other cases our results can lead to hypothesize links between TFs couples and diseases (as for the STAT family that strongly interact with TBM in the promoters of genes belonging to the region of chr5q13 involved in several diseases.<a reveal-id="dsx041-B27 dsx041-B28 dsx041-B29" data-open="dsx041-B27 dsx041-B28 dsx041-B29" class="link link-ref link-reveal xref-bibr"><sup>27–29</sup></a> Finally, some other couples were not previously identified (such as the couple MZF1–MAZ) that would deserves further investigation.</p><p>According to our algorithm two TFs could be associated in different ways, for example they could be co-operative in a strict sense or concurrent in the sense that the presence of one of the two impedes the presence of the other. A small size window, as the one we use, leads us to hypothesize that two transcription factor proteins that could bind sites within the window, either are sufficiently close to each other to physically interact, or only one of the two is able to bind its TFBS because of the steric hindrance.</p><p>This work represents a step in the direction of designing complex gene regulatory networks, and it provides information on TFs association that could be useful in this context. The identification of significant TFs couples could be of help in the view of artificially altering the regulation of genes by inhibiting the interaction between given TFs couples.</p>                            <h2 scrollto-destination=112483415 id="112483415" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=112483417 id="112483417" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx041-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mitchell</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Tjian</div> <div class="given-names">R.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Transcriptional regulation in mammalian cells by sequence-specific DNA binding proteins</div>, <div class="source">Science</div> , <div class="volume">245</div>, <div class="fpage">371</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulation%20in%20mammalian%20cells%20by%20sequence-specific%20DNA%20binding%20proteins&amp;author=P.J.%20Mitchell&amp;author=R.%20Tjian&amp;publication_year=1989&amp;journal=Science&amp;volume=245&amp;pages=371-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2667136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2667136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2667136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2667136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Latchman</div> <div class="given-names">D.S.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Transcription factors: an overview</div>, <div class="source">Int. J. Biochem. Cell Biol</div> ., <div class="volume">29</div>, <div class="fpage">1305</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factors%3A%20an%20overview&amp;author=D.S.%20Latchman&amp;publication_year=1997&amp;journal=Int.%20J.%20Biochem.%20Cell%20Biol&amp;volume=29&amp;pages=1305-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1357-2725(97)00085-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1357-2725(97)00085-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1357-2725(97)00085-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9570129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">H.X.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Rapid search for specific sites on DNA through conformational switch of nonspecifically bound proteins</div>, <div class="source">Proc. Nat. Acad. Sci. U.S.A</div> ., <div class="volume">108</div>, <div class="fpage">8651</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20search%20for%20specific%20sites%20on%20DNA%20through%20conformational%20switch%20of%20nonspecifically%20bound%20proteins&amp;author=H.X.%20Zhou&amp;publication_year=2011&amp;journal=Proc.%20Nat.%20Acad.%20Sci.%20U.S.A&amp;volume=108&amp;pages=8651-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1101555108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1101555108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1101555108"> </span></div> </div></div></div></div></div><div content-id="dsx041-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berg</div> <div class="given-names">O.G.</div></div>, <div class="name string-name"><div class="surname">Winter</div> <div class="given-names">R.B.</div></div>, <div class="name string-name"><div class="surname">Von Hippel</div> <div class="given-names">P.H.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">Diffusion-driven mechanisms of protein translocation on nucleic acids. 1. Models and theory</div>, <div class="source">Biochemistry</div> , <div class="volume">20</div>, <div class="fpage">6929</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diffusion-driven%20mechanisms%20of%20protein%20translocation%20on%20nucleic%20acids.%201.%20Models%20and%20theory&amp;author=O.G.%20Berg&amp;author=R.B.%20Winter&amp;author=P.H.%20Von%20Hippel&amp;publication_year=1981&amp;journal=Biochemistry&amp;volume=20&amp;pages=6929-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00527a028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00527a028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00527a028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7317363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pahl</div> <div class="given-names">H.L.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Activators and target genes of Rel/NF-kB transcription factors</div>, <div class="source">Oncogene</div> , <div class="volume">18</div>, <div class="fpage">6853</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Activators%20and%20target%20genes%20of%20Rel%2FNF-kB%20transcription%20factors&amp;author=H.L.%20Pahl&amp;publication_year=1999&amp;journal=Oncogene&amp;volume=18&amp;pages=6853-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1203239" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1203239" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1203239"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10602461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wingender</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hehl</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2000</div>, 
<div class="article-title">TRANSFAC: an integrated system for gene expression regulation</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">28</div>, <div class="fpage">316</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TRANSFAC%3A%20an%20integrated%20system%20for%20gene%20expression%20regulation&amp;author=E.%20Wingender&amp;author=X.%20Chen&amp;author=R.%20Hehl&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;pages=316-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.316" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.316" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.316"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wingender</div> <div class="given-names">E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The TRANSFAC project as an example of framework technology that supports the analysis of genomic regulation</div>, <div class="source">Brief. Bioinform</div> ., <div class="volume">9</div>, <div class="fpage">326</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TRANSFAC%20project%20as%20an%20example%20of%20framework%20technology%20that%20supports%20the%20analysis%20of%20genomic%20regulation&amp;author=E.%20Wingender&amp;publication_year=2008&amp;journal=Brief.%20Bioinform&amp;volume=9&amp;pages=326-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbn016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18436575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sandelin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Alkema</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Engström</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Wasserman</div> <div class="given-names">W.W.</div></div>, <div class="name string-name"><div class="surname">Lenhard</div> <div class="given-names">B.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">JASPAR: an open-access database for eukaryotic transcription factor binding profiles</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">D91</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=JASPAR%3A%20an%20open-access%20database%20for%20eukaryotic%20transcription%20factor%20binding%20profiles&amp;author=A.%20Sandelin&amp;author=W.%20Alkema&amp;author=P.%20Engstr%C3%B6m&amp;author=W.W.%20Wasserman&amp;author=B.%20Lenhard&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=D91-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tompa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">L.T.</div></div>, <div class="name string-name"><div class="surname">Church</div> <div class="given-names">G.M.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Assessing computational tools for the discovery of transcription factor binding sites</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">23</div>, <div class="fpage">137</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20computational%20tools%20for%20the%20discovery%20of%20transcription%20factor%20binding%20sites&amp;author=M.%20Tompa&amp;author=N.%20Li&amp;author=L.T.%20Bailey&amp;author=G.M.%20Church&amp;publication_year=2005&amp;journal=Nat.%20Biotechnol&amp;volume=23&amp;pages=137-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15637633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suryamohan</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Halfon</div> <div class="given-names">M.S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Identifying transcriptional cis-regulatory modules in animal genomes</div>, <div class="source">Wires. Dev. Biol</div> ., <div class="volume">4</div>, <div class="fpage">59</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20transcriptional%20cis-regulatory%20modules%20in%20animal%20genomes&amp;author=K.%20Suryamohan&amp;author=M.S.%20Halfon&amp;publication_year=2015&amp;journal=Wires.%20Dev.%20Biol&amp;volume=4&amp;pages=59-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/wdev.168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwdev.168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fwdev.168"> </span></div> </div></div></div></div></div><div content-id="dsx041-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blanchette</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bataille</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Genome-wide computational prediction of transcriptional regulatory modules reveals new insights into human gene expression. <em>Genome.</em></div> <div class="source">Res</div> ., <div class="volume">16</div>, <div class="fpage">656</div>–<div class="lpage">68</div>.</div></div></div></div><div content-id="dsx041-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Prabhakar</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Poulin</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Shoukry</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Close sequence comparisons are sufficient to identify human cis-regulatory elements</div>, <div class="source">Genome Res</div> ., <div class="volume">16</div>, <div class="fpage">855</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Close%20sequence%20comparisons%20are%20sufficient%20to%20identify%20human%20cis-regulatory%20elements&amp;author=S.%20Prabhakar&amp;author=F.%20Poulin&amp;author=M.%20Shoukry&amp;publication_year=2006&amp;journal=Genome%20Res&amp;volume=16&amp;pages=855-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4717506" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4717506" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4717506"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Collins</div> <div class="given-names">J.F.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Prediction of synergistic transcription factors by function conservation</div>, <div class="source">Genome Biol</div> ., <div class="volume">8</div>, <div class="fpage">R257</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20synergistic%20transcription%20factors%20by%20function%20conservation&amp;author=Z.%20Hu&amp;author=B.%20Hu&amp;author=J.F.%20Collins&amp;publication_year=2007&amp;journal=Genome%20Biol&amp;volume=8&amp;pages=R257" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-12-r257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-12-r257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-12-r257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18053230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zack</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Computational analysis of tissue-specific combinatorial gene regulation: predicting interaction between transcription factors in human tissues</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">4925</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20analysis%20of%20tissue-specific%20combinatorial%20gene%20regulation%3A%20predicting%20interaction%20between%20transcription%20factors%20in%20human%20tissues&amp;author=X.%20Yu&amp;author=J.%20Lin&amp;author=D.J.%20Zack&amp;author=J.%20Qian&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=4925-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl595" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl595"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16982645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nandi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Blais</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ioshikhes</div> <div class="given-names">I.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Identification of cis-regulatory modules in promoters of human genes exploiting mutual positioning of transcription factors</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">8822</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20cis-regulatory%20modules%20in%20promoters%20of%20human%20genes%20exploiting%20mutual%20positioning%20of%20transcription%20factors&amp;author=S.%20Nandi&amp;author=A.%20Blais&amp;author=I.%20Ioshikhes&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=8822-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt578" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt578" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt578"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23913413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hannenhalli</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Levy</div> <div class="given-names">S.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Predicting transcription factor synergism</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">4278</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20transcription%20factor%20synergism&amp;author=S.%20Hannenhalli&amp;author=S.%20Levy&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=4278-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf535" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf535" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf535"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12364607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Sugnet</div> <div class="given-names">C.W.</div></div>, <div class="name string-name"><div class="surname">Furey</div> <div class="given-names">T.S.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">The human genome browser at UCSC</div>, <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">996</div>–<div class="lpage">1006</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20genome%20browser%20at%20UCSC&amp;author=W.J.%20Kent&amp;author=C.W.%20Sugnet&amp;author=T.S.%20Furey&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=996-1006" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12045153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gentleman</div> <div class="given-names">R.C.</div></div>, <div class="name string-name"><div class="surname">Carey</div> <div class="given-names">V.J.</div></div>, <div class="name string-name"><div class="surname">Bates</div> <div class="given-names">D.M.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Bioconductor: open software development for computational biology and bioinformatics</div>, <div class="source">Genome Biol</div> ., <div class="volume">5</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioconductor%3A%20open%20software%20development%20for%20computational%20biology%20and%20bioinformatics&amp;author=R.C.%20Gentleman&amp;author=V.J.%20Carey&amp;author=D.M.%20Bates&amp;publication_year=2004&amp;journal=Genome%20Biol&amp;volume=5&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-10-r80" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-10-r80"> </span></div> </div></div></div></div></div><div content-id="dsx041-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marinescu</div> <div class="given-names">V.D.</div></div>, <div class="name string-name"><div class="surname">Hohane</div> <div class="given-names">I.S.</div></div>, <div class="name string-name"><div class="surname">Riva</div> <div class="given-names">A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The MAPPER database: a multi-genome catalog of putative transcription factor binding sites</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">33 (suppl 1</div>), <div class="fpage">D91</div>–<div class="lpage">7</div>. [CVOCROSSCVO]</div></div></div></div><div content-id="dsx041-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pagès</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Aboyoun</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gentleman</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">DebRoy</div> <div class="given-names">S.</div></div></div> <div class="year">2009</div>, String objects representing biological sequences, and matching algorithms. R package version 2.2.</div></div></div></div><div content-id="dsx041-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div> <div class="given-names">J.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Divergence measures based on the shannon entropy</div>, <div class="source">IEEE Trans. Inform. Theory</div> , <div class="volume">37</div>, <div class="fpage">145</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergence%20measures%20based%20on%20the%20shannon%20entropy&amp;author=J.%20Lin&amp;publication_year=1991&amp;journal=IEEE%20Trans.%20Inform.%20Theory&amp;volume=37&amp;pages=145-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/18.61115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2F18.61115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2F18.61115"> </span></div> </div></div></div></div></div><div content-id="dsx041-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Szklarczyk</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Morris</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Cook</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">The STRING database in 2017: quality-controlled protein-protein association networks, made broadly accessible</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">45</div>, <div class="fpage">D362</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20STRING%20database%20in%202017%3A%20quality-controlled%20protein-protein%20association%20networks%2C%20made%20broadly%20accessible&amp;author=D.%20Szklarczyk&amp;author=J.H.%20Morris&amp;author=H.%20Cook&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=D362-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw937" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw937" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw937"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27924014" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.K.</div></div></div>, ed. <div class="year">2010</div>, <div class="source">Statistical Bioinformatics for Biomedical and Life Science Researchers</div> . 
<div class="publisher-name">Wiley-Blackwell</div>, 
<div class="publisher-loc">Hoboken, New Jersey</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20Bioinformatics%20for%20Biomedical%20and%20Life%20Science%20Researchers&amp;author=J.K.%20Lee&amp;publication_year=2010&amp;book=Statistical%20Bioinformatics%20for%20Biomedical%20and%20Life%20Science%20Researchers" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/9780470567647" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F9780470567647" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F9780470567647"> </span></div> </div></div></div></div></div><div content-id="dsx041-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dannenberg</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">David</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">van der Torre</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">W.H.</div></div>, <div class="name string-name"><div class="surname">Depinho</div> <div class="given-names">R.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">mSin3A corepressor regulates diverse transcriptional networks governing normal and neoplastic growth and survival</div>, <div class="source">Genes Dev</div> ., <div class="volume">19</div>, <div class="fpage">1581</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mSin3A%20corepressor%20regulates%20diverse%20transcriptional%20networks%20governing%20normal%20and%20neoplastic%20growth%20and%20survival&amp;author=J.H.%20Dannenberg&amp;author=G.%20David&amp;author=S.%20Zhong&amp;author=J.%20van%20der%20Torre&amp;author=W.H.%20Wong&amp;author=R.A.%20Depinho&amp;publication_year=2005&amp;journal=Genes%20Dev&amp;volume=19&amp;pages=1581-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1286905" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1286905" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1286905"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15998811" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Allard</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Utley</div> <div class="given-names">R.T.</div></div>, <div class="name string-name"><div class="surname">Savard</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">1999</div>, 
<div class="article-title">NuA4, an essential transcription adaptor/histone H4 acetyltransferase complex containing Esa1p and the ATM-related cofactor Tra1p</div>, <div class="source">Embo J</div> ., <div class="volume">18</div>, <div class="fpage">5108</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NuA4%2C%20an%20essential%20transcription%20adaptor%2Fhistone%20H4%20acetyltransferase%20complex%20containing%20Esa1p%20and%20the%20ATM-related%20cofactor%20Tra1p&amp;author=S.%20Allard&amp;author=R.T.%20Utley&amp;author=J.%20Savard&amp;publication_year=1999&amp;journal=Embo%20J&amp;volume=18&amp;pages=5108-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/18.18.5108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F18.18.5108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F18.18.5108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10487762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albert</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wells</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Funk</div> <div class="given-names">J.O.</div></div></div>, et al.   <div class="year">2001</div>, 
<div class="article-title">The chromatin structure of the dual c-myc promoter P1/P2 is regulated by separate elements</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">276</div>, <div class="fpage">20482</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chromatin%20structure%20of%20the%20dual%20c-myc%20promoter%20P1%2FP2%20is%20regulated%20by%20separate%20elements&amp;author=T.%20Albert&amp;author=J.%20Wells&amp;author=J.O.%20Funk&amp;publication_year=2001&amp;journal=J.%20Biol.%20Chem&amp;volume=276&amp;pages=20482-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M100265200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M100265200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M100265200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11279041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Hartz</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.C.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Copy number variations in 6q14. 1 and 5q13. 2 are associated with alcohol dependence</div>, <div class="source">Alcohol. Clin. Exp. Res</div> ., <div class="volume">36</div>, <div class="fpage">1512</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variations%20in%206q14.%201%20and%205q13.%202%20are%20associated%20with%20alcohol%20dependence&amp;author=P.%20Lin&amp;author=S.M.%20Hartz&amp;author=J.C.%20Wang&amp;publication_year=2012&amp;journal=Alcohol.%20Clin.%20Exp.%20Res&amp;volume=36&amp;pages=1512-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/acer.2012.36.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Facer.2012.36.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Facer.2012.36.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22702843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Also-Rallo</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Alias</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Martinez-Hernandez</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Treatment of spinal muscular atrophy cells with drugs that upregulate SMN expression reveals inter-and intra-patient variability</div>, <div class="source">Eur. J. Hum. Genet</div> ., <div class="volume">19</div>, <div class="fpage">1059</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Treatment%20of%20spinal%20muscular%20atrophy%20cells%20with%20drugs%20that%20upregulate%20SMN%20expression%20reveals%20inter-and%20intra-patient%20variability&amp;author=E.%20Also-Rallo&amp;author=L.%20Alias&amp;author=R.%20Martinez-Hernandez&amp;publication_year=2011&amp;journal=Eur.%20J.%20Hum.%20Genet&amp;volume=19&amp;pages=1059-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ejhg.2011.89" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fejhg.2011.89" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fejhg.2011.89"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21610752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Ivanova</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Merup</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">1999</div>, 
<div class="article-title">Molecular analysis of the human chromosome 5q13. 3 region in patients with hairy cell leukemia and identification of tumor suppressor gene candidates</div>, <div class="source">Genomics</div> , <div class="volume">60</div>, <div class="fpage">161</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20analysis%20of%20the%20human%20chromosome%205q13.%203%20region%20in%20patients%20with%20hairy%20cell%20leukemia%20and%20identification%20of%20tumor%20suppressor%20gene%20candidates&amp;author=X.%20Wu&amp;author=G.%20Ivanova&amp;author=M.%20Merup&amp;publication_year=1999&amp;journal=Genomics&amp;volume=60&amp;pages=161-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1999.5911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1999.5911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1999.5911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10486207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zweier</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Albrecht</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Mitulla</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">‘Mowat-Wilson’ syndrome with and without hirschsprung disease is a distinct, recognizable multiple congenital anomalies-mental retardation syndrome caused by mutations in the zinc finger homeo box 1B</div>, <div class="source">Am. J. Med. Genet</div> ., <div class="volume">108</div>, <div class="fpage">177</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%E2%80%98Mowat-Wilson%E2%80%99%20syndrome%20with%20and%20without%20hirschsprung%20disease%20is%20a%20distinct%2C%20recognizable%20multiple%20congenital%20anomalies-mental%20retardation%20syndrome%20caused%20by%20mutations%20in%20the%20zinc%20finger%20homeo%20box%201B&amp;author=C.%20Zweier&amp;author=B.%20Albrecht&amp;author=B.%20Mitulla&amp;publication_year=2002&amp;journal=Am.%20J.%20Med.%20Genet&amp;volume=108&amp;pages=177-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1096-8628" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1096-8628" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1096-8628"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11891681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lantinga-van Leeuwen</div> <div class="given-names">I.S.</div></div>, <div class="name string-name"><div class="surname">Leonhard</div> <div class="given-names">W.N.</div></div>, <div class="name string-name"><div class="surname">Dauwerse</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Common regulatory elements in the polycystic kidney disease 1 and 2 promoter regions</div>, <div class="source">Eur. J. Hum. Genet</div> ., <div class="volume">13</div>, <div class="fpage">649</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20regulatory%20elements%20in%20the%20polycystic%20kidney%20disease%201%20and%202%20promoter%20regions&amp;author=I.S.%20Lantinga-van%20Leeuwen&amp;author=W.N.%20Leonhard&amp;author=H.%20Dauwerse&amp;publication_year=2005&amp;journal=Eur.%20J.%20Hum.%20Genet&amp;volume=13&amp;pages=649-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.ejhg.5201392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.ejhg.5201392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.ejhg.5201392"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15770226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx041-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx041-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bogni</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Genome-wide approach to identify risk factors for therapy-related myeloid leukemia</div>, <div class="source">Leukemia</div> , <div class="volume">20</div>, <div class="fpage">239</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20approach%20to%20identify%20risk%20factors%20for%20therapy-related%20myeloid%20leukemia&amp;author=A.%20Bogni&amp;author=C.%20Cheng&amp;author=W.%20Liu&amp;publication_year=2006&amp;journal=Leukemia&amp;volume=20&amp;pages=239-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.leu.2404059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.leu.2404059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.leu.2404059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16341039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4562529&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">528</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx041"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4562529"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f103%2f4562529">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F103%2F4562529" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4562529'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Investigating transcription factor synergism in humans" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx041");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f103%2f4562529">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

