

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/5/269/350906","@type":"ScholarlyArticle","name":"Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi016","keywords":["transcription factor","domain organization","DNA binding","prokaryote","comparative genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/5/269/350906","author":[{"name":"Minezaki, Yoshiaki","affiliation":"1  Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems  1111 Yata, Mishima, Shizuoka, 411-8540 Japan","@type":"Person"},{"name":"Homma, Keiichi","affiliation":"1  Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems  1111 Yata, Mishima, Shizuoka, 411-8540 Japan    2  Japan Science and Technology Corporation  1–8, Honcho 4-chome, Kawaguchi City, Saitama, 332-0012, Japan","@type":"Person"},{"name":"Nishikawa, Ken","affiliation":"1  Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems  1111 Yata, Mishima, Shizuoka, 411-8540 Japan","@type":"Person"}],"description":"Abstract.  Assignment of all transcription factors (TFs) from genome sequence data is not a straightforward task due to the wide variation in TFs among differen","pageStart":"269","pageEnd":"280","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/5/0/m_dnaresearch12_5.cover.gif?Expires=2147483647&Signature=aCda7p9Uq5iwZjUzLjBT9gM9GcXHBhpbSpHktSZ3aBfaLgUTRrUBMfrhnbDOftVRcOxJYPlWQoOSDtYCMRCSqLqznp4erEvRKXpqo14qAWzUQzXAUf0R2hzkxTrttMY8s2309oR8tNHgB9rVuF2hpqh3QH4eyjVeWXlnJ40m9gcWoDr-jQH7WNKVcWkYKOzDqj0zz3HLWNzUSiszlQ7mPDPy4j~szH2kDsY0ptS4bBqTMPSDLuVeIksQM7hpXYsVJSgGsIGUFqHvKMFIXeS6xBjZCvV-jy2mqIn0wg5gOITNC5m-u-f0z~ur8QMlAsoeOXCWmD3HyGddqFtbkE9Www__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/5/0/m_dnaresearch12_5.cover.gif?Expires=2147483647&Signature=aCda7p9Uq5iwZjUzLjBT9gM9GcXHBhpbSpHktSZ3aBfaLgUTRrUBMfrhnbDOftVRcOxJYPlWQoOSDtYCMRCSqLqznp4erEvRKXpqo14qAWzUQzXAUf0R2hzkxTrttMY8s2309oR8tNHgB9rVuF2hpqh3QH4eyjVeWXlnJ40m9gcWoDr-jQH7WNKVcWkYKOzDqj0zz3HLWNzUSiszlQ7mPDPy4j~szH2kDsY0ptS4bBqTMPSDLuVeIksQM7hpXYsVJSgGsIGUFqHvKMFIXeS6xBjZCvV-jy2mqIn0wg5gOITNC5m-u-f0z~ur8QMlAsoeOXCWmD3HyGddqFtbkE9Www__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea" />
    <meta property="og:description" content="Abstract.  Assignment of all transcription factors (TFs) from genome sequence data is not a straightforward task due to the wide variation in TFs among differen" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/5/269/350906" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Minezaki, Yoshiaki" /><meta name="citation_author_institution" content="1Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems1111 Yata, Mishima, Shizuoka, 411-8540 Japan" /><meta name="citation_author" content="Homma, Keiichi" /><meta name="citation_author_institution" content="1Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems1111 Yata, Mishima, Shizuoka, 411-8540 Japan" /><meta name="citation_author_institution" content="2Japan Science and Technology Corporation1–8, Honcho 4-chome, Kawaguchi City, Saitama, 332-0012, Japan" /><meta name="citation_author" content="Nishikawa, Ken" /><meta name="citation_author_institution" content="1Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems1111 Yata, Mishima, Shizuoka, 411-8540 Japan" /><meta name="citation_title" content="Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea" /><meta name="citation_firstpage" content="269" /><meta name="citation_lastpage" content="280" /><meta name="citation_doi" content="10.1093/dnares/dsi016" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Functional modulation of Escherichia coli RNA polymerase; Annu. Rev. Microbiol.;  citation_year=2000;  citation_volume=54;  citation_pages=499-518; " /><meta name="citation_reference" content="citation_title=Transcriptional regulation in archaea; Curr. Opin. Genet. Dev.;  citation_year=2004;  citation_volume=14;  citation_pages=133-138; " /><meta name="citation_reference" content="citation_title=Arabidopsis transcription factors: genome-wide comparative analysis among eukaryotes; Science;  citation_year=2000;  citation_volume=290;  citation_pages=2105-2110; " /><meta name="citation_reference" content="citation_title=The phylogenetic diversity of eukaryotic transcription; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=653-660; " /><meta name="citation_reference" content="citation_title=TRANSFAC: transcriptional regulation, from patterns to profiles; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=374-378; " /><meta name="citation_reference" content="citation_title=The repertoire of DNA-binding transcriptional regulators in Escherichia coli K-12; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=1838-1847; " /><meta name="citation_reference" content="citation_title=DNA-binding proteins and evolution of transcription regulation in the archaea; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4658-4670; " /><meta name="citation_reference" content="citation_title=Transcription in archaea; Proc. Natl Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=8545-8550; " /><meta name="citation_reference" content="citation_title=Transcription regulation and environmental adaptation in bacteria; Trends Microbiol.;  citation_year=2003;  citation_volume=11;  citation_pages=248-253; " /><meta name="citation_reference" content="citation_title=Evolution of protein superfamilies and bacterial genome size; J. Mol. Biol.;  citation_year=2004;  citation_volume=336;  citation_pages=871-887; " /><meta name="citation_reference" content="citation_title=CATH—a hierarchic classification of protein domain structures; Structure;  citation_year=1997;  citation_volume=5;  citation_pages=1093-1108; " /><meta name="citation_reference" content="citation_title=BacTregulators: a database of transcriptional regulators in bacteria and archaea; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2787-2791; " /><meta name="citation_reference" content="citation_title=Evolution of transcription factors and the gene regulatory network in Escherichia coli; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=1234-1244; " /><meta name="citation_reference" content="citation_title=SCOP: a structural classification of proteins database for the investigation of sequences and structures; J. Mol. Biol.;  citation_year=1995;  citation_volume=247;  citation_pages=536-540; " /><meta name="citation_reference" content="citation_title=GTOP: a database of protein structures predicted from genome sequences; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=294-298; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=The Protein Data Bank; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=235-242; " /><meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt); Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D154-D159; " /><meta name="citation_reference" content="citation_title=Profile hidden Markov models; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=755-763; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D138-D141; " /><meta name="citation_reference" content="citation_title=Archaeal chromatin and transcription; Mol. Microbiol.;  citation_year=2003;  citation_volume=48;  citation_pages=587-598; " /><meta name="citation_reference" content="citation_title=Rho factor: transcription termination in four steps; Curr. Biol.;  citation_year=2003;  citation_volume=13;  citation_pages=R714-R716; " /><meta name="citation_reference" content="citation_title=Recent developments in bacterial cold-shock response; Curr. Issues Mol. Biol.;  citation_year=2004;  citation_volume=6;  citation_pages=125-36; " /><meta name="citation_reference" content="citation_title=Structure analysis of PH1161 protein, a transcriptional activator TenA homologue from the hyperthermophilic archaeon Pyrococcus horikoshii; Acta Crystallogr. D Biol. Crystallogr.;  citation_year=2004;  citation_volume=60;  citation_pages=1094-1100; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=Supra-domains: evolutionary units larger than single protein domains; J. Mol. Biol.;  citation_year=2004;  citation_volume=336;  citation_pages=809-823; " /><meta name="citation_reference" content="citation_title=Protein family and fold occurrence in genomes: power-law behaviour and evolutionary model; J. Mol. Biol.;  citation_year=2001;  citation_volume=313;  citation_pages=673-681; " /><meta name="citation_reference" content="citation_title=DnaA initiator—also a transcription factor; Mol. Microbiol.;  citation_year=1997;  citation_volume=24;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the marine planctomycete Pirellula sp . strain 1; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=8298-8303; " /><meta name="citation_reference" content="citation_title=Characterization of genes encoding multi-domain proteins in the genome of the filamentous nitrogen-fixing cyanobacterium Anabaena sp . strain PCC 7120; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=271-284; " /><meta name="citation_reference" content="citation_title=The Lrp family of transcriptional regulators; Mol. Microbiol.;  citation_year=2003;  citation_volume=48;  citation_pages=287-294; " /><meta name="citation_reference" content="citation_title=The archaeal feast/famine regulatory protein: potential roles of its assembly forms for regulating transcription; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=2840-2845; " /><meta name="citation_reference" content="citation_title=Transcription-associated protein families are primarily taxon-specific; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=95-97; " /><meta name="citation_reference" content="citation_title=Genome sequence of the endocellular bacterial symbiont of aphids Buchnera sp . APS; Nature;  citation_year=2000;  citation_volume=407;  citation_pages=81-86; " /><meta name="citation_reference" content="citation_title=Regulation of transcription in a reduced bacterial genome: nutrient-provisioning genes of the obligate symbiont Buchnera aphidicola; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=4229-4237; " /><meta name="citation_reference" content="citation_title=The genome of M. acetivorans reveals extensive metabolic and physiological diversity; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=532-542; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the hyperthermophilic, sulphate-reducing archaeon Archaeoglobus fulgidus; Nature;  citation_year=1997;  citation_volume=390;  citation_pages=364-370; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the model actinomycete Streptomyces coelicolor A3(2); Nature;  citation_year=2002;  citation_volume=417;  citation_pages=141-147; " /><meta name="citation_reference" content="citation_title=The genome sequence of Bifidobacterium longum reflects its adaptation to the human gastrointestinal tract; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=14422-14427; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis; Nature;  citation_year=1997;  citation_volume=390;  citation_pages=249-256; " /><meta name="citation_reference" content="citation_title=Genome sequence and comparative analysis of the solvent-producing bacterium Clostridium acetobutylicum; J. Bacteriol.;  citation_year=2001;  citation_volume=183;  citation_pages=4823-4838; " /><meta name="citation_reference" content="citation_title=The genome sequence of the entomopathogenic bacterium Photorhabdus luminescens; Nat. Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=1307-1313; " /><meta name="citation_reference" content="citation_title=The genome sequence of the probiotic intestinal bacterium Lactobacillus johnsonii NCC 533; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=2512-2517; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the lactic acid bacterium Lactococcus lactis ssp. lactis IL1403; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=731-753; " /><meta name="citation_reference" content="citation_title=Genome sequence and analysis of the oral bacterium Fusobacterium nucleatum strain ATCC 25586; J. Bacteriol.;  citation_year=2002;  citation_volume=184;  citation_pages=2005-2018; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the metabolically versatile photosynthetic bacterium Rhodopseudomonas palustris; Nat. Biotechnol.;  citation_year=2004;  citation_volume=22;  citation_pages=55-61; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/5/269/6718920/dsi016.pdf" /><meta name="description" content="Abstract.  Assignment of all transcription factors (TFs) from genome sequence data is not a straightforward task due to the wide variation in TFs among differen" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/5/269/350906" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/5/269/350906" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/5/269/350906");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi016';
    </script>
<title>
	Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./350906" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaBhPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaW5lemFraSwgWW9zaGlha2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxTGFib3JhdG9yeSBvZiBHZW5lLVByb2R1Y3QgSW5mb3JtYXRpY3MsIENlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neeKAk0ROQSBEYXRhIEJhbmsgb2YgSmFwYW4sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgUmVzZWFyY2ggT3JnYW5pemF0aW9uIG9mIEluZm9ybWF0aW9uIGFuZCBTeXN0ZW1zMTExMSBZYXRhLCBNaXNoaW1hLCBTaGl6dW9rYSwgNDExLTg1NDAgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb21tYSwgS2VpaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFMYWJvcmF0b3J5IG9mIEdlbmUtUHJvZHVjdCBJbmZvcm1hdGljcywgQ2VudGVyIGZvciBJbmZvcm1hdGlvbiBCaW9sb2d54oCTRE5BIERhdGEgQmFuayBvZiBKYXBhbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBSZXNlYXJjaCBPcmdhbml6YXRpb24gb2YgSW5mb3JtYXRpb24gYW5kIFN5c3RlbXMxMTExIFlhdGEsIE1pc2hpbWEsIFNoaXp1b2thLCA0MTEtODU0MCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJKYXBhbiBTY2llbmNlIGFuZCBUZWNobm9sb2d5IENvcnBvcmF0aW9uMeKAkzgsIEhvbmNobyA0LWNob21lLCBLYXdhZ3VjaGkgQ2l0eSwgU2FpdGFtYSwgMzMyLTAwMTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmlzaGlrYXdhLCBLZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxTGFib3JhdG9yeSBvZiBHZW5lLVByb2R1Y3QgSW5mb3JtYXRpY3MsIENlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neeKAk0ROQSBEYXRhIEJhbmsgb2YgSmFwYW4sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgUmVzZWFyY2ggT3JnYW5pemF0aW9uIG9mIEluZm9ybWF0aW9uIGFuZCBTeXN0ZW1zMTExMSBZYXRhLCBNaXNoaW1hLCBTaGl6dW9rYSwgNDExLTg1NDAgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS1XaWRlIFN1cnZleSBvZiBUcmFuc2NyaXB0aW9uIEZhY3RvcnMgaW4gUHJva2FyeW90ZXMgUmV2ZWFscyBNYW55IEJhY3RlcmlhLVNwZWNpZmljIEZhbWlsaWVzIE5vdCBGb3VuZCBpbiBBcmNoYWVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjY5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc2kwMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNS8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgbW9kdWxhdGlvbiBvZiBFc2NoZXJpY2hpYSBjb2xpIFJOQSBwb2x5bWVyYXNlOyBBbm51LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz00OTktNTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcmVndWxhdGlvbiBpbiBhcmNoYWVhOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEzMy0xMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIHRyYW5zY3JpcHRpb24gZmFjdG9yczogZ2Vub21lLXdpZGUgY29tcGFyYXRpdmUgYW5hbHlzaXMgYW1vbmcgZXVrYXJ5b3RlczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTA7ICBjaXRhdGlvbl9wYWdlcz0yMTA1LTIxMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwaHlsb2dlbmV0aWMgZGl2ZXJzaXR5IG9mIGV1a2FyeW90aWMgdHJhbnNjcmlwdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NjUzLTY2MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VFJBTlNGQUM6IHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uLCBmcm9tIHBhdHRlcm5zIHRvIHByb2ZpbGVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNzQtMzc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVwZXJ0b2lyZSBvZiBETkEtYmluZGluZyB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9ycyBpbiBFc2NoZXJpY2hpYSBjb2xpIEstMTI7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTE4MzgtMTg0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BLWJpbmRpbmcgcHJvdGVpbnMgYW5kIGV2b2x1dGlvbiBvZiB0cmFuc2NyaXB0aW9uIHJlZ3VsYXRpb24gaW4gdGhlIGFyY2hhZWE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTQ2NTgtNDY3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBpbiBhcmNoYWVhOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9ODU0NS04NTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIHJlZ3VsYXRpb24gYW5kIGVudmlyb25tZW50YWwgYWRhcHRhdGlvbiBpbiBiYWN0ZXJpYTsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNDgtMjUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgcHJvdGVpbiBzdXBlcmZhbWlsaWVzIGFuZCBiYWN0ZXJpYWwgZ2Vub21lIHNpemU7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzM2OyAgY2l0YXRpb25fcGFnZXM9ODcxLTg4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FUSOKAlGEgaGllcmFyY2hpYyBjbGFzc2lmaWNhdGlvbiBvZiBwcm90ZWluIGRvbWFpbiBzdHJ1Y3R1cmVzOyBTdHJ1Y3R1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEwOTMtMTEwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjVHJlZ3VsYXRvcnM6IGEgZGF0YWJhc2Ugb2YgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvcnMgaW4gYmFjdGVyaWEgYW5kIGFyY2hhZWE7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9Mjc4Ny0yNzkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGFuZCB0aGUgZ2VuZSByZWd1bGF0b3J5IG5ldHdvcmsgaW4gRXNjaGVyaWNoaWEgY29saTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTIzNC0xMjQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TQ09QOiBhIHN0cnVjdHVyYWwgY2xhc3NpZmljYXRpb24gb2YgcHJvdGVpbnMgZGF0YWJhc2UgZm9yIHRoZSBpbnZlc3RpZ2F0aW9uIG9mIHNlcXVlbmNlcyBhbmQgc3RydWN0dXJlczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDc7ICBjaXRhdGlvbl9wYWdlcz01MzYtNTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HVE9QOiBhIGRhdGFiYXNlIG9mIHByb3RlaW4gc3RydWN0dXJlcyBwcmVkaWN0ZWQgZnJvbSBnZW5vbWUgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yOTQtMjk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS0zNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUHJvdGVpbiBEYXRhIEJhbms7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTIzNS0yNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBVbml2ZXJzYWwgUHJvdGVpbiBSZXNvdXJjZSAoVW5pUHJvdCk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPUQxNTQtRDE1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZmlsZSBoaWRkZW4gTWFya292IG1vZGVsczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz03NTUtNzYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1EMTM4LUQxNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyY2hhZWFsIGNocm9tYXRpbiBhbmQgdHJhbnNjcmlwdGlvbjsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NTg3LTU5ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmhvIGZhY3RvcjogdHJhbnNjcmlwdGlvbiB0ZXJtaW5hdGlvbiBpbiBmb3VyIHN0ZXBzOyBDdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPVI3MTQtUjcxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjZW50IGRldmVsb3BtZW50cyBpbiBiYWN0ZXJpYWwgY29sZC1zaG9jayByZXNwb25zZTsgQ3Vyci4gSXNzdWVzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIGFuYWx5c2lzIG9mIFBIMTE2MSBwcm90ZWluLCBhIHRyYW5zY3JpcHRpb25hbCBhY3RpdmF0b3IgVGVuQSBob21vbG9ndWUgZnJvbSB0aGUgaHlwZXJ0aGVybW9waGlsaWMgYXJjaGFlb24gUHlyb2NvY2N1cyBob3Jpa29zaGlpOyBBY3RhIENyeXN0YWxsb2dyLiBEIEJpb2wuIENyeXN0YWxsb2dyLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTEwOTQtMTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENPRyBkYXRhYmFzZTogYW4gdXBkYXRlZCB2ZXJzaW9uIGluY2x1ZGVzIGV1a2FyeW90ZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdXByYS1kb21haW5zOiBldm9sdXRpb25hcnkgdW5pdHMgbGFyZ2VyIHRoYW4gc2luZ2xlIHByb3RlaW4gZG9tYWluczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzY7ICBjaXRhdGlvbl9wYWdlcz04MDktODIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGZhbWlseSBhbmQgZm9sZCBvY2N1cnJlbmNlIGluIGdlbm9tZXM6IHBvd2VyLWxhdyBiZWhhdmlvdXIgYW5kIGV2b2x1dGlvbmFyeSBtb2RlbDsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTM7ICBjaXRhdGlvbl9wYWdlcz02NzMtNjgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbmFBIGluaXRpYXRvcuKAlGFsc28gYSB0cmFuc2NyaXB0aW9uIGZhY3RvcjsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIG1hcmluZSBwbGFuY3RvbXljZXRlIFBpcmVsbHVsYSBzcCAuIHN0cmFpbiAxOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTgyOTgtODMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBnZW5lcyBlbmNvZGluZyBtdWx0aS1kb21haW4gcHJvdGVpbnMgaW4gdGhlIGdlbm9tZSBvZiB0aGUgZmlsYW1lbnRvdXMgbml0cm9nZW4tZml4aW5nIGN5YW5vYmFjdGVyaXVtIEFuYWJhZW5hIHNwIC4gc3RyYWluIFBDQyA3MTIwOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjcxLTI4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIExycCBmYW1pbHkgb2YgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvcnM7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTI4Ny0yOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhcmNoYWVhbCBmZWFzdC9mYW1pbmUgcmVndWxhdG9yeSBwcm90ZWluOiBwb3RlbnRpYWwgcm9sZXMgb2YgaXRzIGFzc2VtYmx5IGZvcm1zIGZvciByZWd1bGF0aW5nIHRyYW5zY3JpcHRpb247IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9Mjg0MC0yODQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uLWFzc29jaWF0ZWQgcHJvdGVpbiBmYW1pbGllcyBhcmUgcHJpbWFyaWx5IHRheG9uLXNwZWNpZmljOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTk1LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGVuZG9jZWxsdWxhciBiYWN0ZXJpYWwgc3ltYmlvbnQgb2YgYXBoaWRzIEJ1Y2huZXJhIHNwIC4gQVBTOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA3OyAgY2l0YXRpb25fcGFnZXM9ODEtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2YgdHJhbnNjcmlwdGlvbiBpbiBhIHJlZHVjZWQgYmFjdGVyaWFsIGdlbm9tZTogbnV0cmllbnQtcHJvdmlzaW9uaW5nIGdlbmVzIG9mIHRoZSBvYmxpZ2F0ZSBzeW1iaW9udCBCdWNobmVyYSBhcGhpZGljb2xhOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4NzsgIGNpdGF0aW9uX3BhZ2VzPTQyMjktNDIzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBNLiBhY2V0aXZvcmFucyByZXZlYWxzIGV4dGVuc2l2ZSBtZXRhYm9saWMgYW5kIHBoeXNpb2xvZ2ljYWwgZGl2ZXJzaXR5OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTUzMi01NDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGh5cGVydGhlcm1vcGhpbGljLCBzdWxwaGF0ZS1yZWR1Y2luZyBhcmNoYWVvbiBBcmNoYWVvZ2xvYnVzIGZ1bGdpZHVzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MzkwOyAgY2l0YXRpb25fcGFnZXM9MzY0LTM3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBtb2RlbCBhY3Rpbm9teWNldGUgU3RyZXB0b215Y2VzIGNvZWxpY29sb3IgQTMoMik7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTc7ICBjaXRhdGlvbl9wYWdlcz0xNDEtMTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIEJpZmlkb2JhY3Rlcml1bSBsb25ndW0gcmVmbGVjdHMgaXRzIGFkYXB0YXRpb24gdG8gdGhlIGh1bWFuIGdhc3Ryb2ludGVzdGluYWwgdHJhY3Q7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xNDQyMi0xNDQyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgZ3JhbS1wb3NpdGl2ZSBiYWN0ZXJpdW0gQmFjaWxsdXMgc3VidGlsaXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0zOTA7ICBjaXRhdGlvbl9wYWdlcz0yNDktMjU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2UgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBzb2x2ZW50LXByb2R1Y2luZyBiYWN0ZXJpdW0gQ2xvc3RyaWRpdW0gYWNldG9idXR5bGljdW07IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTgzOyAgY2l0YXRpb25fcGFnZXM9NDgyMy00ODM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBlbnRvbW9wYXRob2dlbmljIGJhY3Rlcml1bSBQaG90b3JoYWJkdXMgbHVtaW5lc2NlbnM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMzA3LTEzMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHByb2Jpb3RpYyBpbnRlc3RpbmFsIGJhY3Rlcml1bSBMYWN0b2JhY2lsbHVzIGpvaG5zb25paSBOQ0MgNTMzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTI1MTItMjUxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbGFjdGljIGFjaWQgYmFjdGVyaXVtIExhY3RvY29jY3VzIGxhY3RpcyBzc3AuIGxhY3RpcyBJTDE0MDM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NzMxLTc1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIGFuZCBhbmFseXNpcyBvZiB0aGUgb3JhbCBiYWN0ZXJpdW0gRnVzb2JhY3Rlcml1bSBudWNsZWF0dW0gc3RyYWluIEFUQ0MgMjU1ODY7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg0OyAgY2l0YXRpb25fcGFnZXM9MjAwNS0yMDE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIG1ldGFib2xpY2FsbHkgdmVyc2F0aWxlIHBob3Rvc3ludGhldGljIGJhY3Rlcml1bSBSaG9kb3BzZXVkb21vbmFzIHBhbHVzdHJpczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTU1LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTIvNS8yNjkvNjcxODkyMC9kc2kwMTYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFzc2lnbm1lbnQgb2YgYWxsIHRyYW5zY3JpcHRpb24gZmFjdG9ycyAoVEZzKSBmcm9tIGdlbm9tZSBzZXF1ZW5jZSBkYXRhIGlzIG5vdCBhIHN0cmFpZ2h0Zm9yd2FyZCB0YXNrIGR1ZSB0byB0aGUgd2lkZSB2YXJpYXRpb24gaW4gVEZzIGFtb25nIGRpZmZlcmVuIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTA5MDYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkyfLnv0xLmLLcwEv3dR1AMhshx/Y=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f269%2f350906" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f269%2f350906" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/5/0/m_dnaresearch12_5.cover.gif?Expires=2147483647&amp;Signature=aCda7p9Uq5iwZjUzLjBT9gM9GcXHBhpbSpHktSZ3aBfaLgUTRrUBMfrhnbDOftVRcOxJYPlWQoOSDtYCMRCSqLqznp4erEvRKXpqo14qAWzUQzXAUf0R2hzkxTrttMY8s2309oR8tNHgB9rVuF2hpqh3QH4eyjVeWXlnJ40m9gcWoDr-jQH7WNKVcWkYKOzDqj0zz3HLWNzUSiszlQ7mPDPy4j~szH2kDsY0ptS4bBqTMPSDLuVeIksQM7hpXYsVJSgGsIGUFqHvKMFIXeS6xBjZCvV-jy2mqIn0wg5gOITNC5m-u-f0z~ur8QMlAsoeOXCWmD3HyGddqFtbkE9Www__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/5">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="35871095">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35871095">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35871097">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35871097">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35871101">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35871101">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35871109">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35871109">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35871128">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35871128">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35871137">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35871137">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/12/5/281/350803">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiaki Minezaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems</div><div class="addr-line">1111 Yata, Mishima, Shizuoka, 411-8540 Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiaki+Minezaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minezaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minezaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiaki Minezaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keiichi Homma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems</div><div class="addr-line">1111 Yata, Mishima, Shizuoka, 411-8540 Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Japan Science and Technology Corporation</div><div class="addr-line">1–8, Honcho 4-chome, Kawaguchi City, Saitama, 332-0012, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keiichi+Homma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Homma K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Homma K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Keiichi Homma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Nishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Laboratory of Gene-Product Informatics, Center for Information Biology–DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems</div><div class="addr-line">1111 Yata, Mishima, Shizuoka, 411-8540 Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup> To whom correspondence should be addressed. Tel. +81-55-981-6859, Fax. +81-55-981-6889, E-mail: <a href="mailto:knishika@genes.nig.ac.jp" target="_blank">knishika@genes.nig.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Nishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishikawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishikawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ken Nishikawa</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 5, 1 January 2005, Pages 269–280, <a href='https://doi.org/10.1093/dnares/dsi016'>https://doi.org/10.1093/dnares/dsi016</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">02 July 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">28 September 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="350906" href="/dnaresearch/article-pdf/12/5/269/6718920/dsi016.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoshiaki Minezaki, Keiichi Homma, Ken Nishikawa;  Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea, <em>DNA Research</em>, Volume 12, Issue 5, 1 January 2005, Pages 269–280, <a href='https://doi.org/10.1093/dnares/dsi016'>https://doi.org/10.1093/dnares/dsi016</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/350906?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/350906?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/350906?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/350906?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/350906?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/350906?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/350906?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20Survey%20of%20Transcription%20Factors%20in%20Prokaryotes%20Reveals%20Many%20Bacteria-Specific%20Families%20Not%20Found%20in%20Archaea&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=5&amp;author=Minezaki%2C%20Yoshiaki%3B%20Homma%2C%20Keiichi&amp;startPage=269&amp;endPage=280&amp;contentId=10.1093%2Fdnares%2Fdsi016&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f269%2f350906" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=35871095 id="35871095" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Assignment of all transcription factors (TFs) from genome sequence data is not a straightforward task due to the wide variation in TFs among different species. A DNA binding domain (DBD) and a contiguous non-DBD with a characteristic SCOP or Pfam domain combination are observed in most members of TF families. We found that most of the experimentally verified TFs in prokaryotes are detectable by a combination of SCOP or Pfam domains assigned to DBDs and non-DBDs. Based on this finding, we set up rules to detect TFs and classify them into 52 TF families. Application of the rules to 154 entirely sequenced prokaryotic genomes detected &gt;18 000 TFs classified into families, which have been made publicly available from the ‘GTOP_TF’ database. Despite the rough proportionality of the number of TFs per genome with genome size, species with reduced genomes, i.e. obligatory parasites and symbionts, have only a few if any TFs, reflecting a nearly complete loss. Also the number of TFs is significantly lower in archaea than in bacteria. In addition, all but 1 of the 19 TF families present in archaea is present in bacteria, whereas 33 TF families are found exclusively in bacteria. This observation indicates that a number of new TF families have evolved in bacteria, making the transcription regulatory system more divergent in bacteria than in archaea.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factor&quot;">transcription factor</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;domain organization&quot;">domain organization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA binding&quot;">DNA binding</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="prokaryote">prokaryote</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a></div>                            <h2 scrollto-destination=35871097 id="35871097" class="section-title" >1. Introduction</h2>
<p> The genetic information contained in DNA is transcribed to RNA by a transcription complex including DNA-directed RNA polymerase (RNAP). A bacterial transcription initiation complex comprised of the core RNAP enzyme and a σ factor binds to a promoter and, upon initiation of RNA synthesis, releases the σ factor. <a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> The archaeal transcription initiation machinery has a combination of different core RNAP proteins and basal transcription factors (TFs) such as TATA-box binding protein (TBP) and transcription factor B, each of which is homologous to the eukaryotic counterpart. <a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> In either case, the transcriptional complex can be easily detected by homology because the essentiality of the transcriptional complex entails high conservation of amino acid sequence in every component. On the other hand, TFs such as repressors, activators and enhancer binding proteins, all of which bind to double-stranded DNA at specific sites to interfere or modulate RNAP function, display an enormous variation. <a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> , <a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> Different types of TFs have dissimilar 3D structures, with the only shared characteristic being the ability to bind double-stranded DNA. The aim of this study is 2-fold: one is to develop a method to systematically detect all kinds of TFs encoded by a genome with the highest possible accuracy, and the other is to compare results among all prokaryotic species, particularly focusing on any distinctions between bacteria and archaea. As we are conducting a separate investigation on eukaryotes, utilizing extensively compiled data of eukaryotic TFs in the TRANSFAC database, <a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> we limit our attention to prokaryotes in this paper. </p><p> Although there have been hitherto no genome-wide surveys for TFs across all prokaryotes, several investigations have been carried out for particular species, taxons or TF families: Perez-Rueda and Collado-Vides <a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> conducted keyword and PROSITE searches to assign 314 regulatory DNA binding proteins in <em>Escherichia coli</em> , and stored them in the RegulonDB database. Aravind and Koonin <a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> examined all the archaeal genomes then available ( <em>Methanocaldococcus jannaschii, Methanothermobacter thermoautotrophicum, Archaeoglobus fulgidus</em> and <em>Pyrococcus horikoshii</em> ) and made a long list of HTH (helix–turn–helix) proteins, which include some non-TFs. Kyrpides and Ouzounis <a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> investigated the same four archaeal species, found 280 transcription-associated proteins including not only TFs but also basal TFs and RNAPs, and classified them into 58 families. A larger scaled survey by Cases et al. <a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> investigated 60 bacterial species and categorized proteins into several functional groups, including a group of transcription-related proteins. Ranea et al. <a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> performed proteomics analyses on 56 prokaryotic species and classified proteins into CATH <a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> superfamilies and functional categories containing one with transcriptional regulators. Recently, Martinez-Bueno et al. <a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> exhaustively identified TFs of the AraC-XylS and TetR families from 123 genomes of archaea and bacteria, and deposited them in the BacTregulators database. All the above-mentioned identifications of TFs are in principle based on sequence alignment of the entire sequence. However, as the DBD of a TF constitutes only a small portion such approach often fails to properly identify a DBD and consequently leads to overidentification. On the other hand, Babu and Teichmann <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> used SCOP <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> domains aligned to DBDs as identifiers, listed 271 TFs of <em>E. coli</em> and classified them into 11 TF families including 1 with an RNA-binding domain. As the same SCOP domains are sometimes found in both DBDs and non-DBDs (see Results) this methodology also produces some overidentified cases. </p><p>In order to reliably detect TFs we developed a novel method employing a combination of a DBD and a contiguous non-DBD with specific SCOP or Pfam domains as the main identifiers of TFs. Our method detects all experimentally verified prokaryotic TFs and is considered to miss or erroneously assign TFs infrequently. Application of the rules for each kind of TFs (i.e. TF family) to entirely sequenced prokaryotes revealed that bacteria have many kingdom-specific families of TFs, whereas archaea share almost all of their TF families with bacteria.</p>                            <h2 scrollto-destination=35871101 id="35871101" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=35871102 id="35871102" class="section-title" >2.1. Genome data</h3>
<p> The main body of the dataset used in the present study comes from the GTOP database <a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> ( <a class="link link-uri" href="http://spock.genes.nig.ac.jp/~genome/gtop.html" target="_blank"></a> ), which has been constructed and maintained in this laboratory. GTOP contains all the open reading frames (ORFs) assigned to each organism by the genome sequencing team, and provides structural and functional information on ORFs analyzed by homology search at the protein level. In GTOP, a PSI-BLAST <a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> search was conducted for each query ORF against the public databases of PDB <a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> (released on 14 November 2003) and SCOP <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> (version 1.65), together with a BLAST search against Swiss-Prot <a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> (version 42.5). The <em>E</em> -value threshold was set at 0.001 in both search methods. The hidden Markov model program (HMMER) <a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> was also utilized for the search against Pfam <a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> (version 11). We used a version of GTOP (released on 19 May 2004) containing the genomic data of 18 archaeal and 136 bacterial species together with a number of eukaryotes and bacteriophages. </p>                            <h3 scrollto-destination=35871104 id="35871104" class="section-title" >2.2. TFs for analyses</h3>
<p> We deal with all kinds of TFs bound to the double-stranded DNA at specific sites to regulate RNAP function but not those bound to DNA in a non-specific manner or involved in the initiation complex of RNAP itself. Accordingly, bacterial σ factors or archaeal general TFs (TBP) were excluded from our analyses. Also omitted are DNA binding proteins that function non-specifically, such as bacterial HU and archaeal histones, as they affect the transcription process in general. <a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> Moreover factors controlling transcription termination like Rho, <a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> which bind to RNA rather than DNA, were kept out of this study. Ambiguous cases were decided individually by consulting the literature; we excluded those that were originally regarded as TFs, but were recently revealed not to be TFs, such as cold-shock proteins <a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> and TenA. <a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> Following the current classification of TFs, each TF family was defined based on the kind of DBD it contains and was named after a representative gene (protein) belonging to the family (see <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> ). </p>                        <a id="35871106" scrollto-destination="35871106"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>List of high confidence TF families characterized by Pfam and SCOP domains.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TF family </th><th>Total count </th><th colspan="2">Pfam </th><th colspan="2">SCOP </th><th>Representative TF </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th> </th></tr><tr><th> </th><th> </th><th>DBD </th><th>Non-DBD </th><th>DBD </th><th>Non-DBD </th><th> </th></tr></thead><tbody><tr><td>LysR </td><td>2530 </td><td><strong>PF00126</strong> </td><td>PF03466 </td><td><strong>a.4.5.8</strong> </td><td>c.94.1.1 </td><td>LysR_ecol </td></tr><tr><td>TetR/AcrR </td><td>1681 </td><td><strong>PF00440</strong> </td><td>— </td><td><strong>a.4.1.9</strong> </td><td>— </td><td>AcrR_ecol </td></tr><tr><td>GntR </td><td>1394 </td><td><strong>PF00392</strong> </td><td>PF00155; PF00532; PF07702 </td><td><strong>a.4.5.6</strong> </td><td>c.67.1.1; c.93.1.1 </td><td>GntR_tten </td></tr><tr><td>AraC </td><td>1375 </td><td><strong>PF00165</strong> </td><td>PF01965; PF02311; PF02805 </td><td><strong>a.4.1.8</strong> </td><td>b.82.1.9; c.23.16.2; c.55.7.1; g.48.1.1 </td><td>AraC_ecol </td></tr><tr><td>CRO/CI/Xre </td><td>1259 </td><td><strong>PF01381</strong> </td><td>— </td><td><strong>a.35.1.2; a.35.1.3</strong> </td><td>— </td><td>DicA_ecol </td></tr><tr><td>OmpR </td><td>1241 </td><td><strong>PF00486</strong> </td><td>PF00072 </td><td><strong>a.4.6.1</strong> </td><td>c.23.1.1; c.23.1.3 </td><td>OmpR_ecol </td></tr><tr><td>LuxR/NarL </td><td>1071 </td><td><strong>PF00196</strong> </td><td>PF00072; PF03472 </td><td><strong>a.4.6.2</strong> </td><td>c.23.1.1; d.110.5.1 </td><td>NarL_ecol </td></tr><tr><td>MarR </td><td>948 </td><td><strong>PF01047</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>MarR_ecol </td></tr><tr><td>LacI </td><td>750 </td><td><strong>PF00356</strong> </td><td>PF00532 </td><td><strong>a.35.1.5</strong> </td><td>c.93.1.1 </td><td>LacI_ecol </td></tr><tr><td>ArsR </td><td>622 </td><td><strong>PF01022</strong> </td><td>— </td><td><strong>a.4.5.5; a.4.5.36</strong> </td><td>— </td><td>ArsR_ecol </td></tr><tr><td>Fis </td><td>570 </td><td><strong>PF02954</strong> </td><td>PF00072; PF00158; PF01590; PF06506 </td><td><strong>a.105.1.1</strong> </td><td>c.23.1.1; c.37.1.20; d.110.2.1 </td><td>Fis_ecol </td></tr><tr><td>MerR </td><td>549 </td><td><strong>PF00376</strong> </td><td>— </td><td><strong>a.6.1.3</strong> </td><td>— </td><td>MerR_styp </td></tr><tr><td>AsnC/Lrp </td><td>439 </td><td>— </td><td>— </td><td><strong>a.4.5.28; a.4.5.32</strong> </td><td>d.58.4.2 </td><td>AsnC_ecol </td></tr><tr><td>DeoR </td><td>355 </td><td>— </td><td>— </td><td><strong>a.4.5.1; a.4.5.24</strong> </td><td>c.35.1.2; c.63.1.3 </td><td>DeoR_ecol </td></tr><tr><td>Crp/Fnr </td><td>353 </td><td><strong>PF00325</strong> </td><td>PF00027 </td><td><strong>a.4.5.4</strong> </td><td>b.82.3.1; b.82.3.2 </td><td>Crp_ecol </td></tr><tr><td>Fur </td><td>265 </td><td><strong>PF01475</strong> </td><td>— </td><td>— </td><td>— </td><td>Fur_ecol </td></tr><tr><td>PadR </td><td>253 </td><td><strong>PF03551</strong> </td><td>— </td><td>— </td><td>— </td><td>PadR_bsub </td></tr><tr><td>RpiR </td><td>227 </td><td><strong>PF01418</strong> </td><td>PF01380 </td><td> </td><td>c.80.1.1; c.80.1.3 </td><td>RpiR_ecol </td></tr><tr><td>Rrf2 </td><td>218 </td><td><strong>PF02082</strong> </td><td>— </td><td>— </td><td>— </td><td>IscR_ecol </td></tr><tr><td>DnaA </td><td>139 </td><td>— </td><td>— </td><td><strong>a.4.12.2</strong> </td><td>c.37.1.20 </td><td>DnaA_ecol </td></tr><tr><td>BolA/YrbA </td><td>121 </td><td><strong>PF01722</strong> </td><td>— </td><td><strong>d.52.6.1</strong> </td><td>— </td><td>BolA_ecol </td></tr><tr><td>ROK/NagC/XylR </td><td>118 </td><td>— </td><td>PF00480 </td><td><strong>a.4.5.1; a.4.5.24; a.4.5.28; a.4.5.32; a.4.5.36</strong> </td><td>— </td><td>NagC_ecol </td></tr><tr><td>LytTR </td><td>115 </td><td><strong>PF04397</strong> </td><td>PF00072 </td><td> </td><td>c.23.1.1 </td><td>LytT_bsub </td></tr><tr><td>SorC </td><td>113 </td><td>— </td><td>PF04198 </td><td><strong>a.4.5.4; a.4.13.2</strong> </td><td>— </td><td>SorC_sfle </td></tr><tr><td>ArgR </td><td>98 </td><td><strong>PF01316</strong> </td><td>PF02863 </td><td><strong>a.4.5.3</strong> </td><td>d.74.2.1 </td><td>ArgR_ecol </td></tr><tr><td>DtxR </td><td>92 </td><td><strong>PF01325</strong> </td><td>PF02742; PF04023 </td><td><strong>a.4.5.24</strong> </td><td>a.76.1.1; b.34.1.2 </td><td>DtxR_cdip </td></tr><tr><td>LexA </td><td>86 </td><td><strong>PF01726</strong> </td><td>PF00717 </td><td><strong>a.4.5.2</strong> </td><td>b.87.1.1 </td><td>LexA_ecol </td></tr><tr><td>TrmB </td><td>68 </td><td><strong>PF01978</strong> </td><td>— </td><td>— </td><td>— </td><td>AF1009_aful </td></tr><tr><td>BirA </td><td>67 </td><td>— </td><td>PF02237; PF03099 </td><td><strong>a.4.5.1</strong> </td><td>b.34.1.1; d.104.1.2 </td><td>BirA_ecol </td></tr><tr><td>PenR/BlaI/MecI </td><td>59 </td><td><strong>PF03965</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>BlaI_bant </td></tr><tr><td>SfsA </td><td>57 </td><td><strong>PF03749</strong> </td><td>— </td><td>— </td><td>— </td><td>SfsA_ecol </td></tr><tr><td>Nlp </td><td>42 </td><td>— </td><td>— </td><td><strong>a.35.1.2</strong> </td><td>— </td><td>SfsB_ecol </td></tr><tr><td>Archaeal HTH-10 </td><td>40 </td><td><strong>PF04967</strong> </td><td>— </td><td>— </td><td>— </td><td>AF0805_aful </td></tr><tr><td>CopG/RepA </td><td>38 </td><td><strong>PF01402</strong> </td><td>— </td><td>— </td><td>— </td><td>NikR_ecol </td></tr><tr><td>PutA </td><td>38 </td><td>— </td><td>PF00171; PF01619 </td><td><strong>a.176.1.1</strong> </td><td>c.1.23.2; c.82.1.1 </td><td>PutA_ecol </td></tr><tr><td>ModE </td><td>31 </td><td><strong>PF02573</strong> </td><td>PF03459 </td><td><strong>a.4.5.8</strong> </td><td>b.40.6.1; b.40.6.2 </td><td>ModE_ecol </td></tr><tr><td>PaiB </td><td>30 </td><td><strong>PF04299</strong> </td><td>— </td><td>— </td><td>— </td><td>PaiB_bsub </td></tr><tr><td>CtsR </td><td>28 </td><td><strong>PF05848</strong> </td><td>— </td><td>— </td><td>— </td><td>CtsR_bsub </td></tr><tr><td>AfsR/DnrI/RedD </td><td>27 </td><td><strong>PF00486</strong> </td><td>PF03704 </td><td>— </td><td>— </td><td>EmbR_mtub </td></tr><tr><td>CodY </td><td>27 </td><td><strong>PF06018</strong> </td><td>— </td><td>— </td><td>— </td><td>CodY_bsub </td></tr><tr><td>TrpR </td><td>25 </td><td><strong>PF01371</strong> </td><td>— </td><td><strong>a.4.12.1</strong> </td><td>— </td><td>TrpR_ecol </td></tr><tr><td>MtlR </td><td>24 </td><td><strong>PF05068</strong> </td><td>— </td><td>— </td><td>— </td><td>MtlR_ecol </td></tr><tr><td>ROS/MUCR </td><td>22 </td><td><strong>PF05443</strong> </td><td>— </td><td>— </td><td>— </td><td>Ros_atum </td></tr><tr><td>MetJ </td><td>21 </td><td><strong>PF01340</strong> </td><td>— </td><td><strong>a.43.1.2</strong> </td><td>— </td><td>MetJ_ecol </td></tr><tr><td>GutM </td><td>17 </td><td><strong>PF06923</strong> </td><td>— </td><td>— </td><td>— </td><td>GutM_ecol </td></tr><tr><td>Crl </td><td>16 </td><td><strong>PF07417</strong> </td><td>— </td><td>— </td><td>— </td><td>Crl_ecol </td></tr><tr><td>ComK </td><td>14 </td><td><strong>PF06338</strong> </td><td>— </td><td>— </td><td>— </td><td>ComK_bsub </td></tr><tr><td>FlhD </td><td>14 </td><td><strong>PF05247</strong> </td><td>— </td><td><strong>a.145.1.1</strong> </td><td>— </td><td>FlhD_ecol </td></tr><tr><td>RtcR </td><td>11 </td><td><strong>PF06956</strong> </td><td>PF00158 </td><td>— </td><td>— </td><td>RtcR_ecol </td></tr><tr><td>Spo0A </td><td>9 </td><td>— </td><td>PF00072 </td><td><strong>a.4.6.3</strong> </td><td>c.23.1.1 </td><td>Spo0A_bsub </td></tr><tr><td>DctR </td><td>6 </td><td>— </td><td>— </td><td><strong>a.4.5.1</strong> </td><td>c.23.1.1 </td><td>DctR_bsub </td></tr><tr><td>NifT/FixU </td><td>6 </td><td><strong>PF06988</strong> </td><td>— </td><td>— </td><td>— </td><td>NifT_anab </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>TF family </th><th>Total count </th><th colspan="2">Pfam </th><th colspan="2">SCOP </th><th>Representative TF </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th> </th></tr><tr><th> </th><th> </th><th>DBD </th><th>Non-DBD </th><th>DBD </th><th>Non-DBD </th><th> </th></tr></thead><tbody><tr><td>LysR </td><td>2530 </td><td><strong>PF00126</strong> </td><td>PF03466 </td><td><strong>a.4.5.8</strong> </td><td>c.94.1.1 </td><td>LysR_ecol </td></tr><tr><td>TetR/AcrR </td><td>1681 </td><td><strong>PF00440</strong> </td><td>— </td><td><strong>a.4.1.9</strong> </td><td>— </td><td>AcrR_ecol </td></tr><tr><td>GntR </td><td>1394 </td><td><strong>PF00392</strong> </td><td>PF00155; PF00532; PF07702 </td><td><strong>a.4.5.6</strong> </td><td>c.67.1.1; c.93.1.1 </td><td>GntR_tten </td></tr><tr><td>AraC </td><td>1375 </td><td><strong>PF00165</strong> </td><td>PF01965; PF02311; PF02805 </td><td><strong>a.4.1.8</strong> </td><td>b.82.1.9; c.23.16.2; c.55.7.1; g.48.1.1 </td><td>AraC_ecol </td></tr><tr><td>CRO/CI/Xre </td><td>1259 </td><td><strong>PF01381</strong> </td><td>— </td><td><strong>a.35.1.2; a.35.1.3</strong> </td><td>— </td><td>DicA_ecol </td></tr><tr><td>OmpR </td><td>1241 </td><td><strong>PF00486</strong> </td><td>PF00072 </td><td><strong>a.4.6.1</strong> </td><td>c.23.1.1; c.23.1.3 </td><td>OmpR_ecol </td></tr><tr><td>LuxR/NarL </td><td>1071 </td><td><strong>PF00196</strong> </td><td>PF00072; PF03472 </td><td><strong>a.4.6.2</strong> </td><td>c.23.1.1; d.110.5.1 </td><td>NarL_ecol </td></tr><tr><td>MarR </td><td>948 </td><td><strong>PF01047</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>MarR_ecol </td></tr><tr><td>LacI </td><td>750 </td><td><strong>PF00356</strong> </td><td>PF00532 </td><td><strong>a.35.1.5</strong> </td><td>c.93.1.1 </td><td>LacI_ecol </td></tr><tr><td>ArsR </td><td>622 </td><td><strong>PF01022</strong> </td><td>— </td><td><strong>a.4.5.5; a.4.5.36</strong> </td><td>— </td><td>ArsR_ecol </td></tr><tr><td>Fis </td><td>570 </td><td><strong>PF02954</strong> </td><td>PF00072; PF00158; PF01590; PF06506 </td><td><strong>a.105.1.1</strong> </td><td>c.23.1.1; c.37.1.20; d.110.2.1 </td><td>Fis_ecol </td></tr><tr><td>MerR </td><td>549 </td><td><strong>PF00376</strong> </td><td>— </td><td><strong>a.6.1.3</strong> </td><td>— </td><td>MerR_styp </td></tr><tr><td>AsnC/Lrp </td><td>439 </td><td>— </td><td>— </td><td><strong>a.4.5.28; a.4.5.32</strong> </td><td>d.58.4.2 </td><td>AsnC_ecol </td></tr><tr><td>DeoR </td><td>355 </td><td>— </td><td>— </td><td><strong>a.4.5.1; a.4.5.24</strong> </td><td>c.35.1.2; c.63.1.3 </td><td>DeoR_ecol </td></tr><tr><td>Crp/Fnr </td><td>353 </td><td><strong>PF00325</strong> </td><td>PF00027 </td><td><strong>a.4.5.4</strong> </td><td>b.82.3.1; b.82.3.2 </td><td>Crp_ecol </td></tr><tr><td>Fur </td><td>265 </td><td><strong>PF01475</strong> </td><td>— </td><td>— </td><td>— </td><td>Fur_ecol </td></tr><tr><td>PadR </td><td>253 </td><td><strong>PF03551</strong> </td><td>— </td><td>— </td><td>— </td><td>PadR_bsub </td></tr><tr><td>RpiR </td><td>227 </td><td><strong>PF01418</strong> </td><td>PF01380 </td><td> </td><td>c.80.1.1; c.80.1.3 </td><td>RpiR_ecol </td></tr><tr><td>Rrf2 </td><td>218 </td><td><strong>PF02082</strong> </td><td>— </td><td>— </td><td>— </td><td>IscR_ecol </td></tr><tr><td>DnaA </td><td>139 </td><td>— </td><td>— </td><td><strong>a.4.12.2</strong> </td><td>c.37.1.20 </td><td>DnaA_ecol </td></tr><tr><td>BolA/YrbA </td><td>121 </td><td><strong>PF01722</strong> </td><td>— </td><td><strong>d.52.6.1</strong> </td><td>— </td><td>BolA_ecol </td></tr><tr><td>ROK/NagC/XylR </td><td>118 </td><td>— </td><td>PF00480 </td><td><strong>a.4.5.1; a.4.5.24; a.4.5.28; a.4.5.32; a.4.5.36</strong> </td><td>— </td><td>NagC_ecol </td></tr><tr><td>LytTR </td><td>115 </td><td><strong>PF04397</strong> </td><td>PF00072 </td><td> </td><td>c.23.1.1 </td><td>LytT_bsub </td></tr><tr><td>SorC </td><td>113 </td><td>— </td><td>PF04198 </td><td><strong>a.4.5.4; a.4.13.2</strong> </td><td>— </td><td>SorC_sfle </td></tr><tr><td>ArgR </td><td>98 </td><td><strong>PF01316</strong> </td><td>PF02863 </td><td><strong>a.4.5.3</strong> </td><td>d.74.2.1 </td><td>ArgR_ecol </td></tr><tr><td>DtxR </td><td>92 </td><td><strong>PF01325</strong> </td><td>PF02742; PF04023 </td><td><strong>a.4.5.24</strong> </td><td>a.76.1.1; b.34.1.2 </td><td>DtxR_cdip </td></tr><tr><td>LexA </td><td>86 </td><td><strong>PF01726</strong> </td><td>PF00717 </td><td><strong>a.4.5.2</strong> </td><td>b.87.1.1 </td><td>LexA_ecol </td></tr><tr><td>TrmB </td><td>68 </td><td><strong>PF01978</strong> </td><td>— </td><td>— </td><td>— </td><td>AF1009_aful </td></tr><tr><td>BirA </td><td>67 </td><td>— </td><td>PF02237; PF03099 </td><td><strong>a.4.5.1</strong> </td><td>b.34.1.1; d.104.1.2 </td><td>BirA_ecol </td></tr><tr><td>PenR/BlaI/MecI </td><td>59 </td><td><strong>PF03965</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>BlaI_bant </td></tr><tr><td>SfsA </td><td>57 </td><td><strong>PF03749</strong> </td><td>— </td><td>— </td><td>— </td><td>SfsA_ecol </td></tr><tr><td>Nlp </td><td>42 </td><td>— </td><td>— </td><td><strong>a.35.1.2</strong> </td><td>— </td><td>SfsB_ecol </td></tr><tr><td>Archaeal HTH-10 </td><td>40 </td><td><strong>PF04967</strong> </td><td>— </td><td>— </td><td>— </td><td>AF0805_aful </td></tr><tr><td>CopG/RepA </td><td>38 </td><td><strong>PF01402</strong> </td><td>— </td><td>— </td><td>— </td><td>NikR_ecol </td></tr><tr><td>PutA </td><td>38 </td><td>— </td><td>PF00171; PF01619 </td><td><strong>a.176.1.1</strong> </td><td>c.1.23.2; c.82.1.1 </td><td>PutA_ecol </td></tr><tr><td>ModE </td><td>31 </td><td><strong>PF02573</strong> </td><td>PF03459 </td><td><strong>a.4.5.8</strong> </td><td>b.40.6.1; b.40.6.2 </td><td>ModE_ecol </td></tr><tr><td>PaiB </td><td>30 </td><td><strong>PF04299</strong> </td><td>— </td><td>— </td><td>— </td><td>PaiB_bsub </td></tr><tr><td>CtsR </td><td>28 </td><td><strong>PF05848</strong> </td><td>— </td><td>— </td><td>— </td><td>CtsR_bsub </td></tr><tr><td>AfsR/DnrI/RedD </td><td>27 </td><td><strong>PF00486</strong> </td><td>PF03704 </td><td>— </td><td>— </td><td>EmbR_mtub </td></tr><tr><td>CodY </td><td>27 </td><td><strong>PF06018</strong> </td><td>— </td><td>— </td><td>— </td><td>CodY_bsub </td></tr><tr><td>TrpR </td><td>25 </td><td><strong>PF01371</strong> </td><td>— </td><td><strong>a.4.12.1</strong> </td><td>— </td><td>TrpR_ecol </td></tr><tr><td>MtlR </td><td>24 </td><td><strong>PF05068</strong> </td><td>— </td><td>— </td><td>— </td><td>MtlR_ecol </td></tr><tr><td>ROS/MUCR </td><td>22 </td><td><strong>PF05443</strong> </td><td>— </td><td>— </td><td>— </td><td>Ros_atum </td></tr><tr><td>MetJ </td><td>21 </td><td><strong>PF01340</strong> </td><td>— </td><td><strong>a.43.1.2</strong> </td><td>— </td><td>MetJ_ecol </td></tr><tr><td>GutM </td><td>17 </td><td><strong>PF06923</strong> </td><td>— </td><td>— </td><td>— </td><td>GutM_ecol </td></tr><tr><td>Crl </td><td>16 </td><td><strong>PF07417</strong> </td><td>— </td><td>— </td><td>— </td><td>Crl_ecol </td></tr><tr><td>ComK </td><td>14 </td><td><strong>PF06338</strong> </td><td>— </td><td>— </td><td>— </td><td>ComK_bsub </td></tr><tr><td>FlhD </td><td>14 </td><td><strong>PF05247</strong> </td><td>— </td><td><strong>a.145.1.1</strong> </td><td>— </td><td>FlhD_ecol </td></tr><tr><td>RtcR </td><td>11 </td><td><strong>PF06956</strong> </td><td>PF00158 </td><td>— </td><td>— </td><td>RtcR_ecol </td></tr><tr><td>Spo0A </td><td>9 </td><td>— </td><td>PF00072 </td><td><strong>a.4.6.3</strong> </td><td>c.23.1.1 </td><td>Spo0A_bsub </td></tr><tr><td>DctR </td><td>6 </td><td>— </td><td>— </td><td><strong>a.4.5.1</strong> </td><td>c.23.1.1 </td><td>DctR_bsub </td></tr><tr><td>NifT/FixU </td><td>6 </td><td><strong>PF06988</strong> </td><td>— </td><td>— </td><td>— </td><td>NifT_anab </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>DBDs characterized by Pfam/SCOP ID codes are indicated in boldface. Only those ID codes of Pfam and SCOP that were used to identify proteins as TFs are indicated in the table. A representative TF for each TF family is shown as a protein name plus the shortened name of the species. The full species names are given in GTOP_TF.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35871106" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>List of high confidence TF families characterized by Pfam and SCOP domains.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TF family </th><th>Total count </th><th colspan="2">Pfam </th><th colspan="2">SCOP </th><th>Representative TF </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th> </th></tr><tr><th> </th><th> </th><th>DBD </th><th>Non-DBD </th><th>DBD </th><th>Non-DBD </th><th> </th></tr></thead><tbody><tr><td>LysR </td><td>2530 </td><td><strong>PF00126</strong> </td><td>PF03466 </td><td><strong>a.4.5.8</strong> </td><td>c.94.1.1 </td><td>LysR_ecol </td></tr><tr><td>TetR/AcrR </td><td>1681 </td><td><strong>PF00440</strong> </td><td>— </td><td><strong>a.4.1.9</strong> </td><td>— </td><td>AcrR_ecol </td></tr><tr><td>GntR </td><td>1394 </td><td><strong>PF00392</strong> </td><td>PF00155; PF00532; PF07702 </td><td><strong>a.4.5.6</strong> </td><td>c.67.1.1; c.93.1.1 </td><td>GntR_tten </td></tr><tr><td>AraC </td><td>1375 </td><td><strong>PF00165</strong> </td><td>PF01965; PF02311; PF02805 </td><td><strong>a.4.1.8</strong> </td><td>b.82.1.9; c.23.16.2; c.55.7.1; g.48.1.1 </td><td>AraC_ecol </td></tr><tr><td>CRO/CI/Xre </td><td>1259 </td><td><strong>PF01381</strong> </td><td>— </td><td><strong>a.35.1.2; a.35.1.3</strong> </td><td>— </td><td>DicA_ecol </td></tr><tr><td>OmpR </td><td>1241 </td><td><strong>PF00486</strong> </td><td>PF00072 </td><td><strong>a.4.6.1</strong> </td><td>c.23.1.1; c.23.1.3 </td><td>OmpR_ecol </td></tr><tr><td>LuxR/NarL </td><td>1071 </td><td><strong>PF00196</strong> </td><td>PF00072; PF03472 </td><td><strong>a.4.6.2</strong> </td><td>c.23.1.1; d.110.5.1 </td><td>NarL_ecol </td></tr><tr><td>MarR </td><td>948 </td><td><strong>PF01047</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>MarR_ecol </td></tr><tr><td>LacI </td><td>750 </td><td><strong>PF00356</strong> </td><td>PF00532 </td><td><strong>a.35.1.5</strong> </td><td>c.93.1.1 </td><td>LacI_ecol </td></tr><tr><td>ArsR </td><td>622 </td><td><strong>PF01022</strong> </td><td>— </td><td><strong>a.4.5.5; a.4.5.36</strong> </td><td>— </td><td>ArsR_ecol </td></tr><tr><td>Fis </td><td>570 </td><td><strong>PF02954</strong> </td><td>PF00072; PF00158; PF01590; PF06506 </td><td><strong>a.105.1.1</strong> </td><td>c.23.1.1; c.37.1.20; d.110.2.1 </td><td>Fis_ecol </td></tr><tr><td>MerR </td><td>549 </td><td><strong>PF00376</strong> </td><td>— </td><td><strong>a.6.1.3</strong> </td><td>— </td><td>MerR_styp </td></tr><tr><td>AsnC/Lrp </td><td>439 </td><td>— </td><td>— </td><td><strong>a.4.5.28; a.4.5.32</strong> </td><td>d.58.4.2 </td><td>AsnC_ecol </td></tr><tr><td>DeoR </td><td>355 </td><td>— </td><td>— </td><td><strong>a.4.5.1; a.4.5.24</strong> </td><td>c.35.1.2; c.63.1.3 </td><td>DeoR_ecol </td></tr><tr><td>Crp/Fnr </td><td>353 </td><td><strong>PF00325</strong> </td><td>PF00027 </td><td><strong>a.4.5.4</strong> </td><td>b.82.3.1; b.82.3.2 </td><td>Crp_ecol </td></tr><tr><td>Fur </td><td>265 </td><td><strong>PF01475</strong> </td><td>— </td><td>— </td><td>— </td><td>Fur_ecol </td></tr><tr><td>PadR </td><td>253 </td><td><strong>PF03551</strong> </td><td>— </td><td>— </td><td>— </td><td>PadR_bsub </td></tr><tr><td>RpiR </td><td>227 </td><td><strong>PF01418</strong> </td><td>PF01380 </td><td> </td><td>c.80.1.1; c.80.1.3 </td><td>RpiR_ecol </td></tr><tr><td>Rrf2 </td><td>218 </td><td><strong>PF02082</strong> </td><td>— </td><td>— </td><td>— </td><td>IscR_ecol </td></tr><tr><td>DnaA </td><td>139 </td><td>— </td><td>— </td><td><strong>a.4.12.2</strong> </td><td>c.37.1.20 </td><td>DnaA_ecol </td></tr><tr><td>BolA/YrbA </td><td>121 </td><td><strong>PF01722</strong> </td><td>— </td><td><strong>d.52.6.1</strong> </td><td>— </td><td>BolA_ecol </td></tr><tr><td>ROK/NagC/XylR </td><td>118 </td><td>— </td><td>PF00480 </td><td><strong>a.4.5.1; a.4.5.24; a.4.5.28; a.4.5.32; a.4.5.36</strong> </td><td>— </td><td>NagC_ecol </td></tr><tr><td>LytTR </td><td>115 </td><td><strong>PF04397</strong> </td><td>PF00072 </td><td> </td><td>c.23.1.1 </td><td>LytT_bsub </td></tr><tr><td>SorC </td><td>113 </td><td>— </td><td>PF04198 </td><td><strong>a.4.5.4; a.4.13.2</strong> </td><td>— </td><td>SorC_sfle </td></tr><tr><td>ArgR </td><td>98 </td><td><strong>PF01316</strong> </td><td>PF02863 </td><td><strong>a.4.5.3</strong> </td><td>d.74.2.1 </td><td>ArgR_ecol </td></tr><tr><td>DtxR </td><td>92 </td><td><strong>PF01325</strong> </td><td>PF02742; PF04023 </td><td><strong>a.4.5.24</strong> </td><td>a.76.1.1; b.34.1.2 </td><td>DtxR_cdip </td></tr><tr><td>LexA </td><td>86 </td><td><strong>PF01726</strong> </td><td>PF00717 </td><td><strong>a.4.5.2</strong> </td><td>b.87.1.1 </td><td>LexA_ecol </td></tr><tr><td>TrmB </td><td>68 </td><td><strong>PF01978</strong> </td><td>— </td><td>— </td><td>— </td><td>AF1009_aful </td></tr><tr><td>BirA </td><td>67 </td><td>— </td><td>PF02237; PF03099 </td><td><strong>a.4.5.1</strong> </td><td>b.34.1.1; d.104.1.2 </td><td>BirA_ecol </td></tr><tr><td>PenR/BlaI/MecI </td><td>59 </td><td><strong>PF03965</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>BlaI_bant </td></tr><tr><td>SfsA </td><td>57 </td><td><strong>PF03749</strong> </td><td>— </td><td>— </td><td>— </td><td>SfsA_ecol </td></tr><tr><td>Nlp </td><td>42 </td><td>— </td><td>— </td><td><strong>a.35.1.2</strong> </td><td>— </td><td>SfsB_ecol </td></tr><tr><td>Archaeal HTH-10 </td><td>40 </td><td><strong>PF04967</strong> </td><td>— </td><td>— </td><td>— </td><td>AF0805_aful </td></tr><tr><td>CopG/RepA </td><td>38 </td><td><strong>PF01402</strong> </td><td>— </td><td>— </td><td>— </td><td>NikR_ecol </td></tr><tr><td>PutA </td><td>38 </td><td>— </td><td>PF00171; PF01619 </td><td><strong>a.176.1.1</strong> </td><td>c.1.23.2; c.82.1.1 </td><td>PutA_ecol </td></tr><tr><td>ModE </td><td>31 </td><td><strong>PF02573</strong> </td><td>PF03459 </td><td><strong>a.4.5.8</strong> </td><td>b.40.6.1; b.40.6.2 </td><td>ModE_ecol </td></tr><tr><td>PaiB </td><td>30 </td><td><strong>PF04299</strong> </td><td>— </td><td>— </td><td>— </td><td>PaiB_bsub </td></tr><tr><td>CtsR </td><td>28 </td><td><strong>PF05848</strong> </td><td>— </td><td>— </td><td>— </td><td>CtsR_bsub </td></tr><tr><td>AfsR/DnrI/RedD </td><td>27 </td><td><strong>PF00486</strong> </td><td>PF03704 </td><td>— </td><td>— </td><td>EmbR_mtub </td></tr><tr><td>CodY </td><td>27 </td><td><strong>PF06018</strong> </td><td>— </td><td>— </td><td>— </td><td>CodY_bsub </td></tr><tr><td>TrpR </td><td>25 </td><td><strong>PF01371</strong> </td><td>— </td><td><strong>a.4.12.1</strong> </td><td>— </td><td>TrpR_ecol </td></tr><tr><td>MtlR </td><td>24 </td><td><strong>PF05068</strong> </td><td>— </td><td>— </td><td>— </td><td>MtlR_ecol </td></tr><tr><td>ROS/MUCR </td><td>22 </td><td><strong>PF05443</strong> </td><td>— </td><td>— </td><td>— </td><td>Ros_atum </td></tr><tr><td>MetJ </td><td>21 </td><td><strong>PF01340</strong> </td><td>— </td><td><strong>a.43.1.2</strong> </td><td>— </td><td>MetJ_ecol </td></tr><tr><td>GutM </td><td>17 </td><td><strong>PF06923</strong> </td><td>— </td><td>— </td><td>— </td><td>GutM_ecol </td></tr><tr><td>Crl </td><td>16 </td><td><strong>PF07417</strong> </td><td>— </td><td>— </td><td>— </td><td>Crl_ecol </td></tr><tr><td>ComK </td><td>14 </td><td><strong>PF06338</strong> </td><td>— </td><td>— </td><td>— </td><td>ComK_bsub </td></tr><tr><td>FlhD </td><td>14 </td><td><strong>PF05247</strong> </td><td>— </td><td><strong>a.145.1.1</strong> </td><td>— </td><td>FlhD_ecol </td></tr><tr><td>RtcR </td><td>11 </td><td><strong>PF06956</strong> </td><td>PF00158 </td><td>— </td><td>— </td><td>RtcR_ecol </td></tr><tr><td>Spo0A </td><td>9 </td><td>— </td><td>PF00072 </td><td><strong>a.4.6.3</strong> </td><td>c.23.1.1 </td><td>Spo0A_bsub </td></tr><tr><td>DctR </td><td>6 </td><td>— </td><td>— </td><td><strong>a.4.5.1</strong> </td><td>c.23.1.1 </td><td>DctR_bsub </td></tr><tr><td>NifT/FixU </td><td>6 </td><td><strong>PF06988</strong> </td><td>— </td><td>— </td><td>— </td><td>NifT_anab </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>TF family </th><th>Total count </th><th colspan="2">Pfam </th><th colspan="2">SCOP </th><th>Representative TF </th></tr><tr><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th> </th></tr><tr><th> </th><th> </th><th>DBD </th><th>Non-DBD </th><th>DBD </th><th>Non-DBD </th><th> </th></tr></thead><tbody><tr><td>LysR </td><td>2530 </td><td><strong>PF00126</strong> </td><td>PF03466 </td><td><strong>a.4.5.8</strong> </td><td>c.94.1.1 </td><td>LysR_ecol </td></tr><tr><td>TetR/AcrR </td><td>1681 </td><td><strong>PF00440</strong> </td><td>— </td><td><strong>a.4.1.9</strong> </td><td>— </td><td>AcrR_ecol </td></tr><tr><td>GntR </td><td>1394 </td><td><strong>PF00392</strong> </td><td>PF00155; PF00532; PF07702 </td><td><strong>a.4.5.6</strong> </td><td>c.67.1.1; c.93.1.1 </td><td>GntR_tten </td></tr><tr><td>AraC </td><td>1375 </td><td><strong>PF00165</strong> </td><td>PF01965; PF02311; PF02805 </td><td><strong>a.4.1.8</strong> </td><td>b.82.1.9; c.23.16.2; c.55.7.1; g.48.1.1 </td><td>AraC_ecol </td></tr><tr><td>CRO/CI/Xre </td><td>1259 </td><td><strong>PF01381</strong> </td><td>— </td><td><strong>a.35.1.2; a.35.1.3</strong> </td><td>— </td><td>DicA_ecol </td></tr><tr><td>OmpR </td><td>1241 </td><td><strong>PF00486</strong> </td><td>PF00072 </td><td><strong>a.4.6.1</strong> </td><td>c.23.1.1; c.23.1.3 </td><td>OmpR_ecol </td></tr><tr><td>LuxR/NarL </td><td>1071 </td><td><strong>PF00196</strong> </td><td>PF00072; PF03472 </td><td><strong>a.4.6.2</strong> </td><td>c.23.1.1; d.110.5.1 </td><td>NarL_ecol </td></tr><tr><td>MarR </td><td>948 </td><td><strong>PF01047</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>MarR_ecol </td></tr><tr><td>LacI </td><td>750 </td><td><strong>PF00356</strong> </td><td>PF00532 </td><td><strong>a.35.1.5</strong> </td><td>c.93.1.1 </td><td>LacI_ecol </td></tr><tr><td>ArsR </td><td>622 </td><td><strong>PF01022</strong> </td><td>— </td><td><strong>a.4.5.5; a.4.5.36</strong> </td><td>— </td><td>ArsR_ecol </td></tr><tr><td>Fis </td><td>570 </td><td><strong>PF02954</strong> </td><td>PF00072; PF00158; PF01590; PF06506 </td><td><strong>a.105.1.1</strong> </td><td>c.23.1.1; c.37.1.20; d.110.2.1 </td><td>Fis_ecol </td></tr><tr><td>MerR </td><td>549 </td><td><strong>PF00376</strong> </td><td>— </td><td><strong>a.6.1.3</strong> </td><td>— </td><td>MerR_styp </td></tr><tr><td>AsnC/Lrp </td><td>439 </td><td>— </td><td>— </td><td><strong>a.4.5.28; a.4.5.32</strong> </td><td>d.58.4.2 </td><td>AsnC_ecol </td></tr><tr><td>DeoR </td><td>355 </td><td>— </td><td>— </td><td><strong>a.4.5.1; a.4.5.24</strong> </td><td>c.35.1.2; c.63.1.3 </td><td>DeoR_ecol </td></tr><tr><td>Crp/Fnr </td><td>353 </td><td><strong>PF00325</strong> </td><td>PF00027 </td><td><strong>a.4.5.4</strong> </td><td>b.82.3.1; b.82.3.2 </td><td>Crp_ecol </td></tr><tr><td>Fur </td><td>265 </td><td><strong>PF01475</strong> </td><td>— </td><td>— </td><td>— </td><td>Fur_ecol </td></tr><tr><td>PadR </td><td>253 </td><td><strong>PF03551</strong> </td><td>— </td><td>— </td><td>— </td><td>PadR_bsub </td></tr><tr><td>RpiR </td><td>227 </td><td><strong>PF01418</strong> </td><td>PF01380 </td><td> </td><td>c.80.1.1; c.80.1.3 </td><td>RpiR_ecol </td></tr><tr><td>Rrf2 </td><td>218 </td><td><strong>PF02082</strong> </td><td>— </td><td>— </td><td>— </td><td>IscR_ecol </td></tr><tr><td>DnaA </td><td>139 </td><td>— </td><td>— </td><td><strong>a.4.12.2</strong> </td><td>c.37.1.20 </td><td>DnaA_ecol </td></tr><tr><td>BolA/YrbA </td><td>121 </td><td><strong>PF01722</strong> </td><td>— </td><td><strong>d.52.6.1</strong> </td><td>— </td><td>BolA_ecol </td></tr><tr><td>ROK/NagC/XylR </td><td>118 </td><td>— </td><td>PF00480 </td><td><strong>a.4.5.1; a.4.5.24; a.4.5.28; a.4.5.32; a.4.5.36</strong> </td><td>— </td><td>NagC_ecol </td></tr><tr><td>LytTR </td><td>115 </td><td><strong>PF04397</strong> </td><td>PF00072 </td><td> </td><td>c.23.1.1 </td><td>LytT_bsub </td></tr><tr><td>SorC </td><td>113 </td><td>— </td><td>PF04198 </td><td><strong>a.4.5.4; a.4.13.2</strong> </td><td>— </td><td>SorC_sfle </td></tr><tr><td>ArgR </td><td>98 </td><td><strong>PF01316</strong> </td><td>PF02863 </td><td><strong>a.4.5.3</strong> </td><td>d.74.2.1 </td><td>ArgR_ecol </td></tr><tr><td>DtxR </td><td>92 </td><td><strong>PF01325</strong> </td><td>PF02742; PF04023 </td><td><strong>a.4.5.24</strong> </td><td>a.76.1.1; b.34.1.2 </td><td>DtxR_cdip </td></tr><tr><td>LexA </td><td>86 </td><td><strong>PF01726</strong> </td><td>PF00717 </td><td><strong>a.4.5.2</strong> </td><td>b.87.1.1 </td><td>LexA_ecol </td></tr><tr><td>TrmB </td><td>68 </td><td><strong>PF01978</strong> </td><td>— </td><td>— </td><td>— </td><td>AF1009_aful </td></tr><tr><td>BirA </td><td>67 </td><td>— </td><td>PF02237; PF03099 </td><td><strong>a.4.5.1</strong> </td><td>b.34.1.1; d.104.1.2 </td><td>BirA_ecol </td></tr><tr><td>PenR/BlaI/MecI </td><td>59 </td><td><strong>PF03965</strong> </td><td>— </td><td><strong>a.4.5.28</strong> </td><td>— </td><td>BlaI_bant </td></tr><tr><td>SfsA </td><td>57 </td><td><strong>PF03749</strong> </td><td>— </td><td>— </td><td>— </td><td>SfsA_ecol </td></tr><tr><td>Nlp </td><td>42 </td><td>— </td><td>— </td><td><strong>a.35.1.2</strong> </td><td>— </td><td>SfsB_ecol </td></tr><tr><td>Archaeal HTH-10 </td><td>40 </td><td><strong>PF04967</strong> </td><td>— </td><td>— </td><td>— </td><td>AF0805_aful </td></tr><tr><td>CopG/RepA </td><td>38 </td><td><strong>PF01402</strong> </td><td>— </td><td>— </td><td>— </td><td>NikR_ecol </td></tr><tr><td>PutA </td><td>38 </td><td>— </td><td>PF00171; PF01619 </td><td><strong>a.176.1.1</strong> </td><td>c.1.23.2; c.82.1.1 </td><td>PutA_ecol </td></tr><tr><td>ModE </td><td>31 </td><td><strong>PF02573</strong> </td><td>PF03459 </td><td><strong>a.4.5.8</strong> </td><td>b.40.6.1; b.40.6.2 </td><td>ModE_ecol </td></tr><tr><td>PaiB </td><td>30 </td><td><strong>PF04299</strong> </td><td>— </td><td>— </td><td>— </td><td>PaiB_bsub </td></tr><tr><td>CtsR </td><td>28 </td><td><strong>PF05848</strong> </td><td>— </td><td>— </td><td>— </td><td>CtsR_bsub </td></tr><tr><td>AfsR/DnrI/RedD </td><td>27 </td><td><strong>PF00486</strong> </td><td>PF03704 </td><td>— </td><td>— </td><td>EmbR_mtub </td></tr><tr><td>CodY </td><td>27 </td><td><strong>PF06018</strong> </td><td>— </td><td>— </td><td>— </td><td>CodY_bsub </td></tr><tr><td>TrpR </td><td>25 </td><td><strong>PF01371</strong> </td><td>— </td><td><strong>a.4.12.1</strong> </td><td>— </td><td>TrpR_ecol </td></tr><tr><td>MtlR </td><td>24 </td><td><strong>PF05068</strong> </td><td>— </td><td>— </td><td>— </td><td>MtlR_ecol </td></tr><tr><td>ROS/MUCR </td><td>22 </td><td><strong>PF05443</strong> </td><td>— </td><td>— </td><td>— </td><td>Ros_atum </td></tr><tr><td>MetJ </td><td>21 </td><td><strong>PF01340</strong> </td><td>— </td><td><strong>a.43.1.2</strong> </td><td>— </td><td>MetJ_ecol </td></tr><tr><td>GutM </td><td>17 </td><td><strong>PF06923</strong> </td><td>— </td><td>— </td><td>— </td><td>GutM_ecol </td></tr><tr><td>Crl </td><td>16 </td><td><strong>PF07417</strong> </td><td>— </td><td>— </td><td>— </td><td>Crl_ecol </td></tr><tr><td>ComK </td><td>14 </td><td><strong>PF06338</strong> </td><td>— </td><td>— </td><td>— </td><td>ComK_bsub </td></tr><tr><td>FlhD </td><td>14 </td><td><strong>PF05247</strong> </td><td>— </td><td><strong>a.145.1.1</strong> </td><td>— </td><td>FlhD_ecol </td></tr><tr><td>RtcR </td><td>11 </td><td><strong>PF06956</strong> </td><td>PF00158 </td><td>— </td><td>— </td><td>RtcR_ecol </td></tr><tr><td>Spo0A </td><td>9 </td><td>— </td><td>PF00072 </td><td><strong>a.4.6.3</strong> </td><td>c.23.1.1 </td><td>Spo0A_bsub </td></tr><tr><td>DctR </td><td>6 </td><td>— </td><td>— </td><td><strong>a.4.5.1</strong> </td><td>c.23.1.1 </td><td>DctR_bsub </td></tr><tr><td>NifT/FixU </td><td>6 </td><td><strong>PF06988</strong> </td><td>— </td><td>— </td><td>— </td><td>NifT_anab </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>DBDs characterized by Pfam/SCOP ID codes are indicated in boldface. Only those ID codes of Pfam and SCOP that were used to identify proteins as TFs are indicated in the table. A representative TF for each TF family is shown as a protein name plus the shortened name of the species. The full species names are given in GTOP_TF.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35871106" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=35871107 id="35871107" class="section-title" >2.3. Selection of TFs with experimental evidence</h3>
<p> We first surveyed all the prokaryotic entries of Swiss-Prot to select TFs annotated with direct experimental evidence, excluding those annotated ‘by similarity’. The selected entries were manually checked as to whether the literature cited therein provided genuine experimental evidence, and those failing such confirmation were removed. The final count of the Swiss-Prot entries that satisfy our aforementioned criteria of TFs is 382, of which 135 were from <em>E. coli</em> and 48 came from <em>Bacillus subtilis.</em> The remaining 199 TFs were taken from various species, including 13 TFs from six archaeal species: <em>Sulfolobus solfataricus</em> , <em>Methanosarcina acetivorans</em> , <em>Halobacterium sp. NRC-1</em> , <em>M. jannaschii</em> , <em>Methanococcus maripaludis</em> and <em>Pyrococcus furiosus</em> . To find more TFs with experimental evidence, we conducted an additional survey of TF candidates in Pfam and SCOP database with experimental evidence, and added their homologs to form a set of TF candidates. Only those candidate TFs that meet the criteria of the previous section and that are not annotated as TFs in Swiss-Prot were retained. We then conducted a literature search for every remaining candidate to identify TFs with direct experimental evidence. With 12 TFs found in this process, we obtained a total of 394 TFs with experimental evidence and designated it as the set of experimentally verified TFs. </p>                            <h2 scrollto-destination=35871109 id="35871109" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=35871110 id="35871110" class="section-title" >3.1. Detection of TFs based on selection rules</h3>
<p> Examining many TFs, we noticed that most of them can be detected by a combination of a DBD and a contiguous non-DBD with specific SCOP or Pfam domains as the identifier of TFs ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1a</a> ). We note that, besides the directionality of DBDs and non-DBDs in the figure, a DBD may be present at the C-terminal end of a non-DBD. <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> Homology search for the entire sequence of gene/protein may detect three proteins (MglB, LacI and AraR) as homologs having similar functions, because a large part of the sequence can be aligned to each other. In reality, however, MglB is a periplasmic ligand binding protein, whereas both LacI and AraR function as TFs (repressors). We note that LacI and AraR are classified into different TF families (the LacI and GntR families, respectively) based on the difference in the N-terminal DBDs, although they possess a common structural domain (SCOP ID: c.93.1.1) at the C-termini. This example illustrates that despite the small size of a DBD, its presence is crucial for a protein to function as a TF. It should be emphasized that the existence of a SCOP or Pfam domain characteristic of a DBD is necessary, but it is not a sufficient condition for a TF: for example, despite having the same SCOP domain (a.4.6.3) as assigned to the DBD of a TF, SCOP domain (c.55.7.1) is annotated as methylated-DNA-protein-cysteine methyltransferase in Swiss-Prot and is not a TF ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1b</a> ). The combination approach thus reduces the number of non-TFs erroneously detected as TFs in approaches involving the DBD only. </p>                        <a id="35871112" scrollto-destination="35871112"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f1.jpeg?Expires=2147483647&Signature=l1sI2~NlflEe7ilThumJ0QP5HKy-9zBtQbczzf~4wg6bdQLVVIrPdA01sOh3JeiUKNpj4nS0d9jmTwJ8aeOdRlHX0ynzmh53yLt0UAf0~s-lTh1qNeduQIae0335YgrwcDCUPq2UScxm5dYQIU1OjxcsvDcIxlm6jN8tLAecL9lxh~eC7hGdVPWIWGdK0V4NaZ6QGk~Ji~axVR-BOHdD9JBv2lGKXK0sLRS19EbB6j9VQAR8Y5FJznF1WKCFzQRP9qSkGcTY-vStL8LZSr3EeHiPJbd974fJNqZtOSR4JRmkoCTB~RIu7OlNTdBVoovrvQmB0hVVaoavctztjzXsGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Domain organization of TFs. ( a ) LacI and AraR are TFs with DBDs, while MglB is not a TF because it has no DBD. ( b ) Despite the presence of the same SCOP domain (a.4.6.3) in Ogt and spo0A, the former is not a TF, while the latter is. " path-from-xml="dsi016f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871112/dsi016f1.jpeg" path-from-xml="dsi016f1.jpeg" target="_blank">View large</a><a section="35871112" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f1.jpeg?Expires=2147483647&Signature=VuOllIQ9Zb5F3mzGfDwT2YOhW0oSpvoXfqCst9vqrdwgn2hiWjFx9MSetflEyfz1cBO8cZ7HlUR3USVYGBLvzF3lcGc8hQAuIpAW55Vr4Af8gYLCj27x3I8YIadAWu1E2hV4yUxx2svaAEXLj-ULlqJcPYedaYLMfwANh2E66nFok1QqrOyFcRpfT9jVUjbkdxnh1JQvHokxI5PXOSZYidm6JT5gYfeHU1ZCSU9w4a4yhOcMbpwBLBq~FhX0syoNymB7Ho8vljBadEuGS0U1YRY8lxduyLaxd7W5STu~xW27Lqv2Quafp~7qu8f0WSYJ723tEWKzsylZ3mjyDM6ouQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871112&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Domain organization of TFs. ( <strong>a</strong> ) LacI and AraR are TFs with DBDs, while MglB is not a TF because it has no DBD. ( <strong>b</strong> ) Despite the presence of the same SCOP domain (a.4.6.3) in Ogt and spo0A, the former is not a TF, while the latter is. </p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f1.jpeg?Expires=2147483647&Signature=l1sI2~NlflEe7ilThumJ0QP5HKy-9zBtQbczzf~4wg6bdQLVVIrPdA01sOh3JeiUKNpj4nS0d9jmTwJ8aeOdRlHX0ynzmh53yLt0UAf0~s-lTh1qNeduQIae0335YgrwcDCUPq2UScxm5dYQIU1OjxcsvDcIxlm6jN8tLAecL9lxh~eC7hGdVPWIWGdK0V4NaZ6QGk~Ji~axVR-BOHdD9JBv2lGKXK0sLRS19EbB6j9VQAR8Y5FJznF1WKCFzQRP9qSkGcTY-vStL8LZSr3EeHiPJbd974fJNqZtOSR4JRmkoCTB~RIu7OlNTdBVoovrvQmB0hVVaoavctztjzXsGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Domain organization of TFs. ( a ) LacI and AraR are TFs with DBDs, while MglB is not a TF because it has no DBD. ( b ) Despite the presence of the same SCOP domain (a.4.6.3) in Ogt and spo0A, the former is not a TF, while the latter is. " path-from-xml="dsi016f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871112/dsi016f1.jpeg" path-from-xml="dsi016f1.jpeg" target="_blank">View large</a><a section="35871112" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f1.jpeg?Expires=2147483647&Signature=VuOllIQ9Zb5F3mzGfDwT2YOhW0oSpvoXfqCst9vqrdwgn2hiWjFx9MSetflEyfz1cBO8cZ7HlUR3USVYGBLvzF3lcGc8hQAuIpAW55Vr4Af8gYLCj27x3I8YIadAWu1E2hV4yUxx2svaAEXLj-ULlqJcPYedaYLMfwANh2E66nFok1QqrOyFcRpfT9jVUjbkdxnh1JQvHokxI5PXOSZYidm6JT5gYfeHU1ZCSU9w4a4yhOcMbpwBLBq~FhX0syoNymB7Ho8vljBadEuGS0U1YRY8lxduyLaxd7W5STu~xW27Lqv2Quafp~7qu8f0WSYJ723tEWKzsylZ3mjyDM6ouQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871112&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Domain organization of TFs. ( <strong>a</strong> ) LacI and AraR are TFs with DBDs, while MglB is not a TF because it has no DBD. ( <strong>b</strong> ) Despite the presence of the same SCOP domain (a.4.6.3) in Ogt and spo0A, the former is not a TF, while the latter is. </p></div></div><p> We individually determined the domain organization pattern of each TF family from the set of experimentally verified TFs. We classified TFs into families according to the specific combinations of Pfam and/or SCOP domains corresponding to the DBDs and non-DBDs they contain. Each TF family consists of members of the set of experimentally verified TFs that have both a DBD and a contiguous non-DBD with specific Pfam or SCOP domains. For example, the rule for the Spo0A family ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1b</a> and <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> ) stipulates that proteins contain at least one of the combinations of a DBD and a non-DBD, a.4.6.3-PF00072 and a.4.6.3-c.23.1.1, and thereby excludes Ogt_bsub, because this protein has neither a PF00072 nor a c.23.1.1 domain juxtaposed to the DBD. </p><p> In this way, we set up selection rules for individual TF families through analyses of the domain organization of experimentally verified TFs. We determined a rule for each TF family by trial and error; when evident false positives were found, then the selection rule was revised and the process was repeated until there remained no false positives. By Pfam or SCOP combinations, we can detect 86.1% of TFs in the experimentally verified set. In addition, the following cases were included in the selection rules. Many TFs with only DBD assignment (e.g. those in the TetR/AcrR family) represent those whose non-DBDs have yet to be assigned. A small number of TFs, e.g. those in the PaiB family, are short and clearly possess DBDs only, because non-DBDs cannot possibly be present in the remainder of the small protein. Both kinds will be called ‘solitary’ TF families below, and together constitute 9.8% of the TFs. The rest in the set of experimentally verified TFs can be detected using alignment to existing members of TF families with the requirement that the length of the hit sequences differ from that of the query by &lt;20% (expressed as ‘aligned to a TF of comparable length’ in the following). The length requirement is intended to ensure alignment for nearly the entire length including the DBD, which tends to be short, and thereby to minimize over-assignment. COG number(s) (from the COG database, <a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> 20 June 2004 release) specifically assigned to most of the TF families serve to verify sequence alignments. <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> presents a list of 52 TF families, each with Pfam and SCOP identifiers and a representative TF. </p><p> Based on the analysis of the experimentally verified TFs, we set up the following criteria for TF selection. A protein with a DBD and a contiguous non-DBD with a combination of Pfam or SCOP domains specific to each TF family ( <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> ) is said to be a high confidence TF. A protein with a DBD of a Pfam or SCOP domain specific for a solitary TF family is considered as a high confidence TF, too, if it is aligned to an experimentally verified TF of comparable length belonging to the same family. Furthermore a protein with a Pfam or SCOP domain characteristic of a TF family but without non-DBD assignment is also classified as a high confidence TF, if it is aligned to a TF family member of comparable length. On the other hand, a protein lacking an identifiable DBD is judged to be a highly probable TF if it is aligned to an experimentally verified TF of comparable length. Finally a protein is regarded as a probable TF if it is detected with a DBD of a Pfam or SCOP domain characteristic of a TF family, but with no homologs in the set of experimentally verified TFs. The high quality set is defined to include only high confidence and highly probable TFs. Our analyses of TFs are based exclusively on high quality TFs (see below). </p><p> The aforementioned TF assignment procedure was applied to 154 wholly sequenced prokaryotic genomes in GTOP. The total number of TFs detected in the high quality set was 18 577, 95% of which were detected as high confidence TFs, while the remainder (5%) were classified as highly probable TFs. <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> lists the total count of high confidence TFs in each TF family. The largest TF family is LysR, followed by TetR/ArcR and GntR. Though we did not take the order of DBDs and non-DBDs into account, each TF family turned out to have a specific order with absolutely no exceptions. This finding is in agreement with a previous investigation <a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> reporting that the order of domains is nearly fixed in prokaryotic proteins. The number distribution of TF families shows that a small number of major TF families and a large number of minor ones exist. This tendency is in good agreement with the reported distribution of <em>E. coli</em> TFs <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> that obeys the power law. <a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> Detailed data of high confidence and highly probable TFs as well as the numbers of TFs and TF families assigned in each genome are available from the GTOP_TF database ( <a class="link link-uri" href="http://spock.genes.nig.ac.jp/~gtop_tf/index2.html" target="_blank"></a> ) and the Supplementary Table ( <a class="link link-uri" href="http://spock.genes.nig.ac.jp/~gtop_tf/tableS1.pdf" target="_blank"></a> ). </p>                            <h3 scrollto-destination=35871117 id="35871117" class="section-title" >3.2. Comparison with previous studies</h3>
<p> In order to check the precision of the TF detection method, it is important to compare the results with those reported in the literature. Such comparisons are, however, not straightforward to make because the definitions of TFs and TF families often differ from one study to another. For instance, Babu and Teichmann <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> employed the SCOP domain as the identifier of TFs in their thorough TF analysis for <em>E. coli</em> . However, as their definition of the TF families is based on DBDs specified by the SCOP superfamilies and not the SCOP and Pfam families as in our case, it is difficult to compare the two results. It is nevertheless possible to compare the two results at the individual protein level using one-to-one correspondence. Babu and Teichmann <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> detected 271 TFs from <em>E. coli</em> , 233 of which were identical to our high quality set ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). Out of the 38 proteins found exclusively in their set, 13 proteins [e.g. CspA (cold-shock protein), Hns (histon-like protein) and RpoE (sigma-E factor)] were attributable to the different definition they use, while the remaining 25 were categorized as ‘probable’ TFs and therefore not included in the high quality set. Almost all of these probable TFs, e.g. YafY, YaiV and YbaQ, are products of hypothetical genes and have been poorly characterized. In contrast 19 TFs with experimental support, including 11 annotated in Swiss-Prot (e.g. BolA, CaiF, and CdaR), were missed by Babu and Teichmann presumably because the unavailability of the structure of the DBDs in these proteins made it impossible to identify them as TFs by SCOP search alone. </p>                        <a id="35871119" scrollto-destination="35871119"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f2.jpeg?Expires=2147483647&Signature=4IZCESMZFh~~TYVMUFE9Ys-1Q9ibN8V4CZF3Uk4hKSyj6Rc5JH96fRhDIFftmcK1Z2808yczeFF-g7K5XD~WoTmOjpHd1sar1ic-vVkXopOLjgPmHCc-RhWKaM7IsR6dxnmO~oUMK8mjnmqOmk~V1Y-E9yd6Aj2ISQyqodi9bj3A3wOEj2rgjD~hFPIUCmoWQhUkKVypYUhMvB5~c3~3yo15NV~a7fVDB2kuSDQ5ZyGDo76nQjZZO5YATLMI7EfPauXFnvmPrYqGWjSetMtf7p1lABFH4HZiw-n4ewX951QifblRZ20blwWTknxDmF8EyRVnk-SzgSq-OjrckHoKww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Relationship of E. coli TFs annotated in Swiss-Prot, the high quality TFs (present study) and TFs annotated by Babu and Teichmann. 13 The 11 proteins shared by Swiss-Prot and GTOP_TF are BolA, CaiF, CdaR Crl, DcuR, DpiA, GutM, MalY, MtlR, NikR and PutA. The 8 proteins assigned only in the present study are SfsA, YehT, YggD, YjgJ, YjhU, YpdB, YqjI and YrbA. The 38 proteins exclusively assigned by Babu and Teichmann 13 are CspA, FecI, FrvR, Hns, IhfA, IhfB, PinQ, RacR, RpoE, StpA, YafY, YagA, YaiV, YbaQ, YdaW, YdcQ, YddM, YeiI, YfeC, YfeD, YffH, YffS, YfgA, YfjR, YgeH, YheO, YhgG, YhiE, YiiF, YjjM, YmfN, YqeH, YqeI, B0373, B0502, B0540, B1027 and B1146. " path-from-xml="dsi016f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871119/dsi016f2.jpeg" path-from-xml="dsi016f2.jpeg" target="_blank">View large</a><a section="35871119" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f2.jpeg?Expires=2147483647&Signature=tRZtegyGbdzAPB7zBo0rt8EBk6AKPg9uxi8OdLE6WE4memacCxEVbJveX7O4z~gPUcUUU-n1qdIu9SQI5ilLjYu0Fag2~~9wYIRMudmwD8arSsyFPCEcHmTopaeVog4IcvflCtpuFii2c9JAw1mudRL6RnUmcilB8dhcC34NVbNAJDWBepnfrtGKT8aobkfAn38GgjVlXlQuaUkhZvL9M6IZ1a3e4qm6L-HloQbGiUnwLTDD019nKGkHDz8aU-zN2DL4xmjN2R6Q37dBQfCYCNbej8VUHmgcD86EZXTqvAxQVtm7dTifLm1p~zetIuhh0X6lafSueI2fPuMoBeCzeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871119&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Relationship of <em>E. coli</em> TFs annotated in Swiss-Prot, the high quality TFs (present study) and TFs annotated by Babu and Teichmann. <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> The 11 proteins shared by Swiss-Prot and GTOP_TF are BolA, CaiF, CdaR Crl, DcuR, DpiA, GutM, MalY, MtlR, NikR and PutA. The 8 proteins assigned only in the present study are SfsA, YehT, YggD, YjgJ, YjhU, YpdB, YqjI and YrbA. The 38 proteins exclusively assigned by Babu and Teichmann <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> are CspA, FecI, FrvR, Hns, IhfA, IhfB, PinQ, RacR, RpoE, StpA, YafY, YagA, YaiV, YbaQ, YdaW, YdcQ, YddM, YeiI, YfeC, YfeD, YffH, YffS, YfgA, YfjR, YgeH, YheO, YhgG, YhiE, YiiF, YjjM, YmfN, YqeH, YqeI, B0373, B0502, B0540, B1027 and B1146. </p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f2.jpeg?Expires=2147483647&Signature=4IZCESMZFh~~TYVMUFE9Ys-1Q9ibN8V4CZF3Uk4hKSyj6Rc5JH96fRhDIFftmcK1Z2808yczeFF-g7K5XD~WoTmOjpHd1sar1ic-vVkXopOLjgPmHCc-RhWKaM7IsR6dxnmO~oUMK8mjnmqOmk~V1Y-E9yd6Aj2ISQyqodi9bj3A3wOEj2rgjD~hFPIUCmoWQhUkKVypYUhMvB5~c3~3yo15NV~a7fVDB2kuSDQ5ZyGDo76nQjZZO5YATLMI7EfPauXFnvmPrYqGWjSetMtf7p1lABFH4HZiw-n4ewX951QifblRZ20blwWTknxDmF8EyRVnk-SzgSq-OjrckHoKww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Relationship of E. coli TFs annotated in Swiss-Prot, the high quality TFs (present study) and TFs annotated by Babu and Teichmann. 13 The 11 proteins shared by Swiss-Prot and GTOP_TF are BolA, CaiF, CdaR Crl, DcuR, DpiA, GutM, MalY, MtlR, NikR and PutA. The 8 proteins assigned only in the present study are SfsA, YehT, YggD, YjgJ, YjhU, YpdB, YqjI and YrbA. The 38 proteins exclusively assigned by Babu and Teichmann 13 are CspA, FecI, FrvR, Hns, IhfA, IhfB, PinQ, RacR, RpoE, StpA, YafY, YagA, YaiV, YbaQ, YdaW, YdcQ, YddM, YeiI, YfeC, YfeD, YffH, YffS, YfgA, YfjR, YgeH, YheO, YhgG, YhiE, YiiF, YjjM, YmfN, YqeH, YqeI, B0373, B0502, B0540, B1027 and B1146. " path-from-xml="dsi016f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871119/dsi016f2.jpeg" path-from-xml="dsi016f2.jpeg" target="_blank">View large</a><a section="35871119" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f2.jpeg?Expires=2147483647&Signature=tRZtegyGbdzAPB7zBo0rt8EBk6AKPg9uxi8OdLE6WE4memacCxEVbJveX7O4z~gPUcUUU-n1qdIu9SQI5ilLjYu0Fag2~~9wYIRMudmwD8arSsyFPCEcHmTopaeVog4IcvflCtpuFii2c9JAw1mudRL6RnUmcilB8dhcC34NVbNAJDWBepnfrtGKT8aobkfAn38GgjVlXlQuaUkhZvL9M6IZ1a3e4qm6L-HloQbGiUnwLTDD019nKGkHDz8aU-zN2DL4xmjN2R6Q37dBQfCYCNbej8VUHmgcD86EZXTqvAxQVtm7dTifLm1p~zetIuhh0X6lafSueI2fPuMoBeCzeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871119&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Relationship of <em>E. coli</em> TFs annotated in Swiss-Prot, the high quality TFs (present study) and TFs annotated by Babu and Teichmann. <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> The 11 proteins shared by Swiss-Prot and GTOP_TF are BolA, CaiF, CdaR Crl, DcuR, DpiA, GutM, MalY, MtlR, NikR and PutA. The 8 proteins assigned only in the present study are SfsA, YehT, YggD, YjgJ, YjhU, YpdB, YqjI and YrbA. The 38 proteins exclusively assigned by Babu and Teichmann <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> are CspA, FecI, FrvR, Hns, IhfA, IhfB, PinQ, RacR, RpoE, StpA, YafY, YagA, YaiV, YbaQ, YdaW, YdcQ, YddM, YeiI, YfeC, YfeD, YffH, YffS, YfgA, YfjR, YgeH, YheO, YhgG, YhiE, YiiF, YjjM, YmfN, YqeH, YqeI, B0373, B0502, B0540, B1027 and B1146. </p></div></div><p> Genome sequencing teams provide detailed lists of individual genes assigned to the genomes and classify them into functional groups. We found 11 such species, including 2 archaea ( <em>M. acetivorans</em> and <em>A. fulgidus</em> ), and compared the assignment of TFs with ours at the TF family level ( <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> ). The results of genome-wide manual annotations and our TF assignments generally agree. Moreover, the agreement is as good in species of archaea as those of bacteria. We therefore claim that the method developed in this research excludes doubtful cases and detects more TFs than those reported in the literature. The results also indicate that our procedure works as well on archaeal species as on bacteria despite the presence of only a small number of archaeal proteins in the set of experimentally verified TFs. </p>                        <a id="35871121" scrollto-destination="35871121"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Comparison of our TF assignments at the TF family level with those in primary annotations.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th></tr></thead><tbody><tr><td colspan="4"><em>M. acetivorans</em><a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a> </td><td colspan="4"><em>Lactobacillus johnsonii</em><a reveal-id="b43" data-open="b43" class="link link-ref link-reveal xref-bibr">43</a> </td></tr><tr><td> </td><td>TetR </td><td>15 </td><td>16 </td><td> </td><td>GntR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>MarR </td><td>11 </td><td>13 </td><td> </td><td>LacI </td><td>7 </td><td>7 </td></tr><tr><td> </td><td>Lrp </td><td>1 </td><td>3 </td><td> </td><td>RpiR </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>A. fulgidus</em><a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> </td><td> </td><td>ArsR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td>TetR </td><td>1 </td><td>1 </td><td> </td><td>LysR </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>MarR </td><td>1 </td><td>2 </td><td> </td><td>AraC </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>Lrp </td><td>14 </td><td>8 </td><td colspan="4"><em>Lactococcus lactis</em><a reveal-id="b44" data-open="b44" class="link link-ref link-reveal xref-bibr">44</a> </td></tr><tr><td colspan="4"><em>S. coelicolor</em><a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> </td><td> </td><td>LacI </td><td>7 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>33 </td><td>35 </td><td> </td><td>LysR </td><td>9 </td><td>6 </td></tr><tr><td> </td><td>WhiB </td><td>8 </td><td>13 </td><td> </td><td>AraC </td><td>3 </td><td>3 </td></tr><tr><td colspan="4"><em>Bifidobacterium longum</em><a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> </td><td> </td><td>GntR </td><td>4 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>22 </td><td>21 </td><td> </td><td>DeoR </td><td>4 </td><td>3 </td></tr><tr><td> </td><td>LysR </td><td>5 </td><td>5 </td><td> </td><td>MarR </td><td>11 </td><td>12 </td></tr><tr><td> </td><td>AraC </td><td>1 </td><td>1 </td><td colspan="4"><em>Fusobacterium nucleatum</em><a reveal-id="b45" data-open="b45" class="link link-ref link-reveal xref-bibr">45</a> </td></tr><tr><td> </td><td>WhiB </td><td>2 </td><td>3 </td><td> </td><td>TetR </td><td>6 </td><td>6 </td></tr><tr><td> </td><td>MerR </td><td>3 </td><td>4 </td><td> </td><td>GntR </td><td>6 </td><td>5 </td></tr><tr><td> </td><td>Fur </td><td>1 </td><td>1 </td><td> </td><td>DeoR </td><td>5 </td><td>2 </td></tr><tr><td colspan="4"><em>B. subtilis</em><a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> </td><td> </td><td>LuxR/LysR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>GntR </td><td>20 </td><td>20 </td><td> </td><td>MarR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LysR </td><td>19 </td><td>19 </td><td> </td><td>Crp </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LacI </td><td>12 </td><td>11 </td><td> </td><td>MerR </td><td>2 </td><td>1 </td></tr><tr><td> </td><td>AraC </td><td>11 </td><td>12 </td><td colspan="4"><em>Rhodopseudomonas palustris</em><a reveal-id="b46" data-open="b46" class="link link-ref link-reveal xref-bibr">46</a> </td></tr><tr><td> </td><td>Lrp </td><td>7 </td><td>6 </td><td> </td><td>AraC </td><td>23 </td><td>23 </td></tr><tr><td> </td><td>DeoR </td><td>6 </td><td>4 </td><td> </td><td>DeoR </td><td>1 </td><td>1 </td></tr><tr><td colspan="4"><em>Clostridium acetobutylicum</em><a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a> </td><td> </td><td>LuxR </td><td>11 </td><td>11 </td></tr><tr><td> </td><td>AcrR/TetR </td><td>28 </td><td>27 </td><td> </td><td>LysR </td><td>27 </td><td>27 </td></tr><tr><td> </td><td>MarR/RmrR </td><td>22 </td><td>18 </td><td> </td><td>MarR </td><td>17 </td><td>17 </td></tr><tr><td> </td><td>LysR </td><td>14 </td><td>14 </td><td> </td><td>ArsR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>LacI </td><td>9 </td><td>7 </td><td> </td><td>AsnC </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>Photorhabdus luminescens</em><a reveal-id="b42" data-open="b42" class="link link-ref link-reveal xref-bibr">42</a> </td><td> </td><td>Crp </td><td>15 </td><td>15 </td></tr><tr><td> </td><td>LuxR </td><td>32 </td><td>40 </td><td> </td><td>GntR </td><td>13 </td><td>13 </td></tr><tr><td> </td><td>LysR </td><td>20 </td><td>29 </td><td> </td><td>IclR </td><td>7 </td><td>6 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>MerR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>TetR </td><td>39 </td><td>38 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>Fis </td><td>13 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>CopG </td><td>2 </td><td>1 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th></tr></thead><tbody><tr><td colspan="4"><em>M. acetivorans</em><a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a> </td><td colspan="4"><em>Lactobacillus johnsonii</em><a reveal-id="b43" data-open="b43" class="link link-ref link-reveal xref-bibr">43</a> </td></tr><tr><td> </td><td>TetR </td><td>15 </td><td>16 </td><td> </td><td>GntR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>MarR </td><td>11 </td><td>13 </td><td> </td><td>LacI </td><td>7 </td><td>7 </td></tr><tr><td> </td><td>Lrp </td><td>1 </td><td>3 </td><td> </td><td>RpiR </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>A. fulgidus</em><a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> </td><td> </td><td>ArsR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td>TetR </td><td>1 </td><td>1 </td><td> </td><td>LysR </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>MarR </td><td>1 </td><td>2 </td><td> </td><td>AraC </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>Lrp </td><td>14 </td><td>8 </td><td colspan="4"><em>Lactococcus lactis</em><a reveal-id="b44" data-open="b44" class="link link-ref link-reveal xref-bibr">44</a> </td></tr><tr><td colspan="4"><em>S. coelicolor</em><a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> </td><td> </td><td>LacI </td><td>7 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>33 </td><td>35 </td><td> </td><td>LysR </td><td>9 </td><td>6 </td></tr><tr><td> </td><td>WhiB </td><td>8 </td><td>13 </td><td> </td><td>AraC </td><td>3 </td><td>3 </td></tr><tr><td colspan="4"><em>Bifidobacterium longum</em><a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> </td><td> </td><td>GntR </td><td>4 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>22 </td><td>21 </td><td> </td><td>DeoR </td><td>4 </td><td>3 </td></tr><tr><td> </td><td>LysR </td><td>5 </td><td>5 </td><td> </td><td>MarR </td><td>11 </td><td>12 </td></tr><tr><td> </td><td>AraC </td><td>1 </td><td>1 </td><td colspan="4"><em>Fusobacterium nucleatum</em><a reveal-id="b45" data-open="b45" class="link link-ref link-reveal xref-bibr">45</a> </td></tr><tr><td> </td><td>WhiB </td><td>2 </td><td>3 </td><td> </td><td>TetR </td><td>6 </td><td>6 </td></tr><tr><td> </td><td>MerR </td><td>3 </td><td>4 </td><td> </td><td>GntR </td><td>6 </td><td>5 </td></tr><tr><td> </td><td>Fur </td><td>1 </td><td>1 </td><td> </td><td>DeoR </td><td>5 </td><td>2 </td></tr><tr><td colspan="4"><em>B. subtilis</em><a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> </td><td> </td><td>LuxR/LysR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>GntR </td><td>20 </td><td>20 </td><td> </td><td>MarR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LysR </td><td>19 </td><td>19 </td><td> </td><td>Crp </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LacI </td><td>12 </td><td>11 </td><td> </td><td>MerR </td><td>2 </td><td>1 </td></tr><tr><td> </td><td>AraC </td><td>11 </td><td>12 </td><td colspan="4"><em>Rhodopseudomonas palustris</em><a reveal-id="b46" data-open="b46" class="link link-ref link-reveal xref-bibr">46</a> </td></tr><tr><td> </td><td>Lrp </td><td>7 </td><td>6 </td><td> </td><td>AraC </td><td>23 </td><td>23 </td></tr><tr><td> </td><td>DeoR </td><td>6 </td><td>4 </td><td> </td><td>DeoR </td><td>1 </td><td>1 </td></tr><tr><td colspan="4"><em>Clostridium acetobutylicum</em><a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a> </td><td> </td><td>LuxR </td><td>11 </td><td>11 </td></tr><tr><td> </td><td>AcrR/TetR </td><td>28 </td><td>27 </td><td> </td><td>LysR </td><td>27 </td><td>27 </td></tr><tr><td> </td><td>MarR/RmrR </td><td>22 </td><td>18 </td><td> </td><td>MarR </td><td>17 </td><td>17 </td></tr><tr><td> </td><td>LysR </td><td>14 </td><td>14 </td><td> </td><td>ArsR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>LacI </td><td>9 </td><td>7 </td><td> </td><td>AsnC </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>Photorhabdus luminescens</em><a reveal-id="b42" data-open="b42" class="link link-ref link-reveal xref-bibr">42</a> </td><td> </td><td>Crp </td><td>15 </td><td>15 </td></tr><tr><td> </td><td>LuxR </td><td>32 </td><td>40 </td><td> </td><td>GntR </td><td>13 </td><td>13 </td></tr><tr><td> </td><td>LysR </td><td>20 </td><td>29 </td><td> </td><td>IclR </td><td>7 </td><td>6 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>MerR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>TetR </td><td>39 </td><td>38 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>Fis </td><td>13 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>CopG </td><td>2 </td><td>1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The ROK, TetR and KorSA/GntR families in <em>S. coelicolor</em> , the LuxS family in <em>B. longum</em> and the Xre family in <em>C. acetobutylicum</em> are omitted because the definitions of the TF families in the studies differ from those in our investigation. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35871121" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Comparison of our TF assignments at the TF family level with those in primary annotations.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th></tr></thead><tbody><tr><td colspan="4"><em>M. acetivorans</em><a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a> </td><td colspan="4"><em>Lactobacillus johnsonii</em><a reveal-id="b43" data-open="b43" class="link link-ref link-reveal xref-bibr">43</a> </td></tr><tr><td> </td><td>TetR </td><td>15 </td><td>16 </td><td> </td><td>GntR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>MarR </td><td>11 </td><td>13 </td><td> </td><td>LacI </td><td>7 </td><td>7 </td></tr><tr><td> </td><td>Lrp </td><td>1 </td><td>3 </td><td> </td><td>RpiR </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>A. fulgidus</em><a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> </td><td> </td><td>ArsR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td>TetR </td><td>1 </td><td>1 </td><td> </td><td>LysR </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>MarR </td><td>1 </td><td>2 </td><td> </td><td>AraC </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>Lrp </td><td>14 </td><td>8 </td><td colspan="4"><em>Lactococcus lactis</em><a reveal-id="b44" data-open="b44" class="link link-ref link-reveal xref-bibr">44</a> </td></tr><tr><td colspan="4"><em>S. coelicolor</em><a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> </td><td> </td><td>LacI </td><td>7 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>33 </td><td>35 </td><td> </td><td>LysR </td><td>9 </td><td>6 </td></tr><tr><td> </td><td>WhiB </td><td>8 </td><td>13 </td><td> </td><td>AraC </td><td>3 </td><td>3 </td></tr><tr><td colspan="4"><em>Bifidobacterium longum</em><a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> </td><td> </td><td>GntR </td><td>4 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>22 </td><td>21 </td><td> </td><td>DeoR </td><td>4 </td><td>3 </td></tr><tr><td> </td><td>LysR </td><td>5 </td><td>5 </td><td> </td><td>MarR </td><td>11 </td><td>12 </td></tr><tr><td> </td><td>AraC </td><td>1 </td><td>1 </td><td colspan="4"><em>Fusobacterium nucleatum</em><a reveal-id="b45" data-open="b45" class="link link-ref link-reveal xref-bibr">45</a> </td></tr><tr><td> </td><td>WhiB </td><td>2 </td><td>3 </td><td> </td><td>TetR </td><td>6 </td><td>6 </td></tr><tr><td> </td><td>MerR </td><td>3 </td><td>4 </td><td> </td><td>GntR </td><td>6 </td><td>5 </td></tr><tr><td> </td><td>Fur </td><td>1 </td><td>1 </td><td> </td><td>DeoR </td><td>5 </td><td>2 </td></tr><tr><td colspan="4"><em>B. subtilis</em><a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> </td><td> </td><td>LuxR/LysR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>GntR </td><td>20 </td><td>20 </td><td> </td><td>MarR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LysR </td><td>19 </td><td>19 </td><td> </td><td>Crp </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LacI </td><td>12 </td><td>11 </td><td> </td><td>MerR </td><td>2 </td><td>1 </td></tr><tr><td> </td><td>AraC </td><td>11 </td><td>12 </td><td colspan="4"><em>Rhodopseudomonas palustris</em><a reveal-id="b46" data-open="b46" class="link link-ref link-reveal xref-bibr">46</a> </td></tr><tr><td> </td><td>Lrp </td><td>7 </td><td>6 </td><td> </td><td>AraC </td><td>23 </td><td>23 </td></tr><tr><td> </td><td>DeoR </td><td>6 </td><td>4 </td><td> </td><td>DeoR </td><td>1 </td><td>1 </td></tr><tr><td colspan="4"><em>Clostridium acetobutylicum</em><a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a> </td><td> </td><td>LuxR </td><td>11 </td><td>11 </td></tr><tr><td> </td><td>AcrR/TetR </td><td>28 </td><td>27 </td><td> </td><td>LysR </td><td>27 </td><td>27 </td></tr><tr><td> </td><td>MarR/RmrR </td><td>22 </td><td>18 </td><td> </td><td>MarR </td><td>17 </td><td>17 </td></tr><tr><td> </td><td>LysR </td><td>14 </td><td>14 </td><td> </td><td>ArsR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>LacI </td><td>9 </td><td>7 </td><td> </td><td>AsnC </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>Photorhabdus luminescens</em><a reveal-id="b42" data-open="b42" class="link link-ref link-reveal xref-bibr">42</a> </td><td> </td><td>Crp </td><td>15 </td><td>15 </td></tr><tr><td> </td><td>LuxR </td><td>32 </td><td>40 </td><td> </td><td>GntR </td><td>13 </td><td>13 </td></tr><tr><td> </td><td>LysR </td><td>20 </td><td>29 </td><td> </td><td>IclR </td><td>7 </td><td>6 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>MerR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>TetR </td><td>39 </td><td>38 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>Fis </td><td>13 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>CopG </td><td>2 </td><td>1 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th><th>Species </th><th>TF family </th><th>Primary annotation </th><th>This study </th></tr></thead><tbody><tr><td colspan="4"><em>M. acetivorans</em><a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a> </td><td colspan="4"><em>Lactobacillus johnsonii</em><a reveal-id="b43" data-open="b43" class="link link-ref link-reveal xref-bibr">43</a> </td></tr><tr><td> </td><td>TetR </td><td>15 </td><td>16 </td><td> </td><td>GntR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>MarR </td><td>11 </td><td>13 </td><td> </td><td>LacI </td><td>7 </td><td>7 </td></tr><tr><td> </td><td>Lrp </td><td>1 </td><td>3 </td><td> </td><td>RpiR </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>A. fulgidus</em><a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> </td><td> </td><td>ArsR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td>TetR </td><td>1 </td><td>1 </td><td> </td><td>LysR </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>MarR </td><td>1 </td><td>2 </td><td> </td><td>AraC </td><td>4 </td><td>4 </td></tr><tr><td> </td><td>Lrp </td><td>14 </td><td>8 </td><td colspan="4"><em>Lactococcus lactis</em><a reveal-id="b44" data-open="b44" class="link link-ref link-reveal xref-bibr">44</a> </td></tr><tr><td colspan="4"><em>S. coelicolor</em><a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> </td><td> </td><td>LacI </td><td>7 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>33 </td><td>35 </td><td> </td><td>LysR </td><td>9 </td><td>6 </td></tr><tr><td> </td><td>WhiB </td><td>8 </td><td>13 </td><td> </td><td>AraC </td><td>3 </td><td>3 </td></tr><tr><td colspan="4"><em>Bifidobacterium longum</em><a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> </td><td> </td><td>GntR </td><td>4 </td><td>5 </td></tr><tr><td> </td><td>LacI </td><td>22 </td><td>21 </td><td> </td><td>DeoR </td><td>4 </td><td>3 </td></tr><tr><td> </td><td>LysR </td><td>5 </td><td>5 </td><td> </td><td>MarR </td><td>11 </td><td>12 </td></tr><tr><td> </td><td>AraC </td><td>1 </td><td>1 </td><td colspan="4"><em>Fusobacterium nucleatum</em><a reveal-id="b45" data-open="b45" class="link link-ref link-reveal xref-bibr">45</a> </td></tr><tr><td> </td><td>WhiB </td><td>2 </td><td>3 </td><td> </td><td>TetR </td><td>6 </td><td>6 </td></tr><tr><td> </td><td>MerR </td><td>3 </td><td>4 </td><td> </td><td>GntR </td><td>6 </td><td>5 </td></tr><tr><td> </td><td>Fur </td><td>1 </td><td>1 </td><td> </td><td>DeoR </td><td>5 </td><td>2 </td></tr><tr><td colspan="4"><em>B. subtilis</em><a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> </td><td> </td><td>LuxR/LysR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>GntR </td><td>20 </td><td>20 </td><td> </td><td>MarR </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LysR </td><td>19 </td><td>19 </td><td> </td><td>Crp </td><td>2 </td><td>2 </td></tr><tr><td> </td><td>LacI </td><td>12 </td><td>11 </td><td> </td><td>MerR </td><td>2 </td><td>1 </td></tr><tr><td> </td><td>AraC </td><td>11 </td><td>12 </td><td colspan="4"><em>Rhodopseudomonas palustris</em><a reveal-id="b46" data-open="b46" class="link link-ref link-reveal xref-bibr">46</a> </td></tr><tr><td> </td><td>Lrp </td><td>7 </td><td>6 </td><td> </td><td>AraC </td><td>23 </td><td>23 </td></tr><tr><td> </td><td>DeoR </td><td>6 </td><td>4 </td><td> </td><td>DeoR </td><td>1 </td><td>1 </td></tr><tr><td colspan="4"><em>Clostridium acetobutylicum</em><a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a> </td><td> </td><td>LuxR </td><td>11 </td><td>11 </td></tr><tr><td> </td><td>AcrR/TetR </td><td>28 </td><td>27 </td><td> </td><td>LysR </td><td>27 </td><td>27 </td></tr><tr><td> </td><td>MarR/RmrR </td><td>22 </td><td>18 </td><td> </td><td>MarR </td><td>17 </td><td>17 </td></tr><tr><td> </td><td>LysR </td><td>14 </td><td>14 </td><td> </td><td>ArsR </td><td>9 </td><td>9 </td></tr><tr><td> </td><td>LacI </td><td>9 </td><td>7 </td><td> </td><td>AsnC </td><td>5 </td><td>5 </td></tr><tr><td colspan="4"><em>Photorhabdus luminescens</em><a reveal-id="b42" data-open="b42" class="link link-ref link-reveal xref-bibr">42</a> </td><td> </td><td>Crp </td><td>15 </td><td>15 </td></tr><tr><td> </td><td>LuxR </td><td>32 </td><td>40 </td><td> </td><td>GntR </td><td>13 </td><td>13 </td></tr><tr><td> </td><td>LysR </td><td>20 </td><td>29 </td><td> </td><td>IclR </td><td>7 </td><td>6 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>MerR </td><td>3 </td><td>3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>TetR </td><td>39 </td><td>38 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>Fis </td><td>13 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td>CopG </td><td>2 </td><td>1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The ROK, TetR and KorSA/GntR families in <em>S. coelicolor</em> , the LuxS family in <em>B. longum</em> and the Xre family in <em>C. acetobutylicum</em> are omitted because the definitions of the TF families in the studies differ from those in our investigation. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35871121" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=35871122 id="35871122" class="section-title" >3.3. Diversity of TFs in prokaryotes</h3>
<p> For comparison of different species, <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> plots the number of TFs per genome (the figures are given in GTOP_TF) against the total number of ORFs, which serves as an indicator of genome size in prokaryotes. Excluding archaea and a few other species, the graph shows an initial lag up to ∼1500 ORFs per genome and then a nearly linear increase. This observation is consistent with the quadratic relationship previously reported. <a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> The initial lag section contains those species that have only a few TFs with a small genome size, i.e. 1500 ORFs or less. These species are all parasitic and symbiotic organisms: seven Chlamydiae; seven Mollicutes; two <em>Rickettsias</em> species in Alphaproteobacteria; three <em>Buchneras</em> species, <em>Wigglesworthia brevipalpis</em> and <em>Candidatus Blochmannia floridanus</em> in Gammaproteobacteria; two <em>Tropherymas</em> species in Actinobacteria and <em>Nanoarchaeum equitans</em> in Nanoarchaea. The number of TFs in these organisms ranges from 2 to 11 (see GTOP_TF). In sharp contrast to its complete absence in archaea, DnaA is found in all bacterial species presumably because this protein plays the dual role of an essential factor involved in DNA replication and of a transcription regulator, <a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> although the latter function is unlikely to be essential. Besides DnaA, the HrcA repressor (a negative regulator of heat shock genes) is universally present in Chlamydiae and Mollicutes species, but is completely absent in <em>Buchnera</em> species. We think it likely that these parasitic and symbiotic organisms have shed most TFs as they retained only the minimum sets of genes for their dependent life style. </p>                        <a id="35871124" scrollto-destination="35871124"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f3.jpeg?Expires=2147483647&Signature=P43LAEBdMXHwRsvaPHqCRHTUEfIS5YrFgzfcwfQ~-ZfQ-4gezndODuD28I9m19p0yfcCkW~rPry0Wqr9wBLtHNDrf6TfxTz05WeqmfC9lX2Nb4qDA4qPwV-nE-SifehJi6i5UzfXTuiSYHvaJbfygzKVE~J-ON5FOB-RtG7jgasXsN4UiGlIRt9PJG0k~J3f1W9f-Z~rOpny5s0IHez85tzi3zomZ8qfTPFCOjWcvE5gxn4NJ4LIi0xGngqL2SYOFzAAE7~TYAJePstvtMtQDNRP5l8jsmkan-ZEEFYMQNNvJ09Q6L-yROhY4HD1skImW0q5we32fKwDgsU9legskQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the number of TFs and genome size. The total number of high quality TFs per genome is plotted against genome size (an indicator of the total number of ORFs)." path-from-xml="dsi016f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871124/dsi016f3.jpeg" path-from-xml="dsi016f3.jpeg" target="_blank">View large</a><a section="35871124" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f3.jpeg?Expires=2147483647&Signature=qysVN30sqqy734qxRQSngquTApW4YwcVle-y5n0MublmVg9T60Hh~FsiJAzCnqcDj3JT3e8zRFCa4xJZcyBQZqStD5E3r25XGG0HkUiVTVcDv2HlIaSuUOpw5DiHcBptW4LGiJ1O5yEIRbYR5sSZyQqjs-k1frb2UrK3lWlnC-LiDdNevlAg0Mmg-3L4Qm-Y8O9Y5aN-MaSbCprh~hImNeYJPX0VeAFh3GafrKq5v2VZvhSbeGVuWxuaqn2YyB5MCzwxmR2l4gNGl9OMuSBf20D9OTob0BH1v0RMn~biPlQA~xgr~6CCM8rVrZNN4p4wmtP7K0GWUHy3LfOzM-KV5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871124&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the number of TFs and genome size. The total number of high quality TFs per genome is plotted against genome size (an indicator of the total number of ORFs).</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f3.jpeg?Expires=2147483647&Signature=P43LAEBdMXHwRsvaPHqCRHTUEfIS5YrFgzfcwfQ~-ZfQ-4gezndODuD28I9m19p0yfcCkW~rPry0Wqr9wBLtHNDrf6TfxTz05WeqmfC9lX2Nb4qDA4qPwV-nE-SifehJi6i5UzfXTuiSYHvaJbfygzKVE~J-ON5FOB-RtG7jgasXsN4UiGlIRt9PJG0k~J3f1W9f-Z~rOpny5s0IHez85tzi3zomZ8qfTPFCOjWcvE5gxn4NJ4LIi0xGngqL2SYOFzAAE7~TYAJePstvtMtQDNRP5l8jsmkan-ZEEFYMQNNvJ09Q6L-yROhY4HD1skImW0q5we32fKwDgsU9legskQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the number of TFs and genome size. The total number of high quality TFs per genome is plotted against genome size (an indicator of the total number of ORFs)." path-from-xml="dsi016f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871124/dsi016f3.jpeg" path-from-xml="dsi016f3.jpeg" target="_blank">View large</a><a section="35871124" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f3.jpeg?Expires=2147483647&Signature=qysVN30sqqy734qxRQSngquTApW4YwcVle-y5n0MublmVg9T60Hh~FsiJAzCnqcDj3JT3e8zRFCa4xJZcyBQZqStD5E3r25XGG0HkUiVTVcDv2HlIaSuUOpw5DiHcBptW4LGiJ1O5yEIRbYR5sSZyQqjs-k1frb2UrK3lWlnC-LiDdNevlAg0Mmg-3L4Qm-Y8O9Y5aN-MaSbCprh~hImNeYJPX0VeAFh3GafrKq5v2VZvhSbeGVuWxuaqn2YyB5MCzwxmR2l4gNGl9OMuSBf20D9OTob0BH1v0RMn~biPlQA~xgr~6CCM8rVrZNN4p4wmtP7K0GWUHy3LfOzM-KV5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871124&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the number of TFs and genome size. The total number of high quality TFs per genome is plotted against genome size (an indicator of the total number of ORFs).</p></div></div><p> The nearly linear section (open symbols and small gray dots in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> ) consists of the majority of bacteria. The positive intersection with the <em>x</em> -axis corresponding to the lag shows that TFs are needed in proportion to the genome size above a certain number of ORFs (∼1500). The TFs may be considered as factors that regulate complex cell functions beyond the minimum level. Though Betaproteobacteria (8 species) and Lactobacillales (13 species) belong to this group, they are shifted upward (open symbols) from the rest of the group. Four species with large genomes including two Actinobacteria ( <em>Streptomyces avermitilis</em> and <em>Streptomyces coelicolor</em> ) and two Alphaproteobacteria ( <em>Bradyrhizobium japonicum</em> and <em>Mesorhizobium loti</em> ) have 453–639 TFs (see also GTOP_TF), although the number of TF families they have is limited to 33–35, which is less than those of many Gammaproteobacteria including <em>E. coli</em> . <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Figure 4</a> presents this point more clearly; the number of TF families levels off with large numbers of TFs. In other words, in species with &gt;300 TFs the number of TF families remains nearly constant (35–40), suggesting the divergence of similar TFs by gene duplication. It is also noticed that Gammaproteobacteria and Bacillales (open symbols in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a> ) have relatively more TF families than others. </p>                        <a id="35871126" scrollto-destination="35871126"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f4.jpeg?Expires=2147483647&Signature=3BwLcvzafYQj7fVPnz3qkk3KO~yZV7FYGU5fwv1IF7omjD3izorCR2NJKMnSU5S9qw1lCJStnt49yeKJqBfZ2PgK0rSSNQzd3el9Z4NU1-UO8uN39Xr7JcVXpfKVoAAgR4EfL1QWp38wXEXN9U5V5AAVv2q19mv1N3fOhotaimpItKorRKEFL~8FKoBonD9peMECNM8Rczp9MFqsCaAAnrZLPzQRWrDyW4tyAlsygAwuYTJ75rUrmhkcJt61TBi3JNBUrwmCT7gY3NU3P0PMHcDM7z4NRGXFxUOcv38-t44~OCetydGgfPm7f9jf3SRMjZBWZnVhdJTUvXPEwc6ufA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the number of TF families and the number of TFs. The number of TF families of each species is plotted against the total number of high quality TFs in the genome." path-from-xml="dsi016f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871126/dsi016f4.jpeg" path-from-xml="dsi016f4.jpeg" target="_blank">View large</a><a section="35871126" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f4.jpeg?Expires=2147483647&Signature=BDOF0u7hfUXU6y9JC8h6UMDiCIuKTiGZI9qLL044jyXgs9zoqwscHEyDqSxqz~miz0zzTIdM56La4ASDbRt5epicIa5bYOKM1s1gd6Mna6SYiSb9kqoGxleE851IOyqSSMZtk7rcRh-YO0GtqT~oo08j3aDgenZJggwQCLm~BqUQrhR8A7FSMBRkMLgjhetA7CuSWVTj3sXeUoRRDiIia2aysgN1xKjhD0cK5Awo41~EdZMCUIcyG6Y-bQ7YBErBlGDpwsqMEHFGnLDyL7R68bUQ2WigK3Eq9H00ApqIxYtJPp2uwcv5800teUoJFad-mo6ne0cdXW-mnLzAn~jRNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871126&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the number of TF families and the number of TFs. The number of TF families of each species is plotted against the total number of high quality TFs in the genome.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f4.jpeg?Expires=2147483647&Signature=3BwLcvzafYQj7fVPnz3qkk3KO~yZV7FYGU5fwv1IF7omjD3izorCR2NJKMnSU5S9qw1lCJStnt49yeKJqBfZ2PgK0rSSNQzd3el9Z4NU1-UO8uN39Xr7JcVXpfKVoAAgR4EfL1QWp38wXEXN9U5V5AAVv2q19mv1N3fOhotaimpItKorRKEFL~8FKoBonD9peMECNM8Rczp9MFqsCaAAnrZLPzQRWrDyW4tyAlsygAwuYTJ75rUrmhkcJt61TBi3JNBUrwmCT7gY3NU3P0PMHcDM7z4NRGXFxUOcv38-t44~OCetydGgfPm7f9jf3SRMjZBWZnVhdJTUvXPEwc6ufA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the number of TF families and the number of TFs. The number of TF families of each species is plotted against the total number of high quality TFs in the genome." path-from-xml="dsi016f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871126/dsi016f4.jpeg" path-from-xml="dsi016f4.jpeg" target="_blank">View large</a><a section="35871126" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f4.jpeg?Expires=2147483647&Signature=BDOF0u7hfUXU6y9JC8h6UMDiCIuKTiGZI9qLL044jyXgs9zoqwscHEyDqSxqz~miz0zzTIdM56La4ASDbRt5epicIa5bYOKM1s1gd6Mna6SYiSb9kqoGxleE851IOyqSSMZtk7rcRh-YO0GtqT~oo08j3aDgenZJggwQCLm~BqUQrhR8A7FSMBRkMLgjhetA7CuSWVTj3sXeUoRRDiIia2aysgN1xKjhD0cK5Awo41~EdZMCUIcyG6Y-bQ7YBErBlGDpwsqMEHFGnLDyL7R68bUQ2WigK3Eq9H00ApqIxYtJPp2uwcv5800teUoJFad-mo6ne0cdXW-mnLzAn~jRNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871126&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the number of TF families and the number of TFs. The number of TF families of each species is plotted against the total number of high quality TFs in the genome.</p></div></div><p> On the other hand, the group lying close to the abscissa (filled symbols in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> ) has no clear lag phase, but instead shows a roughly linear dependence over the entire range with a slope less steep than that in the nearly linear section described above. This group consists of Cyanobacteria (8 species), Spirochaetes (4), Planctomycetes (1) and all 18 archaeal species. Species of this group have characteristically small numbers of TFs. The most remarkable among them is the <em>Pirellula sp</em> ., the only entirely sequenced species in Planctomycetes, <a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> as it has only 91 TFs despite the large genome size (7325 ORFs). The number should be compared with 500 TFs or more expected in bacteria of similar genome sizes. A key to account for this discrepancy lies in the fact that <em>Pirellula sp</em> . has as many as 50 σ factors besides TFs. <a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> Scarcity of TFs is also notable in Cyanobacteria, especially in the <em>Anabaena sp</em> . ( <em>Nostoc sp.</em> ), which has only 133 TFs in 6132 ORFs. The scantiness of TFs in Cyanobacteria can be explained by the fact that Cyanobacteria have a highly developed two-component signal transduction system. <a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> Throughout all the archaeal species examined, the numbers of TFs and TF families are small ( <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figs 3</a> and <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">4</a> ). We consider this to be the most significant finding in the present study and discuss it in the following section. </p>                            <h2 scrollto-destination=35871128 id="35871128" class="section-title" >4. Discussion</h2>
<p>In contrast to previously publicized methods, the newly developed method uses a combination of Pfam or SCOP domains assigned to the DBDs and the non-DBDs of proteins to select TFs. Compared with the single use of the DBDs, this combination approach drastically reduces over-assigned cases. To exclude over-assigned cases in solitary TF families, we introduced an additional requirement, namely alignment to an experimentally verified TF of comparable length. We anticipate a very low number of mistakenly detected cases in the set of TFs selected by our method.</p><p> The sensitivity and therefore the extent of under-assignment of the TF detection method developed in the present study depends mainly on the sensitivity of the homology-search tools utilized, i.e. BLAST, PSI-BLAST and HMMER. In the GTOP database, the average residue-wise fraction of proteins aligned to known a 3D structure (PDB) by PSI-BLAST in the genomes of prokaryotic species is 45.5%, but this fraction is increasing with time as the number of PDB entries increases. The corresponding residue-wise fraction aligned to Pfam is even higher (52.0% if HMMER is used for alignment) presumably reflecting the fact that the Pfam database is constructed independently of the known protein structures. Although the fraction of known domains is considerable and increasing, a significant number of protein domains in each genome remain unknown. As we cover all the experimentally verified TFs in prokaryotes by our detection rules, cases missed by our method arise only from the failure to detect TFs caused by faulty performance of alignment programs or the lack of known homologs. The failure of all the alignment programs to detect homology with known proteins is considered to be infrequent, thanks to the high sensitivity of PSI-BLAST and HMMER. We expect the alignment programs to work almost equally reliably on proteins in archaea as those in bacteria despite the phylogenic remoteness and generally sparser protein annotation in archaea than in bacteria, particularly in <em>E. coli</em> and <em>B. subtilis</em> . In fact, the average residue-wise coverage of proteins by PDB with the use of PSI-BLAST is 42.3% in archaea and is comparable to the corresponding figure, 46.1%, in bacteria. Consequently the only possible major source of under-assignment is the failure to detect DBDs because they belong to unknown families. However, as bacterial TFs are well studied it is unlikely to find many new DBD families in bacteria. Furthermore, considering the fact that more than half of proteins in archaea have SCOP and Pfam domain assignments and that almost no archaea-specific TF families have been found in structurally aligned proteins, we think it improbable to discover many archaea-specific TF families in the future with more PDB data. Comparison of TFs detected by our method with previously reported TFs ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> ) supports the conclusion of theoretical considerations. </p><p> As shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figs 3</a> and <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">4</a> , the numbers of TFs and TF families are lower in archaea than in the majority of bacteria. For example, <em>M. acetivorans</em> , whose genome is larger than that of <em>E. coli</em> , has only 80 high quality TFs classified into 13 TF families (see GTOP_TF). To compare bacteria and archaea more systematically, the phylogenetic pattern for each TF family across major taxa of prokaryotes is presented in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> , where only those taxa having four or more species of known genome are included. The following observations remain valid even if taxa with less than four species are included (Supplemental Table S1 and GTOP_TF). Surprisingly there was only one minor TF family specific to archaea found in this study, Archaeal HTH-10 (the third row, <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> ). We note that there are no TF families commonly shared by archaea and eukaryotes besides those shown in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> ; although all protein sequences of archaea were searched in the GTOP database against all kinds of Pfam domains including eukaryotic DBDs (e.g. zinc-finger, homeobox, and leucine-zipper), no eukaryotic DBDs were detected in archaea. As depicted in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> , archaea and bacteria share 18 TF families including 10 out of a total of 20 major ones (rows 4–21, in which the major TF families are yellow-tinted), including the nearly ubiquitous LysR, TetR/AcrR and GntR families. Since the same domain organization is kept throughout the prokaryotes, as shown with the AsnC/Lrp family, <a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> , <a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> these TF families must have descended from the common ancestor of bacteria and archaea. <a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Notably there are no essential differences between Crenarchaeota and Euryarchaeota (columns 2 and 3), although some TF families (e.g. LysR, MerR and BirA) are absent in Crenarchaeota. The remaining 10 major as well as 23 minor families in the list are unique to bacteria and are also absent in eukaryotes according to our preliminary analyses. Thus, they must have evolved in bacteria after branching off from archaea. Interestingly, well-known TFs such as OmpR, AraC, LacI and Fis fall in this group. At the same time, we notice several taxon-specific TF families: ROS/MUCR (specific to Alphaproteobacteria), AfsR/DnrI/RedD (Actinobacteria) and ComK (Bacillales) as well as MtlR, MetJ and Crl (Gammaproteobacteria), as previously reported. <a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> It should be noted, however, that these taxon/species-specific TF families are all minor families, and are more or less biased to the present knowledge of TFs with experimental evidence. It is relevant to note that Gammaproteobacteria tend to have more TF families detected than other bacteria with comparable numbers of TFs per genome ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a> ), which generally reflect the genome size ( <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> ). We consider it probable that this difference reflects the better-studied nature of Gammaproteobacteria, especially <em>E. coli</em> , than other bacteria. Minor TF families unique to individual taxa or species will increase in the future as experimental evidence accumulates, especially in bacteria other than Gammaproteobacteria. However, the distribution of major TF families is unlikely to be altered, since only minor TF families in bacteria and archaea are possibly not covered by the present selection rules as reasoned in the preceding paragraph. Therefore we consider it probable that approximately half of the major TF families exist exclusively in bacteria, while the rest are shared by the two kingdoms. </p>                        <a id="35871132" scrollto-destination="35871132"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f5.jpeg?Expires=2147483647&Signature=DDV6tkEZsVmNPF074RExRNogst2pvBrZ2fuHxIPTqySS5T0LsxjPyrqZy3Eb-6GVIEPBFknsTzqyanTwJRe~jvRIaMDUbGhkdqBnpu-6w0gyCSFAe81Hereun~xYOmhILOjr~nVftQHmiH3Q3Bn0HaLdaJe8LrPo1YUVEFvJIjFS5AbEoo8gKVVYj2ZVzY1FbR9i6qUcdUstGc7A2YzxTUvRuYVmGCyceR70MTzyF9brw3RTvbYg36oAD1iDqn9MoWRuY5ddj187eAgSfFcK03VWcHQMub5dNV8mCxX2jb1szTvPj5RMfOOL-uOHU2Zx5eJrzenlXmf0rcKVcyR~eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Phylogenetic pattern of high confidence TF families. Presence or absence of each TF family in major phyla is shown. The abbreviation of each phylum with the number of species after removing those of obligatory and semi-obligatory parasites in brackets is Crenarchaeota, Cr (4); Euryarchaeota, Eu (13); Actinobacteria, Ac (10); Cyanobacteria, Cy (8); Bacillales, Ba (12); Clostridia, Cl (4); Lactobacillales, La (13); Alphaproteobacteria, αP (9); Betaproteobacteria, βP (8); Gammaproteobacteria, γP (29); Epsilonproteobacteria, ɛP (5): Spirochaetes, Sp (4); Mollicutes, Mo* (7); and Chlamydiae, Ch* (7). Two major phyla of archaea, 12 major phyla of bacteria as well as two bacterial phyla consisting of obligatory and semi-obligatory parasites are placed from left to right. The TF family unique to archaea is placed at the top, followed by those shared by archaea and bacteria, and lastly by those unique to bacteria. The cells of major TF families, i.e. the top 20 families in Table 1 , are colored yellow. A box colored black or red shows that all the species of the phylum excluding those of obligatory and semi-obligatory parasites have TFs of the corresponding family. A box is tinted grey or pink if at least one species, but not as many species as to be painted black or red, belonging to the phylum has TFs of the family. In archaea, red and pink boxes are used, while in bacteria, black and grey boxes are utilized. A blank box signifies the complete absence of the TF family in the phylum. " path-from-xml="dsi016f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871132/dsi016f5.jpeg" path-from-xml="dsi016f5.jpeg" target="_blank">View large</a><a section="35871132" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f5.jpeg?Expires=2147483647&Signature=sfR9o-JMl4j0-VUrQv1rursY72SUxGp6HnZMRbQk1Tmk-TNfdnJ-Uoh9yaN-gMDk8AzDuAvRdKBxQGBbIQWxR89gy-OaBZSORZSDHhTkS1oXmRkfuyWgP8SjoI2woXR2EDbDxqqNEGyPVTpo6km6jTa1vHmkgu-Ky7Nm10Mn7NDPu6xMbeGPwZKxu~4AwhM6VLf4CeTsDcq5DgiPro-oG-kRaS3aNlYuSZpM0Fbieo12Uv4zos9~2S1oO49ofqoOazsoJACq-WNxIpheRg4AvxgHRZeN8RC3EO5Q43~WIARmzyl-6lv4mocXOwZWPRGzguzjHcNEu-3xZ9hcMkWa5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871132&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Phylogenetic pattern of high confidence TF families. Presence or absence of each TF family in major phyla is shown. The abbreviation of each phylum with the number of species after removing those of obligatory and semi-obligatory parasites in brackets is Crenarchaeota, Cr (4); Euryarchaeota, Eu (13); Actinobacteria, Ac (10); Cyanobacteria, Cy (8); Bacillales, Ba (12); Clostridia, Cl (4); Lactobacillales, La (13); Alphaproteobacteria, αP (9); Betaproteobacteria, βP (8); Gammaproteobacteria, γP (29); Epsilonproteobacteria, ɛP (5): Spirochaetes, Sp (4); Mollicutes, Mo* (7); and Chlamydiae, Ch* (7). Two major phyla of archaea, 12 major phyla of bacteria as well as two bacterial phyla consisting of obligatory and semi-obligatory parasites are placed from left to right. The TF family unique to archaea is placed at the top, followed by those shared by archaea and bacteria, and lastly by those unique to bacteria. The cells of major TF families, i.e. the top 20 families in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> , are colored yellow. A box colored black or red shows that all the species of the phylum excluding those of obligatory and semi-obligatory parasites have TFs of the corresponding family. A box is tinted grey or pink if at least one species, but not as many species as to be painted black or red, belonging to the phylum has TFs of the family. In archaea, red and pink boxes are used, while in bacteria, black and grey boxes are utilized. A blank box signifies the complete absence of the TF family in the phylum. </p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/m_dsi016f5.jpeg?Expires=2147483647&Signature=DDV6tkEZsVmNPF074RExRNogst2pvBrZ2fuHxIPTqySS5T0LsxjPyrqZy3Eb-6GVIEPBFknsTzqyanTwJRe~jvRIaMDUbGhkdqBnpu-6w0gyCSFAe81Hereun~xYOmhILOjr~nVftQHmiH3Q3Bn0HaLdaJe8LrPo1YUVEFvJIjFS5AbEoo8gKVVYj2ZVzY1FbR9i6qUcdUstGc7A2YzxTUvRuYVmGCyceR70MTzyF9brw3RTvbYg36oAD1iDqn9MoWRuY5ddj187eAgSfFcK03VWcHQMub5dNV8mCxX2jb1szTvPj5RMfOOL-uOHU2Zx5eJrzenlXmf0rcKVcyR~eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Phylogenetic pattern of high confidence TF families. Presence or absence of each TF family in major phyla is shown. The abbreviation of each phylum with the number of species after removing those of obligatory and semi-obligatory parasites in brackets is Crenarchaeota, Cr (4); Euryarchaeota, Eu (13); Actinobacteria, Ac (10); Cyanobacteria, Cy (8); Bacillales, Ba (12); Clostridia, Cl (4); Lactobacillales, La (13); Alphaproteobacteria, αP (9); Betaproteobacteria, βP (8); Gammaproteobacteria, γP (29); Epsilonproteobacteria, ɛP (5): Spirochaetes, Sp (4); Mollicutes, Mo* (7); and Chlamydiae, Ch* (7). Two major phyla of archaea, 12 major phyla of bacteria as well as two bacterial phyla consisting of obligatory and semi-obligatory parasites are placed from left to right. The TF family unique to archaea is placed at the top, followed by those shared by archaea and bacteria, and lastly by those unique to bacteria. The cells of major TF families, i.e. the top 20 families in Table 1 , are colored yellow. A box colored black or red shows that all the species of the phylum excluding those of obligatory and semi-obligatory parasites have TFs of the corresponding family. A box is tinted grey or pink if at least one species, but not as many species as to be painted black or red, belonging to the phylum has TFs of the family. In archaea, red and pink boxes are used, while in bacteria, black and grey boxes are utilized. A blank box signifies the complete absence of the TF family in the phylum. " path-from-xml="dsi016f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35871132/dsi016f5.jpeg" path-from-xml="dsi016f5.jpeg" target="_blank">View large</a><a section="35871132" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi016/3/dsi016f5.jpeg?Expires=2147483647&Signature=sfR9o-JMl4j0-VUrQv1rursY72SUxGp6HnZMRbQk1Tmk-TNfdnJ-Uoh9yaN-gMDk8AzDuAvRdKBxQGBbIQWxR89gy-OaBZSORZSDHhTkS1oXmRkfuyWgP8SjoI2woXR2EDbDxqqNEGyPVTpo6km6jTa1vHmkgu-Ky7Nm10Mn7NDPu6xMbeGPwZKxu~4AwhM6VLf4CeTsDcq5DgiPro-oG-kRaS3aNlYuSZpM0Fbieo12Uv4zos9~2S1oO49ofqoOazsoJACq-WNxIpheRg4AvxgHRZeN8RC3EO5Q43~WIARmzyl-6lv4mocXOwZWPRGzguzjHcNEu-3xZ9hcMkWa5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35871132&ar=350906&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi016f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Phylogenetic pattern of high confidence TF families. Presence or absence of each TF family in major phyla is shown. The abbreviation of each phylum with the number of species after removing those of obligatory and semi-obligatory parasites in brackets is Crenarchaeota, Cr (4); Euryarchaeota, Eu (13); Actinobacteria, Ac (10); Cyanobacteria, Cy (8); Bacillales, Ba (12); Clostridia, Cl (4); Lactobacillales, La (13); Alphaproteobacteria, αP (9); Betaproteobacteria, βP (8); Gammaproteobacteria, γP (29); Epsilonproteobacteria, ɛP (5): Spirochaetes, Sp (4); Mollicutes, Mo* (7); and Chlamydiae, Ch* (7). Two major phyla of archaea, 12 major phyla of bacteria as well as two bacterial phyla consisting of obligatory and semi-obligatory parasites are placed from left to right. The TF family unique to archaea is placed at the top, followed by those shared by archaea and bacteria, and lastly by those unique to bacteria. The cells of major TF families, i.e. the top 20 families in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> , are colored yellow. A box colored black or red shows that all the species of the phylum excluding those of obligatory and semi-obligatory parasites have TFs of the corresponding family. A box is tinted grey or pink if at least one species, but not as many species as to be painted black or red, belonging to the phylum has TFs of the family. In archaea, red and pink boxes are used, while in bacteria, black and grey boxes are utilized. A blank box signifies the complete absence of the TF family in the phylum. </p></div></div><p><a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3</a> indicates that Cyanobacteria and Spirochaetes belong to the same group as archaea. <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5</a> shows that these bacteria (Cy and Sp) indeed have a limited number of TF families, but have a different repertoire (e.g. OmpR) from that of archaea. The life of obligatory or semi-obligatory parasites, such as Chlamydiae, Mycoplasma and <em>Buchnera</em> , depends heavily on host cells for nutrition, resulting in deletion of many genes from the genomes. <a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> Hence, it is natural to think that many kinds of TFs have also been deleted in these species. A recent study <a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> revealed that <em>Buchnera</em> has lost all genes but one (metR) that regulate transcriptions of genes involved in syntheses of various amino acids. The idea is corroborated by the TF family distribution of Mollicutes, which include Mycoplasma and Chlamydiae (Mo <sup>*</sup> and Ch <sup>*</sup> in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> ). The fact that they have only a few TFs implies that expression of all of their genes is mostly under no specific regulation just as in the case of housekeeping genes. On the other hand, autotrophic species in Gammaproteobacteria, Bacillales and other phyla tend to have many kinds of TFs to control cellular processes in response to environmental changes. Assuming that the variety in the TF family a given species possesses is indicative of the complexity of the species, Gammaproteobacteria are the most diversified form of prokaryotes ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Figs 4</a> and <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">5</a> ), although some of the minor TF families must be biased to good annotation of <em>E. coli</em> as described above. Thus, by the same measure, we can say that archaea are less diverse than bacteria <a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> ( <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> ), since it is unlikely that our method failed to detect any archaea-specific major TFs families, as argued above. </p><p> Finally, we should point out another fact revealed in the present study. That is, all but one DBDs of known structure fall in the all-α type or class a in the SCOP classification ( <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> ). The only exception to this rule is the DBD of the BolA/YrbA family, which belongs to the α + β type (class <span class="monospace">d</span> ) in SCOP. Remarkably TFs having DBDs of the all-α type, particularly HTH proteins, <a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> are predominant throughout prokaryotes. This implies two things: one is the phylogenetic continuity of bacteria and archaea, and the other is a presumable dichotomy between prokaryotes and eukaryotes because TFs in eukaryotes are known to have various kinds of DBDs including those of the all-β type (class <span class="monospace">b</span> ) as well as an abundance of zinc-finger types, <a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> , <a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> in addition to the all-α type such as the homeobox domain. Comparison of TFs between prokaryotes and eukaryotes will be the subject of our next report. </p><p>The authors thank Dr Nobuyuki Fujita for the valuable advice. This work was supported by a grant-in-aid for the National Project of Protein 3000 from the MEXT, Japan.</p>                            <h2 scrollto-destination=35871137 id="35871137" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishihama</div> <div class="given-names">A.</div></div>. </div><div class="article-title"> Functional modulation of <em>Escherichia coli</em> RNA polymerase </div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">499</div>-<div class="lpage">518</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20modulation%20of%20Escherichia%20coli%20RNA%20polymerase&amp;author=A.%20Ishihama&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=54&amp;pages=499-518" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.54.1.499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.54.1.499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.54.1.499"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11018136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ouhammouch</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transcriptional regulation in archaea</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">133</div>-<div class="lpage">138</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulation%20in%20archaea&amp;author=M.%20Ouhammouch&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=14&amp;pages=133-138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2004.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2004.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2004.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15196459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Riechmann</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Heard</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title"><em>Arabidopsis</em> transcription factors: genome-wide comparative analysis among eukaryotes </div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">290</div> (pg. <div class="fpage">2105</div>-<div class="lpage">2110</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20transcription%20factors%3A%20genome-wide%20comparative%20analysis%20among%20eukaryotes&amp;author=J.%20L.%20Riechmann&amp;author=J.%20Heard&amp;author=G.%20Martin&amp;publication_year=2000&amp;journal=Science&amp;volume=290&amp;pages=2105-2110" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.290.5499.2105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.290.5499.2105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.290.5499.2105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11118137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coulson</div> <div class="given-names">R. M.</div></div>,  <div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">The phylogenetic diversity of eukaryotic transcription</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">653</div>-<div class="lpage">660</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20phylogenetic%20diversity%20of%20eukaryotic%20transcription&amp;author=R.%20M.%20Coulson&amp;author=C.%20A.%20Ouzounis&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=653-660" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12527774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matys</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Fricke</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Geffers</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">TRANSFAC: transcriptional regulation, from patterns to profiles</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">374</div>-<div class="lpage">378</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TRANSFAC%3A%20transcriptional%20regulation%2C%20from%20patterns%20to%20profiles&amp;author=V.%20Matys&amp;author=E.%20Fricke&amp;author=R.%20Geffers&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=374-378" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perez-Rueda</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div>. </div><div class="article-title"> The repertoire of DNA-binding transcriptional regulators in <em>Escherichia coli</em> K-12 </div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1838</div>-<div class="lpage">1847</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20repertoire%20of%20DNA-binding%20transcriptional%20regulators%20in%20Escherichia%20coli%20K-12&amp;author=E.%20Perez-Rueda&amp;author=J.%20Collado-Vides&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=1838-1847" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.8.1838" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.8.1838" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.8.1838"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10734204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>. </div><div class="article-title">DNA-binding proteins and evolution of transcription regulation in the archaea</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4658</div>-<div class="lpage">4670</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-binding%20proteins%20and%20evolution%20of%20transcription%20regulation%20in%20the%20archaea&amp;author=L.%20Aravind&amp;author=E.%20V.%20Koonin&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4658-4670" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4658" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4658" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4658"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kyrpides</div> <div class="given-names">N. C.</div></div>,  <div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">Transcription in archaea</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">8545</div>-<div class="lpage">8550</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20in%20archaea&amp;author=N.%20C.%20Kyrpides&amp;author=C.%20A.%20Ouzounis&amp;publication_year=1999&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=8545-8550" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.15.8545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.15.8545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.15.8545"> </span></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cases</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">de Lorenzo</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">Transcription regulation and environmental adaptation in bacteria</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">248</div>-<div class="lpage">253</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20regulation%20and%20environmental%20adaptation%20in%20bacteria&amp;author=I.%20Cases&amp;author=V.%20de%20Lorenzo&amp;author=C.%20A.%20Ouzounis&amp;publication_year=2003&amp;journal=Trends%20Microbiol.&amp;volume=11&amp;pages=248-253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(03)00103-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(03)00103-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(03)00103-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12823939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ranea</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Buchan</div> <div class="given-names">D. W.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Orengo</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">Evolution of protein superfamilies and bacterial genome size</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">336</div> (pg. <div class="fpage">871</div>-<div class="lpage">887</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20protein%20superfamilies%20and%20bacterial%20genome%20size&amp;author=J.%20A.%20Ranea&amp;author=D.%20W.%20Buchan&amp;author=J.%20M.%20Thornton&amp;author=C.%20A.%20Orengo&amp;publication_year=2004&amp;journal=J.%20Mol.%20Biol.&amp;volume=336&amp;pages=871-887" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2003.12.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2003.12.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2003.12.044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15095866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Orengo</div> <div class="given-names">C. A.</div></div>,  <div class="name"><div class="surname">Michie</div> <div class="given-names">A. D.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">D. T.</div></div>,  <div class="name"><div class="surname">Swindells</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">CATH—a hierarchic classification of protein domain structures</div>, <div class="source">Structure</div> , <div class="year">1997</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1093</div>-<div class="lpage">1108</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CATH%E2%80%94a%20hierarchic%20classification%20of%20protein%20domain%20structures&amp;author=C.%20A.%20Orengo&amp;author=A.%20D.%20Michie&amp;author=S.%20Jones&amp;author=D.%20T.%20Jones&amp;author=M.%20B.%20Swindells&amp;author=J.%20M.%20Thornton&amp;publication_year=1997&amp;journal=Structure&amp;volume=5&amp;pages=1093-1108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0969-2126(97)00260-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0969-2126(97)00260-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0969-2126(97)00260-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9309224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martinez-Bueno</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Molina-Henares</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Pareja</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ramos</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Tobes</div> <div class="given-names">R.</div></div>. </div><div class="article-title">BacTregulators: a database of transcriptional regulators in bacteria and archaea</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2787</div>-<div class="lpage">2791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BacTregulators%3A%20a%20database%20of%20transcriptional%20regulators%20in%20bacteria%20and%20archaea&amp;author=M.%20Martinez-Bueno&amp;author=A.%20J.%20Molina-Henares&amp;author=E.%20Pareja&amp;author=J.%20L.%20Ramos&amp;author=R.%20Tobes&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2787-2791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15166024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Babu</div> <div class="given-names">M. M.</div></div>,  <div class="name"><div class="surname">Teichmann</div> <div class="given-names">S. A.</div></div>. </div><div class="article-title"> Evolution of transcription factors and the gene regulatory network in <em>Escherichia coli</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">1234</div>-<div class="lpage">1244</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20transcription%20factors%20and%20the%20gene%20regulatory%20network%20in%20Escherichia%20coli&amp;author=M.%20M.%20Babu&amp;author=S.%20A.%20Teichmann&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=1234-1244" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murzin</div> <div class="given-names">A. G.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">S. E.</div></div>,  <div class="name"><div class="surname">Hubbard</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Chothia</div> <div class="given-names">C.</div></div>. </div><div class="article-title">SCOP: a structural classification of proteins database for the investigation of sequences and structures</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1995</div>, vol. <div class="volume">247</div> (pg. <div class="fpage">536</div>-<div class="lpage">540</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SCOP%3A%20a%20structural%20classification%20of%20proteins%20database%20for%20the%20investigation%20of%20sequences%20and%20structures&amp;author=A.%20G.%20Murzin&amp;author=S.%20E.%20Brenner&amp;author=T.%20Hubbard&amp;author=C.%20Chothia&amp;publication_year=1995&amp;journal=J.%20Mol.%20Biol.&amp;volume=247&amp;pages=536-540" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7723011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawabata</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fukuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Homma</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">GTOP: a database of protein structures predicted from genome sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">294</div>-<div class="lpage">298</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GTOP%3A%20a%20database%20of%20protein%20structures%20predicted%20from%20genome%20sequences&amp;author=T.%20Kawabata&amp;author=S.%20Fukuchi&amp;author=K.%20Homma&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=294-298" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berman</div> <div class="given-names">H. M.</div></div>,  <div class="name"><div class="surname">Westbrook</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">The Protein Data Bank</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">235</div>-<div class="lpage">242</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Protein%20Data%20Bank&amp;author=H.%20M.%20Berman&amp;author=J.%20Westbrook&amp;author=Z.%20Feng&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=235-242" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.235" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.235" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.235"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">C. H.</div></div>, et al. </div><div class="article-title">The Universal Protein Resource (UniProt)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D154</div>-<div class="lpage">D159</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Universal%20Protein%20Resource%20(UniProt)&amp;author=A.%20Bairoch&amp;author=R.%20Apweiler&amp;author=C.%20H.%20Wu&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D154-D159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">Profile hidden Markov models</div>, <div class="source">Bioinformatics</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">755</div>-<div class="lpage">763</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profile%20hidden%20Markov%20models&amp;author=S.%20R.%20Eddy&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=755-763" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.755" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.9.755" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.9.755"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Coin</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D138</div>-<div class="lpage">D141</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=A.%20Bateman&amp;author=L.%20Coin&amp;author=R.%20Durbin&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D138-D141" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reeve</div> <div class="given-names">J. N.</div></div>. </div><div class="article-title">Archaeal chromatin and transcription</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">587</div>-<div class="lpage">598</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Archaeal%20chromatin%20and%20transcription&amp;author=J.%20N.%20Reeve&amp;publication_year=2003&amp;journal=Mol.%20Microbiol.&amp;volume=48&amp;pages=587-598" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03439.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03439.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03439.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12694606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaplan</div> <div class="given-names">D. L.</div></div>,  <div class="name"><div class="surname">O'Donnell</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Rho factor: transcription termination in four steps</div>, <div class="source">Curr. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">R714</div>-<div class="lpage">R716</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rho%20factor%3A%20transcription%20termination%20in%20four%20steps&amp;author=D.%20L.%20Kaplan&amp;author=M.%20O'Donnell&amp;publication_year=2003&amp;journal=Curr.%20Biol.&amp;volume=13&amp;pages=R714-R716" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2003.08.047" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2003.08.047" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2003.08.047"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13678611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Phadtare</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Recent developments in bacterial cold-shock response</div>, <div class="source">Curr. Issues Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">125</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20developments%20in%20bacterial%20cold-shock%20response&amp;author=S.%20Phadtare&amp;publication_year=2004&amp;journal=Curr.%20Issues%20Mol.%20Biol.&amp;volume=6&amp;pages=125-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15119823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Itou</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">I.</div></div>. </div><div class="article-title"> Structure analysis of PH1161 protein, a transcriptional activator TenA homologue from the hyperthermophilic archaeon <em>Pyrococcus horikoshii</em></div>, <div class="source">Acta Crystallogr. D Biol. Crystallogr.</div> , <div class="year">2004</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">1094</div>-<div class="lpage">1100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20analysis%20of%20PH1161%20protein%2C%20a%20transcriptional%20activator%20TenA%20homologue%20from%20the%20hyperthermophilic%20archaeon%20Pyrococcus%20horikoshii&amp;author=H.%20Itou&amp;author=M.%20Yao&amp;author=N.%20Watanabe&amp;author=I.%20Tanaka&amp;publication_year=2004&amp;journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;volume=60&amp;pages=1094-1100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1107/S0907444904008522" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1107%2FS0907444904008522" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1107%2FS0907444904008522"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15159569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N. D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J. D.</div></div>, et al. </div><div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">41</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.%20L.%20Tatusov&amp;author=N.%20D.%20Fedorova&amp;author=J.%20D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vogel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Berzuini</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bashton</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Supra-domains: evolutionary units larger than single protein domains</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">336</div> (pg. <div class="fpage">809</div>-<div class="lpage">823</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Supra-domains%3A%20evolutionary%20units%20larger%20than%20single%20protein%20domains&amp;author=C.%20Vogel&amp;author=C.%20Berzuini&amp;author=M.%20Bashton&amp;publication_year=2004&amp;journal=J.%20Mol.%20Biol.&amp;volume=336&amp;pages=809-823" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2003.12.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2003.12.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2003.12.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15095989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Luscombe</div> <div class="given-names">N. M.</div></div>,  <div class="name"><div class="surname">Gerstein</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Protein family and fold occurrence in genomes: power-law behaviour and evolutionary model</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">673</div>-<div class="lpage">681</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20family%20and%20fold%20occurrence%20in%20genomes%3A%20power-law%20behaviour%20and%20evolutionary%20model&amp;author=J.%20Qian&amp;author=N.%20M.%20Luscombe&amp;author=M.%20Gerstein&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol.&amp;volume=313&amp;pages=673-681" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2001.5079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2001.5079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2001.5079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11697896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Weigel</div> <div class="given-names">C.</div></div>. </div><div class="article-title">DnaA initiator—also a transcription factor</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaA%20initiator%E2%80%94also%20a%20transcription%20factor&amp;author=W.%20Messer&amp;author=C.%20Weigel&amp;publication_year=1997&amp;journal=Mol.%20Microbiol.&amp;volume=24&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.3171678.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1997.3171678.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1997.3171678.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9140960" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glockner</div> <div class="given-names">F. O.</div></div>,  <div class="name"><div class="surname">Kube</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bauer</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title"> Complete genome sequence of the marine planctomycete <em>Pirellula sp</em> . strain 1 </div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">8298</div>-<div class="lpage">8303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20marine%20planctomycete%20Pirellula%20sp%20.%20strain%201&amp;author=F.%20O.%20Glockner&amp;author=M.%20Kube&amp;author=M.%20Bauer&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=8298-8303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1431443100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1431443100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1431443100"> </span></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohmori</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title"> Characterization of genes encoding multi-domain proteins in the genome of the filamentous nitrogen-fixing cyanobacterium <em>Anabaena sp</em> . strain PCC 7120 </div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">271</div>-<div class="lpage">284</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20genes%20encoding%20multi-domain%20proteins%20in%20the%20genome%20of%20the%20filamentous%20nitrogen-fixing%20cyanobacterium%20Anabaena%20sp%20.%20strain%20PCC%207120&amp;author=M.%20Ohmori&amp;author=M.%20Ikeuchi&amp;author=N.%20Sato&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=271-284" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.271"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11858227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brinkman</div> <div class="given-names">A. B.</div></div>,  <div class="name"><div class="surname">Ettema</div> <div class="given-names">T. J.</div></div>,  <div class="name"><div class="surname">de Vos</div> <div class="given-names">W. M.</div></div>,  <div class="name"><div class="surname">van der Oost</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The Lrp family of transcriptional regulators</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">287</div>-<div class="lpage">294</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Lrp%20family%20of%20transcriptional%20regulators&amp;author=A.%20B.%20Brinkman&amp;author=T.%20J.%20Ettema&amp;author=W.%20M.%20de%20Vos&amp;author=J.%20van%20der%20Oost&amp;publication_year=2003&amp;journal=Mol.%20Microbiol.&amp;volume=48&amp;pages=287-294" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03442.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03442.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03442.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12675791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koike</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ishijima</div> <div class="given-names">S. A.</div></div>,  <div class="name"><div class="surname">Clowney</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The archaeal feast/famine regulatory protein: potential roles of its assembly forms for regulating transcription</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">2840</div>-<div class="lpage">2845</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20archaeal%20feast%2Ffamine%20regulatory%20protein%3A%20potential%20roles%20of%20its%20assembly%20forms%20for%20regulating%20transcription&amp;author=H.%20Koike&amp;author=S.%20A.%20Ishijima&amp;author=L.%20Clowney&amp;author=M.%20Suzuki&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=2840-2845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0400109101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0400109101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0400109101"> </span></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coulson</div> <div class="given-names">R. M. R.</div></div>,  <div class="name"><div class="surname">Enright</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">Transcription-associated protein families are primarily taxon-specific</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">95</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-associated%20protein%20families%20are%20primarily%20taxon-specific&amp;author=R.%20M.%20R.%20Coulson&amp;author=A.%20J.%20Enright&amp;author=C.%20A.%20Ouzounis&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=95-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.1.95" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.1.95" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.1.95"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11222266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shigenobu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ishikawa</div> <div class="given-names">H.</div></div>. </div><div class="article-title"> Genome sequence of the endocellular bacterial symbiont of aphids <em>Buchnera sp</em> . APS </div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">407</div> (pg. <div class="fpage">81</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20endocellular%20bacterial%20symbiont%20of%20aphids%20Buchnera%20sp%20.%20APS&amp;author=S.%20Shigenobu&amp;author=H.%20Watanabe&amp;author=M.%20Hattori&amp;author=Y.%20Sakaki&amp;author=H.%20Ishikawa&amp;publication_year=2000&amp;journal=Nature&amp;volume=407&amp;pages=81-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35024074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35024074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35024074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10993077" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moran</div> <div class="given-names">N. A.</div></div>,  <div class="name"><div class="surname">Dunbar</div> <div class="given-names">H. E.</div></div>,  <div class="name"><div class="surname">Wilcox</div> <div class="given-names">J. L.</div></div>. </div><div class="article-title"> Regulation of transcription in a reduced bacterial genome: nutrient-provisioning genes of the obligate symbiont <em>Buchnera aphidicola</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">4229</div>-<div class="lpage">4237</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20transcription%20in%20a%20reduced%20bacterial%20genome%3A%20nutrient-provisioning%20genes%20of%20the%20obligate%20symbiont%20Buchnera%20aphidicola&amp;author=N.%20A.%20Moran&amp;author=H.%20E.%20Dunbar&amp;author=J.%20L.%20Wilcox&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=4229-4237" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.12.4229-4237.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.12.4229-4237.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.12.4229-4237.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15937185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galagan</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Nusbaum</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Roy</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title"> The genome of <em>M. acetivorans</em> reveals extensive metabolic and physiological diversity </div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">532</div>-<div class="lpage">542</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20M.%20acetivorans%20reveals%20extensive%20metabolic%20and%20physiological%20diversity&amp;author=J.%20E.%20Galagan&amp;author=C.%20Nusbaum&amp;author=A.%20Roy&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=532-542" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.223902" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.223902" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.223902"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klenk</div> <div class="given-names">H. P.</div></div>,  <div class="name"><div class="surname">Clayton</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Tomb</div> <div class="given-names">J. F.</div></div>, et al. </div><div class="article-title"> The complete genome sequence of the hyperthermophilic, sulphate-reducing archaeon <em>Archaeoglobus fulgidus</em></div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">390</div> (pg. <div class="fpage">364</div>-<div class="lpage">370</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20hyperthermophilic%2C%20sulphate-reducing%20archaeon%20Archaeoglobus%20fulgidus&amp;author=H.%20P.%20Klenk&amp;author=R.%20A.%20Clayton&amp;author=J.%20F.%20Tomb&amp;publication_year=1997&amp;journal=Nature&amp;volume=390&amp;pages=364-370" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/37052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F37052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F37052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9389475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bentley</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Chater</div> <div class="given-names">K. F.</div></div>,  <div class="name"><div class="surname">Cerdeno-Tarraga</div> <div class="given-names">A. M.</div></div>, et al. </div><div class="article-title"> Complete genome sequence of the model actinomycete <em>Streptomyces coelicolor</em> A3(2) </div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">417</div> (pg. <div class="fpage">141</div>-<div class="lpage">147</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20model%20actinomycete%20Streptomyces%20coelicolor%20A3(2)&amp;author=S.%20D.%20Bentley&amp;author=K.%20F.%20Chater&amp;author=A.%20M.%20Cerdeno-Tarraga&amp;publication_year=2002&amp;journal=Nature&amp;volume=417&amp;pages=141-147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/417141a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F417141a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F417141a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12000953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schell</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Karmirantzou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Snel</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title"> The genome sequence of <em>Bifidobacterium longum</em> reflects its adaptation to the human gastrointestinal tract </div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">14422</div>-<div class="lpage">14427</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Bifidobacterium%20longum%20reflects%20its%20adaptation%20to%20the%20human%20gastrointestinal%20tract&amp;author=M.%20A.%20Schell&amp;author=M.%20Karmirantzou&amp;author=B.%20Snel&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=14422-14427" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.212527599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.212527599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.212527599"> </span></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunst</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moszer</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title"> The complete genome sequence of the gram-positive bacterium <em>Bacillus subtilis</em></div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">390</div> (pg. <div class="fpage">249</div>-<div class="lpage">256</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20gram-positive%20bacterium%20Bacillus%20subtilis&amp;author=F.%20Kunst&amp;author=N.%20Ogasawara&amp;author=I.%20Moszer&amp;publication_year=1997&amp;journal=Nature&amp;volume=390&amp;pages=249-256" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/36786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F36786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F36786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9384377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nolling</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Breton</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Omelchenko</div> <div class="given-names">M. V.</div></div>, et al. </div><div class="article-title"> Genome sequence and comparative analysis of the solvent-producing bacterium <em>Clostridium acetobutylicum</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">4823</div>-<div class="lpage">4838</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20comparative%20analysis%20of%20the%20solvent-producing%20bacterium%20Clostridium%20acetobutylicum&amp;author=J.%20Nolling&amp;author=G.%20Breton&amp;author=M.%20V.%20Omelchenko&amp;publication_year=2001&amp;journal=J.%20Bacteriol.&amp;volume=183&amp;pages=4823-4838" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.16.4823-4838.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.16.4823-4838.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.16.4823-4838.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11466286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duchaud</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Rusniok</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Frangeul</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title"> The genome sequence of the entomopathogenic bacterium <em>Photorhabdus luminescens</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1307</div>-<div class="lpage">1313</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20entomopathogenic%20bacterium%20Photorhabdus%20luminescens&amp;author=E.%20Duchaud&amp;author=C.%20Rusniok&amp;author=L.%20Frangeul&amp;publication_year=2003&amp;journal=Nat.%20Biotechnol.&amp;volume=21&amp;pages=1307-1313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt886" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt886" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt886"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14528314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pridmore</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Berger</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Desiere</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title"> The genome sequence of the probiotic intestinal bacterium <em>Lactobacillus johnsonii</em> NCC 533 </div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">2512</div>-<div class="lpage">2517</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20probiotic%20intestinal%20bacterium%20Lactobacillus%20johnsonii%20NCC%20533&amp;author=R.%20D.%20Pridmore&amp;author=B.%20Berger&amp;author=F.%20Desiere&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=2512-2517" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307327101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307327101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307327101"> </span></div> </div></div></div></div></div><div content-id="b44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bolotin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wincker</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Mauger</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title"> The complete genome sequence of the lactic acid bacterium <em>Lactococcus lactis</em> ssp. lactis IL1403 </div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">731</div>-<div class="lpage">753</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20lactic%20acid%20bacterium%20Lactococcus%20lactis%20ssp.%20lactis%20IL1403&amp;author=A.%20Bolotin&amp;author=P.%20Wincker&amp;author=S.%20Mauger&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=731-753" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.GR-1697R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.GR-1697R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.GR-1697R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11337471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapatral</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Ivanova</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title"> Genome sequence and analysis of the oral bacterium <em>Fusobacterium nucleatum</em> strain ATCC 25586 </div>, <div class="source">J. Bacteriol.</div> , <div class="year">2002</div>, vol. <div class="volume">184</div> (pg. <div class="fpage">2005</div>-<div class="lpage">2018</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20analysis%20of%20the%20oral%20bacterium%20Fusobacterium%20nucleatum%20strain%20ATCC%2025586&amp;author=V.%20Kapatral&amp;author=I.%20Anderson&amp;author=N.%20Ivanova&amp;publication_year=2002&amp;journal=J.%20Bacteriol.&amp;volume=184&amp;pages=2005-2018" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.184.7.2005-2018.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.184.7.2005-2018.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.184.7.2005-2018.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11889109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larimer</div> <div class="given-names">F. W.</div></div>,  <div class="name"><div class="surname">Chain</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hauser</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title"> Complete genome sequence of the metabolically versatile photosynthetic bacterium <em>Rhodopseudomonas palustris</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">55</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20metabolically%20versatile%20photosynthetic%20bacterium%20Rhodopseudomonas%20palustris&amp;author=F.%20W.%20Larimer&amp;author=P.%20Chain&amp;author=L.%20Hauser&amp;publication_year=2004&amp;journal=Nat.%20Biotechnol.&amp;volume=22&amp;pages=55-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=350906&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">448</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119800001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">37</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi016"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="350906"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119800001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f5%2f269%2f350906">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119800001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (37)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F5%2F269%2F350906" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/350906'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide Survey of Transcription Factors in Prokaryotes Reveals Many Bacteria-Specific Families Not Found in Archaea" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi016");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f269%2f350906">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

