

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":744,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/5/477/2236181","@type":"ScholarlyArticle","name":"Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome","datePublished":"2016-07-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2016-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw031","keywords":["Z-DNA","Z-DNA-binding protein","human genome","ChIP-Seq","active transcription"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/5/477/2236181","author":[{"name":"Shin, So-I.","affiliation":"Department of Life Sciences","@type":"Person"},{"name":"Ham, Seokjin","affiliation":"Department of Life Sciences","@type":"Person"},{"name":"Park, Jihwan","affiliation":"Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea","@type":"Person"},{"name":"Seo, Seong Hye","affiliation":"Department of Life Sciences","@type":"Person"},{"name":"Lim, Chae Hyun","affiliation":"Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea","@type":"Person"},{"name":"Jeon, Hyeongrin","affiliation":"Department of Life Sciences","@type":"Person"},{"name":"Huh, Jounghyun","affiliation":"Department of Life Sciences","@type":"Person"},{"name":"Roh, Tae-Young","affiliation":"Department of Life Sciences  Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea","@type":"Person"}],"description":"Abstract.  Z-DNA, a left-handed double helical DNA is structurally different from the most abundant B-DNA. Z-DNA has been known to play a significant role in tr","pageStart":"477","pageEnd":"486","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome" />
    <meta property="og:description" content="Abstract.  Z-DNA, a left-handed double helical DNA is structurally different from the most abundant B-DNA. Z-DNA has been known to play a significant role in tr" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/5/477/2236181" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-07-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shin, So-I." /><meta name="citation_author_institution" content="Department of Life Sciences" /><meta name="citation_author" content="Ham, Seokjin" /><meta name="citation_author_institution" content="Department of Life Sciences" /><meta name="citation_author" content="Park, Jihwan" /><meta name="citation_author_institution" content="Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea" /><meta name="citation_author" content="Seo, Seong Hye" /><meta name="citation_author_institution" content="Department of Life Sciences" /><meta name="citation_author" content="Lim, Chae Hyun" /><meta name="citation_author_institution" content="Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea" /><meta name="citation_author" content="Jeon, Hyeongrin" /><meta name="citation_author_institution" content="Department of Life Sciences" /><meta name="citation_author" content="Huh, Jounghyun" /><meta name="citation_author_institution" content="Department of Life Sciences" /><meta name="citation_author" content="Roh, Tae-Young" /><meta name="citation_author_institution" content="Department of Life Sciences" /><meta name="citation_author_institution" content="Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea" /><meta name="citation_title" content="Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome" /><meta name="citation_firstpage" content="477" /><meta name="citation_lastpage" content="486" /><meta name="citation_doi" content="10.1093/dnares/dsw031" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2016/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Consecutive A X T pairs can adopt a left-handed DNA structure; citation_author=McLean  M. J.; citation_author=Blaho  J. A.; citation_author=Kilpatrick  M. W.; citation_author=Wells  R. D.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=1986;  citation_volume=83;  citation_pages=5884-8; " /><meta name="citation_reference" content="citation_title=A computer aided thermodynamic approach for predicting the formation of Z-DNA in naturally occurring sequences; citation_author=Ho  P. S.; citation_author=Ellison  M. J.; citation_author=Quigley  G. J.; citation_author=Rich  A.;  citation_journal_title=EMBO J;  citation_year=1986;  citation_volume=5;  citation_pages=2737-44; " /><meta name="citation_reference" content="citation_title=An assessment of the Z-DNA forming potential of alternating dA-dT stretches in supercoiled plasmids; citation_author=Ellison  M. J.; citation_author=Feigon  J.; citation_author=Kelleher  R. J.; citation_author=Wang  A. H.; citation_author=Habener  J. F.; citation_author=Rich  A.;  citation_journal_title=Biochemistry;  citation_year=1986;  citation_volume=25;  citation_pages=3648-55; " /><meta name="citation_reference" content="citation_title=Effects of methylation on a synthetic polynucleotide: the B–Z transition in poly(dG-m5dC).poly(dG-m5dC); citation_author=Behe  M.; citation_author=Felsenfeld  G.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=1981;  citation_volume=78;  citation_pages=1619-23; " /><meta name="citation_reference" content="citation_title=Flipping of cloned d(pCpG)n.d(pCpG)n DNA sequences from right- to left-handed helical structure by salt, Co(III), or negative supercoiling; citation_author=Peck  L. J.; citation_author=Nordheim  A.; citation_author=Rich  A.; citation_author=Wang  J. C.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=1982;  citation_volume=79;  citation_pages=4560-4; " /><meta name="citation_reference" content="citation_title=Proteolytic dissection of Zab, the Z-DNA-binding domain of human ADAR1; citation_author=Schwartz  T.; citation_author=Lowenhaupt  K.; citation_author=Kim  Y. G.;  citation_journal_title=J. Bio. Chem;  citation_year=1999;  citation_volume=274;  citation_pages=2899-906; " /><meta name="citation_reference" content="citation_title=The Zalpha domain from human ADAR1 binds to the Z-DNA conformer of many different sequences; citation_author=Herbert  A.; citation_author=Schade  M.; citation_author=Lowenhaupt  K.;  citation_journal_title=Nucleic. Acids Res;  citation_year=1998;  citation_volume=26;  citation_pages=3486-93; " /><meta name="citation_reference" content="citation_title=The interaction between Z-DNA and the Zab domain of double-stranded RNA adenosine deaminase characterized using fusion nucleases; citation_author=Kim  Y. G.; citation_author=Lowenhaupt  K.; citation_author=Schwartz  T.; citation_author=Rich  A.;  citation_journal_title=J. Bio. Chem;  citation_year=1999;  citation_volume=274;  citation_pages=19081-6; " /><meta name="citation_reference" content="citation_title=A role for Z-DNA binding in vaccinia virus pathogenesis; citation_author=Kim  Y. G.; citation_author=Muralinath  M.; citation_author=Brandt  T.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=2003;  citation_volume=100;  citation_pages=6974-9; " /><meta name="citation_reference" content="citation_title=Evidence that vaccinia virulence factor E3L binds to Z-DNA in vivo: Implications for development of a therapy for poxvirus infection; citation_author=Kim  Y. G.; citation_author=Lowenhaupt  K.; citation_author=Oh  D. B.; citation_author=Kim  K. K.; citation_author=Rich  A.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=2004;  citation_volume=101;  citation_pages=1514-8; " /><meta name="citation_reference" content="citation_title=Mapping Z-DNA in the human genome. Computer-aided mapping reveals a nonrandom distribution of potential Z-DNA-forming sequences in human genes; citation_author=Schroth  G. P.; citation_author=Chou  P. J.; citation_author=Ho  P. S.;  citation_journal_title=J. Biol. Chem;  citation_year=1992;  citation_volume=267;  citation_pages=11846-55; " /><meta name="citation_reference" content="citation_title=Human genomic Z-DNA segments probed by the Z alpha domain of ADAR1; citation_author=Li  H.; citation_author=Xiao  J.; citation_author=Li  J.; citation_author=Lu  L.; citation_author=Feng  S.; citation_author=Droge  P.;  citation_journal_title=Nucleic. Acids. Res;  citation_year=2009;  citation_volume=37;  citation_pages=2737-46; " /><meta name="citation_reference" content="citation_title=Supercoiling of the DNA template during transcription; citation_author=Liu  L. F.; citation_author=Wang  J. C.;  citation_journal_title=Proc. Natl Acad Sci. U S A;  citation_year=1987;  citation_volume=84;  citation_pages=7024-7; " /><meta name="citation_reference" content="citation_title=Transcription of human c-myc in permeabilized nuclei is associated with formation of Z-DNA in three discrete regions of the gene; citation_author=Wittig  B.; citation_author=Wolfl  S.; citation_author=Dorbic  T.; citation_author=Vahrson  W.; citation_author=Rich  A.;  citation_journal_title=EMBO J;  citation_year=1992;  citation_volume=11;  citation_pages=4653-63; " /><meta name="citation_reference" content="citation_title=Regulation of CSF1 promoter by the SWI/SNF-like BAF complex; citation_author=Liu  R.; citation_author=Liu  H.; citation_author=Chen  X.; citation_author=Kirby  M.; citation_author=Brown  P. O.; citation_author=Zhao  K.;  citation_journal_title=Cell;  citation_year=2001;  citation_volume=106;  citation_pages=309-18; " /><meta name="citation_reference" content="citation_title=Nrf2 activation is associated with Z-DNA formation in the human HO-1 promoter; citation_author=Maruyama  A.; citation_author=Mimura  J.; citation_author=Harada  N.; citation_author=Itoh  K.;  citation_journal_title=Nucleic. Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=5223-34; " /><meta name="citation_reference" content="citation_title=Characterization of Z-DNA as a nucleosome-boundary element in yeast Saccharomyces cerevisiae; citation_author=Wong  B.; citation_author=Chen  S.; citation_author=Kwon  J. A.; citation_author=Rich  A.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=2007;  citation_volume=104;  citation_pages=2229-34; " /><meta name="citation_reference" content="citation_title=Non-B DNA structure-induced genetic instability; citation_author=Wang  G.; citation_author=Vasquez  K. M.;  citation_journal_title=Mut. Res;  citation_year=2006;  citation_volume=598;  citation_pages=103-19; " /><meta name="citation_reference" content="citation_title=Z-DNA-forming sequences generate large-scale deletions in mammalian cells; citation_author=Wang  G.; citation_author=Christensen  L. A.; citation_author=Vasquez  K. M.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=2006;  citation_volume=103;  citation_pages=2677-82; " /><meta name="citation_reference" content="citation_title=, A Z-DNA sequence reduces slipped-strand structure formation in the myotonic dystrophy type 2 (CCTG) x (CAGG) repeat; citation_author=Edwards  S. F.; citation_author=Sirito  M.; citation_author=Krahe  R.; citation_author=Sinden  R. R.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=2009;  citation_volume=106;  citation_pages=3270-5; " /><meta name="citation_reference" content="citation_title=SWI/SNF-mediated chromatin remodeling induces Z-DNA formation on a nucleosome; citation_author=Mulholland  N.; citation_author=Xu  Y.; citation_author=Sugiyama  H.; citation_author=Zhao  K.;  citation_journal_title=Cell &amp; Biosci;  citation_year=2012;  citation_volume=2;  citation_pages=3" /><meta name="citation_reference" content="citation_title=Construction of a Z-DNA-specific restriction endonuclease; citation_author=Kim  Y. G.; citation_author=Kim  P. S.; citation_author=Herbert  A.; citation_author=Rich  A.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=1997;  citation_volume=94;  citation_pages=12875-9; " /><meta name="citation_reference" content="citation_title=NFkappaB and STAT3 synergistically activate the expression of FAT10, a gene counteracting the tumor suppressor p53; citation_author=Choi  Y.; citation_author=Kim  J. K.; citation_author=Yoo  J. Y.;  citation_journal_title=Mol. Oncol;  citation_year=2014;  citation_volume=8;  citation_pages=642-55; " /><meta name="citation_reference" content="citation_title=Functional elements demarcated by histone modifications in breast cancer cells; citation_author=Choe  M. K.; citation_author=Hong  C. P.; citation_author=Park  J.; citation_author=Seo  S. H.; citation_author=Roh  T. Y.;  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=2012;  citation_volume=418;  citation_pages=475-82; " /><meta name="citation_reference" content="citation_title=Mapping accessible chromatin regions using Sono-Seq; citation_author=Auerbach  R. K.; citation_author=Euskirchen  G.; citation_author=Rozowsky  J.;  citation_journal_title=Proc. Natl Acad.Sci.U S A;  citation_year=2009;  citation_volume=106;  citation_pages=14926-31; " /><meta name="citation_reference" content="citation_title=ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia; citation_author=Landt  S. G.; citation_author=Marinov  G. K.; citation_author=Kundaje  A.;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=1813-31; " /><meta name="citation_reference" content="citation_title=Deep proteome and transcriptome mapping of a human cancer cell line; citation_author=Nagaraj  N.; citation_author=Wisniewski  J. R.; citation_author=Geiger  T.;  citation_journal_title=Mol. Syst. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=548" /><meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions; citation_author=Kim  D.; citation_author=Pertea  G.; citation_author=Trapnell  C.; citation_author=Pimentel  H.; citation_author=Kelley  R.; citation_author=Salzberg  S. L.;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R36" /><meta name="citation_reference" content="citation_title=Distinct and predictive histone lysine acetylation patterns at promoters, enhancers, and gene bodies; citation_author=Rajagopal  N.; citation_author=Ernst  J.; citation_author=Ray  P.;  citation_journal_title=G3 (Bethesda);  citation_year=2014;  citation_volume=4;  citation_pages=2051-63; " /><meta name="citation_reference" content="citation_title=Z-DNA-forming silencer in the first exon regulates human ADAM-12 gene expression; citation_author=Ray  B. K.; citation_author=Dhar  S.; citation_author=Shakya  A.; citation_author=Ray  A.;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=2011;  citation_volume=108;  citation_pages=103-8; " /><meta name="citation_reference" content="citation_title=Generation of superhelical torsion by ATP-dependent chromatin remodeling activities; citation_author=Havas  K.; citation_author=Flaus  A.; citation_author=Phelan  M.;  citation_journal_title=Cell;  citation_year=2000;  citation_volume=103;  citation_pages=1133-42; " /><meta name="citation_reference" content="citation_title=SWI/SNF chromatin remodeling requires changes in DNA topology; citation_author=Gavin  I.; citation_author=Horn  P. J.; citation_author=Peterson  C. L.;  citation_journal_title=Mol. Cell;  citation_year=2001;  citation_volume=7;  citation_pages=97-104; " /><meta name="citation_reference" content="citation_title=Transcription forms and remodels supercoiling domains unfolding large-scale chromatin structures; citation_author=Naughton  C.; citation_author=Avlonitis  N.; citation_author=Corless  S.;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2013;  citation_volume=20;  citation_pages=387-95; " /><meta name="citation_reference" content="citation_title=Transcription-dependent dynamic supercoiling is a short-range genomic force; citation_author=Kouzine  F.; citation_author=Gupta  A.; citation_author=Baranello  L.;  citation_journal_title=Nat. Struct. Mol Biol;  citation_year=2013;  citation_volume=20;  citation_pages=396-403; " /><meta name="citation_reference" content="citation_title=Active promoters give rise to false positive 'Phantom Peaks' in ChIP-seq experiments; citation_author=Jain  D.; citation_author=Baldi  S.; citation_author=Zabel  A.; citation_author=Straub  T.; citation_author=Becker  P. B.;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=6959-68; " /><meta name="citation_reference" content="citation_title=Impact of artifact removal on ChIP quality metrics in ChIP-seq and ChIP-exo data; citation_author=Carroll  T. S.; citation_author=Liang  Z.; citation_author=Salama  R.; citation_author=Stark  R.; citation_author=de Santiago  I.;  citation_journal_title=Front Genet;  citation_year=2014;  citation_volume=5;  citation_pages=75" /><meta name="citation_fulltext_world_readable" content="" /><meta name="description" content="Abstract.  Z-DNA, a left-handed double helical DNA is structurally different from the most abundant B-DNA. Z-DNA has been known to play a significant role in tr" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/5/477/2236181" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/5/477/2236181" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/5/477/2236181");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw031';
    </script>
<title>
	Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2236181" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb16PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGluLCBTby1JLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhbSwgU2Vva2ppbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhcmssIEppaHdhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEludGVncmF0aXZlIEJpb3NjaWVuY2VzICZhbXA7IEJpb3RlY2hub2xvZ3ksIFBvaGFuZyBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKFBPU1RFQ0gpLCBQb2hhbmcsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2VvLCBTZW9uZyBIeWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIExpZmUgU2NpZW5jZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaW0sIENoYWUgSHl1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEludGVncmF0aXZlIEJpb3NjaWVuY2VzICZhbXA7IEJpb3RlY2hub2xvZ3ksIFBvaGFuZyBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKFBPU1RFQ0gpLCBQb2hhbmcsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmVvbiwgSHllb25ncmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVoLCBKb3VuZ2h5dW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIExpZmUgU2NpZW5jZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSb2gsIFRhZS1Zb3VuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEludGVncmF0aXZlIEJpb3NjaWVuY2VzICZhbXA7IEJpb3RlY2hub2xvZ3ksIFBvaGFuZyBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKFBPU1RFQ0gpLCBQb2hhbmcsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJaLUROQS1mb3JtaW5nIHNpdGVzIGlkZW50aWZpZWQgYnkgQ2hJUC1TZXEgYXJlIGFzc29jaWF0ZWQgd2l0aCBhY3RpdmVseSB0cmFuc2NyaWJlZCByZWdpb25zIGluIHRoZSBodW1hbiBnZW5vbWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0NzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc2VjdXRpdmUgQSBYIFQgcGFpcnMgY2FuIGFkb3B0IGEgbGVmdC1oYW5kZWQgRE5BIHN0cnVjdHVyZTsgY2l0YXRpb25fYXV0aG9yPU1jTGVhbiAgTS4gSi47IGNpdGF0aW9uX2F1dGhvcj1CbGFobyAgSi4gQS47IGNpdGF0aW9uX2F1dGhvcj1LaWxwYXRyaWNrICBNLiBXLjsgY2l0YXRpb25fYXV0aG9yPVdlbGxzICBSLiBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9NTg4NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXB1dGVyIGFpZGVkIHRoZXJtb2R5bmFtaWMgYXBwcm9hY2ggZm9yIHByZWRpY3RpbmcgdGhlIGZvcm1hdGlvbiBvZiBaLUROQSBpbiBuYXR1cmFsbHkgb2NjdXJyaW5nIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPUhvICBQLiBTLjsgY2l0YXRpb25fYXV0aG9yPUVsbGlzb24gIE0uIEouOyBjaXRhdGlvbl9hdXRob3I9UXVpZ2xleSAgRy4gSi47IGNpdGF0aW9uX2F1dGhvcj1SaWNoICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yNzM3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhc3Nlc3NtZW50IG9mIHRoZSBaLUROQSBmb3JtaW5nIHBvdGVudGlhbCBvZiBhbHRlcm5hdGluZyBkQS1kVCBzdHJldGNoZXMgaW4gc3VwZXJjb2lsZWQgcGxhc21pZHM7IGNpdGF0aW9uX2F1dGhvcj1FbGxpc29uICBNLiBKLjsgY2l0YXRpb25fYXV0aG9yPUZlaWdvbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1LZWxsZWhlciAgUi4gSi47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBBLiBILjsgY2l0YXRpb25fYXV0aG9yPUhhYmVuZXIgIEouIEYuOyBjaXRhdGlvbl9hdXRob3I9UmljaCAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTM2NDgtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgbWV0aHlsYXRpb24gb24gYSBzeW50aGV0aWMgcG9seW51Y2xlb3RpZGU6IHRoZSBC4oCTWiB0cmFuc2l0aW9uIGluIHBvbHkoZEctbTVkQykucG9seShkRy1tNWRDKTsgY2l0YXRpb25fYXV0aG9yPUJlaGUgIE0uOyBjaXRhdGlvbl9hdXRob3I9RmVsc2VuZmVsZCAgRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTE2MTktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZsaXBwaW5nIG9mIGNsb25lZCBkKHBDcEcpbi5kKHBDcEcpbiBETkEgc2VxdWVuY2VzIGZyb20gcmlnaHQtIHRvIGxlZnQtaGFuZGVkIGhlbGljYWwgc3RydWN0dXJlIGJ5IHNhbHQsIENvKElJSSksIG9yIG5lZ2F0aXZlIHN1cGVyY29pbGluZzsgY2l0YXRpb25fYXV0aG9yPVBlY2sgIEwuIEouOyBjaXRhdGlvbl9hdXRob3I9Tm9yZGhlaW0gIEEuOyBjaXRhdGlvbl9hdXRob3I9UmljaCAgQS47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBKLiBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9NDU2MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZW9seXRpYyBkaXNzZWN0aW9uIG9mIFphYiwgdGhlIFotRE5BLWJpbmRpbmcgZG9tYWluIG9mIGh1bWFuIEFEQVIxOyBjaXRhdGlvbl9hdXRob3I9U2Nod2FydHogIFQuOyBjaXRhdGlvbl9hdXRob3I9TG93ZW5oYXVwdCAgSy47IGNpdGF0aW9uX2F1dGhvcj1LaW0gIFkuIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW8uIENoZW07ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc0OyAgY2l0YXRpb25fcGFnZXM9Mjg5OS05MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBaYWxwaGEgZG9tYWluIGZyb20gaHVtYW4gQURBUjEgYmluZHMgdG8gdGhlIFotRE5BIGNvbmZvcm1lciBvZiBtYW55IGRpZmZlcmVudCBzZXF1ZW5jZXM7IGNpdGF0aW9uX2F1dGhvcj1IZXJiZXJ0ICBBLjsgY2l0YXRpb25fYXV0aG9yPVNjaGFkZSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Mb3dlbmhhdXB0ICBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYy4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzQ4Ni05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGludGVyYWN0aW9uIGJldHdlZW4gWi1ETkEgYW5kIHRoZSBaYWIgZG9tYWluIG9mIGRvdWJsZS1zdHJhbmRlZCBSTkEgYWRlbm9zaW5lIGRlYW1pbmFzZSBjaGFyYWN0ZXJpemVkIHVzaW5nIGZ1c2lvbiBudWNsZWFzZXM7IGNpdGF0aW9uX2F1dGhvcj1LaW0gIFkuIEcuOyBjaXRhdGlvbl9hdXRob3I9TG93ZW5oYXVwdCAgSy47IGNpdGF0aW9uX2F1dGhvcj1TY2h3YXJ0eiAgVC47IGNpdGF0aW9uX2F1dGhvcj1SaWNoICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3NDsgIGNpdGF0aW9uX3BhZ2VzPTE5MDgxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcm9sZSBmb3IgWi1ETkEgYmluZGluZyBpbiB2YWNjaW5pYSB2aXJ1cyBwYXRob2dlbmVzaXM7IGNpdGF0aW9uX2F1dGhvcj1LaW0gIFkuIEcuOyBjaXRhdGlvbl9hdXRob3I9TXVyYWxpbmF0aCAgTS47IGNpdGF0aW9uX2F1dGhvcj1CcmFuZHQgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVSBTIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9Njk3NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSB0aGF0IHZhY2NpbmlhIHZpcnVsZW5jZSBmYWN0b3IgRTNMIGJpbmRzIHRvIFotRE5BIGluIHZpdm86IEltcGxpY2F0aW9ucyBmb3IgZGV2ZWxvcG1lbnQgb2YgYSB0aGVyYXB5IGZvciBwb3h2aXJ1cyBpbmZlY3Rpb247IGNpdGF0aW9uX2F1dGhvcj1LaW0gIFkuIEcuOyBjaXRhdGlvbl9hdXRob3I9TG93ZW5oYXVwdCAgSy47IGNpdGF0aW9uX2F1dGhvcj1PaCAgRC4gQi47IGNpdGF0aW9uX2F1dGhvcj1LaW0gIEsuIEsuOyBjaXRhdGlvbl9hdXRob3I9UmljaCAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xNTE0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgWi1ETkEgaW4gdGhlIGh1bWFuIGdlbm9tZS4gQ29tcHV0ZXItYWlkZWQgbWFwcGluZyByZXZlYWxzIGEgbm9ucmFuZG9tIGRpc3RyaWJ1dGlvbiBvZiBwb3RlbnRpYWwgWi1ETkEtZm9ybWluZyBzZXF1ZW5jZXMgaW4gaHVtYW4gZ2VuZXM7IGNpdGF0aW9uX2F1dGhvcj1TY2hyb3RoICBHLiBQLjsgY2l0YXRpb25fYXV0aG9yPUNob3UgIFAuIEouOyBjaXRhdGlvbl9hdXRob3I9SG8gIFAuIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI2NzsgIGNpdGF0aW9uX3BhZ2VzPTExODQ2LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBnZW5vbWljIFotRE5BIHNlZ21lbnRzIHByb2JlZCBieSB0aGUgWiBhbHBoYSBkb21haW4gb2YgQURBUjE7IGNpdGF0aW9uX2F1dGhvcj1MaSAgSC47IGNpdGF0aW9uX2F1dGhvcj1YaWFvICBKLjsgY2l0YXRpb25fYXV0aG9yPUxpICBKLjsgY2l0YXRpb25fYXV0aG9yPUx1ICBMLjsgY2l0YXRpb25fYXV0aG9yPUZlbmcgIFMuOyBjaXRhdGlvbl9hdXRob3I9RHJvZ2UgIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljLiBBY2lkcy4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MjczNy00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VwZXJjb2lsaW5nIG9mIHRoZSBETkEgdGVtcGxhdGUgZHVyaW5nIHRyYW5zY3JpcHRpb247IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEwuIEYuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgSi4gQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZCBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9NzAyNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIG9mIGh1bWFuIGMtbXljIGluIHBlcm1lYWJpbGl6ZWQgbnVjbGVpIGlzIGFzc29jaWF0ZWQgd2l0aCBmb3JtYXRpb24gb2YgWi1ETkEgaW4gdGhyZWUgZGlzY3JldGUgcmVnaW9ucyBvZiB0aGUgZ2VuZTsgY2l0YXRpb25fYXV0aG9yPVdpdHRpZyAgQi47IGNpdGF0aW9uX2F1dGhvcj1Xb2xmbCAgUy47IGNpdGF0aW9uX2F1dGhvcj1Eb3JiaWMgIFQuOyBjaXRhdGlvbl9hdXRob3I9VmFocnNvbiAgVy47IGNpdGF0aW9uX2F1dGhvcj1SaWNoICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDY1My02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBDU0YxIHByb21vdGVyIGJ5IHRoZSBTV0kvU05GLWxpa2UgQkFGIGNvbXBsZXg7IGNpdGF0aW9uX2F1dGhvcj1MaXUgIFIuOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBILjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIFguOyBjaXRhdGlvbl9hdXRob3I9S2lyYnkgIE0uOyBjaXRhdGlvbl9hdXRob3I9QnJvd24gIFAuIE8uOyBjaXRhdGlvbl9hdXRob3I9WmhhbyAgSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MzA5LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OcmYyIGFjdGl2YXRpb24gaXMgYXNzb2NpYXRlZCB3aXRoIFotRE5BIGZvcm1hdGlvbiBpbiB0aGUgaHVtYW4gSE8tMSBwcm9tb3RlcjsgY2l0YXRpb25fYXV0aG9yPU1hcnV5YW1hICBBLjsgY2l0YXRpb25fYXV0aG9yPU1pbXVyYSAgSi47IGNpdGF0aW9uX2F1dGhvcj1IYXJhZGEgIE4uOyBjaXRhdGlvbl9hdXRob3I9SXRvaCAgSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTUyMjMtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgWi1ETkEgYXMgYSBudWNsZW9zb21lLWJvdW5kYXJ5IGVsZW1lbnQgaW4geWVhc3QgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBjaXRhdGlvbl9hdXRob3I9V29uZyAgQi47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBTLjsgY2l0YXRpb25fYXV0aG9yPUt3b24gIEouIEEuOyBjaXRhdGlvbl9hdXRob3I9UmljaCAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0yMjI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob24tQiBETkEgc3RydWN0dXJlLWluZHVjZWQgZ2VuZXRpYyBpbnN0YWJpbGl0eTsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEcuOyBjaXRhdGlvbl9hdXRob3I9VmFzcXVleiAgSy4gTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU11dC4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU5ODsgIGNpdGF0aW9uX3BhZ2VzPTEwMy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Wi1ETkEtZm9ybWluZyBzZXF1ZW5jZXMgZ2VuZXJhdGUgbGFyZ2Utc2NhbGUgZGVsZXRpb25zIGluIG1hbW1hbGlhbiBjZWxsczsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEcuOyBjaXRhdGlvbl9hdXRob3I9Q2hyaXN0ZW5zZW4gIEwuIEEuOyBjaXRhdGlvbl9hdXRob3I9VmFzcXVleiAgSy4gTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0yNjc3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT0sIEEgWi1ETkEgc2VxdWVuY2UgcmVkdWNlcyBzbGlwcGVkLXN0cmFuZCBzdHJ1Y3R1cmUgZm9ybWF0aW9uIGluIHRoZSBteW90b25pYyBkeXN0cm9waHkgdHlwZSAyIChDQ1RHKSB4IChDQUdHKSByZXBlYXQ7IGNpdGF0aW9uX2F1dGhvcj1FZHdhcmRzICBTLiBGLjsgY2l0YXRpb25fYXV0aG9yPVNpcml0byAgTS47IGNpdGF0aW9uX2F1dGhvcj1LcmFoZSAgUi47IGNpdGF0aW9uX2F1dGhvcj1TaW5kZW4gIFIuIFIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVSBTIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MzI3MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TV0kvU05GLW1lZGlhdGVkIGNocm9tYXRpbiByZW1vZGVsaW5nIGluZHVjZXMgWi1ETkEgZm9ybWF0aW9uIG9uIGEgbnVjbGVvc29tZTsgY2l0YXRpb25fYXV0aG9yPU11bGhvbGxhbmQgIE4uOyBjaXRhdGlvbl9hdXRob3I9WHUgIFkuOyBjaXRhdGlvbl9hdXRob3I9U3VnaXlhbWEgIEguOyBjaXRhdGlvbl9hdXRob3I9WmhhbyAgSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgJmFtcDsgQmlvc2NpOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgWi1ETkEtc3BlY2lmaWMgcmVzdHJpY3Rpb24gZW5kb251Y2xlYXNlOyBjaXRhdGlvbl9hdXRob3I9S2ltICBZLiBHLjsgY2l0YXRpb25fYXV0aG9yPUtpbSAgUC4gUy47IGNpdGF0aW9uX2F1dGhvcj1IZXJiZXJ0ICBBLjsgY2l0YXRpb25fYXV0aG9yPVJpY2ggIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVSBTIEE7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjg3NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ORmthcHBhQiBhbmQgU1RBVDMgc3luZXJnaXN0aWNhbGx5IGFjdGl2YXRlIHRoZSBleHByZXNzaW9uIG9mIEZBVDEwLCBhIGdlbmUgY291bnRlcmFjdGluZyB0aGUgdHVtb3Igc3VwcHJlc3NvciBwNTM7IGNpdGF0aW9uX2F1dGhvcj1DaG9pICBZLjsgY2l0YXRpb25fYXV0aG9yPUtpbSAgSi4gSy47IGNpdGF0aW9uX2F1dGhvcj1Zb28gIEouIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE9uY29sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz02NDItNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgZWxlbWVudHMgZGVtYXJjYXRlZCBieSBoaXN0b25lIG1vZGlmaWNhdGlvbnMgaW4gYnJlYXN0IGNhbmNlciBjZWxsczsgY2l0YXRpb25fYXV0aG9yPUNob2UgIE0uIEsuOyBjaXRhdGlvbl9hdXRob3I9SG9uZyAgQy4gUC47IGNpdGF0aW9uX2F1dGhvcj1QYXJrICBKLjsgY2l0YXRpb25fYXV0aG9yPVNlbyAgUy4gSC47IGNpdGF0aW9uX2F1dGhvcj1Sb2ggIFQuIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTg7ICBjaXRhdGlvbl9wYWdlcz00NzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYWNjZXNzaWJsZSBjaHJvbWF0aW4gcmVnaW9ucyB1c2luZyBTb25vLVNlcTsgY2l0YXRpb25fYXV0aG9yPUF1ZXJiYWNoICBSLiBLLjsgY2l0YXRpb25fYXV0aG9yPUV1c2tpcmNoZW4gIEcuOyBjaXRhdGlvbl9hdXRob3I9Um96b3dza3kgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuU2NpLlUgUyBBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTE0OTI2LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaElQLXNlcSBndWlkZWxpbmVzIGFuZCBwcmFjdGljZXMgb2YgdGhlIEVOQ09ERSBhbmQgbW9kRU5DT0RFIGNvbnNvcnRpYTsgY2l0YXRpb25fYXV0aG9yPUxhbmR0ICBTLiBHLjsgY2l0YXRpb25fYXV0aG9yPU1hcmlub3YgIEcuIEsuOyBjaXRhdGlvbl9hdXRob3I9S3VuZGFqZSAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xODEzLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWVwIHByb3Rlb21lIGFuZCB0cmFuc2NyaXB0b21lIG1hcHBpbmcgb2YgYSBodW1hbiBjYW5jZXIgY2VsbCBsaW5lOyBjaXRhdGlvbl9hdXRob3I9TmFnYXJhaiAgTi47IGNpdGF0aW9uX2F1dGhvcj1XaXNuaWV3c2tpICBKLiBSLjsgY2l0YXRpb25fYXV0aG9yPUdlaWdlciAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTQ4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0MjogYWNjdXJhdGUgYWxpZ25tZW50IG9mIHRyYW5zY3JpcHRvbWVzIGluIHRoZSBwcmVzZW5jZSBvZiBpbnNlcnRpb25zLCBkZWxldGlvbnMgYW5kIGdlbmUgZnVzaW9uczsgY2l0YXRpb25fYXV0aG9yPUtpbSAgRC47IGNpdGF0aW9uX2F1dGhvcj1QZXJ0ZWEgIEcuOyBjaXRhdGlvbl9hdXRob3I9VHJhcG5lbGwgIEMuOyBjaXRhdGlvbl9hdXRob3I9UGltZW50ZWwgIEguOyBjaXRhdGlvbl9hdXRob3I9S2VsbGV5ICBSLjsgY2l0YXRpb25fYXV0aG9yPVNhbHpiZXJnICBTLiBMLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0aW5jdCBhbmQgcHJlZGljdGl2ZSBoaXN0b25lIGx5c2luZSBhY2V0eWxhdGlvbiBwYXR0ZXJucyBhdCBwcm9tb3RlcnMsIGVuaGFuY2VycywgYW5kIGdlbmUgYm9kaWVzOyBjaXRhdGlvbl9hdXRob3I9UmFqYWdvcGFsICBOLjsgY2l0YXRpb25fYXV0aG9yPUVybnN0ICBKLjsgY2l0YXRpb25fYXV0aG9yPVJheSAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIChCZXRoZXNkYSk7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTIwNTEtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVotRE5BLWZvcm1pbmcgc2lsZW5jZXIgaW4gdGhlIGZpcnN0IGV4b24gcmVndWxhdGVzIGh1bWFuIEFEQU0tMTIgZ2VuZSBleHByZXNzaW9uOyBjaXRhdGlvbl9hdXRob3I9UmF5ICBCLiBLLjsgY2l0YXRpb25fYXV0aG9yPURoYXIgIFMuOyBjaXRhdGlvbl9hdXRob3I9U2hha3lhICBBLjsgY2l0YXRpb25fYXV0aG9yPVJheSAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBvZiBzdXBlcmhlbGljYWwgdG9yc2lvbiBieSBBVFAtZGVwZW5kZW50IGNocm9tYXRpbiByZW1vZGVsaW5nIGFjdGl2aXRpZXM7IGNpdGF0aW9uX2F1dGhvcj1IYXZhcyAgSy47IGNpdGF0aW9uX2F1dGhvcj1GbGF1cyAgQS47IGNpdGF0aW9uX2F1dGhvcj1QaGVsYW4gIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTExMzMtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNXSS9TTkYgY2hyb21hdGluIHJlbW9kZWxpbmcgcmVxdWlyZXMgY2hhbmdlcyBpbiBETkEgdG9wb2xvZ3k7IGNpdGF0aW9uX2F1dGhvcj1HYXZpbiAgSS47IGNpdGF0aW9uX2F1dGhvcj1Ib3JuICBQLiBKLjsgY2l0YXRpb25fYXV0aG9yPVBldGVyc29uICBDLiBMLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz05Ny0xMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gZm9ybXMgYW5kIHJlbW9kZWxzIHN1cGVyY29pbGluZyBkb21haW5zIHVuZm9sZGluZyBsYXJnZS1zY2FsZSBjaHJvbWF0aW4gc3RydWN0dXJlczsgY2l0YXRpb25fYXV0aG9yPU5hdWdodG9uICBDLjsgY2l0YXRpb25fYXV0aG9yPUF2bG9uaXRpcyAgTi47IGNpdGF0aW9uX2F1dGhvcj1Db3JsZXNzICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbi1kZXBlbmRlbnQgZHluYW1pYyBzdXBlcmNvaWxpbmcgaXMgYSBzaG9ydC1yYW5nZSBnZW5vbWljIGZvcmNlOyBjaXRhdGlvbl9hdXRob3I9S291emluZSAgRi47IGNpdGF0aW9uX2F1dGhvcj1HdXB0YSAgQS47IGNpdGF0aW9uX2F1dGhvcj1CYXJhbmVsbG8gIEwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFN0cnVjdC4gTW9sIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0zOTYtNDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3RpdmUgcHJvbW90ZXJzIGdpdmUgcmlzZSB0byBmYWxzZSBwb3NpdGl2ZSAnUGhhbnRvbSBQZWFrcycgaW4gQ2hJUC1zZXEgZXhwZXJpbWVudHM7IGNpdGF0aW9uX2F1dGhvcj1KYWluICBELjsgY2l0YXRpb25fYXV0aG9yPUJhbGRpICBTLjsgY2l0YXRpb25fYXV0aG9yPVphYmVsICBBLjsgY2l0YXRpb25fYXV0aG9yPVN0cmF1YiAgVC47IGNpdGF0aW9uX2F1dGhvcj1CZWNrZXIgIFAuIEIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTY5NTktNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcGFjdCBvZiBhcnRpZmFjdCByZW1vdmFsIG9uIENoSVAgcXVhbGl0eSBtZXRyaWNzIGluIENoSVAtc2VxIGFuZCBDaElQLWV4byBkYXRhOyBjaXRhdGlvbl9hdXRob3I9Q2Fycm9sbCAgVC4gUy47IGNpdGF0aW9uX2F1dGhvcj1MaWFuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1TYWxhbWEgIFIuOyBjaXRhdGlvbl9hdXRob3I9U3RhcmsgIFIuOyBjaXRhdGlvbl9hdXRob3I9ZGUgU2FudGlhZ28gIEkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udCBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBaLUROQSwgYSBsZWZ0LWhhbmRlZCBkb3VibGUgaGVsaWNhbCBETkEgaXMgc3RydWN0dXJhbGx5IGRpZmZlcmVudCBmcm9tIHRoZSBtb3N0IGFidW5kYW50IEItRE5BLiBaLUROQSBoYXMgYmVlbiBrbm93biB0byBwbGF5IGEgc2lnbmlmaWNhbnQgcm9sZSBpbiB0ciIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjIzNjE4MSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGR545ghQUYrF8/0bKl4bkFFZd4VFQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f477%2f2236181" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f477%2f2236181" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=79980" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&amp;Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/5">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200778">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200778">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200780">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200780">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200786">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200786">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200801">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200801">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200829">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200829">Conflic of Interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200831">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200831">Accession number</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200833">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200833">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200835">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200835">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/5/467/2236168">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/5/487/2236185">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">So-I. Shin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Life Sciences</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=So-I.+Shin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shin S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shin S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">So-I. Shin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seokjin Ham</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Life Sciences</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seokjin+Ham">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ham S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ham S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seokjin Ham</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jihwan Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div>Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jihwan+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Park J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jihwan Park</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seong Hye Seo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Life Sciences</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seong+Hye+Seo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seo S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seo S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seong Hye Seo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chae Hyun Lim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div>Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chae+Hyun+Lim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lim C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lim C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chae Hyun Lim</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hyeongrin Jeon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Life Sciences</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hyeongrin+Jeon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeon H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeon H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hyeongrin Jeon</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jounghyun Huh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Life Sciences</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jounghyun+Huh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huh J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huh J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jounghyun Huh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tae-Young Roh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Life Sciences</div><div class="aff"><div class="label -label"><sup>2</sup></div>Division of Integrative Biosciences &amp; Biotechnology, Pohang University of Science and Technology (POSTECH), Pohang, Republic of Korea</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw031-cor1" data-open="dsw031-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Tel. +82 54-279-2350. Fax. +82 54-279-8179. Email: <a href="mailto:tyroh@postech.edu" target="_blank">tyroh@postech.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tae-Young+Roh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Roh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Roh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tae-Young Roh</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw031-FM1"><div class="fn"><div class="footnote-content"><sup>†</sup>Equal contribution first authors.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw031-FM2"><div class="fn"><div class="footnote-content">Edited by Dr Minoru Ko</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 477–486, <a href='https://doi.org/10.1093/dnares/dsw031'>https://doi.org/10.1093/dnares/dsw031</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 July 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 September 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 June 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                            
                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>So-I. Shin, Seokjin Ham, Jihwan Park, Seong Hye Seo, Chae Hyun Lim, Hyeongrin Jeon, Jounghyun Huh, Tae-Young Roh;  Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome, <em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 477–486, <a href='https://doi.org/10.1093/dnares/dsw031'>https://doi.org/10.1093/dnares/dsw031</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2236181?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236181?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236181?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236181?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236181?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236181?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236181?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Z-DNA-forming%20sites%20identified%20by%20ChIP-Seq%20are%20associated%20with%20actively%20transcribed%20regions%20in%20the%20human%20genome&amp;publicationDate=2016-07-03&amp;volumeNum=23&amp;issueNum=5&amp;author=Shin%2C%20So-I.%3B%20Ham%2C%20Seokjin&amp;startPage=477&amp;endPage=486&amp;contentId=10.1093%2Fdnares%2Fdsw031&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f477%2f2236181" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200778 id="87200778" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Z-DNA, a left-handed double helical DNA is structurally different from the most abundant B-DNA. Z-DNA has been known to play a significant role in transcription and genome stability but the biological meaning and positions of Z-DNA-forming sites (ZFSs) in the human genome has not been fully explored. To obtain genome-wide map of ZFSs, Zaa with two Z-DNA-binding domains was used for ChIP-Seq analysis. A total of 391 ZFSs were found and their functions were examined <em>in vivo</em>. A large portion of ZFSs was enriched in the promoter regions and contain sequences with high potential to form Z-DNA. Genes containing ZFSs were occupied by RNA polymerase II at the promoters and showed high levels of expression. Moreover, ZFSs were significantly related to active histone marks such as H3K4me3 and H3K9ac. The association of Z-DNA with active transcription was confirmed by the reporter assay system. Overall, our results suggest that Z-DNA formation depends on chromatin structure as well as sequence composition, and is associated with active transcription in human cells. The global information about ZFSs positioning will provide a useful resource for further understanding of DNA structure-dependent transcriptional regulation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Z-DNA">Z-DNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Z-DNA-binding protein&quot;">Z-DNA-binding protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;human genome&quot;">human genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ChIP-Seq">ChIP-Seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;active transcription&quot;">active transcription</a></div>                            <h2 scrollto-destination=87200780 id="87200780" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Z-DNA is a structural form of DNA that has a left-handed double helix and a zigzagging sugar-phosphate backbone. The bases in Z-DNA alternate between an <em>anti</em> and <em>syn</em> orientation about the N-glycosidic bond, whereas in B-DNA, they uniformly have an <em>anti</em>-conformation. Z-DNA easily occurs in alternating purine-pyrimidine sequences, with those sequences’ propensity for forming Z-DNA in the following order: d(GC)n &gt; d(CA)n &gt; d(CGGG)n &gt; d(AT)n.<a reveal-id="dsw031-B1 dsw031-B2 dsw031-B3" data-open="dsw031-B1 dsw031-B2 dsw031-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> Z-DNA formation requires more energy than B-DNA formation, but Z-DNA can be stable under particular conditions, including those with high salt concentrations, divalent and polyvalent ions, dehydrating agents, and DNA-specific conditions such as the presence of chemically modified bases and negative supercoiling.<a reveal-id="dsw031-B4" data-open="dsw031-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw031-B5" data-open="dsw031-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a></p><p>A family of proteins has the ability to bind Z-DNA through a Z-DNA-binding domain. One such protein is the human double-stranded RNA adenosine deaminase (hADAR1), which catalyzes the deamination of adenosine to inosine in double-stranded RNA. The N-terminal region of this enzyme has a Z-DNA-binding domain composed of Zα and Zβ motifs. Zα belongs to the winged helix-turn-helix family of proteins and recognizes <em>syn</em>-conformed purine residues, meaning it binds specifically to Z-DNA, whereas Zβ does not.<a reveal-id="dsw031-B6" data-open="dsw031-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Za (residues 133–209) binds to Z-DNA better than Zα (residues 121–197). Zaa can be generated by replacing Zβ with Zα, resulting in a much higher affinity and specificity to Z-DNA than Zα.<a reveal-id="dsw031-B7" data-open="dsw031-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw031-B8" data-open="dsw031-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Za and Zaa have been widely used as probes for Z structure <em>in vitro</em> and <em>in vivo</em>. Motifs similar to the Zα of hADAR1 are also present in other Z-DNA-binding proteins, such as E3L and DLM1, and are important for the proteins’ biological functions.<a reveal-id="dsw031-B9" data-open="dsw031-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw031-B10" data-open="dsw031-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Because Za of hADAR1 has a strong affinity to Z-DNA and its characteristics have been extensively studied, Za is useful for identifying Z-DNA-forming sites in cells.</p><p>In human cells, Z-DNA-forming regions (ZDRs) have been predicted genome-wide by the computer programs Z-hunt and Z-catcher. Potential ZDRs are enriched near the transcriptional start sites (TSSs) of genes.<a reveal-id="dsw031-B11" data-open="dsw031-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw031-B12" data-open="dsw031-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> This suggests that Z-DNA functions in transcriptional regulation, and indeed, RNA polymerase II accumulates local negative supercoiling behind it as it moves along the DNA strand during transcription, creating a favourable environment for Z-DNA formation.<a reveal-id="dsw031-B13" data-open="dsw031-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> In addition, experiments show that Z-DNA formation, as detected by a Z-DNA antibody, increases with the expression level of the <em>c-myc</em> gene,<a reveal-id="dsw031-B14" data-open="dsw031-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and Z-DNA formation, enhanced by the BAF complex, plays a role in stabilizing open chromatin, giving RNA polymerase II access to the <em>CSF1</em> gene.<a reveal-id="dsw031-B15" data-open="dsw031-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> In a similar manner, transcription of the <em>HO-1</em> gene is increased by Z-DNA formation.<a reveal-id="dsw031-B16" data-open="dsw031-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Furthermore, the incorporation of Z-DNA into nucleosomes is unfavourable, resulting in the promoter region Z-DNA regulating transcription by establishing the boundaries of neighboring nucleosomes and producing an open chromatin state.<a reveal-id="dsw031-B17" data-open="dsw031-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> In addition, Z-DNA is involved in stimulating homologous recombination, protecting the genome, and genetic instability.<a reveal-id="dsw031-B18 dsw031-B19 dsw031-B20" data-open="dsw031-B18 dsw031-B19 dsw031-B20" class="link link-ref link-reveal xref-bibr"><sup>18–20</sup></a></p><p>Recently, 186 Z-DNA hotspots in human cells were identified using Zα<sub>ADAR1</sub> as a probe in an <em>in vitro</em> chromatin affinity precipitation (ChAP)-Sanger sequencing experiment.<a reveal-id="dsw031-B12" data-open="dsw031-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Among them, 46 hotspots were located in the centromere and were correlated with high densities of single nucleotide polymorphisms (SNPs), a finding that was inconsistent with the <em>in silico</em> predictions of ZDRs being located mostly in TSSs.</p><p>Because ZFSs have never been explored at the human genome level by high-throughput analysis, it is difficult to completely understand the biological functions of Z-DNA. Sanger sequencing technique is low throughput, making the interpretation of the sparse data difficult. In order to overcome this limitation, we used chromatin immunoprecipitation with Zaa that consists of two copies of Za, followed by next-generation sequencing (ChIP-Seq). This method provided information on ZFSs in the human genome at high resolution and coverage. We found ZFSs with high confidence, and determined their genomic and epigenetic features. Our results support the positive correlation between Z-DNA formation and active transcription in human cells.</p>                            <h2 scrollto-destination=87200786 id="87200786" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200787 id="87200787" class="section-title" data-legacy-id=SEC2.1>2.1. Cell culture and the expression of Zaa</h3>
<p>HeLa cells, a human cervix carcinoma cell line, were cultured at 37 °C with 5% CO<sub>2</sub> in DMEM media containing 10% FBS, 50 U/ml penicillin, and 50 μg/ml streptomycin. For transfection, Zaa were amplified by PCR using Zaa-Fok as a template, generated by Mulholland <em>et al.</em><a reveal-id="dsw031-B21" data-open="dsw031-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> PCR products were digested with BclI and XbaI, and cloned into the BamHI/XbaI site of pEF1α, in which a FLAG tag was added. The SV40 nuclear localization signal (NLS) sequence was generated as described<a reveal-id="dsw031-B16" data-open="dsw031-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> and inserted into the XmaI/AscI site of pEF1α, producing pEF1α-Zaa. pEF1α-Zaa was transfected into HeLa cells using GeneJuice (Novagen, Germany).</p>                            <h3 scrollto-destination=87200789 id="87200789" class="section-title" ><em>2.2. In vitro</em> Z-DNA cleavage assay</h3>
<p>pET28a-Fok was constructed by inserting a catalytic domain of <em>Fok</em>I endonuclase into the HindIII/XhoI site of pET28a and a (Gly<sub>4</sub>Ser)<sub>3</sub> linker was added upstream of Fok by cloning into the EcoRI/HindIII site, as described in Mulholland <em>et al</em>.<a reveal-id="dsw031-B21" data-open="dsw031-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and Kim <em>et al.</em><a reveal-id="dsw031-B22" data-open="dsw031-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> For generation of pET28aZa-Fok and pET28aZaa-Fok, Za or Zaa was placed in front of the (Gly<sub>4</sub>Ser)<sub>3</sub> linker by inserting Za or Zaa into the NheI/EcoRI site in pET28a-Fok. BL21 (DE3) bacterial cells containing pET28a-Fok, pET28a-Za-Fok, or pET28a-Zaa-Fok were grown at 37 °C and induced with 0.1 mM IPTG and grown for 12 hr at 18 °C. Fok, Za-Fok, and Zaa-Fok were purified using His-Bind resin (Novagen) charged with Ni<sup>2+</sup>. Purified recombinant proteins were dialyzed and analyzed using SDS-polyacrylamide gel electrophoresis followed by Coomassie Blue staining. Aliquots were stored at -80 °C. pDHg16 and pDPL6 were obtained from Addgene (USA). The supercoiled form of pDHg16 was purified using the LaboPass Mini Kit (Cosmo Genetech, Korea). The linear form was prepared by digesting supercoiled pDHg16 with PstI. For the <em>in vitro</em> Z-DNA cleavage assay, supercoiled or linear plasmid was incubated with Fok, Za-Fok, or Zaa-Fok in digestion buffer (10 mM Tris-Cl [pH 8.0], 50 mM KCl, 1 mM DTT, 2.5% glycerol and 0.05% NP40) at 22 °C. After 20 min, MgCl<sub>2</sub> was added to a final concentration of 10 mM, and the reaction was further incubated for 2 hr. Fok, Za-Fok, or Zaa-Fok was inactivated by heat treatment at 50 °C for 30 min. The supercoiled plasmid DNA was digested with PstI for 1 hr at 37 °C and analyzed by gel electrophoresis in 1% agarose gel. For a generation of pDPL6-ZFSs and pDPL6-negative, predicted short ZDR sequences inside ZFSs or a sequence without potential to form Z-DNA was inserted into the XbaI/SalI site of pDPL6. The resulting pDPL6-ZDRs and pDPL6-negative were used for <em>in vitro</em> Z-DNA cleavage assay.</p>                            <h3 scrollto-destination=87200791 id="87200791" class="section-title" data-legacy-id=SEC2.3>2.3. Immunofluorescence analysis and chromatin immunoprecipitation (ChIP)</h3>
<p>The expression of Zaa in HeLa cells was monitored by immunofluorescence analysis. Forty hours after transfection, HeLa cells were fixed in 4% paraformaldehyde for 10 min at room temperature and then permeabilized with 0.1% Triton X-100 for another 10 min at room temperature. Cells were incubated with blocking solution (0.1% BSA in PBS) for 1 hr at room temperature and sequentially incubated with FLAG M2 antibody in blocking buffer for 2 hr at 37 °C, followed by incubation with Dylight 488–labelled secondary antibody (Abcam, UK) for 1 hr at 37 °C. Nuclei were stained with Hoechst dye, and samples were observed using the Olympus FluoView 1200 confocal microscope. ChIP was performed as described<a reveal-id="dsw031-B23" data-open="dsw031-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> with small changes. Briefly, transfected cells were cross-linked with 1% formaldehyde for 10 min at room temperature. Cell fixation was stopped by adding 2.5 M glycine to a final concentration of 0.1375 M and incubating for 5 min. After, cells were washed twice with cold PBS and collected. The cell nuclei were extracted with buffer 1 (10 mM HEPES [pH 6.5], 0.25% Triton X-100, 10 mM EDTA, 0.5 mM EGTA, and 1 mM PMSF) and buffer 2 (10 mM HEPES [pH 6.5], 200 mM NaCl, 1 mM EDTA, 0.5 mM EGTA, and 1 mM PMSF), and isolated nuclei pellets were resuspended in sonication buffer (50 mM HEPES [pH 7.9], 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% Na-deoxycholate, 0.1% SDS, and 1× protease inhibitor cocktail). Chromatin with a range from 100 to 300 bp was prepared by sonication and subjected to immunoprecipitation with 2 μg of FLAG M2 (Sigma, USA), normal mouse IgG (Santa Cruz, USA), or RNA polymerase II (8WG16, Millipore, USA) antibodies and protein A/G magnetic beads (Thermo, USA). One percent of sonicated chromatin was reserved as input before immunoprecipitation. Proteinase K was added to ChIP and input samples, and the chromatin was reverse-crosslinked by incubating for 10 hr at 65 °C in buffer 4 (10 mM Tris-Cl [8.0], 1 mM EDTA, 0.5% SDS). ChIPed DNA and input DNA was purified and used for PCR or sequencing library construction. The primers for Zaa ChIP-qPCR were shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h3 scrollto-destination=87200793 id="87200793" class="section-title" data-legacy-id=SEC2.4>2.4. ChIP-Seq</h3>
<p>The sequencing library was prepared as described<a reveal-id="dsw031-B24" data-open="dsw031-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> with a few modifications. ChIPed DNA and input DNA fragments were end-repaired, dA tailed, and ligated with the genomic DNA adapters provided by Illumina (USA). Adapter-ligated DNA fragments were purified and amplified by PCR. Gel extraction was performed to isolate 200–400-bp PCR products. Libraries were sequenced using the Illumina Genome Analyzer IIx and HiSeq 2500. Sequence alignment was performed using Bowtie2 (v2.25) on the human genome (hg18). The sequencing data of Zaa rep1, rep2, rep3, input and IgG were deposited in the NCBI GEO database (GSE71682). The overall scheme of ChIP-Seq is shown in <a reveal-id="dsw031-F2" data-open="dsw031-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C.</p>                            <h3 scrollto-destination=87200795 id="87200795" class="section-title" data-legacy-id=SEC2.5>2.5. Data analysis</h3>
<p>The ChIP-Seq data was summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. Uniquely mappable reads were used for the following computational analyses. Using IgG data as a control, Zaa peaks on each Zaa replicate were detected by SPP (v.1.10.1) with the following options: -x:-500:85, -npeak = 300,000. Input DNA was also used as a control but IgG data was applied to further analysis for practical comparison purpose.<a reveal-id="dsw031-B25" data-open="dsw031-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Reproducible Zaa peaks between replicates were determined by Irreproducible Discovery Rate (IDR) analysis.<a reveal-id="dsw031-B26" data-open="dsw031-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Self-consistent peaks between two pseudoreplicates (half of mapped reads in each replicate) were found using IDR ≤ 0.01. Peaks consistent between pooled-pseudoreplicates (half of mapped reads pooled across all replicates) were found using IDR ≤ 0.001. In addition, peaks consistent between three pairs of Zaa replicates were found using IDR ≤ 0.05. Of three consistent peak numbers, the maximum peak number was used as a cutoff. With this cutoff, top-ranked peaks were selected in the pooled replicates. Then, the Zaa peaks were examined if they were overlapped with frequently overrepresented random peaks called blacklisted regions. The blacklist detected in the human genome was downloaded from the website (<a class="link link-uri" href="https://sites.google.com/site/anshulkundaje/projects/blacklists" target="_blank">https://sites.google.com/site/anshulkundaje/projects/blacklists</a>, 26 April 2016, date last accessed). After filtering, the final 391 Zaa peaks were selected as ZFSs and used for further analysis. The Pearson correlation coefficient was calculated by counting reads mapped within ZFSs for every sample and comparing those read counts across samples.</p><p>Genes containing ZFSs near the transcription start site (± 2 kb) were subjected to a Gene Ontology enrichment analysis by DAVID. Known motifs that were significantly enriched in ZFSs were examined by Homer (v4.8). The presence of ZDRs within the ZFSs were predicted by Z-catcher with two cutoff values for the negative supercoiling density, -0.08 or -0.07. For comparison, 10,000 random peaks were selected arbitrarily with the distribution of length of ZFSs. The supercoiling density value required for Z-DNA formation was calculated for ZFSs and random peak regions. The region 2 kb upstream and downstream from the centre of each ZFS and random peak was divided into 20-bp windows. The supercoiling density value was calculated in a 20-bp sliding window overlapped with 18-bp. The value for each window was plotted.</p><p>For the gene expression analysis, public mRNA-Seq data was downloaded and analyzed.<a reveal-id="dsw031-B27" data-open="dsw031-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Sequence reads were mapped by TopHat v2.0.9 program <a reveal-id="dsw031-B28" data-open="dsw031-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> to the human genome reference sequence (hg18). Quantification and normalization for each transcript were performed using the RPKM method (Reads Per Kilobase of exon per Million aligned sequence tags). The colocalization of ZFSs with RNA polymerase II binding sites and histone modifications was examined using the following ChIP-Seq datasets; RNA Polymerase II (SRR349806, SRR349811, and SRR765748) and histones H3K4me3, H3K9ac, H3K9me3, H3K27me3, H3K36me3, and H3K79me3 (SRP002252). The heatmap was drawn by calculating the normalized read counts for 50-bp windows within ZFSs and 3 kb upstream and downstream.</p>                            <h3 scrollto-destination=87200799 id="87200799" class="section-title" data-legacy-id=SEC2.6>2.6. Luciferase reporter assay</h3>
<p>For the <em>in vitro</em> Z-DNA cleavage assay, predicted short ZDRs inside ZFSs and one negative control were selected and cloned into the NheI/XhoI site of pGL3-HS containing minimal heat-shock promoter.<a reveal-id="dsw031-B29" data-open="dsw031-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The resulting pGL3-HS-ZFSs or pGL3-HS-negative were transfected into HeLa cells and luciferase reporter assay was performed using Dual-Luciferase Reporter Assay System (Promega, USA). Luciferase activities of each pGL3-HS-ZFSs and pGL3-HS-negative were normalized to those of pGL3-HS and fold change of relative luciferase activity was calculated by dividing the normalized luciferase activity in pGL3-HS-ZFSs by the normalized luciferase activity of pGL3-HS-negative.</p>                            <h2 scrollto-destination=87200801 id="87200801" class="section-title" data-legacy-id=SEC3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87200802 id="87200802" class="section-title" data-legacy-id=SEC3.1>3.1. Recognition of Z-DNA structure by Za and Zaa</h3>
<p>Z-DNA formation and Z-DNA recognition by Za and Zaa were examined using an <em>in vitro</em> cleavage assay. The Z-DNA-binding domains Za and Zaa were fused with the catalytic domain of <em>Fok</em>I, which is a type IIS restriction endonuclease (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>A). Two fusion proteins, Za-Fok and Zaa-Fok, and Fok as a negative control were expressed and purified. A d(CG)<sub>11</sub> insert within pDHg16 is known to form Z-DNA when the plasmid is negatively supercoiled (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>B). The <em>in vitro</em> Z-DNA cleavage assay showed that Za-Fok and Zaa-Fok did not digest the linear form of pDHg16 (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C, top and middle, lane 1–5), whereas they cleaved the d(CG)<sub>11</sub> insert in the negatively supercoiled pDHg16, producing 1.4-kb and 0.8-kb DNA fragments (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C, top and middle, lane 7–11). These results suggest that the binding of Za and Zaa is dependent on DNA conformation rather than sequence composition, which is consistent with previously reported data.<a reveal-id="dsw031-B7" data-open="dsw031-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw031-B8" data-open="dsw031-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> As the concentration of Za-Fok was increased, the two bands produced by Za-Fok and PstI became thicker, but the increase in activity stopped when eight times more Za-Fok than pDHg16 was added (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C, top, lane 10). Because Zaa has a higher affinity to Z-DNA than Za, the two fragments produced by Zaa-Fok and PstI formed clearer, thicker bands, and Zaa-Fok had more activity than Za-Fok, even at a 2:1 molar ratio of Zaa-Fok:pDHg16 (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C, middle, lane 8). Contrary to Za- and Zaa-Fok, digested fragments of supercoiled pDHg16 were not shown with Fok (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C, bottom, lane 8-12). Therefore, we confirmed that Za and Zaa were capable of recognizing and binding Z-DNA and that Zaa showed higher affinity to Z-DNA than Za. </p>                        <a id="87200804" scrollto-destination="87200804"></a>
<div id="dsw031-F1" data-id="dsw031-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f1p.png?Expires=2147483647&Signature=LaXgkgYA3iaf3Ek9kGbBQiJgS8a7ZGD~pJwtdlWG1P6fP8gxORahR6XiqM39yRxxtB~s2bG6pDBrLZPV4NhrHCHirAUFLIFmA5HK~MJrJDg8YN8YsFdM7~9la92DkcZIIqwv6Dfi2WE~97Y-zKCCTCSMDUXWnzkhDXHhq0RLf1k8rf4TT8wH7WeaLxLSnY8ycsOgOBjWuXan8mFcGhnNLhBMLrN4jtpRqKe~0Sp8cgc~VbTktha-eWbxJQx4X81SaL~gdrKR0rPJBUTljUtxd-Yi69KKD~VrVm59DyLucd6JzmVla9qbL-yibzwO4dYkUlKHoBEJ7NfXLWj1vXSwBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of Z-DNA structure by cleavage with Za- and Zaa-Fok. (A) Schematic constructs of recombinant Za-Fok, Zaa-Fok, and Fok with a His tag and (Gly4Ser)3 linker. (B) pDHg16 contains CpG dinucleotide repeats. Arrows indicate ClaI and PstI cleavage sites. (C) The in vitro Z-DNA cleavage assay was analyzed on a 1% agarose gel. Linear and supercoiled pDHg16 was digested by various concentrations of Za-Fok, Zaa-Fok, or Fok. The molar ratio of Za-Fok, Zaa-Fok, or Fok to pDHg16 was 0, 2, 4, 6 and 8:1 (top and middle, lane 1-5, 7-11, bottom, lane 1-5, 8-12). Arrows indicate the resulting fragments produced by Za-Fok or Zaa-Fok, and PstI digestion (top and bottom, lane 1-5, 7-11). For a positive control, Zaa-Fok was used at the same time that Fok was tested (bottom, lane 6 and 13). Lane 6 (top and middle) and 7 (bottom) has DNA molecular mass markers. Lane 12 (top and middle) and 14 (bottom) has the restriction fragments of ClaI and PstI." path-from-xml="dsw031f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200804/dsw031f1p.tiff" path-from-xml="dsw031f1p.tiff" target="_blank">View large</a><a section="87200804" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f1p.png?Expires=2147483647&Signature=MB2FkLk2iSE0LpEHSM9~8w5xL6oahYoj7t4nEtPYx3~28x685QhsgnRw-uFN7YgcqFpFYjot-8bBazRbukt2dZBcVCfmWLGWH9pnB0fcU-K-lB3A-uLJzhsdjDNuSuE5YCjUP0agK84L0ArbQCb-iEUGCuX3I4-FpZys3QsjGP9qqWuD5kdVsKKnPUGp2GV37uCRLoKAIVnDLGrcyztuSbhU6JsxgKEgS3uTiLe5WzB~Bx9P1HqI1QDYLLZFF8xEPCtl-P79Rznnot6w9HoOj1NgTALcVf1y~QGatEogYZuAl3v1wmPF485gQT0OC2N2c-hTYxa8ToGpVkI-3wMHhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200804&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of Z-DNA structure by cleavage with Za- and Zaa-Fok. (A) Schematic constructs of recombinant Za-Fok, Zaa-Fok, and Fok with a His tag and (Gly<sub>4</sub>Ser)<sub>3</sub> linker. (B) pDHg16 contains CpG dinucleotide repeats. Arrows indicate ClaI and PstI cleavage sites. (C) The <em>in vitro</em> Z-DNA cleavage assay was analyzed on a 1% agarose gel. Linear and supercoiled pDHg16 was digested by various concentrations of Za-Fok, Zaa-Fok, or Fok. The molar ratio of Za-Fok, Zaa-Fok, or Fok to pDHg16 was 0, 2, 4, 6 and 8:1 (top and middle, lane 1-5, 7-11, bottom, lane 1-5, 8-12). Arrows indicate the resulting fragments produced by Za-Fok or Zaa-Fok, and PstI digestion (top and bottom, lane 1-5, 7-11). For a positive control, Zaa-Fok was used at the same time that Fok was tested (bottom, lane 6 and 13). Lane 6 (top and middle) and 7 (bottom) has DNA molecular mass markers. Lane 12 (top and middle) and 14 (bottom) has the restriction fragments of ClaI and PstI.</p></div></div><div content-id="dsw031-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f1p.png?Expires=2147483647&Signature=LaXgkgYA3iaf3Ek9kGbBQiJgS8a7ZGD~pJwtdlWG1P6fP8gxORahR6XiqM39yRxxtB~s2bG6pDBrLZPV4NhrHCHirAUFLIFmA5HK~MJrJDg8YN8YsFdM7~9la92DkcZIIqwv6Dfi2WE~97Y-zKCCTCSMDUXWnzkhDXHhq0RLf1k8rf4TT8wH7WeaLxLSnY8ycsOgOBjWuXan8mFcGhnNLhBMLrN4jtpRqKe~0Sp8cgc~VbTktha-eWbxJQx4X81SaL~gdrKR0rPJBUTljUtxd-Yi69KKD~VrVm59DyLucd6JzmVla9qbL-yibzwO4dYkUlKHoBEJ7NfXLWj1vXSwBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of Z-DNA structure by cleavage with Za- and Zaa-Fok. (A) Schematic constructs of recombinant Za-Fok, Zaa-Fok, and Fok with a His tag and (Gly4Ser)3 linker. (B) pDHg16 contains CpG dinucleotide repeats. Arrows indicate ClaI and PstI cleavage sites. (C) The in vitro Z-DNA cleavage assay was analyzed on a 1% agarose gel. Linear and supercoiled pDHg16 was digested by various concentrations of Za-Fok, Zaa-Fok, or Fok. The molar ratio of Za-Fok, Zaa-Fok, or Fok to pDHg16 was 0, 2, 4, 6 and 8:1 (top and middle, lane 1-5, 7-11, bottom, lane 1-5, 8-12). Arrows indicate the resulting fragments produced by Za-Fok or Zaa-Fok, and PstI digestion (top and bottom, lane 1-5, 7-11). For a positive control, Zaa-Fok was used at the same time that Fok was tested (bottom, lane 6 and 13). Lane 6 (top and middle) and 7 (bottom) has DNA molecular mass markers. Lane 12 (top and middle) and 14 (bottom) has the restriction fragments of ClaI and PstI." path-from-xml="dsw031f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200804/dsw031f1p.tiff" path-from-xml="dsw031f1p.tiff" target="_blank">View large</a><a section="87200804" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f1p.png?Expires=2147483647&Signature=MB2FkLk2iSE0LpEHSM9~8w5xL6oahYoj7t4nEtPYx3~28x685QhsgnRw-uFN7YgcqFpFYjot-8bBazRbukt2dZBcVCfmWLGWH9pnB0fcU-K-lB3A-uLJzhsdjDNuSuE5YCjUP0agK84L0ArbQCb-iEUGCuX3I4-FpZys3QsjGP9qqWuD5kdVsKKnPUGp2GV37uCRLoKAIVnDLGrcyztuSbhU6JsxgKEgS3uTiLe5WzB~Bx9P1HqI1QDYLLZFF8xEPCtl-P79Rznnot6w9HoOj1NgTALcVf1y~QGatEogYZuAl3v1wmPF485gQT0OC2N2c-hTYxa8ToGpVkI-3wMHhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200804&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of Z-DNA structure by cleavage with Za- and Zaa-Fok. (A) Schematic constructs of recombinant Za-Fok, Zaa-Fok, and Fok with a His tag and (Gly<sub>4</sub>Ser)<sub>3</sub> linker. (B) pDHg16 contains CpG dinucleotide repeats. Arrows indicate ClaI and PstI cleavage sites. (C) The <em>in vitro</em> Z-DNA cleavage assay was analyzed on a 1% agarose gel. Linear and supercoiled pDHg16 was digested by various concentrations of Za-Fok, Zaa-Fok, or Fok. The molar ratio of Za-Fok, Zaa-Fok, or Fok to pDHg16 was 0, 2, 4, 6 and 8:1 (top and middle, lane 1-5, 7-11, bottom, lane 1-5, 8-12). Arrows indicate the resulting fragments produced by Za-Fok or Zaa-Fok, and PstI digestion (top and bottom, lane 1-5, 7-11). For a positive control, Zaa-Fok was used at the same time that Fok was tested (bottom, lane 6 and 13). Lane 6 (top and middle) and 7 (bottom) has DNA molecular mass markers. Lane 12 (top and middle) and 14 (bottom) has the restriction fragments of ClaI and PstI.</p></div></div>                            <h3 scrollto-destination=87200805 id="87200805" class="section-title" data-legacy-id=SEC3.2>3.2. Identification of ZFSs in human genome</h3>
<p>The specificity of selectivity of antibody was examined. Zaa was overexpressed and localized mostly in the nucleus in HeLa cells. Confocal microscopic analysis determined that the expressed Zaa accumulated in the nuclei, confirming a high level of expression and nuclear localization of Zaa. Additionally, non-specific protein bands were not observed using FLAG antibody, indicating that the antibody is highly specific to Zaa (<a reveal-id="dsw031-F2" data-open="dsw031-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>A and B). Even though some commercial antibodies against Z-DNA are available and several groups have succeeded in detecting Z-DNA in human cells,<a reveal-id="dsw031-B14" data-open="dsw031-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw031-B30" data-open="dsw031-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> it is hard to identify sites enriched for Z-DNA-antibody binding in HeLa cells (data not shown), likely because the intensive sonication required for fragmenting chromatin may cause Z-DNA more unstable. Therefore, we used Z-DNA-binding proteins as probes for detecting Z-DNA-forming sites. ChIP-Seq experiments were performed with Zaa-transfected HeLa cells (<a reveal-id="dsw031-F2" data-open="dsw031-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C) and sequencing data were obtained from three biological replicates and two negative controls, IgG and Input DNA. To identify confident ZFSs, IDR framework analysis was applied to find Zaa peaks consistent between replicates. The numbers of self-consistent peaks between two pseudoreplicates were 1,024 for Zaa repl1, 520 for Zaa rep2, and 814 for Zaa rep3 (IDR ≤ 0.01) and the number of peaks consistent between pooled-pseudoreplicates is 649 (IDR ≤ 0.001). Additionally, the numbers of peaks consistent between pairs of Zaa replicates are 197 between rep1 and rep2, 229 between rep1 and rep3, and 427 between rep2 and rep3 (IDR ≤ 0.05). Of these three, the maximum peak number (427) was used as a cutoff value. Top 427 Zaa peaks in pooled Zaa replicates were further processed with blacklist regions and finally 391 regions were selected as ZFSs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The list of 391 ZFSs and their surrounding genomic features is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. We confirmed that the correlations of mapped reads were higher between Zaa replicate pairs than between unrelated sample pairs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The Pearson’s correlation coefficients between Zaa replicates were shown; 0.962 for rep1-rep2, 0.774 for rep1-rep3, 0.814 for rep2-rep3, 0.375 for rep1-IgG, 0.379 for rep2-IgG, and 0.342 for rep3-IgG, 0.385 for rep1-Input, 0.421 for rep2-Input, 0.295 for rep3-Input, and 0.760 for IgG-Input. The correlation coefficients are much higher between Zaa replicates than among pairs of replicate-control. As shown in the comprehensive genome-wide view of ZFSs, the overall ZFSs were distributed all over the chromosome (<a reveal-id="dsw031-F2" data-open="dsw031-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>D). As examples, sequence tag distributions of replicates, input and IgG are visualized by the UCSC genome browser (<a reveal-id="dsw031-F2" data-open="dsw031-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>E, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). </p>                        <a id="87200807" scrollto-destination="87200807"></a>
<div id="dsw031-F2" data-id="dsw031-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f2p.png?Expires=2147483647&Signature=OfMzP~M5WhNLnaQhdyIHGu-wuk9XwsYchzPqSZQCGDKxehXMjomz2f6kEHClesJnCkMrzl8HnyowRP3vwD-Ec71UvtbOzuTLlXhQlzyIh2loslPwlQB6giUU~ozi0al19sY6wOjs~~YwkRisLMBBaDCQcQF2RkbMlsXR7u2lPfJds~Vf-a3A17iYrbzpFciA2mtXrRfHj47rxhYQgb5xjs7zGx17916zrXGKVpGruyoRbYmt7XItclawC8HG0N~BrAsUwQRL2fs~JUSTAmzzqZFzaz4D~YFn7pNxXg6fFTTZDxOkS23GLyTIdhB3qx8AuDpGkI7Pa97q7uDslFbgfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide mapping of Z-DNA using Zaa. (A) Schematic of Zaa with FLAG tag and SV40NLS. (B) The localization of Zaa in HeLa cells was observed by confocal microscopy. Zaa was labeled with FLAG antibody, followed by a secondary antibody conjugated to green-fluorescent dye. Nuclei were stained using Hoechst dye (blue). Pictures are representative of three independent experiments. Scale bars indicate 10 μm. The Western blot image is shown on the right panel. (C) Flowchart illustrating the ChIP-Seq process. Zaa construct was transiently transfected into HeLa cells. (D) Circos plot displaying the genome-wide distribution of genes, ZFSs, and polymerase II (gray bar: RefSeq gene, blue bar: ZFSs, orange bar: RNA polymerase II peak). One representative region in chromosome 1 is shown on the UCSC genome browser. The y-axis of the UCSC genome browser is the normalized read count. Peak regions, ZFS, and predicted ZDRs in ZFSs (cutoff of − 0.08) are indicated as gray bars, blue bars, and pink bars, respectively. (E) Examples of ZFSs are shown. The UCSC genome browser was used to visualize read distributions of three Zaa replicates, IgG, and input in ANKRD11, SRSF6, and SIK1. The PCR primer positions for ChIP validation are marked by big arrow." path-from-xml="dsw031f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200807/dsw031f2p.tiff" path-from-xml="dsw031f2p.tiff" target="_blank">View large</a><a section="87200807" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f2p.png?Expires=2147483647&Signature=rk3Ng8uzQ-a90zUjdV2cEDdhhOLqWotp~JYCnsVNVGcnEBKLo8A4c96R0MoSUt2bWSNc4-Nc5RxL5codtbl8l85vKQrX9KiSl9htXsWWaCSdarDawVHcNKEAKOvyO3A6Yeu46wV-iwL6-ni1b9vpCTdb7Qxfr7EigQvF6K1io0iqJSUwhJSdxHGkwmN8mFR1Fd2GEsuoFJulIuVY1Sd1JhoRhos3-MW8vxVLw7C4VB0fruvtvZwL9QSAGnOQDwy~Jwm80dKSWHCKFwCcSrLsAwySspENrIVFwr7ZbKuFaJO042DVuvWvtQmKktTmpynNdmOqFtB-o04zsLBWDDo9pA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200807&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide mapping of Z-DNA using Zaa. (A) Schematic of Zaa with FLAG tag and SV40NLS. (B) The localization of Zaa in HeLa cells was observed by confocal microscopy. Zaa was labeled with FLAG antibody, followed by a secondary antibody conjugated to green-fluorescent dye. Nuclei were stained using Hoechst dye (blue). Pictures are representative of three independent experiments. Scale bars indicate 10 μm. The Western blot image is shown on the right panel. (C) Flowchart illustrating the ChIP-Seq process. Zaa construct was transiently transfected into HeLa cells. (D) Circos plot displaying the genome-wide distribution of genes, ZFSs, and polymerase II (gray bar: RefSeq gene, blue bar: ZFSs, orange bar: RNA polymerase II peak). One representative region in chromosome 1 is shown on the UCSC genome browser. The y-axis of the UCSC genome browser is the normalized read count. Peak regions, ZFS, and predicted ZDRs in ZFSs (cutoff of − 0.08) are indicated as gray bars, blue bars, and pink bars, respectively. (E) Examples of ZFSs are shown. The UCSC genome browser was used to visualize read distributions of three Zaa replicates, IgG, and input in <em>ANKRD11, SRSF6</em>, and <em>SIK1</em>. The PCR primer positions for ChIP validation are marked by big arrow.</p></div></div><div content-id="dsw031-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f2p.png?Expires=2147483647&Signature=OfMzP~M5WhNLnaQhdyIHGu-wuk9XwsYchzPqSZQCGDKxehXMjomz2f6kEHClesJnCkMrzl8HnyowRP3vwD-Ec71UvtbOzuTLlXhQlzyIh2loslPwlQB6giUU~ozi0al19sY6wOjs~~YwkRisLMBBaDCQcQF2RkbMlsXR7u2lPfJds~Vf-a3A17iYrbzpFciA2mtXrRfHj47rxhYQgb5xjs7zGx17916zrXGKVpGruyoRbYmt7XItclawC8HG0N~BrAsUwQRL2fs~JUSTAmzzqZFzaz4D~YFn7pNxXg6fFTTZDxOkS23GLyTIdhB3qx8AuDpGkI7Pa97q7uDslFbgfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide mapping of Z-DNA using Zaa. (A) Schematic of Zaa with FLAG tag and SV40NLS. (B) The localization of Zaa in HeLa cells was observed by confocal microscopy. Zaa was labeled with FLAG antibody, followed by a secondary antibody conjugated to green-fluorescent dye. Nuclei were stained using Hoechst dye (blue). Pictures are representative of three independent experiments. Scale bars indicate 10 μm. The Western blot image is shown on the right panel. (C) Flowchart illustrating the ChIP-Seq process. Zaa construct was transiently transfected into HeLa cells. (D) Circos plot displaying the genome-wide distribution of genes, ZFSs, and polymerase II (gray bar: RefSeq gene, blue bar: ZFSs, orange bar: RNA polymerase II peak). One representative region in chromosome 1 is shown on the UCSC genome browser. The y-axis of the UCSC genome browser is the normalized read count. Peak regions, ZFS, and predicted ZDRs in ZFSs (cutoff of − 0.08) are indicated as gray bars, blue bars, and pink bars, respectively. (E) Examples of ZFSs are shown. The UCSC genome browser was used to visualize read distributions of three Zaa replicates, IgG, and input in ANKRD11, SRSF6, and SIK1. The PCR primer positions for ChIP validation are marked by big arrow." path-from-xml="dsw031f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200807/dsw031f2p.tiff" path-from-xml="dsw031f2p.tiff" target="_blank">View large</a><a section="87200807" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f2p.png?Expires=2147483647&Signature=rk3Ng8uzQ-a90zUjdV2cEDdhhOLqWotp~JYCnsVNVGcnEBKLo8A4c96R0MoSUt2bWSNc4-Nc5RxL5codtbl8l85vKQrX9KiSl9htXsWWaCSdarDawVHcNKEAKOvyO3A6Yeu46wV-iwL6-ni1b9vpCTdb7Qxfr7EigQvF6K1io0iqJSUwhJSdxHGkwmN8mFR1Fd2GEsuoFJulIuVY1Sd1JhoRhos3-MW8vxVLw7C4VB0fruvtvZwL9QSAGnOQDwy~Jwm80dKSWHCKFwCcSrLsAwySspENrIVFwr7ZbKuFaJO042DVuvWvtQmKktTmpynNdmOqFtB-o04zsLBWDDo9pA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200807&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide mapping of Z-DNA using Zaa. (A) Schematic of Zaa with FLAG tag and SV40NLS. (B) The localization of Zaa in HeLa cells was observed by confocal microscopy. Zaa was labeled with FLAG antibody, followed by a secondary antibody conjugated to green-fluorescent dye. Nuclei were stained using Hoechst dye (blue). Pictures are representative of three independent experiments. Scale bars indicate 10 μm. The Western blot image is shown on the right panel. (C) Flowchart illustrating the ChIP-Seq process. Zaa construct was transiently transfected into HeLa cells. (D) Circos plot displaying the genome-wide distribution of genes, ZFSs, and polymerase II (gray bar: RefSeq gene, blue bar: ZFSs, orange bar: RNA polymerase II peak). One representative region in chromosome 1 is shown on the UCSC genome browser. The y-axis of the UCSC genome browser is the normalized read count. Peak regions, ZFS, and predicted ZDRs in ZFSs (cutoff of − 0.08) are indicated as gray bars, blue bars, and pink bars, respectively. (E) Examples of ZFSs are shown. The UCSC genome browser was used to visualize read distributions of three Zaa replicates, IgG, and input in <em>ANKRD11, SRSF6</em>, and <em>SIK1</em>. The PCR primer positions for ChIP validation are marked by big arrow.</p></div></div>                            <h3 scrollto-destination=87200808 id="87200808" class="section-title" data-legacy-id=SEC3.3>3.3. Validation of ZFSs by Zaa ChIP-qPCR and an <em>in vitro</em> Z-DNA cleavage assay</h3>
<p>In order to validate ZFSs, the existence of predicted ZDRs in ZFSs was first examined using the Z-catcher program. The advantage of Z-catcher is that it can predict ZDRs genome-wide using different σ levels which specify the superhelical density, a proxy for the energy that can cause a B-to-Z transition. About 80% and 50% of ZFSs contained more than one predicted ZDR at the cutoff of σ = −0.08 and −0.07, respectively. d(CG) or d(CA/TG) repeating sequence is well known to form Z-DNA. However, little is known about the Z-DNA formation in other sequences, except for alternating repeat sequences. To compare the efficiency of Z-DNA formation between different sequences, we randomly selected 10 ZFSs for validation and then divided them into two groups based on the number of alternating purine/pyrimidine dinucleotides of predicted ZDRs: an alternating (Pu)(Py) group (≥ 6 dinucleotides) and a non-alternating (Pu)(Py) group (&lt; 6 dinucleotides). <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> lists the genes that were closest to each ZFS and the predicted ZDR sequence. Four negative controls were designed not to contain any Zaa peaks, 5–10 kb away from a ZFS. ChIP-qPCR found that Zaa was significantly enriched in 9 out of 10 ZFSs (90%), relative to the average of the ZFS-negative regions (<em>P</em>-value &lt; 0.05) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A</a></span>, marked by cross). The control IgG had low Zaa enrichment for all ZFSs. There was little difference between the enrichment levels of Zaa in the alternating (Pu)(Py) group and the non-alternating (Pu)(Py) group, even though the predicted ZDR sequences were not identical and could not be directly compared. Among nine Zaa-enriched ZFSs (<em>ANKRD11</em>, <em>ROR</em>, <em>SNX12, SRSF6</em>, TFAP2A, <em>SIK1</em>, <em>HIST2H2AC</em>, <em>STX16</em>, <em>and PIM3</em>), the most enriched ZFSs by Zaa is <em>SIK1</em> whose ZDR does not contain alternating purine/pyrimidine repeat sequences. In addition, <em>STX16</em> and <em>PIM3</em> also show relatively high levels of Zaa enrichment. The enrichment level of Zaa seemed not to be closely correlated with the presence of an alternating purine/pyrimidine repeat sequence. Presumably, alternating purine/pyrimidine repeat sequences might not be the only major factor required to form the Z-DNA structure in cells.</p><p>We next performed an <em>in vitro</em> Z-DNA cleavage assay to confirm whether the predicted ZDRs in ZFSs were able to form Z-DNA in a negatively supercoiled plasmid. Zaa-Fok digested 4 of the 5 predicted ZDRs in the alternating (Pu)(Py) group (pDPL6-<em>ANKRD11</em>, -<em>ROR1</em>, -<em>SNX12</em>, and -<em>SRSF6</em>; each ZDR labeled by its corresponding gene name) and 2 of the 5 predicted ZDRs in the non-alternating (Pu)(Py) group (pDPL6-<em>SIK1</em> and -<em>PIM3</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4B</a></span>). In contrast to the supercoiled forms of the pDPL6-ZDRs, the supercoiled pDPL6-negative was not digested by Zaa-Fok. Therefore, 6 out of the 10 predicted ZDRs (60%) formed the Z-DNA structure in negatively supercoiled plasmids. The intensities of the two bands produced by Zaa-Fok digestion were quantitated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4C</a></span>). The digestion efficiency of Zaa-Fok for each predicted ZDR had a low correlation with the enrichment level of Zaa (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4D</a></span>). For example, pDPL6-<em>SIK1</em> was digested by Zaa-Fok with a similar digestion efficiency as pDPL6-<em>SRSF6</em>, meaning that they had sequences with similar potential to form Z-DNA <em>in vitro</em>. However, the ZFS in <em>SIK1</em> had more Zaa enrichment than the ZFSs in <em>SRSF6</em>, as demonstrated by ChIP-qPCR. This difference between the <em>in vitro</em> Z-DNA cleavage assay and the Zaa ChIP-qPCR might be caused by the ZFSs located in the chromatin context in cells. Contrary to Zaa ChIP-qPCR, only 60% of predicted ZDRs in the ZFSs were confirmed by an <em>in vitro</em> Z-DNA cleavage assay. The remaining 40% of predicted ZDRs were not digested by Zaa-Fok probably because the ZDR sequences <em>per se</em> had a low potential to form Z-DNA and levels of negative supercoiling were restricted <em>in vitro</em>. However, the formation of DNA supercoiling <em>in vivo</em> is affected by chromatin structure related with biological processes so that supercoil density might dynamically change. In the presence of a high degree of negative supercoiling, Z-DNA can be formed even at ZDRs with the relatively low potential to form Z-DNA.</p><p>Consequently, Z-DNA formation in cells may be affected by the dynamic regulation of chromatin structure, such as chromatin remodeling and nucleosome positioning, in addition to the sequence composition and length of the ZDRs. In summary, we confirmed that the enrichment level of Zaa was not simply determined by the existence of known alternating purine/pyrimidine repeat sequences in ZFSs.</p>                            <h3 scrollto-destination=87200812 id="87200812" class="section-title" data-legacy-id=SEC3.4>3.4. Characterization of genomic features of ZFSs</h3>
<p>ZFSs were significantly enriched in the promoter regions of genes (46.2%), in the gene body (29.7%), and in intergenic regions (24.1%). On the other hand, only 1.5% of random peaks were present in promoter regions, and a large portion of random peaks were found in intergenic regions (57.2%; <a reveal-id="dsw031-F3" data-open="dsw031-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A). As demonstrated by the <em>in vitro</em> Z-DNA cleavage assay (<a reveal-id="dsw031-F1" data-open="dsw031-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4B</a></span>), negative supercoiling is required to induce Z-DNA formation. Thus, ZFSs are likely to be negatively supercoiled. We analyzed the distribution of the superhelical density in and around ZFSs as well as random peaks. Because, in Z-catcher, the free energy requirement for B-to-Z transition was translated into the superhelical density (σ) which provides energy for the transition, given sequences which require less supercoiling (closer to zero) have greater Z-DNA forming potentials. As shown in <a reveal-id="dsw031-F3" data-open="dsw031-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>B, the superhelical density required for Z-DNA formation was lower in the centre of ZFSs than random peaks, meaning that ZFSs have concentrations of Z-DNA forming potentials. In addition, the existence of short ZDRs in 391 ZFSs was examined using the Z-catcher. Among ZFSs, 81.0% carried potential ZDRs with the cutoff of −0.08, but only 32.5% of random peaks had potential ZDRs. When the cutoff value was increased to −0.07, 50.3% of ZFSs and 10.0% of random peaks contain predicted ZDRs. However, considering the ratio of ZFS to random peak ratio in frequency at the cut off of −0.07, 5 times more ZFSs had ZDRs than random peaks (Chi-Square <em>P</em>-value: 1.60e-131) whereas ZFSs containing predicted ZDRs are just 2.5 times more than random peaks at the cut off of -0.08 (Chi-Square <em>P</em>-value: 2.88e-87, <a reveal-id="dsw031-F3" data-open="dsw031-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>C). Thus, ZFSs significantly contain sequences with higher potential for forming Z-DNA than random peaks. </p>                        <a id="87200814" scrollto-destination="87200814"></a>
<div id="dsw031-F3" data-id="dsw031-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f3p.png?Expires=2147483647&Signature=FGLFAtgfGMJBGqvT2ffmfsgvSdz5tiCh7EzpgXqo5drAuuLSWXb1-i2~gvPOoKcdTaXO52vQmjHI4X1xxwtkQG-KiqKDOhvCT9HMOEh8Gud0tYGZReK7dNV04U9E-CR8UMUIdmo5q6xgnOkwtIR8S4OSHKuIZkZ7~9JvwvU~y20eTj5jBWgZnSnWS3C8zHTC46feJ9~jJ40OuXOUJImCpqvto45REBmh0jUySlnZYfUJ4UqPal33ExZYqjWyAQq8FcprnystS-lqzjzi9eE3WFbTR4TdFKM9fggW8xY9~oFBiVGIquvXSDPIX-pJKwCdYWg7kiOGDuiC6CEotb5kMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic features at ZFSs. (A) The pie charts show the genomic distribution of ZFSs and 10,000 random peaks. About 46% of ZFSs were found in promoter regions which are defined as within 2 kb of the TSS. (B) The supercoiling density was analyzed as a function of the distance from the peak using Z-catcher. The centre of the peaks is at zero. (C) Peak frequency indicates the number of ZFSs or random peaks that contain predicted ZDRs (black bar). Predicted ZDRs in peak regions were identified using the cutoff of -0.07 or -0.08 in Z-catcher. At -0.08, ZFSs and random peaks containing predicted ZDRs accounted for 81.07% and 32.58% of peaks, respectively. At -0.07, ZFSs and random peaks containing predicted ZDRs accounted for 50.38% and 10.09% of peaks, respectively. A two-sample proportion test was performed using a 99% confidence level. Stars indicate Fisher’s P-value &lt; 2.2e-16. (D) Gene ontology (GO) analysis was performed using the DAVID functional annotation tool with genes that had a ZFS within 2 kb of their TSS. The x-axis indicates −log10 P-value and the y-axis displays the enriched GO terms. (E) Known motif enrichment analysis was carried out using Homer. The top three motifs for transcription factors in ZFSs are shown. Enrichment is shown as −log10 P-value. Letter size indicates the frequency of nucleotides." path-from-xml="dsw031f3p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200814/dsw031f3p.tiff" path-from-xml="dsw031f3p.tiff" target="_blank">View large</a><a section="87200814" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f3p.png?Expires=2147483647&Signature=XN0Cp~CW0YFBLImx7jeeZX80Iy3kr10QC0tRUKAt50U1ph1NoJFtChsMRt87kM9t2MA-pIAQWwkxxKZCf6VnqcQs~AKt8FeRWVhkKL7~2TwVimf1drwYGA~nc6xDKHF8yomiz~YlenjPpnjCwCOKtPWM72y73pMUF3uWqR~9HCxp1EqcHzFiLEPMeLp9bAM83lkPATFhyIXk2q2I8lkNVBnFSF1lhCB383fxIzJpU9IioEPjVpIIBe6HRRdI4B0U7BE4-fFhG85-ydIR-dBpow-ix~rCdnTSr4jQtfA0Vp~La7wIR~XQDCZo9ezHVtaA6ibsnIvCxWfTgvTQaXvW-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200814&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f3p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic features at ZFSs. (A) The pie charts show the genomic distribution of ZFSs and 10,000 random peaks. About 46% of ZFSs were found in promoter regions which are defined as within 2 kb of the TSS. (B) The supercoiling density was analyzed as a function of the distance from the peak using Z-catcher. The centre of the peaks is at zero. (C) Peak frequency indicates the number of ZFSs or random peaks that contain predicted ZDRs (black bar). Predicted ZDRs in peak regions were identified using the cutoff of -0.07 or -0.08 in Z-catcher. At -0.08, ZFSs and random peaks containing predicted ZDRs accounted for 81.07% and 32.58% of peaks, respectively. At -0.07, ZFSs and random peaks containing predicted ZDRs accounted for 50.38% and 10.09% of peaks, respectively. A two-sample proportion test was performed using a 99% confidence level. Stars indicate Fisher’s <em>P</em>-value &lt; 2.2e-16. (D) Gene ontology (GO) analysis was performed using the DAVID functional annotation tool with genes that had a ZFS within 2 kb of their TSS. The x-axis indicates −log10 <em>P</em>-value and the y-axis displays the enriched GO terms. (E) Known motif enrichment analysis was carried out using Homer. The top three motifs for transcription factors in ZFSs are shown. Enrichment is shown as −log10 <em>P</em>-value. Letter size indicates the frequency of nucleotides.</p></div></div><div content-id="dsw031-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f3p.png?Expires=2147483647&Signature=FGLFAtgfGMJBGqvT2ffmfsgvSdz5tiCh7EzpgXqo5drAuuLSWXb1-i2~gvPOoKcdTaXO52vQmjHI4X1xxwtkQG-KiqKDOhvCT9HMOEh8Gud0tYGZReK7dNV04U9E-CR8UMUIdmo5q6xgnOkwtIR8S4OSHKuIZkZ7~9JvwvU~y20eTj5jBWgZnSnWS3C8zHTC46feJ9~jJ40OuXOUJImCpqvto45REBmh0jUySlnZYfUJ4UqPal33ExZYqjWyAQq8FcprnystS-lqzjzi9eE3WFbTR4TdFKM9fggW8xY9~oFBiVGIquvXSDPIX-pJKwCdYWg7kiOGDuiC6CEotb5kMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic features at ZFSs. (A) The pie charts show the genomic distribution of ZFSs and 10,000 random peaks. About 46% of ZFSs were found in promoter regions which are defined as within 2 kb of the TSS. (B) The supercoiling density was analyzed as a function of the distance from the peak using Z-catcher. The centre of the peaks is at zero. (C) Peak frequency indicates the number of ZFSs or random peaks that contain predicted ZDRs (black bar). Predicted ZDRs in peak regions were identified using the cutoff of -0.07 or -0.08 in Z-catcher. At -0.08, ZFSs and random peaks containing predicted ZDRs accounted for 81.07% and 32.58% of peaks, respectively. At -0.07, ZFSs and random peaks containing predicted ZDRs accounted for 50.38% and 10.09% of peaks, respectively. A two-sample proportion test was performed using a 99% confidence level. Stars indicate Fisher’s P-value &lt; 2.2e-16. (D) Gene ontology (GO) analysis was performed using the DAVID functional annotation tool with genes that had a ZFS within 2 kb of their TSS. The x-axis indicates −log10 P-value and the y-axis displays the enriched GO terms. (E) Known motif enrichment analysis was carried out using Homer. The top three motifs for transcription factors in ZFSs are shown. Enrichment is shown as −log10 P-value. Letter size indicates the frequency of nucleotides." path-from-xml="dsw031f3p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200814/dsw031f3p.tiff" path-from-xml="dsw031f3p.tiff" target="_blank">View large</a><a section="87200814" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f3p.png?Expires=2147483647&Signature=XN0Cp~CW0YFBLImx7jeeZX80Iy3kr10QC0tRUKAt50U1ph1NoJFtChsMRt87kM9t2MA-pIAQWwkxxKZCf6VnqcQs~AKt8FeRWVhkKL7~2TwVimf1drwYGA~nc6xDKHF8yomiz~YlenjPpnjCwCOKtPWM72y73pMUF3uWqR~9HCxp1EqcHzFiLEPMeLp9bAM83lkPATFhyIXk2q2I8lkNVBnFSF1lhCB383fxIzJpU9IioEPjVpIIBe6HRRdI4B0U7BE4-fFhG85-ydIR-dBpow-ix~rCdnTSr4jQtfA0Vp~La7wIR~XQDCZo9ezHVtaA6ibsnIvCxWfTgvTQaXvW-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200814&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f3p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic features at ZFSs. (A) The pie charts show the genomic distribution of ZFSs and 10,000 random peaks. About 46% of ZFSs were found in promoter regions which are defined as within 2 kb of the TSS. (B) The supercoiling density was analyzed as a function of the distance from the peak using Z-catcher. The centre of the peaks is at zero. (C) Peak frequency indicates the number of ZFSs or random peaks that contain predicted ZDRs (black bar). Predicted ZDRs in peak regions were identified using the cutoff of -0.07 or -0.08 in Z-catcher. At -0.08, ZFSs and random peaks containing predicted ZDRs accounted for 81.07% and 32.58% of peaks, respectively. At -0.07, ZFSs and random peaks containing predicted ZDRs accounted for 50.38% and 10.09% of peaks, respectively. A two-sample proportion test was performed using a 99% confidence level. Stars indicate Fisher’s <em>P</em>-value &lt; 2.2e-16. (D) Gene ontology (GO) analysis was performed using the DAVID functional annotation tool with genes that had a ZFS within 2 kb of their TSS. The x-axis indicates −log10 <em>P</em>-value and the y-axis displays the enriched GO terms. (E) Known motif enrichment analysis was carried out using Homer. The top three motifs for transcription factors in ZFSs are shown. Enrichment is shown as −log10 <em>P</em>-value. Letter size indicates the frequency of nucleotides.</p></div></div><p>A functional association of ZFSs with gene was examined by GO analysis. As <a reveal-id="dsw031-F3" data-open="dsw031-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>D is shown, Z-DNA may be involved in a wide variety of biological processes such as nucleosome assembly, translation, cellular macromolecular complex subunit organization, translational elongation, and the regulation of transcription from RNA polymerase II promoters, <em>etc</em>. Some studies have reported that transcription factors and remodeling complexes need Z-DNA-forming sequences to bind DNA.<a reveal-id="dsw031-B15" data-open="dsw031-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw031-B16" data-open="dsw031-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Based on this, we hypothesized that known and unknown factors are binding within ZFSs. We analyzed the ZFSs for transcription factor binding motifs. As shown in <a reveal-id="dsw031-F3" data-open="dsw031-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>E, the NRF motif is the most enriched motif in ZFSs. In addition, there are other transcription factor binding motifs for ARNT and ELK4. These proteins are interesting candidates for follow-up studies on the association of Z-DNA to transcription. According to the result of <em>de novo</em> motif analysis, about 40% of ZFSs contained alternating purine/pyrimidine repeat sequences which were not found in the remaining ZFSs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). This result is also consistent with the Zaa ChIP-qPCR assay; alternating purine/pyrimidine repeat sequences might not be the only major factor required to form the Z-DNA structure in cells.</p><p>Negative supercoils are generated behind RNA polymerase II as it moves along the DNA strand and during the nucleosome remodelling.<a reveal-id="dsw031-B31" data-open="dsw031-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw031-B32" data-open="dsw031-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Furthermore, negative supercoiling is found in open chromatin regions, which contain actively transcribed genes.<a reveal-id="dsw031-B33" data-open="dsw031-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw031-B34" data-open="dsw031-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Z-DNA is known to be stabilized by negative supercoiling, as determined <em>in vivo</em>. Our experiments demonstrating that Zaa-Fok digests predicted ZDRs in a negatively supercoiled plasmid and that ZFSs have high potential to form Z-DNA also support the assertion that Z-DNA can form during negative supercoiling. Moreover, we found a large portion of ZFSs were located in promoter regions. All of these results suggest the possibility that the Z-DNA formation is related to active transcription.</p><p>Recently, it has been reported that false-positive signals from ChIP-seq could be caused by artefacts and biases inherent in the ChIP-seq procedures such as batch-to-batch variation in sonication efficiency, cross-reactivity of antibodies with non-specific proteins, differential chromatin accessibility, and so on.<a reveal-id="dsw031-B35" data-open="dsw031-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsw031-B36" data-open="dsw031-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Even though input and IgG controls are used, artificially high ChIP signals cannot be fully removed. However, most Zaa peaks (91%) were not overlapped with ultra-high signal artefact regions (data not shown), which suggests that Zaa peaks are reliable to find a biological function of Z-DNA.</p>                            <h3 scrollto-destination=87200818 id="87200818" class="section-title" data-legacy-id=SEC3.5>3.5. Differences of distribution of ZFSs and Z-DNA hotspots</h3>
<p>Z-DNA hotspots found by ChAP and cloning, followed by Sanger sequencing, has been reported that they were mainly found in centromeres and telomeres.<a reveal-id="dsw031-B12" data-open="dsw031-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Because many of our ZFSs were detected in the promoter regions of human genes and Zaa enrichment in several Z-DNA hotspots was not observed in our Zaa ChIP-qPCR data, we further examined this discrepancy. Out of 186 Z-DNA hotspots, only 6 Zaa peaks were colocalized with Z-DNA hotspots (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6A</a></span>). Notably, our input and IgG also showed strong read enrichment in Z-DNA hotspots (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6B</a></span>).</p><p>There are four possible implications of such differences; First, different methods were used for detecting Z-DNA in human cells. Li <em>et al.</em> used dual cross-linking ChAP followed by cloning and Sanger sequencing. Because this method has less coverage than ChIP-Seq, it is possible that Za-binding regions were only locally mapped, which might not give enough information for identifying the role of Z-DNA. Second, they applied purified Za to fixed and permeabilized cells and then cross-linked Za to DNA. This strategy might result in identifying Z-DNA formed in other genomic locations than the locations found in our approach. Third, different constructs were used to detect Z-DNA. Since Zaa more strongly and specifically binds to Z-DNA than Za, different maps of Z-DNA forming sites might be generated by Za and Zaa. Finally, different cell lines were used. Different cells have different characteristics, and Z-DNA locations may be associated with different biological processes. Future studies are needed to find Z-DNA-forming sites and to compare the biological functions of Z-DNA in different cells using each method.</p>                            <h3 scrollto-destination=87200821 id="87200821" class="section-title" data-legacy-id=SEC3.6>3.6. Association of ZFSs with active transcription</h3>
<p>To examine the relationship between Z-DNA and transcription, the expression levels of genes associated with ZFSs were analyzed. Higher gene expression was observed in genes with ZFS at their promoter than in the gene body (Mann-Whitney U test <em>P</em>-values for promoter vs. gene body, total genes vs. gene body, and total genes vs. promoter are 2.201e-03, 4.322e-06, and &lt; 2.2e-16, respectively. <a reveal-id="dsw031-F4" data-open="dsw031-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>A). Genes were divided into three groups (‘top’, ‘mid’, and ‘bottom’) according to their expression levels and normalized read counts in ZFSs for each Zaa replicate were plotted depending on the gene expression level. The reads mapped in ZFSs were highly enriched in the ‘top’ gene group, especially at the TSSs (<a reveal-id="dsw031-F4" data-open="dsw031-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>B). The same pattern is also shown in other two Zaa replicates (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). Next, we investigated whether ZFSs are associated with RNA polymerase II deposition. RNA polymerase II ChIP-Seq data generated from three different data sets (Pol II-1, Pol II-2, and Pol II-3) were analyzed. Among the 391 ZFSs, 231 ZFSs (59%) overlapped with the RNA polymerase II peaks (<em>P</em>-value &lt; 1.0E-05, Fisher’s exact test, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). ZFSs were substantially correlated with RNA polymerase II binding sites, as shown in the Circos diagram and an example region in chromosome 1 (<a reveal-id="dsw031-F2" data-open="dsw031-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>D). Additionally, the association of ZFSs with RNA polymerase II binding sites was experimentally validated in the top seven genes, which had been selected according to Zaa enrichment in the Zaa ChIP-qPCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A</a></span>). First, as the binding of RNA polymerase II at Zaa peaks was examined in all seven ZFSs via the UCSC genome browser, and the position of RNA polymerase II was overlapped with most ZFSs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8A</a></span>). By ChIP-qPCR, RNA polymerase II mostly showed significant enrichment near selected ZFSs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8B</a></span>). Therefore, these results confirm that Z-DNA formation is strongly correlated with active transcription. </p>                        <a id="87200823" scrollto-destination="87200823"></a>
<div id="dsw031-F4" data-id="dsw031-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f4p.png?Expires=2147483647&Signature=X~BIHcFgFAuX439nTf0QMlLd2PTuGNr-aBz~EBNX~-Aes1FeaTl7mV256V9bFJ-b5MIONgRpzbs5VjD-lIaxvI-ja2dIakAcQnsQpONG~MApcIDpx3liJ6YHXj2kNWR7SI2LYQpVi~4voWRhXcGM-X3-92-pqMxJq2kZ7JZUjVLAhcfU0TxlvF3hwjFc-wL~q-qONWQCUROhksok208cUkDEvnIu-ZwCkPY9s5IB-uGeDLINVR2Tlvy3Ky1r76mtZfBN-TV7Y5tCaJWdOzrebWaWGXFF2orFu37GJPU1m6Ud4BhIZrM4yHUngtBKTpAekOxcz94xeEyACHrNs34VPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of ZFSs with active transcriptional regions. (A) Gene expression analysis was performed. Public mRNA sequencing data were downloaded and mapped to the human reference genome. Each transcript was quantified and normalized using the RPKM (reads per kilobase per million). ‘Promoter’ or ‘Gene body’ indicates where the genes have the ZFS. (B) Genes were ranked based on their RPKM values and divided into three groups based on their expression: top (1,000 highest expressing genes, thick line), mid (1,000 genes between the top and bottom group, dashed line), bottom (6,661 unexpressed genes (RPKM = 0), dotted line). Reads of each Zaa replicate were normalized by Read count per million mapped reads and normalized reads were counted within 5 kb of the TSSs. Shaded regions indicate P-value &lt; 0.05. (C) Heatmap showing the enrichment of various histone marks and RNA polymerase II at ZFSs. ZFSs were ranked by the enrichment of Zaa, calculated using normalized read counts in ZFS ± 3kb. The enrichment level is shown by the color scale (white: depletion, black: enrichment). (D) The correlation of Z-DNA with RNA polymerase II and histone marks is illustrated. In promoter regions of actively transcribed genes, Z-DNA is associated with RNA polymerase II and active histone marks such as H3K4me3 and H3K9ac." path-from-xml="dsw031f4p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200823/dsw031f4p.tiff" path-from-xml="dsw031f4p.tiff" target="_blank">View large</a><a section="87200823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f4p.png?Expires=2147483647&Signature=GRAOPtwetDINSc0m7wLUS0ZeGqY72HEBc6mWkeNY2h1-WMH0sANSMkPALEABWLnw7ite5MJ-lxVg8YpVhhyRkyBfnf41hM1R6-pbO33pHdGRfn0EvKVmvx0WKbmkrHKXMO8~kXXfPDGxUca8qDNhyFw7am-40X0AFyoGufustWkt9-jbDu5vZEWOCZYMfyb1N7WXILlIYABsjP1g9kRnCip~~q5dL5jWN9bg7DABemdH6J9W6Q2sWoAWzSZxkOYp~0edah5G4-i1s3Mq3JZwM37M2d62RD2H03v8yvDgJJmEo~iddmT95w0HHwNVUoYE5nWB7L0BlckzJH2tTutQ6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200823&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f4p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of ZFSs with active transcriptional regions. (A) Gene expression analysis was performed. Public mRNA sequencing data were downloaded and mapped to the human reference genome. Each transcript was quantified and normalized using the RPKM (reads per kilobase per million). ‘Promoter’ or ‘Gene body’ indicates where the genes have the ZFS. (B) Genes were ranked based on their RPKM values and divided into three groups based on their expression: top (1,000 highest expressing genes, thick line), mid (1,000 genes between the top and bottom group, dashed line), bottom (6,661 unexpressed genes (RPKM = 0), dotted line). Reads of each Zaa replicate were normalized by Read count per million mapped reads and normalized reads were counted within 5 kb of the TSSs. Shaded regions indicate <em>P</em>-value &lt; 0.05. (C) Heatmap showing the enrichment of various histone marks and RNA polymerase II at ZFSs. ZFSs were ranked by the enrichment of Zaa, calculated using normalized read counts in ZFS ± 3kb. The enrichment level is shown by the color scale (white: depletion, black: enrichment). (D) The correlation of Z-DNA with RNA polymerase II and histone marks is illustrated. In promoter regions of actively transcribed genes, Z-DNA is associated with RNA polymerase II and active histone marks such as H3K4me3 and H3K9ac.</p></div></div><div content-id="dsw031-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/m_dsw031f4p.png?Expires=2147483647&Signature=X~BIHcFgFAuX439nTf0QMlLd2PTuGNr-aBz~EBNX~-Aes1FeaTl7mV256V9bFJ-b5MIONgRpzbs5VjD-lIaxvI-ja2dIakAcQnsQpONG~MApcIDpx3liJ6YHXj2kNWR7SI2LYQpVi~4voWRhXcGM-X3-92-pqMxJq2kZ7JZUjVLAhcfU0TxlvF3hwjFc-wL~q-qONWQCUROhksok208cUkDEvnIu-ZwCkPY9s5IB-uGeDLINVR2Tlvy3Ky1r76mtZfBN-TV7Y5tCaJWdOzrebWaWGXFF2orFu37GJPU1m6Ud4BhIZrM4yHUngtBKTpAekOxcz94xeEyACHrNs34VPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of ZFSs with active transcriptional regions. (A) Gene expression analysis was performed. Public mRNA sequencing data were downloaded and mapped to the human reference genome. Each transcript was quantified and normalized using the RPKM (reads per kilobase per million). ‘Promoter’ or ‘Gene body’ indicates where the genes have the ZFS. (B) Genes were ranked based on their RPKM values and divided into three groups based on their expression: top (1,000 highest expressing genes, thick line), mid (1,000 genes between the top and bottom group, dashed line), bottom (6,661 unexpressed genes (RPKM = 0), dotted line). Reads of each Zaa replicate were normalized by Read count per million mapped reads and normalized reads were counted within 5 kb of the TSSs. Shaded regions indicate P-value &lt; 0.05. (C) Heatmap showing the enrichment of various histone marks and RNA polymerase II at ZFSs. ZFSs were ranked by the enrichment of Zaa, calculated using normalized read counts in ZFS ± 3kb. The enrichment level is shown by the color scale (white: depletion, black: enrichment). (D) The correlation of Z-DNA with RNA polymerase II and histone marks is illustrated. In promoter regions of actively transcribed genes, Z-DNA is associated with RNA polymerase II and active histone marks such as H3K4me3 and H3K9ac." path-from-xml="dsw031f4p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200823/dsw031f4p.tiff" path-from-xml="dsw031f4p.tiff" target="_blank">View large</a><a section="87200823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031f4p.png?Expires=2147483647&Signature=GRAOPtwetDINSc0m7wLUS0ZeGqY72HEBc6mWkeNY2h1-WMH0sANSMkPALEABWLnw7ite5MJ-lxVg8YpVhhyRkyBfnf41hM1R6-pbO33pHdGRfn0EvKVmvx0WKbmkrHKXMO8~kXXfPDGxUca8qDNhyFw7am-40X0AFyoGufustWkt9-jbDu5vZEWOCZYMfyb1N7WXILlIYABsjP1g9kRnCip~~q5dL5jWN9bg7DABemdH6J9W6Q2sWoAWzSZxkOYp~0edah5G4-i1s3Mq3JZwM37M2d62RD2H03v8yvDgJJmEo~iddmT95w0HHwNVUoYE5nWB7L0BlckzJH2tTutQ6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200823&ar=2236181&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw031f4p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of ZFSs with active transcriptional regions. (A) Gene expression analysis was performed. Public mRNA sequencing data were downloaded and mapped to the human reference genome. Each transcript was quantified and normalized using the RPKM (reads per kilobase per million). ‘Promoter’ or ‘Gene body’ indicates where the genes have the ZFS. (B) Genes were ranked based on their RPKM values and divided into three groups based on their expression: top (1,000 highest expressing genes, thick line), mid (1,000 genes between the top and bottom group, dashed line), bottom (6,661 unexpressed genes (RPKM = 0), dotted line). Reads of each Zaa replicate were normalized by Read count per million mapped reads and normalized reads were counted within 5 kb of the TSSs. Shaded regions indicate <em>P</em>-value &lt; 0.05. (C) Heatmap showing the enrichment of various histone marks and RNA polymerase II at ZFSs. ZFSs were ranked by the enrichment of Zaa, calculated using normalized read counts in ZFS ± 3kb. The enrichment level is shown by the color scale (white: depletion, black: enrichment). (D) The correlation of Z-DNA with RNA polymerase II and histone marks is illustrated. In promoter regions of actively transcribed genes, Z-DNA is associated with RNA polymerase II and active histone marks such as H3K4me3 and H3K9ac.</p></div></div><p>As the chromatin structure has been suggested to be one of the important factors in Z-DNA formation and our Zaa ChIP-qPCR and the <em>in vitro</em> Z-DNA cleavage assay implied the involvement of other factors than nucleotide composition, we tested if specific epigenetic marks were associated with Z-DNA by analyzing histone ChIP-Seq data. Zaa-enriched regions were strongly correlated with the accumulation of H3K4me3 and H3K9ac, which are mainly found in the promoter and enhancer regions of actively transcribed genes. On the other hand, Zaa-enriched regions were not associated with the histone marks H3K9me3, H3K27me3, H3K36me3 and H3K79me3, which are largely in the gene body or are repressive modifications. Moreover, the enrichment of RNA polymerase II was again found in Zaa-enriched regions (<a reveal-id="dsw031-F4" data-open="dsw031-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>C).</p><p>The association of Z-DNA formation with active transcription was further examined by luciferase reporter assay. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S9</a></span>, 90% ZFSs (9/10) showed significantly increased luciferase reporter activities relative to the negative control, supporting a positive correlation between Z-DNA formation and transcriptional activation. Collectively, it is evident that the Z-DNA formation is tightly linked with the open chromatin structure of active transcription. Z-DNA is formed near the TSS of actively transcribed genes, with RNA polymerase II and active histone marks located in the promoter regions and, importantly, Z-DNA formation might play a crucial role in activating transcription (<a reveal-id="dsw031-F4" data-open="dsw031-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>D).</p>                            <h3 scrollto-destination=87200826 id="87200826" class="section-title" data-legacy-id=SEC3.7>3.7. Conclusion</h3>
<p>Z-DNA is one of the most well studied non-B-DNA structures, and <em>in vivo</em> and <em>in vitro</em> formation of Z-DNA has been reported in many studies. However, the general biological function of Z-DNA in human cells has not been well elucidated because only a few studies have investigated the genome-wide distribution of Z-DNA-forming sites. Using Zaa as a probe, we identified highly confident 391 ZFSs genome-wide in HeLa cells with ChIP-seq. The majority of 10 ZFSs was validated by Zaa ChIP-qPCR and an <em>in vitro</em> Z-DNA cleavage assay. We found that ZFSs are mostly located in promoter regions and contain sequences with the highest potential to form Z-DNA. ZFSs are significantly associated with highly expressed genes and have an enrichment of RNA polymerase II binding. In addition, active histone marks were significantly correlated with Z-DNA formation, meaning that Z-DNA formation in cells relies not only on sequence composition, but also on chromatin structure. Furthermore, we confirmed that the Z-DNA formation might transcriptionally activate genes. These results suggest that there is a strong correlation between Z-DNA formation and active transcription.</p><p>This ZFS mapping information could be useful for identifying disease genes whose expression can be altered if Z-DNA formation can be regulated by drugs such as inhibitors of epigenetic regulators, topoisomerase inhibitor, or other reagents that release negative supercoils.</p>                            <h2 scrollto-destination=87200829 id="87200829" class="section-title" >Conflic of Interest</h2>
<p>None declared</p>                            <h2 scrollto-destination=87200831 id="87200831" class="section-title" >Accession number</h2>
<p>The accession number of ChIP-Seq data is GSE71682.</p>                            <h2 scrollto-destination=87200833 id="87200833" class="section-title" >Funding</h2>
<p>This work was supported by grants from the National Research Foundation of Korea (2015M3A9B4051044, 2010-0020259 and 2015048159, and 2014M3C9A3064548 to TYR), the POSTECH BSRI research fund (to TYR), and the BK21 PLUS fellowship program funded by the National Research Foundation of Korea (G16CN40T1201 to SIS, JP, SH, SHS, CHL and HJ). Zaa-Fok plasmid was kindly provided by Dr. Keji Zhao.</p>                            <h2 scrollto-destination=87200835 id="87200835" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw031-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">McLean</div>
               <div class="given-names">M. J.</div>
            </div>
             <div class="name">
              <div class="surname">Blaho</div>
               <div class="given-names">J. A.</div>
            </div>
             <div class="name">
              <div class="surname">Kilpatrick</div>
               <div class="given-names">M. W.</div>
            </div>
             <div class="name">
              <div class="surname">Wells</div>
               <div class="given-names">R. D.</div>
            </div>
          </div>
          <div class="year">1986</div>, 
<div class="article-title">Consecutive A X T pairs can adopt a left-handed DNA structure</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> ., <div class="volume">83</div>, <div class="fpage">5884</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consecutive%20A%20X%20T%20pairs%20can%20adopt%20a%20left-handed%20DNA%20structure&amp;author=M.%20J.%20McLean&amp;author=J.%20A.%20Blaho&amp;author=M.%20W.%20Kilpatrick&amp;author=R.%20D.%20Wells&amp;publication_year=1986&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=83&amp;pages=5884-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.83.16.5884" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.83.16.5884" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.83.16.5884"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3016726" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ho</div>
               <div class="given-names">P. S.</div>
            </div>
             <div class="name">
              <div class="surname">Ellison</div>
               <div class="given-names">M. J.</div>
            </div>
             <div class="name">
              <div class="surname">Quigley</div>
               <div class="given-names">G. J.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1986</div>, 
<div class="article-title">A computer aided thermodynamic approach for predicting the formation of Z-DNA in naturally occurring sequences</div>. <div class="source">EMBO J</div> ., <div class="volume">5</div>, <div class="fpage">2737</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20computer%20aided%20thermodynamic%20approach%20for%20predicting%20the%20formation%20of%20Z-DNA%20in%20naturally%20occurring%20sequences&amp;author=P.%20S.%20Ho&amp;author=M.%20J.%20Ellison&amp;author=G.%20J.%20Quigley&amp;author=A.%20Rich&amp;publication_year=1986&amp;journal=EMBO%20J&amp;volume=5&amp;pages=2737-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3780676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ellison</div>
               <div class="given-names">M. J.</div>
            </div>
             <div class="name">
              <div class="surname">Feigon</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Kelleher</div>
               <div class="given-names">R. J.</div>
              3rd
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">A. H.</div>
            </div>
             <div class="name">
              <div class="surname">Habener</div>
               <div class="given-names">J. F.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1986</div>, 
<div class="article-title">An assessment of the Z-DNA forming potential of alternating dA-dT stretches in supercoiled plasmids</div>. <div class="source">Biochemistry</div> , <div class="volume">25</div>, <div class="fpage">3648</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20assessment%20of%20the%20Z-DNA%20forming%20potential%20of%20alternating%20dA-dT%20stretches%20in%20supercoiled%20plasmids&amp;author=M.%20J.%20Ellison&amp;author=J.%20Feigon&amp;author=R.%20J.%20Kelleher&amp;author=A.%20H.%20Wang&amp;author=J.%20F.%20Habener&amp;author=A.%20Rich&amp;publication_year=1986&amp;journal=Biochemistry&amp;volume=25&amp;pages=3648-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00360a026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00360a026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00360a026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3718951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Behe</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Felsenfeld</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">1981</div>, 
<div class="article-title">Effects of methylation on a synthetic polynucleotide: the B–Z transition in poly(dG-m5dC).poly(dG-m5dC)</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> ., <div class="volume">78</div>, <div class="fpage">1619</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20methylation%20on%20a%20synthetic%20polynucleotide%3A%20the%20B%E2%80%93Z%20transition%20in%20poly(dG-m5dC).poly(dG-m5dC)&amp;author=M.%20Behe&amp;author=G.%20Felsenfeld&amp;publication_year=1981&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=78&amp;pages=1619-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.78.3.1619" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.78.3.1619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.78.3.1619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6262820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Peck</div>
               <div class="given-names">L. J.</div>
            </div>
             <div class="name">
              <div class="surname">Nordheim</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">J. C.</div>
            </div>
          </div>
          <div class="year">1982</div>, 
<div class="article-title">Flipping of cloned d(pCpG)n.d(pCpG)n DNA sequences from right- to left-handed helical structure by salt, Co(III), or negative supercoiling</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> ., <div class="volume">79</div>, <div class="fpage">4560</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flipping%20of%20cloned%20d(pCpG)n.d(pCpG)n%20DNA%20sequences%20from%20right-%20to%20left-handed%20helical%20structure%20by%20salt%2C%20Co(III)%2C%20or%20negative%20supercoiling&amp;author=L.%20J.%20Peck&amp;author=A.%20Nordheim&amp;author=A.%20Rich&amp;author=J.%20C.%20Wang&amp;publication_year=1982&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=79&amp;pages=4560-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.79.15.4560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.79.15.4560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.79.15.4560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6956879" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Schwartz</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Lowenhaupt</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">Y. G.</div>
            </div>
          </div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Proteolytic dissection of Zab, the Z-DNA-binding domain of human ADAR1</div>. <div class="source">J. Bio. Chem</div> ., <div class="volume">274</div>, <div class="fpage">2899</div>–<div class="lpage">906</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteolytic%20dissection%20of%20Zab%2C%20the%20Z-DNA-binding%20domain%20of%20human%20ADAR1&amp;author=T.%20Schwartz&amp;author=K.%20Lowenhaupt&amp;author=Y.%20G.%20Kim&amp;publication_year=1999&amp;journal=J.%20Bio.%20Chem&amp;volume=274&amp;pages=2899-906" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.274.5.2899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.274.5.2899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.274.5.2899"> </span></div> </div></div></div></div></div><div content-id="dsw031-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Herbert</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Schade</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Lowenhaupt</div>
               <div class="given-names">K.</div>
            </div>
          </div>, et al.  . <div class="year">1998</div>, 
<div class="article-title">The Zalpha domain from human ADAR1 binds to the Z-DNA conformer of many different sequences</div>. <div class="source">Nucleic. Acids Res</div> ., <div class="volume">26</div>, <div class="fpage">3486</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Zalpha%20domain%20from%20human%20ADAR1%20binds%20to%20the%20Z-DNA%20conformer%20of%20many%20different%20sequences&amp;author=A.%20Herbert&amp;author=M.%20Schade&amp;author=K.%20Lowenhaupt&amp;publication_year=1998&amp;journal=Nucleic.%20Acids%20Res&amp;volume=26&amp;pages=3486-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.15.3486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.15.3486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.15.3486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9671809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">Y. G.</div>
            </div>
             <div class="name">
              <div class="surname">Lowenhaupt</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Schwartz</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1999</div>, 
<div class="article-title">The interaction between Z-DNA and the Zab domain of double-stranded RNA adenosine deaminase characterized using fusion nucleases</div>. <div class="source">J. Bio. Chem</div> ., <div class="volume">274</div>, <div class="fpage">19081</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20interaction%20between%20Z-DNA%20and%20the%20Zab%20domain%20of%20double-stranded%20RNA%20adenosine%20deaminase%20characterized%20using%20fusion%20nucleases&amp;author=Y.%20G.%20Kim&amp;author=K.%20Lowenhaupt&amp;author=T.%20Schwartz&amp;author=A.%20Rich&amp;publication_year=1999&amp;journal=J.%20Bio.%20Chem&amp;volume=274&amp;pages=19081-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.274.27.19081" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.274.27.19081" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.274.27.19081"> </span></div> </div></div></div></div></div><div content-id="dsw031-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">Y. G.</div>
            </div>
             <div class="name">
              <div class="surname">Muralinath</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Brandt</div>
               <div class="given-names">T.</div>
            </div>
          </div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">A role for Z-DNA binding in vaccinia virus pathogenesis</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> ., <div class="volume">100</div>, <div class="fpage">6974</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20Z-DNA%20binding%20in%20vaccinia%20virus%20pathogenesis&amp;author=Y.%20G.%20Kim&amp;author=M.%20Muralinath&amp;author=T.%20Brandt&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=100&amp;pages=6974-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0431131100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0431131100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0431131100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12777633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">Y. G.</div>
            </div>
             <div class="name">
              <div class="surname">Lowenhaupt</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Oh</div>
               <div class="given-names">D. B.</div>
            </div>
             <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">K. K.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2004</div>, 
<div class="article-title">Evidence that vaccinia virulence factor E3L binds to Z-DNA in vivo: Implications for development of a therapy for poxvirus infection</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> ., <div class="volume">101</div>, <div class="fpage">1514</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20that%20vaccinia%20virulence%20factor%20E3L%20binds%20to%20Z-DNA%20in%20vivo%3A%20Implications%20for%20development%20of%20a%20therapy%20for%20poxvirus%20infection&amp;author=Y.%20G.%20Kim&amp;author=K.%20Lowenhaupt&amp;author=D.%20B.%20Oh&amp;author=K.%20K.%20Kim&amp;author=A.%20Rich&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=101&amp;pages=1514-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0308260100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0308260100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0308260100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14757814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Schroth</div>
               <div class="given-names">G. P.</div>
            </div>
             <div class="name">
              <div class="surname">Chou</div>
               <div class="given-names">P. J.</div>
            </div>
             <div class="name">
              <div class="surname">Ho</div>
               <div class="given-names">P. S.</div>
            </div>
          </div>
          <div class="year">1992</div>, 
<div class="article-title">Mapping Z-DNA in the human genome. Computer-aided mapping reveals a nonrandom distribution of potential Z-DNA-forming sequences in human genes</div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">267</div>, <div class="fpage">11846</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20Z-DNA%20in%20the%20human%20genome.%20Computer-aided%20mapping%20reveals%20a%20nonrandom%20distribution%20of%20potential%20Z-DNA-forming%20sequences%20in%20human%20genes&amp;author=G.%20P.%20Schroth&amp;author=P.%20J.%20Chou&amp;author=P.%20S.%20Ho&amp;publication_year=1992&amp;journal=J.%20Biol.%20Chem&amp;volume=267&amp;pages=11846-55" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1601856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Xiao</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Lu</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Feng</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Droge</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">2009</div>, 
<div class="article-title">Human genomic Z-DNA segments probed by the Z alpha domain of ADAR1</div>. <div class="source">Nucleic. Acids. Res</div> ., <div class="volume">37</div>, <div class="fpage">2737</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20genomic%20Z-DNA%20segments%20probed%20by%20the%20Z%20alpha%20domain%20of%20ADAR1&amp;author=H.%20Li&amp;author=J.%20Xiao&amp;author=J.%20Li&amp;author=L.%20Lu&amp;author=S.%20Feng&amp;author=P.%20Droge&amp;publication_year=2009&amp;journal=Nucleic.%20Acids.%20Res&amp;volume=37&amp;pages=2737-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp124" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp124" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp124"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19276205" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">L. F.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">J. C.</div>
            </div>
          </div>
          <div class="year">1987</div>, 
<div class="article-title">Supercoiling of the DNA template during transcription</div>. <div class="source">Proc. Natl Acad Sci. U S A</div> , <div class="volume">84</div>, <div class="fpage">7024</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Supercoiling%20of%20the%20DNA%20template%20during%20transcription&amp;author=L.%20F.%20Liu&amp;author=J.%20C.%20Wang&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad%20Sci.%20U%20S%20A&amp;volume=84&amp;pages=7024-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.20.7024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.20.7024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.20.7024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2823250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wittig</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Wolfl</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Dorbic</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Vahrson</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1992</div>, 
<div class="article-title">Transcription of human c-myc in permeabilized nuclei is associated with formation of Z-DNA in three discrete regions of the gene</div>. <div class="source">EMBO J</div> ., <div class="volume">11</div>, <div class="fpage">4653</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20of%20human%20c-myc%20in%20permeabilized%20nuclei%20is%20associated%20with%20formation%20of%20Z-DNA%20in%20three%20discrete%20regions%20of%20the%20gene&amp;author=B.%20Wittig&amp;author=S.%20Wolfl&amp;author=T.%20Dorbic&amp;author=W.%20Vahrson&amp;author=A.%20Rich&amp;publication_year=1992&amp;journal=EMBO%20J&amp;volume=11&amp;pages=4653-63" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1330542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Kirby</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Brown</div>
               <div class="given-names">P. O.</div>
            </div>
             <div class="name">
              <div class="surname">Zhao</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">2001</div>, 
<div class="article-title">Regulation of CSF1 promoter by the SWI/SNF-like BAF complex</div>. <div class="source">Cell</div> , <div class="volume">106</div>, <div class="fpage">309</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20CSF1%20promoter%20by%20the%20SWI%2FSNF-like%20BAF%20complex&amp;author=R.%20Liu&amp;author=H.%20Liu&amp;author=X.%20Chen&amp;author=M.%20Kirby&amp;author=P.%20O.%20Brown&amp;author=K.%20Zhao&amp;publication_year=2001&amp;journal=Cell&amp;volume=106&amp;pages=309-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(01)00446-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(01)00446-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(01)00446-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11509180" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Maruyama</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Mimura</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Harada</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Itoh</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">2013</div>, 
<div class="article-title">Nrf2 activation is associated with Z-DNA formation in the human HO-1 promoter</div>. <div class="source">Nucleic. Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">5223</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nrf2%20activation%20is%20associated%20with%20Z-DNA%20formation%20in%20the%20human%20HO-1%20promoter&amp;author=A.%20Maruyama&amp;author=J.%20Mimura&amp;author=N.%20Harada&amp;author=K.%20Itoh&amp;publication_year=2013&amp;journal=Nucleic.%20Acids%20Res&amp;volume=41&amp;pages=5223-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23571756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wong</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Kwon</div>
               <div class="given-names">J. A.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2007</div>, 
<div class="article-title">Characterization of Z-DNA as a nucleosome-boundary element in yeast Saccharomyces cerevisiae</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> , <div class="volume">104</div>, <div class="fpage">2229</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Z-DNA%20as%20a%20nucleosome-boundary%20element%20in%20yeast%20Saccharomyces%20cerevisiae&amp;author=B.%20Wong&amp;author=S.%20Chen&amp;author=J.%20A.%20Kwon&amp;author=A.%20Rich&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=104&amp;pages=2229-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611447104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611447104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611447104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17284586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Vasquez</div>
               <div class="given-names">K. M.</div>
            </div>
          </div>
          <div class="year">2006</div>, 
<div class="article-title">Non-B DNA structure-induced genetic instability</div>. <div class="source">Mut. Res</div> ., <div class="volume">598</div>, <div class="fpage">103</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-B%20DNA%20structure-induced%20genetic%20instability&amp;author=G.%20Wang&amp;author=K.%20M.%20Vasquez&amp;publication_year=2006&amp;journal=Mut.%20Res&amp;volume=598&amp;pages=103-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mrfmmm.2006.01.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mrfmmm.2006.01.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mrfmmm.2006.01.019"> </span></div> </div></div></div></div></div><div content-id="dsw031-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Christensen</div>
               <div class="given-names">L. A.</div>
            </div>
             <div class="name">
              <div class="surname">Vasquez</div>
               <div class="given-names">K. M.</div>
            </div>
          </div>
          <div class="year">2006</div>, 
<div class="article-title">Z-DNA-forming sequences generate large-scale deletions in mammalian cells</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> , <div class="volume">103</div>, <div class="fpage">2677</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Z-DNA-forming%20sequences%20generate%20large-scale%20deletions%20in%20mammalian%20cells&amp;author=G.%20Wang&amp;author=L.%20A.%20Christensen&amp;author=K.%20M.%20Vasquez&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=103&amp;pages=2677-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0511084103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0511084103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0511084103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16473937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Edwards</div>
               <div class="given-names">S. F.</div>
            </div>
             <div class="name">
              <div class="surname">Sirito</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Krahe</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Sinden</div>
               <div class="given-names">R. R.</div>
            </div>
          </div>
          <div class="year">2009</div>
          <div class="article-title">, A Z-DNA sequence reduces slipped-strand structure formation in the myotonic dystrophy type 2 (CCTG) x (CAGG) repeat</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> , <div class="volume">106</div>, <div class="fpage">3270</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%2C%20A%20Z-DNA%20sequence%20reduces%20slipped-strand%20structure%20formation%20in%20the%20myotonic%20dystrophy%20type%202%20(CCTG)%20x%20(CAGG)%20repeat&amp;author=S.%20F.%20Edwards&amp;author=M.%20Sirito&amp;author=R.%20Krahe&amp;author=R.%20R.%20Sinden&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=106&amp;pages=3270-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0807699106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0807699106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0807699106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19218442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mulholland</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Xu</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Sugiyama</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Zhao</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">SWI/SNF-mediated chromatin remodeling induces Z-DNA formation on a nucleosome</div>. <div class="source">Cell &amp; Biosci</div> ., <div class="volume">2</div>, <div class="fpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SWI%2FSNF-mediated%20chromatin%20remodeling%20induces%20Z-DNA%20formation%20on%20a%20nucleosome&amp;author=N.%20Mulholland&amp;author=Y.%20Xu&amp;author=H.%20Sugiyama&amp;author=K.%20Zhao&amp;publication_year=2012&amp;journal=Cell%20%26%20Biosci&amp;volume=2&amp;pages=3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2045-3701-2-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2045-3701-2-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2045-3701-2-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22264354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">Y. G.</div>
            </div>
             <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">P. S.</div>
            </div>
             <div class="name">
              <div class="surname">Herbert</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1997</div>, 
<div class="article-title">Construction of a Z-DNA-specific restriction endonuclease</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> , <div class="volume">94</div>, <div class="fpage">12875</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20Z-DNA-specific%20restriction%20endonuclease&amp;author=Y.%20G.%20Kim&amp;author=P.%20S.%20Kim&amp;author=A.%20Herbert&amp;author=A.%20Rich&amp;publication_year=1997&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=94&amp;pages=12875-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.24.12875" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.24.12875" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.24.12875"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9371768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Choi</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">J. K.</div>
            </div>
             <div class="name">
              <div class="surname">Yoo</div>
               <div class="given-names">J. Y.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">NFkappaB and STAT3 synergistically activate the expression of FAT10, a gene counteracting the tumor suppressor p53</div>. <div class="source">Mol. Oncol</div> ., <div class="volume">8</div>, <div class="fpage">642</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NFkappaB%20and%20STAT3%20synergistically%20activate%20the%20expression%20of%20FAT10%2C%20a%20gene%20counteracting%20the%20tumor%20suppressor%20p53&amp;author=Y.%20Choi&amp;author=J.%20K.%20Kim&amp;author=J.%20Y.%20Yoo&amp;publication_year=2014&amp;journal=Mol.%20Oncol&amp;volume=8&amp;pages=642-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molonc.2014.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molonc.2014.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molonc.2014.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24518302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Choe</div>
               <div class="given-names">M. K.</div>
            </div>
             <div class="name">
              <div class="surname">Hong</div>
               <div class="given-names">C. P.</div>
            </div>
             <div class="name">
              <div class="surname">Park</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Seo</div>
               <div class="given-names">S. H.</div>
            </div>
             <div class="name">
              <div class="surname">Roh</div>
               <div class="given-names">T. Y.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Functional elements demarcated by histone modifications in breast cancer cells</div>. <div class="source">Biochem. Biophys. Res. Commun</div> ., <div class="volume">418</div>, <div class="fpage">475</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20elements%20demarcated%20by%20histone%20modifications%20in%20breast%20cancer%20cells&amp;author=M.%20K.%20Choe&amp;author=C.%20P.%20Hong&amp;author=J.%20Park&amp;author=S.%20H.%20Seo&amp;author=T.%20Y.%20Roh&amp;publication_year=2012&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=418&amp;pages=475-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2012.01.042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2012.01.042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2012.01.042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22285863" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Auerbach</div>
               <div class="given-names">R. K.</div>
            </div>
             <div class="name">
              <div class="surname">Euskirchen</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Rozowsky</div>
               <div class="given-names">J.</div>
            </div>
          </div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Mapping accessible chromatin regions using Sono-Seq</div>. <div class="source">Proc. Natl Acad.Sci.U S A</div> , <div class="volume">106</div>, <div class="fpage">14926</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20accessible%20chromatin%20regions%20using%20Sono-Seq&amp;author=R.%20K.%20Auerbach&amp;author=G.%20Euskirchen&amp;author=J.%20Rozowsky&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.Sci.U%20S%20A&amp;volume=106&amp;pages=14926-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905443106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905443106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905443106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19706456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Landt</div>
               <div class="given-names">S. G.</div>
            </div>
             <div class="name">
              <div class="surname">Marinov</div>
               <div class="given-names">G. K.</div>
            </div>
             <div class="name">
              <div class="surname">Kundaje</div>
               <div class="given-names">A.</div>
            </div>
          </div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia</div>. <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">1813</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ChIP-seq%20guidelines%20and%20practices%20of%20the%20ENCODE%20and%20modENCODE%20consortia&amp;author=S.%20G.%20Landt&amp;author=G.%20K.%20Marinov&amp;author=A.%20Kundaje&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=1813-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.136184.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.136184.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.136184.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955991" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nagaraj</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Wisniewski</div>
               <div class="given-names">J. R.</div>
            </div>
             <div class="name">
              <div class="surname">Geiger</div>
               <div class="given-names">T.</div>
            </div>
          </div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Deep proteome and transcriptome mapping of a human cancer cell line</div>. <div class="source">Mol. Syst. Biol</div> ., <div class="volume">7</div>, <div class="fpage">548</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20proteome%20and%20transcriptome%20mapping%20of%20a%20human%20cancer%20cell%20line&amp;author=N.%20Nagaraj&amp;author=J.%20R.%20Wisniewski&amp;author=T.%20Geiger&amp;publication_year=2011&amp;journal=Mol.%20Syst.%20Biol&amp;volume=7&amp;pages=548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2011.81" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2011.81" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2011.81"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22068331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Pertea</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Pimentel</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Kelley</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Salzberg</div>
               <div class="given-names">S. L.</div>
            </div>
          </div>
          <div class="year">2013</div>, 
<div class="article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</div>. <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.%20L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-4-r36" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r36" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-4-r36"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rajagopal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Ernst</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">P.</div>
            </div>
          </div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Distinct and predictive histone lysine acetylation patterns at promoters, enhancers, and gene bodies</div>. <div class="source">G3 (Bethesda)</div> , <div class="volume">4</div>, <div class="fpage">2051</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20and%20predictive%20histone%20lysine%20acetylation%20patterns%20at%20promoters%2C%20enhancers%2C%20and%20gene%20bodies&amp;author=N.%20Rajagopal&amp;author=J.%20Ernst&amp;author=P.%20Ray&amp;publication_year=2014&amp;journal=G3%20(Bethesda)&amp;volume=4&amp;pages=2051-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.013565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.013565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.013565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25122670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">B. K.</div>
            </div>
             <div class="name">
              <div class="surname">Dhar</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Shakya</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Ray</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">Z-DNA-forming silencer in the first exon regulates human ADAM-12 gene expression</div>. <div class="source">Proc. Natl Acad. Sci. U S A</div> , <div class="volume">108</div>, <div class="fpage">103</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Z-DNA-forming%20silencer%20in%20the%20first%20exon%20regulates%20human%20ADAM-12%20gene%20expression&amp;author=B.%20K.%20Ray&amp;author=S.%20Dhar&amp;author=A.%20Shakya&amp;author=A.%20Ray&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=108&amp;pages=103-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1008831108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1008831108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1008831108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21173277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Havas</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Flaus</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Phelan</div>
               <div class="given-names">M.</div>
            </div>
          </div>, et al.  . <div class="year">2000</div>, 
<div class="article-title">Generation of superhelical torsion by ATP-dependent chromatin remodeling activities</div>. <div class="source">Cell</div> , <div class="volume">103</div>, <div class="fpage">1133</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20superhelical%20torsion%20by%20ATP-dependent%20chromatin%20remodeling%20activities&amp;author=K.%20Havas&amp;author=A.%20Flaus&amp;author=M.%20Phelan&amp;publication_year=2000&amp;journal=Cell&amp;volume=103&amp;pages=1133-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)00215-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)00215-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)00215-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11163188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gavin</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Horn</div>
               <div class="given-names">P. J.</div>
            </div>
             <div class="name">
              <div class="surname">Peterson</div>
               <div class="given-names">C. L.</div>
            </div>
          </div>
          <div class="year">2001</div>, 
<div class="article-title">SWI/SNF chromatin remodeling requires changes in DNA topology</div>. <div class="source">Mol. Cell</div> , <div class="volume">7</div>, <div class="fpage">97</div>–<div class="lpage">104</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SWI%2FSNF%20chromatin%20remodeling%20requires%20changes%20in%20DNA%20topology&amp;author=I.%20Gavin&amp;author=P.%20J.%20Horn&amp;author=C.%20L.%20Peterson&amp;publication_year=2001&amp;journal=Mol.%20Cell&amp;volume=7&amp;pages=97-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1097-2765(01)00158-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1097-2765(01)00158-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1097-2765(01)00158-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11172715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Naughton</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Avlonitis</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Corless</div>
               <div class="given-names">S.</div>
            </div>
          </div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Transcription forms and remodels supercoiling domains unfolding large-scale chromatin structures</div>. <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">20</div>, <div class="fpage">387</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20forms%20and%20remodels%20supercoiling%20domains%20unfolding%20large-scale%20chromatin%20structures&amp;author=C.%20Naughton&amp;author=N.%20Avlonitis&amp;author=S.%20Corless&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=20&amp;pages=387-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23416946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kouzine</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Gupta</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Baranello</div>
               <div class="given-names">L.</div>
            </div>
          </div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Transcription-dependent dynamic supercoiling is a short-range genomic force</div>. <div class="source">Nat. Struct. Mol Biol</div> ., <div class="volume">20</div>, <div class="fpage">396</div>–<div class="lpage">403</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-dependent%20dynamic%20supercoiling%20is%20a%20short-range%20genomic%20force&amp;author=F.%20Kouzine&amp;author=A.%20Gupta&amp;author=L.%20Baranello&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol%20Biol&amp;volume=20&amp;pages=396-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2517"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23416947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Jain</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Baldi</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Zabel</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Straub</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Becker</div>
               <div class="given-names">P. B.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Active promoters give rise to false positive 'Phantom Peaks' in ChIP-seq experiments</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">6959</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Active%20promoters%20give%20rise%20to%20false%20positive%20'Phantom%20Peaks'%20in%20ChIP-seq%20experiments&amp;author=D.%20Jain&amp;author=S.%20Baldi&amp;author=A.%20Zabel&amp;author=T.%20Straub&amp;author=P.%20B.%20Becker&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=6959-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv637" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv637" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv637"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26117547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw031-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw031-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Carroll</div>
               <div class="given-names">T. S.</div>
            </div>
             <div class="name">
              <div class="surname">Liang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Salama</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Stark</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">de Santiago</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Impact of artifact removal on ChIP quality metrics in ChIP-seq and ChIP-exo data</div>. <div class="source">Front Genet</div> ., <div class="volume">5</div>, <div class="fpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20artifact%20removal%20on%20ChIP%20quality%20metrics%20in%20ChIP-seq%20and%20ChIP-exo%20data&amp;author=T.%20S.%20Carroll&amp;author=Z.%20Liang&amp;author=R.%20Salama&amp;author=R.%20Stark&amp;author=I.%20de%20Santiago&amp;publication_year=2014&amp;journal=Front%20Genet&amp;volume=5&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2014.00075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2014.00075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2014.00075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24782889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2236181&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw031/1/dsw031_Supplementary_Data.zip?Expires=2147483647&amp;Signature=2xzMOCCKAZDnLWHSlS4FAKJFKCh06OI0NJ0e2c7iaGYW636YNwwhAkkpoSeNNVz3K8wb-wHGmzZSH3ow1rdjJE-Z7~YVgqfzzmjeOsQPA7Pz8zM11jhRMhXSD9OZHML4~wiulKxZwtakLooveJx28aEec09eAvnlzKIlW3SR2qvRU8nZ5tJ~~L4Hhz13y6anSrYjx-P28nCp8OIssF2O6m9-P9Ne5AuCKdqdzhYkWosVdRpg4mlILK7-ZUXxxAyD3Dl0OvfW1wXIR0F3426hbqB1B64p1BgJP6R-NGsLDqm-89j0srMCk1Hhuv4mGhcWTVpq6iPUcNPgM2VqJCxj6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">748</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000386964800006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw031"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2236181"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000386964800006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f5%2f477%2f2236181">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000386964800006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F5%2F477%2F2236181" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2236181'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw031");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f477%2f2236181">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

