

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/5/287/421945","@type":"ScholarlyArticle","name":"A Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses","datePublished":"2009-10-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2009-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp018","keywords":["batch learning SOM","oligopeptide frequency","protein function","metagenome","alignment-free clustering"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/5/287/421945","author":[{"name":"Abe, Takashi","affiliation":"1  Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"},{"name":"Kanaya, Shigehiko","affiliation":"2  Department of Bioinformatics and Genomes, Graduate School of Information Science ,  Nara Institute of Science and Technology ,  Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Uehara, Hiroshi","affiliation":"1  Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"},{"name":"Ikemura, Toshimichi","affiliation":"1  Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829 ,  Japan","@type":"Person"}],"description":"Abstract.  As a result of remarkable progresses of DNA sequencing technology, vast quantities of genomic sequences have been decoded. Homology search for amino ","pageStart":"287","pageEnd":"297","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses" />
    <meta property="og:description" content="Abstract.  As a result of remarkable progresses of DNA sequencing technology, vast quantities of genomic sequences have been decoded. Homology search for amino " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/5/287/421945" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-10-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Abe, Takashi" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_author" content="Kanaya, Shigehiko" /><meta name="citation_author_institution" content="Department of Bioinformatics and Genomes, Graduate School of Information Science, Nara Institute of Science and Technology, Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Uehara, Hiroshi" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_author" content="Ikemura, Toshimichi" /><meta name="citation_author_institution" content="Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829, Japan" /><meta name="citation_title" content="A Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses" /><meta name="citation_firstpage" content="287" /><meta name="citation_lastpage" content="297" /><meta name="citation_doi" content="10.1093/dnares/dsp018" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2009/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=FUGUE: sequence-structure homology recognition using environment-specific substitution tables and structure-dependent gap penalties; J. Mol. Biol.;  citation_year=2001;  citation_volume=310;  citation_pages=243-57; " /><meta name="citation_reference" content="citation_title=Data growth and its impact on the SCOP database: new developments; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D419-25; " /><meta name="citation_reference" content="citation_title=The CATH classification revisited—architectures reviewed and new ways to characterize structural divergence in superfamilies; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D310-4; " /><meta name="citation_reference" content="citation_title=Evolution of function in protein superfamilies, from a structural perspective; J. Mol. Biol.;  citation_year=2001;  citation_volume=307;  citation_pages=1113-43; " /><meta name="citation_reference" content="citation_title=Predicting protein function from sequence and structure; Nat. Rev. Mol. Cell Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=995-1005; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Twilight zone of protein sequence alignments; Protein Eng.;  citation_year=1999;  citation_volume=12;  citation_pages=85-94; " /><meta name="citation_reference" content="citation_title=Phylogenetic profiles reveal evolutionary relationships within the “twilight zone” of sequence similarity; Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=1374-9; " /><meta name="citation_reference" content="citation_title=The self-organizing map; Proc. IEEE;  citation_year=1990;  citation_volume=78;  citation_pages=1464-80; " /><meta name="citation_reference" content="citation_title=Engineering applications of the self-organizing map; Proc. IEEE;  citation_year=1996;  citation_volume=84;  citation_pages=1358-84; " /><meta name="citation_reference" content="citation_title=Self-organized neural maps of human protein sequences; Protein Sci.;  citation_year=1994;  citation_volume=3;  citation_pages=507-21; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the E. coli O157 genome; Gene;  citation_year=2001;  citation_volume=276;  citation_pages=89-99; " /><meta name="citation_reference" content="citation_title=Informatics for unveiling hidden genome signatures; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=693-702; " /><meta name="citation_reference" content="citation_title=Novel phylogenetic studies of genomic sequence fragments derived from uncultured microbe mixtures in environmental and clinical samples; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=281-90; " /><meta name="citation_reference" content="citation_title=A large-scale self-organizing map (SOM) unveils sequence characteristics of a wide range of eukaryote genomes; Gene;  citation_year=2006;  citation_volume=365;  citation_pages=27-34; " /><meta name="citation_reference" content="citation_title=Sequences from almost all prokaryotic, eukaryotic, and viral genomes available could be classified according to genomes on a large-scale self-organizing map constructed with the earth simulator; J. Earth Simulator;  citation_year=2006;  citation_volume=6;  citation_pages=17-23; " /><meta name="citation_reference" content="citation_title=Substrate-induced gene-expression screening of environmental metagenome libraries for isolation of catabolic genes; Nat. Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=88-93; " /><meta name="citation_reference" content="citation_title=Direct cloning of genes encoding novel xylanases from human gut; Can. J. Microbiol.;  citation_year=2005;  citation_volume=51;  citation_pages=251-9; " /><meta name="citation_reference" content="citation_title=The genome of Pelotomaculum thermopropionicum reveals niche-associated evolution in anaerobic microbiota; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=442-8; " /><meta name="citation_reference" content="citation_title=A novel bioinformatics tool for phylogenetic classification of genomic sequence fragments derived from mixed genomes of environmental uncultured microbes; Polar Biosci.;  citation_year=2006;  citation_volume=20;  citation_pages=103-12; " /><meta name="citation_reference" content="citation_title=A genomic perspective on protein families; Science;  citation_year=1997;  citation_volume=278;  citation_pages=631-7; " /><meta name="citation_reference" content="citation_title=The role of lineage-specific gene family expansion in the evolution of eukaryotes; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1048-59; " /><meta name="citation_reference" content="citation_title=The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=5691-702; " /><meta name="citation_reference" content="citation_title=Genomics of bacteria and Archaea: the emerging dynamic view of the prokaryotic world; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=6688-719; " /><meta name="citation_reference" content="citation_title=Microbial population genomics and ecology; Curr. Opin. Microbiol.;  citation_year=2002;  citation_volume=5;  citation_pages=520-4; " /><meta name="citation_reference" content="citation_title=Biotechnological prospects from metagenomics; Curr. Opin. Biotechnol.;  citation_year=2003;  citation_volume=14;  citation_pages=303-10; " /><meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea; Science;  citation_year=2004;  citation_volume=304;  citation_pages=66-74; " /><meta name="citation_reference" content="citation_title=Exploration and grading of possible genes in 183 bacterial strains by a common fine protocol lead to new genes: Gene Trek in Prokaryote Space (GTPS); DNA Res.;  citation_year=2006;  citation_volume=13;  citation_pages=245-54; " /><meta name="citation_reference" content="citation_title=The PRINTS database: a resource for identification of protein families; Brief Bioinform.;  citation_year=2002;  citation_volume=3;  citation_pages=252-63; " /><meta name="citation_reference" content="citation_title=The 20 years of PROSITE; Nucleic Acids Res.;  citation_year=2007;  citation_volume=36;  citation_pages=D245-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/5/287/1167835/dsp018.pdf" /><meta name="description" content="Abstract.  As a result of remarkable progresses of DNA sequencing technology, vast quantities of genomic sequences have been decoded. Homology search for amino " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/5/287/421945" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/5/287/421945" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/5/287/421945");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp018';
    </script>
<title>
	Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./421945" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BetFPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmUsIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYWdhaGFtYSBJbnN0aXR1dGUgb2YgQmlvLVNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFRhbXVyYS1jaG8gMTI2NiwgTmFnYWhhbWEtc2hpLCBTaGlnYS1rZW4gNTI2LTA4MjksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FuYXlhLCBTaGlnZWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzIGFuZCBHZW5vbWVzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVlaGFyYSwgSGlyb3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hZ2FoYW1hIEluc3RpdHV0ZSBvZiBCaW8tU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVGFtdXJhLWNobyAxMjY2LCBOYWdhaGFtYS1zaGksIFNoaWdhLWtlbiA1MjYtMDgyOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJa2VtdXJhLCBUb3NoaW1pY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmFnYWhhbWEgSW5zdGl0dXRlIG9mIEJpby1TY2llbmNlIGFuZCBUZWNobm9sb2d5LCBUYW11cmEtY2hvIDEyNjYsIE5hZ2FoYW1hLXNoaSwgU2hpZ2Eta2VuIDUyNi0wODI5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQSBOb3ZlbCBCaW9pbmZvcm1hdGljcyBTdHJhdGVneSBmb3IgRnVuY3Rpb24gUHJlZGljdGlvbiBvZiBQb29ybHktQ2hhcmFjdGVyaXplZCBQcm90ZWluIEdlbmVzIE9idGFpbmVkIGZyb20gTWV0YWdlbm9tZSBBbmFseXNlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjk3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNwMDE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDkvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GVUdVRTogc2VxdWVuY2Utc3RydWN0dXJlIGhvbW9sb2d5IHJlY29nbml0aW9uIHVzaW5nIGVudmlyb25tZW50LXNwZWNpZmljIHN1YnN0aXR1dGlvbiB0YWJsZXMgYW5kIHN0cnVjdHVyZS1kZXBlbmRlbnQgZ2FwIHBlbmFsdGllczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTA7ICBjaXRhdGlvbl9wYWdlcz0yNDMtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGEgZ3Jvd3RoIGFuZCBpdHMgaW1wYWN0IG9uIHRoZSBTQ09QIGRhdGFiYXNlOiBuZXcgZGV2ZWxvcG1lbnRzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1ENDE5LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ0FUSCBjbGFzc2lmaWNhdGlvbiByZXZpc2l0ZWTigJRhcmNoaXRlY3R1cmVzIHJldmlld2VkIGFuZCBuZXcgd2F5cyB0byBjaGFyYWN0ZXJpemUgc3RydWN0dXJhbCBkaXZlcmdlbmNlIGluIHN1cGVyZmFtaWxpZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQzMTAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGZ1bmN0aW9uIGluIHByb3RlaW4gc3VwZXJmYW1pbGllcywgZnJvbSBhIHN0cnVjdHVyYWwgcGVyc3BlY3RpdmU7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzA3OyAgY2l0YXRpb25fcGFnZXM9MTExMy00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyBwcm90ZWluIGZ1bmN0aW9uIGZyb20gc2VxdWVuY2UgYW5kIHN0cnVjdHVyZTsgTmF0LiBSZXYuIE1vbC4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTk1LTEwMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdpbGlnaHQgem9uZSBvZiBwcm90ZWluIHNlcXVlbmNlIGFsaWdubWVudHM7IFByb3RlaW4gRW5nLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTg1LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgcHJvZmlsZXMgcmV2ZWFsIGV2b2x1dGlvbmFyeSByZWxhdGlvbnNoaXBzIHdpdGhpbiB0aGUg4oCcdHdpbGlnaHQgem9uZeKAnSBvZiBzZXF1ZW5jZSBzaW1pbGFyaXR5OyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTEzNzQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlbGYtb3JnYW5pemluZyBtYXA7IFByb2MuIElFRUU7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz0xNDY0LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmdpbmVlcmluZyBhcHBsaWNhdGlvbnMgb2YgdGhlIHNlbGYtb3JnYW5pemluZyBtYXA7IFByb2MuIElFRUU7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz0xMzU4LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxmLW9yZ2FuaXplZCBuZXVyYWwgbWFwcyBvZiBodW1hbiBwcm90ZWluIHNlcXVlbmNlczsgUHJvdGVpbiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz01MDctMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNvZG9uIHVzYWdlIGRpdmVyc2l0eSBvZiBiYWN0ZXJpYWwgZ2VuZXMgd2l0aCBhIHNlbGYtb3JnYW5pemluZyBtYXAgKFNPTSk6IGNoYXJhY3Rlcml6YXRpb24gb2YgaG9yaXpvbnRhbGx5IHRyYW5zZmVycmVkIGdlbmVzIHdpdGggZW1waGFzaXMgb24gdGhlIEUuIGNvbGkgTzE1NyBnZW5vbWU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjc2OyAgY2l0YXRpb25fcGFnZXM9ODktOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZm9ybWF0aWNzIGZvciB1bnZlaWxpbmcgaGlkZGVuIGdlbm9tZSBzaWduYXR1cmVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTY5My03MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIHBoeWxvZ2VuZXRpYyBzdHVkaWVzIG9mIGdlbm9taWMgc2VxdWVuY2UgZnJhZ21lbnRzIGRlcml2ZWQgZnJvbSB1bmN1bHR1cmVkIG1pY3JvYmUgbWl4dHVyZXMgaW4gZW52aXJvbm1lbnRhbCBhbmQgY2xpbmljYWwgc2FtcGxlczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbGFyZ2Utc2NhbGUgc2VsZi1vcmdhbml6aW5nIG1hcCAoU09NKSB1bnZlaWxzIHNlcXVlbmNlIGNoYXJhY3RlcmlzdGljcyBvZiBhIHdpZGUgcmFuZ2Ugb2YgZXVrYXJ5b3RlIGdlbm9tZXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzY1OyAgY2l0YXRpb25fcGFnZXM9MjctMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlcyBmcm9tIGFsbW9zdCBhbGwgcHJva2FyeW90aWMsIGV1a2FyeW90aWMsIGFuZCB2aXJhbCBnZW5vbWVzIGF2YWlsYWJsZSBjb3VsZCBiZSBjbGFzc2lmaWVkIGFjY29yZGluZyB0byBnZW5vbWVzIG9uIGEgbGFyZ2Utc2NhbGUgc2VsZi1vcmdhbml6aW5nIG1hcCBjb25zdHJ1Y3RlZCB3aXRoIHRoZSBlYXJ0aCBzaW11bGF0b3I7IEouIEVhcnRoIFNpbXVsYXRvcjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTctMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1YnN0cmF0ZS1pbmR1Y2VkIGdlbmUtZXhwcmVzc2lvbiBzY3JlZW5pbmcgb2YgZW52aXJvbm1lbnRhbCBtZXRhZ2Vub21lIGxpYnJhcmllcyBmb3IgaXNvbGF0aW9uIG9mIGNhdGFib2xpYyBnZW5lczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTg4LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXJlY3QgY2xvbmluZyBvZiBnZW5lcyBlbmNvZGluZyBub3ZlbCB4eWxhbmFzZXMgZnJvbSBodW1hbiBndXQ7IENhbi4gSi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTI1MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIFBlbG90b21hY3VsdW0gdGhlcm1vcHJvcGlvbmljdW0gcmV2ZWFscyBuaWNoZS1hc3NvY2lhdGVkIGV2b2x1dGlvbiBpbiBhbmFlcm9iaWMgbWljcm9iaW90YTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz00NDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBiaW9pbmZvcm1hdGljcyB0b29sIGZvciBwaHlsb2dlbmV0aWMgY2xhc3NpZmljYXRpb24gb2YgZ2Vub21pYyBzZXF1ZW5jZSBmcmFnbWVudHMgZGVyaXZlZCBmcm9tIG1peGVkIGdlbm9tZXMgb2YgZW52aXJvbm1lbnRhbCB1bmN1bHR1cmVkIG1pY3JvYmVzOyBQb2xhciBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTAzLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9taWMgcGVyc3BlY3RpdmUgb24gcHJvdGVpbiBmYW1pbGllczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzg7ICBjaXRhdGlvbl9wYWdlcz02MzEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgbGluZWFnZS1zcGVjaWZpYyBnZW5lIGZhbWlseSBleHBhbnNpb24gaW4gdGhlIGV2b2x1dGlvbiBvZiBldWthcnlvdGVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEwNDgtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdWJzeXN0ZW1zIGFwcHJvYWNoIHRvIGdlbm9tZSBhbm5vdGF0aW9uIGFuZCBpdHMgdXNlIGluIHRoZSBwcm9qZWN0IHRvIGFubm90YXRlIDEwMDAgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTY5MS03MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIGJhY3RlcmlhIGFuZCBBcmNoYWVhOiB0aGUgZW1lcmdpbmcgZHluYW1pYyB2aWV3IG9mIHRoZSBwcm9rYXJ5b3RpYyB3b3JsZDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NjY4OC03MTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBwb3B1bGF0aW9uIGdlbm9taWNzIGFuZCBlY29sb2d5OyBDdXJyLiBPcGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01MjAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvdGVjaG5vbG9naWNhbCBwcm9zcGVjdHMgZnJvbSBtZXRhZ2Vub21pY3M7IEN1cnIuIE9waW4uIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnZpcm9ubWVudGFsIGdlbm9tZSBzaG90Z3VuIHNlcXVlbmNpbmcgb2YgdGhlIFNhcmdhc3NvIFNlYTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDQ7ICBjaXRhdGlvbl9wYWdlcz02Ni03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwbG9yYXRpb24gYW5kIGdyYWRpbmcgb2YgcG9zc2libGUgZ2VuZXMgaW4gMTgzIGJhY3RlcmlhbCBzdHJhaW5zIGJ5IGEgY29tbW9uIGZpbmUgcHJvdG9jb2wgbGVhZCB0byBuZXcgZ2VuZXM6IEdlbmUgVHJlayBpbiBQcm9rYXJ5b3RlIFNwYWNlIChHVFBTKTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yNDUtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQUklOVFMgZGF0YWJhc2U6IGEgcmVzb3VyY2UgZm9yIGlkZW50aWZpY2F0aW9uIG9mIHByb3RlaW4gZmFtaWxpZXM7IEJyaWVmIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTI1Mi02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIDIwIHllYXJzIG9mIFBST1NJVEU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQyNDUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzUvMjg3LzExNjc4MzUvZHNwMDE4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBcyBhIHJlc3VsdCBvZiByZW1hcmthYmxlIHByb2dyZXNzZXMgb2YgRE5BIHNlcXVlbmNpbmcgdGVjaG5vbG9neSwgdmFzdCBxdWFudGl0aWVzIG9mIGdlbm9taWMgc2VxdWVuY2VzIGhhdmUgYmVlbiBkZWNvZGVkLiBIb21vbG9neSBzZWFyY2ggZm9yIGFtaW5vICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDIxOTQ1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZGViC4WMtNl1ecD3/EtNN2MuqKqU" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f287%2f421945" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f287%2f421945" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&amp;Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/5">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6252178">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252178">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6252180">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252180">1 Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6252185">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252185">2 Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6252198">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252198">3 Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6252224">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252224">4 Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6252231">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252231">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6252236">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252236">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6252239">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6252239">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/5/275/421566">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/5/299/420917">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 749-64-8126. Fax. <div class="fax">+81 749-64-8126</div>. E-mail: <a href="mailto:takaabe@nagahama-i-bio.ac.jp" target="_blank">takaabe@nagahama-i-bio.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Abe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigehiko Kanaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioinformatics and Genomes, Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigehiko+Kanaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanaya S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanaya S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigehiko Kanaya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Uehara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Uehara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Uehara H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Uehara H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Uehara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshimichi Ikemura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Nagahama Institute of Bio-Science and Technology, Tamura-cho 1266, Nagahama-shi, Shiga-ken 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshimichi+Ikemura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikemura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikemura T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshimichi Ikemura</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 287–297, <a href='https://doi.org/10.1093/dnares/dsp018'>https://doi.org/10.1093/dnares/dsp018</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 October 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 July 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 September 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="421945" href="/dnaresearch/article-pdf/16/5/287/1167835/dsp018.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takashi Abe, Shigehiko Kanaya, Hiroshi Uehara, Toshimichi Ikemura;  A Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses, <em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 287–297, <a href='https://doi.org/10.1093/dnares/dsp018'>https://doi.org/10.1093/dnares/dsp018</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/421945?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/421945?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/421945?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/421945?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/421945?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/421945?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/421945?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Novel%20Bioinformatics%20Strategy%20for%20Function%20Prediction%20of%20Poorly-Characterized%20Protein%20Genes%20Obtained%20from%20Metagenome%20Analyses&amp;publicationDate=2009-10-03&amp;volumeNum=16&amp;issueNum=5&amp;author=Abe%2C%20Takashi%3B%20Kanaya%2C%20Shigehiko&amp;startPage=287&amp;endPage=297&amp;contentId=10.1093%2Fdnares%2Fdsp018&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f287%2f421945" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6252178 id="6252178" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>As a result of remarkable progresses of DNA sequencing technology, vast quantities of genomic sequences have been decoded. Homology search for amino acid sequences, such as BLAST, has become a basic tool for assigning functions of genes/proteins when genomic sequences are decoded. Although the homology search has clearly been a powerful and irreplaceable method, the functions of only 50% or fewer of genes can be predicted when a novel genome is decoded. A prediction method independent of the homology search is urgently needed. By analyzing oligonucleotide compositions in genomic sequences, we previously developed a modified Self-Organizing Map ‘BLSOM’ that clustered genomic fragments according to phylotype with no advance knowledge of phylotype. Using BLSOM for di-, tri- and tetrapeptide compositions, we developed a system to enable separation (self-organization) of proteins by function. Analyzing oligopeptide frequencies in proteins previously classified into COGs (clusters of orthologous groups of proteins), BLSOMs could faithfully reproduce the COG classifications. This indicated that proteins, whose functions are unknown because of lack of significant sequence similarity with function-known proteins, can be related to function-known proteins based on similarity in oligopeptide composition. BLSOM was applied to predict functions of vast quantities of proteins derived from mixed genomes in environmental samples.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;batch learning SOM&quot;">batch learning SOM</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;oligopeptide frequency&quot;">oligopeptide frequency</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein function&quot;">protein function</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenome">metagenome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alignment-free clustering&quot;">alignment-free clustering</a></div>                            <h2 scrollto-destination=6252180 id="6252180" class="section-title" data-legacy-id=s1>1 Introduction</h2>
<p>A large number of protein genes whose functions cannot be predicted by the sequence homology search have progressively accumulated in the International DNA Sequence Databanks and remain of very little use to science and industry. Currently, the situation has become worse, since the number of proteins with unknown functions, such as those found in novel unculturable species derived from environmental samples, has increased rapidly. Considering that such annotation is done based on protein sequence alignment, other methods that are complementary to the sequence alignment would be required. One approach for the prediction of function is detection of similarity in three-dimensional (3D) structure of proteins, by direct comparison of coordinates or threading approach. Another successful approach is an integration of 3D structure information with one-dimensional sequence similarity, and the programs and the structural domain databases (e.g. FUGUE, SCOP and CATH) have been published.<sup><a reveal-id="DSP018C1  DSP018C5" data-open="DSP018C1  DSP018C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> Protein sequence alignments in twilight-zone of similarity have also been studied intensively.<sup><a reveal-id="DSP018C6  DSP018C8" data-open="DSP018C6  DSP018C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> As described earlier, however, the number of proteins with unknown functions, such as those found in novel unculturable species derived from environmental samples, has increased rapidly, and in light of technical limitations and the cost required, it is difficult to apply routinely the higher-order structure determination of proteins for prediction of functions of the increasingly vast quantity of such novel proteins.</p><p>In the present study, we have developed a sequence alignment- and structural information-free clustering method that will complement the function prediction for proteins for which no or only ambiguous prediction could be obtained even by a combination of conventional tools. Self-organizing map (SOM) is an unsupervised clustering algorithm developed by Kohonen <em>et al</em>.,<sup><a reveal-id="DSP018C9" data-open="DSP018C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSP018C10" data-open="DSP018C10" class="link link-ref link-reveal xref-bibr">10</a></sup> which enables us to efficiently and easily interpret the clustering of high-dimensional data utilizing visualization on a two-dimensional plane. More than 15 years ago, Ferran <em>et al</em>.<sup><a reveal-id="DSP018C11" data-open="DSP018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> performed pioneering and extensive SOM analyses for dipeptide compositions in ∼2000 human proteins stored in the SwissProt Database. They reported clustering (self-organization) of the proteins by both function and higher-order structure with no advance knowledge during the SOM calculation. Although this unsupervised clustering method can be considered useful for predicting protein function, the study was conducted long before decoding of genome sequences, and at that time proteins of unknown function were rare. Furthermore, because a long computation time was required for the conventional SOM of the dipeptide compositions (400-dimensional vectorial data), even using high-performance computers at that time, and because the final map was dependent on both the order of the vectorial data input and the initial vectorial values, the SOM method has rarely been used for prediction of protein function.</p><p>Previously, we developed a modified SOM (batch-learning SOM: BLSOM), which depends on neither the order of data input nor the initial condition, for codon frequencies in gene sequences<sup><a reveal-id="DSP018C12" data-open="DSP018C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and oligonucleotide frequencies in genomic sequences.<sup><a reveal-id="DSP018C13" data-open="DSP018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The BLSOM recognizes species-specific characteristics of codon or oligonucleotide composition, permitting clustering (self-organization) of genes and genomic sequences according to species without the need for species information during BLSOM calculation. Various high-performance supercomputers are now available for biological studies, and the BLSOM, although not the conventional SOM, is suitable for actualizing high-performance parallel-computing<sup><a reveal-id="DSP018C14  DSP018C16" data-open="DSP018C14  DSP018C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> and thus for a large-scale computation of a huge quantity of data using a high-performance supercomputer. In our previous studies, we used the BLSOM for phylogenetic classification of genomic sequence fragments obtained from mixed genomes of environmental microorganisms by analyzing tetranucleotide frequencies (256-dimensional vectorial data).<sup><a reveal-id="DSP018C17  DSP018C20" data-open="DSP018C17  DSP018C20" class="link link-ref link-reveal xref-bibr">17–20</a></sup> The unsupervised method clustered the genomic fragment sequences (e.g. 10 kb) according to phylotype without phylotype information.</p><p>For the present study, we developed BLSOM to predict protein function on the basis of similarity in oligopeptide composition (di-, tri- and tetrapeptide compositions in this study) of proteins. We searched the BLSOM condition to faithfully reproduce separation (self-organization) of function-known proteins according to functional categories. As a test data set for this purpose, we focused on protein genes which were classified into COGs (clusters of orthologous groups of proteins) by an NCBI group.<sup><a reveal-id="DSP018C21" data-open="DSP018C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The NCBI COG (abbreviated as NcCOG) is the functional category identified with best-hit relationships between the completely sequenced genomes using the sequence homology search. The NcCOG is not a simple, decisive categorization because of the co-orthologous relationship of protein genes produced by duplication of orthologous genes subsequent to a speciation event.<sup><a reveal-id="DSP018C22  DSP018C24" data-open="DSP018C22  DSP018C24" class="link link-ref link-reveal xref-bibr">22–24</a></sup> The NcCOG, however, is undoubtedly a useful categorization of proteins according to function, especially from the view point of the method-oriented purpose of the present study because proteins belonging to a single NcCOG most likely have the same function. In the first approach, by analyzing di-, tri- and tetrapeptide compositions in the 83 962 proteins classified by the NCBI group<sup><a reveal-id="DSP018C21" data-open="DSP018C21" class="link link-ref link-reveal xref-bibr">21</a></sup> into 2853 function-known NcCOGs, we could obtain BLSOM conditions that faithfully reproduced the NcCOG classifications without an advanced knowledge of the NcCOG information. Then, we applied the BLSOMs to predict functions of proteins obtained from environmental mixed genomes.</p>                            <h2 scrollto-destination=6252185 id="6252185" class="section-title" data-legacy-id=s2>2 Materials and methods</h2>
                            <h3 scrollto-destination=6252186 id="6252186" class="section-title" data-legacy-id=s2a>2.1 Amino acid sequences</h3>
<p>Amino acid sequences were obtained from <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/GenBank/">http://www.ncbi.nlm.nih.gov/GenBank/</a>. Proteins shorter than 200 amino acids in length were not included in the present study. To reduce the computation time as described by Ferran <em>et al</em>.,<sup><a reveal-id="DSP018C11" data-open="DSP018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> BLSOM was constructed with tripeptide frequencies of the degenerate 11 groups of residues: {V, L, I}, {T, S}, {N, Q}, {E, D}, {K, R, H}, {Y, F, W}, {M}, {P}, {C}, {A} and {G}. BLSOM was also constructed with tetrapeptide frequencies of degenerate six groups of residues: {V, L, I, M}, {T, S, P, G, A}, {E, D, N, Q}, {K, R, H}, {Y, F, W} and {C}.<sup><a reveal-id="DSP018C11" data-open="DSP018C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p>                            <h3 scrollto-destination=6252188 id="6252188" class="section-title" data-legacy-id=s2b>2.2 Blsom construction</h3>
<p>We previously modified the conventional Kohonen's SOM<sup><a reveal-id="DSP018C9" data-open="DSP018C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSP018C10" data-open="DSP018C10" class="link link-ref link-reveal xref-bibr">10</a></sup> for genome informatics to make the learning process and resulting map independent of the order of data input.<sup><a reveal-id="DSP018C12" data-open="DSP018C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP018C13" data-open="DSP018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In the case of the conventional SOM, the initial vectorial data are set at a random value, but this results in a final map which is changed by each initial data set. In contrast, in our BLSOM for genome analyses, we could obtain a reproducible map using the first and second primary components in the principal component analysis of oligonucleotide composition for the initial vectorial data. The initial weight vectors (<em>w</em><sub><em>ij</em></sub>) were arranged in the two-dimensional lattice denoted by <em>i</em> (=0, 1, …, <em>I</em> − 1) and <em>j</em> (=0, 1, …, <em>J</em> − 1). <em>σ</em><sub>1</sub> and <em>σ</em><sub>2</sub> were the standard deviations of the first and second principal components, respectively, and <em>J</em> was defined by the nearest integer greater than (<em>σ</em><sub>2</sub>/<em>σ</em><sub>1</sub>) × <em>I</em>. Under this condition, <em>I</em> was set to attain the map size which provides a mean ca. 8 data points per lattice point, as described previously.<sup><a reveal-id="DSP018C12" data-open="DSP018C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP018C13" data-open="DSP018C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>When this strategy was applied to dipeptide composition in proteins in a preliminary study, the first component tended to reflect the length of proteins. Although the length of a protein undoubtedly relates to its function, the length sometimes differs significantly even between proteins with the same function. Furthermore, to predict functions of proteins with high novelty, it would become important to compare the oligopeptide composition of proteins with similar functions derived from the genomes with divergent evolutionary origins, and lengths of proteins with similar functions may differ significantly from each other. Additionally, when the protein-coding regions are estimated computationally from sequences of poorly characterized genomes, including those obtained by metagenome analyses, it is often difficult to specify accurately the initiation codon position and thus the protein length. To develop a method that is less dependent on protein length, we provided a window of 200 amino acids that is moved with a 50-amino acid step. For the final, residual sequence shorter than 200 amino acids, a 200-amino acid window was newly set from the last amino acid of the protein. The BLSOM was constructed for all these overlapped 200-amino acid sequences according to the method described for the oligonucleotide composition in genomic sequnces.<sup><a reveal-id="DSP018C13" data-open="DSP018C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Introduction of the window with the sliding step enables us to analyze multi-functional and multidomain proteins, which originated often from the fusion of distinct proteins during evolution, collectively with smaller proteins. Only in Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a>A and Table <a reveal-id="DSP018TB1" data-open="DSP018TB1" class="link link-reveal link-table xref-fig">1</a>, dipeptide frequencies in the full-length sequences of proteins were analyzed, resulting in a lower level of proper clustering according to the COG category.</p>                        <a id="6252191" scrollto-destination="6252191"></a>
<div id="DSP018F1" data-id="DSP018F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01801.jpeg?Expires=2147483647&Signature=g36ajmMkRPQYBSaB4MrITVk1bbfeWX8SR6uJe1usNKZY1RdughHW4uasOTHlRqbf-kPpdcK4P82kX1QkYTcFZk3UHuYhkJSaKX9whWUAenpYZI1Mzmf-rQ55p5CxCVmfD~HTdwzd0uLUkpVBWn8jPwFRqjD8BpuPwJov8W7fe7e5TY2clWOe-Rapssu844W5nYcLPGrit7~r5uDnSgjH9EIjYIpweHkQy3DwnPBp7DVUDfS5r~cj77uPFn0a9Hg-ORaT1vcjk0NGcvHMbnh6idNJMMxUd3PdDg~iQOqtin3tCROsuBkuWQJ8BzgM~J8QTJAwb9uvOyosWNo0DVcvng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of pure lattice points. (A) Di20-w/o window; 20-amino acid groups, full-length sequences without window. (B) Di20-W200; 20-amino acid groups, 200-amino acid windows. (C) Di11-W200; 11-amino acid groups, 200-amino acid windows. (D) Tri11-W100; 11-amino acid groups, 100-amino acid windows. (E) Tri11-W200; 11-amino acid groups, 200-amino acid windows. (F) Tetra6-W200; 6-amino acid groups, 200-amino acid windows. ‘Pure lattice points’ are colored in red and ‘mixed lattice points’ are colored in blue. Lattice points without sequence or with one sequence are shown in white." path-from-xml="dsp01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252191/dsp01801.jpeg" path-from-xml="dsp01801.jpeg" target="_blank">View large</a><a section="6252191" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01801.jpeg?Expires=2147483647&Signature=BRNb7mWxs0nyFeefIGcVWcCDSWcZaZlJpsNiNHY4G-hRer4xd3bHEK5EzZrQHalQdSZb~ZrztZvnwskTJ5siVIqUu9nw1OjHfI5KabyBLboAEmBPqftwG6y4ybePhx6qHkvSWo9k8FtuWzgwWXExKcFYoe~Sj9NN1FsCMoUaUK9Pcw611Kq-GmhrP7IHDkwt-9XS3mj8tgNWRGRa2m4~bXzpnrIn0XzVQm1xFME5aZ643HsAjjGtyys8Lgwc67vYDXLk9IxSlH3szrVKtunP-qk99iSyk1~M-zBzGNNGMtHId9QMJAf2ANh6xq5JmDcAvm3FIji3djPljc55XrHcEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252191&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of pure lattice points. (A) Di20-w/o window; 20-amino acid groups, full-length sequences without window. (B) Di20-W200; 20-amino acid groups, 200-amino acid windows. (C) Di11-W200; 11-amino acid groups, 200-amino acid windows. (D) Tri11-W100; 11-amino acid groups, 100-amino acid windows. (E) Tri11-W200; 11-amino acid groups, 200-amino acid windows. (F) Tetra6-W200; 6-amino acid groups, 200-amino acid windows. ‘Pure lattice points’ are colored in red and ‘mixed lattice points’ are colored in blue. Lattice points without sequence or with one sequence are shown in white.</p></div></div><div content-id="DSP018F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01801.jpeg?Expires=2147483647&Signature=g36ajmMkRPQYBSaB4MrITVk1bbfeWX8SR6uJe1usNKZY1RdughHW4uasOTHlRqbf-kPpdcK4P82kX1QkYTcFZk3UHuYhkJSaKX9whWUAenpYZI1Mzmf-rQ55p5CxCVmfD~HTdwzd0uLUkpVBWn8jPwFRqjD8BpuPwJov8W7fe7e5TY2clWOe-Rapssu844W5nYcLPGrit7~r5uDnSgjH9EIjYIpweHkQy3DwnPBp7DVUDfS5r~cj77uPFn0a9Hg-ORaT1vcjk0NGcvHMbnh6idNJMMxUd3PdDg~iQOqtin3tCROsuBkuWQJ8BzgM~J8QTJAwb9uvOyosWNo0DVcvng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of pure lattice points. (A) Di20-w/o window; 20-amino acid groups, full-length sequences without window. (B) Di20-W200; 20-amino acid groups, 200-amino acid windows. (C) Di11-W200; 11-amino acid groups, 200-amino acid windows. (D) Tri11-W100; 11-amino acid groups, 100-amino acid windows. (E) Tri11-W200; 11-amino acid groups, 200-amino acid windows. (F) Tetra6-W200; 6-amino acid groups, 200-amino acid windows. ‘Pure lattice points’ are colored in red and ‘mixed lattice points’ are colored in blue. Lattice points without sequence or with one sequence are shown in white." path-from-xml="dsp01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252191/dsp01801.jpeg" path-from-xml="dsp01801.jpeg" target="_blank">View large</a><a section="6252191" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01801.jpeg?Expires=2147483647&Signature=BRNb7mWxs0nyFeefIGcVWcCDSWcZaZlJpsNiNHY4G-hRer4xd3bHEK5EzZrQHalQdSZb~ZrztZvnwskTJ5siVIqUu9nw1OjHfI5KabyBLboAEmBPqftwG6y4ybePhx6qHkvSWo9k8FtuWzgwWXExKcFYoe~Sj9NN1FsCMoUaUK9Pcw611Kq-GmhrP7IHDkwt-9XS3mj8tgNWRGRa2m4~bXzpnrIn0XzVQm1xFME5aZ643HsAjjGtyys8Lgwc67vYDXLk9IxSlH3szrVKtunP-qk99iSyk1~M-zBzGNNGMtHId9QMJAf2ANh6xq5JmDcAvm3FIji3djPljc55XrHcEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252191&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of pure lattice points. (A) Di20-w/o window; 20-amino acid groups, full-length sequences without window. (B) Di20-W200; 20-amino acid groups, 200-amino acid windows. (C) Di11-W200; 11-amino acid groups, 200-amino acid windows. (D) Tri11-W100; 11-amino acid groups, 100-amino acid windows. (E) Tri11-W200; 11-amino acid groups, 200-amino acid windows. (F) Tetra6-W200; 6-amino acid groups, 200-amino acid windows. ‘Pure lattice points’ are colored in red and ‘mixed lattice points’ are colored in blue. Lattice points without sequence or with one sequence are shown in white.</p></div></div>                        <a id="6252192" scrollto-destination="6252192"></a>
<div content-id="DSP018TB1" class="table-modal"><div class="table-wrap"><div id="DSP018TB1" class="table-wrap-title" data-id="DSP018TB1"><div class="label -label">Table 1</div><div class="caption"><p>Occurrence levels of pure lattice points</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Analysis condition </th><th>Proportion (%) of pure lattice points </th></tr></thead><tbody><tr><td>Di20-w/o window </td><td>15.2 </td></tr><tr><td>Di20-W200 </td><td>35.1 </td></tr><tr><td>Di11-W200 </td><td>10.7 </td></tr><tr><td>Tri11-W100 </td><td>19.1 </td></tr><tr><td>Tri11-W200 </td><td>45.4 </td></tr><tr><td>Tetra6-W200 </td><td>18.1 </td></tr></tbody></table></div><div content-id="DSP018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Analysis condition </th><th>Proportion (%) of pure lattice points </th></tr></thead><tbody><tr><td>Di20-w/o window </td><td>15.2 </td></tr><tr><td>Di20-W200 </td><td>35.1 </td></tr><tr><td>Di11-W200 </td><td>10.7 </td></tr><tr><td>Tri11-W100 </td><td>19.1 </td></tr><tr><td>Tri11-W200 </td><td>45.4 </td></tr><tr><td>Tetra6-W200 </td><td>18.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6252192" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP018TB1" class="table-wrap-title" data-id="DSP018TB1"><div class="label -label">Table 1</div><div class="caption"><p>Occurrence levels of pure lattice points</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Analysis condition </th><th>Proportion (%) of pure lattice points </th></tr></thead><tbody><tr><td>Di20-w/o window </td><td>15.2 </td></tr><tr><td>Di20-W200 </td><td>35.1 </td></tr><tr><td>Di11-W200 </td><td>10.7 </td></tr><tr><td>Tri11-W100 </td><td>19.1 </td></tr><tr><td>Tri11-W200 </td><td>45.4 </td></tr><tr><td>Tetra6-W200 </td><td>18.1 </td></tr></tbody></table></div><div content-id="DSP018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Analysis condition </th><th>Proportion (%) of pure lattice points </th></tr></thead><tbody><tr><td>Di20-w/o window </td><td>15.2 </td></tr><tr><td>Di20-W200 </td><td>35.1 </td></tr><tr><td>Di11-W200 </td><td>10.7 </td></tr><tr><td>Tri11-W100 </td><td>19.1 </td></tr><tr><td>Tri11-W200 </td><td>45.4 </td></tr><tr><td>Tetra6-W200 </td><td>18.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6252192" target="_blank">View Large</a></div></div><p>The present BLSOM was suitable for actualizing high-performance parallel-computing with high-performance supercomputers. Using 136 CPUs of ‘the Earth Simulator’, calculations in this study could be performed primarily within 2 days. The BLSOM program suitable for PC cluster systems and a PC program for mapping of new sequences on a large-scale BLSOM constructed with high-performance supercomputers could be obtained from our group (<a href="mailto:takaabe@nagahama-i-bio.ac.jp" target="_blank">takaabe@nagahama-i-bio.ac.jp</a>) and UNTROD, Inc. (<a href="mailto:k_wada@nagahama-i-bio.ac.jp" target="_blank">k_wada@nagahama-i-bio.ac.jp</a>). Twenty thousands protein sequences could be mapped within a day using the PC system.</p>                            <h3 scrollto-destination=6252194 id="6252194" class="section-title" data-legacy-id=s2c>2.3 Assignment of NcCOGs highly associated with each other</h3>
<p>To examine whether the sharing of one lattice point with different NcCOG sequences reflected similarity of function, we focused on all lattice points that contained 200-amino acid fragments from more than one NcCOG (designated as ‘mixed lattice points’) and then selected the lattice points that had more than four fragments at least of two NcCOGs in order to prevents cases of mere, accidental association; it should be noted that the number of lattice points on each BLSOM was designed to have eight data per lattice point on average. Next, for each of the mixed lattice points thus selected, we listed all pairs of NcCOGs having more than four fragments and summed up the pairs for all of the mixed lattice points. Then, for each pair of NcCOGs, the number of lattice points giving the pair was counted, and the pair found only once was omitted to reduce the case of an accidental association.</p>                            <h3 scrollto-destination=6252196 id="6252196" class="section-title" data-legacy-id=s2d>2.4 Function prediction for protein-gene candidates obtained by metagenome analyses</h3>
<p>For each Sargasso protein, a 200-amino acid window moved with a 10-amino acid step was provided and each 200-amino acid segment was mapped on a BLSOM constructed in advance with NcCOG sequences, by identifying the lattice-point with the minimum Euclidean distances in the multidimensional space, as published previously.<sup><a reveal-id="DSP018C14" data-open="DSP018C14" class="link link-ref link-reveal xref-bibr">14</a></sup> As to every lattice point on which Sargasso fragments were mapped, the most abundant and the second abundant NcCOG segments greater than three were identified, and the mapped Sargasso segments were tentatively assumed to belong to the NcCOGs. Considering the finding that proteins belonging to different but functionally related COGs were often associated with each other on BLSOMs, the second most abundant NcCOG was included in the analysis, but the results were similar to those of the analysis in which only the most abundant NcCOG segment was chosen. Finally, when more than 60% of the number of the 200-amino acid fragments derived from one Sargasso protein gave the same NcCOG category, the Sargasso protein was assigned to that COG and designated as SomCOG, showing that the respective Sargasso protein was assigned to this COG by BLSOM. The 10-amino acid step, rather than the 50-amino acid used for the BLSOM construction, was set for this mapping process in order to reduce the position effect of the 200-amino acid segmentation. This provided five times more mapping data and reduced in part the sequence length-dependency of the prediction accuracy. Although this mapping could be conducted with PC-level computers, BLSOMs could be constructed by using high-performance supercomputers. The 10-amino acid step was not used for BLSOM construction because of the five times more computation time needed.</p>                            <h2 scrollto-destination=6252198 id="6252198" class="section-title" data-legacy-id=s3>3 Results</h2>
                            <h3 scrollto-destination=6252200 id="6252200" class="section-title" data-legacy-id=s3a>3.1 Blsoms for proteins belonging to function-known NcCOGs</h3>
<p>For the test data set to examine whether proteins are clustered according to function by BLSOM, we chose proteins that were classified into 2853 function-known COGs by an NCBI group<sup><a reveal-id="DSP018C21" data-open="DSP018C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (NcCOGs) and analyzed oligopeptide frequencies in 83 962 proteins belonging to the function-known NcCOGs. Ferran <em>et al</em>.<sup><a reveal-id="DSP018C11" data-open="DSP018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> reported, by using the conventional SOM, that the SOM for dipeptide composition after grouping of 20 amino acids into 11 categories (11<sup>2</sup> = 121 dimensional vectorial data), in which amino acids having similar physico-chemical properties were grouped as the same residue (see Materials and methods), led to a clustering that was similar to the one obtained with the 400 (=20<sup>2</sup>) dimensional vectorial representation for 20 amino acids. This grouping of amino acids resulted in a substantial reduction in the computation time. To further reduce the computation time, they also attempted the amino-acid grouping into six categories (6<sup>2</sup> = 36 dimensional data; see Materials and methods) and found reduction in proper clustering into functional categories in the case of the dipeptide frequencies. Taking the pioneering findings by Ferran <em>et al</em>.<sup><a reveal-id="DSP018C11" data-open="DSP018C11" class="link link-ref link-reveal xref-bibr">11</a></sup> into account, we investigated four different BLSOM conditions to determine which gave the best accuracy and to what degree similar results were obtained among these conditions. One was the BLSOM for the dipeptide composition of 20 amino acids (abbreviated as Di20-BLSOM). The second and the third were the BLSOMs for the dipeptide and the tripeptide composition after grouping into 11 categories, 121 (=11<sup>2</sup>) or 1331 (=11<sup>3</sup>) dimensional data (abbreviated as Di11- or Tri11-BLSOM, respectively). The fourth was the BLSOM for the tetrapeptide composition after grouping into six categories, 1296 (=6<sup>4</sup>) dimensional data (Tetra6-BLSOM). BLSOMs for much higher dimensional data such as those for the tripeptide composition of 20 amino acids (8000-dimensional data) and for the tetrapeptide composition after grouping 11 categories (14 641-dimensional data) were impossible in this study because of the limitation of resources of the supercomputers currently available to our group.</p><p>In order to introduce a method that is less dependent on the amino acid sequence length (see Materials and methods), we provided a window of 200 amino acids that is moved with a 50-amino acid step for proteins longer than 200 amino acids, and the BLSOM was constructed for the overlapped 200-amino acid sequences (a total of 472 572 sequences). Introduction of a window with the sliding step enabled us to analyze multifunctional and multidomain proteins, which originated often from the fusion of distinct proteins during evolution, collectively with smaller proteins.</p><p>The most critical part of the present study is to know the level at which each lattice point on a BLSOM contains fragments derived from a single NcCOG. The number of the function-known NcCOG categories is 2835, and the map size of a BLSOM was chosen so as to provide a mean ca. 8 data points per lattice point on the BLSOM. If sequences were randomly distributed, the probability that all fragments associated within one lattice point were derived from a single NcCOG by chance should be extremely low, e.g. (1/2853)<sup>8</sup> =2.3 × 10<sup>−28</sup>, whereas this value should depend on the number of fragments derived from proteins belonging to the respective NcCOG. We designate here the lattice point that contained fragments derived from only one NcCOG as a ‘pure lattice point’. By this definition, the lattice points that contained only one sequence fragment were not assigned to the pure lattice point, although such lattice points were minor.</p><p>Considering that the occurrence probability of the pure lattice point as an accidental event is extremely low, we next compared this occurrence level among six different BLSOMs listed in Table <a reveal-id="DSP018TB1" data-open="DSP018TB1" class="link link-reveal link-table xref-fig">1</a>. To graphically show the difference among these BLSOMs, pure lattice points were colored as red and ‘mixed lattice points’, which contained fragments derived from different NcCOGs, were colored as blue (Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a>). Even no NcCOG category information was given during BLSOM calculation, a high percentage of pure lattice points, i.e. clustering (self organization) of proteins according to one NcCOG category, was obtained, and pure lattice points distributed all over the maps. The highest occurrence level of pure lattice points was observed on the Tri11-BLSOM for 200-amino acid fragments (Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a>E); ∼45% of lattice points of the Tri11-BLSOM contained sequences derived from one NcCOG (Table <a reveal-id="DSP018TB1" data-open="DSP018TB1" class="link link-reveal link-table xref-fig">1</a>). Shorting of the fragment size into 100 amino acids reduced significantly the occurrence level of pure lattice points (Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a>D). The dipeptide frequencies in the full-length sequences of proteins (Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a>A) gave a lower occurrence of pure lattice points than the dipeptide frequencies in the 200-amino acid window (Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a>B).</p><p>Although the results in Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a> and Table <a reveal-id="DSP018TB1" data-open="DSP018TB1" class="link link-reveal link-table xref-fig">1</a> show that the occurrence level of pure lattice points was many orders of magnitude higher than that of random association, it was not clear in what way the 200-amino acid segments belonging to a single NcCOG were distributed on a BLSOM. In Fig. <a reveal-id="DSP018F2" data-open="DSP018F2" class="link link-reveal link-table xref-fig">2</a>, the number of sequences classified into each pure lattice-point on a BLSOM was shown by the height of the vertical bar with a color representing each of the 20 NcCOG cases. These 20 examples were chosen because they were well separated from each other on a BLSOM and thus were easily visualized on one plane. Sequences belonging to a single NcCOG were localized not in a single lattice point, but in the neighboring points, resulting in a high peak composed of neighboring high bars. A few high peaks with the same color located far apart from each other were also observed. Detailed inspection showed that these detached high peaks are mostly due to the different 200-amino acid segments (e.g. anterior and posterior portions) derived from one protein, which have distinct oligonucleotide compositions and likely represented distinct structural and/or functional domains of the respective protein. This type of distinct but major peak may be informative for prediction of functions of multidomain, multifunctional proteins.</p>                        <a id="6252206" scrollto-destination="6252206"></a>
<div id="DSP018F2" data-id="DSP018F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01802.jpeg?Expires=2147483647&Signature=bglZZtN8QuhjoOBUEDRxl72Aoog03XDuXB5Gazn6jYy2MOi7gkPxrcMx1YpytZhmvC6sd0UMqgV47RxpM8aWLUENJohy6dndswKhCMGdfaPI8dYsyCEGwaVvrLJtBuGpvffnCSCkS9mESQvnc0bxO8Ycg8HGQtFmiDEy6fvQ8q0BW2zf8SwSuBFAmOVurKgdTnImFBco7VY97Y0sr4n7jUl7ws-GSAexaG8hgkBdH4Mu3xpDl4AK7vuBiZZ6XfLNLjnMKWA-DcHhChkanWdIc9-3exPDtM9ZKw5d6Ka5U3Nl6GT9W0qw~gGLFObvFUnON5-LFAdUubFrIH15qonMnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clustering of protein sequences according to COG. (A) Di20-W200. (B) Tri11-W200. (C) Tetra6-W200. Sequences of 20 COG examples are presented. Numbers of sequences classified into each lattice point are presented by the height of the vertical bar with a color representing each of the 20 NcCOGs." path-from-xml="dsp01802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252206/dsp01802.jpeg" path-from-xml="dsp01802.jpeg" target="_blank">View large</a><a section="6252206" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01802.jpeg?Expires=2147483647&Signature=23nsrjASShv81-zShn4JehLFpV5nB9247gso1~jdGbVHSdO~Y3rF7MHUMo6tMKRLMjt9YDf2gnYrbcNVAZKcKXymE3-LGf2DPOmj1UwmMe~K-vbjaIfOtQ0kfzdcxNe4GoUAMVE1~3BDelNgxrVV6FauqmYUnWsZLmdWRmvSGmB7kmj1EgxkERmS-oIyEnrylfh86UmhkC3aQS2J5rKx5dkgBiodGvolCqemHxav5XzIc1125gLnFGgQQ8GnaDlIPCvotlO5Zqsx7SUu4r8kk3ybCEFPAdr5-vQ0eujy6~HGeVBSwQALknjyZmoaO-6Ik15fdvqrixnqM1faxKI88w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252206&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clustering of protein sequences according to COG. (A) Di20-W200. (B) Tri11-W200. (C) Tetra6-W200. Sequences of 20 COG examples are presented. Numbers of sequences classified into each lattice point are presented by the height of the vertical bar with a color representing each of the 20 NcCOGs.</p></div></div><div content-id="DSP018F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01802.jpeg?Expires=2147483647&Signature=bglZZtN8QuhjoOBUEDRxl72Aoog03XDuXB5Gazn6jYy2MOi7gkPxrcMx1YpytZhmvC6sd0UMqgV47RxpM8aWLUENJohy6dndswKhCMGdfaPI8dYsyCEGwaVvrLJtBuGpvffnCSCkS9mESQvnc0bxO8Ycg8HGQtFmiDEy6fvQ8q0BW2zf8SwSuBFAmOVurKgdTnImFBco7VY97Y0sr4n7jUl7ws-GSAexaG8hgkBdH4Mu3xpDl4AK7vuBiZZ6XfLNLjnMKWA-DcHhChkanWdIc9-3exPDtM9ZKw5d6Ka5U3Nl6GT9W0qw~gGLFObvFUnON5-LFAdUubFrIH15qonMnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clustering of protein sequences according to COG. (A) Di20-W200. (B) Tri11-W200. (C) Tetra6-W200. Sequences of 20 COG examples are presented. Numbers of sequences classified into each lattice point are presented by the height of the vertical bar with a color representing each of the 20 NcCOGs." path-from-xml="dsp01802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252206/dsp01802.jpeg" path-from-xml="dsp01802.jpeg" target="_blank">View large</a><a section="6252206" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01802.jpeg?Expires=2147483647&Signature=23nsrjASShv81-zShn4JehLFpV5nB9247gso1~jdGbVHSdO~Y3rF7MHUMo6tMKRLMjt9YDf2gnYrbcNVAZKcKXymE3-LGf2DPOmj1UwmMe~K-vbjaIfOtQ0kfzdcxNe4GoUAMVE1~3BDelNgxrVV6FauqmYUnWsZLmdWRmvSGmB7kmj1EgxkERmS-oIyEnrylfh86UmhkC3aQS2J5rKx5dkgBiodGvolCqemHxav5XzIc1125gLnFGgQQ8GnaDlIPCvotlO5Zqsx7SUu4r8kk3ybCEFPAdr5-vQ0eujy6~HGeVBSwQALknjyZmoaO-6Ik15fdvqrixnqM1faxKI88w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252206&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clustering of protein sequences according to COG. (A) Di20-W200. (B) Tri11-W200. (C) Tetra6-W200. Sequences of 20 COG examples are presented. Numbers of sequences classified into each lattice point are presented by the height of the vertical bar with a color representing each of the 20 NcCOGs.</p></div></div>                            <h3 scrollto-destination=6252207 id="6252207" class="section-title" data-legacy-id=s3b>3.2 NcCOGs highly associated with each other on BLSOMs</h3>
<p>Protein sequences contain all the following information: (i) evolutionary origin of the sequence; (ii) functional requirements (e.g. those needed to form active sites, such as functional motifs); and (iii) structural demands (e.g. the need to form a certain 3D structure or to fit into a membrane medium). Proteins belonging to a single NcCOG should have all the common information. At the same time, it should be stressed that the paralogous genes have primarily the same/similar functions. When we consider the prediction of function of the proteins derived from poorly characterized genomes such as those analyzed in metagenomic approaches, proteins that originated by diversifications of one gene during evolution and likely belong to different NcCOGs (e.g. paralogous genes) should become important because they have likely the same/similar functions. In other words, the COG category may be too strict to judge the level of clustering of proteins according to function.</p><p>Proteins with the same/similar functions such as those belonging to one paralogous group might be associated with each other on BLSOMs. To examine whether the sharing of one lattice point with different COG sequences reflected similarity of protein function, we focused on mixed lattice points, which had sequences derived from different NcCOGs, and selected a set of NcCOGs whose sequences were highly associated with each on Di20-, Tri11- and Tetra6-BLSOMs, as described in Materials and methods. Table <a reveal-id="DSP018TB2" data-open="DSP018TB2" class="link link-reveal link-table xref-fig">2</a> lists the associated NcCOG pairs commonly found for the three BLSOMs along with the annotation of the NcCOG. When we referred to their annotations, 10 out of 11 NcCOG pairs were found to have a closely related function (e.g. ATPase, carboxylase or permease) for each pair, showing that the BLSOM can cluster proteins according to function. In the case of the exceptional one pair (COGs 0515 and 5099), a significant sequence similarity could not found. However, after fine tuning of conditions for a dot-matrix method, a weak local similarity for 10 segments was detected, whereas the segments were not linearly arranged along the primary sequence. Detailed study on proteins with such local, weak similarity will be one of our future works.</p>                        <a id="6252210" scrollto-destination="6252210"></a>
<div content-id="DSP018TB2" class="table-modal"><div class="table-wrap"><div id="DSP018TB2" class="table-wrap-title" data-id="DSP018TB2"><div class="label -label">Table 2</div><div class="caption"><p>COG pairs associated commonly on three BLSOMs</p></div> </div><div class="table-overflow"><table><tbody><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG0497 ATPase involved in DNA repair </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG1196 Chromosome segregation ATPases </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG4942 Membrane-bound metallopeptidase </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG1196 Chromosome segregation ATPases </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG0439 Biotin carboxylase </td><td>COG4770 Acetyl/propionyl-CoA carboxylase, alpha subunit </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG0697 Permeases of the drug/metabolite transporter (DMT) superfamily </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG2814 Arabinose efflux permease </td></tr><tr><td>COG0515 Serine/threonine protein kinase </td><td>COG5099 RNA-binding protein of the Puf family, translational repressor </td></tr><tr><td>COG1298 Flagellar biosynthesis pathway, component FlhA </td><td>COG4789 Type III secretory pathway, component EscV </td></tr><tr><td>COG3839 ABC-type sugar transport systems, ATPase components </td><td>COG3842 ABC-type spermidine/putrescine transport systems, ATPase components </td></tr></tbody></table></div><div content-id="DSP018TB2" class="table-modal"><table><tbody><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG0497 ATPase involved in DNA repair </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG1196 Chromosome segregation ATPases </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG4942 Membrane-bound metallopeptidase </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG1196 Chromosome segregation ATPases </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG0439 Biotin carboxylase </td><td>COG4770 Acetyl/propionyl-CoA carboxylase, alpha subunit </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG0697 Permeases of the drug/metabolite transporter (DMT) superfamily </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG2814 Arabinose efflux permease </td></tr><tr><td>COG0515 Serine/threonine protein kinase </td><td>COG5099 RNA-binding protein of the Puf family, translational repressor </td></tr><tr><td>COG1298 Flagellar biosynthesis pathway, component FlhA </td><td>COG4789 Type III secretory pathway, component EscV </td></tr><tr><td>COG3839 ABC-type sugar transport systems, ATPase components </td><td>COG3842 ABC-type spermidine/putrescine transport systems, ATPase components </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6252210" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP018TB2" class="table-wrap-title" data-id="DSP018TB2"><div class="label -label">Table 2</div><div class="caption"><p>COG pairs associated commonly on three BLSOMs</p></div> </div><div class="table-overflow"><table><tbody><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG0497 ATPase involved in DNA repair </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG1196 Chromosome segregation ATPases </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG4942 Membrane-bound metallopeptidase </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG1196 Chromosome segregation ATPases </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG0439 Biotin carboxylase </td><td>COG4770 Acetyl/propionyl-CoA carboxylase, alpha subunit </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG0697 Permeases of the drug/metabolite transporter (DMT) superfamily </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG2814 Arabinose efflux permease </td></tr><tr><td>COG0515 Serine/threonine protein kinase </td><td>COG5099 RNA-binding protein of the Puf family, translational repressor </td></tr><tr><td>COG1298 Flagellar biosynthesis pathway, component FlhA </td><td>COG4789 Type III secretory pathway, component EscV </td></tr><tr><td>COG3839 ABC-type sugar transport systems, ATPase components </td><td>COG3842 ABC-type spermidine/putrescine transport systems, ATPase components </td></tr></tbody></table></div><div content-id="DSP018TB2" class="table-modal"><table><tbody><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG0497 ATPase involved in DNA repair </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG1196 Chromosome segregation ATPases </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG4942 Membrane-bound metallopeptidase </td></tr><tr><td>COG0419 ATPase involved in DNA repair </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG1196 Chromosome segregation ATPases </td><td>COG5022 Myosin heavy chain </td></tr><tr><td>COG0439 Biotin carboxylase </td><td>COG4770 Acetyl/propionyl-CoA carboxylase, alpha subunit </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG0697 Permeases of the drug/metabolite transporter (DMT) superfamily </td></tr><tr><td>COG0477 Permeases of the major facilitator superfamily </td><td>COG2814 Arabinose efflux permease </td></tr><tr><td>COG0515 Serine/threonine protein kinase </td><td>COG5099 RNA-binding protein of the Puf family, translational repressor </td></tr><tr><td>COG1298 Flagellar biosynthesis pathway, component FlhA </td><td>COG4789 Type III secretory pathway, component EscV </td></tr><tr><td>COG3839 ABC-type sugar transport systems, ATPase components </td><td>COG3842 ABC-type spermidine/putrescine transport systems, ATPase components </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6252210" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6252211 id="6252211" class="section-title" data-legacy-id=s3c>3.3 Mapping of function-known environmental proteins on BLSOMs constructed with NcCOG sequences</h3>
<p>Environmental microorganisms cannot be cultured easily under laboratory conditions, and therefore, genomes of unculturable microorganisms have remained mostly uncharacterized but are thought to contain a wide range of novel protein genes of scientific and industrial usefulness. Metagenomic approaches, which determine the sequence of mixed genomes of uncultured environmental microbes, have been recently developed to identify wide varieties of novel and industrially useful genes.<sup><a reveal-id="DSP018C25" data-open="DSP018C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSP018C26" data-open="DSP018C26" class="link link-ref link-reveal xref-bibr">26</a></sup> One important contribution of the present alignment-free clustering method is to systematically and efficiently predict functions of the increasingly vast quantity of function-unknown proteins derived from poorly characterized microbe genomes. To examine the feasibility of BLSOMs for function prediction for such environmental samples and compare the BLSOM results with those obtained by the sequence homology search, we next used, as a test data set, protein candidates that were obtained from genomic fragments derived from metagenome libraries originating in the Sargasso Sea<sup><a reveal-id="DSP018C27" data-open="DSP018C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and examined the similarities and differences in predictions obtained by BLAST and BLSOM. We chose the Sargasso Sea sequences because the sequences covered a wide variety of genes from a wide variety of phylotypes, and in particular we focused on the 97 838 protein candidates that have been registered as ORFs in the International DNA Databanks without function annotation and are longer than 200 amino acids. From these proteins, we initially selected those that could be robustly assigned to COGs with BLAST basing on the strict criterion described by the International DNA Bank of Japan (DDBJ).<sup><a reveal-id="DSP018C28" data-open="DSP018C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>It should be mentioned that, as the test data used in Fig. <a reveal-id="DSP018F1" data-open="DSP018F1" class="link link-reveal link-table xref-fig">1</a> and Table <a reveal-id="DSP018TB1" data-open="DSP018TB1" class="link link-reveal link-table xref-fig">1</a>, we focused only on proteins belonging to function-known NcCOGs (2853 groups) to examine self-organization of proteins according to function. However, environmental sequences such as those of the Sargasso samples should have functions corresponding not only to the function-known NcCOGs, but also the function-unknown NcCOGs (2020 groups). Therefore, both types of NcCOGs (a total of 4873 groups) were included for this selection of Sargasso proteins robustly assignable to NcCOGs with BLAST.<sup><a reveal-id="DSP018C28" data-open="DSP018C28" class="link link-ref link-reveal xref-bibr">28</a></sup> A total of 4240 Sargasso proteins were assigned to the NcCOGs, and the COGs thus assigned were called HomCOGs (COGs assigned by the homology search). Then, we divided the HomCOG Sargasso proteins into 200-amino acid segments with a sliding step of 10 amino acids, obtaining a total of 77 504 segments.</p><p>As a separate analysis, we prepared the Di20-, Tri11- and Tetra6-BLSOMs with 200-amino acid segments with a sliding step of 50 amino acids, which were derived from the function-known plus-unknown NcCOGs (a total of 4873 groups: 124 292 sequences: 693 553 segments). Then, on these BLSOMs, each of the 200-amino acid segments that were derived from the HomCOG Sargasso proteins was mapped by finding the lattice point with the minimum Euclidian distance in the multidimensional space, as described in Materials and methods. For each HomCOG Sargasso protein, we summed up the mapping results of all 200-amino acid segments. If more than 60% of the mapped results gave the same NcCOG category, the Sargasso protein was assigned to this COG and this COG was designated as the SomCOG (the COG assigned with BLSOM). Because the number of COG categories used here was 4873, a probability of random chance that the multiple segments derived from one Sargasso protein were classified into the lattice points tentatively assigned to the same NcCOG should be extremely low. A Venn diagram shows high levels of overlap of the COG assignments obtained by three BLSOMs (Fig. <a reveal-id="DSP018F3" data-open="DSP018F3" class="link link-reveal link-table xref-fig">3</a>A).</p>                        <a id="6252215" scrollto-destination="6252215"></a>
<div id="DSP018F3" data-id="DSP018F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01803.jpeg?Expires=2147483647&Signature=N8w3OWlutOLdLyivBgvTbaRHXvT5tCVw81ht3J9lCzp4nDN2wCrJ2nf3SAaVoWxL0TYcTQY2dFad01DBbmwGn3MNdUD9jZ0BfU8RSPriIXmpayw2EP6fUdH7n~X~X9doGgTShcUnfQkqFTAKwkXd3lXjwoxo1fWElH9hcLgcMyQR4oUHAU78dZeh8YRTtHnMRco8YIjnLcizjyrVdCHRz~BNEEscmTd9UfGX1r6JOg4vb8wJMHswfrruLnbWYH0YNeKA28ROGDUEDY5nKC29apVbN5iQaYRAWytWG4T4GJB1-Luh0PM-o7-Oj1EnlaD-JDulzg1jNFnhT5B9zjSGHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagrams representing COG predictions obtained by three BLSOMs. (A) Sargasso HomCOG proteins. The number and percentage in a parenthesis show the number of Sargasso proteins properly assigned to COG with each BLSOM and its percentage, respectively. (B) Sargasso proteins unassigned to COG with BLAST. The number in a parenthesis shows the number of Sargasso proteins assigned to COG with each BLSOM." path-from-xml="dsp01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252215/dsp01803.jpeg" path-from-xml="dsp01803.jpeg" target="_blank">View large</a><a section="6252215" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01803.jpeg?Expires=2147483647&Signature=BX1nSVEoApjcDV3qxRXHkW92EH-75dhX~zSUpQ62FhorpkfeaHPSJfpjluwXMqwTVbW4PvXz7OPUFFYAwQ21lTp3bVSfN7fPyExEdbgedtidzSdC9fV7w8Ruis4Lv-Rt8A1dk46ok6h-uUgSGJuT4025swsAo8ecv4oNbNCsrvrQblNZ8ORNM7E0JD4JPRXCOhFFlQS1oBvefR-tmmpma8hAlC-~FIBI7YFg28pUAoxEKsMqanuMMntGD-qzmfKHXeM0~yTDEjd8q7Xn9yp7FuIxI0V~udzj7QN3BJYhxbNAxjE8lujaV8gO8V7gLa1JKJzOcbBmvwAdYfAVA84Xhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252215&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagrams representing COG predictions obtained by three BLSOMs. (A) Sargasso HomCOG proteins. The number and percentage in a parenthesis show the number of Sargasso proteins properly assigned to COG with each BLSOM and its percentage, respectively. (B) Sargasso proteins unassigned to COG with BLAST. The number in a parenthesis shows the number of Sargasso proteins assigned to COG with each BLSOM.</p></div></div><div content-id="DSP018F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01803.jpeg?Expires=2147483647&Signature=N8w3OWlutOLdLyivBgvTbaRHXvT5tCVw81ht3J9lCzp4nDN2wCrJ2nf3SAaVoWxL0TYcTQY2dFad01DBbmwGn3MNdUD9jZ0BfU8RSPriIXmpayw2EP6fUdH7n~X~X9doGgTShcUnfQkqFTAKwkXd3lXjwoxo1fWElH9hcLgcMyQR4oUHAU78dZeh8YRTtHnMRco8YIjnLcizjyrVdCHRz~BNEEscmTd9UfGX1r6JOg4vb8wJMHswfrruLnbWYH0YNeKA28ROGDUEDY5nKC29apVbN5iQaYRAWytWG4T4GJB1-Luh0PM-o7-Oj1EnlaD-JDulzg1jNFnhT5B9zjSGHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagrams representing COG predictions obtained by three BLSOMs. (A) Sargasso HomCOG proteins. The number and percentage in a parenthesis show the number of Sargasso proteins properly assigned to COG with each BLSOM and its percentage, respectively. (B) Sargasso proteins unassigned to COG with BLAST. The number in a parenthesis shows the number of Sargasso proteins assigned to COG with each BLSOM." path-from-xml="dsp01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252215/dsp01803.jpeg" path-from-xml="dsp01803.jpeg" target="_blank">View large</a><a section="6252215" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01803.jpeg?Expires=2147483647&Signature=BX1nSVEoApjcDV3qxRXHkW92EH-75dhX~zSUpQ62FhorpkfeaHPSJfpjluwXMqwTVbW4PvXz7OPUFFYAwQ21lTp3bVSfN7fPyExEdbgedtidzSdC9fV7w8Ruis4Lv-Rt8A1dk46ok6h-uUgSGJuT4025swsAo8ecv4oNbNCsrvrQblNZ8ORNM7E0JD4JPRXCOhFFlQS1oBvefR-tmmpma8hAlC-~FIBI7YFg28pUAoxEKsMqanuMMntGD-qzmfKHXeM0~yTDEjd8q7Xn9yp7FuIxI0V~udzj7QN3BJYhxbNAxjE8lujaV8gO8V7gLa1JKJzOcbBmvwAdYfAVA84Xhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252215&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagrams representing COG predictions obtained by three BLSOMs. (A) Sargasso HomCOG proteins. The number and percentage in a parenthesis show the number of Sargasso proteins properly assigned to COG with each BLSOM and its percentage, respectively. (B) Sargasso proteins unassigned to COG with BLAST. The number in a parenthesis shows the number of Sargasso proteins assigned to COG with each BLSOM.</p></div></div><p>We next compared the results of the COG assignment obtained by BLSOM (SomCOG) with those of the sequence homology search (HomCOG). Approximately 88%, 85% and 79% of the 4240 Sargasso HomCOG proteins were classified into COG categories same to those of BLAST on Tri11-, Di20- and Tetra6-BLSOMs, respectively. The highest level of congruity was found for Tri11-BLSOM. The reason why the 100% congruity and the complete overlap were not found (Fig. <a reveal-id="DSP018F3" data-open="DSP018F3" class="link link-reveal link-table xref-fig">3</a>A) is thought to be related to the aforementioned finding that proteins belonging to different COGs, but having closely related functions, were associated with each other on BLSOMs. Levels of false-positives (i.e. assignment to unrelated COGs) were 6.3%, 6.4% and 7.1% on Tri11-, Di20- and Tetra6-BLSOMs, respectively, and the level was less than 0.2% for the overlapped zones of the two BLSOMs.</p>                            <h3 scrollto-destination=6252217 id="6252217" class="section-title" data-legacy-id=s3d>3.4 Comparison of function predictions by BLSOM and BLAST</h3>
<p>In the above analysis, we focused on the Sargasso proteins assigned robustly to NcCOGs by BLAST (HomCOGs) in order to search for conditions giving high prediction accuracy. The most important contribution of the present alignment-free method should be to predict functions that cannot be predicted or can only be hypothesized ambiguously by the sequence homology search. To examine this potentiality, we used, as a test data set, the residual 93 598 Sargasso proteins that have been registered as ORFs in the International DNA Databanks but could not be robustly assigned to NcCOGs with BLAST. We mapped 200-amino acid segments derived from all 93 598 Sargasso proteins on the Di20-, Tri11- and Tetra6-BLSOMs constructed with NcCOG proteins, as described earlier. The overlap levels of the COG assignments for the residual 93 598 Sargasso proteins (Fig. <a reveal-id="DSP018F3" data-open="DSP018F3" class="link link-reveal link-table xref-fig">3</a>B) were lower than those found for the 4240 Sargasso HomCOG proteins (Fig. <a reveal-id="DSP018F3" data-open="DSP018F3" class="link link-reveal link-table xref-fig">3</a>A). This appears to fit the view that the residual Sargasso proteins presumably contain distant relatives of NcCOG proteins (e.g. paralogous genes) and are difficult to be assigned to a single NcCOG.</p><p>When the same prediction was obtained by different BLSOMs in Fig. <a reveal-id="DSP018F3" data-open="DSP018F3" class="link link-reveal link-table xref-fig">3</a>B, the prediction would be meaningful, rather than accidental. This is because the three BLSOMs differed not only in the length of peptides (2, 3 and 4), but also in the categorization of amino acid residues (20, 11 and 6). A common SomCOG was assigned at least with two BLSOMs for 20 476 Sargasso proteins and was assigned with all three BLSOMs for 11 652 Sargasso proteins. To examine whether such predictions actually have biological meaning, BLAST search was conducted against NcCOG proteins, using each of the Sargasso COG proteins commonly assigned with three BLSOMs as a query. The identity and coverage levels found for the NcCOG protein with the lowest <em>e</em> value for each Sargasso query are plotted in Fig. <a reveal-id="DSP018F4" data-open="DSP018F4" class="link link-reveal link-table xref-fig">4</a>A. Although a major portion of the data had more than 50% level of identity and more than 80% level of coverage, seven proteins had less than 30% identity, for which BLAST may not give convincing assignments. Next, we focused on Sargasso proteins assigned at least with two BLSOMs and conducted the above BLAST search. Data points with low identity and coverage levels increased significantly (Fig. <a reveal-id="DSP018F4" data-open="DSP018F4" class="link link-reveal link-table xref-fig">4</a>B); for example, there were 66 proteins with less than 30% identity. In the final analysis, we focused on 63 752 Sargasso proteins that were assigned with any of the three BLSOMs and conducted the same BLAST search. The number of proteins with low levels of identity and coverage increased significantly, and there were 3097 proteins with less than 30% identity.</p>                        <a id="6252220" scrollto-destination="6252220"></a>
<div id="DSP018F4" data-id="DSP018F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01804.jpeg?Expires=2147483647&Signature=kLmYrCDMncrMbEgqWzQ-pd1ejqdtSMVtWSyQeSSt8POsBlNo~aPYT3Hibxq6p~aAwfkzRWNbWMjaS4~t4x540cKovevbtoROoYA~FXb-uh6LEZ3m7~7YheNyqUDQT7rBGqRUifJNgdqVey~9qxOZ9lT33PbFSvy7MFuh6b6s7gDT65nXwHS~sK5hUKjEI4lumbcIcAeH9VDr8aE-MnmGNORRtbKsb6mnd5PJ0y19qQXXa3IT486xCBwddojC8CQteBB1RXRRN6bEtz0Nz1b1LK7I5wvAmgjDAM372btcITozSg888hKfkesyMHf9i2g6eobHAZ4938HPV1TEx0QIdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identity and coverage levels found for the NcCOG protein with the lowest e value for each of Sargasso queries, which included HomCOG proteins. Sargasso proteins commonly assigned (A) with three BLSOMs, (B) with at least two BLSOMs and (C) with any BLSOM, respectively." path-from-xml="dsp01804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252220/dsp01804.jpeg" path-from-xml="dsp01804.jpeg" target="_blank">View large</a><a section="6252220" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01804.jpeg?Expires=2147483647&Signature=gpW~DUAUHz3ZhMtnVduA4XOrs3GiiaNuRsfxkOZ0tCpO1gBNUoBQaI9YJjjqLzvpuTfJ4VovNCSF0etC3WtKM4HvVdgloamVmhCGSmnlD51UDpyYnf5nGty-GU9HKKl~85vvZW8aAilaqIr-DE5XvaLYIX16D9Jpi-DEaCYmA67Ht~tQc8xC8LDf2vFgRPOqOPwVfox8qG76oZOyG79A~Wjuv37IDMmeLCEjegeNN3rKwoIYe54ZxFWJxuPuLOfxdJGw3LfMuRZmQfD~nQT1s34-UzBB8TAhSgrfpH~YK3c2WPzVPFqRiz2hAO1n2ngeumgpYsqSKLkiTwl1uh8fSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252220&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identity and coverage levels found for the NcCOG protein with the lowest <em>e</em> value for each of Sargasso queries, which included HomCOG proteins. Sargasso proteins commonly assigned (A) with three BLSOMs, (B) with at least two BLSOMs and (C) with any BLSOM, respectively.</p></div></div><div content-id="DSP018F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/m_dsp01804.jpeg?Expires=2147483647&Signature=kLmYrCDMncrMbEgqWzQ-pd1ejqdtSMVtWSyQeSSt8POsBlNo~aPYT3Hibxq6p~aAwfkzRWNbWMjaS4~t4x540cKovevbtoROoYA~FXb-uh6LEZ3m7~7YheNyqUDQT7rBGqRUifJNgdqVey~9qxOZ9lT33PbFSvy7MFuh6b6s7gDT65nXwHS~sK5hUKjEI4lumbcIcAeH9VDr8aE-MnmGNORRtbKsb6mnd5PJ0y19qQXXa3IT486xCBwddojC8CQteBB1RXRRN6bEtz0Nz1b1LK7I5wvAmgjDAM372btcITozSg888hKfkesyMHf9i2g6eobHAZ4938HPV1TEx0QIdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identity and coverage levels found for the NcCOG protein with the lowest e value for each of Sargasso queries, which included HomCOG proteins. Sargasso proteins commonly assigned (A) with three BLSOMs, (B) with at least two BLSOMs and (C) with any BLSOM, respectively." path-from-xml="dsp01804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6252220/dsp01804.jpeg" path-from-xml="dsp01804.jpeg" target="_blank">View large</a><a section="6252220" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp018/2/dsp01804.jpeg?Expires=2147483647&Signature=gpW~DUAUHz3ZhMtnVduA4XOrs3GiiaNuRsfxkOZ0tCpO1gBNUoBQaI9YJjjqLzvpuTfJ4VovNCSF0etC3WtKM4HvVdgloamVmhCGSmnlD51UDpyYnf5nGty-GU9HKKl~85vvZW8aAilaqIr-DE5XvaLYIX16D9Jpi-DEaCYmA67Ht~tQc8xC8LDf2vFgRPOqOPwVfox8qG76oZOyG79A~Wjuv37IDMmeLCEjegeNN3rKwoIYe54ZxFWJxuPuLOfxdJGw3LfMuRZmQfD~nQT1s34-UzBB8TAhSgrfpH~YK3c2WPzVPFqRiz2hAO1n2ngeumgpYsqSKLkiTwl1uh8fSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6252220&ar=421945&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identity and coverage levels found for the NcCOG protein with the lowest <em>e</em> value for each of Sargasso queries, which included HomCOG proteins. Sargasso proteins commonly assigned (A) with three BLSOMs, (B) with at least two BLSOMs and (C) with any BLSOM, respectively.</p></div></div>                            <h3 scrollto-destination=6252221 id="6252221" class="section-title" data-legacy-id=s3e>3.5 Integrative assessment of function predictions by BLAST and BLSOM</h3>
<p>The sequence homology searches, such as BLAST and PSI-BLAST,<sup><a reveal-id="DSP018C6" data-open="DSP018C6" class="link link-ref link-reveal xref-bibr">6</a></sup> undoubtedly are essential tools for predicting protein function, but there has been accumulated a vast quantity of function-unknown proteins. One important contribution of BLSOM should be to provide an independent, separate prediction judgment, permitting us to get integrative assessment. For example, in the cases of Sargasso proteins with less than 60% levels of identity and coverage in the BLAST search, multiple COGs often became the candidates, and it was usually difficult to tell which of the candidates was the proper answer. When the BLSOM prediction fit to one of the candidates obtained by BLAST, this will become more reliable because the two methods are based on clearly distinct principles; sequence alignment-free and -dependent methods. In the cases of proteins with less than a 30% identity level (Fig. <a reveal-id="DSP018F4" data-open="DSP018F4" class="link link-reveal link-table xref-fig">4</a>), convincing prediction will not be obtained with BLAST. For such twilight zone of alignments, various useful strategies have been developed to get reliable alignments.<sup><a reveal-id="DSP018C1  DSP018C8" data-open="DSP018C1  DSP018C8" class="link link-ref link-reveal xref-bibr">1–8</a></sup> BLSOM is a sequence alignment- and structural information-free method, and therefore, BLSOM prediction will become the novel, irreplaceable information.</p><p>The proof of functions, especially for novel proteins, can only be obtained by experimental approaches, and the purpose of any informatics prediction for such novel proteins is to provide the supporting-evidence/guideline for experimental studies. BLSOM will become a novel, irreplaceable strategy for function prediction of proteins for which only ambiguous or no prediction was obtained with sequence homology searches.</p>                            <h2 scrollto-destination=6252224 id="6252224" class="section-title" data-legacy-id=s4>4 Discussion</h2>
<p>Oligopeptides are component parts of a protein and involved in formation of both functional motifs and 3D structures. The BLSOM analyzing oligopeptide composition, therefore, appears to be positioned near with the functional motif searches, such as PRINTS and PROSITE,<sup><a reveal-id="DSP018C29" data-open="DSP018C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSP018C30" data-open="DSP018C30" class="link link-ref link-reveal xref-bibr">30</a></sup> which are powerful, indispensable methods for the function prediction of proteins. The functional motif search, however, does not adequately incorporate information pertaining to 3D structure formation. In the case of BLSOM, not only the oligopeptides for functional motifs, but also those contributing to 3D structure formation were included for predicting functions. Furthermore, because the functional motifs are obtained mostly from experimentally well-characterized proteins, the method might become less useful in the function prediction for the less-characterized proteins originating from poorly characterized genomes. The present unsupervised clustering algorism has the advantage that no advance knowledge about the target protein is required, and therefore, is thought to be appropriate for analyses of diverse novel proteins.</p><p>Sequence similarity and functional motif searches undoubtedly are essential, indispensable tools for predicting protein functions, but even after combining these powerful methods, a large number of function-unknown proteins remains. In order to complement these conventional methods and provide the additional new information helpful for integrative assessments, we developed the BLSOM method. The most important contribution of the present unsupervised, alignment-free method should be to predict functions of increasingly vast quantity of function-unknown proteins derived from less characterized genomes, such as those studied in the metagenomic approaches. For identifying functions of novel function-unknown proteins, a large-scale BLSOM that has analyzed all function-known proteins in databases must be constructed to extract their characteristics in advance. Various high-performance supercomputers are now available for biological studies, and the BLSOM developed here is suitable for actualizing high-performance parallel-computing with high-performance supercomputers such as the Earth Simulator ‘ES’.<sup><a reveal-id="DSP018C14  DSP018C16" data-open="DSP018C14  DSP018C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> The present unsupervised, self-classifying strategy to find association of function-unknown proteins to function-known proteins on a large-scale BLSOM constructed with high-performance supercomputers should serve as a new and powerful tool in the post-genome era. In this study, we did not compare the performance of our approach with those of the structure-based methods, which would be one of our future works.</p><p>It is noteworthy that once the large-scale BLSOM is constructed using a high performance supercomputer and publicized, researchers can predict the function of individual proteins (e.g. those obtained from a newly sequenced genome) by mapping their proteins of interest on the large-scale BLSOM. This mapping is possible using a PC-level computer. The programs for the mapping, as well as the large-scale BLSOMs constructed by supercomputers, were available from our group (<a href="mailto:takaabe@nagahama-i-bio.ac.jp" target="_blank">takaabe@nagahama-i-bio.ac.jp</a>). This approach should add a new and powerful tool to the genomics and proteomics fields for the systematical and efficient prediction of functions of huge quantities of function-unknown proteins progressively accumulated in the International DNA Banks.</p>                            <h2 scrollto-destination=6252231 id="6252231" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by Grant-in-Aid for Scientific Research (C) and for Young Scientists (B) from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=6252236 id="6252236" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The computation was done with the Earth Simulator of Japan Agency for Marine-Earth Science and Technology.</p>                            <h2 scrollto-destination=6252239 id="6252239" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP018C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Blundell</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Mizuguchi</div> <div class="given-names">K.</div></div>. </div><div class="article-title">FUGUE: sequence-structure homology recognition using environment-specific substitution tables and structure-dependent gap penalties</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">310</div> (pg. <div class="fpage">243</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FUGUE%3A%20sequence-structure%20homology%20recognition%20using%20environment-specific%20substitution%20tables%20and%20structure-dependent%20gap%20penalties&amp;author=J.%20Shi&amp;author=T.L.%20Blundell&amp;author=K.%20Mizuguchi&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol.&amp;volume=310&amp;pages=243-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2001.4762" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2001.4762" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2001.4762"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11419950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andreeva</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Howorth</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chandonia</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Data growth and its impact on the SCOP database: new developments</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D419</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Data%20growth%20and%20its%20impact%20on%20the%20SCOP%20database%3A%20new%20developments&amp;author=A.%20Andreeva&amp;author=D.%20Howorth&amp;author=J.%20Chandonia&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D419-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm993" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm993" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm993"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18000004" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cuff</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Sillitoe</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lewis</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">The CATH classification revisited—architectures reviewed and new ways to characterize structural divergence in superfamilies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D310</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20CATH%20classification%20revisited%E2%80%94architectures%20reviewed%20and%20new%20ways%20to%20characterize%20structural%20divergence%20in%20superfamilies&amp;author=A.L.%20Cuff&amp;author=I.%20Sillitoe&amp;author=T.%20Lewis&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D310-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18996897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Todd</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Orengo</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Evolution of function in protein superfamilies, from a structural perspective</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">307</div> (pg. <div class="fpage">1113</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20function%20in%20protein%20superfamilies%2C%20from%20a%20structural%20perspective&amp;author=A.E.%20Todd&amp;author=C.A.%20Orengo&amp;author=J.M.%20Thornton&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol.&amp;volume=307&amp;pages=1113-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2001.4513" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2001.4513" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2001.4513"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11286560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Redfern</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Orengo</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Predicting protein function from sequence and structure</div>, <div class="source">Nat. Rev. Mol. Cell Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">995</div>-<div class="lpage">1005</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20protein%20function%20from%20sequence%20and%20structure&amp;author=D.%20Lee&amp;author=O.%20Redfern&amp;author=C.%20Orengo&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;volume=8&amp;pages=995-1005" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm2281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm2281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm2281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18037900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.L.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rost</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Twilight zone of protein sequence alignments</div>, <div class="source">Protein Eng.</div> , <div class="year">1999</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">85</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Twilight%20zone%20of%20protein%20sequence%20alignments&amp;author=B.%20Rost&amp;publication_year=1999&amp;journal=Protein%20Eng.&amp;volume=12&amp;pages=85-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/protein/12.2.85" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fprotein%2F12.2.85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fprotein%2F12.2.85"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10195279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">G.S.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ko</div> <div class="given-names">K.D.</div></div>, et al. </div><div class="article-title">Phylogenetic profiles reveal evolutionary relationships within the “twilight zone” of sequence similarity</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">1374</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20profiles%20reveal%20evolutionary%20relationships%20within%20the%20%E2%80%9Ctwilight%20zone%E2%80%9D%20of%20sequence%20similarity&amp;author=G.S.%20Chang&amp;author=Y.%20Hong&amp;author=K.D.%20Ko&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=1374-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0710986105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0710986105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0710986105"> </span></div> </div></div></div></div></div><div content-id="DSP018C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The self-organizing map</div>, <div class="source">Proc. IEEE</div> , <div class="year">1990</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">1464</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20self-organizing%20map&amp;author=T.%20Kohonen&amp;publication_year=1990&amp;journal=Proc.%20IEEE&amp;volume=78&amp;pages=1464-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/5.58325" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2F5.58325" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2F5.58325"> </span></div> </div></div></div></div></div><div content-id="DSP018C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Oja</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Simula</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Visa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kangas</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Engineering applications of the self-organizing map</div>, <div class="source">Proc. IEEE</div> , <div class="year">1996</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">1358</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20applications%20of%20the%20self-organizing%20map&amp;author=T.%20Kohonen&amp;author=E.%20Oja&amp;author=O.%20Simula&amp;author=A.%20Visa&amp;author=J.%20Kangas&amp;publication_year=1996&amp;journal=Proc.%20IEEE&amp;volume=84&amp;pages=1358-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/5.537105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2F5.537105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2F5.537105"> </span></div> </div></div></div></div></div><div content-id="DSP018C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferran</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Pflugfelder</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ferrara</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Self-organized neural maps of human protein sequences</div>, <div class="source">Protein Sci.</div> , <div class="year">1994</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">507</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Self-organized%20neural%20maps%20of%20human%20protein%20sequences&amp;author=E.A.%20Ferran&amp;author=B.%20Pflugfelder&amp;author=P.%20Ferrara&amp;publication_year=1994&amp;journal=Protein%20Sci.&amp;volume=3&amp;pages=507-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/pro.v3:3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fpro.v3:3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fpro.v3:3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8019421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the E. coli O157 genome</div>, <div class="source">Gene</div> , <div class="year">2001</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">89</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20codon%20usage%20diversity%20of%20bacterial%20genes%20with%20a%20self-organizing%20map%20(SOM)%3A%20characterization%20of%20horizontally%20transferred%20genes%20with%20emphasis%20on%20the%20E.%20coli%20O157%20genome&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;author=T.%20Abe&amp;publication_year=2001&amp;journal=Gene&amp;volume=276&amp;pages=89-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(01)00673-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(01)00673-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(01)00673-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ichiba</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kozuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Informatics for unveiling hidden genome signatures</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">693</div>-<div class="lpage">702</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Informatics%20for%20unveiling%20hidden%20genome%20signatures&amp;author=T.%20Abe&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;author=Y.%20Ichiba&amp;author=T.%20Kozuki&amp;author=T.%20Ikemura&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=693-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.634603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.634603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.634603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Novel phylogenetic studies of genomic sequence fragments derived from uncultured microbe mixtures in environmental and clinical samples</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">281</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20phylogenetic%20studies%20of%20genomic%20sequence%20fragments%20derived%20from%20uncultured%20microbe%20mixtures%20in%20environmental%20and%20clinical%20samples&amp;author=T.%20Abe&amp;author=H.%20Sugawara&amp;author=M.%20Kinouchi&amp;author=S.%20Kanaya&amp;author=T.%20Ikemura&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=281-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A large-scale self-organizing map (SOM) unveils sequence characteristics of a wide range of eukaryote genomes</div>, <div class="source">Gene</div> , <div class="year">2006</div>, vol. <div class="volume">365</div> (pg. <div class="fpage">27</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large-scale%20self-organizing%20map%20(SOM)%20unveils%20sequence%20characteristics%20of%20a%20wide%20range%20of%20eukaryote%20genomes&amp;author=T.%20Abe&amp;author=H.%20Sugawara&amp;author=M.%20Kinouchi&amp;author=S.%20Kanaya&amp;author=T.%20Ikemura&amp;publication_year=2006&amp;journal=Gene&amp;volume=365&amp;pages=27-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.09.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.09.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.09.040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16364569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Sequences from almost all prokaryotic, eukaryotic, and viral genomes available could be classified according to genomes on a large-scale self-organizing map constructed with the earth simulator</div>, <div class="source">J. Earth Simulator</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">17</div>-<div class="lpage">23</div>)</div></div></div></div><div content-id="DSP018C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Substrate-induced gene-expression screening of environmental metagenome libraries for isolation of catabolic genes</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">88</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substrate-induced%20gene-expression%20screening%20of%20environmental%20metagenome%20libraries%20for%20isolation%20of%20catabolic%20genes&amp;author=T.%20Uchiyama&amp;author=T.%20Abe&amp;author=T.%20Ikemura&amp;author=K.%20Watanabe&amp;publication_year=2005&amp;journal=Nat.%20Biotechnol.&amp;volume=23&amp;pages=88-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakamoto</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Direct cloning of genes encoding novel xylanases from human gut</div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">251</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20cloning%20of%20genes%20encoding%20novel%20xylanases%20from%20human%20gut&amp;author=H.%20Hayashi&amp;author=T.%20Abe&amp;author=M.%20Sakamoto&amp;publication_year=2005&amp;journal=Can.%20J.%20Microbiol.&amp;volume=51&amp;pages=251-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/w04-136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fw04-136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fw04-136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15920623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shimoyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ishii</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">K.</div></div>. </div><div class="article-title">The genome of Pelotomaculum thermopropionicum reveals niche-associated evolution in anaerobic microbiota</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">442</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Pelotomaculum%20thermopropionicum%20reveals%20niche-associated%20evolution%20in%20anaerobic%20microbiota&amp;author=T.%20Kosaka&amp;author=S.%20Kato&amp;author=T.%20Shimoyama&amp;author=S.%20Ishii&amp;author=T.%20Abe&amp;author=K.%20Watanabe&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=442-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.7136508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.7136508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.7136508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A novel bioinformatics tool for phylogenetic classification of genomic sequence fragments derived from mixed genomes of environmental uncultured microbes</div>, <div class="source">Polar Biosci.</div> , <div class="year">2006</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">103</div>-<div class="lpage">12</div>)</div></div></div></div><div content-id="DSP018C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatsusov</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">A genomic perspective on protein families</div>, <div class="source">Science</div> , <div class="year">1997</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">631</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20perspective%20on%20protein%20families&amp;author=R.L.%20Tatsusov&amp;author=E.V.%20Koonin&amp;author=D.J.%20Lipman&amp;publication_year=1997&amp;journal=Science&amp;volume=278&amp;pages=631-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.278.5338.631" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.278.5338.631" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.278.5338.631"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9381173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lespinet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>,  <div class="name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The role of lineage-specific gene family expansion in the evolution of eukaryotes</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1048</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20lineage-specific%20gene%20family%20expansion%20in%20the%20evolution%20of%20eukaryotes&amp;author=O.%20Lespinet&amp;author=Y.I.%20Wolf&amp;author=E.V.%20Koonin&amp;author=L.%20Aravind&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1048-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.174302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.174302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.174302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12097341" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Overbeek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Begley</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">R.M.</div></div>, et al. </div><div class="article-title">The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">5691</div>-<div class="lpage">702</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20subsystems%20approach%20to%20genome%20annotation%20and%20its%20use%20in%20the%20project%20to%20annotate%201000%20genomes&amp;author=R.%20Overbeek&amp;author=T.%20Begley&amp;author=R.M.%20Butler&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=5691-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki866" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki866" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki866"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16214803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>,  <div class="name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Genomics of bacteria and Archaea: the emerging dynamic view of the prokaryotic world</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">6688</div>-<div class="lpage">719</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20bacteria%20and%20Archaea%3A%20the%20emerging%20dynamic%20view%20of%20the%20prokaryotic%20world&amp;author=E.V.%20Koonin&amp;author=L.%20Aravind&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=6688-719" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn668" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn668" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn668"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DeLong</div> <div class="given-names">E.F.</div></div>. </div><div class="article-title">Microbial population genomics and ecology</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2002</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">520</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20population%20genomics%20and%20ecology&amp;author=E.F.%20DeLong&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=5&amp;pages=520-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(02)00353-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(02)00353-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(02)00353-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12354561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schloss</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Handelsman</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Biotechnological prospects from metagenomics</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">303</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biotechnological%20prospects%20from%20metagenomics&amp;author=P.D.%20Schloss&amp;author=J.%20Handelsman&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=14&amp;pages=303-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(03)00067-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(03)00067-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(03)00067-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12849784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venter</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Remington</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J.F.</div></div>, et al. </div><div class="article-title">Environmental genome shotgun sequencing of the Sargasso Sea</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">304</div> (pg. <div class="fpage">66</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20genome%20shotgun%20sequencing%20of%20the%20Sargasso%20Sea&amp;author=J.C.%20Venter&amp;author=K.%20Remington&amp;author=J.F.%20Heidelberg&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=66-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1093857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1093857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1093857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15001713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosuge</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okido</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Exploration and grading of possible genes in 183 bacterial strains by a common fine protocol lead to new genes: Gene Trek in Prokaryote Space (GTPS)</div>, <div class="source">DNA Res.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">245</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploration%20and%20grading%20of%20possible%20genes%20in%20183%20bacterial%20strains%20by%20a%20common%20fine%20protocol%20lead%20to%20new%20genes%3A%20Gene%20Trek%20in%20Prokaryote%20Space%20(GTPS)&amp;author=T.%20Kosuge&amp;author=T.%20Abe&amp;author=T.%20Okido&amp;publication_year=2006&amp;journal=DNA%20Res.&amp;volume=13&amp;pages=245-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsl014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsl014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsl014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17166861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Attwood</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The PRINTS database: a resource for identification of protein families</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">252</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20PRINTS%20database%3A%20a%20resource%20for%20identification%20of%20protein%20families&amp;author=T.%20Attwood&amp;publication_year=2002&amp;journal=Brief%20Bioinform.&amp;volume=3&amp;pages=252-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/3.3.252" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2F3.3.252" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2F3.3.252"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12230034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP018C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP018C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hulo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bulliard</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">The 20 years of PROSITE</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D245</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%2020%20years%20of%20PROSITE&amp;author=N.%20Hulo&amp;author=A.%20Bairoch&amp;author=V.%20Bulliard&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D245-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm977" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm977" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm977"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18003654" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=421945&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">79</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp018"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="421945"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f5%2f287%2f421945">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000271610400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F5%2F287%2F421945" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/421945'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Novel Bioinformatics Strategy for Function Prediction of Poorly-Characterized Protein Genes Obtained from Metagenome Analyses" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp018");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f287%2f421945">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

