

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/1/37/503258","@type":"ScholarlyArticle","name":"A Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice","datePublished":"2007-04-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm005","keywords":["polymorphism","microsatellite or MS","database","whole-genome sequences","rice"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/1/37/503258","author":[{"name":"Zhang, Zhonghua","affiliation":"1  James D. Watson Institute of Genome Science ,  Zhejiang University ,  Hangzhou 310008 ,  China    5  Institute of Vegetables and Flowers ,  Chinese Academy of Agricultural Sciences ,  Beijing 100081 ,  China","@type":"Person"},{"name":"Deng, Yajun","affiliation":"2  Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics ,  Chinese Academy of Sciences ,  Beijing 101300 ,  China","@type":"Person"},{"name":"Tan, Jun","affiliation":"4  Institute of Bioinformation ,  Chongqing University of Posts and Telecommunications ,  Chongqing 400065 ,  China","@type":"Person"},{"name":"Hu, Songnian","affiliation":"1  James D. Watson Institute of Genome Science ,  Zhejiang University ,  Hangzhou 310008 ,  China    2  Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics ,  Chinese Academy of Sciences ,  Beijing 101300 ,  China","@type":"Person"},{"name":"Yu, Jun","affiliation":"1  James D. Watson Institute of Genome Science ,  Zhejiang University ,  Hangzhou 310008 ,  China    2  Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics ,  Chinese Academy of Sciences ,  Beijing 101300 ,  China    3  Institute of Computing Technology ,  Chinese Academy of Sciences ,  Beijing 101300 ,  China","@type":"Person"},{"name":"Xue, Qingzhong","affiliation":"1  James D. Watson Institute of Genome Science ,  Zhejiang University ,  Hangzhou 310008 ,  China    6  Department of Agronomy ,  College of Agriculture and Biotechnology, Zhejiang University ,  Hangzhou 310029 ,  China","@type":"Person"}],"description":"Abstract.  Microsatellite (MS) polymorphism is an important source of genetic diversity, providing support for map-based cloning and molecular breeding. We have","pageStart":"37","pageEnd":"45","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/1/0/m_dnaresearch14_1.cover.gif?Expires=2147483647&Signature=FMRJ1vzWw3n6RteSNegoL3XeMjVWMPUkUw1DAy9B1s7Po~XXJmQxiq7D2osJ~-qo14VmnnqqwET2h-DyJlv~6ABtVT~LvncFBeJjZ1wxx~y6M3vE3Qs5ixEluS1GT9PiuYZ8KtSanZd2ABbw5KLKYLu92qJh~vhgbf1vjy6lnHZMmx0r7uOO0pacC-GB~PSHCXdcgMyTCPmkGvO2wO9EFShBH0VO0-A5aQVXeyhe0MKcDIjFoBlFl1AmYllwMnIhSIvKtmeMe1oBwYyjgpTM9K4V2BjTo7iH4RO-DPaTutzNC88PpWqK0TxkAqEw7mIyX1rh07TGg~hEL4HMNF39xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/1/0/m_dnaresearch14_1.cover.gif?Expires=2147483647&Signature=FMRJ1vzWw3n6RteSNegoL3XeMjVWMPUkUw1DAy9B1s7Po~XXJmQxiq7D2osJ~-qo14VmnnqqwET2h-DyJlv~6ABtVT~LvncFBeJjZ1wxx~y6M3vE3Qs5ixEluS1GT9PiuYZ8KtSanZd2ABbw5KLKYLu92qJh~vhgbf1vjy6lnHZMmx0r7uOO0pacC-GB~PSHCXdcgMyTCPmkGvO2wO9EFShBH0VO0-A5aQVXeyhe0MKcDIjFoBlFl1AmYllwMnIhSIvKtmeMe1oBwYyjgpTM9K4V2BjTo7iH4RO-DPaTutzNC88PpWqK0TxkAqEw7mIyX1rh07TGg~hEL4HMNF39xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice" />
    <meta property="og:description" content="Abstract.  Microsatellite (MS) polymorphism is an important source of genetic diversity, providing support for map-based cloning and molecular breeding. We have" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/1/37/503258" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-04-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhang, Zhonghua" /><meta name="citation_author_institution" content="James D. Watson Institute of Genome Science, Zhejiang University, Hangzhou 310008, China" /><meta name="citation_author_institution" content="Institute of Vegetables and Flowers, Chinese Academy of Agricultural Sciences, Beijing 100081, China" /><meta name="citation_author" content="Deng, Yajun" /><meta name="citation_author_institution" content="Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 101300, China" /><meta name="citation_author" content="Tan, Jun" /><meta name="citation_author_institution" content="Institute of Bioinformation, Chongqing University of Posts and Telecommunications, Chongqing 400065, China" /><meta name="citation_author" content="Hu, Songnian" /><meta name="citation_author_institution" content="James D. Watson Institute of Genome Science, Zhejiang University, Hangzhou 310008, China" /><meta name="citation_author_institution" content="Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 101300, China" /><meta name="citation_author" content="Yu, Jun" /><meta name="citation_author_institution" content="James D. Watson Institute of Genome Science, Zhejiang University, Hangzhou 310008, China" /><meta name="citation_author_institution" content="Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 101300, China" /><meta name="citation_author_institution" content="Institute of Computing Technology, Chinese Academy of Sciences, Beijing 101300, China" /><meta name="citation_author" content="Xue, Qingzhong" /><meta name="citation_author_institution" content="James D. Watson Institute of Genome Science, Zhejiang University, Hangzhou 310008, China" /><meta name="citation_author_institution" content="Department of Agronomy, College of Agriculture and Biotechnology, Zhejiang University, Hangzhou 310029, China" /><meta name="citation_title" content="A Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice" /><meta name="citation_firstpage" content="37" /><meta name="citation_lastpage" content="45" /><meta name="citation_doi" content="10.1093/dnares/dsm005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Simple sequence repeats in Escherichia coli: abundance, distribution, composition, and polymorphism; Genome Res;  citation_year=2000;  citation_volume=10;  citation_pages=62-71; " /><meta name="citation_reference" content="citation_title=Generation and analysis of 25 Mb of genomic DNA from the pufferfish Fugu rubripes by sequence scanning; Genome Res;  citation_year=1999;  citation_volume=9;  citation_pages=960-971; " /><meta name="citation_reference" content="citation_title=Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes; Nat. Genet;  citation_year=2002;  citation_volume=30;  citation_pages=194-200; " /><meta name="citation_reference" content="citation_title=Diversity of microsatellites derived from genomic libraries and GenBank sequences in rice (Oryza sativa L.); Theor. Appl. Genet;  citation_year=2000;  citation_volume=100;  citation_pages=713-722; " /><meta name="citation_reference" content="citation_title=Identification of trait-improving quantitative trait loci alleles from a wild rice relative; Oryza rufipogon, Genetics;  citation_year=1998;  citation_volume=150;  citation_pages=899-909; " /><meta name="citation_reference" content="citation_title=Mapping quantitative trait loci controlling sheath blight resistance into rice cultivars (Oryza sativa L.); Theor. Appl. Genet;  citation_year=2000;  citation_volume=101;  citation_pages=569-573; " /><meta name="citation_reference" content="citation_title=Hypervariability of simple sequences as a general source for polymorphic DNA markers; Nucleic Acids Res;  citation_year=1989;  citation_volume=17;  citation_pages=6463-6471; " /><meta name="citation_reference" content="citation_title=Development of microsatellite markers and characterization of simple sequence length polymorphism (SSLP) in rice (Oryza sativa L.); Mol. Gen. Genet;  citation_year=1996;  citation_volume=252;  citation_pages=597-607; " /><meta name="citation_reference" content="citation_title=Abundance, polymorphism and genetic mapping of microsatellites in rice; Mol. Gen. Genet;  citation_year=1993;  citation_volume=241;  citation_pages=225-235; " /><meta name="citation_reference" content="citation_title=Microsatellite DNA markers for rice chromosomes; Theor. Appl. Genet.;  citation_year=1996;  citation_volume=94;  citation_pages=61-67; " /><meta name="citation_reference" content="citation_title=Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, length variation, transposon associations, and genetic marker potential; Genome Res;  citation_year=2001;  citation_volume=11;  citation_pages=1441-1452; " /><meta name="citation_reference" content="citation_title=Mapping and genome organization of microsatellite sequences in rice (Oryza sativa L.); Theor. Appl. Genet;  citation_year=2000;  citation_volume=100;  citation_pages=697-712; " /><meta name="citation_reference" content="citation_title=Development and mapping of 2240 new SSR markers for rice (Oryza sativa L.); DNA Res;  citation_year=2002;  citation_volume=9;  citation_pages=199-207; " /><meta name="citation_reference" content="citation_title=The Genomes of Oryza sativa: a history of duplications; PLoS Biol;  citation_year=2005;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome; citation_author=IRGSP; Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=Sequence variations of simple sequence repeats on chromosome-4 in two subspecies of the Asian cultivated rice; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=392-400; " /><meta name="citation_reference" content="citation_title=BGI-RIS: an integrated information resource and comparative analysis workbench for rice genomics; Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=D377-D382; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; citation_publisher=Humana Press: Totowa, NJ, ; Bioinformatics Methods and Protocols: Methods in Molecular Biology;  citation_year=2000;  citation_pages=365-386.; " /><meta name="citation_reference" content="citation_title=Rapid isolation of high molecular weight plant DNA; Nucleic Acids Res;  citation_year=1980;  citation_volume=8;  citation_pages=4321-4325; " /><meta name="citation_reference" content="citation_title=A novel feature of microsatellites in plants: a distribution gradient along the direction of transcription; FEBS Lett;  citation_year=2003;  citation_volume=554;  citation_pages=17-22; " /><meta name="citation_reference" content="citation_title=Preference of simple sequence repeats in coding and non-coding regions of Arabidopsis thaliana; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=1081-1086; " /><meta name="citation_reference" content="citation_title=Comparing the whole-genome-shotgun and map-based sequences of the rice genome; Trends Plant Sci;  citation_year=2006;  citation_volume=11;  citation_pages=387-391; " /><meta name="citation_reference" content="citation_title=Mono-nucleotide repeats (MNRs): a neglected polymorphism for generating high density genetic maps in silicon; Hum. Genet;  citation_year=2004;  citation_volume=115;  citation_pages=213-220; " /><meta name="citation_reference" content="citation_title=EST-SSR: A new class of genetic markers in cotton; J. Cotton Sci.;  citation_year=2004;  citation_volume=8;  citation_pages=112-123; " /><meta name="citation_reference" content="citation_title=Characterization of RFLP probe sequences for gene discovery and SSR development in Sorghum bicolor (L.) Moench; Theor. Appl. Genet;  citation_year=2002;  citation_volume=105;  citation_pages=912-920; " /><meta name="citation_reference" content="citation_title=Slippage synthesis of simple sequence DNA; Nucleic Acids Res;  citation_year=1992;  citation_volume=20;  citation_pages=211-215; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/1/37/17462700/dsm005.pdf" /><meta name="description" content="Abstract.  Microsatellite (MS) polymorphism is an important source of genetic diversity, providing support for map-based cloning and molecular breeding. We have" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/1/37/503258" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/1/37/503258" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/1/37/503258");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm005';
    </script>
<title>
	Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./503258" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BftHPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgWmhvbmdodWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKYW1lcyBELiBXYXRzb24gSW5zdGl0dXRlIG9mIEdlbm9tZSBTY2llbmNlLCBaaGVqaWFuZyBVbml2ZXJzaXR5LCBIYW5nemhvdSAzMTAwMDgsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIFZlZ2V0YWJsZXMgYW5kIEZsb3dlcnMsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEJlaWppbmcgMTAwMDgxLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRlbmcsIFlhanVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgR2Vub21lIFNjaWVuY2VzIGFuZCBJbmZvcm1hdGlvbiwgQmVpamluZyBJbnN0aXR1dGUgb2YgR2Vub21pY3MsIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZyAxMDEzMDAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuLCBKdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpb24sIENob25ncWluZyBVbml2ZXJzaXR5IG9mIFBvc3RzIGFuZCBUZWxlY29tbXVuaWNhdGlvbnMsIENob25ncWluZyA0MDAwNjUsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHUsIFNvbmduaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSmFtZXMgRC4gV2F0c29uIEluc3RpdHV0ZSBvZiBHZW5vbWUgU2NpZW5jZSwgWmhlamlhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UgMzEwMDA4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEdlbm9tZSBTY2llbmNlcyBhbmQgSW5mb3JtYXRpb24sIEJlaWppbmcgSW5zdGl0dXRlIG9mIEdlbm9taWNzLCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJlaWppbmcgMTAxMzAwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1LCBKdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKYW1lcyBELiBXYXRzb24gSW5zdGl0dXRlIG9mIEdlbm9tZSBTY2llbmNlLCBaaGVqaWFuZyBVbml2ZXJzaXR5LCBIYW5nemhvdSAzMTAwMDgsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgR2Vub21lIFNjaWVuY2VzIGFuZCBJbmZvcm1hdGlvbiwgQmVpamluZyBJbnN0aXR1dGUgb2YgR2Vub21pY3MsIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZyAxMDEzMDAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIENvbXB1dGluZyBUZWNobm9sb2d5LCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJlaWppbmcgMTAxMzAwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilh1ZSwgUWluZ3pob25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSmFtZXMgRC4gV2F0c29uIEluc3RpdHV0ZSBvZiBHZW5vbWUgU2NpZW5jZSwgWmhlamlhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UgMzEwMDA4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyb25vbXksIENvbGxlZ2Ugb2YgQWdyaWN1bHR1cmUgYW5kIEJpb3RlY2hub2xvZ3ksIFpoZWppYW5nIFVuaXZlcnNpdHksIEhhbmd6aG91IDMxMDAyOSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgR2Vub21lLXdpZGUgTWljcm9zYXRlbGxpdGUgUG9seW1vcnBoaXNtIERhdGFiYXNlIGZvciB0aGUgSW5kaWNhIGFuZCBKYXBvbmljYSBSaWNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNtMDA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDcvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBpbiBFc2NoZXJpY2hpYSBjb2xpOiBhYnVuZGFuY2UsIGRpc3RyaWJ1dGlvbiwgY29tcG9zaXRpb24sIGFuZCBwb2x5bW9ycGhpc207IEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02Mi03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBhbmQgYW5hbHlzaXMgb2YgMjUgTWIgb2YgZ2Vub21pYyBETkEgZnJvbSB0aGUgcHVmZmVyZmlzaCBGdWd1IHJ1YnJpcGVzIGJ5IHNlcXVlbmNlIHNjYW5uaW5nOyBHZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz05NjAtOTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXMgYXJlIHByZWZlcmVudGlhbGx5IGFzc29jaWF0ZWQgd2l0aCBub25yZXBldGl0aXZlIEROQSBpbiBwbGFudCBnZW5vbWVzOyBOYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTk0LTIwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaXR5IG9mIG1pY3Jvc2F0ZWxsaXRlcyBkZXJpdmVkIGZyb20gZ2Vub21pYyBsaWJyYXJpZXMgYW5kIEdlbkJhbmsgc2VxdWVuY2VzIGluIHJpY2UgKE9yeXphIHNhdGl2YSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz03MTMtNzIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiB0cmFpdC1pbXByb3ZpbmcgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgYWxsZWxlcyBmcm9tIGEgd2lsZCByaWNlIHJlbGF0aXZlOyBPcnl6YSBydWZpcG9nb24sIEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTg5OS05MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgY29udHJvbGxpbmcgc2hlYXRoIGJsaWdodCByZXNpc3RhbmNlIGludG8gcmljZSBjdWx0aXZhcnMgKE9yeXphIHNhdGl2YSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz01NjktNTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IeXBlcnZhcmlhYmlsaXR5IG9mIHNpbXBsZSBzZXF1ZW5jZXMgYXMgYSBnZW5lcmFsIHNvdXJjZSBmb3IgcG9seW1vcnBoaWMgRE5BIG1hcmtlcnM7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NjQ2My02NDcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNpbXBsZSBzZXF1ZW5jZSBsZW5ndGggcG9seW1vcnBoaXNtIChTU0xQKSBpbiByaWNlIChPcnl6YSBzYXRpdmEgTC4pOyBNb2wuIEdlbi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjUyOyAgY2l0YXRpb25fcGFnZXM9NTk3LTYwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJ1bmRhbmNlLCBwb2x5bW9ycGhpc20gYW5kIGdlbmV0aWMgbWFwcGluZyBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gcmljZTsgTW9sLiBHZW4uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI0MTsgIGNpdGF0aW9uX3BhZ2VzPTIyNS0yMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIEROQSBtYXJrZXJzIGZvciByaWNlIGNocm9tb3NvbWVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTk0OyAgY2l0YXRpb25fcGFnZXM9NjEtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYW5kIGV4cGVyaW1lbnRhbCBhbmFseXNpcyBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTogZnJlcXVlbmN5LCBsZW5ndGggdmFyaWF0aW9uLCB0cmFuc3Bvc29uIGFzc29jaWF0aW9ucywgYW5kIGdlbmV0aWMgbWFya2VyIHBvdGVudGlhbDsgR2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE0NDEtMTQ1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBhbmQgZ2Vub21lIG9yZ2FuaXphdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZSBzZXF1ZW5jZXMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTY5Ny03MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBtYXBwaW5nIG9mIDIyNDAgbmV3IFNTUiBtYXJrZXJzIGZvciByaWNlIChPcnl6YSBzYXRpdmEgTC4pOyBETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xOTktMjA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2Vub21lcyBvZiBPcnl6YSBzYXRpdmE6IGEgaGlzdG9yeSBvZiBkdXBsaWNhdGlvbnM7IFBMb1MgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWFwLWJhc2VkIHNlcXVlbmNlIG9mIHRoZSByaWNlIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUlSR1NQOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM2OyAgY2l0YXRpb25fcGFnZXM9NzkzLTgwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgdmFyaWF0aW9ucyBvZiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBvbiBjaHJvbW9zb21lLTQgaW4gdHdvIHN1YnNwZWNpZXMgb2YgdGhlIEFzaWFuIGN1bHRpdmF0ZWQgcmljZTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0zOTItNDAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CR0ktUklTOiBhbiBpbnRlZ3JhdGVkIGluZm9ybWF0aW9uIHJlc291cmNlIGFuZCBjb21wYXJhdGl2ZSBhbmFseXNpcyB3b3JrYmVuY2ggZm9yIHJpY2UgZ2Vub21pY3M7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDM3Ny1EMzgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltZXIzIG9uIHRoZSBXV1cgZm9yIGdlbmVyYWwgdXNlcnMgYW5kIGZvciBiaW9sb2dpc3QgcHJvZ3JhbW1lcnM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1IdW1hbmEgUHJlc3M6IFRvdG93YSwgTkosIDsgQmlvaW5mb3JtYXRpY3MgTWV0aG9kcyBhbmQgUHJvdG9jb2xzOiBNZXRob2RzIGluIE1vbGVjdWxhciBCaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fcGFnZXM9MzY1LTM4Ni47ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGlzb2xhdGlvbiBvZiBoaWdoIG1vbGVjdWxhciB3ZWlnaHQgcGxhbnQgRE5BOyBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDMyMS00MzI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIGZlYXR1cmUgb2YgbWljcm9zYXRlbGxpdGVzIGluIHBsYW50czogYSBkaXN0cmlidXRpb24gZ3JhZGllbnQgYWxvbmcgdGhlIGRpcmVjdGlvbiBvZiB0cmFuc2NyaXB0aW9uOyBGRUJTIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTU0OyAgY2l0YXRpb25fcGFnZXM9MTctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWZlcmVuY2Ugb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgaW4gY29kaW5nIGFuZCBub24tY29kaW5nIHJlZ2lvbnMgb2YgQXJhYmlkb3BzaXMgdGhhbGlhbmE7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTA4MS0xMDg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpbmcgdGhlIHdob2xlLWdlbm9tZS1zaG90Z3VuIGFuZCBtYXAtYmFzZWQgc2VxdWVuY2VzIG9mIHRoZSByaWNlIGdlbm9tZTsgVHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny0zOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbm8tbnVjbGVvdGlkZSByZXBlYXRzIChNTlJzKTogYSBuZWdsZWN0ZWQgcG9seW1vcnBoaXNtIGZvciBnZW5lcmF0aW5nIGhpZ2ggZGVuc2l0eSBnZW5ldGljIG1hcHMgaW4gc2lsaWNvbjsgSHVtLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz0yMTMtMjIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FU1QtU1NSOiBBIG5ldyBjbGFzcyBvZiBnZW5ldGljIG1hcmtlcnMgaW4gY290dG9uOyBKLiBDb3R0b24gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTEyLTEyMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBSRkxQIHByb2JlIHNlcXVlbmNlcyBmb3IgZ2VuZSBkaXNjb3ZlcnkgYW5kIFNTUiBkZXZlbG9wbWVudCBpbiBTb3JnaHVtIGJpY29sb3IgKEwuKSBNb2VuY2g7IFRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz05MTItOTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbGlwcGFnZSBzeW50aGVzaXMgb2Ygc2ltcGxlIHNlcXVlbmNlIEROQTsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yMTEtMjE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTQvMS8zNy8xNzQ2MjcwMC9kc20wMDUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIE1pY3Jvc2F0ZWxsaXRlIChNUykgcG9seW1vcnBoaXNtIGlzIGFuIGltcG9ydGFudCBzb3VyY2Ugb2YgZ2VuZXRpYyBkaXZlcnNpdHksIHByb3ZpZGluZyBzdXBwb3J0IGZvciBtYXAtYmFzZWQgY2xvbmluZyBhbmQgbW9sZWN1bGFyIGJyZWVkaW5nLiBXZSBoYXZlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj01MDMyNTgmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk7NLnh5OVVsodgfsOvyM/TlOGhmM=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f37%2f503258" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f37%2f503258" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19906" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/1/0/m_dnaresearch14_1.cover.gif?Expires=2147483647&amp;Signature=FMRJ1vzWw3n6RteSNegoL3XeMjVWMPUkUw1DAy9B1s7Po~XXJmQxiq7D2osJ~-qo14VmnnqqwET2h-DyJlv~6ABtVT~LvncFBeJjZ1wxx~y6M3vE3Qs5ixEluS1GT9PiuYZ8KtSanZd2ABbw5KLKYLu92qJh~vhgbf1vjy6lnHZMmx0r7uOO0pacC-GB~PSHCXdcgMyTCPmkGvO2wO9EFShBH0VO0-A5aQVXeyhe0MKcDIjFoBlFl1AmYllwMnIhSIvKtmeMe1oBwYyjgpTM9K4V2BjTo7iH4RO-DPaTutzNC88PpWqK0TxkAqEw7mIyX1rh07TGg~hEL4HMNF39xw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/1">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195892">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195892">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195894">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195894">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195899">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195899">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195909">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195909">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195960">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195960">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195962">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195962">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/1/25/503145">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhonghua Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">James D. Watson Institute of Genome Science</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Hangzhou 310008</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Institute of Vegetables and Flowers</div>, <div class="insititution">Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Beijing 100081</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhonghua+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhonghua Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yajun Deng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 101300</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yajun+Deng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Deng Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Deng Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yajun Deng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Tan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Institute of Bioinformation</div>, <div class="insititution">Chongqing University of Posts and Telecommunications</div>, <div class="addr-line">Chongqing 400065</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Tan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tan J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tan J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jun Tan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Songnian Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">James D. Watson Institute of Genome Science</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Hangzhou 310008</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 101300</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Songnian+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Songnian Hu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">James D. Watson Institute of Genome Science</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Hangzhou 310008</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 101300</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Institute of Computing Technology</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 101300</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jun Yu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qingzhong Xue</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">James D. Watson Institute of Genome Science</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Hangzhou 310008</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Agronomy</div>, <div class="insititution">College of Agriculture and Biotechnology, Zhejiang University</div>, <div class="addr-line">Hangzhou 310029</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86-571-86971611. Fax. <div class="fax">+86-571-86971117</div>, E-mail: <a href="mailto:qzhxue@hotmail.com" target="_blank">qzhxue@hotmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qingzhong+Xue">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xue Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xue Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qingzhong Xue</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <label class="label label--footnote">†</label><div content-id="FN1"><div class="fn"><div class="footnote-content">These authors contributed equally to this paper.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 1, 1 January 2007, Pages 37–45, <a href='https://doi.org/10.1093/dnares/dsm005'>https://doi.org/10.1093/dnares/dsm005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 April 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 July 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">11 March 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="503258" href="/dnaresearch/article-pdf/14/1/37/17462700/dsm005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Zhonghua Zhang, Yajun Deng, Jun Tan, Songnian Hu, Jun Yu, Qingzhong Xue;  A Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice, <em>DNA Research</em>, Volume 14, Issue 1, 1 January 2007, Pages 37–45, <a href='https://doi.org/10.1093/dnares/dsm005'>https://doi.org/10.1093/dnares/dsm005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/503258?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/503258?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/503258?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/503258?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/503258?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/503258?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/503258?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Genome-wide%20Microsatellite%20Polymorphism%20Database%20for%20the%20Indica%20and%20Japonica%20Rice&amp;publicationDate=2007-04-23&amp;volumeNum=14&amp;issueNum=1&amp;author=Zhang%2C%20Zhonghua%3B%20Deng%2C%20Yajun&amp;startPage=37&amp;endPage=45&amp;contentId=10.1093%2Fdnares%2Fdsm005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f37%2f503258" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195892 id="87195892" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Microsatellite (MS) polymorphism is an important source of genetic diversity, providing support for map-based cloning and molecular breeding. We have developed a new database that contains 52 845 polymorphic MS loci between indica and japonica, composed of ample Class II MS markers, and integrated 18 828 MS loci from IRGSP and genetic markers from RGP. Based on genetic marker positions on the rice genome (<a class="link link-uri" href="http://rise.genomics.org.cn/rice2/index.jsp">http://rise.genomics.org.cn/rice2/index.jsp</a> ), we determined the approximate genetic distances of these MS loci and validated 100 randomly selected markers experimentally with 90% success rate. In addition, we recorded polymorphic MS positions in indica cv. 9311 that is the most important paternal parent of the two-line hybrid rice in China. Our database will undoubtedly facilitate the application of MS markers in genetic researches and marker-assisted breeding. The data set is freely available from <a class="link link-uri" href="http://www.wigs.zju.edu.cn/achievment/polySSR">www.wigs.zju.edu.cn/achievment/polySSR</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="polymorphism">polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microsatellite or MS&quot;">microsatellite or MS</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="database">database</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome sequences&quot;">whole-genome sequences</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="rice">rice</a></div>                            <h2 scrollto-destination=87195894 id="87195894" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microsatellites (MSs), or simple sequence repeats (SSRs), are short consecutive repetitive DNA sequence motifs with 1–6 bp in length. The frequency and distribution of MSs have been studied in detail in genomes of bacteria,<sup><a reveal-id="DSM005C1" data-open="DSM005C1" class="link link-ref link-reveal xref-bibr">1</a></sup> animals,<sup><a reveal-id="DSM005C2" data-open="DSM005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and plants.<sup><a reveal-id="DSM005C3" data-open="DSM005C3" class="link link-ref link-reveal xref-bibr">3</a></sup> MSs have become a valuable source of molecular markers, widely used in genetic diversity analysis<sup><a reveal-id="DSM005C4" data-open="DSM005C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and positional cloning in rice<sup><a reveal-id="DSM005C5" data-open="DSM005C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSM005C6" data-open="DSM005C6" class="link link-ref link-reveal xref-bibr">6</a></sup> because of their ubiquity and polymorphism.<sup><a reveal-id="DSM005C7" data-open="DSM005C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Starting with screening genomic DNA libraries with MS probes,<sup><a reveal-id="DSM005C8" data-open="DSM005C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSM005C9" data-open="DSM005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> many investigators have been involved in generating new MS markers<sup><a reveal-id="DSM005C10  DSM005C12" data-open="DSM005C10  DSM005C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> and integrating them into the rice genetic map.<sup><a reveal-id="DSM005C13" data-open="DSM005C13" class="link link-ref link-reveal xref-bibr">13</a></sup> However, the desire to have more markers is still strong, since the amount and their density over rice genomes are not sufficient for satisfying the need of geneticists and breeders.</p><p>It is fortunate that Beijing Genomics Institute<sup><a reveal-id="DSM005C14" data-open="DSM005C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (BGI) and the International Rice Genome Sequencing Project<sup><a reveal-id="DSM005C15" data-open="DSM005C15" class="link link-ref link-reveal xref-bibr">15</a></sup> have released their rice genomic data that provide comparative basis for a thorough analysis on MS between indica and japonica subspecies. Although such a survey has been done recently for chromosome 4 without eliminating MSs with multiple copies,<sup><a reveal-id="DSM005C16" data-open="DSM005C16" class="link link-ref link-reveal xref-bibr">16</a></sup> the report focused on MSs in japonica rice cv. Nipponbare, mainly composed of DNRs, TNRs, and TTRs. In this study, we emphasize our newly discovered MS makers through a comparative analysis on MSs between the indica and japonica genome sequences, and have developed a genome-wide MS polymorphism database that have markers indexed in both rice genomes to facilitate molecular studies and marker-assisted breeding practice.</p>                            <h2 scrollto-destination=87195899 id="87195899" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87195900 id="87195900" class="section-title" data-legacy-id=s2a>2.1. Rice genome sequences and MS screening</h3>
<p>The genome sequences for indica (<em>Oryza sativa</em> ssp. indica cv. 9311) and japonica (<em>O. sativa</em> ssp. japonica cv. Nipponbare) (Version build4) were downloaded from <a class="link link-uri" href="http://rise.genomics.org.cn/rice2/index.jsp">http://rise.genomics.org.cn/rice2/index.jsp</a><sup><a reveal-id="DSM005C17" data-open="DSM005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and <a class="link link-uri" href="http://rgp.dna.affrc.go.jp">http://rgp.dna.affrc.go.jp</a><sup><a reveal-id="DSM005C15" data-open="DSM005C15" class="link link-ref link-reveal xref-bibr">15</a></sup> respectively. The 5′-UTR, coding, intronic, 3′-UTR, and intergenic regions were determined based on their original annotations. An MS motif scanning tool, SSRIT<sup><a reveal-id="DSM005C11" data-open="DSM005C11" class="link link-ref link-reveal xref-bibr">11</a></sup> written in Perl was downloaded from Cornell University (<a class="link link-uri" href="http://www.gramene.org/">http://www.gramene.org/</a>) and modified to search for perfect MS repeats in the two rice genome sequences, including mono-nucleotide repeats (MNRs), di-nucleotide repeats (DNRs), tri-nucleotide repeats (TNRs), tetra-nucleotide repeats (TTRs), penta-nucleotide repeats (PNRs), and hexa-nucleotide repeats (HNRs) with numbers of repeat units of 10, 7, 6, 5, 4 and 4, respectively. Some of these MS sequences were combined into one repeat complex if the lengths between two adjacent units were less than 20 bp.</p>                            <h3 scrollto-destination=87195903 id="87195903" class="section-title" data-legacy-id=s2b>2.2. MS Polymorphism and primer design</h3>
<p>An MS locus is defined as a tract of genomic sequences that include a perfect repeat motif and two unique 160-bp flanking sequences on each sides of the repeat. These MS candidate sequences were aligned to their source genome sequences with BLASTN (E-value cutoff of 1e-40); those with unique hit were selected as candidate MS loci together with their specific flanking sequences. The selected MS sequences were then used as queries against another genome without filtering low complexity sequences. We wrote a Perl script (Supplementary script: polymorphism.pl) to determine the presence of polymorphisms among MS motifs between indica and japonica. Primers pairs were automatically designed on the basis of unique flanking sequences aided by Primer3<sup><a reveal-id="DSM005C18" data-open="DSM005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> with its default parameters. Candidate primer sequences were subsequently filtered through following parameters: length range from 18 to 24 bp, devoid of consecutive tracts of single nucleotide, GC content around 50% with a theoretical T<sub>m</sub> approximately 60°C, and perfect ending with G- or C-rich at the 3′ end.</p>                            <h3 scrollto-destination=87195905 id="87195905" class="section-title" data-legacy-id=s2c>2.3. Experimental validation</h3>
<p>One hundred primer pairs covering all rice chromosomes and covering all MS types were used for our validation experiment. The DNA templates used in this experiment are <em>O. sativa</em> ssp. indica cv. 9311 and <em>O. sativa</em> ssp. japonica cv. Nipponbare grown in a green house. Genomic DNA was extracted from 1-week-old seedlings, using the CTAB method.<sup><a reveal-id="DSM005C19" data-open="DSM005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The quality of DNA was evaluated electrophoretically with 0.8% agarose gel.</p><p>Oligonucleotide primers were made by Shanghai DNA Biotechnologies Co., Ltd. PCR was performed in 25 µl reactions containing 3.0 µl buffer, 2.5 µl MgCl2 (25 mM), 2.0 µl dNTP (2.5 mM), 0.5 µl <em>Taq</em> polymerase (5 U/µl), 3.0 µl template DNA (10 ng/µl), 13 µl ddH<sub>2</sub>O, 0.1 µg primers. A PTC-100TM thermal cycler (MJ Research Inc. Incline Village, NV, USA) was used along with the following PCR profile settings: 2 min at 96°C, 35 cycles of 40 s at 95°C, 30 s at 61°C, 1 min 30 s at 72°C, and a final extension at 72°C for 10 min. PCR products were electrophoresed on 1.0% agarose gel containing ethidium bromide. Montage TM PCR96 Cleanup Kits were used to purify PCR products before being sequenced with an ABI 377 DNA sequencer.</p>                            <h2 scrollto-destination=87195909 id="87195909" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87195910 id="87195910" class="section-title" data-legacy-id=s3a>3.1. MS identification and distribution in the indica and japonica genomes</h3>
<p>We identified 102 706 and 102 423 candidate MSs in the indica and japonica genomes, respectively, and the average distances between the repeat units are shorter in the indica genome (3.6 kb) than in the japonica (3.7 kb) genome. These candidate MSs reside largely within intergenic regions (58 094 or 56.6% for the indica and 58 812 or 57.4% for the japonica), as compared to those in intronic regions (20 868 or 20.3% in the indica and 19 631 or 19.2% for the japonica) (Table <a reveal-id="DSM005TB1" data-open="DSM005TB1" class="link link-reveal link-table xref-fig">1</a>). However, the average MS lengths in the indica and japonica genomes were found similar, from 17.4 bp to 17.8 bp. The fractions of MSs with unique flanking sequences in the indica and japonica are 67.1% and 72.5%, respectively. We also found that one-third of the total MSs are polymorphic and that the number of MS with unique flanking sequences was slightly lower in the indica (68 895) than in the japonica (74 227), but the number of MS sequences with polymorphisms and their average lengths were nearly the same between the two subspecies (Table <a reveal-id="DSM005TB1" data-open="DSM005TB1" class="link link-reveal link-table xref-fig">1</a>). It implies that the number of MS with polymorphisms and the average length of polymorphic SSRs are independent parameters in the two rice subspecies. MS loci most likely to be useful as genetic markers are those that are single copy in both genomes and polymorphic when the genomes are compared.</p>                        <a id="87195913" scrollto-destination="87195913"></a>
<div content-id="DSM005TB1" class="table-modal"><div class="table-wrap"><div id="DSM005TB1" class="table-wrap-title" data-id="DSM005TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Frequency and polymorphism of MSs in different genomic regions of indica and japonica genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Genome region </th><th colspan="4">All<sup>a</sup> </th><th colspan="3">Wsfs<sup>b</sup> </th><th colspan="3">Wpm<sup>c</sup> </th><th rowspan="2">Rate of pm<sup>d</sup> (%) </th></tr><tr><th>Count </th><th>Interval (kbp) </th><th>Length<sup>e</sup> </th><th>GC (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>f</sup> (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>g</sup> (%) </th></tr></thead><tbody><tr><td>Indica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 411 </td><td>2.1 </td><td>17.4 </td><td>40.1 </td><td>9324 </td><td>2.8 </td><td>75.1 </td><td>4613 </td><td>5.6 </td><td>37.2 </td><td>49.5 </td></tr><tr><td> CDS </td><td>5406 </td><td>8.1 </td><td>20.8 </td><td>81.4 </td><td>4217 </td><td>10.4 </td><td>78 </td><td>1162 </td><td>37.6 </td><td>21.5 </td><td>27.6 </td></tr><tr><td> Intron </td><td>20 868 </td><td>2.9 </td><td>15.7 </td><td>21.5 </td><td>15 670 </td><td>3.8 </td><td>75.1 </td><td>8551 </td><td>7 </td><td>41 </td><td>54.6 </td></tr><tr><td> 3′-UTR </td><td>5927 </td><td>3.2 </td><td>16.6 </td><td>20.1 </td><td>4338 </td><td>4.4 </td><td>73.2 </td><td>2312 </td><td>8.3 </td><td>39 </td><td>53.3 </td></tr><tr><td> Intergenic </td><td>58 094 </td><td>3.9 </td><td>17.8 </td><td>19.9 </td><td>35 346 </td><td>6.4 </td><td>60.8 </td><td>19 053 </td><td>11.8 </td><td>32.8 </td><td>53.9 </td></tr><tr><td> Total </td><td>102 706 </td><td>3.6 </td><td>17.4 </td><td>26.5 </td><td>68 895 </td><td>5.4 </td><td>67.1 </td><td>35 691 </td><td>10.5 </td><td>34.8 </td><td>51.8 </td></tr><tr><td>Japonica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 569 </td><td>2 </td><td>17.6 </td><td>42.0 </td><td>10 462 </td><td>2.4 </td><td>83.2 </td><td>4871 </td><td>5.1 </td><td>38.8 </td><td>46.6 </td></tr><tr><td> CDS </td><td>5685 </td><td>7.7 </td><td>20.6 </td><td>81.4 </td><td>4854 </td><td>9 </td><td>85.4 </td><td>1249 </td><td>35 </td><td>22 </td><td>25.7 </td></tr><tr><td> Intron </td><td>19 631 </td><td>2.9 </td><td>15.7 </td><td>22.2 </td><td>16 316 </td><td>3.5 </td><td>83.1 </td><td>8281 </td><td>6.9 </td><td>42.2 </td><td>50.8 </td></tr><tr><td> 3′-UTR </td><td>5726 </td><td>3.2 </td><td>16.7 </td><td>20.8 </td><td>4647 </td><td>3.9 </td><td>81.2 </td><td>2321 </td><td>7.9 </td><td>40.5 </td><td>49.9 </td></tr><tr><td> Intergenic </td><td>58 812 </td><td>4 </td><td>18.5 </td><td>21.1 </td><td>37 948 </td><td>6.3 </td><td>64.5 </td><td>19 057 </td><td>12.5 </td><td>32.4 </td><td>50.2 </td></tr><tr><td> Total </td><td>102 423 </td><td>3.7 </td><td>17.8 </td><td>27.7 </td><td>74 227 </td><td>5.1 </td><td>72.5 </td><td>35 779 </td><td>10.7 </td><td>34.9 </td><td>48.2 </td></tr></tbody></table></div><div content-id="DSM005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Genome region </th><th colspan="4">All<sup>a</sup> </th><th colspan="3">Wsfs<sup>b</sup> </th><th colspan="3">Wpm<sup>c</sup> </th><th rowspan="2">Rate of pm<sup>d</sup> (%) </th></tr><tr><th>Count </th><th>Interval (kbp) </th><th>Length<sup>e</sup> </th><th>GC (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>f</sup> (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>g</sup> (%) </th></tr></thead><tbody><tr><td>Indica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 411 </td><td>2.1 </td><td>17.4 </td><td>40.1 </td><td>9324 </td><td>2.8 </td><td>75.1 </td><td>4613 </td><td>5.6 </td><td>37.2 </td><td>49.5 </td></tr><tr><td> CDS </td><td>5406 </td><td>8.1 </td><td>20.8 </td><td>81.4 </td><td>4217 </td><td>10.4 </td><td>78 </td><td>1162 </td><td>37.6 </td><td>21.5 </td><td>27.6 </td></tr><tr><td> Intron </td><td>20 868 </td><td>2.9 </td><td>15.7 </td><td>21.5 </td><td>15 670 </td><td>3.8 </td><td>75.1 </td><td>8551 </td><td>7 </td><td>41 </td><td>54.6 </td></tr><tr><td> 3′-UTR </td><td>5927 </td><td>3.2 </td><td>16.6 </td><td>20.1 </td><td>4338 </td><td>4.4 </td><td>73.2 </td><td>2312 </td><td>8.3 </td><td>39 </td><td>53.3 </td></tr><tr><td> Intergenic </td><td>58 094 </td><td>3.9 </td><td>17.8 </td><td>19.9 </td><td>35 346 </td><td>6.4 </td><td>60.8 </td><td>19 053 </td><td>11.8 </td><td>32.8 </td><td>53.9 </td></tr><tr><td> Total </td><td>102 706 </td><td>3.6 </td><td>17.4 </td><td>26.5 </td><td>68 895 </td><td>5.4 </td><td>67.1 </td><td>35 691 </td><td>10.5 </td><td>34.8 </td><td>51.8 </td></tr><tr><td>Japonica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 569 </td><td>2 </td><td>17.6 </td><td>42.0 </td><td>10 462 </td><td>2.4 </td><td>83.2 </td><td>4871 </td><td>5.1 </td><td>38.8 </td><td>46.6 </td></tr><tr><td> CDS </td><td>5685 </td><td>7.7 </td><td>20.6 </td><td>81.4 </td><td>4854 </td><td>9 </td><td>85.4 </td><td>1249 </td><td>35 </td><td>22 </td><td>25.7 </td></tr><tr><td> Intron </td><td>19 631 </td><td>2.9 </td><td>15.7 </td><td>22.2 </td><td>16 316 </td><td>3.5 </td><td>83.1 </td><td>8281 </td><td>6.9 </td><td>42.2 </td><td>50.8 </td></tr><tr><td> 3′-UTR </td><td>5726 </td><td>3.2 </td><td>16.7 </td><td>20.8 </td><td>4647 </td><td>3.9 </td><td>81.2 </td><td>2321 </td><td>7.9 </td><td>40.5 </td><td>49.9 </td></tr><tr><td> Intergenic </td><td>58 812 </td><td>4 </td><td>18.5 </td><td>21.1 </td><td>37 948 </td><td>6.3 </td><td>64.5 </td><td>19 057 </td><td>12.5 </td><td>32.4 </td><td>50.2 </td></tr><tr><td> Total </td><td>102 423 </td><td>3.7 </td><td>17.8 </td><td>27.7 </td><td>74 227 </td><td>5.1 </td><td>72.5 </td><td>35 779 </td><td>10.7 </td><td>34.9 </td><td>48.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>All MSs in the genomes.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>MSs with unique flanking sequences.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>MSs with polymorphisms.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Rate of MS Wpm in MS Wsfs.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Average length of MSs expressed in base pairs (bp).</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Percent of MSs with unique flanking sequences against all.</p></div></div><div content-id=""><div class="fn"><p><sup>g</sup>Percent of MSs with polymorphisms against all.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195913" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM005TB1" class="table-wrap-title" data-id="DSM005TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Frequency and polymorphism of MSs in different genomic regions of indica and japonica genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Genome region </th><th colspan="4">All<sup>a</sup> </th><th colspan="3">Wsfs<sup>b</sup> </th><th colspan="3">Wpm<sup>c</sup> </th><th rowspan="2">Rate of pm<sup>d</sup> (%) </th></tr><tr><th>Count </th><th>Interval (kbp) </th><th>Length<sup>e</sup> </th><th>GC (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>f</sup> (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>g</sup> (%) </th></tr></thead><tbody><tr><td>Indica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 411 </td><td>2.1 </td><td>17.4 </td><td>40.1 </td><td>9324 </td><td>2.8 </td><td>75.1 </td><td>4613 </td><td>5.6 </td><td>37.2 </td><td>49.5 </td></tr><tr><td> CDS </td><td>5406 </td><td>8.1 </td><td>20.8 </td><td>81.4 </td><td>4217 </td><td>10.4 </td><td>78 </td><td>1162 </td><td>37.6 </td><td>21.5 </td><td>27.6 </td></tr><tr><td> Intron </td><td>20 868 </td><td>2.9 </td><td>15.7 </td><td>21.5 </td><td>15 670 </td><td>3.8 </td><td>75.1 </td><td>8551 </td><td>7 </td><td>41 </td><td>54.6 </td></tr><tr><td> 3′-UTR </td><td>5927 </td><td>3.2 </td><td>16.6 </td><td>20.1 </td><td>4338 </td><td>4.4 </td><td>73.2 </td><td>2312 </td><td>8.3 </td><td>39 </td><td>53.3 </td></tr><tr><td> Intergenic </td><td>58 094 </td><td>3.9 </td><td>17.8 </td><td>19.9 </td><td>35 346 </td><td>6.4 </td><td>60.8 </td><td>19 053 </td><td>11.8 </td><td>32.8 </td><td>53.9 </td></tr><tr><td> Total </td><td>102 706 </td><td>3.6 </td><td>17.4 </td><td>26.5 </td><td>68 895 </td><td>5.4 </td><td>67.1 </td><td>35 691 </td><td>10.5 </td><td>34.8 </td><td>51.8 </td></tr><tr><td>Japonica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 569 </td><td>2 </td><td>17.6 </td><td>42.0 </td><td>10 462 </td><td>2.4 </td><td>83.2 </td><td>4871 </td><td>5.1 </td><td>38.8 </td><td>46.6 </td></tr><tr><td> CDS </td><td>5685 </td><td>7.7 </td><td>20.6 </td><td>81.4 </td><td>4854 </td><td>9 </td><td>85.4 </td><td>1249 </td><td>35 </td><td>22 </td><td>25.7 </td></tr><tr><td> Intron </td><td>19 631 </td><td>2.9 </td><td>15.7 </td><td>22.2 </td><td>16 316 </td><td>3.5 </td><td>83.1 </td><td>8281 </td><td>6.9 </td><td>42.2 </td><td>50.8 </td></tr><tr><td> 3′-UTR </td><td>5726 </td><td>3.2 </td><td>16.7 </td><td>20.8 </td><td>4647 </td><td>3.9 </td><td>81.2 </td><td>2321 </td><td>7.9 </td><td>40.5 </td><td>49.9 </td></tr><tr><td> Intergenic </td><td>58 812 </td><td>4 </td><td>18.5 </td><td>21.1 </td><td>37 948 </td><td>6.3 </td><td>64.5 </td><td>19 057 </td><td>12.5 </td><td>32.4 </td><td>50.2 </td></tr><tr><td> Total </td><td>102 423 </td><td>3.7 </td><td>17.8 </td><td>27.7 </td><td>74 227 </td><td>5.1 </td><td>72.5 </td><td>35 779 </td><td>10.7 </td><td>34.9 </td><td>48.2 </td></tr></tbody></table></div><div content-id="DSM005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Genome region </th><th colspan="4">All<sup>a</sup> </th><th colspan="3">Wsfs<sup>b</sup> </th><th colspan="3">Wpm<sup>c</sup> </th><th rowspan="2">Rate of pm<sup>d</sup> (%) </th></tr><tr><th>Count </th><th>Interval (kbp) </th><th>Length<sup>e</sup> </th><th>GC (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>f</sup> (%) </th><th>Count </th><th>Interval (kbp) </th><th>Per<sup>g</sup> (%) </th></tr></thead><tbody><tr><td>Indica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 411 </td><td>2.1 </td><td>17.4 </td><td>40.1 </td><td>9324 </td><td>2.8 </td><td>75.1 </td><td>4613 </td><td>5.6 </td><td>37.2 </td><td>49.5 </td></tr><tr><td> CDS </td><td>5406 </td><td>8.1 </td><td>20.8 </td><td>81.4 </td><td>4217 </td><td>10.4 </td><td>78 </td><td>1162 </td><td>37.6 </td><td>21.5 </td><td>27.6 </td></tr><tr><td> Intron </td><td>20 868 </td><td>2.9 </td><td>15.7 </td><td>21.5 </td><td>15 670 </td><td>3.8 </td><td>75.1 </td><td>8551 </td><td>7 </td><td>41 </td><td>54.6 </td></tr><tr><td> 3′-UTR </td><td>5927 </td><td>3.2 </td><td>16.6 </td><td>20.1 </td><td>4338 </td><td>4.4 </td><td>73.2 </td><td>2312 </td><td>8.3 </td><td>39 </td><td>53.3 </td></tr><tr><td> Intergenic </td><td>58 094 </td><td>3.9 </td><td>17.8 </td><td>19.9 </td><td>35 346 </td><td>6.4 </td><td>60.8 </td><td>19 053 </td><td>11.8 </td><td>32.8 </td><td>53.9 </td></tr><tr><td> Total </td><td>102 706 </td><td>3.6 </td><td>17.4 </td><td>26.5 </td><td>68 895 </td><td>5.4 </td><td>67.1 </td><td>35 691 </td><td>10.5 </td><td>34.8 </td><td>51.8 </td></tr><tr><td>Japonica </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> 5′-UTR </td><td>12 569 </td><td>2 </td><td>17.6 </td><td>42.0 </td><td>10 462 </td><td>2.4 </td><td>83.2 </td><td>4871 </td><td>5.1 </td><td>38.8 </td><td>46.6 </td></tr><tr><td> CDS </td><td>5685 </td><td>7.7 </td><td>20.6 </td><td>81.4 </td><td>4854 </td><td>9 </td><td>85.4 </td><td>1249 </td><td>35 </td><td>22 </td><td>25.7 </td></tr><tr><td> Intron </td><td>19 631 </td><td>2.9 </td><td>15.7 </td><td>22.2 </td><td>16 316 </td><td>3.5 </td><td>83.1 </td><td>8281 </td><td>6.9 </td><td>42.2 </td><td>50.8 </td></tr><tr><td> 3′-UTR </td><td>5726 </td><td>3.2 </td><td>16.7 </td><td>20.8 </td><td>4647 </td><td>3.9 </td><td>81.2 </td><td>2321 </td><td>7.9 </td><td>40.5 </td><td>49.9 </td></tr><tr><td> Intergenic </td><td>58 812 </td><td>4 </td><td>18.5 </td><td>21.1 </td><td>37 948 </td><td>6.3 </td><td>64.5 </td><td>19 057 </td><td>12.5 </td><td>32.4 </td><td>50.2 </td></tr><tr><td> Total </td><td>102 423 </td><td>3.7 </td><td>17.8 </td><td>27.7 </td><td>74 227 </td><td>5.1 </td><td>72.5 </td><td>35 779 </td><td>10.7 </td><td>34.9 </td><td>48.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>All MSs in the genomes.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>MSs with unique flanking sequences.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>MSs with polymorphisms.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Rate of MS Wpm in MS Wsfs.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Average length of MSs expressed in base pairs (bp).</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Percent of MSs with unique flanking sequences against all.</p></div></div><div content-id=""><div class="fn"><p><sup>g</sup>Percent of MSs with polymorphisms against all.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195913" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195914 id="87195914" class="section-title" data-legacy-id=s3b>3.2. MS polymorphisms in different genomic regions</h3>
<p>We have noticed that the average MS length and distance between candidate MS loci within coding regions of the two subspecies were slightly different. The indica genome has an average MS length of 20.8 bp and an average distance between candidate MS loci of 8.1 kb; the corresponding numbers in the japonica genome are 20.6 bp and 7.7 kb, respectively. In the regions other than the coding sequences, the average MS length and the average distance between candidate MS loci become 2.1–3.9 bp and 2.0–4.0 kb, respectively, although MS sequences are most abundant in 5′-UTR, followed in an order of intronic, 3′-UTR, intergenic, and coding regions. The trend was found consistent with previous reports in <em>Arabidopsis</em> and rice.<sup><a reveal-id="DSM005C3" data-open="DSM005C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSM005C20" data-open="DSM005C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSM005C21" data-open="DSM005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In addition, GC contents of MS in other regions (19.9–40.1 and 20.8–42.0) are far lower than that of coding regions that are 81.4% and 81.4 % for the indica and japonica, respectively.</p><p>We also looked into details of MSs and their unique flanking sequences among different regions in the indica genome and found that there are higher numbers of MSs with unique flanking sequences in intergenic, intronic, and 5′-UTR regions, and the trend is similar in japonica. MSs with both polymorphism and unique flanking sequences vary in different genomic regions, there are as low as 21.5 and 22% in coding regions in the indica and japonica genomes, respectively, whereas there is up to 40% observed in the intronic region (Table <a reveal-id="DSM005TB1" data-open="DSM005TB1" class="link link-reveal link-table xref-fig">1</a>). We concluded that MSs within intronic regions are most polymorphic, and followed by UTR, intergenic, and coding regions.</p>                            <h3 scrollto-destination=87195918 id="87195918" class="section-title" data-legacy-id=s3c>3.3. Polymorphisms and distribution of different MS repeat types</h3>
<p>In our collection, there are 62.5% MNRs, 17.2% DNRs, and 10.6% TNRs, which form the rice genome sequences; these repeat types are collectively over 90% of the total MS sequences. The distributions of these MS types are rather similar between indica and japonica in general. TNRs were predominately found in coding regions (87.5% for indica and 87.6% for japonica, respectively). MNRs and DNRs are highly dispersed in intronic, 3′-UTR, intergenic, and 5′-UTR regions (Fig. <a reveal-id="DSM005F1" data-open="DSM005F1" class="link link-reveal link-table xref-fig">1</a>A). The distribution of polymorphic MS loci is similar among all types, except the percentage of TNRs in various genomic regions varies significantly (Fig. <a reveal-id="DSM005F1" data-open="DSM005F1" class="link link-reveal link-table xref-fig">1</a>B). With respect to the polymorphism rate (the ratio between MSs with polymorphism and the total) and those with unique flanking sequences, we found that MNRs and DNRs are better than TNRs in all genomic regions (Fig. <a reveal-id="DSM005F1" data-open="DSM005F1" class="link link-reveal link-table xref-fig">1</a>C). The polymorphism rate of TTRs, PNRs, and HNRs varies from 32 to 40% (data not shown), and they may provide additional sources for molecular makers when called upon.</p>                        <a id="87195922" scrollto-destination="87195922"></a>
<div id="DSM005F1" data-id="DSM005F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/m_dsm00501.jpeg?Expires=2147483647&Signature=AjQepLm4w9ExqdV6uIU4pzSYD9kmXRe-ijWkdDb2I2Syta5y7YXTp~4QwTI1Ev5GfD9k6PEwFwLuNDr4FkxjSpqiMm5xoC3FegjX8uCzNJP7ZhOvLAD7iVIan1tQvDgrhfI7ucGhotcQC14yk5GLEjgb7qpS8nrNipD1IEpw3venRMFCudkOTj91t~vuCl1Z8sDsfBZakbJwV0w2k4XHinbRxRr9KYZLB55YWO6wdgBTTmx40I~1Nd-ZMWoyjzlKOi2lNjbrmyb8baKabgn7GK-22vVx01NI6svOnYcrPKWe6uvGozc~BlQnlFQV1AKV8DFJgBHe0FrPnrM4ssHDaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Percentages and polymorphic rates of MNR, DNR, and TNR in different regions of the two rice genomes. (A) Percentages of MNR, DNR, and TNR in the overall MS. (B) Percentages of MNR, DNR, and TNR with polymorphisms. (C) Polymorphic rates of MNR, DNR, and TNR." path-from-xml="dsm00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195922/dsm00501.jpeg" path-from-xml="dsm00501.jpeg" target="_blank">View large</a><a section="87195922" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/dsm00501.jpeg?Expires=2147483647&Signature=IxYq-p1tXM3-jWojpQvy1-~P96nCHQ7qG6TNuWUSQL0zvZMqBBbAbgNSZTDVe8SEDEya~lxxi0JW1i55quN6XfDCFk11vyw6orKbJLQhpb3YJ-JlAa978O8-0eU0OCxLw1VBQSJVeGJ7C1L5DC~xfa1ikOc-EBG~foNAR6LXdilibX1ER3dT6SNo2QkYd1LMWOkek6Vaepz0M9Y0dIHINHer1IJFgUHXDW4NmZPda~CsE9YKjcpOonn4M0wqOAhEfeGCpN3mcSLG7YqY4TOl536TRbvJKf6tzHwvYsEatn5t~f4CBODCv7ez0B19vWwzZO4AgT5efUWHALYZ9rGWeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195922&ar=503258&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Percentages and polymorphic rates of MNR, DNR, and TNR in different regions of the two rice genomes. (<strong>A</strong>) Percentages of MNR, DNR, and TNR in the overall MS. (<strong>B</strong>) Percentages of MNR, DNR, and TNR with polymorphisms. (<strong>C</strong>) Polymorphic rates of MNR, DNR, and TNR.</p></div></div><div content-id="DSM005F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/m_dsm00501.jpeg?Expires=2147483647&Signature=AjQepLm4w9ExqdV6uIU4pzSYD9kmXRe-ijWkdDb2I2Syta5y7YXTp~4QwTI1Ev5GfD9k6PEwFwLuNDr4FkxjSpqiMm5xoC3FegjX8uCzNJP7ZhOvLAD7iVIan1tQvDgrhfI7ucGhotcQC14yk5GLEjgb7qpS8nrNipD1IEpw3venRMFCudkOTj91t~vuCl1Z8sDsfBZakbJwV0w2k4XHinbRxRr9KYZLB55YWO6wdgBTTmx40I~1Nd-ZMWoyjzlKOi2lNjbrmyb8baKabgn7GK-22vVx01NI6svOnYcrPKWe6uvGozc~BlQnlFQV1AKV8DFJgBHe0FrPnrM4ssHDaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Percentages and polymorphic rates of MNR, DNR, and TNR in different regions of the two rice genomes. (A) Percentages of MNR, DNR, and TNR in the overall MS. (B) Percentages of MNR, DNR, and TNR with polymorphisms. (C) Polymorphic rates of MNR, DNR, and TNR." path-from-xml="dsm00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195922/dsm00501.jpeg" path-from-xml="dsm00501.jpeg" target="_blank">View large</a><a section="87195922" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/dsm00501.jpeg?Expires=2147483647&Signature=IxYq-p1tXM3-jWojpQvy1-~P96nCHQ7qG6TNuWUSQL0zvZMqBBbAbgNSZTDVe8SEDEya~lxxi0JW1i55quN6XfDCFk11vyw6orKbJLQhpb3YJ-JlAa978O8-0eU0OCxLw1VBQSJVeGJ7C1L5DC~xfa1ikOc-EBG~foNAR6LXdilibX1ER3dT6SNo2QkYd1LMWOkek6Vaepz0M9Y0dIHINHer1IJFgUHXDW4NmZPda~CsE9YKjcpOonn4M0wqOAhEfeGCpN3mcSLG7YqY4TOl536TRbvJKf6tzHwvYsEatn5t~f4CBODCv7ez0B19vWwzZO4AgT5efUWHALYZ9rGWeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195922&ar=503258&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Percentages and polymorphic rates of MNR, DNR, and TNR in different regions of the two rice genomes. (<strong>A</strong>) Percentages of MNR, DNR, and TNR in the overall MS. (<strong>B</strong>) Percentages of MNR, DNR, and TNR with polymorphisms. (<strong>C</strong>) Polymorphic rates of MNR, DNR, and TNR.</p></div></div>                            <h3 scrollto-destination=87195925 id="87195925" class="section-title" data-legacy-id=s3d>3.4. Polymorphisms of the candidate MS loci with different repeat units</h3>
<p>The number of A/T repeats is 8.5 and 6.8 times of all repeat types in the indica and japonica genomes, respectively; it is 7.5 and 6.6 times more polymorphic than C/G repeats. The average MS length (13.8 bp in the indica and 13.6 bp in the japonica) and the average length of MSs with polymorphisms (14.4 bp in indica and 14.1 bp in japonica) in C/G repeats are slightly longer than those of A/T repeats (10.8–11.2 bp). The polymorphism rate of C/G MNR class of SSR is slightly higher (68.6% for indica and 61.3% for japonica) than that of A/T MNRs (53.5% for indica and 49.6% for japonica). Therefore, when targeting MNRs as the basis for developing genetic markers, the C/G repeat class is likely to be more useful, at least when comparing indica and japonica genotypes. Among DNRs, both AG/CT (59.8–56.4%) and AT/TA (&gt;50%) repeats are more polymorphic than CG/GC repeats (15.6%). AT/TA repeats in the indica and japonica assemblies have a longer average length (&gt;30 bp) than AG/CT repeats (22.6–22.5 bp; Table <a reveal-id="DSM005TB2" data-open="DSM005TB2" class="link link-reveal link-table xref-fig">2</a>). The polymorphism rate of GC-rich TNRs is relatively lower than that of GC-poor TNRs, despite the fact that CCG/CGG repeats are the most abundant, contributing more than 40% to the total TNRs. The GC-poor repeats, such as AAT/ATT, AAG/CTT, and ACT/AGT, have polymorphic rates greater than 40% (Table <a reveal-id="DSM005TB2" data-open="DSM005TB2" class="link link-reveal link-table xref-fig">2</a>). Another example is AGG/CCT that contributes to 15% of the total TNRs but the polymorphic rate of AGG/CCT repeats is relatively low, 35.5% for indica and 31.7% for japonica. The numbers of polymorphic ATC/GCT, AAC/GTT, and ACT/AGT repeats are scarce. Among TTRs, AGAT/ATCT, AAAT/ATTT, AAAG/CTTT, ACAT/ATGT, AGCT/AGCT, and AATT/AATT are quite dominant, and the two kinds of TTR (ACAT/ATGT and AGAT/ATCT) have longer average lengths and higher polymorphism rates. Among PNRs, AAAAG/CTTTT, AAAAT/ATTTT, AGAGG/CCTCT, and AGGGG/CCCCT are rather dominant, AGAGG/CCTCT is more polymorphic, 24.8% in indica and 31.7% in japonica; all other kinds of PNR have much lower polymorphic rates (data not shown).</p>                        <a id="87195930" scrollto-destination="87195930"></a>
<div content-id="DSM005TB2" class="table-modal"><div class="table-wrap"><div id="DSM005TB2" class="table-wrap-title" data-id="DSM005TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Polymorphism distribution of MS motifs in indica and japonica genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Motifs </th><th colspan="4">Indica </th><th>Rate of pm </th><th colspan="4">Japonica </th><th>Rate of pm </th></tr><tr><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th></tr><tr><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th></tr></thead><tbody><tr><td>A/T </td><td>57 467 </td><td>10.9 </td><td>21 393 </td><td>11.2 </td><td>53.5 </td><td>53 940 </td><td>10.8 </td><td>20 637 </td><td>11.1 </td><td>49.6 </td></tr><tr><td>C/G </td><td>6710 </td><td>13.8 </td><td>2849 </td><td>14.4 </td><td>68.6 </td><td>7923 </td><td>13.6 </td><td>3104 </td><td>14.1 </td><td>61.3 </td></tr><tr><td>AC/GT </td><td>1405 </td><td>18.2 </td><td>541 </td><td>19.6 </td><td>54.4 </td><td>1318 </td><td>18.1 </td><td>560 </td><td>19.4 </td><td>52.5 </td></tr><tr><td>AG/CT </td><td>8371 </td><td>20.1 </td><td>3220 </td><td>22.6 </td><td>59.8 </td><td>8664 </td><td>19.7 </td><td>3351 </td><td>22.5 </td><td>56.4 </td></tr><tr><td>AT/AT </td><td>7271 </td><td>34.5 </td><td>2147 </td><td>36.8 </td><td>56.6 </td><td>8041 </td><td>35.9 </td><td>2287 </td><td>38.1 </td><td>54.4 </td></tr><tr><td>CG/CG </td><td>668 </td><td>14.8 </td><td>59 </td><td>15.7 </td><td>15.6 </td><td>755 </td><td>14.7 </td><td>64 </td><td>15.5 </td><td>15.6 </td></tr><tr><td>AAC/GTT </td><td>186 </td><td>24.5 </td><td>45 </td><td>30.7 </td><td>37.8 </td><td>185 </td><td>22.7 </td><td>57 </td><td>25.2 </td><td>41.6 </td></tr><tr><td>AAG/CTT </td><td>892 </td><td>25.1 </td><td>304 </td><td>29.0 </td><td>46.5 </td><td>863 </td><td>26.0 </td><td>305 </td><td>30.1 </td><td>41.8 </td></tr><tr><td>AAT/ATT </td><td>683 </td><td>45.0 </td><td>248 </td><td>51.7 </td><td>55.6 </td><td>671 </td><td>49.5 </td><td>253 </td><td>59.5 </td><td>49.6 </td></tr><tr><td>ACC/GGT </td><td>603 </td><td>20.1 </td><td>111 </td><td>21.3 </td><td>23.8 </td><td>632 </td><td>20.1 </td><td>120 </td><td>21.7 </td><td>22.2 </td></tr><tr><td>ACG/CGT </td><td>832 </td><td>19.7 </td><td>117 </td><td>21.6 </td><td>20.8 </td><td>942 </td><td>19.5 </td><td>122 </td><td>20.6 </td><td>19.5 </td></tr><tr><td>ACT/AGT </td><td>151 </td><td>22.2 </td><td>44 </td><td>24.8 </td><td>40.7 </td><td>139 </td><td>22.2 </td><td>57 </td><td>24.9 </td><td>44.5 </td></tr><tr><td>AGC/GCT </td><td>958 </td><td>19.7 </td><td>216 </td><td>20.9 </td><td>31.3 </td><td>999 </td><td>19.9 </td><td>264 </td><td>20.7 </td><td>33.2 </td></tr><tr><td>AGG/CCT </td><td>1540 </td><td>20.6 </td><td>424 </td><td>22.1 </td><td>35.5 </td><td>1557 </td><td>20.5 </td><td>415 </td><td>21.8 </td><td>31.7 </td></tr><tr><td>ATC/GAT </td><td>310 </td><td>22.2 </td><td>70 </td><td>22.7 </td><td>37.8 </td><td>303 </td><td>21.6 </td><td>74 </td><td>22.4 </td><td>37.0 </td></tr><tr><td>CCG/CGG </td><td>4734 </td><td>19.7 </td><td>999 </td><td>21.2 </td><td>28.0 </td><td>5009 </td><td>19.6 </td><td>1087 </td><td>20.8 </td><td>26.6 </td></tr></tbody></table></div><div content-id="DSM005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Motifs </th><th colspan="4">Indica </th><th>Rate of pm </th><th colspan="4">Japonica </th><th>Rate of pm </th></tr><tr><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th></tr><tr><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th></tr></thead><tbody><tr><td>A/T </td><td>57 467 </td><td>10.9 </td><td>21 393 </td><td>11.2 </td><td>53.5 </td><td>53 940 </td><td>10.8 </td><td>20 637 </td><td>11.1 </td><td>49.6 </td></tr><tr><td>C/G </td><td>6710 </td><td>13.8 </td><td>2849 </td><td>14.4 </td><td>68.6 </td><td>7923 </td><td>13.6 </td><td>3104 </td><td>14.1 </td><td>61.3 </td></tr><tr><td>AC/GT </td><td>1405 </td><td>18.2 </td><td>541 </td><td>19.6 </td><td>54.4 </td><td>1318 </td><td>18.1 </td><td>560 </td><td>19.4 </td><td>52.5 </td></tr><tr><td>AG/CT </td><td>8371 </td><td>20.1 </td><td>3220 </td><td>22.6 </td><td>59.8 </td><td>8664 </td><td>19.7 </td><td>3351 </td><td>22.5 </td><td>56.4 </td></tr><tr><td>AT/AT </td><td>7271 </td><td>34.5 </td><td>2147 </td><td>36.8 </td><td>56.6 </td><td>8041 </td><td>35.9 </td><td>2287 </td><td>38.1 </td><td>54.4 </td></tr><tr><td>CG/CG </td><td>668 </td><td>14.8 </td><td>59 </td><td>15.7 </td><td>15.6 </td><td>755 </td><td>14.7 </td><td>64 </td><td>15.5 </td><td>15.6 </td></tr><tr><td>AAC/GTT </td><td>186 </td><td>24.5 </td><td>45 </td><td>30.7 </td><td>37.8 </td><td>185 </td><td>22.7 </td><td>57 </td><td>25.2 </td><td>41.6 </td></tr><tr><td>AAG/CTT </td><td>892 </td><td>25.1 </td><td>304 </td><td>29.0 </td><td>46.5 </td><td>863 </td><td>26.0 </td><td>305 </td><td>30.1 </td><td>41.8 </td></tr><tr><td>AAT/ATT </td><td>683 </td><td>45.0 </td><td>248 </td><td>51.7 </td><td>55.6 </td><td>671 </td><td>49.5 </td><td>253 </td><td>59.5 </td><td>49.6 </td></tr><tr><td>ACC/GGT </td><td>603 </td><td>20.1 </td><td>111 </td><td>21.3 </td><td>23.8 </td><td>632 </td><td>20.1 </td><td>120 </td><td>21.7 </td><td>22.2 </td></tr><tr><td>ACG/CGT </td><td>832 </td><td>19.7 </td><td>117 </td><td>21.6 </td><td>20.8 </td><td>942 </td><td>19.5 </td><td>122 </td><td>20.6 </td><td>19.5 </td></tr><tr><td>ACT/AGT </td><td>151 </td><td>22.2 </td><td>44 </td><td>24.8 </td><td>40.7 </td><td>139 </td><td>22.2 </td><td>57 </td><td>24.9 </td><td>44.5 </td></tr><tr><td>AGC/GCT </td><td>958 </td><td>19.7 </td><td>216 </td><td>20.9 </td><td>31.3 </td><td>999 </td><td>19.9 </td><td>264 </td><td>20.7 </td><td>33.2 </td></tr><tr><td>AGG/CCT </td><td>1540 </td><td>20.6 </td><td>424 </td><td>22.1 </td><td>35.5 </td><td>1557 </td><td>20.5 </td><td>415 </td><td>21.8 </td><td>31.7 </td></tr><tr><td>ATC/GAT </td><td>310 </td><td>22.2 </td><td>70 </td><td>22.7 </td><td>37.8 </td><td>303 </td><td>21.6 </td><td>74 </td><td>22.4 </td><td>37.0 </td></tr><tr><td>CCG/CGG </td><td>4734 </td><td>19.7 </td><td>999 </td><td>21.2 </td><td>28.0 </td><td>5009 </td><td>19.6 </td><td>1087 </td><td>20.8 </td><td>26.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Labels are the same as in Table <a reveal-id="DSM005TB1" data-open="DSM005TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195930" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM005TB2" class="table-wrap-title" data-id="DSM005TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Polymorphism distribution of MS motifs in indica and japonica genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Motifs </th><th colspan="4">Indica </th><th>Rate of pm </th><th colspan="4">Japonica </th><th>Rate of pm </th></tr><tr><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th></tr><tr><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th></tr></thead><tbody><tr><td>A/T </td><td>57 467 </td><td>10.9 </td><td>21 393 </td><td>11.2 </td><td>53.5 </td><td>53 940 </td><td>10.8 </td><td>20 637 </td><td>11.1 </td><td>49.6 </td></tr><tr><td>C/G </td><td>6710 </td><td>13.8 </td><td>2849 </td><td>14.4 </td><td>68.6 </td><td>7923 </td><td>13.6 </td><td>3104 </td><td>14.1 </td><td>61.3 </td></tr><tr><td>AC/GT </td><td>1405 </td><td>18.2 </td><td>541 </td><td>19.6 </td><td>54.4 </td><td>1318 </td><td>18.1 </td><td>560 </td><td>19.4 </td><td>52.5 </td></tr><tr><td>AG/CT </td><td>8371 </td><td>20.1 </td><td>3220 </td><td>22.6 </td><td>59.8 </td><td>8664 </td><td>19.7 </td><td>3351 </td><td>22.5 </td><td>56.4 </td></tr><tr><td>AT/AT </td><td>7271 </td><td>34.5 </td><td>2147 </td><td>36.8 </td><td>56.6 </td><td>8041 </td><td>35.9 </td><td>2287 </td><td>38.1 </td><td>54.4 </td></tr><tr><td>CG/CG </td><td>668 </td><td>14.8 </td><td>59 </td><td>15.7 </td><td>15.6 </td><td>755 </td><td>14.7 </td><td>64 </td><td>15.5 </td><td>15.6 </td></tr><tr><td>AAC/GTT </td><td>186 </td><td>24.5 </td><td>45 </td><td>30.7 </td><td>37.8 </td><td>185 </td><td>22.7 </td><td>57 </td><td>25.2 </td><td>41.6 </td></tr><tr><td>AAG/CTT </td><td>892 </td><td>25.1 </td><td>304 </td><td>29.0 </td><td>46.5 </td><td>863 </td><td>26.0 </td><td>305 </td><td>30.1 </td><td>41.8 </td></tr><tr><td>AAT/ATT </td><td>683 </td><td>45.0 </td><td>248 </td><td>51.7 </td><td>55.6 </td><td>671 </td><td>49.5 </td><td>253 </td><td>59.5 </td><td>49.6 </td></tr><tr><td>ACC/GGT </td><td>603 </td><td>20.1 </td><td>111 </td><td>21.3 </td><td>23.8 </td><td>632 </td><td>20.1 </td><td>120 </td><td>21.7 </td><td>22.2 </td></tr><tr><td>ACG/CGT </td><td>832 </td><td>19.7 </td><td>117 </td><td>21.6 </td><td>20.8 </td><td>942 </td><td>19.5 </td><td>122 </td><td>20.6 </td><td>19.5 </td></tr><tr><td>ACT/AGT </td><td>151 </td><td>22.2 </td><td>44 </td><td>24.8 </td><td>40.7 </td><td>139 </td><td>22.2 </td><td>57 </td><td>24.9 </td><td>44.5 </td></tr><tr><td>AGC/GCT </td><td>958 </td><td>19.7 </td><td>216 </td><td>20.9 </td><td>31.3 </td><td>999 </td><td>19.9 </td><td>264 </td><td>20.7 </td><td>33.2 </td></tr><tr><td>AGG/CCT </td><td>1540 </td><td>20.6 </td><td>424 </td><td>22.1 </td><td>35.5 </td><td>1557 </td><td>20.5 </td><td>415 </td><td>21.8 </td><td>31.7 </td></tr><tr><td>ATC/GAT </td><td>310 </td><td>22.2 </td><td>70 </td><td>22.7 </td><td>37.8 </td><td>303 </td><td>21.6 </td><td>74 </td><td>22.4 </td><td>37.0 </td></tr><tr><td>CCG/CGG </td><td>4734 </td><td>19.7 </td><td>999 </td><td>21.2 </td><td>28.0 </td><td>5009 </td><td>19.6 </td><td>1087 </td><td>20.8 </td><td>26.6 </td></tr></tbody></table></div><div content-id="DSM005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Motifs </th><th colspan="4">Indica </th><th>Rate of pm </th><th colspan="4">Japonica </th><th>Rate of pm </th></tr><tr><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th><th colspan="2">All </th><th colspan="2">Wpm </th><th rowspan="2"> </th></tr><tr><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th><th>Count </th><th>Length </th></tr></thead><tbody><tr><td>A/T </td><td>57 467 </td><td>10.9 </td><td>21 393 </td><td>11.2 </td><td>53.5 </td><td>53 940 </td><td>10.8 </td><td>20 637 </td><td>11.1 </td><td>49.6 </td></tr><tr><td>C/G </td><td>6710 </td><td>13.8 </td><td>2849 </td><td>14.4 </td><td>68.6 </td><td>7923 </td><td>13.6 </td><td>3104 </td><td>14.1 </td><td>61.3 </td></tr><tr><td>AC/GT </td><td>1405 </td><td>18.2 </td><td>541 </td><td>19.6 </td><td>54.4 </td><td>1318 </td><td>18.1 </td><td>560 </td><td>19.4 </td><td>52.5 </td></tr><tr><td>AG/CT </td><td>8371 </td><td>20.1 </td><td>3220 </td><td>22.6 </td><td>59.8 </td><td>8664 </td><td>19.7 </td><td>3351 </td><td>22.5 </td><td>56.4 </td></tr><tr><td>AT/AT </td><td>7271 </td><td>34.5 </td><td>2147 </td><td>36.8 </td><td>56.6 </td><td>8041 </td><td>35.9 </td><td>2287 </td><td>38.1 </td><td>54.4 </td></tr><tr><td>CG/CG </td><td>668 </td><td>14.8 </td><td>59 </td><td>15.7 </td><td>15.6 </td><td>755 </td><td>14.7 </td><td>64 </td><td>15.5 </td><td>15.6 </td></tr><tr><td>AAC/GTT </td><td>186 </td><td>24.5 </td><td>45 </td><td>30.7 </td><td>37.8 </td><td>185 </td><td>22.7 </td><td>57 </td><td>25.2 </td><td>41.6 </td></tr><tr><td>AAG/CTT </td><td>892 </td><td>25.1 </td><td>304 </td><td>29.0 </td><td>46.5 </td><td>863 </td><td>26.0 </td><td>305 </td><td>30.1 </td><td>41.8 </td></tr><tr><td>AAT/ATT </td><td>683 </td><td>45.0 </td><td>248 </td><td>51.7 </td><td>55.6 </td><td>671 </td><td>49.5 </td><td>253 </td><td>59.5 </td><td>49.6 </td></tr><tr><td>ACC/GGT </td><td>603 </td><td>20.1 </td><td>111 </td><td>21.3 </td><td>23.8 </td><td>632 </td><td>20.1 </td><td>120 </td><td>21.7 </td><td>22.2 </td></tr><tr><td>ACG/CGT </td><td>832 </td><td>19.7 </td><td>117 </td><td>21.6 </td><td>20.8 </td><td>942 </td><td>19.5 </td><td>122 </td><td>20.6 </td><td>19.5 </td></tr><tr><td>ACT/AGT </td><td>151 </td><td>22.2 </td><td>44 </td><td>24.8 </td><td>40.7 </td><td>139 </td><td>22.2 </td><td>57 </td><td>24.9 </td><td>44.5 </td></tr><tr><td>AGC/GCT </td><td>958 </td><td>19.7 </td><td>216 </td><td>20.9 </td><td>31.3 </td><td>999 </td><td>19.9 </td><td>264 </td><td>20.7 </td><td>33.2 </td></tr><tr><td>AGG/CCT </td><td>1540 </td><td>20.6 </td><td>424 </td><td>22.1 </td><td>35.5 </td><td>1557 </td><td>20.5 </td><td>415 </td><td>21.8 </td><td>31.7 </td></tr><tr><td>ATC/GAT </td><td>310 </td><td>22.2 </td><td>70 </td><td>22.7 </td><td>37.8 </td><td>303 </td><td>21.6 </td><td>74 </td><td>22.4 </td><td>37.0 </td></tr><tr><td>CCG/CGG </td><td>4734 </td><td>19.7 </td><td>999 </td><td>21.2 </td><td>28.0 </td><td>5009 </td><td>19.6 </td><td>1087 </td><td>20.8 </td><td>26.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Labels are the same as in Table <a reveal-id="DSM005TB1" data-open="DSM005TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195930" target="_blank">View Large</a></div></div><p>We detected 35 691 polymorphic MS loci with unique flanking sequences in the indica genome and 35 779 in the japonica genome (Table <a reveal-id="DSM005TB3" data-open="DSM005TB3" class="link link-reveal link-table xref-fig">3</a>). If we took away 18 016 loci common for both, there were 53 454 subspecific MS loci left, and we were able to design primer pairs for 52 845 MSs (Table <a reveal-id="DSM005TB3" data-open="DSM005TB3" class="link link-reveal link-table xref-fig">3</a>), which gave rise to one polymorphic MS for every 7.1 and 7.2 kb in lengths in indica and japonica, respectively. Of these MSs, there were 36 374 MNRs, 8298 DNRs, 3878 TNRs, 802 TTRs, 683 PNRs, 487 HNRs, and 2323 complex repeats that comprised two or more repeat units (Table <a reveal-id="DSM005TB3" data-open="DSM005TB3" class="link link-reveal link-table xref-fig">3</a>). We also analyzed the length polymorphism of MSs in indica and japonica sequences (Fig. <a reveal-id="DSM005F2" data-open="DSM005F2" class="link link-reveal link-table xref-fig">2</a>), and found that approximately 38% of MSs have a length difference of 1 bp. Only about 8% of MSs have a length polymorphism of more than 10 bp, and the number of MSs with polymorphism length of 5 and 7 bp is relatively small (Fig. <a reveal-id="DSM005F2" data-open="DSM005F2" class="link link-reveal link-table xref-fig">2</a>), since the length polymorphism is related to the length of the repeat units or types.</p>                        <a id="87195936" scrollto-destination="87195936"></a>
<div id="DSM005F2" data-id="DSM005F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/m_dsm00502.jpeg?Expires=2147483647&Signature=pf3OwRsxyvLkGVupQJ0mSW5mDrESn2jqT~K0-wiFdapT0OupVrQDF~kBXMxE4WMDPeBIvCAnKSpRj9ZOOGSjKs5JIY6sMlRgwu8cewiZeGU~AG28Fcso8TRdtPYdYhZuZxX-TX02aE6IxVJokO7Io91MnP2aXqaFKRZEZGwm3dd2PILgr~j0qti~ypnuV1Qr8pVxiz1FLsVjDcquZObTDLOJ2LIEZkhn-~y~lEe50uQ1G6CiyFs9yjCPbb~fGebrgt~jD8J-3-~vTw~yWhMw2HK1jrIHBaNKCdw0mhHAQ-DSPd~4CBddsywxvBmbV7z0BMWT91lmkKFYU1-iwEj89Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distribution of polymorphic MSs in indica and japonica rice genomes." path-from-xml="dsm00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195936/dsm00502.jpeg" path-from-xml="dsm00502.jpeg" target="_blank">View large</a><a section="87195936" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/dsm00502.jpeg?Expires=2147483647&Signature=jZG2v5Gn-U3gZklEo7BQtOkyh3HD9GERsptN8WkWE8pOYnnKI5lxq18cD8ke9hIum4tPRDjFdMk8mcXkxs6kMB12X0zzeE6Km4NgIl~95LtvGhR2r2q8hP5L2DJqCayzD6vEJ84mxC5cfnCyNeWuG9qqR1td8qLTjVyLKlvs3nFZj43UpHyB6whV0VnnKOkfm4yC4haaBtnU-MRhtvLtkNKM3gAO6dni3juY~njxJhTHXAn5vDXw5nsI2N9c3hw7ekYPINg4YFUxlH2OGplClmWVVqqLz0WIibH3V~1OVu4o3LybJPK~MmqV-iq5gxuWnKV-Y8a1MHvF7Ae58T2eDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195936&ar=503258&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distribution of polymorphic MSs in <em>indica</em> and <em>japonica</em> rice genomes.</p></div></div><div content-id="DSM005F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/m_dsm00502.jpeg?Expires=2147483647&Signature=pf3OwRsxyvLkGVupQJ0mSW5mDrESn2jqT~K0-wiFdapT0OupVrQDF~kBXMxE4WMDPeBIvCAnKSpRj9ZOOGSjKs5JIY6sMlRgwu8cewiZeGU~AG28Fcso8TRdtPYdYhZuZxX-TX02aE6IxVJokO7Io91MnP2aXqaFKRZEZGwm3dd2PILgr~j0qti~ypnuV1Qr8pVxiz1FLsVjDcquZObTDLOJ2LIEZkhn-~y~lEe50uQ1G6CiyFs9yjCPbb~fGebrgt~jD8J-3-~vTw~yWhMw2HK1jrIHBaNKCdw0mhHAQ-DSPd~4CBddsywxvBmbV7z0BMWT91lmkKFYU1-iwEj89Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distribution of polymorphic MSs in indica and japonica rice genomes." path-from-xml="dsm00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195936/dsm00502.jpeg" path-from-xml="dsm00502.jpeg" target="_blank">View large</a><a section="87195936" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/dsm00502.jpeg?Expires=2147483647&Signature=jZG2v5Gn-U3gZklEo7BQtOkyh3HD9GERsptN8WkWE8pOYnnKI5lxq18cD8ke9hIum4tPRDjFdMk8mcXkxs6kMB12X0zzeE6Km4NgIl~95LtvGhR2r2q8hP5L2DJqCayzD6vEJ84mxC5cfnCyNeWuG9qqR1td8qLTjVyLKlvs3nFZj43UpHyB6whV0VnnKOkfm4yC4haaBtnU-MRhtvLtkNKM3gAO6dni3juY~njxJhTHXAn5vDXw5nsI2N9c3hw7ekYPINg4YFUxlH2OGplClmWVVqqLz0WIibH3V~1OVu4o3LybJPK~MmqV-iq5gxuWnKV-Y8a1MHvF7Ae58T2eDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195936&ar=503258&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distribution of polymorphic MSs in <em>indica</em> and <em>japonica</em> rice genomes.</p></div></div>                        <a id="87195939" scrollto-destination="87195939"></a>
<div content-id="DSM005TB3" class="table-modal"><div class="table-wrap"><div id="DSM005TB3" class="table-wrap-title" data-id="DSM005TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of polymorphic MS loci</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Complex </th><th>Total </th></tr></thead><tbody><tr><td>(1) </td><td>Indica MS </td><td>24 242 </td><td>5967 </td><td>2578 </td><td>477 </td><td>394 </td><td>281 </td><td>1752 </td><td>35 691 </td></tr><tr><td>(2) </td><td>Japonica MS </td><td>23 741 </td><td>6262 </td><td>2754 </td><td>562 </td><td>451 </td><td>275 </td><td>1734 </td><td>35 779 </td></tr><tr><td>(3) </td><td>Common MS </td><td>11 378 </td><td>3658 </td><td>1413 </td><td>231 </td><td>156 </td><td>53 </td><td>1127 </td><td>18 016 </td></tr><tr><td>(4) </td><td>Theoretical estimates of polymorphic MS </td><td>36 605 </td><td>8571 </td><td>3919 </td><td>808 </td><td>689 </td><td>503 </td><td>2359 </td><td>53 454 </td></tr><tr><td>(5) </td><td>Observed values of polymorphic MS </td><td>36 555 </td><td>8546 </td><td>3942 </td><td>803 </td><td>688 </td><td>503 </td><td>2417 </td><td>53 454 </td></tr><tr><td>(6) </td><td>MS with primers </td><td>36 374 </td><td>8298 </td><td>3878 </td><td>802 </td><td>683 </td><td>487 </td><td>2323 </td><td>52 845 </td></tr></tbody></table></div><div content-id="DSM005TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Complex </th><th>Total </th></tr></thead><tbody><tr><td>(1) </td><td>Indica MS </td><td>24 242 </td><td>5967 </td><td>2578 </td><td>477 </td><td>394 </td><td>281 </td><td>1752 </td><td>35 691 </td></tr><tr><td>(2) </td><td>Japonica MS </td><td>23 741 </td><td>6262 </td><td>2754 </td><td>562 </td><td>451 </td><td>275 </td><td>1734 </td><td>35 779 </td></tr><tr><td>(3) </td><td>Common MS </td><td>11 378 </td><td>3658 </td><td>1413 </td><td>231 </td><td>156 </td><td>53 </td><td>1127 </td><td>18 016 </td></tr><tr><td>(4) </td><td>Theoretical estimates of polymorphic MS </td><td>36 605 </td><td>8571 </td><td>3919 </td><td>808 </td><td>689 </td><td>503 </td><td>2359 </td><td>53 454 </td></tr><tr><td>(5) </td><td>Observed values of polymorphic MS </td><td>36 555 </td><td>8546 </td><td>3942 </td><td>803 </td><td>688 </td><td>503 </td><td>2417 </td><td>53 454 </td></tr><tr><td>(6) </td><td>MS with primers </td><td>36 374 </td><td>8298 </td><td>3878 </td><td>802 </td><td>683 </td><td>487 </td><td>2323 </td><td>52 845 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>(1) The identified polymorphic MS loci in indica; (2) The identified polymorphic MS loci in japonica; (3) The common polymorphic MS loci that are found to be the same in both indica and japonica; (4) The theoretical estimates of all polymorphic MS loci in rice that can be estimated according to the following formula polymorphic MS = (1) + (2) − (3); (5) Observed values of all polymorphic MS loci in rice. The data discrepancies between (4) and (5) are due in part to the fact that some MS in indica belong to MNR or DNR, also some MS in japonica can be changed into the Complex kinds; (6) MS with primers: polymorphic MS loci with primers that were designed successfully with Primer3 (<a class="link link-uri" href="http://www.genome.wi.mit.edu/cgi-bin/primer/primer3">http://www.genome.wi.mit.edu/cgi-bin/primer/primer3</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195939" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM005TB3" class="table-wrap-title" data-id="DSM005TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of polymorphic MS loci</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Complex </th><th>Total </th></tr></thead><tbody><tr><td>(1) </td><td>Indica MS </td><td>24 242 </td><td>5967 </td><td>2578 </td><td>477 </td><td>394 </td><td>281 </td><td>1752 </td><td>35 691 </td></tr><tr><td>(2) </td><td>Japonica MS </td><td>23 741 </td><td>6262 </td><td>2754 </td><td>562 </td><td>451 </td><td>275 </td><td>1734 </td><td>35 779 </td></tr><tr><td>(3) </td><td>Common MS </td><td>11 378 </td><td>3658 </td><td>1413 </td><td>231 </td><td>156 </td><td>53 </td><td>1127 </td><td>18 016 </td></tr><tr><td>(4) </td><td>Theoretical estimates of polymorphic MS </td><td>36 605 </td><td>8571 </td><td>3919 </td><td>808 </td><td>689 </td><td>503 </td><td>2359 </td><td>53 454 </td></tr><tr><td>(5) </td><td>Observed values of polymorphic MS </td><td>36 555 </td><td>8546 </td><td>3942 </td><td>803 </td><td>688 </td><td>503 </td><td>2417 </td><td>53 454 </td></tr><tr><td>(6) </td><td>MS with primers </td><td>36 374 </td><td>8298 </td><td>3878 </td><td>802 </td><td>683 </td><td>487 </td><td>2323 </td><td>52 845 </td></tr></tbody></table></div><div content-id="DSM005TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Complex </th><th>Total </th></tr></thead><tbody><tr><td>(1) </td><td>Indica MS </td><td>24 242 </td><td>5967 </td><td>2578 </td><td>477 </td><td>394 </td><td>281 </td><td>1752 </td><td>35 691 </td></tr><tr><td>(2) </td><td>Japonica MS </td><td>23 741 </td><td>6262 </td><td>2754 </td><td>562 </td><td>451 </td><td>275 </td><td>1734 </td><td>35 779 </td></tr><tr><td>(3) </td><td>Common MS </td><td>11 378 </td><td>3658 </td><td>1413 </td><td>231 </td><td>156 </td><td>53 </td><td>1127 </td><td>18 016 </td></tr><tr><td>(4) </td><td>Theoretical estimates of polymorphic MS </td><td>36 605 </td><td>8571 </td><td>3919 </td><td>808 </td><td>689 </td><td>503 </td><td>2359 </td><td>53 454 </td></tr><tr><td>(5) </td><td>Observed values of polymorphic MS </td><td>36 555 </td><td>8546 </td><td>3942 </td><td>803 </td><td>688 </td><td>503 </td><td>2417 </td><td>53 454 </td></tr><tr><td>(6) </td><td>MS with primers </td><td>36 374 </td><td>8298 </td><td>3878 </td><td>802 </td><td>683 </td><td>487 </td><td>2323 </td><td>52 845 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>(1) The identified polymorphic MS loci in indica; (2) The identified polymorphic MS loci in japonica; (3) The common polymorphic MS loci that are found to be the same in both indica and japonica; (4) The theoretical estimates of all polymorphic MS loci in rice that can be estimated according to the following formula polymorphic MS = (1) + (2) − (3); (5) Observed values of all polymorphic MS loci in rice. The data discrepancies between (4) and (5) are due in part to the fact that some MS in indica belong to MNR or DNR, also some MS in japonica can be changed into the Complex kinds; (6) MS with primers: polymorphic MS loci with primers that were designed successfully with Primer3 (<a class="link link-uri" href="http://www.genome.wi.mit.edu/cgi-bin/primer/primer3">http://www.genome.wi.mit.edu/cgi-bin/primer/primer3</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195939" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195941 id="87195941" class="section-title" data-legacy-id=s3e>3.5. Development of the polymorphic MS database for rice subspecies</h3>
<p>A total of 18 828 SSR markers have been recently reported based on sequence analysis of the japonica cv. Nipponbare genome,<sup><a reveal-id="DSM005C15" data-open="DSM005C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and 2240 of the total were experimentally confirmed.<sup><a reveal-id="DSM005C13" data-open="DSM005C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Since some of the markers between the two genomes may not be polymorphic and their physical positions may not be unique, we may not be able to include all of them in our database. Although some MS polymorphisms in chromosome 4 between the indica and japonica have been reported,<sup><a reveal-id="DSM005C16" data-open="DSM005C16" class="link link-ref link-reveal xref-bibr">16</a></sup> it was limited to a single chromosome without eliminating multiple-hitting copies. In addition, previous works have mainly focused on Class I MSs, such as DNRs, TNRs, and TTRs, whereas Class II MSs, including MNRs, PNRs, and HNRs have not drawn enough attention for marker development.</p><p>By comparing MS sequences between the sequenced indica and japonica genomes, we developed a new database for 52 845 (Table <a reveal-id="DSM005TB3" data-open="DSM005TB3" class="link link-reveal link-table xref-fig">3</a>) polymorphic MS loci with specific flanking sequences, including polymorphic ample Class II MSs. To avoid confusion in MS naming convention used by the rice community, we integrated 18 828 MSs published by IRGSP<sup><a reveal-id="DSM005C15" data-open="DSM005C15" class="link link-ref link-reveal xref-bibr">15</a></sup> into our collection, and the same loci are cross-referenced (6294 MSs from IRGSP). However, 6641 MSs were excluded because they have multiple copies in the rice genome. Some 3988 MSs were found lack of polymorphism between the indica and japonica genomes, and 1905 MSs were not mapped due to the absence of reciprocal polymorphic sequences between the two genomes. Only 4509 Class I polymorphic MS loci were detected in the indica genome (Table <a reveal-id="DSM005TB4" data-open="DSM005TB4" class="link link-reveal link-table xref-fig">4</a>), 1775 (6284 − 4509) loci were missed because the length of their repeat motif was less than 20 bp and do not satisfy the criteria of Class I in indica, although they were Class I loci in japonica. These 1775 shorter MS alleles in indica may not represent any technical or biological anomalies but simply happened by chance since MSs are in general short in length. In addition, we also mapped the genetic markers consisting of mostly RFLPs from RGP (<a class="link link-uri" href="http://rgp.dna.affrc.go.jp">http://rgp.dna.affrc.go.jp</a>) on the indica (from <a class="link link-uri" href="http://rise.genomics.org.cn">http://rise.genomics.org.cn</a>), IRGSP, and TIGR pseudomolecules <em>in silico</em>. Although there is a misunderstanding about the BGI's sequence assemblies, the indica pseudomolecules are believed as useful as the BAC-to-BAC assembly when evaluated carefully according to scientific rationales.<sup><a reveal-id="DSM005C22" data-open="DSM005C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                        <a id="87195947" scrollto-destination="87195947"></a>
<div content-id="DSM005TB4" class="table-modal"><div class="table-wrap"><div id="DSM005TB4" class="table-wrap-title" data-id="DSM005TB4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Comparative analysis of MSs identified in indica with different cutoffs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Class I </th><th colspan="6">Class II </th></tr><tr><th> </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th></tr><tr><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th></tr></thead><tbody><tr><td>MNRs </td><td>789 </td><td>641 </td><td>81.2 </td><td>417 </td><td>52.9 </td><td>65.1 </td><td>61 074 </td><td>45 996 </td><td>75.3 </td><td>23 324 </td><td>38.2 </td><td>50.7 </td></tr><tr><td>DNRs </td><td>6753 </td><td>4071 </td><td>60.3 </td><td>2747 </td><td>40.7 </td><td>67.5 </td><td>16 947 </td><td>9654 </td><td>57.0 </td><td>3825 </td><td>22.6 </td><td>39.6 </td></tr><tr><td>TNRs </td><td>3763 </td><td>2770 </td><td>73.6 </td><td>1345 </td><td>35.7 </td><td>48.6 </td><td>54 824 </td><td>33 480 </td><td>61.1 </td><td>2638 </td><td>4.8 </td><td>7.9 </td></tr></tbody></table></div><div content-id="DSM005TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Class I </th><th colspan="6">Class II </th></tr><tr><th> </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th></tr><tr><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th></tr></thead><tbody><tr><td>MNRs </td><td>789 </td><td>641 </td><td>81.2 </td><td>417 </td><td>52.9 </td><td>65.1 </td><td>61 074 </td><td>45 996 </td><td>75.3 </td><td>23 324 </td><td>38.2 </td><td>50.7 </td></tr><tr><td>DNRs </td><td>6753 </td><td>4071 </td><td>60.3 </td><td>2747 </td><td>40.7 </td><td>67.5 </td><td>16 947 </td><td>9654 </td><td>57.0 </td><td>3825 </td><td>22.6 </td><td>39.6 </td></tr><tr><td>TNRs </td><td>3763 </td><td>2770 </td><td>73.6 </td><td>1345 </td><td>35.7 </td><td>48.6 </td><td>54 824 </td><td>33 480 </td><td>61.1 </td><td>2638 </td><td>4.8 </td><td>7.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Class I: MNRs with 20, DNRs with 10, and TNRs with seven repeat units; Class II: MNRs with 10 repeat units, DNRs with seven, and TNRs with six repeat units. The rest of the labels are the same as in Table <a reveal-id="DSM005TB1" data-open="DSM005TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195947" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM005TB4" class="table-wrap-title" data-id="DSM005TB4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Comparative analysis of MSs identified in indica with different cutoffs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Class I </th><th colspan="6">Class II </th></tr><tr><th> </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th></tr><tr><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th></tr></thead><tbody><tr><td>MNRs </td><td>789 </td><td>641 </td><td>81.2 </td><td>417 </td><td>52.9 </td><td>65.1 </td><td>61 074 </td><td>45 996 </td><td>75.3 </td><td>23 324 </td><td>38.2 </td><td>50.7 </td></tr><tr><td>DNRs </td><td>6753 </td><td>4071 </td><td>60.3 </td><td>2747 </td><td>40.7 </td><td>67.5 </td><td>16 947 </td><td>9654 </td><td>57.0 </td><td>3825 </td><td>22.6 </td><td>39.6 </td></tr><tr><td>TNRs </td><td>3763 </td><td>2770 </td><td>73.6 </td><td>1345 </td><td>35.7 </td><td>48.6 </td><td>54 824 </td><td>33 480 </td><td>61.1 </td><td>2638 </td><td>4.8 </td><td>7.9 </td></tr></tbody></table></div><div content-id="DSM005TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Class I </th><th colspan="6">Class II </th></tr><tr><th> </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th><th>All count </th><th colspan="2">Wsfs </th><th colspan="2">Wpm </th><th>Rate of pm </th></tr><tr><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th><th> </th><th>Count </th><th>Per<sup>f</sup> </th><th>Count </th><th>Per<sup>g</sup> </th><th> </th></tr></thead><tbody><tr><td>MNRs </td><td>789 </td><td>641 </td><td>81.2 </td><td>417 </td><td>52.9 </td><td>65.1 </td><td>61 074 </td><td>45 996 </td><td>75.3 </td><td>23 324 </td><td>38.2 </td><td>50.7 </td></tr><tr><td>DNRs </td><td>6753 </td><td>4071 </td><td>60.3 </td><td>2747 </td><td>40.7 </td><td>67.5 </td><td>16 947 </td><td>9654 </td><td>57.0 </td><td>3825 </td><td>22.6 </td><td>39.6 </td></tr><tr><td>TNRs </td><td>3763 </td><td>2770 </td><td>73.6 </td><td>1345 </td><td>35.7 </td><td>48.6 </td><td>54 824 </td><td>33 480 </td><td>61.1 </td><td>2638 </td><td>4.8 </td><td>7.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Class I: MNRs with 20, DNRs with 10, and TNRs with seven repeat units; Class II: MNRs with 10 repeat units, DNRs with seven, and TNRs with six repeat units. The rest of the labels are the same as in Table <a reveal-id="DSM005TB1" data-open="DSM005TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195947" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195950 id="87195950" class="section-title" data-legacy-id=s3f>3.6. Classes of complex MS, such as MNRs, PNRs, and HNRs, could be used as genetic markers in the future</h3>
<p>At present, only some of the MSs, such as DNRs, TNRs, and TTRs, are used as genetic markers for rice, and other types have been largely ignored, perhaps due to the lack of adequate experimental methods, such as in the case of MNRs, although they are accounted for more than 60% of all MSs and approximately half of them are believed polymorphic. In this report, we validated a few MNRs (such as P005 and P010 in Fig. <a reveal-id="DSM005F3" data-open="DSM005F3" class="link link-reveal link-table xref-fig">3</a>), proving their usefulness, and a recent report in typing method<sup><a reveal-id="DSM005C23" data-open="DSM005C23" class="link link-ref link-reveal xref-bibr">23</a></sup> also suggested a new opportunity for the MNR application as genetic markers for rice. Although PNRs and HNRs have been previously used as molecular markers in sorghum and cotton,<sup><a reveal-id="DSM005C24" data-open="DSM005C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSM005C25" data-open="DSM005C25" class="link link-ref link-reveal xref-bibr">25</a></sup> they have not been developed for rice. We found many polymorphic PNRs and HNRs with polymorphisms in the rice genome, and verified some of them experimentally, including eight PNRs, five HNRs, and four complex MSs. We believe that they should be used as genetic markers in rice in the future. It is important to note that polymorphisms found between 9311 and Nipponbare may not be generalized to other indica and japonica genomes, and additional sequence and genotyping information from diverse cultivars will broaden our understanding of SSR polymorphism in <em>O. sativa.</em></p>                        <a id="87195954" scrollto-destination="87195954"></a>
<div id="DSM005F3" data-id="DSM005F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/m_dsm00503.jpeg?Expires=2147483647&Signature=ONqXfQBpx02IjOouhEMVAHIuX7NXulP-PyTunS-d4irR8p-Da41RB5lfcxtxhFNCzUKqTv3~aNSmYbdZmNXfnpEnffuRBo1RcBuxYRSz00pJA210oJ80rh8nV4PdBviM-RGYKxAojrH0TMTkU2fXqgRoPAkuual~0Ujj5-yloFyFZv99X2~k0VDdxrDFEAziErP5WoLYDyPxsAQNr295Q0~5hb2imwU0OOF394qORgSLHmqVMpq7lnIZpapmki50y1iQe616ak-VrgV4RS4rylAI~1k6uvfBu-F~HzTl~Lbvefp86obKA3s-Lei~MJVBZ3PahSlRj38yh7zkeHTEMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation (PCR) of polymorphic MSs. Left lanes show PCR products from indica cv. 9311, and right lanes are samples from japonica cv. Nipponbare. MS involved MNR (P005, P010), DNR (P041, P031, P050, P043, P009, P039, P026, P040, P025, P024, P027), TNR (P055, P058, P062), TTR (P077), and other complex MS (P008, P095, P097, P100). Four PCR products for 9311 (P025, P026, P058, P062) and two for Nipponbare (P025, P058) showed complex band patterns and seven products (P008, P010, P025, P050, P055, P058, P062) appeared polymorphic." path-from-xml="dsm00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195954/dsm00503.jpeg" path-from-xml="dsm00503.jpeg" target="_blank">View large</a><a section="87195954" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/dsm00503.jpeg?Expires=2147483647&Signature=oJib0W7M5xtxDlYwQNA8EvG7UZbsaIfPaT3wg5tsoz2YHmwvoSJ51C8nKzOkZtwqvYOW~t8o-PGH2LK5uQMqaIEFMH5ihgI-VcNFCdx9~atmO65DqjmL7QYo4d5Et-mSoxq9yKxXbUPprNBzYPphVMF5XM8fhdoOAzBaW~KlgzvTbZFt~eCR0oB63m1fpZmfMNgRkEiDhH2qsR01IUT8tul06Jec1WeAzpqAHSnc8M9ltnSCueTPNcEEvVHuOufM3xXeAVM1OLdsI3bmHGtpqNd-K7~hdaeOUTqF2Jp3yXpbeke1vc9gas4sJEf4uqk1pPIcxmcmhjheZHfPCJ0tZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195954&ar=503258&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation (PCR) of polymorphic MSs. Left lanes show PCR products from indica cv. 9311, and right lanes are samples from japonica cv. Nipponbare. MS involved MNR (P005, P010), DNR (P041, P031, P050, P043, P009, P039, P026, P040, P025, P024, P027), TNR (P055, P058, P062), TTR (P077), and other complex MS (P008, P095, P097, P100). Four PCR products for 9311 (P025, P026, P058, P062) and two for Nipponbare (P025, P058) showed complex band patterns and seven products (P008, P010, P025, P050, P055, P058, P062) appeared polymorphic.</p></div></div><div content-id="DSM005F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/m_dsm00503.jpeg?Expires=2147483647&Signature=ONqXfQBpx02IjOouhEMVAHIuX7NXulP-PyTunS-d4irR8p-Da41RB5lfcxtxhFNCzUKqTv3~aNSmYbdZmNXfnpEnffuRBo1RcBuxYRSz00pJA210oJ80rh8nV4PdBviM-RGYKxAojrH0TMTkU2fXqgRoPAkuual~0Ujj5-yloFyFZv99X2~k0VDdxrDFEAziErP5WoLYDyPxsAQNr295Q0~5hb2imwU0OOF394qORgSLHmqVMpq7lnIZpapmki50y1iQe616ak-VrgV4RS4rylAI~1k6uvfBu-F~HzTl~Lbvefp86obKA3s-Lei~MJVBZ3PahSlRj38yh7zkeHTEMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation (PCR) of polymorphic MSs. Left lanes show PCR products from indica cv. 9311, and right lanes are samples from japonica cv. Nipponbare. MS involved MNR (P005, P010), DNR (P041, P031, P050, P043, P009, P039, P026, P040, P025, P024, P027), TNR (P055, P058, P062), TTR (P077), and other complex MS (P008, P095, P097, P100). Four PCR products for 9311 (P025, P026, P058, P062) and two for Nipponbare (P025, P058) showed complex band patterns and seven products (P008, P010, P025, P050, P055, P058, P062) appeared polymorphic." path-from-xml="dsm00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195954/dsm00503.jpeg" path-from-xml="dsm00503.jpeg" target="_blank">View large</a><a section="87195954" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/dsm00503.jpeg?Expires=2147483647&Signature=oJib0W7M5xtxDlYwQNA8EvG7UZbsaIfPaT3wg5tsoz2YHmwvoSJ51C8nKzOkZtwqvYOW~t8o-PGH2LK5uQMqaIEFMH5ihgI-VcNFCdx9~atmO65DqjmL7QYo4d5Et-mSoxq9yKxXbUPprNBzYPphVMF5XM8fhdoOAzBaW~KlgzvTbZFt~eCR0oB63m1fpZmfMNgRkEiDhH2qsR01IUT8tul06Jec1WeAzpqAHSnc8M9ltnSCueTPNcEEvVHuOufM3xXeAVM1OLdsI3bmHGtpqNd-K7~hdaeOUTqF2Jp3yXpbeke1vc9gas4sJEf4uqk1pPIcxmcmhjheZHfPCJ0tZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195954&ar=503258&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation (PCR) of polymorphic MSs. Left lanes show PCR products from indica cv. 9311, and right lanes are samples from japonica cv. Nipponbare. MS involved MNR (P005, P010), DNR (P041, P031, P050, P043, P009, P039, P026, P040, P025, P024, P027), TNR (P055, P058, P062), TTR (P077), and other complex MS (P008, P095, P097, P100). Four PCR products for 9311 (P025, P026, P058, P062) and two for Nipponbare (P025, P058) showed complex band patterns and seven products (P008, P010, P025, P050, P055, P058, P062) appeared polymorphic.</p></div></div><p>Comparison of MSs identified in indica were made with different cut-off values. MS polymorphisms have two basic parameters, types of the repeat units and numbers of units. By adjusting cut-off values for the number of repeat units toward a lower boundary, we discovered more polymorphic MS loci that are both highly polymorphic and can be uniquely identified in the genomes. We did try to define Class 1 MS by using a standard protocol, but only detected 417 MNRs, 2747 DNRs, and 1345 TNRs in the indica genome assembly. In this report, we found that when the MNR length reaches 10 bp, the polymorphism numbers increases up to 23 324 for MNR with a length of 10 bp, 3825 for DNR with a length of 7 bp, and 2638 for TNR with a length of 6 bp (Table <a reveal-id="DSM005TB3" data-open="DSM005TB3" class="link link-reveal link-table xref-fig">3</a>). It was evident that polymorphism counts of most Class II MNRs, DNRs, and TNRs are far higher than those of many Class I MS; the result suggests that different numbers of repeat units should be used for screening different MS types as genetic markers, such as 10 for MNRs, 7 for DNRs, 7 for TNRs, 6 for TTRs, 5 for PNRs, and 5 for HNRs.</p><p>Most MS repeats have unique flanking sequences (62.1% for indica and 73.1% for japonica in our study), and MSs may be preferentially associated with non-repetitive regions in rice genome as reported by Morgante et al.<sup><a reveal-id="DSM005C3" data-open="DSM005C3" class="link link-ref link-reveal xref-bibr">3</a></sup> except for AT/TA and AAT/ATT repeats.<sup><a reveal-id="DSM005C3" data-open="DSM005C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSM005C10" data-open="DSM005C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSM005C11" data-open="DSM005C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Furthermore, the fraction of MSs with unique flanking sequences in the indica is about 10% lower than that in the japonica, suggesting that there may be more repetitive sequences assembled in the indica genome when compared with the japonica genome. It was reported previously that MS polymorphism is positively correlated with their length,<sup><a reveal-id="DSM005C4" data-open="DSM005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSM005C12" data-open="DSM005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and the distribution of GC-rich DNRs and TNRs have a bias toward exons where polymorphisms occur scarcely.<sup><a reveal-id="DSM005C3" data-open="DSM005C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSM005C11" data-open="DSM005C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSM005C17" data-open="DSM005C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSM005C21" data-open="DSM005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> We also found in our current study that the degree of the polymorphisms in MSs depends on their genomic locations and the repeat types. For instance, higher polymorphism rates were found within intronic and UTR regions, whereas the lowest within coding regions. The relationship between GC content and MS polymorphism is unclear. For instance, A/T-containing MNRs have a slightly lower polymorphism rate than the G/C containing repeats. The opposite seems true in DNRs, where AG/CG repeats have higher polymorphism rates of 52–60%, than those in CG/GC repeats (15–16%). Similarly, CCG/CGG repeats contributing almost half of the total TNRs have a polymorphism rate of less than 30%, whereas AAT/ATT AAG/CTT and ACT/AGT have polymorphism rates of greater than 40%. Our results are in agreement to previous observations.<sup><a reveal-id="DSM005C4" data-open="DSM005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSM005C12" data-open="DSM005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The suggested reason was that GC-rich regions are relatively stable and resulted in less replication slippage.<sup><a reveal-id="DSM005C26" data-open="DSM005C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=87195957 id="87195957" class="section-title" data-legacy-id=s3g>3.7. Experimental validation of 100 polymorphic MS loci</h3>
<p>To validate our results, we chose 100 polymorphic MSs randomly for validation, using genomic DNA from indica cv. 9311 and japonica cv. Nipponbare as PCR templates. The PCR products were sequenced, and the result is summarized in Table <a reveal-id="DSM005TB5" data-open="DSM005TB5" class="link link-reveal link-table xref-fig">5</a> (other detailed information was summarized in Supplementary Table 1). We have 98 polymorphic MSs PCR-amplified in the two cultivars, and four of them worked for 9311 (P025, P026, P058, and P062 in Fig. <a reveal-id="DSM005F3" data-open="DSM005F3" class="link link-reveal link-table xref-fig">3</a>) and two for Nipponbare (P025 and P058), showing complex banding patterns that were perhaps due to lack of sequence specificity in the genomic DNA sample. Sequencing results confirmed 91 PCR products to be polymorphic, and seven (P008, P010, P025, P050, P055, P058, and P062) of the remaining PCR products that failed in sequencing appeared polymorphic based on the existence of multiple DNA bands on agarose gels. Only one (P055) showed unexpected products with a length less than 150 bp. In addition, the sequences amplified with six primer pairs (P060, P016, P037, P087, P020, and P030 in Supplementary Table 1) appeared inconsistent with the expected genomic sequences. This could be explained by the fact that the low-complexity sequences were difficult to be sequenced and relatively error-prone. We concluded that our new MS collection contains at least 90% valid polymorphic MS markers.</p>                        <a id="87195959" scrollto-destination="87195959"></a>
<div content-id="DSM005TB5" class="table-modal"><div class="table-wrap"><div id="DSM005TB5" class="table-wrap-title" data-id="DSM005TB5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Summary of PCR amplification and sequencing in indica and japonica genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">MS type </th><th rowspan="2">Total No. </th><th colspan="2">9311 </th><th colspan="2">Nipponbare </th><th rowspan="2">No. both sequenced </th></tr><tr><th>No. of amplified </th><th>No. of sequenced </th><th>No. of amplified </th><th>No. of sequenced </th></tr></thead><tbody><tr><td>MNRs </td><td>12 </td><td>12 </td><td>10 </td><td>12 </td><td>11 </td><td>10 </td></tr><tr><td>DNRs </td><td>37 </td><td>35 (1) </td><td>34 </td><td>36 (1) </td><td>36 </td><td>34 </td></tr><tr><td>TNRs </td><td>23 </td><td>23 (2) </td><td>22 </td><td>23 (1) </td><td>19 </td><td>19 </td></tr><tr><td>TTRs </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td>PNRs </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>HNRs </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>Complex </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>Total </td><td>100 </td><td>98 </td><td>94 </td><td>99 </td><td>94 </td><td>91 </td></tr></tbody></table></div><div content-id="DSM005TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">MS type </th><th rowspan="2">Total No. </th><th colspan="2">9311 </th><th colspan="2">Nipponbare </th><th rowspan="2">No. both sequenced </th></tr><tr><th>No. of amplified </th><th>No. of sequenced </th><th>No. of amplified </th><th>No. of sequenced </th></tr></thead><tbody><tr><td>MNRs </td><td>12 </td><td>12 </td><td>10 </td><td>12 </td><td>11 </td><td>10 </td></tr><tr><td>DNRs </td><td>37 </td><td>35 (1) </td><td>34 </td><td>36 (1) </td><td>36 </td><td>34 </td></tr><tr><td>TNRs </td><td>23 </td><td>23 (2) </td><td>22 </td><td>23 (1) </td><td>19 </td><td>19 </td></tr><tr><td>TTRs </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td>PNRs </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>HNRs </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>Complex </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>Total </td><td>100 </td><td>98 </td><td>94 </td><td>99 </td><td>94 </td><td>91 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Numbers in parentheses are the number of products with complex band pattern.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195959" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM005TB5" class="table-wrap-title" data-id="DSM005TB5"><div class="label -label"><strong>Table 5</strong></div><div class="caption"><p>Summary of PCR amplification and sequencing in indica and japonica genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">MS type </th><th rowspan="2">Total No. </th><th colspan="2">9311 </th><th colspan="2">Nipponbare </th><th rowspan="2">No. both sequenced </th></tr><tr><th>No. of amplified </th><th>No. of sequenced </th><th>No. of amplified </th><th>No. of sequenced </th></tr></thead><tbody><tr><td>MNRs </td><td>12 </td><td>12 </td><td>10 </td><td>12 </td><td>11 </td><td>10 </td></tr><tr><td>DNRs </td><td>37 </td><td>35 (1) </td><td>34 </td><td>36 (1) </td><td>36 </td><td>34 </td></tr><tr><td>TNRs </td><td>23 </td><td>23 (2) </td><td>22 </td><td>23 (1) </td><td>19 </td><td>19 </td></tr><tr><td>TTRs </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td>PNRs </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>HNRs </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>Complex </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>Total </td><td>100 </td><td>98 </td><td>94 </td><td>99 </td><td>94 </td><td>91 </td></tr></tbody></table></div><div content-id="DSM005TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">MS type </th><th rowspan="2">Total No. </th><th colspan="2">9311 </th><th colspan="2">Nipponbare </th><th rowspan="2">No. both sequenced </th></tr><tr><th>No. of amplified </th><th>No. of sequenced </th><th>No. of amplified </th><th>No. of sequenced </th></tr></thead><tbody><tr><td>MNRs </td><td>12 </td><td>12 </td><td>10 </td><td>12 </td><td>11 </td><td>10 </td></tr><tr><td>DNRs </td><td>37 </td><td>35 (1) </td><td>34 </td><td>36 (1) </td><td>36 </td><td>34 </td></tr><tr><td>TNRs </td><td>23 </td><td>23 (2) </td><td>22 </td><td>23 (1) </td><td>19 </td><td>19 </td></tr><tr><td>TTRs </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td>PNRs </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>HNRs </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>Complex </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td><td>8 </td></tr><tr><td>Total </td><td>100 </td><td>98 </td><td>94 </td><td>99 </td><td>94 </td><td>91 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Numbers in parentheses are the number of products with complex band pattern.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195959" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87195960 id="87195960" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was supported by National Natural Science Foundation of China (No. 30571146), the Key Research Project of Zhejiang Province (No. 2003C22007), and Rice Project of Zhejiang Province (04-06).</p>                            <h2 scrollto-destination=87195962 id="87195962" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM005C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gur-Arie</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cohen</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Eitan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shelef</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hallerman</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Kashi</div> <div class="given-names">Y</div></div>. </div><div class="article-title">Simple sequence repeats in <em>Escherichia coli</em>: abundance, distribution, composition, and polymorphism</div>, <div class="source">Genome Res</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">62</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20in%20Escherichia%20coli%3A%20abundance%2C%20distribution%2C%20composition%2C%20and%20polymorphism&amp;author=R.%20Gur-Arie&amp;author=C.%20J.%20Cohen&amp;author=Y.%20Eitan&amp;author=L.%20Shelef&amp;author=E.%20M.%20Hallerman&amp;author=Y%20Kashi&amp;publication_year=2000&amp;journal=Genome%20Res&amp;volume=10&amp;pages=62-71" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10645951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elgar</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">M. S.</div></div>,  <div class="name"><div class="surname">Meek</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Generation and analysis of 25 Mb of genomic DNA from the pufferfish <em>Fugu rubripes</em> by sequence scanning</div>, <div class="source">Genome Res</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">960</div>-<div class="lpage">971</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20and%20analysis%20of%2025%20Mb%20of%20genomic%20DNA%20from%20the%20pufferfish%20Fugu%20rubripes%20by%20sequence%20scanning&amp;author=G.%20Elgar&amp;author=M.%20S.%20Clark&amp;author=S.%20Meek&amp;publication_year=1999&amp;journal=Genome%20Res&amp;volume=9&amp;pages=960-971" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.10.960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.10.960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.10.960"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10523524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hanafey</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">W</div></div>. </div><div class="article-title">Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes</div>, <div class="source">Nat. Genet</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">194</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20are%20preferentially%20associated%20with%20nonrepetitive%20DNA%20in%20plant%20genomes&amp;author=M.%20Morgante&amp;author=M.%20Hanafey&amp;author=W%20Powell&amp;publication_year=2002&amp;journal=Nat.%20Genet&amp;volume=30&amp;pages=194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng822" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11799393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">Y. G.</div></div>,  <div class="name"><div class="surname">Ishii</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Diversity of microsatellites derived from genomic libraries and GenBank sequences in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Theor. Appl. Genet</div> , <div class="year">2000</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">713</div>-<div class="lpage">722</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20microsatellites%20derived%20from%20genomic%20libraries%20and%20GenBank%20sequences%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=Y.%20G.%20Cho&amp;author=T.%20Ishii&amp;author=S.%20Temnykh&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet&amp;volume=100&amp;pages=713-722" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051343"> </span></div> </div></div></div></div></div><div content-id="DSM005C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Grandillo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ahn</div> <div class="given-names">S. N.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S. R</div></div>. </div><div class="article-title">Identification of trait-improving quantitative trait loci alleles from a wild rice relative</div>, <div class="source">Oryza rufipogon, Genetics</div> , <div class="year">1998</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">899</div>-<div class="lpage">909</div>)</div></div></div></div><div content-id="DSM005C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zou</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">X. B.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Z. X.</div></div>, et al. </div><div class="article-title">Mapping quantitative trait loci controlling sheath blight resistance into rice cultivars (<em>Oryza sativa</em> L.)</div>, <div class="source">Theor. Appl. Genet</div> , <div class="year">2000</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">569</div>-<div class="lpage">573</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20quantitative%20trait%20loci%20controlling%20sheath%20blight%20resistance%20into%20rice%20cultivars%20(Oryza%20sativa%20L.)&amp;author=J.%20H.%20Zou&amp;author=X.%20B.%20Pan&amp;author=Z.%20X.%20Chen&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet&amp;volume=101&amp;pages=569-573" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051517"> </span></div> </div></div></div></div></div><div content-id="DSM005C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tautz</div> <div class="given-names">D</div></div>. </div><div class="article-title">Hypervariability of simple sequences as a general source for polymorphic DNA markers</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1989</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">6463</div>-<div class="lpage">6471</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypervariability%20of%20simple%20sequences%20as%20a%20general%20source%20for%20polymorphic%20DNA%20markers&amp;author=D%20Tautz&amp;publication_year=1989&amp;journal=Nucleic%20Acids%20Res&amp;volume=17&amp;pages=6463-6471" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/17.16.6463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F17.16.6463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F17.16.6463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2780284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Panaud</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S. R</div></div>. </div><div class="article-title">Development of microsatellite markers and characterization of simple sequence length polymorphism (SSLP) in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Mol. Gen. Genet</div> , <div class="year">1996</div>, vol. <div class="volume">252</div> (pg. <div class="fpage">597</div>-<div class="lpage">607</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20microsatellite%20markers%20and%20characterization%20of%20simple%20sequence%20length%20polymorphism%20(SSLP)%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=O.%20Panaud&amp;author=X.%20Chen&amp;author=S.%20R%20McCouch&amp;publication_year=1996&amp;journal=Mol.%20Gen.%20Genet&amp;volume=252&amp;pages=597-607" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8914521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">K. S.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S. D</div></div>. </div><div class="article-title">Abundance, polymorphism and genetic mapping of microsatellites in rice</div>, <div class="source">Mol. Gen. Genet</div> , <div class="year">1993</div>, vol. <div class="volume">241</div> (pg. <div class="fpage">225</div>-<div class="lpage">235</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%2C%20polymorphism%20and%20genetic%20mapping%20of%20microsatellites%20in%20rice&amp;author=K.%20S.%20Wu&amp;author=S.%20D%20Tanksley&amp;publication_year=1993&amp;journal=Mol.%20Gen.%20Genet&amp;volume=241&amp;pages=225-235" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00280220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00280220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00280220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7901751" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akagi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yokozeki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Inagaki</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Fujimura</div> <div class="given-names">T</div></div>. </div><div class="article-title">Microsatellite DNA markers for rice chromosomes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1996</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">61</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20DNA%20markers%20for%20rice%20chromosomes&amp;author=H.%20Akagi&amp;author=Y.%20Yokozeki&amp;author=A.%20Inagaki&amp;author=T%20Fujimura&amp;publication_year=1996&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=94&amp;pages=61-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050382" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050382" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050382"> </span></div> </div></div></div></div></div><div content-id="DSM005C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">DeClerck</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lukashova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cartinhour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S. R</div></div>. </div><div class="article-title">Computational and experimental analysis of microsatellites in rice (<em>Oryza sativa</em> L.): frequency, length variation, transposon associations, and genetic marker potential</div>, <div class="source">Genome Res</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1441</div>-<div class="lpage">1452</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20analysis%20of%20microsatellites%20in%20rice%20(Oryza%20sativa%20L.)%3A%20frequency%2C%20length%20variation%2C%20transposon%20associations%2C%20and%20genetic%20marker%20potential&amp;author=S.%20Temnykh&amp;author=G.%20DeClerck&amp;author=A.%20Lukashova&amp;author=L.%20Lipovich&amp;author=S.%20Cartinhour&amp;author=S.%20R%20McCouch&amp;publication_year=2001&amp;journal=Genome%20Res&amp;volume=11&amp;pages=1441-1452" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.184001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.184001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.184001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">W. D.</div></div>,  <div class="name"><div class="surname">Ayers</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Mapping and genome organization of microsatellite sequences in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Theor. Appl. Genet</div> , <div class="year">2000</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">697</div>-<div class="lpage">712</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20genome%20organization%20of%20microsatellite%20sequences%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=S.%20Temnykh&amp;author=W.%20D.%20Park&amp;author=N.%20Ayers&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet&amp;volume=100&amp;pages=697-712" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051342" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051342" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051342"> </span></div> </div></div></div></div></div><div content-id="DSM005C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCouch</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Teytelman</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Development and mapping of 2240 new SSR markers for rice (<em>Oryza sativa</em> L.)</div>, <div class="source">DNA Res</div> , <div class="year">2002</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">199</div>-<div class="lpage">207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20mapping%20of%202240%20new%20SSR%20markers%20for%20rice%20(Oryza%20sativa%20L.)&amp;author=S.%20R.%20McCouch&amp;author=L.%20Teytelman&amp;author=Y.%20Xu&amp;publication_year=2002&amp;journal=DNA%20Res&amp;volume=9&amp;pages=199-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.6.199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.6.199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.6.199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12597276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Genomes of <em>Oryza sativa</em>: a history of duplications</div>, <div class="source">PLoS Biol</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> </div></div></div></div><div content-id="DSM005C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="collab">IRGSP</div><div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">436</div> (pg. <div class="fpage">793</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ying</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">B</div></div>. </div><div class="article-title">Sequence variations of simple sequence repeats on chromosome-4 in two subspecies of the Asian cultivated rice</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">392</div>-<div class="lpage">400</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20variations%20of%20simple%20sequence%20repeats%20on%20chromosome-4%20in%20two%20subspecies%20of%20the%20Asian%20cultivated%20rice&amp;author=C.%20Li&amp;author=Y.%20Zhang&amp;author=K.%20Ying&amp;author=X.%20Liang&amp;author=B%20Han&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=392-400" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1457-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1457-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1457-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14564393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">BGI-RIS: an integrated information resource and comparative analysis workbench for rice genomics</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D377</div>-<div class="lpage">D382</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BGI-RIS%3A%20an%20integrated%20information%20resource%20and%20comparative%20analysis%20workbench%20for%20rice%20genomics&amp;author=W.%20Zhao&amp;author=J.%20Wang&amp;author=X.%20He&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=D377-D382" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Krawetz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Misener</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Bioinformatics Methods and Protocols: Methods in Molecular Biology</div> , <div class="year">2000</div><div class="publisher-name">Humana Press: Totowa, NJ</div>(pg. <div class="fpage">365</div>-<div class="lpage">386.</div>)</div></div></div></div><div content-id="DSM005C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murray</div> <div class="given-names">M. G.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">W. F</div></div>. </div><div class="article-title">Rapid isolation of high molecular weight plant DNA</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1980</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">4321</div>-<div class="lpage">4325</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20isolation%20of%20high%20molecular%20weight%20plant%20DNA&amp;author=M.%20G.%20Murray&amp;author=W.%20F%20Thompson&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res&amp;volume=8&amp;pages=4321-4325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/8.19.4321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F8.19.4321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F8.19.4321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7433111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujimori</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Washio</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Higo</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A novel feature of microsatellites in plants: a distribution gradient along the direction of transcription</div>, <div class="source">FEBS Lett</div> , <div class="year">2003</div>, vol. <div class="volume">554</div> (pg. <div class="fpage">17</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20feature%20of%20microsatellites%20in%20plants%3A%20a%20distribution%20gradient%20along%20the%20direction%20of%20transcription&amp;author=S.%20Fujimori&amp;author=T.%20Washio&amp;author=K.%20Higo&amp;publication_year=2003&amp;journal=FEBS%20Lett&amp;volume=554&amp;pages=17-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(03)01041-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(03)01041-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(03)01041-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14596907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Preference of simple sequence repeats in coding and non-coding regions of <em>Arabidopsis thaliana</em></div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1081</div>-<div class="lpage">1086</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preference%20of%20simple%20sequence%20repeats%20in%20coding%20and%20non-coding%20regions%20of%20Arabidopsis%20thaliana&amp;author=L.%20Zhang&amp;author=D.%20Yuan&amp;author=S.%20Yu&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=1081-1086" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ni</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">G.-K. S</div></div>. </div><div class="article-title">Comparing the whole-genome-shotgun and map-based sequences of the rice genome</div>, <div class="source">Trends Plant Sci</div> , <div class="year">2006</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">387</div>-<div class="lpage">391</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparing%20the%20whole-genome-shotgun%20and%20map-based%20sequences%20of%20the%20rice%20genome&amp;author=J.%20Yu&amp;author=P.%20Ni&amp;author=G.-K.%20S%20Wong&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci&amp;volume=11&amp;pages=387-391" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2006.06.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2006.06.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2006.06.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16843033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cohen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Danin-Poleg</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Cohen</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Sprecher</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Darvasi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kashi</div> <div class="given-names">Y</div></div>. </div><div class="article-title">Mono-nucleotide repeats (MNRs): a neglected polymorphism for generating high density genetic maps in silicon</div>, <div class="source">Hum. Genet</div> , <div class="year">2004</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">213</div>-<div class="lpage">220</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mono-nucleotide%20repeats%20(MNRs)%3A%20a%20neglected%20polymorphism%20for%20generating%20high%20density%20genetic%20maps%20in%20silicon&amp;author=H.%20Cohen&amp;author=Y.%20Danin-Poleg&amp;author=C.%20J.%20Cohen&amp;author=E.%20Sprecher&amp;author=A.%20Darvasi&amp;author=Y%20Kashi&amp;publication_year=2004&amp;journal=Hum.%20Genet&amp;volume=115&amp;pages=213-220" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00439-004-1135-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00439-004-1135-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00439-004-1135-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15232733" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Samina</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sukumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ramesh</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Jenkins</div> <div class="given-names">J</div></div>. </div><div class="article-title">EST-SSR: A new class of genetic markers in cotton</div>, <div class="source">J. Cotton Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">112</div>-<div class="lpage">123</div>)</div></div></div></div><div content-id="DSM005C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schloss</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mitchell</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Characterization of RFLP probe sequences for gene discovery and SSR development in <em>Sorghum bicolor</em> (L.) Moench</div>, <div class="source">Theor. Appl. Genet</div> , <div class="year">2002</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">912</div>-<div class="lpage">920</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20RFLP%20probe%20sequences%20for%20gene%20discovery%20and%20SSR%20development%20in%20Sorghum%20bicolor%20(L.)%20Moench&amp;author=J.%20Schloss&amp;author=E.%20Mitchell&amp;author=M.%20White&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet&amp;volume=105&amp;pages=912-920" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-0991-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-0991-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-0991-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM005C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM005C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlotterer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tautz</div> <div class="given-names">D</div></div>. </div><div class="article-title">Slippage synthesis of simple sequence DNA</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1992</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">211</div>-<div class="lpage">215</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Slippage%20synthesis%20of%20simple%20sequence%20DNA&amp;author=C.%20Schlotterer&amp;author=D%20Tautz&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res&amp;volume=20&amp;pages=211-215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.2.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.2.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.2.211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1741246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=503258&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm005/1/dsm005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=ab0GRSNDQHeL0kz9WWjA812YxBY0wUaOX~Rq5myWbQulNqOuHPHHn7~y98gZ0j1CWQplmNjUxwTV2oIe1YI-2owvQW70uYD9W5SZ3G6eJyVeIdnAlDAp-yqWhpNkXljpZcix9kjI32Spymcc89sQR7MS~nXJHoFw~7HBHvR1paOvHPeW2AQqWsqdVSU81vPk86zd5lvVXNVSldoRn8iJuEGbGbdfKtKJ7wzJRLkar1n8QLFimiRUgL047R0tt2wt23A~fAx-v9zFp6DROcrlp2W70wS~U7ustqfxq-DIpyy8YEGETlHYbkdf1PuIMM0V-s8KsbDrLsX~F9xmP~vHSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">240</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248042500004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">49</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="503258"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248042500004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f1%2f37%2f503258">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000248042500004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (49)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F1%2F37%2F503258" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/503258'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide Microsatellite Polymorphism Database for the Indica and Japonica Rice" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f37%2f503258">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

