

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/4/279/348084","@type":"ScholarlyArticle","name":"Making a chocolate chip: development and evaluation of a 6K SNP array for  Theobroma cacao ","datePublished":"2015-06-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2015-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv009","keywords":["cacao","SNP","mapping","breeding","markers"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/4/279/348084","author":[{"name":"Livingstone, Donald","affiliation":"1  Mars, Incorporated ,  McLean, VA 22101 ,  USA","@type":"Person"},{"name":"Royaert, Stefan","affiliation":"1  Mars, Incorporated ,  McLean, VA 22101 ,  USA","@type":"Person"},{"name":"Stack, Conrad","affiliation":"1  Mars, Incorporated ,  McLean, VA 22101 ,  USA","@type":"Person"},{"name":"Mockaitis, Keithanne","affiliation":"2  Department of Biology ,  Indiana University ,  Bloomington, IN 47405 ,  USA","@type":"Person"},{"name":"May, Greg","affiliation":"3  National Center for Genome Resources ,  Santa Fe, NM 87505 ,  USA","@type":"Person"},{"name":"Farmer, Andrew","affiliation":"3  National Center for Genome Resources ,  Santa Fe, NM 87505 ,  USA","@type":"Person"},{"name":"Saski, Christopher","affiliation":"4  Clemson University Genomics and Computational Biology Laboratory, Institute for Translational Genomics ,  Clemson, SC 29634 ,  USA","@type":"Person"},{"name":"Schnell, Ray","affiliation":"1  Mars, Incorporated ,  McLean, VA 22101 ,  USA","@type":"Person"},{"name":"Kuhn, David","affiliation":"5  United States Department of Agriculture-Agriculture Research Service ,  Subtropical, Horticulture Research Station ,  Miami, FL 33158 ,  USA","@type":"Person"},{"name":"Motamayor, Juan Carlos","affiliation":"1  Mars, Incorporated ,  McLean, VA 22101 ,  USA","@type":"Person"}],"description":"Abstract.  Theobroma cacao, the key ingredient in chocolate production, is one of the world's most important tree fruit crops, with ∼4,000,000 metric tons produ","pageStart":"279","pageEnd":"291","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Making a chocolate chip: development and evaluation of a 6K SNP array for  Theobroma cacao ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Making a chocolate chip: development and evaluation of a 6K SNP array for  Theobroma cacao " />
    <meta property="og:description" content="Abstract.  Theobroma cacao, the key ingredient in chocolate production, is one of the world&#39;s most important tree fruit crops, with ∼4,000,000 metric tons produ" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/4/279/348084" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-06-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Livingstone, Donald" /><meta name="citation_author_institution" content="Mars, Incorporated, McLean, VA 22101, USA" /><meta name="citation_author" content="Royaert, Stefan" /><meta name="citation_author_institution" content="Mars, Incorporated, McLean, VA 22101, USA" /><meta name="citation_author" content="Stack, Conrad" /><meta name="citation_author_institution" content="Mars, Incorporated, McLean, VA 22101, USA" /><meta name="citation_author" content="Mockaitis, Keithanne" /><meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, IN 47405, USA" /><meta name="citation_author" content="May, Greg" /><meta name="citation_author_institution" content="National Center for Genome Resources, Santa Fe, NM 87505, USA" /><meta name="citation_author" content="Farmer, Andrew" /><meta name="citation_author_institution" content="National Center for Genome Resources, Santa Fe, NM 87505, USA" /><meta name="citation_author" content="Saski, Christopher" /><meta name="citation_author_institution" content="Clemson University Genomics and Computational Biology Laboratory, Institute for Translational Genomics, Clemson, SC 29634, USA" /><meta name="citation_author" content="Schnell, Ray" /><meta name="citation_author_institution" content="Mars, Incorporated, McLean, VA 22101, USA" /><meta name="citation_author" content="Kuhn, David" /><meta name="citation_author_institution" content="United States Department of Agriculture-Agriculture Research Service, Subtropical, Horticulture Research Station, Miami, FL 33158, USA" /><meta name="citation_author" content="Motamayor, Juan Carlos" /><meta name="citation_author_institution" content="Mars, Incorporated, McLean, VA 22101, USA" /><meta name="citation_title" content="Making a chocolate chip: development and evaluation of a 6K SNP array for Theobroma cacao" /><meta name="citation_firstpage" content="279" /><meta name="citation_lastpage" content="291" /><meta name="citation_doi" content="10.1093/dnares/dsv009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2015/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="Food and Agriculture Organization of the United Nations. 2015. FAOSTAT. Crops (National Production).http://data.fao.org/ref/29920434-c74e-4ea2-beed-01b832e60609.html?version=1.0(6 March 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=Cacao and its allies: a taxonomic revision og the genus Theobroma; citation_author=Cuatrecasas  J.;  citation_journal_title=Contrib. US Herbarium;  citation_year=1964;  citation_volume=35;  citation_pages=542-3; " /><meta name="citation_reference" content="citation_title=The status of cacao (Theobroma-Cacao, Sterculiaceae) in the western-hemisphere; citation_author=Hunter  J.R.;  citation_journal_title=Econ. Bot.;  citation_year=1990;  citation_volume=44;  citation_pages=425-39; " /><meta name="citation_reference" content="citation_title=Cacao domestication I: the origin of the cacao cultivated by the Mayas; citation_author=Motamayor  J.C.; citation_author=Risterucci  A.M.; citation_author=Lopez  P.A.; citation_author=Ortiz  C.F.; citation_author=Moreno  A.; citation_author=Lanaud  C.;  citation_journal_title=Heredity;  citation_year=2002;  citation_volume=89;  citation_pages=380-6; " /><meta name="citation_reference" content="citation_author=Fredholm  A.; citation_publisher=The West India Committee Circular, ;  citation_title=Breeding of cacao;  citation_year=1911;  citation_pages=102-3; " /><meta name="citation_reference" content="citation_title=Geographic and genetic population differentiation of the Amazonian chocolate tree (Theobroma cacao L); citation_author=Motamayor  J.C.; citation_author=Lachenaud  P.; citation_author=da Silva e Mota  J.W.;  citation_journal_title=PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3311" /><meta name="citation_reference" content="citation_title=Cocoa breeding in the 21st century; citation_author=Warren  J.; citation_publisher=Port of Spain, Trinidad and Tobago;  citation_title=International Workshop on Conservation, Characterisation and Utilisation of Cocoa Genetic Resources in the 21st Century;  citation_year=1992; " /><meta name="citation_reference" content="citation_title=A systematic description of cacao clones. III. Relationships between clones, between characteristics and some consequences for the cacao breeding; citation_author=Engels  J.;  citation_journal_title=Euphytica;  citation_year=1983;  citation_volume=32;  citation_pages=719-33; " /><meta name="citation_reference" content="citation_title=Germplasm enhancement for resistance to black pod and witches' broom diseases; citation_author=Iwaro  A.D.; citation_author=Butler  D.P.; citation_publisher=Cocoa Producers Alliance, Lagos, Nigeria;  citation_title=Proceedings of the 13th International Cocoa Research Conference: Towards the Effective and Optimum Promotion of Cocoa through Research and Development;  citation_year=2000; " /><meta name="citation_reference" content="citation_title=Genetic diversity, conservation, and utilization of Theobroma cacao L.: genetic resources in the Dominican Republic; citation_author=Boza  E.J.; citation_author=Irish  B.M.; citation_author=Meerow  A.W.;  citation_journal_title=Genet. Resour. Crop Evol.;  citation_year=2013;  citation_volume=60;  citation_pages=605-19; " /><meta name="citation_reference" content="citation_title=Microsatellite fingerprinting of the USDA-ARS Tropical Agriculture Research Station Cacao (Theobroma cacao L.) germplasm collection; citation_author=Irish  B.M.; citation_author=Goenaga  R.; citation_author=Zhang  D.; citation_author=Schnell  R.; citation_author=Brown  J.S.; citation_author=Motamayor  J.C.;  citation_journal_title=Crop Sci.;  citation_year=2010;  citation_volume=50;  citation_pages=656-67; " /><meta name="citation_reference" content="citation_title=Retrospective determination of the parental population of superior cacao (Theobroma cacao L.) seedlings and association of microsatellite alleles with productivity; citation_author=Schnell  R.J.; citation_author=Olano  C.T.; citation_author=Brown  J.S.;  citation_journal_title=J. Am. Soc. Hortic. Sci.;  citation_year=2005;  citation_volume=130;  citation_pages=181-90; " /><meta name="citation_reference" content="citation_title=A high-density linkage map of Theobroma cacao L; citation_author=Risterucci  A.M.; citation_author=Grivet  L.; citation_author=N'Goran  J.A.K.; citation_author=Pieretti  I.; citation_author=Flament  M.H.; citation_author=Lanaud  C.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2000;  citation_volume=101;  citation_pages=948-55; " /><meta name="citation_reference" content="citation_title=Resistance gene homologues in Theobroma cacao as useful genetic markers; citation_author=Kuhn  D.N.; citation_author=Heath  M.; citation_author=Wisser  R.J.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=191-202; " /><meta name="citation_reference" content="citation_title=Isolation, characterization, and development of WRKY genes as useful genetic markers in Theobroma cacao; citation_author=Borrone  J.; citation_author=Kuhn  D.; citation_author=Schnell  R.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=495-507; " /><meta name="citation_reference" content="citation_title=Development of a marker assisted selection program for cacao; citation_author=Schnell  R.J.; citation_author=Kuhn  D.N.; citation_author=Brown  J.S.;  citation_journal_title=Phytopathology;  citation_year=2007;  citation_volume=97;  citation_pages=1664-9; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphisms from Theobroma cacao expressed sequence tags associated with witches’ broom disease in cacao; citation_author=Lima  L.; citation_author=Gramacho  K.; citation_author=Carels  N.;  citation_journal_title=Genet. Mol. Res.;  citation_year=2009;  citation_volume=8;  citation_pages=799-808; " /><meta name="citation_reference" content="citation_title=SSCP markers provide a useful alternative to microsatellites in genotyping and estimating genetic diversity in populations and germplasm collections of plant specialty crops; citation_author=Kuhn  D.N.; citation_author=Motamayor  J.C.; citation_author=Meerow  A.W.; citation_author=Borrone  J.W.; citation_author=Schnell  R.J.;  citation_journal_title=Electrophoresis;  citation_year=2008;  citation_volume=29;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=The relic Criollo cacao in Belize - genetic diversity and relationship with Trinitario and other cacao clones held in the International Cocoa Genebank, Trinidad; citation_author=Motilal  L.A.; citation_author=Zhang  D.P.; citation_author=Umaharan  P.; citation_author=Mischke  S.; citation_author=Mooleedhar  V.; citation_author=Meinhardt  L.W.;  citation_journal_title=Plant Genet. Resour. C.;  citation_year=2010;  citation_volume=8;  citation_pages=106-15; " /><meta name="citation_reference" content="citation_title=Microsatellite markers developed from Theobroma cacao L. expressed sequence tags; citation_author=Borrone  J.W.; citation_author=Brown  J.S.; citation_author=Kuhn  D.N.; citation_author=Motamayor  J.C.; citation_author=Schnell  R.J.;  citation_journal_title=Mol. Ecol. Notes;  citation_year=2007;  citation_volume=7;  citation_pages=236-9; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of microsatellites in Theobroma cacao L; citation_author=Lanaud  C.; citation_author=Risterucci  A.M.; citation_author=Pieretti  I.; citation_author=Falque  M.; citation_author=Bouet  A.; citation_author=Lagoda  P.J.;  citation_journal_title=Mol. Ecol.;  citation_year=1999;  citation_volume=8;  citation_pages=2141-3; " /><meta name="citation_reference" content="citation_title=Development of single nucleotide polymorphism markers in Theobroma cacao and comparison to simple sequence repeat markers for genotyping of Cameroon clones; citation_author=Livingstone  D.S.; citation_author=Motamayor  J.C.; citation_author=Schnell  R.J.;  citation_journal_title=Mol. Breed.;  citation_year=2011;  citation_volume=27;  citation_pages=93-106; " /><meta name="citation_reference" content="citation_title=The genome of Theobroma cacao; citation_author=Argout  X.; citation_author=Salse  J.; citation_author=Aury  J.M.;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=The genome sequence of the most widely cultivated cacao type and its use to identify candidate genes regulating pod color; citation_author=Motamayor  J.C.; citation_author=Mockaitis  K.; citation_author=Schmutz  J.;  citation_journal_title=Genome Biol.;  citation_year=2013;  citation_volume=14;  citation_pages=R53" /><meta name="citation_reference" content="citation_title=Evaluation of single-nucleotide polymorphisms in CAPN1 for association with meat tenderness in cattle; citation_author=Page  B.T.; citation_author=Casas  E.; citation_author=Heaton  M.P.;  citation_journal_title=J. Anim. Sci.;  citation_year=2002;  citation_volume=80;  citation_pages=3077-85; " /><meta name="citation_reference" content="citation_title=Development of SNP assays for marker-assisted selection of two southern root-knot nematode resistance QTL in soybean; citation_author=Ha  B.K.; citation_author=Hussey  R.S.; citation_author=Boerma  H.R.;  citation_journal_title=Crop Sci.;  citation_year=2007;  citation_volume=47;  citation_pages=S73-82; " /><meta name="citation_reference" content="citation_title=An SNP map of the human genome generated by reduced representation shotgun sequencing; citation_author=Altshuler  D.; citation_author=Pollara  V.J.; citation_author=Cowles  C.R.;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=407;  citation_pages=513-6; " /><meta name="citation_reference" content="citation_title=Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds; citation_author=The Bovine HapMap Consortium;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=528-32; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP detection, validation, and development of an 8K SNP array for apple; citation_author=Chagné  D.; citation_author=Crowhurst  R.N.; citation_author=Troggio  M.;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e31745" /><meta name="citation_reference" content="citation_title=SNP discovery through next-generation sequencing and its applications; citation_author=Kumar  S.; citation_author=Banks  T.W.; citation_author=Cloutier  S.;  citation_journal_title=Int. J. Plant Genomics;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries; citation_author=Van Tassell  C.P.; citation_author=Smith  T.P.L.; citation_author=Matukumalli  L.K.;  citation_journal_title=Nat. Meth.;  citation_year=2008;  citation_volume=5;  citation_pages=247-52; " /><meta name="citation_reference" content="citation_title=Applying SNP marker technology in the cacao breeding programme in Ghana; citation_author=Takrama  J.; citation_author=Dadzie  A.M.; citation_author=Opoku  S.Y.;  citation_journal_title=Afr. Crop Sci. J.;  citation_year=2012;  citation_volume=20;  citation_pages=67-75; " /><meta name="citation_reference" content="citation_title=Evaluating Theobroma grandiflorum for comparative genomic studies with Theobroma cacao; citation_author=Kuhn  D.N.; citation_author=Figueira  A.; citation_author=Lopes  U.;  citation_journal_title=Tree Genet. Genomes;  citation_year=2010;  citation_volume=6;  citation_pages=783-92; " /><meta name="citation_reference" content="citation_title=Optimization of a SNP assay for genotyping Theobroma cacao under field conditions; citation_author=Livingstone  D.S.; citation_author=Freeman  B.; citation_author=Motamayor  J.C.;  citation_journal_title=Mol. Breed.;  citation_year=2012;  citation_volume=30;  citation_pages=33-52; " /><meta name="citation_reference" content="citation_title=Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of Theobroma cacao L; citation_author=Allegre  M.; citation_author=Argout  X.; citation_author=Boccara  M.;  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=23-35; " /><meta name="citation_reference" content="citation_title=Accurate determination of genetic identity for a single cacao bean, using molecular markers with a nanofluidic system, ensures cocoa authentication; citation_author=Fang  W.; citation_author=Meinhardt  L.W.; citation_author=Mischke  S.; citation_author=Bellato  C.M.; citation_author=Motilal  L.; citation_author=Zhang  D.;  citation_journal_title=J. Agric. Food Chem.;  citation_year=2013;  citation_volume=62;  citation_pages=481-7; " /><meta name="citation_reference" content="citation_title=Development of a dense SNP-based linkage map of an apple rootstock progeny using the Malus Infinium whole genome genotyping array; citation_author=Antanaviciute  L.; citation_author=Fernández-Fernández  F.; citation_author=Jansen  J.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=203" /><meta name="citation_reference" content="citation_title=A simple and efficient method for isolating RNA from pine trees; citation_author=Chang  S.; citation_author=Puryear  J.; citation_author=Cairney  J.;  citation_journal_title=Plant Mol. Biol. Rep.;  citation_year=1993;  citation_volume=11;  citation_pages=113-6; " /><meta name="citation_reference" content="citation_title=Gene expression in leaves of Theobroma cacao in response to mechanical wounding, ethylene, and/or methyl jasmonate; citation_author=Bailey  B.A.; citation_author=Strem  M.D.; citation_author=Bae  H.H.; citation_author=de Mayolo  G.A.; citation_author=Guiltinan  M.J.;  citation_journal_title=Plant Sci.;  citation_year=2005;  citation_volume=168;  citation_pages=1247-58; " /><meta name="citation_reference" content="citation_title=Identification and mapping of conserved ortholog set (COS) II sequences of cacao and their conversion to SNP markers for marker-assisted selection in Theobroma cacao and comparative genomics studies; citation_author=Kuhn  D.N.; citation_author=Livingstone  D.; citation_author=Main  D.;  citation_journal_title=Tree Genet. Genomes;  citation_year=2012;  citation_volume=8;  citation_pages=97-111; " /><meta name="citation_reference" content="citation_title=Management of high-throughput DNA sequencing projects: alpheus; citation_author=Miller  N.A.; citation_author=Kingsmore  S.F.; citation_author=Farmer  A.;  citation_journal_title=J. Comput. Sci. Syst. Biol.;  citation_year=2008;  citation_volume=1;  citation_pages=132" /><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=Purcell  S.; citation_author=Neale  B.; citation_author=Todd-Brown  K.;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2007;  citation_volume=81;  citation_pages=559-75; " /><meta name="citation_reference" content="citation_author=RC Team;  citation_title=R: a language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing; 2012;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Construction of integrated genetic linkage maps by means of a new computer package: Join Map; citation_author=Stam  P.;  citation_journal_title=Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=739-44; " /><meta name="citation_reference" content="citation_title=Towards the understanding of the cocoa transcriptome: production and analysis of an exhaustive dataset of ESTs of Theobroma cacao L. generated from various tissues and under various conditions; citation_author=Argout  X.; citation_author=Fouet  O.; citation_author=Wincker  P.;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=512" /><meta name="citation_reference" content="citation_title=Cacao domestication II: progenitor germplasm of the Trinitario cacao cultivar; citation_author=Motamayor  J.C.; citation_author=Risterucci  A.M.; citation_author=Heath  M.; citation_author=Lanaud  C.;  citation_journal_title=Heredity;  citation_year=2003;  citation_volume=91;  citation_pages=322-30; " /><meta name="citation_reference" content="citation_title=Mapping QTLs for resistance to frosty pod and black pod diseases and horticultural traits in Theobroma cacao L; citation_author=Brown  J.S.; citation_author=Phillips-Mora  W.; citation_author=Power  E.J.;  citation_journal_title=Crop Sci.;  citation_year=2007;  citation_volume=47;  citation_pages=1851-8; " /><meta name="citation_reference" content="citation_title=A composite linkage map from three crosses between commercial clones of cacao, Theobroma cacao L; citation_author=Brown  J.; citation_author=Sautter  R.; citation_author=Olano  C.;  citation_journal_title=Trop. Plant Biol.;  citation_year=2008;  citation_volume=1;  citation_pages=120-30; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of Theobroma cacao L; citation_author=Lanaud  C.; citation_author=Risterucci  A.; citation_author=N'goran  A.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=1995;  citation_volume=91;  citation_pages=987-93; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/4/279/997447/dsv009.pdf" /><meta name="description" content="Abstract.  Theobroma cacao, the key ingredient in chocolate production, is one of the world's most important tree fruit crops, with ∼4,000,000 metric tons produ" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/4/279/348084" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/4/279/348084" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/4/279/348084");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv009';
    </script>
<title>
	Making a chocolate chip: development and evaluation of a 6K SNP array for Theobroma cacao | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348084" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdCgATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl2aW5nc3RvbmUsIERvbmFsZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1hcnMsIEluY29ycG9yYXRlZCwgTWNMZWFuLCBWQSAyMjEwMSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm95YWVydCwgU3RlZmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFycywgSW5jb3Jwb3JhdGVkLCBNY0xlYW4sIFZBIDIyMTAxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdGFjaywgQ29ucmFkIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFycywgSW5jb3Jwb3JhdGVkLCBNY0xlYW4sIFZBIDIyMTAxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb2NrYWl0aXMsIEtlaXRoYW5uZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSwgSW5kaWFuYSBVbml2ZXJzaXR5LCBCbG9vbWluZ3RvbiwgSU4gNDc0MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1heSwgR3JlZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIENlbnRlciBmb3IgR2Vub21lIFJlc291cmNlcywgU2FudGEgRmUsIE5NIDg3NTA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGYXJtZXIsIEFuZHJldyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIENlbnRlciBmb3IgR2Vub21lIFJlc291cmNlcywgU2FudGEgRmUsIE5NIDg3NTA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXNraSwgQ2hyaXN0b3BoZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbGVtc29uIFVuaXZlcnNpdHkgR2Vub21pY3MgYW5kIENvbXB1dGF0aW9uYWwgQmlvbG9neSBMYWJvcmF0b3J5LCBJbnN0aXR1dGUgZm9yIFRyYW5zbGF0aW9uYWwgR2Vub21pY3MsIENsZW1zb24sIFNDIDI5NjM0LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2huZWxsLCBSYXkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYXJzLCBJbmNvcnBvcmF0ZWQsIE1jTGVhbiwgVkEgMjIxMDEsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1aG4sIERhdmlkIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdGVkIFN0YXRlcyBEZXBhcnRtZW50IG9mIEFncmljdWx0dXJlLUFncmljdWx0dXJlIFJlc2VhcmNoIFNlcnZpY2UsIFN1YnRyb3BpY2FsLCBIb3J0aWN1bHR1cmUgUmVzZWFyY2ggU3RhdGlvbiwgTWlhbWksIEZMIDMzMTU4LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3RhbWF5b3IsIEp1YW4gQ2FybG9zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFycywgSW5jb3Jwb3JhdGVkLCBNY0xlYW4sIFZBIDIyMTAxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik1ha2luZyBhIGNob2NvbGF0ZSBjaGlwOiBkZXZlbG9wbWVudCBhbmQgZXZhbHVhdGlvbiBvZiBhIDZLIFNOUCBhcnJheSBmb3IgVGhlb2Jyb21hIGNhY2FvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3YwMDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkZvb2QgYW5kIEFncmljdWx0dXJlIE9yZ2FuaXphdGlvbiBvZiB0aGUgVW5pdGVkIE5hdGlvbnMuIDIwMTUuIEZBT1NUQVQuIENyb3BzIChOYXRpb25hbCBQcm9kdWN0aW9uKS5odHRwOi8vZGF0YS5mYW8ub3JnL3JlZi8yOTkyMDQzNC1jNzRlLTRlYTItYmVlZC0wMWI4MzJlNjA2MDkuaHRtbD92ZXJzaW9uPTEuMCg2IE1hcmNoIDIwMTQsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhY2FvIGFuZCBpdHMgYWxsaWVzOiBhIHRheG9ub21pYyByZXZpc2lvbiBvZyB0aGUgZ2VudXMgVGhlb2Jyb21hOyBjaXRhdGlvbl9hdXRob3I9Q3VhdHJlY2FzYXMgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db250cmliLiBVUyBIZXJiYXJpdW07ICBjaXRhdGlvbl95ZWFyPTE5NjQ7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz01NDItMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN0YXR1cyBvZiBjYWNhbyAoVGhlb2Jyb21hLUNhY2FvLCBTdGVyY3VsaWFjZWFlKSBpbiB0aGUgd2VzdGVybi1oZW1pc3BoZXJlOyBjaXRhdGlvbl9hdXRob3I9SHVudGVyICBKLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FY29uLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9NDI1LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYWNhbyBkb21lc3RpY2F0aW9uIEk6IHRoZSBvcmlnaW4gb2YgdGhlIGNhY2FvIGN1bHRpdmF0ZWQgYnkgdGhlIE1heWFzOyBjaXRhdGlvbl9hdXRob3I9TW90YW1heW9yICBKLkMuOyBjaXRhdGlvbl9hdXRob3I9UmlzdGVydWNjaSAgQS5NLjsgY2l0YXRpb25fYXV0aG9yPUxvcGV6ICBQLkEuOyBjaXRhdGlvbl9hdXRob3I9T3J0aXogIEMuRi47IGNpdGF0aW9uX2F1dGhvcj1Nb3Jlbm8gIEEuOyBjaXRhdGlvbl9hdXRob3I9TGFuYXVkICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz0zODAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPUZyZWRob2xtICBBLjsgY2l0YXRpb25fcHVibGlzaGVyPVRoZSBXZXN0IEluZGlhIENvbW1pdHRlZSBDaXJjdWxhciwgOyAgY2l0YXRpb25fdGl0bGU9QnJlZWRpbmcgb2YgY2FjYW87ICBjaXRhdGlvbl95ZWFyPTE5MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDItMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VvZ3JhcGhpYyBhbmQgZ2VuZXRpYyBwb3B1bGF0aW9uIGRpZmZlcmVudGlhdGlvbiBvZiB0aGUgQW1hem9uaWFuIGNob2NvbGF0ZSB0cmVlIChUaGVvYnJvbWEgY2FjYW8gTCk7IGNpdGF0aW9uX2F1dGhvcj1Nb3RhbWF5b3IgIEouQy47IGNpdGF0aW9uX2F1dGhvcj1MYWNoZW5hdWQgIFAuOyBjaXRhdGlvbl9hdXRob3I9ZGEgU2lsdmEgZSBNb3RhICBKLlcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTMzMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2NvYSBicmVlZGluZyBpbiB0aGUgMjFzdCBjZW50dXJ5OyBjaXRhdGlvbl9hdXRob3I9V2FycmVuICBKLjsgY2l0YXRpb25fcHVibGlzaGVyPVBvcnQgb2YgU3BhaW4sIFRyaW5pZGFkIGFuZCBUb2JhZ287ICBjaXRhdGlvbl90aXRsZT1JbnRlcm5hdGlvbmFsIFdvcmtzaG9wIG9uIENvbnNlcnZhdGlvbiwgQ2hhcmFjdGVyaXNhdGlvbiBhbmQgVXRpbGlzYXRpb24gb2YgQ29jb2EgR2VuZXRpYyBSZXNvdXJjZXMgaW4gdGhlIDIxc3QgQ2VudHVyeTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzeXN0ZW1hdGljIGRlc2NyaXB0aW9uIG9mIGNhY2FvIGNsb25lcy4gSUlJLiBSZWxhdGlvbnNoaXBzIGJldHdlZW4gY2xvbmVzLCBiZXR3ZWVuIGNoYXJhY3RlcmlzdGljcyBhbmQgc29tZSBjb25zZXF1ZW5jZXMgZm9yIHRoZSBjYWNhbyBicmVlZGluZzsgY2l0YXRpb25fYXV0aG9yPUVuZ2VscyAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTcxOS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VybXBsYXNtIGVuaGFuY2VtZW50IGZvciByZXNpc3RhbmNlIHRvIGJsYWNrIHBvZCBhbmQgd2l0Y2hlcycgYnJvb20gZGlzZWFzZXM7IGNpdGF0aW9uX2F1dGhvcj1Jd2FybyAgQS5ELjsgY2l0YXRpb25fYXV0aG9yPUJ1dGxlciAgRC5QLjsgY2l0YXRpb25fcHVibGlzaGVyPUNvY29hIFByb2R1Y2VycyBBbGxpYW5jZSwgTGFnb3MsIE5pZ2VyaWE7ICBjaXRhdGlvbl90aXRsZT1Qcm9jZWVkaW5ncyBvZiB0aGUgMTN0aCBJbnRlcm5hdGlvbmFsIENvY29hIFJlc2VhcmNoIENvbmZlcmVuY2U6IFRvd2FyZHMgdGhlIEVmZmVjdGl2ZSBhbmQgT3B0aW11bSBQcm9tb3Rpb24gb2YgQ29jb2EgdGhyb3VnaCBSZXNlYXJjaCBhbmQgRGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5LCBjb25zZXJ2YXRpb24sIGFuZCB1dGlsaXphdGlvbiBvZiBUaGVvYnJvbWEgY2FjYW8gTC46IGdlbmV0aWMgcmVzb3VyY2VzIGluIHRoZSBEb21pbmljYW4gUmVwdWJsaWM7IGNpdGF0aW9uX2F1dGhvcj1Cb3phICBFLkouOyBjaXRhdGlvbl9hdXRob3I9SXJpc2ggIEIuTS47IGNpdGF0aW9uX2F1dGhvcj1NZWVyb3cgIEEuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0LiBSZXNvdXIuIENyb3AgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz02MDUtMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIGZpbmdlcnByaW50aW5nIG9mIHRoZSBVU0RBLUFSUyBUcm9waWNhbCBBZ3JpY3VsdHVyZSBSZXNlYXJjaCBTdGF0aW9uIENhY2FvIChUaGVvYnJvbWEgY2FjYW8gTC4pIGdlcm1wbGFzbSBjb2xsZWN0aW9uOyBjaXRhdGlvbl9hdXRob3I9SXJpc2ggIEIuTS47IGNpdGF0aW9uX2F1dGhvcj1Hb2VuYWdhICBSLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBELjsgY2l0YXRpb25fYXV0aG9yPVNjaG5lbGwgIFIuOyBjaXRhdGlvbl9hdXRob3I9QnJvd24gIEouUy47IGNpdGF0aW9uX2F1dGhvcj1Nb3RhbWF5b3IgIEouQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV0cm9zcGVjdGl2ZSBkZXRlcm1pbmF0aW9uIG9mIHRoZSBwYXJlbnRhbCBwb3B1bGF0aW9uIG9mIHN1cGVyaW9yIGNhY2FvIChUaGVvYnJvbWEgY2FjYW8gTC4pIHNlZWRsaW5ncyBhbmQgYXNzb2NpYXRpb24gb2YgbWljcm9zYXRlbGxpdGUgYWxsZWxlcyB3aXRoIHByb2R1Y3Rpdml0eTsgY2l0YXRpb25fYXV0aG9yPVNjaG5lbGwgIFIuSi47IGNpdGF0aW9uX2F1dGhvcj1PbGFubyAgQy5ULjsgY2l0YXRpb25fYXV0aG9yPUJyb3duICBKLlMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbS4gU29jLiBIb3J0aWMuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTMwOyAgY2l0YXRpb25fcGFnZXM9MTgxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtZGVuc2l0eSBsaW5rYWdlIG1hcCBvZiBUaGVvYnJvbWEgY2FjYW8gTDsgY2l0YXRpb25fYXV0aG9yPVJpc3RlcnVjY2kgIEEuTS47IGNpdGF0aW9uX2F1dGhvcj1Hcml2ZXQgIEwuOyBjaXRhdGlvbl9hdXRob3I9TidHb3JhbiAgSi5BLksuOyBjaXRhdGlvbl9hdXRob3I9UGllcmV0dGkgIEkuOyBjaXRhdGlvbl9hdXRob3I9RmxhbWVudCAgTS5ILjsgY2l0YXRpb25fYXV0aG9yPUxhbmF1ZCAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9OTQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNpc3RhbmNlIGdlbmUgaG9tb2xvZ3VlcyBpbiBUaGVvYnJvbWEgY2FjYW8gYXMgdXNlZnVsIGdlbmV0aWMgbWFya2VyczsgY2l0YXRpb25fYXV0aG9yPUt1aG4gIEQuTi47IGNpdGF0aW9uX2F1dGhvcj1IZWF0aCAgTS47IGNpdGF0aW9uX2F1dGhvcj1XaXNzZXIgIFIuSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MTkxLTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uLCBjaGFyYWN0ZXJpemF0aW9uLCBhbmQgZGV2ZWxvcG1lbnQgb2YgV1JLWSBnZW5lcyBhcyB1c2VmdWwgZ2VuZXRpYyBtYXJrZXJzIGluIFRoZW9icm9tYSBjYWNhbzsgY2l0YXRpb25fYXV0aG9yPUJvcnJvbmUgIEouOyBjaXRhdGlvbl9hdXRob3I9S3VobiAgRC47IGNpdGF0aW9uX2F1dGhvcj1TY2huZWxsICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz00OTUtNTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIG1hcmtlciBhc3Npc3RlZCBzZWxlY3Rpb24gcHJvZ3JhbSBmb3IgY2FjYW87IGNpdGF0aW9uX2F1dGhvcj1TY2huZWxsICBSLkouOyBjaXRhdGlvbl9hdXRob3I9S3VobiAgRC5OLjsgY2l0YXRpb25fYXV0aG9yPUJyb3duICBKLlMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHl0b3BhdGhvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTE2NjQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBmcm9tIFRoZW9icm9tYSBjYWNhbyBleHByZXNzZWQgc2VxdWVuY2UgdGFncyBhc3NvY2lhdGVkIHdpdGggd2l0Y2hlc+KAmSBicm9vbSBkaXNlYXNlIGluIGNhY2FvOyBjaXRhdGlvbl9hdXRob3I9TGltYSAgTC47IGNpdGF0aW9uX2F1dGhvcj1HcmFtYWNobyAgSy47IGNpdGF0aW9uX2F1dGhvcj1DYXJlbHMgIE4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gTW9sLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz03OTktODA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TU0NQIG1hcmtlcnMgcHJvdmlkZSBhIHVzZWZ1bCBhbHRlcm5hdGl2ZSB0byBtaWNyb3NhdGVsbGl0ZXMgaW4gZ2Vub3R5cGluZyBhbmQgZXN0aW1hdGluZyBnZW5ldGljIGRpdmVyc2l0eSBpbiBwb3B1bGF0aW9ucyBhbmQgZ2VybXBsYXNtIGNvbGxlY3Rpb25zIG9mIHBsYW50IHNwZWNpYWx0eSBjcm9wczsgY2l0YXRpb25fYXV0aG9yPUt1aG4gIEQuTi47IGNpdGF0aW9uX2F1dGhvcj1Nb3RhbWF5b3IgIEouQy47IGNpdGF0aW9uX2F1dGhvcj1NZWVyb3cgIEEuVy47IGNpdGF0aW9uX2F1dGhvcj1Cb3Jyb25lICBKLlcuOyBjaXRhdGlvbl9hdXRob3I9U2NobmVsbCAgUi5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWxlY3Ryb3Bob3Jlc2lzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlbGljIENyaW9sbG8gY2FjYW8gaW4gQmVsaXplIC0gZ2VuZXRpYyBkaXZlcnNpdHkgYW5kIHJlbGF0aW9uc2hpcCB3aXRoIFRyaW5pdGFyaW8gYW5kIG90aGVyIGNhY2FvIGNsb25lcyBoZWxkIGluIHRoZSBJbnRlcm5hdGlvbmFsIENvY29hIEdlbmViYW5rLCBUcmluaWRhZDsgY2l0YXRpb25fYXV0aG9yPU1vdGlsYWwgIEwuQS47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgRC5QLjsgY2l0YXRpb25fYXV0aG9yPVVtYWhhcmFuICBQLjsgY2l0YXRpb25fYXV0aG9yPU1pc2Noa2UgIFMuOyBjaXRhdGlvbl9hdXRob3I9TW9vbGVlZGhhciAgVi47IGNpdGF0aW9uX2F1dGhvcj1NZWluaGFyZHQgIEwuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEdlbmV0LiBSZXNvdXIuIEMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMDYtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZGV2ZWxvcGVkIGZyb20gVGhlb2Jyb21hIGNhY2FvIEwuIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOyBjaXRhdGlvbl9hdXRob3I9Qm9ycm9uZSAgSi5XLjsgY2l0YXRpb25fYXV0aG9yPUJyb3duICBKLlMuOyBjaXRhdGlvbl9hdXRob3I9S3VobiAgRC5OLjsgY2l0YXRpb25fYXV0aG9yPU1vdGFtYXlvciAgSi5DLjsgY2l0YXRpb25fYXV0aG9yPVNjaG5lbGwgIFIuSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTIzNi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgbWljcm9zYXRlbGxpdGVzIGluIFRoZW9icm9tYSBjYWNhbyBMOyBjaXRhdGlvbl9hdXRob3I9TGFuYXVkICBDLjsgY2l0YXRpb25fYXV0aG9yPVJpc3RlcnVjY2kgIEEuTS47IGNpdGF0aW9uX2F1dGhvcj1QaWVyZXR0aSAgSS47IGNpdGF0aW9uX2F1dGhvcj1GYWxxdWUgIE0uOyBjaXRhdGlvbl9hdXRob3I9Qm91ZXQgIEEuOyBjaXRhdGlvbl9hdXRob3I9TGFnb2RhICBQLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMTQxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBtYXJrZXJzIGluIFRoZW9icm9tYSBjYWNhbyBhbmQgY29tcGFyaXNvbiB0byBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0IG1hcmtlcnMgZm9yIGdlbm90eXBpbmcgb2YgQ2FtZXJvb24gY2xvbmVzOyBjaXRhdGlvbl9hdXRob3I9TGl2aW5nc3RvbmUgIEQuUy47IGNpdGF0aW9uX2F1dGhvcj1Nb3RhbWF5b3IgIEouQy47IGNpdGF0aW9uX2F1dGhvcj1TY2huZWxsICBSLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTkzLTEwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBUaGVvYnJvbWEgY2FjYW87IGNpdGF0aW9uX2F1dGhvcj1BcmdvdXQgIFguOyBjaXRhdGlvbl9hdXRob3I9U2Fsc2UgIEouOyBjaXRhdGlvbl9hdXRob3I9QXVyeSAgSi5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbW9zdCB3aWRlbHkgY3VsdGl2YXRlZCBjYWNhbyB0eXBlIGFuZCBpdHMgdXNlIHRvIGlkZW50aWZ5IGNhbmRpZGF0ZSBnZW5lcyByZWd1bGF0aW5nIHBvZCBjb2xvcjsgY2l0YXRpb25fYXV0aG9yPU1vdGFtYXlvciAgSi5DLjsgY2l0YXRpb25fYXV0aG9yPU1vY2thaXRpcyAgSy47IGNpdGF0aW9uX2F1dGhvcj1TY2htdXR6ICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UjUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhbHVhdGlvbiBvZiBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIENBUE4xIGZvciBhc3NvY2lhdGlvbiB3aXRoIG1lYXQgdGVuZGVybmVzcyBpbiBjYXR0bGU7IGNpdGF0aW9uX2F1dGhvcj1QYWdlICBCLlQuOyBjaXRhdGlvbl9hdXRob3I9Q2FzYXMgIEUuOyBjaXRhdGlvbl9hdXRob3I9SGVhdG9uICBNLlAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbmltLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9MzA3Ny04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgU05QIGFzc2F5cyBmb3IgbWFya2VyLWFzc2lzdGVkIHNlbGVjdGlvbiBvZiB0d28gc291dGhlcm4gcm9vdC1rbm90IG5lbWF0b2RlIHJlc2lzdGFuY2UgUVRMIGluIHNveWJlYW47IGNpdGF0aW9uX2F1dGhvcj1IYSAgQi5LLjsgY2l0YXRpb25fYXV0aG9yPUh1c3NleSAgUi5TLjsgY2l0YXRpb25fYXV0aG9yPUJvZXJtYSAgSC5SLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9UzczLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBTTlAgbWFwIG9mIHRoZSBodW1hbiBnZW5vbWUgZ2VuZXJhdGVkIGJ5IHJlZHVjZWQgcmVwcmVzZW50YXRpb24gc2hvdGd1biBzZXF1ZW5jaW5nOyBjaXRhdGlvbl9hdXRob3I9QWx0c2h1bGVyICBELjsgY2l0YXRpb25fYXV0aG9yPVBvbGxhcmEgIFYuSi47IGNpdGF0aW9uX2F1dGhvcj1Db3dsZXMgIEMuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDc7ICBjaXRhdGlvbl9wYWdlcz01MTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgc3VydmV5IG9mIFNOUCB2YXJpYXRpb24gdW5jb3ZlcnMgdGhlIGdlbmV0aWMgc3RydWN0dXJlIG9mIGNhdHRsZSBicmVlZHM7IGNpdGF0aW9uX2F1dGhvcj1UaGUgQm92aW5lIEhhcE1hcCBDb25zb3J0aXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTUyOC0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgU05QIGRldGVjdGlvbiwgdmFsaWRhdGlvbiwgYW5kIGRldmVsb3BtZW50IG9mIGFuIDhLIFNOUCBhcnJheSBmb3IgYXBwbGU7IGNpdGF0aW9uX2F1dGhvcj1DaGFnbsOpICBELjsgY2l0YXRpb25fYXV0aG9yPUNyb3dodXJzdCAgUi5OLjsgY2l0YXRpb25fYXV0aG9yPVRyb2dnaW8gIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTMxNzQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSB0aHJvdWdoIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGFuZCBpdHMgYXBwbGljYXRpb25zOyBjaXRhdGlvbl9hdXRob3I9S3VtYXIgIFMuOyBjaXRhdGlvbl9hdXRob3I9QmFua3MgIFQuVy47IGNpdGF0aW9uX2F1dGhvcj1DbG91dGllciAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gUGxhbnQgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBkaXNjb3ZlcnkgYW5kIGFsbGVsZSBmcmVxdWVuY3kgZXN0aW1hdGlvbiBieSBkZWVwIHNlcXVlbmNpbmcgb2YgcmVkdWNlZCByZXByZXNlbnRhdGlvbiBsaWJyYXJpZXM7IGNpdGF0aW9uX2F1dGhvcj1WYW4gVGFzc2VsbCAgQy5QLjsgY2l0YXRpb25fYXV0aG9yPVNtaXRoICBULlAuTC47IGNpdGF0aW9uX2F1dGhvcj1NYXR1a3VtYWxsaSAgTC5LLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRoLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjQ3LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBseWluZyBTTlAgbWFya2VyIHRlY2hub2xvZ3kgaW4gdGhlIGNhY2FvIGJyZWVkaW5nIHByb2dyYW1tZSBpbiBHaGFuYTsgY2l0YXRpb25fYXV0aG9yPVRha3JhbWEgIEouOyBjaXRhdGlvbl9hdXRob3I9RGFkemllICBBLk0uOyBjaXRhdGlvbl9hdXRob3I9T3Bva3UgIFMuWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFmci4gQ3JvcCBTY2kuIEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjctNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpbmcgVGhlb2Jyb21hIGdyYW5kaWZsb3J1bSBmb3IgY29tcGFyYXRpdmUgZ2Vub21pYyBzdHVkaWVzIHdpdGggVGhlb2Jyb21hIGNhY2FvOyBjaXRhdGlvbl9hdXRob3I9S3VobiAgRC5OLjsgY2l0YXRpb25fYXV0aG9yPUZpZ3VlaXJhICBBLjsgY2l0YXRpb25fYXV0aG9yPUxvcGVzICBVLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NzgzLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWl6YXRpb24gb2YgYSBTTlAgYXNzYXkgZm9yIGdlbm90eXBpbmcgVGhlb2Jyb21hIGNhY2FvIHVuZGVyIGZpZWxkIGNvbmRpdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1MaXZpbmdzdG9uZSAgRC5TLjsgY2l0YXRpb25fYXV0aG9yPUZyZWVtYW4gIEIuOyBjaXRhdGlvbl9hdXRob3I9TW90YW1heW9yICBKLkMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTMzLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3ZlcnkgYW5kIG1hcHBpbmcgb2YgYSBuZXcgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZy1zaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gYW5kIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgcGFuZWwgZm9yIGxhcmdlLXNjYWxlIGdlbmV0aWMgc3R1ZGllcyBhbmQgYnJlZWRpbmcgb2YgVGhlb2Jyb21hIGNhY2FvIEw7IGNpdGF0aW9uX2F1dGhvcj1BbGxlZ3JlICBNLjsgY2l0YXRpb25fYXV0aG9yPUFyZ291dCAgWC47IGNpdGF0aW9uX2F1dGhvcj1Cb2NjYXJhICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yMy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgZGV0ZXJtaW5hdGlvbiBvZiBnZW5ldGljIGlkZW50aXR5IGZvciBhIHNpbmdsZSBjYWNhbyBiZWFuLCB1c2luZyBtb2xlY3VsYXIgbWFya2VycyB3aXRoIGEgbmFub2ZsdWlkaWMgc3lzdGVtLCBlbnN1cmVzIGNvY29hIGF1dGhlbnRpY2F0aW9uOyBjaXRhdGlvbl9hdXRob3I9RmFuZyAgVy47IGNpdGF0aW9uX2F1dGhvcj1NZWluaGFyZHQgIEwuVy47IGNpdGF0aW9uX2F1dGhvcj1NaXNjaGtlICBTLjsgY2l0YXRpb25fYXV0aG9yPUJlbGxhdG8gIEMuTS47IGNpdGF0aW9uX2F1dGhvcj1Nb3RpbGFsICBMLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQWdyaWMuIEZvb2QgQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz00ODEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgYSBkZW5zZSBTTlAtYmFzZWQgbGlua2FnZSBtYXAgb2YgYW4gYXBwbGUgcm9vdHN0b2NrIHByb2dlbnkgdXNpbmcgdGhlIE1hbHVzIEluZmluaXVtIHdob2xlIGdlbm9tZSBnZW5vdHlwaW5nIGFycmF5OyBjaXRhdGlvbl9hdXRob3I9QW50YW5hdmljaXV0ZSAgTC47IGNpdGF0aW9uX2F1dGhvcj1GZXJuw6FuZGV6LUZlcm7DoW5kZXogIEYuOyBjaXRhdGlvbl9hdXRob3I9SmFuc2VuICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUgYW5kIGVmZmljaWVudCBtZXRob2QgZm9yIGlzb2xhdGluZyBSTkEgZnJvbSBwaW5lIHRyZWVzOyBjaXRhdGlvbl9hdXRob3I9Q2hhbmcgIFMuOyBjaXRhdGlvbl9hdXRob3I9UHVyeWVhciAgSi47IGNpdGF0aW9uX2F1dGhvcj1DYWlybmV5ICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTEzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBpbiBsZWF2ZXMgb2YgVGhlb2Jyb21hIGNhY2FvIGluIHJlc3BvbnNlIHRvIG1lY2hhbmljYWwgd291bmRpbmcsIGV0aHlsZW5lLCBhbmQvb3IgbWV0aHlsIGphc21vbmF0ZTsgY2l0YXRpb25fYXV0aG9yPUJhaWxleSAgQi5BLjsgY2l0YXRpb25fYXV0aG9yPVN0cmVtICBNLkQuOyBjaXRhdGlvbl9hdXRob3I9QmFlICBILkguOyBjaXRhdGlvbl9hdXRob3I9ZGUgTWF5b2xvICBHLkEuOyBjaXRhdGlvbl9hdXRob3I9R3VpbHRpbmFuICBNLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE2ODsgIGNpdGF0aW9uX3BhZ2VzPTEyNDctNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBtYXBwaW5nIG9mIGNvbnNlcnZlZCBvcnRob2xvZyBzZXQgKENPUykgSUkgc2VxdWVuY2VzIG9mIGNhY2FvIGFuZCB0aGVpciBjb252ZXJzaW9uIHRvIFNOUCBtYXJrZXJzIGZvciBtYXJrZXItYXNzaXN0ZWQgc2VsZWN0aW9uIGluIFRoZW9icm9tYSBjYWNhbyBhbmQgY29tcGFyYXRpdmUgZ2Vub21pY3Mgc3R1ZGllczsgY2l0YXRpb25fYXV0aG9yPUt1aG4gIEQuTi47IGNpdGF0aW9uX2F1dGhvcj1MaXZpbmdzdG9uZSAgRC47IGNpdGF0aW9uX2F1dGhvcj1NYWluICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTctMTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYW5hZ2VtZW50IG9mIGhpZ2gtdGhyb3VnaHB1dCBETkEgc2VxdWVuY2luZyBwcm9qZWN0czogYWxwaGV1czsgY2l0YXRpb25fYXV0aG9yPU1pbGxlciAgTi5BLjsgY2l0YXRpb25fYXV0aG9yPUtpbmdzbW9yZSAgUy5GLjsgY2l0YXRpb25fYXV0aG9yPUZhcm1lciAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENvbXB1dC4gU2NpLiBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UExJTks6IGEgdG9vbCBzZXQgZm9yIHdob2xlLWdlbm9tZSBhc3NvY2lhdGlvbiBhbmQgcG9wdWxhdGlvbi1iYXNlZCBsaW5rYWdlIGFuYWx5c2VzOyBjaXRhdGlvbl9hdXRob3I9UHVyY2VsbCAgUy47IGNpdGF0aW9uX2F1dGhvcj1OZWFsZSAgQi47IGNpdGF0aW9uX2F1dGhvcj1Ub2RkLUJyb3duICBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9NTU5LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9UkMgVGVhbTsgIGNpdGF0aW9uX3RpdGxlPVI6IGEgbGFuZ3VhZ2UgYW5kIGVudmlyb25tZW50IGZvciBzdGF0aXN0aWNhbCBjb21wdXRpbmcuIFZpZW5uYSwgQXVzdHJpYTogUiBGb3VuZGF0aW9uIGZvciBTdGF0aXN0aWNhbCBDb21wdXRpbmc7IDIwMTI7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBpbnRlZ3JhdGVkIGdlbmV0aWMgbGlua2FnZSBtYXBzIGJ5IG1lYW5zIG9mIGEgbmV3IGNvbXB1dGVyIHBhY2thZ2U6IEpvaW4gTWFwOyBjaXRhdGlvbl9hdXRob3I9U3RhbSAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz03MzktNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZHMgdGhlIHVuZGVyc3RhbmRpbmcgb2YgdGhlIGNvY29hIHRyYW5zY3JpcHRvbWU6IHByb2R1Y3Rpb24gYW5kIGFuYWx5c2lzIG9mIGFuIGV4aGF1c3RpdmUgZGF0YXNldCBvZiBFU1RzIG9mIFRoZW9icm9tYSBjYWNhbyBMLiBnZW5lcmF0ZWQgZnJvbSB2YXJpb3VzIHRpc3N1ZXMgYW5kIHVuZGVyIHZhcmlvdXMgY29uZGl0aW9uczsgY2l0YXRpb25fYXV0aG9yPUFyZ291dCAgWC47IGNpdGF0aW9uX2F1dGhvcj1Gb3VldCAgTy47IGNpdGF0aW9uX2F1dGhvcj1XaW5ja2VyICBQLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz01MTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYWNhbyBkb21lc3RpY2F0aW9uIElJOiBwcm9nZW5pdG9yIGdlcm1wbGFzbSBvZiB0aGUgVHJpbml0YXJpbyBjYWNhbyBjdWx0aXZhcjsgY2l0YXRpb25fYXV0aG9yPU1vdGFtYXlvciAgSi5DLjsgY2l0YXRpb25fYXV0aG9yPVJpc3RlcnVjY2kgIEEuTS47IGNpdGF0aW9uX2F1dGhvcj1IZWF0aCAgTS47IGNpdGF0aW9uX2F1dGhvcj1MYW5hdWQgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTMyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBRVExzIGZvciByZXNpc3RhbmNlIHRvIGZyb3N0eSBwb2QgYW5kIGJsYWNrIHBvZCBkaXNlYXNlcyBhbmQgaG9ydGljdWx0dXJhbCB0cmFpdHMgaW4gVGhlb2Jyb21hIGNhY2FvIEw7IGNpdGF0aW9uX2F1dGhvcj1Ccm93biAgSi5TLjsgY2l0YXRpb25fYXV0aG9yPVBoaWxsaXBzLU1vcmEgIFcuOyBjaXRhdGlvbl9hdXRob3I9UG93ZXIgIEUuSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTE4NTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wb3NpdGUgbGlua2FnZSBtYXAgZnJvbSB0aHJlZSBjcm9zc2VzIGJldHdlZW4gY29tbWVyY2lhbCBjbG9uZXMgb2YgY2FjYW8sIFRoZW9icm9tYSBjYWNhbyBMOyBjaXRhdGlvbl9hdXRob3I9QnJvd24gIEouOyBjaXRhdGlvbl9hdXRob3I9U2F1dHRlciAgUi47IGNpdGF0aW9uX2F1dGhvcj1PbGFubyAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyb3AuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMjAtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiBUaGVvYnJvbWEgY2FjYW8gTDsgY2l0YXRpb25fYXV0aG9yPUxhbmF1ZCAgQy47IGNpdGF0aW9uX2F1dGhvcj1SaXN0ZXJ1Y2NpICBBLjsgY2l0YXRpb25fYXV0aG9yPU4nZ29yYW4gIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9OTg3LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvNC8yNzkvOTk3NDQ3L2RzdjAwOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlb2Jyb21hIGNhY2FvLCB0aGUga2V5IGluZ3JlZGllbnQgaW4gY2hvY29sYXRlIHByb2R1Y3Rpb24sIGlzIG9uZSBvZiB0aGUgd29ybGQncyBtb3N0IGltcG9ydGFudCB0cmVlIGZydWl0IGNyb3BzLCB3aXRoIOKIvDQsMDAwLDAwMCBtZXRyaWMgdG9ucyBwcm9kdSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ4MDg0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZCWfRFX5c61SUjWmE21O0s7Gjf02" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f279%2f348084" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f279%2f348084" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13443" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&amp;Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/4">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4938938">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4938938">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4938940">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4938940">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4938945">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4938945">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4938967">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4938967">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4938996">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4938996">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4938998">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4938998">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4939000">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4939000">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4939002">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4939002">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4939004">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4939004">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/4/269/348206">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/4/293/348350">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Making a chocolate chip: development and evaluation of a 6K SNP array for <em>Theobroma cacao</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Donald Livingstone</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Mars, Incorporated</div>, <div class="addr-line">McLean, VA 22101</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Donald+Livingstone">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Livingstone D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Livingstone D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Donald Livingstone</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stefan Royaert</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Mars, Incorporated</div>, <div class="addr-line">McLean, VA 22101</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stefan+Royaert">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Royaert S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Royaert S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stefan Royaert</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Conrad Stack</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Mars, Incorporated</div>, <div class="addr-line">McLean, VA 22101</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Conrad+Stack">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stack C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Stack C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Conrad Stack</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keithanne Mockaitis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biology</div>, <div class="insititution">Indiana University</div>, <div class="addr-line">Bloomington, IN 47405</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keithanne+Mockaitis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mockaitis K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mockaitis K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Keithanne Mockaitis</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Greg May</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Center for Genome Resources</div>, <div class="addr-line">Santa Fe, NM 87505</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Greg+May">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=May G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;May G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Greg May</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew Farmer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Center for Genome Resources</div>, <div class="addr-line">Santa Fe, NM 87505</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+Farmer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Farmer A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Farmer A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andrew Farmer</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christopher Saski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Clemson University Genomics and Computational Biology Laboratory, Institute for Translational Genomics</div>, <div class="addr-line">Clemson, SC 29634</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christopher+Saski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saski C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saski C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christopher Saski</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ray Schnell</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Mars, Incorporated</div>, <div class="addr-line">McLean, VA 22101</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ray+Schnell">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schnell R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schnell R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ray Schnell</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David Kuhn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">United States Department of Agriculture-Agriculture Research Service</div>, <div class="insititution">Subtropical, Horticulture Research Station</div>, <div class="addr-line">Miami, FL 33158</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+Kuhn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuhn D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuhn D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">David Kuhn</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Juan Carlos Motamayor</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Mars, Incorporated</div>, <div class="addr-line">McLean, VA 22101</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 786-573-7091. Fax. +1 786-573-7102. E-mail: <a href="mailto:juan.motamayor@effem.com" target="_blank">juan.motamayor@effem.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Juan+Carlos+Motamayor">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Motamayor J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Motamayor J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Juan Carlos Motamayor</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 279–291, <a href='https://doi.org/10.1093/dnares/dsv009'>https://doi.org/10.1093/dnares/dsv009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 June 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 December 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 May 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348084" href="/dnaresearch/article-pdf/22/4/279/997447/dsv009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Donald Livingstone, Stefan Royaert, Conrad Stack, Keithanne Mockaitis, Greg May, Andrew Farmer, Christopher Saski, Ray Schnell, David Kuhn, Juan Carlos Motamayor;  Making a chocolate chip: development and evaluation of a 6K SNP array for <em>Theobroma cacao</em>, <em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 279–291, <a href='https://doi.org/10.1093/dnares/dsv009'>https://doi.org/10.1093/dnares/dsv009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348084?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348084?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348084?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348084?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348084?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348084?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348084?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Making%20a%20chocolate%20chip%3A%20development%20and%20evaluation%20of%20a%206K%20SNP%20array%20for%20%26lt%3Bem%26gt%3BTheobroma%20cacao%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2015-06-11&amp;volumeNum=22&amp;issueNum=4&amp;author=Livingstone%2C%20Donald%3B%20Royaert%2C%20Stefan&amp;startPage=279&amp;endPage=291&amp;contentId=10.1093%2Fdnares%2Fdsv009&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f279%2f348084" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4938938 id="4938938" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Theobroma cacao</em>, the key ingredient in chocolate production, is one of the world's most important tree fruit crops, with ∼4,000,000 metric tons produced across 50 countries. To move towards gene discovery and marker-assisted breeding in cacao, a single-nucleotide polymorphism (SNP) identification project was undertaken using RNAseq data from 16 diverse cacao cultivars. RNA sequences were aligned to the assembled transcriptome of the cultivar Matina 1-6, and 330,000 SNPs within coding regions were identified. From these SNPs, a subset of 6,000 high-quality SNPs were selected for inclusion on an Illumina Infinium SNP array: the Cacao6kSNP array. Using Cacao6KSNP array data from over 1,000 cacao samples, we demonstrate that our custom array produces a saturated genetic map and can be used to distinguish among even closely related genotypes. Our study enhances and expands the genetic resources available to the cacao research community, and provides the genome-scale set of tools that are critical for advancing breeding with molecular markers in an agricultural species with high genetic diversity.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="cacao">cacao</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mapping">mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="breeding">breeding</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="markers">markers</a></div>                            <h2 scrollto-destination=4938940 id="4938940" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The beans of <em>Theobroma cacao</em>, prized worldwide for their cocoa solids and cocoa butter that serve as the defining ingredient of chocolate, are now commercially produced in over 50 countries,<strong><sup><a reveal-id="DSV009C1" data-open="DSV009C1" class="link link-ref link-reveal xref-bibr">1</a></sup></strong> including tropical regions of Central and South America, Asia and Africa. Although the species was initially domesticated from wild ancestors in Central America and Mexico,<sup><a reveal-id="DSV009C2  DSV009C4" data-open="DSV009C2  DSV009C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> modern efforts to improve cacao were initiated in the early 1900s,<sup><a reveal-id="DSV009C5" data-open="DSV009C5" class="link link-ref link-reveal xref-bibr">5</a></sup> with the classification of cacao germplasm into three major morphological/geographic groups: Criollo, Forastero and Trinitario. Only since the advent of molecular marker technology, however, has modern molecular breeding become possible for cacao. Using the precision afforded by such molecular markers, Motamayor et al.<sup><a reveal-id="DSV009C6" data-open="DSV009C6" class="link link-ref link-reveal xref-bibr">6</a></sup> more accurately classified cacao germplasm into 10 major genetic clusters: Marañon, Curaray, Criollo, Iquitos, Nanay, Contamana, Amelonado, Purús, Nacional and Guiana. The tremendous genetic diversity extant within the cacao species represents a key resource for the improvement of flavour, yield and disease resistance. Moreover, breeders can exploit the heterosis that occurs in crosses between trees of different genetic groups to improve production.<sup><a reveal-id="DSV009C7" data-open="DSV009C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Crop improvement programs rely on the accurate characterization of genetic resources (germplasm) for breeding and propagation. Historically, germplasm collections were curated based on morphology and the agronomic characteristics of individual clones.<sup><a reveal-id="DSV009C8" data-open="DSV009C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSV009C9" data-open="DSV009C9" class="link link-ref link-reveal xref-bibr">9</a></sup> More recently, however, efforts have been underway to assess, describe and characterize cacao collections based on population structure, genetic diversity and evolutionary relationships using molecular markers.<sup><a reveal-id="DSV009C6" data-open="DSV009C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV009C10  DSV009C12" data-open="DSV009C10  DSV009C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> In addition to assisting in the curation of existing cacao germplasm collections, a key goal of an integrated cacao genetics program is to use molecular markers to select for improved varieties and maintain these in breeding programs.<sup><a reveal-id="DSV009C11" data-open="DSV009C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV009C13  DSV009C19" data-open="DSV009C13  DSV009C19" class="link link-ref link-reveal xref-bibr">13–19</a></sup></p><p>Although simple sequence repeats (SSRs) have served as genetic markers in most cacao studies to date,<sup><a reveal-id="DSV009C6" data-open="DSV009C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV009C11" data-open="DSV009C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV009C20" data-open="DSV009C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSV009C21" data-open="DSV009C21" class="link link-ref link-reveal xref-bibr">21</a></sup> SSR data can exhibit platform-to-platform variation, which complicates the sharing of genotype data among different research groups.<sup><a reveal-id="DSV009C22" data-open="DSV009C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The recent sequencing of two cacao genomes (Criollo B97-61/B2<sup><a reveal-id="DSV009C23" data-open="DSV009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and Matina 1-6<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup>) has substantially increased the number of resources available to cacao researchers for the identification of novel molecular markers. The most abundant marker type is the single-nucleotide polymorphism (SNP). The use of SNPs has revolutionized genetic research in many species, including cacao.<sup><a reveal-id="DSV009C25  DSV009C28" data-open="DSV009C25  DSV009C28" class="link link-ref link-reveal xref-bibr">25–28</a></sup> SNPs are detected by aligning and comparing sequence data from genetically diverse individuals.<sup><a reveal-id="DSV009C29  DSV009C31" data-open="DSV009C29  DSV009C31" class="link link-ref link-reveal xref-bibr">29–31</a></sup> In cacao, ∼1,300 SNPs have been discovered and used for comparative genomic studies, production of consensus genetic maps, marker-assisted breeding and for curation of germplasm collections.<sup><a reveal-id="DSV009C22" data-open="DSV009C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSV009C32  DSV009C36" data-open="DSV009C32  DSV009C36" class="link link-ref link-reveal xref-bibr">32–36</a></sup></p><p>The ease of use and abundance of SNPs make them particularly well suited for generating saturated linkage maps. SNP-based linkage maps have been built for a variety of tree crops, including cacao.<sup><a reveal-id="DSV009C22" data-open="DSV009C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSV009C37" data-open="DSV009C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In the present study, we describe the discovery of hundreds of thousands of SNPs using next-generation sequencing, laying the foundation for future genetic mapping studies. We present results from the design and use of a 6k Illumina Infinium SNP array and demonstrate that the array is useful for the generation of a saturated genetic linkage map and the differentiation of two closely related cacao accessions.</p>                            <h2 scrollto-destination=4938945 id="4938945" class="section-title" data-legacy-id=s2>2. Methods</h2>
                            <h3 scrollto-destination=4938946 id="4938946" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>The 16 cacao accessions making up the diversity panel used here for SNP discovery (Fig. <a reveal-id="DSV009F1" data-open="DSV009F1" class="link link-reveal link-table xref-fig">1</a>) represent the 10 distinct genetic clusters of cacao identified by Motamayor et al.<sup><a reveal-id="DSV009C6" data-open="DSV009C6" class="link link-ref link-reveal xref-bibr">6</a></sup> All plant tissues were collected at the USDA-ARS SHRS germplasm collection in Miami except EBC-148, which was from the Centro Agronomico Tropical de Investigacion y Ensenanza (CATIE), Turrialba, Costa Rica. Mature pistil tissue from unpollinated flowers of UF273 Type I was also collected at CATIE. Tissue samples from CATIE were placed into RNAlater (Life Technologies, NY, USA) for stable shipment prior to RNA extraction. RNA samples were used for transcript sequencing and SNP calling. </p>                        <a id="4938948" scrollto-destination="4938948"></a>
<div id="DSV009F1" data-id="DSV009F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00901.png?Expires=2147483647&Signature=viFnjRiGQMLEiV9eFmKRFNhTuFpG8RfFWcDZkQ0OPOLdfysa6Xd1ouE3~6YEqJqa4DLfGV9osly2D9qwi9yp6J-Sh8QMV~ENVXU6H4bnl4cAJVlawRVwrBO8GuaGVlMFiBhjwJ4a1T0CYzkgu613AGSJ-CRlATZ2uwAIwhNGPpPOOdD8Umh~y7WzfvqMmBxK-1HM5pB-JXlcw-OK0~BvY-wkTzEJ4PQQSlGfDxSykjRyC2my7wHj2yvj7YLSVMimQASzqGbScuHwaYiCaDAkLSeLDtIHmTVBNM9On-q7i40Hlz80ONV5TVaXW5kCMmqps-9hFuQ72nXQpt7gXDJVng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNP discovery and filtering for the selection of 6,000 SNPs for inclusion on the Cacao6kSNP array. Variants identified by alignment to an early Matina reference transcriptome. Standard filtering was applied to provide confidence in the existence of the variants. SNP Filter was applied to retain only biallelic SNP variants, referred to as the Original SNP Report. As a measure of added confidence, only those SNPs for which at least two accessions displayed the variant allele and two accessions displayed the reference allele were kept (Selected Filtered SNPs). Additional loci filtering was applied to reduce the number of selected SNPs down to the targeted final filtered SNPs, which were included on the Cacao6kSNP array." path-from-xml="dsv00901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938948/dsv00901.tiff" path-from-xml="dsv00901.tiff" target="_blank">View large</a><a section="4938948" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00901.png?Expires=2147483647&Signature=4j-eSqwDIsBb19wnJVqVBlBf6QenB8f9S~8jVIkiprL1avxCiEIeE25zCbjhnV4kG752HOrOOIkAxq-UHYFN1jH-AmCWHk76JHVFyK1hwMjmMaeV4F3p4Pjx0dRAvrivRe8jw2KhQYkLJO2mU8MmZoatG8aQPsvyhmhst~ihktoRD~t5DjHJdWe2BgtWNspTvhrQljGJpeZzaLcVLtDbh8Gk2d0em1g4r~IbWYgPfJlvma3NBcHVRIgxNJjSaPCjJ3Fu4TlFEkZWzeLqbifjh5rP-khOXUx-tgP2wNEZriuNxQWOG3pOMIWGoiTi8vzg2cG3z2yqHF0G52an6mvBEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938948&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNP discovery and filtering for the selection of 6,000 SNPs for inclusion on the Cacao6kSNP array. Variants identified by alignment to an early Matina reference transcriptome. Standard filtering was applied to provide confidence in the existence of the variants. SNP Filter was applied to retain only biallelic SNP variants, referred to as the Original SNP Report. As a measure of added confidence, only those SNPs for which at least two accessions displayed the variant allele and two accessions displayed the reference allele were kept (Selected Filtered SNPs). Additional loci filtering was applied to reduce the number of selected SNPs down to the targeted final filtered SNPs, which were included on the Cacao6kSNP array.</p></div></div><div content-id="DSV009F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00901.png?Expires=2147483647&Signature=viFnjRiGQMLEiV9eFmKRFNhTuFpG8RfFWcDZkQ0OPOLdfysa6Xd1ouE3~6YEqJqa4DLfGV9osly2D9qwi9yp6J-Sh8QMV~ENVXU6H4bnl4cAJVlawRVwrBO8GuaGVlMFiBhjwJ4a1T0CYzkgu613AGSJ-CRlATZ2uwAIwhNGPpPOOdD8Umh~y7WzfvqMmBxK-1HM5pB-JXlcw-OK0~BvY-wkTzEJ4PQQSlGfDxSykjRyC2my7wHj2yvj7YLSVMimQASzqGbScuHwaYiCaDAkLSeLDtIHmTVBNM9On-q7i40Hlz80ONV5TVaXW5kCMmqps-9hFuQ72nXQpt7gXDJVng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNP discovery and filtering for the selection of 6,000 SNPs for inclusion on the Cacao6kSNP array. Variants identified by alignment to an early Matina reference transcriptome. Standard filtering was applied to provide confidence in the existence of the variants. SNP Filter was applied to retain only biallelic SNP variants, referred to as the Original SNP Report. As a measure of added confidence, only those SNPs for which at least two accessions displayed the variant allele and two accessions displayed the reference allele were kept (Selected Filtered SNPs). Additional loci filtering was applied to reduce the number of selected SNPs down to the targeted final filtered SNPs, which were included on the Cacao6kSNP array." path-from-xml="dsv00901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938948/dsv00901.tiff" path-from-xml="dsv00901.tiff" target="_blank">View large</a><a section="4938948" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00901.png?Expires=2147483647&Signature=4j-eSqwDIsBb19wnJVqVBlBf6QenB8f9S~8jVIkiprL1avxCiEIeE25zCbjhnV4kG752HOrOOIkAxq-UHYFN1jH-AmCWHk76JHVFyK1hwMjmMaeV4F3p4Pjx0dRAvrivRe8jw2KhQYkLJO2mU8MmZoatG8aQPsvyhmhst~ihktoRD~t5DjHJdWe2BgtWNspTvhrQljGJpeZzaLcVLtDbh8Gk2d0em1g4r~IbWYgPfJlvma3NBcHVRIgxNJjSaPCjJ3Fu4TlFEkZWzeLqbifjh5rP-khOXUx-tgP2wNEZriuNxQWOG3pOMIWGoiTi8vzg2cG3z2yqHF0G52an6mvBEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938948&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNP discovery and filtering for the selection of 6,000 SNPs for inclusion on the Cacao6kSNP array. Variants identified by alignment to an early Matina reference transcriptome. Standard filtering was applied to provide confidence in the existence of the variants. SNP Filter was applied to retain only biallelic SNP variants, referred to as the Original SNP Report. As a measure of added confidence, only those SNPs for which at least two accessions displayed the variant allele and two accessions displayed the reference allele were kept (Selected Filtered SNPs). Additional loci filtering was applied to reduce the number of selected SNPs down to the targeted final filtered SNPs, which were included on the Cacao6kSNP array.</p></div></div><p>A total of 1,152 cacao DNA samples were collected and submitted to Illumina for genotyping. These samples consisted of trees from the USDA cacao collection in Miami, including 9 of the 16 accessions from the diversity panel described above. Also included were five separate F1 mapping populations: Pound 7 × UF273 Type I (181 trees, CATIE Type 1), Pound 7 × UF273 Type II (68 trees, CATIE Type 2), KA2-101 × K82 and reciprocal (340 trees, PNG) and TSH1188 × CCN51 (498 trees, MCCS). The remaining 56 samples represent duplicated controls (36 samples) and select germplasm lines (20 samples). Leaves from the CATIE Type 1 and Type 2 populations were collected in CATIE, samples from PNG 1 and PNG 8 came from the Cacao and Coconut Institute in Papua New Guinea and the MCCS population came from the Mars Center for Cacao Science in Brazil.</p>                            <h3 scrollto-destination=4938950 id="4938950" class="section-title" data-legacy-id=s2b>2.2. RNA extraction for SNP discovery</h3>
<p>Following a modified form of the method described by Chang et al.,<sup><a reveal-id="DSV009C38" data-open="DSV009C38" class="link link-ref link-reveal xref-bibr">38</a></sup> RNA was extracted from leaf tissue for all cacao accessions in the diversity panel, with the exception of UF273 Type I, for which pistil tissue was used. A total of 1 g of leaf tissue (excluding the mid-rib) was collected from each tree and placed immediately in liquid nitrogen, then ground to a fine powder and homogenized in 15 ml of 65°C extraction buffer<sup><a reveal-id="DSV009C39" data-open="DSV009C39" class="link link-ref link-reveal xref-bibr">39</a></sup> using a PRO200 Homogenizer with a 10 × 115 mm generator (PRO Scientific, Inc., CT, USA). Pistil tissue was first dissected from mature, unopened flowers in RNAlater with a scalpel and dissecting microscope. Approximately 20 dissected pistils were ground in liquid nitrogen, homogenized and then treated identically as leaf samples. All samples were extracted twice with equal volumes chloroform (1 min homogenization with the PRO200 Homogenizer and 30 min centrifugation at 10,500 × <em>g</em>). One-third volume of 8 M LiCl was then added, the sample mixed and the RNA precipitated overnight or over the weekend at 0°C (in an ice water bath in a 4°C refrigerator). The RNA pellet was obtained via centrifugation at 10,500 × <em>g</em> for 1 h at 4°C. The pellet was then processed with the Qiagen RNeasy Mini Kit (Qiagen, CA, USA) following the manufacturer's instructions. The clean RNA was treated with DNase by adding 100 µl water, 20 µl DNase buffer and 4 µl DNase I (Roche, IN, USA) and incubating for 30 min. RNA was extracted with 180 µl phenol : chloroform : isoamyl alcohol (25 : 24 : 1) with centrifugation at 14,000 × <em>g</em> for 10 min at 4°C. Next, 20 µl 3M NaOAc and 600 µl 100% ethanol were added to the supernatant and left to precipitate overnight at −80°C. The pellet was obtained via centrifugation at 14,000 × <em>g</em> for 30 min at 4°C and then rinsed with 500 µl 80% EtOH. The dried pellet was resuspended in 50 µl RNase-free water.</p>                            <h3 scrollto-destination=4938952 id="4938952" class="section-title" data-legacy-id=s2c>2.3. DNA extraction for genotyping</h3>
<p>Purification of genomic DNA from <em>T. cacao</em> is difficult and not standardized for use in automated DNA extraction pipelines because of the high concentrations of carbohydrates, polyphenolic compounds and other secondary metabolites that oxidize nucleic acids. Here, ∼300 mg of tissue was ground to a powder in liquid nitrogen and immediately put in 1 ml of an inert wash buffer containing final concentrations of 100 mM Hepes, 0.1% PVP 40 (w/v) and 4% 2-mercapto-ethanol in 1.2 ml centrifuge tubes. The tubes were vortexed on a medium setting for 5 min and centrifuged at 8,400 × <em>g</em> for 5 min, decanted and washed 4–6 more times until the supernatant was no longer viscous. After the final wash, the pellet was suspended in nuclei extraction buffer (15% sucrose, 50 mM Tris–HCl, 50 mM EDTA pH 8.0 and 500 mM NaCl) by vortexing. The solution was incubated at 50°C for 15 min, mixing gently every 2 min and then centrifuged at 8,400 × <em>g</em> for 5 min. The supernatant was decanted, and the pelleted plant material resuspended in 450 µl of 20 mM Tris–HCl with 10 mM EDTA. Next, 80 µl of 10% SDS was added, and the samples were incubated for 15 min at 70°C. Samples were allowed to cool to room temperature and then mixed with 300 µl of 7.5 M NH<sub>4</sub>OAc (stored at −20°C) and incubated on ice for 30 min. The solution was centrifuged at top speed for 15 min at 4°C. The aqueous layer was transferred to a clean 1.2-ml tube, combined with an equal amount of isopropanol, and centrifuged at top speed for 15 min at 4°C. The DNA pellet was washed two times with ice-cold 70% ETOH and resuspended in 100 µl of 10 mM Tris–HCl.</p>                            <h3 scrollto-destination=4938954 id="4938954" class="section-title" data-legacy-id=s2d>2.4. Creation of the transcriptome reference sequence</h3>
<p>Total leaf RNA of Matina 1-6 (800 ng) was used for a library preparation optimized for Roche/454 GS FLX Titanium sequencing described previously (NCBI SRX172965<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSV009C40" data-open="DSV009C40" class="link link-ref link-reveal xref-bibr">40</a></sup>). High-quality sequence reads (1,510,557) were trimmed and assembled using Newbler v2.0 into 29,787 contigs of average length 725 (SD = 558).<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h3 scrollto-destination=4938956 id="4938956" class="section-title" data-legacy-id=s2e>2.5. SNP discovery in the diversity panel</h3>
<p>The Matina 1-6 leaf transcriptome assembly was used as a reference sequence for SNP discovery among the members of a diversity panel that contains 16 accessions. mRNA isolated from each accession was used in double-stranded Illumina libraries after two rounds of oligo dT selection from total RNA. Three hundred base-pair library fragments were agarose gel purified, amplified by PCR and quality assessed using a Bioanalyzer 2100 (Agilent, Inc., Santa Clara, CA, USA). Libraries were sequenced on the Illumina GAII to 54 nt length using protocols of the manufacturer (Illumina, San Diego, CA, USA). Sequence reads are deposited in the short read archive (SRA) of NCBI under BioProject PRJNA51633 (Accession no. SRX278177 and SRX278006).</p><p>Sequence variants were identified by aligning the Illumina reads to the Matina 1-6 leaf transcriptome using the Alpheus analysis pipeline detailed by Miller et al.<sup><a reveal-id="DSV009C41" data-open="DSV009C41" class="link link-ref link-reveal xref-bibr">41</a></sup> In brief, paired-end reads were aligned to the reference using the GSNAP aligner configured to allow up to 5% mismatch across the paired read considered as a whole. The data were filtered to reduce false positives. Only variants within reads that aligned uniquely to the reference were considered. Unique mapping ranged among accessions of the diversity panel from ∼20 to 45% of total reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S1</a>). Sequence read data quality and depth standards (Fig. <a reveal-id="DSV009F1" data-open="DSV009F1" class="link link-reveal link-table xref-fig">1</a>, standard filter) required the variant to be present in at least two reads covering the base position, present in at least 20% of the total number of reads covering the variant, and to possess a sequence quality score of Q20 or higher. A variant report was generated from the Alpheus output, with SNPs named with respect to individual Matina 1-6 transcriptome contigs.</p>                            <h3 scrollto-destination=4938959 id="4938959" class="section-title" data-legacy-id=s2f>2.6. Selection of sequences for genotyping array</h3>
<p>Additional filtering was applied to the variants in a stepwise manner to optimize selection and further reduce markers to the number appropriate for the array. Only bi-allelic SNPs were retained after sequence quality and depth filtering: all insertion/deletions (indels) and multi-allelic polymorphisms were excluded. The Original SNP Report contained ∼340 K sequences after these exclusions (Fig. <a reveal-id="DSV009F1" data-open="DSV009F1" class="link link-reveal link-table xref-fig">1</a>). The next major filtering step considered data of the diversity panel as a whole. Here, SNPs were retained only if the reference (Matina 1-6) allele appeared in two accessions and the alternative allele also appeared in two accessions. This filtering step provided selection confidence against potential artifacts of single RNA samples and resulted in the list of 48,408 termed Selected Filtered SNPs (Fig. <a reveal-id="DSV009F1" data-open="DSV009F1" class="link link-reveal link-table xref-fig">1</a>). Each potential marker was evaluated on the basis of its internal sequence with each sequence subjected to review of their expected performance in the Illumina Infinium technology. First, all markers in the Selected Filtered SNP set were submitted to Illumina and assigned an Assay Design Tool score. This score predicts the success of an SNP assay on the array by taking into account the GC content and sequence adjacent to the SNP site and assigning a value from zero to one. Only SNPs with an Illumina Infinium Assay Design Tool score &gt;0.9 were included in the Final Filtered SNPs.</p><p>Furthermore, minor allele frequency (MAF) was calculated for each locus. Markers with a MAF &gt;0.2 were selected for the Final Filtered set. Sequences surrounding the SNPs were screened for the presence of intron–exon junctions using BLASTn on a preliminary Matina 1-6 genome assembly<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and any containing a putative junction site was removed. Because the array was to be used on multiple mapping populations, SNPs heterozygous in at least one, but not all, diversity panel accessions were selected for inclusion on the array. As the Infinium array uses a primer extension bead-based array for genotyping, the number of markers that can be included depends on the bead type of each marker, which in turn is dependent on the nucleotides being assayed. Markers that distinguish A/T or C/G alleles require two beads. To increase the number of loci represented, only single bead-type SNPs (A/C, A/G, T/C, T/G) were chosen for inclusion on the Cacao6kSNP array. Finally, when possible, two SNPs per transcript were selected in an effort to ensure locus representation should one marker fail.</p>                            <h3 scrollto-destination=4938962 id="4938962" class="section-title" data-legacy-id=s2g>2.7. SNP evaluation and linkage mapping</h3>
<p>Data generated from all 1,152 samples were analysed with GenomeStudio (Illumina), and genotypes were called using a GenCall threshold of 0.15. Additionally, to serve as a quality control for genotype calling, only markers with GenTrain and GenCall 10% scores above 0.4 and 0.2, respectively, were included for data analysis. The data were exported and separated into populations, with germplasm and diversity panel accessions (grouped per genetic affinity) comprising their own ‘populations'. For genetic mapping, the markers present in each dataset were reduced so that only SNPs showing segregation in one or both of the parental lines remained.</p><p>Pair-wise Identity by descent (IBD) values were estimated with PLINK v1.07<sup><a reveal-id="DSV009C42" data-open="DSV009C42" class="link link-ref link-reveal xref-bibr">42</a></sup> with 5,149 SNPs for two mapping populations: CATIE Type 1 (Pound 7 × UF273 Type I) and CATIE Type 2 (Pound 7 × UF273 Type II), with 180 and 69 progenies, respectively. The total number of pair-wise calculations including the three parents (Pound 7, UF273 Type I and UF273 Type II) is 31,626. A pedigree file was used to allow PLINK to relate the two populations correctly. Assignments of ‘parent-offspring’, ‘full-sibling’, ‘half-sibs’ (i.e. CATIE Type 1 versus CATIE Type 2 progenies) and ‘other-related’ (i.e. mother/father) were assigned. The IBD distribution was plotted with R.<sup><a reveal-id="DSV009C43" data-open="DSV009C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>A conservative estimation of the number of SNP loci needed to differentiate related cacao accessions (i.e. determine off-types) was performed. The genotyping error rate was calculated from five replicated DNA samples of Pound 7, UF273 Type I and UF273 Type II included on the Cacao6kSNP array. The genotyping error rate was small for all three parents, with UF273 Type II samples exhibiting the highest number of non-concordant genotypes with 14 out of 5,149 (0.00272). To be conservative, we rounded the calculated genotyping error rate of 0.00272 to 0.01 mistypes per locus, and the probability of differentiating any two cacao accessions was determined from the binomial distribution. As direct siblings are likely to share many alleles, the probability of detecting an off-type between any two pair of individuals was defined as the probability of observing more genotype differences between them than expected with a genotyping error rate of 0.01. We calculated this probability for different numbers of SNPs sampled and for three different levels of relatedness (i.e. expected number of genotype differences). The proportion of differing loci between the closely related accessions UF273 Type I and UF273 Type II (0.162) was used as a reference point likely to be encountered in practical situations. Additionally, proportion values, or levels of relatedness, of 0.1 and 0.3 were also included to indicate the number of SNP loci that might be needed to differentiate more or less related accessions.</p><p>A genetic linkage map was constructed using the CATIE Type 2 mapping population. This population resulted from the cross of Pound 7 × UF273 Type II and consists of 68 individual trees. The initial mapping dataset included 2,857 SNPs that demonstrated segregation in the population. To generate the linkage map, the JoinMap 4.1 software (Kyazma, the Netherlands) was used by first converting the raw genotype data into the JoinMap format using the parental genotypes to define the segregation type and then assigning the appropriate allele code based upon the genotype data.<sup><a reveal-id="DSV009C44" data-open="DSV009C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Markers with a significant amount of segregation distortion (15 SNPs) and with &gt;10% missing data (11 SNPs) were removed from the linkage mapping analysis. Marker grouping was selected with a recombination frequency of 0.1 or higher, resulting in the formation of 10 linkage groups (one for each chromosome of cacao). Mapping itself was performed using maximum likelihood analysis. The fit and stress values, a measure of linkage strength, of the markers on each linkage group were examined, and any markers with a value &gt;7 were removed. The linkage mapping was re-iterated until all markers had values &lt;7. The small size of the CATIE Type 2 population contributed to a large number of markers with high fit and stress values. In total, 242 SNPs were removed due to fit and stress values above the stated threshold, resulting in a final linkage map consisting of 68 individuals with 2,589 SNPs.</p>                            <h2 scrollto-destination=4938967 id="4938967" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4938968 id="4938968" class="section-title" data-legacy-id=s3a>3.1. Filtering and SNP statistics</h3>
<p>Among the transcriptome reference sequences built for Matina 1-6 genome annotation efforts,<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> a Matina 1-6 leaf transcriptome assembly was used for the SNP discovery described here. This transcriptome represented ∼22 Mb of sequence in 29,787 isotigs.<sup><a reveal-id="DSV009C40" data-open="DSV009C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Paired-end Illumina GAII reads for 16 highly diverse cacao accessions were obtained and mapped as described (SRA BioProject PRJNA51633). <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> summarizes the read coverage against the transcriptome assembly for each accession used for SNP calling. Unique mapped reads (red) were used to call SNPs. The repetitive (green) reads had at least one end of the read-pair map to multiple places within the transcriptome, whereas the unmapped (blue) reads failed to map under the described thresholds (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). For each accession, the average number of reads per final filtered SNP selected for the Cacao6kSNP array ranged from 13 to 67 reads per SNP (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>The original SNP report identified 339,330 SNPs, which were reduced to 48,408 selected filtered SNPs, and finally to 6,000 SNPs selected for inclusion on the Cacao6kSNP array (Fig. <a reveal-id="DSV009F1" data-open="DSV009F1" class="link link-reveal link-table xref-fig">1</a>). Figure <a reveal-id="DSV009F2" data-open="DSV009F2" class="link link-reveal link-table xref-fig">2</a> plots the location along the cacao genome of the selected filtered (middle ring) and the final filtered (inner ring) SNPs. Originally designed to be broadly applicable to various germplasm collections and mapping populations, our filtering methods favoured high-quality SNPs with large minor allele frequencies across the set, rather than SNPs specific to any one genotype. Although rare SNPs may identify a specific genetic background or unusual phenotype in germplasm collections, they are likely to be homozygous and hence unmappable in mapping populations. When possible, redundancy of marker usefulness among genotypes was preferred. The inclusion of two SNPs per contig that met all the filtering criteria was a measure of precaution against failed genotyping at any one locus. This precaution was unnecessary as 5,149 of the 5,388 successfully synthesized beads (96%) yielded reliable genotype data. </p>                        <a id="4938971" scrollto-destination="4938971"></a>
<div id="DSV009F2" data-id="DSV009F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00902.png?Expires=2147483647&Signature=tM8zV9H4h-sNfbuwoCJc2XltObdNkgaGKyhXJS8OscDDxAeiS4xPHjsrKmDhNG4-u2AXqx~WdXJIDz4GJBsqa5nepGnhmOCy~7bOWLk8YKd68~ini6BZ2JfSGoKAceeuxgS0IH2iKT8LUBUU~JiVfzeQDNlYPEkTz99k4n3IwKlG0EOYt4CP9PANxcTFOilKzj2AnR4Wn6wUQUMrxv8Ug1TALu2s0QNUwUapHvZYguP1vRTADIz13v0QtuQsJ7XJX1ETCnNrP5PR6qhuQ5Ols5xiKXZcpKdbQDLsf41WpWtVS4wCGqfGrJIu2hxRAJZKKdSK29d6psSE2J4YbLg6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Location of SNP sets along the Matina 1-6 genome. The 10 chromosomes of the Matina 1-6 genome assembly are represented along the outer ring (LG01–LG10). The middle ring highlights the locations of 48,408 select filtered SNPs, while the inner ring highlights the locations of the 6,000 final filtered SNPs. This figure is available in black and white in print and in colour at DNA Research online" path-from-xml="dsv00902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938971/dsv00902.tiff" path-from-xml="dsv00902.tiff" target="_blank">View large</a><a section="4938971" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00902.png?Expires=2147483647&Signature=x0RZID-rGhdMBo9oGRod3VTHGwctC4~aAYCsQV~pDyMdut~Ro5BsKNmASeVeF7~z0-h~fo5~fEAX7FnegpJKto6cHm24DzQWcvFNOuKM4KgkJ7U8kRpwtzbCNrVg48QDDzKqV8p4BZhG8BPbUlZuFKkKl4t5WZW05zT4ylp~34xU2qVWDMZ3mCPEA8Oko1Igwze94grjZPVCx7N1euvBYTQGAAWA5MWQLCSNLRrZdqbBXwQF3Gc6HSYpoap8EKtYA~LGpNaeB1MwVMQ~3UmEuLDePnIpjtb0QGSPYbp3sq~6q3Jstsyeu8k21jVS0OuXqT2lS7L1kQB-jan-vbNpag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938971&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Location of SNP sets along the Matina 1-6 genome. The 10 chromosomes of the Matina 1-6 genome assembly are represented along the outer ring (LG01–LG10). The middle ring highlights the locations of 48,408 select filtered SNPs, while the inner ring highlights the locations of the 6,000 final filtered SNPs. This figure is available in black and white in print and in colour at <em>DNA Research</em> online</p></div></div><div content-id="DSV009F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00902.png?Expires=2147483647&Signature=tM8zV9H4h-sNfbuwoCJc2XltObdNkgaGKyhXJS8OscDDxAeiS4xPHjsrKmDhNG4-u2AXqx~WdXJIDz4GJBsqa5nepGnhmOCy~7bOWLk8YKd68~ini6BZ2JfSGoKAceeuxgS0IH2iKT8LUBUU~JiVfzeQDNlYPEkTz99k4n3IwKlG0EOYt4CP9PANxcTFOilKzj2AnR4Wn6wUQUMrxv8Ug1TALu2s0QNUwUapHvZYguP1vRTADIz13v0QtuQsJ7XJX1ETCnNrP5PR6qhuQ5Ols5xiKXZcpKdbQDLsf41WpWtVS4wCGqfGrJIu2hxRAJZKKdSK29d6psSE2J4YbLg6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Location of SNP sets along the Matina 1-6 genome. The 10 chromosomes of the Matina 1-6 genome assembly are represented along the outer ring (LG01–LG10). The middle ring highlights the locations of 48,408 select filtered SNPs, while the inner ring highlights the locations of the 6,000 final filtered SNPs. This figure is available in black and white in print and in colour at DNA Research online" path-from-xml="dsv00902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938971/dsv00902.tiff" path-from-xml="dsv00902.tiff" target="_blank">View large</a><a section="4938971" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00902.png?Expires=2147483647&Signature=x0RZID-rGhdMBo9oGRod3VTHGwctC4~aAYCsQV~pDyMdut~Ro5BsKNmASeVeF7~z0-h~fo5~fEAX7FnegpJKto6cHm24DzQWcvFNOuKM4KgkJ7U8kRpwtzbCNrVg48QDDzKqV8p4BZhG8BPbUlZuFKkKl4t5WZW05zT4ylp~34xU2qVWDMZ3mCPEA8Oko1Igwze94grjZPVCx7N1euvBYTQGAAWA5MWQLCSNLRrZdqbBXwQF3Gc6HSYpoap8EKtYA~LGpNaeB1MwVMQ~3UmEuLDePnIpjtb0QGSPYbp3sq~6q3Jstsyeu8k21jVS0OuXqT2lS7L1kQB-jan-vbNpag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938971&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Location of SNP sets along the Matina 1-6 genome. The 10 chromosomes of the Matina 1-6 genome assembly are represented along the outer ring (LG01–LG10). The middle ring highlights the locations of 48,408 select filtered SNPs, while the inner ring highlights the locations of the 6,000 final filtered SNPs. This figure is available in black and white in print and in colour at <em>DNA Research</em> online</p></div></div><p>In the final filtered set, 4,632 contigs were represented on the Cacao6kSNP array by 6,000 loci. This represents an average of 1.3 SNPs/contig. Because an early transcriptome assembly was used to identify SNPs, many of these contigs were later collapsed into one another during the gene model prediction and annotation stage of genome assembly.<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The final number of gene models represented by the selected SNPs was determined and shows a slight reduction to 4,305 gene models (1.4 SNPs/gene model on average). The gene models represented on the Cacao6kSNP array display a cross-section of cellular components, molecular functions and biological processes as determined via GOslim annotation (Fig. <a reveal-id="DSV009F3" data-open="DSV009F3" class="link link-reveal link-table xref-fig">3</a>). This diversity in the types of genes represented suggests that the SNPs selected for inclusion on the Cacao6kSNP array are likely to have broad impacts on plant physiology, making them desirable for general marker-trait association analysis. Large proportions of genes involved in protein (13%) and nucleotide (10%) binding are represented, along with those with transferase (9%) and hydrolase (8%) activities. The largest biological processes represented include metabolic, cellular and protein metabolism. Of immediate interest to cacao breeding efforts are the SNPs present in gene models involved in stress and abiotic or biotic response (4%) and those linked to developmental processes (2%). </p>                        <a id="4938973" scrollto-destination="4938973"></a>
<div id="DSV009F3" data-id="DSV009F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00903.png?Expires=2147483647&Signature=ftRHgC-VjlV6kttKLQz5xujdnCZ0lKcO4VlV-5FO1mrlOO5hWVTVUeD4dPwsdQlPo16VwBN3biHspPpUKavFH3VZN8mhdUP0hLrDZPHSRf2e8F0ekrOpsQcvkBSEE8qDGPX4DWysS8Ljj4TvxTA7sUgZPpSo6f~T3prND2muFuslKZ~X0ozrAnGXyiMJTzzqmUmW55UuzAPgb4JryAPDGrSMF6nvDorhHM1Ywbe894OM9kvFKM5xqCmaMWd2kdqdX1cN8phhetYltSbpClQWOG6VPJBQWdJETvAi4fZT7EU1hJy0jqdIFzgn7KKIGBmGmIDLRg76Buss8tC5dWwQhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GOslim annotation of SNP-containing gene models. Gene models from the Matina 1-6 assembly were screened to identify the final selected SNP set. Goslim annotation was used to classify the SNP-containing gene models. The top chart represents cellular components, the middle describes molecular function and the bottom refers to biological process. The X-axis shows the total number of SNP-containing gene models. The percentage after the bar represents the percent of SNP-containing gene models within each annotated group." path-from-xml="dsv00903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938973/dsv00903.tiff" path-from-xml="dsv00903.tiff" target="_blank">View large</a><a section="4938973" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00903.png?Expires=2147483647&Signature=pVkIKBgHmofXgy56Q8OQMGfS~uVN7~sG6Or4KkGjnUOGJTA8QicT~HgdjlUCibiKYvkLvwFX~nlohyhr9GTAP280t9wXM09cAoyApeu2EjYK92z7pWmwDyCMIaDHfkJi23EaL5Gn9gs-8oIceQMvqBmsNQPgGMBx1y0-1EG~ZSObDyTvTiN46P9eY1W8wD2yUbcJE~DVkHv1LGxijCMmQoYPDG~VT8m6c43aiBasan0L6bA7xcIQISfzGcko6aQ-L70dFsisWqYP2MqPiQ39goKeRjAPxXc3NuwqDE5dUa0tbQzCpHkFP1Jj7JnK-JICr3QDdxz68D04ZHtP0HMOpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938973&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GOslim annotation of SNP-containing gene models. Gene models from the Matina 1-6 assembly were screened to identify the final selected SNP set. Goslim annotation was used to classify the SNP-containing gene models. The top chart represents cellular components, the middle describes molecular function and the bottom refers to biological process. The <em>X</em>-axis shows the total number of SNP-containing gene models. The percentage after the bar represents the percent of SNP-containing gene models within each annotated group.</p></div></div><div content-id="DSV009F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00903.png?Expires=2147483647&Signature=ftRHgC-VjlV6kttKLQz5xujdnCZ0lKcO4VlV-5FO1mrlOO5hWVTVUeD4dPwsdQlPo16VwBN3biHspPpUKavFH3VZN8mhdUP0hLrDZPHSRf2e8F0ekrOpsQcvkBSEE8qDGPX4DWysS8Ljj4TvxTA7sUgZPpSo6f~T3prND2muFuslKZ~X0ozrAnGXyiMJTzzqmUmW55UuzAPgb4JryAPDGrSMF6nvDorhHM1Ywbe894OM9kvFKM5xqCmaMWd2kdqdX1cN8phhetYltSbpClQWOG6VPJBQWdJETvAi4fZT7EU1hJy0jqdIFzgn7KKIGBmGmIDLRg76Buss8tC5dWwQhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GOslim annotation of SNP-containing gene models. Gene models from the Matina 1-6 assembly were screened to identify the final selected SNP set. Goslim annotation was used to classify the SNP-containing gene models. The top chart represents cellular components, the middle describes molecular function and the bottom refers to biological process. The X-axis shows the total number of SNP-containing gene models. The percentage after the bar represents the percent of SNP-containing gene models within each annotated group." path-from-xml="dsv00903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938973/dsv00903.tiff" path-from-xml="dsv00903.tiff" target="_blank">View large</a><a section="4938973" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00903.png?Expires=2147483647&Signature=pVkIKBgHmofXgy56Q8OQMGfS~uVN7~sG6Or4KkGjnUOGJTA8QicT~HgdjlUCibiKYvkLvwFX~nlohyhr9GTAP280t9wXM09cAoyApeu2EjYK92z7pWmwDyCMIaDHfkJi23EaL5Gn9gs-8oIceQMvqBmsNQPgGMBx1y0-1EG~ZSObDyTvTiN46P9eY1W8wD2yUbcJE~DVkHv1LGxijCMmQoYPDG~VT8m6c43aiBasan0L6bA7xcIQISfzGcko6aQ-L70dFsisWqYP2MqPiQ39goKeRjAPxXc3NuwqDE5dUa0tbQzCpHkFP1Jj7JnK-JICr3QDdxz68D04ZHtP0HMOpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938973&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GOslim annotation of SNP-containing gene models. Gene models from the Matina 1-6 assembly were screened to identify the final selected SNP set. Goslim annotation was used to classify the SNP-containing gene models. The top chart represents cellular components, the middle describes molecular function and the bottom refers to biological process. The <em>X</em>-axis shows the total number of SNP-containing gene models. The percentage after the bar represents the percent of SNP-containing gene models within each annotated group.</p></div></div>                            <h3 scrollto-destination=4938974 id="4938974" class="section-title" data-legacy-id=s3b>3.2. SNP array evaluation</h3>
<p>Of the 6,000 SNPs submitted for inclusion on the bead array, 5,388 were successfully synthesized. Failing to produce genotype data for any sample, 171 SNPs were removed before analysis. Removing markers with low GenTrain (&lt;0.4) and GenCall 10% (&lt;0.2) scores resulted in a final set of 5,149 SNPs. The Cacao6kSNP array was originally run with 1,152 DNA samples (including duplicated controls), but seven DNA samples failed completely. In total, 5,149 SNPs were genotyped in 1,145 DNA samples for a total of 5,895,605 data points. Only 6,171 of these data points provided no definitive genotype representing 0.1% missing data within the total analysed dataset. The submitted Cacao6kSNP represented 6,912,000 potential data, of which 85% provided useful genotype data.</p>                            <h3 scrollto-destination=4938976 id="4938976" class="section-title" data-legacy-id=s3c>3.3. Comparison of allele calls from chip array and sequencing</h3>
<p>A cursory survey was performed to compare the initial sequencing-based SNP calls from RNAseq with the genotype data generated from the Cacao6kSNP array. Nine accessions present in both datasets were examined for differences in allele calls (Table <a reveal-id="DSV009TB1" data-open="DSV009TB1" class="link link-reveal link-table xref-fig">1</a>). Concordance between genotypes can be found along the diagonal for each accession and ranged from 86 to 99% across the nine accessions compared (Table <a reveal-id="DSV009TB1" data-open="DSV009TB1" class="link link-reveal link-table xref-fig">1</a>). Overall, 39,399 data points were compared and found to be 93% concordant (36,724 data points). The majority of non-concordance events observed for any particular accession appear to represent differences in the sensitivity of heterozygous allele detection, that is, when one dataset called a marker heterozygous and the other set called the same marker homozygous. Homozygous changes, or those for which the sequencing reads called one homozygous allele and the array called homozygous for the other allele, were much less common. In total, 1,916 SNPs generated differing genotype calls between the Cacao6kSNP array and RNAseq data. This number was striking and represents variation for 37% of the total SNPs genotyped. A closer examination of the differences was performed. The number of non-concordant markers was calculated and sorted based on the number of accessions showing discrepancies for that particular SNP (Table <a reveal-id="DSV009TB2" data-open="DSV009TB2" class="link link-reveal link-table xref-fig">2</a>). This analysis revealed that the bulk (1,433) of the identified non-concordant SNPs differed within a single accession and that 97% (1,383) of the differences observed were heterozygous changes. Visual inspection of these SNPs in GenomeStudio displayed three well-defined clusters for the majority of the markers. Additionally, for discrepant SNPs in the parents of the CATIE mapping population, Pound 7 and UF273, Mendelian segregation was used to infer the correct genotype of the parent. For all instances when either parent was the only accession displaying discrepancies between the Cacao6kSNP array and RNAseq, segregation of the CATIE mapping populations supported the genotype value obtained by the Cacao6kSNP array. In contrast, non-congruent SNPs with three or more accessions showing differences display a larger proportion of opposing homozygotes (24–100%), and the number of discrepant markers dropped to only 136 SNPs for this whole subset (Table <a reveal-id="DSV009TB2" data-open="DSV009TB2" class="link link-reveal link-table xref-fig">2</a>). When SNPs with three or more non-concordant accessions were examined in GenomeStudio, over half showed poor or single cluster formation, indicating issues with the Cacao6kSNP data for these SNPs. Those displaying three well-defined clusters had less than six sequencing reads for the discrepant accessions. Although experiments specifically designed to assess the accuracy of the chip array versus short read sequencing calls are still required, from our data, it appears as though the majority of non-concordant markers are the result of improper allele calls from the use of sequencing reads, based on the strong cluster formation, segregation data and small number of discrepant accessions affected by these SNPs. These issues are possibly the result of low read coverage or strand (allele) bias at the mismatched SNP causing heterozygotes to be misidentified as homozygotes. However, the less common SNPs with the opposing homozygous alleles are more likely issues with the chip array, as evidenced by the increase in poor cluster formation for these SNPs. </p>                        <a id="4938978" scrollto-destination="4938978"></a>
<div content-id="DSV009TB1" class="table-modal"><div class="table-wrap"><div id="DSV009TB1" class="table-wrap-title" data-id="DSV009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of Cacao6kSNP array and sequencing calls across common accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Clone </th><th rowspan="2">Concordant SNPs </th><th rowspan="2">Non-concordant SNPs </th><th rowspan="2">Per cent concordance </th><th rowspan="2"> </th><th rowspan="2"> </th><th colspan="3">RNAseq genotypes<hr /> </th></tr><tr><th>Homozygous reference </th><th>Heterozygous </th><th>Homozygous variant </th></tr></thead><tbody><tr><td> </td><td> </td><td> </td><td> </td><td rowspan="27">Cacao6kSNP Array genotypes </td><td>Homozygous reference </td><td>4,685 </td><td>23 </td><td>3 </td></tr><tr><td>Matina 1-6 </td><td>4,687 </td><td>47 </td><td>99 </td><td>Heterozygous </td><td>2 </td><td>0 </td><td>0 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>9 </td><td>10 </td><td>2 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,477 </td><td>24 </td><td>14 </td></tr><tr><td>Criollo 13 </td><td>4,104 </td><td>117 </td><td>97 </td><td>Heterozygous </td><td>1 </td><td>3 </td><td>1 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>59 </td><td>2,626 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>3,237 </td><td>51 </td><td>6 </td></tr><tr><td>Pound 5 </td><td>4,519 </td><td>169 </td><td>96 </td><td>Heterozygous </td><td>21 </td><td>104 </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>23 </td><td>54 </td><td>1,178 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,712 </td><td>91 </td><td>12 </td></tr><tr><td>GU255 </td><td>4,558 </td><td>211 </td><td>96 </td><td>Heterozygous </td><td>17 </td><td>147 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>8 </td><td>70 </td><td>1,699 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,220 </td><td>14 </td><td>12 </td></tr><tr><td>NAP 30 </td><td>3,685 </td><td>185 </td><td>95 </td><td>Heterozygous </td><td>42 </td><td>182 </td><td>55 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>16 </td><td>46 </td><td>2,283 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>971 </td><td>50 </td><td>14 </td></tr><tr><td>SCA-6 </td><td>3,493 </td><td>414 </td><td>89 </td><td>Heterozygous </td><td>83 </td><td>524 </td><td>144 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>11 </td><td>112 </td><td>1,998 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,103 </td><td>49 </td><td>9 </td></tr><tr><td>Pound 7 </td><td>3,529 </td><td>513 </td><td>87 </td><td>Heterozygous </td><td>245 </td><td>820 </td><td>164 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>12 </td><td>34 </td><td>606 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,304 </td><td>64 </td><td>5 </td></tr><tr><td>EBC 148 </td><td>4,273 </td><td>369 </td><td>92 </td><td>Heterozygous </td><td>118 </td><td>870 </td><td>67 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>97 </td><td>1,099 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,144 </td><td>47 </td><td>14 </td></tr><tr><td>UF273 type I </td><td>3,876 </td><td>650 </td><td>86 </td><td>Heterozygous </td><td>280 </td><td>1,351 </td><td>185 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>13 </td><td>112 </td><td>1,381 </td></tr></tbody></table></div><div content-id="DSV009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Clone </th><th rowspan="2">Concordant SNPs </th><th rowspan="2">Non-concordant SNPs </th><th rowspan="2">Per cent concordance </th><th rowspan="2"> </th><th rowspan="2"> </th><th colspan="3">RNAseq genotypes<hr /> </th></tr><tr><th>Homozygous reference </th><th>Heterozygous </th><th>Homozygous variant </th></tr></thead><tbody><tr><td> </td><td> </td><td> </td><td> </td><td rowspan="27">Cacao6kSNP Array genotypes </td><td>Homozygous reference </td><td>4,685 </td><td>23 </td><td>3 </td></tr><tr><td>Matina 1-6 </td><td>4,687 </td><td>47 </td><td>99 </td><td>Heterozygous </td><td>2 </td><td>0 </td><td>0 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>9 </td><td>10 </td><td>2 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,477 </td><td>24 </td><td>14 </td></tr><tr><td>Criollo 13 </td><td>4,104 </td><td>117 </td><td>97 </td><td>Heterozygous </td><td>1 </td><td>3 </td><td>1 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>59 </td><td>2,626 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>3,237 </td><td>51 </td><td>6 </td></tr><tr><td>Pound 5 </td><td>4,519 </td><td>169 </td><td>96 </td><td>Heterozygous </td><td>21 </td><td>104 </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>23 </td><td>54 </td><td>1,178 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,712 </td><td>91 </td><td>12 </td></tr><tr><td>GU255 </td><td>4,558 </td><td>211 </td><td>96 </td><td>Heterozygous </td><td>17 </td><td>147 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>8 </td><td>70 </td><td>1,699 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,220 </td><td>14 </td><td>12 </td></tr><tr><td>NAP 30 </td><td>3,685 </td><td>185 </td><td>95 </td><td>Heterozygous </td><td>42 </td><td>182 </td><td>55 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>16 </td><td>46 </td><td>2,283 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>971 </td><td>50 </td><td>14 </td></tr><tr><td>SCA-6 </td><td>3,493 </td><td>414 </td><td>89 </td><td>Heterozygous </td><td>83 </td><td>524 </td><td>144 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>11 </td><td>112 </td><td>1,998 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,103 </td><td>49 </td><td>9 </td></tr><tr><td>Pound 7 </td><td>3,529 </td><td>513 </td><td>87 </td><td>Heterozygous </td><td>245 </td><td>820 </td><td>164 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>12 </td><td>34 </td><td>606 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,304 </td><td>64 </td><td>5 </td></tr><tr><td>EBC 148 </td><td>4,273 </td><td>369 </td><td>92 </td><td>Heterozygous </td><td>118 </td><td>870 </td><td>67 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>97 </td><td>1,099 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,144 </td><td>47 </td><td>14 </td></tr><tr><td>UF273 type I </td><td>3,876 </td><td>650 </td><td>86 </td><td>Heterozygous </td><td>280 </td><td>1,351 </td><td>185 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>13 </td><td>112 </td><td>1,381 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4938978" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV009TB1" class="table-wrap-title" data-id="DSV009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of Cacao6kSNP array and sequencing calls across common accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Clone </th><th rowspan="2">Concordant SNPs </th><th rowspan="2">Non-concordant SNPs </th><th rowspan="2">Per cent concordance </th><th rowspan="2"> </th><th rowspan="2"> </th><th colspan="3">RNAseq genotypes<hr /> </th></tr><tr><th>Homozygous reference </th><th>Heterozygous </th><th>Homozygous variant </th></tr></thead><tbody><tr><td> </td><td> </td><td> </td><td> </td><td rowspan="27">Cacao6kSNP Array genotypes </td><td>Homozygous reference </td><td>4,685 </td><td>23 </td><td>3 </td></tr><tr><td>Matina 1-6 </td><td>4,687 </td><td>47 </td><td>99 </td><td>Heterozygous </td><td>2 </td><td>0 </td><td>0 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>9 </td><td>10 </td><td>2 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,477 </td><td>24 </td><td>14 </td></tr><tr><td>Criollo 13 </td><td>4,104 </td><td>117 </td><td>97 </td><td>Heterozygous </td><td>1 </td><td>3 </td><td>1 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>59 </td><td>2,626 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>3,237 </td><td>51 </td><td>6 </td></tr><tr><td>Pound 5 </td><td>4,519 </td><td>169 </td><td>96 </td><td>Heterozygous </td><td>21 </td><td>104 </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>23 </td><td>54 </td><td>1,178 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,712 </td><td>91 </td><td>12 </td></tr><tr><td>GU255 </td><td>4,558 </td><td>211 </td><td>96 </td><td>Heterozygous </td><td>17 </td><td>147 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>8 </td><td>70 </td><td>1,699 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,220 </td><td>14 </td><td>12 </td></tr><tr><td>NAP 30 </td><td>3,685 </td><td>185 </td><td>95 </td><td>Heterozygous </td><td>42 </td><td>182 </td><td>55 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>16 </td><td>46 </td><td>2,283 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>971 </td><td>50 </td><td>14 </td></tr><tr><td>SCA-6 </td><td>3,493 </td><td>414 </td><td>89 </td><td>Heterozygous </td><td>83 </td><td>524 </td><td>144 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>11 </td><td>112 </td><td>1,998 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,103 </td><td>49 </td><td>9 </td></tr><tr><td>Pound 7 </td><td>3,529 </td><td>513 </td><td>87 </td><td>Heterozygous </td><td>245 </td><td>820 </td><td>164 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>12 </td><td>34 </td><td>606 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,304 </td><td>64 </td><td>5 </td></tr><tr><td>EBC 148 </td><td>4,273 </td><td>369 </td><td>92 </td><td>Heterozygous </td><td>118 </td><td>870 </td><td>67 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>97 </td><td>1,099 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,144 </td><td>47 </td><td>14 </td></tr><tr><td>UF273 type I </td><td>3,876 </td><td>650 </td><td>86 </td><td>Heterozygous </td><td>280 </td><td>1,351 </td><td>185 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>13 </td><td>112 </td><td>1,381 </td></tr></tbody></table></div><div content-id="DSV009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Clone </th><th rowspan="2">Concordant SNPs </th><th rowspan="2">Non-concordant SNPs </th><th rowspan="2">Per cent concordance </th><th rowspan="2"> </th><th rowspan="2"> </th><th colspan="3">RNAseq genotypes<hr /> </th></tr><tr><th>Homozygous reference </th><th>Heterozygous </th><th>Homozygous variant </th></tr></thead><tbody><tr><td> </td><td> </td><td> </td><td> </td><td rowspan="27">Cacao6kSNP Array genotypes </td><td>Homozygous reference </td><td>4,685 </td><td>23 </td><td>3 </td></tr><tr><td>Matina 1-6 </td><td>4,687 </td><td>47 </td><td>99 </td><td>Heterozygous </td><td>2 </td><td>0 </td><td>0 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>9 </td><td>10 </td><td>2 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,477 </td><td>24 </td><td>14 </td></tr><tr><td>Criollo 13 </td><td>4,104 </td><td>117 </td><td>97 </td><td>Heterozygous </td><td>1 </td><td>3 </td><td>1 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>59 </td><td>2,626 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>3,237 </td><td>51 </td><td>6 </td></tr><tr><td>Pound 5 </td><td>4,519 </td><td>169 </td><td>96 </td><td>Heterozygous </td><td>21 </td><td>104 </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>23 </td><td>54 </td><td>1,178 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,712 </td><td>91 </td><td>12 </td></tr><tr><td>GU255 </td><td>4,558 </td><td>211 </td><td>96 </td><td>Heterozygous </td><td>17 </td><td>147 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>8 </td><td>70 </td><td>1,699 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,220 </td><td>14 </td><td>12 </td></tr><tr><td>NAP 30 </td><td>3,685 </td><td>185 </td><td>95 </td><td>Heterozygous </td><td>42 </td><td>182 </td><td>55 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>16 </td><td>46 </td><td>2,283 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>971 </td><td>50 </td><td>14 </td></tr><tr><td>SCA-6 </td><td>3,493 </td><td>414 </td><td>89 </td><td>Heterozygous </td><td>83 </td><td>524 </td><td>144 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>11 </td><td>112 </td><td>1,998 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,103 </td><td>49 </td><td>9 </td></tr><tr><td>Pound 7 </td><td>3,529 </td><td>513 </td><td>87 </td><td>Heterozygous </td><td>245 </td><td>820 </td><td>164 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>12 </td><td>34 </td><td>606 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>2,304 </td><td>64 </td><td>5 </td></tr><tr><td>EBC 148 </td><td>4,273 </td><td>369 </td><td>92 </td><td>Heterozygous </td><td>118 </td><td>870 </td><td>67 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>18 </td><td>97 </td><td>1,099 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous reference </td><td>1,144 </td><td>47 </td><td>14 </td></tr><tr><td>UF273 type I </td><td>3,876 </td><td>650 </td><td>86 </td><td>Heterozygous </td><td>280 </td><td>1,351 </td><td>185 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>Homozygous variant </td><td>13 </td><td>112 </td><td>1,381 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4938978" target="_blank">View Large</a></div></div>                        <a id="4938979" scrollto-destination="4938979"></a>
<div content-id="DSV009TB2" class="table-modal"><div class="table-wrap"><div id="DSV009TB2" class="table-wrap-title" data-id="DSV009TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of non-concordant SNPs between Cacao6kSNP array and sequencing arranged by number of discrepant accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number of non-concordant accessions identified </th><th>Number of unique non-concordant SNPs </th><th>Number of unique SNPs with opposing homozygous genotypes </th><th>Per cent of opposing homozygous genotypes </th></tr></thead><tbody><tr><td>1 </td><td>1,433 </td><td>50 </td><td>3 </td></tr><tr><td>2 </td><td>347 </td><td>27 </td><td>8 </td></tr><tr><td>3 </td><td>71 </td><td>17 </td><td>24 </td></tr><tr><td>4 </td><td>28 </td><td>17 </td><td>61 </td></tr><tr><td>5 </td><td>16 </td><td>11 </td><td>69 </td></tr><tr><td>6 </td><td>8 </td><td>7 </td><td>88 </td></tr><tr><td>7 </td><td>9 </td><td>9 </td><td>100 </td></tr><tr><td>8 </td><td>4 </td><td>3 </td><td>75 </td></tr><tr><td>9 </td><td>0 </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>1,916 </td><td>141 </td><td>7 </td></tr></tbody></table></div><div content-id="DSV009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Number of non-concordant accessions identified </th><th>Number of unique non-concordant SNPs </th><th>Number of unique SNPs with opposing homozygous genotypes </th><th>Per cent of opposing homozygous genotypes </th></tr></thead><tbody><tr><td>1 </td><td>1,433 </td><td>50 </td><td>3 </td></tr><tr><td>2 </td><td>347 </td><td>27 </td><td>8 </td></tr><tr><td>3 </td><td>71 </td><td>17 </td><td>24 </td></tr><tr><td>4 </td><td>28 </td><td>17 </td><td>61 </td></tr><tr><td>5 </td><td>16 </td><td>11 </td><td>69 </td></tr><tr><td>6 </td><td>8 </td><td>7 </td><td>88 </td></tr><tr><td>7 </td><td>9 </td><td>9 </td><td>100 </td></tr><tr><td>8 </td><td>4 </td><td>3 </td><td>75 </td></tr><tr><td>9 </td><td>0 </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>1,916 </td><td>141 </td><td>7 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4938979" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV009TB2" class="table-wrap-title" data-id="DSV009TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of non-concordant SNPs between Cacao6kSNP array and sequencing arranged by number of discrepant accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number of non-concordant accessions identified </th><th>Number of unique non-concordant SNPs </th><th>Number of unique SNPs with opposing homozygous genotypes </th><th>Per cent of opposing homozygous genotypes </th></tr></thead><tbody><tr><td>1 </td><td>1,433 </td><td>50 </td><td>3 </td></tr><tr><td>2 </td><td>347 </td><td>27 </td><td>8 </td></tr><tr><td>3 </td><td>71 </td><td>17 </td><td>24 </td></tr><tr><td>4 </td><td>28 </td><td>17 </td><td>61 </td></tr><tr><td>5 </td><td>16 </td><td>11 </td><td>69 </td></tr><tr><td>6 </td><td>8 </td><td>7 </td><td>88 </td></tr><tr><td>7 </td><td>9 </td><td>9 </td><td>100 </td></tr><tr><td>8 </td><td>4 </td><td>3 </td><td>75 </td></tr><tr><td>9 </td><td>0 </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>1,916 </td><td>141 </td><td>7 </td></tr></tbody></table></div><div content-id="DSV009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Number of non-concordant accessions identified </th><th>Number of unique non-concordant SNPs </th><th>Number of unique SNPs with opposing homozygous genotypes </th><th>Per cent of opposing homozygous genotypes </th></tr></thead><tbody><tr><td>1 </td><td>1,433 </td><td>50 </td><td>3 </td></tr><tr><td>2 </td><td>347 </td><td>27 </td><td>8 </td></tr><tr><td>3 </td><td>71 </td><td>17 </td><td>24 </td></tr><tr><td>4 </td><td>28 </td><td>17 </td><td>61 </td></tr><tr><td>5 </td><td>16 </td><td>11 </td><td>69 </td></tr><tr><td>6 </td><td>8 </td><td>7 </td><td>88 </td></tr><tr><td>7 </td><td>9 </td><td>9 </td><td>100 </td></tr><tr><td>8 </td><td>4 </td><td>3 </td><td>75 </td></tr><tr><td>9 </td><td>0 </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>1,916 </td><td>141 </td><td>7 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4938979" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4938980 id="4938980" class="section-title" data-legacy-id=s3d>3.4. Complete SNP dataset comparison among selected genotypes</h3>
<p>To assess the utility of the Cacao6kSNP array data in distinguishing among accessions, representative accessions of the two sequenced genomes (Matina 1-6 and Criollo 13) known to belong to divergent genetic groups<sup><a reveal-id="DSV009C6" data-open="DSV009C6" class="link link-ref link-reveal xref-bibr">6</a></sup> were compared. Both accessions showed extreme homozygosity across 5,117 SNPs, with only two and six heterozygotes identified in Matina 1-6 and Criollo 13, respectively, resulting in 99.8% homozygosity. This finding is consistent with previous analyses using microsatellite markers<sup><a reveal-id="DSV009C23" data-open="DSV009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and was helpful in the generation of the respective genome assemblies. Despite their high levels of homozygosity, these accessions share only a meager 38% of their alleles (1,936 out of 5,117). A list of the markers with differing alleles between Matina 1-6 and Criollo 13 can be found in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. This allelic divergence is perhaps due to geographic isolation and differing evolutionary paths. In terms of cacao breeding, an examination of the differences between these accessions may identify alleles specific for particular traits of interest. Criollo is often preferred by the fine flavour chocolate producers but tends to suffer from poor yields,<sup><a reveal-id="DSV009C45" data-open="DSV009C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSV009C46" data-open="DSV009C46" class="link link-ref link-reveal xref-bibr">46</a></sup> whereas Matina, an Amelonado type, one of the most grown genetic groups, is higher yielding and makes up a large proportion of bulk chocolate. A better understanding of the genetic differences between these two clones may provide insight into the mechanism and control of the traits that make each desirable to cacao growers and producers.</p><p>Two other genotypes, this time closely related: UF273 Type I and UF273 Type II, were also examined. These two accessions, respectively, serve as the fathers for the CATIE Type 1 and CATIE Type 2 populations generated at CATIE, Turrialba, Costa Rica.<sup><a reveal-id="DSV009C47" data-open="DSV009C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Originally intended to be a single F1 population from the cross of Pound 7 × UF273, five cloned trees of UF273 were used as parents. After the F1 population was planted, it was discovered that one tree out of the five UF273 trees used as a pollen donor was genetically distinct from the other four clones (UF273 Type I) and was henceforth referred to as UF273 Type II. An SSR analysis of 180 markers revealed variation in UF273 Type II alleles for 22 markers (12%), with most often one allele differing at the locus.<sup><a reveal-id="DSV009C47" data-open="DSV009C47" class="link link-ref link-reveal xref-bibr">47</a></sup> A comparison of UF273 Type I and UF273 Type II was performed using 5,123 successful genotypes from the Cacao6kSNP array. UF273 Type I shows 60% homozygousity (3,101 out of 5,123 markers) and UF273 Type II only slightly more at 64% (3,276 out of 5,123). Further distinguishing themselves from the Matina 1-6 and Criollo 13 accessions, UF273 Type I and UF273 Type II share 84% identity (4,294 markers) and 99% of their alleles, having only eight opposing homozygous SNPs out of the 5,123 markers compared. The high proportion of shared alleles between these two accessions suggests a close genetic relationship. IBD values of all pair-wise comparisons for both the CATIE Type 1 and Type 2 mapping populations were determined and are summarized in Fig. <a reveal-id="DSV009F4" data-open="DSV009F4" class="link link-reveal link-table xref-fig">4</a>. A pair-wise IBD value of 0.92 was determined for UF273 Type I versus UF273 Type II, indicating that it is a first-degree (full-sib) relationship (Fig. <a reveal-id="DSV009F4" data-open="DSV009F4" class="link link-reveal link-table xref-fig">4</a>). There are only two cases where the IBD is as high, or higher, than those of the UF273 accessions. Two full-sibs from the CATIE Type 1 population also share an IBD of 0.92, and another comparison shows an IBD of 1 (genetically identical), which can be attributed to a field error. The UF273 Type I and Type II accessions are so similar that they have often been lumped together as few genetic differences between the two were previously observed.<sup><a reveal-id="DSV009C47" data-open="DSV009C47" class="link link-ref link-reveal xref-bibr">47</a></sup> However, this pooling often creates problems for cacao breeders as progeny generated from crosses with these accessions has been shown to segregate differently for multiple traits (Dr Wilbert Phillips, personal communication). The availability of SNPs to easily distinguish these accessions from one another would serve to reassure breeding efforts utilizing one or both. A total of 827 markers differentiating UF273 Type I and Type II accessions were reported and can be found in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. </p>                        <a id="4938983" scrollto-destination="4938983"></a>
<div id="DSV009F4" data-id="DSV009F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00904.png?Expires=2147483647&Signature=S6GJEJ8o~NdeEE0vljQ9vGluM-3fE7p2i35xk1PPO76DAU7cFWDM4ZHCtIzL~8U70UCc2H5ERvGJSge-McQ6SbbE6bApD58HckMy9YKAz5OD3sM-2tcu21dWY3Kdhe7ZUNkFB~Iwx98tTsz4P3DFciV0sykzmUqS0IqDcscpPwE8IHE4I0kS3I2D6n6TG8H0HzMBwt0T~kSbI3bklfeSlInbPqfe72KQwlfrhBYiuAvrhEF-kE2nPnIBKOuwWbMd8PjuF7X3Y6fPgOgVAC78~CRw0PNJ9Z~0rb-dAAARd-iXWBMM4rH7gnOvLTV9~iRvi-j-6UjDGoH-gk0DocjCDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="IBD distribution for all CATIE Type 1 and Type 2 pair-wise comparisons (including parents). The left mode of the bimodal curve reflects the IBD distribution for parent–offspring relationships (as well as two parent-to-parent IBDs: Pound 7 versus UF273 Type I and Pound 7 versus UF273 Type II). The right mode reflects the full-sibling and half-sibling pair-wise IBDs. The dashed line represents the UF273 Type I versus UF273 Type II pair-wise IBD value (0.92), indicating a first-degree relationship. This figure is available in black and white in print and in colour at DNA Research online" path-from-xml="dsv00904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938983/dsv00904.tiff" path-from-xml="dsv00904.tiff" target="_blank">View large</a><a section="4938983" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00904.png?Expires=2147483647&Signature=IAcWt8jqs820SjC-gmYGdEJWh5yW~XI7X-krf2aLCOWWNPAseqsymeYoWsVU~PXK87L8MijjRFqKmG6Y4dgmjmu6OhiM6WBLL~N~g5TwHlMXz6CdIxw7xK6lWYRlA-fCzIWldYPyu2q~NA2esBqIXvJTbYDm0xirNxunJ3QwpiWsGuWLplB0hC3GeUx9ik~mODLgDMI3CQ9N~JZ0s7SmDWjmqjjKAUyJ2kzcVGRy2gtmAV3wcdOinHJIb45GfvWvQ-NgEW8m9ZVDD0KaN~0x-Y9afF54lJ2cUnxvgnR4ahubed7bNhf8cKm0RLWStlFuaX9z3nFzvLjW8T9mmXkyaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938983&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>IBD distribution for all CATIE Type 1 and Type 2 pair-wise comparisons (including parents). The left mode of the bimodal curve reflects the IBD distribution for parent–offspring relationships (as well as two parent-to-parent IBDs: Pound 7 versus UF273 Type I and Pound 7 versus UF273 Type II). The right mode reflects the full-sibling and half-sibling pair-wise IBDs. The dashed line represents the UF273 Type I versus UF273 Type II pair-wise IBD value (0.92), indicating a first-degree relationship. This figure is available in black and white in print and in colour at <em>DNA Research</em> online</p></div></div><div content-id="DSV009F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00904.png?Expires=2147483647&Signature=S6GJEJ8o~NdeEE0vljQ9vGluM-3fE7p2i35xk1PPO76DAU7cFWDM4ZHCtIzL~8U70UCc2H5ERvGJSge-McQ6SbbE6bApD58HckMy9YKAz5OD3sM-2tcu21dWY3Kdhe7ZUNkFB~Iwx98tTsz4P3DFciV0sykzmUqS0IqDcscpPwE8IHE4I0kS3I2D6n6TG8H0HzMBwt0T~kSbI3bklfeSlInbPqfe72KQwlfrhBYiuAvrhEF-kE2nPnIBKOuwWbMd8PjuF7X3Y6fPgOgVAC78~CRw0PNJ9Z~0rb-dAAARd-iXWBMM4rH7gnOvLTV9~iRvi-j-6UjDGoH-gk0DocjCDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="IBD distribution for all CATIE Type 1 and Type 2 pair-wise comparisons (including parents). The left mode of the bimodal curve reflects the IBD distribution for parent–offspring relationships (as well as two parent-to-parent IBDs: Pound 7 versus UF273 Type I and Pound 7 versus UF273 Type II). The right mode reflects the full-sibling and half-sibling pair-wise IBDs. The dashed line represents the UF273 Type I versus UF273 Type II pair-wise IBD value (0.92), indicating a first-degree relationship. This figure is available in black and white in print and in colour at DNA Research online" path-from-xml="dsv00904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938983/dsv00904.tiff" path-from-xml="dsv00904.tiff" target="_blank">View large</a><a section="4938983" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00904.png?Expires=2147483647&Signature=IAcWt8jqs820SjC-gmYGdEJWh5yW~XI7X-krf2aLCOWWNPAseqsymeYoWsVU~PXK87L8MijjRFqKmG6Y4dgmjmu6OhiM6WBLL~N~g5TwHlMXz6CdIxw7xK6lWYRlA-fCzIWldYPyu2q~NA2esBqIXvJTbYDm0xirNxunJ3QwpiWsGuWLplB0hC3GeUx9ik~mODLgDMI3CQ9N~JZ0s7SmDWjmqjjKAUyJ2kzcVGRy2gtmAV3wcdOinHJIb45GfvWvQ-NgEW8m9ZVDD0KaN~0x-Y9afF54lJ2cUnxvgnR4ahubed7bNhf8cKm0RLWStlFuaX9z3nFzvLjW8T9mmXkyaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938983&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>IBD distribution for all CATIE Type 1 and Type 2 pair-wise comparisons (including parents). The left mode of the bimodal curve reflects the IBD distribution for parent–offspring relationships (as well as two parent-to-parent IBDs: Pound 7 versus UF273 Type I and Pound 7 versus UF273 Type II). The right mode reflects the full-sibling and half-sibling pair-wise IBDs. The dashed line represents the UF273 Type I versus UF273 Type II pair-wise IBD value (0.92), indicating a first-degree relationship. This figure is available in black and white in print and in colour at <em>DNA Research</em> online</p></div></div><p>The first-order relationship between UF273 Type I and Type II provides an opportunity to examine the number of SNP markers needed to successfully differentiate between two closely related accessions, a task that is often essential in cacao breeding programs. Using a number of randomly selected loci genotyped in the Cacao6kSNP array as a pool of markers, we determined the probability of successfully differentiating accessions with 10%, 16.2% (as observed with UF273 Type I and Type II) and 30% genetic variation (Fig. <a reveal-id="DSV009F5" data-open="DSV009F5" class="link link-reveal link-table xref-fig">5</a>). A confidence threshold of 95%, taking into account the genotyping error rate, was used to identify different accessions. Serving as a measure of relatedness, the proportion of loci that were different between the first-order related UF273 accessions (0.162) was used as a reference point as differentiation between two full-sibs is likely to be a practical situation encountered in breeding programs. Proportion values of 0.1 and 0.3 were also included to show how the results might change with more or less genetically related accessions as described in the Materials and Methods. For the example of UF273 Type I and Type II, 24 loci are required to confidently distinguish one accession from another, whereas for accessions displaying only 10% variation, 30 loci would be required (Fig. <a reveal-id="DSV009F5" data-open="DSV009F5" class="link link-reveal link-table xref-fig">5</a>). These values are consistent with those identified in bovine systems and in other cacao studies,<sup><a reveal-id="DSV009C28" data-open="DSV009C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV009C36" data-open="DSV009C36" class="link link-ref link-reveal xref-bibr">36</a></sup> which reported that ∼40 SNP loci are required to differentiate genetically distinct individuals. The number of loci required to distinguish related accessions can be genotyped easily using modern SNP genotyping systems (Fluidigm, Sequenome, Golden Gate, etc.) and will be of value for marker-assisted selection breeding programs. </p>                        <a id="4938985" scrollto-destination="4938985"></a>
<div id="DSV009F5" data-id="DSV009F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00905.png?Expires=2147483647&Signature=udzrvTif5zbmjwlWuJZhqEGiV9jzqa231WretsiB8YICr1lgwof1IX~xNtV01Cc~E6GjSafLdjJNjr2oscAQWTvESvk6hYdB-MMQ4llFuh0L7-W1Co93K-gDiVc7gf0wrHcWGcEDwZYmExWMqyTqmZdWuyaAuoY51xZQiihOHIHPoveg9BlIdf9xNhzMauTiZmbhXdahyKwrkcEOcvmuFVu1Xf3Uw5pisYQFuuJLD8gddUvVKfU1JFsL8s4uFitadd9Nqjjj-t5nS2c~ZfK3OrXFNbeEaqc0ZSMyWcedUTHJoDigmEISNP5nHpvduglMBVZtZ2RkS9h-pdnwX-ppbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Probability of differentiating accessions of various levels of identity using increasing numbers of SNP markers. The probability of successfully differentiating accessions with 10% (Δ), 16.2% (□; as observed with UF273 Type I and Type II) and 30% (∇) genetic variation using a number of randomly selected loci as calculated using the binomial distribution. A confidence threshold of 95%, with a 0.01 genotyping error rate, was selected to identify different accessions. Thirty SNPs are sufficient to distinguish accessions with only 10% variation." path-from-xml="dsv00905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938985/dsv00905.tiff" path-from-xml="dsv00905.tiff" target="_blank">View large</a><a section="4938985" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00905.png?Expires=2147483647&Signature=v4O23LJ2KsbfCCVNIOxm~qbWveWTShwIiWPmZ3MeAldDDg-fORRfsCTOUVAzDaU~WD--wMlnI7bPtuhmlucQqhEaI9KqAhRf~VK6jvx-b6tOzq-F18AhJAT53D1gQRJHmw5Eouth4ovOfm6KYD293oSiToOi-K3ynym4omTOREorKU6tn2c2la64GwGxo7wGtuTuV52y4qiGS~x3sSyW8TSPYObwKaSaVEZq9CBZvN4WKIk7JhkEN6yiXpMTq2ujsH756647SP0NZ7pIZMpq5TzqH6V-ugp3wFe6PrirPPjhbjSU7jH~cRZV21~Lh7ygD-lnTvVI~5XmklmnkUdG8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938985&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Probability of differentiating accessions of various levels of identity using increasing numbers of SNP markers. The probability of successfully differentiating accessions with 10% (Δ), 16.2% (□; as observed with UF273 Type I and Type II) and 30% (∇) genetic variation using a number of randomly selected loci as calculated using the binomial distribution. A confidence threshold of 95%, with a 0.01 genotyping error rate, was selected to identify different accessions. Thirty SNPs are sufficient to distinguish accessions with only 10% variation.</p></div></div><div content-id="DSV009F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00905.png?Expires=2147483647&Signature=udzrvTif5zbmjwlWuJZhqEGiV9jzqa231WretsiB8YICr1lgwof1IX~xNtV01Cc~E6GjSafLdjJNjr2oscAQWTvESvk6hYdB-MMQ4llFuh0L7-W1Co93K-gDiVc7gf0wrHcWGcEDwZYmExWMqyTqmZdWuyaAuoY51xZQiihOHIHPoveg9BlIdf9xNhzMauTiZmbhXdahyKwrkcEOcvmuFVu1Xf3Uw5pisYQFuuJLD8gddUvVKfU1JFsL8s4uFitadd9Nqjjj-t5nS2c~ZfK3OrXFNbeEaqc0ZSMyWcedUTHJoDigmEISNP5nHpvduglMBVZtZ2RkS9h-pdnwX-ppbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Probability of differentiating accessions of various levels of identity using increasing numbers of SNP markers. The probability of successfully differentiating accessions with 10% (Δ), 16.2% (□; as observed with UF273 Type I and Type II) and 30% (∇) genetic variation using a number of randomly selected loci as calculated using the binomial distribution. A confidence threshold of 95%, with a 0.01 genotyping error rate, was selected to identify different accessions. Thirty SNPs are sufficient to distinguish accessions with only 10% variation." path-from-xml="dsv00905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938985/dsv00905.tiff" path-from-xml="dsv00905.tiff" target="_blank">View large</a><a section="4938985" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00905.png?Expires=2147483647&Signature=v4O23LJ2KsbfCCVNIOxm~qbWveWTShwIiWPmZ3MeAldDDg-fORRfsCTOUVAzDaU~WD--wMlnI7bPtuhmlucQqhEaI9KqAhRf~VK6jvx-b6tOzq-F18AhJAT53D1gQRJHmw5Eouth4ovOfm6KYD293oSiToOi-K3ynym4omTOREorKU6tn2c2la64GwGxo7wGtuTuV52y4qiGS~x3sSyW8TSPYObwKaSaVEZq9CBZvN4WKIk7JhkEN6yiXpMTq2ujsH756647SP0NZ7pIZMpq5TzqH6V-ugp3wFe6PrirPPjhbjSU7jH~cRZV21~Lh7ygD-lnTvVI~5XmklmnkUdG8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938985&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Probability of differentiating accessions of various levels of identity using increasing numbers of SNP markers. The probability of successfully differentiating accessions with 10% (Δ), 16.2% (□; as observed with UF273 Type I and Type II) and 30% (∇) genetic variation using a number of randomly selected loci as calculated using the binomial distribution. A confidence threshold of 95%, with a 0.01 genotyping error rate, was selected to identify different accessions. Thirty SNPs are sufficient to distinguish accessions with only 10% variation.</p></div></div>                            <h3 scrollto-destination=4938986 id="4938986" class="section-title" data-legacy-id=s3e>3.5. Linkage mapping</h3>
<p>In preparation for genetic linkage mapping with the Cacao6kSNP data, SNPs displaying segregation within the different mapping populations were separated from the uninformative loci. Segregating SNPs, or those where one or both parents are heterozygous, can be used to generate linkage maps in these F1 populations. The number of segregating SNPs within and between mapping populations is shown as a Venn diagram (Fig. <a reveal-id="DSV009F6" data-open="DSV009F6" class="link link-reveal link-table xref-fig">6</a>). The total number of segregating SNPs ranges from 2,543 to 3,563, which is ∼10 times as many markers as in the composite SSR map created by Brown et al.<sup><a reveal-id="DSV009C48" data-open="DSV009C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and 2 to 3 times the number of markers in the composite map generated by Allegre et al.<sup><a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a></sup> A number of SNPs specific to each of the mapping populations were identified (Fig. <a reveal-id="DSV009F6" data-open="DSV009F6" class="link link-reveal link-table xref-fig">6</a>). Furthermore, 260 loci segregate in CATIE Type 2 but do not segregate in CATIE Type 1. Additionally, CATIE Type 1 has 346 segregating markers that do not segregate in the Type 2 population (Fig. <a reveal-id="DSV009F6" data-open="DSV009F6" class="link link-reveal link-table xref-fig">6</a>). These population-specific SNPs may prove instrumental in determining associations with traits specific to one population. Conversely, the 956 SNPs held in common across all populations can be used as anchor markers in a composite linkage map. Placement of the markers of this set will provide higher overall precision than use of markers from any one individual map.<sup><a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSV009C47" data-open="DSV009C47" class="link link-ref link-reveal xref-bibr">47</a></sup><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> lists all SNPs present on the Cacao6kSNP array, any population they segregate within, their positions on the Matina 1-6 genome, nearby SNPs presented by Allegre et al.<sup><a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a></sup> as well as transcript sequences flanking each SNP. </p>                        <a id="4938988" scrollto-destination="4938988"></a>
<div id="DSV009F6" data-id="DSV009F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00906.png?Expires=2147483647&Signature=Yt6Mmrh3MdcHr9-lja0ilSYVkLvLBGt4~6rBZlbPszSD5lk8rgivxSG2IYCwdZXnn7UNqH8ctkMlIfT9OiWUd7wpc8WvNXOpysWZ0z7Tt88c3LZOGgub44yGyiuDW~AxAk8LMpsanKEDIKgh5tpU3z1umqk~20d9aEMrONMJj78XVQHk4pWSjiuYQI8LWl~J0--ce8glLUpgQZAxhEpd6T72XMQ5G~14vEKOtWG9Du4qQA8zpyKAK2syp0uGL19xDsBMfjrgD2eP7gSB80hR2C0Aux5VJWU9AoYqWBWYeK8SkOw3KGJ~xxWFad8o7j6KpvzjMIhUuHQCBlco6Vs~sA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram detailing the number of SNP markers that can be mapped, i.e. those with at least one heterozygous parent, in a particular population. The population name is listed above with the total number of markers that can be mapped in parenthesis. Markers that segregate within each population are represented by a coloured rectangle: CATIE Type 1 markers in blue, CATIE Type 2 markers in green, PNG in red and MCCS in purple. Overlapping rectangles indicate markers shared between those populations. Numbers within rectangles represent the number of markers that can be mapped within those population(s)." path-from-xml="dsv00906.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938988/dsv00906.tiff" path-from-xml="dsv00906.tiff" target="_blank">View large</a><a section="4938988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00906.png?Expires=2147483647&Signature=Ae9NuiiZV7Il7u-dY8RAJ~8CST8nSpSEdi0TAOnin64PpG5al1Z~qEMEunuHgVSsQK5cpCnYPD8h3P4iSQxX2aCOn-XMyTcVVblUQpzs6X5x8EXuuzvUfPYrmNGzxAH~DfA84L9LJ5RBG0MiGjStReMdUsIabUD-z~A79uWWX2J45YyNY2iNuI2WFXyJ5Li3y98zN0C1orHAyVbkbBPHDLSgnjMDCIqI9Khun7Jvuk0k6rzQ2etw0uFCizcG6mTknLhQ2XsSPhOI6i4Wy~76Jr1Phy-D2xUV5JtTyUV0gmWYeFDYe3TxhOEIfj3oebQIrZti4sLUYkYCb30lHGfqHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938988&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00906.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram detailing the number of SNP markers that can be mapped, i.e. those with at least one heterozygous parent, in a particular population. The population name is listed above with the total number of markers that can be mapped in parenthesis. Markers that segregate within each population are represented by a coloured rectangle: CATIE Type 1 markers in blue, CATIE Type 2 markers in green, PNG in red and MCCS in purple. Overlapping rectangles indicate markers shared between those populations. Numbers within rectangles represent the number of markers that can be mapped within those population(s).</p></div></div><div content-id="DSV009F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00906.png?Expires=2147483647&Signature=Yt6Mmrh3MdcHr9-lja0ilSYVkLvLBGt4~6rBZlbPszSD5lk8rgivxSG2IYCwdZXnn7UNqH8ctkMlIfT9OiWUd7wpc8WvNXOpysWZ0z7Tt88c3LZOGgub44yGyiuDW~AxAk8LMpsanKEDIKgh5tpU3z1umqk~20d9aEMrONMJj78XVQHk4pWSjiuYQI8LWl~J0--ce8glLUpgQZAxhEpd6T72XMQ5G~14vEKOtWG9Du4qQA8zpyKAK2syp0uGL19xDsBMfjrgD2eP7gSB80hR2C0Aux5VJWU9AoYqWBWYeK8SkOw3KGJ~xxWFad8o7j6KpvzjMIhUuHQCBlco6Vs~sA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram detailing the number of SNP markers that can be mapped, i.e. those with at least one heterozygous parent, in a particular population. The population name is listed above with the total number of markers that can be mapped in parenthesis. Markers that segregate within each population are represented by a coloured rectangle: CATIE Type 1 markers in blue, CATIE Type 2 markers in green, PNG in red and MCCS in purple. Overlapping rectangles indicate markers shared between those populations. Numbers within rectangles represent the number of markers that can be mapped within those population(s)." path-from-xml="dsv00906.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938988/dsv00906.tiff" path-from-xml="dsv00906.tiff" target="_blank">View large</a><a section="4938988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00906.png?Expires=2147483647&Signature=Ae9NuiiZV7Il7u-dY8RAJ~8CST8nSpSEdi0TAOnin64PpG5al1Z~qEMEunuHgVSsQK5cpCnYPD8h3P4iSQxX2aCOn-XMyTcVVblUQpzs6X5x8EXuuzvUfPYrmNGzxAH~DfA84L9LJ5RBG0MiGjStReMdUsIabUD-z~A79uWWX2J45YyNY2iNuI2WFXyJ5Li3y98zN0C1orHAyVbkbBPHDLSgnjMDCIqI9Khun7Jvuk0k6rzQ2etw0uFCizcG6mTknLhQ2XsSPhOI6i4Wy~76Jr1Phy-D2xUV5JtTyUV0gmWYeFDYe3TxhOEIfj3oebQIrZti4sLUYkYCb30lHGfqHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938988&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00906.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram detailing the number of SNP markers that can be mapped, i.e. those with at least one heterozygous parent, in a particular population. The population name is listed above with the total number of markers that can be mapped in parenthesis. Markers that segregate within each population are represented by a coloured rectangle: CATIE Type 1 markers in blue, CATIE Type 2 markers in green, PNG in red and MCCS in purple. Overlapping rectangles indicate markers shared between those populations. Numbers within rectangles represent the number of markers that can be mapped within those population(s).</p></div></div><p>A genetic linkage map of the CATIE Type 2 population (Pound 7 × UF273 Type II) was generated with 68 individuals and 2,589 SNP markers (Fig. <a reveal-id="DSV009F7" data-open="DSV009F7" class="link link-reveal link-table xref-fig">7</a>), producing a map of 1,268 cM total length. Ten linkage groups corresponding to the 10 chromosomes of cacao were identified, and linkage group designation was assigned by comparing mapped markers to the chromosomes of the Matina 1-6 genome assembly using the convention defined by Lanaud.<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSV009C49" data-open="DSV009C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Linkage mapping generally relies on large numbers of individuals within the population to identify recombination events, which are used to order markers along the map. Although quite small for linkage map generation, this population is typical of many cacao mapping populations, which are often studied in regions where resources are limited. Combined with the large land requirements of tree breeding and the need to maintain trees for many years, cacao mapping populations seldom have more than a hundred individuals, although a few exceptions exist at larger cacao research stations. The substantial increase in the number of markers should help to identify recombination events that may not be noticed with fewer markers. The genetic map generated for the CATIE Type 2 population serves as a proof of concept that larger numbers of markers can help, but not completely overcome, a limited number of individuals. A summary of the linkage map is presented in Table <a reveal-id="DSV009TB3" data-open="DSV009TB3" class="link link-reveal link-table xref-fig">3</a>, as is the composite linkage map generated by Allegre et al. for comparison.<sup><a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                        <a id="4938990" scrollto-destination="4938990"></a>
<div content-id="DSV009TB3" class="table-modal"><div class="table-wrap"><div id="DSV009TB3" class="table-wrap-title" data-id="DSV009TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of CATIE type 2 linkage map generated with SNPs (top) and the composite Allegre<sup><a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a></sup> linkage map generated with SNPs (bottom)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>No. of SNPs </th><th>Length (cM) </th><th>Gap length (cM) </th><th>Length (bp) </th><th>SNPs/cM </th><th>bp/cM </th></tr></thead><tbody><tr><td>LG01 </td><td>265 </td><td>134.2 </td><td>5.6 </td><td>35,128,701 </td><td>2.0 </td><td>261,764 ± 40,679 </td></tr><tr><td>LG02 </td><td>454 </td><td>178.2 </td><td>7.1 </td><td>35,095,207 </td><td>2.5 </td><td>196,976 ± 58,270 </td></tr><tr><td>LG03 </td><td>128 </td><td>107.3 </td><td>13.4 </td><td>29,936,525 </td><td>1.2 </td><td>279,100 ± 58,820 </td></tr><tr><td>LG04 </td><td>280 </td><td>146.5 </td><td>6.2 </td><td>28,498,427 </td><td>1.9 </td><td>194,555 ± 48,217 </td></tr><tr><td>LG05 </td><td>206 </td><td>117.1 </td><td>14.0 </td><td>33,047,117 </td><td>1.8 </td><td>282,230 ± 89,324 </td></tr><tr><td>LG06 </td><td>261 </td><td>99.5 </td><td>3.3 </td><td>21,387,731 </td><td>2.6 </td><td>214,924 ± 83,893 </td></tr><tr><td>LG07 </td><td>153 </td><td>93.8 </td><td>7.2 </td><td>19,274,282 </td><td>1.6 </td><td>205,424 ± 76,948 </td></tr><tr><td>LG08 </td><td>309 </td><td>120.7 </td><td>3.9 </td><td>16,539,538 </td><td>2.6 </td><td>137,005 ± 58,617 </td></tr><tr><td>LG09 </td><td>354 </td><td>178.5 </td><td>7.9 </td><td>35,247,141 </td><td>2.0 </td><td>197,447 ± 53,776 </td></tr><tr><td>LG10 </td><td>179 </td><td>92.4 </td><td>6.3 </td><td>20,306,549 </td><td>1.9 </td><td>219,756 ± 78,700 </td></tr><tr><td>Total </td><td>2,589 </td><td>1,268 </td><td>7.5 </td><td>274,461,215 </td><td>2.0 </td><td>216,420 ± 62,443 </td></tr><tr><td>LG01 </td><td>179 </td><td>94.1 </td><td>2.8 </td><td>35,128,701 </td><td>1.9 </td><td>373,312 ± 58,014 </td></tr><tr><td>LG02 </td><td>155 </td><td>101.1 </td><td>7.6 </td><td>35,095,207 </td><td>1.5 </td><td>347,134 ± 102,691 </td></tr><tr><td>LG03 </td><td>163 </td><td>76.9 </td><td>2.3 </td><td>29,936,525 </td><td>2.1 </td><td>389,292 ± 82,043 </td></tr><tr><td>LG04 </td><td>135 </td><td>64.2 </td><td>2.6 </td><td>28,498,427 </td><td>2.1 </td><td>443,901 ± 110,012 </td></tr><tr><td>LG05 </td><td>147 </td><td>78.1 </td><td>4.4 </td><td>33,047,117 </td><td>1.9 </td><td>423,138 ± 133,921 </td></tr><tr><td>LG06 </td><td>81 </td><td>64.0 </td><td>4.1 </td><td>21,387,731 </td><td>1.3 </td><td>334,183 ± 130,445 </td></tr><tr><td>LG07 </td><td>62 </td><td>52.6 </td><td>4.0 </td><td>19,274,282 </td><td>1.2 </td><td>366,431 ± 137,259 </td></tr><tr><td>LG08 </td><td>73 </td><td>59.2 </td><td>6.0 </td><td>16,539,538 </td><td>1.2 </td><td>279,384 ± 119,532 </td></tr><tr><td>LG09 </td><td>182 </td><td>100.9 </td><td>4.0 </td><td>35,247,141 </td><td>1.8 </td><td>349,327 ± 95,141 </td></tr><tr><td>LG10 </td><td>63 </td><td>59.5 </td><td>5.4 </td><td>20,306,549 </td><td>1.1 </td><td>341,287 ± 122,223 </td></tr><tr><td>Total </td><td>1,240 </td><td>751 </td><td>4.3 </td><td>274,461,215 </td><td>1.7 </td><td>365,656 ± 105,501 </td></tr></tbody></table></div><div content-id="DSV009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>No. of SNPs </th><th>Length (cM) </th><th>Gap length (cM) </th><th>Length (bp) </th><th>SNPs/cM </th><th>bp/cM </th></tr></thead><tbody><tr><td>LG01 </td><td>265 </td><td>134.2 </td><td>5.6 </td><td>35,128,701 </td><td>2.0 </td><td>261,764 ± 40,679 </td></tr><tr><td>LG02 </td><td>454 </td><td>178.2 </td><td>7.1 </td><td>35,095,207 </td><td>2.5 </td><td>196,976 ± 58,270 </td></tr><tr><td>LG03 </td><td>128 </td><td>107.3 </td><td>13.4 </td><td>29,936,525 </td><td>1.2 </td><td>279,100 ± 58,820 </td></tr><tr><td>LG04 </td><td>280 </td><td>146.5 </td><td>6.2 </td><td>28,498,427 </td><td>1.9 </td><td>194,555 ± 48,217 </td></tr><tr><td>LG05 </td><td>206 </td><td>117.1 </td><td>14.0 </td><td>33,047,117 </td><td>1.8 </td><td>282,230 ± 89,324 </td></tr><tr><td>LG06 </td><td>261 </td><td>99.5 </td><td>3.3 </td><td>21,387,731 </td><td>2.6 </td><td>214,924 ± 83,893 </td></tr><tr><td>LG07 </td><td>153 </td><td>93.8 </td><td>7.2 </td><td>19,274,282 </td><td>1.6 </td><td>205,424 ± 76,948 </td></tr><tr><td>LG08 </td><td>309 </td><td>120.7 </td><td>3.9 </td><td>16,539,538 </td><td>2.6 </td><td>137,005 ± 58,617 </td></tr><tr><td>LG09 </td><td>354 </td><td>178.5 </td><td>7.9 </td><td>35,247,141 </td><td>2.0 </td><td>197,447 ± 53,776 </td></tr><tr><td>LG10 </td><td>179 </td><td>92.4 </td><td>6.3 </td><td>20,306,549 </td><td>1.9 </td><td>219,756 ± 78,700 </td></tr><tr><td>Total </td><td>2,589 </td><td>1,268 </td><td>7.5 </td><td>274,461,215 </td><td>2.0 </td><td>216,420 ± 62,443 </td></tr><tr><td>LG01 </td><td>179 </td><td>94.1 </td><td>2.8 </td><td>35,128,701 </td><td>1.9 </td><td>373,312 ± 58,014 </td></tr><tr><td>LG02 </td><td>155 </td><td>101.1 </td><td>7.6 </td><td>35,095,207 </td><td>1.5 </td><td>347,134 ± 102,691 </td></tr><tr><td>LG03 </td><td>163 </td><td>76.9 </td><td>2.3 </td><td>29,936,525 </td><td>2.1 </td><td>389,292 ± 82,043 </td></tr><tr><td>LG04 </td><td>135 </td><td>64.2 </td><td>2.6 </td><td>28,498,427 </td><td>2.1 </td><td>443,901 ± 110,012 </td></tr><tr><td>LG05 </td><td>147 </td><td>78.1 </td><td>4.4 </td><td>33,047,117 </td><td>1.9 </td><td>423,138 ± 133,921 </td></tr><tr><td>LG06 </td><td>81 </td><td>64.0 </td><td>4.1 </td><td>21,387,731 </td><td>1.3 </td><td>334,183 ± 130,445 </td></tr><tr><td>LG07 </td><td>62 </td><td>52.6 </td><td>4.0 </td><td>19,274,282 </td><td>1.2 </td><td>366,431 ± 137,259 </td></tr><tr><td>LG08 </td><td>73 </td><td>59.2 </td><td>6.0 </td><td>16,539,538 </td><td>1.2 </td><td>279,384 ± 119,532 </td></tr><tr><td>LG09 </td><td>182 </td><td>100.9 </td><td>4.0 </td><td>35,247,141 </td><td>1.8 </td><td>349,327 ± 95,141 </td></tr><tr><td>LG10 </td><td>63 </td><td>59.5 </td><td>5.4 </td><td>20,306,549 </td><td>1.1 </td><td>341,287 ± 122,223 </td></tr><tr><td>Total </td><td>1,240 </td><td>751 </td><td>4.3 </td><td>274,461,215 </td><td>1.7 </td><td>365,656 ± 105,501 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4938990" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV009TB3" class="table-wrap-title" data-id="DSV009TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of CATIE type 2 linkage map generated with SNPs (top) and the composite Allegre<sup><a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a></sup> linkage map generated with SNPs (bottom)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>No. of SNPs </th><th>Length (cM) </th><th>Gap length (cM) </th><th>Length (bp) </th><th>SNPs/cM </th><th>bp/cM </th></tr></thead><tbody><tr><td>LG01 </td><td>265 </td><td>134.2 </td><td>5.6 </td><td>35,128,701 </td><td>2.0 </td><td>261,764 ± 40,679 </td></tr><tr><td>LG02 </td><td>454 </td><td>178.2 </td><td>7.1 </td><td>35,095,207 </td><td>2.5 </td><td>196,976 ± 58,270 </td></tr><tr><td>LG03 </td><td>128 </td><td>107.3 </td><td>13.4 </td><td>29,936,525 </td><td>1.2 </td><td>279,100 ± 58,820 </td></tr><tr><td>LG04 </td><td>280 </td><td>146.5 </td><td>6.2 </td><td>28,498,427 </td><td>1.9 </td><td>194,555 ± 48,217 </td></tr><tr><td>LG05 </td><td>206 </td><td>117.1 </td><td>14.0 </td><td>33,047,117 </td><td>1.8 </td><td>282,230 ± 89,324 </td></tr><tr><td>LG06 </td><td>261 </td><td>99.5 </td><td>3.3 </td><td>21,387,731 </td><td>2.6 </td><td>214,924 ± 83,893 </td></tr><tr><td>LG07 </td><td>153 </td><td>93.8 </td><td>7.2 </td><td>19,274,282 </td><td>1.6 </td><td>205,424 ± 76,948 </td></tr><tr><td>LG08 </td><td>309 </td><td>120.7 </td><td>3.9 </td><td>16,539,538 </td><td>2.6 </td><td>137,005 ± 58,617 </td></tr><tr><td>LG09 </td><td>354 </td><td>178.5 </td><td>7.9 </td><td>35,247,141 </td><td>2.0 </td><td>197,447 ± 53,776 </td></tr><tr><td>LG10 </td><td>179 </td><td>92.4 </td><td>6.3 </td><td>20,306,549 </td><td>1.9 </td><td>219,756 ± 78,700 </td></tr><tr><td>Total </td><td>2,589 </td><td>1,268 </td><td>7.5 </td><td>274,461,215 </td><td>2.0 </td><td>216,420 ± 62,443 </td></tr><tr><td>LG01 </td><td>179 </td><td>94.1 </td><td>2.8 </td><td>35,128,701 </td><td>1.9 </td><td>373,312 ± 58,014 </td></tr><tr><td>LG02 </td><td>155 </td><td>101.1 </td><td>7.6 </td><td>35,095,207 </td><td>1.5 </td><td>347,134 ± 102,691 </td></tr><tr><td>LG03 </td><td>163 </td><td>76.9 </td><td>2.3 </td><td>29,936,525 </td><td>2.1 </td><td>389,292 ± 82,043 </td></tr><tr><td>LG04 </td><td>135 </td><td>64.2 </td><td>2.6 </td><td>28,498,427 </td><td>2.1 </td><td>443,901 ± 110,012 </td></tr><tr><td>LG05 </td><td>147 </td><td>78.1 </td><td>4.4 </td><td>33,047,117 </td><td>1.9 </td><td>423,138 ± 133,921 </td></tr><tr><td>LG06 </td><td>81 </td><td>64.0 </td><td>4.1 </td><td>21,387,731 </td><td>1.3 </td><td>334,183 ± 130,445 </td></tr><tr><td>LG07 </td><td>62 </td><td>52.6 </td><td>4.0 </td><td>19,274,282 </td><td>1.2 </td><td>366,431 ± 137,259 </td></tr><tr><td>LG08 </td><td>73 </td><td>59.2 </td><td>6.0 </td><td>16,539,538 </td><td>1.2 </td><td>279,384 ± 119,532 </td></tr><tr><td>LG09 </td><td>182 </td><td>100.9 </td><td>4.0 </td><td>35,247,141 </td><td>1.8 </td><td>349,327 ± 95,141 </td></tr><tr><td>LG10 </td><td>63 </td><td>59.5 </td><td>5.4 </td><td>20,306,549 </td><td>1.1 </td><td>341,287 ± 122,223 </td></tr><tr><td>Total </td><td>1,240 </td><td>751 </td><td>4.3 </td><td>274,461,215 </td><td>1.7 </td><td>365,656 ± 105,501 </td></tr></tbody></table></div><div content-id="DSV009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>No. of SNPs </th><th>Length (cM) </th><th>Gap length (cM) </th><th>Length (bp) </th><th>SNPs/cM </th><th>bp/cM </th></tr></thead><tbody><tr><td>LG01 </td><td>265 </td><td>134.2 </td><td>5.6 </td><td>35,128,701 </td><td>2.0 </td><td>261,764 ± 40,679 </td></tr><tr><td>LG02 </td><td>454 </td><td>178.2 </td><td>7.1 </td><td>35,095,207 </td><td>2.5 </td><td>196,976 ± 58,270 </td></tr><tr><td>LG03 </td><td>128 </td><td>107.3 </td><td>13.4 </td><td>29,936,525 </td><td>1.2 </td><td>279,100 ± 58,820 </td></tr><tr><td>LG04 </td><td>280 </td><td>146.5 </td><td>6.2 </td><td>28,498,427 </td><td>1.9 </td><td>194,555 ± 48,217 </td></tr><tr><td>LG05 </td><td>206 </td><td>117.1 </td><td>14.0 </td><td>33,047,117 </td><td>1.8 </td><td>282,230 ± 89,324 </td></tr><tr><td>LG06 </td><td>261 </td><td>99.5 </td><td>3.3 </td><td>21,387,731 </td><td>2.6 </td><td>214,924 ± 83,893 </td></tr><tr><td>LG07 </td><td>153 </td><td>93.8 </td><td>7.2 </td><td>19,274,282 </td><td>1.6 </td><td>205,424 ± 76,948 </td></tr><tr><td>LG08 </td><td>309 </td><td>120.7 </td><td>3.9 </td><td>16,539,538 </td><td>2.6 </td><td>137,005 ± 58,617 </td></tr><tr><td>LG09 </td><td>354 </td><td>178.5 </td><td>7.9 </td><td>35,247,141 </td><td>2.0 </td><td>197,447 ± 53,776 </td></tr><tr><td>LG10 </td><td>179 </td><td>92.4 </td><td>6.3 </td><td>20,306,549 </td><td>1.9 </td><td>219,756 ± 78,700 </td></tr><tr><td>Total </td><td>2,589 </td><td>1,268 </td><td>7.5 </td><td>274,461,215 </td><td>2.0 </td><td>216,420 ± 62,443 </td></tr><tr><td>LG01 </td><td>179 </td><td>94.1 </td><td>2.8 </td><td>35,128,701 </td><td>1.9 </td><td>373,312 ± 58,014 </td></tr><tr><td>LG02 </td><td>155 </td><td>101.1 </td><td>7.6 </td><td>35,095,207 </td><td>1.5 </td><td>347,134 ± 102,691 </td></tr><tr><td>LG03 </td><td>163 </td><td>76.9 </td><td>2.3 </td><td>29,936,525 </td><td>2.1 </td><td>389,292 ± 82,043 </td></tr><tr><td>LG04 </td><td>135 </td><td>64.2 </td><td>2.6 </td><td>28,498,427 </td><td>2.1 </td><td>443,901 ± 110,012 </td></tr><tr><td>LG05 </td><td>147 </td><td>78.1 </td><td>4.4 </td><td>33,047,117 </td><td>1.9 </td><td>423,138 ± 133,921 </td></tr><tr><td>LG06 </td><td>81 </td><td>64.0 </td><td>4.1 </td><td>21,387,731 </td><td>1.3 </td><td>334,183 ± 130,445 </td></tr><tr><td>LG07 </td><td>62 </td><td>52.6 </td><td>4.0 </td><td>19,274,282 </td><td>1.2 </td><td>366,431 ± 137,259 </td></tr><tr><td>LG08 </td><td>73 </td><td>59.2 </td><td>6.0 </td><td>16,539,538 </td><td>1.2 </td><td>279,384 ± 119,532 </td></tr><tr><td>LG09 </td><td>182 </td><td>100.9 </td><td>4.0 </td><td>35,247,141 </td><td>1.8 </td><td>349,327 ± 95,141 </td></tr><tr><td>LG10 </td><td>63 </td><td>59.5 </td><td>5.4 </td><td>20,306,549 </td><td>1.1 </td><td>341,287 ± 122,223 </td></tr><tr><td>Total </td><td>1,240 </td><td>751 </td><td>4.3 </td><td>274,461,215 </td><td>1.7 </td><td>365,656 ± 105,501 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4938990" target="_blank">View Large</a></div></div>                        <a id="4938991" scrollto-destination="4938991"></a>
<div id="DSV009F7" data-id="DSV009F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00907.png?Expires=2147483647&Signature=DKu4JhGtQGHipTQ~wq1PklWEqafwa9nXrq8zFGkIYxeyzbd8eIhD3btzAqmggDGXJVqC~Lgyn5DF3wsLN6CJNwDSHgaU8bRJEouy5H5iZGqfZO56~SDSP-YtmWR-nQfDN0qDk8rwIEO3-pEYfyCPpLbRw8MFo5AqsdM2CvZYlNFJc7VJfSIpOxZAw0dWHJCaK4bymJ~Pz6wbMMRQSkyqVfRpBpoRyXfqoio5uAhaqLqOETA-fAvZihzIITtF-AKRc5z62HFgP2HlH7pUaLC3hF-70ucjb80dZ3YOVcDSQe428H8sLOClTHn9XG4aBlqzege1zBV~c-kflSgb3f7KLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Pound 7 × UF273 Type II mapping population. Linkage map representing the 10 identified chromosomes of cacao. The map was generated with JoinMap 4.1 and contains 68 individuals with 2,589 markers. Markers are depicted as black lines with their position (cM) reported to the left. Linkage group designation is indicated across the top, markers are identified as horizontal lines and cM position is indicated along the left." path-from-xml="dsv00907.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938991/dsv00907.tiff" path-from-xml="dsv00907.tiff" target="_blank">View large</a><a section="4938991" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00907.png?Expires=2147483647&Signature=TBwWCAGfgaIYFj~p6tLsJjYllL4xFgOA-cqXa9eaXZLCMBinffiXeAzsOvfVWvL7kHFCsV~9NtshYa8o3qPIn1m5EnWZnSHcosVhTBe61YNGC7SlfAEQW6GRZfVYHzRUANCcxVgZZLDg79zQHDO8T3tIaqB-MpT2HKDmaTlwcI4pMx6eln-8tp83xKbsawIejXECj9j12D~EnFEurvBnnEFEn875BLQKgFAw5T6gRYP~9lb1JbT1~EtWLzOv1hkf0h2ibPdH91nIERvRX0bnlWA69gIDQX~75JwoWd0Jq3BGj22ehEu8MBcEDrs8nNGTITKPBS0V7X39yvvBnkxkkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938991&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00907.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage map of the Pound 7 × UF273 Type II mapping population. Linkage map representing the 10 identified chromosomes of cacao. The map was generated with JoinMap 4.1 and contains 68 individuals with 2,589 markers. Markers are depicted as black lines with their position (cM) reported to the left. Linkage group designation is indicated across the top, markers are identified as horizontal lines and cM position is indicated along the left.</p></div></div><div content-id="DSV009F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00907.png?Expires=2147483647&Signature=DKu4JhGtQGHipTQ~wq1PklWEqafwa9nXrq8zFGkIYxeyzbd8eIhD3btzAqmggDGXJVqC~Lgyn5DF3wsLN6CJNwDSHgaU8bRJEouy5H5iZGqfZO56~SDSP-YtmWR-nQfDN0qDk8rwIEO3-pEYfyCPpLbRw8MFo5AqsdM2CvZYlNFJc7VJfSIpOxZAw0dWHJCaK4bymJ~Pz6wbMMRQSkyqVfRpBpoRyXfqoio5uAhaqLqOETA-fAvZihzIITtF-AKRc5z62HFgP2HlH7pUaLC3hF-70ucjb80dZ3YOVcDSQe428H8sLOClTHn9XG4aBlqzege1zBV~c-kflSgb3f7KLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage map of the Pound 7 × UF273 Type II mapping population. Linkage map representing the 10 identified chromosomes of cacao. The map was generated with JoinMap 4.1 and contains 68 individuals with 2,589 markers. Markers are depicted as black lines with their position (cM) reported to the left. Linkage group designation is indicated across the top, markers are identified as horizontal lines and cM position is indicated along the left." path-from-xml="dsv00907.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938991/dsv00907.tiff" path-from-xml="dsv00907.tiff" target="_blank">View large</a><a section="4938991" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00907.png?Expires=2147483647&Signature=TBwWCAGfgaIYFj~p6tLsJjYllL4xFgOA-cqXa9eaXZLCMBinffiXeAzsOvfVWvL7kHFCsV~9NtshYa8o3qPIn1m5EnWZnSHcosVhTBe61YNGC7SlfAEQW6GRZfVYHzRUANCcxVgZZLDg79zQHDO8T3tIaqB-MpT2HKDmaTlwcI4pMx6eln-8tp83xKbsawIejXECj9j12D~EnFEurvBnnEFEn875BLQKgFAw5T6gRYP~9lb1JbT1~EtWLzOv1hkf0h2ibPdH91nIERvRX0bnlWA69gIDQX~75JwoWd0Jq3BGj22ehEu8MBcEDrs8nNGTITKPBS0V7X39yvvBnkxkkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938991&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00907.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage map of the Pound 7 × UF273 Type II mapping population. Linkage map representing the 10 identified chromosomes of cacao. The map was generated with JoinMap 4.1 and contains 68 individuals with 2,589 markers. Markers are depicted as black lines with their position (cM) reported to the left. Linkage group designation is indicated across the top, markers are identified as horizontal lines and cM position is indicated along the left.</p></div></div><p>Compared with a composite map generated with 1,240 SNPs (Table <a reveal-id="DSV009TB3" data-open="DSV009TB3" class="link link-reveal link-table xref-fig">3</a>), the total length of all linkage groups was longer. The CATIE Type 2 map has a largest gap length between markers of 14 cM on LG05 and an average longest gap length of 7.5 cM across all linkage groups. In comparison, the Allegre composite SNP map has a 7.6 cM gap (LG02) and an average longest gap length of 4.3 cM.<sup><a reveal-id="DSV009C35" data-open="DSV009C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Despite the increased number of SNPs on the CATIE Type 2 map, the gap size did not decrease. The redundancy in SNP selection for the Cacao6kSNP array is the most likely reason for this observation. Although more SNPs were present, they co-located (i.e. had the same cM position) along the map. In the CATIE Type 2 linkage map (Fig. <a reveal-id="DSV009F7" data-open="DSV009F7" class="link link-reveal link-table xref-fig">7</a>), ∼1,600 SNPs were co-located. If these SNPs were selected from additional transcripts, fewer co-locating markers, and thus smaller gaps, would be expected. Additionally, the identification of SNPs from the transcriptome limited marker positions to the gene space. As a result, gene-rich regions may contain an abundance of markers. In the future, to better distribute SNPs across each linkage group, it will be beneficial to represent more transcripts or non-transcribed SNPs, perhaps evenly positioned across the chromosome, rather than to prioritize marker redundancy.</p><p>The generation of a composite map increases the effective number of individuals within the mapping population, improving the chance of observing recombination events. The smaller size of the CATIE Type 2 population reduces the number of observed recombinations, causing markers to co-locate and generating gaps between clumps of markers. The uncertainty introduced by ordering co-located markers can also result in somewhat larger linkage groups. An estimate of base- pairs per cM was calculated for each map, first using the length of each sequenced genome separately and then averaging the resulting values together; the error presented represents the standard deviation (Table <a reveal-id="DSV009TB3" data-open="DSV009TB3" class="link link-reveal link-table xref-fig">3</a>). An average size was used, because Matina has one of the larger cacao genome sizes, whereas Criollo has one of the smaller genome sizes.<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The CATIE Type 2 map, at 2 SNPs/cM, generated an average size of 216,420 bp/cM, whereas the Allegre composite map, with 1.7 SNPs/cM, predicted 365,656 bp/cM.</p><p>Under ideal conditions, SNPs with fit and stress values (a statistical measure of whether a locus fits well between its neighbours; lower values represent a better fit) over 2 are removed from the map. To generate this map, 242 SNPs with a fit and stress value over 7 were removed; this relatively high number of markers is due to the limited number of individuals in the population, which reduces the strength of the linkages. When data from the Cacao6kSNP array is used to map a much larger population, the increased number of individuals in that population contributes to significantly improve fit and stress values for the mapped SNPs, further emphasizing that for linkage mapping, additional markers cannot completely overcome a lack of individuals. Despite this issue, the markers on the CATIE Type 2 linkage map show strong co-linearity to both the Matina 1-6 v1.1 and Criollo v1 genome assemblies (Fig. <a reveal-id="DSV009F8" data-open="DSV009F8" class="link link-reveal link-table xref-fig">8</a>). Marker positions of the linkage map were compared with the genomic positions of both assembled cacao genomes. Marker sequences for the 2,589 SNPs included on the CATIE Type 2 linkage map were blasted to each assembled genome to determine marker position, and all of the linkage groups that contain a mapped marker are displayed with identical markers connected by grey lines (Fig. <a reveal-id="DSV009F8" data-open="DSV009F8" class="link link-reveal link-table xref-fig">8</a>). Eleven linkage groups are presented for the Criollo v1 assembly as some markers were found on Tc00, a scaffold used to hold sequences that could not be placed on the original Criollo v1 assembly.<sup><a reveal-id="DSV009C45" data-open="DSV009C45" class="link link-ref link-reveal xref-bibr">45</a></sup> As such, the increased number of markers on this new linkage map may have helped to anchor additional regions of the Criollo v1 assembly (Tc00) onto the 10 chromosomes. </p>                        <a id="4938995" scrollto-destination="4938995"></a>
<div id="DSV009F8" data-id="DSV009F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00908.png?Expires=2147483647&Signature=s3fgBKThtvgn7GrFvjpRIlpjx28jYzETrFnxcld5qXXPUwulTLdJV7LXIKiKqaZ3xtVCFaKXag9dELpUFozE4Eoxmb-iJjkV7Nho4WN-yFgc7WAcAmNeatvwS5OOU84bs9gr5O~uNqv6oMmBs3PztIfTG9NNJzoIVpMlJtYM0mkjMOLbWXDc5--Mo52OQnRFpcpwpM4fJsdZVhpRYOUF-Eh5WMNcUQ7iOpjRdymYnDey4143wczcKYdE1GZbFNlcdyutusNvSY5wOlvxN9BdQkYDAgvurjSIwt-VBEBBpeLl2DP5CVkfcqY4g4sRvMjnyQZ39Hv2SuhAUhqIMvNgvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of SNP marker positions. Marker positions are depicted as grey lines and show marker position either by linkage mapping (CATIE Type 2) or blasting 121-mers containing the SNP to the genomic assemblies of Criollo23 or Matina.24 Tc00 contains unanchored sequences in the Criollo assembly. No markers present on the CATIE Type 2 map were located on unanchored sequences in the Matina assembly. This demonstrates the advantages of a more saturated genetic map in genome assembly." path-from-xml="dsv00908.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938995/dsv00908.tiff" path-from-xml="dsv00908.tiff" target="_blank">View large</a><a section="4938995" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00908.png?Expires=2147483647&Signature=kpDU7FEbkbJ0hPBbRPOqfRnTCYoSfHxRhuLz54krsa7ucHIN8rouLH0IKfEnt8JjsT5Pu~juWik4yN8TdYCj3i-Nz6tsb1N0yITiESOnW19Ia8F6aspnul6M7lgemHBnwEam0NAD4spAjHKbs6mSKzEqCJvKMg6iZRoPEY37MCyvKG0jDQJcrPLTt-IM0q0xvVHhSbcIyqgUqPbTvL5tTz2cx3v0XjwnhLlxJEBg~Av2h3vWYNMRObtMMtVIPH8-pOaVN5eoteAhnHjl6hR071huimcrXFzTQclgAnDd7MEEuScZfLbcrImuZ~d7JlfApLF58S2Az9FIY1DqVmebiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938995&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00908.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of SNP marker positions. Marker positions are depicted as grey lines and show marker position either by linkage mapping (CATIE Type 2) or blasting 121-mers containing the SNP to the genomic assemblies of Criollo<sup><a reveal-id="DSV009C23" data-open="DSV009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> or Matina.<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Tc00 contains unanchored sequences in the Criollo assembly. No markers present on the CATIE Type 2 map were located on unanchored sequences in the Matina assembly. This demonstrates the advantages of a more saturated genetic map in genome assembly.</p></div></div><div content-id="DSV009F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/m_dsv00908.png?Expires=2147483647&Signature=s3fgBKThtvgn7GrFvjpRIlpjx28jYzETrFnxcld5qXXPUwulTLdJV7LXIKiKqaZ3xtVCFaKXag9dELpUFozE4Eoxmb-iJjkV7Nho4WN-yFgc7WAcAmNeatvwS5OOU84bs9gr5O~uNqv6oMmBs3PztIfTG9NNJzoIVpMlJtYM0mkjMOLbWXDc5--Mo52OQnRFpcpwpM4fJsdZVhpRYOUF-Eh5WMNcUQ7iOpjRdymYnDey4143wczcKYdE1GZbFNlcdyutusNvSY5wOlvxN9BdQkYDAgvurjSIwt-VBEBBpeLl2DP5CVkfcqY4g4sRvMjnyQZ39Hv2SuhAUhqIMvNgvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of SNP marker positions. Marker positions are depicted as grey lines and show marker position either by linkage mapping (CATIE Type 2) or blasting 121-mers containing the SNP to the genomic assemblies of Criollo23 or Matina.24 Tc00 contains unanchored sequences in the Criollo assembly. No markers present on the CATIE Type 2 map were located on unanchored sequences in the Matina assembly. This demonstrates the advantages of a more saturated genetic map in genome assembly." path-from-xml="dsv00908.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4938995/dsv00908.tiff" path-from-xml="dsv00908.tiff" target="_blank">View large</a><a section="4938995" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv00908.png?Expires=2147483647&Signature=kpDU7FEbkbJ0hPBbRPOqfRnTCYoSfHxRhuLz54krsa7ucHIN8rouLH0IKfEnt8JjsT5Pu~juWik4yN8TdYCj3i-Nz6tsb1N0yITiESOnW19Ia8F6aspnul6M7lgemHBnwEam0NAD4spAjHKbs6mSKzEqCJvKMg6iZRoPEY37MCyvKG0jDQJcrPLTt-IM0q0xvVHhSbcIyqgUqPbTvL5tTz2cx3v0XjwnhLlxJEBg~Av2h3vWYNMRObtMMtVIPH8-pOaVN5eoteAhnHjl6hR071huimcrXFzTQclgAnDd7MEEuScZfLbcrImuZ~d7JlfApLF58S2Az9FIY1DqVmebiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4938995&ar=348084&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00908.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of SNP marker positions. Marker positions are depicted as grey lines and show marker position either by linkage mapping (CATIE Type 2) or blasting 121-mers containing the SNP to the genomic assemblies of Criollo<sup><a reveal-id="DSV009C23" data-open="DSV009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> or Matina.<sup><a reveal-id="DSV009C24" data-open="DSV009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Tc00 contains unanchored sequences in the Criollo assembly. No markers present on the CATIE Type 2 map were located on unanchored sequences in the Matina assembly. This demonstrates the advantages of a more saturated genetic map in genome assembly.</p></div></div>                            <h2 scrollto-destination=4938996 id="4938996" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>We have identified nearly 400,000 SNP markers across a cross-section of the cacao transcriptome diversity and from these constructed a Cacao6kSNP array. With this ‘chocolate chip’, 5,149 markers provided genotype data for 1,152 samples, including five mapping populations. A linkage map with 2,589 SNPs was generated for the smallest of these mapping populations. The data generated from this chip have allowed the comparison of important cacao genotypes. Additionally, we determined a minimum number of SNP markers required to distinguish closely related accessions, with the aim of helping further cacao breeding efforts. Future QTL and association studies using these larger SNP datasets promise to identify markers that can be utilized in marker-assisted selection breeding programs throughout the world to help ensure a ready supply of chocolate.</p>                            <h2 scrollto-destination=4938998 id="4938998" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4939000 id="4939000" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This project was part of the Matina 1-6 cacao sequencing project, and was funded by MARS, Inc. Funding to pay the Open Access publication charges for this article was provided by MARS, Inc.</p>                            <h2 scrollto-destination=4939002 id="4939002" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Dr Wilbert Phillips from the Centro Agronomico Tropical de Investigacion y Ensenanza (CATIE), Turrialba, Costa Rica for kindly supplying many of the tissue samples utilized in this study. We thank Dr Sean Myles for his invaluable advice and assistance editing this manuscript, as well as Mark Hansen from Illumina for his expert advice and guidance when preparing and analysing the Illumina Infinium array that would become the Cacao6kSNP chip.</p>                            <h2 scrollto-destination=4939004 id="4939004" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV009C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="collab">Food and Agriculture Organization of the United Nations</div>. <div class="year">2015</div>. <div class="comment">FAOSTAT. Crops (National Production). <a class="link link-uri" href="http://data.fao.org/ref/29920434-c74e-4ea2-beed-01b832e60609.html?version=1.0">http://data.fao.org/ref/29920434-c74e-4ea2-beed-01b832e60609.html?version=1.0</a> (6 March 2014, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Food+and+Agriculture+Organization+of+the+United+Nations.+2015.+FAOSTAT.+Crops+(National+Production).+http://data.fao.org/ref/29920434-c74e-4ea2-beed-01b832e60609.html?version=1.0+(6+March+2014,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV009C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cuatrecasas</div> <div class="given-names">J.</div></div></div> <div class="year">1964</div>, <div class="article-title">Cacao and its allies: a taxonomic revision og the genus <em>Theobroma</em></div>, <div class="source">Contrib. US Herbarium</div> , <div class="volume">35</div>, <div class="fpage">542</div>–<div class="lpage">3</div>.</div></div></div></div><div content-id="DSV009C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hunter</div> <div class="given-names">J.R.</div></div></div> <div class="year">1990</div>, <div class="article-title">The status of cacao (<em>Theobroma-Cacao</em>, Sterculiaceae) in the western-hemisphere</div>, <div class="source">Econ. Bot.</div> , <div class="volume">44</div>, <div class="fpage">425</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20status%20of%20cacao%20(Theobroma-Cacao%2C%20Sterculiaceae)%20in%20the%20western-hemisphere&amp;author=J.R.%20Hunter&amp;publication_year=1990&amp;journal=Econ.%20Bot.&amp;volume=44&amp;pages=425-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02859775" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02859775" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02859775"> </span></div> </div></div></div></div></div><div content-id="DSV009C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.M.</div></div> <div class="name"><div class="surname">Lopez</div> <div class="given-names">P.A.</div></div> <div class="name"><div class="surname">Ortiz</div> <div class="given-names">C.F.</div></div> <div class="name"><div class="surname">Moreno</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div></div> <div class="year">2002</div>, <div class="article-title">Cacao domestication I: the origin of the cacao cultivated by the Mayas</div>, <div class="source">Heredity</div> , <div class="volume">89</div>, <div class="fpage">380</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cacao%20domestication%20I%3A%20the%20origin%20of%20the%20cacao%20cultivated%20by%20the%20Mayas&amp;author=J.C.%20Motamayor&amp;author=A.M.%20Risterucci&amp;author=P.A.%20Lopez&amp;author=C.F.%20Ortiz&amp;author=A.%20Moreno&amp;author=C.%20Lanaud&amp;publication_year=2002&amp;journal=Heredity&amp;volume=89&amp;pages=380-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6800156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6800156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12399997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Fredholm</div> <div class="given-names">A.</div></div></div> <div class="year">1911</div>, <div class="source">Breeding of cacao</div> , <div class="publisher-name">The West India Committee Circular</div>, <div class="fpage">102</div>–<div class="lpage">3</div>.</div></div></div></div><div content-id="DSV009C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Lachenaud</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">da Silva e Mota</div> <div class="given-names">J.W.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Geographic and genetic population differentiation of the Amazonian chocolate tree (<em>Theobroma cacao</em> L)</div>, <div class="source">PLoS ONE</div> , <div class="volume">3</div>, <div class="fpage">e3311</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Geographic%20and%20genetic%20population%20differentiation%20of%20the%20Amazonian%20chocolate%20tree%20(Theobroma%20cacao%20L)&amp;author=J.C.%20Motamayor&amp;author=P.%20Lachenaud&amp;author=J.W.%20da%20Silva%20e%20Mota&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e3311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18827930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Warren</div> <div class="given-names">J.</div></div></div> <div class="year">1992</div>, <div class="article-title">Cocoa breeding in the 21st century</div>. In: <div class="source">International Workshop on Conservation, Characterisation and Utilisation of Cocoa Genetic Resources in the 21st Century</div> . <div class="publisher-loc">Port of Spain, Trinidad and Tobago</div>.</div></div></div></div><div content-id="DSV009C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Engels</div> <div class="given-names">J.</div></div></div> <div class="year">1983</div>, <div class="article-title">A systematic description of cacao clones. III. Relationships between clones, between characteristics and some consequences for the cacao breeding</div>, <div class="source">Euphytica</div> , <div class="volume">32</div>, <div class="fpage">719</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20systematic%20description%20of%20cacao%20clones.%20III.%20Relationships%20between%20clones%2C%20between%20characteristics%20and%20some%20consequences%20for%20the%20cacao%20breeding&amp;author=J.%20Engels&amp;publication_year=1983&amp;journal=Euphytica&amp;volume=32&amp;pages=719-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00042152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00042152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00042152"> </span></div> </div></div></div></div></div><div content-id="DSV009C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Iwaro</div> <div class="given-names">A.D.</div></div> <div class="name"><div class="surname">Butler</div> <div class="given-names">D.P.</div></div></div> <div class="year">2000</div>, <div class="article-title">Germplasm enhancement for resistance to black pod and witches' broom diseases</div>. In: <div class="source">Proceedings of the 13th International Cocoa Research Conference: Towards the Effective and Optimum Promotion of Cocoa through Research and Development</div> . <div class="publisher-name">Cocoa Producers Alliance</div>, <div class="publisher-loc">Lagos, Nigeria</div>.</div></div></div></div><div content-id="DSV009C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Boza</div> <div class="given-names">E.J.</div></div> <div class="name"><div class="surname">Irish</div> <div class="given-names">B.M.</div></div> <div class="name"><div class="surname">Meerow</div> <div class="given-names">A.W.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Genetic diversity, conservation, and utilization of <em>Theobroma cacao</em> L.: genetic resources in the Dominican Republic</div>, <div class="source">Genet. Resour. Crop Evol.</div> , <div class="volume">60</div>, <div class="fpage">605</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%2C%20conservation%2C%20and%20utilization%20of%20Theobroma%20cacao%20L.%3A%20genetic%20resources%20in%20the%20Dominican%20Republic&amp;author=E.J.%20Boza&amp;author=B.M.%20Irish&amp;author=A.W.%20Meerow&amp;publication_year=2013&amp;journal=Genet.%20Resour.%20Crop%20Evol.&amp;volume=60&amp;pages=605-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-012-9860-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-012-9860-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-012-9860-4"> </span></div> </div></div></div></div></div><div content-id="DSV009C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Irish</div> <div class="given-names">B.M.</div></div> <div class="name"><div class="surname">Goenaga</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Schnell</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Brown</div> <div class="given-names">J.S.</div></div> <div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div></div> <div class="year">2010</div>, <div class="article-title">Microsatellite fingerprinting of the USDA-ARS Tropical Agriculture Research Station Cacao (<em>Theobroma cacao</em> L.) germplasm collection</div>, <div class="source">Crop Sci.</div> , <div class="volume">50</div>, <div class="fpage">656</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20fingerprinting%20of%20the%20USDA-ARS%20Tropical%20Agriculture%20Research%20Station%20Cacao%20(Theobroma%20cacao%20L.)%20germplasm%20collection&amp;author=B.M.%20Irish&amp;author=R.%20Goenaga&amp;author=D.%20Zhang&amp;author=R.%20Schnell&amp;author=J.S.%20Brown&amp;author=J.C.%20Motamayor&amp;publication_year=2010&amp;journal=Crop%20Sci.&amp;volume=50&amp;pages=656-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2009.06.0299" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2009.06.0299" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2009.06.0299"> </span></div> </div></div></div></div></div><div content-id="DSV009C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Schnell</div> <div class="given-names">R.J.</div></div> <div class="name"><div class="surname">Olano</div> <div class="given-names">C.T.</div></div> <div class="name"><div class="surname">Brown</div> <div class="given-names">J.S.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">Retrospective determination of the parental population of superior cacao (<em>Theobroma cacao</em> L.) seedlings and association of microsatellite alleles with productivity</div>, <div class="source">J. Am. Soc. Hortic. Sci.</div> , <div class="volume">130</div>, <div class="fpage">181</div>–<div class="lpage">90</div>.</div></div></div></div><div content-id="DSV009C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.M.</div></div> <div class="name"><div class="surname">Grivet</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">N'Goran</div> <div class="given-names">J.A.K.</div></div> <div class="name"><div class="surname">Pieretti</div> <div class="given-names">I.</div></div> <div class="name"><div class="surname">Flament</div> <div class="given-names">M.H.</div></div> <div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div></div> <div class="year">2000</div>, <div class="article-title">A high-density linkage map of <em>Theobroma cacao</em> L</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">101</div>, <div class="fpage">948</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20linkage%20map%20of%20Theobroma%20cacao%20L&amp;author=A.M.%20Risterucci&amp;author=L.%20Grivet&amp;author=J.A.K.%20N'Goran&amp;author=I.%20Pieretti&amp;author=M.H.%20Flament&amp;author=C.%20Lanaud&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=101&amp;pages=948-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051566"> </span></div> </div></div></div></div></div><div content-id="DSV009C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.N.</div></div> <div class="name"><div class="surname">Heath</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Wisser</div> <div class="given-names">R.J.</div></div>et al.  </div> <div class="year">2003</div>, <div class="article-title">Resistance gene homologues in <em>Theobroma cacao</em> as useful genetic markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">107</div>, <div class="fpage">191</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resistance%20gene%20homologues%20in%20Theobroma%20cacao%20as%20useful%20genetic%20markers&amp;author=D.N.%20Kuhn&amp;author=M.%20Heath&amp;author=R.J.%20Wisser&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=191-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1239-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1239-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1239-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12845435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Borrone</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Schnell</div> <div class="given-names">R.</div></div></div> <div class="year">2004</div>, <div class="article-title">Isolation, characterization, and development of WRKY genes as useful genetic markers in <em>Theobroma cacao</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">109</div>, <div class="fpage">495</div>–<div class="lpage">507</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%2C%20characterization%2C%20and%20development%20of%20WRKY%20genes%20as%20useful%20genetic%20markers%20in%20Theobroma%20cacao&amp;author=J.%20Borrone&amp;author=D.%20Kuhn&amp;author=R.%20Schnell&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=495-507" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1662-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1662-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1662-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15148571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Schnell</div> <div class="given-names">R.J.</div></div> <div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.N.</div></div> <div class="name"><div class="surname">Brown</div> <div class="given-names">J.S.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Development of a marker assisted selection program for cacao</div>, <div class="source">Phytopathology</div> , <div class="volume">97</div>, <div class="fpage">1664</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20marker%20assisted%20selection%20program%20for%20cacao&amp;author=R.J.%20Schnell&amp;author=D.N.%20Kuhn&amp;author=J.S.%20Brown&amp;publication_year=2007&amp;journal=Phytopathology&amp;volume=97&amp;pages=1664-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO-97-12-1664" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO-97-12-1664" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO-97-12-1664"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18943731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lima</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Gramacho</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Carels</div> <div class="given-names">N.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">Single nucleotide polymorphisms from <em>Theobroma cacao</em> expressed sequence tags associated with witches’ broom disease in cacao</div>, <div class="source">Genet. Mol. Res.</div> , <div class="volume">8</div>, <div class="fpage">799</div>–<div class="lpage">808</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphisms%20from%20Theobroma%20cacao%20expressed%20sequence%20tags%20associated%20with%20witches%E2%80%99%20broom%20disease%20in%20cacao&amp;author=L.%20Lima&amp;author=K.%20Gramacho&amp;author=N.%20Carels&amp;publication_year=2009&amp;journal=Genet.%20Mol.%20Res.&amp;volume=8&amp;pages=799-808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4238/vol8-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4238%2Fvol8-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4238%2Fvol8-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19681032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.N.</div></div> <div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Meerow</div> <div class="given-names">A.W.</div></div> <div class="name"><div class="surname">Borrone</div> <div class="given-names">J.W.</div></div> <div class="name"><div class="surname">Schnell</div> <div class="given-names">R.J.</div></div></div> <div class="year">2008</div>, <div class="article-title">SSCP markers provide a useful alternative to microsatellites in genotyping and estimating genetic diversity in populations and germplasm collections of plant specialty crops</div>, <div class="source">Electrophoresis</div> , <div class="volume">29</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SSCP%20markers%20provide%20a%20useful%20alternative%20to%20microsatellites%20in%20genotyping%20and%20estimating%20genetic%20diversity%20in%20populations%20and%20germplasm%20collections%20of%20plant%20specialty%20crops&amp;author=D.N.%20Kuhn&amp;author=J.C.%20Motamayor&amp;author=A.W.%20Meerow&amp;author=J.W.%20Borrone&amp;author=R.J.%20Schnell&amp;publication_year=2008&amp;journal=Electrophoresis&amp;volume=29&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/elps.v29:19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Felps.v29:19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Felps.v29:19"> </span></div> </div></div></div></div></div><div content-id="DSV009C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Motilal</div> <div class="given-names">L.A.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.P.</div></div> <div class="name"><div class="surname">Umaharan</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Mischke</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Mooleedhar</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Meinhardt</div> <div class="given-names">L.W.</div></div></div> <div class="year">2010</div>, <div class="article-title">The relic Criollo cacao in Belize - genetic diversity and relationship with Trinitario and other cacao clones held in the International Cocoa Genebank, Trinidad</div>, <div class="source">Plant Genet. Resour. C.</div> , <div class="volume">8</div>, <div class="fpage">106</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relic%20Criollo%20cacao%20in%20Belize%20-%20genetic%20diversity%20and%20relationship%20with%20Trinitario%20and%20other%20cacao%20clones%20held%20in%20the%20International%20Cocoa%20Genebank%2C%20Trinidad&amp;author=L.A.%20Motilal&amp;author=D.P.%20Zhang&amp;author=P.%20Umaharan&amp;author=S.%20Mischke&amp;author=V.%20Mooleedhar&amp;author=L.W.%20Meinhardt&amp;publication_year=2010&amp;journal=Plant%20Genet.%20Resour.%20C.&amp;volume=8&amp;pages=106-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1479262109990232" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1479262109990232" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1479262109990232"> </span></div> </div></div></div></div></div><div content-id="DSV009C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Borrone</div> <div class="given-names">J.W.</div></div> <div class="name"><div class="surname">Brown</div> <div class="given-names">J.S.</div></div> <div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.N.</div></div> <div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Schnell</div> <div class="given-names">R.J.</div></div></div> <div class="year">2007</div>, <div class="article-title">Microsatellite markers developed from <em>Theobroma cacao</em> L. expressed sequence tags</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="volume">7</div>, <div class="fpage">236</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20developed%20from%20Theobroma%20cacao%20L.%20expressed%20sequence%20tags&amp;author=J.W.%20Borrone&amp;author=J.S.%20Brown&amp;author=D.N.%20Kuhn&amp;author=J.C.%20Motamayor&amp;author=R.J.%20Schnell&amp;publication_year=2007&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=7&amp;pages=236-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1471-8286.2006.01561.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1471-8286.2006.01561.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1471-8286.2006.01561.x"> </span></div> </div></div></div></div></div><div content-id="DSV009C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.M.</div></div> <div class="name"><div class="surname">Pieretti</div> <div class="given-names">I.</div></div> <div class="name"><div class="surname">Falque</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Bouet</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Lagoda</div> <div class="given-names">P.J.</div></div></div> <div class="year">1999</div>, <div class="article-title">Isolation and characterization of microsatellites in <em>Theobroma cacao</em> L</div>, <div class="source">Mol. Ecol.</div> , <div class="volume">8</div>, <div class="fpage">2141</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20microsatellites%20in%20Theobroma%20cacao%20L&amp;author=C.%20Lanaud&amp;author=A.M.%20Risterucci&amp;author=I.%20Pieretti&amp;author=M.%20Falque&amp;author=A.%20Bouet&amp;author=P.J.%20Lagoda&amp;publication_year=1999&amp;journal=Mol.%20Ecol.&amp;volume=8&amp;pages=2141-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-294x.1999.00802.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-294x.1999.00802.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-294x.1999.00802.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10632866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Livingstone</div> <div class="given-names">D.S.</div></div> <div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Schnell</div> <div class="given-names">R.J.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Development of single nucleotide polymorphism markers in <em>Theobroma cacao</em> and comparison to simple sequence repeat markers for genotyping of Cameroon clones</div>, <div class="source">Mol. Breed.</div> , <div class="volume">27</div>, <div class="fpage">93</div>–<div class="lpage">106</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20single%20nucleotide%20polymorphism%20markers%20in%20Theobroma%20cacao%20and%20comparison%20to%20simple%20sequence%20repeat%20markers%20for%20genotyping%20of%20Cameroon%20clones&amp;author=D.S.%20Livingstone&amp;author=J.C.%20Motamayor&amp;author=R.J.%20Schnell&amp;publication_year=2011&amp;journal=Mol.%20Breed.&amp;volume=27&amp;pages=93-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-010-9416-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-010-9416-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-010-9416-2"> </span></div> </div></div></div></div></div><div content-id="DSV009C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Salse</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">The genome of <em>Theobroma cacao</em></div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">101</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Theobroma%20cacao&amp;author=X.%20Argout&amp;author=J.%20Salse&amp;author=J.M.%20Aury&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.736" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.736" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.736"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Mockaitis</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">The genome sequence of the most widely cultivated cacao type and its use to identify candidate genes regulating pod color</div>, <div class="source">Genome Biol.</div> , <div class="volume">14</div>, <div class="fpage">R53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20most%20widely%20cultivated%20cacao%20type%20and%20its%20use%20to%20identify%20candidate%20genes%20regulating%20pod%20color&amp;author=J.C.%20Motamayor&amp;author=K.%20Mockaitis&amp;author=J.%20Schmutz&amp;publication_year=2013&amp;journal=Genome%20Biol.&amp;volume=14&amp;pages=R53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-6-r53" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-6-r53" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-6-r53"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23731509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Page</div> <div class="given-names">B.T.</div></div> <div class="name"><div class="surname">Casas</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Heaton</div> <div class="given-names">M.P.</div></div>et al.  </div> <div class="year">2002</div>, <div class="article-title">Evaluation of single-nucleotide polymorphisms in CAPN1 for association with meat tenderness in cattle</div>, <div class="source">J. Anim. Sci.</div> , <div class="volume">80</div>, <div class="fpage">3077</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20single-nucleotide%20polymorphisms%20in%20CAPN1%20for%20association%20with%20meat%20tenderness%20in%20cattle&amp;author=B.T.%20Page&amp;author=E.%20Casas&amp;author=M.P.%20Heaton&amp;publication_year=2002&amp;journal=J.%20Anim.%20Sci.&amp;volume=80&amp;pages=3077-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/2002.80123077x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2F2002.80123077x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2F2002.80123077x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12542147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ha</div> <div class="given-names">B.K.</div></div> <div class="name"><div class="surname">Hussey</div> <div class="given-names">R.S.</div></div> <div class="name"><div class="surname">Boerma</div> <div class="given-names">H.R.</div></div></div> <div class="year">2007</div>, <div class="article-title">Development of SNP assays for marker-assisted selection of two southern root-knot nematode resistance QTL in soybean</div>, <div class="source">Crop Sci.</div> , <div class="volume">47</div>, <div class="fpage">S73</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20SNP%20assays%20for%20marker-assisted%20selection%20of%20two%20southern%20root-knot%20nematode%20resistance%20QTL%20in%20soybean&amp;author=B.K.%20Ha&amp;author=R.S.%20Hussey&amp;author=H.R.%20Boerma&amp;publication_year=2007&amp;journal=Crop%20Sci.&amp;volume=47&amp;pages=S73-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2006.10.0645" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2006.10.0645" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2006.10.0645"> </span></div> </div></div></div></div></div><div content-id="DSV009C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Altshuler</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Pollara</div> <div class="given-names">V.J.</div></div> <div class="name"><div class="surname">Cowles</div> <div class="given-names">C.R.</div></div>et al.  </div> <div class="year">2000</div>, <div class="article-title">An SNP map of the human genome generated by reduced representation shotgun sequencing</div>, <div class="source">Nature</div> , <div class="volume">407</div>, <div class="fpage">513</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20SNP%20map%20of%20the%20human%20genome%20generated%20by%20reduced%20representation%20shotgun%20sequencing&amp;author=D.%20Altshuler&amp;author=V.J.%20Pollara&amp;author=C.R.%20Cowles&amp;publication_year=2000&amp;journal=Nature&amp;volume=407&amp;pages=513-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35035083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35035083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35035083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11029002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="collab">The Bovine HapMap Consortium</div> <div class="year">2009</div>, <div class="article-title">Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds</div>, <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">528</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1167936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1167936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1167936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19390050" class="link link-pub-id" target="_blank">PubMed</a></div> </div><div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=The+Bovine+HapMap+Consortium+2009,+Genome-wide+survey+of+SNP+variation+uncovers+the+genetic+structure+of+cattle+breeds,+Science,+324,+528–32." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV009C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chagné</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div> <div class="name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Genome-wide SNP detection, validation, and development of an 8K SNP array for apple</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e31745</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20detection%2C%20validation%2C%20and%20development%20of%20an%208K%20SNP%20array%20for%20apple&amp;author=D.%20Chagn%C3%A9&amp;author=R.N.%20Crowhurst&amp;author=M.%20Troggio&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e31745" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0031745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0031745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0031745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22363718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Banks</div> <div class="given-names">T.W.</div></div> <div class="name"><div class="surname">Cloutier</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, <div class="article-title">SNP discovery through next-generation sequencing and its applications</div>, <div class="source">Int. J. Plant Genomics</div> , <div class="comment">15 pages</div>.</div></div></div></div><div content-id="DSV009C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Van Tassell</div> <div class="given-names">C.P.</div></div> <div class="name"><div class="surname">Smith</div> <div class="given-names">T.P.L.</div></div> <div class="name"><div class="surname">Matukumalli</div> <div class="given-names">L.K.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries</div>, <div class="source">Nat. Meth.</div> , <div class="volume">5</div>, <div class="fpage">247</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20allele%20frequency%20estimation%20by%20deep%20sequencing%20of%20reduced%20representation%20libraries&amp;author=C.P.%20Van%20Tassell&amp;author=T.P.L.%20Smith&amp;author=L.K.%20Matukumalli&amp;publication_year=2008&amp;journal=Nat.%20Meth.&amp;volume=5&amp;pages=247-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1185"> </span></div> </div></div></div></div></div><div content-id="DSV009C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Takrama</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Dadzie</div> <div class="given-names">A.M.</div></div> <div class="name"><div class="surname">Opoku</div> <div class="given-names">S.Y.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Applying SNP marker technology in the cacao breeding programme in Ghana</div>, <div class="source">Afr. Crop Sci. J.</div> , <div class="volume">20</div>, <div class="fpage">67</div>–<div class="lpage">75</div>.</div></div></div></div><div content-id="DSV009C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.N.</div></div> <div class="name"><div class="surname">Figueira</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Lopes</div> <div class="given-names">U.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">Evaluating Theobroma grandiflorum for comparative genomic studies with <em>Theobroma cacao</em></div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">6</div>, <div class="fpage">783</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluating%20Theobroma%20grandiflorum%20for%20comparative%20genomic%20studies%20with%20Theobroma%20cacao&amp;author=D.N.%20Kuhn&amp;author=A.%20Figueira&amp;author=U.%20Lopes&amp;publication_year=2010&amp;journal=Tree%20Genet.%20Genomes&amp;volume=6&amp;pages=783-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-010-0291-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-010-0291-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-010-0291-0"> </span></div> </div></div></div></div></div><div content-id="DSV009C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Livingstone</div> <div class="given-names">D.S.</div></div> <div class="name"><div class="surname">Freeman</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Optimization of a SNP assay for genotyping <em>Theobroma cacao</em> under field conditions</div>, <div class="source">Mol. Breed.</div> , <div class="volume">30</div>, <div class="fpage">33</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20of%20a%20SNP%20assay%20for%20genotyping%20Theobroma%20cacao%20under%20field%20conditions&amp;author=D.S.%20Livingstone&amp;author=B.%20Freeman&amp;author=J.C.%20Motamayor&amp;publication_year=2012&amp;journal=Mol.%20Breed.&amp;volume=30&amp;pages=33-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9596-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9596-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9596-4"> </span></div> </div></div></div></div></div><div content-id="DSV009C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Allegre</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Boccara</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of <em>Theobroma cacao</em> L</div>, <div class="source">DNA Res.</div> , <div class="volume">19</div>, <div class="fpage">23</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20and%20mapping%20of%20a%20new%20expressed%20sequence%20tag-single%20nucleotide%20polymorphism%20and%20simple%20sequence%20repeat%20panel%20for%20large-scale%20genetic%20studies%20and%20breeding%20of%20Theobroma%20cacao%20L&amp;author=M.%20Allegre&amp;author=X.%20Argout&amp;author=M.%20Boccara&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=23-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22210604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Fang</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Meinhardt</div> <div class="given-names">L.W.</div></div> <div class="name"><div class="surname">Mischke</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Bellato</div> <div class="given-names">C.M.</div></div> <div class="name"><div class="surname">Motilal</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div></div> <div class="year">2013</div>, <div class="article-title">Accurate determination of genetic identity for a single cacao bean, using molecular markers with a nanofluidic system, ensures cocoa authentication</div>, <div class="source">J. Agric. Food Chem.</div> , <div class="volume">62</div>, <div class="fpage">481</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20determination%20of%20genetic%20identity%20for%20a%20single%20cacao%20bean%2C%20using%20molecular%20markers%20with%20a%20nanofluidic%20system%2C%20ensures%20cocoa%20authentication&amp;author=W.%20Fang&amp;author=L.W.%20Meinhardt&amp;author=S.%20Mischke&amp;author=C.M.%20Bellato&amp;author=L.%20Motilal&amp;author=D.%20Zhang&amp;publication_year=2013&amp;journal=J.%20Agric.%20Food%20Chem.&amp;volume=62&amp;pages=481-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jf404402v" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf404402v" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjf404402v"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24354624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Antanaviciute</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Fernández-Fernández</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Jansen</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Development of a dense SNP-based linkage map of an apple rootstock progeny using the Malus Infinium whole genome genotyping array</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20dense%20SNP-based%20linkage%20map%20of%20an%20apple%20rootstock%20progeny%20using%20the%20Malus%20Infinium%20whole%20genome%20genotyping%20array&amp;author=L.%20Antanaviciute&amp;author=F.%20Fern%C3%A1ndez-Fern%C3%A1ndez&amp;author=J.%20Jansen&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22631220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Puryear</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Cairney</div> <div class="given-names">J.</div></div></div> <div class="year">1993</div>, <div class="article-title">A simple and efficient method for isolating RNA from pine trees</div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="volume">11</div>, <div class="fpage">113</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20and%20efficient%20method%20for%20isolating%20RNA%20from%20pine%20trees&amp;author=S.%20Chang&amp;author=J.%20Puryear&amp;author=J.%20Cairney&amp;publication_year=1993&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=11&amp;pages=113-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02670468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02670468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02670468"> </span></div> </div></div></div></div></div><div content-id="DSV009C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">B.A.</div></div> <div class="name"><div class="surname">Strem</div> <div class="given-names">M.D.</div></div> <div class="name"><div class="surname">Bae</div> <div class="given-names">H.H.</div></div> <div class="name"><div class="surname">de Mayolo</div> <div class="given-names">G.A.</div></div> <div class="name"><div class="surname">Guiltinan</div> <div class="given-names">M.J.</div></div></div> <div class="year">2005</div>, <div class="article-title">Gene expression in leaves of <em>Theobroma cacao</em> in response to mechanical wounding, ethylene, and/or methyl jasmonate</div>, <div class="source">Plant Sci.</div> , <div class="volume">168</div>, <div class="fpage">1247</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20in%20leaves%20of%20Theobroma%20cacao%20in%20response%20to%20mechanical%20wounding%2C%20ethylene%2C%20and%2For%20methyl%20jasmonate&amp;author=B.A.%20Bailey&amp;author=M.D.%20Strem&amp;author=H.H.%20Bae&amp;author=G.A.%20de%20Mayolo&amp;author=M.J.%20Guiltinan&amp;publication_year=2005&amp;journal=Plant%20Sci.&amp;volume=168&amp;pages=1247-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2005.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2005.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2005.01.002"> </span></div> </div></div></div></div></div><div content-id="DSV009C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.N.</div></div> <div class="name"><div class="surname">Livingstone</div> <div class="given-names">D.</div>III</div> <div class="name"><div class="surname">Main</div> <div class="given-names">D.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Identification and mapping of conserved ortholog set (COS) II sequences of cacao and their conversion to SNP markers for marker-assisted selection in <em>Theobroma cacao</em> and comparative genomics studies</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">8</div>, <div class="fpage">97</div>–<div class="lpage">111</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20mapping%20of%20conserved%20ortholog%20set%20(COS)%20II%20sequences%20of%20cacao%20and%20their%20conversion%20to%20SNP%20markers%20for%20marker-assisted%20selection%20in%20Theobroma%20cacao%20and%20comparative%20genomics%20studies&amp;author=D.N.%20Kuhn&amp;author=D.%20Livingstone&amp;author=D.%20Main&amp;publication_year=2012&amp;journal=Tree%20Genet.%20Genomes&amp;volume=8&amp;pages=97-111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-011-0424-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-011-0424-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-011-0424-0"> </span></div> </div></div></div></div></div><div content-id="DSV009C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">N.A.</div></div> <div class="name"><div class="surname">Kingsmore</div> <div class="given-names">S.F.</div></div> <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Management of high-throughput DNA sequencing projects: alpheus</div>, <div class="source">J. Comput. Sci. Syst. Biol.</div> , <div class="volume">1</div>, <div class="fpage">132</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Management%20of%20high-throughput%20DNA%20sequencing%20projects%3A%20alpheus&amp;author=N.A.%20Miller&amp;author=S.F.%20Kingsmore&amp;author=A.%20Farmer&amp;publication_year=2008&amp;journal=J.%20Comput.%20Sci.%20Syst.%20Biol.&amp;volume=1&amp;pages=132" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20151039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Purcell</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Neale</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Todd-Brown</div> <div class="given-names">K.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">81</div>, <div class="fpage">559</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd-Brown&amp;publication_year=2007&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=81&amp;pages=559-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/519795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F519795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F519795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17701901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="collab">RC Team</div> <div class="year">2014</div>, <div class="source">R: a language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing; 2012</div> . <div class="comment"><a class="link link-uri" href="http://cran.r-project.org">http://cran.r-project.org</a>. (20 May 2015, date last accessed)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=RC+Team+2014,+R:+a+language+and+environment+for+statistical+computing.+Vienna,+Austria:+R+Foundation+for+Statistical+Computing;+2012.+http://cran.r-project.org.+(20+May+2015,+date+last+accessed)." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSV009C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Stam</div> <div class="given-names">P.</div></div></div> <div class="year">1993</div>, <div class="article-title">Construction of integrated genetic linkage maps by means of a new computer package: Join Map</div>, <div class="source">Plant J.</div> , <div class="volume">3</div>, <div class="fpage">739</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20integrated%20genetic%20linkage%20maps%20by%20means%20of%20a%20new%20computer%20package%3A%20Join%20Map&amp;author=P.%20Stam&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=739-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1993.3.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1993.3.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1993.3.issue-5"> </span></div> </div></div></div></div></div><div content-id="DSV009C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Fouet</div> <div class="given-names">O.</div></div> <div class="name"><div class="surname">Wincker</div> <div class="given-names">P.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Towards the understanding of the cocoa transcriptome: production and analysis of an exhaustive dataset of ESTs of <em>Theobroma cacao</em> L. generated from various tissues and under various conditions</div>, <div class="source">BMC Genomics</div> , <div class="volume">9</div>, <div class="fpage">512</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20the%20understanding%20of%20the%20cocoa%20transcriptome%3A%20production%20and%20analysis%20of%20an%20exhaustive%20dataset%20of%20ESTs%20of%20Theobroma%20cacao%20L.%20generated%20from%20various%20tissues%20and%20under%20various%20conditions&amp;author=X.%20Argout&amp;author=O.%20Fouet&amp;author=P.%20Wincker&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=512" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-512"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18973681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.M.</div></div> <div class="name"><div class="surname">Heath</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div></div> <div class="year">2003</div>, <div class="article-title">Cacao domestication II: progenitor germplasm of the Trinitario cacao cultivar</div>, <div class="source">Heredity</div> , <div class="volume">91</div>, <div class="fpage">322</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cacao%20domestication%20II%3A%20progenitor%20germplasm%20of%20the%20Trinitario%20cacao%20cultivar&amp;author=J.C.%20Motamayor&amp;author=A.M.%20Risterucci&amp;author=M.%20Heath&amp;author=C.%20Lanaud&amp;publication_year=2003&amp;journal=Heredity&amp;volume=91&amp;pages=322-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6800298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6800298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12939635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV009C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">J.S.</div></div> <div class="name"><div class="surname">Phillips-Mora</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Power</div> <div class="given-names">E.J.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Mapping QTLs for resistance to frosty pod and black pod diseases and horticultural traits in <em>Theobroma cacao</em> L</div>, <div class="source">Crop Sci.</div> , <div class="volume">47</div>, <div class="fpage">1851</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20QTLs%20for%20resistance%20to%20frosty%20pod%20and%20black%20pod%20diseases%20and%20horticultural%20traits%20in%20Theobroma%20cacao%20L&amp;author=J.S.%20Brown&amp;author=W.%20Phillips-Mora&amp;author=E.J.%20Power&amp;publication_year=2007&amp;journal=Crop%20Sci.&amp;volume=47&amp;pages=1851-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2006.11.0753" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2006.11.0753" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2006.11.0753"> </span></div> </div></div></div></div></div><div content-id="DSV009C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Sautter</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Olano</div> <div class="given-names">C.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">A composite linkage map from three crosses between commercial clones of cacao, <em>Theobroma cacao</em> L</div>, <div class="source">Trop. Plant Biol.</div> , <div class="volume">1</div>, <div class="fpage">120</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20composite%20linkage%20map%20from%20three%20crosses%20between%20commercial%20clones%20of%20cacao%2C%20Theobroma%20cacao%20L&amp;author=J.%20Brown&amp;author=R.%20Sautter&amp;author=C.%20Olano&amp;publication_year=2008&amp;journal=Trop.%20Plant%20Biol.&amp;volume=1&amp;pages=120-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12042-008-9011-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12042-008-9011-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12042-008-9011-4"> </span></div> </div></div></div></div></div><div content-id="DSV009C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV009C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">N'goran</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">1995</div>, <div class="article-title">A genetic linkage map of <em>Theobroma cacao</em> L</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">91</div>, <div class="fpage">987</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20Theobroma%20cacao%20L&amp;author=C.%20Lanaud&amp;author=A.%20Risterucci&amp;author=A.%20N'goran&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=91&amp;pages=987-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348084&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv009-dsv009supp_table3.xlsx?Expires=2147483647&amp;Signature=PlW8U2Sanhq91dMY5YETTMb4A3Rf~P-Qyhl6f9fc4lVM2USaIv687ZRr~SiTWzmqfxqsVMTgJ9hBvi1A4HW4e1u6691Yfgxc9g~jnaRwLMalNcEEYQSLvJtwWCMQBnJEKu0ded-2gPjFXZ47SzfOLquX6dna5rGtYx7wM8Nno9KsiZA0b7FIcNXmSdIZKE1lT3h7RdJfSTCz943INPsk3RYpiPo5kGU6v5HArWXicD8OEIxWNHwMj0vq4-79HrdIyz6-2zbZ0NE51~QoahcB0YFc3zVv5oR8cisMKMC1ZImM5na1Zajko7e~S~RCUevSkvRuLHEaxx3mqsLLY1IXfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv009-dsv009supp_table2.xlsx?Expires=2147483647&amp;Signature=Pc9cWRgoog6HSsxj2CQuK4hwJFebZ8Ho-5ll6wGlhFcUZek5RtkivO87bv6SoJhdHUPsbkOFPFvL7705wUx9a4nH0nPO5LztvNSCm4goxGyK9rMVIkb4vg7fQXQ1dQ7~MpfpwnqHRRQ8nIkdWCrgmOju3pfOEB~9r56F4UIxhMgR3UtSLXWYNsy8EogQ8LcfO5arRG7sYcIGPCWl-CvO9AaJ7JG3Q-7sXajyzJv7JntxYFNnLtTCMueEFr~0BRETyDeXpAxUFCkW2U9xc5SrhoQY76f3FU~CRuOkK5rbChBscY3JLjSgJtmpTdedCywtXf5dFwygzUqG2C2jzQZkoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv009-dsv009supp_table1.xlsx?Expires=2147483647&amp;Signature=g7~vfLKuacL6pd53V4UIKzBa2kENf3hAMrwSm4-n1ff7TyWJ0W7LKvX23aNXfa2FQnDzflWbFESDrWRLuZOR~lM0fV6iQ72mBwPYkc-0cn16oepB8AZtaS6o~LlC~~JLKCHxqVGkDVTQTyrd238lH-nySlynQJ2UoTjhrLtcrW2xlOISlloSgR4P1SnNhydhx0QJZXNt34rfTOZDXBgV-O9kTJrCMpNqt~iijITavNQFMPJvhRK4JEOh4c3SC1xLz3m01NLa-X4sPuDZY3vklPYGP7wonRLS2uWBey5c01SErEvE4po4B2fbtbywrqKjW~G5j8Mh5u3KLdDfNtVGkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv009/2/dsv009-dsv009supp_fig1.doc?Expires=2147483647&amp;Signature=rcuSIdhSvwpCmPJdtKrCBIuvt94-nf46Yp9Zp5OPlJ854scpn72~qGgyYIrPOf645iXN2fQA4mGzINq5JuKdIe3H00PRc-i5zsqF5nuINqqBDyuiv3j2qzWzZN9Vol01JC2z1grzoqRX~iCgqV5lnKKohQOBt-yxBRl4SyzRx7c1J8PLPOLL-HrbMTDOKHknNt3TuV770AeCO0n-Am~0-NHxfxmLjz6mOTCOY6KAoNkIwicGaXk8WS6v9LjpTCmu3Y4SSomys6GhJ7JBFhREvyqC90Pg6avenkkJ6mPQ8Lb8xUsVEZ4zqLYsZIWm58zy-IpxeiooaP9tmAzb2wlQvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">624</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348084"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f4%2f279%2f348084">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362346000004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F4%2F279%2F348084" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348084'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Making a chocolate chip: development and evaluation of a 6K SNP array for Theobroma cacao" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f279%2f348084">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

