

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/623/3953933","@type":"ScholarlyArticle","name":"Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains","datePublished":"2017-07-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx030","keywords":["translation rate","co-adaption","amino acid usage","tRNA gene copy number","minimum cost"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/623/3953933","author":[{"name":"Du, Meng-Ze","affiliation":"School of Life Science and Technology","@type":"Person"},{"name":"Wei, Wen","affiliation":"School of Life Science and Technology","@type":"Person"},{"name":"Qin, Lei","affiliation":"School of Life Science and Technology","@type":"Person"},{"name":"Liu, Shuo","affiliation":"School of Life Science and Technology","@type":"Person"},{"name":"Zhang, An-Ying","affiliation":"School of Life Science and Technology  Centre for Informational Biology","@type":"Person"},{"name":"Zhang, Yong","affiliation":"School of Life Science and Technology  Centre for Informational Biology","@type":"Person"},{"name":"Zhou, Hong","affiliation":"School of Life Science and Technology  Centre for Informational Biology","@type":"Person"},{"name":"Guo, Feng-Biao","affiliation":"School of Life Science and Technology  Centre for Informational Biology  Key Laboratory for Neuroinformation of the Ministry of Education, University of Electronic Science and Technology of China, Chengdu, China","@type":"Person"}],"description":"Abstract.  Although more and more entangled participants of translation process were realized, how they cooperate and co-determine the final translation efficie","pageStart":"623","pageEnd":"633","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains" />
    <meta property="og:description" content="Abstract.  Although more and more entangled participants of translation process were realized, how they cooperate and co-determine the final translation efficie" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/623/3953933" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-07-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Du, Meng-Ze" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author" content="Wei, Wen" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author" content="Qin, Lei" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author" content="Liu, Shuo" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author" content="Zhang, An-Ying" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author_institution" content="Centre for Informational Biology" /><meta name="citation_author" content="Zhang, Yong" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author_institution" content="Centre for Informational Biology" /><meta name="citation_author" content="Zhou, Hong" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author_institution" content="Centre for Informational Biology" /><meta name="citation_author" content="Guo, Feng-Biao" /><meta name="citation_author_institution" content="School of Life Science and Technology" /><meta name="citation_author_institution" content="Centre for Informational Biology" /><meta name="citation_author_institution" content="Key Laboratory for Neuroinformation of the Ministry of Education, University of Electronic Science and Technology of China, Chengdu, China" /><meta name="citation_title" content="Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains" /><meta name="citation_firstpage" content="623" /><meta name="citation_lastpage" content="633" /><meta name="citation_doi" content="10.1093/dnares/dsx030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Determinants of translation efficiency and accuracy,;  citation_journal_title=Mol. Sys. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=481" /><meta name="citation_reference" content="citation_title=Codon-by-codon modulation of translational speed and accuracy via mRNA folding,;  citation_journal_title=PLoS Biol;  citation_year=2014;  citation_volume=12;  citation_pages=e1001910" /><meta name="citation_reference" content="citation_title=Optimality and evolutionary tuning of the expression level of a protein,;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=436;  citation_pages=588-92; " /><meta name="citation_reference" content="citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications,;  citation_journal_title=Nucleic Acids Res;  citation_year=1987;  citation_volume=15;  citation_pages=1281-95; " /><meta name="citation_reference" content="citation_title=Solving the riddle of codon usage preferences: a test for translational selection,;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=5036-44; " /><meta name="citation_reference" content="citation_title=Ribosome footprint profiling of translation throughout the genome,;  citation_journal_title=Cell;  citation_year=2016;  citation_volume=165;  citation_pages=22-33; " /><meta name="citation_reference" content="citation_title=Dynamics of translation of single mRNA molecules in vivo,;  citation_journal_title=Cell;  citation_year=2016;  citation_volume=165;  citation_pages=976-89; " /><meta name="citation_reference" content="citation_title=How the sequence of a gene can tune its translation,;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=227-29; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation,;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-54; " /><meta name="citation_reference" content="citation_title=Adjacent codons act in concert to modulate translation efficiency in yeast,;  citation_journal_title=Cell;  citation_year=2016;  citation_volume=166;  citation_pages=679-90; " /><meta name="citation_reference" content="citation_title=Selective charging of tRNA isoacceptors explains patterns of codon usage,;  citation_journal_title=Science;  citation_year=2003;  citation_volume=300;  citation_pages=1718-22; " /><meta name="citation_reference" content="citation_title=GtRNAdb: a database of transfer RNA genes detected in genomic sequence,;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=D93-97; " /><meta name="citation_reference" content="citation_title=Optimization of codon translation rates via tRNA modifications maintains proteome integrity,;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=161;  citation_pages=1606-18; " /><meta name="citation_reference" content="citation_title=A role for codon order in translation dynamics,;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=355-67; " /><meta name="citation_reference" content="citation_title=Tissue-specific differences in human transfer RNA expression,;  citation_journal_title=PLoS Genet;  citation_year=2006;  citation_volume=2;  citation_pages=e221" /><meta name="citation_reference" content="citation_title=The ribosome uses two active mechanisms to unwind messenger RNA during translation,;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=118-21; " /><meta name="citation_reference" content="citation_title=The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria,;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=484;  citation_pages=538-41; " /><meta name="citation_reference" content="citation_title=Rate-limiting steps in yeast protein translation,;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=153;  citation_pages=1589-1601; " /><meta name="citation_reference" content="citation_title=Codon usage influences the local rate of translation elongation to regulate co-translational protein folding,;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=744-54; " /><meta name="citation_reference" content="citation_title=Translation efficiency is determined by both codon bias and folding energy,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=3645-50; " /><meta name="citation_reference" content="citation_title=Structural biology: Wobble puts RNA on target,;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=510;  citation_pages=480-81; " /><meta name="citation_reference" content="citation_title=An integrated approach reveals regulatory controls on bacterial translation elongation,;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=159;  citation_pages=1200-11; " /><meta name="citation_reference" content="citation_title=New structural insights into translational miscoding;  citation_journal_title=Trends Biochemi. Sci;  citation_year=2016;  citation_volume=41;  citation_pages=798-814; " /><meta name="citation_reference" content="citation_title=Translation initiation is controlled by RNA folding kinetics via a ribosome drafting mechanism;  citation_journal_title=J. Am. Chem. Soc;  citation_year=2016;  citation_volume=138;  citation_pages=7016" /><meta name="citation_reference" content="citation_title=Translation dynamics of single mRNAs in live cells and neurons;  citation_journal_title=Science;  citation_year=2016;  citation_volume=352;  citation_pages=1430" /><meta name="citation_reference" content="citation_title=Starvation for different nutrients in Escherichia coli results in differential modulation of RpoS levels and stability,;  citation_journal_title=J. Bacteriol;  citation_year=2005;  citation_volume=187;  citation_pages=434-442; " /><meta name="citation_reference" content="citation_publisher=Wiley, New York;  citation_title=Current protocols in molecular biology New York;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=GenBank,;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D36-D42; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=tRNAdb 2009: compilation of tRNA sequences and tRNA genes,;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=D159-D162; " /><meta name="citation_reference" content="citation_title=PaxDb, a database of protein abundance averages across all three domains of life,;  citation_journal_title=Mol. Cell. Proteomics;  citation_year=2012;  citation_volume=11;  citation_pages=492-500; " /><meta name="citation_reference" content="citation_title=Translational selection in human: more pronounced in housekeeping genes,;  citation_journal_title=Biol. Direct;  citation_year=2014;  citation_volume=9;  citation_pages=17" /><meta name="citation_reference" content="citation_title=Old genes experience stronger translational selection than young genes,;  citation_journal_title=Gene;  citation_year=2016;  citation_volume=590;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_title=Genomic complexity places less restrictions on the evolution of young coexpression networks than protein–protein interactions,;  citation_journal_title=Genome Biol. Evol;  citation_year=2016;  citation_volume=8;  citation_pages=2624-31; " /><meta name="citation_reference" content="citation_title=HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=187-9; " /><meta name="citation_reference" content="citation_title=Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization,;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=2279-86; " /><meta name="citation_reference" content="citation_title=Levels of tRNAs in bacterial cells as affected by amino acid usage in proteins,;  citation_journal_title=Nucleic acids Res;  citation_year=1991;  citation_volume=19;  citation_pages=6119-6122; " /><meta name="citation_reference" content="citation_title=Nature of the protein universe,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=11079-11084; " /><meta name="citation_reference" content="citation_title=Growth, metabolic partitioning, and the size of microorganisms,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=495-500; " /><meta name="citation_reference" content="citation_title=Evolutionary tradeoffs in cellular composition across diverse bacteria,;  citation_journal_title=ISME J;  citation_year=2016;  citation_volume=1010;  citation_pages=21452145-21572157; " /><meta name="citation_reference" content="citation_title=The effect of tRNA levels on decoding times of mRNA codons,;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=9171-81; " /><meta name="citation_reference" content="citation_title=Toxin-resistant sodium channels: parallel adaptive evolution across a complete gene family,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2008;  citation_volume=25;  citation_pages=1016-24; " /><meta name="citation_reference" content="citation_title=Direct binding of FtsZ to ZipA, an essential component of the septal ring structure that mediates cell division in E. coli;  citation_journal_title=Cell;  citation_year=1997;  citation_volume=88;  citation_pages=175-85; " /><meta name="citation_reference" content="citation_title=The two-component system,;  citation_journal_title=Regulation of diverse signaling pathways in prokaryotes and eukaryotes. Plant Physiol;  citation_year=1998;  citation_volume=117;  citation_pages=723-31; " /><meta name="citation_reference" content="citation_title=DNA mismatch repair,;  citation_journal_title=Annu. Rev. Biochem;  citation_year=2005;  citation_volume=74;  citation_pages=681-710; " /><meta name="citation_reference" content="citation_title=Global control of sugar metabolism: a gram-positive solution;  citation_journal_title=Antonie Van Leeuwenhoek;  citation_year=2002;  citation_volume=82;  citation_pages=59-71; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling,;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=218-223; " /><meta name="citation_reference" content="citation_title=Improvement of human interferon HUIFNalpha2 and HCV core protein expression levels in Escherichia coli but not of HUIFNalpha8 by using the tRNA(AGA/AGG);  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=2002;  citation_volume=296;  citation_pages=1303-1309; " /><meta name="citation_reference" content="citation_title=Bacterial growth laws reflect the evolutionary importance of energy efficiency,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=406-411; " /><meta name="citation_reference" content="citation_title=Metabolic modelling in a dynamic evolutionary framework predicts adaptive diversification of bacteria in a long-term evolution experiment;  citation_journal_title=BMC Evol. Biol;  citation_year=2016;  citation_volume=16;  citation_pages=163" /><meta name="citation_reference" content="citation_title=Codon bias as a means to fine-tune gene expression,;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=149-61; " /><meta name="citation_reference" content="citation_title=tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes,;  citation_journal_title=Trends Genet;  citation_year=2000;  citation_volume=16;  citation_pages=287-89; " /><meta name="citation_reference" content="citation_title=Environmental perturbations lift the degeneracy of the genetic code to regulate protein levels in bacteria;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=2419-24; " /><meta name="citation_reference" content="citation_title=Codon usage is an important determinant of gene expression levels largely through its effects on transcription,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2016;  citation_volume=113;  citation_pages=E6117" /><meta name="citation_reference" content="citation_title=Gene expression levels are correlated with synonymous codon usage, amino acid composition, and gene architecture in the red flour beetle, Tribolium castaneum,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=3755-66; " /><meta name="citation_reference" content="citation_title=Amino Acid metabolism conflicts with protein diversity,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2014;  citation_volume=31;  citation_pages=2905-12; " /><meta name="citation_reference" content="citation_title=Coevolution of codon usage and tRNA genes leads to alternative stable states of biased codon usage,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2008;  citation_volume=25;  citation_pages=2279-91; " /><meta name="citation_reference" content="citation_title=The modern synthesis in the light of microbial genomics,;  citation_journal_title=Annu. Rev. Microbiol;  citation_year=2016;  citation_volume=70;  citation_pages=279-297; " /><meta name="citation_reference" content="citation_title=The origins of genome complexity,;  citation_journal_title=Science;  citation_year=2003;  citation_volume=302;  citation_pages=1401-4; " /><meta name="citation_reference" content="citation_title=Origins and principles of translational control;  citation_journal_title=Cold Spring Harbor Monogr. Arch;  citation_year=2000;  citation_volume=39;  citation_pages=1-31; " /><meta name="citation_reference" content="citation_title=mRNA composition and control of bacterial gene expression,;  citation_journal_title=J. Bacteriol;  citation_year=2000;  citation_volume=182;  citation_pages=3037-3044; " /><meta name="citation_reference" content="citation_title=Comparison of two codon optimization strategies to enhance recombinant protein production in Escherichia coli,;  citation_journal_title=Microb. Cell Fact;  citation_year=2011;  citation_volume=10;  citation_issue=1; " /><meta name="citation_reference" content="citation_title=Design and synthesis of a minimal bacterial genome,;  citation_journal_title=Science;  citation_year=2016;  citation_volume=351;  citation_pages=aad6253" /><meta name="citation_reference" content="citation_title=The minimal genome comes of age,;  citation_journal_title=Nat. Biotechnol;  citation_year=2016;  citation_volume=34;  citation_pages=623-24; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/623/22295803/dsx030.pdf" /><meta name="description" content="Abstract.  Although more and more entangled participants of translation process were realized, how they cooperate and co-determine the final translation efficie" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/623/3953933" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/623/3953933" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/623/3953933");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx030';
    </script>
<title>
	Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3953933" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfGPATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRHUsIE1lbmctWmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2VpLCBXZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUWluLCBMZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBTaHVvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIExpZmUgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBBbi1ZaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIExpZmUgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBmb3IgSW5mb3JtYXRpb25hbCBCaW9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIFlvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGZvciBJbmZvcm1hdGlvbmFsIEJpb2xvZ3kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaG91LCBIb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIExpZmUgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBmb3IgSW5mb3JtYXRpb25hbCBCaW9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3VvLCBGZW5nLUJpYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGZvciBJbmZvcm1hdGlvbmFsIEJpb2xvZ3kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBmb3IgTmV1cm9pbmZvcm1hdGlvbiBvZiB0aGUgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBVbml2ZXJzaXR5IG9mIEVsZWN0cm9uaWMgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBvZiBDaGluYSwgQ2hlbmdkdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNvLWFkYXB0aW9uIG9mIHRSTkEgZ2VuZSBjb3B5IG51bWJlciBhbmQgYW1pbm8gYWNpZCB1c2FnZSBpbmZsdWVuY2VzIHRyYW5zbGF0aW9uIHJhdGVzIGluIHRocmVlIGxpZmUgZG9tYWlucyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjYyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNjMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlcm1pbmFudHMgb2YgdHJhbnNsYXRpb24gZWZmaWNpZW5jeSBhbmQgYWNjdXJhY3ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5cy4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NDgxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24tYnktY29kb24gbW9kdWxhdGlvbiBvZiB0cmFuc2xhdGlvbmFsIHNwZWVkIGFuZCBhY2N1cmFjeSB2aWEgbVJOQSBmb2xkaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDE5MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWFsaXR5IGFuZCBldm9sdXRpb25hcnkgdHVuaW5nIG9mIHRoZSBleHByZXNzaW9uIGxldmVsIG9mIGEgcHJvdGVpbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzY7ICBjaXRhdGlvbl9wYWdlcz01ODgtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2RvbiBhZGFwdGF0aW9uIGluZGV4LWEgbWVhc3VyZSBvZiBkaXJlY3Rpb25hbCBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGJpYXMsIGFuZCBpdHMgcG90ZW50aWFsIGFwcGxpY2F0aW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTI4MS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29sdmluZyB0aGUgcmlkZGxlIG9mIGNvZG9uIHVzYWdlIHByZWZlcmVuY2VzOiBhIHRlc3QgZm9yIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz01MDM2LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaWJvc29tZSBmb290cHJpbnQgcHJvZmlsaW5nIG9mIHRyYW5zbGF0aW9uIHRocm91Z2hvdXQgdGhlIGdlbm9tZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9MjItMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWNzIG9mIHRyYW5zbGF0aW9uIG9mIHNpbmdsZSBtUk5BIG1vbGVjdWxlcyBpbiB2aXZvLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjU7ICBjaXRhdGlvbl9wYWdlcz05NzYtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvdyB0aGUgc2VxdWVuY2Ugb2YgYSBnZW5lIGNhbiB0dW5lIGl0cyB0cmFuc2xhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9MjI3LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcmlseSBjb25zZXJ2ZWQgbWVjaGFuaXNtIGZvciBjb250cm9sbGluZyB0aGUgZWZmaWNpZW5jeSBvZiBwcm90ZWluIHRyYW5zbGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0zNDQtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkamFjZW50IGNvZG9ucyBhY3QgaW4gY29uY2VydCB0byBtb2R1bGF0ZSB0cmFuc2xhdGlvbiBlZmZpY2llbmN5IGluIHllYXN0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz02NzktOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGl2ZSBjaGFyZ2luZyBvZiB0Uk5BIGlzb2FjY2VwdG9ycyBleHBsYWlucyBwYXR0ZXJucyBvZiBjb2RvbiB1c2FnZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzAwOyAgY2l0YXRpb25fcGFnZXM9MTcxOC0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3RSTkFkYjogYSBkYXRhYmFzZSBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgZGV0ZWN0ZWQgaW4gZ2Vub21pYyBzZXF1ZW5jZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9RDkzLTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWl6YXRpb24gb2YgY29kb24gdHJhbnNsYXRpb24gcmF0ZXMgdmlhIHRSTkEgbW9kaWZpY2F0aW9ucyBtYWludGFpbnMgcHJvdGVvbWUgaW50ZWdyaXR5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjE7ICBjaXRhdGlvbl9wYWdlcz0xNjA2LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJvbGUgZm9yIGNvZG9uIG9yZGVyIGluIHRyYW5zbGF0aW9uIGR5bmFtaWNzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0zNTUtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRpc3N1ZS1zcGVjaWZpYyBkaWZmZXJlbmNlcyBpbiBodW1hbiB0cmFuc2ZlciBSTkEgZXhwcmVzc2lvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUyMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmlib3NvbWUgdXNlcyB0d28gYWN0aXZlIG1lY2hhbmlzbXMgdG8gdW53aW5kIG1lc3NlbmdlciBSTkEgZHVyaW5nIHRyYW5zbGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ3NTsgIGNpdGF0aW9uX3BhZ2VzPTExOC0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFudGktU2hpbmUtRGFsZ2Fybm8gc2VxdWVuY2UgZHJpdmVzIHRyYW5zbGF0aW9uYWwgcGF1c2luZyBhbmQgY29kb24gY2hvaWNlIGluIGJhY3RlcmlhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4NDsgIGNpdGF0aW9uX3BhZ2VzPTUzOC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmF0ZS1saW1pdGluZyBzdGVwcyBpbiB5ZWFzdCBwcm90ZWluIHRyYW5zbGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTM7ICBjaXRhdGlvbl9wYWdlcz0xNTg5LTE2MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGluZmx1ZW5jZXMgdGhlIGxvY2FsIHJhdGUgb2YgdHJhbnNsYXRpb24gZWxvbmdhdGlvbiB0byByZWd1bGF0ZSBjby10cmFuc2xhdGlvbmFsIHByb3RlaW4gZm9sZGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTc0NC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb24gZWZmaWNpZW5jeSBpcyBkZXRlcm1pbmVkIGJ5IGJvdGggY29kb24gYmlhcyBhbmQgZm9sZGluZyBlbmVyZ3ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0zNjQ1LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGJpb2xvZ3k6IFdvYmJsZSBwdXRzIFJOQSBvbiB0YXJnZXQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTEwOyAgY2l0YXRpb25fcGFnZXM9NDgwLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIGFwcHJvYWNoIHJldmVhbHMgcmVndWxhdG9yeSBjb250cm9scyBvbiBiYWN0ZXJpYWwgdHJhbnNsYXRpb24gZWxvbmdhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU5OyAgY2l0YXRpb25fcGFnZXM9MTIwMC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IHN0cnVjdHVyYWwgaW5zaWdodHMgaW50byB0cmFuc2xhdGlvbmFsIG1pc2NvZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb2NoZW1pLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz03OTgtODE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBpbml0aWF0aW9uIGlzIGNvbnRyb2xsZWQgYnkgUk5BIGZvbGRpbmcga2luZXRpY3MgdmlhIGEgcmlib3NvbWUgZHJhZnRpbmcgbWVjaGFuaXNtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbS4gQ2hlbS4gU29jOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTEzODsgIGNpdGF0aW9uX3BhZ2VzPTcwMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBkeW5hbWljcyBvZiBzaW5nbGUgbVJOQXMgaW4gbGl2ZSBjZWxscyBhbmQgbmV1cm9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTI7ICBjaXRhdGlvbl9wYWdlcz0xNDMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhcnZhdGlvbiBmb3IgZGlmZmVyZW50IG51dHJpZW50cyBpbiBFc2NoZXJpY2hpYSBjb2xpIHJlc3VsdHMgaW4gZGlmZmVyZW50aWFsIG1vZHVsYXRpb24gb2YgUnBvUyBsZXZlbHMgYW5kIHN0YWJpbGl0eSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz00MzQtNDQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXksIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9Q3VycmVudCBwcm90b2NvbHMgaW4gbW9sZWN1bGFyIGJpb2xvZ3kgTmV3IFlvcms7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbkJhbmssOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUQzNi1ENDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05NTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFkYiAyMDA5OiBjb21waWxhdGlvbiBvZiB0Uk5BIHNlcXVlbmNlcyBhbmQgdFJOQSBnZW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9RDE1OS1EMTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXhEYiwgYSBkYXRhYmFzZSBvZiBwcm90ZWluIGFidW5kYW5jZSBhdmVyYWdlcyBhY3Jvc3MgYWxsIHRocmVlIGRvbWFpbnMgb2YgbGlmZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbC4gUHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ5Mi01MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uIGluIGh1bWFuOiBtb3JlIHByb25vdW5jZWQgaW4gaG91c2VrZWVwaW5nIGdlbmVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC4gRGlyZWN0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9sZCBnZW5lcyBleHBlcmllbmNlIHN0cm9uZ2VyIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uIHRoYW4geW91bmcgZ2VuZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTU5MDsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGNvbXBsZXhpdHkgcGxhY2VzIGxlc3MgcmVzdHJpY3Rpb25zIG9uIHRoZSBldm9sdXRpb24gb2YgeW91bmcgY29leHByZXNzaW9uIG5ldHdvcmtzIHRoYW4gcHJvdGVpbuKAk3Byb3RlaW4gaW50ZXJhY3Rpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTI2MjQtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhHVC1EQjogYSBkYXRhYmFzZSBvZiBwdXRhdGl2ZSBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgaW4gcHJva2FyeW90aWMgY29tcGxldGUgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xODctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYmlhcyBmcm9tIHRSTkEncyBwb2ludCBvZiB2aWV3OiByZWR1bmRhbmN5LCBzcGVjaWFsaXphdGlvbiwgYW5kIGVmZmljaWVudCBkZWNvZGluZyBmb3IgdHJhbnNsYXRpb24gb3B0aW1pemF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIyNzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxldmVscyBvZiB0Uk5BcyBpbiBiYWN0ZXJpYWwgY2VsbHMgYXMgYWZmZWN0ZWQgYnkgYW1pbm8gYWNpZCB1c2FnZSBpbiBwcm90ZWlucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgYWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NjExOS02MTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmUgb2YgdGhlIHByb3RlaW4gdW5pdmVyc2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xMTA3OS0xMTA4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3Jvd3RoLCBtZXRhYm9saWMgcGFydGl0aW9uaW5nLCBhbmQgdGhlIHNpemUgb2YgbWljcm9vcmdhbmlzbXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz00OTUtNTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgdHJhZGVvZmZzIGluIGNlbGx1bGFyIGNvbXBvc2l0aW9uIGFjcm9zcyBkaXZlcnNlIGJhY3RlcmlhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SVNNRSBKOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTEwMTA7ICBjaXRhdGlvbl9wYWdlcz0yMTQ1MjE0NS0yMTU3MjE1NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmVjdCBvZiB0Uk5BIGxldmVscyBvbiBkZWNvZGluZyB0aW1lcyBvZiBtUk5BIGNvZG9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTE3MS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG94aW4tcmVzaXN0YW50IHNvZGl1bSBjaGFubmVsczogcGFyYWxsZWwgYWRhcHRpdmUgZXZvbHV0aW9uIGFjcm9zcyBhIGNvbXBsZXRlIGdlbmUgZmFtaWx5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTAxNi0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IGJpbmRpbmcgb2YgRnRzWiB0byBaaXBBLCBhbiBlc3NlbnRpYWwgY29tcG9uZW50IG9mIHRoZSBzZXB0YWwgcmluZyBzdHJ1Y3R1cmUgdGhhdCBtZWRpYXRlcyBjZWxsIGRpdmlzaW9uIGluIEUuIGNvbGk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0d28tY29tcG9uZW50IHN5c3RlbSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJlZ3VsYXRpb24gb2YgZGl2ZXJzZSBzaWduYWxpbmcgcGF0aHdheXMgaW4gcHJva2FyeW90ZXMgYW5kIGV1a2FyeW90ZXMuIFBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9NzIzLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWlzbWF0Y2ggcmVwYWlyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9NjgxLTcxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGNvbnRyb2wgb2Ygc3VnYXIgbWV0YWJvbGlzbTogYSBncmFtLXBvc2l0aXZlIHNvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbnRvbmllIFZhbiBMZWV1d2VuaG9lazsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTU5LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBpbiB2aXZvIG9mIHRyYW5zbGF0aW9uIHdpdGggbnVjbGVvdGlkZSByZXNvbHV0aW9uIHVzaW5nIHJpYm9zb21lIHByb2ZpbGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9MjE4LTIyMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZW1lbnQgb2YgaHVtYW4gaW50ZXJmZXJvbiBIVUlGTmFscGhhMiBhbmQgSENWIGNvcmUgcHJvdGVpbiBleHByZXNzaW9uIGxldmVscyBpbiBFc2NoZXJpY2hpYSBjb2xpIGJ1dCBub3Qgb2YgSFVJRk5hbHBoYTggYnkgdXNpbmcgdGhlIHRSTkEoQUdBL0FHRyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5NjsgIGNpdGF0aW9uX3BhZ2VzPTEzMDMtMTMwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGdyb3d0aCBsYXdzIHJlZmxlY3QgdGhlIGV2b2x1dGlvbmFyeSBpbXBvcnRhbmNlIG9mIGVuZXJneSBlZmZpY2llbmN5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9NDA2LTQxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWJvbGljIG1vZGVsbGluZyBpbiBhIGR5bmFtaWMgZXZvbHV0aW9uYXJ5IGZyYW1ld29yayBwcmVkaWN0cyBhZGFwdGl2ZSBkaXZlcnNpZmljYXRpb24gb2YgYmFjdGVyaWEgaW4gYSBsb25nLXRlcm0gZXZvbHV0aW9uIGV4cGVyaW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYmlhcyBhcyBhIG1lYW5zIHRvIGZpbmUtdHVuZSBnZW5lIGV4cHJlc3Npb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0xNDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkEgZ2VuZSBudW1iZXIgYW5kIGNvZG9uIHVzYWdlIGluIHRoZSBDLiBlbGVnYW5zIGdlbm9tZSBhcmUgY28tYWRhcHRlZCBmb3Igb3B0aW1hbCB0cmFuc2xhdGlvbiBvZiBoaWdobHkgZXhwcmVzc2VkIGdlbmVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mjg3LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnZpcm9ubWVudGFsIHBlcnR1cmJhdGlvbnMgbGlmdCB0aGUgZGVnZW5lcmFjeSBvZiB0aGUgZ2VuZXRpYyBjb2RlIHRvIHJlZ3VsYXRlIHByb3RlaW4gbGV2ZWxzIGluIGJhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0yNDE5LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpcyBhbiBpbXBvcnRhbnQgZGV0ZXJtaW5hbnQgb2YgZ2VuZSBleHByZXNzaW9uIGxldmVscyBsYXJnZWx5IHRocm91Z2ggaXRzIGVmZmVjdHMgb24gdHJhbnNjcmlwdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPUU2MTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBleHByZXNzaW9uIGxldmVscyBhcmUgY29ycmVsYXRlZCB3aXRoIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UsIGFtaW5vIGFjaWQgY29tcG9zaXRpb24sIGFuZCBnZW5lIGFyY2hpdGVjdHVyZSBpbiB0aGUgcmVkIGZsb3VyIGJlZXRsZSwgVHJpYm9saXVtIGNhc3RhbmV1bSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTM3NTUtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtaW5vIEFjaWQgbWV0YWJvbGlzbSBjb25mbGljdHMgd2l0aCBwcm90ZWluIGRpdmVyc2l0eSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI5MDUtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZXZvbHV0aW9uIG9mIGNvZG9uIHVzYWdlIGFuZCB0Uk5BIGdlbmVzIGxlYWRzIHRvIGFsdGVybmF0aXZlIHN0YWJsZSBzdGF0ZXMgb2YgYmlhc2VkIGNvZG9uIHVzYWdlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjI3OS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vZGVybiBzeW50aGVzaXMgaW4gdGhlIGxpZ2h0IG9mIG1pY3JvYmlhbCBnZW5vbWljcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9Mjc5LTI5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG9yaWdpbnMgb2YgZ2Vub21lIGNvbXBsZXhpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMwMjsgIGNpdGF0aW9uX3BhZ2VzPTE0MDEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2lucyBhbmQgcHJpbmNpcGxlcyBvZiB0cmFuc2xhdGlvbmFsIGNvbnRyb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbGQgU3ByaW5nIEhhcmJvciBNb25vZ3IuIEFyY2g7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0xLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1tUk5BIGNvbXBvc2l0aW9uIGFuZCBjb250cm9sIG9mIGJhY3RlcmlhbCBnZW5lIGV4cHJlc3Npb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTgyOyAgY2l0YXRpb25fcGFnZXM9MzAzNy0zMDQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIHR3byBjb2RvbiBvcHRpbWl6YXRpb24gc3RyYXRlZ2llcyB0byBlbmhhbmNlIHJlY29tYmluYW50IHByb3RlaW4gcHJvZHVjdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iLiBDZWxsIEZhY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9pc3N1ZT0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXNpZ24gYW5kIHN5bnRoZXNpcyBvZiBhIG1pbmltYWwgYmFjdGVyaWFsIGdlbm9tZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzUxOyAgY2l0YXRpb25fcGFnZXM9YWFkNjI1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtaW5pbWFsIGdlbm9tZSBjb21lcyBvZiBhZ2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz02MjMtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC82LzYyMy8yMjI5NTgwMy9kc3gwMzAucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFsdGhvdWdoIG1vcmUgYW5kIG1vcmUgZW50YW5nbGVkIHBhcnRpY2lwYW50cyBvZiB0cmFuc2xhdGlvbiBwcm9jZXNzIHdlcmUgcmVhbGl6ZWQsIGhvdyB0aGV5IGNvb3BlcmF0ZSBhbmQgY28tZGV0ZXJtaW5lIHRoZSBmaW5hbCB0cmFuc2xhdGlvbiBlZmZpY2llIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zOTUzOTMzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJtlP9G7PNnx6yJMZNZXUkPIn6th" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f623%2f3953933" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f623%2f3953933" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104086552">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086552">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086554">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086554">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086559">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086559">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086571">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086571">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086591">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086591">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086604">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086604">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086612">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086612">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/6/609/3904549">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/6/635/3979487">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Meng-Ze Du</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Meng-Ze+Du">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Du M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Du M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Meng-Ze Du</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wen Wei</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wen+Wei">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wei W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wei W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wen Wei</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lei Qin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lei+Qin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qin L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qin L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lei Qin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuo Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuo+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shuo Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">An-Ying Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div><div class="aff">Centre for Informational Biology</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=An-Ying+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">An-Ying Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div><div class="aff">Centre for Informational Biology</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yong Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hong Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div><div class="aff">Centre for Informational Biology</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hong+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hong Zhou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Feng-Biao Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Life Science and Technology</div><div class="aff">Centre for Informational Biology</div><div class="aff">Key Laboratory for Neuroinformation of the Ministry of Education, University of Electronic Science and Technology of China, Chengdu, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx030-cor1" data-open="dsx030-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel: +86 28 83202351. Fax. +86 28 83202351. Email: <a href="mailto:fbguo@uestc.edu.cn" target="_blank">fbguo@uestc.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Feng-Biao+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Feng-Biao Guo</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx030-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 623–633, <a href='https://doi.org/10.1093/dnares/dsx030'>https://doi.org/10.1093/dnares/dsx030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 July 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 February 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">15 June 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3953933" href="/dnaresearch/article-pdf/24/6/623/22295803/dsx030.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Meng-Ze Du, Wen Wei, Lei Qin, Shuo Liu, An-Ying Zhang, Yong Zhang, Hong Zhou, Feng-Biao Guo;  Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 623–633, <a href='https://doi.org/10.1093/dnares/dsx030'>https://doi.org/10.1093/dnares/dsx030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3953933?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3953933?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3953933?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3953933?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3953933?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3953933?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3953933?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Co-adaption%20of%20tRNA%20gene%20copy%20number%20and%20amino%20acid%20usage%20influences%20translation%20rates%20in%20three%20life%20domains&amp;publicationDate=2017-07-11&amp;volumeNum=24&amp;issueNum=6&amp;author=Du%2C%20Meng-Ze%3B%20Wei%2C%20Wen&amp;startPage=623&amp;endPage=633&amp;contentId=10.1093%2Fdnares%2Fdsx030&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f623%2f3953933" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104086552 id="104086552" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Although more and more entangled participants of translation process were realized, how they cooperate and co-determine the final translation efficiency still lacks details. Here, we reasoned that the basic translation components, tRNAs and amino acids should be consistent to maximize the efficiency and minimize the cost. We firstly revealed that 310 out of 410 investigated genomes of three domains had significant co-adaptions between the tRNA gene copy numbers and amino acid compositions, indicating that maximum efficiency constitutes ubiquitous selection pressure on protein translation. Furthermore, fast-growing and larger bacteria are found to have significantly better co-adaption and confirmed the effect of this pressure. Within organism, highly expressed proteins and those connected to acute responses have higher co-adaption intensity. Thus, the better co-adaption probably speeds up the growing of cells through accelerating the translation of special proteins. Experimentally, manipulating the tRNA gene copy number to optimize co-adaption between enhanced green fluorescent protein (EGFP) and tRNA gene set of <em>Escherichia coli</em> indeed lifted the translation rate (speed). Finally, as a newly confirmed translation rate regulating mechanism, the co-adaption reflecting translation rate not only deepens our understanding on translation process but also provides an easy and practicable method to improve protein translation rates and productivity.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;translation rate&quot;">translation rate</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="co-adaption">co-adaption</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;amino acid usage&quot;">amino acid usage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tRNA gene copy number&quot;">tRNA gene copy number</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;minimum cost&quot;">minimum cost</a></div>                            <h2 scrollto-destination=104086554 id="104086554" class="section-title" >1. Introduction</h2>
<p>Translation initiation, elongation and termination involve many factors, that balance translation rate (speed) and accuracy.<a reveal-id="dsx030-B1" data-open="dsx030-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx030-B2" data-open="dsx030-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The final translation efficiency for a given protein is restricted by the cost of its production and organization.<a reveal-id="dsx030-B3" data-open="dsx030-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Therefore, evolving a genome-wide translation regulation regime can efficiently determine the translation rates of various genes under different conditions.<a reveal-id="dsx030-B1" data-open="dsx030-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Conventional computations of translation elongation efficiency refer to codon usage<a reveal-id="dsx030-B4" data-open="dsx030-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and tRNA availability.<a reveal-id="dsx030-B5" data-open="dsx030-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The relationship between codon usage and tRNA abundance predicts translation efficiency with reasonable accuracy.<a reveal-id="dsx030-B1" data-open="dsx030-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a></p><p>Additional theories have been proposed with constantly emerging experimental technologies<a reveal-id="dsx030-B6" data-open="dsx030-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx030-B7" data-open="dsx030-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> to cope with challenges to the simplified assumptions about translation described above. Thus, the effects of codon order,<a reveal-id="dsx030-B8 dsx030-B9 dsx030-B10" data-open="dsx030-B8 dsx030-B9 dsx030-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> local tRNA availability,<a reveal-id="dsx030-B11 dsx030-B12 dsx030-B13" data-open="dsx030-B11 dsx030-B12 dsx030-B13" class="link link-ref link-reveal xref-bibr"><sup>11–13</sup></a> regulation of expression of the tRNA gene,<a reveal-id="dsx030-B14" data-open="dsx030-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> the diverse demands of the transcriptomes,<a reveal-id="dsx030-B9" data-open="dsx030-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx030-B15" data-open="dsx030-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> ribosomes,<a reveal-id="dsx030-B16" data-open="dsx030-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> mRNA structures<a reveal-id="dsx030-B17 dsx030-B18 dsx030-B19" data-open="dsx030-B17 dsx030-B18 dsx030-B19" class="link link-ref link-reveal xref-bibr"><sup>17–19</sup></a> and folding energy<a reveal-id="dsx030-B20" data-open="dsx030-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> were included in the translation efficiency models. Among these factors, tRNA availability repeatedly emphasized decides the supply of aminoacyl-tRNA,<a reveal-id="dsx030-B21" data-open="dsx030-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> which influences the translocation of ribosomes on mRNA.<a reveal-id="dsx030-B22 dsx030-B23 dsx030-B24 dsx030-B25" data-open="dsx030-B22 dsx030-B23 dsx030-B24 dsx030-B25" class="link link-ref link-reveal xref-bibr"><sup>22–25</sup></a> Nutriment limitations, such as amino acid shortage, also have influences on the cellular supply of aminoacyl-tRNA.<a reveal-id="dsx030-B26" data-open="dsx030-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> However, how the formation of aminoacyl-tRNA influences translation efficiency is still unclear.</p><p>In the translation process, tRNAs can be thought of as tools and the amino acids as the raw materials. Each species of tRNA corresponds to a particular amino acid, and each of the former is responsible for carrying one of the latter. We hypothesize that the levels of the tRNAs and the corresponding amino acids should be well matched to synthesize proteins more efficiently. Such a consistency would maximize efficiency and minimize resource/energy costs.</p><p>Here, we firstly examined the association between the tRNA gene copy numbers and amino acid compositions in various organisms. We sought to validate two points of reasoning: at the organismal scale, most organisms evolve co-adaption between tRNA gene copy number and amino acid composition; and at the second scale of individual proteins, the co-adaption intensity may vary among the proteins within an organism. Some proteins need to be expressed rapidly to maintain a high quantity or to satisfy the requirements of acute responses. We speculated that such proteins would have higher co-adaption to increase their translation efficiency. Computational analyses were employed to elucidate co-adaption between tRNA gene copy numbers and amino acid usage for proteins/proteomes in three domains of life, indicating the effects of maximum efficiency and the minimum cost principle. Then, we further correlated the co-adaption with proteins’ translation rates, which were validated by growth rates of bacteria and production rates of target proteins. The target proteins’ translation rates were observed to be lifted through changing the proportions of gene copy numbers of tRNAs, providing clues for applied biology.</p>                            <h2 scrollto-destination=104086559 id="104086559" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=104086560 id="104086560" class="section-title" >2.1. <em>E. coli</em> strain and methods</h3>
<p>DNA amplification and expression were performed in <em>E. coli</em> Top10 cells (F- <em>mcrA</em> Δ(<em>mrr-hsd</em>RMS-<em>mcr</em>BC) Φ80<em>lac</em>ZΔM15 Δ <em>lac</em>X74 <em>rec</em>A1 <em>ara</em>D139 Δ(<em>araleu</em>)7697 <em>gal</em>U <em>gal</em>K <em>rps</em>L (StrR) <em>end</em>A1 <em>nup</em>G). All bacterial media and methods used in this study were as described in Current Protocols in Molecular Biology.<a reveal-id="dsx030-B27" data-open="dsx030-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a></p>                            <h3 scrollto-destination=104086562 id="104086562" class="section-title" >2.2. Production of synthetic genes</h3>
<p>Oligonucleotides were synthesized using PCR amplification. The fragments were recombined to generate the target coding sequences, which were inserted behind the arabinose promoter. Positive clones were screened by resistance screening and confirmed by PCR and sequencing.</p>                            <h3 scrollto-destination=104086564 id="104086564" class="section-title" >2.3. Detection of target polypeptides</h3>
<p>Cells were grown overnight at 30°C in Luria–Bertani (LB) culture medium, and were inoculated in LB culture medium with ampicillin at OD600 = 2. After hours of constant shaking (OD600 = 0.6), L-arabinose (0.05%) was added to the culture medium to induce heterologous expression. Samples were collected at different time points and put on ice. When all samples were prepared, aliquots of the cells were observed through confocal microscopy (Leica TCS SP8, Germany), and the rest were collected by centrifugation (4000×<em>g</em>, 20 min). The cells were then washed three times with cold PB (4000×<em>g</em>, 10 min), and cell lysis buffer [phenylmethylsulfonyl fluoride (PMSF) 0.1 mM, PB 10 mM, lysozyme 1 mg/ml] was added to lyse the cells for 15 min before sonication (3 min). After ultrasonic breakage, the samples were centrifuged, and the supernatants were collected. EGFP in the supernatants was measured using fluorospectro-photometer (Hitachi F7000, EX WL: 460.0 nm) and was quantified using a Bicinchoninic Acid (BCA) Protein Assay Kit (CWBIO) before adding loading buffer and boiling for five minutes. Thirty micrograms of the protein extract samples were loaded on 12% stain-free SDS-polyacrylamide gel (Bio-Rad), subjected to electrophoresis, and transferred to 0.2 µm polyvinylidene difluoride membranes (Millipore). After blocking with 5% non-fat milk in Tris-buffered saline buffer with Tween 20 (TBS/T) for 4 h at room temperature, the membrane was incubated with the appropriate primary antibodies (Abmart GFP-tag mAb, 1:1000) for 18 h at 4 °C. Next, horseradish peroxidase (HRP)-conjugated goat anti-mouse secondary antibody (1:5000, ZSGB-BIO) was applied for 2 h at room temperature. Finally, the signals were visualized using an Enhanced Chemiluminescence (ECL) kit (Roche).</p>                            <h3 scrollto-destination=104086566 id="104086566" class="section-title" >2.4. TAAI</h3>
<div class="block-child-p">TAAI is short for the ‘tRNA gene copy number and amino acid usage accordance index’, and it measures the co-adaption of amino acid usage and tRNA gene copy number. For a given protein sequence, the frequencies of the 20 types of amino acids are unique. For a specific organism, the average frequencies of amino acid usage also differ among organisms. <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>X</mi><mi>i</mi></msub><mo>=</mo><mfrac><mrow><mi>N</mi><mi>t</mi><mi>R</mi><mi>N</mi><msub><mi>A</mi><mi>i</mi></msub></mrow><mrow><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>20</mn></mrow></munderover><mrow><mi>N</mi><mi>t</mi><mi>R</mi><mi>N</mi><msub><mi>A</mi><mi>k</mi></msub></mrow></mrow></mfrac><mo>,</mo><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mn>3…20</mn></mrow></math></div></div><div class="label">(1)</div></div><a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>Y</mi><mi>i</mi></msub><mo>=</mo><mfrac><mrow><mi>N</mi><mi>a</mi><msub><mi>a</mi><mi>i</mi></msub></mrow><mrow><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>20</mn></mrow></munderover><mrow><mi>N</mi><mi>a</mi><msub><mi>a</mi><mi>k</mi></msub></mrow></mrow></mfrac><mo>,</mo><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mn>3…20</mn></mrow></math></div></div><div class="label">(2)</div></div><a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>T</mi><mi>A</mi><mi>A</mi><mi>I</mi><mo>=</mo><mfrac><mrow><msub><mo>∑</mo><mi>i</mi></msub><mrow><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo>−</mo><mrow><mover accent="true"><mi>x</mi><mo>¯</mo></mover></mrow><mo stretchy="false">)</mo><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow><mo stretchy="false">)</mo></mrow></mrow><mrow><msqrt><mrow><msub><mo>∑</mo><mi>i</mi></msub><mrow><msup><mrow><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo>−</mo><mrow><mover accent="true"><mi>x</mi><mo>¯</mo></mover></mrow><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><msub><mo>∑</mo><mi>i</mi></msub><mrow><msup><mrow><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><mrow><mover accent="true"><mi>y</mi><mo>¯</mo></mover></mrow><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></mrow></mrow></msqrt></mrow></mfrac><mo>,</mo><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mn>3…20</mn></mrow></math></div></div><div class="label">(3)</div></div> where X<sub>i</sub> is the frequency of the gene copy of tRNA i in one organism’s genome in <a href="#E1" class="sectionLink xref-sec">Equation (1)</a>, and let Y<sub>i</sub> be the frequency of amino acid i of in a specific protein in the <a href="#E2" class="sectionLink xref-sec">Equation (2)</a>. The value NtRNA<sub>i</sub> is the corresponding gene copy number of tRNAi decoding all codons for the ith amino acid. The value <em>N</em>aa<sub>i</sub> is the corresponding counts of the ith amino acid in special protein or genome. TAAI in <a href="#E3" class="sectionLink xref-sec">Equation (3)</a> is the Spearman correlation index of X and Y. In addition to the TAAI of one protein, we also calculated the general TAAI of an organism to denote its overall co-adaption. The overall value equals the average of all proteins’ TAAIs for that organism.</div>                            <h3 scrollto-destination=104086568 id="104086568" class="section-title" >2.5. Bioinformatics data source</h3>
<p>This work requires tRNA gene annotation information that is as accurate as possible; therefore, we compared annotation information from three databases. We chose three widely used databases: GenBank,<a reveal-id="dsx030-B28" data-open="dsx030-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> a comprehensive bioinformatics database; the Genomic tRNA Database,<a reveal-id="dsx030-B12" data-open="dsx030-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> which uses tRNAscan-SE;<a reveal-id="dsx030-B29" data-open="dsx030-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and tRNAdb,<a reveal-id="dsx030-B30" data-open="dsx030-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> which contains more than 12,000 tRNA genes. A total of 410 genomes have the same tRNA gene annotation information in all three databases. If the tRNA gene annotations for one organism were consistent in the three databases, we deemed it as reliable and then employed this organism in further analysis. Nucleotide and protein sequences were downloaded from GenBank,<a reveal-id="dsx030-B28" data-open="dsx030-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and protein abundance values were acquired from PaxDb.<a reveal-id="dsx030-B31" data-open="dsx030-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> We further compared the co-adaption of specific gene categories. Further analysis employed housekeeping genes (expression-invariable ones) and tissue-specific genes (expressed only in one tissue) from Ma <em>et al.</em><a reveal-id="dsx030-B32" data-open="dsx030-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> The young and old genes of human were acquired from Yin <em>et al.</em><a reveal-id="dsx030-B33" data-open="dsx030-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and Wei <em>et al.</em><a reveal-id="dsx030-B34" data-open="dsx030-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The horizontal transfer gene lists for bacteria were downloaded from HGT-DB.<a reveal-id="dsx030-B35" data-open="dsx030-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p><p>We employed 53 organisms’ growth times (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s2_dsx030.xlsx?Expires=2147483647&Signature=LRyyJlT2jg2RlKz0os7yJhlocPB4zcjLsaN5M40wsTQwaO2eE8e1Dl5v68Ra6485z5Y7x90fhVFUMhA5MXHpbIHOGstG-9S9iGRAVvil2yJKJmKlewyxguplHy8oeo8y4B4C3F61Re~XMJIIT25NX9lKkPbW0R6KuZEWqvE36FjHt~0DZUCbbD1DCI3YNqOKbhLlf6AIUKAFN6p2~~zEf6tuZWGA7EunHx7Ozi3s~9Z8Gh8AY1ELReQ~YONovCJOWkPZnXoNEKv6-uPQlehr3cEtKa~IcT-TtVCxRG4sG5SmMA9NpFoj6NI4n1S0Vi4YKseim16EyQ7vN-EeSCkgdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Here, we chose those archaea and bacteria with growth times from Rocha <em>et al.</em><a reveal-id="dsx030-B36" data-open="dsx030-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Among 410 organisms filtered for reliable tRNA gene annotation, 376 archaea and bacteria were chosen to perform the analysis of correlation between TAAI and genome sizes. When comparing the TAAIs among proteins within a specific genome, we analyzed the protein abundances and TAAIs in six model organisms, which have integrated abundances in PaxDb.<a reveal-id="dsx030-B31" data-open="dsx030-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a></p>                            <h2 scrollto-destination=104086571 id="104086571" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=104086572 id="104086572" class="section-title" >3.1. At the genome scale the tRNA and corresponding amino acid co-adapt to maximize the efficiency</h3>
<p>During translation, tRNAs transport amino acids to ribosomes, and co-operation between the two components has been reported in a few organisms.<a reveal-id="dsx030-B37" data-open="dsx030-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Previous researches demonstrated that the tRNA gene copy numbers were different among organisms/strains/species, and that the protein sequences varied greatly.<a reveal-id="dsx030-B38" data-open="dsx030-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> To check whether the amino acid usage of proteins generally co-adapts with the corresponding organism’s tRNA gene copy number, we calculated and compared independent frequencies of the two in 410 genomes from three domains of life (17 archaea; 359 bacteria; 34 eukaryotes), using more accurate tRNA gene annotations in GenBank,<a reveal-id="dsx030-B28" data-open="dsx030-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> the Genomic tRNA database<a reveal-id="dsx030-B12" data-open="dsx030-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and tRNAdb.<a reveal-id="dsx030-B30" data-open="dsx030-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> First, the frequencies of 20 standard tRNA genes (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s1_dsx030.xlsx?Expires=2147483647&Signature=yB5TbdLaKY81NBGR~GiyngN34Db9jJBhv-SmYIkf06aXvoB6uDpXS5qWAR6ompO8SZlTQQ852MvQNLR7cmm4TQkdKTvXoO14NA6CSBmfQZqwpcORwuc~fffCFp-Vh8bF3k2o0vnbV4uc0xz0mv4ZSkfLjZ7-6TLMIOcDxm8ZNnkD6aQmPQujy4RVt~sQB2G2riTKH6NNiX~pEAMkninpUPDIQ~7ZnsXjW~88MuVqevi8yC0p5cPMpPy9jesvNA~TWefPtVuG3Ah0Ef8B7aHd5P-TfJRrPD2Okp4WHKjiBu2rRdFAcCX6cHKMAWDNT2IEE3QqhK8qbeJ5nK2qpdCfMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were computed by counting cognate tRNA gene copies and being divided by the organism’s total tRNA gene counts. Second, frequencies of the 20 amino acids in the proteome (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s1_dsx030.xlsx?Expires=2147483647&Signature=yB5TbdLaKY81NBGR~GiyngN34Db9jJBhv-SmYIkf06aXvoB6uDpXS5qWAR6ompO8SZlTQQ852MvQNLR7cmm4TQkdKTvXoO14NA6CSBmfQZqwpcORwuc~fffCFp-Vh8bF3k2o0vnbV4uc0xz0mv4ZSkfLjZ7-6TLMIOcDxm8ZNnkD6aQmPQujy4RVt~sQB2G2riTKH6NNiX~pEAMkninpUPDIQ~7ZnsXjW~88MuVqevi8yC0p5cPMpPy9jesvNA~TWefPtVuG3Ah0Ef8B7aHd5P-TfJRrPD2Okp4WHKjiBu2rRdFAcCX6cHKMAWDNT2IEE3QqhK8qbeJ5nK2qpdCfMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were computed by dividing the count of each amino acid by the sum of the twenty amino acid counts. After obtaining these two types of data for each organism, we performed linear fit and correlation analyses.</p><p>The linear fit results showed variable co-adaption relationships (<a reveal-id="dsx030-F1" data-open="dsx030-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s1_dsx030.xlsx?Expires=2147483647&Signature=yB5TbdLaKY81NBGR~GiyngN34Db9jJBhv-SmYIkf06aXvoB6uDpXS5qWAR6ompO8SZlTQQ852MvQNLR7cmm4TQkdKTvXoO14NA6CSBmfQZqwpcORwuc~fffCFp-Vh8bF3k2o0vnbV4uc0xz0mv4ZSkfLjZ7-6TLMIOcDxm8ZNnkD6aQmPQujy4RVt~sQB2G2riTKH6NNiX~pEAMkninpUPDIQ~7ZnsXjW~88MuVqevi8yC0p5cPMpPy9jesvNA~TWefPtVuG3Ah0Ef8B7aHd5P-TfJRrPD2Okp4WHKjiBu2rRdFAcCX6cHKMAWDNT2IEE3QqhK8qbeJ5nK2qpdCfMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), illustrating that the two factors (tRNA gene copy number and the amino acid frequency) are not independent from each other. Indeed, although the slopes of the fitted lines differed, in all cases, the tRNA gene copy numbers showed positive correlations with corresponding amino acid usages. Spearman rank correlation coefficients (<em>r</em>) were calculated after least square fitting (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s1_dsx030.xlsx?Expires=2147483647&Signature=yB5TbdLaKY81NBGR~GiyngN34Db9jJBhv-SmYIkf06aXvoB6uDpXS5qWAR6ompO8SZlTQQ852MvQNLR7cmm4TQkdKTvXoO14NA6CSBmfQZqwpcORwuc~fffCFp-Vh8bF3k2o0vnbV4uc0xz0mv4ZSkfLjZ7-6TLMIOcDxm8ZNnkD6aQmPQujy4RVt~sQB2G2riTKH6NNiX~pEAMkninpUPDIQ~7ZnsXjW~88MuVqevi8yC0p5cPMpPy9jesvNA~TWefPtVuG3Ah0Ef8B7aHd5P-TfJRrPD2Okp4WHKjiBu2rRdFAcCX6cHKMAWDNT2IEE3QqhK8qbeJ5nK2qpdCfMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Specifically, 99.27% had correlation coefficients greater than 0.1 (<a reveal-id="dsx030-F1" data-open="dsx030-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>), and 75.61% showed significant correlations (<em>P</em> &lt; 0.05). Finally, a general linear relationship exists between tRNA gene copy and amino acid usage. For four representative organisms, the archaebacterium <em>Methanosphaera stadtmanae</em> (<em>r</em> = 0.17, <em>P</em> = 0.46), the bacterium <em>Escherichia coli</em> (<em>r</em> = 0.54, <em>P</em> = 0.01), and the eukaryotes <em>Saccharomyces cerevisiae</em> (<em>r</em> = 0.74, <em>P</em> = 1.67e−04) and <em>Homo sapiens</em> (<em>r</em> = 0.56, <em>P</em> = 0.01), the linear models presented different co-adaption intensities (<a reveal-id="dsx030-F1" data-open="dsx030-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s1_dsx030.xlsx?Expires=2147483647&Signature=yB5TbdLaKY81NBGR~GiyngN34Db9jJBhv-SmYIkf06aXvoB6uDpXS5qWAR6ompO8SZlTQQ852MvQNLR7cmm4TQkdKTvXoO14NA6CSBmfQZqwpcORwuc~fffCFp-Vh8bF3k2o0vnbV4uc0xz0mv4ZSkfLjZ7-6TLMIOcDxm8ZNnkD6aQmPQujy4RVt~sQB2G2riTKH6NNiX~pEAMkninpUPDIQ~7ZnsXjW~88MuVqevi8yC0p5cPMpPy9jesvNA~TWefPtVuG3Ah0Ef8B7aHd5P-TfJRrPD2Okp4WHKjiBu2rRdFAcCX6cHKMAWDNT2IEE3QqhK8qbeJ5nK2qpdCfMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Compared with the other three organisms, yeast had the best linear fit. However, for <em>M. stadtmanae</em>, having the most unbalanced constitution of tRNA genes, the observed points were not well fitted. In general, most organisms’ tRNA gene copy numbers and amino acid usages showed a positive linear relationship. </p>                        <a id="104086575" scrollto-destination="104086575"></a>
<div id="dsx030-F1" data-id="dsx030-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f1.png?Expires=2147483647&Signature=FzruukroO3Iw0i76NKT-onp0HnFpyNc34y0uHrLd7jKljNvCwrWNAAI4SUJSjv1h8M-mBa5j0Xht7egruJv9oUd7oCXZZJLzpM9ddhnejJkAJQfGjyNkQeogb8J8SBQ2Mosoak68jVPxHgGb7VHGe-xn3QxvdyOGgWt-WIFspt~e5UPc3t9n9z79x7m8-lksLmG9GM9qeBg467OCkf9lfpAWfcUFbyiWfkyW8rzHdEZzj4UdyUtSNGJlnz674XJr3MiTNHsP1hRb3X3-KlY8~97Tczfa9Zq2rESmoFrLRqi3As~sBSiOQbdfNyV0h9U3LB1s2JDvBQ~xk28978rtBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Co-adaption between frequencies of tRNA gene copy numbers and amino acid usage. (A) Linear fitting results for 410 organisms. Dotted lines represent organisms with the P values of linear fitting are greater than 0.05; straight lines indicate that P values less than 0.05. (B) Corresponding Spearman rank correlation coefficients r for the linear fit of 410 genomes. (C) Linear fit of four model organisms." path-from-xml="dsx030f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086575/dsx030f1.tif" path-from-xml="dsx030f1.tif" target="_blank">View large</a><a section="104086575" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f1.png?Expires=2147483647&Signature=pTbH-8KTy4UNgmWYJ4pAmwSr7ubGRzRrDG5ahe6dsKYKyosUKzfr8DaWlj9l90IuSmyN6sALZnqkual1k19d-R2u-vSUZSVEHppfemFQ1ZlIEzT1bOmTj6ETL3CPL6~85m34OBRCzUBEbpde8RmOk5Q4Wq5zRTZfp2s~1SgfnNASbb81AT8~waffITOLh2Ms2s1FD5-BYMFLrYb2TZXaMgacdCYnkvzWFxBICnogk0n-Zq2t6TnKRJFpXz3jmABPjQ2dpW9R1AoJWb9oROTJT6GUrwPgRDJmR8jvbNU3k6c44leGhhlHNWnOGaAXap~EVz-hENUxc2fdlp7OluP2Cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086575&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Co-adaption between frequencies of tRNA gene copy numbers and amino acid usage. (A) Linear fitting results for 410 organisms. Dotted lines represent organisms with the <em>P</em> values of linear fitting are greater than 0.05; straight lines indicate that <em>P</em> values less than 0.05. (B) Corresponding Spearman rank correlation coefficients <em>r</em> for the linear fit of 410 genomes. (C) Linear fit of four model organisms.</p></div></div><div content-id="dsx030-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f1.png?Expires=2147483647&Signature=FzruukroO3Iw0i76NKT-onp0HnFpyNc34y0uHrLd7jKljNvCwrWNAAI4SUJSjv1h8M-mBa5j0Xht7egruJv9oUd7oCXZZJLzpM9ddhnejJkAJQfGjyNkQeogb8J8SBQ2Mosoak68jVPxHgGb7VHGe-xn3QxvdyOGgWt-WIFspt~e5UPc3t9n9z79x7m8-lksLmG9GM9qeBg467OCkf9lfpAWfcUFbyiWfkyW8rzHdEZzj4UdyUtSNGJlnz674XJr3MiTNHsP1hRb3X3-KlY8~97Tczfa9Zq2rESmoFrLRqi3As~sBSiOQbdfNyV0h9U3LB1s2JDvBQ~xk28978rtBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Co-adaption between frequencies of tRNA gene copy numbers and amino acid usage. (A) Linear fitting results for 410 organisms. Dotted lines represent organisms with the P values of linear fitting are greater than 0.05; straight lines indicate that P values less than 0.05. (B) Corresponding Spearman rank correlation coefficients r for the linear fit of 410 genomes. (C) Linear fit of four model organisms." path-from-xml="dsx030f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086575/dsx030f1.tif" path-from-xml="dsx030f1.tif" target="_blank">View large</a><a section="104086575" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f1.png?Expires=2147483647&Signature=pTbH-8KTy4UNgmWYJ4pAmwSr7ubGRzRrDG5ahe6dsKYKyosUKzfr8DaWlj9l90IuSmyN6sALZnqkual1k19d-R2u-vSUZSVEHppfemFQ1ZlIEzT1bOmTj6ETL3CPL6~85m34OBRCzUBEbpde8RmOk5Q4Wq5zRTZfp2s~1SgfnNASbb81AT8~waffITOLh2Ms2s1FD5-BYMFLrYb2TZXaMgacdCYnkvzWFxBICnogk0n-Zq2t6TnKRJFpXz3jmABPjQ2dpW9R1AoJWb9oROTJT6GUrwPgRDJmR8jvbNU3k6c44leGhhlHNWnOGaAXap~EVz-hENUxc2fdlp7OluP2Cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086575&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Co-adaption between frequencies of tRNA gene copy numbers and amino acid usage. (A) Linear fitting results for 410 organisms. Dotted lines represent organisms with the <em>P</em> values of linear fitting are greater than 0.05; straight lines indicate that <em>P</em> values less than 0.05. (B) Corresponding Spearman rank correlation coefficients <em>r</em> for the linear fit of 410 genomes. (C) Linear fit of four model organisms.</p></div></div><p>We measured the co-adaption intensity using a <strong><em>t</em></strong>RNA gene copy and <strong><em>a</em></strong>mino <strong><em>a</em></strong>cid usage accordance <strong><em>i</em></strong>ndex (TAAI), which equals to the <em>r</em> value of the Spearman rank correlation. During protein production, tRNA genes will be transcribed to tRNAs, and then loaded with amino acids for protein translation. Resource allocation would be the most efficient if the supply of tRNAs just meets the required amount of amino acids. Based on the results, our species/strain/organism scale reasoning was confirmed. In other words, most genomes had significant co-adaption between the tRNA gene copy number and the frequencies of amino acid usage and hence maximized their translation efficiency and minimized their energy/resource costs.</p><p>Different genomes (species/strain/organism) may face different translation selection pressure when translating different numbers of proteins in a given time. For example, large genomes have more proteins, and fast-growing bacteria need to synthesize more proteins simultaneously. In fact, large bacterial genomes are often associated with short generation times.<a reveal-id="dsx030-B36" data-open="dsx030-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> According to the maximum efficiency/minimum cost principle, fast-growing/large bacteria should have higher TAAIs than slow-growing and/or small bacteria. To test this possibility, we compared the TAAIs of 53 bacteria (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s2_dsx030.xlsx?Expires=2147483647&Signature=LRyyJlT2jg2RlKz0os7yJhlocPB4zcjLsaN5M40wsTQwaO2eE8e1Dl5v68Ra6485z5Y7x90fhVFUMhA5MXHpbIHOGstG-9S9iGRAVvil2yJKJmKlewyxguplHy8oeo8y4B4C3F61Re~XMJIIT25NX9lKkPbW0R6KuZEWqvE36FjHt~0DZUCbbD1DCI3YNqOKbhLlf6AIUKAFN6p2~~zEf6tuZWGA7EunHx7Ozi3s~9Z8Gh8AY1ELReQ~YONovCJOWkPZnXoNEKv6-uPQlehr3cEtKa~IcT-TtVCxRG4sG5SmMA9NpFoj6NI4n1S0Vi4YKseim16EyQ7vN-EeSCkgdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) and grouped them by growth time.<a reveal-id="dsx030-B36" data-open="dsx030-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> The fast, had growth times below the mean of the 53 ones; the slow, had growth times greater than the average. The two groups had similar variances and GC contents, while the slow group had significantly lower TAAIs than the fast group (<a reveal-id="dsx030-F2" data-open="dsx030-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Thus, co-adaption showed an effect on growth rate. This result is consistent with the idea that population growth rate is a fundamental ecological and evolutionary characteristic of living organisms.<a reveal-id="dsx030-B39" data-open="dsx030-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Similarly, larger bacteria have larger genomes and more proteins that need to be translated than bacteria with smaller genomes.<a reveal-id="dsx030-B39" data-open="dsx030-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx030-B40" data-open="dsx030-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Therefore, we also compared the TAAIs of prokaryotic organisms grouped by genome size (small, medium and large). These three groups had significantly divergent mean TAAIs of 0.37, 0.60 and 0.65 (the Student’s <em>t</em> test: <em>P</em> &lt; 2.2e−16; <a reveal-id="dsx030-F2" data-open="dsx030-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). That the relatively larger genomes have higher TAAIs supports the conclusion that bacteria under higher selective pressure have higher TAAIs, thus conforming to our first hypothesis based on the principle of efficiency described above. Bacteria with smaller genomes and slow growing speeds would suffer less pressure from protein translations and hence have lower TAAIs. </p>                        <a id="104086578" scrollto-destination="104086578"></a>
<div id="dsx030-F2" data-id="dsx030-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f2.png?Expires=2147483647&Signature=g23SNnYcJv3Y3Kumz8gyx7YEKKbRBwb1tf~HXIED~ofYN6ZfhGsfJpC~Pdvo7LsgPBnx6RMmjI0wzc~0SV9lFnfIzsSCySOfr~yvBFvKmoNQ3Fgic6NBY8xS6nB0Mjxu~qnbxe3mVztGE5M3QM4Sry3awxGYMlZNgpFKJdirfeS7Z7ZPMkztX2LIVDzyPkbBmMMXycS-slcqL2kDodtAreJ4XZ9MO~P~vBK4-Y0e9suKWHsUitxioiHEIWyl3nJTVyk3vNK3Xl5gDo2ZIb9egy5Bcii7Xyr~4d2dlEk317SZca-99xUS6U-IlXHV1sBTi6PLSPVtJhhTzHAXXtL6AA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Co-adaption at the genome scale. (A) TAAIs of bacteria divided into two groups based on their growth rates. The dataset includes 53 bacteria with available information on growth rates: which were classed into two groups by the mean growth time. The fast group has higher TAAIs than the slow. The GC content of the two groups are similar (The students’ t test: P = 0.42), their genome sizes varied non-significantly (The fast generally has larger genome than the slow: P = 0.07). (B) Prokaryotic organisms’ TAAIs, associated with corresponding genome sizes. The prokaryotic organisms were divided into three groups, showing significantly different TAAIs. Here, 376 prokaryotic genomes were involved in the analysis of correlation between TAAIs and genome sizes. In addition, the GC content also affect the TAAIs along with the increase of genome sizes: GC small &lt; GC median &lt; GC large (P &lt; 2.2e−16)." path-from-xml="dsx030f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086578/dsx030f2.tif" path-from-xml="dsx030f2.tif" target="_blank">View large</a><a section="104086578" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f2.png?Expires=2147483647&Signature=AS23BG8BXdaJcn8wqwjZup1ueXoA1fOuhAXGfnw5fpcyWwSjSVwqPGD1jRcFjj9IEGiji7BGx8dOop0DGhUjBkE98MZysr6HvFIuOV0Q3SujHFTGkNWY4zDCW2iJPJ-0sMTOBLFUdiK7bXgFp7vfl~ncakTpqYCa~pryLrcrh7mikDw3DSr8pLMvrTyknnBetPOXgz3lOQuIc3yVZ7K3ixuzBMRP5lZnCD4Wthqc-zuXDoysj9G9bHYMvzFsTboO0-q3QyJQTscTVUCLi5dSbl7qt28R5SAL5QCzSlH5CWnMTzrpnduHtfuSR2pGLGnZLU2l4gwaY7b41pLJGZ05mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086578&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Co-adaption at the genome scale. (A) TAAIs of bacteria divided into two groups based on their growth rates. The dataset includes 53 bacteria with available information on growth rates: which were classed into two groups by the mean growth time. The fast group has higher TAAIs than the slow. The GC content of the two groups are similar (The students’ <em>t</em> test: <em>P</em> = 0.42), their genome sizes varied non-significantly (The fast generally has larger genome than the slow: <em>P</em> = 0.07). (B) Prokaryotic organisms’ TAAIs, associated with corresponding genome sizes. The prokaryotic organisms were divided into three groups, showing significantly different TAAIs. Here, 376 prokaryotic genomes were involved in the analysis of correlation between TAAIs and genome sizes. In addition, the GC content also affect the TAAIs along with the increase of genome sizes: GC <sub>small</sub> &lt; GC <sub>median</sub> &lt; GC <sub>large</sub> (<em>P</em> &lt; 2.2e−16).</p></div></div><div content-id="dsx030-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f2.png?Expires=2147483647&Signature=g23SNnYcJv3Y3Kumz8gyx7YEKKbRBwb1tf~HXIED~ofYN6ZfhGsfJpC~Pdvo7LsgPBnx6RMmjI0wzc~0SV9lFnfIzsSCySOfr~yvBFvKmoNQ3Fgic6NBY8xS6nB0Mjxu~qnbxe3mVztGE5M3QM4Sry3awxGYMlZNgpFKJdirfeS7Z7ZPMkztX2LIVDzyPkbBmMMXycS-slcqL2kDodtAreJ4XZ9MO~P~vBK4-Y0e9suKWHsUitxioiHEIWyl3nJTVyk3vNK3Xl5gDo2ZIb9egy5Bcii7Xyr~4d2dlEk317SZca-99xUS6U-IlXHV1sBTi6PLSPVtJhhTzHAXXtL6AA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Co-adaption at the genome scale. (A) TAAIs of bacteria divided into two groups based on their growth rates. The dataset includes 53 bacteria with available information on growth rates: which were classed into two groups by the mean growth time. The fast group has higher TAAIs than the slow. The GC content of the two groups are similar (The students’ t test: P = 0.42), their genome sizes varied non-significantly (The fast generally has larger genome than the slow: P = 0.07). (B) Prokaryotic organisms’ TAAIs, associated with corresponding genome sizes. The prokaryotic organisms were divided into three groups, showing significantly different TAAIs. Here, 376 prokaryotic genomes were involved in the analysis of correlation between TAAIs and genome sizes. In addition, the GC content also affect the TAAIs along with the increase of genome sizes: GC small &lt; GC median &lt; GC large (P &lt; 2.2e−16)." path-from-xml="dsx030f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086578/dsx030f2.tif" path-from-xml="dsx030f2.tif" target="_blank">View large</a><a section="104086578" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f2.png?Expires=2147483647&Signature=AS23BG8BXdaJcn8wqwjZup1ueXoA1fOuhAXGfnw5fpcyWwSjSVwqPGD1jRcFjj9IEGiji7BGx8dOop0DGhUjBkE98MZysr6HvFIuOV0Q3SujHFTGkNWY4zDCW2iJPJ-0sMTOBLFUdiK7bXgFp7vfl~ncakTpqYCa~pryLrcrh7mikDw3DSr8pLMvrTyknnBetPOXgz3lOQuIc3yVZ7K3ixuzBMRP5lZnCD4Wthqc-zuXDoysj9G9bHYMvzFsTboO0-q3QyJQTscTVUCLi5dSbl7qt28R5SAL5QCzSlH5CWnMTzrpnduHtfuSR2pGLGnZLU2l4gwaY7b41pLJGZ05mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086578&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Co-adaption at the genome scale. (A) TAAIs of bacteria divided into two groups based on their growth rates. The dataset includes 53 bacteria with available information on growth rates: which were classed into two groups by the mean growth time. The fast group has higher TAAIs than the slow. The GC content of the two groups are similar (The students’ <em>t</em> test: <em>P</em> = 0.42), their genome sizes varied non-significantly (The fast generally has larger genome than the slow: <em>P</em> = 0.07). (B) Prokaryotic organisms’ TAAIs, associated with corresponding genome sizes. The prokaryotic organisms were divided into three groups, showing significantly different TAAIs. Here, 376 prokaryotic genomes were involved in the analysis of correlation between TAAIs and genome sizes. In addition, the GC content also affect the TAAIs along with the increase of genome sizes: GC <sub>small</sub> &lt; GC <sub>median</sub> &lt; GC <sub>large</sub> (<em>P</em> &lt; 2.2e−16).</p></div></div><p>Some genomes have non-significant TAAIs and it would be beneficial to understand why. In prokaryotes, genome size and TAAI correlated well (<em>r</em> = 0.49, <em>P</em> &lt; 2.2e−16) and almost all the 96 prokaryotes with bad TAAI/co-adaption do have genome sizes smaller than 2.5 Mb, whereas almost all genomes with good co-adaption are larger than 2.6 Mb. The weak TAAI of smaller genomes is obviously caused by their deficiency in request of translation efficiency: less selection pressure, which could be measured based on the genome size in these organisms. However, in eukaryotes, alternative splicing of messenger RNA results in an inconsistency between genome size and the number of proteins produced. Using the quotient of the number of proteins divided by genome size should be a more reliable reflection of eukaryotes’ actual translation demand (also selection pressure). Consequently, four eukaryotes, <em>Bos taurus</em> (cow), <em>Felis catus</em> (cat), <em>Strongylocentrotus purpuratus</em> (Sea urchin) and <em>Plasmodium falciparum</em> with non-significant TAAIs indeed have smaller quotient. Therefore, it is reasonable that the lower translation demand (also selective pressure) leads some genomes to have bad TAAIs.</p>                            <h3 scrollto-destination=104086580 id="104086580" class="section-title" >3.2. Highly expressed proteins and those connected to acute response tend to have higher TAAI for fast production</h3>
<p>The aforementioned results validated the maximum efficiency principle at the genome (species/strain/organism) scale. Next, we asked whether there are co-adaption divergences within genomes and what such divergences may signify. The proteins within a genome also have different adaptions (variable TAAIs) for their different amino acid compositions (<a reveal-id="dsx030-F3" data-open="dsx030-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Taking <em>H. sapiens</em> as an example, a distinct difference was noted when comparing the co-adaption of the four proteins with the highest TAAIs and the ones of four proteins with the lowest TAAIs, which belong to a gene set with similar GC content and gene lengths (<a reveal-id="dsx030-F3" data-open="dsx030-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The amino acid frequencies of the top four were more consistent with the corresponding genomic tRNA gene copy frequencies. Within a given genome, this co-adaption divergence generally occurred among proteins. </p>                        <a id="104086582" scrollto-destination="104086582"></a>
<div id="dsx030-F3" data-id="dsx030-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f3.png?Expires=2147483647&Signature=cgoevchtyOJC0H03uM5o90-WkM9cfUabEC-3e95S9IZtqrAUx4MSBdsANG1wkSkYjX33rTO39IAQTmr2Z-PJJo5qZJ7zES9Ouhzpx3oLUmRkIyJID1LZa0c6eTEqX2rVn18kyZzd0YGiD3ksCjuWfNcZNkuCzCQAQtvnN1b~CiBmDDZwhM~DK~-P2AjSCcSX1L-IE2flaV28hhd-EJVNdgSu2IAnH3LPI8o-QV5jpBB2I~1730VR38fPR7KSgiGoz-D2LWChAshZngu2KkFaCMo9U~gHtuZqFVENCnbcFsLvgPnJtC4Ryj1YevJZ2Cmha0JwCEgzxp3TWXKQMYyZ5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intra-genome variation and translation factors. (A) TAAI distribution in three model organisms. (B) For human proteins having similar backgrounds (GC%: 0.64–0.66; lengths: 14,300–14,700 bp), they have variable intensities of co-adaption. Amino acid compositions of the top four proteins (Black ones) linearly correlated with the tRNA gene copy numbers, and Amino acid compositions of the bottom four (Red ones) do not change with the growth of the tRNA gene copy numbers, the gene names were listed in the subfigures. (C) Analysis of six model organisms’ abundance shows that highly expressed proteins generally have higher TAAIs than proteins with lower expression levels. Proteins in the high and the low groups have the similar GC content and lengths (The students’ t test: P &gt; 0.05). Their GC contents are: A. thaliana 0.41–0.42; C. elegans 0.34–0.35; S. cerevisiae 0.39–0.41; E. coli 0.50–0.53; B. subtilis 0.50∼0.53; H. sapiens 0.5–0.53. We try to determine the most appropriate GC contents under which more proteins can be included into the analysis. (D) The housekeeping genes and old genes have higher TAAIs than those tissue specific and young genes. These results accord with the results by Ma et al. and Yin et al. that housekeeping and old genes suffered more translation selection." path-from-xml="dsx030f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086582/dsx030f3.tif" path-from-xml="dsx030f3.tif" target="_blank">View large</a><a section="104086582" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f3.png?Expires=2147483647&Signature=XSQ4KOXlTmIjqfeOfcCus5I1fLlEXoJqlHQJ6T2eQTqYhgHKZg598o4OcG04kg4hHw4e0BUv2bIBHIjru17Pn~gN0Y8qwnCU4nvnvrioiTg0WtxzLuNCvu9zpGfr~pQ~TmeOZE6EliKyQ-aNVCxaWFL2~QGDSZA~j46ZtBIiKp1JFqj7UOdN9scbQdkmcxZ69sifa4CUS2Qsp~o06TIo4WS1j2qFZfGXT8-etOASke0LPrRXUgqJVvAyJuGmakUdpoEJqrfKRJLA0vo7r8EVfPCPT0iFxWw1x5jS3T7LXyBmXQAGn2KSEl1bSmYCeFDIkQ2b~GPVVRz9bnq2Gx3FdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086582&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intra-genome variation and translation factors. (A) TAAI distribution in three model organisms. (B) For human proteins having similar backgrounds (GC%: 0.64–0.66; lengths: 14,300–14,700 bp), they have variable intensities of co-adaption. Amino acid compositions of the top four proteins (Black ones) linearly correlated with the tRNA gene copy numbers, and Amino acid compositions of the bottom four (Red ones) do not change with the growth of the tRNA gene copy numbers, the gene names were listed in the subfigures. (C) Analysis of six model organisms’ abundance shows that highly expressed proteins generally have higher TAAIs than proteins with lower expression levels. Proteins in the high and the low groups have the similar GC content and lengths (The students’ <em>t</em> test: <em>P</em> &gt; 0.05). Their GC contents are: <em>A. thaliana</em> 0.41–0.42; <em>C. elegans</em> 0.34–0.35; <em>S. cerevisiae</em> 0.39–0.41; <em>E. coli</em> 0.50–0.53; <em>B. subtilis</em> 0.50∼0.53; <em>H. sapiens</em> 0.5–0.53. We try to determine the most appropriate GC contents under which more proteins can be included into the analysis. (D) The housekeeping genes and old genes have higher TAAIs than those tissue specific and young genes. These results accord with the results by Ma <em>et al.</em> and Yin <em>et al.</em> that housekeeping and old genes suffered more translation selection.</p></div></div><div content-id="dsx030-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f3.png?Expires=2147483647&Signature=cgoevchtyOJC0H03uM5o90-WkM9cfUabEC-3e95S9IZtqrAUx4MSBdsANG1wkSkYjX33rTO39IAQTmr2Z-PJJo5qZJ7zES9Ouhzpx3oLUmRkIyJID1LZa0c6eTEqX2rVn18kyZzd0YGiD3ksCjuWfNcZNkuCzCQAQtvnN1b~CiBmDDZwhM~DK~-P2AjSCcSX1L-IE2flaV28hhd-EJVNdgSu2IAnH3LPI8o-QV5jpBB2I~1730VR38fPR7KSgiGoz-D2LWChAshZngu2KkFaCMo9U~gHtuZqFVENCnbcFsLvgPnJtC4Ryj1YevJZ2Cmha0JwCEgzxp3TWXKQMYyZ5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intra-genome variation and translation factors. (A) TAAI distribution in three model organisms. (B) For human proteins having similar backgrounds (GC%: 0.64–0.66; lengths: 14,300–14,700 bp), they have variable intensities of co-adaption. Amino acid compositions of the top four proteins (Black ones) linearly correlated with the tRNA gene copy numbers, and Amino acid compositions of the bottom four (Red ones) do not change with the growth of the tRNA gene copy numbers, the gene names were listed in the subfigures. (C) Analysis of six model organisms’ abundance shows that highly expressed proteins generally have higher TAAIs than proteins with lower expression levels. Proteins in the high and the low groups have the similar GC content and lengths (The students’ t test: P &gt; 0.05). Their GC contents are: A. thaliana 0.41–0.42; C. elegans 0.34–0.35; S. cerevisiae 0.39–0.41; E. coli 0.50–0.53; B. subtilis 0.50∼0.53; H. sapiens 0.5–0.53. We try to determine the most appropriate GC contents under which more proteins can be included into the analysis. (D) The housekeeping genes and old genes have higher TAAIs than those tissue specific and young genes. These results accord with the results by Ma et al. and Yin et al. that housekeeping and old genes suffered more translation selection." path-from-xml="dsx030f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086582/dsx030f3.tif" path-from-xml="dsx030f3.tif" target="_blank">View large</a><a section="104086582" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f3.png?Expires=2147483647&Signature=XSQ4KOXlTmIjqfeOfcCus5I1fLlEXoJqlHQJ6T2eQTqYhgHKZg598o4OcG04kg4hHw4e0BUv2bIBHIjru17Pn~gN0Y8qwnCU4nvnvrioiTg0WtxzLuNCvu9zpGfr~pQ~TmeOZE6EliKyQ-aNVCxaWFL2~QGDSZA~j46ZtBIiKp1JFqj7UOdN9scbQdkmcxZ69sifa4CUS2Qsp~o06TIo4WS1j2qFZfGXT8-etOASke0LPrRXUgqJVvAyJuGmakUdpoEJqrfKRJLA0vo7r8EVfPCPT0iFxWw1x5jS3T7LXyBmXQAGn2KSEl1bSmYCeFDIkQ2b~GPVVRz9bnq2Gx3FdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086582&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intra-genome variation and translation factors. (A) TAAI distribution in three model organisms. (B) For human proteins having similar backgrounds (GC%: 0.64–0.66; lengths: 14,300–14,700 bp), they have variable intensities of co-adaption. Amino acid compositions of the top four proteins (Black ones) linearly correlated with the tRNA gene copy numbers, and Amino acid compositions of the bottom four (Red ones) do not change with the growth of the tRNA gene copy numbers, the gene names were listed in the subfigures. (C) Analysis of six model organisms’ abundance shows that highly expressed proteins generally have higher TAAIs than proteins with lower expression levels. Proteins in the high and the low groups have the similar GC content and lengths (The students’ <em>t</em> test: <em>P</em> &gt; 0.05). Their GC contents are: <em>A. thaliana</em> 0.41–0.42; <em>C. elegans</em> 0.34–0.35; <em>S. cerevisiae</em> 0.39–0.41; <em>E. coli</em> 0.50–0.53; <em>B. subtilis</em> 0.50∼0.53; <em>H. sapiens</em> 0.5–0.53. We try to determine the most appropriate GC contents under which more proteins can be included into the analysis. (D) The housekeeping genes and old genes have higher TAAIs than those tissue specific and young genes. These results accord with the results by Ma <em>et al.</em> and Yin <em>et al.</em> that housekeeping and old genes suffered more translation selection.</p></div></div><p>Selective pressure within genomes is reflected by the direct results of translation efficiency: proteins’ abundances, even though the two values are not entirely equivalent. Co-adaption should reflect the supply of aminoacyl-tRNA, which ultimately affects the final protein synthesis. We compared six model organisms’ TAAIs and found that proteins with higher expression levels had clearly higher TAAIs than proteins with lower expression levels by Student's <em>t</em> test, even these proteins are with similar GC content and gene lengths (<a reveal-id="dsx030-F3" data-open="dsx030-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). Furthermore, when using a linear fit, the TAAIs and the protein abundances showed significantly positive correlations (<em>P</em> &lt; 1e−6; <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s3_dsx030.docx?Expires=2147483647&Signature=etWKG95watVSRTGmXeNLT0vW7d4cICEzOETuLyiPjZZTVoVw1wP3N32CCJnNqy8TN~LsvTESo4poAIcbCykl-4-x35jHykD1Es-0SqutZ6Vpgiy~Qv69LkvskOguGSwumsBhpHS-26iQJ0PfZHGohrbFosHqGMkNrhQxCIUwZ35TT-NRRZkyVmr37Y~NsdxzUPpEb3lH3KmT3dPGCx4GLb-WrgD~r3lKj9PZij4sA3BwlopYmjfqqNJYosi3O01ctvkNQKS7dV22XsxAm1vxfABlD7QXhHMXpAlSytWqS04jFGE3aZw-n9mdqFeKYWE309wxSsNWeiAhZqLldySYQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). This result is consistent with the idea that tRNA level has direct effects on translation efficiency.<a reveal-id="dsx030-B41" data-open="dsx030-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Thus, as a reflection of the translation rate, protein abundances correlate with TAAIs to a certain extent, and their relationship seems to be a consequence of selection pressure to keep a suitable translation rate.</p><p>To further explore this finding, and considering that paralogous genes in the same family have similar molecular evolutionary stresses and changes,<a reveal-id="dsx030-B42" data-open="dsx030-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> we compared the TAAIs in <em>E. coli</em> and yeast according to gene function groups: ribosome subunits, cell division,<a reveal-id="dsx030-B43" data-open="dsx030-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> two-component system (including response regulators and sensors,<a reveal-id="dsx030-B44" data-open="dsx030-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> mismatch repair<a reveal-id="dsx030-B45" data-open="dsx030-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and sugar metabolism.<a reveal-id="dsx030-B46" data-open="dsx030-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The average TAAIs and protein abundances for five groups of genes were calculated (<a reveal-id="dsx030-T1" data-open="dsx030-T1" class="link link-reveal link-table xref-fig">Table 1</a>). For the <em>E. coli</em> and yeast genomes, proteins from all these groups had average TAAIs higher than the genome average (Student’s <em>t</em> test: <em>P</em> = 3.54e−12). Three of the five groups correspond to acute responses (cell division, two-component system, and mismatch repair), and the other two groups are related to fast growth (ribosome subunits and sugar metabolism). Ribosome subunits are important participants in the translation process, and there is no doubt that ribosome subunits have the highest abundance, TAAI and codon usage bias index CAI.<a reveal-id="dsx030-B4" data-open="dsx030-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Sugar metabolism proteins, which includes proteins in ‘Amino sugar and nucleotide sugar metabolism’, also has higher TAAI and abundance than genome average. Although the abundances of proteins in the other three categories are lower than genome average, their TAAIs were higher. We further compared the protein abundance and TAAIs of experimentally determined upregulated yeast genes,<a reveal-id="dsx030-B47" data-open="dsx030-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> and observed similar results (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s4_dsx030.docx?Expires=2147483647&Signature=ErFxiRxhSdSIqhaotruYxWkdWONWd5MZyQyAz6y0jZK~cvRNQsVSk3htI8JjFvJ5rBG5XE9Kf96E0DBiMX6nwR~LDOvUDjBBlfRZ7FvaqoyXPfhlHH9YqlIIVYTFLaw36llFFmWClTpafjOppBz-Mo~a5knBin9CJMVZi56MiaE~NuBopmZntLc2FoTULYNUB2DX6HN8QFkpMQV-qPy-oKZQR~aTNGI4lfQRRnH6ZEpQB8lpg9RXj-34ckURP5dOFVluurQFlsvRV~3lOCazuvn8FxQ94ffDdq5BOGNRmFEHiHWsKSyE1y3eI2DBPRYVci3yShaEc7OhgnOme5xuMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Thus, proteins involved in acute responses under more selective pressure, generally have good co-adaption relationships. <div content-id="dsx030-T1" class="table-modal"><div class="table-wrap"><div id="dsx030-T1" data-id="dsx030-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Analysis of the potentially rapidly expressed proteins according to their functions in <em>E. coli</em> and <em>S. cerevisiae</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="4"><em>E. coli</em><hr /> </th><th colspan="4"><em>S. cerevisiae</em><hr /> </th></tr><tr><th> </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th></tr></thead><tbody><tr><td>Whole genome </td><td>3133 </td><td>319.2 </td><td>0.51 </td><td>0.35 </td><td>6087 </td><td>163.8 </td><td>0.62 </td><td>0.18 </td></tr><tr><td>Ribosome subunits </td><td>57 </td><td>4724.28 </td><td>0.59 </td><td>0.63 </td><td>178 </td><td>1231.65 </td><td>0.74 </td><td>0.48 </td></tr><tr><td>Cell division </td><td>19 </td><td><strong>131.84</strong> </td><td>0.57 </td><td>0.35 </td><td>20 </td><td><strong>15.09</strong> </td><td>0.67 </td><td><strong>0.16</strong> </td></tr><tr><td>Two-component system </td><td>58 </td><td><strong>80.7</strong> </td><td>0.57 </td><td><strong>0.29</strong> </td><td>216 </td><td>203.86 </td><td>0.65 </td><td>0.19 </td></tr><tr><td>Mismatch repair </td><td>20 </td><td><strong>57.21</strong> </td><td>0.53 </td><td><strong>0.34</strong> </td><td>19 </td><td><strong>32.51</strong> </td><td>0.72 </td><td><strong>0.16</strong> </td></tr><tr><td>Sugar metabolism </td><td>47 </td><td>325.52 </td><td>0.52 </td><td>0.41 </td><td>28 </td><td>563.53 </td><td>0.69 </td><td>0.26 </td></tr></tbody></table></div><div content-id="dsx030-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="4"><em>E. coli</em><hr /> </th><th colspan="4"><em>S. cerevisiae</em><hr /> </th></tr><tr><th> </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th></tr></thead><tbody><tr><td>Whole genome </td><td>3133 </td><td>319.2 </td><td>0.51 </td><td>0.35 </td><td>6087 </td><td>163.8 </td><td>0.62 </td><td>0.18 </td></tr><tr><td>Ribosome subunits </td><td>57 </td><td>4724.28 </td><td>0.59 </td><td>0.63 </td><td>178 </td><td>1231.65 </td><td>0.74 </td><td>0.48 </td></tr><tr><td>Cell division </td><td>19 </td><td><strong>131.84</strong> </td><td>0.57 </td><td>0.35 </td><td>20 </td><td><strong>15.09</strong> </td><td>0.67 </td><td><strong>0.16</strong> </td></tr><tr><td>Two-component system </td><td>58 </td><td><strong>80.7</strong> </td><td>0.57 </td><td><strong>0.29</strong> </td><td>216 </td><td>203.86 </td><td>0.65 </td><td>0.19 </td></tr><tr><td>Mismatch repair </td><td>20 </td><td><strong>57.21</strong> </td><td>0.53 </td><td><strong>0.34</strong> </td><td>19 </td><td><strong>32.51</strong> </td><td>0.72 </td><td><strong>0.16</strong> </td></tr><tr><td>Sugar metabolism </td><td>47 </td><td>325.52 </td><td>0.52 </td><td>0.41 </td><td>28 </td><td>563.53 </td><td>0.69 </td><td>0.26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Proteins in bold face had lower corresponding values than genome’s average values.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx030-T1" data-id="dsx030-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Analysis of the potentially rapidly expressed proteins according to their functions in <em>E. coli</em> and <em>S. cerevisiae</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="4"><em>E. coli</em><hr /> </th><th colspan="4"><em>S. cerevisiae</em><hr /> </th></tr><tr><th> </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th></tr></thead><tbody><tr><td>Whole genome </td><td>3133 </td><td>319.2 </td><td>0.51 </td><td>0.35 </td><td>6087 </td><td>163.8 </td><td>0.62 </td><td>0.18 </td></tr><tr><td>Ribosome subunits </td><td>57 </td><td>4724.28 </td><td>0.59 </td><td>0.63 </td><td>178 </td><td>1231.65 </td><td>0.74 </td><td>0.48 </td></tr><tr><td>Cell division </td><td>19 </td><td><strong>131.84</strong> </td><td>0.57 </td><td>0.35 </td><td>20 </td><td><strong>15.09</strong> </td><td>0.67 </td><td><strong>0.16</strong> </td></tr><tr><td>Two-component system </td><td>58 </td><td><strong>80.7</strong> </td><td>0.57 </td><td><strong>0.29</strong> </td><td>216 </td><td>203.86 </td><td>0.65 </td><td>0.19 </td></tr><tr><td>Mismatch repair </td><td>20 </td><td><strong>57.21</strong> </td><td>0.53 </td><td><strong>0.34</strong> </td><td>19 </td><td><strong>32.51</strong> </td><td>0.72 </td><td><strong>0.16</strong> </td></tr><tr><td>Sugar metabolism </td><td>47 </td><td>325.52 </td><td>0.52 </td><td>0.41 </td><td>28 </td><td>563.53 </td><td>0.69 </td><td>0.26 </td></tr></tbody></table></div><div content-id="dsx030-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="4"><em>E. coli</em><hr /> </th><th colspan="4"><em>S. cerevisiae</em><hr /> </th></tr><tr><th> </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th><th>Number </th><th>Abundance </th><th>TAAI </th><th>CAI </th></tr></thead><tbody><tr><td>Whole genome </td><td>3133 </td><td>319.2 </td><td>0.51 </td><td>0.35 </td><td>6087 </td><td>163.8 </td><td>0.62 </td><td>0.18 </td></tr><tr><td>Ribosome subunits </td><td>57 </td><td>4724.28 </td><td>0.59 </td><td>0.63 </td><td>178 </td><td>1231.65 </td><td>0.74 </td><td>0.48 </td></tr><tr><td>Cell division </td><td>19 </td><td><strong>131.84</strong> </td><td>0.57 </td><td>0.35 </td><td>20 </td><td><strong>15.09</strong> </td><td>0.67 </td><td><strong>0.16</strong> </td></tr><tr><td>Two-component system </td><td>58 </td><td><strong>80.7</strong> </td><td>0.57 </td><td><strong>0.29</strong> </td><td>216 </td><td>203.86 </td><td>0.65 </td><td>0.19 </td></tr><tr><td>Mismatch repair </td><td>20 </td><td><strong>57.21</strong> </td><td>0.53 </td><td><strong>0.34</strong> </td><td>19 </td><td><strong>32.51</strong> </td><td>0.72 </td><td><strong>0.16</strong> </td></tr><tr><td>Sugar metabolism </td><td>47 </td><td>325.52 </td><td>0.52 </td><td>0.41 </td><td>28 </td><td>563.53 </td><td>0.69 </td><td>0.26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Proteins in bold face had lower corresponding values than genome’s average values.</p></div></div></div></div></p><p>According to Yin <em>et al.</em><a reveal-id="dsx030-B33" data-open="dsx030-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and Ma <em>et al.,</em><a reveal-id="dsx030-B32" data-open="dsx030-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> old genes and housekeeping experience stronger translational selection. Thus, the co-adaption in these genes should be better. We employed the gene lists from this two papers and arrived at the expectant conclusion (<a reveal-id="dsx030-F3" data-open="dsx030-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>, <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s5_dsx030.xlsx?Expires=2147483647&amp;Signature=pdAMIpw9EL77tnye1WVRhNLHBcrTFtsMI0Elrg7Rtiaf5pFXcN23CdOH9DMtCCwb7H5uxnz3vKNxvDDAuNmGL0KKh1L4JG0EXEZPkRU3W9ON49UhK5t5ddZUD3ned3-jDu2q2GvhFnR2gsoe0Ku4UKfebw4b4P2-0TQJUKRMBFyFu8E6jrKgV7LYESQsYFi4x1bu7ZO17P~JED-UV9Bh7OlcbwNC9bBr85XV9vT1IG54fCuPLrWaoR8DsoDt3Hscf-8aj74P061p~H-gm1oFbpdo0acxs12wfeoAdjJDQmUGt23ZBYve0bWj3HK4GmQY1wDWpZH8-7oQzyDKJIc6yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Additionally, those horizontally transferred genes experiencing less translation selection also have bad co-adaptions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig1_dsx030.png?Expires=2147483647&amp;Signature=3OZxq~SIXi7yuOp1eA~jzW3LxGVmA4YwRY9MeNCEFUDnBywaP~R2tEb8U2QEXNJ1T9CF0PGo4ZM7wpbwGeytKrYZqykK~zv-18xKko-Rn8B4SYGKzLHXDormcl3z1aGYIFpZ3F8N-v2GPjATsICkYaKlXMxyFxreCQzM99QyiOb74dUbTqTsmO9cnOt9WHW~j3TejjNZ66bI-ESSrjFtfcaXCJGB0Oj6Pk8t2g54ly2TJtJ8KKA5mjJSfkBFlhNG0lJsISKtUGQBAtyP3UI6L7rkvOx3jfSrVzBenIPRz7lF0DZGDfA0DGxdOLIxNlLEiM5~sVkB~dM~3Hu5BqyioQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Therefore, our second reasoning was also validated: at the scale of individual proteins, co-adaption intensity may vary among the protein collective within a genome. Some proteins need to be expressed rapidly to maintain their quantity to be connected with an acute response.</p>                            <h3 scrollto-destination=104086586 id="104086586" class="section-title" >3.3. Experimentally optimizing co-adaption indeed lift the protein expression</h3>
<p>According to the principle and the results above, co-adaption was associated with the translation rate, prompting us to examine whether the proteins with high TAAIs indeed have high translation rates <em>in vivo</em>. To test this conjecture, one copy of a specific tRNA gene that might increase or decrease the TAAI was introduced to <em>E. coli</em> along with a gene encoding enhanced green fluorescent protein (EGFP) after which EGFP protein synthesis was analyzed.</p><p>Proteins with higher TAAIs might have higher translation rates, and thus higher production levels. EGFP is easy to express and detect, and constructs for tRNA overexpression have previously been designed and tested.<a reveal-id="dsx030-B48" data-open="dsx030-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Therefore, by combining the expression of EGFP with a tRNA gene, we can detect the effect of a specific tRNA on EGFP expression. Increasing the copy number of the corresponding tRNA gene may increase or decrease the TAAI for EGFP and the whole genome (<a reveal-id="dsx030-F4" data-open="dsx030-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). EGFP had a TAAI of 0.45 when expressed with the original frequencies of <em>E. coli</em> tRNA genes. When one gene copy encoding either <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>U</mi><mi>C</mi></mrow><mrow><mi>A</mi><mi>s</mi><mi>p</mi></mrow></msubsup></mrow></math>⁠</span>, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>A</mi><mi>T</mi></mrow><mrow><mi>I</mi><mi>l</mi><mi>e</mi></mrow></msubsup></mrow></math></span> or <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>T</mi><mi>A</mi></mrow><mrow><mi>T</mi><mi>y</mi><mi>r</mi></mrow></msubsup></mrow></math></span> was introduced, the corresponding <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Δ</mi><mi>T</mi><mi>A</mi><mi>A</mi><mi>I</mi><mi>s</mi></mrow></math></span> for EGFP were 0.03, 0.0008 or −0.007, respectively, and the cumulative <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Δ</mi><mi>T</mi><mi>A</mi><mi>A</mi><mi>I</mi><mi>s</mi></mrow></math></span> for all proteins of the genome were −28.68, −43.77 or 78.68, respectively. The following sequences were constructed in plasmids: <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>E</mi><mi>G</mi><mi>F</mi><mi>P</mi></mrow></math></span> (control group), <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>U</mi><mi>C</mi></mrow><mrow><mi>A</mi><mi>s</mi><mi>p</mi></mrow></msubsup><mo>−</mo><mi>E</mi><mi>G</mi><mi>F</mi><mi>P</mi></mrow></math></span> (tRNA-Asp), <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>A</mi><mi>T</mi></mrow><mrow><mi>I</mi><mi>l</mi><mi>e</mi></mrow></msubsup><mo>−</mo><mi>E</mi><mi>G</mi><mi>F</mi><mi>P</mi></mrow></math></span> (tRNA-Ile) and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>T</mi><mi>A</mi></mrow><mrow><mi>T</mi><mi>y</mi><mi>r</mi></mrow></msubsup><mo>−</mo><mi>E</mi><mi>G</mi><mi>F</mi><mi>P</mi></mrow></math></span> (tRNA-Tyr), which were used to transfer <em>E. coli</em> Top10 cells (<a reveal-id="dsx030-F4" data-open="dsx030-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). In <em>E. coli</em>, there is only one type of tRNA (isoacceptor tRNA) for Asp, Tyr and Ile, which means that codon bias has no observable effect on the increase in EGFP expression. We observed fluorescence intensity with confocal microscopy and found that the EGFP yield of the experimental group expressing <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>U</mi><mi>C</mi></mrow><mrow><mi>A</mi><mi>s</mi><mi>p</mi></mrow></msubsup><mo>−</mo><mi>E</mi><mi>G</mi><mi>F</mi><mi>P</mi></mrow></math></span> was significantly higher than the others at the same time point (<a reveal-id="dsx030-F4" data-open="dsx030-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). Detailed fluorescence intensities of the four groups acquired with a fluorospectro photometer (<a reveal-id="dsx030-F4" data-open="dsx030-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a><strong>)</strong>, showed that the EGFP production efficiency of the four groups from low to high was as follows: Control, tRNA-Ile, tRNA-Tyr and tRNA-Asp. The fluorescence intensities were consistent with western blotting results (<a reveal-id="dsx030-F4" data-open="dsx030-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>). The experimental group tRNA-Asp produced ten times more EGFP than the Control. Considering the dynamic processes involved in EGFP abundance variation, the slope of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>U</mi><mi>C</mi></mrow><mrow><mi>A</mi><mi>s</mi><mi>p</mi></mrow></msubsup><mo>−</mo><mi>E</mi><mi>G</mi><mi>F</mi><mi>P</mi></mrow></math></span> abundance was much higher than that of the control, indicating that the translation rate of the former was higher. </p>                        <a id="104086589" scrollto-destination="104086589"></a>
<div id="dsx030-F4" data-id="dsx030-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f4.png?Expires=2147483647&amp;Signature=gRplCX0tSrXz5JjCX6dCdCKq4r4DQj3KKypY4DH4Fz30D10~t6-V3zVbPJB~dRkuFLZFAt~VllphL-a8aMd4PXf02yJj2J8swyq~Pc1HP3-JN~f~d~emayna5PG4ZEof9zsTNqTbybGeq1H~UWJuoi9BC8zl5rSj1MjSMDZczZ9BNHeKMVYqhGYcf3CijX8tN3RrWnmyrPCgbZI6NntrViHGkGhP80iW6pJMuU6b5rhC9q6FDTsTIX7bX0aCYP4McD-LzlhFc1YseI2wJXDWtFLbsFdT8hCgorNtA0E8p-jVYDAmaX64~9CyGJPcVgaliFXITo2ZXTOucMkB7xKeiQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="EGFP expression of original and optimized TAAIs in E. coli. (A) ΔTAAI is the upregulated TAAI value resulting from adding a copy of one of the 20 standard tRNAs. The star marks tRNA-Asp, tRNA-Ile and tRNA-Tyr. (B) Confocal micrographs of the control and experimental groups present different fluorescence intensities in the EGFP channel; the corresponding merge figures of the bright-field and EGFP channel are shown in Supplementary Fig. S2. (C) Fluorescence intensities of four nascent sequences with EGFP at 513 nm from 0 to 2.5 h. All of these results showed that in the experimental group transformed with the Asp tRNA gene, there was a lift approximately ten-fold. (D) Western blot results for the nascent sequences. The following histograms show the normalized density of the corresponding lane, and the chemiluminescence intensity of the corresponding target band using stain-free technology (Bio-Rad). The corresponding electrophoretogram, shown in Supplementary Fig. S2B, reflects the loading volume of the total proteins." path-from-xml="dsx030f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086589/dsx030f4.tif" path-from-xml="dsx030f4.tif" target="_blank">View large</a><a section="104086589" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f4.png?Expires=2147483647&amp;Signature=1uJjuSumaIgHSntlu6fFzr~Nl3aYJxBlonoTCYZPiu~HWabkm-v676PUOwSZgMAKsP0lBH0JHSSUmmOtMJPobnihz7NgOj4I8w5SHE3pnibj7l08HDNRD2crv2DCxjRLPy7xXHJd-W0stGblVp024z2pGE35fratEWGwMxy9r1cIu9fQmFjVUM2cRYfuBNwcvDkBLe~3apsuTtVfnynuzLWJLjsxc1XfWyPorlaQERpPxtYwFtyM~T7SxrdpTFxMq3qJqlndfLKk8k4Qhy5UZ8rPwpZWo~4GrU4K0s6KFNNkjH455W-VrS02Uusjlq4bun6X1hYpdMI5tpuqF60aSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=104086589&amp;ar=3953933&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx030f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>EGFP expression of original and optimized TAAIs in <em>E. coli</em>. (A) <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Δ</mi><mi>T</mi><mi>A</mi><mi>A</mi><mi>I</mi></mrow></math></span> is the upregulated TAAI value resulting from adding a copy of one of the 20 standard tRNAs. The star marks tRNA-Asp, tRNA-Ile and tRNA-Tyr. (B) Confocal micrographs of the control and experimental groups present different fluorescence intensities in the EGFP channel; the corresponding merge figures of the bright-field and EGFP channel are shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&amp;Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. (C) Fluorescence intensities of four nascent sequences with EGFP at 513 nm from 0 to 2.5 h. All of these results showed that in the experimental group transformed with the Asp tRNA gene, there was a lift approximately ten-fold. (D) Western blot results for the nascent sequences. The following histograms show the normalized density of the corresponding lane, and the chemiluminescence intensity of the corresponding target band using stain-free technology (Bio-Rad). The corresponding electrophoretogram, shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&amp;Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>, reflects the loading volume of the total proteins.</p></div></div><div content-id="dsx030-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f4.png?Expires=2147483647&amp;Signature=gRplCX0tSrXz5JjCX6dCdCKq4r4DQj3KKypY4DH4Fz30D10~t6-V3zVbPJB~dRkuFLZFAt~VllphL-a8aMd4PXf02yJj2J8swyq~Pc1HP3-JN~f~d~emayna5PG4ZEof9zsTNqTbybGeq1H~UWJuoi9BC8zl5rSj1MjSMDZczZ9BNHeKMVYqhGYcf3CijX8tN3RrWnmyrPCgbZI6NntrViHGkGhP80iW6pJMuU6b5rhC9q6FDTsTIX7bX0aCYP4McD-LzlhFc1YseI2wJXDWtFLbsFdT8hCgorNtA0E8p-jVYDAmaX64~9CyGJPcVgaliFXITo2ZXTOucMkB7xKeiQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="EGFP expression of original and optimized TAAIs in E. coli. (A) ΔTAAI is the upregulated TAAI value resulting from adding a copy of one of the 20 standard tRNAs. The star marks tRNA-Asp, tRNA-Ile and tRNA-Tyr. (B) Confocal micrographs of the control and experimental groups present different fluorescence intensities in the EGFP channel; the corresponding merge figures of the bright-field and EGFP channel are shown in Supplementary Fig. S2. (C) Fluorescence intensities of four nascent sequences with EGFP at 513 nm from 0 to 2.5 h. All of these results showed that in the experimental group transformed with the Asp tRNA gene, there was a lift approximately ten-fold. (D) Western blot results for the nascent sequences. The following histograms show the normalized density of the corresponding lane, and the chemiluminescence intensity of the corresponding target band using stain-free technology (Bio-Rad). The corresponding electrophoretogram, shown in Supplementary Fig. S2B, reflects the loading volume of the total proteins." path-from-xml="dsx030f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086589/dsx030f4.tif" path-from-xml="dsx030f4.tif" target="_blank">View large</a><a section="104086589" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f4.png?Expires=2147483647&amp;Signature=1uJjuSumaIgHSntlu6fFzr~Nl3aYJxBlonoTCYZPiu~HWabkm-v676PUOwSZgMAKsP0lBH0JHSSUmmOtMJPobnihz7NgOj4I8w5SHE3pnibj7l08HDNRD2crv2DCxjRLPy7xXHJd-W0stGblVp024z2pGE35fratEWGwMxy9r1cIu9fQmFjVUM2cRYfuBNwcvDkBLe~3apsuTtVfnynuzLWJLjsxc1XfWyPorlaQERpPxtYwFtyM~T7SxrdpTFxMq3qJqlndfLKk8k4Qhy5UZ8rPwpZWo~4GrU4K0s6KFNNkjH455W-VrS02Uusjlq4bun6X1hYpdMI5tpuqF60aSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=104086589&amp;ar=3953933&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx030f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>EGFP expression of original and optimized TAAIs in <em>E. coli</em>. (A) <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Δ</mi><mi>T</mi><mi>A</mi><mi>A</mi><mi>I</mi></mrow></math></span> is the upregulated TAAI value resulting from adding a copy of one of the 20 standard tRNAs. The star marks tRNA-Asp, tRNA-Ile and tRNA-Tyr. (B) Confocal micrographs of the control and experimental groups present different fluorescence intensities in the EGFP channel; the corresponding merge figures of the bright-field and EGFP channel are shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&amp;Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. (C) Fluorescence intensities of four nascent sequences with EGFP at 513 nm from 0 to 2.5 h. All of these results showed that in the experimental group transformed with the Asp tRNA gene, there was a lift approximately ten-fold. (D) Western blot results for the nascent sequences. The following histograms show the normalized density of the corresponding lane, and the chemiluminescence intensity of the corresponding target band using stain-free technology (Bio-Rad). The corresponding electrophoretogram, shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&amp;Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>, reflects the loading volume of the total proteins.</p></div></div><p>To further rule out the possibility of codon usage influence, we compared the number and coding order of codons for the three tRNAs in EGFP (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2C</a></span>). The EGFP mRNA sequence has one type of Ile codon (ATC), two types of Tyr codons (TAT: 9%, TAC: 91%), and two Asp codons (GAT: 11%; GAC: 89%). All preferred codons are the corresponding codons for the <em>E. coli</em> cognate tRNAs. Therefore, there should be no significant variant effect of varying from the preferred codon. Then, we calculated the dispersion degree by analysis of variance, as the order of tRNA can influence its recycling during translation.<a reveal-id="dsx030-B14" data-open="dsx030-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> We analyzed the variance of amino acid sites both locally and globally. The variance of the first eleven sites for Ile, Tyr and Asp are: 53, 61 and 46. The corresponding recycling effects should be weakened when increasing specific tRNA gene copy numbers. In fact, increasing the Ile tRNA gene copy number does not significantly increase EGFP production. Together, the results do not indicate an influence of tRNA recycling or preferred codon usage. This experiment confirmed that co-adaption has a clear effect on translation rate. Thus, optimizing co-adaption could significantly promote translation production of foreign proteins.</p>                            <h2 scrollto-destination=104086591 id="104086591" class="section-title" >4. Discussion</h2>
                            <h3 scrollto-destination=104086592 id="104086592" class="section-title" >4.1. This co-adaption relationship benefits translation rate differently from codon usage</h3>
<p>Cells are believed to evolve to maximize efficiency and minimize resource and energy cost.<a reveal-id="dsx030-B49" data-open="dsx030-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsx030-B50" data-open="dsx030-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> We hypothesized that this principle would affect translation mechanisms, and we tested this conjecture based on the basic translation ‘tool’ tRNAs and the ‘raw material’ amino acids. To maximally utilize the resources, we reasoned that the quantities of the 20 tRNAs and amino acids in a species should be consistent based on this principle. For simple and convenient analysis, we used the tRNA gene copy number as the proxy for the former, and used the amino acid frequency as the latter. The genome has an average vector of amino acid frequency and each protein also has its vector form of amino acid frequency. Hence, there would be a general co-adaption value for each genome and a specific co-adaption value for each protein. Using correlation and abundance (functional group) analyses we validated our two conjectures, which are logical outcomes of the maximum efficiency and minimum cost principle. Based on the results and analyses, the genome’s TAAI could be regarded as a proxy for general translation efficiency or actual translation needs. Proteins’ TAAIs reflect the highest translation efficiency or translation need in extreme conditions.</p><p>Co-adaption is a global effect exerted on proteins and organisms. Each organism has a specific amino acid usage and a co-adapted tRNA gene copy number. This co-adaption maximizes the translation efficiency of the complete proteomes. The larger translation pressure the organism is exposed to, the higher average TAAI it has. In a genome, almost all the proteins have positive TAAI values (<a reveal-id="dsx030-F3" data-open="dsx030-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Hence, this co-adaption as a translation rate associated factor is applicable to all three domains of life and all proteins within an organism. However, facing multi rate-limiting factors, it is uncertain how much of the final translation rate can be determined by the TAAI.</p><p>The fact that codons affect the expression is well-known.<a reveal-id="dsx030-B1" data-open="dsx030-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx030-B2" data-open="dsx030-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx030-B8" data-open="dsx030-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx030-B51" data-open="dsx030-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsx030-B52" data-open="dsx030-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Then we wonder whether the adaption of codon usage to isoacceptor tRNA gene copy number resulted in co-adaption between amino acid composition and tRNA gene copy number. We computed the correlation index according to codon usage of multi-organisms from three domains and multi-genes, and detected very weak adaption for codon usage to important translation rate-limiting factor tRNA copy number (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplemental_table_s6_dsx030.xlsx?Expires=2147483647&amp;Signature=Vq~tN5s~~dywi6GLp1HgNzrtujOmmvakBGs4JvcadgXf~1xnEjM1KdknlmJkpxq0ortgriGZCQ6nfq-FkEyt2gtROgqDEVz0ei4OnUpGQkgjBPHh7JCy7Eok82CGUInsTTEmhfBK6fGcueBxUCxpFzwR0BKCSjKsfyRe~4LWRWahUPku8fJ~VkK~IsRZfr7MoR9Hno-23Tm3NGYmaOfeK9PHcwwPlPz7P5gZIQZAN~K17o~UdU6lPcGJ-sd620V9w5~dWYASzdJJng9pUezt~9agQAR44ALi3IfYgTYr48TyWvICkqwNGjb~IucJKNDFcAJn05CzlNsldQNMTbXuzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig1_dsx030.png?Expires=2147483647&amp;Signature=3OZxq~SIXi7yuOp1eA~jzW3LxGVmA4YwRY9MeNCEFUDnBywaP~R2tEb8U2QEXNJ1T9CF0PGo4ZM7wpbwGeytKrYZqykK~zv-18xKko-Rn8B4SYGKzLHXDormcl3z1aGYIFpZ3F8N-v2GPjATsICkYaKlXMxyFxreCQzM99QyiOb74dUbTqTsmO9cnOt9WHW~j3TejjNZ66bI-ESSrjFtfcaXCJGB0Oj6Pk8t2g54ly2TJtJ8KKA5mjJSfkBFlhNG0lJsISKtUGQBAtyP3UI6L7rkvOx3jfSrVzBenIPRz7lF0DZGDfA0DGxdOLIxNlLEiM5~sVkB~dM~3Hu5BqyioQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1B</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&amp;Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S3A</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig3_dsx030.png?Expires=2147483647&amp;Signature=UM6ARvGVdUpwW9mBaNgYYO4x3N5coet12Ay0XP8BQjIPy9bj-SF~QoR4RWx0Y25EJ3ZJSzMAisCM9MDW8l~lS1ldbWpRaDvnX-G2BzvjwsMeXwEg-MKwE8zczEXpw7pFU-s48dHg-EQhgux3RN5FrrIFc3TASxZia45s3SzQgeIN6Ky936DJsIwQqyM4mP-hKU4lWHbrzPS6rZzGfnscMGGTxo8eEdwywD-sxYxOzmHbDZ95~LFska7yQDS1O5B8P4iknfNPPEP7KItbn8zHHU1ub7qYxxtcPrEut7KGyr6FOGYckKikar1p6hMwAcKU22broiBuKtbPuN4DaFXuEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S3B</a></span>). On the contrary of codon effects, the amino acid compositions should suffer the translation selection independently.</p><p>However, more and more researchers found that amino acid composition is under more translation selection than codon usage. Li <em>et al.</em> firstly reported that the codons decoded by rare transfer RNAs did not lead to slow translation under nutrient-rich conditions.<a reveal-id="dsx030-B17" data-open="dsx030-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Then, Subramaniam <em>et al.</em> showed that protein synthesis rates were highly similar across yellow fluorescent protein variants during amino acid-rich growth, while some synonymous codons were highly sensitive to environmental perturbation during limitation for corresponding amino acids.<a reveal-id="dsx030-B53" data-open="dsx030-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Zhou <em>et al.</em><a reveal-id="dsx030-B54" data-open="dsx030-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> lately reported that impacts of codon usage are mainly due to their effects on transcription and largely independent of translation. Williford and Demuth compared several expression related factors and introduced that highly expressed genes have stronger selection for amino acid composition than codon usage.<a reveal-id="dsx030-B55" data-open="dsx030-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> It is proved that the cost minimization principle causes the amino acid relative abundances to connect with metabolic cost.<a reveal-id="dsx030-B56" data-open="dsx030-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> In contrast to the consistency between the local effects of synonymous codon usage and tRNA gene copy number on translation rates,<a reveal-id="dsx030-B17" data-open="dsx030-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx030-B52" data-open="dsx030-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> the TAAI is a translation rate associated factors with universal and global effects.</p>                            <h3 scrollto-destination=104086597 id="104086597" class="section-title" >4.2. Energy efficiency and selective pressure for co-adaption</h3>
<p>Co-adaption arises from energy efficiency and selective pressure. Organisms evolve to maximize efficiency and minimize energy cost by adapting through genetic mechanisms.<a reveal-id="dsx030-B40" data-open="dsx030-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx030-B49" data-open="dsx030-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Such global co-adaption might raise the translation efficiency globally, coinciding with the energy efficiency/ecological dynamics principle.<a reveal-id="dsx030-B50" data-open="dsx030-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Previously, Higgs and Ran analyzed 80 bacterial genomes and found that tRNA gene copy numbers evolved in response to translational selection.<a reveal-id="dsx030-B57" data-open="dsx030-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> It is notable that they observed consistency between synonymous codon usage and tRNA gene copy number and that the unequal usage of synonymous codons encoding the same amino acid was involved. However, our current study on co-adaption focuses on disequilibrium frequencies among the twenty standard amino acids. Here, the co-adaption reflects a balance between tRNA gene copy number and the amino acids needed by the proteome. Redundant excessive tRNA gene copies will ultimately be a waste of translation resources (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>t</mi><mi>R</mi><mi>N</mi><msubsup><mi>A</mi><mrow><mi>G</mi><mi>A</mi><mi>T</mi></mrow><mrow><mi>I</mi><mi>l</mi><mi>e</mi></mrow></msubsup></mrow></math></span> did not increase the EGFP production in <em>E. coli</em>). Selective pressure drives the co-adaption at the protein level. Experiments showed that increasing the TAAI could indeed improve the translation speed of the proteins and hence validate that the co-adaption is caused directly by translation pressure. These results indicate that translation selection causes co-variation at the scale of organisms and individual proteins.</p><p>If we expand our view to the domains of life, which have evolutionary connections<a reveal-id="dsx030-B58" data-open="dsx030-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsx030-B59" data-open="dsx030-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a>, such selection also exists. We found that eukaryotic genomes had much better adaption values than the other two domains (<a reveal-id="dsx030-F5" data-open="dsx030-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The translation rate for eukaryotic genomes is ∼3–8 amino acids per second,<a reveal-id="dsx030-B60" data-open="dsx030-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> compared to 10–20 amino acids per second for bacterial genomes.<a reveal-id="dsx030-B61" data-open="dsx030-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> In contrast, eukaryotes have much larger genomes and, hence, many more proteins. Thus, eukaryotic proteins would undergo stricter translation selective pressure. This higher pressure may be one of the reasons for the higher co-adaption observed in eukaryotes. Focusing on the bacterial domain, we observed that larger bacterial genomes tended to have higher TAAIs, and the TAAI value correlated positively with the genome size. Higher selective pressure may be the reason for this positive correlation. Taking all of the results into account, co-adaption is one effect of translation selection at all three levels (domain, genome and protein) and the selection complies with the maximum efficiency and minimum cost principle. </p>                        <a id="104086600" scrollto-destination="104086600"></a>
<div id="dsx030-F5" data-id="dsx030-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f5.png?Expires=2147483647&Signature=DlMhIE8l9drjSVtXhAOc27gPiyF~rb4dbwPrHX6Tvnk15Np7H8ShvcpcWED398TUyGsmnV~HucNThY9fp1~aHSsGCOQ6DmuOA~FQNbY9fDoN4yE4rrbFvWUCeTBBW4LWcUcvBaHavNh2TX4v-lARxIeW4xpx0WlzihCQJn2io3O8JKALlaqoVZknhnBqZeyF0n7E9howBkzj3knBvOCF1FENjK91KRQq~QNztXQdEfrFs5Q-7PbQ3GUdfxZuiQsfYYUWGbRcDnERZdeS2WUu54LmYSxz5FoJHrXstmww8F28U5vOYVods02mAthlSmwtWtyqNHcs~2w0GgfO5U9CFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the co-adaption (TAAI) of three domains. The medians for Archaea, Bacteria and Eukarya were 0.37, 0.61 and 0.69. Archaea had relatively low TAAIs, and Eukarya had the highest. Analysis of variance revealed that the three domains were significantly different, with a P value close to zero (P = 2.57e−06)." path-from-xml="dsx030f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086600/dsx030f5.tif" path-from-xml="dsx030f5.tif" target="_blank">View large</a><a section="104086600" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f5.png?Expires=2147483647&Signature=y67zqK9UDu4NBMptfsN9OiyqfbLfPPoaahvq1Efb8DrsC7~sM0MaDZYzSvhjyvipnyK-h1Fyy45eGI14caY7-rHiq8fuNZk10GB-lGmcJRViCKC74-kZ9b9GP5~zV5zPPdoN-AohDAeGyxlCptMGnXfS~jy52P7o267kjFSwmA7KrbhpEnUy6PqXIzOhxljYCCxrjRNoOqoCPiSbCcuKkiGFB6S1r7mwOaFq0P1wm2HRRsOdU544RjhhdJhzgPbhwTUKtjkmn6J38YVjp4SD9FUjAa6-cdoOMzxxqJy7Q85rNy7KW09wWL7bkxSTHxT-hEn9BQSvNMvFj1bPSprlsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086600&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the co-adaption (TAAI) of three domains. The medians for Archaea, Bacteria and Eukarya were 0.37, 0.61 and 0.69. Archaea had relatively low TAAIs, and Eukarya had the highest. Analysis of variance revealed that the three domains were significantly different, with a <em>P</em> value close to zero (<em>P</em> = 2.57e−06).</p></div></div><div content-id="dsx030-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/m_dsx030f5.png?Expires=2147483647&Signature=DlMhIE8l9drjSVtXhAOc27gPiyF~rb4dbwPrHX6Tvnk15Np7H8ShvcpcWED398TUyGsmnV~HucNThY9fp1~aHSsGCOQ6DmuOA~FQNbY9fDoN4yE4rrbFvWUCeTBBW4LWcUcvBaHavNh2TX4v-lARxIeW4xpx0WlzihCQJn2io3O8JKALlaqoVZknhnBqZeyF0n7E9howBkzj3knBvOCF1FENjK91KRQq~QNztXQdEfrFs5Q-7PbQ3GUdfxZuiQsfYYUWGbRcDnERZdeS2WUu54LmYSxz5FoJHrXstmww8F28U5vOYVods02mAthlSmwtWtyqNHcs~2w0GgfO5U9CFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the co-adaption (TAAI) of three domains. The medians for Archaea, Bacteria and Eukarya were 0.37, 0.61 and 0.69. Archaea had relatively low TAAIs, and Eukarya had the highest. Analysis of variance revealed that the three domains were significantly different, with a P value close to zero (P = 2.57e−06)." path-from-xml="dsx030f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086600/dsx030f5.tif" path-from-xml="dsx030f5.tif" target="_blank">View large</a><a section="104086600" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/dsx030f5.png?Expires=2147483647&Signature=y67zqK9UDu4NBMptfsN9OiyqfbLfPPoaahvq1Efb8DrsC7~sM0MaDZYzSvhjyvipnyK-h1Fyy45eGI14caY7-rHiq8fuNZk10GB-lGmcJRViCKC74-kZ9b9GP5~zV5zPPdoN-AohDAeGyxlCptMGnXfS~jy52P7o267kjFSwmA7KrbhpEnUy6PqXIzOhxljYCCxrjRNoOqoCPiSbCcuKkiGFB6S1r7mwOaFq0P1wm2HRRsOdU544RjhhdJhzgPbhwTUKtjkmn6J38YVjp4SD9FUjAa6-cdoOMzxxqJy7Q85rNy7KW09wWL7bkxSTHxT-hEn9BQSvNMvFj1bPSprlsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086600&ar=3953933&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx030f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the co-adaption (TAAI) of three domains. The medians for Archaea, Bacteria and Eukarya were 0.37, 0.61 and 0.69. Archaea had relatively low TAAIs, and Eukarya had the highest. Analysis of variance revealed that the three domains were significantly different, with a <em>P</em> value close to zero (<em>P</em> = 2.57e−06).</p></div></div>                            <h3 scrollto-destination=104086601 id="104086601" class="section-title" >4.3. Potential application in lifting translation rate and hence protein production</h3>
<p>This co-adaption can be applied to enhance translation efficiency in practice. Traditionally, in industrial application, the yield of a specific protein is improved by optimizing its synonymous codon usage.<a reveal-id="dsx030-B62" data-open="dsx030-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> A higher ratio of optimal codons could facilitate the transcription efficiency by frequent usage of abundant or efficient tRNAs.<a reveal-id="dsx030-B14" data-open="dsx030-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Here, the yield of EGFP was improved markedly in <em>E. coli</em> by optimizing TAAI through increasing the gene copy number of specific tRNAs, thus increasing the translation speed at least tenfold. This finding may be applied in industrial production. To obtain higher output of one protein, we could optimize its co-adaption between tRNA gene copy number and amino acid usage by adding specific tRNA gene copies. Thus, the protein’s translation could be accelerated quickly. One of the prominent advantages of such an operation is that the yields of multiple proteins could be improved in one round. The production of multiple proteins could be increased by adding specific tRNA gene copies corresponding to their amino acid usage. This ideal result is based on the supposition that adding a specific tRNA gene could increase the TAAIs of many proteins simultaneously. A more practical method would be to divide all target proteins into groups based on similar amino acid frequencies. If the tRNA genes to import are carefully chosen, the target group of proteins will have higher expression levels but the other proteins should remain almost unchanged.</p><p>In the field of synthetic biology, it is hoped to devise and construct a general bacterial chassis cell that integrates functional synthetic parts, devices and systems. In practice, such a chassis has often been constructed or synthesized based on small and slowly growing bacteria.<a reveal-id="dsx030-B63" data-open="dsx030-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsx030-B64" data-open="dsx030-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> However, slow growth may limit their capacity to produce enough target molecules in a short time. Our strategy of importing certain tRNA genes may help to address this problem when designing chassis cells.</p>                            <h2 scrollto-destination=104086604 id="104086604" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Mr. Lu-Wen Ning for suggestions, Juan Feng’s lab and Lixia Tang’s lab for providing help with experimental materials and methods, and our lab members for fruitful discussion.</p>                            <h3 scrollto-destination=104086606 id="104086606" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig1_dsx030.png?Expires=2147483647&Signature=3OZxq~SIXi7yuOp1eA~jzW3LxGVmA4YwRY9MeNCEFUDnBywaP~R2tEb8U2QEXNJ1T9CF0PGo4ZM7wpbwGeytKrYZqykK~zv-18xKko-Rn8B4SYGKzLHXDormcl3z1aGYIFpZ3F8N-v2GPjATsICkYaKlXMxyFxreCQzM99QyiOb74dUbTqTsmO9cnOt9WHW~j3TejjNZ66bI-ESSrjFtfcaXCJGB0Oj6Pk8t2g54ly2TJtJ8KKA5mjJSfkBFlhNG0lJsISKtUGQBAtyP3UI6L7rkvOx3jfSrVzBenIPRz7lF0DZGDfA0DGxdOLIxNlLEiM5~sVkB~dM~3Hu5BqyioQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=104086608 id="104086608" class="section-title" >Funding</h3>
<p>This work was supported by the National Natural Science Foundation of China (31501063) and the Fundamental Research Funds for the Central Universities of China (ZYGX2016J117, ZYGX2015J144).</p>                            <h3 scrollto-destination=104086610 id="104086610" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=104086612 id="104086612" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx030-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gingold</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Pilpel</div> <div class="given-names">Y.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Determinants of translation efficiency and accuracy,</div> <div class="source">Mol. Sys. Biol</div> ., <div class="volume">7</div>, <div class="fpage">481</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determinants%20of%20translation%20efficiency%20and%20accuracy%2C&amp;author=H.%20Gingold&amp;author=Y.%20Pilpel&amp;publication_year=2011&amp;journal=Mol.%20Sys.%20Biol&amp;volume=7&amp;pages=481" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2011.14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2011.14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2011.14"> </span></div> </div></div></div></div></div><div content-id="dsx030-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.-R.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Codon-by-codon modulation of translational speed and accuracy via mRNA folding,</div> <div class="source">PLoS Biol</div> ., <div class="volume">12</div>, <div class="fpage">e1001910</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon-by-codon%20modulation%20of%20translational%20speed%20and%20accuracy%20via%20mRNA%20folding%2C&amp;author=J.-R.%20Yang&amp;author=X.%20Chen&amp;author=J.%20Zhang&amp;publication_year=2014&amp;journal=PLoS%20Biol&amp;volume=12&amp;pages=e1001910" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1001910" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1001910" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1001910"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25051069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dekel</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Alon</div> <div class="given-names">U.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Optimality and evolutionary tuning of the expression level of a protein,</div> <div class="source">Nature</div> , <div class="volume">436</div>, <div class="fpage">588</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimality%20and%20evolutionary%20tuning%20of%20the%20expression%20level%20of%20a%20protein%2C&amp;author=E.%20Dekel&amp;author=U.%20Alon&amp;publication_year=2005&amp;journal=Nature&amp;volume=436&amp;pages=588-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16049495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.-H.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">15</div>, <div class="fpage">1281</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20adaptation%20index-a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications%2C&amp;author=P.%20M.%20Sharp&amp;author=W.-H.%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res&amp;volume=15&amp;pages=1281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">dos Reis</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Savva</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Solving the riddle of codon usage preferences: a test for translational selection,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">5036</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solving%20the%20riddle%20of%20codon%20usage%20preferences%3A%20a%20test%20for%20translational%20selection%2C&amp;author=M.%20dos%20Reis&amp;author=R.%20Savva&amp;author=L.%20Wernisch&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=5036-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ingolia</div> <div class="given-names">N.T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Ribosome footprint profiling of translation throughout the genome,</div> <div class="source">Cell</div> , <div class="volume">165</div>, <div class="fpage">22</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosome%20footprint%20profiling%20of%20translation%20throughout%20the%20genome%2C&amp;author=N.T.%20Ingolia&amp;publication_year=2016&amp;journal=Cell&amp;volume=165&amp;pages=22-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2016.02.066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2016.02.066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2016.02.066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27015305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hoek</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Vale</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Tanenbaum</div> <div class="given-names">M.E.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Dynamics of translation of single mRNA molecules in vivo,</div> <div class="source">Cell</div> , <div class="volume">165</div>, <div class="fpage">976</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamics%20of%20translation%20of%20single%20mRNA%20molecules%20in%20vivo%2C&amp;author=X.%20Yan&amp;author=T.A.%20Hoek&amp;author=R.D.%20Vale&amp;author=M.E.%20Tanenbaum&amp;publication_year=2016&amp;journal=Cell&amp;volume=165&amp;pages=976-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2016.04.034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2016.04.034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2016.04.034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27153498" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fredrick</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ibba</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">How the sequence of a gene can tune its translation,</div> <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">227</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20the%20sequence%20of%20a%20gene%20can%20tune%20its%20translation%2C&amp;author=K.%20Fredrick&amp;author=M.%20Ibba&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=227-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Carmi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vestsigian</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation,</div> <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">344</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation%2C&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gamble</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Brule</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Fields</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Grayhack</div> <div class="given-names">E.J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Adjacent codons act in concert to modulate translation efficiency in yeast,</div> <div class="source">Cell</div> , <div class="volume">166</div>, <div class="fpage">679</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adjacent%20codons%20act%20in%20concert%20to%20modulate%20translation%20efficiency%20in%20yeast%2C&amp;author=C.%20E.%20Gamble&amp;author=C.%20E.%20Brule&amp;author=K.M.%20Dean&amp;author=S.%20Fields&amp;author=E.J.%20Grayhack&amp;publication_year=2016&amp;journal=Cell&amp;volume=166&amp;pages=679-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2016.05.070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2016.05.070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2016.05.070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27374328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elf</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Nilsson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tenson</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ehrenberg</div> <div class="given-names">M.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Selective charging of tRNA isoacceptors explains patterns of codon usage,</div> <div class="source">Science</div> , <div class="volume">300</div>, <div class="fpage">1718</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20charging%20of%20tRNA%20isoacceptors%20explains%20patterns%20of%20codon%20usage%2C&amp;author=J.%20Elf&amp;author=D.%20Nilsson&amp;author=T.%20Tenson&amp;author=M.%20Ehrenberg&amp;publication_year=2003&amp;journal=Science&amp;volume=300&amp;pages=1718-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1083811" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1083811" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1083811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12805541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chan</div> <div class="given-names">P.P.</div></div>, <div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">GtRNAdb: a database of transfer RNA genes detected in genomic sequence,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">D93</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GtRNAdb%3A%20a%20database%20of%20transfer%20RNA%20genes%20detected%20in%20genomic%20sequence%2C&amp;author=P.P.%20Chan&amp;author=T.M.%20Lowe&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=D93-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn787" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn787" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn787"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18984615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nedialkova</div> <div class="given-names">D.D.</div></div>, <div class="name string-name"><div class="surname">Leidel</div> <div class="given-names">S.A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Optimization of codon translation rates via tRNA modifications maintains proteome integrity,</div> <div class="source">Cell</div> , <div class="volume">161</div>, <div class="fpage">1606</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20of%20codon%20translation%20rates%20via%20tRNA%20modifications%20maintains%20proteome%20integrity%2C&amp;author=D.D.%20Nedialkova&amp;author=S.A.%20Leidel&amp;publication_year=2015&amp;journal=Cell&amp;volume=161&amp;pages=1606-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.05.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.05.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.05.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26052047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cannarozzi</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Schraudolph</div> <div class="given-names">N. N.</div></div>, <div class="name string-name"><div class="surname">Faty</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">A role for codon order in translation dynamics,</div> <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">355</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20codon%20order%20in%20translation%20dynamics%2C&amp;author=G.%20Cannarozzi&amp;author=N.%20N.%20Schraudolph&amp;author=M.%20Faty&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=355-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.02.036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.02.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.02.036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dittmar</div> <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">Goodenbour</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">T.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Tissue-specific differences in human transfer RNA expression,</div> <div class="source">PLoS Genet</div> ., <div class="volume">2</div>, <div class="fpage">e221</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tissue-specific%20differences%20in%20human%20transfer%20RNA%20expression%2C&amp;author=K.A.%20Dittmar&amp;author=J.M.%20Goodenbour&amp;author=T.%20Pan&amp;publication_year=2006&amp;journal=PLoS%20Genet&amp;volume=2&amp;pages=e221" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17194224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wen</div> <div class="given-names">J.-D.</div></div>, <div class="name string-name"><div class="surname">Lancaster</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Noller</div> <div class="given-names">H.F.</div></div>, <div class="name string-name"><div class="surname">Bustamante</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Tinoco</div> <div class="given-names">I.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The ribosome uses two active mechanisms to unwind messenger RNA during translation,</div> <div class="source">Nature</div> , <div class="volume">475</div>, <div class="fpage">118</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ribosome%20uses%20two%20active%20mechanisms%20to%20unwind%20messenger%20RNA%20during%20translation%2C&amp;author=X.%20Qu&amp;author=J.-D.%20Wen&amp;author=L.%20Lancaster&amp;author=H.F.%20Noller&amp;author=C.%20Bustamante&amp;author=I.%20Tinoco&amp;publication_year=2011&amp;journal=Nature&amp;volume=475&amp;pages=118-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21734708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">G.-W.</div></div>, <div class="name string-name"><div class="surname">Oh</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J.S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria,</div> <div class="source">Nature</div> , <div class="volume">484</div>, <div class="fpage">538</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20anti-Shine-Dalgarno%20sequence%20drives%20translational%20pausing%20and%20codon%20choice%20in%20bacteria%2C&amp;author=G.-W.%20Li&amp;author=E.%20Oh&amp;author=J.S.%20Weissman&amp;publication_year=2012&amp;journal=Nature&amp;volume=484&amp;pages=538-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10965" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10965" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10965"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22456704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shah</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Niemczyk</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Rate-limiting steps in yeast protein translation,</div> <div class="source">Cell</div> , <div class="volume">153</div>, <div class="fpage">1589</div>–<div class="lpage">1601</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rate-limiting%20steps%20in%20yeast%20protein%20translation%2C&amp;author=P.%20Shah&amp;author=Y.%20Ding&amp;author=M.%20Niemczyk&amp;author=G.%20Kudla&amp;author=J.B.%20Plotkin&amp;publication_year=2013&amp;journal=Cell&amp;volume=153&amp;pages=1589-1601" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.05.049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.05.049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.05.049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23791185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.-H.</div></div>, <div class="name string-name"><div class="surname">Dang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Codon usage influences the local rate of translation elongation to regulate co-translational protein folding,</div> <div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">744</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20influences%20the%20local%20rate%20of%20translation%20elongation%20to%20regulate%20co-translational%20protein%20folding%2C&amp;author=C.-H.%20Yu&amp;author=Y.%20Dang&amp;author=Z.%20Zhou&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=744-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.07.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.07.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.07.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26321254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Waldman</div> <div class="given-names">Y.Y.</div></div>, <div class="name string-name"><div class="surname">Kupiec</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ruppin</div> <div class="given-names">E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Translation efficiency is determined by both codon bias and folding energy,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">3645</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20efficiency%20is%20determined%20by%20both%20codon%20bias%20and%20folding%20energy%2C&amp;author=T.%20Tuller&amp;author=Y.Y.%20Waldman&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=3645-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909910107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909910107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909910107"> </span></div> </div></div></div></div></div><div content-id="dsx030-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vargas-Rodriguez</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Musier-Forsyth</div> <div class="given-names">K.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Structural biology: Wobble puts RNA on target,</div> <div class="source">Nature</div> , <div class="volume">510</div>, <div class="fpage">480</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20biology%3A%20Wobble%20puts%20RNA%20on%20target%2C&amp;author=O.%20Vargas-Rodriguez&amp;author=K.%20Musier-Forsyth&amp;publication_year=2014&amp;journal=Nature&amp;volume=510&amp;pages=480-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24919145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Subramaniam</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Zid</div> <div class="given-names">B.M.</div></div>, <div class="name string-name"><div class="surname">O’Shea</div> <div class="given-names">E.K.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">An integrated approach reveals regulatory controls on bacterial translation elongation,</div> <div class="source">Cell</div> , <div class="volume">159</div>, <div class="fpage">1200</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20approach%20reveals%20regulatory%20controls%20on%20bacterial%20translation%20elongation%2C&amp;author=A.R.%20Subramaniam&amp;author=B.M.%20Zid&amp;author=E.K.%20O%E2%80%99Shea&amp;publication_year=2014&amp;journal=Cell&amp;volume=159&amp;pages=1200-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.10.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.10.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.10.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25416955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rozov</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Demeshkina</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Westhof</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Yusupov</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yusupova</div> <div class="given-names">G.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">New structural insights into translational miscoding</div>. <div class="source">Trends Biochemi. Sci</div> ., <div class="volume">41</div>, <div class="fpage">798</div>–<div class="lpage">814</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20structural%20insights%20into%20translational%20miscoding&amp;author=A.%20Rozov&amp;author=N.%20Demeshkina&amp;author=E.%20Westhof&amp;author=M.%20Yusupov&amp;author=G.%20Yusupova&amp;publication_year=2016&amp;journal=Trends%20Biochemi.%20Sci&amp;volume=41&amp;pages=798-814" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibs.2016.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibs.2016.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibs.2016.06.001"> </span></div> </div></div></div></div></div><div content-id="dsx030-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Espah Borujeni</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Salis</div> <div class="given-names">H.M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Translation initiation is controlled by RNA folding kinetics via a ribosome drafting mechanism</div>, <div class="source">J. Am. Chem. Soc</div> , <div class="volume">138</div>, <div class="fpage">7016</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20initiation%20is%20controlled%20by%20RNA%20folding%20kinetics%20via%20a%20ribosome%20drafting%20mechanism&amp;author=A.%20Espah%20Borujeni&amp;author=H.M.%20Salis&amp;publication_year=2016&amp;journal=J.%20Am.%20Chem.%20Soc&amp;volume=138&amp;pages=7016" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jacs.6b01453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjacs.6b01453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjacs.6b01453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27199273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Eliscovich</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Yoon</div> <div class="given-names">Y.J.</div></div>, <div class="name string-name"><div class="surname">Singer</div> <div class="given-names">R.H.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Translation dynamics of single mRNAs in live cells and neurons</div>, <div class="source">Science</div> , <div class="volume">352</div>, <div class="fpage">1430</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20dynamics%20of%20single%20mRNAs%20in%20live%20cells%20and%20neurons&amp;author=B.%20Wu&amp;author=C.%20Eliscovich&amp;author=Y.J.%20Yoon&amp;author=R.H.%20Singer&amp;publication_year=2016&amp;journal=Science&amp;volume=352&amp;pages=1430" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aaf1084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aaf1084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aaf1084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27313041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mandel</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Silhavy</div> <div class="given-names">T.J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Starvation for different nutrients in Escherichia coli results in differential modulation of RpoS levels and stability,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">187</div>, <div class="fpage">434</div>–<div class="lpage">442</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Starvation%20for%20different%20nutrients%20in%20Escherichia%20coli%20results%20in%20differential%20modulation%20of%20RpoS%20levels%20and%20stability%2C&amp;author=M.J.%20Mandel&amp;author=T.J.%20Silhavy&amp;publication_year=2005&amp;journal=J.%20Bacteriol&amp;volume=187&amp;pages=434-442" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.2.434-442.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.2.434-442.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.2.434-442.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ausubel</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Brent</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kingston</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">1987</div>, <div class="source">Current protocols in molecular biology New York</div> , 
<div class="publisher-loc">New York</div>: 
<div class="publisher-name">Wiley</div>.</div></div></div></div><div content-id="dsx030-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benson</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Cavanaugh</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">GenBank,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D36</div>–<div class="lpage">D42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenBank%2C&amp;author=D.A.%20Benson&amp;author=M.%20Cavanaugh&amp;author=K.%20Clark&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D36-D42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jühling</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Mörl</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hartmann</div> <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Sprinzl</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stadler</div> <div class="given-names">P.F.</div></div>, <div class="name string-name"><div class="surname">Pütz</div> <div class="given-names">J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">tRNAdb 2009: compilation of tRNA sequences and tRNA genes,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">D159</div>–<div class="lpage">D162</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAdb%202009%3A%20compilation%20of%20tRNA%20sequences%20and%20tRNA%20genes%2C&amp;author=F.%20J%C3%BChling&amp;author=M.%20M%C3%B6rl&amp;author=R.K.%20Hartmann&amp;author=M.%20Sprinzl&amp;author=P.F.%20Stadler&amp;author=J.%20P%C3%BCtz&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=D159-D162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn772" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn772" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn772"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18957446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Weiss</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Simonovic</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">PaxDb, a database of protein abundance averages across all three domains of life,</div> <div class="source">Mol. Cell. Proteomics</div> , <div class="volume">11</div>, <div class="fpage">492</div>–<div class="lpage">500</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PaxDb%2C%20a%20database%20of%20protein%20abundance%20averages%20across%20all%20three%20domains%20of%20life%2C&amp;author=M.%20Wang&amp;author=M.%20Weiss&amp;author=M.%20Simonovic&amp;publication_year=2012&amp;journal=Mol.%20Cell.%20Proteomics&amp;volume=11&amp;pages=492-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/mcp.O111.014704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fmcp.O111.014704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fmcp.O111.014704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22535208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cui</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Translational selection in human: more pronounced in housekeeping genes,</div> <div class="source">Biol. Direct</div> , <div class="volume">9</div>, <div class="fpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20selection%20in%20human%3A%20more%20pronounced%20in%20housekeeping%20genes%2C&amp;author=L.%20Ma&amp;author=P.%20Cui&amp;author=J.%20Zhu&amp;author=Z.%20Zhang&amp;author=Z.%20Zhang&amp;publication_year=2014&amp;journal=Biol.%20Direct&amp;volume=9&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-9-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-9-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-9-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25011537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yin</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Old genes experience stronger translational selection than young genes,</div> <div class="source">Gene</div> , <div class="volume">590</div>, <div class="fpage">29</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Old%20genes%20experience%20stronger%20translational%20selection%20than%20young%20genes%2C&amp;author=H.%20Yin&amp;author=L.%20Ma&amp;author=G.%20Wang&amp;author=M.%20Li&amp;author=Z.%20Zhang&amp;publication_year=2016&amp;journal=Gene&amp;volume=590&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2016.05.041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2016.05.041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2016.05.041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27259662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wei</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">Y. T.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">M. Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Rao</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">F. B.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genomic complexity places less restrictions on the evolution of young coexpression networks than protein–protein interactions,</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">8</div>, <div class="fpage">2624</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20complexity%20places%20less%20restrictions%20on%20the%20evolution%20of%20young%20coexpression%20networks%20than%20protein%E2%80%93protein%20interactions%2C&amp;author=W.%20Wei&amp;author=Y.%20T.%20Jin&amp;author=M.%20Z.%20Du&amp;author=J.%20Wang&amp;author=N.%20Rao&amp;author=F.%20B.%20Guo&amp;publication_year=2016&amp;journal=Genome%20Biol.%20Evol&amp;volume=8&amp;pages=2624-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evw198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevw198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevw198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27521813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garciavallve</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Guzman</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Montero</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Romeu</div> <div class="given-names">A.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">187</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HGT-DB%3A%20a%20database%20of%20putative%20horizontally%20transferred%20genes%20in%20prokaryotic%20complete%20genomes&amp;author=S.%20Garciavallve&amp;author=E.%20Guzman&amp;author=M.A.%20Montero&amp;author=A.%20Romeu&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=187-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rocha</div> <div class="given-names">E.P.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization,</div> <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">2279</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20from%20tRNA's%20point%20of%20view%3A%20redundancy%2C%20specialization%2C%20and%20efficient%20decoding%20for%20translation%20optimization%2C&amp;author=E.P.%20Rocha&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=2279-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2896904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2896904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2896904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15479947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamao</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Andachi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Muto</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Osawa</div> <div class="given-names">S.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Levels of tRNAs in bacterial cells as affected by amino acid usage in proteins,</div> <div class="source">Nucleic acids Res</div> ., <div class="volume">19</div>, <div class="fpage">6119</div>–<div class="lpage">6122</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Levels%20of%20tRNAs%20in%20bacterial%20cells%20as%20affected%20by%20amino%20acid%20usage%20in%20proteins%2C&amp;author=F.%20Yamao&amp;author=Y.%20Andachi&amp;author=A.%20Muto&amp;author=T.%20Ikemura&amp;author=S.%20Osawa&amp;publication_year=1991&amp;journal=Nucleic%20acids%20Res&amp;volume=19&amp;pages=6119-6122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/19.22.6119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F19.22.6119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F19.22.6119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1956771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Levitt</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Nature of the protein universe,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">106</div>, <div class="fpage">11079</div>–<div class="lpage">11084</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nature%20of%20the%20protein%20universe%2C&amp;author=M.%20Levitt&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=11079-11084" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905029106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905029106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905029106"> </span></div> </div></div></div></div></div><div content-id="dsx030-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kempes</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Dutkiewicz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Follows</div> <div class="given-names">M.J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Growth, metabolic partitioning, and the size of microorganisms,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">495</div>–<div class="lpage">500</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Growth%2C%20metabolic%20partitioning%2C%20and%20the%20size%20of%20microorganisms%2C&amp;author=C.P.%20Kempes&amp;author=S.%20Dutkiewicz&amp;author=M.J.%20Follows&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=495-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1115585109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1115585109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1115585109"> </span></div> </div></div></div></div></div><div content-id="dsx030-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kempes</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Amend</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hoehler</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Evolutionary tradeoffs in cellular composition across diverse bacteria,</div> <div class="source">ISME J</div> ., <div class="volume">1010</div>, <div class="fpage">21452145</div>–<div class="lpage">21572157</div></div></div></div></div><div content-id="dsx030-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dana</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The effect of tRNA levels on decoding times of mRNA codons,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">9171</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20tRNA%20levels%20on%20decoding%20times%20of%20mRNA%20codons%2C&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=9171-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25056313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jost</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Hillis</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kyle</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Fozzard</div> <div class="given-names">H.A.</div></div>, <div class="name string-name"><div class="surname">Zakon</div> <div class="given-names">H.H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Toxin-resistant sodium channels: parallel adaptive evolution across a complete gene family,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">1016</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toxin-resistant%20sodium%20channels%3A%20parallel%20adaptive%20evolution%20across%20a%20complete%20gene%20family%2C&amp;author=M.C.%20Jost&amp;author=D.M.%20Hillis&amp;author=Y.%20Lu&amp;author=J.W.%20Kyle&amp;author=H.A.%20Fozzard&amp;author=H.H.%20Zakon&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol&amp;volume=25&amp;pages=1016-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18258611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hale</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">de Boer</div> <div class="given-names">P.A.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Direct binding of FtsZ to ZipA, an essential component of the septal ring structure that mediates cell division in E. coli</div>. <div class="source">Cell</div> , <div class="volume">88</div>, <div class="fpage">175</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20binding%20of%20FtsZ%20to%20ZipA%2C%20an%20essential%20component%20of%20the%20septal%20ring%20structure%20that%20mediates%20cell%20division%20in%20E.%20coli&amp;author=C.A.%20Hale&amp;author=P.A.%20de%20Boer&amp;publication_year=1997&amp;journal=Cell&amp;volume=88&amp;pages=175-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)81838-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)81838-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)81838-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9008158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Stewart</div> <div class="given-names">R.C.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">The two-component system,</div> <div class="source">Regulation of diverse signaling pathways in prokaryotes and eukaryotes. Plant Physiol</div> ., <div class="volume">117</div>, <div class="fpage">723</div>–<div class="lpage">31</div>.</div></div></div></div><div content-id="dsx030-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kunkel</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Erie</div> <div class="given-names">D.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">DNA mismatch repair,</div> <div class="source">Annu. Rev. Biochem</div> ., <div class="volume">74</div>, <div class="fpage">681</div>–<div class="lpage">710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20mismatch%20repair%2C&amp;author=T.A.%20Kunkel&amp;author=D.A.%20Erie&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=74&amp;pages=681-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biochem.74.082803.133243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biochem.74.082803.133243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biochem.74.082803.133243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15952900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Titgemeyer</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Hillen</div> <div class="given-names">W.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Global control of sugar metabolism: a gram-positive solution</div>. <div class="source">Antonie Van Leeuwenhoek</div> , <div class="volume">82</div>, <div class="fpage">59</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20control%20of%20sugar%20metabolism%3A%20a%20gram-positive%20solution&amp;author=F.%20Titgemeyer&amp;author=W.%20Hillen&amp;publication_year=2002&amp;journal=Antonie%20Van%20Leeuwenhoek&amp;volume=82&amp;pages=59-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1020628909429" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1020628909429" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1020628909429"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12369205" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ingolia</div> <div class="given-names">N.T.</div></div>, <div class="name string-name"><div class="surname">Ghaemmaghami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Newman</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J.S.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling,</div> <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">218</div>–<div class="lpage">223</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20in%20vivo%20of%20translation%20with%20nucleotide%20resolution%20using%20ribosome%20profiling%2C&amp;author=N.T.%20Ingolia&amp;author=S.%20Ghaemmaghami&amp;author=J.R.%20Newman&amp;author=J.S.%20Weissman&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=218-223" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1168978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1168978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1168978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19213877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Acosta-Rivero</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Morales</div> <div class="given-names">J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Improvement of human interferon HUIFNalpha2 and HCV core protein expression levels in Escherichia coli but not of HUIFNalpha8 by using the tRNA(AGA/AGG)</div>. <div class="source">Biochem. Biophys. Res. Commun</div> ., <div class="volume">296</div>, <div class="fpage">1303</div>–<div class="lpage">1309</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvement%20of%20human%20interferon%20HUIFNalpha2%20and%20HCV%20core%20protein%20expression%20levels%20in%20Escherichia%20coli%20but%20not%20of%20HUIFNalpha8%20by%20using%20the%20tRNA(AGA%2FAGG)&amp;author=N.%20Acosta-Rivero&amp;author=J.C.%20Sanchez&amp;author=J.%20Morales&amp;publication_year=2002&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=296&amp;pages=1303-1309" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0006-291X(02)02056-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0006-291X(02)02056-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0006-291X(02)02056-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12207916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maitra</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dill</div> <div class="given-names">K.A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Bacterial growth laws reflect the evolutionary importance of energy efficiency,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">406</div>–<div class="lpage">411</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20growth%20laws%20reflect%20the%20evolutionary%20importance%20of%20energy%20efficiency%2C&amp;author=A.%20Maitra&amp;author=K.A.%20Dill&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=406-411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1421138111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1421138111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1421138111"> </span></div> </div></div></div></div></div><div content-id="dsx030-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grosskopf</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Consuegra</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gaffe</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Metabolic modelling in a dynamic evolutionary framework predicts adaptive diversification of bacteria in a long-term evolution experiment</div>, <div class="source">BMC Evol. Biol</div> ., <div class="volume">16</div>, <div class="fpage">163</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20modelling%20in%20a%20dynamic%20evolutionary%20framework%20predicts%20adaptive%20diversification%20of%20bacteria%20in%20a%20long-term%20evolution%20experiment&amp;author=T.%20Grosskopf&amp;author=J.%20Consuegra&amp;author=J.%20Gaffe&amp;publication_year=2016&amp;journal=BMC%20Evol.%20Biol&amp;volume=16&amp;pages=163" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-016-0733-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-016-0733-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-016-0733-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27544664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quax</div> <div class="given-names">T.E.</div></div>, <div class="name string-name"><div class="surname">Claassens</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Söll</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">van der Oost</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Codon bias as a means to fine-tune gene expression,</div> <div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">149</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20as%20a%20means%20to%20fine-tune%20gene%20expression%2C&amp;author=T.E.%20Quax&amp;author=N.J.%20Claassens&amp;author=D.%20S%C3%B6ll&amp;author=J.%20van%20der%20Oost&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=149-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.05.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.05.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.05.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26186290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Duret</div> <div class="given-names">L.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">tRNA gene number and codon usage in the C. elegans genome are co-adapted for optimal translation of highly expressed genes,</div> <div class="source">Trends Genet</div> ., <div class="volume">16</div>, <div class="fpage">287</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%20gene%20number%20and%20codon%20usage%20in%20the%20C.%20elegans%20genome%20are%20co-adapted%20for%20optimal%20translation%20of%20highly%20expressed%20genes%2C&amp;author=L.%20Duret&amp;publication_year=2000&amp;journal=Trends%20Genet&amp;volume=16&amp;pages=287-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02041-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02041-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02041-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10858656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Subramaniam</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cluzel</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Environmental perturbations lift the degeneracy of the genetic code to regulate protein levels in bacteria</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">2419</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20perturbations%20lift%20the%20degeneracy%20of%20the%20genetic%20code%20to%20regulate%20protein%20levels%20in%20bacteria&amp;author=A.R.%20Subramaniam&amp;author=T.%20Pan&amp;author=P.%20Cluzel&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=2419-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1211077110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1211077110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1211077110"> </span></div> </div></div></div></div></div><div content-id="dsx030-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Dang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Codon usage is an important determinant of gene expression levels largely through its effects on transcription,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">113</div>, <div class="fpage">E6117</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20is%20an%20important%20determinant%20of%20gene%20expression%20levels%20largely%20through%20its%20effects%20on%20transcription%2C&amp;author=Z.%20Zhou&amp;author=Y.%20Dang&amp;author=M.%20Zhou&amp;publication_year=2016&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=113&amp;pages=E6117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1606724113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1606724113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1606724113"> </span></div> </div></div></div></div></div><div content-id="dsx030-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Williford</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Demuth</div> <div class="given-names">J. P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Gene expression levels are correlated with synonymous codon usage, amino acid composition, and gene architecture in the red flour beetle, Tribolium castaneum,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">3755</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20levels%20are%20correlated%20with%20synonymous%20codon%20usage%2C%20amino%20acid%20composition%2C%20and%20gene%20architecture%20in%20the%20red%20flour%20beetle%2C%20Tribolium%20castaneum%2C&amp;author=A.%20Williford&amp;author=J.%20P.%20Demuth&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=3755-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22826459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krick</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Verstraete</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Alonso</div> <div class="given-names">L.G.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Amino Acid metabolism conflicts with protein diversity,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">31</div>, <div class="fpage">2905</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amino%20Acid%20metabolism%20conflicts%20with%20protein%20diversity%2C&amp;author=T.%20Krick&amp;author=N.%20Verstraete&amp;author=L.G.%20Alonso&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol&amp;volume=31&amp;pages=2905-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25086000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Higgs</div> <div class="given-names">P.G.</div></div>, <div class="name string-name"><div class="surname">Ran</div> <div class="given-names">W.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Coevolution of codon usage and tRNA genes leads to alternative stable states of biased codon usage,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">2279</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coevolution%20of%20codon%20usage%20and%20tRNA%20genes%20leads%20to%20alternative%20stable%20states%20of%20biased%20codon%20usage%2C&amp;author=P.G.%20Higgs&amp;author=W.%20Ran&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol&amp;volume=25&amp;pages=2279-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18687657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Booth</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mariscal</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Doolittle</div> <div class="given-names">W.F.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The modern synthesis in the light of microbial genomics,</div> <div class="source">Annu. Rev. Microbiol</div> ., <div class="volume">70</div>, <div class="fpage">279</div>–<div class="lpage">297</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20modern%20synthesis%20in%20the%20light%20of%20microbial%20genomics%2C&amp;author=A.%20Booth&amp;author=C.%20Mariscal&amp;author=W.F.%20Doolittle&amp;publication_year=2016&amp;journal=Annu.%20Rev.%20Microbiol&amp;volume=70&amp;pages=279-297" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-micro-102215-095456" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-micro-102215-095456" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-micro-102215-095456"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27482743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Conery</div> <div class="given-names">J.S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">The origins of genome complexity,</div> <div class="source">Science</div> , <div class="volume">302</div>, <div class="fpage">1401</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origins%20of%20genome%20complexity%2C&amp;author=M.%20Lynch&amp;author=J.S.%20Conery&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=1401-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1089370" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1089370" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1089370"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14631042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mathews</div> <div class="given-names">M.B.</div></div>, <div class="name string-name"><div class="surname">Sonenberg</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Hershey</div> <div class="given-names">J.W.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Origins and principles of translational control</div>, <div class="source">Cold Spring Harbor Monogr. Arch</div> ., <div class="volume">39</div>, <div class="fpage">1</div>–<div class="lpage">31</div>.</div></div></div></div><div content-id="dsx030-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liang</div> <div class="given-names">S.-T.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Y.-C.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bremer</div> <div class="given-names">H.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">mRNA composition and control of bacterial gene expression,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">182</div>, <div class="fpage">3037</div>–<div class="lpage">3044</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mRNA%20composition%20and%20control%20of%20bacterial%20gene%20expression%2C&amp;author=S.-T.%20Liang&amp;author=Y.-C.%20Xu&amp;author=P.%20Dennis&amp;author=H.%20Bremer&amp;publication_year=2000&amp;journal=J.%20Bacteriol&amp;volume=182&amp;pages=3037-3044" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.11.3037-3044.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.11.3037-3044.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.11.3037-3044.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10809680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Menzella</div> <div class="given-names">H.G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Comparison of two codon optimization strategies to enhance recombinant protein production in Escherichia coli,</div> <div class="source">Microb. Cell Fact</div> ., <div class="volume">10</div>, <div class="issue">1</div>.</div></div></div></div><div content-id="dsx030-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hutchison</div> <div class="given-names">C.A.</div>3rd</div>, <div class="name string-name"><div class="surname">Chuang</div> <div class="given-names">R.Y.</div></div>, <div class="name string-name"><div class="surname">Noskov</div> <div class="given-names">V.N.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Design and synthesis of a minimal bacterial genome,</div> <div class="source">Science</div> , <div class="volume">351</div>, <div class="fpage">aad6253</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Design%20and%20synthesis%20of%20a%20minimal%20bacterial%20genome%2C&amp;author=C.A.%20Hutchison&amp;author=R.Y.%20Chuang&amp;author=V.N.%20Noskov&amp;publication_year=2016&amp;journal=Science&amp;volume=351&amp;pages=aad6253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aad6253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aad6253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aad6253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27013737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx030-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx030-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vickers</div> <div class="given-names">C.E.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The minimal genome comes of age,</div> <div class="source">Nat. Biotechnol</div> ., <div class="volume">34</div>, <div class="fpage">623</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20minimal%20genome%20comes%20of%20age%2C&amp;author=C.E.%20Vickers&amp;publication_year=2016&amp;journal=Nat.%20Biotechnol&amp;volume=34&amp;pages=623-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3593" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3593" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3593"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27281422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3953933&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig1_dsx030.png?Expires=2147483647&amp;Signature=3OZxq~SIXi7yuOp1eA~jzW3LxGVmA4YwRY9MeNCEFUDnBywaP~R2tEb8U2QEXNJ1T9CF0PGo4ZM7wpbwGeytKrYZqykK~zv-18xKko-Rn8B4SYGKzLHXDormcl3z1aGYIFpZ3F8N-v2GPjATsICkYaKlXMxyFxreCQzM99QyiOb74dUbTqTsmO9cnOt9WHW~j3TejjNZ66bI-ESSrjFtfcaXCJGB0Oj6Pk8t2g54ly2TJtJ8KKA5mjJSfkBFlhNG0lJsISKtUGQBAtyP3UI6L7rkvOx3jfSrVzBenIPRz7lF0DZGDfA0DGxdOLIxNlLEiM5~sVkB~dM~3Hu5BqyioQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig2_dsx030.png?Expires=2147483647&amp;Signature=mFHVXQsuJ2bgj4FI~PQG-sT9o3~eygh13OYcgZ7ls5Uu6pYpNk4t7SeMFFGPNRyYu8FGag1H82cBQEIR8iVPoaMQLZzhQN1s68znoksRZJrLKQpZuubz-Hai4uCQOm2872RMKZ3yGWJhm9v9n1UuN42fZyTzZgKQC5hy-PuEHrFRJLNy1Mei~P51CntTjfdxylv8Xu222pF-OaX8VxQAY2OLP5s~MzHUc8ce7IbHYDn55dVYikA-g1AuiPjRkPnsbFJqHMQwjy4Nmjor7S3SmTAVSF6Dh3rY8fz7HOjCcmGb9~8~OZcJDn0QbVqwAnWz6Z17QhiwIQXbsfXCbcby7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_fig3_dsx030.png?Expires=2147483647&amp;Signature=UM6ARvGVdUpwW9mBaNgYYO4x3N5coet12Ay0XP8BQjIPy9bj-SF~QoR4RWx0Y25EJ3ZJSzMAisCM9MDW8l~lS1ldbWpRaDvnX-G2BzvjwsMeXwEg-MKwE8zczEXpw7pFU-s48dHg-EQhgux3RN5FrrIFc3TASxZia45s3SzQgeIN6Ky936DJsIwQqyM4mP-hKU4lWHbrzPS6rZzGfnscMGGTxo8eEdwywD-sxYxOzmHbDZ95~LFska7yQDS1O5B8P4iknfNPPEP7KItbn8zHHU1ub7qYxxtcPrEut7KGyr6FOGYckKikar1p6hMwAcKU22broiBuKtbPuN4DaFXuEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S3</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s1_dsx030.xlsx?Expires=2147483647&amp;Signature=yB5TbdLaKY81NBGR~GiyngN34Db9jJBhv-SmYIkf06aXvoB6uDpXS5qWAR6ompO8SZlTQQ852MvQNLR7cmm4TQkdKTvXoO14NA6CSBmfQZqwpcORwuc~fffCFp-Vh8bF3k2o0vnbV4uc0xz0mv4ZSkfLjZ7-6TLMIOcDxm8ZNnkD6aQmPQujy4RVt~sQB2G2riTKH6NNiX~pEAMkninpUPDIQ~7ZnsXjW~88MuVqevi8yC0p5cPMpPy9jesvNA~TWefPtVuG3Ah0Ef8B7aHd5P-TfJRrPD2Okp4WHKjiBu2rRdFAcCX6cHKMAWDNT2IEE3QqhK8qbeJ5nK2qpdCfMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s2_dsx030.xlsx?Expires=2147483647&amp;Signature=LRyyJlT2jg2RlKz0os7yJhlocPB4zcjLsaN5M40wsTQwaO2eE8e1Dl5v68Ra6485z5Y7x90fhVFUMhA5MXHpbIHOGstG-9S9iGRAVvil2yJKJmKlewyxguplHy8oeo8y4B4C3F61Re~XMJIIT25NX9lKkPbW0R6KuZEWqvE36FjHt~0DZUCbbD1DCI3YNqOKbhLlf6AIUKAFN6p2~~zEf6tuZWGA7EunHx7Ozi3s~9Z8Gh8AY1ELReQ~YONovCJOWkPZnXoNEKv6-uPQlehr3cEtKa~IcT-TtVCxRG4sG5SmMA9NpFoj6NI4n1S0Vi4YKseim16EyQ7vN-EeSCkgdA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s3_dsx030.docx?Expires=2147483647&amp;Signature=etWKG95watVSRTGmXeNLT0vW7d4cICEzOETuLyiPjZZTVoVw1wP3N32CCJnNqy8TN~LsvTESo4poAIcbCykl-4-x35jHykD1Es-0SqutZ6Vpgiy~Qv69LkvskOguGSwumsBhpHS-26iQJ0PfZHGohrbFosHqGMkNrhQxCIUwZ35TT-NRRZkyVmr37Y~NsdxzUPpEb3lH3KmT3dPGCx4GLb-WrgD~r3lKj9PZij4sA3BwlopYmjfqqNJYosi3O01ctvkNQKS7dV22XsxAm1vxfABlD7QXhHMXpAlSytWqS04jFGE3aZw-n9mdqFeKYWE309wxSsNWeiAhZqLldySYQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s4_dsx030.docx?Expires=2147483647&amp;Signature=ErFxiRxhSdSIqhaotruYxWkdWONWd5MZyQyAz6y0jZK~cvRNQsVSk3htI8JjFvJ5rBG5XE9Kf96E0DBiMX6nwR~LDOvUDjBBlfRZ7FvaqoyXPfhlHH9YqlIIVYTFLaw36llFFmWClTpafjOppBz-Mo~a5knBin9CJMVZi56MiaE~NuBopmZntLc2FoTULYNUB2DX6HN8QFkpMQV-qPy-oKZQR~aTNGI4lfQRRnH6ZEpQB8lpg9RXj-34ckURP5dOFVluurQFlsvRV~3lOCazuvn8FxQ94ffDdq5BOGNRmFEHiHWsKSyE1y3eI2DBPRYVci3yShaEc7OhgnOme5xuMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_table_s5_dsx030.xlsx?Expires=2147483647&amp;Signature=pdAMIpw9EL77tnye1WVRhNLHBcrTFtsMI0Elrg7Rtiaf5pFXcN23CdOH9DMtCCwb7H5uxnz3vKNxvDDAuNmGL0KKh1L4JG0EXEZPkRU3W9ON49UhK5t5ddZUD3ned3-jDu2q2GvhFnR2gsoe0Ku4UKfebw4b4P2-0TQJUKRMBFyFu8E6jrKgV7LYESQsYFi4x1bu7ZO17P~JED-UV9Bh7OlcbwNC9bBr85XV9vT1IG54fCuPLrWaoR8DsoDt3Hscf-8aj74P061p~H-gm1oFbpdo0acxs12wfeoAdjJDQmUGt23ZBYve0bWj3HK4GmQY1wDWpZH8-7oQzyDKJIc6yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplemental_table_s6_dsx030.xlsx?Expires=2147483647&amp;Signature=Vq~tN5s~~dywi6GLp1HgNzrtujOmmvakBGs4JvcadgXf~1xnEjM1KdknlmJkpxq0ortgriGZCQ6nfq-FkEyt2gtROgqDEVz0ei4OnUpGQkgjBPHh7JCy7Eok82CGUInsTTEmhfBK6fGcueBxUCxpFzwR0BKCSjKsfyRe~4LWRWahUPku8fJ~VkK~IsRZfr7MoR9Hno-23Tm3NGYmaOfeK9PHcwwPlPz7P5gZIQZAN~K17o~UdU6lPcGJ-sd620V9w5~dWYASzdJJng9pUezt~9agQAR44ALi3IfYgTYr48TyWvICkqwNGjb~IucJKNDFcAJn05CzlNsldQNMTbXuzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S6</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx030/1/supplementary_figures_dsx030.docx?Expires=2147483647&amp;Signature=c~Phi5lStY6AKR-904am8-~L-GrftHSGINf4ulS4GPGGl6cZWoXi7az2hn5~A0TQSHllSJCIrKYTXflDXgpfS7MVseerRIX9yqd1vG2N2~0pTfvVrHSZbedZi00kkOLOaCtIU9IlyUvxr5y-u2gw3zOSo2jkU8EIrREc3SxenSCza9qW4KGCnDyrIZjY4UJ9yi-SMz8gMdF7iz~caeqQDvNsH6VYYyywSWzQa~ftCYiqS1JaceixRe127v-s2JJsUx7mI4mfWGgwe7xcGCCpEJzDWTUjrI15bIHEwQB8csPqY7wFDSlnQinccaU-pSN8AMFEXRt8lPbzHXwznAuyDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures</div></a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">890</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx030"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3953933"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f623%2f3953933">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000418420400007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F623%2F3953933" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3953933'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Co-adaption of tRNA gene copy number and amino acid usage influences translation rates in three life domains" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx030");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f623%2f3953933">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

