

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/2/81/1745287","@type":"ScholarlyArticle","name":"An ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper ( Capsicum annuum )","datePublished":"2016-01-06","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2016-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv038","keywords":["pepper","NGS","bin map","QTL","morphological trait"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/2/81/1745287","author":[{"name":"Han, Koeun","affiliation":"1  Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences ,  Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Jeong, Hee-Jin","affiliation":"1  Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences ,  Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Yang, Hee-Bum","affiliation":"1  Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences ,  Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Kang, Sung-Min","affiliation":"2  Department of Computer Science ,  College of Information Science and Technology, KAIST ,  Daejeon 305-701 ,  Republic of Korea","@type":"Person"},{"name":"Kwon, Jin-Kyung","affiliation":"1  Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences ,  Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Seungill","affiliation":"3  Department of Plant Science, Plant Genomics and Breeding Institute, College of Agriculture and Life Sciences ,  Seoul National University ,  599 Gwanak-ro Gwank-gu, Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Choi, Doil","affiliation":"3  Department of Plant Science, Plant Genomics and Breeding Institute, College of Agriculture and Life Sciences ,  Seoul National University ,  599 Gwanak-ro Gwank-gu, Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Kang, Byoung-Cheorl","affiliation":"1  Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences ,  Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"}],"description":"Abstract.  Most agricultural traits are controlled by quantitative trait loci (QTLs); however, there are few studies on QTL mapping of horticultural traits in p","pageStart":"81","pageEnd":"91","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"An ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper ( Capsicum annuum )","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper ( Capsicum annuum )" />
    <meta property="og:description" content="Abstract.  Most agricultural traits are controlled by quantitative trait loci (QTLs); however, there are few studies on QTL mapping of horticultural traits in p" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/2/81/1745287" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-01-06" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Han, Koeun" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Jeong, Hee-Jin" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Yang, Hee-Bum" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Kang, Sung-Min" /><meta name="citation_author_institution" content="Department of Computer Science, College of Information Science and Technology, KAIST, Daejeon 305-701, Republic of Korea" /><meta name="citation_author" content="Kwon, Jin-Kyung" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Kim, Seungill" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, College of Agriculture and Life Sciences, Seoul National University, 599 Gwanak-ro Gwank-gu, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Choi, Doil" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, College of Agriculture and Life Sciences, Seoul National University, 599 Gwanak-ro Gwank-gu, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Kang, Byoung-Cheorl" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_title" content="An ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper (Capsicum annuum)" /><meta name="citation_firstpage" content="81" /><meta name="citation_lastpage" content="91" /><meta name="citation_doi" content="10.1093/dnares/dsv038" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2016/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetic and molecular regulation of fruit and plant domestication traits in tomato and pepper;  citation_journal_title=J. Exp. Bot.;  citation_year=2007;  citation_volume=58;  citation_pages=3841-52; " /><meta name="citation_reference" content="citation_title=The taxonomy of the genus Capsicum (Solanaceae);  citation_journal_title=Phytologia;  citation_year=1980;  citation_volume=47;  citation_pages=153-66; " /><meta name="citation_reference" content="citation_title=Genetic variability in domesticated Capsicum spp. as assessed by morphological and agronomic data in mixed statistical analysis;  citation_journal_title=Genet. Mol. Res.;  citation_year=2010;  citation_volume=9;  citation_pages=283-94; " /><meta name="citation_reference" content="citation_title=Estimation of phenotypic divergence in a collection of Capsicum species for yield-related traits;  citation_journal_title=Euphytica;  citation_year=2009;  citation_volume=168;  citation_pages=189-96; " /><meta name="citation_reference" content="citation_title=QTL mapping of fruit-related traits in pepper (Capsicum annuum);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=1016-28; " /><meta name="citation_reference" content="citation_title=Linkage of the A locus for the presence of anthocyanin and fs10.1, a major fruit-shape QTL in pepper;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=889-94; " /><meta name="citation_reference" content="citation_title=Mapping of yield-related QTLs in pepper in an interspecific cross of Capsicum annuum and C. frutescens;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=1457-66; " /><meta name="citation_reference" content="citation_title=QTLs mapping for fruit size and shape in chromosomes 2 and 4 in pepper and a comparison of the pepper QTL map with that of tomato;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=437-45; " /><meta name="citation_reference" content="citation_title=fs3.1: a major fruit shape QTL conserved in Capsicum;  citation_journal_title=Genome;  citation_year=2003;  citation_volume=46;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Identification of QTLs for capsaicinoids, fruit quality, and plant architecture-related traits in an interspecific Capsicum RIL population;  citation_journal_title=Genome;  citation_year=2013;  citation_volume=56;  citation_pages=61-74; " /><meta name="citation_reference" content="citation_title=An interspecific (Capsicum annuum× C. chinese) F2 linkage map in pepper using RFLP and AFLP markers;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=531-9; " /><meta name="citation_reference" content="citation_title=A high-resolution, intraspecific linkage map of pepper (Capsicum annuum L.) and selection of reduced recombinant inbred line subsets for fast mapping;  citation_journal_title=Genome;  citation_year=2007;  citation_volume=50;  citation_pages=51-60; " /><meta name="citation_reference" content="citation_title=Characterization of Capsicum annuum genetic diversity and population structure based on parallel polymorphism discovery with a 30K unigene Pepper GeneChip;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e56200" /><meta name="citation_reference" content="citation_title=Genotype and SNP calling from next-generation sequencing data;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=443-51; " /><meta name="citation_reference" content="citation_title=Low-coverage sequencing: implications for design of complex trait association studies;  citation_journal_title=Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=940-51; " /><meta name="citation_reference" content="citation_title=Extremely low-coverage sequencing and imputation increases power for genome-wide association studies;  citation_journal_title=Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=631-5; " /><meta name="citation_reference" content="citation_title=High-throughput genotyping by whole-genome resequencing;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1068-76; " /><meta name="citation_reference" content="citation_title=Mapping 49 quantitative trait loci at high resolution through sequencing-based genotyping of rice recombinant inbred lines;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=327-40; " /><meta name="citation_reference" content="citation_title=Identification of QTLs for eight agronomically important traits using an ultra-high-density map based on SNPs generated from high-throughput sequencing in sorghum under contrasting photoperiods;  citation_journal_title=J. Exp. Bot.;  citation_year=2012;  citation_volume=63;  citation_pages=5451-62; " /><meta name="citation_reference" content="citation_title=An ultra-high density bin-map for rapid QTL mapping for tassel and ear architecture in a large F2 maize population;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=433" /><meta name="citation_reference" content="citation_title=CARHTA GENE: multipopulation integrated genetic and radiation hybrid mapping;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1703-4; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs;  citation_journal_title=J Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Development of a controlled vocabulary and software application to analyze fruit shape variation in tomato and other plant species;  citation_journal_title=Plant Physiol.;  citation_year=2006;  citation_volume=141;  citation_pages=15-25; " /><meta name="citation_reference" content="citation_title=Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species;  citation_journal_title=Nat. Genet.;  citation_year=2014;  citation_volume=46;  citation_pages=270-8; " /><meta name="citation_reference" content="  Wang S., Basten C.J., Zeng Z.B.  2012 , Windows QTL Cartographer 2.5. Department of Statistics. North Carolina State University. http://statgen.ncsu.edu/qtlcart/WQTLCart.htm. " /><meta name="citation_reference" content="citation_title=The Pun1 gene for pungency in pepper encodes a putative acyltransferase;  citation_journal_title=Plant J.;  citation_year=2005;  citation_volume=42;  citation_pages=675-88; " /><meta name="citation_reference" content="citation_title=From centiMorgans to base pairs: homologous recombination in plants;  citation_journal_title=Trends Plant Sci.;  citation_year=1996;  citation_volume=1;  citation_pages=340-8; " /><meta name="citation_reference" content="citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2013;  citation_volume=126;  citation_pages=2699-716; " /><meta name="citation_reference" content="citation_title=Widely distributed hot and cold spots in meiotic recombination as shown by the sequencing of rice F2 plants;  citation_journal_title=New Phytol.;  citation_year=2015;  citation_volume=206;  citation_pages=1491-502; " /><meta name="citation_reference" content="citation_title=Sequencing depth and coverage: key considerations in genomic analyses;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2014;  citation_volume=15;  citation_pages=121-32; " /><meta name="citation_reference" content="citation_title=Biosynthesis of capsinoid is controlled by the Pun1 locus in pepper;  citation_journal_title=Mol Breed;  citation_year=2012;  citation_volume=31;  citation_pages=537-48; " /><meta name="citation_reference" content="citation_title=Marker development for erect versus pendant-orientated fruit in Capsicum annuum L;  citation_journal_title=Mol. Cells;  citation_year=2008;  citation_volume=26;  citation_pages=548-53; " /><meta name="citation_reference" content="citation_title=QTL analysis of plant development and fruit traits in pepper and performance of selective phenotyping;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=1157-71; " /><meta name="citation_reference" content="citation_title=Characterization of fs10.1, a major QTL controlling fruit elongation in Capsicum;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=657-65; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/2/81/17462833/dsv038.pdf" /><meta name="description" content="Abstract.  Most agricultural traits are controlled by quantitative trait loci (QTLs); however, there are few studies on QTL mapping of horticultural traits in p" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/2/81/1745287" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/2/81/1745287" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/2/81/1745287");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv038';
    </script>
<title>
	ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper (Capsicum annuum) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745287" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd9hPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYW4sIEtvZXVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBQbGFudCBHZW5vbWljcyBhbmQgQnJlZWRpbmcgSW5zdGl0dXRlIGFuZCBWZWdldGFibGUgQnJlZWRpbmcgUmVzZWFyY2ggQ2VudGVyLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAxNTEtOTIxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikplb25nLCBIZWUtSmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBQbGFudCBHZW5vbWljcyBhbmQgQnJlZWRpbmcgSW5zdGl0dXRlIGFuZCBWZWdldGFibGUgQnJlZWRpbmcgUmVzZWFyY2ggQ2VudGVyLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAxNTEtOTIxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbmcsIEhlZS1CdW0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2UsIFBsYW50IEdlbm9taWNzIGFuZCBCcmVlZGluZyBJbnN0aXR1dGUgYW5kIFZlZ2V0YWJsZSBCcmVlZGluZyBSZXNlYXJjaCBDZW50ZXIsIENvbGxlZ2Ugb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZXMsIFNlb3VsIE5hdGlvbmFsIFVuaXZlcnNpdHksIFNlb3VsIDE1MS05MjEsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FuZywgU3VuZy1NaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGVyIFNjaWVuY2UsIENvbGxlZ2Ugb2YgSW5mb3JtYXRpb24gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgS0FJU1QsIERhZWplb24gMzA1LTcwMSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLd29uLCBKaW4tS3l1bmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2UsIFBsYW50IEdlbm9taWNzIGFuZCBCcmVlZGluZyBJbnN0aXR1dGUgYW5kIFZlZ2V0YWJsZSBCcmVlZGluZyBSZXNlYXJjaCBDZW50ZXIsIENvbGxlZ2Ugb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZXMsIFNlb3VsIE5hdGlvbmFsIFVuaXZlcnNpdHksIFNlb3VsIDE1MS05MjEsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBTZXVuZ2lsbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgU2NpZW5jZSwgUGxhbnQgR2Vub21pY3MgYW5kIEJyZWVkaW5nIEluc3RpdHV0ZSwgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgNTk5IEd3YW5hay1ybyBHd2Fuay1ndSwgU2VvdWwgMTUxLTkyMSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaG9pLCBEb2lsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBQbGFudCBHZW5vbWljcyBhbmQgQnJlZWRpbmcgSW5zdGl0dXRlLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCA1OTkgR3dhbmFrLXJvIEd3YW5rLWd1LCBTZW91bCAxNTEtOTIxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthbmcsIEJ5b3VuZy1DaGVvcmwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2UsIFBsYW50IEdlbm9taWNzIGFuZCBCcmVlZGluZyBJbnN0aXR1dGUgYW5kIFZlZ2V0YWJsZSBCcmVlZGluZyBSZXNlYXJjaCBDZW50ZXIsIENvbGxlZ2Ugb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZXMsIFNlb3VsIE5hdGlvbmFsIFVuaXZlcnNpdHksIFNlb3VsIDE1MS05MjEsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBbiB1bHRyYS1oaWdoLWRlbnNpdHkgYmluIG1hcCBmYWNpbGl0YXRlcyBoaWdoLXRocm91Z2hwdXQgUVRMIG1hcHBpbmcgb2YgaG9ydGljdWx0dXJhbCB0cmFpdHMgaW4gcGVwcGVyIChDYXBzaWN1bSBhbm51dW0pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuZCBtb2xlY3VsYXIgcmVndWxhdGlvbiBvZiBmcnVpdCBhbmQgcGxhbnQgZG9tZXN0aWNhdGlvbiB0cmFpdHMgaW4gdG9tYXRvIGFuZCBwZXBwZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTM4NDEtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0YXhvbm9teSBvZiB0aGUgZ2VudXMgQ2Fwc2ljdW0gKFNvbGFuYWNlYWUpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHl0b2xvZ2lhOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9MTUzLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHZhcmlhYmlsaXR5IGluIGRvbWVzdGljYXRlZCBDYXBzaWN1bSBzcHAuIGFzIGFzc2Vzc2VkIGJ5IG1vcnBob2xvZ2ljYWwgYW5kIGFncm9ub21pYyBkYXRhIGluIG1peGVkIHN0YXRpc3RpY2FsIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gTW9sLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yODMtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpb24gb2YgcGhlbm90eXBpYyBkaXZlcmdlbmNlIGluIGEgY29sbGVjdGlvbiBvZiBDYXBzaWN1bSBzcGVjaWVzIGZvciB5aWVsZC1yZWxhdGVkIHRyYWl0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2ODsgIGNpdGF0aW9uX3BhZ2VzPTE4OS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIG1hcHBpbmcgb2YgZnJ1aXQtcmVsYXRlZCB0cmFpdHMgaW4gcGVwcGVyIChDYXBzaWN1bSBhbm51dW0pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMTYtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmthZ2Ugb2YgdGhlIEEgbG9jdXMgZm9yIHRoZSBwcmVzZW5jZSBvZiBhbnRob2N5YW5pbiBhbmQgZnMxMC4xLCBhIG1ham9yIGZydWl0LXNoYXBlIFFUTCBpbiBwZXBwZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9ODg5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIG9mIHlpZWxkLXJlbGF0ZWQgUVRMcyBpbiBwZXBwZXIgaW4gYW4gaW50ZXJzcGVjaWZpYyBjcm9zcyBvZiBDYXBzaWN1bSBhbm51dW0gYW5kIEMuIGZydXRlc2NlbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTQ1Ny02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMcyBtYXBwaW5nIGZvciBmcnVpdCBzaXplIGFuZCBzaGFwZSBpbiBjaHJvbW9zb21lcyAyIGFuZCA0IGluIHBlcHBlciBhbmQgYSBjb21wYXJpc29uIG9mIHRoZSBwZXBwZXIgUVRMIG1hcCB3aXRoIHRoYXQgb2YgdG9tYXRvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTQzNy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9ZnMzLjE6IGEgbWFqb3IgZnJ1aXQgc2hhcGUgUVRMIGNvbnNlcnZlZCBpbiBDYXBzaWN1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBRVExzIGZvciBjYXBzYWljaW5vaWRzLCBmcnVpdCBxdWFsaXR5LCBhbmQgcGxhbnQgYXJjaGl0ZWN0dXJlLXJlbGF0ZWQgdHJhaXRzIGluIGFuIGludGVyc3BlY2lmaWMgQ2Fwc2ljdW0gUklMIHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTYxLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlcnNwZWNpZmljIChDYXBzaWN1bSBhbm51dW3DlyBDLiBjaGluZXNlKSBGMiBsaW5rYWdlIG1hcCBpbiBwZXBwZXIgdXNpbmcgUkZMUCBhbmQgQUZMUCBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTUzMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtcmVzb2x1dGlvbiwgaW50cmFzcGVjaWZpYyBsaW5rYWdlIG1hcCBvZiBwZXBwZXIgKENhcHNpY3VtIGFubnV1bSBMLikgYW5kIHNlbGVjdGlvbiBvZiByZWR1Y2VkIHJlY29tYmluYW50IGluYnJlZCBsaW5lIHN1YnNldHMgZm9yIGZhc3QgbWFwcGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9NTEtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgQ2Fwc2ljdW0gYW5udXVtIGdlbmV0aWMgZGl2ZXJzaXR5IGFuZCBwb3B1bGF0aW9uIHN0cnVjdHVyZSBiYXNlZCBvbiBwYXJhbGxlbCBwb2x5bW9ycGhpc20gZGlzY292ZXJ5IHdpdGggYSAzMEsgdW5pZ2VuZSBQZXBwZXIgR2VuZUNoaXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTYyMDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwZSBhbmQgU05QIGNhbGxpbmcgZnJvbSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDQzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3ctY292ZXJhZ2Ugc2VxdWVuY2luZzogaW1wbGljYXRpb25zIGZvciBkZXNpZ24gb2YgY29tcGxleCB0cmFpdCBhc3NvY2lhdGlvbiBzdHVkaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTk0MC01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0cmVtZWx5IGxvdy1jb3ZlcmFnZSBzZXF1ZW5jaW5nIGFuZCBpbXB1dGF0aW9uIGluY3JlYXNlcyBwb3dlciBmb3IgZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz02MzEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IGdlbm90eXBpbmcgYnkgd2hvbGUtZ2Vub21lIHJlc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMDY4LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIDQ5IHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGF0IGhpZ2ggcmVzb2x1dGlvbiB0aHJvdWdoIHNlcXVlbmNpbmctYmFzZWQgZ2Vub3R5cGluZyBvZiByaWNlIHJlY29tYmluYW50IGluYnJlZCBsaW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz0zMjctNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFFUTHMgZm9yIGVpZ2h0IGFncm9ub21pY2FsbHkgaW1wb3J0YW50IHRyYWl0cyB1c2luZyBhbiB1bHRyYS1oaWdoLWRlbnNpdHkgbWFwIGJhc2VkIG9uIFNOUHMgZ2VuZXJhdGVkIGZyb20gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgaW4gc29yZ2h1bSB1bmRlciBjb250cmFzdGluZyBwaG90b3BlcmlvZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTU0NTEtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIHVsdHJhLWhpZ2ggZGVuc2l0eSBiaW4tbWFwIGZvciByYXBpZCBRVEwgbWFwcGluZyBmb3IgdGFzc2VsIGFuZCBlYXIgYXJjaGl0ZWN0dXJlIGluIGEgbGFyZ2UgRjIgbWFpemUgcG9wdWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NDMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FSSFRBIEdFTkU6IG11bHRpcG9wdWxhdGlvbiBpbnRlZ3JhdGVkIGdlbmV0aWMgYW5kIHJhZGlhdGlvbiBoeWJyaWQgbWFwcGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xNzAzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBzb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KIEhlcmVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgY29udHJvbGxlZCB2b2NhYnVsYXJ5IGFuZCBzb2Z0d2FyZSBhcHBsaWNhdGlvbiB0byBhbmFseXplIGZydWl0IHNoYXBlIHZhcmlhdGlvbiBpbiB0b21hdG8gYW5kIG90aGVyIHBsYW50IHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGhvdCBwZXBwZXIgcHJvdmlkZXMgaW5zaWdodHMgaW50byB0aGUgZXZvbHV0aW9uIG9mIHB1bmdlbmN5IGluIENhcHNpY3VtIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MjcwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiAgV2FuZyBTLiwgQmFzdGVuIEMuSi4sIFplbmcgWi5CLiAgMjAxMiAsIFdpbmRvd3MgUVRMIENhcnRvZ3JhcGhlciAyLjUuIERlcGFydG1lbnQgb2YgU3RhdGlzdGljcy4gTm9ydGggQ2Fyb2xpbmEgU3RhdGUgVW5pdmVyc2l0eS4gaHR0cDovL3N0YXRnZW4ubmNzdS5lZHUvcXRsY2FydC9XUVRMQ2FydC5odG0uICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQdW4xIGdlbmUgZm9yIHB1bmdlbmN5IGluIHBlcHBlciBlbmNvZGVzIGEgcHV0YXRpdmUgYWN5bHRyYW5zZmVyYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTY3NS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJvbSBjZW50aU1vcmdhbnMgdG8gYmFzZSBwYWlyczogaG9tb2xvZ291cyByZWNvbWJpbmF0aW9uIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTM0MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CcmlkZ2luZyB0aGUgZ2Vub3R5cGluZyBnYXA6IHVzaW5nIGdlbm90eXBpbmcgYnkgc2VxdWVuY2luZyAoR0JTKSB0byBhZGQgaGlnaC1kZW5zaXR5IFNOUCBtYXJrZXJzIGFuZCBuZXcgdmFsdWUgdG8gdHJhZGl0aW9uYWwgYmktcGFyZW50YWwgbWFwcGluZyBhbmQgYnJlZWRpbmcgcG9wdWxhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MjY5OS03MTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdpZGVseSBkaXN0cmlidXRlZCBob3QgYW5kIGNvbGQgc3BvdHMgaW4gbWVpb3RpYyByZWNvbWJpbmF0aW9uIGFzIHNob3duIGJ5IHRoZSBzZXF1ZW5jaW5nIG9mIHJpY2UgRjIgcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDY7ICBjaXRhdGlvbl9wYWdlcz0xNDkxLTUwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBkZXB0aCBhbmQgY292ZXJhZ2U6IGtleSBjb25zaWRlcmF0aW9ucyBpbiBnZW5vbWljIGFuYWx5c2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTIxLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9zeW50aGVzaXMgb2YgY2Fwc2lub2lkIGlzIGNvbnRyb2xsZWQgYnkgdGhlIFB1bjEgbG9jdXMgaW4gcGVwcGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wgQnJlZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz01MzctNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcmtlciBkZXZlbG9wbWVudCBmb3IgZXJlY3QgdmVyc3VzIHBlbmRhbnQtb3JpZW50YXRlZCBmcnVpdCBpbiBDYXBzaWN1bSBhbm51dW0gTDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTU0OC01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIGFuYWx5c2lzIG9mIHBsYW50IGRldmVsb3BtZW50IGFuZCBmcnVpdCB0cmFpdHMgaW4gcGVwcGVyIGFuZCBwZXJmb3JtYW5jZSBvZiBzZWxlY3RpdmUgcGhlbm90eXBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MTE1Ny03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBmczEwLjEsIGEgbWFqb3IgUVRMIGNvbnRyb2xsaW5nIGZydWl0IGVsb25nYXRpb24gaW4gQ2Fwc2ljdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9NjU3LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvMi84MS8xNzQ2MjgzMy9kc3YwMzgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIE1vc3QgYWdyaWN1bHR1cmFsIHRyYWl0cyBhcmUgY29udHJvbGxlZCBieSBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSAoUVRMcyk7IGhvd2V2ZXIsIHRoZXJlIGFyZSBmZXcgc3R1ZGllcyBvbiBRVEwgbWFwcGluZyBvZiBob3J0aWN1bHR1cmFsIHRyYWl0cyBpbiBwIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0xNzQ1Mjg3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZLbiD702Id0lgBp4xaS0df4y0ekM" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f81%2f1745287" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f81%2f1745287" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69060" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&amp;Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/2">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87196808">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196808">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196810">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196810">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196815">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196815">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196828">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196828">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196912">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196912">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196918">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196918">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196920">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196920">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/23/2/93/1745299">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">An ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper (<em>Capsicum annuum</em>)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koeun Han</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koeun+Han">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Han K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Han K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Koeun Han</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hee-Jin Jeong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hee-Jin+Jeong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeong H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeong H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hee-Jin Jeong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hee-Bum Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hee-Bum+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hee-Bum Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sung-Min Kang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Computer Science</div>, <div class="insititution">College of Information Science and Technology, KAIST</div>, <div class="addr-line">Daejeon 305-701</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sung-Min+Kang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kang S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sung-Min Kang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jin-Kyung Kwon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jin-Kyung+Kwon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kwon J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kwon J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jin-Kyung Kwon</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seungill Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Plant Science, Plant Genomics and Breeding Institute, College of Agriculture and Life Sciences</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">599 Gwanak-ro Gwank-gu, Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seungill+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seungill Kim</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Doil Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Plant Science, Plant Genomics and Breeding Institute, College of Agriculture and Life Sciences</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">599 Gwanak-ro Gwank-gu, Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 2-880-4563. Fax. +82 2-873-2056. E-mail: <a href="mailto:bk54@snu.ac.kr" target="_blank">bk54@snu.ac.kr</a> (B.-C.K.); Tel. +82-2-880-4568. Fax. +82-2-873-2056. E-mail: <a href="mailto:doil@snu.ac.kr" target="_blank">doil@snu.ac.kr</a> (D.C.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Doil+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Doil Choi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Byoung-Cheorl Kang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science, Plant Genomics and Breeding Institute and Vegetable Breeding Research Center, College of Agriculture and Life Sciences</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 2-880-4563. Fax. +82 2-873-2056. E-mail: <a href="mailto:bk54@snu.ac.kr" target="_blank">bk54@snu.ac.kr</a> (B.-C.K.); Tel. +82-2-880-4568. Fax. +82-2-873-2056. E-mail: <a href="mailto:doil@snu.ac.kr" target="_blank">doil@snu.ac.kr</a> (D.C.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Byoung-Cheorl+Kang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kang B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kang B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Byoung-Cheorl Kang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 81–91, <a href='https://doi.org/10.1093/dnares/dsv038'>https://doi.org/10.1093/dnares/dsv038</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 January 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 September 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 November 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745287" href="/dnaresearch/article-pdf/23/2/81/17462833/dsv038.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Koeun Han, Hee-Jin Jeong, Hee-Bum Yang, Sung-Min Kang, Jin-Kyung Kwon, Seungill Kim, Doil Choi, Byoung-Cheorl Kang;  An ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper (<em>Capsicum annuum</em>), <em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 81–91, <a href='https://doi.org/10.1093/dnares/dsv038'>https://doi.org/10.1093/dnares/dsv038</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745287?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745287?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745287?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745287?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745287?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745287?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745287?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=An%20ultra-high-density%20bin%20map%20facilitates%20high-throughput%20QTL%20mapping%20of%20horticultural%20traits%20in%20pepper%20(%26lt%3Bem%26gt%3BCapsicum%20annuum%26lt%3B%2Fem%26gt%3B)&amp;publicationDate=2016-01-06&amp;volumeNum=23&amp;issueNum=2&amp;author=Han%2C%20Koeun%3B%20Jeong%2C%20Hee-Jin&amp;startPage=81&amp;endPage=91&amp;contentId=10.1093%2Fdnares%2Fdsv038&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f81%2f1745287" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87196808 id="87196808" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Most agricultural traits are controlled by quantitative trait loci (QTLs); however, there are few studies on QTL mapping of horticultural traits in pepper (<em>Capsicum</em> spp.) due to the lack of high-density molecular maps and the sequence information. In this study, an ultra-high-density map and 120 recombinant inbred lines (RILs) derived from a cross between <em>C. annuum</em> ‘Perennial’ and <em>C. annuum</em> ‘Dempsey’ were used for QTL mapping of horticultural traits. Parental lines and RILs were resequenced at 18× and 1× coverage, respectively. Using a sliding window approach, an ultra-high-density bin map containing 2,578 bins was constructed. The total map length of the map was 1,372 cM, and the average interval between bins was 0.53 cM. A total of 86 significant QTLs controlling 17 horticultural traits were detected. Among these, 32 QTLs controlling 13 traits were major QTLs. Our research shows that the construction of bin maps using low-coverage sequence is a powerful method for QTL mapping, and that the short intervals between bins are helpful for fine-mapping of QTLs. Furthermore, bin maps can be used to improve the quality of reference genomes by elucidating the genetic order of unordered regions and anchoring unassigned scaffolds to linkage groups.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="pepper">pepper</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="NGS">NGS</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bin map&quot;">bin map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="QTL">QTL</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;morphological trait&quot;">morphological trait</a></div>                            <h2 scrollto-destination=87196810 id="87196810" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Pepper (<em>Capsicum</em> spp.) is one of the first domesticated vegetables<sup><a reveal-id="DSV038C1" data-open="DSV038C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and shows enormous variation in plant architecture, flower-, leaf-, and fruit-related traits.<sup><a reveal-id="DSV038C2  DSV038C4" data-open="DSV038C2  DSV038C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Most yield and fruit quality traits important in pepper breeding are regulated by multiple genes or so-called quantitative trait loci (QTLs). QTL analyses of morphological traits in pepper have been performed using intraspecific and interspecific populations.<sup><a reveal-id="DSV038C5  DSV038C8" data-open="DSV038C5  DSV038C8" class="link link-ref link-reveal xref-bibr">5–8</a></sup> Most QTL analyses for morphological traits have concentrated on fruit-associated traits. For instance, <em>fs3.1</em> and <em>fs10.1</em> control fruit shape (ratio of fruit length to fruit width).<sup><a reveal-id="DSV038C5" data-open="DSV038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV038C9" data-open="DSV038C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Major QTLs controlling fruit weight are present in chromosomes 2 and 4, and in comparison with the tomato genetic map, some of these QTLs were found to be conserved between the two crops.<sup><a reveal-id="DSV038C7" data-open="DSV038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV038C8" data-open="DSV038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> QTL analyses for plant height, leaf area, stem length, and other traits have also been performed,<sup><a reveal-id="DSV038C5" data-open="DSV038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV038C7" data-open="DSV038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV038C10" data-open="DSV038C10" class="link link-ref link-reveal xref-bibr">10</a></sup> but no major QTLs were detected and the genes controlling QTLs have not been characterized.</p><p>For QTL mapping, restriction fragment length polymorphism, amplified fragment length polymorphism, random amplified polymorphic DNA, and microsatellite markers have been widely used.<sup><a reveal-id="DSV038C5" data-open="DSV038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV038C7" data-open="DSV038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV038C11" data-open="DSV038C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV038C12" data-open="DSV038C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, for accurate detection and characterization of QTLs, high-density genetic maps are required. Recently, a pepper GeneChip containing 30,815 unigenes derived from EST sequences was used for genotyping a recombinant inbred line (RIL) population.<sup><a reveal-id="DSV038C10" data-open="DSV038C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV038C13" data-open="DSV038C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Using a high-density EST-based genetic map, 96 QTLs for 38 traits were mapped.<sup><a reveal-id="DSV038C10" data-open="DSV038C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, the genes controlling the QTL regions were not discovered in this analysis.</p><p>Next-generation sequencing (NGS) methods are commonly used for whole-genome sequencing.<sup><a reveal-id="DSV038C14" data-open="DSV038C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV038C15" data-open="DSV038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Once a reference genome is available, SNP markers can be easily developed by aligning resequencing data to the reference genome. Although the time and cost involved in sequencing has been greatly reduced, resequencing large populations or organisms with a large genome remains challenging. Alternatively, low-coverage sequencing is a cost-efficient way to genotype for QTL analyses or association studies using a large number of samples.<sup><a reveal-id="DSV038C14" data-open="DSV038C14" class="link link-ref link-reveal xref-bibr">14</a></sup> However, due to the low coverage of sequencing reads, this approach misses a large number of genotypes, and new computational tools have been needed to address the resulting reduction in mapping accuracy. Imputation of missing data using a reference genome can increase the power of SNP calling in genetic studies.<sup><a reveal-id="DSV038C15" data-open="DSV038C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSV038C16" data-open="DSV038C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Generally, imputation tools predict the genotype of missing data based on the haplotype ratio at that site. This method can increase the accuracy of mapping, and enables usage of most of the sequencing data. An alternative method is the sliding window approach, which has been used in cereal crops such as rice, sorghum, and maize to construct bin maps and for QTL analysis.<sup><a reveal-id="DSV038C17  DSV038C20" data-open="DSV038C17  DSV038C20" class="link link-ref link-reveal xref-bibr">17–20</a></sup> Instead of using all SNPs as independent markers, the sliding window approach merges consecutive SNPs into one bin, which can greatly reduce errors caused by inaccurate SNP calling.</p><p>The first assembled reference genomes of cultivated hot pepper (<em>Capsicum annuum</em>) and wild pepper (<em>C. chinense</em> and <em>C. annuum</em> var. <em>glabriusculum</em>) were released in 2014.<sup><a reveal-id="DSV038C21" data-open="DSV038C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV038C22" data-open="DSV038C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The availability of reference genome information facilitates the identification of horticulturally important genes, the development of useful markers, and improvement of cultivars. In this study, we identified QTLs controlling morphological traits using RILs derived from a cross between <em>C. annuum</em> ‘Perennial’ and <em>C. annuum</em> ‘Dempsey’. To increase the efficiency of QTL analysis, we improved the ultra-high-density genetic map for the reference genome using a sliding window approach. Horticultural traits were evaluated in four environments to detect major QTLs, and the ultra-high-density bin map increased the speed of QTL mapping. Furthermore, we improved the reference genome of pepper by correcting the genetic order of pseudomolecules and anchoring unassigned scaffolds into linkage groups.</p>                            <h2 scrollto-destination=87196815 id="87196815" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87196816 id="87196816" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>A total of 120 F<sub>7</sub>–F<sub>10</sub> RILs obtained from a cross between pungent <em>Capsicum annuum</em> ‘Perennial’ and non-pungent <em>C. annuum</em> ‘Dempsey’ were used. RILs were grown for 3 years (2011, 2012, and 2014) to reduce environmental effects in the detection of QTLs. Plants were grown under greenhouse conditions in two locations, Anseong (2011 and 2012a) and Suwon, Korea (2012b and 2014). In 2011 and 2012a, plants were grown in the ground, whereas plants were planted in pots in 2012b and 2014. Seeds were disinfected using 2% sodium chlorate and 10% trisodium phosphate. Five plants per line were planted and fruits were harvested separately.</p>                            <h3 scrollto-destination=87196818 id="87196818" class="section-title" data-legacy-id=s2b>2.2. Phenotype evaluation</h3>
<p>A total of 18 horticultural traits were evaluated for ‘Perennial’, ‘Dempsey’, and 120 RILs based on the definitions of RDA-Genebank, Korea. Stem colour, flower size, fruit position, calyx shape, and immature fruit colour were graded according to the definitions for efficient evaluations (Table <a reveal-id="DSV038TB1" data-open="DSV038TB1" class="link link-reveal link-table xref-fig">1</a>). The other 14 traits were evaluated in three plants per line using the relevant units and average values were calculated. Traits related to plant architecture, leaf, and flower were evaluated 9 weeks after transplanting in 2011, and 10 weeks after transplanting in 2012a, 2012b, and 2014. Fruit-related traits were evaluated after harvest. For precise measurement of the fruit length and width, the image of hemisected fruits was scanned (Epson V30) and analysed using Tomato Analyser 3.0.<sup><a reveal-id="DSV038C23" data-open="DSV038C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Pearson correlation coefficients were calculated to analyse the relationship between traits in each environment. </p>                        <a id="87196820" scrollto-destination="87196820"></a>
<div content-id="DSV038TB1" class="table-modal"><div class="table-wrap"><div id="DSV038TB1" class="table-wrap-title" data-id="DSV038TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Morphological traits evaluated in RILs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Phenotype </th><th>Description </th></tr></thead><tbody><tr><td colspan="2">Plant architecture </td></tr><tr><td> Plant height (cm) </td><td>From soil to head of the plant </td></tr><tr><td> Plant width (cm) </td><td>Wide part of the plant </td></tr><tr><td> Main stem length (cm) </td><td>From soil to the first branch </td></tr><tr><td> Stem thickness (cm) </td><td>Thickness of basal stem </td></tr><tr><td> Lateral branch number </td><td>Basal lateral branch number before the first branch </td></tr><tr><td> Internode length (cm) </td><td>Length of internode between the third and fourth node </td></tr><tr><td> Stem colour </td><td>1: Green, 2: Green with purple, 3: Purple </td></tr><tr><td colspan="2">Leaf </td></tr><tr><td> Leaf length (cm) </td><td>Length of completely grown leaf </td></tr><tr><td> Leaf width (cm) </td><td>Width of completely grown leaf </td></tr><tr><td colspan="2">Flower </td></tr><tr><td> Flower size </td><td>1: Small, 2: Intermediate, 3: Big </td></tr><tr><td> Stamen number </td><td>Most frequent number of stamen </td></tr><tr><td colspan="2">Fruit </td></tr><tr><td> Fruit length (cm) </td><td>Average length of fruit </td></tr><tr><td> Fruit diameter (cm) </td><td>Average width of fruit </td></tr><tr><td> Fruit shape </td><td>Ratio between fruit length and fruit diameter </td></tr><tr><td> Fruit weight (g) </td><td>Average weight of fresh fruit </td></tr><tr><td> Fruit position </td><td>1: Erect, 2: Intermediate, 3: Pendant </td></tr><tr><td> Calyx shape </td><td>1: Cup-shaped, 2: Intermediate, 3: Saucer-shaped </td></tr><tr><td> Immature fruit colour </td><td>1: Light green, 2: Green, 3: Dark-green </td></tr></tbody></table></div><div content-id="DSV038TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Phenotype </th><th>Description </th></tr></thead><tbody><tr><td colspan="2">Plant architecture </td></tr><tr><td> Plant height (cm) </td><td>From soil to head of the plant </td></tr><tr><td> Plant width (cm) </td><td>Wide part of the plant </td></tr><tr><td> Main stem length (cm) </td><td>From soil to the first branch </td></tr><tr><td> Stem thickness (cm) </td><td>Thickness of basal stem </td></tr><tr><td> Lateral branch number </td><td>Basal lateral branch number before the first branch </td></tr><tr><td> Internode length (cm) </td><td>Length of internode between the third and fourth node </td></tr><tr><td> Stem colour </td><td>1: Green, 2: Green with purple, 3: Purple </td></tr><tr><td colspan="2">Leaf </td></tr><tr><td> Leaf length (cm) </td><td>Length of completely grown leaf </td></tr><tr><td> Leaf width (cm) </td><td>Width of completely grown leaf </td></tr><tr><td colspan="2">Flower </td></tr><tr><td> Flower size </td><td>1: Small, 2: Intermediate, 3: Big </td></tr><tr><td> Stamen number </td><td>Most frequent number of stamen </td></tr><tr><td colspan="2">Fruit </td></tr><tr><td> Fruit length (cm) </td><td>Average length of fruit </td></tr><tr><td> Fruit diameter (cm) </td><td>Average width of fruit </td></tr><tr><td> Fruit shape </td><td>Ratio between fruit length and fruit diameter </td></tr><tr><td> Fruit weight (g) </td><td>Average weight of fresh fruit </td></tr><tr><td> Fruit position </td><td>1: Erect, 2: Intermediate, 3: Pendant </td></tr><tr><td> Calyx shape </td><td>1: Cup-shaped, 2: Intermediate, 3: Saucer-shaped </td></tr><tr><td> Immature fruit colour </td><td>1: Light green, 2: Green, 3: Dark-green </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196820" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV038TB1" class="table-wrap-title" data-id="DSV038TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Morphological traits evaluated in RILs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Phenotype </th><th>Description </th></tr></thead><tbody><tr><td colspan="2">Plant architecture </td></tr><tr><td> Plant height (cm) </td><td>From soil to head of the plant </td></tr><tr><td> Plant width (cm) </td><td>Wide part of the plant </td></tr><tr><td> Main stem length (cm) </td><td>From soil to the first branch </td></tr><tr><td> Stem thickness (cm) </td><td>Thickness of basal stem </td></tr><tr><td> Lateral branch number </td><td>Basal lateral branch number before the first branch </td></tr><tr><td> Internode length (cm) </td><td>Length of internode between the third and fourth node </td></tr><tr><td> Stem colour </td><td>1: Green, 2: Green with purple, 3: Purple </td></tr><tr><td colspan="2">Leaf </td></tr><tr><td> Leaf length (cm) </td><td>Length of completely grown leaf </td></tr><tr><td> Leaf width (cm) </td><td>Width of completely grown leaf </td></tr><tr><td colspan="2">Flower </td></tr><tr><td> Flower size </td><td>1: Small, 2: Intermediate, 3: Big </td></tr><tr><td> Stamen number </td><td>Most frequent number of stamen </td></tr><tr><td colspan="2">Fruit </td></tr><tr><td> Fruit length (cm) </td><td>Average length of fruit </td></tr><tr><td> Fruit diameter (cm) </td><td>Average width of fruit </td></tr><tr><td> Fruit shape </td><td>Ratio between fruit length and fruit diameter </td></tr><tr><td> Fruit weight (g) </td><td>Average weight of fresh fruit </td></tr><tr><td> Fruit position </td><td>1: Erect, 2: Intermediate, 3: Pendant </td></tr><tr><td> Calyx shape </td><td>1: Cup-shaped, 2: Intermediate, 3: Saucer-shaped </td></tr><tr><td> Immature fruit colour </td><td>1: Light green, 2: Green, 3: Dark-green </td></tr></tbody></table></div><div content-id="DSV038TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Phenotype </th><th>Description </th></tr></thead><tbody><tr><td colspan="2">Plant architecture </td></tr><tr><td> Plant height (cm) </td><td>From soil to head of the plant </td></tr><tr><td> Plant width (cm) </td><td>Wide part of the plant </td></tr><tr><td> Main stem length (cm) </td><td>From soil to the first branch </td></tr><tr><td> Stem thickness (cm) </td><td>Thickness of basal stem </td></tr><tr><td> Lateral branch number </td><td>Basal lateral branch number before the first branch </td></tr><tr><td> Internode length (cm) </td><td>Length of internode between the third and fourth node </td></tr><tr><td> Stem colour </td><td>1: Green, 2: Green with purple, 3: Purple </td></tr><tr><td colspan="2">Leaf </td></tr><tr><td> Leaf length (cm) </td><td>Length of completely grown leaf </td></tr><tr><td> Leaf width (cm) </td><td>Width of completely grown leaf </td></tr><tr><td colspan="2">Flower </td></tr><tr><td> Flower size </td><td>1: Small, 2: Intermediate, 3: Big </td></tr><tr><td> Stamen number </td><td>Most frequent number of stamen </td></tr><tr><td colspan="2">Fruit </td></tr><tr><td> Fruit length (cm) </td><td>Average length of fruit </td></tr><tr><td> Fruit diameter (cm) </td><td>Average width of fruit </td></tr><tr><td> Fruit shape </td><td>Ratio between fruit length and fruit diameter </td></tr><tr><td> Fruit weight (g) </td><td>Average weight of fresh fruit </td></tr><tr><td> Fruit position </td><td>1: Erect, 2: Intermediate, 3: Pendant </td></tr><tr><td> Calyx shape </td><td>1: Cup-shaped, 2: Intermediate, 3: Saucer-shaped </td></tr><tr><td> Immature fruit colour </td><td>1: Light green, 2: Green, 3: Dark-green </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196820" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87196821 id="87196821" class="section-title" data-legacy-id=s2c>2.3. Genetic mapping and bin map construction</h3>
<p>Sequencing of ‘Perennial’, ‘Dempsey’, and 120 RILs was as described by Kim et al.<sup><a reveal-id="DSV038C24" data-open="DSV038C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Parental lines were sequenced to 18× coverage and RILs were sequenced to 1× coverage. All sequence data are available under the Sequence Read Archive of NCBI (PRJNA298503). To redo SNP calling, sequencing reads were aligned to <em>C. annuum</em> ‘CM334’ chromosome v.1.55 using BWA version 0.7.12 with default parameters, and SNPs were called using GATK UnifiedGenotyper. High-quality SNPs with minimum sequencing depth 3 for each RIL, quality score 30, and called in at least 12 RILs were selected for further analysis.</p><p>A slightly modified sliding window approach<sup><a reveal-id="DSV038C17" data-open="DSV038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> was used to investigate recombination breakpoints and construct a bin map of RILs. The genotype of each window was called with a window size of 5 Mb and step size of 0.5 Mb. The ratio of SNPs with ‘Perennial’ and ‘Dempsey’ genotypes was calculated. When &gt;70% of SNPs had one parental genotype, the window was called as homozygous; otherwise, the window was called as heterozygous. Recombination breakpoints of each RIL were determined by the physical locations where the genotype of window changes. On the basis of the recombination breakpoint position, a physical bin map was constructed as described by Huang et al.<sup><a reveal-id="DSV038C17" data-open="DSV038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Briefly, when 120 RILs did not have the recombination breakpoints in 100 kb interval, these regions were combined to one bin. While there are more than two recombination breakpoints in 100 kb interval, they were determined as one breakpoint. To prevent detection of excessive crossovers, further processing was performed. When short heterozygous regions were located in the middle of consecutive ‘Perennial’ or ‘Dempsey’ genotype regions in the raw bin map and these heterozygous regions were shorter than window size, the genotype was manually corrected to match with the adjacent genotype. Bins were used as markers to construct an ultra-high-density genetic map using the Carthagene program<sup><a reveal-id="DSV038C21" data-open="DSV038C21" class="link link-ref link-reveal xref-bibr">21</a></sup> with default threshold. The Kosambi mapping function was used to calculate the distance (cM) of each bin. Twelve linkage groups were drawn with the MapChart 2.2 program.<sup><a reveal-id="DSV038C22" data-open="DSV038C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The physical map and genetic map were compared using Marker Browser developed by the Phyzen Genomics Institute (Seoul, Korea).</p>                            <h3 scrollto-destination=87196824 id="87196824" class="section-title" data-legacy-id=s2d>2.4. Anchoring of unassigned scaffolds</h3>
<p>Unmapped reads of ‘Perennial’, ‘Dempsey’, and 120 RILs were aligned to ‘CM334’ unassigned scaffolds v.1.55 (<a class="link link-uri" href="http://peppergenome.snu.ac.kr/">http://peppergenome.snu.ac.kr/</a>, accessed 7 December 2015). The alignment and SNP calling pipeline was the same as that used for bin map construction. To assign the unassigned scaffolds, each scaffold was considered as one bin marker. The genotype of the scaffold in each RIL was determined based on the ratio between the SNPs and the ‘Perennial’ and ‘Dempsey’ genotypes. Genotyped scaffolds were mapped on the bin map using the Carthagene program<sup><a reveal-id="DSV038C21" data-open="DSV038C21" class="link link-ref link-reveal xref-bibr">21</a></sup> with an logarithm of odds (LOD) threshold of 5 and a distance threshold of 20 cM.</p>                            <h3 scrollto-destination=87196826 id="87196826" class="section-title" data-legacy-id=s2e>2.5. QTL analysis</h3>
<p>All phenotyping data were used for QTL analysis. Composite interval mapping was performed with Windows QTL Cartographer 2.5.<sup><a reveal-id="DSV038C25" data-open="DSV038C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Values for each trait in three to four locations were analysed separately to detect QTLs. The LOD threshold was determined by applying 500 permutation tests with 5% probability for each trait. The proportion of phenotypic variation explained by each QTL was estimated using <em>R</em><sup>2</sup> (%) value. The 95% confidence interval was used as the location of QTL, and QTLs detected in more than two different environments were considered as significant QTLs.</p>                            <h2 scrollto-destination=87196828 id="87196828" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87196829 id="87196829" class="section-title" data-legacy-id=s3a>3.1. Variation of morphological traits in RILs</h3>
<p>The maternal parent ‘Perennial’ produces tall plants, with thin and purple-striped stems, and small leaves and flowers. ‘Perennial’ fruits are small, erect, and pungent. The paternal parent ‘Dempsey’ has relatively short plants, thick and green stems, and large leaves and flowers. ‘Dempsey’ produces large bell-type fruits that are pendent and non-pungent. We evaluated a total of 18 horticultural traits at least three different environments (Table <a reveal-id="DSV038TB1" data-open="DSV038TB1" class="link link-reveal link-table xref-fig">1</a>), and all traits segregated in RILs generated from these two parents (Table <a reveal-id="DSV038TB2" data-open="DSV038TB2" class="link link-reveal link-table xref-fig">2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Plant height, plant width, and internode length were greatly affected by environmental conditions. The plant height of both parents and RILs was highest in environment 2012a, and lowest in environment 2011. Plant width and internode length of ‘Dempsey’ and RILs were widest and longest in environment 2012a. Immature fruit colour was segregated only in RILs grown in 2012b and 2014. Both parents had green immature fruit, whereas the RILs showed 1 : 1 segregation of light green and green colour. </p>                        <a id="87196831" scrollto-destination="87196831"></a>
<div content-id="DSV038TB2" class="table-modal"><div class="table-wrap"><div id="DSV038TB2" class="table-wrap-title" data-id="DSV038TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Phenotypic variation of ‘Perennial’, ‘Dempsey’, and RILs grown in four different environments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">2011<hr /> </th><th colspan="3">2012a<hr /> </th><th colspan="3">2012b<hr /> </th><th colspan="3">2014<hr /> </th></tr><tr><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th></tr></thead><tbody><tr><td colspan="13">Plant architecture </td></tr><tr><td> Plant height </td><td>132 </td><td>53.5 </td><td>139 ± 24.9<sup>a</sup> </td><td>208 ± 2.5 </td><td>128 ± 6.0 </td><td>165 ± 26.6 </td><td>157 ± 5.1 </td><td>89 ± 1.2 </td><td>130 ± 40.5 </td><td>161 ± 3.3 </td><td>60 ± 1.2 </td><td>126 ± 26.9 </td></tr><tr><td> Plant width </td><td>46 </td><td>30 </td><td>81 ± 12.4 </td><td>64 ± 6.0 </td><td>86 ± 2.5 </td><td>86 ± 17.8 </td><td>68 ± 5.0 </td><td>63 ± 1.2 </td><td>72 ± 9.8 </td><td>68 ± 5.2 </td><td>57 ± 6.2 </td><td>65 ± 9.8 </td></tr><tr><td> Main stem length </td><td>–<sup>b</sup> </td><td>– </td><td>– </td><td>41 ± 4.5 </td><td>20 ± 1.0 </td><td>25 ± 7.8 </td><td>33 ± 1.2 </td><td>21 ± 2.1 </td><td>23 ± 6.8 </td><td>38 ± 4.4 </td><td>23 ± 0.6 </td><td>25 ± 7.0 </td></tr><tr><td> Stem thickness </td><td>– </td><td>– </td><td>– </td><td>2 ± 0.2 </td><td>3 ± 0.4 </td><td>2 ± 0.3 </td><td>1 ± 0.0 </td><td>2 ± 0.0 </td><td>1 ± 0.2 </td><td>1 ± 0.0 </td><td>1 ± 0.0 </td><td>1 ± 0.1 </td></tr><tr><td> Lateral branch number </td><td>– </td><td>– </td><td>– </td><td>22 ± 1.5 </td><td>12 ± 1.0 </td><td>13 ± 2.3 </td><td>20 ± 0.8 </td><td>7 ± 0.5 </td><td>13 ± 2.9 </td><td>19 ± 2.1 </td><td>14 ± 0.8 </td><td>13 ± 3.0 </td></tr><tr><td> Internode length </td><td>– </td><td>– </td><td>– </td><td>9 ± 2.3 </td><td>10 ± 2.3 </td><td>9 ± 2.4 </td><td>16 ± 0.5 </td><td>4 ± 1.1 </td><td>6 ± 2.4 </td><td>10 ± 2.1 </td><td>6 ± 0.5 </td><td>8 ± 2.3 </td></tr><tr><td> Stem colour </td><td>2 </td><td>1 </td><td>1 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td></tr><tr><td colspan="13">Leaf </td></tr><tr><td> Leaf length </td><td>12.3 </td><td>12.8 </td><td>8 ± 1.6 </td><td>12 ± 0.5 </td><td>18 ± 1.0 </td><td>11 ± 1.2 </td><td>8 ± 0.1 </td><td>14 ± 0.8 </td><td>11 ± 1.5 </td><td>12 ± 0.2 </td><td>16 ± 0.6 </td><td>13 ± 1.6 </td></tr><tr><td> Leaf width </td><td>4.7 </td><td>8.4 </td><td>4 ± 1.0 </td><td>7 ± 0.3 </td><td>10 ± 0.0 </td><td>7 ± 0.9 </td><td>5 ± 0.3 </td><td>8 ± 0.4 </td><td>6 ± 1.0 </td><td>6 ± 0.1 </td><td>9 ± 0.7 </td><td>7 ± 1.0 </td></tr><tr><td colspan="13">Flower </td></tr><tr><td> Flower size </td><td>2 </td><td>3 </td><td>2 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td></tr><tr><td> Stamen number </td><td>6 </td><td>6 </td><td>5 ± 0.5 </td><td>– </td><td>– </td><td>– </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td></tr><tr><td colspan="13">Fruit </td></tr><tr><td> Fruit length </td><td>1.8 </td><td>5.8 </td><td>7 ± 1.6 </td><td>– </td><td>– </td><td>6 ± 1.9 </td><td>3 ± 0.3 </td><td>8 ± 0.3 </td><td>6 ± 1.7 </td><td>3 ± 0.1 </td><td>8 ± 0.4 </td><td>6 ± 1.8 </td></tr><tr><td> Fruit diameter </td><td>1.8 </td><td>5.3 </td><td>2 ± 0.8 </td><td>– </td><td>– </td><td>2 ± 0.8 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td></tr><tr><td> Fruit shape </td><td>1 </td><td>1.1 </td><td>3 ± 1.2 </td><td>– </td><td>– </td><td>3 ± 1.4 </td><td>4 </td><td>1 </td><td>3 ± 1.5 </td><td>4 </td><td>1 </td><td>3 ± 1.4 </td></tr><tr><td> Fruit weight </td><td>1.4 </td><td>37.5 </td><td>11 ± 7.2 </td><td>1 </td><td>90 </td><td>10 ± 6.4 </td><td>1 ± 0.0 </td><td>115 ± 23.2 </td><td>7 ± 4.0 </td><td>1 ± 0.0 </td><td>116 ± 9.6 </td><td>6 ± 3.5 </td></tr><tr><td> Fruit position </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td></tr><tr><td> Calyx shape </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td></tr><tr><td> Immature fruit colour </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td></tr></tbody></table></div><div content-id="DSV038TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">2011<hr /> </th><th colspan="3">2012a<hr /> </th><th colspan="3">2012b<hr /> </th><th colspan="3">2014<hr /> </th></tr><tr><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th></tr></thead><tbody><tr><td colspan="13">Plant architecture </td></tr><tr><td> Plant height </td><td>132 </td><td>53.5 </td><td>139 ± 24.9<sup>a</sup> </td><td>208 ± 2.5 </td><td>128 ± 6.0 </td><td>165 ± 26.6 </td><td>157 ± 5.1 </td><td>89 ± 1.2 </td><td>130 ± 40.5 </td><td>161 ± 3.3 </td><td>60 ± 1.2 </td><td>126 ± 26.9 </td></tr><tr><td> Plant width </td><td>46 </td><td>30 </td><td>81 ± 12.4 </td><td>64 ± 6.0 </td><td>86 ± 2.5 </td><td>86 ± 17.8 </td><td>68 ± 5.0 </td><td>63 ± 1.2 </td><td>72 ± 9.8 </td><td>68 ± 5.2 </td><td>57 ± 6.2 </td><td>65 ± 9.8 </td></tr><tr><td> Main stem length </td><td>–<sup>b</sup> </td><td>– </td><td>– </td><td>41 ± 4.5 </td><td>20 ± 1.0 </td><td>25 ± 7.8 </td><td>33 ± 1.2 </td><td>21 ± 2.1 </td><td>23 ± 6.8 </td><td>38 ± 4.4 </td><td>23 ± 0.6 </td><td>25 ± 7.0 </td></tr><tr><td> Stem thickness </td><td>– </td><td>– </td><td>– </td><td>2 ± 0.2 </td><td>3 ± 0.4 </td><td>2 ± 0.3 </td><td>1 ± 0.0 </td><td>2 ± 0.0 </td><td>1 ± 0.2 </td><td>1 ± 0.0 </td><td>1 ± 0.0 </td><td>1 ± 0.1 </td></tr><tr><td> Lateral branch number </td><td>– </td><td>– </td><td>– </td><td>22 ± 1.5 </td><td>12 ± 1.0 </td><td>13 ± 2.3 </td><td>20 ± 0.8 </td><td>7 ± 0.5 </td><td>13 ± 2.9 </td><td>19 ± 2.1 </td><td>14 ± 0.8 </td><td>13 ± 3.0 </td></tr><tr><td> Internode length </td><td>– </td><td>– </td><td>– </td><td>9 ± 2.3 </td><td>10 ± 2.3 </td><td>9 ± 2.4 </td><td>16 ± 0.5 </td><td>4 ± 1.1 </td><td>6 ± 2.4 </td><td>10 ± 2.1 </td><td>6 ± 0.5 </td><td>8 ± 2.3 </td></tr><tr><td> Stem colour </td><td>2 </td><td>1 </td><td>1 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td></tr><tr><td colspan="13">Leaf </td></tr><tr><td> Leaf length </td><td>12.3 </td><td>12.8 </td><td>8 ± 1.6 </td><td>12 ± 0.5 </td><td>18 ± 1.0 </td><td>11 ± 1.2 </td><td>8 ± 0.1 </td><td>14 ± 0.8 </td><td>11 ± 1.5 </td><td>12 ± 0.2 </td><td>16 ± 0.6 </td><td>13 ± 1.6 </td></tr><tr><td> Leaf width </td><td>4.7 </td><td>8.4 </td><td>4 ± 1.0 </td><td>7 ± 0.3 </td><td>10 ± 0.0 </td><td>7 ± 0.9 </td><td>5 ± 0.3 </td><td>8 ± 0.4 </td><td>6 ± 1.0 </td><td>6 ± 0.1 </td><td>9 ± 0.7 </td><td>7 ± 1.0 </td></tr><tr><td colspan="13">Flower </td></tr><tr><td> Flower size </td><td>2 </td><td>3 </td><td>2 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td></tr><tr><td> Stamen number </td><td>6 </td><td>6 </td><td>5 ± 0.5 </td><td>– </td><td>– </td><td>– </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td></tr><tr><td colspan="13">Fruit </td></tr><tr><td> Fruit length </td><td>1.8 </td><td>5.8 </td><td>7 ± 1.6 </td><td>– </td><td>– </td><td>6 ± 1.9 </td><td>3 ± 0.3 </td><td>8 ± 0.3 </td><td>6 ± 1.7 </td><td>3 ± 0.1 </td><td>8 ± 0.4 </td><td>6 ± 1.8 </td></tr><tr><td> Fruit diameter </td><td>1.8 </td><td>5.3 </td><td>2 ± 0.8 </td><td>– </td><td>– </td><td>2 ± 0.8 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td></tr><tr><td> Fruit shape </td><td>1 </td><td>1.1 </td><td>3 ± 1.2 </td><td>– </td><td>– </td><td>3 ± 1.4 </td><td>4 </td><td>1 </td><td>3 ± 1.5 </td><td>4 </td><td>1 </td><td>3 ± 1.4 </td></tr><tr><td> Fruit weight </td><td>1.4 </td><td>37.5 </td><td>11 ± 7.2 </td><td>1 </td><td>90 </td><td>10 ± 6.4 </td><td>1 ± 0.0 </td><td>115 ± 23.2 </td><td>7 ± 4.0 </td><td>1 ± 0.0 </td><td>116 ± 9.6 </td><td>6 ± 3.5 </td></tr><tr><td> Fruit position </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td></tr><tr><td> Calyx shape </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td></tr><tr><td> Immature fruit colour </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>P, Perennial; D, Dempsey.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Mean ± standard deviation.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Phenotypes were not evaluated.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196831" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV038TB2" class="table-wrap-title" data-id="DSV038TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Phenotypic variation of ‘Perennial’, ‘Dempsey’, and RILs grown in four different environments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">2011<hr /> </th><th colspan="3">2012a<hr /> </th><th colspan="3">2012b<hr /> </th><th colspan="3">2014<hr /> </th></tr><tr><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th></tr></thead><tbody><tr><td colspan="13">Plant architecture </td></tr><tr><td> Plant height </td><td>132 </td><td>53.5 </td><td>139 ± 24.9<sup>a</sup> </td><td>208 ± 2.5 </td><td>128 ± 6.0 </td><td>165 ± 26.6 </td><td>157 ± 5.1 </td><td>89 ± 1.2 </td><td>130 ± 40.5 </td><td>161 ± 3.3 </td><td>60 ± 1.2 </td><td>126 ± 26.9 </td></tr><tr><td> Plant width </td><td>46 </td><td>30 </td><td>81 ± 12.4 </td><td>64 ± 6.0 </td><td>86 ± 2.5 </td><td>86 ± 17.8 </td><td>68 ± 5.0 </td><td>63 ± 1.2 </td><td>72 ± 9.8 </td><td>68 ± 5.2 </td><td>57 ± 6.2 </td><td>65 ± 9.8 </td></tr><tr><td> Main stem length </td><td>–<sup>b</sup> </td><td>– </td><td>– </td><td>41 ± 4.5 </td><td>20 ± 1.0 </td><td>25 ± 7.8 </td><td>33 ± 1.2 </td><td>21 ± 2.1 </td><td>23 ± 6.8 </td><td>38 ± 4.4 </td><td>23 ± 0.6 </td><td>25 ± 7.0 </td></tr><tr><td> Stem thickness </td><td>– </td><td>– </td><td>– </td><td>2 ± 0.2 </td><td>3 ± 0.4 </td><td>2 ± 0.3 </td><td>1 ± 0.0 </td><td>2 ± 0.0 </td><td>1 ± 0.2 </td><td>1 ± 0.0 </td><td>1 ± 0.0 </td><td>1 ± 0.1 </td></tr><tr><td> Lateral branch number </td><td>– </td><td>– </td><td>– </td><td>22 ± 1.5 </td><td>12 ± 1.0 </td><td>13 ± 2.3 </td><td>20 ± 0.8 </td><td>7 ± 0.5 </td><td>13 ± 2.9 </td><td>19 ± 2.1 </td><td>14 ± 0.8 </td><td>13 ± 3.0 </td></tr><tr><td> Internode length </td><td>– </td><td>– </td><td>– </td><td>9 ± 2.3 </td><td>10 ± 2.3 </td><td>9 ± 2.4 </td><td>16 ± 0.5 </td><td>4 ± 1.1 </td><td>6 ± 2.4 </td><td>10 ± 2.1 </td><td>6 ± 0.5 </td><td>8 ± 2.3 </td></tr><tr><td> Stem colour </td><td>2 </td><td>1 </td><td>1 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td></tr><tr><td colspan="13">Leaf </td></tr><tr><td> Leaf length </td><td>12.3 </td><td>12.8 </td><td>8 ± 1.6 </td><td>12 ± 0.5 </td><td>18 ± 1.0 </td><td>11 ± 1.2 </td><td>8 ± 0.1 </td><td>14 ± 0.8 </td><td>11 ± 1.5 </td><td>12 ± 0.2 </td><td>16 ± 0.6 </td><td>13 ± 1.6 </td></tr><tr><td> Leaf width </td><td>4.7 </td><td>8.4 </td><td>4 ± 1.0 </td><td>7 ± 0.3 </td><td>10 ± 0.0 </td><td>7 ± 0.9 </td><td>5 ± 0.3 </td><td>8 ± 0.4 </td><td>6 ± 1.0 </td><td>6 ± 0.1 </td><td>9 ± 0.7 </td><td>7 ± 1.0 </td></tr><tr><td colspan="13">Flower </td></tr><tr><td> Flower size </td><td>2 </td><td>3 </td><td>2 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td></tr><tr><td> Stamen number </td><td>6 </td><td>6 </td><td>5 ± 0.5 </td><td>– </td><td>– </td><td>– </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td></tr><tr><td colspan="13">Fruit </td></tr><tr><td> Fruit length </td><td>1.8 </td><td>5.8 </td><td>7 ± 1.6 </td><td>– </td><td>– </td><td>6 ± 1.9 </td><td>3 ± 0.3 </td><td>8 ± 0.3 </td><td>6 ± 1.7 </td><td>3 ± 0.1 </td><td>8 ± 0.4 </td><td>6 ± 1.8 </td></tr><tr><td> Fruit diameter </td><td>1.8 </td><td>5.3 </td><td>2 ± 0.8 </td><td>– </td><td>– </td><td>2 ± 0.8 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td></tr><tr><td> Fruit shape </td><td>1 </td><td>1.1 </td><td>3 ± 1.2 </td><td>– </td><td>– </td><td>3 ± 1.4 </td><td>4 </td><td>1 </td><td>3 ± 1.5 </td><td>4 </td><td>1 </td><td>3 ± 1.4 </td></tr><tr><td> Fruit weight </td><td>1.4 </td><td>37.5 </td><td>11 ± 7.2 </td><td>1 </td><td>90 </td><td>10 ± 6.4 </td><td>1 ± 0.0 </td><td>115 ± 23.2 </td><td>7 ± 4.0 </td><td>1 ± 0.0 </td><td>116 ± 9.6 </td><td>6 ± 3.5 </td></tr><tr><td> Fruit position </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td></tr><tr><td> Calyx shape </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td></tr><tr><td> Immature fruit colour </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td></tr></tbody></table></div><div content-id="DSV038TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">2011<hr /> </th><th colspan="3">2012a<hr /> </th><th colspan="3">2012b<hr /> </th><th colspan="3">2014<hr /> </th></tr><tr><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th><th>P </th><th>D </th><th>RIL </th></tr></thead><tbody><tr><td colspan="13">Plant architecture </td></tr><tr><td> Plant height </td><td>132 </td><td>53.5 </td><td>139 ± 24.9<sup>a</sup> </td><td>208 ± 2.5 </td><td>128 ± 6.0 </td><td>165 ± 26.6 </td><td>157 ± 5.1 </td><td>89 ± 1.2 </td><td>130 ± 40.5 </td><td>161 ± 3.3 </td><td>60 ± 1.2 </td><td>126 ± 26.9 </td></tr><tr><td> Plant width </td><td>46 </td><td>30 </td><td>81 ± 12.4 </td><td>64 ± 6.0 </td><td>86 ± 2.5 </td><td>86 ± 17.8 </td><td>68 ± 5.0 </td><td>63 ± 1.2 </td><td>72 ± 9.8 </td><td>68 ± 5.2 </td><td>57 ± 6.2 </td><td>65 ± 9.8 </td></tr><tr><td> Main stem length </td><td>–<sup>b</sup> </td><td>– </td><td>– </td><td>41 ± 4.5 </td><td>20 ± 1.0 </td><td>25 ± 7.8 </td><td>33 ± 1.2 </td><td>21 ± 2.1 </td><td>23 ± 6.8 </td><td>38 ± 4.4 </td><td>23 ± 0.6 </td><td>25 ± 7.0 </td></tr><tr><td> Stem thickness </td><td>– </td><td>– </td><td>– </td><td>2 ± 0.2 </td><td>3 ± 0.4 </td><td>2 ± 0.3 </td><td>1 ± 0.0 </td><td>2 ± 0.0 </td><td>1 ± 0.2 </td><td>1 ± 0.0 </td><td>1 ± 0.0 </td><td>1 ± 0.1 </td></tr><tr><td> Lateral branch number </td><td>– </td><td>– </td><td>– </td><td>22 ± 1.5 </td><td>12 ± 1.0 </td><td>13 ± 2.3 </td><td>20 ± 0.8 </td><td>7 ± 0.5 </td><td>13 ± 2.9 </td><td>19 ± 2.1 </td><td>14 ± 0.8 </td><td>13 ± 3.0 </td></tr><tr><td> Internode length </td><td>– </td><td>– </td><td>– </td><td>9 ± 2.3 </td><td>10 ± 2.3 </td><td>9 ± 2.4 </td><td>16 ± 0.5 </td><td>4 ± 1.1 </td><td>6 ± 2.4 </td><td>10 ± 2.1 </td><td>6 ± 0.5 </td><td>8 ± 2.3 </td></tr><tr><td> Stem colour </td><td>2 </td><td>1 </td><td>1 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td><td>2 </td><td>1 </td><td>2 ± 0.8 </td></tr><tr><td colspan="13">Leaf </td></tr><tr><td> Leaf length </td><td>12.3 </td><td>12.8 </td><td>8 ± 1.6 </td><td>12 ± 0.5 </td><td>18 ± 1.0 </td><td>11 ± 1.2 </td><td>8 ± 0.1 </td><td>14 ± 0.8 </td><td>11 ± 1.5 </td><td>12 ± 0.2 </td><td>16 ± 0.6 </td><td>13 ± 1.6 </td></tr><tr><td> Leaf width </td><td>4.7 </td><td>8.4 </td><td>4 ± 1.0 </td><td>7 ± 0.3 </td><td>10 ± 0.0 </td><td>7 ± 0.9 </td><td>5 ± 0.3 </td><td>8 ± 0.4 </td><td>6 ± 1.0 </td><td>6 ± 0.1 </td><td>9 ± 0.7 </td><td>7 ± 1.0 </td></tr><tr><td colspan="13">Flower </td></tr><tr><td> Flower size </td><td>2 </td><td>3 </td><td>2 ± 0.4 </td><td>– </td><td>– </td><td>– </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td><td>1 </td><td>3 </td><td>2 ± 0.5 </td></tr><tr><td> Stamen number </td><td>6 </td><td>6 </td><td>5 ± 0.5 </td><td>– </td><td>– </td><td>– </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td><td>5 </td><td>6 </td><td>5 ± 0.5 </td></tr><tr><td colspan="13">Fruit </td></tr><tr><td> Fruit length </td><td>1.8 </td><td>5.8 </td><td>7 ± 1.6 </td><td>– </td><td>– </td><td>6 ± 1.9 </td><td>3 ± 0.3 </td><td>8 ± 0.3 </td><td>6 ± 1.7 </td><td>3 ± 0.1 </td><td>8 ± 0.4 </td><td>6 ± 1.8 </td></tr><tr><td> Fruit diameter </td><td>1.8 </td><td>5.3 </td><td>2 ± 0.8 </td><td>– </td><td>– </td><td>2 ± 0.8 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td><td>1 ± 0.0 </td><td>8 ± 0.1 </td><td>2 ± 0.6 </td></tr><tr><td> Fruit shape </td><td>1 </td><td>1.1 </td><td>3 ± 1.2 </td><td>– </td><td>– </td><td>3 ± 1.4 </td><td>4 </td><td>1 </td><td>3 ± 1.5 </td><td>4 </td><td>1 </td><td>3 ± 1.4 </td></tr><tr><td> Fruit weight </td><td>1.4 </td><td>37.5 </td><td>11 ± 7.2 </td><td>1 </td><td>90 </td><td>10 ± 6.4 </td><td>1 ± 0.0 </td><td>115 ± 23.2 </td><td>7 ± 4.0 </td><td>1 ± 0.0 </td><td>116 ± 9.6 </td><td>6 ± 3.5 </td></tr><tr><td> Fruit position </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td><td>1 </td><td>3 </td><td>2 ± 0.9 </td></tr><tr><td> Calyx shape </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.6 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td><td>1 </td><td>3 </td><td>2 ± 0.8 </td></tr><tr><td> Immature fruit colour </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>2 ± 0.0 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td><td>2 </td><td>2 </td><td>1 ± 0.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>P, Perennial; D, Dempsey.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Mean ± standard deviation.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Phenotypes were not evaluated.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196831" target="_blank">View Large</a></div></div><p>Positive and negative correlations were detected between evaluated traits (Fig. <a reveal-id="DSV038F1" data-open="DSV038F1" class="link link-reveal link-table xref-fig">1</a>). In particular, fruit-related traits were highly correlated with each other compared with other traits. Fruit weight, fruit diameter, and calyx shape were positively correlated. As the fruit shape can be represented by fruit length divided by fruit diameter, it has positive and negative correlation with fruit length and fruit diameter, respectively. Flower size was positively correlated with fruit diameter and fruit weight, and negatively with main stem length. Phenotypic correlation in the parents and RILs indicated that one gene or closely linked genes control multiple fruit-related traits. </p>                        <a id="87196833" scrollto-destination="87196833"></a>
<div id="DSV038F1" data-id="DSV038F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/m_dsv03801.jpeg?Expires=2147483647&Signature=Fe09tdzu333pRYxQgyiRbXXQYnXnIFp2glJ~in1UXqJc45azCYGDWQ~a7xe1wPW06leHiOelSjq1OFbK0fM3MdL83YPa7FMwEc07FrtV9ydLhRngzQ3wJTw8oFItPfLpQG8zSexRUaDecrTfRKm-960OvWXVp317Kaz0dKRQMNMwRVRexbwHq8D16rCwWrbDr20vq49jhs8yULVycvEkZftwPn-0NjcJYc7GpWB8siXl3Zr-j2P-eQJ3zfVfa~5cfkdqAHcj4RLVJ4dHKgc-c4mCDHtu6KUz~A2T7WLcLCZmIyjVzkylnj09TzAa4fHHy0hnnsfrXxhfsTiEKfVF2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between morphological traits evaluated in the RIL population. Average phenotypic values from four environments were used for Pearson correlation test. Red and blue blocks show positive and negative correlation, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196833/dsv03801.jpeg" path-from-xml="dsv03801.jpeg" target="_blank">View large</a><a section="87196833" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv03801.jpeg?Expires=2147483647&Signature=adYjAbfzBEUmqy9Lcg~cJ49IDgyusPFNLE1ieMsIsHX~~qLr2DSTl2dDY9C8LGYJBEZFwbNzekrfa7Eou857a1NfUewYo30u3t74a0pUU1OX7zyADv6WwTb9GVVWU-xOcW2G~yEqmsMA1lOhE1ByBHaL0P~-4XcytG7Qlh1~BnUYS-wM4ROrLhdQYGyHTJz4-CR9rWt3gw7VyguuKkHrILPHX3O-kpDvHP9Kkd2EGtjM~d~e-LEtumTpoE83Xa7D8S67Vn5BD15rBPAMeND2hCnnMj5t5rYtjJ0GLEAT1IIqwu-fUhSf8IOrqoE7HMs-hA7Fiw15lr4-HcFrGG9fMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196833&ar=1745287&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between morphological traits evaluated in the RIL population. Average phenotypic values from four environments were used for Pearson correlation test. Red and blue blocks show positive and negative correlation, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV038F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/m_dsv03801.jpeg?Expires=2147483647&Signature=Fe09tdzu333pRYxQgyiRbXXQYnXnIFp2glJ~in1UXqJc45azCYGDWQ~a7xe1wPW06leHiOelSjq1OFbK0fM3MdL83YPa7FMwEc07FrtV9ydLhRngzQ3wJTw8oFItPfLpQG8zSexRUaDecrTfRKm-960OvWXVp317Kaz0dKRQMNMwRVRexbwHq8D16rCwWrbDr20vq49jhs8yULVycvEkZftwPn-0NjcJYc7GpWB8siXl3Zr-j2P-eQJ3zfVfa~5cfkdqAHcj4RLVJ4dHKgc-c4mCDHtu6KUz~A2T7WLcLCZmIyjVzkylnj09TzAa4fHHy0hnnsfrXxhfsTiEKfVF2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between morphological traits evaluated in the RIL population. Average phenotypic values from four environments were used for Pearson correlation test. Red and blue blocks show positive and negative correlation, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196833/dsv03801.jpeg" path-from-xml="dsv03801.jpeg" target="_blank">View large</a><a section="87196833" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv03801.jpeg?Expires=2147483647&Signature=adYjAbfzBEUmqy9Lcg~cJ49IDgyusPFNLE1ieMsIsHX~~qLr2DSTl2dDY9C8LGYJBEZFwbNzekrfa7Eou857a1NfUewYo30u3t74a0pUU1OX7zyADv6WwTb9GVVWU-xOcW2G~yEqmsMA1lOhE1ByBHaL0P~-4XcytG7Qlh1~BnUYS-wM4ROrLhdQYGyHTJz4-CR9rWt3gw7VyguuKkHrILPHX3O-kpDvHP9Kkd2EGtjM~d~e-LEtumTpoE83Xa7D8S67Vn5BD15rBPAMeND2hCnnMj5t5rYtjJ0GLEAT1IIqwu-fUhSf8IOrqoE7HMs-hA7Fiw15lr4-HcFrGG9fMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196833&ar=1745287&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between morphological traits evaluated in the RIL population. Average phenotypic values from four environments were used for Pearson correlation test. Red and blue blocks show positive and negative correlation, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87196834 id="87196834" class="section-title" data-legacy-id=s3b>3.2. Construction of bin map using low-coverage sequence</h3>
<p>‘Perennial’ and ‘Dempsey’ were sequenced with 18× coverage, whereas RILs were sequenced with 1× coverage. RILs had large amounts of missing data in SNP regions and the reliability of individual SNPs was very low due to the low coverage. Therefore, to construct an accurate ultra-high-density genetic map efficiently, we implemented a modified sliding window approach.<sup><a reveal-id="DSV038C17" data-open="DSV038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Consecutive SNPs were joined in one bin, and the genotype of each bin was determined based on the ratio between SNPs from the two parents. Bins instead of individual SNPs were used as markers to increase mapping efficiency. A total of 1,431,214 SNPs were detected between ‘Perennial’ and ‘Dempsey’ and used to construct a bin map (Table <a reveal-id="DSV038TB3" data-open="DSV038TB3" class="link link-reveal link-table xref-fig">3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). </p>                        <a id="87196836" scrollto-destination="87196836"></a>
<div content-id="DSV038TB3" class="table-modal"><div class="table-wrap"><div id="DSV038TB3" class="table-wrap-title" data-id="DSV038TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Comparison of physical length and genetic distance in the bin map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr. </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of bins </th><th colspan="2">Physical length of bin (Mb)<hr /> </th><th colspan="2">Genetic distance of bin (cM)<hr /> </th></tr><tr><th>Mean </th><th>Total </th><th>Mean </th><th>Total </th></tr></thead><tbody><tr><td>1 </td><td>82,966 </td><td>370 </td><td>0.74 </td><td>272.6 </td><td>0.56 </td><td>208.5 </td></tr><tr><td>2 </td><td>80,141 </td><td>195 </td><td>0.88 </td><td>171.1 </td><td>0.55 </td><td>107.5 </td></tr><tr><td>3 </td><td>87,793 </td><td>261 </td><td>0.99 </td><td>257.9 </td><td>0.45 </td><td>118.5 </td></tr><tr><td>4 </td><td>54,657 </td><td>216 </td><td>1.03 </td><td>222.5 </td><td>0.54 </td><td>116.5 </td></tr><tr><td>5 </td><td>82,413 </td><td>190 </td><td>1.23 </td><td>233.4 </td><td>0.53 </td><td>100.6 </td></tr><tr><td>6 </td><td>107,015 </td><td>220 </td><td>1.08 </td><td>236.9 </td><td>0.47 </td><td>102.6 </td></tr><tr><td>7 </td><td>84,339 </td><td>175 </td><td>1.33 </td><td>231.9 </td><td>0.53 </td><td>92.5 </td></tr><tr><td>8 </td><td>24,383 </td><td>217 </td><td>0.67 </td><td>144.8 </td><td>0.71 </td><td>153.7 </td></tr><tr><td>9 </td><td>275,842 </td><td>161 </td><td>1.57 </td><td>252.7 </td><td>0.54 </td><td>86.2 </td></tr><tr><td>10 </td><td>230,360 </td><td>154 </td><td>1.52 </td><td>233.6 </td><td>0.67 </td><td>103.9 </td></tr><tr><td>11 </td><td>252,765 </td><td>196 </td><td>1.33 </td><td>259.7 </td><td>0.44 </td><td>86.8 </td></tr><tr><td>12 </td><td>68,540 </td><td>223 </td><td>1.06 </td><td>235.7 </td><td>0.43 </td><td>94.9 </td></tr><tr><td>Total </td><td>1,431,214 </td><td>2,578 </td><td>1.07 </td><td>2,752.8 </td><td>0.53 </td><td>1,372.2 </td></tr></tbody></table></div><div content-id="DSV038TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr. </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of bins </th><th colspan="2">Physical length of bin (Mb)<hr /> </th><th colspan="2">Genetic distance of bin (cM)<hr /> </th></tr><tr><th>Mean </th><th>Total </th><th>Mean </th><th>Total </th></tr></thead><tbody><tr><td>1 </td><td>82,966 </td><td>370 </td><td>0.74 </td><td>272.6 </td><td>0.56 </td><td>208.5 </td></tr><tr><td>2 </td><td>80,141 </td><td>195 </td><td>0.88 </td><td>171.1 </td><td>0.55 </td><td>107.5 </td></tr><tr><td>3 </td><td>87,793 </td><td>261 </td><td>0.99 </td><td>257.9 </td><td>0.45 </td><td>118.5 </td></tr><tr><td>4 </td><td>54,657 </td><td>216 </td><td>1.03 </td><td>222.5 </td><td>0.54 </td><td>116.5 </td></tr><tr><td>5 </td><td>82,413 </td><td>190 </td><td>1.23 </td><td>233.4 </td><td>0.53 </td><td>100.6 </td></tr><tr><td>6 </td><td>107,015 </td><td>220 </td><td>1.08 </td><td>236.9 </td><td>0.47 </td><td>102.6 </td></tr><tr><td>7 </td><td>84,339 </td><td>175 </td><td>1.33 </td><td>231.9 </td><td>0.53 </td><td>92.5 </td></tr><tr><td>8 </td><td>24,383 </td><td>217 </td><td>0.67 </td><td>144.8 </td><td>0.71 </td><td>153.7 </td></tr><tr><td>9 </td><td>275,842 </td><td>161 </td><td>1.57 </td><td>252.7 </td><td>0.54 </td><td>86.2 </td></tr><tr><td>10 </td><td>230,360 </td><td>154 </td><td>1.52 </td><td>233.6 </td><td>0.67 </td><td>103.9 </td></tr><tr><td>11 </td><td>252,765 </td><td>196 </td><td>1.33 </td><td>259.7 </td><td>0.44 </td><td>86.8 </td></tr><tr><td>12 </td><td>68,540 </td><td>223 </td><td>1.06 </td><td>235.7 </td><td>0.43 </td><td>94.9 </td></tr><tr><td>Total </td><td>1,431,214 </td><td>2,578 </td><td>1.07 </td><td>2,752.8 </td><td>0.53 </td><td>1,372.2 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196836" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV038TB3" class="table-wrap-title" data-id="DSV038TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Comparison of physical length and genetic distance in the bin map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr. </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of bins </th><th colspan="2">Physical length of bin (Mb)<hr /> </th><th colspan="2">Genetic distance of bin (cM)<hr /> </th></tr><tr><th>Mean </th><th>Total </th><th>Mean </th><th>Total </th></tr></thead><tbody><tr><td>1 </td><td>82,966 </td><td>370 </td><td>0.74 </td><td>272.6 </td><td>0.56 </td><td>208.5 </td></tr><tr><td>2 </td><td>80,141 </td><td>195 </td><td>0.88 </td><td>171.1 </td><td>0.55 </td><td>107.5 </td></tr><tr><td>3 </td><td>87,793 </td><td>261 </td><td>0.99 </td><td>257.9 </td><td>0.45 </td><td>118.5 </td></tr><tr><td>4 </td><td>54,657 </td><td>216 </td><td>1.03 </td><td>222.5 </td><td>0.54 </td><td>116.5 </td></tr><tr><td>5 </td><td>82,413 </td><td>190 </td><td>1.23 </td><td>233.4 </td><td>0.53 </td><td>100.6 </td></tr><tr><td>6 </td><td>107,015 </td><td>220 </td><td>1.08 </td><td>236.9 </td><td>0.47 </td><td>102.6 </td></tr><tr><td>7 </td><td>84,339 </td><td>175 </td><td>1.33 </td><td>231.9 </td><td>0.53 </td><td>92.5 </td></tr><tr><td>8 </td><td>24,383 </td><td>217 </td><td>0.67 </td><td>144.8 </td><td>0.71 </td><td>153.7 </td></tr><tr><td>9 </td><td>275,842 </td><td>161 </td><td>1.57 </td><td>252.7 </td><td>0.54 </td><td>86.2 </td></tr><tr><td>10 </td><td>230,360 </td><td>154 </td><td>1.52 </td><td>233.6 </td><td>0.67 </td><td>103.9 </td></tr><tr><td>11 </td><td>252,765 </td><td>196 </td><td>1.33 </td><td>259.7 </td><td>0.44 </td><td>86.8 </td></tr><tr><td>12 </td><td>68,540 </td><td>223 </td><td>1.06 </td><td>235.7 </td><td>0.43 </td><td>94.9 </td></tr><tr><td>Total </td><td>1,431,214 </td><td>2,578 </td><td>1.07 </td><td>2,752.8 </td><td>0.53 </td><td>1,372.2 </td></tr></tbody></table></div><div content-id="DSV038TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr. </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of bins </th><th colspan="2">Physical length of bin (Mb)<hr /> </th><th colspan="2">Genetic distance of bin (cM)<hr /> </th></tr><tr><th>Mean </th><th>Total </th><th>Mean </th><th>Total </th></tr></thead><tbody><tr><td>1 </td><td>82,966 </td><td>370 </td><td>0.74 </td><td>272.6 </td><td>0.56 </td><td>208.5 </td></tr><tr><td>2 </td><td>80,141 </td><td>195 </td><td>0.88 </td><td>171.1 </td><td>0.55 </td><td>107.5 </td></tr><tr><td>3 </td><td>87,793 </td><td>261 </td><td>0.99 </td><td>257.9 </td><td>0.45 </td><td>118.5 </td></tr><tr><td>4 </td><td>54,657 </td><td>216 </td><td>1.03 </td><td>222.5 </td><td>0.54 </td><td>116.5 </td></tr><tr><td>5 </td><td>82,413 </td><td>190 </td><td>1.23 </td><td>233.4 </td><td>0.53 </td><td>100.6 </td></tr><tr><td>6 </td><td>107,015 </td><td>220 </td><td>1.08 </td><td>236.9 </td><td>0.47 </td><td>102.6 </td></tr><tr><td>7 </td><td>84,339 </td><td>175 </td><td>1.33 </td><td>231.9 </td><td>0.53 </td><td>92.5 </td></tr><tr><td>8 </td><td>24,383 </td><td>217 </td><td>0.67 </td><td>144.8 </td><td>0.71 </td><td>153.7 </td></tr><tr><td>9 </td><td>275,842 </td><td>161 </td><td>1.57 </td><td>252.7 </td><td>0.54 </td><td>86.2 </td></tr><tr><td>10 </td><td>230,360 </td><td>154 </td><td>1.52 </td><td>233.6 </td><td>0.67 </td><td>103.9 </td></tr><tr><td>11 </td><td>252,765 </td><td>196 </td><td>1.33 </td><td>259.7 </td><td>0.44 </td><td>86.8 </td></tr><tr><td>12 </td><td>68,540 </td><td>223 </td><td>1.06 </td><td>235.7 </td><td>0.43 </td><td>94.9 </td></tr><tr><td>Total </td><td>1,431,214 </td><td>2,578 </td><td>1.07 </td><td>2,752.8 </td><td>0.53 </td><td>1,372.2 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196836" target="_blank">View Large</a></div></div><p>To set appropriate parameters for the sliding window approach, window length from 1 to 6 Mb and step size from 0.2 to 0.5 Mb for all chromosomes were tested. Window length and step sizes of 5 and 0.5 Mb, and 4 and 0.4 Mb showed a reasonable number of recombination breakpoints per chromosome, whereas the other values showed too large or low number (data not shown). A window length of 5 Mb and a step size of 0.5 Mb gave the best match of the recombination breakpoints determined manually in five randomly selected RILs. Therefore, this parameter was used to construct a bin map.</p><p>Using a sliding window approach, 3,983 recombination breakpoints were identified from 120 RILs, with an average of 33.2 per RIL (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). For the RILs, the mean number of crossovers per chromosome per line was 2.8. All SNPs were grouped into 2,578 bins (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The average length of bins was 1.07 Mb and ranged from 100 kb to 141 Mb (Table <a reveal-id="DSV038TB3" data-open="DSV038TB3" class="link link-reveal link-table xref-fig">3</a>). The total genetic distance of the bin map was 1,372.2 cM, and the mean distance of the bins was 0.53 cM. On average, 564 SNPs were joined into each bin, and 54% of the bins contained &lt;100 SNPs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>). The proportion of heterozygous bins in the bin map was calculated to measure the heterozygosity of RILs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The average heterozygosity of the 120 RILs was 2.6%, whereas three RILs exhibited 31.8, 23.0, and 17.7% heterozygosity. Excluding these three RILs, the average heterozygosity of RILs in different generations was near 2.0%.</p><p>The locations of bins in the genetic map were compared with those of the physical map based on the ‘CM334’ reference genome (Fig. <a reveal-id="DSV038F2" data-open="DSV038F2" class="link link-reveal link-table xref-fig">2</a>). Inconsistencies of bin order were detected in all chromosomes and most of the inconsistencies were located near the putative centromeric regions. For example, bins located 20–40 and 60–70 Mb on chromosome 4 were inverted in location on the genetic map. Similar patterns were detected at 95–120 Mb on chromosome 3, 80–100 Mb on chromosome 7, 40–50 Mb on chromosome 9, and 110–130 Mb on chromosome 11. Extensive inconsistencies were detected on the upper region of chromosome 8. </p>                        <a id="87196840" scrollto-destination="87196840"></a>
<div id="DSV038F2" data-id="DSV038F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/m_dsv03802.jpeg?Expires=2147483647&Signature=BpDjw~0ZP87YWNJGWcqv12I~IttVmar4j-U0LdD~DQG8CL4pZuaYifXyRAw7TNTrQOYDB1ciiVyJWB5lTj-fC0-Uh5OQajctBWtV4ESQM2OV6dt0u-AzGJ4dDWXZzNUqYif0txguIQSt9URVb0fkKl-03BMe3DmDSOAMDwudobX4xhngPB5CuGLZ0QMI9fuA1G0kNQjeysVlnf5~tCM1fgq~DPW-sqFjMFX932stmKw9LrH7h9X1Gz4qqV56QhP-CIOgXYUfdAfDJM9OXmaRnw5ykkVVERjnsGaza2W51CPaY2b~Cwswb-fZURRm0pEoizDhO1wgSV-IPg~d2wRtBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of physical map and genetic map constructed with bins. Left and right map shows genetic and physical map, respectively. Markers are linked by grey lines between two maps. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196840/dsv03802.jpeg" path-from-xml="dsv03802.jpeg" target="_blank">View large</a><a section="87196840" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv03802.jpeg?Expires=2147483647&Signature=h14evj32SgaWGu5T-TpRAVIGTNm9r6DQgcHq1rObSMkQc~wBnYnp1fbGw0ZfHhUBk6R-10QGeuDQoEUzFv1ChmJ7FOhPp7d4CDkQX1US0l~4lSdOvZ-SiqotRTbofbZr9xZICt9QQiiK3L0d5N-xVtt5Y7BDkhlk9X1-j-DVcS~4BFWSjxO8uaBTw8o2NWvoh334Pw8VoIXI~KKj~RuEQI2xcjJGOrkhAmjn7RV9QYyRrLKzzhNbfRX~J4bbJudLNPRKaAdCPe35cX8hCev5PLS0LQHyOHZ2LC5Sb8oBTmefyCSF-mUaloy8Z-u-gQ9oTflMy-~Jab46XBp2NTTc9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196840&ar=1745287&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of physical map and genetic map constructed with bins. Left and right map shows genetic and physical map, respectively. Markers are linked by grey lines between two maps. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV038F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/m_dsv03802.jpeg?Expires=2147483647&Signature=BpDjw~0ZP87YWNJGWcqv12I~IttVmar4j-U0LdD~DQG8CL4pZuaYifXyRAw7TNTrQOYDB1ciiVyJWB5lTj-fC0-Uh5OQajctBWtV4ESQM2OV6dt0u-AzGJ4dDWXZzNUqYif0txguIQSt9URVb0fkKl-03BMe3DmDSOAMDwudobX4xhngPB5CuGLZ0QMI9fuA1G0kNQjeysVlnf5~tCM1fgq~DPW-sqFjMFX932stmKw9LrH7h9X1Gz4qqV56QhP-CIOgXYUfdAfDJM9OXmaRnw5ykkVVERjnsGaza2W51CPaY2b~Cwswb-fZURRm0pEoizDhO1wgSV-IPg~d2wRtBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of physical map and genetic map constructed with bins. Left and right map shows genetic and physical map, respectively. Markers are linked by grey lines between two maps. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196840/dsv03802.jpeg" path-from-xml="dsv03802.jpeg" target="_blank">View large</a><a section="87196840" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv03802.jpeg?Expires=2147483647&Signature=h14evj32SgaWGu5T-TpRAVIGTNm9r6DQgcHq1rObSMkQc~wBnYnp1fbGw0ZfHhUBk6R-10QGeuDQoEUzFv1ChmJ7FOhPp7d4CDkQX1US0l~4lSdOvZ-SiqotRTbofbZr9xZICt9QQiiK3L0d5N-xVtt5Y7BDkhlk9X1-j-DVcS~4BFWSjxO8uaBTw8o2NWvoh334Pw8VoIXI~KKj~RuEQI2xcjJGOrkhAmjn7RV9QYyRrLKzzhNbfRX~J4bbJudLNPRKaAdCPe35cX8hCev5PLS0LQHyOHZ2LC5Sb8oBTmefyCSF-mUaloy8Z-u-gQ9oTflMy-~Jab46XBp2NTTc9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196840&ar=1745287&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of physical map and genetic map constructed with bins. Left and right map shows genetic and physical map, respectively. Markers are linked by grey lines between two maps. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To confirm that the bin map was constructed properly, we mapped the <em>Pun1</em> gene to the bin map and compared the results with the physical location of the <em>Pun1</em> gene in ‘CM334’ reference genome. <em>Pun1</em> is the gene controlling pungency, and the recessive allele <em>pun1</em> confers non-pungency of pepper.<sup><a reveal-id="DSV038C26" data-open="DSV038C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Owing to the pungent parent ‘Perennial’ and non-pungent parent ‘Dempsey’, pungency in the RILs segregated to a 1 : 1 ratio (data not shown). The <em>Pun1</em> gene was mapped between PD2-bin139 and PD2-bin141 on chromosome 2 of the bin map. The physical location of PD2-bin139 and PD2-bin141 was 149.8 and 151.0 Mb, respectively. On the ‘CM334’ reference genome, the <em>Pun1</em> gene is located at 152.6 Mb, 1.6 Mb from the linked bin. Thus, although there was a slight difference from the physical location, the <em>Pun1</em> gene was mapped quite precisely.</p>                            <h3 scrollto-destination=87196882 id="87196882" class="section-title" data-legacy-id=s3c>3.3. QTL analysis</h3>
<p>In four environments, a total of 434 QTLs were detected, and those QTLs detected in more than two environments were selected as significant. QTLs controlling the same trait and commonly detected in different environments with overlapped 95% confidence intervals were consolidated to one QTL. A total of 86 significant QTLs controlling 17 traits were selected (Table <a reveal-id="DSV038TB4" data-open="DSV038TB4" class="link link-reveal link-table xref-fig">4</a> and Fig. <a reveal-id="DSV038F3" data-open="DSV038F3" class="link link-reveal link-table xref-fig">3</a>). Consolidated QTLs were named based on abbreviation of the trait name and the chromosome number. For each morphological trait, 2–10 QTLs were detected and these QTLs were distributed throughout the 12 pepper chromosomes. The phenotypic variation explained by each QTL ranged from 5.6 to 63.9%. Using a threshold <em>R</em><sup>2</sup> of 10%, 32 major QTLs were identified from 13 traits. <em>LBN-2.1</em> and <em>LBN-2.2</em>, for lateral branch number, were each detected in three environments, and their <em>R</em><sup>2</sup> value ranges were 16–40 and 10–23, respectively. These large variations in <em>R</em><sup>2</sup> value indicate that lateral branch number is highly affected by the environment. The <em>INL-2</em>, for internode length, and <em>FL-3.4</em>, for fruit length, were also each detected in three environments, but their variations of <em>R</em><sup>2</sup> were smaller compared with those of <em>LBN-2.1</em> and <em>LBN-2.2. FS-3.1</em>, for fruit shape, <em>FP-1</em>, and <em>FP-12.2</em>, for fruit position, were identified in all environments. In particular, <em>FP-12.2</em> could explain &gt;40% of phenotypic variation, which suggests the possible existence of a major gene controlling fruit position. </p>                        <a id="87196894" scrollto-destination="87196894"></a>
<div content-id="DSV038TB4" class="table-modal"><div class="table-wrap"><div id="DSV038TB4" class="table-wrap-title" data-id="DSV038TB4"><div class="label -label">Table 4.</div><div class="caption"><p>QTLs controlling morphological traits detected in more than two environments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>Year </th><th>QTL </th><th>Chr. </th><th>Location (cM) </th><th>LOD </th><th><em>R</em><sup>2</sup> (%) </th><th>Direction </th><th>Additive effect </th></tr></thead><tbody><tr><td rowspan="6">Plant height </td><td>2012a, 2014 </td><td><em>PH-2</em> </td><td>2 </td><td>92.7–100.9 </td><td>3.7–4.0 </td><td>11.1–11.8 </td><td>+ </td><td>9.2–10.0 </td></tr><tr><td>2011, 2014 </td><td><em>PH-4</em> </td><td>4 </td><td>96.2–102.3 </td><td>2.3–2.6 </td><td>7.8–10.0 </td><td>− </td><td>8.3–10.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>PH-6</em> </td><td>6 </td><td>63.2–80.2 </td><td>2.1–3.4 </td><td>6.4–11.6 </td><td>+ </td><td>7.8–12.8 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-7</em> </td><td>7 </td><td>77.3–90.9 </td><td>2.6–3.4 </td><td>8.3–11.2 </td><td>+ </td><td>9.5–13.8 </td></tr><tr><td>2012b, 2014 </td><td><em>PH-8.1</em> </td><td>8 </td><td>3.8–21.1 </td><td>2.6–2.8 </td><td>5.6–8.3 </td><td>+ </td><td>9.3–11.7 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-8.2</em> </td><td>8 </td><td>37.2–54.8 </td><td>2.4–2.9 </td><td>5.9–8.6 </td><td>− </td><td>10.7–13.8 </td></tr><tr><td rowspan="2">Plant width </td><td>2012b, 2014 </td><td><em>PW-2</em> </td><td>2 </td><td>34.5–44.9 </td><td>2.2–2.4 </td><td>7.1–8.0 </td><td>− </td><td>3.0–4.4 </td></tr><tr><td>2012b, 2014 </td><td><em>PW-5</em> </td><td>5 </td><td>33.3–41.2 </td><td>2.4–3.6 </td><td>8.6–12.5 </td><td>+ </td><td>3.5–6.0 </td></tr><tr><td rowspan="6">Main stem length </td><td>2012b, 2014 </td><td><em>MSL-7</em> </td><td>7 </td><td>47.2–51.5 </td><td>2.4–5.2 </td><td>8.8–18.6 </td><td>+ </td><td>2.6–4.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-8.1</em> </td><td>8 </td><td>84.2–94.9 </td><td>4.6–6.3 </td><td>14.0–18.3 </td><td>+ </td><td>3.3–4.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-8.2</em> </td><td>8 </td><td>111.1–133.2 </td><td>2.2–4.3 </td><td>7.4–13.4 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-10</em> </td><td>10 </td><td>58.4–62.6 </td><td>2.6–3.3 </td><td>8.7–12.3 </td><td>− </td><td>5.9–7.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-11</em> </td><td>11 </td><td>76.9–82.7 </td><td>2.6–2.7 </td><td>9.8–10.4 </td><td>− </td><td>2.8–3.0 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-12</em> </td><td>12 </td><td>57.8–63.0 </td><td>1.9–2.6 </td><td>6.5–9.9 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td rowspan="2">Lateral branch number </td><td>2012a, 2012b, 2014 </td><td><em>LBN-2.1</em> </td><td>2 </td><td>91.4–98.3 </td><td>4.6–12.5 </td><td>16.0–40.0 </td><td>+ </td><td>0.9–1.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LBN-2.2</em> </td><td>2 </td><td>99.7–104.5 </td><td>2.8–6.4 </td><td>10.1–23.1 </td><td>+ </td><td>0.7–1.4 </td></tr><tr><td rowspan="5">Internode length </td><td>2012a, 2014 </td><td><em>INL-1</em> </td><td>1 </td><td>28.8–32.9 </td><td>2.6–2.9 </td><td>9.2–11.0 </td><td>+ </td><td>0.8–0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>INL-2</em> </td><td>2 </td><td>82.3–90.8 </td><td>3.2–4.8 </td><td>11.0–16.9 </td><td>+ </td><td>0.8–1.0 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-6.1</em> </td><td>6 </td><td>0.0–4.9 </td><td>2.4–3.6 </td><td>7.5–11.9 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>INL-6.2</em> </td><td>6 </td><td>25.8–37.2 </td><td>2.2–3.1 </td><td>7.4–10.2 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-10</em> </td><td>10 </td><td>0.6–22.5 </td><td>2.1–2.4 </td><td>7.1–7.4 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td rowspan="6">Stem colour </td><td>2012b, 2014 </td><td><em>SC-2</em> </td><td>2 </td><td>17.6–22.3 </td><td>3.0 </td><td>11.4 </td><td>− </td><td>0.5 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-4</em> </td><td>4 </td><td>34.7–41.0 </td><td>2.6 </td><td>9.7 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-7</em> </td><td>7 </td><td>0.0–2.5 </td><td>3.2 </td><td>10.4 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>SC-9</em> </td><td>9 </td><td>51.8–59.6 </td><td>2.4–3.4 </td><td>9.3–10.3 </td><td>+ </td><td>0.1–0.6 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-11</em> </td><td>11 </td><td>24.3–28.9 </td><td>3.3 </td><td>11.6 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-12</em> </td><td>12 </td><td>61.1–71.1 </td><td>2.8 </td><td>11.1 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="4">Leaf length </td><td>2012b, 2014 </td><td><em>LL-6</em> </td><td>6 </td><td>42.0–52.7 </td><td>2.3–4.5 </td><td>8.7–16.6 </td><td>+ </td><td>0.6–0.7 </td></tr><tr><td>2012b, 2014 </td><td><em>LL-9</em> </td><td>9 </td><td>75.3–84.9 </td><td>2.2–6.2 </td><td>7.7–20.8 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>LL-11.1</em> </td><td>11 </td><td>47.3–57.1 </td><td>2.3–4.2 </td><td>7.9–16.1 </td><td>+ </td><td>0.4–1.0 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LL-11.2</em> </td><td>11 </td><td>65.3–82.4 </td><td>2.4–3.6 </td><td>8.0–12.3 </td><td>− </td><td>0.4–0.9 </td></tr><tr><td>Leaf width </td><td>2012a, 2014 </td><td><em>LW-8</em> </td><td>8 </td><td>56.7–70.6 </td><td>2.3–5.8 </td><td>9.8–18.2 </td><td>− </td><td>0.4–0.5 </td></tr><tr><td rowspan="8">Flower size </td><td>2012b, 2014 </td><td><em>FLS-1</em> </td><td>1 </td><td>113.7–115.9 </td><td>5.8 </td><td>19.6 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-2</em> </td><td>2 </td><td>60.2–63.9 </td><td>2.8 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.1</em> </td><td>3 </td><td>18.2–23.3 </td><td>3.7 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.2</em> </td><td>3 </td><td>70.7–74.7 </td><td>2.9 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.3</em> </td><td>3 </td><td>89.1–93.7 </td><td>2.6 </td><td>7.8 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-7</em> </td><td>7 </td><td>46.8–53.1 </td><td>2.4 </td><td>8.8 </td><td>− </td><td>0.1 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-8</em> </td><td>8 </td><td>89.7–96.6 </td><td>2.7 </td><td>12.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-10</em> </td><td>10 </td><td>57.0–58.4 </td><td>2.4 </td><td>8.7 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="3">Stamen number </td><td>2012b, 2014 </td><td><em>STN-3</em> </td><td>3 </td><td>108.3–115.8 </td><td>2.6 </td><td>9.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-10</em> </td><td>10 </td><td>99.8–102.6 </td><td>2.8 </td><td>10.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-11</em> </td><td>11 </td><td>31.0–36.2 </td><td>2.3 </td><td>8.5 </td><td>− </td><td>0.2 </td></tr><tr><td rowspan="6">Fruit length </td><td>2011, 2012a, 2012b </td><td><em>FL-3.1</em> </td><td>3 </td><td>40.6–48.0 </td><td>2.7–8.6 </td><td>9.3–23.6 </td><td>+ </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012b </td><td><em>FL-3.2</em> </td><td>3 </td><td>48.5–52.1 </td><td>4.4–10.0 </td><td>14.4–26.7 </td><td>+ </td><td>0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FL-3.3</em> </td><td>3 </td><td>54.1–60.8 </td><td>2.5–6.3 </td><td>8.5–21.3 </td><td>+ </td><td>0.6–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FL-3.4</em> </td><td>3 </td><td>93.3–97.3 </td><td>3.2–4.9 </td><td>10.4–12.0 </td><td>− </td><td>0.7–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FL-3.5</em> </td><td>3 </td><td>99.7–106.0 </td><td>2.9–3.4 </td><td>8.7–10.0 </td><td>− </td><td>0.6–0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FL-8</em> </td><td>8 </td><td>51.9–63.8 </td><td>2.4–6.5 </td><td>9.1–20.8 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td rowspan="5">Fruit diameter </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-1</em> </td><td>1 </td><td>108.8–121.3 </td><td>2.2–5.6 </td><td>8.4–20.0 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FD-2</em> </td><td>2 </td><td>0.0–10.1 </td><td>3.0–7.1 </td><td>8.7–16.0 </td><td>+ </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.1</em> </td><td>3 </td><td>37.7–41.4 </td><td>2.7–6.0 </td><td>8.0–17.5 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.2</em> </td><td>3 </td><td>47.2–59.1 </td><td>2.7–7.4 </td><td>8.3–20.5 </td><td>− </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FD-11</em> </td><td>11 </td><td>21.5–24.6 </td><td>2.1–2.8 </td><td>7.7–10.1 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td rowspan="4">Fruit shape </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-3.1</em> </td><td>3 </td><td>47.9–51.8 </td><td>8.8–13.6 </td><td>24.1–37.0 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FS-3.2</em> </td><td>3 </td><td>54.1–59.0 </td><td>8.4–11.7 </td><td>22.2–35.3 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-8</em> </td><td>8 </td><td>52.7–62.6 </td><td>2.3–3.9 </td><td>7.4–13.6 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FS-11</em> </td><td>11 </td><td>23.4–32.2 </td><td>2.5–2.8 </td><td>9.6–10.2 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td rowspan="6">Fruit weight </td><td>2011, 2014 </td><td><em>FW-1</em> </td><td>1 </td><td>112.5–116.4 </td><td>3.1–4.3 </td><td>10.9–15.8 </td><td>− </td><td>1.4–2.4 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FW-2.1</em> </td><td>2 </td><td>0.0–8.0 </td><td>2.9–3.0 </td><td>9.8–11.4 </td><td>+ </td><td>1.4–2.9 </td></tr><tr><td>2011, 2014 </td><td><em>FW-2.2</em> </td><td>2 </td><td>64.3–73.1 </td><td>2.2–3.5 </td><td>6.6–12.1 </td><td>− </td><td>1.2–2.0 </td></tr><tr><td>2011, 2012a </td><td><em>FW-3</em> </td><td>3 </td><td>47.6–62.5 </td><td>3.4–3.7 </td><td>9.8–12.7 </td><td>− </td><td>2.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FW-6.1</em> </td><td>6 </td><td>53.7–60.2 </td><td>3.0 </td><td>11.3–11.7 </td><td>− </td><td>1.2–1.4 </td></tr><tr><td>2012a, 2014 </td><td><em>FW-6.2</em> </td><td>6 </td><td>59.8–68.1 </td><td>2.2–2.3 </td><td>6.8–8.5 </td><td>− </td><td>1.0–1.7 </td></tr><tr><td rowspan="6">Fruit position </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-1</em> </td><td>1 </td><td>103.7–117.7 </td><td>3.1–3.8 </td><td>10.2–14.3 </td><td>− </td><td>0.3–0.4 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.1</em> </td><td>4 </td><td>35.1–42.6 </td><td>3.3 </td><td>11.0 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.2</em> </td><td>4 </td><td>108.0–115.7 </td><td>2.2 </td><td>7.5 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-12.1</em> </td><td>12 </td><td>49.0–50.5 </td><td>20.0 </td><td>53.6 </td><td>− </td><td>0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-12.2</em> </td><td>12 </td><td>57.7–58.6 </td><td>12.7–26.9 </td><td>40.2–63.9 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FP-12.3</em> </td><td>12 </td><td>65.7–68.1 </td><td>4.4–17.1 </td><td>11.6–47.0 </td><td>− </td><td>0.5–0.7 </td></tr><tr><td rowspan="10">Calyx shape </td><td>2012b, 2014 </td><td><em>CS-1.1</em> </td><td>1 </td><td>109.0–115.4 </td><td>3.7 </td><td>12.9 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-1.2</em> </td><td>1 </td><td>178.5–184.1 </td><td>3.3–3.5 </td><td>11.4–11.8 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.1</em> </td><td>3 </td><td>4.9–15.1 </td><td>2.2 </td><td>7.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.2</em> </td><td>3 </td><td>15.1–20.3 </td><td>2.6–3.5 </td><td>8.0–10.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.3</em> </td><td>3 </td><td>20.3–23.3 </td><td>2.4 </td><td>8.4 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.4</em> </td><td>3 </td><td>45.5–52.1 </td><td>2.8–3.3 </td><td>8.8–9.4 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.5</em> </td><td>3 </td><td>54.1–59.1 </td><td>3.3 </td><td>11.1 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-5</em> </td><td>5 </td><td>53.1–56.0 </td><td>2.8–3.1 </td><td>10.8–11.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-9</em> </td><td>9 </td><td>77.9–81.7 </td><td>2.9 </td><td>10.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-11</em> </td><td>11 </td><td>9.8–14.2 </td><td>2.6 </td><td>9.5 </td><td>+ </td><td>0.3 </td></tr><tr><td rowspan="6">Immature fruit colour </td><td>2012b, 2014 </td><td><em>IFC-10.1</em> </td><td>10 </td><td>66.2–69.0 </td><td>3.5 </td><td>9.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.2</em> </td><td>10 </td><td>75.7–81.8 </td><td>14.3 </td><td>40.2 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.3</em> </td><td>10 </td><td>82.6–86.1 </td><td>5.8 </td><td>19.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.1</em> </td><td>11 </td><td>6.8–13.8 </td><td>2.3 </td><td>8.2 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.2</em> </td><td>11 </td><td>81.3–85.6 </td><td>2.6 </td><td>8.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-12</em> </td><td>12 </td><td>17.5–24.1 </td><td>3.2 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr></tbody></table></div><div content-id="DSV038TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>Year </th><th>QTL </th><th>Chr. </th><th>Location (cM) </th><th>LOD </th><th><em>R</em><sup>2</sup> (%) </th><th>Direction </th><th>Additive effect </th></tr></thead><tbody><tr><td rowspan="6">Plant height </td><td>2012a, 2014 </td><td><em>PH-2</em> </td><td>2 </td><td>92.7–100.9 </td><td>3.7–4.0 </td><td>11.1–11.8 </td><td>+ </td><td>9.2–10.0 </td></tr><tr><td>2011, 2014 </td><td><em>PH-4</em> </td><td>4 </td><td>96.2–102.3 </td><td>2.3–2.6 </td><td>7.8–10.0 </td><td>− </td><td>8.3–10.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>PH-6</em> </td><td>6 </td><td>63.2–80.2 </td><td>2.1–3.4 </td><td>6.4–11.6 </td><td>+ </td><td>7.8–12.8 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-7</em> </td><td>7 </td><td>77.3–90.9 </td><td>2.6–3.4 </td><td>8.3–11.2 </td><td>+ </td><td>9.5–13.8 </td></tr><tr><td>2012b, 2014 </td><td><em>PH-8.1</em> </td><td>8 </td><td>3.8–21.1 </td><td>2.6–2.8 </td><td>5.6–8.3 </td><td>+ </td><td>9.3–11.7 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-8.2</em> </td><td>8 </td><td>37.2–54.8 </td><td>2.4–2.9 </td><td>5.9–8.6 </td><td>− </td><td>10.7–13.8 </td></tr><tr><td rowspan="2">Plant width </td><td>2012b, 2014 </td><td><em>PW-2</em> </td><td>2 </td><td>34.5–44.9 </td><td>2.2–2.4 </td><td>7.1–8.0 </td><td>− </td><td>3.0–4.4 </td></tr><tr><td>2012b, 2014 </td><td><em>PW-5</em> </td><td>5 </td><td>33.3–41.2 </td><td>2.4–3.6 </td><td>8.6–12.5 </td><td>+ </td><td>3.5–6.0 </td></tr><tr><td rowspan="6">Main stem length </td><td>2012b, 2014 </td><td><em>MSL-7</em> </td><td>7 </td><td>47.2–51.5 </td><td>2.4–5.2 </td><td>8.8–18.6 </td><td>+ </td><td>2.6–4.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-8.1</em> </td><td>8 </td><td>84.2–94.9 </td><td>4.6–6.3 </td><td>14.0–18.3 </td><td>+ </td><td>3.3–4.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-8.2</em> </td><td>8 </td><td>111.1–133.2 </td><td>2.2–4.3 </td><td>7.4–13.4 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-10</em> </td><td>10 </td><td>58.4–62.6 </td><td>2.6–3.3 </td><td>8.7–12.3 </td><td>− </td><td>5.9–7.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-11</em> </td><td>11 </td><td>76.9–82.7 </td><td>2.6–2.7 </td><td>9.8–10.4 </td><td>− </td><td>2.8–3.0 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-12</em> </td><td>12 </td><td>57.8–63.0 </td><td>1.9–2.6 </td><td>6.5–9.9 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td rowspan="2">Lateral branch number </td><td>2012a, 2012b, 2014 </td><td><em>LBN-2.1</em> </td><td>2 </td><td>91.4–98.3 </td><td>4.6–12.5 </td><td>16.0–40.0 </td><td>+ </td><td>0.9–1.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LBN-2.2</em> </td><td>2 </td><td>99.7–104.5 </td><td>2.8–6.4 </td><td>10.1–23.1 </td><td>+ </td><td>0.7–1.4 </td></tr><tr><td rowspan="5">Internode length </td><td>2012a, 2014 </td><td><em>INL-1</em> </td><td>1 </td><td>28.8–32.9 </td><td>2.6–2.9 </td><td>9.2–11.0 </td><td>+ </td><td>0.8–0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>INL-2</em> </td><td>2 </td><td>82.3–90.8 </td><td>3.2–4.8 </td><td>11.0–16.9 </td><td>+ </td><td>0.8–1.0 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-6.1</em> </td><td>6 </td><td>0.0–4.9 </td><td>2.4–3.6 </td><td>7.5–11.9 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>INL-6.2</em> </td><td>6 </td><td>25.8–37.2 </td><td>2.2–3.1 </td><td>7.4–10.2 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-10</em> </td><td>10 </td><td>0.6–22.5 </td><td>2.1–2.4 </td><td>7.1–7.4 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td rowspan="6">Stem colour </td><td>2012b, 2014 </td><td><em>SC-2</em> </td><td>2 </td><td>17.6–22.3 </td><td>3.0 </td><td>11.4 </td><td>− </td><td>0.5 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-4</em> </td><td>4 </td><td>34.7–41.0 </td><td>2.6 </td><td>9.7 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-7</em> </td><td>7 </td><td>0.0–2.5 </td><td>3.2 </td><td>10.4 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>SC-9</em> </td><td>9 </td><td>51.8–59.6 </td><td>2.4–3.4 </td><td>9.3–10.3 </td><td>+ </td><td>0.1–0.6 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-11</em> </td><td>11 </td><td>24.3–28.9 </td><td>3.3 </td><td>11.6 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-12</em> </td><td>12 </td><td>61.1–71.1 </td><td>2.8 </td><td>11.1 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="4">Leaf length </td><td>2012b, 2014 </td><td><em>LL-6</em> </td><td>6 </td><td>42.0–52.7 </td><td>2.3–4.5 </td><td>8.7–16.6 </td><td>+ </td><td>0.6–0.7 </td></tr><tr><td>2012b, 2014 </td><td><em>LL-9</em> </td><td>9 </td><td>75.3–84.9 </td><td>2.2–6.2 </td><td>7.7–20.8 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>LL-11.1</em> </td><td>11 </td><td>47.3–57.1 </td><td>2.3–4.2 </td><td>7.9–16.1 </td><td>+ </td><td>0.4–1.0 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LL-11.2</em> </td><td>11 </td><td>65.3–82.4 </td><td>2.4–3.6 </td><td>8.0–12.3 </td><td>− </td><td>0.4–0.9 </td></tr><tr><td>Leaf width </td><td>2012a, 2014 </td><td><em>LW-8</em> </td><td>8 </td><td>56.7–70.6 </td><td>2.3–5.8 </td><td>9.8–18.2 </td><td>− </td><td>0.4–0.5 </td></tr><tr><td rowspan="8">Flower size </td><td>2012b, 2014 </td><td><em>FLS-1</em> </td><td>1 </td><td>113.7–115.9 </td><td>5.8 </td><td>19.6 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-2</em> </td><td>2 </td><td>60.2–63.9 </td><td>2.8 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.1</em> </td><td>3 </td><td>18.2–23.3 </td><td>3.7 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.2</em> </td><td>3 </td><td>70.7–74.7 </td><td>2.9 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.3</em> </td><td>3 </td><td>89.1–93.7 </td><td>2.6 </td><td>7.8 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-7</em> </td><td>7 </td><td>46.8–53.1 </td><td>2.4 </td><td>8.8 </td><td>− </td><td>0.1 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-8</em> </td><td>8 </td><td>89.7–96.6 </td><td>2.7 </td><td>12.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-10</em> </td><td>10 </td><td>57.0–58.4 </td><td>2.4 </td><td>8.7 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="3">Stamen number </td><td>2012b, 2014 </td><td><em>STN-3</em> </td><td>3 </td><td>108.3–115.8 </td><td>2.6 </td><td>9.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-10</em> </td><td>10 </td><td>99.8–102.6 </td><td>2.8 </td><td>10.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-11</em> </td><td>11 </td><td>31.0–36.2 </td><td>2.3 </td><td>8.5 </td><td>− </td><td>0.2 </td></tr><tr><td rowspan="6">Fruit length </td><td>2011, 2012a, 2012b </td><td><em>FL-3.1</em> </td><td>3 </td><td>40.6–48.0 </td><td>2.7–8.6 </td><td>9.3–23.6 </td><td>+ </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012b </td><td><em>FL-3.2</em> </td><td>3 </td><td>48.5–52.1 </td><td>4.4–10.0 </td><td>14.4–26.7 </td><td>+ </td><td>0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FL-3.3</em> </td><td>3 </td><td>54.1–60.8 </td><td>2.5–6.3 </td><td>8.5–21.3 </td><td>+ </td><td>0.6–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FL-3.4</em> </td><td>3 </td><td>93.3–97.3 </td><td>3.2–4.9 </td><td>10.4–12.0 </td><td>− </td><td>0.7–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FL-3.5</em> </td><td>3 </td><td>99.7–106.0 </td><td>2.9–3.4 </td><td>8.7–10.0 </td><td>− </td><td>0.6–0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FL-8</em> </td><td>8 </td><td>51.9–63.8 </td><td>2.4–6.5 </td><td>9.1–20.8 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td rowspan="5">Fruit diameter </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-1</em> </td><td>1 </td><td>108.8–121.3 </td><td>2.2–5.6 </td><td>8.4–20.0 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FD-2</em> </td><td>2 </td><td>0.0–10.1 </td><td>3.0–7.1 </td><td>8.7–16.0 </td><td>+ </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.1</em> </td><td>3 </td><td>37.7–41.4 </td><td>2.7–6.0 </td><td>8.0–17.5 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.2</em> </td><td>3 </td><td>47.2–59.1 </td><td>2.7–7.4 </td><td>8.3–20.5 </td><td>− </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FD-11</em> </td><td>11 </td><td>21.5–24.6 </td><td>2.1–2.8 </td><td>7.7–10.1 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td rowspan="4">Fruit shape </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-3.1</em> </td><td>3 </td><td>47.9–51.8 </td><td>8.8–13.6 </td><td>24.1–37.0 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FS-3.2</em> </td><td>3 </td><td>54.1–59.0 </td><td>8.4–11.7 </td><td>22.2–35.3 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-8</em> </td><td>8 </td><td>52.7–62.6 </td><td>2.3–3.9 </td><td>7.4–13.6 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FS-11</em> </td><td>11 </td><td>23.4–32.2 </td><td>2.5–2.8 </td><td>9.6–10.2 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td rowspan="6">Fruit weight </td><td>2011, 2014 </td><td><em>FW-1</em> </td><td>1 </td><td>112.5–116.4 </td><td>3.1–4.3 </td><td>10.9–15.8 </td><td>− </td><td>1.4–2.4 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FW-2.1</em> </td><td>2 </td><td>0.0–8.0 </td><td>2.9–3.0 </td><td>9.8–11.4 </td><td>+ </td><td>1.4–2.9 </td></tr><tr><td>2011, 2014 </td><td><em>FW-2.2</em> </td><td>2 </td><td>64.3–73.1 </td><td>2.2–3.5 </td><td>6.6–12.1 </td><td>− </td><td>1.2–2.0 </td></tr><tr><td>2011, 2012a </td><td><em>FW-3</em> </td><td>3 </td><td>47.6–62.5 </td><td>3.4–3.7 </td><td>9.8–12.7 </td><td>− </td><td>2.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FW-6.1</em> </td><td>6 </td><td>53.7–60.2 </td><td>3.0 </td><td>11.3–11.7 </td><td>− </td><td>1.2–1.4 </td></tr><tr><td>2012a, 2014 </td><td><em>FW-6.2</em> </td><td>6 </td><td>59.8–68.1 </td><td>2.2–2.3 </td><td>6.8–8.5 </td><td>− </td><td>1.0–1.7 </td></tr><tr><td rowspan="6">Fruit position </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-1</em> </td><td>1 </td><td>103.7–117.7 </td><td>3.1–3.8 </td><td>10.2–14.3 </td><td>− </td><td>0.3–0.4 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.1</em> </td><td>4 </td><td>35.1–42.6 </td><td>3.3 </td><td>11.0 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.2</em> </td><td>4 </td><td>108.0–115.7 </td><td>2.2 </td><td>7.5 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-12.1</em> </td><td>12 </td><td>49.0–50.5 </td><td>20.0 </td><td>53.6 </td><td>− </td><td>0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-12.2</em> </td><td>12 </td><td>57.7–58.6 </td><td>12.7–26.9 </td><td>40.2–63.9 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FP-12.3</em> </td><td>12 </td><td>65.7–68.1 </td><td>4.4–17.1 </td><td>11.6–47.0 </td><td>− </td><td>0.5–0.7 </td></tr><tr><td rowspan="10">Calyx shape </td><td>2012b, 2014 </td><td><em>CS-1.1</em> </td><td>1 </td><td>109.0–115.4 </td><td>3.7 </td><td>12.9 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-1.2</em> </td><td>1 </td><td>178.5–184.1 </td><td>3.3–3.5 </td><td>11.4–11.8 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.1</em> </td><td>3 </td><td>4.9–15.1 </td><td>2.2 </td><td>7.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.2</em> </td><td>3 </td><td>15.1–20.3 </td><td>2.6–3.5 </td><td>8.0–10.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.3</em> </td><td>3 </td><td>20.3–23.3 </td><td>2.4 </td><td>8.4 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.4</em> </td><td>3 </td><td>45.5–52.1 </td><td>2.8–3.3 </td><td>8.8–9.4 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.5</em> </td><td>3 </td><td>54.1–59.1 </td><td>3.3 </td><td>11.1 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-5</em> </td><td>5 </td><td>53.1–56.0 </td><td>2.8–3.1 </td><td>10.8–11.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-9</em> </td><td>9 </td><td>77.9–81.7 </td><td>2.9 </td><td>10.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-11</em> </td><td>11 </td><td>9.8–14.2 </td><td>2.6 </td><td>9.5 </td><td>+ </td><td>0.3 </td></tr><tr><td rowspan="6">Immature fruit colour </td><td>2012b, 2014 </td><td><em>IFC-10.1</em> </td><td>10 </td><td>66.2–69.0 </td><td>3.5 </td><td>9.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.2</em> </td><td>10 </td><td>75.7–81.8 </td><td>14.3 </td><td>40.2 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.3</em> </td><td>10 </td><td>82.6–86.1 </td><td>5.8 </td><td>19.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.1</em> </td><td>11 </td><td>6.8–13.8 </td><td>2.3 </td><td>8.2 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.2</em> </td><td>11 </td><td>81.3–85.6 </td><td>2.6 </td><td>8.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-12</em> </td><td>12 </td><td>17.5–24.1 </td><td>3.2 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196894" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV038TB4" class="table-wrap-title" data-id="DSV038TB4"><div class="label -label">Table 4.</div><div class="caption"><p>QTLs controlling morphological traits detected in more than two environments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>Year </th><th>QTL </th><th>Chr. </th><th>Location (cM) </th><th>LOD </th><th><em>R</em><sup>2</sup> (%) </th><th>Direction </th><th>Additive effect </th></tr></thead><tbody><tr><td rowspan="6">Plant height </td><td>2012a, 2014 </td><td><em>PH-2</em> </td><td>2 </td><td>92.7–100.9 </td><td>3.7–4.0 </td><td>11.1–11.8 </td><td>+ </td><td>9.2–10.0 </td></tr><tr><td>2011, 2014 </td><td><em>PH-4</em> </td><td>4 </td><td>96.2–102.3 </td><td>2.3–2.6 </td><td>7.8–10.0 </td><td>− </td><td>8.3–10.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>PH-6</em> </td><td>6 </td><td>63.2–80.2 </td><td>2.1–3.4 </td><td>6.4–11.6 </td><td>+ </td><td>7.8–12.8 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-7</em> </td><td>7 </td><td>77.3–90.9 </td><td>2.6–3.4 </td><td>8.3–11.2 </td><td>+ </td><td>9.5–13.8 </td></tr><tr><td>2012b, 2014 </td><td><em>PH-8.1</em> </td><td>8 </td><td>3.8–21.1 </td><td>2.6–2.8 </td><td>5.6–8.3 </td><td>+ </td><td>9.3–11.7 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-8.2</em> </td><td>8 </td><td>37.2–54.8 </td><td>2.4–2.9 </td><td>5.9–8.6 </td><td>− </td><td>10.7–13.8 </td></tr><tr><td rowspan="2">Plant width </td><td>2012b, 2014 </td><td><em>PW-2</em> </td><td>2 </td><td>34.5–44.9 </td><td>2.2–2.4 </td><td>7.1–8.0 </td><td>− </td><td>3.0–4.4 </td></tr><tr><td>2012b, 2014 </td><td><em>PW-5</em> </td><td>5 </td><td>33.3–41.2 </td><td>2.4–3.6 </td><td>8.6–12.5 </td><td>+ </td><td>3.5–6.0 </td></tr><tr><td rowspan="6">Main stem length </td><td>2012b, 2014 </td><td><em>MSL-7</em> </td><td>7 </td><td>47.2–51.5 </td><td>2.4–5.2 </td><td>8.8–18.6 </td><td>+ </td><td>2.6–4.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-8.1</em> </td><td>8 </td><td>84.2–94.9 </td><td>4.6–6.3 </td><td>14.0–18.3 </td><td>+ </td><td>3.3–4.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-8.2</em> </td><td>8 </td><td>111.1–133.2 </td><td>2.2–4.3 </td><td>7.4–13.4 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-10</em> </td><td>10 </td><td>58.4–62.6 </td><td>2.6–3.3 </td><td>8.7–12.3 </td><td>− </td><td>5.9–7.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-11</em> </td><td>11 </td><td>76.9–82.7 </td><td>2.6–2.7 </td><td>9.8–10.4 </td><td>− </td><td>2.8–3.0 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-12</em> </td><td>12 </td><td>57.8–63.0 </td><td>1.9–2.6 </td><td>6.5–9.9 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td rowspan="2">Lateral branch number </td><td>2012a, 2012b, 2014 </td><td><em>LBN-2.1</em> </td><td>2 </td><td>91.4–98.3 </td><td>4.6–12.5 </td><td>16.0–40.0 </td><td>+ </td><td>0.9–1.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LBN-2.2</em> </td><td>2 </td><td>99.7–104.5 </td><td>2.8–6.4 </td><td>10.1–23.1 </td><td>+ </td><td>0.7–1.4 </td></tr><tr><td rowspan="5">Internode length </td><td>2012a, 2014 </td><td><em>INL-1</em> </td><td>1 </td><td>28.8–32.9 </td><td>2.6–2.9 </td><td>9.2–11.0 </td><td>+ </td><td>0.8–0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>INL-2</em> </td><td>2 </td><td>82.3–90.8 </td><td>3.2–4.8 </td><td>11.0–16.9 </td><td>+ </td><td>0.8–1.0 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-6.1</em> </td><td>6 </td><td>0.0–4.9 </td><td>2.4–3.6 </td><td>7.5–11.9 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>INL-6.2</em> </td><td>6 </td><td>25.8–37.2 </td><td>2.2–3.1 </td><td>7.4–10.2 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-10</em> </td><td>10 </td><td>0.6–22.5 </td><td>2.1–2.4 </td><td>7.1–7.4 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td rowspan="6">Stem colour </td><td>2012b, 2014 </td><td><em>SC-2</em> </td><td>2 </td><td>17.6–22.3 </td><td>3.0 </td><td>11.4 </td><td>− </td><td>0.5 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-4</em> </td><td>4 </td><td>34.7–41.0 </td><td>2.6 </td><td>9.7 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-7</em> </td><td>7 </td><td>0.0–2.5 </td><td>3.2 </td><td>10.4 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>SC-9</em> </td><td>9 </td><td>51.8–59.6 </td><td>2.4–3.4 </td><td>9.3–10.3 </td><td>+ </td><td>0.1–0.6 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-11</em> </td><td>11 </td><td>24.3–28.9 </td><td>3.3 </td><td>11.6 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-12</em> </td><td>12 </td><td>61.1–71.1 </td><td>2.8 </td><td>11.1 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="4">Leaf length </td><td>2012b, 2014 </td><td><em>LL-6</em> </td><td>6 </td><td>42.0–52.7 </td><td>2.3–4.5 </td><td>8.7–16.6 </td><td>+ </td><td>0.6–0.7 </td></tr><tr><td>2012b, 2014 </td><td><em>LL-9</em> </td><td>9 </td><td>75.3–84.9 </td><td>2.2–6.2 </td><td>7.7–20.8 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>LL-11.1</em> </td><td>11 </td><td>47.3–57.1 </td><td>2.3–4.2 </td><td>7.9–16.1 </td><td>+ </td><td>0.4–1.0 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LL-11.2</em> </td><td>11 </td><td>65.3–82.4 </td><td>2.4–3.6 </td><td>8.0–12.3 </td><td>− </td><td>0.4–0.9 </td></tr><tr><td>Leaf width </td><td>2012a, 2014 </td><td><em>LW-8</em> </td><td>8 </td><td>56.7–70.6 </td><td>2.3–5.8 </td><td>9.8–18.2 </td><td>− </td><td>0.4–0.5 </td></tr><tr><td rowspan="8">Flower size </td><td>2012b, 2014 </td><td><em>FLS-1</em> </td><td>1 </td><td>113.7–115.9 </td><td>5.8 </td><td>19.6 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-2</em> </td><td>2 </td><td>60.2–63.9 </td><td>2.8 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.1</em> </td><td>3 </td><td>18.2–23.3 </td><td>3.7 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.2</em> </td><td>3 </td><td>70.7–74.7 </td><td>2.9 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.3</em> </td><td>3 </td><td>89.1–93.7 </td><td>2.6 </td><td>7.8 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-7</em> </td><td>7 </td><td>46.8–53.1 </td><td>2.4 </td><td>8.8 </td><td>− </td><td>0.1 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-8</em> </td><td>8 </td><td>89.7–96.6 </td><td>2.7 </td><td>12.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-10</em> </td><td>10 </td><td>57.0–58.4 </td><td>2.4 </td><td>8.7 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="3">Stamen number </td><td>2012b, 2014 </td><td><em>STN-3</em> </td><td>3 </td><td>108.3–115.8 </td><td>2.6 </td><td>9.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-10</em> </td><td>10 </td><td>99.8–102.6 </td><td>2.8 </td><td>10.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-11</em> </td><td>11 </td><td>31.0–36.2 </td><td>2.3 </td><td>8.5 </td><td>− </td><td>0.2 </td></tr><tr><td rowspan="6">Fruit length </td><td>2011, 2012a, 2012b </td><td><em>FL-3.1</em> </td><td>3 </td><td>40.6–48.0 </td><td>2.7–8.6 </td><td>9.3–23.6 </td><td>+ </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012b </td><td><em>FL-3.2</em> </td><td>3 </td><td>48.5–52.1 </td><td>4.4–10.0 </td><td>14.4–26.7 </td><td>+ </td><td>0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FL-3.3</em> </td><td>3 </td><td>54.1–60.8 </td><td>2.5–6.3 </td><td>8.5–21.3 </td><td>+ </td><td>0.6–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FL-3.4</em> </td><td>3 </td><td>93.3–97.3 </td><td>3.2–4.9 </td><td>10.4–12.0 </td><td>− </td><td>0.7–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FL-3.5</em> </td><td>3 </td><td>99.7–106.0 </td><td>2.9–3.4 </td><td>8.7–10.0 </td><td>− </td><td>0.6–0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FL-8</em> </td><td>8 </td><td>51.9–63.8 </td><td>2.4–6.5 </td><td>9.1–20.8 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td rowspan="5">Fruit diameter </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-1</em> </td><td>1 </td><td>108.8–121.3 </td><td>2.2–5.6 </td><td>8.4–20.0 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FD-2</em> </td><td>2 </td><td>0.0–10.1 </td><td>3.0–7.1 </td><td>8.7–16.0 </td><td>+ </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.1</em> </td><td>3 </td><td>37.7–41.4 </td><td>2.7–6.0 </td><td>8.0–17.5 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.2</em> </td><td>3 </td><td>47.2–59.1 </td><td>2.7–7.4 </td><td>8.3–20.5 </td><td>− </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FD-11</em> </td><td>11 </td><td>21.5–24.6 </td><td>2.1–2.8 </td><td>7.7–10.1 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td rowspan="4">Fruit shape </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-3.1</em> </td><td>3 </td><td>47.9–51.8 </td><td>8.8–13.6 </td><td>24.1–37.0 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FS-3.2</em> </td><td>3 </td><td>54.1–59.0 </td><td>8.4–11.7 </td><td>22.2–35.3 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-8</em> </td><td>8 </td><td>52.7–62.6 </td><td>2.3–3.9 </td><td>7.4–13.6 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FS-11</em> </td><td>11 </td><td>23.4–32.2 </td><td>2.5–2.8 </td><td>9.6–10.2 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td rowspan="6">Fruit weight </td><td>2011, 2014 </td><td><em>FW-1</em> </td><td>1 </td><td>112.5–116.4 </td><td>3.1–4.3 </td><td>10.9–15.8 </td><td>− </td><td>1.4–2.4 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FW-2.1</em> </td><td>2 </td><td>0.0–8.0 </td><td>2.9–3.0 </td><td>9.8–11.4 </td><td>+ </td><td>1.4–2.9 </td></tr><tr><td>2011, 2014 </td><td><em>FW-2.2</em> </td><td>2 </td><td>64.3–73.1 </td><td>2.2–3.5 </td><td>6.6–12.1 </td><td>− </td><td>1.2–2.0 </td></tr><tr><td>2011, 2012a </td><td><em>FW-3</em> </td><td>3 </td><td>47.6–62.5 </td><td>3.4–3.7 </td><td>9.8–12.7 </td><td>− </td><td>2.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FW-6.1</em> </td><td>6 </td><td>53.7–60.2 </td><td>3.0 </td><td>11.3–11.7 </td><td>− </td><td>1.2–1.4 </td></tr><tr><td>2012a, 2014 </td><td><em>FW-6.2</em> </td><td>6 </td><td>59.8–68.1 </td><td>2.2–2.3 </td><td>6.8–8.5 </td><td>− </td><td>1.0–1.7 </td></tr><tr><td rowspan="6">Fruit position </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-1</em> </td><td>1 </td><td>103.7–117.7 </td><td>3.1–3.8 </td><td>10.2–14.3 </td><td>− </td><td>0.3–0.4 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.1</em> </td><td>4 </td><td>35.1–42.6 </td><td>3.3 </td><td>11.0 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.2</em> </td><td>4 </td><td>108.0–115.7 </td><td>2.2 </td><td>7.5 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-12.1</em> </td><td>12 </td><td>49.0–50.5 </td><td>20.0 </td><td>53.6 </td><td>− </td><td>0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-12.2</em> </td><td>12 </td><td>57.7–58.6 </td><td>12.7–26.9 </td><td>40.2–63.9 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FP-12.3</em> </td><td>12 </td><td>65.7–68.1 </td><td>4.4–17.1 </td><td>11.6–47.0 </td><td>− </td><td>0.5–0.7 </td></tr><tr><td rowspan="10">Calyx shape </td><td>2012b, 2014 </td><td><em>CS-1.1</em> </td><td>1 </td><td>109.0–115.4 </td><td>3.7 </td><td>12.9 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-1.2</em> </td><td>1 </td><td>178.5–184.1 </td><td>3.3–3.5 </td><td>11.4–11.8 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.1</em> </td><td>3 </td><td>4.9–15.1 </td><td>2.2 </td><td>7.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.2</em> </td><td>3 </td><td>15.1–20.3 </td><td>2.6–3.5 </td><td>8.0–10.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.3</em> </td><td>3 </td><td>20.3–23.3 </td><td>2.4 </td><td>8.4 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.4</em> </td><td>3 </td><td>45.5–52.1 </td><td>2.8–3.3 </td><td>8.8–9.4 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.5</em> </td><td>3 </td><td>54.1–59.1 </td><td>3.3 </td><td>11.1 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-5</em> </td><td>5 </td><td>53.1–56.0 </td><td>2.8–3.1 </td><td>10.8–11.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-9</em> </td><td>9 </td><td>77.9–81.7 </td><td>2.9 </td><td>10.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-11</em> </td><td>11 </td><td>9.8–14.2 </td><td>2.6 </td><td>9.5 </td><td>+ </td><td>0.3 </td></tr><tr><td rowspan="6">Immature fruit colour </td><td>2012b, 2014 </td><td><em>IFC-10.1</em> </td><td>10 </td><td>66.2–69.0 </td><td>3.5 </td><td>9.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.2</em> </td><td>10 </td><td>75.7–81.8 </td><td>14.3 </td><td>40.2 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.3</em> </td><td>10 </td><td>82.6–86.1 </td><td>5.8 </td><td>19.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.1</em> </td><td>11 </td><td>6.8–13.8 </td><td>2.3 </td><td>8.2 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.2</em> </td><td>11 </td><td>81.3–85.6 </td><td>2.6 </td><td>8.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-12</em> </td><td>12 </td><td>17.5–24.1 </td><td>3.2 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr></tbody></table></div><div content-id="DSV038TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>Year </th><th>QTL </th><th>Chr. </th><th>Location (cM) </th><th>LOD </th><th><em>R</em><sup>2</sup> (%) </th><th>Direction </th><th>Additive effect </th></tr></thead><tbody><tr><td rowspan="6">Plant height </td><td>2012a, 2014 </td><td><em>PH-2</em> </td><td>2 </td><td>92.7–100.9 </td><td>3.7–4.0 </td><td>11.1–11.8 </td><td>+ </td><td>9.2–10.0 </td></tr><tr><td>2011, 2014 </td><td><em>PH-4</em> </td><td>4 </td><td>96.2–102.3 </td><td>2.3–2.6 </td><td>7.8–10.0 </td><td>− </td><td>8.3–10.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>PH-6</em> </td><td>6 </td><td>63.2–80.2 </td><td>2.1–3.4 </td><td>6.4–11.6 </td><td>+ </td><td>7.8–12.8 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-7</em> </td><td>7 </td><td>77.3–90.9 </td><td>2.6–3.4 </td><td>8.3–11.2 </td><td>+ </td><td>9.5–13.8 </td></tr><tr><td>2012b, 2014 </td><td><em>PH-8.1</em> </td><td>8 </td><td>3.8–21.1 </td><td>2.6–2.8 </td><td>5.6–8.3 </td><td>+ </td><td>9.3–11.7 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>PH-8.2</em> </td><td>8 </td><td>37.2–54.8 </td><td>2.4–2.9 </td><td>5.9–8.6 </td><td>− </td><td>10.7–13.8 </td></tr><tr><td rowspan="2">Plant width </td><td>2012b, 2014 </td><td><em>PW-2</em> </td><td>2 </td><td>34.5–44.9 </td><td>2.2–2.4 </td><td>7.1–8.0 </td><td>− </td><td>3.0–4.4 </td></tr><tr><td>2012b, 2014 </td><td><em>PW-5</em> </td><td>5 </td><td>33.3–41.2 </td><td>2.4–3.6 </td><td>8.6–12.5 </td><td>+ </td><td>3.5–6.0 </td></tr><tr><td rowspan="6">Main stem length </td><td>2012b, 2014 </td><td><em>MSL-7</em> </td><td>7 </td><td>47.2–51.5 </td><td>2.4–5.2 </td><td>8.8–18.6 </td><td>+ </td><td>2.6–4.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-8.1</em> </td><td>8 </td><td>84.2–94.9 </td><td>4.6–6.3 </td><td>14.0–18.3 </td><td>+ </td><td>3.3–4.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-8.2</em> </td><td>8 </td><td>111.1–133.2 </td><td>2.2–4.3 </td><td>7.4–13.4 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td>2012b, 2014 </td><td><em>MSL-10</em> </td><td>10 </td><td>58.4–62.6 </td><td>2.6–3.3 </td><td>8.7–12.3 </td><td>− </td><td>5.9–7.5 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-11</em> </td><td>11 </td><td>76.9–82.7 </td><td>2.6–2.7 </td><td>9.8–10.4 </td><td>− </td><td>2.8–3.0 </td></tr><tr><td>2012a, 2014 </td><td><em>MSL-12</em> </td><td>12 </td><td>57.8–63.0 </td><td>1.9–2.6 </td><td>6.5–9.9 </td><td>− </td><td>2.6–3.0 </td></tr><tr><td rowspan="2">Lateral branch number </td><td>2012a, 2012b, 2014 </td><td><em>LBN-2.1</em> </td><td>2 </td><td>91.4–98.3 </td><td>4.6–12.5 </td><td>16.0–40.0 </td><td>+ </td><td>0.9–1.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LBN-2.2</em> </td><td>2 </td><td>99.7–104.5 </td><td>2.8–6.4 </td><td>10.1–23.1 </td><td>+ </td><td>0.7–1.4 </td></tr><tr><td rowspan="5">Internode length </td><td>2012a, 2014 </td><td><em>INL-1</em> </td><td>1 </td><td>28.8–32.9 </td><td>2.6–2.9 </td><td>9.2–11.0 </td><td>+ </td><td>0.8–0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>INL-2</em> </td><td>2 </td><td>82.3–90.8 </td><td>3.2–4.8 </td><td>11.0–16.9 </td><td>+ </td><td>0.8–1.0 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-6.1</em> </td><td>6 </td><td>0.0–4.9 </td><td>2.4–3.6 </td><td>7.5–11.9 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>INL-6.2</em> </td><td>6 </td><td>25.8–37.2 </td><td>2.2–3.1 </td><td>7.4–10.2 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td>2012a, 2012b </td><td><em>INL-10</em> </td><td>10 </td><td>0.6–22.5 </td><td>2.1–2.4 </td><td>7.1–7.4 </td><td>+ </td><td>0.7–0.8 </td></tr><tr><td rowspan="6">Stem colour </td><td>2012b, 2014 </td><td><em>SC-2</em> </td><td>2 </td><td>17.6–22.3 </td><td>3.0 </td><td>11.4 </td><td>− </td><td>0.5 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-4</em> </td><td>4 </td><td>34.7–41.0 </td><td>2.6 </td><td>9.7 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-7</em> </td><td>7 </td><td>0.0–2.5 </td><td>3.2 </td><td>10.4 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>SC-9</em> </td><td>9 </td><td>51.8–59.6 </td><td>2.4–3.4 </td><td>9.3–10.3 </td><td>+ </td><td>0.1–0.6 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-11</em> </td><td>11 </td><td>24.3–28.9 </td><td>3.3 </td><td>11.6 </td><td>+ </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>SC-12</em> </td><td>12 </td><td>61.1–71.1 </td><td>2.8 </td><td>11.1 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="4">Leaf length </td><td>2012b, 2014 </td><td><em>LL-6</em> </td><td>6 </td><td>42.0–52.7 </td><td>2.3–4.5 </td><td>8.7–16.6 </td><td>+ </td><td>0.6–0.7 </td></tr><tr><td>2012b, 2014 </td><td><em>LL-9</em> </td><td>9 </td><td>75.3–84.9 </td><td>2.2–6.2 </td><td>7.7–20.8 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2012a, 2014 </td><td><em>LL-11.1</em> </td><td>11 </td><td>47.3–57.1 </td><td>2.3–4.2 </td><td>7.9–16.1 </td><td>+ </td><td>0.4–1.0 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>LL-11.2</em> </td><td>11 </td><td>65.3–82.4 </td><td>2.4–3.6 </td><td>8.0–12.3 </td><td>− </td><td>0.4–0.9 </td></tr><tr><td>Leaf width </td><td>2012a, 2014 </td><td><em>LW-8</em> </td><td>8 </td><td>56.7–70.6 </td><td>2.3–5.8 </td><td>9.8–18.2 </td><td>− </td><td>0.4–0.5 </td></tr><tr><td rowspan="8">Flower size </td><td>2012b, 2014 </td><td><em>FLS-1</em> </td><td>1 </td><td>113.7–115.9 </td><td>5.8 </td><td>19.6 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-2</em> </td><td>2 </td><td>60.2–63.9 </td><td>2.8 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.1</em> </td><td>3 </td><td>18.2–23.3 </td><td>3.7 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.2</em> </td><td>3 </td><td>70.7–74.7 </td><td>2.9 </td><td>9.0 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-3.3</em> </td><td>3 </td><td>89.1–93.7 </td><td>2.6 </td><td>7.8 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-7</em> </td><td>7 </td><td>46.8–53.1 </td><td>2.4 </td><td>8.8 </td><td>− </td><td>0.1 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-8</em> </td><td>8 </td><td>89.7–96.6 </td><td>2.7 </td><td>12.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>FLS-10</em> </td><td>10 </td><td>57.0–58.4 </td><td>2.4 </td><td>8.7 </td><td>+ </td><td>0.5 </td></tr><tr><td rowspan="3">Stamen number </td><td>2012b, 2014 </td><td><em>STN-3</em> </td><td>3 </td><td>108.3–115.8 </td><td>2.6 </td><td>9.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-10</em> </td><td>10 </td><td>99.8–102.6 </td><td>2.8 </td><td>10.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>STN-11</em> </td><td>11 </td><td>31.0–36.2 </td><td>2.3 </td><td>8.5 </td><td>− </td><td>0.2 </td></tr><tr><td rowspan="6">Fruit length </td><td>2011, 2012a, 2012b </td><td><em>FL-3.1</em> </td><td>3 </td><td>40.6–48.0 </td><td>2.7–8.6 </td><td>9.3–23.6 </td><td>+ </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012b </td><td><em>FL-3.2</em> </td><td>3 </td><td>48.5–52.1 </td><td>4.4–10.0 </td><td>14.4–26.7 </td><td>+ </td><td>0.9 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FL-3.3</em> </td><td>3 </td><td>54.1–60.8 </td><td>2.5–6.3 </td><td>8.5–21.3 </td><td>+ </td><td>0.6–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FL-3.4</em> </td><td>3 </td><td>93.3–97.3 </td><td>3.2–4.9 </td><td>10.4–12.0 </td><td>− </td><td>0.7–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FL-3.5</em> </td><td>3 </td><td>99.7–106.0 </td><td>2.9–3.4 </td><td>8.7–10.0 </td><td>− </td><td>0.6–0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FL-8</em> </td><td>8 </td><td>51.9–63.8 </td><td>2.4–6.5 </td><td>9.1–20.8 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td rowspan="5">Fruit diameter </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-1</em> </td><td>1 </td><td>108.8–121.3 </td><td>2.2–5.6 </td><td>8.4–20.0 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FD-2</em> </td><td>2 </td><td>0.0–10.1 </td><td>3.0–7.1 </td><td>8.7–16.0 </td><td>+ </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.1</em> </td><td>3 </td><td>37.7–41.4 </td><td>2.7–6.0 </td><td>8.0–17.5 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FD-3.2</em> </td><td>3 </td><td>47.2–59.1 </td><td>2.7–7.4 </td><td>8.3–20.5 </td><td>− </td><td>0.2–0.4 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FD-11</em> </td><td>11 </td><td>21.5–24.6 </td><td>2.1–2.8 </td><td>7.7–10.1 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td rowspan="4">Fruit shape </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-3.1</em> </td><td>3 </td><td>47.9–51.8 </td><td>8.8–13.6 </td><td>24.1–37.0 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2014 </td><td><em>FS-3.2</em> </td><td>3 </td><td>54.1–59.0 </td><td>8.4–11.7 </td><td>22.2–35.3 </td><td>+ </td><td>0.7–0.9 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FS-8</em> </td><td>8 </td><td>52.7–62.6 </td><td>2.3–3.9 </td><td>7.4–13.6 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td>2011, 2012b, 2014 </td><td><em>FS-11</em> </td><td>11 </td><td>23.4–32.2 </td><td>2.5–2.8 </td><td>9.6–10.2 </td><td>+ </td><td>0.5–0.6 </td></tr><tr><td rowspan="6">Fruit weight </td><td>2011, 2014 </td><td><em>FW-1</em> </td><td>1 </td><td>112.5–116.4 </td><td>3.1–4.3 </td><td>10.9–15.8 </td><td>− </td><td>1.4–2.4 </td></tr><tr><td>2012a, 2012b, 2014 </td><td><em>FW-2.1</em> </td><td>2 </td><td>0.0–8.0 </td><td>2.9–3.0 </td><td>9.8–11.4 </td><td>+ </td><td>1.4–2.9 </td></tr><tr><td>2011, 2014 </td><td><em>FW-2.2</em> </td><td>2 </td><td>64.3–73.1 </td><td>2.2–3.5 </td><td>6.6–12.1 </td><td>− </td><td>1.2–2.0 </td></tr><tr><td>2011, 2012a </td><td><em>FW-3</em> </td><td>3 </td><td>47.6–62.5 </td><td>3.4–3.7 </td><td>9.8–12.7 </td><td>− </td><td>2.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FW-6.1</em> </td><td>6 </td><td>53.7–60.2 </td><td>3.0 </td><td>11.3–11.7 </td><td>− </td><td>1.2–1.4 </td></tr><tr><td>2012a, 2014 </td><td><em>FW-6.2</em> </td><td>6 </td><td>59.8–68.1 </td><td>2.2–2.3 </td><td>6.8–8.5 </td><td>− </td><td>1.0–1.7 </td></tr><tr><td rowspan="6">Fruit position </td><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-1</em> </td><td>1 </td><td>103.7–117.7 </td><td>3.1–3.8 </td><td>10.2–14.3 </td><td>− </td><td>0.3–0.4 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.1</em> </td><td>4 </td><td>35.1–42.6 </td><td>3.3 </td><td>11.0 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-4.2</em> </td><td>4 </td><td>108.0–115.7 </td><td>2.2 </td><td>7.5 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>FP-12.1</em> </td><td>12 </td><td>49.0–50.5 </td><td>20.0 </td><td>53.6 </td><td>− </td><td>0.7 </td></tr><tr><td>2011, 2012a, 2012b, 2014 </td><td><em>FP-12.2</em> </td><td>12 </td><td>57.7–58.6 </td><td>12.7–26.9 </td><td>40.2–63.9 </td><td>− </td><td>0.6–0.8 </td></tr><tr><td>2011, 2012a </td><td><em>FP-12.3</em> </td><td>12 </td><td>65.7–68.1 </td><td>4.4–17.1 </td><td>11.6–47.0 </td><td>− </td><td>0.5–0.7 </td></tr><tr><td rowspan="10">Calyx shape </td><td>2012b, 2014 </td><td><em>CS-1.1</em> </td><td>1 </td><td>109.0–115.4 </td><td>3.7 </td><td>12.9 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-1.2</em> </td><td>1 </td><td>178.5–184.1 </td><td>3.3–3.5 </td><td>11.4–11.8 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.1</em> </td><td>3 </td><td>4.9–15.1 </td><td>2.2 </td><td>7.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.2</em> </td><td>3 </td><td>15.1–20.3 </td><td>2.6–3.5 </td><td>8.0–10.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.3</em> </td><td>3 </td><td>20.3–23.3 </td><td>2.4 </td><td>8.4 </td><td>− </td><td>0.2 </td></tr><tr><td>2011, 2012a </td><td><em>CS-3.4</em> </td><td>3 </td><td>45.5–52.1 </td><td>2.8–3.3 </td><td>8.8–9.4 </td><td>− </td><td>0.2–0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-3.5</em> </td><td>3 </td><td>54.1–59.1 </td><td>3.3 </td><td>11.1 </td><td>− </td><td>0.3 </td></tr><tr><td>2011, 2012a </td><td><em>CS-5</em> </td><td>5 </td><td>53.1–56.0 </td><td>2.8–3.1 </td><td>10.8–11.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-9</em> </td><td>9 </td><td>77.9–81.7 </td><td>2.9 </td><td>10.3 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>CS-11</em> </td><td>11 </td><td>9.8–14.2 </td><td>2.6 </td><td>9.5 </td><td>+ </td><td>0.3 </td></tr><tr><td rowspan="6">Immature fruit colour </td><td>2012b, 2014 </td><td><em>IFC-10.1</em> </td><td>10 </td><td>66.2–69.0 </td><td>3.5 </td><td>9.2 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.2</em> </td><td>10 </td><td>75.7–81.8 </td><td>14.3 </td><td>40.2 </td><td>− </td><td>0.3 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-10.3</em> </td><td>10 </td><td>82.6–86.1 </td><td>5.8 </td><td>19.3 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.1</em> </td><td>11 </td><td>6.8–13.8 </td><td>2.3 </td><td>8.2 </td><td>+ </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-11.2</em> </td><td>11 </td><td>81.3–85.6 </td><td>2.6 </td><td>8.8 </td><td>− </td><td>0.2 </td></tr><tr><td>2012b, 2014 </td><td><em>IFC-12</em> </td><td>12 </td><td>17.5–24.1 </td><td>3.2 </td><td>11.3 </td><td>− </td><td>0.2 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196894" target="_blank">View Large</a></div></div>                        <a id="87196898" scrollto-destination="87196898"></a>
<div id="DSV038F3" data-id="DSV038F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/m_dsv03803.jpeg?Expires=2147483647&Signature=M~4pNC7bR3O9tSjaRPYihZw0xmZKflUlrUV7~XJbvkrNzy7gx49UFBFdvvkGThzRSlnf-QVLY-f6HkM-zMUEH5JRdO9bwk1zWHscH~uZgshagLac0i4lpz0fpLLWgR7iIHeZUwBpcrrcEDCoyTLrhbuI9lh1HwCuW8h5I9CfnawwQqbpgBbmZVR68zK5yI3AqsqOsyfsEIIIf8k~oWwj0YY~bDsz2TXvdZm-SmEQcH1dsOPwI1FJ52gGK4o~h71gI0d0pt5u1fuvCh-2~oOTtJ8IugDM7zcTzepBw7x4kxUERJbzw2eUGfr0LL2y1SO5~UM1eaPJsVfZVTMQkiy2OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chromosomal distribution of significant QTLs. Coloured bars show the location of QTLs, and the names of QTLs are listed in Table 4. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196898/dsv03803.jpeg" path-from-xml="dsv03803.jpeg" target="_blank">View large</a><a section="87196898" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv03803.jpeg?Expires=2147483647&Signature=Hnx13lK-snU739qTQDMjbr1NnYD~xUOFOszvIk4AeKov3oIWxbXWndw24DE1rOUu88dc1n6hj8O0jUjbclCj1toPQRalAZArzAIPBSQcmugUXU1j~tXA4uQXUrG6oC909FdyLlpjDfk~JV7iv0jfsrrZIq6mybh6xdg6RsoYNSr5pCpYx5RxsdUfN7QzqkMa380g8vMsB-c6qaymwBCj4xzfj22TXA9yUk8PEb7e4~ZPJ8du--B6KvzwpnesoVWlepNUYtzj~BQK-Vo6vW6z1aNA2suh~TRcEGz7y~qyXnnhuHdb~9NwdGt5xwUQDmxr10~HeWh8kyS7OlJR-c2tew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196898&ar=1745287&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal distribution of significant QTLs. Coloured bars show the location of QTLs, and the names of QTLs are listed in Table <a reveal-id="DSV038TB4" data-open="DSV038TB4" class="link link-reveal link-table xref-fig">4</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV038F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/m_dsv03803.jpeg?Expires=2147483647&Signature=M~4pNC7bR3O9tSjaRPYihZw0xmZKflUlrUV7~XJbvkrNzy7gx49UFBFdvvkGThzRSlnf-QVLY-f6HkM-zMUEH5JRdO9bwk1zWHscH~uZgshagLac0i4lpz0fpLLWgR7iIHeZUwBpcrrcEDCoyTLrhbuI9lh1HwCuW8h5I9CfnawwQqbpgBbmZVR68zK5yI3AqsqOsyfsEIIIf8k~oWwj0YY~bDsz2TXvdZm-SmEQcH1dsOPwI1FJ52gGK4o~h71gI0d0pt5u1fuvCh-2~oOTtJ8IugDM7zcTzepBw7x4kxUERJbzw2eUGfr0LL2y1SO5~UM1eaPJsVfZVTMQkiy2OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chromosomal distribution of significant QTLs. Coloured bars show the location of QTLs, and the names of QTLs are listed in Table 4. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196898/dsv03803.jpeg" path-from-xml="dsv03803.jpeg" target="_blank">View large</a><a section="87196898" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv03803.jpeg?Expires=2147483647&Signature=Hnx13lK-snU739qTQDMjbr1NnYD~xUOFOszvIk4AeKov3oIWxbXWndw24DE1rOUu88dc1n6hj8O0jUjbclCj1toPQRalAZArzAIPBSQcmugUXU1j~tXA4uQXUrG6oC909FdyLlpjDfk~JV7iv0jfsrrZIq6mybh6xdg6RsoYNSr5pCpYx5RxsdUfN7QzqkMa380g8vMsB-c6qaymwBCj4xzfj22TXA9yUk8PEb7e4~ZPJ8du--B6KvzwpnesoVWlepNUYtzj~BQK-Vo6vW6z1aNA2suh~TRcEGz7y~qyXnnhuHdb~9NwdGt5xwUQDmxr10~HeWh8kyS7OlJR-c2tew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196898&ar=1745287&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal distribution of significant QTLs. Coloured bars show the location of QTLs, and the names of QTLs are listed in Table <a reveal-id="DSV038TB4" data-open="DSV038TB4" class="link link-reveal link-table xref-fig">4</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>On chromosome 3, 19 QTLs were detected and 15 of them controlled fruit-related traits (Fig. <a reveal-id="DSV038F3" data-open="DSV038F3" class="link link-reveal link-table xref-fig">3</a>). Notably, 10 QTLs were located between 37.7 and 59.1 cM and all controlled fruit-related traits. <em>FS-3.1</em>, <em>FS-3.2</em>, <em>FL-3.1</em>, <em>FL-3.2</em>, and <em>FL-3.3</em> had positive additive effects, meaning that RILs with the maternal genotype had higher values for fruit shape and fruit length (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A and B</a></span>). The two traits controlled by these QTLs were positively correlated with each other. <em>CS-3.4</em>, <em>CS-3.5</em>, <em>FD-3.1</em>, <em>FD-3.2</em>, and <em>FW-3</em> showed negative additive effects, and the traits showed positive correlation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4C and D</a></span>). Co-localization of correlated QTLs was detected on chromosomes 1, 2, 3, 7, 8, 10, and 11. Most of these co-localized QTLs controlled fruit-related traits, whereas negatively correlated flower size and main stem length also showed co-localization of QTLs on chromosomes 7, 8, and 10.</p>                            <h3 scrollto-destination=87196903 id="87196903" class="section-title" data-legacy-id=s3d>3.4. Anchoring of unassigned scaffolds</h3>
<p>Sequencing reads that did not map to ‘CM334’ chromosomes were realigned to the unassigned scaffolds. The longest scaffold was 3.4 Mb, and we used the assumption that there were no recombinant breakpoints within a scaffold. Therefore, scaffolds with more than 30 SNPs were used as markers. The ratio between the SNPs with ‘Perennial’ and ‘Dempsey’ genotypes was calculated to genotype the scaffolds of each RIL. Genotyped scaffolds were mapped to the bin map constructed with the sliding window approach. The number of unassigned scaffolds was 36,487, of which 263 scaffolds were anchored to the bin map. We removed 59 scaffolds that mapped to more than two chromosomes, and 204 scaffolds could ultimately be assigned using the bin map (Table <a reveal-id="DSV038TB5" data-open="DSV038TB5" class="link link-reveal link-table xref-fig">5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). As we used bins instead of individual SNPs, the precise position of each scaffold was not determined here. </p>                        <a id="87196911" scrollto-destination="87196911"></a>
<div content-id="DSV038TB5" class="table-modal"><div class="table-wrap"><div id="DSV038TB5" class="table-wrap-title" data-id="DSV038TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Distribution of scaffolds assigned using the bin map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Number of assigned scaffolds </th><th>Average length of scaffold (kb) </th></tr></thead><tbody><tr><td>1 </td><td>39 </td><td>624 </td></tr><tr><td>2 </td><td>12 </td><td>512 </td></tr><tr><td>3 </td><td>24 </td><td>511 </td></tr><tr><td>4 </td><td>– </td><td>– </td></tr><tr><td>5 </td><td>14 </td><td>572 </td></tr><tr><td>6 </td><td>26 </td><td>457 </td></tr><tr><td>7 </td><td>– </td><td>– </td></tr><tr><td>8 </td><td>8 </td><td>1,228 </td></tr><tr><td>9 </td><td>27 </td><td>552 </td></tr><tr><td>10 </td><td>10 </td><td>247 </td></tr><tr><td>11 </td><td>24 </td><td>393 </td></tr><tr><td>12 </td><td>20 </td><td>601 </td></tr><tr><td>Total </td><td>204 </td><td>545 </td></tr></tbody></table></div><div content-id="DSV038TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Number of assigned scaffolds </th><th>Average length of scaffold (kb) </th></tr></thead><tbody><tr><td>1 </td><td>39 </td><td>624 </td></tr><tr><td>2 </td><td>12 </td><td>512 </td></tr><tr><td>3 </td><td>24 </td><td>511 </td></tr><tr><td>4 </td><td>– </td><td>– </td></tr><tr><td>5 </td><td>14 </td><td>572 </td></tr><tr><td>6 </td><td>26 </td><td>457 </td></tr><tr><td>7 </td><td>– </td><td>– </td></tr><tr><td>8 </td><td>8 </td><td>1,228 </td></tr><tr><td>9 </td><td>27 </td><td>552 </td></tr><tr><td>10 </td><td>10 </td><td>247 </td></tr><tr><td>11 </td><td>24 </td><td>393 </td></tr><tr><td>12 </td><td>20 </td><td>601 </td></tr><tr><td>Total </td><td>204 </td><td>545 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196911" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV038TB5" class="table-wrap-title" data-id="DSV038TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Distribution of scaffolds assigned using the bin map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Number of assigned scaffolds </th><th>Average length of scaffold (kb) </th></tr></thead><tbody><tr><td>1 </td><td>39 </td><td>624 </td></tr><tr><td>2 </td><td>12 </td><td>512 </td></tr><tr><td>3 </td><td>24 </td><td>511 </td></tr><tr><td>4 </td><td>– </td><td>– </td></tr><tr><td>5 </td><td>14 </td><td>572 </td></tr><tr><td>6 </td><td>26 </td><td>457 </td></tr><tr><td>7 </td><td>– </td><td>– </td></tr><tr><td>8 </td><td>8 </td><td>1,228 </td></tr><tr><td>9 </td><td>27 </td><td>552 </td></tr><tr><td>10 </td><td>10 </td><td>247 </td></tr><tr><td>11 </td><td>24 </td><td>393 </td></tr><tr><td>12 </td><td>20 </td><td>601 </td></tr><tr><td>Total </td><td>204 </td><td>545 </td></tr></tbody></table></div><div content-id="DSV038TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Number of assigned scaffolds </th><th>Average length of scaffold (kb) </th></tr></thead><tbody><tr><td>1 </td><td>39 </td><td>624 </td></tr><tr><td>2 </td><td>12 </td><td>512 </td></tr><tr><td>3 </td><td>24 </td><td>511 </td></tr><tr><td>4 </td><td>– </td><td>– </td></tr><tr><td>5 </td><td>14 </td><td>572 </td></tr><tr><td>6 </td><td>26 </td><td>457 </td></tr><tr><td>7 </td><td>– </td><td>– </td></tr><tr><td>8 </td><td>8 </td><td>1,228 </td></tr><tr><td>9 </td><td>27 </td><td>552 </td></tr><tr><td>10 </td><td>10 </td><td>247 </td></tr><tr><td>11 </td><td>24 </td><td>393 </td></tr><tr><td>12 </td><td>20 </td><td>601 </td></tr><tr><td>Total </td><td>204 </td><td>545 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196911" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87196912 id="87196912" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this work, we constructed an ultra-high-density bin map for pepper via a sliding window approach and used it to detect QTLs controlling morphological traits. Our bin map helps improve the accuracy of the pepper reference genome in two ways. First, the bins can elucidate the order of unordered scaffolds in the chromosomes. Comparison of the genetic and physical locations of bins revealed extended physical length of bins and inconsistencies for bins located near the middle of chromosomes (Fig. <a reveal-id="DSV038F2" data-open="DSV038F2" class="link link-reveal link-table xref-fig">2</a>). These regions are likely centromeric or pericentromeric, as recombination rates tend to be low in the centromeric regions.<sup><a reveal-id="DSV038C27" data-open="DSV038C27" class="link link-ref link-reveal xref-bibr">27</a>–<a reveal-id="DSV038C29" data-open="DSV038C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In addition, unordered scaffolds are concentrated in putative centromeric regions of the reference genome.<sup><a reveal-id="DSV038C24" data-open="DSV038C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Thus, even though the lengths of bins in putative centromeric regions are longer than in other regions, they provide useful information for the reference genome by revealing the genetic order of bins. Second, our bin map refines the pepper reference genome by allowing us to anchor unassigned scaffolds to linkage groups. We were able to assign &gt;200 scaffolds to chromosomes (Table <a reveal-id="DSV038TB5" data-open="DSV038TB5" class="link link-reveal link-table xref-fig">5</a>). The total length of newly assigned scaffolds was 111 Mb, which is ∼3.2% of expected pepper genome size. This result shows that genetic maps constructed using the same population and sequencing data but with a different mapping approach can improve the reference genome by anchoring unassigned scaffolds.</p><p>The ultra-high-density bin map constructed in this study is the first SNP-based bin map in pepper, although the sliding window approach has been used to construct bin maps in cereal crops.<sup><a reveal-id="DSV038C17" data-open="DSV038C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV038C19" data-open="DSV038C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSV038C20" data-open="DSV038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> To determine recombination breakpoints of 150 RILs in rice and 708 F<sub>2</sub> plants in maize, 15 and 18 consecutive SNPs were used as one window in previous researches, respectively.<sup><a reveal-id="DSV038C17" data-open="DSV038C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV038C20" data-open="DSV038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> To construct our bin map in pepper, we modified the sliding window approach. Instead of fixing the number of SNPs included in the window, we set the window length as 5 Mb. Non-uniform distributions of SNPs were detected by counting the number of SNPs in every 1.5 Mb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>). When we set a window as 31 consecutive SNPs, recombination breakpoints were predicted to be excessively concentrated on regions where the density of SNPs was high (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span>). Sequencing or genotyping error from low-coverage sequences can produce low-quality SNPs,<sup><a reveal-id="DSV038C30" data-open="DSV038C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and there will be high correlation between the number of SNPs and recombination breakpoints. The sliding window approach can reduce the noise created by individual SNPs, but the original (set SNP number per window) approach cannot reduce the effects of SNP density. In contrast, recombination breakpoints detected using a 5-Mb window were less affected by the density of SNPs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span>). Our constructed bin map contains 2,578 bins, with an average of 33 recombination breakpoints per line, similar to the bin map of rice.<sup><a reveal-id="DSV038C17" data-open="DSV038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The number of recombination breakpoints was higher in rice and pepper RILs compared with that of maize F<sub>2</sub> plants, confirming that the construction of bin maps using the sliding window approach can detect the differences in recombination frequency between RIL and F<sub>2</sub> plants.</p><p>We confirmed the accuracy of our bin map by mapping the <em>Pun1</em> gene controlling pungency of pepper. There was ∼1.6 Mb difference between the position in this study and the physical location of the <em>Pun1</em> gene in the pepper genome. This may be caused by the misgenotyping of heterozygous <em>Pun1</em>/<em>pun1</em> alleles as homozygous <em>Pun1</em>/<em>Pun1</em> alleles. In this study, the <em>Pun1</em> gene was genotyped using dominant molecular markers,<sup><a reveal-id="DSV038C31" data-open="DSV038C31" class="link link-ref link-reveal xref-bibr">31</a></sup> but five RILs showed heterozygous bins near the <em>Pun1</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Therefore, it is highly possible that the discrepancy was due to the misgenotyped RILs. In addition to the <em>Pun1</em> gene, comparisons of the location of QTLs detected in this study with those in previous studies can verify the accuracy. Here, we detected <em>FP-12.2</em> as a major QTL controlling fruit position and explaining &gt;40% of the phenotypic variation. This QTL was located at 199.6 Mb on chromosome 12 in the ‘CM334’ reference genome. The <em>up</em> locus that is known to control fruit position is also located on chromosome 12 by previous research.<sup><a reveal-id="DSV038C32" data-open="DSV038C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The physical location of the <em>up</em> locus-linked marker is 197.1 Mb, and this marker is 4.3 cM apart from the <em>up</em> locus. Thus, mapping of the <em>Pun1</em> gene and the <em>up</em> locus using our bin map demonstrate the accuracy of the bin map, and show that QTL mapping with an ultra-high-density bin map will appropriately map the genetic locus.</p><p>Fruit-related traits are of vital importance for pepper researchers and breeders. A number of QTLs controlling fruit weight, diameter, and shape have been identified in intraspecific and interspecific populations.<sup><a reveal-id="DSV038C5" data-open="DSV038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV038C7" data-open="DSV038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV038C10" data-open="DSV038C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV038C33" data-open="DSV038C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSV038C34" data-open="DSV038C34" class="link link-ref link-reveal xref-bibr">34</a></sup> In particular, QTLs controlling fruit weight were previously detected on chromosome 2.<sup><a reveal-id="DSV038C5" data-open="DSV038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV038C7" data-open="DSV038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV038C8" data-open="DSV038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> However, direct comparison of QTL locations has been difficult due to the lack of common markers. Therefore, we compared the physical locations of markers linked to QTLs using the ‘CM334’ reference genome sequence. <em>fw2.1</em> detected in <em>C. chinense</em> and <em>C. frutescens</em> introgression lines were located between 156 and 158 Mb on chromosome 2.<sup><a reveal-id="DSV038C8" data-open="DSV038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The QTL with the same name as <em>fw2.1</em> was detected in a backcross population from a cross between <em>C. annuum</em> and <em>C. frutescens</em> and was also located ∼156 Mb of chromosome 2.<sup><a reveal-id="DSV038C7" data-open="DSV038C7" class="link link-ref link-reveal xref-bibr">7</a></sup><em>FW-2.1</em> and <em>FW-2.2</em>, for fruit weight, detected in our research were located at 1.5 and 146 Mb, respectively. These findings show that <em>FW-2.2</em> could be the same QTL as that previously reported, and that <em>FW-2.1</em> is a newly detected QTL controlling fruit weight on chromosome 2. A slight difference in the physical location of <em>FW-2.2</em> with <em>fw2.1</em> could be caused by different statistical methods used for QTL analysis, or from the difference of the species used for the construction of genetic map. Using the same approach, we compared the location of QTLs controlling fruit shape on chromosome 3. The most tightly linked marker to the QTL, TG130, was located ∼194 Mb on chromosome 3.<sup><a reveal-id="DSV038C5" data-open="DSV038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV038C7" data-open="DSV038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV038C9" data-open="DSV038C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In the current work, we identified <em>FS-3.1</em> and <em>FS-3.2</em>. Due to rearrangement of bins near <em>FS-3.1</em>, the physical location of <em>FS-3.1</em> was spread in the ranges 60–75, 98–124, and 174–200 Mb on chromosome 3. <em>FS-3.2</em> was located ∼206–211 Mb. It was not possible to determine whether the QTLs were the same or not, but the repeated detection of QTLs on chromosomes 2 and 3 for fruit weight and fruit shape demonstrates the significance and importance of those QTLs. Therefore, fine-mapping of <em>FW-2.1</em>, <em>FW-2.2</em>, <em>FS-3.1</em>, and <em>FW-3.2</em> as well as comparative study with syntenic QTLs in tomato will likely be helpful to elucidate the underlying genetic factors.</p><p>In summary, in this study, we identified QTLs for not only fruit-related traits but also plant architecture- and leaf-related traits. In addition, we report QTLs controlling flower-related traits in pepper for the first time. The ultra-high-density bin map we constructed using a sliding window approach improves the pepper genetic map and increases the speed of QTL mapping. This bin map will help to revise the reference genome and as all bins were determined based on SNPs, the bins can be used directly to develop molecular markers linked to QTLs.</p>                            <h2 scrollto-destination=87196918 id="87196918" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was carried out with the support of Cooperative Research Program for Agriculture Science and Technology Development (Plant Molecular Breeding Center No. PJ01120501 and the Agricultural Genome Center No. PJ01120401 and No. PJ01127501), Rural Development Administration, Republic of Korea. Funding to pay the Open Access publication charges for this article was provided by Seoul National University.</p>                            <h2 scrollto-destination=87196920 id="87196920" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV038C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paran</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">van der Knaap</div> <div class="given-names">E.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Genetic and molecular regulation of fruit and plant domestication traits in tomato and pepper</div>, <div class="source">J. Exp. Bot.</div> , <div class="volume">58</div>, <div class="fpage">3841</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20molecular%20regulation%20of%20fruit%20and%20plant%20domestication%20traits%20in%20tomato%20and%20pepper&amp;author=I.%20Paran&amp;author=E.%20van%20der%20Knaap&amp;publication_year=2007&amp;journal=J.%20Exp.%20Bot.&amp;volume=58&amp;pages=3841-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erm257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferm257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferm257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18037678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eshbaugh</div>
               <div class="given-names">W.H.</div>
            </div>
          </div>
          <div class="year">1980</div>, <div class="article-title">The taxonomy of the genus <em>Capsicum</em> (Solanaceae)</div>, <div class="source">Phytologia</div> , <div class="volume">47</div>, <div class="fpage">153</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20taxonomy%20of%20the%20genus%20Capsicum%20(Solanaceae)&amp;author=W.H.%20Eshbaugh&amp;publication_year=1980&amp;journal=Phytologia&amp;volume=47&amp;pages=153-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5962/bhl.part.4455" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5962%2Fbhl.part.4455" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5962%2Fbhl.part.4455"> </span></div> </div></div></div></div></div><div content-id="DSV038C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sudré</div>
               <div class="given-names">C.P.</div>
            </div>, <div class="name string-name"><div class="surname">Gonçalves</div> <div class="given-names">L.S.A.</div></div>, <div class="name string-name"><div class="surname">Rodrigues</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Júnior</div> <div class="given-names">A.T.D.A.</div></div>, <div class="name string-name"><div class="surname">Riva-Souza</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Bento</div> <div class="given-names">C.D.S.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Genetic variability in domesticated <em>Capsicum</em> spp. as assessed by morphological and agronomic data in mixed statistical analysis</div>, <div class="source">Genet. Mol. Res.</div> , <div class="volume">9</div>, <div class="fpage">283</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variability%20in%20domesticated%20Capsicum%20spp.%20as%20assessed%20by%20morphological%20and%20agronomic%20data%20in%20mixed%20statistical%20analysis&amp;author=C.P.%20Sudr%C3%A9&amp;author=L.S.A.%20Gon%C3%A7alves&amp;author=R.%20Rodrigues&amp;author=A.T.D.A.%20J%C3%BAnior&amp;author=E.M.%20Riva-Souza&amp;author=C.D.S.%20Bento&amp;publication_year=2010&amp;journal=Genet.%20Mol.%20Res.&amp;volume=9&amp;pages=283-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4238/vol9-1gmr698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4238%2Fvol9-1gmr698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4238%2Fvol9-1gmr698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20198584" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thul</div>
               <div class="given-names">S.T.</div>
            </div>, <div class="name string-name"><div class="surname">Lal</div> <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Shasany</div> <div class="given-names">A.K.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Estimation of phenotypic divergence in a collection of <em>Capsicum</em> species for yield-related traits</div>, <div class="source">Euphytica</div> , <div class="volume">168</div>, <div class="fpage">189</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20phenotypic%20divergence%20in%20a%20collection%20of%20Capsicum%20species%20for%20yield-related%20traits&amp;author=S.T.%20Thul&amp;author=R.K.%20Lal&amp;author=A.K.%20Shasany&amp;publication_year=2009&amp;journal=Euphytica&amp;volume=168&amp;pages=189-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-009-9882-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-009-9882-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-009-9882-y"> </span></div> </div></div></div></div></div><div content-id="DSV038C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ben-Chaim</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Paran</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Grube</div> <div class="given-names">R.C.</div></div>, <div class="name string-name"><div class="surname">Jahn</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wijk</div> <div class="given-names">R.V.</div></div>, <div class="name string-name"><div class="surname">Peleman</div> <div class="given-names">J.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">QTL mapping of fruit-related traits in pepper (<em>Capsicum annuum</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">102</div>, <div class="fpage">1016</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20of%20fruit-related%20traits%20in%20pepper%20(Capsicum%20annuum)&amp;author=A.%20Ben-Chaim&amp;author=I.%20Paran&amp;author=R.C.%20Grube&amp;author=M.%20Jahn&amp;author=R.V.%20Wijk&amp;author=J.%20Peleman&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=1016-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220000461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220000461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220000461"> </span></div> </div></div></div></div></div><div content-id="DSV038C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chaim</div>
               <div class="given-names">A.B.</div>
            </div>, <div class="name string-name"><div class="surname">Borovsky</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Jong</div> <div class="given-names">W.D.</div></div>, <div class="name string-name"><div class="surname">Paran</div> <div class="given-names">I.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Linkage of the <em>A</em> locus for the presence of anthocyanin and <em>fs10.1</em>, a major fruit-shape QTL in pepper</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">106</div>, <div class="fpage">889</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20of%20the%20A%20locus%20for%20the%20presence%20of%20anthocyanin%20and%20fs10.1%2C%20a%20major%20fruit-shape%20QTL%20in%20pepper&amp;author=A.B.%20Chaim&amp;author=Y.%20Borovsky&amp;author=W.D.%20Jong&amp;author=I.%20Paran&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=889-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-1132-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-1132-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-1132-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12647064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rao</div>
               <div class="given-names">G.U.</div>
            </div>, <div class="name string-name"><div class="surname">Chaim</div> <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Borovsky</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Paran</div> <div class="given-names">I.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Mapping of yield-related QTLs in pepper in an interspecific cross of <em>Capsicum annuum</em> and <em>C. frutescens</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">106</div>, <div class="fpage">1457</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20yield-related%20QTLs%20in%20pepper%20in%20an%20interspecific%20cross%20of%20Capsicum%20annuum%20and%20C.%20frutescens&amp;author=G.U.%20Rao&amp;author=A.B.%20Chaim&amp;author=Y.%20Borovsky&amp;author=I.%20Paran&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=1457-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1204-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1204-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1204-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12750789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zygier</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Chaim</div> <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Efrati</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kaluzky</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Borovsky</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Paran</div> <div class="given-names">I.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">QTLs mapping for fruit size and shape in chromosomes 2 and 4 in pepper and a comparison of the pepper QTL map with that of tomato</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">111</div>, <div class="fpage">437</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTLs%20mapping%20for%20fruit%20size%20and%20shape%20in%20chromosomes%202%20and%204%20in%20pepper%20and%20a%20comparison%20of%20the%20pepper%20QTL%20map%20with%20that%20of%20tomato&amp;author=S.%20Zygier&amp;author=A.B.%20Chaim&amp;author=A.%20Efrati&amp;author=G.%20Kaluzky&amp;author=Y.%20Borovsky&amp;author=I.%20Paran&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=437-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-2015-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-2015-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-2015-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15983758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ben-Chaim</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Borovsky</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Rao</div> <div class="given-names">G.U.</div></div>, <div class="name string-name"><div class="surname">Tanyolac</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Paran</div> <div class="given-names">I.</div></div></div>
          <div class="year">2003</div>, <div class="article-title"><em>fs3.1</em>: a major fruit shape QTL conserved in <em>Capsicum</em></div>, <div class="source">Genome</div> , <div class="volume">46</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=fs3.1%3A%20a%20major%20fruit%20shape%20QTL%20conserved%20in%20Capsicum&amp;author=A.%20Ben-Chaim&amp;author=Y.%20Borovsky&amp;author=G.U.%20Rao&amp;author=B.%20Tanyolac&amp;author=I.%20Paran&amp;publication_year=2003&amp;journal=Genome&amp;volume=46&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g02-096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg02-096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg02-096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12669791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yarnes</div>
               <div class="given-names">S.C.</div>
            </div>, <div class="name string-name"><div class="surname">Ashrafi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Reyes-Chin-Wo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hill</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Stoffel</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Identification of QTLs for capsaicinoids, fruit quality, and plant architecture-related traits in an interspecific <em>Capsicum</em> RIL population</div>, <div class="source">Genome</div> , <div class="volume">56</div>, <div class="fpage">61</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20QTLs%20for%20capsaicinoids%2C%20fruit%20quality%2C%20and%20plant%20architecture-related%20traits%20in%20an%20interspecific%20Capsicum%20RIL%20population&amp;author=S.C.%20Yarnes&amp;author=H.%20Ashrafi&amp;author=S.%20Reyes-Chin-Wo&amp;author=T.A.%20Hill&amp;author=K.M.%20Stoffel&amp;author=A.%20Van%20Deynze&amp;publication_year=2013&amp;journal=Genome&amp;volume=56&amp;pages=61-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-2012-0083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-2012-0083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-2012-0083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23379339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kang</div>
               <div class="given-names">B.C.</div>
            </div>, <div class="name string-name"><div class="surname">Nahm</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Huh</div> <div class="given-names">J.H.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">An interspecific (<em>Capsicum annuum</em>× <em>C. chinese</em>) F2 linkage map in pepper using RFLP and AFLP markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">102</div>, <div class="fpage">531</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20interspecific%20(Capsicum%20annuum%C3%97%20C.%20chinese)%20F2%20linkage%20map%20in%20pepper%20using%20RFLP%20and%20AFLP%20markers&amp;author=B.C.%20Kang&amp;author=S.H.%20Nahm&amp;author=J.H.%20Huh&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=531-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051678" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051678" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051678"> </span></div> </div></div></div></div></div><div content-id="DSV038C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Barchi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Bonnet</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Boudet</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">A high-resolution, intraspecific linkage map of pepper (<em>Capsicum annuum</em> L.) and selection of reduced recombinant inbred line subsets for fast mapping</div>, <div class="source">Genome</div> , <div class="volume">50</div>, <div class="fpage">51</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%2C%20intraspecific%20linkage%20map%20of%20pepper%20(Capsicum%20annuum%20L.)%20and%20selection%20of%20reduced%20recombinant%20inbred%20line%20subsets%20for%20fast%20mapping&amp;author=L.%20Barchi&amp;author=J.%20Bonnet&amp;author=C.%20Boudet&amp;publication_year=2007&amp;journal=Genome&amp;volume=50&amp;pages=51-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g06-140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg06-140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg06-140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17546071" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hill</div>
               <div class="given-names">T.A.</div>
            </div>, <div class="name string-name"><div class="surname">Ashrafi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Reyes-Chin-Wo</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Characterization of <em>Capsicum annuum</em> genetic diversity and population structure based on parallel polymorphism discovery with a 30K unigene Pepper GeneChip</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e56200</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Capsicum%20annuum%20genetic%20diversity%20and%20population%20structure%20based%20on%20parallel%20polymorphism%20discovery%20with%20a%2030K%20unigene%20Pepper%20GeneChip&amp;author=T.A.%20Hill&amp;author=H.%20Ashrafi&amp;author=S.%20Reyes-Chin-Wo&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e56200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0056200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0056200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0056200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23409153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nielsen</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Paul</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Albrechtsen</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">Y.S.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Genotype and SNP calling from next-generation sequencing data</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">12</div>, <div class="fpage">443</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotype%20and%20SNP%20calling%20from%20next-generation%20sequencing%20data&amp;author=R.%20Nielsen&amp;author=J.S.%20Paul&amp;author=A.%20Albrechtsen&amp;author=Y.S.%20Song&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=443-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21587300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Sidore</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">H.M.</div></div>, <div class="name string-name"><div class="surname">Boehnke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Abecasis</div> <div class="given-names">G.R.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Low-coverage sequencing: implications for design of complex trait association studies</div>, <div class="source">Genome Res.</div> , <div class="volume">21</div>, <div class="fpage">940</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low-coverage%20sequencing%3A%20implications%20for%20design%20of%20complex%20trait%20association%20studies&amp;author=Y.%20Li&amp;author=C.%20Sidore&amp;author=H.M.%20Kang&amp;author=M.%20Boehnke&amp;author=G.R.%20Abecasis&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=940-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.117259.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.117259.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.117259.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21460063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pasaniuc</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Rohland</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">McLaren</div> <div class="given-names">P.J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Extremely low-coverage sequencing and imputation increases power for genome-wide association studies</div>, <div class="source">Nat. Genet.</div> , <div class="volume">44</div>, <div class="fpage">631</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extremely%20low-coverage%20sequencing%20and%20imputation%20increases%20power%20for%20genome-wide%20association%20studies&amp;author=B.%20Pasaniuc&amp;author=N.%20Rohland&amp;author=P.J.%20McLaren&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=631-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2283" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2283"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22610117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">Q.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">High-throughput genotyping by whole-genome resequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">19</div>, <div class="fpage">1068</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20by%20whole-genome%20resequencing&amp;author=X.%20Huang&amp;author=Q.%20Feng&amp;author=Q.%20Qian&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1068-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089516.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089516.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089516.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Mapping 49 quantitative trait loci at high resolution through sequencing-based genotyping of rice recombinant inbred lines</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">122</div>, <div class="fpage">327</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%2049%20quantitative%20trait%20loci%20at%20high%20resolution%20through%20sequencing-based%20genotyping%20of%20rice%20recombinant%20inbred%20lines&amp;author=L.%20Wang&amp;author=A.%20Wang&amp;author=X.%20Huang&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=327-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1449-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1449-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1449-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20878143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zou</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Zhai</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">Q.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Identification of QTLs for eight agronomically important traits using an ultra-high-density map based on SNPs generated from high-throughput sequencing in sorghum under contrasting photoperiods</div>, <div class="source">J. Exp. Bot.</div> , <div class="volume">63</div>, <div class="fpage">5451</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20QTLs%20for%20eight%20agronomically%20important%20traits%20using%20an%20ultra-high-density%20map%20based%20on%20SNPs%20generated%20from%20high-throughput%20sequencing%20in%20sorghum%20under%20contrasting%20photoperiods&amp;author=G.%20Zou&amp;author=G.%20Zhai&amp;author=Q.%20Feng&amp;publication_year=2012&amp;journal=J.%20Exp.%20Bot.&amp;volume=63&amp;pages=5451-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/ers205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Fers205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Fers205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22859680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Dong</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">An ultra-high density bin-map for rapid QTL mapping for tassel and ear architecture in a large F<sub>2</sub> maize population</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">433</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ultra-high%20density%20bin-map%20for%20rapid%20QTL%20mapping%20for%20tassel%20and%20ear%20architecture%20in%20a%20large%20F2%20maize%20population&amp;author=Z.%20Chen&amp;author=B.%20Wang&amp;author=X.%20Dong&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=433" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-433" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-433" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-433"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24898122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">De Givry</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Bouchez</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chabrier</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Milan</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Schiex</div> <div class="given-names">T.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">CARHTA GENE: multipopulation integrated genetic and radiation hybrid mapping</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">1703</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CARHTA%20GENE%3A%20multipopulation%20integrated%20genetic%20and%20radiation%20hybrid%20mapping&amp;author=S.%20De%20Givry&amp;author=M.%20Bouchez&amp;author=P.%20Chabrier&amp;author=D.%20Milan&amp;author=T.%20Schiex&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1703-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti222" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti222" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti222"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15598829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Voorrips</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="year">2002</div>, <div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J Hered.</div> , <div class="volume">93</div>, <div class="fpage">77</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brewer</div>
               <div class="given-names">M.T.</div>
            </div>, <div class="name string-name"><div class="surname">Lang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Fujimura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Dujmovic</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Gray</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">van der Knaap</div> <div class="given-names">E.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Development of a controlled vocabulary and software application to analyze fruit shape variation in tomato and other plant species</div>, <div class="source">Plant Physiol.</div> , <div class="volume">141</div>, <div class="fpage">15</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20controlled%20vocabulary%20and%20software%20application%20to%20analyze%20fruit%20shape%20variation%20in%20tomato%20and%20other%20plant%20species&amp;author=M.T.%20Brewer&amp;author=L.%20Lang&amp;author=K.%20Fujimura&amp;author=N.%20Dujmovic&amp;author=S.%20Gray&amp;author=E.%20van%20der%20Knaap&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=141&amp;pages=15-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.077867" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.077867" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.077867"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16684933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yeom</div> <div class="given-names">S.I.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome sequence of the hot pepper provides insights into the evolution of pungency in <em>Capsicum</em> species</div>, <div class="source">Nat. Genet.</div> , <div class="volume">46</div>, <div class="fpage">270</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20hot%20pepper%20provides%20insights%20into%20the%20evolution%20of%20pungency%20in%20Capsicum%20species&amp;author=S.%20Kim&amp;author=M.%20Park&amp;author=S.I.%20Yeom&amp;publication_year=2014&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=270-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24441736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Basten</div> <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">Z.B.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Windows QTL Cartographer 2.5</div>. <div class="publisher-name">Department of Statistics. North Carolina State University</div>. <div class="comment"><a class="link link-uri" href="http://statgen.ncsu.edu/qtlcart/WQTLCart.htm">http://statgen.ncsu.edu/qtlcart/WQTLCart.htm</a></div>.</div></div></div></div><div content-id="DSV038C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stewart</div>
               <div class="given-names">C.</div>
              Jr
            </div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">B.C.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">The <em>Pun1</em> gene for pungency in pepper encodes a putative acyltransferase</div>, <div class="source">Plant J.</div> , <div class="volume">42</div>, <div class="fpage">675</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pun1%20gene%20for%20pungency%20in%20pepper%20encodes%20a%20putative%20acyltransferase&amp;author=C.%20Stewart&amp;author=B.C.%20Kang&amp;author=K.%20Liu&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=42&amp;pages=675-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.42.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.42.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.42.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15918882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Puchta</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Hohn</div> <div class="given-names">B.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">From centiMorgans to base pairs: homologous recombination in plants</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">1</div>, <div class="fpage">340</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20centiMorgans%20to%20base%20pairs%3A%20homologous%20recombination%20in%20plants&amp;author=H.%20Puchta&amp;author=B.%20Hohn&amp;publication_year=1996&amp;journal=Trends%20Plant%20Sci.&amp;volume=1&amp;pages=340-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(96)82595-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(96)82595-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(96)82595-0"> </span></div> </div></div></div></div></div><div content-id="DSV038C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Spindel</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">126</div>, <div class="fpage">2699</div>–<div class="lpage">716</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bridging%20the%20genotyping%20gap%3A%20using%20genotyping%20by%20sequencing%20(GBS)%20to%20add%20high-density%20SNP%20markers%20and%20new%20value%20to%20traditional%20bi-parental%20mapping%20and%20breeding%20populations&amp;author=J.%20Spindel&amp;author=M.%20Wright&amp;author=C.%20Chen&amp;publication_year=2013&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=126&amp;pages=2699-716" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-013-2166-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2166-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-013-2166-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23918062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Si</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Widely distributed hot and cold spots in meiotic recombination as shown by the sequencing of rice F2 plants</div>, <div class="source">New Phytol.</div> , <div class="volume">206</div>, <div class="fpage">1491</div>–<div class="lpage">502</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widely%20distributed%20hot%20and%20cold%20spots%20in%20meiotic%20recombination%20as%20shown%20by%20the%20sequencing%20of%20rice%20F2%20plants&amp;author=W.%20Si&amp;author=Y.%20Yuan&amp;author=J.%20Huang&amp;publication_year=2015&amp;journal=New%20Phytol.&amp;volume=206&amp;pages=1491-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25664766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sims</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Sudbery</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Ilott</div> <div class="given-names">N.E.</div></div>, <div class="name string-name"><div class="surname">Heger</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Sequencing depth and coverage: key considerations in genomic analyses</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">15</div>, <div class="fpage">121</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20depth%20and%20coverage%3A%20key%20considerations%20in%20genomic%20analyses&amp;author=D.%20Sims&amp;author=I.%20Sudbery&amp;author=N.E.%20Ilott&amp;author=A.%20Heger&amp;author=C.P.%20Ponting&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=15&amp;pages=121-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24434847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Han</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Jeong</div> <div class="given-names">H.-J.</div></div>, <div class="name string-name"><div class="surname">Sung</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Biosynthesis of capsinoid is controlled by the <em>Pun1</em> locus in pepper</div>, <div class="source">Mol Breed</div> , <div class="volume">31</div>, <div class="fpage">537</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20capsinoid%20is%20controlled%20by%20the%20Pun1%20locus%20in%20pepper&amp;author=K.%20Han&amp;author=H.-J.%20Jeong&amp;author=J.%20Sung&amp;publication_year=2012&amp;journal=Mol%20Breed&amp;volume=31&amp;pages=537-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-012-9811-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-012-9811-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-012-9811-y"> </span></div> </div></div></div></div></div><div content-id="DSV038C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">H.-R.</div>
            </div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">M.-C.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.-J.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">S.-W.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">B.-D.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Marker development for erect versus pendant-orientated fruit in <em>Capsicum annuum</em> L</div>, <div class="source">Mol. Cells</div> , <div class="volume">26</div>, <div class="fpage">548</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Marker%20development%20for%20erect%20versus%20pendant-orientated%20fruit%20in%20Capsicum%20annuum%20L&amp;author=H.-R.%20Lee&amp;author=M.-C.%20Cho&amp;author=H.-J.%20Kim&amp;author=S.-W.%20Park&amp;author=B.-D.%20Kim&amp;publication_year=2008&amp;journal=Mol.%20Cells&amp;volume=26&amp;pages=548-53" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18824887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Barchi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Lefebvre</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Sage-Palloix</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Lanteri</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Palloix</div> <div class="given-names">A.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">QTL analysis of plant development and fruit traits in pepper and performance of selective phenotyping</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">118</div>, <div class="fpage">1157</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20plant%20development%20and%20fruit%20traits%20in%20pepper%20and%20performance%20of%20selective%20phenotyping&amp;author=L.%20Barchi&amp;author=V.%20Lefebvre&amp;author=A.M.%20Sage-Palloix&amp;author=S.%20Lanteri&amp;author=A.%20Palloix&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=1157-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-0970-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-0970-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-0970-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19219599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV038C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV038C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Borovsky</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Paran</div> <div class="given-names">I.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Characterization of <em>fs10.1</em>, a major QTL controlling fruit elongation in <em>Capsicum</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">123</div>, <div class="fpage">657</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20fs10.1%2C%20a%20major%20QTL%20controlling%20fruit%20elongation%20in%20Capsicum&amp;author=Y.%20Borovsky&amp;author=I.%20Paran&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=657-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1615-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1615-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1615-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21603875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745287&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv038/2/dsv038_Supplementary_Data.zip?Expires=2147483647&amp;Signature=qLD68WTlDbxXJW2Cyoqen-ZW8IUi1UdCXAqCzZhdXQ20gSZ3K2mqNFwdWUpIiYV-sRsv14rviro4iLZHb5i-44b3U4ZC6vzVv8~mO1pbB4XcOpObGHZoe-CUPeV-~WnZHtFcAtCHPJIWpTTgyaQPLi9SI15doqNt7d8iAriL7phZEKTzSF0Yh1DbtIv7tOUyl96NGQUgsQYxlSxLUIrLX5jnRF8fcd878svr9kQCxvo3uLnUxmoR52iI8hrwbCmcwJJHKT79rmVyecoM0Bg17Q7lknib8EObKKmREgeAL1FXZ~yy9sLrUHmclvsc5MxuNafUTqi2wQ4brKGuYfDbkw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,070</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">15</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv038"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745287"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f2%2f81%2f1745287">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000375088800001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (15)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F2%2F81%2F1745287" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745287'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="ultra-high-density bin map facilitates high-throughput QTL mapping of horticultural traits in pepper (Capsicum annuum)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv038");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f81%2f1745287">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

