

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/71/4265514","@type":"ScholarlyArticle","name":"Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens","datePublished":"2017-10-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx039","keywords":["circular RNA","chicken","skeletal muscle","embryonic development","cell proliferation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/71/4265514","author":[{"name":"Ouyang, Hongjia","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Chen, Xiaolan","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Wang, Zhijun","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Yu, Jiao","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Jia, Xinzheng","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Li, Zhenhui","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Luo, Wei","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Abdalla, Bahareldin Ali","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Jebessa, Endashaw","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Nie, Qinghua","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"},{"name":"Zhang, Xiquan","affiliation":"Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China","@type":"Person"}],"description":"Abstract.  The growth and development of skeletal muscle is regulated by proteins as well as non-coding RNAs. Circular RNAs (circRNAs) are universally expressed","pageStart":"71","pageEnd":"86","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens" />
    <meta property="og:description" content="Abstract.  The growth and development of skeletal muscle is regulated by proteins as well as non-coding RNAs. Circular RNAs (circRNAs) are universally expressed" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/71/4265514" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-10-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ouyang, Hongjia" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Chen, Xiaolan" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Wang, Zhijun" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Yu, Jiao" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Jia, Xinzheng" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Li, Zhenhui" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Luo, Wei" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Abdalla, Bahareldin Ali" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Jebessa, Endashaw" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Nie, Qinghua" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author" content="Zhang, Xiquan" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China" /><meta name="citation_title" content="Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens" /><meta name="citation_firstpage" content="71" /><meta name="citation_lastpage" content="86" /><meta name="citation_doi" content="10.1093/dnares/dsx039" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Landscape of transcription in human cells,;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=489;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=Long noncoding RNAs: cellular address codes in development and disease,;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=152;  citation_pages=1298-307; " /><meta name="citation_reference" content="citation_title=MicroRNAs: target recognition and regulatory functions,;  citation_journal_title=Cell;  citation_year=2009;  citation_volume=136;  citation_pages=215-33; " /><meta name="citation_reference" content="citation_title=Evolution and functions of long noncoding RNAs,;  citation_journal_title=Cell;  citation_year=2009;  citation_volume=136;  citation_pages=629-41; " /><meta name="citation_reference" content="citation_title=Modular regulatory principles of large non-coding RNAs,;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=482;  citation_pages=339-46; " /><meta name="citation_reference" content="citation_title=Viroids are single-stranded covalently closed circular RNA molecules existing as highly base-paired rod-like structures;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1976;  citation_volume=73;  citation_pages=3852-6; " /><meta name="citation_reference" content="citation_title=Circular transcripts of the testis-determining gene Sry in adult mouse testis,;  citation_journal_title=Cell;  citation_year=1993;  citation_volume=73;  citation_pages=1019-30; " /><meta name="citation_reference" content="citation_title=Circular RNAs from transcripts of the rat cytochrome P450 2C24 gene: correlation with exon skipping,;  citation_journal_title=Proc Natl Acad. Sci. U S A;  citation_year=1996;  citation_volume=93;  citation_pages=6536-41; " /><meta name="citation_reference" content="citation_title=Exon skipping and circular RNA formation in transcripts of the human cytochrome P-450 2C18 gene in epidermis and of the rat androgen binding protein gene in testis,;  citation_journal_title=Mol. Cell. Biol;  citation_year=1997;  citation_volume=17;  citation_pages=2985-93; " /><meta name="citation_reference" content="citation_title=A circularized sodium-calcium exchanger exon 2 transcript,;  citation_journal_title=J. Biol. Chem;  citation_year=1999;  citation_volume=274;  citation_pages=8153-60; " /><meta name="citation_reference" content="citation_title=Mis-splicing yields circular RNA molecules,;  citation_journal_title=FASEB J;  citation_year=1993;  citation_volume=7;  citation_pages=155-60; " /><meta name="citation_reference" content="citation_title=Cell-type specific features of circular RNA expression,;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003777" /><meta name="citation_reference" content="citation_title=Circular RNAs are a large class of animal RNAs with regulatory potency,;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=495;  citation_pages=333-8; " /><meta name="citation_reference" content="citation_title=Natural RNA circles function as efficient microRNA sponges,;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=495;  citation_pages=384-8; " /><meta name="citation_reference" content="citation_title=Circular RNAs in the mammalian brain are highly abundant, conserved, and dynamically expressed,;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=58;  citation_pages=870-85; " /><meta name="citation_reference" content="citation_title=Analysis of intron sequences reveals hallmarks of circular RNA biogenesis in animals,;  citation_journal_title=Cell Rep;  citation_year=2015;  citation_volume=10;  citation_pages=170-7; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of drosophila circular RNAs reveals their structural and sequence properties and age-dependent neural accumulation,;  citation_journal_title=Cell Rep;  citation_year=2014;  citation_volume=9;  citation_pages=1966-80; " /><meta name="citation_reference" content="citation_title=Circular RNA is expressed across the eukaryotic tree of life,;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e90859" /><meta name="citation_reference" content="citation_title=Widespread noncoding circular RNAs in plants,;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=208;  citation_pages=88-95; " /><meta name="citation_reference" content="citation_title=Expanded identification and characterization of mammalian circular RNAs,;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=409" /><meta name="citation_reference" content="citation_title=Circular RNAs are the predominant transcript isoform from hundreds of human genes in diverse cell types,;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e30733" /><meta name="citation_reference" content="citation_title=Circular RNAs are abundant, conserved, and associated with ALU repeats,;  citation_journal_title=RNA;  citation_year=2013;  citation_volume=19;  citation_pages=141-57; " /><meta name="citation_reference" content="citation_title=Circular intronic long noncoding RNAs,;  citation_journal_title=Mol. Cell;  citation_year=2013;  citation_volume=51;  citation_pages=792-806; " /><meta name="citation_reference" content="citation_title=Exon-intron circular RNAs regulate transcription in the nucleus;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2015;  citation_volume=22;  citation_pages=256-64; " /><meta name="citation_reference" content="citation_title=circRNA biogenesis competes with pre-mRNA splicing,;  citation_journal_title=Mol. Cell;  citation_year=2014;  citation_volume=56;  citation_pages=55-66; " /><meta name="citation_reference" content="citation_title=The complexity of the translation ability of circRNAs,;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2016;  citation_volume=1859;  citation_pages=1245-51; " /><meta name="citation_reference" content="citation_title=circRNADb: A comprehensive database for human circular RNAs with protein-coding annotations,;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=34985" /><meta name="citation_reference" content="citation_title=Extensive translation of circular RNAs driven by N6-methyladenosine,;  citation_journal_title=Cell Res;  citation_year=2017;  citation_volume=27;  citation_pages=626-41; " /><meta name="citation_reference" content="citation_title=Circ-ZNF609 Is a Circular RNA that Can Be Translated and Functions in Myogenesis,;  citation_journal_title=Mol. Cell;  citation_year=2017;  citation_volume=66;  citation_pages=22-37; " /><meta name="citation_reference" content="citation_title=Signaling pathways in skeletal muscle remodeling,;  citation_journal_title=Annu. Rev. Biochem;  citation_year=2006;  citation_volume=75;  citation_pages=19-37; " /><meta name="citation_reference" content="citation_title=Activity-dependent signaling pathways controlling muscle diversity and plasticity;  citation_journal_title=Physiology (Bethesda);  citation_year=2007;  citation_volume=22;  citation_pages=269-78; " /><meta name="citation_reference" content="citation_title=MicroRNAs involved in skeletal muscle differentiation,;  citation_journal_title=J. Genet. Genomics;  citation_year=2013;  citation_volume=40;  citation_pages=107-16; " /><meta name="citation_reference" content="citation_title=Noncoding RNAs, Emerging Regulators of Skeletal Muscle Development and Diseases,;  citation_journal_title=Biomed. Res. Int;  citation_year=2015;  citation_volume=2015;  citation_pages=676575" /><meta name="citation_reference" content="citation_title=Circular RNAs: Novel Regulators of Neuronal Development,;  citation_journal_title=Front. Mol. Neurosci;  citation_year=2016;  citation_volume=9;  citation_pages=74" /><meta name="citation_reference" content="citation_title=Disease onset and aging in the world of circular RNAs;  citation_journal_title=J. Transl. Sci;  citation_year=2016;  citation_volume=2;  citation_pages=327-9; " /><meta name="citation_reference" content="citation_title=The emerging landscape of circular RNA in life processes;  citation_journal_title=RNA Biol;  citation_year=2016;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Circular RNAs in monkey muscle: age-dependent changes,;  citation_journal_title=Aging (Albany NY);  citation_year=2015;  citation_volume=7;  citation_pages=903-10; " /><meta name="citation_reference" content="citation_title=A simple and universal method for molecular sexing of non-ratite birds,;  citation_journal_title=J. Avian Biol;  citation_year=1999;  citation_volume=30;  citation_pages=116-21; " /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation,;  citation_journal_title=Nat. Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=CIRI: an efficient and unbiased algorithm for de novo circular RNA identification,;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Spatio-temporal regulation of circular RNA expression during porcine embryonic brain development;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=245" /><meta name="citation_reference" content="citation_title=Circular RNA of cattle casein genes are highly expressed in bovine mammary gland,;  citation_journal_title=J. Dairy Sci;  citation_year=2016;  citation_volume=99;  citation_pages=4750-60; " /><meta name="citation_reference" content="citation_title=Pattern of Pax7 expression during myogenesis in the posthatch chicken establishes a model for satellite cell differentiation and renewal;  citation_journal_title=Dev. Dyn;  citation_year=2004;  citation_volume=231;  citation_pages=489-502; " /><meta name="citation_reference" content="citation_title=Enhancement of meat production by environmental manipulations in embryo and young broilers,;  citation_journal_title=World. Poultry Sci. J;  citation_year=2006;  citation_volume=62;  citation_pages=485-97; " /><meta name="citation_reference" content="citation_title=Statistically based splicing detection reveals neural enrichment and tissue-specific induction of circular RNA during human fetal development,;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=126" /><meta name="citation_reference" content="citation_title=Circular RNA ITCH has inhibitory effect on ESCC by suppressing the Wnt/beta-catenin pathway,;  citation_journal_title=Oncotarget;  citation_year=2015;  citation_volume=6;  citation_pages=6001-13; " /><meta name="citation_reference" content="citation_title=Circular RNA Related to the Chondrocyte ECM Regulates MMP13 Expression by Functioning as a MiR-136 ‘Sponge’ in Human Cartilage Degradation,;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=22572" /><meta name="citation_reference" content="citation_title=A circular RNA protects the heart from pathological hypertrophy and heart failure by targeting miR-223,;  citation_journal_title=Eur. Heart J;  citation_year=2016;  citation_volume=37;  citation_pages=2602-11; " /><meta name="citation_reference" content="citation_title=Foxo3 activity promoted by non-coding effects of circular RNA and Foxo3 pseudogene in the inhibition of tumor growth and angiogenesis,;  citation_journal_title=Oncogene;  citation_year=2016;  citation_volume=35;  citation_pages=3919-31; " /><meta name="citation_reference" content="citation_title=Circular RNA profiling reveals an abundant circHIPK3 that regulates cell growth by sponging multiple miRNAs,;  citation_journal_title=Nat. Commun;  citation_year=2016;  citation_volume=7;  citation_pages=11215" /><meta name="citation_reference" content="citation_title=MicroRNAs flex their muscles,;  citation_journal_title=Trends Genet;  citation_year=2008;  citation_volume=24;  citation_pages=159-66; " /><meta name="citation_reference" content="citation_title=MicroRNAs 1, 133, and 206: critical factors of skeletal and cardiac muscle development, function, and disease,;  citation_journal_title=Int. J. Biochem. Cell Biol;  citation_year=2010;  citation_volume=42;  citation_pages=1252-5; " /><meta name="citation_reference" content="citation_title=Characterization of RNase R-digested cellular RNA source that consists of lariat and circular RNAs from pre-mRNA splicing;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=e63" /><meta name="citation_reference" content="citation_title=A novel regulatory network among LncRpa, CircRar1, MiR-671 and apoptotic genes promotes lead-induced neuronal cell apoptosis,;  citation_journal_title=Arch. Toxicol;  citation_year=2017;  citation_volume=91;  citation_pages=1671-84; " /><meta name="citation_reference" content="citation_title=Circular RNA circ-ABCB10 promotes breast cancer proliferation and progression through sponging miR-1271;  citation_journal_title=Am J Cancer Res;  citation_year=2017;  citation_volume=7;  citation_pages=1566-76; " /><meta name="citation_reference" content="citation_title=Circular RNA hsa-circ-0016347 promotes proliferation, invasion and metastasis of osteosarcoma cells;  citation_journal_title=Oncotarget;  citation_year=2017;  citation_volume=8;  citation_pages=25571-81; " /><meta name="citation_reference" content="citation_title=Selective repression of the oncogene cyclin D1 by the tumor suppressor miR-206 in cancers;  citation_journal_title=Oncogenesis;  citation_year=2014;  citation_volume=3;  citation_pages=e113" /><meta name="citation_reference" content="citation_title=miR-206 inhibits gastric cancer proliferation in part by repressing cyclinD2,;  citation_journal_title=Cancer Lett;  citation_year=2013;  citation_volume=332;  citation_pages=94-101; " /><meta name="citation_reference" content="citation_title=TGF-beta regulates miR-206 and miR-29 to control myogenic differentiation through regulation of HDAC4,;  citation_journal_title=J. Biol. Chem;  citation_year=2011;  citation_volume=286;  citation_pages=13805-14; " /><meta name="citation_reference" content="citation_title=Neural circular RNAs are derived from synaptic genes and regulated by development and plasticity,;  citation_journal_title=Nat. Neurosci;  citation_year=2015;  citation_volume=18;  citation_pages=603-10; " /><meta name="citation_reference" content="citation_title=Screening and validation of lncRNAs and circRNAs as miRNA sponges;  citation_journal_title=Brief. Bioinform;  citation_year=2017;  citation_volume=18;  citation_pages=780-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/71/24151585/dsx039.pdf" /><meta name="description" content="Abstract.  The growth and development of skeletal muscle is regulated by proteins as well as non-coding RNAs. Circular RNAs (circRNAs) are universally expressed" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/71/4265514" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/71/4265514" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/71/4265514");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx039';
    </script>
<title>
	Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4265514" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZWwATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT3V5YW5nLCBIb25namlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIGFuZCBLZXkgTGFiIG9mIENoaWNrZW4gR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVuLCBYaWFvbGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIGFuZCBLZXkgTGFiIG9mIENoaWNrZW4gR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBaaGlqdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcgYW5kIFJlcHJvZHVjdGlvbiwgQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSwgU291dGggQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEd1YW5nemhvdSA1MTA2NDIsIEd1YW5nZG9uZywgUGVvcGxl4oCZcyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikd1YW5nZG9uZyBQcm92aW5jaWFsIEtleSBMYWIgb2YgQWdyby1BbmltYWwgR2Vub21pY3MgYW5kIE1vbGVjdWxhciBCcmVlZGluZywgYW5kIEtleSBMYWIgb2YgQ2hpY2tlbiBHZW5ldGljcywgQnJlZWRpbmcgYW5kIFJlcHJvZHVjdGlvbiwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIEd1YW5nemhvdSA1MTA2NDIsIEd1YW5nZG9uZywgUGVvcGxl4oCZcyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1LCBKaWFvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIGFuZCBLZXkgTGFiIG9mIENoaWNrZW4gR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaWEsIFhpbnpoZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIGFuZCBLZXkgTGFiIG9mIENoaWNrZW4gR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgWmhlbmh1aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZyBhbmQgUmVwcm9kdWN0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlLCBTb3V0aCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgR3Vhbmd6aG91IDUxMDY0MiwgR3Vhbmdkb25nLCBQZW9wbGXigJlzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3Vhbmdkb25nIFByb3ZpbmNpYWwgS2V5IExhYiBvZiBBZ3JvLUFuaW1hbCBHZW5vbWljcyBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBhbmQgS2V5IExhYiBvZiBDaGlja2VuIEdlbmV0aWNzLCBCcmVlZGluZyBhbmQgUmVwcm9kdWN0aW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgR3Vhbmd6aG91IDUxMDY0MiwgR3Vhbmdkb25nLCBQZW9wbGXigJlzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTHVvLCBXZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcgYW5kIFJlcHJvZHVjdGlvbiwgQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSwgU291dGggQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEd1YW5nemhvdSA1MTA2NDIsIEd1YW5nZG9uZywgUGVvcGxl4oCZcyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikd1YW5nZG9uZyBQcm92aW5jaWFsIEtleSBMYWIgb2YgQWdyby1BbmltYWwgR2Vub21pY3MgYW5kIE1vbGVjdWxhciBCcmVlZGluZywgYW5kIEtleSBMYWIgb2YgQ2hpY2tlbiBHZW5ldGljcywgQnJlZWRpbmcgYW5kIFJlcHJvZHVjdGlvbiwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIEd1YW5nemhvdSA1MTA2NDIsIEd1YW5nZG9uZywgUGVvcGxl4oCZcyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFiZGFsbGEsIEJhaGFyZWxkaW4gQWxpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIGFuZCBLZXkgTGFiIG9mIENoaWNrZW4gR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKZWJlc3NhLCBFbmRhc2hhdyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZyBhbmQgUmVwcm9kdWN0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlLCBTb3V0aCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgR3Vhbmd6aG91IDUxMDY0MiwgR3Vhbmdkb25nLCBQZW9wbGXigJlzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3Vhbmdkb25nIFByb3ZpbmNpYWwgS2V5IExhYiBvZiBBZ3JvLUFuaW1hbCBHZW5vbWljcyBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBhbmQgS2V5IExhYiBvZiBDaGlja2VuIEdlbmV0aWNzLCBCcmVlZGluZyBhbmQgUmVwcm9kdWN0aW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgR3Vhbmd6aG91IDUxMDY0MiwgR3Vhbmdkb25nLCBQZW9wbGXigJlzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmllLCBRaW5naHVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIGFuZCBLZXkgTGFiIG9mIENoaWNrZW4gR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgWGlxdWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIGFuZCBLZXkgTGFiIG9mIENoaWNrZW4gR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBHdWFuZ3pob3UgNTEwNjQyLCBHdWFuZ2RvbmcsIFBlb3BsZeKAmXMgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNpcmN1bGFyIFJOQXMgYXJlIGFidW5kYW50IGFuZCBkeW5hbWljYWxseSBleHByZXNzZWQgZHVyaW5nIGVtYnJ5b25pYyBtdXNjbGUgZGV2ZWxvcG1lbnQgaW4gY2hpY2tlbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iODYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhbmRzY2FwZSBvZiB0cmFuc2NyaXB0aW9uIGluIGh1bWFuIGNlbGxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4OTsgIGNpdGF0aW9uX3BhZ2VzPTEwMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb25nIG5vbmNvZGluZyBSTkFzOiBjZWxsdWxhciBhZGRyZXNzIGNvZGVzIGluIGRldmVsb3BtZW50IGFuZCBkaXNlYXNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTI7ICBjaXRhdGlvbl9wYWdlcz0xMjk4LTMwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkFzOiB0YXJnZXQgcmVjb2duaXRpb24gYW5kIHJlZ3VsYXRvcnkgZnVuY3Rpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzY7ICBjaXRhdGlvbl9wYWdlcz0yMTUtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBhbmQgZnVuY3Rpb25zIG9mIGxvbmcgbm9uY29kaW5nIFJOQXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEzNjsgIGNpdGF0aW9uX3BhZ2VzPTYyOS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kdWxhciByZWd1bGF0b3J5IHByaW5jaXBsZXMgb2YgbGFyZ2Ugbm9uLWNvZGluZyBSTkFzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4MjsgIGNpdGF0aW9uX3BhZ2VzPTMzOS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Vmlyb2lkcyBhcmUgc2luZ2xlLXN0cmFuZGVkIGNvdmFsZW50bHkgY2xvc2VkIGNpcmN1bGFyIFJOQSBtb2xlY3VsZXMgZXhpc3RpbmcgYXMgaGlnaGx5IGJhc2UtcGFpcmVkIHJvZC1saWtlIHN0cnVjdHVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5NzY7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz0zODUyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIHRyYW5zY3JpcHRzIG9mIHRoZSB0ZXN0aXMtZGV0ZXJtaW5pbmcgZ2VuZSBTcnkgaW4gYWR1bHQgbW91c2UgdGVzdGlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMTktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIFJOQXMgZnJvbSB0cmFuc2NyaXB0cyBvZiB0aGUgcmF0IGN5dG9jaHJvbWUgUDQ1MOKAiTJDMjQgZ2VuZTogY29ycmVsYXRpb24gd2l0aCBleG9uIHNraXBwaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYyBOYXRsIEFjYWQuIFNjaS4gVSBTIEE7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz02NTM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeG9uIHNraXBwaW5nIGFuZCBjaXJjdWxhciBSTkEgZm9ybWF0aW9uIGluIHRyYW5zY3JpcHRzIG9mIHRoZSBodW1hbiBjeXRvY2hyb21lIFAtNDUw4oCJMkMxOCBnZW5lIGluIGVwaWRlcm1pcyBhbmQgb2YgdGhlIHJhdCBhbmRyb2dlbiBiaW5kaW5nIHByb3RlaW4gZ2VuZSBpbiB0ZXN0aXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGwuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yOTg1LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNpcmN1bGFyaXplZCBzb2RpdW0tY2FsY2l1bSBleGNoYW5nZXIgZXhvbiAyIHRyYW5zY3JpcHQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3NDsgIGNpdGF0aW9uX3BhZ2VzPTgxNTMtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pcy1zcGxpY2luZyB5aWVsZHMgY2lyY3VsYXIgUk5BIG1vbGVjdWxlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZBU0VCIEo7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE1NS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbC10eXBlIHNwZWNpZmljIGZlYXR1cmVzIG9mIGNpcmN1bGFyIFJOQSBleHByZXNzaW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDM3NzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhciBSTkFzIGFyZSBhIGxhcmdlIGNsYXNzIG9mIGFuaW1hbCBSTkFzIHdpdGggcmVndWxhdG9yeSBwb3RlbmN5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5NTsgIGNpdGF0aW9uX3BhZ2VzPTMzMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIFJOQSBjaXJjbGVzIGZ1bmN0aW9uIGFzIGVmZmljaWVudCBtaWNyb1JOQSBzcG9uZ2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5NTsgIGNpdGF0aW9uX3BhZ2VzPTM4NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhciBSTkFzIGluIHRoZSBtYW1tYWxpYW4gYnJhaW4gYXJlIGhpZ2hseSBhYnVuZGFudCwgY29uc2VydmVkLCBhbmQgZHluYW1pY2FsbHkgZXhwcmVzc2VkLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9ODcwLTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBpbnRyb24gc2VxdWVuY2VzIHJldmVhbHMgaGFsbG1hcmtzIG9mIGNpcmN1bGFyIFJOQSBiaW9nZW5lc2lzIGluIGFuaW1hbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE3MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBkcm9zb3BoaWxhIGNpcmN1bGFyIFJOQXMgcmV2ZWFscyB0aGVpciBzdHJ1Y3R1cmFsIGFuZCBzZXF1ZW5jZSBwcm9wZXJ0aWVzIGFuZCBhZ2UtZGVwZW5kZW50IG5ldXJhbCBhY2N1bXVsYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTk2Ni04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BIGlzIGV4cHJlc3NlZCBhY3Jvc3MgdGhlIGV1a2FyeW90aWMgdHJlZSBvZiBsaWZlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWU5MDg1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdpZGVzcHJlYWQgbm9uY29kaW5nIGNpcmN1bGFyIFJOQXMgaW4gcGxhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDg7ICBjaXRhdGlvbl9wYWdlcz04OC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwYW5kZWQgaWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgbWFtbWFsaWFuIGNpcmN1bGFyIFJOQXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTQwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIFJOQXMgYXJlIHRoZSBwcmVkb21pbmFudCB0cmFuc2NyaXB0IGlzb2Zvcm0gZnJvbSBodW5kcmVkcyBvZiBodW1hbiBnZW5lcyBpbiBkaXZlcnNlIGNlbGwgdHlwZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTMwNzMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BcyBhcmUgYWJ1bmRhbnQsIGNvbnNlcnZlZCwgYW5kIGFzc29jaWF0ZWQgd2l0aCBBTFUgcmVwZWF0cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE0MS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgaW50cm9uaWMgbG9uZyBub25jb2RpbmcgUk5Bcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTc5Mi04MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4b24taW50cm9uIGNpcmN1bGFyIFJOQXMgcmVndWxhdGUgdHJhbnNjcmlwdGlvbiBpbiB0aGUgbnVjbGV1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI1Ni02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Y2lyY1JOQSBiaW9nZW5lc2lzIGNvbXBldGVzIHdpdGggcHJlLW1STkEgc3BsaWNpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz01NS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXhpdHkgb2YgdGhlIHRyYW5zbGF0aW9uIGFiaWxpdHkgb2YgY2lyY1JOQXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE4NTk7ICBjaXRhdGlvbl9wYWdlcz0xMjQ1LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1jaXJjUk5BRGI6IEEgY29tcHJlaGVuc2l2ZSBkYXRhYmFzZSBmb3IgaHVtYW4gY2lyY3VsYXIgUk5BcyB3aXRoIHByb3RlaW4tY29kaW5nIGFubm90YXRpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTM0OTg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW5zaXZlIHRyYW5zbGF0aW9uIG9mIGNpcmN1bGFyIFJOQXMgZHJpdmVuIGJ5IE42LW1ldGh5bGFkZW5vc2luZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NjI2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjLVpORjYwOSBJcyBhIENpcmN1bGFyIFJOQSB0aGF0IENhbiBCZSBUcmFuc2xhdGVkIGFuZCBGdW5jdGlvbnMgaW4gTXlvZ2VuZXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTIyLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduYWxpbmcgcGF0aHdheXMgaW4gc2tlbGV0YWwgbXVzY2xlIHJlbW9kZWxpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzU7ICBjaXRhdGlvbl9wYWdlcz0xOS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWN0aXZpdHktZGVwZW5kZW50IHNpZ25hbGluZyBwYXRod2F5cyBjb250cm9sbGluZyBtdXNjbGUgZGl2ZXJzaXR5IGFuZCBwbGFzdGljaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHlzaW9sb2d5IChCZXRoZXNkYSk7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yNjktNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvUk5BcyBpbnZvbHZlZCBpbiBza2VsZXRhbCBtdXNjbGUgZGlmZmVyZW50aWF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTA3LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25jb2RpbmcgUk5BcywgRW1lcmdpbmcgUmVndWxhdG9ycyBvZiBTa2VsZXRhbCBNdXNjbGUgRGV2ZWxvcG1lbnQgYW5kIERpc2Vhc2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbWVkLiBSZXMuIEludDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDE1OyAgY2l0YXRpb25fcGFnZXM9Njc2NTc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BczogTm92ZWwgUmVndWxhdG9ycyBvZiBOZXVyb25hbCBEZXZlbG9wbWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBNb2wuIE5ldXJvc2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz03NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2Vhc2Ugb25zZXQgYW5kIGFnaW5nIGluIHRoZSB3b3JsZCBvZiBjaXJjdWxhciBSTkFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBUcmFuc2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MzI3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlbWVyZ2luZyBsYW5kc2NhcGUgb2YgY2lyY3VsYXIgUk5BIGluIGxpZmUgcHJvY2Vzc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkEgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3BhZ2VzPTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BcyBpbiBtb25rZXkgbXVzY2xlOiBhZ2UtZGVwZW5kZW50IGNoYW5nZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BZ2luZyAoQWxiYW55IE5ZKTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9OTAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBhbmQgdW5pdmVyc2FsIG1ldGhvZCBmb3IgbW9sZWN1bGFyIHNleGluZyBvZiBub24tcmF0aXRlIGJpcmRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQXZpYW4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTExNi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTEwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdCBhc3NlbWJseSBhbmQgcXVhbnRpZmljYXRpb24gYnkgUk5BLVNlcSByZXZlYWxzIHVuYW5ub3RhdGVkIHRyYW5zY3JpcHRzIGFuZCBpc29mb3JtIHN3aXRjaGluZyBkdXJpbmcgY2VsbCBkaWZmZXJlbnRpYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01MTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0lSSTogYW4gZWZmaWNpZW50IGFuZCB1bmJpYXNlZCBhbGdvcml0aG0gZm9yIGRlIG5vdm8gY2lyY3VsYXIgUk5BIGlkZW50aWZpY2F0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz00IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BhdGlvLXRlbXBvcmFsIHJlZ3VsYXRpb24gb2YgY2lyY3VsYXIgUk5BIGV4cHJlc3Npb24gZHVyaW5nIHBvcmNpbmUgZW1icnlvbmljIGJyYWluIGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIFJOQSBvZiBjYXR0bGUgY2FzZWluIGdlbmVzIGFyZSBoaWdobHkgZXhwcmVzc2VkIGluIGJvdmluZSBtYW1tYXJ5IGdsYW5kLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRGFpcnkgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9NDc1MC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybiBvZiBQYXg3IGV4cHJlc3Npb24gZHVyaW5nIG15b2dlbmVzaXMgaW4gdGhlIHBvc3RoYXRjaCBjaGlja2VuIGVzdGFibGlzaGVzIGEgbW9kZWwgZm9yIHNhdGVsbGl0ZSBjZWxsIGRpZmZlcmVudGlhdGlvbiBhbmQgcmVuZXdhbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RGV2LiBEeW47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjMxOyAgY2l0YXRpb25fcGFnZXM9NDg5LTUwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5oYW5jZW1lbnQgb2YgbWVhdCBwcm9kdWN0aW9uIGJ5IGVudmlyb25tZW50YWwgbWFuaXB1bGF0aW9ucyBpbiBlbWJyeW8gYW5kIHlvdW5nIGJyb2lsZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9V29ybGQuIFBvdWx0cnkgU2NpLiBKOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9NDg1LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGF0aXN0aWNhbGx5IGJhc2VkIHNwbGljaW5nIGRldGVjdGlvbiByZXZlYWxzIG5ldXJhbCBlbnJpY2htZW50IGFuZCB0aXNzdWUtc3BlY2lmaWMgaW5kdWN0aW9uIG9mIGNpcmN1bGFyIFJOQSBkdXJpbmcgaHVtYW4gZmV0YWwgZGV2ZWxvcG1lbnQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIFJOQSBJVENIIGhhcyBpbmhpYml0b3J5IGVmZmVjdCBvbiBFU0NDIGJ5IHN1cHByZXNzaW5nIHRoZSBXbnQvYmV0YS1jYXRlbmluIHBhdGh3YXksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1PbmNvdGFyZ2V0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz02MDAxLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhciBSTkEgUmVsYXRlZCB0byB0aGUgQ2hvbmRyb2N5dGUgRUNNIFJlZ3VsYXRlcyBNTVAxMyBFeHByZXNzaW9uIGJ5IEZ1bmN0aW9uaW5nIGFzIGEgTWlSLTEzNiDigJhTcG9uZ2XigJkgaW4gSHVtYW4gQ2FydGlsYWdlIERlZ3JhZGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIyNTcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjaXJjdWxhciBSTkEgcHJvdGVjdHMgdGhlIGhlYXJ0IGZyb20gcGF0aG9sb2dpY2FsIGh5cGVydHJvcGh5IGFuZCBoZWFydCBmYWlsdXJlIGJ5IHRhcmdldGluZyBtaVItMjIzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVyLiBIZWFydCBKOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MjYwMi0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm94bzMgYWN0aXZpdHkgcHJvbW90ZWQgYnkgbm9uLWNvZGluZyBlZmZlY3RzIG9mIGNpcmN1bGFyIFJOQSBhbmQgRm94bzMgcHNldWRvZ2VuZSBpbiB0aGUgaW5oaWJpdGlvbiBvZiB0dW1vciBncm93dGggYW5kIGFuZ2lvZ2VuZXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU9uY29nZW5lOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MzkxOS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BIHByb2ZpbGluZyByZXZlYWxzIGFuIGFidW5kYW50IGNpcmNISVBLMyB0aGF0IHJlZ3VsYXRlcyBjZWxsIGdyb3d0aCBieSBzcG9uZ2luZyBtdWx0aXBsZSBtaVJOQXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTEyMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb1JOQXMgZmxleCB0aGVpciBtdXNjbGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb1JOQXMgMSwgMTMzLCBhbmQgMjA2OiBjcml0aWNhbCBmYWN0b3JzIG9mIHNrZWxldGFsIGFuZCBjYXJkaWFjIG11c2NsZSBkZXZlbG9wbWVudCwgZnVuY3Rpb24sIGFuZCBkaXNlYXNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBCaW9jaGVtLiBDZWxsIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMjUyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgUk5hc2UgUi1kaWdlc3RlZCBjZWxsdWxhciBSTkEgc291cmNlIHRoYXQgY29uc2lzdHMgb2YgbGFyaWF0IGFuZCBjaXJjdWxhciBSTkFzIGZyb20gcHJlLW1STkEgc3BsaWNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9ZTYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCByZWd1bGF0b3J5IG5ldHdvcmsgYW1vbmcgTG5jUnBhLCBDaXJjUmFyMSwgTWlSLTY3MSBhbmQgYXBvcHRvdGljIGdlbmVzIHByb21vdGVzIGxlYWQtaW5kdWNlZCBuZXVyb25hbCBjZWxsIGFwb3B0b3Npcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFyY2guIFRveGljb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xNjcxLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhciBSTkEgY2lyYy1BQkNCMTAgcHJvbW90ZXMgYnJlYXN0IGNhbmNlciBwcm9saWZlcmF0aW9uIGFuZCBwcm9ncmVzc2lvbiB0aHJvdWdoIHNwb25naW5nIG1pUi0xMjcxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbSBKIENhbmNlciBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE1NjYtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIFJOQSBoc2EtY2lyYy0wMDE2MzQ3IHByb21vdGVzIHByb2xpZmVyYXRpb24sIGludmFzaW9uIGFuZCBtZXRhc3Rhc2lzIG9mIG9zdGVvc2FyY29tYSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9T25jb3RhcmdldDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjU1NzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGl2ZSByZXByZXNzaW9uIG9mIHRoZSBvbmNvZ2VuZSBjeWNsaW4gRDEgYnkgdGhlIHR1bW9yIHN1cHByZXNzb3IgbWlSLTIwNiBpbiBjYW5jZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1PbmNvZ2VuZXNpczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTExMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1pUi0yMDYgaW5oaWJpdHMgZ2FzdHJpYyBjYW5jZXIgcHJvbGlmZXJhdGlvbiBpbiBwYXJ0IGJ5IHJlcHJlc3NpbmcgY3ljbGluRDIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DYW5jZXIgTGV0dDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzI7ICBjaXRhdGlvbl9wYWdlcz05NC0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRHRi1iZXRhIHJlZ3VsYXRlcyBtaVItMjA2IGFuZCBtaVItMjkgdG8gY29udHJvbCBteW9nZW5pYyBkaWZmZXJlbnRpYXRpb24gdGhyb3VnaCByZWd1bGF0aW9uIG9mIEhEQUM0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODY7ICBjaXRhdGlvbl9wYWdlcz0xMzgwNS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV1cmFsIGNpcmN1bGFyIFJOQXMgYXJlIGRlcml2ZWQgZnJvbSBzeW5hcHRpYyBnZW5lcyBhbmQgcmVndWxhdGVkIGJ5IGRldmVsb3BtZW50IGFuZCBwbGFzdGljaXR5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBOZXVyb3NjaTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTYwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NyZWVuaW5nIGFuZCB2YWxpZGF0aW9uIG9mIGxuY1JOQXMgYW5kIGNpcmNSTkFzIGFzIG1pUk5BIHNwb25nZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyaWVmLiBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz03ODAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzEvNzEvMjQxNTE1ODUvZHN4MDM5LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgZ3Jvd3RoIGFuZCBkZXZlbG9wbWVudCBvZiBza2VsZXRhbCBtdXNjbGUgaXMgcmVndWxhdGVkIGJ5IHByb3RlaW5zIGFzIHdlbGwgYXMgbm9uLWNvZGluZyBSTkFzLiBDaXJjdWxhciBSTkFzIChjaXJjUk5BcykgYXJlIHVuaXZlcnNhbGx5IGV4cHJlc3NlZCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDI2NTUxNCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTY6EP8lLv8P6SBCV9KM8eGDt1ARw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f71%2f4265514" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f71%2f4265514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112483106">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483106">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483108">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483108">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483113">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483113">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483145">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483145">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483177">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483177">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112483187">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112483187">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/61/4191345">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/87/4282306">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hongjia Ouyang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hongjia+Ouyang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ouyang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ouyang H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hongjia Ouyang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaolan Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaolan+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xiaolan Chen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhijun Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhijun+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang Z&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Zhijun Wang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiao Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiao+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jiao Yu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xinzheng Jia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xinzheng+Jia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jia X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jia X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xinzheng Jia</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhenhui Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhenhui+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Z&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Zhenhui Li</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Luo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wei+Luo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Luo W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Luo W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wei Luo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bahareldin Ali Abdalla</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bahareldin+Ali+Abdalla">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abdalla B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abdalla B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Bahareldin Ali Abdalla</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Endashaw Jebessa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Endashaw+Jebessa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jebessa E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jebessa E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Endashaw Jebessa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qinghua Nie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx039-cor1" data-open="dsx039-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 20 85285759. Fax. +86 20 85280740. Email: <a href="mailto:nqinghua@scau.edu.cn" target="_blank">nqinghua@scau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qinghua+Nie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nie Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nie Q&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Qinghua Nie</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiquan Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou 510642, Guangdong, People’s Republic of China</div><div class="aff">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, and Key Lab of Chicken Genetics, Breeding and Reproduction, Ministry of Agriculture, Guangzhou 510642, Guangdong, People’s Republic of China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiquan+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang X&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Xiquan Zhang</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx039-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 71–86, <a href='https://doi.org/10.1093/dnares/dsx039'>https://doi.org/10.1093/dnares/dsx039</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 October 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 January 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">07 September 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4265514" href="/dnaresearch/article-pdf/25/1/71/24151585/dsx039.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hongjia Ouyang, Xiaolan Chen, Zhijun Wang, Jiao Yu, Xinzheng Jia, Zhenhui Li, Wei Luo, Bahareldin Ali Abdalla, Endashaw Jebessa, Qinghua Nie, Xiquan Zhang;  Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 71–86, <a href='https://doi.org/10.1093/dnares/dsx039'>https://doi.org/10.1093/dnares/dsx039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4265514?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4265514?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4265514?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4265514?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4265514?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4265514?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4265514?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Circular%20RNAs%20are%20abundant%20and%20dynamically%20expressed%20during%20embryonic%20muscle%20development%20in%20chickens&amp;publicationDate=2017-10-03&amp;volumeNum=25&amp;issueNum=1&amp;author=Ouyang%2C%20Hongjia%3B%20Chen%2C%20Xiaolan&amp;startPage=71&amp;endPage=86&amp;contentId=10.1093%2Fdnares%2Fdsx039&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f71%2f4265514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112483106 id="112483106" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The growth and development of skeletal muscle is regulated by proteins as well as non-coding RNAs. Circular RNAs (circRNAs) are universally expressed in various tissues and cell types, and regulate gene expression in eukaryotes. To identify the circRNAs during chicken embryonic skeletal muscle development, leg muscles of female Xinghua (XH) chicken at three developmental time points 11 embryo age (E11), 16 embryo age (E16) and 1 day post hatch (P1) were performed RNA sequencing. We identified 13,377 circRNAs with 3,036 abundantly expressed and most were derived from coding exons. A total of 462 differentially expressed circRNAs were identified (fold change &gt; 2; q-value &lt; 0.05). Parental genes of differentially expressed circRNAs were related to muscle biological processes. There were 946 exonic circRNAs have been found that harbored one or more miRNA-binding site for 150 known miRNAs. We validated that circRBFOX2s promoted cell proliferation through interacted with miR-206. These data collectively indicate that circRNAs are abundant and dynamically expressed during embryonic muscle development and could play key roles through sequestering miRNAs as well as other functions.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;circular RNA&quot;">circular RNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chicken">chicken</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;skeletal muscle&quot;">skeletal muscle</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;embryonic development&quot;">embryonic development</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cell proliferation&quot;">cell proliferation</a></div>                            <h2 scrollto-destination=112483108 id="112483108" class="section-title" >1. Introduction</h2>
<p>Non-coding RNA such as microRNAs (miRNA) and long non-coding RNAs exist in many cells and regulate gene expression and possibly perform other biological functions.<a reveal-id="dsx039-B1 dsx039-B2 dsx039-B3 dsx039-B4 dsx039-B5" data-open="dsx039-B1 dsx039-B2 dsx039-B3 dsx039-B4 dsx039-B5" class="link link-ref link-reveal xref-bibr"><sup>1–5</sup></a> Circular RNAs are a novel non-coding RNA that forms a covalently closed continuous loop and have been observed for decades in plant viroids<a reveal-id="dsx039-B6" data-open="dsx039-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and in a few mammalian genes.<a reveal-id="dsx039-B7 dsx039-B8 dsx039-B9 dsx039-B10" data-open="dsx039-B7 dsx039-B8 dsx039-B9 dsx039-B10" class="link link-ref link-reveal xref-bibr"><sup>7–10</sup></a> CircRNAs were generally considered to be splicing artifacts or by-products until recent advances in RNA-sequencing (RNA-seq) technology.<a reveal-id="dsx039-B11" data-open="dsx039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> CircRNAs are served as a new class of RNAs that have been identified widespread in eukaryotic tissues and cells from humans, mice, nematodes, fruit flies and plants.<a reveal-id="dsx039-B12 dsx039-B13 dsx039-B14 dsx039-B15 dsx039-B16 dsx039-B17 dsx039-B18 dsx039-B19" data-open="dsx039-B12 dsx039-B13 dsx039-B14 dsx039-B15 dsx039-B16 dsx039-B17 dsx039-B18 dsx039-B19" class="link link-ref link-reveal xref-bibr"><sup>12–19</sup></a></p><p>CircRNAs can arise from exons, introns, untranslated regions, non-coding RNA loci as well as intergenic and antisense transcripts. They are abundantly and widely expressed in eukaryotes, and often show tissue and developmental stage-specific expression patterns.<a reveal-id="dsx039-B12" data-open="dsx039-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx039-B15" data-open="dsx039-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx039-B20" data-open="dsx039-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> In some genes, circular RNA isoforms are even more abundant than their linear counterparts.<a reveal-id="dsx039-B21" data-open="dsx039-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> CircRNAs are also expressed across eukaryotes, and in humans, hundreds of exonic circRNAs are found to be with circular orthologues in murine.<a reveal-id="dsx039-B18 dsx039-B19 dsx039-B20 dsx039-B21 dsx039-B22" data-open="dsx039-B18 dsx039-B19 dsx039-B20 dsx039-B21 dsx039-B22" class="link link-ref link-reveal xref-bibr"><sup>18–22</sup></a></p><p>The biological functions of circRNA are still largely unknown but they are implicated in the regulation of gene expression at multiple levels. Some exonic circRNAs (eciRNA) can function as miRNA sponges, which compete with mRNAs for miRNA binding and thus up-regulate the expression of miRNAs target gene. One representative example is the cerebellar degeneration-related 1 antisense transcript (<em>CDR1as</em>) that contains over 60 conserved miR-7 target sites.<a reveal-id="dsx039-B13" data-open="dsx039-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx039-B14" data-open="dsx039-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> A <em>CDR1as</em> knockdown leads to reduced expression of mRNAs containing miR-7-binding sites. Overexpression of <em>CDR1as</em> has an effect similar to a miR-7 knockdown and this impaired midbrain development in zebrafish.<a reveal-id="dsx039-B13" data-open="dsx039-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Circular RNAs can regulate transcription <em>via</em> RNA Pol II interactions and have been demonstrated for the ciRNA (intronic circular RNA) <em>ci-ankrd52</em> and <em>ci-sirt7</em>.<a reveal-id="dsx039-B23" data-open="dsx039-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> EIciRNAs (exon-intron circular RNA) regulate their parental genes through interaction with the spliceosome component U1 snRNP (small nuclear ribonucleoprotein).<a reveal-id="dsx039-B24" data-open="dsx039-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> CircRNA biogenesis may also compete with pre-mRNA splicing and thus regulate linear mRNA levels.<a reveal-id="dsx039-B25" data-open="dsx039-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> In addition, some circRNAs have been found to serve as templates for coding peptides or proteins, especially in that modified by N6-methyladenosine or with the presence of internal ribosomal entry sites (IRES).<a reveal-id="dsx039-B26 dsx039-B27 dsx039-B28 dsx039-B29" data-open="dsx039-B26 dsx039-B27 dsx039-B28 dsx039-B29" class="link link-ref link-reveal xref-bibr"><sup>26–29</sup></a></p><p>Skeletal muscle is the most important component of food animals and directly correlates with meat quantity and quality. The growth and development of skeletal muscle involves a series of very complex biological process, regulated by many signaling pathways, genes, transcription factors and ncRNAs.<a reveal-id="dsx039-B30 dsx039-B31 dsx039-B32 dsx039-B33" data-open="dsx039-B30 dsx039-B31 dsx039-B32 dsx039-B33" class="link link-ref link-reveal xref-bibr"><sup>30–33</sup></a> CircRNAs can play important roles in biologic processes as well as human disease.<a reveal-id="dsx039-B34 dsx039-B35 dsx039-B36" data-open="dsx039-B34 dsx039-B35 dsx039-B36" class="link link-ref link-reveal xref-bibr"><sup>34–36</sup></a> CircRNAs also have been identified abundantly expressed in monkey skeletal muscle and may play key roles in growth and development.<a reveal-id="dsx039-B37" data-open="dsx039-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In this study, we identified circRNAs during chicken embryonic skeletal muscle development by RNA sequencing to explore the functions of circRNAs in chicken skeletal muscle.</p>                            <h2 scrollto-destination=112483113 id="112483113" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=112483114 id="112483114" class="section-title" >2.1. Ethics standards</h3>
<p>Animal experiments were handled in compliance and all efforts were made to minimize suffering. It was approved by the Animal Care Committee of South China Agricultural University (Guangzhou, China) with approval number SCAU#0014.</p>                            <h3 scrollto-destination=112483116 id="112483116" class="section-title" >2.2. Samples for circular RNA sequencing</h3>
<p>A total of 240 XH chickens of E10 embryonic age were obtained from the Chicken Breeding Farm of South China Agricultural University (Guangzhou, China), and incubated in Automatic Incubator (Oscilla, Shandong, China) at 37.8 °C, with 60 ± 10% humidity. During E10 to 1 day post hatch (P1), leg muscles of 20 XH chickens were collected daily. The sex of chicken embryo was identified by PCR of the <em>CHD1</em> gene.<a reveal-id="dsx039-B38" data-open="dsx039-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Leg muscles of female XH chickens at E11, E16 and P1 were used for circular RNA sequencing.</p>                            <h3 scrollto-destination=112483118 id="112483118" class="section-title" >2.3. Circular RNA library construction and Illumina sequencing</h3>
<p>Total RNAs of six female chickens (E11, E16, and P1; each stage two individuals), were isolated by using Trizol reagent (Invitrogen, Carlsbad, CA, USA) and then treated with DNase (Promega, Madison, WI, USA) following the manufacturer’s instructions. The quantity and quality of RNAs were evaluated by Nanodrop2000 (Thermo, Waltham, MA, USA) and gel electrophoresis. Total RNA samples (5 µg) were treated with the Ribo-Zero-magnetic-kit (Epicenter, Madison, Wisconsin, USA) to remove rRNA, and then digested with 20 U of RNase R (RNR07250, Epicenter). RNA-Seq libraries were prepared using the Tru Seq RNA LT Sample Prep Kit v2 (Illumina, San Diego, CA, USA) following the manufacturer’s instructions. The libraries were sequenced using an Illumina Hiseq 3000 instrument with a Paired-End module (at a depth of 50 million reads) at Genergy Biotechnology Co., Ltd. (Shanghai, China).</p>                            <h3 scrollto-destination=112483120 id="112483120" class="section-title" >2.4. Annotation of chicken circRNAs</h3>
<p>All sequencing data are available in the Gene Expression Omnibus (GEO) with accession number (GSE89355). For all raw sequencing data of each sample, adapter reads and low-quality reads were removed using Trim Galore (<a class="link link-uri" href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a> (16 September 2017, date last accessed)). The filtered data were mapped to the chicken genome (Gallus_gallus-4.0/galGal4) using TopHat software.<a reveal-id="dsx039-B39" data-open="dsx039-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The mapping reads was performed transcript assembly using Cufflinks software.<a reveal-id="dsx039-B40" data-open="dsx039-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Identification of circRNAs was performed using CIRI.<a reveal-id="dsx039-B41" data-open="dsx039-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a></p>                            <h3 scrollto-destination=112483122 id="112483122" class="section-title" >2.5. Differential expression analysis</h3>
<p>Expression levels of circRNAs were quantified using the number of reads spanning back-spliced junctions (circular reads). The relative expression of circRNAs was denoted as BSRP (back-spliced reads per million mapped reads), using circular reads normalized to per million mapped reads. Differentially expressed circRNAs among three groups (E11, E16 and P1) were identified using the DESEQ software package (<a class="link link-uri" href="http://www.bioconductor.org/packages/release/bioc/html/DEGseq.html" target="_blank">http://www.bioconductor.org/packages/release/bioc/html/DEGseq.html</a> (16 September 2017, date last accessed)) with a <em>t</em> test <em>q</em>-value &lt; 0.05 and fold change &gt; 2. The top 200 expressed circRNAs were log2 transformed, gene mean centered and visualized as heatmaps using the Multi Experiment Viewer (<a class="link link-uri" href="http://www.tm4.org/mev.html" target="_blank">http://www.tm4.org/</a> (16 September 2017, date last accessed)).</p>                            <h3 scrollto-destination=112483124 id="112483124" class="section-title" >2.6. Target miRNA prediction, pathway and network analysis</h3>
<p>All exonic circRNAs were used to predict miRNAs potential binding sites using miRanda (<a class="link link-uri" href="http://www.microrna.org/microrna/home.do" target="_blank">http://www.microrna.org/microrna/home.do</a> (16 September 2017, date last accessed)) with threshold parameters as follows: single-residue-pair match scores &gt; 140, ΔG &lt; −10 kcal/mol and demand strict 5’ seed pairing. All parental genes of differentially expressed circRNAs were subjected to Gene Ontology (GO) (<a class="link link-uri" href="http://www.geneontology.org/" target="_blank">http://www.geneontology.org/</a> (16 September 2017, date last accessed)) and KEGG (<a class="link link-uri" href="http://www.genome.jp/kegg/" target="_blank">http://www.genome.jp/kegg/</a> (16 September 2017, date last accessed)) pathway enrichment analysis using DAVID 6.7 Functional Annotation Tool (<a class="link link-uri" href="http://david.abcc.ncifcrf.gov/" target="_blank">http://david.abcc.ncifcrf.gov/</a> (16 September 2017, date last accessed)). All parental genes of circRNAs were set as the background gene list. Gene network analysis was performed by Ingenuity Pathways analysis (IPA; Ingenuity Systems; <a class="link link-uri" href="http://www.ingenuity.com" target="_blank">http://www.ingenuity.com</a> (16 September 2017, date last accessed)).</p>                            <h3 scrollto-destination=112483126 id="112483126" class="section-title" >2.7. Validation of circRNAs by Sanger sequencing</h3>
<p>The circRNAs were validated using PCR with divergent and convergent primers as previously described.<a reveal-id="dsx039-B22" data-open="dsx039-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Divergent primers were designed in regions about 100 bp from a junction, and convergent primers were designed in regions of one exon (Details of primers are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). To confirm the junction sequence of circRNAs, PCR products of divergent primers were gel purified and submitted for Sanger sequencing at Sangon Biotech Co. Ltd (Shanghai, China).</p>                            <h3 scrollto-destination=112483128 id="112483128" class="section-title" >2.8. cDNA synthesis and quantitative real-time PCR (qRT-PCR)</h3>
<p>Total RNA was used for reverse transcription using RevertAid First Strand cDNA Synthesis Kit (Fermentas, Waltham, MA, USA) with either random hexamers or specific primers for miRNAs as indicated. The relative expression levels of circRNAs or miRNAs were determined by qRT-PCR using SsoFast Eva Green Supermix (Bio-Rad, Hercules, CA, USA) in a final volume of 20 µl. To check the sensitivity of circRNA to RNaseR, qRT-PCR was also performed using RNA samples with and without RNaseR treatment. Primers used for circRNAs were designed as divergent primers to detect backsplice junctions, and bulge-loop primers were synthesized by Ribobio (Guangzhou, China) for miRNAs (Details of primers are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The <em>β-actin</em> gene was used as reference genes for circRNAs, and U6 snRNA was used as reference gene for miRNAs. The qRT-PCR program was performed in a BIO-RAD CFX96 system as follows: 95 °C for 3 min; 40 cycles of 95 °C for 10 s, annealing temperature (58–62 °C) for 30 s, and 72 °C for 30 s; and 72 °C for 1 min. The relative expression level of miRNA was calculated using the comparative 2<sup>−ΔCt</sup> (ΔCt = Ct<sub>target gene</sub>– Ct<sub>reference gene</sub>). Fold change values were calculated using the comparative 2<sup>−ΔΔCt</sup>, in which ΔΔCt = ΔCt (target sample) − ΔCt (control sample). All reactions were run in triplicate and presented as means ± <span class="small-caps">s</span>.<span class="small-caps">e</span>.<span class="small-caps">m</span>. The Student’s <em>t</em>-test was used to compare expression levels among different groups.</p>                            <h3 scrollto-destination=112483130 id="112483130" class="section-title" >2.9. Vector construction and RNA oligonucleotides</h3>
<p>The perfect match sequence of gga-miR-1a-3p° or gga-miR-206 was synthesized and cloned into the psiCHECK-2 vector (Promega) using the <em>NotI</em> and <em>XhoI</em> restriction sites. The circRNA overexpression vectors were constructed using the linear sequences of <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> amplified from chicken leg muscle cDNA using PCR. They were then cloned into pCD-ciR2.1vector (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s1_dsx039.png?Expires=2147483647&amp;Signature=3lFoad0PPPRn9ld~g90ig~qJDhoC41zi5LDlFGtKGa6EmJFKECUbcvVnUa1bUnDg6FxzDGkSuZbBDLdEV6E6aD~iYR314aZ3rZWeu5ItHc-7R7gQpQ8h5sODXHMqx2iqeApcyPRqcHVB1tUdl0F0rpI0LkEonNl52hFS32T9kLToT3H13g4Eu7NIvNjq7QQwONfmIFchXFt29kAi2D9eLXOYDd-4AH6wclizxNNDT-saLpA0CDR7DXbscbkdyHmW1i0aLv0Nf6Mig4nEbUjuQht0npYXKV4JwDUN5KJ2vtSEf9KhyRTBD5idodSCJbsfQqwK-rO7JpkBNpnorA8rwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>, Geneseed Biotech, Guangzhou, China) according to the manufacturer’s protocol using the <em>KpnI</em> and <em>BamHI</em> restriction sites. The wild type and mutated 3’UTRs sequences of CCND1, CCND2, PAX7 and HDAC4 were synthesized and cloned into the pmirGLO dual-luciferase reporter vector (Promega) using the <em>NheI</em> and <em>XhoI</em> restriction sites. The exon2-3 or exon2-4 of RBFOX2 were synthesized and cloned into the pcDNA3.1+ vector (Invitrogen) using the <em>NheI</em> and <em>SalI</em> restriction sites. Small interfering RNA (siRNA) of circRNAs (sequences are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&amp;Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), miRNA mimics and inhibitors were synthesized by Ribobio.</p>                            <h3 scrollto-destination=112483132 id="112483132" class="section-title" >2.10. Cell culture and transfection</h3>
<p>Chicken primary myoblasts were isolated from the leg muscles of E11 chickens. Leg muscle (1 g) was minced into sections of approximately 1 mm with scissors and digested with 0.25% trypsin (Gibco, Grand Island, NY, USA) at 37 °C in a shaking water bath (90 oscillations/min). Digestions were terminated by adding foetal bovine serum (Gibco) after 30 min. The mixture was filtered through a nylon mesh with 70 µm pores (BD Falcon). The filtered cells were centrifuged at 350g for 5 min, and maintained in Dulbecco’s modified Eagle medium (DMEM) (Gibco), supplemented with 20% foetal bovine serum, and 0.2% penicillin/streptomycin (Invitrogen, Carlsbad, CA, USA) at 37 °C in a 5% CO<sub>2</sub>, humidified atmosphere. Serial plating was performed to enrich myoblasts and remove fibroblasts. DF-1 cells were cultured in DMEM with 10% foetal bovine serum and 0.2% penicillin/streptomycin. QM-7 cells were cultured in high-glucose M199 medium (Gibco) with 10% foetal bovine serum, 10% tryptose phosphate broth solution (Sigma, Louis, MO, USA) and 0.2% penicillin/streptomycin. Cells were transfected with 50 nM of miRNA mimics, 100 nM of miRNA inhibitors, 100 nM of siRNA or plasmid (1 µg/ml) using Lipofectamine 3000 reagent (Invitrogen) according to the manufacturer’s instructions.</p>                            <h3 scrollto-destination=112483134 id="112483134" class="section-title" >2.11. Luciferase reporter assay and Northern blot analysis</h3>
<p>DF-1 cells were seeded in 96-well plates and co-transfected with the dual-luciferase reporter (psiCHECK-2) and miRNA mimics or NC (negative control) and with circRNA overexpression vector, circRNA mutated vector or empty vector (EV) using Lipofectamine 3000 reagent. After transfection for 48 h, the luminescent signals of <em>firefly</em> and <em>Renilla</em> luciferase were detected using Dual-GLO Luciferase Assay System Kit (Promega) with a Fluorescence/Multi-Detection Microplate Reader (Biotek,Winooski, VT, USA). Northern blotting for miR-206 and U6 were performed with MiRNA Northern Blot Assay Kit (Signosis, Santa Clara, CA, USA) following the manufacturer’s instructions.</p>                            <h3 scrollto-destination=112483136 id="112483136" class="section-title" >2.12. Flow cytometry analysis of the cell cycle</h3>
<p>After transfection 48 h, myoblasts were collected and fixed in 70% ethanol overnight at −20 °C. The cells were incubated with 50 µg/ml propidium iodide (Sigma), 10 µg/ml RNaseA (Takara) and 0.2% (v/v) Triton X-100 (Sigma) for 30 min at 4 °C. Analyses were performed using a BD AccuriC6 flow cytometer (BD Biosciences, San Jose, CA, USA) and FlowJo (v7.6) software (Treestar Incorporated, Ashland, OR, USA).</p>                            <h3 scrollto-destination=112483138 id="112483138" class="section-title" >2.13. EdU assay</h3>
<p>QM-7 cells were seeded in 12-well plates, and at 80% confluence, cell were transfected with overexpression vector or siRNA of circRNAs, miRNA mimic, miRNA inhibitor or negative control. After transfection for 36 h, cell proliferation was tested using a Cell-Light EdU Apollo 567 in Vitro Flow Cytometry Kit (Ribobio). The cells were first incubated with 50 µM EDU for 2 h at 37 °C, and then fixed in 4% paraformaldehyde for 30 min and permeabilized with 0.5% Triton X-100. The proliferating cells were stained with Apollo Dye Solution and Hoechst 33342 (used as control). The EdU-stained cells were visualized by fluorescence microscopy (Nikon, Tokyo, Japan). The cell proliferation rate was calculated using images of randomly selected fields obtained from the fluorescence microscope. We performed four repeats for each group, and three images were used to calculate the cell proliferation rate in each repeat.</p>                            <h3 scrollto-destination=112483140 id="112483140" class="section-title" >2.14. Histology</h3>
<p>The leg muscle samples were ﬁxed in 4% paraformaldehyde for a minimum of 24 h and then embedded in paraffin and 10-mm thick serial sections were made. The sections were then subjected to H-E staining following standard protocols. Images were taken using a Moticam 2306 CCD imaging system (Motic Instruments, CA, USA).</p>                            <h3 scrollto-destination=112483142 id="112483142" class="section-title" >2.15. RNA immunoprecipitation and biotin-coupled miRNA pull down</h3>
<p>RIP experiments were performed by using the Magna RIP RNA-Binding Protein Immunoprecipitation Kit (Millipore, Bedford, MA) following the manufacturer’s instructions. Immunoprecipitation of AGO2 was performed in QM7 cells co-transfected with circRBFOX2s expression vector and miR-1a-3p°, miR-206 or miR-203 control. The mRNA levels of circRBFOX2s were quantified by qRT–PCR and were normalized to <em>GAPDH</em> gene. The relative immunoprecipitate/input ratios are plotted.</p><p>The 3’ end biotinylated miR-1a-3p°, miR-206 or miR-203 mimic (RiboBio) were transfected into QM7 cells along with circRBFOX2s expression vector. The biotin-coupled RNA complex was pull-downed by Dynabeads MyOne Streptavidin C1 kit (Invitrogen) following the manufacturer’s instructions. RNA bound to the beads (pull-down RNA) was isolated using Trizol LS reagent (Invitrogen). The mRNA levels of circRBFOX2s in the streptavidin captured fractions were quantified by qRT–PCR and the enrichment ratios of the miR-1a-3p° or miR-206 to the miR-203 control were plotted.</p>                            <h2 scrollto-destination=112483145 id="112483145" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=112483146 id="112483146" class="section-title" >3.1. Overview of circular RNA deep sequencing data</h3>
<p>In recent years, a number of reports demonstrated that circRNAs were widespread and abundant existed in various eukaryotes, especially in mammals. However, little information is known regarding circRNAs in domestic animals with the exception of brain cortex of pig<a reveal-id="dsx039-B42" data-open="dsx039-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> and mammary gland tissues of cattle.<a reveal-id="dsx039-B43" data-open="dsx039-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Here, we performed the genome-wide identification and potential function analysis of circRNAs in chickens. The total number of muscle fibers is the determining factor in muscle mass and this is regulated during embryogenesis or early post hatch in chickens.<a reveal-id="dsx039-B44" data-open="dsx039-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx039-B45" data-open="dsx039-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> To identify the circRNA during the chicken embryonic skeletal muscle development, leg muscle tissues of two female XH chicken for each at days E11, E16 and P1 were used for RNA sequencing after rRNA-depletion and RNase R treatment (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s2_dsx039.png?Expires=2147483647&amp;Signature=W9NmDcc-nlC3V6hCDnqd5Y-qXt7ypIxXMRbzaNKKs4tGbW3SwrHG-FBO1Rhq~0Kgu3JxTyRPiIevlXt7-c0qSaWzZW0KjrL58Nwjv6YBtbbCWYVgVtnZNccP0mJc9Rubn1pzrrNktLUYtfADMx4ppvoJRsHjqkxOoFsPPKa6wjz9EVuY3gZvlfzSs07AIXhb~WWZZUqFvIGqg4IkUC-w1msboUODtS9adSVHr~u3J4L6kuLxRjvLtskghzTulMR6c99aT1HrJsyxIAvKGMyTxMPU7H-k~Nalvb9iX3~c64-MFWkUEC4MRtGwwD-3PAtaLb5ZHM~qRGjzO5kE7RdV3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The sex of XH chicken embryos was identified (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s3_dsx039.png?Expires=2147483647&amp;Signature=LyyaEcoDrCTep5xHiOQDhfzAYQq62IYDuPWQpg6wZIE3APDy~OnO3HlfAaiHmMeqb4ybK5W5n2OJmuDvPXih7CrdbGTkDy2w313Fk77KhB9hnw~xGX79vcqjdgXYF78ymwBxcVVxmWKK406Y5~pwtKOcFmDgoBr4BF4G2AQ-Ky5YVzjpwhV-DBP6rE8doG-vBPm2AgDhZ7DaTb2WArrDNymjgUTekxNa9ZupPb1YxJbX58x9WzuQkubfQQO7RQbzwbe35JzXsXXia59t1uzxmpABjlVUkiGhdx3nHfAJVupbzVe-kfrqd35D3uk7tSaoA0pSnJnlzo6MgDiRd24gQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>) and leg muscle tissues of female chickens were performed using hematoxylin-eosin (H-E) staining to visual the muscle fibers during 10 embryo age to post hatch (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s3_dsx039.png?Expires=2147483647&amp;Signature=LyyaEcoDrCTep5xHiOQDhfzAYQq62IYDuPWQpg6wZIE3APDy~OnO3HlfAaiHmMeqb4ybK5W5n2OJmuDvPXih7CrdbGTkDy2w313Fk77KhB9hnw~xGX79vcqjdgXYF78ymwBxcVVxmWKK406Y5~pwtKOcFmDgoBr4BF4G2AQ-Ky5YVzjpwhV-DBP6rE8doG-vBPm2AgDhZ7DaTb2WArrDNymjgUTekxNa9ZupPb1YxJbX58x9WzuQkubfQQO7RQbzwbe35JzXsXXia59t1uzxmpABjlVUkiGhdx3nHfAJVupbzVe-kfrqd35D3uk7tSaoA0pSnJnlzo6MgDiRd24gQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span>).</p><p>For RNA-seq, a total of about 750 million reads were generated from all six samples, and each sample yielded more than 100 million reads. Raw data were processed to remove adapter and low quality sequences, and then mapped to the chicken reference genome (Gallus_gallus-4.0/galGal4) (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The mapping reads were uniformly distributed in each chromosome, except the abundance in the chromosome MT is much larger than that of the other chromosomes (<a reveal-id="dsx039-F1" data-open="dsx039-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>).From the six muscle tissues, a total of 13,377 circRNAs were detected; 7,914 in E11, 10,949 in E16, and 9,309 in P1, and 5,176 were detected in all three stages (<a reveal-id="dsx039-F1" data-open="dsx039-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) using the CIRI algorithm method.<a reveal-id="dsx039-B41" data-open="dsx039-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> These results agree with the scope and abundance of circRNAs that have been identified in mammals.<a reveal-id="dsx039-B18" data-open="dsx039-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx039-B21" data-open="dsx039-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> According to their genomic locus, the chicken circRNAs were grouped as exon (including 5’UTR, 5’UTR-CDS, CDS-3’UTR, 3’UTR and 5’UTR -CDS-3’UTR), intron, exon-intron, intergenic regions or others. Although circRNAs can arise from almost any location in a genome, they are most originate from coding exons.<a reveal-id="dsx039-B13" data-open="dsx039-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx039-B17 dsx039-B18 dsx039-B19" data-open="dsx039-B17 dsx039-B18 dsx039-B19" class="link link-ref link-reveal xref-bibr"><sup>17–19</sup></a> We also found 11,064 of 13,377 circRNAs were derived from known chicken genes, especially in coding DNA sequences (CDS) (<em>n</em> = 8,110). The other circRNAs were derived from introns, exon-introns, intergenic regions or other regions (<a reveal-id="dsx039-F1" data-open="dsx039-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). All these circRNAs were originated from 3,589 chicken genes. Among them, 1,555 parental genes generated only one circRNA, and the other genes could yield two or more circular isoforms, even smaller fractions yielded more than ten distinct circular isoforms (<a reveal-id="dsx039-F1" data-open="dsx039-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>), similar to the results in pigs.<a reveal-id="dsx039-B42" data-open="dsx039-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                        <a id="112483149" scrollto-destination="112483149"></a>
<div id="dsx039-F1" data-id="dsx039-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f1.png?Expires=2147483647&Signature=rXmDdEOMRNsrNGB2cZlH-oSpoAcCWsxTVG2xSkTTL8fms5hgnNTES5JcoqcPDDVEyXRNrT3q0Md-lPiXoH7kzlr5WzMtFBTxbC3pDWIpFdzQYLva9H2k2To~-i4fycsKaZE8cLxRbVx1KMqw06PJ1efF~~E2rMijoFoUa-mHeL32kJq0jWb9M0tzZIW9FQfHBwMf7po2WJFlhRUA0VRO4ZW9c-Um4kkRIm0Tyrk0GHljEQytgt48sBmTObk454CvYd2LQGSGwIsMxDsTlw8ouc8Sm7rKY4eR4f1IEP4NUFOSqADyXgTKL0MnA7bwYyKe5zsl-5nECYpHjv~gt-VYLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of chicken embryonic muscle circRNA. (A) Distribution of sequencing reads on chicken chromosome. (B) Venn diagrams of circRNAs in embryonic muscle with three different development stages. E11, 11 embryo age; E16, 16 embryo age; P1, 1 day post hatch. (C) Genomic origin of chicken circRNA. (D) Distribution of circRNAs among genes; 11,064 circRNAs in 3,589 genes. (E, F) The number of circRNAs expressed at various cutoff expression levels. (G) Venn diagrams of highly expressed circRNAs (n = 3,036, BSRP &gt; 1). (H) Numbers and percentages of different types of highly expressed circRNAs. The Y axis (left) represents the numbers of highly expressed circRNAs, and the Y axis (right) represents the percentage is the number of abundant circRNAs of type divided by the total number of circRNAs specific type. eciRNA, exonic circRNA; EIciRNA, exon-intron circRNA; ciRNA, intronic circRNA." path-from-xml="dsx039f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483149/dsx039f1.tif" path-from-xml="dsx039f1.tif" target="_blank">View large</a><a section="112483149" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f1.png?Expires=2147483647&Signature=E4bycbznXsDtHEz2qM29QF78eQOV8OCvyDAmky~FBWJuavgb3M-uW3trDfmCUOo50BrVxDvoGIKvhb32zsTtXuQwaavKu7z78yRfHk4Qp2ZHSw7xbSaIWT1xKlXmSsDQ17vYiOeSH4nbEYcMBJijQcLEVA~KZKLgzxHMfG0JtKMzIFb4DDUI0lezkf~hR11hMzGkbZaqCrse-F-79E9pb5imjkrjHkEKZy31Sw8nWN1QxXZn-HCbJghATGjuiH5KtioaFb3yvAwFzKRxaKL5CTTsOzI52MXRVopl49oD6LeLXF09UQ-8WHg7MyoAa~nClwVLcfA9sm7DAwWo7JAskQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483149&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of chicken embryonic muscle circRNA. (A) Distribution of sequencing reads on chicken chromosome. (B) Venn diagrams of circRNAs in embryonic muscle with three different development stages. E11, 11 embryo age; E16, 16 embryo age; P1, 1 day post hatch. (C) Genomic origin of chicken circRNA. (D) Distribution of circRNAs among genes; 11,064 circRNAs in 3,589 genes. (E, F) The number of circRNAs expressed at various cutoff expression levels. (G) Venn diagrams of highly expressed circRNAs (<em>n</em> = 3,036, BSRP &gt; 1). (H) Numbers and percentages of different types of highly expressed circRNAs. The <em>Y</em> axis (left) represents the numbers of highly expressed circRNAs, and the <em>Y</em> axis (right) represents the percentage is the number of abundant circRNAs of type divided by the total number of circRNAs specific type. eciRNA, exonic circRNA; EIciRNA, exon-intron circRNA; ciRNA, intronic circRNA.</p></div></div><div content-id="dsx039-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f1.png?Expires=2147483647&Signature=rXmDdEOMRNsrNGB2cZlH-oSpoAcCWsxTVG2xSkTTL8fms5hgnNTES5JcoqcPDDVEyXRNrT3q0Md-lPiXoH7kzlr5WzMtFBTxbC3pDWIpFdzQYLva9H2k2To~-i4fycsKaZE8cLxRbVx1KMqw06PJ1efF~~E2rMijoFoUa-mHeL32kJq0jWb9M0tzZIW9FQfHBwMf7po2WJFlhRUA0VRO4ZW9c-Um4kkRIm0Tyrk0GHljEQytgt48sBmTObk454CvYd2LQGSGwIsMxDsTlw8ouc8Sm7rKY4eR4f1IEP4NUFOSqADyXgTKL0MnA7bwYyKe5zsl-5nECYpHjv~gt-VYLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of chicken embryonic muscle circRNA. (A) Distribution of sequencing reads on chicken chromosome. (B) Venn diagrams of circRNAs in embryonic muscle with three different development stages. E11, 11 embryo age; E16, 16 embryo age; P1, 1 day post hatch. (C) Genomic origin of chicken circRNA. (D) Distribution of circRNAs among genes; 11,064 circRNAs in 3,589 genes. (E, F) The number of circRNAs expressed at various cutoff expression levels. (G) Venn diagrams of highly expressed circRNAs (n = 3,036, BSRP &gt; 1). (H) Numbers and percentages of different types of highly expressed circRNAs. The Y axis (left) represents the numbers of highly expressed circRNAs, and the Y axis (right) represents the percentage is the number of abundant circRNAs of type divided by the total number of circRNAs specific type. eciRNA, exonic circRNA; EIciRNA, exon-intron circRNA; ciRNA, intronic circRNA." path-from-xml="dsx039f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483149/dsx039f1.tif" path-from-xml="dsx039f1.tif" target="_blank">View large</a><a section="112483149" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f1.png?Expires=2147483647&Signature=E4bycbznXsDtHEz2qM29QF78eQOV8OCvyDAmky~FBWJuavgb3M-uW3trDfmCUOo50BrVxDvoGIKvhb32zsTtXuQwaavKu7z78yRfHk4Qp2ZHSw7xbSaIWT1xKlXmSsDQ17vYiOeSH4nbEYcMBJijQcLEVA~KZKLgzxHMfG0JtKMzIFb4DDUI0lezkf~hR11hMzGkbZaqCrse-F-79E9pb5imjkrjHkEKZy31Sw8nWN1QxXZn-HCbJghATGjuiH5KtioaFb3yvAwFzKRxaKL5CTTsOzI52MXRVopl49oD6LeLXF09UQ-8WHg7MyoAa~nClwVLcfA9sm7DAwWo7JAskQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483149&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of chicken embryonic muscle circRNA. (A) Distribution of sequencing reads on chicken chromosome. (B) Venn diagrams of circRNAs in embryonic muscle with three different development stages. E11, 11 embryo age; E16, 16 embryo age; P1, 1 day post hatch. (C) Genomic origin of chicken circRNA. (D) Distribution of circRNAs among genes; 11,064 circRNAs in 3,589 genes. (E, F) The number of circRNAs expressed at various cutoff expression levels. (G) Venn diagrams of highly expressed circRNAs (<em>n</em> = 3,036, BSRP &gt; 1). (H) Numbers and percentages of different types of highly expressed circRNAs. The <em>Y</em> axis (left) represents the numbers of highly expressed circRNAs, and the <em>Y</em> axis (right) represents the percentage is the number of abundant circRNAs of type divided by the total number of circRNAs specific type. eciRNA, exonic circRNA; EIciRNA, exon-intron circRNA; ciRNA, intronic circRNA.</p></div></div><p>The abundance of expressed circRNAs in each sample were normalized as number of back-spliced reads per million mapped reads (BSRP), and a minimum BSRP cutoff was set to 0.1 for all samples. The number of circRNAs expressed at various cutoff expression levels in each sample are shown in <a reveal-id="dsx039-F1" data-open="dsx039-F1" class="link link-reveal link-table xref-fig">Figure 1E and F</a>. A total of 3,036 circRNAs were defined as highly expressed circRNA (BSRP &gt; 1), and 2,579 of them were detected in the three developmental stages we examined (<a reveal-id="dsx039-F1" data-open="dsx039-F1" class="link link-reveal link-table xref-fig">Fig. 1G</a>). The 10 most abundant circRNAs expressed in embryonic muscle are listed in <a reveal-id="dsx039-T1" data-open="dsx039-T1" class="link link-reveal link-table xref-fig">Table 1</a>. The majority of highly expressed circRNAs was derived from exons and as a group was more abundant than other circRNA types (<a reveal-id="dsx039-F1" data-open="dsx039-F1" class="link link-reveal link-table xref-fig">Fig. 1H</a>). Our RNA sequencing data indicating circRNAs are also abundant in chicken embryonic muscle. <div content-id="dsx039-T1" class="table-modal"><div class="table-wrap"><div id="dsx039-T1" data-id="dsx039-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>The top 10 circRNAs expressed in chicken embryonic muscle</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>circRNA_ID </th><th>circRNA_type </th><th>Parental gene </th><th>Junction </th><th>Linear distance of junction (nt) </th><th>Length (nt) </th><th>BSRP </th></tr></thead><tbody><tr><td>1:106020993|106024456 </td><td>eciRNA </td><td>MORC3 </td><td>Exon 5-7 </td><td>3,464 </td><td>428 </td><td>328.06 </td></tr><tr><td>5:24409706|24411456 </td><td>eciRNA </td><td>MAPKBP1 </td><td>Exon 25-27 </td><td>1,751 </td><td>396 </td><td>279.82 </td></tr><tr><td>2:14642365|14648660 </td><td>EIciRNA </td><td>SVIL </td><td>Exon6-intron11(P) </td><td>6,296 </td><td>6296 </td><td>271.62 </td></tr><tr><td>2:14633486|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-14 </td><td>15,174 </td><td>1594 </td><td>268.80 </td></tr><tr><td>2:14645249|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-10 </td><td>3,412 </td><td>504 </td><td>179.63 </td></tr><tr><td>7:14755660|14757838 </td><td>eciRNA </td><td>MYPN </td><td>Exon 100-104 </td><td>2,179 </td><td>501 </td><td>154.00 </td></tr><tr><td>1:51782860|51783084 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 2 </td><td>2,25 </td><td>225 </td><td>134.95 </td></tr><tr><td>4:1285647|1289769 </td><td>eciRNA </td><td>FNDC3A </td><td>Exon 5-9 </td><td>4,123 </td><td>695 </td><td>126.41 </td></tr><tr><td>26:5104947|5107131 </td><td>eciRNA </td><td>TAF8 </td><td>Exon 2-5 </td><td>2,185 </td><td>444 </td><td>120.12 </td></tr><tr><td>1:51819796|51833118 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 9-11 </td><td>13,323 </td><td>262 </td><td>106.49 </td></tr></tbody></table></div><div content-id="dsx039-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>circRNA_ID </th><th>circRNA_type </th><th>Parental gene </th><th>Junction </th><th>Linear distance of junction (nt) </th><th>Length (nt) </th><th>BSRP </th></tr></thead><tbody><tr><td>1:106020993|106024456 </td><td>eciRNA </td><td>MORC3 </td><td>Exon 5-7 </td><td>3,464 </td><td>428 </td><td>328.06 </td></tr><tr><td>5:24409706|24411456 </td><td>eciRNA </td><td>MAPKBP1 </td><td>Exon 25-27 </td><td>1,751 </td><td>396 </td><td>279.82 </td></tr><tr><td>2:14642365|14648660 </td><td>EIciRNA </td><td>SVIL </td><td>Exon6-intron11(P) </td><td>6,296 </td><td>6296 </td><td>271.62 </td></tr><tr><td>2:14633486|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-14 </td><td>15,174 </td><td>1594 </td><td>268.80 </td></tr><tr><td>2:14645249|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-10 </td><td>3,412 </td><td>504 </td><td>179.63 </td></tr><tr><td>7:14755660|14757838 </td><td>eciRNA </td><td>MYPN </td><td>Exon 100-104 </td><td>2,179 </td><td>501 </td><td>154.00 </td></tr><tr><td>1:51782860|51783084 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 2 </td><td>2,25 </td><td>225 </td><td>134.95 </td></tr><tr><td>4:1285647|1289769 </td><td>eciRNA </td><td>FNDC3A </td><td>Exon 5-9 </td><td>4,123 </td><td>695 </td><td>126.41 </td></tr><tr><td>26:5104947|5107131 </td><td>eciRNA </td><td>TAF8 </td><td>Exon 2-5 </td><td>2,185 </td><td>444 </td><td>120.12 </td></tr><tr><td>1:51819796|51833118 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 9-11 </td><td>13,323 </td><td>262 </td><td>106.49 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The expressed abundances of circRNAs were normalized as number of back-spliced reads per million mapped reads (BSRP). This column represented the sum of the BSRP values for all samples. eciRNA, exonic circRNA; EIciRNA, exon-intron circRNA.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx039-T1" data-id="dsx039-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>The top 10 circRNAs expressed in chicken embryonic muscle</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>circRNA_ID </th><th>circRNA_type </th><th>Parental gene </th><th>Junction </th><th>Linear distance of junction (nt) </th><th>Length (nt) </th><th>BSRP </th></tr></thead><tbody><tr><td>1:106020993|106024456 </td><td>eciRNA </td><td>MORC3 </td><td>Exon 5-7 </td><td>3,464 </td><td>428 </td><td>328.06 </td></tr><tr><td>5:24409706|24411456 </td><td>eciRNA </td><td>MAPKBP1 </td><td>Exon 25-27 </td><td>1,751 </td><td>396 </td><td>279.82 </td></tr><tr><td>2:14642365|14648660 </td><td>EIciRNA </td><td>SVIL </td><td>Exon6-intron11(P) </td><td>6,296 </td><td>6296 </td><td>271.62 </td></tr><tr><td>2:14633486|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-14 </td><td>15,174 </td><td>1594 </td><td>268.80 </td></tr><tr><td>2:14645249|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-10 </td><td>3,412 </td><td>504 </td><td>179.63 </td></tr><tr><td>7:14755660|14757838 </td><td>eciRNA </td><td>MYPN </td><td>Exon 100-104 </td><td>2,179 </td><td>501 </td><td>154.00 </td></tr><tr><td>1:51782860|51783084 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 2 </td><td>2,25 </td><td>225 </td><td>134.95 </td></tr><tr><td>4:1285647|1289769 </td><td>eciRNA </td><td>FNDC3A </td><td>Exon 5-9 </td><td>4,123 </td><td>695 </td><td>126.41 </td></tr><tr><td>26:5104947|5107131 </td><td>eciRNA </td><td>TAF8 </td><td>Exon 2-5 </td><td>2,185 </td><td>444 </td><td>120.12 </td></tr><tr><td>1:51819796|51833118 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 9-11 </td><td>13,323 </td><td>262 </td><td>106.49 </td></tr></tbody></table></div><div content-id="dsx039-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>circRNA_ID </th><th>circRNA_type </th><th>Parental gene </th><th>Junction </th><th>Linear distance of junction (nt) </th><th>Length (nt) </th><th>BSRP </th></tr></thead><tbody><tr><td>1:106020993|106024456 </td><td>eciRNA </td><td>MORC3 </td><td>Exon 5-7 </td><td>3,464 </td><td>428 </td><td>328.06 </td></tr><tr><td>5:24409706|24411456 </td><td>eciRNA </td><td>MAPKBP1 </td><td>Exon 25-27 </td><td>1,751 </td><td>396 </td><td>279.82 </td></tr><tr><td>2:14642365|14648660 </td><td>EIciRNA </td><td>SVIL </td><td>Exon6-intron11(P) </td><td>6,296 </td><td>6296 </td><td>271.62 </td></tr><tr><td>2:14633486|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-14 </td><td>15,174 </td><td>1594 </td><td>268.80 </td></tr><tr><td>2:14645249|14648660 </td><td>eciRNA </td><td>SVIL </td><td>Exon 6-10 </td><td>3,412 </td><td>504 </td><td>179.63 </td></tr><tr><td>7:14755660|14757838 </td><td>eciRNA </td><td>MYPN </td><td>Exon 100-104 </td><td>2,179 </td><td>501 </td><td>154.00 </td></tr><tr><td>1:51782860|51783084 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 2 </td><td>2,25 </td><td>225 </td><td>134.95 </td></tr><tr><td>4:1285647|1289769 </td><td>eciRNA </td><td>FNDC3A </td><td>Exon 5-9 </td><td>4,123 </td><td>695 </td><td>126.41 </td></tr><tr><td>26:5104947|5107131 </td><td>eciRNA </td><td>TAF8 </td><td>Exon 2-5 </td><td>2,185 </td><td>444 </td><td>120.12 </td></tr><tr><td>1:51819796|51833118 </td><td>eciRNA </td><td>RBFOX2 </td><td>Exon 9-11 </td><td>13,323 </td><td>262 </td><td>106.49 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The expressed abundances of circRNAs were normalized as number of back-spliced reads per million mapped reads (BSRP). This column represented the sum of the BSRP values for all samples. eciRNA, exonic circRNA; EIciRNA, exon-intron circRNA.</p></div></div></div></div></p>                            <h3 scrollto-destination=112483151 id="112483151" class="section-title" >3.2. Identification of differentially expressed circRNAs</h3>
<p>CircRNAs are usually expressed in tissue and developmental stage specific manners. Especially during embryonic development, circRNA expression levels exhibit dynamic global changes.<a reveal-id="dsx039-B17" data-open="dsx039-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx039-B42" data-open="dsx039-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsx039-B46" data-open="dsx039-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> To address the potential functions of circRNA, differentially-expressed circRNAs (DEcircRNAs) were identified by DEGseq analysis (fold change &gt; 2, q-value &lt; 0.05) and clustered based on their expression profiles (<a reveal-id="dsx039-F2" data-open="dsx039-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s4_dsx039.png?Expires=2147483647&amp;Signature=MW3gW9cpPMOGbjU6WcD1pybiz119iRF~AMZA8sAK87YgxKpCZlQvNOCZRgQMsOniIgGIhdcvO0mhR7SocBMHwfaI~CtMr-i6XuOKQagoEyjnHl0sSqSDGa9WjX99F~C~EFt7fgTJrSXcMyXr8h20eoN8z-BPjm6nUNY2Ld~8~aeMEzKVHPaqesGwfNAtNBB18A6K28zThNFE1BsSbwoRo2jMMWC3rt6~rXkJksG33AF0F-htvnKnchVtnEFYF4c8KRHgG-QIj96Z7dZZopYXQyI3kvYzDiQhct~4mljPaRTNZDBqppCtniZq3mPGjbvXbKmlRgawLmy0tK7qTGCo8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). A total of 462 DEcircRNAs were detected in the three developmental groups, 236, 285 and 89 circRNAs in E11_VS_E16, E11_VS_P1, and E16_VS_P1 comparison groups, respectively. These circRNAs were more abundant in E16 and P1 than in E11, whereas a small fraction of group E11 was quantitatively higher (<a reveal-id="dsx039-F2" data-open="dsx039-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>, <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&amp;Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S4–S6</a></span>). We found that 223, 526 and 255 circRNAs were specifically expressed in E11, E16 and P1, respectively (<a reveal-id="dsx039-F2" data-open="dsx039-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). A larger number of circRNAs are differentially expressed in chicken leg muscle development. We performed RNA sequencing used two individuals in each stage but not mix RNA pool. The variation of circRNAs abundance was also existed between the two duplicates used for sequencing. The numbers of circRNAs were larger in E16 or P1 than that in E11 of leg muscle, consistent with that circRNAs were found to accumulate during aging in Drosophila heads.<a reveal-id="dsx039-B17" data-open="dsx039-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p>                        <a id="112483153" scrollto-destination="112483153"></a>
<div id="dsx039-F2" data-id="dsx039-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f2.png?Expires=2147483647&Signature=4mf6SAhcbgG~owMXt3m9yZojsvAop2gZxMybtZ8~NR9zbEjokX~XiPDZQVA9arRes8D1CfjpJXgtq7DCowJNrXKi5w6Zb96h2z~AThc0ImV1iL73tFmM18t2-m2BRgc8DICGXPzNWed4vRgT2eAeCVUgOQu0tk9UhstT-s1wVl8ywK-YvyXFxFz3lP55SkiKWpmUJ6JGSmdsVNzipDs3P05q6mDRnyUMwjSQsu44-zImnOYxdVqTtvKZYihIoTCxqsNR8Q966vPy9i7JoM42jrv24XfCtgZ5nqUBWazf7smFg8af4AoKY5~fiCpotxO0-5fTTB4-DS0yBs~0dvuEcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differentially expressed circRNAs in three different development stages of embryonic muscle. (A) Heatmap of top 100 differentially expressed circRNAs in E11, E16 and P1 of leg muscle. (B) Venn diagrams of differentially expressed circRNAs (n = 462; q value &lt; 0.05, fold change &gt; 2). (C) The circRNA specific expression in three different development stages of embryonic muscle. The top 10 GO enrichment term for the parental genes of differentially expressed circRNAs in E11_VS_E16 (D), E11_VS_P1 (E) and E16_VS_P1 (F). The Y axis represents GO terms and the X axis represents –Log p-value." path-from-xml="dsx039f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483153/dsx039f2.tif" path-from-xml="dsx039f2.tif" target="_blank">View large</a><a section="112483153" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f2.png?Expires=2147483647&Signature=0QidqrqCxiJf-PE1WoAVC6000IXqOL0Xc99ZZ95eSaB7zP6seuapCNO8mdcK8XwlSqLqbq4nxzZJbBXan1PL2hrttaUa1LiHtGJ5PYmL9xPXuF9WKSBxCKEsL8LKpKMrT7vsos0ao6R~s-mTkRaoybK4NkFGhoICJUDIAu7lUTc3-M~d064bCjSZMXTARjI1xuDLZPWcotctVdvlhY8338nK78dn0V4jmIt6ay5yZ~rhnLQjUByBhkl5~K0qQQYlywTc~REfLeGYMR7~S3PgP7gkXwGi~Xp2lQkd9v2IRlvcMuJcMtyxgKR1ZT-oHIrhlZTxSXk41KbbD8Hu14j~1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483153&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differentially expressed circRNAs in three different development stages of embryonic muscle. (A) Heatmap of top 100 differentially expressed circRNAs in E11, E16 and P1 of leg muscle. (B) Venn diagrams of differentially expressed circRNAs (<em>n</em> = 462; q value &lt; 0.05, fold change &gt; 2). (C) The circRNA specific expression in three different development stages of embryonic muscle. The top 10 GO enrichment term for the parental genes of differentially expressed circRNAs in E11_VS_E16 (D), E11_VS_P1 (E) and E16_VS_P1 (F). The <em>Y</em> axis represents GO terms and the <em>X</em> axis represents –Log <em>p</em>-value.</p></div></div><div content-id="dsx039-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f2.png?Expires=2147483647&Signature=4mf6SAhcbgG~owMXt3m9yZojsvAop2gZxMybtZ8~NR9zbEjokX~XiPDZQVA9arRes8D1CfjpJXgtq7DCowJNrXKi5w6Zb96h2z~AThc0ImV1iL73tFmM18t2-m2BRgc8DICGXPzNWed4vRgT2eAeCVUgOQu0tk9UhstT-s1wVl8ywK-YvyXFxFz3lP55SkiKWpmUJ6JGSmdsVNzipDs3P05q6mDRnyUMwjSQsu44-zImnOYxdVqTtvKZYihIoTCxqsNR8Q966vPy9i7JoM42jrv24XfCtgZ5nqUBWazf7smFg8af4AoKY5~fiCpotxO0-5fTTB4-DS0yBs~0dvuEcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Differentially expressed circRNAs in three different development stages of embryonic muscle. (A) Heatmap of top 100 differentially expressed circRNAs in E11, E16 and P1 of leg muscle. (B) Venn diagrams of differentially expressed circRNAs (n = 462; q value &lt; 0.05, fold change &gt; 2). (C) The circRNA specific expression in three different development stages of embryonic muscle. The top 10 GO enrichment term for the parental genes of differentially expressed circRNAs in E11_VS_E16 (D), E11_VS_P1 (E) and E16_VS_P1 (F). The Y axis represents GO terms and the X axis represents –Log p-value." path-from-xml="dsx039f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483153/dsx039f2.tif" path-from-xml="dsx039f2.tif" target="_blank">View large</a><a section="112483153" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f2.png?Expires=2147483647&Signature=0QidqrqCxiJf-PE1WoAVC6000IXqOL0Xc99ZZ95eSaB7zP6seuapCNO8mdcK8XwlSqLqbq4nxzZJbBXan1PL2hrttaUa1LiHtGJ5PYmL9xPXuF9WKSBxCKEsL8LKpKMrT7vsos0ao6R~s-mTkRaoybK4NkFGhoICJUDIAu7lUTc3-M~d064bCjSZMXTARjI1xuDLZPWcotctVdvlhY8338nK78dn0V4jmIt6ay5yZ~rhnLQjUByBhkl5~K0qQQYlywTc~REfLeGYMR7~S3PgP7gkXwGi~Xp2lQkd9v2IRlvcMuJcMtyxgKR1ZT-oHIrhlZTxSXk41KbbD8Hu14j~1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483153&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differentially expressed circRNAs in three different development stages of embryonic muscle. (A) Heatmap of top 100 differentially expressed circRNAs in E11, E16 and P1 of leg muscle. (B) Venn diagrams of differentially expressed circRNAs (<em>n</em> = 462; q value &lt; 0.05, fold change &gt; 2). (C) The circRNA specific expression in three different development stages of embryonic muscle. The top 10 GO enrichment term for the parental genes of differentially expressed circRNAs in E11_VS_E16 (D), E11_VS_P1 (E) and E16_VS_P1 (F). The <em>Y</em> axis represents GO terms and the <em>X</em> axis represents –Log <em>p</em>-value.</p></div></div><p>The biogenesis of circRNA can competes with pre-mRNA splicing, and intron or exon-intron circRNAs can regulate the transcription of their parental gene.<a reveal-id="dsx039-B23 dsx039-B24 dsx039-B25" data-open="dsx039-B23 dsx039-B24 dsx039-B25" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a> In this study, we performed GO Enrichment Analysis for the parental genes of differentially-expressed circRNAs (fold change &gt; 2, <em>P</em>-value &lt; 0.05) in E11_VS_E16, E11_VS_P1 and E16_VS_P1 comparison groups (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&amp;Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7–S9</a></span>). The parental gene functions were involved in muscle-related biological processes, including muscle structure development, muscle tissue development, muscle system processes and muscle cell differentiation (<a reveal-id="dsx039-F2" data-open="dsx039-F2" class="link link-reveal link-table xref-fig">Fig. 2D–F</a>). Using this data, we organized a functional network using Ingenuity pathway analysis (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&amp;Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). The most prevalent diseases and function interaction network was related to skeletal and muscular disorders and this group involved 29 parental genes (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s5_dsx039.png?Expires=2147483647&amp;Signature=RjDwDx-FzHMB7ci~RaFT3gZfGuUGIWAF0rDdbS4vPZmgIj1XBcB5ajsYmpBLd5uhVKO92X4n-KmZSJudbsiK04BfRoYIhb71kgbM7t5H1nJv0g2P5HgNfbhHjtBWwk2TmlgI9lQRqDexLJc4GgdmSoyiDcT3yo1Hey6LnL8W-yfoLYegVoAbQ7iu8LfcQQaIggLVBNIw~PO3HtxRgmJf0JATK~VYvE9qXFQTKBZcxm~WMfH3hlABids9abdnsnLbLYetreb-w342ySVosEUu9VvXeei7EJZdxjTUKNv4ubaENCxz2iBgvSp65GYUSaIwkbSbpG6eUkJLe6l5fnnW3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p>                            <h3 scrollto-destination=112483155 id="112483155" class="section-title" >3.3. Putative functions of chicken circRNAs as miRNA sponges</h3>
<p>The spatial-temporal grouping of circRNAs suggested that they are functional molecules. Since the discovery of <em>CDR1as</em> can serve as miR-7 sponges,<a reveal-id="dsx039-B13" data-open="dsx039-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx039-B14" data-open="dsx039-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> there are growing evidences indicate that circRNAs can function as miRNA sponges or competing endogenous RNAs.<a reveal-id="dsx039-B17" data-open="dsx039-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx039-B47 dsx039-B48 dsx039-B49 dsx039-B50 dsx039-B51" data-open="dsx039-B47 dsx039-B48 dsx039-B49 dsx039-B50 dsx039-B51" class="link link-ref link-reveal xref-bibr"><sup>47–51</sup></a> However, unlike <em>CDR1as</em> contain multiple binding sites to sponge miR-7, most circRNAs bind more than one species of miRNA as was shown for <em>circFOXO3</em><a reveal-id="dsx039-B52" data-open="dsx039-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and <em>circHIPK3</em>.<a reveal-id="dsx039-B53" data-open="dsx039-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> We identified the miRNA binding capabilities of our differentially expressed exonic circRNAs, and found that 946 of 1,401 exonic circRNAs harbored one or more miRNA binding site and involved 150 known miRNAs (The top 10 of these circRNAs were listed in <a reveal-id="dsx039-T2" data-open="dsx039-T2" class="link link-reveal link-table xref-fig">Table 2</a>). We used the 20 most abundant differentially expressed exonic circRNAs from each comparison group and matched them with their potential target miRNAs. These data were then used to construct an interactive network map (<a reveal-id="dsx039-F3" data-open="dsx039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). We identified several miRNAs with key roles in skeletal muscle development and differentiation such as miR-1, miR-133, miR-206 and miR-203 in this network. The high abundance of these DEcircRNAs could be served as candidate circRNAs to further study in chicken skeletal muscle. <div content-id="dsx039-T2" class="table-modal"><div class="table-wrap"><div id="dsx039-T2" data-id="dsx039-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The top 10 differentially expressed eciRNAs with miRNA binding sites</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>circRNA_ID </th><th>E11_base Mean </th><th>E16_base Mean </th><th>P1_base Mean </th><th>Parental gene </th><th>Junction </th><th>Length (nt) </th><th>Target miRNAs </th></tr></thead><tbody><tr><td>2:14633486|14648660 </td><td>313.35 </td><td>5345.71 </td><td>7148.93 </td><td>SVIL </td><td>Exon6-14 </td><td>1594 </td><td>miR-31-5p (2), miR-103-3p (2), miR-135a-5p (2), miR-203a, miR-1720-3p, miR-1737 </td></tr><tr><td>7:14755660|14757838 </td><td>14.79 </td><td>2547.25 </td><td>5155.27 </td><td>TNN </td><td>Exon100-104 </td><td>501 </td><td>miR-133a-5p, miR-1a-2-5p, miR-22-5, miR-451 </td></tr><tr><td>26:5104947|5107131 </td><td>561.47 </td><td>2995.87 </td><td>2333.66 </td><td>TAF8 </td><td>Exon2-5 </td><td>444 </td><td>miR-34a-5p </td></tr><tr><td>1:51819796|51833118 </td><td>626.95 </td><td>2756.90 </td><td>1891.00 </td><td>RBFOX2 </td><td>Exon9-11 </td><td>262 </td><td>miR-18b-3p </td></tr><tr><td>2:14633486|14644033 </td><td>354.24 </td><td>1777.35 </td><td>1973.62 </td><td>SVIL </td><td>Exon11-14 </td><td>1090 </td><td>miR-31-5p, miR-103-3p (2), miR-135a-5p (2), miR-1720-3p </td></tr><tr><td>5:13396775|13397627 </td><td>0.00 </td><td>3666.12 </td><td>0.00 </td><td>TNNT3 </td><td>Exon15-17 </td><td>297 </td><td>miR-6631-5p </td></tr><tr><td>1:84430731|84434121 </td><td>1992.44 </td><td>171.80 </td><td>690.86 </td><td>ABI3BP </td><td>Exon31-34 </td><td>273 </td><td>miR-499-3p (2) </td></tr><tr><td>6:30457170|30471496 </td><td>2015.87 </td><td>101.28 </td><td>47.48 </td><td>FGFR2 </td><td>Exon3-6 </td><td>636 </td><td>miR-15a, miR-200a-3p </td></tr><tr><td>1:51782860|51793309 </td><td>165.19 </td><td>775.20 </td><td>1025.64 </td><td>RBFOX2 </td><td>Exon2-3 </td><td>372 </td><td>miR-1a-3p, miR-206 </td></tr><tr><td>7:35565240|35577784 </td><td>246.91 </td><td>912.57 </td><td>451.10 </td><td>GPD2 </td><td>Exon1-5 </td><td>667 </td><td>miR-103-3p (2), miR-133a-3p, miR-133b, miR-133c-3p, miR-30a-5p </td></tr></tbody></table></div><div content-id="dsx039-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>circRNA_ID </th><th>E11_base Mean </th><th>E16_base Mean </th><th>P1_base Mean </th><th>Parental gene </th><th>Junction </th><th>Length (nt) </th><th>Target miRNAs </th></tr></thead><tbody><tr><td>2:14633486|14648660 </td><td>313.35 </td><td>5345.71 </td><td>7148.93 </td><td>SVIL </td><td>Exon6-14 </td><td>1594 </td><td>miR-31-5p (2), miR-103-3p (2), miR-135a-5p (2), miR-203a, miR-1720-3p, miR-1737 </td></tr><tr><td>7:14755660|14757838 </td><td>14.79 </td><td>2547.25 </td><td>5155.27 </td><td>TNN </td><td>Exon100-104 </td><td>501 </td><td>miR-133a-5p, miR-1a-2-5p, miR-22-5, miR-451 </td></tr><tr><td>26:5104947|5107131 </td><td>561.47 </td><td>2995.87 </td><td>2333.66 </td><td>TAF8 </td><td>Exon2-5 </td><td>444 </td><td>miR-34a-5p </td></tr><tr><td>1:51819796|51833118 </td><td>626.95 </td><td>2756.90 </td><td>1891.00 </td><td>RBFOX2 </td><td>Exon9-11 </td><td>262 </td><td>miR-18b-3p </td></tr><tr><td>2:14633486|14644033 </td><td>354.24 </td><td>1777.35 </td><td>1973.62 </td><td>SVIL </td><td>Exon11-14 </td><td>1090 </td><td>miR-31-5p, miR-103-3p (2), miR-135a-5p (2), miR-1720-3p </td></tr><tr><td>5:13396775|13397627 </td><td>0.00 </td><td>3666.12 </td><td>0.00 </td><td>TNNT3 </td><td>Exon15-17 </td><td>297 </td><td>miR-6631-5p </td></tr><tr><td>1:84430731|84434121 </td><td>1992.44 </td><td>171.80 </td><td>690.86 </td><td>ABI3BP </td><td>Exon31-34 </td><td>273 </td><td>miR-499-3p (2) </td></tr><tr><td>6:30457170|30471496 </td><td>2015.87 </td><td>101.28 </td><td>47.48 </td><td>FGFR2 </td><td>Exon3-6 </td><td>636 </td><td>miR-15a, miR-200a-3p </td></tr><tr><td>1:51782860|51793309 </td><td>165.19 </td><td>775.20 </td><td>1025.64 </td><td>RBFOX2 </td><td>Exon2-3 </td><td>372 </td><td>miR-1a-3p, miR-206 </td></tr><tr><td>7:35565240|35577784 </td><td>246.91 </td><td>912.57 </td><td>451.10 </td><td>GPD2 </td><td>Exon1-5 </td><td>667 </td><td>miR-103-3p (2), miR-133a-3p, miR-133b, miR-133c-3p, miR-30a-5p </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Read counts of each sample were normalized by DESEQ software. Base Mean is the average normalized value of two samples at same time point. The numbers in brackets are target sites for each miRNA.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx039-T2" data-id="dsx039-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The top 10 differentially expressed eciRNAs with miRNA binding sites</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>circRNA_ID </th><th>E11_base Mean </th><th>E16_base Mean </th><th>P1_base Mean </th><th>Parental gene </th><th>Junction </th><th>Length (nt) </th><th>Target miRNAs </th></tr></thead><tbody><tr><td>2:14633486|14648660 </td><td>313.35 </td><td>5345.71 </td><td>7148.93 </td><td>SVIL </td><td>Exon6-14 </td><td>1594 </td><td>miR-31-5p (2), miR-103-3p (2), miR-135a-5p (2), miR-203a, miR-1720-3p, miR-1737 </td></tr><tr><td>7:14755660|14757838 </td><td>14.79 </td><td>2547.25 </td><td>5155.27 </td><td>TNN </td><td>Exon100-104 </td><td>501 </td><td>miR-133a-5p, miR-1a-2-5p, miR-22-5, miR-451 </td></tr><tr><td>26:5104947|5107131 </td><td>561.47 </td><td>2995.87 </td><td>2333.66 </td><td>TAF8 </td><td>Exon2-5 </td><td>444 </td><td>miR-34a-5p </td></tr><tr><td>1:51819796|51833118 </td><td>626.95 </td><td>2756.90 </td><td>1891.00 </td><td>RBFOX2 </td><td>Exon9-11 </td><td>262 </td><td>miR-18b-3p </td></tr><tr><td>2:14633486|14644033 </td><td>354.24 </td><td>1777.35 </td><td>1973.62 </td><td>SVIL </td><td>Exon11-14 </td><td>1090 </td><td>miR-31-5p, miR-103-3p (2), miR-135a-5p (2), miR-1720-3p </td></tr><tr><td>5:13396775|13397627 </td><td>0.00 </td><td>3666.12 </td><td>0.00 </td><td>TNNT3 </td><td>Exon15-17 </td><td>297 </td><td>miR-6631-5p </td></tr><tr><td>1:84430731|84434121 </td><td>1992.44 </td><td>171.80 </td><td>690.86 </td><td>ABI3BP </td><td>Exon31-34 </td><td>273 </td><td>miR-499-3p (2) </td></tr><tr><td>6:30457170|30471496 </td><td>2015.87 </td><td>101.28 </td><td>47.48 </td><td>FGFR2 </td><td>Exon3-6 </td><td>636 </td><td>miR-15a, miR-200a-3p </td></tr><tr><td>1:51782860|51793309 </td><td>165.19 </td><td>775.20 </td><td>1025.64 </td><td>RBFOX2 </td><td>Exon2-3 </td><td>372 </td><td>miR-1a-3p, miR-206 </td></tr><tr><td>7:35565240|35577784 </td><td>246.91 </td><td>912.57 </td><td>451.10 </td><td>GPD2 </td><td>Exon1-5 </td><td>667 </td><td>miR-103-3p (2), miR-133a-3p, miR-133b, miR-133c-3p, miR-30a-5p </td></tr></tbody></table></div><div content-id="dsx039-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>circRNA_ID </th><th>E11_base Mean </th><th>E16_base Mean </th><th>P1_base Mean </th><th>Parental gene </th><th>Junction </th><th>Length (nt) </th><th>Target miRNAs </th></tr></thead><tbody><tr><td>2:14633486|14648660 </td><td>313.35 </td><td>5345.71 </td><td>7148.93 </td><td>SVIL </td><td>Exon6-14 </td><td>1594 </td><td>miR-31-5p (2), miR-103-3p (2), miR-135a-5p (2), miR-203a, miR-1720-3p, miR-1737 </td></tr><tr><td>7:14755660|14757838 </td><td>14.79 </td><td>2547.25 </td><td>5155.27 </td><td>TNN </td><td>Exon100-104 </td><td>501 </td><td>miR-133a-5p, miR-1a-2-5p, miR-22-5, miR-451 </td></tr><tr><td>26:5104947|5107131 </td><td>561.47 </td><td>2995.87 </td><td>2333.66 </td><td>TAF8 </td><td>Exon2-5 </td><td>444 </td><td>miR-34a-5p </td></tr><tr><td>1:51819796|51833118 </td><td>626.95 </td><td>2756.90 </td><td>1891.00 </td><td>RBFOX2 </td><td>Exon9-11 </td><td>262 </td><td>miR-18b-3p </td></tr><tr><td>2:14633486|14644033 </td><td>354.24 </td><td>1777.35 </td><td>1973.62 </td><td>SVIL </td><td>Exon11-14 </td><td>1090 </td><td>miR-31-5p, miR-103-3p (2), miR-135a-5p (2), miR-1720-3p </td></tr><tr><td>5:13396775|13397627 </td><td>0.00 </td><td>3666.12 </td><td>0.00 </td><td>TNNT3 </td><td>Exon15-17 </td><td>297 </td><td>miR-6631-5p </td></tr><tr><td>1:84430731|84434121 </td><td>1992.44 </td><td>171.80 </td><td>690.86 </td><td>ABI3BP </td><td>Exon31-34 </td><td>273 </td><td>miR-499-3p (2) </td></tr><tr><td>6:30457170|30471496 </td><td>2015.87 </td><td>101.28 </td><td>47.48 </td><td>FGFR2 </td><td>Exon3-6 </td><td>636 </td><td>miR-15a, miR-200a-3p </td></tr><tr><td>1:51782860|51793309 </td><td>165.19 </td><td>775.20 </td><td>1025.64 </td><td>RBFOX2 </td><td>Exon2-3 </td><td>372 </td><td>miR-1a-3p, miR-206 </td></tr><tr><td>7:35565240|35577784 </td><td>246.91 </td><td>912.57 </td><td>451.10 </td><td>GPD2 </td><td>Exon1-5 </td><td>667 </td><td>miR-103-3p (2), miR-133a-3p, miR-133b, miR-133c-3p, miR-30a-5p </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Read counts of each sample were normalized by DESEQ software. Base Mean is the average normalized value of two samples at same time point. The numbers in brackets are target sites for each miRNA.</p></div></div></div></div></p>                        <a id="112483157" scrollto-destination="112483157"></a>
<div id="dsx039-F3" data-id="dsx039-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f3.png?Expires=2147483647&Signature=Wh2TCNa-wyd5Cv5gMIq7FFtyAf3XVQqEPOTfszWjzPEXEsqh07YmcFDJbSSdv7EwfZXjLPMm9i9ZMkYxdC96pJNR8IkHp-QhJhxYlPPvfg34TixK1qGvA5EG1eUKc1ASfGFCO0uSO1PO~aI50B2Op1WqZh2pryImpmDfByGXehSBRb3s5FU2jXJXMsCYswL9qxeSkbkArpQh9yb1XuvFKCVSJmwOrqq89n7hA1fToaefgDbD18KX7Hdus7DrrY7yR74Q683EMeSyNMmrsN75jQ8iVKqVVtsAE1St3eAFvZoxGpHSQdMO7ItN5uPzNYzSdKHN5VyD7HFmobWFaT1dww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Interaction network of eciRNA and their potential target miRNAs. The top 20 differentially expressed exonic circRNAs of each comparison group and their potential target miRNAs were used for their interaction network analysis. Rectangle nodes represent circRNAs and circle nodes represent miRNAs." path-from-xml="dsx039f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483157/dsx039f3.tif" path-from-xml="dsx039f3.tif" target="_blank">View large</a><a section="112483157" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f3.png?Expires=2147483647&Signature=R1SfC9vJjZooK4GqPOujWVz9V1fUItxICz1DUuSGZITpHWARw5XiQHKK-b3gQ~XdHKfu9C1joBGbePnxC3ZnPu37eLLaZsugEPASqOkjsPrdFn6swKu1dpkkpBFHivWzpEWnsZUxC7Ax8lkiAQm4cBL~~dpJ3vBgmwlYT2GUC0vobjpd1THWOlmkKamoVtg85M52l-YDxCbdxRi1z5ZfPRVvGUs2hZafz4GQDtO~N2WBAXB5vAPqOZvL9q03AcoJ3VHisoAeDCRiG4GQlv40PD9LgpKOLfxIWm-ywRQJoQUoRnk53scKCFNPP5nutKbMyTc3l1UD1dbDjV9HbMTtqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483157&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interaction network of eciRNA and their potential target miRNAs. The top 20 differentially expressed exonic circRNAs of each comparison group and their potential target miRNAs were used for their interaction network analysis. Rectangle nodes represent circRNAs and circle nodes represent miRNAs.</p></div></div><div content-id="dsx039-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f3.png?Expires=2147483647&Signature=Wh2TCNa-wyd5Cv5gMIq7FFtyAf3XVQqEPOTfszWjzPEXEsqh07YmcFDJbSSdv7EwfZXjLPMm9i9ZMkYxdC96pJNR8IkHp-QhJhxYlPPvfg34TixK1qGvA5EG1eUKc1ASfGFCO0uSO1PO~aI50B2Op1WqZh2pryImpmDfByGXehSBRb3s5FU2jXJXMsCYswL9qxeSkbkArpQh9yb1XuvFKCVSJmwOrqq89n7hA1fToaefgDbD18KX7Hdus7DrrY7yR74Q683EMeSyNMmrsN75jQ8iVKqVVtsAE1St3eAFvZoxGpHSQdMO7ItN5uPzNYzSdKHN5VyD7HFmobWFaT1dww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Interaction network of eciRNA and their potential target miRNAs. The top 20 differentially expressed exonic circRNAs of each comparison group and their potential target miRNAs were used for their interaction network analysis. Rectangle nodes represent circRNAs and circle nodes represent miRNAs." path-from-xml="dsx039f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483157/dsx039f3.tif" path-from-xml="dsx039f3.tif" target="_blank">View large</a><a section="112483157" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f3.png?Expires=2147483647&Signature=R1SfC9vJjZooK4GqPOujWVz9V1fUItxICz1DUuSGZITpHWARw5XiQHKK-b3gQ~XdHKfu9C1joBGbePnxC3ZnPu37eLLaZsugEPASqOkjsPrdFn6swKu1dpkkpBFHivWzpEWnsZUxC7Ax8lkiAQm4cBL~~dpJ3vBgmwlYT2GUC0vobjpd1THWOlmkKamoVtg85M52l-YDxCbdxRi1z5ZfPRVvGUs2hZafz4GQDtO~N2WBAXB5vAPqOZvL9q03AcoJ3VHisoAeDCRiG4GQlv40PD9LgpKOLfxIWm-ywRQJoQUoRnk53scKCFNPP5nutKbMyTc3l1UD1dbDjV9HbMTtqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483157&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interaction network of eciRNA and their potential target miRNAs. The top 20 differentially expressed exonic circRNAs of each comparison group and their potential target miRNAs were used for their interaction network analysis. Rectangle nodes represent circRNAs and circle nodes represent miRNAs.</p></div></div><p>The miR-1 and miR-206 are well-studied muscle-specific miRNAs that are necessary for skeletal muscle development as well as proliferation and differentiation of muscle cells.<a reveal-id="dsx039-B52" data-open="dsx039-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsx039-B53" data-open="dsx039-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Interestingly, we found that circRNA derived from the <em>RBFOX2</em> gene harbored binding sites for gga-miR-1a-3p and gga-miR-206. We identified 11 isoforms of <em>RBFOX2</em> circRNAs (<a reveal-id="dsx039-F4" data-open="dsx039-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>) and six of them were expressed abundantly and differentially in muscle (<a reveal-id="dsx039-F4" data-open="dsx039-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). Four isoforms <em>circRBFOX2.2-3</em>, <em>2-4</em>, <em>2-6</em>, and <em>2-9</em> harbored miR-1a-3p and miR-206 binding sites (<a reveal-id="dsx039-F4" data-open="dsx039-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). The interaction of circRBFOX2s with miR-1a-3p and miR-206 was further validated in our study. </p>                        <a id="112483159" scrollto-destination="112483159"></a>
<div id="dsx039-F4" data-id="dsx039-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f4.png?Expires=2147483647&Signature=GqOQRP0dXkdyddRuEC1g15A58oXrl5pbgWO29TJQrxZELTFBQx0GoO105Zz079918NNUlAb8tluqm3L-lSuCkMIRh62UW4dYWuTLNGHu9-xsnpBvqch3sNNUtWSQEAcV65l7i5Egn2ntmFwH1S5BvKRGIkH9GPkasBpoUDTZ5rlf-rutPQuo8hgAvFvcz9CJMYi2Xmoki2nnheg0F2S~MjtXiN7vxfSljn15tQNlHQW-nJlsr4ytI-ReyUvzM0ZIFF~btat~z6GILi7GSWdqbniK2fUCM~HTzMRcU2oAdGH8ewdamFULIvbH5tMlOKUmjhydX082w5bAuXIs3tKJKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of circular RBFOX2 gene. (A) All circRNA isoforms of the RBFOX2 gene in chicken embryonic muscle. (B) The abundance of circular RBFOX2 in chicken embryonic muscle. The scale bar 0–32 represents the abundance of circRNAs (BSRP value). (C) Network of circular RBFOX2 with miRNAs. Dark gray rectangle nodes represent highly-expressed circRNAs, light gray rectangle nodes represent low expressed circRNAs and circle nodes represent miRNAs." path-from-xml="dsx039f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483159/dsx039f4.tif" path-from-xml="dsx039f4.tif" target="_blank">View large</a><a section="112483159" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f4.png?Expires=2147483647&Signature=GDAfXkkB6Q~EBh9MG4Fz8XWPOBEd40uKIttFOgCH0IWEoMKNQUINlPfnqX0QgRCbnrKj9IZexJx~wxhkZiT5tOuO9kx~YdNhO7rzTP4eeadQ5e2uSuY7a4UO2UqWVTdkhKHKFMEgywEnDauijwBPTLL-bTpnzYmZGsqBV6uq1K1F8FkMu8z51~98qRmGj8xROoXvcjomQbK0D2t3cK~iVh6RCgYXQIeqb41TkrhN5BEvk-10r5TNQJyVpASit3pZ6b~6CyzYyI~Bb-E~AY4Y8Oexa1bMonxfoiX76htWoHAybShBCcLW~HJGjKcM37hYhxO7GIdAA9tnzbZnBpccSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483159&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of circular RBFOX2 gene. (A) All circRNA isoforms of the RBFOX2 gene in chicken embryonic muscle. (B) The abundance of circular RBFOX2 in chicken embryonic muscle. The scale bar 0–32 represents the abundance of circRNAs (BSRP value). (C) Network of circular <em>RBFOX2</em> with miRNAs. Dark gray rectangle nodes represent highly-expressed circRNAs, light gray rectangle nodes represent low expressed circRNAs and circle nodes represent miRNAs.</p></div></div><div content-id="dsx039-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f4.png?Expires=2147483647&Signature=GqOQRP0dXkdyddRuEC1g15A58oXrl5pbgWO29TJQrxZELTFBQx0GoO105Zz079918NNUlAb8tluqm3L-lSuCkMIRh62UW4dYWuTLNGHu9-xsnpBvqch3sNNUtWSQEAcV65l7i5Egn2ntmFwH1S5BvKRGIkH9GPkasBpoUDTZ5rlf-rutPQuo8hgAvFvcz9CJMYi2Xmoki2nnheg0F2S~MjtXiN7vxfSljn15tQNlHQW-nJlsr4ytI-ReyUvzM0ZIFF~btat~z6GILi7GSWdqbniK2fUCM~HTzMRcU2oAdGH8ewdamFULIvbH5tMlOKUmjhydX082w5bAuXIs3tKJKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of circular RBFOX2 gene. (A) All circRNA isoforms of the RBFOX2 gene in chicken embryonic muscle. (B) The abundance of circular RBFOX2 in chicken embryonic muscle. The scale bar 0–32 represents the abundance of circRNAs (BSRP value). (C) Network of circular RBFOX2 with miRNAs. Dark gray rectangle nodes represent highly-expressed circRNAs, light gray rectangle nodes represent low expressed circRNAs and circle nodes represent miRNAs." path-from-xml="dsx039f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483159/dsx039f4.tif" path-from-xml="dsx039f4.tif" target="_blank">View large</a><a section="112483159" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f4.png?Expires=2147483647&Signature=GDAfXkkB6Q~EBh9MG4Fz8XWPOBEd40uKIttFOgCH0IWEoMKNQUINlPfnqX0QgRCbnrKj9IZexJx~wxhkZiT5tOuO9kx~YdNhO7rzTP4eeadQ5e2uSuY7a4UO2UqWVTdkhKHKFMEgywEnDauijwBPTLL-bTpnzYmZGsqBV6uq1K1F8FkMu8z51~98qRmGj8xROoXvcjomQbK0D2t3cK~iVh6RCgYXQIeqb41TkrhN5BEvk-10r5TNQJyVpASit3pZ6b~6CyzYyI~Bb-E~AY4Y8Oexa1bMonxfoiX76htWoHAybShBCcLW~HJGjKcM37hYhxO7GIdAA9tnzbZnBpccSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483159&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of circular RBFOX2 gene. (A) All circRNA isoforms of the RBFOX2 gene in chicken embryonic muscle. (B) The abundance of circular RBFOX2 in chicken embryonic muscle. The scale bar 0–32 represents the abundance of circRNAs (BSRP value). (C) Network of circular <em>RBFOX2</em> with miRNAs. Dark gray rectangle nodes represent highly-expressed circRNAs, light gray rectangle nodes represent low expressed circRNAs and circle nodes represent miRNAs.</p></div></div>                            <h3 scrollto-destination=112483160 id="112483160" class="section-title" >3.4. Experimental validation of chicken circRNAs</h3>
<p>The expression and back-splicing sites of several circRNAs were validated by divergent reverse-transcription PCR, RNaseR digestion and qRT-PCR, according to previously described methodologies.<a reveal-id="dsx039-B13" data-open="dsx039-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx039-B54" data-open="dsx039-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Divergent primers and convergent primers were designed to amplify six candidate circRNAs in cDNA and genomic DNA samples. Divergent primers from each circRNA produced a single distinct band of the expected product size only in cDNA samples suggesting the presence of back-splicing junctions but not genomic rearrangements (<a reveal-id="dsx039-F5" data-open="dsx039-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). PCR products of divergent primers were further detected by Sanger sequencing to confirm the back-splicing junctions (<a reveal-id="dsx039-F5" data-open="dsx039-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s6_dsx039.png?Expires=2147483647&Signature=q289IXksnNvNBIa3CcLuBrYqQ5RV5IxHOjIZ3KaAHiI8VNth81JIG937xtcG9EoZEXP1zTQna~DhhffbTaeFbifcMxve4YgiqchfWHCkFAIxn1ulxc9irzYhj3KHU5FPdvxK03h29Qa1pDmhEmKmp3-Yz2rm-LNxGyoNxpY6mfo48UozgzqcSeVj6PcJ9iARAbTP53JDc7oVN-mOZhNXVpzMVoe17PIQkwvGtN0szL0lB~VDlAHdWU8TLDGdSQI38PzeLrIdKUdBUEAzEu416tpBliO2SNi3r7f0PE9Cd76JHGzL8qnn5ql-CursOjPahXyNzGA2Xau8ZQEp4SxMfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). To detect the resistance of circRNA to the digestion by RNaseR, we quantified the six candidate circRNAs with RNaseR treatment compared to control by qRT-PCR. All tested circRNAs showed much more resistant than the linear mRNA control (<a reveal-id="dsx039-F5" data-open="dsx039-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). In addition, the expression of six differentially expressed circRNAs was also validated by qRT-PCR in E11_VS_E16, E11_VS_P1, and E16_VS_P1 comparison groups. Except the expression of <em>circRBFOX2.2-4</em> and <em>circFGFR2.3-6</em> in P1 VS E16 compared group, the expression patterns of these circRNAs were consistent with the RNA-Seq results, indicating that the deep sequencing results were reliable (<a reveal-id="dsx039-F5" data-open="dsx039-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). </p>                        <a id="112483162" scrollto-destination="112483162"></a>
<div id="dsx039-F5" data-id="dsx039-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f5.png?Expires=2147483647&Signature=Z9sFZimI9A99xJQueJzQlEcXNSFKlF-nios~K~F~PqXIARkEqMimb1233HBquAoRj3FGitqabofU-yH1BHi87Tw8LceSccDnbCnJPcGQ0aXz01rZNrWlz4xScYGZjp7NUxMcr~3yLF1FD6CjLtmWIKtVYWFZBXdYrOZ1lsLK1cTD7SA4IubKNMliVxQUu0~dYLKVcrcrH-Unk-fiUMSgJTUYKOsQzVplT7ICJStKy~VfDxOZ7~K8wvC98XjaCz2nQ7xzz3JSAEBTKJ2Y6LrUHqEHMFMSy~p8qBJKfIUcZ7S0ERUb4V~M~6Pc8XME19g8ZGc0pu2UkuWMYnqFt6MNGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of circular RNAs. (A) Divergent primers amplify circRNAs in cDNA but not genomic DNA (gDNA). White triangles represent convergent primers and black triangles represent divergent primers. (B) Sanger sequencing confirmed the back-splicing junction sequence of circRNAs. (C) qRT-PCR showing resistance of circRNAs to RNaseR digestion. (D) qRT-PCR validation of six differentially expressed circRNAs in all three comparisons. qRT-PCR reactions were run in triplicate and presented as means ± s.e.m. The Student’s t-test was used to compare expression levels among different groups. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483162/dsx039f5.tif" path-from-xml="dsx039f5.tif" target="_blank">View large</a><a section="112483162" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f5.png?Expires=2147483647&Signature=tPsjuQevm422KlgnLzjmIlaheKAbRwkcmxHJYviadDK2tCSxbeIw8T-T18axZOP3qILC-3kT2JVv2yvWsAw~qoevGdmu9kp~7VcTgCKpnuau0vuQhnBjuhj368EKBLVXauMOmTNFUhGrSmctNemC3-lYWrelSuTy6klf-fPOzXdnyh7eXPP6cxkZsHJP9LbFltDg9hir0Q2csdVFnipFVRewIrwPGF53hJtOy~-DUC6dcqwxCpR85Gk5xLtqrsJfxYysxShh7pq85aVJ2pRFafNieSJ6vPgdzDabX4MHzaOkjt2YXaGpUwjCAlIS24xh2RyPGw~wP5OhdE5-aRjZLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483162&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of circular RNAs. (A) Divergent primers amplify circRNAs in cDNA but not genomic DNA (gDNA). White triangles represent convergent primers and black triangles represent divergent primers. (B) Sanger sequencing confirmed the back-splicing junction sequence of circRNAs. (C) qRT-PCR showing resistance of circRNAs to RNaseR digestion. (D) qRT-PCR validation of six differentially expressed circRNAs in all three comparisons. qRT-PCR reactions were run in triplicate and presented as means ± <span class="small-caps">s</span>.<span class="small-caps">e</span>.<span class="small-caps">m</span>. The Student’s t-test was used to compare expression levels among different groups. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div><div content-id="dsx039-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f5.png?Expires=2147483647&Signature=Z9sFZimI9A99xJQueJzQlEcXNSFKlF-nios~K~F~PqXIARkEqMimb1233HBquAoRj3FGitqabofU-yH1BHi87Tw8LceSccDnbCnJPcGQ0aXz01rZNrWlz4xScYGZjp7NUxMcr~3yLF1FD6CjLtmWIKtVYWFZBXdYrOZ1lsLK1cTD7SA4IubKNMliVxQUu0~dYLKVcrcrH-Unk-fiUMSgJTUYKOsQzVplT7ICJStKy~VfDxOZ7~K8wvC98XjaCz2nQ7xzz3JSAEBTKJ2Y6LrUHqEHMFMSy~p8qBJKfIUcZ7S0ERUb4V~M~6Pc8XME19g8ZGc0pu2UkuWMYnqFt6MNGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of circular RNAs. (A) Divergent primers amplify circRNAs in cDNA but not genomic DNA (gDNA). White triangles represent convergent primers and black triangles represent divergent primers. (B) Sanger sequencing confirmed the back-splicing junction sequence of circRNAs. (C) qRT-PCR showing resistance of circRNAs to RNaseR digestion. (D) qRT-PCR validation of six differentially expressed circRNAs in all three comparisons. qRT-PCR reactions were run in triplicate and presented as means ± s.e.m. The Student’s t-test was used to compare expression levels among different groups. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483162/dsx039f5.tif" path-from-xml="dsx039f5.tif" target="_blank">View large</a><a section="112483162" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f5.png?Expires=2147483647&Signature=tPsjuQevm422KlgnLzjmIlaheKAbRwkcmxHJYviadDK2tCSxbeIw8T-T18axZOP3qILC-3kT2JVv2yvWsAw~qoevGdmu9kp~7VcTgCKpnuau0vuQhnBjuhj368EKBLVXauMOmTNFUhGrSmctNemC3-lYWrelSuTy6klf-fPOzXdnyh7eXPP6cxkZsHJP9LbFltDg9hir0Q2csdVFnipFVRewIrwPGF53hJtOy~-DUC6dcqwxCpR85Gk5xLtqrsJfxYysxShh7pq85aVJ2pRFafNieSJ6vPgdzDabX4MHzaOkjt2YXaGpUwjCAlIS24xh2RyPGw~wP5OhdE5-aRjZLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483162&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of circular RNAs. (A) Divergent primers amplify circRNAs in cDNA but not genomic DNA (gDNA). White triangles represent convergent primers and black triangles represent divergent primers. (B) Sanger sequencing confirmed the back-splicing junction sequence of circRNAs. (C) qRT-PCR showing resistance of circRNAs to RNaseR digestion. (D) qRT-PCR validation of six differentially expressed circRNAs in all three comparisons. qRT-PCR reactions were run in triplicate and presented as means ± <span class="small-caps">s</span>.<span class="small-caps">e</span>.<span class="small-caps">m</span>. The Student’s t-test was used to compare expression levels among different groups. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div>                            <h3 scrollto-destination=112483163 id="112483163" class="section-title" >3.5. CircRBFOX2 interacts with miR-206 and miR-1a</h3>
<p>Two circular isoforms of <em>RBFOX2</em> which we designated as <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> were derived from exon2-3 and exon 2-4 of <em>RBFOX2</em>, respectively. Both these molecules possessed miR-1a-3p and miR-206 binding sites (<a reveal-id="dsx039-F6" data-open="dsx039-F6" class="link link-reveal link-table xref-fig">Fig. 6A and B</a>). The prediction of a miRNA target site does not necessarily mean that a miRNA binds to that site. To further validate that these two circRBFOX2s can be interacted with miR-1a-3p and miR-206, we constructed two dual-luciferase reporters by inserting perfect miR-1a-3p or miR-206 target sites into the 3’ end of <em>Renilla</em> luciferase. The knockdown potential of the miRNA was assessed by the presence or absence of circRBFOX2s. After co-transfecting the dual-luciferase reporter with their corresponding miRNAs, the relative luminescence was significantly decreased compared with miR-NC. However, the relative luminescence returned when co-transfected with either <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> expression vectors, but not the mutated vectors of circRBFOX2s (<a reveal-id="dsx039-F6" data-open="dsx039-F6" class="link link-reveal link-table xref-fig">Fig. 6C and D</a>). These results demonstrated that the knockdown potential of miR-1a-3p or miR-206 was significantly diminished in the presence of <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em>. Immunoprecipitation of AGO2 from myoblasts co-transfected with miR-1a-3p, miR-206 or miR-203 and circRBFOX2 overexpression vector was performed and the circRNA levels were quantified by qRT–PCR and normalized to <em>GAPDH</em> (<a reveal-id="dsx039-F6" data-open="dsx039-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>). Compared with miR-203 transfected cells, immunoprecipitation of AGO2 from miR-1a-3p or miR-206 transfected cells were both resulted in an enrichment of circRBFOX2. In addition, compared with miR-203 control, <em>circRBFOX2.2-4</em> was specifically captured by both 3-biotinylated miR-1a-3p and miR-206, and <em>circRBFOX2.2-3</em> was also captured by 3-biotinylated miR-206. Together, these results suggested that circRBFOX2s are able to interact with miR-1a-3p and miR-206. </p>                        <a id="112483165" scrollto-destination="112483165"></a>
<div id="dsx039-F6" data-id="dsx039-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f6.png?Expires=2147483647&Signature=u52DCzfv97okqlGZdvkP95oNNtNqOQNtvSoWJzHcU9fDP8X~ZP4BN0XuM7IGH-5ST18dEuhCWlBUyvU2UJQMGXi0q8PqftKSgLU7OHVp1yAl5qE1yDfgxVF~CvUAU4bk1C1qkJmLgEsDHE3viTJphr0-pBFKBj2P3Shy31aEMvlP0n2ij6Bs5DPdZECMvHPUaClS9Cp4KX8uu~jjzQLo-dSt1WKhozoeV2MTFIiGCqHFUlHaPkoQld3eenIG-8MHRwm1BfL-MmpIJPMYJWTmGAGkfu29OX30IXsk4sPfE4OTlwPfE1QG4YOg0vgUz9gSgqlMbNMtaGa1EYhoDfSHkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CircRBFOX2s interacted with miR-206 and miR-1a-3p. (A) The predicted binding site of miR-1a-3p and miR-206 in the exon3 of RBFOXO2. Mut indicates the mutation sequences of binding sites. (B) miR-1a-3p and miR-206 targeting site in circRBFOX2s analysed by RNAhybrid software. Luciferase reporter assays for miR-1a-3p (C) and miR-206 (D). Luminescence was measured 48 h after transfected with the luciferase reporter and miRNA mimics or NC (negative control) and with circRNAs overexpression vector, circRNA mutated vector or empty vector. The relative levels of Renilla luminescence normalized to firefly luminescence are plotted. Error bars represent s.d. (n = 6). OV1, overexpression vector of circRBFOX2.2-3; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector. (E) Immunoprecipitation of AGO2 from myoblasts co-transfected with miR-1a-3p, miR-206 or miR-203 and circRBFOX2 overexpression vector. The circRNA levels were quantified by qRT–PCR and normalized to the GAPDH, the fold change of immunoprecipitate/input are plotted. Error bars represent s.d. (n = 3). (F) qRT–PCR analysis of circRBFOX2s level in the streptavidin captured fractions from the myoblast lysates after transfection with 3-end biotinylated miR-1a-3p, miR-206 or miR-203 control. Error bars represent s.d. (n = 3). Student’s t test (two-tailed) was performed for data analysis. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483165/dsx039f6.tif" path-from-xml="dsx039f6.tif" target="_blank">View large</a><a section="112483165" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f6.png?Expires=2147483647&Signature=pdpmNa~H4~XbQuQSbIsdi5RuyxSjz7vyjTT-RrQ6PbInTfhxcIvuXQOZS9iG36owv7kUj8dBB97vgOrzF8qXU8na5Sd4y4QEyN9Jmz-kXi22XgqvcCHgQFiCzL2PXqgxLjfnUCJdbULmyaY4YYp1uocVBXn9i5tUd5f1CNphgx3VdOgxz5YLzzaT1v~9dZ2vlxlNVXAU56It-2~UB7MsbXV-XiH9TTbhse1Qycfwk1mnDvP5Q~ujIDTzLQNGruqnruwbClgZJXKNumUv1rGrq4Jau5mptrfmcYm~d6IrlKJXau6IzNOBgaqkjeBl00UDXzupafFmZcsQkZqo6GeTYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483165&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CircRBFOX2s interacted with miR-206 and miR-1a-3p. (A) The predicted binding site of miR-1a-3p and miR-206 in the exon3 of RBFOXO2. Mut indicates the mutation sequences of binding sites. (B) miR-1a-3p and miR-206 targeting site in circRBFOX2s analysed by RNAhybrid software. Luciferase reporter assays for miR-1a-3p (C) and miR-206 (D). Luminescence was measured 48 h after transfected with the luciferase reporter and miRNA mimics or NC (negative control) and with circRNAs overexpression vector, circRNA mutated vector or empty vector. The relative levels of <em>Renilla</em> luminescence normalized to firefly luminescence are plotted. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 6). OV1, overexpression vector of <em>circRBFOX2.2-3</em>; OV2, overexpression vector of <em>circRBFOX2.2-4</em>; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of <em>circRBFOX2.2-4</em>; EV, empty vector. (E) Immunoprecipitation of AGO2 from myoblasts co-transfected with miR-1a-3p, miR-206 or miR-203 and circRBFOX2 overexpression vector. The circRNA levels were quantified by qRT–PCR and normalized to the GAPDH, the fold change of immunoprecipitate/input are plotted. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3). (F) qRT–PCR analysis of circRBFOX2s level in the streptavidin captured fractions from the myoblast lysates after transfection with 3-end biotinylated miR-1a-3p, miR-206 or miR-203 control. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3). Student’s t test (two-tailed) was performed for data analysis. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div><div content-id="dsx039-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f6.png?Expires=2147483647&Signature=u52DCzfv97okqlGZdvkP95oNNtNqOQNtvSoWJzHcU9fDP8X~ZP4BN0XuM7IGH-5ST18dEuhCWlBUyvU2UJQMGXi0q8PqftKSgLU7OHVp1yAl5qE1yDfgxVF~CvUAU4bk1C1qkJmLgEsDHE3viTJphr0-pBFKBj2P3Shy31aEMvlP0n2ij6Bs5DPdZECMvHPUaClS9Cp4KX8uu~jjzQLo-dSt1WKhozoeV2MTFIiGCqHFUlHaPkoQld3eenIG-8MHRwm1BfL-MmpIJPMYJWTmGAGkfu29OX30IXsk4sPfE4OTlwPfE1QG4YOg0vgUz9gSgqlMbNMtaGa1EYhoDfSHkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CircRBFOX2s interacted with miR-206 and miR-1a-3p. (A) The predicted binding site of miR-1a-3p and miR-206 in the exon3 of RBFOXO2. Mut indicates the mutation sequences of binding sites. (B) miR-1a-3p and miR-206 targeting site in circRBFOX2s analysed by RNAhybrid software. Luciferase reporter assays for miR-1a-3p (C) and miR-206 (D). Luminescence was measured 48 h after transfected with the luciferase reporter and miRNA mimics or NC (negative control) and with circRNAs overexpression vector, circRNA mutated vector or empty vector. The relative levels of Renilla luminescence normalized to firefly luminescence are plotted. Error bars represent s.d. (n = 6). OV1, overexpression vector of circRBFOX2.2-3; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector. (E) Immunoprecipitation of AGO2 from myoblasts co-transfected with miR-1a-3p, miR-206 or miR-203 and circRBFOX2 overexpression vector. The circRNA levels were quantified by qRT–PCR and normalized to the GAPDH, the fold change of immunoprecipitate/input are plotted. Error bars represent s.d. (n = 3). (F) qRT–PCR analysis of circRBFOX2s level in the streptavidin captured fractions from the myoblast lysates after transfection with 3-end biotinylated miR-1a-3p, miR-206 or miR-203 control. Error bars represent s.d. (n = 3). Student’s t test (two-tailed) was performed for data analysis. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483165/dsx039f6.tif" path-from-xml="dsx039f6.tif" target="_blank">View large</a><a section="112483165" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f6.png?Expires=2147483647&Signature=pdpmNa~H4~XbQuQSbIsdi5RuyxSjz7vyjTT-RrQ6PbInTfhxcIvuXQOZS9iG36owv7kUj8dBB97vgOrzF8qXU8na5Sd4y4QEyN9Jmz-kXi22XgqvcCHgQFiCzL2PXqgxLjfnUCJdbULmyaY4YYp1uocVBXn9i5tUd5f1CNphgx3VdOgxz5YLzzaT1v~9dZ2vlxlNVXAU56It-2~UB7MsbXV-XiH9TTbhse1Qycfwk1mnDvP5Q~ujIDTzLQNGruqnruwbClgZJXKNumUv1rGrq4Jau5mptrfmcYm~d6IrlKJXau6IzNOBgaqkjeBl00UDXzupafFmZcsQkZqo6GeTYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483165&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CircRBFOX2s interacted with miR-206 and miR-1a-3p. (A) The predicted binding site of miR-1a-3p and miR-206 in the exon3 of RBFOXO2. Mut indicates the mutation sequences of binding sites. (B) miR-1a-3p and miR-206 targeting site in circRBFOX2s analysed by RNAhybrid software. Luciferase reporter assays for miR-1a-3p (C) and miR-206 (D). Luminescence was measured 48 h after transfected with the luciferase reporter and miRNA mimics or NC (negative control) and with circRNAs overexpression vector, circRNA mutated vector or empty vector. The relative levels of <em>Renilla</em> luminescence normalized to firefly luminescence are plotted. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 6). OV1, overexpression vector of <em>circRBFOX2.2-3</em>; OV2, overexpression vector of <em>circRBFOX2.2-4</em>; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of <em>circRBFOX2.2-4</em>; EV, empty vector. (E) Immunoprecipitation of AGO2 from myoblasts co-transfected with miR-1a-3p, miR-206 or miR-203 and circRBFOX2 overexpression vector. The circRNA levels were quantified by qRT–PCR and normalized to the GAPDH, the fold change of immunoprecipitate/input are plotted. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3). (F) qRT–PCR analysis of circRBFOX2s level in the streptavidin captured fractions from the myoblast lysates after transfection with 3-end biotinylated miR-1a-3p, miR-206 or miR-203 control. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3). Student’s t test (two-tailed) was performed for data analysis. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div>                            <h3 scrollto-destination=112483166 id="112483166" class="section-title" >3.6. Effects of circRBFOX2 on expression of miRNAs and their target genes</h3>
<p>We over expressed or knocked down <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> in chicken primary myoblast, and then measured miR-1°-3p and miR-206 expression levels by qRT-PCR. We constructed the pcDNA3.1 vector with linear exon2-3 or exon2-4 of RBFOX2 as control, which give rise to linear RBFOX2 products. We transfected these linear control vector in myoblast and then the miR-206/miR-1a-3p level were quantified by qRT-PCR. The results showed that linear RBFOX2 did not affect the expression of miR-206 and miR-1a-3p (<a reveal-id="dsx039-F7" data-open="dsx039-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a>). </p>                        <a id="112483168" scrollto-destination="112483168"></a>
<div id="dsx039-F7" data-id="dsx039-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f7.png?Expires=2147483647&Signature=UapdjlVsv6iIvI-PGN-88p7~XV~yoljNJEObxCHr5b68OKFmz4cv4mPlO66Z6k-6AV5beChe8R6LJKDkKcufRLgu61-kzubtRhhnGVee6nnWuawo3Z0PHBHos8qQo6K8nX67lNjSC7aYJzRTCFqlPL1xCMyNeQomWDphE21NUAPcl-rvpd6bSvZT3gmqxv6fKLi9kVwM6oXQyeINldpBGny41P-FcrzTlrx6t83hqN2IzSOyvj6a-hsiExdv7Gz6irf3v~zOoMqJkXlJo3ewUjSQVhh5qbe4SsyU1XVzRYmhf4bO0JctP6KzofwyqT2n8Edffq97MJY7givUXu84Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effects of circRBFOX2 on expression of miRNAs and their target gene. (A) Linear RBFOX2 did not affect the expression of miR-206 and miR-1a-3p. The miR-206 and miR-1a-3p level were quantified by qRT-PCR after transfected with linear RBFOX2 overexpression vector or pcDNA3.1 empty vector. L1 represents linear exon2-3 of RBFOX2 overexpression vector; L2 represents linear exon2-4 of RBFOX2 overexpression vector; NC control represents pcDNA3.1 empty vector. (B) Overexpression of circRBFOX2.2-3 or circRBFOX2.2-4 inhibits miR-1a-3p and miR-206 expression. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with circRNAs overexpression vector, circRNA mutated vector or empty vector. Bars represent S.D. (n = 4). (C) The effects different fold change of circRBFOX2.2-3 or circRBFOX2.2-4 on miR-1a-3p and miR-206 expression. (D) Knockdown of circRBFOX2.2-3 or circRBFOX2.2-4 increases miR-1a-3p and miR-206 abundance. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with siRNA of circRBFOX2 or si-NC. Bars represent s.d. (n = 4). (E) Luciferase assay for miR-206 and their target genes. The relative luminescence was measured 48 h after co-transfected with miR-206 mimic and luciferase reporters of no-insert control, wide type target sequence of target genes (WT) or mutated target sequence of target genes (MUT) in DF-1 cell. Error bars represent s.d. (n = 4). The relative levels of CCND2 and HDAC4 were quantified by qRT–PCR after overexpressed (F) or knockdown (G) of circRBFOX2.2-3 or circRBFOX2.2-4 in chicken myoblast. Error bars represent s.d. (n = 4). (H) Knockdown of circRBFOX2.2-3 or circRBFOX2.2-4 increases the expression of miR-206. The expression level of miR-206 was detected after transfected with siRNA of circRBFOX2 or si-NC by Northern blotting. OV1, overexpression vector of circRBFOX2.2-3; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector; Si-1, siRNA of circRBFOX2.2-3; Si-2, siRNA of circRBFOX2.2-4; Si-NC, negative control. Student’s t test (two-tailed) was performed for data analysis. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483168/dsx039f7.tif" path-from-xml="dsx039f7.tif" target="_blank">View large</a><a section="112483168" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f7.png?Expires=2147483647&Signature=qpvIb6L-W5ix~600~moIq~~RME1yDNsfH7AJAWwU1tNuIGuKqjAPfNSXHwkgcQvz1gaSeBZra9ZULtzBPkkMFROmFIK256a-itstFn83XXVgS11-QKn2LwHevSxBwPEohyqshSwFc3itjJjGyQtNmLDhSz02XgR4ORpQQcqn1oLm~-AJRKPp0IJjnTtcJyvB0hHXGIBaMjlFMv2tg6iHrY49~adu721IBkX5SOIbvBy9pL9-Ct67QFOD23fIf3~A3gYehQP~OmyJCFOg~XOnY9hTDMvw0YqyKTrvhREt7cwFrSzfNlugdHSmcsbpyK0h7QAskRV8iOIslSBtz3ptNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483168&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effects of circRBFOX2 on expression of miRNAs and their target gene. (A) Linear <em>RBFOX2</em> did not affect the expression of miR-206 and miR-1a-3p. The miR-206 and miR-1a-3p level were quantified by qRT-PCR after transfected with linear <em>RBFOX2</em> overexpression vector or pcDNA3.1 empty vector. L1 represents linear exon2-3 of <em>RBFOX2</em> overexpression vector; L2 represents linear exon2-4 of <em>RBFOX2</em> overexpression vector; NC control represents pcDNA3.1 empty vector. (B) Overexpression of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> inhibits miR-1a-3p and miR-206 expression. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with circRNAs overexpression vector, circRNA mutated vector or empty vector. Bars represent S.D. (<em>n</em> = 4). (C) The effects different fold change of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> on miR-1a-3p and miR-206 expression. (D) Knockdown of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> increases miR-1a-3p and miR-206 abundance. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with siRNA of circRBFOX2 or si-NC. Bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). (E) Luciferase assay for miR-206 and their target genes. The relative luminescence was measured 48 h after co-transfected with miR-206 mimic and luciferase reporters of no-insert control, wide type target sequence of target genes (WT) or mutated target sequence of target genes (MUT) in DF-1 cell. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). The relative levels of <em>CCND2</em> and <em>HDAC4</em> were quantified by qRT–PCR after overexpressed (F) or knockdown (G) of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> in chicken myoblast. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). (H) Knockdown of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> increases the expression of miR-206. The expression level of miR-206 was detected after transfected with siRNA of circRBFOX2 or si-NC by Northern blotting. OV1, overexpression vector of <em>circRBFOX2.2-3</em>; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of <em>circRBFOX2.2-3</em>; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector; Si-1, siRNA of <em>circRBFOX2.2-3</em>; Si-2, siRNA of <em>circRBFOX2.2-4</em>; Si-NC, negative control. Student’s <em>t</em> test (two-tailed) was performed for data analysis. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div><div content-id="dsx039-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f7.png?Expires=2147483647&Signature=UapdjlVsv6iIvI-PGN-88p7~XV~yoljNJEObxCHr5b68OKFmz4cv4mPlO66Z6k-6AV5beChe8R6LJKDkKcufRLgu61-kzubtRhhnGVee6nnWuawo3Z0PHBHos8qQo6K8nX67lNjSC7aYJzRTCFqlPL1xCMyNeQomWDphE21NUAPcl-rvpd6bSvZT3gmqxv6fKLi9kVwM6oXQyeINldpBGny41P-FcrzTlrx6t83hqN2IzSOyvj6a-hsiExdv7Gz6irf3v~zOoMqJkXlJo3ewUjSQVhh5qbe4SsyU1XVzRYmhf4bO0JctP6KzofwyqT2n8Edffq97MJY7givUXu84Nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effects of circRBFOX2 on expression of miRNAs and their target gene. (A) Linear RBFOX2 did not affect the expression of miR-206 and miR-1a-3p. The miR-206 and miR-1a-3p level were quantified by qRT-PCR after transfected with linear RBFOX2 overexpression vector or pcDNA3.1 empty vector. L1 represents linear exon2-3 of RBFOX2 overexpression vector; L2 represents linear exon2-4 of RBFOX2 overexpression vector; NC control represents pcDNA3.1 empty vector. (B) Overexpression of circRBFOX2.2-3 or circRBFOX2.2-4 inhibits miR-1a-3p and miR-206 expression. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with circRNAs overexpression vector, circRNA mutated vector or empty vector. Bars represent S.D. (n = 4). (C) The effects different fold change of circRBFOX2.2-3 or circRBFOX2.2-4 on miR-1a-3p and miR-206 expression. (D) Knockdown of circRBFOX2.2-3 or circRBFOX2.2-4 increases miR-1a-3p and miR-206 abundance. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with siRNA of circRBFOX2 or si-NC. Bars represent s.d. (n = 4). (E) Luciferase assay for miR-206 and their target genes. The relative luminescence was measured 48 h after co-transfected with miR-206 mimic and luciferase reporters of no-insert control, wide type target sequence of target genes (WT) or mutated target sequence of target genes (MUT) in DF-1 cell. Error bars represent s.d. (n = 4). The relative levels of CCND2 and HDAC4 were quantified by qRT–PCR after overexpressed (F) or knockdown (G) of circRBFOX2.2-3 or circRBFOX2.2-4 in chicken myoblast. Error bars represent s.d. (n = 4). (H) Knockdown of circRBFOX2.2-3 or circRBFOX2.2-4 increases the expression of miR-206. The expression level of miR-206 was detected after transfected with siRNA of circRBFOX2 or si-NC by Northern blotting. OV1, overexpression vector of circRBFOX2.2-3; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector; Si-1, siRNA of circRBFOX2.2-3; Si-2, siRNA of circRBFOX2.2-4; Si-NC, negative control. Student’s t test (two-tailed) was performed for data analysis. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483168/dsx039f7.tif" path-from-xml="dsx039f7.tif" target="_blank">View large</a><a section="112483168" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f7.png?Expires=2147483647&Signature=qpvIb6L-W5ix~600~moIq~~RME1yDNsfH7AJAWwU1tNuIGuKqjAPfNSXHwkgcQvz1gaSeBZra9ZULtzBPkkMFROmFIK256a-itstFn83XXVgS11-QKn2LwHevSxBwPEohyqshSwFc3itjJjGyQtNmLDhSz02XgR4ORpQQcqn1oLm~-AJRKPp0IJjnTtcJyvB0hHXGIBaMjlFMv2tg6iHrY49~adu721IBkX5SOIbvBy9pL9-Ct67QFOD23fIf3~A3gYehQP~OmyJCFOg~XOnY9hTDMvw0YqyKTrvhREt7cwFrSzfNlugdHSmcsbpyK0h7QAskRV8iOIslSBtz3ptNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483168&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effects of circRBFOX2 on expression of miRNAs and their target gene. (A) Linear <em>RBFOX2</em> did not affect the expression of miR-206 and miR-1a-3p. The miR-206 and miR-1a-3p level were quantified by qRT-PCR after transfected with linear <em>RBFOX2</em> overexpression vector or pcDNA3.1 empty vector. L1 represents linear exon2-3 of <em>RBFOX2</em> overexpression vector; L2 represents linear exon2-4 of <em>RBFOX2</em> overexpression vector; NC control represents pcDNA3.1 empty vector. (B) Overexpression of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> inhibits miR-1a-3p and miR-206 expression. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with circRNAs overexpression vector, circRNA mutated vector or empty vector. Bars represent S.D. (<em>n</em> = 4). (C) The effects different fold change of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> on miR-1a-3p and miR-206 expression. (D) Knockdown of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> increases miR-1a-3p and miR-206 abundance. The expression level of miR-1a-3p, miR-206 and miR-203 control was detected after transfected with siRNA of circRBFOX2 or si-NC. Bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). (E) Luciferase assay for miR-206 and their target genes. The relative luminescence was measured 48 h after co-transfected with miR-206 mimic and luciferase reporters of no-insert control, wide type target sequence of target genes (WT) or mutated target sequence of target genes (MUT) in DF-1 cell. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). The relative levels of <em>CCND2</em> and <em>HDAC4</em> were quantified by qRT–PCR after overexpressed (F) or knockdown (G) of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> in chicken myoblast. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). (H) Knockdown of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> increases the expression of miR-206. The expression level of miR-206 was detected after transfected with siRNA of circRBFOX2 or si-NC by Northern blotting. OV1, overexpression vector of <em>circRBFOX2.2-3</em>; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of <em>circRBFOX2.2-3</em>; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector; Si-1, siRNA of <em>circRBFOX2.2-3</em>; Si-2, siRNA of <em>circRBFOX2.2-4</em>; Si-NC, negative control. Student’s <em>t</em> test (two-tailed) was performed for data analysis. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div><p>Compared to empty vector control, overexpressed <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> inhibited both miR-1a-3p and miR-206, while the mutation of <em>circRBFOX2</em> vector did not have significant change (<a reveal-id="dsx039-F7" data-open="dsx039-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>). Furthermore, the expression level of miR-206/miR-1a-3p tend to be lower when the level of circRBFOX2s higher (<a reveal-id="dsx039-F7" data-open="dsx039-F7" class="link link-reveal link-table xref-fig">Fig. 7C</a>). We knock down <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> using siRNAs, and the results showed that the siRNAs could knockdown the circRBFOX2s but not linear RBFOX2. Knockdown of either <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> increased both the expression of miR-1a-3p and miR-206 (<a reveal-id="dsx039-F7" data-open="dsx039-F7" class="link link-reveal link-table xref-fig">Fig. 7D</a>). Northern blot analysis for miR-206 was also confirmed that the miR-206 expression level was increased by knockdown of circRBFOX2s (<a reveal-id="dsx039-F7" data-open="dsx039-F7" class="link link-reveal link-table xref-fig">Fig. 7H</a>). The expression of miR-203 used as control did not affect by neither overexpressed nor knockdown of circRBFOX2s. Together, our results found that circRBFOX2s downregulated the expression of miR-1a-3p and miR-206. This finding was not consisted with the standard model of circRNA miRNA sponge. The circRNAs that act as miRNA sponge can sequesters the miRNA away from other targets, but does not necessarily alter the expression of the miRNA. Recently, there are some circRNAs, such as <em>CircRar1</em>, <em>circ-ABCB10</em> and hsa-circ-0016347 were also reported to downregulated the expression of their target miRNA.<a reveal-id="dsx039-B55 dsx039-B56 dsx039-B57" data-open="dsx039-B55 dsx039-B56 dsx039-B57" class="link link-ref link-reveal xref-bibr"><sup>55–57</sup></a> However, the exact mechanism for this association is unknown, probably upregulation of circRNAs might degrade miRNA expression.</p><p>We further validated whether <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> regulated the target gene of miR-206. According to previous studies and predicted by Targetsclan (<a class="link link-uri" href="http://www.targetscan.org/vert_71/" target="_blank">http://www.targetscan.org/vert_71/</a> (16 September 2017, date last accessed)), we chose the CCND1, CCND2, PAX3 and HDAC4 as candidate target gene for miR-206, and then verified their relations by luciferase assay.<a reveal-id="dsx039-B58 dsx039-B59 dsx039-B60" data-open="dsx039-B58 dsx039-B59 dsx039-B60" class="link link-ref link-reveal xref-bibr"><sup>58–60</sup></a> The CCND2 and HDAC4 were confirmed as the target gene of miR-206 (<a reveal-id="dsx039-F7" data-open="dsx039-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a>). Thus, we further determine their expression level after overexpression/knockdown of circRBFOX2s in myoblast. The results showed that both circRBFOX2.2-3 and circRBFOX2.2-4 upregulated the mRNA level of CCND2 but not HDAC4 (<a reveal-id="dsx039-F7" data-open="dsx039-F7" class="link link-reveal link-table xref-fig">Fig. 7F and G</a>).</p>                            <h3 scrollto-destination=112483171 id="112483171" class="section-title" >3.7. Effects of circRBFOX2 and miR-206 on myoblast proliferation</h3>
<p>To observe the functions of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> on myoblast, we over expressed and knocked down the <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> by transfected with overexpression vector or siRNAs. Overexpression of either <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> resulted in increased levels of <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> above that of the empty vector control (<a reveal-id="dsx039-F8" data-open="dsx039-F8" class="link link-reveal link-table xref-fig">Fig. 8A and B</a>). Similarly, when we transfected <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> siRNAs, the transcript levels of both these RNAs decreased (<a reveal-id="dsx039-F8" data-open="dsx039-F8" class="link link-reveal link-table xref-fig">Fig. 8C and D</a>). These results indicated that our expression system worked in myoblasts. We used flow cytometry for cell cycle analysis of transfected myoblasts. Overexpression of <em>circRBFOX2.2-3</em> resulted in a greater number of S-phase cells than controls, and fewer G0/G1 cells (<a reveal-id="dsx039-F8" data-open="dsx039-F8" class="link link-reveal link-table xref-fig">Fig. 8E and F</a>, <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s7_dsx039.png?Expires=2147483647&amp;Signature=PUQWtDy-a0RpVPHIGny9yg5MEdfWSrpY9jQ6S13DvKV-mAurBvIXFzYbGLxlxgmnRh4Ud58frQ5v3SQ34CILrc~ah81Y1FhahIT38ZBmxphhZrlUGPlMh0DDR6sJe2UvMKiOrW~GXMPe1nqZ3GadT7alJ0RtfYSEBE9JwHyIp8IJttoTzEUpFEokIcjTluzC5VpA-abNrScdU8v5Suge7exxM6txG7yAdHLSaXn8U0g8JiSfp~jwPQ0c1gGm27YRKr2CFeHneELRcQGSRS9ooFkSTU4~16AUqPoAdfVy92fSppREkdKlVml8dmF6GCEg1r-dmiu9eUN0zVOb7jWecQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s8_dsx039.png?Expires=2147483647&amp;Signature=QKSB30C9rhasN8PQeJu3bIop4~WvFNuMrte1atjCWFq8AvlfGoCUP3y2pKgYCuehEwaBRFcV-o79f-FWdHfGV-ucaO-v1gVKUDkPHy6ME7Fjoosk~NTCeJxkiuW3-~7kW5XHB6MVMv68OwXIUnv5lY34CPk-oj7TBZoJQMj-O4dZTAgc0ZHhCXlKVlF7D2TlrajDZ2qq5cz7mQ5WjpN09P7won85vSITlKn2rnjx07dEKMGrdOXl5aJNlipsMKl0~z-M11DvsLigKcYyKik7fhfd49CKD8bk3u0pW94irCUW0jhLpVebiKMUCX7X-g4DobcJYJAfVYoJg0tfKq0PkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S8</a></span>). Both the mutated vector of <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> did not affect the cell cycle of myoblast. Knockdown of <em>circRBFOX2.2-3</em> resulted in a greater number of G0/G1 cells and fewer S and G2/M phase cells than controls. However, the <em>circRBFOX2.2-4</em> knockdown and overexpression groups showed significant differences only in S-phase (<a reveal-id="dsx039-F8" data-open="dsx039-F8" class="link link-reveal link-table xref-fig">Fig. 8E and F</a>, <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s7_dsx039.png?Expires=2147483647&amp;Signature=PUQWtDy-a0RpVPHIGny9yg5MEdfWSrpY9jQ6S13DvKV-mAurBvIXFzYbGLxlxgmnRh4Ud58frQ5v3SQ34CILrc~ah81Y1FhahIT38ZBmxphhZrlUGPlMh0DDR6sJe2UvMKiOrW~GXMPe1nqZ3GadT7alJ0RtfYSEBE9JwHyIp8IJttoTzEUpFEokIcjTluzC5VpA-abNrScdU8v5Suge7exxM6txG7yAdHLSaXn8U0g8JiSfp~jwPQ0c1gGm27YRKr2CFeHneELRcQGSRS9ooFkSTU4~16AUqPoAdfVy92fSppREkdKlVml8dmF6GCEg1r-dmiu9eUN0zVOb7jWecQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s8_dsx039.png?Expires=2147483647&amp;Signature=QKSB30C9rhasN8PQeJu3bIop4~WvFNuMrte1atjCWFq8AvlfGoCUP3y2pKgYCuehEwaBRFcV-o79f-FWdHfGV-ucaO-v1gVKUDkPHy6ME7Fjoosk~NTCeJxkiuW3-~7kW5XHB6MVMv68OwXIUnv5lY34CPk-oj7TBZoJQMj-O4dZTAgc0ZHhCXlKVlF7D2TlrajDZ2qq5cz7mQ5WjpN09P7won85vSITlKn2rnjx07dEKMGrdOXl5aJNlipsMKl0~z-M11DvsLigKcYyKik7fhfd49CKD8bk3u0pW94irCUW0jhLpVebiKMUCX7X-g4DobcJYJAfVYoJg0tfKq0PkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S8</a></span>). The 5-Ethynyl-2’-deoxyuridine (EdU) incorporation assays were also performed after over expressed and knocked down the <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> in myoblast. The number of EdU-stained cells increased in both the <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> overexpression groups. In contrast, proliferation decreased when <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> were knocked down (<a reveal-id="dsx039-F8" data-open="dsx039-F8" class="link link-reveal link-table xref-fig">Fig. 8G and H</a>). </p>                        <a id="112483173" scrollto-destination="112483173"></a>
<div id="dsx039-F8" data-id="dsx039-F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f8.png?Expires=2147483647&Signature=QvafeV0r55RW7Vnsg0U1t-eba~kipdGO8WpM7RiqXdjxGHIV1JTbfnf49UV~Zgc52WwjUNNcsE~QOUPWOUp~9pBiVDDY52~NtLmQM1mpFzaJ2iKzWnRcunHYLmqQnzL7HeJ2po5PR9bbByKoRSYTpFt3b1CCMa9jXvaHdK9Y7kNhc6FT1ZDiQvsnEhutEunPwkvPbCiaXsKs5RhT1iTALW2w8O8IirgUB7ypys81svhrH9q2pHMj5NCxUTWEoWUtNZ8ruOldgrnSvNZTH7JvaAij5oAPk58J1KWSgkikhNVBqeN9vLO2S8n65tSGnH6wpJJt9~t81sa6UyUnkUwMoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chicken circRBFOX2.2-3 and circRBFOX2.2-4 regulate QM-7 cell proliferation. Overexpression of circRBFOX2.2-3 (A) and circRBFOX2.2-4 (B). The relative expression was determined by qRT-PCR in QM-7 after transfected with overexpression vector or EV control. Knocked down of circRBFOX2.2-3 (C) and circRBFOX2.2-4 (D). The relative expression was determined by qRT-PCR in QM-7 after transfected with si-circRBFOX2.2-3, si-circRBFOX2.2-4 or oligo control. Cell cycle analysis of myoblasts transfected with (E) overexpression vectors, mutated vector or (F) siRNAs were performed using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent s.d. (n = 4). OV1, overexpression vector of circRBFOX2.2-3; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector. (G) Photomicrographs of QM-7 myoblasts stained with EdU to assess proliferation. Cells were transfected with either overexpression vectors or siRNAs and assayed after 48 h. EdU (red) fluorescence indicates proliferation. Nuclei are indicated by Hoechst (blue) fluorescence. All photomicrographs are at 200× magnification. (H) The percentage of EdU-stained cells per total cell numbers. Error bars represent s.d. (n = 3). Student’s t test (two-tailed) was performed for data analysis. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483173/dsx039f8.tif" path-from-xml="dsx039f8.tif" target="_blank">View large</a><a section="112483173" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f8.png?Expires=2147483647&Signature=zCQb7Y6yCVPhwoSgqkgWyIj3BEZnK9T-nea7l5bkC9QED7fbvtjVMhi4t6pyt~cztT~Adl9zGZ3RJRdfLnRdpHNIkYcYa7OfG0JUusCLL0kWouxfsS56WuuaNgsJLd1Z7nCeZ2WdcMBdXEih8KOXuZXMtXcFAePoQVayNcc7XAbsMWnVPK-w1oMxQAE-RuQ0ta-42uAETRukOFrPESM6JP~EP2JQN52UyZBtbthGmkZ5RGWn8ploL9i56ivODhBsLqcxaVRXYMgiFq8KiUFMQpOq78IwQHeVxMPPhhYYHRSAWH1-57bvmjMKoBClAF~cj7HK0gywQDGuiDG6BXcidA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483173&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chicken circRBFOX2.2-3 and circRBFOX2.2-4 regulate QM-7 cell proliferation. Overexpression of <em>circRBFOX2.2-3</em> (A) and <em>circRBFOX2.2-4</em> (B). The relative expression was determined by qRT-PCR in QM-7 after transfected with overexpression vector or EV control. Knocked down of <em>circRBFOX2.2-3</em> (C) and <em>circRBFOX2.2-4</em> (D). The relative expression was determined by qRT-PCR in QM-7 after transfected with si-circRBFOX2.2-3, si-circRBFOX2.2-4 or oligo control. Cell cycle analysis of myoblasts transfected with (E) overexpression vectors, mutated vector or (F) siRNAs were performed using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). OV1, overexpression vector of <em>circRBFOX2.2-3</em>; OV2, overexpression vector of <em>circRBFOX2.2-4</em>; Mut1, mutated vector of <em>circRBFOX2.2-3</em>; Mut2, mutated vector of <em>circRBFOX2.2-4</em>; EV, empty vector. (G) Photomicrographs of QM-7 myoblasts stained with EdU to assess proliferation. Cells were transfected with either overexpression vectors or siRNAs and assayed after 48 h. EdU (red) fluorescence indicates proliferation. Nuclei are indicated by Hoechst (blue) fluorescence. All photomicrographs are at 200× magnification. (H) The percentage of EdU-stained cells per total cell numbers. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3). Student’s t test (two-tailed) was performed for data analysis. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div><div content-id="dsx039-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f8.png?Expires=2147483647&Signature=QvafeV0r55RW7Vnsg0U1t-eba~kipdGO8WpM7RiqXdjxGHIV1JTbfnf49UV~Zgc52WwjUNNcsE~QOUPWOUp~9pBiVDDY52~NtLmQM1mpFzaJ2iKzWnRcunHYLmqQnzL7HeJ2po5PR9bbByKoRSYTpFt3b1CCMa9jXvaHdK9Y7kNhc6FT1ZDiQvsnEhutEunPwkvPbCiaXsKs5RhT1iTALW2w8O8IirgUB7ypys81svhrH9q2pHMj5NCxUTWEoWUtNZ8ruOldgrnSvNZTH7JvaAij5oAPk58J1KWSgkikhNVBqeN9vLO2S8n65tSGnH6wpJJt9~t81sa6UyUnkUwMoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chicken circRBFOX2.2-3 and circRBFOX2.2-4 regulate QM-7 cell proliferation. Overexpression of circRBFOX2.2-3 (A) and circRBFOX2.2-4 (B). The relative expression was determined by qRT-PCR in QM-7 after transfected with overexpression vector or EV control. Knocked down of circRBFOX2.2-3 (C) and circRBFOX2.2-4 (D). The relative expression was determined by qRT-PCR in QM-7 after transfected with si-circRBFOX2.2-3, si-circRBFOX2.2-4 or oligo control. Cell cycle analysis of myoblasts transfected with (E) overexpression vectors, mutated vector or (F) siRNAs were performed using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent s.d. (n = 4). OV1, overexpression vector of circRBFOX2.2-3; OV2, overexpression vector of circRBFOX2.2-4; Mut1, mutated vector of circRBFOX2.2-3; Mut2, mutated vector of circRBFOX2.2-4; EV, empty vector. (G) Photomicrographs of QM-7 myoblasts stained with EdU to assess proliferation. Cells were transfected with either overexpression vectors or siRNAs and assayed after 48 h. EdU (red) fluorescence indicates proliferation. Nuclei are indicated by Hoechst (blue) fluorescence. All photomicrographs are at 200× magnification. (H) The percentage of EdU-stained cells per total cell numbers. Error bars represent s.d. (n = 3). Student’s t test (two-tailed) was performed for data analysis. *P &lt; 0.05; **P &lt; 0.01." path-from-xml="dsx039f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483173/dsx039f8.tif" path-from-xml="dsx039f8.tif" target="_blank">View large</a><a section="112483173" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f8.png?Expires=2147483647&Signature=zCQb7Y6yCVPhwoSgqkgWyIj3BEZnK9T-nea7l5bkC9QED7fbvtjVMhi4t6pyt~cztT~Adl9zGZ3RJRdfLnRdpHNIkYcYa7OfG0JUusCLL0kWouxfsS56WuuaNgsJLd1Z7nCeZ2WdcMBdXEih8KOXuZXMtXcFAePoQVayNcc7XAbsMWnVPK-w1oMxQAE-RuQ0ta-42uAETRukOFrPESM6JP~EP2JQN52UyZBtbthGmkZ5RGWn8ploL9i56ivODhBsLqcxaVRXYMgiFq8KiUFMQpOq78IwQHeVxMPPhhYYHRSAWH1-57bvmjMKoBClAF~cj7HK0gywQDGuiDG6BXcidA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483173&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chicken circRBFOX2.2-3 and circRBFOX2.2-4 regulate QM-7 cell proliferation. Overexpression of <em>circRBFOX2.2-3</em> (A) and <em>circRBFOX2.2-4</em> (B). The relative expression was determined by qRT-PCR in QM-7 after transfected with overexpression vector or EV control. Knocked down of <em>circRBFOX2.2-3</em> (C) and <em>circRBFOX2.2-4</em> (D). The relative expression was determined by qRT-PCR in QM-7 after transfected with si-circRBFOX2.2-3, si-circRBFOX2.2-4 or oligo control. Cell cycle analysis of myoblasts transfected with (E) overexpression vectors, mutated vector or (F) siRNAs were performed using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). OV1, overexpression vector of <em>circRBFOX2.2-3</em>; OV2, overexpression vector of <em>circRBFOX2.2-4</em>; Mut1, mutated vector of <em>circRBFOX2.2-3</em>; Mut2, mutated vector of <em>circRBFOX2.2-4</em>; EV, empty vector. (G) Photomicrographs of QM-7 myoblasts stained with EdU to assess proliferation. Cells were transfected with either overexpression vectors or siRNAs and assayed after 48 h. EdU (red) fluorescence indicates proliferation. Nuclei are indicated by Hoechst (blue) fluorescence. All photomicrographs are at 200× magnification. (H) The percentage of EdU-stained cells per total cell numbers. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3). Student’s t test (two-tailed) was performed for data analysis. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></div></div><p>Muscle-specific miR-206 has been reported to inhibit cell proliferation in human and mouse.<a reveal-id="dsx039-B58 dsx039-B59" data-open="dsx039-B58 dsx039-B59" class="link link-ref link-reveal xref-bibr"><sup>58–59</sup></a> The effects of miR-206 in myoblast proliferation were also examined using cell cycle and EdU incorporation analysis. We overexpressed and knocked down miR-206 by transfecting myoblasts with a miR-206 mimic or anti-miR-206 (<a reveal-id="dsx039-F9" data-open="dsx039-F9" class="link link-reveal link-table xref-fig">Fig. 9A and B</a>). Myoblasts transfected with the miR-206 mimic decreased the number of S-phase cells compared with the scrambled negative control group (<a reveal-id="dsx039-F9" data-open="dsx039-F9" class="link link-reveal link-table xref-fig">Fig. 9C</a>). Conversely, S-phase cells in the anti-miR-206 group were significantly increased (<a reveal-id="dsx039-F9" data-open="dsx039-F9" class="link link-reveal link-table xref-fig">Fig. 9D</a>). Consistent with the result of cell cycle analysis, EdU incorporation indicated that myoblast proliferation was suppressed by miR-206 overexpression and increased by miR-206 knockdown (<a reveal-id="dsx039-F9" data-open="dsx039-F9" class="link link-reveal link-table xref-fig">Fig. 9A and F</a>). Interestingly, the effects of miR-206 on myoblast proliferation was impaired when either <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> vectors were co-transfected (<a reveal-id="dsx039-F9" data-open="dsx039-F9" class="link link-reveal link-table xref-fig">Fig. 9E and F</a>). Together, these data implied that miR-206 could suppress myoblast proliferation, whereas <em>circRBFOX2.2-3</em> and <em>circRBFOX2.2-4</em> can promote myoblast proliferation, and antagonize the functions of miR-206 in myoblasts. Our results support previous findings that circRNAs sequester miRNAs to suppress their functions in chickens. Nevertheless, only a few circRNAs that function in this manner have been described. Some reports argue that most circRNAs do not function as miRNA sponges in human and mouse.<a reveal-id="dsx039-B20" data-open="dsx039-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx039-B61" data-open="dsx039-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a><sup>,</sup><a reveal-id="dsx039-B62" data-open="dsx039-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Whether this mechanism is a common theme in circRNA biology awaits future research. </p>                        <a id="112483175" scrollto-destination="112483175"></a>
<div id="dsx039-F9" data-id="dsx039-F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f9.png?Expires=2147483647&Signature=OtmD~14Ih7AdVyatGz90uC6XUK4jddJmmuEQMnAzu7BtbDCyBmOw5cRVnnahGbw3e1gFOmadBBhLOUVx~tEvMmBv2glUmAxQfCMUTrUy44XdwuV0ObT5cduxc8NxExSe1DUfiD7ycDuDO-g6NEWBQJLc8VS8AkQM-imsjvDTASRl~xlsfIF8-fC4VbX6I-RYjpWZybpqBToIg5R6K~wxF-uVPwIS5UZKjMc-Px7WHNjHn4uENW0awKimw5s~~kmww2ywJG2GECWsE5ATaQ0Ez44k8rR8qaNJGKWJHdQdjP9HPB-nxIbxfMFeqRj5XIwhmXOCUv3ISK3aTHqNVLKG8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The effect of miR-206 on QM-7 cell proliferation was rescued by circRBFOX2s. (A) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with a miR-206 mimic or control duplexes. (B) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with anti-miR-206 or anti-NC. Cell cycle analysis of myoblasts transfected with miR-206 mimic (C) or anti-miR-206 (D) using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent s.d. (n = 4). *P &lt; 0.05; **P &lt; 0.01. (E) EdU assays were performed in QM-7cells after 48 h co-transfected a miR-206 mimic, mimic control, anti-miR-206, anti-NC, miR-206 mimic with circRBFOX2.2-3 or circRBFOX2.2-4 overexpression vector. Cells were fluorescently stained with EdU (red). Nuclei were stained with Hoechst (blue). All photomicrographs are at 200 × magnification. (F) The percentage of EdU-stained cells per total number. Error bars represent s.d. (n = 3); *P &lt; 0.05." path-from-xml="dsx039f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483175/dsx039f9.tif" path-from-xml="dsx039f9.tif" target="_blank">View large</a><a section="112483175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f9.png?Expires=2147483647&Signature=Et-0fK6Yox6eXuPhRM8SsMzThjyqvGBVnm-FnhKrAL134Pz7-9dTYzzY-TqzTeNDUpB7h~vdzwMZWoSq8hc6iv5bMUafaBilAMazIhuW30rcauNPKOZYEoWln7oZoSKhK5uDqO02NgV8Rod1DNpA-cwt06r6ow4ijtxrE6gEWumQOngIVXEG338xPwsR1D~~u26EyjygcOzIfa2s8K35tCLVexwsbdcmxRWXvTYyum5Kw05tJ-VauiuqFoMIt7WspFDiEsVKar5ElPT6yowIaqCIackT8dPFg6rjHy0jKH50iin-j-IlMbJgpILgx62U7BrUCQbsLuhba-KaxwVdBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483175&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The effect of miR-206 on QM-7 cell proliferation was rescued by circRBFOX2s. (A) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with a miR-206 mimic or control duplexes. (B) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with anti-miR-206 or anti-NC. Cell cycle analysis of myoblasts transfected with miR-206 mimic (C) or anti-miR-206 (D) using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). *<em>P</em> &lt; 0.05; **<em>P </em>&lt; 0.01. (E) EdU assays were performed in QM-7cells after 48 h co-transfected a miR-206 mimic, mimic control, anti-miR-206, anti-NC, miR-206 mimic with <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> overexpression vector. Cells were fluorescently stained with EdU (red). Nuclei were stained with Hoechst (blue). All photomicrographs are at 200 × magnification. (F) The percentage of EdU-stained cells per total number. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3); *<em>P </em>&lt; 0.05.</p></div></div><div content-id="dsx039-F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/m_dsx039f9.png?Expires=2147483647&Signature=OtmD~14Ih7AdVyatGz90uC6XUK4jddJmmuEQMnAzu7BtbDCyBmOw5cRVnnahGbw3e1gFOmadBBhLOUVx~tEvMmBv2glUmAxQfCMUTrUy44XdwuV0ObT5cduxc8NxExSe1DUfiD7ycDuDO-g6NEWBQJLc8VS8AkQM-imsjvDTASRl~xlsfIF8-fC4VbX6I-RYjpWZybpqBToIg5R6K~wxF-uVPwIS5UZKjMc-Px7WHNjHn4uENW0awKimw5s~~kmww2ywJG2GECWsE5ATaQ0Ez44k8rR8qaNJGKWJHdQdjP9HPB-nxIbxfMFeqRj5XIwhmXOCUv3ISK3aTHqNVLKG8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The effect of miR-206 on QM-7 cell proliferation was rescued by circRBFOX2s. (A) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with a miR-206 mimic or control duplexes. (B) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with anti-miR-206 or anti-NC. Cell cycle analysis of myoblasts transfected with miR-206 mimic (C) or anti-miR-206 (D) using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent s.d. (n = 4). *P &lt; 0.05; **P &lt; 0.01. (E) EdU assays were performed in QM-7cells after 48 h co-transfected a miR-206 mimic, mimic control, anti-miR-206, anti-NC, miR-206 mimic with circRBFOX2.2-3 or circRBFOX2.2-4 overexpression vector. Cells were fluorescently stained with EdU (red). Nuclei were stained with Hoechst (blue). All photomicrographs are at 200 × magnification. (F) The percentage of EdU-stained cells per total number. Error bars represent s.d. (n = 3); *P &lt; 0.05." path-from-xml="dsx039f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112483175/dsx039f9.tif" path-from-xml="dsx039f9.tif" target="_blank">View large</a><a section="112483175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/dsx039f9.png?Expires=2147483647&Signature=Et-0fK6Yox6eXuPhRM8SsMzThjyqvGBVnm-FnhKrAL134Pz7-9dTYzzY-TqzTeNDUpB7h~vdzwMZWoSq8hc6iv5bMUafaBilAMazIhuW30rcauNPKOZYEoWln7oZoSKhK5uDqO02NgV8Rod1DNpA-cwt06r6ow4ijtxrE6gEWumQOngIVXEG338xPwsR1D~~u26EyjygcOzIfa2s8K35tCLVexwsbdcmxRWXvTYyum5Kw05tJ-VauiuqFoMIt7WspFDiEsVKar5ElPT6yowIaqCIackT8dPFg6rjHy0jKH50iin-j-IlMbJgpILgx62U7BrUCQbsLuhba-KaxwVdBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112483175&ar=4265514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx039f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The effect of miR-206 on QM-7 cell proliferation was rescued by circRBFOX2s. (A) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with a miR-206 mimic or control duplexes. (B) miR-206 expression was determined by qRT-PCR in QM-7 cells after transfection with anti-miR-206 or anti-NC. Cell cycle analysis of myoblasts transfected with miR-206 mimic (C) or anti-miR-206 (D) using propidium iodide staining for DNA content and a FACSAriaII flow cytometer. Bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 4). *<em>P</em> &lt; 0.05; **<em>P </em>&lt; 0.01. (E) EdU assays were performed in QM-7cells after 48 h co-transfected a miR-206 mimic, mimic control, anti-miR-206, anti-NC, miR-206 mimic with <em>circRBFOX2.2-3</em> or <em>circRBFOX2.2-4</em> overexpression vector. Cells were fluorescently stained with EdU (red). Nuclei were stained with Hoechst (blue). All photomicrographs are at 200 × magnification. (F) The percentage of EdU-stained cells per total number. Error bars represent <span class="small-caps">s</span>.<span class="small-caps">d</span>. (<em>n</em> = 3); *<em>P </em>&lt; 0.05.</p></div></div><p>In summary, we performed genome-wide identification of chicken circRNAs by RNA sequencing, and found they are abundant and differentially expressed during chicken embryonic development. Most of the exonic circRNAs harbored miRNA binding sites and could play key roles in embryonic muscle development through sequestering miRNAs as well as other functions.</p>                            <h2 scrollto-destination=112483177 id="112483177" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Prof. Zhang’s lab members for helpful discussions and the SCAU chicken farm for providing breeding eggs.</p>                            <h3 scrollto-destination=112483179 id="112483179" class="section-title" >Accession numbers</h3>
<p>RNA-seq data are available in the Gene Expression Omnibus (GEO) with accession number (GSE89355).</p>                            <h3 scrollto-destination=112483181 id="112483181" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s1_dsx039.png?Expires=2147483647&Signature=3lFoad0PPPRn9ld~g90ig~qJDhoC41zi5LDlFGtKGa6EmJFKECUbcvVnUa1bUnDg6FxzDGkSuZbBDLdEV6E6aD~iYR314aZ3rZWeu5ItHc-7R7gQpQ8h5sODXHMqx2iqeApcyPRqcHVB1tUdl0F0rpI0LkEonNl52hFS32T9kLToT3H13g4Eu7NIvNjq7QQwONfmIFchXFt29kAi2D9eLXOYDd-4AH6wclizxNNDT-saLpA0CDR7DXbscbkdyHmW1i0aLv0Nf6Mig4nEbUjuQht0npYXKV4JwDUN5KJ2vtSEf9KhyRTBD5idodSCJbsfQqwK-rO7JpkBNpnorA8rwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=112483183 id="112483183" class="section-title" >Funding</h3>
<p>This research was supported by the Natural Scientific Foundation of China (31472090), the China Agriculture Research System (CARS-41-G03), and the Foundation for High-level Talents in Higher Education in Guangdong, China.</p>                            <h3 scrollto-destination=112483185 id="112483185" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=112483187 id="112483187" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx039-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Djebali</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Davis</div> <div class="given-names">C. A.</div></div>, <div class="name string-name"><div class="surname">Merkel</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Landscape of transcription in human cells,</div> <div class="source">Nature</div> , <div class="volume">489</div>, <div class="fpage">101</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Landscape%20of%20transcription%20in%20human%20cells%2C&amp;author=S.%20Djebali&amp;author=C.%20A.%20Davis&amp;author=A.%20Merkel&amp;publication_year=2012&amp;journal=Nature&amp;volume=489&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Batista</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H. Y.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Long noncoding RNAs: cellular address codes in development and disease,</div> <div class="source">Cell</div> , <div class="volume">152</div>, <div class="fpage">1298</div>–<div class="lpage">307</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20noncoding%20RNAs%3A%20cellular%20address%20codes%20in%20development%20and%20disease%2C&amp;author=P.%20J.%20Batista&amp;author=H.%20Y.%20Chang&amp;publication_year=2013&amp;journal=Cell&amp;volume=152&amp;pages=1298-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.02.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.02.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.02.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23498938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D. P.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">MicroRNAs: target recognition and regulatory functions,</div> <div class="source">Cell</div> , <div class="volume">136</div>, <div class="fpage">215</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAs%3A%20target%20recognition%20and%20regulatory%20functions%2C&amp;author=D.%20P.%20Bartel&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=215-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19167326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C. P.</div></div>, <div class="name string-name"><div class="surname">Oliver</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Reik</div> <div class="given-names">W.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Evolution and functions of long noncoding RNAs,</div> <div class="source">Cell</div> , <div class="volume">136</div>, <div class="fpage">629</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20functions%20of%20long%20noncoding%20RNAs%2C&amp;author=C.%20P.%20Ponting&amp;author=P.%20L.%20Oliver&amp;author=W.%20Reik&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=629-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19239885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rinn</div> <div class="given-names">J. L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Modular regulatory principles of large non-coding RNAs,</div> <div class="source">Nature</div> , <div class="volume">482</div>, <div class="fpage">339</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modular%20regulatory%20principles%20of%20large%20non-coding%20RNAs%2C&amp;author=M.%20Guttman&amp;author=J.%20L.%20Rinn&amp;publication_year=2012&amp;journal=Nature&amp;volume=482&amp;pages=339-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10887" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10887" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10887"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22337053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sanger</div> <div class="given-names">H. L.</div></div>, <div class="name string-name"><div class="surname">Klotz</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Riesner</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gross</div> <div class="given-names">H. J.</div></div>, <div class="name string-name"><div class="surname">Kleinschmidt</div> <div class="given-names">A. K.</div></div></div> <div class="year">1976</div>, 
<div class="article-title">Viroids are single-stranded covalently closed circular RNA molecules existing as highly base-paired rod-like structures</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">73</div>, <div class="fpage">3852</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Viroids%20are%20single-stranded%20covalently%20closed%20circular%20RNA%20molecules%20existing%20as%20highly%20base-paired%20rod-like%20structures&amp;author=H.%20L.%20Sanger&amp;author=G.%20Klotz&amp;author=D.%20Riesner&amp;author=H.%20J.%20Gross&amp;author=A.%20K.%20Kleinschmidt&amp;publication_year=1976&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=73&amp;pages=3852-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.73.11.3852" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.73.11.3852" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.73.11.3852"> </span></div> </div></div></div></div></div><div content-id="dsx039-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Capel</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Swain</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nicolis</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">1993</div>, 
<div class="article-title">Circular transcripts of the testis-determining gene Sry in adult mouse testis,</div> <div class="source">Cell</div> , <div class="volume">73</div>, <div class="fpage">1019</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20transcripts%20of%20the%20testis-determining%20gene%20Sry%20in%20adult%20mouse%20testis%2C&amp;author=B.%20Capel&amp;author=A.%20Swain&amp;author=S.%20Nicolis&amp;publication_year=1993&amp;journal=Cell&amp;volume=73&amp;pages=1019-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(93)90279-Y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(93)90279-Y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(93)90279-Y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7684656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zaphiropoulos</div> <div class="given-names">P. G.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Circular RNAs from transcripts of the rat cytochrome P450 2C24 gene: correlation with exon skipping,</div> <div class="source">Proc Natl Acad. Sci. U S A</div> , <div class="volume">93</div>, <div class="fpage">6536</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20from%20transcripts%20of%20the%20rat%20cytochrome%20P450%E2%80%892C24%20gene%3A%20correlation%20with%20exon%20skipping%2C&amp;author=P.%20G.%20Zaphiropoulos&amp;publication_year=1996&amp;journal=Proc%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=93&amp;pages=6536-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.13.6536" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.13.6536" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.13.6536"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8692851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zaphiropoulos</div> <div class="given-names">P. G.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Exon skipping and circular RNA formation in transcripts of the human cytochrome P-450 2C18 gene in epidermis and of the rat androgen binding protein gene in testis,</div> <div class="source">Mol. Cell. Biol</div> ., <div class="volume">17</div>, <div class="fpage">2985</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exon%20skipping%20and%20circular%20RNA%20formation%20in%20transcripts%20of%20the%20human%20cytochrome%20P-450%E2%80%892C18%20gene%20in%20epidermis%20and%20of%20the%20rat%20androgen%20binding%20protein%20gene%20in%20testis%2C&amp;author=P.%20G.%20Zaphiropoulos&amp;publication_year=1997&amp;journal=Mol.%20Cell.%20Biol&amp;volume=17&amp;pages=2985-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.17.6.2985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.17.6.2985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.17.6.2985"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9154796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">X. F.</div></div>, <div class="name string-name"><div class="surname">Lytton</div> <div class="given-names">J.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">A circularized sodium-calcium exchanger exon 2 transcript,</div> <div class="source">J. Biol. Chem</div> ., <div class="volume">274</div>, <div class="fpage">8153</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20circularized%20sodium-calcium%20exchanger%20exon%202%20transcript%2C&amp;author=X.%20F.%20Li&amp;author=J.%20Lytton&amp;publication_year=1999&amp;journal=J.%20Biol.%20Chem&amp;volume=274&amp;pages=8153-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.274.12.8153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.274.12.8153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.274.12.8153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10075718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cocquerelle</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Mascrez</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Hetuin</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Bailleul</div> <div class="given-names">B.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Mis-splicing yields circular RNA molecules,</div> <div class="source">FASEB J</div> ., <div class="volume">7</div>, <div class="fpage">155</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mis-splicing%20yields%20circular%20RNA%20molecules%2C&amp;author=C.%20Cocquerelle&amp;author=B.%20Mascrez&amp;author=D.%20Hetuin&amp;author=B.%20Bailleul&amp;publication_year=1993&amp;journal=FASEB%20J&amp;volume=7&amp;pages=155-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1096/fasebj.7.1.7678559" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1096%2Ffasebj.7.1.7678559" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1096%2Ffasebj.7.1.7678559"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7678559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salzman</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">R. E.</div></div>, <div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">M. N.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Cell-type specific features of circular RNA expression,</div> <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003777</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell-type%20specific%20features%20of%20circular%20RNA%20expression%2C&amp;author=J.%20Salzman&amp;author=R.%20E.%20Chen&amp;author=M.%20N.%20Olsen&amp;author=P.%20L.%20Wang&amp;author=P.%20O.%20Brown&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003777" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003777" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003777" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003777"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24039610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Memczak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Elefsinioti</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Circular RNAs are a large class of animal RNAs with regulatory potency,</div> <div class="source">Nature</div> , <div class="volume">495</div>, <div class="fpage">333</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20are%20a%20large%20class%20of%20animal%20RNAs%20with%20regulatory%20potency%2C&amp;author=S.%20Memczak&amp;author=M.%20Jens&amp;author=A.%20Elefsinioti&amp;publication_year=2013&amp;journal=Nature&amp;volume=495&amp;pages=333-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23446348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">T. B.</div></div>, <div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">T. I.</div></div>, <div class="name string-name"><div class="surname">Clausen</div> <div class="given-names">B. H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Natural RNA circles function as efficient microRNA sponges,</div> <div class="source">Nature</div> , <div class="volume">495</div>, <div class="fpage">384</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20RNA%20circles%20function%20as%20efficient%20microRNA%20sponges%2C&amp;author=T.%20B.%20Hansen&amp;author=T.%20I.%20Jensen&amp;author=B.%20H.%20Clausen&amp;publication_year=2013&amp;journal=Nature&amp;volume=495&amp;pages=384-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11993" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11993" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11993"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23446346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rybak-Wolf</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Stottmeister</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Glazar</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Circular RNAs in the mammalian brain are highly abundant, conserved, and dynamically expressed,</div> <div class="source">Mol. Cell</div> , <div class="volume">58</div>, <div class="fpage">870</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20in%20the%20mammalian%20brain%20are%20highly%20abundant%2C%20conserved%2C%20and%20dynamically%20expressed%2C&amp;author=A.%20Rybak-Wolf&amp;author=C.%20Stottmeister&amp;author=P.%20Glazar&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=58&amp;pages=870-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.03.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.03.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.03.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25921068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ivanov</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Memczak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wyler</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Analysis of intron sequences reveals hallmarks of circular RNA biogenesis in animals,</div> <div class="source">Cell Rep</div> ., <div class="volume">10</div>, <div class="fpage">170</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20intron%20sequences%20reveals%20hallmarks%20of%20circular%20RNA%20biogenesis%20in%20animals%2C&amp;author=A.%20Ivanov&amp;author=S.%20Memczak&amp;author=E.%20Wyler&amp;publication_year=2015&amp;journal=Cell%20Rep&amp;volume=10&amp;pages=170-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2014.12.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2014.12.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2014.12.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25558066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Westholm</div> <div class="given-names">J. O.</div></div>, <div class="name string-name"><div class="surname">Miura</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Olson</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genome-wide analysis of drosophila circular RNAs reveals their structural and sequence properties and age-dependent neural accumulation,</div> <div class="source">Cell Rep</div> ., <div class="volume">9</div>, <div class="fpage">1966</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20drosophila%20circular%20RNAs%20reveals%20their%20structural%20and%20sequence%20properties%20and%20age-dependent%20neural%20accumulation%2C&amp;author=J.%20O.%20Westholm&amp;author=P.%20Miura&amp;author=S.%20Olson&amp;publication_year=2014&amp;journal=Cell%20Rep&amp;volume=9&amp;pages=1966-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2014.10.062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2014.10.062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2014.10.062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25544350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yee</div> <div class="given-names">M. C.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Circular RNA is expressed across the eukaryotic tree of life,</div> <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e90859</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20is%20expressed%20across%20the%20eukaryotic%20tree%20of%20life%2C&amp;author=P.%20L.%20Wang&amp;author=Y.%20Bao&amp;author=M.%20C.%20Yee&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e90859" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0090859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0090859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0090859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24609083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">C. Y.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Q. H.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Widespread noncoding circular RNAs in plants,</div> <div class="source">New Phytol</div> ., <div class="volume">208</div>, <div class="fpage">88</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20noncoding%20circular%20RNAs%20in%20plants%2C&amp;author=C.%20Y.%20Ye&amp;author=L.%20Chen&amp;author=C.%20Liu&amp;author=Q.%20H.%20Zhu&amp;author=L.%20Fan&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=208&amp;pages=88-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13585" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13585" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13585"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26204923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">J. U.</div></div>, <div class="name string-name"><div class="surname">Agarwal</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D. P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Expanded identification and characterization of mammalian circular RNAs,</div> <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">409</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expanded%20identification%20and%20characterization%20of%20mammalian%20circular%20RNAs%2C&amp;author=J.%20U.%20Guo&amp;author=V.%20Agarwal&amp;author=H.%20Guo&amp;author=D.%20P.%20Bartel&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0409-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0409-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0409-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25070500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salzman</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gawad</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Lacayo</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Circular RNAs are the predominant transcript isoform from hundreds of human genes in diverse cell types,</div> <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e30733</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20are%20the%20predominant%20transcript%20isoform%20from%20hundreds%20of%20human%20genes%20in%20diverse%20cell%20types%2C&amp;author=J.%20Salzman&amp;author=C.%20Gawad&amp;author=P.%20L.%20Wang&amp;author=N.%20Lacayo&amp;author=P.%20O.%20Brown&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e30733" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030733" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030733" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030733"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22319583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jeck</div> <div class="given-names">W. R.</div></div>, <div class="name string-name"><div class="surname">Sorrentino</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Circular RNAs are abundant, conserved, and associated with ALU repeats,</div> <div class="source">RNA</div> , <div class="volume">19</div>, <div class="fpage">141</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20are%20abundant%2C%20conserved%2C%20and%20associated%20with%20ALU%20repeats%2C&amp;author=W.%20R.%20Jeck&amp;author=J.%20A.%20Sorrentino&amp;author=K.%20Wang&amp;publication_year=2013&amp;journal=RNA&amp;volume=19&amp;pages=141-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.035667.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.035667.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.035667.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23249747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X. O.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Circular intronic long noncoding RNAs,</div> <div class="source">Mol. Cell</div> , <div class="volume">51</div>, <div class="fpage">792</div>–<div class="lpage">806</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20intronic%20long%20noncoding%20RNAs%2C&amp;author=Y.%20Zhang&amp;author=X.%20O.%20Zhang&amp;author=T.%20Chen&amp;publication_year=2013&amp;journal=Mol.%20Cell&amp;volume=51&amp;pages=792-806" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2013.08.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2013.08.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2013.08.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24035497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Exon-intron circular RNAs regulate transcription in the nucleus</div>, <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">22</div>, <div class="fpage">256</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exon-intron%20circular%20RNAs%20regulate%20transcription%20in%20the%20nucleus&amp;author=Z.%20Li&amp;author=C.%20Huang&amp;author=C.%20Bao&amp;publication_year=2015&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=22&amp;pages=256-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2959" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2959" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2959"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25664725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashwal-Fluss</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pamudurti</div> <div class="given-names">N. R.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">circRNA biogenesis competes with pre-mRNA splicing,</div> <div class="source">Mol. Cell</div> , <div class="volume">56</div>, <div class="fpage">55</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=circRNA%20biogenesis%20competes%20with%20pre-mRNA%20splicing%2C&amp;author=R.%20Ashwal-Fluss&amp;author=M.%20Meyer&amp;author=N.%20R.%20Pamudurti&amp;publication_year=2014&amp;journal=Mol.%20Cell&amp;volume=56&amp;pages=55-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2014.08.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2014.08.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2014.08.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25242144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Granados-Riveron</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Aquino-Jarquin</div> <div class="given-names">G.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The complexity of the translation ability of circRNAs,</div> <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1859</div>, <div class="fpage">1245</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complexity%20of%20the%20translation%20ability%20of%20circRNAs%2C&amp;author=J.%20T.%20Granados-Riveron&amp;author=G.%20Aquino-Jarquin&amp;publication_year=2016&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1859&amp;pages=1245-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbagrm.2016.07.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbagrm.2016.07.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbagrm.2016.07.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27449861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">circRNADb: A comprehensive database for human circular RNAs with protein-coding annotations,</div> <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">34985</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=circRNADb%3A%20A%20comprehensive%20database%20for%20human%20circular%20RNAs%20with%20protein-coding%20annotations%2C&amp;author=X.%20Chen&amp;author=P.%20Han&amp;author=T.%20Zhou&amp;author=X.%20Guo&amp;author=X.%20Song&amp;author=Y.%20Li&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=34985" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep34985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep34985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep34985"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27725737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Extensive translation of circular RNAs driven by N6-methyladenosine,</div> <div class="source">Cell Res</div> ., <div class="volume">27</div>, <div class="fpage">626</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20translation%20of%20circular%20RNAs%20driven%20by%20N6-methyladenosine%2C&amp;author=Y.%20Yang&amp;author=X.%20Fan&amp;author=M.%20Mao&amp;publication_year=2017&amp;journal=Cell%20Res&amp;volume=27&amp;pages=626-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cr.2017.31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcr.2017.31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcr.2017.31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28281539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Legnini</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Di Timoteo</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Rossi</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Circ-ZNF609 Is a Circular RNA that Can Be Translated and Functions in Myogenesis,</div> <div class="source">Mol. Cell</div> , <div class="volume">66</div>, <div class="fpage">22</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circ-ZNF609%20Is%20a%20Circular%20RNA%20that%20Can%20Be%20Translated%20and%20Functions%20in%20Myogenesis%2C&amp;author=I.%20Legnini&amp;author=G.%20Di%20Timoteo&amp;author=F.%20Rossi&amp;publication_year=2017&amp;journal=Mol.%20Cell&amp;volume=66&amp;pages=22-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2017.02.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2017.02.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2017.02.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28344082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bassel-Duby</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Olson</div> <div class="given-names">E. N.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Signaling pathways in skeletal muscle remodeling,</div> <div class="source">Annu. Rev. Biochem</div> ., <div class="volume">75</div>, <div class="fpage">19</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signaling%20pathways%20in%20skeletal%20muscle%20remodeling%2C&amp;author=R.%20Bassel-Duby&amp;author=E.%20N.%20Olson&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=75&amp;pages=19-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biochem.75.103004.142622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biochem.75.103004.142622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biochem.75.103004.142622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16756483" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schiaffino</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sandri</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Murgia</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Activity-dependent signaling pathways controlling muscle diversity and plasticity</div>, <div class="source">Physiology (Bethesda)</div> , <div class="volume">22</div>, <div class="fpage">269</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Activity-dependent%20signaling%20pathways%20controlling%20muscle%20diversity%20and%20plasticity&amp;author=S.%20Schiaffino&amp;author=M.%20Sandri&amp;author=M.%20Murgia&amp;publication_year=2007&amp;journal=Physiology%20(Bethesda)&amp;volume=22&amp;pages=269-78" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17699880" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Nie</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">MicroRNAs involved in skeletal muscle differentiation,</div> <div class="source">J. Genet. Genomics</div> , <div class="volume">40</div>, <div class="fpage">107</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAs%20involved%20in%20skeletal%20muscle%20differentiation%2C&amp;author=W.%20Luo&amp;author=Q.%20Nie&amp;author=X.%20Zhang&amp;publication_year=2013&amp;journal=J.%20Genet.%20Genomics&amp;volume=40&amp;pages=107-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jgg.2013.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jgg.2013.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jgg.2013.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23522383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nie</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">Z. L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">D. Z.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Noncoding RNAs, Emerging Regulators of Skeletal Muscle Development and Diseases,</div> <div class="source">Biomed. Res. Int</div> ., <div class="volume">2015</div>, <div class="fpage">676575</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noncoding%20RNAs%2C%20Emerging%20Regulators%20of%20Skeletal%20Muscle%20Development%20and%20Diseases%2C&amp;author=M.%20Nie&amp;author=Z.%20L.%20Deng&amp;author=J.%20Liu&amp;author=D.%20Z.%20Wang&amp;publication_year=2015&amp;journal=Biomed.%20Res.%20Int&amp;volume=2015&amp;pages=676575" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26258142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Rossum</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Verheijen</div> <div class="given-names">B. M.</div></div>, <div class="name string-name"><div class="surname">Pasterkamp</div> <div class="given-names">R. J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Circular RNAs: Novel Regulators of Neuronal Development,</div> <div class="source">Front. Mol. Neurosci</div> ., <div class="volume">9</div>, <div class="fpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%3A%20Novel%20Regulators%20of%20Neuronal%20Development%2C&amp;author=D.%20van%20Rossum&amp;author=B.%20M.%20Verheijen&amp;author=R.%20J.%20Pasterkamp&amp;publication_year=2016&amp;journal=Front.%20Mol.%20Neurosci&amp;volume=9&amp;pages=74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fnmol.2016.00074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffnmol.2016.00074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffnmol.2016.00074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27616979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maiese</div> <div class="given-names">K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Disease onset and aging in the world of circular RNAs</div>, <div class="source">J. Transl. Sci</div> ., <div class="volume">2</div>, <div class="fpage">327</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disease%20onset%20and%20aging%20in%20the%20world%20of%20circular%20RNAs&amp;author=K.%20Maiese&amp;publication_year=2016&amp;journal=J.%20Transl.%20Sci&amp;volume=2&amp;pages=327-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27642518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shang</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The emerging landscape of circular RNA in life processes</div>, <div class="source">RNA Biol</div> ., <div class="fpage">1</div>–<div class="lpage">8</div>.</div></div></div></div><div content-id="dsx039-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abdelmohsen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Panda</div> <div class="given-names">A. C.</div></div>, <div class="name string-name"><div class="surname">De</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Circular RNAs in monkey muscle: age-dependent changes,</div> <div class="source">Aging (Albany NY)</div> , <div class="volume">7</div>, <div class="fpage">903</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20in%20monkey%20muscle%3A%20age-dependent%20changes%2C&amp;author=K.%20Abdelmohsen&amp;author=A.%20C.%20Panda&amp;author=S.%20De&amp;publication_year=2015&amp;journal=Aging%20(Albany%20NY)&amp;volume=7&amp;pages=903-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.18632/aging.v7i11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.18632%2Faging.v7i11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.18632%2Faging.v7i11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26546448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fridolfsson</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">A simple and universal method for molecular sexing of non-ratite birds,</div> <div class="source">J. Avian Biol</div> ., <div class="volume">30</div>, <div class="fpage">116</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20and%20universal%20method%20for%20molecular%20sexing%20of%20non-ratite%20birds%2C&amp;author=A.%20Fridolfsson&amp;author=H.%20Ellegren&amp;publication_year=1999&amp;journal=J.%20Avian%20Biol&amp;volume=30&amp;pages=116-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/3677252" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F3677252" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F3677252"> </span></div> </div></div></div></div></div><div content-id="dsx039-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq%2C&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.%20L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B. A.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation,</div> <div class="source">Nat. Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation%2C&amp;author=C.%20Trapnell&amp;author=B.%20A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">F.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">CIRI: an efficient and unbiased algorithm for de novo circular RNA identification,</div> <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CIRI%3A%20an%20efficient%20and%20unbiased%20algorithm%20for%20de%20novo%20circular%20RNA%20identification%2C&amp;author=Y.%20Gao&amp;author=J.%20Wang&amp;author=F.%20Zhao&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0571-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0571-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0571-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25583365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Veno</div> <div class="given-names">M. T.</div></div>, <div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">T. B.</div></div>, <div class="name string-name"><div class="surname">Veno</div> <div class="given-names">S. T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Spatio-temporal regulation of circular RNA expression during porcine embryonic brain development</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">245</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatio-temporal%20regulation%20of%20circular%20RNA%20expression%20during%20porcine%20embryonic%20brain%20development&amp;author=M.%20T.%20Veno&amp;author=T.%20B.%20Hansen&amp;author=S.%20T.%20Veno&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=245" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0801-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0801-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0801-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26541409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Circular RNA of cattle casein genes are highly expressed in bovine mammary gland,</div> <div class="source">J. Dairy Sci</div> ., <div class="volume">99</div>, <div class="fpage">4750</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20of%20cattle%20casein%20genes%20are%20highly%20expressed%20in%20bovine%20mammary%20gland%2C&amp;author=C.%20Zhang&amp;author=H.%20Wu&amp;author=Y.%20Wang&amp;publication_year=2016&amp;journal=J.%20Dairy%20Sci&amp;volume=99&amp;pages=4750-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3168/jds.2015-10381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3168%2Fjds.2015-10381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3168%2Fjds.2015-10381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27040791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Halevy</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Piestun</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Allouh</div> <div class="given-names">M. Z.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Pattern of Pax7 expression during myogenesis in the posthatch chicken establishes a model for satellite cell differentiation and renewal</div>, <div class="source">Dev. Dyn</div> ., <div class="volume">231</div>, <div class="fpage">489</div>–<div class="lpage">502</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pattern%20of%20Pax7%20expression%20during%20myogenesis%20in%20the%20posthatch%20chicken%20establishes%20a%20model%20for%20satellite%20cell%20differentiation%20and%20renewal&amp;author=O.%20Halevy&amp;author=Y.%20Piestun&amp;author=M.%20Z.%20Allouh&amp;publication_year=2004&amp;journal=Dev.%20Dyn&amp;volume=231&amp;pages=489-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0177" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0177" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0177"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15390217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Halevy</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Yahav</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Rozenboim</div> <div class="given-names">I.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Enhancement of meat production by environmental manipulations in embryo and young broilers,</div> <div class="source">World. Poultry Sci. J</div> ., <div class="volume">62</div>, <div class="fpage">485</div>–<div class="lpage">97</div>.</div></div></div></div><div content-id="dsx039-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Szabo</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Morey</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Palpant</div> <div class="given-names">N. J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Statistically based splicing detection reveals neural enrichment and tissue-specific induction of circular RNA during human fetal development,</div> <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">126</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistically%20based%20splicing%20detection%20reveals%20neural%20enrichment%20and%20tissue-specific%20induction%20of%20circular%20RNA%20during%20human%20fetal%20development%2C&amp;author=L.%20Szabo&amp;author=R.%20Morey&amp;author=N.%20J.%20Palpant&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=126" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0690-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0690-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0690-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26076956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Circular RNA ITCH has inhibitory effect on ESCC by suppressing the Wnt/beta-catenin pathway,</div> <div class="source">Oncotarget</div> , <div class="volume">6</div>, <div class="fpage">6001</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20ITCH%20has%20inhibitory%20effect%20on%20ESCC%20by%20suppressing%20the%20Wnt%2Fbeta-catenin%20pathway%2C&amp;author=F.%20Li&amp;author=L.%20Zhang&amp;author=W.%20Li&amp;publication_year=2015&amp;journal=Oncotarget&amp;volume=6&amp;pages=6001-13" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25749389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">X.,</div></div></div> et al.   <div class="year">2016</div>, 
<div class="article-title">Circular RNA Related to the Chondrocyte ECM Regulates MMP13 Expression by Functioning as a MiR-136 ‘Sponge’ in Human Cartilage Degradation,</div> <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">22572</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20Related%20to%20the%20Chondrocyte%20ECM%20Regulates%20MMP13%20Expression%20by%20Functioning%20as%20a%20MiR-136%20%E2%80%98Sponge%E2%80%99%20in%20Human%20Cartilage%20Degradation%2C&amp;author=Q.%20Liu&amp;author=X.%20Zhang&amp;author=X.%2C%20Hu&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=22572" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep22572" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep22572" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep22572"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26931159" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A circular RNA protects the heart from pathological hypertrophy and heart failure by targeting miR-223,</div> <div class="source">Eur. Heart J</div> ., <div class="volume">37</div>, <div class="fpage">2602</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20circular%20RNA%20protects%20the%20heart%20from%20pathological%20hypertrophy%20and%20heart%20failure%20by%20targeting%20miR-223%2C&amp;author=K.%20Wang&amp;author=B.%20Long&amp;author=F.%20Liu&amp;publication_year=2016&amp;journal=Eur.%20Heart%20J&amp;volume=37&amp;pages=2602-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/eurheartj/ehv713" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Feurheartj%2Fehv713" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Feurheartj%2Fehv713"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26802132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">WW</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yee</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">B. B.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Foxo3 activity promoted by non-coding effects of circular RNA and Foxo3 pseudogene in the inhibition of tumor growth and angiogenesis,</div> <div class="source">Oncogene</div> , <div class="volume">35</div>, <div class="fpage">3919</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foxo3%20activity%20promoted%20by%20non-coding%20effects%20of%20circular%20RNA%20and%20Foxo3%20pseudogene%20in%20the%20inhibition%20of%20tumor%20growth%20and%20angiogenesis%2C&amp;author=W.%20Yang&amp;author=WW%20Du&amp;author=X.%20Li&amp;author=A.%20J.%20Yee&amp;author=B.%20B.%20Yang&amp;publication_year=2016&amp;journal=Oncogene&amp;volume=35&amp;pages=3919-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/onc.2015.460" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fonc.2015.460" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fonc.2015.460"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26657152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Circular RNA profiling reveals an abundant circHIPK3 that regulates cell growth by sponging multiple miRNAs,</div> <div class="source">Nat. Commun</div> ., <div class="volume">7</div>, <div class="fpage">11215</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20profiling%20reveals%20an%20abundant%20circHIPK3%20that%20regulates%20cell%20growth%20by%20sponging%20multiple%20miRNAs%2C&amp;author=Q.%20Zheng&amp;author=C.%20Bao&amp;author=W.%20Guo&amp;publication_year=2016&amp;journal=Nat.%20Commun&amp;volume=7&amp;pages=11215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms11215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms11215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms11215"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27050392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Rooij</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Olson</div> <div class="given-names">E. N.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">MicroRNAs flex their muscles,</div> <div class="source">Trends Genet</div> ., <div class="volume">24</div>, <div class="fpage">159</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAs%20flex%20their%20muscles%2C&amp;author=E.%20van%20Rooij&amp;author=N.%20Liu&amp;author=E.%20N.%20Olson&amp;publication_year=2008&amp;journal=Trends%20Genet&amp;volume=24&amp;pages=159-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2008.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2008.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2008.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18325627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Townley-Tilson</div> <div class="given-names">W. H.</div></div>, <div class="name string-name"><div class="surname">Callis</div> <div class="given-names">T. E.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">D.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">MicroRNAs 1, 133, and 206: critical factors of skeletal and cardiac muscle development, function, and disease,</div> <div class="source">Int. J. Biochem. Cell Biol</div> ., <div class="volume">42</div>, <div class="fpage">1252</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAs%201%2C%20133%2C%20and%20206%3A%20critical%20factors%20of%20skeletal%20and%20cardiac%20muscle%20development%2C%20function%2C%20and%20disease%2C&amp;author=W.%20H.%20Townley-Tilson&amp;author=T.%20E.%20Callis&amp;author=D.%20Wang&amp;publication_year=2010&amp;journal=Int.%20J.%20Biochem.%20Cell%20Biol&amp;volume=42&amp;pages=1252-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biocel.2009.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biocel.2009.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biocel.2009.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20619221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zuo</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">M. Q.</div></div>, <div class="name string-name"><div class="surname">Malhotra</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mayeda</div> <div class="given-names">A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Characterization of RNase R-digested cellular RNA source that consists of lariat and circular RNAs from pre-mRNA splicing</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">e63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20RNase%20R-digested%20cellular%20RNA%20source%20that%20consists%20of%20lariat%20and%20circular%20RNAs%20from%20pre-mRNA%20splicing&amp;author=H.%20Suzuki&amp;author=Y.%20Zuo&amp;author=J.%20Wang&amp;author=M.%20Q.%20Zhang&amp;author=A.%20Malhotra&amp;author=A.%20Mayeda&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=e63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16682442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">A novel regulatory network among LncRpa, CircRar1, MiR-671 and apoptotic genes promotes lead-induced neuronal cell apoptosis,</div> <div class="source">Arch. Toxicol</div> ., <div class="volume">91</div>, <div class="fpage">1671</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20regulatory%20network%20among%20LncRpa%2C%20CircRar1%2C%20MiR-671%20and%20apoptotic%20genes%20promotes%20lead-induced%20neuronal%20cell%20apoptosis%2C&amp;author=A.%20Nan&amp;author=L.%20Chen&amp;author=N.%20Zhang&amp;publication_year=2017&amp;journal=Arch.%20Toxicol&amp;volume=91&amp;pages=1671-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00204-016-1837-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00204-016-1837-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00204-016-1837-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27604105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liang</div> <div class="given-names">H. F.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X. Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B. G.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Circular RNA circ-ABCB10 promotes breast cancer proliferation and progression through sponging miR-1271</div>, <div class="source">Am J Cancer Res</div> ., <div class="volume">7</div>, <div class="fpage">1566</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20circ-ABCB10%20promotes%20breast%20cancer%20proliferation%20and%20progression%20through%20sponging%20miR-1271&amp;author=H.%20F.%20Liang&amp;author=X.%20Z.%20Zhang&amp;author=B.%20G.%20Liu&amp;publication_year=2017&amp;journal=Am%20J%20Cancer%20Res&amp;volume=7&amp;pages=1566-76" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28744405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jin</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Circular RNA hsa-circ-0016347 promotes proliferation, invasion and metastasis of osteosarcoma cells</div>, <div class="source">Oncotarget</div> ., <div class="volume">8</div>, <div class="fpage">25571</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20hsa-circ-0016347%20promotes%20proliferation%2C%20invasion%20and%20metastasis%20of%20osteosarcoma%20cells&amp;author=H.%20Jin&amp;author=X.%20Jin&amp;author=H.%20Zhang&amp;author=W.%20Wang&amp;publication_year=2017&amp;journal=Oncotarget&amp;volume=8&amp;pages=25571-81" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28424426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elliman</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Howley</div> <div class="given-names">B. V.</div></div>, <div class="name string-name"><div class="surname">Mehta</div> <div class="given-names">D. S.</div></div>, <div class="name string-name"><div class="surname">Fearnhead</div> <div class="given-names">H. O.</div></div>, <div class="name string-name"><div class="surname">Kemp</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Barkley</div> <div class="given-names">L. R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Selective repression of the oncogene cyclin D1 by the tumor suppressor miR-206 in cancers</div>, <div class="source">Oncogenesis</div> , <div class="volume">3</div>, <div class="fpage">e113</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20repression%20of%20the%20oncogene%20cyclin%20D1%20by%20the%20tumor%20suppressor%20miR-206%20in%20cancers&amp;author=S.%20J.%20Elliman&amp;author=B.%20V.%20Howley&amp;author=D.%20S.%20Mehta&amp;author=H.%20O.%20Fearnhead&amp;author=D.%20M.%20Kemp&amp;author=L.%20R.%20Barkley&amp;publication_year=2014&amp;journal=Oncogenesis&amp;volume=3&amp;pages=e113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/oncsis.2014.26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Foncsis.2014.26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Foncsis.2014.26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25111862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">miR-206 inhibits gastric cancer proliferation in part by repressing cyclinD2,</div> <div class="source">Cancer Lett</div> ., <div class="volume">332</div>, <div class="fpage">94</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=miR-206%20inhibits%20gastric%20cancer%20proliferation%20in%20part%20by%20repressing%20cyclinD2%2C&amp;author=L.%20Zhang&amp;author=X.%20Liu&amp;author=H.%20Jin&amp;publication_year=2013&amp;journal=Cancer%20Lett&amp;volume=332&amp;pages=94-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.canlet.2013.01.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.canlet.2013.01.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.canlet.2013.01.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23348698" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Winbanks</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Beyer</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">TGF-beta regulates miR-206 and miR-29 to control myogenic differentiation through regulation of HDAC4,</div> <div class="source">J. Biol. Chem</div> ., <div class="volume">286</div>, <div class="fpage">13805</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TGF-beta%20regulates%20miR-206%20and%20miR-29%20to%20control%20myogenic%20differentiation%20through%20regulation%20of%20HDAC4%2C&amp;author=C.%20E.%20Winbanks&amp;author=B.%20Wang&amp;author=C.%20Beyer&amp;publication_year=2011&amp;journal=J.%20Biol.%20Chem&amp;volume=286&amp;pages=13805-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M110.192625" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M110.192625" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M110.192625"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21324893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">You</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Vlatkovic</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Babic</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Neural circular RNAs are derived from synaptic genes and regulated by development and plasticity,</div> <div class="source">Nat. Neurosci</div> ., <div class="volume">18</div>, <div class="fpage">603</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Neural%20circular%20RNAs%20are%20derived%20from%20synaptic%20genes%20and%20regulated%20by%20development%20and%20plasticity%2C&amp;author=X.%20You&amp;author=I.%20Vlatkovic&amp;author=A.%20Babic&amp;publication_year=2015&amp;journal=Nat.%20Neurosci&amp;volume=18&amp;pages=603-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nn.3975" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnn.3975" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnn.3975"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25714049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx039-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx039-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Militello</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Weirick</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">John</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Doring</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Dimmeler</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Uchida</div> <div class="given-names">S.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Screening and validation of lncRNAs and circRNAs as miRNA sponges</div>, <div class="source">Brief. Bioinform</div> ., <div class="volume">18</div>, <div class="fpage">780</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Screening%20and%20validation%20of%20lncRNAs%20and%20circRNAs%20as%20miRNA%20sponges&amp;author=G.%20Militello&amp;author=T.%20Weirick&amp;author=D.%20John&amp;author=C.%20Doring&amp;author=S.%20Dimmeler&amp;author=S.%20Uchida&amp;publication_year=2017&amp;journal=Brief.%20Bioinform&amp;volume=18&amp;pages=780-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27373735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4265514&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s1_dsx039.png?Expires=2147483647&amp;Signature=3lFoad0PPPRn9ld~g90ig~qJDhoC41zi5LDlFGtKGa6EmJFKECUbcvVnUa1bUnDg6FxzDGkSuZbBDLdEV6E6aD~iYR314aZ3rZWeu5ItHc-7R7gQpQ8h5sODXHMqx2iqeApcyPRqcHVB1tUdl0F0rpI0LkEonNl52hFS32T9kLToT3H13g4Eu7NIvNjq7QQwONfmIFchXFt29kAi2D9eLXOYDd-4AH6wclizxNNDT-saLpA0CDR7DXbscbkdyHmW1i0aLv0Nf6Mig4nEbUjuQht0npYXKV4JwDUN5KJ2vtSEf9KhyRTBD5idodSCJbsfQqwK-rO7JpkBNpnorA8rwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s2_dsx039.png?Expires=2147483647&amp;Signature=W9NmDcc-nlC3V6hCDnqd5Y-qXt7ypIxXMRbzaNKKs4tGbW3SwrHG-FBO1Rhq~0Kgu3JxTyRPiIevlXt7-c0qSaWzZW0KjrL58Nwjv6YBtbbCWYVgVtnZNccP0mJc9Rubn1pzrrNktLUYtfADMx4ppvoJRsHjqkxOoFsPPKa6wjz9EVuY3gZvlfzSs07AIXhb~WWZZUqFvIGqg4IkUC-w1msboUODtS9adSVHr~u3J4L6kuLxRjvLtskghzTulMR6c99aT1HrJsyxIAvKGMyTxMPU7H-k~Nalvb9iX3~c64-MFWkUEC4MRtGwwD-3PAtaLb5ZHM~qRGjzO5kE7RdV3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s3_dsx039.png?Expires=2147483647&amp;Signature=LyyaEcoDrCTep5xHiOQDhfzAYQq62IYDuPWQpg6wZIE3APDy~OnO3HlfAaiHmMeqb4ybK5W5n2OJmuDvPXih7CrdbGTkDy2w313Fk77KhB9hnw~xGX79vcqjdgXYF78ymwBxcVVxmWKK406Y5~pwtKOcFmDgoBr4BF4G2AQ-Ky5YVzjpwhV-DBP6rE8doG-vBPm2AgDhZ7DaTb2WArrDNymjgUTekxNa9ZupPb1YxJbX58x9WzuQkubfQQO7RQbzwbe35JzXsXXia59t1uzxmpABjlVUkiGhdx3nHfAJVupbzVe-kfrqd35D3uk7tSaoA0pSnJnlzo6MgDiRd24gQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S3</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s4_dsx039.png?Expires=2147483647&amp;Signature=MW3gW9cpPMOGbjU6WcD1pybiz119iRF~AMZA8sAK87YgxKpCZlQvNOCZRgQMsOniIgGIhdcvO0mhR7SocBMHwfaI~CtMr-i6XuOKQagoEyjnHl0sSqSDGa9WjX99F~C~EFt7fgTJrSXcMyXr8h20eoN8z-BPjm6nUNY2Ld~8~aeMEzKVHPaqesGwfNAtNBB18A6K28zThNFE1BsSbwoRo2jMMWC3rt6~rXkJksG33AF0F-htvnKnchVtnEFYF4c8KRHgG-QIj96Z7dZZopYXQyI3kvYzDiQhct~4mljPaRTNZDBqppCtniZq3mPGjbvXbKmlRgawLmy0tK7qTGCo8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S4</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s5_dsx039.png?Expires=2147483647&amp;Signature=RjDwDx-FzHMB7ci~RaFT3gZfGuUGIWAF0rDdbS4vPZmgIj1XBcB5ajsYmpBLd5uhVKO92X4n-KmZSJudbsiK04BfRoYIhb71kgbM7t5H1nJv0g2P5HgNfbhHjtBWwk2TmlgI9lQRqDexLJc4GgdmSoyiDcT3yo1Hey6LnL8W-yfoLYegVoAbQ7iu8LfcQQaIggLVBNIw~PO3HtxRgmJf0JATK~VYvE9qXFQTKBZcxm~WMfH3hlABids9abdnsnLbLYetreb-w342ySVosEUu9VvXeei7EJZdxjTUKNv4ubaENCxz2iBgvSp65GYUSaIwkbSbpG6eUkJLe6l5fnnW3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S5</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s6_dsx039.png?Expires=2147483647&amp;Signature=q289IXksnNvNBIa3CcLuBrYqQ5RV5IxHOjIZ3KaAHiI8VNth81JIG937xtcG9EoZEXP1zTQna~DhhffbTaeFbifcMxve4YgiqchfWHCkFAIxn1ulxc9irzYhj3KHU5FPdvxK03h29Qa1pDmhEmKmp3-Yz2rm-LNxGyoNxpY6mfo48UozgzqcSeVj6PcJ9iARAbTP53JDc7oVN-mOZhNXVpzMVoe17PIQkwvGtN0szL0lB~VDlAHdWU8TLDGdSQI38PzeLrIdKUdBUEAzEu416tpBliO2SNi3r7f0PE9Cd76JHGzL8qnn5ql-CursOjPahXyNzGA2Xau8ZQEp4SxMfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S6</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s7_dsx039.png?Expires=2147483647&amp;Signature=PUQWtDy-a0RpVPHIGny9yg5MEdfWSrpY9jQ6S13DvKV-mAurBvIXFzYbGLxlxgmnRh4Ud58frQ5v3SQ34CILrc~ah81Y1FhahIT38ZBmxphhZrlUGPlMh0DDR6sJe2UvMKiOrW~GXMPe1nqZ3GadT7alJ0RtfYSEBE9JwHyIp8IJttoTzEUpFEokIcjTluzC5VpA-abNrScdU8v5Suge7exxM6txG7yAdHLSaXn8U0g8JiSfp~jwPQ0c1gGm27YRKr2CFeHneELRcQGSRS9ooFkSTU4~16AUqPoAdfVy92fSppREkdKlVml8dmF6GCEg1r-dmiu9eUN0zVOb7jWecQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S7</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/figure_s8_dsx039.png?Expires=2147483647&amp;Signature=QKSB30C9rhasN8PQeJu3bIop4~WvFNuMrte1atjCWFq8AvlfGoCUP3y2pKgYCuehEwaBRFcV-o79f-FWdHfGV-ucaO-v1gVKUDkPHy6ME7Fjoosk~NTCeJxkiuW3-~7kW5XHB6MVMv68OwXIUnv5lY34CPk-oj7TBZoJQMj-O4dZTAgc0ZHhCXlKVlF7D2TlrajDZ2qq5cz7mQ5WjpN09P7won85vSITlKn2rnjx07dEKMGrdOXl5aJNlipsMKl0~z-M11DvsLigKcYyKik7fhfd49CKD8bk3u0pW94irCUW0jhLpVebiKMUCX7X-g4DobcJYJAfVYoJg0tfKq0PkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S8</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx039/3/tables_s1-s10_dsx039.xls?Expires=2147483647&amp;Signature=hkhXG1EjOnyf4l-zoZqpm0sLkPiL1yPs~0uxaKZpwDz~a5gZ~vCO79BPxIwocCUg7L6WpC4F8LdtHK3wA6UR1yxoYv9ZiVVQCRxsfdpEh5R8uGSFqHpX8mvtejxRr5FLX1Pz3QI7n4pYm3prUhupoB7CJVtiw5h8Y~ntt0QYgEWxtKm9GeRV82CgWpgr4cCrK1Vx29wTAmsYoZ7ivG2kYeS-8dlCv3VRZin600u60fFzSvgN1DoDD~HsGiLLPKE48uw8gzZZM29AqDwBa4xEiHAlfxtCnqeQlQ~IBtxNfZn5CHhoo6jZkWIMlem28C2OZRZPSa5q05PJkTMiXlibwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Tables S1-S10</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,305</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx039"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4265514"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f71%2f4265514">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000425954100007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F71%2F4265514" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4265514'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx039");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f71%2f4265514">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

