

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/1/67/485988","@type":"ScholarlyArticle","name":"Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of  Synechocystis  sp. PCC 6803","datePublished":"2011-12-22","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2012-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr042","keywords":["genome sequence; massive parallel sequencer","substrain","<italic>Synechocystis</italic> sp. PCC 6803"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/1/67/485988","author":[{"name":"Kanesaki, Yu","affiliation":"1  Genome Research Center, NODAI Research Institute ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Shiwa, Yuh","affiliation":"1  Genome Research Center, NODAI Research Institute ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Tajima, Naoyuki","affiliation":"2  Department of Life Sciences ,  Graduate School of Arts and Sciences, The University of Tokyo ,  3-8-1 Komaba, Meguro-ku, Tokyo 153-8902 ,  Japan","@type":"Person"},{"name":"Suzuki, Marie","affiliation":"3  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Watanabe, Satoru","affiliation":"3  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Sato, Naoki","affiliation":"2  Department of Life Sciences ,  Graduate School of Arts and Sciences, The University of Tokyo ,  3-8-1 Komaba, Meguro-ku, Tokyo 153-8902 ,  Japan","@type":"Person"},{"name":"Ikeuchi, Masahiko","affiliation":"2  Department of Life Sciences ,  Graduate School of Arts and Sciences, The University of Tokyo ,  3-8-1 Komaba, Meguro-ku, Tokyo 153-8902 ,  Japan","@type":"Person"},{"name":"Yoshikawa, Hirofumi","affiliation":"1  Genome Research Center, NODAI Research Institute ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan    3  Department of Bioscience ,  Tokyo University of Agriculture ,  1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502 ,  Japan","@type":"Person"}],"description":"Abstract.  The cyanobacterium, Synechocystis sp. PCC 6803, was the first photosynthetic organism whose genome sequence was determined in 1996 (Kazusa strain). I","pageStart":"67","pageEnd":"79","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of  Synechocystis  sp. PCC 6803","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of  Synechocystis  sp. PCC 6803" />
    <meta property="og:description" content="Abstract.  The cyanobacterium, Synechocystis sp. PCC 6803, was the first photosynthetic organism whose genome sequence was determined in 1996 (Kazusa strain). I" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/1/67/485988" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-12-22" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kanesaki, Yu" /><meta name="citation_author_institution" content="Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Shiwa, Yuh" /><meta name="citation_author_institution" content="Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Tajima, Naoyuki" /><meta name="citation_author_institution" content="Department of Life Sciences, Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba, Meguro-ku, Tokyo 153-8902, Japan" /><meta name="citation_author" content="Suzuki, Marie" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Watanabe, Satoru" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Sato, Naoki" /><meta name="citation_author_institution" content="Department of Life Sciences, Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba, Meguro-ku, Tokyo 153-8902, Japan" /><meta name="citation_author" content="Ikeuchi, Masahiko" /><meta name="citation_author_institution" content="Department of Life Sciences, Graduate School of Arts and Sciences, The University of Tokyo, 3-8-1 Komaba, Meguro-ku, Tokyo 153-8902, Japan" /><meta name="citation_author" content="Yoshikawa, Hirofumi" /><meta name="citation_author_institution" content="Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, 1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502, Japan" /><meta name="citation_title" content="Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of Synechocystis sp. PCC 6803" /><meta name="citation_firstpage" content="67" /><meta name="citation_lastpage" content="79" /><meta name="citation_doi" content="10.1093/dnares/dsr042" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2012/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetic systems in cyanobacteria; Methods Enzymol.;  citation_year=1991;  citation_volume=204;  citation_pages=418-30; " /><meta name="citation_reference" content="citation_title=DNA transformation; Methods Enzymol.;  citation_year=1988;  citation_volume=167;  citation_pages=703-12; " /><meta name="citation_reference" content="citation_title=Construction of specific mutations in photosystem II photosynthetic reaction center by genetic engineering methods in Synechocystis 6803; Methods Enzymol.;  citation_year=1988;  citation_volume=167;  citation_pages=766-78; " /><meta name="citation_reference" content="citation_title=Genetic assignments, strain histories and properties of pure cultures of cyanobacteria; J. Gen. Microbiol.;  citation_year=1979;  citation_volume=111;  citation_pages=1-61; " /><meta name="citation_reference" content="citation_title=Sequence analysis of the genome of the unicellular cyanobacterium Synechocystis sp. strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions; DNA Res.;  citation_year=1996;  citation_volume=3;  citation_pages=109-36; " /><meta name="citation_reference" content="citation_title=Cyanobacterial postgenomic research and systems biology; Trends Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=504-11; " /><meta name="citation_reference" content="citation_title=Purification and properties of unicellular blue-green algae (order Chroococcales); Bacteriol. Rev.;  citation_year=1971;  citation_volume=35;  citation_pages=171-205; " /><meta name="citation_reference" content="citation_title=Synechocystis sp. PCC 6803—a useful tool in the study of the genetics of cyanobacteria; Photosynth. Res.;  citation_year=2001;  citation_volume=70;  citation_pages=73-83; " /><meta name="citation_reference" content="citation_title=Catalogue of strains; citation_publisher=Institut Pasteur, Paris; Pasteur Culture Collection of Cyanobacterial Strains in Axenic Culture;  citation_year=1992;  citation_volume=vol. I; " /><meta name="citation_reference" content="citation_title=Expression of photosynthesis genes in the cyanobacterium Synechocystis sp. PCC 6803: psaA-psaB and psbA transcripts accumulate in dark-grown cells; Plant Mol. Biol.;  citation_year=1991;  citation_volume=17;  citation_pages=959-71; " /><meta name="citation_reference" content="citation_title=Identification of a novel gene (slr2031) involved in high-light resistance in the cyanobacterium Synechocystis sp. PCC 6803; citation_publisher=Kluwer Academic Publishers, Dordrecht, The Netherlands; Photosynthesis: Mechanism and Effects;  citation_year=1998;  citation_pages=2901-5; " /><meta name="citation_reference" content="citation_title=A eukaryotic-type protein kinase, SpkA, is required for normal motility of the unicellular cyanobacterium Synechocystis sp. strain PCC 6803; J. Bacteriol.;  citation_year=2001;  citation_volume=183;  citation_pages=1505-10; " /><meta name="citation_reference" content="citation_title=A possible role of 154-base pair nucleotides located upstream of ORF440 on CO2 transport of Synechocystis PCC6803; citation_text=Kluwer Academic Publishers: Dordrecht, The Netherlands,; Photosynthesis: From Light to Biosphere;  citation_year=1995;  citation_volume=3;  citation_pages=481-4; " /><meta name="citation_reference" content="citation_title=Experimental analysis of recently transposed insertion sequences in the cyanobacterium Synechocystis sp. PCC 6803; DNA Res.;  citation_year=1999;  citation_volume=6;  citation_pages=265-73; " /><meta name="citation_reference" content="citation_title=Mutational analysis of genes involved in pilus structure, motility and transformation competency in the unicellular motile cyanobacterium Synechocystis sp. PCC 6803; Plant Cell. Physiol.;  citation_year=2001;  citation_volume=42;  citation_pages=63-73; " /><meta name="citation_reference" content="citation_title=Mutation in a novel gene required for photomixotrophic growth leads to enhanced photoautotrophic growth of Synechocystis sp. PCC 6803; Photosynth. Res.;  citation_year=1997;  citation_volume=53;  citation_pages=129-39; " /><meta name="citation_reference" content="citation_title=A novel gene, pmgA, specifically regulates photosystem stoichiometry in the cyanobacterium Synechocystis species PCC 6803 in response to high light; Plant Physiol.;  citation_year=1998;  citation_volume=117;  citation_pages=1205-16; " /><meta name="citation_reference" content="citation_title=High-precision, whole-genome sequencing of laboratory strains facilitates genetic studies; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000139" /><meta name="citation_reference" content="citation_title=Novel putative photoreceptor and regulatory genes Required for the positive phototactic movement of the unicellular motile cyanobacterium Synechocystis sp. PCC 6803; Plant Cell Physiol.;  citation_year=2000;  citation_volume=41;  citation_pages=1299-304; " /><meta name="citation_reference" content="citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1851-8; " /><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=589-95; " /><meta name="citation_reference" content="citation_title=1000 Genome Project Data Processing Subgroup, The sequence alignment/map format and SAMtools; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=BreakDancer: An algorithm for high-resolution mapping of genomic structural variation; Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=677-81; " /><meta name="citation_reference" content="citation_title=Velvet: Algorithms for de novo short read assembly using de Bruijn graphs; Genome Res.;  citation_year=2009;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Open source MUMmer 3.0 is described in ‘Versatile and open software for comparing large genomes; Genome Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=R12" /><meta name="citation_reference" content="citation_title=CyanoClust: comparative genome resources of cyanobacteria and plastids; citation_text=bap025; Database (Oxford);  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Genomic structure of the cyanobacterium Synechocystis sp. PCC 6803 strain GT-S; DNA Res.;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=PHYLIP—Phylogeny Inference Package (Version 3.2); Cladistics;  citation_year=1989;  citation_volume=5;  citation_pages=164-6; " /><meta name="citation_reference" content="citation_title=Histidine kinases play important roles in the perception and signal transduction of hydrogen peroxide in the cyanobacterium Synechocystis sp. PCC 6803; Plant J.;  citation_year=2007;  citation_volume=49;  citation_pages=313-24; " /><meta name="citation_reference" content="citation_title=Five histidine kinases perceive osmotic stress and regulate distinct sets of genes in Synechocystis; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=53078-86; " /><meta name="citation_reference" content="citation_title=Cold-regulated genes under control of the cold sensor Hik33 in Synechocystis; Mol. Microbiol.;  citation_year=2001;  citation_volume=40;  citation_pages=235-44; " /><meta name="citation_reference" content="citation_title=Stock culture heterogeneity rather than new mutational variation complicates short-term cell physiology studies of Escherichia coli K-12 MG1655 in continuous culture; Microbiology;  citation_year=2011;  citation_volume=157;  citation_pages=2604-10; " /><meta name="citation_reference" content="citation_title=Mutagenesis by random cloning of an Escherichia coli kanamycin resistance gene into the genome of the cyanobacterium Synechocystis PCC 6803: Selection of mutants defective in photosynthesis; Mol. Gen. Genet.;  citation_year=1989;  citation_volume=216;  citation_pages=51-9; " /><meta name="citation_reference" content="citation_title=Type IV pilus biogenesis and motility in the cyanobacterium Synechocystis sp. PCC6803; Mol. Microbiol.;  citation_year=2000;  citation_volume=37;  citation_pages=941-51; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/1/67/1313838/dsr042.pdf" /><meta name="description" content="Abstract.  The cyanobacterium, Synechocystis sp. PCC 6803, was the first photosynthetic organism whose genome sequence was determined in 1996 (Kazusa strain). I" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/1/67/485988" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/1/67/485988" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/1/67/485988");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr042';
    </script>
<title>
	Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of Synechocystis sp. PCC 6803 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./485988" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeNbPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5lc2FraSwgWXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHZW5vbWUgUmVzZWFyY2ggQ2VudGVyLCBOT0RBSSBSZXNlYXJjaCBJbnN0aXR1dGUsIFRva3lvIFVuaXZlcnNpdHkgb2YgQWdyaWN1bHR1cmUsIDEtMS0xIFNha3VyYWdhb2thLCBTZXRhZ2F5YS1rdSwgVG9reW8gMTU2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpd2EsIFl1aCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlbm9tZSBSZXNlYXJjaCBDZW50ZXIsIE5PREFJIFJlc2VhcmNoIEluc3RpdHV0ZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgMS0xLTEgU2FrdXJhZ2Fva2EsIFNldGFnYXlhLWt1LCBUb2t5byAxNTYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWppbWEsIE5hb3l1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIExpZmUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBBcnRzIGFuZCBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDMtOC0xIEtvbWFiYSwgTWVndXJvLWt1LCBUb2t5byAxNTMtODkwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIE1hcmllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlLCBUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlLCAxLTEtMSBTYWt1cmFnYW9rYSwgU2V0YWdheWEta3UsIFRva3lvIDE1Ni04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhdGFuYWJlLCBTYXRvcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2UsIFRva3lvIFVuaXZlcnNpdHkgb2YgQWdyaWN1bHR1cmUsIDEtMS0xIFNha3VyYWdhb2thLCBTZXRhZ2F5YS1rdSwgVG9reW8gMTU2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0bywgTmFva2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIExpZmUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBBcnRzIGFuZCBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDMtOC0xIEtvbWFiYSwgTWVndXJvLWt1LCBUb2t5byAxNTMtODkwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJa2V1Y2hpLCBNYXNhaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEFydHMgYW5kIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgMy04LTEgS29tYWJhLCBNZWd1cm8ta3UsIFRva3lvIDE1My04OTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Illvc2hpa2F3YSwgSGlyb2Z1bWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHZW5vbWUgUmVzZWFyY2ggQ2VudGVyLCBOT0RBSSBSZXNlYXJjaCBJbnN0aXR1dGUsIFRva3lvIFVuaXZlcnNpdHkgb2YgQWdyaWN1bHR1cmUsIDEtMS0xIFNha3VyYWdhb2thLCBTZXRhZ2F5YS1rdSwgVG9reW8gMTU2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlLCBUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlLCAxLTEtMSBTYWt1cmFnYW9rYSwgU2V0YWdheWEta3UsIFRva3lvIDE1Ni04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSWRlbnRpZmljYXRpb24gb2YgU3Vic3RyYWluLVNwZWNpZmljIE11dGF0aW9ucyBieSBNYXNzaXZlbHkgUGFyYWxsZWwgV2hvbGUtR2Vub21lIFJlc2VxdWVuY2luZyBvZiBTeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjA0MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEyLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBzeXN0ZW1zIGluIGN5YW5vYmFjdGVyaWE7IE1ldGhvZHMgRW56eW1vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjA0OyAgY2l0YXRpb25fcGFnZXM9NDE4LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgdHJhbnNmb3JtYXRpb247IE1ldGhvZHMgRW56eW1vbC47ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9NzAzLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2Ygc3BlY2lmaWMgbXV0YXRpb25zIGluIHBob3Rvc3lzdGVtIElJIHBob3Rvc3ludGhldGljIHJlYWN0aW9uIGNlbnRlciBieSBnZW5ldGljIGVuZ2luZWVyaW5nIG1ldGhvZHMgaW4gU3luZWNob2N5c3RpcyA2ODAzOyBNZXRob2RzIEVuenltb2wuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTE2NzsgIGNpdGF0aW9uX3BhZ2VzPTc2Ni03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhc3NpZ25tZW50cywgc3RyYWluIGhpc3RvcmllcyBhbmQgcHJvcGVydGllcyBvZiBwdXJlIGN1bHR1cmVzIG9mIGN5YW5vYmFjdGVyaWE7IEouIEdlbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk3OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0xLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBhbmFseXNpcyBvZiB0aGUgZ2Vub21lIG9mIHRoZSB1bmljZWxsdWxhciBjeWFub2JhY3Rlcml1bSBTeW5lY2hvY3lzdGlzIHNwLiBzdHJhaW4gUENDNjgwMy4gSUkuIFNlcXVlbmNlIGRldGVybWluYXRpb24gb2YgdGhlIGVudGlyZSBnZW5vbWUgYW5kIGFzc2lnbm1lbnQgb2YgcG90ZW50aWFsIHByb3RlaW4tY29kaW5nIHJlZ2lvbnM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMDktMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5YW5vYmFjdGVyaWFsIHBvc3RnZW5vbWljIHJlc2VhcmNoIGFuZCBzeXN0ZW1zIGJpb2xvZ3k7IFRyZW5kcyBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTUwNC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHVyaWZpY2F0aW9uIGFuZCBwcm9wZXJ0aWVzIG9mIHVuaWNlbGx1bGFyIGJsdWUtZ3JlZW4gYWxnYWUgKG9yZGVyIENocm9vY29jY2FsZXMpOyBCYWN0ZXJpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5NzE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0xNzEtMjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwM+KAlGEgdXNlZnVsIHRvb2wgaW4gdGhlIHN0dWR5IG9mIHRoZSBnZW5ldGljcyBvZiBjeWFub2JhY3RlcmlhOyBQaG90b3N5bnRoLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9NzMtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhdGFsb2d1ZSBvZiBzdHJhaW5zOyBjaXRhdGlvbl9wdWJsaXNoZXI9SW5zdGl0dXQgUGFzdGV1ciwgUGFyaXM7IFBhc3RldXIgQ3VsdHVyZSBDb2xsZWN0aW9uIG9mIEN5YW5vYmFjdGVyaWFsIFN0cmFpbnMgaW4gQXhlbmljIEN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9dm9sLiBJOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIG9mIHBob3Rvc3ludGhlc2lzIGdlbmVzIGluIHRoZSBjeWFub2JhY3Rlcml1bSBTeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwMzogcHNhQS1wc2FCIGFuZCBwc2JBIHRyYW5zY3JpcHRzIGFjY3VtdWxhdGUgaW4gZGFyay1ncm93biBjZWxsczsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTk1OS03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBub3ZlbCBnZW5lIChzbHIyMDMxKSBpbnZvbHZlZCBpbiBoaWdoLWxpZ2h0IHJlc2lzdGFuY2UgaW4gdGhlIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzOyBjaXRhdGlvbl9wdWJsaXNoZXI9S2x1d2VyIEFjYWRlbWljIFB1Ymxpc2hlcnMsIERvcmRyZWNodCwgVGhlIE5ldGhlcmxhbmRzOyBQaG90b3N5bnRoZXNpczogTWVjaGFuaXNtIGFuZCBFZmZlY3RzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fcGFnZXM9MjkwMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGV1a2FyeW90aWMtdHlwZSBwcm90ZWluIGtpbmFzZSwgU3BrQSwgaXMgcmVxdWlyZWQgZm9yIG5vcm1hbCBtb3RpbGl0eSBvZiB0aGUgdW5pY2VsbHVsYXIgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBzcC4gc3RyYWluIFBDQyA2ODAzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE4MzsgIGNpdGF0aW9uX3BhZ2VzPTE1MDUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcG9zc2libGUgcm9sZSBvZiAxNTQtYmFzZSBwYWlyIG51Y2xlb3RpZGVzIGxvY2F0ZWQgdXBzdHJlYW0gb2YgT1JGNDQwIG9uIENPMiB0cmFuc3BvcnQgb2YgU3luZWNob2N5c3RpcyBQQ0M2ODAzOyBjaXRhdGlvbl90ZXh0PUtsdXdlciBBY2FkZW1pYyBQdWJsaXNoZXJzOiBEb3JkcmVjaHQsIFRoZSBOZXRoZXJsYW5kcyw7IFBob3Rvc3ludGhlc2lzOiBGcm9tIExpZ2h0IHRvIEJpb3NwaGVyZTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NDgxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBhbmFseXNpcyBvZiByZWNlbnRseSB0cmFuc3Bvc2VkIGluc2VydGlvbiBzZXF1ZW5jZXMgaW4gdGhlIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjY1LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbmFsIGFuYWx5c2lzIG9mIGdlbmVzIGludm9sdmVkIGluIHBpbHVzIHN0cnVjdHVyZSwgbW90aWxpdHkgYW5kIHRyYW5zZm9ybWF0aW9uIGNvbXBldGVuY3kgaW4gdGhlIHVuaWNlbGx1bGFyIG1vdGlsZSBjeWFub2JhY3Rlcml1bSBTeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwMzsgUGxhbnQgQ2VsbC4gUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz02My03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb24gaW4gYSBub3ZlbCBnZW5lIHJlcXVpcmVkIGZvciBwaG90b21peG90cm9waGljIGdyb3d0aCBsZWFkcyB0byBlbmhhbmNlZCBwaG90b2F1dG90cm9waGljIGdyb3d0aCBvZiBTeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwMzsgUGhvdG9zeW50aC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTEyOS0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBnZW5lLCBwbWdBLCBzcGVjaWZpY2FsbHkgcmVndWxhdGVzIHBob3Rvc3lzdGVtIHN0b2ljaGlvbWV0cnkgaW4gdGhlIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3BlY2llcyBQQ0MgNjgwMyBpbiByZXNwb25zZSB0byBoaWdoIGxpZ2h0OyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz0xMjA1LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXByZWNpc2lvbiwgd2hvbGUtZ2Vub21lIHNlcXVlbmNpbmcgb2YgbGFib3JhdG9yeSBzdHJhaW5zIGZhY2lsaXRhdGVzIGdlbmV0aWMgc3R1ZGllczsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwMTM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgcHV0YXRpdmUgcGhvdG9yZWNlcHRvciBhbmQgcmVndWxhdG9yeSBnZW5lcyBSZXF1aXJlZCBmb3IgdGhlIHBvc2l0aXZlIHBob3RvdGFjdGljIG1vdmVtZW50IG9mIHRoZSB1bmljZWxsdWxhciBtb3RpbGUgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDM7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz0xMjk5LTMwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBzaG9ydCBETkEgc2VxdWVuY2luZyByZWFkcyBhbmQgY2FsbGluZyB2YXJpYW50cyB1c2luZyBtYXBwaW5nIHF1YWxpdHkgc2NvcmVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE4NTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgbG9uZy1yZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT0xMDAwIEdlbm9tZSBQcm9qZWN0IERhdGEgUHJvY2Vzc2luZyBTdWJncm91cCwgVGhlIHNlcXVlbmNlIGFsaWdubWVudC9tYXAgZm9ybWF0IGFuZCBTQU10b29sczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJyZWFrRGFuY2VyOiBBbiBhbGdvcml0aG0gZm9yIGhpZ2gtcmVzb2x1dGlvbiBtYXBwaW5nIG9mIGdlbm9taWMgc3RydWN0dXJhbCB2YXJpYXRpb247IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9Njc3LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZWx2ZXQ6IEFsZ29yaXRobXMgZm9yIGRlIG5vdm8gc2hvcnQgcmVhZCBhc3NlbWJseSB1c2luZyBkZSBCcnVpam4gZ3JhcGhzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTgyMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcGVuIHNvdXJjZSBNVU1tZXIgMy4wIGlzIGRlc2NyaWJlZCBpbiDigJhWZXJzYXRpbGUgYW5kIG9wZW4gc29mdHdhcmUgZm9yIGNvbXBhcmluZyBsYXJnZSBnZW5vbWVzOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPVIxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5YW5vQ2x1c3Q6IGNvbXBhcmF0aXZlIGdlbm9tZSByZXNvdXJjZXMgb2YgY3lhbm9iYWN0ZXJpYSBhbmQgcGxhc3RpZHM7IGNpdGF0aW9uX3RleHQ9YmFwMDI1OyBEYXRhYmFzZSAoT3hmb3JkKTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBzdHJ1Y3R1cmUgb2YgdGhlIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzIHN0cmFpbiBHVC1TOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEhZTElQ4oCUUGh5bG9nZW55IEluZmVyZW5jZSBQYWNrYWdlIChWZXJzaW9uIDMuMik7IENsYWRpc3RpY3M7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE2NC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaXN0aWRpbmUga2luYXNlcyBwbGF5IGltcG9ydGFudCByb2xlcyBpbiB0aGUgcGVyY2VwdGlvbiBhbmQgc2lnbmFsIHRyYW5zZHVjdGlvbiBvZiBoeWRyb2dlbiBwZXJveGlkZSBpbiB0aGUgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MzEzLTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaXZlIGhpc3RpZGluZSBraW5hc2VzIHBlcmNlaXZlIG9zbW90aWMgc3RyZXNzIGFuZCByZWd1bGF0ZSBkaXN0aW5jdCBzZXRzIG9mIGdlbmVzIGluIFN5bmVjaG9jeXN0aXM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3OTsgIGNpdGF0aW9uX3BhZ2VzPTUzMDc4LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2xkLXJlZ3VsYXRlZCBnZW5lcyB1bmRlciBjb250cm9sIG9mIHRoZSBjb2xkIHNlbnNvciBIaWszMyBpbiBTeW5lY2hvY3lzdGlzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0yMzUtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0b2NrIGN1bHR1cmUgaGV0ZXJvZ2VuZWl0eSByYXRoZXIgdGhhbiBuZXcgbXV0YXRpb25hbCB2YXJpYXRpb24gY29tcGxpY2F0ZXMgc2hvcnQtdGVybSBjZWxsIHBoeXNpb2xvZ3kgc3R1ZGllcyBvZiBFc2NoZXJpY2hpYSBjb2xpIEstMTIgTUcxNjU1IGluIGNvbnRpbnVvdXMgY3VsdHVyZTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE1NzsgIGNpdGF0aW9uX3BhZ2VzPTI2MDQtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGFnZW5lc2lzIGJ5IHJhbmRvbSBjbG9uaW5nIG9mIGFuIEVzY2hlcmljaGlhIGNvbGkga2FuYW15Y2luIHJlc2lzdGFuY2UgZ2VuZSBpbnRvIHRoZSBnZW5vbWUgb2YgdGhlIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgUENDIDY4MDM6IFNlbGVjdGlvbiBvZiBtdXRhbnRzIGRlZmVjdGl2ZSBpbiBwaG90b3N5bnRoZXNpczsgTW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTY7ICBjaXRhdGlvbl9wYWdlcz01MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UeXBlIElWIHBpbHVzIGJpb2dlbmVzaXMgYW5kIG1vdGlsaXR5IGluIHRoZSBjeWFub2JhY3Rlcml1bSBTeW5lY2hvY3lzdGlzIHNwLiBQQ0M2ODAzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz05NDEtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS8xLzY3LzEzMTM4MzgvZHNyMDQyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgY3lhbm9iYWN0ZXJpdW0sIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzLCB3YXMgdGhlIGZpcnN0IHBob3Rvc3ludGhldGljIG9yZ2FuaXNtIHdob3NlIGdlbm9tZSBzZXF1ZW5jZSB3YXMgZGV0ZXJtaW5lZCBpbiAxOTk2IChLYXp1c2Egc3RyYWluKS4gSSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDg1OTg4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZG1E8wcpIxhVSlXefzpj2VuRthU1" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f67%2f485988" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f67%2f485988" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19070" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&amp;Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/1">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7394558">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394558">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7394562">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394562">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7394572">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394572">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7394605">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394605">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7394639">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394639">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7394656">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394656">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7394660">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394660">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7394664">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7394664">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/1/51/485713">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/1/81/486167">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of <em>Synechocystis</em> sp. PCC 6803
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yu Kanesaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Genome Research Center, NODAI Research Institute</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yu+Kanesaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanesaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanesaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yu Kanesaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuh Shiwa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Genome Research Center, NODAI Research Institute</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuh+Shiwa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shiwa Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shiwa Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuh Shiwa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoyuki Tajima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Life Sciences</div>, <div class="insititution">Graduate School of Arts and Sciences, The University of Tokyo</div>, <div class="addr-line">3-8-1 Komaba, Meguro-ku, Tokyo 153-8902</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoyuki+Tajima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tajima N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tajima N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naoyuki Tajima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marie Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marie+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Marie Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoru Watanabe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Life Sciences</div>, <div class="insititution">Graduate School of Arts and Sciences, The University of Tokyo</div>, <div class="addr-line">3-8-1 Komaba, Meguro-ku, Tokyo 153-8902</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naoki Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahiko Ikeuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Life Sciences</div>, <div class="insititution">Graduate School of Arts and Sciences, The University of Tokyo</div>, <div class="addr-line">3-8-1 Komaba, Meguro-ku, Tokyo 153-8902</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahiko+Ikeuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikeuchi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikeuchi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masahiko Ikeuchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirofumi Yoshikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Genome Research Center, NODAI Research Institute</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">1-1-1 Sakuragaoka, Setagaya-ku, Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 3-5477-2769. Fax. +81 3-5477-2377. E-mail: <a href="mailto:hiyoshik@nodai.ac.jp" target="_blank">hiyoshik@nodai.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirofumi+Yoshikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshikawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshikawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirofumi Yoshikawa</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 67–79, <a href='https://doi.org/10.1093/dnares/dsr042'>https://doi.org/10.1093/dnares/dsr042</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">22 December 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 September 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">13 November 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="485988" href="/dnaresearch/article-pdf/19/1/67/1313838/dsr042.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yu Kanesaki, Yuh Shiwa, Naoyuki Tajima, Marie Suzuki, Satoru Watanabe, Naoki Sato, Masahiko Ikeuchi, Hirofumi Yoshikawa;  Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of <em>Synechocystis</em> sp. PCC 6803, <em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 67–79, <a href='https://doi.org/10.1093/dnares/dsr042'>https://doi.org/10.1093/dnares/dsr042</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/485988?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/485988?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/485988?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/485988?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/485988?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/485988?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/485988?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20of%20Substrain-Specific%20Mutations%20by%20Massively%20Parallel%20Whole-Genome%20Resequencing%20of%20%26lt%3Bem%26gt%3BSynechocystis%26lt%3B%2Fem%26gt%3B%20sp.%20PCC%206803&amp;publicationDate=2011-12-22&amp;volumeNum=19&amp;issueNum=1&amp;author=Kanesaki%2C%20Yu%3B%20Shiwa%2C%20Yuh&amp;startPage=67&amp;endPage=79&amp;contentId=10.1093%2Fdnares%2Fdsr042&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f67%2f485988" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7394558 id="7394558" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The cyanobacterium, <em>Synechocystis</em> sp. PCC 6803, was the first photosynthetic organism whose genome sequence was determined in 1996 (Kazusa strain). It thus plays an important role in basic research on the mechanism, evolution, and molecular genetics of the photosynthetic machinery. There are many substrains or laboratory strains derived from the original Berkeley strain including glucose-tolerant (GT) strains. To establish reliable genomic sequence data of this cyanobacterium, we performed resequencing of the genomes of three substrains (GT-I, PCC-P, and PCC-N) and compared the data obtained with those of the original Kazusa strain stored in the public database. We found that each substrain has sequence differences some of which are likely to reflect specific mutations that may contribute to its altered phenotype. Our resequence data of the PCC substrains along with the proposed corrections/refinements of the sequence data for the Kazusa strain and its derivatives are expected to contribute to investigations of the evolutionary events in the photosynthetic and related systems that have occurred in <em>Synechocystis</em> as well as in other cyanobacteria.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequence; massive parallel sequencer&quot;">genome sequence; massive parallel sequencer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="substrain">substrain</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Synechocystis sp. PCC 6803&quot;"><em>Synechocystis</em> sp. PCC 6803</a></div>                            <h2 scrollto-destination=7394562 id="7394562" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Cyanobacteria are capable of oxygenic photosynthesis; they are thought to be the progenitor of plant plastids. <em>Synechocystis</em> sp. PCC 6803 is one of the most widely used cyanobacterial species for genetic studies for several major reasons; (i) it is naturally competent by incorporating exogenous DNA into cells that is integrated into the genome by homologous recombination at high frequency;<sup><a reveal-id="DSR042C1  DSR042C3" data-open="DSR042C1  DSR042C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> (ii) it grows heterotrophically in the presence of glucose;<sup><a reveal-id="DSR042C3" data-open="DSR042C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR042C4" data-open="DSR042C4" class="link link-ref link-reveal xref-bibr">4</a></sup> (iii) the entire genome sequence was determined early on by Kaneko <em>et al.</em><sup><a reveal-id="DSR042C5" data-open="DSR042C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The availability of the entire genome sequence facilitated post-genomic investigations such as transcriptome-, proteome-, and functional genomics studies.<sup><a reveal-id="DSR042C6" data-open="DSR042C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p><p>The original strain of <em>Synechocystis</em> was isolated from California freshwater by Kunisawa and colleagues and called the Berkeley strain;<sup><a reveal-id="DSR042C7" data-open="DSR042C7" class="link link-ref link-reveal xref-bibr">7</a></sup> it was deposited in the Pasteur Culture Collection (PCC strain) and the American Type Culture Collection (ATCC strain). Williams<sup><a reveal-id="DSR042C3" data-open="DSR042C3" class="link link-ref link-reveal xref-bibr">3</a></sup> subsequently isolated the glucose-tolerant (GT) strain from the ATCC strain.<sup><a reveal-id="DSR042C3" data-open="DSR042C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The Kazusa strain, whose genome sequence was published in 1996,<sup><a reveal-id="DSR042C5" data-open="DSR042C5" class="link link-ref link-reveal xref-bibr">5</a></sup> is a derivative of a GT strain. A single representative clone of the GT strain was established for complete genome sequencing as the Kazusa strain; other strains were maintained and transferred without further cloning such as single colony isolation.<sup><a reveal-id="DSR042C8" data-open="DSR042C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Therefore, four substrains, PCC-, ATCC-, GT-, and Kazusa strains, all derived from the original Berkeley strain, were distributed to a number of laboratories although all of them were grouped together under the name <em>Synechocystis</em> sp. PCC 6803.<sup><a reveal-id="DSR042C9" data-open="DSR042C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Ikeuchi and Tabata<sup><a reveal-id="DSR042C8" data-open="DSR042C8" class="link link-ref link-reveal xref-bibr">8</a></sup> reported that each substrain had specific mutations such as single nucleotide polymorphisms (SNPs) and indels, and some exhibited a specific phenotype. Some of the mutated loci that were different from the sequence in the database derived from the Kazusa strain have been identified such as a SNP,<sup><a reveal-id="DSR042C10" data-open="DSR042C10" class="link link-ref link-reveal xref-bibr">10</a></sup> indels,<sup><a reveal-id="DSR042C6" data-open="DSR042C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR042C11  DSR042C13" data-open="DSR042C11  DSR042C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup> and IS mobilization.<sup><a reveal-id="DSR042C14" data-open="DSR042C14" class="link link-ref link-reveal xref-bibr">14</a></sup> However, the total number of mutations in the whole genome of each strain remained unknown and sequence variations in these major strains can be expected to raise problems in the evaluation of phenotypes of mutants constructed from these strains. The history of these major substrains and additional substrains, isolated as a single colony from the PCC- and GT strain (GT-I strain; the standard strain in Dr Ikeuchi's group) was summarized by Ikeuchi and Tabata.<sup><a reveal-id="DSR042C8" data-open="DSR042C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The single colonies isolated from the PCC strain were designated PCC-P (positive phototaxis) strain and PCC-N (negative phototaxis) strain based on the direction of phototactic movement.<sup><a reveal-id="DSR042C15" data-open="DSR042C15" class="link link-ref link-reveal xref-bibr">15</a></sup> A derivative of the GT-I strain that acquired high light tolerance and a glucose-sensitive phenotype was designated the WL strain, which has an SNP in the <em>pmgA</em> gene.<sup><a reveal-id="DSR042C16" data-open="DSR042C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR042C17" data-open="DSR042C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Thus, there are two fundamental problems for post-genomic research in bacterial molecular genetics. One is the heterogeneity of cells in the frozen stock of the culture collection centres; the other is the frequent spontaneous mutation in bacterial genomes, an event that may be unavoidable during the long cultivation of bacterial cells. As revealed in <em>Bacillus subtilis</em>, whole-genome resequencing is a powerful solution for obtaining the sequence information of such spontaneous mutants.<sup><a reveal-id="DSR042C18" data-open="DSR042C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Without question, laboratories should start their post-genomic research with genome sequence data of the ‘reference’ or ‘standard’ strain. We deciphered the three substrains of <em>Synechocystis</em> sp. PCC 6803, i.e. PCC-P, PCC-N, and GT-I, to reconstruct the informatics basis of the molecular biology of <em>Synechocystis</em> sp. PCC 6803. We identified a number of SNPs and indels in these substrains and introduced a genetic strategy to identify the mutated loci on a genome-wide level using the massive parallel sequencer. Especially, determination of the genome sequence of PCC substrains will widely contribute cyanobacterial researches using the frozen stock cells supplied from the PCC.</p>                            <h2 scrollto-destination=7394572 id="7394572" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=7394574 id="7394574" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and genomic DNA</h3>
<p><em>Synechocystis</em> PCC-P, PCC-N, and GT-I strains were maintained as frozen stocks in the laboratory of Dr Masahiko Ikeuchi at The University of Tokyo, Japan. The PCC-P and PCC-N strains that exhibited positive- or negative-direction movement under phototaxis test conditions, respectively, were isolated by Yoshihara <em>et al</em>.<sup><a reveal-id="DSR042C19" data-open="DSR042C19" class="link link-ref link-reveal xref-bibr">19</a></sup> as a single colony from frozen stock obtained from the French Pasteur Culture Collection (PCC strain; see catalogue of strains.<sup><a reveal-id="DSR042C9" data-open="DSR042C9" class="link link-ref link-reveal xref-bibr">9</a></sup>). Genomic DNA was extracted with the hot-phenol method.<sup><a reveal-id="DSR042C16" data-open="DSR042C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=7394577 id="7394577" class="section-title" data-legacy-id=s2b>2.2. Sequencing methods</h3>
<p>DNA was uniformly sheared into 300-bp portions using Adaptive Focused Acoustics (Covaris Inc., Woburn, MA, USA). We constructed a DNA library with a median insert size of 300 bp for a paired-end read format. The quality of the DNA library was checked with the Sanger method by <em>Escherichia coli</em> transformation of aliquots of the library solution. The library was sequenced on a Genome Analyser II (Illumina Inc., San Diego, CA, USA). Sample preparation, cluster generation, and 50-base paired-end sequencing were according to the manufacturer's protocols with minor modifications (Illumina paired-end cluster generation kit GAII ver. 2, 36-cycle sequencing kit ver. 3) with multiplex method using the single lane of the 8 lane flow-cell. Image analysis and ELAND alignment were with Illumina's Pipeline Analysis software ver. 1.6. Sequences passing standard Illumina GA pipeline filters were retained.</p>                            <h3 scrollto-destination=7394580 id="7394580" class="section-title" data-legacy-id=s2c>2.3. Mapping analyses using short-read sequences</h3>
<p>For short-read alignment and calling variants (SNPs and InDels), we used the short-read mapping software MAQ version 0.7.1,<sup><a reveal-id="DSR042C20" data-open="DSR042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> BWA ver. 0.5.1,<sup><a reveal-id="DSR042C21" data-open="DSR042C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and SAMtools ver. 0.1.9.<sup><a reveal-id="DSR042C22" data-open="DSR042C22" class="link link-ref link-reveal xref-bibr">22</a></sup> MAQ alignments were done using the ‘easyrun’ option of the maq-pl script using the default parameter settings. The SNP filtering was performed using the default parameters except for the minimum consensus quality for SNPs (-q 40). BWA alignments and the subsequent variants calling using SAMtools were done using the default parameter settings. Finally, we applied the following filtering criteria to the lists of SNPs/indels: minimum read depth for SNPs calling = 3, minimum read depth for indel calling = 10, and a 60% cut-off of the percent of aligned reads calling the SNP/indel per total mapped reads at the non-reference allele sites. We also used BWA to estimate the sequence read depth affecting the coverage and accuracy of the variant calls. Structural variations were identified using BreakDancer<sup><a reveal-id="DSR042C23" data-open="DSR042C23" class="link link-ref link-reveal xref-bibr">23</a></sup> with default parameters.</p>                            <h3 scrollto-destination=7394584 id="7394584" class="section-title" data-legacy-id=s2d>2.4. Mapping analyses using contigs assembled <em>de novo</em></h3>
<p>Read sequences were assembled <em>de novo</em> with the Velvet assembly programme.<sup><a reveal-id="DSR042C24" data-open="DSR042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> For optimization of the hash value of the assembly process we used the N50 size. The <em>de novo</em> assembled contigs were mapped on the genome sequence of a database derived from the Kazusa strain using MUMmer sequence alignment package<sup><a reveal-id="DSR042C25" data-open="DSR042C25" class="link link-ref link-reveal xref-bibr">25</a></sup> with default settings. We then employed show-SNPs functions of the MUMmer program to produce lists of SNPs/indels which were applied the filtering criteria describe above.</p>                            <h3 scrollto-destination=7394588 id="7394588" class="section-title" data-legacy-id=s2e>2.5. Annotation and creation of the SNP/indel list</h3>
<p>The list of SNPs/indels was then annotated with in-house developed software Variant Annotator (VA) that was specifically designed to check amino acid substitutions attributable to the large number of identified SNPs/indels using Genbank annotation files. We used the GenBank, RefSeq, cyanobacterial database Cyanobase (<a class="link link-uri" href="http://genome.kazusa.or.jp/cyanobase">http://genome.kazusa.or.jp/cyanobase</a>), CyanoClust,<sup><a reveal-id="DSR042C26" data-open="DSR042C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and also ORF information of the GT-S strain,<sup><a reveal-id="DSR042C27" data-open="DSR042C27" class="link link-ref link-reveal xref-bibr">27</a></sup> which is the recently resequenced substrain of <em>Synechocystis</em> sp. PCC 6803, for precise annotation of each ORF.</p>                            <h3 scrollto-destination=7394592 id="7394592" class="section-title" data-legacy-id=s2f>2.6. Capillary sequencing with the Sanger method for SNP/indel confirmation</h3>
<p>About 200-base genomic regions around the SNPs and indels called by the mapping programmes were amplified by PCR and sequenced on a capillary sequencer with the Sanger method using the commercial sequence service of MACROGEN (Tokyo, Japan). To confirm the SNPs located near IS elements or repetitive regions, the longer DNA fragments were amplified to avoid the amplification of other homologous regions in the <em>Synechocystis</em> genome. The primers used for confirmation are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=7394595 id="7394595" class="section-title" data-legacy-id=s2g>2.7. Uploading the genome sequence in the database</h3>
<p>Short-read data, obtained on a Genome Analyzer II (Illumina Inc., San Diego, CA, USA), of the substrains of <em>Synechocystis</em> sp. PCC 6803, PCC-P, PCC-N, and GT-I, were deposited in the DRA (<span class="underline">D</span>DBJ Sequence <span class="underline">R</span>ead <span class="underline">A</span>rchive; <a class="link link-uri" href="http://trace.ddbj.nig.ac.jp/DRASearch/">http://trace.ddbj.nig.ac.jp/DRASearch/</a>); the accession number is DRA000401. The genome sequences and gene annotations of the substrain GT-I, PCC-P, and PCC-N were also deposited in the DDBJ/GenBank/EMBL database with the accession numbers for each substrain, GT-I (AP012276), PCC-P (AP012278), and PCC-N (AP012277).</p>                            <h3 scrollto-destination=7394600 id="7394600" class="section-title" data-legacy-id=s2h>2.8. Phylogenetic analysis of <em>Synechocystis</em> sp. PCC 6803 substrains</h3>
<p>Phylogenetic relationship of various strains was estimated by the maximum parsimony method by assuming that both base change and indel are treated as a single event. The computation was performed by the dolpenny software of the Phylip package version 3.67,<sup><a reveal-id="DSR042C28" data-open="DSR042C28" class="link link-ref link-reveal xref-bibr">28</a></sup> using the polymorphism option. Each branch length was set as the number of events occurring along the branch.</p>                            <h2 scrollto-destination=7394605 id="7394605" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7394608 id="7394608" class="section-title" data-legacy-id=s3a>3.1. Analytical scheme applied to the massive short-read data obtained by next-generation sequencing</h3>
<p>The amplified DNA library was sequenced by GAII with 50-base paired-end methods using the parameter settings described in Materials and methods section. We obtained 250, 257, and 221 Mb read data for GT-I, PCC-N, and PCC-P substrains, respectively (Table <a reveal-id="DSR042TB1" data-open="DSR042TB1" class="link link-reveal link-table xref-fig">1</a>). These read depths correspond to more than 60 times the genome size of <em>Synechocystis</em>. Read data were mapped using three analyses to identify the genomic position of the SNPs, indels, and rearrangements (Fig. <a reveal-id="DSR042F1" data-open="DSR042F1" class="link link-reveal link-table xref-fig">1</a>): (i) BWA<sup><a reveal-id="DSR042C21" data-open="DSR042C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and MAQ<sup><a reveal-id="DSR042C20" data-open="DSR042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> for mapping analysis using raw read data; (ii) Velvet<sup><a reveal-id="DSR042C24" data-open="DSR042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and MUMmer<sup><a reveal-id="DSR042C25" data-open="DSR042C25" class="link link-ref link-reveal xref-bibr">25</a></sup> for mapping analysis using <em>de novo</em> assembled contigs; and (iii) BreakDancer<sup><a reveal-id="DSR042C23" data-open="DSR042C23" class="link link-ref link-reveal xref-bibr">23</a></sup> for rearrangement analysis such as IS movement. The number of mutations was called by each programme and passed through the filter settings of the SAMtools program<sup><a reveal-id="DSR042C22" data-open="DSR042C22" class="link link-ref link-reveal xref-bibr">22</a></sup> (Table <a reveal-id="DSR042TB1" data-open="DSR042TB1" class="link link-reveal link-table xref-fig">1</a>). Distributions of averaged read depth in each 1 kb along with the entire genome, obtained by BWA and MAQ were shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplemental Fig. S1</a>. At least 15 times read depth was obtained even in the lowest read depth region. It also indicates that patterns of the read depth distribution depend on the algorithm of each mapping programme. The mapping programmes BWA, MAQ, and MUMer called 76, 69, and 85 potential mutation points, respectively, for the GT-I strain as primary data, 89, 75, and 109 points for the PCC-P strain, and 78, 79, and 104 points for the PCC-N strain. To confirm these results, we checked the sequence around all these loci by Sanger sequencing; regions of ∼200 bp were amplified around the position of the mutations. Depending on the parameter settings, read depth and sequence specificity, it happens that common SNPs in all three substrains were detected only in one or two substrains in each programme. Even if the SNP is called only in one strain, we performed Sanger sequencing of the same locus in all three strains. Whole oligo DNA primers prepared for PCR reactions are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplemental Table S1</a>. </p>                        <a id="7394612" scrollto-destination="7394612"></a>
<div content-id="DSR042TB1" class="table-modal"><div class="table-wrap"><div id="DSR042TB1" class="table-wrap-title" data-id="DSR042TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of mapping analyses using the read data (BWA, MAQ) or the <em>de novo</em> assembled contigs (Velvet and MUMmer)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="3"><em>Synechocystis</em> sp. PCC 6803 substrains<hr /> </th></tr><tr><th>GT-I </th><th>PCC-N </th><th>PCC-P </th></tr></thead><tbody><tr><td>Total read bases (Mb) </td><td>250 </td><td>257 </td><td>221 </td></tr><tr><td>Averaged read depth </td><td>70 </td><td>72 </td><td>62 </td></tr><tr><td>Genome coverage (%) </td><td>99.99 </td><td>99.99 </td><td>99.99 </td></tr><tr><td>Mapping programmes </td><td colspan="3">Number of SNPs and indels called by each programmes (Final number of differences/number of differences including false-positive data) </td></tr><tr><td>MAQ </td><td>16/76 </td><td>26/78 </td><td>23/89 </td></tr><tr><td>BWA </td><td>19/69 </td><td>32/79 </td><td>28/75 </td></tr><tr><td>Velvet and MUMmer </td><td>22/85 </td><td>33/104 </td><td>29/109 </td></tr><tr><td>BreakDancer </td><td>3/3 </td><td>3/3 </td><td>3/3 </td></tr><tr><td>Final number of differences to the database </td><td>28 </td><td>44 </td><td>39 </td></tr></tbody></table></div><div content-id="DSR042TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="3"><em>Synechocystis</em> sp. PCC 6803 substrains<hr /> </th></tr><tr><th>GT-I </th><th>PCC-N </th><th>PCC-P </th></tr></thead><tbody><tr><td>Total read bases (Mb) </td><td>250 </td><td>257 </td><td>221 </td></tr><tr><td>Averaged read depth </td><td>70 </td><td>72 </td><td>62 </td></tr><tr><td>Genome coverage (%) </td><td>99.99 </td><td>99.99 </td><td>99.99 </td></tr><tr><td>Mapping programmes </td><td colspan="3">Number of SNPs and indels called by each programmes (Final number of differences/number of differences including false-positive data) </td></tr><tr><td>MAQ </td><td>16/76 </td><td>26/78 </td><td>23/89 </td></tr><tr><td>BWA </td><td>19/69 </td><td>32/79 </td><td>28/75 </td></tr><tr><td>Velvet and MUMmer </td><td>22/85 </td><td>33/104 </td><td>29/109 </td></tr><tr><td>BreakDancer </td><td>3/3 </td><td>3/3 </td><td>3/3 </td></tr><tr><td>Final number of differences to the database </td><td>28 </td><td>44 </td><td>39 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7394612" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR042TB1" class="table-wrap-title" data-id="DSR042TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of mapping analyses using the read data (BWA, MAQ) or the <em>de novo</em> assembled contigs (Velvet and MUMmer)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="3"><em>Synechocystis</em> sp. PCC 6803 substrains<hr /> </th></tr><tr><th>GT-I </th><th>PCC-N </th><th>PCC-P </th></tr></thead><tbody><tr><td>Total read bases (Mb) </td><td>250 </td><td>257 </td><td>221 </td></tr><tr><td>Averaged read depth </td><td>70 </td><td>72 </td><td>62 </td></tr><tr><td>Genome coverage (%) </td><td>99.99 </td><td>99.99 </td><td>99.99 </td></tr><tr><td>Mapping programmes </td><td colspan="3">Number of SNPs and indels called by each programmes (Final number of differences/number of differences including false-positive data) </td></tr><tr><td>MAQ </td><td>16/76 </td><td>26/78 </td><td>23/89 </td></tr><tr><td>BWA </td><td>19/69 </td><td>32/79 </td><td>28/75 </td></tr><tr><td>Velvet and MUMmer </td><td>22/85 </td><td>33/104 </td><td>29/109 </td></tr><tr><td>BreakDancer </td><td>3/3 </td><td>3/3 </td><td>3/3 </td></tr><tr><td>Final number of differences to the database </td><td>28 </td><td>44 </td><td>39 </td></tr></tbody></table></div><div content-id="DSR042TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="3"><em>Synechocystis</em> sp. PCC 6803 substrains<hr /> </th></tr><tr><th>GT-I </th><th>PCC-N </th><th>PCC-P </th></tr></thead><tbody><tr><td>Total read bases (Mb) </td><td>250 </td><td>257 </td><td>221 </td></tr><tr><td>Averaged read depth </td><td>70 </td><td>72 </td><td>62 </td></tr><tr><td>Genome coverage (%) </td><td>99.99 </td><td>99.99 </td><td>99.99 </td></tr><tr><td>Mapping programmes </td><td colspan="3">Number of SNPs and indels called by each programmes (Final number of differences/number of differences including false-positive data) </td></tr><tr><td>MAQ </td><td>16/76 </td><td>26/78 </td><td>23/89 </td></tr><tr><td>BWA </td><td>19/69 </td><td>32/79 </td><td>28/75 </td></tr><tr><td>Velvet and MUMmer </td><td>22/85 </td><td>33/104 </td><td>29/109 </td></tr><tr><td>BreakDancer </td><td>3/3 </td><td>3/3 </td><td>3/3 </td></tr><tr><td>Final number of differences to the database </td><td>28 </td><td>44 </td><td>39 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7394612" target="_blank">View Large</a></div></div>                        <a id="7394614" scrollto-destination="7394614"></a>
<div id="DSR042F1" data-id="DSR042F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04201.jpeg?Expires=2147483647&Signature=LSbWX1QUgP~rvWgzvQi1fHh7KSEGdkDqm40hiy~YEHU3pDzsno7c4A~E99akRIPGepJiT5Pe2nmq6Qnaypcwlr~P~HhKui7iATtFFjeP2JypF49cHh0LYoNM-ndfWw9pzLvj1fh3DPrRaxD5sTb24zEV50Ix1OAHHzWySGKDbZj2bkWOYC~3AqGpcwbpZP-4Z8zErASGehrWuuWiTzWcWKiOnZ-VNsqCyIlvblt~QK~7to1TzWddqs0oYGAjRM0OHH9EOOkfch5ITIbmBZhPP55vLyQnHOf6jLdoVKWq1Mr6lVtBXe0i7WoteYFU7Y-QZ45rBQyg2iaJCRSn49p71w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analytical scheme of the read data obtained by massive parallel sequencing. The preparation of the DNA library is described in Materials and methods section. The mapping programmes BWA and MAQ were used for short-read data; the de novo assembly programme was Velvet, and MUMmer was the mapping programme for assembled contigs." path-from-xml="dsr04201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394614/dsr04201.jpeg" path-from-xml="dsr04201.jpeg" target="_blank">View large</a><a section="7394614" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04201.jpeg?Expires=2147483647&Signature=E-ruEjkyAIAZCAvj4v5uUF2wQpie3z8p0mUON03MKEg1upkANW35KUd2ngpDLyGy0q1xvsTXlOEg3cUQuxOI7sMaAQ0-wio-ZGZChz3aOmCSJurxSkS-ZcIFG6-llvf0O-49r6aARGFfUc1UazENcCrqnL53EzWqA9VK5Du6AIAnctbnnfqt06cT-ZnT05czO5xwUK5uLbWDnX21KDnJKbzq-akW4BsmWrNwDJBM~ojrzauMEFgEWE5~2U8a4~KRFR9zmv~btFUu4ltgK0jcbYtB2mJVYZ1hmDCI7VU6RsByOQQbH~A5GxV2D-E29gGEuLXiVyjqKOynngUFnBLDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394614&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analytical scheme of the read data obtained by massive parallel sequencing. The preparation of the DNA library is described in Materials and methods section. The mapping programmes BWA and MAQ were used for short-read data; the <em>de novo</em> assembly programme was Velvet, and MUMmer was the mapping programme for assembled contigs.</p></div></div><div content-id="DSR042F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04201.jpeg?Expires=2147483647&Signature=LSbWX1QUgP~rvWgzvQi1fHh7KSEGdkDqm40hiy~YEHU3pDzsno7c4A~E99akRIPGepJiT5Pe2nmq6Qnaypcwlr~P~HhKui7iATtFFjeP2JypF49cHh0LYoNM-ndfWw9pzLvj1fh3DPrRaxD5sTb24zEV50Ix1OAHHzWySGKDbZj2bkWOYC~3AqGpcwbpZP-4Z8zErASGehrWuuWiTzWcWKiOnZ-VNsqCyIlvblt~QK~7to1TzWddqs0oYGAjRM0OHH9EOOkfch5ITIbmBZhPP55vLyQnHOf6jLdoVKWq1Mr6lVtBXe0i7WoteYFU7Y-QZ45rBQyg2iaJCRSn49p71w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analytical scheme of the read data obtained by massive parallel sequencing. The preparation of the DNA library is described in Materials and methods section. The mapping programmes BWA and MAQ were used for short-read data; the de novo assembly programme was Velvet, and MUMmer was the mapping programme for assembled contigs." path-from-xml="dsr04201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394614/dsr04201.jpeg" path-from-xml="dsr04201.jpeg" target="_blank">View large</a><a section="7394614" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04201.jpeg?Expires=2147483647&Signature=E-ruEjkyAIAZCAvj4v5uUF2wQpie3z8p0mUON03MKEg1upkANW35KUd2ngpDLyGy0q1xvsTXlOEg3cUQuxOI7sMaAQ0-wio-ZGZChz3aOmCSJurxSkS-ZcIFG6-llvf0O-49r6aARGFfUc1UazENcCrqnL53EzWqA9VK5Du6AIAnctbnnfqt06cT-ZnT05czO5xwUK5uLbWDnX21KDnJKbzq-akW4BsmWrNwDJBM~ojrzauMEFgEWE5~2U8a4~KRFR9zmv~btFUu4ltgK0jcbYtB2mJVYZ1hmDCI7VU6RsByOQQbH~A5GxV2D-E29gGEuLXiVyjqKOynngUFnBLDVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394614&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analytical scheme of the read data obtained by massive parallel sequencing. The preparation of the DNA library is described in Materials and methods section. The mapping programmes BWA and MAQ were used for short-read data; the <em>de novo</em> assembly programme was Velvet, and MUMmer was the mapping programme for assembled contigs.</p></div></div>                            <h3 scrollto-destination=7394617 id="7394617" class="section-title" data-legacy-id=s3b>3.2. Combinatorial use of the mapping programmes contributes to the identification of SNPs and indels</h3>
<p>Confirmation by Sanger sequencing alerted to a number of false-positive SNP- and indel-calls. Final numbers of SNPs/indels in each substrain were only about 20–40% of the called numbers by each programme as shown in Table <a reveal-id="DSR042TB1" data-open="DSR042TB1" class="link link-reveal link-table xref-fig">1</a>. Most of the false-positive SNPs or indels were located in repetitive regions or in highly homologous genes in the <em>Synechocystis</em> genome. We expected that the cut-off value of mapping programmes as 60% is enough to detect a number of heterogeneous SNPs of <em>Synechocystis</em> which has multi-copy genome, because 80% SNPs were called as heterogeneous SNPs by BWA in case of GT-I strain. However, we could not detect any heterogeneous SNPs among them by the Sanger method. It indicates that there are technical problems in expecting the total number of the heterogeneous SNPs in the whole genome using cut-off value settings or base-call percentage data obtained by these mapping programmes. On the other hand, all 16 SNPs called homogeneous SNPs by BWA in GT-I strain were also confirmed by the Sanger method. The total number of mutations confirmed by the Sanger method is shown in Fig. <a reveal-id="DSR042F2" data-open="DSR042F2" class="link link-reveal link-table xref-fig">2</a> with the number of mutations including false-positive data in parenthesis. These numbers are the sum of results obtained for the three substrains. We found that the combinatorial use of several programmes is necessary for the comprehensive detection of SNPs/indels and for the identification of mutations. Mutation loci detected commonly by all three programmes were more reliable than that detected by only the single programme (Fig. <a reveal-id="DSR042F2" data-open="DSR042F2" class="link link-reveal link-table xref-fig">2</a>). Difference of the distribution pattern of the read depth in each mapping programme also suggests that combination use of several programmes is more adequate (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplemental Fig. S1</a>). However, it is worth noting, SNPs/indels identified commonly by three programmes covered only 60% of the total number of mutations. The correct detection of IS movements reported by Okamoto <em>et al</em>.<sup><a reveal-id="DSR042C14" data-open="DSR042C14" class="link link-ref link-reveal xref-bibr">14</a></sup> was possible only with BreakDancer, a programme developed for the detection of genome rearrangements.<sup><a reveal-id="DSR042C23" data-open="DSR042C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                        <a id="7394622" scrollto-destination="7394622"></a>
<div id="DSR042F2" data-id="DSR042F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04202.jpeg?Expires=2147483647&Signature=ODsANNpNtOCktruRNDOeEWr6JaYQtyYamwCzihllf3wLw9cDhsVbqce~BVHNChx4EzTdihkboQZFErfoRtCaIxVD1mvRdsa1u3lWB6z9QSrDXD5~i21UxWF~PirR6RubcV0RcIubh38M8-iowOybLwnETfL8a3qjny018xRj6LNhSZlTTKGbpUQg7-ouZaPAqWpLAisaZiUdpHeNOpbep7byxUCdBb7Ei7ci6EKgURrS8xyOyY8njvqDoCtGUvr9EpvB6vVzuDmdoEDp72QNKHNyc7I7BuI4UocO8ynw6OsIM5tyRBg4etwAMeJOO-fxdVja8aPlGQgHX9dCAhmuyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Diagram of the mutations identified by each programme. The number of mutations (SNPs and indels) confirmed by the Sanger method is shown in each circle with the number of mutations including false-positive data in parenthesis. The number of mutations detected by plural programmes is indicated in the circle overlap region. Threshold (cut-off) value of 60% was used in mapping programmes; BWA and MAQ (see Materials and methods section). Mutations detected commonly by all three programmes were more reliable. The combinatorial use of the mapping programmes is important for the genome-wide identification of the mutation loci. Numbers labelled with an asterisk contain miss-called results indicated by parenthesis in Tables 2 and 3." path-from-xml="dsr04202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394622/dsr04202.jpeg" path-from-xml="dsr04202.jpeg" target="_blank">View large</a><a section="7394622" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04202.jpeg?Expires=2147483647&Signature=U4oVY~I3T14kC6SLgVZ67j75~-v~2qWOg0ITmZ4QTB3HIlwg8VCD96oeScEeQpxk~qK-XCGd3nD3ux9txKsvmZ8aV4M5tJm8tE4Nfa04tFm2LudyzRxEbf1tsZZjT8UUSr0GZmCNleevRDPDVcgFtns-ZUSXZeFmoVQSS8pSg4fE2lCcVbJgHlNNUdxz3HxWC5H8RCbE0~n4dIR7Qc-Xhd9VOIH3be52N4Y5nJatG0qG~tLCzYUzFulfYGFNX-FnBgUBcffZFbIj6QGqN7F3rXo9Fx4fAKaA9aR0JWj5g1q~lrXDvlA4uPuhmHBqj1cq1I~GgQX04B0sppNZO3Sixg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394622&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Diagram of the mutations identified by each programme. The number of mutations (SNPs and indels) confirmed by the Sanger method is shown in each circle with the number of mutations including false-positive data in parenthesis. The number of mutations detected by plural programmes is indicated in the circle overlap region. Threshold (cut-off) value of 60% was used in mapping programmes; BWA and MAQ (see Materials and methods section). Mutations detected commonly by all three programmes were more reliable. The combinatorial use of the mapping programmes is important for the genome-wide identification of the mutation loci. Numbers labelled with an asterisk contain miss-called results indicated by parenthesis in Tables <a reveal-id="DSR042TB2" data-open="DSR042TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR042TB3" data-open="DSR042TB3" class="link link-reveal link-table xref-fig">3</a>.</p></div></div><div content-id="DSR042F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04202.jpeg?Expires=2147483647&Signature=ODsANNpNtOCktruRNDOeEWr6JaYQtyYamwCzihllf3wLw9cDhsVbqce~BVHNChx4EzTdihkboQZFErfoRtCaIxVD1mvRdsa1u3lWB6z9QSrDXD5~i21UxWF~PirR6RubcV0RcIubh38M8-iowOybLwnETfL8a3qjny018xRj6LNhSZlTTKGbpUQg7-ouZaPAqWpLAisaZiUdpHeNOpbep7byxUCdBb7Ei7ci6EKgURrS8xyOyY8njvqDoCtGUvr9EpvB6vVzuDmdoEDp72QNKHNyc7I7BuI4UocO8ynw6OsIM5tyRBg4etwAMeJOO-fxdVja8aPlGQgHX9dCAhmuyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Diagram of the mutations identified by each programme. The number of mutations (SNPs and indels) confirmed by the Sanger method is shown in each circle with the number of mutations including false-positive data in parenthesis. The number of mutations detected by plural programmes is indicated in the circle overlap region. Threshold (cut-off) value of 60% was used in mapping programmes; BWA and MAQ (see Materials and methods section). Mutations detected commonly by all three programmes were more reliable. The combinatorial use of the mapping programmes is important for the genome-wide identification of the mutation loci. Numbers labelled with an asterisk contain miss-called results indicated by parenthesis in Tables 2 and 3." path-from-xml="dsr04202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394622/dsr04202.jpeg" path-from-xml="dsr04202.jpeg" target="_blank">View large</a><a section="7394622" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04202.jpeg?Expires=2147483647&Signature=U4oVY~I3T14kC6SLgVZ67j75~-v~2qWOg0ITmZ4QTB3HIlwg8VCD96oeScEeQpxk~qK-XCGd3nD3ux9txKsvmZ8aV4M5tJm8tE4Nfa04tFm2LudyzRxEbf1tsZZjT8UUSr0GZmCNleevRDPDVcgFtns-ZUSXZeFmoVQSS8pSg4fE2lCcVbJgHlNNUdxz3HxWC5H8RCbE0~n4dIR7Qc-Xhd9VOIH3be52N4Y5nJatG0qG~tLCzYUzFulfYGFNX-FnBgUBcffZFbIj6QGqN7F3rXo9Fx4fAKaA9aR0JWj5g1q~lrXDvlA4uPuhmHBqj1cq1I~GgQX04B0sppNZO3Sixg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394622&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Diagram of the mutations identified by each programme. The number of mutations (SNPs and indels) confirmed by the Sanger method is shown in each circle with the number of mutations including false-positive data in parenthesis. The number of mutations detected by plural programmes is indicated in the circle overlap region. Threshold (cut-off) value of 60% was used in mapping programmes; BWA and MAQ (see Materials and methods section). Mutations detected commonly by all three programmes were more reliable. The combinatorial use of the mapping programmes is important for the genome-wide identification of the mutation loci. Numbers labelled with an asterisk contain miss-called results indicated by parenthesis in Tables <a reveal-id="DSR042TB2" data-open="DSR042TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR042TB3" data-open="DSR042TB3" class="link link-reveal link-table xref-fig">3</a>.</p></div></div>                            <h3 scrollto-destination=7394625 id="7394625" class="section-title" data-legacy-id=s3c>3.3. Comparison of the identified SNPs/indels and the genome sequence in the database</h3>
<p>The confirmed mutations are listed in Tables <a reveal-id="DSR042TB2" data-open="DSR042TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR042TB3" data-open="DSR042TB3" class="link link-reveal link-table xref-fig">3</a>. The three substrains, GT-I, PCC-P, and PCC-N, manifested at least 22 common different sites compared with the sequence of the Kazusa strain in the database. Among these mutation sites, 15 sites were different from the database sequence, but not real differences in the genomic loci as revealed by Tajima <em>et al</em>.<sup><a reveal-id="DSR042C27" data-open="DSR042C27" class="link link-ref link-reveal xref-bibr">27</a></sup> (Table <a reveal-id="DSR042TB2" data-open="DSR042TB2" class="link link-reveal link-table xref-fig">2</a>). We also found that there are totally 14 mutations between the GT/Kazusa- and the PCC-P/PCC-N strains (Table <a reveal-id="DSR042TB3" data-open="DSR042TB3" class="link link-reveal link-table xref-fig">3</a>). The PCC-P and PCC-N substrains contained three and eight additional specific mutations, respectively (Table <a reveal-id="DSR042TB3" data-open="DSR042TB3" class="link link-reveal link-table xref-fig">3</a>). These may be potential mutations that elicited the known difference in the phototactic phenotype of the PCC substrains. For example, the PCC-N substrain has mutation in the <em>gspE2</em> (<em>pilB2</em>) gene for pilus assembly, which also moderately affects the transformation efficiency.<sup><a reveal-id="DSR042C15" data-open="DSR042C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The PCC-N strain also has a 12-base deletion in the kinase domain of the <em>hik33</em> gene for the histidine kinase without a frameshift. Hik33 is the multi-stress sensor in <em>Synechocystis</em> and it is conserved in all cyanobacterial species.<sup><a reveal-id="DSR042C29" data-open="DSR042C29" class="link link-ref link-reveal xref-bibr">29</a>–<a reveal-id="DSR042C31" data-open="DSR042C31" class="link link-ref link-reveal xref-bibr">31</a></sup> This suggests that this substrain may lose the Hik33-dependent regulation of global gene expression, although the relationship between <em>hik33</em> and phototaxis remains to be determined. </p>                        <a id="7394630" scrollto-destination="7394630"></a>
<div content-id="DSR042TB2" class="table-modal"><div class="table-wrap"><div id="DSR042TB2" class="table-wrap-title" data-id="DSR042TB2"><div class="label -label">Table 2.</div><div class="caption"><p>List of the genomic loci of SNPs and indels found in all GT-I, PCC-P, and PCC-N strains compared with the nucleotide sequence in the database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>943495 </td><td>SNP </td><td>G </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1834</em> </td><td><em>psaA</em> </td><td>V604I </td><td>Smart and McIntosh (10). Error of the Database (27) </td></tr><tr><td>1012958 </td><td>SNP </td><td>G </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3177-sll1633</em> </td><td><em>repA-ftsZ</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>1200143–1201488 (1200306) </td><td>Indel (SNP) </td><td>IS (C) </td><td>IS </td><td>— </td><td>—(A) </td><td>—(A) </td><td>—(A) </td><td>(108 233) 99 </td><td>(MAQ BWA) BreakDancer </td><td><em>sll1780</em> </td><td><em>ISY203b</em> </td><td>— </td><td>Insertion of transposase (14). GT-Kazusa specific (27). MAQ and BWA detected this indel region as SNP </td></tr><tr><td>1364187 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0838</em> </td><td><em>pyrF</em> </td><td>Silent </td><td>Error of the Database (27) </td></tr><tr><td>2092571 </td><td>SNP </td><td>A </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0422</em> </td><td><em>sll0422</em> </td><td>L313* Stop codon </td><td>Error of the Database (27) </td></tr><tr><td>2198893 </td><td>SNP </td><td>T </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0142</em> </td><td><em>sll0142</em> </td><td>689 Silent </td><td>Error of the Database (27) </td></tr><tr><td>2204584 </td><td>Indel </td><td>G </td><td>G </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>slr0162</em> </td><td><em>gspF</em> </td><td><em>gspF+pilC</em> </td><td>G-insertion in GT-Kazusa strain causes split of the original <em>pilC</em> gene (34). GT-Kazusa specific (27) </td></tr><tr><td>2301721 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0168</em> </td><td><em>slr0168</em> </td><td>K403E </td><td>Error of the Database (27) </td></tr><tr><td>2350285–2350286 </td><td>Indel </td><td>— </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>317 </td><td>BWA mummer </td><td><em>intergenic region sml0001-slr0363</em> </td><td><em>psbI-slr0363</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>2360245–2360246 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>323 </td><td>BWA mummer </td><td><em>slr0364</em> </td><td><em>slr0364</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2409244 </td><td>Indel </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>sll0762</em> </td><td><em>sll0762</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2419399 </td><td>Indel </td><td>T </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>302 </td><td>BWA mummer </td><td><em>sll0752</em> </td><td><em>sll0752</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2544044–2544045 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>180 </td><td>BWA mummer </td><td><em>ssl0787</em> </td><td><em>ssl0787</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2602717 </td><td>SNP </td><td>C </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>H82Q </td><td>Error of the Database (27) </td></tr><tr><td>2602734 </td><td>SNP </td><td>T </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>I88N </td><td>Error of the Database (27) </td></tr><tr><td>2748897 </td><td>SNP </td><td>C </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region slr0210-ssr0332</em> </td><td><em>slr0210-ssr0332</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>3142651 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0045</em> </td><td><em>sps</em> </td><td>75 Silent </td><td>Error of the Database (27) </td></tr><tr><td>3260096 </td><td>Indel </td><td>C </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>Mummer </td><td><em>intergenic region sll0529-sll0528</em> </td><td><em>sll0529-sll0528</em> </td><td>— </td><td>GT-Kazusa specific (27) </td></tr><tr><td>3400322–3401506 </td><td>Indel </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>sll1474</em> </td><td><em>ISY203g</em> </td><td><em>sll1473+sll1475</em> </td><td>Insertion of transposase. IS-insertion causes split of the original <em>hik32</em> gene (14). GT-Kazusa specific (27) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>386410- 386411 (386406) </td><td>Indel (SNP) </td><td>—(T) </td><td>— </td><td>— </td><td>102 bp (A) </td><td>102 bp (A) </td><td>102 bp (A) </td><td>(68) </td><td>(MAQ) </td><td><em>slr1084</em> </td><td><em>slr1084</em> </td><td>34 amino acids deletion (V77D) </td><td>This indel region was called as SNP by MAQ as shown in parentheses. This indel region was not detected in the GT-S strain (27). CTGGGGGAAAAATGTTGGATTGATAACCTCGCCCCGGTTACCATTGAGTCCCATGTGTGTATTTCCCAGGGCGTTTACCTATGCACTGGCAACCACGATTGG </td></tr><tr><td>1192983 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>C/A </td><td>C/A </td><td>C/A </td><td> </td><td>Mummer </td><td><em>slr1855</em> </td><td><em>slr1855</em> </td><td>T167P </td><td>Potential heterogeneous nucleotide (Intensity of the peaks due to C and A were almost equal.) This SNP was not detected in the GT-S strain (27) </td></tr><tr><td>2048341–2049583 </td><td>Indel </td><td>IS </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>slr1635</em> </td><td><em>ISY203e</em> </td><td>— </td><td>Insertion of transposase (14). Specific IS in GT-Kazusa and GT-S strains (27) </td></tr></tbody></table></div><div content-id="DSR042TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>943495 </td><td>SNP </td><td>G </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1834</em> </td><td><em>psaA</em> </td><td>V604I </td><td>Smart and McIntosh (10). Error of the Database (27) </td></tr><tr><td>1012958 </td><td>SNP </td><td>G </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3177-sll1633</em> </td><td><em>repA-ftsZ</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>1200143–1201488 (1200306) </td><td>Indel (SNP) </td><td>IS (C) </td><td>IS </td><td>— </td><td>—(A) </td><td>—(A) </td><td>—(A) </td><td>(108 233) 99 </td><td>(MAQ BWA) BreakDancer </td><td><em>sll1780</em> </td><td><em>ISY203b</em> </td><td>— </td><td>Insertion of transposase (14). GT-Kazusa specific (27). MAQ and BWA detected this indel region as SNP </td></tr><tr><td>1364187 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0838</em> </td><td><em>pyrF</em> </td><td>Silent </td><td>Error of the Database (27) </td></tr><tr><td>2092571 </td><td>SNP </td><td>A </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0422</em> </td><td><em>sll0422</em> </td><td>L313* Stop codon </td><td>Error of the Database (27) </td></tr><tr><td>2198893 </td><td>SNP </td><td>T </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0142</em> </td><td><em>sll0142</em> </td><td>689 Silent </td><td>Error of the Database (27) </td></tr><tr><td>2204584 </td><td>Indel </td><td>G </td><td>G </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>slr0162</em> </td><td><em>gspF</em> </td><td><em>gspF+pilC</em> </td><td>G-insertion in GT-Kazusa strain causes split of the original <em>pilC</em> gene (34). GT-Kazusa specific (27) </td></tr><tr><td>2301721 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0168</em> </td><td><em>slr0168</em> </td><td>K403E </td><td>Error of the Database (27) </td></tr><tr><td>2350285–2350286 </td><td>Indel </td><td>— </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>317 </td><td>BWA mummer </td><td><em>intergenic region sml0001-slr0363</em> </td><td><em>psbI-slr0363</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>2360245–2360246 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>323 </td><td>BWA mummer </td><td><em>slr0364</em> </td><td><em>slr0364</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2409244 </td><td>Indel </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>sll0762</em> </td><td><em>sll0762</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2419399 </td><td>Indel </td><td>T </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>302 </td><td>BWA mummer </td><td><em>sll0752</em> </td><td><em>sll0752</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2544044–2544045 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>180 </td><td>BWA mummer </td><td><em>ssl0787</em> </td><td><em>ssl0787</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2602717 </td><td>SNP </td><td>C </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>H82Q </td><td>Error of the Database (27) </td></tr><tr><td>2602734 </td><td>SNP </td><td>T </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>I88N </td><td>Error of the Database (27) </td></tr><tr><td>2748897 </td><td>SNP </td><td>C </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region slr0210-ssr0332</em> </td><td><em>slr0210-ssr0332</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>3142651 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0045</em> </td><td><em>sps</em> </td><td>75 Silent </td><td>Error of the Database (27) </td></tr><tr><td>3260096 </td><td>Indel </td><td>C </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>Mummer </td><td><em>intergenic region sll0529-sll0528</em> </td><td><em>sll0529-sll0528</em> </td><td>— </td><td>GT-Kazusa specific (27) </td></tr><tr><td>3400322–3401506 </td><td>Indel </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>sll1474</em> </td><td><em>ISY203g</em> </td><td><em>sll1473+sll1475</em> </td><td>Insertion of transposase. IS-insertion causes split of the original <em>hik32</em> gene (14). GT-Kazusa specific (27) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>386410- 386411 (386406) </td><td>Indel (SNP) </td><td>—(T) </td><td>— </td><td>— </td><td>102 bp (A) </td><td>102 bp (A) </td><td>102 bp (A) </td><td>(68) </td><td>(MAQ) </td><td><em>slr1084</em> </td><td><em>slr1084</em> </td><td>34 amino acids deletion (V77D) </td><td>This indel region was called as SNP by MAQ as shown in parentheses. This indel region was not detected in the GT-S strain (27). CTGGGGGAAAAATGTTGGATTGATAACCTCGCCCCGGTTACCATTGAGTCCCATGTGTGTATTTCCCAGGGCGTTTACCTATGCACTGGCAACCACGATTGG </td></tr><tr><td>1192983 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>C/A </td><td>C/A </td><td>C/A </td><td> </td><td>Mummer </td><td><em>slr1855</em> </td><td><em>slr1855</em> </td><td>T167P </td><td>Potential heterogeneous nucleotide (Intensity of the peaks due to C and A were almost equal.) This SNP was not detected in the GT-S strain (27) </td></tr><tr><td>2048341–2049583 </td><td>Indel </td><td>IS </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>slr1635</em> </td><td><em>ISY203e</em> </td><td>— </td><td>Insertion of transposase (14). Specific IS in GT-Kazusa and GT-S strains (27) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The left column shows the genomic locus of each mutation in the database (NCBI accession number; NC_000911). Quality scores indicate the phred-scaled scores called by MAQ and BWA, respectively. Quality scores given by BreakDancer is a software-original value. The upper table listed the mutations that were suggested as the error of the database and also that the GT-Kazusa strain-specific mutations such as <em>ISY203b</em>, <em>ISY203g</em>, and the locus 2204584 (31). Lower table shows additional differences found only in GT-I, PCC-P and PCC-N strains. Greyed columns emphasize the different sites and their details. Several indel regions miscalled as SNP by MAQ and BWA were shown in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7394630" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR042TB2" class="table-wrap-title" data-id="DSR042TB2"><div class="label -label">Table 2.</div><div class="caption"><p>List of the genomic loci of SNPs and indels found in all GT-I, PCC-P, and PCC-N strains compared with the nucleotide sequence in the database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>943495 </td><td>SNP </td><td>G </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1834</em> </td><td><em>psaA</em> </td><td>V604I </td><td>Smart and McIntosh (10). Error of the Database (27) </td></tr><tr><td>1012958 </td><td>SNP </td><td>G </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3177-sll1633</em> </td><td><em>repA-ftsZ</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>1200143–1201488 (1200306) </td><td>Indel (SNP) </td><td>IS (C) </td><td>IS </td><td>— </td><td>—(A) </td><td>—(A) </td><td>—(A) </td><td>(108 233) 99 </td><td>(MAQ BWA) BreakDancer </td><td><em>sll1780</em> </td><td><em>ISY203b</em> </td><td>— </td><td>Insertion of transposase (14). GT-Kazusa specific (27). MAQ and BWA detected this indel region as SNP </td></tr><tr><td>1364187 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0838</em> </td><td><em>pyrF</em> </td><td>Silent </td><td>Error of the Database (27) </td></tr><tr><td>2092571 </td><td>SNP </td><td>A </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0422</em> </td><td><em>sll0422</em> </td><td>L313* Stop codon </td><td>Error of the Database (27) </td></tr><tr><td>2198893 </td><td>SNP </td><td>T </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0142</em> </td><td><em>sll0142</em> </td><td>689 Silent </td><td>Error of the Database (27) </td></tr><tr><td>2204584 </td><td>Indel </td><td>G </td><td>G </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>slr0162</em> </td><td><em>gspF</em> </td><td><em>gspF+pilC</em> </td><td>G-insertion in GT-Kazusa strain causes split of the original <em>pilC</em> gene (34). GT-Kazusa specific (27) </td></tr><tr><td>2301721 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0168</em> </td><td><em>slr0168</em> </td><td>K403E </td><td>Error of the Database (27) </td></tr><tr><td>2350285–2350286 </td><td>Indel </td><td>— </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>317 </td><td>BWA mummer </td><td><em>intergenic region sml0001-slr0363</em> </td><td><em>psbI-slr0363</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>2360245–2360246 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>323 </td><td>BWA mummer </td><td><em>slr0364</em> </td><td><em>slr0364</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2409244 </td><td>Indel </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>sll0762</em> </td><td><em>sll0762</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2419399 </td><td>Indel </td><td>T </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>302 </td><td>BWA mummer </td><td><em>sll0752</em> </td><td><em>sll0752</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2544044–2544045 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>180 </td><td>BWA mummer </td><td><em>ssl0787</em> </td><td><em>ssl0787</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2602717 </td><td>SNP </td><td>C </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>H82Q </td><td>Error of the Database (27) </td></tr><tr><td>2602734 </td><td>SNP </td><td>T </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>I88N </td><td>Error of the Database (27) </td></tr><tr><td>2748897 </td><td>SNP </td><td>C </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region slr0210-ssr0332</em> </td><td><em>slr0210-ssr0332</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>3142651 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0045</em> </td><td><em>sps</em> </td><td>75 Silent </td><td>Error of the Database (27) </td></tr><tr><td>3260096 </td><td>Indel </td><td>C </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>Mummer </td><td><em>intergenic region sll0529-sll0528</em> </td><td><em>sll0529-sll0528</em> </td><td>— </td><td>GT-Kazusa specific (27) </td></tr><tr><td>3400322–3401506 </td><td>Indel </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>sll1474</em> </td><td><em>ISY203g</em> </td><td><em>sll1473+sll1475</em> </td><td>Insertion of transposase. IS-insertion causes split of the original <em>hik32</em> gene (14). GT-Kazusa specific (27) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>386410- 386411 (386406) </td><td>Indel (SNP) </td><td>—(T) </td><td>— </td><td>— </td><td>102 bp (A) </td><td>102 bp (A) </td><td>102 bp (A) </td><td>(68) </td><td>(MAQ) </td><td><em>slr1084</em> </td><td><em>slr1084</em> </td><td>34 amino acids deletion (V77D) </td><td>This indel region was called as SNP by MAQ as shown in parentheses. This indel region was not detected in the GT-S strain (27). CTGGGGGAAAAATGTTGGATTGATAACCTCGCCCCGGTTACCATTGAGTCCCATGTGTGTATTTCCCAGGGCGTTTACCTATGCACTGGCAACCACGATTGG </td></tr><tr><td>1192983 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>C/A </td><td>C/A </td><td>C/A </td><td> </td><td>Mummer </td><td><em>slr1855</em> </td><td><em>slr1855</em> </td><td>T167P </td><td>Potential heterogeneous nucleotide (Intensity of the peaks due to C and A were almost equal.) This SNP was not detected in the GT-S strain (27) </td></tr><tr><td>2048341–2049583 </td><td>Indel </td><td>IS </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>slr1635</em> </td><td><em>ISY203e</em> </td><td>— </td><td>Insertion of transposase (14). Specific IS in GT-Kazusa and GT-S strains (27) </td></tr></tbody></table></div><div content-id="DSR042TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>943495 </td><td>SNP </td><td>G </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1834</em> </td><td><em>psaA</em> </td><td>V604I </td><td>Smart and McIntosh (10). Error of the Database (27) </td></tr><tr><td>1012958 </td><td>SNP </td><td>G </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3177-sll1633</em> </td><td><em>repA-ftsZ</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>1200143–1201488 (1200306) </td><td>Indel (SNP) </td><td>IS (C) </td><td>IS </td><td>— </td><td>—(A) </td><td>—(A) </td><td>—(A) </td><td>(108 233) 99 </td><td>(MAQ BWA) BreakDancer </td><td><em>sll1780</em> </td><td><em>ISY203b</em> </td><td>— </td><td>Insertion of transposase (14). GT-Kazusa specific (27). MAQ and BWA detected this indel region as SNP </td></tr><tr><td>1364187 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0838</em> </td><td><em>pyrF</em> </td><td>Silent </td><td>Error of the Database (27) </td></tr><tr><td>2092571 </td><td>SNP </td><td>A </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0422</em> </td><td><em>sll0422</em> </td><td>L313* Stop codon </td><td>Error of the Database (27) </td></tr><tr><td>2198893 </td><td>SNP </td><td>T </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0142</em> </td><td><em>sll0142</em> </td><td>689 Silent </td><td>Error of the Database (27) </td></tr><tr><td>2204584 </td><td>Indel </td><td>G </td><td>G </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>slr0162</em> </td><td><em>gspF</em> </td><td><em>gspF+pilC</em> </td><td>G-insertion in GT-Kazusa strain causes split of the original <em>pilC</em> gene (34). GT-Kazusa specific (27) </td></tr><tr><td>2301721 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0168</em> </td><td><em>slr0168</em> </td><td>K403E </td><td>Error of the Database (27) </td></tr><tr><td>2350285–2350286 </td><td>Indel </td><td>— </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>317 </td><td>BWA mummer </td><td><em>intergenic region sml0001-slr0363</em> </td><td><em>psbI-slr0363</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>2360245–2360246 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>323 </td><td>BWA mummer </td><td><em>slr0364</em> </td><td><em>slr0364</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2409244 </td><td>Indel </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>mummer </td><td><em>sll0762</em> </td><td><em>sll0762</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2419399 </td><td>Indel </td><td>T </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>302 </td><td>BWA mummer </td><td><em>sll0752</em> </td><td><em>sll0752</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2544044–2544045 </td><td>Indel </td><td>— </td><td>C </td><td>C </td><td>C </td><td>C </td><td>C </td><td>180 </td><td>BWA mummer </td><td><em>ssl0787</em> </td><td><em>ssl0787</em> </td><td>Frameshift </td><td>Error of the Database (27) </td></tr><tr><td>2602717 </td><td>SNP </td><td>C </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>H82Q </td><td>Error of the Database (27) </td></tr><tr><td>2602734 </td><td>SNP </td><td>T </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0468</em> </td><td><em>slr0468</em> </td><td>I88N </td><td>Error of the Database (27) </td></tr><tr><td>2748897 </td><td>SNP </td><td>C </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region slr0210-ssr0332</em> </td><td><em>slr0210-ssr0332</em> </td><td>— </td><td>Error of the Database (27) </td></tr><tr><td>3142651 </td><td>SNP </td><td>A </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0045</em> </td><td><em>sps</em> </td><td>75 Silent </td><td>Error of the Database (27) </td></tr><tr><td>3260096 </td><td>Indel </td><td>C </td><td>C </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td><td>Mummer </td><td><em>intergenic region sll0529-sll0528</em> </td><td><em>sll0529-sll0528</em> </td><td>— </td><td>GT-Kazusa specific (27) </td></tr><tr><td>3400322–3401506 </td><td>Indel </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>sll1474</em> </td><td><em>ISY203g</em> </td><td><em>sll1473+sll1475</em> </td><td>Insertion of transposase. IS-insertion causes split of the original <em>hik32</em> gene (14). GT-Kazusa specific (27) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>386410- 386411 (386406) </td><td>Indel (SNP) </td><td>—(T) </td><td>— </td><td>— </td><td>102 bp (A) </td><td>102 bp (A) </td><td>102 bp (A) </td><td>(68) </td><td>(MAQ) </td><td><em>slr1084</em> </td><td><em>slr1084</em> </td><td>34 amino acids deletion (V77D) </td><td>This indel region was called as SNP by MAQ as shown in parentheses. This indel region was not detected in the GT-S strain (27). CTGGGGGAAAAATGTTGGATTGATAACCTCGCCCCGGTTACCATTGAGTCCCATGTGTGTATTTCCCAGGGCGTTTACCTATGCACTGGCAACCACGATTGG </td></tr><tr><td>1192983 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>C/A </td><td>C/A </td><td>C/A </td><td> </td><td>Mummer </td><td><em>slr1855</em> </td><td><em>slr1855</em> </td><td>T167P </td><td>Potential heterogeneous nucleotide (Intensity of the peaks due to C and A were almost equal.) This SNP was not detected in the GT-S strain (27) </td></tr><tr><td>2048341–2049583 </td><td>Indel </td><td>IS </td><td>IS </td><td>IS </td><td>— </td><td>— </td><td>— </td><td>99 </td><td>BreakDancer </td><td><em>slr1635</em> </td><td><em>ISY203e</em> </td><td>— </td><td>Insertion of transposase (14). Specific IS in GT-Kazusa and GT-S strains (27) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The left column shows the genomic locus of each mutation in the database (NCBI accession number; NC_000911). Quality scores indicate the phred-scaled scores called by MAQ and BWA, respectively. Quality scores given by BreakDancer is a software-original value. The upper table listed the mutations that were suggested as the error of the database and also that the GT-Kazusa strain-specific mutations such as <em>ISY203b</em>, <em>ISY203g</em>, and the locus 2204584 (31). Lower table shows additional differences found only in GT-I, PCC-P and PCC-N strains. Greyed columns emphasize the different sites and their details. Several indel regions miscalled as SNP by MAQ and BWA were shown in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7394630" target="_blank">View Large</a></div></div>                        <a id="7394632" scrollto-destination="7394632"></a>
<div content-id="DSR042TB3" class="table-modal"><div class="table-wrap"><div id="DSR042TB3" class="table-wrap-title" data-id="DSR042TB3"><div class="label -label">Table 3.</div><div class="caption"><p>List of the genomic loci of SNPs and indels found in the specific strains compared with the nucleotide sequence in the database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>126257 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>D63N </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>731367 </td><td>Indel </td><td>T </td><td>T </td><td>T </td><td>T </td><td>— </td><td>— </td><td>155 </td><td>BWA mummer </td><td><em>sll1574</em> </td><td><em>sll1574</em> </td><td><em>sll1574+sll1575</em> </td><td>T insertion in GT strains causes gene split of the original <em>spkA</em> gene. Different site between GT/ATCC strains and PCC strains (12) </td></tr><tr><td>781625–781626 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>154 bp </td><td>154 bp </td><td>— </td><td>— </td><td><em>intergenic region slr2030-slr2031</em> </td><td><em>slr2030-slr2031</em> </td><td><em>—</em> </td><td>Different site between GT strains and PCC strains (13). TTTAAACGTCATGCACCAATCTCTGATTTACTGGTTTATTCATCTATCAATTCCATAGGCTTTTTGCTTCATCGCTCCAACTAACTTTTCTGGGATGTCCTCCATGCCCCCCGTGCCTAGCTTACCGTCCACCGATGCCGTTATTCCCCCCGGC </td></tr><tr><td>831647 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3441-sll1815</em> </td><td><em>infA-adk</em> </td><td>— </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1204616 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1865</em> </td><td><em>slr1865</em> </td><td>C114Y </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1300941–1300985 (1300977) </td><td>Indel (SNP) </td><td>45 bp </td><td>45 bp </td><td>45 bp </td><td>45 bp (C) </td><td>—(T) </td><td>—(T) </td><td>(164 255) </td><td>(MAQ BWA) </td><td><em>slr1819</em> </td><td><em>slr1819</em> </td><td>15 amino acids deletion </td><td>Putative PCC strains-specific 45bp deletion without frameshift. This indel region was called as SNP by MAQ and BWA as shown in parentheses. GGGCTATCCTGCGGGATAGCGACATGACCCTGGCCACTCTCCAGG </td></tr><tr><td>1423340–1423341 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>A </td><td>A </td><td>386 </td><td>BWA mummer </td><td><em>sll1951</em> </td><td><em>sll1951</em> </td><td>N1438* Stop codon </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1437389 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1993</em> </td><td><em>thl</em> </td><td>N6S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1812419 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1983</em> </td><td><em>slr1983</em> </td><td>A225V </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2521013 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0222</em> </td><td><em>slr0222</em> </td><td>F898S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2736514–2736515 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>T </td><td>257 </td><td>BWA mummer </td><td><em>sll0182</em> </td><td><em>sll0182</em> </td><td>Frameshift </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3014665 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0302</em> </td><td><em>slr0302</em> </td><td>92 Silent </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3096187 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>66 </td><td>MAQ </td><td><em>ssr1175</em> </td><td><em>ssr1175</em> </td><td>I47T </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3098707 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>217 224 </td><td>MAQ BWA </td><td><em>ssr1176</em> </td><td><em>ssr1176</em> </td><td>C95R </td><td>Different site between GT strains and PCC strains. Potential heterogenous nucleotide (Small T peak was also detected in the PCC strains) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>387006 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1085</em> </td><td><em>slr1085</em> </td><td>P109L </td><td>GT-I strain-specific </td></tr><tr><td>842060 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1799</em> </td><td><em>rplC</em> </td><td>R185Q </td><td>GT-I strain-specific </td></tr><tr><td>909360 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1968</em> </td><td><em>pmgA</em> </td><td>E93K </td><td>GT-I strain-specific </td></tr><tr><td>1392586 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1250</em> </td><td><em>pstB</em> </td><td>L204S </td><td>GT-I strain-specific </td></tr><tr><td>1470212 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1605</em> </td><td><em>fabZ</em> </td><td>R46C </td><td>GT-I strain-specific </td></tr><tr><td>1764198 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>G </td><td>T </td><td>T </td><td>234 244 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>F158C </td><td>GT-I strain-specific </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>125218 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>T409M </td><td>PCC-P strain-specific </td></tr><tr><td>1437136 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1992</em> </td><td><em>slr1992</em> </td><td>146 Silent </td><td>PCC-P strain-specific </td></tr><tr><td>2674108 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0645</em> </td><td><em>slr0645</em> </td><td>A3V </td><td>PCC-P strain-specific </td></tr><tr><td>69849 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1119</em> </td><td><em>slr1119</em> </td><td>R189Q </td><td>PCC-N strain-specific </td></tr><tr><td>125262–125273 </td><td>Indel </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>— </td><td> </td><td>Mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>Four amino acids deletion </td><td>PCC-N strain-specific 12base deletion without frameshift. CTGGGTCAACAT </td></tr><tr><td>1597057 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1510</em> </td><td><em>plsX</em> </td><td>V88G </td><td>PCC-N strain-specific </td></tr><tr><td>1763998 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>E91D </td><td>PCC-N strain-specific </td></tr><tr><td>2370197 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0370</em> </td><td><em>gabD</em> </td><td>T306A </td><td>PCC-N strain-specific </td></tr><tr><td>2580625 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2580626 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2881614–2881615 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>269 </td><td>Bwa </td><td><em>slr0079</em> </td><td><em>gspE</em> </td><td>Frameshift </td><td>PCC-N strain-specific </td></tr></tbody></table></div><div content-id="DSR042TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>126257 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>D63N </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>731367 </td><td>Indel </td><td>T </td><td>T </td><td>T </td><td>T </td><td>— </td><td>— </td><td>155 </td><td>BWA mummer </td><td><em>sll1574</em> </td><td><em>sll1574</em> </td><td><em>sll1574+sll1575</em> </td><td>T insertion in GT strains causes gene split of the original <em>spkA</em> gene. Different site between GT/ATCC strains and PCC strains (12) </td></tr><tr><td>781625–781626 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>154 bp </td><td>154 bp </td><td>— </td><td>— </td><td><em>intergenic region slr2030-slr2031</em> </td><td><em>slr2030-slr2031</em> </td><td><em>—</em> </td><td>Different site between GT strains and PCC strains (13). TTTAAACGTCATGCACCAATCTCTGATTTACTGGTTTATTCATCTATCAATTCCATAGGCTTTTTGCTTCATCGCTCCAACTAACTTTTCTGGGATGTCCTCCATGCCCCCCGTGCCTAGCTTACCGTCCACCGATGCCGTTATTCCCCCCGGC </td></tr><tr><td>831647 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3441-sll1815</em> </td><td><em>infA-adk</em> </td><td>— </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1204616 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1865</em> </td><td><em>slr1865</em> </td><td>C114Y </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1300941–1300985 (1300977) </td><td>Indel (SNP) </td><td>45 bp </td><td>45 bp </td><td>45 bp </td><td>45 bp (C) </td><td>—(T) </td><td>—(T) </td><td>(164 255) </td><td>(MAQ BWA) </td><td><em>slr1819</em> </td><td><em>slr1819</em> </td><td>15 amino acids deletion </td><td>Putative PCC strains-specific 45bp deletion without frameshift. This indel region was called as SNP by MAQ and BWA as shown in parentheses. GGGCTATCCTGCGGGATAGCGACATGACCCTGGCCACTCTCCAGG </td></tr><tr><td>1423340–1423341 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>A </td><td>A </td><td>386 </td><td>BWA mummer </td><td><em>sll1951</em> </td><td><em>sll1951</em> </td><td>N1438* Stop codon </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1437389 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1993</em> </td><td><em>thl</em> </td><td>N6S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1812419 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1983</em> </td><td><em>slr1983</em> </td><td>A225V </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2521013 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0222</em> </td><td><em>slr0222</em> </td><td>F898S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2736514–2736515 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>T </td><td>257 </td><td>BWA mummer </td><td><em>sll0182</em> </td><td><em>sll0182</em> </td><td>Frameshift </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3014665 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0302</em> </td><td><em>slr0302</em> </td><td>92 Silent </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3096187 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>66 </td><td>MAQ </td><td><em>ssr1175</em> </td><td><em>ssr1175</em> </td><td>I47T </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3098707 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>217 224 </td><td>MAQ BWA </td><td><em>ssr1176</em> </td><td><em>ssr1176</em> </td><td>C95R </td><td>Different site between GT strains and PCC strains. Potential heterogenous nucleotide (Small T peak was also detected in the PCC strains) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>387006 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1085</em> </td><td><em>slr1085</em> </td><td>P109L </td><td>GT-I strain-specific </td></tr><tr><td>842060 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1799</em> </td><td><em>rplC</em> </td><td>R185Q </td><td>GT-I strain-specific </td></tr><tr><td>909360 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1968</em> </td><td><em>pmgA</em> </td><td>E93K </td><td>GT-I strain-specific </td></tr><tr><td>1392586 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1250</em> </td><td><em>pstB</em> </td><td>L204S </td><td>GT-I strain-specific </td></tr><tr><td>1470212 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1605</em> </td><td><em>fabZ</em> </td><td>R46C </td><td>GT-I strain-specific </td></tr><tr><td>1764198 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>G </td><td>T </td><td>T </td><td>234 244 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>F158C </td><td>GT-I strain-specific </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>125218 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>T409M </td><td>PCC-P strain-specific </td></tr><tr><td>1437136 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1992</em> </td><td><em>slr1992</em> </td><td>146 Silent </td><td>PCC-P strain-specific </td></tr><tr><td>2674108 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0645</em> </td><td><em>slr0645</em> </td><td>A3V </td><td>PCC-P strain-specific </td></tr><tr><td>69849 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1119</em> </td><td><em>slr1119</em> </td><td>R189Q </td><td>PCC-N strain-specific </td></tr><tr><td>125262–125273 </td><td>Indel </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>— </td><td> </td><td>Mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>Four amino acids deletion </td><td>PCC-N strain-specific 12base deletion without frameshift. CTGGGTCAACAT </td></tr><tr><td>1597057 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1510</em> </td><td><em>plsX</em> </td><td>V88G </td><td>PCC-N strain-specific </td></tr><tr><td>1763998 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>E91D </td><td>PCC-N strain-specific </td></tr><tr><td>2370197 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0370</em> </td><td><em>gabD</em> </td><td>T306A </td><td>PCC-N strain-specific </td></tr><tr><td>2580625 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2580626 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2881614–2881615 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>269 </td><td>Bwa </td><td><em>slr0079</em> </td><td><em>gspE</em> </td><td>Frameshift </td><td>PCC-N strain-specific </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The left column shows the genomic locus of each mutation in the database (NCBI accession number; NC_000911). Quality scores indicate the phred-scaled scores called by MAQ and BWA, respectively. Greyed columns emphasize the different sites and their details. Several indel regions miscalled as SNP by MAQ and BWA were shown in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7394632" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR042TB3" class="table-wrap-title" data-id="DSR042TB3"><div class="label -label">Table 3.</div><div class="caption"><p>List of the genomic loci of SNPs and indels found in the specific strains compared with the nucleotide sequence in the database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>126257 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>D63N </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>731367 </td><td>Indel </td><td>T </td><td>T </td><td>T </td><td>T </td><td>— </td><td>— </td><td>155 </td><td>BWA mummer </td><td><em>sll1574</em> </td><td><em>sll1574</em> </td><td><em>sll1574+sll1575</em> </td><td>T insertion in GT strains causes gene split of the original <em>spkA</em> gene. Different site between GT/ATCC strains and PCC strains (12) </td></tr><tr><td>781625–781626 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>154 bp </td><td>154 bp </td><td>— </td><td>— </td><td><em>intergenic region slr2030-slr2031</em> </td><td><em>slr2030-slr2031</em> </td><td><em>—</em> </td><td>Different site between GT strains and PCC strains (13). TTTAAACGTCATGCACCAATCTCTGATTTACTGGTTTATTCATCTATCAATTCCATAGGCTTTTTGCTTCATCGCTCCAACTAACTTTTCTGGGATGTCCTCCATGCCCCCCGTGCCTAGCTTACCGTCCACCGATGCCGTTATTCCCCCCGGC </td></tr><tr><td>831647 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3441-sll1815</em> </td><td><em>infA-adk</em> </td><td>— </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1204616 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1865</em> </td><td><em>slr1865</em> </td><td>C114Y </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1300941–1300985 (1300977) </td><td>Indel (SNP) </td><td>45 bp </td><td>45 bp </td><td>45 bp </td><td>45 bp (C) </td><td>—(T) </td><td>—(T) </td><td>(164 255) </td><td>(MAQ BWA) </td><td><em>slr1819</em> </td><td><em>slr1819</em> </td><td>15 amino acids deletion </td><td>Putative PCC strains-specific 45bp deletion without frameshift. This indel region was called as SNP by MAQ and BWA as shown in parentheses. GGGCTATCCTGCGGGATAGCGACATGACCCTGGCCACTCTCCAGG </td></tr><tr><td>1423340–1423341 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>A </td><td>A </td><td>386 </td><td>BWA mummer </td><td><em>sll1951</em> </td><td><em>sll1951</em> </td><td>N1438* Stop codon </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1437389 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1993</em> </td><td><em>thl</em> </td><td>N6S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1812419 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1983</em> </td><td><em>slr1983</em> </td><td>A225V </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2521013 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0222</em> </td><td><em>slr0222</em> </td><td>F898S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2736514–2736515 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>T </td><td>257 </td><td>BWA mummer </td><td><em>sll0182</em> </td><td><em>sll0182</em> </td><td>Frameshift </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3014665 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0302</em> </td><td><em>slr0302</em> </td><td>92 Silent </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3096187 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>66 </td><td>MAQ </td><td><em>ssr1175</em> </td><td><em>ssr1175</em> </td><td>I47T </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3098707 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>217 224 </td><td>MAQ BWA </td><td><em>ssr1176</em> </td><td><em>ssr1176</em> </td><td>C95R </td><td>Different site between GT strains and PCC strains. Potential heterogenous nucleotide (Small T peak was also detected in the PCC strains) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>387006 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1085</em> </td><td><em>slr1085</em> </td><td>P109L </td><td>GT-I strain-specific </td></tr><tr><td>842060 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1799</em> </td><td><em>rplC</em> </td><td>R185Q </td><td>GT-I strain-specific </td></tr><tr><td>909360 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1968</em> </td><td><em>pmgA</em> </td><td>E93K </td><td>GT-I strain-specific </td></tr><tr><td>1392586 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1250</em> </td><td><em>pstB</em> </td><td>L204S </td><td>GT-I strain-specific </td></tr><tr><td>1470212 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1605</em> </td><td><em>fabZ</em> </td><td>R46C </td><td>GT-I strain-specific </td></tr><tr><td>1764198 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>G </td><td>T </td><td>T </td><td>234 244 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>F158C </td><td>GT-I strain-specific </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>125218 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>T409M </td><td>PCC-P strain-specific </td></tr><tr><td>1437136 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1992</em> </td><td><em>slr1992</em> </td><td>146 Silent </td><td>PCC-P strain-specific </td></tr><tr><td>2674108 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0645</em> </td><td><em>slr0645</em> </td><td>A3V </td><td>PCC-P strain-specific </td></tr><tr><td>69849 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1119</em> </td><td><em>slr1119</em> </td><td>R189Q </td><td>PCC-N strain-specific </td></tr><tr><td>125262–125273 </td><td>Indel </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>— </td><td> </td><td>Mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>Four amino acids deletion </td><td>PCC-N strain-specific 12base deletion without frameshift. CTGGGTCAACAT </td></tr><tr><td>1597057 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1510</em> </td><td><em>plsX</em> </td><td>V88G </td><td>PCC-N strain-specific </td></tr><tr><td>1763998 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>E91D </td><td>PCC-N strain-specific </td></tr><tr><td>2370197 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0370</em> </td><td><em>gabD</em> </td><td>T306A </td><td>PCC-N strain-specific </td></tr><tr><td>2580625 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2580626 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2881614–2881615 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>269 </td><td>Bwa </td><td><em>slr0079</em> </td><td><em>gspE</em> </td><td>Frameshift </td><td>PCC-N strain-specific </td></tr></tbody></table></div><div content-id="DSR042TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Genomic loci </th><th>Type </th><th>Data base </th><th>GT-Kazusa </th><th>GT-S strain </th><th>GT-I strain </th><th>PCC-P strain </th><th>PCC-N strain </th><th>Quality score </th><th>Source </th><th>Gene ID </th><th>Annotation </th><th>Amino acid change </th><th>Comment </th></tr></thead><tbody><tr><td>126257 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>D63N </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>731367 </td><td>Indel </td><td>T </td><td>T </td><td>T </td><td>T </td><td>— </td><td>— </td><td>155 </td><td>BWA mummer </td><td><em>sll1574</em> </td><td><em>sll1574</em> </td><td><em>sll1574+sll1575</em> </td><td>T insertion in GT strains causes gene split of the original <em>spkA</em> gene. Different site between GT/ATCC strains and PCC strains (12) </td></tr><tr><td>781625–781626 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>154 bp </td><td>154 bp </td><td>— </td><td>— </td><td><em>intergenic region slr2030-slr2031</em> </td><td><em>slr2030-slr2031</em> </td><td><em>—</em> </td><td>Different site between GT strains and PCC strains (13). TTTAAACGTCATGCACCAATCTCTGATTTACTGGTTTATTCATCTATCAATTCCATAGGCTTTTTGCTTCATCGCTCCAACTAACTTTTCTGGGATGTCCTCCATGCCCCCCGTGCCTAGCTTACCGTCCACCGATGCCGTTATTCCCCCCGGC </td></tr><tr><td>831647 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl3441-sll1815</em> </td><td><em>infA-adk</em> </td><td>— </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1204616 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1865</em> </td><td><em>slr1865</em> </td><td>C114Y </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1300941–1300985 (1300977) </td><td>Indel (SNP) </td><td>45 bp </td><td>45 bp </td><td>45 bp </td><td>45 bp (C) </td><td>—(T) </td><td>—(T) </td><td>(164 255) </td><td>(MAQ BWA) </td><td><em>slr1819</em> </td><td><em>slr1819</em> </td><td>15 amino acids deletion </td><td>Putative PCC strains-specific 45bp deletion without frameshift. This indel region was called as SNP by MAQ and BWA as shown in parentheses. GGGCTATCCTGCGGGATAGCGACATGACCCTGGCCACTCTCCAGG </td></tr><tr><td>1423340–1423341 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>A </td><td>A </td><td>386 </td><td>BWA mummer </td><td><em>sll1951</em> </td><td><em>sll1951</em> </td><td>N1438* Stop codon </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1437389 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1993</em> </td><td><em>thl</em> </td><td>N6S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>1812419 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1983</em> </td><td><em>slr1983</em> </td><td>A225V </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2521013 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0222</em> </td><td><em>slr0222</em> </td><td>F898S </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>2736514–2736515 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>T </td><td>257 </td><td>BWA mummer </td><td><em>sll0182</em> </td><td><em>sll0182</em> </td><td>Frameshift </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3014665 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0302</em> </td><td><em>slr0302</em> </td><td>92 Silent </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3096187 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>66 </td><td>MAQ </td><td><em>ssr1175</em> </td><td><em>ssr1175</em> </td><td>I47T </td><td>Different site between GT strains and PCC strains </td></tr><tr><td>3098707 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>C </td><td>C </td><td>217 224 </td><td>MAQ BWA </td><td><em>ssr1176</em> </td><td><em>ssr1176</em> </td><td>C95R </td><td>Different site between GT strains and PCC strains. Potential heterogenous nucleotide (Small T peak was also detected in the PCC strains) </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>387006 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1085</em> </td><td><em>slr1085</em> </td><td>P109L </td><td>GT-I strain-specific </td></tr><tr><td>842060 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1799</em> </td><td><em>rplC</em> </td><td>R185Q </td><td>GT-I strain-specific </td></tr><tr><td>909360 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1968</em> </td><td><em>pmgA</em> </td><td>E93K </td><td>GT-I strain-specific </td></tr><tr><td>1392586 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>C </td><td>T </td><td>T </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1250</em> </td><td><em>pstB</em> </td><td>L204S </td><td>GT-I strain-specific </td></tr><tr><td>1470212 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll1605</em> </td><td><em>fabZ</em> </td><td>R46C </td><td>GT-I strain-specific </td></tr><tr><td>1764198 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>G </td><td>T </td><td>T </td><td>234 244 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>F158C </td><td>GT-I strain-specific </td></tr><tr><td>Genomic loci </td><td>Type </td><td>Data base </td><td>GT-Kazusa </td><td>GT-S strain </td><td>GT-I strain </td><td>PCC-P strain </td><td>PCC-N strain </td><td>Quality score </td><td>Source </td><td>Gene ID </td><td>Annotation </td><td>Amino acid change </td><td>Comment </td></tr><tr><td>125218 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>T409M </td><td>PCC-P strain-specific </td></tr><tr><td>1437136 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1992</em> </td><td><em>slr1992</em> </td><td>146 Silent </td><td>PCC-P strain-specific </td></tr><tr><td>2674108 </td><td>SNP </td><td>C </td><td>C </td><td>C </td><td>C </td><td>T </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0645</em> </td><td><em>slr0645</em> </td><td>A3V </td><td>PCC-P strain-specific </td></tr><tr><td>69849 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1119</em> </td><td><em>slr1119</em> </td><td>R189Q </td><td>PCC-N strain-specific </td></tr><tr><td>125262–125273 </td><td>Indel </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>12 bp </td><td>— </td><td> </td><td>Mummer </td><td><em>sll0698</em> </td><td><em>hik33</em> </td><td>Four amino acids deletion </td><td>PCC-N strain-specific 12base deletion without frameshift. CTGGGTCAACAT </td></tr><tr><td>1597057 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1510</em> </td><td><em>plsX</em> </td><td>V88G </td><td>PCC-N strain-specific </td></tr><tr><td>1763998 </td><td>SNP </td><td>G </td><td>G </td><td>G </td><td>G </td><td>G </td><td>C </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr1962</em> </td><td><em>slr1962</em> </td><td>E91D </td><td>PCC-N strain-specific </td></tr><tr><td>2370197 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>slr0370</em> </td><td><em>gabD</em> </td><td>T306A </td><td>PCC-N strain-specific </td></tr><tr><td>2580625 </td><td>SNP </td><td>T </td><td>T </td><td>T </td><td>T </td><td>T </td><td>A </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2580626 </td><td>SNP </td><td>A </td><td>A </td><td>A </td><td>A </td><td>A </td><td>G </td><td>255 255 </td><td>MAQ BWA mummer </td><td><em>intergenic region ssl0105-sll0063</em> </td><td><em>ssl0105-sll0063</em> </td><td>— </td><td>PCC-N strain-specific </td></tr><tr><td>2881614–2881615 </td><td>Indel </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>T </td><td>269 </td><td>Bwa </td><td><em>slr0079</em> </td><td><em>gspE</em> </td><td>Frameshift </td><td>PCC-N strain-specific </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The left column shows the genomic locus of each mutation in the database (NCBI accession number; NC_000911). Quality scores indicate the phred-scaled scores called by MAQ and BWA, respectively. Greyed columns emphasize the different sites and their details. Several indel regions miscalled as SNP by MAQ and BWA were shown in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7394632" target="_blank">View Large</a></div></div><p>A part of the indel regions was miss-called as SNPs by the mapping programmes. We aligned the genome sequence of the substrains around the indels (Fig. <a reveal-id="DSR042F3" data-open="DSR042F3" class="link link-reveal link-table xref-fig">3</a>) and found that these indels were located in the middle of two direct repeat- or direct repeat-like sequences. Interestingly, these direct repeats found in the deleted regions were not common sequence. Mapping analyses using <em>de novo</em> assembled contigs (Velvet and MUMmer) help to correct the false-positive SNP/indel-calls made by mapping programmes such as BWA and MAQ (Table <a reveal-id="DSR042TB2" data-open="DSR042TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="7394635" scrollto-destination="7394635"></a>
<div id="DSR042F3" data-id="DSR042F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04203.jpeg?Expires=2147483647&Signature=DWxeJ~-AscaoIj4CSnlYXj8SlU4QNnAef3MMVevVoJbnikkW36rS6TiAo0dBde1wIzP4J75BBvrXE2FKifB85ho-4sCT8tTBNsjYEj6fYWSvjEZy8xXewQnmJnHBFt5rMSvhecoAgbwNYk5Vw4a2Lh-4REYTwd1WuLn~F08JIaYdYzpNJMd48qzKUw0-M~GagfvC9xUqrKEgLrQrXEpI6vSc2yPEwKoFymYUDS8t6YTOpD7aibL1l~Ta0F8BUk3B08NxPCYi~jWLbMEr-jML3nX1nnbAyoCSigWXQMF9tcX7ySs7CqXEa7zk1PxPkU2V-v9HAX-YkVOUewjSs1uZ1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the specific indel regions whose consensus read bases were miss-called. (A) The 154-base deletion in the slr2031 gene13 in the GT strains. (B) The 12-base deletion in the hik33 gene in the PCC-N strain. (C) The 45-base deletion in the slr1819 gene in PCC-P and PCC-N strains. (D) The 102-base deletion in the slr1084 gene in the GT-S and Kazusa strains. Deleted regions were underlined and direct-repeat sequences were emphasized by grey colour. These deleted loci were situated in the middle of the direct-repeat sequences." path-from-xml="dsr04203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394635/dsr04203.jpeg" path-from-xml="dsr04203.jpeg" target="_blank">View large</a><a section="7394635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04203.jpeg?Expires=2147483647&Signature=QDbxsDBMsAfilsvLYacn-tF0keh1kUz3clah1dZpeKA13YPt4qWUgxPzg82JpCSnNKS9v4uLWNyYhIsug1qo2P3XMBteyx9TKndqc7JtHLoARR0EC99aVkdWTGEhfnFrUYtQcE5~mLr~~-EdX3uIG9FmO-7FRwz45Fgj5R-q4DBKiWLS5ZC8VSP20~jIxexmyCF4~gIo32t8iqsBFH7QL2tNbDwkEpKlRYl1tYCc6JGIr~BGDlmE7l7F0iMPqC--bHZr4lQjkGHWRo5-po9vf01k0SGF5ai5yCQsqQwRcyJ-B3myOEnqa0uZxJQsMqTA7qcX8euM-JWxpq~HFAzyXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394635&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of the specific indel regions whose consensus read bases were miss-called. (A) The 154-base deletion in the <em>slr2031</em> gene<sup><a reveal-id="DSR042C13" data-open="DSR042C13" class="link link-ref link-reveal xref-bibr">13</a></sup> in the GT strains. (B) The 12-base deletion in the <em>hik33</em> gene in the PCC-N strain. (C) The 45-base deletion in the <em>slr1819</em> gene in PCC-P and PCC-N strains. (D) The 102-base deletion in the <em>slr1084</em> gene in the GT-S and Kazusa strains. Deleted regions were underlined and direct-repeat sequences were emphasized by grey colour. These deleted loci were situated in the middle of the direct-repeat sequences.</p></div></div><div content-id="DSR042F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04203.jpeg?Expires=2147483647&Signature=DWxeJ~-AscaoIj4CSnlYXj8SlU4QNnAef3MMVevVoJbnikkW36rS6TiAo0dBde1wIzP4J75BBvrXE2FKifB85ho-4sCT8tTBNsjYEj6fYWSvjEZy8xXewQnmJnHBFt5rMSvhecoAgbwNYk5Vw4a2Lh-4REYTwd1WuLn~F08JIaYdYzpNJMd48qzKUw0-M~GagfvC9xUqrKEgLrQrXEpI6vSc2yPEwKoFymYUDS8t6YTOpD7aibL1l~Ta0F8BUk3B08NxPCYi~jWLbMEr-jML3nX1nnbAyoCSigWXQMF9tcX7ySs7CqXEa7zk1PxPkU2V-v9HAX-YkVOUewjSs1uZ1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of the specific indel regions whose consensus read bases were miss-called. (A) The 154-base deletion in the slr2031 gene13 in the GT strains. (B) The 12-base deletion in the hik33 gene in the PCC-N strain. (C) The 45-base deletion in the slr1819 gene in PCC-P and PCC-N strains. (D) The 102-base deletion in the slr1084 gene in the GT-S and Kazusa strains. Deleted regions were underlined and direct-repeat sequences were emphasized by grey colour. These deleted loci were situated in the middle of the direct-repeat sequences." path-from-xml="dsr04203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394635/dsr04203.jpeg" path-from-xml="dsr04203.jpeg" target="_blank">View large</a><a section="7394635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04203.jpeg?Expires=2147483647&Signature=QDbxsDBMsAfilsvLYacn-tF0keh1kUz3clah1dZpeKA13YPt4qWUgxPzg82JpCSnNKS9v4uLWNyYhIsug1qo2P3XMBteyx9TKndqc7JtHLoARR0EC99aVkdWTGEhfnFrUYtQcE5~mLr~~-EdX3uIG9FmO-7FRwz45Fgj5R-q4DBKiWLS5ZC8VSP20~jIxexmyCF4~gIo32t8iqsBFH7QL2tNbDwkEpKlRYl1tYCc6JGIr~BGDlmE7l7F0iMPqC--bHZr4lQjkGHWRo5-po9vf01k0SGF5ai5yCQsqQwRcyJ-B3myOEnqa0uZxJQsMqTA7qcX8euM-JWxpq~HFAzyXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394635&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of the specific indel regions whose consensus read bases were miss-called. (A) The 154-base deletion in the <em>slr2031</em> gene<sup><a reveal-id="DSR042C13" data-open="DSR042C13" class="link link-ref link-reveal xref-bibr">13</a></sup> in the GT strains. (B) The 12-base deletion in the <em>hik33</em> gene in the PCC-N strain. (C) The 45-base deletion in the <em>slr1819</em> gene in PCC-P and PCC-N strains. (D) The 102-base deletion in the <em>slr1084</em> gene in the GT-S and Kazusa strains. Deleted regions were underlined and direct-repeat sequences were emphasized by grey colour. These deleted loci were situated in the middle of the direct-repeat sequences.</p></div></div><p>Some of the mutations confirmed by the Sanger method were putative heterogeneous SNPs. At these SNP loci, we detected a second peak from the other nucleotide. As the <em>Synechocystis</em> genome is multi-copy, some of the genome copies may harbour heterogeneous SNPs. These loci were not in a multi-repeat region of the genome.</p><p>A phylogenetic scheme of the history of the <em>Synechocystis</em> sp. PCC 6803 substrains is presented in Fig. <a reveal-id="DSR042F4" data-open="DSR042F4" class="link link-reveal link-table xref-fig">4</a>. The predicted phylogenetic relationship of various strains indicated that the putative root may exist between the PCC branch and the GT branch. It suggests that all existing substrains do not have the original sequence of this organism isolated in 1968. </p>                        <a id="7394638" scrollto-destination="7394638"></a>
<div id="DSR042F4" data-id="DSR042F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04204.jpeg?Expires=2147483647&Signature=q5WdQJddnwTjBduLoYw1vWIJXyQnuVmLlRBTCpFCqxvKZN8I1jHMMqSAA9GDZoRuDzyK368XEuFJA0gjFaKv-R2wtTGAnzM-BTWt7IdP1emhthLGKb2gQB2kyzkmPBqyClzLxtFk2bmr57l4uMgvhEsohfRfuT8BdOo3~RxnIXf1v64cY6qFMKqzJIpB0OZSUfouFJHgxLfmltO6CwoKelCTT7Ri42ewU7bf~u1nPizUryzsx60ficc7d~lenF8576qC-VtaYEEkwdTx8q-2vYY1XK-QEPp7q4aDtuG5nY1Tju5R8zxD-s6c2wqad9oI284jFCUYIFsZRsDl1xv22Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unrooted tree of phylogenetic relationship of various strains of Synechocystis sp. PCC 6803. Known events are indicated on each branch. The number of mutations in each substrain to the database sequence (Kazusa strain) was indicated. The scale bar indicates the distance of branch corresponding to the number of mutations." path-from-xml="dsr04204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394638/dsr04204.jpeg" path-from-xml="dsr04204.jpeg" target="_blank">View large</a><a section="7394638" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04204.jpeg?Expires=2147483647&Signature=yjodjaD1ogGMKSpbG0eBN87JOuXKj9KLoyzNqjFBIHVRgM4bcdCf52XJ3FQPTXyhleao02ut-pElWEbmDwZ-tGw00oHRjigBJNaoYMB8oorZLBNdOpajJyOVtmYFU4bYevisStgHv-ss9iGszhvhbTBxNFuf22I6cQbsepouMinn5ZBhEe4cQ-ET4E1CiVb8uT~RWMM3TVhGgP8Jc2Q-UdCsxjW0PHySNZuHN1lS95cvifjXtYIJPAcXnuFbWfv9GZJ15APdffj5ZimQDIIaZnJ6lY2E1Mw1U4Awe-VE3h1BVBE4McWT1PYEyVc-ccCTq8b6WPop11Q17rQvEmwV4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394638&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unrooted tree of phylogenetic relationship of various strains of <em>Synechocystis</em> sp. PCC 6803. Known events are indicated on each branch. The number of mutations in each substrain to the database sequence (Kazusa strain) was indicated. The scale bar indicates the distance of branch corresponding to the number of mutations.</p></div></div><div content-id="DSR042F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/m_dsr04204.jpeg?Expires=2147483647&Signature=q5WdQJddnwTjBduLoYw1vWIJXyQnuVmLlRBTCpFCqxvKZN8I1jHMMqSAA9GDZoRuDzyK368XEuFJA0gjFaKv-R2wtTGAnzM-BTWt7IdP1emhthLGKb2gQB2kyzkmPBqyClzLxtFk2bmr57l4uMgvhEsohfRfuT8BdOo3~RxnIXf1v64cY6qFMKqzJIpB0OZSUfouFJHgxLfmltO6CwoKelCTT7Ri42ewU7bf~u1nPizUryzsx60ficc7d~lenF8576qC-VtaYEEkwdTx8q-2vYY1XK-QEPp7q4aDtuG5nY1Tju5R8zxD-s6c2wqad9oI284jFCUYIFsZRsDl1xv22Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unrooted tree of phylogenetic relationship of various strains of Synechocystis sp. PCC 6803. Known events are indicated on each branch. The number of mutations in each substrain to the database sequence (Kazusa strain) was indicated. The scale bar indicates the distance of branch corresponding to the number of mutations." path-from-xml="dsr04204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7394638/dsr04204.jpeg" path-from-xml="dsr04204.jpeg" target="_blank">View large</a><a section="7394638" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr04204.jpeg?Expires=2147483647&Signature=yjodjaD1ogGMKSpbG0eBN87JOuXKj9KLoyzNqjFBIHVRgM4bcdCf52XJ3FQPTXyhleao02ut-pElWEbmDwZ-tGw00oHRjigBJNaoYMB8oorZLBNdOpajJyOVtmYFU4bYevisStgHv-ss9iGszhvhbTBxNFuf22I6cQbsepouMinn5ZBhEe4cQ-ET4E1CiVb8uT~RWMM3TVhGgP8Jc2Q-UdCsxjW0PHySNZuHN1lS95cvifjXtYIJPAcXnuFbWfv9GZJ15APdffj5ZimQDIIaZnJ6lY2E1Mw1U4Awe-VE3h1BVBE4McWT1PYEyVc-ccCTq8b6WPop11Q17rQvEmwV4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7394638&ar=485988&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unrooted tree of phylogenetic relationship of various strains of <em>Synechocystis</em> sp. PCC 6803. Known events are indicated on each branch. The number of mutations in each substrain to the database sequence (Kazusa strain) was indicated. The scale bar indicates the distance of branch corresponding to the number of mutations.</p></div></div>                            <h2 scrollto-destination=7394639 id="7394639" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=7394640 id="7394640" class="section-title" data-legacy-id=s4a>4.1. Problems associated with the heterogeneity of frozen stocks of bacterial strains</h3>
<p>Frozen stocks in culture centres are basically stored as heterogeneous cell groups of the particular bacteria, such as <em>E. coli</em> K-12 MG1655.<sup><a reveal-id="DSR042C32" data-open="DSR042C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Thus, when aliquots of the frozen cells are thawed, selection bias due to the growth conditions arises; this affects the major genotype of the cells in culture. We posit that the heterogeneity of cells in different laboratories affected the results of genetic research or phenotypic analyses and we suggest that post-genomic research on cyanobacteria in the next generation should be based on resequenced strains as the new reference for each laboratory. In studies on post-genomic science using various mutants, it is better to prepare the frozen stock cells of single colonies one by one and also of the parental cell as soon as possible after the isolation of the mutants. It is also unquestionable that long-term cultivation of the cells on the gels or in the liquid cultures is not appropriate to keep the genotype of the cells.</p>                            <h3 scrollto-destination=7394642 id="7394642" class="section-title" data-legacy-id=s4b>4.2. Potential factors that affect phenotypic differences in 6803 substrains</h3>
<p>We found a small number of substrain-specific mutations in PCC-P and PCC-N strains. It can be expected that a mutation accounts for the difference in the phototactic phenotype of PCC-P and PCC-N strains.<sup><a reveal-id="DSR042C19" data-open="DSR042C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Furthermore, the 14 mutations found in the GT/Kazusa- and the PCC-P/PCC-N strains suggest that their loci may affect the cell motility of these strains<sup><a reveal-id="DSR042C8" data-open="DSR042C8" class="link link-ref link-reveal xref-bibr">8</a></sup> or their glucose tolerance.<sup><a reveal-id="DSR042C3" data-open="DSR042C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Our findings may be useful in functional studies on the gene that is disrupted by SNPs or indels in specific substrains. As Kamei <em>et al</em>.<sup><a reveal-id="DSR042C12" data-open="DSR042C12" class="link link-ref link-reveal xref-bibr">12</a></sup> or Okamoto <em>et al</em>.<sup><a reveal-id="DSR042C14" data-open="DSR042C14" class="link link-ref link-reveal xref-bibr">14</a></sup> suggested, the real function of several genes could only be studied in specific substrains which have the original nucleotide sequence without any SNPs, indels, or IS insertions.</p>                            <h3 scrollto-destination=7394645 id="7394645" class="section-title" data-legacy-id=s4c>4.3. Indels located between two direct repeats result in miss-calls by the mapping programme</h3>
<p>Confirmation of our results with the Sanger method revealed that specific deletion patterns were miss-called or undetected at high frequency by mapping analysis of short-read type data. Several deleted regions in the middle of direct-repeat sequences were miss-called as SNPs or undetected (Tables <a reveal-id="DSR042TB2" data-open="DSR042TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR042TB3" data-open="DSR042TB3" class="link link-reveal link-table xref-fig">3</a>). Alignment of the DNA sequence of the three substrains clearly showed direct repeats on both sides of the deletion locus (Fig. <a reveal-id="DSR042F3" data-open="DSR042F3" class="link link-reveal link-table xref-fig">3</a>). After the deletion event only a single direct repeat sequence remained, leading to the hypothesis that the deletion was due to self-crossover. At present we do not know what mechanisms or factors trigger these deletions, but caution should be exercised when we perform long-term cultivation of cyanobacterial cells. This is also a technical problem of mapping analysis to detect the exact positions of SNPs and indels using massive short-read data of next-generation sequencers.</p>                            <h3 scrollto-destination=7394650 id="7394650" class="section-title" data-legacy-id=s4d>4.4. Problems raised by heterozygous and homozygous SNPs</h3>
<p>In this analysis, we found that the threshold value for SNP detection by BWA and MAQ was more than 60% of read data covering the SNP positions. However, <em>Synechocystis</em> sp. strain PCC6803 has a multi-copy genome,<sup><a reveal-id="DSR042C33" data-open="DSR042C33" class="link link-ref link-reveal xref-bibr">33</a></sup> suggesting that the genome contains unidentified heterozygous SNPs below the threshold value. It is technically difficult to identify all heterozygous SNPs in the genome; if we lower the threshold, the number of false-positive SNPs increases drastically. Mutations hidden as minor heterozygous SNPs may become major SNPs under specific conditions and affect the cell phenotype, an observation reported by Hihara and Ikeuchi<sup><a reveal-id="DSR042C16" data-open="DSR042C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and Hihara <em>et al</em>.,<sup><a reveal-id="DSR042C17" data-open="DSR042C17" class="link link-ref link-reveal xref-bibr">17</a></sup> who studied the <em>pmgA</em> mutant. The active retention of heterogeneity may be a strategy of cyanobacteria for acclimation to environmental changes. Heterogeneous SNPs found in the same locus in several substrains such as the loci 1192983 and 3098707 were the candidates to understand such mechanisms. The detection of minor heterozygous SNPs is a future problem for mapping using massive parallel sequencing.</p><p>In this study, we identified a number of differences in the genome sequence of laboratory strains and published sequence data derived from the Kazusa strain. Resequence data on PCC substrains will be useful for considering evolutionary events among <em>Synechocystis</em> intra-species. For the reconstruction of informatics in post-genomic studies on <em>Synechocystis</em> sp. PCC 6803, resequence analysis is effective and represents a powerful genetic strategy to identify potential mutation loci in spontaneous mutants with altered phenotypes.</p>                            <h2 scrollto-destination=7394656 id="7394656" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data is available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7394660 id="7394660" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported by Grants-in-Aid for Scientific Research from the Ministry of Education, Culture, Sports, Science and Technology (S0801025).</p>                            <h2 scrollto-destination=7394664 id="7394664" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR042C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haselkorn</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Genetic systems in cyanobacteria</div>, <div class="source">Methods Enzymol.</div> , <div class="year">1991</div>, vol. <div class="volume">204</div> (pg. <div class="fpage">418</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0076-6879(91)04022-G" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0076-6879(91)04022-G</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20systems%20in%20cyanobacteria&amp;author=R.%20Haselkorn&amp;publication_year=1991&amp;journal=Methods%20Enzymol.&amp;volume=204&amp;pages=418-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1658567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Porter</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">DNA transformation</div>, <div class="source">Methods Enzymol.</div> , <div class="year">1988</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">703</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0076-6879(88)67081-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0076-6879(88)67081-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20transformation&amp;author=R.D.%20Porter&amp;publication_year=1988&amp;journal=Methods%20Enzymol.&amp;volume=167&amp;pages=703-12" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3148838" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Williams</div> <div class="given-names">J.G.K.</div></div>. </div><div class="article-title">Construction of specific mutations in photosystem II photosynthetic reaction center by genetic engineering methods in <em>Synechocystis</em> 6803</div>, <div class="source">Methods Enzymol.</div> , <div class="year">1988</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">766</div>-<div class="lpage">78</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0076-6879(88)67088-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0076-6879(88)67088-1</a></div></div></div></div></div><div content-id="DSR042C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rippka</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Deruelles</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Waterbury</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Herdman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stanier</div> <div class="given-names">R.Y.</div></div>. </div><div class="article-title">Genetic assignments, strain histories and properties of pure cultures of cyanobacteria</div>, <div class="source">J. Gen. Microbiol.</div> , <div class="year">1979</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">1</div>-<div class="lpage">61</div>)</div></div></div></div><div content-id="DSR042C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kotani</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Sequence analysis of the genome of the unicellular cyanobacterium <em>Synechocystis</em> sp. strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions</div>, <div class="source">DNA Res.</div> , <div class="year">1996</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">109</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/3.3.109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/3.3.109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20the%20genome%20of%20the%20unicellular%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC6803.%20II.%20Sequence%20determination%20of%20the%20entire%20genome%20and%20assignment%20of%20potential%20protein-coding%20regions&amp;author=T.%20Kaneko&amp;author=S.%20Sato&amp;author=H.%20Kotani&amp;publication_year=1996&amp;journal=DNA%20Res.&amp;volume=3&amp;pages=109-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/3.3.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F3.3.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F3.3.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burja</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Dhamwichukorn</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">P.C.</div></div>. </div><div class="article-title">Cyanobacterial postgenomic research and systems biology</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">504</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tibtech.2003.08.008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tibtech.2003.08.008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cyanobacterial%20postgenomic%20research%20and%20systems%20biology&amp;author=A.M.%20Burja&amp;author=S.%20Dhamwichukorn&amp;author=P.C.%20Wright&amp;publication_year=2003&amp;journal=Trends%20Biotechnol.&amp;volume=21&amp;pages=504-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2003.08.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2003.08.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2003.08.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14573364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanier</div> <div class="given-names">R.Y.</div></div>,  <div class="name"><div class="surname">Kunisawa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Mandel</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cohen-Bazire</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Purification and properties of unicellular blue-green algae (order Chroococcales)</div>, <div class="source">Bacteriol. Rev.</div> , <div class="year">1971</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">171</div>-<div class="lpage">205</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20properties%20of%20unicellular%20blue-green%20algae%20(order%20Chroococcales)&amp;author=R.Y.%20Stanier&amp;author=R.%20Kunisawa&amp;author=M.%20Mandel&amp;author=G.%20Cohen-Bazire&amp;publication_year=1971&amp;journal=Bacteriol.%20Rev.&amp;volume=35&amp;pages=171-205" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4998365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>. </div><div class="article-title"><em>Synechocystis</em> sp. PCC 6803—a useful tool in the study of the genetics of cyanobacteria</div>, <div class="source">Photosynth. Res.</div> , <div class="year">2001</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">73</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1013887908680" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1013887908680</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synechocystis%20sp.%20PCC%206803%E2%80%94a%20useful%20tool%20in%20the%20study%20of%20the%20genetics%20of%20cyanobacteria&amp;author=M.%20Ikeuchi&amp;author=S.%20Tabata&amp;publication_year=2001&amp;journal=Photosynth.%20Res.&amp;volume=70&amp;pages=73-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1013887908680" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1013887908680" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1013887908680"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16228363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rippka</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Herdman</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Catalogue of strains</div>, <div class="source">Pasteur Culture Collection of Cyanobacterial Strains in Axenic Culture</div> , <div class="year">1992</div>, vol. <div class="volume">vol. I</div> <div class="publisher-loc">Paris</div><div class="publisher-name">Institut Pasteur</div></div></div></div></div><div content-id="DSR042C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smart</div> <div class="given-names">L.B.</div></div>,  <div class="name"><div class="surname">McIntosh</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Expression of photosynthesis genes in the cyanobacterium <em>Synechocystis</em> sp. PCC 6803: <em>psaA-psaB</em> and <em>psbA</em> transcripts accumulate in dark-grown cells</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">959</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF00037136" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF00037136</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20photosynthesis%20genes%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20PCC%206803%3A%20psaA-psaB%20and%20psbA%20transcripts%20accumulate%20in%20dark-grown%20cells&amp;author=L.B.%20Smart&amp;author=L.%20McIntosh&amp;publication_year=1991&amp;journal=Plant%20Mol.%20Biol.&amp;volume=17&amp;pages=959-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00037136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00037136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00037136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1932686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kamei</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Garab</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Identification of a novel gene (<em>slr2031</em>) involved in high-light resistance in the cyanobacterium <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">Photosynthesis: Mechanism and Effects</div> , <div class="year">1998</div><div class="publisher-loc">Dordrecht, The Netherlands</div><div class="publisher-name">Kluwer Academic Publishers</div>(pg. <div class="fpage">2901</div>-<div class="lpage">5</div>)</div></div></div></div><div content-id="DSR042C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kamei</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yuasa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Orikawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Geng</div> <div class="given-names">X.X.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A eukaryotic-type protein kinase, SpkA, is required for normal motility of the unicellular cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">1505</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/JB.183.5.1505-1510.2001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/JB.183.5.1505-1510.2001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20eukaryotic-type%20protein%20kinase%2C%20SpkA%2C%20is%20required%20for%20normal%20motility%20of%20the%20unicellular%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=A.%20Kamei&amp;author=T.%20Yuasa&amp;author=K.%20Orikawa&amp;author=X.X.%20Geng&amp;author=M.%20Ikeuchi&amp;publication_year=2001&amp;journal=J.%20Bacteriol.&amp;volume=183&amp;pages=1505-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.5.1505-1510.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.5.1505-1510.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.5.1505-1510.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11160079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sonoda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A possible role of 154-base pair nucleotides located upstream of ORF440 on CO<sub>2</sub> transport of Synechocystis PCC6803</div>, <div class="source">Photosynthesis: From Light to Biosphere</div> , <div class="year">1995</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">481</div>-<div class="lpage">4</div>) <div class="comment">Kluwer Academic Publishers: Dordrecht, The Netherlands,</div></div></div></div></div><div content-id="DSR042C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okamoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohmori</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Experimental analysis of recently transposed insertion sequences in the cyanobacterium <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">DNA Res.</div> , <div class="year">1999</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">265</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/6.5.265" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/6.5.265</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20analysis%20of%20recently%20transposed%20insertion%20sequences%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20PCC%206803&amp;author=S.%20Okamoto&amp;author=M.%20Ikeuchi&amp;author=M.%20Ohmori&amp;publication_year=1999&amp;journal=DNA%20Res.&amp;volume=6&amp;pages=265-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/6.5.265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F6.5.265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F6.5.265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10574452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoshihara</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Geng</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Mutational analysis of genes involved in pilus structure, motility and transformation competency in the unicellular motile cyanobacterium <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">Plant Cell. Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">63</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pce007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/pcp/pce007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20analysis%20of%20genes%20involved%20in%20pilus%20structure%2C%20motility%20and%20transformation%20competency%20in%20the%20unicellular%20motile%20cyanobacterium%20Synechocystis%20sp.%20PCC%206803&amp;author=S.%20Yoshihara&amp;author=X.%20Geng&amp;author=S.%20Okamoto&amp;publication_year=2001&amp;journal=Plant%20Cell.%20Physiol.&amp;volume=42&amp;pages=63-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pce007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpce007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpce007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11158445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mutation in a novel gene required for photomixotrophic growth leads to enhanced photoautotrophic growth of <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">Photosynth. Res.</div> , <div class="year">1997</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">129</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1005815321295" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1005815321295</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20in%20a%20novel%20gene%20required%20for%20photomixotrophic%20growth%20leads%20to%20enhanced%20photoautotrophic%20growth%20of%20Synechocystis%20sp.%20PCC%206803&amp;author=Y.%20Hihara&amp;author=M.%20Ikeuchi&amp;publication_year=1997&amp;journal=Photosynth.%20Res.&amp;volume=53&amp;pages=129-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1005879905365" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1005879905365" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1005879905365"> </span></div> </div></div></div></div></div><div content-id="DSR042C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sonoike</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A novel gene, <em>pmgA</em>, specifically regulates photosystem stoichiometry in the cyanobacterium <em>Synechocystis</em> species PCC 6803 in response to high light</div>, <div class="source">Plant Physiol.</div> , <div class="year">1998</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">1205</div>-<div class="lpage">16</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.117.4.1205" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.117.4.1205</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20gene%2C%20pmgA%2C%20specifically%20regulates%20photosystem%20stoichiometry%20in%20the%20cyanobacterium%20Synechocystis%20species%20PCC%206803%20in%20response%20to%20high%20light&amp;author=Y.%20Hihara&amp;author=K.%20Sonoike&amp;author=M.%20Ikeuchi&amp;publication_year=1998&amp;journal=Plant%20Physiol.&amp;volume=117&amp;pages=1205-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.117.4.1205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.117.4.1205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.117.4.1205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9701577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Srivatsan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">High-precision, whole-genome sequencing of laboratory strains facilitates genetic studies</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000139</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pgen.1000139" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pgen.1000139</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-precision%2C%20whole-genome%20sequencing%20of%20laboratory%20strains%20facilitates%20genetic%20studies&amp;author=A.%20Srivatsan&amp;author=Y.%20Han&amp;author=J.%20Peng&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000139" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18670626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoshihara</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Fujita</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Geng</div> <div class="given-names">X.X.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Novel putative photoreceptor and regulatory genes Required for the positive phototactic movement of the unicellular motile cyanobacterium <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2000</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">1299</div>-<div class="lpage">304</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pce010" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/pcp/pce010</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20putative%20photoreceptor%20and%20regulatory%20genes%20Required%20for%20the%20positive%20phototactic%20movement%20of%20the%20unicellular%20motile%20cyanobacterium%20Synechocystis%20sp.%20PCC%206803&amp;author=S.%20Yoshihara&amp;author=F.%20Suzuki&amp;author=H.%20Fujita&amp;author=X.X.%20Geng&amp;author=M.%20Ikeuchi&amp;publication_year=2000&amp;journal=Plant%20Cell%20Physiol.&amp;volume=41&amp;pages=1299-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pce010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpce010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpce010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11134414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1851</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.078212.108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.078212.108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20short%20DNA%20sequencing%20reads%20and%20calling%20variants%20using%20mapping%20quality%20scores&amp;author=H.%20Li&amp;author=J.%20Ruan&amp;author=R.%20Durbin&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1851-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.078212.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.078212.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.078212.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18714091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">589</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp698" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp698</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=589-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">1000 Genome Project Data Processing Subgroup, The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2078</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=1000%20Genome%20Project%20Data%20Processing%20Subgroup%2C%20The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wallis</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">McLellan</div> <div class="given-names">M.D.</div></div>, et al. </div><div class="article-title">BreakDancer: An algorithm for high-resolution mapping of genomic structural variation</div>, <div class="source">Nat. Methods</div> , <div class="year">2009</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">677</div>-<div class="lpage">81</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1363" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1363</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BreakDancer%3A%20An%20algorithm%20for%20high-resolution%20mapping%20of%20genomic%20structural%20variation&amp;author=K.%20Chen&amp;author=J.W.%20Wallis&amp;author=M.D.%20McLellan&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=677-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19668202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: Algorithms for <em>de novo</em> short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">821</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.074492.107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.074492.107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20Algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div> </div></div></div></div></div><div content-id="DSR042C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>, et al. </div><div class="article-title">Open source MUMmer 3.0 is described in ‘Versatile and open software for comparing large genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">R12</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2004-5-2-r12" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2004-5-2-r12</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Open%20source%20MUMmer%203.0%20is%20described%20in%20%E2%80%98Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&amp;author=S.%20Kurtz&amp;author=A.%20Phillippy&amp;author=A.L.%20Delcher&amp;publication_year=2004&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-2-r12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-2-r12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">N.V.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">N.</div></div>. </div><div class="article-title">CyanoClust: comparative genome resources of cyanobacteria and plastids</div>, <div class="source">Database (Oxford)</div> , <div class="year">2010</div> <div class="comment">bap025</div></div></div></div></div><div content-id="DSR042C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tajima</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Maruyama</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genomic structure of the cyanobacterium <em>Synechocystis</em> sp. PCC 6803 strain GT-S</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div><div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsr026" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsr026</a></div></div></div></div></div><div content-id="DSR042C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J.</div></div>. </div><div class="article-title">PHYLIP—Phylogeny Inference Package (Version 3.2)</div>, <div class="source">Cladistics</div> , <div class="year">1989</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">164</div>-<div class="lpage">6</div>)</div></div></div></div><div content-id="DSR042C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanesaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Paithoonrangsarid</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Histidine kinases play important roles in the perception and signal transduction of hydrogen peroxide in the cyanobacterium <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">313</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2006.02959.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-313X.2006.02959.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Histidine%20kinases%20play%20important%20roles%20in%20the%20perception%20and%20signal%20transduction%20of%20hydrogen%20peroxide%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20PCC%206803&amp;author=Y.%20Kanesaki&amp;author=H.%20Yamamoto&amp;author=K.%20Paithoonrangsarid&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=49&amp;pages=313-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2006.02959.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2006.02959.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2006.02959.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17181779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paithoonrangsarid</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shoumskaya</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Kanesaki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Five histidine kinases perceive osmotic stress and regulate distinct sets of genes in <em>Synechocystis</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">53078</div>-<div class="lpage">86</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.M410162200" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.M410162200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Five%20histidine%20kinases%20perceive%20osmotic%20stress%20and%20regulate%20distinct%20sets%20of%20genes%20in%20Synechocystis&amp;author=K.%20Paithoonrangsarid&amp;author=M.A.%20Shoumskaya&amp;author=Y.%20Kanesaki&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=53078-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M410162200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M410162200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M410162200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15471853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kanesaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mikami</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Cold-regulated genes under control of the cold sensor Hik33 in <em>Synechocystis</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">235</div>-<div class="lpage">44</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-2958.2001.02379.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-2958.2001.02379.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cold-regulated%20genes%20under%20control%20of%20the%20cold%20sensor%20Hik33%20in%20Synechocystis&amp;author=I.%20Suzuki&amp;author=Y.%20Kanesaki&amp;author=K.%20Mikami&amp;author=M.%20Kanehisa&amp;author=N.%20Murata&amp;publication_year=2001&amp;journal=Mol.%20Microbiol.&amp;volume=40&amp;pages=235-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02379.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2001.02379.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2001.02379.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11298290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nahku</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Peebo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Valgepea</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Barrick</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Adamberg</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Vilu</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Stock culture heterogeneity rather than new mutational variation complicates short-term cell physiology studies of <em>Escherichia coli</em> K-12 MG1655 in continuous culture</div>, <div class="source">Microbiology</div> , <div class="year">2011</div>, vol. <div class="volume">157</div> (pg. <div class="fpage">2604</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1099/mic.0.050658-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1099/mic.0.050658-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stock%20culture%20heterogeneity%20rather%20than%20new%20mutational%20variation%20complicates%20short-term%20cell%20physiology%20studies%20of%20Escherichia%20coli%20K-12%20MG1655%20in%20continuous%20culture&amp;author=R.%20Nahku&amp;author=K.%20Peebo&amp;author=K.%20Valgepea&amp;author=J.E.%20Barrick&amp;author=K.%20Adamberg&amp;author=R.%20Vilu&amp;publication_year=2011&amp;journal=Microbiology&amp;volume=157&amp;pages=2604-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.050658-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.050658-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.050658-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chauvat</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Rouet</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bottiu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Boussac</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Mutagenesis by random cloning of an <em>Escherichia coli</em> kanamycin resistance gene into the genome of the cyanobacterium <em>Synechocystis</em> PCC 6803: Selection of mutants defective in photosynthesis</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1989</div>, vol. <div class="volume">216</div> (pg. <div class="fpage">51</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF00332230" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF00332230</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutagenesis%20by%20random%20cloning%20of%20an%20Escherichia%20coli%20kanamycin%20resistance%20gene%20into%20the%20genome%20of%20the%20cyanobacterium%20Synechocystis%20PCC%206803%3A%20Selection%20of%20mutants%20defective%20in%20photosynthesis&amp;author=F.%20Chauvat&amp;author=P.%20Rouet&amp;author=H.%20Bottiu&amp;author=A.%20Boussac&amp;publication_year=1989&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=216&amp;pages=51-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00332230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00332230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00332230"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2499763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR042C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR042C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bhaya</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bianco</div> <div class="given-names">N.R.</div></div>,  <div class="name"><div class="surname">Bryant</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Grossman</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Type IV pilus biogenesis and motility in the cyanobacterium <em>Synechocystis</em> sp. PCC6803</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">941</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-2958.2000.02068.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-2958.2000.02068.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Type%20IV%20pilus%20biogenesis%20and%20motility%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20PCC6803&amp;author=D.%20Bhaya&amp;author=N.R.%20Bianco&amp;author=D.%20Bryant&amp;author=A.%20Grossman&amp;publication_year=2000&amp;journal=Mol.%20Microbiol.&amp;volume=37&amp;pages=941-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.02068.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.02068.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.02068.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10972813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=485988&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr042/2/dsr042-dsr042supp.pdf?Expires=2147483647&amp;Signature=nwmd8I~pNHCdnUqfN-hDAaN-JMEkfC31s7NVOkSZ3bBFVeAOSqlEV6JZZZV-dNjfPkus-ssEKqN9WBSMQnATQ~fBR7jhPB-uz-~OVFvVS0TA5fk-7thvv71RZxjai1MQa056YMfYkj-SBDl~vpJuMdg651Oh0ISGD2W0blU1F9508hmrzINe00Go3g6b8Um1WowyE5bvonY1izdAqOtz9Wvpc98WGxhx8aUesW-YY1jSXAfcCTmcXexpICNTDCVW9-QhpJYhVs2GHvPLbVi9A7Z5cIKtWGziccmby8ndQHUHtn6XJUrRI5v8fah5rMpk7UTWFk-s11X28rV94pjBNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">298</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">65</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr042"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="485988"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f1%2f67%2f485988">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000300760000006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (65)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F1%2F67%2F485988" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/485988'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification of Substrain-Specific Mutations by Massively Parallel Whole-Genome Resequencing of Synechocystis sp. PCC 6803" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr042");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f67%2f485988">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

