

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/5/201/350973","@type":"ScholarlyArticle","name":"DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins","datePublished":"2007-11-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm019","keywords":["mitochondrion","strand asymmetry","amino acids","codon usage"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/5/201/350973","author":[{"name":"Min, Xiang Jia","affiliation":"Department of Biology ,  Concordia University ,  7141 Sherbrooke West, Montreal, Quebec ,  Canada  H4B 1R6","@type":"Person"},{"name":"Hickey, Donal A.","affiliation":"Department of Biology ,  Concordia University ,  7141 Sherbrooke West, Montreal, Quebec ,  Canada  H4B 1R6","@type":"Person"}],"description":"Abstract.  Variations in GC content between genomes have been extensively documented. Genomes with comparable GC contents can, however, still differ in the appo","pageStart":"201","pageEnd":"206","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins" />
    <meta property="og:description" content="Abstract.  Variations in GC content between genomes have been extensively documented. Genomes with comparable GC contents can, however, still differ in the appo" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/5/201/350973" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-11-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Min, Xiang Jia" /><meta name="citation_author_institution" content="Department of Biology, Concordia University, 7141 Sherbrooke West, Montreal, Quebec, CanadaH4B 1R6" /><meta name="citation_author" content="Hickey, Donal A." /><meta name="citation_author_institution" content="Department of Biology, Concordia University, 7141 Sherbrooke West, Montreal, Quebec, CanadaH4B 1R6" /><meta name="citation_title" content="DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins" /><meta name="citation_firstpage" content="201" /><meta name="citation_lastpage" content="206" /><meta name="citation_doi" content="10.1093/dnares/dsm019" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=DNA sequence evolution: the sounds of silence; Philos. Trans. R. Soc. Lond. B Biol. Sci.;  citation_year=1995;  citation_volume=349;  citation_pages=241-247; " /><meta name="citation_reference" content="citation_title=Influence of genomic G + C content on average amino-acid composition of proteins from 59 bacterial species; Gene;  citation_year=1997;  citation_volume=205;  citation_pages=309-316; " /><meta name="citation_reference" content="citation_title=Nucleotide bias causes a genomewide bias in the amino acid composition of proteins; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1581-1588; " /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria; Mol. Biol. Evol.;  citation_year=1996;  citation_volume=13;  citation_pages=660-665; " /><meta name="citation_reference" content="citation_title=Two aspects of DNA base composition: G + C content and translation-coupled deviation from intra-strand rule of A = T and G = C; J. Mol. Evol.;  citation_year=1999;  citation_volume=49;  citation_pages=49-62; " /><meta name="citation_reference" content="citation_title=Patterns of nucleotide composition at fourfold degenerate sites of animal mitochondrial genomes; J. Mol. Evol.;  citation_year=1995;  citation_volume=41;  citation_pages=353-358; " /><meta name="citation_reference" content="citation_title=Strand compositional asymmetry in bacterial and large viral genomes; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=3720-3725; " /><meta name="citation_reference" content="citation_title=Base composition skews, replication orientation, and gene orientation in 12 prokaryote genomes; J. Mol. Evol.;  citation_year=1998;  citation_volume=47;  citation_pages=691-696; " /><meta name="citation_reference" content="citation_title=Universal replication biases in bacteria; Mol. Microbiol.;  citation_year=1999;  citation_volume=32;  citation_pages=11-16; " /><meta name="citation_reference" content="citation_title=Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases, Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=1642-1649; " /><meta name="citation_reference" content="citation_title=Nucleotide composition bias affects amino acid content in proteins coded by animal mitochondria; J. Mol. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=282-288; " /><meta name="citation_reference" content="citation_title=Codon usage and bias in mitochondrial genomes of parasitic platyhelminthes; Korean J. Parasitol.;  citation_year=2004;  citation_volume=42;  citation_pages=159-167; " /><meta name="citation_reference" content="citation_title=Genomics of parasitic flatworms; Int. J. Parasitol.;  citation_year=2004;  citation_volume=34;  citation_pages=153-158; " /><meta name="citation_reference" content="citation_title=Ongoing evolution of strand composition in bacterial genomes; Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=1789-1799; " /><meta name="citation_reference" content="citation_title=Similar compositional biases are caused by very different mutational effects; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1537-1547; " /><meta name="citation_reference" content="citation_title=Likelihood analysis of asymmetrical mutation bias gradients in vertebrate mitochondrial genomes; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=735-745; " /><meta name="citation_reference" content="citation_title=Strand compositional asymmetries of nuclear DNA in eukaryotes; J. Mol. Evol.;  citation_year=2003;  citation_volume=57;  citation_pages=325-334; " /><meta name="citation_reference" content="citation_title=Transcription-coupled and splicing-coupled strand asymmetries in eukaryotic genomes; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=4969-4978; " /><meta name="citation_reference" content="citation_title=Phylogeny of Arthropoda inferred from mitochondrial sequences: strategies for limiting the misleading effects of multiple changes in pattern and rates of substitution; Mol. Phylogenet. Evol.;  citation_year=2006;  citation_volume=38;  citation_pages=100-116; " /><meta name="citation_reference" content="citation_title=The effect of model choice on phylogenetic inference using mitochondrial sequence data: lessons from the scorpions; Mol. Phylogenet. Evol.;  citation_year=2007;  citation_volume=43;  citation_pages=583-595; " /><meta name="citation_reference" content="citation_title=A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes; citation_text=RESEARCH0010; Genome Biol.;  citation_year=2001;  citation_volume=2; " /><meta name="citation_reference" content="citation_title=Mutational bias affects protein evolution in flowering plants; Mol. Biol. Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=90-96; " /><meta name="citation_reference" content="citation_title=Transcription-coupled TA and GC strand asymmetries in the human genome; FEBS Lett.;  citation_year=2003;  citation_volume=555;  citation_pages=579-582; " /><meta name="citation_reference" content="citation_title=Replication-associated strand asymmetries in mammalian genomes: toward detection of replication origins; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=9836-9841; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/5/201/17462590/dsm019.pdf" /><meta name="description" content="Abstract.  Variations in GC content between genomes have been extensively documented. Genomes with comparable GC contents can, however, still differ in the appo" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/5/201/350973" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/5/201/350973" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/5/201/350973");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm019';
    </script>
<title>
	DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./350973" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bac3PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaW4sIFhpYW5nIEppYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSwgQ29uY29yZGlhIFVuaXZlcnNpdHksIDcxNDEgU2hlcmJyb29rZSBXZXN0LCBNb250cmVhbCwgUXVlYmVjLCBDYW5hZGFINEIgMVI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlja2V5LCBEb25hbCBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSwgQ29uY29yZGlhIFVuaXZlcnNpdHksIDcxNDEgU2hlcmJyb29rZSBXZXN0LCBNb250cmVhbCwgUXVlYmVjLCBDYW5hZGFINEIgMVI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJETkEgQXN5bW1ldHJpYyBTdHJhbmQgQmlhcyBBZmZlY3RzIHRoZSBBbWlubyBBY2lkIENvbXBvc2l0aW9uIG9mIE1pdG9jaG9uZHJpYWwgUHJvdGVpbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA3LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHNlcXVlbmNlIGV2b2x1dGlvbjogdGhlIHNvdW5kcyBvZiBzaWxlbmNlOyBQaGlsb3MuIFRyYW5zLiBSLiBTb2MuIExvbmQuIEIgQmlvbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zNDk7ICBjaXRhdGlvbl9wYWdlcz0yNDEtMjQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZsdWVuY2Ugb2YgZ2Vub21pYyBHICsgQyBjb250ZW50IG9uIGF2ZXJhZ2UgYW1pbm8tYWNpZCBjb21wb3NpdGlvbiBvZiBwcm90ZWlucyBmcm9tIDU5IGJhY3RlcmlhbCBzcGVjaWVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTIwNTsgIGNpdGF0aW9uX3BhZ2VzPTMwOS0zMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb3RpZGUgYmlhcyBjYXVzZXMgYSBnZW5vbWV3aWRlIGJpYXMgaW4gdGhlIGFtaW5vIGFjaWQgY29tcG9zaXRpb24gb2YgcHJvdGVpbnM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xNTgxLTE1ODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzeW1tZXRyaWMgc3Vic3RpdHV0aW9uIHBhdHRlcm5zIGluIHRoZSB0d28gRE5BIHN0cmFuZHMgb2YgYmFjdGVyaWE7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NjAtNjY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gYXNwZWN0cyBvZiBETkEgYmFzZSBjb21wb3NpdGlvbjogRyArIEMgY29udGVudCBhbmQgdHJhbnNsYXRpb24tY291cGxlZCBkZXZpYXRpb24gZnJvbSBpbnRyYS1zdHJhbmQgcnVsZSBvZiBBID0gVCBhbmQgRyA9IEM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz00OS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgb2YgbnVjbGVvdGlkZSBjb21wb3NpdGlvbiBhdCBmb3VyZm9sZCBkZWdlbmVyYXRlIHNpdGVzIG9mIGFuaW1hbCBtaXRvY2hvbmRyaWFsIGdlbm9tZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz0zNTMtMzU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhbmQgY29tcG9zaXRpb25hbCBhc3ltbWV0cnkgaW4gYmFjdGVyaWFsIGFuZCBsYXJnZSB2aXJhbCBnZW5vbWVzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTM3MjAtMzcyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzZSBjb21wb3NpdGlvbiBza2V3cywgcmVwbGljYXRpb24gb3JpZW50YXRpb24sIGFuZCBnZW5lIG9yaWVudGF0aW9uIGluIDEyIHByb2thcnlvdGUgZ2Vub21lczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTY5MS02OTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaXZlcnNhbCByZXBsaWNhdGlvbiBiaWFzZXMgaW4gYmFjdGVyaWE7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTExLTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZW9tZSBjb21wb3NpdGlvbiBhbmQgY29kb24gdXNhZ2UgaW4gc3Bpcm9jaGFldGVzOiBzcGVjaWVzLXNwZWNpZmljIGFuZCBETkEgc3RyYW5kLXNwZWNpZmljIG11dGF0aW9uYWwgYmlhc2VzLCBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTE2NDItMTY0OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvdGlkZSBjb21wb3NpdGlvbiBiaWFzIGFmZmVjdHMgYW1pbm8gYWNpZCBjb250ZW50IGluIHByb3RlaW5zIGNvZGVkIGJ5IGFuaW1hbCBtaXRvY2hvbmRyaWE7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0yODItMjg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBhbmQgYmlhcyBpbiBtaXRvY2hvbmRyaWFsIGdlbm9tZXMgb2YgcGFyYXNpdGljIHBsYXR5aGVsbWludGhlczsgS29yZWFuIEouIFBhcmFzaXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xNTktMTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBwYXJhc2l0aWMgZmxhdHdvcm1zOyBJbnQuIEouIFBhcmFzaXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xNTMtMTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbmdvaW5nIGV2b2x1dGlvbiBvZiBzdHJhbmQgY29tcG9zaXRpb24gaW4gYmFjdGVyaWFsIGdlbm9tZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNzg5LTE3OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbWlsYXIgY29tcG9zaXRpb25hbCBiaWFzZXMgYXJlIGNhdXNlZCBieSB2ZXJ5IGRpZmZlcmVudCBtdXRhdGlvbmFsIGVmZmVjdHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTUzNy0xNTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaWtlbGlob29kIGFuYWx5c2lzIG9mIGFzeW1tZXRyaWNhbCBtdXRhdGlvbiBiaWFzIGdyYWRpZW50cyBpbiB2ZXJ0ZWJyYXRlIG1pdG9jaG9uZHJpYWwgZ2Vub21lczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9NzM1LTc0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyYW5kIGNvbXBvc2l0aW9uYWwgYXN5bW1ldHJpZXMgb2YgbnVjbGVhciBETkEgaW4gZXVrYXJ5b3RlczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTMyNS0zMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24tY291cGxlZCBhbmQgc3BsaWNpbmctY291cGxlZCBzdHJhbmQgYXN5bW1ldHJpZXMgaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz00OTY5LTQ5Nzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VueSBvZiBBcnRocm9wb2RhIGluZmVycmVkIGZyb20gbWl0b2Nob25kcmlhbCBzZXF1ZW5jZXM6IHN0cmF0ZWdpZXMgZm9yIGxpbWl0aW5nIHRoZSBtaXNsZWFkaW5nIGVmZmVjdHMgb2YgbXVsdGlwbGUgY2hhbmdlcyBpbiBwYXR0ZXJuIGFuZCByYXRlcyBvZiBzdWJzdGl0dXRpb247IE1vbC4gUGh5bG9nZW5ldC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMDAtMTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZWZmZWN0IG9mIG1vZGVsIGNob2ljZSBvbiBwaHlsb2dlbmV0aWMgaW5mZXJlbmNlIHVzaW5nIG1pdG9jaG9uZHJpYWwgc2VxdWVuY2UgZGF0YTogbGVzc29ucyBmcm9tIHRoZSBzY29ycGlvbnM7IE1vbC4gUGh5bG9nZW5ldC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz01ODMtNTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBtb2RlbCBiYXNlZCBvbiBtdXRhdGlvbiBhbmQgc2VsZWN0aW9uIGV4cGxhaW5zIHRyZW5kcyBpbiBjb2RvbiBhbmQgYW1pbm8tYWNpZCB1c2FnZSBhbmQgR0MgY29tcG9zaXRpb24gd2l0aGluIGFuZCBhY3Jvc3MgZ2Vub21lczsgY2l0YXRpb25fdGV4dD1SRVNFQVJDSDAwMTA7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbmFsIGJpYXMgYWZmZWN0cyBwcm90ZWluIGV2b2x1dGlvbiBpbiBmbG93ZXJpbmcgcGxhbnRzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9OTAtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24tY291cGxlZCBUQSBhbmQgR0Mgc3RyYW5kIGFzeW1tZXRyaWVzIGluIHRoZSBodW1hbiBnZW5vbWU7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTU1OyAgY2l0YXRpb25fcGFnZXM9NTc5LTU4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwbGljYXRpb24tYXNzb2NpYXRlZCBzdHJhbmQgYXN5bW1ldHJpZXMgaW4gbWFtbWFsaWFuIGdlbm9tZXM6IHRvd2FyZCBkZXRlY3Rpb24gb2YgcmVwbGljYXRpb24gb3JpZ2luczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9OTgzNi05ODQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTQvNS8yMDEvMTc0NjI1OTAvZHNtMDE5LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBWYXJpYXRpb25zIGluIEdDIGNvbnRlbnQgYmV0d2VlbiBnZW5vbWVzIGhhdmUgYmVlbiBleHRlbnNpdmVseSBkb2N1bWVudGVkLiBHZW5vbWVzIHdpdGggY29tcGFyYWJsZSBHQyBjb250ZW50cyBjYW4sIGhvd2V2ZXIsIHN0aWxsIGRpZmZlciBpbiB0aGUgYXBwbyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzUwOTczJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZBDNG/mSrDxM8JWtlGfAbLVfizIs" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f201%2f350973" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f201%2f350973" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13580" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&amp;Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/5">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195025">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195025">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195027">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195027">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195032">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195032">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195038">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195038">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195052">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195052">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195054">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195054">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/14/5/207/351235">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiang Jia Min</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biology</div>, <div class="insititution">Concordia University</div>, <div class="addr-line">7141 Sherbrooke West, Montreal, Quebec</div>, <div class="country">Canada</div><div class="addr-line">H4B 1R6</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiang+Jia+Min">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Min X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Min X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiang Jia Min</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Donal A. Hickey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biology</div>, <div class="insititution">Concordia University</div>, <div class="addr-line">7141 Sherbrooke West, Montreal, Quebec</div>, <div class="country">Canada</div><div class="addr-line">H4B 1R6</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 514-848-2424 ext 5184. Fax. <div class="fax">+1 514-848-2881</div>. E-mail: <a href="mailto:dhickey@alcor.concordia.ca" target="_blank">dhickey@alcor.concordia.ca</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Donal+A.+Hickey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hickey D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hickey D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Donal A. Hickey</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr. Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 5, 1 January 2007, Pages 201206, <a href='https://doi.org/10.1093/dnares/dsm019'>https://doi.org/10.1093/dnares/dsm019</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 November 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 July 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 October 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="350973" href="/dnaresearch/article-pdf/14/5/201/17462590/dsm019.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Xiang Jia Min, Donal A. Hickey;  DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins, <em>DNA Research</em>, Volume 14, Issue 5, 1 January 2007, Pages 201206, <a href='https://doi.org/10.1093/dnares/dsm019'>https://doi.org/10.1093/dnares/dsm019</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/350973?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/350973?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/350973?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/350973?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/350973?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/350973?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/350973?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=DNA%20Asymmetric%20Strand%20Bias%20Affects%20the%20Amino%20Acid%20Composition%20of%20Mitochondrial%20Proteins&amp;publicationDate=2007-11-01&amp;volumeNum=14&amp;issueNum=5&amp;author=Min%2C%20Xiang%20Jia%3B%20Hickey%2C%20Donal%20A.&amp;startPage=201&amp;endPage=206&amp;contentId=10.1093%2Fdnares%2Fdsm019&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f201%2f350973" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195025 id="87195025" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Variations in GC content between genomes have been extensively documented. Genomes with comparable GC contents can, however, still differ in the apportionment of the G and C nucleotides between the two DNA strands. This asymmetric strand bias is known as GC skew. Here, we have investigated the impact of differences in nucleotide skew on the amino acid composition of the encoded proteins. We compared orthologous genes between animal mitochondrial genomes that show large differences in GC and AT skews. Specifically, we compared the mitochondrial genomes of mammals, which are characterized by a negative GC skew and a positive AT skew, to those of flatworms, which show the opposite skews for both GC and AT base pairs. We found that the mammalian proteins are highly enriched in amino acids encoded by CA-rich codons (as predicted by their negative GC and positive AT skews), whereas their flatworm orthologs were enriched in amino acids encoded by GT-rich codons (also as predicted from their skews). We found that these differences in mitochondrial strand asymmetry (measured as GC and AT skews) can have very large, predictable effects on the composition of the encoded proteins.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="mitochondrion">mitochondrion</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;strand asymmetry&quot;">strand asymmetry</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;amino acids&quot;">amino acids</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage&quot;">codon usage</a></div>                            <h2 scrollto-destination=87195027 id="87195027" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genome-wide biases in nucleotide content have been extensively studied in a wide variety of organisms.<sup><a reveal-id="DSM019C1" data-open="DSM019C1" class="link link-ref link-reveal xref-bibr">1</a></sup> During the past decade, there has been accumulating evidence that these variations at the DNA level can result in parallel changes in the frequencies of amino acids in the encoded proteins. For example, the GC content of bacterial genomes has been shown to influence the amino acid composition of the proteome.<sup><a reveal-id="DSM019C2" data-open="DSM019C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSM019C3" data-open="DSM019C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>In addition to the variations in GC content, however, bacterial genomes can also show significant compositional asymmetry between the two DNA strands.<sup><a reveal-id="DSM019C4" data-open="DSM019C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSM019C5" data-open="DSM019C5" class="link link-ref link-reveal xref-bibr">5</a></sup> This strand asymmetry is usually measured as GC or AT skew<sup><a reveal-id="DSM019C6" data-open="DSM019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> (see Methods) and it can be due to such factors as different substitutional patterns between the leading and lagging strands during replication.<sup><a reveal-id="DSM019C4" data-open="DSM019C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSM019C7" data-open="DSM019C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSM019C8" data-open="DSM019C8" class="link link-ref link-reveal xref-bibr">8</a></sup> By comparing the amino acid compositions of proteins encoded on the leading strand with those encoded on the lagging strand, it has been shown that these nucleotide skews can also affect the amino acid composition of bacterial proteins.<sup><a reveal-id="DSM019C9" data-open="DSM019C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSM019C10" data-open="DSM019C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>Among eukaryotes, the correlations between nucleotide content and amino acid compositions have been studied primarily in animal mitochondrial genomes. Animal mitochondria show considerable variation between species in both GC content and GC skew.<sup><a reveal-id="DSM019C6" data-open="DSM019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> It has been shown that the variations in GC content affect the amino acid composition of the encoded proteins<sup><a reveal-id="DSM019C11" data-open="DSM019C11" class="link link-ref link-reveal xref-bibr">11</a></sup> but the effects of variations in nucleotide skew between genomes have not been studied. The recent availability of a very large number of completely sequenced mitochondrial genomes allows us to fill this gap. Mammalian mitochondria, including those of humans, are characterized by negative GC skews and positive AT skews, i.e. the major coding strand of mammalian mitochondria is relatively rich in the nucleotides C and A, and correspondingly poor in G and T. For instance, as noted by Perna and Kocher (1995),<sup><a reveal-id="DSM019C6" data-open="DSM019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> although the human mitochondrial genome contains more than 40% GC pairs, the frequency of G on the coding strand is only 5%. The strength of the skew varies between species, and some species of invertebrate show opposite skews to those found in mammals, i.e. the coding strand is rich in G and poor in C.<sup><a reveal-id="DSM019C6" data-open="DSM019C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p><p>In recent years, there has been an explosion of data on mitochondrial whole genome sequences. Since our goal was to measure the effect of strand asymmetry on amino acid composition, we chose a group of species, the Platyhelminthes (flatworms), in which the GC and AT skews are opposite to those seen in mammals.<sup><a reveal-id="DSM019C12" data-open="DSM019C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSM019C13" data-open="DSM019C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Among the flatworms, the coding sequences are rich in G and T and correspondingly poor in C and A, i.e. a complete contrast to the patterns seen in mammals. For example, the main mitochondrial coding strand in <em>Schistosoma mansoni</em> (a flatworm) contains only 6.7% C, but it contains 25% G.<sup><a reveal-id="DSM019C12" data-open="DSM019C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Despite the contrasting GC and AT skew patterns, however, both mammals and flatworms contain essentially the same set of homologous mitochondrial genes and they span similar ranges of GC content. Thus a comparison between these two groups provides us with an opportunity to assess the potential effects of DNA strand bias on the amino acid composition of a well-characterized set of orthologous proteins.</p>                            <h2 scrollto-destination=87195032 id="87195032" class="section-title" data-legacy-id=s2>2. Methods</h2>
<p>We downloaded all of the publicly available complete mitochondrial (mt) genome (mtDNA) sequences from both mammals and Platyhelminthes from the NCBI RefSeq organelle genome database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genomes/ORGANELLES/mztax_short.html">http://www.ncbi.nlm.nih.gov/genomes/ORGANELLES/mztax_short.html</a>) (released in July, 2006). There were a total of 170 mt genomes from mammals and 13 mt genomes from Platyhelminthes (Supplementary Table S1).</p><p>The nucleotide frequencies and compositional biases of mtDNA sequences were calculated from the  + -strand (the major coding strand used by NCBI for annotation) for each species. The GC and AT asymmetry is measured in terms of GC- and AT-skews according to the following formulae given in Perna and Kocher<sup><a reveal-id="DSM019C6" data-open="DSM019C6" class="link link-ref link-reveal xref-bibr">6</a></sup>: GC-skew = (GC)/(G + C); AT-skew = (A  T)/(A + T), where C, G, A, and T are the occurrences of the four nucleotides.</p><p>We predicted the amino acid compositions based on the mitochondrial genetic code (Table<a reveal-id="DSM019TB1" data-open="DSM019TB1" class="link link-reveal link-table xref-fig">1</a>) by partitioning the mitochondrial codons into CA-rich codons, GT-rich codons, and other codons. The frequency of synonymous codon usage was measured by the nucleotide content of G + T or C + A at the third codon positions of fourfold degenerate codon families: GGN (Glycine), GTN (Valine), CGN (Arginine), ACN (Threonine), GCN (Alanine), and CCN (Proline) (Supplementary Fig. S1).</p>                        <a id="87195036" scrollto-destination="87195036"></a>
<div content-id="DSM019TB1" class="table-modal"><div class="table-wrap"><div id="DSM019TB1" class="table-wrap-title" data-id="DSM019TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Partition of the vertebrate mitochondrial genetic code into GT-rich, CA-rich and other codons</p></div> </div><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01904.jpeg?Expires=2147483647&Signature=XkK0JiY9WL4~oad-kzDwtlKMxIo7Q04tXWfJTjkIBrwo-QxKPaNBBGlLLo5DjtaIbSFoCOXxGH5RWrdZeTTjE0xYBrDjvMOJOfKp8lF7VuPQ781K-mb3o5gbyoOokqrvsbPnzppCHIyV-9Bp6CAUCH-z5bBoD1cIjp9KpGTKu7WdOnglVYpjjpU33Yq0IVo~d7wvKOPN7CrbtljNJvPTVRhdaC2sbVSEFufZW2DGrhfGvPX4FB7c56CmSZforRziQBRLtm0eyH2HmkLRDWuJBTGllCropEik1gY4nQi3K6AN1u4PB6ojtI5w-cj0GETINJ097X~j23BEF6mwlmDN5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01904.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01904.jpeg?Expires=2147483647&Signature=Dky7cQicQjZKxTick6fnnOrn7LeGq0WGgZLkt7l32WvNJcs495Qs9bw2Qi2ztcdrV~pG17plSkLyi3WU-s6JYdNdv72H5BkXiu7HVCuJXJcs6quh9OHzhWIfsijsHck7MUcBSavBOKgMtREZkHgFAnjq2SABTJmrVBAYr0brN1SaqfYu~sQZz4DFY7YEJKL69tycMIyTO7NK4zsLKyW7iwywZSK8ag-0iGQo5Pk72wRTO1RsuqC2HQ~y89aMVsZK8XMJhaeBwhZz4QkxM6yAXlPc8dxl~AfY-mCmVaAnY6SU4xPnzu7mjruVk7RSp~Pd00prR48nBhk5u8JsfnGMDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Partition of the vertebrate mitochondrial genetic code into GT-rich, CA-rich and other codons" path-from-xml="dsm01904.jpeg" /></a></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>GT-rich codons (italic) include GT, TG, GG, TT codons at the first two codon positions. TTA and TTG are excluded as there are other codons for Leu. CA-rich codons (bold) include CA, AC, CC, AA codons at the first two codon positions. Different codon assignments in Mammals and Platyhelminthes are underlined. In Platyhelminthes, AGA and AGG code for Ser, ATA for Ile, and AAA for Asn. The numbers following each codon are codon usage per thousand codons in the 11 conserved proteins of mammals (the first number) and Platyhelminthes (the second number).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195036" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM019TB1" class="table-wrap-title" data-id="DSM019TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Partition of the vertebrate mitochondrial genetic code into GT-rich, CA-rich and other codons</p></div> </div><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01904.jpeg?Expires=2147483647&Signature=XkK0JiY9WL4~oad-kzDwtlKMxIo7Q04tXWfJTjkIBrwo-QxKPaNBBGlLLo5DjtaIbSFoCOXxGH5RWrdZeTTjE0xYBrDjvMOJOfKp8lF7VuPQ781K-mb3o5gbyoOokqrvsbPnzppCHIyV-9Bp6CAUCH-z5bBoD1cIjp9KpGTKu7WdOnglVYpjjpU33Yq0IVo~d7wvKOPN7CrbtljNJvPTVRhdaC2sbVSEFufZW2DGrhfGvPX4FB7c56CmSZforRziQBRLtm0eyH2HmkLRDWuJBTGllCropEik1gY4nQi3K6AN1u4PB6ojtI5w-cj0GETINJ097X~j23BEF6mwlmDN5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01904.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01904.jpeg?Expires=2147483647&Signature=Dky7cQicQjZKxTick6fnnOrn7LeGq0WGgZLkt7l32WvNJcs495Qs9bw2Qi2ztcdrV~pG17plSkLyi3WU-s6JYdNdv72H5BkXiu7HVCuJXJcs6quh9OHzhWIfsijsHck7MUcBSavBOKgMtREZkHgFAnjq2SABTJmrVBAYr0brN1SaqfYu~sQZz4DFY7YEJKL69tycMIyTO7NK4zsLKyW7iwywZSK8ag-0iGQo5Pk72wRTO1RsuqC2HQ~y89aMVsZK8XMJhaeBwhZz4QkxM6yAXlPc8dxl~AfY-mCmVaAnY6SU4xPnzu7mjruVk7RSp~Pd00prR48nBhk5u8JsfnGMDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Partition of the vertebrate mitochondrial genetic code into GT-rich, CA-rich and other codons" path-from-xml="dsm01904.jpeg" /></a></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>GT-rich codons (italic) include GT, TG, GG, TT codons at the first two codon positions. TTA and TTG are excluded as there are other codons for Leu. CA-rich codons (bold) include CA, AC, CC, AA codons at the first two codon positions. Different codon assignments in Mammals and Platyhelminthes are underlined. In Platyhelminthes, AGA and AGG code for Ser, ATA for Ile, and AAA for Asn. The numbers following each codon are codon usage per thousand codons in the 11 conserved proteins of mammals (the first number) and Platyhelminthes (the second number).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195036" target="_blank">View Large</a></div></div><p>The statistical significance of the average differences in amino acid composition between the two groups of species was scored using a Student's <em>t</em>-test.</p>                            <h2 scrollto-destination=87195038 id="87195038" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
<p>We first confirmed the published reports (see Introduction) of contrasting GC and AT skews in the mitochondria of mammals and flatworms. As can be seen from Fig.<a reveal-id="DSM019F1" data-open="DSM019F1" class="link link-reveal link-table xref-fig">1</a>, there is a negative GC skew, and a positive AT skew, in the major coding strand among the mammalian species, whereas the opposite is true among the flatworms. Despite the variations between species within both groups, there is a large average difference between them and this difference between the groups is statistically highly significant (<em>P</em> &lt; 0.0001). As previously noted by Perna and Kocher<sup><a reveal-id="DSM019C6" data-open="DSM019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and Le et al.,<sup><a reveal-id="DSM019C12" data-open="DSM019C12" class="link link-ref link-reveal xref-bibr">12</a></sup> the major coding sequence of mammalian mtDNA is relatively rich in C and A, whereas G and T are much more common in the flatworm coding sequences.</p>                        <a id="87195040" scrollto-destination="87195040"></a>
<div id="DSM019F1" data-id="DSM019F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01901.jpeg?Expires=2147483647&Signature=WIA9SWOCxikEwxcSjjizij2B0ExPRoyJuJNbWsnL2GH9vpTSSwm4MvvchzJewxv2Kmu--SMEW3Hjz02KFk0B2bWlRsToljWLWBu5dw9sVVFbZaipr0yC~HBGqU18Gr7acBkXiaYJGW6jzAG97tgMb5P-3GyL9iG-trZTU~CO0fspW8JOoDUv85lCLZFgA5s9GMtTYLv0v9rBmWyOhAXAKtrmX~LNx3HBIKepjuWqtpw0a7sZ1lrGiQAvnMsOhbiMkt8Bv~M9VLKP~~rkQP-qpPExdvGFenBfjA4H6WyI4dHqFJNWXUkCvHZwGe0yWkqLMJUvwZaMS~8Hr6b8DzbSww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contrasting patterns of DNA strand asymmetry in the mitochondrial coding sequences of mammals and Platyhelminthes. (A) The GC skew, (G  C)/(G + C), is negative for all mammalian species (shown in blue), and positive for all Platyhelminthe species (shown in red). (B) The AT skew, (A  T)/(A + T), is positive for mammals (shown in blue) and negative for Platyhelminthes (shown in red)." path-from-xml="dsm01901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195040/dsm01901.jpeg" path-from-xml="dsm01901.jpeg" target="_blank">View large</a><a section="87195040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01901.jpeg?Expires=2147483647&Signature=e9DnvXstQyGi07L-Ul82fqJ90L31uvxvJlrhT3II6icHkDy7vCCPEQFJcJEZK188A9Wr8fWUhmlE-lIAvQcP6WoNWDU6NfFpVqhLEGEe08gerGj8OW9JNqwxNLE~0sts0dfyGqH2FQkYuFR0gkBVDFyu1XhxRICUZd8Vai1vy0PVeQBoV2rKGLjZojsnmQ1HDYs9Itxy45EAIFtvTltbmSdml3icw9VZfhVumW1Jg~OvuEREN62BGqW0iMu8xfFLDeNZWG9hwC46hlVue3Zk8EzxIKXe1rnna1w1pP9aZjx~bZP~~rTlCFXfG5~vQZCudFo4Tzy6-RH~ehw7TbeZfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195040&ar=350973&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contrasting patterns of DNA strand asymmetry in the mitochondrial coding sequences of mammals and Platyhelminthes. (A) The GC skew, (G  C)/(G + C), is negative for all mammalian species (shown in blue), and positive for all Platyhelminthe species (shown in red). (B) The AT skew, (A  T)/(A + T), is positive for mammals (shown in blue) and negative for Platyhelminthes (shown in red).</p></div></div><div content-id="DSM019F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01901.jpeg?Expires=2147483647&Signature=WIA9SWOCxikEwxcSjjizij2B0ExPRoyJuJNbWsnL2GH9vpTSSwm4MvvchzJewxv2Kmu--SMEW3Hjz02KFk0B2bWlRsToljWLWBu5dw9sVVFbZaipr0yC~HBGqU18Gr7acBkXiaYJGW6jzAG97tgMb5P-3GyL9iG-trZTU~CO0fspW8JOoDUv85lCLZFgA5s9GMtTYLv0v9rBmWyOhAXAKtrmX~LNx3HBIKepjuWqtpw0a7sZ1lrGiQAvnMsOhbiMkt8Bv~M9VLKP~~rkQP-qpPExdvGFenBfjA4H6WyI4dHqFJNWXUkCvHZwGe0yWkqLMJUvwZaMS~8Hr6b8DzbSww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contrasting patterns of DNA strand asymmetry in the mitochondrial coding sequences of mammals and Platyhelminthes. (A) The GC skew, (G  C)/(G + C), is negative for all mammalian species (shown in blue), and positive for all Platyhelminthe species (shown in red). (B) The AT skew, (A  T)/(A + T), is positive for mammals (shown in blue) and negative for Platyhelminthes (shown in red)." path-from-xml="dsm01901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195040/dsm01901.jpeg" path-from-xml="dsm01901.jpeg" target="_blank">View large</a><a section="87195040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01901.jpeg?Expires=2147483647&Signature=e9DnvXstQyGi07L-Ul82fqJ90L31uvxvJlrhT3II6icHkDy7vCCPEQFJcJEZK188A9Wr8fWUhmlE-lIAvQcP6WoNWDU6NfFpVqhLEGEe08gerGj8OW9JNqwxNLE~0sts0dfyGqH2FQkYuFR0gkBVDFyu1XhxRICUZd8Vai1vy0PVeQBoV2rKGLjZojsnmQ1HDYs9Itxy45EAIFtvTltbmSdml3icw9VZfhVumW1Jg~OvuEREN62BGqW0iMu8xfFLDeNZWG9hwC46hlVue3Zk8EzxIKXe1rnna1w1pP9aZjx~bZP~~rTlCFXfG5~vQZCudFo4Tzy6-RH~ehw7TbeZfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195040&ar=350973&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contrasting patterns of DNA strand asymmetry in the mitochondrial coding sequences of mammals and Platyhelminthes. (A) The GC skew, (G  C)/(G + C), is negative for all mammalian species (shown in blue), and positive for all Platyhelminthe species (shown in red). (B) The AT skew, (A  T)/(A + T), is positive for mammals (shown in blue) and negative for Platyhelminthes (shown in red).</p></div></div><p>Despite their differences in strand asymmetry, the major coding strands of the two groups encode essentially the same set of 11 conserved mitochondrial proteins including cytochrome b, three subunits of cytochrome c oxidase (subunit 1, 2, and 3), six subunits of NADH dehydrogenase (subunit 1, 2, 3, 4, 4L, and 5), and ATP synthase F0 subunit 6 (ATP6). The exception is that NADH dehydrogenase subunit 6 (ND6) is encoded on the major coding strand in flatworms, but is encoded on the opposite strand in mammals.</p><p>Given the contrasting patterns of mitochondrial DNA strand asymmetry between these two groups of animals, we wished to investigate if there was a corresponding difference between the two groups in the frequencies of encoded amino acids. Specifically, because of their negative GC skews and positive AT skews (which reduce the frequencies of G and T nucleotides on the coding strand), we expected the mammalian coding strands to encode proteins that are relatively low in the proportions of cysteine (C), Valine (V), phenylalanine (F), Glycine (G), and Tryptophan (W), all of which are encoded by GT-rich codons (Table<a reveal-id="DSM019TB1" data-open="DSM019TB1" class="link link-reveal link-table xref-fig">1</a>). This is exactly what we observed (Fig.<a reveal-id="DSM019F2" data-open="DSM019F2" class="link link-reveal link-table xref-fig">2</a>A). On average, the combined proportions of these five amino acids among the mammals are approximately half the value observed in the flatworm orthologs. Since these GC and AT skews result in a corresponding enrichment of C and A nucleotides on the mammalian coding strand, we expected the mammalian proteins to show a corresponding relative increase in the proportions of Glutamine (G), Threonine (T), Proline (P), Histidine (H), Asparagine (N), and Lysine (K), all of which are encoded by CA-rich codons. Again, this prediction is borne out (Fig. 2B) and again the average difference between the two groups of species is approximately twofold. Not only are the average differences in the predicted direction, but they are statistically highly significant (<em>P</em> &lt; 0.0001) and they are consistent over all species within each group (see details in Supplementary Table S1). In addition to the consistency over species, there is also a consistency over all amino acids within the two codon groups. These differences are surprisingly large, given that we are dealing with a set of conserved orthologous proteins. Although taken as a group, the amino acid frequencies show approximately a twofold difference between the two groups (Fig.<a reveal-id="DSM019F2" data-open="DSM019F2" class="link link-reveal link-table xref-fig">2</a>), at the level of individual amino acids, some of these differences are threefold or greater (Fig.<a reveal-id="DSM019F3" data-open="DSM019F3" class="link link-reveal link-table xref-fig">3</a>). Again, the individual amino acid differences shown in Fig. 3 are highly statistically significant (<em>P</em> &lt; 0.0001). Thus, we can conclude that strand asymmetries at the level of DNA have had a major influence on the composition of these mitochondrial protein sequences.</p>                        <a id="87195043" scrollto-destination="87195043"></a>
<div id="DSM019F2" data-id="DSM019F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01902.jpeg?Expires=2147483647&Signature=4Fcw0WwW-gZTjA1HeB1LrYa7N2Jyxkm7foDpway~NeUm3ovxu7OHSwxSyxL~t8ysYxmPd6ZEWJ1rr5e8yxx~4OImx2ylHrZa6lldUpk-~CJ5epHRp4s03e3ald1ssgZ2xUQIyJEwNYBPhX4qBpyRTfgWXJGo3-iLeN-pBPKtAaNdgQCsgnfVAqGyISUNbet6fuIUReJQy0osRYumO19dFlHQFbvyVQbTRJfc4K6ou8NzI4ExRxYPeOtFhVtL5~BfeFZBL~TwhbzXMf2e9sMfzeYAdPxGEjVzLwgSusjbNJw0r80AQndb8RnLPdKR4FIJb~nbvJdqhIFGwcug7HZkiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The amino composition of mitochondrial proteins reflects the DNA strand asymmetry. (A) The proportions of cysteine (C), Valine (V), phenylalanine (F), Glycine (G), and Tryptophan (W) are relatively low in mammalian proteins (shown in blue) and relatively high in flatworms (shown in red). These amino acids are encoded by GT-rich codons. (B) The proportions Glutamine (Q), Threonine (T), Proline (P), Histidine (H), Asparagine (N), and Lysine (K) are, in contrast, relatively high in mammals (shown in blue) and relatively low in flatworms (shown in red). These amino acids are encoded by CA-rich codons." path-from-xml="dsm01902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195043/dsm01902.jpeg" path-from-xml="dsm01902.jpeg" target="_blank">View large</a><a section="87195043" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01902.jpeg?Expires=2147483647&Signature=14fe4jkzSapIwzABW4Nkh56ggEkE8vCndAba7g5RZ3fLPn~p~5JVn-BjkrbugHFD0sH0BpmL4FC7a79Fk-Xm8xpj7JiomA3gEKqzFwpBhAXjZmbIQxRa97pqUETq5TQq3ZaAX8e6KKbApm3-0wzHcZ16Pj1wHEp5ns7HFPSgJho3FTeyYM35BMyJTHDYnAxkN164K6RY1IifHmbSi~Z8l9nkY8SydAazmU59JEJuznkPQdfPTr43ldd~R6QkiXoLgLhIOaEYlBRI4juzg3YXbNDMHmJ-arFCuQMdauc56l~Z~Qz5CQ2RNEgKt9Ys5baZctSUCIzSm8kTj63Pt0Tpow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195043&ar=350973&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The amino composition of mitochondrial proteins reflects the DNA strand asymmetry. (A) The proportions of cysteine (C), Valine (V), phenylalanine (F), Glycine (G), and Tryptophan (W) are relatively low in mammalian proteins (shown in blue) and relatively high in flatworms (shown in red). These amino acids are encoded by GT-rich codons. (B) The proportions Glutamine (Q), Threonine (T), Proline (P), Histidine (H), Asparagine (N), and Lysine (K) are, in contrast, relatively high in mammals (shown in blue) and relatively low in flatworms (shown in red). These amino acids are encoded by CA-rich codons.</p></div></div><div content-id="DSM019F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01902.jpeg?Expires=2147483647&Signature=4Fcw0WwW-gZTjA1HeB1LrYa7N2Jyxkm7foDpway~NeUm3ovxu7OHSwxSyxL~t8ysYxmPd6ZEWJ1rr5e8yxx~4OImx2ylHrZa6lldUpk-~CJ5epHRp4s03e3ald1ssgZ2xUQIyJEwNYBPhX4qBpyRTfgWXJGo3-iLeN-pBPKtAaNdgQCsgnfVAqGyISUNbet6fuIUReJQy0osRYumO19dFlHQFbvyVQbTRJfc4K6ou8NzI4ExRxYPeOtFhVtL5~BfeFZBL~TwhbzXMf2e9sMfzeYAdPxGEjVzLwgSusjbNJw0r80AQndb8RnLPdKR4FIJb~nbvJdqhIFGwcug7HZkiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The amino composition of mitochondrial proteins reflects the DNA strand asymmetry. (A) The proportions of cysteine (C), Valine (V), phenylalanine (F), Glycine (G), and Tryptophan (W) are relatively low in mammalian proteins (shown in blue) and relatively high in flatworms (shown in red). These amino acids are encoded by GT-rich codons. (B) The proportions Glutamine (Q), Threonine (T), Proline (P), Histidine (H), Asparagine (N), and Lysine (K) are, in contrast, relatively high in mammals (shown in blue) and relatively low in flatworms (shown in red). These amino acids are encoded by CA-rich codons." path-from-xml="dsm01902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195043/dsm01902.jpeg" path-from-xml="dsm01902.jpeg" target="_blank">View large</a><a section="87195043" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01902.jpeg?Expires=2147483647&Signature=14fe4jkzSapIwzABW4Nkh56ggEkE8vCndAba7g5RZ3fLPn~p~5JVn-BjkrbugHFD0sH0BpmL4FC7a79Fk-Xm8xpj7JiomA3gEKqzFwpBhAXjZmbIQxRa97pqUETq5TQq3ZaAX8e6KKbApm3-0wzHcZ16Pj1wHEp5ns7HFPSgJho3FTeyYM35BMyJTHDYnAxkN164K6RY1IifHmbSi~Z8l9nkY8SydAazmU59JEJuznkPQdfPTr43ldd~R6QkiXoLgLhIOaEYlBRI4juzg3YXbNDMHmJ-arFCuQMdauc56l~Z~Qz5CQ2RNEgKt9Ys5baZctSUCIzSm8kTj63Pt0Tpow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195043&ar=350973&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The amino composition of mitochondrial proteins reflects the DNA strand asymmetry. (A) The proportions of cysteine (C), Valine (V), phenylalanine (F), Glycine (G), and Tryptophan (W) are relatively low in mammalian proteins (shown in blue) and relatively high in flatworms (shown in red). These amino acids are encoded by GT-rich codons. (B) The proportions Glutamine (Q), Threonine (T), Proline (P), Histidine (H), Asparagine (N), and Lysine (K) are, in contrast, relatively high in mammals (shown in blue) and relatively low in flatworms (shown in red). These amino acids are encoded by CA-rich codons.</p></div></div>                        <a id="87195044" scrollto-destination="87195044"></a>
<div id="DSM019F3" data-id="DSM019F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01903.jpeg?Expires=2147483647&Signature=L4VYBMEMeh6vHnPB6VIxLgSeX6WLLp5XuFpntr7M6grhPLAgl4WVBu4ATOXH6PFcY9inGX9VMgHHXoznKWvMgzY6j-p5HcfEIrVodWP6xvKT57YAyXiuJr7vLx2Y5AJnPr9~5VUB6tXtsSY26lmLM6AdBtwgv8Lzd1iZwG8mRZODOxY~UX26-APbOsWx0hOhdsQvWmwkwLm2OvoX0LT85BuwmH6lwaIetSMr9SU~p9UptmsCusx5xlJaKqO1OqMEr-GowBOy6sBcD-IF3VhueYJIR36ff8y7lZOoLzICEuVz6n3GPlJCDHhALCS9NCpEjp6y6YmcWl2xI8n9o5rpSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The proportions of individual amino acids that were most affected by DNA strand asymmetry in mammals and Platyhelminthes. (A) The proportions of cysteine and Valine are low in mammalian proteins (shown in blue) and high in flatworm proteins (shown in red). (B) The proportions of Glutamine and Threonine are high in mammals (shown in blue) and low in flatworms (shown in red)." path-from-xml="dsm01903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195044/dsm01903.jpeg" path-from-xml="dsm01903.jpeg" target="_blank">View large</a><a section="87195044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01903.jpeg?Expires=2147483647&Signature=Cssg7p2O0P-suCTT49XfAChZ4oIdBPpdljKku0O-dtq43N0bYVG59~Uo9V4EwyE0l55g3VRhQlQfO~XKbO5rYHanbd6QDCVLo8Dy3kKBlW9zTwhTKJXYoGC0zgXszgb75SxvtcabMBKAZVR0TiNM754vRvoWr7VHATsjrxsD8tZHNilOdgu4HRaIWT7jAoD7YrvD-L3dZjPmVxUQl5-dKx8DeuUGR4N1yxYLITjNKQt9U8dsjKYWJof2UkxPe36F9mhVkIwvgEiYsk83KsEk2nSpDzXXVZzU~UDk20mAlMIm-DzixWGIYSeU~QLJEKZgnB6QY5fX6uV-hSPm5icCVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195044&ar=350973&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The proportions of individual amino acids that were most affected by DNA strand asymmetry in mammals and Platyhelminthes. (A) The proportions of cysteine and Valine are low in mammalian proteins (shown in blue) and high in flatworm proteins (shown in red). (B) The proportions of Glutamine and Threonine are high in mammals (shown in blue) and low in flatworms (shown in red).</p></div></div><div content-id="DSM019F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/m_dsm01903.jpeg?Expires=2147483647&Signature=L4VYBMEMeh6vHnPB6VIxLgSeX6WLLp5XuFpntr7M6grhPLAgl4WVBu4ATOXH6PFcY9inGX9VMgHHXoznKWvMgzY6j-p5HcfEIrVodWP6xvKT57YAyXiuJr7vLx2Y5AJnPr9~5VUB6tXtsSY26lmLM6AdBtwgv8Lzd1iZwG8mRZODOxY~UX26-APbOsWx0hOhdsQvWmwkwLm2OvoX0LT85BuwmH6lwaIetSMr9SU~p9UptmsCusx5xlJaKqO1OqMEr-GowBOy6sBcD-IF3VhueYJIR36ff8y7lZOoLzICEuVz6n3GPlJCDHhALCS9NCpEjp6y6YmcWl2xI8n9o5rpSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The proportions of individual amino acids that were most affected by DNA strand asymmetry in mammals and Platyhelminthes. (A) The proportions of cysteine and Valine are low in mammalian proteins (shown in blue) and high in flatworm proteins (shown in red). (B) The proportions of Glutamine and Threonine are high in mammals (shown in blue) and low in flatworms (shown in red)." path-from-xml="dsm01903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195044/dsm01903.jpeg" path-from-xml="dsm01903.jpeg" target="_blank">View large</a><a section="87195044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm01903.jpeg?Expires=2147483647&Signature=Cssg7p2O0P-suCTT49XfAChZ4oIdBPpdljKku0O-dtq43N0bYVG59~Uo9V4EwyE0l55g3VRhQlQfO~XKbO5rYHanbd6QDCVLo8Dy3kKBlW9zTwhTKJXYoGC0zgXszgb75SxvtcabMBKAZVR0TiNM754vRvoWr7VHATsjrxsD8tZHNilOdgu4HRaIWT7jAoD7YrvD-L3dZjPmVxUQl5-dKx8DeuUGR4N1yxYLITjNKQt9U8dsjKYWJof2UkxPe36F9mhVkIwvgEiYsk83KsEk2nSpDzXXVZzU~UDk20mAlMIm-DzixWGIYSeU~QLJEKZgnB6QY5fX6uV-hSPm5icCVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195044&ar=350973&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The proportions of individual amino acids that were most affected by DNA strand asymmetry in mammals and Platyhelminthes. (A) The proportions of cysteine and Valine are low in mammalian proteins (shown in blue) and high in flatworm proteins (shown in red). (B) The proportions of Glutamine and Threonine are high in mammals (shown in blue) and low in flatworms (shown in red).</p></div></div><p>These results show that differences in the patterns of strand asymmetry between the coding and template strands of a mitochondrial gene can produce very significant changes in the amino acid composition of the encoded proteins. The magnitude of these changes is comparable to those noted previously by Foster et al.<sup><a reveal-id="DSM019C11" data-open="DSM019C11" class="link link-ref link-reveal xref-bibr">11</a></sup> for the effects of differences in mitochondrial nucleotide composition, i.e. GC content. It should be noted, however, that the strand asymmetries described in this study do not affect the same subsets of the amino acids as those affected by changes GC content. As an illustrative example, we can compare a single pair of mitochondrial genomes, one mammal and one flatworm, those of the red deer (<em>Cervus elaphus</em>) and the liver fluke (<em>Fasciola hepatica</em>). In both species, the overall nucleotide content of the mitochondrial coding sequences is virtually identical at 38% GC and proportions of GC-rich (GARP) and AT-rich (FYMINK) amino acids (see Foster et al.<sup><a reveal-id="DSM019C11" data-open="DSM019C11" class="link link-ref link-reveal xref-bibr">11</a></sup> for details) are also very similar between these two species. In other words, their similarity in nucleotide composition is reflected in a similarity in the proportions of GC-rich and AT-rich amino acids. But when we compare the same two species for the levels of CA-rich (QTPHNK) and GT-rich (CVFGW) amino acids, we see large differences reflecting the differences in strand asymmetry between the mammals and the flatworms. For example, the liver fluke proteins contain more than twice as many Valine residues and more than five times as many cysteine residues as do their orthologs in the red deer. On the other hand, the liver flukes have approximately a third as many Glutamines and Threonines as are found in the red deer. These differences are also statistically highly significant (<em>P</em> &lt; 0.001) and they are entirely consistent with what we see for the average differences between mammals and flatworms (Fig.<a reveal-id="DSM019F3" data-open="DSM019F3" class="link link-reveal link-table xref-fig">3</a> and Supplementary Table S1).</p><p>A possible alternative explanation for these results is that the amino acid differences are the cause, rather than the consequence, of the strand asymmetries. We can exclude this possibility, however, for two reasons. First, there is an even larger strand asymmetry at the synonymous codon sites (Supplementary Fig. S1) suggesting that the nucleotide skew is counterbalanced, to some extent, by functional constraint at the protein level. In other words, protein function does have an effect, but as a constraint rather than a cause. Another way to illustrate this point is to calculate the GC skew at each codon position separately. The results (Supplementary Table S2) show that the greatest differences in GC skew occur at the third codon position and the least skew occurs at the second position. This is consistent with the fact that many changes at the third codon position alter the codon usage but do not affect the protein sequence. Secondly, in mammalian mitochondria, one gene (<em>ND6</em>) is encoded on the opposite strand from the other 12 genes and, in accordance with our prediction, the amino acid composition of this mammalian protein displays a pattern that is similar to that of the flatworm proteins rather than the other 12 mammalian proteins (Supplementary Fig. S2). Both of these observations indicate that the primary effect is at the level nucleotide asymmetry between the two strands of the mitochondrial genome and that this DNA bias causes a secondary effect at the level of protein composition.</p><p>We performed a number of further tests to explore the interplay between functional constraint at the protein level and nucleotide skew at the DNA level. Since we limited our comparison to orthologous mitochondrial genes, we have eliminated the effect of different types of proteins in the two groups of species. Moreover, since mitochondrial function is highly conserved in metazoan animals, we expect that the orthologous proteins are performing essentially the same functions in the two species groups. There is still, however, the remote possibility that there are differences in physiological conditions between the mitochondria of mammals and those of flatworms and that these differences could contribute to the differences in protein composition which we observe between the two groups. To control for this possibility, we examined the sequences of three mitochondrial ribosomal protein genes: L11, L15, and L20 (Supplementary Table S2). Although these proteins function in the mitochondria, they are encoded by genes in the nuclear genome. The results show that the gene sequences do not show the characteristic mitochondrial gene skews and there is no significant difference between mammals and Platyhelminthes, either at the DNA level or at the protein level. This indicates that the key factor underlying the species differences is not the functional environment of the proteins but rather the location of the genes encoding those proteins.</p><p>Since our results point to a mutational force at the DNA level that is counteracted by functional constraint at the protein level, we asked what would happen if we confined our analysis to a region of the protein where there was reason to believe that the functional constraint would be especially strong. The transmembrane domain portions of mitochondrial proteins provide an opportunity to do such an analysis. Specifically, we looked at the patterns of GC skew and amino acid composition in the transmembrane domain regions of the COXI gene. The results (Supplementary Table S2) show that the increased functional constraint on amino acid composition does indeed lead to a decreased difference in GC skew between the two species groups. For example, the difference in GC skew between mammals and flatworms for the complete dataset is 0.84, whereas it is reduced to 0.57 for the transmembrane domains, but the same pattern remains both at the DNA and protein levels. For instance, even within the transmembrane domains, the mammalian sequences have less cysteine and valine, and more glutamine and threonine than do their flatworm orthologs, reflecting the same patterns as shown in Figure<a reveal-id="DSM019F3" data-open="DSM019F3" class="link link-reveal link-table xref-fig">3</a>. In other words, increased functional constraint decreases the effect of nucleotide skew, but it does not completely eliminate it.</p><p>Overall, our results show that the DNA strand asymmetry of animal mitochondrial genomes affects the amino acid composition of encoded proteins. A similar effect has been noted previously in bacterial genomes<sup><a reveal-id="DSM019C9" data-open="DSM019C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSM019C10" data-open="DSM019C10" class="link link-ref link-reveal xref-bibr">10</a></sup> but it has not been reported in animal or plant genomes. Since we infer that the DNA strand bias is the cause rather than the consequence of the protein differences, this raises the question of what causes the DNA strand bias in the first place. In bacterial genomes, there is good evidence that it is related to DNA replication, based primarily on the fact that the direction of the bias switches at the origin of replication.<sup><a reveal-id="DSM019C4" data-open="DSM019C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSM019C7" data-open="DSM019C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSM019C8" data-open="DSM019C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Recent work indicates that although DNA strand biases are widespread in prokaryotes, eukaryotes, and viruses, the magnitude and the direction of the bias is variable, suggesting that the underlying causes are multifactorial.<sup><a reveal-id="DSM019C14" data-open="DSM019C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSM019C15" data-open="DSM019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In animal mitochondrial genomes, the strand bias appears to be caused by varying durations of time that the heavy strand spends in the mutagenic single-strand state during replication.<sup><a reveal-id="DSM019C16" data-open="DSM019C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In addition to the replication-associated effects, there is evidence that transcription can also generate DNA strand asymmetry in eukaryotes due to transcription-coupled mutations.<sup><a reveal-id="DSM019C17" data-open="DSM019C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSM019C18" data-open="DSM019C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>A recent study has shown that strand bias in mitochondrial sequences can lead to artefactual results in phylogenetic reconstructions.<sup><a reveal-id="DSM019C19" data-open="DSM019C19" class="link link-ref link-reveal xref-bibr">19</a></sup> This misleading result can be minimized by a recoding scheme that excludes transitions at rapidly evolving neutral sites.<sup><a reveal-id="DSM019C19" data-open="DSM019C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSM019C20" data-open="DSM019C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Our results, however, show that mitochondrial strand biases can also have significant effects at non-neutral sites that change the amino acid sequence. This means that some degree of bias remains even after the correction has been implemented. As stated by Jones et al. (2007)<sup><a reveal-id="DSM019C20" data-open="DSM019C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In the case of mitochondrial genes, strand-bias should be of particular concern and the previous use of mitochondrial genomes in resolving deep phylogenies requires critical re-evaluation.</p><p>It remains to be seen if DNA strand asymmetry can also affect the composition of proteins encoded by eukaryotic nuclear genes, as has been shown for biases in the GC content of nuclear genes.<sup><a reveal-id="DSM019C21" data-open="DSM019C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSM019C22" data-open="DSM019C22" class="link link-ref link-reveal xref-bibr">22</a></sup> It is already known that DNA strand asymmetries exist in nuclear genes<sup><a reveal-id="DSM019C17" data-open="DSM019C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSM019C18" data-open="DSM019C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSM019C23" data-open="DSM019C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSM019C24" data-open="DSM019C24" class="link link-ref link-reveal xref-bibr">24</a></sup> but their effects on the composition of nuclear-encoded proteins have not been studied.</p>                            <h2 scrollto-destination=87195052 id="87195052" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by a grant to DAH from the Natural Science and Engineering Research Council of Canada.</p>                            <h2 scrollto-destination=87195054 id="87195054" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM019C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>, <div class="name"><div class="surname">Averof</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A. T.</div></div>, <div class="name"><div class="surname">Matassi</div> <div class="given-names">G.</div></div>, <div class="name"><div class="surname">Peden</div> <div class="given-names">J. F.</div></div>.</div><div class="article-title">DNA sequence evolution: the sounds of silence</div>,<div class="source">Philos. Trans. R. Soc. Lond. B Biol. Sci.</div>,<div class="year">1995</div>,vol.<div class="volume">349</div>(pg.<div class="fpage">241</div>-<div class="lpage">247</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20sequence%20evolution%3A%20the%20sounds%20of%20silence&amp;author=P.%20M.%20Sharp&amp;author=M.%20Averof&amp;author=A.%20T.%20Lloyd&amp;author=G.%20Matassi&amp;author=J.%20F.%20Peden&amp;publication_year=1995&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;volume=349&amp;pages=241-247" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.1995.0108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.1995.0108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.1995.0108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8577834" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>.</div><div class="article-title">Influence of genomic G + C content on average amino-acid composition of proteins from 59 bacterial species</div>,<div class="source">Gene</div>,<div class="year">1997</div>,vol.<div class="volume">205</div>(pg.<div class="fpage">309</div>-<div class="lpage">316</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Influence%20of%20genomic%20G%20%2B%20C%20content%20on%20average%20amino-acid%20composition%20of%20proteins%20from%2059%20bacterial%20species&amp;author=J.%20R.%20Lobry&amp;publication_year=1997&amp;journal=Gene&amp;volume=205&amp;pages=309-316" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(97)00403-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(97)00403-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(97)00403-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9461405" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singer</div> <div class="given-names">G. A.</div></div>, <div class="name"><div class="surname">Hickey</div> <div class="given-names">D. A.</div></div>.</div><div class="article-title">Nucleotide bias causes a genomewide bias in the amino acid composition of proteins</div>,<div class="source">Mol. Biol. Evol.</div>,<div class="year">2000</div>,vol.<div class="volume">17</div>(pg.<div class="fpage">1581</div>-<div class="lpage">1588</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20bias%20causes%20a%20genomewide%20bias%20in%20the%20amino%20acid%20composition%20of%20proteins&amp;author=G.%20A.%20Singer&amp;author=D.%20A.%20Hickey&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1581-1588" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11070046" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>.</div><div class="article-title">Asymmetric substitution patterns in the two DNA strands of bacteria</div>,<div class="source">Mol. Biol. Evol.</div>,<div class="year">1996</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">660</div>-<div class="lpage">665</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%20in%20the%20two%20DNA%20strands%20of%20bacteria&amp;author=J.%20R.%20Lobry&amp;publication_year=1996&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=13&amp;pages=660-665" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025626" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025626"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8676740" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>.</div><div class="article-title">Two aspects of DNA base composition: G + C content and translation-coupled deviation from intra-strand rule of A = T and G = C</div>,<div class="source">J. Mol. Evol.</div>,<div class="year">1999</div>,vol.<div class="volume">49</div>(pg.<div class="fpage">49</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20aspects%20of%20DNA%20base%20composition%3A%20G%20%2B%20C%20content%20and%20translation-coupled%20deviation%20from%20intra-strand%20rule%20of%20A%20%3D%20T%20and%20G%20%3D%20C&amp;author=N.%20Sueoka&amp;publication_year=1999&amp;journal=J.%20Mol.%20Evol.&amp;volume=49&amp;pages=49-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10368434" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perna</div> <div class="given-names">N. T.</div></div>, <div class="name"><div class="surname">Kocher</div> <div class="given-names">T. D.</div></div>.</div><div class="article-title">Patterns of nucleotide composition at fourfold degenerate sites of animal mitochondrial genomes</div>,<div class="source">J. Mol. Evol.</div>,<div class="year">1995</div>,vol.<div class="volume">41</div>(pg.<div class="fpage">353</div>-<div class="lpage">358</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20nucleotide%20composition%20at%20fourfold%20degenerate%20sites%20of%20animal%20mitochondrial%20genomes&amp;author=N.%20T.%20Perna&amp;author=T.%20D.%20Kocher&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol.&amp;volume=41&amp;pages=353-358" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01215182" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01215182" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01215182"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7563121" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrzek</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>.</div><div class="article-title">Strand compositional asymmetry in bacterial and large viral genomes</div>,<div class="source">Proc. Natl. Acad. Sci. USA</div>,<div class="year">1998</div>,vol.<div class="volume">95</div>(pg.<div class="fpage">3720</div>-<div class="lpage">3725</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20compositional%20asymmetry%20in%20bacterial%20and%20large%20viral%20genomes&amp;author=J.%20Mr%C3%A1zek&amp;author=S.%20Karlin&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=3720-3725" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.7.3720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.7.3720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.7.3720"> </span></div></div></div></div></div></div><div content-id="DSM019C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLean</div> <div class="given-names">M. J.</div></div>, <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>, <div class="name"><div class="surname">Devine</div> <div class="given-names">K. M.</div></div>.</div><div class="article-title">Base composition skews, replication orientation, and gene orientation in 12 prokaryote genomes</div>,<div class="source">J. Mol. Evol.</div>,<div class="year">1998</div>,vol.<div class="volume">47</div>(pg.<div class="fpage">691</div>-<div class="lpage">696</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base%20composition%20skews%2C%20replication%20orientation%2C%20and%20gene%20orientation%20in%2012%20prokaryote%20genomes&amp;author=M.%20J.%20McLean&amp;author=K.%20H.%20Wolfe&amp;author=K.%20M.%20Devine&amp;publication_year=1998&amp;journal=J.%20Mol.%20Evol.&amp;volume=47&amp;pages=691-696" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847411" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P. C.</div></div>, <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Viari</div> <div class="given-names">A.</div></div>.</div><div class="article-title">Universal replication biases in bacteria</div>,<div class="source">Mol. Microbiol.</div>,<div class="year">1999</div>,vol.<div class="volume">32</div>(pg.<div class="fpage">11</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20replication%20biases%20in%20bacteria&amp;author=E.%20P.%20C.%20Rocha&amp;author=A.%20Danchin&amp;author=A.%20Viari&amp;publication_year=1999&amp;journal=Mol.%20Microbiol.&amp;volume=32&amp;pages=11-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01334.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1999.01334.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1999.01334.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10216855" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lafay</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A. T.</div></div>, <div class="name"><div class="surname">McLean</div> <div class="given-names">M. J.</div></div>, <div class="name"><div class="surname">Devine</div> <div class="given-names">K. M.</div></div>, <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>, <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>.</div><div class="article-title">Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases, Nucleic Acids Res</div>,<div class="year">1999</div>,vol.<div class="volume">27</div>(pg.<div class="fpage">1642</div>-<div class="lpage">1649</div>)</div></div></div></div><div content-id="DSM019C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foster</div> <div class="given-names">P. G.</div></div>, <div class="name"><div class="surname">Jermiin</div> <div class="given-names">L. S.</div></div>, <div class="name"><div class="surname">Hickey</div> <div class="given-names">D. A.</div></div>.</div><div class="article-title">Nucleotide composition bias affects amino acid content in proteins coded by animal mitochondria</div>,<div class="source">J. Mol. Evol.</div>,<div class="year">1997</div>,vol.<div class="volume">44</div>(pg.<div class="fpage">282</div>-<div class="lpage">288</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20composition%20bias%20affects%20amino%20acid%20content%20in%20proteins%20coded%20by%20animal%20mitochondria&amp;author=P.%20G.%20Foster&amp;author=L.%20S.%20Jermiin&amp;author=D.%20A.%20Hickey&amp;publication_year=1997&amp;journal=J.%20Mol.%20Evol.&amp;volume=44&amp;pages=282-288" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006145"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9060394" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le</div> <div class="given-names">T. H.</div></div>, <div class="name"><div class="surname">McManus</div> <div class="given-names">D.P.</div></div>, <div class="name"><div class="surname">Blair</div> <div class="given-names">D.</div></div>.</div><div class="article-title">Codon usage and bias in mitochondrial genomes of parasitic platyhelminthes</div>,<div class="source">Korean J. Parasitol.</div>,<div class="year">2004</div>,vol.<div class="volume">42</div>(pg.<div class="fpage">159</div>-<div class="lpage">167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20bias%20in%20mitochondrial%20genomes%20of%20parasitic%20platyhelminthes&amp;author=T.%20H.%20Le&amp;author=D.P.%20McManus&amp;author=D.%20Blair&amp;publication_year=2004&amp;journal=Korean%20J.%20Parasitol.&amp;volume=42&amp;pages=159-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3347/kjp.2004.42.4.159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3347%2Fkjp.2004.42.4.159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3347%2Fkjp.2004.42.4.159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15591833" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McManus</div> <div class="given-names">D. P.</div></div>, <div class="name"><div class="surname">Le</div> <div class="given-names">T. H.</div></div>, <div class="name"><div class="surname">Blair</div> <div class="given-names">D.</div></div>.</div><div class="article-title">Genomics of parasitic flatworms</div>,<div class="source">Int. J. Parasitol.</div>,<div class="year">2004</div>,vol.<div class="volume">34</div>(pg.<div class="fpage">153</div>-<div class="lpage">158</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20parasitic%20flatworms&amp;author=D.%20P.%20McManus&amp;author=T.%20H.%20Le&amp;author=D.%20Blair&amp;publication_year=2004&amp;journal=Int.%20J.%20Parasitol.&amp;volume=34&amp;pages=153-158" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijpara.2003.11.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijpara.2003.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijpara.2003.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15037102" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>, <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>.</div><div class="article-title">Ongoing evolution of strand composition in bacterial genomes</div>,<div class="source">Mol. Biol. Evol.</div>,<div class="year">2001</div>,vol.<div class="volume">18</div>(pg.<div class="fpage">1789</div>-<div class="lpage">1799</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ongoing%20evolution%20of%20strand%20composition%20in%20bacterial%20genomes&amp;author=E.%20P.%20Rocha&amp;author=A.%20Danchin&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=1789-1799" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003966" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003966" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003966"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11504858" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>, <div class="name"><div class="surname">Touchon</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Feil</div> <div class="given-names">E. J.</div></div>.</div><div class="article-title">Similar compositional biases are caused by very different mutational effects</div>,<div class="source">Genome Res.</div>,<div class="year">2006</div>,vol.<div class="volume">16</div>(pg.<div class="fpage">1537</div>-<div class="lpage">1547</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Similar%20compositional%20biases%20are%20caused%20by%20very%20different%20mutational%20effects&amp;author=E.%20P.%20Rocha&amp;author=M.%20Touchon&amp;author=E.%20J.%20Feil&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1537-1547" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5525106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5525106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5525106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17068325" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Faith</div> <div class="given-names">J. J.</div></div>, <div class="name"><div class="surname">Pollock</div> <div class="given-names">D. D</div></div>.</div><div class="article-title">Likelihood analysis of asymmetrical mutation bias gradients in vertebrate mitochondrial genomes</div>,<div class="source">Genetics</div>,<div class="year">2003</div>,vol.<div class="volume">165</div>(pg.<div class="fpage">735</div>-<div class="lpage">745</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Likelihood%20analysis%20of%20asymmetrical%20mutation%20bias%20gradients%20in%20vertebrate%20mitochondrial%20genomes&amp;author=J.%20J.%20Faith&amp;author=D.%20D%20Pollock&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=735-745" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14573484" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Niu</div> <div class="given-names">D. K.</div></div>, <div class="name"><div class="surname">Lin</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Y.</div></div>.</div><div class="article-title">Strand compositional asymmetries of nuclear DNA in eukaryotes</div>,<div class="source">J. Mol. Evol.</div>,<div class="year">2003</div>,vol.<div class="volume">57</div>(pg.<div class="fpage">325</div>-<div class="lpage">334</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20compositional%20asymmetries%20of%20nuclear%20DNA%20in%20eukaryotes&amp;author=D.%20K.%20Niu&amp;author=K.%20Lin&amp;author=D.Y.%20Zhang&amp;publication_year=2003&amp;journal=J.%20Mol.%20Evol.&amp;volume=57&amp;pages=325-334" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-003-2483-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-003-2483-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-003-2483-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14629042" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Touchon</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Arneodo</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">d'Aubenton-Carafa</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Thermes</div> <div class="given-names">C.</div></div>.</div><div class="article-title">Transcription-coupled and splicing-coupled strand asymmetries in eukaryotic genomes</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">2004</div>,vol.<div class="volume">32</div>(pg.<div class="fpage">4969</div>-<div class="lpage">4978</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-coupled%20and%20splicing-coupled%20strand%20asymmetries%20in%20eukaryotic%20genomes&amp;author=M.%20Touchon&amp;author=A.%20Arneodo&amp;author=Y.%20d'Aubenton-Carafa&amp;author=C.%20Thermes&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=4969-4978" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh823" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh823" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh823"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15388799" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hassanin</div> <div class="given-names">A.</div></div>.</div><div class="article-title">Phylogeny of Arthropoda inferred from mitochondrial sequences: strategies for limiting the misleading effects of multiple changes in pattern and rates of substitution</div>,<div class="source">Mol. Phylogenet. Evol.</div>,<div class="year">2006</div>,vol.<div class="volume">38</div>(pg.<div class="fpage">100</div>-<div class="lpage">116</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20Arthropoda%20inferred%20from%20mitochondrial%20sequences%3A%20strategies%20for%20limiting%20the%20misleading%20effects%20of%20multiple%20changes%20in%20pattern%20and%20rates%20of%20substitution&amp;author=A.%20Hassanin&amp;publication_year=2006&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=38&amp;pages=100-116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2005.09.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2005.09.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2005.09.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16290034" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Gantenbein</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Fet</div> <div class="given-names">V.</div></div>, <div class="name"><div class="surname">Blaxter</div> <div class="given-names">M.</div></div>.</div><div class="article-title">The effect of model choice on phylogenetic inference using mitochondrial sequence data: lessons from the scorpions</div>,<div class="source">Mol. Phylogenet. Evol.</div>,<div class="year">2007</div>,vol.<div class="volume">43</div>(pg.<div class="fpage">583</div>-<div class="lpage">595</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20model%20choice%20on%20phylogenetic%20inference%20using%20mitochondrial%20sequence%20data%3A%20lessons%20from%20the%20scorpions&amp;author=M.%20Jones&amp;author=B.%20Gantenbein&amp;author=V.%20Fet&amp;author=M.%20Blaxter&amp;publication_year=2007&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=43&amp;pages=583-595" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2006.11.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2006.11.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2006.11.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17275351" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Knight</div> <div class="given-names">R. D.</div></div>, <div class="name"><div class="surname">Freeland</div> <div class="given-names">S. J.</div></div>, <div class="name"><div class="surname">Landweber</div> <div class="given-names">L. F.</div></div>.</div><div class="article-title">A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes</div>,<div class="source">Genome Biol.</div>,<div class="year">2001</div>,vol.<div class="volume">2</div><div class="comment">RESEARCH0010</div></div></div></div></div><div content-id="DSM019C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H. C.</div></div>, <div class="name"><div class="surname">Singer</div> <div class="given-names">G. A.</div></div>, <div class="name"><div class="surname">Hickey</div> <div class="given-names">D. A.</div></div>.</div><div class="article-title">Mutational bias affects protein evolution in flowering plants</div>,<div class="source">Mol. Biol. Evol.</div>,<div class="year">2004</div>,vol.<div class="volume">21</div>(pg.<div class="fpage">90</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20bias%20affects%20protein%20evolution%20in%20flowering%20plants&amp;author=H.%20C.%20Wang&amp;author=G.%20A.%20Singer&amp;author=D.%20A.%20Hickey&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=21&amp;pages=90-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14595101" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Touchon</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Nicolay</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Arneodo</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">d'Aubenton-Carafa</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Thermes</div> <div class="given-names">C.</div></div>.</div><div class="article-title">Transcription-coupled TA and GC strand asymmetries in the human genome</div>,<div class="source">FEBS Lett.</div>,<div class="year">2003</div>,vol.<div class="volume">555</div>(pg.<div class="fpage">579</div>-<div class="lpage">582</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-coupled%20TA%20and%20GC%20strand%20asymmetries%20in%20the%20human%20genome&amp;author=M.%20Touchon&amp;author=S.%20Nicolay&amp;author=A.%20Arneodo&amp;author=Y.%20d'Aubenton-Carafa&amp;author=C.%20Thermes&amp;publication_year=2003&amp;journal=FEBS%20Lett.&amp;volume=555&amp;pages=579-582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(03)01306-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(03)01306-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(03)01306-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14675777" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSM019C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM019C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Touchon</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Nicolay</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Audit</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Brodie</div> <div class="given-names">E. B.</div></div>, <div class="name"><div class="surname">d'Aubenton-Carafa</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Arneodo</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Thermes</div> <div class="given-names">C.</div></div>.</div><div class="article-title">Replication-associated strand asymmetries in mammalian genomes: toward detection of replication origins</div>,<div class="source">Proc. Natl. Acad. Sci. USA</div>,<div class="year">2005</div>,vol.<div class="volume">102</div>(pg.<div class="fpage">9836</div>-<div class="lpage">9841</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication-associated%20strand%20asymmetries%20in%20mammalian%20genomes%3A%20toward%20detection%20of%20replication%20origins&amp;author=M.%20Touchon&amp;author=S.%20Nicolay&amp;author=B.%20Audit&amp;author=E.%20B.%20Brodie&amp;author=Y.%20d'Aubenton-Carafa&amp;author=A.%20Arneodo&amp;author=C.%20Thermes&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=9836-9841" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0500577102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0500577102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0500577102"> </span></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2007. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=350973&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm019/1/dsm019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=c85mFXeWGbo~qMtpt0vRH5lhj7CeGKb8GUesloh0t04eL9htqp1qYdRm0HP9wh090vjvLkNKjzf1NpzRdTZeGOTE3qQd2tKqlqwAoMhC5sdCJKemRm9upSfm~TssZo3O3zNnlN~i4yL~~kucLhgXc15uqe4S8OxDsXlKvz7qFnHJqkitU9-87CGAkl68M4viOEGcmLe4whdhU1XgFvw4Y8190AKorC0KQkT7~BURQf~A7KJJCQ~TZ0RKSTBgGluPoXN~1s0S8402gEOdpYUzzoux2gTFQn3WW5QY~F~wDdxy2yiP~BhCAmxou4kTnbKG8EMdP8LHvJGvHN3ENQIk9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">156</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000251879300001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">28</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm019"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="350973"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000251879300001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f5%2f201%2f350973">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000251879300001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (28)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F5%2F201%2F350973" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/350973'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="DNA Asymmetric Strand Bias Affects the Amino Acid Composition of Mitochondrial Proteins" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm019");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f201%2f350973">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

