

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/2/103/375923","@type":"ScholarlyArticle","name":"Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients","datePublished":"2008-02-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2008-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm035","keywords":["focused microarray","RNA diagnosis","PMBC","allergic granulomatosis angiitis","Churg–Strauss syndrome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/2/103/375923","author":[{"name":"Tougan, Takahiro","affiliation":"1  Department of Molecular Genetics ,  Osaka University ,  Osaka 565-0871 ,  Japan    2  DNA-chip Development Center for Infectious Diseases;  Research Institute for Microbial Diseases; Osaka University ,  3-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Onda, Hiroaki","affiliation":"1  Department of Molecular Genetics ,  Osaka University ,  Osaka 565-0871 ,  Japan    3  Innovation Plaza Osaka ,  3-1-10 Technostage, Izumi, Osaka 594-1144 ,  Japan","@type":"Person"},{"name":"Okuzaki, Daisuke","affiliation":"1  Department of Molecular Genetics ,  Osaka University ,  Osaka 565-0871 ,  Japan    2  DNA-chip Development Center for Infectious Diseases;  Research Institute for Microbial Diseases; Osaka University ,  3-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Kobayashi, Shigeto","affiliation":"4  Department of Rheumatology and Internal Medicine ,  Juntendo University School of Medicine ,  2-1-1 Hongo, Bunkyo-ku, Tokyo 113-8421 ,  Japan","@type":"Person"},{"name":"Hashimoto, Hiroshi","affiliation":"4  Department of Rheumatology and Internal Medicine ,  Juntendo University School of Medicine ,  2-1-1 Hongo, Bunkyo-ku, Tokyo 113-8421 ,  Japan","@type":"Person"},{"name":"Nojima, Hiroshi","affiliation":"1  Department of Molecular Genetics ,  Osaka University ,  Osaka 565-0871 ,  Japan    3  Innovation Plaza Osaka ,  3-1-10 Technostage, Izumi, Osaka 594-1144 ,  Japan","@type":"Person"}],"description":"Abstract.  DNA diagnostics are useful but are hampered by difficult ethical issues. Moreover, it cannot provide enough information on the environmental factors ","pageStart":"103","pageEnd":"114","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients" />
    <meta property="og:description" content="Abstract.  DNA diagnostics are useful but are hampered by difficult ethical issues. Moreover, it cannot provide enough information on the environmental factors " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/2/103/375923" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-02-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tougan, Takahiro" /><meta name="citation_author_institution" content="Department of Molecular Genetics, Osaka University, Osaka 565-0871, Japan" /><meta name="citation_author_institution" content="DNA-chip Development Center for Infectious Diseases;Research Institute for Microbial Diseases; Osaka University, 3-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Onda, Hiroaki" /><meta name="citation_author_institution" content="Department of Molecular Genetics, Osaka University, Osaka 565-0871, Japan" /><meta name="citation_author_institution" content="Innovation Plaza Osaka, 3-1-10 Technostage, Izumi, Osaka 594-1144, Japan" /><meta name="citation_author" content="Okuzaki, Daisuke" /><meta name="citation_author_institution" content="Department of Molecular Genetics, Osaka University, Osaka 565-0871, Japan" /><meta name="citation_author_institution" content="DNA-chip Development Center for Infectious Diseases;Research Institute for Microbial Diseases; Osaka University, 3-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Kobayashi, Shigeto" /><meta name="citation_author_institution" content="Department of Rheumatology and Internal Medicine, Juntendo University School of Medicine, 2-1-1 Hongo, Bunkyo-ku, Tokyo 113-8421, Japan" /><meta name="citation_author" content="Hashimoto, Hiroshi" /><meta name="citation_author_institution" content="Department of Rheumatology and Internal Medicine, Juntendo University School of Medicine, 2-1-1 Hongo, Bunkyo-ku, Tokyo 113-8421, Japan" /><meta name="citation_author" content="Nojima, Hiroshi" /><meta name="citation_author_institution" content="Department of Molecular Genetics, Osaka University, Osaka 565-0871, Japan" /><meta name="citation_author_institution" content="Innovation Plaza Osaka, 3-1-10 Technostage, Izumi, Osaka 594-1144, Japan" /><meta name="citation_title" content="Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients" /><meta name="citation_firstpage" content="103" /><meta name="citation_lastpage" content="114" /><meta name="citation_doi" content="10.1093/dnares/dsm035" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2008/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Extensivepolymorphisms observed in HIV-1 clade B protease gene using high-density oligonucleotide arrays; Nat. Med.;  citation_year=1996;  citation_volume=2;  citation_pages=753-759; " /><meta name="citation_reference" content="citation_title=DNA microarrays: a powerful genomic tool for biomedical and clinical research; Mol. Med.;  citation_year=2007;  citation_volume=13;  citation_pages=527-541; " /><meta name="citation_reference" content="citation_title=Elucidating genetic diversity with oligonucleotide arrays; Chromosome Res.;  citation_year=2005;  citation_volume=13;  citation_pages=225-235; " /><meta name="citation_reference" content="citation_title=Use of stepwise subtraction to comprehensively isolate mouse genes whose transcription is up-regulated during spermiogenesis; EMBO Rep.;  citation_year=2002;  citation_volume=3;  citation_pages=367-372; " /><meta name="citation_reference" content="citation_title=The American College of Rheumatology 1990 criteria for the classification of Churg-Strauss syndrome (allergic granulomatosis and angiitis); Arthritis Rheum;  citation_year=1990;  citation_volume=33;  citation_pages=1094-1100; " /><meta name="citation_reference" content="citation_title=Isolation and expression profiling of genes upregulated in the peripheral blood cells of systemic lupus erythematosus patients; DNA Res;  citation_year=2005;  citation_volume=112;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Large scale isolation of osteoclast-specific genes by an improved method involving the preparation of a subtracted cDNA library; Genes Cells;  citation_year=1998;  citation_volume=3;  citation_pages=459-475; " /><meta name="citation_reference" content="citation_title=The RNA recognition motif, a plastic RNA-binding platform to regulate post-transcriptional gene expression; FEBS J;  citation_year=2005;  citation_volume=272;  citation_pages=2118-2131; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of a novel mouse Siglec, mSiglec-F: differential evolution of the mouse and human (CD33) Siglec-3-related gene clusters; J. Biol. Chem.;  citation_year=2001;  citation_volume=276;  citation_pages=45128-45136; " /><meta name="citation_reference" content="citation_title=Phosphoesterase domains associated with DNA polymerases of diverse origins; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=3746-3752; " /><meta name="citation_reference" content="citation_title=Paxillin family members function as Csk binding proteins that regulate Lyn activity in human and murine platelets; Biochem. J.;  citation_year=2007;  citation_volume=403;  citation_pages=275-281; " /><meta name="citation_reference" content="citation_title=Paxillin: adapting to change; Physiol Rev.;  citation_year=2004;  citation_volume=84;  citation_pages=1315-1339; " /><meta name="citation_reference" content="citation_title=A new sequence motif linking lissencephaly, Treacher Collins and oral-facial-digital type 1 syndromes, microtubule dynamics and cell migration; Hum. Mol. Genet.;  citation_year=2001;  citation_volume=10;  citation_pages=2813-2820; " /><meta name="citation_reference" content="citation_title=Laminin isoforms and lung development: all isoforms are not equal; Dev. Biol.;  citation_year=2006;  citation_volume=294;  citation_pages=271-279; " /><meta name="citation_reference" content="citation_title=Deciphering B-ZIP transcription factor interactions in vitro and in vivo; Biochim. Biophys. Acta.;  citation_year=2006;  citation_volume=1759;  citation_pages=4-12; " /><meta name="citation_reference" content="citation_title=DNA-binding properties of the recombinant high-mobility-group-like AT-hook-containing region from human BRG1 protein; Biol. Chem.;  citation_year=2006;  citation_volume=387;  citation_pages=1469-1478; " /><meta name="citation_reference" content="citation_title=Mammalian sirtuins—emerging roles in physiology, aging, and calorie restriction; Genes Dev.;  citation_year=2006;  citation_volume=20;  citation_pages=2913-2921; " /><meta name="citation_reference" content="citation_title=The Arabidopsis root hair cell wall formation mutant lrx1 is suppressed by mutations in the RHM1 gene encoding a UDP-L-rhamnose synthase; Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=1630-1641; " /><meta name="citation_reference" content="citation_title=The coiled-coil domain of the adenovirus type 5 protein IX is dispensable for capsid incorporation and thermostability; J. Virol.;  citation_year=2005;  citation_volume=79;  citation_pages=3206-3210; " /><meta name="citation_reference" content="citation_title=Identification, cloning, and sequencing of a gene required for ferric vibriobactin utilization by Vibrio cholerae; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=5631-5638; " /><meta name="citation_reference" content="citation_title=Churg-Strauss syndrome; Curr Opin. Rheumatol;  citation_year=2007;  citation_volume=19;  citation_pages=25-32; " /><meta name="citation_reference" content="citation_title=Coactosin-like protein supports 5-lipoxygenase enzyme activity and up-regulates leukotriene A4 production; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=13150-13155; " /><meta name="citation_reference" content="citation_title=Cathepsin D, a lysosomal protease, regulates ABCA1-mediated lipid efflux; J. Biol. Chem;  citation_year=2006;  citation_volume=281;  citation_pages=39971-39981; " /><meta name="citation_reference" content="citation_title=Lysosomal proteolysis of prosaposin, the precursor of saposins (sphingolipid activator proteins): its mechanism and inhibition by ganglioside; Arch. Biochem. Biophys;  citation_year=1997;  citation_volume=341;  citation_pages=17-24; " /><meta name="citation_reference" content="citation_title=Ceramides and other bioactive sphingolipid backbones in health and disease: lipidomic analysis, metabolism and roles in membrane structure, dynamics, signaling and autophagy; Biochim. Biophys. Acta.;  citation_year=2006;  citation_volume=1758;  citation_pages=1864-1884; " /><meta name="citation_reference" content="citation_title=Lipopolysaccharide-induced c-Src expression plays a role in nitric oxide and TNFalpha secretion in macrophages; Mol. Immunol.;  citation_year=2006;  citation_volume=43;  citation_pages=308-316; " /><meta name="citation_reference" content="citation_title=Fgr deficiency results in defective eosinophil recruitment to the lung during allergic airway inflammation; J. Immunol.;  citation_year=2002;  citation_volume=168;  citation_pages=6446-6454; " /><meta name="citation_reference" content="citation_title=Src family kinases mediate neutrophil adhesion to adherent platelets; Blood;  citation_year=2007;  citation_volume=109;  citation_pages=2461-2469; " /><meta name="citation_reference" content="citation_title=Lysozyme amyloidosis: report of 4 cases and a review of the literature; Medicine (Baltimore);  citation_year=2007;  citation_volume=85;  citation_pages=66-73; " /><meta name="citation_reference" content="citation_title=Conjunctival inflammation and amyloidosis in allergic granulomatosis and angiitis (Churg-Strauss syndrome); Am. J. Ophthalmol.;  citation_year=1981;  citation_volume=91;  citation_pages=216-219; " /><meta name="citation_reference" content="citation_title=Abeta-related angiitis: primary angiitis of the central nervous system associated with cerebral amyloid angiopathy; Brain;  citation_year=2005;  citation_volume=128;  citation_pages=500-515; " /><meta name="citation_reference" content="citation_title=Receptor-type protein tyrosine phosphatase epsilon (PTPepsilonM) is a negative regulator of insulin signaling in primary hepatocytes and liver; Zoolog. Sci;  citation_year=2005;  citation_volume=22;  citation_pages=169-175; " /><meta name="citation_reference" content="citation_title=Protein tyrosine phosphatase epsilonC selectively inhibits interleukin-6- and interleukin- 10-induced JAK-STAT signaling; Blood;  citation_year=2001;  citation_volume=98;  citation_pages=3030-3034; " /><meta name="citation_reference" content="citation_title=The CXC-chemokine platelet factor 4 promotes monocyte survival and induces monocyte differentiation into macrophages; Blood;  citation_year=2000;  citation_volume=95;  citation_pages=1158-1166; " /><meta name="citation_reference" content="citation_title=Nicastrin functions as a gamma-secretase-substrate receptor; Cell;  citation_year=2005;  citation_volume=122;  citation_pages=435-447; " /><meta name="citation_reference" content="citation_title=Gamma-secretase is a functional component of phagosomes; J. Biol. Chem.;  citation_year=2005;  citation_volume=280;  citation_pages=36310-36317; " /><meta name="citation_reference" content="citation_title=Human P-selectin glycoprotein ligand-1 (PSGL-1) interacts with the skin-associated chemokine CCL27 via sulfated tyrosines at the PSGL-1 amino terminus; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=51775-51782; " /><meta name="citation_reference" content="citation_title=Autoantibody response against a novel testicular antigen protein highly expressed in testis (PHET) in SSc patients; Autoimmun. Rev.;  citation_year=2007;  citation_volume=6;  citation_pages=228-231; " /><meta name="citation_reference" content="citation_title=A role for heterologous gap junctions between melanoma and endothelial cells in metastasis; J. Clin. Invest.;  citation_year=2000;  citation_volume=105;  citation_pages=1189-1197; " /><meta name="citation_reference" content="citation_title=Comprehensive isolation of meiosis-specific genes identifies novel proteins and unusual non-coding transcripts in Schizosaccharomyces pombe; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=2327-2337; " /><meta name="citation_reference" content="citation_title=Isolation and expression profiling of genes upregulated in bone marrow-derived mononuclear cells of rheumatoid arthritis patients; DNA Res.;  citation_year=2006;  citation_volume=13;  citation_pages=169-183; " /><meta name="citation_reference" content="citation_title=Abundant poly(A)-bearing RNAs that lack open reading frames in Schizosaccharomyces pombe; DNA Res.;  citation_year=2002;  citation_volume=9;  citation_pages=209-215; " /><meta name="citation_reference" content="citation_title=Antineutrophil cytoplasmic antibodies (ANCA); Autoimmunity;  citation_year=2005;  citation_volume=38;  citation_pages=93-103; " /><meta name="citation_reference" content="citation_title=Mechanisms of Disease: pathogenesis and treatment of ANCA-associated vasculitides; Nat. Clin. Pract. Rheumatol;  citation_year=2006;  citation_volume=2;  citation_pages=661-670; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/2/103/1068495/dsm035.pdf" /><meta name="description" content="Abstract.  DNA diagnostics are useful but are hampered by difficult ethical issues. Moreover, it cannot provide enough information on the environmental factors " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/2/103/375923" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/2/103/375923" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/2/103/375923");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm035';
    </script>
<title>
	Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./375923" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYtsPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb3VnYW4sIFRha2FoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgR2VuZXRpY3MsIE9zYWthIFVuaXZlcnNpdHksIE9zYWthIDU2NS0wODcxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkROQS1jaGlwIERldmVsb3BtZW50IENlbnRlciBmb3IgSW5mZWN0aW91cyBEaXNlYXNlcztSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIE1pY3JvYmlhbCBEaXNlYXNlczsgT3Nha2EgVW5pdmVyc2l0eSwgMy0xIFlhbWFkYW9rYSwgU3VpdGEsIE9zYWthIDU2NS0wODcxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9uZGEsIEhpcm9ha2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBHZW5ldGljcywgT3Nha2EgVW5pdmVyc2l0eSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5ub3ZhdGlvbiBQbGF6YSBPc2FrYSwgMy0xLTEwIFRlY2hub3N0YWdlLCBJenVtaSwgT3Nha2EgNTk0LTExNDQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2t1emFraSwgRGFpc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIEdlbmV0aWNzLCBPc2FrYSBVbml2ZXJzaXR5LCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJETkEtY2hpcCBEZXZlbG9wbWVudCBDZW50ZXIgZm9yIEluZmVjdGlvdXMgRGlzZWFzZXM7UmVzZWFyY2ggSW5zdGl0dXRlIGZvciBNaWNyb2JpYWwgRGlzZWFzZXM7IE9zYWthIFVuaXZlcnNpdHksIDMtMSBZYW1hZGFva2EsIFN1aXRhLCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2JheWFzaGksIFNoaWdldG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFJoZXVtYXRvbG9neSBhbmQgSW50ZXJuYWwgTWVkaWNpbmUsIEp1bnRlbmRvIFVuaXZlcnNpdHkgU2Nob29sIG9mIE1lZGljaW5lLCAyLTEtMSBIb25nbywgQnVua3lvLWt1LCBUb2t5byAxMTMtODQyMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXNoaW1vdG8sIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFJoZXVtYXRvbG9neSBhbmQgSW50ZXJuYWwgTWVkaWNpbmUsIEp1bnRlbmRvIFVuaXZlcnNpdHkgU2Nob29sIG9mIE1lZGljaW5lLCAyLTEtMSBIb25nbywgQnVua3lvLWt1LCBUb2t5byAxMTMtODQyMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb2ppbWEsIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBHZW5ldGljcywgT3Nha2EgVW5pdmVyc2l0eSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5ub3ZhdGlvbiBQbGF6YSBPc2FrYSwgMy0xLTEwIFRlY2hub3N0YWdlLCBJenVtaSwgT3Nha2EgNTk0LTExNDQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJGb2N1c2VkIE1pY3JvYXJyYXkgQW5hbHlzaXMgb2YgUGVyaXBoZXJhbCBNb25vbnVjbGVhciBCbG9vZCBDZWxscyBmcm9tIENodXJn4oCTU3RyYXVzcyBTeW5kcm9tZSBQYXRpZW50cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNtMDM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmVwb2x5bW9ycGhpc21zIG9ic2VydmVkIGluIEhJVi0xIGNsYWRlIEIgcHJvdGVhc2UgZ2VuZSB1c2luZyBoaWdoLWRlbnNpdHkgb2xpZ29udWNsZW90aWRlIGFycmF5czsgTmF0LiBNZWQuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz03NTMtNzU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWljcm9hcnJheXM6IGEgcG93ZXJmdWwgZ2Vub21pYyB0b29sIGZvciBiaW9tZWRpY2FsIGFuZCBjbGluaWNhbCByZXNlYXJjaDsgTW9sLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NTI3LTU0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWx1Y2lkYXRpbmcgZ2VuZXRpYyBkaXZlcnNpdHkgd2l0aCBvbGlnb251Y2xlb3RpZGUgYXJyYXlzOyBDaHJvbW9zb21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMjUtMjM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2Ugb2Ygc3RlcHdpc2Ugc3VidHJhY3Rpb24gdG8gY29tcHJlaGVuc2l2ZWx5IGlzb2xhdGUgbW91c2UgZ2VuZXMgd2hvc2UgdHJhbnNjcmlwdGlvbiBpcyB1cC1yZWd1bGF0ZWQgZHVyaW5nIHNwZXJtaW9nZW5lc2lzOyBFTUJPIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTM2Ny0zNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBbWVyaWNhbiBDb2xsZWdlIG9mIFJoZXVtYXRvbG9neSAxOTkwIGNyaXRlcmlhIGZvciB0aGUgY2xhc3NpZmljYXRpb24gb2YgQ2h1cmctU3RyYXVzcyBzeW5kcm9tZSAoYWxsZXJnaWMgZ3JhbnVsb21hdG9zaXMgYW5kIGFuZ2lpdGlzKTsgQXJ0aHJpdGlzIFJoZXVtOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTA5NC0xMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGV4cHJlc3Npb24gcHJvZmlsaW5nIG9mIGdlbmVzIHVwcmVndWxhdGVkIGluIHRoZSBwZXJpcGhlcmFsIGJsb29kIGNlbGxzIG9mIHN5c3RlbWljIGx1cHVzIGVyeXRoZW1hdG9zdXMgcGF0aWVudHM7IEROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9MS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Ugc2NhbGUgaXNvbGF0aW9uIG9mIG9zdGVvY2xhc3Qtc3BlY2lmaWMgZ2VuZXMgYnkgYW4gaW1wcm92ZWQgbWV0aG9kIGludm9sdmluZyB0aGUgcHJlcGFyYXRpb24gb2YgYSBzdWJ0cmFjdGVkIGNETkEgbGlicmFyeTsgR2VuZXMgQ2VsbHM7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQ1OS00NzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSTkEgcmVjb2duaXRpb24gbW90aWYsIGEgcGxhc3RpYyBSTkEtYmluZGluZyBwbGF0Zm9ybSB0byByZWd1bGF0ZSBwb3N0LXRyYW5zY3JpcHRpb25hbCBnZW5lIGV4cHJlc3Npb247IEZFQlMgSjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz0yMTE4LTIxMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBub3ZlbCBtb3VzZSBTaWdsZWMsIG1TaWdsZWMtRjogZGlmZmVyZW50aWFsIGV2b2x1dGlvbiBvZiB0aGUgbW91c2UgYW5kIGh1bWFuIChDRDMzKSBTaWdsZWMtMy1yZWxhdGVkIGdlbmUgY2x1c3RlcnM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3NjsgIGNpdGF0aW9uX3BhZ2VzPTQ1MTI4LTQ1MTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaG9zcGhvZXN0ZXJhc2UgZG9tYWlucyBhc3NvY2lhdGVkIHdpdGggRE5BIHBvbHltZXJhc2VzIG9mIGRpdmVyc2Ugb3JpZ2luczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9Mzc0Ni0zNzUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXhpbGxpbiBmYW1pbHkgbWVtYmVycyBmdW5jdGlvbiBhcyBDc2sgYmluZGluZyBwcm90ZWlucyB0aGF0IHJlZ3VsYXRlIEx5biBhY3Rpdml0eSBpbiBodW1hbiBhbmQgbXVyaW5lIHBsYXRlbGV0czsgQmlvY2hlbS4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDAzOyAgY2l0YXRpb25fcGFnZXM9Mjc1LTI4MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF4aWxsaW46IGFkYXB0aW5nIHRvIGNoYW5nZTsgUGh5c2lvbCBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9MTMxNS0xMzM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBzZXF1ZW5jZSBtb3RpZiBsaW5raW5nIGxpc3NlbmNlcGhhbHksIFRyZWFjaGVyIENvbGxpbnMgYW5kIG9yYWwtZmFjaWFsLWRpZ2l0YWwgdHlwZSAxIHN5bmRyb21lcywgbWljcm90dWJ1bGUgZHluYW1pY3MgYW5kIGNlbGwgbWlncmF0aW9uOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MjgxMy0yODIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYW1pbmluIGlzb2Zvcm1zIGFuZCBsdW5nIGRldmVsb3BtZW50OiBhbGwgaXNvZm9ybXMgYXJlIG5vdCBlcXVhbDsgRGV2LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTQ7ICBjaXRhdGlvbl9wYWdlcz0yNzEtMjc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWNpcGhlcmluZyBCLVpJUCB0cmFuc2NyaXB0aW9uIGZhY3RvciBpbnRlcmFjdGlvbnMgaW4gdml0cm8gYW5kIGluIHZpdm87IEJpb2NoaW0uIEJpb3BoeXMuIEFjdGEuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3NTk7ICBjaXRhdGlvbl9wYWdlcz00LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEtYmluZGluZyBwcm9wZXJ0aWVzIG9mIHRoZSByZWNvbWJpbmFudCBoaWdoLW1vYmlsaXR5LWdyb3VwLWxpa2UgQVQtaG9vay1jb250YWluaW5nIHJlZ2lvbiBmcm9tIGh1bWFuIEJSRzEgcHJvdGVpbjsgQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg3OyAgY2l0YXRpb25fcGFnZXM9MTQ2OS0xNDc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYW1tYWxpYW4gc2lydHVpbnPigJRlbWVyZ2luZyByb2xlcyBpbiBwaHlzaW9sb2d5LCBhZ2luZywgYW5kIGNhbG9yaWUgcmVzdHJpY3Rpb247IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yOTEzLTI5MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBcmFiaWRvcHNpcyByb290IGhhaXIgY2VsbCB3YWxsIGZvcm1hdGlvbiBtdXRhbnQgbHJ4MSBpcyBzdXBwcmVzc2VkIGJ5IG11dGF0aW9ucyBpbiB0aGUgUkhNMSBnZW5lIGVuY29kaW5nIGEgVURQLUwtcmhhbW5vc2Ugc3ludGhhc2U7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNjMwLTE2NDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2lsZWQtY29pbCBkb21haW4gb2YgdGhlIGFkZW5vdmlydXMgdHlwZSA1IHByb3RlaW4gSVggaXMgZGlzcGVuc2FibGUgZm9yIGNhcHNpZCBpbmNvcnBvcmF0aW9uIGFuZCB0aGVybW9zdGFiaWxpdHk7IEouIFZpcm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT03OTsgIGNpdGF0aW9uX3BhZ2VzPTMyMDYtMzIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24sIGNsb25pbmcsIGFuZCBzZXF1ZW5jaW5nIG9mIGEgZ2VuZSByZXF1aXJlZCBmb3IgZmVycmljIHZpYnJpb2JhY3RpbiB1dGlsaXphdGlvbiBieSBWaWJyaW8gY2hvbGVyYWU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9NTYzMS01NjM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHVyZy1TdHJhdXNzIHN5bmRyb21lOyBDdXJyIE9waW4uIFJoZXVtYXRvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTI1LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2FjdG9zaW4tbGlrZSBwcm90ZWluIHN1cHBvcnRzIDUtbGlwb3h5Z2VuYXNlIGVuenltZSBhY3Rpdml0eSBhbmQgdXAtcmVndWxhdGVzIGxldWtvdHJpZW5lIEE0IHByb2R1Y3Rpb247IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTEzMTUwLTEzMTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXRoZXBzaW4gRCwgYSBseXNvc29tYWwgcHJvdGVhc2UsIHJlZ3VsYXRlcyBBQkNBMS1tZWRpYXRlZCBsaXBpZCBlZmZsdXg7IEouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjgxOyAgY2l0YXRpb25fcGFnZXM9Mzk5NzEtMzk5ODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUx5c29zb21hbCBwcm90ZW9seXNpcyBvZiBwcm9zYXBvc2luLCB0aGUgcHJlY3Vyc29yIG9mIHNhcG9zaW5zIChzcGhpbmdvbGlwaWQgYWN0aXZhdG9yIHByb3RlaW5zKTogaXRzIG1lY2hhbmlzbSBhbmQgaW5oaWJpdGlvbiBieSBnYW5nbGlvc2lkZTsgQXJjaC4gQmlvY2hlbS4gQmlvcGh5czsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0zNDE7ICBjaXRhdGlvbl9wYWdlcz0xNy0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VyYW1pZGVzIGFuZCBvdGhlciBiaW9hY3RpdmUgc3BoaW5nb2xpcGlkIGJhY2tib25lcyBpbiBoZWFsdGggYW5kIGRpc2Vhc2U6IGxpcGlkb21pYyBhbmFseXNpcywgbWV0YWJvbGlzbSBhbmQgcm9sZXMgaW4gbWVtYnJhbmUgc3RydWN0dXJlLCBkeW5hbWljcywgc2lnbmFsaW5nIGFuZCBhdXRvcGhhZ3k7IEJpb2NoaW0uIEJpb3BoeXMuIEFjdGEuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3NTg7ICBjaXRhdGlvbl9wYWdlcz0xODY0LTE4ODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpcG9wb2x5c2FjY2hhcmlkZS1pbmR1Y2VkIGMtU3JjIGV4cHJlc3Npb24gcGxheXMgYSByb2xlIGluIG5pdHJpYyBveGlkZSBhbmQgVE5GYWxwaGEgc2VjcmV0aW9uIGluIG1hY3JvcGhhZ2VzOyBNb2wuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MzA4LTMxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmdyIGRlZmljaWVuY3kgcmVzdWx0cyBpbiBkZWZlY3RpdmUgZW9zaW5vcGhpbCByZWNydWl0bWVudCB0byB0aGUgbHVuZyBkdXJpbmcgYWxsZXJnaWMgYWlyd2F5IGluZmxhbW1hdGlvbjsgSi4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTY4OyAgY2l0YXRpb25fcGFnZXM9NjQ0Ni02NDU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcmMgZmFtaWx5IGtpbmFzZXMgbWVkaWF0ZSBuZXV0cm9waGlsIGFkaGVzaW9uIHRvIGFkaGVyZW50IHBsYXRlbGV0czsgQmxvb2Q7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MjQ2MS0yNDY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MeXNvenltZSBhbXlsb2lkb3NpczogcmVwb3J0IG9mIDQgY2FzZXMgYW5kIGEgcmV2aWV3IG9mIHRoZSBsaXRlcmF0dXJlOyBNZWRpY2luZSAoQmFsdGltb3JlKTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTY2LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25qdW5jdGl2YWwgaW5mbGFtbWF0aW9uIGFuZCBhbXlsb2lkb3NpcyBpbiBhbGxlcmdpYyBncmFudWxvbWF0b3NpcyBhbmQgYW5naWl0aXMgKENodXJnLVN0cmF1c3Mgc3luZHJvbWUpOyBBbS4gSi4gT3BodGhhbG1vbC47ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0yMTYtMjE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYmV0YS1yZWxhdGVkIGFuZ2lpdGlzOiBwcmltYXJ5IGFuZ2lpdGlzIG9mIHRoZSBjZW50cmFsIG5lcnZvdXMgc3lzdGVtIGFzc29jaWF0ZWQgd2l0aCBjZXJlYnJhbCBhbXlsb2lkIGFuZ2lvcGF0aHk7IEJyYWluOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTUwMC01MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY2VwdG9yLXR5cGUgcHJvdGVpbiB0eXJvc2luZSBwaG9zcGhhdGFzZSBlcHNpbG9uIChQVFBlcHNpbG9uTSkgaXMgYSBuZWdhdGl2ZSByZWd1bGF0b3Igb2YgaW5zdWxpbiBzaWduYWxpbmcgaW4gcHJpbWFyeSBoZXBhdG9jeXRlcyBhbmQgbGl2ZXI7IFpvb2xvZy4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTY5LTE3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiB0eXJvc2luZSBwaG9zcGhhdGFzZSBlcHNpbG9uQyBzZWxlY3RpdmVseSBpbmhpYml0cyBpbnRlcmxldWtpbi02LSBhbmQgaW50ZXJsZXVraW4tIDEwLWluZHVjZWQgSkFLLVNUQVQgc2lnbmFsaW5nOyBCbG9vZDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTMwMzAtMzAzNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENYQy1jaGVtb2tpbmUgcGxhdGVsZXQgZmFjdG9yIDQgcHJvbW90ZXMgbW9ub2N5dGUgc3Vydml2YWwgYW5kIGluZHVjZXMgbW9ub2N5dGUgZGlmZmVyZW50aWF0aW9uIGludG8gbWFjcm9waGFnZXM7IEJsb29kOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MTE1OC0xMTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OaWNhc3RyaW4gZnVuY3Rpb25zIGFzIGEgZ2FtbWEtc2VjcmV0YXNlLXN1YnN0cmF0ZSByZWNlcHRvcjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz00MzUtNDQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYW1tYS1zZWNyZXRhc2UgaXMgYSBmdW5jdGlvbmFsIGNvbXBvbmVudCBvZiBwaGFnb3NvbWVzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yODA7ICBjaXRhdGlvbl9wYWdlcz0zNjMxMC0zNjMxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gUC1zZWxlY3RpbiBnbHljb3Byb3RlaW4gbGlnYW5kLTEgKFBTR0wtMSkgaW50ZXJhY3RzIHdpdGggdGhlIHNraW4tYXNzb2NpYXRlZCBjaGVtb2tpbmUgQ0NMMjcgdmlhIHN1bGZhdGVkIHR5cm9zaW5lcyBhdCB0aGUgUFNHTC0xIGFtaW5vIHRlcm1pbnVzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzk7ICBjaXRhdGlvbl9wYWdlcz01MTc3NS01MTc4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b2FudGlib2R5IHJlc3BvbnNlIGFnYWluc3QgYSBub3ZlbCB0ZXN0aWN1bGFyIGFudGlnZW4gcHJvdGVpbiBoaWdobHkgZXhwcmVzc2VkIGluIHRlc3RpcyAoUEhFVCkgaW4gU1NjIHBhdGllbnRzOyBBdXRvaW1tdW4uIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIyOC0yMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcm9sZSBmb3IgaGV0ZXJvbG9nb3VzIGdhcCBqdW5jdGlvbnMgYmV0d2VlbiBtZWxhbm9tYSBhbmQgZW5kb3RoZWxpYWwgY2VsbHMgaW4gbWV0YXN0YXNpczsgSi4gQ2xpbi4gSW52ZXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0xMTg5LTExOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgaXNvbGF0aW9uIG9mIG1laW9zaXMtc3BlY2lmaWMgZ2VuZXMgaWRlbnRpZmllcyBub3ZlbCBwcm90ZWlucyBhbmQgdW51c3VhbCBub24tY29kaW5nIHRyYW5zY3JpcHRzIGluIFNjaGl6b3NhY2NoYXJvbXljZXMgcG9tYmU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTIzMjctMjMzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBleHByZXNzaW9uIHByb2ZpbGluZyBvZiBnZW5lcyB1cHJlZ3VsYXRlZCBpbiBib25lIG1hcnJvdy1kZXJpdmVkIG1vbm9udWNsZWFyIGNlbGxzIG9mIHJoZXVtYXRvaWQgYXJ0aHJpdGlzIHBhdGllbnRzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTE2OS0xODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFidW5kYW50IHBvbHkoQSktYmVhcmluZyBSTkFzIHRoYXQgbGFjayBvcGVuIHJlYWRpbmcgZnJhbWVzIGluIFNjaGl6b3NhY2NoYXJvbXljZXMgcG9tYmU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMDktMjE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpbmV1dHJvcGhpbCBjeXRvcGxhc21pYyBhbnRpYm9kaWVzIChBTkNBKTsgQXV0b2ltbXVuaXR5OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9OTMtMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc21zIG9mIERpc2Vhc2U6IHBhdGhvZ2VuZXNpcyBhbmQgdHJlYXRtZW50IG9mIEFOQ0EtYXNzb2NpYXRlZCB2YXNjdWxpdGlkZXM7IE5hdC4gQ2xpbi4gUHJhY3QuIFJoZXVtYXRvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NjYxLTY3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE1LzIvMTAzLzEwNjg0OTUvZHNtMDM1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBETkEgZGlhZ25vc3RpY3MgYXJlIHVzZWZ1bCBidXQgYXJlIGhhbXBlcmVkIGJ5IGRpZmZpY3VsdCBldGhpY2FsIGlzc3Vlcy4gTW9yZW92ZXIsIGl0IGNhbm5vdCBwcm92aWRlIGVub3VnaCBpbmZvcm1hdGlvbiBvbiB0aGUgZW52aXJvbm1lbnRhbCBmYWN0b3JzICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mzc1OTIzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZNfG9dekR0ZCb8SXp8sepFG7Xlg6" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f103%2f375923" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f103%2f375923" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14648" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&amp;Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/2">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5472233">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472233">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472235">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472235">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472240">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472240">2. Patients, Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472254">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472254">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472285">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472285">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472293">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472293">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472295">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472295">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472297">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472297">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5472299">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5472299">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/2/93/376250">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Tougan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Genetics</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">Osaka 565-0871</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">DNA-chip Development Center for Infectious Diseases;</div><div class="insititution">Research Institute for Microbial Diseases; Osaka University</div>, <div class="addr-line">3-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Tougan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tougan T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tougan T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takahiro Tougan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroaki Onda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Genetics</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">Osaka 565-0871</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Innovation Plaza Osaka</div>, <div class="addr-line">3-1-10 Technostage, Izumi, Osaka 594-1144</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroaki+Onda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Onda H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Onda H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroaki Onda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daisuke Okuzaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Genetics</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">Osaka 565-0871</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">DNA-chip Development Center for Infectious Diseases;</div><div class="insititution">Research Institute for Microbial Diseases; Osaka University</div>, <div class="addr-line">3-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daisuke+Okuzaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okuzaki D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okuzaki D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daisuke Okuzaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeto Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Rheumatology and Internal Medicine</div>, <div class="insititution">Juntendo University School of Medicine</div>, <div class="addr-line">2-1-1 Hongo, Bunkyo-ku, Tokyo 113-8421</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeto+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigeto Kobayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Hashimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Rheumatology and Internal Medicine</div>, <div class="insititution">Juntendo University School of Medicine</div>, <div class="addr-line">2-1-1 Hongo, Bunkyo-ku, Tokyo 113-8421</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Hashimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hashimoto H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hashimoto H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Hashimoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Nojima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Genetics</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">Osaka 565-0871</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Innovation Plaza Osaka</div>, <div class="addr-line">3-1-10 Technostage, Izumi, Osaka 594-1144</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-6-6875-3980. Fax. <div class="fax">+81-6-6875-5192</div>. E-mail: <a href="mailto:snj-0212@biken.osaka-u.ac.jp" target="_blank">snj-0212@biken.osaka-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Nojima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nojima H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nojima H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Nojima</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Mitsuo Oshimura</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These two authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 2, 1 April 2008, Pages 103–114, <a href='https://doi.org/10.1093/dnares/dsm035'>https://doi.org/10.1093/dnares/dsm035</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 February 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 October 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 December 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="375923" href="/dnaresearch/article-pdf/15/2/103/1068495/dsm035.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takahiro Tougan, Hiroaki Onda, Daisuke Okuzaki, Shigeto Kobayashi, Hiroshi Hashimoto, Hiroshi Nojima;  Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients, <em>DNA Research</em>, Volume 15, Issue 2, 1 April 2008, Pages 103–114, <a href='https://doi.org/10.1093/dnares/dsm035'>https://doi.org/10.1093/dnares/dsm035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/375923?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/375923?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/375923?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/375923?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/375923?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/375923?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/375923?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Focused%20Microarray%20Analysis%20of%20Peripheral%20Mononuclear%20Blood%20Cells%20from%20Churg%E2%80%93Strauss%20Syndrome%20Patients&amp;publicationDate=2008-02-07&amp;volumeNum=15&amp;issueNum=2&amp;author=Tougan%2C%20Takahiro%3B%20Onda%2C%20Hiroaki&amp;startPage=103&amp;endPage=114&amp;contentId=10.1093%2Fdnares%2Fdsm035&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f103%2f375923" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5472233 id="5472233" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>DNA diagnostics are useful but are hampered by difficult ethical issues. Moreover, it cannot provide enough information on the environmental factors that are important for pathogenesis of certain diseases. However, this is not a problem for RNA diagnostics, which evaluate the expression of the gene in question. We here report a novel RNA diagnostics tool that can be employed with peripheral blood mononuclear cells (PBMCs). To establish this tool, we identified 290 genes that are highly expressed in normal PBMCs but not in TIG-1, a normal human fibroblast cell. These genes were entitled <em>PREP</em> after predominantly expressed in PBMC and included 50 uncharacterized genes. We then conducted <em>PREP</em> gene-focused microarray analysis on PBMCs from seven cases of Churg–Strauss syndrome (CSS), which is a small-vessel necrotizing vasculitis. We found that <em>PREP135</em> (coactosin-like protein), <em>PREP77</em> (prosaposin), <em>PREP191</em> (cathepsin D), <em>PREP234</em> (<em>c</em>-fgr), and <em>PREP136</em> (lysozyme) were very highly up-regulated in all seven CSS patients. Another 28 genes were also up-regulated, albeit more moderately, and three were down-regulated in all CSS patients. The nature of these up- and down-regulated genes suggest that the immune systems of the patients are activated in response to invading microorganisms. These observations indicate that focused microarray analysis of PBMCs may be a practical, useful, and low-cost bedside diagnostics tool.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;focused microarray&quot;">focused microarray</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RNA diagnosis&quot;">RNA diagnosis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="PMBC">PMBC</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;allergic granulomatosis angiitis&quot;">allergic granulomatosis angiitis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Churg–Strauss syndrome&quot;">Churg–Strauss syndrome</a></div>                            <h2 scrollto-destination=5472235 id="5472235" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The advent of array technology and the subsequent development of high-density oligonucleotide arrays<sup><a reveal-id="DSM035C1" data-open="DSM035C1" class="link link-ref link-reveal xref-bibr">1</a></sup> have been enormously helpful in improving our understanding of the genome-wide transcriptional profiles of many biological systems in both basic and applied research.<sup><a reveal-id="DSM035C2" data-open="DSM035C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Array technology has also been extremely useful for discovering and developing diagnostic gene markers for disease subcategories, disease prognosis, and treatment outcome; this has paved the way for effective pharmaceutical drug discovery, the development of novel strategies for molecular (DNA, RNA) diagnostics, and the design of personalized drug regimens.<sup><a reveal-id="DSM035C2" data-open="DSM035C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>Oligonucleotide microarrays, which were initially designed to analyze genome-wide gene expression levels, have turned out to be particularly useful in DNA diagnostics as they can be used for many different applications, including discovering polymorphisms and genotyping patients by identifying inheritable genetic markers.<sup><a reveal-id="DSM035C3" data-open="DSM035C3" class="link link-ref link-reveal xref-bibr">3</a></sup> With regard to the latter applications, such DNA diagnostics have greatly improved our understanding of and ability to detect the causative mutations of various diseases. However, this technology is hampered by major ethical and privacy issues since the genome sequence of a person carries unchangeable private information whose discovery can affect the life of not only that person, but also to some extent their family members. The need to protect patient privacy upon DNA diagnostics testing and the practical problems this constitutes also imposes extra costs on the testing procedure. Moreover, many kinds of diseases are caused not only by the DNA polymorphism, but also by environmental status. These environmental factors can be reflected in the RNA status, but DNA diagnostics usually cannot provide enough information on the environments. Thus, the activity of the disease can be detected by RNA or protein level but not by DNA level.</p><p>These problems are not faced by RNA diagnostics, in which the genome-wide mRNA levels are monitored by oligonucleotide (or cDNA) microarrays. Ethical issues of the nature described above do not apply here because the information obtained from RNA diagnostics does not necessarily relate to the DNA sequence. Instead, this technology monitors the dynamic changes in gene activities, namely, the increased or decreased transcriptional (mRNA) levels in the samples provided by the individual. The transcriptional levels would vary in the same individual depending on the health of the individual. In this sense, DNA diagnostics is a static test of the genome, whereas RNA diagnostics is a dynamic test of the genome.</p><p>However, it is not possible to monitor all 44 000 kinds of mRNA species that are transcribed from the human genome for many samples at the same time. A novel way to circumvent this problem is to develop a ‘focused array’ in which a limited number of mRNAs are tested in a low-density array. In the present study, we developed a focused oligonucleotide (or cDNA) array for use with patient peripheral blood mononuclear cells (PBMCs). For this, we selected the genes that are predominantly expressed in normal PBMCs, as determined by stepwise subtractive hybridization<sup><a reveal-id="DSM035C4" data-open="DSM035C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and by genome-wide cDNA microarray analysis. From the 290 ‘PBMC-focused’ genes we identified, we can prepare the PBMC-focused cDNA array. We examined the expression levels of these <em>PREP</em> genes to analyze the PBMC RNAs obtained from patients suffering the autoimmune disease Churg–Strauss syndrome (CSS), which is an alternative name of allergic granulomatosis angiitis, because the autoimmune response of CSS patients is expected to disturb the expression levels of immune-related genes in PBMC. Indeed, we identified several genes whose expressions are markedly up- or down-regulated in all CSS patients tested. These observations suggest that this low-cost RNA diagnostics test is useful, practical, and can be used at the bedside.</p>                            <h2 scrollto-destination=5472240 id="5472240" class="section-title" data-legacy-id=s2>2. Patients, Materials and methods</h2>
                            <h3 scrollto-destination=5472241 id="5472241" class="section-title" data-legacy-id=s2a>2.1. Human subjects: patients and healthy controls</h3>
<p>Blood was obtained from eight healthy volunteers (four males and four females; aged 25–49 years) for the cDNA library preparation. Blood was also obtained from seven cases of CSS patients whose profiles are shown in Supplementary Table S1 and 18 healthy controls (six males and 11 females; aged 25–86 years) for focused microarray analysis. CSS patients were diagnosed according to the diagnostic criteria of the American College of Rheumatology.<sup><a reveal-id="DSM035C5" data-open="DSM035C5" class="link link-ref link-reveal xref-bibr">5</a></sup> This study was reviewed and approved by the Internal Review Board of the Research Institute for Microbial Diseases, Osaka University. In accordance with the requirements of the Board, a written informed consent was obtained from each participant before venous blood samples were obtained. Serum samples were consecutively obtained regardless of the patient's symptom, active, or inactive phase.</p>                            <h3 scrollto-destination=5472243 id="5472243" class="section-title" data-legacy-id=s2b>2.2. Preparation of RNA</h3>
<p>The RNA of the PBMCs obtained from healthy volunteers was prepared as described previously.<sup><a reveal-id="DSM035C6" data-open="DSM035C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Briefly, heparinized venous blood (10 mL) was mixed with an equal volume of 2% dextran/saline solution and incubated at room temperature for 30 min to precipitate the red blood cells. Total RNA was extracted from the PBMC pellet by adding guanidine–thiocyanate solution and the samples were used for cDNA library preparation and subtractive hybridization.<sup><a reveal-id="DSM035C7" data-open="DSM035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Total RNA was also prepared by acid guanidinium–phenol–chloroform extraction for the DNA microarray, northern blot, and RT–PCR analyses. In some experiments, total RNA was synthesized using Ribo Max kit (Promega, Madison, WI) from the PBMC cDNA library. Total RNA or mRNA from human fibroblast TIG-1 cells was prepared as described previously.<sup><a reveal-id="DSM035C7" data-open="DSM035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Ex<em>Taq</em> DNA polymerase for RT–PCR was purchased from TaKaRa Co. Ltd. (Otsu, Japan). Probe labeling and detection for northern blots were performed by using the Gene Images Random-Prime Labelling and Detection System (GE Healthcare Bio-Sciences Corp., Piscataway, NJ).</p>                            <h3 scrollto-destination=5472245 id="5472245" class="section-title" data-legacy-id=s2c>2.3. Preparation of the subtracted cDNA library and stepwise subtraction</h3>
<p>Poly(A)<sup>+</sup> RNA was purified from total RNA by oligo(dT) cellulose chromatography. A cDNA library with eight million independent clones was constructed from the PBMC mRNAs by using the linker-primer method and the pAP3neo vector as described previously.<sup><a reveal-id="DSM035C7" data-open="DSM035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The poly(A)<sup>+</sup> RNAs from exponentially growing TIG-1 cells that had been incubated with 10% serum in tissue culture plates were also purified and biotinylated by using photobiotin. After converting the cDNA library to a single-stranded form by transfection with an f1 helper phage, we hybridized it with the biotinylated mRNAs and subtracted it by biotin–avidin interactions.<sup><a reveal-id="DSM035C7" data-open="DSM035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The unhybridized clones were converted to the double-stranded form and used to transform competent <em>Escherichia coli</em> cells. This generated a first-stage subtracted cDNA library of 11 million independent clones.</p><p>We then prepared plasmid DNA from ∼800 randomly selected cDNA clones and numbered and digested an aliquot of each plasmid DNA with <em>Sma</em>I and <em>Not</em>I restriction enzymes to prepare 10 sheets of Southern blots, each of which included 80 clones arranged in order. We then purified the cDNA inserts of clones 1–20 on 1% agarose gels by digesting them with <em>Eco</em>RI and <em>Not</em>I; these inserts were labeled with fluorescent dye and then used as probes for northern analysis with PBMC and TIG-1 RNAs to determine which clones contained <em>PREP</em> genes (data not shown). The DNA sequences of the <em>PREP</em> clones from the 5′ end of the cDNA inserts were determined by the dideoxy-chain termination reaction using an automatic DNA sequencer (Licor 4000L; Lincoln, NE). After these analyses, we selected the next 20 unhybridized clones on the Southern blot (from 21 onwards) for the next round of cDNA insert preparation, fluorescent labeling, and northern analysis. This procedure was repeated until we finished testing all 800 unhybridized clones. The <em>PREP</em> genes identified in the preceding step were then converted to RNA, biotinylated with photobiotin, and used to subtract the first-stage subtracted cDNA library. The second-stage subtracted cDNA library was then analyzed as described above and subtracted again. This process was repeated three times as described previously.<sup><a reveal-id="DSM035C4" data-open="DSM035C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>After the clones whose transcription was conspicuously up-regulated in PBMCs as compared with TIG-1 cells were sequenced, the DNA sequences were used to search the EST database by using the BLAST algorithm (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>).</p>                            <h3 scrollto-destination=5472249 id="5472249" class="section-title" data-legacy-id=s2d>2.4. Human cDNA microarray analysis for identification of PREP genes</h3>
<p>The total RNAs (500 ng) from the normal human PBMCs were pooled, as were the TIG-1 cell RNAs, after which both pools were reverse-transcribed by using oligo-dT primers containing the T7 RNA polymerase promoter sequence. The cDNAs were then subjected to <em>in vitro</em> transcription using T7 RNA polymerase to label the complementary RNAs (cRNAs) with cyanine 3 (Cy3)-CTP or cyanine 5 (Cy5)-CTP (Amersham Pharmacia Biotech, Piscataway, NJ). The Cy-labeled cRNAs from the normal human PBMCs (1 µg) were then mixed with the same amount of reverse color Cy-labeled TIG-1 cell-derived cRNAs. Hybridizations, rinsing, scanning, and gene analysis on the Agilent's all human cDNA microarray (Hu44K) were conducted according to the manufacturer's protocol (G2940BA; Agilent Technologies, Inc., Palo Alto, CA). Fluorophore reversal (dye swap) duplicates were used in two-color DNA microarray experiments. The 399 genes that showed the highest level of up- or down-regulation were selected and subjected to RT–PCR analysis using the normal human PBMC and TIG-1 RNAs (Fig. <a reveal-id="DSM035F1" data-open="DSM035F1" class="link link-reveal link-table xref-fig">1</a>) with relevant oligonucleotide primers (Supplementary Table S2). Of these, 122 genes were identified as <em>PREP</em> genes, 33 of which were already identified as <em>PREP</em> genes by stepwise subtraction.<sup><a reveal-id="DSM035C4" data-open="DSM035C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p>                        <a id="5472251" scrollto-destination="5472251"></a>
<div id="DSM035F1" data-id="DSM035F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03501a.jpeg?Expires=2147483647&amp;Signature=Q5hvRaXu9rOSQfk8rhF9X6PsWP6w-zYbsevifyYnrE5in2pbkHvn0rel95doYDFrD25xoRKMmDm3sv400gBh~mqF-F6QiyNLj8JUaKdRfUFVsspuL8dh0aENYCZdoMYCohAci5LlaMqm5id83BowDCPXzO3BCKjz2PwNRzQ53VebspE5RkqZdbak-eb1JCVoumTR7x4Z0eb4dcZXDnUox~pYHGoreL3jwts3wfn2Vr7Jl5KvcMuTyOFOrrEJbA5Il~7mTBsxZubr3ilzOPhno9Ktre8lPG0JoyJDxVeCEyebsafuWs8I4LtD1wdS9yA48PPeq6n6zxCRGcB4TZzFNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of PREP genes. Individual PREP cDNA clones were subjected to northern blot or RT–PCR analysis to compare their gene expression levels in the PBMCs of healthy volunteers (right lanes, denoted as P) and human fibroblast TIG-1 cells (left lanes, denoted as T). The names of the PREP genes are shown in Table 1. The RT–PCR data for GAPDH and the northern blot data for β-actin are also shown as loading controls. The annealing temperature and amplification cycles used in the RT–PCRs are denoted to the left of each image by a–h: a, 50°C and 35 cycles, b, 50°C and 30 cycles, c, 55°C and 35 cycles, d, 55°C and 40 cycles, e, 55°C and 45 cycles, f, 55°C and 30 cycles, and g, 50°C and 40 cycles, respectively. The northern blots are denoted as n." path-from-xml="dsm03501a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472251/dsm03501a.jpeg" path-from-xml="dsm03501a.jpeg" target="_blank">View large</a><a section="5472251" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03501a.jpeg?Expires=2147483647&amp;Signature=Z6E7~Zhrvh3AIeP6W3nQ-63e20RHDW91R7URRzc2ELD~y4iuw4PNgKzQJY9UGZSLAfAOnu6kvzQR-5rCcqScgmGbJy6gCuEmX2I8ctpdgBraZU9AMtdoiNeV~YlIQWl8HMIDJ9LVOaJAwb9ztZP6RziqQedP4Wf-5VHFqv~uZM-fQkf3HQGxVEhtqOwiZe-~ctvj-tGmirQNNER8Ecw5F2uIXfHMYvw9wO7AZjWE2WUPxggV4IgTC3xgFjjgdvXXQOSLetQh7z5Ml9WlAqyqyUx~hL~r~eXnZg05pxgF4LuI3Kbq6wdhufFTnpZ1lmBFL1eXdF2MCfMbyz5uDRHuag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5472251&amp;ar=375923&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm03501a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03501b.jpeg?Expires=2147483647&amp;Signature=NRZvOMbPM0k-51w6sgx4NUHx69tAKcz9dK6hhl7fZCL9DJhBNFbQmzeUCU5jV1om5g9f59DYUAsiVp5wswX~V78Ib6bR55x781QfotCItGVxICgYO3Ll-rF0ZdOXor1ZrZk1zu~ARC1XWotSYbK0Y~PGvT~siWRIH1s56pGRbwJLnBub5Fx3lziubbR2lgp3rR~4BHdivF5Zm6VZL6uLopHTU7SEBQBow-49d-IMYWJQbrm9PCacN0GqMG2LefrB-mA~~ZWvJu93jXX5JxKgfy~ZiardiZx4gQm7RPsOReExFTx-g9p7dtuL8Cke2PethGXVtXoCqaZDP82JwYE6FQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of PREP genes. Individual PREP cDNA clones were subjected to northern blot or RT–PCR analysis to compare their gene expression levels in the PBMCs of healthy volunteers (right lanes, denoted as P) and human fibroblast TIG-1 cells (left lanes, denoted as T). The names of the PREP genes are shown in Table 1. The RT–PCR data for GAPDH and the northern blot data for β-actin are also shown as loading controls. The annealing temperature and amplification cycles used in the RT–PCRs are denoted to the left of each image by a–h: a, 50°C and 35 cycles, b, 50°C and 30 cycles, c, 55°C and 35 cycles, d, 55°C and 40 cycles, e, 55°C and 45 cycles, f, 55°C and 30 cycles, and g, 50°C and 40 cycles, respectively. The northern blots are denoted as n." path-from-xml="dsm03501b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472251/dsm03501b.jpeg" path-from-xml="dsm03501b.jpeg" target="_blank">View large</a><a section="5472251" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03501b.jpeg?Expires=2147483647&amp;Signature=fmrT3XfCs7zaGwhIqrXtrPNND9Bm9NsLH~VJb9EJjTLxzfYiy~OlStHW~PH35o35RORPH6gQ6CwuIuujEZgoFcL0YsOSwZy~MJbgnatJ2Jvec4hyHAH2hoZRDctemYAU2LbuAVimVhi0JR6fjRH24Se-MK76~UdyaekQNbdtu5shD01NaKY1aDT3~ScgWZrX0om-VIkYf1tmfjjg6q10CLc~6J5HxoKc2HuKO4uX2NJUi-nsKZFjLZVSGQTiIolQnC3SIU5ui8yRG5UaAZ6fVZ~bQJzJirmOk4uvJNvBTdyKcw8E0JH8JRRcWQo0WbgYJTP-qInyesCzWK4a5Gkmhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5472251&amp;ar=375923&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm03501b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of <em>PREP</em> genes. Individual <em>PREP</em> cDNA clones were subjected to northern blot or RT–PCR analysis to compare their gene expression levels in the PBMCs of healthy volunteers (right lanes, denoted as P) and human fibroblast TIG-1 cells (left lanes, denoted as T). The names of the <em>PREP</em> genes are shown in Table 1. The RT–PCR data for <em>GAPDH</em> and the northern blot data for β-actin are also shown as loading controls. The annealing temperature and amplification cycles used in the RT–PCRs are denoted to the left of each image by a–h: a, 50°C and 35 cycles, b, 50°C and 30 cycles, c, 55°C and 35 cycles, d, 55°C and 40 cycles, e, 55°C and 45 cycles, f, 55°C and 30 cycles, and g, 50°C and 40 cycles, respectively. The northern blots are denoted as n.</p></div></div><div content-id="DSM035F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03501a.jpeg?Expires=2147483647&amp;Signature=Q5hvRaXu9rOSQfk8rhF9X6PsWP6w-zYbsevifyYnrE5in2pbkHvn0rel95doYDFrD25xoRKMmDm3sv400gBh~mqF-F6QiyNLj8JUaKdRfUFVsspuL8dh0aENYCZdoMYCohAci5LlaMqm5id83BowDCPXzO3BCKjz2PwNRzQ53VebspE5RkqZdbak-eb1JCVoumTR7x4Z0eb4dcZXDnUox~pYHGoreL3jwts3wfn2Vr7Jl5KvcMuTyOFOrrEJbA5Il~7mTBsxZubr3ilzOPhno9Ktre8lPG0JoyJDxVeCEyebsafuWs8I4LtD1wdS9yA48PPeq6n6zxCRGcB4TZzFNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of PREP genes. Individual PREP cDNA clones were subjected to northern blot or RT–PCR analysis to compare their gene expression levels in the PBMCs of healthy volunteers (right lanes, denoted as P) and human fibroblast TIG-1 cells (left lanes, denoted as T). The names of the PREP genes are shown in Table 1. The RT–PCR data for GAPDH and the northern blot data for β-actin are also shown as loading controls. The annealing temperature and amplification cycles used in the RT–PCRs are denoted to the left of each image by a–h: a, 50°C and 35 cycles, b, 50°C and 30 cycles, c, 55°C and 35 cycles, d, 55°C and 40 cycles, e, 55°C and 45 cycles, f, 55°C and 30 cycles, and g, 50°C and 40 cycles, respectively. The northern blots are denoted as n." path-from-xml="dsm03501a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472251/dsm03501a.jpeg" path-from-xml="dsm03501a.jpeg" target="_blank">View large</a><a section="5472251" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03501a.jpeg?Expires=2147483647&amp;Signature=Z6E7~Zhrvh3AIeP6W3nQ-63e20RHDW91R7URRzc2ELD~y4iuw4PNgKzQJY9UGZSLAfAOnu6kvzQR-5rCcqScgmGbJy6gCuEmX2I8ctpdgBraZU9AMtdoiNeV~YlIQWl8HMIDJ9LVOaJAwb9ztZP6RziqQedP4Wf-5VHFqv~uZM-fQkf3HQGxVEhtqOwiZe-~ctvj-tGmirQNNER8Ecw5F2uIXfHMYvw9wO7AZjWE2WUPxggV4IgTC3xgFjjgdvXXQOSLetQh7z5Ml9WlAqyqyUx~hL~r~eXnZg05pxgF4LuI3Kbq6wdhufFTnpZ1lmBFL1eXdF2MCfMbyz5uDRHuag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5472251&amp;ar=375923&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm03501a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03501b.jpeg?Expires=2147483647&amp;Signature=NRZvOMbPM0k-51w6sgx4NUHx69tAKcz9dK6hhl7fZCL9DJhBNFbQmzeUCU5jV1om5g9f59DYUAsiVp5wswX~V78Ib6bR55x781QfotCItGVxICgYO3Ll-rF0ZdOXor1ZrZk1zu~ARC1XWotSYbK0Y~PGvT~siWRIH1s56pGRbwJLnBub5Fx3lziubbR2lgp3rR~4BHdivF5Zm6VZL6uLopHTU7SEBQBow-49d-IMYWJQbrm9PCacN0GqMG2LefrB-mA~~ZWvJu93jXX5JxKgfy~ZiardiZx4gQm7RPsOReExFTx-g9p7dtuL8Cke2PethGXVtXoCqaZDP82JwYE6FQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of PREP genes. Individual PREP cDNA clones were subjected to northern blot or RT–PCR analysis to compare their gene expression levels in the PBMCs of healthy volunteers (right lanes, denoted as P) and human fibroblast TIG-1 cells (left lanes, denoted as T). The names of the PREP genes are shown in Table 1. The RT–PCR data for GAPDH and the northern blot data for β-actin are also shown as loading controls. The annealing temperature and amplification cycles used in the RT–PCRs are denoted to the left of each image by a–h: a, 50°C and 35 cycles, b, 50°C and 30 cycles, c, 55°C and 35 cycles, d, 55°C and 40 cycles, e, 55°C and 45 cycles, f, 55°C and 30 cycles, and g, 50°C and 40 cycles, respectively. The northern blots are denoted as n." path-from-xml="dsm03501b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472251/dsm03501b.jpeg" path-from-xml="dsm03501b.jpeg" target="_blank">View large</a><a section="5472251" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03501b.jpeg?Expires=2147483647&amp;Signature=fmrT3XfCs7zaGwhIqrXtrPNND9Bm9NsLH~VJb9EJjTLxzfYiy~OlStHW~PH35o35RORPH6gQ6CwuIuujEZgoFcL0YsOSwZy~MJbgnatJ2Jvec4hyHAH2hoZRDctemYAU2LbuAVimVhi0JR6fjRH24Se-MK76~UdyaekQNbdtu5shD01NaKY1aDT3~ScgWZrX0om-VIkYf1tmfjjg6q10CLc~6J5HxoKc2HuKO4uX2NJUi-nsKZFjLZVSGQTiIolQnC3SIU5ui8yRG5UaAZ6fVZ~bQJzJirmOk4uvJNvBTdyKcw8E0JH8JRRcWQo0WbgYJTP-qInyesCzWK4a5Gkmhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5472251&amp;ar=375923&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm03501b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of <em>PREP</em> genes. Individual <em>PREP</em> cDNA clones were subjected to northern blot or RT–PCR analysis to compare their gene expression levels in the PBMCs of healthy volunteers (right lanes, denoted as P) and human fibroblast TIG-1 cells (left lanes, denoted as T). The names of the <em>PREP</em> genes are shown in Table 1. The RT–PCR data for <em>GAPDH</em> and the northern blot data for β-actin are also shown as loading controls. The annealing temperature and amplification cycles used in the RT–PCRs are denoted to the left of each image by a–h: a, 50°C and 35 cycles, b, 50°C and 30 cycles, c, 55°C and 35 cycles, d, 55°C and 40 cycles, e, 55°C and 45 cycles, f, 55°C and 30 cycles, and g, 50°C and 40 cycles, respectively. The northern blots are denoted as n.</p></div></div>                            <h3 scrollto-destination=5472252 id="5472252" class="section-title" data-legacy-id=s2e>2.5. Individual microarray analysis on CSS patients</h3>
<p>The quality of the RNA samples obtained from PBMCs of each CSS patient was examined by using the RNA 6000 Nano LabChip Kit (p/n 5065-4476) on the Agilent 2100 Bioanalyzer. To conduct the individual cDNA microarray (Hu44K) analysis on CSS patients and normal volunteers, we generated fluorescently labeled cRNA by <em>in vitro</em> transcription with T7 RNA polymerase in the presence of Cy5-CTP or Cy3-CTP using a Low RNA Input Fluorescent Linear Amplification Kit (Agilent Technologies, Inc.) as described above. The Cy5-labled cRNA of each patient was then mixed with the Cy3-labled cRNA of normal volunteers to probe the cDNA microarray. <em>PREP</em> gene expression analysis was then conducted by using the Gene Spring software 7.3.1 (Agilent Technologies, Inc.) by setting appropriate parameters to select 33 or 3 <em>PREP</em> genes whose expressions are up-regulated (&gt;1.2-fold change) or down-regulated (&lt;1.0-fold change) in all seven CSS patients as compared with healthy volunteers. We also conducted similar analysis to select nine conspicuously up-regulated <em>PREP</em> genes (&gt;2.0-fold change).</p>                            <h2 scrollto-destination=5472254 id="5472254" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5472255 id="5472255" class="section-title" data-legacy-id=s3a>3.1. Isolation of putative PBMC-specific genes</h3>
<p>To isolate putative human PBMC-specific genes, we first prepared mRNAs from the PBMCs of healthy volunteers and generated a human PBMC cDNA library with 80 million independent clones by using the linker-primer method described previously.<sup><a reveal-id="DSM035C7" data-open="DSM035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> We also prepared mRNAs from exponentially growing normal human fibroblast TIG-1 cells, which are control cells that express non-PBMC transcripts. These mRNAs were biotinylated with photobiotin<sup><a reveal-id="DSM035C7" data-open="DSM035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and used to subtract the PBMC cDNA library as described previously<sup><a reveal-id="DSM035C4" data-open="DSM035C4" class="link link-ref link-reveal xref-bibr">4</a></sup> to remove the housekeeping and non-PBMC-specific genes. Briefly, after converting the PBMC cDNA library to a single-stranded form by transfection with f1 helper phage, we hybridized it with the biotinylated TIG-1 mRNAs and subtracted it by biotin–avidin interactions. The unhybridized clones were converted to the double-stranded form, which was then used to transform competent <em>E. coli</em> cells, thereby generating a subtracted cDNA library.</p><p>To analyze this subtracted cDNA library, we prepared plasmid DNA from several hundred randomly selected and numbered cDNA clones. An aliquot of each plasmid DNA was then digested with <em>Eco</em>RI and <em>Not</em>I and their cDNA inserts were purified on 1% agarose gels and fluorescently labeled. These probes were then used in northern analysis to identify those genes whose transcript levels were much higher in PBMCs than in TIG-1 cells, namely, those genes that showed almost a plus/minus type difference. These genes were named <em>PREP</em> after predominantly expressed in PBMC. As shown in Fig. <a reveal-id="DSM035F1" data-open="DSM035F1" class="link link-reveal link-table xref-fig">1</a>, each northern sheet only included two lanes, one each for the RNAs extracted from PBMCs and TIG-1. The DNA sequences of these candidate <em>PREP</em> genes were determined from the 5′ end of the cDNA inserts by the dideoxy-chain termination reaction and were used to search the EST database by employing the BLAST algorithm (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>). For those candidate <em>PREP</em> genes whose northern blot signals were weak or undetectable, we examined whether these candidate genes were indeed up-regulated by RT–PCR using oligonucleotide primers based on the DNA sequences of these genes (Supplementary Table S2) (Fig. <a reveal-id="DSM035F1" data-open="DSM035F1" class="link link-reveal link-table xref-fig">1</a>). These procedures were repeated until almost all subtracted cDNAs were tested, as described previously.<sup><a reveal-id="DSM035C4" data-open="DSM035C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The end result was the isolation of 197 <em>PREP</em> genes.</p><p>To reduce the possibility of missing important PBMC-specific pathogenic genes by the stepwise subtraction method, we also performed in parallel a genome-wide cDNA microarray analysis by using Agilent's Hu44K array with the same pooled RNA samples obtained from the PBMCs and TIG-1. The 399 genes that showed the greatest up-regulation in PBMCs compared with TIG-1 were then tested by RT–PCR. This resulted in the identification of 126 <em>PREP</em> genes, 33 of which were the same as those obtained by the stepwise subtraction analysis. The candidate <em>PREP</em> genes were tested by northern and/or RT–PCR analyses to confirm their PBMC-specifc expression (Fig. <a reveal-id="DSM035F1" data-open="DSM035F1" class="link link-reveal link-table xref-fig">1</a>). We did not test those genes that showed lower up-regulation in PBMCs than the 399 genes because the efficiency of detecting a <em>PREP</em> was conspicuously lower in the top 151–400 microarray genes than in the top 1–150 genes. Thus, it appears that while the stepwise subtraction method failed to identify 93 <em>PREP</em> genes, the DNA microarray analysis failed to detect 164 <em>PREP</em> genes. This disparity is mainly due to the differently sized drivers or probes used for hybridization; cDNA subtraction uses kilobase-order biotinylated mRNA as the driver for subtraction, while DNA microarrays are probed by 60-base oligonucleotides. These data together indicate that we identified 290 <em>PREP</em> genes (Supplementary Table S3).</p>                            <h3 scrollto-destination=5472259 id="5472259" class="section-title" data-legacy-id=s3b>3.2. Characterization of the unknown <em>PREP</em> genes</h3>
<p>Of the 290 <em>PREP</em> genes that we identified, <em>PREP1–PREP50</em> are uncharacterized genes. Homology and motif search of the gene products (Prep1–Prep50 proteins) using the Motif Scan algorithm (<a class="link link-uri" href="http://myhits.isb-sib.ch/cgi-bin/motif_scan">http://myhits.isb-sib.ch/cgi-bin/motif_scan</a>) revealed the following notable motifs in <em>PREP1–PREP11</em> (Fig. <a reveal-id="DSM035F2" data-open="DSM035F2" class="link link-reveal link-table xref-fig">2</a>), which may suggest their putative physiological functions. The other uncharacterized <em>PREP</em> genes did not have any notable motifs.</p>                        <a id="5472261" scrollto-destination="5472261"></a>
<div id="DSM035F2" data-id="DSM035F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03502.jpeg?Expires=2147483647&Signature=sC0-Ok2tWGpjlM0GojNIsAS-X1LqkpjVwS-mZ16ZxZGX5Am2l9T31hursNGYRjuqr7GQtRbBUho0KRBEK3yESW-gjbzdNRu2Xb22riWngflIBOVbOAGaV9qPI8r4xAD-STHLEIld06AFN97AGiAGJ-jtoXvfc-1E6favsUV9dECYJ7~xE58dmOAAWe8jiDi5gV6-702D2AB9--M84KZNXEpBzjDuVG24MqbEYouXngknspngzcfWQtJ4HtrY6hrOVRD0zx~3RRYNxgceR30daMyr-o5zq6j8BeE00nbbHuZhCP3Id7Yf8JmbbNwWVEz7QqzuYpvkt714MBgroGVdug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic depiction of the uncharacterized gene products Prep1-Prep11. Numbers indicate the amino acids of the protein. RRM, RNA recognition motif; GEF, Guanine nucleotide exchange factor; PH, pleckstrin homology; LisH, LIS1 (lissencephaly) homology; bZIP, basic region/leucine zipper; Sir2, sirtuin 2; FAD, flavin adenine dinucleotide." path-from-xml="dsm03502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472261/dsm03502.jpeg" path-from-xml="dsm03502.jpeg" target="_blank">View large</a><a section="5472261" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03502.jpeg?Expires=2147483647&Signature=xn5Te6eX~s4MoEb4sPdkKAt1GseR2Wio3Ng~h48vcVtMqxBLix~YngcCNOzkozWaqBgioeS7WYAWItL90fDLNI0bjxVwJfyCQGTDCnEovmUgYE0GaLxcKH7AXboiDVXQpePdZ1M7l-wn-xDtrZMus4ph6LMDGN9cDhTLIU5HW~TbY4LL1EXzecGp3QT~ff85GQJMwgz6f53NiNnJZzQeMmCg5gMhyI4gkht9kq~9Jmf1L6NgLrY6QZUH7nXFNv14ECMn8q3YWsGNlWEyUDoYtOI5KT~IMM7a6FbPyzVVJ0UoN~oaGMPV3LcvAML5Qjy1U1iUYmFZVv~qk9ggz4xVKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472261&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic depiction of the uncharacterized gene products Prep1-Prep11. Numbers indicate the amino acids of the protein. RRM, RNA recognition motif; GEF, Guanine nucleotide exchange factor; PH, pleckstrin homology; LisH, LIS1 (lissencephaly) homology; bZIP, basic region/leucine zipper; Sir2, sirtuin 2; FAD, flavin adenine dinucleotide.</p></div></div><div content-id="DSM035F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03502.jpeg?Expires=2147483647&Signature=sC0-Ok2tWGpjlM0GojNIsAS-X1LqkpjVwS-mZ16ZxZGX5Am2l9T31hursNGYRjuqr7GQtRbBUho0KRBEK3yESW-gjbzdNRu2Xb22riWngflIBOVbOAGaV9qPI8r4xAD-STHLEIld06AFN97AGiAGJ-jtoXvfc-1E6favsUV9dECYJ7~xE58dmOAAWe8jiDi5gV6-702D2AB9--M84KZNXEpBzjDuVG24MqbEYouXngknspngzcfWQtJ4HtrY6hrOVRD0zx~3RRYNxgceR30daMyr-o5zq6j8BeE00nbbHuZhCP3Id7Yf8JmbbNwWVEz7QqzuYpvkt714MBgroGVdug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic depiction of the uncharacterized gene products Prep1-Prep11. Numbers indicate the amino acids of the protein. RRM, RNA recognition motif; GEF, Guanine nucleotide exchange factor; PH, pleckstrin homology; LisH, LIS1 (lissencephaly) homology; bZIP, basic region/leucine zipper; Sir2, sirtuin 2; FAD, flavin adenine dinucleotide." path-from-xml="dsm03502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472261/dsm03502.jpeg" path-from-xml="dsm03502.jpeg" target="_blank">View large</a><a section="5472261" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03502.jpeg?Expires=2147483647&Signature=xn5Te6eX~s4MoEb4sPdkKAt1GseR2Wio3Ng~h48vcVtMqxBLix~YngcCNOzkozWaqBgioeS7WYAWItL90fDLNI0bjxVwJfyCQGTDCnEovmUgYE0GaLxcKH7AXboiDVXQpePdZ1M7l-wn-xDtrZMus4ph6LMDGN9cDhTLIU5HW~TbY4LL1EXzecGp3QT~ff85GQJMwgz6f53NiNnJZzQeMmCg5gMhyI4gkht9kq~9Jmf1L6NgLrY6QZUH7nXFNv14ECMn8q3YWsGNlWEyUDoYtOI5KT~IMM7a6FbPyzVVJ0UoN~oaGMPV3LcvAML5Qjy1U1iUYmFZVv~qk9ggz4xVKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472261&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic depiction of the uncharacterized gene products Prep1-Prep11. Numbers indicate the amino acids of the protein. RRM, RNA recognition motif; GEF, Guanine nucleotide exchange factor; PH, pleckstrin homology; LisH, LIS1 (lissencephaly) homology; bZIP, basic region/leucine zipper; Sir2, sirtuin 2; FAD, flavin adenine dinucleotide.</p></div></div><p><em>PREP1</em> encodes an RNA-binding protein because it harbors three RNA recognition motifs.<sup><a reveal-id="DSM035C8" data-open="DSM035C8" class="link link-ref link-reveal xref-bibr">8</a></sup><em>PREP2</em> encodes a protein that harbors a V-set Ig-like domain that is responsible for binding to sialic acid.<sup><a reveal-id="DSM035C9" data-open="DSM035C9" class="link link-ref link-reveal xref-bibr">9</a></sup> A specialized subgroup of the Ig superfamily, called sialic acid-binding immunoglobulin-like lectins (Siglecs), can recognize sialylated glycoconjugates and play a role in cell–cell recognition and intracellular signaling. At least 11 related genes have been identified in the human genome, all of which encode type 1 membrane proteins that carry an N-terminal sialic acid-binding V-set Ig domain and varying numbers of C2-set Ig domains.<sup><a reveal-id="DSM035C9" data-open="DSM035C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Thus, Prep2 may be a Siglec-type protein. Prep3 protein belongs to the large calcineurin-like phosphoesterase superfamily. Members of this superfamily include the polymerase-associated B-subunits and all carry an active site harboring two divalent metal ions for catalysis.<sup><a reveal-id="DSM035C10" data-open="DSM035C10" class="link link-ref link-reveal xref-bibr">10</a></sup> This suggests that Prep3 protein may participate in cell growth by regulating DNA replication. <em>PREP4</em> encodes a novel member of the paxillin family proteins that bear the RhoGEF and PH domains.<sup><a reveal-id="DSM035C11" data-open="DSM035C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Paxillin is a multi-domain adaptor that recruits associated proteins to focal adhesions, where it plays a pivotal role in transducing cell signaling, thereby eliciting changes in cell migration and gene expression.<sup><a reveal-id="DSM035C12" data-open="DSM035C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Given that the RhoGEF and PH domains both are also found in these cell signaling molecules, Prep4 protein appears to function in the signal transduction of growth signals. <em>PREP5</em> encodes a protein with a LisH (LIS1 homology) motif, which is found in many signaling proteins of the WD-40 family and is believed to help regulate microtubule dynamics by mediating dimerization or binding to cytoplasmic dynein heavy chain.<sup><a reveal-id="DSM035C13" data-open="DSM035C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Prep6 protein is homologous to laminin A, which is one of the 15 laminin isoforms that are the major component of basement membranes.<sup><a reveal-id="DSM035C14" data-open="DSM035C14" class="link link-ref link-reveal xref-bibr">14</a></sup><em>PREP7</em> encodes a basic-leucine zipper (bZip) transcription factor<sup><a reveal-id="DSM035C15" data-open="DSM035C15" class="link link-ref link-reveal xref-bibr">15</a></sup> with a DNA-binding motif called AT-hook.<sup><a reveal-id="DSM035C16" data-open="DSM035C16" class="link link-ref link-reveal xref-bibr">16</a></sup><em>PREP8</em> encodes a protein that is weakly homologous to Sir2, an NAD-dependent deacetylase that links metabolism with longevity in yeast, flies, and worms.<sup><a reveal-id="DSM035C17" data-open="DSM035C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Prep9 protein harbors a region that is weakly homologous to extensin, a hydroxyproline-rich cell wall structural glycoprotein of higher plants.<sup><a reveal-id="DSM035C18" data-open="DSM035C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The N-terminal region of Prep10 protein bears a domain that has homology to the adenovirus hexon-associated protein IX.<sup><a reveal-id="DSM035C19" data-open="DSM035C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Prep11 protein carries in its C-terminus a domain with homology to the flavin adenine dinucleotide (FAD)-binding domain of siderophore-interacting protein which may be involved in iron chelation and iron utilization.<sup><a reveal-id="DSM035C20" data-open="DSM035C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                            <h3 scrollto-destination=5472263 id="5472263" class="section-title" data-legacy-id=s3c>3.3. Expression pattern of PREP1–PREP11 and proto-oncogene PREP genes in PBMCs</h3>
<p>To determine whether the uncharacterized <em>PREP1</em>–<em>PREP11</em> genes are expressed in particular human blood cells, we performed RT–PCR on multiple tissue cDNA (MTC) panels from Clontech (Palo Alto, CA). As shown in Fig. <a reveal-id="DSM035F3" data-open="DSM035F3" class="link link-reveal link-table xref-fig">3</a>, <em>PREP3</em>, <em>PREP4</em>, and <em>PREP6</em> mRNAs were only detected in resting CD14+ cells (monocytes; M), whereas <em>PREP1</em> mRNAs were predominantly found in monocytes but were also faintly expressed in resting CD19+ cells (B cells). In addition, <em>PREP11</em> showed a strong signal in monocytes and B cells and a weak signal in T-suppressor/cytotoxic cells and activated CD4+ T cells. <em>PREP2</em> was strongly detected in monocytes, T-helper/inducer, and B cells, and weakly in T-suppressor/cytotoxic and activated CD4+ cells. <em>PREP7</em> was strongly expressed in all mononuclear cells (B cells, T cells, and monocytes), weakly expressed in activated CD8+ T cells, and faintly expressed in activated CD4 + , CD19 + , and mononuclear cells. <em>PREP10</em> was detected strongly in all resting mononuclear cells, but only faintly in the other cell types examined. The other <em>PREP</em> genes were ubiquitously expressed at almost equal levels in all cell types examined and indeed could serve as a loading control like GAPDH.</p>                        <a id="5472265" scrollto-destination="5472265"></a>
<div id="DSM035F3" data-id="DSM035F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03503.jpeg?Expires=2147483647&Signature=sS-zoA6KD-ZEn9ntH1D-gCTK~uHkPxfERgOGN9M74Xij-0NsJrQujuyiLz08Hlr-KdspfQ-Z3y3UvIyK4Hb0U2NriHzsoHQIGPBL07f~EUvbWw8T80TMXjJHUm14P85~jMwl7qIdf5RwIPOVqRc04oSzKX0ac9XIUgIKkLu5HTpGrZhfgmoNX13s-ny3AcsuSbx4Xy-oifqfb4L~zHAdbg3q8lCb4q2DGkP0nFxVNqnOr7t5lDlkIiUMk2XfXgLSmIjwnyh4rBSYlYCOK-IP~3BfFTLlhZyCpoXaYRwHlEdWusvP5ZG7RvDA8ElFs-R7s0Po2tKObMbjMTa9IZlJIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Determination by RT–PCR of the human blood cells that express PREP1–PREP11. RT–PCR was performed using a multiple tissue cDNA panel for human blood fractions (MTC, Clontech). GAPDH was also amplified as a loading control. PCR amplifications involved 40 cycles at 55°C except as indicated to the right of the panels: 35 cycles at 55°C (#) or 35 cycles at 58°C ($). Lane 1, mononuclear cells (B cells, T cells, and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells." path-from-xml="dsm03503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472265/dsm03503.jpeg" path-from-xml="dsm03503.jpeg" target="_blank">View large</a><a section="5472265" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03503.jpeg?Expires=2147483647&Signature=1cpJUvLfVQfF9gmzFQrlIYmNRzr8aVPEUeMGDYnXle1BnDv8YfsfLi6iA3uNzyN5ZndlT7Led8ckQWvmM8oGYwIqXDgbl7ozMoBR-aCFy1EcnYqc52N6JO7sRVo37~hoNq~RKIGQXDB7fQVjRIGMYM5EL9DyBWBvLfHmTMie29cO1khq0s~2n7YBfq0Sr1JJ5Or~xtN4XK8lXV9ZCFFOB6KSy1701ieELzBJTT-xQzOQ5r6mMzGWu1rmuP5vErpmviz~tydfLaatEzPvMHuMCAZs-gkAruq5RPbSqlIM5QPgO623YCxOllDHG1Xz-lLNyE1RrYeZQ8O3hClNYPxyUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472265&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Determination by RT–PCR of the human blood cells that express <em>PREP1</em>–<em>PREP11.</em> RT–PCR was performed using a multiple tissue cDNA panel for human blood fractions (MTC, Clontech). <em>GAPDH</em> was also amplified as a loading control. PCR amplifications involved 40 cycles at 55°C except as indicated to the right of the panels: 35 cycles at 55°C (#) or 35 cycles at 58°C ($). Lane 1, mononuclear cells (B cells, T cells, and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells.</p></div></div><div content-id="DSM035F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03503.jpeg?Expires=2147483647&Signature=sS-zoA6KD-ZEn9ntH1D-gCTK~uHkPxfERgOGN9M74Xij-0NsJrQujuyiLz08Hlr-KdspfQ-Z3y3UvIyK4Hb0U2NriHzsoHQIGPBL07f~EUvbWw8T80TMXjJHUm14P85~jMwl7qIdf5RwIPOVqRc04oSzKX0ac9XIUgIKkLu5HTpGrZhfgmoNX13s-ny3AcsuSbx4Xy-oifqfb4L~zHAdbg3q8lCb4q2DGkP0nFxVNqnOr7t5lDlkIiUMk2XfXgLSmIjwnyh4rBSYlYCOK-IP~3BfFTLlhZyCpoXaYRwHlEdWusvP5ZG7RvDA8ElFs-R7s0Po2tKObMbjMTa9IZlJIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Determination by RT–PCR of the human blood cells that express PREP1–PREP11. RT–PCR was performed using a multiple tissue cDNA panel for human blood fractions (MTC, Clontech). GAPDH was also amplified as a loading control. PCR amplifications involved 40 cycles at 55°C except as indicated to the right of the panels: 35 cycles at 55°C (#) or 35 cycles at 58°C ($). Lane 1, mononuclear cells (B cells, T cells, and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells." path-from-xml="dsm03503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472265/dsm03503.jpeg" path-from-xml="dsm03503.jpeg" target="_blank">View large</a><a section="5472265" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03503.jpeg?Expires=2147483647&Signature=1cpJUvLfVQfF9gmzFQrlIYmNRzr8aVPEUeMGDYnXle1BnDv8YfsfLi6iA3uNzyN5ZndlT7Led8ckQWvmM8oGYwIqXDgbl7ozMoBR-aCFy1EcnYqc52N6JO7sRVo37~hoNq~RKIGQXDB7fQVjRIGMYM5EL9DyBWBvLfHmTMie29cO1khq0s~2n7YBfq0Sr1JJ5Or~xtN4XK8lXV9ZCFFOB6KSy1701ieELzBJTT-xQzOQ5r6mMzGWu1rmuP5vErpmviz~tydfLaatEzPvMHuMCAZs-gkAruq5RPbSqlIM5QPgO623YCxOllDHG1Xz-lLNyE1RrYeZQ8O3hClNYPxyUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472265&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Determination by RT–PCR of the human blood cells that express <em>PREP1</em>–<em>PREP11.</em> RT–PCR was performed using a multiple tissue cDNA panel for human blood fractions (MTC, Clontech). <em>GAPDH</em> was also amplified as a loading control. PCR amplifications involved 40 cycles at 55°C except as indicated to the right of the panels: 35 cycles at 55°C (#) or 35 cycles at 58°C ($). Lane 1, mononuclear cells (B cells, T cells, and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells.</p></div></div><p>Unexpectedly, we found six proto-oncogenes were <em>PREP</em> genes, namely, c-<em>ets</em>, c-<em>src</em>, c-<em>yes</em>, c-<em>fgr</em>, c-<em>fos</em>, and c-<em>vav</em>. We initially asked whether the augmented expression of these proto-oncogenes is due to infection-induced T-cell or macrophage stimulation in one of the PBMC donors. To test this, we examined the expression of these proto-oncogenes by RT–PCR on the MTC panel described above, since the sample RNAs used to generate the panel is not the same as our RNAs (Fig. <a reveal-id="DSM035F4" data-open="DSM035F4" class="link link-reveal link-table xref-fig">4</a>). To our surprise, we found all cell types examined strongly expressed c-<em>ets</em>, c-<em>src</em>, and c-<em>yes</em>. All mononuclear cells (B cells, T cells, and monocytes) and activated CD4+ and CD19+ cells, but not activated CD8+ and mononuclear cells, also expressed c-<em>vav</em>. In contrast, c-<em>fgr</em> was only detected in T-suppressor/cytotoxic cells, while c-<em>fos</em> was only detected in resting T-suppressor/cytotoxic cells, B cells, and mononuclear cells. Thus, the expression of these proto-oncogenes is a commonly observed phenomenon.</p>                        <a id="5472267" scrollto-destination="5472267"></a>
<div id="DSM035F4" data-id="DSM035F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03504.jpeg?Expires=2147483647&Signature=Yu0jkbB-RB0f57O3jO530rX6KkK2-XhdTXlh4pCeqq9wXLYgYdrj-NIPA0Mb~07VBbIja0t8~pA1Rrxe6mqVivZ1DmVModZ3RyRISQZA5my1wIDdt9p9PSCRg3uPfjAjdM94hb1q7fnoIDQ9hPnvIj5N0Vbrcn6YbsgR-4iv6B8Y7-QVu5Qcvx3qEoe910KzJr3ppwnz3MRaLZaPnVaXjNZvZu4PLunIueawPsRFunphqwonbopZxDyMWgjrXdtMWAnXHZMn1aqqRuyJHt1O8K8zcI-hC5CprI4Dlf~AKE-o2hh5h~gFzZsN8VyxdQyv8WohHp3zVxQQVp0N8yX0FQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of the indicated proto-oncogenes in human blood fractions. RT–PCR was performed using the multiple tissue cDNA panel (MTC, Clontech). GAPDH was also amplified as a loading control. PCR amplifications involved 35 cycles at 55°C for all samples. Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells." path-from-xml="dsm03504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472267/dsm03504.jpeg" path-from-xml="dsm03504.jpeg" target="_blank">View large</a><a section="5472267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03504.jpeg?Expires=2147483647&Signature=FHGBWEM63BEbxwYYTiSTuKgxxre32G9WXVn4-BkDYQ~zzLLCBNzSNhwJxkcp1hEFrCYTSVuIBaCEB2ablmITYYuZJjAErf9DzPx-vC5J~MUHmIgeRELKStUl3xp-B~duv05nB9Ok-eHPzUt2Mbf9LyootUCRaizfN4U6SMl1nlkPXZcdEUuP4kyIbeiW13BITsDieq8QaKpob1bbdT9nMiwP0RWJ1hwCIsdRIftW3DdgWP6lL3rvJvJUlSNsrOoVv9-e-G~cZwrQgrpp2TM-X~0LH1nUi3USbJLMjjFBzgvhV-994iXmPDuyQJ92GYry6kzJo3lk5xHkxlFBlwG0jA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472267&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of the indicated proto-oncogenes in human blood fractions. RT–PCR was performed using the multiple tissue cDNA panel (MTC, Clontech)<em>. GAPDH</em> was also amplified as a loading control. PCR amplifications involved 35 cycles at 55°C for all samples. Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells.</p></div></div><div content-id="DSM035F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03504.jpeg?Expires=2147483647&Signature=Yu0jkbB-RB0f57O3jO530rX6KkK2-XhdTXlh4pCeqq9wXLYgYdrj-NIPA0Mb~07VBbIja0t8~pA1Rrxe6mqVivZ1DmVModZ3RyRISQZA5my1wIDdt9p9PSCRg3uPfjAjdM94hb1q7fnoIDQ9hPnvIj5N0Vbrcn6YbsgR-4iv6B8Y7-QVu5Qcvx3qEoe910KzJr3ppwnz3MRaLZaPnVaXjNZvZu4PLunIueawPsRFunphqwonbopZxDyMWgjrXdtMWAnXHZMn1aqqRuyJHt1O8K8zcI-hC5CprI4Dlf~AKE-o2hh5h~gFzZsN8VyxdQyv8WohHp3zVxQQVp0N8yX0FQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of the indicated proto-oncogenes in human blood fractions. RT–PCR was performed using the multiple tissue cDNA panel (MTC, Clontech). GAPDH was also amplified as a loading control. PCR amplifications involved 35 cycles at 55°C for all samples. Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells." path-from-xml="dsm03504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472267/dsm03504.jpeg" path-from-xml="dsm03504.jpeg" target="_blank">View large</a><a section="5472267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03504.jpeg?Expires=2147483647&Signature=FHGBWEM63BEbxwYYTiSTuKgxxre32G9WXVn4-BkDYQ~zzLLCBNzSNhwJxkcp1hEFrCYTSVuIBaCEB2ablmITYYuZJjAErf9DzPx-vC5J~MUHmIgeRELKStUl3xp-B~duv05nB9Ok-eHPzUt2Mbf9LyootUCRaizfN4U6SMl1nlkPXZcdEUuP4kyIbeiW13BITsDieq8QaKpob1bbdT9nMiwP0RWJ1hwCIsdRIftW3DdgWP6lL3rvJvJUlSNsrOoVv9-e-G~cZwrQgrpp2TM-X~0LH1nUi3USbJLMjjFBzgvhV-994iXmPDuyQJ92GYry6kzJo3lk5xHkxlFBlwG0jA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472267&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of the indicated proto-oncogenes in human blood fractions. RT–PCR was performed using the multiple tissue cDNA panel (MTC, Clontech)<em>. GAPDH</em> was also amplified as a loading control. PCR amplifications involved 35 cycles at 55°C for all samples. Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells.</p></div></div>                            <h3 scrollto-destination=5472268 id="5472268" class="section-title" data-legacy-id=s3d>3.4. Focused PREP analysis of CSS PBMCs</h3>
<p>We next examined whether the <em>PREP</em> genes can serve in a focused array that can be used to diagnostically. For this purpose, we used the array to determine if CSS patients can be characterized by a particular PREP mRNA expression pattern. CSS is a rare autoimmune disease (ca. 2.4 per million), namely, a small-vessel necrotizing vasculitis that is typically characterized by asthma, lung infiltrates, extravascular necrotizing granulomas, and hypereosinophilia.<sup><a reveal-id="DSM035C21" data-open="DSM035C21" class="link link-ref link-reveal xref-bibr">21</a></sup> This disease was chosen because diagnostic gene markers for this disease have not yet been reported. Moreover, it is likely that the symptoms of CSS patients will be reflected in abnormal gene expressions in their PBMCs. Indeed, we found with the aid of expression profiling analysis (GeneSpring) that, compared with normal volunteers, 33 <em>PREP</em> genes are consistently up-regulated in the PBMCs of all seven cases of CSS patients tested (Fig. <a reveal-id="DSM035F5" data-open="DSM035F5" class="link link-reveal link-table xref-fig">5</a>A, uppermost panel). Of these, <em>PREP135</em>, <em>PREP77</em>, <em>PREP191</em>, <em>PREP234</em>, and <em>PREP136</em>, which are highlighted by larger arrows in Fig. <a reveal-id="DSM035F5" data-open="DSM035F5" class="link link-reveal link-table xref-fig">5</a>A, showed the most conspicuous up-regulation in all CSS patients (&gt;2.5-fold change).</p>                        <a id="5472270" scrollto-destination="5472270"></a>
<div id="DSM035F5" data-id="DSM035F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03505.jpeg?Expires=2147483647&Signature=fPjdIRrc3J35dl4LhEZjIkasIqphFnThg7B-~mNDf3PAgHiY4fhFHGw4LjRatsC7HkeQAO2hXthD5P3B~YmAy1-xlz2uYXpHBJ5TPagyLTh5MOf8WDhfB6Y1i2jStTF7fpFsGMD63BSryZO1uOz45rH9JQrHZVyHq1WydwoYoJWOdAqlA2HdmE8-2ocENvtRQYA4LF8XpGdimA8S2GKh4ij3wTZ5gMJzD963q6Mnm7-4frvPTMoK4dn7D-VxWLC8zltShR4zD9AzufChlliPWnFpoWcIF4k~c80Ox~DJgzlAxfU1siVzv7VyHc6X7ddhV3BuVjuUVJ3LYDPjOgqMAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of PREP genes in CSS patient PBMCs. Agilent's whole human genome DNA microarray was employed for this analysis. (A) Presentation of mosaic tiles for the 33 and 3 PREP genes whose expressions are up-regulated (&gt;1.2-fold change) or down-regulated (&lt;1.0-fold change) in all seven CSS patients as compared with healthy volunteers. The gene names are indicated on the right of the panels. The PREP genes were selected by a filter and their expressions in all seven cases of CSS patients were grouped together by using GeneSpring software 7.3 (Agilent Technologies, Inc.). In other words, their raw expression values were normalized according to the manufacturer's instructions by conducting a ratio calculation of the two channels. This was followed by a per-gene normalization to ensure all ratios for each gene were normalized to 1. The nine conspicuously up-regulated PREP genes (&gt;2.0-fold change) are denoted by arrows; the five large arrows signify more enhanced up-regulation (&gt;2.5-fold change). Tile colors indicate the mean relative transcript level; green corresponds to a log10 ratio of 0.01 (down-regulation), red corresponds to a log10 ratio of 100 (up-regulation), and yellow indicates unchanged (bottom-most panel) when CSS and normal PBMCs are compared (bottom-most panel). (B) Graphical presentation of the nine CSS-up-regulated and three CSS-down-regulated PREP genes (denoted by arrows in Fig. 5A). Mean values were plotted for the genes that showed a significant (P &lt; 0.05) difference in expression. The ordinate scale indicates relative log ratio values." path-from-xml="dsm03505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472270/dsm03505.jpeg" path-from-xml="dsm03505.jpeg" target="_blank">View large</a><a section="5472270" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03505.jpeg?Expires=2147483647&Signature=w90mfNiZlx2nwKV~36Q2PK8gzKGEDMVfzRhTfOEj01xLmQBGcwix~yq5EoZRuEIFJpf1H48LAaqy56nn2aACAp9UeAD1X99qI1D0rHf~-WtH75LE8jNxfa7Me0NFrv7YYFhWANG6S2r~2oFiDJPXJ7IlUYP~vO3mrDudtSXeW3L64j96sI-izIgzLPnj6eitM1Ny9EqXp53wrmDh2A8fK6EWaDDBaOkXELAUGD4-~dkpBL8Evn8daslJaGQH1D9vulMg87qAcBaO9Yw~oXHiBOpTcwQen5MgqhLUwYvwu~dD1aILoNLqrXmrtK9RRz6LT11kHv-WQ76mFKnVRQaYag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472270&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>PREP</em> genes in CSS patient PBMCs. Agilent's whole human genome DNA microarray was employed for this analysis. (<strong>A</strong>) Presentation of mosaic tiles for the 33 and 3 <em>PREP</em> genes whose expressions are up-regulated (&gt;1.2-fold change) or down-regulated (&lt;1.0-fold change) in all seven CSS patients as compared with healthy volunteers. The gene names are indicated on the right of the panels. The <em>PREP</em> genes were selected by a filter and their expressions in all seven cases of CSS patients were grouped together by using GeneSpring software 7.3 (Agilent Technologies, Inc.). In other words, their raw expression values were normalized according to the manufacturer's instructions by conducting a ratio calculation of the two channels. This was followed by a per-gene normalization to ensure all ratios for each gene were normalized to 1. The nine conspicuously up-regulated <em>PREP</em> genes (&gt;2.0-fold change) are denoted by arrows; the five large arrows signify more enhanced up-regulation (&gt;2.5-fold change). Tile colors indicate the mean relative transcript level; green corresponds to a log10 ratio of 0.01 (down-regulation), red corresponds to a log10 ratio of 100 (up-regulation), and yellow indicates unchanged (bottom-most panel) when CSS and normal PBMCs are compared (bottom-most panel). (<strong>B</strong>) Graphical presentation of the nine CSS-up-regulated and three CSS-down-regulated <em>PREP</em> genes (denoted by arrows in Fig. 5A). Mean values were plotted for the genes that showed a significant (<em>P</em> &lt; 0.05) difference in expression. The ordinate scale indicates relative log ratio values.</p></div></div><div content-id="DSM035F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03505.jpeg?Expires=2147483647&Signature=fPjdIRrc3J35dl4LhEZjIkasIqphFnThg7B-~mNDf3PAgHiY4fhFHGw4LjRatsC7HkeQAO2hXthD5P3B~YmAy1-xlz2uYXpHBJ5TPagyLTh5MOf8WDhfB6Y1i2jStTF7fpFsGMD63BSryZO1uOz45rH9JQrHZVyHq1WydwoYoJWOdAqlA2HdmE8-2ocENvtRQYA4LF8XpGdimA8S2GKh4ij3wTZ5gMJzD963q6Mnm7-4frvPTMoK4dn7D-VxWLC8zltShR4zD9AzufChlliPWnFpoWcIF4k~c80Ox~DJgzlAxfU1siVzv7VyHc6X7ddhV3BuVjuUVJ3LYDPjOgqMAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of PREP genes in CSS patient PBMCs. Agilent's whole human genome DNA microarray was employed for this analysis. (A) Presentation of mosaic tiles for the 33 and 3 PREP genes whose expressions are up-regulated (&gt;1.2-fold change) or down-regulated (&lt;1.0-fold change) in all seven CSS patients as compared with healthy volunteers. The gene names are indicated on the right of the panels. The PREP genes were selected by a filter and their expressions in all seven cases of CSS patients were grouped together by using GeneSpring software 7.3 (Agilent Technologies, Inc.). In other words, their raw expression values were normalized according to the manufacturer's instructions by conducting a ratio calculation of the two channels. This was followed by a per-gene normalization to ensure all ratios for each gene were normalized to 1. The nine conspicuously up-regulated PREP genes (&gt;2.0-fold change) are denoted by arrows; the five large arrows signify more enhanced up-regulation (&gt;2.5-fold change). Tile colors indicate the mean relative transcript level; green corresponds to a log10 ratio of 0.01 (down-regulation), red corresponds to a log10 ratio of 100 (up-regulation), and yellow indicates unchanged (bottom-most panel) when CSS and normal PBMCs are compared (bottom-most panel). (B) Graphical presentation of the nine CSS-up-regulated and three CSS-down-regulated PREP genes (denoted by arrows in Fig. 5A). Mean values were plotted for the genes that showed a significant (P &lt; 0.05) difference in expression. The ordinate scale indicates relative log ratio values." path-from-xml="dsm03505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472270/dsm03505.jpeg" path-from-xml="dsm03505.jpeg" target="_blank">View large</a><a section="5472270" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03505.jpeg?Expires=2147483647&Signature=w90mfNiZlx2nwKV~36Q2PK8gzKGEDMVfzRhTfOEj01xLmQBGcwix~yq5EoZRuEIFJpf1H48LAaqy56nn2aACAp9UeAD1X99qI1D0rHf~-WtH75LE8jNxfa7Me0NFrv7YYFhWANG6S2r~2oFiDJPXJ7IlUYP~vO3mrDudtSXeW3L64j96sI-izIgzLPnj6eitM1Ny9EqXp53wrmDh2A8fK6EWaDDBaOkXELAUGD4-~dkpBL8Evn8daslJaGQH1D9vulMg87qAcBaO9Yw~oXHiBOpTcwQen5MgqhLUwYvwu~dD1aILoNLqrXmrtK9RRz6LT11kHv-WQ76mFKnVRQaYag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472270&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>PREP</em> genes in CSS patient PBMCs. Agilent's whole human genome DNA microarray was employed for this analysis. (<strong>A</strong>) Presentation of mosaic tiles for the 33 and 3 <em>PREP</em> genes whose expressions are up-regulated (&gt;1.2-fold change) or down-regulated (&lt;1.0-fold change) in all seven CSS patients as compared with healthy volunteers. The gene names are indicated on the right of the panels. The <em>PREP</em> genes were selected by a filter and their expressions in all seven cases of CSS patients were grouped together by using GeneSpring software 7.3 (Agilent Technologies, Inc.). In other words, their raw expression values were normalized according to the manufacturer's instructions by conducting a ratio calculation of the two channels. This was followed by a per-gene normalization to ensure all ratios for each gene were normalized to 1. The nine conspicuously up-regulated <em>PREP</em> genes (&gt;2.0-fold change) are denoted by arrows; the five large arrows signify more enhanced up-regulation (&gt;2.5-fold change). Tile colors indicate the mean relative transcript level; green corresponds to a log10 ratio of 0.01 (down-regulation), red corresponds to a log10 ratio of 100 (up-regulation), and yellow indicates unchanged (bottom-most panel) when CSS and normal PBMCs are compared (bottom-most panel). (<strong>B</strong>) Graphical presentation of the nine CSS-up-regulated and three CSS-down-regulated <em>PREP</em> genes (denoted by arrows in Fig. 5A). Mean values were plotted for the genes that showed a significant (<em>P</em> &lt; 0.05) difference in expression. The ordinate scale indicates relative log ratio values.</p></div></div><p><em>PREP135</em> encodes the coactosin-like protein (CLP), which is a small, evolutionarily conserved F-actin-binding protein that can also bind to 5-lipoxygenase (5LO) and regulate its activity.<sup><a reveal-id="DSM035C22" data-open="DSM035C22" class="link link-ref link-reveal xref-bibr">22</a></sup><em>PREP77</em> encodes prosaposin, the precursor of the sphingolipid activator proteins (saposin A, B, C, and D) that are required for the enzymatic hydrolysis of sphingolipids by specific lysosomal hydrolases.<sup><a reveal-id="DSM035C23" data-open="DSM035C23" class="link link-ref link-reveal xref-bibr">23</a></sup><em>PREP191</em>, which encodes a lysosomal aspartyl peptidase cathepsin D (CTSD), is known to be involved in prosaposin proteolysis.<sup><a reveal-id="DSM035C24" data-open="DSM035C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Thus, the enhanced expression of <em>PREP77</em> and <em>PREP191</em> suggests sphingolipid levels are increased in CSS. This is consistent with the pivotal role sphingolipids are known to play as proinflammatory factors.<sup><a reveal-id="DSM035C25" data-open="DSM035C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Thus, sphingolipids and related enzymes may not only be diagnostic marker for CSS, they may also be novel therapeutic targets in CSS (see Discussion).</p><p><em>PREP234</em> encodes Gardner–Rasheed feline sarcoma viral (<em>v</em>-fgr) oncogene homolog (<em>c</em>-fgr: FGR), which is known to play an important role in lipopolysaccharide (LPS)-induced macrophage activation.<sup><a reveal-id="DSM035C26" data-open="DSM035C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Indeed, Fgr-deficient mice fail to develop lung eosinophilia in response to repeated challenge with aerosolized ovalbumin (OVA)<sup><a reveal-id="DSM035C27" data-open="DSM035C27" class="link link-ref link-reveal xref-bibr">27</a></sup>. Src-family tyrosine kinases are required to support the accumulation of polymorphonuclear leukocytes (PMN) along with adherent platelets at the site of vascular injury.<sup><a reveal-id="DSM035C28" data-open="DSM035C28" class="link link-ref link-reveal xref-bibr">28</a></sup><em>PREP136</em> encodes lysozyme, which is one of the genes responsible for autosomal dominant hereditary amyloidosis.<sup><a reveal-id="DSM035C29" data-open="DSM035C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Indeed, there are several case reports of amyloidosis associated with CSS that occurs together with a destructive inflammatory or granulomatous reaction to amyloid.<sup><a reveal-id="DSM035C30" data-open="DSM035C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Several case reports have also described granulomatous angiitis in patients with sporadic, amyloid beta peptide (Abeta)-related cerebral amyloid angiopathy and Abeta-related angiitis with development of amyloid-associated inflammation.<sup><a reveal-id="DSM035C31" data-open="DSM035C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p><p>There are also four other less conspicuously CSS-up-regulated <em>PREP</em> genes (2.0–2.5-fold change). Of these, <em>PREP226</em> encodes a receptor (membrane)-type variant of the protein tyrosine phosphatase (PTPepsilonM) family protein that negatively regulates insulin receptor (IR) signaling by dephosphorylating IR, thereby suppressing the phosphorylation of IR downstream enzymes such as Akt, extracellular regulated kinase (ERK), and glycogen synthase kinase 3 (GSK3).<sup><a reveal-id="DSM035C32" data-open="DSM035C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Notably, the cytosolic isoform of PTPepsilon (PTPepsilonC) selectively inhibits interleukin-6 (IL-6)- and IL-10-induced Janus kinases (JAK)-signal transducer and activator of transcription 3 (STAT3) signaling in blood cells.<sup><a reveal-id="DSM035C33" data-open="DSM035C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p><p>Another of the four less conspicuously up-regulated <em>PREP</em> genes, <em>PREP113</em>, encodes platelet factor 4 (PF4; CXC chemokine ligand 4), which is a specific marker for megakaryocyte cells and that induces the differentiation of monocytes into macrophages during the inflammatory process.<sup><a reveal-id="DSM035C34" data-open="DSM035C34" class="link link-ref link-reveal xref-bibr">34</a></sup><em>PREP286</em> encodes lysosomal-associated multi-spanning membrane protein-5 (LAPTM5). While the function of LAPTM5 is unknown, it may participate in host defense in collaboration with other lysosome-related genes (<em>PREP77</em>, <em>PREP191</em>, and <em>PREP136</em>) in the blood cells of CSS patients. These results suggest that the regulation of cell signaling in blood cells from CSS patients is abnormal. The last of the four less conspicuously up-regulated <em>PREP</em> genes is <em>PREP203</em>, which encodes the spliceosomal protein thioredoxin-like 4B (TXNL4B). However, its physiological significance in the pathogenesis of CSS remains unclear.</p><p>We also identified three <em>PREP</em> genes that, compared with normal volunteers, are down-regulated in the PBMCs of all seven CSS patients (Fig. <a reveal-id="DSM035F5" data-open="DSM035F5" class="link link-reveal link-table xref-fig">5</a>A, middle panel). <em>PREP76</em> encodes nicastrin;<sup><a reveal-id="DSM035C35" data-open="DSM035C35" class="link link-ref link-reveal xref-bibr">35</a></sup> its down-regulation may disturb the phagocytic response of macrophages to inflammatory cytokines since gamma-secretase catalyzes the intramembrane cleavage of its protein substrates and targets phagocytosis-related proteins of macrophages.<sup><a reveal-id="DSM035C36" data-open="DSM035C36" class="link link-ref link-reveal xref-bibr">36</a></sup><em>PREP114</em> encodes selectin P ligand (SELPLG), which stimulates T lymphocytes, and plays a critical role in the tethering of these cells to activated platelets or endothelia expressing P-selectin.<sup><a reveal-id="DSM035C37" data-open="DSM035C37" class="link link-ref link-reveal xref-bibr">37</a></sup><em>PREP283</em> encodes sperm-associated antigen 9 (SPAG9), which has also been identified as a serum target of the autoantibody produced in systemic sclerosis (SSc).<sup><a reveal-id="DSM035C38" data-open="DSM035C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Thus, the reduced expression of these three genes could also serve as diagnostic markers of CSS. Unfortunately, we could not correlate the expression levels of these genes to the profiles of tested CSS patients (Supplementary Table S1).</p>                            <h3 scrollto-destination=5472278 id="5472278" class="section-title" data-legacy-id=s3e>3.5. Expression pattern of CSS-up-regulated PREP genes in PBMCs</h3>
<p>To examine whether these 12 CSS-up- or -down-regulated <em>PREP</em> genes are expressed in particular human blood cells, we performed RT–PCR on MTC panels (Fig. <a reveal-id="DSM035F6" data-open="DSM035F6" class="link link-reveal link-table xref-fig">6</a>). <em>PREP76</em>, <em>PREP135</em>, and <em>PREP203</em> were expressed by nearly all cells, and were the only genes that were expressed by activated CD19+ and activated CD8+ cells. Most of the 12 <em>PREP</em> genes were expressed by resting CD14+ cells (monocyte; M) and T-helper/inducer (T-h/i) cells, albeit with different expression levels. All genes except <em>PREP283</em> were expressed by T-suppressor/cytotoxic (T-s/c) cells. <em>PREP76</em>, <em>PREP113</em>, <em>PREP114</em>, <em>PREP135</em>, and <em>PREP203</em> were expressed at varying expression levels by activated mononuclear cells (aMNC). All genes except <em>PREP114</em>, <em>PREP135</em>, and <em>PREP283</em> were expressed by B cells, albeit comparatively weakly. Thus, many of the CSS-up-regulated or -down-regulated <em>PREP</em> genes are expressed by T cells or monocytes. Unfortunately, we could not perform the same analysis on CSS patients because of the low numbers of PBMCs available.</p>                        <a id="5472283" scrollto-destination="5472283"></a>
<div id="DSM035F6" data-id="DSM035F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03506.jpeg?Expires=2147483647&Signature=XVfqmfaSffn4NQ7ygyOhJiwSOw9otkqoCz~abEstaK3Irdr7M7tCwk5qWnfxycuyE7uNdIgMCwyYDYKfmn3SgT7-kpcd~a6hvJw58wKV3bxAGMcb7MPoe0~EMXWku8eOugb7CNo0ByBcA2~vuyeNPoXbQv1Q~OJedPc93kmwaCwhZIX9vnDBCdEj37Q4nnIRKyNDcnctKZLomFkIxylk~MA8Q9e~222Eyvwp2XpuRQATGScTxbNBj4I6BfgY8GWhP9RD6br1RY6Q-5Kli5mioPIk0KTMgYLVK4WuU2MJkB9UBdVP2ObQG2fimWFgs4VCOR-X3pjDCR7AFyqkuRF-KA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of CSS-up-regulated or -down-regulated PREP genes by various human blood cells. RT–PCR was performed with the MTC. The GAPDH expression profile served as a loading control. All PCR amplifications involved 35 cycles at 55°C. We could not detect an amplified band for PREP119 even though we tested three different primer sets (Supplementary Table S1). Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells." path-from-xml="dsm03506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472283/dsm03506.jpeg" path-from-xml="dsm03506.jpeg" target="_blank">View large</a><a section="5472283" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03506.jpeg?Expires=2147483647&Signature=w5hqfA5sLureCanCrIuF~LD1nj-f-H4FZB3xH2dw7gT6huWJA6DCTpikZJTunrNOk53pLBVuGHN~4WqrOM5ClGGq33pfOOx2C0whu4YUVW6Iks6NwxLlI2bG1mb6mckdltg88C6FpFdg4~FrF~Gthjhl0b18ssOivd5UjY11fgVUj4bqivqkcc1SMubq2XvxCsjzmuF~e48CRdCPlTJLNJ64PJPAF240yM9MKZmjfX0f6z6pjnG-NTilrbXw13TXqnPn703g3ogCttcReZ6olVeC0WoJLag~kQcyAeJfEMUX8KBwcLniWyYl0uMOkU37ZH6062oRS0BmrlC4dIT3RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472283&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of CSS-up-regulated or -down-regulated <em>PREP</em> genes by various human blood cells. RT–PCR was performed with the MTC. The <em>GAPDH</em> expression profile served as a loading control. All PCR amplifications involved 35 cycles at 55°C. We could not detect an amplified band for <em>PREP119</em> even though we tested three different primer sets (Supplementary Table S1). Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells.</p></div></div><div content-id="DSM035F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/m_dsm03506.jpeg?Expires=2147483647&Signature=XVfqmfaSffn4NQ7ygyOhJiwSOw9otkqoCz~abEstaK3Irdr7M7tCwk5qWnfxycuyE7uNdIgMCwyYDYKfmn3SgT7-kpcd~a6hvJw58wKV3bxAGMcb7MPoe0~EMXWku8eOugb7CNo0ByBcA2~vuyeNPoXbQv1Q~OJedPc93kmwaCwhZIX9vnDBCdEj37Q4nnIRKyNDcnctKZLomFkIxylk~MA8Q9e~222Eyvwp2XpuRQATGScTxbNBj4I6BfgY8GWhP9RD6br1RY6Q-5Kli5mioPIk0KTMgYLVK4WuU2MJkB9UBdVP2ObQG2fimWFgs4VCOR-X3pjDCR7AFyqkuRF-KA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of CSS-up-regulated or -down-regulated PREP genes by various human blood cells. RT–PCR was performed with the MTC. The GAPDH expression profile served as a loading control. All PCR amplifications involved 35 cycles at 55°C. We could not detect an amplified band for PREP119 even though we tested three different primer sets (Supplementary Table S1). Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells." path-from-xml="dsm03506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5472283/dsm03506.jpeg" path-from-xml="dsm03506.jpeg" target="_blank">View large</a><a section="5472283" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm03506.jpeg?Expires=2147483647&Signature=w5hqfA5sLureCanCrIuF~LD1nj-f-H4FZB3xH2dw7gT6huWJA6DCTpikZJTunrNOk53pLBVuGHN~4WqrOM5ClGGq33pfOOx2C0whu4YUVW6Iks6NwxLlI2bG1mb6mckdltg88C6FpFdg4~FrF~Gthjhl0b18ssOivd5UjY11fgVUj4bqivqkcc1SMubq2XvxCsjzmuF~e48CRdCPlTJLNJ64PJPAF240yM9MKZmjfX0f6z6pjnG-NTilrbXw13TXqnPn703g3ogCttcReZ6olVeC0WoJLag~kQcyAeJfEMUX8KBwcLniWyYl0uMOkU37ZH6062oRS0BmrlC4dIT3RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5472283&ar=375923&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of CSS-up-regulated or -down-regulated <em>PREP</em> genes by various human blood cells. RT–PCR was performed with the MTC. The <em>GAPDH</em> expression profile served as a loading control. All PCR amplifications involved 35 cycles at 55°C. We could not detect an amplified band for <em>PREP119</em> even though we tested three different primer sets (Supplementary Table S1). Lane 1, mononuclear cells (B, T cells and monocytes). Lane 2, resting CD8+ cells (T-suppressor/cytotoxic cells). Lane 3, resting CD4+ cells (T-helper/inducer). Lane 4, resting CD14+ cells (monocytes). Lane 5, resting CD19+ cells (B cells). Lane 6, activated CD19+ cells. Lane 7, activated mononuclear cells (aMNC). Lane 8, activated CD4+ cells. Lane 9, activated CD8+ cells.</p></div></div>                            <h2 scrollto-destination=5472285 id="5472285" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5472286 id="5472286" class="section-title" data-legacy-id=s4a>4.1. Isolation of genes that are predominantly expressed in human PBMCs</h3>
<p>Here we show for the first time (to our knowledge) that RNA diagnostics can be performed by using a PBMC-specific focused array. Such RNA diagnostics methods have the advantage over DNA diagnostics in that ethical problems concerning the genetic information of individuals are avoided. To select the PBMC-specific genes for use in RNA diagnostics, we utilized a technique that we developed previously, namely, stepwise subtraction.<sup><a reveal-id="DSM035C4" data-open="DSM035C4" class="link link-ref link-reveal xref-bibr">4</a></sup> In this technique, all genes expressed by a particular cell, tissue, or whole organism are used to generate a cDNA library. This library is then systematically subtracted in a stepwise manner by using the biotinylated mRNAs from a control population to isolate essentially all those genes that are up-regulated relative to the control population expression profile. This technique has proven to be useful for isolating the protein-coding genes that are specifically expressed during cancer metastasis, meiosis, and spermatogenesis, and in patients with autoimmune diseases.<sup><a reveal-id="DSM035C4" data-open="DSM035C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSM035C39  DSM035C41" data-open="DSM035C39  DSM035C41" class="link link-ref link-reveal xref-bibr">39–41</a></sup> It has also been used to identify putative non-coding RNAs that cannot be identified by microarray screening.<sup><a reveal-id="DSM035C42" data-open="DSM035C42" class="link link-ref link-reveal xref-bibr">42</a></sup> By using this technique along with high-density oligonucleotide microarray analysis, we identified 290 <em>PREP</em> genes that are predominantly expressed in human PBMCs as compared with fibroblast (TIG-1) cells (Fig. <a reveal-id="DSM035F1" data-open="DSM035F1" class="link link-reveal link-table xref-fig">1</a> and Table 1). These genes could be used to generate a PBMC-specific focused array that could be used to characterize the abnormal expression profiles in the PBMCs of patients with a particular disease. The abnormal profiles could then serve as a diagnostic tool for other patients presenting with this disease.</p><p>The 290 <em>PREP</em> genes we identified include 50 uncharacterized genes, 11 of which possess notable amino acid motifs (Fig. <a reveal-id="DSM035F2" data-open="DSM035F2" class="link link-reveal link-table xref-fig">2</a>). Two of these, <em>PREP1</em> and <em>PREP6</em>, were almost exclusively expressed in monocytes (Fig. <a reveal-id="DSM035F6" data-open="DSM035F6" class="link link-reveal link-table xref-fig">6</a>). Of the 240 <em>PREP</em> genes that have been characterized previously, 11, eight, eight, seven, six, five, five, nine, and 36 are HLA genes, CD antigen genes, ribosomal protein genes, interferon (IFN)-related genes, S100 calcium-binding protein-related genes, interleukin (IL)-related genes, platelet-related genes, signal transduction-related genes, and immunity-related genes, respectively (Supplementary Table S4). We did not identify any other immunity-related genes as <em>PREP</em> genes by these techniques, probably because their expression levels are lower than the <em>PREP</em> genes.</p>                            <h3 scrollto-destination=5472289 id="5472289" class="section-title" data-legacy-id=s4b>4.2. PBMC-specific array analysis with CSS patients</h3>
<p>To explore the usefulness of the PBMC-specific focused array analysis in RNA diagnostics, we tested the PBMCs of seven cases of CSS patients.<sup><a reveal-id="DSM035C21" data-open="DSM035C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The precise pathogenetic mechanisms of CSS remain elusive, in part because the rarity of this disease. We found that compared with normal volunteers, 22 <em>PREP</em> genes were up-regulated by &gt;1.5-fold in the PBMCs of all seven CSS patients (Fig. <a reveal-id="DSM035F5" data-open="DSM035F5" class="link link-reveal link-table xref-fig">5</a>). Of these, <em>PREP135</em>, <em>PREP77</em>, <em>PREP191</em>, <em>PREP234</em>, <em>PREP136</em>, <em>PREP203</em>, and <em>PREP113</em> were the most conspicuously up-regulated in CSS patients. How these genes contribute to the pathogenesis of CSS is suggested by the known functions of their products, as follows. <em>PREP77</em> (prosaposin) and <em>PREP135</em> (CLP) may indicate the enhanced immune responses that cause vascular inflammation; <em>PREP203</em> (TXNL4B) and <em>PREP234</em> (<em>c-</em>fgr) may reflect the augmented intracellular signaling that occurs after an infection causes inflammation; and the increased expression of the lysosome-related genes <em>PREP77</em>, <em>PREP136</em>, <em>PREP191</em>, and <em>PREP286</em> may indicate the mounting of an effective defense against invading microorganisms by the lysosomes in blood cells. Thus, our <em>PREP</em> gene-focused analysis of CSS patients has helped to identify genes that may be putative diagnostic and/or therapeutic targets.</p><p>There are few molecular tools that can be used to aid the diagnosis of CSS. The main tool that is commonly employed is the detection of antineutrophil cytoplasmic autoantibodies (ANCA), which can be detected by an immunofluorescent assay (IFA) on ethanol-fixed neutrophils. ANCA are associated with systemic necrotizing vasculitis and glomerulonephritis diseases such as CSS, Wegener's granulomatosis (WG), and microscopic polyarteritis (MPA).<sup><a reveal-id="DSM035C43" data-open="DSM035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Two different ANCA immunostaining patterns are observed, namely, a diffuse cytoplasmic staining pattern (c-ANCA) that recognizes proteinase 3, and a perinuclear/nuclear pattern (p-ANCA) that commonly recognizes myeloperoxidase (MPO), which is a neutrophil granule protein that helps generate oxygen radicals and is associated with the antimicrobial properties of neutrophils. The c-ANCA pattern is most frequently observed in WG patients, whereas the p-ANCA pattern is detected in CSS and MPA patients.<sup><a reveal-id="DSM035C44" data-open="DSM035C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Notably, ANCAs against alternative antigens such as cathepsin G, which is a major protease released by activated neutrophils, are also occasionally observed.</p><p>Since antibodies against cathepsin D (<em>PREP191</em>), PTPRE (<em>PREP226</em>), FGR (<em>PREP234</em>), PF4 (<em>PREP113</em>), CLP (<em>PREP135</em>), and prosaposin (<em>PREP77</em>) are currently commercially available, it may be worthwhile to use these antibodies in IFAs on ethanol-fixed blood cell samples to determine whether these antibodies have a relationship with the ANCA-staining patterns in CSS and whether they can help confirm the diagnosis in ANCA-negative CSS patients. Unfortunately, these experiments will have to be performed in the future because the CSS blood samples we obtained were used up during RNA preparation. Moreover, as CSS occurs only very rarely (only 2.4 patients per 1 million people in Japan), it will take some time to accumulate enough patients for such a study.</p>                            <h2 scrollto-destination=5472293 id="5472293" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported in part by grants-in-aid from Bio-Medical Cluster Project In Saito, Innovation Plaza Osaka of the Japan Science and Technology Agency (JST), Scientific Research on Priority Areas ‘Applied Genomics’, Scientific Research (S), Exploratory Research, and the Science and Technology Incubation Program in Advanced Regions, from the Ministry of Education, Culture, Sports, Science and Technology of Japan to H.N. This study was also supported in part by grants-in-aid from the Ministry of Health and Welfare of Japan to K.S. and H.H.</p>                            <h2 scrollto-destination=5472295 id="5472295" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the patients and healthy volunteers who participated in this study. We also thank Ms. Akie Tanigawa, Ms. Azumi Fujimori, Ms. Tomoko Motoyama, Ms. Kumiko Ikeue, Ms. Maki Masuda, Ms. Yuki Hamada, and Ms. Maki Fukuda for technical assistance, and Dr. Patrick Hughes for critically reading the manuscript.</p>                            <h2 scrollto-destination=5472297 id="5472297" class="backsection-title" data-legacy-id=s6>Supplementary Data</h2>
<p>Supplementary data are available online at www.dnaresearch.oxfordjournals.org.</p>                            <h2 scrollto-destination=5472299 id="5472299" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM035C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kozal</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Extensivepolymorphisms observed in HIV-1 clade B protease gene using high-density oligonucleotide arrays</div>, <div class="source">Nat. Med.</div> , <div class="year">1996</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">753</div>-<div class="lpage">759</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensivepolymorphisms%20observed%20in%20HIV-1%20clade%20B%20protease%20gene%20using%20high-density%20oligonucleotide%20arrays&amp;author=M.%20J.%20Kozal&amp;author=N.%20Shah&amp;author=N.%20Shen&amp;publication_year=1996&amp;journal=Nat.%20Med.&amp;volume=2&amp;pages=753-759" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm0796-753" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm0796-753" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm0796-753"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8673920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trevino</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Falciani</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Barrera-Saldaña</div> <div class="given-names">H. A.</div></div>. </div><div class="article-title">DNA microarrays: a powerful genomic tool for biomedical and clinical research</div>, <div class="source">Mol. Med.</div> , <div class="year">2007</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">527</div>-<div class="lpage">541</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20microarrays%3A%20a%20powerful%20genomic%20tool%20for%20biomedical%20and%20clinical%20research&amp;author=V.%20Trevino&amp;author=F.%20Falciani&amp;author=H.%20A.%20Barrera-Salda%C3%B1a&amp;publication_year=2007&amp;journal=Mol.%20Med.&amp;volume=13&amp;pages=527-541" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2119/2006-00107.Trevino" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2119%2F2006-00107.Trevino" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2119%2F2006-00107.Trevino"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17660860" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kidgell</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Winzeler</div> <div class="given-names">E. A.</div></div>. </div><div class="article-title">Elucidating genetic diversity with oligonucleotide arrays</div>, <div class="source">Chromosome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">225</div>-<div class="lpage">235</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elucidating%20genetic%20diversity%20with%20oligonucleotide%20arrays&amp;author=C.%20Kidgell&amp;author=E.%20A.%20Winzeler&amp;publication_year=2005&amp;journal=Chromosome%20Res.&amp;volume=13&amp;pages=225-235" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-005-1503-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-005-1503-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-005-1503-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15868417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujii</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Masai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nishimune</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nojima</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Use of stepwise subtraction to comprehensively isolate mouse genes whose transcription is up-regulated during spermiogenesis</div>, <div class="source">EMBO Rep.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">367</div>-<div class="lpage">372</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20stepwise%20subtraction%20to%20comprehensively%20isolate%20mouse%20genes%20whose%20transcription%20is%20up-regulated%20during%20spermiogenesis&amp;author=T.%20Fujii&amp;author=K.%20Tamura&amp;author=K.%20Masai&amp;author=H.%20Tanaka&amp;author=Y.%20Nishimune&amp;author=H.%20Nojima&amp;publication_year=2002&amp;journal=EMBO%20Rep.&amp;volume=3&amp;pages=367-372" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/embo-reports/kvf073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fembo-reports%2Fkvf073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fembo-reports%2Fkvf073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11943763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Masi</div> <div class="given-names">A. T.</div></div>,  <div class="name"><div class="surname">Hunder</div> <div class="given-names">G. G.</div></div>,  <div class="name"><div class="surname">Lie</div> <div class="given-names">J. T.</div></div>, et al. </div><div class="article-title">The American College of Rheumatology 1990 criteria for the classification of Churg-Strauss syndrome (allergic granulomatosis and angiitis)</div>, <div class="source">Arthritis Rheum</div> , <div class="year">1990</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1094</div>-<div class="lpage">1100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20American%20College%20of%20Rheumatology%201990%20criteria%20for%20the%20classification%20of%20Churg-Strauss%20syndrome%20(allergic%20granulomatosis%20and%20angiitis)&amp;author=A.%20T.%20Masi&amp;author=G.%20G.%20Hunder&amp;author=J.%20T.%20Lie&amp;publication_year=1990&amp;journal=Arthritis%20Rheum&amp;volume=33&amp;pages=1094-1100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/art.1780330806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fart.1780330806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fart.1780330806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2202307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishii</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Onda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tanigawa</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Isolation and expression profiling of genes upregulated in the peripheral blood cells of systemic lupus erythematosus patients</div>, <div class="source">DNA Res</div> , <div class="year">2005</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">1</div>-<div class="lpage">11</div>)</div></div></div></div><div content-id="DSM035C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobori</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ikeda</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nara</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Large scale isolation of osteoclast-specific genes by an improved method involving the preparation of a subtracted cDNA library</div>, <div class="source">Genes Cells</div> , <div class="year">1998</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">459</div>-<div class="lpage">475</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20scale%20isolation%20of%20osteoclast-specific%20genes%20by%20an%20improved%20method%20involving%20the%20preparation%20of%20a%20subtracted%20cDNA%20library&amp;author=M.%20Kobori&amp;author=Y.%20Ikeda&amp;author=H.%20Nara&amp;publication_year=1998&amp;journal=Genes%20Cells&amp;volume=3&amp;pages=459-475" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2443.1998.00202.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2443.1998.00202.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2443.1998.00202.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9753427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maris</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dominguez</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Allain</div> <div class="given-names">F. H.</div></div>. </div><div class="article-title">The RNA recognition motif, a plastic RNA-binding platform to regulate post-transcriptional gene expression</div>, <div class="source">FEBS J</div> , <div class="year">2005</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">2118</div>-<div class="lpage">2131</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RNA%20recognition%20motif%2C%20a%20plastic%20RNA-binding%20platform%20to%20regulate%20post-transcriptional%20gene%20expression&amp;author=C.%20Maris&amp;author=C.%20Dominguez&amp;author=F.%20H.%20Allain&amp;publication_year=2005&amp;journal=FEBS%20J&amp;volume=272&amp;pages=2118-2131" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1742-4658.2005.04653.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2005.04653.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1742-4658.2005.04653.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15853797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Angata</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hingorani</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Varki</div> <div class="given-names">N. M.</div></div>,  <div class="name"><div class="surname">Varki</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Cloning and characterization of a novel mouse Siglec, mSiglec-F: differential evolution of the mouse and human (CD33) Siglec-3-related gene clusters</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2001</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">45128</div>-<div class="lpage">45136</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20a%20novel%20mouse%20Siglec%2C%20mSiglec-F%3A%20differential%20evolution%20of%20the%20mouse%20and%20human%20(CD33)%20Siglec-3-related%20gene%20clusters&amp;author=T.%20Angata&amp;author=R.%20Hingorani&amp;author=N.%20M.%20Varki&amp;author=A.%20Varki&amp;publication_year=2001&amp;journal=J.%20Biol.%20Chem.&amp;volume=276&amp;pages=45128-45136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M108573200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M108573200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M108573200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11579105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>. </div><div class="article-title">Phosphoesterase domains associated with DNA polymerases of diverse origins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">3746</div>-<div class="lpage">3752</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phosphoesterase%20domains%20associated%20with%20DNA%20polymerases%20of%20diverse%20origins&amp;author=L.%20Aravind&amp;author=E.%20V.%20Koonin&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=3746-3752" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.16.3746" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.16.3746" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.16.3746"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9685491" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rathore</div> <div class="given-names">V. B.</div></div>,  <div class="name"><div class="surname">Okada</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Newman</div> <div class="given-names">P. J.</div></div>,  <div class="name"><div class="surname">Newman</div> <div class="given-names">D. K.</div></div>. </div><div class="article-title">Paxillin family members function as Csk binding proteins that regulate Lyn activity in human and murine platelets</div>, <div class="source">Biochem. J.</div> , <div class="year">2007</div>, vol. <div class="volume">403</div> (pg. <div class="fpage">275</div>-<div class="lpage">281</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paxillin%20family%20members%20function%20as%20Csk%20binding%20proteins%20that%20regulate%20Lyn%20activity%20in%20human%20and%20murine%20platelets&amp;author=V.%20B.%20Rathore&amp;author=M.%20Okada&amp;author=P.%20J.%20Newman&amp;author=D.%20K.%20Newman&amp;publication_year=2007&amp;journal=Biochem.%20J.&amp;volume=403&amp;pages=275-281" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BJ20061618" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBJ20061618" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBJ20061618"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17233630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">C. E.</div></div>. </div><div class="article-title">Paxillin: adapting to change</div>, <div class="source">Physiol Rev.</div> , <div class="year">2004</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">1315</div>-<div class="lpage">1339</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paxillin%3A%20adapting%20to%20change&amp;author=M.%20C.%20Brown&amp;author=C.%20E.%20Turner&amp;publication_year=2004&amp;journal=Physiol%20Rev.&amp;volume=84&amp;pages=1315-1339" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/physrev.00002.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fphysrev.00002.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fphysrev.00002.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15383653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Emes</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Ponting</div> <div class="given-names">C. P.</div></div>. </div><div class="article-title">A new sequence motif linking lissencephaly, Treacher Collins and oral-facial-digital type 1 syndromes, microtubule dynamics and cell migration</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">2813</div>-<div class="lpage">2820</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20sequence%20motif%20linking%20lissencephaly%2C%20Treacher%20Collins%20and%20oral-facial-digital%20type%201%20syndromes%2C%20microtubule%20dynamics%20and%20cell%20migration&amp;author=R.%20D.%20Emes&amp;author=C.%20P.%20Ponting&amp;publication_year=2001&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=10&amp;pages=2813-2820" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/10.24.2813" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F10.24.2813" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F10.24.2813"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11734546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nguyen</div> <div class="given-names">N. M.</div></div>,  <div class="name"><div class="surname">Senior</div> <div class="given-names">R. M.</div></div>. </div><div class="article-title">Laminin isoforms and lung development: all isoforms are not equal</div>, <div class="source">Dev. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">294</div> (pg. <div class="fpage">271</div>-<div class="lpage">279</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Laminin%20isoforms%20and%20lung%20development%3A%20all%20isoforms%20are%20not%20equal&amp;author=N.%20M.%20Nguyen&amp;author=R.%20M.%20Senior&amp;publication_year=2006&amp;journal=Dev.%20Biol.&amp;volume=294&amp;pages=271-279" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2006.03.032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ydbio.2006.03.032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ydbio.2006.03.032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16643883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vinson</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Acharya</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Taparowsky</div> <div class="given-names">E. J.</div></div>. </div><div class="article-title">Deciphering B-ZIP transcription factor interactions in vitro and in vivo</div>, <div class="source">Biochim. Biophys. Acta.</div> , <div class="year">2006</div>, vol. <div class="volume">1759</div> (pg. <div class="fpage">4</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20B-ZIP%20transcription%20factor%20interactions%20in%20vitro%20and%20in%20vivo&amp;author=C.%20Vinson&amp;author=A.%20Acharya&amp;author=E.%20J.%20Taparowsky&amp;publication_year=2006&amp;journal=Biochim.%20Biophys.%20Acta.&amp;volume=1759&amp;pages=4-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbaexp.2005.12.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbaexp.2005.12.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbaexp.2005.12.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16580748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">D'Silva</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Holak</div> <div class="given-names">T. A.</div></div>. </div><div class="article-title">DNA-binding properties of the recombinant high-mobility-group-like AT-hook-containing region from human BRG1 protein</div>, <div class="source">Biol. Chem.</div> , <div class="year">2006</div>, vol. <div class="volume">387</div> (pg. <div class="fpage">1469</div>-<div class="lpage">1478</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-binding%20properties%20of%20the%20recombinant%20high-mobility-group-like%20AT-hook-containing%20region%20from%20human%20BRG1%20protein&amp;author=M.%20Singh&amp;author=L.%20D'Silva&amp;author=T.%20A.%20Holak&amp;publication_year=2006&amp;journal=Biol.%20Chem.&amp;volume=387&amp;pages=1469-1478" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1515/BC.2006.184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1515%2FBC.2006.184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1515%2FBC.2006.184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17081121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haigis</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Guarente</div> <div class="given-names">L. P.</div></div>. </div><div class="article-title">Mammalian sirtuins—emerging roles in physiology, aging, and calorie restriction</div>, <div class="source">Genes Dev.</div> , <div class="year">2006</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2913</div>-<div class="lpage">2921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mammalian%20sirtuins%E2%80%94emerging%20roles%20in%20physiology%2C%20aging%2C%20and%20calorie%20restriction&amp;author=M.%20C.%20Haigis&amp;author=L.%20P.%20Guarente&amp;publication_year=2006&amp;journal=Genes%20Dev.&amp;volume=20&amp;pages=2913-2921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1467506" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1467506" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1467506"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17079682" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diet</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Link</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Seifert</div> <div class="given-names">G. J.</div></div>, et al. </div><div class="article-title">The Arabidopsis root hair cell wall formation mutant lrx1 is suppressed by mutations in the RHM1 gene encoding a UDP-L-rhamnose synthase</div>, <div class="source">Plant Cell</div> , <div class="year">2006</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1630</div>-<div class="lpage">1641</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20root%20hair%20cell%20wall%20formation%20mutant%20lrx1%20is%20suppressed%20by%20mutations%20in%20the%20RHM1%20gene%20encoding%20a%20UDP-L-rhamnose%20synthase&amp;author=A.%20Diet&amp;author=B.%20Link&amp;author=G.%20J.%20Seifert&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=1630-1641" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.038653" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.038653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.038653"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16766693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vellinga</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">van-den-Wollenberg</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">van-der-Heijdt</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rabelink</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Hoeben</div> <div class="given-names">R. C.</div></div>. </div><div class="article-title">The coiled-coil domain of the adenovirus type 5 protein IX is dispensable for capsid incorporation and thermostability</div>, <div class="source">J. Virol.</div> , <div class="year">2005</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">3206</div>-<div class="lpage">3210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20coiled-coil%20domain%20of%20the%20adenovirus%20type%205%20protein%20IX%20is%20dispensable%20for%20capsid%20incorporation%20and%20thermostability&amp;author=J.%20Vellinga&amp;author=D.%20J.%20van-den-Wollenberg&amp;author=S.%20van-der-Heijdt&amp;author=M.%20J.%20Rabelink&amp;author=R.%20C.%20Hoeben&amp;publication_year=2005&amp;journal=J.%20Virol.&amp;volume=79&amp;pages=3206-3210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JVI.79.5.3206-3210.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJVI.79.5.3206-3210.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJVI.79.5.3206-3210.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15709043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Butterton</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Calderwood</div> <div class="given-names">S. B.</div></div>. </div><div class="article-title">Identification, cloning, and sequencing of a gene required for ferric vibriobactin utilization by <em>Vibrio cholerae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">5631</div>-<div class="lpage">5638</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%2C%20cloning%2C%20and%20sequencing%20of%20a%20gene%20required%20for%20ferric%20vibriobactin%20utilization%20by%20Vibrio%20cholerae&amp;author=J.%20R.%20Butterton&amp;author=S.%20B.%20Calderwood&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=5631-5638" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8083157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pagnoux</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Guilpain</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Guillevin</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Churg-Strauss syndrome</div>, <div class="source">Curr Opin. Rheumatol</div> , <div class="year">2007</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">25</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Churg-Strauss%20syndrome&amp;author=C.%20Pagnoux&amp;author=P.%20Guilpain&amp;author=L.%20Guillevin&amp;publication_year=2007&amp;journal=Curr%20Opin.%20Rheumatol&amp;volume=19&amp;pages=25-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1097/BOR.0b013e3280119854" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1097%2FBOR.0b013e3280119854" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1097%2FBOR.0b013e3280119854"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17143092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rakonjac</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fischer</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Provost</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Coactosin-like protein supports 5-lipoxygenase enzyme activity and up-regulates leukotriene A4 production</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">13150</div>-<div class="lpage">13155</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coactosin-like%20protein%20supports%205-lipoxygenase%20enzyme%20activity%20and%20up-regulates%20leukotriene%20A4%20production&amp;author=M.%20Rakonjac&amp;author=L.%20Fischer&amp;author=P.%20Provost&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=13150-13155" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0605150103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0605150103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0605150103"> </span></div> </div></div></div></div></div><div content-id="DSM035C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haidar</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Kiss</div> <div class="given-names">R. S.</div></div>,  <div class="name"><div class="surname">Sarov-Blat</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Cathepsin D, a lysosomal protease, regulates ABCA1-mediated lipid efflux</div>, <div class="source">J. Biol. Chem</div> , <div class="year">2006</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">39971</div>-<div class="lpage">39981</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cathepsin%20D%2C%20a%20lysosomal%20protease%2C%20regulates%20ABCA1-mediated%20lipid%20efflux&amp;author=B.%20Haidar&amp;author=R.%20S.%20Kiss&amp;author=L.%20Sarov-Blat&amp;publication_year=2006&amp;journal=J.%20Biol.%20Chem&amp;volume=281&amp;pages=39971-39981" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M605095200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M605095200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M605095200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17032648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiraiwa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">B. M.</div></div>,  <div class="name"><div class="surname">Kishimoto</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Conner</div> <div class="given-names">G. E.</div></div>,  <div class="name"><div class="surname">Tsuji</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">O'Brien</div> <div class="given-names">J. S.</div></div>. </div><div class="article-title">Lysosomal proteolysis of prosaposin, the precursor of saposins (sphingolipid activator proteins): its mechanism and inhibition by ganglioside</div>, <div class="source">Arch. Biochem. Biophys</div> , <div class="year">1997</div>, vol. <div class="volume">341</div> (pg. <div class="fpage">17</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysosomal%20proteolysis%20of%20prosaposin%2C%20the%20precursor%20of%20saposins%20(sphingolipid%20activator%20proteins)%3A%20its%20mechanism%20and%20inhibition%20by%20ganglioside&amp;author=M.%20Hiraiwa&amp;author=B.%20M.%20Martin&amp;author=Y.%20Kishimoto&amp;author=G.%20E.%20Conner&amp;author=S.%20Tsuji&amp;author=J.%20S.%20O'Brien&amp;publication_year=1997&amp;journal=Arch.%20Biochem.%20Biophys&amp;volume=341&amp;pages=17-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/abbi.1997.9958" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fabbi.1997.9958" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fabbi.1997.9958"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9143348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zheng</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Kollmeyer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Symolon</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Ceramides and other bioactive sphingolipid backbones in health and disease: lipidomic analysis, metabolism and roles in membrane structure, dynamics, signaling and autophagy</div>, <div class="source">Biochim. Biophys. Acta.</div> , <div class="year">2006</div>, vol. <div class="volume">1758</div> (pg. <div class="fpage">1864</div>-<div class="lpage">1884</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ceramides%20and%20other%20bioactive%20sphingolipid%20backbones%20in%20health%20and%20disease%3A%20lipidomic%20analysis%2C%20metabolism%20and%20roles%20in%20membrane%20structure%2C%20dynamics%2C%20signaling%20and%20autophagy&amp;author=W.%20Zheng&amp;author=J.%20Kollmeyer&amp;author=H.%20Symolon&amp;publication_year=2006&amp;journal=Biochim.%20Biophys.%20Acta.&amp;volume=1758&amp;pages=1864-1884" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbamem.2006.08.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbamem.2006.08.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbamem.2006.08.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17052686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leu</div> <div class="given-names">T. H.</div></div>,  <div class="name"><div class="surname">Charoenfuprasert</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yen</div> <div class="given-names">C. K.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">C. W.</div></div>,  <div class="name"><div class="surname">Maa</div> <div class="given-names">M. C.</div></div>. </div><div class="article-title">Lipopolysaccharide-induced c-Src expression plays a role in nitric oxide and TNFalpha secretion in macrophages</div>, <div class="source">Mol. Immunol.</div> , <div class="year">2006</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">308</div>-<div class="lpage">316</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lipopolysaccharide-induced%20c-Src%20expression%20plays%20a%20role%20in%20nitric%20oxide%20and%20TNFalpha%20secretion%20in%20macrophages&amp;author=T.%20H.%20Leu&amp;author=S.%20Charoenfuprasert&amp;author=C.%20K.%20Yen&amp;author=C.%20W.%20Fan&amp;author=M.%20C.%20Maa&amp;publication_year=2006&amp;journal=Mol.%20Immunol.&amp;volume=43&amp;pages=308-316" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molimm.2005.03.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molimm.2005.03.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molimm.2005.03.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15869794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vicentini</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Mazzi</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Caveggion</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Fgr deficiency results in defective eosinophil recruitment to the lung during allergic airway inflammation</div>, <div class="source">J. Immunol.</div> , <div class="year">2002</div>, vol. <div class="volume">168</div> (pg. <div class="fpage">6446</div>-<div class="lpage">6454</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fgr%20deficiency%20results%20in%20defective%20eosinophil%20recruitment%20to%20the%20lung%20during%20allergic%20airway%20inflammation&amp;author=L.%20Vicentini&amp;author=P.%20Mazzi&amp;author=E.%20Caveggion&amp;publication_year=2002&amp;journal=J.%20Immunol.&amp;volume=168&amp;pages=6446-6454" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4049/jimmunol.168.12.6446" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4049%2Fjimmunol.168.12.6446" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4049%2Fjimmunol.168.12.6446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12055264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Evangelista</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Pamuklar</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Piccoli</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Src family kinases mediate neutrophil adhesion to adherent platelets</div>, <div class="source">Blood</div> , <div class="year">2007</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">2461</div>-<div class="lpage">2469</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Src%20family%20kinases%20mediate%20neutrophil%20adhesion%20to%20adherent%20platelets&amp;author=V.%20Evangelista&amp;author=Z.%20Pamuklar&amp;author=A.%20Piccoli&amp;publication_year=2007&amp;journal=Blood&amp;volume=109&amp;pages=2461-2469" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1182/blood-2006-06-029082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1182%2Fblood-2006-06-029082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1182%2Fblood-2006-06-029082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17095622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Granel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Valleix</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Serratrice</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Lysozyme amyloidosis: report of 4 cases and a review of the literature</div>, <div class="source">Medicine (Baltimore)</div> , <div class="year">2007</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">66</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysozyme%20amyloidosis%3A%20report%20of%204%20cases%20and%20a%20review%20of%20the%20literature&amp;author=B.%20Granel&amp;author=S.%20Valleix&amp;author=J.%20Serratrice&amp;publication_year=2007&amp;journal=Medicine%20(Baltimore)&amp;volume=85&amp;pages=66-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1097/01.md.0000200467.51816.6d" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1097%2F01.md.0000200467.51816.6d" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1097%2F01.md.0000200467.51816.6d"> </span></div> </div></div></div></div></div><div content-id="DSM035C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meisler</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Stock</div> <div class="given-names">E. L.</div></div>,  <div class="name"><div class="surname">Wertz</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Khadem</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chaudhuri</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">O'Grady</div> <div class="given-names">R. B.</div></div>. </div><div class="article-title">Conjunctival inflammation and amyloidosis in allergic granulomatosis and angiitis (Churg-Strauss syndrome)</div>, <div class="source">Am. J. Ophthalmol.</div> , <div class="year">1981</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">216</div>-<div class="lpage">219</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conjunctival%20inflammation%20and%20amyloidosis%20in%20allergic%20granulomatosis%20and%20angiitis%20(Churg-Strauss%20syndrome)&amp;author=D.%20M.%20Meisler&amp;author=E.%20L.%20Stock&amp;author=R.%20D.%20Wertz&amp;author=M.%20Khadem&amp;author=B.%20Chaudhuri&amp;author=R.%20B.%20O'Grady&amp;publication_year=1981&amp;journal=Am.%20J.%20Ophthalmol.&amp;volume=91&amp;pages=216-219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0002-9394(81)90176-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0002-9394(81)90176-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0002-9394(81)90176-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7468737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scolding</div> <div class="given-names">N. J.</div></div>,  <div class="name"><div class="surname">Joseph</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kirby</div> <div class="given-names">P. A.</div></div>, et al. </div><div class="article-title">Abeta-related angiitis: primary angiitis of the central nervous system associated with cerebral amyloid angiopathy</div>, <div class="source">Brain</div> , <div class="year">2005</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">500</div>-<div class="lpage">515</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abeta-related%20angiitis%3A%20primary%20angiitis%20of%20the%20central%20nervous%20system%20associated%20with%20cerebral%20amyloid%20angiopathy&amp;author=N.%20J.%20Scolding&amp;author=F.%20Joseph&amp;author=P.%20A.%20Kirby&amp;publication_year=2005&amp;journal=Brain&amp;volume=128&amp;pages=500-515" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/brain/awh379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbrain%2Fawh379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbrain%2Fawh379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659428" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakagawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Aoyama</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Receptor-type protein tyrosine phosphatase epsilon (PTPepsilonM) is a negative regulator of insulin signaling in primary hepatocytes and liver</div>, <div class="source">Zoolog. Sci</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">169</div>-<div class="lpage">175</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Receptor-type%20protein%20tyrosine%20phosphatase%20epsilon%20(PTPepsilonM)%20is%20a%20negative%20regulator%20of%20insulin%20signaling%20in%20primary%20hepatocytes%20and%20liver&amp;author=Y.%20Nakagawa&amp;author=N.%20Aoki&amp;author=K.%20Aoyama&amp;publication_year=2005&amp;journal=Zoolog.%20Sci&amp;volume=22&amp;pages=169-175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2108/zsj.22.169" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2108%2Fzsj.22.169" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2108%2Fzsj.22.169"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15738637" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanuma</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Shima</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kikuchi</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Protein tyrosine phosphatase epsilonC selectively inhibits interleukin-6- and interleukin- 10-induced JAK-STAT signaling</div>, <div class="source">Blood</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">3030</div>-<div class="lpage">3034</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20tyrosine%20phosphatase%20epsilonC%20selectively%20inhibits%20interleukin-6-%20and%20interleukin-%2010-induced%20JAK-STAT%20signaling&amp;author=N.%20Tanuma&amp;author=H.%20Shima&amp;author=K.%20Nakamura&amp;author=K.%20Kikuchi&amp;publication_year=2001&amp;journal=Blood&amp;volume=98&amp;pages=3030-3034" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1182/blood.V98.10.3030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1182%2Fblood.V98.10.3030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1182%2Fblood.V98.10.3030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11698287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scheuerer</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ernst</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Durrbaum-Landmann</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">The CXC-chemokine platelet factor 4 promotes monocyte survival and induces monocyte differentiation into macrophages</div>, <div class="source">Blood</div> , <div class="year">2000</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">1158</div>-<div class="lpage">1166</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20CXC-chemokine%20platelet%20factor%204%20promotes%20monocyte%20survival%20and%20induces%20monocyte%20differentiation%20into%20macrophages&amp;author=B.%20Scheuerer&amp;author=M.%20Ernst&amp;author=I.%20Durrbaum-Landmann&amp;publication_year=2000&amp;journal=Blood&amp;volume=95&amp;pages=1158-1166" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10666185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shah</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Tabuchi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Nicastrin functions as a gamma-secretase-substrate receptor</div>, <div class="source">Cell</div> , <div class="year">2005</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">435</div>-<div class="lpage">447</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nicastrin%20functions%20as%20a%20gamma-secretase-substrate%20receptor&amp;author=S.%20Shah&amp;author=S.%20F.%20Lee&amp;author=K.%20Tabuchi&amp;publication_year=2005&amp;journal=Cell&amp;volume=122&amp;pages=435-447" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2005.05.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2005.05.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2005.05.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16096062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jutras</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Laplante</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Boulais</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Brunet</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Thinakaran</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Desjardins</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Gamma-secretase is a functional component of phagosomes</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2005</div>, vol. <div class="volume">280</div> (pg. <div class="fpage">36310</div>-<div class="lpage">36317</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gamma-secretase%20is%20a%20functional%20component%20of%20phagosomes&amp;author=I.%20Jutras&amp;author=A.%20Laplante&amp;author=J.%20Boulais&amp;author=S.%20Brunet&amp;author=G.%20Thinakaran&amp;author=M.%20Desjardins&amp;publication_year=2005&amp;journal=J.%20Biol.%20Chem.&amp;volume=280&amp;pages=36310-36317" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M504069200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M504069200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M504069200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16103123" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirata</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Furukawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">B. G.</div></div>, et al. </div><div class="article-title">Human P-selectin glycoprotein ligand-1 (PSGL-1) interacts with the skin-associated chemokine CCL27 via sulfated tyrosines at the PSGL-1 amino terminus</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">51775</div>-<div class="lpage">51782</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20P-selectin%20glycoprotein%20ligand-1%20(PSGL-1)%20interacts%20with%20the%20skin-associated%20chemokine%20CCL27%20via%20sulfated%20tyrosines%20at%20the%20PSGL-1%20amino%20terminus&amp;author=T.%20Hirata&amp;author=Y.%20Furukawa&amp;author=B.%20G.%20Yang&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=51775-51782" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M409868200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M409868200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M409868200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yasuoka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kuwana</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Autoantibody response against a novel testicular antigen protein highly expressed in testis (PHET) in SSc patients</div>, <div class="source">Autoimmun. Rev.</div> , <div class="year">2007</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">228</div>-<div class="lpage">231</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Autoantibody%20response%20against%20a%20novel%20testicular%20antigen%20protein%20highly%20expressed%20in%20testis%20(PHET)%20in%20SSc%20patients&amp;author=H.%20Yasuoka&amp;author=M.%20Kuwana&amp;publication_year=2007&amp;journal=Autoimmun.%20Rev.&amp;volume=6&amp;pages=228-231" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.autrev.2006.08.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.autrev.2006.08.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.autrev.2006.08.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17317613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Katoh</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kataoka</div> <div class="given-names">T. R.</div></div>, et al. </div><div class="article-title">A role for heterologous gap junctions between melanoma and endothelial cells in metastasis</div>, <div class="source">J. Clin. Invest.</div> , <div class="year">2000</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">1189</div>-<div class="lpage">1197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20heterologous%20gap%20junctions%20between%20melanoma%20and%20endothelial%20cells%20in%20metastasis&amp;author=A.%20Ito&amp;author=F.%20Katoh&amp;author=T.%20R.%20Kataoka&amp;publication_year=2000&amp;journal=J.%20Clin.%20Invest.&amp;volume=105&amp;pages=1189-1197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1172/JCI8257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1172%2FJCI8257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1172%2FJCI8257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10791993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watanabe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Miyashita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">T. T.</div></div>, et al. </div><div class="article-title">Comprehensive isolation of meiosis-specific genes identifies novel proteins and unusual non-coding transcripts in <em>Schizosaccharomyces pombe</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">2327</div>-<div class="lpage">2337</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20isolation%20of%20meiosis-specific%20genes%20identifies%20novel%20proteins%20and%20unusual%20non-coding%20transcripts%20in%20Schizosaccharomyces%20pombe&amp;author=T.%20Watanabe&amp;author=K.%20Miyashita&amp;author=T.%20T.%20Saito&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=2327-2337" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.11.2327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.11.2327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.11.2327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11376151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Shimaoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tougan</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Isolation and expression profiling of genes upregulated in bone marrow-derived mononuclear cells of rheumatoid arthritis patients</div>, <div class="source">DNA Res.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">169</div>-<div class="lpage">183</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20expression%20profiling%20of%20genes%20upregulated%20in%20bone%20marrow-derived%20mononuclear%20cells%20of%20rheumatoid%20arthritis%20patients&amp;author=N.%20Nakamura&amp;author=Y.%20Shimaoka&amp;author=T.%20Tougan&amp;publication_year=2006&amp;journal=DNA%20Res.&amp;volume=13&amp;pages=169-183" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsl006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsl006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsl006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17082220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watanabe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Miyashita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">T. T.</div></div>,  <div class="name"><div class="surname">Nabeshima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nojima</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Abundant poly(A)-bearing RNAs that lack open reading frames in Schizosaccharomyces pombe</div>, <div class="source">DNA Res.</div> , <div class="year">2002</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">209</div>-<div class="lpage">215</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundant%20poly(A)-bearing%20RNAs%20that%20lack%20open%20reading%20frames%20in%20Schizosaccharomyces%20pombe&amp;author=T.%20Watanabe&amp;author=K.%20Miyashita&amp;author=T.%20T.%20Saito&amp;author=K.%20Nabeshima&amp;author=H.%20Nojima&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=9&amp;pages=209-215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.6.209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.6.209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.6.209"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12597277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Radice</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sinico</div> <div class="given-names">R. A.</div></div>. </div><div class="article-title">Antineutrophil cytoplasmic antibodies (ANCA)</div>, <div class="source">Autoimmunity</div> , <div class="year">2005</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">93</div>-<div class="lpage">103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antineutrophil%20cytoplasmic%20antibodies%20(ANCA)&amp;author=A.%20Radice&amp;author=R.%20A.%20Sinico&amp;publication_year=2005&amp;journal=Autoimmunity&amp;volume=38&amp;pages=93-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/08916930400022673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F08916930400022673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F08916930400022673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15804710" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM035C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM035C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kallenberg</div> <div class="given-names">C. G.</div></div>,  <div class="name"><div class="surname">Heeringa</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Stegeman</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">Mechanisms of Disease: pathogenesis and treatment of ANCA-associated vasculitides</div>, <div class="source">Nat. Clin. Pract. Rheumatol</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">661</div>-<div class="lpage">670</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20Disease%3A%20pathogenesis%20and%20treatment%20of%20ANCA-associated%20vasculitides&amp;author=C.%20G.%20Kallenberg&amp;author=P.%20Heeringa&amp;author=C.%20A.%20Stegeman&amp;publication_year=2006&amp;journal=Nat.%20Clin.%20Pract.%20Rheumatol&amp;volume=2&amp;pages=661-670" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncprheum0355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncprheum0355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncprheum0355"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17133251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=375923&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm035/2/dsm035-dsm035supp.pdf?Expires=2147483647&amp;Signature=zcyq9~Nn~YKLEvzABtWHAFvUW1P7okv4FOSvAtvCpLh~0yd006yl2LYiqbARKeXooQg-v8CWJt-SZDwALKqNVsahLZS1U3dTm3vu~-8naLMw1-ndP10RAB6jCId9LNFQQz~zBm~S7VjGADuGD5krFInmcSIN9XB0Ey3F7w3ELE9yIjIXBaq~4Tvs58K4jinHeqG36kPLKt9g9fvRs7YGqfcpYck-h6O9CyCfbJ2ftt~NAWx3yh7sy9bFCJpQ683MXs8CEDEfh1rHrzer4ijePAoK~AfPzploCaVpGXDeGEP4tApSKR3Jv5CiHCwqgD3aPB8wWaOoLj0AE6d~jQ1fyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">148</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000256489900005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm035"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="375923"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000256489900005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f2%2f103%2f375923">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000256489900005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F2%2F103%2F375923" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/375923'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Focused Microarray Analysis of Peripheral Mononuclear Blood Cells from Churg–Strauss Syndrome Patients" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm035");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f103%2f375923">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

