

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/2/155/1745431","@type":"ScholarlyArticle","name":"Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome","datePublished":"2016-02-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2016-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw005","keywords":["3D genome organization","long-range chromatin interactions","replication","expression noise","evolutionary constraints","essential genes"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/2/155/1745431","author":[{"name":"Singh, Arashdeep","affiliation":"Department of Biological Sciences ,  Indian Institute of Science Education and Research (IISER)-Mohali ,  SAS Nagar 140306 ,  India","@type":"Person"},{"name":"Bagadia, Meenakshi","affiliation":"Department of Biological Sciences ,  Indian Institute of Science Education and Research (IISER)-Mohali ,  SAS Nagar 140306 ,  India","@type":"Person"},{"name":"Sandhu, Kuljeet Singh","affiliation":"Department of Biological Sciences ,  Indian Institute of Science Education and Research (IISER)-Mohali ,  SAS Nagar 140306 ,  India","@type":"Person"}],"description":"Abstract.  Despite recent advances, the underlying functional constraints that shape the three-dimensional organization of eukaryotic genome are not entirely cl","pageStart":"155","pageEnd":"169","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome" />
    <meta property="og:description" content="Abstract.  Despite recent advances, the underlying functional constraints that shape the three-dimensional organization of eukaryotic genome are not entirely cl" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/2/155/1745431" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-02-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Singh, Arashdeep" /><meta name="citation_author_institution" content="Department of Biological Sciences, Indian Institute of Science Education and Research (IISER)-Mohali, SAS Nagar 140306, India" /><meta name="citation_author" content="Bagadia, Meenakshi" /><meta name="citation_author_institution" content="Department of Biological Sciences, Indian Institute of Science Education and Research (IISER)-Mohali, SAS Nagar 140306, India" /><meta name="citation_author" content="Sandhu, Kuljeet Singh" /><meta name="citation_author_institution" content="Department of Biological Sciences, Indian Institute of Science Education and Research (IISER)-Mohali, SAS Nagar 140306, India" /><meta name="citation_title" content="Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome" /><meta name="citation_firstpage" content="155" /><meta name="citation_lastpage" content="169" /><meta name="citation_doi" content="10.1093/dnares/dsw005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2016/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Evolutionary conservation of chromosome territory arrangements in cell nuclei from higher primates;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=4424-9; " /><meta name="citation_reference" content="citation_title=Enhancer and promoter interactions-long distance calls;  citation_journal_title=Curr. Opin. Genet. Dev.;  citation_year=2012;  citation_volume=22;  citation_pages=79-85; " /><meta name="citation_reference" content="citation_title=DNA looping induced by a transcriptional enhancer in vivo;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=3743-50; " /><meta name="citation_reference" content="citation_title=Enhancers: the abundance and function of regulatory sequences beyond promoters;  citation_journal_title=Dev. Biol.;  citation_year=2010;  citation_volume=339;  citation_pages=250-7; " /><meta name="citation_reference" content="citation_title=Enhancer loops appear stable during development and are associated with paused polymerase;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=512;  citation_pages=96-100; " /><meta name="citation_reference" content="citation_title=Transcription factors mediate long-range enhancer-promoter interactions;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=20222-27; " /><meta name="citation_reference" content="citation_title=Remote control of gene transcription;  citation_journal_title=Hum. Mol. Genet.;  citation_year=2005;  citation_volume=14;  citation_pages=R101-11; " /><meta name="citation_reference" content="citation_title=Enhancer function: new insights into the regulation of tissue-specific gene expression;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=283-93; " /><meta name="citation_reference" content="citation_title=Dynamic genome architecture in the nuclear space: regulation of gene expression in three dimensions;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2007;  citation_volume=8;  citation_pages=104-15; " /><meta name="citation_reference" content="citation_title=Nuclear organization of the genome and the potential for gene regulation;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=447;  citation_pages=413-7; " /><meta name="citation_reference" content="citation_title=Active genes dynamically colocalize to shared sites of ongoing transcription;  citation_journal_title=Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=1065-71; " /><meta name="citation_reference" content="citation_title=Ultrastructural study of transcription factories in mouse erythroblasts;  citation_journal_title=J. Cell Sci.;  citation_year=2011;  citation_volume=124;  citation_pages=3676-83; " /><meta name="citation_reference" content="citation_title=Transcription factories: quantitative studies of nanostructures in the mammalian nucleus;  citation_journal_title=Chromosome Res.;  citation_year=2003;  citation_volume=11;  citation_pages=461-70; " /><meta name="citation_reference" content="citation_title=A model for all genomes: the role of transcription factories;  citation_journal_title=J. Mol. Biol.;  citation_year=2010;  citation_volume=395;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Transcription factories: gene expression in unions?;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=457-66; " /><meta name="citation_reference" content="citation_title=Numbers and organization of RNA polymerases, nascent transcripts, and transcription units in HeLa nuclei;  citation_journal_title=Mol. Biol. Cell;  citation_year=1998;  citation_volume=9;  citation_pages=1523-36; " /><meta name="citation_reference" content="citation_title=Capturing chromosome conformation;  citation_journal_title=Science;  citation_year=2002;  citation_volume=295;  citation_pages=1306-11; " /><meta name="citation_reference" content="citation_title=Circular chromosome conformation capture (4C) uncovers extensive networks of epigenetically regulated intra- and interchromosomal interactions;  citation_journal_title=Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=1341-7; " /><meta name="citation_reference" content="citation_title=An oestrogen-receptor-alpha-bound human chromatin interactome;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=462;  citation_pages=58-64; " /><meta name="citation_reference" content="citation_title=Chromosome Conformation Capture Carbon Copy (5C): a massively parallel solution for mapping interactions between genomic elements;  citation_journal_title=Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1299-309; " /><meta name="citation_reference" content="citation_title=Comprehensive mapping of long-range interactions reveals folding principles of the human genome;  citation_journal_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=289-93; " /><meta name="citation_reference" content="citation_title=A three-dimensional model of the yeast genome;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=465;  citation_pages=363-7; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of local chromatin packing in Arabidopsis thaliana;  citation_journal_title=Genome Res.;  citation_year=2015;  citation_volume=25;  citation_pages=246-56; " /><meta name="citation_reference" content="citation_title=Three-dimensional folding and functional organization principles of the Drosophila genome;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=148;  citation_pages=458-72; " /><meta name="citation_reference" content="citation_title=The three-dimensional architecture of a bacterial genome and its alteration by genetic perturbation;  citation_journal_title=Mol. Cell;  citation_year=2011;  citation_volume=44;  citation_pages=252-64; " /><meta name="citation_reference" content="citation_title=Mapping of long-range associations throughout the fission yeast genome reveals global genome organization linked to transcriptional regulation;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=8164-77; " /><meta name="citation_reference" content="citation_title=Topological domains in mammalian genomes identified by analysis of chromatin interactions;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=485;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=Chromatin architecture reorganization during stem cell differentiation;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=518;  citation_pages=331-6; " /><meta name="citation_reference" content="citation_title=Chromatin-driven behavior of topologically associating domains;  citation_journal_title=J. Mol. Biol.;  citation_year=2015;  citation_volume=427;  citation_pages=608-25; " /><meta name="citation_reference" content="citation_title=Topologically associating domains are stable units of replication-timing regulation;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=515;  citation_pages=402-5; " /><meta name="citation_reference" content="citation_title=Chromatin connectivity maps reveal dynamic promoter-enhancer long-range associations;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=504;  citation_pages=306-10; " /><meta name="citation_reference" content="citation_title=A high-resolution map of the three-dimensional chromatin interactome in human cells;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=503;  citation_pages=290-4; " /><meta name="citation_reference" content="citation_title=The long-range interaction landscape of gene promoters;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=489;  citation_pages=109-13; " /><meta name="citation_reference" content="citation_title=Extensive promoter-centered chromatin interactions provide a topological basis for transcription regulation;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=148;  citation_pages=84-98; " /><meta name="citation_reference" content="citation_title=The pluripotent regulatory circuitry connecting promoters to their long-range interacting elements;  citation_journal_title=Genome Res.;  citation_year=2015;  citation_volume=25;  citation_pages=582-97; " /><meta name="citation_reference" content="citation_title=Characterization of genome-wide enhancer-promoter interactions reveals co-expression of interacting genes and modes of higher order chromatin organization;  citation_journal_title=Cell Res.;  citation_year=2012;  citation_volume=22;  citation_pages=490-503; " /><meta name="citation_reference" content="citation_title=The 3D organization of the yeast genome correlates with co-expression and reflects functional relations between genes;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e54699" /><meta name="citation_reference" content="citation_title=Nuclear colocalization of transcription factor target genes strengthens coregulation in yeast;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=27-36; " /><meta name="citation_reference" content="citation_title=On the assessment of statistical significance of three-dimensional colocalization of sets of genomic elements;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=3849-55; " /><meta name="citation_reference" content="citation_title=The yeast genome undergoes significant topological reorganization in quiescence;  citation_journal_title=Nucleic Acids Res.;  citation_year=2015;  citation_volume=43;  citation_pages=8299-313; " /><meta name="citation_reference" content="citation_title=The temporal program of chromosome replication: genomewide replication in clb5{Delta} Saccharomyces cerevisiae;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=180;  citation_pages=1833-47; " /><meta name="citation_reference" content="citation_title=Mutation rates across budding yeast chromosome VI are correlated with replication timing;  citation_journal_title=Genome Biol. Evol.;  citation_year=2011;  citation_volume=3;  citation_pages=799-811; " /><meta name="citation_reference" content="citation_title=Nucleosomes positioned by ORC facilitate the initiation of DNA replication;  citation_journal_title=Mol. Cell;  citation_year=2001;  citation_volume=7;  citation_pages=21-30; " /><meta name="citation_reference" content="citation_title=Multiple determinants controlling activation of yeast replication origins late in S phase;  citation_journal_title=Genes Dev.;  citation_year=1996;  citation_volume=10;  citation_pages=1595-607; " /><meta name="citation_reference" content="citation_title=Diversity of eukaryotic DNA replication origins revealed by genome-wide analysis of chromatin structure;  citation_journal_title=PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1001092" /><meta name="citation_reference" content="citation_title=Evolution of chromosome organization driven by selection for reduced gene expression noise;  citation_journal_title=Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=945-9; " /><meta name="citation_reference" content="citation_title=Points of significance: replication;  citation_journal_title=Nat. Methods;  citation_year=2014;  citation_volume=11;  citation_pages=879-80; " /><meta name="citation_reference" content="citation_title=Somatic mosaicism in healthy human tissues;  citation_journal_title=Trends Genet.;  citation_year=2011;  citation_volume=27;  citation_pages=217-23; " /><meta name="citation_reference" content="citation_title=Stochastic and genetic factors influence tissue-specific decline in ageing C. elegans;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=419;  citation_pages=808-14; " /><meta name="citation_reference" content="citation_title=Overcoming technical variation and biological variation in quantitative proteomics;  citation_journal_title=Proteomics;  citation_year=2003;  citation_volume=3;  citation_pages=1912-9; " /><meta name="citation_reference" content="citation_title=Somatic mosaicism: on the road to cancer;  citation_journal_title=Nat. Rev. Cancer;  citation_year=2016;  citation_volume=16;  citation_pages=43-55; " /><meta name="citation_reference" content="citation_title=Interplay between gene expression noise and regulatory network architecture;  citation_journal_title=Trends Genet.;  citation_year=2012;  citation_volume=28;  citation_pages=221-32; " /><meta name="citation_reference" content="citation_title=Systematic characterization of the conformation and dynamics of budding yeast chromosome XII;  citation_journal_title=J. Cell Biol.;  citation_year=2013;  citation_volume=202;  citation_pages=201-10; " /><meta name="citation_reference" content="citation_title=Chromosome dynamics in the yeast interphase nucleus;  citation_journal_title=Science;  citation_year=2001;  citation_volume=294;  citation_pages=2181-6; " /><meta name="citation_reference" content="citation_title=A genome-wide analysis of promoter-mediated phenotypic noise in Escherichia coli;  citation_journal_title=PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002443" /><meta name="citation_reference" content="citation_title=Spatial features for Escherichia coli genome organization;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=37" /><meta name="citation_reference" content="citation_title=Genome conformation capture reveals that the Escherichia coli chromosome is organized by replication and transcription;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=6058-71; " /><meta name="citation_reference" content="citation_title=Tracing the derivation of embryonic stem cells from the inner cell mass by single-cell RNA-Seq analysis;  citation_journal_title=Cell Stem Cell;  citation_year=2010;  citation_volume=6;  citation_pages=468-78; " /><meta name="citation_reference" content="citation_title=The emergence of lineage-specific chromosomal topologies from coordinate gene regulation;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=6679-84; " /><meta name="citation_reference" content="citation_title=Genomic positions of co-expressed genes: echoes of chromosome organisation in gene expression data;  citation_journal_title=BMC Res. Notes;  citation_year=2013;  citation_volume=6;  citation_pages=229" /><meta name="citation_reference" content="citation_title=Preferential associations between co-regulated genes reveal a transcriptional interactome in erythroid cells;  citation_journal_title=Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=53-61; " /><meta name="citation_reference" content="citation_title=Colocalization of coregulated genes: a steered molecular dynamics study of human chromosome 19;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2013;  citation_volume=9;  citation_pages=e1003019" /><meta name="citation_reference" content="citation_title=Temporally coordinated assembly and disassembly of replication factories in the absence of DNA synthesis;  citation_journal_title=Nat. Cell Biol.;  citation_year=2000;  citation_volume=2;  citation_pages=686-94; " /><meta name="citation_reference" content="citation_title=Stochastic association of neighboring replicons creates replication factories in budding yeast;  citation_journal_title=J. Cell Biol.;  citation_year=2013;  citation_volume=202;  citation_pages=1001-12; " /><meta name="citation_reference" content="citation_title=Replication timing and epigenetic reprogramming of gene expression: a two-way relationship?;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=269-76; " /><meta name="citation_reference" content="citation_title=DNA replication-dependent induction of gene proximity by androgen;  citation_journal_title=Hum. Mol. Genet.;  citation_year=2015;  citation_volume=24;  citation_pages=963-71; " /><meta name="citation_reference" content="citation_title=Different nucleosomal architectures at early and late replicating origins in Saccharomyces cerevisiae;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=791" /><meta name="citation_reference" content="citation_title=The positioning and dynamics of origins of replication in the budding yeast nucleus;  citation_journal_title=J. Cell Biol.;  citation_year=2001;  citation_volume=152;  citation_pages=385-400; " /><meta name="citation_reference" content="citation_title=DNA replication timing and higher-order nuclear organization determine single-nucleotide substitution patterns in cancer genomes;  citation_journal_title=Nat. Commun.;  citation_year=2013;  citation_volume=4;  citation_pages=1502" /><meta name="citation_reference" content="citation_title=DNA replication timing and long-range DNA interactions predict mutational landscapes of cancer genomes;  citation_journal_title=Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=1103-8; " /><meta name="citation_reference" content="citation_title=Coordinate control of gene expression noise and interchromosomal interactions in a MAP kinase pathway;  citation_journal_title=Nat. Cell Biol.;  citation_year=2010;  citation_volume=12;  citation_pages=954-62; " /><meta name="citation_reference" content="citation_title=Chromatin motion is constrained by association with nuclear compartments in human cells;  citation_journal_title=Curr. Biol.;  citation_year=2002;  citation_volume=12;  citation_pages=439-45; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/2/155/17463214/dsw005.pdf" /><meta name="description" content="Abstract.  Despite recent advances, the underlying functional constraints that shape the three-dimensional organization of eukaryotic genome are not entirely cl" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/2/155/1745431" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/2/155/1745431" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/2/155/1745431");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw005';
    </script>
<title>
	Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745431" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcabATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2luZ2gsIEFyYXNoZGVlcCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgSW5kaWFuIEluc3RpdHV0ZSBvZiBTY2llbmNlIEVkdWNhdGlvbiBhbmQgUmVzZWFyY2ggKElJU0VSKS1Nb2hhbGksIFNBUyBOYWdhciAxNDAzMDYsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmFnYWRpYSwgTWVlbmFrc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBJbmRpYW4gSW5zdGl0dXRlIG9mIFNjaWVuY2UgRWR1Y2F0aW9uIGFuZCBSZXNlYXJjaCAoSUlTRVIpLU1vaGFsaSwgU0FTIE5hZ2FyIDE0MDMwNiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYW5kaHUsIEt1bGplZXQgU2luZ2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIEluZGlhbiBJbnN0aXR1dGUgb2YgU2NpZW5jZSBFZHVjYXRpb24gYW5kIFJlc2VhcmNoIChJSVNFUiktTW9oYWxpLCBTQVMgTmFnYXIgMTQwMzA2LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iU3BhdGlhbGx5IGNvb3JkaW5hdGVkIHJlcGxpY2F0aW9uIGFuZCBtaW5pbWl6YXRpb24gb2YgZXhwcmVzc2lvbiBub2lzZSBjb25zdHJhaW4gdGhyZWUtZGltZW5zaW9uYWwgb3JnYW5pemF0aW9uIG9mIHllYXN0IGdlbm9tZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE1NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTY5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgY29uc2VydmF0aW9uIG9mIGNocm9tb3NvbWUgdGVycml0b3J5IGFycmFuZ2VtZW50cyBpbiBjZWxsIG51Y2xlaSBmcm9tIGhpZ2hlciBwcmltYXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTQ0MjQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5oYW5jZXIgYW5kIHByb21vdGVyIGludGVyYWN0aW9ucy1sb25nIGRpc3RhbmNlIGNhbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTc5LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbG9vcGluZyBpbmR1Y2VkIGJ5IGEgdHJhbnNjcmlwdGlvbmFsIGVuaGFuY2VyIGluIHZpdm87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTM3NDMtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuaGFuY2VyczogdGhlIGFidW5kYW5jZSBhbmQgZnVuY3Rpb24gb2YgcmVndWxhdG9yeSBzZXF1ZW5jZXMgYmV5b25kIHByb21vdGVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RGV2LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzk7ICBjaXRhdGlvbl9wYWdlcz0yNTAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5oYW5jZXIgbG9vcHMgYXBwZWFyIHN0YWJsZSBkdXJpbmcgZGV2ZWxvcG1lbnQgYW5kIGFyZSBhc3NvY2lhdGVkIHdpdGggcGF1c2VkIHBvbHltZXJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MTI7ICBjaXRhdGlvbl9wYWdlcz05Ni0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gZmFjdG9ycyBtZWRpYXRlIGxvbmctcmFuZ2UgZW5oYW5jZXItcHJvbW90ZXIgaW50ZXJhY3Rpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTIwMjIyLTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZW1vdGUgY29udHJvbCBvZiBnZW5lIHRyYW5zY3JpcHRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gTW9sLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMTAxLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmhhbmNlciBmdW5jdGlvbjogbmV3IGluc2lnaHRzIGludG8gdGhlIHJlZ3VsYXRpb24gb2YgdGlzc3VlLXNwZWNpZmljIGdlbmUgZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI4My05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyBnZW5vbWUgYXJjaGl0ZWN0dXJlIGluIHRoZSBudWNsZWFyIHNwYWNlOiByZWd1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiB0aHJlZSBkaW1lbnNpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMDQtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlYXIgb3JnYW5pemF0aW9uIG9mIHRoZSBnZW5vbWUgYW5kIHRoZSBwb3RlbnRpYWwgZm9yIGdlbmUgcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0NzsgIGNpdGF0aW9uX3BhZ2VzPTQxMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3RpdmUgZ2VuZXMgZHluYW1pY2FsbHkgY29sb2NhbGl6ZSB0byBzaGFyZWQgc2l0ZXMgb2Ygb25nb2luZyB0cmFuc2NyaXB0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTEwNjUtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhc3RydWN0dXJhbCBzdHVkeSBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvcmllcyBpbiBtb3VzZSBlcnl0aHJvYmxhc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDZWxsIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9MzY3Ni04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBmYWN0b3JpZXM6IHF1YW50aXRhdGl2ZSBzdHVkaWVzIG9mIG5hbm9zdHJ1Y3R1cmVzIGluIHRoZSBtYW1tYWxpYW4gbnVjbGV1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb21vc29tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDYxLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1vZGVsIGZvciBhbGwgZ2Vub21lczogdGhlIHJvbGUgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3JpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzk1OyAgY2l0YXRpb25fcGFnZXM9MS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBmYWN0b3JpZXM6IGdlbmUgZXhwcmVzc2lvbiBpbiB1bmlvbnM/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NDU3LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdW1iZXJzIGFuZCBvcmdhbml6YXRpb24gb2YgUk5BIHBvbHltZXJhc2VzLCBuYXNjZW50IHRyYW5zY3JpcHRzLCBhbmQgdHJhbnNjcmlwdGlvbiB1bml0cyBpbiBIZUxhIG51Y2xlaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xNTIzLTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXB0dXJpbmcgY2hyb21vc29tZSBjb25mb3JtYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk1OyAgY2l0YXRpb25fcGFnZXM9MTMwNi0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgY2hyb21vc29tZSBjb25mb3JtYXRpb24gY2FwdHVyZSAoNEMpIHVuY292ZXJzIGV4dGVuc2l2ZSBuZXR3b3JrcyBvZiBlcGlnZW5ldGljYWxseSByZWd1bGF0ZWQgaW50cmEtIGFuZCBpbnRlcmNocm9tb3NvbWFsIGludGVyYWN0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMzQxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIG9lc3Ryb2dlbi1yZWNlcHRvci1hbHBoYS1ib3VuZCBodW1hbiBjaHJvbWF0aW4gaW50ZXJhY3RvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NjI7ICBjaXRhdGlvbl9wYWdlcz01OC02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tZSBDb25mb3JtYXRpb24gQ2FwdHVyZSBDYXJib24gQ29weSAoNUMpOiBhIG1hc3NpdmVseSBwYXJhbGxlbCBzb2x1dGlvbiBmb3IgbWFwcGluZyBpbnRlcmFjdGlvbnMgYmV0d2VlbiBnZW5vbWljIGVsZW1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEyOTktMzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIG1hcHBpbmcgb2YgbG9uZy1yYW5nZSBpbnRlcmFjdGlvbnMgcmV2ZWFscyBmb2xkaW5nIHByaW5jaXBsZXMgb2YgdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0yODktOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGhyZWUtZGltZW5zaW9uYWwgbW9kZWwgb2YgdGhlIHllYXN0IGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NTsgIGNpdGF0aW9uX3BhZ2VzPTM2My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBsb2NhbCBjaHJvbWF0aW4gcGFja2luZyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNDYtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRocmVlLWRpbWVuc2lvbmFsIGZvbGRpbmcgYW5kIGZ1bmN0aW9uYWwgb3JnYW5pemF0aW9uIHByaW5jaXBsZXMgb2YgdGhlIERyb3NvcGhpbGEgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0ODsgIGNpdGF0aW9uX3BhZ2VzPTQ1OC03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRocmVlLWRpbWVuc2lvbmFsIGFyY2hpdGVjdHVyZSBvZiBhIGJhY3RlcmlhbCBnZW5vbWUgYW5kIGl0cyBhbHRlcmF0aW9uIGJ5IGdlbmV0aWMgcGVydHVyYmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0yNTItNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgb2YgbG9uZy1yYW5nZSBhc3NvY2lhdGlvbnMgdGhyb3VnaG91dCB0aGUgZmlzc2lvbiB5ZWFzdCBnZW5vbWUgcmV2ZWFscyBnbG9iYWwgZ2Vub21lIG9yZ2FuaXphdGlvbiBsaW5rZWQgdG8gdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTgxNjQtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvcG9sb2dpY2FsIGRvbWFpbnMgaW4gbWFtbWFsaWFuIGdlbm9tZXMgaWRlbnRpZmllZCBieSBhbmFseXNpcyBvZiBjaHJvbWF0aW4gaW50ZXJhY3Rpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg1OyAgY2l0YXRpb25fcGFnZXM9Mzc2LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbWF0aW4gYXJjaGl0ZWN0dXJlIHJlb3JnYW5pemF0aW9uIGR1cmluZyBzdGVtIGNlbGwgZGlmZmVyZW50aWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTE4OyAgY2l0YXRpb25fcGFnZXM9MzMxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tYXRpbi1kcml2ZW4gYmVoYXZpb3Igb2YgdG9wb2xvZ2ljYWxseSBhc3NvY2lhdGluZyBkb21haW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQyNzsgIGNpdGF0aW9uX3BhZ2VzPTYwOC0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wb2xvZ2ljYWxseSBhc3NvY2lhdGluZyBkb21haW5zIGFyZSBzdGFibGUgdW5pdHMgb2YgcmVwbGljYXRpb24tdGltaW5nIHJlZ3VsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MTU7ICBjaXRhdGlvbl9wYWdlcz00MDItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21hdGluIGNvbm5lY3Rpdml0eSBtYXBzIHJldmVhbCBkeW5hbWljIHByb21vdGVyLWVuaGFuY2VyIGxvbmctcmFuZ2UgYXNzb2NpYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTA0OyAgY2l0YXRpb25fcGFnZXM9MzA2LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtcmVzb2x1dGlvbiBtYXAgb2YgdGhlIHRocmVlLWRpbWVuc2lvbmFsIGNocm9tYXRpbiBpbnRlcmFjdG9tZSBpbiBodW1hbiBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTUwMzsgIGNpdGF0aW9uX3BhZ2VzPTI5MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbG9uZy1yYW5nZSBpbnRlcmFjdGlvbiBsYW5kc2NhcGUgb2YgZ2VuZSBwcm9tb3RlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODk7ICBjaXRhdGlvbl9wYWdlcz0xMDktMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBwcm9tb3Rlci1jZW50ZXJlZCBjaHJvbWF0aW4gaW50ZXJhY3Rpb25zIHByb3ZpZGUgYSB0b3BvbG9naWNhbCBiYXNpcyBmb3IgdHJhbnNjcmlwdGlvbiByZWd1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0ODsgIGNpdGF0aW9uX3BhZ2VzPTg0LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGx1cmlwb3RlbnQgcmVndWxhdG9yeSBjaXJjdWl0cnkgY29ubmVjdGluZyBwcm9tb3RlcnMgdG8gdGhlaXIgbG9uZy1yYW5nZSBpbnRlcmFjdGluZyBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz01ODItOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgZ2Vub21lLXdpZGUgZW5oYW5jZXItcHJvbW90ZXIgaW50ZXJhY3Rpb25zIHJldmVhbHMgY28tZXhwcmVzc2lvbiBvZiBpbnRlcmFjdGluZyBnZW5lcyBhbmQgbW9kZXMgb2YgaGlnaGVyIG9yZGVyIGNocm9tYXRpbiBvcmdhbml6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTQ5MC01MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSAzRCBvcmdhbml6YXRpb24gb2YgdGhlIHllYXN0IGdlbm9tZSBjb3JyZWxhdGVzIHdpdGggY28tZXhwcmVzc2lvbiBhbmQgcmVmbGVjdHMgZnVuY3Rpb25hbCByZWxhdGlvbnMgYmV0d2VlbiBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU1NDY5OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlYXIgY29sb2NhbGl6YXRpb24gb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3IgdGFyZ2V0IGdlbmVzIHN0cmVuZ3RoZW5zIGNvcmVndWxhdGlvbiBpbiB5ZWFzdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MjctMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIHRoZSBhc3Nlc3NtZW50IG9mIHN0YXRpc3RpY2FsIHNpZ25pZmljYW5jZSBvZiB0aHJlZS1kaW1lbnNpb25hbCBjb2xvY2FsaXphdGlvbiBvZiBzZXRzIG9mIGdlbm9taWMgZWxlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTM4NDktNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB5ZWFzdCBnZW5vbWUgdW5kZXJnb2VzIHNpZ25pZmljYW50IHRvcG9sb2dpY2FsIHJlb3JnYW5pemF0aW9uIGluIHF1aWVzY2VuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTgyOTktMzEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdGVtcG9yYWwgcHJvZ3JhbSBvZiBjaHJvbW9zb21lIHJlcGxpY2F0aW9uOiBnZW5vbWV3aWRlIHJlcGxpY2F0aW9uIGluIGNsYjV7RGVsdGF9IFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9MTgzMy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb24gcmF0ZXMgYWNyb3NzIGJ1ZGRpbmcgeWVhc3QgY2hyb21vc29tZSBWSSBhcmUgY29ycmVsYXRlZCB3aXRoIHJlcGxpY2F0aW9uIHRpbWluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz03OTktODExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW9zb21lcyBwb3NpdGlvbmVkIGJ5IE9SQyBmYWNpbGl0YXRlIHRoZSBpbml0aWF0aW9uIG9mIEROQSByZXBsaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgZGV0ZXJtaW5hbnRzIGNvbnRyb2xsaW5nIGFjdGl2YXRpb24gb2YgeWVhc3QgcmVwbGljYXRpb24gb3JpZ2lucyBsYXRlIGluIFMgcGhhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNTk1LTYwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaXR5IG9mIGV1a2FyeW90aWMgRE5BIHJlcGxpY2F0aW9uIG9yaWdpbnMgcmV2ZWFsZWQgYnkgZ2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgY2hyb21hdGluIHN0cnVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMDkyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGNocm9tb3NvbWUgb3JnYW5pemF0aW9uIGRyaXZlbiBieSBzZWxlY3Rpb24gZm9yIHJlZHVjZWQgZ2VuZSBleHByZXNzaW9uIG5vaXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTk0NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2ludHMgb2Ygc2lnbmlmaWNhbmNlOiByZXBsaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODc5LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb21hdGljIG1vc2FpY2lzbSBpbiBoZWFsdGh5IGh1bWFuIHRpc3N1ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMTctMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0b2NoYXN0aWMgYW5kIGdlbmV0aWMgZmFjdG9ycyBpbmZsdWVuY2UgdGlzc3VlLXNwZWNpZmljIGRlY2xpbmUgaW4gYWdlaW5nIEMuIGVsZWdhbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTk7ICBjaXRhdGlvbl9wYWdlcz04MDgtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU92ZXJjb21pbmcgdGVjaG5pY2FsIHZhcmlhdGlvbiBhbmQgYmlvbG9naWNhbCB2YXJpYXRpb24gaW4gcXVhbnRpdGF0aXZlIHByb3Rlb21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTE5MTItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29tYXRpYyBtb3NhaWNpc206IG9uIHRoZSByb2FkIHRvIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIENhbmNlcjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQzLTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcnBsYXkgYmV0d2VlbiBnZW5lIGV4cHJlc3Npb24gbm9pc2UgYW5kIHJlZ3VsYXRvcnkgbmV0d29yayBhcmNoaXRlY3R1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yMjEtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgY29uZm9ybWF0aW9uIGFuZCBkeW5hbWljcyBvZiBidWRkaW5nIHllYXN0IGNocm9tb3NvbWUgWElJOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwMjsgIGNpdGF0aW9uX3BhZ2VzPTIwMS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tZSBkeW5hbWljcyBpbiB0aGUgeWVhc3QgaW50ZXJwaGFzZSBudWNsZXVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5NDsgIGNpdGF0aW9uX3BhZ2VzPTIxODEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBwcm9tb3Rlci1tZWRpYXRlZCBwaGVub3R5cGljIG5vaXNlIGluIEVzY2hlcmljaGlhIGNvbGk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjQ0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwYXRpYWwgZmVhdHVyZXMgZm9yIEVzY2hlcmljaGlhIGNvbGkgZ2Vub21lIG9yZ2FuaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgY29uZm9ybWF0aW9uIGNhcHR1cmUgcmV2ZWFscyB0aGF0IHRoZSBFc2NoZXJpY2hpYSBjb2xpIGNocm9tb3NvbWUgaXMgb3JnYW5pemVkIGJ5IHJlcGxpY2F0aW9uIGFuZCB0cmFuc2NyaXB0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz02MDU4LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFjaW5nIHRoZSBkZXJpdmF0aW9uIG9mIGVtYnJ5b25pYyBzdGVtIGNlbGxzIGZyb20gdGhlIGlubmVyIGNlbGwgbWFzcyBieSBzaW5nbGUtY2VsbCBSTkEtU2VxIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIFN0ZW0gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NDY4LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZW1lcmdlbmNlIG9mIGxpbmVhZ2Utc3BlY2lmaWMgY2hyb21vc29tYWwgdG9wb2xvZ2llcyBmcm9tIGNvb3JkaW5hdGUgZ2VuZSByZWd1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTY2NzktODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgcG9zaXRpb25zIG9mIGNvLWV4cHJlc3NlZCBnZW5lczogZWNob2VzIG9mIGNocm9tb3NvbWUgb3JnYW5pc2F0aW9uIGluIGdlbmUgZXhwcmVzc2lvbiBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUmVzLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZmVyZW50aWFsIGFzc29jaWF0aW9ucyBiZXR3ZWVuIGNvLXJlZ3VsYXRlZCBnZW5lcyByZXZlYWwgYSB0cmFuc2NyaXB0aW9uYWwgaW50ZXJhY3RvbWUgaW4gZXJ5dGhyb2lkIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2xvY2FsaXphdGlvbiBvZiBjb3JlZ3VsYXRlZCBnZW5lczogYSBzdGVlcmVkIG1vbGVjdWxhciBkeW5hbWljcyBzdHVkeSBvZiBodW1hbiBjaHJvbW9zb21lIDE5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzMDE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVtcG9yYWxseSBjb29yZGluYXRlZCBhc3NlbWJseSBhbmQgZGlzYXNzZW1ibHkgb2YgcmVwbGljYXRpb24gZmFjdG9yaWVzIGluIHRoZSBhYnNlbmNlIG9mIEROQSBzeW50aGVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9Njg2LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdG9jaGFzdGljIGFzc29jaWF0aW9uIG9mIG5laWdoYm9yaW5nIHJlcGxpY29ucyBjcmVhdGVzIHJlcGxpY2F0aW9uIGZhY3RvcmllcyBpbiBidWRkaW5nIHllYXN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMDEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGxpY2F0aW9uIHRpbWluZyBhbmQgZXBpZ2VuZXRpYyByZXByb2dyYW1taW5nIG9mIGdlbmUgZXhwcmVzc2lvbjogYSB0d28td2F5IHJlbGF0aW9uc2hpcD87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNjktNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSByZXBsaWNhdGlvbi1kZXBlbmRlbnQgaW5kdWN0aW9uIG9mIGdlbmUgcHJveGltaXR5IGJ5IGFuZHJvZ2VuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9OTYzLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnQgbnVjbGVvc29tYWwgYXJjaGl0ZWN0dXJlcyBhdCBlYXJseSBhbmQgbGF0ZSByZXBsaWNhdGluZyBvcmlnaW5zIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NzkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBvc2l0aW9uaW5nIGFuZCBkeW5hbWljcyBvZiBvcmlnaW5zIG9mIHJlcGxpY2F0aW9uIGluIHRoZSBidWRkaW5nIHllYXN0IG51Y2xldXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTUyOyAgY2l0YXRpb25fcGFnZXM9Mzg1LTQwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHJlcGxpY2F0aW9uIHRpbWluZyBhbmQgaGlnaGVyLW9yZGVyIG51Y2xlYXIgb3JnYW5pemF0aW9uIGRldGVybWluZSBzaW5nbGUtbnVjbGVvdGlkZSBzdWJzdGl0dXRpb24gcGF0dGVybnMgaW4gY2FuY2VyIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTUwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSByZXBsaWNhdGlvbiB0aW1pbmcgYW5kIGxvbmctcmFuZ2UgRE5BIGludGVyYWN0aW9ucyBwcmVkaWN0IG11dGF0aW9uYWwgbGFuZHNjYXBlcyBvZiBjYW5jZXIgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTExMDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29vcmRpbmF0ZSBjb250cm9sIG9mIGdlbmUgZXhwcmVzc2lvbiBub2lzZSBhbmQgaW50ZXJjaHJvbW9zb21hbCBpbnRlcmFjdGlvbnMgaW4gYSBNQVAga2luYXNlIHBhdGh3YXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTk1NC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21hdGluIG1vdGlvbiBpcyBjb25zdHJhaW5lZCBieSBhc3NvY2lhdGlvbiB3aXRoIG51Y2xlYXIgY29tcGFydG1lbnRzIGluIGh1bWFuIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQzOS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzIvMTU1LzE3NDYzMjE0L2RzdzAwNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRGVzcGl0ZSByZWNlbnQgYWR2YW5jZXMsIHRoZSB1bmRlcmx5aW5nIGZ1bmN0aW9uYWwgY29uc3RyYWludHMgdGhhdCBzaGFwZSB0aGUgdGhyZWUtZGltZW5zaW9uYWwgb3JnYW5pemF0aW9uIG9mIGV1a2FyeW90aWMgZ2Vub21lIGFyZSBub3QgZW50aXJlbHkgY2wiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTE3NDU0MzEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkHBMnELn0IdZE1R44rRLNf3TDa7Q=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f155%2f1745431" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f155%2f1745431" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69060" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&amp;Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/2">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200058">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200058">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200060">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200060">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200063">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200063">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200078">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200078">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200105">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200105">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200112">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200112">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200114">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200114">Authors’ contribution</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200116">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200116">Conflict of interest statement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200118">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200118">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200120">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200120">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200122">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200122">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/2/145/1745392">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/2/171/1745492">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arashdeep Singh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">Indian Institute of Science Education and Research (IISER)-Mohali</div>, <div class="addr-line">SAS Nagar 140306</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arashdeep+Singh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Singh A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Singh A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Arashdeep Singh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Meenakshi Bagadia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">Indian Institute of Science Education and Research (IISER)-Mohali</div>, <div class="addr-line">SAS Nagar 140306</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Meenakshi+Bagadia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bagadia M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bagadia M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Meenakshi Bagadia</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kuljeet Singh Sandhu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">Indian Institute of Science Education and Research (IISER)-Mohali</div>, <div class="addr-line">SAS Nagar 140306</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 172-2293182. Fax. +91 172-2240266. E-mail: <a href="mailto:sandhuks@iisermohali.ac.in" target="_blank">sandhuks@iisermohali.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kuljeet+Singh+Sandhu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sandhu K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sandhu K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kuljeet Singh Sandhu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 155–169, <a href='https://doi.org/10.1093/dnares/dsw005'>https://doi.org/10.1093/dnares/dsw005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 February 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">04 November 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">31 January 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745431" href="/dnaresearch/article-pdf/23/2/155/17463214/dsw005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Arashdeep Singh, Meenakshi Bagadia, Kuljeet Singh Sandhu;  Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome, <em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 155–169, <a href='https://doi.org/10.1093/dnares/dsw005'>https://doi.org/10.1093/dnares/dsw005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745431?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745431?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745431?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745431?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745431?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745431?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745431?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Spatially%20coordinated%20replication%20and%20minimization%20of%20expression%20noise%20constrain%20three-dimensional%20organization%20of%20yeast%20genome&amp;publicationDate=2016-02-29&amp;volumeNum=23&amp;issueNum=2&amp;author=Singh%2C%20Arashdeep%3B%20Bagadia%2C%20Meenakshi&amp;startPage=155&amp;endPage=169&amp;contentId=10.1093%2Fdnares%2Fdsw005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f155%2f1745431" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200058 id="87200058" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Despite recent advances, the underlying functional constraints that shape the three-dimensional organization of eukaryotic genome are not entirely clear. Through comprehensive multivariate analyses of genome-wide datasets, we show that <em>cis</em> and <em>trans</em> interactions in yeast genome have significantly distinct functional associations. In particular, (i) the <em>trans</em> interactions are constrained by coordinated replication and co-varying mutation rates of early replicating domains through interactions among early origins, while <em>cis</em> interactions are constrained by coordination of late replication through interactions among late origins; (ii) <em>cis</em> and <em>trans</em> interactions exhibit differential preference for nucleosome occupancy; (iii) <em>cis</em> interactions are also constrained by the essentiality and co-fitness of interacting genes. Essential gene clusters associate with high average interaction frequency, relatively short-range interactions of low variance, and exhibit less fluctuations in chromatin conformation, marking a physically restrained state of engaged loci that, we suggest, is important to mitigate the epigenetic errors by restricting the spatial mobility of loci. Indeed, the genes with lower expression noise associate with relatively short-range interactions of lower variance and exhibit relatively higher average interaction frequency, a property that is conserved across <em>Escherichia coli,</em> yeast, and mESCs. Altogether, our observations highlight the coordination of replication and the minimization of expression noise, not necessarily co-expression of genes, as potent evolutionary constraints shaping the spatial organization of yeast genome.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;3D genome organization&quot;">3D genome organization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;long-range chromatin interactions&quot;">long-range chromatin interactions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="replication">replication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;expression noise&quot;">expression noise</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;evolutionary constraints&quot;">evolutionary constraints</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;essential genes&quot;">essential genes</a></div>                            <h2 scrollto-destination=87200060 id="87200060" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Eukaryotic genes and their regulatory elements communicate with each other through a complex wiring of long-range interactions.<sup><a reveal-id="DSW005C1" data-open="DSW005C1" class="link link-ref link-reveal xref-bibr">1</a></sup> It is now well established that distal enhancers can physically juxtapose to their cognate promoters for transcriptional regulation.<sup><a reveal-id="DSW005C2  DSW005C8" data-open="DSW005C2  DSW005C8" class="link link-ref link-reveal xref-bibr">2–8</a></sup> Interestingly, distant genes can also co-localize in nuclear space.<sup><a reveal-id="DSW005C9" data-open="DSW005C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW005C10" data-open="DSW005C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The prevailing view is that the genes spatially cluster at concentrated foci of RNA polymerase II, also known as transcription factories.<sup><a reveal-id="DSW005C11  DSW005C16" data-open="DSW005C11  DSW005C16" class="link link-ref link-reveal xref-bibr">11–16</a></sup> It is suggested that the spatial convergence of genes at transcription factories provide a topological basis of co-expression of engaged genes; however, such proposals have not been subjected to proper scrutiny. Recent advent of high throughput derivatives of Chromosome Conformation Capture (3C) has availed genome-wide quantitative data of long-range chromatin interactions across diverse spectra of model systems.<sup><a reveal-id="DSW005C17  DSW005C26" data-open="DSW005C17  DSW005C26" class="link link-ref link-reveal xref-bibr">17–26</a></sup> Briefly, in 3C-derived techniques, the chromatin is cross-linked with formaldehyde, restriction digested, and the open ends of cross-linked products are ligated in diluted conditions to prefer intra-molecular ligation over inter-molecular. In HiC, the ligated junctions are then pulled down and sequenced using deep sequencing to unravel all-to-all chromatin interactions.<sup><a reveal-id="DSW005C21" data-open="DSW005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> HiC has revealed large topologically associated domains (TADs) that exhibit high density of intra-connectivity of chromatin and are largely conserved across cell lineages.<sup><a reveal-id="DSW005C27" data-open="DSW005C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSW005C28" data-open="DSW005C28" class="link link-ref link-reveal xref-bibr">28</a></sup> TADs are tightly associated with the chromatin type and replication timing, and are marked by CTCF on boundaries.<sup><a reveal-id="DSW005C27" data-open="DSW005C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSW005C29" data-open="DSW005C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSW005C30" data-open="DSW005C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Wide-spread enhancer-to-promoter interactions, that are mostly cell-type specific, have been uncovered across several systems.<sup><a reveal-id="DSW005C28" data-open="DSW005C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSW005C31  DSW005C33" data-open="DSW005C31  DSW005C33" class="link link-ref link-reveal xref-bibr">31–33</a></sup> Zhang et al.<sup><a reveal-id="DSW005C31" data-open="DSW005C31" class="link link-ref link-reveal xref-bibr">31</a></sup> have suggested differential usage of enhancers during embryonic stem cell differentiation. Some studies have also revealed promoter-to-terminator interactions commonly found for housekeeping genes,<sup><a reveal-id="DSW005C34" data-open="DSW005C34" class="link link-ref link-reveal xref-bibr">34</a></sup> possibly ascribing a circular template for recurrent transcription. Most interesting of all is the widespread promoter-to-promoter interactions among genes impinging from neighbouring regions to form discrete multi-gene complexes.<sup><a reveal-id="DSW005C34" data-open="DSW005C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSW005C35" data-open="DSW005C35" class="link link-ref link-reveal xref-bibr">35</a></sup> However, what functional and evolutionary constraints might have shaped the large-scale organization of promoter–promoter interactions is not entirely clear. Although the genes within multi-gene complexes are shown to be co-expressed,<sup><a reveal-id="DSW005C34" data-open="DSW005C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSW005C36" data-open="DSW005C36" class="link link-ref link-reveal xref-bibr">36</a></sup> whether or not co-expression of engaged genes is dependent on their spatial, but not the linear, proximity remains to be seen. Moreover, it is hypothesized that interacting promoters can influence transcriptional states of each other and that the promoter of one gene can function as an enhancer of other gene.<sup><a reveal-id="DSW005C34" data-open="DSW005C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Nevertheless, these proposals are yet not established as fact. Importantly, most of these studies have primarily focussed on intra-chromosomal (referred as ‘<em>cis’</em> in this study) interactions and whether or not distant genes converging from different chromosomes (referred as <em>trans</em> interactions) have functional association is yet not clear. Comprehensive statistical analyses of accumulated HiC like datasets can answer several questions pertaining to non-random genome organization. Here, we ask whether we can delineate evolutionary constraints of three-dimensional organization of genome.</p><p>Multivariate analyses provide a statistical platform to assess the association of several different functional variables in an unbiased manner. Availability of various genome-wide datasets and high-resolution data of <em>cis</em> as well as <em>trans</em> chromatin interactions makes budding yeast an ideal candidate for multivariate analysis to identify the potential functional constraints shaping the non-random spatial organization of genome. The article by Duan et al.<sup><a reveal-id="DSW005C22" data-open="DSW005C22" class="link link-ref link-reveal xref-bibr">22</a></sup> suggested following key features of three-dimensional organization of budding yeast genome: (i) interactions among the centromeres, (ii) interactions among the sites of early origin and not the late origins, and (iii) interactions among t-RNA genes. A few follow-up studies suggested a link between chromatin interactions and co-expression of involved genes.<sup><a reveal-id="DSW005C37" data-open="DSW005C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSW005C38" data-open="DSW005C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Another report, on the contrary, dismissed the claims of proximity of co-expressed genes in yeast.<sup><a reveal-id="DSW005C39" data-open="DSW005C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Moreover, the possibility that <em>cis</em> and <em>trans</em> chromatin interactions might have been shaped under different evolutionary constraints has not been explored. In this study, using comprehensive statistical analysis, we show that functional and evolutionary constraints of <em>cis</em> and <em>trans</em> interactions are significantly distinct and are not necessarily associated with co-expression of genes. We show that the <em>trans</em> interactions are primarily constrained by coordinated replication through converged early origins, while <em>cis</em> interactions are shaped by coordination through late origins and by the minimization of expression noise of engaged genes in an evolutionarily conserved manner.</p>                            <h2 scrollto-destination=87200063 id="87200063" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200064 id="87200064" class="section-title" data-legacy-id=s2a>2.1. Data sources</h3>
<p>We obtained the publicly available genome-scale datasets from different sources; details of which are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h3 scrollto-destination=87200066 id="87200066" class="section-title" data-legacy-id=s2b>2.2. Methods</h3>
<p>Detailed methodology to process the datasets is given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>.</p>                            <h4 scrollto-destination=87200068 id="87200068" class="section-title" data-legacy-id=s2b1>2.2.1. Binning of data</h4>
<p>The interaction frequency data (frq) were clustered into bins of equal size of 1 unit, and average value of each functional attribute was calculated for each bin. The master tables for the binned and the original data are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Information</a></span>.</p>                            <h4 scrollto-destination=87200070 id="87200070" class="section-title" data-legacy-id=s2b2>2.2.2. Correlogram analyses</h4>
<p>Correlograms were plotted for the binned data using ‘corrgram’ R-package (<a class="link link-uri" href="http://cran.r-project.org/web/packages/corrgram/index.html">http://cran.r-project.org/web/packages/corrgram/index.html</a>). Pearson's correlation coefficients were calculated using <em>cor.test()</em> function in R and <em>P</em>-values for multiple comparisons (no. 55) were corrected using Bonferroni's method. Significant <em>P</em>-values after correction are marked with triple asterisk (***) in correlograms.</p>                            <h4 scrollto-destination=87200072 id="87200072" class="section-title" data-legacy-id=s2b3>2.2.3. Partial least squared regression</h4>
<div class="block-child-p">Partial least squared regression (PLSR) models the relation between input matrix <strong>X</strong> (<em>n</em> × <em>p</em> matrix with <em>n</em> dimensions of <em>p</em> input variables, genomic/functional attributes in this case) and response matrix <strong>Y</strong> (<em>n</em> × 1 matrix with <em>n</em> dimensions of 1 response variable, interaction frequency in this case) by decomposing them as following: <a id="DSW005M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSW005M1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="bold">X</mi></mrow><mo xmlns="">=</mo><mrow xmlns=""><mi mathvariant="bold">T</mi></mrow><mrow xmlns=""><msup><mrow><mi mathvariant="bold">P</mi></mrow><mrow><mi mathvariant="bold">T</mi></mrow></msup></mrow><mo xmlns="">+</mo><mrow xmlns=""><mi mathvariant="bold">E</mi></mrow></math></div></div><div class="label">(1)</div></div><a id="DSW005M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSW005M2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="bold">Y</mi></mrow><mo xmlns="">=</mo><mrow xmlns=""><mi mathvariant="bold">U</mi></mrow><mrow xmlns=""><msup><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi mathvariant="bold">T</mi></mrow></msup></mrow><mo xmlns="">+</mo><mrow xmlns=""><mi mathvariant="bold">F</mi></mrow></math></div></div><div class="label">(2)</div></div> Where <strong>T</strong> and <strong>U</strong> are <em>n</em> × <em>r</em> matrices with <em>r</em> extracted latent vectors (or scores). <strong>P</strong> and <strong>Q</strong> are <em>p</em> × <em>r</em> and 1 × <em>r</em> matrices of X and Y loadings, respectively. <strong>E</strong> and <strong>F</strong> are <em>n</em> × <em>p</em> and <em>n</em> × 1 matrices of residuals. In kernel PLS regression, following inner relation between T and U is assumed;<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="bold">U</mi></mrow><mo xmlns="">=</mo><mrow xmlns=""><mi mathvariant="bold">T</mi><mi mathvariant="bold">B</mi></mrow><mo xmlns="">+</mo><mrow xmlns=""><mi mathvariant="bold">H</mi></mrow></math></div></div></div> where <strong>B</strong> is the <em>r</em> × <em>r</em> diagonal matrix of regression coefficients and <strong>H</strong> is matrix of residuals. Accordingly, equation (2) can be rewritten as<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="bold">Y</mi></mrow><mo xmlns="">=</mo><mrow xmlns=""><mi mathvariant="bold">T</mi><mi mathvariant="bold">B</mi></mrow><mrow xmlns=""><msup><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi mathvariant="bold">T</mi></mrow></msup></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mi mathvariant="bold">H</mi></mrow><mrow xmlns=""><msup><mrow><mi mathvariant="bold">Q</mi></mrow><mrow><mi mathvariant="bold">T</mi></mrow></msup></mrow><mo xmlns="">+</mo><mrow xmlns=""><mi mathvariant="bold">F</mi></mrow><mo stretchy="false" xmlns="">)</mo></math></div></div></div> which defines the final PLS regression model,<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="bold">Y</mi></mrow><mo xmlns="">=</mo><mrow xmlns=""><mi mathvariant="bold">T</mi></mrow><mrow xmlns=""><msup><mrow><mi mathvariant="bold">C</mi></mrow><mrow><mi mathvariant="bold">T</mi></mrow></msup></mrow><mo xmlns="">+</mo><msup xmlns=""><mrow><mi mathvariant="bold">F</mi></mrow><mo>∗</mo></msup></math></div></div></div> where <strong>C<sup>T</sup></strong> = <strong>BQ<sup>T</sup></strong> and <strong>F*</strong> = <strong>HQ<sup>T</sup></strong> + <strong>F</strong></div><p>‘PLS’ R-package was used for this (<a class="link link-uri" href="http://cran.r-project.org/web/packages/pls/index.html">http://cran.r-project.org/web/packages/pls/index.html</a>). Prior to multivariate regression analyses, the columns of X were <em>Z</em>-normalized.</p>                            <h4 scrollto-destination=87200075 id="87200075" class="section-title" data-legacy-id=s2b4>2.2.4. Random re-sampling to compile null distributions</h4>
<div class="block-child-p">To assess the non-random connectivity among replication origins, we used re-sampling approach given by Witten and Noble.<sup><a reveal-id="DSW005C39" data-open="DSW005C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Briefly, the positions of the origins in the genome were randomized 10<sup>3</sup> times keeping the chromosomal distribution same as in the original set. P-value was calculated using following equation:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">p</mi><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mn>1</mn><mi>B</mi></mfrac></mrow><mfenced open="(" close=")"><mrow><mn>1</mn><mo>+</mo><msubsup><mo movablelimits="false">∑</mo><mrow><mi>b</mi><mo>=</mo><mn>1</mn></mrow><mi>B</mi></msubsup><mrow><mrow><msub><mrow><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mrow><mstyle displaystyle="true"><mrow><mfrac><mrow><mrow><msub><mi>k</mi><mi>b</mi></msub></mrow></mrow><mrow><mrow><msub><mrow><msup><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">′</mi></mrow></msup></mrow><mi>b</mi></msub></mrow></mrow></mfrac></mrow><mo>&gt;</mo><mstyle><mrow><mfrac><mi>k</mi><mrow><msup><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">′</mi></mrow></msup></mrow></mfrac></mrow></mstyle></mstyle></mrow></msub></mrow></mrow></mrow></mfenced></mstyle></math></div></div></div> Where, <em>B</em>= number of re-samplings (no. 1,000); <em>k<sub>b</sub></em> = Observed number of interactions among randomized coordinates; <em>k′<sub>b</sub></em> = Expected number of interactions among the randomized coordinates; <strong><em>k</em></strong> = Observed number of interactions among original coordinates. <em>k</em>′ = Expected number of interactions among original coordinates.</div><p>We mapped the feature coordinates onto restriction fragments and considered a restriction fragment only once regardless of how many feature coordinates fall inside. This takes care of local clustering of feature coordinates.</p>                            <h2 scrollto-destination=87200078 id="87200078" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87200079 id="87200079" class="section-title" data-legacy-id=s3a>3.1. Distinct functional attributes of <em>cis</em> and <em>trans</em> chromatin interactions in yeast</h3>
<p>Chromatin in the interphase nucleus is generally present in the form of chromosomal territories. The chromatin loci embedded inside the territory would, therefore, be expected to have greater number of <em>cis</em> interactions, while the ones near territorial edge would have abundant <em>trans</em> interactions. This is particularly true for the metazoan genomes; the distinction into such intra- and inter-territorial organization of genes is not well studied in yeast however. If yeast genome conforms into distinct territories, certain regions of chromosomes would exhibit greater number of either <em>cis</em> or <em>trans</em> interactions marking intra-territorial and inter-territorial locations of the loci. Therefore, we first tested whether the number of <em>cis</em> interactions of each locus correlated with the <em>trans</em> interactions of corresponding locus in the yeast genome. We smoothened the number of <em>cis</em> and <em>trans</em> interactions using an arbitrary window of 20 kb along the chromosomes and plotted number of <em>cis</em> vs<em>. trans</em> interactions. We observed that the number of <em>cis</em> and <em>trans</em> interactions per locus poorly correlated with each other (<em>ρ</em>=0.05) and that there were regions, which were enriched either with <em>cis</em> or <em>trans</em> interactions (highlighted in ellipses in Fig. <a reveal-id="DSW005F1" data-open="DSW005F1" class="link link-reveal link-table xref-fig">1</a>a). The observed distinction of <em>cis</em> and <em>trans</em> interactions was also confirmed by plotting the <em>Z</em>-normalized number of <em>cis</em> and <em>trans</em> interactions side-by-side as a function of chromosomal coordinates. As shown in the Fig. <a reveal-id="DSW005F1" data-open="DSW005F1" class="link link-reveal link-table xref-fig">1</a>b, there were domains in the genome having greater number of either <em>cis</em> or <em>trans</em> chromatin interactions. We further calculated average frequency of all the <em>cis</em> and all the <em>trans</em> interactions of each genomic locus independently. By plotting average <em>cis</em> and <em>trans</em> interaction frequencies of corresponding genomic loci against each other, we once again observed regions having higher <em>cis</em> interaction frequency and relatively lower <em>trans</em> frequency and vice versa supporting that there were regions in the genome having high frequency of either <em>cis</em> or <em>trans</em> interactions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). We further showed an example of chromosome 4 in the rendered 3D model of yeast genome (as provided by Duan et al.) in Fig. <a reveal-id="DSW005F1" data-open="DSW005F1" class="link link-reveal link-table xref-fig">1</a>c, highlighting a region enriched with local clusters of abundant <em>cis</em> interactions (dashed box). Rest of the chromosome 4 could be seen as an extended arm extensively intermingling with other chromosomes. Quantitative data for Fig. <a reveal-id="DSW005F1" data-open="DSW005F1" class="link link-reveal link-table xref-fig">1</a>c were shown in the Fig. <a reveal-id="DSW005F1" data-open="DSW005F1" class="link link-reveal link-table xref-fig">1</a>d. The plot clearly showed greater number of <em>cis</em> interactions in the boxed area, which corresponds to the same boxed area as in 3D model. Our observations through Fig. <a reveal-id="DSW005F1" data-open="DSW005F1" class="link link-reveal link-table xref-fig">1</a> highlighted that certain genomic domains can have preferred folding in <em>cis</em> while others intermingles extensively with other chromosomes in <em>trans</em>, supporting the notion of ‘chromosome territories’. This also raised a possibility of <em>cis</em> and <em>trans</em> interactomes to have evolved under distinct evolutionary constraints. We, therefore, attempted to delineate the functional attributes that explained best the observed distinction of <em>cis</em> and <em>trans</em> chromatin interactions by analysing the datasets of <em>cis</em> and <em>trans</em> chromatin interactions independently. </p>                        <a id="87200081" scrollto-destination="87200081"></a>
<div id="DSW005F1" data-id="DSW005F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00501.jpeg?Expires=2147483647&amp;Signature=Wf3-Zk1tEOTrL6j0BJfbOAxPXugp8hcW4f6uv5lkbzkKwmjO6gcIyqtX5289rri1XEM0wdSRnkptWvqnkvJSJTcdJwf4bCsPETVA1YmQk-hmZbWqrkjwryCqX2BCaBRKdgEH4iMb5qrlJpuDXDI1b7HxqGt0epLy3sH-MDpzkYmkKr9WQCv~VEiRdNImQG1vnCwHoT8P1Eu1jGDuLNc~ePve~ER7ZfTxlPKyLrqW3AATRL~C5zzharKB7kBS1SRYIwNpKbpDYnmjOf5ZVbdGfsgLaTwaoZMkb9gRY1CYoX6Ka~HIbAbYbCMBIYo9WOE8iC~-BRAzJTFxwb-dpkUWBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Preferred domains of cis and trans chromatin interactions. (a) Scatter plot of number of cis and trans interactions of each locus in yeast genome. X–Y coordinates were binned into 100 bins and represented as shade intensity. (b) Linear view of yeast genome (chr1–16 are concatenated in that order) depicting Z-normalized number of cis and trans chromatin interactions of each locus. Thicker lines represent moving average of 10 values. (c) 3D model of yeast genome, as provided by Duan et al., rendered using VMD software. As an example, a region rich in cis interactions on chromosome 4 is highlighted in dashed box. (d) Z-normalized number of cis and trans interactions of each locus on chromosome 4 as a function of chromosomal coordinates. Boxed area highlights the same cis-rich region as in (c). Data plotted in a, b, and d are given in the Supplementary Table S2. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200081/dsw00501.jpeg" path-from-xml="dsw00501.jpeg" target="_blank">View large</a><a section="87200081" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00501.jpeg?Expires=2147483647&amp;Signature=xzAIe3aCaGXuCaz~iiO8I~InOdhod6OsxOXEVYqW6yooULE~6B2zAr7~pv2RYCK4IvPVsRNoZtHf2OcQiKouVuoEysqg~2jJ9gf1M3zqjs1zhacIf2R2qE74t1InX9CcFhplBLM4Ws1yuwyrfe0ucUvoCTqX386LVOzyrbXgO19QZPvWxLSkIEQ4YLs0HCu7DuChjS-fMWmRjxr9VLsllAuV2FCGyT1hUaJOmcZdmzcfLSQhK7yA~xbPJ0mnQXyVC44RreGdiUrMz5Rcp8vkJGZahb8a5Jk9EzV4SpY-m1XdBh9qHQNVelxtwCiWU-Jtj15mQht1s1oEyRd-T4TgFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200081&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Preferred domains of <em>cis</em> and <em>trans</em> chromatin interactions. (a) Scatter plot of number of <em>cis</em> and <em>trans</em> interactions of each locus in yeast genome. X–Y coordinates were binned into 100 bins and represented as shade intensity. (b) Linear view of yeast genome (chr1–16 are concatenated in that order) depicting <em>Z</em>-normalized number of <em>cis</em> and <em>trans</em> chromatin interactions of each locus. Thicker lines represent moving average of 10 values. (c) 3D model of yeast genome, as provided by Duan et al., rendered using VMD software. As an example, a region rich in <em>cis</em> interactions on chromosome 4 is highlighted in dashed box. (d) <em>Z</em>-normalized number of <em>cis</em> and <em>trans</em> interactions of each locus on chromosome 4 as a function of chromosomal coordinates. Boxed area highlights the same <em>cis</em>-rich region as in (c). Data plotted in a, b, and d are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00501.jpeg?Expires=2147483647&amp;Signature=Wf3-Zk1tEOTrL6j0BJfbOAxPXugp8hcW4f6uv5lkbzkKwmjO6gcIyqtX5289rri1XEM0wdSRnkptWvqnkvJSJTcdJwf4bCsPETVA1YmQk-hmZbWqrkjwryCqX2BCaBRKdgEH4iMb5qrlJpuDXDI1b7HxqGt0epLy3sH-MDpzkYmkKr9WQCv~VEiRdNImQG1vnCwHoT8P1Eu1jGDuLNc~ePve~ER7ZfTxlPKyLrqW3AATRL~C5zzharKB7kBS1SRYIwNpKbpDYnmjOf5ZVbdGfsgLaTwaoZMkb9gRY1CYoX6Ka~HIbAbYbCMBIYo9WOE8iC~-BRAzJTFxwb-dpkUWBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Preferred domains of cis and trans chromatin interactions. (a) Scatter plot of number of cis and trans interactions of each locus in yeast genome. X–Y coordinates were binned into 100 bins and represented as shade intensity. (b) Linear view of yeast genome (chr1–16 are concatenated in that order) depicting Z-normalized number of cis and trans chromatin interactions of each locus. Thicker lines represent moving average of 10 values. (c) 3D model of yeast genome, as provided by Duan et al., rendered using VMD software. As an example, a region rich in cis interactions on chromosome 4 is highlighted in dashed box. (d) Z-normalized number of cis and trans interactions of each locus on chromosome 4 as a function of chromosomal coordinates. Boxed area highlights the same cis-rich region as in (c). Data plotted in a, b, and d are given in the Supplementary Table S2. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200081/dsw00501.jpeg" path-from-xml="dsw00501.jpeg" target="_blank">View large</a><a section="87200081" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00501.jpeg?Expires=2147483647&amp;Signature=xzAIe3aCaGXuCaz~iiO8I~InOdhod6OsxOXEVYqW6yooULE~6B2zAr7~pv2RYCK4IvPVsRNoZtHf2OcQiKouVuoEysqg~2jJ9gf1M3zqjs1zhacIf2R2qE74t1InX9CcFhplBLM4Ws1yuwyrfe0ucUvoCTqX386LVOzyrbXgO19QZPvWxLSkIEQ4YLs0HCu7DuChjS-fMWmRjxr9VLsllAuV2FCGyT1hUaJOmcZdmzcfLSQhK7yA~xbPJ0mnQXyVC44RreGdiUrMz5Rcp8vkJGZahb8a5Jk9EzV4SpY-m1XdBh9qHQNVelxtwCiWU-Jtj15mQht1s1oEyRd-T4TgFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200081&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Preferred domains of <em>cis</em> and <em>trans</em> chromatin interactions. (a) Scatter plot of number of <em>cis</em> and <em>trans</em> interactions of each locus in yeast genome. X–Y coordinates were binned into 100 bins and represented as shade intensity. (b) Linear view of yeast genome (chr1–16 are concatenated in that order) depicting <em>Z</em>-normalized number of <em>cis</em> and <em>trans</em> chromatin interactions of each locus. Thicker lines represent moving average of 10 values. (c) 3D model of yeast genome, as provided by Duan et al., rendered using VMD software. As an example, a region rich in <em>cis</em> interactions on chromosome 4 is highlighted in dashed box. (d) <em>Z</em>-normalized number of <em>cis</em> and <em>trans</em> interactions of each locus on chromosome 4 as a function of chromosomal coordinates. Boxed area highlights the same <em>cis</em>-rich region as in (c). Data plotted in a, b, and d are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Multi-dimensional genomic datasets, i.e. data where measurements were done for all the yeast genes across several different time-points or conditions, were used to calculate similarity (mostly Pearson's correlation coefficient; Fig. <a reveal-id="DSW005F2" data-open="DSW005F2" class="link link-reveal link-table xref-fig">2</a>; Table <a reveal-id="DSW005TB1" data-open="DSW005TB1" class="link link-reveal link-table xref-fig">1</a>; Materials and Methods) between interacting genes. Analysis of correlations among several functional attributes suggested that the frequency of <em>trans</em> interactions was best correlated with the similarity in % replicated DNA or co-replication (‘rep’, Table <a reveal-id="DSW005TB1" data-open="DSW005TB1" class="link link-reveal link-table xref-fig">1</a>) of interacting loci (<em>ρ</em>=0.47, <em>P</em>-value &lt; 10<sup>−4</sup>, Fig. <a reveal-id="DSW005F3" data-open="DSW005F3" class="link link-reveal link-table xref-fig">3</a>a). Interestingly, co-expression of genes (i) through cell cycle (cct), (ii) following transcription factor perturbation (tfp), and (iii) following environmental perturbations did not show strong correlation with the <em>trans</em> interaction frequency as claimed elsewhere<sup><a reveal-id="DSW005C38" data-open="DSW005C38" class="link link-ref link-reveal xref-bibr">38</a></sup> (<em>ρ</em>=0.07, 0.09, and 0.13, respectively; Fig. <a reveal-id="DSW005F3" data-open="DSW005F3" class="link link-reveal link-table xref-fig">3</a>a, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). On the contrary, <em>cis</em> interactions among loci, which were at least 20 kb apart, did not exhibit strong correlation with co-replication (<em>ρ</em>= 0.14; <em>P</em>-value &lt; 10<sup>−3</sup>; FDR &gt; 0.01) and were, instead, strongly correlated with the co-fitness of interacting genes as measured through chemical genomic screens (<em>ρ</em>= 0.25; <em>P</em>-value &lt; 10<sup>−4</sup>; FDR &lt; 0.01; Fig. <a reveal-id="DSW005F3" data-open="DSW005F3" class="link link-reveal link-table xref-fig">3</a>b). Again, the general cell-cycle-related co-expression of genes showed weak correlation, though slightly higher compared with the <em>trans</em> interactions (<em>ρ</em> = 0.11 vs. 0.07). To further scrutinize the significance of observed correlations, we rewired the chromatin interactions using the strategy given in the ‘Materials and methods’ section to obtain null distribution and calculate <em>P</em>-values (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Examples of correlograms generated from rewired <em>cis</em> and <em>trans</em> interactions were shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>a and b, and their comparisons with observed values of correlations were drawn in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>c and d. In particular, we confirmed that (i) the correlation of <em>trans</em> interaction frequency with the coordinated replication and that of <em>cis</em> interaction frequency with the co-fitness of interacting genes were (i) significant compared with rewired control (<em>P</em>-value = 1.0e–09 and 3.3e–09, respectively, FDR &lt; 0.001 for both comparisons, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>a–e); and (ii) significantly different for <em>cis</em> and <em>trans</em> interactions, i.e. correlation with ‘rep’ was significantly higher for <em>trans</em> (<em>P</em>-value = 3.1e–12) and lower for <em>cis</em>, while correlation with co-fitness was significantly higher for <em>cis</em> (<em>P</em>-value = 8.5e–08) and lower for <em>trans</em> interactions. We also confirmed these observations for the chromatin interactions that were concomitantly captured in the HiC-library generated using <em>Eco</em>RI restriction enzyme, highlighting the robustness of the analyses (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>a and b). We further cross-validated the differential association of <em>cis</em> and <em>trans</em> interaction frequencies with the coordinated replication (rep) and co-fitness (cof) of engaged genes using recently published HiC dataset of exponentially growing budding yeast cells<sup><a reveal-id="DSW005C40" data-open="DSW005C40" class="link link-ref link-reveal xref-bibr">40</a></sup> (Fig. <a reveal-id="DSW005F3" data-open="DSW005F3" class="link link-reveal link-table xref-fig">3</a>c). </p>                        <a id="87200083" scrollto-destination="87200083"></a>
<div content-id="DSW005TB1" class="table-modal"><div class="table-wrap"><div id="DSW005TB1" class="table-wrap-title" data-id="DSW005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Abbreviations used for different kind of functional similarities between genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Abbreviations </th><th>Attribute </th><th>Similarity </th><th>No. in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> </th></tr></thead><tbody><tr><td>frq </td><td>Interaction frequency </td><td><em>cis</em> and <em>trans</em> chromatin interaction frequencies between genes were taken from Duan et al. and normalized using HiCNorm package </td><td>1 </td></tr><tr><td>cct </td><td>Cell cycle time course </td><td>Pearson correlation between time course expression values of interacting genes </td><td>2 </td></tr><tr><td>cfp </td><td>Chromatin factor perturbation </td><td>Pearson correlation between expression values of interacting genes across chromatin factor mutant strains </td><td>3 </td></tr><tr><td>tfp </td><td>Transcription factor perturbation </td><td>Pearson correlation between expression values of interacting genes across transcription factor mutant strains </td><td>4 </td></tr><tr><td>env </td><td>Environmental response </td><td>Pearson's correlation between expression values of interacting genes across different environmental conditions </td><td>5 </td></tr><tr><td>ace </td><td>Acetylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone acetylations </td><td>7 </td></tr><tr><td>met </td><td>Methylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone methylations </td><td>8 </td></tr><tr><td>rep </td><td>Co-replication </td><td>Pearson's correlation between % replication DNA of 500 bp genomic bins, as provided by the authors, mapping to interacting pairs of genes </td><td>9 </td></tr><tr><td>cof </td><td>Co-fitness </td><td>Co-fitness of interacting genes were directly taken from this study </td><td>11 </td></tr><tr><td>ppi </td><td>Protein–protein interaction </td><td>Socio-Affinity (SA) index, which measures the log-odds of observed number of times two proteins interact relative to the expected value deduced from their frequency in the dataset, were obtained for each protein-protein interaction </td><td>14 </td></tr><tr><td>fsm </td><td>Functional similarity </td><td>Functional similarity (fsm) was calculated using ‘GOSemSim’ R-package. </td><td>16 </td></tr><tr><td>erc </td><td>Evolutionary rate co-variation </td><td>Evolutionary rate co-variation (erc) was taken as it is from the source </td><td>17 </td></tr></tbody></table></div><div content-id="DSW005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Abbreviations </th><th>Attribute </th><th>Similarity </th><th>No. in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> </th></tr></thead><tbody><tr><td>frq </td><td>Interaction frequency </td><td><em>cis</em> and <em>trans</em> chromatin interaction frequencies between genes were taken from Duan et al. and normalized using HiCNorm package </td><td>1 </td></tr><tr><td>cct </td><td>Cell cycle time course </td><td>Pearson correlation between time course expression values of interacting genes </td><td>2 </td></tr><tr><td>cfp </td><td>Chromatin factor perturbation </td><td>Pearson correlation between expression values of interacting genes across chromatin factor mutant strains </td><td>3 </td></tr><tr><td>tfp </td><td>Transcription factor perturbation </td><td>Pearson correlation between expression values of interacting genes across transcription factor mutant strains </td><td>4 </td></tr><tr><td>env </td><td>Environmental response </td><td>Pearson's correlation between expression values of interacting genes across different environmental conditions </td><td>5 </td></tr><tr><td>ace </td><td>Acetylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone acetylations </td><td>7 </td></tr><tr><td>met </td><td>Methylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone methylations </td><td>8 </td></tr><tr><td>rep </td><td>Co-replication </td><td>Pearson's correlation between % replication DNA of 500 bp genomic bins, as provided by the authors, mapping to interacting pairs of genes </td><td>9 </td></tr><tr><td>cof </td><td>Co-fitness </td><td>Co-fitness of interacting genes were directly taken from this study </td><td>11 </td></tr><tr><td>ppi </td><td>Protein–protein interaction </td><td>Socio-Affinity (SA) index, which measures the log-odds of observed number of times two proteins interact relative to the expected value deduced from their frequency in the dataset, were obtained for each protein-protein interaction </td><td>14 </td></tr><tr><td>fsm </td><td>Functional similarity </td><td>Functional similarity (fsm) was calculated using ‘GOSemSim’ R-package. </td><td>16 </td></tr><tr><td>erc </td><td>Evolutionary rate co-variation </td><td>Evolutionary rate co-variation (erc) was taken as it is from the source </td><td>17 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200083" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW005TB1" class="table-wrap-title" data-id="DSW005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Abbreviations used for different kind of functional similarities between genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Abbreviations </th><th>Attribute </th><th>Similarity </th><th>No. in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> </th></tr></thead><tbody><tr><td>frq </td><td>Interaction frequency </td><td><em>cis</em> and <em>trans</em> chromatin interaction frequencies between genes were taken from Duan et al. and normalized using HiCNorm package </td><td>1 </td></tr><tr><td>cct </td><td>Cell cycle time course </td><td>Pearson correlation between time course expression values of interacting genes </td><td>2 </td></tr><tr><td>cfp </td><td>Chromatin factor perturbation </td><td>Pearson correlation between expression values of interacting genes across chromatin factor mutant strains </td><td>3 </td></tr><tr><td>tfp </td><td>Transcription factor perturbation </td><td>Pearson correlation between expression values of interacting genes across transcription factor mutant strains </td><td>4 </td></tr><tr><td>env </td><td>Environmental response </td><td>Pearson's correlation between expression values of interacting genes across different environmental conditions </td><td>5 </td></tr><tr><td>ace </td><td>Acetylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone acetylations </td><td>7 </td></tr><tr><td>met </td><td>Methylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone methylations </td><td>8 </td></tr><tr><td>rep </td><td>Co-replication </td><td>Pearson's correlation between % replication DNA of 500 bp genomic bins, as provided by the authors, mapping to interacting pairs of genes </td><td>9 </td></tr><tr><td>cof </td><td>Co-fitness </td><td>Co-fitness of interacting genes were directly taken from this study </td><td>11 </td></tr><tr><td>ppi </td><td>Protein–protein interaction </td><td>Socio-Affinity (SA) index, which measures the log-odds of observed number of times two proteins interact relative to the expected value deduced from their frequency in the dataset, were obtained for each protein-protein interaction </td><td>14 </td></tr><tr><td>fsm </td><td>Functional similarity </td><td>Functional similarity (fsm) was calculated using ‘GOSemSim’ R-package. </td><td>16 </td></tr><tr><td>erc </td><td>Evolutionary rate co-variation </td><td>Evolutionary rate co-variation (erc) was taken as it is from the source </td><td>17 </td></tr></tbody></table></div><div content-id="DSW005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Abbreviations </th><th>Attribute </th><th>Similarity </th><th>No. in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> </th></tr></thead><tbody><tr><td>frq </td><td>Interaction frequency </td><td><em>cis</em> and <em>trans</em> chromatin interaction frequencies between genes were taken from Duan et al. and normalized using HiCNorm package </td><td>1 </td></tr><tr><td>cct </td><td>Cell cycle time course </td><td>Pearson correlation between time course expression values of interacting genes </td><td>2 </td></tr><tr><td>cfp </td><td>Chromatin factor perturbation </td><td>Pearson correlation between expression values of interacting genes across chromatin factor mutant strains </td><td>3 </td></tr><tr><td>tfp </td><td>Transcription factor perturbation </td><td>Pearson correlation between expression values of interacting genes across transcription factor mutant strains </td><td>4 </td></tr><tr><td>env </td><td>Environmental response </td><td>Pearson's correlation between expression values of interacting genes across different environmental conditions </td><td>5 </td></tr><tr><td>ace </td><td>Acetylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone acetylations </td><td>7 </td></tr><tr><td>met </td><td>Methylation </td><td>Pearson's correlation between ChIP enrichment values of interacting gene promoters across different histone methylations </td><td>8 </td></tr><tr><td>rep </td><td>Co-replication </td><td>Pearson's correlation between % replication DNA of 500 bp genomic bins, as provided by the authors, mapping to interacting pairs of genes </td><td>9 </td></tr><tr><td>cof </td><td>Co-fitness </td><td>Co-fitness of interacting genes were directly taken from this study </td><td>11 </td></tr><tr><td>ppi </td><td>Protein–protein interaction </td><td>Socio-Affinity (SA) index, which measures the log-odds of observed number of times two proteins interact relative to the expected value deduced from their frequency in the dataset, were obtained for each protein-protein interaction </td><td>14 </td></tr><tr><td>fsm </td><td>Functional similarity </td><td>Functional similarity (fsm) was calculated using ‘GOSemSim’ R-package. </td><td>16 </td></tr><tr><td>erc </td><td>Evolutionary rate co-variation </td><td>Evolutionary rate co-variation (erc) was taken as it is from the source </td><td>17 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200083" target="_blank">View Large</a></div></div>                        <a id="87200084" scrollto-destination="87200084"></a>
<div id="DSW005F2" data-id="DSW005F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00502.jpeg?Expires=2147483647&Signature=hYb9r6T9y0cC~YUAfmyM95iS413hOLhIX8GBMA4SQq158oRFROnDF-MCHJrUj9PgkdfA5Pz-QrFMl~5Vf~ftnWabxFk50w5VLfa1SB4YhchiL6iItax85hnPXEJCwjTjyt50kTWqbmpeVifeaRlJjBonT5mPhTd9QiR32aUpjdBtVNuzBjKMXli5Mh5EMh5bIJhj3cfWhaZL0EtUsyFCajtC2s9xjVnd9IwZ9U7VlnxzF3vxqLTNG7qhwMjBOWLdOMNcyVSW4-F0Ur5aZJ6XCEe~God2ozk3PZBOLcg~vsumZBCoM1dFonAv46CUvB9EY5MrMh2ojcpb~oEx8e6Kgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of overall analysis. Pairs of interacting genes were obtained from chromatin interaction data and split into cis and trans pairs. For each interacting pair of genes, epigenetic/functional similarity was calculated, mostly by calculating Pearson's correlation between attribute profiles across conditions or time points. Vectors of cis and trans interaction frequencies were compiled and binned into equal size of 1 unit change in interaction frequency. The functional similarity scores were accordingly averaged for each bin. The resulting interaction frequency and functional similarity matrices (Y and X) were than subjected to correlation and PLSR analysis. The functional attributes that exhibited best association in the multivariate analyses were subjected to comprehensive follow-up analyses. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200084/dsw00502.jpeg" path-from-xml="dsw00502.jpeg" target="_blank">View large</a><a section="87200084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00502.jpeg?Expires=2147483647&Signature=F23wzpEGSIyBnLZN~0erDgNJ2mc2jkOK3anKkSPJbNhUyi2iVaT6tdiFb6DQp~6DUWk39Sea6515XFXKnC7P0bWPk8BI-ACWGxmooivzt6GmUiasXuLHzUhVGIcPMcmLxPeepFK4FBSbPvpMERWm8C15VezxRnpiYvP8afGF4zfIjvplxVHa6OWl0Ow86G6ijiQgSjqdcBOA-qJ1KL3lE9ZpmUP1yOfLUmCQEiTAqoz5JzEqA6o-LxQ4HXgyBdgOPidR9Kje9IKmYsCWUlxQUr9ob2tg8gZUUs10I41qe~5KRqMDG9QW5ygT1Hw-lkY7VCgbsf7TIKUAuXpNlcNznA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200084&ar=1745431&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of overall analysis. Pairs of interacting genes were obtained from chromatin interaction data and split into <em>cis</em> and <em>trans</em> pairs. For each interacting pair of genes, epigenetic/functional similarity was calculated, mostly by calculating Pearson's correlation between attribute profiles across conditions or time points. Vectors of <em>cis</em> and <em>trans</em> interaction frequencies were compiled and binned into equal size of 1 unit change in interaction frequency. The functional similarity scores were accordingly averaged for each bin. The resulting interaction frequency and functional similarity matrices (Y and X) were than subjected to correlation and PLSR analysis. The functional attributes that exhibited best association in the multivariate analyses were subjected to comprehensive follow-up analyses. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00502.jpeg?Expires=2147483647&Signature=hYb9r6T9y0cC~YUAfmyM95iS413hOLhIX8GBMA4SQq158oRFROnDF-MCHJrUj9PgkdfA5Pz-QrFMl~5Vf~ftnWabxFk50w5VLfa1SB4YhchiL6iItax85hnPXEJCwjTjyt50kTWqbmpeVifeaRlJjBonT5mPhTd9QiR32aUpjdBtVNuzBjKMXli5Mh5EMh5bIJhj3cfWhaZL0EtUsyFCajtC2s9xjVnd9IwZ9U7VlnxzF3vxqLTNG7qhwMjBOWLdOMNcyVSW4-F0Ur5aZJ6XCEe~God2ozk3PZBOLcg~vsumZBCoM1dFonAv46CUvB9EY5MrMh2ojcpb~oEx8e6Kgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of overall analysis. Pairs of interacting genes were obtained from chromatin interaction data and split into cis and trans pairs. For each interacting pair of genes, epigenetic/functional similarity was calculated, mostly by calculating Pearson's correlation between attribute profiles across conditions or time points. Vectors of cis and trans interaction frequencies were compiled and binned into equal size of 1 unit change in interaction frequency. The functional similarity scores were accordingly averaged for each bin. The resulting interaction frequency and functional similarity matrices (Y and X) were than subjected to correlation and PLSR analysis. The functional attributes that exhibited best association in the multivariate analyses were subjected to comprehensive follow-up analyses. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200084/dsw00502.jpeg" path-from-xml="dsw00502.jpeg" target="_blank">View large</a><a section="87200084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00502.jpeg?Expires=2147483647&Signature=F23wzpEGSIyBnLZN~0erDgNJ2mc2jkOK3anKkSPJbNhUyi2iVaT6tdiFb6DQp~6DUWk39Sea6515XFXKnC7P0bWPk8BI-ACWGxmooivzt6GmUiasXuLHzUhVGIcPMcmLxPeepFK4FBSbPvpMERWm8C15VezxRnpiYvP8afGF4zfIjvplxVHa6OWl0Ow86G6ijiQgSjqdcBOA-qJ1KL3lE9ZpmUP1yOfLUmCQEiTAqoz5JzEqA6o-LxQ4HXgyBdgOPidR9Kje9IKmYsCWUlxQUr9ob2tg8gZUUs10I41qe~5KRqMDG9QW5ygT1Hw-lkY7VCgbsf7TIKUAuXpNlcNznA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200084&ar=1745431&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of overall analysis. Pairs of interacting genes were obtained from chromatin interaction data and split into <em>cis</em> and <em>trans</em> pairs. For each interacting pair of genes, epigenetic/functional similarity was calculated, mostly by calculating Pearson's correlation between attribute profiles across conditions or time points. Vectors of <em>cis</em> and <em>trans</em> interaction frequencies were compiled and binned into equal size of 1 unit change in interaction frequency. The functional similarity scores were accordingly averaged for each bin. The resulting interaction frequency and functional similarity matrices (Y and X) were than subjected to correlation and PLSR analysis. The functional attributes that exhibited best association in the multivariate analyses were subjected to comprehensive follow-up analyses. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="87200085" scrollto-destination="87200085"></a>
<div id="DSW005F3" data-id="DSW005F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00503.jpeg?Expires=2147483647&amp;Signature=JH88WKf4Y-ldXPf8fz8JcG-moY0pF0N-9mTBZ2Yr2-tFgFVc7xtTybccRKprzJbxtpCGvB51avBzjvGC2ZdfIqndhJzTXvRHNZojqfT09T8IZG9xiniZ92GS-xNB9DNeXjbQao~Y130QYcR-5HD0b7DRw58TZdvJclk3xmjFUGU8k8kWPaKCiXWnc2oVvl8SgHNaOau9oimSfFmVUBW7CLw86THV-zuiAf-8o3osdHahNZPjtC9rpOPJueANQxrSS9PHxCbfvQxodNgk-5FF0Qrh6qQfvpHA8HzLyessLQ6mr-H5BgZtjX34p8eoT-QKt7Rh6u~Om49sYA1VBb~J6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multivariate analyses of trans and cis chromatin interactions in yeast genome. (a and b) Correlograms of distinct functional attributes of trans- and cis-interacting genes. The upper triangle shows the heatmap and the correlation values for each comparison. Lower triangle is a pie chart representation of Pearson's correlation coefficients. Triple asterisks (***) indicate P-value &lt; 10−4 that corresponds to FDR &lt; 0.01 for multiple comparisons of all 55 correlations between chromatin interaction frequency and other functional variables. Double asterisks indicate P-value &lt; 10−3, and single asterisk indicates P-value &lt; 10−2. Plots are made using ‘corrgram’ package on R. Frq, interaction frequency; cof, co-fitness; ppi, protein–protein interaction; cct, cell cycle time course; tfp, transcription factor perturbation; ace, acetylation; met, methylation; erc, evolutionary rate covariation; cfp, chromatin factor perturbation; fsm, functional similarity; rep, replication. Data in correlograms are ordered as per PCA clustering. ‘ρ’ stands for Pearson's correlation coefficient. Detailed control analyses are given in Supplementary Fig. S3. Details of three-letter abbreviations are given in the Table 1. (c) Cross-validation of differential correlations of cis and trans interaction frequencies with ‘rep‘ and ‘cof‘ through an independent HiC data of exponentially growing budding yeast cells (Rutledge et al.40). ‘ρ’ stands for Pearson's correlation coefficient. (d and e) Bi-plot of leading two X-loading vectors p1 and p2 (columns of matrix P) of PLSR model for (d) trans- and (e) cis-interacting gene pairs. The arrows are shaded as per p1 loading. Loading vectors p1 and p2 correspond to leading latent vectors t1 and t2 of matrix T, which explain 32.3 and 5.6% variance of Y (n× 1 matrix of interaction frequency) for trans interactions and 36.9 and 3.96% variance of cis interaction frequency, respectively. The datasets plotted in a and b are given in the Supplementary Tables S3 and S4. The loadings for all the components are given in the Supplementary Fig. S5. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200085/dsw00503.jpeg" path-from-xml="dsw00503.jpeg" target="_blank">View large</a><a section="87200085" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00503.jpeg?Expires=2147483647&amp;Signature=saGkVtx8qiANnS04Ld~xaZAPR5Frloea7nMPzAvmozhuUX0YJobKpzD3otWXNpjy8xM~IYJZz5j7kzjhfZoiQpqfu-GETgV5w~3DXjj-CDZzCHAGizSXEItfu8aRvf-TEYx7vwGwKsJ3oy0YrAsTtLtlDsN0pm8mN5ZYDRo7F3msXyOOmM-8gQgWJc3ymeHxTjcbOMWWKJJoCyJm0nnrWmgE7VEE7aoARVzZh7A2Dj80qUZQLxW-7-XI27FVHXcyy2rm6XpHza0QKRGYtnRyT-Ui8Tfs7Kg0RINEsMbjECvW4OtdVi6K4ENNCESHAB~0ZdqKepuSw7-uX4zD4xu-VQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200085&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multivariate analyses of <em>trans</em> and <em>cis</em> chromatin interactions in yeast genome. (a and b) Correlograms of distinct functional attributes of <em>trans-</em> and <em>cis-</em>interacting genes. The upper triangle shows the heatmap and the correlation values for each comparison. Lower triangle is a pie chart representation of Pearson's correlation coefficients. Triple asterisks (***) indicate <em>P</em>-value &lt; 10<sup>−4</sup> that corresponds to FDR &lt; 0.01 for multiple comparisons of all 55 correlations between chromatin interaction frequency and other functional variables. Double asterisks indicate <em>P</em>-value &lt; 10<sup>−3</sup>, and single asterisk indicates <em>P</em>-value &lt; 10<sup>−2</sup>. Plots are made using ‘corrgram’ package on R. Frq, interaction frequency; cof, co-fitness; ppi, protein–protein interaction; cct, cell cycle time course; tfp, transcription factor perturbation; ace, acetylation; met, methylation; erc, evolutionary rate covariation; cfp, chromatin factor perturbation; fsm, functional similarity; rep, replication. Data in correlograms are ordered as per PCA clustering. ‘<em>ρ</em>’ stands for Pearson's correlation coefficient. Detailed control analyses are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>. Details of three-letter abbreviations are given in the Table <a reveal-id="DSW005TB1" data-open="DSW005TB1" class="link link-reveal link-table xref-fig">1</a>. (c) Cross-validation of differential correlations of <em>cis</em> and <em>tran</em>s interaction frequencies with ‘rep‘ and ‘cof‘ through an independent HiC data of exponentially growing budding yeast cells (Rutledge et al.<sup><a reveal-id="DSW005C40" data-open="DSW005C40" class="link link-ref link-reveal xref-bibr">40</a></sup>). ‘<em>ρ</em>’ stands for Pearson's correlation coefficient. (d and e) Bi-plot of leading two X-loading vectors p1 and p2 (columns of matrix P) of PLSR model for (d) <em>trans-</em> and (e) <em>cis</em>-interacting gene pairs. The arrows are shaded as per p1 loading. Loading vectors p1 and p2 correspond to leading latent vectors t1 and t2 of matrix T, which explain 32.3 and 5.6% variance of Y (<em>n</em>× 1 matrix of interaction frequency) for <em>trans</em> interactions and 36.9 and 3.96% variance of <em>cis</em> interaction frequency, respectively. The datasets plotted in a and b are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>. The loadings for all the components are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00503.jpeg?Expires=2147483647&amp;Signature=JH88WKf4Y-ldXPf8fz8JcG-moY0pF0N-9mTBZ2Yr2-tFgFVc7xtTybccRKprzJbxtpCGvB51avBzjvGC2ZdfIqndhJzTXvRHNZojqfT09T8IZG9xiniZ92GS-xNB9DNeXjbQao~Y130QYcR-5HD0b7DRw58TZdvJclk3xmjFUGU8k8kWPaKCiXWnc2oVvl8SgHNaOau9oimSfFmVUBW7CLw86THV-zuiAf-8o3osdHahNZPjtC9rpOPJueANQxrSS9PHxCbfvQxodNgk-5FF0Qrh6qQfvpHA8HzLyessLQ6mr-H5BgZtjX34p8eoT-QKt7Rh6u~Om49sYA1VBb~J6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multivariate analyses of trans and cis chromatin interactions in yeast genome. (a and b) Correlograms of distinct functional attributes of trans- and cis-interacting genes. The upper triangle shows the heatmap and the correlation values for each comparison. Lower triangle is a pie chart representation of Pearson's correlation coefficients. Triple asterisks (***) indicate P-value &lt; 10−4 that corresponds to FDR &lt; 0.01 for multiple comparisons of all 55 correlations between chromatin interaction frequency and other functional variables. Double asterisks indicate P-value &lt; 10−3, and single asterisk indicates P-value &lt; 10−2. Plots are made using ‘corrgram’ package on R. Frq, interaction frequency; cof, co-fitness; ppi, protein–protein interaction; cct, cell cycle time course; tfp, transcription factor perturbation; ace, acetylation; met, methylation; erc, evolutionary rate covariation; cfp, chromatin factor perturbation; fsm, functional similarity; rep, replication. Data in correlograms are ordered as per PCA clustering. ‘ρ’ stands for Pearson's correlation coefficient. Detailed control analyses are given in Supplementary Fig. S3. Details of three-letter abbreviations are given in the Table 1. (c) Cross-validation of differential correlations of cis and trans interaction frequencies with ‘rep‘ and ‘cof‘ through an independent HiC data of exponentially growing budding yeast cells (Rutledge et al.40). ‘ρ’ stands for Pearson's correlation coefficient. (d and e) Bi-plot of leading two X-loading vectors p1 and p2 (columns of matrix P) of PLSR model for (d) trans- and (e) cis-interacting gene pairs. The arrows are shaded as per p1 loading. Loading vectors p1 and p2 correspond to leading latent vectors t1 and t2 of matrix T, which explain 32.3 and 5.6% variance of Y (n× 1 matrix of interaction frequency) for trans interactions and 36.9 and 3.96% variance of cis interaction frequency, respectively. The datasets plotted in a and b are given in the Supplementary Tables S3 and S4. The loadings for all the components are given in the Supplementary Fig. S5. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200085/dsw00503.jpeg" path-from-xml="dsw00503.jpeg" target="_blank">View large</a><a section="87200085" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00503.jpeg?Expires=2147483647&amp;Signature=saGkVtx8qiANnS04Ld~xaZAPR5Frloea7nMPzAvmozhuUX0YJobKpzD3otWXNpjy8xM~IYJZz5j7kzjhfZoiQpqfu-GETgV5w~3DXjj-CDZzCHAGizSXEItfu8aRvf-TEYx7vwGwKsJ3oy0YrAsTtLtlDsN0pm8mN5ZYDRo7F3msXyOOmM-8gQgWJc3ymeHxTjcbOMWWKJJoCyJm0nnrWmgE7VEE7aoARVzZh7A2Dj80qUZQLxW-7-XI27FVHXcyy2rm6XpHza0QKRGYtnRyT-Ui8Tfs7Kg0RINEsMbjECvW4OtdVi6K4ENNCESHAB~0ZdqKepuSw7-uX4zD4xu-VQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200085&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multivariate analyses of <em>trans</em> and <em>cis</em> chromatin interactions in yeast genome. (a and b) Correlograms of distinct functional attributes of <em>trans-</em> and <em>cis-</em>interacting genes. The upper triangle shows the heatmap and the correlation values for each comparison. Lower triangle is a pie chart representation of Pearson's correlation coefficients. Triple asterisks (***) indicate <em>P</em>-value &lt; 10<sup>−4</sup> that corresponds to FDR &lt; 0.01 for multiple comparisons of all 55 correlations between chromatin interaction frequency and other functional variables. Double asterisks indicate <em>P</em>-value &lt; 10<sup>−3</sup>, and single asterisk indicates <em>P</em>-value &lt; 10<sup>−2</sup>. Plots are made using ‘corrgram’ package on R. Frq, interaction frequency; cof, co-fitness; ppi, protein–protein interaction; cct, cell cycle time course; tfp, transcription factor perturbation; ace, acetylation; met, methylation; erc, evolutionary rate covariation; cfp, chromatin factor perturbation; fsm, functional similarity; rep, replication. Data in correlograms are ordered as per PCA clustering. ‘<em>ρ</em>’ stands for Pearson's correlation coefficient. Detailed control analyses are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>. Details of three-letter abbreviations are given in the Table <a reveal-id="DSW005TB1" data-open="DSW005TB1" class="link link-reveal link-table xref-fig">1</a>. (c) Cross-validation of differential correlations of <em>cis</em> and <em>tran</em>s interaction frequencies with ‘rep‘ and ‘cof‘ through an independent HiC data of exponentially growing budding yeast cells (Rutledge et al.<sup><a reveal-id="DSW005C40" data-open="DSW005C40" class="link link-ref link-reveal xref-bibr">40</a></sup>). ‘<em>ρ</em>’ stands for Pearson's correlation coefficient. (d and e) Bi-plot of leading two X-loading vectors p1 and p2 (columns of matrix P) of PLSR model for (d) <em>trans-</em> and (e) <em>cis</em>-interacting gene pairs. The arrows are shaded as per p1 loading. Loading vectors p1 and p2 correspond to leading latent vectors t1 and t2 of matrix T, which explain 32.3 and 5.6% variance of Y (<em>n</em>× 1 matrix of interaction frequency) for <em>trans</em> interactions and 36.9 and 3.96% variance of <em>cis</em> interaction frequency, respectively. The datasets plotted in a and b are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>. The loadings for all the components are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Correlation analysis does not always imply if a variable independently accounts for the observed response (interaction frequency in this case) or it is due to co-linearity among variables. To address this, we performed PLSR of the variables to identify the ones that explained the observed variance in the frequency of chromatin interactions. The analyses suggested that 32.3 and 36.9% of total variance in <em>trans</em> and <em>cis</em> interaction data, respectively, were explained by one major component, primarily comprised of co-replication (rep) followed by similar susceptibility to chromatin factor perturbations (cfp) of interacting genes in case of <em>trans</em> interactions and co-fitness (cof) followed by functional similarity (fsm) of interacting genes in case of <em>cis</em> interactions (Fig. <a reveal-id="DSW005F3" data-open="DSW005F3" class="link link-reveal link-table xref-fig">3</a>d and e). The second best component explained only 5.6 and 3.9% of variance in <em>trans</em> and <em>cis</em> interaction frequency, respectively. Details of relative contributions of functional attributes to each component were given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>. Importantly, second component was also associated with co-replication to a large extent (second best contributor) in case of <em>trans</em> interactions and with co-fitness (second best contributor) in case of <em>cis</em> interactions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Interestingly, the cell-cycle-related co-expression (cct) of genes, in general, did not show significant contributions to the first two components in PLSR analysis of either <em>trans</em> or <em>cis</em> interactions. Overall, the PLSR and the correlation analyses concomitantly suggested that the <em>trans</em> interactions in yeast were primarily shaped by similarity in replication states of interacting loci, while <em>cis</em> interactions were mainly constrained by co-fitness of interacting genes highlighting an overall distinction of constraints shaping <em>cis</em> and <em>trans</em> chromatin interactomes.</p>                            <h3 scrollto-destination=87200087 id="87200087" class="section-title" data-legacy-id=s3b>3.2. <em>trans</em> and <em>cis</em> interactions associate with the coordination of early and late replication, respectively</h3>
<p>It has been shown that the early firing, but not the late firing, origins from different chromosomes non-randomly collide with each other in the nuclear space.<sup><a reveal-id="DSW005C22" data-open="DSW005C22" class="link link-ref link-reveal xref-bibr">22</a></sup> We first confirmed this observation (<em>P</em>-value = 3.1e–02, Fig. <a reveal-id="DSW005F4" data-open="DSW005F4" class="link link-reveal link-table xref-fig">4</a>a). Interestingly, we observed that the late, but not the early, firing origins non-randomly interacted in <em>cis</em> (<em>P</em>-value = 1.9e–02<em>;</em> Fig. <a reveal-id="DSW005F4" data-open="DSW005F4" class="link link-reveal link-table xref-fig">4</a>a), suggesting that the close physical proximity might be a common property of origins of replication and that early origins were <em>trans</em> interacting and late origins were <em>cis</em> interacting. Early and late origins showed significantly lesser number of <em>cis</em> and <em>trans</em> interactions with each other compared with null distribution, i.e. early-to-late interactions were under-represented compared with null distribution, strongly suggesting that they do not interact with each other in <em>cis</em> or <em>trans</em> (<em>P</em>-value<em><sub>cis</sub></em> = 0.003, <em>P</em>-value<em><sub>trans</sub></em> = 0.012; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). It is noteworthy that the significant <em>P</em>-values in the Fig. <a reveal-id="DSW005F4" data-open="DSW005F4" class="link link-reveal link-table xref-fig">4</a>a did not approach extreme partly due to small sample sizes (<em>n</em> = 78 and 122 for early and late origins) and stringent null models. </p>                        <a id="87200089" scrollto-destination="87200089"></a>
<div id="DSW005F4" data-id="DSW005F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00504.jpeg?Expires=2147483647&amp;Signature=HYa9s8HNh7UCg7ZM82NMZXORMkwtehyvbaC1ckjDBmzh4N5QszLAXBup1PaYL77VHZYTNIHm3pPIB0NxVr~RzfgBO3xRQbuP-8q5FOiqgFe5e1Mor7ymakaOM6DS2sO-q40rSqboURHsAdH0TtAuXVjddSx7akoESGAS~J56Vk0OkKFZ8-tkf4mowZ94aSdsRi5D2AJ60cKNWC7YdUnyiB87gQXu5LfajUejI~pGdrBjfke6ZaXkq3KRpl6gDfD0krwAk7cyY5MFBEIFM~cZ-a2aBWwrxuj5BaNCSyIk-rcRgTDSefHZ8mPxFKcS5p-yVpjg1cGuBUgOVnsXUTsQrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis and trans chromatin interactions with the coordinated replication and co-varying mutation rate. (a) Spatial connectivity (observed/expected number of interactions) among CLb5-independent early firing origins and Clb5-dependent late firing origins, with respect to null distribution generated through re-sampling-based method. Cartoon on the left-hand side represents the atypical cell-cycle curve representing number of cells (y-axis) as a function of replicated DNA content (x-axis). The first peak represents cells in G1 phase with 2N DNA followed by S-phase, and the second peak representing cells in G2 phase with 4N total DNA. The vertical bars in the cartoon represent early and late S-phase respectively. (b) Significance of correlation between interaction frequency and coordinated replication (upper panel), and similarity of mutation rate (lower panel) for cis and trans interactions. Interactions are split into EE (early-to-early) and LL (late-to-late) categories. Horizontal dotted lines represent P-value of 0.05. Over each bar, corresponding Pearson's correlations and 95% confidence values are given. Datasets of b are given in the Supplementary Tables S5 and S6. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200089/dsw00504.jpeg" path-from-xml="dsw00504.jpeg" target="_blank">View large</a><a section="87200089" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00504.jpeg?Expires=2147483647&amp;Signature=O8-kz8fgepyIlv2940qu21HWczANVAnJmabt29W9Qsjs~wnJ9ly9zxUPnF2uZ14Y~I2Hvrg3am4tdCGzwMBBrjSQdsqEvA4twQ9Jyvomo9ahW98~je0RZJ6JSo2xK9VsYCE9f9EF~V8peHo-7vEtrK9PDek8u1AHTETPCPzV79RAriJirGzAtIyAojvZp4eR6VnLlejHmM0Sa-4aB3vJnTlkQ6Dsif6gVsotwDijgmqQ5y7yIypz5hLKPmJp4IU-EG2Zv7PpbzTDkeCgUWDYTkTaBrbSTKiEuRy9MP-uR41P3n6UOG5lnoZ~M9kxs8sKYzWY6EvNN8nTyBxwG9KMTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200089&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> and <em>trans</em> chromatin interactions with the coordinated replication and co-varying mutation rate. (a) Spatial connectivity (observed/expected number of interactions) among CLb5-independent early firing origins and Clb5-dependent late firing origins, with respect to null distribution generated through re-sampling-based method. Cartoon on the left-hand side represents the atypical cell-cycle curve representing number of cells (<em>y</em>-axis) as a function of replicated DNA content (<em>x</em>-axis). The first peak represents cells in G1 phase with 2N DNA followed by S-phase, and the second peak representing cells in G2 phase with 4N total DNA. The vertical bars in the cartoon represent early and late S-phase respectively. (b) Significance of correlation between interaction frequency and coordinated replication (upper panel), and similarity of mutation rate (lower panel) for <em>cis</em> and <em>trans</em> interactions. Interactions are split into EE (early-to-early) and LL (late-to-late) categories. Horizontal dotted lines represent <em>P</em>-value of 0.05. Over each bar, corresponding Pearson's correlations and 95% confidence values are given. Datasets of b are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5 and S6</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00504.jpeg?Expires=2147483647&amp;Signature=HYa9s8HNh7UCg7ZM82NMZXORMkwtehyvbaC1ckjDBmzh4N5QszLAXBup1PaYL77VHZYTNIHm3pPIB0NxVr~RzfgBO3xRQbuP-8q5FOiqgFe5e1Mor7ymakaOM6DS2sO-q40rSqboURHsAdH0TtAuXVjddSx7akoESGAS~J56Vk0OkKFZ8-tkf4mowZ94aSdsRi5D2AJ60cKNWC7YdUnyiB87gQXu5LfajUejI~pGdrBjfke6ZaXkq3KRpl6gDfD0krwAk7cyY5MFBEIFM~cZ-a2aBWwrxuj5BaNCSyIk-rcRgTDSefHZ8mPxFKcS5p-yVpjg1cGuBUgOVnsXUTsQrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis and trans chromatin interactions with the coordinated replication and co-varying mutation rate. (a) Spatial connectivity (observed/expected number of interactions) among CLb5-independent early firing origins and Clb5-dependent late firing origins, with respect to null distribution generated through re-sampling-based method. Cartoon on the left-hand side represents the atypical cell-cycle curve representing number of cells (y-axis) as a function of replicated DNA content (x-axis). The first peak represents cells in G1 phase with 2N DNA followed by S-phase, and the second peak representing cells in G2 phase with 4N total DNA. The vertical bars in the cartoon represent early and late S-phase respectively. (b) Significance of correlation between interaction frequency and coordinated replication (upper panel), and similarity of mutation rate (lower panel) for cis and trans interactions. Interactions are split into EE (early-to-early) and LL (late-to-late) categories. Horizontal dotted lines represent P-value of 0.05. Over each bar, corresponding Pearson's correlations and 95% confidence values are given. Datasets of b are given in the Supplementary Tables S5 and S6. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200089/dsw00504.jpeg" path-from-xml="dsw00504.jpeg" target="_blank">View large</a><a section="87200089" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00504.jpeg?Expires=2147483647&amp;Signature=O8-kz8fgepyIlv2940qu21HWczANVAnJmabt29W9Qsjs~wnJ9ly9zxUPnF2uZ14Y~I2Hvrg3am4tdCGzwMBBrjSQdsqEvA4twQ9Jyvomo9ahW98~je0RZJ6JSo2xK9VsYCE9f9EF~V8peHo-7vEtrK9PDek8u1AHTETPCPzV79RAriJirGzAtIyAojvZp4eR6VnLlejHmM0Sa-4aB3vJnTlkQ6Dsif6gVsotwDijgmqQ5y7yIypz5hLKPmJp4IU-EG2Zv7PpbzTDkeCgUWDYTkTaBrbSTKiEuRy9MP-uR41P3n6UOG5lnoZ~M9kxs8sKYzWY6EvNN8nTyBxwG9KMTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200089&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> and <em>trans</em> chromatin interactions with the coordinated replication and co-varying mutation rate. (a) Spatial connectivity (observed/expected number of interactions) among CLb5-independent early firing origins and Clb5-dependent late firing origins, with respect to null distribution generated through re-sampling-based method. Cartoon on the left-hand side represents the atypical cell-cycle curve representing number of cells (<em>y</em>-axis) as a function of replicated DNA content (<em>x</em>-axis). The first peak represents cells in G1 phase with 2N DNA followed by S-phase, and the second peak representing cells in G2 phase with 4N total DNA. The vertical bars in the cartoon represent early and late S-phase respectively. (b) Significance of correlation between interaction frequency and coordinated replication (upper panel), and similarity of mutation rate (lower panel) for <em>cis</em> and <em>trans</em> interactions. Interactions are split into EE (early-to-early) and LL (late-to-late) categories. Horizontal dotted lines represent <em>P</em>-value of 0.05. Over each bar, corresponding Pearson's correlations and 95% confidence values are given. Datasets of b are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5 and S6</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>These observations suggested that the correlation between <em>trans</em> interaction frequency and correlated replication seen in Fig. <a reveal-id="DSW005F2" data-open="DSW005F2" class="link link-reveal link-table xref-fig">2</a>a might be associated with the non-random <em>trans</em> interactions among early origins. On similar lines, relatively weaker correlation (<em>ρ</em>=0.14) of <em>cis</em> interaction frequency with the coordinated replication might be associated with the non-random <em>cis</em> interactions among late origins (Fig. <a reveal-id="DSW005F3" data-open="DSW005F3" class="link link-reveal link-table xref-fig">3</a>b). To further explore whether <em>trans</em> and <em>cis</em> interactions associate with the coordination of early and late replication, respectively, we split the yeast genome into early and late replicating domains as per data from McCune et al.<sup><a reveal-id="DSW005C41" data-open="DSW005C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and accordingly compiled early-to-early and late-to-late interaction categories. Early-to-early (EE) interaction category exhibited significant correlation between coordinated replication and the <em>trans</em>, but not the <em>cis</em>, interaction frequency (<em>ρ<sub>trans</sub></em> = 0.37, <em>ρ<sub>cis</sub></em> = 0.0; Fig. <a reveal-id="DSW005F4" data-open="DSW005F4" class="link link-reveal link-table xref-fig">4</a>b). On the contrary, in the late-to-late category, <em>cis</em> and <em>trans</em> interactions showed marginally differing correlations with coordinated replication (<em>ρ<sub>cis</sub></em> = 0.26, <em>ρ<sub>trans</sub></em> = 0.29; Fig. <a reveal-id="DSW005F4" data-open="DSW005F4" class="link link-reveal link-table xref-fig">4</a>b). However, importantly, the <em>cis</em> interaction frequency in the late-to-late category showed greater significance and power of correlation with the coordinated replication compared with the <em>trans</em> interaction frequency (<em>P</em>-value<em><sub>cis</sub></em> = 8.7 × 10<sup>−8</sup>, <em>P</em>-value<em><sub>trans</sub></em> = 2.7 × 10<sup>−4</sup>; power<em><sub>cis</sub></em> = 0.99 and power<em><sub>trans</sub></em> = 0.83; Fig. <a reveal-id="DSW005F4" data-open="DSW005F4" class="link link-reveal link-table xref-fig">4</a>b). Since the Pearson's correlation coefficient decreases and the power of the observed effect increases exponentially as the sample size increases, we reason that the differing sample sizes of <em>cis</em> and <em>trans</em> interactions of late replicating regions (<em>n</em> = 711 and 146 binned values, respectively) explain the observed difference in significance of apparently similar correlation values. It is also notable that the <em>cis</em> interactions, as a whole, were only weakly associated with the coordinated replication as demonstrated in Fig. <a reveal-id="DSW005F3" data-open="DSW005F3" class="link link-reveal link-table xref-fig">3</a>b, the significance of correlation of late replication with the <em>cis</em> interactions was, therefore, non-trivial. This is also reinforced by our earlier observation that late origins interacted in <em>cis</em>, but not in <em>trans</em>. These results supported that the <em>trans</em> and <em>cis</em> interactions were organized vis-a-vis to correlated early and late replication states, respectively, possibly marking spatial segregation of early and late replication factories. Furthermore, replication is the major cause of mutations in the genome, and correlation between replication timing and mutation rates has been observed in the past.<sup><a reveal-id="DSW005C42" data-open="DSW005C42" class="link link-ref link-reveal xref-bibr">42</a></sup> We, therefore, asked whether spatially proximal loci exhibited similar mutation rates. We obtained synonymous single-nucleotide substitution rates, which were adjusted for neutrality, for this purpose (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Again, we observed stronger correlation between similarity in mutation rates and the <em>trans</em> interaction frequency and relatively weaker correlation with the <em>cis</em> interaction frequency (<em>ρ<sub>cis</sub></em> = 0.14, <em>ρ<sub>trans</sub></em> = 0.36; <em>P</em>-value<em><sub>cis</sub></em> = 0.63, <em>P</em>-value<em><sub>trans</sub></em> = 2.2e–05; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). Early-to-early interactions in <em>trans</em> and late-to-late interactions in <em>cis</em> exhibited greater significance of correlations with the similarity in mutation rates of engaged genes (<em>P</em>-value<sub>EE,<em>cis</em></sub> = 0.15, <em>P</em>-value<sub>EE,<em>trans</em></sub> = 1.4e–03, <em>P</em>-value<sub>LL,<em>cis</em></sub> = 5.4e–02, <em>P</em>-value<sub>LL,<em>trans</em></sub> = 0.46; Fig. <a reveal-id="DSW005F4" data-open="DSW005F4" class="link link-reveal link-table xref-fig">4</a>b), highlighting a possible role of 3D genome organization in mutagenic mechanisms that might have shaped the co-evolution of spatially proximal genes in yeast. We hypothesize that the genes sharing the same sub-nuclear compartment might experience similar microenvironment that causes single-nucleotide mutations during replication or similar deficiency/efficiency of DNA repair factors.</p>                            <h3 scrollto-destination=87200091 id="87200091" class="section-title" data-legacy-id=s3c>3.3. <em>trans</em> and <em>cis</em> interactions exhibit differential preference for nucleosome occupancy</h3>
<p>Since the process of replication is intricately linked with the chromatin structure, we tested whether the observed distinction between <em>trans</em> and <em>cis</em> interactomes was associated with nucleosome occupancy and explains their association with early and late replication, respectively. <em>Z</em>-normalized nucleosome occupancy in 3 kb bins across yeast genome was plotted against the number of <em>cis</em>, <em>trans</em> interactions, and the ratio of the two. The plots clearly showed significantly negative and positive correlations with <em>cis</em> and <em>trans</em> interactions, respectively (<em>ρ<sub>cis</sub></em> = −0.12, <em>ρ<sub>trans</sub></em> = 0.13; <em>P</em>-value<em><sub>cis</sub></em> &lt; 2.2e–16, <em>P</em>-value<em><sub>trans</sub></em> &lt; 2.2e–16; Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>a). Accordingly, the ratio of number of <em>cis</em> and <em>trans</em> interactions inversely scaled with the nucleosome occupancy suggesting that <em>cis</em> and <em>trans</em> interactions, in general, associate with low and high nucleosome occupancy, respectively (<em>ρ<sub>cis/trans</sub></em> = −0.10; <em>P</em>-value<em><sub>cis/trans</sub></em> = 7.2e–08; Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>a). The Pearson's correlation coefficients observed in the Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>a, though statistically significant, were relatively small in magnitude, which can be attributed to large sample size (<em>n</em> = 4,031). It is also noteworthy that the present study does not aim to claim the scaling of <em>cis</em> and <em>trans</em> interactions with the nucleosome occupancy, but the distinct level of nucleosome occupancy of regions significantly enriched with <em>cis</em> or <em>trans</em> interactions. Therefore, we identified the <em>cis</em>- and <em>trans</em>-rich regions using sliding window (20 kb) approach and applying cut-off for <em>cis</em>/<em>trans</em> peaks as 50% of maximal window value in the genome (Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>b). Demarcating the regions that were enriched with <em>cis-</em> or <em>trans</em> interactions clearly highlighted greater nucleosome occupancy in the regions enriched with <em>trans</em> interactions and <em>vice versa</em> (<em>P</em>-value &lt; 2.2e–16; Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>b, boxplot). These observations suggested that the nucleosome-depleted ‘open chromatin’ regions in the yeast genome were more likely to fold in <em>cis</em>, possibly highlighting a mechanism of spatially insulating the highly transcribing open chromatin from the nucleosome-rich repressive or regulated domains. We indeed observed that <em>cis</em>-enriched, nucleosome-depleted domains had relatively higher transcriptional activity compared with <em>trans</em>-enriched regions (<em>P</em> = 5.0e–03; Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>b, boxplot). We also confirmed that these correlations were not the artefacts of abundant <em>trans</em> interactions among centromeres, which are generally heterochromatized and might show higher nucleosomal occupancy. We removed the regions that were proximal (&lt;50 kb) to centro/telomeres and recalculated the correlations. We consistently found significant negative and positive correlations of nucleosome occupancy with the <em>cis</em> and <em>trans</em> interaction frequencies, respectively (<em>ρ<sub>cis</sub></em> = −0.12, <em>ρ<sub>cis</sub></em> = 0.19; <em>P</em>-value<em><sub>cis</sub></em> &lt; 2.2e–16, <em>P</em>-value<em><sub>trans</sub></em> &lt; 2.2e–16). </p>                        <a id="87200093" scrollto-destination="87200093"></a>
<div id="DSW005F5" data-id="DSW005F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00505.jpeg?Expires=2147483647&amp;Signature=Xd2LuViAolcgo6IksLLYFLx62Yrk3Wi9sordAe9xezQOA-tpkS90hyG~6Z01paLUAH0aUrf4~hEby-imml~tACRLKBGew4lPNO5eTWPQbaDyh1LTwYZe1TJ-dY0wVbTWo3bSozXNwxyBedMP4~eB3cuvf3z~FbjfGkohHJSYV0Kuq2xK4Vr2-o5ULDPb6hMvWXcL0lTXU-8lvVWbtYPLogEjVcPhdeLqop5eMs0tmHrFcn8dkPkiHyQkrN5tRRQ8eT8qhQVluxoax8ruXdNd9F2ej~H400MsPbXR95QiBD1TW65ANLZyg0UvmfCPO2BiRTLZ~M40s2PG5qhUjqQ7fw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis and trans chromatin interactions with the nucleosome occupancy. (a) Left: Regression plot between the ratio of cis to trans interactions and nucleosomal occupancy. Right: regressions showing relationships between nucleosome occupancy and cis/trans interaction frequency. (b) Linear view of nucleosome occupancy and gene expression (SAGE counts) aligned along the number of cis/trans chromatin interactions. cis-Enriched and trans-enriched regions are highlighted. The peaks were identified as windows having sliding mean (averaged over 10 consecutive windows) greater than the 50% of the maximum window average in the genome. Boxplots on the right-hand side show distributions of nucleosome occupancy and SAGE counts in the cis-rich and trans-rich regions. (c) Left panel: average nucleosome occupancy (G1 phase) around ±10 kb of early and late origins. Right panel: average nucleosome occupancy profile around early and late origins. Shades represent the standard errors across all the early and late origins respectively. The datasets of b and c are given in Supplementary Tables S7 and S8, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200093/dsw00505.jpeg" path-from-xml="dsw00505.jpeg" target="_blank">View large</a><a section="87200093" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00505.jpeg?Expires=2147483647&amp;Signature=ZENST8Xebu4366U9pviXUxivCIJInUDdfSg37PdfdxQMEqg97Q7VzkIb27zmtILD9i6JxsovgSKX17qLxM3jJXxPOi0M9mkxxHyh9hsxvrea4rY2gdAdkz1DJWrWJfoP~wdPCrEGpzLdQW-VeQvqWPPQET7PsxPaUekjg3D8RtD4bm7u7XblxZfCXhL61EB1hUGD0ZuhqPzEwBMlV3YtnvAH4Zbs0RN6K~W~8Jo5ljGEdhqCl4zc-PASW6d3IhhKlvbB1F-ARMeOSYVusVvkQ9MfN9yj7CLPEep9MvOTiiDJ2crQH9jMzYiW3Fr-F8NL31XwPambHLw1sVIBLZxWhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200093&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> and <em>trans</em> chromatin interactions with the nucleosome occupancy. (a) Left: Regression plot between the ratio of <em>cis</em> to <em>trans</em> interactions and nucleosomal occupancy. Right: regressions showing relationships between nucleosome occupancy and <em>cis/trans</em> interaction frequency. (b) Linear view of nucleosome occupancy and gene expression (SAGE counts) aligned along the number of <em>cis/trans</em> chromatin interactions. <em>cis</em>-Enriched and <em>trans-</em>enriched regions are highlighted. The peaks were identified as windows having sliding mean (averaged over 10 consecutive windows) greater than the 50% of the maximum window average in the genome. Boxplots on the right-hand side show distributions of nucleosome occupancy and SAGE counts in the <em>cis</em>-rich and <em>trans</em>-rich regions. (c) Left panel: average nucleosome occupancy (G1 phase) around ±10 kb of early and late origins. Right panel: average nucleosome occupancy profile around early and late origins. Shades represent the standard errors across all the early and late origins respectively. The datasets of b and c are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7 and S8</a></span>, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00505.jpeg?Expires=2147483647&amp;Signature=Xd2LuViAolcgo6IksLLYFLx62Yrk3Wi9sordAe9xezQOA-tpkS90hyG~6Z01paLUAH0aUrf4~hEby-imml~tACRLKBGew4lPNO5eTWPQbaDyh1LTwYZe1TJ-dY0wVbTWo3bSozXNwxyBedMP4~eB3cuvf3z~FbjfGkohHJSYV0Kuq2xK4Vr2-o5ULDPb6hMvWXcL0lTXU-8lvVWbtYPLogEjVcPhdeLqop5eMs0tmHrFcn8dkPkiHyQkrN5tRRQ8eT8qhQVluxoax8ruXdNd9F2ej~H400MsPbXR95QiBD1TW65ANLZyg0UvmfCPO2BiRTLZ~M40s2PG5qhUjqQ7fw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis and trans chromatin interactions with the nucleosome occupancy. (a) Left: Regression plot between the ratio of cis to trans interactions and nucleosomal occupancy. Right: regressions showing relationships between nucleosome occupancy and cis/trans interaction frequency. (b) Linear view of nucleosome occupancy and gene expression (SAGE counts) aligned along the number of cis/trans chromatin interactions. cis-Enriched and trans-enriched regions are highlighted. The peaks were identified as windows having sliding mean (averaged over 10 consecutive windows) greater than the 50% of the maximum window average in the genome. Boxplots on the right-hand side show distributions of nucleosome occupancy and SAGE counts in the cis-rich and trans-rich regions. (c) Left panel: average nucleosome occupancy (G1 phase) around ±10 kb of early and late origins. Right panel: average nucleosome occupancy profile around early and late origins. Shades represent the standard errors across all the early and late origins respectively. The datasets of b and c are given in Supplementary Tables S7 and S8, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200093/dsw00505.jpeg" path-from-xml="dsw00505.jpeg" target="_blank">View large</a><a section="87200093" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00505.jpeg?Expires=2147483647&amp;Signature=ZENST8Xebu4366U9pviXUxivCIJInUDdfSg37PdfdxQMEqg97Q7VzkIb27zmtILD9i6JxsovgSKX17qLxM3jJXxPOi0M9mkxxHyh9hsxvrea4rY2gdAdkz1DJWrWJfoP~wdPCrEGpzLdQW-VeQvqWPPQET7PsxPaUekjg3D8RtD4bm7u7XblxZfCXhL61EB1hUGD0ZuhqPzEwBMlV3YtnvAH4Zbs0RN6K~W~8Jo5ljGEdhqCl4zc-PASW6d3IhhKlvbB1F-ARMeOSYVusVvkQ9MfN9yj7CLPEep9MvOTiiDJ2crQH9jMzYiW3Fr-F8NL31XwPambHLw1sVIBLZxWhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200093&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> and <em>trans</em> chromatin interactions with the nucleosome occupancy. (a) Left: Regression plot between the ratio of <em>cis</em> to <em>trans</em> interactions and nucleosomal occupancy. Right: regressions showing relationships between nucleosome occupancy and <em>cis/trans</em> interaction frequency. (b) Linear view of nucleosome occupancy and gene expression (SAGE counts) aligned along the number of <em>cis/trans</em> chromatin interactions. <em>cis</em>-Enriched and <em>trans-</em>enriched regions are highlighted. The peaks were identified as windows having sliding mean (averaged over 10 consecutive windows) greater than the 50% of the maximum window average in the genome. Boxplots on the right-hand side show distributions of nucleosome occupancy and SAGE counts in the <em>cis</em>-rich and <em>trans</em>-rich regions. (c) Left panel: average nucleosome occupancy (G1 phase) around ±10 kb of early and late origins. Right panel: average nucleosome occupancy profile around early and late origins. Shades represent the standard errors across all the early and late origins respectively. The datasets of b and c are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7 and S8</a></span>, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To reconcile the observation in the context of replication, we assessed the nucleosome occupancy around the early origins and compared with that of late origins. To nullify the differential levels of replicated DNA at early and late origins during S phase, we considered nucleosome occupancy in G1 phase of cell cycle only. Since we showed that the <em>trans</em> interactions were associated with early origins as well as with higher overall nucleosome occupancy, by inference, we expected the early origins to be associated with the higher nucleosome occupancy and vice versa for late origins. Indeed, we confirmed greater overall nucleosome occupancy around early origins compared with late origins (±10 kb, <em>P</em>-value = 5.0e-02, Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>c). Interestingly, by analysing the nucleosome occupancy at single nucleosome resolution, we observed that the early origins were located in a narrow (∼350 bp) nucleosome-depleted region (NDR) despite having an overall higher nucleosome occupancy (Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>c). Such nucleosomal organization had been shown to facilitate the replication initiation.<sup><a reveal-id="DSW005C43" data-open="DSW005C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Late origins, on the contrary, were located in a relatively wider NDR (∼750 bp), significance of which is not entirely clear yet (Fig. <a reveal-id="DSW005F5" data-open="DSW005F5" class="link link-reveal link-table xref-fig">5</a>c). However, we speculated that the late origins might need accessibility to additional sequence features for the regulatory factors involved in late replication. Indeed, such sequence features have been proposed earlier by others too.<sup><a reveal-id="DSW005C44" data-open="DSW005C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSW005C45" data-open="DSW005C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Role of <em>cis</em> regulatory elements in late firing of origins is further discussed in the Discussion section.</p>                            <h3 scrollto-destination=87200095 id="87200095" class="section-title" data-legacy-id=s3d>3.4. Association of <em>cis</em> interactions with the essential gene clusters</h3>
<p>Strong correlation between <em>cis</em> interaction frequency and the co-fitness of interacting genes suggested that the genes with similar fitness defects across different environments tend to co-localize more frequently. This would also mean that genes with the same extreme growth defects, like lethality, would tend to interact with greater interaction frequency. We, therefore, tested whether the essential genes were interacting with each other in <em>cis</em> with a significantly greater frequency compared with random null model. We extracted the <em>cis</em> and <em>trans</em> interactions that had essential genes on both restriction fragments. Indeed, the re-sampling approach uncovered significantly greater frequency of <em>cis</em> (<em>P</em>-value = 5e–02), but not the <em>trans</em> (<em>P</em>-value = 0.16), interactions among essential genes (Fig. <a reveal-id="DSW005F6" data-open="DSW005F6" class="link link-reveal link-table xref-fig">6</a>a). Essential genes are generally located in the regions having an overall low nucleosome occupancy and consequently exhibit lower expression noise.<sup><a reveal-id="DSW005C46" data-open="DSW005C46" class="link link-ref link-reveal xref-bibr">46</a></sup> We, therefore, propose a hypothesis that the essential genes should also be relatively stable in the three-dimensional nuclear space to keep the expression noise low. Greater mobility might introduce noise in the expression, which can be deleterious in case of essential genes. We presumed that a stable locus would be expected to show: (i) high average value of its interaction frequencies to all its <em>cis</em>-interacting partners, that we referred as ‘average interaction frequency’; (ii) relatively short-range interactions and lower variance in the genomic distances between interacting loci; and (iii) greater consistency of chromatin interactions across biological replicates in HiC experiments, measured as overlap of interactions of a locus commonly captured in <em>Eco</em>RI and <em>Hin</em>dIII HiC libraries to all its interactions captured in <em>Hin</em>dIII library. Although it is not possible to distinguish biological variations from technical variations in the present scenario, some studies have shown that variation between biological replicates are largely biological than technical and can be attributed to genetic, epigenetic, or stochastic mosaicism of cells.<sup><a reveal-id="DSW005C47" data-open="DSW005C47" class="link link-ref link-reveal xref-bibr">47</a>–<a reveal-id="DSW005C52" data-open="DSW005C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The chromatin loci exhibiting greater consistency of chromatin interactions in biological replicates can, therefore, be considered relatively restrained in the nuclear space compared with the ones showing greater variations. By applying the aforementioned measures, we observed that the loci having essential gene clusters (&gt;2 gene in 5 kb bin) consistently showed: (i) higher average interaction frequency (<em>P</em>-value = 3.8e–06<em>;</em> Fig. <a reveal-id="DSW005F6" data-open="DSW005F6" class="link link-reveal link-table xref-fig">6</a>b), (ii) relatively short-range interactions as depicted by mean distance between interacting sites (<em>P</em>-value = 2.9e–02; Fig. <a reveal-id="DSW005F6" data-open="DSW005F6" class="link link-reveal link-table xref-fig">6</a>c); (iii) lower variance in genomic distances between interacting sites (<em>P</em>-value = 3.5e–02; Fig. <a reveal-id="DSW005F6" data-open="DSW005F6" class="link link-reveal link-table xref-fig">6</a>c); and (iv) greater overlap between biological replicates of HiC data, compared with rest of genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Further, the average interaction frequency decreases as a function of genomic distance from the essential gene clusters, marking the selective enrichment of high frequency interaction on and around essential gene clusters (Fig. <a reveal-id="DSW005F6" data-open="DSW005F6" class="link link-reveal link-table xref-fig">6</a>d and e, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Thus, the higher average interaction frequency, relatively short-range interactions of lower variance, and consistency across HiC replicates might suggest spatially clustered and stable nature of genomic regions having essential gene clusters. We also confirmed the link between gene essentiality and the average interaction frequency analysing average fitness defect, as measure from chemical genomics data, at sites of high average interaction frequency (<em>P</em>-value = 9.0e–06; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). </p>                        <a id="87200097" scrollto-destination="87200097"></a>
<div id="DSW005F6" data-id="DSW005F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00506.jpeg?Expires=2147483647&amp;Signature=P5BLXb1t8VASkIzcOGsNB7tQBtY3rRiSt3KRh4A-Vf4BgzcLVJd6pigeDYp9ilcnDrx4c5-hNELqKC9jXvDTOKvnplL-WTqBLhT5lwJZjb9k3yd7EmuaQprrLPoiLp7c5Fuocylq1rtHBrb1deXoYSy71avwMV~M3u0RRQZPUldlXNqWO3CSy6po0hZo7H~5GH9zvkRtzReUwWZleXc17lc576eM1usK40nGfM1tXi59BzPSvvjf3Ui51RVyDQM1X7pr9wAIrSA07ch6qA15DB6bqNhOwjIuF-YN8~GhQ8PlE77fRl-M7nk9clctFFgiuxhk-gmiB6gzNSvRkg-eMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis interactions with the gene essentiality. (a) Average frequency of trans and cis interactions among essential genes overlaid on the null distribution of average values across 1,000 random samples. Cartoon on the top depicts all cis interactions of a locus (grey-coloured node), from which the average interaction frequency of that locus was calculated. (b) Average frequency of cis interactions of genomic loci having and not having essential gene clusters. (c) Mean and variance of genomic distances between interaction sites at genomic loci having and not having essential gene clusters. Cartoon on the top represents the genomic distance between cis-interacting sites of grey-coloured node. (d) Average cis interaction frequency as a function of relative distance from the essential gene cluster (defined as &gt;2 essential genes in 5 kb window). (e) Average cis interaction frequency (moving average of three consecutive restriction fragments) and essential gene density along Chromosome 3. Data plotted in b–e are given in the Supplementary Table S9. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200097/dsw00506.jpeg" path-from-xml="dsw00506.jpeg" target="_blank">View large</a><a section="87200097" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00506.jpeg?Expires=2147483647&amp;Signature=oAF-Rk4QoXDb--tpldLoMZGj~PsUOyMueQhkmIjwwzYJkaFmoH1yv7OKCmL-tLEzcZzS5TuYAHeX8zrmC8c9uILqAflR6-GmfReCMLjU0hzHWCPYvxry8nboPQZBkNYCL8GPRWSwia255rXg41-m2PfDGuypvhEsxX8Xz1TaHqME56JQUQlJbCZfEqHyEFTCIG4ovyLplioakGhdA7a4QzfFzaPeXFMqAR5YTRZ~9BGX5XnYy7hMDyDY4r65Zot6MtSeqwxOOx81LJyEchTIPRaUYDAhv9yajp2fBFPH84qgNw8zICxL6mfdWF2xYMPp-5P7uZ86jnd0XYPCqOBZOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200097&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> interactions with the gene essentiality. (a) Average frequency of <em>trans</em> and <em>cis</em> interactions among essential genes overlaid on the null distribution of average values across 1,000 random samples. Cartoon on the top depicts all <em>cis</em> interactions of a locus (grey-coloured node), from which the average interaction frequency of that locus was calculated. (b) Average frequency of <em>cis</em> interactions of genomic loci having and not having essential gene clusters. (c) Mean and variance of genomic distances between interaction sites at genomic loci having and not having essential gene clusters. Cartoon on the top represents the genomic distance between <em>cis</em>-interacting sites of grey-coloured node. (d) Average <em>cis</em> interaction frequency as a function of relative distance from the essential gene cluster (defined as &gt;2 essential genes in 5 kb window). (e) Average <em>cis</em> interaction frequency (moving average of three consecutive restriction fragments) and essential gene density along Chromosome 3. Data plotted in b–e are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00506.jpeg?Expires=2147483647&amp;Signature=P5BLXb1t8VASkIzcOGsNB7tQBtY3rRiSt3KRh4A-Vf4BgzcLVJd6pigeDYp9ilcnDrx4c5-hNELqKC9jXvDTOKvnplL-WTqBLhT5lwJZjb9k3yd7EmuaQprrLPoiLp7c5Fuocylq1rtHBrb1deXoYSy71avwMV~M3u0RRQZPUldlXNqWO3CSy6po0hZo7H~5GH9zvkRtzReUwWZleXc17lc576eM1usK40nGfM1tXi59BzPSvvjf3Ui51RVyDQM1X7pr9wAIrSA07ch6qA15DB6bqNhOwjIuF-YN8~GhQ8PlE77fRl-M7nk9clctFFgiuxhk-gmiB6gzNSvRkg-eMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis interactions with the gene essentiality. (a) Average frequency of trans and cis interactions among essential genes overlaid on the null distribution of average values across 1,000 random samples. Cartoon on the top depicts all cis interactions of a locus (grey-coloured node), from which the average interaction frequency of that locus was calculated. (b) Average frequency of cis interactions of genomic loci having and not having essential gene clusters. (c) Mean and variance of genomic distances between interaction sites at genomic loci having and not having essential gene clusters. Cartoon on the top represents the genomic distance between cis-interacting sites of grey-coloured node. (d) Average cis interaction frequency as a function of relative distance from the essential gene cluster (defined as &gt;2 essential genes in 5 kb window). (e) Average cis interaction frequency (moving average of three consecutive restriction fragments) and essential gene density along Chromosome 3. Data plotted in b–e are given in the Supplementary Table S9. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200097/dsw00506.jpeg" path-from-xml="dsw00506.jpeg" target="_blank">View large</a><a section="87200097" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00506.jpeg?Expires=2147483647&amp;Signature=oAF-Rk4QoXDb--tpldLoMZGj~PsUOyMueQhkmIjwwzYJkaFmoH1yv7OKCmL-tLEzcZzS5TuYAHeX8zrmC8c9uILqAflR6-GmfReCMLjU0hzHWCPYvxry8nboPQZBkNYCL8GPRWSwia255rXg41-m2PfDGuypvhEsxX8Xz1TaHqME56JQUQlJbCZfEqHyEFTCIG4ovyLplioakGhdA7a4QzfFzaPeXFMqAR5YTRZ~9BGX5XnYy7hMDyDY4r65Zot6MtSeqwxOOx81LJyEchTIPRaUYDAhv9yajp2fBFPH84qgNw8zICxL6mfdWF2xYMPp-5P7uZ86jnd0XYPCqOBZOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200097&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> interactions with the gene essentiality. (a) Average frequency of <em>trans</em> and <em>cis</em> interactions among essential genes overlaid on the null distribution of average values across 1,000 random samples. Cartoon on the top depicts all <em>cis</em> interactions of a locus (grey-coloured node), from which the average interaction frequency of that locus was calculated. (b) Average frequency of <em>cis</em> interactions of genomic loci having and not having essential gene clusters. (c) Mean and variance of genomic distances between interaction sites at genomic loci having and not having essential gene clusters. Cartoon on the top represents the genomic distance between <em>cis</em>-interacting sites of grey-coloured node. (d) Average <em>cis</em> interaction frequency as a function of relative distance from the essential gene cluster (defined as &gt;2 essential genes in 5 kb window). (e) Average <em>cis</em> interaction frequency (moving average of three consecutive restriction fragments) and essential gene density along Chromosome 3. Data plotted in b–e are given in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200098 id="87200098" class="section-title" data-legacy-id=s3e>3.5. <em>cis</em> interactions are constrained by minimization of expression noise</h3>
<p>To directly test whether or not spatially restrained loci were associated with the lower noise in the expression, we extracted high noise and low noise genes (upper and lower quartile of genome-wide abundance-corrected expression noise data) and assessed their average interaction frequencies, mean and variance of genomic distances between interacting sites, and the overlap ratio of their interactions captured in the HiC replicates. The genes with low expression noise exhibited greater average interaction frequency (<em>P</em>-value = 2e–02; Fig. <a reveal-id="DSW005F7" data-open="DSW005F7" class="link link-reveal link-table xref-fig">7</a>a), relatively lower mean and variance of genomic distances between interacting loci (<em>P</em>-value<sub>mean</sub> = 9e–03, <em>P</em>-value<sub>variance</sub> = 4e–02; Fig. <a reveal-id="DSW005F7" data-open="DSW005F7" class="link link-reveal link-table xref-fig">7</a>b), and greater overlap ratio of interactions in the HiC biological replicates (<em>P</em>-value = 3.3e05; Fig. <a reveal-id="DSW005F7" data-open="DSW005F7" class="link link-reveal link-table xref-fig">7</a>), supporting the hypothesis that spatially restrained loci would tend to have lower expression noise. Indeed, the expression noise of essential genes inversely scaled with the average <em>cis,</em> but not the <em>trans</em> interaction frequency of the loci (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Further, it is known that the genes that are toxic when over-expressed also exhibit low expression noise.<sup><a reveal-id="DSW005C46" data-open="DSW005C46" class="link link-ref link-reveal xref-bibr">46</a></sup> We, therefore, tested whether these genes are also spatially constrained by high frequency of <em>cis</em> interactions. Surprisingly, genes exhibiting over-expression toxicity did not show significantly greater average interaction frequency as in the case of essential gene clusters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>). We propose that spatially restrained or ‘tethered’ property is specifically associated with the mechanism that minimizes stochastic loss of expression but not the abrupt gain of expression. </p>                        <a id="87200100" scrollto-destination="87200100"></a>
<div id="DSW005F7" data-id="DSW005F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00507.jpeg?Expires=2147483647&amp;Signature=LUfwlEqlmAMJD5D0FHopcgeC0lKlXloZl8gNQuHKqei5eoanKluyG-TysbGHtdkQDQbB1A6tvz6nBhzGfsrKBrtDzKfz5MhKGu4-CFw54A7M-nRk~~r0P2HMDyf4JYiLVbzoAHUHnprjieTMBOaFHpXppWKpVMg3~OWGZ6PtE8Ytb3yj2jXbqDT-DhixK-vu6Qs64x1F2HWQrpR4dlHgmY8Ub4Yg8SYB55HJm4unMt89wWeeHthbO7d0cVdG0lCyuqdKL-s5S-0VDtVANxOtZICYQpTfN~X5w9w7w93eLvKCl8~05So6L5GHnzuxAfCNfSWSglwLrZF38Py8sadPjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis interactions with expression noise and chromatin fluctuations. (a) Distribution of average cis interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes. Cartoon on the top depicts all cis interactions of a locus (grey-coloured node), from which the average interaction frequency of grey-coloured locus was calculated. (b) Distribution of mean and variance of genomic distances between interacting sites for low and high noise genes. Cartoon on the top represents the genomic distance between cis-interacting sites of grey-coloured node. (c) Distribution of overlap ratio of chromatin interactions between HindIII and EcoRI HiC replicates for low and high noise genes. Horizontal dotted line represents the genome-wide median value. (d–f) Scatter plot between and experimentally determined chromatin fluctuations and (d) average cis interaction frequency, (e) expression noise, and (f) variance of genomic distances between interacting sites on yeast Chromosome 12. (g) Expression noise, average cis interaction frequency, variance of genomic distances, and chromatin fluctuations aligned along Chromosome 12. Data plotted in a–c and d–g are given in Supplementary Tables S9 and S10, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200100/dsw00507.jpeg" path-from-xml="dsw00507.jpeg" target="_blank">View large</a><a section="87200100" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00507.jpeg?Expires=2147483647&amp;Signature=jIF00-FDCY0SZ9j5C1ghhL3isdIu~SdPpFACyJcbCOodyUTNNy1HvP0d1v3E7VO10Gw28CNbxBTiVxdqab15xIZO7amheAIKqoFJjBfh0l8nZXYrxNcN0ZX0cDAnuI5vQGVopCzJegeLZVviN9qD3zKrwu6XV6knW8hRQuvrv41ePVFvGeDBjUqRVcQIkEL-AIeOMDw0bNZVpM7qAkACNoRGjj3TUxkcUBy-w6zp09IA7APyO2peiNwmBbOpV9ifsQQ396yLzzx8k-0kw~1EVVA68t5HQpIqysyejSqoXumCUeKemvw8fgLFvFDYMl9A-B5xqQEuXtQFrjsPNloTOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200100&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> interactions with expression noise and chromatin fluctuations. (a) Distribution of average <em>cis</em> interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes. Cartoon on the top depicts all <em>cis</em> interactions of a locus (grey-coloured node), from which the average interaction frequency of grey-coloured locus was calculated. (b) Distribution of mean and variance of genomic distances between interacting sites for low and high noise genes. Cartoon on the top represents the genomic distance between <em>cis</em>-interacting sites of grey-coloured node. (c) Distribution of overlap ratio of chromatin interactions between <em>Hin</em>dIII and <em>Eco</em>RI HiC replicates for low and high noise genes. Horizontal dotted line represents the genome-wide median value. (d–f) Scatter plot between and experimentally determined chromatin fluctuations and (d) average <em>cis</em> interaction frequency, (e) expression noise, and (f) variance of genomic distances between interacting sites on yeast Chromosome 12. (g) Expression noise, average <em>cis</em> interaction frequency, variance of genomic distances, and chromatin fluctuations aligned along Chromosome 12. Data plotted in a–c and d–g are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S9 and S10</a></span>, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00507.jpeg?Expires=2147483647&amp;Signature=LUfwlEqlmAMJD5D0FHopcgeC0lKlXloZl8gNQuHKqei5eoanKluyG-TysbGHtdkQDQbB1A6tvz6nBhzGfsrKBrtDzKfz5MhKGu4-CFw54A7M-nRk~~r0P2HMDyf4JYiLVbzoAHUHnprjieTMBOaFHpXppWKpVMg3~OWGZ6PtE8Ytb3yj2jXbqDT-DhixK-vu6Qs64x1F2HWQrpR4dlHgmY8Ub4Yg8SYB55HJm4unMt89wWeeHthbO7d0cVdG0lCyuqdKL-s5S-0VDtVANxOtZICYQpTfN~X5w9w7w93eLvKCl8~05So6L5GHnzuxAfCNfSWSglwLrZF38Py8sadPjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of cis interactions with expression noise and chromatin fluctuations. (a) Distribution of average cis interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes. Cartoon on the top depicts all cis interactions of a locus (grey-coloured node), from which the average interaction frequency of grey-coloured locus was calculated. (b) Distribution of mean and variance of genomic distances between interacting sites for low and high noise genes. Cartoon on the top represents the genomic distance between cis-interacting sites of grey-coloured node. (c) Distribution of overlap ratio of chromatin interactions between HindIII and EcoRI HiC replicates for low and high noise genes. Horizontal dotted line represents the genome-wide median value. (d–f) Scatter plot between and experimentally determined chromatin fluctuations and (d) average cis interaction frequency, (e) expression noise, and (f) variance of genomic distances between interacting sites on yeast Chromosome 12. (g) Expression noise, average cis interaction frequency, variance of genomic distances, and chromatin fluctuations aligned along Chromosome 12. Data plotted in a–c and d–g are given in Supplementary Tables S9 and S10, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200100/dsw00507.jpeg" path-from-xml="dsw00507.jpeg" target="_blank">View large</a><a section="87200100" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00507.jpeg?Expires=2147483647&amp;Signature=jIF00-FDCY0SZ9j5C1ghhL3isdIu~SdPpFACyJcbCOodyUTNNy1HvP0d1v3E7VO10Gw28CNbxBTiVxdqab15xIZO7amheAIKqoFJjBfh0l8nZXYrxNcN0ZX0cDAnuI5vQGVopCzJegeLZVviN9qD3zKrwu6XV6knW8hRQuvrv41ePVFvGeDBjUqRVcQIkEL-AIeOMDw0bNZVpM7qAkACNoRGjj3TUxkcUBy-w6zp09IA7APyO2peiNwmBbOpV9ifsQQ396yLzzx8k-0kw~1EVVA68t5HQpIqysyejSqoXumCUeKemvw8fgLFvFDYMl9A-B5xqQEuXtQFrjsPNloTOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200100&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of <em>cis</em> interactions with expression noise and chromatin fluctuations. (a) Distribution of average <em>cis</em> interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes. Cartoon on the top depicts all <em>cis</em> interactions of a locus (grey-coloured node), from which the average interaction frequency of grey-coloured locus was calculated. (b) Distribution of mean and variance of genomic distances between interacting sites for low and high noise genes. Cartoon on the top represents the genomic distance between <em>cis</em>-interacting sites of grey-coloured node. (c) Distribution of overlap ratio of chromatin interactions between <em>Hin</em>dIII and <em>Eco</em>RI HiC replicates for low and high noise genes. Horizontal dotted line represents the genome-wide median value. (d–f) Scatter plot between and experimentally determined chromatin fluctuations and (d) average <em>cis</em> interaction frequency, (e) expression noise, and (f) variance of genomic distances between interacting sites on yeast Chromosome 12. (g) Expression noise, average <em>cis</em> interaction frequency, variance of genomic distances, and chromatin fluctuations aligned along Chromosome 12. Data plotted in a–c and d–g are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S9 and S10</a></span>, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To further scrutinize our observations, we performed following additional analyses. (i) We identified the regions of significantly greater average frequency of <em>cis</em> interactions from the interactions commonly present in the two HiC replicates. The distribution plot of average <em>cis</em> interaction frequency across yeast genome revealed a distinct population of regions having significantly greater interaction frequency than the rest (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>a). This is analogous to identifying ‘peaks’ in ChIP-Seq genomic tracks. The ‘peak’ nature of certain regions was also confirmed by plotting QQ plot of the average interaction frequency against a random normal distribution (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. 13</a></span>b and c). We extracted the genes located in these regions and compared the expression noise with the null distribution. Again, we observed significantly lower (<em>P</em>-value = 1e–03) noise for the genes located in the regions of high average interaction frequency. (ii) Lower diversity of interactions might also represent relatively stable state of a locus. We measured the variation in chromatin environment using coefficient of variation (<em>σ</em>/<em>μ</em>) and observed that the genes having greater variation in their chromatin interactions exhibited higher expression noise (<em>P</em>-value = 3e–02; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). (iii) More importantly, we compared the experimentally determined chromatin mobility of several loci<sup><a reveal-id="DSW005C53" data-open="DSW005C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSW005C54" data-open="DSW005C54" class="link link-ref link-reveal xref-bibr">54</a></sup> with their average <em>cis</em> interaction frequencies, variances of distances between interacting sites, and the expression noise (Fig. <a reveal-id="DSW005F7" data-open="DSW005F7" class="link link-reveal link-table xref-fig">7</a>d–g, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S15</a></span>). The analyses showed striking correlation of chromatin fluctuations with the expression noise (<em>ρ</em>=0.78, Fig. <a reveal-id="DSW005F7" data-open="DSW005F7" class="link link-reveal link-table xref-fig">7</a>d), moderate correlation with the variance of genomic distances between interacting loci (<em>ρ</em>=0.39, Fig. <a reveal-id="DSW005F7" data-open="DSW005F7" class="link link-reveal link-table xref-fig">7</a>e), and strong anticorrelation with the <em>cis</em> interaction frequency (<em>ρ</em>=−0.65, Fig. <a reveal-id="DSW005F7" data-open="DSW005F7" class="link link-reveal link-table xref-fig">7</a>f), strongly supporting our proposal that the chromatin mobility associates with the expression noise of the underlying genes.</p><p>Although we largely restrict all our claims to the budding yeast genome, we now present a few lines of evidence which suggest that minimization of expression noise through physically restrained environment might be a general evolutionary constraint shaping the genome organization in other radically different systems like bacteria and the mammalian cells. The expression noise data had been available for <em>Escherichia coli</em> too<sup><a reveal-id="DSW005C55" data-open="DSW005C55" class="link link-ref link-reveal xref-bibr">55</a></sup> and very recently genome organization data of <em>E. coli</em> have also been generated using Genome Conformation Capture technique.<sup><a reveal-id="DSW005C56" data-open="DSW005C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSW005C57" data-open="DSW005C57" class="link link-ref link-reveal xref-bibr">57</a></sup> It would, therefore, be interesting to test whether the proposed relationship between interaction frequency and expression noise also exists in Bacteria. We performed similar analyses in <em>E. coli</em> by taking normalized interaction frequency data for 10 kb bins as provided by Xie et al.,<sup><a reveal-id="DSW005C56" data-open="DSW005C56" class="link link-ref link-reveal xref-bibr">56</a></sup> number of essential genes and the maximal expression noise value in each bin. We found strikingly similar association among gene essentiality, expression noise, and average interaction frequency in <em>E. coli</em>, as in yeast (Fig. <a reveal-id="DSW005F8" data-open="DSW005F8" class="link link-reveal link-table xref-fig">8</a>a–c). This is an important observation highlighting an evolutionarily conserved association between expression noise and genome organization despite having radically distinct mechanisms of gene expression. Further, genome-wide single-cell gene expression data are now available for several model systems. We obtained one such dataset for mouse embryonic stem cells (mESCs), for which genome-wide chromatin interaction data was also available.<sup><a reveal-id="DSW005C35" data-open="DSW005C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSW005C58" data-open="DSW005C58" class="link link-ref link-reveal xref-bibr">58</a></sup> We calculated the noise in gene expression for mESCs and corrected for the transcript abundance using residuals of lowess fit between transcript abundance and noise. Again, the average interaction frequency of low noise genes was significantly greater than that of high noise genes (<em>P</em>-value = 3.0e–11; Fig. <a reveal-id="DSW005F8" data-open="DSW005F8" class="link link-reveal link-table xref-fig">8</a>d). Due to lack of whole-genome single-cell gene expression data along with HiC data on other cell types of mammalian genome, our observation on mESCs is presently not generalizable to different cell types in higher eukaryotes. </p>                        <a id="87200103" scrollto-destination="87200103"></a>
<div id="DSW005F8" data-id="DSW005F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00508.jpeg?Expires=2147483647&amp;Signature=Im0L8UCoK-DiTjyCPQJH-LXtoM3UfX2YEbsT5XUQqQOjOF6mi8iXvreOBJf~Zny2whIQYRn4k2vWrcC0udc7Xa6cjBfgNtkXL72GC7ZhlRXEoPGia2NHcxoyvFesZa~BjcwpZ-Ft7D09NkuQMtmttC1iVsN03nQTj~w7ydhlwEX7OJXCCVJ~VJFY8oWxTfmHflRBn6diZXftbMo7Xbofm-3pwh6gKH0GuKvibc5sF6c7X9k5FLt6RZJvUHYaU0myMMvn1vK9qJ04E~jZioj-JgdRd8Mka3nANYz6wbsLHbWbVpNiqJ5I3NMaaAA3opTjjCtd-53jJD-R6zyNHR6f0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of chromatin interactions with the expression noise in other systems (a) Average frequency of interactions among genomic bins having essential genes overlaid on the null distribution of average values across 1,000 random re-samplings in Escherichiacoli. (b) Violin plots of average interaction frequencies of loci having distinct densities of essential genes per 5 kb in E. coli genome. The categories &gt;0, &gt;1, and &gt;2 are inclusive and not mutually exclusive. (c) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in E. coli. (d) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in mESCs. (e) Distribution of number of RNAPII-associated chromatin interactions of low and high noise genes in mESCs. Mapped E. coli and mESC datasets are given in Supplementary Table S11 and S12. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00508.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200103/dsw00508.jpeg" path-from-xml="dsw00508.jpeg" target="_blank">View large</a><a section="87200103" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00508.jpeg?Expires=2147483647&amp;Signature=5GwdLmtcNx6U7WXBH8bfr0ct40tj5gcK83GsYt12inTYa2~DEphF1IK5KlbIuB1IO59gIyWoQ6wG~LyGDDztWpBNETb3DfvUWJHK8Xf8zUrhiaRNJlP4uVhV911PRsWyB9mMsob~QmRKWb3K0hFuJceM27AAzKT4qhKYg3a-vH-0BTMgyENNerd1Xn9GlcrFhOiGGRPs-nMLlAG3lV0sfwMzdNFOCl3bo1v5822NTuWDDiiXCBTPpxqL8iEIRBbZRORNvO9z0FUTH6nBz1GZ6iMuPQe1LPAeYJnvEs3MuoxFg9Zudq3r85HH3G~wlUBxD0nXxhupOxW-9Ts5gI7~yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200103&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00508.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of chromatin interactions with the expression noise in other systems (a) Average frequency of interactions among genomic bins having essential genes overlaid on the null distribution of average values across 1,000 random re-samplings in <em>Escherichia</em><em>coli</em>. (b) Violin plots of average interaction frequencies of loci having distinct densities of essential genes per 5 kb in <em>E. coli</em> genome. The categories &gt;0, &gt;1, and &gt;2 are inclusive and not mutually exclusive. (c) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in <em>E. coli</em>. (d) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in mESCs. (e) Distribution of number of RNAPII-associated chromatin interactions of low and high noise genes in mESCs. Mapped <em>E. coli</em> and mESC datasets are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11 and S12</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00508.jpeg?Expires=2147483647&amp;Signature=Im0L8UCoK-DiTjyCPQJH-LXtoM3UfX2YEbsT5XUQqQOjOF6mi8iXvreOBJf~Zny2whIQYRn4k2vWrcC0udc7Xa6cjBfgNtkXL72GC7ZhlRXEoPGia2NHcxoyvFesZa~BjcwpZ-Ft7D09NkuQMtmttC1iVsN03nQTj~w7ydhlwEX7OJXCCVJ~VJFY8oWxTfmHflRBn6diZXftbMo7Xbofm-3pwh6gKH0GuKvibc5sF6c7X9k5FLt6RZJvUHYaU0myMMvn1vK9qJ04E~jZioj-JgdRd8Mka3nANYz6wbsLHbWbVpNiqJ5I3NMaaAA3opTjjCtd-53jJD-R6zyNHR6f0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Association of chromatin interactions with the expression noise in other systems (a) Average frequency of interactions among genomic bins having essential genes overlaid on the null distribution of average values across 1,000 random re-samplings in Escherichiacoli. (b) Violin plots of average interaction frequencies of loci having distinct densities of essential genes per 5 kb in E. coli genome. The categories &gt;0, &gt;1, and &gt;2 are inclusive and not mutually exclusive. (c) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in E. coli. (d) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in mESCs. (e) Distribution of number of RNAPII-associated chromatin interactions of low and high noise genes in mESCs. Mapped E. coli and mESC datasets are given in Supplementary Table S11 and S12. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00508.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200103/dsw00508.jpeg" path-from-xml="dsw00508.jpeg" target="_blank">View large</a><a section="87200103" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00508.jpeg?Expires=2147483647&amp;Signature=5GwdLmtcNx6U7WXBH8bfr0ct40tj5gcK83GsYt12inTYa2~DEphF1IK5KlbIuB1IO59gIyWoQ6wG~LyGDDztWpBNETb3DfvUWJHK8Xf8zUrhiaRNJlP4uVhV911PRsWyB9mMsob~QmRKWb3K0hFuJceM27AAzKT4qhKYg3a-vH-0BTMgyENNerd1Xn9GlcrFhOiGGRPs-nMLlAG3lV0sfwMzdNFOCl3bo1v5822NTuWDDiiXCBTPpxqL8iEIRBbZRORNvO9z0FUTH6nBz1GZ6iMuPQe1LPAeYJnvEs3MuoxFg9Zudq3r85HH3G~wlUBxD0nXxhupOxW-9Ts5gI7~yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87200103&amp;ar=1745431&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw00508.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Association of chromatin interactions with the expression noise in other systems (a) Average frequency of interactions among genomic bins having essential genes overlaid on the null distribution of average values across 1,000 random re-samplings in <em>Escherichia</em><em>coli</em>. (b) Violin plots of average interaction frequencies of loci having distinct densities of essential genes per 5 kb in <em>E. coli</em> genome. The categories &gt;0, &gt;1, and &gt;2 are inclusive and not mutually exclusive. (c) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in <em>E. coli</em>. (d) Distribution of interaction frequencies for low (≤1st quartile) and high noise (≥3rd quartile) genes in mESCs. (e) Distribution of number of RNAPII-associated chromatin interactions of low and high noise genes in mESCs. Mapped <em>E. coli</em> and mESC datasets are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11 and S12</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The above analyses clearly strengthened our proposal that essential genes exhibit lesser variation in its chromatin environment and that such a property contributes in reducing the expression noise of engaged loci.</p>                            <h2 scrollto-destination=87200105 id="87200105" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>It is known that spatial positioning of genes in the nucleus is highly non-random. Genes positioned interior to the nucleus experience transcriptionally permissive environment, while the ones located near the nuclear periphery are generally repressed or lowly expressed.<sup><a reveal-id="DSW005C1" data-open="DSW005C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Whether or not spatial co-localization of genes plays an active role in regulating essential genomic functions is presently subjected to intense scrutiny. It has been presumed that genes co-localize to synchronize their transcriptional states.<sup><a reveal-id="DSW005C59" data-open="DSW005C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DSW005C60" data-open="DSW005C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Besides sporadic reports of co-expression of individual loci,<sup><a reveal-id="DSW005C19" data-open="DSW005C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSW005C61" data-open="DSW005C61" class="link link-ref link-reveal xref-bibr">61</a>,<a reveal-id="DSW005C62" data-open="DSW005C62" class="link link-ref link-reveal xref-bibr">62</a></sup> genome-wide proposal has been made in case of multi-gene complexes (relatively short-range intra-chromosomal interactions) in mammals<sup><a reveal-id="DSW005C34" data-open="DSW005C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and for both intra- and inter-chromosomal interactions in lower eukaryotes like yeast.<sup><a reveal-id="DSW005C37" data-open="DSW005C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSW005C38" data-open="DSW005C38" class="link link-ref link-reveal xref-bibr">38</a></sup> However, these studies are not subjected to comprehensive multivariate analyses and in the absence of other genomic variables, these reports do not present an unbiased view of possible functional constraints shaping the three-dimensional genome organization of eukaryotic genome. Here we analysed several functional attributes of yeast genome to identify the potential constraints of genome-wide chromatin interactions. Our analyses suggested that inter- and intra-chromosomal interactions were under distinct evolutionary constraints. While inter-chromosomal interactions were primarily associated with the coordination of Clb5-independent replication, intra-chromosomal interactions were constrained by the coordination of Clb5-dependent replication and minimization of expression noise of engaged loci. Correlation of interaction frequency with the co-expression of engaged genes was very weak. The correlation coefficient of similar magnitude was also reported earlier by others.<sup><a reveal-id="DSW005C37" data-open="DSW005C37" class="link link-ref link-reveal xref-bibr">37</a></sup> However, the authors compared the correlation coefficient with the average correlation of all possible gene pairs in the genome and found the correlation of 0.09 to be significantly higher. We argue that genome-wide average might not serve as an appropriate control, because the sample size of gene pairs will be disproportionally greater for the whole genome compared with the ones that are spatially proximal. Rewired contact networks and re-sampling-based approach can serve as better control. Re-assessment of reports claiming co-localization of co-regulated genes have been questioned elsewhere too.<sup><a reveal-id="DSW005C39" data-open="DSW005C39" class="link link-ref link-reveal xref-bibr">39</a></sup> More importantly, in the absence of other functional variables, it is not entirely justified to claim that co-expression if the major constraint of genome organization.</p><p>The specific association of <em>trans</em> and <em>cis</em> interaction frequency with early and the late replication, respectively, supports the notion of spatially segregated early and late replication factories, which had been proposed earlier by several authors.<sup><a reveal-id="DSW005C63  DSW005C65" data-open="DSW005C63  DSW005C65" class="link link-ref link-reveal xref-bibr">63–65</a></sup> Interestingly, a recent report has clearly shown that androgen-induced proximity between TMPRSS2 and ERG genes is due to AR-controlled replication, but not the transcription.<sup><a reveal-id="DSW005C66" data-open="DSW005C66" class="link link-ref link-reveal xref-bibr">66</a></sup> Further, the association of preferred domains of <em>cis</em> and <em>trans</em> interactions with the low and high nucleosome occupancy, respectively, supports the spatial segregation of open and closed chromatin. The observation that the early origins were flanked by well-positioned nucleosome around a narrow trough of nucleosome-depleted region was in line with earlier reports suggesting the role ORC mediated nucleosome positioning in establishment of pre-initiation complex around early origins.<sup><a reveal-id="DSW005C43" data-open="DSW005C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Relatively wider NDRs around late origins can be explained through following arguments: (i) Late firing of origins might need additional <em>cis</em> regulatory elements in the proximity. It is known that after recycling from early origins, replication factors are present in a very limited amount for late origins. Late origins, therefore, cannot afford the delay caused by the remodelling of nucleosomes to open up the binding sites of replication factors. Constitutively open chromatin and spatial co-localization of late origins therefore might help in efficient and rapid usage of replication factors and ensure the complete replication of the genome. (ii) Late origins are known to be present predominantly in the intergenic regions (IGRs) between convergent genes.<sup><a reveal-id="DSW005C67" data-open="DSW005C67" class="link link-ref link-reveal xref-bibr">67</a></sup> The abundant occurrence of transcription termination around convergent IGRs can interfere with the nucleosome stability at late origins. The nucleosome depletion around late origins can thus be an artefact of genomic neighbourhood. (iii) It is known that the radial positioning early origins in the nuclear space is mostly random, while late origins often localize towards nuclear periphery. The accessibility to sequences flaking late origins can help targeting the late origins to nuclear periphery.<sup><a reveal-id="DSW005C68" data-open="DSW005C68" class="link link-ref link-reveal xref-bibr">68</a></sup></p><p>The association of chromatin interactions with the mutation rate variation suggests a potential role of spatial genome organization in mutagenic mechanisms. Coordinated replication through spatial convergence might also ascribe similar susceptibility to genetic errors at engaged loci. Such properties had earlier been proposed in the context of cancer genomes.<sup><a reveal-id="DSW005C69" data-open="DSW005C69" class="link link-ref link-reveal xref-bibr">69</a>,<a reveal-id="DSW005C70" data-open="DSW005C70" class="link link-ref link-reveal xref-bibr">70</a></sup></p><p>Intra-chromosomal interactions, on the other hand, were also constrained by co-fitness of genes. It has been proposed that the gene pairs with higher co-fitness represent their close functional similarity. It can, therefore, partly be explained by the co-functionality of <em>cis</em>-interacting genes as shown in the Fig. <a reveal-id="DSW005F2" data-open="DSW005F2" class="link link-reveal link-table xref-fig">2</a>e. We further attempted to explain the link between fitness defect and genome organization by taking essential genes as example. We hypothesized that the high frequency intra-chromosomal interactions restrict the mobility of engaged loci, while low frequency interactions of a locus might represent a relatively mobile state of chromatin. The presumption that the high interaction frequency and the lower coefficient of variation represent spatially restrained loci in genome is also supported by comparison with the experimentally determined mobility of certain loci<sup><a reveal-id="DSW005C54" data-open="DSW005C54" class="link link-ref link-reveal xref-bibr">54</a></sup> (Fig. <a reveal-id="DSW005F8" data-open="DSW005F8" class="link link-reveal link-table xref-fig">8</a>d–g, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>). Restricted mobility of interacting loci might be important to reduce the transcriptional noise of interacting genes. We reconciled this hypothesis by taking the example of essential genes. Essential gene clusters are known to exhibit lower expression noise and consistently remain in transcriptionally permissive open chromatin state. Their non-randomly greater frequency of interactions aligns to our hypothesis. It was noteworthy that we did not consider the chromatin interactions between loci which were &lt;20 kb apart, and therefore, the linear clustering of essential genes would not have impacted this observation. Further, the association of low noise genes with greater mean interaction frequency of loci and <em>vice versa</em> across bacteria, yeast, and mouse genomes clearly suggests that spatially stable nature of genes assist in minimizing the stochastic transcriptional errors. Indeed, it has earlier been proposed that the long-range chromatin interactions might occur at the cost of increased expression noise.<sup><a reveal-id="DSW005C71" data-open="DSW005C71" class="link link-ref link-reveal xref-bibr">71</a></sup> To which sub-nuclear compartment these loci might tether to? We propose that for active genes, the most likely tethering foci would be transcription factories. Though this would need thorough scrutiny, we explored the RNAPII-associated chromatin interaction data generated through ChIA-PET technique for this purpose. Multi-gene complexes in that data have been proposed to be equivalent to transcription factories.<sup><a reveal-id="DSW005C34" data-open="DSW005C34" class="link link-ref link-reveal xref-bibr">34</a></sup> We asked whether abundance-corrected transcriptional noise associate with these complexes. We observed that the number of RNAPII-tethered promoter-to-promoter interactions, marking multi-gene complexes, was significantly higher for low noise genes compared with high noise genes (<em>P</em> &lt; 3e–11), suggesting that genes engaged in multi-gene complexes or transcription factories tend to have lower transcriptional noise (Fig. <a reveal-id="DSW005F8" data-open="DSW005F8" class="link link-reveal link-table xref-fig">8</a>e). Other nuclear compartments like nucleoli have been shown experimentally to constrain the chromatin movement, and their disruption leads to increased chromatin mobility.<sup><a reveal-id="DSW005C72" data-open="DSW005C72" class="link link-ref link-reveal xref-bibr">72</a></sup></p><p>Altogether, our unbiased approach of analysing distinct functional constraints provides a different perspective of evolution of three-dimensional genome organizations, which appears to be conserved in bacteria, yeast, and mouse. <em>Escherichia coli</em> genome is known to have an organization tightly linked with the replication,<sup><a reveal-id="DSW005C57" data-open="DSW005C57" class="link link-ref link-reveal xref-bibr">57</a></sup> and here we showed that it is also associated with gene essentiality and expression noise, very similar to what we observed in yeast. Finally, based on our observations, we propose a model for functional constraints shaping the <em>cis</em> and <em>trans</em> organization of chromatin. This is illustrated in the Fig. <a reveal-id="DSW005F9" data-open="DSW005F9" class="link link-reveal link-table xref-fig">9</a>. </p>                        <a id="87200111" scrollto-destination="87200111"></a>
<div id="DSW005F9" data-id="DSW005F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00509.jpeg?Expires=2147483647&Signature=WAOiVjIioLYmpTEG6l2wC-KlUH5Zl5qVWgD5JcSKbPd1~2gzSm~x6Ru46ghg049j19wWrnkh5ExnnCkkHHnnS7JFcivxTqtZUvRmjd3XkYKip-kNd1wjA9GbmY5glIHyoT7WlXZtdDz5PT42Tqt7yK4JzGr84hyh7GuHRL60OMYqSd9dWQZ9t-5nEGCwYMWLaT8f7QCFWGAOa0h9zqxH5yJSeMObyEfaZusIHSklAz1kYNZ~zRB5mEeuqz8Ys2Mj2sPKVs~p~KKV~HPGSnV57C3oGbBoDgd8yM-VReZdv6q7zU5N7jqKszFxF79aWew6xVKCOP4yaoMpIp0VS2OT6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of overall observations. At higher resolution, the yeast genome is organized into preferred domains of inter- and intra-chromosomal interactions, which is associated with the spatial segregation of early and late replication. As inferred from high average interaction frequency and chromatin fluctuations, essential gene loci are hypothesized to be physically restrained possibly by tethering to nuclear sub-compartments or foci. The proposed physically restrained nature of essential genes might be important to mitigate the epigenetic fluctuations or errors that predispose the genes to stochastic variation in expression. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00509.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200111/dsw00509.jpeg" path-from-xml="dsw00509.jpeg" target="_blank">View large</a><a section="87200111" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00509.jpeg?Expires=2147483647&Signature=CWH5-1zUTqzCSVSZemZS1-BOQ7oAbDw2gHCSMpvyW4JftidhTRE~bKSHFvKtNOQ2wYQZayhsheBmZdItO4P0Vn3Q3vvdVsAxNieQlnjCZ4YVQB97-XKc9ppbYvwKlX4NexSXZPeGEFSYvjlrOk8ViDFjf3BnmQGBbjg7YoZppO1BSZ~kiU6NHm~-wDxIhDYtJ5VfAtGVQvxRIdhp9emyK6ct08uFK1Lkr85Wvskikm9zP56q8nCNnh-h3o483ZV2DYOz8RFfOZ08WCBW0-~OKC73ZhwTJhxojtHdyHlhRUxdKaNaLWiTLD0dsCYKkvwZfaZdx79wjvXBIQZvH31wLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200111&ar=1745431&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00509.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of overall observations. At higher resolution, the yeast genome is organized into preferred domains of inter- and intra-chromosomal interactions, which is associated with the spatial segregation of early and late replication. As inferred from high average interaction frequency and chromatin fluctuations, essential gene loci are hypothesized to be physically restrained possibly by tethering to nuclear sub-compartments or foci. The proposed physically restrained nature of essential genes might be important to mitigate the epigenetic fluctuations or errors that predispose the genes to stochastic variation in expression. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW005F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/m_dsw00509.jpeg?Expires=2147483647&Signature=WAOiVjIioLYmpTEG6l2wC-KlUH5Zl5qVWgD5JcSKbPd1~2gzSm~x6Ru46ghg049j19wWrnkh5ExnnCkkHHnnS7JFcivxTqtZUvRmjd3XkYKip-kNd1wjA9GbmY5glIHyoT7WlXZtdDz5PT42Tqt7yK4JzGr84hyh7GuHRL60OMYqSd9dWQZ9t-5nEGCwYMWLaT8f7QCFWGAOa0h9zqxH5yJSeMObyEfaZusIHSklAz1kYNZ~zRB5mEeuqz8Ys2Mj2sPKVs~p~KKV~HPGSnV57C3oGbBoDgd8yM-VReZdv6q7zU5N7jqKszFxF79aWew6xVKCOP4yaoMpIp0VS2OT6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of overall observations. At higher resolution, the yeast genome is organized into preferred domains of inter- and intra-chromosomal interactions, which is associated with the spatial segregation of early and late replication. As inferred from high average interaction frequency and chromatin fluctuations, essential gene loci are hypothesized to be physically restrained possibly by tethering to nuclear sub-compartments or foci. The proposed physically restrained nature of essential genes might be important to mitigate the epigenetic fluctuations or errors that predispose the genes to stochastic variation in expression. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00509.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200111/dsw00509.jpeg" path-from-xml="dsw00509.jpeg" target="_blank">View large</a><a section="87200111" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw00509.jpeg?Expires=2147483647&Signature=CWH5-1zUTqzCSVSZemZS1-BOQ7oAbDw2gHCSMpvyW4JftidhTRE~bKSHFvKtNOQ2wYQZayhsheBmZdItO4P0Vn3Q3vvdVsAxNieQlnjCZ4YVQB97-XKc9ppbYvwKlX4NexSXZPeGEFSYvjlrOk8ViDFjf3BnmQGBbjg7YoZppO1BSZ~kiU6NHm~-wDxIhDYtJ5VfAtGVQvxRIdhp9emyK6ct08uFK1Lkr85Wvskikm9zP56q8nCNnh-h3o483ZV2DYOz8RFfOZ08WCBW0-~OKC73ZhwTJhxojtHdyHlhRUxdKaNaLWiTLD0dsCYKkvwZfaZdx79wjvXBIQZvH31wLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200111&ar=1745431&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00509.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of overall observations. At higher resolution, the yeast genome is organized into preferred domains of inter- and intra-chromosomal interactions, which is associated with the spatial segregation of early and late replication. As inferred from high average interaction frequency and chromatin fluctuations, essential gene loci are hypothesized to be physically restrained possibly by tethering to nuclear sub-compartments or foci. The proposed physically restrained nature of essential genes might be important to mitigate the epigenetic fluctuations or errors that predispose the genes to stochastic variation in expression. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h2 scrollto-destination=87200112 id="87200112" class="section-title" data-legacy-id=s5>5. Conclusion</h2>
<p>The study suggests that there are different set of functional constraints that shape intra- and inter-chromosomal interactomes in eukaryotes. Distinct spatial organization of early and late origins and the underlying coordination of replication strongly support the presence of discrete early and late replication factories. A tethered intra-chromosomal microenvironment might ascribe physical stability to a locus which can be important to reduce the transcriptional noise of engaged loci, particularly, the ones that are functionally indispensable. Therefore, coordinated replication and gene essentiality, not necessarily the co-expression of genes, seem major functional and evolutionary constraints shaping the three-dimensional genome organization of eukaryotes as well as of prokaryotes.</p>                            <h2 scrollto-destination=87200114 id="87200114" class="section-title" data-legacy-id=s6>Authors’ contribution</h2>
<p>A.S. performed most of the analyses. M.B. helped with the data analysis of mouse embryonic stem cells and performed some of the control analyses. K.S.S. conceived and supervised the project. K.S.S. has performed the statistical tests and drawn the figures.</p>                            <h2 scrollto-destination=87200116 id="87200116" class="section-title" data-legacy-id=s7>Conflict of interest statement</h2>
<p>None declared.</p>                            <h2 scrollto-destination=87200118 id="87200118" class="section-title" data-legacy-id=s9>Funding</h2>
<p>Funding to pay the Open Access publication charges for this article was provided by Ministry of Human Resource Development (MHRD), India.</p>                            <h2 scrollto-destination=87200120 id="87200120" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Authors acknowledge the financial support from Ministry of Human Resource and Development (MHRD), India.</p>                            <h2 scrollto-destination=87200122 id="87200122" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW005C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanabe</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Muller</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Neusser</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Evolutionary conservation of chromosome territory arrangements in cell nuclei from higher primates</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">99</div>, <div class="fpage">4424</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20conservation%20of%20chromosome%20territory%20arrangements%20in%20cell%20nuclei%20from%20higher%20primates&amp;author=H.%20Tanabe&amp;author=S.%20Muller&amp;author=M.%20Neusser&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=4424-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.072618599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.072618599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.072618599"> </span></div> </div></div></div></div></div><div content-id="DSW005C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krivega</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">A.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Enhancer and promoter interactions-long distance calls</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="volume">22</div>, <div class="fpage">79</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhancer%20and%20promoter%20interactions-long%20distance%20calls&amp;author=I.%20Krivega&amp;author=A.%20Dean&amp;publication_year=2012&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=22&amp;pages=79-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2011.11.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2011.11.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2011.11.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22169023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Petrascheck</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Escher</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Mahmoudi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Verrijzer</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Schaffner</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Barberis</div> <div class="given-names">A.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">DNA looping induced by a transcriptional enhancer in vivo</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">3743</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20looping%20induced%20by%20a%20transcriptional%20enhancer%20in%20vivo&amp;author=M.%20Petrascheck&amp;author=D.%20Escher&amp;author=T.%20Mahmoudi&amp;author=C.P.%20Verrijzer&amp;author=W.%20Schaffner&amp;author=A.%20Barberis&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=3743-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki689" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16002789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bulger</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Groudine</div> <div class="given-names">M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Enhancers: the abundance and function of regulatory sequences beyond promoters</div>, <div class="source">Dev. Biol.</div> , <div class="volume">339</div>, <div class="fpage">250</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhancers%3A%20the%20abundance%20and%20function%20of%20regulatory%20sequences%20beyond%20promoters&amp;author=M.%20Bulger&amp;author=M.%20Groudine&amp;publication_year=2010&amp;journal=Dev.%20Biol.&amp;volume=339&amp;pages=250-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2009.11.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ydbio.2009.11.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ydbio.2009.11.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20025863" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ghavi-Helm</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Klein</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Pakozdi</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Enhancer loops appear stable during development and are associated with paused polymerase</div>, <div class="source">Nature</div> , <div class="volume">512</div>, <div class="fpage">96</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhancer%20loops%20appear%20stable%20during%20development%20and%20are%20associated%20with%20paused%20polymerase&amp;author=Y.%20Ghavi-Helm&amp;author=F.A.%20Klein&amp;author=T.%20Pakozdi&amp;publication_year=2014&amp;journal=Nature&amp;volume=512&amp;pages=96-100" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25043061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nolis</div>
               <div class="given-names">I.K.</div>
            </div>, <div class="name string-name"><div class="surname">McKay</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Mantouvalou</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Lomvardas</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Merika</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Thanos</div> <div class="given-names">D.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Transcription factors mediate long-range enhancer-promoter interactions</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">106</div>, <div class="fpage">20222</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factors%20mediate%20long-range%20enhancer-promoter%20interactions&amp;author=I.K.%20Nolis&amp;author=D.J.%20McKay&amp;author=E.%20Mantouvalou&amp;author=S.%20Lomvardas&amp;author=M.%20Merika&amp;author=D.%20Thanos&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=20222-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0902454106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0902454106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0902454106"> </span></div> </div></div></div></div></div><div content-id="DSW005C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">West</div>
               <div class="given-names">A.G.</div>
            </div>, <div class="name string-name"><div class="surname">Fraser</div> <div class="given-names">P.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Remote control of gene transcription</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="volume">14</div>, <div class="fpage">R101</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Remote%20control%20of%20gene%20transcription&amp;author=A.G.%20West&amp;author=P.%20Fraser&amp;publication_year=2005&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=14&amp;pages=R101-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddi104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddi104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddi104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15809261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ong</div>
               <div class="given-names">C.T.</div>
            </div>, <div class="name string-name"><div class="surname">Corces</div> <div class="given-names">V.G.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Enhancer function: new insights into the regulation of tissue-specific gene expression</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">12</div>, <div class="fpage">283</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhancer%20function%3A%20new%20insights%20into%20the%20regulation%20of%20tissue-specific%20gene%20expression&amp;author=C.T.%20Ong&amp;author=V.G.%20Corces&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=283-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2957" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2957" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2957"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21358745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lanctot</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Cheutin</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cremer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cavalli</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Cremer</div> <div class="given-names">T.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Dynamic genome architecture in the nuclear space: regulation of gene expression in three dimensions</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">8</div>, <div class="fpage">104</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20genome%20architecture%20in%20the%20nuclear%20space%3A%20regulation%20of%20gene%20expression%20in%20three%20dimensions&amp;author=C.%20Lanctot&amp;author=T.%20Cheutin&amp;author=M.%20Cremer&amp;author=G.%20Cavalli&amp;author=T.%20Cremer&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=8&amp;pages=104-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17230197" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fraser</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Bickmore</div> <div class="given-names">W.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Nuclear organization of the genome and the potential for gene regulation</div>, <div class="source">Nature</div> , <div class="volume">447</div>, <div class="fpage">413</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20organization%20of%20the%20genome%20and%20the%20potential%20for%20gene%20regulation&amp;author=P.%20Fraser&amp;author=W.%20Bickmore&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=413-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05916" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05916" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05916"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17522674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Osborne</div>
               <div class="given-names">C.S.</div>
            </div>, <div class="name string-name"><div class="surname">Chakalova</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">K.E.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Active genes dynamically colocalize to shared sites of ongoing transcription</div>, <div class="source">Nat. Genet.</div> , <div class="volume">36</div>, <div class="fpage">1065</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Active%20genes%20dynamically%20colocalize%20to%20shared%20sites%20of%20ongoing%20transcription&amp;author=C.S.%20Osborne&amp;author=L.%20Chakalova&amp;author=K.E.%20Brown&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=1065-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1423" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1423" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1423"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15361872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eskiw</div>
               <div class="given-names">C.H.</div>
            </div>, <div class="name string-name"><div class="surname">Fraser</div> <div class="given-names">P.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Ultrastructural study of transcription factories in mouse erythroblasts</div>, <div class="source">J. Cell Sci.</div> , <div class="volume">124</div>, <div class="fpage">3676</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrastructural%20study%20of%20transcription%20factories%20in%20mouse%20erythroblasts&amp;author=C.H.%20Eskiw&amp;author=P.%20Fraser&amp;publication_year=2011&amp;journal=J.%20Cell%20Sci.&amp;volume=124&amp;pages=3676-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.087981" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.087981" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.087981"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22045738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martin</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Pombo</div> <div class="given-names">A.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Transcription factories: quantitative studies of nanostructures in the mammalian nucleus</div>, <div class="source">Chromosome Res.</div> , <div class="volume">11</div>, <div class="fpage">461</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factories%3A%20quantitative%20studies%20of%20nanostructures%20in%20the%20mammalian%20nucleus&amp;author=S.%20Martin&amp;author=A.%20Pombo&amp;publication_year=2003&amp;journal=Chromosome%20Res.&amp;volume=11&amp;pages=461-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1024926710797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1024926710797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1024926710797"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12971722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cook</div>
               <div class="given-names">P.R.</div>
            </div>
          </div>
          <div class="year">2010</div>, <div class="article-title">A model for all genomes: the role of transcription factories</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">395</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20model%20for%20all%20genomes%3A%20the%20role%20of%20transcription%20factories&amp;author=P.R.%20Cook&amp;publication_year=2010&amp;journal=J.%20Mol.%20Biol.&amp;volume=395&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2009.10.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2009.10.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2009.10.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19852969" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sutherland</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Bickmore</div> <div class="given-names">W.A.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Transcription factories: gene expression in unions?</div><div class="source">Nat. Rev. Genet.</div> , <div class="volume">10</div>, <div class="fpage">457</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factories%3A%20gene%20expression%20in%20unions%3F&amp;author=H.%20Sutherland&amp;author=W.A.%20Bickmore&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=457-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2592" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2592" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19506577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jackson</div>
               <div class="given-names">D.A.</div>
            </div>, <div class="name string-name"><div class="surname">Iborra</div> <div class="given-names">F.J.</div></div>, <div class="name string-name"><div class="surname">Manders</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Cook</div> <div class="given-names">P.R.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Numbers and organization of RNA polymerases, nascent transcripts, and transcription units in HeLa nuclei</div>, <div class="source">Mol. Biol. Cell</div> , <div class="volume">9</div>, <div class="fpage">1523</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Numbers%20and%20organization%20of%20RNA%20polymerases%2C%20nascent%20transcripts%2C%20and%20transcription%20units%20in%20HeLa%20nuclei&amp;author=D.A.%20Jackson&amp;author=F.J.%20Iborra&amp;author=E.M.%20Manders&amp;author=P.R.%20Cook&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Cell&amp;volume=9&amp;pages=1523-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.9.6.1523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.9.6.1523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.9.6.1523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9614191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dekker</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Rippe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Dekker</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kleckner</div> <div class="given-names">N.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Capturing chromosome conformation</div>, <div class="source">Science</div> , <div class="volume">295</div>, <div class="fpage">1306</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Capturing%20chromosome%20conformation&amp;author=J.%20Dekker&amp;author=K.%20Rippe&amp;author=M.%20Dekker&amp;author=N.%20Kleckner&amp;publication_year=2002&amp;journal=Science&amp;volume=295&amp;pages=1306-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1067799" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1067799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1067799"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11847345" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Tavoosidana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Sjolinder</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Circular chromosome conformation capture (4C) uncovers extensive networks of epigenetically regulated intra- and interchromosomal interactions</div>, <div class="source">Nat. Genet.</div> , <div class="volume">38</div>, <div class="fpage">1341</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20chromosome%20conformation%20capture%20(4C)%20uncovers%20extensive%20networks%20of%20epigenetically%20regulated%20intra-%20and%20interchromosomal%20interactions&amp;author=Z.%20Zhao&amp;author=G.%20Tavoosidana&amp;author=M.%20Sjolinder&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=1341-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17033624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fullwood</div>
               <div class="given-names">M.J.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Y.F.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">An oestrogen-receptor-alpha-bound human chromatin interactome</div>, <div class="source">Nature</div> , <div class="volume">462</div>, <div class="fpage">58</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20oestrogen-receptor-alpha-bound%20human%20chromatin%20interactome&amp;author=M.J.%20Fullwood&amp;author=M.H.%20Liu&amp;author=Y.F.%20Pan&amp;publication_year=2009&amp;journal=Nature&amp;volume=462&amp;pages=58-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19890323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dostie</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Richmond</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Arnaout</div> <div class="given-names">R.A.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Chromosome Conformation Capture Carbon Copy (5C): a massively parallel solution for mapping interactions between genomic elements</div>, <div class="source">Genome Res.</div> , <div class="volume">16</div>, <div class="fpage">1299</div>–<div class="lpage">309</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20Conformation%20Capture%20Carbon%20Copy%20(5C)%3A%20a%20massively%20parallel%20solution%20for%20mapping%20interactions%20between%20genomic%20elements&amp;author=J.%20Dostie&amp;author=T.A.%20Richmond&amp;author=R.A.%20Arnaout&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1299-309" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5571506" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5571506" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5571506"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16954542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lieberman-Aiden</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">van Berkum</div> <div class="given-names">N.L.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Comprehensive mapping of long-range interactions reveals folding principles of the human genome</div>, <div class="source">Science</div> , <div class="volume">326</div>, <div class="fpage">289</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&amp;author=E.%20Lieberman-Aiden&amp;author=N.L.%20van%20Berkum&amp;author=L.%20Williams&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=289-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1181369" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181369" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1181369"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19815776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duan</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Andronescu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schutz</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">A three-dimensional model of the yeast genome</div>, <div class="source">Nature</div> , <div class="volume">465</div>, <div class="fpage">363</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20three-dimensional%20model%20of%20the%20yeast%20genome&amp;author=Z.%20Duan&amp;author=M.%20Andronescu&amp;author=K.%20Schutz&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=363-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08973" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08973" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08973"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436457" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Roqueiro</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Genome-wide analysis of local chromatin packing in Arabidopsis thaliana</div>, <div class="source">Genome Res.</div> , <div class="volume">25</div>, <div class="fpage">246</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20local%20chromatin%20packing%20in%20Arabidopsis%20thaliana&amp;author=C.%20Wang&amp;author=C.%20Liu&amp;author=D.%20Roqueiro&amp;publication_year=2015&amp;journal=Genome%20Res.&amp;volume=25&amp;pages=246-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170332.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170332.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170332.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25367294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sexton</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Yaffe</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kenigsberg</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Three-dimensional folding and functional organization principles of the Drosophila genome</div>, <div class="source">Cell</div> , <div class="volume">148</div>, <div class="fpage">458</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three-dimensional%20folding%20and%20functional%20organization%20principles%20of%20the%20Drosophila%20genome&amp;author=T.%20Sexton&amp;author=E.%20Yaffe&amp;author=E.%20Kenigsberg&amp;publication_year=2012&amp;journal=Cell&amp;volume=148&amp;pages=458-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.01.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.01.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.01.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22265598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Umbarger</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Toro</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">M.A.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">The three-dimensional architecture of a bacterial genome and its alteration by genetic perturbation</div>, <div class="source">Mol. Cell</div> , <div class="volume">44</div>, <div class="fpage">252</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20three-dimensional%20architecture%20of%20a%20bacterial%20genome%20and%20its%20alteration%20by%20genetic%20perturbation&amp;author=M.A.%20Umbarger&amp;author=E.%20Toro&amp;author=M.A.%20Wright&amp;publication_year=2011&amp;journal=Mol.%20Cell&amp;volume=44&amp;pages=252-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2011.09.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2011.09.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2011.09.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22017872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanizawa</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Iwasaki</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Mapping of long-range associations throughout the fission yeast genome reveals global genome organization linked to transcriptional regulation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">8164</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20long-range%20associations%20throughout%20the%20fission%20yeast%20genome%20reveals%20global%20genome%20organization%20linked%20to%20transcriptional%20regulation&amp;author=H.%20Tanizawa&amp;author=O.%20Iwasaki&amp;author=A.%20Tanaka&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=8164-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21030438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dixon</div>
               <div class="given-names">J.R.</div>
            </div>, <div class="name string-name"><div class="surname">Selvaraj</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yue</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Topological domains in mammalian genomes identified by analysis of chromatin interactions</div>, <div class="source">Nature</div> , <div class="volume">485</div>, <div class="fpage">376</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Topological%20domains%20in%20mammalian%20genomes%20identified%20by%20analysis%20of%20chromatin%20interactions&amp;author=J.R.%20Dixon&amp;author=S.%20Selvaraj&amp;author=F.%20Yue&amp;publication_year=2012&amp;journal=Nature&amp;volume=485&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dixon</div>
               <div class="given-names">J.R.</div>
            </div>, <div class="name string-name"><div class="surname">Jung</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Selvaraj</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Chromatin architecture reorganization during stem cell differentiation</div>, <div class="source">Nature</div> , <div class="volume">518</div>, <div class="fpage">331</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20architecture%20reorganization%20during%20stem%20cell%20differentiation&amp;author=J.R.%20Dixon&amp;author=I.%20Jung&amp;author=S.%20Selvaraj&amp;publication_year=2015&amp;journal=Nature&amp;volume=518&amp;pages=331-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature14222" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature14222" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature14222"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25693564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ciabrelli</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Cavalli</div> <div class="given-names">G.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Chromatin-driven behavior of topologically associating domains</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">427</div>, <div class="fpage">608</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin-driven%20behavior%20of%20topologically%20associating%20domains&amp;author=F.%20Ciabrelli&amp;author=G.%20Cavalli&amp;publication_year=2015&amp;journal=J.%20Mol.%20Biol.&amp;volume=427&amp;pages=608-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2014.09.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2014.09.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2014.09.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25280896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pope</div>
               <div class="given-names">B.D.</div>
            </div>, <div class="name string-name"><div class="surname">Ryba</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Dileep</div> <div class="given-names">V.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Topologically associating domains are stable units of replication-timing regulation</div>, <div class="source">Nature</div> , <div class="volume">515</div>, <div class="fpage">402</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Topologically%20associating%20domains%20are%20stable%20units%20of%20replication-timing%20regulation&amp;author=B.D.%20Pope&amp;author=T.%20Ryba&amp;author=V.%20Dileep&amp;publication_year=2014&amp;journal=Nature&amp;volume=515&amp;pages=402-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">C.H.</div></div>, <div class="name string-name"><div class="surname">Birnbaum</div> <div class="given-names">R.Y.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Chromatin connectivity maps reveal dynamic promoter-enhancer long-range associations</div>, <div class="source">Nature</div> , <div class="volume">504</div>, <div class="fpage">306</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20connectivity%20maps%20reveal%20dynamic%20promoter-enhancer%20long-range%20associations&amp;author=Y.%20Zhang&amp;author=C.H.%20Wong&amp;author=R.Y.%20Birnbaum&amp;publication_year=2013&amp;journal=Nature&amp;volume=504&amp;pages=306-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12716" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12716" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12716"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24213634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jin</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Dixon</div> <div class="given-names">J.R.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">A high-resolution map of the three-dimensional chromatin interactome in human cells</div>, <div class="source">Nature</div> , <div class="volume">503</div>, <div class="fpage">290</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%20map%20of%20the%20three-dimensional%20chromatin%20interactome%20in%20human%20cells&amp;author=F.%20Jin&amp;author=Y.%20Li&amp;author=J.R.%20Dixon&amp;publication_year=2013&amp;journal=Nature&amp;volume=503&amp;pages=290-4" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24141950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sanyal</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Lajoie</div> <div class="given-names">B.R.</div></div>, <div class="name string-name"><div class="surname">Jain</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Dekker</div> <div class="given-names">J.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">The long-range interaction landscape of gene promoters</div>, <div class="source">Nature</div> , <div class="volume">489</div>, <div class="fpage">109</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20long-range%20interaction%20landscape%20of%20gene%20promoters&amp;author=A.%20Sanyal&amp;author=B.R.%20Lajoie&amp;author=G.%20Jain&amp;author=J.%20Dekker&amp;publication_year=2012&amp;journal=Nature&amp;volume=489&amp;pages=109-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11279"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Ruan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Auerbach</div> <div class="given-names">R.K.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Extensive promoter-centered chromatin interactions provide a topological basis for transcription regulation</div>, <div class="source">Cell</div> , <div class="volume">148</div>, <div class="fpage">84</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20promoter-centered%20chromatin%20interactions%20provide%20a%20topological%20basis%20for%20transcription%20regulation&amp;author=G.%20Li&amp;author=X.%20Ruan&amp;author=R.K.%20Auerbach&amp;publication_year=2012&amp;journal=Cell&amp;volume=148&amp;pages=84-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.12.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.12.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.12.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22265404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schoenfelder</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Furlan-Magaril</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mifsud</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">The pluripotent regulatory circuitry connecting promoters to their long-range interacting elements</div>, <div class="source">Genome Res.</div> , <div class="volume">25</div>, <div class="fpage">582</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pluripotent%20regulatory%20circuitry%20connecting%20promoters%20to%20their%20long-range%20interacting%20elements&amp;author=S.%20Schoenfelder&amp;author=M.%20Furlan-Magaril&amp;author=B.%20Mifsud&amp;publication_year=2015&amp;journal=Genome%20Res.&amp;volume=25&amp;pages=582-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.185272.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.185272.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.185272.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25752748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chepelev</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wangsa</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">K.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Characterization of genome-wide enhancer-promoter interactions reveals co-expression of interacting genes and modes of higher order chromatin organization</div>, <div class="source">Cell Res.</div> , <div class="volume">22</div>, <div class="fpage">490</div>–<div class="lpage">503</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20genome-wide%20enhancer-promoter%20interactions%20reveals%20co-expression%20of%20interacting%20genes%20and%20modes%20of%20higher%20order%20chromatin%20organization&amp;author=I.%20Chepelev&amp;author=G.%20Wei&amp;author=D.%20Wangsa&amp;author=Q.%20Tang&amp;author=K.%20Zhao&amp;publication_year=2012&amp;journal=Cell%20Res.&amp;volume=22&amp;pages=490-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cr.2012.15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcr.2012.15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcr.2012.15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22270183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Homouz</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Kudlicki</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">The 3D organization of the yeast genome correlates with co-expression and reflects functional relations between genes</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e54699</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%203D%20organization%20of%20the%20yeast%20genome%20correlates%20with%20co-expression%20and%20reflects%20functional%20relations%20between%20genes&amp;author=D.%20Homouz&amp;author=A.S.%20Kudlicki&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e54699" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0054699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0054699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0054699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dai</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Dai</div> <div class="given-names">X.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Nuclear colocalization of transcription factor target genes strengthens coregulation in yeast</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">27</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20colocalization%20of%20transcription%20factor%20target%20genes%20strengthens%20coregulation%20in%20yeast&amp;author=Z.%20Dai&amp;author=X.%20Dai&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=27-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr689" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21880591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Witten</div>
               <div class="given-names">D.M.</div>
            </div>, <div class="name string-name"><div class="surname">Noble</div> <div class="given-names">W.S.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">On the assessment of statistical significance of three-dimensional colocalization of sets of genomic elements</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">3849</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20assessment%20of%20statistical%20significance%20of%20three-dimensional%20colocalization%20of%20sets%20of%20genomic%20elements&amp;author=D.M.%20Witten&amp;author=W.S.%20Noble&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=3849-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22266657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rutledge</div>
               <div class="given-names">M.T.</div>
            </div>, <div class="name string-name"><div class="surname">Russo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Belton</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Dekker</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Broach</div> <div class="given-names">J.R.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">The yeast genome undergoes significant topological reorganization in quiescence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">43</div>, <div class="fpage">8299</div>–<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20yeast%20genome%20undergoes%20significant%20topological%20reorganization%20in%20quiescence&amp;author=M.T.%20Rutledge&amp;author=M.%20Russo&amp;author=J.M.%20Belton&amp;author=J.%20Dekker&amp;author=J.R.%20Broach&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res.&amp;volume=43&amp;pages=8299-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26202961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McCune</div>
               <div class="given-names">H.J.</div>
            </div>, <div class="name string-name"><div class="surname">Danielson</div> <div class="given-names">L.S.</div></div>, <div class="name string-name"><div class="surname">Alvino</div> <div class="given-names">G.M.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">The temporal program of chromosome replication: genomewide replication in clb5{Delta} Saccharomyces cerevisiae</div>, <div class="source">Genetics</div> , <div class="volume">180</div>, <div class="fpage">1833</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20temporal%20program%20of%20chromosome%20replication%3A%20genomewide%20replication%20in%20clb5%7BDelta%7D%20Saccharomyces%20cerevisiae&amp;author=H.J.%20McCune&amp;author=L.S.%20Danielson&amp;author=G.M.%20Alvino&amp;publication_year=2008&amp;journal=Genetics&amp;volume=180&amp;pages=1833-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.094359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.094359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.094359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18832352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lang</div>
               <div class="given-names">G.I.</div>
            </div>, <div class="name string-name"><div class="surname">Murray</div> <div class="given-names">A.W.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Mutation rates across budding yeast chromosome VI are correlated with replication timing</div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">3</div>, <div class="fpage">799</div>–<div class="lpage">811</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20rates%20across%20budding%20yeast%20chromosome%20VI%20are%20correlated%20with%20replication%20timing&amp;author=G.I.%20Lang&amp;author=A.W.%20Murray&amp;publication_year=2011&amp;journal=Genome%20Biol.%20Evol.&amp;volume=3&amp;pages=799-811" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evr054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevr054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevr054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21666225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lipford</div>
               <div class="given-names">J.R.</div>
            </div>, <div class="name string-name"><div class="surname">Bell</div> <div class="given-names">S.P.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Nucleosomes positioned by ORC facilitate the initiation of DNA replication</div>, <div class="source">Mol. Cell</div> , <div class="volume">7</div>, <div class="fpage">21</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosomes%20positioned%20by%20ORC%20facilitate%20the%20initiation%20of%20DNA%20replication&amp;author=J.R.%20Lipford&amp;author=S.P.%20Bell&amp;publication_year=2001&amp;journal=Mol.%20Cell&amp;volume=7&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1097-2765(01)00151-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1097-2765(01)00151-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1097-2765(01)00151-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11172708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Friedman</div>
               <div class="given-names">K.L.</div>
            </div>, <div class="name string-name"><div class="surname">Diller</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Ferguson</div> <div class="given-names">B.M.</div></div>, <div class="name string-name"><div class="surname">Nyland</div> <div class="given-names">S.V.</div></div>, <div class="name string-name"><div class="surname">Brewer</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Fangman</div> <div class="given-names">W.L.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Multiple determinants controlling activation of yeast replication origins late in S phase</div>, <div class="source">Genes Dev.</div> , <div class="volume">10</div>, <div class="fpage">1595</div>–<div class="lpage">607</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20determinants%20controlling%20activation%20of%20yeast%20replication%20origins%20late%20in%20S%20phase&amp;author=K.L.%20Friedman&amp;author=J.D.%20Diller&amp;author=B.M.%20Ferguson&amp;author=S.V.%20Nyland&amp;author=B.J.%20Brewer&amp;author=W.L.%20Fangman&amp;publication_year=1996&amp;journal=Genes%20Dev.&amp;volume=10&amp;pages=1595-607" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.10.13.1595" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.10.13.1595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.10.13.1595"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8682291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Berbenetz</div>
               <div class="given-names">N.M.</div>
            </div>, <div class="name string-name"><div class="surname">Nislow</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">G.W.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Diversity of eukaryotic DNA replication origins revealed by genome-wide analysis of chromatin structure</div>, <div class="source">PLoS Genet.</div> , <div class="volume">6</div>, <div class="fpage">e1001092</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20eukaryotic%20DNA%20replication%20origins%20revealed%20by%20genome-wide%20analysis%20of%20chromatin%20structure&amp;author=N.M.%20Berbenetz&amp;author=C.%20Nislow&amp;author=G.W.%20Brown&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1001092" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20824081" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Batada</div>
               <div class="given-names">N.N.</div>
            </div>, <div class="name string-name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Evolution of chromosome organization driven by selection for reduced gene expression noise</div>, <div class="source">Nat. Genet.</div> , <div class="volume">39</div>, <div class="fpage">945</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20chromosome%20organization%20driven%20by%20selection%20for%20reduced%20gene%20expression%20noise&amp;author=N.N.%20Batada&amp;author=L.D.%20Hurst&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=945-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng2071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng2071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng2071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17660811" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blainey</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Krzywinski</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Altman</div> <div class="given-names">N.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Points of significance: replication</div>, <div class="source">Nat. Methods</div> , <div class="volume">11</div>, <div class="fpage">879</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Points%20of%20significance%3A%20replication&amp;author=P.%20Blainey&amp;author=M.%20Krzywinski&amp;author=N.%20Altman&amp;publication_year=2014&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=879-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25317452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">De</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2011</div>, <div class="article-title">Somatic mosaicism in healthy human tissues</div>, <div class="source">Trends Genet.</div> , <div class="volume">27</div>, <div class="fpage">217</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Somatic%20mosaicism%20in%20healthy%20human%20tissues&amp;author=S.%20De&amp;publication_year=2011&amp;journal=Trends%20Genet.&amp;volume=27&amp;pages=217-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2011.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2011.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2011.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21496937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Herndon</div>
               <div class="given-names">L.A.</div>
            </div>, <div class="name string-name"><div class="surname">Schmeissner</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Dudaronek</div> <div class="given-names">J.M.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Stochastic and genetic factors influence tissue-specific decline in ageing C. elegans</div>, <div class="source">Nature</div> , <div class="volume">419</div>, <div class="fpage">808</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochastic%20and%20genetic%20factors%20influence%20tissue-specific%20decline%20in%20ageing%20C.%20elegans&amp;author=L.A.%20Herndon&amp;author=P.J.%20Schmeissner&amp;author=J.M.%20Dudaronek&amp;publication_year=2002&amp;journal=Nature&amp;volume=419&amp;pages=808-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12397350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Molloy</div>
               <div class="given-names">M.P.</div>
            </div>, <div class="name string-name"><div class="surname">Brzezinski</div> <div class="given-names">E.E.</div></div>, <div class="name string-name"><div class="surname">Hang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">McDowell</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">VanBogelen</div> <div class="given-names">R.A.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Overcoming technical variation and biological variation in quantitative proteomics</div>, <div class="source">Proteomics</div> , <div class="volume">3</div>, <div class="fpage">1912</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overcoming%20technical%20variation%20and%20biological%20variation%20in%20quantitative%20proteomics&amp;author=M.P.%20Molloy&amp;author=E.E.%20Brzezinski&amp;author=J.%20Hang&amp;author=M.T.%20McDowell&amp;author=R.A.%20VanBogelen&amp;publication_year=2003&amp;journal=Proteomics&amp;volume=3&amp;pages=1912-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14625853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fernandez</div>
               <div class="given-names">L.C.</div>
            </div>, <div class="name string-name"><div class="surname">Torres</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Real</div> <div class="given-names">F.X.</div></div></div>
          <div class="year">2016</div>, <div class="article-title">Somatic mosaicism: on the road to cancer</div>, <div class="source">Nat. Rev. Cancer</div> , <div class="volume">16</div>, <div class="fpage">43</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Somatic%20mosaicism%3A%20on%20the%20road%20to%20cancer&amp;author=L.C.%20Fernandez&amp;author=M.%20Torres&amp;author=F.X.%20Real&amp;publication_year=2016&amp;journal=Nat.%20Rev.%20Cancer&amp;volume=16&amp;pages=43-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrc.2015.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrc.2015.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrc.2015.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26678315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chalancon</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Ravarani</div> <div class="given-names">C.N.</div></div>, <div class="name string-name"><div class="surname">Balaji</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Interplay between gene expression noise and regulatory network architecture</div>, <div class="source">Trends Genet.</div> , <div class="volume">28</div>, <div class="fpage">221</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interplay%20between%20gene%20expression%20noise%20and%20regulatory%20network%20architecture&amp;author=G.%20Chalancon&amp;author=C.N.%20Ravarani&amp;author=S.%20Balaji&amp;publication_year=2012&amp;journal=Trends%20Genet.&amp;volume=28&amp;pages=221-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2012.01.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2012.01.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2012.01.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22365642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Albert</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Mathon</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Shukla</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Systematic characterization of the conformation and dynamics of budding yeast chromosome XII</div>, <div class="source">J. Cell Biol.</div> , <div class="volume">202</div>, <div class="fpage">201</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20characterization%20of%20the%20conformation%20and%20dynamics%20of%20budding%20yeast%20chromosome%20XII&amp;author=B.%20Albert&amp;author=J.%20Mathon&amp;author=A.%20Shukla&amp;publication_year=2013&amp;journal=J.%20Cell%20Biol.&amp;volume=202&amp;pages=201-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.201208186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.201208186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.201208186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23878273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Heun</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Laroche</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shimada</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Furrer</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gasser</div> <div class="given-names">S.M.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Chromosome dynamics in the yeast interphase nucleus</div>, <div class="source">Science</div> , <div class="volume">294</div>, <div class="fpage">2181</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20dynamics%20in%20the%20yeast%20interphase%20nucleus&amp;author=P.%20Heun&amp;author=T.%20Laroche&amp;author=K.%20Shimada&amp;author=P.%20Furrer&amp;author=S.M.%20Gasser&amp;publication_year=2001&amp;journal=Science&amp;volume=294&amp;pages=2181-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1065366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1065366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1065366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11739961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Silander</div>
               <div class="given-names">O.K.</div>
            </div>, <div class="name string-name"><div class="surname">Nikolic</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Zaslaver</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">A genome-wide analysis of promoter-mediated phenotypic noise in Escherichia coli</div>, <div class="source">PLoS Genet.</div> , <div class="volume">8</div>, <div class="fpage">e1002443</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20analysis%20of%20promoter-mediated%20phenotypic%20noise%20in%20Escherichia%20coli&amp;author=O.K.%20Silander&amp;author=N.%20Nikolic&amp;author=A.%20Zaslaver&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002443" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002443" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002443" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002443"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22275871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xie</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">L.Y.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Q.Y.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Spatial features for Escherichia coli genome organization</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatial%20features%20for%20Escherichia%20coli%20genome%20organization&amp;author=T.%20Xie&amp;author=L.Y.%20Fu&amp;author=Q.Y.%20Yang&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1258-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1258-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1258-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25652224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cagliero</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Grand</div> <div class="given-names">R.S.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">M.B.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">O'Sullivan</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Genome conformation capture reveals that the Escherichia coli chromosome is organized by replication and transcription</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">6058</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20conformation%20capture%20reveals%20that%20the%20Escherichia%20coli%20chromosome%20is%20organized%20by%20replication%20and%20transcription&amp;author=C.%20Cagliero&amp;author=R.S.%20Grand&amp;author=M.B.%20Jones&amp;author=D.J.%20Jin&amp;author=J.M.%20O'Sullivan&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=6058-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt325" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt325" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt325"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23632166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tang</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Barbacioru</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Tracing the derivation of embryonic stem cells from the inner cell mass by single-cell RNA-Seq analysis</div>, <div class="source">Cell Stem Cell</div> , <div class="volume">6</div>, <div class="fpage">468</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracing%20the%20derivation%20of%20embryonic%20stem%20cells%20from%20the%20inner%20cell%20mass%20by%20single-cell%20RNA-Seq%20analysis&amp;author=F.%20Tang&amp;author=C.%20Barbacioru&amp;author=S.%20Bao&amp;publication_year=2010&amp;journal=Cell%20Stem%20Cell&amp;volume=6&amp;pages=468-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.stem.2010.03.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.stem.2010.03.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.stem.2010.03.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20452321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rajapakse</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Perlman</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Scalzo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kooperberg</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Groudine</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kosak</div> <div class="given-names">S.T.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">The emergence of lineage-specific chromosomal topologies from coordinate gene regulation</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">106</div>, <div class="fpage">6679</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20emergence%20of%20lineage-specific%20chromosomal%20topologies%20from%20coordinate%20gene%20regulation&amp;author=I.%20Rajapakse&amp;author=M.D.%20Perlman&amp;author=D.%20Scalzo&amp;author=C.%20Kooperberg&amp;author=M.%20Groudine&amp;author=S.T.%20Kosak&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=6679-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0900986106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0900986106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0900986106"> </span></div> </div></div></div></div></div><div content-id="DSW005C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Szczepinska</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Pawlowski</div> <div class="given-names">K.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Genomic positions of co-expressed genes: echoes of chromosome organisation in gene expression data</div>, <div class="source">BMC Res. Notes</div> , <div class="volume">6</div>, <div class="fpage">229</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20positions%20of%20co-expressed%20genes%3A%20echoes%20of%20chromosome%20organisation%20in%20gene%20expression%20data&amp;author=T.%20Szczepinska&amp;author=K.%20Pawlowski&amp;publication_year=2013&amp;journal=BMC%20Res.%20Notes&amp;volume=6&amp;pages=229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-6-229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-6-229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-6-229"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23764369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schoenfelder</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Sexton</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Chakalova</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Preferential associations between co-regulated genes reveal a transcriptional interactome in erythroid cells</div>, <div class="source">Nat. Genet.</div> , <div class="volume">42</div>, <div class="fpage">53</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preferential%20associations%20between%20co-regulated%20genes%20reveal%20a%20transcriptional%20interactome%20in%20erythroid%20cells&amp;author=S.%20Schoenfelder&amp;author=T.%20Sexton&amp;author=L.%20Chakalova&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=53-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.496" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.496" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.496"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Di Stefano</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Rosa</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Belcastro</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">di Bernardo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Micheletti</div> <div class="given-names">C.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Colocalization of coregulated genes: a steered molecular dynamics study of human chromosome 19</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">9</div>, <div class="fpage">e1003019</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Colocalization%20of%20coregulated%20genes%3A%20a%20steered%20molecular%20dynamics%20study%20of%20human%20chromosome%2019&amp;author=M.%20Di%20Stefano&amp;author=A.%20Rosa&amp;author=V.%20Belcastro&amp;author=D.%20di%20Bernardo&amp;author=C.%20Micheletti&amp;publication_year=2013&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=9&amp;pages=e1003019" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1003019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dimitrova</div>
               <div class="given-names">D.S.</div>
            </div>, <div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">D.M.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Temporally coordinated assembly and disassembly of replication factories in the absence of DNA synthesis</div>, <div class="source">Nat. Cell Biol.</div> , <div class="volume">2</div>, <div class="fpage">686</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temporally%20coordinated%20assembly%20and%20disassembly%20of%20replication%20factories%20in%20the%20absence%20of%20DNA%20synthesis&amp;author=D.S.%20Dimitrova&amp;author=D.M.%20Gilbert&amp;publication_year=2000&amp;journal=Nat.%20Cell%20Biol.&amp;volume=2&amp;pages=686-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35036309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35036309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35036309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11025658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saner</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Karschau</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Natsume</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Stochastic association of neighboring replicons creates replication factories in budding yeast</div>, <div class="source">J. Cell Biol.</div> , <div class="volume">202</div>, <div class="fpage">1001</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochastic%20association%20of%20neighboring%20replicons%20creates%20replication%20factories%20in%20budding%20yeast&amp;author=N.%20Saner&amp;author=J.%20Karschau&amp;author=T.%20Natsume&amp;publication_year=2013&amp;journal=J.%20Cell%20Biol.&amp;volume=202&amp;pages=1001-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.201306143" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.201306143" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.201306143"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24062338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gondor</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Ohlsson</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Replication timing and epigenetic reprogramming of gene expression: a two-way relationship?</div><div class="source">Nat. Rev. Genet.</div> , <div class="volume">10</div>, <div class="fpage">269</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication%20timing%20and%20epigenetic%20reprogramming%20of%20gene%20expression%3A%20a%20two-way%20relationship%3F&amp;author=A.%20Gondor&amp;author=R.%20Ohlsson&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=269-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19274048" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Coll-Bastus</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Young</div> <div class="given-names">B.D.</div></div>, <div class="name string-name"><div class="surname">Sheer</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Y.J.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">DNA replication-dependent induction of gene proximity by androgen</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="volume">24</div>, <div class="fpage">963</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20replication-dependent%20induction%20of%20gene%20proximity%20by%20androgen&amp;author=N.%20Coll-Bastus&amp;author=X.%20Mao&amp;author=B.D.%20Young&amp;author=D.%20Sheer&amp;author=Y.J.%20Lu&amp;publication_year=2015&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=24&amp;pages=963-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddu508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddu508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddu508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25281662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Soriano</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Morafraile</div> <div class="given-names">E.C.</div></div>, <div class="name string-name"><div class="surname">Vazquez</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Antequera</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Segurado</div> <div class="given-names">M.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Different nucleosomal architectures at early and late replicating origins in Saccharomyces cerevisiae</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">791</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20nucleosomal%20architectures%20at%20early%20and%20late%20replicating%20origins%20in%20Saccharomyces%20cerevisiae&amp;author=I.%20Soriano&amp;author=E.C.%20Morafraile&amp;author=E.%20Vazquez&amp;author=F.%20Antequera&amp;author=M.%20Segurado&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-791" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-791" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-791"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25218085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Heun</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Laroche</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Raghuraman</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Gasser</div> <div class="given-names">S.M.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">The positioning and dynamics of origins of replication in the budding yeast nucleus</div>, <div class="source">J. Cell Biol.</div> , <div class="volume">152</div>, <div class="fpage">385</div>–<div class="lpage">400</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20positioning%20and%20dynamics%20of%20origins%20of%20replication%20in%20the%20budding%20yeast%20nucleus&amp;author=P.%20Heun&amp;author=T.%20Laroche&amp;author=M.K.%20Raghuraman&amp;author=S.M.%20Gasser&amp;publication_year=2001&amp;journal=J.%20Cell%20Biol.&amp;volume=152&amp;pages=385-400" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.152.2.385" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.152.2.385" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.152.2.385"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11266454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">De</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Michor</div> <div class="given-names">F.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">DNA replication timing and higher-order nuclear organization determine single-nucleotide substitution patterns in cancer genomes</div>, <div class="source">Nat. Commun.</div> , <div class="volume">4</div>, <div class="fpage">1502</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20replication%20timing%20and%20higher-order%20nuclear%20organization%20determine%20single-nucleotide%20substitution%20patterns%20in%20cancer%20genomes&amp;author=L.%20Liu&amp;author=S.%20De&amp;author=F.%20Michor&amp;publication_year=2013&amp;journal=Nat.%20Commun.&amp;volume=4&amp;pages=1502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms2502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms2502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23422670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">De</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Michor</div> <div class="given-names">F.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">DNA replication timing and long-range DNA interactions predict mutational landscapes of cancer genomes</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">29</div>, <div class="fpage">1103</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20replication%20timing%20and%20long-range%20DNA%20interactions%20predict%20mutational%20landscapes%20of%20cancer%20genomes&amp;author=S.%20De&amp;author=F.%20Michor&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=1103-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22101487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McCullagh</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Seshan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">El-Samad</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Madhani</div> <div class="given-names">H.D.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Coordinate control of gene expression noise and interchromosomal interactions in a MAP kinase pathway</div>, <div class="source">Nat. Cell Biol.</div> , <div class="volume">12</div>, <div class="fpage">954</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coordinate%20control%20of%20gene%20expression%20noise%20and%20interchromosomal%20interactions%20in%20a%20MAP%20kinase%20pathway&amp;author=E.%20McCullagh&amp;author=A.%20Seshan&amp;author=H.%20El-Samad&amp;author=H.D.%20Madhani&amp;publication_year=2010&amp;journal=Nat.%20Cell%20Biol.&amp;volume=12&amp;pages=954-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncb2097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncb2097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncb2097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20852627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW005C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW005C72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chubb</div>
               <div class="given-names">J.R.</div>
            </div>, <div class="name string-name"><div class="surname">Boyle</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Perry</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bickmore</div> <div class="given-names">W.A.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Chromatin motion is constrained by association with nuclear compartments in human cells</div>, <div class="source">Curr. Biol.</div> , <div class="volume">12</div>, <div class="fpage">439</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20motion%20is%20constrained%20by%20association%20with%20nuclear%20compartments%20in%20human%20cells&amp;author=J.R.%20Chubb&amp;author=S.%20Boyle&amp;author=P.%20Perry&amp;author=W.A.%20Bickmore&amp;publication_year=2002&amp;journal=Curr.%20Biol.&amp;volume=12&amp;pages=439-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(02)00695-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(02)00695-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(02)00695-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11909528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745431&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw005/2/dsw005_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HUSHvpG9q-LQE4g1cg2zjuBXMQXaofivvdMnuWz3TweVZHopJHebtNDWmRvm3Vt88xeB2OW0PoOhJAOs8LMRx7oTDJumOVpNavKdnwgLq~p2Q49nmSG7c7YPCPbpyMqCry3ddYzWTcaW6D5lpoC4xA7NuaC0GFcxRPKO0~Rsm-cBp9WyunAqgWpo~tSVYArh8cpPUv38PzzoRLfbgDAytb022QvAPwi9XUrhOeKq6JlQUPVb238h1h7M2ZYxi1Y~4BIhDaM04nfdUNpzHxUvrQUDe1ZJwRpv1-3eT7DliHJY6dTKMrNqPm2fKWc5Oa4k2rLJ~~bZMBWiIyS73P7yBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">382</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745431"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f2%2f155%2f1745431">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000375088800008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F2%2F155%2F1745431" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745431'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Spatially coordinated replication and minimization of expression noise constrain three-dimensional organization of yeast genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f155%2f1745431">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

