

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/2/77/344797","@type":"ScholarlyArticle","name":"An Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi029","keywords":["whole genome amplification","isothermal","comparative genomic hybridization","genotyping","mutations"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/2/77/344797","author":[{"name":"Lee, Cheryl I. P.","@type":"Person"},{"name":"Leong, Siew Hong","@type":"Person"},{"name":"Png, Adrian E. H.","affiliation":"1  Bioinformatics Group, Nanyang Polytechnic  180 Ang Mo Kio Avenue 8, Singapore 569830","@type":"Person"},{"name":"Choo, Keng Wah","affiliation":"1  Bioinformatics Group, Nanyang Polytechnic  180 Ang Mo Kio Avenue 8, Singapore 569830","@type":"Person"},{"name":"Syn, Christopher","affiliation":"2  Centre for Forensic Science, Health Sciences Authority  11 Outram Road, Singapore 169078","@type":"Person"},{"name":"Lim, Dennis T. H.","affiliation":"3  Department of General Surgery, Singapore General Hospital  Outram Road, Singapore 169608","@type":"Person"},{"name":"Law, Hai Yang","affiliation":"4  Department of Paediatric Medicine, KK Women's &amp; Children's Hospital  100 Bukit Timah Road, Singapore 229899","@type":"Person"},{"name":"Kon, Oi Lian","@type":"Person"}],"description":"Abstract.  Molecular genotyping has important biomedical and forensic applications. However, limiting amounts of human biological material often yield genomic D","pageStart":"77","pageEnd":"88","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/2/0/m_dnaresearch13_2.cover.gif?Expires=2147483647&Signature=F40l18FMMrHb6CuOHJFXS03~VtBt1lG7g-odY5VEQUX1q0IM3hsT3kQORluBF8TnhhhNDVn7NJ2JJ-lW--ItgbBtnDkWHBWu3Vh4vaN4At1mYuSVWHgMIRqNfy8ZxuoH5cWSkzZObVXIjlncNZwn4Ig4Ynou09p4BxS6t2GDTY9xY2f2xwCGfZvQCsvQoiWpv5Y0Fz1eTmH2br04qtKXDhBLaxooaMOHzDWM3WvkvXdYEFoy6gAfvxpdvG3aVHgOULaJ4SNuIKS3K9n7amjBoW8otGN2PcXRc2jsZyBh0roj8zIISl-LQJPTkWg2DplR9vREdMFAXUJVke0z1sah4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"An Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/2/0/m_dnaresearch13_2.cover.gif?Expires=2147483647&Signature=F40l18FMMrHb6CuOHJFXS03~VtBt1lG7g-odY5VEQUX1q0IM3hsT3kQORluBF8TnhhhNDVn7NJ2JJ-lW--ItgbBtnDkWHBWu3Vh4vaN4At1mYuSVWHgMIRqNfy8ZxuoH5cWSkzZObVXIjlncNZwn4Ig4Ynou09p4BxS6t2GDTY9xY2f2xwCGfZvQCsvQoiWpv5Y0Fz1eTmH2br04qtKXDhBLaxooaMOHzDWM3WvkvXdYEFoy6gAfvxpdvG3aVHgOULaJ4SNuIKS3K9n7amjBoW8otGN2PcXRc2jsZyBh0roj8zIISl-LQJPTkWg2DplR9vREdMFAXUJVke0z1sah4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection" />
    <meta property="og:description" content="Abstract.  Molecular genotyping has important biomedical and forensic applications. However, limiting amounts of human biological material often yield genomic D" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/2/77/344797" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Lee, Cheryl I. P." /><meta name="citation_author" content="Leong, Siew Hong" /><meta name="citation_author" content="Png, Adrian E. H." /><meta name="citation_author_institution" content="1Bioinformatics Group, Nanyang Polytechnic180 Ang Mo Kio Avenue 8, Singapore 569830" /><meta name="citation_author" content="Choo, Keng Wah" /><meta name="citation_author_institution" content="1Bioinformatics Group, Nanyang Polytechnic180 Ang Mo Kio Avenue 8, Singapore 569830" /><meta name="citation_author" content="Syn, Christopher" /><meta name="citation_author_institution" content="2Centre for Forensic Science, Health Sciences Authority11 Outram Road, Singapore 169078" /><meta name="citation_author" content="Lim, Dennis T. H." /><meta name="citation_author_institution" content="3Department of General Surgery, Singapore General HospitalOutram Road, Singapore 169608" /><meta name="citation_author" content="Law, Hai Yang" /><meta name="citation_author_institution" content="4Department of Paediatric Medicine, KK Women's &amp; Children's Hospital100 Bukit Timah Road, Singapore 229899" /><meta name="citation_author" content="Kon, Oi Lian" /><meta name="citation_title" content="An Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection" /><meta name="citation_firstpage" content="77" /><meta name="citation_lastpage" content="88" /><meta name="citation_doi" content="10.1093/dnares/dsi029" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genomic profiling of gastric cancer predicts lymph node status and survival; Oncogene;  citation_year=2003;  citation_volume=22;  citation_pages=1872-1879; " /><meta name="citation_reference" content="citation_title=Further evidence for LBP-1c/CP2/LSF association in Alzheimer's disease families; J. Med. Genet.;  citation_year=2005;  citation_volume=42;  citation_pages=857-862; " /><meta name="citation_reference" content="citation_title=Validation of short tandem repeats (STRs) for forensic usage: performance testing of fluorescent multiplex STR systems and analysis of authentic and simulated forensic samples; J. Forensic Sci.;  citation_year=2001;  citation_volume=46;  citation_pages=647-660; " /><meta name="citation_reference" content="citation_title=Comprehen-sive human genome amplification using multiple displacement amplification; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=5261-5266; " /><meta name="citation_reference" content="citation_title=Whole genome analysis of genetic alterations in small DNA samples using hyperbranched strand displacement amplification and array-CGH; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=294-307; " /><meta name="citation_reference" content="citation_title=Degenerate oligonucleotide-primed PCR: general amplification of target DNA by a single degenerate primer; Genomics;  citation_year=1992;  citation_volume=13;  citation_pages=718-725; " /><meta name="citation_reference" content="citation_title=Whole genome amplification from a single cell: implications for genetic analysis; Proc. Natl Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=5847-5851; " /><meta name="citation_reference" content="citation_title=Comparative genomic hybridization, loss of heterozygosity, and DNA sequence analysis of single cells; Proc. Natl Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=4494-4499; " /><meta name="citation_reference" content="citation_title=Balanced-PCR amplification allows unbiased identification of genomic copy changes in minute cell and tissue samples; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=e76" /><meta name="citation_reference" content="citation_title=DNA amplification method tolerant to sample degradation; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2357-2366; " /><meta name="citation_reference" content="citation_title=Rubicon Genomics, Inc.; Pharmacogenomics;  citation_year=2002;  citation_volume=3;  citation_pages=557-560; " /><meta name="citation_reference" content="citation_title=A technique for radiolabeling DNA restriction endonuclease fragments to high specific activity; Anal. Biochem.;  citation_year=1983;  citation_volume=132;  citation_pages=6-13; " /><meta name="citation_reference" content="citation_title=Use of random primer extension for concurrent amplification and nonradioactive labeling of nucleic acids; Anal. Biochem.;  citation_year=1993;  citation_volume=212;  citation_pages=428-435; " /><meta name="citation_reference" content="citation_title=A combined comparative genomic hybridisation and expression microarray analysis of gastric cancer research reveals novel molecular subtypes; Cancer Res.;  citation_year=2003;  citation_volume=63;  citation_pages=3309-3316; " /><meta name="citation_reference" content="citation_title=Fully automatic quantification of microarray image data; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=325-332; " /><meta name="citation_reference" content="citation_title=Population data on the thirteen CODIS core short tandem repeat loci in African Americans, U.S. Caucasians, Hispanics, Bahamians, Jamaicans, and Trinidians; J. Forensic Sci.;  citation_year=1999;  citation_volume=44;  citation_pages=1277-1286; " /><meta name="citation_reference" content="citation_title=Population data on the STR loci D2S1338 and D19S433; citation_text=(); Forensic Sci. Commun.;  citation_year=2001;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=A study on the effects of degradation and template concentration on the amplification efficiency of the STR Miniplex primer sets; J. Forensic Sci.;  citation_year=2004;  citation_volume=49;  citation_pages=733-740; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/2/77/17462466/dsi029.pdf" /><meta name="description" content="Abstract.  Molecular genotyping has important biomedical and forensic applications. However, limiting amounts of human biological material often yield genomic D" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/2/77/344797" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/2/77/344797" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/2/77/344797");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi029';
    </script>
<title>
	Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./344797" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYc0PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWUsIENoZXJ5bCBJLiBQLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikxlb25nLCBTaWV3IEhvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQbmcsIEFkcmlhbiBFLiBILiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFCaW9pbmZvcm1hdGljcyBHcm91cCwgTmFueWFuZyBQb2x5dGVjaG5pYzE4MCBBbmcgTW8gS2lvIEF2ZW51ZSA4LCBTaW5nYXBvcmUgNTY5ODMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hvbywgS2VuZyBXYWgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxQmlvaW5mb3JtYXRpY3MgR3JvdXAsIE5hbnlhbmcgUG9seXRlY2huaWMxODAgQW5nIE1vIEtpbyBBdmVudWUgOCwgU2luZ2Fwb3JlIDU2OTgzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN5biwgQ2hyaXN0b3BoZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIyQ2VudHJlIGZvciBGb3JlbnNpYyBTY2llbmNlLCBIZWFsdGggU2NpZW5jZXMgQXV0aG9yaXR5MTEgT3V0cmFtIFJvYWQsIFNpbmdhcG9yZSAxNjkwNzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaW0sIERlbm5pcyBULiBILiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjNEZXBhcnRtZW50IG9mIEdlbmVyYWwgU3VyZ2VyeSwgU2luZ2Fwb3JlIEdlbmVyYWwgSG9zcGl0YWxPdXRyYW0gUm9hZCwgU2luZ2Fwb3JlIDE2OTYwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxhdywgSGFpIFlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSI0RGVwYXJ0bWVudCBvZiBQYWVkaWF0cmljIE1lZGljaW5lLCBLSyBXb21lbidzICZhbXA7IENoaWxkcmVuJ3MgSG9zcGl0YWwxMDAgQnVraXQgVGltYWggUm9hZCwgU2luZ2Fwb3JlIDIyOTg5OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvbiwgT2kgTGlhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQW4gSXNvdGhlcm1hbCBNZXRob2QgZm9yIFdob2xlIEdlbm9tZSBBbXBsaWZpY2F0aW9uIG9mIEZyZXNoIGFuZCBEZWdyYWRlZCBETkEgZm9yIENvbXBhcmF0aXZlIEdlbm9taWMgSHlicmlkaXphdGlvbiwgR2Vub3R5cGluZyBhbmQgTXV0YXRpb24gRGV0ZWN0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9Ijg4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNpMDI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDYvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHByb2ZpbGluZyBvZiBnYXN0cmljIGNhbmNlciBwcmVkaWN0cyBseW1waCBub2RlIHN0YXR1cyBhbmQgc3Vydml2YWw7IE9uY29nZW5lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTg3Mi0xODc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdXJ0aGVyIGV2aWRlbmNlIGZvciBMQlAtMWMvQ1AyL0xTRiBhc3NvY2lhdGlvbiBpbiBBbHpoZWltZXIncyBkaXNlYXNlIGZhbWlsaWVzOyBKLiBNZWQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTg1Ny04NjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhbGlkYXRpb24gb2Ygc2hvcnQgdGFuZGVtIHJlcGVhdHMgKFNUUnMpIGZvciBmb3JlbnNpYyB1c2FnZTogcGVyZm9ybWFuY2UgdGVzdGluZyBvZiBmbHVvcmVzY2VudCBtdWx0aXBsZXggU1RSIHN5c3RlbXMgYW5kIGFuYWx5c2lzIG9mIGF1dGhlbnRpYyBhbmQgc2ltdWxhdGVkIGZvcmVuc2ljIHNhbXBsZXM7IEouIEZvcmVuc2ljIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz02NDctNjYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW4tc2l2ZSBodW1hbiBnZW5vbWUgYW1wbGlmaWNhdGlvbiB1c2luZyBtdWx0aXBsZSBkaXNwbGFjZW1lbnQgYW1wbGlmaWNhdGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTUyNjEtNTI2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIGFuYWx5c2lzIG9mIGdlbmV0aWMgYWx0ZXJhdGlvbnMgaW4gc21hbGwgRE5BIHNhbXBsZXMgdXNpbmcgaHlwZXJicmFuY2hlZCBzdHJhbmQgZGlzcGxhY2VtZW50IGFtcGxpZmljYXRpb24gYW5kIGFycmF5LUNHSDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yOTQtMzA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWdlbmVyYXRlIG9saWdvbnVjbGVvdGlkZS1wcmltZWQgUENSOiBnZW5lcmFsIGFtcGxpZmljYXRpb24gb2YgdGFyZ2V0IEROQSBieSBhIHNpbmdsZSBkZWdlbmVyYXRlIHByaW1lcjsgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz03MTgtNzI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZSBnZW5vbWUgYW1wbGlmaWNhdGlvbiBmcm9tIGEgc2luZ2xlIGNlbGw6IGltcGxpY2F0aW9ucyBmb3IgZ2VuZXRpYyBhbmFseXNpczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTU4NDctNTg1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pYyBoeWJyaWRpemF0aW9uLCBsb3NzIG9mIGhldGVyb3p5Z29zaXR5LCBhbmQgRE5BIHNlcXVlbmNlIGFuYWx5c2lzIG9mIHNpbmdsZSBjZWxsczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTQ0OTQtNDQ5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFsYW5jZWQtUENSIGFtcGxpZmljYXRpb24gYWxsb3dzIHVuYmlhc2VkIGlkZW50aWZpY2F0aW9uIG9mIGdlbm9taWMgY29weSBjaGFuZ2VzIGluIG1pbnV0ZSBjZWxsIGFuZCB0aXNzdWUgc2FtcGxlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9ZTc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGFtcGxpZmljYXRpb24gbWV0aG9kIHRvbGVyYW50IHRvIHNhbXBsZSBkZWdyYWRhdGlvbjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yMzU3LTIzNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJ1Ymljb24gR2Vub21pY3MsIEluYy47IFBoYXJtYWNvZ2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTU1Ny01NjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGVjaG5pcXVlIGZvciByYWRpb2xhYmVsaW5nIEROQSByZXN0cmljdGlvbiBlbmRvbnVjbGVhc2UgZnJhZ21lbnRzIHRvIGhpZ2ggc3BlY2lmaWMgYWN0aXZpdHk7IEFuYWwuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTYtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzZSBvZiByYW5kb20gcHJpbWVyIGV4dGVuc2lvbiBmb3IgY29uY3VycmVudCBhbXBsaWZpY2F0aW9uIGFuZCBub25yYWRpb2FjdGl2ZSBsYWJlbGluZyBvZiBudWNsZWljIGFjaWRzOyBBbmFsLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTI7ICBjaXRhdGlvbl9wYWdlcz00MjgtNDM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbWJpbmVkIGNvbXBhcmF0aXZlIGdlbm9taWMgaHlicmlkaXNhdGlvbiBhbmQgZXhwcmVzc2lvbiBtaWNyb2FycmF5IGFuYWx5c2lzIG9mIGdhc3RyaWMgY2FuY2VyIHJlc2VhcmNoIHJldmVhbHMgbm92ZWwgbW9sZWN1bGFyIHN1YnR5cGVzOyBDYW5jZXIgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTMzMDktMzMxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVsbHkgYXV0b21hdGljIHF1YW50aWZpY2F0aW9uIG9mIG1pY3JvYXJyYXkgaW1hZ2UgZGF0YTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zMjUtMzMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIGRhdGEgb24gdGhlIHRoaXJ0ZWVuIENPRElTIGNvcmUgc2hvcnQgdGFuZGVtIHJlcGVhdCBsb2NpIGluIEFmcmljYW4gQW1lcmljYW5zLCBVLlMuIENhdWNhc2lhbnMsIEhpc3BhbmljcywgQmFoYW1pYW5zLCBKYW1haWNhbnMsIGFuZCBUcmluaWRpYW5zOyBKLiBGb3JlbnNpYyBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MTI3Ny0xMjg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIGRhdGEgb24gdGhlIFNUUiBsb2NpIEQyUzEzMzggYW5kIEQxOVM0MzM7IGNpdGF0aW9uX3RleHQ9KCk7IEZvcmVuc2ljIFNjaS4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN0dWR5IG9uIHRoZSBlZmZlY3RzIG9mIGRlZ3JhZGF0aW9uIGFuZCB0ZW1wbGF0ZSBjb25jZW50cmF0aW9uIG9uIHRoZSBhbXBsaWZpY2F0aW9uIGVmZmljaWVuY3kgb2YgdGhlIFNUUiBNaW5pcGxleCBwcmltZXIgc2V0czsgSi4gRm9yZW5zaWMgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTczMy03NDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xMy8yLzc3LzE3NDYyNDY2L2RzaTAyOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTW9sZWN1bGFyIGdlbm90eXBpbmcgaGFzIGltcG9ydGFudCBiaW9tZWRpY2FsIGFuZCBmb3JlbnNpYyBhcHBsaWNhdGlvbnMuIEhvd2V2ZXIsIGxpbWl0aW5nIGFtb3VudHMgb2YgaHVtYW4gYmlvbG9naWNhbCBtYXRlcmlhbCBvZnRlbiB5aWVsZCBnZW5vbWljIEQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0NDc5NyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTRMVF4v5YSLnk6Y0wTa5us4P6XZw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f2%2f77%2f344797" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f2%2f77%2f344797" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13265" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/2/0/m_dnaresearch13_2.cover.gif?Expires=2147483647&amp;Signature=F40l18FMMrHb6CuOHJFXS03~VtBt1lG7g-odY5VEQUX1q0IM3hsT3kQORluBF8TnhhhNDVn7NJ2JJ-lW--ItgbBtnDkWHBWu3Vh4vaN4At1mYuSVWHgMIRqNfy8ZxuoH5cWSkzZObVXIjlncNZwn4Ig4Ynou09p4BxS6t2GDTY9xY2f2xwCGfZvQCsvQoiWpv5Y0Fz1eTmH2br04qtKXDhBLaxooaMOHzDWM3WvkvXdYEFoy6gAfvxpdvG3aVHgOULaJ4SNuIKS3K9n7amjBoW8otGN2PcXRc2jsZyBh0roj8zIISl-LQJPTkWg2DplR9vREdMFAXUJVke0z1sah4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/2">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87194213">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194213">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194215">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194215">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194220">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194220">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194246">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194263">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194263">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194273">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194273">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/2/65/344571">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">An Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cheryl I. P. Lee</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Cheryl+I.+P.+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Cheryl I. P. Lee</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Siew Hong Leong</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Siew+Hong+Leong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Leong S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Leong S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Siew Hong Leong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Adrian E. H. Png</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Bioinformatics Group, Nanyang Polytechnic</div><div class="addr-line">180 Ang Mo Kio Avenue 8, Singapore 569830</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Adrian+E.+H.+Png">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Png A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Png A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Adrian E. H. Png</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keng Wah Choo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Bioinformatics Group, Nanyang Polytechnic</div><div class="addr-line">180 Ang Mo Kio Avenue 8, Singapore 569830</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keng+Wah+Choo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choo K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choo K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Keng Wah Choo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christopher Syn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Centre for Forensic Science, Health Sciences Authority</div><div class="addr-line">11 Outram Road, Singapore 169078</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christopher+Syn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Syn C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Syn C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christopher Syn</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dennis T. H. Lim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Department of General Surgery, Singapore General Hospital</div><div class="addr-line">Outram Road, Singapore 169608</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dennis+T.+H.+Lim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lim D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lim D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dennis T. H. Lim</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hai Yang Law</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Department of Paediatric Medicine, KK Women's &amp; Children's Hospital</div><div class="addr-line">100 Bukit Timah Road, Singapore 229899</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hai+Yang+Law">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Law H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Law H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hai Yang Law</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Oi Lian Kon</div>
    </div>


                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +65-6436-8319/8307, Fax. +65-6372-0161, Email: <a href="mailto:dmskol@nccs.com.sg" target="_blank">dmskol@nccs.com.sg</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Oi+Lian+Kon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kon O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kon O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Oi Lian Kon</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Michio Oishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 2, 1 January 2006, Pages 7788, <a href='https://doi.org/10.1093/dnares/dsi029'>https://doi.org/10.1093/dnares/dsi029</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 December 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">24 January 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="344797" href="/dnaresearch/article-pdf/13/2/77/17462466/dsi029.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Cheryl I. P. Lee, Siew Hong Leong, Adrian E. H. Png, Keng Wah Choo, Christopher Syn, Dennis T. H. Lim, Hai Yang Law, Oi Lian Kon;  An Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection, <em>DNA Research</em>, Volume 13, Issue 2, 1 January 2006, Pages 7788, <a href='https://doi.org/10.1093/dnares/dsi029'>https://doi.org/10.1093/dnares/dsi029</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/344797?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/344797?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/344797?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/344797?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/344797?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/344797?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/344797?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=An%20Isothermal%20Method%20for%20Whole%20Genome%20Amplification%20of%20Fresh%20and%20Degraded%20DNA%20for%20Comparative%20Genomic%20Hybridization%2C%20Genotyping%20and%20Mutation%20Detection&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=2&amp;author=Lee%2C%20Cheryl%20I.%20P.%3B%20Leong%2C%20Siew%20Hong&amp;startPage=77&amp;endPage=88&amp;contentId=10.1093%2Fdnares%2Fdsi029&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f2%2f77%2f344797" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87194213 id="87194213" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Molecular genotyping has important biomedical and forensic applications. However, limiting amounts of human biological material often yield genomic DNA (gDNA) in insufficient quantity and of poor quality for a reliable analysis. This motivated the development of an efficient whole genome amplification method with quantitatively unbiased representation usable on fresh and degraded gDNA. Amplification of fresh frozen, formalin-fixed paraffin-embedded (FFPE) and DNase-degraded DNA using degenerate oligonucleotide-primed PCR or primer extension amplification using a short primer sequence bioinformatically optimized for coverage of the human genome was compared with amplification using current primers by chromosome-based and BAC-array comparative genomic hybridization (CGH), genotyping at short tandem repeats (STRs) and single base mutation detection. Compared with current primers, genome amplification using the bioinformatically optimized primer was significantly less biased on CGH in selfself hybridizations, and replicated tumour genome copy number aberrations, even from FFPE tissue. STR genotyping could be performed on degraded gDNA amplified using our technique but failed with multiple displacement amplification. Of the 18 different single base mutations 16 (89.5%) were correctly identified by sequencing gDNA amplified from clinical samples using our technique. This simple and efficient isothermal method should be helpful for genetic research and clinical and forensic applications.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole genome amplification&quot;">whole genome amplification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="isothermal">isothermal</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomic hybridization&quot;">comparative genomic hybridization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genotyping">genotyping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mutations">mutations</a></div>                            <h2 scrollto-destination=87194215 id="87194215" class="section-title" >1. Introduction</h2>
<p>Genomic analyses, such as comparative genomic hybridization (CGH), genotyping of polymorphic loci and detection of disease gene mutations, are important in genomic medicine and forensic science. For example, CGH of cancer genomes has identified recurrent copy number aberrations of likely pathogenic significance,<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> certain genotypes are associated with altered disease susceptibility<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> and genotyping of highly polymorphic short tandem repeats (STR) is the most widely used method of human identification.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a></p><p>Genomic techniques require nanogram to microgram quantities of genomic DNA (gDNA). Hence, DNA samples of insufficient quantity must be pre-amplified before the actual genomic analysis can be performed. Multiple displacement amplification (MDA),<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> a widely used whole genome amplification (WGA) method, is efficient (&gt;1000-fold amplification) and yields minimally biased representation from sub-nanogram quantities of fresh undegraded gDNA. However, MDA performs poorly on degraded gDNA.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a></p><p>Other WGA methods are mainly, though not entirely, PCR based. Degenerate oligonucleotide-based PCR (DOP-PCR),<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> primer extension preamplification<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> and ligation-mediated PCR<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> are known to introduce copy number bias of dispersed genomic regions. Balanced PCR amplification,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> restriction and circularization-aided rolling circle amplification,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> and the OmniPlex<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> methods require substantial enzymatic manipulations that may be impractical with very small starting samples.</p><p>We have adapted an isothermal method originally developed for DNA labelling<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> but which also amplifies input DNA.<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> Our modified method uses a primer that is selected after parsing the human genome for oligomer sequences that have maximum genome coverage with minimal overlaps (A. E. H. Png, K. W. Choo, C. I. P. Lee, S. H. Leong and O. L. Kon, manuscript in preparation). The method we describe here performs better than other current techniques on gDNA from formalin-fixed paraffin-embedded (FFPE) sources and degraded gDNA.</p>                            <h2 scrollto-destination=87194220 id="87194220" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=87194221 id="87194221" class="section-title" >2.1. gDNA sources and extraction</h3>
<p>Sources of gDNA were peripheral human blood, fresh and FFPE Hs 746T tumour tissue. Tumours were generated by subcutaneously implanting Hs746T human gastric cancer cells (American Type Culture Collection, Manassas, VA, USA) in scid mice. Excised Hs 746T tumours were fresh frozen at 80C or fixed in neutral buffered 10% formalin for 16 h at room temperature and dehydrated in increasing ethanol concentrations (70100%), followed by three changes each of absolute ethanol, xylene and paraffin. gDNA was extracted from blood and the fresh tumour using the Blood &amp; Cell Culture DNA kit (Qiagen GmbH, Hilden, Germany), according to the manufacturer's protocol except for the following modification for tumour tissue. Ground tissue (300 mg) was incubated for 1618 h at 50C in 19 ml Buffer G2 and at least one additional aliquot (0.5 ml) of Qiagen Protease solution was added during this period to obtain a clear lysate.</p><p>DNA was extracted from tissue cores (1 mm diameter, total weight of cores 0.5 mg) prepared from paraffin blocks of Hs 746T tumour using the PUREGENE<sup></sup> DNA Purification Tissue Kit (Gentra Systems Inc., Minneapolis, MN, USA) according to the manufacturer's protocol with the following modification. De-paraffinized tissue was incubated with cell lysis solution at 65C for 60 min with shaking (300 r.p.m.). gDNA was quantified by absorbance at 260 nm and used if the ratio of absorbance at 260 nm/280 nm was 1.82.0.</p>                            <h3 scrollto-destination=87194224 id="87194224" class="section-title" >2.2. Primers</h3>
<p>The following primers were used: (i) NYP (Nanyang Polytechnic) primer, 5-NNNTGAGAT-3; (ii) random octamers and (iii) DOP-PCR primer, 5-CCGACTCGAGNNNNNNATGTGG-3.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> The sequence TGAGAT of the NYP primer was bioinformatically selected for its predicted ability to generate 23 kb amplicons uniformly distributed throughout the human genome.</p>                            <h3 scrollto-destination=87194226 id="87194226" class="section-title" >2.3. DNA amplification</h3>
<p>Two methods were used. DOP-PCR using the 22-base primer was performed with 40 ng DNA, 2 M primer, 4  200 M dNTPs, 1.5 mM MgCl<sub>2</sub> and 5 U AmpliTaq Gold<sup></sup> DNA polymerase (Applied Biosystems, Foster City, CA, USA) in a reaction volume of 0.1 ml. Thermal cycling conditions were 94C  9 min, 9 cycles of 94C  1 min/30C  1.5 min/72C  3 min, 30 cycles of 94C  1 min/62C  1 min/72C  1.5 min and a final extension of 8 min at 72C. DOP-PCR using NYP primer was performed with 40 ng DNA, 2 M primer, 4  200 M dNTPs, 1.5 mM MgCl<sub>2</sub>, 2 units DyNAzyme EXT DNA polymerase (Finnzymes Oy, Espoo, Finland) in a reaction volume of 0.1 ml. Thermal cycling conditions were 94C  5 min, 40 cycles of 94C  1 min/22C  1 min/0.1C per second transition to 72C/72C  1 min and a final extension of 8 min at 72C. Two more units of DyNAzyme EXT DNA polymerase were added to the reaction immediately after completion of the 20th cycle. Amplified DNA was purified using the QIAquick PCR Purification Kit (Qiagen).</p><p>Isothermal primer extension amplification<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> was performed by modification of the BioPrime DNA Labeling system (Invitrogen Corporation, USA). A total of 300 ng gDNA were denatured in the presence of 15 g random octamers or NYP primer by heating to 99C for 10 min and were snap-cooled on ice. Klenow polymerase (40 U) and 0.12 mM each of dATP, dCTP, dGTP and dTTP were added to the chilled mixture in a total reaction volume of 50 l. The reaction was incubated at 37C for 1416 h. DNA was purified using a Microcon YM-30 centrifugal filter unit (Millipore Corp., USA).</p>                            <h3 scrollto-destination=87194229 id="87194229" class="section-title" >2.4. DNA labelling</h3>
<p>Unamplified gDNA and gDNA pre-amplified by primer extension for chromosome CGH were labelled with fluorescein-12-dUTP (test DNA) or Texas Red-5-dUTP (reference DNA) using the Invitrogen's Nick Translation System according to the manufacturer's protocol to obtain 5002000 bp labelled fragments.</p><p>DNA samples for BAC array CGH were labelled with Cyanine 3-dUTP (Cy 3-dUTP for test DNA) or Cyanine 5-dUTP (Cy 5-dUTP for reference DNA) using the BioPrime DNA Labeling System (Invitrogen) and the supplier's protocol. All fluorophore-labelled deoxynucleotides were from PerkinElmer Life Sciences (USA). DNA was purified after labelling with Microcon YM-30 centrifugal filter units.</p>                            <h3 scrollto-destination=87194232 id="87194232" class="section-title" >2.5. CGH</h3>
<p>Chromosome CGH was performed as described previously.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> One microgram each of Hs 746T and reference gDNA (from fresh lymphocytes or paraffin cores of healthy male spleen) were labelled with fluorescein-12-dUTP and Texas Red-5-dUTP, respectively. Labelled DNA and 20 g Cot-1 DNA (Invitrogen) were co-precipitated with ethanol and dissolved in 10 l hybridization buffer (50% formamide, 10% dextran sulfate and 2 SSC). The probe mixture was denatured (74C for 6 min) and hybridized to normal metaphase chromosomes (prepared from phytohemagglutinin-stimulated peripheral lymphocytes using standard protocols) that had been denatured (74C for 2 min) in 70% formamide/2 SSC and dehydrated in an ethanol series. After a 48 h hybridization, slides were washed once in 50% formamide/2 SSC (10 min), twice in 2 SSC and once in 0.1 SSC (all 5 min each, at 45C), followed by 1 PBD buffer [12 mM NaHCO<sub>3</sub> containing 0.5% (v/v) NP-40] and deionized water at room temperature (5 min each). Slides were mounted in antifade solution with 0.1 g/ml 4,6-diamidino-2-phenylindole (DAPI).</p><p>BAC array CGH was performed on 2464-clone arrays (each clone spotted in triplicate) from the University of California San Francisco Comprehensive Cancer Center Microarray Core. Hybridization was performed as detailed in the Array CGH Hybridization Protocol (<a class="link link-uri" href="http://cc.ucsf.edu/microarray/index.asp" target="_blank"></a>). Briefly, gDNA was digested for 5 h with <em>Dpn</em>II (New England Biolabs, USA), purified using the QIAquick PCR purification kit and labelled as described above. Labelled reference and test DNAs (34 g each) were co-precipitated with 35 g human Cot-1 DNA. The precipitate was collected by high-speed centrifugation, air-dried and re-dissolved in a final volume of 0.1 ml 1 SSC containing 70% formamide (extra pure grade, Merck KgaA, Germany), 3.4% SDS (Bio-Rad Laboratories, USA) and 5 mg dextran sulfate (sodium salt, MW 500000, Sigma Chemical Co., USA). DNA was denatured at 73C for 12 min and snap-cooled before applying to the array slide on which a plastic frame seal (2.3 cm<sup>2</sup>, Fisher Scientific, USA) had been placed firmly around the region of spotted BAC clones. The array slide was pre-wetted with 50 l 2 SSC containing 20 mg dextran sulfate and 50% formamide (pre-hybridization solution pre-warmed to 37C). After the entire array surface was wet, 20 l of the pre-hybridization solution was withdrawn and the denatured combined reference and test DNA solution (0.1 ml) was quickly applied on the array. The assembled hybridization array slide was placed in a humidified and well-sealed slide chamber box. Uniform hybridization was achieved by rocking the slide on a tilting platform (2 oscillations/min) at 37C for 72 h. The position of the array slide with respect to the tilting axis was moved by consecutive 90 turns at 24 h intervals.</p><p>Post-hybridization washing in dim light was performed in the following sequence: (i) 50% formamide in 2 SSC, pH 7 for 15 min at 50C; (ii) 0.1 M Na<sub>2</sub>HPO<sub>4</sub> containing 0.1% Nonidet P40, pH 8 for 15 min at room temperature; (iii) 2 SSC for 2 min; (iv) 70%, 85% and absolute ethanol, each for 2 min. The slide was dried by centrifugation (130 <em>g</em> for 2 min) before scanning.</p>                            <h3 scrollto-destination=87194236 id="87194236" class="section-title" >2.6. Imaging and analysis</h3>
<p>DAPI, fluorescein and Texas Red images were captured from 810 metaphases on chromosome CGH slides using a fluorescence microscope and a charge-coupled device camera. Digital images were analysed by the CV Chromofluor system (Applied Imaging, UK) using 99% standard reference intervals. The mean fluorescence ratio exceeded 1.2 in regions of gain and was below 0.8 in regions of loss. High-level amplifications were chromosomal regions with mean ratios &gt;1.5.</p><p>Array CGH slides were scanned on an Axon GenePix 4000B scanner (Axon Instruments, USA) after adjusting the count ratio of channel 1/channel 2 to 1. TIFF images were analysed using the SPOT program, which segmented the spots, performed local background subtraction and generated log<sub>2</sub> ratio data of Cy 3/Cy 5 intensities for each spot.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> SPOT output data were further processed using the SPROC program (<a class="link link-uri" href="http://jainlab.ucsf.edu/Downloads.html" target="_blank"></a>), which generated the mean log<sub>2</sub> Cy 3/Cy 5 ratio of triplicate spots for each BAC clone that survived quality constraints. SPOT and SPROC data files were exported to a custom program, NEAT, which generated an average intensity grid by grid plot for Cy 3 and Cy 5 to visually inspect hybridization quality and accuracy of the initial count ratio. Cy3/Cy5 ratios were normalized by total intensity normalization and normalized ratio plots were generated for every chromosome. The thresholds for amplification and deletions were +0.33 and 0.33, which are the log<sub>2</sub> transformed values corresponding to ratios of 1.25 and 0.75, respectively, of chromosome CGH.</p>                            <h3 scrollto-destination=87194239 id="87194239" class="section-title" >2.7. STR analysis</h3>
<p>Control and whole genome amplified DNA were profiled using the commercial AmpF<em>I</em>STR<sup></sup> Identifiler<sup></sup> PCR Amplification Kit (Applied Biosystems) and the supplied protocol. This kit allows amplification/detection of 15 STR loci comprising 13 US CODIS (<span class="underline">CO</span>mbinded <span class="underline">D</span>NA <span class="underline">I</span>ndex <span class="underline">S</span>ystem) loci, D2S1338, D19S433 and the gender-determining locus amelogenin. All genotyping reactions were performed on a GeneAmp 9700 PCR system (Applied Biosystems). Where necessary, DNA template quantity was modified for PCR optimization. PCR products were separated by capillary electrophoresis in an ABI PRISM<sup></sup> 3100 Genetic Analyzer (Applied Biosystems). One microlitre of PCR product was injected at 1 kV for 22 s with the minimum allele detection threshold set at 50 relative fluorescence units.</p>                            <h3 scrollto-destination=87194241 id="87194241" class="section-title" >2.8. Mutation detection</h3>
<p>A total of 50 ng of lymphocyte gDNA was amplified by primer extension using NYP primer as described above (DNA amplification) followed by purification using the QIAquick PCR Purification Kit. PCR with gene-specific primers (forward primer 5-GTACGGCTGTCATCACTTAGACCTCA-3; reverse primer 5-TCCCATAGACTCACCCTGAA-3; 2 M each) was performed on 200 ng of amplified gDNA. Other reaction conditions were 0.8 mM dNTPs, 1.5 mM MgCl<sub>2</sub> and 0.02 U DyNAzyme EXT DNA polymerase. Thermal cycling conditions were 94C for 2 min followed by 30 cycles of 94C  30 s/55C  30 s/72C  60 s and a terminal extension for 10 min at 72C.</p><p>A second PCR using a nested reverse primer (5-TGCAGCTTGTCACAGTGCAGCTCACT-3) was performed on 1 l of the completed first reaction product to yield a single 602 bp DNA fragment, which was resolved by electrophoresis on 1.5% agarose and extracted using the QIAquick Gel Extraction Kit (Qiagen). This was sequenced using the CEQ Dye Terminator Cycle Sequencing Quick Start Kit (Beckman Coulter, USA) according to the supplied protocol. After 30 cycles, 5 l of stop solution/glycogen mixture (1.2 M sodium acetate, 40 mM disodium EDTA and 4 mg glycogen/ml) was added and the sample was ethanol precipitated, vacuum dried for 3 min and resuspended in 20 l of the provided sample loading solution before loading on the CEQ capillary sequencer (Beckman Coulter).</p>                            <h3 scrollto-destination=87194244 id="87194244" class="section-title" >2.8. Statistical tests</h3>
<p>The significance of differences in variance and differences in proportion of outlier clones were determined using Levene's and Fisher's exact tests, respectively (SPSS statistical analysis software, Chicago, USA).</p>                            <h2 scrollto-destination=87194246 id="87194246" class="section-title" >3. Results</h2>
<p><a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Figure 1</a> is a comparison of copy number profiles obtained by chromosome CGH after amplification using two different primers. The sample in each experiment was lymphocyte gDNA from the same normal healthy male amplified by DOP-PCR using the bioinformatically optimized NYP primer, 5-NNNTGAGAT-3, or the widely used primer of Telenius <em>et al.</em><a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> 5-CCGACTCGAGNNNNNNATGTGG-3. WGA products were labelled with fluorescein-12-dUTP while the cognate unamplified gDNA was labelled with Texas Red-5-dUTP by nick translation. Selfself hybridization of unamplified and DOP-PCR amplified gDNA showed that WGA products generated using NYP primer resulted in false deletions in only 2 chromosomes, without any spurious copy number gains (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Figure 1</a>, upper panel). In contrast, DOP-PCR using the primer of Telenius <em>et al</em>.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> resulted in major false deletions in no fewer than 11 chromosome arms (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Figure 1</a>, lower panel).</p>                        <a id="87194248" scrollto-destination="87194248"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f1.jpeg?Expires=2147483647&Signature=ZMuYFOWCXBFNT~0qrF5Qkj5TP6M1HfHLYhgNfhrHlqydZ6637rIQwtKRhMCFWoojaZuoydk0kH-eF5n9HQYJtCIQpAWUP863hpHisiZWJC4ddUrdTRM7ny9YWaLQBdcDdOfGvcR1G~HpG8FljJ-p~heA9Y6raxLpzqPZOkCzosJp5lWKxo6A89i8NZaQQb42vpHXtjMkH2eAihhKMvNy5AZ7Bwe4NrRLOeYcvl4pVySP4hH0a7adJ0Lc~4IBeUxRQIws8ttLYbtgQYXX4I~lP682pGPCtXff2b5qO2ZQk5ldQvkP5FIvtXrVSrJlPbzhijh6vj6KLRYCRnDceSemAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of genomic representation after WGA using NYP (upper panel) or Telenius (lower panel) primer in DOP-PCR. Each panel shows chromosomal CGH selfself hybridization. The same gDNA was used in each experiment. It was extracted from normal lymphocytes of a healthy male. In each CGH experiment, unamplified gDNA was hybridized against itself after DOP-PCR amplification (i.e. unamplified self versus amplified self). Chromosomal CGH was performed as described previously.14 A colour version of Figure 1 can be found online as supplementary data at " path-from-xml="dsi029f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194248/dsi029f1.jpeg" path-from-xml="dsi029f1.jpeg" target="_blank">View large</a><a section="87194248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f1.jpeg?Expires=2147483647&Signature=S2LJ4annNPkAHSD6Xr~qLAnnMopwpVfN8sq-v60Ei6FEcLYhTWKGqQQitJBYv04ZFzPwbKttjc7UNXK1UH9wbOsHgINyLGtZFadQuVl44WoF4C88nJX7qPLSEuGWJNbfbRFYHOGEE5nN~u4UAa1KjnnLXga8G4O0-8L2GMuS9SG1tQmylrZjWJDwraNTtQ9bcx784QuABdy5vSMh8MzoQ-Uu5QP7MBJQUQge7jUepnDTf3orCR7qJwzdiQoNQ49gdqbVmia11xwZYjZmyL8pAGdyY~hdfn5LoF7XH~2LLMXEaLjKFP7KasVwtrS6luur2SmUE5voKGOtv4D7iNfWFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194248&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of genomic representation after WGA using NYP (upper panel) or Telenius (lower panel) primer in DOP-PCR. Each panel shows chromosomal CGH selfself hybridization. The same gDNA was used in each experiment. It was extracted from normal lymphocytes of a healthy male. In each CGH experiment, unamplified gDNA was hybridized against itself after DOP-PCR amplification (i.e. unamplified self versus amplified self). Chromosomal CGH was performed as described previously.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> A colour version of Figure 1 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f1.jpeg?Expires=2147483647&Signature=ZMuYFOWCXBFNT~0qrF5Qkj5TP6M1HfHLYhgNfhrHlqydZ6637rIQwtKRhMCFWoojaZuoydk0kH-eF5n9HQYJtCIQpAWUP863hpHisiZWJC4ddUrdTRM7ny9YWaLQBdcDdOfGvcR1G~HpG8FljJ-p~heA9Y6raxLpzqPZOkCzosJp5lWKxo6A89i8NZaQQb42vpHXtjMkH2eAihhKMvNy5AZ7Bwe4NrRLOeYcvl4pVySP4hH0a7adJ0Lc~4IBeUxRQIws8ttLYbtgQYXX4I~lP682pGPCtXff2b5qO2ZQk5ldQvkP5FIvtXrVSrJlPbzhijh6vj6KLRYCRnDceSemAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of genomic representation after WGA using NYP (upper panel) or Telenius (lower panel) primer in DOP-PCR. Each panel shows chromosomal CGH selfself hybridization. The same gDNA was used in each experiment. It was extracted from normal lymphocytes of a healthy male. In each CGH experiment, unamplified gDNA was hybridized against itself after DOP-PCR amplification (i.e. unamplified self versus amplified self). Chromosomal CGH was performed as described previously.14 A colour version of Figure 1 can be found online as supplementary data at " path-from-xml="dsi029f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194248/dsi029f1.jpeg" path-from-xml="dsi029f1.jpeg" target="_blank">View large</a><a section="87194248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f1.jpeg?Expires=2147483647&Signature=S2LJ4annNPkAHSD6Xr~qLAnnMopwpVfN8sq-v60Ei6FEcLYhTWKGqQQitJBYv04ZFzPwbKttjc7UNXK1UH9wbOsHgINyLGtZFadQuVl44WoF4C88nJX7qPLSEuGWJNbfbRFYHOGEE5nN~u4UAa1KjnnLXga8G4O0-8L2GMuS9SG1tQmylrZjWJDwraNTtQ9bcx784QuABdy5vSMh8MzoQ-Uu5QP7MBJQUQge7jUepnDTf3orCR7qJwzdiQoNQ49gdqbVmia11xwZYjZmyL8pAGdyY~hdfn5LoF7XH~2LLMXEaLjKFP7KasVwtrS6luur2SmUE5voKGOtv4D7iNfWFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194248&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of genomic representation after WGA using NYP (upper panel) or Telenius (lower panel) primer in DOP-PCR. Each panel shows chromosomal CGH selfself hybridization. The same gDNA was used in each experiment. It was extracted from normal lymphocytes of a healthy male. In each CGH experiment, unamplified gDNA was hybridized against itself after DOP-PCR amplification (i.e. unamplified self versus amplified self). Chromosomal CGH was performed as described previously.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> A colour version of Figure 1 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div><p>WGA in all subsequent experiments was performed by primer extension amplification because DOP-PCR is now recognized to cause significant amplification bias. The extent of gDNA amplification by primer extension was inversely related to the amount of input DNA. About 5500-fold amplification was achieved when input DNA was 1 ng. With 10 and 50 ng input gDNA, the amplification factors were 580 and 144, respectively. The size of WGA products ranged from 200 to &gt;23 kb (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Figure 2</a>).</p>                        <a id="87194250" scrollto-destination="87194250"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f2.jpeg?Expires=2147483647&Signature=LNQJj8ymzXNjQ1NZcc9DbcmnLTj~eavRo~u~TTYZMWbHx9HFa~skGsIul1PyHDs3RDumyY00y20uNXe--yR1VyDabenu6qZardwPX9AKxHn6QDZgu~8z8HRaLZzBwiMVwyqSby9y6V~ZjGq1cWV51vh4rseHgnqCGjYZJRKuvVZqpgyd3s584f~JtxLVjGVpmgwgUEzKC5DCqSfPF8gR~Dw1YmILfNu8WMzqYMSlhqe8Y1TFYfnvyN4v5fHrGs7gm2CZppdsIUphRKHIkSJmlx8wgoQIfI7swK5tkidvGucJFVotbkuwkHjf5t9H~n1TddJcMmtU59bmq983twA2Dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Agarose gel electrophoresis of primer extension amplification products showing varying amounts of fresh gDNA input, DNA size standards and amplification factors." path-from-xml="dsi029f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194250/dsi029f2.jpeg" path-from-xml="dsi029f2.jpeg" target="_blank">View large</a><a section="87194250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f2.jpeg?Expires=2147483647&Signature=XxxI6gS1uGLoRiEkode9aLcuKrSJnMCczk5ejU2lNtJuWA-e6O62Ppy3kA94ARlnTFojEXEeKy-3tA4We2TpZEDwHxm536ybCYJE34mfgV~2ANj5G3oh~V-9vi6PC7g9QsrXcwnG8JoyrXz4cvUf6BL33gNbRQ42f2Gr1MHQFIYfbP~jE54QBa453UxVZaD1JGJgSfIMEPwm8S~TNc3IvaVz8K5GgS6UBjSsiPoxtaG0TxOS47XwtksrCS1tXa5HiUnT-YaCZ1IBLwtWCWlNlb1oFSM-F2OozZqxqxbMsiBbZFgQhDOrWrmDkQsdceHL8qNPTlnZlREvjBhWyf1rYA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194250&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Agarose gel electrophoresis of primer extension amplification products showing varying amounts of fresh gDNA input, DNA size standards and amplification factors.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f2.jpeg?Expires=2147483647&Signature=LNQJj8ymzXNjQ1NZcc9DbcmnLTj~eavRo~u~TTYZMWbHx9HFa~skGsIul1PyHDs3RDumyY00y20uNXe--yR1VyDabenu6qZardwPX9AKxHn6QDZgu~8z8HRaLZzBwiMVwyqSby9y6V~ZjGq1cWV51vh4rseHgnqCGjYZJRKuvVZqpgyd3s584f~JtxLVjGVpmgwgUEzKC5DCqSfPF8gR~Dw1YmILfNu8WMzqYMSlhqe8Y1TFYfnvyN4v5fHrGs7gm2CZppdsIUphRKHIkSJmlx8wgoQIfI7swK5tkidvGucJFVotbkuwkHjf5t9H~n1TddJcMmtU59bmq983twA2Dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Agarose gel electrophoresis of primer extension amplification products showing varying amounts of fresh gDNA input, DNA size standards and amplification factors." path-from-xml="dsi029f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194250/dsi029f2.jpeg" path-from-xml="dsi029f2.jpeg" target="_blank">View large</a><a section="87194250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f2.jpeg?Expires=2147483647&Signature=XxxI6gS1uGLoRiEkode9aLcuKrSJnMCczk5ejU2lNtJuWA-e6O62Ppy3kA94ARlnTFojEXEeKy-3tA4We2TpZEDwHxm536ybCYJE34mfgV~2ANj5G3oh~V-9vi6PC7g9QsrXcwnG8JoyrXz4cvUf6BL33gNbRQ42f2Gr1MHQFIYfbP~jE54QBa453UxVZaD1JGJgSfIMEPwm8S~TNc3IvaVz8K5GgS6UBjSsiPoxtaG0TxOS47XwtksrCS1tXa5HiUnT-YaCZ1IBLwtWCWlNlb1oFSM-F2OozZqxqxbMsiBbZFgQhDOrWrmDkQsdceHL8qNPTlnZlREvjBhWyf1rYA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194250&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Agarose gel electrophoresis of primer extension amplification products showing varying amounts of fresh gDNA input, DNA size standards and amplification factors.</p></div></div><p>We next compared copy number profiles of unamplified gDNA extracted from fresh frozen Hs 746T tumour tissue (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3A</a>) with unamplified gDNA extracted from the same tumour after formalin fixation and paraffin embedding (FFPE) (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3B</a>) to determine the extent to which tissue preservation alone affects CGH analysis. Reference gDNA was unamplified fresh human lymphocyte gDNA (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3A</a>) or unamplified human spleen gDNA extracted from paraffin blocks (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3B</a>). <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3A and B</a> show similar copy number profiles but with several discordant aberrations i.e. spurious copy number losses of 5q, 10p, 12q, 18p and 19q; spurious copy number gain of 20p; and failure to detect 9p loss in CGH of gDNA extracted from FFPE tissue. As neither gDNA sample in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3A and B</a> was amplified, these discrepant copy number changes were most likely due to effects of chemical fixation and embedding on the quality of extracted gDNA e.g. DNAprotein cross-linking and DNA degradation. We proceeded to determine the fidelity of copy number analysis after amplifying gDNA from FFPE Hs 746T tumour tissue. A total of 50 ng of gDNA from FFPE Hs 746T tumour tissue and FFPE normal human spleen (reference DNA) was amplified by primer extension using NYP primer followed by fluorescence labelling using nick translation. <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3C</a> shows that the copy number profile of amplified gDNA from FFPE tissue largely replicates the true aberrations of unamplified gDNA prepared from fresh Hs 746T tumour tissue. Copy number gains in 1p, 3pq, 6q, 7q, 8q, 9q, 10q, 11q, 14q and 17q were replicated in gDNA from fixed tumour tissues, as were copy number losses in 1q, 2pq, 4q, 8p, 13q, 15q, 16p, 20p, 21q and 22q. Certain copy number abnormalities, however, only appeared in CGH of amplified gDNA prepared from FFPE tumour tissue (i.e. deletions of 5q, 10p, 12q and 19q; gain of 20q) while gain of 7p and loss of 9p detected in fresh tumour tissue were not detected in gDNA from FFPE tissue. However, most of these apparently artefactual findings were present already in unamplified gDNA extracted from FFPE tissues, i.e. deletions of 5q, 10p, 12q, 18p and gain of 20q (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3B</a>), and cannot be attributed to quantitatively skewed representation caused by WGA.</p>                        <a id="87194252" scrollto-destination="87194252"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f3.jpeg?Expires=2147483647&Signature=Ixdjpxhai1NM0Xk6lRfMy5TRcHAUKq0Sv1OqggCvjWwrV63eW6l-1cTTVoaBWQ-AnomlXTQ3qWqbYOD1mU4qskfTYKsfhQI13upMoN9HQUfMkOBwnQK0oTZXCsfTHD0GpT9zQqWOQulSlvWiv7mhSZspUA3TbsUyxDzD6h0SshLj5L~1ObcmpTniKZzFMK9IgXyhgDPPanVq~rfRggB0F3XhgUewTASBlINEnKGhVOzmQdsLTkntLK5V2ecYR2ph2NMJdJodHXcm050W1DNkhDHSOn4zwDMEB37HSLrM-2Qu2mPB3rgnPacNki9HaJ9GiufUfWlQSYzBa9JgcMWIHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Primer extension amplification using NYP primer replicates copy number aberrations of unamplified tumour gDNA. cCGH of a xenotumour (Hs 746T) using (A) unamplified gDNA extracted from fresh frozen tissue; (B) unamplified gDNA from FFPE tissue of the same tumour and (C) gDNA from FFPE tumour tissue amplified with NYP primer. Reference gDNA was from normal male lymphocytes (A) or from FFPE normal human spleen (B and C). A colour version of figure 3 can be found online as supplementary data at " path-from-xml="dsi029f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194252/dsi029f3.jpeg" path-from-xml="dsi029f3.jpeg" target="_blank">View large</a><a section="87194252" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f3.jpeg?Expires=2147483647&Signature=HFIQuuJm0tEcAh2uZNJRAKagh8o-oolOhHoppSZd7jdsHvQR099HU5tzUTISQn1v1IUYhIfg0CsCbJDqs5HJRV7J9uGhCCUMCCy54svKirWEbQ-DYMl-n-7ZiBEO1kKp4dAlEOUk0CxIuszAkos~0-b-ZIddkr0p-Oh31btYkhjfC3aEpy6ol3DBLTO5b1kSsEKF5qtnTE5qU-LZKDsg1JixtC7m1HSREUqnxgWfSn2hiZxQSWEiWkDZRims9QRG4w0KUo4wBQebY68vZamumll4LhAAB--AYQIrZuKX7CP8i8m5Wkeq60qF8oSYNAPtNe664K2x2eOgKs2puJ3LsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194252&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Primer extension amplification using NYP primer replicates copy number aberrations of unamplified tumour gDNA. cCGH of a xenotumour (Hs 746T) using (<strong>A</strong>) unamplified gDNA extracted from fresh frozen tissue; (<strong>B</strong>) unamplified gDNA from FFPE tissue of the same tumour and (<strong>C</strong>) gDNA from FFPE tumour tissue amplified with NYP primer. Reference gDNA was from normal male lymphocytes (A) or from FFPE normal human spleen (B and C). A colour version of figure 3 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f3.jpeg?Expires=2147483647&Signature=Ixdjpxhai1NM0Xk6lRfMy5TRcHAUKq0Sv1OqggCvjWwrV63eW6l-1cTTVoaBWQ-AnomlXTQ3qWqbYOD1mU4qskfTYKsfhQI13upMoN9HQUfMkOBwnQK0oTZXCsfTHD0GpT9zQqWOQulSlvWiv7mhSZspUA3TbsUyxDzD6h0SshLj5L~1ObcmpTniKZzFMK9IgXyhgDPPanVq~rfRggB0F3XhgUewTASBlINEnKGhVOzmQdsLTkntLK5V2ecYR2ph2NMJdJodHXcm050W1DNkhDHSOn4zwDMEB37HSLrM-2Qu2mPB3rgnPacNki9HaJ9GiufUfWlQSYzBa9JgcMWIHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Primer extension amplification using NYP primer replicates copy number aberrations of unamplified tumour gDNA. cCGH of a xenotumour (Hs 746T) using (A) unamplified gDNA extracted from fresh frozen tissue; (B) unamplified gDNA from FFPE tissue of the same tumour and (C) gDNA from FFPE tumour tissue amplified with NYP primer. Reference gDNA was from normal male lymphocytes (A) or from FFPE normal human spleen (B and C). A colour version of figure 3 can be found online as supplementary data at " path-from-xml="dsi029f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194252/dsi029f3.jpeg" path-from-xml="dsi029f3.jpeg" target="_blank">View large</a><a section="87194252" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f3.jpeg?Expires=2147483647&Signature=HFIQuuJm0tEcAh2uZNJRAKagh8o-oolOhHoppSZd7jdsHvQR099HU5tzUTISQn1v1IUYhIfg0CsCbJDqs5HJRV7J9uGhCCUMCCy54svKirWEbQ-DYMl-n-7ZiBEO1kKp4dAlEOUk0CxIuszAkos~0-b-ZIddkr0p-Oh31btYkhjfC3aEpy6ol3DBLTO5b1kSsEKF5qtnTE5qU-LZKDsg1JixtC7m1HSREUqnxgWfSn2hiZxQSWEiWkDZRims9QRG4w0KUo4wBQebY68vZamumll4LhAAB--AYQIrZuKX7CP8i8m5Wkeq60qF8oSYNAPtNe664K2x2eOgKs2puJ3LsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194252&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Primer extension amplification using NYP primer replicates copy number aberrations of unamplified tumour gDNA. cCGH of a xenotumour (Hs 746T) using (<strong>A</strong>) unamplified gDNA extracted from fresh frozen tissue; (<strong>B</strong>) unamplified gDNA from FFPE tissue of the same tumour and (<strong>C</strong>) gDNA from FFPE tumour tissue amplified with NYP primer. Reference gDNA was from normal male lymphocytes (A) or from FFPE normal human spleen (B and C). A colour version of figure 3 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div><p>Array CGH has considerably higher resolution than chromosomal CGH and is increasingly used in preference. We therefore compared the quality of amplified gDNA generated by primer extension amplification performed with NYP primer or random octamers by hybridization with a 2464 BAC clone array. <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Figure 4</a> compares normalized log<sub>2</sub> Cy3/Cy5 ratios of selfself hybridizations in which normal human lymphocyte gDNA was amplified using NYP primer (upper panel) or random octamers (lower panel). Copy number analysis across the human genome showed significantly less scatter around the zero value (signifying balanced copy number) when WGA was performed with NYP primer compared with random octamers (<em>P</em> &lt; 0.0001 for the difference between variances using Levene's test). Although both datasets included outlier signals, defined as log<sub>2</sub> greater than +0.33 or less than 0.33 that falsely signified copy number gain or loss, respectively, there were only 18 such outliers in NYP-primed WGA compared with 65 outliers in random octamer-primed WGA (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>, <em>P</em> &lt; 0.0001, Fisher's exact test). We observed that outlier clones did not cluster in genomic regions but were instead uniformly dispersed in all chromosomes.</p>                        <a id="87194254" scrollto-destination="87194254"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f4.jpeg?Expires=2147483647&Signature=Esl~zLYV0TCpe0Fq1yU4GJhjzjHeV8D0wOu3qAvu-ShRQd4QbPDXcgwHOPyngP-doBruw39wq9La0D2PSGIu9vv2kJs7WPc9znJlzb1FsaGpceA2qXkbsY3Bn3jQFDutDr5coOU4P74~gadlVNd0rcuNoWu1ZGTJsTL5Yq4kYg~lphkM0KlXj2Gibke8klSLEbv2mIrW40iOWKCXWP2SR-3cuZ9Srsa-w5OKhuSb9nY1LelF1fxpLOkbV8Q2WgytUd3kzhLF7ml5SHlXFoP-IM6oSBR1utC8~MpK2qyOepY8jaUJYApQHwRaN9BuSZuV09besuhN8PivGBBGnlaP4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Primer extension WGA using NYP primer (upper panel) and random octamers (lower panel) compared for artefactually biased representation. Data are normalized log2 Cy3/Cy5 ratios (vertical axis) of selfself hybridizations of normal lymphocyte gDNA plotted against chromosomal positions on the horizontal axis (chromosomes 1 to Y, left to right). Vertical lines denote boundaries between BAC clones that are mapped to consecutive chromosomes. Horizontal grey lines correspond to log2 values of +0.33 and 0.33." path-from-xml="dsi029f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194254/dsi029f4.jpeg" path-from-xml="dsi029f4.jpeg" target="_blank">View large</a><a section="87194254" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f4.jpeg?Expires=2147483647&Signature=ZgaZ3so3tJwFw-aiF~gC3tRRj~FIiFyziiSxUNzJUs~FQlZaKHnqyJH6paReSEAO8~GqgGKTt4gHxXbVxZXQyViJgQSQR1RnxOVftwghMXmMsXBNJg764kKiFCq0HyOoOT5IJrHlih3TEn1QB2AB~ipMoa5~VCEaAq9YpLBBsdHWQ4rce3CqSmNIwBuQZd9JUJy6dkh7SL57KjilVN7kNJFi8mbeVxVvZGllf8U0VFJ0MFGEfg53M5sfqMf4v0~K5qomfSqOVfNXS1KuiScgyUt2G6Y5IRWepLIRKqQWTc26dBVik4WjYNbwjf-NKZdkY7T5xEeOkz7-ptvmx7zOlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194254&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Primer extension WGA using NYP primer (upper panel) and random octamers (lower panel) compared for artefactually biased representation. Data are normalized log<sub>2</sub> Cy3/Cy5 ratios (vertical axis) of selfself hybridizations of normal lymphocyte gDNA plotted against chromosomal positions on the horizontal axis (chromosomes 1 to Y, left to right). Vertical lines denote boundaries between BAC clones that are mapped to consecutive chromosomes. Horizontal grey lines correspond to log<sub>2</sub> values of +0.33 and 0.33.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f4.jpeg?Expires=2147483647&Signature=Esl~zLYV0TCpe0Fq1yU4GJhjzjHeV8D0wOu3qAvu-ShRQd4QbPDXcgwHOPyngP-doBruw39wq9La0D2PSGIu9vv2kJs7WPc9znJlzb1FsaGpceA2qXkbsY3Bn3jQFDutDr5coOU4P74~gadlVNd0rcuNoWu1ZGTJsTL5Yq4kYg~lphkM0KlXj2Gibke8klSLEbv2mIrW40iOWKCXWP2SR-3cuZ9Srsa-w5OKhuSb9nY1LelF1fxpLOkbV8Q2WgytUd3kzhLF7ml5SHlXFoP-IM6oSBR1utC8~MpK2qyOepY8jaUJYApQHwRaN9BuSZuV09besuhN8PivGBBGnlaP4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Primer extension WGA using NYP primer (upper panel) and random octamers (lower panel) compared for artefactually biased representation. Data are normalized log2 Cy3/Cy5 ratios (vertical axis) of selfself hybridizations of normal lymphocyte gDNA plotted against chromosomal positions on the horizontal axis (chromosomes 1 to Y, left to right). Vertical lines denote boundaries between BAC clones that are mapped to consecutive chromosomes. Horizontal grey lines correspond to log2 values of +0.33 and 0.33." path-from-xml="dsi029f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194254/dsi029f4.jpeg" path-from-xml="dsi029f4.jpeg" target="_blank">View large</a><a section="87194254" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f4.jpeg?Expires=2147483647&Signature=ZgaZ3so3tJwFw-aiF~gC3tRRj~FIiFyziiSxUNzJUs~FQlZaKHnqyJH6paReSEAO8~GqgGKTt4gHxXbVxZXQyViJgQSQR1RnxOVftwghMXmMsXBNJg764kKiFCq0HyOoOT5IJrHlih3TEn1QB2AB~ipMoa5~VCEaAq9YpLBBsdHWQ4rce3CqSmNIwBuQZd9JUJy6dkh7SL57KjilVN7kNJFi8mbeVxVvZGllf8U0VFJ0MFGEfg53M5sfqMf4v0~K5qomfSqOVfNXS1KuiScgyUt2G6Y5IRWepLIRKqQWTc26dBVik4WjYNbwjf-NKZdkY7T5xEeOkz7-ptvmx7zOlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194254&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Primer extension WGA using NYP primer (upper panel) and random octamers (lower panel) compared for artefactually biased representation. Data are normalized log<sub>2</sub> Cy3/Cy5 ratios (vertical axis) of selfself hybridizations of normal lymphocyte gDNA plotted against chromosomal positions on the horizontal axis (chromosomes 1 to Y, left to right). Vertical lines denote boundaries between BAC clones that are mapped to consecutive chromosomes. Horizontal grey lines correspond to log<sub>2</sub> values of +0.33 and 0.33.</p></div></div>                        <a id="87194255" scrollto-destination="87194255"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Comparison of outlier clones in BAC array CGH selfself hybridizations of gDNA amplified using NYP primer or random octamers.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th></th><th colspan="2">Primers for whole genome amplfication</th></tr><tr><th></th><th colspan="2"><hr /></th></tr><tr><th></th><th>NYP</th><th>Random octamers</th></tr></thead><tbody><tr><td>No. of outlier BAC clones</td><td>18</td><td>65</td></tr><tr><td>% of outlier BAC clones</td><td>0.73</td><td>2.64</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th></th><th colspan="2">Primers for whole genome amplfication</th></tr><tr><th></th><th colspan="2"><hr /></th></tr><tr><th></th><th>NYP</th><th>Random octamers</th></tr></thead><tbody><tr><td>No. of outlier BAC clones</td><td>18</td><td>65</td></tr><tr><td>% of outlier BAC clones</td><td>0.73</td><td>2.64</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Normal human lymphocyte gDNA was subjected to primer extension amplification, labelled by nick translation and hybridized on 2464-clone arrays. WGA using NYP primer generated significantly fewer false copy number changes (<em>P</em> &lt; 0.0001, Fisher's exact test).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194255" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Comparison of outlier clones in BAC array CGH selfself hybridizations of gDNA amplified using NYP primer or random octamers.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th></th><th colspan="2">Primers for whole genome amplfication</th></tr><tr><th></th><th colspan="2"><hr /></th></tr><tr><th></th><th>NYP</th><th>Random octamers</th></tr></thead><tbody><tr><td>No. of outlier BAC clones</td><td>18</td><td>65</td></tr><tr><td>% of outlier BAC clones</td><td>0.73</td><td>2.64</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th></th><th colspan="2">Primers for whole genome amplfication</th></tr><tr><th></th><th colspan="2"><hr /></th></tr><tr><th></th><th>NYP</th><th>Random octamers</th></tr></thead><tbody><tr><td>No. of outlier BAC clones</td><td>18</td><td>65</td></tr><tr><td>% of outlier BAC clones</td><td>0.73</td><td>2.64</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Normal human lymphocyte gDNA was subjected to primer extension amplification, labelled by nick translation and hybridized on 2464-clone arrays. WGA using NYP primer generated significantly fewer false copy number changes (<em>P</em> &lt; 0.0001, Fisher's exact test).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194255" target="_blank">View Large</a></div></div><p>We next evaluated the performance of primer extension amplification on degraded gDNA for STR genotyping. Normal human lymphocyte gDNA was digested with DNase I for 230 min to generate five different size ranges: (i) 15003000 bp, (ii) 8001500 bp, (iii) 500800 bp, (iv) 200500 bp and (v) &lt;200 bp (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5</a>). DNA in each range was gel purified after electrophoresis on 1% agarose and used as the input template for primer extension amplification using NYP primer or random octamers (<a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Figure 6</a>). <a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Figure 7</a> shows the genotype profile obtained at 15 STR loci and amelogenin XY with whole genome amplified DNA generated by NYP primer (panel B) or random octamers (panel C). Compared with reference undegraded gDNA (<a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Figure 7A</a>), 9, 9 and 8 loci were successfully detected in gDNA that had been degraded to 15003000 bp, 8001500 bp (<a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Figure 7B</a>) and 500800 bp, respectively, and then whole genome amplified with the NYP primer. Seven loci (D8S1179, D3S1358, D19S322, vWA, D13S317, D5S818 and amelogenin) were detected in all three degraded DNA samples. Genotyping of WGA products was not associated with imbalance at biparental alleles if sufficient amplified gDNA was used, and did not increase stutter peaks. In contrast, genotyping the same degraded gDNA after MDA was uniformly unsuccessful in all size ranges and severe imbalances at heterozygous loci were observed (<a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Figure 7D</a>).</p>                        <a id="87194257" scrollto-destination="87194257"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f5.jpeg?Expires=2147483647&Signature=I1p0BT-oNRTF662-VAo2FzT2JCx2~1k55NB3VlnQR~Z0tW1Q1RJb35fDtLjarh9aqf6TNEfGeScqQbdjWens8PuxH0rpygyuDiBUO2TI3mM7nWToFNepHjnVEqQEeAd3tDCocJOFeTTFOYZ-E5GhMoruVBZwCfrL7t8E4-1kgZu60hwZuBsmizC2DQVaizIwaFpLVhV3VW0cmbX8jPsykJ5rggOnhuXnH-G4afH9O~Z0hQ0gzhz4AtfJvAo6r5ql1rY1lkKR7HJHpsWNQ8Z2wEhvCkGsSejmP7YA-36HRDK6z7xxRYsRC7QhpXrZf0gsp~VDEOLlqhhzF9yYw-TB5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNase-digested gDNA was fractionated by agarose gel electrophoresis into five different size bands: (i) 15003000 bp; (ii) 8001500 bp; (iii) 500800 bp; (iv) 200500 bp; (v) &lt;200 bp. Each size range of degraded gDNA was gel purified and used as input DNA for primer extension amplification (shown in Figure 6)." path-from-xml="dsi029f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194257/dsi029f5.jpeg" path-from-xml="dsi029f5.jpeg" target="_blank">View large</a><a section="87194257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f5.jpeg?Expires=2147483647&Signature=XkSD4R4LCufYXmSIdlXXd~QcWjc97abs48YM~8XF-ozxZEcmigeWFXp-cDIZ7FywVDdQ6cr9Z-Gm23~s9CWLBzRUt5GTeZT0WxENlcZXz26QDyGND~A7Gooea~iQirumS7y-tMu134AkQc0zlLpfFthK7m9AAdCgyYBigdFHHYkabUYs54Ml10tilATGP0DzABso0QZI6yTt5r1XFTN0UunlKAM3xBHYKCpnH9PG8SfgqikoRI~FwU~ojXDOJ8VmAjUDf3brE-ius~JkFjNSZ2imZkyIVToBgmMwlwLQ8uOMM9wYGRwQX6SepSVoGCJxEZmjf5g1lWVRno3YkSjm0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194257&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNase-digested gDNA was fractionated by agarose gel electrophoresis into five different size bands: (i) 15003000 bp; (ii) 8001500 bp; (iii) 500800 bp; (iv) 200500 bp; (v) &lt;200 bp. Each size range of degraded gDNA was gel purified and used as input DNA for primer extension amplification (shown in <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Figure 6</a>).</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f5.jpeg?Expires=2147483647&Signature=I1p0BT-oNRTF662-VAo2FzT2JCx2~1k55NB3VlnQR~Z0tW1Q1RJb35fDtLjarh9aqf6TNEfGeScqQbdjWens8PuxH0rpygyuDiBUO2TI3mM7nWToFNepHjnVEqQEeAd3tDCocJOFeTTFOYZ-E5GhMoruVBZwCfrL7t8E4-1kgZu60hwZuBsmizC2DQVaizIwaFpLVhV3VW0cmbX8jPsykJ5rggOnhuXnH-G4afH9O~Z0hQ0gzhz4AtfJvAo6r5ql1rY1lkKR7HJHpsWNQ8Z2wEhvCkGsSejmP7YA-36HRDK6z7xxRYsRC7QhpXrZf0gsp~VDEOLlqhhzF9yYw-TB5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNase-digested gDNA was fractionated by agarose gel electrophoresis into five different size bands: (i) 15003000 bp; (ii) 8001500 bp; (iii) 500800 bp; (iv) 200500 bp; (v) &lt;200 bp. Each size range of degraded gDNA was gel purified and used as input DNA for primer extension amplification (shown in Figure 6)." path-from-xml="dsi029f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194257/dsi029f5.jpeg" path-from-xml="dsi029f5.jpeg" target="_blank">View large</a><a section="87194257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f5.jpeg?Expires=2147483647&Signature=XkSD4R4LCufYXmSIdlXXd~QcWjc97abs48YM~8XF-ozxZEcmigeWFXp-cDIZ7FywVDdQ6cr9Z-Gm23~s9CWLBzRUt5GTeZT0WxENlcZXz26QDyGND~A7Gooea~iQirumS7y-tMu134AkQc0zlLpfFthK7m9AAdCgyYBigdFHHYkabUYs54Ml10tilATGP0DzABso0QZI6yTt5r1XFTN0UunlKAM3xBHYKCpnH9PG8SfgqikoRI~FwU~ojXDOJ8VmAjUDf3brE-ius~JkFjNSZ2imZkyIVToBgmMwlwLQ8uOMM9wYGRwQX6SepSVoGCJxEZmjf5g1lWVRno3YkSjm0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194257&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNase-digested gDNA was fractionated by agarose gel electrophoresis into five different size bands: (i) 15003000 bp; (ii) 8001500 bp; (iii) 500800 bp; (iv) 200500 bp; (v) &lt;200 bp. Each size range of degraded gDNA was gel purified and used as input DNA for primer extension amplification (shown in <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Figure 6</a>).</p></div></div>                        <a id="87194258" scrollto-destination="87194258"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f6.jpeg?Expires=2147483647&Signature=q2iPic4OwWf9RhP7XgD4z5u-mzQtXYCjBHz6tqFg1lLVb7hV~iwG22irpT6lziIC9kDrYDyKPPm5E2a6ycclZeEosQMJi~6NUw7bLV87I63~ntWvuA4j6pwUJ3BN~uReA0x8WdC5yq~-msqoFRp82CaIljMs~703Yl8dOMTqH~MMSmNQXQQdvULYrrPqjz0R6jMki9X6ydncZ9Rh~OQZMK0JDkSKJkR4WPHC0p4OzeQLC8KPFzu89u830JvqD6WFE7QQEDBNxmi05IEvpxqtmTOXeo2auay2eXyGx1DbSTx2Z1YQ5yq2yi1LETsfwbPoACCDeE~EnlldCmqpr5Jumw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Primer extension amplification of degraded gDNA from five different size bands (Figure 5) using random octamers or NYP primer. Each amplified reaction was used for genotyping at 16 STR loci (shown in Figure 7)." path-from-xml="dsi029f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194258/dsi029f6.jpeg" path-from-xml="dsi029f6.jpeg" target="_blank">View large</a><a section="87194258" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f6.jpeg?Expires=2147483647&Signature=0DcdpuYLSN1iTjqT~ZKKRGA7IDerSX35WvKa4ZAeGOcBxMPm~hGncMqBldnhRxsdPBjfGKfvqKK3Ca7hqYlKR7JxEBnRn~gqyYcWSaMiq10vy-o6-MrUDi95m9jDMwnQ13sp-lFKof8mWXxIP2bkT~S0qapt08BEIhbaRpHimdj~BhWeAii0xagQ-~jOX~kqxgqf7dkhQC6-9Uw5UlXLDQ7hRY8uriAxuyGuJBjFALLr0g3Xdlu3GZCtCkMXHbvajbkDOs2BJK0aERFOVaFD~-GlOxPfzy7mouM2xu0nVp9HOg4RE60GZ6W1OHOGD3bxqW17V49bpdRvjHJ5ct9kdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194258&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Primer extension amplification of degraded gDNA from five different size bands (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5</a>) using random octamers or NYP primer. Each amplified reaction was used for genotyping at 16 STR loci (shown in <a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Figure 7</a>).</p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f6.jpeg?Expires=2147483647&Signature=q2iPic4OwWf9RhP7XgD4z5u-mzQtXYCjBHz6tqFg1lLVb7hV~iwG22irpT6lziIC9kDrYDyKPPm5E2a6ycclZeEosQMJi~6NUw7bLV87I63~ntWvuA4j6pwUJ3BN~uReA0x8WdC5yq~-msqoFRp82CaIljMs~703Yl8dOMTqH~MMSmNQXQQdvULYrrPqjz0R6jMki9X6ydncZ9Rh~OQZMK0JDkSKJkR4WPHC0p4OzeQLC8KPFzu89u830JvqD6WFE7QQEDBNxmi05IEvpxqtmTOXeo2auay2eXyGx1DbSTx2Z1YQ5yq2yi1LETsfwbPoACCDeE~EnlldCmqpr5Jumw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Primer extension amplification of degraded gDNA from five different size bands (Figure 5) using random octamers or NYP primer. Each amplified reaction was used for genotyping at 16 STR loci (shown in Figure 7)." path-from-xml="dsi029f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194258/dsi029f6.jpeg" path-from-xml="dsi029f6.jpeg" target="_blank">View large</a><a section="87194258" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f6.jpeg?Expires=2147483647&Signature=0DcdpuYLSN1iTjqT~ZKKRGA7IDerSX35WvKa4ZAeGOcBxMPm~hGncMqBldnhRxsdPBjfGKfvqKK3Ca7hqYlKR7JxEBnRn~gqyYcWSaMiq10vy-o6-MrUDi95m9jDMwnQ13sp-lFKof8mWXxIP2bkT~S0qapt08BEIhbaRpHimdj~BhWeAii0xagQ-~jOX~kqxgqf7dkhQC6-9Uw5UlXLDQ7hRY8uriAxuyGuJBjFALLr0g3Xdlu3GZCtCkMXHbvajbkDOs2BJK0aERFOVaFD~-GlOxPfzy7mouM2xu0nVp9HOg4RE60GZ6W1OHOGD3bxqW17V49bpdRvjHJ5ct9kdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194258&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Primer extension amplification of degraded gDNA from five different size bands (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5</a>) using random octamers or NYP primer. Each amplified reaction was used for genotyping at 16 STR loci (shown in <a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Figure 7</a>).</p></div></div>                        <a id="87194259" scrollto-destination="87194259"></a>
<div id="fig7" data-id="fig7" class="fig fig-section"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f7.jpeg?Expires=2147483647&Signature=L00K-TruBzcJd0OOT5YoZOXyb7ZujuqANpEhg~RNKJrc87OnDmdjqRUd3gICZMmj7tttrJDYU55MjVVFh~AZ16xCPsz7t3TZodvTkjnc4Wh3ZWimDfF6ObXRZXhjY8E9fn5e7cSfirfy9E4RQQKtex9GiYEPLnMWZ4Gl~3xwBN6VMsOGzGe5QC6pXlCTKN8tZvtbxgYT-sE3thc6HZ2gSfxoFy01rwJSPYm6FTWPg-TXuPw3eSFRpzgiyF8KNY9~RM68GgQc-6pRQ3Oub9KIS9~onRP1vKM7MJzeZb1m5cwvcLh-itOURmEZ9ySZvq3~4ko3XsyA9-Tsrl7eytuz~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genotyping profiles at 16 simple tandem repeat loci (AmpFISTR Identifiler panel, Applied Biosystems, USA) of gDNA degraded to the size range of 8001500 bp (Figure 5) (A) without WGA; (B) after primer extension amplification using NYP primer; (C) after primer extension amplification using random octamers; and (D) after MDA (GenomiPhi Kit, Amersham Biosciences). A colour version of figure 7 can be found online as supplementary data at " path-from-xml="dsi029f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194259/dsi029f7.jpeg" path-from-xml="dsi029f7.jpeg" target="_blank">View large</a><a section="87194259" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f7.jpeg?Expires=2147483647&Signature=mcV1o5iefqj1YzKt3VVNNkVil-ksZAfZH0nglWsYcoG6EmLYHgGOPN96aok3AwBy1-lBVOzHnUumESko94dL-FVF4vDpSIDwXY3q139IroOAxhGmBm7miEmm~4pCnLvWhMaABzPCjtO8I~KSGdQ6-ujLoisk6NIWhnqKOYlxAnsid143dEKHVzovCs4dF6s9P~f44vGk48NFM9Lre2K9svBdXipUtsw~JdPHVolfvLgHRZARxSvxgY8gOZvN-eeu6A~u6irH1BorsZv~iktqx3l-JwU9A7Pm9lyzSY6ehhyMLmI4-Yk6I9PhJHltN51ugKsT292h~RzuV6DJ4fGQ6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194259&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genotyping profiles at 16 simple tandem repeat loci (AmpF<em>I</em>STR<sup></sup> Identifiler<sup></sup> panel, Applied Biosystems, USA) of gDNA degraded to the size range of 8001500 bp (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5</a>) (<strong>A</strong>) without WGA; (<strong>B</strong>) after primer extension amplification using NYP primer; (<strong>C</strong>) after primer extension amplification using random octamers; and (<strong>D</strong>) after MDA (GenomiPhi Kit, Amersham Biosciences). A colour version of figure 7 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div><div content-id="fig7" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f7.jpeg?Expires=2147483647&Signature=L00K-TruBzcJd0OOT5YoZOXyb7ZujuqANpEhg~RNKJrc87OnDmdjqRUd3gICZMmj7tttrJDYU55MjVVFh~AZ16xCPsz7t3TZodvTkjnc4Wh3ZWimDfF6ObXRZXhjY8E9fn5e7cSfirfy9E4RQQKtex9GiYEPLnMWZ4Gl~3xwBN6VMsOGzGe5QC6pXlCTKN8tZvtbxgYT-sE3thc6HZ2gSfxoFy01rwJSPYm6FTWPg-TXuPw3eSFRpzgiyF8KNY9~RM68GgQc-6pRQ3Oub9KIS9~onRP1vKM7MJzeZb1m5cwvcLh-itOURmEZ9ySZvq3~4ko3XsyA9-Tsrl7eytuz~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genotyping profiles at 16 simple tandem repeat loci (AmpFISTR Identifiler panel, Applied Biosystems, USA) of gDNA degraded to the size range of 8001500 bp (Figure 5) (A) without WGA; (B) after primer extension amplification using NYP primer; (C) after primer extension amplification using random octamers; and (D) after MDA (GenomiPhi Kit, Amersham Biosciences). A colour version of figure 7 can be found online as supplementary data at " path-from-xml="dsi029f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194259/dsi029f7.jpeg" path-from-xml="dsi029f7.jpeg" target="_blank">View large</a><a section="87194259" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f7.jpeg?Expires=2147483647&Signature=mcV1o5iefqj1YzKt3VVNNkVil-ksZAfZH0nglWsYcoG6EmLYHgGOPN96aok3AwBy1-lBVOzHnUumESko94dL-FVF4vDpSIDwXY3q139IroOAxhGmBm7miEmm~4pCnLvWhMaABzPCjtO8I~KSGdQ6-ujLoisk6NIWhnqKOYlxAnsid143dEKHVzovCs4dF6s9P~f44vGk48NFM9Lre2K9svBdXipUtsw~JdPHVolfvLgHRZARxSvxgY8gOZvN-eeu6A~u6irH1BorsZv~iktqx3l-JwU9A7Pm9lyzSY6ehhyMLmI4-Yk6I9PhJHltN51ugKsT292h~RzuV6DJ4fGQ6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194259&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genotyping profiles at 16 simple tandem repeat loci (AmpF<em>I</em>STR<sup></sup> Identifiler<sup></sup> panel, Applied Biosystems, USA) of gDNA degraded to the size range of 8001500 bp (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5</a>) (<strong>A</strong>) without WGA; (<strong>B</strong>) after primer extension amplification using NYP primer; (<strong>C</strong>) after primer extension amplification using random octamers; and (<strong>D</strong>) after MDA (GenomiPhi Kit, Amersham Biosciences). A colour version of figure 7 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div><p>The ability of primer extension WGA to accurately detect single nucleotide mutations was investigated in a series of 18 clinical samples of lymphocyte gDNA from genetically characterized cases of -thalassemia. Each exon-specific PCR product obtained after primer extension amplification with NYP primer was sequenced and the correct mutation was detected in 16 cases (89% accuracy; <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>). Representative DNA sequence chromatograms of whole genome amplified samples are shown in <a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Figure 8</a>.</p>                        <a id="87194261" scrollto-destination="87194261"></a>
<div id="fig8" data-id="fig8" class="fig fig-section"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f8.jpeg?Expires=2147483647&Signature=fD-~fYuFAFe0FUn7OFAy8KeE5bucv67fokaWGXLwX-ozenyptzp0YTLpLne8Noi5DUhcdGqkWINBQGd-0QE2PMJxvOeU77NdKkrxJLzyCUTUMqDndClXZbc5wXlTBsW-FSifbXaQLsd0ab5lU7~JIEjkH1zY5v5WII05fhJaC1z-hVixprigJP7Bc-jKYfR8pvNTtbrKW5vrvQtN-dexmuMXSJjBdlkl8PAQWqwRopNlOQhdDvtjVG63~AlQQrriASoHik~KYBZD3Fg6YFOoHy8LE~fAbldw6qH~gPhZEVr~LCS7S2NfjYA8yrSEI0fGwImRV8iZhNAh-MViaGn1OA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative examples of whole genome amplified DNA clinical samples showing detection of a single nucleotide substitution (A and B), insertion (C) and deletion (D). All samples were heterozygous. A colour version of figure 8 can be found online as supplementary data at " path-from-xml="dsi029f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194261/dsi029f8.jpeg" path-from-xml="dsi029f8.jpeg" target="_blank">View large</a><a section="87194261" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f8.jpeg?Expires=2147483647&Signature=4exEvoLwHnuLcDvk2HRA6iJNx2QhTqyEfBoIcc5igk~b-N4Sc6uR5R8FY3kA4SJPNgj6DgTy59AbTJRErHbPyKl85WtR8PQIPntnVaGkkGLZAU2pa75A50iLdSfZipyXc6j9~ka2GF4eVctZWeoYIcgjyQp~SlsnwYJxwv04--ktrMXkYORkNxpNSh5OsYDP5LDT5xM7mFGWR4tg1OcHKLM5dzGM5s47nx8U0ofvu8VUOlO9VBWo1QsPkMib~yist2N1yFlcxmwLdr77mCpmNqa1EiQr1yOVRbb1AstDgTozn1Yrm8SbBTI9YXpOkyYsOJ61S4bTbecg07p~Ih5a7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194261&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative examples of whole genome amplified DNA clinical samples showing detection of a single nucleotide substitution (<strong>A</strong> and <strong>B</strong>), insertion (<strong>C</strong>) and deletion (<strong>D</strong>). All samples were heterozygous. A colour version of figure 8 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div><div content-id="fig8" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/m_dsi029f8.jpeg?Expires=2147483647&Signature=fD-~fYuFAFe0FUn7OFAy8KeE5bucv67fokaWGXLwX-ozenyptzp0YTLpLne8Noi5DUhcdGqkWINBQGd-0QE2PMJxvOeU77NdKkrxJLzyCUTUMqDndClXZbc5wXlTBsW-FSifbXaQLsd0ab5lU7~JIEjkH1zY5v5WII05fhJaC1z-hVixprigJP7Bc-jKYfR8pvNTtbrKW5vrvQtN-dexmuMXSJjBdlkl8PAQWqwRopNlOQhdDvtjVG63~AlQQrriASoHik~KYBZD3Fg6YFOoHy8LE~fAbldw6qH~gPhZEVr~LCS7S2NfjYA8yrSEI0fGwImRV8iZhNAh-MViaGn1OA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative examples of whole genome amplified DNA clinical samples showing detection of a single nucleotide substitution (A and B), insertion (C) and deletion (D). All samples were heterozygous. A colour version of figure 8 can be found online as supplementary data at " path-from-xml="dsi029f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194261/dsi029f8.jpeg" path-from-xml="dsi029f8.jpeg" target="_blank">View large</a><a section="87194261" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029f8.jpeg?Expires=2147483647&Signature=4exEvoLwHnuLcDvk2HRA6iJNx2QhTqyEfBoIcc5igk~b-N4Sc6uR5R8FY3kA4SJPNgj6DgTy59AbTJRErHbPyKl85WtR8PQIPntnVaGkkGLZAU2pa75A50iLdSfZipyXc6j9~ka2GF4eVctZWeoYIcgjyQp~SlsnwYJxwv04--ktrMXkYORkNxpNSh5OsYDP5LDT5xM7mFGWR4tg1OcHKLM5dzGM5s47nx8U0ofvu8VUOlO9VBWo1QsPkMib~yist2N1yFlcxmwLdr77mCpmNqa1EiQr1yOVRbb1AstDgTozn1Yrm8SbBTI9YXpOkyYsOJ61S4bTbecg07p~Ih5a7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87194261&ar=344797&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi029f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative examples of whole genome amplified DNA clinical samples showing detection of a single nucleotide substitution (<strong>A</strong> and <strong>B</strong>), insertion (<strong>C</strong>) and deletion (<strong>D</strong>). All samples were heterozygous. A colour version of figure 8 can be found online as supplementary data at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p></div></div>                        <a id="87194262" scrollto-destination="87194262"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Detection of single base mutations in WGA samples.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample</th><th>-Globin gene mutation</th><th>Detection on amplified gDNA</th></tr></thead><tbody><tr><td>1</td><td>CAP +1(AC)</td><td>No</td></tr><tr><td>2</td><td>HbC (CD6 GA)</td><td>No</td></tr><tr><td>3</td><td>Initiation CD (TG)</td><td>Yes</td></tr><tr><td>4</td><td>28 (AG)</td><td>Yes</td></tr><tr><td>5</td><td>HbE (CD26 GAGAAG)</td><td>Yes</td></tr><tr><td>6</td><td>IVS1-5 (GC)</td><td>Yes</td></tr><tr><td>7</td><td>IVS1-1 (GT)</td><td>Yes</td></tr><tr><td>8</td><td>CD71/72 (+A)</td><td>Yes</td></tr><tr><td>9</td><td>CD43 (GT)</td><td>Yes</td></tr><tr><td>10</td><td>CD16 (C)</td><td>Yes</td></tr><tr><td>11</td><td>CD8/9 (+G)</td><td>Yes</td></tr><tr><td>12</td><td>HbS (CD6 GAGGTG)</td><td>Yes</td></tr><tr><td>13</td><td>29 (AG)</td><td>Yes</td></tr><tr><td>14</td><td>CD15 (GA)</td><td>Yes</td></tr><tr><td>15</td><td>CD27/28 (+C)</td><td>Yes</td></tr><tr><td>16</td><td>CD19 (AG)</td><td>Yes</td></tr><tr><td>17</td><td>CD14/15 (+G)</td><td>Yes</td></tr><tr><td>18</td><td>CD35 (C)</td><td>Yes</td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample</th><th>-Globin gene mutation</th><th>Detection on amplified gDNA</th></tr></thead><tbody><tr><td>1</td><td>CAP +1(AC)</td><td>No</td></tr><tr><td>2</td><td>HbC (CD6 GA)</td><td>No</td></tr><tr><td>3</td><td>Initiation CD (TG)</td><td>Yes</td></tr><tr><td>4</td><td>28 (AG)</td><td>Yes</td></tr><tr><td>5</td><td>HbE (CD26 GAGAAG)</td><td>Yes</td></tr><tr><td>6</td><td>IVS1-5 (GC)</td><td>Yes</td></tr><tr><td>7</td><td>IVS1-1 (GT)</td><td>Yes</td></tr><tr><td>8</td><td>CD71/72 (+A)</td><td>Yes</td></tr><tr><td>9</td><td>CD43 (GT)</td><td>Yes</td></tr><tr><td>10</td><td>CD16 (C)</td><td>Yes</td></tr><tr><td>11</td><td>CD8/9 (+G)</td><td>Yes</td></tr><tr><td>12</td><td>HbS (CD6 GAGGTG)</td><td>Yes</td></tr><tr><td>13</td><td>29 (AG)</td><td>Yes</td></tr><tr><td>14</td><td>CD15 (GA)</td><td>Yes</td></tr><tr><td>15</td><td>CD27/28 (+C)</td><td>Yes</td></tr><tr><td>16</td><td>CD19 (AG)</td><td>Yes</td></tr><tr><td>17</td><td>CD14/15 (+G)</td><td>Yes</td></tr><tr><td>18</td><td>CD35 (C)</td><td>Yes</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Lymphocyte DNA samples from 18 -thalassemia patients with known -globin gene mutations were amplified by primer extension amplification with NYP primer. Exon-specific PCR was performed on each amplified sample and the PCR product was sequenced. Of the 18 WGA (whole genome amplification) samples, the correct mutation was detected in 16 (89%).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194262" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Detection of single base mutations in WGA samples.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample</th><th>-Globin gene mutation</th><th>Detection on amplified gDNA</th></tr></thead><tbody><tr><td>1</td><td>CAP +1(AC)</td><td>No</td></tr><tr><td>2</td><td>HbC (CD6 GA)</td><td>No</td></tr><tr><td>3</td><td>Initiation CD (TG)</td><td>Yes</td></tr><tr><td>4</td><td>28 (AG)</td><td>Yes</td></tr><tr><td>5</td><td>HbE (CD26 GAGAAG)</td><td>Yes</td></tr><tr><td>6</td><td>IVS1-5 (GC)</td><td>Yes</td></tr><tr><td>7</td><td>IVS1-1 (GT)</td><td>Yes</td></tr><tr><td>8</td><td>CD71/72 (+A)</td><td>Yes</td></tr><tr><td>9</td><td>CD43 (GT)</td><td>Yes</td></tr><tr><td>10</td><td>CD16 (C)</td><td>Yes</td></tr><tr><td>11</td><td>CD8/9 (+G)</td><td>Yes</td></tr><tr><td>12</td><td>HbS (CD6 GAGGTG)</td><td>Yes</td></tr><tr><td>13</td><td>29 (AG)</td><td>Yes</td></tr><tr><td>14</td><td>CD15 (GA)</td><td>Yes</td></tr><tr><td>15</td><td>CD27/28 (+C)</td><td>Yes</td></tr><tr><td>16</td><td>CD19 (AG)</td><td>Yes</td></tr><tr><td>17</td><td>CD14/15 (+G)</td><td>Yes</td></tr><tr><td>18</td><td>CD35 (C)</td><td>Yes</td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample</th><th>-Globin gene mutation</th><th>Detection on amplified gDNA</th></tr></thead><tbody><tr><td>1</td><td>CAP +1(AC)</td><td>No</td></tr><tr><td>2</td><td>HbC (CD6 GA)</td><td>No</td></tr><tr><td>3</td><td>Initiation CD (TG)</td><td>Yes</td></tr><tr><td>4</td><td>28 (AG)</td><td>Yes</td></tr><tr><td>5</td><td>HbE (CD26 GAGAAG)</td><td>Yes</td></tr><tr><td>6</td><td>IVS1-5 (GC)</td><td>Yes</td></tr><tr><td>7</td><td>IVS1-1 (GT)</td><td>Yes</td></tr><tr><td>8</td><td>CD71/72 (+A)</td><td>Yes</td></tr><tr><td>9</td><td>CD43 (GT)</td><td>Yes</td></tr><tr><td>10</td><td>CD16 (C)</td><td>Yes</td></tr><tr><td>11</td><td>CD8/9 (+G)</td><td>Yes</td></tr><tr><td>12</td><td>HbS (CD6 GAGGTG)</td><td>Yes</td></tr><tr><td>13</td><td>29 (AG)</td><td>Yes</td></tr><tr><td>14</td><td>CD15 (GA)</td><td>Yes</td></tr><tr><td>15</td><td>CD27/28 (+C)</td><td>Yes</td></tr><tr><td>16</td><td>CD19 (AG)</td><td>Yes</td></tr><tr><td>17</td><td>CD14/15 (+G)</td><td>Yes</td></tr><tr><td>18</td><td>CD35 (C)</td><td>Yes</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Lymphocyte DNA samples from 18 -thalassemia patients with known -globin gene mutations were amplified by primer extension amplification with NYP primer. Exon-specific PCR was performed on each amplified sample and the PCR product was sequenced. Of the 18 WGA (whole genome amplification) samples, the correct mutation was detected in 16 (89%).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194262" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87194263 id="87194263" class="section-title" >4. Discussion</h2>
<p>We have reported a simple isothermal WGA method based on primer extension that performs well for several different genomic analyses, i.e. chromosomal and BAC array CGH, STR genotyping and single base mutation detection. The method is tolerant of degraded DNA and works well on DNA extracted from fixed tissues. It does not require specially modified primers (unlike MDA), ligation steps (unlike balanced PCR, ligation-mediated PCR and rolling circle amplification techniques) or library construction (unlike Omniplex technology).</p><p>Parsing the complete human genome sequence made it possible to identify a short primer sequence that is uniformly distributed throughout the entire human genome, thus favouring uniform amplification while at the same time having minimal overlap to avoid over-amplifying certain regions. We further specified that, if used as a primer for PCR-based WGA, the selected sequence should amplify products of 23 kb. The hexamer, TGAGAT, met all these criteria but was efficient in priming gDNA only when the 5 end was extended by three degenerate bases. Hence, the WGA primer we used for DOP-PCR and for primer extension amplification was 5-NNNTGAGAT-3.</p><p>WGA products obtained by DOP-PCR using NYP primer gave minimal deviations from a balanced copy number profile in selfself hybridization on chromosome CGH compared with DOP-PCR with the widely used primer of Telenius <em>et al.</em><a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> that yielded multiple regions of spurious deletions (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Figure 1</a>). Identical selfself hybridizations on BAC array CGH also showed that, compared with random octamers, NYP primer generated a more representative and less biased amplification of gDNA (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Figure 4</a>).</p><p>Although isothermal random primer extension is commonly employed for DNA labelling, it has not been fully investigated as a method for WGA. Our data showed that this reaction could amplify gDNA by several thousand times (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Figure 2</a>), confirming an earlier study.<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a></p><p>gDNA extracted from FFPE tissues is notoriously difficult to be amplified and analysed using molecular techniques probably because of DNA cross-linking and degradation. We were motivated to test our approach on FFPE tissue because archived human tissues in paraffin blocks are a rich research resource. Our results show that the major characteristic copy number abnormalities can be reproduced in gDNA extracted from FFPE tissue although the process of tissue preservation itself, independent of WGA, introduces artefactual copy number alterations (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3A and B</a>). Despite this, reasonable fidelity of copy number profile was achieved in gDNA amplified with NYP primer from FFPE tumour tissue compared with the same freshly frozen tumour tissue (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Figure 3A and C</a>).</p><p>Other potentially useful applications of WGA are in forensic science and genetic diagnosis, where samples are frequently of poor quality or available in minute quantities. We were able to genotype DNA that had been degraded to 500800 bp at a minimum of six standard simple tandem repeat loci using 8 ng of amplified gDNA representing 56 pg of original degraded input DNA, giving a conservatively estimated power of discrimination of 0.999998.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> Primers designed to generate smaller amplicons of STR loci have been shown to successfully genotype DNA degraded to &lt;126 bp when standard methods fail.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> Such an approach, combined with primer extension WGA, should be useful in situations requiring the analysis of highly degraded DNA.</p><p>In summary, we have shown that primer extension amplification using a bioinformatically optimized universal primer is a simple and efficient isothermal method for WGA. The same primer can also be used for PCR-based genome amplification and achieves less biased amplification than a widely used DOP-PCR primer.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> Isothermal primer extension amplification does not require extensive sample processing and manipulation or chemically modified primers. The method we have described performed well in chromosome and BAC array CGH using gDNA from both fresh and archived tissues. It generated usable STR genotype profiles from degraded gDNA and correctly detected a high proportion of single nucleotide mutations in clinical samples. These performance characteristics should make this method useful for practical application in genome science, genomic medicine and forensic science.</p><p>The authors thank Dr Alvin Eng for guidance with array CGH experiments, Dr Ann Lee for use of the rocker platform, Dr Patrick Tan for use of the Axon GenePix scanner, Dr Balram Chowbay's laboratory for assistance in sequencing and Mark EH Tan for statistical analysis. This work was supported by the National Medical Research Council, Singapore.</p>                            <h2 scrollto-destination=87194273 id="87194273" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weiss</div> <div class="given-names">M. M.</div></div>, <div class="name"><div class="surname">Kuipers</div> <div class="given-names">E. J.</div></div>, <div class="name"><div class="surname">Postma</div> <div class="given-names">C.</div></div>,et al.</div><div class="article-title">Genomic profiling of gastric cancer predicts lymph node status and survival</div>,<div class="source">Oncogene</div>,<div class="year">2003</div>,vol.<div class="volume">22</div>(pg.<div class="fpage">1872</div>-<div class="lpage">1879</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20profiling%20of%20gastric%20cancer%20predicts%20lymph%20node%20status%20and%20survival&amp;author=M.%20M.%20Weiss&amp;author=E.%20J.%20Kuipers&amp;author=C.%20Postma&amp;publication_year=2003&amp;journal=Oncogene&amp;volume=22&amp;pages=1872-1879" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1206350" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1206350" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1206350"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12660823" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bertram</div> <div class="given-names">L.</div></div>, <div class="name"><div class="surname">Parkinson</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">McQueen</div> <div class="given-names">M. B.</div></div>,et al.</div><div class="article-title">Further evidence for LBP-1c/CP2/LSF association in Alzheimer's disease families</div>,<div class="source">J. Med. Genet.</div>,<div class="year">2005</div>,vol.<div class="volume">42</div>(pg.<div class="fpage">857</div>-<div class="lpage">862</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Further%20evidence%20for%20LBP-1c%2FCP2%2FLSF%20association%20in%20Alzheimer's%20disease%20families&amp;author=L.%20Bertram&amp;author=M.%20Parkinson&amp;author=M.%20B.%20McQueen&amp;publication_year=2005&amp;journal=J.%20Med.%20Genet.&amp;volume=42&amp;pages=857-862" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/jmg.2004.024596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fjmg.2004.024596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fjmg.2004.024596"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16272261" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moretti</div> <div class="given-names">T. R.</div></div>, <div class="name"><div class="surname">Baumstark</div> <div class="given-names">A. L.</div></div>, <div class="name"><div class="surname">Defenbaugh</div> <div class="given-names">D. A.</div></div>, <div class="name"><div class="surname">Keys</div> <div class="given-names">K. M.</div></div>, <div class="name"><div class="surname">Smerick</div> <div class="given-names">J. B.</div></div>, <div class="name"><div class="surname">Budowle</div> <div class="given-names">B.</div></div>.</div><div class="article-title">Validation of short tandem repeats (STRs) for forensic usage: performance testing of fluorescent multiplex STR systems and analysis of authentic and simulated forensic samples</div>,<div class="source">J. Forensic Sci.</div>,<div class="year">2001</div>,vol.<div class="volume">46</div>(pg.<div class="fpage">647</div>-<div class="lpage">660</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Validation%20of%20short%20tandem%20repeats%20(STRs)%20for%20forensic%20usage%3A%20performance%20testing%20of%20fluorescent%20multiplex%20STR%20systems%20and%20analysis%20of%20authentic%20and%20simulated%20forensic%20samples&amp;author=T.%20R.%20Moretti&amp;author=A.%20L.%20Baumstark&amp;author=D.%20A.%20Defenbaugh&amp;author=K.%20M.%20Keys&amp;author=J.%20B.%20Smerick&amp;author=B.%20Budowle&amp;publication_year=2001&amp;journal=J.%20Forensic%20Sci.&amp;volume=46&amp;pages=647-660" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11373004" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dean</div> <div class="given-names">F. B.</div></div>, <div class="name"><div class="surname">Hosono</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>,et al.</div><div class="article-title">Comprehen-sive human genome amplification using multiple displacement amplification</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">2002</div>,vol.<div class="volume">99</div>(pg.<div class="fpage">5261</div>-<div class="lpage">5266</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehen-sive%20human%20genome%20amplification%20using%20multiple%20displacement%20amplification&amp;author=F.%20B.%20Dean&amp;author=S.%20Hosono&amp;author=L.%20Fang&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=5261-5266" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.082089499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.082089499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.082089499"> </span></div></div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lage</div> <div class="given-names">J. M.</div></div>, <div class="name"><div class="surname">Leamon</div> <div class="given-names">J. H.</div></div>, <div class="name"><div class="surname">Pejovic</div> <div class="given-names">T.</div></div>,et al.</div><div class="article-title">Whole genome analysis of genetic alterations in small DNA samples using hyperbranched strand displacement amplification and array-CGH</div>,<div class="source">Genome Res.</div>,<div class="year">2003</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">294</div>-<div class="lpage">307</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20analysis%20of%20genetic%20alterations%20in%20small%20DNA%20samples%20using%20hyperbranched%20strand%20displacement%20amplification%20and%20array-CGH&amp;author=J.%20M.%20Lage&amp;author=J.%20H.%20Leamon&amp;author=T.%20Pejovic&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=294-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.377203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.377203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.377203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12566408" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Telenius</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Carter</div> <div class="given-names">N. P.</div></div>, <div class="name"><div class="surname">Bebb</div> <div class="given-names">C. E.</div></div>, <div class="name"><div class="surname">Nordenskjold</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Ponder</div> <div class="given-names">B. A. J.</div></div>, <div class="name"><div class="surname">Tunnacliffe</div> <div class="given-names">A.</div></div>.</div><div class="article-title">Degenerate oligonucleotide-primed PCR: general amplification of target DNA by a single degenerate primer</div>,<div class="source">Genomics</div>,<div class="year">1992</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">718</div>-<div class="lpage">725</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Degenerate%20oligonucleotide-primed%20PCR%3A%20general%20amplification%20of%20target%20DNA%20by%20a%20single%20degenerate%20primer&amp;author=H.%20Telenius&amp;author=N.%20P.%20Carter&amp;author=C.%20E.%20Bebb&amp;author=M.%20Nordenskjold&amp;author=B.%20A.%20J.%20Ponder&amp;author=A.%20Tunnacliffe&amp;publication_year=1992&amp;journal=Genomics&amp;volume=13&amp;pages=718-725" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(92)90147-K" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(92)90147-K" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(92)90147-K"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1639399" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name"><div class="surname">Cui</div> <div class="given-names">X.</div></div>, <div class="name"><div class="surname">Schmitt</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Hubert</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Navidi</div> <div class="given-names">W.</div></div>, <div class="name"><div class="surname">Arnheim</div> <div class="given-names">N.</div></div>.</div><div class="article-title">Whole genome amplification from a single cell: implications for genetic analysis</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">1992</div>,vol.<div class="volume">89</div>(pg.<div class="fpage">5847</div>-<div class="lpage">5851</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20amplification%20from%20a%20single%20cell%3A%20implications%20for%20genetic%20analysis&amp;author=L.%20Zhang&amp;author=X.%20Cui&amp;author=K.%20Schmitt&amp;author=R.%20Hubert&amp;author=W.%20Navidi&amp;author=N.%20Arnheim&amp;publication_year=1992&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=5847-5851" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.13.5847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.13.5847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.13.5847"> </span></div></div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klein</div> <div class="given-names">C. A.</div></div>, <div class="name"><div class="surname">Schmidt-Kittler</div> <div class="given-names">O.</div></div>, <div class="name"><div class="surname">Schardt</div> <div class="given-names">J. A.</div></div>, <div class="name"><div class="surname">Pantel</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Speicher</div> <div class="given-names">M. R.</div></div>, <div class="name"><div class="surname">Riethmuller</div> <div class="given-names">G.</div></div>.</div><div class="article-title">Comparative genomic hybridization, loss of heterozygosity, and DNA sequence analysis of single cells</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">1999</div>,vol.<div class="volume">96</div>(pg.<div class="fpage">4494</div>-<div class="lpage">4499</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20hybridization%2C%20loss%20of%20heterozygosity%2C%20and%20DNA%20sequence%20analysis%20of%20single%20cells&amp;author=C.%20A.%20Klein&amp;author=O.%20Schmidt-Kittler&amp;author=J.%20A.%20Schardt&amp;author=K.%20Pantel&amp;author=M.%20R.%20Speicher&amp;author=G.%20Riethmuller&amp;publication_year=1999&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=4494-4499" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.8.4494" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.8.4494" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.8.4494"> </span></div></div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">G.</div></div>, <div class="name"><div class="surname">Brennan</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Rook</div> <div class="given-names">M.</div></div>,et al.</div><div class="article-title">Balanced-PCR amplification allows unbiased identification of genomic copy changes in minute cell and tissue samples</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">2004</div>,vol.<div class="volume">32</div>pg.<div class="fpage">e76</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Balanced-PCR%20amplification%20allows%20unbiased%20identification%20of%20genomic%20copy%20changes%20in%20minute%20cell%20and%20tissue%20samples&amp;author=G.%20Wang&amp;author=C.%20Brennan&amp;author=M.%20Rook&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=e76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gnh070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgnh070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgnh070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15155823" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">G.</div></div>, <div class="name"><div class="surname">Maher</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Brennan</div> <div class="given-names">C.</div></div>,et al.</div><div class="article-title">DNA amplification method tolerant to sample degradation</div>,<div class="source">Genome Res.</div>,<div class="year">2004</div>,vol.<div class="volume">14</div>(pg.<div class="fpage">2357</div>-<div class="lpage">2366</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20amplification%20method%20tolerant%20to%20sample%20degradation&amp;author=G.%20Wang&amp;author=E.%20Maher&amp;author=C.%20Brennan&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2357-2366" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2813404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2813404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2813404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15520297" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Langmore</div> <div class="given-names">J. P.</div></div>.</div><div class="article-title">Rubicon Genomics, Inc.</div>,<div class="source">Pharmacogenomics</div>,<div class="year">2002</div>,vol.<div class="volume">3</div>(pg.<div class="fpage">557</div>-<div class="lpage">560</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rubicon%20Genomics%2C%20Inc.&amp;author=J.%20P.%20Langmore&amp;publication_year=2002&amp;journal=Pharmacogenomics&amp;volume=3&amp;pages=557-560" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1517/14622416.3.4.557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1517%2F14622416.3.4.557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1517%2F14622416.3.4.557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12164778" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feinberg</div> <div class="given-names">A.P.</div></div>, <div class="name"><div class="surname">Vogelstein</div> <div class="given-names">B.</div></div>.</div><div class="article-title">A technique for radiolabeling DNA restriction endonuclease fragments to high specific activity</div>,<div class="source">Anal. Biochem.</div>,<div class="year">1983</div>,vol.<div class="volume">132</div>(pg.<div class="fpage">6</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20technique%20for%20radiolabeling%20DNA%20restriction%20endonuclease%20fragments%20to%20high%20specific%20activity&amp;author=A.P.%20Feinberg&amp;author=B.%20Vogelstein&amp;publication_year=1983&amp;journal=Anal.%20Biochem.&amp;volume=132&amp;pages=6-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0003-2697(83)90418-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0003-2697(83)90418-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0003-2697(83)90418-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6312838" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mackey</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Darfler</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Nisson</div> <div class="given-names">P.</div></div>, <div class="name"><div class="surname">Rashtchian</div> <div class="given-names">A.</div></div>.</div><div class="article-title">Use of random primer extension for concurrent amplification and nonradioactive labeling of nucleic acids</div>,<div class="source">Anal. Biochem.</div>,<div class="year">1993</div>,vol.<div class="volume">212</div>(pg.<div class="fpage">428</div>-<div class="lpage">435</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20random%20primer%20extension%20for%20concurrent%20amplification%20and%20nonradioactive%20labeling%20of%20nucleic%20acids&amp;author=J.%20Mackey&amp;author=M.%20Darfler&amp;author=P.%20Nisson&amp;author=A.%20Rashtchian&amp;publication_year=1993&amp;journal=Anal.%20Biochem.&amp;volume=212&amp;pages=428-435" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/abio.1993.1351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fabio.1993.1351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fabio.1993.1351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8214584" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tay</div> <div class="given-names">S. T.</div></div>, <div class="name"><div class="surname">Leong</div> <div class="given-names">S. H.</div></div>, <div class="name"><div class="surname">Yu</div> <div class="given-names">K.</div></div>,et al.</div><div class="article-title">A combined comparative genomic hybridisation and expression microarray analysis of gastric cancer research reveals novel molecular subtypes</div>,<div class="source">Cancer Res.</div>,<div class="year">2003</div>,vol.<div class="volume">63</div>(pg.<div class="fpage">3309</div>-<div class="lpage">3316</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20combined%20comparative%20genomic%20hybridisation%20and%20expression%20microarray%20analysis%20of%20gastric%20cancer%20research%20reveals%20novel%20molecular%20subtypes&amp;author=S.%20T.%20Tay&amp;author=S.%20H.%20Leong&amp;author=K.%20Yu&amp;publication_year=2003&amp;journal=Cancer%20Res.&amp;volume=63&amp;pages=3309-3316" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12810664" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">A. N.</div></div>, <div class="name"><div class="surname">Tokuyasu</div> <div class="given-names">T. A.</div></div>, <div class="name"><div class="surname">Snijders</div> <div class="given-names">A. M.</div></div>, <div class="name"><div class="surname">Segraves</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Albertson</div> <div class="given-names">D. G.</div></div>, <div class="name"><div class="surname">Pinkel</div> <div class="given-names">D.</div></div>.</div><div class="article-title">Fully automatic quantification of microarray image data</div>,<div class="source">Genome Res.</div>,<div class="year">2002</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">325</div>-<div class="lpage">332</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fully%20automatic%20quantification%20of%20microarray%20image%20data&amp;author=A.%20N.%20Jain&amp;author=T.%20A.%20Tokuyasu&amp;author=A.%20M.%20Snijders&amp;author=R.%20Segraves&amp;author=D.%20G.%20Albertson&amp;author=D.%20Pinkel&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=325-332" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.210902" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.210902" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.210902"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11827952" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Budowle</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Moretti</div> <div class="given-names">T. R.</div></div>, <div class="name"><div class="surname">Baumstark</div> <div class="given-names">A. L.</div></div>, <div class="name"><div class="surname">Defenbaugh</div> <div class="given-names">D. A.</div></div>, <div class="name"><div class="surname">Keys</div> <div class="given-names">K. M.</div></div>.</div><div class="article-title">Population data on the thirteen CODIS core short tandem repeat loci in African Americans, U.S. Caucasians, Hispanics, Bahamians, Jamaicans, and Trinidians</div>,<div class="source">J. Forensic Sci.</div>,<div class="year">1999</div>,vol.<div class="volume">44</div>(pg.<div class="fpage">1277</div>-<div class="lpage">1286</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20data%20on%20the%20thirteen%20CODIS%20core%20short%20tandem%20repeat%20loci%20in%20African%20Americans%2C%20U.S.%20Caucasians%2C%20Hispanics%2C%20Bahamians%2C%20Jamaicans%2C%20and%20Trinidians&amp;author=B.%20Budowle&amp;author=T.%20R.%20Moretti&amp;author=A.%20L.%20Baumstark&amp;author=D.%20A.%20Defenbaugh&amp;author=K.%20M.%20Keys&amp;publication_year=1999&amp;journal=J.%20Forensic%20Sci.&amp;volume=44&amp;pages=1277-1286" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1520/JFS14601J" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1520%2FJFS14601J" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1520%2FJFS14601J"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10582369" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Budowle</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Collin</div> <div class="given-names">P. J.</div></div>, <div class="name"><div class="surname">Dimsoski</div> <div class="given-names">P.</div></div>,et al.</div><div class="article-title">Population data on the STR loci D2S1338 and D19S433</div>,<div class="source">Forensic Sci. Commun.</div>,<div class="year">2001</div>,vol.<div class="volume">3</div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chung</div> <div class="given-names">D. T.</div></div>, <div class="name"><div class="surname">Drabek</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Opel</div> <div class="given-names">K. L.</div></div>, <div class="name"><div class="surname">Butler</div> <div class="given-names">J. M.</div></div>, <div class="name"><div class="surname">McCord</div> <div class="given-names">B. R.</div></div>.</div><div class="article-title">A study on the effects of degradation and template concentration on the amplification efficiency of the STR Miniplex primer sets</div>,<div class="source">J. Forensic Sci.</div>,<div class="year">2004</div>,vol.<div class="volume">49</div>(pg.<div class="fpage">733</div>-<div class="lpage">740</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20study%20on%20the%20effects%20of%20degradation%20and%20template%20concentration%20on%20the%20amplification%20efficiency%20of%20the%20STR%20Miniplex%20primer%20sets&amp;author=D.%20T.%20Chung&amp;author=J.%20Drabek&amp;author=K.%20L.%20Opel&amp;author=J.%20M.%20Butler&amp;author=B.%20R.%20McCord&amp;publication_year=2004&amp;journal=J.%20Forensic%20Sci.&amp;volume=49&amp;pages=733-740" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1520/JFS2003269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1520%2FJFS2003269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1520%2FJFS2003269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15317187" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=344797&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/2/10.1093_dnares_dsi029/1/dsi029_Supplementary_Data.zip?Expires=2147483647&amp;Signature=sU~1pcZzXx-VOyD~ctTruizopcUypF5vHyFK9JysJ0jYAai2w2VlbgOvCrIVDGmiclgmMM2HdfPZxL7SO7CsFPaYwvNN6pRNn1rCzhCjFmWzyooTG8~oK2Pr7aTo3VVoJMiGJmmXDXSo4TSxHTd9KsPSUUneaqezBZf~KedqBH9HFHbQWIu8u9ojg9uUOLZRIas5HDRDtxs4kt6usHnwEtmnOWHInVDFeQL0hDjF0yLveLd8~lEax9L87YDCU-NVtOQLjTynblv3r-6zYDHhlLJ-mj-V683lE~LY0uoTttBnhbEypdg9klFUmBdyQl1LV8t2Zn9E2qlnDpRM3j~nuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">324</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250119700004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi029"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="344797"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250119700004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f2%2f77%2f344797">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000250119700004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F2%2F77%2F344797" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/344797'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Isothermal Method for Whole Genome Amplification of Fresh and Degraded DNA for Comparative Genomic Hybridization, Genotyping and Mutation Detection" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi029");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f2%2f77%2f344797">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

