

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/5/217/351025","@type":"ScholarlyArticle","name":"Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits","datePublished":"2007-11-13","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm020","keywords":["genotype matrix mapping","QTL interaction","genetic variation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/5/217/351025","author":[{"name":"Isobe, Sachiko","affiliation":"1  National Agricultural Research Center for Hokkaido Region  ,  Histujigaoka 1, Toyohira, Sapporo 062-8555  ,  Japan    3  Kazusa DNA Research Institute  ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818  ,  Japan","@type":"Person"},{"name":"Nakaya, Akihiro","affiliation":"2  Department of computational Biology  ,  The University of Tokyo  ,  5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561  ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"3  Kazusa DNA Research Institute  ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818  ,  Japan","@type":"Person"}],"description":"Abstract.   In order to reveal quantitative trait loci (QTL) interactions and the relationship between various interactions in complex traits, we have developed","pageStart":"217","pageEnd":"225","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits" />
    <meta property="og:description" content="Abstract.   In order to reveal quantitative trait loci (QTL) interactions and the relationship between various interactions in complex traits, we have developed" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/5/217/351025" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-11-13" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="National Agricultural Research Center for Hokkaido Region , Histujigaoka 1, Toyohira, Sapporo 062-8555 , Japan" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute , Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 , Japan" /><meta name="citation_author" content="Nakaya, Akihiro" /><meta name="citation_author_institution" content="Department of computational Biology , The University of Tokyo , 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561 , Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute , Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 , Japan" /><meta name="citation_title" content="Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits" /><meta name="citation_firstpage" content="217" /><meta name="citation_lastpage" content="225" /><meta name="citation_doi" content="10.1093/dnares/dsm020" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetic association mapping and genome organization of maize; Curr. Opin. Biotech.;  citation_year=2006;  citation_volume=17;  citation_pages=155-160; " /><meta name="citation_reference" content="citation_title=Quantitative genetic variation: a post-modern view; Hum. Mol. Genet.;  citation_year=2004;  citation_volume=13;  citation_pages=R1-R7; " /><meta name="citation_reference" content="citation_title=Mapping quantitative trait loci in humans: achievements and limitations; J. Clin. Invest.;  citation_year=2005;  citation_volume=115;  citation_pages=1419-1424; " /><meta name="citation_reference" content="citation_title=Genomics of the future: identification of quantitative trait loci in the mouse; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1741-1745; " /><meta name="citation_reference" content="citation_title=Hd1 , a major photoperiod sensitivity quantitative trait locus in rice, is closely related to the Arabidopsis flowering time gene CONSTANS; Plant Cell;  citation_year=2000;  citation_volume=12;  citation_pages=2473-2484; " /><meta name="citation_reference" content="citation_title=Linkage disequilibrium maps and association mapping; J. Clin. Invest;  citation_year=2005;  citation_volume=115;  citation_pages=1425-1430; " /><meta name="citation_reference" content="citation_title=Theoretical basis of separation of multiple linked gene effects on mapping quantitative trait loci; Proc. Natl. Acad. Sci. USA;  citation_year=1993;  citation_volume=90;  citation_pages=10972-10976; " /><meta name="citation_reference" content="citation_title=High resolution of quantitative trait into multiple loci via interval mapping; Genetics;  citation_year=1994;  citation_volume=136;  citation_pages=1447-1445; " /><meta name="citation_reference" content="citation_title=Locating multiple interacting quantitative trait loci using rank-based model selection; Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=1845-1854; " /><meta name="citation_reference" content="citation_title=The Challenge for genetic epidemiologist: how to analyze large number of SNPs in relation to complex diseases; BMC Genetics;  citation_year=2006;  citation_volume=7;  citation_pages=23" /><meta name="citation_reference" content="citation_title=Modeling epistasis of quantitative trait loci using Cckerham's model; Genetics;  citation_year=2002;  citation_volume=160;  citation_pages=1243-1261; " /><meta name="citation_reference" content="citation_title=Modifying the Schwarz Bayesian information criterion to locate multiple interacting quantitative trait loci; Genetics;  citation_year=2004;  citation_volume=167;  citation_pages=989-999; " /><meta name="citation_reference" content="citation_title=Detection of multiple QTL with epistatic effects under a mixed inheritance model in an outbred population; Genet. Sel. Evol.;  citation_year=2004;  citation_volume=36;  citation_pages=415-433; " /><meta name="citation_reference" content="citation_title=Multifactor-dimensionality reduction reveals high-order interactions among estrogen-metabolism genes in sporadic breast cancer; Am. J. Hum. Genet.;  citation_year=2001;  citation_volume=69;  citation_pages=138-147; " /><meta name="citation_reference" content="citation_title=An application of conditional logistic regression and multifactor dimensionality reduction for detecting geneâ€“gene interactions on risk of myocardial infarction: the importance of model validation; BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=49" /><meta name="citation_reference" content="citation_title=Multifactor-dimensionality reduction shows a two locus interaction associated with Type 2 diabetes mellitus; Diabetologia;  citation_year=2004;  citation_volume=47;  citation_pages=549-554; " /><meta name="citation_reference" content="citation_title=Comprehensive structural analysis of the genome of red clover ( Trifolium pratense L.); DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=301-364; " /><meta name="citation_reference" content="citation_title=Swiss Mattenklee landraces, a distinct and diverse genetic resource of red clover ( Trifolium pratense L.); Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=306-315; " /><meta name="citation_reference" content="citation_title=Inference of Population Structure using multilocus genotype data; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-959; " /><meta name="citation_reference" content="citation_title=Tracing synergetic behavior of the QTLs affecting oral glucose tolerance in the OLETF rat; Genome Informatics;  citation_year=1999;  citation_volume=10;  citation_pages=155-165; " /><meta name="citation_reference" content="citation_title=Constrained clusters of gene expression profiles with pathological features; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=3137-3145; " /><meta name="citation_reference" content="citation_title=Test for interaction between two unlinked loci; Am. J. Hum. Genet.;  citation_year=2006;  citation_volume=79;  citation_pages=831-845; " /><meta name="citation_reference" content="citation_title=Identifying quantitative trait locus by genetic background interactions in association studies; Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=553-561; " /><meta name="citation_reference" content="citation_title=QTL analysis of seed yield components in red clover ( Trifolium pratense L.); Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=536-545; " /><meta name="citation_reference" content="citation_title=Genetics of gene expression surveyed in maize, mouse and man; Nature;  citation_year=2003;  citation_volume=422;  citation_pages=297-302; " /><meta name="citation_reference" content="citation_title=eQTL Viewer: visualizing how sequence variation affects genome-wide transcription; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=7-11; " /><meta name="citation_reference" content="citation_title=Genetic dissection of complex traits; Science;  citation_year=1994;  citation_volume=265;  citation_pages=2037-2048; " /><meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=203-208; " /><meta name="citation_reference" content="citation_title=A bias-ed assessment of the use of SNPs in human complex traits; Curr. Opin. Genet. Develop.;  citation_year=2002;  citation_volume=12;  citation_pages=726-734; " /><meta name="citation_reference" content="citation_title=Assessing the power of tag SNPs in the mapping of quantitative trait loci (QTL) with extremal and random samples; BMC Genetics;  citation_year=2005;  citation_volume=6;  citation_pages=51" /><meta name="citation_reference" content="citation_title=Experimental designs for reliable detection of linkage disequilibrium in unstructured random population association studies; Genetics;  citation_year=2005;  citation_volume=170;  citation_pages=859-873; " /><meta name="citation_reference" content="citation_title=Power and precision of alternate methods for linkage disequilibrium mapping of quantitative trait loci; Genetics;  citation_year=2007;  citation_volume=175;  citation_pages=1975-1986; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/5/217/7525556/dsm020.pdf" /><meta name="description" content="Abstract.   In order to reveal quantitative trait loci (QTL) interactions and the relationship between various interactions in complex traits, we have developed" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/5/217/351025" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/5/217/351025" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/5/217/351025");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm020';
    </script>
<title>
	Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./351025" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfFIPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc29iZSwgU2FjaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIgZm9yIEhva2thaWRvIFJlZ2lvbiAsIEhpc3R1amlnYW9rYSAxLCBUb3lvaGlyYSwgU2FwcG9ybyAwNjItODU1NSAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUgLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYXlhLCBBa2loaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBjb21wdXRhdGlvbmFsIEJpb2xvZ3kgLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5byAsIDUtMS01LCBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEgLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhYmF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlICwgS2F6dXNhLUthbWF0YXJpIDItNi03LCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTggLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub3R5cGUgTWF0cml4IE1hcHBpbmc6IFNlYXJjaGluZyBmb3IgUXVhbnRpdGF0aXZlIFRyYWl0IExvY2kgSW50ZXJhY3Rpb25zIGluIEdlbmV0aWMgVmFyaWF0aW9uIGluIENvbXBsZXggVHJhaXRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc20wMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNy8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYXNzb2NpYXRpb24gbWFwcGluZyBhbmQgZ2Vub21lIG9yZ2FuaXphdGlvbiBvZiBtYWl6ZTsgQ3Vyci4gT3Bpbi4gQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xNTUtMTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgZ2VuZXRpYyB2YXJpYXRpb246IGEgcG9zdC1tb2Rlcm4gdmlldzsgSHVtLiBNb2wuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPVIxLVI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGluIGh1bWFuczogYWNoaWV2ZW1lbnRzIGFuZCBsaW1pdGF0aW9uczsgSi4gQ2xpbi4gSW52ZXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz0xNDE5LTE0MjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIHRoZSBmdXR1cmU6IGlkZW50aWZpY2F0aW9uIG9mIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGluIHRoZSBtb3VzZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNzQxLTE3NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhkMSAsIGEgbWFqb3IgcGhvdG9wZXJpb2Qgc2Vuc2l0aXZpdHkgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIGluIHJpY2UsIGlzIGNsb3NlbHkgcmVsYXRlZCB0byB0aGUgQXJhYmlkb3BzaXMgZmxvd2VyaW5nIHRpbWUgZ2VuZSBDT05TVEFOUzsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI0NzMtMjQ4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2FnZSBkaXNlcXVpbGlicml1bSBtYXBzIGFuZCBhc3NvY2lhdGlvbiBtYXBwaW5nOyBKLiBDbGluLiBJbnZlc3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9MTQyNS0xNDMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGVvcmV0aWNhbCBiYXNpcyBvZiBzZXBhcmF0aW9uIG9mIG11bHRpcGxlIGxpbmtlZCBnZW5lIGVmZmVjdHMgb24gbWFwcGluZyBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0xMDk3Mi0xMDk3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCByZXNvbHV0aW9uIG9mIHF1YW50aXRhdGl2ZSB0cmFpdCBpbnRvIG11bHRpcGxlIGxvY2kgdmlhIGludGVydmFsIG1hcHBpbmc7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTEzNjsgIGNpdGF0aW9uX3BhZ2VzPTE0NDctMTQ0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9jYXRpbmcgbXVsdGlwbGUgaW50ZXJhY3RpbmcgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgdXNpbmcgcmFuay1iYXNlZCBtb2RlbCBzZWxlY3Rpb247IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTE4NDUtMTg1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENoYWxsZW5nZSBmb3IgZ2VuZXRpYyBlcGlkZW1pb2xvZ2lzdDogaG93IHRvIGFuYWx5emUgbGFyZ2UgbnVtYmVyIG9mIFNOUHMgaW4gcmVsYXRpb24gdG8gY29tcGxleCBkaXNlYXNlczsgQk1DIEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGVsaW5nIGVwaXN0YXNpcyBvZiBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSB1c2luZyBDY2tlcmhhbSdzIG1vZGVsOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjA7ICBjaXRhdGlvbl9wYWdlcz0xMjQzLTEyNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGlmeWluZyB0aGUgU2Nod2FyeiBCYXllc2lhbiBpbmZvcm1hdGlvbiBjcml0ZXJpb24gdG8gbG9jYXRlIG11bHRpcGxlIGludGVyYWN0aW5nIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjc7ICBjaXRhdGlvbl9wYWdlcz05ODktOTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgbXVsdGlwbGUgUVRMIHdpdGggZXBpc3RhdGljIGVmZmVjdHMgdW5kZXIgYSBtaXhlZCBpbmhlcml0YW5jZSBtb2RlbCBpbiBhbiBvdXRicmVkIHBvcHVsYXRpb247IEdlbmV0LiBTZWwuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NDE1LTQzMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlmYWN0b3ItZGltZW5zaW9uYWxpdHkgcmVkdWN0aW9uIHJldmVhbHMgaGlnaC1vcmRlciBpbnRlcmFjdGlvbnMgYW1vbmcgZXN0cm9nZW4tbWV0YWJvbGlzbSBnZW5lcyBpbiBzcG9yYWRpYyBicmVhc3QgY2FuY2VyOyBBbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Njk7ICBjaXRhdGlvbl9wYWdlcz0xMzgtMTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhcHBsaWNhdGlvbiBvZiBjb25kaXRpb25hbCBsb2dpc3RpYyByZWdyZXNzaW9uIGFuZCBtdWx0aWZhY3RvciBkaW1lbnNpb25hbGl0eSByZWR1Y3Rpb24gZm9yIGRldGVjdGluZyBnZW5l4oCTZ2VuZSBpbnRlcmFjdGlvbnMgb24gcmlzayBvZiBteW9jYXJkaWFsIGluZmFyY3Rpb246IHRoZSBpbXBvcnRhbmNlIG9mIG1vZGVsIHZhbGlkYXRpb247IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aWZhY3Rvci1kaW1lbnNpb25hbGl0eSByZWR1Y3Rpb24gc2hvd3MgYSB0d28gbG9jdXMgaW50ZXJhY3Rpb24gYXNzb2NpYXRlZCB3aXRoIFR5cGUgMiBkaWFiZXRlcyBtZWxsaXR1czsgRGlhYmV0b2xvZ2lhOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NTQ5LTU1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBzdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgb2YgcmVkIGNsb3ZlciAoIFRyaWZvbGl1bSBwcmF0ZW5zZSBMLik7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzAxLTM2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3dpc3MgTWF0dGVua2xlZSBsYW5kcmFjZXMsIGEgZGlzdGluY3QgYW5kIGRpdmVyc2UgZ2VuZXRpYyByZXNvdXJjZSBvZiByZWQgY2xvdmVyICggVHJpZm9saXVtIHByYXRlbnNlIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0zMDYtMzE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgUG9wdWxhdGlvbiBTdHJ1Y3R1cmUgdXNpbmcgbXVsdGlsb2N1cyBnZW5vdHlwZSBkYXRhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTU7ICBjaXRhdGlvbl9wYWdlcz05NDUtOTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFjaW5nIHN5bmVyZ2V0aWMgYmVoYXZpb3Igb2YgdGhlIFFUTHMgYWZmZWN0aW5nIG9yYWwgZ2x1Y29zZSB0b2xlcmFuY2UgaW4gdGhlIE9MRVRGIHJhdDsgR2Vub21lIEluZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTU1LTE2NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RyYWluZWQgY2x1c3RlcnMgb2YgZ2VuZSBleHByZXNzaW9uIHByb2ZpbGVzIHdpdGggcGF0aG9sb2dpY2FsIGZlYXR1cmVzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTMxMzctMzE0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVzdCBmb3IgaW50ZXJhY3Rpb24gYmV0d2VlbiB0d28gdW5saW5rZWQgbG9jaTsgQW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9ODMxLTg0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIGJ5IGdlbmV0aWMgYmFja2dyb3VuZCBpbnRlcmFjdGlvbnMgaW4gYXNzb2NpYXRpb24gc3R1ZGllczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9NTUzLTU2MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIGFuYWx5c2lzIG9mIHNlZWQgeWllbGQgY29tcG9uZW50cyBpbiByZWQgY2xvdmVyICggVHJpZm9saXVtIHByYXRlbnNlIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz01MzYtNTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBvZiBnZW5lIGV4cHJlc3Npb24gc3VydmV5ZWQgaW4gbWFpemUsIG1vdXNlIGFuZCBtYW47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjI7ICBjaXRhdGlvbl9wYWdlcz0yOTctMzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1lUVRMIFZpZXdlcjogdmlzdWFsaXppbmcgaG93IHNlcXVlbmNlIHZhcmlhdGlvbiBhZmZlY3RzIGdlbm9tZS13aWRlIHRyYW5zY3JpcHRpb247IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Ny0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXNzZWN0aW9uIG9mIGNvbXBsZXggdHJhaXRzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTI2NTsgIGNpdGF0aW9uX3BhZ2VzPTIwMzctMjA0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB1bmlmaWVkIG1peGVkLW1vZGVsIG1ldGhvZCBmb3IgYXNzb2NpYXRpb24gbWFwcGluZyB0aGF0IGFjY291bnRzIGZvciBtdWx0aXBsZSBsZXZlbHMgb2YgcmVsYXRlZG5lc3M7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MjAzLTIwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBiaWFzLWVkIGFzc2Vzc21lbnQgb2YgdGhlIHVzZSBvZiBTTlBzIGluIGh1bWFuIGNvbXBsZXggdHJhaXRzOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2ZWxvcC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz03MjYtNzM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgdGhlIHBvd2VyIG9mIHRhZyBTTlBzIGluIHRoZSBtYXBwaW5nIG9mIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIChRVEwpIHdpdGggZXh0cmVtYWwgYW5kIHJhbmRvbSBzYW1wbGVzOyBCTUMgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwZXJpbWVudGFsIGRlc2lnbnMgZm9yIHJlbGlhYmxlIGRldGVjdGlvbiBvZiBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGluIHVuc3RydWN0dXJlZCByYW5kb20gcG9wdWxhdGlvbiBhc3NvY2lhdGlvbiBzdHVkaWVzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzA7ICBjaXRhdGlvbl9wYWdlcz04NTktODczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3dlciBhbmQgcHJlY2lzaW9uIG9mIGFsdGVybmF0ZSBtZXRob2RzIGZvciBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIG1hcHBpbmcgb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2k7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NTsgIGNpdGF0aW9uX3BhZ2VzPTE5NzUtMTk4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE0LzUvMjE3Lzc1MjU1NTYvZHNtMDIwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICAgSW4gb3JkZXIgdG8gcmV2ZWFsIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIChRVEwpIGludGVyYWN0aW9ucyBhbmQgdGhlIHJlbGF0aW9uc2hpcCBiZXR3ZWVuIHZhcmlvdXMgaW50ZXJhY3Rpb25zIGluIGNvbXBsZXggdHJhaXRzLCB3ZSBoYXZlIGRldmVsb3BlZCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzUxMDI1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZL6OO4Yh0FtklmhIFdkWG4+9Xkm+" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f217%2f351025" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f217%2f351025" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13580" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&amp;Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/5">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="42021762">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021762">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42021764">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021764">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42021770">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021770">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42021779">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021779">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42021797">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021797">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42021805">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021805">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42021807">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021807">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42021809">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42021809">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/5/207/351235">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/5/227/351394">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Agricultural Research Center for Hokkaido Region</div> , <div class="addr-line">Histujigaoka 1, Toyohira, Sapporo 062-8555</div> , <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Kazusa DNA Research Institute</div> , <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div> , <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div> To whom correspondence should be addressed. <div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7, Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div> , <div class="country">Japan</div> . Tel. +81 438-52-3928 . Fax <div class="fax">+81 438-52-3934</div> . E-mail: <a href="mailto:sisobe@kazusa.or.jp" target="_blank">sisobe@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akihiro Nakaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of computational Biology</div> , <div class="insititution">The University of Tokyo</div> , <div class="addr-line">5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akihiro+Nakaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakaya A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakaya A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akihiro Nakaya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Kazusa DNA Research Institute</div> , <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 5, 1 January 2007, Pages 217â€“225, <a href='https://doi.org/10.1093/dnares/dsm020'>https://doi.org/10.1093/dnares/dsm020</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">13 November 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 July 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 October 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="351025" href="/dnaresearch/article-pdf/14/5/217/7525556/dsm020.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sachiko Isobe, Akihiro Nakaya, Satoshi Tabata;  Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits, <em>DNA Research</em>, Volume 14, Issue 5, 1 January 2007, Pages 217â€“225, <a href='https://doi.org/10.1093/dnares/dsm020'>https://doi.org/10.1093/dnares/dsm020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/351025?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/351025?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/351025?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/351025?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/351025?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/351025?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/351025?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genotype%20Matrix%20Mapping%3A%20Searching%20for%20Quantitative%20Trait%20Loci%20Interactions%20in%20Genetic%20Variation%20in%20Complex%20Traits&amp;publicationDate=2007-11-13&amp;volumeNum=14&amp;issueNum=5&amp;author=Isobe%2C%20Sachiko%3B%20Nakaya%2C%20Akihiro&amp;startPage=217&amp;endPage=225&amp;contentId=10.1093%2Fdnares%2Fdsm020&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f217%2f351025" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=42021762 id="42021762" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> In order to reveal quantitative trait loci (QTL) interactions and the relationship between various interactions in complex traits, we have developed a new QTL mapping approach, named genotype matrix mapping (GMM), which searches for QTL interactions in genetic variation. The central approach in GMM is the following. (1) Each tested marker is given a virtual matrix, named a genotype matrix (GM), containing intersecting lines and rows equal to the total allele number for that marker in the population analyzed. (2) QTL interactions are then estimated and compared through virtual networks among the GMs. To evaluate the contribution of marker combinations to a quantitative phenotype, the GMM method divides the samples into two non-overlapping subclasses, <em>S</em><sub>0</sub> and <em>S</em><sub>1</sub> ; the former contains the samples that have a specific genotype pattern to be evaluated, and the latter contains samples that do not. Based on this division, the F-measure is calculated as an index of significance. With the GMM method, we extracted significant marker combinations consisting of one to three interacting markers. The results indicated there were multiple QTL interactions affecting the phenotype (flowering date). GMM will be a valuable approach to identify QTL interactions in genetic variation of a complex trait within a variety of organisms. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genotype matrix mapping&quot;">genotype matrix mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;QTL interaction&quot;">QTL interaction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic variation&quot;">genetic variation</a></div>                            <h2 scrollto-destination=42021764 id="42021764" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p> Many complex traits of medical and agricultural importance, such as blood pressure, diabetes, crop yield, and plant stress resistance, are controlled by quantitative trait loci (QTLs). <sup><a reveal-id="DSM020C1" data-open="DSM020C1" class="link link-ref link-reveal xref-bibr">1</a> , <a reveal-id="DSM020C2" data-open="DSM020C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The phenotypic variation of a complex trait usually results from multiple QTLs, QTLâ€“QTL interactions, and QTLâ€“environmental interactions. <sup><a reveal-id="DSM020C3" data-open="DSM020C3" class="link link-ref link-reveal xref-bibr">3</a></sup> These complex interactions make identification of individual QTL and QTL interactions difficult. However, recent advances in molecular biology and genomics will now enable us to identify and dissect QTLs related to complex traits in connection with genomic information by employing a statistical method. Once candidate QTLs have been identified, such genetic regions can be broken down into responsible genes by a map-based cloning approach with the aid of DNA markers and genetic linkage maps. <sup><a reveal-id="DSM020C4" data-open="DSM020C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The first successful example of gene identification and cloning from a naturally occurring allelic variation was <em>Hd1</em> , a major QTL responsible for photoperiod sensitivity in rice. <sup><a reveal-id="DSM020C5" data-open="DSM020C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p> Two major approaches have been used to investigate QTLs contributing to complex traits: linkage analysis and association analysis. Linkage analysis exploits the shared inheritance of functional polymorphisms and adjacent markers within families or pedigrees of known ancestry, <sup><a reveal-id="DSM020C1" data-open="DSM020C1" class="link link-ref link-reveal xref-bibr">1</a></sup> whereas association analysis is an approach to detect QTL localization based on linkage disequilibrium in unrelated individuals or natural populations. <sup><a reveal-id="DSM020C6" data-open="DSM020C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Both these approaches have contributed to our understanding of single QTLs in a great variety of species; however, neither approach can be used to detect interactions among multiple QTLs. Although composite interval mapping (CIM) <sup><a reveal-id="DSM020C7" data-open="DSM020C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and multiple QTL mapping (MQM) <sup><a reveal-id="DSM020C8" data-open="DSM020C8" class="link link-ref link-reveal xref-bibr">8</a></sup> can identify multiple QTLs by including additional background markers in the model, these approaches can only precisely locate QTLs with significant main effects and do not detect epistatic QTLs. <sup><a reveal-id="DSM020C9" data-open="DSM020C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p> Recently, several approaches for the detection of epigenetic gene interactions have been developed, primarily in the field of human genetics, by application of regression analysis, neural networks, and non-parametric methods. <sup><a reveal-id="DSM020C10" data-open="DSM020C10" class="link link-ref link-reveal xref-bibr">10</a></sup> One of the typical approaches is fitting a multiple regression model and, thus, relating the trait values to marker genotypes. <sup><a reveal-id="DSM020C9" data-open="DSM020C9" class="link link-ref link-reveal xref-bibr">9</a> , <a reveal-id="DSM020C11  DSM020C13" data-open="DSM020C11  DSM020C13" class="link link-ref link-reveal xref-bibr">11â€“13</a></sup> In this approach, the number of QTLs in the population by first making estimates based on the Bayesian information criterion and then QTL location and interaction fitting in the model are assumed. The weak point of this approach is that the detection power decreases as the number of parameters increases. So far, identification of presumptive epistatic QTLs using the multiple regression model has been demonstrated only with inter-crossed populations. The other commonly used approach is the multifactor dimensionality reduction (MDR) method, which uses non-parametric methods and is most efficiently used with caseâ€“control data. <sup><a reveal-id="DSM020C14  DSM020C16" data-open="DSM020C14  DSM020C16" class="link link-ref link-reveal xref-bibr">14â€“16</a></sup> The MDR method pools multi-locus genotypes into a single dimension with two groups classified as either case or control data. Because this approach was developed for genetic epidemiologists, sequential phenotypic data cannot be adapted to this analysis. </p><p> Similar to the multiple regression model and MDR method, other current approaches available for detection of epistatic QTLs have their own advantages and disadvantages. For example, some methods are incapable of detecting epistatic QTLs in the absence of main effect QTLs or interactions among more than three loci, and others have not developed software programs for general users. <sup><a reveal-id="DSM020C10" data-open="DSM020C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In addition, we recognized a common disadvantage in these current approaches; that is, all of them were developed to generate a best solution (QTL interaction) among numerous candidate interactions, and do not consider relations among subsets of interacting QTLs. Because the final phenotypic value of a complex trait should be determined through multifactor interactions, there may not be just one best interaction in a complex trait, but multiple interactions related to a complex trait. Therefore, we considered that comparison of many QTL interactions at once is vital for a comprehensive understanding of QTL expression affecting a complex trait. </p><p>In order to reveal QTL interactions and interactionâ€“interaction relationships in complex traits, we have developed a new QTL mapping approach named genotype matrix mapping (GMM), which searches for QTL interactions not only in family data, but also in various genetic backgrounds. In GMM, each marker is given a matrix in which each of the total number of alleles for the marker in the tested population is represented by intersecting lines and rows. QTL interactions are estimated and compared through virtual networks generated among the locus matrixes. In this report, we described the concept of GMM and investigate the efficiency of the concept using a real genotypeâ€“phenotype data set.</p>                            <h2 scrollto-destination=42021770 id="42021770" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=42021771 id="42021771" class="section-title" data-legacy-id=s2a>2.1. Red clover data set</h3>
<p> For development and investigation of the GMM algorithm, a real genotypeâ€“phenotype data set derived from red clover ( <em>Trifolium pratense</em> L.) was used. Red clover is diploid and the genome size is less than 440 Mb. <sup><a reveal-id="DSM020C17" data-open="DSM020C17" class="link link-ref link-reveal xref-bibr">17</a></sup> It is an allogamous species and each variety contains a high level of heterozygosity. <sup><a reveal-id="DSM020C18" data-open="DSM020C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p> A total of 74 red clover individuals originating from ten varieties bred in different countries were used: â€˜Natsuyuâ€™ (Japan), â€˜Hokusekiâ€™ (Japan), â€˜Sapporoâ€™ (Japan), â€˜Rannij2â€™ (Russia), â€˜Startâ€™ (Czechoslovakia), â€˜Kuranoâ€™ (Denmark), â€˜Renovaâ€™ (Switzerland), â€˜Merviotâ€™ (Belgium), â€˜Kenlandâ€™ (USA) and â€˜Altaswedeâ€™ (Canada). Genomic DNA was extracted from five to eight individuals of each variety and subjected to PCR examination using 106 primer pairs for selected microsatellite markers distributed throughout the genome. <sup><a reveal-id="DSM020C17" data-open="DSM020C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The presence or absence of amplification and differences in fragment size were scored as different alleles. For phenotype data, the flowering date was estimated for each individual grown under conditions of 20Â°C and an 18 h light/6 h dark cycle in a greenhouse. The possibility of population structure in the tested 74 individuals was estimated by the â€˜Structureâ€™ (version 2.0) program with the following parameters: length of burning period = 10 000, Number of MCMC population in the burning period = 1000 ( <a class="link link-uri" href="http://pritch.bsd.uchicago.edu/software.html">http://pritch.bsd.uchicago.edu/software.html</a> ). <sup><a reveal-id="DSM020C19" data-open="DSM020C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p> Phenotype values for red clover flowering date were distributed over 26 days (Fig.Â  <a reveal-id="DSM020F1" data-open="DSM020F1" class="link link-reveal link-table xref-fig">1</a> A). The average number of alleles per marker was 6.5, and the range was between 2 and 10 (Fig.Â  <a reveal-id="DSM020F1" data-open="DSM020F1" class="link link-reveal link-table xref-fig">1</a> B). The number of genotypes (observed two allele combinations per marker) ranged from 2 to 31 (Fig.Â  <a reveal-id="DSM020F1" data-open="DSM020F1" class="link link-reveal link-table xref-fig">1</a> C). The absence of population structure was confirmed by comparison of Ln <em>P</em> ( <em>D</em> ) = âˆ’22476.9 ( <em>K</em> = 1) and Ln <em>P</em> ( <em>D</em> ) = âˆ’22507.9 ( <em>K</em> = 2). A schematic view of the dataset is presented in Fig.Â  <a reveal-id="DSM020F2" data-open="DSM020F2" class="link link-reveal link-table xref-fig">2</a> . </p>                        <a id="42021775" scrollto-destination="42021775"></a>
<div id="DSM020F1" data-id="DSM020F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02001.jpeg?Expires=2147483647&Signature=xx7BMk81i-cRMcLncUtCyT6sHcYg1fIOPofFudnPtF6ibbH23GndVG~mtfjDnK~pSpNBrzgF6bl2l1xRA6EyvZ8I3pxYaWiLPDiBYJE5crI3zoSLR9uzc~HAB8AFiHcKj29t9ALeSGIoTYacyO09oQUE3BSvDLE98s3QpmIJTErEO6~~o8fbijNUo4Ofb5VahvW5P1WBip8OTYzuGf6bDRiUAUWW9tqx26y6L6ekx8AdOUQP0Dd3wcizFbEMmp~dFD6gQ-bbxhwGCVzHVuak99vFIHZ5mADgUJQQVN44DwDnWgiqUHFwICjFE98dG3lZlFlXzPAKnqTdS2nFuqsvgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Phenotype and genotype data from the red clover germplasms. Distribution of flowering date ( A ), number of alleles per locus ( B ), and number of genotypes per marker ( C ) are summarized. " path-from-xml="dsm02001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021775/dsm02001.jpeg" path-from-xml="dsm02001.jpeg" target="_blank">View large</a><a section="42021775" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02001.jpeg?Expires=2147483647&Signature=IDIPixpNAkrMIs93wwrmoBwbyL5k4UsVTHOs6vIvS7KEcPCqg6~pwHR30scTtcKr5N7FzavgN23JTyYoqpiB5GgRuBdxJYX9NMNyHe8PYIazUF8R0FmaCF9pP5SVCqzdgJbyz4~NkQ5BLJKXRzX~56d6sGIGPsvcQ2WwwwVZQg4MwzaoVNvlLbbe~gvkh00P2TVoLX1HjgZYzCaQZnasi9ACq07kdj93IxLSp~T9DwrRZnIKBf2UR24rUikx7dLCyNkRK1sq29FQDqs4XOHt~A3tZDdNW9Ns0OoAAW7M7Ou4IO4uoSb-0om-JVQVabyEZb6JZb8fl3n88s8Qlg~M8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021775&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Phenotype and genotype data from the red clover germplasms. Distribution of flowering date ( <strong>A</strong> ), number of alleles per locus ( <strong>B</strong> ), and number of genotypes per marker ( <strong>C</strong> ) are summarized. </p></div></div><div content-id="DSM020F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02001.jpeg?Expires=2147483647&Signature=xx7BMk81i-cRMcLncUtCyT6sHcYg1fIOPofFudnPtF6ibbH23GndVG~mtfjDnK~pSpNBrzgF6bl2l1xRA6EyvZ8I3pxYaWiLPDiBYJE5crI3zoSLR9uzc~HAB8AFiHcKj29t9ALeSGIoTYacyO09oQUE3BSvDLE98s3QpmIJTErEO6~~o8fbijNUo4Ofb5VahvW5P1WBip8OTYzuGf6bDRiUAUWW9tqx26y6L6ekx8AdOUQP0Dd3wcizFbEMmp~dFD6gQ-bbxhwGCVzHVuak99vFIHZ5mADgUJQQVN44DwDnWgiqUHFwICjFE98dG3lZlFlXzPAKnqTdS2nFuqsvgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Phenotype and genotype data from the red clover germplasms. Distribution of flowering date ( A ), number of alleles per locus ( B ), and number of genotypes per marker ( C ) are summarized. " path-from-xml="dsm02001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021775/dsm02001.jpeg" path-from-xml="dsm02001.jpeg" target="_blank">View large</a><a section="42021775" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02001.jpeg?Expires=2147483647&Signature=IDIPixpNAkrMIs93wwrmoBwbyL5k4UsVTHOs6vIvS7KEcPCqg6~pwHR30scTtcKr5N7FzavgN23JTyYoqpiB5GgRuBdxJYX9NMNyHe8PYIazUF8R0FmaCF9pP5SVCqzdgJbyz4~NkQ5BLJKXRzX~56d6sGIGPsvcQ2WwwwVZQg4MwzaoVNvlLbbe~gvkh00P2TVoLX1HjgZYzCaQZnasi9ACq07kdj93IxLSp~T9DwrRZnIKBf2UR24rUikx7dLCyNkRK1sq29FQDqs4XOHt~A3tZDdNW9Ns0OoAAW7M7Ou4IO4uoSb-0om-JVQVabyEZb6JZb8fl3n88s8Qlg~M8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021775&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Phenotype and genotype data from the red clover germplasms. Distribution of flowering date ( <strong>A</strong> ), number of alleles per locus ( <strong>B</strong> ), and number of genotypes per marker ( <strong>C</strong> ) are summarized. </p></div></div>                        <a id="42021776" scrollto-destination="42021776"></a>
<div id="DSM020F2" data-id="DSM020F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02002.jpeg?Expires=2147483647&Signature=LRCS6kvwq0auXowlZhvwmciH5I31E-YL8Z37gFVnslZY3J-Zp0q~2uJSikHJjCOry3xa18XZEzqsnMuitvxT9dyD~g36OC3ZGFr~PmnMuTz77GV4TdKiEpIVrMyFHw43pEL1PiHadrVB7gxW~JzzjXvbOWBkDGqhGIjUpmhAxR5lhpsXbQLwY5ju9KkSwu8GNPcXq4RWj~UG9Wr9S1ndne5BPaUzsvtFsJUshusGTZbkgIqxj5GGLGeWvDtwzTVHCcgDOTENL7K74X3d7DRoNRq3MDwOMBkZjMpi1vohgTghfW100T1witIw-82NWJz~WVOXl3gl5VOJpc6evT4zZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic view of the genotype data set. Each line and row indicates red clover individuals and marker genotypes, respectively. Colors of cells indicate allele combination as listed at the bottom. Allele types are indicated by lower case, and â€˜-â€™ indicates the absence of amplifications or undetermined alleles of dominant markers. The matrix at the right indicates phenotypic values (flowering date) of each individual (red dots) from early to late, corresponding to 29 October 2004 to 24 November 2004, respectively." path-from-xml="dsm02002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021776/dsm02002.jpeg" path-from-xml="dsm02002.jpeg" target="_blank">View large</a><a section="42021776" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02002.jpeg?Expires=2147483647&Signature=eIF2pUCqZeh63qnrv7BuRyXHWIH~jbAzBhIs32b5yYGubnIdZRcb6Zd~yy9EudAHfcicJyueneWsz7x~Yu6OgEYNNY7GSAQet471gcqsrnK7qpmYfK-dEgbHcD8XcFgMqoHHzxeipWss4RRoic1kax7IjA~cwMZ-cH3A4LB~tOHmnYcpylFiXz-8VrdWEpDhwNl1vplMkABQ5Sd8th9R2YempImAwDofMrhZ49Eztv1ydIsvoupbpJNd9G2mr5UpTgolPodMQRagGepuGqxzKlNm~3Xo6c4RLCcjcDtBo4ojW4tYZDZ-iLaeZrQfB5NUo~y0IX5vXZ7nGnubZ4mwew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021776&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic view of the genotype data set. Each line and row indicates red clover individuals and marker genotypes, respectively. Colors of cells indicate allele combination as listed at the bottom. Allele types are indicated by lower case, and â€˜-â€™ indicates the absence of amplifications or undetermined alleles of dominant markers. The matrix at the right indicates phenotypic values (flowering date) of each individual (red dots) from early to late, corresponding to 29 October 2004 to 24 November 2004, respectively.</p></div></div><div content-id="DSM020F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02002.jpeg?Expires=2147483647&Signature=LRCS6kvwq0auXowlZhvwmciH5I31E-YL8Z37gFVnslZY3J-Zp0q~2uJSikHJjCOry3xa18XZEzqsnMuitvxT9dyD~g36OC3ZGFr~PmnMuTz77GV4TdKiEpIVrMyFHw43pEL1PiHadrVB7gxW~JzzjXvbOWBkDGqhGIjUpmhAxR5lhpsXbQLwY5ju9KkSwu8GNPcXq4RWj~UG9Wr9S1ndne5BPaUzsvtFsJUshusGTZbkgIqxj5GGLGeWvDtwzTVHCcgDOTENL7K74X3d7DRoNRq3MDwOMBkZjMpi1vohgTghfW100T1witIw-82NWJz~WVOXl3gl5VOJpc6evT4zZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic view of the genotype data set. Each line and row indicates red clover individuals and marker genotypes, respectively. Colors of cells indicate allele combination as listed at the bottom. Allele types are indicated by lower case, and â€˜-â€™ indicates the absence of amplifications or undetermined alleles of dominant markers. The matrix at the right indicates phenotypic values (flowering date) of each individual (red dots) from early to late, corresponding to 29 October 2004 to 24 November 2004, respectively." path-from-xml="dsm02002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021776/dsm02002.jpeg" path-from-xml="dsm02002.jpeg" target="_blank">View large</a><a section="42021776" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02002.jpeg?Expires=2147483647&Signature=eIF2pUCqZeh63qnrv7BuRyXHWIH~jbAzBhIs32b5yYGubnIdZRcb6Zd~yy9EudAHfcicJyueneWsz7x~Yu6OgEYNNY7GSAQet471gcqsrnK7qpmYfK-dEgbHcD8XcFgMqoHHzxeipWss4RRoic1kax7IjA~cwMZ-cH3A4LB~tOHmnYcpylFiXz-8VrdWEpDhwNl1vplMkABQ5Sd8th9R2YempImAwDofMrhZ49Eztv1ydIsvoupbpJNd9G2mr5UpTgolPodMQRagGepuGqxzKlNm~3Xo6c4RLCcjcDtBo4ojW4tYZDZ-iLaeZrQfB5NUo~y0IX5vXZ7nGnubZ4mwew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021776&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic view of the genotype data set. Each line and row indicates red clover individuals and marker genotypes, respectively. Colors of cells indicate allele combination as listed at the bottom. Allele types are indicated by lower case, and â€˜-â€™ indicates the absence of amplifications or undetermined alleles of dominant markers. The matrix at the right indicates phenotypic values (flowering date) of each individual (red dots) from early to late, corresponding to 29 October 2004 to 24 November 2004, respectively.</p></div></div>                            <h3 scrollto-destination=42021777 id="42021777" class="section-title" data-legacy-id=s2b>2.2. Genotype matrix mapping</h3>
<p> To carry out our analysis, we used a newly developed program written in C++ language. This program uses the obtained marker genotypes and the phenotypes of the individuals as its input data set (Fig.Â  <a reveal-id="DSM020F2" data-open="DSM020F2" class="link link-reveal link-table xref-fig">2</a> ), and extracts all significant QTLs and QTL interactions in an exhaustive manner without omission. The results are available in a computer-parsable format and can be converted into graphical presentations, as shown in this report. In contrast to interval mapping, the GMM algorithm does not necessarily require relative genetic distances between markers, since it does not estimate genotypes at putative loci between two flanking marker loci. It uses map information, if available, only for visualizing the results. </p>                            <h2 scrollto-destination=42021779 id="42021779" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=42021780 id="42021780" class="section-title" data-legacy-id=s3a>3.1. Central concept of genotype matrix mapping</h3>
<p> Any type of population, including unrelated individuals, family data, and mapping populations for linkage analysis, can be used for GMM, providing that there is no population structure within the tested data set. The number of alleles in the population should be determined for every marker before analysis. Next, each marker is given a virtual matrix, named a genotype matrix (GM), in which line and row numbers are allotted based on the total number of alleles for the marker in the population (Fig.Â  <a reveal-id="DSM020F3" data-open="DSM020F3" class="link link-reveal link-table xref-fig">3</a> A). For example, when the number of alleles of â€˜Marker Aâ€™ is five, â€˜Marker Aâ€™ is given half a 5 Ã— 5 matrix covering all possible allele combinations. Individual genotypes are plotting onto the matrix. For example, â€˜Individual Xâ€™ with genotype â€˜abâ€™ for â€˜Marker Aâ€™, plots to the cell where line â€˜aâ€™ and row â€˜bâ€™ intersect. The estimated existence of a significant relationship between genotype and phenotype values based on analysis of variance (ANOVA), as in association analysis, is determined when a number of individuals are plotted to the same cell. Allelic interaction between â€˜aâ€™ and â€˜bâ€™ on â€˜Marker Aâ€™ is estimated by comparison the phenotype distribution of â€˜aaâ€™, â€˜abâ€™, and â€˜bbâ€™ cells on the GM of â€˜Marker Aâ€™. Additionally, the general effect of allele â€˜aâ€™ can be estimated by comparison the phenotypes among all the cells in â€˜line aâ€™. </p>                        <a id="42021782" scrollto-destination="42021782"></a>
<div id="DSM020F3" data-id="DSM020F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02003.jpeg?Expires=2147483647&Signature=NaAfRGdPSGWgHNFozL9uVRJnWRrbSMShOXKUGhKzeSacF0XuG9H-4GcTvXkNPPy5WCDSOk5Ic~7Ja6BYCl1~vPrK7JouLvD67NpukRncCSjQMQjXlHO4jddrds7hSwVn-vRBDH~-yHdNG6z-FwIJqJrnlZBgZvNPk13pv6jtGSLiknWp0JICcq4xc7P985Fzcoo6FC1oQR0lVI58mG3fyDBiNWQD~WlTbqwt1mxER1r6mYkClRp1ZrFnQh7jXqqZZGGsr2t9RcyTY2915zXSIT47Dm04eJ3wpUzewx7B2TrYk3Vs-bBzcEdAkdp6CBjwvmm-dJnbAZ8D53h2mtVYHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Schematic representation of GM and GMN. ( A ) A virtual matrix given for â€˜Marker Aâ€™, which is composed of five alleles. The pink cell indicates the â€˜abâ€™ genotype. ( B ) Detection of marker locus interactions by virtual networks among multiple matrices. Red, orange, and green lines indicate interactions which influence the phenotype to various extent (see text). " path-from-xml="dsm02003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021782/dsm02003.jpeg" path-from-xml="dsm02003.jpeg" target="_blank">View large</a><a section="42021782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02003.jpeg?Expires=2147483647&Signature=zvaL6M5aKs5VYWkJrs-1U03BZmkkvStWNQHAfLMwFypo-zV7cFQPLwgTKPuOSXLzpJZJir6DFANL9Yu0Xuf-Gjtaxd~s6StMBBG9r91OsY5Asu5vfhMjU3yVwMZ7R7z9MjrCqqITSKbIin1BV9rZvKtvQioWezf9RvdkDTABjMFJ6I2LSCjhBAIyCQ36AoVgZI3WTYQh9mI54PZQt7yNppDOJR0Xi6Ku33vdMGJCKGQ809aipbRMFQvvi8hEZattNpb7f1qOsq3gQcTlFRTX1p2sngTgMsaYXid7lLDVzv7cjb9lIm3veKhpHIU6TXeLjplAKHhETsWeLx2ZBiYoIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021782&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Schematic representation of GM and GMN. ( <strong>A</strong> ) A virtual matrix given for â€˜Marker Aâ€™, which is composed of five alleles. The pink cell indicates the â€˜abâ€™ genotype. ( <strong>B</strong> ) Detection of marker locus interactions by virtual networks among multiple matrices. Red, orange, and green lines indicate interactions which influence the phenotype to various extent (see text). </p></div></div><div content-id="DSM020F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02003.jpeg?Expires=2147483647&Signature=NaAfRGdPSGWgHNFozL9uVRJnWRrbSMShOXKUGhKzeSacF0XuG9H-4GcTvXkNPPy5WCDSOk5Ic~7Ja6BYCl1~vPrK7JouLvD67NpukRncCSjQMQjXlHO4jddrds7hSwVn-vRBDH~-yHdNG6z-FwIJqJrnlZBgZvNPk13pv6jtGSLiknWp0JICcq4xc7P985Fzcoo6FC1oQR0lVI58mG3fyDBiNWQD~WlTbqwt1mxER1r6mYkClRp1ZrFnQh7jXqqZZGGsr2t9RcyTY2915zXSIT47Dm04eJ3wpUzewx7B2TrYk3Vs-bBzcEdAkdp6CBjwvmm-dJnbAZ8D53h2mtVYHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Schematic representation of GM and GMN. ( A ) A virtual matrix given for â€˜Marker Aâ€™, which is composed of five alleles. The pink cell indicates the â€˜abâ€™ genotype. ( B ) Detection of marker locus interactions by virtual networks among multiple matrices. Red, orange, and green lines indicate interactions which influence the phenotype to various extent (see text). " path-from-xml="dsm02003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021782/dsm02003.jpeg" path-from-xml="dsm02003.jpeg" target="_blank">View large</a><a section="42021782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02003.jpeg?Expires=2147483647&Signature=zvaL6M5aKs5VYWkJrs-1U03BZmkkvStWNQHAfLMwFypo-zV7cFQPLwgTKPuOSXLzpJZJir6DFANL9Yu0Xuf-Gjtaxd~s6StMBBG9r91OsY5Asu5vfhMjU3yVwMZ7R7z9MjrCqqITSKbIin1BV9rZvKtvQioWezf9RvdkDTABjMFJ6I2LSCjhBAIyCQ36AoVgZI3WTYQh9mI54PZQt7yNppDOJR0Xi6Ku33vdMGJCKGQ809aipbRMFQvvi8hEZattNpb7f1qOsq3gQcTlFRTX1p2sngTgMsaYXid7lLDVzv7cjb9lIm3veKhpHIU6TXeLjplAKHhETsWeLx2ZBiYoIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021782&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Schematic representation of GM and GMN. ( <strong>A</strong> ) A virtual matrix given for â€˜Marker Aâ€™, which is composed of five alleles. The pink cell indicates the â€˜abâ€™ genotype. ( <strong>B</strong> ) Detection of marker locus interactions by virtual networks among multiple matrices. Red, orange, and green lines indicate interactions which influence the phenotype to various extent (see text). </p></div></div><p> Finally, virtual networks created by through multiple GMs, named genotype matrix networks (GMNs), connected by each cell are generated. When a particular network indicates a significant relationship to the phenotype, the markerâ€“allele combinations assigned on the GMs are considered a QTL interaction combination. In the example in Fig.Â  <a reveal-id="DSM020F3" data-open="DSM020F3" class="link link-reveal link-table xref-fig">3</a> B, the combination of Marker A(ab)â€“Marker B(bc)â€“Marker C(bd) strongly influences the phenotype (red line), the combination of Marker A(ab)â€“Marker B(ac)â€“Marker C(bd) has an intermediate influence (orange line), and the combination of Marker A(ab)â€“Marker B(ac)â€“Marker C(ae) has a weak influence (green line). The letters in parentheses indicate allele types. In summary, QTL interactions are identified by finding significant interactions between multiple GMs using GMNs. In theory, the potential number of conjunct markers per combination ranges from one to the maximum number of tested markers. </p>                            <h3 scrollto-destination=42021784 id="42021784" class="section-title" data-legacy-id=s3b>3.2. Detection of QTL interactions</h3>
<div class="block-child-p"> The first step of the GMM method is to construct a list of locus combinations. Each locus combination in the list is then depicted on the GM of the marker, as explained above. To evaluate the significance of the locus combinations, we used the <em>F</em> -measure. <sup><a reveal-id="DSM020C20" data-open="DSM020C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The total set <em>S</em> that consists of <em>N</em> individuals is divided into two non-overlapping subclasses <em>S</em><sub>0</sub> and <em>S</em><sub>1</sub> according to their marker genotypes. Here, we have defined <em>S</em><sub>1</sub> as the samples that have the specific genotype pattern to be evaluated, and <em>S</em><sub>0</sub> as the samples that do not. When the aggregative effect of markers is evaluated, the samples are subdivided using the multiple markers instead of a single marker. Next, the mean square among classes is calculated as follows: <a id="" class="jumplink-placeholder">Â </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02007.gif?Expires=2147483647&Signature=tGZK7ZA58PZR-oKmum30TsPtKrsIZMDyRU1i55ybzbQEyVYdFUbxNP~QfnZ-WjsexVAozXi7nN2FJ50k11NiauqK8YmoO-dOBcMgbAA-cBPQmALNrdeAtENZmaw-k1jbMFh28xzviZ6kOcu17SSHeiRpPp5Kcub407SmLXOtBkWv-~klkq1xwUioUwltgWUyp~kwOgituCiDG5A47zIFB0vsT2GtGf1cQ02XJAOCD0Rr8bbcodFoyBOe52W4hc0tuH6xd7Obxae5retuF2y4HSC4WiJouXaEZirGd5JogWM~de5oSHVv1lJKjEIZqbPgwqyyYDQEeBdU1psYsv-jUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm02007.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02007.gif?Expires=2147483647&Signature=m1NmRE3TOU8tWzKxKiOovf-E-cwNT~5xnt7oNTxVSDISle~2JJbNk4dYLdqzEiem~wZqsLxPnHLMjazVi2QHcIDBDVO5QaA9~WEZUBxDCdmXU~77PQrZ5TnV2V-OcWve0CopUm5UaDw0AoS32zHWTFQDGFKTdpHhVyvOAGK94ssYfU5A~ZiZdvI-0XuLCJy1-JG6QFURqMOKMe-znYHdWVMoAycDSYqIawH81NIkExR9Cye8aNeV6XvnuBRxgBl0PYf0G1pLsBVhBVwO5~JPfr8Tp5UMdfefarHoRz~D9bXWjUW4NHr4X5SPCJPm06upvI6JjV3Rf3hYG6IUSOTheg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm02007.gif" /></a></div></div></div></div><p> Here, Âµ, Âµ <sub>0</sub> , and Âµ <sub>1</sub> are the means of the phenotype values in <em>S</em> , <em>S</em><sub>0</sub> , and <em>S</em><sub>1</sub> , respectively. |X| is the number of individuals in <em>X</em> . </p><div class="block-child-p"> The mean square within each class is defined as follows: <a id="" class="jumplink-placeholder">Â </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02008.gif?Expires=2147483647&Signature=aIlU0Yefwfs-gKZhyo8KlKQXZIHvTuzUchhxqVM5c4gfOSf7FuypvySCJE3UN9n1T35Za3imu-xigAszHA9DSnwXKctOglyN0f8sZFOj67MQRa0QgmR1NDEr~ZdAR7zRS0NbP4cnlbsHmUI-ONxLIR-i0AuSZtOVVX1SJPN4pa~cZYPu99ot4ZpLg6dnmZeYhXz5-kSjoNbLfrG1vrefpkw0yZ~7w~wEbpbMWFG-Zs47gN7Wl9sRQ0iWaoT-TT4QabEpUGDKxlxBOq3x0YNKpNpU-Oxmoqsailsnuy6sW6-KR3wNLRGmw3eKU9WIIWkw4kiurfFBT3yW5gdo3gLgQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm02008.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02008.gif?Expires=2147483647&Signature=QJqy~omFF~w1rNjh3u4a87QdLv61ML7H41dxE7Qso6owf6KoGUO8G~rfPHzGwcI0LVW6w~~sZF2RgJoZmXfWHvVL6CQaLPxA7iHxR9i0c-3G78trbtHymasM40GuS4pokEP27wEboxNrGwLwVOw6CimZOL8389lKeM7HiIWytsolLnL5FZ2vQC~HXOQFMld9StwOMuARcJxeP~3MfuYMTiBuDNM-hf0nlYF8G9--NMWBJz0--jeUhs9FF8iJ8~wTY~MrMGe5f8uhBaN2V1yQd315sjWKJDz4TieiDL0cAxFywIodysLibGeHYo163MqIW7VEqlZs7fNa-1Pwhqy6Fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm02008.gif" /></a></div></div></div></div><p> Here, <em>P</em><sub><em>i</em></sub> is the phenotype value of the <em>i</em> -th individual <em>s</em><sub><em>i</em></sub> . The <em>F</em> -measure is obtained by dividing the MSA by the MSW ( <em>F</em> = MSA/MSW) and indicates the bias of the distribution of phenotype values in the two subclasses. If the distribution of the phenotype in the two subclasses differs, one can conclude that the condition (i.e., the pattern of marker genotypes) used for sample division is associated with the phenotype of interest. In such cases, the <em>F</em> -measure value is large. </p><p> Significant locus combinations that have large <em>F</em> -measure values are searched in an incremental manner. During the searching procedure, the maximum <em>F</em> -measure obtained is <em>F</em><sub>opt</sub> , and the value of <em>F</em><sub>opt</sub> is updated when a better combination whose <em>F</em> -measure is higher than the current <em>F</em><sub>opt</sub> is identified. Additionally, <em>d</em> is a given margin value and the final result of this method includes all combinations whose <em>F</em> -measure is less than the maximum by this margin value <em>d</em> . To fulfill this condition, starting with a single marker, another marker is concatenated to the first marker with each round of the incremental search until the given maximum length <em>L</em> is reached. When each additional marker is concatenated to a combination, the <em>F</em> -measure is evaluated to judge whether the combination is significant or not (i.e., higher than <em>F</em><sub>opt</sub> - <em>d</em> or not). If the combination is significant, it is included in the result, and the upper boundary of the <em>F</em> -measure that the combination can reach by further concatenation is calculated. <sup><a reveal-id="DSM020C21" data-open="DSM020C21" class="link link-ref link-reveal xref-bibr">21</a></sup> If this upper boundary is less than <em>F</em><sub>opt</sub> - <em>d</em> , significant combinations cannot be obtained by further computation, and the current search is therefore terminated and evaluation of another set of combinations is started. When all searches are terminated, all the optimum and sub-optimum combinations are included in the result. </p>                            <h3 scrollto-destination=42021790 id="42021790" class="section-title" data-legacy-id=s3c>3.3. GMM search on a red clover data set</h3>
<p> We examined the feasibility of the GMM method using the datasets from red clover, which consist of 106 microsatellite markers distributed along the entire genome for the genotype dataset and the flowering date for the phenotype dataset. Using the GMM algorithm, we tentatively extracted the significant locus combinations consisting of at most <em>L</em> = 3 markers. TableÂ  <a reveal-id="DSM020TB1" data-open="DSM020TB1" class="link link-reveal link-table xref-fig">1</a> shows all the combinations of markers whose <em>F</em> -measure value was higher than 35.0 (searching was carried out setting the margin value <em>d</em> = 10.0 and only combinations higher than <em>F</em> = 35.0 are shown). All the combinations with high <em>F</em> -measure values listed in TableÂ  <a reveal-id="DSM020TB1" data-open="DSM020TB1" class="link link-reveal link-table xref-fig">1</a> are concatenations of three loci. This means that the <em>F</em> -measure values obtained using combinations of one or two loci were lower than those obtained using combinations of three loci. The combination of RCS2958(b-), RCS0914(b-), and RCS1300(ab) produced the highest <em>F</em> -measure score (43.9), and extracts samples with the three highest phenotype values (late-flowering samples). Fig.Â  <a reveal-id="DSM020F4" data-open="DSM020F4" class="link link-reveal link-table xref-fig">4</a> illustrates the distribution of phenotype values extracted using each combination consisting of one, two, or three of the three loci. The <em>F</em> -measure values obtained using the single loci RCS2958(b-), RCS0914(b-) and RCS1300(ab) were 7.9 Ã— 10 <sup>âˆ’1</sup> , 3.9 Ã— 10 <sup>0</sup> , and 3.7 Ã— 10 <sup>âˆ’4</sup> , respectively. These low values indicate that there is almost no, or only a weak, effect of each single locus alone. Similar to the single locus results, combinations of two loci did not strongly affect the phenotype. However, the triple loci combinations are highly associated with the phenotype. </p>                        <a id="42021792" scrollto-destination="42021792"></a>
<div id="DSM020F4" data-id="DSM020F4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02004.jpeg?Expires=2147483647&Signature=VbGK1vzvGK7BYehGhzBEeGstUKFJaT9R-ly3dI80mIT~dbJt3Vj6FcvsuuR8m0wIJp7wrNPjbl30NdK-TMiLGKEHQp4AGMT75OurMiM1x34vSf0nuHFKQR8exNlvEDfeNCzweL0H1IvWAF0~ts89nLQI4LG0SuW33G3RQe1EtA2VLPaqP8QQg2kFGZtnRbcH-twmAm0u1QrYbJ9ZOjfJ0a395jGCRxTBikSlMQgwSREjGSubNm4yo4U57xzBSOyC3PrXZ126LT~w2Yufxky1fPyyfQ4ejEuO6YioATYNfJ89b2ctwCaCtYPrTfjPWHe~qwbOtRZYV1e5Z~X4w3yoog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Distribution of phenotype values (flowering date) resulting from multiple locus/allele combinations. Combinations of two out of three locus/allele types are shown by green, blue, and yellow lines, and that of three types by red lines. Orange and gray bars indicate samples possessing the relevant locus/allele combinations and others, respectively (see TableÂ  1 ). " path-from-xml="dsm02004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021792/dsm02004.jpeg" path-from-xml="dsm02004.jpeg" target="_blank">View large</a><a section="42021792" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02004.jpeg?Expires=2147483647&Signature=CSIM3y41E3EEd~OA9nRg9rkWtVTn7h2YtR1AVxXNfjm7oLvhDiI0PkSriFSNi9K1fKMHrx7YX-XDYXTxn681RPyjVedTZ7c1T1F55gxb5WWRiuI3VDTxso~8EkdN~3S4nHVo1BsORZpAS-ttb4Sga2tmjttCGiKl0-nCeSBJpy2HU0QOqsEkkAx0Ngq8dz5WVbmB6ZuZXkltDpmCrVFvYb33vBWCvJDbx~Er4qMIZJPWN9Fl6cSCeFlSS6uspqkPXq-qjDJ~IY5aDKOiGJScili2Ts~UmxGjQPvADowiQEMM0hpfpIhYD0BNtWiApmFSFLcujl-7pvMmroTSy6e1Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021792&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Distribution of phenotype values (flowering date) resulting from multiple locus/allele combinations. Combinations of two out of three locus/allele types are shown by green, blue, and yellow lines, and that of three types by red lines. Orange and gray bars indicate samples possessing the relevant locus/allele combinations and others, respectively (see TableÂ  <a reveal-id="DSM020TB1" data-open="DSM020TB1" class="link link-reveal link-table xref-fig">1</a> ). </p></div></div><div content-id="DSM020F4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02004.jpeg?Expires=2147483647&Signature=VbGK1vzvGK7BYehGhzBEeGstUKFJaT9R-ly3dI80mIT~dbJt3Vj6FcvsuuR8m0wIJp7wrNPjbl30NdK-TMiLGKEHQp4AGMT75OurMiM1x34vSf0nuHFKQR8exNlvEDfeNCzweL0H1IvWAF0~ts89nLQI4LG0SuW33G3RQe1EtA2VLPaqP8QQg2kFGZtnRbcH-twmAm0u1QrYbJ9ZOjfJ0a395jGCRxTBikSlMQgwSREjGSubNm4yo4U57xzBSOyC3PrXZ126LT~w2Yufxky1fPyyfQ4ejEuO6YioATYNfJ89b2ctwCaCtYPrTfjPWHe~qwbOtRZYV1e5Z~X4w3yoog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Distribution of phenotype values (flowering date) resulting from multiple locus/allele combinations. Combinations of two out of three locus/allele types are shown by green, blue, and yellow lines, and that of three types by red lines. Orange and gray bars indicate samples possessing the relevant locus/allele combinations and others, respectively (see TableÂ  1 ). " path-from-xml="dsm02004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021792/dsm02004.jpeg" path-from-xml="dsm02004.jpeg" target="_blank">View large</a><a section="42021792" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02004.jpeg?Expires=2147483647&Signature=CSIM3y41E3EEd~OA9nRg9rkWtVTn7h2YtR1AVxXNfjm7oLvhDiI0PkSriFSNi9K1fKMHrx7YX-XDYXTxn681RPyjVedTZ7c1T1F55gxb5WWRiuI3VDTxso~8EkdN~3S4nHVo1BsORZpAS-ttb4Sga2tmjttCGiKl0-nCeSBJpy2HU0QOqsEkkAx0Ngq8dz5WVbmB6ZuZXkltDpmCrVFvYb33vBWCvJDbx~Er4qMIZJPWN9Fl6cSCeFlSS6uspqkPXq-qjDJ~IY5aDKOiGJScili2Ts~UmxGjQPvADowiQEMM0hpfpIhYD0BNtWiApmFSFLcujl-7pvMmroTSy6e1Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021792&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Distribution of phenotype values (flowering date) resulting from multiple locus/allele combinations. Combinations of two out of three locus/allele types are shown by green, blue, and yellow lines, and that of three types by red lines. Orange and gray bars indicate samples possessing the relevant locus/allele combinations and others, respectively (see TableÂ  <a reveal-id="DSM020TB1" data-open="DSM020TB1" class="link link-reveal link-table xref-fig">1</a> ). </p></div></div>                        <a id="42021793" scrollto-destination="42021793"></a>
<div content-id="DSM020TB1" class="table-modal"><div class="table-wrap"><div id="DSM020TB1" class="table-wrap-title" data-id="DSM020TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p> Significant locus combinations consisting of one to three interacting markers whose <em>F</em> -measures were &gt;35.0 </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>F</em> -measure Â </th><th colspan="2"> Relevant samples <sup>a</sup>Â </th><th colspan="2">OthersÂ </th><th>Number of lociÂ </th><th colspan="3">Interacted locus (allele)Â </th></tr><tr><th>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS3665(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS2741(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td><td>RCS0937(-)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS0914(b-)Â </td><td>RCS3719(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2958(b-)Â </td><td>RCS1113(bc)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1541(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS2689(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2645(ab)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr></tbody></table></div><div content-id="DSM020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th><em>F</em> -measure Â </th><th colspan="2"> Relevant samples <sup>a</sup>Â </th><th colspan="2">OthersÂ </th><th>Number of lociÂ </th><th colspan="3">Interacted locus (allele)Â </th></tr><tr><th>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS3665(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS2741(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td><td>RCS0937(-)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS0914(b-)Â </td><td>RCS3719(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2958(b-)Â </td><td>RCS1113(bc)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1541(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS2689(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2645(ab)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup> Samples harbouring the locus (allele) combination in the right-most column. </p></div></div><div content-id=""><div class="fn"><p><sup>b</sup> Mean value of phenotypes (flowering date). </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/42021793" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM020TB1" class="table-wrap-title" data-id="DSM020TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p> Significant locus combinations consisting of one to three interacting markers whose <em>F</em> -measures were &gt;35.0 </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>F</em> -measure Â </th><th colspan="2"> Relevant samples <sup>a</sup>Â </th><th colspan="2">OthersÂ </th><th>Number of lociÂ </th><th colspan="3">Interacted locus (allele)Â </th></tr><tr><th>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS3665(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS2741(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td><td>RCS0937(-)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS0914(b-)Â </td><td>RCS3719(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2958(b-)Â </td><td>RCS1113(bc)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1541(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS2689(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2645(ab)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr></tbody></table></div><div content-id="DSM020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th><em>F</em> -measure Â </th><th colspan="2"> Relevant samples <sup>a</sup>Â </th><th colspan="2">OthersÂ </th><th>Number of lociÂ </th><th colspan="3">Interacted locus (allele)Â </th></tr><tr><th>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Number of samplesÂ </th><th> Mean <sup>b</sup>Â </th><th>Â </th><th>Â </th><th>Â </th><th>Â </th></tr></thead><tbody><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>43.9Â </td><td>3Â </td><td>20.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS0914(b-)Â </td><td>RCS1300(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS3665(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS2741(a-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>41.0Â </td><td>3Â </td><td>19.7Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td><td>RCS0937(-)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0914(b-)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>38.2Â </td><td>5Â </td><td>15.8Â </td><td>69Â </td><td>3.6Â </td><td>3Â </td><td>RCS1366(a-)Â </td><td>RCS0914(b-)Â </td><td>RCS3719(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2958(b-)Â </td><td>RCS1113(bc)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1541(a-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS3315(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1113(bc)Â </td><td>RCS2689(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS2645(ab)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1325(cd)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2958(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1113(bc)Â </td><td>RCS1167(b-)Â </td><td>RCS1022(ab)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS2645(ab)Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td></tr><tr><td>35.8Â </td><td>3Â </td><td>19.0Â </td><td>71Â </td><td>3.8Â </td><td>3Â </td><td>RCS1167(b-)Â </td><td>RCS0235(-)Â </td><td>RCS1022(ab)Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup> Samples harbouring the locus (allele) combination in the right-most column. </p></div></div><div content-id=""><div class="fn"><p><sup>b</sup> Mean value of phenotypes (flowering date). </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/42021793" target="_blank">View Large</a></div></div><p> Several loci are repeatedly listed in TableÂ  <a reveal-id="DSM020TB1" data-open="DSM020TB1" class="link link-reveal link-table xref-fig">1</a> , which indicates that they interact with multiple combinations of loci. For example, RCS1022(ab) and RCS1167(b-) were each found in five combinations in TableÂ  <a reveal-id="DSM020TB1" data-open="DSM020TB1" class="link link-reveal link-table xref-fig">1</a> . The redundancy of the list presentation does not facilitate an intuitive understanding of the results, and the obtained information was therefore also visualized in two styles of graphical presentation, as shown in FigsÂ  <a reveal-id="DSM020F5" data-open="DSM020F5" class="link link-reveal link-table xref-fig">5</a> and <a reveal-id="DSM020F6" data-open="DSM020F6" class="link link-reveal link-table xref-fig">6</a> . Fig.Â  <a reveal-id="DSM020F5" data-open="DSM020F5" class="link link-reveal link-table xref-fig">5</a> illustrates the combinations of interacting triple loci and their positional information on the red clover linkage map. <sup><a reveal-id="DSM020C17" data-open="DSM020C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Locations of interacting loci are interlinked by lines on the linkage group maps. By illustrating the positions of interacting loci, the loci which affect the phenotype through interactions with multiple combinations of genotypes are readily identified; in this case, 50â€“52 cM and 84â€“89 cM on LG2, 76 cM on LG5, and 27 cM on LG6. In a different type of graphic, Fig.Â  <a reveal-id="DSM020F6" data-open="DSM020F6" class="link link-reveal link-table xref-fig">6</a> illustrates the relationships between interacting loci and allele type using GMs and a GMN. Line colors (GMNs) and cell colors (GMs) represent the magnitude of <em>F</em> -measure values for interacting and single locus/allele effects, respectively. By comparing several GMNs, the existence of â€˜hub loci/allelesâ€™ in these interactions emerges, such as RCS1022(ab) and RCS1167(b-), as well as multiple GMNs. The hub loci/alleles, however, did not show the highest effect on phenotypes when evaluated as a single locus/allele. </p>                        <a id="42021795" scrollto-destination="42021795"></a>
<div id="DSM020F5" data-id="DSM020F5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02005.jpeg?Expires=2147483647&Signature=bXY-i86RJlp0vXa8CsBRz1niFV-B1AF46YhyhrRaa4hZVql5s-OTHYKoP8tamQQQ2eH1lhhLC6thWYZd~SIX2DUyS0eQk82tCqlS4nV5kSHsiBoPENo-gHMyilqjJtFz83ksKvE-xNOTFEebhlJXEclGbW974TbidTz4MMd5uoCrRbTi7eCaH-1IhRXCOA5H5kzmYsDngYqUinX0upIM62cGp8MHf5tKH6dzBBFOvqV45fhYqigimY6buDq5uR3K71yV-vKlQ6NtT8hvkmX3HkfZE4z-fHkgxCt2GV38RkDWC7SHfyCLSpuUOjX9qC6daFgUfDb36AJ7MuaN0JCEmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Graphical presentation of interacting triple loci and their positions on the genetic linkage map. Seven linkage groups and unmapped markers are arranged tandemly as a circle. Triangles in the circle indicate GMNs, i.e. interacted triple loci combination. Magnitude of the F -measure is show by a color-code. " path-from-xml="dsm02005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021795/dsm02005.jpeg" path-from-xml="dsm02005.jpeg" target="_blank">View large</a><a section="42021795" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02005.jpeg?Expires=2147483647&Signature=JESKzVrX6g5KX6ToNKsqquJtZridM6VUwPMYnySBgijgMBjLDFETsqIAeBnqu5BNJxpDETDWjsYT7DuWwhQyWbwJTgrgrFwAN7n5syz0XNrUmV-oCtHK3pwqpB~9DZW3lXZfKbyDnyP51APWdz9eB0HNfQcs9RJ6gL-ZbMv3mIfphxZwz0PRlt00aVrWvjxmoHTWTOSkEpIaph4o06Lx9L1wdzhvjcli5ZGlc-HFejWX8MVfp5DaZSy-TPippzhSby1r01Riw~UxegXFfyQs2BhZ3GDqnlXP96eseO71-zGbo6XZJiM~Ei8UI3HQjW0oGIHx7heeQtzvNuNrmTJGNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021795&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Graphical presentation of interacting triple loci and their positions on the genetic linkage map. Seven linkage groups and unmapped markers are arranged tandemly as a circle. Triangles in the circle indicate GMNs, i.e. interacted triple loci combination. Magnitude of the <em>F</em> -measure is show by a color-code. </p></div></div><div content-id="DSM020F5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02005.jpeg?Expires=2147483647&Signature=bXY-i86RJlp0vXa8CsBRz1niFV-B1AF46YhyhrRaa4hZVql5s-OTHYKoP8tamQQQ2eH1lhhLC6thWYZd~SIX2DUyS0eQk82tCqlS4nV5kSHsiBoPENo-gHMyilqjJtFz83ksKvE-xNOTFEebhlJXEclGbW974TbidTz4MMd5uoCrRbTi7eCaH-1IhRXCOA5H5kzmYsDngYqUinX0upIM62cGp8MHf5tKH6dzBBFOvqV45fhYqigimY6buDq5uR3K71yV-vKlQ6NtT8hvkmX3HkfZE4z-fHkgxCt2GV38RkDWC7SHfyCLSpuUOjX9qC6daFgUfDb36AJ7MuaN0JCEmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Graphical presentation of interacting triple loci and their positions on the genetic linkage map. Seven linkage groups and unmapped markers are arranged tandemly as a circle. Triangles in the circle indicate GMNs, i.e. interacted triple loci combination. Magnitude of the F -measure is show by a color-code. " path-from-xml="dsm02005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021795/dsm02005.jpeg" path-from-xml="dsm02005.jpeg" target="_blank">View large</a><a section="42021795" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02005.jpeg?Expires=2147483647&Signature=JESKzVrX6g5KX6ToNKsqquJtZridM6VUwPMYnySBgijgMBjLDFETsqIAeBnqu5BNJxpDETDWjsYT7DuWwhQyWbwJTgrgrFwAN7n5syz0XNrUmV-oCtHK3pwqpB~9DZW3lXZfKbyDnyP51APWdz9eB0HNfQcs9RJ6gL-ZbMv3mIfphxZwz0PRlt00aVrWvjxmoHTWTOSkEpIaph4o06Lx9L1wdzhvjcli5ZGlc-HFejWX8MVfp5DaZSy-TPippzhSby1r01Riw~UxegXFfyQs2BhZ3GDqnlXP96eseO71-zGbo6XZJiM~Ei8UI3HQjW0oGIHx7heeQtzvNuNrmTJGNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021795&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Graphical presentation of interacting triple loci and their positions on the genetic linkage map. Seven linkage groups and unmapped markers are arranged tandemly as a circle. Triangles in the circle indicate GMNs, i.e. interacted triple loci combination. Magnitude of the <em>F</em> -measure is show by a color-code. </p></div></div>                        <a id="42021796" scrollto-destination="42021796"></a>
<div id="DSM020F6" data-id="DSM020F6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02006.jpeg?Expires=2147483647&Signature=l-29681g7guIaAxF4VT6kqSIAnLVKeQ-mPdICnN4kHzG57dlZbTCmJ40K0Egblkyuzeu7sNfgUbaJ-OJPMw69wky5cwlZz86MbOA~4uLK~nlfyNYzDB9JN4YyQ4iNvy4sA0VmlHzrRy5Wa4uGrxc9c60x52NaFoRKYQoQAAeAM2U2jHmj1JI6rYbU9Cn6AeRWeHuaTmqA2PagD2Eki0SQhCL1ovgvi1C8~54OdHXLc1nVUuK7iM6RhfSY8uY1eFLHVw0u7nv5EOQ2i3fEm29I9yecp3fTSlMLSatNKtEd6SZD9wYJ3kaGEg021EGDan9K67RiXNpORhL2e3mnz2pww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Graphical presentation of interacting loci and allele type by GMs and a GMN. Significant locus/allele combinations of one to three interacting loci whose F -measures were &gt;35.0 are shown by GMs and GMN. Matrices and connecting lines indicate GMs and GMNs, respectively. Magnitude of the F -measure of combination and single locus/alleles effects, respectively, is shown by a color-code. " path-from-xml="dsm02006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021796/dsm02006.jpeg" path-from-xml="dsm02006.jpeg" target="_blank">View large</a><a section="42021796" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02006.jpeg?Expires=2147483647&Signature=hF2BErvoC3~hxV1BMqp2-zacCUFsdMGRgKYlojMRQW~3c9F8JpPDHVzOvkm1K~gwuEuG6tiOEowkoiIDAy31S6NYBl~BU1c7yTdCrWybs9DoLIatjgcpTlu71yaIqNGTV1jYEjj6Ed6h1jaK9EVxXmsP3vrNwwcILZD0Hc3oJJ-R5mfMzO821K3zVfmxt6krlk9zZCMy~bunnGkN~jese5CVaAsVaE3YmbKJdcZqd3GY6wqhXMDEt4ZTUHTaYWPOdiTb4n3PGO4dN2lFE-fNpMMPG989Qz59v1Un8E9A8agc44a5CyaiLXR76HsHBrOChaycJ8vrR229mZY5Hhoo-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021796&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Graphical presentation of interacting loci and allele type by GMs and a GMN. Significant locus/allele combinations of one to three interacting loci whose <em>F</em> -measures were &gt;35.0 are shown by GMs and GMN. Matrices and connecting lines indicate GMs and GMNs, respectively. Magnitude of the <em>F</em> -measure of combination and single locus/alleles effects, respectively, is shown by a color-code. </p></div></div><div content-id="DSM020F6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/m_dsm02006.jpeg?Expires=2147483647&Signature=l-29681g7guIaAxF4VT6kqSIAnLVKeQ-mPdICnN4kHzG57dlZbTCmJ40K0Egblkyuzeu7sNfgUbaJ-OJPMw69wky5cwlZz86MbOA~4uLK~nlfyNYzDB9JN4YyQ4iNvy4sA0VmlHzrRy5Wa4uGrxc9c60x52NaFoRKYQoQAAeAM2U2jHmj1JI6rYbU9Cn6AeRWeHuaTmqA2PagD2Eki0SQhCL1ovgvi1C8~54OdHXLc1nVUuK7iM6RhfSY8uY1eFLHVw0u7nv5EOQ2i3fEm29I9yecp3fTSlMLSatNKtEd6SZD9wYJ3kaGEg021EGDan9K67RiXNpORhL2e3mnz2pww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Graphical presentation of interacting loci and allele type by GMs and a GMN. Significant locus/allele combinations of one to three interacting loci whose F -measures were &gt;35.0 are shown by GMs and GMN. Matrices and connecting lines indicate GMs and GMNs, respectively. Magnitude of the F -measure of combination and single locus/alleles effects, respectively, is shown by a color-code. " path-from-xml="dsm02006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42021796/dsm02006.jpeg" path-from-xml="dsm02006.jpeg" target="_blank">View large</a><a section="42021796" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm020/3/dsm02006.jpeg?Expires=2147483647&Signature=hF2BErvoC3~hxV1BMqp2-zacCUFsdMGRgKYlojMRQW~3c9F8JpPDHVzOvkm1K~gwuEuG6tiOEowkoiIDAy31S6NYBl~BU1c7yTdCrWybs9DoLIatjgcpTlu71yaIqNGTV1jYEjj6Ed6h1jaK9EVxXmsP3vrNwwcILZD0Hc3oJJ-R5mfMzO821K3zVfmxt6krlk9zZCMy~bunnGkN~jese5CVaAsVaE3YmbKJdcZqd3GY6wqhXMDEt4ZTUHTaYWPOdiTb4n3PGO4dN2lFE-fNpMMPG989Qz59v1Un8E9A8agc44a5CyaiLXR76HsHBrOChaycJ8vrR229mZY5Hhoo-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42021796&ar=351025&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Graphical presentation of interacting loci and allele type by GMs and a GMN. Significant locus/allele combinations of one to three interacting loci whose <em>F</em> -measures were &gt;35.0 are shown by GMs and GMN. Matrices and connecting lines indicate GMs and GMNs, respectively. Magnitude of the <em>F</em> -measure of combination and single locus/alleles effects, respectively, is shown by a color-code. </p></div></div>                            <h2 scrollto-destination=42021797 id="42021797" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p> Although there is a growing awareness of the importance of gene interactions in genetic studies of complex traits, classical genetic analysis either ignores gene interactions or defines the effect of gene interactions as a deviation from genetic additive effects. <sup><a reveal-id="DSM020C22" data-open="DSM020C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The demonstration in this study, using a real data set from red clover, indicates that the GMM algorithm efficiently detects multi-QTL interactions in genetic variation in a complex phenotype. The novel aspect of GMM is that the algorithm is capable of comparing multiple QTL interactions at once, which has not previously been possible in QTL detection approaches. </p><p> One of the advantages of comparison of multiple QTL interactions is the detection of hub locus/allele in the interactions. In this study, GMM demonstrated the existence of loci which affected the phenotype through multiple combinations of interacting loci. Interestingly, these hub loci/alleles did not necessarily display the highest effect on the phenotype as a single locus. To date, most findings have suggested that quantitative variation is determined by a few QTLs with a relatively large effect and a large number of genes having progressively smaller effects. <sup><a reveal-id="DSM020C2" data-open="DSM020C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, Jannink <sup><a reveal-id="DSM020C23" data-open="DSM020C23" class="link link-ref link-reveal xref-bibr">23</a></sup> recently identified QTLs by analyzing genetic background interactions in association studies, and was able to detect loci that have no main effect but which influence a trait only through their interactions with other loci. Our results, together with those of Jannink, suggest that multiple QTL interactions might be buried under the smaller effect of single QTLs. The hub locus/allele may be a key locus for identifying QTLs networks in which each component contributes directly to the final phenotype. </p><p> The hub loci/alleles that appeared in the multiple interactions that affected the flowering date of red clover were identified on 50â€“52 cM and 84â€“89 cM on LG2, 76 cM on LG5 and 27 cM on LG6. Herrmann et al. <sup><a reveal-id="DSM020C24" data-open="DSM020C24" class="link link-ref link-reveal xref-bibr">24</a></sup> have detected seven QTLs for the flowering date of red clover on all of the linkage groups except for on LG1. Although the tested DNA markers and the total lengths of the reference maps were different between Herrmann et al.'s (444 cM) and our study (869 cM), it could be presumed that the hub loci/alleles found on LG2 and LG5 will correspond to the QTLs identified by Herrmann et al., once a comparison of the two maps used in these studies has been performed. In this study, we used a set of unrelated individuals to demonstrate the GMM algorithm; however, any type of population, including family data, which is used for interval mapping analysis, can be analyzed by the GMM algorithm. Therefore, using GMM for reassessment of QTLs that have been identified by other methods (e.g., interval mapping) might provide us with insights into QTL interactions in the analyzed population. It is expected that the comparison of multiple combinations of interacting QTLs will lead to the identification of genes related to complex traits. Recently, expression quantitative trait locus (eQTL) mapping, which is a combination of gene expression profiling and classical genetic mapping, has been adopted to reveal quantitative heritable variation in the transcriptome. <sup><a reveal-id="DSM020C25" data-open="DSM020C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In this method, however, thousands of expression profiles are related with sequence polymorphisms across the genome through their correlated variations, which results in a large number of mappings that make it difficult to consider simultaneously the relationships between multiple genomic regions and multiple expression profiles. <sup><a reveal-id="DSM020C26" data-open="DSM020C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Prioritization and re-organization of a large number of original results by the GMM method may facilitate efficient dissection of expression QTLs and subsequent identification of genes. </p><p> Though the concept of GMM and its potential feasibility were demonstrated with a real genotypeâ€“phenotype data set in this article, there are several issues yet to be evaluated, examined, and solved before GMM can be applied more precisely for the detection of QTL interactions. The most prioritized issue to be discussed is the population structure. We used a data set of unrelated individuals without population structure in the present demonstration. However, QTL detection by GMM is performed on the basis of analysis of variance; thus, it is possible that hidden population structure leads to false positive associations between genotypes and phenotypes, as was observed in LD mapping. <sup><a reveal-id="DSM020C27" data-open="DSM020C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Therefore, estimation of the degree of population structure of a data set prior to GMM calculation should be an essential requirement. In order to address the problem of population structure in population-based samples, Yu et al. <sup><a reveal-id="DSM020C28" data-open="DSM020C28" class="link link-ref link-reveal xref-bibr">28</a></sup> have developed a new statistical method for LD mapping that simultaneously accounts for both population structure and familial relatedness. A combination of such method with GMM might provide a solution to the problem of the current version of GMM. </p><p> Effectual data size and degree of significance are also indispensable issues to be evaluated. One of the most probable reasons for insufficient output from the GMM analysis is the limited size of the data set. Increasing the number of individuals for data acquisition would improve the accuracy of the whole analysis, but this is not always feasible, especially in the early stages of data preparation. On the other hand, there is a strong demand to sort locus combinations according to their significance so that the loci that are expected to interact together can be prioritized. Therefore, searching for probable locus combinations will be useful from a practical point of view. In LD mapping, for example, many simulation studies have been carried out to obtain the optimal experiment design to detect QTLs, by considering a wide range of parameters, such as population size, number of markers, heritability and effects of QTLs, length of LDs, and the number of generations. <sup><a reveal-id="DSM020C29  DSM020C32" data-open="DSM020C29  DSM020C32" class="link link-ref link-reveal xref-bibr">29â€“32</a></sup> As with LD mapping, it is probable that the appropriate size of a data set for GMM will vary depending on the experimental design. Moreover, if the number of alleles of each locus increases, the risk rate of detecting false-positive interactions could also increase. To solve such problems, we need to evaluate the detection power of GMM in relation to the number of possible combinations of individuals and their genotypes. Both simulation studies and the application of the GMM method to real data sets will be useful to estimate the effectual data size. Additionally, the robustness of the significance test should also be investigated in the near future. </p><p> For easy handling and presentation of the results for users, it will be fundamental to design an appropriate interface for the GMM software program. Because of the large number of, and sometimes redundant, candidate QTL combinations detected by the GMM analysis, extraction of meaningful information from a tabulated set of data will often be difficult. In this study, we presented two types of graphic presentations for the results from the GMM analysis: a circular linkage map (Fig.Â  <a reveal-id="DSM020F5" data-open="DSM020F5" class="link link-reveal link-table xref-fig">5</a> ) and a combination GM and GMN chart (Fig.Â  <a reveal-id="DSM020F6" data-open="DSM020F6" class="link link-reveal link-table xref-fig">6</a> ). Development of a user interface that allows intuitive understanding of the relationships among multiple QTL interactions is under way. </p><p>Though GMM currently has several unclear issues which have to be resolved, it should give us an additional dimension of QTL impacts by indication of multiple QTL interactions. We hope the new ideas for dissection of QTL expression will be sparked on genetics and genomics by handling GMM. The GMM service and software package will be provided at http://www.kazusa.or.jp/GMM in December, 2007.</p>                            <h2 scrollto-destination=42021805 id="42021805" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the â€œDevelopment of DNA Marker-aided Selection Technology for Plants and Animals (DM-1604)â€ Project, Ministry of Agriculture, Forestry and Fisheries, Japan, and the Kazusa DNA Research Foundation.</p>                            <h2 scrollto-destination=42021807 id="42021807" class="backacknowledgements-title" >Acknowledgement</h2>
<p>We thank S. Sasamoto for excellent technical assistance.</p>                            <h2 scrollto-destination=42021809 id="42021809" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM020C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E. S.</div></div>.Â </div><div class="article-title">Genetic association mapping and genome organization of maize</div>,Â <div class="source">Curr. Opin. Biotech.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">155</div>-<div class="lpage">160</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20association%20mapping%20and%20genome%20organization%20of%20maize&amp;author=J.%20Yu&amp;author=E.%20S.%20Buckler&amp;publication_year=2006&amp;journal=Curr.%20Opin.%20Biotech.&amp;volume=17&amp;pages=155-160" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.copbio.2006.02.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.copbio.2006.02.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.copbio.2006.02.003"> </span></div>Â </div></div></div></div></div><div content-id="DSM020C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Farrall</div> <div class="given-names">M</div></div>.Â </div><div class="article-title">Quantitative genetic variation: a post-modern view</div>,Â <div class="source">Hum. Mol. Genet.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">13</div>Â (pg.Â <div class="fpage">R1</div>-<div class="lpage">R7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20genetic%20variation%3A%20a%20post-modern%20view&amp;author=M%20Farrall&amp;publication_year=2004&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=13&amp;pages=R1-R7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddh084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddh084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddh084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14962979" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Majumder</div> <div class="given-names">P. P.</div></div>,Â  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">S</div></div>.Â </div><div class="article-title">Mapping quantitative trait loci in humans: achievements and limitations</div>,Â <div class="source">J. Clin. Invest.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">115</div>Â (pg.Â <div class="fpage">1419</div>-<div class="lpage">1424</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20quantitative%20trait%20loci%20in%20humans%3A%20achievements%20and%20limitations&amp;author=P.%20P.%20Majumder&amp;author=S%20Ghosh&amp;publication_year=2005&amp;journal=J.%20Clin.%20Invest.&amp;volume=115&amp;pages=1419-1424" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1172/JCI24757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1172%2FJCI24757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1172%2FJCI24757"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15931376" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flaherty</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Herron</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Symula</div> <div class="given-names">D</div></div>.Â </div><div class="article-title">Genomics of the future: identification of quantitative trait loci in the mouse</div>,Â <div class="source">Genome Res.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">15</div>Â (pg.Â <div class="fpage">1741</div>-<div class="lpage">1745</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20the%20future%3A%20identification%20of%20quantitative%20trait%20loci%20in%20the%20mouse&amp;author=L.%20Flaherty&amp;author=B.%20Herron&amp;author=D%20Symula&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1741-1745" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3841405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3841405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3841405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339372" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yano</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Katayose</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Ashikari</div> <div class="given-names">M.</div></div>,Â et al.Â </div><div class="article-title"><em>Hd1</em> , a major photoperiod sensitivity quantitative trait locus in rice, is closely related to the Arabidopsis flowering time gene <em>CONSTANS</em></div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">12</div>Â (pg.Â <div class="fpage">2473</div>-<div class="lpage">2484</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hd1%20%2C%20a%20major%20photoperiod%20sensitivity%20quantitative%20trait%20locus%20in%20rice%2C%20is%20closely%20related%20to%20the%20Arabidopsis%20flowering%20time%20gene%20CONSTANS&amp;author=M.%20Yano&amp;author=Y.%20Katayose&amp;author=M.%20Ashikari&amp;publication_year=2000&amp;journal=Plant%20Cell&amp;volume=12&amp;pages=2473-2484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.12.12.2473" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.12.12.2473" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.12.12.2473"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11148291" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morton</div> <div class="given-names">N. E</div></div>.Â </div><div class="article-title">Linkage disequilibrium maps and association mapping</div>,Â <div class="source">J. Clin. Invest</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">115</div>Â (pg.Â <div class="fpage">1425</div>-<div class="lpage">1430</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20disequilibrium%20maps%20and%20association%20mapping&amp;author=N.%20E%20Morton&amp;publication_year=2005&amp;journal=J.%20Clin.%20Invest&amp;volume=115&amp;pages=1425-1430" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1172/JCI25032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1172%2FJCI25032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1172%2FJCI25032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15931377" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">Z. B</div></div>.Â </div><div class="article-title">Theoretical basis of separation of multiple linked gene effects on mapping quantitative trait loci</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">1993</div>,Â vol.Â <div class="volume">90</div>Â (pg.Â <div class="fpage">10972</div>-<div class="lpage">10976</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Theoretical%20basis%20of%20separation%20of%20multiple%20linked%20gene%20effects%20on%20mapping%20quantitative%20trait%20loci&amp;author=Z.%20B%20Zeng&amp;publication_year=1993&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=90&amp;pages=10972-10976" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.90.23.10972" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.23.10972" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.90.23.10972"> </span></div>Â </div></div></div></div></div><div content-id="DSM020C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jansen</div> <div class="given-names">R. C.</div></div>,Â  <div class="name"><div class="surname">Stam</div> <div class="given-names">P</div></div>.Â </div><div class="article-title">High resolution of quantitative trait into multiple loci via interval mapping</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">1994</div>,Â vol.Â <div class="volume">136</div>Â (pg.Â <div class="fpage">1447</div>-<div class="lpage">1445</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20resolution%20of%20quantitative%20trait%20into%20multiple%20loci%20via%20interval%20mapping&amp;author=R.%20C.%20Jansen&amp;author=P%20Stam&amp;publication_year=1994&amp;journal=Genetics&amp;volume=136&amp;pages=1447-1445" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8013917" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zak</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Baierl</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Bogdan</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Futschik</div> <div class="given-names">A</div></div>.Â </div><div class="article-title">Locating multiple interacting quantitative trait loci using rank-based model selection</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">176</div>Â (pg.Â <div class="fpage">1845</div>-<div class="lpage">1854</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20multiple%20interacting%20quantitative%20trait%20loci%20using%20rank-based%20model%20selection&amp;author=M.%20Zak&amp;author=A.%20Baierl&amp;author=M.%20Bogdan&amp;author=A%20Futschik&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=1845-1854" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.068031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.068031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.068031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17507685" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heidema</div> <div class="given-names">A. G.</div></div>,Â  <div class="name"><div class="surname">Boer</div> <div class="given-names">J. M. A.</div></div>,Â  <div class="name"><div class="surname">Nagelkerke</div> <div class="given-names">N.</div></div>,Â et al.Â </div><div class="article-title">The Challenge for genetic epidemiologist: how to analyze large number of SNPs in relation to complex diseases</div>,Â <div class="source">BMC Genetics</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">7</div>Â pg.Â <div class="fpage">23</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Challenge%20for%20genetic%20epidemiologist%3A%20how%20to%20analyze%20large%20number%20of%20SNPs%20in%20relation%20to%20complex%20diseases&amp;author=A.%20G.%20Heidema&amp;author=J.%20M.%20A.%20Boer&amp;author=N.%20Nagelkerke&amp;publication_year=2006&amp;journal=BMC%20Genetics&amp;volume=7&amp;pages=23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-7-23" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-7-23" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-7-23"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16630340" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kao</div> <div class="given-names">C. H.</div></div>,Â  <div class="name"><div class="surname">Zeng</div> <div class="given-names">Z. B</div></div>.Â </div><div class="article-title">Modeling epistasis of quantitative trait loci using Cckerham's model</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">160</div>Â (pg.Â <div class="fpage">1243</div>-<div class="lpage">1261</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20epistasis%20of%20quantitative%20trait%20loci%20using%20Cckerham's%20model&amp;author=C.%20H.%20Kao&amp;author=Z.%20B%20Zeng&amp;publication_year=2002&amp;journal=Genetics&amp;volume=160&amp;pages=1243-1261" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11901137" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bogdan</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">J. K.</div></div>,Â  <div class="name"><div class="surname">Doerge</div> <div class="given-names">R. W</div></div>.Â </div><div class="article-title">Modifying the Schwarz Bayesian information criterion to locate multiple interacting quantitative trait loci</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">167</div>Â (pg.Â <div class="fpage">989</div>-<div class="lpage">999</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modifying%20the%20Schwarz%20Bayesian%20information%20criterion%20to%20locate%20multiple%20interacting%20quantitative%20trait%20loci&amp;author=M.%20Bogdan&amp;author=J.%20K.%20Ghosh&amp;author=R.%20W%20Doerge&amp;publication_year=2004&amp;journal=Genetics&amp;volume=167&amp;pages=989-999" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.103.021683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.103.021683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.103.021683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15238547" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Narita</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">Y</div></div>.Â </div><div class="article-title">Detection of multiple QTL with epistatic effects under a mixed inheritance model in an outbred population</div>,Â <div class="source">Genet. Sel. Evol.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">36</div>Â (pg.Â <div class="fpage">415</div>-<div class="lpage">433</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20multiple%20QTL%20with%20epistatic%20effects%20under%20a%20mixed%20inheritance%20model%20in%20an%20outbred%20population&amp;author=A.%20Narita&amp;author=Y%20Sasaki&amp;publication_year=2004&amp;journal=Genet.%20Sel.%20Evol.&amp;volume=36&amp;pages=415-433" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1297-9686-36-4-415" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1297-9686-36-4-415" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1297-9686-36-4-415"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15231232" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ritchie</div> <div class="given-names">M. D.</div></div>,Â  <div class="name"><div class="surname">Hahn</div> <div class="given-names">L. W.</div></div>,Â  <div class="name"><div class="surname">Roodi</div> <div class="given-names">N.</div></div>,Â et al.Â </div><div class="article-title">Multifactor-dimensionality reduction reveals high-order interactions among estrogen-metabolism genes in sporadic breast cancer</div>,Â <div class="source">Am. J. Hum. Genet.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">69</div>Â (pg.Â <div class="fpage">138</div>-<div class="lpage">147</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multifactor-dimensionality%20reduction%20reveals%20high-order%20interactions%20among%20estrogen-metabolism%20genes%20in%20sporadic%20breast%20cancer&amp;author=M.%20D.%20Ritchie&amp;author=L.%20W.%20Hahn&amp;author=N.%20Roodi&amp;publication_year=2001&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=69&amp;pages=138-147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/321276" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F321276" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F321276"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11404819" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coffey</div> <div class="given-names">C. S.</div></div>,Â  <div class="name"><div class="surname">Hebert</div> <div class="given-names">P. R.</div></div>,Â  <div class="name"><div class="surname">Ritchie</div> <div class="given-names">M. D.</div></div>,Â et al.Â </div><div class="article-title">An application of conditional logistic regression and multifactor dimensionality reduction for detecting geneâ€“gene interactions on risk of myocardial infarction: the importance of model validation</div>,Â <div class="source">BMC Bioinformatics</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">5</div>Â pg.Â <div class="fpage">49</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20application%20of%20conditional%20logistic%20regression%20and%20multifactor%20dimensionality%20reduction%20for%20detecting%20gene%E2%80%93gene%20interactions%20on%20risk%20of%20myocardial%20infarction%3A%20the%20importance%20of%20model%20validation&amp;author=C.%20S.%20Coffey&amp;author=P.%20R.%20Hebert&amp;author=M.%20D.%20Ritchie&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-49" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-49" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-49"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15119966" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">Y. M.</div></div>,Â  <div class="name"><div class="surname">Ritchie</div> <div class="given-names">M. D.</div></div>,Â  <div class="name"><div class="surname">Moore</div> <div class="given-names">J. H.</div></div>,Â et al.Â </div><div class="article-title">Multifactor-dimensionality reduction shows a two locus interaction associated with Type 2 diabetes mellitus</div>,Â <div class="source">Diabetologia</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">47</div>Â (pg.Â <div class="fpage">549</div>-<div class="lpage">554</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multifactor-dimensionality%20reduction%20shows%20a%20two%20locus%20interaction%20associated%20with%20Type%202%20diabetes%20mellitus&amp;author=Y.%20M.%20Cho&amp;author=M.%20D.%20Ritchie&amp;author=J.%20H.%20Moore&amp;publication_year=2004&amp;journal=Diabetologia&amp;volume=47&amp;pages=549-554" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00125-004-1419-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00125-004-1419-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00125-004-1419-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14730379" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,Â et al.Â </div><div class="article-title"> Comprehensive structural analysis of the genome of red clover ( <em>Trifolium pratense</em> L.) </div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">12</div>Â (pg.Â <div class="fpage">301</div>-<div class="lpage">364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20structural%20analysis%20of%20the%20genome%20of%20red%20clover%20(%20Trifolium%20pratense%20L.)&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;author=E.%20Asamizu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=301-364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769692" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">KÃ¶lliker</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Herrmann</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Boller</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Widmer</div> <div class="given-names">F</div></div>.Â </div><div class="article-title"> Swiss Mattenklee landraces, a distinct and diverse genetic resource of red clover ( <em>Trifolium pratense</em> L.) </div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">107</div>Â (pg.Â <div class="fpage">306</div>-<div class="lpage">315</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Swiss%20Mattenklee%20landraces%2C%20a%20distinct%20and%20diverse%20genetic%20resource%20of%20red%20clover%20(%20Trifolium%20pratense%20L.)&amp;author=R.%20K%C3%B6lliker&amp;author=D.%20Herrmann&amp;author=B.%20Boller&amp;author=F%20Widmer&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=306-315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1248-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1248-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1248-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12845445" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pritchard</div> <div class="given-names">J. K.</div></div>,Â  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">P</div></div>.Â </div><div class="article-title">Inference of Population Structure using multilocus genotype data</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">155</div>Â (pg.Â <div class="fpage">945</div>-<div class="lpage">959</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20Population%20Structure%20using%20multilocus%20genotype%20data&amp;author=J.%20K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-959" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakaya</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Hishigaki</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Morishita</div> <div class="given-names">S</div></div>.Â </div><div class="article-title">Tracing synergetic behavior of the QTLs affecting oral glucose tolerance in the OLETF rat</div>,Â <div class="source">Genome Informatics</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">10</div>Â (pg.Â <div class="fpage">155</div>-<div class="lpage">165</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracing%20synergetic%20behavior%20of%20the%20QTLs%20affecting%20oral%20glucose%20tolerance%20in%20the%20OLETF%20rat&amp;author=A.%20Nakaya&amp;author=H.%20Hishigaki&amp;author=S%20Morishita&amp;publication_year=1999&amp;journal=Genome%20Informatics&amp;volume=10&amp;pages=155-165" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11072352" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sese</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Monden</div> <div class="given-names">M.</div></div>,Â et al.Â </div><div class="article-title">Constrained clusters of gene expression profiles with pathological features</div>,Â <div class="source">Bioinformatics</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">20</div>Â (pg.Â <div class="fpage">3137</div>-<div class="lpage">3145</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constrained%20clusters%20of%20gene%20expression%20profiles%20with%20pathological%20features&amp;author=J.%20Sese&amp;author=Y.%20Kurokawa&amp;author=M.%20Monden&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=3137-3145" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth373" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth373" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth373"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15217814" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Jin</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Xiong</div> <div class="given-names">M</div></div>.Â </div><div class="article-title">Test for interaction between two unlinked loci</div>,Â <div class="source">Am. J. Hum. Genet.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">79</div>Â (pg.Â <div class="fpage">831</div>-<div class="lpage">845</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Test%20for%20interaction%20between%20two%20unlinked%20loci&amp;author=J.%20Zhao&amp;author=L.%20Jin&amp;author=M%20Xiong&amp;publication_year=2006&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=79&amp;pages=831-845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/508571" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F508571" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F508571"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17033960" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jannink</div> <div class="given-names">J</div></div>.Â </div><div class="article-title">Identifying quantitative trait locus by genetic background interactions in association studies</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">176</div>Â (pg.Â <div class="fpage">553</div>-<div class="lpage">561</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20quantitative%20trait%20locus%20by%20genetic%20background%20interactions%20in%20association%20studies&amp;author=J%20Jannink&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=553-561" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.062992" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.062992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.062992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17179077" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Herrmann</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Boller</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Studer</div> <div class="given-names">B.</div></div>,Â et al.Â </div><div class="article-title"> QTL analysis of seed yield components in red clover ( <em>Trifolium pratense</em> L.) </div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">112</div>Â (pg.Â <div class="fpage">536</div>-<div class="lpage">545</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20seed%20yield%20components%20in%20red%20clover%20(%20Trifolium%20pratense%20L.)&amp;author=D.%20Herrmann&amp;author=B.%20Boller&amp;author=B.%20Studer&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=536-545" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0158-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0158-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0158-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16331477" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schadt</div> <div class="given-names">E. E.</div></div>,Â  <div class="name"><div class="surname">Monks</div> <div class="given-names">S. A.</div></div>,Â  <div class="name"><div class="surname">Drake</div> <div class="given-names">T. A.</div></div>,Â et al.Â </div><div class="article-title">Genetics of gene expression surveyed in maize, mouse and man</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">422</div>Â (pg.Â <div class="fpage">297</div>-<div class="lpage">302</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20gene%20expression%20surveyed%20in%20maize%2C%20mouse%20and%20man&amp;author=E.%20E.%20Schadt&amp;author=S.%20A.%20Monks&amp;author=T.%20A.%20Drake&amp;publication_year=2003&amp;journal=Nature&amp;volume=422&amp;pages=297-302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01434" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01434" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01434"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12646919" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zou</div> <div class="given-names">W.</div></div>,Â  <div class="name"><div class="surname">Aylor</div> <div class="given-names">D. L.</div></div>,Â  <div class="name"><div class="surname">Zeng</div> <div class="given-names">Z. B</div></div>.Â </div><div class="article-title">eQTL Viewer: visualizing how sequence variation affects genome-wide transcription</div>,Â <div class="source">BMC Bioinformatics</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">8</div>Â (pg.Â <div class="fpage">7</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=eQTL%20Viewer%3A%20visualizing%20how%20sequence%20variation%20affects%20genome-wide%20transcription&amp;author=W.%20Zou&amp;author=D.%20L.%20Aylor&amp;author=Z.%20B%20Zeng&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=7-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17212828" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E. S.</div></div>,Â  <div class="name"><div class="surname">Schork</div> <div class="given-names">N. J</div></div>.Â </div><div class="article-title">Genetic dissection of complex traits</div>,Â <div class="source">Science</div>Â ,Â <div class="year">1994</div>,Â vol.Â <div class="volume">265</div>Â (pg.Â <div class="fpage">2037</div>-<div class="lpage">2048</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20complex%20traits&amp;author=E.%20S.%20Lander&amp;author=N.%20J%20Schork&amp;publication_year=1994&amp;journal=Science&amp;volume=265&amp;pages=2037-2048" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.8091226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.8091226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.8091226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8091226" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Pressoir</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Briggs</div> <div class="given-names">W. H.</div></div>,Â et al.Â </div><div class="article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">38</div>Â (pg.Â <div class="fpage">203</div>-<div class="lpage">208</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20mixed-model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.%20H.%20Briggs&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=203-208" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380716" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Terwilliger</div> <div class="given-names">J. D.</div></div>,Â  <div class="name"><div class="surname">Haghighi</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Hiekkalinna</div> <div class="given-names">T. S.</div></div>,Â et al.Â </div><div class="article-title">A bias-ed assessment of the use of SNPs in human complex traits</div>,Â <div class="source">Curr. Opin. Genet. Develop.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">12</div>Â (pg.Â <div class="fpage">726</div>-<div class="lpage">734</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20bias-ed%20assessment%20of%20the%20use%20of%20SNPs%20in%20human%20complex%20traits&amp;author=J.%20D.%20Terwilliger&amp;author=F.%20Haghighi&amp;author=T.%20S.%20Hiekkalinna&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Genet.%20Develop.&amp;volume=12&amp;pages=726-734" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(02)00357-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(02)00357-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(02)00357-X"> </span></div>Â </div></div></div></div></div><div content-id="DSM020C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Sun</div> <div class="given-names">F</div></div>.Â </div><div class="article-title">Assessing the power of tag SNPs in the mapping of quantitative trait loci (QTL) with extremal and random samples</div>,Â <div class="source">BMC Genetics</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">6</div>Â pg.Â <div class="fpage">51</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20power%20of%20tag%20SNPs%20in%20the%20mapping%20of%20quantitative%20trait%20loci%20(QTL)%20with%20extremal%20and%20random%20samples&amp;author=K.%20Zhang&amp;author=F%20Sun&amp;publication_year=2005&amp;journal=BMC%20Genetics&amp;volume=6&amp;pages=51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-6-51" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-6-51" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-6-51"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16236175" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ball</div> <div class="given-names">R. D</div></div>.Â </div><div class="article-title">Experimental designs for reliable detection of linkage disequilibrium in unstructured random population association studies</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">170</div>Â (pg.Â <div class="fpage">859</div>-<div class="lpage">873</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20designs%20for%20reliable%20detection%20of%20linkage%20disequilibrium%20in%20unstructured%20random%20population%20association%20studies&amp;author=R.%20D%20Ball&amp;publication_year=2005&amp;journal=Genetics&amp;volume=170&amp;pages=859-873" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.103.024752" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.103.024752" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.103.024752"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15781715" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM020C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM020C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">H. H.</div></div>,Â  <div class="name"><div class="surname">Fernando</div> <div class="given-names">R. L.</div></div>,Â  <div class="name"><div class="surname">Dekkers</div> <div class="given-names">J. C. M</div></div>.Â </div><div class="article-title">Power and precision of alternate methods for linkage disequilibrium mapping of quantitative trait loci</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">175</div>Â (pg.Â <div class="fpage">1975</div>-<div class="lpage">1986</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Power%20and%20precision%20of%20alternate%20methods%20for%20linkage%20disequilibrium%20mapping%20of%20quantitative%20trait%20loci&amp;author=H.%20H.%20Zhao&amp;author=R.%20L.%20Fernando&amp;author=J.%20C.%20M%20Dekkers&amp;publication_year=2007&amp;journal=Genetics&amp;volume=175&amp;pages=1975-1986" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.066480" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.066480" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.066480"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17277369" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2007. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=351025&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">179</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000251879300003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">18</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm020"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="351025"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000251879300003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f5%2f217%2f351025">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000251879300003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (18)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F5%2F217%2F351025" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/351025'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genotype Matrix Mapping: Searching for Quantitative Trait Loci Interactions in Genetic Variation in Complex Traits" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm020");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f217%2f351025">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

