

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/3/315/394421","@type":"ScholarlyArticle","name":" Prdm9  Polymorphism Unveils Mouse Evolutionary Tracks","datePublished":"2014-01-20","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2014-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst059","keywords":["Prdm9","mouse","polymorphism","evolution"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/3/315/394421","author":[{"name":"Kono, Hiromitsu","affiliation":"1  Mammalian Genetics Laboratory ,  National Institute of Genetics ,  1111 Yata, Mishima, Shizuoka 411-8540 ,  Japan    2  Department of Biophysics and Biochemistry ,  The University of Tokyo ,  Tokyo 153-8902 ,  Japan","@type":"Person"},{"name":"Tamura, Masaru","affiliation":"1  Mammalian Genetics Laboratory ,  National Institute of Genetics ,  1111 Yata, Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"},{"name":"Osada, Naoki","affiliation":"3  Division of Evolutionary Genetics ,  National Institute of Genetics ,  Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"},{"name":"Suzuki, Hitoshi","affiliation":"4  Laboratory of Ecology and Genetics, Graduate School of Environmental Earth Science ,  Hokkaido University ,  Sapporo, Hokkaido 060-0810 ,  Japan","@type":"Person"},{"name":"Abe, Kuniya","affiliation":"5  BioResource Center ,  RIKEN Tsukuba Institute ,  Tsukuba, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Moriwaki, Kazuo","affiliation":"5  BioResource Center ,  RIKEN Tsukuba Institute ,  Tsukuba, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Ohta, Kunihiro","affiliation":"2  Department of Biophysics and Biochemistry ,  The University of Tokyo ,  Tokyo 153-8902 ,  Japan    6  Department of Life Sciences ,  The University of Tokyo ,  Tokyo 153-8902 ,  Japan","@type":"Person"},{"name":"Shiroishi, Toshihiko","affiliation":"1  Mammalian Genetics Laboratory ,  National Institute of Genetics ,  1111 Yata, Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"}],"description":"Abstract.  PR/SET domain containing 9 (Prdm9) mediates histone modifications such as H3K4me3 and marks hotspots of meiotic recombination. In many mammalian spec","pageStart":"315","pageEnd":"326","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Prdm9  Polymorphism Unveils Mouse Evolutionary Tracks","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Prdm9  Polymorphism Unveils Mouse Evolutionary Tracks" />
    <meta property="og:description" content="Abstract.  PR/SET domain containing 9 (Prdm9) mediates histone modifications such as H3K4me3 and marks hotspots of meiotic recombination. In many mammalian spec" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/3/315/394421" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-01-20" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kono, Hiromitsu" /><meta name="citation_author_institution" content="Mammalian Genetics Laboratory, National Institute of Genetics, 1111 Yata, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="Department of Biophysics and Biochemistry, The University of Tokyo, Tokyo 153-8902, Japan" /><meta name="citation_author" content="Tamura, Masaru" /><meta name="citation_author_institution" content="Mammalian Genetics Laboratory, National Institute of Genetics, 1111 Yata, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Osada, Naoki" /><meta name="citation_author_institution" content="Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Suzuki, Hitoshi" /><meta name="citation_author_institution" content="Laboratory of Ecology and Genetics, Graduate School of Environmental Earth Science, Hokkaido University, Sapporo, Hokkaido 060-0810, Japan" /><meta name="citation_author" content="Abe, Kuniya" /><meta name="citation_author_institution" content="BioResource Center, RIKEN Tsukuba Institute, Tsukuba, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Moriwaki, Kazuo" /><meta name="citation_author_institution" content="BioResource Center, RIKEN Tsukuba Institute, Tsukuba, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Ohta, Kunihiro" /><meta name="citation_author_institution" content="Department of Biophysics and Biochemistry, The University of Tokyo, Tokyo 153-8902, Japan" /><meta name="citation_author_institution" content="Department of Life Sciences, The University of Tokyo, Tokyo 153-8902, Japan" /><meta name="citation_author" content="Shiroishi, Toshihiko" /><meta name="citation_author_institution" content="Mammalian Genetics Laboratory, National Institute of Genetics, 1111 Yata, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_title" content="Prdm9 Polymorphism Unveils Mouse Evolutionary Tracks" /><meta name="citation_firstpage" content="315" /><meta name="citation_lastpage" content="326" /><meta name="citation_doi" content="10.1093/dnares/dst059" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2014/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Meiotic recombination hot spots and cold spots; Nat. Rev. Genet.;  citation_year=2001;  citation_volume=2;  citation_pages=360-9; " /><meta name="citation_reference" content="citation_title=Where the crossovers are: recombination distributions in mammals; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=413-24; " /><meta name="citation_reference" content="citation_title=A new wild-derived H-2 haplotype enhancing K-IA recombination; Nature;  citation_year=1982;  citation_volume=300;  citation_pages=370-2; " /><meta name="citation_reference" content="citation_title=Genetic control of sex-dependent meiotic recombination in the major histocompatibility complex of the mouse; EMBO J.;  citation_year=1991;  citation_volume=10;  citation_pages=681-6; " /><meta name="citation_reference" content="citation_title=Prdm9 controls activation of mammalian recombination hotspots; Science;  citation_year=2010;  citation_volume=327;  citation_pages=835" /><meta name="citation_reference" content="citation_title=Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination; Science;  citation_year=2010;  citation_volume=327;  citation_pages=876-9; " /><meta name="citation_reference" content="citation_title=PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice; Science;  citation_year=2010;  citation_volume=327;  citation_pages=836-40; " /><meta name="citation_reference" content="citation_title=High diversity at PRDM9 in chimpanzees and bonobos; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e39064" /><meta name="citation_reference" content="citation_title=Variants of the protein PRDM9 differentially regulate a set of human meiotic recombination hotspots highly active in African populations; Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=12378-83; " /><meta name="citation_reference" content="citation_title=PRDM9 variation strongly influences recombination hot-spot activity and meiotic instability in humans; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=859-63; " /><meta name="citation_reference" content="citation_title=Genetic recombination is directed away from functional genomic elements in mice; Nature;  citation_year=2012;  citation_volume=485;  citation_pages=642-5; " /><meta name="citation_reference" content="citation_title=Genetic variants in REC8, RNF212, and PRDM9 influence male recombination in cattle; PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002854" /><meta name="citation_reference" content="citation_title=A histone H3 methyltransferase controls epigenetic events required for meiotic prophase; Nature;  citation_year=2005;  citation_volume=438;  citation_pages=374-8; " /><meta name="citation_reference" content="citation_title=A mouse speciation gene encodes a meiotic histone H3 methyltransferase; Science;  citation_year=2009;  citation_volume=323;  citation_pages=373-5; " /><meta name="citation_reference" content="citation_title=Interallelic and intergenic incompatibilities of the Prdm9 (Hst1) gene in mouse hybrid sterility; PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1003044" /><meta name="citation_reference" content="citation_title=Extraordinary molecular evolution in the PRDM9 fertility gene; PloS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e8505" /><meta name="citation_reference" content="citation_title=Accelerated evolution of the Prdm9 speciation gene across diverse metazoan taxa; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000753" /><meta name="citation_reference" content="citation_title=Selection of DNA binding sites for zinc fingers using rationally randomized DNA reveals coded interactions; Proc. Natl. Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=11168-72; " /><meta name="citation_reference" content="citation_title=DNA binding specificities of the long zinc-finger recombination protein PRDM9; Genome Biol.;  citation_year=2013;  citation_volume=14;  citation_pages=R35" /><meta name="citation_reference" content="citation_title=Temporal, spatial, and ecological modes of evolution of Eurasian Mus based on mitochondrial and nuclear gene sequences; Mol. Phylogenet. Evol.;  citation_year=2004;  citation_volume=33;  citation_pages=626-46; " /><meta name="citation_reference" content="citation_title=The evolution of house mice; Annu. Rev. Ecol. Evol. Syst;  citation_year=1993;  citation_volume=24;  citation_pages=119-52; " /><meta name="citation_reference" content="citation_title=Evolutionary and dispersal history of Eurasian house mice Mus musculus clarified by more extensive geographic sampling of mitochondrial DNA; Heredity;  citation_year=2013;  citation_volume=111;  citation_pages=375-90; " /><meta name="citation_reference" content="citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation; Nature;  citation_year=2011;  citation_volume=477;  citation_pages=289-94; " /><meta name="citation_reference" content="citation_title=Origins of laboratory mice deduced from restriction patterns of mitochondrial DNA; Differentiation;  citation_year=1982;  citation_volume=22;  citation_pages=222-6; " /><meta name="citation_reference" content="citation_title=On the subspecific origin of the laboratory mouse; Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=1100-7; " /><meta name="citation_reference" content="citation_title=Evolution of the mouse t haplotype: recent and worldwide introgression to Mus musculus; Proc. Natl. Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=6851-5; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-80; " /><meta name="citation_reference" content="citation_title=A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences; J. Mol. Evol.;  citation_year=1980;  citation_volume=16;  citation_pages=111-20; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-25; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Subspecific origin and haplotype diversity in the laboratory mouse; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=648-55; " /><meta name="citation_reference" content="citation_title=Positive Darwinian selection promotes charge profile diversity in the antigen-binding cleft of class I major-histocompatibility-complex molecules; Mol. Biol. Evol.;  citation_year=1990;  citation_volume=7;  citation_pages=515-24; " /><meta name="citation_reference" content="citation_title=Rare allelic forms of PRDM9 associated with childhood leukemogenesis; Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=419-30; " /><meta name="citation_reference" content="citation_title=Hybrid origin of Japanese mice “Mus musculus molossinus”: evidence from restriction analysis of mitochondrial DNA; Mol. Biol. Evol.;  citation_year=1988;  citation_volume=5;  citation_pages=63-78; " /><meta name="citation_reference" content="citation_title=Origin and genetic status of Mus musculus molossinus: a typical example of reticulate evolution in the genus Mus; citation_publisher=Cambridge University Press, Cambridge; Evolution of the House Mouse;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=The ancestor of extant Japanese fancy mice contributed to the mosaic genomes of classical inbred strains; Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=1329-38; " /><meta name="citation_reference" content="citation_title=Origin and radiation of the house mouse: clues from nuclear genes; J. Evol. Biol.;  citation_year=1996;  citation_volume=9;  citation_pages=519-39; " /><meta name="citation_reference" content="citation_title=Mitochondrial DNA sequence variation in the eastern house mouse Mus musculus: comparison with other house mice and report of a 75-bp tandem repeat; Genetics;  citation_year=1996;  citation_volume=143;  citation_pages=427-46; " /><meta name="citation_reference" content="citation_title=Origin and radiation of the house mouse: mitochondrial DNA phylogeny; J. Evol. Biol.;  citation_year=1996;  citation_volume=9;  citation_pages=391-415; " /><meta name="citation_reference" content="citation_title=Mitochondrial DNA variation in the western house mouse (Mus musculus domesticus) close to its site of origin: studies in Turkey; Biol. J. Linn. Soc.;  citation_year=2005;  citation_volume=84;  citation_pages=473-85; " /><meta name="citation_reference" content="citation_title=Genetic differentiation of the house mouse around the Mediterranean basin: matrilineal footprints of early and late colonization; Proc. R. Soc. Lond. B Biol. Sci.;  citation_year=2011;  citation_volume=278;  citation_pages=1034-43; " /><meta name="citation_reference" content="citation_title=House mouse phylogeography; citation_publisher=Cambridge University Press, Cambridge; Evolution of the House Mouse;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=Transmission ratio distortion in mice; Annu. Rev. Genet.;  citation_year=2003;  citation_volume=37;  citation_pages=393-408; " /><meta name="citation_reference" content="citation_title=A novel mouse chromosome 17 hybrid sterility locus: implications for the origin of t haplotypes; Genetics;  citation_year=1991;  citation_volume=129;  citation_pages=237-46; " /><meta name="citation_reference" content="citation_title=Peculiar journey of a selfish chromosome: mouse t-haplotype and meiotic drive; Trends Genet.;  citation_year=1994;  citation_volume=9;  citation_pages=250-4; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of the alpha-globin pseudogene-4 (Hba-ps4) locus in the house mouse species complex reveals a stepwise evolution of t haplotypes; Mol. Biol. Evol.;  citation_year=1993;  citation_volume=10;  citation_pages=971-1001; " /><meta name="citation_reference" content="citation_title=Distribution of t-haplotypes in natural populations of wild house mice; J. Mammal.;  citation_year=1988;  citation_volume=69;  citation_pages=489-99; " /><meta name="citation_reference" content="citation_title=Low diversity of t haplotypes in the eastern form of the house mouse, Mus. musculus L.; , Genetics;  citation_year=1991;  citation_volume=127;  citation_pages=161-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/3/315/1109575/dst059.pdf" /><meta name="description" content="Abstract.  PR/SET domain containing 9 (Prdm9) mediates histone modifications such as H3K4me3 and marks hotspots of meiotic recombination. In many mammalian spec" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/3/315/394421" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/3/315/394421" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/3/315/394421");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst059';
    </script>
<title>
	Prdm9 Polymorphism Unveils Mouse Evolutionary Tracks | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./394421" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdxuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb25vLCBIaXJvbWl0c3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYW1tYWxpYW4gR2VuZXRpY3MgTGFib3JhdG9yeSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCAxMTExIFlhdGEsIE1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvcGh5c2ljcyBhbmQgQmlvY2hlbWlzdHJ5LCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgVG9reW8gMTUzLTg5MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFtdXJhLCBNYXNhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYW1tYWxpYW4gR2VuZXRpY3MgTGFib3JhdG9yeSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCAxMTExIFlhdGEsIE1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9zYWRhLCBOYW9raSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEV2b2x1dGlvbmFyeSBHZW5ldGljcywgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBNaXNoaW1hLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIEhpdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEVjb2xvZ3kgYW5kIEdlbmV0aWNzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRW52aXJvbm1lbnRhbCBFYXJ0aCBTY2llbmNlLCBIb2trYWlkbyBVbml2ZXJzaXR5LCBTYXBwb3JvLCBIb2trYWlkbyAwNjAtMDgxMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmUsIEt1bml5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb1Jlc291cmNlIENlbnRlciwgUklLRU4gVHN1a3ViYSBJbnN0aXR1dGUsIFRzdWt1YmEsIEliYXJha2kgMzA1LTAwNzQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yaXdha2ksIEthenVvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvUmVzb3VyY2UgQ2VudGVyLCBSSUtFTiBUc3VrdWJhIEluc3RpdHV0ZSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtMDA3NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaHRhLCBLdW5paGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvcGh5c2ljcyBhbmQgQmlvY2hlbWlzdHJ5LCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgVG9reW8gMTUzLTg5MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgVG9reW8gMTUzLTg5MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpcm9pc2hpLCBUb3NoaWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYW1tYWxpYW4gR2VuZXRpY3MgTGFib3JhdG9yeSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCAxMTExIFlhdGEsIE1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUHJkbTkgUG9seW1vcnBoaXNtIFVudmVpbHMgTW91c2UgRXZvbHV0aW9uYXJ5IFRyYWNrcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN0MDU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTQvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWlvdGljIHJlY29tYmluYXRpb24gaG90IHNwb3RzIGFuZCBjb2xkIHNwb3RzOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zNjAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hlcmUgdGhlIGNyb3Nzb3ZlcnMgYXJlOiByZWNvbWJpbmF0aW9uIGRpc3RyaWJ1dGlvbnMgaW4gbWFtbWFsczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDEzLTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyB3aWxkLWRlcml2ZWQgSC0yIGhhcGxvdHlwZSBlbmhhbmNpbmcgSy1JQSByZWNvbWJpbmF0aW9uOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9MzAwOyAgY2l0YXRpb25fcGFnZXM9MzcwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgY29udHJvbCBvZiBzZXgtZGVwZW5kZW50IG1laW90aWMgcmVjb21iaW5hdGlvbiBpbiB0aGUgbWFqb3IgaGlzdG9jb21wYXRpYmlsaXR5IGNvbXBsZXggb2YgdGhlIG1vdXNlOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NjgxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZG05IGNvbnRyb2xzIGFjdGl2YXRpb24gb2YgbWFtbWFsaWFuIHJlY29tYmluYXRpb24gaG90c3BvdHM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzI3OyAgY2l0YXRpb25fcGFnZXM9ODM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJpdmUgYWdhaW5zdCBob3RzcG90IG1vdGlmcyBpbiBwcmltYXRlcyBpbXBsaWNhdGVzIHRoZSBQUkRNOSBnZW5lIGluIG1laW90aWMgcmVjb21iaW5hdGlvbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjc7ICBjaXRhdGlvbl9wYWdlcz04NzYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UFJETTkgaXMgYSBtYWpvciBkZXRlcm1pbmFudCBvZiBtZWlvdGljIHJlY29tYmluYXRpb24gaG90c3BvdHMgaW4gaHVtYW5zIGFuZCBtaWNlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyNzsgIGNpdGF0aW9uX3BhZ2VzPTgzNi00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCBkaXZlcnNpdHkgYXQgUFJETTkgaW4gY2hpbXBhbnplZXMgYW5kIGJvbm9ib3M7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzkwNjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYW50cyBvZiB0aGUgcHJvdGVpbiBQUkRNOSBkaWZmZXJlbnRpYWxseSByZWd1bGF0ZSBhIHNldCBvZiBodW1hbiBtZWlvdGljIHJlY29tYmluYXRpb24gaG90c3BvdHMgaGlnaGx5IGFjdGl2ZSBpbiBBZnJpY2FuIHBvcHVsYXRpb25zOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMjM3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UFJETTkgdmFyaWF0aW9uIHN0cm9uZ2x5IGluZmx1ZW5jZXMgcmVjb21iaW5hdGlvbiBob3Qtc3BvdCBhY3Rpdml0eSBhbmQgbWVpb3RpYyBpbnN0YWJpbGl0eSBpbiBodW1hbnM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9ODU5LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHJlY29tYmluYXRpb24gaXMgZGlyZWN0ZWQgYXdheSBmcm9tIGZ1bmN0aW9uYWwgZ2Vub21pYyBlbGVtZW50cyBpbiBtaWNlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg1OyAgY2l0YXRpb25fcGFnZXM9NjQyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWFudHMgaW4gUkVDOCwgUk5GMjEyLCBhbmQgUFJETTkgaW5mbHVlbmNlIG1hbGUgcmVjb21iaW5hdGlvbiBpbiBjYXR0bGU7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjg1NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlzdG9uZSBIMyBtZXRoeWx0cmFuc2ZlcmFzZSBjb250cm9scyBlcGlnZW5ldGljIGV2ZW50cyByZXF1aXJlZCBmb3IgbWVpb3RpYyBwcm9waGFzZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzODsgIGNpdGF0aW9uX3BhZ2VzPTM3NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1vdXNlIHNwZWNpYXRpb24gZ2VuZSBlbmNvZGVzIGEgbWVpb3RpYyBoaXN0b25lIEgzIG1ldGh5bHRyYW5zZmVyYXNlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyMzsgIGNpdGF0aW9uX3BhZ2VzPTM3My01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFsbGVsaWMgYW5kIGludGVyZ2VuaWMgaW5jb21wYXRpYmlsaXRpZXMgb2YgdGhlIFByZG05IChIc3QxKSBnZW5lIGluIG1vdXNlIGh5YnJpZCBzdGVyaWxpdHk7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzA0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dHJhb3JkaW5hcnkgbW9sZWN1bGFyIGV2b2x1dGlvbiBpbiB0aGUgUFJETTkgZmVydGlsaXR5IGdlbmU7IFBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lODUwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY2VsZXJhdGVkIGV2b2x1dGlvbiBvZiB0aGUgUHJkbTkgc3BlY2lhdGlvbiBnZW5lIGFjcm9zcyBkaXZlcnNlIG1ldGF6b2FuIHRheGE7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDc1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGlvbiBvZiBETkEgYmluZGluZyBzaXRlcyBmb3IgemluYyBmaW5nZXJzIHVzaW5nIHJhdGlvbmFsbHkgcmFuZG9taXplZCBETkEgcmV2ZWFscyBjb2RlZCBpbnRlcmFjdGlvbnM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MTExNjgtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBiaW5kaW5nIHNwZWNpZmljaXRpZXMgb2YgdGhlIGxvbmcgemluYy1maW5nZXIgcmVjb21iaW5hdGlvbiBwcm90ZWluIFBSRE05OyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UZW1wb3JhbCwgc3BhdGlhbCwgYW5kIGVjb2xvZ2ljYWwgbW9kZXMgb2YgZXZvbHV0aW9uIG9mIEV1cmFzaWFuIE11cyBiYXNlZCBvbiBtaXRvY2hvbmRyaWFsIGFuZCBudWNsZWFyIGdlbmUgc2VxdWVuY2VzOyBNb2wuIFBoeWxvZ2VuZXQuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NjI2LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIGhvdXNlIG1pY2U7IEFubnUuIFJldi4gRWNvbC4gRXZvbC4gU3lzdDsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTExOS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGFuZCBkaXNwZXJzYWwgaGlzdG9yeSBvZiBFdXJhc2lhbiBob3VzZSBtaWNlIE11cyBtdXNjdWx1cyBjbGFyaWZpZWQgYnkgbW9yZSBleHRlbnNpdmUgZ2VvZ3JhcGhpYyBzYW1wbGluZyBvZiBtaXRvY2hvbmRyaWFsIEROQTsgSGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9Mzc1LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3VzZSBnZW5vbWljIHZhcmlhdGlvbiBhbmQgaXRzIGVmZmVjdCBvbiBwaGVub3R5cGVzIGFuZCBnZW5lIHJlZ3VsYXRpb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00Nzc7ICBjaXRhdGlvbl9wYWdlcz0yODktOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaWdpbnMgb2YgbGFib3JhdG9yeSBtaWNlIGRlZHVjZWQgZnJvbSByZXN0cmljdGlvbiBwYXR0ZXJucyBvZiBtaXRvY2hvbmRyaWFsIEROQTsgRGlmZmVyZW50aWF0aW9uOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIHRoZSBzdWJzcGVjaWZpYyBvcmlnaW4gb2YgdGhlIGxhYm9yYXRvcnkgbW91c2U7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTEwMC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgdGhlIG1vdXNlIHQgaGFwbG90eXBlOiByZWNlbnQgYW5kIHdvcmxkd2lkZSBpbnRyb2dyZXNzaW9uIHRvIE11cyBtdXNjdWx1czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz02ODUxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUwgVzogaW1wcm92aW5nIHRoZSBzZW5zaXRpdml0eSBvZiBwcm9ncmVzc2l2ZSBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgdGhyb3VnaCBzZXF1ZW5jZSB3ZWlnaHRpbmcsIHBvc2l0aW9uLXNwZWNpZmljIGdhcCBwZW5hbHRpZXMgYW5kIHdlaWdodCBtYXRyaXggY2hvaWNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00NjczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBtZXRob2QgZm9yIGVzdGltYXRpbmcgZXZvbHV0aW9uYXJ5IHJhdGVzIG9mIGJhc2Ugc3Vic3RpdHV0aW9ucyB0aHJvdWdoIGNvbXBhcmF0aXZlIHN0dWRpZXMgb2YgbnVjbGVvdGlkZSBzZXF1ZW5jZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZWlnaGJvci1qb2luaW5nIG1ldGhvZDogYSBuZXcgbWV0aG9kIGZvciByZWNvbnN0cnVjdGluZyBwaHlsb2dlbmV0aWMgdHJlZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWJzcGVjaWZpYyBvcmlnaW4gYW5kIGhhcGxvdHlwZSBkaXZlcnNpdHkgaW4gdGhlIGxhYm9yYXRvcnkgbW91c2U7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NjQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3NpdGl2ZSBEYXJ3aW5pYW4gc2VsZWN0aW9uIHByb21vdGVzIGNoYXJnZSBwcm9maWxlIGRpdmVyc2l0eSBpbiB0aGUgYW50aWdlbi1iaW5kaW5nIGNsZWZ0IG9mIGNsYXNzIEkgbWFqb3ItaGlzdG9jb21wYXRpYmlsaXR5LWNvbXBsZXggbW9sZWN1bGVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz01MTUtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcmUgYWxsZWxpYyBmb3JtcyBvZiBQUkRNOSBhc3NvY2lhdGVkIHdpdGggY2hpbGRob29kIGxldWtlbW9nZW5lc2lzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQxOS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHlicmlkIG9yaWdpbiBvZiBKYXBhbmVzZSBtaWNlIOKAnE11cyBtdXNjdWx1cyBtb2xvc3NpbnVz4oCdOiBldmlkZW5jZSBmcm9tIHJlc3RyaWN0aW9uIGFuYWx5c2lzIG9mIG1pdG9jaG9uZHJpYWwgRE5BOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz02My03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIGFuZCBnZW5ldGljIHN0YXR1cyBvZiBNdXMgbXVzY3VsdXMgbW9sb3NzaW51czogYSB0eXBpY2FsIGV4YW1wbGUgb2YgcmV0aWN1bGF0ZSBldm9sdXRpb24gaW4gdGhlIGdlbnVzIE11czsgY2l0YXRpb25fcHVibGlzaGVyPUNhbWJyaWRnZSBVbml2ZXJzaXR5IFByZXNzLCBDYW1icmlkZ2U7IEV2b2x1dGlvbiBvZiB0aGUgSG91c2UgTW91c2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhbmNlc3RvciBvZiBleHRhbnQgSmFwYW5lc2UgZmFuY3kgbWljZSBjb250cmlidXRlZCB0byB0aGUgbW9zYWljIGdlbm9tZXMgb2YgY2xhc3NpY2FsIGluYnJlZCBzdHJhaW5zOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEzMjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaWdpbiBhbmQgcmFkaWF0aW9uIG9mIHRoZSBob3VzZSBtb3VzZTogY2x1ZXMgZnJvbSBudWNsZWFyIGdlbmVzOyBKLiBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NTE5LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvY2hvbmRyaWFsIEROQSBzZXF1ZW5jZSB2YXJpYXRpb24gaW4gdGhlIGVhc3Rlcm4gaG91c2UgbW91c2UgTXVzIG11c2N1bHVzOiBjb21wYXJpc29uIHdpdGggb3RoZXIgaG91c2UgbWljZSBhbmQgcmVwb3J0IG9mIGEgNzUtYnAgdGFuZGVtIHJlcGVhdDsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTQzOyAgY2l0YXRpb25fcGFnZXM9NDI3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gYW5kIHJhZGlhdGlvbiBvZiB0aGUgaG91c2UgbW91c2U6IG1pdG9jaG9uZHJpYWwgRE5BIHBoeWxvZ2VueTsgSi4gRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM5MS00MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pdG9jaG9uZHJpYWwgRE5BIHZhcmlhdGlvbiBpbiB0aGUgd2VzdGVybiBob3VzZSBtb3VzZSAoTXVzIG11c2N1bHVzIGRvbWVzdGljdXMpIGNsb3NlIHRvIGl0cyBzaXRlIG9mIG9yaWdpbjogc3R1ZGllcyBpbiBUdXJrZXk7IEJpb2wuIEouIExpbm4uIFNvYy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz00NzMtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGlmZmVyZW50aWF0aW9uIG9mIHRoZSBob3VzZSBtb3VzZSBhcm91bmQgdGhlIE1lZGl0ZXJyYW5lYW4gYmFzaW46IG1hdHJpbGluZWFsIGZvb3RwcmludHMgb2YgZWFybHkgYW5kIGxhdGUgY29sb25pemF0aW9uOyBQcm9jLiBSLiBTb2MuIExvbmQuIEIgQmlvbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzg7ICBjaXRhdGlvbl9wYWdlcz0xMDM0LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3VzZSBtb3VzZSBwaHlsb2dlb2dyYXBoeTsgY2l0YXRpb25fcHVibGlzaGVyPUNhbWJyaWRnZSBVbml2ZXJzaXR5IFByZXNzLCBDYW1icmlkZ2U7IEV2b2x1dGlvbiBvZiB0aGUgSG91c2UgTW91c2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbWlzc2lvbiByYXRpbyBkaXN0b3J0aW9uIGluIG1pY2U7IEFubnUuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MzkzLTQwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBtb3VzZSBjaHJvbW9zb21lIDE3IGh5YnJpZCBzdGVyaWxpdHkgbG9jdXM6IGltcGxpY2F0aW9ucyBmb3IgdGhlIG9yaWdpbiBvZiB0IGhhcGxvdHlwZXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTEyOTsgIGNpdGF0aW9uX3BhZ2VzPTIzNy00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVjdWxpYXIgam91cm5leSBvZiBhIHNlbGZpc2ggY2hyb21vc29tZTogbW91c2UgdC1oYXBsb3R5cGUgYW5kIG1laW90aWMgZHJpdmU7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI1MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgYW5hbHlzaXMgb2YgdGhlIGFscGhhLWdsb2JpbiBwc2V1ZG9nZW5lLTQgKEhiYS1wczQpIGxvY3VzIGluIHRoZSBob3VzZSBtb3VzZSBzcGVjaWVzIGNvbXBsZXggcmV2ZWFscyBhIHN0ZXB3aXNlIGV2b2x1dGlvbiBvZiB0IGhhcGxvdHlwZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz05NzEtMTAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdHJpYnV0aW9uIG9mIHQtaGFwbG90eXBlcyBpbiBuYXR1cmFsIHBvcHVsYXRpb25zIG9mIHdpbGQgaG91c2UgbWljZTsgSi4gTWFtbWFsLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTQ4OS05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG93IGRpdmVyc2l0eSBvZiB0IGhhcGxvdHlwZXMgaW4gdGhlIGVhc3Rlcm4gZm9ybSBvZiB0aGUgaG91c2UgbW91c2UsIE11cy4gbXVzY3VsdXMgTC47ICwgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTI3OyAgY2l0YXRpb25fcGFnZXM9MTYxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8zLzMxNS8xMTA5NTc1L2RzdDA1OS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUFIvU0VUIGRvbWFpbiBjb250YWluaW5nIDkgKFByZG05KSBtZWRpYXRlcyBoaXN0b25lIG1vZGlmaWNhdGlvbnMgc3VjaCBhcyBIM0s0bWUzIGFuZCBtYXJrcyBob3RzcG90cyBvZiBtZWlvdGljIHJlY29tYmluYXRpb24uIEluIG1hbnkgbWFtbWFsaWFuIHNwZWMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM5NDQyMSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSboIC5qSbWHWq942/qspRvsDkVhw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f315%2f394421" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f315%2f394421" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15369" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&amp;Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/3">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5821328">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821328">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821330">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821330">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821343">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821343">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821357">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821357">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821455">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821455">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821475">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821475">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821479">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821479">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821483">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821483">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5821487">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5821487">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/3/299/394334">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/3/327/394723">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"><em>Prdm9</em> Polymorphism Unveils Mouse Evolutionary Tracks
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiromitsu Kono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Mammalian Genetics Laboratory</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">1111 Yata, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biophysics and Biochemistry</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">Tokyo 153-8902</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiromitsu+Kono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kono H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kono H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiromitsu Kono</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaru Tamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Mammalian Genetics Laboratory</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">1111 Yata, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaru+Tamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tamura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tamura M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masaru Tamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Osada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Division of Evolutionary Genetics</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Osada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Osada N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Osada N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naoki Osada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hitoshi Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Laboratory of Ecology and Genetics, Graduate School of Environmental Earth Science</div>, <div class="insititution">Hokkaido University</div>, <div class="addr-line">Sapporo, Hokkaido 060-0810</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hitoshi+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hitoshi Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kuniya Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">BioResource Center</div>, <div class="insititution">RIKEN Tsukuba Institute</div>, <div class="addr-line">Tsukuba, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kuniya+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kuniya Abe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuo Moriwaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">BioResource Center</div>, <div class="insititution">RIKEN Tsukuba Institute</div>, <div class="addr-line">Tsukuba, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuo+Moriwaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Moriwaki K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Moriwaki K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuo Moriwaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kunihiro Ohta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biophysics and Biochemistry</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">Tokyo 153-8902</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Life Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">Tokyo 153-8902</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kunihiro+Ohta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohta K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohta K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kunihiro Ohta</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshihiko Shiroishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Mammalian Genetics Laboratory</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">1111 Yata, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 55-981-6818. Fax. +81 55-981-6817. Email: <a href="mailto:tshirois@lab.nig.ac.jp" target="_blank">tshirois@lab.nig.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshihiko+Shiroishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shiroishi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shiroishi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshihiko Shiroishi</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 315–326, <a href='https://doi.org/10.1093/dnares/dst059'>https://doi.org/10.1093/dnares/dst059</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">20 January 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">02 September 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">11 December 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="394421" href="/dnaresearch/article-pdf/21/3/315/1109575/dst059.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hiromitsu Kono, Masaru Tamura, Naoki Osada, Hitoshi Suzuki, Kuniya Abe, Kazuo Moriwaki, Kunihiro Ohta, Toshihiko Shiroishi;  <em>Prdm9</em> Polymorphism Unveils Mouse Evolutionary Tracks, <em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 315–326, <a href='https://doi.org/10.1093/dnares/dst059'>https://doi.org/10.1093/dnares/dst059</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/394421?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/394421?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/394421?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/394421?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/394421?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/394421?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/394421?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BPrdm9%26lt%3B%2Fem%26gt%3B%20Polymorphism%20Unveils%20Mouse%20Evolutionary%20Tracks&amp;publicationDate=2014-01-20&amp;volumeNum=21&amp;issueNum=3&amp;author=Kono%2C%20Hiromitsu%3B%20Tamura%2C%20Masaru&amp;startPage=315&amp;endPage=326&amp;contentId=10.1093%2Fdnares%2Fdst059&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f315%2f394421" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5821328 id="5821328" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>PR/SET domain containing 9 (Prdm9) mediates histone modifications such as H3K4me3 and marks hotspots of meiotic recombination. In many mammalian species, the <em>Prdm9</em> gene is highly polymorphic. <em>Prdm9</em> polymorphism is assumed to play two critical roles in evolution: to diversify the spectrum of meiotic recombination hotspots and to cause male hybrid sterility, leading to reproductive isolation and speciation. Nevertheless, information about <em>Prdm9</em> sequences in natural populations is very limited. In this study, we conducted a comprehensive population survey on <em>Prdm9</em> polymorphism in the house mouse, <em>Mus musculus.</em> Overall <em>M. musculus Prdm9</em> displays an extraordinarily high level of polymorphism, particularly in regions encoding zinc finger repeats, which recognize recombination hotspots. <em>Prdm9</em> alleles specific to various <em>M. musculus</em> subspecies dominate in subspecies territories. Moreover, introgression into other subspecies territories was found for highly divergent <em>Prdm9</em> alleles associated with <em>t-</em>haplotype. The results of our phylogeographical analysis suggest that the requirement for hotspot diversity depends on geographical range and time span in mouse evolution, and that <em>Prdm9</em> polymorphism has not been maintained by a simple balanced selection in the population of each subspecies.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Prdm9">Prdm9</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mouse">mouse</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="polymorphism">polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a></div>                            <h2 scrollto-destination=5821330 id="5821330" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Meiotic recombination enhances the genetic diversity in natural populations and contributes to genome evolution. In organisms as diverse as yeasts and mammals, meiotic recombination events do not take place at random but are clustered at specific genomic regions, referred to as recombination hotspots.<sup><a reveal-id="DST059C1" data-open="DST059C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST059C2" data-open="DST059C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Nevertheless, until recently, the molecular basis underlying determination of the hotspots has been elusive.</p><p>We previously reported that <em>wm7</em>, a wild mouse-derived haplotype of the major histocompatibility complex (MHC) on chromosome 17, enhances meiotic recombination at a hotspot within the MHC.<sup><a reveal-id="DST059C3" data-open="DST059C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Subsequently, we found that a factor genetically linked to this hotspot determines its recombination rate, and that the <em>wm7</em> haplotype carries a recombination-enhancing factor.<sup><a reveal-id="DST059C4" data-open="DST059C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Recently, the factor was reported to be a histone methyltransferase, PR/SET domain containing 9 (Prdm9).<sup><a reveal-id="DST059C5  DST059C7" data-open="DST059C5  DST059C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> Prdm9 mediates histone modifications, such as H3K4me3, and is thought to mark recombination hotspots. Since its identification, many reports have shown that <em>Prdm9</em> polymorphisms correlate well with site variation in hotspots in different mammalian species, including humans, chimpanzees, and mice.<sup><a reveal-id="DST059C8  DST059C12" data-open="DST059C8  DST059C12" class="link link-ref link-reveal xref-bibr">8–12</a></sup> Genome-wide ChIP analysis with antibodies that recognize DMC1 and RAD51 performed for two different <em>Prdm9</em> alleles in a common genetic background revealed that <em>Prdm9</em> variation can account for the site specificity of almost all of the DNA double-strand breaks that initiate meiotic recombination.<sup><a reveal-id="DST059C11" data-open="DST059C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Thus, <em>Prdm9</em> appears to be a major trans-acting factor for determining the spectrum of hotspots in mice and humans, and perhaps in other mammalian species as well.<sup><a reveal-id="DST059C8  DST059C11" data-open="DST059C8  DST059C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup></p><p>A <em>Prdm9</em> knockout mouse shows meiosis arrest, indicating that Prdm9 histone methyltransferase activity is involved in progression of meiosis.<sup><a reveal-id="DST059C13" data-open="DST059C13" class="link link-ref link-reveal xref-bibr">13</a></sup> This function has also been implicated in reproductive isolation, a process that prevents free exchange of genes between two genetically divergent populations, leading to speciation. In crosses between the mouse subspecies <em>Mus musculus domesticus</em> and <em>M. m. musculus</em>, the F1 male hybrids are sometimes sterile<em>.</em> The locus responsible for this hybrid sterility was named Hybrid sterility 1 (<em>Hst1</em>), and mapped to chromosome 17. Recent results revealed that <em>Hst1</em> is identical to <em>Prdm9</em>, such that <em>Prdm9</em> became the first speciation gene to be reported in mammalian species.<sup><a reveal-id="DST059C14" data-open="DST059C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST059C15" data-open="DST059C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Comprehensive information on <em>Prdm9</em> polymorphisms in natural populations should provide an important insight into <em>Prdm9</em> functions, especially in evolution. Past studies have focused on <em>Prdm9</em> polymorphism in human populations. The results revealed that the worldwide population is quite diverse, with differences including repeat number variations of the zinc finger (ZF) DNA-binding repeat and amino acid substitutions in the zinc finger array (ZFA) of the <em>Prdm9</em> C-terminal domain.<sup><a reveal-id="DST059C7" data-open="DST059C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST059C10" data-open="DST059C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST059C16" data-open="DST059C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST059C17" data-open="DST059C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Importantly, non-synonymous substitutions preferentially occur at three amino acids along the α-helix domain of the ZF, which are involved in recognition of the hotspot nucleotide motifs.<sup><a reveal-id="DST059C18" data-open="DST059C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Currently, it is thought that hotspot diversity in natural human populations can be attributed to polymorphism of the Prdm9 ZFA.<sup><a reveal-id="DST059C19" data-open="DST059C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>There are marked advantages to studying <em>Prdm9</em> polymorphism in natural populations of <em>M. musculus.</em> First, the phylogenetics of <em>M. musculus</em> is well established.<sup><a reveal-id="DST059C20" data-open="DST059C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST059C21" data-open="DST059C21" class="link link-ref link-reveal xref-bibr">21</a></sup><em>Mus musculus</em> is a complex species, and comprises distinct ‘phylogroups’ or subspecies. The results of extensive phylogenetic analysis of Eurasian wild mice revealed that these subspecies diverged roughly 0.5–1.0 million years ago.<sup><a reveal-id="DST059C20" data-open="DST059C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Their habitats are demarcated throughout the Eurasian continent.<sup><a reveal-id="DST059C21" data-open="DST059C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST059C22" data-open="DST059C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Secondly, whereas subspecies of <em>M. musculus</em> are thought to be in an early stage of speciation, neighbouring species, including <em>M. spretus</em>, <em>M. macedonicus</em>, and <em>M. spicilegus</em>, inhabit areas overlapping those of <em>M. musculus.</em><sup><a reveal-id="DST059C22" data-open="DST059C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Thus, <em>M. musculus</em> and its neighbouring species provide an ideal model system to study phylogeography and speciation. Thus far, mouse <em>Prdm9</em> polymorphisms has been investigated in commonly used laboratory inbred strains and inbred strains derived from wild mice.<sup><a reveal-id="DST059C5" data-open="DST059C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST059C7" data-open="DST059C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST059C23" data-open="DST059C23" class="link link-ref link-reveal xref-bibr">23</a></sup> However, in these studies, sample collection was limited, as laboratory inbred strains originate predominantly from a single western European subspecies, <em>M. m. domesticus.</em><sup><a reveal-id="DST059C24" data-open="DST059C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST059C25" data-open="DST059C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Moreover, only a limited number of inbred strains derived from wild-captured mice were included in previous studies.<sup><a reveal-id="DST059C5" data-open="DST059C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST059C23" data-open="DST059C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p><p>In this study, we extended the population survey to wild mice collected in natural populations of <em>M. musculus</em> subspecies and neighbouring species, as well as inbred strains derived from wild mice. We also investigated <em>Prdm9</em> polymorphism in mice with the <em>t-</em>haplotype chromosome variant, which is characterized by long inversions on chromosome 17 and is linked to the <em>Prdm9</em> locus. Genes on the <em>t</em>-haplotype are highly divergent from those on wild-type chromosome 17.<sup><a reveal-id="DST059C26" data-open="DST059C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The results of our study confirm that <em>Prdm9</em> polymorphisms are concentrated in ZFA with extensive variation of the ZF repeat number and hyper-variation of amino acids at the three DNA recognition sites within the ZF. Our survey of 79 wild-captured mice and 37 inbred strains revealed as many as 57 different <em>Prdm9</em> alleles in <em>M. musculus.</em> In contrast, some alleles were predominant in two subspecies, <em>M. m. domesticus</em> and <em>M. m. musculus</em>. The overall phylogeography of mouse <em>Prdm9</em> reflects evolutionary episodes of this species. More importantly, <em>Prdm9</em> alleles that predominate in one subspecies are often found in territories of other subspecies. Likewise, highly divergent <em>Prdm9</em> alleles associated with the <em>t-</em>haplotype were found to introgress into all subspecies of <em>M. musculus</em>.</p>                            <h2 scrollto-destination=5821343 id="5821343" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5821344 id="5821344" class="section-title" data-legacy-id=s2a>2.1. Mice</h3>
<p>Nine mouse strains (<em>M. m. molossinus</em>, MSM/Ms; <em>M. m. musculus</em>, NJL/Ms, KJR/Ms, BLG2/Ms, SWN/Ms, CHD/Ms; <em>M. m. castaneus</em>, HMI/Ms; <em>M. m. domesticus</em>, PGN2/Ms, BFM/Ms), one Japanese fancy mouse-derived strain (<em>M. m. molossinus</em>, JF1/Ms), and two MHC congenic mouse strains (B10.R209, B10D2.TCH/+) were maintained at the Genetic Strains Research Center, National Institute of Genetics (NIG). A classical laboratory mouse strain, C57BL/10Snf, was purchased from the Jackson Laboratory and maintained at the NIG. The inbred strain SPR2/Rbrc, derived from <em>M. spretus</em>, was provided by the RIKEN BioResource Center (BRC) through the National BioResource Project, which is funded by the Ministry of Education, Culture, Sports, Science and Technology (MEXT), Japan. Inbred strains used in this study are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. All animal experiments were performed in accordance with protocols approved by the Animal Care and Use Committee of NIG.</p>                            <h3 scrollto-destination=5821346 id="5821346" class="section-title" data-legacy-id=s2b>2.2. <em>Prdm9</em> cDNA synthesis</h3>
<p>Total testes RNA from each mouse strain was isolated with Isogen (Nippon Gene). Complementary DNA (cDNA) was synthesized using the Primescript RT reagent kit (TAKARA) according to the manufacturer's instructions.</p>                            <h3 scrollto-destination=5821348 id="5821348" class="section-title" data-legacy-id=s2c>2.3. Mouse genomic DNA samples and PCR conditions</h3>
<p>Most genomic DNA samples, including classical inbred and wild-captured mice, were prepared by the Genetic Strains Research Center at the NIG. Some were prepared at Hokkaido University. Several genomic DNA samples were purchased from the Jackson Laboratory. Genomic DNA samples from <em>t</em>-haplotype mice were kindly provided by Joe Nadeau or by the RIKEN BRC. All the genomic DNA samples are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. To prevent errors in PCR, we used high-fidelity DNA polymerases (KAPA HiFi from Kapa Biosystems or KOD Neo FX from Toyobo). In addition, we repeated independent amplification at least three times for each sample. PCR primer sets and conditions are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3,</a> and the amplified sites are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>.</p>                            <h3 scrollto-destination=5821350 id="5821350" class="section-title" data-legacy-id=s2d>2.4. Sequence analysis</h3>
<p>To determine the sequences of cDNA, ZFA, high SNP region 1 (HSR1) of <em>Prdm9</em>, and intron of <em>T-complex protein 1</em> (<em>Tcp1</em>), we sequenced PCR products directly or after subcloning. For subcloning, PCR products were extracted with a QIAquick Gel Extraction Kit (QIAGEN) after electrophoresis. Then, the purified PCR products were subcloned into pCR-Blunt II-TOPO (Invitrogen). For sequencing, we used the BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems) and a 3130XL DNA Analyser (Applied Biosystems). Primers used for sequence analyses are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>. We analysed at least six clones per sample and carried out multiple independent experiments for each allele. All sequence data from this study were submitted to the DDBJ Sequence Read Archive (Accession numbers AB843858 to AB844116 and AB846828).</p>                            <h3 scrollto-destination=5821352 id="5821352" class="section-title" data-legacy-id=s2e>2.5. Coding of ZF repeats</h3>
<p>The <em>Prdm9</em> ZFA nucleotide sequence was conceptually translated (Fig. <a reveal-id="DST059F1" data-open="DST059F1" class="link link-reveal link-table xref-fig">1</a>B), and then the sequence of 28 amino acids from 512S to the last residue, which corresponds to the ZFA, was extracted from all ZF repeats. We assigned a one-letter box with a given colour to every ZF repeat that had a given amino acid triad at the most variable positions of the α-helix domain (−1, 3, and 6). When amino acid variation was found for a ZF repeat at less variable positions (−5, −2, and 1), the right bottom of the box was labelled with the variant amino acids. </p>                        <a id="5821354" scrollto-destination="5821354"></a>
<div id="DST059F1" data-id="DST059F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05901.png?Expires=2147483647&Signature=BCggeGKvgxQudLdDbRF3xaKl3ciGdJjNUuucxLXQjjRlyCFsY3Z-lV6-8Y-OnW~sXZIiG2DjLdqhdDlLiI7WsL9pmi-b9GDwIc2vc4ToHHhqkSHIk9JJKgYAbiomev2532MGlDZN2qAqI30tsGqiWUeouqWJwXvRe3WJO10HplQ18C-cxwYq9kpPqeKD7LcZheMZR2wg8RvTUyf08T7hyWdLv66~ydSW5Gcui5vm13OJBdWvImprrXyXo0S0E0GMwSsSGkwV6ndNvZgJ5Dt3o7SBIbiWBEWF~QsrcM3IZFsgUtO87WHsqb7ewM6BgMTiO8bvmmyjBAGzqAQlY5skmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Polymorphisms in Prdm9 cDNA sequences from inbred mouse strains. Amino acid polymorphism of Prdm9 excluding ZFA is summarized. Full-length cDNA of Prdm9 was sequenced for 12 inbred strains derived from M. musculus and one inbred strain derived from a neighbouring species, M. spretus. Nucleotide sequences of the inbred strains were compared with the C57BL/6J reference sequence (NCBI mm9). The C57BL/10J sequence was identical to the reference. Comparison revealed nucleotide substitutions that lead to 16 amino acid substitutions in total, as well as two insertions and one deletion of an amino acid relative to the reference. No amino acid substitution was observed in the PR/SET domains. Upper arrows indicate positions of amino acid variation. An asterisk (*) indicates an interspecific variation (or variation in B10.D2-TCH/+). All amino acid variations are listed in the table below the diagram of Prdm9 protein. The letters S, I, and D in variant types indicate substitution, insertion, and deletion of an amino acid, respectively." path-from-xml="dst05901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821354/dst05901.tiff" path-from-xml="dst05901.tiff" target="_blank">View large</a><a section="5821354" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05901.png?Expires=2147483647&Signature=tHFlYWksEm07L3sk2muLRGLaY9DGHzNrIeoECFesFc1IJhMiynhp8mEiKzbPijLB-q1EQMh1VUp6upntgGKgcEIfPJuQP4Yc5sCh1WVqfl9sIwF-82S3KM3d77MbNzRw46tbqGglGTylBHj-BJ2mNmxhiESO48fnxJ0BmI2kq2msg6cGsHMnNEVRXxHWVArZ0-MSUrWTSp4lDhVk83fdR5U9cvrUpxQhaxMOxEu2BFDKInATUgZOxbn-hjaaFtb6yEpyAaLEjAjT1TkDnNC2ZkZ0~Fk--Kt6-uUYLgm8J6zQkg54btNd8eldbYYMRryX~di0Lczf~Opovw~1wa2DcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821354&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Polymorphisms in <em>Prdm9</em> cDNA sequences from inbred mouse strains. Amino acid polymorphism of <em>Prdm9</em> excluding ZFA is summarized. Full-length cDNA of <em>Prdm9</em> was sequenced for 12 inbred strains derived from <em>M. musculus</em> and one inbred strain derived from a neighbouring species, <em>M. spretus</em>. Nucleotide sequences of the inbred strains were compared with the C57BL/6J reference sequence (NCBI mm9). The C57BL/10J sequence was identical to the reference. Comparison revealed nucleotide substitutions that lead to 16 amino acid substitutions in total, as well as two insertions and one deletion of an amino acid relative to the reference. No amino acid substitution was observed in the PR/SET domains. Upper arrows indicate positions of amino acid variation. An asterisk (*) indicates an interspecific variation (or variation in B10.D2-TCH<em>/+</em>). All amino acid variations are listed in the table below the diagram of Prdm9 protein. The letters S, I, and D in variant types indicate substitution, insertion, and deletion of an amino acid, respectively.</p></div></div><div content-id="DST059F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05901.png?Expires=2147483647&Signature=BCggeGKvgxQudLdDbRF3xaKl3ciGdJjNUuucxLXQjjRlyCFsY3Z-lV6-8Y-OnW~sXZIiG2DjLdqhdDlLiI7WsL9pmi-b9GDwIc2vc4ToHHhqkSHIk9JJKgYAbiomev2532MGlDZN2qAqI30tsGqiWUeouqWJwXvRe3WJO10HplQ18C-cxwYq9kpPqeKD7LcZheMZR2wg8RvTUyf08T7hyWdLv66~ydSW5Gcui5vm13OJBdWvImprrXyXo0S0E0GMwSsSGkwV6ndNvZgJ5Dt3o7SBIbiWBEWF~QsrcM3IZFsgUtO87WHsqb7ewM6BgMTiO8bvmmyjBAGzqAQlY5skmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Polymorphisms in Prdm9 cDNA sequences from inbred mouse strains. Amino acid polymorphism of Prdm9 excluding ZFA is summarized. Full-length cDNA of Prdm9 was sequenced for 12 inbred strains derived from M. musculus and one inbred strain derived from a neighbouring species, M. spretus. Nucleotide sequences of the inbred strains were compared with the C57BL/6J reference sequence (NCBI mm9). The C57BL/10J sequence was identical to the reference. Comparison revealed nucleotide substitutions that lead to 16 amino acid substitutions in total, as well as two insertions and one deletion of an amino acid relative to the reference. No amino acid substitution was observed in the PR/SET domains. Upper arrows indicate positions of amino acid variation. An asterisk (*) indicates an interspecific variation (or variation in B10.D2-TCH/+). All amino acid variations are listed in the table below the diagram of Prdm9 protein. The letters S, I, and D in variant types indicate substitution, insertion, and deletion of an amino acid, respectively." path-from-xml="dst05901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821354/dst05901.tiff" path-from-xml="dst05901.tiff" target="_blank">View large</a><a section="5821354" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05901.png?Expires=2147483647&Signature=tHFlYWksEm07L3sk2muLRGLaY9DGHzNrIeoECFesFc1IJhMiynhp8mEiKzbPijLB-q1EQMh1VUp6upntgGKgcEIfPJuQP4Yc5sCh1WVqfl9sIwF-82S3KM3d77MbNzRw46tbqGglGTylBHj-BJ2mNmxhiESO48fnxJ0BmI2kq2msg6cGsHMnNEVRXxHWVArZ0-MSUrWTSp4lDhVk83fdR5U9cvrUpxQhaxMOxEu2BFDKInATUgZOxbn-hjaaFtb6yEpyAaLEjAjT1TkDnNC2ZkZ0~Fk--Kt6-uUYLgm8J6zQkg54btNd8eldbYYMRryX~di0Lczf~Opovw~1wa2DcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821354&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Polymorphisms in <em>Prdm9</em> cDNA sequences from inbred mouse strains. Amino acid polymorphism of <em>Prdm9</em> excluding ZFA is summarized. Full-length cDNA of <em>Prdm9</em> was sequenced for 12 inbred strains derived from <em>M. musculus</em> and one inbred strain derived from a neighbouring species, <em>M. spretus</em>. Nucleotide sequences of the inbred strains were compared with the C57BL/6J reference sequence (NCBI mm9). The C57BL/10J sequence was identical to the reference. Comparison revealed nucleotide substitutions that lead to 16 amino acid substitutions in total, as well as two insertions and one deletion of an amino acid relative to the reference. No amino acid substitution was observed in the PR/SET domains. Upper arrows indicate positions of amino acid variation. An asterisk (*) indicates an interspecific variation (or variation in B10.D2-TCH<em>/+</em>). All amino acid variations are listed in the table below the diagram of Prdm9 protein. The letters S, I, and D in variant types indicate substitution, insertion, and deletion of an amino acid, respectively.</p></div></div>                            <h3 scrollto-destination=5821355 id="5821355" class="section-title" data-legacy-id=s2f>2.6. Phylogenetic analysis</h3>
<p>To construct a phylogenetic tree of ZFA, we aligned ZFA repeat units using a progressive multiple sequence alignment algorithm implemented in ClustalW.<sup><a reveal-id="DST059C27" data-open="DST059C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Briefly, each repeat unit was converted to a one-letter code according to the amino acid residues at five signature sites, i.e. the three most variable and two less variable among ZF repeats. Mismatch scores between repeat units were given by the number of different signature sites between two units. The gap open penalty was set to 0.5 and the gap extension penalty we used was 0.1. The algorithm aligns ZF repeat unit sequences by maximizing the alignment score, as is typical for nucleotide and protein sequence alignment. After alignment, the repeat unit sequences were transformed into nucleotide sequences and nucleotide distance was measured using Kimura's two-parameter method.<sup><a reveal-id="DST059C28" data-open="DST059C28" class="link link-ref link-reveal xref-bibr">28</a></sup> A phylogenetic tree was constructed using the neighbour-joining method<sup><a reveal-id="DST059C29" data-open="DST059C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and implemented using the MEGA 5 software.<sup><a reveal-id="DST059C30" data-open="DST059C30" class="link link-ref link-reveal xref-bibr">30</a></sup> A phylogenetic tree of <em>HSR1</em> and <em>Tcp1</em> was similarly constructed using neighbour-joining with Kimura's two-parameter distances. Bootstrap resampling tests were conducted with 1000 iterations.</p>                            <h2 scrollto-destination=5821357 id="5821357" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5821358 id="5821358" class="section-title" data-legacy-id=s3a>3.1. Polymorphism in the cDNA sequence of <em>Prdm9</em></h3>
<p>To determine the overall nature of mouse <em>Prdm9</em> polymorphism, we first cloned <em>Prdm9</em> cDNAs by PCR from total RNAs prepared from the testes of 12 inbred strains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These comprised seven strains derived from wild-captured mice belonging to different subspecies: two congenic strains harbouring wild-derived <em>Prdm9</em> alleles, one inbred strain of the Japanese fancy mouse (JF1/Ms), and one inbred strain derived from a different species, <em>M. spretus.</em> The PCR products showed marked variation in size. For each sample, the major band was cloned and then sequenced using a capillary sequencer.</p><p>Comparison between these 12 sequences yielded a total of 28 nucleotide changes in the 1565-bp <em>Prdm9</em> ORF, excluding the ZFA (Fig. <a reveal-id="DST059F1" data-open="DST059F1" class="link link-reveal link-table xref-fig">1</a>). Of these, 19 cause amino acid changes. When compared with the C57BL/6 reference sequence (NCBI mm9) and our C57BL/10 sequence (this study), insertions and deletions (in/dels) without frame shifts were observed in two strains, CHD/Ms and B10.D2-TCH/+ (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). No difference was found in the PR/SET domains of the 12 strains examined. In contrast, the repeat numbers of ZF were largely variable among these strains, consistent with changes in the electrophoretic mobility of the PCR products (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>A). In addition, we found numerous nucleotide changes in ZFA. Almost all of these were associated with amino acid changes.</p>                            <h3 scrollto-destination=5821361 id="5821361" class="section-title" data-legacy-id=s3b>3.2. ZFA polymorphism of wild-captured mice</h3>
<p>We next extended our survey of <em>Prdm9</em> ZFA polymorphisms to wild-captured mice. The entire ZFA region is encoded by the last exon. We directly amplified this exon by PCR from genomic DNA from total of 79 wild mice collected at different locations (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The samples include populations belonging to three major <em>M. musculus</em> subspecies and a neighbouring species<em>.</em> To avoid artificial PCR products, we used high-fidelity DNA polymerase and highly stringent conditions. Moreover, PCR primer sets were designed for regions 200 bp apart from both ends of the ZFA to prevent mis-annealing of PCR intermediates. PCR products were separated on an agarose gel and the major band, which displays extensive variation in size among samples (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>B), was subjected to subcloning. When two major bands existed, the both bands were subcloned, as they are likely to represent heterozygous alleles. Reproducibility was confirmed by repeated independent PCR amplifications. More than six clones were sequenced for each sample with two different primer sets. If two types of ZFA sequences were reproducibly obtained, they were judged as heterozygous alleles.</p><p>We aligned all ZF repeats identified in this study (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>A). The first ZF repeat appears to be uniform, but the internal ZF repeats were highly variable. In particular, DNA recognition positions −1, 3, and 6 were highly variable (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>A), consistent with a previous report.<sup><a reveal-id="DST059C17" data-open="DST059C17" class="link link-ref link-reveal xref-bibr">17</a></sup> A lower level of variation was found at positions, −5, −2, and 1. The last repeat tends to be less variable than the internal repeats. In total, we identified 36 unique ZF repeats in ZFA. Of these, 24 are newly identified in this study (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>A). </p>                        <a id="5821365" scrollto-destination="5821365"></a>
<div id="DST059F2" data-id="DST059F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05902.png?Expires=2147483647&Signature=prD6Tm6ypf2JPvLBno2Znnux5BmV9WK74tgJ-UR5nv1eSBqB~zudFpNu~k9q0Lx~zQstlZCPJ4lfyJxzGmjPMz1IZsB9DqCVr61AMI2nzb9JTS~M2-RIkKU0JSFyKsLGc5SmT7~1rbKRJKYijGr5siyH5gmjTvvvwTYD0z3jTiT6g4jQ3ErB9t0XU9X5JtEudYTEJaUIaJEqdqfazNNqA4~upZKgGPYzLkj7SLk0AfVJ4VQ85dlWj6H9k9oxs3UWrWAUs~sOuTe2QgvxuVATCLLzWVpOGUpjQH~LSxytidllSOKylqdJXz8BmFdpsmUHf8m5G7jyO1E7M~v18UX-gA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amino acid variation in Prdm9 ZF repeats. (A) Multiple alignment of ZF repeat sequences from inbred strains and wild-captured mice. The alignments were made separately for the first repeat, the internal repeats of the ZFA, and the last repeat, as their degrees of variation are different (see text). The α-helix domain of the ZF is shown at top of the internal repeat. The one-letter code is shown to the left of the repeats. A novel repeat found in this study is indicated by # to the right of the repeat. Amino acid variation at specific positions along aligned ZF repeats are shown in colour. The star on the left side in the first repeat corresponds to those shown in Fig. 3. (B) Multiple alignment of ZF repeats of human PRDM9 based on the publically available data (http://www.ncbi.nlm.nih.gov/nuccore). The format of the alignment is the same as that used for mouse Prdm9." path-from-xml="dst05902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821365/dst05902.tiff" path-from-xml="dst05902.tiff" target="_blank">View large</a><a section="5821365" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05902.png?Expires=2147483647&Signature=jUuLHHQT-Tc3x0xuHvxwkYx~EMsaPNfJIHfqE1KT6wZPYfFt-G8keh4OzwBfgLXWOzQP-5sXAiOLPN~K~i5v3k7TzWJLKJ4iypVn~~Xw44kncmwYvGpuK3GMPr7xxGEMgtMet-GZ3JL4Ni2FKPZGz-8J1~lQK2lmysO2Ah0z4F2qLmCHfhrCBv8hUSOF7OWpdHZjLnHNMU~G8PQPtUGVF5XiCTVoDyloSLy4KeVBdAcXoN~26B0e4RhQI-MlDXQnLq~ajXMAKXBK7tap1QC30-05hT2wLG~PiPrGXNpBIDfxmCCbr8txuVgd3wCVXGK~XOm41DKX-gmoIsYdpdoSAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821365&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amino acid variation in <em>Prdm9</em> ZF repeats. (A) Multiple alignment of ZF repeat sequences from inbred strains and wild-captured mice. The alignments were made separately for the first repeat, the internal repeats of the ZFA, and the last repeat, as their degrees of variation are different (see text). The α-helix domain of the ZF is shown at top of the internal repeat. The one-letter code is shown to the left of the repeats. A novel repeat found in this study is indicated by # to the right of the repeat. Amino acid variation at specific positions along aligned ZF repeats are shown in colour. The star on the left side in the first repeat corresponds to those shown in Fig. <a reveal-id="DST059F3" data-open="DST059F3" class="link link-reveal link-table xref-fig">3</a>. (B) Multiple alignment of ZF repeats of human PRDM9 based on the publically available data (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/nuccore">http://www.ncbi.nlm.nih.gov/nuccore</a>). The format of the alignment is the same as that used for mouse Prdm9.</p></div></div><div content-id="DST059F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05902.png?Expires=2147483647&Signature=prD6Tm6ypf2JPvLBno2Znnux5BmV9WK74tgJ-UR5nv1eSBqB~zudFpNu~k9q0Lx~zQstlZCPJ4lfyJxzGmjPMz1IZsB9DqCVr61AMI2nzb9JTS~M2-RIkKU0JSFyKsLGc5SmT7~1rbKRJKYijGr5siyH5gmjTvvvwTYD0z3jTiT6g4jQ3ErB9t0XU9X5JtEudYTEJaUIaJEqdqfazNNqA4~upZKgGPYzLkj7SLk0AfVJ4VQ85dlWj6H9k9oxs3UWrWAUs~sOuTe2QgvxuVATCLLzWVpOGUpjQH~LSxytidllSOKylqdJXz8BmFdpsmUHf8m5G7jyO1E7M~v18UX-gA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amino acid variation in Prdm9 ZF repeats. (A) Multiple alignment of ZF repeat sequences from inbred strains and wild-captured mice. The alignments were made separately for the first repeat, the internal repeats of the ZFA, and the last repeat, as their degrees of variation are different (see text). The α-helix domain of the ZF is shown at top of the internal repeat. The one-letter code is shown to the left of the repeats. A novel repeat found in this study is indicated by # to the right of the repeat. Amino acid variation at specific positions along aligned ZF repeats are shown in colour. The star on the left side in the first repeat corresponds to those shown in Fig. 3. (B) Multiple alignment of ZF repeats of human PRDM9 based on the publically available data (http://www.ncbi.nlm.nih.gov/nuccore). The format of the alignment is the same as that used for mouse Prdm9." path-from-xml="dst05902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821365/dst05902.tiff" path-from-xml="dst05902.tiff" target="_blank">View large</a><a section="5821365" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05902.png?Expires=2147483647&Signature=jUuLHHQT-Tc3x0xuHvxwkYx~EMsaPNfJIHfqE1KT6wZPYfFt-G8keh4OzwBfgLXWOzQP-5sXAiOLPN~K~i5v3k7TzWJLKJ4iypVn~~Xw44kncmwYvGpuK3GMPr7xxGEMgtMet-GZ3JL4Ni2FKPZGz-8J1~lQK2lmysO2Ah0z4F2qLmCHfhrCBv8hUSOF7OWpdHZjLnHNMU~G8PQPtUGVF5XiCTVoDyloSLy4KeVBdAcXoN~26B0e4RhQI-MlDXQnLq~ajXMAKXBK7tap1QC30-05hT2wLG~PiPrGXNpBIDfxmCCbr8txuVgd3wCVXGK~XOm41DKX-gmoIsYdpdoSAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821365&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amino acid variation in <em>Prdm9</em> ZF repeats. (A) Multiple alignment of ZF repeat sequences from inbred strains and wild-captured mice. The alignments were made separately for the first repeat, the internal repeats of the ZFA, and the last repeat, as their degrees of variation are different (see text). The α-helix domain of the ZF is shown at top of the internal repeat. The one-letter code is shown to the left of the repeats. A novel repeat found in this study is indicated by # to the right of the repeat. Amino acid variation at specific positions along aligned ZF repeats are shown in colour. The star on the left side in the first repeat corresponds to those shown in Fig. <a reveal-id="DST059F3" data-open="DST059F3" class="link link-reveal link-table xref-fig">3</a>. (B) Multiple alignment of ZF repeats of human PRDM9 based on the publically available data (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/nuccore">http://www.ncbi.nlm.nih.gov/nuccore</a>). The format of the alignment is the same as that used for mouse Prdm9.</p></div></div><p>We further compared variation in the mouse ZF repeats with those in human PRDM9, as obtained from a public database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/nuccore">http://www.ncbi.nlm.nih.gov/nuccore</a>) (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>B). Amino acids in the backbone of C<sub>2</sub>H<sub>2</sub>-class ZFs are well conserved between the two species; however, some species-specific amino acid variations are found. For example, the amino acid at position −2 of the α-helix domain is mostly tyrosine (T) in the mouse, but serine (S) or arginine (R) in humans. The amino acid at position 5 is mostly isoleucine (I) in mouse, but exclusively leucine (L) in human. These positions reside in the α-helix domain, close to the most variable DNA recognition positions, i.e. positions −1 and 6. Amino acid variation is also found within each species. It is conceivable that this variation results in changes to DNA recognition patterns.</p>                            <h3 scrollto-destination=5821376 id="5821376" class="section-title" data-legacy-id=s3c>3.3. Phylogeography of polymorphisms in the ZFA of <em>Prdm9</em> in natural populations</h3>
<p>To simplify annotation of <em>Prdm9</em> ZFAs in subsequent analyses, we used a one-letter code to identify each ZF repeat depending on the amino acid triads present at the most variable three positions, −1, 3, and 6, as well as additional alterations (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>A) (see Materials and Methods for detail). Figure <a reveal-id="DST059F3" data-open="DST059F3" class="link link-reveal link-table xref-fig">3</a> presents an alignment of all ZFA diagrams of wild-captured mice, inbred strains derived from wild mice, and commonly used laboratory inbred strains. The data indicate that wild-captured mice have ZFAs of various lengths. The ZF repeat numbers were between 9 and 16. We could identify as many as 57 ZFA variants within a single species, <em>M. musculus.</em> In addition, we found that neighbouring species of <em>M. musculus</em> have ZFA variant types different from those of <em>M. musculus</em>. </p>                        <a id="5821387" scrollto-destination="5821387"></a>
<div id="DST059F3" data-id="DST059F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05903.png?Expires=2147483647&Signature=0f9WIBGTrKw5hfpS2gqUP5Y9SQzmcPz-JkPBu8Myy41W7le3Srfo-mADKEc3ICb3uQvQZ~Rfj9b67EOC4CHoWK8opkCdQiCij1YV~I~12J9n10Rrmf1jgP5zqrBGFIO6EjY~t267XpARr0CKDyRL-n5JnyLRKZdRQwfFkvaVZY2zilLrzkALw4-z6OgOhY5gNsoaIT5lrKyHEnLNBdEJf0n~5FUsQad6OfWqfm8QMhUokHHoUbUIWYldxXgwvUA48AkiF6JPnWhRdcwZymC4q43QcGJtVgbjjGMWbrLY4IZWhIONWqsdi3sCmkagnKzDOKqRNDsuTS0YNNnn13Y72Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ZFA alignment for inbred and wild-captured mice. We first assigned a one-letter coding system to ZFAs (see text), then aligned the ZFA diagrams based on the sequences of classical laboratory strains, wild-derived inbred strains, wild-captured mice, neighbouring species of M. musculus, and t-haplotype bearing mice. Using this code, we easily classified all variant types of ZFA. ZFA boxes are aligned from the N-terminal (left of the diagram) to the C-terminal end of the protein (right of the diagram). The diagrams are categorized into DOM- and CAS-related ZFA variant types (left block), MUS-related ZFA variant type (centre block), and other ZFA types of neighbouring species and t-haplotype (right block). The identification (ID) code for each ZFA diagram is indicated onto the left of the diagram. The names of inbred strains and taxonomy are indicated to the left of the ID codes." path-from-xml="dst05903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821387/dst05903.tiff" path-from-xml="dst05903.tiff" target="_blank">View large</a><a section="5821387" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05903.png?Expires=2147483647&Signature=GevdRaLiuU9ppx46FQkl7XyrKeyvIshir8mZGPgHMDgFGjAIkIvq4oWvRidfVy1hc09~YOcpY5WBnttA1DY5ASjJ5zSHAezpMio5b06rlAHEP5Jz6Sb0gQut-tdwUbs~7PSAOjcz2in9P41b2hAztdHu82NRdnVwJeE1KThufo0BCnhN7XdOR6u2hLVHomPBp7WClhu0nb0rUAtFTzGVADyhHm328SRXN~5wZHTOxxtQ0f1gjiNPq29jLJtBE0-sTVDhcfDvtFFxNaF55enVzwiXLfaQ-u7~dkX0cGN-B-SRZlTDVSIRNCnUs4Y7XPnmwPML3kj4bgInygOebzyxkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821387&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ZFA alignment for inbred and wild-captured mice. We first assigned a one-letter coding system to ZFAs (see text), then aligned the ZFA diagrams based on the sequences of classical laboratory strains, wild-derived inbred strains, wild-captured mice, neighbouring species of <em>M. musculus</em>, and <em>t-</em>haplotype bearing mice. Using this code, we easily classified all variant types of ZFA. ZFA boxes are aligned from the N-terminal (left of the diagram) to the C-terminal end of the protein (right of the diagram). The diagrams are categorized into DOM- and CAS-related ZFA variant types (left block), MUS-related ZFA variant type (centre block), and other ZFA types of neighbouring species and <em>t</em>-haplotype (right block). The identification (ID) code for each ZFA diagram is indicated onto the left of the diagram. The names of inbred strains and taxonomy are indicated to the left of the ID codes.</p></div></div><div content-id="DST059F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05903.png?Expires=2147483647&Signature=0f9WIBGTrKw5hfpS2gqUP5Y9SQzmcPz-JkPBu8Myy41W7le3Srfo-mADKEc3ICb3uQvQZ~Rfj9b67EOC4CHoWK8opkCdQiCij1YV~I~12J9n10Rrmf1jgP5zqrBGFIO6EjY~t267XpARr0CKDyRL-n5JnyLRKZdRQwfFkvaVZY2zilLrzkALw4-z6OgOhY5gNsoaIT5lrKyHEnLNBdEJf0n~5FUsQad6OfWqfm8QMhUokHHoUbUIWYldxXgwvUA48AkiF6JPnWhRdcwZymC4q43QcGJtVgbjjGMWbrLY4IZWhIONWqsdi3sCmkagnKzDOKqRNDsuTS0YNNnn13Y72Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ZFA alignment for inbred and wild-captured mice. We first assigned a one-letter coding system to ZFAs (see text), then aligned the ZFA diagrams based on the sequences of classical laboratory strains, wild-derived inbred strains, wild-captured mice, neighbouring species of M. musculus, and t-haplotype bearing mice. Using this code, we easily classified all variant types of ZFA. ZFA boxes are aligned from the N-terminal (left of the diagram) to the C-terminal end of the protein (right of the diagram). The diagrams are categorized into DOM- and CAS-related ZFA variant types (left block), MUS-related ZFA variant type (centre block), and other ZFA types of neighbouring species and t-haplotype (right block). The identification (ID) code for each ZFA diagram is indicated onto the left of the diagram. The names of inbred strains and taxonomy are indicated to the left of the ID codes." path-from-xml="dst05903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821387/dst05903.tiff" path-from-xml="dst05903.tiff" target="_blank">View large</a><a section="5821387" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05903.png?Expires=2147483647&Signature=GevdRaLiuU9ppx46FQkl7XyrKeyvIshir8mZGPgHMDgFGjAIkIvq4oWvRidfVy1hc09~YOcpY5WBnttA1DY5ASjJ5zSHAezpMio5b06rlAHEP5Jz6Sb0gQut-tdwUbs~7PSAOjcz2in9P41b2hAztdHu82NRdnVwJeE1KThufo0BCnhN7XdOR6u2hLVHomPBp7WClhu0nb0rUAtFTzGVADyhHm328SRXN~5wZHTOxxtQ0f1gjiNPq29jLJtBE0-sTVDhcfDvtFFxNaF55enVzwiXLfaQ-u7~dkX0cGN-B-SRZlTDVSIRNCnUs4Y7XPnmwPML3kj4bgInygOebzyxkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821387&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ZFA alignment for inbred and wild-captured mice. We first assigned a one-letter coding system to ZFAs (see text), then aligned the ZFA diagrams based on the sequences of classical laboratory strains, wild-derived inbred strains, wild-captured mice, neighbouring species of <em>M. musculus</em>, and <em>t-</em>haplotype bearing mice. Using this code, we easily classified all variant types of ZFA. ZFA boxes are aligned from the N-terminal (left of the diagram) to the C-terminal end of the protein (right of the diagram). The diagrams are categorized into DOM- and CAS-related ZFA variant types (left block), MUS-related ZFA variant type (centre block), and other ZFA types of neighbouring species and <em>t</em>-haplotype (right block). The identification (ID) code for each ZFA diagram is indicated onto the left of the diagram. The names of inbred strains and taxonomy are indicated to the left of the ID codes.</p></div></div><p>To elucidate the phylogenetic relationships among ZFA variants in <em>M. musculus</em>, multiple alignment of the ZFA was performed using the one-letter code. After alignment, the phylogenetic tree was constructed using the neighbour-joining method.<sup><a reveal-id="DST059C29" data-open="DST059C29" class="link link-ref link-reveal xref-bibr">29</a></sup> This phylogenetic analysis revealed that the ZFA variants could be divided into five major groups (Fig. <a reveal-id="DST059F4" data-open="DST059F4" class="link link-reveal link-table xref-fig">4</a>A). The localities where the wild mice were collected are then plotted on the territories of three major subspecies of <em>M. musculus</em> (Fig. <a reveal-id="DST059F4" data-open="DST059F4" class="link link-reveal link-table xref-fig">4</a>B). </p>                        <a id="5821398" scrollto-destination="5821398"></a>
<div id="DST059F4" data-id="DST059F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05904.png?Expires=2147483647&Signature=w8XpoX6NtJM5cahLIcNcMswFYtvqW2EiuEOaMv8OEHrCtiUl0kP8YHjHZcKf03fweqEgTm0YwIWdz77IKPGKAk95NCQQDzjBSu6zfe8uGwXiaiEsgZTcXZTFT22RbSAaglQsq0CjHwOVifwHSgRuK1QxvOa9Oi7bXHU2~~ppMC21NALrUFYYD9phlwCPBzESrJ7FRTku6sR01ykzBZozyinB29ZEG15vU8OgmKnLwFWGIa7uy307ondRmUR3dOWm871FYCHRE~buRHDURIib4V0zmSTvkzjIOIpPqfkEQMdTLgw9X4Yr2wETJ5zB4Mbe9WhE9RxxarZXOm5ZxIM5Fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny and phylogeography of Prdm9 ZFAs. (A) Phylogenic tree constructed from 57 Prdm9 ZFAs. The identification codes of ZFA variant types are labelled according to the ZFA diagrams (Fig. 3). The 57 ZFA variant types can be divided into five major groups. The names of inbred strains carrying a given ZFA variant type are shown. (B) Points of collection of wild-captured mice are shown on a world map. Background colours indicate the ranges (territories) of each M. musculus subspecies as follows. Blue, DOM; green, MUS; purple, CAS. The red line in Europe indicates a DOM–MUS hybrid zone, where the two subspecies come into contact. Dashed lines indicate borders between other subspecies. For some species, the border regions are not clear. Japanese wild mice, M. m. molossinus, are a hybrid of two subspecies, MUS and CAS." path-from-xml="dst05904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821398/dst05904.tiff" path-from-xml="dst05904.tiff" target="_blank">View large</a><a section="5821398" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05904.png?Expires=2147483647&Signature=SEatr-a-26O8Dr8-HCPqCz1Yb3Jfcnre8JDJh2zOzBqSXkS37PqvhN3zfllXrGnPY0N~gw3oA~IZiCZnh4WyJ4GTmp8~cQPPZ8HIqhj-R443PpY~6u7Hekxt6pUmfHbiuJH7PjrJ2NWrd8qzNKOXOz875hkA1y~kl6fQM8763RRpE7fH3UJz7Doqak8Fgit68SIbVuFTatUjQtZqHNoyNLP4w9QEyp5Mffr63QISZtdEqc6EdOSYVYoGGSLSulLkH9YgxxYmpbenyewvEBWbbh5ufuMxpUPxuMJM2jRgQDXcYjwAKax1cgiNpOhc-rpZXxdtUUvdQKYYpvW7iHWYaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821398&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny and phylogeography of <em>Prdm9</em> ZFAs. (A) Phylogenic tree constructed from 57 <em>Prdm9</em> ZFAs. The identification codes of ZFA variant types are labelled according to the ZFA diagrams (Fig. <a reveal-id="DST059F3" data-open="DST059F3" class="link link-reveal link-table xref-fig">3</a>). The 57 ZFA variant types can be divided into five major groups. The names of inbred strains carrying a given ZFA variant type are shown. (B) Points of collection of wild-captured mice are shown on a world map. Background colours indicate the ranges (territories) of each <em>M. musculus</em> subspecies as follows. Blue, DOM; green, MUS; purple, CAS. The red line in Europe indicates a DOM–MUS hybrid zone, where the two subspecies come into contact. Dashed lines indicate borders between other subspecies. For some species, the border regions are not clear. Japanese wild mice, <em>M. m. molossinus</em>, are a hybrid of two subspecies, MUS and CAS.</p></div></div><div content-id="DST059F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05904.png?Expires=2147483647&Signature=w8XpoX6NtJM5cahLIcNcMswFYtvqW2EiuEOaMv8OEHrCtiUl0kP8YHjHZcKf03fweqEgTm0YwIWdz77IKPGKAk95NCQQDzjBSu6zfe8uGwXiaiEsgZTcXZTFT22RbSAaglQsq0CjHwOVifwHSgRuK1QxvOa9Oi7bXHU2~~ppMC21NALrUFYYD9phlwCPBzESrJ7FRTku6sR01ykzBZozyinB29ZEG15vU8OgmKnLwFWGIa7uy307ondRmUR3dOWm871FYCHRE~buRHDURIib4V0zmSTvkzjIOIpPqfkEQMdTLgw9X4Yr2wETJ5zB4Mbe9WhE9RxxarZXOm5ZxIM5Fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny and phylogeography of Prdm9 ZFAs. (A) Phylogenic tree constructed from 57 Prdm9 ZFAs. The identification codes of ZFA variant types are labelled according to the ZFA diagrams (Fig. 3). The 57 ZFA variant types can be divided into five major groups. The names of inbred strains carrying a given ZFA variant type are shown. (B) Points of collection of wild-captured mice are shown on a world map. Background colours indicate the ranges (territories) of each M. musculus subspecies as follows. Blue, DOM; green, MUS; purple, CAS. The red line in Europe indicates a DOM–MUS hybrid zone, where the two subspecies come into contact. Dashed lines indicate borders between other subspecies. For some species, the border regions are not clear. Japanese wild mice, M. m. molossinus, are a hybrid of two subspecies, MUS and CAS." path-from-xml="dst05904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821398/dst05904.tiff" path-from-xml="dst05904.tiff" target="_blank">View large</a><a section="5821398" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05904.png?Expires=2147483647&Signature=SEatr-a-26O8Dr8-HCPqCz1Yb3Jfcnre8JDJh2zOzBqSXkS37PqvhN3zfllXrGnPY0N~gw3oA~IZiCZnh4WyJ4GTmp8~cQPPZ8HIqhj-R443PpY~6u7Hekxt6pUmfHbiuJH7PjrJ2NWrd8qzNKOXOz875hkA1y~kl6fQM8763RRpE7fH3UJz7Doqak8Fgit68SIbVuFTatUjQtZqHNoyNLP4w9QEyp5Mffr63QISZtdEqc6EdOSYVYoGGSLSulLkH9YgxxYmpbenyewvEBWbbh5ufuMxpUPxuMJM2jRgQDXcYjwAKax1cgiNpOhc-rpZXxdtUUvdQKYYpvW7iHWYaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821398&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny and phylogeography of <em>Prdm9</em> ZFAs. (A) Phylogenic tree constructed from 57 <em>Prdm9</em> ZFAs. The identification codes of ZFA variant types are labelled according to the ZFA diagrams (Fig. <a reveal-id="DST059F3" data-open="DST059F3" class="link link-reveal link-table xref-fig">3</a>). The 57 ZFA variant types can be divided into five major groups. The names of inbred strains carrying a given ZFA variant type are shown. (B) Points of collection of wild-captured mice are shown on a world map. Background colours indicate the ranges (territories) of each <em>M. musculus</em> subspecies as follows. Blue, DOM; green, MUS; purple, CAS. The red line in Europe indicates a DOM–MUS hybrid zone, where the two subspecies come into contact. Dashed lines indicate borders between other subspecies. For some species, the border regions are not clear. Japanese wild mice, <em>M. m. molossinus</em>, are a hybrid of two subspecies, MUS and CAS.</p></div></div><p>Group 1 exclusively includes mice collected in the territory of subspecies <em>M. m. musculus</em> (hereafter, MUS), which extends on the North Eurasian continent from Eastern Europe to the Far East. Groups 2, 4, and 5 mainly include mice collected in the territory of subspecies <em>M. m. castaneus</em> (CAS), which includes a wide range, from Southwestern Asia to India, and extends to Southeast Asia, South China, Indonesia, and the southern and northern edges of Japanese islands. Group 3 includes mice collected in the territory of subspecies <em>M. m. domesticus</em> (DOM), which extends from western and southern Europe to the Middle East and the shores around the Mediterranean Sea. Group 3 territory also includes the New World, North and South America, coincident with human migration. This group also includes commonly used classical inbred strains, which are overwhelmingly derived from the DOM lineage.<sup><a reveal-id="DST059C31" data-open="DST059C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Groups 2 and 5 also include a small number of mice collected in the MUS and DOM territories, respectively. Likewise, Group 3 includes one mouse collected in the CAS territory. In the Japanese population, 12 of 13 mice (including two inbred strains) showed only a single ZFA variant type (Ma5), classified as Group 1. The remaining strain has the variant type of Group 4 (Ca2).</p><p>Altogether, although <em>M. musculus</em> as a whole holds huge variations in ZFA, the mice collected in the MUS and DOM territories exhibit mutually exclusive ZFA variant patterns (Group 1 for MUS and Group 3 for DOM, respectively). The existence of such predominant variant types is not seen in the case of the mice collected in the CAS territory<em>.</em> In this region, ZFA variant types can be further divided into at least the three major groups.</p><p>We calculated the frequency of wild-captured mice heterozygous for ZFA variant types in different territories of subspecies (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). The value for the total <em>M. musculus</em> population is 35% (26/74); for DOM, 10% (2/20); for MUS, 44% (11/25); for CAS, 50% (12/24); and for Japanese wild mice (<em>M. m. molossinus</em>), 10% (1/10)<em>.</em> These results are consistent with the degree of ZFA diversity observed in the three subspecies populations.</p>                            <h3 scrollto-destination=5821430 id="5821430" class="section-title" data-legacy-id=s3d>3.4. Phylogeny of <em>Prdm9</em> ZFA in <em>t</em>-haplotype mice</h3>
<p>B10.D2-TCH/+ is a heterozygous MHC haplotype mouse stock that harbours a recessive lethal mutation. In this stock, the wild-derived MHC haplotype is transmitted to the next generation at a highly distorted ratio (unpublished data). Therefore, we inferred that B10.D2-TCH/+ has <em>t-</em>haplotype (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). <em>Prdm9</em> cDNA from this strain did not have polymorphisms observed for other strains in the PR/SET domain; however, unique substitutions were found outside the ZFA, as for <em>M. spretus</em>, in addition to a DOM-type sequence. The result suggests that this stock is heterozygous for DOM-type and more divergent <em>Prdm9</em> alleles (Fig. <a reveal-id="DST059F1" data-open="DST059F1" class="link link-reveal link-table xref-fig">1</a>). Its ZFA has 11 repeats of ZF with a rare type of the amino acid triad. To analyse other strains carrying the <em>t</em>-haplotype, we amplified the ZFA from seven <em>t-</em>haplotype samples, <em>t<sup>w5</sup></em>/+, <em>t<sup>w71</sup></em>/+, <em>t<sup>w75</sup></em>/+, <em>t<sup>12</sup>/+</em>, <em>t<sup>w12</sup>/+</em>, <em>t<sup>0</sup>/+</em>, and <em>t<sup>w2</sup>/ t<sup>w2</sup></em>. With the exception of for <em>t<sup>w2</sup></em>, all samples were heterozygous for a DOM-type chromosome. Sequencing of ZFA from all strains showed a rare variant type in addition to a DOM-type ZFA. Notably, the sequence of the rare variant is identical among samples carrying the <em>t-h</em>aplotype and B10.D2-TCH/+. Phylogenetic analysis showed that the ZFA sequence associated with the <em>t</em>-haplotype is more divergent from those of the five major groups in <em>M. musculus</em> (Figs <a reveal-id="DST059F3" data-open="DST059F3" class="link link-reveal link-table xref-fig">3</a> and <a reveal-id="DST059F4" data-open="DST059F4" class="link link-reveal link-table xref-fig">4</a>A).</p><p>We compared <em>Prdm9</em> genome sequences among three inbred strains, C57BL/6J, CAST/EiJ, and PWK/PhJ, which are derived from three subspecies, DOM, CAS, and MUS, respectively. We found nucleotide sequence polymorphism in a 750-bp intronic region of <em>Prdm9</em> residing in the interval between two exons that encode the PR/SET domain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>)<em>.</em> We named this HSR1 (Fig. <a reveal-id="DST059F5" data-open="DST059F5" class="link link-reveal link-table xref-fig">5</a>A). To clarify the phylogeny of regions outside the <em>Prdm9</em> ZFA, we sequenced HSR1 from different subspecies of wild-captured <em>M. musculus</em> and neighbouring species. We also sequenced a 2.38-kb region that includes introns 8–11 of <em>Tcp1</em>, a <em>t-</em>haplotype marker,<sup><a reveal-id="DST059C26" data-open="DST059C26" class="link link-ref link-reveal xref-bibr">26</a></sup> to analyse the phylogeny of a gene linked to <em>Prdm9</em>. We constructed phylogenetic trees from these two sequences, HSR1 and <em>Tcp1</em>, using the neighbour-joining method.<sup><a reveal-id="DST059C29" data-open="DST059C29" class="link link-ref link-reveal xref-bibr">29</a></sup> For tree construction, <em>M. caroli</em>, which is more divergent from <em>M. musculus</em> than other neighbouring species,<sup><a reveal-id="DST059C20" data-open="DST059C20" class="link link-ref link-reveal xref-bibr">20</a></sup> was used as an outgroup<em>.</em> The overall topology of the <em>Tcp1</em> tree is similar to the tree based on <em>Prdm9</em> ZFA (Fig. <a reveal-id="DST059F5" data-open="DST059F5" class="link link-reveal link-table xref-fig">5</a>C). In particular, <em>Tcp1</em> in the <em>t</em>-haplotype has diverged from those in the wild-type chromosome, consistent with a previous report.<sup><a reveal-id="DST059C26" data-open="DST059C26" class="link link-ref link-reveal xref-bibr">26</a></sup> However, the sequence is similar to that found in some mice from the CAS territory. In contrast, the phylogenetic tree of the HSR1 sequence shows that <em>Prdm9</em> in the <em>t-</em>haplotype is similar to that found in mice from the CAS and DOM territories, but distant from those of mice from the MUS territory (Fig. <a reveal-id="DST059F5" data-open="DST059F5" class="link link-reveal link-table xref-fig">5</a>B). These results indicate that the single <em>Prdm9</em> allele in <em>t</em>-haplotype introgressed into all subspecies of <em>M. musculus</em>, and suggest that intragenic recombination somewhere in the interval between ZFA and HSR1 in the <em>Prdm9</em> gene occurred in the past. </p>                        <a id="5821447" scrollto-destination="5821447"></a>
<div id="DST059F5" data-id="DST059F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05905.png?Expires=2147483647&Signature=H6Uk10CF0XACmNzaIZZiLOL32tgY80EfqLjLNVj2JPexRlx81e30rmqJwMarxnh~jHnmeXe2M~OUOhfi7WqUI6lYrIKTyzZfgJNou2orGg84i8TkObkxYMXVdFMx5QTcvJ7sTxBgXSYd0gYDBQe0nU-c~qAg~LCCN2TIRE0SyWMNEsIf49T9B2KNGAFRxTOOZhKfoTgXPAo9MTSsVqNXlHdHliRx-e9ygKaqSOF1RjZEtyBSJXcoJcSdvKt1Kk1rVKLhzYCZTKFqaBchjrK-CwY8nm~KzpB2ooOLrEG6Le6yZMaKT3TOXO3b9XWRd9aGUPG2kuGecWaBaX7kikW~yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny of the Prdm9 intronic sequence (HSR1) and Tcp1. (A) Upper diagram, map of the chromosomal region containing Tcp1 and Prdm9. Lower diagram, view of the exon–intron organization of Prdm9. HSR1 is located in an intron between two exons that encode the PR/SET domain. (B) Phylogenetic tree of Prdm9 HSR1. (C) Phylogenetic tree of Tcp1 sequence in introns 8–11. M. caroli was used as the outgroup for both trees. An asterisk (*) to the right of a strain name (ZFA variant) indicates that the samples are heterozygous for Prdm9 HSR1 (B) and Tcp1 intronic (C) sequences." path-from-xml="dst05905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821447/dst05905.tiff" path-from-xml="dst05905.tiff" target="_blank">View large</a><a section="5821447" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05905.png?Expires=2147483647&Signature=Saosn17eEtR4G07aDNq3nOOTdyokLJKlVh~0H06avlvPhyyA55pM0zs002abp-taYLhyY45zlFvP4yrPvW-oE2KukAXjFoQzt3Z7NA1D~ISuj0bXMg9Scgc4MBb48Rq0ArxAlMAJ7eOo-47OS9eLZy4aiguFpKryltbFkmDaM-WQSxxeNwmkax2rLLh5VezuCzAnHcf0KCDHoSWZ66cMPBGObxAumsfa2a7~ix4RH2gydwdpvCqCfaPXjeAWA8RblObb4G3DTKqeP~XbDG2JmlwY5wnTPEiVuh2tpe~p4CDL5xAGOsRqRaDbhILlhFa~9EGhAytavbXVY5CwH9kO0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821447&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny of the <em>Prdm9</em> intronic sequence (HSR1) and <em>Tcp1.</em> (A) Upper diagram, map of the chromosomal region containing <em>Tcp1</em> and <em>Prdm9</em>. Lower diagram, view of the exon–intron organization of <em>Prdm9.</em> HSR1 is located in an intron between two exons that encode the PR/SET domain. (B) Phylogenetic tree of <em>Prdm9</em> HSR1. (C) Phylogenetic tree of <em>Tcp1</em> sequence in introns 8–11. <em>M. caroli</em> was used as the outgroup for both trees. An asterisk (*) to the right of a strain name (ZFA variant) indicates that the samples are heterozygous for <em>Prdm9</em> HSR1 (B) and <em>Tcp1</em> intronic (C) sequences.</p></div></div><div content-id="DST059F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/m_dst05905.png?Expires=2147483647&Signature=H6Uk10CF0XACmNzaIZZiLOL32tgY80EfqLjLNVj2JPexRlx81e30rmqJwMarxnh~jHnmeXe2M~OUOhfi7WqUI6lYrIKTyzZfgJNou2orGg84i8TkObkxYMXVdFMx5QTcvJ7sTxBgXSYd0gYDBQe0nU-c~qAg~LCCN2TIRE0SyWMNEsIf49T9B2KNGAFRxTOOZhKfoTgXPAo9MTSsVqNXlHdHliRx-e9ygKaqSOF1RjZEtyBSJXcoJcSdvKt1Kk1rVKLhzYCZTKFqaBchjrK-CwY8nm~KzpB2ooOLrEG6Le6yZMaKT3TOXO3b9XWRd9aGUPG2kuGecWaBaX7kikW~yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny of the Prdm9 intronic sequence (HSR1) and Tcp1. (A) Upper diagram, map of the chromosomal region containing Tcp1 and Prdm9. Lower diagram, view of the exon–intron organization of Prdm9. HSR1 is located in an intron between two exons that encode the PR/SET domain. (B) Phylogenetic tree of Prdm9 HSR1. (C) Phylogenetic tree of Tcp1 sequence in introns 8–11. M. caroli was used as the outgroup for both trees. An asterisk (*) to the right of a strain name (ZFA variant) indicates that the samples are heterozygous for Prdm9 HSR1 (B) and Tcp1 intronic (C) sequences." path-from-xml="dst05905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5821447/dst05905.tiff" path-from-xml="dst05905.tiff" target="_blank">View large</a><a section="5821447" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst05905.png?Expires=2147483647&Signature=Saosn17eEtR4G07aDNq3nOOTdyokLJKlVh~0H06avlvPhyyA55pM0zs002abp-taYLhyY45zlFvP4yrPvW-oE2KukAXjFoQzt3Z7NA1D~ISuj0bXMg9Scgc4MBb48Rq0ArxAlMAJ7eOo-47OS9eLZy4aiguFpKryltbFkmDaM-WQSxxeNwmkax2rLLh5VezuCzAnHcf0KCDHoSWZ66cMPBGObxAumsfa2a7~ix4RH2gydwdpvCqCfaPXjeAWA8RblObb4G3DTKqeP~XbDG2JmlwY5wnTPEiVuh2tpe~p4CDL5xAGOsRqRaDbhILlhFa~9EGhAytavbXVY5CwH9kO0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5821447&ar=394421&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny of the <em>Prdm9</em> intronic sequence (HSR1) and <em>Tcp1.</em> (A) Upper diagram, map of the chromosomal region containing <em>Tcp1</em> and <em>Prdm9</em>. Lower diagram, view of the exon–intron organization of <em>Prdm9.</em> HSR1 is located in an intron between two exons that encode the PR/SET domain. (B) Phylogenetic tree of <em>Prdm9</em> HSR1. (C) Phylogenetic tree of <em>Tcp1</em> sequence in introns 8–11. <em>M. caroli</em> was used as the outgroup for both trees. An asterisk (*) to the right of a strain name (ZFA variant) indicates that the samples are heterozygous for <em>Prdm9</em> HSR1 (B) and <em>Tcp1</em> intronic (C) sequences.</p></div></div>                            <h2 scrollto-destination=5821455 id="5821455" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The results of this study support that the mouse <em>Prdm9</em> polymorphism is highly polymorphic. Indeed, the degree of the polymorphism of <em>Prdm9</em> is comparable with that of MHC, which may be maintained by selective advantage.<sup><a reveal-id="DST059C32" data-open="DST059C32" class="link link-ref link-reveal xref-bibr">32</a></sup><em>Prdm9</em> polymorphisms converged in the ZF repeats and in amino acid positions at −1, 3, and 6 of the α-helix domain, which are extremely variable and are thought to recognize DNA sequences. A lower level of variation was found for amino acid positions −2 and 1 in both mouse and human ZF repeats. We infer that all these positions have been subjected to positive selection to increase the diversity of <em>Prdm9</em> polymorphism,<sup><a reveal-id="DST059C17" data-open="DST059C17" class="link link-ref link-reveal xref-bibr">17</a></sup> as non-synonymous substitutions preferentially occur at positions −2 and 1, both in mice and in humans. In humans, 19 unique ZF repeats have been identified by extensive analysis of genome sequences from various human populations.<sup><a reveal-id="DST059C5" data-open="DST059C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST059C7" data-open="DST059C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST059C10" data-open="DST059C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST059C17" data-open="DST059C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST059C33" data-open="DST059C33" class="link link-ref link-reveal xref-bibr">33</a></sup> This number is lower than that in mouse (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>B). It is likely explained by the shorter time of divergence between different human populations, in contrast to a longer period of divergence between different mouse subspecies (i.e. 0.5–1.0 million years).<sup><a reveal-id="DST059C20" data-open="DST059C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p><p>The first and last ZF repeats showed a lower level of variation when compared with internal repeats. In the first repeat, the first cysteine, which is involved in tertiary structure of the C<sub>2</sub>H<sub>2</sub>-class ZF, has been lost. As a consequence, it may not function as authentic ZF (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>A and B). For the last repeat, the C<sub>2</sub>H<sub>2</sub>-class ZF is conserved and amino acids at positions 3 and 6 are variable in mice (Fig. <a reveal-id="DST059F2" data-open="DST059F2" class="link link-reveal link-table xref-fig">2</a>A). These positions show a higher ratio of Ka/Ks value (4 : 1), suggesting that they participate in DNA recognition.</p><p>The most prominent feature of ZFA polymorphism is repeat number variation. In mouse populations, the minimum number of repeats is 9, including the first and last repeats (Fig. <a reveal-id="DST059F3" data-open="DST059F3" class="link link-reveal link-table xref-fig">3</a>). This number was found in three mouse subspecies. Longer repeats (15 and 16) are enriched in the MUS territory, although some MUS mice have shorter repeats. In the MUS territory, although mice inhabiting the border with CAS territory carry such shorter repeats, they share ZFA characteristic of the MUS type.</p><p>The results of our phylogeograpical study clearly show that <em>M. musculus</em> as a species holds extensive <em>Prdm9</em> polymorphism, owing to large degrees of variation in the ZFA. Within a subspecies lineage, the ZFA variant types tend to be similar, with the exception of the CAS lineage. Even though MUS territory extends long distances, reaching from the Northern Eurasian continent to Eastern Europe and the Far East, most mice collected in this range are exclusively included in Group 1 (Fig. <a reveal-id="DST059F4" data-open="DST059F4" class="link link-reveal link-table xref-fig">4</a>A and B). The Japanese population, <em>M. m. molossinus</em>, is a hybrid of two subspecies, MUS and CAS, but its genome is overwhelmingly derived from MUS.<sup><a reveal-id="DST059C34" data-open="DST059C34" class="link link-ref link-reveal xref-bibr">34</a></sup> We found that the majority of mice collected in different localities in Japanese islands have a single ZFA variant type (Ma5) of Group 1. Another variant type (Ca2) in Group 2 (CAS) is carried by one mouse sample with the wm7 MHC haplotype, which was first used for identification of <em>Prdm9</em> as the hotspot determinant.<sup><a reveal-id="DST059C7" data-open="DST059C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Thus, the present result supports a hybrid origin of <em>M. m. molossinus.</em><sup><a reveal-id="DST059C34  DST059C36" data-open="DST059C34  DST059C36" class="link link-ref link-reveal xref-bibr">34–36</a></sup></p><p>Recent studies suggested that Southwest Asia and North India are the likely places of origin of <em>M. musculus.</em><sup><a reveal-id="DST059C37  DST059C39" data-open="DST059C37  DST059C39" class="link link-ref link-reveal xref-bibr">37–39</a></sup> In this region, three major subspecies lineages, DOM, MUS, and CAS, likely separated from one another in subdivided regions and diverged over a relatively long evolutionary time period of 0.5–1.0 million years. Subsequently, the three lineages dispersed to their present ranges, probably associated with agricultural dispersal by humans.<sup><a reveal-id="DST059C22" data-open="DST059C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST059C37  DST059C39" data-open="DST059C37  DST059C39" class="link link-ref link-reveal xref-bibr">37–39</a></sup> The latter event is estimated to have occurred relatively recently, 10–20 thousand years ago.<sup><a reveal-id="DST059C22" data-open="DST059C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST059C40" data-open="DST059C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST059C41" data-open="DST059C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Eastward MUS lineages from the origin of <em>M. musculus</em> may have reached to the Far East, then migrated to Japanese islands 2–3 thousand years ago through the Korean peninsula as stowaways during the transportation of rice, following preceding CAS migration from Southeast Asia, which might have occurred 5–10 thousand years ago.<sup><a reveal-id="DST059C22" data-open="DST059C22" class="link link-ref link-reveal xref-bibr">22</a></sup> If these evolutionary episodes of <em>M. musculus</em> are correct, then it appears that a large degree of <em>Prdm9</em> diversity is not always required for survival in natural populations. A single version of the hotspot repertoire has been sufficient to maintain the Japanese population in a time span of at least 2–3 thousand years. Likewise, local populations in MUS and DOM territories have survived for 10–20 thousand years with a limited polymorphism at <em>Prdm9</em>. Regarding the CAS lineage, the high heterogeneity of ZFA in these mice is consistent with data suggesting that CAS consists of multiple sublineages, which show a relatively large degree of genome divergence from one another.<sup><a reveal-id="DST059C22" data-open="DST059C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Given that the CAS territory includes the likely place of origin of <em>M. musculus</em>,<sup><a reveal-id="DST059C41" data-open="DST059C41" class="link link-ref link-reveal xref-bibr">41</a></sup> its heterogeneity is reminiscent of the African human population, which shows greater diversity of <em>Prdm9</em> ZFA.<sup><a reveal-id="DST059C9" data-open="DST059C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Thus, overall, the phylogeographical features of <em>Prdm9</em> polymorphism in <em>M. musculus</em> are similar to what has been inferred from information about many other genes.<sup><a reveal-id="DST059C21" data-open="DST059C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST059C22" data-open="DST059C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST059C42" data-open="DST059C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Furthermore, we infer that the requirement for hotspot diversity depends on geographical range and time span in evolution, and that <em>Prdm9</em> polymorphism has not been maintained by a simple balanced selection in the population of each subspecies.</p><p>The results of this study show that subspecies-specific <em>Prdm9</em> variant groups prevail in the demarcated territories of <em>M. musculus</em> subspecies; however, the data also revealed intermingled variant groups in areas where the territory of one subspecies borders another (Fig. <a reveal-id="DST059F4" data-open="DST059F4" class="link link-reveal link-table xref-fig">4</a>B). Moreover, except for Group 1, each of the major ZFA variant groups contains small numbers of mice collected in the territory of other subspecies. A more prominent example of intermingled <em>Prdm9</em> alleles was observed in the case of <em>t-</em>haplotype.</p><p>Mouse <em>t</em>-haplotype harbours three inversions in chromosome 17.<sup><a reveal-id="DST059C43  DST059C45" data-open="DST059C43  DST059C45" class="link link-ref link-reveal xref-bibr">43–45</a></sup> The nucleotide sequences of <em>t</em>-haplotype associated genes are largely diverged from those of the wild-type chromosome of <em>M. musculus.</em><sup><a reveal-id="DST059C44" data-open="DST059C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DST059C46" data-open="DST059C46" class="link link-ref link-reveal xref-bibr">46</a></sup> At present, in mouse natural populations, <em>t-</em>haplotype is observed in all subspecies of <em>M. musculus</em> at 10–40% of frequency.<sup><a reveal-id="DST059C47" data-open="DST059C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DST059C48" data-open="DST059C48" class="link link-ref link-reveal xref-bibr">48</a></sup> It is inferred that <em>t</em>-haplotype has been introgressed into all subspecies of <em>M. musculus</em> 10–20 thousand years ago as a single event, and expanded to all subspecies during agricultural dispersal, due to high distortion of the transmission ratio of <em>t-</em>haplotype relative to wild-type chromosome.<sup><a reveal-id="DST059C44" data-open="DST059C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DST059C46" data-open="DST059C46" class="link link-ref link-reveal xref-bibr">46</a></sup> This study clearly shows that <em>t-</em>haplotype mice have unique and characteristic <em>Prdm9</em> ZFA and intronic (HSR1) sequences. These data support the idea that the <em>t</em>-haplotype is of monophyletic origin<em>.</em> In our phylogenetic trees, <em>Tcp1</em> and <em>Prdm9</em> HSR1 in <em>t-</em>haplotype appear in the same clade as those from mice collected in the CAS territory. This suggests that <em>t</em>-haplotype originated in a sublineage of the CAS subspecies and rapidly introgressed into all subspecies of <em>M. musculus</em>.</p><p>Our phylogenetic analysis also revealed that the topology of the ZFA tree differs from that of the HSR1 tree. This implies that intragenic recombination occurred in the interval between these two regions, despite the fact that they are only 9 kb apart. If ZFA contains a recombination hotspot, an associated frequent and unequal recombination might give rise to repeat number variation of ZFs.</p>                            <h2 scrollto-destination=5821475 id="5821475" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5821479 id="5821479" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported in part by Grant-in-Aid for Scientific Research on Innovative Areas titled ‘Systematical studies of chromosome adaptation’ and ‘Non-coding DNA’ from MEXT, Japan, to T.S. and K.O., respectively, and a grant from the Japan Society for the Promotion of Science (JSPS) Research Fellowship to H.K.</p>                            <h2 scrollto-destination=5821483 id="5821483" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank P. Vogel, K.P. Aplin, E. Nevo, L.V. Yakimenko, L.V. Frisman, and A. Kryukov for providing us with wild-captured mice, and K. Artzt and J. Nadeau for providing us with <em>t</em>-haplotype mice. We also thank T. Takada for discussion of phylogenetic analysis of <em>Prdm9</em> and J. Galipon for editing the manuscript.</p>                            <h2 scrollto-destination=5821487 id="5821487" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST059C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petes</div> <div class="given-names">T.D.</div></div>. </div><div class="article-title">Meiotic recombination hot spots and cold spots</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">360</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meiotic%20recombination%20hot%20spots%20and%20cold%20spots&amp;author=T.D.%20Petes&amp;publication_year=2001&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=2&amp;pages=360-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35072078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35072078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35072078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11331902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kauppi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jeffreys</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Keeney</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Where the crossovers are: recombination distributions in mammals</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">413</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Where%20the%20crossovers%20are%3A%20recombination%20distributions%20in%20mammals&amp;author=L.%20Kauppi&amp;author=A.J.%20Jeffreys&amp;author=S.%20Keeney&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=413-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shiroishi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sagai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Moriwaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A new wild-derived H-2 haplotype enhancing K-IA recombination</div>, <div class="source">Nature</div> , <div class="year">1982</div>, vol. <div class="volume">300</div> (pg. <div class="fpage">370</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20wild-derived%20H-2%20haplotype%20enhancing%20K-IA%20recombination&amp;author=T.%20Shiroishi&amp;author=T.%20Sagai&amp;author=K.%20Moriwaki&amp;publication_year=1982&amp;journal=Nature&amp;volume=300&amp;pages=370-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/300370a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F300370a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F300370a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6815537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shiroishi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sagai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hanzawa</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gotoh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Moriwaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genetic control of sex-dependent meiotic recombination in the major histocompatibility complex of the mouse</div>, <div class="source">EMBO J.</div> , <div class="year">1991</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">681</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20sex-dependent%20meiotic%20recombination%20in%20the%20major%20histocompatibility%20complex%20of%20the%20mouse&amp;author=T.%20Shiroishi&amp;author=T.%20Sagai&amp;author=N.%20Hanzawa&amp;author=H.%20Gotoh&amp;author=K.%20Moriwaki&amp;publication_year=1991&amp;journal=EMBO%20J.&amp;volume=10&amp;pages=681-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2001681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parvanov</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Petkov</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Paigen</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Prdm9 controls activation of mammalian recombination hotspots</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> pg. <div class="fpage">835</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prdm9%20controls%20activation%20of%20mammalian%20recombination%20hotspots&amp;author=E.D.%20Parvanov&amp;author=P.M.%20Petkov&amp;author=K.%20Paigen&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=835" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1181495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1181495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Myers</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bowden</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Tumian</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">876</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Drive%20against%20hotspot%20motifs%20in%20primates%20implicates%20the%20PRDM9%20gene%20in%20meiotic%20recombination&amp;author=S.%20Myers&amp;author=R.%20Bowden&amp;author=A.%20Tumian&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=876-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1182363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1182363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1182363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baudat</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Buard</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Grey</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">836</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PRDM9%20is%20a%20major%20determinant%20of%20meiotic%20recombination%20hotspots%20in%20humans%20and%20mice&amp;author=F.%20Baudat&amp;author=J.%20Buard&amp;author=C.%20Grey&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=836-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1183439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1183439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1183439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Groeneveld</div> <div class="given-names">L.F.</div></div>,  <div class="name"><div class="surname">Atencia</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Garriga</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Vigilant</div> <div class="given-names">L.</div></div>. </div><div class="article-title">High diversity at PRDM9 in chimpanzees and bonobos</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e39064</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20diversity%20at%20PRDM9%20in%20chimpanzees%20and%20bonobos&amp;author=L.F.%20Groeneveld&amp;author=R.%20Atencia&amp;author=R.M.%20Garriga&amp;author=L.%20Vigilant&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e39064" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0039064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0039064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0039064"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22768294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berg</div> <div class="given-names">I.L.</div></div>,  <div class="name"><div class="surname">Neumann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sarbajna</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Odenthal-Hesse</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Jeffreys</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Variants of the protein PRDM9 differentially regulate a set of human meiotic recombination hotspots highly active in African populations</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2011</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">12378</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variants%20of%20the%20protein%20PRDM9%20differentially%20regulate%20a%20set%20of%20human%20meiotic%20recombination%20hotspots%20highly%20active%20in%20African%20populations&amp;author=I.L.%20Berg&amp;author=R.%20Neumann&amp;author=S.%20Sarbajna&amp;author=L.%20Odenthal-Hesse&amp;author=N.J.%20Butler&amp;author=A.J.%20Jeffreys&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=12378-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1109531108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1109531108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1109531108"> </span></div> </div></div></div></div></div><div content-id="DST059C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berg</div> <div class="given-names">I.L.</div></div>,  <div class="name"><div class="surname">Neumann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Lam</div> <div class="given-names">K.W.</div></div>,  <div class="name"><div class="surname">Sarbajna</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Odenthal-Hesse</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Jeffreys</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">PRDM9 variation strongly influences recombination hot-spot activity and meiotic instability in humans</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">859</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PRDM9%20variation%20strongly%20influences%20recombination%20hot-spot%20activity%20and%20meiotic%20instability%20in%20humans&amp;author=I.L.%20Berg&amp;author=R.%20Neumann&amp;author=K.W.%20Lam&amp;author=S.%20Sarbajna&amp;author=L.%20Odenthal-Hesse&amp;author=C.A.%20May&amp;author=A.J.%20Jeffreys&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=859-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.658" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.658" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.658"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20818382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brick</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Smagulova</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Khil</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Camerini-Otero</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Petukhova</div> <div class="given-names">G.V.</div></div>. </div><div class="article-title">Genetic recombination is directed away from functional genomic elements in mice</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">485</div> (pg. <div class="fpage">642</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20recombination%20is%20directed%20away%20from%20functional%20genomic%20elements%20in%20mice&amp;author=K.%20Brick&amp;author=F.%20Smagulova&amp;author=P.%20Khil&amp;author=R.D.%20Camerini-Otero&amp;author=G.V.%20Petukhova&amp;publication_year=2012&amp;journal=Nature&amp;volume=485&amp;pages=642-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandor</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Coppieters</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Druet</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Charlier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Georges</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genetic variants in <em>REC8</em>, <em>RNF212</em>, and <em>PRDM9</em> influence male recombination in cattle</div>, <div class="source">PLoS Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002854</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variants%20in%20REC8%2C%20RNF212%2C%20and%20PRDM9%20influence%20male%20recombination%20in%20cattle&amp;author=C.%20Sandor&amp;author=W.%20Li&amp;author=W.%20Coppieters&amp;author=T.%20Druet&amp;author=C.%20Charlier&amp;author=M.%20Georges&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002854" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002854" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002854" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002854"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22844258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Matsui</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">A histone H3 methyltransferase controls epigenetic events required for meiotic prophase</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">438</div> (pg. <div class="fpage">374</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20histone%20H3%20methyltransferase%20controls%20epigenetic%20events%20required%20for%20meiotic%20prophase&amp;author=K.%20Hayashi&amp;author=K.%20Yoshida&amp;author=Y.%20Matsui&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=374-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16292313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mihola</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Trachtulec</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Vlcek</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schimenti</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Forejt</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A mouse speciation gene encodes a meiotic histone H3 methyltransferase</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">323</div> (pg. <div class="fpage">373</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20mouse%20speciation%20gene%20encodes%20a%20meiotic%20histone%20H3%20methyltransferase&amp;author=O.%20Mihola&amp;author=Z.%20Trachtulec&amp;author=C.%20Vlcek&amp;author=J.C.%20Schimenti&amp;author=J.%20Forejt&amp;publication_year=2009&amp;journal=Science&amp;volume=323&amp;pages=373-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1163601" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1163601" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1163601"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19074312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flachs</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Mihola</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Simecek</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Interallelic and intergenic incompatibilities of the <em>Prdm9</em> (<em>Hst1</em>) gene in mouse hybrid sterility</div>, <div class="source">PLoS Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1003044</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interallelic%20and%20intergenic%20incompatibilities%20of%20the%20Prdm9%20(Hst1)%20gene%20in%20mouse%20hybrid%20sterility&amp;author=P.%20Flachs&amp;author=O.%20Mihola&amp;author=P.%20Simecek&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1003044" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23133405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Emerson</div> <div class="given-names">R.O.</div></div>,  <div class="name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Extraordinary molecular evolution in the PRDM9 fertility gene</div>, <div class="source">PloS ONE</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e8505</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extraordinary%20molecular%20evolution%20in%20the%20PRDM9%20fertility%20gene&amp;author=J.H.%20Thomas&amp;author=R.O.%20Emerson&amp;author=J.%20Shendure&amp;publication_year=2009&amp;journal=PloS%20ONE&amp;volume=4&amp;pages=e8505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0008505" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0008505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0008505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20041164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliver</div> <div class="given-names">P.L.</div></div>,  <div class="name"><div class="surname">Goodstadt</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bayes</div> <div class="given-names">J.J.</div></div>, et al. </div><div class="article-title">Accelerated evolution of the Prdm9 speciation gene across diverse metazoan taxa</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000753</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20evolution%20of%20the%20Prdm9%20speciation%20gene%20across%20diverse%20metazoan%20taxa&amp;author=P.L.%20Oliver&amp;author=L.%20Goodstadt&amp;author=J.J.%20Bayes&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000753" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000753" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000753" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000753"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19997497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Klug</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Selection of DNA binding sites for zinc fingers using rationally randomized DNA reveals coded interactions</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">11168</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20of%20DNA%20binding%20sites%20for%20zinc%20fingers%20using%20rationally%20randomized%20DNA%20reveals%20coded%20interactions&amp;author=Y.%20Choo&amp;author=A.%20Klug&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=11168-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.23.11168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.23.11168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.23.11168"> </span></div> </div></div></div></div></div><div content-id="DST059C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Billings</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Parvanov</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Baker</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Walker</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Paigen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Petkov</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">DNA binding specificities of the long zinc-finger recombination protein PRDM9</div>, <div class="source">Genome Biol.</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">R35</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20binding%20specificities%20of%20the%20long%20zinc-finger%20recombination%20protein%20PRDM9&amp;author=T.%20Billings&amp;author=E.D.%20Parvanov&amp;author=C.L.%20Baker&amp;author=M.%20Walker&amp;author=K.%20Paigen&amp;author=P.M.%20Petkov&amp;publication_year=2013&amp;journal=Genome%20Biol.&amp;volume=14&amp;pages=R35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-4-r35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-4-r35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shimada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Terashima</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tsuchiya</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Aplin</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Temporal, spatial, and ecological modes of evolution of Eurasian <em>Mus</em> based on mitochondrial and nuclear gene sequences</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">626</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temporal%2C%20spatial%2C%20and%20ecological%20modes%20of%20evolution%20of%20Eurasian%20Mus%20based%20on%20mitochondrial%20and%20nuclear%20gene%20sequences&amp;author=H.%20Suzuki&amp;author=T.%20Shimada&amp;author=M.%20Terashima&amp;author=K.%20Tsuchiya&amp;author=K.%20Aplin&amp;publication_year=2004&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=33&amp;pages=626-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2004.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2004.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2004.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15522792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boursot</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Auffray</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Britton-Davidian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bonhomme</div> <div class="given-names">F.</div></div>. </div><div class="article-title">The evolution of house mice</div>, <div class="source">Annu. Rev. Ecol. Evol. Syst</div> , <div class="year">1993</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">119</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20house%20mice&amp;author=P.%20Boursot&amp;author=J.C.%20Auffray&amp;author=J.%20Britton-Davidian&amp;author=F.%20Bonhomme&amp;publication_year=1993&amp;journal=Annu.%20Rev.%20Ecol.%20Evol.%20Syst&amp;volume=24&amp;pages=119-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.es.24.110193.001003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.es.24.110193.001003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.es.24.110193.001003"> </span></div> </div></div></div></div></div><div content-id="DST059C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nunome</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Evolutionary and dispersal history of Eurasian house mice <em>Mus musculus</em> clarified by more extensive geographic sampling of mitochondrial DNA</div>, <div class="source">Heredity</div> , <div class="year">2013</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">375</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20and%20dispersal%20history%20of%20Eurasian%20house%20mice%20Mus%20musculus%20clarified%20by%20more%20extensive%20geographic%20sampling%20of%20mitochondrial%20DNA&amp;author=H.%20Suzuki&amp;author=M.%20Nunome&amp;author=G.%20Kinoshita&amp;publication_year=2013&amp;journal=Heredity&amp;volume=111&amp;pages=375-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2013.60" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2013.60" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2013.60"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23820581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keane</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Goodstadt</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Danecek</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Mouse genomic variation and its effect on phenotypes and gene regulation</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">477</div> (pg. <div class="fpage">289</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mouse%20genomic%20variation%20and%20its%20effect%20on%20phenotypes%20and%20gene%20regulation&amp;author=T.M.%20Keane&amp;author=L.%20Goodstadt&amp;author=P.%20Danecek&amp;publication_year=2011&amp;journal=Nature&amp;volume=477&amp;pages=289-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10413" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10413"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yonekawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Moriwaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Gotoh</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Origins of laboratory mice deduced from restriction patterns of mitochondrial DNA</div>, <div class="source">Differentiation</div> , <div class="year">1982</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">222</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20of%20laboratory%20mice%20deduced%20from%20restriction%20patterns%20of%20mitochondrial%20DNA&amp;author=H.%20Yonekawa&amp;author=K.%20Moriwaki&amp;author=O.%20Gotoh&amp;publication_year=1982&amp;journal=Differentiation&amp;volume=22&amp;pages=222-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1432-0436.1982.tb01255.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1432-0436.1982.tb01255.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1432-0436.1982.tb01255.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6293902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bell</div> <div class="given-names">T.A.</div></div>,  <div class="name"><div class="surname">Churchill</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Pardo-Manuel de Villena</div> <div class="given-names">F.</div></div>. </div><div class="article-title">On the subspecific origin of the laboratory mouse</div>, <div class="source">Nat. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">1100</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20subspecific%20origin%20of%20the%20laboratory%20mouse&amp;author=H.%20Yang&amp;author=T.A.%20Bell&amp;author=G.A.%20Churchill&amp;author=F.%20Pardo-Manuel%20de%20Villena&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=1100-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng2087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng2087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng2087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17660819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morita</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kubota</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Evolution of the mouse <em>t</em> haplotype: recent and worldwide introgression to <em>Mus musculus</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">6851</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20mouse%20t%20haplotype%3A%20recent%20and%20worldwide%20introgression%20to%20Mus%20musculus&amp;author=T.%20Morita&amp;author=H.%20Kubota&amp;author=K.%20Murata&amp;publication_year=1992&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=6851-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.15.6851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.15.6851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.15.6851"> </span></div> </div></div></div></div></div><div content-id="DST059C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1980</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">111</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20estimating%20evolutionary%20rates%20of%20base%20substitutions%20through%20comparative%20studies%20of%20nucleotide%20sequences&amp;author=M.%20Kimura&amp;publication_year=1980&amp;journal=J.%20Mol.%20Evol.&amp;volume=16&amp;pages=111-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01731581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01731581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01731581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7463489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Didion</div> <div class="given-names">J.P.</div></div>, et al. </div><div class="article-title">Subspecific origin and haplotype diversity in the laboratory mouse</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">648</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Subspecific%20origin%20and%20haplotype%20diversity%20in%20the%20laboratory%20mouse&amp;author=H.%20Yang&amp;author=J.R.%20Wang&amp;author=J.P.%20Didion&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=648-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.847"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21623374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hughes</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Positive Darwinian selection promotes charge profile diversity in the antigen-binding cleft of class I major-histocompatibility-complex molecules</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1990</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">515</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positive%20Darwinian%20selection%20promotes%20charge%20profile%20diversity%20in%20the%20antigen-binding%20cleft%20of%20class%20I%20major-histocompatibility-complex%20molecules&amp;author=A.L.%20Hughes&amp;author=T.%20Ota&amp;author=M.%20Nei&amp;publication_year=1990&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=7&amp;pages=515-24" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2283951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hussin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sinnett</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Casals</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Rare allelic forms of PRDM9 associated with childhood leukemogenesis</div>, <div class="source">Genome Res.</div> , <div class="year">2013</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">419</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rare%20allelic%20forms%20of%20PRDM9%20associated%20with%20childhood%20leukemogenesis&amp;author=J.%20Hussin&amp;author=D.%20Sinnett&amp;author=F.%20Casals&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=419-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.144188.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.144188.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.144188.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23222848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yonekawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Moriwaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Gotoh</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Hybrid origin of Japanese mice “<em>Mus musculus molossinus”</em>: evidence from restriction analysis of mitochondrial DNA</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1988</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">63</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hybrid%20origin%20of%20Japanese%20mice%20%E2%80%9CMus%20musculus%20molossinus%E2%80%9D%3A%20evidence%20from%20restriction%20analysis%20of%20mitochondrial%20DNA&amp;author=H.%20Yonekawa&amp;author=K.%20Moriwaki&amp;author=O.%20Gotoh&amp;publication_year=1988&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=5&amp;pages=63-78" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2833677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yonekawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Moriwaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Origin and genetic status of <em>Mus</em> musculus molossinus: a typical example of reticulate evolution in the genus <em>Mus</em></div>, <div class="source">Evolution of the House Mouse</div> , <div class="year">2012</div><div class="publisher-loc">Cambridge</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DST059C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ebata</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The ancestor of extant Japanese fancy mice contributed to the mosaic genomes of classical inbred strains</div>, <div class="source">Genome Res.</div> , <div class="year">2013</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1329</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ancestor%20of%20extant%20Japanese%20fancy%20mice%20contributed%20to%20the%20mosaic%20genomes%20of%20classical%20inbred%20strains&amp;author=T.%20Takada&amp;author=T.%20Ebata&amp;author=H.%20Noguchi&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=1329-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.156497.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.156497.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.156497.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23604024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Din</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Anand</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Boursot</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Origin and radiation of the house mouse: clues from nuclear genes</div>, <div class="source">J. Evol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">519</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20radiation%20of%20the%20house%20mouse%3A%20clues%20from%20nuclear%20genes&amp;author=W.%20Din&amp;author=R.%20Anand&amp;author=P.%20Boursot&amp;publication_year=1996&amp;journal=J.%20Evol.%20Biol.&amp;volume=9&amp;pages=519-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1420-9101.1996.9050519.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1420-9101.1996.9050519.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1420-9101.1996.9050519.x"> </span></div> </div></div></div></div></div><div content-id="DST059C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prager</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Tichy</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sage</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Mitochondrial DNA sequence variation in the eastern house mouse <em>Mus musculus</em>: comparison with other house mice and report of a 75-bp tandem repeat</div>, <div class="source">Genetics</div> , <div class="year">1996</div>, vol. <div class="volume">143</div> (pg. <div class="fpage">427</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20sequence%20variation%20in%20the%20eastern%20house%20mouse%20Mus%20musculus%3A%20comparison%20with%20other%20house%20mice%20and%20report%20of%20a%2075-bp%20tandem%20repeat&amp;author=E.M.%20Prager&amp;author=H.%20Tichy&amp;author=R.D.%20Sage&amp;publication_year=1996&amp;journal=Genetics&amp;volume=143&amp;pages=427-46" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8722794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boursot</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Din</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Anand</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Origin and radiation of the house mouse: mitochondrial DNA phylogeny</div>, <div class="source">J. Evol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">391</div>-<div class="lpage">415</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20radiation%20of%20the%20house%20mouse%3A%20mitochondrial%20DNA%20phylogeny&amp;author=P.%20Boursot&amp;author=W.%20Din&amp;author=R.%20Anand&amp;publication_year=1996&amp;journal=J.%20Evol.%20Biol.&amp;volume=9&amp;pages=391-415" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1420-9101.1996.9040391.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1420-9101.1996.9040391.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1420-9101.1996.9040391.x"> </span></div> </div></div></div></div></div><div content-id="DST059C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gündüz</div> <div class="given-names">I</div></div>,  <div class="name"><div class="surname">Rambau</div> <div class="given-names">R.V.</div></div>,  <div class="name"><div class="surname">Tez</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Searle</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Mitochondrial DNA variation in the western house mouse (<em>Mus musculus domesticus</em>) close to its site of origin: studies in Turkey</div>, <div class="source">Biol. J. Linn. Soc.</div> , <div class="year">2005</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">473</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20variation%20in%20the%20western%20house%20mouse%20(Mus%20musculus%20domesticus)%20close%20to%20its%20site%20of%20origin%3A%20studies%20in%20Turkey&amp;author=I%20G%C3%BCnd%C3%BCz&amp;author=R.V.%20Rambau&amp;author=C.%20Tez&amp;author=J.B.%20Searle&amp;publication_year=2005&amp;journal=Biol.%20J.%20Linn.%20Soc.&amp;volume=84&amp;pages=473-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1095-8312.2005.00448.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1095-8312.2005.00448.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1095-8312.2005.00448.x"> </span></div> </div></div></div></div></div><div content-id="DST059C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bonhomme</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Orth</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cucchi</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genetic differentiation of the house mouse around the Mediterranean basin: matrilineal footprints of early and late colonization</div>, <div class="source">Proc. R. Soc. Lond. B Biol. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">1034</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20differentiation%20of%20the%20house%20mouse%20around%20the%20Mediterranean%20basin%3A%20matrilineal%20footprints%20of%20early%20and%20late%20colonization&amp;author=F.%20Bonhomme&amp;author=A.%20Orth&amp;author=T.%20Cucchi&amp;publication_year=2011&amp;journal=Proc.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;volume=278&amp;pages=1034-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2010.1228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2010.1228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.2010.1228"> </span></div> </div></div></div></div></div><div content-id="DST059C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bonhomme</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Searle</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">House mouse phylogeography</div>, <div class="source">Evolution of the House Mouse</div> , <div class="year">2012</div><div class="publisher-loc">Cambridge</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DST059C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lyon</div> <div class="given-names">M.F.</div></div>. </div><div class="article-title">Transmission ratio distortion in mice</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">393</div>-<div class="lpage">408</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transmission%20ratio%20distortion%20in%20mice&amp;author=M.F.%20Lyon&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=37&amp;pages=393-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.37.110801.143030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.37.110801.143030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.37.110801.143030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14616067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pilder</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Hammer</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Silver</div> <div class="given-names">L.M.</div></div>. </div><div class="article-title">A novel mouse chromosome 17 hybrid sterility locus: implications for the origin of <em>t</em> haplotypes</div>, <div class="source">Genetics</div> , <div class="year">1991</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">237</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20mouse%20chromosome%2017%20hybrid%20sterility%20locus%3A%20implications%20for%20the%20origin%20of%20t%20haplotypes&amp;author=S.H.%20Pilder&amp;author=M.F.%20Hammer&amp;author=L.M.%20Silver&amp;publication_year=1991&amp;journal=Genetics&amp;volume=129&amp;pages=237-46" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1936961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Silver</div> <div class="given-names">L.M.</div></div>. </div><div class="article-title">Peculiar journey of a selfish chromosome: mouse <em>t</em>-haplotype and meiotic drive</div>, <div class="source">Trends Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">250</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Peculiar%20journey%20of%20a%20selfish%20chromosome%3A%20mouse%20t-haplotype%20and%20meiotic%20drive&amp;author=L.M.%20Silver&amp;publication_year=1994&amp;journal=Trends%20Genet.&amp;volume=9&amp;pages=250-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9525(93)90090-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9525(93)90090-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9525(93)90090-5"> </span></div> </div></div></div></div></div><div content-id="DST059C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hammer</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Silver</div> <div class="given-names">L.M.</div></div>. </div><div class="article-title">Phylogenetic analysis of the <em>alpha-globin pseudogene-4</em> (<em>Hba-ps4</em>) locus in the house mouse species complex reveals a stepwise evolution of <em>t</em> haplotypes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1993</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">971</div>-<div class="lpage">1001</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20the%20alpha-globin%20pseudogene-4%20(Hba-ps4)%20locus%20in%20the%20house%20mouse%20species%20complex%20reveals%20a%20stepwise%20evolution%20of%20t%20haplotypes&amp;author=M.F.%20Hammer&amp;author=L.M.%20Silver&amp;publication_year=1993&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=10&amp;pages=971-1001" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8105360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST059C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lenington</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Franks</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Distribution of <em>t</em>-haplotypes in natural populations of wild house mice</div>, <div class="source">J. Mammal.</div> , <div class="year">1988</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">489</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20t-haplotypes%20in%20natural%20populations%20of%20wild%20house%20mice&amp;author=S.%20Lenington&amp;author=P.%20Franks&amp;author=J.%20Williams&amp;publication_year=1988&amp;journal=J.%20Mammal.&amp;volume=69&amp;pages=489-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/1381340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F1381340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F1381340"> </span></div> </div></div></div></div></div><div content-id="DST059C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST059C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruvinsky</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Polyakov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Agulnik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tichy</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Figueroa</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Klein</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Low diversity of <em>t</em> haplotypes in the eastern form of the house mouse, <em>Mus. musculus</em> L.</div>, <div class="source">, <em>Genetics</em></div> , <div class="year">1991</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">161</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low%20diversity%20of%20t%20haplotypes%20in%20the%20eastern%20form%20of%20the%20house%20mouse%2C%20Mus.%20musculus%20L.&amp;author=A.%20Ruvinsky&amp;author=A.%20Polyakov&amp;author=A.%20Agulnik&amp;author=H.%20Tichy&amp;author=F.%20Figueroa&amp;author=J.%20Klein&amp;publication_year=1991&amp;journal=%2C%20Genetics&amp;volume=127&amp;pages=161-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2016041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Minoru Ko</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=394421&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst059/2/dst059-dst059supp.pdf?Expires=2147483647&amp;Signature=TYEkx1zZ~XjeBKIYJuhkTrmHtCu-8iMCx0LjjhJSDE5A0ugmpXCF-x1rzbLggHDO7kvw8x~pr53tYT7fRvY1D7LZZwBVOQnGzRFmVHXrgi8l6FPZqg8JoYCD9qAPwlWuycC~G2wiSulbqCY~j-XgEKN9E1AV~f~tVGFbtHuIeA9mQp1Vdua392wZ2~GPKyMHdH4iFeZMulB0GMcjRCJHj9fVWPBxR-VAF35CXw6QQl0COMEyMtfX1ICzCr6sZBcN9STs2rtLmeeq2~3zPqAOFyv9sa--zJT1dwbu8SuGT8tcChiiP4eHAkRRr-YuiQVNB6i3ZSoNOkAr~KnGAH0pMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">401</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">25</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst059"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="394421"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f3%2f315%2f394421">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897900007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (25)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F3%2F315%2F394421" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/394421'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Prdm9 Polymorphism Unveils Mouse Evolutionary Tracks" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst059");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f315%2f394421">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

