

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2097,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/3/193/1745343","@type":"ScholarlyArticle","name":"Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling","datePublished":"2016-03-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2016-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw008","keywords":["ribosome profiling","translation initiation","TetRP","genome annotation","N-terminal"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/3/193/1745343","author":[{"name":"Nakahigashi, Kenji","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka, Yamagata 997-0017 ,  Japan","@type":"Person"},{"name":"Takai, Yuki","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka, Yamagata 997-0017 ,  Japan","@type":"Person"},{"name":"Kimura, Michiko","affiliation":"2  Graduate School of Pharmaceutical Sciences ,  Kyoto University ,  Sakyo-ku, Kyoto 606-8501 ,  Japan","@type":"Person"},{"name":"Abe, Nozomi","affiliation":"1  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka, Yamagata 997-0017 ,  Japan","@type":"Person"},{"name":"Nakayashiki, Toru","affiliation":"3  Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Shiwa, Yuh","affiliation":"4  Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture ,  Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Yoshikawa, Hirofumi","affiliation":"4  Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture ,  Tokyo 156-8502 ,  Japan    5  Department of Bioscience ,  Tokyo University of Agriculture ,  Tokyo 156-8502 ,  Japan","@type":"Person"},{"name":"Wanner, Barry L.","affiliation":"6  Department of Microbiology and Immunobiology ,  Harvard Medical School ,  Boston, MA 02115 ,  USA","@type":"Person"},{"name":"Ishihama, Yasushi","affiliation":"2  Graduate School of Pharmaceutical Sciences ,  Kyoto University ,  Sakyo-ku, Kyoto 606-8501 ,  Japan","@type":"Person"},{"name":"Mori, Hirotada","affiliation":"3  Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  Ikoma, Nara 630-0101 ,  Japan","@type":"Person"}],"description":"Abstract.  Tetracycline-inhibited ribosome profiling (TetRP) provides a powerful new experimental tool for comprehensive genome-wide identification of translati","pageStart":"193","pageEnd":"201","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling" />
    <meta property="og:description" content="Abstract.  Tetracycline-inhibited ribosome profiling (TetRP) provides a powerful new experimental tool for comprehensive genome-wide identification of translati" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/3/193/1745343" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-03-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nakahigashi, Kenji" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata 997-0017, Japan" /><meta name="citation_author" content="Takai, Yuki" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata 997-0017, Japan" /><meta name="citation_author" content="Kimura, Michiko" /><meta name="citation_author_institution" content="Graduate School of Pharmaceutical Sciences, Kyoto University, Sakyo-ku, Kyoto 606-8501, Japan" /><meta name="citation_author" content="Abe, Nozomi" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata 997-0017, Japan" /><meta name="citation_author" content="Nakayashiki, Toru" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Shiwa, Yuh" /><meta name="citation_author_institution" content="Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Yoshikawa, Hirofumi" /><meta name="citation_author_institution" content="Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture, Tokyo 156-8502, Japan" /><meta name="citation_author_institution" content="Department of Bioscience, Tokyo University of Agriculture, Tokyo 156-8502, Japan" /><meta name="citation_author" content="Wanner, Barry L." /><meta name="citation_author_institution" content="Department of Microbiology and Immunobiology, Harvard Medical School, Boston, MA 02115, USA" /><meta name="citation_author" content="Ishihama, Yasushi" /><meta name="citation_author_institution" content="Graduate School of Pharmaceutical Sciences, Kyoto University, Sakyo-ku, Kyoto 606-8501, Japan" /><meta name="citation_author" content="Mori, Hirotada" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, Ikoma, Nara 630-0101, Japan" /><meta name="citation_title" content="Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling" /><meta name="citation_firstpage" content="193" /><meta name="citation_lastpage" content="201" /><meta name="citation_doi" content="10.1093/dnares/dsw008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2016/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Insights from 20 years of bacterial genome sequencing;  citation_journal_title=Funct. Integr. Genomics;  citation_year=2015;  citation_volume=15;  citation_pages=141-61; " /><meta name="citation_reference" content="citation_title=Similarity-based gene detection: using COGs to find evolutionarily-conserved ORFs;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=31" /><meta name="citation_reference" content="citation_title=Escherichia coli K-12: a cooperatively developed annotation snapshot—2005;  citation_journal_title=Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=EcoGene 3.0;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D613-24; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=218-23; " /><meta name="citation_reference" content="citation_title=Ribosome profiling: new views of translation, from single codons to genome scale;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2014;  citation_volume=15;  citation_pages=205-13; " /><meta name="citation_reference" content="citation_title=Translatome profiling: methods for genome-scale analysis of mRNA translation;  citation_journal_title=Brief Funct Genomics;  citation_year=2016;  citation_volume=15;  citation_pages=22-31; " /><meta name="citation_reference" content="citation_title=Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=157;  citation_pages=624-35; " /><meta name="citation_reference" content="citation_title=Genome-wide search for novel human uORFs and N-terminal protein extensions using ribosomal footprinting;  citation_journal_title=Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=2208-18; " /><meta name="citation_reference" content="citation_title=Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=147;  citation_pages=789-802; " /><meta name="citation_reference" content="citation_title=Effect of codon adaptation on codon-level and gene-level translation efficiency in vivo;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1115" /><meta name="citation_reference" content="citation_title=The A-Z of bacterial translation inhibitors;  citation_journal_title=Crit. Rev. Biochem. Mol. Biol.;  citation_year=2009;  citation_volume=44;  citation_pages=393-433; " /><meta name="citation_reference" content="citation_title=Antibiotic mechanisms;  citation_journal_title=Annu. Rev. Pharmacol.;  citation_year=1972;  citation_volume=12;  citation_pages=35-56; " /><meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection;  citation_journal_title=Mol. Syst. Biol.;  citation_year=2006;  citation_volume=2; " /><meta name="citation_reference" content="citation_title=One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=6640-5; " /><meta name="citation_reference" content="citation_title=Transduction of linked genetic characters of the host by bacteriophage P1;  citation_journal_title=Virology;  citation_year=1955;  citation_volume=1;  citation_pages=190-206; " /><meta name="citation_reference" content="citation_title=Culture medium for enterobacteria;  citation_journal_title=J. Bacteriol.;  citation_year=1974;  citation_volume=119;  citation_pages=736-47; " /><meta name="citation_reference" content="citation_title=Gene expression in bacteria using TnphoA and TnphoA‘ elements to make and switch phoA gene, lacZ (op), and lacZ (pr) fusions; citation_publisher=Academic Press, Orlando;  citation_title=Methods in molecular genetics;  citation_year=1994;  citation_volume=Vol. 3;  citation_pages=291-310; " /><meta name="citation_reference" content="citation_title=Antibiotic-sensitive TolC mutants and their suppressors;  citation_journal_title=J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=1851-60; " /><meta name="citation_reference" content="citation_title=Antibiotic susceptibility profiles of Escherichia coli strains lacking multidrug efflux pump genes;  citation_journal_title=Antimicrobial Agents Chemother.;  citation_year=2001;  citation_volume=45;  citation_pages=1126-36; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=HemK, a class of protein methyl transferase with similarity to DNA methyl transferases, methylates polypeptide chain release factors, and hemK knockout induces defects in translational termination;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=1473-8; " /><meta name="citation_reference" content="citation_title=Plasmid vectors for the selection of promoters;  citation_journal_title=Gene;  citation_year=1984;  citation_volume=27;  citation_pages=151-60; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Plainview;  citation_title=A short course in bacterial genetics: a laboratory manual and handbook for Escherichia coli and related bacteria;  citation_year=1992; " /><meta name="citation_reference" content="citation_title=Selecting protein N-terminal peptides by combined fractional diagonal chromatography;  citation_journal_title=Nat. Protoc.;  citation_year=2011;  citation_volume=6;  citation_pages=1130-41; " /><meta name="citation_reference" content="citation_title=A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications;  citation_journal_title=Nat. Biotechnol.;  citation_year=2002;  citation_volume=20;  citation_pages=87-90; " /><meta name="citation_reference" content="citation_title=The tRNA thiolation pathway modulates the intracellular redox state in Escherichia coli;  citation_journal_title=J. Bacteriol.;  citation_year=2013;  citation_volume=195;  citation_pages=2039-49; " /><meta name="citation_reference" content="citation_title=Deep coverage of the Escherichia coli proteome enables the assessment of false discovery rates in simple proteogenomic experiments;  citation_journal_title=Mol. Cell Proteomics;  citation_year=2013;  citation_volume=12;  citation_pages=3420-30; " /><meta name="citation_reference" content="citation_title=Escherichia coli mrsC is an allele of hflB, encoding a membrane-associated ATPase and protease that is required for mRNA decay;  citation_journal_title=J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=1929-38; " /><meta name="citation_reference" content="citation_title=Self-processing of FtsH and its implication for the cleavage specificity of this protease;  citation_journal_title=Biochemistry;  citation_year=1999;  citation_volume=38;  citation_pages=11693-99; " /><meta name="citation_reference" content="citation_title=Identification of novel small RNAs using comparative genomics and microarrays;  citation_journal_title=Genes Dev.;  citation_year=2001;  citation_volume=15;  citation_pages=1637-51; " /><meta name="citation_reference" content="citation_title=Global mapping of translation initiation sites in mammalian cells at single-nucleotide resolution;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=E2424-32; " /><meta name="citation_reference" content="citation_title=Structural basis for the interaction of antibiotics with the peptidyl transferase centre in eubacteria;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=413;  citation_pages=814-21; " /><meta name="citation_reference" content="citation_title=Structures of the Escherichia coli ribosome with antibiotics bound near the peptidyl transferase center explain spectra of drug action;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=17152-57; " /><meta name="citation_reference" content="citation_title=EcoGene-RefSeq: EcoGene tools applied to the RefSeq prokaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=1917-8; " /><meta name="citation_reference" content="citation_title=Site-directed mutagenesis of the dual translational initiation sites of the clpB gene of Escherichia coli and characterization of its gene products;  citation_journal_title=J. Biol. Chem.;  citation_year=1993;  citation_volume=268;  citation_pages=20170-74; " /><meta name="citation_reference" content="citation_title=Dual translational initiation sites control function of the lambda S gene;  citation_journal_title=EMBO J.;  citation_year=1989;  citation_volume=8;  citation_pages=3501-10; " /><meta name="citation_reference" content="citation_title=Membrane localization of small proteins in Escherichia coli;  citation_journal_title=J. Biol. Chem.;  citation_year=2011;  citation_volume=286;  citation_pages=32464-74; " /><meta name="citation_reference" content="citation_title=Small membrane proteins found by comparative genomics and ribosome binding site models;  citation_journal_title=Mol. Microbiol.;  citation_year=2008;  citation_volume=70;  citation_pages=1487-501; " /><meta name="citation_reference" content="citation_title=Unprecedented high-resolution view of bacterial operon architecture revealed by RNA sequencing;  citation_journal_title=MBio;  citation_year=2014;  citation_volume=5;  citation_pages=e01442-14; " /><meta name="citation_reference" content="citation_title=Dark matter in the genome: evidence of widespread transcription detected by microarray tiling experiments;  citation_journal_title=Trends Genet.;  citation_year=2005;  citation_volume=21;  citation_pages=93-102; " /><meta name="citation_reference" content="citation_title=Identification and characterization of upstream open reading frames (uORF) in the 5’ untranslated regions (UTR) of genes in Saccharomyces cerevisiae;  citation_journal_title=Curr. Genet.;  citation_year=2005;  citation_volume=48;  citation_pages=77-87; " /><meta name="citation_reference" content="citation_title=Attenuation in amino acid biosynthetic operons;  citation_journal_title=Annu. Rev. Genet.;  citation_year=1982;  citation_volume=16;  citation_pages=113-34; " /><meta name="citation_reference" content="citation_title=Upstream open reading frames as regulators of mRNA translation;  citation_journal_title=Mol. Cellular Biol.;  citation_year=2000;  citation_volume=20;  citation_pages=8635-42; " /><meta name="citation_reference" content="citation_title=Translation regulation gets its ‘omics’ moment;  citation_journal_title=Wiley Interdiscip. Rev. RNA;  citation_year=2013;  citation_volume=4;  citation_pages=617-30; " /><meta name="citation_reference" content="citation_title=Computational challenges, tools, and resources for analyzing co- and post-transcriptional events in high throughput;  citation_journal_title=Wiley Interdiscip. Rev. RNA;  citation_year=2015;  citation_volume=6;  citation_pages=291-310; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/3/193/17463004/dsw008.pdf" /><meta name="description" content="Abstract.  Tetracycline-inhibited ribosome profiling (TetRP) provides a powerful new experimental tool for comprehensive genome-wide identification of translati" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/3/193/1745343" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/3/193/1745343" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/3/193/1745343");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw008';
    </script>
<title>
	Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745343" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZB2PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthaGlnYXNoaSwgS2VuamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgZm9yIEFkdmFuY2VkIEJpb3NjaWVuY2VzLCBLZWlvIFVuaXZlcnNpdHksIFRzdXJ1b2thLCBZYW1hZ2F0YSA5OTctMDAxNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthaSwgWXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBmb3IgQWR2YW5jZWQgQmlvc2NpZW5jZXMsIEtlaW8gVW5pdmVyc2l0eSwgVHN1cnVva2EsIFlhbWFnYXRhIDk5Ny0wMDE3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktpbXVyYSwgTWljaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBQaGFybWFjZXV0aWNhbCBTY2llbmNlcywgS3lvdG8gVW5pdmVyc2l0eSwgU2FreW8ta3UsIEt5b3RvIDYwNi04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFiZSwgTm96b21pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIGZvciBBZHZhbmNlZCBCaW9zY2llbmNlcywgS2VpbyBVbml2ZXJzaXR5LCBUc3VydW9rYSwgWWFtYWdhdGEgOTk3LTAwMTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYXlhc2hpa2ksIFRvcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgSWtvbWEsIE5hcmEgNjMwLTAxMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpd2EsIFl1aCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlbm9tZSBSZXNlYXJjaCBDZW50ZXIsIE5PREFJIFJlc2VhcmNoIEluc3RpdHV0ZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgVG9reW8gMTU2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWW9zaGlrYXdhLCBIaXJvZnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlbm9tZSBSZXNlYXJjaCBDZW50ZXIsIE5PREFJIFJlc2VhcmNoIEluc3RpdHV0ZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgVG9reW8gMTU2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlLCBUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlLCBUb2t5byAxNTYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5uZXIsIEJhcnJ5IEwuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNaWNyb2Jpb2xvZ3kgYW5kIEltbXVub2Jpb2xvZ3ksIEhhcnZhcmQgTWVkaWNhbCBTY2hvb2wsIEJvc3RvbiwgTUEgMDIxMTUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklzaGloYW1hLCBZYXN1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIFBoYXJtYWNldXRpY2FsIFNjaWVuY2VzLCBLeW90byBVbml2ZXJzaXR5LCBTYWt5by1rdSwgS3lvdG8gNjA2LTg1MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yaSwgSGlyb3RhZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgSWtvbWEsIE5hcmEgNjMwLTAxMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJDb21wcmVoZW5zaXZlIGlkZW50aWZpY2F0aW9uIG9mIHRyYW5zbGF0aW9uIHN0YXJ0IHNpdGVzIGJ5IHRldHJhY3ljbGluZS1pbmhpYml0ZWQgcmlib3NvbWUgcHJvZmlsaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGZyb20gMjAgeWVhcnMgb2YgYmFjdGVyaWFsIGdlbm9tZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5jdC4gSW50ZWdyLiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE0MS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltaWxhcml0eS1iYXNlZCBnZW5lIGRldGVjdGlvbjogdXNpbmcgQ09HcyB0byBmaW5kIGV2b2x1dGlvbmFyaWx5LWNvbnNlcnZlZCBPUkZzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNjaGVyaWNoaWEgY29saSBLLTEyOiBhIGNvb3BlcmF0aXZlbHkgZGV2ZWxvcGVkIGFubm90YXRpb24gc25hcHNob3TigJQyMDA1OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVjb0dlbmUgMy4wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1ENjEzLTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBpbiB2aXZvIG9mIHRyYW5zbGF0aW9uIHdpdGggbnVjbGVvdGlkZSByZXNvbHV0aW9uIHVzaW5nIHJpYm9zb21lIHByb2ZpbGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjQ7ICBjaXRhdGlvbl9wYWdlcz0yMTgtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpYm9zb21lIHByb2ZpbGluZzogbmV3IHZpZXdzIG9mIHRyYW5zbGF0aW9uLCBmcm9tIHNpbmdsZSBjb2RvbnMgdG8gZ2Vub21lIHNjYWxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjA1LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdG9tZSBwcm9maWxpbmc6IG1ldGhvZHMgZm9yIGdlbm9tZS1zY2FsZSBhbmFseXNpcyBvZiBtUk5BIHRyYW5zbGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmllZiBGdW5jdCBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIyLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGlmeWluZyBhYnNvbHV0ZSBwcm90ZWluIHN5bnRoZXNpcyByYXRlcyByZXZlYWxzIHByaW5jaXBsZXMgdW5kZXJseWluZyBhbGxvY2F0aW9uIG9mIGNlbGx1bGFyIHJlc291cmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTc7ICBjaXRhdGlvbl9wYWdlcz02MjQtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHNlYXJjaCBmb3Igbm92ZWwgaHVtYW4gdU9SRnMgYW5kIE4tdGVybWluYWwgcHJvdGVpbiBleHRlbnNpb25zIHVzaW5nIHJpYm9zb21hbCBmb290cHJpbnRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjIwOC0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Umlib3NvbWUgcHJvZmlsaW5nIG9mIG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGxzIHJldmVhbHMgdGhlIGNvbXBsZXhpdHkgYW5kIGR5bmFtaWNzIG9mIG1hbW1hbGlhbiBwcm90ZW9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTQ3OyAgY2l0YXRpb25fcGFnZXM9Nzg5LTgwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0IG9mIGNvZG9uIGFkYXB0YXRpb24gb24gY29kb24tbGV2ZWwgYW5kIGdlbmUtbGV2ZWwgdHJhbnNsYXRpb24gZWZmaWNpZW5jeSBpbiB2aXZvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMTE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEEtWiBvZiBiYWN0ZXJpYWwgdHJhbnNsYXRpb24gaW5oaWJpdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JpdC4gUmV2LiBCaW9jaGVtLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MzkzLTQzMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aWJpb3RpYyBtZWNoYW5pc21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBoYXJtYWNvbC47ICBjaXRhdGlvbl95ZWFyPTE5NzI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zNS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIEVzY2hlcmljaGlhIGNvbGkgSy0xMiBpbi1mcmFtZSwgc2luZ2xlLWdlbmUga25vY2tvdXQgbXV0YW50czogdGhlIEtlaW8gY29sbGVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbmUtc3RlcCBpbmFjdGl2YXRpb24gb2YgY2hyb21vc29tYWwgZ2VuZXMgaW4gRXNjaGVyaWNoaWEgY29saSBLLTEyIHVzaW5nIFBDUiBwcm9kdWN0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTY2NDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNkdWN0aW9uIG9mIGxpbmtlZCBnZW5ldGljIGNoYXJhY3RlcnMgb2YgdGhlIGhvc3QgYnkgYmFjdGVyaW9waGFnZSBQMTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Vmlyb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5NTU7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE5MC0yMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1bHR1cmUgbWVkaXVtIGZvciBlbnRlcm9iYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk3NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz03MzYtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBpbiBiYWN0ZXJpYSB1c2luZyBUbnBob0EgYW5kIFRucGhvQeKAmCBlbGVtZW50cyB0byBtYWtlIGFuZCBzd2l0Y2ggcGhvQSBnZW5lLCBsYWNaIChvcCksIGFuZCBsYWNaIChwcikgZnVzaW9uczsgY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBPcmxhbmRvOyAgY2l0YXRpb25fdGl0bGU9TWV0aG9kcyBpbiBtb2xlY3VsYXIgZ2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9Vm9sLiAzOyAgY2l0YXRpb25fcGFnZXM9MjkxLTMxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aWJpb3RpYy1zZW5zaXRpdmUgVG9sQyBtdXRhbnRzIGFuZCB0aGVpciBzdXBwcmVzc29yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xODY7ICBjaXRhdGlvbl9wYWdlcz0xODUxLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpYmlvdGljIHN1c2NlcHRpYmlsaXR5IHByb2ZpbGVzIG9mIEVzY2hlcmljaGlhIGNvbGkgc3RyYWlucyBsYWNraW5nIG11bHRpZHJ1ZyBlZmZsdXggcHVtcCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW50aW1pY3JvYmlhbCBBZ2VudHMgQ2hlbW90aGVyLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTExMjYtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhZmFzdCBhbmQgbWVtb3J5LWVmZmljaWVudCBhbGlnbm1lbnQgb2Ygc2hvcnQgRE5BIHNlcXVlbmNlcyB0byB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1SMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZW1LLCBhIGNsYXNzIG9mIHByb3RlaW4gbWV0aHlsIHRyYW5zZmVyYXNlIHdpdGggc2ltaWxhcml0eSB0byBETkEgbWV0aHlsIHRyYW5zZmVyYXNlcywgbWV0aHlsYXRlcyBwb2x5cGVwdGlkZSBjaGFpbiByZWxlYXNlIGZhY3RvcnMsIGFuZCBoZW1LIGtub2Nrb3V0IGluZHVjZXMgZGVmZWN0cyBpbiB0cmFuc2xhdGlvbmFsIHRlcm1pbmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTQ3My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFzbWlkIHZlY3RvcnMgZm9yIHRoZSBzZWxlY3Rpb24gb2YgcHJvbW90ZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTUxLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sZCBTcHJpbmcgSGFyYm9yIExhYm9yYXRvcnkgUHJlc3MsIFBsYWludmlldzsgIGNpdGF0aW9uX3RpdGxlPUEgc2hvcnQgY291cnNlIGluIGJhY3RlcmlhbCBnZW5ldGljczogYSBsYWJvcmF0b3J5IG1hbnVhbCBhbmQgaGFuZGJvb2sgZm9yIEVzY2hlcmljaGlhIGNvbGkgYW5kIHJlbGF0ZWQgYmFjdGVyaWE7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGluZyBwcm90ZWluIE4tdGVybWluYWwgcGVwdGlkZXMgYnkgY29tYmluZWQgZnJhY3Rpb25hbCBkaWFnb25hbCBjaHJvbWF0b2dyYXBoeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xMTMwLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHZhcmlhbnQgb2YgeWVsbG93IGZsdW9yZXNjZW50IHByb3RlaW4gd2l0aCBmYXN0IGFuZCBlZmZpY2llbnQgbWF0dXJhdGlvbiBmb3IgY2VsbC1iaW9sb2dpY2FsIGFwcGxpY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTg3LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdFJOQSB0aGlvbGF0aW9uIHBhdGh3YXkgbW9kdWxhdGVzIHRoZSBpbnRyYWNlbGx1bGFyIHJlZG94IHN0YXRlIGluIEVzY2hlcmljaGlhIGNvbGk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTk1OyAgY2l0YXRpb25fcGFnZXM9MjAzOS00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVlcCBjb3ZlcmFnZSBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBwcm90ZW9tZSBlbmFibGVzIHRoZSBhc3Nlc3NtZW50IG9mIGZhbHNlIGRpc2NvdmVyeSByYXRlcyBpbiBzaW1wbGUgcHJvdGVvZ2Vub21pYyBleHBlcmltZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsIFByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zNDIwLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBjb2xpIG1yc0MgaXMgYW4gYWxsZWxlIG9mIGhmbEIsIGVuY29kaW5nIGEgbWVtYnJhbmUtYXNzb2NpYXRlZCBBVFBhc2UgYW5kIHByb3RlYXNlIHRoYXQgaXMgcmVxdWlyZWQgZm9yIG1STkEgZGVjYXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9MTkyOS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZi1wcm9jZXNzaW5nIG9mIEZ0c0ggYW5kIGl0cyBpbXBsaWNhdGlvbiBmb3IgdGhlIGNsZWF2YWdlIHNwZWNpZmljaXR5IG9mIHRoaXMgcHJvdGVhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTExNjkzLTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBub3ZlbCBzbWFsbCBSTkFzIHVzaW5nIGNvbXBhcmF0aXZlIGdlbm9taWNzIGFuZCBtaWNyb2FycmF5czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE2MzctNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBtYXBwaW5nIG9mIHRyYW5zbGF0aW9uIGluaXRpYXRpb24gc2l0ZXMgaW4gbWFtbWFsaWFuIGNlbGxzIGF0IHNpbmdsZS1udWNsZW90aWRlIHJlc29sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9RTI0MjQtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgYmFzaXMgZm9yIHRoZSBpbnRlcmFjdGlvbiBvZiBhbnRpYmlvdGljcyB3aXRoIHRoZSBwZXB0aWR5bCB0cmFuc2ZlcmFzZSBjZW50cmUgaW4gZXViYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQxMzsgIGNpdGF0aW9uX3BhZ2VzPTgxNC0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlcyBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSByaWJvc29tZSB3aXRoIGFudGliaW90aWNzIGJvdW5kIG5lYXIgdGhlIHBlcHRpZHlsIHRyYW5zZmVyYXNlIGNlbnRlciBleHBsYWluIHNwZWN0cmEgb2YgZHJ1ZyBhY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MTcxNTItNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVjb0dlbmUtUmVmU2VxOiBFY29HZW5lIHRvb2xzIGFwcGxpZWQgdG8gdGhlIFJlZlNlcSBwcm9rYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTE5MTctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2l0ZS1kaXJlY3RlZCBtdXRhZ2VuZXNpcyBvZiB0aGUgZHVhbCB0cmFuc2xhdGlvbmFsIGluaXRpYXRpb24gc2l0ZXMgb2YgdGhlIGNscEIgZ2VuZSBvZiBFc2NoZXJpY2hpYSBjb2xpIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGl0cyBnZW5lIHByb2R1Y3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjg7ICBjaXRhdGlvbl9wYWdlcz0yMDE3MC03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHVhbCB0cmFuc2xhdGlvbmFsIGluaXRpYXRpb24gc2l0ZXMgY29udHJvbCBmdW5jdGlvbiBvZiB0aGUgbGFtYmRhIFMgZ2VuZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MzUwMS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVtYnJhbmUgbG9jYWxpemF0aW9uIG9mIHNtYWxsIHByb3RlaW5zIGluIEVzY2hlcmljaGlhIGNvbGk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4NjsgIGNpdGF0aW9uX3BhZ2VzPTMyNDY0LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbWFsbCBtZW1icmFuZSBwcm90ZWlucyBmb3VuZCBieSBjb21wYXJhdGl2ZSBnZW5vbWljcyBhbmQgcmlib3NvbWUgYmluZGluZyBzaXRlIG1vZGVsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9MTQ4Ny01MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVucHJlY2VkZW50ZWQgaGlnaC1yZXNvbHV0aW9uIHZpZXcgb2YgYmFjdGVyaWFsIG9wZXJvbiBhcmNoaXRlY3R1cmUgcmV2ZWFsZWQgYnkgUk5BIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1CaW87ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUwMTQ0Mi0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGFyayBtYXR0ZXIgaW4gdGhlIGdlbm9tZTogZXZpZGVuY2Ugb2Ygd2lkZXNwcmVhZCB0cmFuc2NyaXB0aW9uIGRldGVjdGVkIGJ5IG1pY3JvYXJyYXkgdGlsaW5nIGV4cGVyaW1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9OTMtMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiB1cHN0cmVhbSBvcGVuIHJlYWRpbmcgZnJhbWVzICh1T1JGKSBpbiB0aGUgNeKAmSB1bnRyYW5zbGF0ZWQgcmVnaW9ucyAoVVRSKSBvZiBnZW5lcyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdHRlbnVhdGlvbiBpbiBhbWlubyBhY2lkIGJpb3N5bnRoZXRpYyBvcGVyb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTExMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXBzdHJlYW0gb3BlbiByZWFkaW5nIGZyYW1lcyBhcyByZWd1bGF0b3JzIG9mIG1STkEgdHJhbnNsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbHVsYXIgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz04NjM1LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiByZWd1bGF0aW9uIGdldHMgaXRzIOKAmG9taWNz4oCZIG1vbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9V2lsZXkgSW50ZXJkaXNjaXAuIFJldi4gUk5BOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz02MTctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgY2hhbGxlbmdlcywgdG9vbHMsIGFuZCByZXNvdXJjZXMgZm9yIGFuYWx5emluZyBjby0gYW5kIHBvc3QtdHJhbnNjcmlwdGlvbmFsIGV2ZW50cyBpbiBoaWdoIHRocm91Z2hwdXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVdpbGV5IEludGVyZGlzY2lwLiBSZXYuIFJOQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjkxLTMxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzMvMTkzLzE3NDYzMDA0L2RzdzAwOC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGV0cmFjeWNsaW5lLWluaGliaXRlZCByaWJvc29tZSBwcm9maWxpbmcgKFRldFJQKSBwcm92aWRlcyBhIHBvd2VyZnVsIG5ldyBleHBlcmltZW50YWwgdG9vbCBmb3IgY29tcHJlaGVuc2l2ZSBnZW5vbWUtd2lkZSBpZGVudGlmaWNhdGlvbiBvZiB0cmFuc2xhdGkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTE3NDUzNDMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkMEIL5ImHDkbpMpz34dp/20x97YY=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f193%2f1745343" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f193%2f1745343" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69063" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&amp;Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/3">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87198187">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198187">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198189">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198189">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198194">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198194">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198216">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198216">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198239">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198239">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198244">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198244">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198246">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198248">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198248">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/3/181/1745331">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/3/203/1745366">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenji Nakahigashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka, Yamagata 997-0017</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 235-29-0521. Fax. +81 235-29-0536. E-mail: <a href="mailto:knakahig@pobox.com" target="_blank">knakahig@pobox.com</a> (K.N.); Tel. +81 743-72-5660. Fax. +81 743-72-5669. E-mail: <a href="mailto:hmori@gtc.naist.jp" target="_blank">hmori@gtc.naist.jp</a> (H.M.) 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenji+Nakahigashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakahigashi K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakahigashi K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenji Nakahigashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Takai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka, Yamagata 997-0017</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Takai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takai Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takai Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Takai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michiko Kimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Pharmaceutical Sciences</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Sakyo-ku, Kyoto 606-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michiko+Kimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kimura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kimura M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michiko Kimura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nozomi Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka, Yamagata 997-0017</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nozomi+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nozomi Abe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toru Nakayashiki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toru+Nakayashiki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayashiki T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayashiki T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toru Nakayashiki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuh Shiwa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture</div>, <div class="addr-line">Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuh+Shiwa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shiwa Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shiwa Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuh Shiwa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirofumi Yoshikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Genome Research Center, NODAI Research Institute, Tokyo University of Agriculture</div>, <div class="addr-line">Tokyo 156-8502</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Tokyo University of Agriculture</div>, <div class="addr-line">Tokyo 156-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirofumi+Yoshikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshikawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshikawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirofumi Yoshikawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Barry L. Wanner</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Microbiology and Immunobiology</div>, <div class="insititution">Harvard Medical School</div>, <div class="addr-line">Boston, MA 02115</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Barry+L.+Wanner">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wanner B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wanner B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Barry L. Wanner</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasushi Ishihama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Pharmaceutical Sciences</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Sakyo-ku, Kyoto 606-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasushi+Ishihama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishihama Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishihama Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasushi Ishihama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirotada Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 235-29-0521. Fax. +81 235-29-0536. E-mail: <a href="mailto:knakahig@pobox.com" target="_blank">knakahig@pobox.com</a> (K.N.); Tel. +81 743-72-5660. Fax. +81 743-72-5669. E-mail: <a href="mailto:hmori@gtc.naist.jp" target="_blank">hmori@gtc.naist.jp</a> (H.M.) 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirotada+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirotada Mori</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present Address: Green Earth Institute, 2-5-9 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present Address: Division of Biobank and Data Management, Iwate Tohoku Medical Megabank Organization, Iwate Medical University Disaster Reconstruction Center, 2-1-1 Nishitokuda, Yahaba-cho, Shiwa-gun, Iwate 028-3694, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN3"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 193–201, <a href='https://doi.org/10.1093/dnares/dsw008'>https://doi.org/10.1093/dnares/dsw008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 March 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 February 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745343" href="/dnaresearch/article-pdf/23/3/193/17463004/dsw008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenji Nakahigashi, Yuki Takai, Michiko Kimura, Nozomi Abe, Toru Nakayashiki, Yuh Shiwa, Hirofumi Yoshikawa, Barry L. Wanner, Yasushi Ishihama, Hirotada Mori;  Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling, <em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 193–201, <a href='https://doi.org/10.1093/dnares/dsw008'>https://doi.org/10.1093/dnares/dsw008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745343?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745343?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745343?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745343?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745343?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745343?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745343?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comprehensive%20identification%20of%20translation%20start%20sites%20by%20tetracycline-inhibited%20ribosome%20profiling&amp;publicationDate=2016-03-23&amp;volumeNum=23&amp;issueNum=3&amp;author=Nakahigashi%2C%20Kenji%3B%20Takai%2C%20Yuki&amp;startPage=193&amp;endPage=201&amp;contentId=10.1093%2Fdnares%2Fdsw008&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f193%2f1745343" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87198187 id="87198187" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Tetracycline-inhibited ribosome profiling (TetRP) provides a powerful new experimental tool for comprehensive genome-wide identification of translation initiation sites in bacteria. We validated TetRP by confirming the translation start sites of protein-coding genes in accordance with the 2006 version of <em>Escherichia coli</em> K-12 annotation record (GenBank U00096.2) and found ∼150 new start sites within 60 nucleotides of the annotated site. This analysis revealed 72 per cent of the genes whose initiation site annotations were changed from the 2006 GenBank record to the newer 2014 annotation record (GenBank U00096.3), indicating a high sensitivity. Also, results from reporter fusion and proteomics of N-terminally enriched peptides showed high specificity of the TetRP results. In addition, we discovered over 300 translation start sites within non-coding, intergenic regions of the genome, using a threshold that retains ∼2,000 known coding genes. While some appear to correspond to pseudogenes, others may encode small peptides or have previously unforeseen roles. In summary, we showed that ribosome profiling upon translation inhibition by tetracycline offers a simple, reliable and comprehensive experimental tool for precise annotation of translation start sites of expressed genes in bacteria.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;ribosome profiling&quot;">ribosome profiling</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;translation initiation&quot;">translation initiation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="TetRP">TetRP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome annotation&quot;">genome annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="N-terminal">N-terminal</a></div>                            <h2 scrollto-destination=87198189 id="87198189" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Recent advances in DNA sequencing has permitted rapid determination of the complete genomes of thousands of bacteria.<sup><a reveal-id="DSW008C1" data-open="DSW008C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Computational analyses of these genomes by ORF scan and homology to known genes have greatly facilitated the annotation of protein-coding genes.<sup><a reveal-id="DSW008C2" data-open="DSW008C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, precise identification of the N-termini of ORFs has proven to be difficult, as documented from the continuous re-annotation of <em>Escherichia coli</em> K-12. In early 2006, Riley et al.<sup><a reveal-id="DSW008C3" data-open="DSW008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> reported innumerable updates to gene annotations with 100 s of start site changes, which were supported by a variety of experimental, computational, and database resources. Yet, the 2014 <em>E. coli</em> K-12 MG1655 annotation record (NCBI Reference Sequence: NC_000913.3) included 223 start site changes from the 2006 GenBank record, including 133 new start sites for functional genes and 90 start sites for pseudogenes, which were compiled from various newer experimental results and by comparisons with related genomes.<sup><a reveal-id="DSW008C4" data-open="DSW008C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Ribosome profiling (RP) is an application of high-throughput sequencing, in which mRNA protected from RNase digestion by bound ribosomes is used as the source of the sequence library and consequent identification of translated regions of expressed genes.<sup><a reveal-id="DSW008C5  DSW008C7" data-open="DSW008C5  DSW008C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> Indeed, RP has provided a wealth of data on translation efficiency,<sup><a reveal-id="DSW008C5" data-open="DSW008C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW008C8" data-open="DSW008C8" class="link link-ref link-reveal xref-bibr">8</a></sup> while also changing our understanding of translational control and revealing unexpected translated regions such as upstream micro ORFs and N-terminal identifications.<sup><a reveal-id="DSW008C9" data-open="DSW008C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW008C10" data-open="DSW008C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, RP has not been sufficient for precise identification of translation start sites, at least not in bacteria.</p><p>With the goal towards describing translation efficiency in <em>E. coli</em>, we conducted RP in the presence of the translation inhibitors chloramphenicol (Cm) and tetracycline (Tet),<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> which were chosen because they inhibit translation differently.<sup><a reveal-id="DSW008C12" data-open="DSW008C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSW008C13" data-open="DSW008C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Cm blocks translation elongation by targeting the peptidyl transferase centre on the large ribosomal subunit while Tet inhibits translation by preventing the stable binding of tRNA to the ribosome by directly overlapping with the anticodon stem-loop of tRNA at A-site. While Cm and Tet produced similar RP patterns within central regions of protein-encoding genes, they produced dramatically different patterns near known translation start sites. Importantly, Cm produced broad high-density peaks from the initiation codon to ∼50 nucleotides downstream of the coding region, while Tet produced an RP pattern in which nearly one-half of the signal was sharply concentrated at the location where the initiation codon was at P-site of the ribosome, corresponding to the location of the translating initiation complex.<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> These results suggested the hypothesis that Tet-inhibited RP (TetRP) may be a powerful new tool for comprehensive and precise identification of translation start sites in bacteria.</p><p>Here we validated the usefulness of TetRP for experimental determination of translation start sites and identification of previously unknown translated regions. Results from analysing our TetRP data<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> confirmed the utility of TetRP for defining functional translation initiation codons.</p>                            <h2 scrollto-destination=87198194 id="87198194" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87198195 id="87198195" class="section-title" data-legacy-id=s2a>2.1. Strains and culture conditions</h3>
<p><em>Escherichia coli</em> K-12 BW25113 and its derivatives<sup><a reveal-id="DSW008C14" data-open="DSW008C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSW008C15" data-open="DSW008C15" class="link link-ref link-reveal xref-bibr">15</a></sup> were used throughout. LB medium<sup><a reveal-id="DSW008C16" data-open="DSW008C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and LB agar were used as rich media. Glucose MOPS medium<sup><a reveal-id="DSW008C17" data-open="DSW008C17" class="link link-ref link-reveal xref-bibr">17</a></sup> prepared as described<sup><a reveal-id="DSW008C18" data-open="DSW008C18" class="link link-ref link-reveal xref-bibr">18</a></sup> was used as minimal medium.</p>                            <h3 scrollto-destination=87198197 id="87198197" class="section-title" data-legacy-id=s2b>2.2. Ribosome profile data used and data processing</h3>
<p>The ribosome profile dataset used here has been published<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and is available at DDBJ as BioProject ID:PRJDB2960. In brief, samples were taken from glucose-limited continuous cultures of <em>E. coli</em> K-12 BW25113 and its <em>smpB</em> deletion mutant<sup><a reveal-id="DSW008C14" data-open="DSW008C14" class="link link-ref link-reveal xref-bibr">14</a></sup> in glucose MOPS medium after 30 s treatment with Cm (100 μg/ml) or Tet (40 μg/ml). RP following treatment with clindamycin (10 µg/ml; clindamycin hydrochloride monohydrate, Tokyo Chemical Industry, Japan) and pactamycin (5 µg/ml, Sigma) were similarly performed using an <em>E. coli</em> K-12 BW25113 <em>tolC</em> mutant,<sup><a reveal-id="DSW008C14" data-open="DSW008C14" class="link link-ref link-reveal xref-bibr">14</a></sup> which is more sensitive to these antibiotics.<sup><a reveal-id="DSW008C19" data-open="DSW008C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSW008C20" data-open="DSW008C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p><p>Because the sequence libraries were constructed by adding polyA to the 3′-ends of the short RNA fragments produced with RNase I, the polyA sequences were computationally eliminated using fastx_clipper in the FASTX-Toolkit (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit/index.html">http://hannonlab.cshl.edu/fastx_toolkit/index.html</a>, accessed 27 February 2016) and mapped to the <em>E. coli</em> K-12 genome (GenBank: U00096.2) using bowtie.<sup><a reveal-id="DSW008C21" data-open="DSW008C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Number of reads mapped to coding region was about one to a few million (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Mapped length of reads distributes mainly from 25 to 50 nucleotides, due to size selection of digested mRNA fragments by polyacrylamide gel electrophoresis.<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The length distribution of TetRP reads mapped to CDS, indicating protected length by ribosome, is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>. To compare the read depth at each genomic position, the mapping information of each read was used to summarize the read depth at each position on both strands. For precise mapping, positions corresponding to 3′ ends of reads were used for calculation. The read depth was normalized between samples by defining the average depth over the entire coding region as one; the resultant depth was used as the signal strength at each position. Because 3′ adenines were removed during data processing, reads ending before and after a 3′ adenine were counted as the same position, depth at this position was used for both positions. While this process reduced resolution, we found this step was a reasonable solution for purposes of data analysis.<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Though the protected length by ribosome ranges mainly from 25 to 50 nucleotides in the datasets, our previous analyses showed that it extends 12- to 13-nt in the 3′-direction from the first base of A-site codon but various lengths in the 5′-direction;<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> thus, an average read depth over 15- to 16-nt 3′ was used as the signal to identify codons starting from a specific genome position.</p>                            <h3 scrollto-destination=87198200 id="87198200" class="section-title" data-legacy-id=s2c>2.3. Screening new translation start sites of known coding genes</h3>
<p>In-frame NTG codons &lt;60-nt 5′ from the initiating codon of coding genes<sup><a reveal-id="DSW008C3" data-open="DSW008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> were listed, and signal ratio of the codon to the corresponding new start codon was calculated after addition of 0.5 to both values to avoid division by zero. Candidates for alternate start codons were then selected as described in Results.</p>                            <h3 scrollto-destination=87198202 id="87198202" class="section-title" data-legacy-id=s2d>2.4. Construction of β-galactosidase gene fusions to confirm new translation start sites</h3>
<p>The modified <em>lacZ</em> DNA gene behind the <em>lacUV5 promoter</em> without an operator from pKK232-Z (GGA)<sup><a reveal-id="DSW008C22" data-open="DSW008C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSW008C23" data-open="DSW008C23" class="link link-ref link-reveal xref-bibr">23</a></sup> was replaced with the <em>lacZ</em> alpha region from the low copy number plasmid pMW228 (Nippon Gene Co. Ltd., Tokyo), to construct the low copy number plasmid pMW-base-lacZ, which expresses <em>lacZ</em> constitutively. For this, PCR fragments were generated using pMW228 and pKK232-Z(GGA) as templates and primers XbaI-pMW218-1874R ccttctagACAGCTTTGAATGCACCAAA with XhoI-pMW218-2570F ctcctcgagTTTCTCATAGCTCACGCTGT, and XbaI-TP17-F ccttctagAATTCAGCCCGCCTAATGAG with XhoI-rrnBterm ctcctcgagTGCTTTCCTGATGCAAAAAC, respectively, digested with XbaI and XhoI and ligated. <em>lacZ</em> gene fusions were made by replacing DNA from the BamHI site downstream of the <em>lacZ</em> transcription start site to 6th codon of <em>lacZ</em> with the test sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). To confirm translation was initiated from the test sequence, the central base of either or both possible translation start codons in the region were changed from T to C. All segments generated by PCR were confirmed by DNA sequencing of the relevant regions in the respective plasmids.</p>                            <h3 scrollto-destination=87198204 id="87198204" class="section-title" data-legacy-id=s2e>2.5. β-galactosidase assays</h3>
<p><em>Escherichia coli</em> K-12 BW25113 harboring each plasmid was grown aerobically in minimal medium containing 0.4% glucose and 25 µg/ml ampicillin to 0.5 OD<sub>660</sub> as measured with a digital spectrophotometer (miniphoto 518R, Taitec Corp, Japan). Cultures were cooled on ice and assayed for β-galactosidase as described.<sup><a reveal-id="DSW008C24" data-open="DSW008C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Averages and standard deviations from triplicated cultures were determined.</p>                            <h3 scrollto-destination=87198206 id="87198206" class="section-title" data-legacy-id=s2f>2.6. Proteomics of N-terminal peptides</h3>
<p><em>Escherichia coli</em> K-12 BW25113 was grown to mid-log phase in LB broth with vigorous shaking at 37°C. Six replicated cultures were analysed as follows. Cells were collected by centrifugation and re-suspended in buffer containing 100 mM Tris–HCl (pH 9.0), 12 mM sodium deoxycholate, and 12 mM sodium lauroyl sarcosinate. The protein crude extract was treated by reductive dimethylation followed by trypsin digestion. Protein N-terminal peptides were enriched by COFRADIC<sup><a reveal-id="DSW008C25" data-open="DSW008C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and desalted with a SDB-XC StageTip (GL Sciences, Tokyo, Japan).</p><p>NanoLC-MS/MS analyses were performed on TripleTOF 5600 (AB SCIEX) system, connected to a Thermofisher Scientific UltiMate 3000 pump (Germering, Germany) and a HTC-PAL autosampler (CTC Analytics). Peptides were separated in a self-pulled needle column (150 mm length × 100 µm ID, 6 µm opening) packed with Reprosil-C18 3 μm reversed-phase material (Dr Maisch GmbH, Germany). The mobile phases consisted of (A) 0.5% acetic acid and (B) 0.5% acetic acid and 80% acetonitrile. A three-step linear gradient of 5–10% B in 5 min, 10–40% B in 60 min, 40–100% B in 5 min, and 100% B for 10 min was employed. The mass scan ranges were <em>m/z</em> 300–1,500, and top 10 precursor ions were selected in each MS scan for subsequent MS/MS scans.</p><p>Peptides and proteins were identified by automated database searching using Mascot v2.3 (Matrix Science, London, UK) with a precursor mass tolerance of 20 ppm, a fragment ion mass tolerance of 0.1 Da, and strict trypsin specificity allowing for two missed cleavages. The protein database searched was produced from the 2006 <em>E. coli</em> K-12 MG1665 genome record,<sup><a reveal-id="DSW008C3" data-open="DSW008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> with or without an update from the TetRP results. Peptides were considered identified if the Mascot score was over the 95% confidence limit (<em>P</em> &lt; 0.05) for each peptide. False discovery rate was set to be 1% at peptide level.</p>                            <h3 scrollto-destination=87198210 id="87198210" class="section-title" data-legacy-id=s2g>2.7. Screening for new translation start sites within intergenic regions</h3>
<p>ATG codons outside coding regions, excluding ones &lt;30-nt from the 5′-end of coding regions (betMet), were selected from the 2006 <em>E. coli</em> K-12 annotation<sup><a reveal-id="DSW008C3" data-open="DSW008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and used to identify possible new start sites as described in Results.</p>                            <h3 scrollto-destination=87198212 id="87198212" class="section-title" data-legacy-id=s2h>2.8. Construction of genomic Venus fusions to intergenic regions</h3>
<p>Venus fusions to predicted intergenic coding regions were made by replacing the termination codon of the targeted ORF with a DNA fragment containing Venus (without a start codon) and chloramphenicol resistance (<em>cat</em>) gene by λ Red-mediated recombination.<sup><a reveal-id="DSW008C15" data-open="DSW008C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The Venus-<em>cat</em> template plasmid was made by inserting the Venus coding region<sup><a reveal-id="DSW008C26" data-open="DSW008C26" class="link link-ref link-reveal xref-bibr">26</a></sup> into pKD3.<sup><a reveal-id="DSW008C15" data-open="DSW008C15" class="link link-ref link-reveal xref-bibr">15</a></sup> DNA sequences of the template and PCR primers are in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>.</p>                            <h3 scrollto-destination=87198214 id="87198214" class="section-title" data-legacy-id=s2i>2.9. Flow cytometry</h3>
<p>Genomic Venus fusion strains were grown in glucose minimum medium to 0.3 OD<sub>600</sub> or LB broth overnight and subjected to flow cytometry using a FACScan flow cytometer (Becton, Dickinson) as described elsewhere.<sup><a reveal-id="DSW008C27" data-open="DSW008C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h2 scrollto-destination=87198216 id="87198216" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87198217 id="87198217" class="section-title" data-legacy-id=s3a>3.1. Different patterns of Tet- and Cm-inhibited ribosome profiling</h3>
<p>In an earlier study, we found huge differences in RP read patterns between samples from Cm- and Tet-treated cultures. Whereas Cm-inhibited RP (CmRP) patterns show signals distributed to all coding regions, TetRP patterns have sharp peaks at translational initiation sites (Fig. <a reveal-id="DSW008F1" data-open="DSW008F1" class="link link-reveal link-table xref-fig">1</a>a). At around initiating sites, Cm-inhibited RP (CmRP) patterns show broad high-density reads from the initiating codon to ∼50 nucleotides downstream of the coding region, instead TetRP patterns have peaks at positions where the initiating codon is at the ribosome P-site (Fig. <a reveal-id="DSW008F1" data-open="DSW008F1" class="link link-reveal link-table xref-fig">1</a>b and c).<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Although the mechanism responsible for these differences is not well understood, the characteristic RP pattern from TetRP samples led us to propose the hypothesis that TetRP may be useful for identification of translational start sites in bacteria, which we tested as described below. </p>                        <a id="87198219" scrollto-destination="87198219"></a>
<div id="DSW008F1" data-id="DSW008F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00801.png?Expires=2147483647&Signature=TnaPSw6BQMaJ8~zCuM8UHqrwR3Cht8opQ4H0eN~NjPx4oO~ldkEfuGYpZHZdVJrAMuRmHDEcyVswGHs8z-79NuPMNfcmh7NNpP25gB10Gp~-PZgBNnZFAv48tHCyGXqvtaqxqu57R9zILjzKiaHbEuNjNiMc2SmZtobKbjpY6XmAG7qrW-G1-EX-7x4A28o3uXl02V4Yy7au96jfCod0EWZSVlsCWyBuIIyWBZee96LukoJuxybPSvG~tiglwrFYgRau4v5gni7J6p7pjPDiHbVsObNJT33vHaBEBppKbgF08USK5sJbTh2AklRvtXJRQcItugYp~6rpJzKtx8y7AQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Difference in TetRP and CmRP signal patterns. (a) TetRP and CmRP patterns of a ribosomal protein operon. TetRP (Black) and CmRP (light gray) patterns over an operon coding ribosomal proteins are shown. Location of each gene is shown by an arrow. (b) CmRP and (c) TetRP patterns near 5′ end of coding region of genes. Genes longer than 500-bp were selected, and read depths from −5 to +495 from the first base of initiation codon were normalized to an average value of the region, then the depth of all genes was averaged. Results from three independent cultures are shown in different colours. (d, e) Relative TetRP signal distribution at in-frame ATG codons near initiation codons of known genes is compared with signals at the corresponding initiation codon. (d) Median and (e) average signals at a position from three samples were used for calculation. Median, upper and lower quartiles of all ATG codons at indicated distances are boxed. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198219/dsw00801.tiff" path-from-xml="dsw00801.tiff" target="_blank">View large</a><a section="87198219" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00801.png?Expires=2147483647&Signature=QDyS58eI0YmJKrfDyphFmIBLcJ-QsJnKQiqF5kH3lyYxq2LBOls53Zn6edmchCbBPmyWms1fUJqgkKnfP0ynKMnTGCfENwhRppVZ0UggulbEUUrSkrUausBJ8UBethHBUYJcD5Rp5EycX9-YZIWScdLviaPNPAaje3ueP3Enz-tDcvZ7HmawVGDtfD2ql9R92H8S0N7FegWmRTIrR8SEN3asoWlBMBLV9Z~17a8~nXdkHDoqYFvSm5Xv1YYRWm0PgCQjnRhbolEzqLVpCwWXm8HXnm-O32UqdQIcxQB6cs1~5sRUDs1dyNU3ipMgXpP4DaQh4x4dX~KX6By80LSSRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198219&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Difference in TetRP and CmRP signal patterns. (a) TetRP and CmRP patterns of a ribosomal protein operon. TetRP (Black) and CmRP (light gray) patterns over an operon coding ribosomal proteins are shown. Location of each gene is shown by an arrow. (b) CmRP and (c) TetRP patterns near 5′ end of coding region of genes. Genes longer than 500-bp were selected, and read depths from −5 to +495 from the first base of initiation codon were normalized to an average value of the region, then the depth of all genes was averaged. Results from three independent cultures are shown in different colours. (d, e) Relative TetRP signal distribution at in-frame ATG codons near initiation codons of known genes is compared with signals at the corresponding initiation codon. (d) Median and (e) average signals at a position from three samples were used for calculation. Median, upper and lower quartiles of all ATG codons at indicated distances are boxed. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW008F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00801.png?Expires=2147483647&Signature=TnaPSw6BQMaJ8~zCuM8UHqrwR3Cht8opQ4H0eN~NjPx4oO~ldkEfuGYpZHZdVJrAMuRmHDEcyVswGHs8z-79NuPMNfcmh7NNpP25gB10Gp~-PZgBNnZFAv48tHCyGXqvtaqxqu57R9zILjzKiaHbEuNjNiMc2SmZtobKbjpY6XmAG7qrW-G1-EX-7x4A28o3uXl02V4Yy7au96jfCod0EWZSVlsCWyBuIIyWBZee96LukoJuxybPSvG~tiglwrFYgRau4v5gni7J6p7pjPDiHbVsObNJT33vHaBEBppKbgF08USK5sJbTh2AklRvtXJRQcItugYp~6rpJzKtx8y7AQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Difference in TetRP and CmRP signal patterns. (a) TetRP and CmRP patterns of a ribosomal protein operon. TetRP (Black) and CmRP (light gray) patterns over an operon coding ribosomal proteins are shown. Location of each gene is shown by an arrow. (b) CmRP and (c) TetRP patterns near 5′ end of coding region of genes. Genes longer than 500-bp were selected, and read depths from −5 to +495 from the first base of initiation codon were normalized to an average value of the region, then the depth of all genes was averaged. Results from three independent cultures are shown in different colours. (d, e) Relative TetRP signal distribution at in-frame ATG codons near initiation codons of known genes is compared with signals at the corresponding initiation codon. (d) Median and (e) average signals at a position from three samples were used for calculation. Median, upper and lower quartiles of all ATG codons at indicated distances are boxed. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198219/dsw00801.tiff" path-from-xml="dsw00801.tiff" target="_blank">View large</a><a section="87198219" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00801.png?Expires=2147483647&Signature=QDyS58eI0YmJKrfDyphFmIBLcJ-QsJnKQiqF5kH3lyYxq2LBOls53Zn6edmchCbBPmyWms1fUJqgkKnfP0ynKMnTGCfENwhRppVZ0UggulbEUUrSkrUausBJ8UBethHBUYJcD5Rp5EycX9-YZIWScdLviaPNPAaje3ueP3Enz-tDcvZ7HmawVGDtfD2ql9R92H8S0N7FegWmRTIrR8SEN3asoWlBMBLV9Z~17a8~nXdkHDoqYFvSm5Xv1YYRWm0PgCQjnRhbolEzqLVpCwWXm8HXnm-O32UqdQIcxQB6cs1~5sRUDs1dyNU3ipMgXpP4DaQh4x4dX~KX6By80LSSRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198219&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Difference in TetRP and CmRP signal patterns. (a) TetRP and CmRP patterns of a ribosomal protein operon. TetRP (Black) and CmRP (light gray) patterns over an operon coding ribosomal proteins are shown. Location of each gene is shown by an arrow. (b) CmRP and (c) TetRP patterns near 5′ end of coding region of genes. Genes longer than 500-bp were selected, and read depths from −5 to +495 from the first base of initiation codon were normalized to an average value of the region, then the depth of all genes was averaged. Results from three independent cultures are shown in different colours. (d, e) Relative TetRP signal distribution at in-frame ATG codons near initiation codons of known genes is compared with signals at the corresponding initiation codon. (d) Median and (e) average signals at a position from three samples were used for calculation. Median, upper and lower quartiles of all ATG codons at indicated distances are boxed. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>We initially tested our TetRP hypothesis by comparing TetRP reads at initiating codons with RP reads at nearby in-frame AUG codons (alt-AUG) in known coding genes. Figure <a reveal-id="DSW008F1" data-open="DSW008F1" class="link link-reveal link-table xref-fig">1</a>d and e shows the ratio (reads at alt-AUG codons)/(reads at corresponding initiating codons) against the relative position to the initiating codon. Results showed that read depths at initiating codons were &gt;10-fold higher than at most distal positions. Differences were less at closer positions, and depths at +1 positions (the second codon following the initiating codons) were almost the same as at the initiating codon, possibly due to functioning of alternative initiating codons, the limits of resolution of our data, or both.</p>                            <h3 scrollto-destination=87198221 id="87198221" class="section-title" data-legacy-id=s3b>3.2. Re-mapping start sites of known <em>E. coli</em> protein-encoding genes</h3>
<p>Based on the results above, we attempted to re-annotate the starts of all coding genes, noting that by nature RP data pertain only to genes expressed under experimental growth conditions. We did this by comparing our results with the 2006 annotation record of the <em>E. coli</em> K-12 MG1655 genome (GenBank U00096.2), but excluding pseudogenes and IS genes.<sup><a reveal-id="DSW008C3" data-open="DSW008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> By comparing read depth at initiating codons with nearby (&lt;60-nt) in-frame NUG codons, we selected 177 (1.2% of 17,319) possible alternative initiating (alt-init) codons, which showed higher read depth in two of three samples (median ≥1) and ≥10-fold higher read depth on average. These 177 sites correspond to 165 genes, including 10 genes with 2 new start site candidates and 1 gene with 3 start site candidates. Also, 11 alt-init sites lying upstream of the 2006 annotated site have in-frame stop codons preceding the annotated initiating codon. The remaining 154 genes are listed as alt-init codon(s) in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p><p>We next compared the TetRP data with the 2014 <em>E. coli</em> K-12 MG1655 annotation record (GenBank U00096.3), which was released while this study was in progress. The 2014 GenBank record has 95 genes with re-annotated start sites within our search range, including 63 that match our TetRP results (Fig. <a reveal-id="DSW008F2" data-open="DSW008F2" class="link link-reveal link-table xref-fig">2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>); 5 other genes have new start sites by our analyses, which differ from the re-annotated start sites. Figure <a reveal-id="DSW008F3" data-open="DSW008F3" class="link link-reveal link-table xref-fig">3</a> shows the read depth near the start sites of these 5 genes. Inspection of the signal at three possible initiating sites show that the major signals for <em>ycjY</em>, <em>icdC</em>, <em>yciX</em> (<em>ymiA</em>), and <em>tfaS</em> are at the start site that we mapped and little signal lies at start sites in the 2006 and 2014 <em>E. coli</em> K-12 MG1655 annotation records. For <em>rfaB</em>, the sites from both the TetRP results and the 2014 annotation show signals. An additional strong signal exists near the 2014 <em>rfaB</em> annotated site, indicating that its major initiation site is at this site. </p>                        <a id="87198224" scrollto-destination="87198224"></a>
<div id="DSW008F2" data-id="DSW008F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00802.png?Expires=2147483647&Signature=fCGkHKiS5pLNSuWNeScn-IEJfSVF71T4e3AYwIoo7Pyzg8vQOXAKIxffQLA5qcMuZ26dLwfm0P~Q155aCMVJQLJOJPI09Xynb45UzDdKrKuQx2yvhtcs6B71i3lIuxcEHeRXyqAkkWL1T388Z5cmPhXoqs6nSuBlxz44~2dNR~~Lrxfk4xH95MCTHzafjxq~b9VXWpbtI9-3hwlOPd~ZTe0VR38ycsxBLvpvfKnxRZtWJg015Py9HmErH9I6mF5nA1WsW8bTZtgpUYrson7q08nu08dNdCr8h8dmbx0xdK2k0Lglum5AJw1RRSfCcUNfc6vo4NmuaVLaonirkUZXSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Confirmation flowchart of possible new initiation sites of known genes. Boxes with dark gray (red) lines indicate genes changed annotation in 2014 annotation record; dashed lines indicate unconfirmed or inconclusive changes. Shadowed boxes (green) indicate genes changed by our analysis; stripes indicate unconfirmed or inconclusive changes. Boxes denoted *1 and *2 are 2014 annotation changes that did not match with our analysis. See text for further explanation. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198224/dsw00802.tiff" path-from-xml="dsw00802.tiff" target="_blank">View large</a><a section="87198224" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00802.png?Expires=2147483647&Signature=SwEjsX3HqwSso~PsNUo7jTDP3rXDS3Xg2-N6Jh2F0JFbJbqAMNYVS5ka24eE37-gR3D~UYHLJyn1szpyYNt~Mr~RAg18V-i3wL2QMu1QdMaHakFFiJdnDZQjln3amDoSpLxNwKINIgFFswRs~mzKd~-mTMWzVOkuHFz6Fr1FXaoxEm-zAcR3zP75qP0HRY1qjr-vvl7CUWFpOGe1UaRDXIWtPSBwlNmQapHNSSk3q57BBa2iZhP7T1bsqM1TZw9tlkfDj1j187Z50Jhj6UaAkhQV4EyyUpmlI466c9ft62Wg9EL1xhZLshk9Sh4M8kyanVZ5IgCNYblgFdjdK3j2ew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198224&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Confirmation flowchart of possible new initiation sites of known genes. Boxes with dark gray (red) lines indicate genes changed annotation in 2014 annotation record; dashed lines indicate unconfirmed or inconclusive changes. Shadowed boxes (green) indicate genes changed by our analysis; stripes indicate unconfirmed or inconclusive changes. Boxes denoted *1 and *2 are 2014 annotation changes that did not match with our analysis. See text for further explanation. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW008F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00802.png?Expires=2147483647&Signature=fCGkHKiS5pLNSuWNeScn-IEJfSVF71T4e3AYwIoo7Pyzg8vQOXAKIxffQLA5qcMuZ26dLwfm0P~Q155aCMVJQLJOJPI09Xynb45UzDdKrKuQx2yvhtcs6B71i3lIuxcEHeRXyqAkkWL1T388Z5cmPhXoqs6nSuBlxz44~2dNR~~Lrxfk4xH95MCTHzafjxq~b9VXWpbtI9-3hwlOPd~ZTe0VR38ycsxBLvpvfKnxRZtWJg015Py9HmErH9I6mF5nA1WsW8bTZtgpUYrson7q08nu08dNdCr8h8dmbx0xdK2k0Lglum5AJw1RRSfCcUNfc6vo4NmuaVLaonirkUZXSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Confirmation flowchart of possible new initiation sites of known genes. Boxes with dark gray (red) lines indicate genes changed annotation in 2014 annotation record; dashed lines indicate unconfirmed or inconclusive changes. Shadowed boxes (green) indicate genes changed by our analysis; stripes indicate unconfirmed or inconclusive changes. Boxes denoted *1 and *2 are 2014 annotation changes that did not match with our analysis. See text for further explanation. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198224/dsw00802.tiff" path-from-xml="dsw00802.tiff" target="_blank">View large</a><a section="87198224" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00802.png?Expires=2147483647&Signature=SwEjsX3HqwSso~PsNUo7jTDP3rXDS3Xg2-N6Jh2F0JFbJbqAMNYVS5ka24eE37-gR3D~UYHLJyn1szpyYNt~Mr~RAg18V-i3wL2QMu1QdMaHakFFiJdnDZQjln3amDoSpLxNwKINIgFFswRs~mzKd~-mTMWzVOkuHFz6Fr1FXaoxEm-zAcR3zP75qP0HRY1qjr-vvl7CUWFpOGe1UaRDXIWtPSBwlNmQapHNSSk3q57BBa2iZhP7T1bsqM1TZw9tlkfDj1j187Z50Jhj6UaAkhQV4EyyUpmlI466c9ft62Wg9EL1xhZLshk9Sh4M8kyanVZ5IgCNYblgFdjdK3j2ew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198224&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Confirmation flowchart of possible new initiation sites of known genes. Boxes with dark gray (red) lines indicate genes changed annotation in 2014 annotation record; dashed lines indicate unconfirmed or inconclusive changes. Shadowed boxes (green) indicate genes changed by our analysis; stripes indicate unconfirmed or inconclusive changes. Boxes denoted *1 and *2 are 2014 annotation changes that did not match with our analysis. See text for further explanation. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="87198225" scrollto-destination="87198225"></a>
<div id="DSW008F3" data-id="DSW008F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00803.png?Expires=2147483647&Signature=vqm186MONpCC4Gz6J98L6UBC5Osq72PFkds7A7bVJTzw96ocU3cucQlA3XeSRufrt5NKtN5zkSYGnrd82XrKVYT4FyKMyLq-37LpthRZ-Z5KXGnm58QcWCCU0EeHp3t5t0oxRc6sB4pNSWqK~uop7vjsHUvnPAxERE4M0i5cr-ahfkYxlCB4YfyI1UZv~MvVrwOZzlq3kqGrC7Ywl7UYOCdA1Li1mWyCWl3ho9h9M-CvF9Tl1Zmd92j-ttxb2~SGerssGuCoNlGH7f1sHrPuqaodu~cGfUKKRjajtzC7HXT-CDx3Gojh49wX5RxPWiQOm6OLaDU3BPCfg2dRmQaXQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TetRP signals near initiation region of triply annotated genes. An average + SD value of TetRP signal at each genome position is shown by a black bar and a line. (The TetRP signal is shown at 15-nt 5’ from the 3’-end of the read, corresponding to the first base of P-site codon.) The calculated hybridization stability with anti-ribosome binding site of 16S rRNA (ΔG) is shown by gray line. Coding region starting from the 2006 (top, black), 2014 (middle, blue), and TetRP (bottom, red) annotations are indicated as thick arrows and an upstream gene is sown by a thick gray arrow. A, I, or M in X-axis indicates, A: a position 15-nt. 3’, where sequence reads giving the signal at the position, is adenine, I: end or start position of a gene, or M: both of them. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00803.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198225/dsw00803.tiff" path-from-xml="dsw00803.tiff" target="_blank">View large</a><a section="87198225" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00803.png?Expires=2147483647&Signature=wut1hEv3a7WKtRy0fO~ozYqIi-3H8JmT7nR8zAYJKQQPavrLxNyX-VGgFi8tJWn52nEcZ~rQT4eQW9VZm~HBINFEtB~W7gH-5NHJ-IosoQobE0wB2qFAufF9PwT-ydf~uUDk8Yczp7ZUu79FjeedqDE0BAgeaO9dHEblWmrTA9eTwKlO0dOnE03klL8G4v9oCqHsghumkePhD1ffi-VyIP-MMbVu~ibJN1EIAt8EHb6ACYqb4EeT-OR~pAfJVtlXXPhLlps3jf8TS6XcR7pGt9ji1ZAn4a8V53lcdyFVnYysdgbxCyXLqj33ceRnr1dx8AckrFXqGZDRAvwPuAmBgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198225&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00803.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TetRP signals near initiation region of triply annotated genes. An average + SD value of TetRP signal at each genome position is shown by a black bar and a line. (The TetRP signal is shown at 15-nt 5’ from the 3’-end of the read, corresponding to the first base of P-site codon.) The calculated hybridization stability with anti-ribosome binding site of 16S rRNA (ΔG) is shown by gray line. Coding region starting from the 2006 (top, black), 2014 (middle, blue), and TetRP (bottom, red) annotations are indicated as thick arrows and an upstream gene is sown by a thick gray arrow. A, I, or M in X-axis indicates, A: a position 15-nt. 3’, where sequence reads giving the signal at the position, is adenine, I: end or start position of a gene, or M: both of them. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW008F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00803.png?Expires=2147483647&Signature=vqm186MONpCC4Gz6J98L6UBC5Osq72PFkds7A7bVJTzw96ocU3cucQlA3XeSRufrt5NKtN5zkSYGnrd82XrKVYT4FyKMyLq-37LpthRZ-Z5KXGnm58QcWCCU0EeHp3t5t0oxRc6sB4pNSWqK~uop7vjsHUvnPAxERE4M0i5cr-ahfkYxlCB4YfyI1UZv~MvVrwOZzlq3kqGrC7Ywl7UYOCdA1Li1mWyCWl3ho9h9M-CvF9Tl1Zmd92j-ttxb2~SGerssGuCoNlGH7f1sHrPuqaodu~cGfUKKRjajtzC7HXT-CDx3Gojh49wX5RxPWiQOm6OLaDU3BPCfg2dRmQaXQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TetRP signals near initiation region of triply annotated genes. An average + SD value of TetRP signal at each genome position is shown by a black bar and a line. (The TetRP signal is shown at 15-nt 5’ from the 3’-end of the read, corresponding to the first base of P-site codon.) The calculated hybridization stability with anti-ribosome binding site of 16S rRNA (ΔG) is shown by gray line. Coding region starting from the 2006 (top, black), 2014 (middle, blue), and TetRP (bottom, red) annotations are indicated as thick arrows and an upstream gene is sown by a thick gray arrow. A, I, or M in X-axis indicates, A: a position 15-nt. 3’, where sequence reads giving the signal at the position, is adenine, I: end or start position of a gene, or M: both of them. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00803.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198225/dsw00803.tiff" path-from-xml="dsw00803.tiff" target="_blank">View large</a><a section="87198225" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00803.png?Expires=2147483647&Signature=wut1hEv3a7WKtRy0fO~ozYqIi-3H8JmT7nR8zAYJKQQPavrLxNyX-VGgFi8tJWn52nEcZ~rQT4eQW9VZm~HBINFEtB~W7gH-5NHJ-IosoQobE0wB2qFAufF9PwT-ydf~uUDk8Yczp7ZUu79FjeedqDE0BAgeaO9dHEblWmrTA9eTwKlO0dOnE03klL8G4v9oCqHsghumkePhD1ffi-VyIP-MMbVu~ibJN1EIAt8EHb6ACYqb4EeT-OR~pAfJVtlXXPhLlps3jf8TS6XcR7pGt9ji1ZAn4a8V53lcdyFVnYysdgbxCyXLqj33ceRnr1dx8AckrFXqGZDRAvwPuAmBgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198225&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00803.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TetRP signals near initiation region of triply annotated genes. An average + SD value of TetRP signal at each genome position is shown by a black bar and a line. (The TetRP signal is shown at 15-nt 5’ from the 3’-end of the read, corresponding to the first base of P-site codon.) The calculated hybridization stability with anti-ribosome binding site of 16S rRNA (ΔG) is shown by gray line. Coding region starting from the 2006 (top, black), 2014 (middle, blue), and TetRP (bottom, red) annotations are indicated as thick arrows and an upstream gene is sown by a thick gray arrow. A, I, or M in X-axis indicates, A: a position 15-nt. 3’, where sequence reads giving the signal at the position, is adenine, I: end or start position of a gene, or M: both of them. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>We similarly investigated 27 genes whose start sites were changed from the 2006 to 2014 annotation records and were not revealed by our initial analyses (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4 and Table S3</a></span>). Upon manual inspection, we found that 12 have reads only at the 2014 annotation start sites. We failed to find these initially due to variability among the triplicate samples. Five have signals only at the 2006 annotation start sites, supporting the validity of our analyses for these genes. Six have signals at both the 2006 and 2014 annotation start sites, suggesting alternative start sites function for these genes. Four have no signal at either start site.</p><p>Of the 9 cases where the 2006 to 2014 annotation changes did not match with our analysis (marked *1 and *2 in Fig. <a reveal-id="DSW008F2" data-open="DSW008F2" class="link link-reveal link-table xref-fig">2</a>), two revisions of the 2014 record were made based on proteomics data. In <em>yciX</em> (<em>ymiA</em>), a peptide not expected from 2006 annotation was found,<sup><a reveal-id="DSW008C28" data-open="DSW008C28" class="link link-ref link-reveal xref-bibr">28</a></sup> but which can be derived from proteins initiated from the 2014 annotation and TetRP site. Accordingly, we were unable to determine whether the 2014 or TetRP site is correct from the peptide data. For <em>trpC</em>, an N-terminal peptide corresponding only to the 2014 annotation was found,<sup><a reveal-id="DSW008C28" data-open="DSW008C28" class="link link-ref link-reveal xref-bibr">28</a></sup> contrary to our finding that little TetRP signal was at this site. In seven other cases, the 2006 to 2014 revision appears to have no experimental support (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Thus, excluding these 8 of 9 cases, our TetRP analyses agree with 72% [63/(95 − 8)] of the start sites re-annotated from 2006 to 2014.</p><p>We also closely inspected reads near the initiation sites of 87 genes for which we found new start sites and which were unchanged in the 2006–14 annotation records (Fig. <a reveal-id="DSW008F2" data-open="DSW008F2" class="link link-reveal link-table xref-fig">2</a>). We removed 20 cases with one codon difference from these analyses, because the differences in the read depth between the initiation codon and an adjacent ATG codon were small (Fig. <a reveal-id="DSW008F1" data-open="DSW008F1" class="link link-reveal link-table xref-fig">1</a> d and e). We analysed the remaining 42 ATG, 10 GTG, 9 TTG, and 6 CTG new start sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Of these 67 cases, 39 (27 ATG, 4 GTG, and 8 TTG) have clear signals mainly at the new start sites; 22 (14 ATG, 5 GTG, and 3 CTG) have signals at both the original and the newly annotated sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>); and 6 cases were inconclusive due to weak signal or broad signal distribution near the initiation site(s). For two genes (<em>ymdC</em> and <em>surE</em>), we were unable to ascertain whether the signal near the start sites was real due to overlap of the initiation sites by coding regions of strongly expressed upstream genes. Accordingly, 39–61 (39 + 22) of these 67 genes appear to initiate from the newly identified start sites. Our data not only show high validity but also reveal many genes with dual translation initiation sites.</p>                            <h3 scrollto-destination=87198229 id="87198229" class="section-title" data-legacy-id=s3c>3.3. Use of β-galactosidase gene fusions to confirm new translation start sites</h3>
<p>It is notable that we found 9 new UUG start sites, as only 81 UUG start codons had been previously annotated.<sup><a reveal-id="DSW008C3" data-open="DSW008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> We therefore confirmed several new minor initiation codons (6 UUG and 1 GUG) by comparing their ability to initiate translation with the originally annotated AUG codon by using <em>lacZ</em> reporter gene fusions. To do this, we made a series of <em>lacZ</em> fusions to DNA fragments containing both the original and newly identified initiating codons plus 15-nt 5′- and 3′-flanking sequences to <em>lacZ</em> without a start codon, in which expression was driven by the <em>lacUV5</em> promoter on a low copy number plasmid (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). We then destroyed either or both initiating codons by mutating its second position T (U) to C. The amounts of β-galactosidase made by the resultant constructs are in Fig. <a reveal-id="DSW008F4" data-open="DSW008F4" class="link link-reveal link-table xref-fig">4</a>. Results show that in four of seven cases (<em>ftsH</em>, <em>potA</em>, <em>rfaQ</em>, and <em>yedI</em>) elimination of the new start site (marked by an asterisk in Fig. <a reveal-id="DSW008F4" data-open="DSW008F4" class="link link-reveal link-table xref-fig">4</a>) has a greater effect on reducing β-galactosidase levels than alteration of the original start codon, indicating more efficient translation from the new start codon. In the case of <em>yigE</em>, removal of original (upper) initiating codon has a larger effect. However, examination of <em>yigE</em> transcription in the culture used for RP revealed that transcription initiates between the original and newly annotated start sites; thus, the start site showing higher translation for the <em>lacZ</em> fusion is not used in the genome context (Fig. <a reveal-id="DSW008F4" data-open="DSW008F4" class="link link-reveal link-table xref-fig">4</a> lower right panel). We were unable to draw a definitive conclusion for two genes because (i) the β-galactosidase levels were too low in all cases for <em>ybjO</em> fusions (data not shown) or (ii) the <em>trkG</em> fusion unexpectedly showed the highest level in the construct in which both initiation codons altered (Fig. <a reveal-id="DSW008F4" data-open="DSW008F4" class="link link-reveal link-table xref-fig">4</a>). Thus, five of the seven cases, including 4 with UUGs, indicated higher translation for the newly annotated codon than the original AUG codon, while one case (<em>yigE</em>) was true only in its genomic context. Our data show that these non-ATG codons function efficiently. It is notable that our newly found UUG start site for <em>ftsH</em> had been previously reported,<sup><a reveal-id="DSW008C29" data-open="DSW008C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSW008C30" data-open="DSW008C30" class="link link-ref link-reveal xref-bibr">30</a></sup> but it had not been included in the 2006 nor 2014 GenBank annotation records. </p>                        <a id="87198231" scrollto-destination="87198231"></a>
<div id="DSW008F4" data-id="DSW008F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00804.png?Expires=2147483647&Signature=vnrpCLgxwWaw9~V1Tl4ZWNJbYXwBODVGSBYlVuzLv6EB7UuIY9S4fxECLc2ej-rQmw2S0MtolUip~TWQonHkUBcf2PYNEMCn6fLNyR5YcEuME0RzjWO~P1ZjkZrKjnnuvGf7u1MSBAaqNwF5BBoi1oBQDZwgsgh7T9vESQQ5M0W5RI-XBQdC~Sm3khhwTKYZ3M7ozu3sN2-Sjslfw019UFpTbBs1pAP198EDNk-7inE5B1-9QsLg~X7MDkUPvL0jsZ9ZJySnSAkkvHEkiGxPO8Ga7gx2-UIMcjnNYYo0bOuA6B8H35b55ILwxvcKGZFhi9lM7Fpz27bvHcEUBguSlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="β-Galactoside activity from lacZ reporter gene fusions. Results from fusions to initiation regions (A) with both possible translation initiation codons (wild type), (B) with upstream initiation codon mutated, (C) with downstream initiation codon mutated, and (D) with both initiation codons mutated. The construct with the new annotated initiation codon mutated is marked with an asterisk. Upper left panel shows the structures of the gene fusions with an X marking mutated sites. Lower right panel shows TetRP (black), transcription (gray), and DNA sequence near the yigE initiation codon. -35 and -10 regions of the possible promoter are highlighted (red). The 2006 ATG initiation codon and transcription start site are enlarged. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00804.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198231/dsw00804.tiff" path-from-xml="dsw00804.tiff" target="_blank">View large</a><a section="87198231" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00804.png?Expires=2147483647&Signature=beoIPpbttROcEtD9OBsu5AeEdInBaZm7aERmiXs5iFXbKhF-GY9w2D7IMn1sScBb-pS4MCJ26OCzPXX3NY5a4Lpk9-WgUM4ljlHi50v3Bi392x3Lr5t-ORdUGsOZwPx8S0sSBnGPeNsBy~HdMpaRk8iAKA-BQ7WyRbM2BvmJn~QSq9NDnOKY2ODM9v1XpVe68~CQ3QKds5LVjrQM05Gyegz9lIE~BFHdhdjlLkFCWCkiW~XzPU4yKU14EfDPGi1yPyVXCxZtmkyuYX4ljiRs-UClheTn6WsNs69tELoh1ryGTSGMGtTVIDnqLPliY9hOMJIauoK12nQ-mGGyZakBmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198231&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00804.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>β-Galactoside activity from <em>lacZ</em> reporter gene fusions. Results from fusions to initiation regions (A) with both possible translation initiation codons (wild type), (B) with upstream initiation codon mutated, (C) with downstream initiation codon mutated, and (D) with both initiation codons mutated. The construct with the new annotated initiation codon mutated is marked with an asterisk. Upper left panel shows the structures of the gene fusions with an <em>X</em> marking mutated sites. Lower right panel shows TetRP (black), transcription (gray), and DNA sequence near the <em>yigE</em> initiation codon. -35 and -10 regions of the possible promoter are highlighted (red). The 2006 ATG initiation codon and transcription start site are enlarged. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW008F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00804.png?Expires=2147483647&Signature=vnrpCLgxwWaw9~V1Tl4ZWNJbYXwBODVGSBYlVuzLv6EB7UuIY9S4fxECLc2ej-rQmw2S0MtolUip~TWQonHkUBcf2PYNEMCn6fLNyR5YcEuME0RzjWO~P1ZjkZrKjnnuvGf7u1MSBAaqNwF5BBoi1oBQDZwgsgh7T9vESQQ5M0W5RI-XBQdC~Sm3khhwTKYZ3M7ozu3sN2-Sjslfw019UFpTbBs1pAP198EDNk-7inE5B1-9QsLg~X7MDkUPvL0jsZ9ZJySnSAkkvHEkiGxPO8Ga7gx2-UIMcjnNYYo0bOuA6B8H35b55ILwxvcKGZFhi9lM7Fpz27bvHcEUBguSlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="β-Galactoside activity from lacZ reporter gene fusions. Results from fusions to initiation regions (A) with both possible translation initiation codons (wild type), (B) with upstream initiation codon mutated, (C) with downstream initiation codon mutated, and (D) with both initiation codons mutated. The construct with the new annotated initiation codon mutated is marked with an asterisk. Upper left panel shows the structures of the gene fusions with an X marking mutated sites. Lower right panel shows TetRP (black), transcription (gray), and DNA sequence near the yigE initiation codon. -35 and -10 regions of the possible promoter are highlighted (red). The 2006 ATG initiation codon and transcription start site are enlarged. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00804.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198231/dsw00804.tiff" path-from-xml="dsw00804.tiff" target="_blank">View large</a><a section="87198231" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00804.png?Expires=2147483647&Signature=beoIPpbttROcEtD9OBsu5AeEdInBaZm7aERmiXs5iFXbKhF-GY9w2D7IMn1sScBb-pS4MCJ26OCzPXX3NY5a4Lpk9-WgUM4ljlHi50v3Bi392x3Lr5t-ORdUGsOZwPx8S0sSBnGPeNsBy~HdMpaRk8iAKA-BQ7WyRbM2BvmJn~QSq9NDnOKY2ODM9v1XpVe68~CQ3QKds5LVjrQM05Gyegz9lIE~BFHdhdjlLkFCWCkiW~XzPU4yKU14EfDPGi1yPyVXCxZtmkyuYX4ljiRs-UClheTn6WsNs69tELoh1ryGTSGMGtTVIDnqLPliY9hOMJIauoK12nQ-mGGyZakBmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198231&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00804.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>β-Galactoside activity from <em>lacZ</em> reporter gene fusions. Results from fusions to initiation regions (A) with both possible translation initiation codons (wild type), (B) with upstream initiation codon mutated, (C) with downstream initiation codon mutated, and (D) with both initiation codons mutated. The construct with the new annotated initiation codon mutated is marked with an asterisk. Upper left panel shows the structures of the gene fusions with an <em>X</em> marking mutated sites. Lower right panel shows TetRP (black), transcription (gray), and DNA sequence near the <em>yigE</em> initiation codon. -35 and -10 regions of the possible promoter are highlighted (red). The 2006 ATG initiation codon and transcription start site are enlarged. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87198232 id="87198232" class="section-title" data-legacy-id=s3d>3.4. Confirmation of the new translational initiation start sites with proteomics</h3>
<p>We also used proteomics to confirm the newly annotated start sites by selectively enriching N-terminal peptides after tryptic digestion and LC-MS identification of peptides. In this analysis, we found N-terminal peptides corresponding to 910 proteins of the 4,300 total known coding genes, including N-terminal peptides for 19 of the 154 genes with new start sites, counting peptides starting with the initiating methionine or the second amino acid (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). We also found N-terminal peptides for 8 of these 154 genes corresponding to the 2006 annotation. Importantly, we found two N-terminal peptides for 5 genes, including one corresponding to our new start site and the other corresponding to the 2006 annotation start site. Peptides corresponding to the new site were found more frequently in four cases, indicating dual translation initiation sites and more efficient translation at the new site. Our discovery rate of 12 per cent (19 N-terminal peptides for 154 genes) is significantly lower (<em>P</em> &lt; 0.5) when compared with 21 per cent (910 N-terminal peptides for 4,300 genes) for total proteins. However, considering 60% specificity will result in similar discovery rates indicated good accuracy of our analysis.</p>                            <h3 scrollto-destination=87198234 id="87198234" class="section-title" data-legacy-id=s3e>3.5. Identification of intergenic translation start sites</h3>
<p>Having shown TetRP to be an accurate and reliable tool for identification of translation start sites of known genes, we sought to identify translated regions within intergenic (non-coding) regions of the <em>E. coli</em> K-12 MG1655 genome in the 2006 GenBank record. We examined reads at 14,942 ATGs between intergenic regions, denoted betMet, excluding ATGs within 30-nt of known start sites, as potential alternative translation initiation sites. We selected intergenic start sites by TetRP signal strength at a threshold of average − standard deviation in log (TetRP signals) of known start sites (initMets), but not using those with no signal. This led to keeping 2,588 of 4,310 initMets but only 409 of the 14,942 betMets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6a</a></span>). We then used the TetRP/CmRP ratio at a threshold, keeping 80% (2,071) of the remaining initMets, and selected 360 betMets as candidates for intergenic translation start sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6b and Table S5</a></span>). These 360 intergenic start sites include 27 that were in the 2014 GenBank annotation record (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), including 18 newly annotated protein-coding genes, 5 pseudogenes, and 4 new start sites of known genes. We also identified 5 others as new start sites of known genes. Thus, 328 may function as start sites of novel intergenic translation units. As expected from their intergenic locations, most predicted coding regions are short: only 16 are predicted to encode &gt;50 amino acid residues and 80% encode fewer than 24 residues (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6c</a></span>).</p>                            <h3 scrollto-destination=87198236 id="87198236" class="section-title" data-legacy-id=s3f>3.6. Use of genomic Venus fusions to confirm translation initiation of newly discovered small ORFs</h3>
<p>To test newly identified intergenic start sites for function in their native genomic context, we constructed a series of Venus (bright yellow fluorescent protein: Ref. <a reveal-id="DSW008C26" data-open="DSW008C26" class="link link-ref link-reveal xref-bibr">26</a>) gene (translation) fusions to the 3′-end of selected ORFs by recombining a Venus-<em>cat</em> cassette into the genome as shown in Fig. <a reveal-id="DSW008F5" data-open="DSW008F5" class="link link-reveal link-table xref-fig">5</a> (upper panel). We chose five small ORFs including <em>yciY</em> (from the 2014 annotation<sup><a reveal-id="DSW008C4" data-open="DSW008C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW008C31" data-open="DSW008C31" class="link link-ref link-reveal xref-bibr">31</a></sup>) for analysis, which we named IVP1, 3 (<em>yciY</em>), 4, 6, and 7 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5 and Fig. S7</a></span>). We also chose two intergenic ORFs with ATGs with lower TetRP signal (IVP2 and 5) than selected, but located 7- and 10-nt from IVP1 and IVP4, respectively, to check the resolution of our analysis. Our flow cytometry data for these strains and a control wild type without Venus are in Fig. <a reveal-id="DSW008F5" data-open="DSW008F5" class="link link-reveal link-table xref-fig">5</a>. On the one hand, in LB medium three of four and in minimal medium two of four fusions to these ORFs produced expression levels similar to the IVP3 (<em>yciY</em>) fusion, indicating that these start sites are functional. On the other hand, the IVP2 and IVP5 fusions showed lower fluorescence in both medium than IVP1 and IVP4, respectively, indicating that TetRP reads can discriminate sites separated by 10 or fewer nt. </p>                        <a id="87198238" scrollto-destination="87198238"></a>
<div id="DSW008F5" data-id="DSW008F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00805.png?Expires=2147483647&Signature=OhJL9iaADhlY1vJAdgmwCcfD9JaX6lEWGOiouLfIu3TI4dZQEYbVsm5C6GqUTvyehPw4JfT-KVIv5YiUFutTlptepGjftRVQYgvSvqpfPu30wLh7DJMm8vbU88~Vq5sGt2nUa2jlZ76Q3xsT6YeERN9DQE9-18eJHwo1qe2HAakwnaKpXhImHGMjNUkXVGVoeH3OO0fvauT6rur8JUXfoVgBCMCtFEBlebzhw~tMNBlDN3ip5H6yS2txPJ8LTg1nAVdnGzARmXNf3VVvILC1cc1tUgIC2kaSd6PMviObGmOcE-1hySfqYhnxHkqaPE6rGImjRifGn6qcqhwCX2K9BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of small (s)ORF-Venus fusion in genomic context. Upper panel: Expression pattern of the sORF in wild type and structure of the fusion gene. TetRP (red, mapped location of 3′-end of sequence reads), CmRP, and transcription (blue and black, respectively; all sequence reads mapped) signals are shown with the location of sRNA and flanking genes. Venus and cat DNA cassette was inserted at the 3′ end of sORF as described in Materials and Methods. Lower panels: Fluorescence from the Venus fusion protein in each strain is shown following growth on LB or glucose minimal medium, as indicated. Fluorescence from wild type lacking Venus was used as background. Signal from positive control (IVP3, yciY) is in red, and negative controls (IVP2 and 5) are in black. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00805.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198238/dsw00805.tiff" path-from-xml="dsw00805.tiff" target="_blank">View large</a><a section="87198238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00805.png?Expires=2147483647&Signature=HJKPU8XQpfBkLkC0HZicCcthwYplBYiA~DTB7D2hahLNXiAl5Rr7di6toM92sU1xMRvRTlFf3xNwJjHdHzi3kXOiUJITT0K02a6mP6W-UkfQXll5D8jQxMDxi3YnIssk4blUkOfK1Ct1L54CnQfapt3VR5bO-yNIpgpgf5FjuHFXnAvvcIomQ728NSUr7~3TLiKUlIT1xthgXgep5Sky4OUf0pFbCipwTZUe1pJObK4RftS1Y9169xfa1ZbXLXCwVjISretM9G296qpVyDc9htnn0i2Vx0muxtNw2lRZh1RycZ~YPx7T~XofHbYgWfhsJ~oUZWucx2MFp61jTWvHoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198238&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00805.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of small (s)ORF-Venus fusion in genomic context. Upper panel: Expression pattern of the sORF in wild type and structure of the fusion gene. TetRP (red, mapped location of 3′-end of sequence reads), CmRP, and transcription (blue and black, respectively; all sequence reads mapped) signals are shown with the location of sRNA and flanking genes. Venus and <em>cat</em> DNA cassette was inserted at the 3′ end of sORF as described in Materials and Methods. Lower panels: Fluorescence from the Venus fusion protein in each strain is shown following growth on LB or glucose minimal medium, as indicated. Fluorescence from wild type lacking Venus was used as background. Signal from positive control (IVP3, <em>yciY</em>) is in red, and negative controls (IVP2 and 5) are in black. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW008F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/m_dsw00805.png?Expires=2147483647&Signature=OhJL9iaADhlY1vJAdgmwCcfD9JaX6lEWGOiouLfIu3TI4dZQEYbVsm5C6GqUTvyehPw4JfT-KVIv5YiUFutTlptepGjftRVQYgvSvqpfPu30wLh7DJMm8vbU88~Vq5sGt2nUa2jlZ76Q3xsT6YeERN9DQE9-18eJHwo1qe2HAakwnaKpXhImHGMjNUkXVGVoeH3OO0fvauT6rur8JUXfoVgBCMCtFEBlebzhw~tMNBlDN3ip5H6yS2txPJ8LTg1nAVdnGzARmXNf3VVvILC1cc1tUgIC2kaSd6PMviObGmOcE-1hySfqYhnxHkqaPE6rGImjRifGn6qcqhwCX2K9BQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of small (s)ORF-Venus fusion in genomic context. Upper panel: Expression pattern of the sORF in wild type and structure of the fusion gene. TetRP (red, mapped location of 3′-end of sequence reads), CmRP, and transcription (blue and black, respectively; all sequence reads mapped) signals are shown with the location of sRNA and flanking genes. Venus and cat DNA cassette was inserted at the 3′ end of sORF as described in Materials and Methods. Lower panels: Fluorescence from the Venus fusion protein in each strain is shown following growth on LB or glucose minimal medium, as indicated. Fluorescence from wild type lacking Venus was used as background. Signal from positive control (IVP3, yciY) is in red, and negative controls (IVP2 and 5) are in black. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw00805.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198238/dsw00805.tiff" path-from-xml="dsw00805.tiff" target="_blank">View large</a><a section="87198238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw00805.png?Expires=2147483647&Signature=HJKPU8XQpfBkLkC0HZicCcthwYplBYiA~DTB7D2hahLNXiAl5Rr7di6toM92sU1xMRvRTlFf3xNwJjHdHzi3kXOiUJITT0K02a6mP6W-UkfQXll5D8jQxMDxi3YnIssk4blUkOfK1Ct1L54CnQfapt3VR5bO-yNIpgpgf5FjuHFXnAvvcIomQ728NSUr7~3TLiKUlIT1xthgXgep5Sky4OUf0pFbCipwTZUe1pJObK4RftS1Y9169xfa1ZbXLXCwVjISretM9G296qpVyDc9htnn0i2Vx0muxtNw2lRZh1RycZ~YPx7T~XofHbYgWfhsJ~oUZWucx2MFp61jTWvHoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198238&ar=1745343&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00805.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of small (s)ORF-Venus fusion in genomic context. Upper panel: Expression pattern of the sORF in wild type and structure of the fusion gene. TetRP (red, mapped location of 3′-end of sequence reads), CmRP, and transcription (blue and black, respectively; all sequence reads mapped) signals are shown with the location of sRNA and flanking genes. Venus and <em>cat</em> DNA cassette was inserted at the 3′ end of sORF as described in Materials and Methods. Lower panels: Fluorescence from the Venus fusion protein in each strain is shown following growth on LB or glucose minimal medium, as indicated. Fluorescence from wild type lacking Venus was used as background. Signal from positive control (IVP3, <em>yciY</em>) is in red, and negative controls (IVP2 and 5) are in black. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h2 scrollto-destination=87198239 id="87198239" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We showed that TetRP is a reliable experimental tool for comprehensive and precise identification of translation start sites for expressed genes in <em>E. coli</em>. It is especially notable that TetRP revealed &gt;70% of start site changes in the latest (2014) GenBank annotation record, which included many sites manually compiled from a number of individual experimental studies in addition to ones identified by homology to closely related genomes,<sup><a reveal-id="DSW008C4" data-open="DSW008C4" class="link link-ref link-reveal xref-bibr">4</a></sup> while also producing a low false discovery rate. The high sensitivity and specificity of finding working initiation sites based on existing annotation data shows TetRP to be reliable for validation of correct translation start sites, following standard gene annotation steps of ORF scan and homology searching.</p><p>RP following treatment with translation inhibitors has been previously used to identify translation initiation sites in eukaryotes,<sup><a reveal-id="DSW008C10" data-open="DSW008C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW008C32" data-open="DSW008C32" class="link link-ref link-reveal xref-bibr">32</a></sup> but not in bacteria. We tested several antibiotics in addition to Cm and Tet and found two others (clindamycin and pactamycin) that showed similar initiation site patterns as Tet (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Since pactamycin is believed to inhibit the first translocation from initiation complex,<sup><a reveal-id="DSW008C12" data-open="DSW008C12" class="link link-ref link-reveal xref-bibr">12</a></sup> emergence of initiation site-specific RP pattern for pactamycin is expected. However, clindamycin and Tet inhibit translation elongation by binding to 50S subunit to inhibit peptidyl transferase reaction and to 30S subunit to block aminoacyl-tRNA entry, respectively.<sup><a reveal-id="DSW008C12" data-open="DSW008C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSW008C33" data-open="DSW008C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSW008C34" data-open="DSW008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> More information on the mode of action is required to understand the basis of the observed initiation site-specific RP patterns. Irrespective of full understanding, three antibiotics are applicable for identification of translation start sites, though Tet is generally more suitable considering its broad specificity and availability.<sup><a reveal-id="DSW008C12" data-open="DSW008C12" class="link link-ref link-reveal xref-bibr">12</a></sup> As described herein, the resolution of TetRP in this report was somewhat limited by the addition of polyA to short RNAs prior to reverse transcription during the construction of sequencing libraries, which led to an ambiguity of the 3′-end positions and required sequencing in the 5′ to 3′ direction with respect to the RNA to find 3′-ends.<sup><a reveal-id="DSW008C11" data-open="DSW008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> While we used this method to avoid quantitative bias in RNA sequencing,<sup><a reveal-id="DSW008C5" data-open="DSW008C5" class="link link-ref link-reveal xref-bibr">5</a></sup> the use of adapter ligation to 3′-ends of short RNA for library construction and sequencing in the 3′ to 5′ direction of the RNA would be useful to enhance resolution.</p><p>With regards to genome annotation of <em>E. coli</em> K-12, results in this work are important not only because we found many new N-termini of known proteins in the extensively re-annotated 2014 GenBank record annotation,<sup><a reveal-id="DSW008C4" data-open="DSW008C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW008C35" data-open="DSW008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> but also because we uncovered multiple translation initiation sites for many genes. We identified 28 genes with two initiation sites by TetRP, including 5 proteins with two start sites by N′-proteomics. Although, only a few cases had been found in <em>E. coli</em>,<sup><a reveal-id="DSW008C36" data-open="DSW008C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSW008C37" data-open="DSW008C37" class="link link-ref link-reveal xref-bibr">37</a></sup> proteins with multiple start sites may be more prevalent than previously had been thought. Our results from testing alternate initiation codons by constructing β-galactosidase gene fusions also showed that in most cases constructs lacking either initiation codon retained some activity while constructs lacking both codons showed lowest activity (Fig. <a reveal-id="DSW008F3" data-open="DSW008F3" class="link link-reveal link-table xref-fig">3</a>), which also suggests functioning of multiple initiation codons. These genes may change the major translation initiation site in response to cellular environments or signals, and TetRP can be a useful means to globally investigate mechanisms of how cells select translation initiation site.</p><p>We also found evidence for &gt;300 new translation units without using homology or ORF scan information. Some such ORFs are likely to have function like other small proteins recently found in the cell membrane<sup><a reveal-id="DSW008C38" data-open="DSW008C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSW008C39" data-open="DSW008C39" class="link link-ref link-reveal xref-bibr">39</a></sup> or are pseudogene fragments like ones that have already been included in the 2014 <em>E. coli</em> K-12 GenBank record (NCBI Reference Sequence: NC_000913.3). However, it is difficult to infer how many of the 328 new intergenic translation units have biological function. Though TetRP signal strength of these possible initiation sites are generally similar with the known initiation sites, many weak sites may result from low level infidelity of ribosomes binding to RNA and the high sensitivity of TetRP to detect such binding sites. For example, we recently uncovered thousands of previously undocumented transcription start sites in <em>E. coli</em> K-12 by strand-specific RNA-seq,<sup><a reveal-id="DSW008C40" data-open="DSW008C40" class="link link-ref link-reveal xref-bibr">40</a></sup> many of which are antisense to coding regions. Whether these have biological roles or correspond to genomic ‘dark matter’ is unknown.<sup><a reveal-id="DSW008C41" data-open="DSW008C41" class="link link-ref link-reveal xref-bibr">41</a></sup> In support of this notion, we found that the length distribution of ORFs from the intergenic start sites was similar to the length distribution of all ORFs within intergenic regions or ORFs randomly produced by the genetic code (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6c</a></span>). Accordingly, peptides produced by these new initiation sites appear not to have been selected by their function. It is also notable that the portion of very short ORFs (&lt;9 residues) is higher in the selected ORFs than in the random or total ORF sets. Especially, the portion of single-codon ORFs, i.e. ones with a start codon followed immediately by a stop codon was higher in the selected ORFs. Translation of the detected small intergenic ORFs may have biological function even if the translation product has no function. Ribosome binding or translation of short ORFs may have key gene regulatory roles in both prokaryotes and eukaryotes.<sup><a reveal-id="DSW008C42  DSW008C45" data-open="DSW008C42  DSW008C45" class="link link-ref link-reveal xref-bibr">42–45</a></sup> Yet, many of the intergenic translation initiation sites were not obviously related to nearby coding genes, though ribosome binding to such RNAs may affect its function by changing its structure or stability.<sup><a reveal-id="DSW008C46" data-open="DSW008C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p>                            <h2 scrollto-destination=87198244 id="87198244" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by Japan Society for the Promotion of Science Grant-in-Aid for Scientific Research (B) (20310117 and 24310148 to K.N.), (A) (25250028 to H.M.), and Grant-in-Aid for Scientific Research on Innovative Areas (26116717 to H.M and K.N.); the National Science Foundation (106394 to B.L.W.); and a co-operative research grant of the Genome Research for Bioresource, NODAI Genome Research Center, Tokyo University of Agriculture. Funding to pay the Open Access publication charges for this article was provided by Japan Society for the Promotion of Science, Grant-in-Aid for Scientific Research (A) (25250028).</p>                            <h2 scrollto-destination=87198246 id="87198246" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Mei Wada and Masayuki Honma for technical assistance and Akio Kanai for valuable discussions. We thank the authorities at Tsuruoka City and Yamagata Prefecture for supporting this study at Keio University Institute for Advanced Biosciences.</p>                            <h2 scrollto-destination=87198248 id="87198248" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW008C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Land</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Hauser</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Jun</div> <div class="given-names">S.R.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Insights from 20 years of bacterial genome sequencing</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="volume">15</div>, <div class="fpage">141</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20from%2020%20years%20of%20bacterial%20genome%20sequencing&amp;author=M.%20Land&amp;author=L.%20Hauser&amp;author=S.R.%20Jun&amp;publication_year=2015&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=15&amp;pages=141-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-015-0433-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-015-0433-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-015-0433-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25722247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Powell</div>
               <div class="given-names">B.C.</div>
            </div>, <div class="name string-name"><div class="surname">Hutchison</div> <div class="given-names">C.A.</div>3rd</div></div>. <div class="year">2006</div>, <div class="article-title">Similarity-based gene detection: using COGs to find evolutionarily-conserved ORFs</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">7</div>, <div class="fpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Similarity-based%20gene%20detection%3A%20using%20COGs%20to%20find%20evolutionarily-conserved%20ORFs&amp;author=B.C.%20Powell&amp;author=C.A.%20Hutchison&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16423288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Riley</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Arnaud</div> <div class="given-names">M.B.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Escherichia coli K-12: a cooperatively developed annotation snapshot—2005</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">34</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20K-12%3A%20a%20cooperatively%20developed%20annotation%20snapshot%E2%80%942005&amp;author=M.%20Riley&amp;author=T.%20Abe&amp;author=M.B.%20Arnaud&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16397293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhou</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Rudd</div> <div class="given-names">K.E.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">EcoGene 3.0</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">D613</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EcoGene%203.0&amp;author=J.%20Zhou&amp;author=K.E.%20Rudd&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D613-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1235" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1235" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1235"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23197660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ingolia</div>
               <div class="given-names">N.T.</div>
            </div>, <div class="name string-name"><div class="surname">Ghaemmaghami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Newman</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J.S.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling</div>, <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">218</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20in%20vivo%20of%20translation%20with%20nucleotide%20resolution%20using%20ribosome%20profiling&amp;author=N.T.%20Ingolia&amp;author=S.%20Ghaemmaghami&amp;author=J.R.%20Newman&amp;author=J.S.%20Weissman&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=218-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1168978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1168978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1168978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19213877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ingolia</div>
               <div class="given-names">N.T.</div>
            </div>
          </div>
          <div class="year">2014</div>, <div class="article-title">Ribosome profiling: new views of translation, from single codons to genome scale</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">15</div>, <div class="fpage">205</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosome%20profiling%3A%20new%20views%20of%20translation%2C%20from%20single%20codons%20to%20genome%20scale&amp;author=N.T.%20Ingolia&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=15&amp;pages=205-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3645" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3645" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3645"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24468696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">King</div>
               <div class="given-names">H.A.</div>
            </div>, <div class="name string-name"><div class="surname">Gerber</div> <div class="given-names">A.P.</div></div></div>
          <div class="year">2016</div>, <div class="article-title">Translatome profiling: methods for genome-scale analysis of mRNA translation</div>, <div class="source">Brief Funct Genomics</div> , <div class="volume">15</div>, <div class="fpage">22</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translatome%20profiling%3A%20methods%20for%20genome-scale%20analysis%20of%20mRNA%20translation&amp;author=H.A.%20King&amp;author=A.P.%20Gerber&amp;publication_year=2016&amp;journal=Brief%20Funct%20Genomics&amp;volume=15&amp;pages=22-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25380596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">G.-W.</div>
            </div>, <div class="name string-name"><div class="surname">Burkhardt</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gross</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J.S.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources</div>, <div class="source">Cell</div> , <div class="volume">157</div>, <div class="fpage">624</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantifying%20absolute%20protein%20synthesis%20rates%20reveals%20principles%20underlying%20allocation%20of%20cellular%20resources&amp;author=G.-W.%20Li&amp;author=D.%20Burkhardt&amp;author=C.%20Gross&amp;author=J.S.%20Weissman&amp;publication_year=2014&amp;journal=Cell&amp;volume=157&amp;pages=624-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.02.033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.02.033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.02.033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24766808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fritsch</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Herrmann</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nothnagel</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Genome-wide search for novel human uORFs and N-terminal protein extensions using ribosomal footprinting</div>, <div class="source">Genome Res.</div> , <div class="volume">22</div>, <div class="fpage">2208</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20search%20for%20novel%20human%20uORFs%20and%20N-terminal%20protein%20extensions%20using%20ribosomal%20footprinting&amp;author=C.%20Fritsch&amp;author=A.%20Herrmann&amp;author=M.%20Nothnagel&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=2208-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.139568.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.139568.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.139568.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22879431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ingolia</div>
               <div class="given-names">N.T.</div>
            </div>, <div class="name string-name"><div class="surname">Lareau</div> <div class="given-names">L.F.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J.S.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes</div>, <div class="source">Cell</div> , <div class="volume">147</div>, <div class="fpage">789</div>–<div class="lpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosome%20profiling%20of%20mouse%20embryonic%20stem%20cells%20reveals%20the%20complexity%20and%20dynamics%20of%20mammalian%20proteomes&amp;author=N.T.%20Ingolia&amp;author=L.F.%20Lareau&amp;author=J.S.%20Weissman&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=789-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22056041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nakahigashi</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Takai</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shiwa</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Effect of codon adaptation on codon-level and gene-level translation efficiency in vivo</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20codon%20adaptation%20on%20codon-level%20and%20gene-level%20translation%20efficiency%20in%20vivo&amp;author=K.%20Nakahigashi&amp;author=Y.%20Takai&amp;author=Y.%20Shiwa&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-1115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-1115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-1115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25512115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wilson</div>
               <div class="given-names">D.N.</div>
            </div>
          </div>
          <div class="year">2009</div>, <div class="article-title">The A-Z of bacterial translation inhibitors</div>, <div class="source">Crit. Rev. Biochem. Mol. Biol.</div> , <div class="volume">44</div>, <div class="fpage">393</div>–<div class="lpage">433</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20A-Z%20of%20bacterial%20translation%20inhibitors&amp;author=D.N.%20Wilson&amp;publication_year=2009&amp;journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol.&amp;volume=44&amp;pages=393-433" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3109/10409230903307311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3109%2F10409230903307311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3109%2F10409230903307311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19929179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hash</div>
               <div class="given-names">J.H.</div>
            </div>
          </div>
          <div class="year">1972</div>, <div class="article-title">Antibiotic mechanisms</div>, <div class="source">Annu. Rev. Pharmacol.</div> , <div class="volume">12</div>, <div class="fpage">35</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antibiotic%20mechanisms&amp;author=J.H.%20Hash&amp;publication_year=1972&amp;journal=Annu.%20Rev.%20Pharmacol.&amp;volume=12&amp;pages=35-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.pa.12.040172.000343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.pa.12.040172.000343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.pa.12.040172.000343"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4114184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Baba</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Ara</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hasegawa</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="volume">2</div>, <div class="comment">2006 0008 (article number)</div>.</div></div></div></div><div content-id="DSW008C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Datsenko</div>
               <div class="given-names">K.A.</div>
            </div>, <div class="name string-name"><div class="surname">Wanner</div> <div class="given-names">B.L.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">97</div>, <div class="fpage">6640</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One-step%20inactivation%20of%20chromosomal%20genes%20in%20Escherichia%20coli%20K-12%20using%20PCR%20products&amp;author=K.A.%20Datsenko&amp;author=B.L.%20Wanner&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=6640-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.120163297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.120163297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.120163297"> </span></div> </div></div></div></div></div><div content-id="DSW008C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lennox</div>
               <div class="given-names">E.S.</div>
            </div>
          </div>
          <div class="year">1955</div>, <div class="article-title">Transduction of linked genetic characters of the host by bacteriophage P1</div>, <div class="source">Virology</div> , <div class="volume">1</div>, <div class="fpage">190</div>–<div class="lpage">206</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transduction%20of%20linked%20genetic%20characters%20of%20the%20host%20by%20bacteriophage%20P1&amp;author=E.S.%20Lennox&amp;publication_year=1955&amp;journal=Virology&amp;volume=1&amp;pages=190-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0042-6822(55)90016-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0042-6822(55)90016-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0042-6822(55)90016-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13267987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Neidhardt</div>
               <div class="given-names">F.C.</div>
            </div>, <div class="name string-name"><div class="surname">Bloch</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">D.F.</div></div></div>
          <div class="year">1974</div>, <div class="article-title">Culture medium for enterobacteria</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">119</div>, <div class="fpage">736</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Culture%20medium%20for%20enterobacteria&amp;author=F.C.%20Neidhardt&amp;author=P.L.%20Bloch&amp;author=D.F.%20Smith&amp;publication_year=1974&amp;journal=J.%20Bacteriol.&amp;volume=119&amp;pages=736-47" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4604283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wanner</div>
               <div class="given-names">B.L.</div>
            </div>
          </div>
          <div class="year">1994</div>, <div class="article-title">Gene expression in bacteria using Tn<em>phoA</em> and Tn<em>phoA</em>‘ elements to make and switch <em>phoA</em> gene, <em>lacZ</em> (op), and <em>lacZ</em> (pr) fusions</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Adolph</div> <div class="given-names">K.W.</div></div></div>, ed., <div class="source">Methods in molecular genetics</div> , <div class="volume">Vol. 3</div>. <div class="publisher-name">Academic Press</div>: <div class="publisher-loc">Orlando</div>, pp. <div class="fpage">291</div>–<div class="lpage">310</div>.</div></div></div></div><div content-id="DSW008C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Augustus</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Celaya</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Husain</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Humbard</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Misra</div> <div class="given-names">R.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Antibiotic-sensitive TolC mutants and their suppressors</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">186</div>, <div class="fpage">1851</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antibiotic-sensitive%20TolC%20mutants%20and%20their%20suppressors&amp;author=A.M.%20Augustus&amp;author=T.%20Celaya&amp;author=F.%20Husain&amp;author=M.%20Humbard&amp;author=R.%20Misra&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=1851-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.6.1851-1860.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.6.1851-1860.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.6.1851-1860.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14996816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sulavik</div>
               <div class="given-names">M.C.</div>
            </div>, <div class="name string-name"><div class="surname">Houseweart</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Cramer</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Antibiotic susceptibility profiles of Escherichia coli strains lacking multidrug efflux pump genes</div>, <div class="source">Antimicrobial Agents Chemother.</div> , <div class="volume">45</div>, <div class="fpage">1126</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antibiotic%20susceptibility%20profiles%20of%20Escherichia%20coli%20strains%20lacking%20multidrug%20efflux%20pump%20genes&amp;author=M.C.%20Sulavik&amp;author=C.%20Houseweart&amp;author=C.%20Cramer&amp;publication_year=2001&amp;journal=Antimicrobial%20Agents%20Chemother.&amp;volume=45&amp;pages=1126-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.45.4.1126-1136.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.45.4.1126-1136.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.45.4.1126-1136.2001"> </span></div> </div></div></div></div></div><div content-id="DSW008C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nakahigashi</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Kubo</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Narita</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">HemK, a class of protein methyl transferase with similarity to DNA methyl transferases, methylates polypeptide chain release factors, and hemK knockout induces defects in translational termination</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">99</div>, <div class="fpage">1473</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HemK%2C%20a%20class%20of%20protein%20methyl%20transferase%20with%20similarity%20to%20DNA%20methyl%20transferases%2C%20methylates%20polypeptide%20chain%20release%20factors%2C%20and%20hemK%20knockout%20induces%20defects%20in%20translational%20termination&amp;author=K.%20Nakahigashi&amp;author=N.%20Kubo&amp;author=S.%20Narita&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=1473-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.032488499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.032488499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.032488499"> </span></div> </div></div></div></div></div><div content-id="DSW008C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brosius</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">1984</div>, <div class="article-title">Plasmid vectors for the selection of promoters</div>, <div class="source">Gene</div> , <div class="volume">27</div>, <div class="fpage">151</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmid%20vectors%20for%20the%20selection%20of%20promoters&amp;author=J.%20Brosius&amp;publication_year=1984&amp;journal=Gene&amp;volume=27&amp;pages=151-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(84)90136-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(84)90136-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(84)90136-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6327464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Miller</div>
               <div class="given-names">J.H.</div>
            </div>
          </div>
          <div class="year">1992</div>, <div class="source">A short course in bacterial genetics: a laboratory manual and handbook for Escherichia coli and related bacteria</div> . <div class="publisher-name">Cold Spring Harbor Laboratory Press</div>: <div class="publisher-loc">Plainview</div>.</div></div></div></div><div content-id="DSW008C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Staes</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Impens</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Van Damme</div> <div class="given-names">P.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Selecting protein N-terminal peptides by combined fractional diagonal chromatography</div>, <div class="source">Nat. Protoc.</div> , <div class="volume">6</div>, <div class="fpage">1130</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selecting%20protein%20N-terminal%20peptides%20by%20combined%20fractional%20diagonal%20chromatography&amp;author=A.%20Staes&amp;author=F.%20Impens&amp;author=P.%20Van%20Damme&amp;publication_year=2011&amp;journal=Nat.%20Protoc.&amp;volume=6&amp;pages=1130-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2011.355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2011.355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2011.355"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21799483" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nagai</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Ibata</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Kubota</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mikoshiba</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Miyawaki</div> <div class="given-names">A.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">20</div>, <div class="fpage">87</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20variant%20of%20yellow%20fluorescent%20protein%20with%20fast%20and%20efficient%20maturation%20for%20cell-biological%20applications&amp;author=T.%20Nagai&amp;author=K.%20Ibata&amp;author=E.S.%20Park&amp;author=M.%20Kubota&amp;author=K.%20Mikoshiba&amp;author=A.%20Miyawaki&amp;publication_year=2002&amp;journal=Nat.%20Biotechnol.&amp;volume=20&amp;pages=87-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt0102-87" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt0102-87" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt0102-87"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11753368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nakayashiki</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Takeuchi</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">The tRNA thiolation pathway modulates the intracellular redox state in Escherichia coli</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">195</div>, <div class="fpage">2039</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tRNA%20thiolation%20pathway%20modulates%20the%20intracellular%20redox%20state%20in%20Escherichia%20coli&amp;author=T.%20Nakayashiki&amp;author=N.%20Saito&amp;author=R.%20Takeuchi&amp;publication_year=2013&amp;journal=J.%20Bacteriol.&amp;volume=195&amp;pages=2039-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.02180-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.02180-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.02180-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23457245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krug</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Carpy</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Behrends</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Matic</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Soares</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Macek</div> <div class="given-names">B.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Deep coverage of the Escherichia coli proteome enables the assessment of false discovery rates in simple proteogenomic experiments</div>, <div class="source">Mol. Cell Proteomics</div> , <div class="volume">12</div>, <div class="fpage">3420</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20coverage%20of%20the%20Escherichia%20coli%20proteome%20enables%20the%20assessment%20of%20false%20discovery%20rates%20in%20simple%20proteogenomic%20experiments&amp;author=K.%20Krug&amp;author=A.%20Carpy&amp;author=G.%20Behrends&amp;author=K.%20Matic&amp;author=N.C.%20Soares&amp;author=B.%20Macek&amp;publication_year=2013&amp;journal=Mol.%20Cell%20Proteomics&amp;volume=12&amp;pages=3420-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/mcp.M113.029165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fmcp.M113.029165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fmcp.M113.029165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23908556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">R.F.</div>
            </div>, <div class="name string-name"><div class="surname">O'Hara</div> <div class="given-names">E.B.</div></div>, <div class="name string-name"><div class="surname">Aldea</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bargmann</div> <div class="given-names">C.I.</div></div>, <div class="name string-name"><div class="surname">Gromley</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kushner</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Escherichia coli mrsC is an allele of hflB, encoding a membrane-associated ATPase and protease that is required for mRNA decay</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">180</div>, <div class="fpage">1929</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20mrsC%20is%20an%20allele%20of%20hflB%2C%20encoding%20a%20membrane-associated%20ATPase%20and%20protease%20that%20is%20required%20for%20mRNA%20decay&amp;author=R.F.%20Wang&amp;author=E.B.%20O'Hara&amp;author=M.%20Aldea&amp;author=C.I.%20Bargmann&amp;author=H.%20Gromley&amp;author=S.R.%20Kushner&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=1929-38" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9537394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akiyama</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">1999</div>, <div class="article-title">Self-processing of FtsH and its implication for the cleavage specificity of this protease</div>, <div class="source">Biochemistry</div> , <div class="volume">38</div>, <div class="fpage">11693</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Self-processing%20of%20FtsH%20and%20its%20implication%20for%20the%20cleavage%20specificity%20of%20this%20protease&amp;author=Y.%20Akiyama&amp;publication_year=1999&amp;journal=Biochemistry&amp;volume=38&amp;pages=11693-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi991177c" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi991177c" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi991177c"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10512625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wassarman</div>
               <div class="given-names">K.M.</div>
            </div>, <div class="name string-name"><div class="surname">Repoila</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Rosenow</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Storz</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Gottesman</div> <div class="given-names">S.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Identification of novel small RNAs using comparative genomics and microarrays</div>, <div class="source">Genes Dev.</div> , <div class="volume">15</div>, <div class="fpage">1637</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20novel%20small%20RNAs%20using%20comparative%20genomics%20and%20microarrays&amp;author=K.M.%20Wassarman&amp;author=F.%20Repoila&amp;author=C.%20Rosenow&amp;author=G.%20Storz&amp;author=S.%20Gottesman&amp;publication_year=2001&amp;journal=Genes%20Dev.&amp;volume=15&amp;pages=1637-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.901001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.901001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.901001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11445539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">S.X.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">S.B.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Global mapping of translation initiation sites in mammalian cells at single-nucleotide resolution</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">E2424</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20mapping%20of%20translation%20initiation%20sites%20in%20mammalian%20cells%20at%20single-nucleotide%20resolution&amp;author=S.%20Lee&amp;author=B.%20Liu&amp;author=S.%20Lee&amp;author=S.X.%20Huang&amp;author=B.%20Shen&amp;author=S.B.%20Qian&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=E2424-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1207846109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1207846109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1207846109"> </span></div> </div></div></div></div></div><div content-id="DSW008C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schlunzen</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Zarivach</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Harms</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Structural basis for the interaction of antibiotics with the peptidyl transferase centre in eubacteria</div>, <div class="source">Nature</div> , <div class="volume">413</div>, <div class="fpage">814</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20the%20interaction%20of%20antibiotics%20with%20the%20peptidyl%20transferase%20centre%20in%20eubacteria&amp;author=F.%20Schlunzen&amp;author=R.%20Zarivach&amp;author=J.%20Harms&amp;publication_year=2001&amp;journal=Nature&amp;volume=413&amp;pages=814-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35101544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35101544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35101544"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11677599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dunkle</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Xiong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Mankin</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Cate</div> <div class="given-names">J.H.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Structures of the Escherichia coli ribosome with antibiotics bound near the peptidyl transferase center explain spectra of drug action</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">17152</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structures%20of%20the%20Escherichia%20coli%20ribosome%20with%20antibiotics%20bound%20near%20the%20peptidyl%20transferase%20center%20explain%20spectra%20of%20drug%20action&amp;author=J.A.%20Dunkle&amp;author=L.%20Xiong&amp;author=A.S.%20Mankin&amp;author=J.H.%20Cate&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=17152-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1007988107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1007988107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1007988107"> </span></div> </div></div></div></div></div><div content-id="DSW008C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhou</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Richardson</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Rudd</div> <div class="given-names">K.E.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">EcoGene-RefSeq: EcoGene tools applied to the RefSeq prokaryotic genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">1917</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EcoGene-RefSeq%3A%20EcoGene%20tools%20applied%20to%20the%20RefSeq%20prokaryotic%20genomes&amp;author=J.%20Zhou&amp;author=A.J.%20Richardson&amp;author=K.E.%20Rudd&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=1917-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23736533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Park</div>
               <div class="given-names">S.K.</div>
            </div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">K.I.</div></div>, <div class="name string-name"><div class="surname">Woo</div> <div class="given-names">K.M.</div></div>et al.  </div>. <div class="year">1993</div>, <div class="article-title">Site-directed mutagenesis of the dual translational initiation sites of the <em>clpB</em> gene of <em>Escherichia coli</em> and characterization of its gene products</div>, <div class="source">J. Biol. Chem.</div> , <div class="volume">268</div>, <div class="fpage">20170</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Site-directed%20mutagenesis%20of%20the%20dual%20translational%20initiation%20sites%20of%20the%20clpB%20gene%20of%20Escherichia%20coli%20and%20characterization%20of%20its%20gene%20products&amp;author=S.K.%20Park&amp;author=K.I.%20Kim&amp;author=K.M.%20Woo&amp;publication_year=1993&amp;journal=J.%20Biol.%20Chem.&amp;volume=268&amp;pages=20170-74" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8376377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blasi</div>
               <div class="given-names">U.</div>
            </div>, <div class="name string-name"><div class="surname">Nam</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hartz</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gold</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Young</div> <div class="given-names">R.</div></div></div>
          <div class="year">1989</div>, <div class="article-title">Dual translational initiation sites control function of the lambda S gene</div>, <div class="source">EMBO J.</div> , <div class="volume">8</div>, <div class="fpage">3501</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dual%20translational%20initiation%20sites%20control%20function%20of%20the%20lambda%20S%20gene&amp;author=U.%20Blasi&amp;author=K.%20Nam&amp;author=D.%20Hartz&amp;author=L.%20Gold&amp;author=R.%20Young&amp;publication_year=1989&amp;journal=EMBO%20J.&amp;volume=8&amp;pages=3501-10" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2531079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fontaine</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Fuchs</div> <div class="given-names">R.T.</div></div>, <div class="name string-name"><div class="surname">Storz</div> <div class="given-names">G.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Membrane localization of small proteins in Escherichia coli</div>, <div class="source">J. Biol. Chem.</div> , <div class="volume">286</div>, <div class="fpage">32464</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Membrane%20localization%20of%20small%20proteins%20in%20Escherichia%20coli&amp;author=F.%20Fontaine&amp;author=R.T.%20Fuchs&amp;author=G.%20Storz&amp;publication_year=2011&amp;journal=J.%20Biol.%20Chem.&amp;volume=286&amp;pages=32464-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M111.245696" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M111.245696" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M111.245696"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21778229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hemm</div>
               <div class="given-names">M.R.</div>
            </div>, <div class="name string-name"><div class="surname">Paul</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Schneider</div> <div class="given-names">T.D.</div></div>, <div class="name string-name"><div class="surname">Storz</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Rudd</div> <div class="given-names">K.E.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Small membrane proteins found by comparative genomics and ribosome binding site models</div>, <div class="source">Mol. Microbiol.</div> , <div class="volume">70</div>, <div class="fpage">1487</div>–<div class="lpage">501</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20membrane%20proteins%20found%20by%20comparative%20genomics%20and%20ribosome%20binding%20site%20models&amp;author=M.R.%20Hemm&amp;author=B.J.%20Paul&amp;author=T.D.%20Schneider&amp;author=G.%20Storz&amp;author=K.E.%20Rudd&amp;publication_year=2008&amp;journal=Mol.%20Microbiol.&amp;volume=70&amp;pages=1487-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2008.70.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2008.70.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2008.70.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Conway</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Creecy</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Maddox</div> <div class="given-names">S.M.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Unprecedented high-resolution view of bacterial operon architecture revealed by RNA sequencing</div>, <div class="source">MBio</div> , <div class="volume">5</div>, <div class="fpage">e01442</div>–<div class="lpage">14</div><div class="comment">(article number)</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unprecedented%20high-resolution%20view%20of%20bacterial%20operon%20architecture%20revealed%20by%20RNA%20sequencing&amp;author=T.%20Conway&amp;author=J.P.%20Creecy&amp;author=S.M.%20Maddox&amp;publication_year=2014&amp;journal=MBio&amp;volume=5&amp;pages=e01442-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/mBio.01442-14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FmBio.01442-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FmBio.01442-14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25006232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Johnson</div>
               <div class="given-names">J.M.</div>
            </div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shoemaker</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Schadt</div> <div class="given-names">E.E.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Dark matter in the genome: evidence of widespread transcription detected by microarray tiling experiments</div>, <div class="source">Trends Genet.</div> , <div class="volume">21</div>, <div class="fpage">93</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dark%20matter%20in%20the%20genome%3A%20evidence%20of%20widespread%20transcription%20detected%20by%20microarray%20tiling%20experiments&amp;author=J.M.%20Johnson&amp;author=S.%20Edwards&amp;author=D.%20Shoemaker&amp;author=E.E.%20Schadt&amp;publication_year=2005&amp;journal=Trends%20Genet.&amp;volume=21&amp;pages=93-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.12.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.12.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.12.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Dietrich</div> <div class="given-names">F.S.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Identification and characterization of upstream open reading frames (uORF) in the 5’ untranslated regions (UTR) of genes in Saccharomyces cerevisiae</div>, <div class="source">Curr. Genet.</div> , <div class="volume">48</div>, <div class="fpage">77</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20upstream%20open%20reading%20frames%20(uORF)%20in%20the%205%E2%80%99%20untranslated%20regions%20(UTR)%20of%20genes%20in%20Saccharomyces%20cerevisiae&amp;author=Z.%20Zhang&amp;author=F.S.%20Dietrich&amp;publication_year=2005&amp;journal=Curr.%20Genet.&amp;volume=48&amp;pages=77-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00294-005-0001-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00294-005-0001-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00294-005-0001-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16012843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kolter</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Yanofsky</div> <div class="given-names">C.</div></div></div>
          <div class="year">1982</div>, <div class="article-title">Attenuation in amino acid biosynthetic operons</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="volume">16</div>, <div class="fpage">113</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Attenuation%20in%20amino%20acid%20biosynthetic%20operons&amp;author=R.%20Kolter&amp;author=C.%20Yanofsky&amp;publication_year=1982&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=16&amp;pages=113-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.16.120182.000553" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.16.120182.000553" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.16.120182.000553"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6186194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morris</div>
               <div class="given-names">D.R.</div>
            </div>, <div class="name string-name"><div class="surname">Geballe</div> <div class="given-names">A.P.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Upstream open reading frames as regulators of mRNA translation</div>, <div class="source">Mol. Cellular Biol.</div> , <div class="volume">20</div>, <div class="fpage">8635</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Upstream%20open%20reading%20frames%20as%20regulators%20of%20mRNA%20translation&amp;author=D.R.%20Morris&amp;author=A.P.%20Geballe&amp;publication_year=2000&amp;journal=Mol.%20Cellular%20Biol.&amp;volume=20&amp;pages=8635-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.20.23.8635-8642.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.20.23.8635-8642.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.20.23.8635-8642.2000"> </span></div> </div></div></div></div></div><div content-id="DSW008C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kuersten</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Radek</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vogel</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Penalva</div> <div class="given-names">L.O.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Translation regulation gets its ‘omics’ moment</div>, <div class="source">Wiley Interdiscip. Rev. RNA</div> , <div class="volume">4</div>, <div class="fpage">617</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20regulation%20gets%20its%20%E2%80%98omics%E2%80%99%20moment&amp;author=S.%20Kuersten&amp;author=A.%20Radek&amp;author=C.%20Vogel&amp;author=L.O.%20Penalva&amp;publication_year=2013&amp;journal=Wiley%20Interdiscip.%20Rev.%20RNA&amp;volume=4&amp;pages=617-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23677826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW008C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW008C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bahrami-Samani</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Vo</div> <div class="given-names">D.T.</div></div>, <div class="name string-name"><div class="surname">de Araujo</div> <div class="given-names">P.R.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Computational challenges, tools, and resources for analyzing co- and post-transcriptional events in high throughput</div>, <div class="source">Wiley Interdiscip. Rev. RNA</div> , <div class="volume">6</div>, <div class="fpage">291</div>–<div class="lpage">310</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20challenges%2C%20tools%2C%20and%20resources%20for%20analyzing%20co-%20and%20post-transcriptional%20events%20in%20high%20throughput&amp;author=E.%20Bahrami-Samani&amp;author=D.T.%20Vo&amp;author=P.R.%20de%20Araujo&amp;publication_year=2015&amp;journal=Wiley%20Interdiscip.%20Rev.%20RNA&amp;volume=6&amp;pages=291-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/wrna.2015.6.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwrna.2015.6.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fwrna.2015.6.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25515586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745343&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw008/1/dsw008_Supplementary_Data.zip?Expires=2147483647&amp;Signature=YpXyaXE6MvtvlN0Y~EiwWyiWvvVGYP4pCb36XCEdnlfO18LMuysZ4WPJzBn1lIkvLWukMVUtEYdCv9Giuu4VKYn1D2CIG2PM90cSvlOy8A~jh5Oo-to60pA2sBSoMFRRdxId4zIMMDlDdtp5g4dA4iVcWHIW6UYdyqOYYAHhDRF73bSBgjqtBa8HgRC4aHn0saYVaxekcTtPCLOleiBO3UPQKf1ZTAKTaUUwUmeCwfPHZbRSASyVT4rpzltLX6v8W-700CPGDwIgTS6cj08GpsK6YoldAy8zioNBN~GCDgdRRHGhP8yQ7w4H5PhhD5-EPJJJhLqeLKB9jF06hkYiKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">821</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745343"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f3%2f193%2f1745343">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000379741500002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F3%2F193%2F1745343" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745343'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f193%2f1745343">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

