

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/3/261/3052236","@type":"ScholarlyArticle","name":"Circadian oscillations of microbial and functional composition in the human salivary microbiome","datePublished":"2017-02-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx001","keywords":["salivary microbiome","circadian rhythm","16S rRNA gene","oral","human"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/3/261/3052236","author":[{"name":"Takayasu, Lena","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Suda, Wataru","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan   2 Department of Microbiology and Immunology, School of Medicine, Keio University, 35 Shinanomachi, Shinjuku-ku, Tokyo 160-8582, Japan","@type":"Person"},{"name":"Takanashi, Kageyasu","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Iioka, Erica","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Kurokawa, Rina","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Shindo, Chie","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Hattori, Yasue","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Yamashita, Naoko","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Nishijima, Suguru","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Oshima, Kenshiro","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"1 Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan   3 Cooperative Major in Advanced Health Science, Graduate School of Advanced Science and Engineering, Faculty of Science and Engineering, Waseda University, 3-4-1 Okubo, Shinjuku-ku, Tokyo 169-8555, Japan","@type":"Person"}],"description":"Abstract.  The human microbiomes across the body evidently interact with various signals in response to biogeographical physiological conditions. To understand ","pageStart":"261","pageEnd":"270","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Circadian oscillations of microbial and functional composition in the human salivary microbiome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Circadian oscillations of microbial and functional composition in the human salivary microbiome" />
    <meta property="og:description" content="Abstract.  The human microbiomes across the body evidently interact with various signals in response to biogeographical physiological conditions. To understand " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/3/261/3052236" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-02-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takayasu, Lena" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Suda, Wataru" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author_institution" content="Department of Microbiology and Immunology, School of Medicine, Keio University, 35 Shinanomachi, Shinjuku-ku, Tokyo 160-8582, Japan" /><meta name="citation_author" content="Takanashi, Kageyasu" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Iioka, Erica" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Kurokawa, Rina" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Shindo, Chie" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Hattori, Yasue" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Yamashita, Naoko" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Nishijima, Suguru" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Oshima, Kenshiro" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author_institution" content="Cooperative Major in Advanced Health Science, Graduate School of Advanced Science and Engineering, Faculty of Science and Engineering, Waseda University, 3-4-1 Okubo, Shinjuku-ku, Tokyo 169-8555, Japan" /><meta name="citation_title" content="Circadian oscillations of microbial and functional composition in the human salivary microbiome" /><meta name="citation_firstpage" content="261" /><meta name="citation_lastpage" content="270" /><meta name="citation_doi" content="10.1093/dnares/dsx001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2017/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Defining the normal bacterial flora of the oral cavity;  citation_journal_title=J. Clin. Microbiol;  citation_year=2005;  citation_volume=43;  citation_pages=5721-5732; " /><meta name="citation_reference" content="citation_title=Defining the healthy “core microbiome” of oral microbial communities;  citation_journal_title=BMC Microbiol;  citation_year=2009;  citation_volume=9;  citation_pages=259" /><meta name="citation_reference" content="citation_title=Bacterial diversity in saliva and oral health-related conditions: the Hisayama Study;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=22164" /><meta name="citation_reference" content="citation_title=Composition of the adult digestive tract bacterial microbiome based on seven mouth surfaces, tonsils, throat and stool samples;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=R42" /><meta name="citation_reference" content="citation_title=Structure, function and diversity of the healthy human microbiome;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=486;  citation_pages=207-214; " /><meta name="citation_reference" content="citation_title=Oligotyping analysis of the human oral microbiome;  citation_journal_title=Proc. Natl. Acad. Sci. USA.;  citation_year=2014;  citation_volume=111;  citation_pages=E2875-E2884; " /><meta name="citation_reference" content="citation_title=A core human microbiome as viewed through 16S rRNA sequence clusters;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e34242" /><meta name="citation_reference" content="citation_title=Study of inter- and intra-individual variations in the salivary microbiota;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=523" /><meta name="citation_reference" content="citation_title=The salivary microbiome for differentiating individuals: proof of principle;  citation_journal_title=Microbes Infect;  citation_year=2016;  citation_volume=18;  citation_pages=399-405; " /><meta name="citation_reference" content="citation_title=Dynamics and associations of microbial community types across the human body;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=509;  citation_pages=357-360; " /><meta name="citation_reference" content="citation_title=Oral pathobiont induces systemic inflammation and metabolic changes associated with alteration of gut microbiota;  citation_journal_title=Sci. Rep;  citation_year=2014;  citation_volume=4;  citation_pages=04828" /><meta name="citation_reference" content="citation_title=The same microbiota and a potentially discriminant metabolome in the saliva of omnivore, ovo-lacto-vegetarian and Vegan individuals;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e112373" /><meta name="citation_reference" content="citation_title=Nurture trumps nature in a longitudinal survey of salivary bacterial communities in twins from early adolescence to early adulthood;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=2146-2152; " /><meta name="citation_reference" content="citation_title=Global diversity in the human salivary microbiome;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=636-643; " /><meta name="citation_reference" content="citation_title=Same exposure but two radically different responses to antibiotics: resilience of the salivary microbiome versus long-term microbial shifts in feces;  citation_journal_title=mBio;  citation_year=2015;  citation_volume=6;  citation_pages=e01693-e01715; " /><meta name="citation_reference" content="citation_title=Differentiation of salivary bacterial profiles of subjects with periodontitis and dental caries;  citation_journal_title=J. Oral Microbiol;  citation_year=2015;  citation_volume=7;  citation_pages=27429" /><meta name="citation_reference" content="citation_title=The oral mucosal and salivary microbial community of Behcet's syndrome and recurrent aphthous stomatitis;  citation_journal_title=J. Oral Microbiol;  citation_year=2015;  citation_volume=7;  citation_pages=27150" /><meta name="citation_reference" content="citation_title=Variations of oral microbiota are associated with pancreatic diseases including pancreatic cancer;  citation_journal_title=Gut;  citation_year=2012;  citation_volume=61;  citation_pages=582-588; " /><meta name="citation_reference" content="citation_title=Dysbiosis of salivary microbiota in inflammatory bowel disease and its association with oral immunological biomarkers;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=15-25; " /><meta name="citation_reference" content="citation_title=salivary microbiota and metabolome associated with celiac disease;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2014;  citation_volume=80;  citation_pages=3416-3425; " /><meta name="citation_reference" content="citation_title=The oral and gut microbiomes are perturbed in rheumatoid arthritis and partly normalized after treatment;  citation_journal_title=Nat. Med;  citation_year=2015;  citation_volume=21;  citation_pages=895-905; " /><meta name="citation_reference" content="citation_title=Salivary microbiota reflects changes in gut microbiota in cirrhosis with hepatic encephalopathy;  citation_journal_title=Hepatology;  citation_year=2015;  citation_volume=62;  citation_pages=1260-1271; " /><meta name="citation_reference" content="citation_title=The circadian clock in oral health and diseases;  citation_journal_title=J. Dent. Res;  citation_year=2014;  citation_volume=93;  citation_pages=27-35; " /><meta name="citation_reference" content="citation_title=Host lifestyle affects human microbiota on daily timescales;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=R89" /><meta name="citation_reference" content="citation_title=The human salivary microbiome exhibits temporal stability in bacterial diversity;  citation_journal_title=FEMS Microbiol. Ecol;  citation_year=2015;  citation_volume=91;  citation_pages=fiv091" /><meta name="citation_reference" content="citation_title=Temporal stability of the salivary microbiota in oral health;  citation_journal_title=PLoS ONE;  citation_year=2016;  citation_volume=11;  citation_pages=e0147472" /><meta name="citation_reference" content="citation_title=Robustness of gut microbiota of healthy adults in response to probiotic intervention revealed by high-throughput pyrosequencing;  citation_journal_title=DNA Res;  citation_year=2013;  citation_volume=20;  citation_pages=241-253; " /><meta name="citation_reference" content="citation_title=UniFrac: a new phylogenetic method for comparing microbial communities;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2005;  citation_volume=71;  citation_pages=8228-8235; " /><meta name="citation_reference" content="citation_title=Fast UniFrac: facilitating high-throughput phylogenetic analyses of microbial communities including analysis of pyrosequencing and PhyloChip data;  citation_journal_title=ISME J;  citation_year=2010;  citation_volume=4;  citation_pages=17-27; " /><meta name="citation_reference" content="citation_title=MetaGeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes;  citation_journal_title=DNA Res;  citation_year=2008;  citation_volume=15;  citation_pages=387-396; " /><meta name="citation_reference" content="citation_title=The gut microbiome of healthy Japanese and its microbial and functional uniqueness;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=125-133; " /><meta name="citation_reference" content="citation_title=JTK_CYCLE: an efficient nonparametric algorithm for detecting rhythmic components in genome-scale data sets;  citation_journal_title=J. Biol. Rhythms;  citation_year=2010;  citation_volume=25;  citation_pages=372-80; " /><meta name="citation_reference" content="citation_title=Transkingdom control of microbiota diurnal oscillations promotes metabolic homeostasis;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=159;  citation_pages=514-529; " /><meta name="citation_reference" content="citation_title=Inter-Individual differences in the oral bacteriome are greater than intra-day fluctuations in individuals;  citation_journal_title=PLoS ONE;  citation_year=2015;  citation_volume=10;  citation_pages=e0131607" /><meta name="citation_reference" content="citation_title=Rhythmicity of the intestinal microbiota is regulated by gender and the host circadian clock;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=10479-10484; " /><meta name="citation_reference" content="citation_title=The human circadian metabolome;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=2625-2629; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/3/261/17743972/dsx001.pdf" /><meta name="description" content="Abstract.  The human microbiomes across the body evidently interact with various signals in response to biogeographical physiological conditions. To understand " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/3/261/3052236" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/3/261/3052236" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/3/261/3052236");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx001';
    </script>
<title>
	Circadian oscillations of microbial and functional composition in the human salivary microbiome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3052236" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZNuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWtheWFzdSwgTGVuYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgTWV0YWdlbm9taWNzLCBEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSwgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1ZGEsIFdhdGFydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgTWV0YWdlbm9taWNzLCBEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSwgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5IGFuZCBJbW11bm9sb2d5LCBTY2hvb2wgb2YgTWVkaWNpbmUsIEtlaW8gVW5pdmVyc2l0eSwgMzUgU2hpbmFub21hY2hpLCBTaGluanVrdS1rdSwgVG9reW8gMTYwLTg1ODIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYW5hc2hpLCBLYWdleWFzdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgTWV0YWdlbm9taWNzLCBEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSwgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iklpb2thLCBFcmljYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgTWV0YWdlbm9taWNzLCBEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSwgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1cm9rYXdhLCBSaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBNZXRhZ2Vub21pY3MsIERlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5IGFuZCBNZWRpY2FsIFNjaWVuY2UsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01LCBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbmRvLCBDaGllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBNZXRhZ2Vub21pY3MsIERlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5IGFuZCBNZWRpY2FsIFNjaWVuY2UsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01LCBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF0dG9yaSwgWWFzdWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1ldGFnZW5vbWljcywgRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3kgYW5kIE1lZGljYWwgU2NpZW5jZSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUsIEthc2hpd2Fub2hhLCBLYXNoaXdhLCBDaGliYSAyNzctODU2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hc2hpdGEsIE5hb2tvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBNZXRhZ2Vub21pY3MsIERlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5IGFuZCBNZWRpY2FsIFNjaWVuY2UsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01LCBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmlzaGlqaW1hLCBTdWd1cnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1ldGFnZW5vbWljcywgRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3kgYW5kIE1lZGljYWwgU2NpZW5jZSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUsIEthc2hpd2Fub2hhLCBLYXNoaXdhLCBDaGliYSAyNzctODU2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIEtlbnNoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBNZXRhZ2Vub21pY3MsIERlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5IGFuZCBNZWRpY2FsIFNjaWVuY2UsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01LCBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF0dG9yaSwgTWFzYWhpcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1ldGFnZW5vbWljcywgRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3kgYW5kIE1lZGljYWwgU2NpZW5jZSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUsIEthc2hpd2Fub2hhLCBLYXNoaXdhLCBDaGliYSAyNzctODU2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb29wZXJhdGl2ZSBNYWpvciBpbiBBZHZhbmNlZCBIZWFsdGggU2NpZW5jZSwgR3JhZHVhdGUgU2Nob29sIG9mIEFkdmFuY2VkIFNjaWVuY2UgYW5kIEVuZ2luZWVyaW5nLCBGYWN1bHR5IG9mIFNjaWVuY2UgYW5kIEVuZ2luZWVyaW5nLCBXYXNlZGEgVW5pdmVyc2l0eSwgMy00LTEgT2t1Ym8sIFNoaW5qdWt1LWt1LCBUb2t5byAxNjktODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNpcmNhZGlhbiBvc2NpbGxhdGlvbnMgb2YgbWljcm9iaWFsIGFuZCBmdW5jdGlvbmFsIGNvbXBvc2l0aW9uIGluIHRoZSBodW1hbiBzYWxpdmFyeSBtaWNyb2Jpb21lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjYxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmluaW5nIHRoZSBub3JtYWwgYmFjdGVyaWFsIGZsb3JhIG9mIHRoZSBvcmFsIGNhdml0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2xpbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NTcyMS01NzMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWZpbmluZyB0aGUgaGVhbHRoeSDigJxjb3JlIG1pY3JvYmlvbWXigJ0gb2Ygb3JhbCBtaWNyb2JpYWwgY29tbXVuaXRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCBkaXZlcnNpdHkgaW4gc2FsaXZhIGFuZCBvcmFsIGhlYWx0aC1yZWxhdGVkIGNvbmRpdGlvbnM6IHRoZSBIaXNheWFtYSBTdHVkeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIyMTY0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcG9zaXRpb24gb2YgdGhlIGFkdWx0IGRpZ2VzdGl2ZSB0cmFjdCBiYWN0ZXJpYWwgbWljcm9iaW9tZSBiYXNlZCBvbiBzZXZlbiBtb3V0aCBzdXJmYWNlcywgdG9uc2lscywgdGhyb2F0IGFuZCBzdG9vbCBzYW1wbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPVI0MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZSwgZnVuY3Rpb24gYW5kIGRpdmVyc2l0eSBvZiB0aGUgaGVhbHRoeSBodW1hbiBtaWNyb2Jpb21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg2OyAgY2l0YXRpb25fcGFnZXM9MjA3LTIxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T2xpZ290eXBpbmcgYW5hbHlzaXMgb2YgdGhlIGh1bWFuIG9yYWwgbWljcm9iaW9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0EuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPUUyODc1LUUyODg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvcmUgaHVtYW4gbWljcm9iaW9tZSBhcyB2aWV3ZWQgdGhyb3VnaCAxNlMgclJOQSBzZXF1ZW5jZSBjbHVzdGVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzNDI0MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0dWR5IG9mIGludGVyLSBhbmQgaW50cmEtaW5kaXZpZHVhbCB2YXJpYXRpb25zIGluIHRoZSBzYWxpdmFyeSBtaWNyb2Jpb3RhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz01MjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2FsaXZhcnkgbWljcm9iaW9tZSBmb3IgZGlmZmVyZW50aWF0aW5nIGluZGl2aWR1YWxzOiBwcm9vZiBvZiBwcmluY2lwbGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmVzIEluZmVjdDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTM5OS00MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWNzIGFuZCBhc3NvY2lhdGlvbnMgb2YgbWljcm9iaWFsIGNvbW11bml0eSB0eXBlcyBhY3Jvc3MgdGhlIGh1bWFuIGJvZHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MDk7ICBjaXRhdGlvbl9wYWdlcz0zNTctMzYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmFsIHBhdGhvYmlvbnQgaW5kdWNlcyBzeXN0ZW1pYyBpbmZsYW1tYXRpb24gYW5kIG1ldGFib2xpYyBjaGFuZ2VzIGFzc29jaWF0ZWQgd2l0aCBhbHRlcmF0aW9uIG9mIGd1dCBtaWNyb2Jpb3RhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MDQ4MjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2FtZSBtaWNyb2Jpb3RhIGFuZCBhIHBvdGVudGlhbGx5IGRpc2NyaW1pbmFudCBtZXRhYm9sb21lIGluIHRoZSBzYWxpdmEgb2Ygb21uaXZvcmUsIG92by1sYWN0by12ZWdldGFyaWFuIGFuZCBWZWdhbiBpbmRpdmlkdWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMTIzNzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdXJ0dXJlIHRydW1wcyBuYXR1cmUgaW4gYSBsb25naXR1ZGluYWwgc3VydmV5IG9mIHNhbGl2YXJ5IGJhY3RlcmlhbCBjb21tdW5pdGllcyBpbiB0d2lucyBmcm9tIGVhcmx5IGFkb2xlc2NlbmNlIHRvIGVhcmx5IGFkdWx0aG9vZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTIxNDYtMjE1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGRpdmVyc2l0eSBpbiB0aGUgaHVtYW4gc2FsaXZhcnkgbWljcm9iaW9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTYzNi02NDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbWUgZXhwb3N1cmUgYnV0IHR3byByYWRpY2FsbHkgZGlmZmVyZW50IHJlc3BvbnNlcyB0byBhbnRpYmlvdGljczogcmVzaWxpZW5jZSBvZiB0aGUgc2FsaXZhcnkgbWljcm9iaW9tZSB2ZXJzdXMgbG9uZy10ZXJtIG1pY3JvYmlhbCBzaGlmdHMgaW4gZmVjZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPW1CaW87ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUwMTY5My1lMDE3MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhdGlvbiBvZiBzYWxpdmFyeSBiYWN0ZXJpYWwgcHJvZmlsZXMgb2Ygc3ViamVjdHMgd2l0aCBwZXJpb2RvbnRpdGlzIGFuZCBkZW50YWwgY2FyaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBPcmFsIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9Mjc0MjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JhbCBtdWNvc2FsIGFuZCBzYWxpdmFyeSBtaWNyb2JpYWwgY29tbXVuaXR5IG9mIEJlaGNldCdzIHN5bmRyb21lIGFuZCByZWN1cnJlbnQgYXBodGhvdXMgc3RvbWF0aXRpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gT3JhbCBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTI3MTUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWF0aW9ucyBvZiBvcmFsIG1pY3JvYmlvdGEgYXJlIGFzc29jaWF0ZWQgd2l0aCBwYW5jcmVhdGljIGRpc2Vhc2VzIGluY2x1ZGluZyBwYW5jcmVhdGljIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R3V0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9NTgyLTU4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHlzYmlvc2lzIG9mIHNhbGl2YXJ5IG1pY3JvYmlvdGEgaW4gaW5mbGFtbWF0b3J5IGJvd2VsIGRpc2Vhc2UgYW5kIGl0cyBhc3NvY2lhdGlvbiB3aXRoIG9yYWwgaW1tdW5vbG9naWNhbCBiaW9tYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTUtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXNhbGl2YXJ5IG1pY3JvYmlvdGEgYW5kIG1ldGFib2xvbWUgYXNzb2NpYXRlZCB3aXRoIGNlbGlhYyBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0zNDE2LTM0MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBvcmFsIGFuZCBndXQgbWljcm9iaW9tZXMgYXJlIHBlcnR1cmJlZCBpbiByaGV1bWF0b2lkIGFydGhyaXRpcyBhbmQgcGFydGx5IG5vcm1hbGl6ZWQgYWZ0ZXIgdHJlYXRtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1lZDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTg5NS05MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbGl2YXJ5IG1pY3JvYmlvdGEgcmVmbGVjdHMgY2hhbmdlcyBpbiBndXQgbWljcm9iaW90YSBpbiBjaXJyaG9zaXMgd2l0aCBoZXBhdGljIGVuY2VwaGFsb3BhdGh5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IZXBhdG9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9MTI2MC0xMjcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY2lyY2FkaWFuIGNsb2NrIGluIG9yYWwgaGVhbHRoIGFuZCBkaXNlYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRGVudC4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9MjctMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvc3QgbGlmZXN0eWxlIGFmZmVjdHMgaHVtYW4gbWljcm9iaW90YSBvbiBkYWlseSB0aW1lc2NhbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPVI4OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiBzYWxpdmFyeSBtaWNyb2Jpb21lIGV4aGliaXRzIHRlbXBvcmFsIHN0YWJpbGl0eSBpbiBiYWN0ZXJpYWwgZGl2ZXJzaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPWZpdjA5MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbXBvcmFsIHN0YWJpbGl0eSBvZiB0aGUgc2FsaXZhcnkgbWljcm9iaW90YSBpbiBvcmFsIGhlYWx0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1lMDE0NzQ3MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvYnVzdG5lc3Mgb2YgZ3V0IG1pY3JvYmlvdGEgb2YgaGVhbHRoeSBhZHVsdHMgaW4gcmVzcG9uc2UgdG8gcHJvYmlvdGljIGludGVydmVudGlvbiByZXZlYWxlZCBieSBoaWdoLXRocm91Z2hwdXQgcHlyb3NlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yNDEtMjUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlGcmFjOiBhIG5ldyBwaHlsb2dlbmV0aWMgbWV0aG9kIGZvciBjb21wYXJpbmcgbWljcm9iaWFsIGNvbW11bml0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz04MjI4LTgyMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgVW5pRnJhYzogZmFjaWxpdGF0aW5nIGhpZ2gtdGhyb3VnaHB1dCBwaHlsb2dlbmV0aWMgYW5hbHlzZXMgb2YgbWljcm9iaWFsIGNvbW11bml0aWVzIGluY2x1ZGluZyBhbmFseXNpcyBvZiBweXJvc2VxdWVuY2luZyBhbmQgUGh5bG9DaGlwIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlTTUUgSjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTctMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFHZW5lQW5ub3RhdG9yOiBkZXRlY3Rpbmcgc3BlY2llcy1zcGVjaWZpYyBwYXR0ZXJucyBvZiByaWJvc29tYWwgYmluZGluZyBzaXRlIGZvciBwcmVjaXNlIGdlbmUgcHJlZGljdGlvbiBpbiBhbm9ueW1vdXMgcHJva2FyeW90aWMgYW5kIHBoYWdlIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0zODctMzk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3V0IG1pY3JvYmlvbWUgb2YgaGVhbHRoeSBKYXBhbmVzZSBhbmQgaXRzIG1pY3JvYmlhbCBhbmQgZnVuY3Rpb25hbCB1bmlxdWVuZXNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTI1LTEzMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SlRLX0NZQ0xFOiBhbiBlZmZpY2llbnQgbm9ucGFyYW1ldHJpYyBhbGdvcml0aG0gZm9yIGRldGVjdGluZyByaHl0aG1pYyBjb21wb25lbnRzIGluIGdlbm9tZS1zY2FsZSBkYXRhIHNldHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIFJoeXRobXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zNzItODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5za2luZ2RvbSBjb250cm9sIG9mIG1pY3JvYmlvdGEgZGl1cm5hbCBvc2NpbGxhdGlvbnMgcHJvbW90ZXMgbWV0YWJvbGljIGhvbWVvc3Rhc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OTsgIGNpdGF0aW9uX3BhZ2VzPTUxNC01Mjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyLUluZGl2aWR1YWwgZGlmZmVyZW5jZXMgaW4gdGhlIG9yYWwgYmFjdGVyaW9tZSBhcmUgZ3JlYXRlciB0aGFuIGludHJhLWRheSBmbHVjdHVhdGlvbnMgaW4gaW5kaXZpZHVhbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTAxMzE2MDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaHl0aG1pY2l0eSBvZiB0aGUgaW50ZXN0aW5hbCBtaWNyb2Jpb3RhIGlzIHJlZ3VsYXRlZCBieSBnZW5kZXIgYW5kIHRoZSBob3N0IGNpcmNhZGlhbiBjbG9jazsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9MTA0NzktMTA0ODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiBjaXJjYWRpYW4gbWV0YWJvbG9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MjYyNS0yNjI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvMy8yNjEvMTc3NDM5NzIvZHN4MDAxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgaHVtYW4gbWljcm9iaW9tZXMgYWNyb3NzIHRoZSBib2R5IGV2aWRlbnRseSBpbnRlcmFjdCB3aXRoIHZhcmlvdXMgc2lnbmFscyBpbiByZXNwb25zZSB0byBiaW9nZW9ncmFwaGljYWwgcGh5c2lvbG9naWNhbCBjb25kaXRpb25zLiBUbyB1bmRlcnN0YW5kICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzA1MjIzNiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQOoR2tzBnHGTlDn+EBBTcbE4/KMA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f261%2f3052236" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f261%2f3052236" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&amp;Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/3">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="88473582">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473582">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473584">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473584">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473589">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473589">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473632">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473632">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473654">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473654">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473661">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473661">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473673">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473673">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/3/251/3058507">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/3/271/2965793">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Circadian oscillations of microbial and functional composition in the human salivary microbiome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lena Takayasu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lena+Takayasu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takayasu L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takayasu L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Lena Takayasu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wataru Suda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div><div class="aff"><div class="label -label">2</div>Department of Microbiology and Immunology, School of Medicine, Keio University, 35 Shinanomachi, Shinjuku-ku, Tokyo 160-8582, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wataru+Suda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suda W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suda W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wataru Suda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kageyasu Takanashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kageyasu+Takanashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takanashi K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takanashi K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kageyasu Takanashi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Erica Iioka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Erica+Iioka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iioka E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iioka E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Erica Iioka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rina Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rina+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Rina Kurokawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chie Shindo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chie+Shindo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shindo C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shindo C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chie Shindo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasue Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasue+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasue Hattori</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoko Yamashita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoko+Yamashita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashita N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamashita N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Naoko Yamashita</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suguru Nishijima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suguru+Nishijima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishijima S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishijima S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Suguru Nishijima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenshiro Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenshiro+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenshiro Oshima</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Metagenomics, Department of Computational Biology and Medical Science, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5, Kashiwanoha, Kashiwa, Chiba 277-8561, Japan</div><div class="aff"><div class="label -label">3</div>Cooperative Major in Advanced Health Science, Graduate School of Advanced Science and Engineering, Faculty of Science and Engineering, Waseda University, 3-4-1 Okubo, Shinjuku-ku, Tokyo 169-8555, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx001-cor1" data-open="dsx001-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 4 7136 4070. Fax. +81 4 7136 4070. Email: <a href="mailto:m-hattori@aoni.waseda.jp" target="_blank">m-hattori@aoni.waseda.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx001-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 261–270, <a href='https://doi.org/10.1093/dnares/dsx001'>https://doi.org/10.1093/dnares/dsx001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 February 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 December 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">17 January 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">20 January 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3052236" href="/dnaresearch/article-pdf/24/3/261/17743972/dsx001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Lena Takayasu, Wataru Suda, Kageyasu Takanashi, Erica Iioka, Rina Kurokawa, Chie Shindo, Yasue Hattori, Naoko Yamashita, Suguru Nishijima, Kenshiro Oshima, Masahira Hattori;  Circadian oscillations of microbial and functional composition in the human salivary microbiome, <em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 261–270, <a href='https://doi.org/10.1093/dnares/dsx001'>https://doi.org/10.1093/dnares/dsx001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3052236?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3052236?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3052236?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3052236?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3052236?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3052236?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3052236?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Circadian%20oscillations%20of%20microbial%20and%20functional%20composition%20in%20the%20human%20salivary%20microbiome&amp;publicationDate=2017-02-23&amp;volumeNum=24&amp;issueNum=3&amp;author=Takayasu%2C%20Lena%3B%20Suda%2C%20Wataru&amp;startPage=261&amp;endPage=270&amp;contentId=10.1093%2Fdnares%2Fdsx001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f261%2f3052236" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=88473582 id="88473582" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The human microbiomes across the body evidently interact with various signals in response to biogeographical physiological conditions. To understand such interactions in detail, we investigated how the salivary microbiome in the oral cavity would be regulated by host-related signals. Here, we show that the microbial abundance and gene participating in keeping the human salivary microbiome exhibit global circadian rhythm. Analysis of the 16S rRNA sequences of salivary microbial samples of six healthy adults collected at 4-h intervals for three days revealed that the microbial genera accounting for 68.4–89.6% of the total abundance were observed to significantly oscillate with the periodicity of ∼24 h. These oscillation patterns showed high variations amongst individuals, and the extent of circadian variations in individuals was generally lower than that of interindividual variations. Of the microbial categories oscillated, those classified by aerobic/anaerobic growth and Gram staining, Firmicutes including <em>Streptococcus</em> and <em>Gemella</em>, and Bacteroidetes including <em>Prevotella</em> showed high association with the circadian oscillation. The circadian oscillation was completely abolished by incubating the saliva <em>in vitro</em>, suggesting that host’s physiological changes mostly contributed to the microbial oscillation. Further metagenomic analysis showed that circadian oscillation enriched the functions of environmental responses such as various transporters and two-component regulatory systems in the evening, and those of metabolisms such as the biosynthesis of vitamins and fatty acids in the morning.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;salivary microbiome&quot;">salivary microbiome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;circadian rhythm&quot;">circadian rhythm</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;16S rRNA gene&quot;">16S rRNA gene</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="oral">oral</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="human">human</a></div>                            <h2 scrollto-destination=88473584 id="88473584" class="section-title" >1. Introduction</h2>
<p>The salivary microbiome is composed of ∼700 species<a reveal-id="dsx001-B1" data-open="dsx001-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> mostly belonging to the five phyla, Firmicutes, Proteobacteria, Actinobacteria, Bacteroidetes, and Fusobacteria,<a reveal-id="dsx001-B2" data-open="dsx001-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx001-B3" data-open="dsx001-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> similar to other oral microbiomes in the tongue, throat, and tonsil but relatively distinct from the plaque microbiome.<a reveal-id="dsx001-B4 dsx001-B5 dsx001-B6" data-open="dsx001-B4 dsx001-B5 dsx001-B6" class="link link-ref link-reveal xref-bibr"><sup>4–6</sup></a> The salivary microbiomes can be characterized by a significantly lower interindividual variability than microbial communities of other body sites such as the gut,<a reveal-id="dsx001-B7" data-open="dsx001-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx001-B8" data-open="dsx001-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> but it is possible to differentiate individuals by phylogenic approaches.<a reveal-id="dsx001-B9" data-open="dsx001-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Though the salivary microbiome is associated with gut microbiome,<a reveal-id="dsx001-B10" data-open="dsx001-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx001-B11" data-open="dsx001-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> the diversity in salivary microbiomes showed relatively low associations with several affecting factors including long-term diet,<a reveal-id="dsx001-B12" data-open="dsx001-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> weight and gender,<a reveal-id="dsx001-B13" data-open="dsx001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> geographic location,<a reveal-id="dsx001-B14" data-open="dsx001-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and antibiotic treatment (high tolerance)<a reveal-id="dsx001-B15" data-open="dsx001-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> compared with that of the gut microbiome. These apparent discrepancies between the salivary microbiome in the oral cavity and the gut microbiome in the intestine suggest the existence of different regulations and responses between them.</p><p>On the other hand, several studies reported the association of the salivary microbiome with not only oral manifestations such as periodontitis and dental caries,<a reveal-id="dsx001-B16" data-open="dsx001-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Behcet’s syndrome and aphthous stomatitis,<a reveal-id="dsx001-B17" data-open="dsx001-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> but also with several systemic diseases including pancreatic cancer,<a reveal-id="dsx001-B18" data-open="dsx001-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> inflammatory bowel disease (IBD),<a reveal-id="dsx001-B19" data-open="dsx001-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> celiac disease,<a reveal-id="dsx001-B20" data-open="dsx001-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> rheumatoid arthritis,<a reveal-id="dsx001-B21" data-open="dsx001-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and hepatic cirrhosis.<a reveal-id="dsx001-B22" data-open="dsx001-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> These reports suggest that physiological changes at the distal lesion sites largely contribute to the alteration of salivary microbiome, and that the salivary microbiome appears to be more sensitive to host-related rather than environment-related factors in comparison with the gut microbiome.</p><p>In this context, it is conceivable that the oral circadian rhythm as a host-related diurnal signal may affect the salivary microbiome in addition to salivary gland and epithelial homeostasis in the oral cavity.<a reveal-id="dsx001-B23" data-open="dsx001-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> There have been several researches on the analysis of changes or stability in longitudinal salivary samples collected daily, monthly, or yearly.<a reveal-id="dsx001-B8" data-open="dsx001-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx001-B13" data-open="dsx001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx001-B24" data-open="dsx001-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx001-B25" data-open="dsx001-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> These studies showed the tendency of high stability of the human salivary microbiome over time with low intra-subject variations. Some of the papers also showed more shared species and lower taxonomic changes among the individual salivary than gut microbiomes.<a reveal-id="dsx001-B13" data-open="dsx001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx001-B24" data-open="dsx001-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Unlike the gut microbiome, these observations further suggested that host-related internal signals had more impact on the salivary microbiome than external signals related to individual lifestyles. However, more recently, the individual salivary microbiomes collected with 4-h intervals in a day was reported to show little or no change in the microbial profiles.<a reveal-id="dsx001-B26" data-open="dsx001-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p><p>In this study, we examined diurnal changes in salivary microbiomes longitudinally collected from six healthy subjects with 4-h intervals for 3 days by 16S rRNA gene and metagenomic sequencing of the salivary DNA. Our data revealed that the majority of salivary microbes exhibited the circadian rhythm profiles with ∼24-h periodicity.</p>                            <h2 scrollto-destination=88473589 id="88473589" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=88473590 id="88473590" class="section-title" >2.1. Subjects and saliva sample collection</h3>
<p>Six healthy volunteers were recruited. All subjects were informed of the purpose of this study. This study was approved by the ethical committees of the University of Tokyo and Azabu University, and written consent was obtained from all subjects. Fresh unstimulated saliva (∼1 ml) were collected every 4 h at 12:00, 16:00, 20:00 in the first day, at 0:00, 4:00, 8:00, 12:00, 16:00, 20:00 in the second and third days, and at 0:00, 4:00, 8:00 in the fourth day, accounting for a total of 18 samples for each subject for 3 days (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). All subjects took a sleep from 0:00 to 8:00, and shortly awakened at 4:00 for sampling. The sampling of saliva was carried out at least 2 h after meal or toothbrushing. Collected saliva samples were kept at 4°C, frozen in liquid nitrogen within 24 h after sampling, and stored at -80°C until use for DNA preparation.</p>                            <h3 scrollto-destination=88473592 id="88473592" class="section-title" >2.2. DNA preparation and sequencing</h3>
<p>Microbial DNA was prepared from saliva samples according to the method described previously.<a reveal-id="dsx001-B19" data-open="dsx001-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> The 16S rRNA gene hypervariable V1–V2 regions were PCR-amplified using universal primers (a barcoded 27Fmod and 338R) and, multiplexed amplicon pyrosequencing was carried out using 454 GS FLX Titanium or 454 GS JUNIOR (Roche Applied Science).</p><p>To obtain metagenomic sequences, we performed the whole-genome shotgun sequencing with the MiSeq (<a reveal-id="dsx001-B2" data-open="dsx001-B2" class="link link-ref link-reveal xref-bibr">2</a> × 300 paired-end run) and IonProton (200-base reads with the Proton I chip) platforms according to the manufacturer’s protocol. Statistics of the sequencing was summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1 and S2</a></span>, respectively.</p>                            <h3 scrollto-destination=88473595 id="88473595" class="section-title" >2.3. Quantitative PCR (qPCR)</h3>
<p>The universal bacterial 16S rRNA primers, 27Fmod (5ʹ-agrgtttgatymtggctcag-3ʹ) and 338R (5ʹ-tgctgcctcccgtaggagt-3ʹ) were used to estimate the microbial cell number by real-time qPCR using <em>E</em><em>scherichia</em><em>coli</em> 16S rRNA gene sequence as a standard.</p>                            <h3 scrollto-destination=88473597 id="88473597" class="section-title" >2.4. Data processing of 16S sequences</h3>
<p>The 16S V1–V2 sequences were assigned to samples on the basis of their barcode sequence. Reads with an average quality value (QV) &lt; 25 and those lacking the primer sequences at both ends were filtered off. Possibly chimeric sequences that had alignment lengths of &lt; 90% coverage with authentic reference 16S sequences in the database (described below) were also removed. Filter-passed reads obtained from these quality-check processes accounted for ∼50% of the total reads. The majority of the removed reads represented those lacking PCR primer sequences. After trimming both primer sequences from the filter-passed reads, 3,000 reads per sample were randomly selected and used for further analyses.<a reveal-id="dsx001-B27" data-open="dsx001-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The high-quality reads were sorted according to the QV<a reveal-id="dsx001-B19" data-open="dsx001-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and then clustered into operational taxonomic units (OTUs) using a 96% pairwise-identity cutoff with the UCLUST program v.5.2.32 (<a class="link link-uri" href="http://drive5.com/usearch/manual/uclust_algo.html" target="_blank">http://drive5.com/usearch/manual/uclust_algo.html</a> (31 January 2017, date last accessed)). The representative sequences of the generated OTUs were blasted to our own 16S database constructed in this study by GLSEARCH program to determine the closest taxa. The 16S database was constructed from three publically available databases: Ribosomal Database Project (RDP) v.10.31, CORE (<a class="link link-uri" href="http://microbiome.osu.edu/" target="_blank">http://microbiome.osu.edu/</a>(31 January 2017, date last accessed)), and the reference genome sequence database obtained from the NCBI FTP site (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genbank/" target="_blank">ftp://ftp.ncbi.nih.gov/genbank/</a> (December 2011, date last accessed)). For assignment of OTUs to the taxa at the phylum, genus and species level, sequence similarity thresholds of 70, 94, and 96% were applied. The results of the taxonomic assignment of the OTUs were summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. UniFrac distance<a reveal-id="dsx001-B28" data-open="dsx001-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsx001-B29" data-open="dsx001-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> was calculated to assess the dissimilarity (distance) between each sample.</p>                            <h3 scrollto-destination=88473599 id="88473599" class="section-title" >2.5. Data processing of metagenomic sequences</h3>
<p>In Ion Proton reads, artificially duplicated sequences were removed using replicate filter if any sequences have a similarity of ≥95% identity to other sequences with identical starting position. Human-derived reads were removed by mapping them to the human genome (HG19) with Newbler. Finally, low-quality reads with average QV &lt; 20 or &lt; 75 bp were removed. For MiSeq reads, low quality bases (QV &lt; 20) at the 5ʹ end of reads were trimmed. Reads in which more than half of bases are QV &lt; 20 and reads of which the length was &lt; 50 bp were filtered out. These procedures were done using FASTX-Tool kit. The filter-passed reads were mapped to the human genome using bowtie2 and the mapped reads were also removed. Reads derived from phiX were also mapped to the phiX genome and the mapped reads were removed.</p><p>The high-quality reads were then assembled into contigs by using the Newbler algorithm (v.1.1.03, provided with the GS FLX sequencer). Gene prediction was performed using MetaGeneAnnotator<a reveal-id="dsx001-B30" data-open="dsx001-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> with the –<em>m</em> option for multiple species. The nucleotide sequences of predicted genes were clustered using cd-hit-est v.4.6 with a ≥95% sequence identity and ≥90% coverage. The clustered gene sequences were then translated to proteins using the NCBI translation table 11. Functional annotation of the predicted proteins was performed using BLASTP v.2.2.25 against KEGG database with an <em>E</em>-value 1e-5. We mapped 100 K high-quality reads onto the non-redundant gene set using bowtie2 to obtain the quantified functional composition in each sample. Also, we quantitatively evaluated the microbial composition by mapping 100 K high-quality reads onto the reference genomes under the conditions described previously.<a reveal-id="dsx001-B31" data-open="dsx001-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a></p>                            <h3 scrollto-destination=88473602 id="88473602" class="section-title" >2.6. Periodicity discrimination method (PDM)</h3>
<p>In this study, we developed a PDM for evaluation of the periodicity of oscillation patterns of the relative abundance of given taxa and microbial categories.</p><div class="block-child-p">The microbial relative abundance data at sampling times in a day (12:00, 16:00, 20:00, 0:00, 4:00, 8:00) was normalized as follows. <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>d</mi><mrow><mi>n</mi><mi>o</mi><mi>r</mi><mi>m</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mi>d</mi><mi>n</mi></msub><mo>-</mo><msub><mi>d</mi><mrow><mi>m</mi><mi>i</mi><mi>n</mi></mrow></msub></mrow><mrow><msub><mi>d</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi><mo>-</mo></mrow></msub><msub><mi>d</mi><mrow><mi>m</mi><mi>i</mi><mi>n</mi></mrow></msub></mrow></mfrac></mrow></math></div></div></div><em>d<sub>n</sub></em> denotes each value of the day, <em>d<sub>norm</sub></em> denotes the normalized value, <em>d<sub>min</sub></em> denotes the minimum value of the day and <em>d<sub>max</sub></em> denotes the maximum value of the day. Normalized values of each day were all in the interval [0, 1].</div><p>To figure out the periodicity of a given taxon <em>X</em> in a subject, the normalized abundance data of every 4 h for 3 days were prepared as a list and the average normalized abundance of each time zone was calculated as shown below.</p><p><div content-id="T1" class="table-modal"><div class="table-wrap"><div id="T1" data-id="T1" class="table-wrap-title"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>12:00 </th><th>16:00 </th><th>20:00 </th><th>0:00 </th><th>4:00 </th><th>8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div><div content-id="T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>12:00 </th><th>16:00 </th><th>20:00 </th><th>0:00 </th><th>4:00 </th><th>8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="T1" data-id="T1" class="table-wrap-title"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>12:00 </th><th>16:00 </th><th>20:00 </th><th>0:00 </th><th>4:00 </th><th>8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div><div content-id="T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>12:00 </th><th>16:00 </th><th>20:00 </th><th>0:00 </th><th>4:00 </th><th>8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div></div></p><p>Then, the list was sorted with time zones (column vector in the shown table) by the average normalized abundance of each time zone. For example, when the average normalized abundance lined as follows; <em>X</em><sub>2</sub> &gt; <em>X</em><sub>4</sub> &gt; <em>X</em><sub>5</sub> &gt; <em>X</em><sub>3</sub> &gt; <em>X</em><sub>1</sub> &gt; <em>X</em><sub>6</sub>, the sorted list would be lined as shown below.</p><p><div content-id="T2" class="table-modal"><div class="table-wrap"><div id="T2" data-id="T2" class="table-wrap-title"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>TimeZone1 16:00 </th><th>TimeZone2 0:00 </th><th>TimeZone3 4:00 </th><th>TimeZone4 20:00 </th><th>TimeZone5 12:00 </th><th>TimeZone6 8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div><div content-id="T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>TimeZone1 16:00 </th><th>TimeZone2 0:00 </th><th>TimeZone3 4:00 </th><th>TimeZone4 20:00 </th><th>TimeZone5 12:00 </th><th>TimeZone6 8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="T2" data-id="T2" class="table-wrap-title"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>TimeZone1 16:00 </th><th>TimeZone2 0:00 </th><th>TimeZone3 4:00 </th><th>TimeZone4 20:00 </th><th>TimeZone5 12:00 </th><th>TimeZone6 8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div><div content-id="T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>TimeZone1 16:00 </th><th>TimeZone2 0:00 </th><th>TimeZone3 4:00 </th><th>TimeZone4 20:00 </th><th>TimeZone5 12:00 </th><th>TimeZone6 8:00 </th></tr></thead><tbody><tr><td>Day1 </td><td><em>x</em><sub>12</sub> </td><td><em>x</em><sub>14</sub> </td><td><em>x</em><sub>15</sub> </td><td><em>x</em><sub>13</sub> </td><td><em>x</em><sub>11</sub> </td><td><em>x</em><sub>16</sub> </td></tr><tr><td>Day2 </td><td><em>x</em><sub>22</sub> </td><td><em>x</em><sub>24</sub> </td><td><em>x</em><sub>25</sub> </td><td><em>x</em><sub>23</sub> </td><td><em>x</em><sub>21</sub> </td><td><em>x</em><sub>26</sub> </td></tr><tr><td>Day3 </td><td><em>x</em><sub>32</sub> </td><td><em>x</em><sub>34</sub> </td><td><em>x</em><sub>35</sub> </td><td><em>x</em><sub>33</sub> </td><td><em>x</em><sub>31</sub> </td><td><em>x</em><sub>36</sub> </td></tr><tr><td>Average </td><td><em>X</em><sub>2</sub> </td><td><em>X</em><sub>4</sub> </td><td><em>X</em><sub>5</sub> </td><td><em>X</em><sub>3</sub> </td><td><em>X</em><sub>1</sub> </td><td><em>X</em><sub>6</sub> </td></tr></tbody></table></div></div></p><p>This sorting operation is especially useful to reduce the calculation resources when the observed time points are further large, so that combination explosion can be avoided for selecting all possible combination of TimeZones.</p><p>Then, sorted time zones in the sorted list are grouped as a combination of two separated ‘TimeZoneCombi’ which discern the deviation of the abundance between the two time zones. By using the following algorithm, the best <em>P</em>-value for a taxon and the TimeZoneCombi can be obtained.</p><p>BestPvalue ← 1</p><p>FOR each TimeZoneCombi</p><p> BestPvalueOfTimeZone ← 1</p><p> FOR each Threshold = each data of taxa <em>X</em></p><p>Pvalue ← CALL FisherExactTest WITH taxa <em>X</em> data in time zones of TimeZoneCombi, taxa <em>X</em> data of other time zones AND Threshold</p><p>  IF Pvalue &lt; BestPvalueOfTimeZone THEN</p><p>   BestPvalueOfTimeZone ← Pvalue</p><p>   BestThresholdOfTimeZone ← Threshold</p><p>  END IF</p><p> END FOR</p><p> IF BestPvalueOfTimeZone &lt; BestPvalue THEN</p><p>  BestPvalue ← BestPvalueOfTimeZone</p><p>  BestThreshold ← BestThresholdOfTimeZone</p><p>  BestTimeZone ← TimeZoneCombi</p><p> END IF</p><p>END FOR</p><p>In the subprocedure ‘FisherExactTest’, the <em>P</em> value from the test is computed according to the 2 × 2 contingency table shown below.</p><p><div content-id="T3" class="table-modal"><div class="table-wrap"><div id="T3" data-id="T3" class="table-wrap-title"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>≥ Threshold </th><th>&lt; Threshold </th></tr></thead><tbody><tr><td>TimeZoneCombi </td><td>a </td><td>b </td></tr><tr><td>Other time zone </td><td>c </td><td>d </td></tr></tbody></table></div><div content-id="T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>≥ Threshold </th><th>&lt; Threshold </th></tr></thead><tbody><tr><td>TimeZoneCombi </td><td>a </td><td>b </td></tr><tr><td>Other time zone </td><td>c </td><td>d </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>a: the number of time points of taxon <em>X</em> data in time zones of TimeZoneCombi which is equal or over the threshold.</p></div></div><div content-id="tblfn2"><div class="fn"><p>b: the number of time points of taxon <em>X</em> data in time zones of TimeZoneCombi which is under the threshold.</p></div></div><div content-id="tblfn3"><div class="fn"><p>c: the number of time points of taxon <em>X</em> data in other time zones which is equal or over the threshold.</p></div></div><div content-id="tblfn4"><div class="fn"><p>d: the number of time points of taxon <em>X</em> data in other time zones which is under the threshold.</p></div></div></div></div></div><div class="table-wrap"><div id="T3" data-id="T3" class="table-wrap-title"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>≥ Threshold </th><th>&lt; Threshold </th></tr></thead><tbody><tr><td>TimeZoneCombi </td><td>a </td><td>b </td></tr><tr><td>Other time zone </td><td>c </td><td>d </td></tr></tbody></table></div><div content-id="T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>≥ Threshold </th><th>&lt; Threshold </th></tr></thead><tbody><tr><td>TimeZoneCombi </td><td>a </td><td>b </td></tr><tr><td>Other time zone </td><td>c </td><td>d </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>a: the number of time points of taxon <em>X</em> data in time zones of TimeZoneCombi which is equal or over the threshold.</p></div></div><div content-id="tblfn2"><div class="fn"><p>b: the number of time points of taxon <em>X</em> data in time zones of TimeZoneCombi which is under the threshold.</p></div></div><div content-id="tblfn3"><div class="fn"><p>c: the number of time points of taxon <em>X</em> data in other time zones which is equal or over the threshold.</p></div></div><div content-id="tblfn4"><div class="fn"><p>d: the number of time points of taxon <em>X</em> data in other time zones which is under the threshold.</p></div></div></div></div></p><p>Thus, the less <em>P</em> value the taxon has, the stronger periodicity it is regarded to have.</p><p>Furthermore, the time zones determined by TimeZoneCombi with the best <em>P</em> value are regarded as proliferating time zones for the taxon <em>X</em>.</p><p>The whole procedures were depicted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>.</p>                            <h2 scrollto-destination=88473632 id="88473632" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=88473633 id="88473633" class="section-title" >3.1. Circadian rhythm in the salivary microbiome</h3>
<p>We longitudinally collected a total of 108 unstimulated salivary samples (18 samples per subject) every 4 h for 3 days from six healthy subjects (two females and four males). We enumerated sequences of the 16S rRNA gene V1–V2 region by pyrosequencing the salivary DNA with the Roche 454 platform (see Materials and methods). The 16S reads were clustered to obtain the OTUs for each subject, and the generated OTUs were assigned to taxa by blasting against the 16S databases (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The phylum- and genus-level assignment apparently showed the diurnal change in the microbial abundance for all the subjects (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). We next performed the UniFrac distance analysis of the microbial communities, in which the distances between the samples at the same times (ST) and those at different times (DT) for 3 days were measured, respectively. The results revealed that the UniFrac distances between the ST samples were lower than those between the DT samples for all the subjects, of which the five subjects showed a statistical significance, indicating that the overall structure of the ST samples was more similar than that of the DT samples (<a reveal-id="dsx001-F1" data-open="dsx001-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>).</p>                        <a id="88473635" scrollto-destination="88473635"></a>
<div id="dsx001-F1" data-id="dsx001-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f1.png?Expires=2147483647&Signature=H6EACl0VfPOPqbed-9upaRhJjOKEmQpVdGUxjO4IHEn8gx7iqHIFCFLglUvsLRDRYC7Pbe-69rvofAlvCVAlKQqd6dA9RaYa4sAnADw6exI~xJZ-6WNOuFoxCn~D3EnlAru43ocYQlaXgZZsPWlAdhHLycdXuoPTTHdwMgFdwUvU8DEThmgUPoStavsWGCl0utoKqYX6E56oxAXKJq97PVJn6lrxKZvz6sIVUg0MBBx57gcpIm~agm0PCLAE1-5C7wFRGUqh0y2-A8a2EAnZKuBy5hu-2yBDxT0jT9DYYAzBvkoEv6FFrskpBPE3awdnUtqClobAsaMkwbadmGy2LQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of diurnal oscillation in salivary microbiomes. (a) Comparison of the weighted UniFrac distances between the ST and DT samples in each subject. Asterisks indicate the statistical significance (P &lt; 0.05, Wilcoxon signed-rank test). (b) Autocorrelation coefficients of the relative abundance of Firmicutes, Bacteroidetes in all the subjects. The horizontal axis represents time lag used for autocorrelation calculation. (c) Correlations between the relative abundance and circadian periodicity of the genera. Genera with ≥0.1% average relative abundance in all the subjects were analysed. The vertical axis represents raw P values of the circadian periodicity of each genus calculated by the PDM. The horizontal axis represents the average relative abundance of each genus." path-from-xml="dsx001f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473635/dsx001f1.tif" path-from-xml="dsx001f1.tif" target="_blank">View large</a><a section="88473635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f1.png?Expires=2147483647&Signature=BN0edbx9XcPHOFNDW09dk76ZF7FpuwkMaPy-pnX7edJXzfaZckFxKUCFJ4Y73gzrOqXWjduhuKD8zGcvPXzJnzLnyHsMepGmS-cObfgzPxcYx-dW0u2vWVhpm707gwBdUgr0ASIQVtCKXn4WJ5h5nvVoMZlC8i9BO5c4xBkv1jXKA86c1zfLLuULN1W3wvNDdpiENRjCw-5j0VU2b22nyPxkn5wZj9jR9lrsloJPT14N~0OH7cchsI5ZtQRzeBysffNJJcGWcZYdAFR-sRe6g8nERuaPv6zH3E1LQg04aILJd33wj9retBrj5sOb4asxD6aZEuU4gNdaSWSvwVMT0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473635&ar=3052236&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx001f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of diurnal oscillation in salivary microbiomes. (a) Comparison of the weighted UniFrac distances between the ST and DT samples in each subject. Asterisks indicate the statistical significance (<em>P</em> &lt; 0.05, Wilcoxon signed-rank test). (b) Autocorrelation coefficients of the relative abundance of Firmicutes, Bacteroidetes in all the subjects. The horizontal axis represents time lag used for autocorrelation calculation. (c) Correlations between the relative abundance and circadian periodicity of the genera. Genera with ≥0.1% average relative abundance in all the subjects were analysed. The vertical axis represents raw <em>P</em> values of the circadian periodicity of each genus calculated by the PDM. The horizontal axis represents the average relative abundance of each genus.</p></div></div><div content-id="dsx001-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f1.png?Expires=2147483647&Signature=H6EACl0VfPOPqbed-9upaRhJjOKEmQpVdGUxjO4IHEn8gx7iqHIFCFLglUvsLRDRYC7Pbe-69rvofAlvCVAlKQqd6dA9RaYa4sAnADw6exI~xJZ-6WNOuFoxCn~D3EnlAru43ocYQlaXgZZsPWlAdhHLycdXuoPTTHdwMgFdwUvU8DEThmgUPoStavsWGCl0utoKqYX6E56oxAXKJq97PVJn6lrxKZvz6sIVUg0MBBx57gcpIm~agm0PCLAE1-5C7wFRGUqh0y2-A8a2EAnZKuBy5hu-2yBDxT0jT9DYYAzBvkoEv6FFrskpBPE3awdnUtqClobAsaMkwbadmGy2LQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of diurnal oscillation in salivary microbiomes. (a) Comparison of the weighted UniFrac distances between the ST and DT samples in each subject. Asterisks indicate the statistical significance (P &lt; 0.05, Wilcoxon signed-rank test). (b) Autocorrelation coefficients of the relative abundance of Firmicutes, Bacteroidetes in all the subjects. The horizontal axis represents time lag used for autocorrelation calculation. (c) Correlations between the relative abundance and circadian periodicity of the genera. Genera with ≥0.1% average relative abundance in all the subjects were analysed. The vertical axis represents raw P values of the circadian periodicity of each genus calculated by the PDM. The horizontal axis represents the average relative abundance of each genus." path-from-xml="dsx001f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473635/dsx001f1.tif" path-from-xml="dsx001f1.tif" target="_blank">View large</a><a section="88473635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f1.png?Expires=2147483647&Signature=BN0edbx9XcPHOFNDW09dk76ZF7FpuwkMaPy-pnX7edJXzfaZckFxKUCFJ4Y73gzrOqXWjduhuKD8zGcvPXzJnzLnyHsMepGmS-cObfgzPxcYx-dW0u2vWVhpm707gwBdUgr0ASIQVtCKXn4WJ5h5nvVoMZlC8i9BO5c4xBkv1jXKA86c1zfLLuULN1W3wvNDdpiENRjCw-5j0VU2b22nyPxkn5wZj9jR9lrsloJPT14N~0OH7cchsI5ZtQRzeBysffNJJcGWcZYdAFR-sRe6g8nERuaPv6zH3E1LQg04aILJd33wj9retBrj5sOb4asxD6aZEuU4gNdaSWSvwVMT0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473635&ar=3052236&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx001f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of diurnal oscillation in salivary microbiomes. (a) Comparison of the weighted UniFrac distances between the ST and DT samples in each subject. Asterisks indicate the statistical significance (<em>P</em> &lt; 0.05, Wilcoxon signed-rank test). (b) Autocorrelation coefficients of the relative abundance of Firmicutes, Bacteroidetes in all the subjects. The horizontal axis represents time lag used for autocorrelation calculation. (c) Correlations between the relative abundance and circadian periodicity of the genera. Genera with ≥0.1% average relative abundance in all the subjects were analysed. The vertical axis represents raw <em>P</em> values of the circadian periodicity of each genus calculated by the PDM. The horizontal axis represents the average relative abundance of each genus.</p></div></div><p>We then performed the autocorrelation analysis for diurnal changes in the relative abundance of the two major phyla, Firmicutes and Bacteroidetes in each subject between the 3 days. The analysis showed that the highest correlation coefficient was observed at ∼24-h interval in all of the subjects (<a reveal-id="dsx001-F1" data-open="dsx001-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>). These data strongly suggested that the observed diurnal changes in the microbial abundance of the salivary microbiomes exhibited a circadian rhythm at the phylum level. Similar results were also obtained from the analysis using the microbial abundance data estimated from mapping of the metagenomic reads on the reference genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p><p>On the other hand, the hierarchical clustering of the samples based on the Spearman's rank correlation coefficient revealed that many of the ST samples but not all were clustered most closely within a subject, whereas all the samples were clearly aggregated by the individuals (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), suggesting that the extent of circadian intra-subject variations was generally lower than that of the inter-subject variations.</p><p>To detect the taxa with a statistically significant circadian periodicity, we developed the PDM that determines the <em>P</em> value for the circadian periodicity of each genus by using the Fisher’s exact test repeatedly for every possible threshold and time-zone combination (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The PDM analysis followed by Bonferroni correction of 19 genera (18 known and one undefined) with ≥ 0.1% relative abundance in all of the six subjects found that the genera accounting for 68.4–89.6% of the total abundance (∼79.3% in average for the six subjects) exhibited a significant oscillation with the 24-h periodicity (<em>P</em> &lt; 0.05), among which <em>Prevotella</em>, <em>Gemella</em>, <em>Peptostreptococcus</em>, <em>Solobacterium</em>, <em>Streptococcus</em>, and <em>Haemophilus</em> were statistically significant for all the subjects (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Furthermore, we found a correlation between the relative abundance of the genera and the <em>P</em> value of the circadian periodicity <em>in toto</em>, which indicated that the higher abundant genera tended to have the stronger circadian periodicity, and <em>vice versa</em> (<a reveal-id="dsx001-F1" data-open="dsx001-F1" class="link link-reveal link-table xref-fig">Fig. 1c</a>). For the major four phyla, the PDM analysis also indicated the high circadian periodicity of them in most of the subjects as well as the autocorrelation analysis mentioned above (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). We also evaluated the circadian periodicity of the genera and phyla by JTK_CYCLE<a reveal-id="dsx001-B32" data-open="dsx001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S4 and S5</a></span>). In the genus level, JTK_CYCLE and PDM both showed the significant periodicity in <em>Prevotella</em> for the five subjects, <em>Neisseria</em> for the four subjects, <em>Gemella</em> and <em>Streptococcus</em> for the three subjects. In the phylum level, Bacteroidetes for the five subjects, Firmicutes for the four subjects, Proteobacteria for the three subjects, and Actinobacteria for the two subjects were significant in the circadian periodicity, respectively.</p><p>In addition, we examined the diurnal change in the OTU number. We compared quantitative differences in all pairs of OTU numbers in the ST and those in the DT samples in each subject, but no significant difference was observed between them for all the subjects, suggesting that OTU number representing the species richness had no circadian oscillation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p><p>We further analysed changes in the microbial cell number by qPCR with universal 16S primers. The results observed statistically significant periodicity of microbial cell numbers in two of the six subjects, in which microbial cell number increased at 4:00 and 16:00 in the subject 3, and at 4:00 and 8:00 in the subject 4 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). These data suggested that microbial cell number also exhibited the circadian oscillation with individual differences.</p>                            <h3 scrollto-destination=88473641 id="88473641" class="section-title" >3.2. Characterization of circadian oscillation patterns</h3>
<p>We characterized the oscillation pattern of salivary microbes at the phylum and genus level. We found that the relative abundance of Firmicutes was increased in the evening, whilst that of Bacteroidetes was in the daytime in all the subjects, indicating that these two major phyla had an inverse relation in the circadian oscillation pattern (<a reveal-id="dsx001-F2" data-open="dsx001-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a>). At the genus level, we showed proliferation time zones of the circadian oscillation of the 12 genera with ≥ 1% relative abundance in all the subjects, respectively (<a reveal-id="dsx001-F2" data-open="dsx001-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>). Among the 12 genera, <em>Prevotella</em>, <em>Gemella</em>, <em>Streptococcus</em>, and <em>Haemophilus</em> showed the significant circadian periodicity for all the subjects, but the other eight genera showed non-significant periodicity in a few subjects (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). The relative abundance of <em>Prevotella</em> was increased in the morning (from 4:00 to 12:00) in all the subjects, and those of <em>Gemella</em> and <em>Streptococcus</em> were increased in the evening (from 16:00 to 0:00) in all the subjects. The relative abundance of <em>Haemophilus</em> was proliferated in the morning (around 8:00) in all the subjects, but different proliferation time zones were also observed in three subjects. These genera are typical examples that had almost same circadian oscillation patterns shared among all the six subjects. The inverse oscillation pattern between Firmicutes and Bacteroidetes described above could be largely due to the strong circadian periodicity of the top two pre-dominant <em>Streptococcus</em> and <em>Prevotella</em> genera belonging to these two phyla, respectively. <em>Rothia</em> had similar proliferation time zones in the relative abundance among four subjects but no significant circadian periodicity in two subjects. The other genera such as <em>Granulicatella</em> and <em>Veillonella</em> did not share similar proliferation time zones among the subjects with the significant circadian periodicity. These data revealed that a few but highly abundant genera had the strong circadian periodicity and similar circadian oscillation patterns shared with most of the subjects. Many of the genera had the significant circadian periodicity but their oscillation patterns were considerably varied among the subjects. Thus, these data suggested the high inter-subject variation in the circadian oscillation profile.</p>                        <a id="88473643" scrollto-destination="88473643"></a>
<div id="dsx001-F2" data-id="dsx001-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f2.png?Expires=2147483647&amp;Signature=oAZcRigEuwgYreFQ2BFP9CDDs1LIMuzNWql9ednStQw3WSeSWs59AZ2~U3tbX7yITtdEf7DM1MA5sy0sZhWIlKfl88D50WniWInZzr8gMK~uY54Swge57oOAwWi5VS86DbpoGzZBMh4~I44dWSPA76BPJUMGAIwlQQgIGkHDHrdpU-EN1Y3nhLe47Qz8rgkFGIeXmdXcrok-bBro5zabfmyHwNlAHfFISxJsBtsLhKJS~1H~hd7BRiIfTCekeMo73a25wIkLSG8WTJOq8rICWCahJjVF9glFOlR2Nih9BKB4OLcBsTLHjFx5FM73-78ARd9h1tktkcZS0grCTo6zYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oscillation patterns of major taxa. (a) Circadian oscillation patterns of Firmicutes and Bacteroidetes. Changes in the normalized relative abundance of Firmicutes and Bacteroidetes in each subject are represented. (b) Circadian proliferation time zones of major genera. The results of the genera with ≥1% relative abundance in all the subjects are shown. Lines cover the proliferation time zones of the genus in each subject determined by PDM, in which the subjects having the proliferation time zones with no significant circadian periodicity were eliminated (see Supplementary Table S4)." path-from-xml="dsx001f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473643/dsx001f2.tif" path-from-xml="dsx001f2.tif" target="_blank">View large</a><a section="88473643" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f2.png?Expires=2147483647&amp;Signature=uYmnfjaMhf238J0D2K6E6hyhx11meaR5J9awAHEpZHTz6rqOdrpepe~NBgilz70q1h3uhm~lNsf5SEOSQNbVMOfLgj63R-e7PuWp~D8rme0~jb5dhQ9MfuHnS1SL-JGaRP54VMv5UDNdsVPNSfaGTDnCCTaApv9RfXlbo6XxGZl5TGnzauN6~QiqhWusPjIR6950i0oQoLP-iw0lHOlm1UphCbFC7PYqPsCAxUCCoiLHpiD6yGetaYavp0gLhpzkSdn2jUZsVbxv5KnDlEZMqHNxoDoYDEjtIkAnNS9n-jvgW62SAqKBErv~-GrEDLF-HkNFZf4o57FedlNk6pFVow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473643&amp;ar=3052236&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx001f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oscillation patterns of major taxa. (a) Circadian oscillation patterns of Firmicutes and Bacteroidetes. Changes in the normalized relative abundance of Firmicutes and Bacteroidetes in each subject are represented. (b) Circadian proliferation time zones of major genera. The results of the genera with ≥1% relative abundance in all the subjects are shown. Lines cover the proliferation time zones of the genus in each subject determined by PDM, in which the subjects having the proliferation time zones with no significant circadian periodicity were eliminated (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&amp;Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p></div></div><div content-id="dsx001-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f2.png?Expires=2147483647&amp;Signature=oAZcRigEuwgYreFQ2BFP9CDDs1LIMuzNWql9ednStQw3WSeSWs59AZ2~U3tbX7yITtdEf7DM1MA5sy0sZhWIlKfl88D50WniWInZzr8gMK~uY54Swge57oOAwWi5VS86DbpoGzZBMh4~I44dWSPA76BPJUMGAIwlQQgIGkHDHrdpU-EN1Y3nhLe47Qz8rgkFGIeXmdXcrok-bBro5zabfmyHwNlAHfFISxJsBtsLhKJS~1H~hd7BRiIfTCekeMo73a25wIkLSG8WTJOq8rICWCahJjVF9glFOlR2Nih9BKB4OLcBsTLHjFx5FM73-78ARd9h1tktkcZS0grCTo6zYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oscillation patterns of major taxa. (a) Circadian oscillation patterns of Firmicutes and Bacteroidetes. Changes in the normalized relative abundance of Firmicutes and Bacteroidetes in each subject are represented. (b) Circadian proliferation time zones of major genera. The results of the genera with ≥1% relative abundance in all the subjects are shown. Lines cover the proliferation time zones of the genus in each subject determined by PDM, in which the subjects having the proliferation time zones with no significant circadian periodicity were eliminated (see Supplementary Table S4)." path-from-xml="dsx001f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473643/dsx001f2.tif" path-from-xml="dsx001f2.tif" target="_blank">View large</a><a section="88473643" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f2.png?Expires=2147483647&amp;Signature=uYmnfjaMhf238J0D2K6E6hyhx11meaR5J9awAHEpZHTz6rqOdrpepe~NBgilz70q1h3uhm~lNsf5SEOSQNbVMOfLgj63R-e7PuWp~D8rme0~jb5dhQ9MfuHnS1SL-JGaRP54VMv5UDNdsVPNSfaGTDnCCTaApv9RfXlbo6XxGZl5TGnzauN6~QiqhWusPjIR6950i0oQoLP-iw0lHOlm1UphCbFC7PYqPsCAxUCCoiLHpiD6yGetaYavp0gLhpzkSdn2jUZsVbxv5KnDlEZMqHNxoDoYDEjtIkAnNS9n-jvgW62SAqKBErv~-GrEDLF-HkNFZf4o57FedlNk6pFVow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473643&amp;ar=3052236&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx001f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oscillation patterns of major taxa. (a) Circadian oscillation patterns of Firmicutes and Bacteroidetes. Changes in the normalized relative abundance of Firmicutes and Bacteroidetes in each subject are represented. (b) Circadian proliferation time zones of major genera. The results of the genera with ≥1% relative abundance in all the subjects are shown. Lines cover the proliferation time zones of the genus in each subject determined by PDM, in which the subjects having the proliferation time zones with no significant circadian periodicity were eliminated (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&amp;Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p></div></div>                            <h3 scrollto-destination=88473644 id="88473644" class="section-title" >3.3. Bacterial phenotypes and oscillation patterns</h3>
<p>To examine if the oscillation patterns of the salivary microbiome is determined by the bacterial phenotype, we compared the oscillation patterns between strict aerobes (AE), facultative anaerobes (FA), and strict anaerobes (AN). The PDM analysis revealed that the relative abundance of AE, FA, and AN had significant circadian oscillations in all the six subjects, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). We analysed changes in the average relative abundance of the three microbial categories through the three days (<a reveal-id="dsx001-F3" data-open="dsx001-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). The results showed differences in the circadian oscillation patterns among the three categories, in which AN was proliferated in the morning between 4:00 and 12:00, and FA was in the evening between 16:00 and 4:00, indicating the circadian oscillation pattern of AN and FA had an inverse relation. On the other hand, the proliferation time zone of AE was the daytime between 8:00 and 20:00, which was intermediate between those of AN and FA. In the 19 genera with ≥0.1% relative abundance in all the subjects, <em>Neisseria</em>, <em>Rothia</em>, and <em>Campylobacter</em> were included in the typical AE pattern, <em>Prevotella</em> and <em>Solobacterium</em> were in the typical AN pattern, and <em>Streptococcus</em>, <em>Actinomyces</em> and <em>Gemella</em> were in the typical FA pattern, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). These data revealed that the three categories defined by the bacterial aerobicity were sequentially proliferated in the relative abundance with some overlap among them in a day. The periodicity of the three microbial categories was also evaluated by JTK_CYCLE, showing similar results to those by the PDM (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>).</p>                        <a id="88473646" scrollto-destination="88473646"></a>
<div id="dsx001-F3" data-id="dsx001-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f3.png?Expires=2147483647&Signature=gXVAIdpmLfSETlfPZ4CrfpydbMVXLbzc86qnHjkHg3Z9hpLvXkKxZr-orLGUhsV6gKhRPhR7U6NuHgdZUMgvMXxk6b9iJ2vukRlxDXkz6HCdx-NidTCfePjBI4M~z1NRGJJUQzfs43-ujpRequXQ82GzVahcdIBku6mdrB6YLZOdClfkCJ~c2B~Bp8C-XdN2Le3YafFoa~F8TU7ZayJ81f-bOI-6C14WwOoH23pZA7yEF4~CfxrsjT44-mwLS65EebRe08Vpbn3r5EkJtZ4EJwOXtYkzrYnDoTKhHhYdBgK2RznNj1wyxocv9TiT8cCgAejXnyGTuUqg9rFGwB0xTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circadian oscillation patterns of the bacterial phenotypes. (a) Oscillation patterns of genera classified by the bacterial aerobicity. The oscillation patterns of strictly aerobic (AE), facultative anaerobic (FA), and strictly anaerobic (AN) genera are shown. (b) Oscillation patterns of genera classified by the Gram stainability. The oscillation patterns of Gram-positive and -negative genera are shown." path-from-xml="dsx001f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473646/dsx001f3.tif" path-from-xml="dsx001f3.tif" target="_blank">View large</a><a section="88473646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f3.png?Expires=2147483647&Signature=06ryUVOivAcZM-DkAHC3hb-fo7Z3hycPn8NP-YfkDDnEsB5-IypiRuLevz24h1HXAuYbep7VJC-nSaWABtT~izSzApKhbG51bACJlFSpTQPs0GRw7hUaLg6gvF2W4croRxemYHBJXwLcfMQxdW5P20vQ2~9TlxbdrU~y3mzv0F37i19-EWsVX6AeTBHN1V5yC~JCsKRtc2n8ShzW3OD9P8vGvJfFkJibH2A1yHDqCJELIMagloL7Jqww9mR0FJ-KE3xLqWEB5jreTMtSvs3ZXS8XlqmckqZqKPBhDSaCsDxwzmolkXSiXvvHJabgEztjFX~VdPKUuTuMkk8jrrM0Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473646&ar=3052236&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx001f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circadian oscillation patterns of the bacterial phenotypes. (a) Oscillation patterns of genera classified by the bacterial aerobicity. The oscillation patterns of strictly aerobic (AE), facultative anaerobic (FA), and strictly anaerobic (AN) genera are shown. (b) Oscillation patterns of genera classified by the Gram stainability. The oscillation patterns of Gram-positive and -negative genera are shown.</p></div></div><div content-id="dsx001-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f3.png?Expires=2147483647&Signature=gXVAIdpmLfSETlfPZ4CrfpydbMVXLbzc86qnHjkHg3Z9hpLvXkKxZr-orLGUhsV6gKhRPhR7U6NuHgdZUMgvMXxk6b9iJ2vukRlxDXkz6HCdx-NidTCfePjBI4M~z1NRGJJUQzfs43-ujpRequXQ82GzVahcdIBku6mdrB6YLZOdClfkCJ~c2B~Bp8C-XdN2Le3YafFoa~F8TU7ZayJ81f-bOI-6C14WwOoH23pZA7yEF4~CfxrsjT44-mwLS65EebRe08Vpbn3r5EkJtZ4EJwOXtYkzrYnDoTKhHhYdBgK2RznNj1wyxocv9TiT8cCgAejXnyGTuUqg9rFGwB0xTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circadian oscillation patterns of the bacterial phenotypes. (a) Oscillation patterns of genera classified by the bacterial aerobicity. The oscillation patterns of strictly aerobic (AE), facultative anaerobic (FA), and strictly anaerobic (AN) genera are shown. (b) Oscillation patterns of genera classified by the Gram stainability. The oscillation patterns of Gram-positive and -negative genera are shown." path-from-xml="dsx001f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473646/dsx001f3.tif" path-from-xml="dsx001f3.tif" target="_blank">View large</a><a section="88473646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f3.png?Expires=2147483647&Signature=06ryUVOivAcZM-DkAHC3hb-fo7Z3hycPn8NP-YfkDDnEsB5-IypiRuLevz24h1HXAuYbep7VJC-nSaWABtT~izSzApKhbG51bACJlFSpTQPs0GRw7hUaLg6gvF2W4croRxemYHBJXwLcfMQxdW5P20vQ2~9TlxbdrU~y3mzv0F37i19-EWsVX6AeTBHN1V5yC~JCsKRtc2n8ShzW3OD9P8vGvJfFkJibH2A1yHDqCJELIMagloL7Jqww9mR0FJ-KE3xLqWEB5jreTMtSvs3ZXS8XlqmckqZqKPBhDSaCsDxwzmolkXSiXvvHJabgEztjFX~VdPKUuTuMkk8jrrM0Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473646&ar=3052236&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx001f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circadian oscillation patterns of the bacterial phenotypes. (a) Oscillation patterns of genera classified by the bacterial aerobicity. The oscillation patterns of strictly aerobic (AE), facultative anaerobic (FA), and strictly anaerobic (AN) genera are shown. (b) Oscillation patterns of genera classified by the Gram stainability. The oscillation patterns of Gram-positive and -negative genera are shown.</p></div></div><p>We also compared the oscillation pattern between Gram-positive and -negative species. The PDM and JTK_CYCLE analyses revealed that the relative abundance of Gram-positive and -negative genera had significant circadian oscillations in all the subjects, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The analysis indicated that the Gram-positive species was proliferated in the evening between 16:00 and 4:00, whereas that of the Gram-negative species was in the day between 4:00 and 16:00, respectively (<a reveal-id="dsx001-F3" data-open="dsx001-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). In the 19 genera with ≥0.1% relative abundance, oscillation patterns of several Gram-negative <em>Prevotella</em> and <em>Haemophilus</em> were similar to that of the all Gram-negative species, and those of several Gram-positive <em>Streptococcus</em>, <em>Rothia</em>, <em>Actinomyces</em> and <em>Gemella</em> were similar to that of the all Gram-positive species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Furthermore, the oscillation pattern of Gram-positive bacteria shown in <a reveal-id="dsx001-F3" data-open="dsx001-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a> appeared to be overlapped with those of AE and FA shown in <a reveal-id="dsx001-F3" data-open="dsx001-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>, whereas that of Gram-negative bacteria appeared to be overlapped with those of AE and AN. The proportion of genera classified by the Gram stainability and aerobicity in samples of the subjects was shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>.</p>                            <h3 scrollto-destination=88473648 id="88473648" class="section-title" >3.4. Disappearance of circadian rhythm in salivary microbiomes <em>in vitro</em></h3>
<p>Salivary samples from two subjects (subjects 1 and 2) were incubated at 37°C <em>in vitro</em> under aerobic conditions, and changes in the microbial abundance in aliquot samples collected every 1 h for 7 h were monitored by 16S rRNA gene sequence analysis. These <em>in vitro</em> data were compared with those of salivary samples freshly collected with the ST intervals as those <em>in vitro</em> from the same two subjects (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). The results indicated that the <em>in vitro</em> samples had no longer change in the microbial composition with very low UniFrac distances between them, whilst freshly collected samples showed the circadian oscillation as expected.</p>                            <h3 scrollto-destination=88473650 id="88473650" class="section-title" >3.5. Functional analysis based on metagenomic data</h3>
<p>For addressing the biological periodicity, we performed the functional assignment of the annotated genes in the assembled metagenomic sequences, and found a total of 370 KEGG modules by using the KEGG database. We then quantified the 370 KEGG modules by mapping 100 K metagenomic reads per sample to them (see Materials and methods). The results detected 55 modules that significantly exhibited 24 h oscillation in more than five subjects (<a reveal-id="dsx001-F4" data-open="dsx001-F4" class="link link-reveal link-table xref-fig">Fig. 4a</a>). These mainly included the modules assigned to four functional categories, ‘Environmental information processing’, ‘Energy metabolism’, ‘Carbohydrate and lipid metabolism’, and ‘Nucleotide and amino acid metabolism’. Many of the modules assigned to ‘Environmental information processing’ were enriched in the evening (between 16:00 and 4:00), whereas those assigned to the other three functional categories were enriched in the morning (between 4:00 and 12:00), respectively, indicating the inverse relation between ‘Environmental information processing’ and the other three functional categories (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). The modules assigned to ‘Environmental information processing’ included the functions related to various transporters including those for metal ions and two-component regulatory systems. On the other hand, the modules assigned to ‘Carbohydrate and lipid metabolism’, ‘Energy metabolism’, and ‘Nucleotide and amino acid metabolism’ included the functions for the biosynthesis of vitamins and fatty acids, polysaccharides degradation, Hem biosynthesis from glutamate, the reductive citrate cycle, and electron transport.</p>                        <a id="88473652" scrollto-destination="88473652"></a>
<div id="dsx001-F4" data-id="dsx001-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f4.png?Expires=2147483647&Signature=Bd~z8EnHhIEn2iRznTMUye-n8A2N8TbuZmILmNKg895fHWVVU67pLRp9hPTgNPi0RI07wgfnfUKKtYssVJ3Iu~YThJLtHi9dA9dZce5DRCjXlKvjj~BBxirUUj5jUL2KqLGs3aNibkMTmXY6Yj4kmbFIk15x3q9vHDjhrIbGyyFITVP~GYZLqTI1a0fuijLAt-I2muO5N8jpOWr-Oc2z5mCmK6N9Fy9o5Jq7BsBcUN-pZVaSH7Y6nPmaVbiDS2sMPHRsYZeEXQ36eCPpGNsgIJk-dn7FHjnRehGjQEfYMZuX9JNq3KAIqb0FihXIvKqqafzSTY7siGJjxPgeSxrX1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The KEGG modules having the circadian oscillation. (a) Heatmap of the relative gene abundances of 55 KEGG modules. Colour gradient indicate the degree of the average abundance of the KEGG modules which had statistically significance in at least five subjects. Colours on right side indicate six upper functional categories of the KEGG modules. (b) Network of the KEGG modules and genera. The results for the significant correlation between the 55 KEGG modules and four genera in more than five subjects are shown. Interactions between nodes were calculated by Spearman’s rank correlation and correlations with P &gt;0.05 or P &lt;0.6 are removed. Coloured rectangles correspond to the KEGG modules and ellipse nodes signify the genera. Size of ellipse nodes corresponds to the relative abundance of the genera. Blue and red lines indicate positive and negative correlation, respectively." path-from-xml="dsx001f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473652/dsx001f4.tif" path-from-xml="dsx001f4.tif" target="_blank">View large</a><a section="88473652" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f4.png?Expires=2147483647&Signature=IXX9lf2l7zKJuU51wx3XNG46zXsdd3Q42Sc8lu7Q~-PoL8-zgNQT8hEsrEXNNy8BBmMBpy0vjlSmuVJLXeqlYFkjOpvubC-BRiTMdvK1i0kqatSNADF2CgFLOVlscpg8eZjVtXHt-uRz0u~B7dQw-IQucJ19YNMe4iNLQ6EmDj~lkrGvkHRe8WXSYq9Zi7uS3Fr4~0RzVSP3onDpVPDXYflflWfzC9F1wQfSdEcxWuJsotVQYS8Ex1KJvyE0AmB0XRFoKSvoXuhdsMPA9jPGV7zo2itObjCIBY0awKmSIjtZHD-Gonc3ySaMUPQN~2GhZ6bli~ZjWaxyip~GbNwMVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473652&ar=3052236&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx001f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The KEGG modules having the circadian oscillation. (a) Heatmap of the relative gene abundances of 55 KEGG modules. Colour gradient indicate the degree of the average abundance of the KEGG modules which had statistically significance in at least five subjects. Colours on right side indicate six upper functional categories of the KEGG modules. (b) Network of the KEGG modules and genera. The results for the significant correlation between the 55 KEGG modules and four genera in more than five subjects are shown. Interactions between nodes were calculated by Spearman’s rank correlation and correlations with <em>P</em> &gt;0.05 or <em>P</em> &lt;0.6 are removed. Coloured rectangles correspond to the KEGG modules and ellipse nodes signify the genera. Size of ellipse nodes corresponds to the relative abundance of the genera. Blue and red lines indicate positive and negative correlation, respectively.</p></div></div><div content-id="dsx001-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/m_dsx001f4.png?Expires=2147483647&Signature=Bd~z8EnHhIEn2iRznTMUye-n8A2N8TbuZmILmNKg895fHWVVU67pLRp9hPTgNPi0RI07wgfnfUKKtYssVJ3Iu~YThJLtHi9dA9dZce5DRCjXlKvjj~BBxirUUj5jUL2KqLGs3aNibkMTmXY6Yj4kmbFIk15x3q9vHDjhrIbGyyFITVP~GYZLqTI1a0fuijLAt-I2muO5N8jpOWr-Oc2z5mCmK6N9Fy9o5Jq7BsBcUN-pZVaSH7Y6nPmaVbiDS2sMPHRsYZeEXQ36eCPpGNsgIJk-dn7FHjnRehGjQEfYMZuX9JNq3KAIqb0FihXIvKqqafzSTY7siGJjxPgeSxrX1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The KEGG modules having the circadian oscillation. (a) Heatmap of the relative gene abundances of 55 KEGG modules. Colour gradient indicate the degree of the average abundance of the KEGG modules which had statistically significance in at least five subjects. Colours on right side indicate six upper functional categories of the KEGG modules. (b) Network of the KEGG modules and genera. The results for the significant correlation between the 55 KEGG modules and four genera in more than five subjects are shown. Interactions between nodes were calculated by Spearman’s rank correlation and correlations with P &gt;0.05 or P &lt;0.6 are removed. Coloured rectangles correspond to the KEGG modules and ellipse nodes signify the genera. Size of ellipse nodes corresponds to the relative abundance of the genera. Blue and red lines indicate positive and negative correlation, respectively." path-from-xml="dsx001f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473652/dsx001f4.tif" path-from-xml="dsx001f4.tif" target="_blank">View large</a><a section="88473652" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001f4.png?Expires=2147483647&Signature=IXX9lf2l7zKJuU51wx3XNG46zXsdd3Q42Sc8lu7Q~-PoL8-zgNQT8hEsrEXNNy8BBmMBpy0vjlSmuVJLXeqlYFkjOpvubC-BRiTMdvK1i0kqatSNADF2CgFLOVlscpg8eZjVtXHt-uRz0u~B7dQw-IQucJ19YNMe4iNLQ6EmDj~lkrGvkHRe8WXSYq9Zi7uS3Fr4~0RzVSP3onDpVPDXYflflWfzC9F1wQfSdEcxWuJsotVQYS8Ex1KJvyE0AmB0XRFoKSvoXuhdsMPA9jPGV7zo2itObjCIBY0awKmSIjtZHD-Gonc3ySaMUPQN~2GhZ6bli~ZjWaxyip~GbNwMVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473652&ar=3052236&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx001f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The KEGG modules having the circadian oscillation. (a) Heatmap of the relative gene abundances of 55 KEGG modules. Colour gradient indicate the degree of the average abundance of the KEGG modules which had statistically significance in at least five subjects. Colours on right side indicate six upper functional categories of the KEGG modules. (b) Network of the KEGG modules and genera. The results for the significant correlation between the 55 KEGG modules and four genera in more than five subjects are shown. Interactions between nodes were calculated by Spearman’s rank correlation and correlations with <em>P</em> &gt;0.05 or <em>P</em> &lt;0.6 are removed. Coloured rectangles correspond to the KEGG modules and ellipse nodes signify the genera. Size of ellipse nodes corresponds to the relative abundance of the genera. Blue and red lines indicate positive and negative correlation, respectively.</p></div></div><p>We further performed the network analysis of the correlation between the 55 modules and the five genera (<em>Streptococcus</em>, <em>Prevotella</em>, <em>Haemophilus</em>, <em>Gemella</em>, and <em>Granulicatella</em>) with ≥1% relative abundance and the significant circadian periodicity shared with more than five subjects (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). The analysis formed two distinct negatively correlated clusters, which were represented by positive correlations between <em>Streptococcus</em> and <em>Gemella</em> and many of the modules assigned to ‘Environmental information processing’, and those between <em>Prevotella</em> and <em>Haemophilus</em> and the modules assigned to ‘Energy metabolism’ (<a reveal-id="dsx001-F4" data-open="dsx001-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a>). <em>Granulicatella</em> had no significant correlation with any of the module. These correlations were also related with the bacterial phenotypes, in which <em>Streptococcus</em> and <em>Gemella</em> were the Gram-positive, and <em>Prevotella</em> and <em>Haemophilus</em> were Gram-negative.</p>                            <h2 scrollto-destination=88473654 id="88473654" class="section-title" >4. Discussion</h2>
<p>In this study, we demonstrated that the majority of the salivary microbes exhibited the circadian oscillation with ∼24-h in the relative abundance by the analysis using the 16S V1–V2 sequences obtained from the samples collected with 4-h intervals for three consecutive days from six healthy adults. We obtained the several pieces of evidence for the circadian oscillation from significant differences in the UniFrac distances between the samples collected at the same and DT in each subject, and apparent diurnal changes in the relative abundance of taxa at the phylum and genus level with the highest autocorrelation coefficient of ∼24-h interval. Furthermore, the circadian periodicity of each taxon was also statistically evaluated by the PMD originally developed in this study. The basis of the PDM is to assess the reproducibility of proliferation time patterns of the relative abundance of each taxon by the Fisher exact test, so that the PDM is also applicable to the relatively minor taxa of which the 16S read number is zero in several samples. Therefore, the PDM generally enables us to detect more taxa with the periodic pattern than the JTK_CYCLE.<a reveal-id="dsx001-B32" data-open="dsx001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> The PDM analysis found that the major genera had the tendency of relatively high circadian periodicity compared with minor genera in the salivary microbiome, similar to the observation in the gut microbiome.<a reveal-id="dsx001-B33" data-open="dsx001-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> However, on the contrary, the recent study reported almost no significant diurnal variation in the salivary microbiome by using the clustering analysis of two sample sets collected every 4 h in 24 h with a 1 week interval from five subjects.<a reveal-id="dsx001-B26" data-open="dsx001-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> We similarly performed the hierarchical clustering of the same sample sets, but could not clearly show the closest relation between the samples collected at the ST by this clustering. Therefore, this inconsistency suggested that the hierarchical clustering was insufficient to differentiate subtle changes in variations within individuals.</p><p>The circadian oscillation completely disappeared with very low UniFrac distances between the samples in the <em>in vitro</em> incubation experiments, suggesting that the circadian oscillation in the salivary microbiome is not an autonomous process, but a response linked to certain host’s signals and/or oral environmental changes associated with them. The salivary microbiome is thought to be formed by microbes adhered to various intraoral surfaces because the saliva is sterilely secreted from the salivary glands into the oral cavity.<a reveal-id="dsx001-B3" data-open="dsx001-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> This implies that the microbes associated with intraoral surfaces may change their releasing level from the surfaces to the saliva and/or their growth rate in the saliva. The microbes in the throat, tonsil, and tongue may be the resources of the salivary microbiome because the salivary microbiome composition is similar to that of these microbiomes.<a reveal-id="dsx001-B4 dsx001-B5 dsx001-B6" data-open="dsx001-B4 dsx001-B5 dsx001-B6" class="link link-ref link-reveal xref-bibr"><sup>4–6</sup></a> In this context, the recent paper showed that the plaque microbiome exhibited little circadian oscillation,<a reveal-id="dsx001-B34" data-open="dsx001-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> suggesting that the sensitivity to stimuli in the oral cavity differed between the plaque and salivary and possibly tongue and tonsil microbiomes.</p><p>We observed differences in the circadian periodicities (<em>P</em> values estimated by the PDM) among various microbial categories by comparing the average <em>P</em> values of 28 microbial categories classified by the aerobicity, Gram stainability, and taxonomically (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). The data revealed that the first ten categories had much higher circadian periodicity than others, which included the three categories by the aerobicity, two by the Gram stainability, two phyla (Firmicutes and Bacteroidetes), and three genera (<em>Streptococcus</em>, <em>Prevotella</em>, and <em>Gemella</em>). These oscillation patterns were clearer than those of the others. These data suggested large associations of these microbial categories to the circadian oscillation of the salivary microbiome. On the other hand, the other 18 genera exhibiting less periodicity than the 10 microbial categories had less similarity of the oscillation patterns among most of the subjects, suggesting the high inter-subject variation of these microbes in the circadian regulation of salivary microbiome.</p><p>Regarding the bacterial phenotypes, our data suggested that the aerobicity in the oral cavity was also diurnally changed, and more aerobic around noon in a day according to the oscillation pattern of the three microbial categories classified by the aerobicity. In addition, the Gram stainability was also largely involved in the circadian oscillation in the salivary microbiome, in which Gram-positive and -negative species had the trade-off relation of circadian oscillation patterns. At present, we have no reasonable interpretation for the relationship between circadian oscillation and the Gram stainability.</p><p>The circadian oscillation of gene functions was also evaluated by metagenomic data. In the daytime, the functions assigned to ‘Energy metabolism’ and ‘Carbohydrate and lipid metabolism’ mainly associated with <em>Prevotella</em> were enriched, and in the evening, those to ‘Environmental information processing’ mainly associated with <em>Streptococcus</em> were enriched, respectively. These findings suggest that functions of salivary microbes devote to environmental responses in the daytime, whereas those devote to metabolisms in the evening, and these functional oscillations may be related with maintenance of the oral homeostasis.</p><p>Recently, it was reported that the gut microbiome exhibited the circadian oscillation in the microbial abundance in mice, and the circadian oscillation was abolished by disruption of a host circadian clock gene.<a reveal-id="dsx001-B33" data-open="dsx001-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsx001-B35" data-open="dsx001-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> From these observations, it can be suggested that the circadian oscillation of the salivary microbiome is regulated by host clock genes. Although ∼15% of the salivary metabolites were reported to exhibit diurnal oscillation,<a reveal-id="dsx001-B36" data-open="dsx001-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> the factors or signals affecting the circadian oscillation of the salivary microbiome are unknown at present and further investigation is needed to address them.</p>                            <h2 scrollto-destination=88473661 id="88473661" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Dr Y. Hirose (Toyohashi University of Technology) for technical support for sequencing, and Drs H. Takayasu (Tokyo Institute of Technology), M. Takayasu (Sony Computer Science Laboratories, Inc.) and T. Yamada (Tokyo Institute of Technology) for their valuable advice in the discussion. The Super-computing resource was provided by the Human Genome Center (The University of Tokyo).</p>                            <h3 scrollto-destination=88473663 id="88473663" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=88473665 id="88473665" class="section-title" >Accession numbers</h3>
<p>DRA005351, DRA005363, DRA5277, DRA3992</p>                            <h3 scrollto-destination=88473667 id="88473667" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=88473669 id="88473669" class="section-title" >Data availability</h3>
<p>All high-quality 16S V1-V2 sequences analyzed in the present study were deposited into the DDBJ/GenBank/EMBL database with accession numbers DRA005351, DRA005363. The filter-passed metagenomic sequences were deposited with accession numbers DRA5277, DRA3992.</p>                            <h3 scrollto-destination=88473671 id="88473671" class="section-title" >Funding</h3>
<p>This work was supported in part by a Grant-in-Aid for JSPS Fellows (No. 7826) to L.T., a Grant-in-Aid for Challenging Exploratory Research (No. 25640105) to M.H., and the global COE project of ‘Genome Information Big Bang’ from the Ministry of Education, Culture, Sports, Science, and Technology (MEXT) of Japan to M.H. and K.O. Funding to pay the Open Access publication charges for this article was provided by an administration grant from the Waseda University.</p>                            <h2 scrollto-destination=88473673 id="88473673" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx001-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aas</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Paster</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Stokes</div> <div class="given-names">L.N.</div></div>, <div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Dewhirst</div> <div class="given-names">F.E.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Defining the normal bacterial flora of the oral cavity</div>. <div class="source">J. Clin. Microbiol</div> ., <div class="volume">43</div>, <div class="fpage">5721</div>–<div class="lpage">5732</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20normal%20bacterial%20flora%20of%20the%20oral%20cavity&amp;author=J.A.%20Aas&amp;author=B.J.%20Paster&amp;author=L.N.%20Stokes&amp;author=I.%20Olsen&amp;author=F.E.%20Dewhirst&amp;publication_year=2005&amp;journal=J.%20Clin.%20Microbiol&amp;volume=43&amp;pages=5721-5732" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.43.11.5721-5732.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.43.11.5721-5732.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.43.11.5721-5732.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16272510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zaura</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Keijser</div> <div class="given-names">B.J.F.</div></div>, <div class="name string-name"><div class="surname">Huse</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Crielaard</div> <div class="given-names">W.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Defining the healthy “core microbiome” of oral microbial communities</div>. <div class="source">BMC Microbiol</div> ., <div class="volume">9</div>, <div class="fpage">259</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20healthy%20%E2%80%9Ccore%20microbiome%E2%80%9D%20of%20oral%20microbial%20communities&amp;author=E.%20Zaura&amp;author=B.J.F.%20Keijser&amp;author=S.M.%20Huse&amp;author=W.%20Crielaard&amp;publication_year=2009&amp;journal=BMC%20Microbiol&amp;volume=9&amp;pages=259" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-9-259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-9-259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-9-259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20003481" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takeshita</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kageyama</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Furuta</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Bacterial diversity in saliva and oral health-related conditions: the Hisayama Study</div>. <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">22164</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20diversity%20in%20saliva%20and%20oral%20health-related%20conditions%3A%20the%20Hisayama%20Study&amp;author=T.%20Takeshita&amp;author=S.%20Kageyama&amp;author=M.%20Furuta&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=22164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep22164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep22164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep22164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26907866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Segata</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Haake</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Mannon</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Composition of the adult digestive tract bacterial microbiome based on seven mouth surfaces, tonsils, throat and stool samples</div>. <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">R42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Composition%20of%20the%20adult%20digestive%20tract%20bacterial%20microbiome%20based%20on%20seven%20mouth%20surfaces%2C%20tonsils%2C%20throat%20and%20stool%20samples&amp;author=N.%20Segata&amp;author=S.K.%20Haake&amp;author=P.%20Mannon&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=R42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-6-r42" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-6-r42" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-6-r42"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22698087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huttenhower</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gevers</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Knight</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Structure, function and diversity of the healthy human microbiome</div>. <div class="source">Nature</div> , <div class="volume">486</div>, <div class="fpage">207</div>–<div class="lpage">214</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%2C%20function%20and%20diversity%20of%20the%20healthy%20human%20microbiome&amp;author=C.%20Huttenhower&amp;author=D.%20Gevers&amp;author=R.%20Knight&amp;publication_year=2012&amp;journal=Nature&amp;volume=486&amp;pages=207-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eren</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Borisy</div> <div class="given-names">G.G.</div></div>, <div class="name string-name"><div class="surname">Huse</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Welch</div> <div class="given-names">J.L.M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Oligotyping analysis of the human oral microbiome</div>. <div class="source">Proc. Natl. Acad. Sci. USA.</div>  <div class="volume">111</div>, <div class="fpage">E2875</div>–<div class="lpage">E2884</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oligotyping%20analysis%20of%20the%20human%20oral%20microbiome&amp;author=A.M.%20Eren&amp;author=G.G.%20Borisy&amp;author=S.M.%20Huse&amp;author=J.L.M.%20Welch&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA.&amp;volume=111&amp;pages=E2875-E2884" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1409644111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1409644111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1409644111"> </span></div> </div></div></div></div></div><div content-id="dsx001-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huse</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Y.Z.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.J.</div></div>, <div class="name string-name"><div class="surname">Fodor</div> <div class="given-names">A.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A core human microbiome as viewed through 16S rRNA sequence clusters</div>. <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e34242</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20core%20human%20microbiome%20as%20viewed%20through%2016S%20rRNA%20sequence%20clusters&amp;author=S.M.%20Huse&amp;author=Y.Z.%20Ye&amp;author=Y.J.%20Zhou&amp;author=A.A.%20Fodor&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e34242" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0034242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0034242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0034242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22719824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lazarevic</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Whiteson</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hernandez</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Francois</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Schrenzel</div> <div class="given-names">J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Study of inter- and intra-individual variations in the salivary microbiota</div>. <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">523</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Study%20of%20inter-%20and%20intra-individual%20variations%20in%20the%20salivary%20microbiota&amp;author=V.%20Lazarevic&amp;author=K.%20Whiteson&amp;author=D.%20Hernandez&amp;author=P.%20Francois&amp;author=J.%20Schrenzel&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=523" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20920195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Leake</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Pagni</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Falquet</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Taroni</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Greub</div> <div class="given-names">G.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The salivary microbiome for differentiating individuals: proof of principle</div>. <div class="source">Microbes Infect</div> ., <div class="volume">18</div>, <div class="fpage">399</div>–<div class="lpage">405</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20salivary%20microbiome%20for%20differentiating%20individuals%3A%20proof%20of%20principle&amp;author=S.L.%20Leake&amp;author=M.%20Pagni&amp;author=L.%20Falquet&amp;author=F.%20Taroni&amp;author=G.%20Greub&amp;publication_year=2016&amp;journal=Microbes%20Infect&amp;volume=18&amp;pages=399-405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.micinf.2016.03.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.micinf.2016.03.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.micinf.2016.03.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27063111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ding</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Schloss</div> <div class="given-names">P.D.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Dynamics and associations of microbial community types across the human body</div>. <div class="source">Nature</div> , <div class="volume">509</div>, <div class="fpage">357</div>–<div class="lpage">360</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamics%20and%20associations%20of%20microbial%20community%20types%20across%20the%20human%20body&amp;author=T.%20Ding&amp;author=P.D.%20Schloss&amp;publication_year=2014&amp;journal=Nature&amp;volume=509&amp;pages=357-360" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13178" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13178" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13178"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24739969" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arimatsu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Miyazawa</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Oral pathobiont induces systemic inflammation and metabolic changes associated with alteration of gut microbiota</div>. <div class="source">Sci. Rep</div> ., <div class="volume">4</div>, <div class="fpage">04828</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oral%20pathobiont%20induces%20systemic%20inflammation%20and%20metabolic%20changes%20associated%20with%20alteration%20of%20gut%20microbiota&amp;author=K.%20Arimatsu&amp;author=H.%20Yamada&amp;author=H.%20Miyazawa&amp;publication_year=2014&amp;journal=Sci.%20Rep&amp;volume=4&amp;pages=04828" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep04828" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep04828" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep04828"> </span></div> </div></div></div></div></div><div content-id="dsx001-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">De Filippis</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Vannini</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">La Storia</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The same microbiota and a potentially discriminant metabolome in the saliva of omnivore, ovo-lacto-vegetarian and Vegan individuals</div>. <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e112373</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20same%20microbiota%20and%20a%20potentially%20discriminant%20metabolome%20in%20the%20saliva%20of%20omnivore%2C%20ovo-lacto-vegetarian%20and%20Vegan%20individuals&amp;author=F.%20De%20Filippis&amp;author=L.%20Vannini&amp;author=A.%20La%20Storia&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e112373" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0112373" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0112373" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0112373"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25372853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stahringer</div> <div class="given-names">S.S.</div></div>, <div class="name string-name"><div class="surname">Clemente</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Corley</div> <div class="given-names">R.P.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Nurture trumps nature in a longitudinal survey of salivary bacterial communities in twins from early adolescence to early adulthood</div>. <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">2146</div>–<div class="lpage">2152</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nurture%20trumps%20nature%20in%20a%20longitudinal%20survey%20of%20salivary%20bacterial%20communities%20in%20twins%20from%20early%20adolescence%20to%20early%20adulthood&amp;author=S.S.%20Stahringer&amp;author=J.C.%20Clemente&amp;author=R.P.%20Corley&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=2146-2152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.140608.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.140608.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.140608.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23064750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nasidze</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Quinque</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Stoneking</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Global diversity in the human salivary microbiome</div>. <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">636</div>–<div class="lpage">643</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20diversity%20in%20the%20human%20salivary%20microbiome&amp;author=I.%20Nasidze&amp;author=J.%20Li&amp;author=D.%20Quinque&amp;author=K.%20Tang&amp;author=M.%20Stoneking&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=636-643" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.084616.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.084616.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.084616.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19251737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zaura</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Brandt</div> <div class="given-names">B.W.</div></div>, <div class="name string-name"><div class="surname">de Mattos</div> <div class="given-names">M.J.T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Same exposure but two radically different responses to antibiotics: resilience of the salivary microbiome versus long-term microbial shifts in feces</div>. <div class="source">mBio</div> , <div class="volume">6</div>, <div class="fpage">e01693</div>–<div class="lpage">e01715</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Same%20exposure%20but%20two%20radically%20different%20responses%20to%20antibiotics%3A%20resilience%20of%20the%20salivary%20microbiome%20versus%20long-term%20microbial%20shifts%20in%20feces&amp;author=E.%20Zaura&amp;author=B.W.%20Brandt&amp;author=M.J.T.%20de%20Mattos&amp;publication_year=2015&amp;journal=mBio&amp;volume=6&amp;pages=e01693-e01715" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/mBio.01693-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FmBio.01693-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FmBio.01693-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26556275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Belstrom</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Fiehn</div> <div class="given-names">N.E.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">C.H.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Differentiation of salivary bacterial profiles of subjects with periodontitis and dental caries</div>. <div class="source">J. Oral Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">27429</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differentiation%20of%20salivary%20bacterial%20profiles%20of%20subjects%20with%20periodontitis%20and%20dental%20caries&amp;author=D.%20Belstrom&amp;author=N.E.%20Fiehn&amp;author=C.H.%20Nielsen&amp;publication_year=2015&amp;journal=J.%20Oral%20Microbiol&amp;volume=7&amp;pages=27429" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3402/jom.v7.27429" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3402%2Fjom.v7.27429" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3402%2Fjom.v7.27429"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25854857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seoudi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Bergmeier</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Drobniewski</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Paster</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Fortune</div> <div class="given-names">F.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The oral mucosal and salivary microbial community of Behcet's syndrome and recurrent aphthous stomatitis</div>. <div class="source">J. Oral Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">27150</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20oral%20mucosal%20and%20salivary%20microbial%20community%20of%20Behcet's%20syndrome%20and%20recurrent%20aphthous%20stomatitis&amp;author=N.%20Seoudi&amp;author=L.A.%20Bergmeier&amp;author=F.%20Drobniewski&amp;author=B.%20Paster&amp;author=F.%20Fortune&amp;publication_year=2015&amp;journal=J.%20Oral%20Microbiol&amp;volume=7&amp;pages=27150" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3402/jom.v7.27150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3402%2Fjom.v7.27150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3402%2Fjom.v7.27150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26037240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Farrell</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Variations of oral microbiota are associated with pancreatic diseases including pancreatic cancer</div>. <div class="source">Gut</div> , <div class="volume">61</div>, <div class="fpage">582</div>–<div class="lpage">588</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variations%20of%20oral%20microbiota%20are%20associated%20with%20pancreatic%20diseases%20including%20pancreatic%20cancer&amp;author=J.J.%20Farrell&amp;author=L.%20Zhang&amp;author=H.%20Zhou&amp;publication_year=2012&amp;journal=Gut&amp;volume=61&amp;pages=582-588" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/gutjnl-2011-300784" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fgutjnl-2011-300784" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fgutjnl-2011-300784"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21994333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Said</div> <div class="given-names">H.S.</div></div>, <div class="name string-name"><div class="surname">Suda</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Nakagome</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Dysbiosis of salivary microbiota in inflammatory bowel disease and its association with oral immunological biomarkers</div>. <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">15</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dysbiosis%20of%20salivary%20microbiota%20in%20inflammatory%20bowel%20disease%20and%20its%20association%20with%20oral%20immunological%20biomarkers&amp;author=H.S.%20Said&amp;author=W.%20Suda&amp;author=S.%20Nakagome&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=15-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24013298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Francavilla</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ercolini</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Piccolo</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">salivary microbiota and metabolome associated with celiac disease</div>. <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">80</div>, <div class="fpage">3416</div>–<div class="lpage">3425</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=salivary%20microbiota%20and%20metabolome%20associated%20with%20celiac%20disease&amp;author=R.%20Francavilla&amp;author=D.%20Ercolini&amp;author=M.%20Piccolo&amp;publication_year=2014&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=80&amp;pages=3416-3425" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.00362-14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00362-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.00362-14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24657864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">D.Y.</div></div>, <div class="name string-name"><div class="surname">Jia</div> <div class="given-names">H.J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The oral and gut microbiomes are perturbed in rheumatoid arthritis and partly normalized after treatment</div>. <div class="source">Nat. Med</div> ., <div class="volume">21</div>, <div class="fpage">895</div>–<div class="lpage">905</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20oral%20and%20gut%20microbiomes%20are%20perturbed%20in%20rheumatoid%20arthritis%20and%20partly%20normalized%20after%20treatment&amp;author=X.%20Zhang&amp;author=D.Y.%20Zhang&amp;author=H.J.%20Jia&amp;publication_year=2015&amp;journal=Nat.%20Med&amp;volume=21&amp;pages=895-905" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm.3914" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm.3914" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm.3914"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26214836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Betrapally</div> <div class="given-names">N.S.</div></div>, <div class="name string-name"><div class="surname">Hylemon</div> <div class="given-names">P.B.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Salivary microbiota reflects changes in gut microbiota in cirrhosis with hepatic encephalopathy</div>. <div class="source">Hepatology</div> , <div class="volume">62</div>, <div class="fpage">1260</div>–<div class="lpage">1271</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salivary%20microbiota%20reflects%20changes%20in%20gut%20microbiota%20in%20cirrhosis%20with%20hepatic%20encephalopathy&amp;author=J.S.%20Bajaj&amp;author=N.S.%20Betrapally&amp;author=P.B.%20Hylemon&amp;publication_year=2015&amp;journal=Hepatology&amp;volume=62&amp;pages=1260-1271" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/hep.27819" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhep.27819" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhep.27819"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25820757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Papagerakis</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Schnell</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The circadian clock in oral health and diseases</div>. <div class="source">J. Dent. Res</div> ., <div class="volume">93</div>, <div class="fpage">27</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20circadian%20clock%20in%20oral%20health%20and%20diseases&amp;author=S.%20Papagerakis&amp;author=L.%20Zheng&amp;author=S.%20Schnell&amp;publication_year=2014&amp;journal=J.%20Dent.%20Res&amp;volume=93&amp;pages=27-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/0022034513505768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F0022034513505768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F0022034513505768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24065634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">David</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Materna</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Friedman</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Host lifestyle affects human microbiota on daily timescales</div>. <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">R89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Host%20lifestyle%20affects%20human%20microbiota%20on%20daily%20timescales&amp;author=L.A.%20David&amp;author=A.C.%20Materna&amp;author=J.%20Friedman&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=R89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-7-r89" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-7-r89" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-7-r89"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25146375" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cameron</div> <div class="given-names">S.J.S.</div></div>, <div class="name string-name"><div class="surname">Huws</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Hegarty</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">D.P.M.</div></div>, <div class="name string-name"><div class="surname">Mur</div> <div class="given-names">L.A.J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The human salivary microbiome exhibits temporal stability in bacterial diversity</div>. <div class="source">FEMS Microbiol. Ecol</div> ., <div class="volume">91</div>, <div class="fpage">fiv091</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20salivary%20microbiome%20exhibits%20temporal%20stability%20in%20bacterial%20diversity&amp;author=S.J.S.%20Cameron&amp;author=S.A.%20Huws&amp;author=M.J.%20Hegarty&amp;author=D.P.M.%20Smith&amp;author=L.A.J.%20Mur&amp;publication_year=2015&amp;journal=FEMS%20Microbiol.%20Ecol&amp;volume=91&amp;pages=fiv091" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsec/fiv091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsec%2Ffiv091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsec%2Ffiv091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26207044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Belstrom</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Holmstrup</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bardow</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kokaras</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fiehn</div> <div class="given-names">N.E.</div></div>, <div class="name string-name"><div class="surname">Paster</div> <div class="given-names">B.J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Temporal stability of the salivary microbiota in oral health</div>. <div class="source">PLoS ONE</div> , <div class="volume">11</div>, <div class="fpage">e0147472</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temporal%20stability%20of%20the%20salivary%20microbiota%20in%20oral%20health&amp;author=D.%20Belstrom&amp;author=P.%20Holmstrup&amp;author=A.%20Bardow&amp;author=A.%20Kokaras&amp;author=N.E.%20Fiehn&amp;author=B.J.%20Paster&amp;publication_year=2016&amp;journal=PLoS%20ONE&amp;volume=11&amp;pages=e0147472" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0147472" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0147472" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0147472"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26799067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Suda</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Robustness of gut microbiota of healthy adults in response to probiotic intervention revealed by high-throughput pyrosequencing</div>. <div class="source">DNA Res</div> ., <div class="volume">20</div>, <div class="fpage">241</div>–<div class="lpage">253</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Robustness%20of%20gut%20microbiota%20of%20healthy%20adults%20in%20response%20to%20probiotic%20intervention%20revealed%20by%20high-throughput%20pyrosequencing&amp;author=S.W.%20Kim&amp;author=W.%20Suda&amp;author=S.%20Kim&amp;publication_year=2013&amp;journal=DNA%20Res&amp;volume=20&amp;pages=241-253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23571675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lozupone</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Knight</div> <div class="given-names">R.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">UniFrac: a new phylogenetic method for comparing microbial communities</div>. <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">71</div>, <div class="fpage">8228</div>–<div class="lpage">8235</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniFrac%3A%20a%20new%20phylogenetic%20method%20for%20comparing%20microbial%20communities&amp;author=C.%20Lozupone&amp;author=R.%20Knight&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=71&amp;pages=8228-8235" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.12.8228-8235.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.12.8228-8235.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.12.8228-8235.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16332807" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lozupone</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Knight</div> <div class="given-names">R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Fast UniFrac: facilitating high-throughput phylogenetic analyses of microbial communities including analysis of pyrosequencing and PhyloChip data</div>. <div class="source">ISME J</div> ., <div class="volume">4</div>, <div class="fpage">17</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20UniFrac%3A%20facilitating%20high-throughput%20phylogenetic%20analyses%20of%20microbial%20communities%20including%20analysis%20of%20pyrosequencing%20and%20PhyloChip%20data&amp;author=M.%20Hamady&amp;author=C.%20Lozupone&amp;author=R.%20Knight&amp;publication_year=2010&amp;journal=ISME%20J&amp;volume=4&amp;pages=17-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2009.97" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2009.97" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2009.97"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19710709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Taniguchi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">MetaGeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes</div>. <div class="source">DNA Res</div> ., <div class="volume">15</div>, <div class="fpage">387</div>–<div class="lpage">396</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MetaGeneAnnotator%3A%20detecting%20species-specific%20patterns%20of%20ribosomal%20binding%20site%20for%20precise%20gene%20prediction%20in%20anonymous%20prokaryotic%20and%20phage%20genomes&amp;author=H.%20Noguchi&amp;author=T.%20Taniguchi&amp;author=T.%20Itoh&amp;publication_year=2008&amp;journal=DNA%20Res&amp;volume=15&amp;pages=387-396" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940874" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nishijima</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Suda</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Oshima</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The gut microbiome of healthy Japanese and its microbial and functional uniqueness</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">125</div>–<div class="lpage">133</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20gut%20microbiome%20of%20healthy%20Japanese%20and%20its%20microbial%20and%20functional%20uniqueness&amp;author=S.%20Nishijima&amp;author=W.%20Suda&amp;author=K.%20Oshima&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=125-133" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsw002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsw002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26951067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hughes</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Hogenesch</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Kornacker</div> <div class="given-names">K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">JTK_CYCLE: an efficient nonparametric algorithm for detecting rhythmic components in genome-scale data sets</div>. <div class="source">J. Biol. Rhythms</div> , <div class="volume">25</div>, <div class="fpage">372</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=JTK_CYCLE%3A%20an%20efficient%20nonparametric%20algorithm%20for%20detecting%20rhythmic%20components%20in%20genome-scale%20data%20sets&amp;author=M.E.%20Hughes&amp;author=J.B.%20Hogenesch&amp;author=K.%20Kornacker&amp;publication_year=2010&amp;journal=J.%20Biol.%20Rhythms&amp;volume=25&amp;pages=372-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/0748730410379711" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F0748730410379711" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F0748730410379711"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20876817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thaiss</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Zeevi</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Levy</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Transkingdom control of microbiota diurnal oscillations promotes metabolic homeostasis</div>. <div class="source">Cell</div> , <div class="volume">159</div>, <div class="fpage">514</div>–<div class="lpage">529</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transkingdom%20control%20of%20microbiota%20diurnal%20oscillations%20promotes%20metabolic%20homeostasis&amp;author=C.A.%20Thaiss&amp;author=D.%20Zeevi&amp;author=M.%20Levy&amp;publication_year=2014&amp;journal=Cell&amp;volume=159&amp;pages=514-529" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.09.048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.09.048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.09.048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25417104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sato</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yamagishi</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yamashita</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Inter-Individual differences in the oral bacteriome are greater than intra-day fluctuations in individuals</div>. <div class="source">PLoS ONE</div> , <div class="volume">10</div>, <div class="fpage">e0131607</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inter-Individual%20differences%20in%20the%20oral%20bacteriome%20are%20greater%20than%20intra-day%20fluctuations%20in%20individuals&amp;author=Y.%20Sato&amp;author=J.%20Yamagishi&amp;author=R.%20Yamashita&amp;publication_year=2015&amp;journal=PLoS%20ONE&amp;volume=10&amp;pages=e0131607" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0131607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0131607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0131607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26121551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx001-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Bushman</div> <div class="given-names">F.D.</div></div>, <div class="name string-name"><div class="surname">FitzGerald</div> <div class="given-names">G.A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Rhythmicity of the intestinal microbiota is regulated by gender and the host circadian clock</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">10479</div>–<div class="lpage">10484</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rhythmicity%20of%20the%20intestinal%20microbiota%20is%20regulated%20by%20gender%20and%20the%20host%20circadian%20clock&amp;author=X.%20Liang&amp;author=F.D.%20Bushman&amp;author=G.A.%20FitzGerald&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=10479-10484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1501305112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1501305112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1501305112"> </span></div> </div></div></div></div></div><div content-id="dsx001-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx001-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dallmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Viola</div> <div class="given-names">A.U.</div></div>, <div class="name string-name"><div class="surname">Tarokh</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cajochen</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">S.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The human circadian metabolome</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">2625</div>–<div class="lpage">2629</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20circadian%20metabolome&amp;author=R.%20Dallmann&amp;author=A.U.%20Viola&amp;author=L.%20Tarokh&amp;author=C.%20Cajochen&amp;author=S.A.%20Brown&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=2625-2629" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1114410109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1114410109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1114410109"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute. </div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3052236&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsx001/1/dsx001_Supp.zip?Expires=2147483647&amp;Signature=T9wWhtJZA-92u7EHcHLgYaGlOtp6zD5~oC9ZdMzmQodp54EiwT4rXy1I7RK9JMnInd-Avi3vZB6MeDQsoh0BBiPnmo3dF6XYiNxtAxeSjCUbY0od30apRyJKyTSO~FDxbLKktu45DHkTYTxFNnkKtqUK3qZoCTtXa54dMA5xGG1xPNa7PggKYO3d~0zlR7gY7HZYscdFySANITs3ub5p120bvtyLcsaTqoetklv5wZHgw6ciWfzbsGKcMCdq~XBGT60nPMjaj2DSNfVjXyRnTfJNKN2sAN7q3zD6DOeL6nnYu11X1~JR~qJ601egoZU6Xk3YclZRo5LeQEIQg6EgSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,885</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3052236"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f3%2f261%2f3052236">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000404439600004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F3%2F261%2F3052236" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3052236'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Circadian oscillations of microbial and functional composition in the human salivary microbiome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f261%2f3052236">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

