

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/5/521/5056130","@type":"ScholarlyArticle","name":"Mobile elements contribute to the uniqueness of human genome with 15,000 human-specific insertions and 14 Mbp sequence increase","datePublished":"2018-07-19","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2018-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy022","keywords":["mobile elements","human genome","evolution"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/5/521/5056130","author":[{"name":"Tang, Wanxiangfu","affiliation":"Department of Biological Sciences, Brock University, St. Catharines, ON, Canada","@type":"Person"},{"name":"Mun, Seyoung","affiliation":"Department of Nanobiomedical Science &amp; BK21 PLUS NBM Global Research, Center for Regenerative Medicine, Dankook University, Cheonan, Republic of Korea","@type":"Person"},{"name":"Joshi, Aditya","affiliation":"Department of Biological Sciences, Brock University, St. Catharines, ON, Canada","@type":"Person"},{"name":"Han, Kyudong","affiliation":"Department of Nanobiomedical Science &amp; BK21 PLUS NBM Global Research, Center for Regenerative Medicine, Dankook University, Cheonan, Republic of Korea","@type":"Person"},{"name":"Liang, Ping","affiliation":"Department of Biological Sciences, Brock University, St. Catharines, ON, Canada","@type":"Person","sameAs":"http://orcid.org/0000-0003-4423-0636"}],"description":"Abstract.  Mobile elements (MEs) collectively contribute to at least 50% of the human genome. Due to their past incremental accumulation and ongoing DNA transpo","pageStart":"521","pageEnd":"533","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Mobile elements contribute to the uniqueness of human genome with 15,000 human-specific insertions and 14 Mbp sequence increase","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Mobile elements contribute to the uniqueness of human genome with 15,000 human-specific insertions and 14&#160;Mbp sequence increase" />
    <meta property="og:description" content="Abstract.  Mobile elements (MEs) collectively contribute to at least 50% of the human genome. Due to their past incremental accumulation and ongoing DNA transpo" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/5/521/5056130" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-07-19" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tang, Wanxiangfu" /><meta name="citation_author_institution" content="Department of Biological Sciences, Brock University, St. Catharines, ON, Canada" /><meta name="citation_author" content="Mun, Seyoung" /><meta name="citation_author_institution" content="Department of Nanobiomedical Science &amp; BK21 PLUS NBM Global Research, Center for Regenerative Medicine, Dankook University, Cheonan, Republic of Korea" /><meta name="citation_author" content="Joshi, Aditya" /><meta name="citation_author_institution" content="Department of Biological Sciences, Brock University, St. Catharines, ON, Canada" /><meta name="citation_author" content="Han, Kyudong" /><meta name="citation_author_institution" content="Department of Nanobiomedical Science &amp; BK21 PLUS NBM Global Research, Center for Regenerative Medicine, Dankook University, Cheonan, Republic of Korea" /><meta name="citation_author" content="Liang, Ping" /><meta name="citation_author_institution" content="Department of Biological Sciences, Brock University, St. Catharines, ON, Canada" /><meta name="citation_title" content="Mobile elements contribute to the uniqueness of human genome with 15,000 human-specific insertions and 14 Mbp sequence increase" /><meta name="citation_firstpage" content="521" /><meta name="citation_lastpage" content="533" /><meta name="citation_doi" content="10.1093/dnares/dsy022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2018/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=Mobile elements and mammalian genome evolution;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2003;  citation_volume=13;  citation_pages=651-8; " /><meta name="citation_reference" content="citation_title=The impact of retrotransposons on human genome evolution;  citation_journal_title=Nat. Rev. Genet;  citation_year=2009;  citation_volume=10;  citation_pages=691-703; " /><meta name="citation_reference" content="citation_title=Mobile elements: drivers of genome evolution;  citation_journal_title=Science;  citation_year=2004;  citation_volume=303;  citation_pages=1626-32; " /><meta name="citation_reference" content="citation_title=Biology of mammalian L1 retrotransposons;  citation_journal_title=Annu. Rev. Genet;  citation_year=2001;  citation_volume=35;  citation_pages=501-38; " /><meta name="citation_reference" content="citation_title=Alu repeats and human genomic diversity;  citation_journal_title=Nat. Rev. Genet;  citation_year=2002;  citation_volume=3;  citation_pages=370-9; " /><meta name="citation_reference" content="citation_title=LINE drive. retrotransposition and genome instability;  citation_journal_title=Cell;  citation_year=2002;  citation_volume=110;  citation_pages=277-80; " /><meta name="citation_reference" content="citation_title=Which transposable elements are active in the human genome?;  citation_journal_title=Trends Genet;  citation_year=2007;  citation_volume=23;  citation_pages=183-91; " /><meta name="citation_reference" content="citation_title=SVA elements are nonautonomous retrotransposons that cause disease in humans;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2003;  citation_volume=73;  citation_pages=1444-51; " /><meta name="citation_reference" content="citation_title=SVA elements: a hominid-specific retroposon family;  citation_journal_title=J. Mol. Biol;  citation_year=2005;  citation_volume=354;  citation_pages=994-1007; " /><meta name="citation_reference" content="citation_title=Selfish genes, the phenotype paradigm and genome evolution;  citation_journal_title=Nature;  citation_year=1980;  citation_volume=284;  citation_pages=601-3; " /><meta name="citation_reference" content="citation_title=Human l1 retrotransposition is associated with genetic instability in vivo;  citation_journal_title=Cell;  citation_year=2002;  citation_volume=110;  citation_pages=327-38; " /><meta name="citation_reference" content="citation_title=Identifying related L1 retrotransposons by analyzing 3' transduced sequences;  citation_journal_title=Genome Biol;  citation_year=2003;  citation_volume=4;  citation_pages=R30" /><meta name="citation_reference" content="citation_title=Transcriptional disruption by the L1 retrotransposon and implications for mammalian transcriptomes;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=429;  citation_pages=268-74; " /><meta name="citation_reference" content="citation_title=Gene-breaking: a new paradigm for human retrotransposon-mediated gene evolution;  citation_journal_title=Genome Res;  citation_year=2005;  citation_volume=15;  citation_pages=1073-8; " /><meta name="citation_reference" content="citation_title=How retrotransposons shape genome regulation;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2016;  citation_volume=37;  citation_pages=90-100; " /><meta name="citation_reference" content="citation_title=Alu retrotransposition-mediated deletion;  citation_journal_title=J. Mol. Biol;  citation_year=2005;  citation_volume=348;  citation_pages=791-800; " /><meta name="citation_reference" content="citation_title=Genomic rearrangements by LINE-1 insertion-mediated deletion in the human and chimpanzee lineages;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=4040-52; " /><meta name="citation_reference" content="citation_title=Human genomic deletions mediated by recombination between Alu elements;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2006;  citation_volume=79;  citation_pages=41-53; " /><meta name="citation_reference" content="citation_title=Alu recombination-mediated structural deletions in the chimpanzee genome;  citation_journal_title=PLoS Genet;  citation_year=2007;  citation_volume=3;  citation_pages=1939-49; " /><meta name="citation_reference" content="citation_title=SVA retrotransposons as modulators of gene expression;  citation_journal_title=Mob. Genet. Elements;  citation_year=2014;  citation_volume=4;  citation_pages=e32102" /><meta name="citation_reference" content="citation_title=A mobile threat to genome stability: the impact of non-LTR retrotransposons upon the human genome;  citation_journal_title=Semin. Cancer Biol;  citation_year=2010;  citation_volume=20;  citation_pages=211-21; " /><meta name="citation_reference" content="citation_title=Active human retrotransposons: variation and disease;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2012;  citation_volume=22;  citation_pages=191-203; " /><meta name="citation_reference" content="citation_title=Retrotransposable elements and human disease;  citation_journal_title=Genome Dyn;  citation_year=2006;  citation_volume=1;  citation_pages=104-15; " /><meta name="citation_reference" content="citation_title=Transposable elements are a significant contributor to tandem repeats in the human genome;  citation_journal_title=Comp. Funct. Genomics;  citation_year=2012;  citation_volume=2012;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Latent regulatory potential of human-specific repetitive elements;  citation_journal_title=Mol. Cell;  citation_year=2013;  citation_volume=49;  citation_pages=262-72; " /><meta name="citation_reference" content="citation_title=Regulatory evolution of innate immunity through co-option of endogenous retroviruses;  citation_journal_title=Science;  citation_year=2016;  citation_volume=351;  citation_pages=1083-7; " /><meta name="citation_reference" content="citation_title=Mobile interspersed repeats are major structural variants in the human genome;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=1171-82; " /><meta name="citation_reference" content="citation_title=The impact of L1 retrotransposons on the human genome;  citation_journal_title=Nat. Genet;  citation_year=1998;  citation_volume=19;  citation_pages=19-24; " /><meta name="citation_reference" content="citation_title=Recently mobilized transposons in the human and chimpanzee genomes;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2006;  citation_volume=78;  citation_pages=671-9; " /><meta name="citation_reference" content="citation_title=Whole genome computational comparative genomics: a fruitful approach for ascertaining Alu insertion polymorphisms;  citation_journal_title=Gene;  citation_year=2006;  citation_volume=365;  citation_pages=11-20; " /><meta name="citation_reference" content="citation_title=Identification of three new Alu Yb subfamilies by source tracking of recently integrated Alu Yb elements;  citation_journal_title=Mob. DNA;  citation_year=2013;  citation_volume=4;  citation_pages=25" /><meta name="citation_reference" content="citation_title=Human-specific subfamilies of HERV-K (HML-2) long terminal repeats: three master genes were active simultaneously during branching of hominoid lineages;  citation_journal_title=Genomics;  citation_year=2003;  citation_volume=81;  citation_pages=149-56; " /><meta name="citation_reference" content="citation_title=Human-specific HERV-K insertion causes genomic variations in the human genome;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e60605" /><meta name="citation_reference" content="citation_title=Initial sequence of the chimpanzee genome and comparison with the human genome;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=437;  citation_pages=69-87; " /><meta name="citation_reference" content="citation_title=Long-read sequence assembly of the gorilla genome;  citation_journal_title=Science;  citation_year=2016;  citation_volume=352;  citation_pages=aae0344" /><meta name="citation_reference" content="citation_title=Comparative and demographic analysis of orang-utan genomes;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=469;  citation_pages=529-33; " /><meta name="citation_reference" content="citation_title=Gibbon genome and the fast karyotype evolution of small apes;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=513;  citation_pages=195-201; " /><meta name="citation_reference" content="citation_title=Evolutionary and biomedical insights from the rhesus macaque genome; citation_author=Rhesus Macaque Genome, S;  citation_journal_title=Science;  citation_year=2007;  citation_volume=316;  citation_pages=222-34; " /><meta name="citation_reference" content="citation_title=A new rhesus macaque assembly and annotation for next-generation sequencing analyses;  citation_journal_title=Biol. Direct;  citation_year=2014;  citation_volume=9;  citation_pages=20" /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=dbRIP: a highly integrated database of retrotransposon insertion polymorphisms in humans;  citation_journal_title=Hum. Mutat;  citation_year=2006;  citation_volume=27;  citation_pages=323-9; " /><meta name="citation_reference" content="citation_title=A comprehensive map of mobile element insertion polymorphisms in humans;  citation_journal_title=PLoS Genet;  citation_year=2011;  citation_volume=7;  citation_pages=e1002236" /><meta name="citation_reference" content="citation_title=An integrated map of structural variation in 2, 504 human genomes;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=526;  citation_pages=75-81; " /><meta name="citation_reference" content="citation_title=Chimpanzee and human Y chromosomes are remarkably divergent in structure and gene content;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=536-9; " /><meta name="citation_reference" content="citation_title=GENCODE: the reference human genome annotation for The ENCODE Project;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=1760-74; " /><meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D61-5; " /><meta name="citation_reference" content="citation_title=An initial map of insertion and deletion (INDEL) variation in the human genome;  citation_journal_title=Genome Res;  citation_year=2006;  citation_volume=16;  citation_pages=1182-90; " /><meta name="citation_reference" content="citation_title=Targeting of human retrotransposon integration is directed by the specificity of the L1 endonuclease for regions of unusual DNA structure;  citation_journal_title=Biochemistry;  citation_year=1998;  citation_volume=37;  citation_pages=18081-93; " /><meta name="citation_reference" content="citation_title=Sequence patterns indicate an enzymatic involvement in integration of mammalian retroposons;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=1997;  citation_volume=94;  citation_pages=1872-7; " /><meta name="citation_reference" content="citation_title=Targeting survival: integration site selection by retroviruses and LTR-retrotransposons;  citation_journal_title=Cell;  citation_year=2003;  citation_volume=115;  citation_pages=135-8; " /><meta name="citation_reference" content="citation_title=A map of human genome variation from population-scale sequencing;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=467;  citation_pages=1061-73; " /><meta name="citation_reference" content="citation_title=Finishing the euchromatic sequence of the human genome; citation_author=International Human Genome Sequencing, C;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=431;  citation_pages=931-45; " /><meta name="citation_reference" content="citation_title=A highly annotated whole-genome sequence of a Korean individual;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=460;  citation_pages=1011-5; " /><meta name="citation_reference" content="citation_title=Many human endogenous retrovirus K (HERV-K) proviruses are unique to humans;  citation_journal_title=Curr. Biol;  citation_year=1999;  citation_volume=9;  citation_pages=861-8; " /><meta name="citation_reference" content="citation_title=Active Alu retrotransposons in the human genome;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=1875-83; " /><meta name="citation_reference" content="citation_title=Human-mouse alignments with BLASTZ;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=103-7; " /><meta name="citation_reference" content="citation_title=Processed pseudogene insertions in somatic cells;  citation_journal_title=Mob. DNA;  citation_year=2014;  citation_volume=5;  citation_pages=20" /><meta name="citation_reference" content="citation_title=The genomic impact of gene retrocopies: what have we learned from comparative genomics, population genomics, and transcriptomic analyses?;  citation_journal_title=Genome Biol. Evol;  citation_year=2017;  citation_volume=9;  citation_pages=1351-73; " /><meta name="citation_reference" content="citation_title=Retrotransposition of gene transcripts leads to structural variation in mammalian genomes;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R22" /><meta name="citation_reference" content="citation_title=Inter-population differences in retrogene loss and expression in humans;  citation_journal_title=PLoS Genet;  citation_year=2015;  citation_volume=11;  citation_pages=e1005579" /><meta name="citation_reference" content="citation_title=Identification and characterization of novel polymorphic LINE-1 insertions through comparison of two human genome sequence assemblies;  citation_journal_title=Gene;  citation_year=2007;  citation_volume=390;  citation_pages=28-38; " /><meta name="citation_reference" content="citation_title=Different evolutionary fates of recently integrated human and chimpanzee LINE-1 retrotransposons;  citation_journal_title=Gene;  citation_year=2007;  citation_volume=390;  citation_pages=18-27; " /><meta name="citation_reference" content="citation_title=Discovery of unfixed endogenous retrovirus insertions in diverse human populations;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2016;  citation_volume=113;  citation_pages=E2326-34; " /><meta name="citation_reference" content="citation_title=Natural mutagenesis of human genomes by endogenous retrotransposons;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=1253-61; " /><meta name="citation_reference" content="citation_title=Mobile element scanning (ME-Scan) identifies thousands of novel Alu insertions in diverse human populations;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=1170-81; " /><meta name="citation_reference" content="citation_title=Biased distributions and decay of long interspersed nuclear elements in the chicken genome;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=178;  citation_pages=573-81; " /><meta name="citation_reference" content="citation_title=Selection against deleterious LINE-1-containing loci in the human lineage;  citation_journal_title=Mol. Biol. Evol;  citation_year=2001;  citation_volume=18;  citation_pages=926-35; " /><meta name="citation_reference" content="citation_title=Is the Y chromosome disappearing? Both sides of the argument;  citation_journal_title=Chromosome Res;  citation_year=2012;  citation_volume=20;  citation_pages=35-45; " /><meta name="citation_reference" content="citation_title=Genetic and physical maps of Saccharomyces cerevisiae;  citation_journal_title=Nature;  citation_year=1997;  citation_volume=387;  citation_pages=67-73; " /><meta name="citation_reference" content="citation_title=Primate segmental duplications: crucibles of evolution, diversity and disease;  citation_journal_title=Nat. Rev. Genet;  citation_year=2006;  citation_volume=7;  citation_pages=552-64; " /><meta name="citation_reference" content="citation_title=Expansion of a novel endogenous retrovirus throughout the pericentromeres of modern humans;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=74" /><meta name="citation_reference" content="citation_title=The complete genome sequence of a Neanderthal from the Altai Mountains;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=505;  citation_pages=43-9; " /><meta name="citation_reference" content="citation_title=An early modern human from Romania with a recent Neanderthal ancestor;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=524;  citation_pages=216-9; " /><meta name="citation_reference" content="citation_title=Archaic human genomics;  citation_journal_title=Am. J. Phys. Anthropol;  citation_year=2012;  citation_volume=149(Suppl 55);  citation_pages=24-39; " /><meta name="citation_reference" content="citation_title=Study of modern human evolution via comparative analysis with the Neanderthal Genome;  citation_journal_title=Genomics Inform;  citation_year=2013;  citation_volume=11;  citation_pages=230-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/5/521/26076917/dsy022.pdf" /><meta name="description" content="Abstract.  Mobile elements (MEs) collectively contribute to at least 50% of the human genome. Due to their past incremental accumulation and ongoing DNA transpo" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/5/521/5056130" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/5/521/5056130" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/5/521/5056130");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy022';
    </script>
<title>
	Mobile elements contribute to the uniqueness of human genome with 15,000 human-specific insertions and 14 Mbp sequence increase | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5056130" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd6eATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuZywgV2FueGlhbmdmdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgQnJvY2sgVW5pdmVyc2l0eSwgU3QuIENhdGhhcmluZXMsIE9OLCBDYW5hZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNdW4sIFNleW91bmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE5hbm9iaW9tZWRpY2FsIFNjaWVuY2UgJmFtcDsgQksyMSBQTFVTIE5CTSBHbG9iYWwgUmVzZWFyY2gsIENlbnRlciBmb3IgUmVnZW5lcmF0aXZlIE1lZGljaW5lLCBEYW5rb29rIFVuaXZlcnNpdHksIENoZW9uYW4sIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSm9zaGksIEFkaXR5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgQnJvY2sgVW5pdmVyc2l0eSwgU3QuIENhdGhhcmluZXMsIE9OLCBDYW5hZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYW4sIEt5dWRvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE5hbm9iaW9tZWRpY2FsIFNjaWVuY2UgJmFtcDsgQksyMSBQTFVTIE5CTSBHbG9iYWwgUmVzZWFyY2gsIENlbnRlciBmb3IgUmVnZW5lcmF0aXZlIE1lZGljaW5lLCBEYW5rb29rIFVuaXZlcnNpdHksIENoZW9uYW4sIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGlhbmcsIFBpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIEJyb2NrIFVuaXZlcnNpdHksIFN0LiBDYXRoYXJpbmVzLCBPTiwgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJNb2JpbGUgZWxlbWVudHMgY29udHJpYnV0ZSB0byB0aGUgdW5pcXVlbmVzcyBvZiBodW1hbiBnZW5vbWUgd2l0aCAxNSwwMDAgaHVtYW4tc3BlY2lmaWMgaW5zZXJ0aW9ucyBhbmQgMTTCoE1icCBzZXF1ZW5jZSBpbmNyZWFzZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjUyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN5MDIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTgvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgZWxlbWVudHMgYW5kIG1hbW1hbGlhbiBnZW5vbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2OyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbXBhY3Qgb2YgcmV0cm90cmFuc3Bvc29ucyBvbiBodW1hbiBnZW5vbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02OTEtNzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgZWxlbWVudHM6IGRyaXZlcnMgb2YgZ2Vub21lIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDM7ICBjaXRhdGlvbl9wYWdlcz0xNjI2LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9sb2d5IG9mIG1hbW1hbGlhbiBMMSByZXRyb3RyYW5zcG9zb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9NTAxLTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHUgcmVwZWF0cyBhbmQgaHVtYW4gZ2Vub21pYyBkaXZlcnNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MzcwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxJTkUgZHJpdmUuIHJldHJvdHJhbnNwb3NpdGlvbiBhbmQgZ2Vub21lIGluc3RhYmlsaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTI3Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hpY2ggdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGFyZSBhY3RpdmUgaW4gdGhlIGh1bWFuIGdlbm9tZT87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE4My05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1ZBIGVsZW1lbnRzIGFyZSBub25hdXRvbm9tb3VzIHJldHJvdHJhbnNwb3NvbnMgdGhhdCBjYXVzZSBkaXNlYXNlIGluIGh1bWFuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz0xNDQ0LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVkEgZWxlbWVudHM6IGEgaG9taW5pZC1zcGVjaWZpYyByZXRyb3Bvc29uIGZhbWlseTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM1NDsgIGNpdGF0aW9uX3BhZ2VzPTk5NC0xMDA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxmaXNoIGdlbmVzLCB0aGUgcGhlbm90eXBlIHBhcmFkaWdtIGFuZCBnZW5vbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9Mjg0OyAgY2l0YXRpb25fcGFnZXM9NjAxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIGwxIHJldHJvdHJhbnNwb3NpdGlvbiBpcyBhc3NvY2lhdGVkIHdpdGggZ2VuZXRpYyBpbnN0YWJpbGl0eSBpbiB2aXZvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTMyNy0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgcmVsYXRlZCBMMSByZXRyb3RyYW5zcG9zb25zIGJ5IGFuYWx5emluZyAzJyB0cmFuc2R1Y2VkIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPVIzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBkaXNydXB0aW9uIGJ5IHRoZSBMMSByZXRyb3RyYW5zcG9zb24gYW5kIGltcGxpY2F0aW9ucyBmb3IgbWFtbWFsaWFuIHRyYW5zY3JpcHRvbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDI5OyAgY2l0YXRpb25fcGFnZXM9MjY4LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lLWJyZWFraW5nOiBhIG5ldyBwYXJhZGlnbSBmb3IgaHVtYW4gcmV0cm90cmFuc3Bvc29uLW1lZGlhdGVkIGdlbmUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTA3My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgcmV0cm90cmFuc3Bvc29ucyBzaGFwZSBnZW5vbWUgcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gR2VuZXQuIERldjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTkwLTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx1IHJldHJvdHJhbnNwb3NpdGlvbi1tZWRpYXRlZCBkZWxldGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM0ODsgIGNpdGF0aW9uX3BhZ2VzPTc5MS04MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgcmVhcnJhbmdlbWVudHMgYnkgTElORS0xIGluc2VydGlvbi1tZWRpYXRlZCBkZWxldGlvbiBpbiB0aGUgaHVtYW4gYW5kIGNoaW1wYW56ZWUgbGluZWFnZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NDA0MC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gZ2Vub21pYyBkZWxldGlvbnMgbWVkaWF0ZWQgYnkgcmVjb21iaW5hdGlvbiBiZXR3ZWVuIEFsdSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Nzk7ICBjaXRhdGlvbl9wYWdlcz00MS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx1IHJlY29tYmluYXRpb24tbWVkaWF0ZWQgc3RydWN0dXJhbCBkZWxldGlvbnMgaW4gdGhlIGNoaW1wYW56ZWUgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xOTM5LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVkEgcmV0cm90cmFuc3Bvc29ucyBhcyBtb2R1bGF0b3JzIG9mIGdlbmUgZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9iLiBHZW5ldC4gRWxlbWVudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWUzMjEwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW9iaWxlIHRocmVhdCB0byBnZW5vbWUgc3RhYmlsaXR5OiB0aGUgaW1wYWN0IG9mIG5vbi1MVFIgcmV0cm90cmFuc3Bvc29ucyB1cG9uIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNlbWluLiBDYW5jZXIgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIxMS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWN0aXZlIGh1bWFuIHJldHJvdHJhbnNwb3NvbnM6IHZhcmlhdGlvbiBhbmQgZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gR2VuZXQuIERldjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE5MS0yMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldHJvdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGFuZCBodW1hbiBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgRHluOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMDQtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zYWJsZSBlbGVtZW50cyBhcmUgYSBzaWduaWZpY2FudCBjb250cmlidXRvciB0byB0YW5kZW0gcmVwZWF0cyBpbiB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wLiBGdW5jdC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjAxMjsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXRlbnQgcmVndWxhdG9yeSBwb3RlbnRpYWwgb2YgaHVtYW4tc3BlY2lmaWMgcmVwZXRpdGl2ZSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MjYyLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0b3J5IGV2b2x1dGlvbiBvZiBpbm5hdGUgaW1tdW5pdHkgdGhyb3VnaCBjby1vcHRpb24gb2YgZW5kb2dlbm91cyByZXRyb3ZpcnVzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzUxOyAgY2l0YXRpb25fcGFnZXM9MTA4My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgaW50ZXJzcGVyc2VkIHJlcGVhdHMgYXJlIG1ham9yIHN0cnVjdHVyYWwgdmFyaWFudHMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0xMTcxLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW1wYWN0IG9mIEwxIHJldHJvdHJhbnNwb3NvbnMgb24gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE5LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnRseSBtb2JpbGl6ZWQgdHJhbnNwb3NvbnMgaW4gdGhlIGh1bWFuIGFuZCBjaGltcGFuemVlIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBIdW0uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9NjcxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlIGdlbm9tZSBjb21wdXRhdGlvbmFsIGNvbXBhcmF0aXZlIGdlbm9taWNzOiBhIGZydWl0ZnVsIGFwcHJvYWNoIGZvciBhc2NlcnRhaW5pbmcgQWx1IGluc2VydGlvbiBwb2x5bW9ycGhpc21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM2NTsgIGNpdGF0aW9uX3BhZ2VzPTExLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiB0aHJlZSBuZXcgQWx1IFliIHN1YmZhbWlsaWVzIGJ5IHNvdXJjZSB0cmFja2luZyBvZiByZWNlbnRseSBpbnRlZ3JhdGVkIEFsdSBZYiBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9iLiBETkE7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4tc3BlY2lmaWMgc3ViZmFtaWxpZXMgb2YgSEVSVi1LIChITUwtMikgbG9uZyB0ZXJtaW5hbCByZXBlYXRzOiB0aHJlZSBtYXN0ZXIgZ2VuZXMgd2VyZSBhY3RpdmUgc2ltdWx0YW5lb3VzbHkgZHVyaW5nIGJyYW5jaGluZyBvZiBob21pbm9pZCBsaW5lYWdlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz0xNDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuLXNwZWNpZmljIEhFUlYtSyBpbnNlcnRpb24gY2F1c2VzIGdlbm9taWMgdmFyaWF0aW9ucyBpbiB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTYwNjA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5pdGlhbCBzZXF1ZW5jZSBvZiB0aGUgY2hpbXBhbnplZSBnZW5vbWUgYW5kIGNvbXBhcmlzb24gd2l0aCB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM3OyAgY2l0YXRpb25fcGFnZXM9NjktODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmctcmVhZCBzZXF1ZW5jZSBhc3NlbWJseSBvZiB0aGUgZ29yaWxsYSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzUyOyAgY2l0YXRpb25fcGFnZXM9YWFlMDM0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGFuZCBkZW1vZ3JhcGhpYyBhbmFseXNpcyBvZiBvcmFuZy11dGFuIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00Njk7ICBjaXRhdGlvbl9wYWdlcz01MjktMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdpYmJvbiBnZW5vbWUgYW5kIHRoZSBmYXN0IGthcnlvdHlwZSBldm9sdXRpb24gb2Ygc21hbGwgYXBlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUxMzsgIGNpdGF0aW9uX3BhZ2VzPTE5NS0yMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBhbmQgYmlvbWVkaWNhbCBpbnNpZ2h0cyBmcm9tIHRoZSByaGVzdXMgbWFjYXF1ZSBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1SaGVzdXMgTWFjYXF1ZSBHZW5vbWUsIFM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE2OyAgY2l0YXRpb25fcGFnZXM9MjIyLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyByaGVzdXMgbWFjYXF1ZSBhc3NlbWJseSBhbmQgYW5ub3RhdGlvbiBmb3IgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgYW5hbHlzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2wuIERpcmVjdDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CTEFU4oCUdGhlIEJMQVNULWxpa2UgYWxpZ25tZW50IHRvb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTYtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWRiUklQOiBhIGhpZ2hseSBpbnRlZ3JhdGVkIGRhdGFiYXNlIG9mIHJldHJvdHJhbnNwb3NvbiBpbnNlcnRpb24gcG9seW1vcnBoaXNtcyBpbiBodW1hbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gTXV0YXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0zMjMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wcmVoZW5zaXZlIG1hcCBvZiBtb2JpbGUgZWxlbWVudCBpbnNlcnRpb24gcG9seW1vcnBoaXNtcyBpbiBodW1hbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyMjM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBtYXAgb2Ygc3RydWN0dXJhbCB2YXJpYXRpb24gaW4gMiwgNTA0IGh1bWFuIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01MjY7ICBjaXRhdGlvbl9wYWdlcz03NS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hpbXBhbnplZSBhbmQgaHVtYW4gWSBjaHJvbW9zb21lcyBhcmUgcmVtYXJrYWJseSBkaXZlcmdlbnQgaW4gc3RydWN0dXJlIGFuZCBnZW5lIGNvbnRlbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz01MzYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0VOQ09ERTogdGhlIHJlZmVyZW5jZSBodW1hbiBnZW5vbWUgYW5ub3RhdGlvbiBmb3IgVGhlIEVOQ09ERSBQcm9qZWN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTc2MC03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkNCSSByZWZlcmVuY2Ugc2VxdWVuY2VzIChSZWZTZXEpOiBhIGN1cmF0ZWQgbm9uLXJlZHVuZGFudCBzZXF1ZW5jZSBkYXRhYmFzZSBvZiBnZW5vbWVzLCB0cmFuc2NyaXB0cyBhbmQgcHJvdGVpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDYxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGluaXRpYWwgbWFwIG9mIGluc2VydGlvbiBhbmQgZGVsZXRpb24gKElOREVMKSB2YXJpYXRpb24gaW4gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTExODItOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGluZyBvZiBodW1hbiByZXRyb3RyYW5zcG9zb24gaW50ZWdyYXRpb24gaXMgZGlyZWN0ZWQgYnkgdGhlIHNwZWNpZmljaXR5IG9mIHRoZSBMMSBlbmRvbnVjbGVhc2UgZm9yIHJlZ2lvbnMgb2YgdW51c3VhbCBETkEgc3RydWN0dXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xODA4MS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgcGF0dGVybnMgaW5kaWNhdGUgYW4gZW56eW1hdGljIGludm9sdmVtZW50IGluIGludGVncmF0aW9uIG9mIG1hbW1hbGlhbiByZXRyb3Bvc29uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLlMuQTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTE4NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0aW5nIHN1cnZpdmFsOiBpbnRlZ3JhdGlvbiBzaXRlIHNlbGVjdGlvbiBieSByZXRyb3ZpcnVzZXMgYW5kIExUUi1yZXRyb3RyYW5zcG9zb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTEzNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1hcCBvZiBodW1hbiBnZW5vbWUgdmFyaWF0aW9uIGZyb20gcG9wdWxhdGlvbi1zY2FsZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDY3OyAgY2l0YXRpb25fcGFnZXM9MTA2MS03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluaXNoaW5nIHRoZSBldWNocm9tYXRpYyBzZXF1ZW5jZSBvZiB0aGUgaHVtYW4gZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBIdW1hbiBHZW5vbWUgU2VxdWVuY2luZywgQzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQzMTsgIGNpdGF0aW9uX3BhZ2VzPTkzMS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdobHkgYW5ub3RhdGVkIHdob2xlLWdlbm9tZSBzZXF1ZW5jZSBvZiBhIEtvcmVhbiBpbmRpdmlkdWFsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDYwOyAgY2l0YXRpb25fcGFnZXM9MTAxMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYW55IGh1bWFuIGVuZG9nZW5vdXMgcmV0cm92aXJ1cyBLIChIRVJWLUspIHByb3ZpcnVzZXMgYXJlIHVuaXF1ZSB0byBodW1hbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg2MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3RpdmUgQWx1IHJldHJvdHJhbnNwb3NvbnMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE4NzUtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuLW1vdXNlIGFsaWdubWVudHMgd2l0aCBCTEFTVFo7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xMDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvY2Vzc2VkIHBzZXVkb2dlbmUgaW5zZXJ0aW9ucyBpbiBzb21hdGljIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2IuIEROQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21pYyBpbXBhY3Qgb2YgZ2VuZSByZXRyb2NvcGllczogd2hhdCBoYXZlIHdlIGxlYXJuZWQgZnJvbSBjb21wYXJhdGl2ZSBnZW5vbWljcywgcG9wdWxhdGlvbiBnZW5vbWljcywgYW5kIHRyYW5zY3JpcHRvbWljIGFuYWx5c2VzPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEzNTEtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldHJvdHJhbnNwb3NpdGlvbiBvZiBnZW5lIHRyYW5zY3JpcHRzIGxlYWRzIHRvIHN0cnVjdHVyYWwgdmFyaWF0aW9uIGluIG1hbW1hbGlhbiBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPVIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyLXBvcHVsYXRpb24gZGlmZmVyZW5jZXMgaW4gcmV0cm9nZW5lIGxvc3MgYW5kIGV4cHJlc3Npb24gaW4gaHVtYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDU1NzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBub3ZlbCBwb2x5bW9ycGhpYyBMSU5FLTEgaW5zZXJ0aW9ucyB0aHJvdWdoIGNvbXBhcmlzb24gb2YgdHdvIGh1bWFuIGdlbm9tZSBzZXF1ZW5jZSBhc3NlbWJsaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5MDsgIGNpdGF0aW9uX3BhZ2VzPTI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnQgZXZvbHV0aW9uYXJ5IGZhdGVzIG9mIHJlY2VudGx5IGludGVncmF0ZWQgaHVtYW4gYW5kIGNoaW1wYW56ZWUgTElORS0xIHJldHJvdHJhbnNwb3NvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzkwOyAgY2l0YXRpb25fcGFnZXM9MTgtMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBvZiB1bmZpeGVkIGVuZG9nZW5vdXMgcmV0cm92aXJ1cyBpbnNlcnRpb25zIGluIGRpdmVyc2UgaHVtYW4gcG9wdWxhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9RTIzMjYtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5hdHVyYWwgbXV0YWdlbmVzaXMgb2YgaHVtYW4gZ2Vub21lcyBieSBlbmRvZ2Vub3VzIHJldHJvdHJhbnNwb3NvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9MTI1My02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9iaWxlIGVsZW1lbnQgc2Nhbm5pbmcgKE1FLVNjYW4pIGlkZW50aWZpZXMgdGhvdXNhbmRzIG9mIG5vdmVsIEFsdSBpbnNlcnRpb25zIGluIGRpdmVyc2UgaHVtYW4gcG9wdWxhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMTcwLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaWFzZWQgZGlzdHJpYnV0aW9ucyBhbmQgZGVjYXkgb2YgbG9uZyBpbnRlcnNwZXJzZWQgbnVjbGVhciBlbGVtZW50cyBpbiB0aGUgY2hpY2tlbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3ODsgIGNpdGF0aW9uX3BhZ2VzPTU3My04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW9uIGFnYWluc3QgZGVsZXRlcmlvdXMgTElORS0xLWNvbnRhaW5pbmcgbG9jaSBpbiB0aGUgaHVtYW4gbGluZWFnZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9OTI2LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JcyB0aGUgWSBjaHJvbW9zb21lIGRpc2FwcGVhcmluZz8gQm90aCBzaWRlcyBvZiB0aGUgYXJndW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNocm9tb3NvbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MzUtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5kIHBoeXNpY2FsIG1hcHMgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mzg3OyAgY2l0YXRpb25fcGFnZXM9NjctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1hdGUgc2VnbWVudGFsIGR1cGxpY2F0aW9uczogY3J1Y2libGVzIG9mIGV2b2x1dGlvbiwgZGl2ZXJzaXR5IGFuZCBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTU1Mi02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwYW5zaW9uIG9mIGEgbm92ZWwgZW5kb2dlbm91cyByZXRyb3ZpcnVzIHRocm91Z2hvdXQgdGhlIHBlcmljZW50cm9tZXJlcyBvZiBtb2Rlcm4gaHVtYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBhIE5lYW5kZXJ0aGFsIGZyb20gdGhlIEFsdGFpIE1vdW50YWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUwNTsgIGNpdGF0aW9uX3BhZ2VzPTQzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGVhcmx5IG1vZGVybiBodW1hbiBmcm9tIFJvbWFuaWEgd2l0aCBhIHJlY2VudCBOZWFuZGVydGhhbCBhbmNlc3RvcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTUyNDsgIGNpdGF0aW9uX3BhZ2VzPTIxNi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmNoYWljIGh1bWFuIGdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gUGh5cy4gQW50aHJvcG9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OShTdXBwbCA1NSk7ICBjaXRhdGlvbl9wYWdlcz0yNC0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3R1ZHkgb2YgbW9kZXJuIGh1bWFuIGV2b2x1dGlvbiB2aWEgY29tcGFyYXRpdmUgYW5hbHlzaXMgd2l0aCB0aGUgTmVhbmRlcnRoYWwgR2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljcyBJbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yMzAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzUvNTIxLzI2MDc2OTE3L2RzeTAyMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTW9iaWxlIGVsZW1lbnRzIChNRXMpIGNvbGxlY3RpdmVseSBjb250cmlidXRlIHRvIGF0IGxlYXN0IDUwJSBvZiB0aGUgaHVtYW4gZ2Vub21lLiBEdWUgdG8gdGhlaXIgcGFzdCBpbmNyZW1lbnRhbCBhY2N1bXVsYXRpb24gYW5kIG9uZ29pbmcgRE5BIHRyYW5zcG8iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUwNTYxMzAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkHQufXVkGMivZqh4nHbGkv0CPCmQ=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f521%2f5056130" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f521%2f5056130" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=110919" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/5">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="122670813">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670813">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670815">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670815">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670821">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670821">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670846">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670846">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670875">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670875">4. Discussions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670901">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670901">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670905">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670905">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/5/511/5047107">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/5/535/5055375">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Mobile elements contribute to the uniqueness of human genome with 15,000 human-specific insertions and 14 Mbp sequence increase
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wanxiangfu Tang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biological Sciences, Brock University, St. Catharines, ON, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wanxiangfu+Tang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tang W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wanxiangfu Tang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seyoung Mun</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Nanobiomedical Science &amp; BK21 PLUS NBM Global Research, Center for Regenerative Medicine, Dankook University, Cheonan, Republic of Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seyoung+Mun">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mun S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mun S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seyoung Mun</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aditya Joshi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biological Sciences, Brock University, St. Catharines, ON, Canada</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aditya+Joshi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Joshi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Joshi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aditya Joshi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyudong Han</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Nanobiomedical Science &amp; BK21 PLUS NBM Global Research, Center for Regenerative Medicine, Dankook University, Cheonan, Republic of Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyudong+Han">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Han K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Han K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kyudong Han</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ping Liang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biological Sciences, Brock University, St. Catharines, ON, Canada</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy022-cor1" data-open="dsy022-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +1 905 688 5550; Ext: 5922. Fax. +1 905 688 1855. Email: <a href="mailto:pliang@brocku.ca" target="_blank">pliang@brocku.ca</a>
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-4423-0636">
                <img class="orchid-icon" alt="orchid id" src="/UI/app/img/mini-icon.png"/>&nbsp;&nbsp;http://orcid.org/0000-0003-4423-0636
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ping+Liang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liang P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liang P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ping Liang</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 521–533, <a href='https://doi.org/10.1093/dnares/dsy022'>https://doi.org/10.1093/dnares/dsy022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">19 July 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 March 2018</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">20 June 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5056130" href="/dnaresearch/article-pdf/25/5/521/26076917/dsy022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Wanxiangfu Tang, Seyoung Mun, Aditya Joshi, Kyudong Han, Ping Liang;  Mobile elements contribute to the uniqueness of human genome with 15,000 human-specific insertions and 14 Mbp sequence increase, <em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 521–533, <a href='https://doi.org/10.1093/dnares/dsy022'>https://doi.org/10.1093/dnares/dsy022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5056130?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5056130?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5056130?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5056130?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5056130?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5056130?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5056130?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Mobile%20elements%20contribute%20to%20the%20uniqueness%20of%20human%20genome%20with%2015%2C000%20human-specific%20insertions%20and%2014%C2%A0Mbp%20sequence%20increase&amp;publicationDate=2018-07-19&amp;volumeNum=25&amp;issueNum=5&amp;author=Tang%2C%20Wanxiangfu%3B%20Mun%2C%20Seyoung&amp;startPage=521&amp;endPage=533&amp;contentId=10.1093%2Fdnares%2Fdsy022&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f521%2f5056130" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=122670813 id="122670813" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Mobile elements (MEs) collectively contribute to at least 50% of the human genome. Due to their past incremental accumulation and ongoing DNA transposition, MEs serve as a significant source for both inter- and intra-species genetic and phenotypic diversity during primate and human evolution. By making use of the most recent genome sequences for human and many other closely related primates and robust multi-way comparative genomic approach, we identified a total of 14,870 human-specific MEs (HS-MEs) with more than 8,000 being newly identified. Collectively, these HS-MEs contribute to a total of 14.2 Mbp net genome sequence increase. Several new observations were made based on these HS-MEs, including the finding of Y chromosome as a strikingly hot target for HS-MEs and a strong mutual preference for SINE-R/VNTR/Alu (SVAs). Furthermore, ∼8,000 of these HS-MEs were found to locate in the vicinity of ∼4,900 genes, and collectively they contribute to ∼84 kb sequences in the human reference transcriptome in association with over 300 genes, including protein-coding sequences for 40 genes. In conclusion, our results demonstrate that MEs made a significant contribution to the evolution of human genome by participating in gene function in a human-specific fashion. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mobile elements&quot;">mobile elements</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;human genome&quot;">human genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a></div>                            <h2 scrollto-destination=122670815 id="122670815" class="section-title" >1. Introduction</h2>
<p>Like in most genomes of higher organisms, transposable elements or mobile elements (‘MEs’ hereafter) constitute a major component in the human genome. The percentage of MEs in the human genome was reported to be from 45 to 48.5%,<a reveal-id="dsy022-B1 dsy022-B2 dsy022-B3" data-open="dsy022-B1 dsy022-B2 dsy022-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> and in this study we revised it to 52.1% based on the most recent reference genome sequences and ME annotation. This percentage is likely to increase slightly with further improvements of the human genome sequences, especially for the constitutive heterochromatin regions, and with tools capable of detecting more diverged and novel MEs.</p><p>In the human genome, MEs are mainly represented by retrotransposons, which propagate in a copy-and-paste fashion via transcribed RNAs as the intermediates, and they are divided into long-terminal repeat (LTR) and non-LTR retrotransposons.<a reveal-id="dsy022-B2" data-open="dsy022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy022-B3" data-open="dsy022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The LTR retrotransposon group is characterized by the presence of LTRs at the two ends of the internal viral sequences, and it is mainly represented by the endogenous retrovirus (ERVs) or human ERVs (HERVs). These ERVs came from exogenous virus affecting germline cells and integrating into the genomes during different stages of primate and human evolution, and they constitute ∼8% of the human genome.<a reveal-id="dsy022-B4" data-open="dsy022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> The non-LTR retrotransposon group consists of several very distinct types, including Short-INterspersed Elements (SINEs), Long INterspersed Elements (LINEs), and a chimeric type of repeat elements (SINE-R/VNTR/Alu or SVA). Non-LTR retrotransposons share the common properties of having a 3′ poly(A) tail and L1-based target-primed reverse transcription (TPRT) mechanism for retrotransposition.<a reveal-id="dsy022-B4" data-open="dsy022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy022-B5" data-open="dsy022-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Among non-LTR MEs, <em>Alu</em> elements, representing the relative young and most successful SINEs by number in primates, have more than 1 million copies and contribute to ∼13% of the human genome.<a reveal-id="dsy022-B4" data-open="dsy022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy022-B6" data-open="dsy022-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> L1s, representing the dominant group of the LINEs, have more than half million copies and make the largest contribution to the human genome by sequence length (∼17%). L1s with full coding capacity are also responsible for transposing all non-LTR MEs.<a reveal-id="dsy022-B7" data-open="dsy022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsy022-B8" data-open="dsy022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> SVAs, emerged as the newest and most active group of retrotransposons that are mainly found within the hominid group of primates, have ∼5,000 copies and contribute to ∼0.1% of the human genome.<a reveal-id="dsy022-B9" data-open="dsy022-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy022-B10" data-open="dsy022-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>Despite initially being considered to junk DNA, implying that they had no function,<a reveal-id="dsy022-B11" data-open="dsy022-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> research work, mostly from the last two decades, has convincingly demonstrated that MEs make very significant contributions in shaping the evolution of genomes and impacting gene function via a variety of mechanisms. These mechanisms range from generation of insertional mutations and causing genomic instability, creation of new genes and splicing isoforms, and exon shuffling to alteration of gene expression and epigenetic regulation.<a reveal-id="dsy022-B12 dsy022-B13 dsy022-B14 dsy022-B15 dsy022-B16 dsy022-B17 dsy022-B18 dsy022-B19 dsy022-B20 dsy022-B21 dsy022-B22" data-open="dsy022-B12 dsy022-B13 dsy022-B14 dsy022-B15 dsy022-B16 dsy022-B17 dsy022-B18 dsy022-B19 dsy022-B20 dsy022-B21 dsy022-B22" class="link link-ref link-reveal xref-bibr"><sup>12–22</sup></a> Functioning also as endogenous insertional mutagens, MEs are known to be responsible for causing certain genetic diseases in humans.<a reveal-id="dsy022-B23" data-open="dsy022-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsy022-B24" data-open="dsy022-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a></p><p>More recently, MEs are also shown to contribute to the generation of tandem repeats and providing definitive regulatory function or potentials. It was shown that at least 23% of all tandem repeats, another type of repetitive elements, were derived from transposable elements.<a reveal-id="dsy022-B25" data-open="dsy022-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> In a recent study, Ward et al. demonstrated that under a heterologous regulatory environment, regulatory sites in MEs, including those specific to humans, can be activated to alter histone modifications and DNA methylation as well as expression of nearby genes in both germline and somatic cells.<a reveal-id="dsy022-B26" data-open="dsy022-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> A profound implication of this observation is that lineage- and species-specific MEs can provide novel regulatory sites to the host genome, which can potentially regulate nearby genes’ expression in a lineage- and species-specific manner and lead to phenotypic differences. A very recent study added such an example by showing that an ERV element is responsible for regulating innate immunity in humans by controlling the expression of adjacent IFN-induced genes.<a reveal-id="dsy022-B27" data-open="dsy022-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a></p><p>Past and ongoing retrotransposition generates genetic diversity among species and among individuals within the same species.<a reveal-id="dsy022-B3" data-open="dsy022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy022-B28 dsy022-B29 dsy022-B30" data-open="dsy022-B28 dsy022-B29 dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>28–30</sup></a> Therefore, analysis of species-specific MEs can help understand the roles of MEs in speciation and species-specific phenotypes. In the human genome, certain members from L1, <em>Alu</em>, SVA, and HERV families are still active in retrotransposition, and they are responsible for generating HS-MEs and MEs that are polymorphic among humans by the presence or absence of the insertions.<a reveal-id="dsy022-B8" data-open="dsy022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy022-B31" data-open="dsy022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsy022-B32" data-open="dsy022-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> So far, a few studies have also examined HS-MEs as being present in the human genome, but not in the orthologous regions of any other primate genomes.<a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsy022-B33" data-open="dsy022-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsy022-B34" data-open="dsy022-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Among these, the study by Mills et al., representing the most comprehensive analysis of species-specific at the genome-scale, identified a total of 7,786 and 2,933 MEs that are specific to human and chimpanzee, respectively.<a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> This study was done with earlier versions of the human and chimpanzee genome sequences (GRHc35/hg17 and CGSC1.1/panTrol1.1), which contained more unsequenced regions and assembly errors, in particular for the chimpanzee genome, and with no other primate genome sequences available. Since then, the genome sequences of human and chimpanzee both have been subjected to several major improvements, and the genome sequences of many additional closely related primates have also become available.<a reveal-id="dsy022-B35 dsy022-B36 dsy022-B37 dsy022-B38 dsy022-B39 dsy022-B40" data-open="dsy022-B35 dsy022-B36 dsy022-B37 dsy022-B38 dsy022-B39 dsy022-B40" class="link link-ref link-reveal xref-bibr"><sup>35–40</sup></a> These additional primate genomes can be useful in providing complementary information to chimpanzee genome sequences for the analysis of HS-MEs. Motivated by these factors, we developed a more robust multi-way comparative genomic approach to compare the human genome with that of nine non-human primate genomes. We identified a total of ∼15,000 HS-MEs, among which more than 8,000 were reported as HS-MEs for the first time. This dataset represents a major improvement over prior studies and permitted us to provide a more complete and accurate picture about the recent DNA transposition in the human genome with several novel observations.</p>                            <h2 scrollto-destination=122670821 id="122670821" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=122670822 id="122670822" class="section-title" >2.1. Sources of primate genomic sequences</h3>
<p>The human genome sequences used in this study was the latest version released in December 2013 (GRCh38/UCSC hg38). The most recent versions of genome sequences for nine other primate species were also included. These include chimpanzee (May 2016, CSAC Pan_tro 3.0/panTro5), gorilla (December 2014, NCBI project 31265/gorGor4.1), orangutan (July 2007, WUSTL version Pongo_albelii-2.0.2/ponAbe2), gibbon (October 2012, GGSC Nleu3.0/nomLeu3), green monkey (March 2014, Chlorocebus_sabeus 1.1/chlSab2), crab-eating macaque (June 2013, WashU Macaca_fascicularis_5.0/macFas5), rhesus monkey (November 2015, BCM Mmul_8.0.1/rheMac8), baboon (anubis) (March 2012, Baylor Panu_2.0/papAnu2), and marmoset (March 2009, WUGSC 3.2/calJac3). All genome sequences in fasta format and the corresponding RepeatMasker annotation files were downloaded from the UCSC genomic website (<a class="link link-uri" href="http://genome.ucsc.edu" target="_blank">http://genome.ucsc.edu</a>) onto our local servers for in-house analyses. </p>                            <h3 scrollto-destination=122670824 id="122670824" class="section-title" >2.2. Identification of HS-MEs</h3>
                            <h4 scrollto-destination=122670825 id="122670825" class="section-title" >2.2.1. Pre-processing of human MEs</h4>
<p>Our starting list of MEs is those annotated in the human genome using RepeatMasker (<a class="link link-uri" href="http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/rmsk.txt.gz" target="_blank">http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/rmsk.txt.gz</a><span class="underline">)</span>. Since RepeatMasker reports fragments of MEs interrupted by other sequences and internal inversions/deletions as individual ME entries, we performed a pre-process to integrate these fragments back to ME sequences representing the original retrotransposition events. Briefly, we examined partial MEs that are next to each other with a distance up to 50 kb, and checked their mapping positions in the repeat consensus sequence and orientation. If two neighbouring partial MEs map to the same repeat consensus sequence in the neighbouring non-overlapping regions and in the same orientation and show the presence of target-site duplications (TSDs) at the revised ME ends, we then treat these ME segments as one ME entry with the start and end positions adjusted accordingly. For LTR retrotransposons, RepeatMasker reports the two LTRs and the internal viral sequences each as a separate entry, and we grouped all components of a full-length LTR as one entry.</p>                            <h4 scrollto-destination=122670827 id="122670827" class="section-title" >2.2.2. Identification of HS-MEs</h4>
<p>As shown in <a reveal-id="dsy022-F1" data-open="dsy022-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, our strategy for identifying HS-MEs is to examine the sequences at the insertion site and its two flanking regions for each of the MEs (after integration) annotated in the human reference genome and compare with the sequences of the orthologous regions in each of the nine non-human primate genomes. If an ME is determined with confidence to be absent from the orthologous regions of all other primate genomes, then it is considered human-specific. Briefly, we used two tools, BLAT<a reveal-id="dsy022-B41" data-open="dsy022-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and liftOver (<a class="link link-uri" href="http://genomes.ucsc.edu" target="_blank">http://genomes.ucsc.edu</a>), for determining the orthologous sequences and the human-specific status of MEs using the aforementioned integrated RepeatMasker ME list as input. Only MEs supported to be unique to human by both tools were included in the final list of HS-MEs. Based on whether the MEs’ flanking sequences were detected in one or more of the out-group genomes, we divided our HS-MEs into two categories. Those with one or both of the flanking sequences detected in the orthologous region of the outgroup genomes are placed in Category I, representing HS-MEs with high confidence. Those with neither of the flanking sequences detected in the orthologous region of the out-group genomes are placed in Category II, likely representing HS-MEs that are located inside another human-specific sequences. A flow chart of the method is shown in <a reveal-id="dsy022-F1" data-open="dsy022-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, and further detailed description is provided in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary materials</a></span> and methods. </p>                        <a id="122670829" scrollto-destination="122670829"></a>
<div id="dsy022-F1" data-id="dsy022-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/m_dsy022f1.png?Expires=2147483647&Signature=qVfufFsA7-UupExMSb3WghfkV7xcQxGUyeqYFNzz2qff-Y4jgw-2N8Q7asySp5iVaBBxjgL2x6R5Y61X78e~cVo57vRVGvvkxcFZLnEAPVqBJOiazHWN3VJJws-qXifKL1R~6yzyya9RxKDS5Wagqvh9u8WiGByNq7NJUxbNw2Wg9L1EEfOEj0Hwy2Nu92qSru0XK5CXrNdN9l7Ve0pXCi8AhH~5Bie6WH9YnVUZoRTgXuqGZNn6oZpIpQgFcEutboazS89e8SoOjpaTDg1fBX~WDlgLDcRi1NyXrtr0GxO3p7TUHbi~M~5x6OE2b-jzPgbmMLhFFK8tHWsfamK8ow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of identifying HS-MEs." path-from-xml="dsy022f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670829/dsy022f1.tif" path-from-xml="dsy022f1.tif" target="_blank">View large</a><a section="122670829" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022f1.png?Expires=2147483647&Signature=S13-Vcj~SgWlPCyR2rpe94uJo8Wo5pPoz4vqa4VmGaD2PD0bWaI49YwX~JDw2pkd09Y0dOiQ-AuEo9eTmcuypuzLvIFq-jHW007Y9taLyejfnduYsjTF6itgmnbflR~zH~pDKGZzsZLpSloar6aI-bfr-XoYGWQO05mKYxs8gQsZ3gfgWOqsAcCgJrfjDOwrVl54cZV59Sx0YeAeUjgIzB35ubvdJo5H2412PIsg9QbNemuTYoaG0BXctQ3VQJ~dyk05s3vsfvQtuzkAMzPTafCy57yCLxtw8EI-ckNELy4F3Oq4gI-Rq0LkPCH2vbXW4ta8wkrnOc7IZuJ~91LayQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670829&ar=5056130&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy022f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of identifying HS-MEs.</p></div></div><div content-id="dsy022-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/m_dsy022f1.png?Expires=2147483647&Signature=qVfufFsA7-UupExMSb3WghfkV7xcQxGUyeqYFNzz2qff-Y4jgw-2N8Q7asySp5iVaBBxjgL2x6R5Y61X78e~cVo57vRVGvvkxcFZLnEAPVqBJOiazHWN3VJJws-qXifKL1R~6yzyya9RxKDS5Wagqvh9u8WiGByNq7NJUxbNw2Wg9L1EEfOEj0Hwy2Nu92qSru0XK5CXrNdN9l7Ve0pXCi8AhH~5Bie6WH9YnVUZoRTgXuqGZNn6oZpIpQgFcEutboazS89e8SoOjpaTDg1fBX~WDlgLDcRi1NyXrtr0GxO3p7TUHbi~M~5x6OE2b-jzPgbmMLhFFK8tHWsfamK8ow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of identifying HS-MEs." path-from-xml="dsy022f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670829/dsy022f1.tif" path-from-xml="dsy022f1.tif" target="_blank">View large</a><a section="122670829" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022f1.png?Expires=2147483647&Signature=S13-Vcj~SgWlPCyR2rpe94uJo8Wo5pPoz4vqa4VmGaD2PD0bWaI49YwX~JDw2pkd09Y0dOiQ-AuEo9eTmcuypuzLvIFq-jHW007Y9taLyejfnduYsjTF6itgmnbflR~zH~pDKGZzsZLpSloar6aI-bfr-XoYGWQO05mKYxs8gQsZ3gfgWOqsAcCgJrfjDOwrVl54cZV59Sx0YeAeUjgIzB35ubvdJo5H2412PIsg9QbNemuTYoaG0BXctQ3VQJ~dyk05s3vsfvQtuzkAMzPTafCy57yCLxtw8EI-ckNELy4F3Oq4gI-Rq0LkPCH2vbXW4ta8wkrnOc7IZuJ~91LayQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670829&ar=5056130&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy022f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of identifying HS-MEs.</p></div></div>                            <h3 scrollto-destination=122670830 id="122670830" class="section-title" >2.3. Validation of HS-MEs</h3>
<p>To assess the accuracy of our HS-MEs both in terms of false-positive and false-negative errors, we performed manual inspection using the UCSC genome browser with a set of 100 randomly sampled HS-ME candidates, and performed further validations using the following 3 methods. </p>                            <h4 scrollto-destination=122670832 id="122670832" class="section-title" >2.3.1. Method 1</h4>
<p>We compared our list against the previous HS-MEs list reported by Mills et al<em>.</em><a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and the human-specific HERV-Ks reported by Shin et al.<a reveal-id="dsy022-B34" data-open="dsy022-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p>                            <h4 scrollto-destination=122670834 id="122670834" class="section-title" >2.3.2. Method 2</h4>
<p>A total of 3,110 polymorphic ME entries present in the reference genome, representing 2,221 or 50% of entries documented in dbRIP,<a reveal-id="dsy022-B42" data-open="dsy022-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> plus additional 779 entries identified by the 1000 Genome Project team<a reveal-id="dsy022-B43" data-open="dsy022-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsy022-B44" data-open="dsy022-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> were collected and cross-matched with the list of HS-ME candidates.</p>                            <h4 scrollto-destination=122670836 id="122670836" class="section-title" >2.3.3. Method 3</h4>
<p>A total of 15 randomly selected HS-ME entries were subjected to PCR validation using primate DNA samples (Details are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary materials</a></span> and methods).</p>                            <h3 scrollto-destination=122670838 id="122670838" class="section-title" >2.4. Identification of TSDs and analysis of sequence motifs at the ME integration sites</h3>
<p>The TSDs as well as transduction and retrotransposon insertion mediated-deletions (RIMDs) for all HS-MEs were identified using in-house Perl scripts incorporating the utility of the NCBI bl2seq and UCSC liftOver tools. The method takes the human genomic sequence covering an HS-ME, plus the flanking sequence on each side and aligns them to the corresponding orthologous sequence from the chimpanzee genome (or the next closest genome with available orthologous sequences), which represents the pre-integration allele. In order to retrieve the orthologous pre-integration allele sequences, the orthologous positions of both flanking sequences of an HS-ME in the out-group genomes were identified using liftOver. For a typical Category I HS-ME, liftOver can find the orthologous region of the immediate flanking regions. However, the immediate flanking sequences of an HS-ME in human genome may not represent the pre-integration sequence due to transduction or RIMD events. Therefore, our scripts liftOver multiple subsequent blocks of flanking sequences retrieved in human genome onto the out-group genomes (100 bp per block for up to 5 kb). The liftOver results were then grouped together to identify the shortest orthologous region containing the pre-integration site, which was used to align against the two human flanking sequences using the NCBI bl2seq tool. The overlapped region in the pre-integration allele between the two aligned regions with the flanking sequences of the ME represents the TSD. For those with TSDs, a 30-bp sequence centred at each insertion site at the pre-integration alleles was extracted after removing the ME sequence and one copy of the TSDs from the ME alleles. The sequence motif for the regions covering 15 bp before and after the insertion sites was obtained using the weblogo tool (<a class="link link-uri" href="http://weblogo.berkeley.edu/logo.cgi" target="_blank">http://weblogo.berkeley.edu/logo.cgi</a>).</p><p>Entries with identified TSDs and extra sequences between the ME and either copy of the TSDs were considered candidates for ME insertion-mediated transductions and were subject to further validation (see details in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary material</a></span>). For entries without TSDs, if there are extra sequences at the pre-integration site in the out-group genomes, they were considered to be candidates for RIMD in the human genome, which were subject to further validation as described in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary materials</a></span> and methods.</p>                            <h3 scrollto-destination=122670841 id="122670841" class="section-title" >2.5. Analysis of HS-MEs in Y chromosome</h3>
<p>The analysis of HS-MEs for Y chromosome required some special considerations for two reasons. First, authentic Y chromosome sequences are currently available only for human, chimpanzee, green monkey, rhesus and marmoset, and they are not as complete as for autosomal chromosomes.<a reveal-id="dsy022-B45" data-open="dsy022-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Also, since the sequences for the pseudoautosomal regions were basically copied from X chromosome of the same genome, we excluded these regions from the analysis for Y chromosome.</p>                            <h3 scrollto-destination=122670843 id="122670843" class="section-title" >2.6. Analysis of HS-MEs’ association with genes and regulatory elements</h3>
<p>The genomic coordinates of genes down to individual exons and coding regions were based on GENCODE<a reveal-id="dsy022-B46" data-open="dsy022-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> and NCBI RefSeq annotation<a reveal-id="dsy022-B47" data-open="dsy022-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> provided in the UCSC Genome Browser. The entire genome was divided into a non-redundant list of categorized regions in gene context as coding sequence (CDS), non-coding RNA, 5′-UTR, 3′-UTR, promoter (1 kb), intron, and intergenic regions using an in-house Perl script. This order of genic region categories as listed above was used to set the priority from high to low in handling overlaps between splice forms of the same gene or different genes. For example, if a region is a CDS for one transcript/gene and is a UTR or intron for another, then this region would be categorized as CDS.</p><p>For identifying HS-MEs overlapping with known transcriptional factor binding sites (TFBS), we used the ENCODE<a reveal-id="dsy022-B46" data-open="dsy022-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Transcription Factor ChIP-seq contained in the wgEncodeRegTfbsClusteredV3.bed.gz available from the UCSC Genome Browser site.</p>                            <h2 scrollto-destination=122670846 id="122670846" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=122670847 id="122670847" class="section-title" >3.1. Revised ME composition in the human genome and a summary of HS-MEs</h3>
<p>To improve the accuracy in identifying HS-MEs and their TSDs and calculating the rate of retrotransposition, we pre-processed MEs annotated by RepeatMasker to integrate fragmented MEs to represent original ME events. This integration process led to a reduction of almost one million (1,180,428) in ME counts from the 5,520,017 RepeatMasker ME entries in NCBI38/hg38 to 4,339,589 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). In the meantime, it resulted an increase of full-length MEs by 21% (data not shown). As seen in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, the number of annotated MEs showed a consistent increase in newer versions of the human reference genomes, especially between earlier updates, mainly due to improved coverage of sequenced regions. Notably, the proportion of MEs in the human reference genome increased from 48.8% for the earlier version<a reveal-id="dsy022-B3" data-open="dsy022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> to 52.1% in the latest version (GRCh38, December 2013) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Based on GRCh38, the copy numbers (after integration) and percentages of the genome for DNA transposons, L1s, <em>Alus</em>, LTRs, SVAs, and the ‘Others’ (all remaining MEs consisting of mostly non-L1 LINEs) are 295,956 (3.5%), 564,195 (17.8%), 1,132,541 (10.5%), 488,208 (9.1%), 4,933 (0.1%), and 1,733,490 (11.2%), respectively. To our knowledge, these data represent the most updated ME composition in the human reference genome. <div content-id="dsy022-T1" class="table-modal"><div class="table-wrap"><div id="dsy022-T1" data-id="dsy022-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of human-specific mobile elements (HS-MEs)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">RM count </th><th rowspan="2">Integrated count </th><th colspan="4">HS-MEs<hr /> </th></tr><tr><th>Category I </th><th>Category II </th><th>Total </th><th>HS% </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>962,085 </td><td>563,594 </td><td>3,654 </td><td>258 </td><td>3,912 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>1,181,072 </td><td>1,129,987 </td><td>8,626 </td><td>191 </td><td>8,817 </td><td>0.8 </td></tr><tr><td><strong>SVA</strong> </td><td>5,397 </td><td>4,928 </td><td>1,512 </td><td>59 </td><td>1,571 </td><td>31.9 </td></tr><tr><td><strong>LTR</strong> </td><td>720,177 </td><td>496,306 </td><td>455 </td><td>75 </td><td>530 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td>2,868,731 </td><td>2,194,815 </td><td>14,247 </td><td>583 </td><td>14,830 </td><td>0.7 </td></tr></tbody></table></div><div content-id="dsy022-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">RM count </th><th rowspan="2">Integrated count </th><th colspan="4">HS-MEs<hr /> </th></tr><tr><th>Category I </th><th>Category II </th><th>Total </th><th>HS% </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>962,085 </td><td>563,594 </td><td>3,654 </td><td>258 </td><td>3,912 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>1,181,072 </td><td>1,129,987 </td><td>8,626 </td><td>191 </td><td>8,817 </td><td>0.8 </td></tr><tr><td><strong>SVA</strong> </td><td>5,397 </td><td>4,928 </td><td>1,512 </td><td>59 </td><td>1,571 </td><td>31.9 </td></tr><tr><td><strong>LTR</strong> </td><td>720,177 </td><td>496,306 </td><td>455 </td><td>75 </td><td>530 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td>2,868,731 </td><td>2,194,815 </td><td>14,247 </td><td>583 </td><td>14,830 </td><td>0.7 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy022-T1" data-id="dsy022-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of human-specific mobile elements (HS-MEs)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">RM count </th><th rowspan="2">Integrated count </th><th colspan="4">HS-MEs<hr /> </th></tr><tr><th>Category I </th><th>Category II </th><th>Total </th><th>HS% </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>962,085 </td><td>563,594 </td><td>3,654 </td><td>258 </td><td>3,912 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>1,181,072 </td><td>1,129,987 </td><td>8,626 </td><td>191 </td><td>8,817 </td><td>0.8 </td></tr><tr><td><strong>SVA</strong> </td><td>5,397 </td><td>4,928 </td><td>1,512 </td><td>59 </td><td>1,571 </td><td>31.9 </td></tr><tr><td><strong>LTR</strong> </td><td>720,177 </td><td>496,306 </td><td>455 </td><td>75 </td><td>530 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td>2,868,731 </td><td>2,194,815 </td><td>14,247 </td><td>583 </td><td>14,830 </td><td>0.7 </td></tr></tbody></table></div><div content-id="dsy022-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">RM count </th><th rowspan="2">Integrated count </th><th colspan="4">HS-MEs<hr /> </th></tr><tr><th>Category I </th><th>Category II </th><th>Total </th><th>HS% </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>962,085 </td><td>563,594 </td><td>3,654 </td><td>258 </td><td>3,912 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>1,181,072 </td><td>1,129,987 </td><td>8,626 </td><td>191 </td><td>8,817 </td><td>0.8 </td></tr><tr><td><strong>SVA</strong> </td><td>5,397 </td><td>4,928 </td><td>1,512 </td><td>59 </td><td>1,571 </td><td>31.9 </td></tr><tr><td><strong>LTR</strong> </td><td>720,177 </td><td>496,306 </td><td>455 </td><td>75 </td><td>530 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td>2,868,731 </td><td>2,194,815 </td><td>14,247 </td><td>583 </td><td>14,830 </td><td>0.7 </td></tr></tbody></table></div></div></p><p>Using a multi-way comparative genomic approach involving comparison of human genome sequences to that of nine other closely related primates as outlined in <a reveal-id="dsy022-F1" data-open="dsy022-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, we identified a total of 14,870 HS-MEs, consisting of 8,817 <em>Alus</em>, 3,912 L1s, 1,571 SVAs, and 530 HERVs. By the presence/absence of flanking sequence in the non-human primate genomes, there are 14,247 in category I and 583 category II (<a reveal-id="dsy022-T1" data-open="dsy022-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Among these HS-MEs, a total of 8,049 MEs were reported as HS-MEs for the first time, while 6,738 entries were shared with the previously reported HS-MEs<a reveal-id="dsy022-B48" data-open="dsy022-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The complete list of HS-MEs is provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary file S1</a></span> and in the dbRIP database (<a class="link link-uri" href="http://dbrip.org" target="_blank">http://dbrip.org</a>).<a reveal-id="dsy022-B42" data-open="dsy022-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                            <h3 scrollto-destination=122670850 id="122670850" class="section-title" >3.2. Validation of HS-MEs</h3>
<p>Due to the complexity of the task in identifying species-specific MEs as further discussed later, data generated by computational methods are very likely to have both false positives and false negatives. While it is cost-prohibitive to experimentally validate all 14,830 HS-MEs due to the extremely large number, we managed to validate the accuracy of our HS-ME list in three ways in addition to manual checking of 100 randomly selected entries on the UCSC Genome Browser, which showed an accuracy of 98% (data not shown). First, we used polymorphic MEs as a test dataset to assess the sensitivity of method [true positive/(true positive +false negative)]. Here, we reason that polymorphic MEs represent most recent insertions into the human genomes, and thus they should be mostly, if not all, HS-MEs. Thus, any polymorphic MEs failed to be identified as human specific ME are considered as false-negatives. By cross-checking the HS-ME list with the 3,110 polymorphic MEs present in the reference genome (2,331 from dbRIP not used in training set and 779 from the 1000 genome project), we obtained a sensitivity of 95.5% (2,972/3,110) (data not shown).</p><p>Second, we cross-checked the HS-MEs with the 7,786 previously reported HS-MEs by Mills et al.,<a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and we obtained 6,738 entries as shared (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Among the 1,048 entries not on our list, we found that 904 actually represented false positives from the previous study and 26 were absent in the new version of the genome sequences, 47 entries for ME types not included in our input list (mainly due to the differences between different versions of Repeatmasker), and 71 entries represent false negatives in our list (these were added to the final list of HS-MEs; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). This converts to a sensitivity of 99.0% (6,738/6,738 + 71)) for our method. Since both the polymorphic MEs and those reported by Mills et al.<a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> are mostly from non-repetitive regions, which are less susceptible to errors in sequencing and computational analysis, the high ratio of overlapping is expected, and these were mostly identified as category I HS-MEs in our analysis. We further compared our list with the human-specific HERV-K list published by Shin et al.<a reveal-id="dsy022-B34" data-open="dsy022-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Among the 28 ME_IN entries, 26 of them overlap with our list while the other entries were not included in the final list due to low confidence.</p><p>Third, we performed validation using PCR, which is the gold standard for ascertaining MEs. We were able to validate 12 of the 13 randomly selected HS-ME loci, for which PCR was successful. This converts to a true positive rate (precision) of 92.3% or a 7% of false positive rate. Representative PCR results can be seen in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and full details of all examined loci are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p><p>Overall, combining the results from the above three methods of validation, we showed that our method has a minimal sensitivity of 95.5% and a precision rate of 92.3%.</p>                            <h3 scrollto-destination=122670855 id="122670855" class="section-title" >3.3. Main sources for newly identified HS-MEs</h3>
<p>We examined the contributions of different factors to the 8,049 novel HS-MEs, among ‘HS-MEs in MEs’, ME integration, non-canonical MEs (transduction, RMID, no-TSD), extra MEs in hg38 (over hg17) and use of multiple genomes, with each of these factors considered independently (redundancy exist among categories, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4a</a></span>) and combined in a step-wise order (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4b</a></span>). When each source considered independently, the top contributors of novel HS-MEs include ‘MEs in MEs’ (3,744) and non-canonical cases (3,256), followed by ME integration (972), use of multiple genomes (822), and new human genome (161) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4a</a></span>). ME integration is the top contributor for novel HS-<em>Alu</em>s, while the non-canonical MEs is the top contributor for L1, SVA, and LTR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4a</a></span>). These sources collectively contributed to more than 6,000 non-redundant entries or 75% of the novel HS-MEs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4b</a></span>). The remaining 25% of the novel HS-MEs are likely due to other factors such as improved version of chimpanzee genome. By percentage of novel HS-MEs among all HS-MEs across four ME types, LTR and L1 showed much higher ratios (78 and 73%, respectively) than <em>Alu</em> and SVA (46 and 44%, respectively) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4b</a></span>).</p>                            <h3 scrollto-destination=122670857 id="122670857" class="section-title" >3.4. HS-MEs contributed 14 Mbp net size increase to the human genome</h3>
<p>ME insertions can lead to genome size increase via insertion of MEs, generation of TSDs, and transductions, and they can also reduce genome size via RIMD. As shown in <a reveal-id="dsy022-T2" data-open="dsy022-T2" class="link link-reveal link-table xref-fig">Table 2</a>, the occurrences of all four mechanisms and a net genome sequence increase were observed for each ME type. Collectively, all HS-MEs contributed to a total of 14.2 Mbp net genome size increase since the last common ancestor (LCA) with chimpanzee. Among ME types, L1s made the largest net increase (∼8.3 Mbp), followed by <em>Alu</em>s (∼2.6 Mbp), SVAs (∼2.4 Mbp), and HERVs (0.8 Mbp). As expected, HS-L1s contributed to the largest size changes from insertion, IMD, and transductions due to their large insert size and high copy number, while HS-<em>Alu</em>s contributed to more size increase via TSDs than any other ME types due to the largest insertions in number of HS-<em>Alu</em>s. Despite having a low retrotransposition activity, LTRs did contribute ∼750 kb size increase to the genome with 530 human-specific copies. It is also interesting to note that by ratio of size increase in relation to all MEs in a type, SVA showed the highest ratio among all (<a reveal-id="dsy022-T2" data-open="dsy022-T2" class="link link-reveal link-table xref-fig">Table 2</a>), seemingly attributed by the combination of a large average size of insertions, the highest ratio of transduction, and the highest ratio of HS-MEs. <div content-id="dsy022-T2" class="table-modal"><div class="table-wrap"><div id="dsy022-T2" data-id="dsy022-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Impact of human-specific mobile elements (HS-MEs) on genome size (kb)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">HS-MEs </th><th rowspan="2">TSDs </th><th rowspan="2">Trans# </th><th rowspan="2">RIMD* </th><th rowspan="2">Net size </th><th colspan="2">Increase ratio<hr /> </th><th>Transd# </th></tr><tr><th>/1K ME </th><th>/Mbp ME </th><th>/1K ME </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>8,422.3 </td><td>38.5 </td><td>380.7 </td><td>−521.5 </td><td>8,320.0 </td><td>14.7 </td><td>15.9 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>2,567.5 </td><td>111.4 </td><td>187.9 </td><td>−265.7 </td><td>2,601.0 </td><td>2.3 </td><td>8.4 </td><td>0.2 </td></tr><tr><td><strong>SVA</strong> </td><td>2,417.9 </td><td>20.2 </td><td>194.5 </td><td>−75.0 </td><td>2,557.5 </td><td>518.5 </td><td>604.8 </td><td>39.4 </td></tr><tr><td><strong>LTR</strong> </td><td>834.3 </td><td>2.0 </td><td>33.4 </td><td>−117.5 </td><td>752.2 </td><td>1.5 </td><td>2.8 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td><strong>14,242.0</strong> </td><td><strong>172.0</strong> </td><td><strong>796.5</strong> </td><td>−<strong>979.7</strong> </td><td><strong>14,230.8</strong> </td><td>6.5 </td><td>12.9 </td><td>0.4 </td></tr></tbody></table></div><div content-id="dsy022-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">HS-MEs </th><th rowspan="2">TSDs </th><th rowspan="2">Trans# </th><th rowspan="2">RIMD* </th><th rowspan="2">Net size </th><th colspan="2">Increase ratio<hr /> </th><th>Transd# </th></tr><tr><th>/1K ME </th><th>/Mbp ME </th><th>/1K ME </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>8,422.3 </td><td>38.5 </td><td>380.7 </td><td>−521.5 </td><td>8,320.0 </td><td>14.7 </td><td>15.9 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>2,567.5 </td><td>111.4 </td><td>187.9 </td><td>−265.7 </td><td>2,601.0 </td><td>2.3 </td><td>8.4 </td><td>0.2 </td></tr><tr><td><strong>SVA</strong> </td><td>2,417.9 </td><td>20.2 </td><td>194.5 </td><td>−75.0 </td><td>2,557.5 </td><td>518.5 </td><td>604.8 </td><td>39.4 </td></tr><tr><td><strong>LTR</strong> </td><td>834.3 </td><td>2.0 </td><td>33.4 </td><td>−117.5 </td><td>752.2 </td><td>1.5 </td><td>2.8 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td><strong>14,242.0</strong> </td><td><strong>172.0</strong> </td><td><strong>796.5</strong> </td><td>−<strong>979.7</strong> </td><td><strong>14,230.8</strong> </td><td>6.5 </td><td>12.9 </td><td>0.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Transd#, transduction; RIMD*, retrotransposition insertion-mediated deletion.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy022-T2" data-id="dsy022-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Impact of human-specific mobile elements (HS-MEs) on genome size (kb)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">HS-MEs </th><th rowspan="2">TSDs </th><th rowspan="2">Trans# </th><th rowspan="2">RIMD* </th><th rowspan="2">Net size </th><th colspan="2">Increase ratio<hr /> </th><th>Transd# </th></tr><tr><th>/1K ME </th><th>/Mbp ME </th><th>/1K ME </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>8,422.3 </td><td>38.5 </td><td>380.7 </td><td>−521.5 </td><td>8,320.0 </td><td>14.7 </td><td>15.9 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>2,567.5 </td><td>111.4 </td><td>187.9 </td><td>−265.7 </td><td>2,601.0 </td><td>2.3 </td><td>8.4 </td><td>0.2 </td></tr><tr><td><strong>SVA</strong> </td><td>2,417.9 </td><td>20.2 </td><td>194.5 </td><td>−75.0 </td><td>2,557.5 </td><td>518.5 </td><td>604.8 </td><td>39.4 </td></tr><tr><td><strong>LTR</strong> </td><td>834.3 </td><td>2.0 </td><td>33.4 </td><td>−117.5 </td><td>752.2 </td><td>1.5 </td><td>2.8 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td><strong>14,242.0</strong> </td><td><strong>172.0</strong> </td><td><strong>796.5</strong> </td><td>−<strong>979.7</strong> </td><td><strong>14,230.8</strong> </td><td>6.5 </td><td>12.9 </td><td>0.4 </td></tr></tbody></table></div><div content-id="dsy022-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">ME type </th><th rowspan="2">HS-MEs </th><th rowspan="2">TSDs </th><th rowspan="2">Trans# </th><th rowspan="2">RIMD* </th><th rowspan="2">Net size </th><th colspan="2">Increase ratio<hr /> </th><th>Transd# </th></tr><tr><th>/1K ME </th><th>/Mbp ME </th><th>/1K ME </th></tr></thead><tbody><tr><td><strong>L1</strong> </td><td>8,422.3 </td><td>38.5 </td><td>380.7 </td><td>−521.5 </td><td>8,320.0 </td><td>14.7 </td><td>15.9 </td><td>0.7 </td></tr><tr><td><strong>Alu</strong> </td><td>2,567.5 </td><td>111.4 </td><td>187.9 </td><td>−265.7 </td><td>2,601.0 </td><td>2.3 </td><td>8.4 </td><td>0.2 </td></tr><tr><td><strong>SVA</strong> </td><td>2,417.9 </td><td>20.2 </td><td>194.5 </td><td>−75.0 </td><td>2,557.5 </td><td>518.5 </td><td>604.8 </td><td>39.4 </td></tr><tr><td><strong>LTR</strong> </td><td>834.3 </td><td>2.0 </td><td>33.4 </td><td>−117.5 </td><td>752.2 </td><td>1.5 </td><td>2.8 </td><td>0.1 </td></tr><tr><td><strong>Total</strong> </td><td><strong>14,242.0</strong> </td><td><strong>172.0</strong> </td><td><strong>796.5</strong> </td><td>−<strong>979.7</strong> </td><td><strong>14,230.8</strong> </td><td>6.5 </td><td>12.9 </td><td>0.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Transd#, transduction; RIMD*, retrotransposition insertion-mediated deletion.</p></div></div></div></div></p>                            <h3 scrollto-destination=122670859 id="122670859" class="section-title" >3.5. Retrotransposition activity level of MEs during early human evolution</h3>
<p>The comprehensive list of HS-MEs provided us an opportunity to re-examine more accurately the activities of MEs in the human genome, especially during the earlier part of human evolution after separation from chimpanzees, which cannot be assessed using the polymorphic MEs. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>, while many observations were similar to what we learned from prior studies based on younger polymorphic MEs.<a reveal-id="dsy022-B3" data-open="dsy022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy022-B8" data-open="dsy022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsy022-B31" data-open="dsy022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> These include that (i) <em>Alu</em>, L1, SVA, and HERVs are the only four types of MEs with retrotransposition during human evolution and (ii) <em>Alu</em>Ya5 and <em>Alu</em>Yb8/9, L1HS, SVA_E/D, and HERV_K are the most active subfamilies for <em>Alu</em>, L1, SVA, and ERV, respectively. In addition, several notable differences were also observed. First, the ratio of HS-MEs in relation to all MEs in the most active subfamilies were much higher (10 times or more) than the corresponding ratios of polymorphic MEs<a reveal-id="dsy022-B8" data-open="dsy022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy022-B31" data-open="dsy022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Second, a few active subfamilies not seen among polymorphic MEs were seen for HS-MEs, and these include <em>Alu</em>Yf, <em>Alu</em>Yk subfamilies, L1P4, L1M, SVA_A, and ERV1 subfamilies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>).</p>                            <h3 scrollto-destination=122670861 id="122670861" class="section-title" >3.6. Patterns of MEs and HS-MEs in the repetitive regions: SVAs are strongly biased for inserting into SVAs</h3>
<p>With HS-MEs in repetitive regions available for the first time, we were interested in knowing whether there are any biases among MEs both as the sources and targets of retrotransposition insertions. We first examined the percentage of MEs inserting into other MEs for all MEs and for HS-MEs separately. For all MEs as the sources of insertions in MEs, there seems to be an increasing trend from LINEs (17.8%), DNA transposons (22.2%), LTRs (28.2%), SINEs (31.2%), and SVAs (36.5%) with the ratio of SVAs being more than doubled of that for LINEs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6a</a></span>). When only HS-MEs were considered (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6b</a></span>), a similar trend (with DNA transposons absent as they were not included in this study) was seen, but the degree of difference among ME types is much smaller than for all MEs. Nevertheless, significant differences among ME types were still observed. For example, HS-<em>Alu</em>s show the highest percentage inserted into MEs (46.5%), followed by HS-SVAs (40.7%), while HS-L1s and HS-LTRs have lower rates (35.5 and 35.3%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6b</a></span>).</p><p>For each ME type as the insertion targets of HS-MEs, we calculated the frequency of HS-MEs as the number of HS-MEs per Mbp of host ME sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6c</a></span>). Among the ME types as targets, the HS-ME density follows a decreasing trend from high to low among LINEs (7.3), DNA transposons (5.0), LTRs (4.0), and SINEs (1.8), seemly correlating with their overall age in the genome from old to new. However, very interestingly, SVA seems to be the striking outlier to this trend by having the highest density of HS-MEs (13.0) despite being the youngest ME type (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6c</a></span>). HS-<em>Alu</em>s seem to be more frequently inserting into MEs than all other three ME types when all MEs are considered as targets seemly due to the highest number of HS-<em>Alu</em>s. HS-<em>Alu</em>s showed the highest ratio in LINEs, LTRs, and SINEs, but not for SVAs. Remarkably, HS-SVAs seem to show an unusually high preference for SVAs as targets at a frequency that is at least 18 times higher than the ratio of HS-SVAs into any other ME types (12.7 vs. 0.7 for LINE) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6c</a></span>). In the meantime, HS-SVAs are also the most frequent seen HS-MEs in SVAs at a frequency that is more than 60 times higher than for other MEs (12.7 vs. 0.2 for HS-<em>Alu</em>s and 0 for HS-L1s and HS-LTRs) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6c</a></span>).</p>                            <h3 scrollto-destination=122670864 id="122670864" class="section-title" >3.7. Y chromosome is a hot target for HS-LTRs</h3>
<p>To examine the distribution pattern of HS-MEs in the genome, we measured the HS-ME density (the number of HS-MEs per million base pairs of non-gaped chromosome sequences) and the ratio of HS-MEs among the same type of MEs in the chromosomes for each ME type or all types combined. As shown in <a reveal-id="dsy022-F2" data-open="dsy022-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>, with all types combined, the HS-ME density varies across chromosomes with Y chromosome showing the highest density (25.4 copies/Mbp), more than 5 times higher than the genome average (5 copies/Mbp). The pattern is different among the 4 ME types (<a reveal-id="dsy022-F2" data-open="dsy022-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Both <em>Alu</em>s and L1s showed a more or less homogenous density among autosomes. However, Y chromosome showed the highest density being four to five times higher than the genome average or more than 10 times higher than that of the chromosomes with the lowest density (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). SVAs showed relatively similar densities among autosomes, except for chromosome 19, which has a density of HS-SVA that is 2 times higher than the genome average. But for Y chromosome, opposite to what seen for HS-L1s and HS-<em>Alu</em>s, the HS-SVA density is more than two times lower than the genome average. LTRs showed the most variable distribution among chromosomes, with very low densities for most chromosomes, but with high and extremely high density in chromosome 19, and Y, respectively. The HS-LTR density in Y chromosome (12.5 copies/2 Mbp) is more than 30 times higher than the genome average (0.4 copies /2 Mbp) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S2</a></span>). Therefore, Y chromosome seems to be a hot target for <em>Alu</em>s, L1s, and LTRs with LTRs showing an extreme high preference over the rest of the genome, while it is the least preferred target for SVAs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Among other chromosomes, chromosome 19 stands out as having the highest density of MEs, genes, HS-SVAs, and HS-LTRs, while chromosome 22 seems to have the least HS-<em>Alu</em>s, and HS-L1s. Correlation analysis showed a strong positive correlation of HS-SVA density with that of genes and with that of all MEs (<em>R</em> = 0.9 in both cases), while HS-L1s showed a moderate negative correlation with gene density (<em>R</em> = −0.4) (data not shown). </p>                        <a id="122670866" scrollto-destination="122670866"></a>
<div id="dsy022-F2" data-id="dsy022-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/m_dsy022f2.png?Expires=2147483647&Signature=FfiZRQLZ7gnRj2n2IuEWrm3M40NMQbCdY9Au-yNBIC2DrnfLZ1yNPyRgVyE06sdtX07N3nkBv3roh3oCWuCp-ui664J5C~C3AXpt5V5p5~vuv2GF9-51mqSyVNZAO-HBhnCXNmwe2q9ltnVja28oVpnVZXmItDTi3-BwTHBIGGnjTzZnJFHumvunoEnSs-uiG7KTOUyqXoxBy8CaQDUHvUE7jU5vR2qzFDFZG1eZA6Xor2-gJ6dRKBDbZZYVd0JmAkFVIB7d1UISQyUQCIry-mKdUYfEEi2PImkzCtcaBhiiV7Aahzv9biFhMiWg914JeWlfvz0amtWaqjsno8xArA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome distribution of HS-MEs. Bar plots showing the density of HS-MEs for each type of MEs among 24 chromosomes and the entire genome." path-from-xml="dsy022f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670866/dsy022f2.tif" path-from-xml="dsy022f2.tif" target="_blank">View large</a><a section="122670866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022f2.png?Expires=2147483647&Signature=XDDOibYcW0DAbQqNfQh4OzmsM2GUH6d9S1s~kl-s97zXu7B6I3Csx5N1Tbmpt9IlCe3F0geDIB8fXewBYGwqXi0UGCH-rX6Uq9O5t2Lnr7VshYlDkdqT-fDOyltEAjnsrX2sU0gAFVAEsgD1xejIMZmBKNYwkXi5E-HDqZarvIowKqmcJYuZmp~SA~43cwBdf1OY2qzXIu~dHxwNan9mUpJfujPbjeTkNf27m3gdPOxSH2UdkzZhZPeblGKVJOSWSrFIW89J9D8~YJ2K~xjYj-yoWGOR1FZQa8P-Gwmkb72x-yt7VlvwgFNsVDjt7PnyHRgoQfN~8BVLjK-FiIyxnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670866&ar=5056130&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy022f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome distribution of HS-MEs. Bar plots showing the density of HS-MEs for each type of MEs among 24 chromosomes and the entire genome.</p></div></div><div content-id="dsy022-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/m_dsy022f2.png?Expires=2147483647&Signature=FfiZRQLZ7gnRj2n2IuEWrm3M40NMQbCdY9Au-yNBIC2DrnfLZ1yNPyRgVyE06sdtX07N3nkBv3roh3oCWuCp-ui664J5C~C3AXpt5V5p5~vuv2GF9-51mqSyVNZAO-HBhnCXNmwe2q9ltnVja28oVpnVZXmItDTi3-BwTHBIGGnjTzZnJFHumvunoEnSs-uiG7KTOUyqXoxBy8CaQDUHvUE7jU5vR2qzFDFZG1eZA6Xor2-gJ6dRKBDbZZYVd0JmAkFVIB7d1UISQyUQCIry-mKdUYfEEi2PImkzCtcaBhiiV7Aahzv9biFhMiWg914JeWlfvz0amtWaqjsno8xArA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome distribution of HS-MEs. Bar plots showing the density of HS-MEs for each type of MEs among 24 chromosomes and the entire genome." path-from-xml="dsy022f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670866/dsy022f2.tif" path-from-xml="dsy022f2.tif" target="_blank">View large</a><a section="122670866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022f2.png?Expires=2147483647&Signature=XDDOibYcW0DAbQqNfQh4OzmsM2GUH6d9S1s~kl-s97zXu7B6I3Csx5N1Tbmpt9IlCe3F0geDIB8fXewBYGwqXi0UGCH-rX6Uq9O5t2Lnr7VshYlDkdqT-fDOyltEAjnsrX2sU0gAFVAEsgD1xejIMZmBKNYwkXi5E-HDqZarvIowKqmcJYuZmp~SA~43cwBdf1OY2qzXIu~dHxwNan9mUpJfujPbjeTkNf27m3gdPOxSH2UdkzZhZPeblGKVJOSWSrFIW89J9D8~YJ2K~xjYj-yoWGOR1FZQa8P-Gwmkb72x-yt7VlvwgFNsVDjt7PnyHRgoQfN~8BVLjK-FiIyxnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670866&ar=5056130&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy022f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome distribution of HS-MEs. Bar plots showing the density of HS-MEs for each type of MEs among 24 chromosomes and the entire genome.</p></div></div>                            <h3 scrollto-destination=122670867 id="122670867" class="section-title" >3.8. The pattern of TSD length and integration site sequence motifs for HS-MEs</h3>
<p>We also surveyed TSD length and insertion site sequence motifs for HS-MEs, and compared among the four ME types. As in <a reveal-id="dsy022-F3" data-open="dsy022-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, <em>Alu</em>s, L1s, and SVAs showed an identical core sequence motif of ‘TT/AAAA’, confirming that all non-LTR retrotransposition use the same TPRT mechanisms.<a reveal-id="dsy022-B9" data-open="dsy022-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy022-B42" data-open="dsy022-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsy022-B49" data-open="dsy022-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsy022-B50" data-open="dsy022-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> LTRs showed basically no recognizable motif signal, an observation not reported before despite reported having certain site preferences.<a reveal-id="dsy022-B51" data-open="dsy022-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> For the pattern of TSD length, as shown in <a reveal-id="dsy022-F3" data-open="dsy022-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, all three non-LTR ME types showed a more or less similar distribution pattern with the TSD length peaking at the 15 bp. However, minor differences in the detailed pattern of the length distribution are also noticeable. For example, SVAs had a narrower peak, while <em>Alu</em>s showed a flatter peak covering 14–16 bp. Interestingly, L1s showed a secondary peak at 8 bp in addition to the main peak at 15 bp, while LTRs also showed a minor peak at 14 bp in addition to the dominant peak at 6 bp. These data clearly indicate the differences of the retrotransposition mechanisms used by LTR and non-LTR insertions, as well as some minor differences among different types of non-LTR insertions. </p>                        <a id="122670869" scrollto-destination="122670869"></a>
<div id="dsy022-F3" data-id="dsy022-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/m_dsy022f3.png?Expires=2147483647&Signature=lJ94FbWNwsmq~yUFYvZRV~4B2auGWOYiXGj2LyvuqdkrSUUbU02tWtu3dI7DolgiOwMFqogiwAtE0W1HOzgvmpHqL3ABHLnKhmxVKnV2ULKazHMW7TEAUkgvPt1AxklN0A4Up-ATqCeFicBB5~7uW1bULm~BpuOxwtX-YANcOa-rPX25MVF4jhHLZH1rJW0hqYCoJbtspzByfXV~LDSx674eJL7oZ4UVi8hIg3ZGpuutyGbYC7qDsnd11lsTanxtFm103lrunB5b2gp-kOri-FaC9fdy5vawhltZqj4eGn5Arkr22Aqo4j49PPhXNEiA61dN-fr1ONNMImIpV9bHvw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics of pre-integration sites and TSD lengths for HS-MEs. (A) Sequence logos for each HS-ME type at the integration sites. (B) Line plots showing the frequencies of TSDs at each length for each type of HS-MEs. " path-from-xml="dsy022f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670869/dsy022f3.tif" path-from-xml="dsy022f3.tif" target="_blank">View large</a><a section="122670869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022f3.png?Expires=2147483647&Signature=nmlWu17u4uzAbqqNma1uIPFMasOTduN-FaPlqqIaejPWVK8o2gGDr3sAQZa20HWXJcaehnz4Qd5zFxg8-m8mwavOXCLWcYWnvYrh4e7T-QtAm1ynjoxEihP9ClfuIjJt1a-~6pr3k0t1p18njQw0GVs0erwXmOIDfCvE~XXLDBS98J~PslutGEf6Bl4w32tX1ECFftIFZjqnhySZsdeDm-mArd5u5lWYCwaBENY64rSw5cX8HfCpWsMwjh2LnQC5vYU6nJkFN1tEP5vgowI8CAuu6r-l2BItiSvpu3yrotPMhEETk3Ew1XT-pbG-ZKiIEcojMQUSGSH0jp1tXISv-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670869&ar=5056130&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy022f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristics of pre-integration sites and TSD lengths for HS-MEs. (A) Sequence logos for each HS-ME type at the integration sites. (B) Line plots showing the frequencies of TSDs at each length for each type of HS-MEs. </p></div></div><div content-id="dsy022-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/m_dsy022f3.png?Expires=2147483647&Signature=lJ94FbWNwsmq~yUFYvZRV~4B2auGWOYiXGj2LyvuqdkrSUUbU02tWtu3dI7DolgiOwMFqogiwAtE0W1HOzgvmpHqL3ABHLnKhmxVKnV2ULKazHMW7TEAUkgvPt1AxklN0A4Up-ATqCeFicBB5~7uW1bULm~BpuOxwtX-YANcOa-rPX25MVF4jhHLZH1rJW0hqYCoJbtspzByfXV~LDSx674eJL7oZ4UVi8hIg3ZGpuutyGbYC7qDsnd11lsTanxtFm103lrunB5b2gp-kOri-FaC9fdy5vawhltZqj4eGn5Arkr22Aqo4j49PPhXNEiA61dN-fr1ONNMImIpV9bHvw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics of pre-integration sites and TSD lengths for HS-MEs. (A) Sequence logos for each HS-ME type at the integration sites. (B) Line plots showing the frequencies of TSDs at each length for each type of HS-MEs. " path-from-xml="dsy022f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670869/dsy022f3.tif" path-from-xml="dsy022f3.tif" target="_blank">View large</a><a section="122670869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022f3.png?Expires=2147483647&Signature=nmlWu17u4uzAbqqNma1uIPFMasOTduN-FaPlqqIaejPWVK8o2gGDr3sAQZa20HWXJcaehnz4Qd5zFxg8-m8mwavOXCLWcYWnvYrh4e7T-QtAm1ynjoxEihP9ClfuIjJt1a-~6pr3k0t1p18njQw0GVs0erwXmOIDfCvE~XXLDBS98J~PslutGEf6Bl4w32tX1ECFftIFZjqnhySZsdeDm-mArd5u5lWYCwaBENY64rSw5cX8HfCpWsMwjh2LnQC5vYU6nJkFN1tEP5vgowI8CAuu6r-l2BItiSvpu3yrotPMhEETk3Ew1XT-pbG-ZKiIEcojMQUSGSH0jp1tXISv-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670869&ar=5056130&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy022f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristics of pre-integration sites and TSD lengths for HS-MEs. (A) Sequence logos for each HS-ME type at the integration sites. (B) Line plots showing the frequencies of TSDs at each length for each type of HS-MEs. </p></div></div>                            <h3 scrollto-destination=122670870 id="122670870" class="section-title" >3.9. HS-MEs contribute to genes and regulatory elements</h3>
<p>To predict the functional impact of HS-MEs, we analysed the gene context of their insertion sites based the most updated version (release 23 July 2015) of gene annotation data from the GENCODE project<a reveal-id="dsy022-B46" data-open="dsy022-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> combined with those in NCBI RefGene.<a reveal-id="dsy022-B47" data-open="dsy022-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> As shown in <a reveal-id="dsy022-T3" data-open="dsy022-T3" class="link link-reveal link-table xref-fig">Table 3</a>, a total of 7,547 HS-MEs, representing more than half of all HS-MEs, are located in protein coding genes and non-coding RNAs as well as transcribed pseudogenes, representing a total of 4,607 unique genes/transcripts (data not shown). While the majority of these HS-MEs are located inside introns (<a reveal-id="dsy022-T3" data-open="dsy022-T3" class="link link-reveal link-table xref-fig">Table 3</a>), a significant number (304) also directly participated in the transcriptomes as part of exon regions of protein coding genes, non-coding RNAs, and transcribed pseudogenes. Collectively, these 304 HS-MEs contributed to a total of 84 kb sequence in the reference transcriptome, which totals in ∼134 Mbp in length (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Among those involved in the exons of protein-coding genes, 40 contribute to protein coding. In all these 40 cases, the HS-ME transcripts represent rare splice forms that have not been documented in the NCBI’s RefSeq genes, but reported in the GENCODE dataset, and interestingly SVAs contributed to 32 or 80% of these CDS HS-MEs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). <div content-id="dsy022-T3" class="table-modal"><div class="table-wrap"><div id="dsy022-T3" data-id="dsy022-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of human-specific mobile elements (HS-MEs) in genic regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene region </th><th>Protein coding </th><th>Non-coding RNA </th><th>Transcribed pseudogenes </th><th>Total counts </th><th>% all HS-MEs </th></tr></thead><tbody><tr><td><strong>1kb Promoter</strong> </td><td><strong>60</strong> </td><td><strong>178</strong> </td><td><strong>4</strong> </td><td><strong>242</strong> </td><td><strong>3.2</strong> </td></tr><tr><td><strong>CDS exon</strong> </td><td><strong>40</strong> </td><td><strong>NA</strong> </td><td><strong>NA</strong> </td><td><strong>40</strong> </td><td><strong>0.5</strong> </td></tr><tr><td><strong>Non-coding exons</strong> </td><td><strong>64</strong> </td><td><strong>195</strong> </td><td><strong>5</strong> </td><td><strong>264</strong> </td><td><strong>3.5</strong> </td></tr><tr><td><strong>Intron</strong> </td><td><strong>5,266</strong> </td><td><strong>1,622</strong> </td><td><strong>113</strong> </td><td><strong>7,001</strong> </td><td><strong>92.8</strong> </td></tr><tr><td><strong>Total</strong> </td><td><strong>5,430</strong> </td><td><strong>1,995</strong> </td><td><strong>122</strong> </td><td><strong>7,547</strong> </td><td><strong>100.0</strong> </td></tr></tbody></table></div><div content-id="dsy022-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene region </th><th>Protein coding </th><th>Non-coding RNA </th><th>Transcribed pseudogenes </th><th>Total counts </th><th>% all HS-MEs </th></tr></thead><tbody><tr><td><strong>1kb Promoter</strong> </td><td><strong>60</strong> </td><td><strong>178</strong> </td><td><strong>4</strong> </td><td><strong>242</strong> </td><td><strong>3.2</strong> </td></tr><tr><td><strong>CDS exon</strong> </td><td><strong>40</strong> </td><td><strong>NA</strong> </td><td><strong>NA</strong> </td><td><strong>40</strong> </td><td><strong>0.5</strong> </td></tr><tr><td><strong>Non-coding exons</strong> </td><td><strong>64</strong> </td><td><strong>195</strong> </td><td><strong>5</strong> </td><td><strong>264</strong> </td><td><strong>3.5</strong> </td></tr><tr><td><strong>Intron</strong> </td><td><strong>5,266</strong> </td><td><strong>1,622</strong> </td><td><strong>113</strong> </td><td><strong>7,001</strong> </td><td><strong>92.8</strong> </td></tr><tr><td><strong>Total</strong> </td><td><strong>5,430</strong> </td><td><strong>1,995</strong> </td><td><strong>122</strong> </td><td><strong>7,547</strong> </td><td><strong>100.0</strong> </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy022-T3" data-id="dsy022-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of human-specific mobile elements (HS-MEs) in genic regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene region </th><th>Protein coding </th><th>Non-coding RNA </th><th>Transcribed pseudogenes </th><th>Total counts </th><th>% all HS-MEs </th></tr></thead><tbody><tr><td><strong>1kb Promoter</strong> </td><td><strong>60</strong> </td><td><strong>178</strong> </td><td><strong>4</strong> </td><td><strong>242</strong> </td><td><strong>3.2</strong> </td></tr><tr><td><strong>CDS exon</strong> </td><td><strong>40</strong> </td><td><strong>NA</strong> </td><td><strong>NA</strong> </td><td><strong>40</strong> </td><td><strong>0.5</strong> </td></tr><tr><td><strong>Non-coding exons</strong> </td><td><strong>64</strong> </td><td><strong>195</strong> </td><td><strong>5</strong> </td><td><strong>264</strong> </td><td><strong>3.5</strong> </td></tr><tr><td><strong>Intron</strong> </td><td><strong>5,266</strong> </td><td><strong>1,622</strong> </td><td><strong>113</strong> </td><td><strong>7,001</strong> </td><td><strong>92.8</strong> </td></tr><tr><td><strong>Total</strong> </td><td><strong>5,430</strong> </td><td><strong>1,995</strong> </td><td><strong>122</strong> </td><td><strong>7,547</strong> </td><td><strong>100.0</strong> </td></tr></tbody></table></div><div content-id="dsy022-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene region </th><th>Protein coding </th><th>Non-coding RNA </th><th>Transcribed pseudogenes </th><th>Total counts </th><th>% all HS-MEs </th></tr></thead><tbody><tr><td><strong>1kb Promoter</strong> </td><td><strong>60</strong> </td><td><strong>178</strong> </td><td><strong>4</strong> </td><td><strong>242</strong> </td><td><strong>3.2</strong> </td></tr><tr><td><strong>CDS exon</strong> </td><td><strong>40</strong> </td><td><strong>NA</strong> </td><td><strong>NA</strong> </td><td><strong>40</strong> </td><td><strong>0.5</strong> </td></tr><tr><td><strong>Non-coding exons</strong> </td><td><strong>64</strong> </td><td><strong>195</strong> </td><td><strong>5</strong> </td><td><strong>264</strong> </td><td><strong>3.5</strong> </td></tr><tr><td><strong>Intron</strong> </td><td><strong>5,266</strong> </td><td><strong>1,622</strong> </td><td><strong>113</strong> </td><td><strong>7,001</strong> </td><td><strong>92.8</strong> </td></tr><tr><td><strong>Total</strong> </td><td><strong>5,430</strong> </td><td><strong>1,995</strong> </td><td><strong>122</strong> </td><td><strong>7,547</strong> </td><td><strong>100.0</strong> </td></tr></tbody></table></div></div></p><p>We also examined the contribution of HS-MEs in regulatory elements, specifically the ENCODE ChIP-seq TFBS for 161 factors.<a reveal-id="dsy022-B52" data-open="dsy022-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Among the HS-MEs, 1,167 have sequence overlap with a total of 3,032 TFBS for 142 transcriptional factors (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). Interestingly, proportionally HS-LTRs showed the highest ratio (14.7%), followed by HS-SVAs (12.5%), which are much higher than that for LINEs (7.9%) and SINEs (6.6%), albert SINEs contribute to the largest number of TFBS (1,621), followed by LINEs (690), SVAs (504), and LTRs (217). Our data suggest that these young HS-MEs have started making their ways in participating in transcriptome, protein coding, and regulation of splicing and transcription.</p>                            <h3 scrollto-destination=122670873 id="122670873" class="section-title" >3.10. Deposition and access of the HS-ME data</h3>
<p>We deposited the HS-ME data into the dbRIP database (htpp://dbrip.org) under a study ID of 2018-01 (available for hg19). In dbRIP, the HS-MEs can be visualized in the same as the polymorphic MEs, <em>i.e</em>. in the UCSC genome browser along with other available data tracks or in detailed data page. HS-MEs were distinguished from polymorphic MEs in the dbRIP with a letter ‘h’ at the end of the ID. The HS-ME data (for hg38) in fasta format with the sequences organized into left flanking (400 bp), TSD1, ME insertion, TSD2, and right flanking (400 bp) and all other related information provided in the definition lines are available for downloading from the dbRIP data download page. The HS-ME data are also made available along with the polymorphic MEs data via Track Hub (dbRIP) at the UCSC genome browser site at <a class="link link-uri" href="http://genome.ucsc.edu" target="_blank">http://genome.ucsc.edu</a>.</p>                            <h2 scrollto-destination=122670875 id="122670875" class="section-title" >4. Discussions</h2>
<p>In this study, we aimed to provide a comprehensive compilation of MEs that are uniquely present in the human genomes. This is necessary resource for a comprehensive assessment of the MEs’ impacts on human biology. By taking the advantages of the much-improved reference genome sequences for humans and the closely related non-human primates and by tackling the MEs inside repetitive sequences and utilizing a more robust multi-way comparative genomics strategy, we were able to identify a total of 14,870 HS-MEs. Among HS-MEs, more than half (8,049) were reported as HS-MEs for the first time, thus representing a significant improvement. Such a comprehensive list of HS-MEs provides unique opportunities in examining the pattern and trend of retrotransposition during human genome evolution since the divergence from chimpanzee and gaining new insights regarding the roles of MEs in human evolution.</p>                            <h3 scrollto-destination=122670877 id="122670877" class="section-title" >4.1. Challenges in identifying HS-MEs</h3>
<p>As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, the number of MEs annotated in the human reference genome increased significantly in the most recent version (GRC38, December 2013) compared with an earlier version released in 2004 (GRC35, UCSC hg17),<a reveal-id="dsy022-B53" data-open="dsy022-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> which covered the first major updates since the initial publication of the human draft genome in 2001.<a reveal-id="dsy022-B1" data-open="dsy022-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> MEs increased ∼289,000 in number and ∼140 Mb in sequence, leading to the increase of ME percentage in the human genome from 48.8% in GRC35 to 52.1% in GRC38 (based on non-gap sequences). The larger increase among ME type is for LTRs (∼34,000 in number and ∼16.5 Mbp in size. L1s had an increase of 42,000 in number and 21.8 Mbp in size. The increase for <em>Alu</em>s is ∼12,300 in number and ∼675 kb in size (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p><p>Despite constant improvements in the quality of the reference genome sequences for human and other primates and of the related bioinformatics tools, obtaining a precise list of MEs uniquely present in the human genomes remains difficult with many challenges. Several factors contribute to the complications in this task, and these include but are not limited to: (i) incomplete coverage of the reference genome sequences for human and more so for other primates as exemplified in a few recent publications related to human and other primates<a reveal-id="dsy022-B36" data-open="dsy022-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsy022-B54" data-open="dsy022-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a>; (ii) assembly errors, particularly in regions rich of MEs; (iii) genome rearrangements occurring in a lineage- and species-specific fashion, often involving or mediated by MEs<a reveal-id="dsy022-B17 dsy022-B18 dsy022-B19 dsy022-B20" data-open="dsy022-B17 dsy022-B18 dsy022-B19 dsy022-B20" class="link link-ref link-reveal xref-bibr"><sup>17–20</sup></a>; (iv) mis-annotation of MEs.</p><p>Prior similar studies on HS-MEs, best represented by Mills et al<em>.</em>,<a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> were limited by all these factors as well as by the limitation of methodologies and study scope that either focused on a specific type of MEs or part of the genome.<a reveal-id="dsy022-B8" data-open="dsy022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy022-B33" data-open="dsy022-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsy022-B55" data-open="dsy022-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsy022-B56" data-open="dsy022-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Most of the MEs in the repetitive regions, including these in the ME-rich regions, were excluded from all previous studies.</p>                            <h3 scrollto-destination=122670881 id="122670881" class="section-title" >4.2. Improvements for identification of HS-MEs</h3>
<p>In our study, we tried to address most of the issues mentioned above. In addition to the use of the best available reference genome sequences for human and other primates, we took an unbiased approach to cover all annotated MEs in the human reference genome, thus representing the first study to include MEs in the repetitive regions, particularly those within the ME-rich regions. This has contributed to 3,744 HS-MEs or 47% of the 8,049 novel HS-MEs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4b</a></span>). Furthermore, the process of integrating MEs in the original RepeatMasker provided a more accurate counting of the transposition events represented by the MEs in the human genome and locations of their flanking sequences. The former leads to more accurate DNA transposition rate, while the latter is critical for identifying HS-MEs and characterizing their sequences including the TSDs. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, the fragmentation affects a significant proportion of MEs (as high as ∼60% in L1s) with the rates seemly to be positively correlated with the element length and relative ages (data not shown). The integration led to the identification of 998 HS-MEs that would not have been identified otherwise, and this has in part contributed to the higher ratio of novel HS-MEs for LTRs and L1s than SVAs and <em>Alu</em>s (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4b</a></span>). LTRs and L1s are much longer in average than SVAs and <em>Alu</em>s, so they have a high chance of being interrupted by genomic rearrangement events, while full-length LTRs were also reported arbitrarily as three entries by RepeatMasker. The use of multiple non-human primate genomes not only helped reduce the false positive by providing orthologous sequences for gaps or regions of rearrangements in the chimpanzee genome, but also helped increase the sensitivity via contributing to 822 novel HS-MEs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4a</a></span>).</p><p>The use of two sequence alignment tools, blat and liftOver, in identifying the orthologous sequences also helped reduce the false positives that can be caused by many of the aforementioned complicating factors. LiftOver uses alignment data linking closely related genomes through blastz,<a reveal-id="dsy022-B57" data-open="dsy022-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> which focuses on large-scale synteny conservation, and is basically the strategy used in the previous analysis of human- and chimp-specific MEs by Mills et al.<a reveal-id="dsy022-B30" data-open="dsy022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The method may work well for well conserved regions, but usually performs poorly for regions involving local rearrangements or with high density of repetitive sequences or misplacement of contigs during assembly. Therefore, it may miss to detect the presence of orthologous MEs and generate high false positives, most likely as category II HS-MEs (missing orthologous flanking regions). In contrast, the blat method focuses on identifying local alignments and performs better in handling regions with species-specific sequence changes near MEs. But it may miss some true HS-MEs by generating some false positive detection of orthologous MEs due to non-orthologous sequence similarity. Therefore, by requiring support of both methods for calling an HS-ME status, we were able to achieve a minimal level of false positive rate in HS-ME detection with an estimated sensitivity and specificity of 95.4 and 97%, respectively. We would like to believe that our current list of HS-MEs may still represent an underestimate of all HS-MEs that may exist in the human genomes due to many complications associated with ME analysis and our emphasis on reducing the false positives. Notably, our list does not cover the processed pseudogenes, which represent copies of mRNAs that were copied back into the genome as a side-product of L1-based transposition,<a reveal-id="dsy022-B58" data-open="dsy022-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsy022-B59" data-open="dsy022-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> and the human-specific processed pseudogenes are likely to be in the order of hundreds based on the level of detected polymorphic levels.<a reveal-id="dsy022-B60" data-open="dsy022-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsy022-B61" data-open="dsy022-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a></p>                            <h3 scrollto-destination=122670884 id="122670884" class="section-title" >4.3. The pattern of HS-MEs in MEs</h3>
<p>MEs in repetitive regions have been mostly ignored by prior studies since it is much more difficult to analyse than those inserted into unique genomic regions. Not only the repetitive regions are more prone to sequence assembly errors, it is also more challenging for computational analysis aiming at specifies-specific and polymorphic ME entries. In a sense, the ignorance of these MEs may be justified for locating in regions assumed to be ‘less functionally important’. However, this may not be true, and by ignoring them, we might be missing useful information. In this study, by taking an unbiased approached, we identified more than 3,700 HS-MEs in the ME regions, which count for approximately a quarter of all HS-MEs. These MEs allowed us to observe some interesting patterns regarding the trend of DNA transposition in the human genome and the different behaviours of different ME types.</p><p>When all MEs were considered, the rates of MEs that inserted inside MEs by ME type showed an increasing trend in the order of L1, DNA, LTR, SINE, and SVA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6a</a></span>). We think these differential ratios reflect the relative overall ages of these ME types with the older groups (<em>e.g</em>. LINEs) having less MEs than the younger groups (e.g. SVAs) in the genome as the targets for insertions.</p><p>In comparison, the ratios of HS-MEs in MEs are more or less similar and lack a clear trend among the ME types ranging from ∼35% for both L1 and LTR to ∼46% for <em>Alu</em> and ∼41% for SVA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6b</a></span>). This is expected since HS-MEs from all types share approximately the same time span in the human genome and thus they had the same amount of MEs as insertion targets. The differences we see here with HS-MEs may represent a more accurate reflection of the preferences for insertions into MEs among ME types. In other words, <em>Alu</em>s are more likely to insert to MEs than L1s, LTRs, and SVAs.</p><p>An unusual observation was made when we examined each ME type as the targets of HS-ME insertion individually (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6c</a></span>). Here we examined the preference of each HS-ME type for each type of MEs as the targets. By the density of HS-MEs, L1, LTR, and <em>Alu</em> all showed a more or less consistent decreasing trend for inserting into LINEs, DNA transposons, LTRs, SINEs, and SVAs, seemly correlating with their overall age in the genome from old to new. This is expected as older MEs have more chance to be inserted by later MEs. For HS-SVAs, while the trend seems to be the same among LINEs, DNAs, LTRs, and SINEs as the targets, they showed a density in SVAs that is more than 120, 40, 30, 16 times higher than that in SINEs, LTRs, DNAs and LINEs, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6c</a></span>). Furthermore, SVAs have the highest density of HS-MEs among the ME types and with over 98% of HE-MEs in SVAs being SVAs and the rest 2% being <em>Alu</em>s. This extreme bias of HS-SVAs for SVAs and vice versa is remarkable since SVAs are the youngest ME type in the human genome, and by random chance we would expect to see the lowest ratio of HS-MEs in SVAs. In fact, very few or no insertion of HS-MEs from L1s, LTRs, and <em>Alu</em>s were seen in SVAs. The reason for this extremely strong preference for SVAs inserting into SVAs and its functional implication are to be investigated in future studies.</p>                            <h3 scrollto-destination=122670889 id="122670889" class="section-title" >4.4. The retrotransposition activity level during early human evolution</h3>
<p>In the last decade, we and others have extensively studied the profiles of active MEs and their relative retrotransposition levels in the human genome based on analysis of polymorphic MEs.<a reveal-id="dsy022-B8" data-open="dsy022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy022-B31" data-open="dsy022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsy022-B32" data-open="dsy022-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsy022-B34" data-open="dsy022-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsy022-B42 dsy022-B43 dsy022-B44" data-open="dsy022-B42 dsy022-B43 dsy022-B44" class="link link-ref link-reveal xref-bibr"><sup>42–44</sup></a><sup>,</sup><a reveal-id="dsy022-B56" data-open="dsy022-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><sup>,</sup><a reveal-id="dsy022-B62 dsy022-B63 dsy022-B64 dsy022-B65 dsy022-B66" data-open="dsy022-B62 dsy022-B63 dsy022-B64 dsy022-B65 dsy022-B66" class="link link-ref link-reveal xref-bibr"><sup>62–66</sup></a> These studies show that L1s, <em>Alu</em>s, SVAs, and HERV-K are ME types remaining ongoing retrotransposition activity. The most active subfamilies for each ME type have also been established. It is worth to point out that these data reflect the retrotransposition profiles in the most recent phase of modern human evolution, during which humans migrates and established as distinct populations, and the rate of retrotransposition in this period might not necessarily be the same as for the earlier part of human evolution. The availability of a comprehensive list of HS-MEs in relation to chimpanzee and other non-human primates can be used to fill this gap.</p><p>As expected, our data confirmed the active retrotransposon subfamilies identified by prior studies, including, Ya5 and Yb8/9 for <em>Alu</em>s, L1HS for L1s, the F-subfamily for SVAs, and HERV-K for LTR retrotransposons (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). In the meantime, notable differences were also observed. First, the ratios of HS-MEs in relation to all MEs in the same subfamilies are much higher than that for polymorphic MEs. Second, many extra active subfamilies were observed, and these include the <em>Alu</em>Yf and <em>Alu</em>Yk subfamilies among <em>Alu</em>s, the L1P4 and L1M subfamilies for L1s, SVA_A subfamily, and ERV1 subfamily for LTR retrotransposons (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). While the higher ratios of HS-MEs compared with that of polymorphic MEs in each active subfamily may be explained merely by the longer time span covered by HS-MEs, the presence of the extra active subfamilies not seen based on polymorphic MEs might be a result of the longer time-span for HS-MEs and the dropping retrotransposition activity in the modern human genomes.</p>                            <h3 scrollto-destination=122670892 id="122670892" class="section-title" >4.5. Y chromosome as a hot target for HS-LTRs</h3>
<p>As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S2</a></span>, and <a reveal-id="dsy022-F2" data-open="dsy022-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, the HS-ME density on Y chromosome is significantly higher than all other chromosomes. By ME type, HS-MEs from <em>Alu</em>s, L1s, and LTRs all showed a higher density for Y chromosome. For <em>Alu</em>s and L1s, the HS-ME density in Y chromosome is at least 4 times higher than the genome average, while for HS-LTRs, the density in Y chromosome is more than 30 times higher than the genome average. In contrary, HS-SVAs showed a strong bias against Y chromosome with the density being only ∼35% of the genome average. This differential bias for Y chromosome among different types of HS-MEs cannot be a result of artefacts, such as poorer sequence quality or lack of Y-chromosome-specific sequences for other primates, as such factors would lead to the same trend of bias for all ME types. It does not seem to be explainable by the gene density either, since chromosome 13 has a lower gene density than Y chromosome, but its HS-ME densities for all 4 types of HS-MEs do not show a strong deviation from the genome average (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). The extreme contrast between the densities of SVAs and LTRs for Y chromosome as well as the differences between Y chromosome and all other chromosomes for the density of all HS-MEs are well visualized in the genome plots of HS-MEs shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. The high preference for Y chromosome by HS-MEs for <em>Alu</em>s and L1s may be partially explained by the lack of homologous recombination-based deletion and lack of selection pressure and relative longer time in male germline.<a reveal-id="dsy022-B67" data-open="dsy022-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a><sup>,</sup><a reveal-id="dsy022-B68" data-open="dsy022-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> However, we cannot explain the extreme strong positive bias for HS-LTRs and negative bias for HS-SVAs. Therefore, the exact reason behind the significant differential bias for different types of HS-MEs in Y chromosome is unknown, and so is the impact of such biases. Nevertheless, our observation does seem to support a recent notion that remodelling and regeneration have dominated chimpanzee and human male specific chromosome evolution, while genetic decay seems to be a general trend in the evolution of Y chromosomes.<a reveal-id="dsy022-B45" data-open="dsy022-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> In addressing the recent heated debate among the science community about whether chromosome Y is disappearing,<a reveal-id="dsy022-B69" data-open="dsy022-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> our result seems to hint that Y chromosome is certainly not disappearing for human and HS-MEs may have contributed to the observed fast evolving pattern on Y chromosome after the human and chimpanzee divergence.<a reveal-id="dsy022-B45" data-open="dsy022-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a></p>                            <h3 scrollto-destination=122670894 id="122670894" class="section-title" >4.6. The impact of MEs on human genome size and gene function</h3>
<p>The 14,870 HS-MEs collectively contributed a net increase of the genome size by 14.2 Mbp since LCA with chimpanzee (<a reveal-id="dsy022-T2" data-open="dsy022-T2" class="link link-reveal link-table xref-fig">Table 2</a>). This size of genome increase is close to one-quarter of the Y chromosome and is larger than the genomes of free-living eukaryotic organisms, such as yeast.<a reveal-id="dsy022-B70" data-open="dsy022-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> This size increase is significant for the relative short period of time and by a single molecular mechanism, and it may represent only well-defined significant change in the human genome during human evolution. The only other molecular mechanism that could contribute a significant size change to the human genome would be genomic segmental duplication.<a reveal-id="dsy022-B71" data-open="dsy022-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> However, no data are available about the exact amount of human-specific segmental duplications occurred for the same period of human evolution.</p><p>We attempted to assess the overall functional impact of these HS-MEs on genes by examining their location in the genome in relation to known genes and functional regulatory elements. A total of 7,547 or 50.7% of these HS-MEs are located inside or in the 1 kb promoter regions of genes for protein coding, non-coding RNAs, as well as transcribed pseudogenes (<a reveal-id="dsy022-T3" data-open="dsy022-T3" class="link link-reveal link-table xref-fig">Table 3</a>), which represent 4,607 unique genes/transcripts (data not shown). Among these, 240 HS-MEs are part of transcripts, representing mostly alternative splice forms. Interestingly, in 40 of these cases, an HS-ME contributes to part of the coding region in the transcript, albeit all of these transcripts represent rare splice forms documented the GENCODE, but are not yet included in NCBI Ref Gene list (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). A profound observation here is that 80% of these CDS HS-MEs are contributed by SVAs, which represent the youngest and most active group of MEs in the human genome.<a reveal-id="dsy022-B10" data-open="dsy022-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy022-B31" data-open="dsy022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> This pattern seems to hold true when all MEs were included in the analysis of MEs’ contribution to human transcriptome (Joshi et al<em>.</em>, manuscript in preparation). These data suggest to us that SVAs as the youngest and most active group of MEs in the human genome might have played a significant role in past human evolution and have the highest potential among all ME types in impacting future human genome evolution.</p><p>Outside of the promote and exon regions, 1,167 of the HS-MEs contribute to 3,032 binding sites for 142 of the 161 examined transcriptional factors (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). While their specific functional impact would be hard to predict computationally and can only be more accurately assessed/validated experimentally, many examples of such functional impact have been demonstrated.<a reveal-id="dsy022-B14" data-open="dsy022-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsy022-B21" data-open="dsy022-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p><p>In summary, our data suggest that, despite being very young in the genome, many of these HS-MEs have already participated in gene function via regulation of transcription, splicing, and protein coding, and there may be more potentials for their future participation as demonstrated by Ward et al<em>.</em><a reveal-id="dsy022-B26" data-open="dsy022-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                            <h3 scrollto-destination=122670899 id="122670899" class="section-title" >4.7. Future directions</h3>
<p>Due to the technical challenges associated with the analysis of MEs and deficiencies of the reference genome sequences for human and other primates, our list of HS-MEs still suffers a certain level of false negatives and false positives. We can expect that the number of HS-MEs continue to increase from regions with sequencing gaps, especially regions highly rich of repetitive sequences, such as the centromere and telomere regions, which may be hot spots for certain types of ME insertion, such as LTRs.<a reveal-id="dsy022-B72" data-open="dsy022-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> A good proportion of the HS-MEs would be shared by other archaic human species/subspecies, notably the Neanderthals and Denisovans.<a reveal-id="dsy022-B73 dsy022-B74 dsy022-B75" data-open="dsy022-B73 dsy022-B74 dsy022-B75" class="link link-ref link-reveal xref-bibr"><sup>73–75</sup></a> Once high quality genome sequences become available for these genomes, we can further break down the human evolution into more phases and examine and compare the DNA transposition profiles among these periods.<a reveal-id="dsy022-B76" data-open="dsy022-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> It would be interesting to find out see how many of these HS-MEs are truly unique to <em>Homo sapiens</em>. Furthermore, a certain portion of the HS-MEs are polymorphic, and it should be useful to generate a list of HS-MEs common to all humans (minimal set of HS-MEs) and another list of HS-MEs that are polymorphic (including also those outside of the reference genome). The former would be useful for analysing MEs’ impact on evolution of all modern humans, while the latter would be useful for studying MEs’ contribution to genetic and phenotypic diversity among human populations and individuals. Additionally, we are extending similar analysis to all other primate genomes, for which genome sequences are available, to access the impact of retrotransposition on primate evolution (work in progress).</p>                            <h2 scrollto-destination=122670901 id="122670901" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work is in part supported by grants from the Canadian Research Chair program, Canadian Foundation of Innovation, Ontario Ministry of Research and Innovation, Canadian Natural Science and Engineering Research Council (NSERC), and Brock University to PL, and was made possible by Compute Canada/SHARCNET high performance computing facilities.</p>                            <h3 scrollto-destination=122670903 id="122670903" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=122670905 id="122670905" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy022-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Linton</div> <div class="given-names">L.M.</div></div>, <div class="name string-name"><div class="surname">Birren</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2001</div>, 
<div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="volume">409</div>, <div class="fpage">860</div>–<div class="lpage">921</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.S.%20Lander&amp;author=L.M.%20Linton&amp;author=B.%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deininger</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Moran</div> <div class="given-names">J.V.</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div></div> <div class="year">2003</div>, 
<div class="article-title">Mobile elements and mammalian genome evolution</div>, <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">13</div>, <div class="fpage">651</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20elements%20and%20mammalian%20genome%20evolution&amp;author=P.L.%20Deininger&amp;author=J.V.%20Moran&amp;author=M.A.%20Batzer&amp;author=H.H.%20Kazazian&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=13&amp;pages=651-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2003.10.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2003.10.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2003.10.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14638329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cordaux</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">The impact of retrotransposons on human genome evolution</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">10</div>, <div class="fpage">691</div>–<div class="lpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20retrotransposons%20on%20human%20genome%20evolution&amp;author=R.%20Cordaux&amp;author=M.A.%20Batzer&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet&amp;volume=10&amp;pages=691-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2640" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2640" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2640"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19763152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div></div> <div class="year">2004</div>, 
<div class="article-title">Mobile elements: drivers of genome evolution</div>, <div class="source">Science</div> , <div class="volume">303</div>, <div class="fpage">1626</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20elements%3A%20drivers%20of%20genome%20evolution&amp;author=H.H.%20Kazazian&amp;publication_year=2004&amp;journal=Science&amp;volume=303&amp;pages=1626-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1089670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1089670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1089670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15016989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ostertag</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div></div> <div class="year">2001</div>, 
<div class="article-title">Biology of mammalian L1 retrotransposons</div>, <div class="source">Annu. Rev. Genet</div> ., <div class="volume">35</div>, <div class="fpage">501</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biology%20of%20mammalian%20L1%20retrotransposons&amp;author=E.M.%20Ostertag&amp;author=H.H.%20Kazazian&amp;publication_year=2001&amp;journal=Annu.%20Rev.%20Genet&amp;volume=35&amp;pages=501-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.35.102401.091032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.35.102401.091032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.35.102401.091032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11700292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Deininger</div> <div class="given-names">P.L.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Alu repeats and human genomic diversity</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">3</div>, <div class="fpage">370</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alu%20repeats%20and%20human%20genomic%20diversity&amp;author=M.A.%20Batzer&amp;author=P.L.%20Deininger&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet&amp;volume=3&amp;pages=370-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg798" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg798"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div>, <div class="name string-name"><div class="surname">Goodier</div> <div class="given-names">J.L.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">LINE drive. retrotransposition and genome instability</div>, <div class="source">Cell</div> , <div class="volume">110</div>, <div class="fpage">277</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LINE%20drive.%20retrotransposition%20and%20genome%20instability&amp;author=H.H.%20Kazazian&amp;author=J.L.%20Goodier&amp;publication_year=2002&amp;journal=Cell&amp;volume=110&amp;pages=277-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(02)00868-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(02)00868-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(02)00868-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Bennett</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Iskow</div> <div class="given-names">R.C.</div></div>, <div class="name string-name"><div class="surname">Devine</div> <div class="given-names">S.E.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Which transposable elements are active in the human genome?</div> <div class="source">Trends Genet</div> ., <div class="volume">23</div>, <div class="fpage">183</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Which%20transposable%20elements%20are%20active%20in%20the%20human%20genome%3F&amp;author=R.E.%20Mills&amp;author=E.A.%20Bennett&amp;author=R.C.%20Iskow&amp;author=S.E.%20Devine&amp;publication_year=2007&amp;journal=Trends%20Genet&amp;volume=23&amp;pages=183-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17331616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ostertag</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Goodier</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div></div> <div class="year">2003</div>, 
<div class="article-title">SVA elements are nonautonomous retrotransposons that cause disease in humans</div>, <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">73</div>, <div class="fpage">1444</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SVA%20elements%20are%20nonautonomous%20retrotransposons%20that%20cause%20disease%20in%20humans&amp;author=E.M.%20Ostertag&amp;author=J.L.%20Goodier&amp;author=Y.%20Zhang&amp;author=H.H.%20Kazazian&amp;publication_year=2003&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=73&amp;pages=1444-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/380207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F380207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F380207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14628287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Xing</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Grover</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">SVA elements: a hominid-specific retroposon family</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">354</div>, <div class="fpage">994</div>–<div class="lpage">1007</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SVA%20elements%3A%20a%20hominid-specific%20retroposon%20family&amp;author=H.%20Wang&amp;author=J.%20Xing&amp;author=D.%20Grover&amp;publication_year=2005&amp;journal=J.%20Mol.%20Biol&amp;volume=354&amp;pages=994-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2005.09.085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2005.09.085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2005.09.085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16288912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doolittle</div> <div class="given-names">W.F.</div></div>, <div class="name string-name"><div class="surname">Sapienza</div> <div class="given-names">C.</div></div></div> <div class="year">1980</div>, 
<div class="article-title">Selfish genes, the phenotype paradigm and genome evolution</div>, <div class="source">Nature</div> , <div class="volume">284</div>, <div class="fpage">601</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selfish%20genes%2C%20the%20phenotype%20paradigm%20and%20genome%20evolution&amp;author=W.F.%20Doolittle&amp;author=C.%20Sapienza&amp;publication_year=1980&amp;journal=Nature&amp;volume=284&amp;pages=601-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/284601a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F284601a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F284601a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6245369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Symer</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Connelly</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Szak</div> <div class="given-names">S.T.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">Human l1 retrotransposition is associated with genetic instability in vivo</div>, <div class="source">Cell</div> , <div class="volume">110</div>, <div class="fpage">327</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20l1%20retrotransposition%20is%20associated%20with%20genetic%20instability%20in%20vivo&amp;author=D.E.%20Symer&amp;author=C.%20Connelly&amp;author=S.T.%20Szak&amp;publication_year=2002&amp;journal=Cell&amp;volume=110&amp;pages=327-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(02)00839-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(02)00839-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(02)00839-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Szak</div> <div class="given-names">S.T.</div></div>, <div class="name string-name"><div class="surname">Pickeral</div> <div class="given-names">O.K.</div></div>, <div class="name string-name"><div class="surname">Landsman</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Boeke</div> <div class="given-names">J.D.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Identifying related L1 retrotransposons by analyzing 3' transduced sequences</div>, <div class="source">Genome Biol</div> ., <div class="volume">4</div>, <div class="fpage">R30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20related%20L1%20retrotransposons%20by%20analyzing%203'%20transduced%20sequences&amp;author=S.T.%20Szak&amp;author=O.K.%20Pickeral&amp;author=D.%20Landsman&amp;author=J.D.%20Boeke&amp;publication_year=2003&amp;journal=Genome%20Biol&amp;volume=4&amp;pages=R30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-4-5-r30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-4-5-r30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-4-5-r30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12734010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Szak</div> <div class="given-names">S.T.</div></div>, <div class="name string-name"><div class="surname">Boeke</div> <div class="given-names">J.D.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Transcriptional disruption by the L1 retrotransposon and implications for mammalian transcriptomes</div>, <div class="source">Nature</div> , <div class="volume">429</div>, <div class="fpage">268</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20disruption%20by%20the%20L1%20retrotransposon%20and%20implications%20for%20mammalian%20transcriptomes&amp;author=J.S.%20Han&amp;author=S.T.%20Szak&amp;author=J.D.%20Boeke&amp;publication_year=2004&amp;journal=Nature&amp;volume=429&amp;pages=268-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02536" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02536" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02536"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15152245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wheelan</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Aizawa</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Boeke</div> <div class="given-names">J.D.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Gene-breaking: a new paradigm for human retrotransposon-mediated gene evolution</div>, <div class="source">Genome Res</div> ., <div class="volume">15</div>, <div class="fpage">1073</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene-breaking%3A%20a%20new%20paradigm%20for%20human%20retrotransposon-mediated%20gene%20evolution&amp;author=S.J.%20Wheelan&amp;author=Y.%20Aizawa&amp;author=J.S.%20Han&amp;author=J.D.%20Boeke&amp;publication_year=2005&amp;journal=Genome%20Res&amp;volume=15&amp;pages=1073-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3688905" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3688905" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3688905"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16024818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mita</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Boeke</div> <div class="given-names">J.D.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">How retrotransposons shape genome regulation</div>, <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">37</div>, <div class="fpage">90</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20retrotransposons%20shape%20genome%20regulation&amp;author=P.%20Mita&amp;author=J.D.%20Boeke&amp;publication_year=2016&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=37&amp;pages=90-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2016.01.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2016.01.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2016.01.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26855260" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Callinan</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Herke</div> <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Garber</div> <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Alu retrotransposition-mediated deletion</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">348</div>, <div class="fpage">791</div>–<div class="lpage">800</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alu%20retrotransposition-mediated%20deletion&amp;author=P.A.%20Callinan&amp;author=J.%20Wang&amp;author=S.W.%20Herke&amp;author=R.K.%20Garber&amp;author=P.%20Liang&amp;author=M.A.%20Batzer&amp;publication_year=2005&amp;journal=J.%20Mol.%20Biol&amp;volume=348&amp;pages=791-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2005.02.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2005.02.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2005.02.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15843013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sen</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Genomic rearrangements by LINE-1 insertion-mediated deletion in the human and chimpanzee lineages</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">4040</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20rearrangements%20by%20LINE-1%20insertion-mediated%20deletion%20in%20the%20human%20and%20chimpanzee%20lineages&amp;author=K.%20Han&amp;author=S.K.%20Sen&amp;author=J.%20Wang&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=4040-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki718" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki718" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki718"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16034026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sen</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Human genomic deletions mediated by recombination between Alu elements</div>, <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">79</div>, <div class="fpage">41</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20genomic%20deletions%20mediated%20by%20recombination%20between%20Alu%20elements&amp;author=S.K.%20Sen&amp;author=K.%20Han&amp;author=J.%20Wang&amp;publication_year=2006&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=79&amp;pages=41-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/504600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F504600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F504600"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16773564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">T.J.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Alu recombination-mediated structural deletions in the chimpanzee genome</div>, <div class="source">PLoS Genet</div> ., <div class="volume">3</div>, <div class="fpage">1939</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alu%20recombination-mediated%20structural%20deletions%20in%20the%20chimpanzee%20genome&amp;author=K.%20Han&amp;author=J.%20Lee&amp;author=T.J.%20Meyer&amp;publication_year=2007&amp;journal=PLoS%20Genet&amp;volume=3&amp;pages=1939-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17953488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quinn</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Bubb</div> <div class="given-names">V.J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">SVA retrotransposons as modulators of gene expression</div>, <div class="source">Mob. Genet. Elements</div> ., <div class="volume">4</div>, <div class="fpage">e32102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SVA%20retrotransposons%20as%20modulators%20of%20gene%20expression&amp;author=J.P.%20Quinn&amp;author=V.J.%20Bubb&amp;publication_year=2014&amp;journal=Mob.%20Genet.%20Elements&amp;volume=4&amp;pages=e32102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/mge.32102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fmge.32102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fmge.32102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25077041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Konkel</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A mobile threat to genome stability: the impact of non-LTR retrotransposons upon the human genome</div>, <div class="source">Semin. Cancer Biol</div> ., <div class="volume">20</div>, <div class="fpage">211</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20mobile%20threat%20to%20genome%20stability%3A%20the%20impact%20of%20non-LTR%20retrotransposons%20upon%20the%20human%20genome&amp;author=M.K.%20Konkel&amp;author=M.A.%20Batzer&amp;publication_year=2010&amp;journal=Semin.%20Cancer%20Biol&amp;volume=20&amp;pages=211-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.semcancer.2010.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.semcancer.2010.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.semcancer.2010.03.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20307669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hancks</div> <div class="given-names">D.C.</div></div>, <div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div></div> <div class="year">2012</div>, 
<div class="article-title">Active human retrotransposons: variation and disease</div>, <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">22</div>, <div class="fpage">191</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Active%20human%20retrotransposons%3A%20variation%20and%20disease&amp;author=D.C.%20Hancks&amp;author=H.H.%20Kazazian&amp;publication_year=2012&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=22&amp;pages=191-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2012.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2012.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2012.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22406018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Callinan</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Retrotransposable elements and human disease</div>, <div class="source">Genome Dyn</div> ., <div class="volume">1</div>, <div class="fpage">104</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposable%20elements%20and%20human%20disease&amp;author=P.A.%20Callinan&amp;author=M.A.%20Batzer&amp;publication_year=2006&amp;journal=Genome%20Dyn&amp;volume=1&amp;pages=104-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000092503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000092503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000092503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18724056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ahmed</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Transposable elements are a significant contributor to tandem repeats in the human genome</div>, <div class="source">Comp. Funct. Genomics</div> , <div class="volume">2012</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%20are%20a%20significant%20contributor%20to%20tandem%20repeats%20in%20the%20human%20genome&amp;author=M.%20Ahmed&amp;author=P.%20Liang&amp;publication_year=2012&amp;journal=Comp.%20Funct.%20Genomics&amp;volume=2012&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/2012/947089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2012%2F947089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2F2012%2F947089"> </span></div> </div></div></div></div></div><div content-id="dsy022-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ward</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Barbosa-Morais</div> <div class="given-names">N.L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Latent regulatory potential of human-specific repetitive elements</div>, <div class="source">Mol. Cell</div> , <div class="volume">49</div>, <div class="fpage">262</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Latent%20regulatory%20potential%20of%20human-specific%20repetitive%20elements&amp;author=M.C.%20Ward&amp;author=M.D.%20Wilson&amp;author=N.L.%20Barbosa-Morais&amp;publication_year=2013&amp;journal=Mol.%20Cell&amp;volume=49&amp;pages=262-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2012.11.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2012.11.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2012.11.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23246434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chuong</div> <div class="given-names">E.B.</div></div>, <div class="name string-name"><div class="surname">Elde</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Regulatory evolution of innate immunity through co-option of endogenous retroviruses</div>, <div class="source">Science</div> , <div class="volume">351</div>, <div class="fpage">1083</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20evolution%20of%20innate%20immunity%20through%20co-option%20of%20endogenous%20retroviruses&amp;author=E.B.%20Chuong&amp;author=N.C.%20Elde&amp;author=C.%20Feschotte&amp;publication_year=2016&amp;journal=Science&amp;volume=351&amp;pages=1083-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aad5497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aad5497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aad5497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26941318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">C.R.</div></div>, <div class="name string-name"><div class="surname">Schneider</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Mobile interspersed repeats are major structural variants in the human genome</div>, <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">1171</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20interspersed%20repeats%20are%20major%20structural%20variants%20in%20the%20human%20genome&amp;author=C.R.%20Huang&amp;author=A.M.%20Schneider&amp;author=Y.%20Lu&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=1171-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.05.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.05.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.05.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20602999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div>, <div class="name string-name"><div class="surname">Moran</div> <div class="given-names">J.V.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">The impact of L1 retrotransposons on the human genome</div>, <div class="source">Nat. Genet</div> ., <div class="volume">19</div>, <div class="fpage">19</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20L1%20retrotransposons%20on%20the%20human%20genome&amp;author=H.H.%20Kazazian&amp;author=J.V.%20Moran&amp;publication_year=1998&amp;journal=Nat.%20Genet&amp;volume=19&amp;pages=19-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0598-19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0598-19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0598-19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9590283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Bennett</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Iskow</div> <div class="given-names">R.C.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Recently mobilized transposons in the human and chimpanzee genomes</div>, <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">78</div>, <div class="fpage">671</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recently%20mobilized%20transposons%20in%20the%20human%20and%20chimpanzee%20genomes&amp;author=R.E.%20Mills&amp;author=E.A.%20Bennett&amp;author=R.C.%20Iskow&amp;publication_year=2006&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=78&amp;pages=671-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/501028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F501028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F501028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16532396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gonder</div> <div class="given-names">M.K.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Whole genome computational comparative genomics: a fruitful approach for ascertaining Alu insertion polymorphisms</div>, <div class="source">Gene</div> , <div class="volume">365</div>, <div class="fpage">11</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20computational%20comparative%20genomics%3A%20a%20fruitful%20approach%20for%20ascertaining%20Alu%20insertion%20polymorphisms&amp;author=J.%20Wang&amp;author=L.%20Song&amp;author=M.K.%20Gonder&amp;publication_year=2006&amp;journal=Gene&amp;volume=365&amp;pages=11-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.09.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.09.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.09.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16376498" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ahmed</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Identification of three new Alu Yb subfamilies by source tracking of recently integrated Alu Yb elements</div>, <div class="source">Mob. DNA</div> , <div class="volume">4</div>, <div class="fpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20three%20new%20Alu%20Yb%20subfamilies%20by%20source%20tracking%20of%20recently%20integrated%20Alu%20Yb%20elements&amp;author=M.%20Ahmed&amp;author=W.%20Li&amp;author=P.%20Liang&amp;publication_year=2013&amp;journal=Mob.%20DNA&amp;volume=4&amp;pages=25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1759-8753-4-25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1759-8753-4-25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1759-8753-4-25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24216009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buzdin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ustyugova</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Khodosevich</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">Human-specific subfamilies of HERV-K (HML-2) long terminal repeats: three master genes were active simultaneously during branching of hominoid lineages</div>, <div class="source">Genomics</div> , <div class="volume">81</div>, <div class="fpage">149</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human-specific%20subfamilies%20of%20HERV-K%20(HML-2)%20long%20terminal%20repeats%3A%20three%20master%20genes%20were%20active%20simultaneously%20during%20branching%20of%20hominoid%20lineages&amp;author=A.%20Buzdin&amp;author=S.%20Ustyugova&amp;author=K.%20Khodosevich&amp;publication_year=2003&amp;journal=Genomics&amp;volume=81&amp;pages=149-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0888-7543(02)00027-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0888-7543(02)00027-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0888-7543(02)00027-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12620392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shin</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Son</div> <div class="given-names">S.Y.</div></div>, <div class="name string-name"><div class="surname">Ahn</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.S.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Human-specific HERV-K insertion causes genomic variations in the human genome</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e60605</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human-specific%20HERV-K%20insertion%20causes%20genomic%20variations%20in%20the%20human%20genome&amp;author=W.%20Shin&amp;author=J.%20Lee&amp;author=S.Y.%20Son&amp;author=K.%20Ahn&amp;author=H.S.%20Kim&amp;author=K.%20Han&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e60605" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0060605" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0060605" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0060605"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23593260" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chimpanzee</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Analysis</div> <div class="given-names">C.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Initial sequence of the chimpanzee genome and comparison with the human genome</div>, <div class="source">Nature</div> , <div class="volume">437</div>, <div class="fpage">69</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequence%20of%20the%20chimpanzee%20genome%20and%20comparison%20with%20the%20human%20genome&amp;author=S.%20Chimpanzee&amp;author=C.%20Analysis&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=69-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16136131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gordon</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Huddleston</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chaisson</div> <div class="given-names">M.J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Long-read sequence assembly of the gorilla genome</div>, <div class="source">Science</div> , <div class="volume">352</div>, <div class="fpage">aae0344</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-read%20sequence%20assembly%20of%20the%20gorilla%20genome&amp;author=D.%20Gordon&amp;author=J.%20Huddleston&amp;author=M.J.%20Chaisson&amp;publication_year=2016&amp;journal=Science&amp;volume=352&amp;pages=aae0344" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aae0344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aae0344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aae0344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27034376" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Locke</div> <div class="given-names">D.P.</div></div>, <div class="name string-name"><div class="surname">Hillier</div> <div class="given-names">L.W.</div></div>, <div class="name string-name"><div class="surname">Warren</div> <div class="given-names">W.C.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Comparative and demographic analysis of orang-utan genomes</div>, <div class="source">Nature</div> , <div class="volume">469</div>, <div class="fpage">529</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20and%20demographic%20analysis%20of%20orang-utan%20genomes&amp;author=D.P.%20Locke&amp;author=L.W.%20Hillier&amp;author=W.C.%20Warren&amp;publication_year=2011&amp;journal=Nature&amp;volume=469&amp;pages=529-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09687" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09687" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09687"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21270892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carbone</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Harris</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Gnerre</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Gibbon genome and the fast karyotype evolution of small apes</div>, <div class="source">Nature</div> , <div class="volume">513</div>, <div class="fpage">195</div>–<div class="lpage">201</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gibbon%20genome%20and%20the%20fast%20karyotype%20evolution%20of%20small%20apes&amp;author=L.%20Carbone&amp;author=R.A.%20Harris&amp;author=S.%20Gnerre&amp;publication_year=2014&amp;journal=Nature&amp;volume=513&amp;pages=195-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25209798" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="collab">Rhesus Macaque Genome, S</div>., <div class="person-group"><div class="name string-name"><div class="surname">Analysis</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gibbs</div> <div class="given-names">R.A.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Evolutionary and biomedical insights from the rhesus macaque genome</div>, <div class="source">Science</div> , <div class="volume">316</div>, <div class="fpage">222</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20and%20biomedical%20insights%20from%20the%20rhesus%20macaque%20genome&amp;author=C.%20Analysis&amp;author=R.A.%20Gibbs&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=222-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1139247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1139247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1139247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17431167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zimin</div> <div class="given-names">A.V.</div></div>, <div class="name string-name"><div class="surname">Cornish</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Maudhoo</div> <div class="given-names">M.D.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A new rhesus macaque assembly and annotation for next-generation sequencing analyses</div>, <div class="source">Biol. Direct</div> ., <div class="volume">9</div>, <div class="fpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20rhesus%20macaque%20assembly%20and%20annotation%20for%20next-generation%20sequencing%20analyses&amp;author=A.V.%20Zimin&amp;author=A.S.%20Cornish&amp;author=M.D.%20Maudhoo&amp;publication_year=2014&amp;journal=Biol.%20Direct&amp;volume=9&amp;pages=20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-9-20" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-9-20" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-9-20"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25319552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Grover</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Azrak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">P.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">dbRIP: a highly integrated database of retrotransposon insertion polymorphisms in humans</div>, <div class="source">Hum. Mutat</div> ., <div class="volume">27</div>, <div class="fpage">323</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dbRIP%3A%20a%20highly%20integrated%20database%20of%20retrotransposon%20insertion%20polymorphisms%20in%20humans&amp;author=J.%20Wang&amp;author=L.%20Song&amp;author=D.%20Grover&amp;author=S.%20Azrak&amp;author=M.A.%20Batzer&amp;author=P.%20Liang&amp;publication_year=2006&amp;journal=Hum.%20Mutat&amp;volume=27&amp;pages=323-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16511833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stewart</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kural</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Stromberg</div> <div class="given-names">M.P.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A comprehensive map of mobile element insertion polymorphisms in humans</div>, <div class="source">PLoS Genet</div> ., <div class="volume">7</div>, <div class="fpage">e1002236</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20map%20of%20mobile%20element%20insertion%20polymorphisms%20in%20humans&amp;author=C.%20Stewart&amp;author=D.%20Kural&amp;author=M.P.%20Stromberg&amp;publication_year=2011&amp;journal=PLoS%20Genet&amp;volume=7&amp;pages=e1002236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21876680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sudmant</div> <div class="given-names">P.H.</div></div>, <div class="name string-name"><div class="surname">Rausch</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Gardner</div> <div class="given-names">E.J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">An integrated map of structural variation in 2, 504 human genomes</div>, <div class="source">Nature</div> , <div class="volume">526</div>, <div class="fpage">75</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20map%20of%20structural%20variation%20in%202%2C%20504%20human%20genomes&amp;author=P.H.%20Sudmant&amp;author=T.%20Rausch&amp;author=E.J.%20Gardner&amp;publication_year=2015&amp;journal=Nature&amp;volume=526&amp;pages=75-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature15394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature15394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature15394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26432246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hughes</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Pyntikova</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Chimpanzee and human Y chromosomes are remarkably divergent in structure and gene content</div>, <div class="source">Nature</div> , <div class="volume">463</div>, <div class="fpage">536</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chimpanzee%20and%20human%20Y%20chromosomes%20are%20remarkably%20divergent%20in%20structure%20and%20gene%20content&amp;author=J.F.%20Hughes&amp;author=H.%20Skaletsky&amp;author=T.%20Pyntikova&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=536-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08700" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08700" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08700"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20072128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harrow</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Frankish</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gonzalez</div> <div class="given-names">J.M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">GENCODE: the reference human genome annotation for The ENCODE Project</div>, <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">1760</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GENCODE%3A%20the%20reference%20human%20genome%20annotation%20for%20The%20ENCODE%20Project&amp;author=J.%20Harrow&amp;author=A.%20Frankish&amp;author=J.M.%20Gonzalez&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=1760-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.135350.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.135350.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.135350.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pruitt</div> <div class="given-names">K.D.</div></div>, <div class="name string-name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Maglott</div> <div class="given-names">D.R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">D61</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequences%20(RefSeq)%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K.D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.R.%20Maglott&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D61-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Luttig</div> <div class="given-names">C.T.</div></div>, <div class="name string-name"><div class="surname">Larkins</div> <div class="given-names">C.E.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">An initial map of insertion and deletion (INDEL) variation in the human genome</div>, <div class="source">Genome Res</div> ., <div class="volume">16</div>, <div class="fpage">1182</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20initial%20map%20of%20insertion%20and%20deletion%20(INDEL)%20variation%20in%20the%20human%20genome&amp;author=R.E.%20Mills&amp;author=C.T.%20Luttig&amp;author=C.E.%20Larkins&amp;publication_year=2006&amp;journal=Genome%20Res&amp;volume=16&amp;pages=1182-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4565806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4565806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4565806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16902084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cost</div> <div class="given-names">G.J.</div></div>, <div class="name string-name"><div class="surname">Boeke</div> <div class="given-names">J.D.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Targeting of human retrotransposon integration is directed by the specificity of the L1 endonuclease for regions of unusual DNA structure</div>, <div class="source">Biochemistry</div> , <div class="volume">37</div>, <div class="fpage">18081</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeting%20of%20human%20retrotransposon%20integration%20is%20directed%20by%20the%20specificity%20of%20the%20L1%20endonuclease%20for%20regions%20of%20unusual%20DNA%20structure&amp;author=G.J.%20Cost&amp;author=J.D.%20Boeke&amp;publication_year=1998&amp;journal=Biochemistry&amp;volume=37&amp;pages=18081-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi981858s" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi981858s" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi981858s"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9922177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Sequence patterns indicate an enzymatic involvement in integration of mammalian retroposons</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">94</div>, <div class="fpage">1872</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20patterns%20indicate%20an%20enzymatic%20involvement%20in%20integration%20of%20mammalian%20retroposons&amp;author=J.%20Jurka&amp;publication_year=1997&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=94&amp;pages=1872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.5.1872" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.5.1872" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.5.1872"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9050872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bushman</div> <div class="given-names">F.D.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Targeting survival: integration site selection by retroviruses and LTR-retrotransposons</div>, <div class="source">Cell</div> , <div class="volume">115</div>, <div class="fpage">135</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeting%20survival%3A%20integration%20site%20selection%20by%20retroviruses%20and%20LTR-retrotransposons&amp;author=F.D.%20Bushman&amp;publication_year=2003&amp;journal=Cell&amp;volume=115&amp;pages=135-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(03)00760-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(03)00760-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(03)00760-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14567911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abecasis</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Altshuler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Auton</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">A map of human genome variation from population-scale sequencing</div>, <div class="source">Nature</div> , <div class="volume">467</div>, <div class="fpage">1061</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&amp;author=G.R.%20Abecasis&amp;author=D.%20Altshuler&amp;author=A.%20Auton&amp;publication_year=2010&amp;journal=Nature&amp;volume=467&amp;pages=1061-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20981092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="collab">International Human Genome Sequencing, C</div>. <div class="year">2004</div>, 
<div class="article-title">Finishing the euchromatic sequence of the human genome</div>, <div class="source">Nature</div> , <div class="volume">431</div>, <div class="fpage">931</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15496913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.I.</div></div>, <div class="name string-name"><div class="surname">Ju</div> <div class="given-names">Y.S.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">A highly annotated whole-genome sequence of a Korean individual</div>, <div class="source">Nature</div> , <div class="volume">460</div>, <div class="fpage">1011</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20highly%20annotated%20whole-genome%20sequence%20of%20a%20Korean%20individual&amp;author=J.I.%20Kim&amp;author=Y.S.%20Ju&amp;author=H.%20Park&amp;publication_year=2009&amp;journal=Nature&amp;volume=460&amp;pages=1011-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19587683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barbulescu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Seaman</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Deinard</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Kidd</div> <div class="given-names">K.K.</div></div>, <div class="name string-name"><div class="surname">Lenz</div> <div class="given-names">J.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Many human endogenous retrovirus K (HERV-K) proviruses are unique to humans</div>, <div class="source">Curr. Biol</div> ., <div class="volume">9</div>, <div class="fpage">861</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Many%20human%20endogenous%20retrovirus%20K%20(HERV-K)%20proviruses%20are%20unique%20to%20humans&amp;author=M.%20Barbulescu&amp;author=G.%20Turner&amp;author=M.I.%20Seaman&amp;author=A.S.%20Deinard&amp;author=K.K.%20Kidd&amp;author=J.%20Lenz&amp;publication_year=1999&amp;journal=Curr.%20Biol&amp;volume=9&amp;pages=861-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(99)80390-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(99)80390-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(99)80390-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10469592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bennett</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Keller</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Active Alu retrotransposons in the human genome</div>, <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">1875</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Active%20Alu%20retrotransposons%20in%20the%20human%20genome&amp;author=E.A.%20Bennett&amp;author=H.%20Keller&amp;author=R.E.%20Mills&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=1875-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.081737.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.081737.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.081737.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18836035" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schwartz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Smit</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">Human-mouse alignments with BLASTZ</div>, <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">103</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human-mouse%20alignments%20with%20BLASTZ&amp;author=S.%20Schwartz&amp;author=W.J.%20Kent&amp;author=A.%20Smit&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=103-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.809403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.809403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.809403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12529312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kazazian</div> <div class="given-names">H.H.</div>Jr</div></div> <div class="year">2014</div>, 
<div class="article-title">Processed pseudogene insertions in somatic cells</div>, <div class="source">Mob. DNA</div> , <div class="volume">5</div>, <div class="fpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Processed%20pseudogene%20insertions%20in%20somatic%20cells&amp;author=H.H.%20Kazazian&amp;publication_year=2014&amp;journal=Mob.%20DNA&amp;volume=5&amp;pages=20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1759-8753-5-20" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1759-8753-5-20" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1759-8753-5-20"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25184004" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Casola</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Betran</div> <div class="given-names">E.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">The genomic impact of gene retrocopies: what have we learned from comparative genomics, population genomics, and transcriptomic analyses?</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">9</div>, <div class="fpage">1351</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20impact%20of%20gene%20retrocopies%3A%20what%20have%20we%20learned%20from%20comparative%20genomics%2C%20population%20genomics%2C%20and%20transcriptomic%20analyses%3F&amp;author=C.%20Casola&amp;author=E.%20Betran&amp;publication_year=2017&amp;journal=Genome%20Biol.%20Evol&amp;volume=9&amp;pages=1351-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evx081" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevx081" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevx081"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28605529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ewing</div> <div class="given-names">A.D.</div></div>, <div class="name string-name"><div class="surname">Ballinger</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Earl</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Retrotransposition of gene transcripts leads to structural variation in mammalian genomes</div>, <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposition%20of%20gene%20transcripts%20leads%20to%20structural%20variation%20in%20mammalian%20genomes&amp;author=A.D.%20Ewing&amp;author=T.J.%20Ballinger&amp;author=D.%20Earl&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-3-r22" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-3-r22" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-3-r22"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23497673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kabza</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kubiak</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Danek</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Inter-population differences in retrogene loss and expression in humans</div>, <div class="source">PLoS Genet</div> ., <div class="volume">11</div>, <div class="fpage">e1005579</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inter-population%20differences%20in%20retrogene%20loss%20and%20expression%20in%20humans&amp;author=M.%20Kabza&amp;author=M.R.%20Kubiak&amp;author=A.%20Danek&amp;publication_year=2015&amp;journal=PLoS%20Genet&amp;volume=11&amp;pages=e1005579" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26474060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Konkel</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Identification and characterization of novel polymorphic LINE-1 insertions through comparison of two human genome sequence assemblies</div>, <div class="source">Gene</div> , <div class="volume">390</div>, <div class="fpage">28</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20novel%20polymorphic%20LINE-1%20insertions%20through%20comparison%20of%20two%20human%20genome%20sequence%20assemblies&amp;author=M.K.%20Konkel&amp;author=J.%20Wang&amp;author=P.%20Liang&amp;author=M.A.%20Batzer&amp;publication_year=2007&amp;journal=Gene&amp;volume=390&amp;pages=28-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.07.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.07.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.07.040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17034961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cordaux</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Different evolutionary fates of recently integrated human and chimpanzee LINE-1 retrotransposons</div>, <div class="source">Gene</div> , <div class="volume">390</div>, <div class="fpage">18</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20evolutionary%20fates%20of%20recently%20integrated%20human%20and%20chimpanzee%20LINE-1%20retrotransposons&amp;author=J.%20Lee&amp;author=R.%20Cordaux&amp;author=K.%20Han&amp;publication_year=2007&amp;journal=Gene&amp;volume=390&amp;pages=18-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.08.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.08.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.08.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17055192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wildschutte</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">Z.H.</div></div>, <div class="name string-name"><div class="surname">Montesion</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Subramanian</div> <div class="given-names">R.P.</div></div>, <div class="name string-name"><div class="surname">Kidd</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Coffin</div> <div class="given-names">J.M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Discovery of unfixed endogenous retrovirus insertions in diverse human populations</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">113</div>, <div class="fpage">E2326</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20unfixed%20endogenous%20retrovirus%20insertions%20in%20diverse%20human%20populations&amp;author=J.H.%20Wildschutte&amp;author=Z.H.%20Williams&amp;author=M.%20Montesion&amp;author=R.P.%20Subramanian&amp;author=J.M.%20Kidd&amp;author=J.M.%20Coffin&amp;publication_year=2016&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=113&amp;pages=E2326-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1602336113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1602336113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1602336113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27001843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iskow</div> <div class="given-names">R.C.</div></div>, <div class="name string-name"><div class="surname">McCabe</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Natural mutagenesis of human genomes by endogenous retrotransposons</div>, <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">1253</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20mutagenesis%20of%20human%20genomes%20by%20endogenous%20retrotransposons&amp;author=R.C.%20Iskow&amp;author=M.T.%20McCabe&amp;author=R.E.%20Mills&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=1253-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.05.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.05.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.05.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20603005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Witherspoon</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xing</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Mobile element scanning (ME-Scan) identifies thousands of novel Alu insertions in diverse human populations</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">1170</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20element%20scanning%20(ME-Scan)%20identifies%20thousands%20of%20novel%20Alu%20insertions%20in%20diverse%20human%20populations&amp;author=D.J.%20Witherspoon&amp;author=Y.%20Zhang&amp;author=J.%20Xing&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1170-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.148973.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.148973.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.148973.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23599355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abrusan</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Krambeck</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Junier</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Giordano</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Warburton</div> <div class="given-names">P.E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Biased distributions and decay of long interspersed nuclear elements in the chicken genome</div>, <div class="source">Genetics</div> , <div class="volume">178</div>, <div class="fpage">573</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20distributions%20and%20decay%20of%20long%20interspersed%20nuclear%20elements%20in%20the%20chicken%20genome&amp;author=G.%20Abrusan&amp;author=H.J.%20Krambeck&amp;author=T.%20Junier&amp;author=J.%20Giordano&amp;author=P.E.%20Warburton&amp;publication_year=2008&amp;journal=Genetics&amp;volume=178&amp;pages=573-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.061861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.061861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.061861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17947446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boissinot</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Entezam</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Furano</div> <div class="given-names">A.V.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Selection against deleterious LINE-1-containing loci in the human lineage</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">18</div>, <div class="fpage">926</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20against%20deleterious%20LINE-1-containing%20loci%20in%20the%20human%20lineage&amp;author=S.%20Boissinot&amp;author=A.%20Entezam&amp;author=A.V.%20Furano&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol&amp;volume=18&amp;pages=926-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003893" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003893" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003893"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11371580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Griffin</div> <div class="given-names">D.K.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Is the Y chromosome disappearing? Both sides of the argument</div>, <div class="source">Chromosome Res</div> ., <div class="volume">20</div>, <div class="fpage">35</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Is%20the%20Y%20chromosome%20disappearing%3F%20Both%20sides%20of%20the%20argument&amp;author=D.K.%20Griffin&amp;publication_year=2012&amp;journal=Chromosome%20Res&amp;volume=20&amp;pages=35-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-011-9252-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-011-9252-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-011-9252-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22083302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cherry</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Ball</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Weng</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">1997</div>, 
<div class="article-title">Genetic and physical maps of Saccharomyces cerevisiae</div>, <div class="source">Nature</div> , <div class="volume">387</div>, <div class="fpage">67</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20physical%20maps%20of%20Saccharomyces%20cerevisiae&amp;author=J.M.%20Cherry&amp;author=C.%20Ball&amp;author=S.%20Weng&amp;publication_year=1997&amp;journal=Nature&amp;volume=387&amp;pages=67-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/387067a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F387067a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F387067a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9169866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Primate segmental duplications: crucibles of evolution, diversity and disease</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">7</div>, <div class="fpage">552</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primate%20segmental%20duplications%3A%20crucibles%20of%20evolution%2C%20diversity%20and%20disease&amp;author=J.A.%20Bailey&amp;author=E.E.%20Eichler&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet&amp;volume=7&amp;pages=552-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16770338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zahn</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kaplan</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Fischer</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Expansion of a novel endogenous retrovirus throughout the pericentromeres of modern humans</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expansion%20of%20a%20novel%20endogenous%20retrovirus%20throughout%20the%20pericentromeres%20of%20modern%20humans&amp;author=J.%20Zahn&amp;author=M.H.%20Kaplan&amp;author=S.%20Fischer&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0641-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0641-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0641-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25886262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Prufer</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Racimo</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Patterson</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The complete genome sequence of a Neanderthal from the Altai Mountains</div>, <div class="source">Nature</div> , <div class="volume">505</div>, <div class="fpage">43</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20a%20Neanderthal%20from%20the%20Altai%20Mountains&amp;author=K.%20Prufer&amp;author=F.%20Racimo&amp;author=N.%20Patterson&amp;publication_year=2014&amp;journal=Nature&amp;volume=505&amp;pages=43-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12886" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12886" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12886"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24352235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Hajdinjak</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Moldovan</div> <div class="given-names">O.T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">An early modern human from Romania with a recent Neanderthal ancestor</div>, <div class="source">Nature</div> , <div class="volume">524</div>, <div class="fpage">216</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20early%20modern%20human%20from%20Romania%20with%20a%20recent%20Neanderthal%20ancestor&amp;author=Q.%20Fu&amp;author=M.%20Hajdinjak&amp;author=O.T.%20Moldovan&amp;publication_year=2015&amp;journal=Nature&amp;volume=524&amp;pages=216-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature14558" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature14558" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature14558"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26098372" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Disotell</div> <div class="given-names">T.R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Archaic human genomics</div>, <div class="source">Am. J. Phys. Anthropol</div> ., <div class="volume">149(Suppl 55)</div>, <div class="fpage">24</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Archaic%20human%20genomics&amp;author=T.R.%20Disotell&amp;publication_year=2012&amp;journal=Am.%20J.%20Phys.%20Anthropol&amp;volume=149(Suppl 55)&amp;pages=24-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ajpa.22159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fajpa.22159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fajpa.22159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23124308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy022-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy022-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ahmed</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Study of modern human evolution via comparative analysis with the Neanderthal Genome</div>, <div class="source">Genomics Inform</div> ., <div class="volume">11</div>, <div class="fpage">230</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Study%20of%20modern%20human%20evolution%20via%20comparative%20analysis%20with%20the%20Neanderthal%20Genome&amp;author=M.%20Ahmed&amp;author=P.%20Liang&amp;publication_year=2013&amp;journal=Genomics%20Inform&amp;volume=11&amp;pages=230-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5808/GI.2013.11.4.230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5808%2FGI.2013.11.4.230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5808%2FGI.2013.11.4.230"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24465235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-18743796" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mikita Suyama</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mikita+Suyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suyama M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suyama M&quot;">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Mikita Suyama</a>
                            </span>
                        <br/>
                    </div>
            </div>
        </div>

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5056130&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy022/1/dsy022_supplementary_tables_figures.pdf?Expires=2147483647&amp;Signature=XnrhljxPtfWhR8J3WwVveiHSrxRLjWGAwWqGxdQnH7zqQinn59B6clVn1ypIwOBZeZ8oK6lZmCocwXWwHZGatSQ1-HNcYXt1zngLd-AnSHzR66hh12k8KGrGL8CRluad69AkK~G0onlpPc767WgR-3A6a~iFnCvduftnPkMvlCGN1yp52YfmdXMa4aY8HhRCQIhPMNfuC9J7HCRDV3LRnk~NTbMSMo~NxD8r4Tr4QdeGloxaouJuK-H4X9Yt7xXyHvTN2S7FOcKxNdX4JaOYSANnTLfqDLrgnA~Dw8RBqADSmY7PKphgN~3LEp1g7y8XxucZsCIubTeijzSXeV09rA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">561</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5056130"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f5%2f521%2f5056130">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F5%2F521%2F5056130" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5056130'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f521%2f5056130">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

