

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/3/123/464521","@type":"ScholarlyArticle","name":"Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method","datePublished":"2008-05-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2008-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn010","keywords":["full-length cDNA","expression profile","transcriptional start site","alternative splicing","antisense transcript"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/3/123/464521","author":[{"name":"Oshikawa, Mio","affiliation":"1  Department of Rehabilitation Engineering ,  Research Institute, National Rehabilitation Center for Persons with Disabilities ,  4-1 Namiki, Tokorozawa, Saitama 359-8555 ,  Japan    2  Department of Biological Applied Chemistry, Graduate School of Engineering ,  Toyo University ,  Kujirai 2100, Kawagoe, Saitama 350-8585 ,  Japan","@type":"Person"},{"name":"Sugai, Yoshiko","affiliation":"1  Department of Rehabilitation Engineering ,  Research Institute, National Rehabilitation Center for Persons with Disabilities ,  4-1 Namiki, Tokorozawa, Saitama 359-8555 ,  Japan    2  Department of Biological Applied Chemistry, Graduate School of Engineering ,  Toyo University ,  Kujirai 2100, Kawagoe, Saitama 350-8585 ,  Japan","@type":"Person"},{"name":"Usami, Ron","affiliation":"2  Department of Biological Applied Chemistry, Graduate School of Engineering ,  Toyo University ,  Kujirai 2100, Kawagoe, Saitama 350-8585 ,  Japan","@type":"Person"},{"name":"Ohtoko, Kuniyo","affiliation":"1  Department of Rehabilitation Engineering ,  Research Institute, National Rehabilitation Center for Persons with Disabilities ,  4-1 Namiki, Tokorozawa, Saitama 359-8555 ,  Japan","@type":"Person"},{"name":"Toyama, Shigeru","affiliation":"1  Department of Rehabilitation Engineering ,  Research Institute, National Rehabilitation Center for Persons with Disabilities ,  4-1 Namiki, Tokorozawa, Saitama 359-8555 ,  Japan","@type":"Person"},{"name":"Kato, Seishi","affiliation":"1  Department of Rehabilitation Engineering ,  Research Institute, National Rehabilitation Center for Persons with Disabilities ,  4-1 Namiki, Tokorozawa, Saitama 359-8555 ,  Japan","@type":"Person"}],"description":"Abstract.  Recently, we have developed a vector-capping method for constructing a full-length cDNA library. In the present study, we performed in-depth analysis","pageStart":"123","pageEnd":"136","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/3/1/m_cover.gif?Expires=2147483647&Signature=Wofi61EvCcdWxJvtysqPZG0U~KWWJOmThdMZDP-AglxAwDtnmCnY2MpChppyQDDpv6NdMaeLAZ~nLGaX7cJuWBVCGvKy3i2prPhQHBCH-vHV3CnTYh7LKJ6XOvu-5EP8bX5lf-70Wga4tfffe84AGB6pX1KlziPK7gdEs5QnVkZBbwLfiRPt-I1COdPZHhK91Imi3IuGiPl9ftt~EYfFKFyDk2nT8pASRTr325Ma~vy8IX6K~kWiKpA09pXB8RYEn5ztSYMyc20h7L3oQyBZ3j8qfpO0VRwhgradKgQz7QarN5Dl5qIio9OjPV0kSCdBe3E3RBSZsEr31pWrM0u5RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/3/1/m_cover.gif?Expires=2147483647&Signature=Wofi61EvCcdWxJvtysqPZG0U~KWWJOmThdMZDP-AglxAwDtnmCnY2MpChppyQDDpv6NdMaeLAZ~nLGaX7cJuWBVCGvKy3i2prPhQHBCH-vHV3CnTYh7LKJ6XOvu-5EP8bX5lf-70Wga4tfffe84AGB6pX1KlziPK7gdEs5QnVkZBbwLfiRPt-I1COdPZHhK91Imi3IuGiPl9ftt~EYfFKFyDk2nT8pASRTr325Ma~vy8IX6K~kWiKpA09pXB8RYEn5ztSYMyc20h7L3oQyBZ3j8qfpO0VRwhgradKgQz7QarN5Dl5qIio9OjPV0kSCdBe3E3RBSZsEr31pWrM0u5RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method" />
    <meta property="og:description" content="Abstract.  Recently, we have developed a vector-capping method for constructing a full-length cDNA library. In the present study, we performed in-depth analysis" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/3/123/464521" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-05-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Oshikawa, Mio" /><meta name="citation_author_institution" content="Department of Rehabilitation Engineering, Research Institute, National Rehabilitation Center for Persons with Disabilities, 4-1 Namiki, Tokorozawa, Saitama 359-8555, Japan" /><meta name="citation_author_institution" content="Department of Biological Applied Chemistry, Graduate School of Engineering, Toyo University, Kujirai 2100, Kawagoe, Saitama 350-8585, Japan" /><meta name="citation_author" content="Sugai, Yoshiko" /><meta name="citation_author_institution" content="Department of Rehabilitation Engineering, Research Institute, National Rehabilitation Center for Persons with Disabilities, 4-1 Namiki, Tokorozawa, Saitama 359-8555, Japan" /><meta name="citation_author_institution" content="Department of Biological Applied Chemistry, Graduate School of Engineering, Toyo University, Kujirai 2100, Kawagoe, Saitama 350-8585, Japan" /><meta name="citation_author" content="Usami, Ron" /><meta name="citation_author_institution" content="Department of Biological Applied Chemistry, Graduate School of Engineering, Toyo University, Kujirai 2100, Kawagoe, Saitama 350-8585, Japan" /><meta name="citation_author" content="Ohtoko, Kuniyo" /><meta name="citation_author_institution" content="Department of Rehabilitation Engineering, Research Institute, National Rehabilitation Center for Persons with Disabilities, 4-1 Namiki, Tokorozawa, Saitama 359-8555, Japan" /><meta name="citation_author" content="Toyama, Shigeru" /><meta name="citation_author_institution" content="Department of Rehabilitation Engineering, Research Institute, National Rehabilitation Center for Persons with Disabilities, 4-1 Namiki, Tokorozawa, Saitama 359-8555, Japan" /><meta name="citation_author" content="Kato, Seishi" /><meta name="citation_author_institution" content="Department of Rehabilitation Engineering, Research Institute, National Rehabilitation Center for Persons with Disabilities, 4-1 Namiki, Tokorozawa, Saitama 359-8555, Japan" /><meta name="citation_title" content="Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method" /><meta name="citation_firstpage" content="123" /><meta name="citation_lastpage" content="136" /><meta name="citation_doi" content="10.1093/dnares/dsn010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2008/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=The sequence of the human genome; Science;  citation_year=2001;  citation_volume=291;  citation_pages=1304-1351; " /><meta name="citation_reference" content="citation_title=Finishing the euchromatic sequence of the human genome; citation_author=International Human Genome Sequencing Consortium; Nature;  citation_year=2004;  citation_volume=431;  citation_pages=931-945; " /><meta name="citation_reference" content="citation_title=Construction of a human full-length cDNA bank; Gene;  citation_year=1994;  citation_volume=150;  citation_pages=243-250; " /><meta name="citation_reference" content="citation_title=High-efficiency full-length cDNA cloning by biotinylated CAP trapper; Genomics;  citation_year=1996;  citation_volume=37;  citation_pages=327-336; " /><meta name="citation_reference" content="citation_title=Construction and characterization of a full length-enriched and a 5'-end-enriched cDNA library; Gene;  citation_year=1997;  citation_volume=200;  citation_pages=149-156; " /><meta name="citation_reference" content="citation_title=Reverse transcriptase template switching: a SMART approach for full-length cDNA library construction; Biotechniques;  citation_year=2001;  citation_volume=30;  citation_pages=892-897; " /><meta name="citation_reference" content="citation_title=Quantitative monitoring of gene expression patterns with a complementary DNA microarray; Science;  citation_year=1995;  citation_volume=270;  citation_pages=467-470; " /><meta name="citation_reference" content="citation_title=Expression monitoring by hybridization to high-density oligonucleotide arrays; Nat. Biotechnol.;  citation_year=1996;  citation_volume=14;  citation_pages=1675-1680; " /><meta name="citation_reference" content="citation_title=Large scale cDNA sequencing for analysis of quantitative and qualitative aspects of gene expression; Nat. Genet.;  citation_year=1992;  citation_volume=2;  citation_pages=173-179; " /><meta name="citation_reference" content="citation_title=Serial analysis of gene expression; Science;  citation_year=1995;  citation_volume=270;  citation_pages=484-487; " /><meta name="citation_reference" content="citation_title=Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=15776-15781; " /><meta name="citation_reference" content="citation_title=5'-end SAGE for the analysis of transcriptional start sites; Nat. Biotechnol.;  citation_year=2004;  citation_volume=22;  citation_pages=1146-1149; " /><meta name="citation_reference" content="citation_title=Tag-based approaches for transcriptome research and genome annotation; Nat. Methods;  citation_year=2005;  citation_volume=2;  citation_pages=495-502; " /><meta name="citation_reference" content="citation_title=Complete sequencing and characterization of 21,243 full-length human cDNAs; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=40-45; " /><meta name="citation_reference" content="citation_title=Integrative annotation of 21,037 human genes validated by full-length cDNA clones; PLoS Biol.;  citation_year=2004;  citation_volume=2;  citation_pages=e162" /><meta name="citation_reference" content="citation_title=The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC); Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2121-2127; " /><meta name="citation_reference" content="citation_title=BodyMap: a collection of 3' ESTs for analysis of human gene expression information; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1817-1827; " /><meta name="citation_reference" content="citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=207-210; " /><meta name="citation_reference" content="citation_title=ArrayExpress—a public repository for microarray gene expression data at the EBI; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=68-71; " /><meta name="citation_reference" content="citation_title=Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=55-65; " /><meta name="citation_reference" content="citation_title=Diverse transcriptional initiation revealed by fine, large-scale mapping of mRNA start sites; EMBO Rep;  citation_year=2001;  citation_volume=2;  citation_pages=388-393; " /><meta name="citation_reference" content="citation_title=Genome-wide detection of alternative splicing in expressed sequences of human genes; Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=2850-2859; " /><meta name="citation_reference" content="citation_title=Patterns of variant polyadenylation signal usage in human genes; Genome Res;  citation_year=2000;  citation_volume=10;  citation_pages=1001-1010; " /><meta name="citation_reference" content="citation_title=Widespread occurrence of antisense transcription in the human genome; Nat. Biotechnol;  citation_year=2003;  citation_volume=21;  citation_pages=379-386; " /><meta name="citation_reference" content="citation_title=Over 20% of human transcripts might form sense-antisense pairs; Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=4812-4820; " /><meta name="citation_reference" content="citation_title=Pieces of the puzzle: expressed sequence tags and the catalog of human genes; J. Mol. Med;  citation_year=1997;  citation_volume=75;  citation_pages=694-698; " /><meta name="citation_reference" content="citation_title=Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22; Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=331-342; " /><meta name="citation_reference" content="citation_title=Global identification of human transcribed sequences with genome tiling arrays; Science;  citation_year=2004;  citation_volume=306;  citation_pages=2242-2246; " /><meta name="citation_reference" content="citation_title=Vector-capping: a simple method for preparing a high-quality full-length cDNA library; DNA Res;  citation_year=2005;  citation_volume=12;  citation_pages=53-62; " /><meta name="citation_reference" content="citation_title=High-efficiency cloning of full-length cDNA; Mol. Cell. Biol.;  citation_year=1982;  citation_volume=2;  citation_pages=161-170; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-410; " /><meta name="citation_reference" content="citation_title=Entrez: molecular biology database and retrieval system; Methods Enzymol;  citation_year=1996;  citation_volume=266;  citation_pages=141-162; " /><meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D61-D65; " /><meta name="citation_reference" content="citation_title=Species richness estimation; citation_text=Species Estimation and Applications. Encyclopedia of Statistical Sciences; citation_publisher=New York, Wiley;  citation_year=2005;  citation_volume=Vol. 12;  citation_pages=7907-7916; " /><meta name="citation_reference" content="citation_title=Estimating the number of classes via sample coverage; J. Am. Stat. Assoc;  citation_year=1992;  citation_volume=87;  citation_pages=210-217; " /><meta name="citation_reference" content="citation_title=Predicting the number of new species in further taxonomic sampling; Ecology;  citation_year=2003;  citation_volume=84;  citation_pages=798-804; " /><meta name="citation_reference" content="citation_title=Alpha B-crystallin is a small heat shock protein; Proc. Natl. Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=3652-3656; " /><meta name="citation_reference" content="citation_title=Hypertonic stress induces alpha B-crystallin expression; Exp. Eye Res;  citation_year=1992;  citation_volume=54;  citation_pages=461-470; " /><meta name="citation_reference" content="citation_title=Coordinated induction of two unrelated glucose-regulated protein genes by a calcium ionophore: human BiP/GRP78 and GAPDH; Biochem. Biophys. Res. Commun.;  citation_year=1990;  citation_volume=171;  citation_pages=431-438; " /><meta name="citation_reference" content="citation_title=An insulin response element in the glyceraldehyde-3-phosphate dehydrogenase gene binds a nuclear protein induced by insulin in cultured cells and by nutritional manipulations in vivo; Proc. Natl. Acad. Sci. USA;  citation_year=1990;  citation_volume=87;  citation_pages=5273-5277; " /><meta name="citation_reference" content="citation_title=Regulation of endothelial cell glyceraldehyde-3-phosphate dehydrogenase expression by hypoxia; J. Biol. Chem;  citation_year=1994;  citation_volume=269;  citation_pages=24446-24453; " /><meta name="citation_reference" content="citation_title=EST analysis of mouse retina and RPE/choroid cDNA libraries; Mol. Vis;  citation_year=2004;  citation_volume=10;  citation_pages=439-444; " /><meta name="citation_reference" content="citation_title=DBTSS: DataBase of human transcription start sites, progress report 2006; Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=D86-D89; " /><meta name="citation_reference" content="citation_title=Giantin, a novel conserved Golgi membrane protein containing a cytoplasmic domain of at least 350 kDa; Mol. Biol. Cell;  citation_year=1993;  citation_volume=4;  citation_pages=679-693; " /><meta name="citation_reference" content="citation_title=An abundance of bidirectional promoters in the human genome; Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=62-66; " /><meta name="citation_reference" content="citation_title=Transcript-specific decapping and regulated stability by the human Dcp2 decapping protein; Mol. Cell. Biol;  citation_year=2008;  citation_volume=28;  citation_pages=939-948; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis of the retina; Genome Biol;  citation_year=2002;  citation_volume=3;  citation_pages=reviews1022.1-1022.4; " /><meta name="citation_reference" content="citation_title=Gene expression profile of cultured adult compared to immortalized human RPE; Mol. Vis.;  citation_year=2006;  citation_volume=12;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Genome-wide detection of tissue-specific alternative splicing in the human transcriptome; Nucleic Acids Res.;  citation_volume=30;  citation_pages=3754-3766; " /><meta name="citation_reference" content="citation_title=Emerging technologies in DNA sequencing; Genome Res;  citation_year=2005;  citation_volume=15;  citation_pages=1767-1776; " /><meta name="citation_reference" content="citation_title=A large-scale full-length cDNA analysis to explore the budding yeast transcriptome; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=17846-17851; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/3/123/1260101/dsn010.pdf" /><meta name="description" content="Abstract.  Recently, we have developed a vector-capping method for constructing a full-length cDNA library. In the present study, we performed in-depth analysis" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/3/123/464521" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/3/123/464521" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/3/123/464521");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn010';
    </script>
<title>
	Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./464521" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaJzPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpa2F3YSwgTWlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBSZWhhYmlsaXRhdGlvbiBFbmdpbmVlcmluZywgUmVzZWFyY2ggSW5zdGl0dXRlLCBOYXRpb25hbCBSZWhhYmlsaXRhdGlvbiBDZW50ZXIgZm9yIFBlcnNvbnMgd2l0aCBEaXNhYmlsaXRpZXMsIDQtMSBOYW1pa2ksIFRva29yb3phd2EsIFNhaXRhbWEgMzU5LTg1NTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIEFwcGxpZWQgQ2hlbWlzdHJ5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRW5naW5lZXJpbmcsIFRveW8gVW5pdmVyc2l0eSwgS3VqaXJhaSAyMTAwLCBLYXdhZ29lLCBTYWl0YW1hIDM1MC04NTg1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1Z2FpLCBZb3NoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBSZWhhYmlsaXRhdGlvbiBFbmdpbmVlcmluZywgUmVzZWFyY2ggSW5zdGl0dXRlLCBOYXRpb25hbCBSZWhhYmlsaXRhdGlvbiBDZW50ZXIgZm9yIFBlcnNvbnMgd2l0aCBEaXNhYmlsaXRpZXMsIDQtMSBOYW1pa2ksIFRva29yb3phd2EsIFNhaXRhbWEgMzU5LTg1NTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIEFwcGxpZWQgQ2hlbWlzdHJ5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRW5naW5lZXJpbmcsIFRveW8gVW5pdmVyc2l0eSwgS3VqaXJhaSAyMTAwLCBLYXdhZ29lLCBTYWl0YW1hIDM1MC04NTg1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVzYW1pLCBSb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgQXBwbGllZCBDaGVtaXN0cnksIEdyYWR1YXRlIFNjaG9vbCBvZiBFbmdpbmVlcmluZywgVG95byBVbml2ZXJzaXR5LCBLdWppcmFpIDIxMDAsIEthd2Fnb2UsIFNhaXRhbWEgMzUwLTg1ODUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2h0b2tvLCBLdW5peW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFJlaGFiaWxpdGF0aW9uIEVuZ2luZWVyaW5nLCBSZXNlYXJjaCBJbnN0aXR1dGUsIE5hdGlvbmFsIFJlaGFiaWxpdGF0aW9uIENlbnRlciBmb3IgUGVyc29ucyB3aXRoIERpc2FiaWxpdGllcywgNC0xIE5hbWlraSwgVG9rb3JvemF3YSwgU2FpdGFtYSAzNTktODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb3lhbWEsIFNoaWdlcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFJlaGFiaWxpdGF0aW9uIEVuZ2luZWVyaW5nLCBSZXNlYXJjaCBJbnN0aXR1dGUsIE5hdGlvbmFsIFJlaGFiaWxpdGF0aW9uIENlbnRlciBmb3IgUGVyc29ucyB3aXRoIERpc2FiaWxpdGllcywgNC0xIE5hbWlraSwgVG9rb3JvemF3YSwgU2FpdGFtYSAzNTktODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRvLCBTZWlzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFJlaGFiaWxpdGF0aW9uIEVuZ2luZWVyaW5nLCBSZXNlYXJjaCBJbnN0aXR1dGUsIE5hdGlvbmFsIFJlaGFiaWxpdGF0aW9uIENlbnRlciBmb3IgUGVyc29ucyB3aXRoIERpc2FiaWxpdGllcywgNC0xIE5hbWlraSwgVG9rb3JvemF3YSwgU2FpdGFtYSAzNTktODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkZpbmUgRXhwcmVzc2lvbiBQcm9maWxpbmcgb2YgRnVsbC1sZW5ndGggVHJhbnNjcmlwdHMgdXNpbmcgYSBTaXplLXVuYmlhc2VkIGNETkEgTGlicmFyeSBQcmVwYXJlZCB3aXRoIHRoZSBWZWN0b3ItY2FwcGluZyBNZXRob2QiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbjAxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA4LzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5pdGlhbCBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiB0aGUgaHVtYW4gZ2Vub21lOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDA5OyAgY2l0YXRpb25fcGFnZXM9ODYwLTkyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIG9mIHRoZSBodW1hbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjkxOyAgY2l0YXRpb25fcGFnZXM9MTMwNC0xMzUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5pc2hpbmcgdGhlIGV1Y2hyb21hdGljIHNlcXVlbmNlIG9mIHRoZSBodW1hbiBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIEh1bWFuIEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW07IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzE7ICBjaXRhdGlvbl9wYWdlcz05MzEtOTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBodW1hbiBmdWxsLWxlbmd0aCBjRE5BIGJhbms7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9MjQzLTI1MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1lZmZpY2llbmN5IGZ1bGwtbGVuZ3RoIGNETkEgY2xvbmluZyBieSBiaW90aW55bGF0ZWQgQ0FQIHRyYXBwZXI7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MzI3LTMzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgZnVsbCBsZW5ndGgtZW5yaWNoZWQgYW5kIGEgNSctZW5kLWVucmljaGVkIGNETkEgbGlicmFyeTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDA7ICBjaXRhdGlvbl9wYWdlcz0xNDktMTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXZlcnNlIHRyYW5zY3JpcHRhc2UgdGVtcGxhdGUgc3dpdGNoaW5nOiBhIFNNQVJUIGFwcHJvYWNoIGZvciBmdWxsLWxlbmd0aCBjRE5BIGxpYnJhcnkgY29uc3RydWN0aW9uOyBCaW90ZWNobmlxdWVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9ODkyLTg5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIG1vbml0b3Jpbmcgb2YgZ2VuZSBleHByZXNzaW9uIHBhdHRlcm5zIHdpdGggYSBjb21wbGVtZW50YXJ5IEROQSBtaWNyb2FycmF5OyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3MDsgIGNpdGF0aW9uX3BhZ2VzPTQ2Ny00NzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gbW9uaXRvcmluZyBieSBoeWJyaWRpemF0aW9uIHRvIGhpZ2gtZGVuc2l0eSBvbGlnb251Y2xlb3RpZGUgYXJyYXlzOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTY3NS0xNjgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZSBzY2FsZSBjRE5BIHNlcXVlbmNpbmcgZm9yIGFuYWx5c2lzIG9mIHF1YW50aXRhdGl2ZSBhbmQgcXVhbGl0YXRpdmUgYXNwZWN0cyBvZiBnZW5lIGV4cHJlc3Npb247IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xNzMtMTc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXJpYWwgYW5hbHlzaXMgb2YgZ2VuZSBleHByZXNzaW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3MDsgIGNpdGF0aW9uX3BhZ2VzPTQ4NC00ODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcCBhbmFseXNpcyBnZW5lIGV4cHJlc3Npb24gZm9yIGhpZ2gtdGhyb3VnaHB1dCBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uYWwgc3RhcnRpbmcgcG9pbnQgYW5kIGlkZW50aWZpY2F0aW9uIG9mIHByb21vdGVyIHVzYWdlOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0xNTc3Ni0xNTc4MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9NSctZW5kIFNBR0UgZm9yIHRoZSBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uYWwgc3RhcnQgc2l0ZXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xMTQ2LTExNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhZy1iYXNlZCBhcHByb2FjaGVzIGZvciB0cmFuc2NyaXB0b21lIHJlc2VhcmNoIGFuZCBnZW5vbWUgYW5ub3RhdGlvbjsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00OTUtNTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBzZXF1ZW5jaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIDIxLDI0MyBmdWxsLWxlbmd0aCBodW1hbiBjRE5BczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz00MC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgYW5ub3RhdGlvbiBvZiAyMSwwMzcgaHVtYW4gZ2VuZXMgdmFsaWRhdGVkIGJ5IGZ1bGwtbGVuZ3RoIGNETkEgY2xvbmVzOyBQTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMTYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN0YXR1cywgcXVhbGl0eSwgYW5kIGV4cGFuc2lvbiBvZiB0aGUgTklIIGZ1bGwtbGVuZ3RoIGNETkEgcHJvamVjdDogdGhlIE1hbW1hbGlhbiBHZW5lIENvbGxlY3Rpb24gKE1HQyk7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjEyMS0yMTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Cb2R5TWFwOiBhIGNvbGxlY3Rpb24gb2YgMycgRVNUcyBmb3IgYW5hbHlzaXMgb2YgaHVtYW4gZ2VuZSBleHByZXNzaW9uIGluZm9ybWF0aW9uOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE4MTctMTgyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBFeHByZXNzaW9uIE9tbmlidXM6IE5DQkkgZ2VuZSBleHByZXNzaW9uIGFuZCBoeWJyaWRpemF0aW9uIGFycmF5IGRhdGEgcmVwb3NpdG9yeTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjA3LTIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJyYXlFeHByZXNz4oCUYSBwdWJsaWMgcmVwb3NpdG9yeSBmb3IgbWljcm9hcnJheSBnZW5lIGV4cHJlc3Npb24gZGF0YSBhdCB0aGUgRUJJOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz02OC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaWZpY2F0aW9uIG9mIHRyYW5zY3JpcHRpb25hbCBtb2R1bGF0aW9uOiBsYXJnZS1zY2FsZSBpZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBwdXRhdGl2ZSBhbHRlcm5hdGl2ZSBwcm9tb3RlcnMgb2YgaHVtYW4gZ2VuZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2UgdHJhbnNjcmlwdGlvbmFsIGluaXRpYXRpb24gcmV2ZWFsZWQgYnkgZmluZSwgbGFyZ2Utc2NhbGUgbWFwcGluZyBvZiBtUk5BIHN0YXJ0IHNpdGVzOyBFTUJPIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9Mzg4LTM5MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZGV0ZWN0aW9uIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nIGluIGV4cHJlc3NlZCBzZXF1ZW5jZXMgb2YgaHVtYW4gZ2VuZXM7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9Mjg1MC0yODU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXR0ZXJucyBvZiB2YXJpYW50IHBvbHlhZGVueWxhdGlvbiBzaWduYWwgdXNhZ2UgaW4gaHVtYW4gZ2VuZXM7IEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMDAxLTEwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdpZGVzcHJlYWQgb2NjdXJyZW5jZSBvZiBhbnRpc2Vuc2UgdHJhbnNjcmlwdGlvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyBOYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNzktMzg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdmVyIDIwJSBvZiBodW1hbiB0cmFuc2NyaXB0cyBtaWdodCBmb3JtIHNlbnNlLWFudGlzZW5zZSBwYWlyczsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz00ODEyLTQ4MjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBpZWNlcyBvZiB0aGUgcHV6emxlOiBleHByZXNzZWQgc2VxdWVuY2UgdGFncyBhbmQgdGhlIGNhdGFsb2cgb2YgaHVtYW4gZ2VuZXM7IEouIE1vbC4gTWVkOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9Njk0LTY5ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgUk5BcyBpZGVudGlmaWVkIGZyb20gYW4gaW4tZGVwdGggYW5hbHlzaXMgb2YgdGhlIHRyYW5zY3JpcHRvbWUgb2YgaHVtYW4gY2hyb21vc29tZXMgMjEgYW5kIDIyOyBHZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzMxLTM0MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGlkZW50aWZpY2F0aW9uIG9mIGh1bWFuIHRyYW5zY3JpYmVkIHNlcXVlbmNlcyB3aXRoIGdlbm9tZSB0aWxpbmcgYXJyYXlzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMwNjsgIGNpdGF0aW9uX3BhZ2VzPTIyNDItMjI0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVjdG9yLWNhcHBpbmc6IGEgc2ltcGxlIG1ldGhvZCBmb3IgcHJlcGFyaW5nIGEgaGlnaC1xdWFsaXR5IGZ1bGwtbGVuZ3RoIGNETkEgbGlicmFyeTsgRE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWVmZmljaWVuY3kgY2xvbmluZyBvZiBmdWxsLWxlbmd0aCBjRE5BOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xNjEtMTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTQxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW50cmV6OiBtb2xlY3VsYXIgYmlvbG9neSBkYXRhYmFzZSBhbmQgcmV0cmlldmFsIHN5c3RlbTsgTWV0aG9kcyBFbnp5bW9sOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI2NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MS0xNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5DQkkgcmVmZXJlbmNlIHNlcXVlbmNlcyAoUmVmU2VxKTogYSBjdXJhdGVkIG5vbi1yZWR1bmRhbnQgc2VxdWVuY2UgZGF0YWJhc2Ugb2YgZ2Vub21lcywgdHJhbnNjcmlwdHMgYW5kIHByb3RlaW5zOyBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQ2MS1ENjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpZXMgcmljaG5lc3MgZXN0aW1hdGlvbjsgY2l0YXRpb25fdGV4dD1TcGVjaWVzIEVzdGltYXRpb24gYW5kIEFwcGxpY2F0aW9ucy4gRW5jeWNsb3BlZGlhIG9mIFN0YXRpc3RpY2FsIFNjaWVuY2VzOyBjaXRhdGlvbl9wdWJsaXNoZXI9TmV3IFlvcmssIFdpbGV5OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPVZvbC4gMTI7ICBjaXRhdGlvbl9wYWdlcz03OTA3LTc5MTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgdGhlIG51bWJlciBvZiBjbGFzc2VzIHZpYSBzYW1wbGUgY292ZXJhZ2U7IEouIEFtLiBTdGF0LiBBc3NvYzsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04NzsgIGNpdGF0aW9uX3BhZ2VzPTIxMC0yMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3RpbmcgdGhlIG51bWJlciBvZiBuZXcgc3BlY2llcyBpbiBmdXJ0aGVyIHRheG9ub21pYyBzYW1wbGluZzsgRWNvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTc5OC04MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFscGhhIEItY3J5c3RhbGxpbiBpcyBhIHNtYWxsIGhlYXQgc2hvY2sgcHJvdGVpbjsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz0zNjUyLTM2NTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5cGVydG9uaWMgc3RyZXNzIGluZHVjZXMgYWxwaGEgQi1jcnlzdGFsbGluIGV4cHJlc3Npb247IEV4cC4gRXllIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTQ2MS00NzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvb3JkaW5hdGVkIGluZHVjdGlvbiBvZiB0d28gdW5yZWxhdGVkIGdsdWNvc2UtcmVndWxhdGVkIHByb3RlaW4gZ2VuZXMgYnkgYSBjYWxjaXVtIGlvbm9waG9yZTogaHVtYW4gQmlQL0dSUDc4IGFuZCBHQVBESDsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTQzMS00Mzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGluc3VsaW4gcmVzcG9uc2UgZWxlbWVudCBpbiB0aGUgZ2x5Y2VyYWxkZWh5ZGUtMy1waG9zcGhhdGUgZGVoeWRyb2dlbmFzZSBnZW5lIGJpbmRzIGEgbnVjbGVhciBwcm90ZWluIGluZHVjZWQgYnkgaW5zdWxpbiBpbiBjdWx0dXJlZCBjZWxscyBhbmQgYnkgbnV0cml0aW9uYWwgbWFuaXB1bGF0aW9ucyBpbiB2aXZvOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT04NzsgIGNpdGF0aW9uX3BhZ2VzPTUyNzMtNTI3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBlbmRvdGhlbGlhbCBjZWxsIGdseWNlcmFsZGVoeWRlLTMtcGhvc3BoYXRlIGRlaHlkcm9nZW5hc2UgZXhwcmVzc2lvbiBieSBoeXBveGlhOyBKLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTI2OTsgIGNpdGF0aW9uX3BhZ2VzPTI0NDQ2LTI0NDUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FU1QgYW5hbHlzaXMgb2YgbW91c2UgcmV0aW5hIGFuZCBSUEUvY2hvcm9pZCBjRE5BIGxpYnJhcmllczsgTW9sLiBWaXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz00MzktNDQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EQlRTUzogRGF0YUJhc2Ugb2YgaHVtYW4gdHJhbnNjcmlwdGlvbiBzdGFydCBzaXRlcywgcHJvZ3Jlc3MgcmVwb3J0IDIwMDY7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9RDg2LUQ4OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2lhbnRpbiwgYSBub3ZlbCBjb25zZXJ2ZWQgR29sZ2kgbWVtYnJhbmUgcHJvdGVpbiBjb250YWluaW5nIGEgY3l0b3BsYXNtaWMgZG9tYWluIG9mIGF0IGxlYXN0IDM1MCBrRGE7IE1vbC4gQmlvbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9Njc5LTY5MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYWJ1bmRhbmNlIG9mIGJpZGlyZWN0aW9uYWwgcHJvbW90ZXJzIGluIHRoZSBodW1hbiBnZW5vbWU7IEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz02Mi02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdC1zcGVjaWZpYyBkZWNhcHBpbmcgYW5kIHJlZ3VsYXRlZCBzdGFiaWxpdHkgYnkgdGhlIGh1bWFuIERjcDIgZGVjYXBwaW5nIHByb3RlaW47IE1vbC4gQ2VsbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTkzOS05NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgdGhlIHJldGluYTsgR2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPXJldmlld3MxMDIyLjEtMTAyMi40OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gcHJvZmlsZSBvZiBjdWx0dXJlZCBhZHVsdCBjb21wYXJlZCB0byBpbW1vcnRhbGl6ZWQgaHVtYW4gUlBFOyBNb2wuIFZpcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBkZXRlY3Rpb24gb2YgdGlzc3VlLXNwZWNpZmljIGFsdGVybmF0aXZlIHNwbGljaW5nIGluIHRoZSBodW1hbiB0cmFuc2NyaXB0b21lOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zNzU0LTM3NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVtZXJnaW5nIHRlY2hub2xvZ2llcyBpbiBETkEgc2VxdWVuY2luZzsgR2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NjctMTc3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBsYXJnZS1zY2FsZSBmdWxsLWxlbmd0aCBjRE5BIGFuYWx5c2lzIHRvIGV4cGxvcmUgdGhlIGJ1ZGRpbmcgeWVhc3QgdHJhbnNjcmlwdG9tZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTc4NDYtMTc4NTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNS8zLzEyMy8xMjYwMTAxL2RzbjAxMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUmVjZW50bHksIHdlIGhhdmUgZGV2ZWxvcGVkIGEgdmVjdG9yLWNhcHBpbmcgbWV0aG9kIGZvciBjb25zdHJ1Y3RpbmcgYSBmdWxsLWxlbmd0aCBjRE5BIGxpYnJhcnkuIEluIHRoZSBwcmVzZW50IHN0dWR5LCB3ZSBwZXJmb3JtZWQgaW4tZGVwdGggYW5hbHlzaXMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ2NDUyMSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQVv4xMvY2wVYeS3ZNOtEAZDNKoxQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f123%2f464521" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f123%2f464521" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18181" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/3/1/m_cover.gif?Expires=2147483647&amp;Signature=Wofi61EvCcdWxJvtysqPZG0U~KWWJOmThdMZDP-AglxAwDtnmCnY2MpChppyQDDpv6NdMaeLAZ~nLGaX7cJuWBVCGvKy3i2prPhQHBCH-vHV3CnTYh7LKJ6XOvu-5EP8bX5lf-70Wga4tfffe84AGB6pX1KlziPK7gdEs5QnVkZBbwLfiRPt-I1COdPZHhK91Imi3IuGiPl9ftt~EYfFKFyDk2nT8pASRTr325Ma~vy8IX6K~kWiKpA09pXB8RYEn5ztSYMyc20h7L3oQyBZ3j8qfpO0VRwhgradKgQz7QarN5Dl5qIio9OjPV0kSCdBe3E3RBSZsEr31pWrM0u5RQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/3">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6976785">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976785">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976788">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976788">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976796">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976796">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976815">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976815">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976854">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976854">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976864">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976864">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976866">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976866">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976868">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976868">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6976870">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6976870">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/3/115/464114">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/3/137/464067">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mio Oshikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Rehabilitation Engineering</div>, <div class="insititution">Research Institute, National Rehabilitation Center for Persons with Disabilities</div>, <div class="addr-line">4-1 Namiki, Tokorozawa, Saitama 359-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Applied Chemistry, Graduate School of Engineering</div>, <div class="insititution">Toyo University</div>, <div class="addr-line">Kujirai 2100, Kawagoe, Saitama 350-8585</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mio+Oshikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshikawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshikawa M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mio Oshikawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiko Sugai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Rehabilitation Engineering</div>, <div class="insititution">Research Institute, National Rehabilitation Center for Persons with Disabilities</div>, <div class="addr-line">4-1 Namiki, Tokorozawa, Saitama 359-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Applied Chemistry, Graduate School of Engineering</div>, <div class="insititution">Toyo University</div>, <div class="addr-line">Kujirai 2100, Kawagoe, Saitama 350-8585</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiko+Sugai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugai Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugai Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiko Sugai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ron Usami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Applied Chemistry, Graduate School of Engineering</div>, <div class="insititution">Toyo University</div>, <div class="addr-line">Kujirai 2100, Kawagoe, Saitama 350-8585</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ron+Usami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Usami R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Usami R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ron Usami</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kuniyo Ohtoko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Rehabilitation Engineering</div>, <div class="insititution">Research Institute, National Rehabilitation Center for Persons with Disabilities</div>, <div class="addr-line">4-1 Namiki, Tokorozawa, Saitama 359-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kuniyo+Ohtoko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtoko K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtoko K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kuniyo Ohtoko</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeru Toyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Rehabilitation Engineering</div>, <div class="insititution">Research Institute, National Rehabilitation Center for Persons with Disabilities</div>, <div class="addr-line">4-1 Namiki, Tokorozawa, Saitama 359-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeru+Toyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyama S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigeru Toyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seishi Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Rehabilitation Engineering</div>, <div class="insititution">Research Institute, National Rehabilitation Center for Persons with Disabilities</div>, <div class="addr-line">4-1 Namiki, Tokorozawa, Saitama 359-8555</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 4-2995-3100. Fax. <div class="fax">+81 4-2995-3132</div>. E-mail: <a href="mailto:seishi@rehab.go.jp" target="_blank">seishi@rehab.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seishi+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seishi Kato</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Minoru Ko</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 3, 1 June 2008, Pages 123–136, <a href='https://doi.org/10.1093/dnares/dsn010'>https://doi.org/10.1093/dnares/dsn010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 May 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 December 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 April 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="464521" href="/dnaresearch/article-pdf/15/3/123/1260101/dsn010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mio Oshikawa, Yoshiko Sugai, Ron Usami, Kuniyo Ohtoko, Shigeru Toyama, Seishi Kato;  Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method, <em>DNA Research</em>, Volume 15, Issue 3, 1 June 2008, Pages 123–136, <a href='https://doi.org/10.1093/dnares/dsn010'>https://doi.org/10.1093/dnares/dsn010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/464521?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/464521?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/464521?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/464521?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/464521?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/464521?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/464521?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Fine%20Expression%20Profiling%20of%20Full-length%20Transcripts%20using%20a%20Size-unbiased%20cDNA%20Library%20Prepared%20with%20the%20Vector-capping%20Method&amp;publicationDate=2008-05-16&amp;volumeNum=15&amp;issueNum=3&amp;author=Oshikawa%2C%20Mio%3B%20Sugai%2C%20Yoshiko&amp;startPage=123&amp;endPage=136&amp;contentId=10.1093%2Fdnares%2Fdsn010&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f123%2f464521" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6976785 id="6976785" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Recently, we have developed a vector-capping method for constructing a full-length cDNA library. In the present study, we performed in-depth analysis of the vector-capped cDNA library prepared from a single type of cell. As a result of single-pass sequencing analysis of 24 000 clones randomly isolated from the unamplified library, we identified 19 951 full-length cDNA clones whose intactness was confirmed by the presence of an additional G at their 5' end. The full-length cDNA content was &gt;95%. Mapping these sequences to the human genome, we identified 4513 transcriptional units that include 36 antisense transcripts against known genes. Comparison of the frequencies of abundant clones showed that the expression profiles of different libraries, including the distribution of transcriptional start sites (TSSs), were reproducible. The analysis of long-sized cDNAs showed that this library contained many cDNAs with a long-sized insert up to 11 199 bp of golgin B, including multiple slicing variants for filamin A and filamin B. These results suggest that the size-unbiased full-length cDNA library constructed using the vector-capping method will be an ideal resource for fine expression profiling of transcriptional variants with alternative TSSs and alternative splicing.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;full-length cDNA&quot;">full-length cDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;expression profile&quot;">expression profile</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptional start site&quot;">transcriptional start site</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;antisense transcript&quot;">antisense transcript</a></div>                            <h2 scrollto-destination=6976788 id="6976788" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The Human Genome Project disclosed sequence information on an entire set of elements that construct and regulate each human cell, tissue, and organ.<sup><a reveal-id="DSN010C1  DSN010C3" data-open="DSN010C1  DSN010C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Our next challenge is to decode the instructions encoded by three billion bases on the genome and to understand the molecular basis of human biology and pathology. In the last decade, post-sequencing projects have taken two approaches: (i) to collect an entire set of full-length transcripts (transcriptome) and in turn map them on the genome to identify their promoter region, and (ii) to compare the expression profiles of genes in different types of cells or tissues at different biological conditions. To achieve these purposes, many technologies have been developed: practical methods for synthesizing full-length cDNA,<sup><a reveal-id="DSN010C4  DSN010C7" data-open="DSN010C4  DSN010C7" class="link link-ref link-reveal xref-bibr">4–7</a></sup> hybridization-based microarray,<sup><a reveal-id="DSN010C8" data-open="DSN010C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSN010C9" data-open="DSN010C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and tag-sequencing-based technologies<sup><a reveal-id="DSN010C10  DSN010C14" data-open="DSN010C10  DSN010C14" class="link link-ref link-reveal xref-bibr">10–14</a></sup> for analyzing an expression profile.</p><p>Large-scale analyses using these technologies have produced an enormous amount of data on human full-length transcripts,<sup><a reveal-id="DSN010C15  DSN010C17" data-open="DSN010C15  DSN010C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup> as well as on expression profiles,<sup><a reveal-id="DSN010C18  DSN010C20" data-open="DSN010C18  DSN010C20" class="link link-ref link-reveal xref-bibr">18–20</a></sup> and have revealed the following facts. (i) Multiple transcripts of different sizes are generated from the same gene locus by alternative promoter usage,<sup><a reveal-id="DSN010C21" data-open="DSN010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> diverse transcriptional initiation,<sup><a reveal-id="DSN010C22" data-open="DSN010C22" class="link link-ref link-reveal xref-bibr">22</a></sup> alternative splicing,<sup><a reveal-id="DSN010C23" data-open="DSN010C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and alternative polyadenylation.<sup><a reveal-id="DSN010C24" data-open="DSN010C24" class="link link-ref link-reveal xref-bibr">24</a></sup> (ii) There are an unexpectedly large amount of sense–antisense pairs of transcripts.<sup><a reveal-id="DSN010C25" data-open="DSN010C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSN010C26" data-open="DSN010C26" class="link link-ref link-reveal xref-bibr">26</a></sup> (iii) Many non-coding RNAs are transcribed.<sup><a reveal-id="DSN010C15" data-open="DSN010C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSN010C16" data-open="DSN010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> (iv) A considerable amount of transcripts estimated by expressed sequence tags (EST) of the UniGene database<sup><a reveal-id="DSN010C27" data-open="DSN010C27" class="link link-ref link-reveal xref-bibr">27</a></sup> or by genome tiling array<sup><a reveal-id="DSN010C28" data-open="DSN010C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSN010C29" data-open="DSN010C29" class="link link-ref link-reveal xref-bibr">29</a></sup> still remain to be annotated because of their low abundance.</p><p>These data sets were accumulated using diverse cell types or tissues, because the first phase of the project was aimed at comprehensive gene collection. Hereafter, a second phase of investigation requires in-depth analysis of transcripts in a single type of cell population to elucidate the intracellular transcriptional regulatory network in a given cell. However, the full-length cDNA libraries constructed using the conventional methods do not meet the requirements for in-depth analyses because of the low complexity of the constituents. Furthermore, the presence of transcriptional variants raises a serious problem regarding the use of the conventional method for expression profiling of genes. If multiple variants transcribed from the same gene locus have a different function, we need to determine the expression profile of each variant to elucidate their role in the regulation network. However, the conventional methods that measure the amount of the limited region of each transcript by counting the number of sequence tags or by quantifying the hybridization signal cannot distinguish these variants without the precedent information of the full sequence of all variants expressed in a given cell.</p><p>The two problems described above, the low complexity of the full-length cDNA library and the presence of multiple variants, can be solved by analyzing all variants that comprise a bias-free full-length cDNA library constructed from a single type of cell. Unfortunately, the conventional methods for synthesizing full-length cDNA were unsuitable for constructing a bias-free cDNA library that reflects an expression level of each transcript in the cell, because they have many processes during which the ratio of components may change. It is especially difficult to construct a bias-free library containing rare or long-sized full-length cDNA clones without changing their content.</p><p>Recently, we have developed the vector-capping (V-capping) method for synthesizing full-length cDNA.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> This method is expected to be suitable to construct a bias-free cDNA library, because it consists of only three steps: the first-strand cDNA synthesis using a vector primer, self-ligation of the cDNA-vector construct, and the replacement of mRNA by the second-strand cDNA. The previous paper showed that we were able to construct the cDNA libraries containing full-length cDNA clones of &gt;95% content without any selection procedure for full-length cDNAs. The further advantage of this library is that we can validate full-length cDNA by the presence of an additional G at its 5' end.</p><p>In this paper, we performed in-depth analysis of the cDNA libraries constructed using the V-capping method from the total RNA isolated from human retinal pigment epithelial cell line ARPE-19, and demonstrated that the constructed library was useful as a starting resource not only for the comprehensive collection of full-length cDNA clones, but also for fine expression profile analysis of transcripts expressed in a single type of cell, including variants generated by alternative promoter usage, alternative transcriptional initiation, alternative splicing, and alternative polyadenylation.</p>                            <h2 scrollto-destination=6976796 id="6976796" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6976797 id="6976797" class="section-title" data-legacy-id=s2a>2.1. Cell culture and RNA preparation</h3>
<p>Human retinal pigment epithelium (RPE) cell line ARPE-19 was obtained from American Type Culture Collection (Manassas, VA, USA). ARPE-19 cells were cultured in Dulbecco's modified eagle's medium: nutrient mixture F-12 (Invitrogen, Carlsbad, CA, USA) containing 10% fetal bovine serum. The cells were incubated for 4 days to confluence in a humidified atmosphere of 5% CO<sub>2</sub> and 95% air at 37°C. The cells were harvested by trypsinization. Total RNA was isolated using ISOGEN (NIPPON GENE, Tokyo, Japan).</p>                            <h3 scrollto-destination=6976800 id="6976800" class="section-title" data-legacy-id=s2b>2.2. Vector primer</h3>
<p>A plasmid vector pGCAP10 was constructed by substituting the cloning site <em>Eco</em>RI–<em>Afl</em>II–<em>Swa</em>I–<em>Kpn</em>I of pGCAP1<sup>30</sup> with <em>Swa</em>I–<em>Eco</em>RI–<em>Fse</em>I–<em>Eco</em>RV–<em>Kpn</em>I. The nucleotide sequence of pGCAP10 is available from GenBank/EMBL/DDBJ under accession no. AB371573. The plasmid pGCAP10 was digested with <em>Kpn</em>I and tailed with ∼60 nucleotides of dT using terminal deoxynucleotidyl transferase (Takara Bio, Ohtsu, Shiga, Japan) according to the Okayama and Berg method.<sup><a reveal-id="DSN010C31" data-open="DSN010C31" class="link link-ref link-reveal xref-bibr">31</a></sup> After digestion with <em>Eco</em>RV, the dT-tailed plasmid vector was purified on agarose gel and used as a vector primer. A cDNA insert can be cut out from the vector with two 8-nucleotide restriction enzymes, <em>Swa</em>I and <em>Not</em>I.</p>                            <h3 scrollto-destination=6976802 id="6976802" class="section-title" data-legacy-id=s2c>2.3. cDNA synthesis with the V-capping method</h3>
<p>Two libraries, Lib-1 (ARe) and Lib-2 (ARi), were constructed using the V-capping method.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Lib-1 has already been described in the previous paper.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Lib-2 was different from Lib-1 in terms of the source of total RNA, a pGCAP10-derived vector primer, and a modified protocol. The experimental conditions were the same as described in the previous paper. A mixture of 5 µg of total RNA and 0.15 µg of pGCAP10-derived vector primer was incubated at 65°C for 5 min. The first-strand cDNA was synthesized using SuperScript III<sup>TM</sup> reverse transcriptase (Invitrogen). The reaction mixture was incubated at 45°C for 3 h. After phenol/chloroform extraction followed by ethanol precipitation, the pellet was dissolved in water. The next step in the original protocol is self-ligation with T4 RNA ligase (Takara Bio). The present protocol includes an <em>Eco</em>RI digestion step before self-ligation. The <em>Eco</em>RI digestion was performed in 200 µL of a reaction mixture containing 50 mM Tris–HCl (pH 7.5), 10 mM MgCl<sub>2</sub>, 1 mM dithiothreitol, 100 mM NaCl, and 0.2 U/μL of <em>Eco</em>RI (Takara Bio). The reaction mixture was incubated at 37°C for 1.5 h. After phenol/chloroform extraction followed by ethanol precipitation, the pellet was dissolved in water. Self-ligation and second-strand synthesis were performed in the same way as in the previous paper.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p>                            <h3 scrollto-destination=6976805 id="6976805" class="section-title" data-legacy-id=s2d>2.4. Construction of cDNA library</h3>
<p>Transformation of <em>Escherichia coli</em> cells DH12S was performed using an electroporation method as previously described.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Transformants were plated on LB agar without amplification. Colonies grown on the plates were picked manually or using a Flexys Colony Picker (Genomic Solutions, Ann Arbor, MI, USA) and suspended in 96-well or 384-well plates. After incubation and the addition of 50% glycerol, the original plates were stored at −80°C.</p>                            <h3 scrollto-destination=6976807 id="6976807" class="section-title" data-legacy-id=s2e>2.5. Plasmid isolation and sequencing</h3>
<p>The isolated plasmid DNA or DNA amplified using the illustra TempliPhi<sup>TM</sup> DNA amplification kit (GE Healthcare, Uppsala, Sweden) was used as a template for sequencing. DNA sequencing from the 5' end of the cDNA insert was carried out with a capillary DNA sequencer (Applied Biosystems Inc., Foster City, CA, USA) using a BigDye<sup>TM</sup> Terminator Cycle sequencing FS Ready reaction kit. The full sequence of the cDNA insert was determined by a primer walking method.</p>                            <h3 scrollto-destination=6976809 id="6976809" class="section-title" data-legacy-id=s2f>2.6. BLAST search and annotation</h3>
<p>First, the 5'-end sequences were used to query our custom database for human full-length cDNA clones (Homo-Protein cDNA bank)<sup><a reveal-id="DSN010C4" data-open="DSN010C4" class="link link-ref link-reveal xref-bibr">4</a></sup> with a software GENETYXR-PDB (GENETYX Co., Tokyo, Japan). Most of the abundant genes, ribosomal RNAs, and mitochondria-derived sequences were identified by this search. Sequences not matching to entries in our custom database were used to query the NCBI Human Genome database (National Center for Biotechnology Information, Bethesda, MD, USA) with the BLAST algorithm.<sup><a reveal-id="DSN010C32" data-open="DSN010C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Each search was carried out manually, and the sequence alignment and map shown on the NCBI's Map Viewer were checked visually by us. Most sequences were mapped to the first exon of a known gene locus. If the query sequence was mapped to the upstream region of a known gene locus in the same direction, the sequence was assigned to that gene. Through the websites linked to the Map Viewer, including Entrez Gene<sup><a reveal-id="DSN010C33" data-open="DSN010C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and UniGene,<sup><a reveal-id="DSN010C27" data-open="DSN010C27" class="link link-ref link-reveal xref-bibr">27</a></sup> we retrieved information on gene name, gene symbol, gene ID, chromosomal location, and RefSeq<sup><a reveal-id="DSN010C34" data-open="DSN010C34" class="link link-ref link-reveal xref-bibr">34</a></sup> accession number. Sequences not mapped to the known gene locus were BLAST-searched against the NCBI database, including non-redundant nucleotide sequences and ESTs. EST sequences not included in Entrez Gene and the determined full sequences of long-sized cDNAs were deposited in GenBank/EMBL/DDBJ under accession numbers AB371430–AB371572 and AB371574–AB371588, respectively.</p>                            <h3 scrollto-destination=6976811 id="6976811" class="section-title" data-legacy-id=s2g>2.7. Estimation of the total number of genes composing libraries</h3>
<p>The total number of genes constituting the library was estimated according to two approaches used for species richness estimation: non-sampling-based extrapolation and statistical sampling approaches.<sup><a reveal-id="DSN010C35" data-open="DSN010C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The former was performed by curve fitting to a gene-accumulation curve using asymptotic models, including negative exponential models and hyperbolic models.<sup><a reveal-id="DSN010C35" data-open="DSN010C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The curve fitting was carried out using software KaleidaGraph (Synergy Software, Reading, PA, USA). The latter approach used an abundance-based coverage estimator model ACE-1, a modified ACE for highly heterogeneous communities.<sup><a reveal-id="DSN010C36" data-open="DSN010C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The calculation was done using the SPADE (Species Prediction and Diversity Estimation) algorithm.<sup><a reveal-id="DSN010C37" data-open="DSN010C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h3 scrollto-destination=6976813 id="6976813" class="section-title" data-legacy-id=s2h>2.8. Quantitative real-time PCR</h3>
<p>First-strand cDNA was synthesized with oligo(dT)<sub>30</sub> as a primer from 20 µg of total RNA using SuperScript III<sup>TM</sup> reverse transcriptase (Invitrogen), and then purified by a Wizard PCR Preps DNA Purification System (Promega, Madison, WI, USA). Real-time PCR was performed using TaqMan Universal Master Mix (Applied Biosystems) on an ABI PRISM 7000 Sequence Detection System (Applied Biosystems) according to the manufacturer's instructions. One microlitter of diluted cDNA, equivalent to 300 ng of the initial total RNA template, was used in each reaction. Probes and primers designed by TaqMan Gene Expression Assays (Applied Biosystems) were used for the assays of ACTB (Hs99999903_m1), CFL1 (Hs00830568_g1), FLNA (Hs99999905_m1), FLNB (Hs00181698_m1), GAPDH (Hs99999905_m1), GUK1 (Hs00176133_m1), MYH9 (Hs00159522_m1), and RAI14 (Hs00210238_m1). The expression level was calculated based on a standard curve prepared for each gene using a plasmid with each cDNA as a template.</p>                            <h2 scrollto-destination=6976815 id="6976815" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6976816 id="6976816" class="section-title" data-legacy-id=s3a>3.1. cDNA Library</h3>
<p>Two cDNA libraries, Lib-1 and Lib-2, were constructed using the V-capping method from the total RNA isolated from ARPE-19. The construction of Lib-1 and part of its analysis were described in a previous paper.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> A total of 10 176 clones from Lib-1 were randomly picked, cultured, and stored as a glycerol stock in 96-well plates. The clones were named ARe and ARf. Lib-2 was prepared from a different lot of total RNA using a slightly modified method that included an <em>Eco</em>RI digestion step after the first-strand cDNA synthesis. Transformation of the <em>E. coli</em> cells by cDNA vectors for Lib-2 was carried out in two batches at different times. The colonies picked from each batch were incubated and stored in 96-well plates (ARi, 6528 clones) and 384-well plates (ARiS, 76 800 clones), respectively. In this study, all of the ARe/ARf and ARi clones and part of ARiS (7296 clones) were analyzed by single-pass sequencing of the 5' end of cDNA.</p><p>Table <a reveal-id="DSN010TB1" data-open="DSN010TB1" class="link link-reveal link-table xref-fig">1</a> shows the contents of each library classified by single-pass sequencing analysis. More than 90% clones provided the high-quality sequence data necessary for sequence analysis. The unreadable sequence may result from (i) deletion of a sequencing primer site on the vector, (ii) mixing of different clones, or (iii) failure of template DNA preparation. Many cases were attributed to the first reason because they showed no sequencing signals and could not be cut by restriction enzymes adjacent to the upstream of the cDNA insert. Each library contained insert-free vectors (3.3–3.9% in content), which may result from uncut vectors escaping from removal during the vector primer preparation process.</p>                        <a id="6976819" scrollto-destination="6976819"></a>
<div content-id="DSN010TB1" class="table-modal"><div class="table-wrap"><div id="DSN010TB1" class="table-wrap-title" data-id="DSN010TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of single-pass sequencing analysis of libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library Name </th><th>Lib-1 </th><th colspan="2">Lib-2 </th><th> </th></tr></thead><tbody><tr><td>Clone Name </td><td>ARe/ARf </td><td>ARi </td><td>ARiS </td></tr><tr><td> Vector </td><td>pKA1U5 </td><td>pGCAP10 </td><td>pGCAP10 </td></tr><tr><td><em> Eco</em>RI cut </td><td>No </td><td>Yes </td><td>Yes </td></tr><tr><td>Total </td><td>10 176 </td><td>6528 </td><td>7296 </td></tr><tr><td> Unreadable </td><td>940 </td><td>340 </td><td>588 </td></tr><tr><td> Readable </td><td>9236 </td><td>6188 </td><td>6708 </td></tr><tr><td>  Insert-free vector </td><td>305 </td><td>244 </td><td>221 </td></tr><tr><td>  dT tail </td><td>177 </td><td>10 </td><td>9 </td></tr><tr><td>  Mitochondria </td><td>86 </td><td>65 </td><td>69 </td></tr><tr><td>  rRNA </td><td>3 </td><td>2 </td><td>4 </td></tr><tr><td>  cDNA insert </td><td>8665 </td><td>5867 </td><td>6405 </td></tr><tr><td>   Full-length </td><td>8275 </td><td>5586 </td><td>6090 </td></tr><tr><td>   Truncated </td><td>310 </td><td>243 </td><td>271 </td></tr><tr><td>   Poly(A) </td><td>80 </td><td>38 </td><td>44 </td></tr></tbody></table></div><div content-id="DSN010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library Name </th><th>Lib-1 </th><th colspan="2">Lib-2 </th><th> </th></tr></thead><tbody><tr><td>Clone Name </td><td>ARe/ARf </td><td>ARi </td><td>ARiS </td></tr><tr><td> Vector </td><td>pKA1U5 </td><td>pGCAP10 </td><td>pGCAP10 </td></tr><tr><td><em> Eco</em>RI cut </td><td>No </td><td>Yes </td><td>Yes </td></tr><tr><td>Total </td><td>10 176 </td><td>6528 </td><td>7296 </td></tr><tr><td> Unreadable </td><td>940 </td><td>340 </td><td>588 </td></tr><tr><td> Readable </td><td>9236 </td><td>6188 </td><td>6708 </td></tr><tr><td>  Insert-free vector </td><td>305 </td><td>244 </td><td>221 </td></tr><tr><td>  dT tail </td><td>177 </td><td>10 </td><td>9 </td></tr><tr><td>  Mitochondria </td><td>86 </td><td>65 </td><td>69 </td></tr><tr><td>  rRNA </td><td>3 </td><td>2 </td><td>4 </td></tr><tr><td>  cDNA insert </td><td>8665 </td><td>5867 </td><td>6405 </td></tr><tr><td>   Full-length </td><td>8275 </td><td>5586 </td><td>6090 </td></tr><tr><td>   Truncated </td><td>310 </td><td>243 </td><td>271 </td></tr><tr><td>   Poly(A) </td><td>80 </td><td>38 </td><td>44 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6976819" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN010TB1" class="table-wrap-title" data-id="DSN010TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of single-pass sequencing analysis of libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library Name </th><th>Lib-1 </th><th colspan="2">Lib-2 </th><th> </th></tr></thead><tbody><tr><td>Clone Name </td><td>ARe/ARf </td><td>ARi </td><td>ARiS </td></tr><tr><td> Vector </td><td>pKA1U5 </td><td>pGCAP10 </td><td>pGCAP10 </td></tr><tr><td><em> Eco</em>RI cut </td><td>No </td><td>Yes </td><td>Yes </td></tr><tr><td>Total </td><td>10 176 </td><td>6528 </td><td>7296 </td></tr><tr><td> Unreadable </td><td>940 </td><td>340 </td><td>588 </td></tr><tr><td> Readable </td><td>9236 </td><td>6188 </td><td>6708 </td></tr><tr><td>  Insert-free vector </td><td>305 </td><td>244 </td><td>221 </td></tr><tr><td>  dT tail </td><td>177 </td><td>10 </td><td>9 </td></tr><tr><td>  Mitochondria </td><td>86 </td><td>65 </td><td>69 </td></tr><tr><td>  rRNA </td><td>3 </td><td>2 </td><td>4 </td></tr><tr><td>  cDNA insert </td><td>8665 </td><td>5867 </td><td>6405 </td></tr><tr><td>   Full-length </td><td>8275 </td><td>5586 </td><td>6090 </td></tr><tr><td>   Truncated </td><td>310 </td><td>243 </td><td>271 </td></tr><tr><td>   Poly(A) </td><td>80 </td><td>38 </td><td>44 </td></tr></tbody></table></div><div content-id="DSN010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library Name </th><th>Lib-1 </th><th colspan="2">Lib-2 </th><th> </th></tr></thead><tbody><tr><td>Clone Name </td><td>ARe/ARf </td><td>ARi </td><td>ARiS </td></tr><tr><td> Vector </td><td>pKA1U5 </td><td>pGCAP10 </td><td>pGCAP10 </td></tr><tr><td><em> Eco</em>RI cut </td><td>No </td><td>Yes </td><td>Yes </td></tr><tr><td>Total </td><td>10 176 </td><td>6528 </td><td>7296 </td></tr><tr><td> Unreadable </td><td>940 </td><td>340 </td><td>588 </td></tr><tr><td> Readable </td><td>9236 </td><td>6188 </td><td>6708 </td></tr><tr><td>  Insert-free vector </td><td>305 </td><td>244 </td><td>221 </td></tr><tr><td>  dT tail </td><td>177 </td><td>10 </td><td>9 </td></tr><tr><td>  Mitochondria </td><td>86 </td><td>65 </td><td>69 </td></tr><tr><td>  rRNA </td><td>3 </td><td>2 </td><td>4 </td></tr><tr><td>  cDNA insert </td><td>8665 </td><td>5867 </td><td>6405 </td></tr><tr><td>   Full-length </td><td>8275 </td><td>5586 </td><td>6090 </td></tr><tr><td>   Truncated </td><td>310 </td><td>243 </td><td>271 </td></tr><tr><td>   Poly(A) </td><td>80 </td><td>38 </td><td>44 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6976819" target="_blank">View Large</a></div></div><p>Lib-1 contained clones carrying a dT tail at the 5' end (1.9% in content). Inspection of the downstream sequence of these clones showed that they lacked a poly(A) tail and contained the inversely inserted cDNA whose 5' end was joined to the <em>Kpn</em>I-cut end of the vector where the 3'-protruding bases were deleted. These clones may be generated by use of an aberrant vector primer that has a dT tail at the opposite end from the intended one, implying that the dT-tail addition to only one end of the vector plasmid occasionally occurred at the dT-tailing step with terminal deoxynucleotidyl transferase. Although the dT tail added to the opposite end should be removed by <em>Eco</em>RV digestion after the dT-tailing reaction, some must remain uncut. These clones starting with the dT tail could be removed by adding an <em>Eco</em>RI digestion step before self-ligation, as shown in the modified protocol. In fact, it worked so well that these artifacts drastically decreased to 0.16% in the ARi library and to 0.13% in the ARiS library.</p><p>The same mechanism can also explain the addition of a vector-derived sequence, ATCCTG in the case of using pKA1U5 as a vector primer, adjacent to the 5' end of ∼5% of cDNA clones isolated from Lib-1. In this case, the opposite end might not have been dT-tailed and in addition might have escaped from <em>Eco</em>RV digestion. Although the <em>Eco</em>RI digestion step in the modified protocol was expected to remove this kind of additional sequence, 2.8% of clones in Lib-2 still had a residual sequence (CGGCCGGCCGAT) derived from the vector sequence located between the <em>Eco</em>RI and <em>Eco</em>RV sites because of incomplete digestion with <em>Eco</em>RI.</p>                            <h3 scrollto-destination=6976822 id="6976822" class="section-title" data-legacy-id=s3b>3.2. Assessment of full-length cDNA</h3>
<p>The 5'-terminal sequences of cDNA inserts were used for the database search. Clones having only poly(A) or the sequence of several bases with poly(A) were classified into a truncated cDNA, because these clones might be derived from degraded mRNA. The sequence similarity was searched against our custom human full-length cDNA database, the NCBI RefSeq database, and the human genome database using the BLAST algorithm. The libraries contained cDNA clones for mitochondria genome-derived transcripts (∼1% of cDNA-carrying clones) and rRNA (&lt;0.06%). Except for mitochondrial clones, all sequences were able to map on the human genome.</p><p>Most of the full-length cDNA clones identified as a known gene have a transcriptional start site (TSS) near to that of RefSeq. However, some sequences did not match the 5'-terminal sequence of RefSeq, presumably because our cDNA had a longer 5' UTR than RefSeq or was transcribed by the usage of an alternative promoter. These sequences were mapped on the human genome and the location of the sequence was determined. If the query sequence was located near the upstream region of the first exon of RefSeq or could be linked to the RefSeq via other mRNA or EST sequences that partly shared the query sequence, the clone was assigned to the gene for the corresponding RefSeq even though there was no shared sequence between the query and RefSeq.</p><p>Consequently, 8275 clones from the ARe/ARf library, 5586 from ARi, and 6090 from ARiS were identified as a full-length cDNA. Most clones (93.6% for Lib-1, 92.3% for Lib-2) had an additional G or NG (N: T, TT, G, etc.) at their 5' end, which is a requisite for full-length cDNA starting from the cap site when using the V-capping method. The previous paper showed that some full-length cDNAs had no additional G.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Thus, if the cDNA not having a 5'-end G started at the upstream region of the first exon of the known gene, it was admitted to be a full-length cDNA for the corresponding gene. Although some cDNAs contained a repetitive sequence such as an Alu repeat, they were also precisely mapped on the genome. Most of the truncated cDNAs existed as a short form with a poly(A) tail. Even such short cDNA was assigned to full-length when it was mapped to the region that was not a known gene locus and had an additional G at the 5' end. Consequently, the content of the full-length clone in all clones carrying a cDNA insert [including only a poly(A) insert] was calculated to be 95.5% for ARe/ARf, 95.2% for ARi, and 95.1% for ARiS.</p><p>Out of 19 951 full-length cDNA clones, 1123 clones (5.6%) lacked an additional 5'-end G. These 5'-G-free genes were classified into two groups. One group (625 clones) consisted of clones starting with a nucleotide A. Another group (309 clones) was a 5'-terminal oligopyrimidine tract (5'-TOP) gene family that started with a pyrimidine-rich sequence, including predominantly ribosomal proteins. The genes that contained more than three 5'-G-free clones (G − ) with the same TSS were listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 1</a>. It should be noted that G-added clones (G + ) corresponding to each G− clone were obtained except for NDUFB11. The content of G− clones was 0.04–0.31 for 16 kinds of 5'-TOP genes. On the other hand, the G− content for 12 out of 15 A-starting genes was 0.38–1.0, higher than for 5'-TOP genes. Although we could not find any conserved sequence in the 5' end of the G-free A-starting genes, the following finding suggests that the 5'-end sequence affected the addition or elimination of a cap structure. We obtained 82 clones starting with 5'-ACCACGCACG… for MT2A, out of which 44 had an additional G and 38 were G-free as shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 1</a>. In addition, we obtained 43 clones starting with the fourth nucleotide A of the previous MT2A clone, i.e. 5'-ACGCACG…. Interestingly, all 43 clones had an additional G, suggesting that the presence of 5'-end three-nucleotide sequence resulted in the production of G-free clones.</p>                            <h3 scrollto-destination=6976827 id="6976827" class="section-title" data-legacy-id=s3c>3.3. Gene annotation</h3>
<p>Mapping a total of 19 951 full-length cDNA sequences to the genome revealed that they were classified into 4513 kinds of transcriptional units (i.e. genes), of which 4370 (96.8%) were included in Entrez Gene. All genes are listed in the order of GeneID in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 2</a>. The list contains the symbol, GeneID, and name, which were retrieved from Entrez Gene, and in addition the accession number of the RefSeq, mRNA size, chromosomal location, and number of clones obtained from each library. Of 4370 genes, 4271 had an accession number with prefix “NM_” that indicates mRNA. The remaining 99 genes (2.3%) were possible non-coding genes.</p>                        <a id="6976829" scrollto-destination="6976829"></a>
<div content-id="DSN010TB2" class="table-modal"><div class="table-wrap"><div id="DSN010TB2" class="table-wrap-title" data-id="DSN010TB2"><div class="label -label">Table 2</div><div class="caption"><p>Long-sized full-length cDNA clones with &gt;7 kb insert</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>HP ID </th><th>Symbol </th><th>Name </th><th>RefSeq </th><th>mRNA (bp) </th><th>Protein (aa) </th><th>Clone ID </th><th>Ac. No. </th><th>cDNA (bp) </th><th>Protein (aa) </th></tr></thead><tbody><tr><td>1 </td><td>HP08164 </td><td>GOLGB1 </td><td>Golgi autoantigen, golgin subfamily b, macrogolgin (with transmembrane signal) 1 </td><td>NM_004487.3 </td><td>11 185 </td><td>3259 </td><td>ARiS161G17 </td><td>AB371588 </td><td>11 198 </td><td>3269<sup>a</sup> </td></tr><tr><td>2 </td><td>HP07459 </td><td>N4BP2 </td><td>Nedd4 binding protein 2 </td><td>NM_018177.2 </td><td>6616 </td><td>1770 </td><td>ARiS023B20 </td><td>AB371584 </td><td>9736 </td><td>1690<sup>b</sup> </td></tr><tr><td>3 </td><td>HP08032 </td><td>ACACA </td><td>Acetyl-Coenzyme A carboxylase alpha </td><td>NM_198836.1 </td><td>9585 </td><td>2346 </td><td>ARiS088K16 </td><td>AB371587 </td><td>9534 </td><td>2346 </td></tr><tr><td>4 </td><td>HP04958 </td><td>FLNB </td><td>Filamin B, beta (actin binding protein 278) </td><td>NM_001457.2 </td><td>9463 </td><td>2602 </td><td>ARe23D04 </td><td>AB191258 </td><td>9405 </td><td>2591 </td></tr><tr><td>5 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe77D06 </td><td>AB371580 </td><td>8059 </td><td>2633 </td></tr><tr><td>6 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe89D09 </td><td>AB371581 </td><td>9366 </td><td>2578 </td></tr><tr><td>7 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARi12F08 </td><td>AB371582 </td><td>7973 </td><td>2602 </td></tr><tr><td>8 </td><td>HP07616 </td><td>FLNC </td><td>Filamin C, gamma (actin binding protein 280) </td><td>NM_001458.3 </td><td>9146 </td><td>2725 </td><td>ARi57A02 </td><td>AB371585 </td><td>9156 </td><td>2725 </td></tr><tr><td>9 </td><td>HP07744 </td><td>SPTBN1 </td><td>Spectrin, beta, non-erythrocytic 1 </td><td>NM_003128.2 </td><td>10 238 </td><td>2364 </td><td>ARiS088A21 </td><td>AB371586 </td><td>8443 </td><td>2364 </td></tr><tr><td>10 </td><td>HP00079 </td><td>FLNA </td><td>Filamin A, alpha (actin binding protein 280) </td><td>NM_001456.2 </td><td>8278 </td><td>2639 </td><td>ARe06F05 </td><td>AB191259 </td><td>8212 </td><td>2612 </td></tr><tr><td>11 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARe27E03 </td><td>AB191260 </td><td>8241 </td><td>2620 </td></tr><tr><td>12 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi13C12 </td><td>AB371574 </td><td>8242 </td><td>2620 </td></tr><tr><td>13 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi37B09 </td><td>AB371575 </td><td>8212 </td><td>2612 </td></tr><tr><td>14 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi47G07 </td><td>AB371576 </td><td>8243 </td><td>2620 </td></tr><tr><td>15 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi50A09 </td><td>AB371577 </td><td>7321 </td><td>2315 </td></tr><tr><td>16 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi66B08 </td><td>AB371578 </td><td>8212 </td><td>2612 </td></tr><tr><td>17 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARiS088J13 </td><td>AB371579 </td><td>8214 </td><td>2612 </td></tr><tr><td>18 </td><td>HP06504 </td><td>COL5A1 </td><td>Collagen, type V, alpha 1 </td><td>NM_000093.3 </td><td>8439 </td><td>1838 </td><td>ARe79B07 </td><td>AB371586 </td><td>8139 </td><td>1838 </td></tr><tr><td>19 </td><td>HP07532 </td><td>GCN1L1 </td><td>GCN1 general control of amino-acid synthesis 1-like 1 (yeast) </td><td>NM_006836.1 </td><td>8699 </td><td>2671 </td><td>ARi43H04 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>20 </td><td>HP06485 </td><td>PTPRF </td><td>Protein tyrosine phosphatase, receptor type, F </td><td>NM_002840.2 </td><td>7718 </td><td>1897 </td><td>ARe76H09 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>21 </td><td>HP05449 </td><td>SPTAN1 </td><td>Spectrin, alpha, non-erythrocytic 1 (alpha-fodrin) </td><td>NM_003127.1 </td><td>7787 </td><td>2472 </td><td>ARe45C02 </td><td>AB191262 </td><td>7791 </td><td>2452 </td></tr><tr><td>22 </td><td>HP00124 </td><td>FN1 </td><td>Fibronectin 1 </td><td>NM_212476.1 </td><td>8272 </td><td>2296 </td><td>ARe05G09 </td><td>AB191261 </td><td>7753 </td><td>2265 </td></tr><tr><td>23 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi53A03 </td><td>— </td><td>7.7 k </td><td> </td></tr><tr><td>24 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS087I09 </td><td>— </td><td>7.9 k </td><td> </td></tr><tr><td>25 </td><td>HP06896 </td><td>PCM1 </td><td>Pericentriolar material 1 </td><td>NM_006197.3 </td><td>8788 </td><td>2024 </td><td>ARi07G10 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>26 </td><td>HP04727 </td><td>GLIS3 </td><td>GLIS family zinc finger 3 </td><td>NM_001042413.1 </td><td>7656 </td><td>930 </td><td>ARe06E05 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>27 </td><td>HP04890 </td><td>BAT2 </td><td>HLA-B associated transcript 2 </td><td>NM_080686.1 </td><td>6750 </td><td>2157 </td><td>ARe17F01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>28 </td><td>HP04890 </td><td>BAT2 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS122P21 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>29 </td><td>HP04715 </td><td>ABCA7 </td><td>ATP-binding cassette, sub-family A (ABC1), member 7 </td><td>NM_019112.2 </td><td>6704 </td><td>2146 </td><td>ARe05B12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>30 </td><td>HP07625 </td><td>WWC2 </td><td>WW, C2 and coiled-coil domain containing 2 </td><td>NM_024949.4 </td><td>6492 </td><td>987 </td><td>ARi58B08 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>31 </td><td>HP04667 </td><td>MYH9 </td><td>myosin, heavy polypeptide 9, non-muscle </td><td>NM_002473.3 </td><td>7274 </td><td>1960 </td><td>ARe01B05 </td><td>AB191263 </td><td>7436 </td><td>1960 </td></tr><tr><td>32 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe12G09 </td><td>AB191263 </td><td>7450 </td><td>1960 </td></tr><tr><td>33 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe58E01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>34 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi26G12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>35 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63B09 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>36 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63H03 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>37 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS045L21 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>38 </td><td>HP04680 </td><td>AGRN </td><td>Agrin </td><td>NM_198576.2 </td><td>7319 </td><td>2045 </td><td>ARe02B04 </td><td>AB191264 </td><td>7319 </td><td>2045 </td></tr><tr><td>39 </td><td>HP04680 </td><td>AGRN </td><td> </td><td> </td><td> </td><td> </td><td>ARiS085N10 </td><td>— </td><td>7.4 k </td><td> </td></tr><tr><td>40 </td><td>HP07482 </td><td>CDC42BPB </td><td>CDC42 binding protein kinase beta (DMPK-like) </td><td>NM_006035.2 </td><td>6782 </td><td>1711 </td><td>ARiS023M02 </td><td>— </td><td>7.2 k </td><td> </td></tr><tr><td>41 </td><td>HP07424 </td><td>MAP9 </td><td>microtubule-associated protein 9 </td><td>NM_001039580.1 </td><td>7333 </td><td>647 </td><td>ARiS109F14 </td><td>— </td><td>7.1 k </td><td> </td></tr><tr><td>42 </td><td>HP07506 </td><td>ROD1 </td><td>ROD1 regulator of differentiation 1 (S. pombe) </td><td>NM_005156.4 </td><td>7230 </td><td>552 </td><td>ARi40D10 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>43 </td><td>HP07574 </td><td>TRAM2 </td><td>Translocation associated membrane protein 2 </td><td>NM_012288.3 </td><td>7065 </td><td>370 </td><td>ARi50G08 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>44 </td><td>HP06927 </td><td>COL4A1 </td><td>Collagen, type IV, alpha 1 </td><td>NM_001845.4 </td><td>6549 </td><td>1669 </td><td>ARe94G08 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>45 </td><td>HP06927 </td><td>COL4A1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi60G12 </td><td>— </td><td>8.2 k </td><td> </td></tr><tr><td>46 </td><td>HP07685 </td><td>ARHGAP23 </td><td>Rho GTPase activating protein 23 </td><td>XM_290799.7 </td><td>6475 </td><td>1684 </td><td>ARi70H06 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>47 </td><td>HP02917 </td><td>PLXNB2 </td><td>Plexin B2 </td><td>XM_371474.4 </td><td>6434 </td><td>1901 </td><td>ARe49A01 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>48 </td><td>HP06882 </td><td>GPAM </td><td>Glycerol-3-phosphate acyltransferase, mitochondrial </td><td>NM_020918.3 </td><td>6386 </td><td>828 </td><td>ARi06G08 </td><td>— </td><td>7.0 k </td><td> </td></tr></tbody></table></div><div content-id="DSN010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>HP ID </th><th>Symbol </th><th>Name </th><th>RefSeq </th><th>mRNA (bp) </th><th>Protein (aa) </th><th>Clone ID </th><th>Ac. No. </th><th>cDNA (bp) </th><th>Protein (aa) </th></tr></thead><tbody><tr><td>1 </td><td>HP08164 </td><td>GOLGB1 </td><td>Golgi autoantigen, golgin subfamily b, macrogolgin (with transmembrane signal) 1 </td><td>NM_004487.3 </td><td>11 185 </td><td>3259 </td><td>ARiS161G17 </td><td>AB371588 </td><td>11 198 </td><td>3269<sup>a</sup> </td></tr><tr><td>2 </td><td>HP07459 </td><td>N4BP2 </td><td>Nedd4 binding protein 2 </td><td>NM_018177.2 </td><td>6616 </td><td>1770 </td><td>ARiS023B20 </td><td>AB371584 </td><td>9736 </td><td>1690<sup>b</sup> </td></tr><tr><td>3 </td><td>HP08032 </td><td>ACACA </td><td>Acetyl-Coenzyme A carboxylase alpha </td><td>NM_198836.1 </td><td>9585 </td><td>2346 </td><td>ARiS088K16 </td><td>AB371587 </td><td>9534 </td><td>2346 </td></tr><tr><td>4 </td><td>HP04958 </td><td>FLNB </td><td>Filamin B, beta (actin binding protein 278) </td><td>NM_001457.2 </td><td>9463 </td><td>2602 </td><td>ARe23D04 </td><td>AB191258 </td><td>9405 </td><td>2591 </td></tr><tr><td>5 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe77D06 </td><td>AB371580 </td><td>8059 </td><td>2633 </td></tr><tr><td>6 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe89D09 </td><td>AB371581 </td><td>9366 </td><td>2578 </td></tr><tr><td>7 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARi12F08 </td><td>AB371582 </td><td>7973 </td><td>2602 </td></tr><tr><td>8 </td><td>HP07616 </td><td>FLNC </td><td>Filamin C, gamma (actin binding protein 280) </td><td>NM_001458.3 </td><td>9146 </td><td>2725 </td><td>ARi57A02 </td><td>AB371585 </td><td>9156 </td><td>2725 </td></tr><tr><td>9 </td><td>HP07744 </td><td>SPTBN1 </td><td>Spectrin, beta, non-erythrocytic 1 </td><td>NM_003128.2 </td><td>10 238 </td><td>2364 </td><td>ARiS088A21 </td><td>AB371586 </td><td>8443 </td><td>2364 </td></tr><tr><td>10 </td><td>HP00079 </td><td>FLNA </td><td>Filamin A, alpha (actin binding protein 280) </td><td>NM_001456.2 </td><td>8278 </td><td>2639 </td><td>ARe06F05 </td><td>AB191259 </td><td>8212 </td><td>2612 </td></tr><tr><td>11 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARe27E03 </td><td>AB191260 </td><td>8241 </td><td>2620 </td></tr><tr><td>12 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi13C12 </td><td>AB371574 </td><td>8242 </td><td>2620 </td></tr><tr><td>13 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi37B09 </td><td>AB371575 </td><td>8212 </td><td>2612 </td></tr><tr><td>14 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi47G07 </td><td>AB371576 </td><td>8243 </td><td>2620 </td></tr><tr><td>15 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi50A09 </td><td>AB371577 </td><td>7321 </td><td>2315 </td></tr><tr><td>16 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi66B08 </td><td>AB371578 </td><td>8212 </td><td>2612 </td></tr><tr><td>17 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARiS088J13 </td><td>AB371579 </td><td>8214 </td><td>2612 </td></tr><tr><td>18 </td><td>HP06504 </td><td>COL5A1 </td><td>Collagen, type V, alpha 1 </td><td>NM_000093.3 </td><td>8439 </td><td>1838 </td><td>ARe79B07 </td><td>AB371586 </td><td>8139 </td><td>1838 </td></tr><tr><td>19 </td><td>HP07532 </td><td>GCN1L1 </td><td>GCN1 general control of amino-acid synthesis 1-like 1 (yeast) </td><td>NM_006836.1 </td><td>8699 </td><td>2671 </td><td>ARi43H04 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>20 </td><td>HP06485 </td><td>PTPRF </td><td>Protein tyrosine phosphatase, receptor type, F </td><td>NM_002840.2 </td><td>7718 </td><td>1897 </td><td>ARe76H09 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>21 </td><td>HP05449 </td><td>SPTAN1 </td><td>Spectrin, alpha, non-erythrocytic 1 (alpha-fodrin) </td><td>NM_003127.1 </td><td>7787 </td><td>2472 </td><td>ARe45C02 </td><td>AB191262 </td><td>7791 </td><td>2452 </td></tr><tr><td>22 </td><td>HP00124 </td><td>FN1 </td><td>Fibronectin 1 </td><td>NM_212476.1 </td><td>8272 </td><td>2296 </td><td>ARe05G09 </td><td>AB191261 </td><td>7753 </td><td>2265 </td></tr><tr><td>23 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi53A03 </td><td>— </td><td>7.7 k </td><td> </td></tr><tr><td>24 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS087I09 </td><td>— </td><td>7.9 k </td><td> </td></tr><tr><td>25 </td><td>HP06896 </td><td>PCM1 </td><td>Pericentriolar material 1 </td><td>NM_006197.3 </td><td>8788 </td><td>2024 </td><td>ARi07G10 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>26 </td><td>HP04727 </td><td>GLIS3 </td><td>GLIS family zinc finger 3 </td><td>NM_001042413.1 </td><td>7656 </td><td>930 </td><td>ARe06E05 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>27 </td><td>HP04890 </td><td>BAT2 </td><td>HLA-B associated transcript 2 </td><td>NM_080686.1 </td><td>6750 </td><td>2157 </td><td>ARe17F01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>28 </td><td>HP04890 </td><td>BAT2 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS122P21 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>29 </td><td>HP04715 </td><td>ABCA7 </td><td>ATP-binding cassette, sub-family A (ABC1), member 7 </td><td>NM_019112.2 </td><td>6704 </td><td>2146 </td><td>ARe05B12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>30 </td><td>HP07625 </td><td>WWC2 </td><td>WW, C2 and coiled-coil domain containing 2 </td><td>NM_024949.4 </td><td>6492 </td><td>987 </td><td>ARi58B08 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>31 </td><td>HP04667 </td><td>MYH9 </td><td>myosin, heavy polypeptide 9, non-muscle </td><td>NM_002473.3 </td><td>7274 </td><td>1960 </td><td>ARe01B05 </td><td>AB191263 </td><td>7436 </td><td>1960 </td></tr><tr><td>32 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe12G09 </td><td>AB191263 </td><td>7450 </td><td>1960 </td></tr><tr><td>33 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe58E01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>34 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi26G12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>35 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63B09 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>36 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63H03 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>37 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS045L21 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>38 </td><td>HP04680 </td><td>AGRN </td><td>Agrin </td><td>NM_198576.2 </td><td>7319 </td><td>2045 </td><td>ARe02B04 </td><td>AB191264 </td><td>7319 </td><td>2045 </td></tr><tr><td>39 </td><td>HP04680 </td><td>AGRN </td><td> </td><td> </td><td> </td><td> </td><td>ARiS085N10 </td><td>— </td><td>7.4 k </td><td> </td></tr><tr><td>40 </td><td>HP07482 </td><td>CDC42BPB </td><td>CDC42 binding protein kinase beta (DMPK-like) </td><td>NM_006035.2 </td><td>6782 </td><td>1711 </td><td>ARiS023M02 </td><td>— </td><td>7.2 k </td><td> </td></tr><tr><td>41 </td><td>HP07424 </td><td>MAP9 </td><td>microtubule-associated protein 9 </td><td>NM_001039580.1 </td><td>7333 </td><td>647 </td><td>ARiS109F14 </td><td>— </td><td>7.1 k </td><td> </td></tr><tr><td>42 </td><td>HP07506 </td><td>ROD1 </td><td>ROD1 regulator of differentiation 1 (S. pombe) </td><td>NM_005156.4 </td><td>7230 </td><td>552 </td><td>ARi40D10 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>43 </td><td>HP07574 </td><td>TRAM2 </td><td>Translocation associated membrane protein 2 </td><td>NM_012288.3 </td><td>7065 </td><td>370 </td><td>ARi50G08 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>44 </td><td>HP06927 </td><td>COL4A1 </td><td>Collagen, type IV, alpha 1 </td><td>NM_001845.4 </td><td>6549 </td><td>1669 </td><td>ARe94G08 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>45 </td><td>HP06927 </td><td>COL4A1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi60G12 </td><td>— </td><td>8.2 k </td><td> </td></tr><tr><td>46 </td><td>HP07685 </td><td>ARHGAP23 </td><td>Rho GTPase activating protein 23 </td><td>XM_290799.7 </td><td>6475 </td><td>1684 </td><td>ARi70H06 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>47 </td><td>HP02917 </td><td>PLXNB2 </td><td>Plexin B2 </td><td>XM_371474.4 </td><td>6434 </td><td>1901 </td><td>ARe49A01 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>48 </td><td>HP06882 </td><td>GPAM </td><td>Glycerol-3-phosphate acyltransferase, mitochondrial </td><td>NM_020918.3 </td><td>6386 </td><td>828 </td><td>ARi06G08 </td><td>— </td><td>7.0 k </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The frame shift owing to the insertion of one nucleotide A in the GOLGB1 clone was corrected.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>RefSeq terminates with a short A-stretch in a 3'-untranslated region.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>In addition, 8 and 5 clones with &gt;6 kb were obtained for COL4A1 and PLXNB2, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6976829" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN010TB2" class="table-wrap-title" data-id="DSN010TB2"><div class="label -label">Table 2</div><div class="caption"><p>Long-sized full-length cDNA clones with &gt;7 kb insert</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>HP ID </th><th>Symbol </th><th>Name </th><th>RefSeq </th><th>mRNA (bp) </th><th>Protein (aa) </th><th>Clone ID </th><th>Ac. No. </th><th>cDNA (bp) </th><th>Protein (aa) </th></tr></thead><tbody><tr><td>1 </td><td>HP08164 </td><td>GOLGB1 </td><td>Golgi autoantigen, golgin subfamily b, macrogolgin (with transmembrane signal) 1 </td><td>NM_004487.3 </td><td>11 185 </td><td>3259 </td><td>ARiS161G17 </td><td>AB371588 </td><td>11 198 </td><td>3269<sup>a</sup> </td></tr><tr><td>2 </td><td>HP07459 </td><td>N4BP2 </td><td>Nedd4 binding protein 2 </td><td>NM_018177.2 </td><td>6616 </td><td>1770 </td><td>ARiS023B20 </td><td>AB371584 </td><td>9736 </td><td>1690<sup>b</sup> </td></tr><tr><td>3 </td><td>HP08032 </td><td>ACACA </td><td>Acetyl-Coenzyme A carboxylase alpha </td><td>NM_198836.1 </td><td>9585 </td><td>2346 </td><td>ARiS088K16 </td><td>AB371587 </td><td>9534 </td><td>2346 </td></tr><tr><td>4 </td><td>HP04958 </td><td>FLNB </td><td>Filamin B, beta (actin binding protein 278) </td><td>NM_001457.2 </td><td>9463 </td><td>2602 </td><td>ARe23D04 </td><td>AB191258 </td><td>9405 </td><td>2591 </td></tr><tr><td>5 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe77D06 </td><td>AB371580 </td><td>8059 </td><td>2633 </td></tr><tr><td>6 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe89D09 </td><td>AB371581 </td><td>9366 </td><td>2578 </td></tr><tr><td>7 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARi12F08 </td><td>AB371582 </td><td>7973 </td><td>2602 </td></tr><tr><td>8 </td><td>HP07616 </td><td>FLNC </td><td>Filamin C, gamma (actin binding protein 280) </td><td>NM_001458.3 </td><td>9146 </td><td>2725 </td><td>ARi57A02 </td><td>AB371585 </td><td>9156 </td><td>2725 </td></tr><tr><td>9 </td><td>HP07744 </td><td>SPTBN1 </td><td>Spectrin, beta, non-erythrocytic 1 </td><td>NM_003128.2 </td><td>10 238 </td><td>2364 </td><td>ARiS088A21 </td><td>AB371586 </td><td>8443 </td><td>2364 </td></tr><tr><td>10 </td><td>HP00079 </td><td>FLNA </td><td>Filamin A, alpha (actin binding protein 280) </td><td>NM_001456.2 </td><td>8278 </td><td>2639 </td><td>ARe06F05 </td><td>AB191259 </td><td>8212 </td><td>2612 </td></tr><tr><td>11 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARe27E03 </td><td>AB191260 </td><td>8241 </td><td>2620 </td></tr><tr><td>12 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi13C12 </td><td>AB371574 </td><td>8242 </td><td>2620 </td></tr><tr><td>13 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi37B09 </td><td>AB371575 </td><td>8212 </td><td>2612 </td></tr><tr><td>14 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi47G07 </td><td>AB371576 </td><td>8243 </td><td>2620 </td></tr><tr><td>15 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi50A09 </td><td>AB371577 </td><td>7321 </td><td>2315 </td></tr><tr><td>16 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi66B08 </td><td>AB371578 </td><td>8212 </td><td>2612 </td></tr><tr><td>17 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARiS088J13 </td><td>AB371579 </td><td>8214 </td><td>2612 </td></tr><tr><td>18 </td><td>HP06504 </td><td>COL5A1 </td><td>Collagen, type V, alpha 1 </td><td>NM_000093.3 </td><td>8439 </td><td>1838 </td><td>ARe79B07 </td><td>AB371586 </td><td>8139 </td><td>1838 </td></tr><tr><td>19 </td><td>HP07532 </td><td>GCN1L1 </td><td>GCN1 general control of amino-acid synthesis 1-like 1 (yeast) </td><td>NM_006836.1 </td><td>8699 </td><td>2671 </td><td>ARi43H04 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>20 </td><td>HP06485 </td><td>PTPRF </td><td>Protein tyrosine phosphatase, receptor type, F </td><td>NM_002840.2 </td><td>7718 </td><td>1897 </td><td>ARe76H09 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>21 </td><td>HP05449 </td><td>SPTAN1 </td><td>Spectrin, alpha, non-erythrocytic 1 (alpha-fodrin) </td><td>NM_003127.1 </td><td>7787 </td><td>2472 </td><td>ARe45C02 </td><td>AB191262 </td><td>7791 </td><td>2452 </td></tr><tr><td>22 </td><td>HP00124 </td><td>FN1 </td><td>Fibronectin 1 </td><td>NM_212476.1 </td><td>8272 </td><td>2296 </td><td>ARe05G09 </td><td>AB191261 </td><td>7753 </td><td>2265 </td></tr><tr><td>23 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi53A03 </td><td>— </td><td>7.7 k </td><td> </td></tr><tr><td>24 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS087I09 </td><td>— </td><td>7.9 k </td><td> </td></tr><tr><td>25 </td><td>HP06896 </td><td>PCM1 </td><td>Pericentriolar material 1 </td><td>NM_006197.3 </td><td>8788 </td><td>2024 </td><td>ARi07G10 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>26 </td><td>HP04727 </td><td>GLIS3 </td><td>GLIS family zinc finger 3 </td><td>NM_001042413.1 </td><td>7656 </td><td>930 </td><td>ARe06E05 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>27 </td><td>HP04890 </td><td>BAT2 </td><td>HLA-B associated transcript 2 </td><td>NM_080686.1 </td><td>6750 </td><td>2157 </td><td>ARe17F01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>28 </td><td>HP04890 </td><td>BAT2 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS122P21 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>29 </td><td>HP04715 </td><td>ABCA7 </td><td>ATP-binding cassette, sub-family A (ABC1), member 7 </td><td>NM_019112.2 </td><td>6704 </td><td>2146 </td><td>ARe05B12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>30 </td><td>HP07625 </td><td>WWC2 </td><td>WW, C2 and coiled-coil domain containing 2 </td><td>NM_024949.4 </td><td>6492 </td><td>987 </td><td>ARi58B08 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>31 </td><td>HP04667 </td><td>MYH9 </td><td>myosin, heavy polypeptide 9, non-muscle </td><td>NM_002473.3 </td><td>7274 </td><td>1960 </td><td>ARe01B05 </td><td>AB191263 </td><td>7436 </td><td>1960 </td></tr><tr><td>32 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe12G09 </td><td>AB191263 </td><td>7450 </td><td>1960 </td></tr><tr><td>33 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe58E01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>34 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi26G12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>35 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63B09 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>36 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63H03 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>37 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS045L21 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>38 </td><td>HP04680 </td><td>AGRN </td><td>Agrin </td><td>NM_198576.2 </td><td>7319 </td><td>2045 </td><td>ARe02B04 </td><td>AB191264 </td><td>7319 </td><td>2045 </td></tr><tr><td>39 </td><td>HP04680 </td><td>AGRN </td><td> </td><td> </td><td> </td><td> </td><td>ARiS085N10 </td><td>— </td><td>7.4 k </td><td> </td></tr><tr><td>40 </td><td>HP07482 </td><td>CDC42BPB </td><td>CDC42 binding protein kinase beta (DMPK-like) </td><td>NM_006035.2 </td><td>6782 </td><td>1711 </td><td>ARiS023M02 </td><td>— </td><td>7.2 k </td><td> </td></tr><tr><td>41 </td><td>HP07424 </td><td>MAP9 </td><td>microtubule-associated protein 9 </td><td>NM_001039580.1 </td><td>7333 </td><td>647 </td><td>ARiS109F14 </td><td>— </td><td>7.1 k </td><td> </td></tr><tr><td>42 </td><td>HP07506 </td><td>ROD1 </td><td>ROD1 regulator of differentiation 1 (S. pombe) </td><td>NM_005156.4 </td><td>7230 </td><td>552 </td><td>ARi40D10 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>43 </td><td>HP07574 </td><td>TRAM2 </td><td>Translocation associated membrane protein 2 </td><td>NM_012288.3 </td><td>7065 </td><td>370 </td><td>ARi50G08 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>44 </td><td>HP06927 </td><td>COL4A1 </td><td>Collagen, type IV, alpha 1 </td><td>NM_001845.4 </td><td>6549 </td><td>1669 </td><td>ARe94G08 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>45 </td><td>HP06927 </td><td>COL4A1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi60G12 </td><td>— </td><td>8.2 k </td><td> </td></tr><tr><td>46 </td><td>HP07685 </td><td>ARHGAP23 </td><td>Rho GTPase activating protein 23 </td><td>XM_290799.7 </td><td>6475 </td><td>1684 </td><td>ARi70H06 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>47 </td><td>HP02917 </td><td>PLXNB2 </td><td>Plexin B2 </td><td>XM_371474.4 </td><td>6434 </td><td>1901 </td><td>ARe49A01 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>48 </td><td>HP06882 </td><td>GPAM </td><td>Glycerol-3-phosphate acyltransferase, mitochondrial </td><td>NM_020918.3 </td><td>6386 </td><td>828 </td><td>ARi06G08 </td><td>— </td><td>7.0 k </td><td> </td></tr></tbody></table></div><div content-id="DSN010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>HP ID </th><th>Symbol </th><th>Name </th><th>RefSeq </th><th>mRNA (bp) </th><th>Protein (aa) </th><th>Clone ID </th><th>Ac. No. </th><th>cDNA (bp) </th><th>Protein (aa) </th></tr></thead><tbody><tr><td>1 </td><td>HP08164 </td><td>GOLGB1 </td><td>Golgi autoantigen, golgin subfamily b, macrogolgin (with transmembrane signal) 1 </td><td>NM_004487.3 </td><td>11 185 </td><td>3259 </td><td>ARiS161G17 </td><td>AB371588 </td><td>11 198 </td><td>3269<sup>a</sup> </td></tr><tr><td>2 </td><td>HP07459 </td><td>N4BP2 </td><td>Nedd4 binding protein 2 </td><td>NM_018177.2 </td><td>6616 </td><td>1770 </td><td>ARiS023B20 </td><td>AB371584 </td><td>9736 </td><td>1690<sup>b</sup> </td></tr><tr><td>3 </td><td>HP08032 </td><td>ACACA </td><td>Acetyl-Coenzyme A carboxylase alpha </td><td>NM_198836.1 </td><td>9585 </td><td>2346 </td><td>ARiS088K16 </td><td>AB371587 </td><td>9534 </td><td>2346 </td></tr><tr><td>4 </td><td>HP04958 </td><td>FLNB </td><td>Filamin B, beta (actin binding protein 278) </td><td>NM_001457.2 </td><td>9463 </td><td>2602 </td><td>ARe23D04 </td><td>AB191258 </td><td>9405 </td><td>2591 </td></tr><tr><td>5 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe77D06 </td><td>AB371580 </td><td>8059 </td><td>2633 </td></tr><tr><td>6 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARe89D09 </td><td>AB371581 </td><td>9366 </td><td>2578 </td></tr><tr><td>7 </td><td>HP04958 </td><td>FLNB </td><td> </td><td> </td><td> </td><td> </td><td>ARi12F08 </td><td>AB371582 </td><td>7973 </td><td>2602 </td></tr><tr><td>8 </td><td>HP07616 </td><td>FLNC </td><td>Filamin C, gamma (actin binding protein 280) </td><td>NM_001458.3 </td><td>9146 </td><td>2725 </td><td>ARi57A02 </td><td>AB371585 </td><td>9156 </td><td>2725 </td></tr><tr><td>9 </td><td>HP07744 </td><td>SPTBN1 </td><td>Spectrin, beta, non-erythrocytic 1 </td><td>NM_003128.2 </td><td>10 238 </td><td>2364 </td><td>ARiS088A21 </td><td>AB371586 </td><td>8443 </td><td>2364 </td></tr><tr><td>10 </td><td>HP00079 </td><td>FLNA </td><td>Filamin A, alpha (actin binding protein 280) </td><td>NM_001456.2 </td><td>8278 </td><td>2639 </td><td>ARe06F05 </td><td>AB191259 </td><td>8212 </td><td>2612 </td></tr><tr><td>11 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARe27E03 </td><td>AB191260 </td><td>8241 </td><td>2620 </td></tr><tr><td>12 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi13C12 </td><td>AB371574 </td><td>8242 </td><td>2620 </td></tr><tr><td>13 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi37B09 </td><td>AB371575 </td><td>8212 </td><td>2612 </td></tr><tr><td>14 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi47G07 </td><td>AB371576 </td><td>8243 </td><td>2620 </td></tr><tr><td>15 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi50A09 </td><td>AB371577 </td><td>7321 </td><td>2315 </td></tr><tr><td>16 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARi66B08 </td><td>AB371578 </td><td>8212 </td><td>2612 </td></tr><tr><td>17 </td><td>HP00079 </td><td>FLNA </td><td> </td><td> </td><td> </td><td> </td><td>ARiS088J13 </td><td>AB371579 </td><td>8214 </td><td>2612 </td></tr><tr><td>18 </td><td>HP06504 </td><td>COL5A1 </td><td>Collagen, type V, alpha 1 </td><td>NM_000093.3 </td><td>8439 </td><td>1838 </td><td>ARe79B07 </td><td>AB371586 </td><td>8139 </td><td>1838 </td></tr><tr><td>19 </td><td>HP07532 </td><td>GCN1L1 </td><td>GCN1 general control of amino-acid synthesis 1-like 1 (yeast) </td><td>NM_006836.1 </td><td>8699 </td><td>2671 </td><td>ARi43H04 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>20 </td><td>HP06485 </td><td>PTPRF </td><td>Protein tyrosine phosphatase, receptor type, F </td><td>NM_002840.2 </td><td>7718 </td><td>1897 </td><td>ARe76H09 </td><td>— </td><td>8.0 k </td><td> </td></tr><tr><td>21 </td><td>HP05449 </td><td>SPTAN1 </td><td>Spectrin, alpha, non-erythrocytic 1 (alpha-fodrin) </td><td>NM_003127.1 </td><td>7787 </td><td>2472 </td><td>ARe45C02 </td><td>AB191262 </td><td>7791 </td><td>2452 </td></tr><tr><td>22 </td><td>HP00124 </td><td>FN1 </td><td>Fibronectin 1 </td><td>NM_212476.1 </td><td>8272 </td><td>2296 </td><td>ARe05G09 </td><td>AB191261 </td><td>7753 </td><td>2265 </td></tr><tr><td>23 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi53A03 </td><td>— </td><td>7.7 k </td><td> </td></tr><tr><td>24 </td><td>HP00124 </td><td>FN1 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS087I09 </td><td>— </td><td>7.9 k </td><td> </td></tr><tr><td>25 </td><td>HP06896 </td><td>PCM1 </td><td>Pericentriolar material 1 </td><td>NM_006197.3 </td><td>8788 </td><td>2024 </td><td>ARi07G10 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>26 </td><td>HP04727 </td><td>GLIS3 </td><td>GLIS family zinc finger 3 </td><td>NM_001042413.1 </td><td>7656 </td><td>930 </td><td>ARe06E05 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>27 </td><td>HP04890 </td><td>BAT2 </td><td>HLA-B associated transcript 2 </td><td>NM_080686.1 </td><td>6750 </td><td>2157 </td><td>ARe17F01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>28 </td><td>HP04890 </td><td>BAT2 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS122P21 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>29 </td><td>HP04715 </td><td>ABCA7 </td><td>ATP-binding cassette, sub-family A (ABC1), member 7 </td><td>NM_019112.2 </td><td>6704 </td><td>2146 </td><td>ARe05B12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>30 </td><td>HP07625 </td><td>WWC2 </td><td>WW, C2 and coiled-coil domain containing 2 </td><td>NM_024949.4 </td><td>6492 </td><td>987 </td><td>ARi58B08 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>31 </td><td>HP04667 </td><td>MYH9 </td><td>myosin, heavy polypeptide 9, non-muscle </td><td>NM_002473.3 </td><td>7274 </td><td>1960 </td><td>ARe01B05 </td><td>AB191263 </td><td>7436 </td><td>1960 </td></tr><tr><td>32 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe12G09 </td><td>AB191263 </td><td>7450 </td><td>1960 </td></tr><tr><td>33 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARe58E01 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>34 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi26G12 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>35 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63B09 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>36 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARi63H03 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>37 </td><td>HP04667 </td><td>MYH9 </td><td> </td><td> </td><td> </td><td> </td><td>ARiS045L21 </td><td>— </td><td>7.5 k </td><td> </td></tr><tr><td>38 </td><td>HP04680 </td><td>AGRN </td><td>Agrin </td><td>NM_198576.2 </td><td>7319 </td><td>2045 </td><td>ARe02B04 </td><td>AB191264 </td><td>7319 </td><td>2045 </td></tr><tr><td>39 </td><td>HP04680 </td><td>AGRN </td><td> </td><td> </td><td> </td><td> </td><td>ARiS085N10 </td><td>— </td><td>7.4 k </td><td> </td></tr><tr><td>40 </td><td>HP07482 </td><td>CDC42BPB </td><td>CDC42 binding protein kinase beta (DMPK-like) </td><td>NM_006035.2 </td><td>6782 </td><td>1711 </td><td>ARiS023M02 </td><td>— </td><td>7.2 k </td><td> </td></tr><tr><td>41 </td><td>HP07424 </td><td>MAP9 </td><td>microtubule-associated protein 9 </td><td>NM_001039580.1 </td><td>7333 </td><td>647 </td><td>ARiS109F14 </td><td>— </td><td>7.1 k </td><td> </td></tr><tr><td>42 </td><td>HP07506 </td><td>ROD1 </td><td>ROD1 regulator of differentiation 1 (S. pombe) </td><td>NM_005156.4 </td><td>7230 </td><td>552 </td><td>ARi40D10 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>43 </td><td>HP07574 </td><td>TRAM2 </td><td>Translocation associated membrane protein 2 </td><td>NM_012288.3 </td><td>7065 </td><td>370 </td><td>ARi50G08 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>44 </td><td>HP06927 </td><td>COL4A1 </td><td>Collagen, type IV, alpha 1 </td><td>NM_001845.4 </td><td>6549 </td><td>1669 </td><td>ARe94G08 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>45 </td><td>HP06927 </td><td>COL4A1 </td><td> </td><td> </td><td> </td><td> </td><td>ARi60G12 </td><td>— </td><td>8.2 k </td><td> </td></tr><tr><td>46 </td><td>HP07685 </td><td>ARHGAP23 </td><td>Rho GTPase activating protein 23 </td><td>XM_290799.7 </td><td>6475 </td><td>1684 </td><td>ARi70H06 </td><td>— </td><td>7.0 k </td><td> </td></tr><tr><td>47 </td><td>HP02917 </td><td>PLXNB2 </td><td>Plexin B2 </td><td>XM_371474.4 </td><td>6434 </td><td>1901 </td><td>ARe49A01 </td><td>— </td><td>7.0 k </td><td><sup>c</sup> </td></tr><tr><td>48 </td><td>HP06882 </td><td>GPAM </td><td>Glycerol-3-phosphate acyltransferase, mitochondrial </td><td>NM_020918.3 </td><td>6386 </td><td>828 </td><td>ARi06G08 </td><td>— </td><td>7.0 k </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The frame shift owing to the insertion of one nucleotide A in the GOLGB1 clone was corrected.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>RefSeq terminates with a short A-stretch in a 3'-untranslated region.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>In addition, 8 and 5 clones with &gt;6 kb were obtained for COL4A1 and PLXNB2, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6976829" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6976830 id="6976830" class="section-title" data-legacy-id=s3d>3.4. Expression profile</h3>
<p>Fig. <a reveal-id="DSN010F1" data-open="DSN010F1" class="link link-reveal link-table xref-fig">1</a> shows the frequency distribution of abundant full-length genes obtained from three libraries. The most abundant genes were GAPDH and FTH1 (each 248 clones, 1.2% in content) followed by ACTB, ACTG1, EEF1A1, VIM, RPL41, MT2A, RPL1, CRYAB, TMSB10, RPS3, and RPL10, each of which gave ≥100 clones (0.5% in content). The number of abundant genes with ≥0.05% content (10 clones) was 310 (6.9% of identified genes). The major components were ribosomal proteins (79 kinds). Of the total genes, 2221 (49.2%) were obtained as a non-redundant transcript.</p>                        <a id="6976832" scrollto-destination="6976832"></a>
<div id="DSN010F1" data-id="DSN010F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01001.jpeg?Expires=2147483647&Signature=uEjzTMW-1-RggoEc31aze1W0CcQipe-SZiUUrYCfMYViQX32IXHhMzw~Y9WzcCxzfK9g3Fxqep2AnkE4~Bm9C9uKaVTCNG0V3bYb1DqY4q8oWQu4YcnmdhSBfJMvwTj7P-E8C7vmZsPxNF6Pb3cXVZdmWGW4uNROu7y-Im6632~OEgKxKmgm3NdUe5noMaeETQeSDD4-5MeX2kSQM51S-SWk7lLbzf0ixyFrSkBh00Y273YkVVUtQnEh77uT-dqEly9IDoeL5Hly5q1W3O7J4Cf7hCG5ZpG4CEc2fuFrwp~ibU~fQ7yg2Q1SOsbTrympo7j2T1u~7jXjjigx1oqlyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequencies of 310 kinds of abundant genes with ≥0.05% content (10 clones) obtained from three libraries. The inlet shows the number of low-redundant genes with ≤0.05% content at each redundancy." path-from-xml="dsn01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976832/dsn01001.jpeg" path-from-xml="dsn01001.jpeg" target="_blank">View large</a><a section="6976832" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01001.jpeg?Expires=2147483647&Signature=CpSEpFRUObpATaJO6tiCGDQfiCNMvkZo5WZCbfn47DESsipVZNFAAOpSmEzzOxPWiAk5wRI7w5xSev36Y4PeDqgs9PdDgw8VhcTqjETtlFMv-zZTYSbTDo85RCtiM9k7A48-nM39W6aKWEWmvV55yNpdY6f8vVahWW9vnpzUtpZrwK7QR15J1s9gqcHTPBmfUor5a3q6M6rxB~zLkV941vDTZKfCpXjIS17UCr30VI2CfvW~APy5Ux6sesCaFUg8ngdvoxV3E55dJR8WMD1J4w-2z~TICmHJkA~sq~ZmsKMWHJZZCq2FyXnxFKm2t-1EmfzC44jSoDjuToCZZND~vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976832&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequencies of 310 kinds of abundant genes with ≥0.05% content (10 clones) obtained from three libraries. The inlet shows the number of low-redundant genes with ≤0.05% content at each redundancy.</p></div></div><div content-id="DSN010F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01001.jpeg?Expires=2147483647&Signature=uEjzTMW-1-RggoEc31aze1W0CcQipe-SZiUUrYCfMYViQX32IXHhMzw~Y9WzcCxzfK9g3Fxqep2AnkE4~Bm9C9uKaVTCNG0V3bYb1DqY4q8oWQu4YcnmdhSBfJMvwTj7P-E8C7vmZsPxNF6Pb3cXVZdmWGW4uNROu7y-Im6632~OEgKxKmgm3NdUe5noMaeETQeSDD4-5MeX2kSQM51S-SWk7lLbzf0ixyFrSkBh00Y273YkVVUtQnEh77uT-dqEly9IDoeL5Hly5q1W3O7J4Cf7hCG5ZpG4CEc2fuFrwp~ibU~fQ7yg2Q1SOsbTrympo7j2T1u~7jXjjigx1oqlyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequencies of 310 kinds of abundant genes with ≥0.05% content (10 clones) obtained from three libraries. The inlet shows the number of low-redundant genes with ≤0.05% content at each redundancy." path-from-xml="dsn01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976832/dsn01001.jpeg" path-from-xml="dsn01001.jpeg" target="_blank">View large</a><a section="6976832" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01001.jpeg?Expires=2147483647&Signature=CpSEpFRUObpATaJO6tiCGDQfiCNMvkZo5WZCbfn47DESsipVZNFAAOpSmEzzOxPWiAk5wRI7w5xSev36Y4PeDqgs9PdDgw8VhcTqjETtlFMv-zZTYSbTDo85RCtiM9k7A48-nM39W6aKWEWmvV55yNpdY6f8vVahWW9vnpzUtpZrwK7QR15J1s9gqcHTPBmfUor5a3q6M6rxB~zLkV941vDTZKfCpXjIS17UCr30VI2CfvW~APy5Ux6sesCaFUg8ngdvoxV3E55dJR8WMD1J4w-2z~TICmHJkA~sq~ZmsKMWHJZZCq2FyXnxFKm2t-1EmfzC44jSoDjuToCZZND~vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976832&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequencies of 310 kinds of abundant genes with ≥0.05% content (10 clones) obtained from three libraries. The inlet shows the number of low-redundant genes with ≤0.05% content at each redundancy.</p></div></div><p>In order to examine the presence of bias on the expression frequency in different libraries, we compared the expression profiles of different libraries. Fig. <a reveal-id="DSN010F2" data-open="DSN010F2" class="link link-reveal link-table xref-fig">2</a>A shows the comparison between frequencies of abundant genes with ≥0.1% content identified from the different pools (ARi and ARiS) of the same library (Lib-2). Although the plots scattered owing to the small sampling number, the correlation between two expression profiles was good (the correlation coefficient = 0.94). As listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 2</a>, there were many genes for which several clones were obtained from one library but no clone from another library, suggesting that we should take the extent of sampling bias into account when we analyze a small number of samples.</p>                        <a id="6976834" scrollto-destination="6976834"></a>
<div id="DSN010F2" data-id="DSN010F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01002.jpeg?Expires=2147483647&Signature=M-8nSSOl8nF8RFYUq~HAG7mZslWXa9w4QWoiVxp7w0P4H39dzNn8FjbI6Tzd~hKOtJM1PavopBHpOwZr2LK26RkurZIIVb6UohN4wFW2PS6gvhVD1EQUs5cmvV8jo1l0NcdRiCZcRdBFsI~Q1gczLLs9KFEXuFegh3F02rmF6K~RmcZQrGSKvef2HClh49yT23mFJGieApFMN8~~ko6wD8kjSdnwtvV~MVl1iqfAds2Mgr1XVYUBRdjTqeUOIkw0gEkYgUhV51~KZEDKmSs21Mxj0unLjS2ixWUTDVftuQTOgFQMV4M5h1jvdO3d1UQij4uiONPegUHPdAjv2RZbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of frequencies of abundant genes. (A) Between different pools from the same library, Lib-2. (B) Between different libraries, Lib-1 and Lib-2. The genes with ≥0.1% content were plotted. The top 10 genes are designated by gene symbols. The line represents the case without bias." path-from-xml="dsn01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976834/dsn01002.jpeg" path-from-xml="dsn01002.jpeg" target="_blank">View large</a><a section="6976834" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01002.jpeg?Expires=2147483647&Signature=bC5V-nWvvI9Cy7EhNhfcBPsrwQPguQpBoTCjavnbqMtQ1e10PI56yOe5aT6ZA7Ib3oQcfEJGKiuGVbkZBWPjiKP1dvxDV3n3TpBG8VTPwDvMxOgyHka1n4V3EAGhfU0hLi-O-77Eud1M4DRAR9jlFMJSPDCb4KL7SEIVIKMCbB2FhldRV9hAG7raft6uY4PM1XEyqa5LY0a1xnUCcwleKkdTdDlPbw0HeB7QSOYkCxUIi4B-U1dQflfyZ149CCIKAICdftoKRo~4SZOmMHMwgkFQRgkQK5JQlBhErdF-Zn3XKRTLYhzmwCVrecii2-pr5QSnC3t4CUDL3z-tY2hpew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976834&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of frequencies of abundant genes. (<strong>A</strong>) Between different pools from the same library, Lib-2. (<strong>B</strong>) Between different libraries, Lib-1 and Lib-2. The genes with ≥0.1% content were plotted. The top 10 genes are designated by gene symbols. The line represents the case without bias.</p></div></div><div content-id="DSN010F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01002.jpeg?Expires=2147483647&Signature=M-8nSSOl8nF8RFYUq~HAG7mZslWXa9w4QWoiVxp7w0P4H39dzNn8FjbI6Tzd~hKOtJM1PavopBHpOwZr2LK26RkurZIIVb6UohN4wFW2PS6gvhVD1EQUs5cmvV8jo1l0NcdRiCZcRdBFsI~Q1gczLLs9KFEXuFegh3F02rmF6K~RmcZQrGSKvef2HClh49yT23mFJGieApFMN8~~ko6wD8kjSdnwtvV~MVl1iqfAds2Mgr1XVYUBRdjTqeUOIkw0gEkYgUhV51~KZEDKmSs21Mxj0unLjS2ixWUTDVftuQTOgFQMV4M5h1jvdO3d1UQij4uiONPegUHPdAjv2RZbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of frequencies of abundant genes. (A) Between different pools from the same library, Lib-2. (B) Between different libraries, Lib-1 and Lib-2. The genes with ≥0.1% content were plotted. The top 10 genes are designated by gene symbols. The line represents the case without bias." path-from-xml="dsn01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976834/dsn01002.jpeg" path-from-xml="dsn01002.jpeg" target="_blank">View large</a><a section="6976834" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01002.jpeg?Expires=2147483647&Signature=bC5V-nWvvI9Cy7EhNhfcBPsrwQPguQpBoTCjavnbqMtQ1e10PI56yOe5aT6ZA7Ib3oQcfEJGKiuGVbkZBWPjiKP1dvxDV3n3TpBG8VTPwDvMxOgyHka1n4V3EAGhfU0hLi-O-77Eud1M4DRAR9jlFMJSPDCb4KL7SEIVIKMCbB2FhldRV9hAG7raft6uY4PM1XEyqa5LY0a1xnUCcwleKkdTdDlPbw0HeB7QSOYkCxUIi4B-U1dQflfyZ149CCIKAICdftoKRo~4SZOmMHMwgkFQRgkQK5JQlBhErdF-Zn3XKRTLYhzmwCVrecii2-pr5QSnC3t4CUDL3z-tY2hpew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976834&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of frequencies of abundant genes. (<strong>A</strong>) Between different pools from the same library, Lib-2. (<strong>B</strong>) Between different libraries, Lib-1 and Lib-2. The genes with ≥0.1% content were plotted. The top 10 genes are designated by gene symbols. The line represents the case without bias.</p></div></div><p>Fig. <a reveal-id="DSN010F2" data-open="DSN010F2" class="link link-reveal link-table xref-fig">2</a>B shows the comparison between frequencies of the different libraries, Lib-1 and Lib-2. Although the order of top 10 genes was different, the correlation was good on the whole (the correlation coefficient = 0.87). These two libraries were constructed from different lots of RNA using slightly different protocols. The variation may result from the difference of RNA lots rather than the difference of protocol. For example, it was reported that the expression level of CRYAB largely varied depending on the conditions of cell culture, such as heat stress<sup><a reveal-id="DSN010C38" data-open="DSN010C38" class="link link-ref link-reveal xref-bibr">38</a></sup> or hypertonic stress.<sup><a reveal-id="DSN010C39" data-open="DSN010C39" class="link link-ref link-reveal xref-bibr">39</a></sup> GAPDH was most abundant in Lib-1, but fifth in Lib-2. The expression level of GAPDH also varied with calcium,<sup><a reveal-id="DSN010C40" data-open="DSN010C40" class="link link-ref link-reveal xref-bibr">40</a></sup> insulin,<sup><a reveal-id="DSN010C41" data-open="DSN010C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and oxygen<sup><a reveal-id="DSN010C42" data-open="DSN010C42" class="link link-ref link-reveal xref-bibr">42</a></sup> in the culture medium. Thus, the difference of culture conditions may induce a different expression level and result in the library-dependent variation in the expression level of these genes.</p>                            <h3 scrollto-destination=6976836 id="6976836" class="section-title" data-legacy-id=s3e>3.5. Estimation of the total number of genes composing the library</h3>
<p>The cumulative number of gene occurrences was plotted as a function of the number of analyzed clones as shown in Fig. <a reveal-id="DSN010F3" data-open="DSN010F3" class="link link-reveal link-table xref-fig">3</a>A. The cumulative number asymptotically increased but did not saturate within the analyzed range. The curve fitting was carried out using six asymptotic models, including negative exponential models and hyperbolic models. As a result, the best fitting was obtained by use of a hyperbolic curve, <em>D<sub>t</sub></em> = <em>St<sup>α</sup></em>/(<em>β</em> + <em>t<sup>α</sup></em>), where <em>D<sub>t</sub></em> denotes the cumulative number of genes for the accumulated number <em>t</em> of sequenced clones, <em>S</em> is an asymptotic value, and <em>α</em> and <em>β</em> are parameters to be estimated from data. The obtained <em>S</em> was 14 348 for Lib-1 and 11 563 for Lib-2. Using the value for Lib-2, the cumulative gene number at 24 000 analyses was calculated to be 4578, which is similar to the observed value, 4513. Even if analyzing 48 000 and 96 000 clones, the cumulative gene number could merely reach 6177 (53.4%) and 7717 (66.7%), respectively.</p>                        <a id="6976838" scrollto-destination="6976838"></a>
<div id="DSN010F3" data-id="DSN010F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01003.jpeg?Expires=2147483647&Signature=AvyWJYlYj8CmWRphqj~M3DcYLBCrhWLwipvSyB4tg5MFvPhXEFVDiWwqSWGJgJowxt5f7I8-sulzEMRvV9MIGwmWadCWEmldCTzZGCaqBw7qzKTChbrlKfvhKqIsqjUWTonF5wWjbfUucpUSlLG2L0YURzbAWHkvCpwWnfAXpWv2MuevZ9xY0-eYai5IkTzgr-OGzmztzyEatjIEEbMAHq5wAHqRuCdYRFNfgIAErzkYGTUL4JnP1kIFY6ofSORpitV9MDUIp0DZ0RhTLFIVOgtY21gcyHr3TFVGxLm2G5F1WvvfDnsYZr0KrzTlOjHNMcVvCdNfXm-jBVtxuYomNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Estimation of the total number of genes composed of each library. (A) The cumulative number of gene occurrences, Dt, was plotted for t, the accumulated number of sequenced clones. In this study, novel gene occurrence per 96 clones was counted. The best fitting curves were obtained using the hyperbolic equation described. The asymptotic value, S, represents the estimate of the total number of genes. (B) Abundance-based coverage estimator model ACE-1. The number of genes containing 10 or fewer clones was used to calculate the gene richness." path-from-xml="dsn01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976838/dsn01003.jpeg" path-from-xml="dsn01003.jpeg" target="_blank">View large</a><a section="6976838" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01003.jpeg?Expires=2147483647&Signature=4mBKoSDTS7uJnWMZw5Ush5TRGw-xsEVITI9XI5PO9FCRSHcQNPstw6q2tnqSl3t6DV4k1udkXceqE078iCO6KBB3Ue-I~en~kTydhsTEyOiRtkgL-MUE2mPHZpI482lWM13d9Y7CiwMfc~VEq1K2SANz0hAMPsiksuOi7w-zuq9Kec7Lt8lv8XwmcRkhz8nWD0ohgi2Rt-mgJG40BScagqDGtZ7gfmQUB2wzvz-3gYa2XEE5A4TMK5xREEel2sJb0oSOfTOOIHbwhH~vb5Tzme8-ncoHRmZfrTlUaCivZs4f56D1DSgJ8BQqm8Th1l-jaK0~O0VKLWlWBmALG3u80g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976838&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Estimation of the total number of genes composed of each library. (<strong>A</strong>) The cumulative number of gene occurrences, <em>D<sub>t</sub></em>, was plotted for <em>t</em>, the accumulated number of sequenced clones. In this study, novel gene occurrence per 96 clones was counted. The best fitting curves were obtained using the hyperbolic equation described. The asymptotic value, <em>S</em>, represents the estimate of the total number of genes. (<strong>B</strong>) Abundance-based coverage estimator model ACE-1. The number of genes containing 10 or fewer clones was used to calculate the gene richness.</p></div></div><div content-id="DSN010F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01003.jpeg?Expires=2147483647&Signature=AvyWJYlYj8CmWRphqj~M3DcYLBCrhWLwipvSyB4tg5MFvPhXEFVDiWwqSWGJgJowxt5f7I8-sulzEMRvV9MIGwmWadCWEmldCTzZGCaqBw7qzKTChbrlKfvhKqIsqjUWTonF5wWjbfUucpUSlLG2L0YURzbAWHkvCpwWnfAXpWv2MuevZ9xY0-eYai5IkTzgr-OGzmztzyEatjIEEbMAHq5wAHqRuCdYRFNfgIAErzkYGTUL4JnP1kIFY6ofSORpitV9MDUIp0DZ0RhTLFIVOgtY21gcyHr3TFVGxLm2G5F1WvvfDnsYZr0KrzTlOjHNMcVvCdNfXm-jBVtxuYomNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Estimation of the total number of genes composed of each library. (A) The cumulative number of gene occurrences, Dt, was plotted for t, the accumulated number of sequenced clones. In this study, novel gene occurrence per 96 clones was counted. The best fitting curves were obtained using the hyperbolic equation described. The asymptotic value, S, represents the estimate of the total number of genes. (B) Abundance-based coverage estimator model ACE-1. The number of genes containing 10 or fewer clones was used to calculate the gene richness." path-from-xml="dsn01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976838/dsn01003.jpeg" path-from-xml="dsn01003.jpeg" target="_blank">View large</a><a section="6976838" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01003.jpeg?Expires=2147483647&Signature=4mBKoSDTS7uJnWMZw5Ush5TRGw-xsEVITI9XI5PO9FCRSHcQNPstw6q2tnqSl3t6DV4k1udkXceqE078iCO6KBB3Ue-I~en~kTydhsTEyOiRtkgL-MUE2mPHZpI482lWM13d9Y7CiwMfc~VEq1K2SANz0hAMPsiksuOi7w-zuq9Kec7Lt8lv8XwmcRkhz8nWD0ohgi2Rt-mgJG40BScagqDGtZ7gfmQUB2wzvz-3gYa2XEE5A4TMK5xREEel2sJb0oSOfTOOIHbwhH~vb5Tzme8-ncoHRmZfrTlUaCivZs4f56D1DSgJ8BQqm8Th1l-jaK0~O0VKLWlWBmALG3u80g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976838&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Estimation of the total number of genes composed of each library. (<strong>A</strong>) The cumulative number of gene occurrences, <em>D<sub>t</sub></em>, was plotted for <em>t</em>, the accumulated number of sequenced clones. In this study, novel gene occurrence per 96 clones was counted. The best fitting curves were obtained using the hyperbolic equation described. The asymptotic value, <em>S</em>, represents the estimate of the total number of genes. (<strong>B</strong>) Abundance-based coverage estimator model ACE-1. The number of genes containing 10 or fewer clones was used to calculate the gene richness.</p></div></div><p>Ida et al.<sup><a reveal-id="DSN010C43" data-open="DSN010C43" class="link link-ref link-reveal xref-bibr">43</a></sup> estimated the number of total gene clusters in mouse retina EST libraries using the abundance-based coverage estimator model ACE. We applied its modified model ACE-1 to estimate the total number of genes composed of Lib-1 and Lib-2. When the number of genes containing 10 or fewer clones was used, the total number was estimated to be 8019 for Lib-1 and 8469 for Lib-2, as shown in Fig. <a reveal-id="DSN010F3" data-open="DSN010F3" class="link link-reveal link-table xref-fig">3</a>B. These values were smaller than those estimated using the cumulative curves. Consequently, the total number of genes comprising the present library was estimated to be between 8000 and 14 000, and thus many genes would remain unidentified.</p>                            <h3 scrollto-destination=6976840 id="6976840" class="section-title" data-legacy-id=s3f>3.6. TSSs of abundant transcripts</h3>
<p>The 5'-terminal sequence analysis of full-length cDNA clones has the advantage of massive production of TSSs.<sup><a reveal-id="DSN010C22" data-open="DSN010C22" class="link link-ref link-reveal xref-bibr">22</a></sup> In particular, the full-length content of the vector-capped cDNA library is so high that the distribution of TSSs of transcriptional initiation variants can be determined for abundant transcripts obtained from each library. The full-length content of the top 11 abundant genes was 94.0–100% for each gene cluster. In addition, truncated cDNAs were easily distinguishable, because most of them started at the position in the last exon and had no additional G at the 5' end.</p><p>The distributions of TSSs for 11 genes, which were obtained from two libraries and DBTSS,<sup><a reveal-id="DSN010C44" data-open="DSN010C44" class="link link-ref link-reveal xref-bibr">44</a></sup> were compared as shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 3</a>. The most frequent TSS for each gene, except for RPL1 and CRYAB, was identical among three distributions. EEF1A1 showed almost only one TSS, but generally multiple preferential TSSs were observed in other genes. Fig. <a reveal-id="DSN010F4" data-open="DSN010F4" class="link link-reveal link-table xref-fig">4</a> shows examples of comparison among distributions of TSSs for GAPDH, ACTG1, and CRYAB. The preferential TSSs formed a cluster in the region of ∼10 nt, presumably owing to the presence of the TATA box upstream of these TSSs. CRYAB also showed widely scattered rare TSSs at the upstream region. In every gene, the distribution patterns of TSSs obtained from two libraries and DBTSS were similar. It should be noted that the data of DBTSS was obtained from various tissues, suggesting that the pattern of TSS distribution shows no tissue specificity in the case of abundant housekeeping genes. DBTSS represented TSSs generated by alternative promoter usage.<sup><a reveal-id="DSN010C44" data-open="DSN010C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The TSS observed in the first intron of GAPDH and VIM, and two TSSs identified at the position ∼1400 nt upstream of the main TSS cluster of CRYAB, may result from transcripts given by alternative promoter usage.</p>                        <a id="6976843" scrollto-destination="6976843"></a>
<div id="DSN010F4" data-id="DSN010F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01004.jpeg?Expires=2147483647&Signature=260Q5eJJ1OXmGIQwriZNXxbx4er29DbVex~o7LbEX3~O5ANb80ivKBSO0jMP7pISqHxOgBWudH0StlBPp0hJgECUmLyImCkn~ZxXTBwQw4ElDGH~BDUAhR-4zTQCkJ4G~eEJ8VMzI5tdfpOnrrWCy6vp7Q3CODoGPaFzgjEfV8ZI5SGqatwPX8gxmtvvaNdT36VZ~qgAiGw59A9gtqOG2fSrrDA5sBiXbkwHnMq8cvHlkS6r2p0Gdasi3rKn70Zna0pupfTkcxsqUIQIhTYNxZj3Qapgz8eQEYk~JqDwJpJS2lcydah6OmPyy2PwRUqwJBNkOe6-c~dXBkkPzstoMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison among distributions of transcriptional start sites. Black bar, Lib-1; Gray bar, Lib-2; White bar, DBTSS. Position 1 is defined as a major TSS." path-from-xml="dsn01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976843/dsn01004.jpeg" path-from-xml="dsn01004.jpeg" target="_blank">View large</a><a section="6976843" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01004.jpeg?Expires=2147483647&Signature=wl-hv4oXVboc6pBOjCXFd52WtkWtOVws39WV4WEkyRo~VrRgsolT73rrrJ1Tn7FzYOJWzI8jrkIzMmeGK4sM61FLe1zm~AM8kFW1NZn3eqTta4A6IDdpmY52Lr9xUYpP9wcS7NeZwWRqPkXNakuGynX85ToQObj4ph~EZIVB5ZSR3IwhbRytkxrqUKPOwg~Hvxh61pER4qbO4voFqCMgxcpL-g3UcnHDmXw16xtiTmTwA6lPkkYHQNAem6A7qAMGRm1zzVeJo2dN6AtKPxQGPa5P~6RXRRCEIfpzO305LyTquV~XeKzeFmiMY4qU6HMBbv90t82loeEs323RYHDyHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976843&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison among distributions of transcriptional start sites. Black bar, Lib-1; Gray bar, Lib-2; White bar, DBTSS. Position 1 is defined as a major TSS.</p></div></div><div content-id="DSN010F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01004.jpeg?Expires=2147483647&Signature=260Q5eJJ1OXmGIQwriZNXxbx4er29DbVex~o7LbEX3~O5ANb80ivKBSO0jMP7pISqHxOgBWudH0StlBPp0hJgECUmLyImCkn~ZxXTBwQw4ElDGH~BDUAhR-4zTQCkJ4G~eEJ8VMzI5tdfpOnrrWCy6vp7Q3CODoGPaFzgjEfV8ZI5SGqatwPX8gxmtvvaNdT36VZ~qgAiGw59A9gtqOG2fSrrDA5sBiXbkwHnMq8cvHlkS6r2p0Gdasi3rKn70Zna0pupfTkcxsqUIQIhTYNxZj3Qapgz8eQEYk~JqDwJpJS2lcydah6OmPyy2PwRUqwJBNkOe6-c~dXBkkPzstoMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison among distributions of transcriptional start sites. Black bar, Lib-1; Gray bar, Lib-2; White bar, DBTSS. Position 1 is defined as a major TSS." path-from-xml="dsn01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976843/dsn01004.jpeg" path-from-xml="dsn01004.jpeg" target="_blank">View large</a><a section="6976843" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01004.jpeg?Expires=2147483647&Signature=wl-hv4oXVboc6pBOjCXFd52WtkWtOVws39WV4WEkyRo~VrRgsolT73rrrJ1Tn7FzYOJWzI8jrkIzMmeGK4sM61FLe1zm~AM8kFW1NZn3eqTta4A6IDdpmY52Lr9xUYpP9wcS7NeZwWRqPkXNakuGynX85ToQObj4ph~EZIVB5ZSR3IwhbRytkxrqUKPOwg~Hvxh61pER4qbO4voFqCMgxcpL-g3UcnHDmXw16xtiTmTwA6lPkkYHQNAem6A7qAMGRm1zzVeJo2dN6AtKPxQGPa5P~6RXRRCEIfpzO305LyTquV~XeKzeFmiMY4qU6HMBbv90t82loeEs323RYHDyHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976843&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison among distributions of transcriptional start sites. Black bar, Lib-1; Gray bar, Lib-2; White bar, DBTSS. Position 1 is defined as a major TSS.</p></div></div>                            <h3 scrollto-destination=6976844 id="6976844" class="section-title" data-legacy-id=s3g>3.7. Long-sized transcripts</h3>
<p>On the basis of the mRNA size of RefSeq for genes with GeneID in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 2</a>, the gene-based and clone-based average lengths of cDNA inserts of our full-length gene collection were calculated to be 2.46 kb (4378 genes) and 1.68 kb (19 758 clones), respectively. The previous paper showed that the V-capping method could synthesize a long-sized full-length cDNA.<sup><a reveal-id="DSN010C30" data-open="DSN010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The present libraries also contained many long-sized full-length clones with a cDNA insert of ≥7 kb, as listed in Table <a reveal-id="DSN010TB2" data-open="DSN010TB2" class="link link-reveal link-table xref-fig">2</a>. The size of the mRNA described in the RefSeq data was often different from the insert size of our cDNA clone because of size variants generated by alternative splicing or alternative polyadenylation. Therefore, cDNA clones that correspond to RefSeq derived from &gt;6 kb mRNA were selected, and then the real size of the insert was examined by restriction enzyme digestion followed by agarose gel electrophoresis. Table <a reveal-id="DSN010TB2" data-open="DSN010TB2" class="link link-reveal link-table xref-fig">2</a> shows the determined size together with the mRNA size of RefSeq. Some clones were fully sequenced and their precise length was listed with an accession number.</p><p>The longest cDNA of 11 199 bp encoded golgin B1 (GOLGB1), which is a Golgi integral membrane protein originally named ‘giantin’ owing to its huge size of ∼400 kDa.<sup><a reveal-id="DSN010C45" data-open="DSN010C45" class="link link-ref link-reveal xref-bibr">45</a></sup> When compared with RefSeq, the coding region of this cDNA had two insertions of 15 bp each that resulted in the insertion of a total of 10 amino acid residues, implying an alternative-splicing variant. A total of nine single-nucleotide variations were observed, one of which was the insertion of one nucleotide A to an A stretch at position 2958–2965 in the coding region, causing a frame-shift. This insertion may result from misreading of reverse transcriptase during first-strand synthesis, because sequencing of the GOLGB1 locus of the ARPE-19 genome showed the absence of such an insertion.</p><p>Redundant cDNA clones in the long-sized cDNAs of ≥8 kb were filamin A (FLNA) and filamin B (FLNB). As a result of full sequencing of a total of eight FLNA clones, three splicing variants were identified, whose exon–intron structures are depicted in Fig. <a reveal-id="DSN010F5" data-open="DSN010F5" class="link link-reveal link-table xref-fig">5</a>A. V1 was the longest variant and three clones showed identical TSS. The V2 clones started at the 6th nucleotide downstream of the TSS of V1, and lacked the 29th exon, which caused deletion of eight amino acid residues. V3 had the same TSS as V2, and lacked the region from the middle of exon 36 to the middle of exon 41 of V1, which caused deletion of 305 amino acid residues. RefSeq seems to correspond to V2, but it is doubtful that it actually dose because RefSeq is constructed using multiple sequences reported by different researchers. Surprisingly, all four FLNB clones showed different splicing patterns, as shown in Fig. <a reveal-id="DSN010F5" data-open="DSN010F5" class="link link-reveal link-table xref-fig">5</a>B.</p>                        <a id="6976848" scrollto-destination="6976848"></a>
<div id="DSN010F5" data-id="DSN010F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01005.jpeg?Expires=2147483647&Signature=PSqfA7M00WXDzDYulqKF~oOQxohfLBxtfdDP1OKxh012VoE51x1J0nb2hSPkHBpw9BykLf~IlqHnY186r6OpF3XqhWe4l~DaHjG3lTxnDQTZ6tloXxGmJRqPThCtfAwM8F7ZPWDROyksQ4WSxbfAGdllLT-ibDmmfKVep7RPgVGcUwG5C7sEb15OOCmHS7qVoDZzFKHkylJxbG5DB2K4ex2K4Vmz9sFZFjFIpKe0rwGYNfgHyOIMQBRqHcrit1TN5XBOqMJRH696xnom1g4saSI5kNNY3B32DR~07TEJCxY9ukMv15N5tndsCyXppLrNUweC~WohSDxbWArcqTiB7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The exon–intron structure of splicing variants for FLNA (A) and FLNB (B). Arrow heads represent exon inclusion or deletion." path-from-xml="dsn01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976848/dsn01005.jpeg" path-from-xml="dsn01005.jpeg" target="_blank">View large</a><a section="6976848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01005.jpeg?Expires=2147483647&Signature=PUVzA84kBD7p~oxxiSCAPjLd6AqaHm4xNybljRSB3YS2WkosmNf8DhHqpCrWzF2Tiq2GqEskueJdJAanQnegbpRWYoAxCAyUK6u62yjuGEFfLd9AJWeXMFrDRPYZJAC6KBDuPM2fWCsZbewD~E14ml4SVSPchgbj6Uuf9poivoL26nt8CWu5ZvUBjFgo08yYJUlZaNtET-WBNzzJHTUDfNFlT~0dOF8cXeB3gkCi1hC27k5NHpcE7PVDz~6sHiwfPwSQ687GkaaCUHW~-~1Djs3cs6ml1OY~PhJDnq5VCdgOu1ZVFmWf2Y8EiVLPZLc6QUmLT3nJ8dbndbn0wUA76g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976848&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The exon–intron structure of splicing variants for FLNA (<strong>A</strong>) and FLNB (<strong>B</strong>). Arrow heads represent exon inclusion or deletion.</p></div></div><div content-id="DSN010F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01005.jpeg?Expires=2147483647&Signature=PSqfA7M00WXDzDYulqKF~oOQxohfLBxtfdDP1OKxh012VoE51x1J0nb2hSPkHBpw9BykLf~IlqHnY186r6OpF3XqhWe4l~DaHjG3lTxnDQTZ6tloXxGmJRqPThCtfAwM8F7ZPWDROyksQ4WSxbfAGdllLT-ibDmmfKVep7RPgVGcUwG5C7sEb15OOCmHS7qVoDZzFKHkylJxbG5DB2K4ex2K4Vmz9sFZFjFIpKe0rwGYNfgHyOIMQBRqHcrit1TN5XBOqMJRH696xnom1g4saSI5kNNY3B32DR~07TEJCxY9ukMv15N5tndsCyXppLrNUweC~WohSDxbWArcqTiB7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The exon–intron structure of splicing variants for FLNA (A) and FLNB (B). Arrow heads represent exon inclusion or deletion." path-from-xml="dsn01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976848/dsn01005.jpeg" path-from-xml="dsn01005.jpeg" target="_blank">View large</a><a section="6976848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01005.jpeg?Expires=2147483647&Signature=PUVzA84kBD7p~oxxiSCAPjLd6AqaHm4xNybljRSB3YS2WkosmNf8DhHqpCrWzF2Tiq2GqEskueJdJAanQnegbpRWYoAxCAyUK6u62yjuGEFfLd9AJWeXMFrDRPYZJAC6KBDuPM2fWCsZbewD~E14ml4SVSPchgbj6Uuf9poivoL26nt8CWu5ZvUBjFgo08yYJUlZaNtET-WBNzzJHTUDfNFlT~0dOF8cXeB3gkCi1hC27k5NHpcE7PVDz~6sHiwfPwSQ687GkaaCUHW~-~1Djs3cs6ml1OY~PhJDnq5VCdgOu1ZVFmWf2Y8EiVLPZLc6QUmLT3nJ8dbndbn0wUA76g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976848&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The exon–intron structure of splicing variants for FLNA (<strong>A</strong>) and FLNB (<strong>B</strong>). Arrow heads represent exon inclusion or deletion.</p></div></div>                            <h3 scrollto-destination=6976849 id="6976849" class="section-title" data-legacy-id=s3h>3.8. Correlation between the number of isolated full-length clones and mRNA content</h3>
<p>The high content of the very long-sized cDNA clones leads us to expect that the present library is unbiased by mRNA size. In order to assess the extent of bias, the mRNA contents for eight genes with different mRNA sizes (1.1–9.5 k) were measured by real-time PCR. As shown in Fig. <a reveal-id="DSN010F6" data-open="DSN010F6" class="link link-reveal link-table xref-fig">6</a>, the total number of full-length clones obtained from the present libraries had good correlation with the content of each mRNA. GUK1 (1.1 k), RAI14 (5.0 k), and FLNB (9.5 k) showed a similar content and were represented by full-length cDNA clones with the number of the same order of magnitude independent of their mRNA sizes. Thus, the bias is expected to be low up to 9.5 kb, explaining the fact that a small-sized library composed of only 20 000 clones contained 48 long-sized full-length clones with a cDNA insert of ≥7 kb (Table <a reveal-id="DSN010TB2" data-open="DSN010TB2" class="link link-reveal link-table xref-fig">2</a>).</p>                        <a id="6976851" scrollto-destination="6976851"></a>
<div id="DSN010F6" data-id="DSN010F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01006.jpeg?Expires=2147483647&Signature=rJuyuXooxysi24L1yAN2cUH9g4fi7gJVLXSzbcos1x4Z8~-JZKGjfGGvYiIQa03oZsRZIsQSoM2DnM58MYQ9HYOLrTltWju5YrFXbjOOt7kZz9i8PlbAqzYTTy5fGvG25YnoHAvMLxILMX90gc-jEUna9~f~QvWm-id1bgCVpajN4BVrNmCggJsQ-xsmXzIrnQoqbCFSgPGt0HkWhyzw58lxvZlkeUTV5ZP0fmF-zsyVE0ir-P8iiNN2SZUaGQkheSVe7FinBOkh0hUk3DZxYm-r2doQAHJMj47Chsc1mVu17Vngchfhf8RkBK82noKPIP1pooYzhOIAIVBk7e1Cvw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of isolated full-length clones and the mRNA content. The size of mRNA was described in parentheses following each gene symbol." path-from-xml="dsn01006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976851/dsn01006.jpeg" path-from-xml="dsn01006.jpeg" target="_blank">View large</a><a section="6976851" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01006.jpeg?Expires=2147483647&Signature=eq9iXMIQzGtGJxCi3qtAUwRx6mvRE8pJ7XsVgVbvWu~6NyjwjNxxUqF757wwyV0d2y4qjRUft5CXfm1gORSAUYIsYEpgpGDAKXCmYxiz3MCaI5jQu-HxEzZGyDjzU5rJtm~kQSatRci0BwbWW9dcQTCiYEUGlZZU6UR2VezrwQG-sz8Vo9JPobN5s5JFua4-AclKAHzACPddkkhyvBa5rzwpucbQyxrsLtkOIi5~bKVxT9zF-6hatfzZURt08xEEf5kcBJutDZV3pOzxIQ72sW-vCEYl0rzqIS3XbbXjLc3DQpNLRTFrehW0j-vZDc8ol4YsLA5HeJXfr1xOnade~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976851&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of isolated full-length clones and the mRNA content. The size of mRNA was described in parentheses following each gene symbol.</p></div></div><div content-id="DSN010F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/m_dsn01006.jpeg?Expires=2147483647&Signature=rJuyuXooxysi24L1yAN2cUH9g4fi7gJVLXSzbcos1x4Z8~-JZKGjfGGvYiIQa03oZsRZIsQSoM2DnM58MYQ9HYOLrTltWju5YrFXbjOOt7kZz9i8PlbAqzYTTy5fGvG25YnoHAvMLxILMX90gc-jEUna9~f~QvWm-id1bgCVpajN4BVrNmCggJsQ-xsmXzIrnQoqbCFSgPGt0HkWhyzw58lxvZlkeUTV5ZP0fmF-zsyVE0ir-P8iiNN2SZUaGQkheSVe7FinBOkh0hUk3DZxYm-r2doQAHJMj47Chsc1mVu17Vngchfhf8RkBK82noKPIP1pooYzhOIAIVBk7e1Cvw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the number of isolated full-length clones and the mRNA content. The size of mRNA was described in parentheses following each gene symbol." path-from-xml="dsn01006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6976851/dsn01006.jpeg" path-from-xml="dsn01006.jpeg" target="_blank">View large</a><a section="6976851" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn01006.jpeg?Expires=2147483647&Signature=eq9iXMIQzGtGJxCi3qtAUwRx6mvRE8pJ7XsVgVbvWu~6NyjwjNxxUqF757wwyV0d2y4qjRUft5CXfm1gORSAUYIsYEpgpGDAKXCmYxiz3MCaI5jQu-HxEzZGyDjzU5rJtm~kQSatRci0BwbWW9dcQTCiYEUGlZZU6UR2VezrwQG-sz8Vo9JPobN5s5JFua4-AclKAHzACPddkkhyvBa5rzwpucbQyxrsLtkOIi5~bKVxT9zF-6hatfzZURt08xEEf5kcBJutDZV3pOzxIQ72sW-vCEYl0rzqIS3XbbXjLc3DQpNLRTFrehW0j-vZDc8ol4YsLA5HeJXfr1xOnade~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6976851&ar=464521&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the number of isolated full-length clones and the mRNA content. The size of mRNA was described in parentheses following each gene symbol.</p></div></div>                            <h3 scrollto-destination=6976852 id="6976852" class="section-title" data-legacy-id=s3i>3.9. Unannotated transcripts</h3>
<p>Of the 4513 full-length transcripts identified, 143 (3.2%) have not been included in Entrez Gene, but 79 out of them hit rare ESTs registered in the UniGene EST database. Although 19 clones did not hit any EST sequence, it did not necessarily mean they were novel genes because our clone may result from a partial sequence not overlapping with known ESTs. The unannotated transcripts can be classified into two groups. One group included an intergenic transcript that was mapped to the gene-unoccupied space between known gene loci. Many of them were transcribed from just upstream of TSS of the known gene toward the opposite direction, presumably owing to the presence of a bi-directional promoter.<sup><a reveal-id="DSN010C46" data-open="DSN010C46" class="link link-ref link-reveal xref-bibr">46</a></sup> The second group (36 genes) was composed of transcripts whose location overlapped the exon of the known gene, generating an antisense transcript against the known exon as listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 4</a>. The antisense transcript against the first exon of the known gene is especially intriguing because of possible involvement in transcriptional regulation. The corresponding ESTs often existed in the UniGene EST database, but they were assigned to the sense strand of the known gene.</p>                            <h2 scrollto-destination=6976854 id="6976854" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In-depth analysis of the full-length cDNA libraries constructed using the V-capping method has revealed that these libraries contained full-length cDNA clones with a wide range of sizes up to 11 kb, suggesting that they meet the requirement for size-unbiased libraries. This success may be attributed to the following reasons. (i) Total RNA was used as a template without purifying poly(A) RNA. (ii) The protocol of cDNA synthesis did not include an intact mRNA selection process such as modification of the cap structure.<sup><a reveal-id="DSN010C4  DSN010C6" data-open="DSN010C4  DSN010C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup> These purification and modification processes might cause mRNA degradation. (iii) A small amount of mRNA was used as a template for reverse transcription. When a large amount of mRNA is used, the synthesis of short abundant cDNAs may consume reverse transcriptase and nucleotide substrates, and rare or long-sized cDNA synthesis may be suppressed. (iv) Size fractionation of cDNAs was not carried out. (v) Restriction enzyme digestion of cDNA was not included. (vi) PCR amplification was not used. (vii) Insertion of cDNA into a small-sized vector of 3.4 kb was done by self-ligation. One conceivable size bias may result from incomplete extension of the first-strand cDNA synthesized from a long-sized mRNA template. The presence of GOLB1 cDNA with an 11.2 kb insert and multiple FLNA cDNAs with an 8.2 kb insert suggested that the termination of reverse transcriptase was not problematic under the present conditions.</p><p>The present analysis also revealed that the full-length content was unexpectedly high despite lacking a full-length selection process. The full-length content of abundant transcripts was 94.0–100%. The most probable explanation for this high content is that most mRNA molecules are intact in the cell. However, the degradation of mRNA, especially long-sized mRNA, seemed to occur during the RNA isolation process. The overall full-length content was 95%, and analysis of the remaining 5% incomplete cDNAs showed that ∼10% of them were derived from the degradation product of the long-sized mRNA of &gt;6 kb (data not shown). Although a size bias due to RNA degradation during the course of its isolation is inevitable, this method can faithfully reflect the composition of a given purified RNA sample.</p><p>The high full-length content and size-unbiased feature of the library enabled us to determine the fine distribution of TSS of genes expressed in a single type of cell. Recently, methods such as CAGE<sup><a reveal-id="DSN010C12" data-open="DSN010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> or 5'SAGE<sup><a reveal-id="DSN010C13" data-open="DSN010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> have been developed to determine the distribution of TSS. For these analyses, a vector-capped library prepared using a modified vector primer for CAGE or 5'SAGE analysis can serve as a starting material. The additional G at the 5' end could ensure the intactness of TSS and provide more precise results.</p><p>The vector-capped cDNA library also enabled us to analyze the presence of a cap structure at the 5' end of mRNA. The large-scale TSS analysis revealed that a small part of full-length clones lacked an additional 5'-end G, indicating the absence of the cap structure of the mRNA. The common feature of the 5'-end sequence of these clones was a pyrimidine-rich sequence or A-starting sequence. Further in-depth analysis of G-free clones suggested that the 5'-end sequence affected the addition or elimination of the cap structure. This finding may partially be explained by the recent results that human decapping enzyme Dcp2 preferentially binds to a subset of mRNAs and identifies sequences at the 5' terminus of the mRNA as a specific substrate.<sup><a reveal-id="DSN010C47" data-open="DSN010C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Further investigation is required to elucidate the mechanism of the cap-free mRNA generation observed in the present study and its biological meaning.</p><p>We identified 4513 kinds of genes out of 19 951 full-length clones isolated from unamplified ARPE-19 cDNA libraries. The total number of genes consisting of the present library was estimated to be 8000–14 000, which is consistent with ∼10 000 estimated for the transcriptome of RPE using UniGene clusters, SAGE tags and ESTs by Swaroop and Zack.<sup><a reveal-id="DSN010C48" data-open="DSN010C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The expression profile analyses using a microarray with 12 600 gene probes identified 5634 ± 65 genes for ARPE-19 and 5580 ± 84 genes for human RPE.<sup><a reveal-id="DSN010C49" data-open="DSN010C49" class="link link-ref link-reveal xref-bibr">49</a></sup> These results suggest the low expression level of unidentified genes that may include non-coding and antisense genes. In order to obtain the remaining rare genes, further large-scale analyses in combination with subtraction/normalization are required.</p><p>Recent investigations have shown that an unexpectedly large amount of alternative splicing variants exist<sup><a reveal-id="DSN010C23" data-open="DSN010C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and that some variants were expressed in a cell-specific manner.<sup><a reveal-id="DSN010C50" data-open="DSN010C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In order to analyze a transcriptional network in a single type of cell, it is necessary to measure the expression levels of these variants. When alternative splicing occurs at the multiple sites, it is difficult to quantify each variant by the RT–PCR method, which measures only the expression level of a limited region of mRNA. The most precise method for determining the expression level of a splicing variant is to count the number of full-length cDNA clones for each variant. This is a particular requirement for long-sized full-length transcripts. The present study demonstrated that multiple splicing variants for long-sized genes such as FLNA and FLNB were expressed in a single type of cell and that the V-capping method was able to provide unique genuine full-length cDNA to each variant. The physiological role of each variant remains to be solved.</p><p>One of the characteristics of the V-capping method is the use of a vector primer with a relatively long dT tail, which has the following merits compared with the use of an oligo dT primer by conventional methods: (i) the unidirectional insertion of the cDNA is guaranteed and makes it easy to identify an antisense transcript against the known gene; (ii) it does not prime a short A-stretch in the mRNA, not generating 3'-truncated cDNAs that were occasionally observed in the RefSeq database (e.g. N4BP2 in Table <a reveal-id="DSN010TB2" data-open="DSN010TB2" class="link link-reveal link-table xref-fig">2</a>). However, we should keep in mind the possibility that the vector primer could fail to capture mRNA with a short poly(A) tail.</p><p>The remaining challenge to analyzing the vector-capped cDNA library is to develop a method for high-throughput single-pass sequencing. When we intend to analyze &gt;100 000 clones, the present single-pass sequencing strategy is not realistic because of high cost and because of the amount of work involved. However, considering that the analysis of the present high-quality full-length cDNA library is expected to enable discovery of rare or long-sized full-length cDNA clones and fine expression profiling of various variants, it makes sense to analyze more than a million clones corresponding to total clones composed of one library. Thus, developing a novel method for massive single-pass sequencing is desired, for example, by applying recently developed technologies.<sup><a reveal-id="DSN010C51" data-open="DSN010C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Furthermore, in order to achieve a comprehensive collection of full-length transcripts, the removal of abundant clones should be carried out by subtraction.</p><p>In conclusion, a full-length cDNA library constructed with the V-capping method has been shown to meet requirements for a size-unbiased library, and thus is expected to be suitable for the comprehensive collection of full-length transcripts and their fine expression profiling. Recently, Miura et al.<sup><a reveal-id="DSN010C52" data-open="DSN010C52" class="link link-ref link-reveal xref-bibr">52</a></sup> analyzed full-length cDNA libraries constructed from budding yeast using the V-capping method and identified novel full-length transcripts, including splicing variants and antisense transcripts.<sup><a reveal-id="DSN010C52" data-open="DSN010C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Even for well-characterized organisms such as yeast, in-depth analysis of the vector-capped cDNA library has been shown to still bring discovery of novel full-length transcripts. To analyze the transcriptome of not only uncharacterized organisms but also well-characterized ones such as human and mouse, the V-capping method will become the first choice for constructing a full-length cDNA library.</p>                            <h2 scrollto-destination=6976864 id="6976864" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6976866 id="6976866" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was partly supported by a grant for the Genome Network Project from the Ministry of Education, Culture, Sports, Science and Technology. Funding for open access charge: Open access charge will be paid by Seishi Kato.</p>                            <h2 scrollto-destination=6976868 id="6976868" class="backacknowledgements-title" >Acknowledgement</h2>
<p>We thank Takumi Nonami for technical assistance for full sequencing of long-sized clones.</p>                            <h2 scrollto-destination=6976870 id="6976870" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN010C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E. S.</div></div>,  <div class="name"><div class="surname">Linton</div> <div class="given-names">L. M.</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.%20S.%20Lander&amp;author=L.%20M.%20Linton&amp;author=B.%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venter</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Adams</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>, et al. </div><div class="article-title">The sequence of the human genome</div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">291</div> (pg. <div class="fpage">1304</div>-<div class="lpage">1351</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20of%20the%20human%20genome&amp;author=J.%20C.%20Venter&amp;author=M.%20D.%20Adams&amp;author=E.%20W.%20Myers&amp;publication_year=2001&amp;journal=Science&amp;volume=291&amp;pages=1304-1351" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1058040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1058040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1058040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11181995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="collab">International Human Genome Sequencing Consortium</div><div class="article-title">Finishing the euchromatic sequence of the human genome</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">431</div> (pg. <div class="fpage">931</div>-<div class="lpage">945</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15496913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sekine</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Oh</div> <div class="given-names">S.-W.</div></div>, et al. </div><div class="article-title">Construction of a human full-length cDNA bank</div>, <div class="source">Gene</div> , <div class="year">1994</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">243</div>-<div class="lpage">250</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20human%20full-length%20cDNA%20bank&amp;author=S.%20Kato&amp;author=S.%20Sekine&amp;author=S.-W.%20Oh&amp;publication_year=1994&amp;journal=Gene&amp;volume=150&amp;pages=243-250" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(94)90433-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(94)90433-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(94)90433-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7821789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kvam</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kitamura</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">High-efficiency full-length cDNA cloning by biotinylated CAP trapper</div>, <div class="source">Genomics</div> , <div class="year">1996</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">327</div>-<div class="lpage">336</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-efficiency%20full-length%20cDNA%20cloning%20by%20biotinylated%20CAP%20trapper&amp;author=P.%20Carninci&amp;author=C.%20Kvam&amp;author=A.%20Kitamura&amp;publication_year=1996&amp;journal=Genomics&amp;volume=37&amp;pages=327-336" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1996.0567" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1996.0567" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1996.0567"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8938445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yoshitomo-Nakagawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Construction and characterization of a full length-enriched and a 5'-end-enriched cDNA library</div>, <div class="source">Gene</div> , <div class="year">1997</div>, vol. <div class="volume">200</div> (pg. <div class="fpage">149</div>-<div class="lpage">156</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20characterization%20of%20a%20full%20length-enriched%20and%20a%205'-end-enriched%20cDNA%20library&amp;author=Y.%20Suzuki&amp;author=K.%20Yoshitomo-Nakagawa&amp;author=K.%20Maruyama&amp;author=A.%20Suyama&amp;author=S.%20Sugano&amp;publication_year=1997&amp;journal=Gene&amp;volume=200&amp;pages=149-156" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(97)00411-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(97)00411-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(97)00411-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9373149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Y. Y.</div></div>,  <div class="name"><div class="surname">Machleder</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Chenchik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Siebert</div> <div class="given-names">P. D.</div></div>. </div><div class="article-title">Reverse transcriptase template switching: a SMART approach for full-length cDNA library construction</div>, <div class="source">Biotechniques</div> , <div class="year">2001</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">892</div>-<div class="lpage">897</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reverse%20transcriptase%20template%20switching%3A%20a%20SMART%20approach%20for%20full-length%20cDNA%20library%20construction&amp;author=Y.%20Y.%20Zhu&amp;author=E.%20M.%20Machleder&amp;author=A.%20Chenchik&amp;author=R.%20Li&amp;author=P.%20D.%20Siebert&amp;publication_year=2001&amp;journal=Biotechniques&amp;volume=30&amp;pages=892-897" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11314272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schena</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shalon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">R. W.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div>. </div><div class="article-title">Quantitative monitoring of gene expression patterns with a complementary DNA microarray</div>, <div class="source">Science</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">467</div>-<div class="lpage">470</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20monitoring%20of%20gene%20expression%20patterns%20with%20a%20complementary%20DNA%20microarray&amp;author=M.%20Schena&amp;author=D.%20Shalon&amp;author=R.%20W.%20Davis&amp;author=P.%20O.%20Brown&amp;publication_year=1995&amp;journal=Science&amp;volume=270&amp;pages=467-470" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.270.5235.467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.270.5235.467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.270.5235.467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7569999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lockhart</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Byrne</div> <div class="given-names">M. C.</div></div>, et al. </div><div class="article-title">Expression monitoring by hybridization to high-density oligonucleotide arrays</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">1996</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1675</div>-<div class="lpage">1680</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20monitoring%20by%20hybridization%20to%20high-density%20oligonucleotide%20arrays&amp;author=D.%20J.%20Lockhart&amp;author=H.%20Dong&amp;author=M.%20C.%20Byrne&amp;publication_year=1996&amp;journal=Nat.%20Biotechnol.&amp;volume=14&amp;pages=1675-1680" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1296-1675" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1296-1675" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1296-1675"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9634850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okubo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hori</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Matoba</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Large scale cDNA sequencing for analysis of quantitative and qualitative aspects of gene expression</div>, <div class="source">Nat. Genet.</div> , <div class="year">1992</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">173</div>-<div class="lpage">179</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20scale%20cDNA%20sequencing%20for%20analysis%20of%20quantitative%20and%20qualitative%20aspects%20of%20gene%20expression&amp;author=K.%20Okubo&amp;author=N.%20Hori&amp;author=R.%20Matoba&amp;publication_year=1992&amp;journal=Nat.%20Genet.&amp;volume=2&amp;pages=173-179" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1192-173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1192-173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1192-173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1345164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Velculescu</div> <div class="given-names">V. E.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Vogelstein</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Kinzler</div> <div class="given-names">K. W.</div></div>. </div><div class="article-title">Serial analysis of gene expression</div>, <div class="source">Science</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">484</div>-<div class="lpage">487</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Serial%20analysis%20of%20gene%20expression&amp;author=V.%20E.%20Velculescu&amp;author=L.%20Zhang&amp;author=B.%20Vogelstein&amp;author=K.%20W.%20Kinzler&amp;publication_year=1995&amp;journal=Science&amp;volume=270&amp;pages=484-487" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.270.5235.484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.270.5235.484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.270.5235.484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7570003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shiraki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kondo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">15776</div>-<div class="lpage">15781</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cap%20analysis%20gene%20expression%20for%20high-throughput%20analysis%20of%20transcriptional%20starting%20point%20and%20identification%20of%20promoter%20usage&amp;author=T.%20Shiraki&amp;author=S.%20Kondo&amp;author=S.%20Katayama&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=15776-15781" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.2136655100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.2136655100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.2136655100"> </span></div> </div></div></div></div></div><div content-id="DSN010C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hashimoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kasai</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">5'-end SAGE for the analysis of transcriptional start sites</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1146</div>-<div class="lpage">1149</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=5'-end%20SAGE%20for%20the%20analysis%20of%20transcriptional%20start%20sites&amp;author=S.%20Hashimoto&amp;author=Y.%20Suzuki&amp;author=Y.%20Kasai&amp;publication_year=2004&amp;journal=Nat.%20Biotechnol.&amp;volume=22&amp;pages=1146-1149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt998" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt998" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt998"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15300261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harbers</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Tag-based approaches for transcriptome research and genome annotation</div>, <div class="source">Nat. Methods</div> , <div class="year">2005</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">495</div>-<div class="lpage">502</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tag-based%20approaches%20for%20transcriptome%20research%20and%20genome%20annotation&amp;author=M.%20Harbers&amp;author=P.%20Carninci&amp;publication_year=2005&amp;journal=Nat.%20Methods&amp;volume=2&amp;pages=495-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15973418" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ota</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Complete sequencing and characterization of 21,243 full-length human cDNAs</div>, <div class="source">Nat. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">40</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20sequencing%20and%20characterization%20of%2021%2C243%20full-length%20human%20cDNAs&amp;author=T.%20Ota&amp;author=Y.%20Suzuki&amp;author=T.%20Nishikawa&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=40-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14702039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Imanishi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Integrative annotation of 21,037 human genes validated by full-length cDNA clones</div>, <div class="source">PLoS Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e162</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20annotation%20of%2021%2C037%20human%20genes%20validated%20by%20full-length%20cDNA%20clones&amp;author=T.%20Imanishi&amp;author=T.%20Itoh&amp;author=Y.%20Suzuki&amp;publication_year=2004&amp;journal=PLoS%20Biol.&amp;volume=2&amp;pages=e162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15103394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gerhard</div> <div class="given-names">D. S.</div></div>,  <div class="name"><div class="surname">Wagner</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Feingold</div> <div class="given-names">E. A.</div></div>, et al. </div><div class="article-title">The status, quality, and expansion of the NIH full-length cDNA project: the Mammalian Gene Collection (MGC)</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2121</div>-<div class="lpage">2127</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20status%2C%20quality%2C%20and%20expansion%20of%20the%20NIH%20full-length%20cDNA%20project%3A%20the%20Mammalian%20Gene%20Collection%20(MGC)&amp;author=D.%20S.%20Gerhard&amp;author=L.%20Wagner&amp;author=E.%20A.%20Feingold&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2121-2127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2596504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2596504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2596504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawamoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yoshii</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">BodyMap: a collection of 3' ESTs for analysis of human gene expression information</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1817</div>-<div class="lpage">1827</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BodyMap%3A%20a%20collection%20of%203'%20ESTs%20for%20analysis%20of%20human%20gene%20expression%20information&amp;author=S.%20Kawamoto&amp;author=J.%20Yoshii&amp;author=K.%20Mizuno&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1817-1827" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.151500" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.151500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.151500"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11076866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Domrachev</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lash</div> <div class="given-names">A. E.</div></div>. </div><div class="article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">207</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20Expression%20Omnibus%3A%20NCBI%20gene%20expression%20and%20hybridization%20array%20data%20repository&amp;author=R.%20Edgar&amp;author=M.%20Domrachev&amp;author=A.%20E.%20Lash&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=207-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brazma</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Parkinson</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sarkans</div> <div class="given-names">U.</div></div>, et al. </div><div class="article-title">ArrayExpress—a public repository for microarray gene expression data at the EBI</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">68</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ArrayExpress%E2%80%94a%20public%20repository%20for%20microarray%20gene%20expression%20data%20at%20the%20EBI&amp;author=A.%20Brazma&amp;author=H.%20Parkinson&amp;author=U.%20Sarkans&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=68-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wakamatsu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Diversification of transcriptional modulation: large-scale identification and characterization of putative alternative promoters of human genes</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">55</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20transcriptional%20modulation%3A%20large-scale%20identification%20and%20characterization%20of%20putative%20alternative%20promoters%20of%20human%20genes&amp;author=K.%20Kimura&amp;author=A.%20Wakamatsu&amp;author=Y.%20Suzuki&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=55-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4039406" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4039406" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4039406"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Taira</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tsunoda</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Diverse transcriptional initiation revealed by fine, large-scale mapping of mRNA start sites</div>, <div class="source">EMBO Rep</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">388</div>-<div class="lpage">393</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diverse%20transcriptional%20initiation%20revealed%20by%20fine%2C%20large-scale%20mapping%20of%20mRNA%20start%20sites&amp;author=Y.%20Suzuki&amp;author=H.%20Taira&amp;author=T.%20Tsunoda&amp;publication_year=2001&amp;journal=EMBO%20Rep&amp;volume=2&amp;pages=388-393" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/embo-reports/kve085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fembo-reports%2Fkve085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fembo-reports%2Fkve085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11375929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Resch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Grasso</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Genome-wide detection of alternative splicing in expressed sequences of human genes</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">2850</div>-<div class="lpage">2859</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20detection%20of%20alternative%20splicing%20in%20expressed%20sequences%20of%20human%20genes&amp;author=B.%20Modrek&amp;author=A.%20Resch&amp;author=C.%20Grasso&amp;author=C.%20Lee&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=2850-2859" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.13.2850" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.13.2850" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.13.2850"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11433032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beaudoing</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Freier</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wyatt</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Gautheret</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Patterns of variant polyadenylation signal usage in human genes</div>, <div class="source">Genome Res</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1001</div>-<div class="lpage">1010</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20variant%20polyadenylation%20signal%20usage%20in%20human%20genes&amp;author=E.%20Beaudoing&amp;author=S.%20Freier&amp;author=J.%20R.%20Wyatt&amp;author=J.%20M.%20Claverie&amp;author=D.%20Gautheret&amp;publication_year=2000&amp;journal=Genome%20Res&amp;volume=10&amp;pages=1001-1010" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.7.1001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.7.1001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.7.1001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10899149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yelin</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dahary</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Widespread occurrence of antisense transcription in the human genome</div>, <div class="source">Nat. Biotechnol</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">379</div>-<div class="lpage">386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20occurrence%20of%20antisense%20transcription%20in%20the%20human%20genome&amp;author=R.%20Yelin&amp;author=D.%20Dahary&amp;author=R.%20Sorek&amp;publication_year=2003&amp;journal=Nat.%20Biotechnol&amp;volume=21&amp;pages=379-386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12640466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kent</div> <div class="given-names">W. J.</div></div>, et al. </div><div class="article-title">Over 20% of human transcripts might form sense-antisense pairs</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">4812</div>-<div class="lpage">4820</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Over%2020%25%20of%20human%20transcripts%20might%20form%20sense-antisense%20pairs&amp;author=J.%20Chen&amp;author=M.%20Sun&amp;author=W.%20J.%20Kent&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=4812-4820" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh818" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh818" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh818"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15356298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuler</div> <div class="given-names">G. D.</div></div>. </div><div class="article-title">Pieces of the puzzle: expressed sequence tags and the catalog of human genes</div>, <div class="source">J. Mol. Med</div> , <div class="year">1997</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">694</div>-<div class="lpage">698</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pieces%20of%20the%20puzzle%3A%20expressed%20sequence%20tags%20and%20the%20catalog%20of%20human%20genes&amp;author=G.%20D.%20Schuler&amp;publication_year=1997&amp;journal=J.%20Mol.%20Med&amp;volume=75&amp;pages=694-698" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001090050155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001090050155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001090050155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9382993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kampa</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kapranov</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22</div>, <div class="source">Genome Res</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">331</div>-<div class="lpage">342</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20RNAs%20identified%20from%20an%20in-depth%20analysis%20of%20the%20transcriptome%20of%20human%20chromosomes%2021%20and%2022&amp;author=D.%20Kampa&amp;author=J.%20Cheng&amp;author=P.%20Kapranov&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=331-342" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2094104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2094104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2094104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14993201" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bertone</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Stolc</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Royce</div> <div class="given-names">T. E.</div></div>, et al. </div><div class="article-title">Global identification of human transcribed sequences with genome tiling arrays</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">306</div> (pg. <div class="fpage">2242</div>-<div class="lpage">2246</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20identification%20of%20human%20transcribed%20sequences%20with%20genome%20tiling%20arrays&amp;author=P.%20Bertone&amp;author=V.%20Stolc&amp;author=T.%20E.%20Royce&amp;publication_year=2004&amp;journal=Science&amp;volume=306&amp;pages=2242-2246" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1103388" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1103388" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1103388"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15539566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ohtoko</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ohtake</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Vector-capping: a simple method for preparing a high-quality full-length cDNA library</div>, <div class="source">DNA Res</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">53</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vector-capping%3A%20a%20simple%20method%20for%20preparing%20a%20high-quality%20full-length%20cDNA%20library&amp;author=S.%20Kato&amp;author=K.%20Ohtoko&amp;author=H.%20Ohtake&amp;author=T.%20Kimura&amp;publication_year=2005&amp;journal=DNA%20Res&amp;volume=12&amp;pages=53-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.1.53" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.1.53" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.1.53"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16106752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okayama</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Berg</div> <div class="given-names">P.</div></div>. </div><div class="article-title">High-efficiency cloning of full-length cDNA</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1982</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">161</div>-<div class="lpage">170</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-efficiency%20cloning%20of%20full-length%20cDNA&amp;author=H.%20Okayama&amp;author=P.%20Berg&amp;publication_year=1982&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=2&amp;pages=161-170" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.2.2.161" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.2.2.161" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.2.2.161"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6287227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuler</div> <div class="given-names">G. D.</div></div>,  <div class="name"><div class="surname">Epstein</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Ohkawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kans</div> <div class="given-names">J. A.</div></div>. </div><div class="article-title">Entrez: molecular biology database and retrieval system</div>, <div class="source">Methods Enzymol</div> , <div class="year">1996</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">141</div>-<div class="lpage">162</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Entrez%3A%20molecular%20biology%20database%20and%20retrieval%20system&amp;author=G.%20D.%20Schuler&amp;author=J.%20A.%20Epstein&amp;author=H.%20Ohkawa&amp;author=J.%20A.%20Kans&amp;publication_year=1996&amp;journal=Methods%20Enzymol&amp;volume=266&amp;pages=141-162" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8743683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K. D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D. R.</div></div>. </div><div class="article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D61</div>-<div class="lpage">D65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequences%20(RefSeq)%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K.%20D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.%20R.%20Maglott&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D61-D65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chao</div> <div class="given-names">A.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Balakrishnan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Read</div> <div class="given-names">C. B.</div></div>,  <div class="name"><div class="surname">Vidakovic</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Species richness estimation</div>, <div class="year">2005</div>, vol. <div class="volume">Vol. 12</div> <div class="publisher-loc">Wiley</div><div class="publisher-name">New York</div>(pg. <div class="fpage">7907</div>-<div class="lpage">7916</div>) <div class="comment">Species Estimation and Applications. Encyclopedia of Statistical Sciences</div></div></div></div></div><div content-id="DSN010C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chao</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">S.-M.</div></div>. </div><div class="article-title">Estimating the number of classes via sample coverage</div>, <div class="source">J. Am. Stat. Assoc</div> , <div class="year">1992</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">210</div>-<div class="lpage">217</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20the%20number%20of%20classes%20via%20sample%20coverage&amp;author=A.%20Chao&amp;author=S.-M.%20Lee&amp;publication_year=1992&amp;journal=J.%20Am.%20Stat.%20Assoc&amp;volume=87&amp;pages=210-217" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/01621459.1992.10475194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F01621459.1992.10475194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F01621459.1992.10475194"> </span></div> </div></div></div></div></div><div content-id="DSN010C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">T.-J.</div></div>,  <div class="name"><div class="surname">Chao</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lina</div> <div class="given-names">C.-F.</div></div>. </div><div class="article-title">Predicting the number of new species in further taxonomic sampling</div>, <div class="source">Ecology</div> , <div class="year">2003</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">798</div>-<div class="lpage">804</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20number%20of%20new%20species%20in%20further%20taxonomic%20sampling&amp;author=T.-J.%20Shen&amp;author=A.%20Chao&amp;author=C.-F.%20Lina&amp;publication_year=2003&amp;journal=Ecology&amp;volume=84&amp;pages=798-804" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1890/0012-9658(2003)084[0798:PTNONS]2.0.CO;2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1890%2F0012-9658(2003)084[0798:PTNONS]2.0.CO;2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1890%2F0012-9658(2003)084[0798:PTNONS]2.0.CO;2"> </span></div> </div></div></div></div></div><div content-id="DSN010C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klemenz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Frohli</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Steiger</div> <div class="given-names">R. H.</div></div>,  <div class="name"><div class="surname">Schafer</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Aoyama</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Alpha B-crystallin is a small heat shock protein</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1991</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">3652</div>-<div class="lpage">3656</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alpha%20B-crystallin%20is%20a%20small%20heat%20shock%20protein&amp;author=R.%20Klemenz&amp;author=E.%20Frohli&amp;author=R.%20H.%20Steiger&amp;author=R.%20Schafer&amp;author=A.%20Aoyama&amp;publication_year=1991&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=3652-3656" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.9.3652" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.9.3652" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.9.3652"> </span></div> </div></div></div></div></div><div content-id="DSN010C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dasgupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hohman</div> <div class="given-names">T. C.</div></div>,  <div class="name"><div class="surname">Carper</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Hypertonic stress induces alpha B-crystallin expression</div>, <div class="source">Exp. Eye Res</div> , <div class="year">1992</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">461</div>-<div class="lpage">470</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypertonic%20stress%20induces%20alpha%20B-crystallin%20expression&amp;author=S.%20Dasgupta&amp;author=T.%20C.%20Hohman&amp;author=D.%20Carper&amp;publication_year=1992&amp;journal=Exp.%20Eye%20Res&amp;volume=54&amp;pages=461-470" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-4835(92)90058-Z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-4835(92)90058-Z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-4835(92)90058-Z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1381680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chao</div> <div class="given-names">C. C.</div></div>,  <div class="name"><div class="surname">Yam</div> <div class="given-names">W. C.</div></div>,  <div class="name"><div class="surname">Lin-Chao</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Coordinated induction of two unrelated glucose-regulated protein genes by a calcium ionophore: human BiP/GRP78 and GAPDH</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">1990</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">431</div>-<div class="lpage">438</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coordinated%20induction%20of%20two%20unrelated%20glucose-regulated%20protein%20genes%20by%20a%20calcium%20ionophore%3A%20human%20BiP%2FGRP78%20and%20GAPDH&amp;author=C.%20C.%20Chao&amp;author=W.%20C.%20Yam&amp;author=S.%20Lin-Chao&amp;publication_year=1990&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=171&amp;pages=431-438" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0006-291X(90)91411-K" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0006-291X(90)91411-K" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0006-291X(90)91411-K"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2118350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nasrin</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ercolani</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Denaro</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kong</div> <div class="given-names">X. F.</div></div>,  <div class="name"><div class="surname">Kang</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Alexander</div> <div class="given-names">M.</div></div>. </div><div class="article-title">An insulin response element in the glyceraldehyde-3-phosphate dehydrogenase gene binds a nuclear protein induced by insulin in cultured cells and by nutritional manipulations in vivo</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1990</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">5273</div>-<div class="lpage">5277</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20insulin%20response%20element%20in%20the%20glyceraldehyde-3-phosphate%20dehydrogenase%20gene%20binds%20a%20nuclear%20protein%20induced%20by%20insulin%20in%20cultured%20cells%20and%20by%20nutritional%20manipulations%20in%20vivo&amp;author=N.%20Nasrin&amp;author=L.%20Ercolani&amp;author=M.%20Denaro&amp;author=X.%20F.%20Kong&amp;author=I.%20Kang&amp;author=M.%20Alexander&amp;publication_year=1990&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=87&amp;pages=5273-5277" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.87.14.5273" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.87.14.5273" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.87.14.5273"> </span></div> </div></div></div></div></div><div content-id="DSN010C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graven</div> <div class="given-names">K. K.</div></div>,  <div class="name"><div class="surname">Troxler</div> <div class="given-names">R. F.</div></div>,  <div class="name"><div class="surname">Kornfeld</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Panchenko</div> <div class="given-names">M. V.</div></div>,  <div class="name"><div class="surname">Farber</div> <div class="given-names">H. W.</div></div>. </div><div class="article-title">Regulation of endothelial cell glyceraldehyde-3-phosphate dehydrogenase expression by hypoxia</div>, <div class="source">J. Biol. Chem</div> , <div class="year">1994</div>, vol. <div class="volume">269</div> (pg. <div class="fpage">24446</div>-<div class="lpage">24453</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20endothelial%20cell%20glyceraldehyde-3-phosphate%20dehydrogenase%20expression%20by%20hypoxia&amp;author=K.%20K.%20Graven&amp;author=R.%20F.%20Troxler&amp;author=H.%20Kornfeld&amp;author=M.%20V.%20Panchenko&amp;author=H.%20W.%20Farber&amp;publication_year=1994&amp;journal=J.%20Biol.%20Chem&amp;volume=269&amp;pages=24446-24453" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7929107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ida</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Boylan</div> <div class="given-names">S. A.</div></div>,  <div class="name"><div class="surname">Weigel</div> <div class="given-names">A. L.</div></div>, et al. </div><div class="article-title">EST analysis of mouse retina and RPE/choroid cDNA libraries</div>, <div class="source">Mol. Vis</div> , <div class="year">2004</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">439</div>-<div class="lpage">444</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST%20analysis%20of%20mouse%20retina%20and%20RPE%2Fchoroid%20cDNA%20libraries&amp;author=H.%20Ida&amp;author=S.%20A.%20Boylan&amp;author=A.%20L.%20Weigel&amp;publication_year=2004&amp;journal=Mol.%20Vis&amp;volume=10&amp;pages=439-444" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15257269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamashita</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wakaguri</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tsuritani</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">DBTSS: DataBase of human transcription start sites, progress report 2006</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D86</div>-<div class="lpage">D89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DBTSS%3A%20DataBase%20of%20human%20transcription%20start%20sites%2C%20progress%20report%202006&amp;author=R.%20Yamashita&amp;author=Y.%20Suzuki&amp;author=H.%20Wakaguri&amp;author=K.%20Tsuritani&amp;author=K.%20Nakai&amp;author=S.%20Sugano&amp;publication_year=2006&amp;book=Nucleic%20Acids%20Res" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Linstedt</div> <div class="given-names">A. D.</div></div>,  <div class="name"><div class="surname">Hauri</div> <div class="given-names">H. P.</div></div>. </div><div class="article-title">Giantin, a novel conserved Golgi membrane protein containing a cytoplasmic domain of at least 350 kDa</div>, <div class="source">Mol. Biol. Cell</div> , <div class="year">1993</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">679</div>-<div class="lpage">693</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Giantin%2C%20a%20novel%20conserved%20Golgi%20membrane%20protein%20containing%20a%20cytoplasmic%20domain%20of%20at%20least%20350%20kDa&amp;author=A.%20D.%20Linstedt&amp;author=H.%20P.%20Hauri&amp;publication_year=1993&amp;journal=Mol.%20Biol.%20Cell&amp;volume=4&amp;pages=679-693" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.4.7.679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.4.7.679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.4.7.679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7691276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trinklein</div> <div class="given-names">N. D.</div></div>,  <div class="name"><div class="surname">Aldred</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Hartman</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Schroeder</div> <div class="given-names">D. I.</div></div>,  <div class="name"><div class="surname">Otillar</div> <div class="given-names">R. P.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">R. M.</div></div>. </div><div class="article-title">An abundance of bidirectional promoters in the human genome</div>, <div class="source">Genome Res</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">62</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20abundance%20of%20bidirectional%20promoters%20in%20the%20human%20genome&amp;author=N.%20D.%20Trinklein&amp;author=S.%20F.%20Aldred&amp;author=S.%20J.%20Hartman&amp;author=D.%20I.%20Schroeder&amp;author=R.%20P.%20Otillar&amp;author=R.%20M.%20Myers&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=62-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1982804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1982804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1982804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14707170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">M. G.</div></div>,  <div class="name"><div class="surname">Kiledjian</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transcript-specific decapping and regulated stability by the human Dcp2 decapping protein</div>, <div class="source">Mol. Cell. Biol</div> , <div class="year">2008</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">939</div>-<div class="lpage">948</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript-specific%20decapping%20and%20regulated%20stability%20by%20the%20human%20Dcp2%20decapping%20protein&amp;author=Y.%20Li&amp;author=M.%20G.%20Song&amp;author=M.%20Kiledjian&amp;publication_year=2008&amp;journal=Mol.%20Cell.%20Biol&amp;volume=28&amp;pages=939-948" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.01727-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.01727-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.01727-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swaroop</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Zack</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Transcriptome analysis of the retina</div>, <div class="source">Genome Biol</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">reviews1022.1</div>-<div class="lpage">1022.4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20the%20retina&amp;author=A.%20Swaroop&amp;author=D.%20J.%20Zack&amp;publication_year=2002&amp;journal=Genome%20Biol&amp;volume=3&amp;pages=reviews1022.1-1022.4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2002-3-8-reviews1022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2002-3-8-reviews1022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2002-3-8-reviews1022"> </span></div> </div></div></div></div></div><div content-id="DSN010C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Del Priore</div> <div class="given-names">L. V.</div></div>. </div><div class="article-title">Gene expression profile of cultured adult compared to immortalized human RPE</div>, <div class="source">Mol. Vis.</div> , <div class="year">2006</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20profile%20of%20cultured%20adult%20compared%20to%20immortalized%20human%20RPE&amp;author=H.%20Cai&amp;author=L.%20V.%20Del%20Priore&amp;publication_year=2006&amp;journal=Mol.%20Vis.&amp;volume=12&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16446697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Genome-wide detection of tissue-specific alternative splicing in the human transcriptome</div>, <div class="source">Nucleic Acids Res.</div> , vol. <div class="volume">30</div> (pg. <div class="fpage">3754</div>-<div class="lpage">3766</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf492" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf492" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf492"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12202761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Metzker</div> <div class="given-names">M. L.</div></div>. </div><div class="article-title">Emerging technologies in DNA sequencing</div>, <div class="source">Genome Res</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1767</div>-<div class="lpage">1776</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emerging%20technologies%20in%20DNA%20sequencing&amp;author=M.%20L.%20Metzker&amp;publication_year=2005&amp;journal=Genome%20Res&amp;volume=15&amp;pages=1767-1776" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3770505" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3770505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3770505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339375" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN010C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN010C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miura</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sese</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A large-scale full-length cDNA analysis to explore the budding yeast transcriptome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">17846</div>-<div class="lpage">17851</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large-scale%20full-length%20cDNA%20analysis%20to%20explore%20the%20budding%20yeast%20transcriptome&amp;author=F.%20Miura&amp;author=N.%20Kawaguchi&amp;author=J.%20Sese&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=17846-17851" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0605645103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0605645103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0605645103"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=464521&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn010-dsn010_Table4.xls?Expires=2147483647&amp;Signature=w5oxw-JaewRVOPHXJ6kXxkwA~rYU2o9S0KHwqsRiQi-OtKbwYwClAM2sOMOHzrObDyTRSfMtMLpU2b8CmpuDTchcn4i8ymF9WRuR0DRLCB775zpLi8IUcVKqgRJqVnbgP9qJMd6CHAXS-CJUwz2yn1D5Zd2zHhQZoyxAZ40fTpD5gAdY-RxW0YD595AQ0pjT1dwhxJ95RMt0YDzvlTmrhFJxl8JrjMdj2VPE5Z9d9EaP6e0XlZu9z8ke8AFPSezB5sWOb6zP13Gwjlo-pakQyH2uVlk8gZkEe1pMSfGPLkF2yhPaz~Mh7ckrEao8RnGlOOD2Eqa1bRj6ciadueJb-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn010-dsn010_Table3.xls?Expires=2147483647&amp;Signature=z5EpfYwHssSI5HJyZk3PCC8eSqjNhTTFQCRqORgaexzKTyciaQ4JpRFcLOh2KtTMEI1cFEr5~oyYGm~iGRHsqjlGkSRTPMH3gB46w6TiT6r2ZVc8nC2FvjnXO189amHxLHqTkCr0vsrgNUZRKY3eZJxyt34YuQHsGXcDozWSVJow322uc8ntRIoluAoVtGa8Vdkujj4BSDFTKkztjC3yYgOrR0N6c6j7eRGQOSUTDNuPYpRo32Ux2r7C6gIZgeJWLaURym1hSMpXene7gOhYje5usRRZERi0r1QpFJQsCWhlPBym7GFXnCBAZ5GxaawUtxHe9xlgLll9cIBSc9bbMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn010-dsn010_Table2.xls?Expires=2147483647&amp;Signature=lXhMP0Lyg-GVSTFBdhpZ53YB~erG9iQZW3dMr1vFfgb--JnSCGcuuPMt9tHk2DUcEGTiX3Zko2kUX-bXnAnqy58PCYzEt4l5nORG4~vONP9XEyUdegpxqTJsb8N1X8q8iwhRYo8qd0~vp74M9cXX~6KsKUK8DvGA~djS0YenhFpadXoLa20ISGRBD0VakY9cZP5h4r8t3Ta0-0vAYhGl~0xsfqF6FcRQ8okwMw~mtLA33cZVoxjAVRlx7zqhdU7n~PE6ZesUhVN1eARlkf1qnMZ5ocJySBZ7s024qmeBqBomI9CHbTnTfkXRndvXxLwduJCx2ZBmRInLH-Z0oTp1OA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn010/2/dsn010-dsn010_Table1.xls?Expires=2147483647&amp;Signature=o6nTwowmHZ9g1FkLm0r8aArJ4GOb~mBmSLrf1VL3W2a32FxGMbHWo5bwhH~dpjsPSNuimYGyfG2yTea04oBB3SQm1SzX1mRy3yuH21Myp0w1ASqBlOly6RBtaJz-jJDaVm0HkQd9NkBD~14YAAFYHCb7gbW-K-3bYSndZexB3N3Mfibk5EKuVbS7xNLptxI54uReDJac44sGm-a5bH~XeN2QQajRJFnjZ73zXlPH6HFago-mphEdkroD~41fEyuMmI7vCgL1ZXQA4daOmL~x0HHTjgFR~BbDo-avS25SRnzYK3lM0HfSdOa4~WLUzmXQboQ07eV~-i1-qcRpOHYQRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">212</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000258424600002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="464521"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000258424600002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f3%2f123%2f464521">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000258424600002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F3%2F123%2F464521" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/464521'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Fine Expression Profiling of Full-length Transcripts using a Size-unbiased cDNA Library Prepared with the Vector-capping Method" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f123%2f464521">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

