

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/5/497/359997","@type":"ScholarlyArticle","name":"Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize","datePublished":"2013-06-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2013-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst026","keywords":["simple sequence repeat","whole-genome sequences","polymorphic SSR markers","teosinte","maize"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/5/497/359997","author":[{"name":"Xu, Jie","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Liu, Ling","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Xu, Yunbi","affiliation":"3  Institute of Crop Sciences, National Key Facilities for Crop Genetic Resources and Improvement ,  Chinese Academy of Agricultural Sciences ,  Beijing 100081 ,  China    4  International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Texcoco ,  Mexico","@type":"Person"},{"name":"Chen, Churun","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Rong, Tingzhao","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Ali, Farhan","affiliation":"5  National Key Laboratory of Crop Genetic Improvement ,  Huazhong Agricultural University ,  Wuhan 430070 ,  China    6  Cereal Crops Research Institute (CCRI) ,  Pirsabak 24100, Nowshera ,  Pakistan","@type":"Person"},{"name":"Zhou, Shufeng","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Wu, Fengkai","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Liu, Yaxi","affiliation":"7  Triticeae Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China","@type":"Person"},{"name":"Wang, Jing","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Cao, Moju","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"},{"name":"Lu, Yanli","affiliation":"1  Maize Research Institute ,  Sichuan Agricultural University ,  Wenjiang 611130, Sichuan ,  China    2  Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region ,  Ministry of Agriculture ,  Wenjiang ,  China","@type":"Person"}],"description":"Abstract.  Simple sequence repeats (SSRs) have been widely used in maize genetics and breeding, because they are co-dominant, easy to score, and highly abundant","pageStart":"497","pageEnd":"509","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize" />
    <meta property="og:description" content="Abstract.  Simple sequence repeats (SSRs) have been widely used in maize genetics and breeding, because they are co-dominant, easy to score, and highly abundant" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/5/497/359997" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-06-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Xu, Jie" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Liu, Ling" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Xu, Yunbi" /><meta name="citation_author_institution" content="Institute of Crop Sciences, National Key Facilities for Crop Genetic Resources and Improvement, Chinese Academy of Agricultural Sciences, Beijing 100081, China" /><meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Texcoco, Mexico" /><meta name="citation_author" content="Chen, Churun" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Rong, Tingzhao" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Ali, Farhan" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author_institution" content="Cereal Crops Research Institute (CCRI), Pirsabak 24100, Nowshera, Pakistan" /><meta name="citation_author" content="Zhou, Shufeng" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Wu, Fengkai" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Liu, Yaxi" /><meta name="citation_author_institution" content="Triticeae Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author" content="Wang, Jing" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Cao, Moju" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_author" content="Lu, Yanli" /><meta name="citation_author_institution" content="Maize Research Institute, Sichuan Agricultural University, Wenjiang 611130, Sichuan, China" /><meta name="citation_author_institution" content="Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region, Ministry of Agriculture, Wenjiang, China" /><meta name="citation_title" content="Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize" /><meta name="citation_firstpage" content="497" /><meta name="citation_lastpage" content="509" /><meta name="citation_doi" content="10.1093/dnares/dst026" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2013/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The B73 maize genome: complexity, diversity, and dynamics; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1112-5; " /><meta name="citation_reference" content="citation_title=Microsatellites: consensus and controversy; Comp. Biochem. Physiol. B Biochem. Mol. Biol.;  citation_year=2000;  citation_volume=126;  citation_pages=455-76; " /><meta name="citation_reference" content="citation_title=Microsatellites: simple sequences with complex evolution; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=435-45; " /><meta name="citation_reference" content="citation_title=Microsatellite marker development, mapping and applications in rice genetics and breeding; Plant Mol. Biol.;  citation_year=1997;  citation_volume=35;  citation_pages=89-99; " /><meta name="citation_reference" content="citation_title=Using next-generation sequencing approaches to isolate simple sequence repeat (SSR) loci in the plant sciences; Am. J. Botany;  citation_year=2012;  citation_volume=99;  citation_pages=193-208; " /><meta name="citation_reference" content="citation_title=The genome-wide determinants of human and chimpanzee microsatellite evolution; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=30-8; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in plants: features and applications; Trends Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=48-55; " /><meta name="citation_reference" content="citation_title=Molecular characterization of global maize breeding germplasm based on genome-wide single nucleotide polymorphisms; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=120;  citation_pages=93-115; " /><meta name="citation_reference" content="citation_title=Genome-wide distribution and organization of microsatellites in plants: an insight into marker development in Brachypodium; PloS One;  citation_year=2011;  citation_volume=6;  citation_pages=e21298" /><meta name="citation_reference" content="citation_title=Abundance of SSR motifs and development of candidate polymorphic SSR markers (BARCSOYSSR_1. 0) in soybean; Crop Sci.;  citation_year=2010;  citation_volume=50;  citation_pages=1950-60; " /><meta name="citation_reference" content="citation_title=Identification of SNP and SSR markers in eggplant using RAD tag sequencing; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=304" /><meta name="citation_reference" content="citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=961-7; " /><meta name="citation_reference" content="citation_title=Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=2117-28; " /><meta name="citation_reference" content="citation_title=Genetic diversity among CIMMYT maize inbred lines investigated with SSR markers; Crop Sci.;  citation_year=2005;  citation_volume=45;  citation_pages=2573-82; " /><meta name="citation_reference" content="citation_title=Genetic diversity in CIMMYT nontemperate maize germplasm: landraces, open pollinated varieties, and inbred lines; Crop Sci.;  citation_year=2008;  citation_volume=48;  citation_pages=617-24; " /><meta name="citation_reference" content="citation_title=SOAP2: an improved ultrafast tool for short read alignment; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1966-7; " /><meta name="citation_reference" content="citation_title=Transcriptome characterization and sequencing-based identification of salt-responsive genes in Millettia pinnata, a semi-mangrove plant; DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=195-207; " /><meta name="citation_reference" content="citation_title=Maize HapMap2 identifies extant variation from a genome in flux; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=803-7; " /><meta name="citation_reference" content="citation_title=An evaluation of the utility of SSR loci as molecular markers in maize (Zea mays L.): comparisons with data from RFLPs and pedigree; Theor. Appl. Genet.;  citation_year=1997;  citation_volume=95;  citation_pages=163-73; " /><meta name="citation_reference" content="citation_title=Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, length variation, transposon associations, and genetic marker potential; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1441-52; " /><meta name="citation_reference" content="citation_title=Sequencing, mapping, and analysis of 27,455 maize full-length cDNAs; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000740" /><meta name="citation_reference" content="citation_title=A genome-wide microsatellite polymorphism database for the indica and japonica rice; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=37-45; " /><meta name="citation_reference" content="citation_title=Distinct patterns of SSR distribution in the Arabidopsis thaliana and rice genomes; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_pages=R14" /><meta name="citation_reference" content="citation_title=Development of a set of public SSR markers derived from genomic sequence of a rapid cycling Brassica oleracea L. genotype; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=117;  citation_pages=977-85; " /><meta name="citation_reference" content="citation_title=Development of genome-wide simple sequence repeat markers using whole-genome shotgun sequences of sorghum (Sorghum bicolor (L.) moench); DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=187-93; " /><meta name="citation_reference" content="citation_title=Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes; Nat. Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=194-200; " /><meta name="citation_reference" content="citation_title=Selection against frameshift mutations limits microsatellite expansion in coding DNA; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=72-80; " /><meta name="citation_reference" content="citation_title=Estimation of sequencing error rates in short reads; BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=185" /><meta name="citation_reference" content="citation_title=Comparative population genomics of maize domestication and improvement; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=808-11; " /><meta name="citation_reference" content="citation_title=Genome-wide patterns of genetic variation among elite maize inbred lines; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=1027-30; " /><meta name="citation_reference" content="citation_title=Genome-wide genetic changes during modern breeding of maize; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=812-5; " /><meta name="citation_reference" content="citation_title=Development and mapping of SSR markers for maize; Plant Mol. Biol.;  citation_year=2002;  citation_volume=48;  citation_pages=463-81; " /><meta name="citation_reference" content="citation_title=Molecular and functional diversity of maize; Curr. Opin. Plant Biol.;  citation_year=2006;  citation_volume=9;  citation_pages=172-6; " /><meta name="citation_reference" content="citation_title=Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize; PloS One;  citation_year=2011;  citation_volume=6;  citation_pages=e24861" /><meta name="citation_reference" content="citation_title=Genome-scale identification of resistance gene analogs and the development of their intron length polymorphism markers in maize; Mol. Breeding;  citation_year=2012;  citation_volume=29;  citation_pages=437-47; " /><meta name="citation_reference" content="citation_title=Genome nucleotide composition shapes variation in simple sequence repeats; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=899-909; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/5/497/1030707/dst026.pdf" /><meta name="description" content="Abstract.  Simple sequence repeats (SSRs) have been widely used in maize genetics and breeding, because they are co-dominant, easy to score, and highly abundant" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/5/497/359997" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/5/497/359997" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/5/497/359997");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst026';
    </script>
<title>
	Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359997" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeFqPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJYdSwgSmllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFpemUgUmVzZWFyY2ggSW5zdGl0dXRlLCBTaWNodWFuIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXZW5qaWFuZyA2MTExMzAsIFNpY2h1YW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBNYWl6ZSBpbiBTb3V0aHdlc3QgUmVnaW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgV2VuamlhbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBMaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFpemUgUmVzZWFyY2ggSW5zdGl0dXRlLCBTaWNodWFuIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXZW5qaWFuZyA2MTExMzAsIFNpY2h1YW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBNYWl6ZSBpbiBTb3V0aHdlc3QgUmVnaW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgV2VuamlhbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWHUsIFl1bmJpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIENyb3AgU2NpZW5jZXMsIE5hdGlvbmFsIEtleSBGYWNpbGl0aWVzIGZvciBDcm9wIEdlbmV0aWMgUmVzb3VyY2VzIGFuZCBJbXByb3ZlbWVudCwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQmVpamluZyAxMDAwODEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW50ZXJuYXRpb25hbCBNYWl6ZSBhbmQgV2hlYXQgSW1wcm92ZW1lbnQgQ2VudGVyIChDSU1NWVQpLCBFbCBCYXRhbiA1NjEzMCwgVGV4Y29jbywgTWV4aWNvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hlbiwgQ2h1cnVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFpemUgUmVzZWFyY2ggSW5zdGl0dXRlLCBTaWNodWFuIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXZW5qaWFuZyA2MTExMzAsIFNpY2h1YW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBNYWl6ZSBpbiBTb3V0aHdlc3QgUmVnaW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgV2VuamlhbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm9uZywgVGluZ3poYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYWl6ZSBSZXNlYXJjaCBJbnN0aXR1dGUsIFNpY2h1YW4gQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFdlbmppYW5nIDYxMTEzMCwgU2ljaHVhbiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBCaW9sb2d5IGFuZCBHZW5ldGljIEltcHJvdmVtZW50IG9mIE1haXplIGluIFNvdXRod2VzdCBSZWdpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBXZW5qaWFuZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBbGksIEZhcmhhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZXJlYWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIChDQ1JJKSwgUGlyc2FiYWsgMjQxMDAsIE5vd3NoZXJhLCBQYWtpc3RhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilpob3UsIFNodWZlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYWl6ZSBSZXNlYXJjaCBJbnN0aXR1dGUsIFNpY2h1YW4gQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFdlbmppYW5nIDYxMTEzMCwgU2ljaHVhbiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBCaW9sb2d5IGFuZCBHZW5ldGljIEltcHJvdmVtZW50IG9mIE1haXplIGluIFNvdXRod2VzdCBSZWdpb24sIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBXZW5qaWFuZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXdSwgRmVuZ2thaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1haXplIFJlc2VhcmNoIEluc3RpdHV0ZSwgU2ljaHVhbiBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV2VuamlhbmcgNjExMTMwLCBTaWNodWFuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEJpb2xvZ3kgYW5kIEdlbmV0aWMgSW1wcm92ZW1lbnQgb2YgTWFpemUgaW4gU291dGh3ZXN0IFJlZ2lvbiwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIFdlbmppYW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgWWF4aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRyaXRpY2VhZSBSZXNlYXJjaCBJbnN0aXR1dGUsIFNpY2h1YW4gQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFdlbmppYW5nIDYxMTEzMCwgU2ljaHVhbiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBKaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFpemUgUmVzZWFyY2ggSW5zdGl0dXRlLCBTaWNodWFuIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXZW5qaWFuZyA2MTExMzAsIFNpY2h1YW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBNYWl6ZSBpbiBTb3V0aHdlc3QgUmVnaW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgV2VuamlhbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FvLCBNb2p1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFpemUgUmVzZWFyY2ggSW5zdGl0dXRlLCBTaWNodWFuIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXZW5qaWFuZyA2MTExMzAsIFNpY2h1YW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBNYWl6ZSBpbiBTb3V0aHdlc3QgUmVnaW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgV2VuamlhbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTHUsIFlhbmxpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFpemUgUmVzZWFyY2ggSW5zdGl0dXRlLCBTaWNodWFuIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXZW5qaWFuZyA2MTExMzAsIFNpY2h1YW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBNYWl6ZSBpbiBTb3V0aHdlc3QgUmVnaW9uLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgV2VuamlhbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZXZlbG9wbWVudCBhbmQgQ2hhcmFjdGVyaXphdGlvbiBvZiBTaW1wbGUgU2VxdWVuY2UgUmVwZWF0IE1hcmtlcnMgUHJvdmlkaW5nIEdlbm9tZS1XaWRlIENvdmVyYWdlIGFuZCBIaWdoIFJlc29sdXRpb24gaW4gTWFpemUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0OTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjUwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEI3MyBtYWl6ZSBnZW5vbWU6IGNvbXBsZXhpdHksIGRpdmVyc2l0eSwgYW5kIGR5bmFtaWNzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNjsgIGNpdGF0aW9uX3BhZ2VzPTExMTItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzOiBjb25zZW5zdXMgYW5kIGNvbnRyb3ZlcnN5OyBDb21wLiBCaW9jaGVtLiBQaHlzaW9sLiBCIEJpb2NoZW0uIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9NDU1LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXM6IHNpbXBsZSBzZXF1ZW5jZXMgd2l0aCBjb21wbGV4IGV2b2x1dGlvbjsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBtYXJrZXIgZGV2ZWxvcG1lbnQsIG1hcHBpbmcgYW5kIGFwcGxpY2F0aW9ucyBpbiByaWNlIGdlbmV0aWNzIGFuZCBicmVlZGluZzsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTg5LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBhcHByb2FjaGVzIHRvIGlzb2xhdGUgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCAoU1NSKSBsb2NpIGluIHRoZSBwbGFudCBzY2llbmNlczsgQW0uIEouIEJvdGFueTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTE5My0yMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUtd2lkZSBkZXRlcm1pbmFudHMgb2YgaHVtYW4gYW5kIGNoaW1wYW56ZWUgbWljcm9zYXRlbGxpdGUgZXZvbHV0aW9uOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTMwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmljIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgaW4gcGxhbnRzOiBmZWF0dXJlcyBhbmQgYXBwbGljYXRpb25zOyBUcmVuZHMgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz00OC01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNoYXJhY3Rlcml6YXRpb24gb2YgZ2xvYmFsIG1haXplIGJyZWVkaW5nIGdlcm1wbGFzbSBiYXNlZCBvbiBnZW5vbWUtd2lkZSBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTkzLTExNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZGlzdHJpYnV0aW9uIGFuZCBvcmdhbml6YXRpb24gb2YgbWljcm9zYXRlbGxpdGVzIGluIHBsYW50czogYW4gaW5zaWdodCBpbnRvIG1hcmtlciBkZXZlbG9wbWVudCBpbiBCcmFjaHlwb2RpdW07IFBsb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjEyOTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnVuZGFuY2Ugb2YgU1NSIG1vdGlmcyBhbmQgZGV2ZWxvcG1lbnQgb2YgY2FuZGlkYXRlIHBvbHltb3JwaGljIFNTUiBtYXJrZXJzIChCQVJDU09ZU1NSXzEuIDApIGluIHNveWJlYW47IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTE5NTAtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFNOUCBhbmQgU1NSIG1hcmtlcnMgaW4gZWdncGxhbnQgdXNpbmcgUkFEIHRhZyBzZXF1ZW5jaW5nOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zMDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIG9mIDE0IGFncm9ub21pYyB0cmFpdHMgaW4gcmljZSBsYW5kcmFjZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTYxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgc3RydWN0dXJlIGFuZCBkaXZlcnNpdHkgYW1vbmcgbWFpemUgaW5icmVkIGxpbmVzIGFzIGluZmVycmVkIGZyb20gRE5BIG1pY3Jvc2F0ZWxsaXRlczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9MjExNy0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgYW1vbmcgQ0lNTVlUIG1haXplIGluYnJlZCBsaW5lcyBpbnZlc3RpZ2F0ZWQgd2l0aCBTU1IgbWFya2VyczsgQ3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MjU3My04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgaW4gQ0lNTVlUIG5vbnRlbXBlcmF0ZSBtYWl6ZSBnZXJtcGxhc206IGxhbmRyYWNlcywgb3BlbiBwb2xsaW5hdGVkIHZhcmlldGllcywgYW5kIGluYnJlZCBsaW5lczsgQ3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NjE3LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQMjogYW4gaW1wcm92ZWQgdWx0cmFmYXN0IHRvb2wgZm9yIHNob3J0IHJlYWQgYWxpZ25tZW50OyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE5NjYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBjaGFyYWN0ZXJpemF0aW9uIGFuZCBzZXF1ZW5jaW5nLWJhc2VkIGlkZW50aWZpY2F0aW9uIG9mIHNhbHQtcmVzcG9uc2l2ZSBnZW5lcyBpbiBNaWxsZXR0aWEgcGlubmF0YSwgYSBzZW1pLW1hbmdyb3ZlIHBsYW50OyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE5NS0yMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1haXplIEhhcE1hcDIgaWRlbnRpZmllcyBleHRhbnQgdmFyaWF0aW9uIGZyb20gYSBnZW5vbWUgaW4gZmx1eDsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz04MDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZhbHVhdGlvbiBvZiB0aGUgdXRpbGl0eSBvZiBTU1IgbG9jaSBhcyBtb2xlY3VsYXIgbWFya2VycyBpbiBtYWl6ZSAoWmVhIG1heXMgTC4pOiBjb21wYXJpc29ucyB3aXRoIGRhdGEgZnJvbSBSRkxQcyBhbmQgcGVkaWdyZWU7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xNjMtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYW5kIGV4cGVyaW1lbnRhbCBhbmFseXNpcyBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTogZnJlcXVlbmN5LCBsZW5ndGggdmFyaWF0aW9uLCB0cmFuc3Bvc29uIGFzc29jaWF0aW9ucywgYW5kIGdlbmV0aWMgbWFya2VyIHBvdGVudGlhbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDQxLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nLCBtYXBwaW5nLCBhbmQgYW5hbHlzaXMgb2YgMjcsNDU1IG1haXplIGZ1bGwtbGVuZ3RoIGNETkFzOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA3NDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIG1pY3Jvc2F0ZWxsaXRlIHBvbHltb3JwaGlzbSBkYXRhYmFzZSBmb3IgdGhlIGluZGljYSBhbmQgamFwb25pY2EgcmljZTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zNy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluY3QgcGF0dGVybnMgb2YgU1NSIGRpc3RyaWJ1dGlvbiBpbiB0aGUgQXJhYmlkb3BzaXMgdGhhbGlhbmEgYW5kIHJpY2UgZ2Vub21lczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1SMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIHNldCBvZiBwdWJsaWMgU1NSIG1hcmtlcnMgZGVyaXZlZCBmcm9tIGdlbm9taWMgc2VxdWVuY2Ugb2YgYSByYXBpZCBjeWNsaW5nIEJyYXNzaWNhIG9sZXJhY2VhIEwuIGdlbm90eXBlOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExNzsgIGNpdGF0aW9uX3BhZ2VzPTk3Ny04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgZ2Vub21lLXdpZGUgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBtYXJrZXJzIHVzaW5nIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNlcXVlbmNlcyBvZiBzb3JnaHVtIChTb3JnaHVtIGJpY29sb3IgKEwuKSBtb2VuY2gpOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzIGFyZSBwcmVmZXJlbnRpYWxseSBhc3NvY2lhdGVkIHdpdGggbm9ucmVwZXRpdGl2ZSBETkEgaW4gcGxhbnQgZ2Vub21lczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xOTQtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24gYWdhaW5zdCBmcmFtZXNoaWZ0IG11dGF0aW9ucyBsaW1pdHMgbWljcm9zYXRlbGxpdGUgZXhwYW5zaW9uIGluIGNvZGluZyBETkE7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NzItODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpb24gb2Ygc2VxdWVuY2luZyBlcnJvciByYXRlcyBpbiBzaG9ydCByZWFkczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcG9wdWxhdGlvbiBnZW5vbWljcyBvZiBtYWl6ZSBkb21lc3RpY2F0aW9uIGFuZCBpbXByb3ZlbWVudDsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz04MDgtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHBhdHRlcm5zIG9mIGdlbmV0aWMgdmFyaWF0aW9uIGFtb25nIGVsaXRlIG1haXplIGluYnJlZCBsaW5lczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMDI3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBnZW5ldGljIGNoYW5nZXMgZHVyaW5nIG1vZGVybiBicmVlZGluZyBvZiBtYWl6ZTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz04MTItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIG1hcHBpbmcgb2YgU1NSIG1hcmtlcnMgZm9yIG1haXplOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NDYzLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYW5kIGZ1bmN0aW9uYWwgZGl2ZXJzaXR5IG9mIG1haXplOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTcyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIFNOUCBhbmQgaGFwbG90eXBlIGFuYWx5c2lzIHJldmVhbHMgYSBoaWdoZXIgZ2VuZXRpYyBkaXZlcnNpdHkgYW5kIHJhcGlkZXIgTEQgZGVjYXkgaW4gdHJvcGljYWwgdGhhbiB0ZW1wZXJhdGUgZ2VybXBsYXNtIGluIG1haXplOyBQbG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTI0ODYxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXNjYWxlIGlkZW50aWZpY2F0aW9uIG9mIHJlc2lzdGFuY2UgZ2VuZSBhbmFsb2dzIGFuZCB0aGUgZGV2ZWxvcG1lbnQgb2YgdGhlaXIgaW50cm9uIGxlbmd0aCBwb2x5bW9ycGhpc20gbWFya2VycyBpbiBtYWl6ZTsgTW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTQzNy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIG51Y2xlb3RpZGUgY29tcG9zaXRpb24gc2hhcGVzIHZhcmlhdGlvbiBpbiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0czsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTg5OS05MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC81LzQ5Ny8xMDMwNzA3L2RzdDAyNi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgU2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgKFNTUnMpIGhhdmUgYmVlbiB3aWRlbHkgdXNlZCBpbiBtYWl6ZSBnZW5ldGljcyBhbmQgYnJlZWRpbmcsIGJlY2F1c2UgdGhleSBhcmUgY28tZG9taW5hbnQsIGVhc3kgdG8gc2NvcmUsIGFuZCBoaWdobHkgYWJ1bmRhbnQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1OTk5NyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQwjzsiB2n2lazJoBc3xI9BHMWUIg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f497%2f359997" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f497%2f359997" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13963" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&amp;Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/5">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5193924">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193924">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193926">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193926">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193933">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193933">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193946">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193946">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193968">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193968">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193974">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193974">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193976">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193976">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5193978">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5193978">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/5/485/359815">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/5/511/360191">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jie Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jie+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jie Xu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ling Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ling+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ling Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yunbi Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Institute of Crop Sciences, National Key Facilities for Crop Genetic Resources and Improvement</div>, <div class="insititution">Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Beijing 100081</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Texcoco</div>, <div class="country">Mexico</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yunbi+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yunbi Xu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Churun Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Churun+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Churun Chen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tingzhao Rong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tingzhao+Rong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rong T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rong T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tingzhao Rong</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Farhan Ali</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">National Key Laboratory of Crop Genetic Improvement</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">6</div><div class="insititution">Cereal Crops Research Institute (CCRI)</div>, <div class="addr-line">Pirsabak 24100, Nowshera</div>, <div class="country">Pakistan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Farhan+Ali">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ali F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ali F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Farhan Ali</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shufeng Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shufeng+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shufeng Zhou</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fengkai Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fengkai+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fengkai Wu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yaxi Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Triticeae Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yaxi+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yaxi Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jing Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jing+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jing Wang</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Moju Cao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Moju+Cao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cao M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cao M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Moju Cao</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanli Lu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Maize Research Institute</div>, <div class="insititution">Sichuan Agricultural University</div>, <div class="addr-line">Wenjiang 611130, Sichuan</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Biology and Genetic Improvement of Maize in Southwest Region</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">Wenjiang</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +86-28-86290916. Fax: +86-28-86290916. E-mail: <a href="mailto:yanli.lu82@hotmail.com" target="_blank">yanli.lu82@hotmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yanli+Lu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lu Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yanli Lu</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">The authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 5, 1 October 2013, Pages 497–509, <a href='https://doi.org/10.1093/dnares/dst026'>https://doi.org/10.1093/dnares/dst026</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 June 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 December 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 May 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359997" href="/dnaresearch/article-pdf/20/5/497/1030707/dst026.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jie Xu, Ling Liu, Yunbi Xu, Churun Chen, Tingzhao Rong, Farhan Ali, Shufeng Zhou, Fengkai Wu, Yaxi Liu, Jing Wang, Moju Cao, Yanli Lu;  Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize, <em>DNA Research</em>, Volume 20, Issue 5, 1 October 2013, Pages 497–509, <a href='https://doi.org/10.1093/dnares/dst026'>https://doi.org/10.1093/dnares/dst026</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359997?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359997?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359997?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359997?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359997?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359997?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359997?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Development%20and%20Characterization%20of%20Simple%20Sequence%20Repeat%20Markers%20Providing%20Genome-Wide%20Coverage%20and%20High%20Resolution%20in%20Maize&amp;publicationDate=2013-06-26&amp;volumeNum=20&amp;issueNum=5&amp;author=Xu%2C%20Jie%3B%20Liu%2C%20Ling&amp;startPage=497&amp;endPage=509&amp;contentId=10.1093%2Fdnares%2Fdst026&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f497%2f359997" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5193924 id="5193924" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Simple sequence repeats (SSRs) have been widely used in maize genetics and breeding, because they are co-dominant, easy to score, and highly abundant. In this study, we used whole-genome sequences from 16 maize inbreds and 1 wild relative to determine SSR abundance and to develop a set of high-density polymorphic SSR markers. A total of 264 658 SSRs were identified across the 17 genomes, with an average of 135 693 SSRs per genome. Marker density was one SSR every of 15.48 kb. (C/G)<em>n</em>, (AT)<em>n</em>, (CAG/CTG)<em>n</em>, and (AAAT/ATTT)<em>n</em> were the most frequent motifs for mono, di-, tri-, and tetra-nucleotide SSRs, respectively. SSRs were most abundant in intergenic region and least frequent in untranslated regions, as revealed by comparing SSR distributions of three representative resequenced genomes. Comparing SSR sequences and e-polymerase chain reaction analysis among the 17 tested genomes created a new database, including 111 887 SSRs, that could be develop as polymorphic markers <em>in silico</em>. Among these markers, 58.00, 26.09, 7.20, 3.00, 3.93, and 1.78% of them had mono, di-, tri-, tetra-, penta-, and hexa-nucleotide motifs, respectively. Polymorphic information content for 35 573 polymorphic SSRs out of 111 887 loci varied from 0.05 to 0.83, with an average of 0.31 in the 17 tested genomes. Experimental validation of polymorphic SSR markers showed that over 70% of the primer pairs could generate the target bands with length polymorphism, and these markers would be very powerful when they are used for genetic populations derived from various types of maize germplasms that were sampled for this study.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;simple sequence repeat&quot;">simple sequence repeat</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome sequences&quot;">whole-genome sequences</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;polymorphic SSR markers&quot;">polymorphic SSR markers</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="teosinte">teosinte</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="maize">maize</a></div>                            <h2 scrollto-destination=5193926 id="5193926" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Maize (<em>Zea mays</em> L.) is one of the most important food, feed, and industrial crops globally and a model system for the study of genetics, evolution, and domestication. The maize genome is large and complex. The estimated total size of genome draft is 2.3 Gb, with over 80% of repeated sequences of various types.<sup><a reveal-id="DST026C1" data-open="DST026C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The genetic variability in the maize genome can be utilized to enhance biotic and abiotic stress tolerance and to improve agronomic traits such as quality, maturity, and yield potential. Types of variation at the whole-genomic level include microsatellites or simple sequence repeats (SSRs), single-nucleotide polymorphisms (SNPs), insertions and deletions (indels), and various types of structure variation.</p><p>SSRs are tandemly repeated mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide sequence motifs flanked by unique sequences.<sup><a reveal-id="DST026C2" data-open="DST026C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST026C3" data-open="DST026C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The unique sequences bordering the SSR motifs provide templates for specific primers to amplify SSR alleles via polymerase chain reaction (PCR), and allelic differences are usually the result of variable numbers of repeat units within a microsatellite structure.<sup><a reveal-id="DST026C4" data-open="DST026C4" class="link link-ref link-reveal xref-bibr">4</a></sup> A larger number of repeated units is generally related to greater genotypic variation, and the shorter motifs such as those with mono-, and di-nucleotides usually possess more repeats than longer motifs such as those with tetra-, penta-, and hexa-nucleotides. However, shorter motifs can produce more slipped-strand mispairing (stuttering) during PCR, which usually lead to genotyping errors.<sup><a reveal-id="DST026C5" data-open="DST026C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST026C6" data-open="DST026C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Based on the repetitive architecture, purity, and complexity of their motifs, SSRs can be classified as perfect (single motif in an uninterrupted array), imperfect, or compound (two or more motifs in interrupted or uninterrupted arrays). As we known, SSR loci with longer or perfect motifs can exhibit a higher level of allelic variability.<sup><a reveal-id="DST026C5" data-open="DST026C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>SSRs have been the genetic markers of choice, because they are easy to score, and have multiallelic nature, co-dominant inheritance, and clear advantages over restriction fragment length polymorphism and amplified fragment length polymorphism markers in terms of technical simplicity, throughput level and automation.<sup><a reveal-id="DST026C7" data-open="DST026C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Compared with SNP markers that are generally biallelic,<sup><a reveal-id="DST026C8" data-open="DST026C8" class="link link-ref link-reveal xref-bibr">8</a></sup> SSR markers are more informative because it can detect multiple alleles per locus, so they are still commonly used nowadays.</p><p>Thanks to the availability of whole-genome or transcriptome sequences in public databases and in the recent advent of bioinformatics tools, development of genetic markers including SSRs has become much easier and more cost-effective. Genetic markers can be obtained by screening genomic, cDNA sequences, or libraries of clones. To facilitate access to and utilization of SSR markers in <em>Brachypodium</em>, 27 329 SSR markers were successfully designed through genome-wide analysis, but only 398 SSR markers have been developed from its bacterial artificial chromosomes end and expressed sequence tag databases.<sup><a reveal-id="DST026C9" data-open="DST026C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The availability of the completed soybean whole-genome sequence also provided an ideal resource for the genome-wide development of locus-specific SSR markers, and 33 065 high-polymorphic SSRs were developed with the availability of their genome positions and primer sequences.<sup><a reveal-id="DST026C10" data-open="DST026C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Barchi <em>et al.</em><sup><a reveal-id="DST026C11" data-open="DST026C11" class="link link-ref link-reveal xref-bibr">11</a></sup> combined the recently developed a restriction site-associated DNA approach with Illumina DNA sequencing to rapidly discover a large number of SNP and SSR markers for eggplant. Huang <em>et al</em>.<sup><a reveal-id="DST026C12" data-open="DST026C12" class="link link-ref link-reveal xref-bibr">12</a></sup> identified over 3.6 million SNPs by sequencing 517 rice landraces, which were used in genome-wide association studies for 14 agronomic traits. These results show that genetic markers such as SSRs and SNPs are abundant in different crop genomes and can be easily scored, making it more accessible to the breeders and geneticists.</p><p>SSRs are abundant and well distributed throughout the maize genome, which can be employed as a preferred marker system. SSR markers have been utilized extensively in maize to characterize the genetic structure and diversity, to construct phylogenetic trees and to define potential heterotic groups, and to identify unique sources of allelic diversity.<sup><a reveal-id="DST026C13  DST026C15" data-open="DST026C13  DST026C15" class="link link-ref link-reveal xref-bibr">13–15</a></sup> Furthermore, SSR markers have been widely used for genetic map construction, quantitative trait locus (QTL) mapping, map-based cloning, and marker-assisted selection (MAS) because of their ubiquity and high level of polymorphism. Hence, enriching the current maize linkage maps with more SSR markers is of great value for the global maize molecular breeding.</p><p>In recent years, many SSR markers have been developed and are publicly available (<a class="link link-uri" href="http://www.maizegdb.org/ssr.php">http://www.maizegdb.org/ssr.php</a>) based on their target sequences among different maize germplasm accessions. However, a relatively low level of polymorphism was observed between cultivated maize and their relatives, and within populations derived from cultivated × teosinte and temperate × tropical maize crosses. The availability of the reference genome sequence and increasingly cost-effective sequencing facilities makes it possible to do whole-genome sequencing for more maize germplasm accessions. We used whole-genome sequence information from 3 typical tropical maize inbreds, 13 typical temperate maize inbreds from different heterotic groups, and 1 teosinte line, to analyse their genetic variation and to develop polymorphic molecular markers that can be used for high-resolution MAS, genomic selection, and QTL mapping. Using germplasm of diverse resources including teosinte and different types of maize lines, we can reveal and utilize unique alleles and loci better. Thus, the objectives of this study were to determine the abundance and characterization of SSRs in the maize genome and to use stringent screening to develop highly polymorphic SSR markers.</p>                            <h2 scrollto-destination=5193933 id="5193933" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5193934 id="5193934" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>Sequence data were generated for 17 genotypes including 16 improved maize inbred lines and 1 wild relative, <em>Z. mays</em> ssp. <em>mexicana</em> (hereafter Z. <em>mexicana</em>), which were listed in Table <a reveal-id="DST026TB1" data-open="DST026TB1" class="link link-reveal link-table xref-fig">1</a>. Among the 16 improved maize inbreds, CML411 and P1 from International Maize and Wheat Improvement Center (CIMMYT) and 81565 from China were chosen to represent tropical/subtropical germplasm, ES40 was derived from traditional Chinese landrace, and the remaining 12 temperate maize inbreds were chosen to represent different heterotic groups in Chinese temperate maize. Temperate maize lines 178, Huangzao4, Ye478, Zheng22, and B73 representing PB, SPT, PA, LRC, and BSSS heterotic groups, respectively, were widely used for commercial hybrid production. For marker validation, additional six teosinte species obtained from United States Department of Agriculture (USDA) and four maize inbreds were also included (Table <a reveal-id="DST026TB1" data-open="DST026TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="5193936" scrollto-destination="5193936"></a>
<div content-id="DST026TB1" class="table-modal"><div class="table-wrap"><div id="DST026TB1" class="table-wrap-title" data-id="DST026TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Maize genotypes used in the study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Pedigree </th><th>Adaptation </th></tr></thead><tbody><tr><td>178<sup>a</sup> </td><td>Selected from an introduced hybrid </td><td>Temperate </td></tr><tr><td>18Red<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>18White<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>48-2<sup>a</sup> </td><td>Synthesized population </td><td>Temperate </td></tr><tr><td>A318 </td><td>Improved S37 </td><td>Tropical </td></tr><tr><td>B73<sup>a</sup> </td><td>BSSS </td><td>Temperate </td></tr><tr><td>Dan598<sup>a</sup> </td><td>(Dan340 × Danhuang11) × (Danhuang02 × Dan599) </td><td>Temperate </td></tr><tr><td>ES40<sup>a</sup> </td><td>Landrace Linshuidadudu selected from Sichuan </td><td>Temperate </td></tr><tr><td>Han21<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>Huangzao4<sup>a</sup> </td><td>Improved from Landrace, TangSiPingTou </td><td>Temperate </td></tr><tr><td>Lu9801 </td><td>Ye502 × H21 </td><td>Temperate </td></tr><tr><td>Mo17<sup>a</sup> </td><td>C103 × 187-2 </td><td>Temperate </td></tr><tr><td>RP125<sup>a</sup> </td><td>Derived from hybrid Chuandan9 </td><td>Temperate </td></tr><tr><td>Ye478<sup>a</sup> </td><td>U8112 × Shen5003 </td><td>Temperate </td></tr><tr><td>Zheng22<sup>a</sup> </td><td>(Duqing × E28) × Lu Jiu Kuan </td><td>Temperate </td></tr><tr><td>81565<sup>a</sup> </td><td>(Huobai × Jin03)S2 × Heibai94 </td><td>Tropical/subtropical </td></tr><tr><td>CML411<sup>a</sup> </td><td>P28C7-S4-#-BBBBBBBBBBB </td><td>Tropical/subtropical </td></tr><tr><td>CML206 </td><td>[EV7992#/EVPO44SRBC3]#BF37SR-2-3SR-2-4-3-BB </td><td>Tropical/subtropical </td></tr><tr><td>CML85 </td><td>P34C5F21-2-#1-2-2-# </td><td>Tropical/subtropical </td></tr><tr><td>P1<sup>a</sup> </td><td>Unknown </td><td>Tropical/subtropical </td></tr><tr><td><em>Z. mexicana</em><sup>a</sup> </td><td><em>Zea mays</em> ssp. <em>mexicana</em> </td><td>Tropical </td></tr><tr><td><em>Z. parviglumis</em> </td><td><em>Zea mays ssp. parviglumis</em> </td><td>Tropical </td></tr><tr><td><em>Z. huehuetenangensis</em> </td><td><em>Zea mays ssp. huehuetenangensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. nicaraguensis</em> </td><td><em>Zea nicaraguensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. luxurians</em> </td><td><em>Zea luxurians</em> </td><td>Tropical </td></tr><tr><td><em>Z. perennis</em> </td><td><em>Zea perennis</em> </td><td>Tropical </td></tr><tr><td><em>Z. diploperennis</em> </td><td><em>Zea diploperennis</em> </td><td>Tropical </td></tr></tbody></table></div><div content-id="DST026TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Pedigree </th><th>Adaptation </th></tr></thead><tbody><tr><td>178<sup>a</sup> </td><td>Selected from an introduced hybrid </td><td>Temperate </td></tr><tr><td>18Red<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>18White<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>48-2<sup>a</sup> </td><td>Synthesized population </td><td>Temperate </td></tr><tr><td>A318 </td><td>Improved S37 </td><td>Tropical </td></tr><tr><td>B73<sup>a</sup> </td><td>BSSS </td><td>Temperate </td></tr><tr><td>Dan598<sup>a</sup> </td><td>(Dan340 × Danhuang11) × (Danhuang02 × Dan599) </td><td>Temperate </td></tr><tr><td>ES40<sup>a</sup> </td><td>Landrace Linshuidadudu selected from Sichuan </td><td>Temperate </td></tr><tr><td>Han21<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>Huangzao4<sup>a</sup> </td><td>Improved from Landrace, TangSiPingTou </td><td>Temperate </td></tr><tr><td>Lu9801 </td><td>Ye502 × H21 </td><td>Temperate </td></tr><tr><td>Mo17<sup>a</sup> </td><td>C103 × 187-2 </td><td>Temperate </td></tr><tr><td>RP125<sup>a</sup> </td><td>Derived from hybrid Chuandan9 </td><td>Temperate </td></tr><tr><td>Ye478<sup>a</sup> </td><td>U8112 × Shen5003 </td><td>Temperate </td></tr><tr><td>Zheng22<sup>a</sup> </td><td>(Duqing × E28) × Lu Jiu Kuan </td><td>Temperate </td></tr><tr><td>81565<sup>a</sup> </td><td>(Huobai × Jin03)S2 × Heibai94 </td><td>Tropical/subtropical </td></tr><tr><td>CML411<sup>a</sup> </td><td>P28C7-S4-#-BBBBBBBBBBB </td><td>Tropical/subtropical </td></tr><tr><td>CML206 </td><td>[EV7992#/EVPO44SRBC3]#BF37SR-2-3SR-2-4-3-BB </td><td>Tropical/subtropical </td></tr><tr><td>CML85 </td><td>P34C5F21-2-#1-2-2-# </td><td>Tropical/subtropical </td></tr><tr><td>P1<sup>a</sup> </td><td>Unknown </td><td>Tropical/subtropical </td></tr><tr><td><em>Z. mexicana</em><sup>a</sup> </td><td><em>Zea mays</em> ssp. <em>mexicana</em> </td><td>Tropical </td></tr><tr><td><em>Z. parviglumis</em> </td><td><em>Zea mays ssp. parviglumis</em> </td><td>Tropical </td></tr><tr><td><em>Z. huehuetenangensis</em> </td><td><em>Zea mays ssp. huehuetenangensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. nicaraguensis</em> </td><td><em>Zea nicaraguensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. luxurians</em> </td><td><em>Zea luxurians</em> </td><td>Tropical </td></tr><tr><td><em>Z. perennis</em> </td><td><em>Zea perennis</em> </td><td>Tropical </td></tr><tr><td><em>Z. diploperennis</em> </td><td><em>Zea diploperennis</em> </td><td>Tropical </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All the materials were used for experimental validation.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Materials were only used for SSR identification and markers development. The chromosome number is 2<em>n</em> = 40 for <em>Z. perennis</em> and 2<em>n</em> = 20 for other species.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193936" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST026TB1" class="table-wrap-title" data-id="DST026TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Maize genotypes used in the study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Pedigree </th><th>Adaptation </th></tr></thead><tbody><tr><td>178<sup>a</sup> </td><td>Selected from an introduced hybrid </td><td>Temperate </td></tr><tr><td>18Red<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>18White<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>48-2<sup>a</sup> </td><td>Synthesized population </td><td>Temperate </td></tr><tr><td>A318 </td><td>Improved S37 </td><td>Tropical </td></tr><tr><td>B73<sup>a</sup> </td><td>BSSS </td><td>Temperate </td></tr><tr><td>Dan598<sup>a</sup> </td><td>(Dan340 × Danhuang11) × (Danhuang02 × Dan599) </td><td>Temperate </td></tr><tr><td>ES40<sup>a</sup> </td><td>Landrace Linshuidadudu selected from Sichuan </td><td>Temperate </td></tr><tr><td>Han21<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>Huangzao4<sup>a</sup> </td><td>Improved from Landrace, TangSiPingTou </td><td>Temperate </td></tr><tr><td>Lu9801 </td><td>Ye502 × H21 </td><td>Temperate </td></tr><tr><td>Mo17<sup>a</sup> </td><td>C103 × 187-2 </td><td>Temperate </td></tr><tr><td>RP125<sup>a</sup> </td><td>Derived from hybrid Chuandan9 </td><td>Temperate </td></tr><tr><td>Ye478<sup>a</sup> </td><td>U8112 × Shen5003 </td><td>Temperate </td></tr><tr><td>Zheng22<sup>a</sup> </td><td>(Duqing × E28) × Lu Jiu Kuan </td><td>Temperate </td></tr><tr><td>81565<sup>a</sup> </td><td>(Huobai × Jin03)S2 × Heibai94 </td><td>Tropical/subtropical </td></tr><tr><td>CML411<sup>a</sup> </td><td>P28C7-S4-#-BBBBBBBBBBB </td><td>Tropical/subtropical </td></tr><tr><td>CML206 </td><td>[EV7992#/EVPO44SRBC3]#BF37SR-2-3SR-2-4-3-BB </td><td>Tropical/subtropical </td></tr><tr><td>CML85 </td><td>P34C5F21-2-#1-2-2-# </td><td>Tropical/subtropical </td></tr><tr><td>P1<sup>a</sup> </td><td>Unknown </td><td>Tropical/subtropical </td></tr><tr><td><em>Z. mexicana</em><sup>a</sup> </td><td><em>Zea mays</em> ssp. <em>mexicana</em> </td><td>Tropical </td></tr><tr><td><em>Z. parviglumis</em> </td><td><em>Zea mays ssp. parviglumis</em> </td><td>Tropical </td></tr><tr><td><em>Z. huehuetenangensis</em> </td><td><em>Zea mays ssp. huehuetenangensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. nicaraguensis</em> </td><td><em>Zea nicaraguensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. luxurians</em> </td><td><em>Zea luxurians</em> </td><td>Tropical </td></tr><tr><td><em>Z. perennis</em> </td><td><em>Zea perennis</em> </td><td>Tropical </td></tr><tr><td><em>Z. diploperennis</em> </td><td><em>Zea diploperennis</em> </td><td>Tropical </td></tr></tbody></table></div><div content-id="DST026TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Pedigree </th><th>Adaptation </th></tr></thead><tbody><tr><td>178<sup>a</sup> </td><td>Selected from an introduced hybrid </td><td>Temperate </td></tr><tr><td>18Red<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>18White<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>48-2<sup>a</sup> </td><td>Synthesized population </td><td>Temperate </td></tr><tr><td>A318 </td><td>Improved S37 </td><td>Tropical </td></tr><tr><td>B73<sup>a</sup> </td><td>BSSS </td><td>Temperate </td></tr><tr><td>Dan598<sup>a</sup> </td><td>(Dan340 × Danhuang11) × (Danhuang02 × Dan599) </td><td>Temperate </td></tr><tr><td>ES40<sup>a</sup> </td><td>Landrace Linshuidadudu selected from Sichuan </td><td>Temperate </td></tr><tr><td>Han21<sup>a</sup> </td><td>American hybrid P78599 </td><td>Temperate </td></tr><tr><td>Huangzao4<sup>a</sup> </td><td>Improved from Landrace, TangSiPingTou </td><td>Temperate </td></tr><tr><td>Lu9801 </td><td>Ye502 × H21 </td><td>Temperate </td></tr><tr><td>Mo17<sup>a</sup> </td><td>C103 × 187-2 </td><td>Temperate </td></tr><tr><td>RP125<sup>a</sup> </td><td>Derived from hybrid Chuandan9 </td><td>Temperate </td></tr><tr><td>Ye478<sup>a</sup> </td><td>U8112 × Shen5003 </td><td>Temperate </td></tr><tr><td>Zheng22<sup>a</sup> </td><td>(Duqing × E28) × Lu Jiu Kuan </td><td>Temperate </td></tr><tr><td>81565<sup>a</sup> </td><td>(Huobai × Jin03)S2 × Heibai94 </td><td>Tropical/subtropical </td></tr><tr><td>CML411<sup>a</sup> </td><td>P28C7-S4-#-BBBBBBBBBBB </td><td>Tropical/subtropical </td></tr><tr><td>CML206 </td><td>[EV7992#/EVPO44SRBC3]#BF37SR-2-3SR-2-4-3-BB </td><td>Tropical/subtropical </td></tr><tr><td>CML85 </td><td>P34C5F21-2-#1-2-2-# </td><td>Tropical/subtropical </td></tr><tr><td>P1<sup>a</sup> </td><td>Unknown </td><td>Tropical/subtropical </td></tr><tr><td><em>Z. mexicana</em><sup>a</sup> </td><td><em>Zea mays</em> ssp. <em>mexicana</em> </td><td>Tropical </td></tr><tr><td><em>Z. parviglumis</em> </td><td><em>Zea mays ssp. parviglumis</em> </td><td>Tropical </td></tr><tr><td><em>Z. huehuetenangensis</em> </td><td><em>Zea mays ssp. huehuetenangensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. nicaraguensis</em> </td><td><em>Zea nicaraguensis</em> </td><td>Tropical </td></tr><tr><td><em>Z. luxurians</em> </td><td><em>Zea luxurians</em> </td><td>Tropical </td></tr><tr><td><em>Z. perennis</em> </td><td><em>Zea perennis</em> </td><td>Tropical </td></tr><tr><td><em>Z. diploperennis</em> </td><td><em>Zea diploperennis</em> </td><td>Tropical </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All the materials were used for experimental validation.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Materials were only used for SSR identification and markers development. The chromosome number is 2<em>n</em> = 40 for <em>Z. perennis</em> and 2<em>n</em> = 20 for other species.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193936" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5193937 id="5193937" class="section-title" data-legacy-id=s2b>2.2. Maize genome sequences</h3>
<p>For maize lines 81565, 18Red, 18White, 48-2, Dan598, ES40, RP125, and Z. <em>mexicana</em>, sequences were generated, and paired-end libraries were constructed according to the Illumina manufacturer's instructions. An average resequencing depth was 13× and genome coverage was 85% for maize inbreds. One teosinte species <em>Mexican</em> was sequenced with an average of resequencing depth of 9× and genome coverage of 74%. The genome sequences for the remaining maize lines were downloaded from the NCBI Sequence Read Archive (SRA) database (SRA049859 and SRA051245) and NCBI GenBank (JQ886798–JQ887980). All sequence reads were aligned against the maize B73 reference genome (<a class="link link-uri" href="http://www.maizesequence.org">www.maizesequence.org</a> Release 4a.53) using Short Oligonucleotide Alignment Program 2 (<a class="link link-uri" href="http://soap.genomics.org.cn/">http://soap.genomics.org.cn/</a>). Sequencing and reads mapping were carried out at Beijing Genomics Institute (Shenzhen, China).<sup><a reveal-id="DST026C16  DST026C18" data-open="DST026C16  DST026C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup></p>                            <h3 scrollto-destination=5193939 id="5193939" class="section-title" data-legacy-id=s2c>2.3. SSR identification and primer design</h3>
<p>SSR motifs were identified in 17 genomes using MISA (MIcroSAtellite identification tool) program downloaded from the Leibniz Institute of Plant Genetics and Crop Plant Research website (<a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>). Only perfect SSRs including mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide motifs with numbers of uninterrupted repeat units more than 10, 7, 6, 5, 4, and 4, respectively, were targeted. The 5′- and 3′-untranslated regions (UTR), protein coding sequence (CDS), intron, and intergenic regions were determined based on their original annotations of the maize B73 reference genome (<a class="link link-uri" href="http://www.maizesequence.org">www.maizesequence.org</a> Release 4a.53). Promoter sequences were determined at 2 kb upstream of the transcription initiation site.</p><p>Any SSR locus to be used to develop genetic markers should include a perfect repeat motif and two unique flanking sequences with 300 bp on each sides of the repeat. In our study, SSR candidate sequences were used for BLASTN search against the genome sequences (<em>e</em>-value cut-off of 1<em>e</em><sup>−10</sup>), and filtered with &gt;90% of identity and minimum alignment length with &gt;85% of the flanking sequences. Those with unique hit, together with their specific flanking sequences, were identified as candidate SSR loci. Then, we wrote a Perl script to combine SSRs within 5 kb of different genomes with the same motif and to identify polymorphic SSR loci among 17 genotypes depending on the presence of motifs.</p><p>The forward and reverse primers were designed based on unique flanking sequences using Primer 3 (<a class="link link-uri" href="http://primer3.sourceforge.net/">http://primer3.sourceforge.net/</a>). Input parameters for the primer design were as follows: minimum, maximum, and optimal sizes were 18, 27, and 20 nt; minimum and maximum GC were 20 and 80%; and minimum, maximum, and optimal <em>T</em><sub>m</sub> were 57, 63, and 60°C, respectively. The deviation of amplicon size of each SSR primer ranged from 30 to 500 bp based on the expected SSR sequence length.</p><p>In addition, electronic polymerase chain reaction (e-PCR) programme (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/projects/e-pcr/">http://www.ncbi.nlm.nih.gov/projects/e-pcr/</a>) was applied to check the uniqueness and specificity of designed primers in the genomes. The parameters were set as following: the word size was 9, the discontiguous word was 1, the maximal allowed deviation of hit product size was 100, the maximum mismatches allowed, and the maximum indels allowed were 1, respectively. On the other hand, the published SSR markers reposited in MaizeGDB (<a class="link link-uri" href="http://www.maizegdb.org/">http://www.maizegdb.org/</a>) were downloaded and amplified <em>in silicon</em> through e-PCR programme for further comparison.</p>                            <h3 scrollto-destination=5193944 id="5193944" class="section-title" data-legacy-id=s2d>2.4. Experimental validation of polymorphic SSR markers</h3>
<p>To assess the value of identified SSR markers, 151 primer pairs from 10 chromosomes including all types of SSR were chosen for experimental validation. The samples used in this experiment included 20 improved maize lines and 7 teosinte lines (Table <a reveal-id="DST026TB1" data-open="DST026TB1" class="link link-reveal link-table xref-fig">1</a>). Genomic DNA was extracted from seedlings using the CTAB method. Primers were made by Shanghai DNA Biotechnologies Co., Ltd. PCR was performed in 25 μl reactions containing 2.5 μl buffer, 2.5 μl MgCl<sub>2</sub> (25 mM), 4.0 μl dNTP (2.5 mM), 0.2 μl Taq polymerase (5 U/μl), 1 μl template DNA (100 ng/μl),13.8 μl ddH<sub>2</sub>O, and 0.1 μg primers. The PCR conditions were as follows: 1 cycle at 94°C for 5 min; 35 cycles at 94°C for 30 s, 60°C for 30 s, 72°C for 1 min, and 1 cycle at 72°C for 10 min. PCR products mixed with loading buffer were heated at 95°C for 5 min and quickly chilled on ice. The entire mixture was electrophoresed on 6% denaturing polyacrylamide gel, and the genotype was scored after silver staining. The number of alleles was recorded and the polymorphism information content (PIC) was calculated as described by Smith <em>et al.</em><sup><a reveal-id="DST026C19" data-open="DST026C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                            <h2 scrollto-destination=5193946 id="5193946" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5193947 id="5193947" class="section-title" data-legacy-id=s3a>3.1. The abundance of SSRs in the maize genome</h3>
<p>A large number of perfect SSRs with mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide motifs were identified, but the numbers varied among different genomes (Table <a reveal-id="DST026TB2" data-open="DST026TB2" class="link link-reveal link-table xref-fig">2</a>). The average number of SSRs was 135 693 in 17 genotypes, ranging from 133 346 loci observed in <em>mexicana</em> to 136 723 loci in tropical/subtropical maize inbred 81565. Some reads from Z. <em>mexicana</em> could not be mapped onto the reference genome, which resulted in relatively lower genome coverage and thus, less SSRs identified compared with other maize inbreds. A total of 264 658 unique SSR loci were detected in 17 genomes, of which mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs were 153 231, 65 236, 25 910, 6572, 8839, and 4870, respectively. The mono-nucleotide motif is the most abundant, accounted for 57.90%. There were 38 971 common SSRs (15% of the total) observed to be the same across 17 genotypes. The SSR density was calculated based on the maize reference genome size of 2.1 Gb, and there was a little difference among 17 genotypes for each nucleotide motif, with an average interval of 15.48 kb between SSR loci for every genome. However, the average intervals for mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs were remarkably different, which were 26.93, 60.88, 150.95, 717.70, 505.05, and 942.55 kb, respectively (Table <a reveal-id="DST026TB2" data-open="DST026TB2" class="link link-reveal link-table xref-fig">2</a>). SSRs were considerably abundant and distributed throughout the maize genome, with a small average marker interval (7.93 kb) for all detected loci. </p>                        <a id="5193949" scrollto-destination="5193949"></a>
<div content-id="DST026TB2" class="table-modal"><div class="table-wrap"><div id="DST026TB2" class="table-wrap-title" data-id="DST026TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Numbers and density of SSR loci identified in 17 maize genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Genotypes </th><th colspan="7">SSR numbers<hr /> </th><th colspan="7">SSR interval (kb)<hr /> </th></tr><tr><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th></tr></thead><tbody><tr><td>178 </td><td>78 367 </td><td>34 604 </td><td>13 929 </td><td>2 964 </td><td>4172 </td><td>2226 </td><td>136 262 </td><td>26.80 </td><td>60.69 </td><td>150.76 </td><td>708.50 </td><td>503.36 </td><td>943.40 </td><td>15.41 </td></tr><tr><td>81565 </td><td>79 042 </td><td>34 457 </td><td>13 893 </td><td>2929 </td><td>4178 </td><td>2224 </td><td>136 723 </td><td>26.57 </td><td>60.95 </td><td>151.16 </td><td>716.97 </td><td>502.63 </td><td>944.24 </td><td>15.36 </td></tr><tr><td>18White </td><td>77 144 </td><td>34 451 </td><td>13 931 </td><td>2928 </td><td>4166 </td><td>2217 </td><td>134 837 </td><td>27.22 </td><td>60.96 </td><td>150.74 </td><td>717.21 </td><td>504.08 </td><td>947.23 </td><td>15.57 </td></tr><tr><td>18Red </td><td>77 049 </td><td>34 495 </td><td>13 919 </td><td>2933 </td><td>4188 </td><td>2221 </td><td>134 805 </td><td>27.26 </td><td>60.88 </td><td>150.87 </td><td>715.99 </td><td>501.43 </td><td>945.52 </td><td>15.58 </td></tr><tr><td>48-2 </td><td>76 920 </td><td>34 448 </td><td>13 884 </td><td>2924 </td><td>4123 </td><td>2219 </td><td>134 518 </td><td>27.30 </td><td>60.96 </td><td>151.25 </td><td>718.19 </td><td>509.34 </td><td>946.37 </td><td>15.61 </td></tr><tr><td>B73 </td><td>77 888 </td><td>34 755 </td><td>14 028 </td><td>2948 </td><td>4181 </td><td>2239 </td><td>136 039 </td><td>26.96 </td><td>60.42 </td><td>149.70 </td><td>712.35 </td><td>502.27 </td><td>937.92 </td><td>15.44 </td></tr><tr><td>CML411 </td><td>78 591 </td><td>34 546 </td><td>13 900 </td><td>2935 </td><td>4189 </td><td>2225 </td><td>136 386 </td><td>26.72 </td><td>60.79 </td><td>151.08 </td><td>715.50 </td><td>501.31 </td><td>943.82 </td><td>15.40 </td></tr><tr><td>Dan598 </td><td>78 558 </td><td>34 559 </td><td>13 954 </td><td>2904 </td><td>4159 </td><td>2227 </td><td>136 361 </td><td>26.73 </td><td>60.77 </td><td>150.49 </td><td>723.14 </td><td>504.93 </td><td>942.97 </td><td>15.40 </td></tr><tr><td>ES40 </td><td>78 978 </td><td>34 372 </td><td>13 894 </td><td>2924 </td><td>4161 </td><td>2221 </td><td>136 550 </td><td>26.59 </td><td>61.10 </td><td>151.14 </td><td>718.19 </td><td>504.69 </td><td>945.52 </td><td>15.38 </td></tr><tr><td>Han21 </td><td>78 539 </td><td>34 584 </td><td>13 951 </td><td>2898 </td><td>4157 </td><td>2233 </td><td>136 362 </td><td>26.74 </td><td>60.72 </td><td>150.53 </td><td>724.64 </td><td>505.17 </td><td>940.44 </td><td>15.40 </td></tr><tr><td>Huangzao4 </td><td>76 773 </td><td>34 445 </td><td>13 878 </td><td>2907 </td><td>4165 </td><td>2224 </td><td>134 392 </td><td>27.35 </td><td>60.97 </td><td>151.32 </td><td>722.39 </td><td>504.20 </td><td>944.24 </td><td>15.63 </td></tr><tr><td>Mo17 </td><td>78 360 </td><td>34 442 </td><td>13 910 </td><td>2902 </td><td>4108 </td><td>2220 </td><td>135 942 </td><td>26.80 </td><td>60.97 </td><td>150.97 </td><td>723.64 </td><td>511.20 </td><td>945.95 </td><td>15.45 </td></tr><tr><td>P1 </td><td>78 975 </td><td>34 447 </td><td>13 849 </td><td>2961 </td><td>4146 </td><td>2235 </td><td>136 613 </td><td>26.59 </td><td>60.96 </td><td>151.64 </td><td>709.22 </td><td>506.51 </td><td>939.60 </td><td>15.37 </td></tr><tr><td>RP125 </td><td>76 880 </td><td>34 523 </td><td>13 934 </td><td>2929 </td><td>4198 </td><td>2244 </td><td>134 708 </td><td>27.32 </td><td>60.83 </td><td>150.71 </td><td>716.97 </td><td>500.24 </td><td>935.83 </td><td>15.59 </td></tr><tr><td><em>Z. mexicana</em> </td><td>75 997 </td><td>34 339 </td><td>13 806 </td><td>2886 </td><td>4085 </td><td>2233 </td><td>133 346 </td><td>27.63 </td><td>61.15 </td><td>152.11 </td><td>727.65 </td><td>514.08 </td><td>940.44 </td><td>15.75 </td></tr><tr><td>Ye478 </td><td>78 571 </td><td>34 508 </td><td>13 973 </td><td>2944 </td><td>4159 </td><td>2234 </td><td>136 389 </td><td>26.73 </td><td>60.86 </td><td>150.29 </td><td>713.32 </td><td>504.93 </td><td>940.02 </td><td>15.40 </td></tr><tr><td>Zheng22 </td><td>78 921 </td><td>34 455 </td><td>13 870 </td><td>2920 </td><td>4156 </td><td>2230 </td><td>136 552 </td><td>26.61 </td><td>60.95 </td><td>151.41 </td><td>719.18 </td><td>505.29 </td><td>941.70 </td><td>15.38 </td></tr><tr><td>Average </td><td>77 974 </td><td>34 496 </td><td>13 912 </td><td>2926 </td><td>4158 </td><td>2228 </td><td>135 693 </td><td>26.93 </td><td>60.88 </td><td>150.95 </td><td>717.70 </td><td>505.05 </td><td>942.55 </td><td>15.48 </td></tr><tr><td>Total </td><td>153 231 </td><td>65 236 </td><td>25 910 </td><td>6572 </td><td>8839 </td><td>4870 </td><td>264 658 </td><td>13.70 </td><td>32.19 </td><td>81.05 </td><td>319.54 </td><td>237.58 </td><td>431.21 </td><td>7.93 </td></tr><tr><td>Common </td><td>22 453 </td><td>9963 </td><td>4603 </td><td>553 </td><td>929 </td><td>470 </td><td>38 971 </td><td>93.53 </td><td>210.78 </td><td>456.22 </td><td>3797.47 </td><td>2260.50 </td><td>4468.09 </td><td>53.89 </td></tr></tbody></table></div><div content-id="DST026TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Genotypes </th><th colspan="7">SSR numbers<hr /> </th><th colspan="7">SSR interval (kb)<hr /> </th></tr><tr><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th></tr></thead><tbody><tr><td>178 </td><td>78 367 </td><td>34 604 </td><td>13 929 </td><td>2 964 </td><td>4172 </td><td>2226 </td><td>136 262 </td><td>26.80 </td><td>60.69 </td><td>150.76 </td><td>708.50 </td><td>503.36 </td><td>943.40 </td><td>15.41 </td></tr><tr><td>81565 </td><td>79 042 </td><td>34 457 </td><td>13 893 </td><td>2929 </td><td>4178 </td><td>2224 </td><td>136 723 </td><td>26.57 </td><td>60.95 </td><td>151.16 </td><td>716.97 </td><td>502.63 </td><td>944.24 </td><td>15.36 </td></tr><tr><td>18White </td><td>77 144 </td><td>34 451 </td><td>13 931 </td><td>2928 </td><td>4166 </td><td>2217 </td><td>134 837 </td><td>27.22 </td><td>60.96 </td><td>150.74 </td><td>717.21 </td><td>504.08 </td><td>947.23 </td><td>15.57 </td></tr><tr><td>18Red </td><td>77 049 </td><td>34 495 </td><td>13 919 </td><td>2933 </td><td>4188 </td><td>2221 </td><td>134 805 </td><td>27.26 </td><td>60.88 </td><td>150.87 </td><td>715.99 </td><td>501.43 </td><td>945.52 </td><td>15.58 </td></tr><tr><td>48-2 </td><td>76 920 </td><td>34 448 </td><td>13 884 </td><td>2924 </td><td>4123 </td><td>2219 </td><td>134 518 </td><td>27.30 </td><td>60.96 </td><td>151.25 </td><td>718.19 </td><td>509.34 </td><td>946.37 </td><td>15.61 </td></tr><tr><td>B73 </td><td>77 888 </td><td>34 755 </td><td>14 028 </td><td>2948 </td><td>4181 </td><td>2239 </td><td>136 039 </td><td>26.96 </td><td>60.42 </td><td>149.70 </td><td>712.35 </td><td>502.27 </td><td>937.92 </td><td>15.44 </td></tr><tr><td>CML411 </td><td>78 591 </td><td>34 546 </td><td>13 900 </td><td>2935 </td><td>4189 </td><td>2225 </td><td>136 386 </td><td>26.72 </td><td>60.79 </td><td>151.08 </td><td>715.50 </td><td>501.31 </td><td>943.82 </td><td>15.40 </td></tr><tr><td>Dan598 </td><td>78 558 </td><td>34 559 </td><td>13 954 </td><td>2904 </td><td>4159 </td><td>2227 </td><td>136 361 </td><td>26.73 </td><td>60.77 </td><td>150.49 </td><td>723.14 </td><td>504.93 </td><td>942.97 </td><td>15.40 </td></tr><tr><td>ES40 </td><td>78 978 </td><td>34 372 </td><td>13 894 </td><td>2924 </td><td>4161 </td><td>2221 </td><td>136 550 </td><td>26.59 </td><td>61.10 </td><td>151.14 </td><td>718.19 </td><td>504.69 </td><td>945.52 </td><td>15.38 </td></tr><tr><td>Han21 </td><td>78 539 </td><td>34 584 </td><td>13 951 </td><td>2898 </td><td>4157 </td><td>2233 </td><td>136 362 </td><td>26.74 </td><td>60.72 </td><td>150.53 </td><td>724.64 </td><td>505.17 </td><td>940.44 </td><td>15.40 </td></tr><tr><td>Huangzao4 </td><td>76 773 </td><td>34 445 </td><td>13 878 </td><td>2907 </td><td>4165 </td><td>2224 </td><td>134 392 </td><td>27.35 </td><td>60.97 </td><td>151.32 </td><td>722.39 </td><td>504.20 </td><td>944.24 </td><td>15.63 </td></tr><tr><td>Mo17 </td><td>78 360 </td><td>34 442 </td><td>13 910 </td><td>2902 </td><td>4108 </td><td>2220 </td><td>135 942 </td><td>26.80 </td><td>60.97 </td><td>150.97 </td><td>723.64 </td><td>511.20 </td><td>945.95 </td><td>15.45 </td></tr><tr><td>P1 </td><td>78 975 </td><td>34 447 </td><td>13 849 </td><td>2961 </td><td>4146 </td><td>2235 </td><td>136 613 </td><td>26.59 </td><td>60.96 </td><td>151.64 </td><td>709.22 </td><td>506.51 </td><td>939.60 </td><td>15.37 </td></tr><tr><td>RP125 </td><td>76 880 </td><td>34 523 </td><td>13 934 </td><td>2929 </td><td>4198 </td><td>2244 </td><td>134 708 </td><td>27.32 </td><td>60.83 </td><td>150.71 </td><td>716.97 </td><td>500.24 </td><td>935.83 </td><td>15.59 </td></tr><tr><td><em>Z. mexicana</em> </td><td>75 997 </td><td>34 339 </td><td>13 806 </td><td>2886 </td><td>4085 </td><td>2233 </td><td>133 346 </td><td>27.63 </td><td>61.15 </td><td>152.11 </td><td>727.65 </td><td>514.08 </td><td>940.44 </td><td>15.75 </td></tr><tr><td>Ye478 </td><td>78 571 </td><td>34 508 </td><td>13 973 </td><td>2944 </td><td>4159 </td><td>2234 </td><td>136 389 </td><td>26.73 </td><td>60.86 </td><td>150.29 </td><td>713.32 </td><td>504.93 </td><td>940.02 </td><td>15.40 </td></tr><tr><td>Zheng22 </td><td>78 921 </td><td>34 455 </td><td>13 870 </td><td>2920 </td><td>4156 </td><td>2230 </td><td>136 552 </td><td>26.61 </td><td>60.95 </td><td>151.41 </td><td>719.18 </td><td>505.29 </td><td>941.70 </td><td>15.38 </td></tr><tr><td>Average </td><td>77 974 </td><td>34 496 </td><td>13 912 </td><td>2926 </td><td>4158 </td><td>2228 </td><td>135 693 </td><td>26.93 </td><td>60.88 </td><td>150.95 </td><td>717.70 </td><td>505.05 </td><td>942.55 </td><td>15.48 </td></tr><tr><td>Total </td><td>153 231 </td><td>65 236 </td><td>25 910 </td><td>6572 </td><td>8839 </td><td>4870 </td><td>264 658 </td><td>13.70 </td><td>32.19 </td><td>81.05 </td><td>319.54 </td><td>237.58 </td><td>431.21 </td><td>7.93 </td></tr><tr><td>Common </td><td>22 453 </td><td>9963 </td><td>4603 </td><td>553 </td><td>929 </td><td>470 </td><td>38 971 </td><td>93.53 </td><td>210.78 </td><td>456.22 </td><td>3797.47 </td><td>2260.50 </td><td>4468.09 </td><td>53.89 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>MNR, DNR, TNR, TTR, PNR, and HNR indicate mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193949" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST026TB2" class="table-wrap-title" data-id="DST026TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Numbers and density of SSR loci identified in 17 maize genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Genotypes </th><th colspan="7">SSR numbers<hr /> </th><th colspan="7">SSR interval (kb)<hr /> </th></tr><tr><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th></tr></thead><tbody><tr><td>178 </td><td>78 367 </td><td>34 604 </td><td>13 929 </td><td>2 964 </td><td>4172 </td><td>2226 </td><td>136 262 </td><td>26.80 </td><td>60.69 </td><td>150.76 </td><td>708.50 </td><td>503.36 </td><td>943.40 </td><td>15.41 </td></tr><tr><td>81565 </td><td>79 042 </td><td>34 457 </td><td>13 893 </td><td>2929 </td><td>4178 </td><td>2224 </td><td>136 723 </td><td>26.57 </td><td>60.95 </td><td>151.16 </td><td>716.97 </td><td>502.63 </td><td>944.24 </td><td>15.36 </td></tr><tr><td>18White </td><td>77 144 </td><td>34 451 </td><td>13 931 </td><td>2928 </td><td>4166 </td><td>2217 </td><td>134 837 </td><td>27.22 </td><td>60.96 </td><td>150.74 </td><td>717.21 </td><td>504.08 </td><td>947.23 </td><td>15.57 </td></tr><tr><td>18Red </td><td>77 049 </td><td>34 495 </td><td>13 919 </td><td>2933 </td><td>4188 </td><td>2221 </td><td>134 805 </td><td>27.26 </td><td>60.88 </td><td>150.87 </td><td>715.99 </td><td>501.43 </td><td>945.52 </td><td>15.58 </td></tr><tr><td>48-2 </td><td>76 920 </td><td>34 448 </td><td>13 884 </td><td>2924 </td><td>4123 </td><td>2219 </td><td>134 518 </td><td>27.30 </td><td>60.96 </td><td>151.25 </td><td>718.19 </td><td>509.34 </td><td>946.37 </td><td>15.61 </td></tr><tr><td>B73 </td><td>77 888 </td><td>34 755 </td><td>14 028 </td><td>2948 </td><td>4181 </td><td>2239 </td><td>136 039 </td><td>26.96 </td><td>60.42 </td><td>149.70 </td><td>712.35 </td><td>502.27 </td><td>937.92 </td><td>15.44 </td></tr><tr><td>CML411 </td><td>78 591 </td><td>34 546 </td><td>13 900 </td><td>2935 </td><td>4189 </td><td>2225 </td><td>136 386 </td><td>26.72 </td><td>60.79 </td><td>151.08 </td><td>715.50 </td><td>501.31 </td><td>943.82 </td><td>15.40 </td></tr><tr><td>Dan598 </td><td>78 558 </td><td>34 559 </td><td>13 954 </td><td>2904 </td><td>4159 </td><td>2227 </td><td>136 361 </td><td>26.73 </td><td>60.77 </td><td>150.49 </td><td>723.14 </td><td>504.93 </td><td>942.97 </td><td>15.40 </td></tr><tr><td>ES40 </td><td>78 978 </td><td>34 372 </td><td>13 894 </td><td>2924 </td><td>4161 </td><td>2221 </td><td>136 550 </td><td>26.59 </td><td>61.10 </td><td>151.14 </td><td>718.19 </td><td>504.69 </td><td>945.52 </td><td>15.38 </td></tr><tr><td>Han21 </td><td>78 539 </td><td>34 584 </td><td>13 951 </td><td>2898 </td><td>4157 </td><td>2233 </td><td>136 362 </td><td>26.74 </td><td>60.72 </td><td>150.53 </td><td>724.64 </td><td>505.17 </td><td>940.44 </td><td>15.40 </td></tr><tr><td>Huangzao4 </td><td>76 773 </td><td>34 445 </td><td>13 878 </td><td>2907 </td><td>4165 </td><td>2224 </td><td>134 392 </td><td>27.35 </td><td>60.97 </td><td>151.32 </td><td>722.39 </td><td>504.20 </td><td>944.24 </td><td>15.63 </td></tr><tr><td>Mo17 </td><td>78 360 </td><td>34 442 </td><td>13 910 </td><td>2902 </td><td>4108 </td><td>2220 </td><td>135 942 </td><td>26.80 </td><td>60.97 </td><td>150.97 </td><td>723.64 </td><td>511.20 </td><td>945.95 </td><td>15.45 </td></tr><tr><td>P1 </td><td>78 975 </td><td>34 447 </td><td>13 849 </td><td>2961 </td><td>4146 </td><td>2235 </td><td>136 613 </td><td>26.59 </td><td>60.96 </td><td>151.64 </td><td>709.22 </td><td>506.51 </td><td>939.60 </td><td>15.37 </td></tr><tr><td>RP125 </td><td>76 880 </td><td>34 523 </td><td>13 934 </td><td>2929 </td><td>4198 </td><td>2244 </td><td>134 708 </td><td>27.32 </td><td>60.83 </td><td>150.71 </td><td>716.97 </td><td>500.24 </td><td>935.83 </td><td>15.59 </td></tr><tr><td><em>Z. mexicana</em> </td><td>75 997 </td><td>34 339 </td><td>13 806 </td><td>2886 </td><td>4085 </td><td>2233 </td><td>133 346 </td><td>27.63 </td><td>61.15 </td><td>152.11 </td><td>727.65 </td><td>514.08 </td><td>940.44 </td><td>15.75 </td></tr><tr><td>Ye478 </td><td>78 571 </td><td>34 508 </td><td>13 973 </td><td>2944 </td><td>4159 </td><td>2234 </td><td>136 389 </td><td>26.73 </td><td>60.86 </td><td>150.29 </td><td>713.32 </td><td>504.93 </td><td>940.02 </td><td>15.40 </td></tr><tr><td>Zheng22 </td><td>78 921 </td><td>34 455 </td><td>13 870 </td><td>2920 </td><td>4156 </td><td>2230 </td><td>136 552 </td><td>26.61 </td><td>60.95 </td><td>151.41 </td><td>719.18 </td><td>505.29 </td><td>941.70 </td><td>15.38 </td></tr><tr><td>Average </td><td>77 974 </td><td>34 496 </td><td>13 912 </td><td>2926 </td><td>4158 </td><td>2228 </td><td>135 693 </td><td>26.93 </td><td>60.88 </td><td>150.95 </td><td>717.70 </td><td>505.05 </td><td>942.55 </td><td>15.48 </td></tr><tr><td>Total </td><td>153 231 </td><td>65 236 </td><td>25 910 </td><td>6572 </td><td>8839 </td><td>4870 </td><td>264 658 </td><td>13.70 </td><td>32.19 </td><td>81.05 </td><td>319.54 </td><td>237.58 </td><td>431.21 </td><td>7.93 </td></tr><tr><td>Common </td><td>22 453 </td><td>9963 </td><td>4603 </td><td>553 </td><td>929 </td><td>470 </td><td>38 971 </td><td>93.53 </td><td>210.78 </td><td>456.22 </td><td>3797.47 </td><td>2260.50 </td><td>4468.09 </td><td>53.89 </td></tr></tbody></table></div><div content-id="DST026TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Genotypes </th><th colspan="7">SSR numbers<hr /> </th><th colspan="7">SSR interval (kb)<hr /> </th></tr><tr><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th><th>MNR </th><th>DNR </th><th>TNR </th><th>TTR </th><th>PNR </th><th>HNR </th><th>Total </th></tr></thead><tbody><tr><td>178 </td><td>78 367 </td><td>34 604 </td><td>13 929 </td><td>2 964 </td><td>4172 </td><td>2226 </td><td>136 262 </td><td>26.80 </td><td>60.69 </td><td>150.76 </td><td>708.50 </td><td>503.36 </td><td>943.40 </td><td>15.41 </td></tr><tr><td>81565 </td><td>79 042 </td><td>34 457 </td><td>13 893 </td><td>2929 </td><td>4178 </td><td>2224 </td><td>136 723 </td><td>26.57 </td><td>60.95 </td><td>151.16 </td><td>716.97 </td><td>502.63 </td><td>944.24 </td><td>15.36 </td></tr><tr><td>18White </td><td>77 144 </td><td>34 451 </td><td>13 931 </td><td>2928 </td><td>4166 </td><td>2217 </td><td>134 837 </td><td>27.22 </td><td>60.96 </td><td>150.74 </td><td>717.21 </td><td>504.08 </td><td>947.23 </td><td>15.57 </td></tr><tr><td>18Red </td><td>77 049 </td><td>34 495 </td><td>13 919 </td><td>2933 </td><td>4188 </td><td>2221 </td><td>134 805 </td><td>27.26 </td><td>60.88 </td><td>150.87 </td><td>715.99 </td><td>501.43 </td><td>945.52 </td><td>15.58 </td></tr><tr><td>48-2 </td><td>76 920 </td><td>34 448 </td><td>13 884 </td><td>2924 </td><td>4123 </td><td>2219 </td><td>134 518 </td><td>27.30 </td><td>60.96 </td><td>151.25 </td><td>718.19 </td><td>509.34 </td><td>946.37 </td><td>15.61 </td></tr><tr><td>B73 </td><td>77 888 </td><td>34 755 </td><td>14 028 </td><td>2948 </td><td>4181 </td><td>2239 </td><td>136 039 </td><td>26.96 </td><td>60.42 </td><td>149.70 </td><td>712.35 </td><td>502.27 </td><td>937.92 </td><td>15.44 </td></tr><tr><td>CML411 </td><td>78 591 </td><td>34 546 </td><td>13 900 </td><td>2935 </td><td>4189 </td><td>2225 </td><td>136 386 </td><td>26.72 </td><td>60.79 </td><td>151.08 </td><td>715.50 </td><td>501.31 </td><td>943.82 </td><td>15.40 </td></tr><tr><td>Dan598 </td><td>78 558 </td><td>34 559 </td><td>13 954 </td><td>2904 </td><td>4159 </td><td>2227 </td><td>136 361 </td><td>26.73 </td><td>60.77 </td><td>150.49 </td><td>723.14 </td><td>504.93 </td><td>942.97 </td><td>15.40 </td></tr><tr><td>ES40 </td><td>78 978 </td><td>34 372 </td><td>13 894 </td><td>2924 </td><td>4161 </td><td>2221 </td><td>136 550 </td><td>26.59 </td><td>61.10 </td><td>151.14 </td><td>718.19 </td><td>504.69 </td><td>945.52 </td><td>15.38 </td></tr><tr><td>Han21 </td><td>78 539 </td><td>34 584 </td><td>13 951 </td><td>2898 </td><td>4157 </td><td>2233 </td><td>136 362 </td><td>26.74 </td><td>60.72 </td><td>150.53 </td><td>724.64 </td><td>505.17 </td><td>940.44 </td><td>15.40 </td></tr><tr><td>Huangzao4 </td><td>76 773 </td><td>34 445 </td><td>13 878 </td><td>2907 </td><td>4165 </td><td>2224 </td><td>134 392 </td><td>27.35 </td><td>60.97 </td><td>151.32 </td><td>722.39 </td><td>504.20 </td><td>944.24 </td><td>15.63 </td></tr><tr><td>Mo17 </td><td>78 360 </td><td>34 442 </td><td>13 910 </td><td>2902 </td><td>4108 </td><td>2220 </td><td>135 942 </td><td>26.80 </td><td>60.97 </td><td>150.97 </td><td>723.64 </td><td>511.20 </td><td>945.95 </td><td>15.45 </td></tr><tr><td>P1 </td><td>78 975 </td><td>34 447 </td><td>13 849 </td><td>2961 </td><td>4146 </td><td>2235 </td><td>136 613 </td><td>26.59 </td><td>60.96 </td><td>151.64 </td><td>709.22 </td><td>506.51 </td><td>939.60 </td><td>15.37 </td></tr><tr><td>RP125 </td><td>76 880 </td><td>34 523 </td><td>13 934 </td><td>2929 </td><td>4198 </td><td>2244 </td><td>134 708 </td><td>27.32 </td><td>60.83 </td><td>150.71 </td><td>716.97 </td><td>500.24 </td><td>935.83 </td><td>15.59 </td></tr><tr><td><em>Z. mexicana</em> </td><td>75 997 </td><td>34 339 </td><td>13 806 </td><td>2886 </td><td>4085 </td><td>2233 </td><td>133 346 </td><td>27.63 </td><td>61.15 </td><td>152.11 </td><td>727.65 </td><td>514.08 </td><td>940.44 </td><td>15.75 </td></tr><tr><td>Ye478 </td><td>78 571 </td><td>34 508 </td><td>13 973 </td><td>2944 </td><td>4159 </td><td>2234 </td><td>136 389 </td><td>26.73 </td><td>60.86 </td><td>150.29 </td><td>713.32 </td><td>504.93 </td><td>940.02 </td><td>15.40 </td></tr><tr><td>Zheng22 </td><td>78 921 </td><td>34 455 </td><td>13 870 </td><td>2920 </td><td>4156 </td><td>2230 </td><td>136 552 </td><td>26.61 </td><td>60.95 </td><td>151.41 </td><td>719.18 </td><td>505.29 </td><td>941.70 </td><td>15.38 </td></tr><tr><td>Average </td><td>77 974 </td><td>34 496 </td><td>13 912 </td><td>2926 </td><td>4158 </td><td>2228 </td><td>135 693 </td><td>26.93 </td><td>60.88 </td><td>150.95 </td><td>717.70 </td><td>505.05 </td><td>942.55 </td><td>15.48 </td></tr><tr><td>Total </td><td>153 231 </td><td>65 236 </td><td>25 910 </td><td>6572 </td><td>8839 </td><td>4870 </td><td>264 658 </td><td>13.70 </td><td>32.19 </td><td>81.05 </td><td>319.54 </td><td>237.58 </td><td>431.21 </td><td>7.93 </td></tr><tr><td>Common </td><td>22 453 </td><td>9963 </td><td>4603 </td><td>553 </td><td>929 </td><td>470 </td><td>38 971 </td><td>93.53 </td><td>210.78 </td><td>456.22 </td><td>3797.47 </td><td>2260.50 </td><td>4468.09 </td><td>53.89 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>MNR, DNR, TNR, TTR, PNR, and HNR indicate mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193949" target="_blank">View Large</a></div></div><p>We also examined different SSR repeat types in the genome for all tested genotypes. The frequencies of different nucleotide repeat types in each motif were different, but they showed similar frequency patterns in different genomes. Here, we compared the frequencies of different SSR repeat types by taking the reference line B73 as an example (Table <a reveal-id="DST026TB3" data-open="DST026TB3" class="link link-reveal link-table xref-fig">3</a>). Of mononucleotide motifs, C/G repeats accounted for ∼54.4%, which was slightly higher than A/T repeats. Of the di-nucleotide motifs, (AT)<em>n</em> were most frequent (24.93%), followed by (AG/CT)<em>n</em> (24.36%), (TA)<em>n</em> (21.03%), and (GA/TC) (20.64%), while the (CG)<em>n</em> motif was least frequent (0.80%). Of the tri-nucleotide motifs, (CAG/CTG)<em>n</em> was the most abundant (15.86%), while other nucleotide repeat types had lower frequencies (0.4–6%). Of the tetra-nucleotide SSRs, (AAAT/ATTT)<em>n</em>, was most frequent (10.35%), and the frequencies for the rest nucleotide repeat types were all lower than 7%. There were many types of penta- and hexa-nucleotide SSRs, each with low frequencies, ranging from 0.04 to 4%. The numbers of mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide motifs in different repeat unit classes are also listed in Table <a reveal-id="DST026TB3" data-open="DST026TB3" class="link link-reveal link-table xref-fig">3</a>. The average repeat lengths were different among various motifs ranging from 11.02 for (A/T)<em>n</em> to 58.84 for (AGT/ACT)<em>n</em>. </p>                        <a id="5193951" scrollto-destination="5193951"></a>
<div content-id="DST026TB3" class="table-modal"><div class="table-wrap"><div id="DST026TB3" class="table-wrap-title" data-id="DST026TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of SSRs in different repeat classes in the maize genome B73</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="9">Repeats number<hr /> </th><th rowspan="2">Total </th><th rowspan="2">Average repeat number </th><th rowspan="2">Average repeat length (bp) </th></tr><tr><th>&lt;5 </th><th>5–7 </th><th>8–10 </th><th>11–15 </th><th>16–20 </th><th>21–25 </th><th>26–30 </th><th>31–40 </th><th>&gt;40 </th></tr></thead><tbody><tr><td>G/C </td><td>0 </td><td>0 </td><td>13 085 </td><td>24 297 </td><td>3 832 </td><td>866 </td><td>243 </td><td>45 </td><td>1 </td><td>42 369 </td><td>12.31 </td><td>12.31 </td></tr><tr><td>A/T </td><td>0 </td><td>0 </td><td>22 410 </td><td>11 687 </td><td>1114 </td><td>192 </td><td>57 </td><td>32 </td><td>27 </td><td>35 519 </td><td>11.02 </td><td>11.02 </td></tr><tr><td>AT </td><td>0 </td><td>1875 </td><td>3268 </td><td>1379 </td><td>708 </td><td>478 </td><td>378 </td><td>466 </td><td>111 </td><td>8663 </td><td>13.28 </td><td>26.56 </td></tr><tr><td>CT/AG </td><td>0 </td><td>3615 </td><td>3504 </td><td>740 </td><td>265 </td><td>127 </td><td>72 </td><td>95 </td><td>47 </td><td>8465 </td><td>9.42 </td><td>18.84 </td></tr><tr><td>TA </td><td>0 </td><td>1725 </td><td>2417 </td><td>1087 </td><td>625 </td><td>465 </td><td>364 </td><td>492 </td><td>134 </td><td>7309 </td><td>14.03 </td><td>28.05 </td></tr><tr><td>GA/TC </td><td>0 </td><td>3220 </td><td>2938 </td><td>519 </td><td>237 </td><td>103 </td><td>71 </td><td>57 </td><td>28 </td><td>7173 </td><td>9.19 </td><td>18.39 </td></tr><tr><td>CA/TG </td><td>0 </td><td>666 </td><td>542 </td><td>72 </td><td>15 </td><td>5 </td><td>1 </td><td>5 </td><td>1 </td><td>1307 </td><td>8.24 </td><td>16.48 </td></tr><tr><td>GT/AC </td><td>0 </td><td>582 </td><td>588 </td><td>73 </td><td>12 </td><td>5 </td><td>4 </td><td>4 </td><td>0 </td><td>1268 </td><td>8.32 </td><td>16.64 </td></tr><tr><td>GC </td><td>0 </td><td>223 </td><td>66 </td><td>3 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>295 </td><td>7.42 </td><td>14.83 </td></tr><tr><td>CG </td><td>0 </td><td>219 </td><td>55 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>275 </td><td>7.29 </td><td>14.59 </td></tr><tr><td>CAG/CTG </td><td>0 </td><td>1980 </td><td>242 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2225 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GAT/ATC </td><td>0 </td><td>526 </td><td>264 </td><td>46 </td><td>5 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>843 </td><td>7.49 </td><td>22.48 </td></tr><tr><td>GCA/TGC </td><td>0 </td><td>711 </td><td>99 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>810 </td><td>6.42 </td><td>19.25 </td></tr><tr><td>GAC/GTC </td><td>0 </td><td>587 </td><td>184 </td><td>28 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>800 </td><td>7.02 </td><td>21.06 </td></tr><tr><td>ATT/AAT </td><td>0 </td><td>352 </td><td>124 </td><td>65 </td><td>42 </td><td>28 </td><td>18 </td><td>10 </td><td>4 </td><td>643 </td><td>10.25 </td><td>30.74 </td></tr><tr><td>TTA/TAA </td><td>0 </td><td>359 </td><td>96 </td><td>66 </td><td>32 </td><td>27 </td><td>24 </td><td>14 </td><td>2 </td><td>620 </td><td>10.31 </td><td>30.94 </td></tr><tr><td>CGT/ACG </td><td>0 </td><td>429 </td><td>131 </td><td>34 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>594 </td><td>7.18 </td><td>21.55 </td></tr><tr><td>TGA/TCA </td><td>0 </td><td>350 </td><td>167 </td><td>58 </td><td>9 </td><td>3 </td><td>1 </td><td>0 </td><td>0 </td><td>588 </td><td>7.86 </td><td>23.59 </td></tr><tr><td>CGA/TCG </td><td>0 </td><td>474 </td><td>100 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>578 </td><td>6.69 </td><td>20.06 </td></tr><tr><td>CGC/GCG </td><td>0 </td><td>505 </td><td>49 </td><td>4 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>559 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GCC/GGC </td><td>0 </td><td>490 </td><td>42 </td><td>5 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>537 </td><td>6.49 </td><td>19.46 </td></tr><tr><td>TAT/ATA </td><td>0 </td><td>246 </td><td>103 </td><td>43 </td><td>47 </td><td>35 </td><td>13 </td><td>13 </td><td>5 </td><td>505 </td><td>11.39 </td><td>34.16 </td></tr><tr><td>CGG/CCG </td><td>0 </td><td>416 </td><td>48 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>467 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>TAC/GTA </td><td>0 </td><td>232 </td><td>50 </td><td>26 </td><td>16 </td><td>6 </td><td>1 </td><td>5 </td><td>1 </td><td>337 </td><td>8.54 </td><td>25.63 </td></tr><tr><td>TTG/CAA </td><td>0 </td><td>230 </td><td>80 </td><td>16 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>330 </td><td>7.32 </td><td>21.96 </td></tr><tr><td>TTC/GAA </td><td>0 </td><td>288 </td><td>26 </td><td>9 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td><td>329 </td><td>7.42 </td><td>22.26 </td></tr><tr><td>ATG/CAT </td><td>0 </td><td>260 </td><td>45 </td><td>11 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>321 </td><td>7.01 </td><td>21.04 </td></tr><tr><td>GCT/AGC </td><td>0 </td><td>291 </td><td>16 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>309 </td><td>6.36 </td><td>19.09 </td></tr><tr><td>TGG/CCA </td><td>0 </td><td>280 </td><td>24 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>306 </td><td>6.4 </td><td>19.2 </td></tr><tr><td>TAG/CTA </td><td>0 </td><td>186 </td><td>49 </td><td>26 </td><td>11 </td><td>10 </td><td>8 </td><td>3 </td><td>3 </td><td>296 </td><td>9.63 </td><td>28.9 </td></tr><tr><td>CTC/GAG </td><td>0 </td><td>232 </td><td>46 </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>284 </td><td>6.82 </td><td>20.45 </td></tr><tr><td>CTT/AAG </td><td>0 </td><td>245 </td><td>31 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>1 </td><td>1 </td><td>281 </td><td>7 </td><td>21 </td></tr><tr><td>TCC/GGA </td><td>0 </td><td>193 </td><td>43 </td><td>8 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>245 </td><td>6.91 </td><td>20.74 </td></tr><tr><td>CCT/AGG </td><td>0 </td><td>187 </td><td>32 </td><td>5 </td><td>2 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>227 </td><td>6.93 </td><td>20.79 </td></tr><tr><td>CAC/GTG </td><td>0 </td><td>198 </td><td>18 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>217 </td><td>6.46 </td><td>19.38 </td></tr><tr><td>ACC/GGT </td><td>0 </td><td>191 </td><td>16 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>208 </td><td>6.39 </td><td>19.17 </td></tr><tr><td>ACA/TGT </td><td>0 </td><td>125 </td><td>58 </td><td>10 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>194 </td><td>7.34 </td><td>22.02 </td></tr><tr><td>AGA/TCT </td><td>0 </td><td>169 </td><td>21 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>193 </td><td>6.58 </td><td>19.74 </td></tr><tr><td>GTT/AAC </td><td>0 </td><td>85 </td><td>29 </td><td>4 </td><td>0 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>120 </td><td>7.45 </td><td>22.35 </td></tr><tr><td>AAAT/ATTT </td><td>0 </td><td>288 </td><td>15 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>305 </td><td>5.58 </td><td>22.33 </td></tr><tr><td>AGGC/GCCT </td><td>0 </td><td>153 </td><td>11 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>168 </td><td>5.9 </td><td>23.6 </td></tr><tr><td>TATT/AATA </td><td>0 </td><td>195 </td><td>10 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>207 </td><td>5.72 </td><td>22.88 </td></tr><tr><td>TCGT/ACGA </td><td>0 </td><td>126 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>126 </td><td>5.07 </td><td>20.29 </td></tr><tr><td>TTAT/ATAA </td><td>0 </td><td>111 </td><td>16 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>127 </td><td>5.88 </td><td>23.53 </td></tr><tr><td>TTTA/TAAA </td><td>0 </td><td>157 </td><td>7 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>166 </td><td>5.67 </td><td>22.67 </td></tr><tr><td>CGAGC/GCTCG </td><td>153 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>177 </td><td>4.16 </td><td>24.95 </td></tr><tr><td>TTTTA/TAAAA </td><td>82 </td><td>28 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>110 </td><td>4.29 </td><td>25.75 </td></tr><tr><td>ATTTT/AAAAT </td><td>69 </td><td>35 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>105 </td><td>4.42 </td><td>22.1 </td></tr></tbody></table></div><div content-id="DST026TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="9">Repeats number<hr /> </th><th rowspan="2">Total </th><th rowspan="2">Average repeat number </th><th rowspan="2">Average repeat length (bp) </th></tr><tr><th>&lt;5 </th><th>5–7 </th><th>8–10 </th><th>11–15 </th><th>16–20 </th><th>21–25 </th><th>26–30 </th><th>31–40 </th><th>&gt;40 </th></tr></thead><tbody><tr><td>G/C </td><td>0 </td><td>0 </td><td>13 085 </td><td>24 297 </td><td>3 832 </td><td>866 </td><td>243 </td><td>45 </td><td>1 </td><td>42 369 </td><td>12.31 </td><td>12.31 </td></tr><tr><td>A/T </td><td>0 </td><td>0 </td><td>22 410 </td><td>11 687 </td><td>1114 </td><td>192 </td><td>57 </td><td>32 </td><td>27 </td><td>35 519 </td><td>11.02 </td><td>11.02 </td></tr><tr><td>AT </td><td>0 </td><td>1875 </td><td>3268 </td><td>1379 </td><td>708 </td><td>478 </td><td>378 </td><td>466 </td><td>111 </td><td>8663 </td><td>13.28 </td><td>26.56 </td></tr><tr><td>CT/AG </td><td>0 </td><td>3615 </td><td>3504 </td><td>740 </td><td>265 </td><td>127 </td><td>72 </td><td>95 </td><td>47 </td><td>8465 </td><td>9.42 </td><td>18.84 </td></tr><tr><td>TA </td><td>0 </td><td>1725 </td><td>2417 </td><td>1087 </td><td>625 </td><td>465 </td><td>364 </td><td>492 </td><td>134 </td><td>7309 </td><td>14.03 </td><td>28.05 </td></tr><tr><td>GA/TC </td><td>0 </td><td>3220 </td><td>2938 </td><td>519 </td><td>237 </td><td>103 </td><td>71 </td><td>57 </td><td>28 </td><td>7173 </td><td>9.19 </td><td>18.39 </td></tr><tr><td>CA/TG </td><td>0 </td><td>666 </td><td>542 </td><td>72 </td><td>15 </td><td>5 </td><td>1 </td><td>5 </td><td>1 </td><td>1307 </td><td>8.24 </td><td>16.48 </td></tr><tr><td>GT/AC </td><td>0 </td><td>582 </td><td>588 </td><td>73 </td><td>12 </td><td>5 </td><td>4 </td><td>4 </td><td>0 </td><td>1268 </td><td>8.32 </td><td>16.64 </td></tr><tr><td>GC </td><td>0 </td><td>223 </td><td>66 </td><td>3 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>295 </td><td>7.42 </td><td>14.83 </td></tr><tr><td>CG </td><td>0 </td><td>219 </td><td>55 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>275 </td><td>7.29 </td><td>14.59 </td></tr><tr><td>CAG/CTG </td><td>0 </td><td>1980 </td><td>242 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2225 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GAT/ATC </td><td>0 </td><td>526 </td><td>264 </td><td>46 </td><td>5 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>843 </td><td>7.49 </td><td>22.48 </td></tr><tr><td>GCA/TGC </td><td>0 </td><td>711 </td><td>99 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>810 </td><td>6.42 </td><td>19.25 </td></tr><tr><td>GAC/GTC </td><td>0 </td><td>587 </td><td>184 </td><td>28 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>800 </td><td>7.02 </td><td>21.06 </td></tr><tr><td>ATT/AAT </td><td>0 </td><td>352 </td><td>124 </td><td>65 </td><td>42 </td><td>28 </td><td>18 </td><td>10 </td><td>4 </td><td>643 </td><td>10.25 </td><td>30.74 </td></tr><tr><td>TTA/TAA </td><td>0 </td><td>359 </td><td>96 </td><td>66 </td><td>32 </td><td>27 </td><td>24 </td><td>14 </td><td>2 </td><td>620 </td><td>10.31 </td><td>30.94 </td></tr><tr><td>CGT/ACG </td><td>0 </td><td>429 </td><td>131 </td><td>34 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>594 </td><td>7.18 </td><td>21.55 </td></tr><tr><td>TGA/TCA </td><td>0 </td><td>350 </td><td>167 </td><td>58 </td><td>9 </td><td>3 </td><td>1 </td><td>0 </td><td>0 </td><td>588 </td><td>7.86 </td><td>23.59 </td></tr><tr><td>CGA/TCG </td><td>0 </td><td>474 </td><td>100 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>578 </td><td>6.69 </td><td>20.06 </td></tr><tr><td>CGC/GCG </td><td>0 </td><td>505 </td><td>49 </td><td>4 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>559 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GCC/GGC </td><td>0 </td><td>490 </td><td>42 </td><td>5 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>537 </td><td>6.49 </td><td>19.46 </td></tr><tr><td>TAT/ATA </td><td>0 </td><td>246 </td><td>103 </td><td>43 </td><td>47 </td><td>35 </td><td>13 </td><td>13 </td><td>5 </td><td>505 </td><td>11.39 </td><td>34.16 </td></tr><tr><td>CGG/CCG </td><td>0 </td><td>416 </td><td>48 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>467 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>TAC/GTA </td><td>0 </td><td>232 </td><td>50 </td><td>26 </td><td>16 </td><td>6 </td><td>1 </td><td>5 </td><td>1 </td><td>337 </td><td>8.54 </td><td>25.63 </td></tr><tr><td>TTG/CAA </td><td>0 </td><td>230 </td><td>80 </td><td>16 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>330 </td><td>7.32 </td><td>21.96 </td></tr><tr><td>TTC/GAA </td><td>0 </td><td>288 </td><td>26 </td><td>9 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td><td>329 </td><td>7.42 </td><td>22.26 </td></tr><tr><td>ATG/CAT </td><td>0 </td><td>260 </td><td>45 </td><td>11 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>321 </td><td>7.01 </td><td>21.04 </td></tr><tr><td>GCT/AGC </td><td>0 </td><td>291 </td><td>16 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>309 </td><td>6.36 </td><td>19.09 </td></tr><tr><td>TGG/CCA </td><td>0 </td><td>280 </td><td>24 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>306 </td><td>6.4 </td><td>19.2 </td></tr><tr><td>TAG/CTA </td><td>0 </td><td>186 </td><td>49 </td><td>26 </td><td>11 </td><td>10 </td><td>8 </td><td>3 </td><td>3 </td><td>296 </td><td>9.63 </td><td>28.9 </td></tr><tr><td>CTC/GAG </td><td>0 </td><td>232 </td><td>46 </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>284 </td><td>6.82 </td><td>20.45 </td></tr><tr><td>CTT/AAG </td><td>0 </td><td>245 </td><td>31 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>1 </td><td>1 </td><td>281 </td><td>7 </td><td>21 </td></tr><tr><td>TCC/GGA </td><td>0 </td><td>193 </td><td>43 </td><td>8 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>245 </td><td>6.91 </td><td>20.74 </td></tr><tr><td>CCT/AGG </td><td>0 </td><td>187 </td><td>32 </td><td>5 </td><td>2 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>227 </td><td>6.93 </td><td>20.79 </td></tr><tr><td>CAC/GTG </td><td>0 </td><td>198 </td><td>18 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>217 </td><td>6.46 </td><td>19.38 </td></tr><tr><td>ACC/GGT </td><td>0 </td><td>191 </td><td>16 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>208 </td><td>6.39 </td><td>19.17 </td></tr><tr><td>ACA/TGT </td><td>0 </td><td>125 </td><td>58 </td><td>10 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>194 </td><td>7.34 </td><td>22.02 </td></tr><tr><td>AGA/TCT </td><td>0 </td><td>169 </td><td>21 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>193 </td><td>6.58 </td><td>19.74 </td></tr><tr><td>GTT/AAC </td><td>0 </td><td>85 </td><td>29 </td><td>4 </td><td>0 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>120 </td><td>7.45 </td><td>22.35 </td></tr><tr><td>AAAT/ATTT </td><td>0 </td><td>288 </td><td>15 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>305 </td><td>5.58 </td><td>22.33 </td></tr><tr><td>AGGC/GCCT </td><td>0 </td><td>153 </td><td>11 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>168 </td><td>5.9 </td><td>23.6 </td></tr><tr><td>TATT/AATA </td><td>0 </td><td>195 </td><td>10 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>207 </td><td>5.72 </td><td>22.88 </td></tr><tr><td>TCGT/ACGA </td><td>0 </td><td>126 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>126 </td><td>5.07 </td><td>20.29 </td></tr><tr><td>TTAT/ATAA </td><td>0 </td><td>111 </td><td>16 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>127 </td><td>5.88 </td><td>23.53 </td></tr><tr><td>TTTA/TAAA </td><td>0 </td><td>157 </td><td>7 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>166 </td><td>5.67 </td><td>22.67 </td></tr><tr><td>CGAGC/GCTCG </td><td>153 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>177 </td><td>4.16 </td><td>24.95 </td></tr><tr><td>TTTTA/TAAAA </td><td>82 </td><td>28 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>110 </td><td>4.29 </td><td>25.75 </td></tr><tr><td>ATTTT/AAAAT </td><td>69 </td><td>35 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>105 </td><td>4.42 </td><td>22.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SSR motifs with repeats number &gt;100 in total were listed here.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193951" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST026TB3" class="table-wrap-title" data-id="DST026TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of SSRs in different repeat classes in the maize genome B73</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="9">Repeats number<hr /> </th><th rowspan="2">Total </th><th rowspan="2">Average repeat number </th><th rowspan="2">Average repeat length (bp) </th></tr><tr><th>&lt;5 </th><th>5–7 </th><th>8–10 </th><th>11–15 </th><th>16–20 </th><th>21–25 </th><th>26–30 </th><th>31–40 </th><th>&gt;40 </th></tr></thead><tbody><tr><td>G/C </td><td>0 </td><td>0 </td><td>13 085 </td><td>24 297 </td><td>3 832 </td><td>866 </td><td>243 </td><td>45 </td><td>1 </td><td>42 369 </td><td>12.31 </td><td>12.31 </td></tr><tr><td>A/T </td><td>0 </td><td>0 </td><td>22 410 </td><td>11 687 </td><td>1114 </td><td>192 </td><td>57 </td><td>32 </td><td>27 </td><td>35 519 </td><td>11.02 </td><td>11.02 </td></tr><tr><td>AT </td><td>0 </td><td>1875 </td><td>3268 </td><td>1379 </td><td>708 </td><td>478 </td><td>378 </td><td>466 </td><td>111 </td><td>8663 </td><td>13.28 </td><td>26.56 </td></tr><tr><td>CT/AG </td><td>0 </td><td>3615 </td><td>3504 </td><td>740 </td><td>265 </td><td>127 </td><td>72 </td><td>95 </td><td>47 </td><td>8465 </td><td>9.42 </td><td>18.84 </td></tr><tr><td>TA </td><td>0 </td><td>1725 </td><td>2417 </td><td>1087 </td><td>625 </td><td>465 </td><td>364 </td><td>492 </td><td>134 </td><td>7309 </td><td>14.03 </td><td>28.05 </td></tr><tr><td>GA/TC </td><td>0 </td><td>3220 </td><td>2938 </td><td>519 </td><td>237 </td><td>103 </td><td>71 </td><td>57 </td><td>28 </td><td>7173 </td><td>9.19 </td><td>18.39 </td></tr><tr><td>CA/TG </td><td>0 </td><td>666 </td><td>542 </td><td>72 </td><td>15 </td><td>5 </td><td>1 </td><td>5 </td><td>1 </td><td>1307 </td><td>8.24 </td><td>16.48 </td></tr><tr><td>GT/AC </td><td>0 </td><td>582 </td><td>588 </td><td>73 </td><td>12 </td><td>5 </td><td>4 </td><td>4 </td><td>0 </td><td>1268 </td><td>8.32 </td><td>16.64 </td></tr><tr><td>GC </td><td>0 </td><td>223 </td><td>66 </td><td>3 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>295 </td><td>7.42 </td><td>14.83 </td></tr><tr><td>CG </td><td>0 </td><td>219 </td><td>55 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>275 </td><td>7.29 </td><td>14.59 </td></tr><tr><td>CAG/CTG </td><td>0 </td><td>1980 </td><td>242 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2225 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GAT/ATC </td><td>0 </td><td>526 </td><td>264 </td><td>46 </td><td>5 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>843 </td><td>7.49 </td><td>22.48 </td></tr><tr><td>GCA/TGC </td><td>0 </td><td>711 </td><td>99 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>810 </td><td>6.42 </td><td>19.25 </td></tr><tr><td>GAC/GTC </td><td>0 </td><td>587 </td><td>184 </td><td>28 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>800 </td><td>7.02 </td><td>21.06 </td></tr><tr><td>ATT/AAT </td><td>0 </td><td>352 </td><td>124 </td><td>65 </td><td>42 </td><td>28 </td><td>18 </td><td>10 </td><td>4 </td><td>643 </td><td>10.25 </td><td>30.74 </td></tr><tr><td>TTA/TAA </td><td>0 </td><td>359 </td><td>96 </td><td>66 </td><td>32 </td><td>27 </td><td>24 </td><td>14 </td><td>2 </td><td>620 </td><td>10.31 </td><td>30.94 </td></tr><tr><td>CGT/ACG </td><td>0 </td><td>429 </td><td>131 </td><td>34 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>594 </td><td>7.18 </td><td>21.55 </td></tr><tr><td>TGA/TCA </td><td>0 </td><td>350 </td><td>167 </td><td>58 </td><td>9 </td><td>3 </td><td>1 </td><td>0 </td><td>0 </td><td>588 </td><td>7.86 </td><td>23.59 </td></tr><tr><td>CGA/TCG </td><td>0 </td><td>474 </td><td>100 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>578 </td><td>6.69 </td><td>20.06 </td></tr><tr><td>CGC/GCG </td><td>0 </td><td>505 </td><td>49 </td><td>4 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>559 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GCC/GGC </td><td>0 </td><td>490 </td><td>42 </td><td>5 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>537 </td><td>6.49 </td><td>19.46 </td></tr><tr><td>TAT/ATA </td><td>0 </td><td>246 </td><td>103 </td><td>43 </td><td>47 </td><td>35 </td><td>13 </td><td>13 </td><td>5 </td><td>505 </td><td>11.39 </td><td>34.16 </td></tr><tr><td>CGG/CCG </td><td>0 </td><td>416 </td><td>48 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>467 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>TAC/GTA </td><td>0 </td><td>232 </td><td>50 </td><td>26 </td><td>16 </td><td>6 </td><td>1 </td><td>5 </td><td>1 </td><td>337 </td><td>8.54 </td><td>25.63 </td></tr><tr><td>TTG/CAA </td><td>0 </td><td>230 </td><td>80 </td><td>16 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>330 </td><td>7.32 </td><td>21.96 </td></tr><tr><td>TTC/GAA </td><td>0 </td><td>288 </td><td>26 </td><td>9 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td><td>329 </td><td>7.42 </td><td>22.26 </td></tr><tr><td>ATG/CAT </td><td>0 </td><td>260 </td><td>45 </td><td>11 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>321 </td><td>7.01 </td><td>21.04 </td></tr><tr><td>GCT/AGC </td><td>0 </td><td>291 </td><td>16 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>309 </td><td>6.36 </td><td>19.09 </td></tr><tr><td>TGG/CCA </td><td>0 </td><td>280 </td><td>24 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>306 </td><td>6.4 </td><td>19.2 </td></tr><tr><td>TAG/CTA </td><td>0 </td><td>186 </td><td>49 </td><td>26 </td><td>11 </td><td>10 </td><td>8 </td><td>3 </td><td>3 </td><td>296 </td><td>9.63 </td><td>28.9 </td></tr><tr><td>CTC/GAG </td><td>0 </td><td>232 </td><td>46 </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>284 </td><td>6.82 </td><td>20.45 </td></tr><tr><td>CTT/AAG </td><td>0 </td><td>245 </td><td>31 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>1 </td><td>1 </td><td>281 </td><td>7 </td><td>21 </td></tr><tr><td>TCC/GGA </td><td>0 </td><td>193 </td><td>43 </td><td>8 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>245 </td><td>6.91 </td><td>20.74 </td></tr><tr><td>CCT/AGG </td><td>0 </td><td>187 </td><td>32 </td><td>5 </td><td>2 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>227 </td><td>6.93 </td><td>20.79 </td></tr><tr><td>CAC/GTG </td><td>0 </td><td>198 </td><td>18 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>217 </td><td>6.46 </td><td>19.38 </td></tr><tr><td>ACC/GGT </td><td>0 </td><td>191 </td><td>16 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>208 </td><td>6.39 </td><td>19.17 </td></tr><tr><td>ACA/TGT </td><td>0 </td><td>125 </td><td>58 </td><td>10 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>194 </td><td>7.34 </td><td>22.02 </td></tr><tr><td>AGA/TCT </td><td>0 </td><td>169 </td><td>21 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>193 </td><td>6.58 </td><td>19.74 </td></tr><tr><td>GTT/AAC </td><td>0 </td><td>85 </td><td>29 </td><td>4 </td><td>0 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>120 </td><td>7.45 </td><td>22.35 </td></tr><tr><td>AAAT/ATTT </td><td>0 </td><td>288 </td><td>15 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>305 </td><td>5.58 </td><td>22.33 </td></tr><tr><td>AGGC/GCCT </td><td>0 </td><td>153 </td><td>11 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>168 </td><td>5.9 </td><td>23.6 </td></tr><tr><td>TATT/AATA </td><td>0 </td><td>195 </td><td>10 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>207 </td><td>5.72 </td><td>22.88 </td></tr><tr><td>TCGT/ACGA </td><td>0 </td><td>126 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>126 </td><td>5.07 </td><td>20.29 </td></tr><tr><td>TTAT/ATAA </td><td>0 </td><td>111 </td><td>16 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>127 </td><td>5.88 </td><td>23.53 </td></tr><tr><td>TTTA/TAAA </td><td>0 </td><td>157 </td><td>7 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>166 </td><td>5.67 </td><td>22.67 </td></tr><tr><td>CGAGC/GCTCG </td><td>153 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>177 </td><td>4.16 </td><td>24.95 </td></tr><tr><td>TTTTA/TAAAA </td><td>82 </td><td>28 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>110 </td><td>4.29 </td><td>25.75 </td></tr><tr><td>ATTTT/AAAAT </td><td>69 </td><td>35 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>105 </td><td>4.42 </td><td>22.1 </td></tr></tbody></table></div><div content-id="DST026TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="9">Repeats number<hr /> </th><th rowspan="2">Total </th><th rowspan="2">Average repeat number </th><th rowspan="2">Average repeat length (bp) </th></tr><tr><th>&lt;5 </th><th>5–7 </th><th>8–10 </th><th>11–15 </th><th>16–20 </th><th>21–25 </th><th>26–30 </th><th>31–40 </th><th>&gt;40 </th></tr></thead><tbody><tr><td>G/C </td><td>0 </td><td>0 </td><td>13 085 </td><td>24 297 </td><td>3 832 </td><td>866 </td><td>243 </td><td>45 </td><td>1 </td><td>42 369 </td><td>12.31 </td><td>12.31 </td></tr><tr><td>A/T </td><td>0 </td><td>0 </td><td>22 410 </td><td>11 687 </td><td>1114 </td><td>192 </td><td>57 </td><td>32 </td><td>27 </td><td>35 519 </td><td>11.02 </td><td>11.02 </td></tr><tr><td>AT </td><td>0 </td><td>1875 </td><td>3268 </td><td>1379 </td><td>708 </td><td>478 </td><td>378 </td><td>466 </td><td>111 </td><td>8663 </td><td>13.28 </td><td>26.56 </td></tr><tr><td>CT/AG </td><td>0 </td><td>3615 </td><td>3504 </td><td>740 </td><td>265 </td><td>127 </td><td>72 </td><td>95 </td><td>47 </td><td>8465 </td><td>9.42 </td><td>18.84 </td></tr><tr><td>TA </td><td>0 </td><td>1725 </td><td>2417 </td><td>1087 </td><td>625 </td><td>465 </td><td>364 </td><td>492 </td><td>134 </td><td>7309 </td><td>14.03 </td><td>28.05 </td></tr><tr><td>GA/TC </td><td>0 </td><td>3220 </td><td>2938 </td><td>519 </td><td>237 </td><td>103 </td><td>71 </td><td>57 </td><td>28 </td><td>7173 </td><td>9.19 </td><td>18.39 </td></tr><tr><td>CA/TG </td><td>0 </td><td>666 </td><td>542 </td><td>72 </td><td>15 </td><td>5 </td><td>1 </td><td>5 </td><td>1 </td><td>1307 </td><td>8.24 </td><td>16.48 </td></tr><tr><td>GT/AC </td><td>0 </td><td>582 </td><td>588 </td><td>73 </td><td>12 </td><td>5 </td><td>4 </td><td>4 </td><td>0 </td><td>1268 </td><td>8.32 </td><td>16.64 </td></tr><tr><td>GC </td><td>0 </td><td>223 </td><td>66 </td><td>3 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>295 </td><td>7.42 </td><td>14.83 </td></tr><tr><td>CG </td><td>0 </td><td>219 </td><td>55 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>275 </td><td>7.29 </td><td>14.59 </td></tr><tr><td>CAG/CTG </td><td>0 </td><td>1980 </td><td>242 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2225 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GAT/ATC </td><td>0 </td><td>526 </td><td>264 </td><td>46 </td><td>5 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>843 </td><td>7.49 </td><td>22.48 </td></tr><tr><td>GCA/TGC </td><td>0 </td><td>711 </td><td>99 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>810 </td><td>6.42 </td><td>19.25 </td></tr><tr><td>GAC/GTC </td><td>0 </td><td>587 </td><td>184 </td><td>28 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>800 </td><td>7.02 </td><td>21.06 </td></tr><tr><td>ATT/AAT </td><td>0 </td><td>352 </td><td>124 </td><td>65 </td><td>42 </td><td>28 </td><td>18 </td><td>10 </td><td>4 </td><td>643 </td><td>10.25 </td><td>30.74 </td></tr><tr><td>TTA/TAA </td><td>0 </td><td>359 </td><td>96 </td><td>66 </td><td>32 </td><td>27 </td><td>24 </td><td>14 </td><td>2 </td><td>620 </td><td>10.31 </td><td>30.94 </td></tr><tr><td>CGT/ACG </td><td>0 </td><td>429 </td><td>131 </td><td>34 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>594 </td><td>7.18 </td><td>21.55 </td></tr><tr><td>TGA/TCA </td><td>0 </td><td>350 </td><td>167 </td><td>58 </td><td>9 </td><td>3 </td><td>1 </td><td>0 </td><td>0 </td><td>588 </td><td>7.86 </td><td>23.59 </td></tr><tr><td>CGA/TCG </td><td>0 </td><td>474 </td><td>100 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>578 </td><td>6.69 </td><td>20.06 </td></tr><tr><td>CGC/GCG </td><td>0 </td><td>505 </td><td>49 </td><td>4 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>559 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>GCC/GGC </td><td>0 </td><td>490 </td><td>42 </td><td>5 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>537 </td><td>6.49 </td><td>19.46 </td></tr><tr><td>TAT/ATA </td><td>0 </td><td>246 </td><td>103 </td><td>43 </td><td>47 </td><td>35 </td><td>13 </td><td>13 </td><td>5 </td><td>505 </td><td>11.39 </td><td>34.16 </td></tr><tr><td>CGG/CCG </td><td>0 </td><td>416 </td><td>48 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>467 </td><td>6.5 </td><td>19.51 </td></tr><tr><td>TAC/GTA </td><td>0 </td><td>232 </td><td>50 </td><td>26 </td><td>16 </td><td>6 </td><td>1 </td><td>5 </td><td>1 </td><td>337 </td><td>8.54 </td><td>25.63 </td></tr><tr><td>TTG/CAA </td><td>0 </td><td>230 </td><td>80 </td><td>16 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>330 </td><td>7.32 </td><td>21.96 </td></tr><tr><td>TTC/GAA </td><td>0 </td><td>288 </td><td>26 </td><td>9 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td><td>329 </td><td>7.42 </td><td>22.26 </td></tr><tr><td>ATG/CAT </td><td>0 </td><td>260 </td><td>45 </td><td>11 </td><td>4 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>321 </td><td>7.01 </td><td>21.04 </td></tr><tr><td>GCT/AGC </td><td>0 </td><td>291 </td><td>16 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>309 </td><td>6.36 </td><td>19.09 </td></tr><tr><td>TGG/CCA </td><td>0 </td><td>280 </td><td>24 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>306 </td><td>6.4 </td><td>19.2 </td></tr><tr><td>TAG/CTA </td><td>0 </td><td>186 </td><td>49 </td><td>26 </td><td>11 </td><td>10 </td><td>8 </td><td>3 </td><td>3 </td><td>296 </td><td>9.63 </td><td>28.9 </td></tr><tr><td>CTC/GAG </td><td>0 </td><td>232 </td><td>46 </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>284 </td><td>6.82 </td><td>20.45 </td></tr><tr><td>CTT/AAG </td><td>0 </td><td>245 </td><td>31 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>1 </td><td>1 </td><td>281 </td><td>7 </td><td>21 </td></tr><tr><td>TCC/GGA </td><td>0 </td><td>193 </td><td>43 </td><td>8 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>245 </td><td>6.91 </td><td>20.74 </td></tr><tr><td>CCT/AGG </td><td>0 </td><td>187 </td><td>32 </td><td>5 </td><td>2 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>227 </td><td>6.93 </td><td>20.79 </td></tr><tr><td>CAC/GTG </td><td>0 </td><td>198 </td><td>18 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>217 </td><td>6.46 </td><td>19.38 </td></tr><tr><td>ACC/GGT </td><td>0 </td><td>191 </td><td>16 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>208 </td><td>6.39 </td><td>19.17 </td></tr><tr><td>ACA/TGT </td><td>0 </td><td>125 </td><td>58 </td><td>10 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>194 </td><td>7.34 </td><td>22.02 </td></tr><tr><td>AGA/TCT </td><td>0 </td><td>169 </td><td>21 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>193 </td><td>6.58 </td><td>19.74 </td></tr><tr><td>GTT/AAC </td><td>0 </td><td>85 </td><td>29 </td><td>4 </td><td>0 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>120 </td><td>7.45 </td><td>22.35 </td></tr><tr><td>AAAT/ATTT </td><td>0 </td><td>288 </td><td>15 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>305 </td><td>5.58 </td><td>22.33 </td></tr><tr><td>AGGC/GCCT </td><td>0 </td><td>153 </td><td>11 </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>168 </td><td>5.9 </td><td>23.6 </td></tr><tr><td>TATT/AATA </td><td>0 </td><td>195 </td><td>10 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>207 </td><td>5.72 </td><td>22.88 </td></tr><tr><td>TCGT/ACGA </td><td>0 </td><td>126 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>126 </td><td>5.07 </td><td>20.29 </td></tr><tr><td>TTAT/ATAA </td><td>0 </td><td>111 </td><td>16 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>127 </td><td>5.88 </td><td>23.53 </td></tr><tr><td>TTTA/TAAA </td><td>0 </td><td>157 </td><td>7 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>166 </td><td>5.67 </td><td>22.67 </td></tr><tr><td>CGAGC/GCTCG </td><td>153 </td><td>24 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>177 </td><td>4.16 </td><td>24.95 </td></tr><tr><td>TTTTA/TAAAA </td><td>82 </td><td>28 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>110 </td><td>4.29 </td><td>25.75 </td></tr><tr><td>ATTTT/AAAAT </td><td>69 </td><td>35 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>105 </td><td>4.42 </td><td>22.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SSR motifs with repeats number &gt;100 in total were listed here.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193951" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5193952 id="5193952" class="section-title" data-legacy-id=s3b>3.2. Screening of SSR loci and development of maize SSR markers</h3>
<p>A total of 2034 SSR markers have been recently developed and reported on MaizeGDB website (<a class="link link-uri" href="http://www.maizegdb.org">www.maizegdb.org</a>). Among the public markers, 1556 SSRs have genomic positions. Through e-PCR programme conducted in B73 genome, 827 SSR markers have specific amplicon, 60 SSR markers have more than one binding sites, and the remaining markers have no proper binding sites on the 10 chromosomes. Here, we developed a new database containing more SSR markers with unique flanking sequences. From the SSRs that could be detected (264 658) across 17 maize genomes, 189 087 (71.45%) of them were identified with unique flanking sequences with an average of 82 741.9 SSR loci for each genome (Table <a reveal-id="DST026TB4" data-open="DST026TB4" class="link link-reveal link-table xref-fig">4</a>). The average numbers of SSR loci with different motifs for each genome were notably different, accounting for 55.19, 74.60, 48.09, 81.94, 80.69, and 68.94% of the total SSRs for mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide motifs, respectively (Table <a reveal-id="DST026TB4" data-open="DST026TB4" class="link link-reveal link-table xref-fig">4</a>). It implies that over 80% of tetra- and penta- nucleotide motifs in the maize genome can be used to design SSR markers. A total of 25 437 SSRs with unique flanking sequences were found to be shared across 17 tested genomes, of which 9240 (36.33%) were polymorphic. </p>                        <a id="5193954" scrollto-destination="5193954"></a>
<div content-id="DST026TB4" class="table-modal"><div class="table-wrap"><div id="DST026TB4" class="table-wrap-title" data-id="DST026TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary of SSR loci with unique flanking sequences identified in tested maize genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="2">Average SSRs<hr /> </th><th colspan="2">Total SSRs<hr /> </th><th colspan="2">Common SSRs<hr /> </th><th colspan="2">Common SSRs with polymorphism<hr /> </th></tr><tr><th>No. </th><th>%<sup>a</sup> </th><th>No. </th><th>%<sup>b</sup> </th><th>No. </th><th>%<sup>c</sup> </th><th>No. </th><th>% </th></tr></thead><tbody><tr><td>MNR </td><td>43 030.4 </td><td>55.19 </td><td>103 486 </td><td>67.54 </td><td>12 029 </td><td>54.31 </td><td>4588 </td><td>38.14 </td></tr><tr><td>DNR </td><td>25 733.2 </td><td>74.6 </td><td>52 876 </td><td>81.05 </td><td>8630 </td><td>79.87 </td><td>3776 </td><td>43.75 </td></tr><tr><td>TNR </td><td>6689.9 </td><td>48.09 </td><td>15 946 </td><td>61.54 </td><td>2755 </td><td>59.53 </td><td>560 </td><td>20.33 </td></tr><tr><td>TTR </td><td>2397.2 </td><td>81.94 </td><td>5658 </td><td>86.09 </td><td>652 </td><td>83.27 </td><td>142 </td><td>21.78 </td></tr><tr><td>PNR </td><td>3355.4 </td><td>80.69 </td><td>7481 </td><td>84.64 </td><td>950 </td><td>83.11 </td><td>129 </td><td>13.58 </td></tr><tr><td>HNR </td><td>1535.8 </td><td>68.94 </td><td>3640 </td><td>74.74 </td><td>422 </td><td>74.17 </td><td>45 </td><td>10.66 </td></tr><tr><td>Total </td><td>82 741.9 </td><td>60.98 </td><td>189 087 </td><td>71.45 </td><td>25 437 </td><td>63.47 </td><td>9240 </td><td>36.33 </td></tr></tbody></table></div><div content-id="DST026TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="2">Average SSRs<hr /> </th><th colspan="2">Total SSRs<hr /> </th><th colspan="2">Common SSRs<hr /> </th><th colspan="2">Common SSRs with polymorphism<hr /> </th></tr><tr><th>No. </th><th>%<sup>a</sup> </th><th>No. </th><th>%<sup>b</sup> </th><th>No. </th><th>%<sup>c</sup> </th><th>No. </th><th>% </th></tr></thead><tbody><tr><td>MNR </td><td>43 030.4 </td><td>55.19 </td><td>103 486 </td><td>67.54 </td><td>12 029 </td><td>54.31 </td><td>4588 </td><td>38.14 </td></tr><tr><td>DNR </td><td>25 733.2 </td><td>74.6 </td><td>52 876 </td><td>81.05 </td><td>8630 </td><td>79.87 </td><td>3776 </td><td>43.75 </td></tr><tr><td>TNR </td><td>6689.9 </td><td>48.09 </td><td>15 946 </td><td>61.54 </td><td>2755 </td><td>59.53 </td><td>560 </td><td>20.33 </td></tr><tr><td>TTR </td><td>2397.2 </td><td>81.94 </td><td>5658 </td><td>86.09 </td><td>652 </td><td>83.27 </td><td>142 </td><td>21.78 </td></tr><tr><td>PNR </td><td>3355.4 </td><td>80.69 </td><td>7481 </td><td>84.64 </td><td>950 </td><td>83.11 </td><td>129 </td><td>13.58 </td></tr><tr><td>HNR </td><td>1535.8 </td><td>68.94 </td><td>3640 </td><td>74.74 </td><td>422 </td><td>74.17 </td><td>45 </td><td>10.66 </td></tr><tr><td>Total </td><td>82 741.9 </td><td>60.98 </td><td>189 087 </td><td>71.45 </td><td>25 437 </td><td>63.47 </td><td>9240 </td><td>36.33 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>MNR, DNR, TNR, TTR, PNR, and HNR indicate mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Percentage of the average number of SSRs with unique flanking sequences against all for every tested maize genome.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Percentage of total SSR number with unique flanking sequences against all identified in 17 maize lines.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Percentage of the common loci against all that are the same in 17 maize lines.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193954" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST026TB4" class="table-wrap-title" data-id="DST026TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary of SSR loci with unique flanking sequences identified in tested maize genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="2">Average SSRs<hr /> </th><th colspan="2">Total SSRs<hr /> </th><th colspan="2">Common SSRs<hr /> </th><th colspan="2">Common SSRs with polymorphism<hr /> </th></tr><tr><th>No. </th><th>%<sup>a</sup> </th><th>No. </th><th>%<sup>b</sup> </th><th>No. </th><th>%<sup>c</sup> </th><th>No. </th><th>% </th></tr></thead><tbody><tr><td>MNR </td><td>43 030.4 </td><td>55.19 </td><td>103 486 </td><td>67.54 </td><td>12 029 </td><td>54.31 </td><td>4588 </td><td>38.14 </td></tr><tr><td>DNR </td><td>25 733.2 </td><td>74.6 </td><td>52 876 </td><td>81.05 </td><td>8630 </td><td>79.87 </td><td>3776 </td><td>43.75 </td></tr><tr><td>TNR </td><td>6689.9 </td><td>48.09 </td><td>15 946 </td><td>61.54 </td><td>2755 </td><td>59.53 </td><td>560 </td><td>20.33 </td></tr><tr><td>TTR </td><td>2397.2 </td><td>81.94 </td><td>5658 </td><td>86.09 </td><td>652 </td><td>83.27 </td><td>142 </td><td>21.78 </td></tr><tr><td>PNR </td><td>3355.4 </td><td>80.69 </td><td>7481 </td><td>84.64 </td><td>950 </td><td>83.11 </td><td>129 </td><td>13.58 </td></tr><tr><td>HNR </td><td>1535.8 </td><td>68.94 </td><td>3640 </td><td>74.74 </td><td>422 </td><td>74.17 </td><td>45 </td><td>10.66 </td></tr><tr><td>Total </td><td>82 741.9 </td><td>60.98 </td><td>189 087 </td><td>71.45 </td><td>25 437 </td><td>63.47 </td><td>9240 </td><td>36.33 </td></tr></tbody></table></div><div content-id="DST026TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Motifs </th><th colspan="2">Average SSRs<hr /> </th><th colspan="2">Total SSRs<hr /> </th><th colspan="2">Common SSRs<hr /> </th><th colspan="2">Common SSRs with polymorphism<hr /> </th></tr><tr><th>No. </th><th>%<sup>a</sup> </th><th>No. </th><th>%<sup>b</sup> </th><th>No. </th><th>%<sup>c</sup> </th><th>No. </th><th>% </th></tr></thead><tbody><tr><td>MNR </td><td>43 030.4 </td><td>55.19 </td><td>103 486 </td><td>67.54 </td><td>12 029 </td><td>54.31 </td><td>4588 </td><td>38.14 </td></tr><tr><td>DNR </td><td>25 733.2 </td><td>74.6 </td><td>52 876 </td><td>81.05 </td><td>8630 </td><td>79.87 </td><td>3776 </td><td>43.75 </td></tr><tr><td>TNR </td><td>6689.9 </td><td>48.09 </td><td>15 946 </td><td>61.54 </td><td>2755 </td><td>59.53 </td><td>560 </td><td>20.33 </td></tr><tr><td>TTR </td><td>2397.2 </td><td>81.94 </td><td>5658 </td><td>86.09 </td><td>652 </td><td>83.27 </td><td>142 </td><td>21.78 </td></tr><tr><td>PNR </td><td>3355.4 </td><td>80.69 </td><td>7481 </td><td>84.64 </td><td>950 </td><td>83.11 </td><td>129 </td><td>13.58 </td></tr><tr><td>HNR </td><td>1535.8 </td><td>68.94 </td><td>3640 </td><td>74.74 </td><td>422 </td><td>74.17 </td><td>45 </td><td>10.66 </td></tr><tr><td>Total </td><td>82 741.9 </td><td>60.98 </td><td>189 087 </td><td>71.45 </td><td>25 437 </td><td>63.47 </td><td>9240 </td><td>36.33 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>MNR, DNR, TNR, TTR, PNR, and HNR indicate mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Percentage of the average number of SSRs with unique flanking sequences against all for every tested maize genome.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Percentage of total SSR number with unique flanking sequences against all identified in 17 maize lines.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Percentage of the common loci against all that are the same in 17 maize lines.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193954" target="_blank">View Large</a></div></div><p>Of 189 087 candidate SSRs, 188 571 loci have specific physical position and would be developed as genetic markers in the study. Primer pairs were then designed for the 188 571 SSR loci, with 13 344 (chromosome 10) to 29 779 (chromosome 1) SSRs on each chromosome, and 173 587 of them were polymorphic with length differences and present-absent variation in 17 genomes. E-PCR programme was further conducted to validate and refine the specificity of new designed SSR markers, and 111 887 primer pairs of them could bind as expected and the others were amplified with multiple binding sites or false match. Through comparing SSR sequences among 17 tested genomes, a new database was developed to include 111 887 SSR markers with specific physical positions, with proportion of 59% of the candidate SSR loci with specific flanking sequences (Table <a reveal-id="DST026TB5" data-open="DST026TB5" class="link link-reveal link-table xref-fig">5</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Among these markers, SSRs with mono-, di-, tri-, tetra-, penta-, and hexa- nucleotide motifs accounted for 58.00, 26.09, 7.20, 3.00, 3.93, and 1.78%, respectively. A total of 35 573 SSR loci, accounting for 31.8% of the refined SSR markers, showed length polymorphism in the 17 tested genotypes. The PIC for these polymorphic SSRs varied from 0.05 to 0.83, with an average of 0.31 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). SSR markers with mono- and di-nucleotide motifs showed higher levels of polymorphism (33.87 and 37.31%, respectively) than other SSR markers with tetra-, penta-, and hexa-nucleotide motifs (7.44–17.19%). Comparing with the SSR markers in MaizeGDB database, there were 18 606 SSR markers, accounting for 16.6% of the newly developed SSR markers, shared the same loci with the public SSR markers with various motifs. However, only 527 (0.47%) newly developed SSR markers had completely compatible position with public SSR primers. Additionally, the average SSR lengths and number of loci across 10 chromosomes were calculated for three SSR datasets, all SSRs, SSRs with unique flanking sequences, and polymorphic SSRs (Fig. <a reveal-id="DST026F1" data-open="DST026F1" class="link link-reveal link-table xref-fig">1</a>). In each of the three SSR datasets, the numbers of loci gradually declined with the increase of SSR lengths, the same as shown in previous studies.<sup><a reveal-id="DST026C20" data-open="DST026C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                        <a id="5193956" scrollto-destination="5193956"></a>
<div content-id="DST026TB5" class="table-modal"><div class="table-wrap"><div id="DST026TB5" class="table-wrap-title" data-id="DST026TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Numbers of candidate SSR markers, and polymorphic SSR markers detected in 17 maize lines and previously developed SSR in MaizeGDB database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Chr </th><th colspan="12">Candidate SSR markers<hr /> </th><th rowspan="3">Poly. (%)<sup>a</sup> </th><th rowspan="3">SSRs in MaizeGDB database </th></tr><tr><th colspan="2">MNR<hr /> </th><th colspan="2">DNR<hr /> </th><th colspan="2">TNR<hr /> </th><th colspan="2">TTR<hr /> </th><th colspan="2">PNR<hr /> </th><th colspan="2">HNR<hr /> </th></tr><tr><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th></tr></thead><tbody><tr><td>1 </td><td>6957 </td><td>3422 </td><td>2958 </td><td>1737 </td><td>1129 </td><td>242 </td><td>439 </td><td>84 </td><td>643 </td><td>73 </td><td>304 </td><td>12 </td><td>30.94 </td><td>293 </td></tr><tr><td>2 </td><td>4875 </td><td>2488 </td><td>2118 </td><td>1212 </td><td>697 </td><td>157 </td><td>294 </td><td>76 </td><td>418 </td><td>46 </td><td>203 </td><td>17 </td><td>31.71 </td><td>226 </td></tr><tr><td>3 </td><td>5079 </td><td>2414 </td><td>2289 </td><td>1217 </td><td>777 </td><td>177 </td><td>330 </td><td>69 </td><td>514 </td><td>49 </td><td>209 </td><td>17 </td><td>30.01 </td><td>224 </td></tr><tr><td>4 </td><td>4831 </td><td>2331 </td><td>2246 </td><td>1264 </td><td>774 </td><td>166 </td><td>331 </td><td>73 </td><td>422 </td><td>45 </td><td>192 </td><td>23 </td><td>30.73 </td><td>141 </td></tr><tr><td>5 </td><td>4623 </td><td>2498 </td><td>1851 </td><td>1140 </td><td>681 </td><td>164 </td><td>276 </td><td>67 </td><td>461 </td><td>55 </td><td>198 </td><td>18 </td><td>32.76 </td><td>146 </td></tr><tr><td>6 </td><td>3501 </td><td>1816 </td><td>1499 </td><td>844 </td><td>579 </td><td>118 </td><td>248 </td><td>44 </td><td>345 </td><td>25 </td><td>158 </td><td>12 </td><td>31.11 </td><td>111 </td></tr><tr><td>7 </td><td>3548 </td><td>1870 </td><td>1473 </td><td>929 </td><td>505 </td><td>150 </td><td>271 </td><td>41 </td><td>285 </td><td>44 </td><td>150 </td><td>12 </td><td>32.83 </td><td>112 </td></tr><tr><td>8 </td><td>3553 </td><td>1845 </td><td>1391 </td><td>901 </td><td>504 </td><td>118 </td><td>185 </td><td>32 </td><td>315 </td><td>50 </td><td>180 </td><td>13 </td><td>32.56 </td><td>118 </td></tr><tr><td>9 </td><td>3059 </td><td>1726 </td><td>1262 </td><td>847 </td><td>451 </td><td>100 </td><td>203 </td><td>39 </td><td>297 </td><td>40 </td><td>137 </td><td>16 </td><td>33.85 </td><td>102 </td></tr><tr><td>10 </td><td>2884 </td><td>1572 </td><td>1214 </td><td>803 </td><td>436 </td><td>127 </td><td>203 </td><td>52 </td><td>248 </td><td>26 </td><td>111 </td><td>8 </td><td>33.68 </td><td>83 </td></tr><tr><td>Total </td><td>42 910 </td><td>21 982 </td><td>18 301 </td><td>10 894 </td><td>6533 </td><td>1519 </td><td>2780 </td><td>577 </td><td>3948 </td><td>453 </td><td>1842 </td><td>148 </td><td>31.79 </td><td>1556 </td></tr></tbody></table></div><div content-id="DST026TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Chr </th><th colspan="12">Candidate SSR markers<hr /> </th><th rowspan="3">Poly. (%)<sup>a</sup> </th><th rowspan="3">SSRs in MaizeGDB database </th></tr><tr><th colspan="2">MNR<hr /> </th><th colspan="2">DNR<hr /> </th><th colspan="2">TNR<hr /> </th><th colspan="2">TTR<hr /> </th><th colspan="2">PNR<hr /> </th><th colspan="2">HNR<hr /> </th></tr><tr><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th></tr></thead><tbody><tr><td>1 </td><td>6957 </td><td>3422 </td><td>2958 </td><td>1737 </td><td>1129 </td><td>242 </td><td>439 </td><td>84 </td><td>643 </td><td>73 </td><td>304 </td><td>12 </td><td>30.94 </td><td>293 </td></tr><tr><td>2 </td><td>4875 </td><td>2488 </td><td>2118 </td><td>1212 </td><td>697 </td><td>157 </td><td>294 </td><td>76 </td><td>418 </td><td>46 </td><td>203 </td><td>17 </td><td>31.71 </td><td>226 </td></tr><tr><td>3 </td><td>5079 </td><td>2414 </td><td>2289 </td><td>1217 </td><td>777 </td><td>177 </td><td>330 </td><td>69 </td><td>514 </td><td>49 </td><td>209 </td><td>17 </td><td>30.01 </td><td>224 </td></tr><tr><td>4 </td><td>4831 </td><td>2331 </td><td>2246 </td><td>1264 </td><td>774 </td><td>166 </td><td>331 </td><td>73 </td><td>422 </td><td>45 </td><td>192 </td><td>23 </td><td>30.73 </td><td>141 </td></tr><tr><td>5 </td><td>4623 </td><td>2498 </td><td>1851 </td><td>1140 </td><td>681 </td><td>164 </td><td>276 </td><td>67 </td><td>461 </td><td>55 </td><td>198 </td><td>18 </td><td>32.76 </td><td>146 </td></tr><tr><td>6 </td><td>3501 </td><td>1816 </td><td>1499 </td><td>844 </td><td>579 </td><td>118 </td><td>248 </td><td>44 </td><td>345 </td><td>25 </td><td>158 </td><td>12 </td><td>31.11 </td><td>111 </td></tr><tr><td>7 </td><td>3548 </td><td>1870 </td><td>1473 </td><td>929 </td><td>505 </td><td>150 </td><td>271 </td><td>41 </td><td>285 </td><td>44 </td><td>150 </td><td>12 </td><td>32.83 </td><td>112 </td></tr><tr><td>8 </td><td>3553 </td><td>1845 </td><td>1391 </td><td>901 </td><td>504 </td><td>118 </td><td>185 </td><td>32 </td><td>315 </td><td>50 </td><td>180 </td><td>13 </td><td>32.56 </td><td>118 </td></tr><tr><td>9 </td><td>3059 </td><td>1726 </td><td>1262 </td><td>847 </td><td>451 </td><td>100 </td><td>203 </td><td>39 </td><td>297 </td><td>40 </td><td>137 </td><td>16 </td><td>33.85 </td><td>102 </td></tr><tr><td>10 </td><td>2884 </td><td>1572 </td><td>1214 </td><td>803 </td><td>436 </td><td>127 </td><td>203 </td><td>52 </td><td>248 </td><td>26 </td><td>111 </td><td>8 </td><td>33.68 </td><td>83 </td></tr><tr><td>Total </td><td>42 910 </td><td>21 982 </td><td>18 301 </td><td>10 894 </td><td>6533 </td><td>1519 </td><td>2780 </td><td>577 </td><td>3948 </td><td>453 </td><td>1842 </td><td>148 </td><td>31.79 </td><td>1556 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>MNR, DNR, TNR, TTR, PNR, and HNR indicate mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs.</p></div></div><div content-id=""><div class="fn"><p>Mono: monomorphism; poly: polymorphism; Chr: chromosome.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Percent of polymorphic SSR markers over all of candidate SSR markers <em>in silicon</em> analysis.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193956" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST026TB5" class="table-wrap-title" data-id="DST026TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Numbers of candidate SSR markers, and polymorphic SSR markers detected in 17 maize lines and previously developed SSR in MaizeGDB database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Chr </th><th colspan="12">Candidate SSR markers<hr /> </th><th rowspan="3">Poly. (%)<sup>a</sup> </th><th rowspan="3">SSRs in MaizeGDB database </th></tr><tr><th colspan="2">MNR<hr /> </th><th colspan="2">DNR<hr /> </th><th colspan="2">TNR<hr /> </th><th colspan="2">TTR<hr /> </th><th colspan="2">PNR<hr /> </th><th colspan="2">HNR<hr /> </th></tr><tr><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th></tr></thead><tbody><tr><td>1 </td><td>6957 </td><td>3422 </td><td>2958 </td><td>1737 </td><td>1129 </td><td>242 </td><td>439 </td><td>84 </td><td>643 </td><td>73 </td><td>304 </td><td>12 </td><td>30.94 </td><td>293 </td></tr><tr><td>2 </td><td>4875 </td><td>2488 </td><td>2118 </td><td>1212 </td><td>697 </td><td>157 </td><td>294 </td><td>76 </td><td>418 </td><td>46 </td><td>203 </td><td>17 </td><td>31.71 </td><td>226 </td></tr><tr><td>3 </td><td>5079 </td><td>2414 </td><td>2289 </td><td>1217 </td><td>777 </td><td>177 </td><td>330 </td><td>69 </td><td>514 </td><td>49 </td><td>209 </td><td>17 </td><td>30.01 </td><td>224 </td></tr><tr><td>4 </td><td>4831 </td><td>2331 </td><td>2246 </td><td>1264 </td><td>774 </td><td>166 </td><td>331 </td><td>73 </td><td>422 </td><td>45 </td><td>192 </td><td>23 </td><td>30.73 </td><td>141 </td></tr><tr><td>5 </td><td>4623 </td><td>2498 </td><td>1851 </td><td>1140 </td><td>681 </td><td>164 </td><td>276 </td><td>67 </td><td>461 </td><td>55 </td><td>198 </td><td>18 </td><td>32.76 </td><td>146 </td></tr><tr><td>6 </td><td>3501 </td><td>1816 </td><td>1499 </td><td>844 </td><td>579 </td><td>118 </td><td>248 </td><td>44 </td><td>345 </td><td>25 </td><td>158 </td><td>12 </td><td>31.11 </td><td>111 </td></tr><tr><td>7 </td><td>3548 </td><td>1870 </td><td>1473 </td><td>929 </td><td>505 </td><td>150 </td><td>271 </td><td>41 </td><td>285 </td><td>44 </td><td>150 </td><td>12 </td><td>32.83 </td><td>112 </td></tr><tr><td>8 </td><td>3553 </td><td>1845 </td><td>1391 </td><td>901 </td><td>504 </td><td>118 </td><td>185 </td><td>32 </td><td>315 </td><td>50 </td><td>180 </td><td>13 </td><td>32.56 </td><td>118 </td></tr><tr><td>9 </td><td>3059 </td><td>1726 </td><td>1262 </td><td>847 </td><td>451 </td><td>100 </td><td>203 </td><td>39 </td><td>297 </td><td>40 </td><td>137 </td><td>16 </td><td>33.85 </td><td>102 </td></tr><tr><td>10 </td><td>2884 </td><td>1572 </td><td>1214 </td><td>803 </td><td>436 </td><td>127 </td><td>203 </td><td>52 </td><td>248 </td><td>26 </td><td>111 </td><td>8 </td><td>33.68 </td><td>83 </td></tr><tr><td>Total </td><td>42 910 </td><td>21 982 </td><td>18 301 </td><td>10 894 </td><td>6533 </td><td>1519 </td><td>2780 </td><td>577 </td><td>3948 </td><td>453 </td><td>1842 </td><td>148 </td><td>31.79 </td><td>1556 </td></tr></tbody></table></div><div content-id="DST026TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Chr </th><th colspan="12">Candidate SSR markers<hr /> </th><th rowspan="3">Poly. (%)<sup>a</sup> </th><th rowspan="3">SSRs in MaizeGDB database </th></tr><tr><th colspan="2">MNR<hr /> </th><th colspan="2">DNR<hr /> </th><th colspan="2">TNR<hr /> </th><th colspan="2">TTR<hr /> </th><th colspan="2">PNR<hr /> </th><th colspan="2">HNR<hr /> </th></tr><tr><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th><th>Mono- </th><th>Poly- </th></tr></thead><tbody><tr><td>1 </td><td>6957 </td><td>3422 </td><td>2958 </td><td>1737 </td><td>1129 </td><td>242 </td><td>439 </td><td>84 </td><td>643 </td><td>73 </td><td>304 </td><td>12 </td><td>30.94 </td><td>293 </td></tr><tr><td>2 </td><td>4875 </td><td>2488 </td><td>2118 </td><td>1212 </td><td>697 </td><td>157 </td><td>294 </td><td>76 </td><td>418 </td><td>46 </td><td>203 </td><td>17 </td><td>31.71 </td><td>226 </td></tr><tr><td>3 </td><td>5079 </td><td>2414 </td><td>2289 </td><td>1217 </td><td>777 </td><td>177 </td><td>330 </td><td>69 </td><td>514 </td><td>49 </td><td>209 </td><td>17 </td><td>30.01 </td><td>224 </td></tr><tr><td>4 </td><td>4831 </td><td>2331 </td><td>2246 </td><td>1264 </td><td>774 </td><td>166 </td><td>331 </td><td>73 </td><td>422 </td><td>45 </td><td>192 </td><td>23 </td><td>30.73 </td><td>141 </td></tr><tr><td>5 </td><td>4623 </td><td>2498 </td><td>1851 </td><td>1140 </td><td>681 </td><td>164 </td><td>276 </td><td>67 </td><td>461 </td><td>55 </td><td>198 </td><td>18 </td><td>32.76 </td><td>146 </td></tr><tr><td>6 </td><td>3501 </td><td>1816 </td><td>1499 </td><td>844 </td><td>579 </td><td>118 </td><td>248 </td><td>44 </td><td>345 </td><td>25 </td><td>158 </td><td>12 </td><td>31.11 </td><td>111 </td></tr><tr><td>7 </td><td>3548 </td><td>1870 </td><td>1473 </td><td>929 </td><td>505 </td><td>150 </td><td>271 </td><td>41 </td><td>285 </td><td>44 </td><td>150 </td><td>12 </td><td>32.83 </td><td>112 </td></tr><tr><td>8 </td><td>3553 </td><td>1845 </td><td>1391 </td><td>901 </td><td>504 </td><td>118 </td><td>185 </td><td>32 </td><td>315 </td><td>50 </td><td>180 </td><td>13 </td><td>32.56 </td><td>118 </td></tr><tr><td>9 </td><td>3059 </td><td>1726 </td><td>1262 </td><td>847 </td><td>451 </td><td>100 </td><td>203 </td><td>39 </td><td>297 </td><td>40 </td><td>137 </td><td>16 </td><td>33.85 </td><td>102 </td></tr><tr><td>10 </td><td>2884 </td><td>1572 </td><td>1214 </td><td>803 </td><td>436 </td><td>127 </td><td>203 </td><td>52 </td><td>248 </td><td>26 </td><td>111 </td><td>8 </td><td>33.68 </td><td>83 </td></tr><tr><td>Total </td><td>42 910 </td><td>21 982 </td><td>18 301 </td><td>10 894 </td><td>6533 </td><td>1519 </td><td>2780 </td><td>577 </td><td>3948 </td><td>453 </td><td>1842 </td><td>148 </td><td>31.79 </td><td>1556 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>MNR, DNR, TNR, TTR, PNR, and HNR indicate mono-, di-, tri-, tetra-, penta-, and hexa-nucleotide SSRs.</p></div></div><div content-id=""><div class="fn"><p>Mono: monomorphism; poly: polymorphism; Chr: chromosome.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Percent of polymorphic SSR markers over all of candidate SSR markers <em>in silicon</em> analysis.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193956" target="_blank">View Large</a></div></div>                        <a id="5193957" scrollto-destination="5193957"></a>
<div id="DST026F1" data-id="DST026F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/m_dst02601.jpeg?Expires=2147483647&Signature=CQagifpG7F0Yysjcua1lSI4VUWmZSLYPJZkLu75SN9~z71nKndvCdfwtWkKJ0EJ55Wks7LTXAxdsbDjsw9XkH3UyMxITBqviQR0Yq7akEbSxg54OyQV-9~WyfWuwntluXBgz9nLo1dinpUeWuDOeBY-wn3zJ7F6RtbC85IsscrD8QjVwhpdxGW3OrPtQw7s0Lyj2JfU8U~fYcDTpMysKfjFz15PVNb75iPvKfQdz1x3pbrG7ApLAV7G4I-7eNjstovvMmf2AJpU4pRGBEHL-NzqwoRyudGYtSAMm0Lv8TmxG44VP4tk1zFeYezCBs1-3xrb9HQKz-W-86zjJh3iiyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributions of 263,423 SSR loci (a) and 111,887 new developed SSR markers (b) with unique physical positions across 10 chromosomes in the B73 reference genome (www.maizesequence.org Release 4a.53). Different colors represent levels of density of SSRs." path-from-xml="dst02601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193957/dst02601.jpeg" path-from-xml="dst02601.jpeg" target="_blank">View large</a><a section="5193957" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst02601.jpeg?Expires=2147483647&Signature=ZCeRw33S4-NU9zVC7XrqNmUPcMnVZV7CzePXG~ij-IWHz4YczpdkmyJy0pgyZgTDSytzGawlw7G7f7Yf~9nAlafM43dkRMywlSv~DSIOuIzZQw29AHbn4K1bQ73uX0OG5MVXKVNIiIFgm9hoKWYtUwDrBi2Xlf6WID8ufk4ifITLtyKOEwMpKYYcbFfVXqjdInTON3yFfLUnUCsimgUUuYnkcQilRswiT4305lBb~Vb7nPwRpkJA0IXGaqJcOr54n49X3fu4RaGymama3XknDbtYHjlnyTwRNytEmLzHbCp0NnWtTYkErWwD7H4UkwWiiHxaqEEfN8epnMZHtP-MRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193957&ar=359997&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of 263,423 SSR loci (a) and 111,887 new developed SSR markers (b) with unique physical positions across 10 chromosomes in the B73 reference genome (www.maizesequence.org Release 4a.53). Different colors represent levels of density of SSRs.</p></div></div><div content-id="DST026F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/m_dst02601.jpeg?Expires=2147483647&Signature=CQagifpG7F0Yysjcua1lSI4VUWmZSLYPJZkLu75SN9~z71nKndvCdfwtWkKJ0EJ55Wks7LTXAxdsbDjsw9XkH3UyMxITBqviQR0Yq7akEbSxg54OyQV-9~WyfWuwntluXBgz9nLo1dinpUeWuDOeBY-wn3zJ7F6RtbC85IsscrD8QjVwhpdxGW3OrPtQw7s0Lyj2JfU8U~fYcDTpMysKfjFz15PVNb75iPvKfQdz1x3pbrG7ApLAV7G4I-7eNjstovvMmf2AJpU4pRGBEHL-NzqwoRyudGYtSAMm0Lv8TmxG44VP4tk1zFeYezCBs1-3xrb9HQKz-W-86zjJh3iiyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributions of 263,423 SSR loci (a) and 111,887 new developed SSR markers (b) with unique physical positions across 10 chromosomes in the B73 reference genome (www.maizesequence.org Release 4a.53). Different colors represent levels of density of SSRs." path-from-xml="dst02601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193957/dst02601.jpeg" path-from-xml="dst02601.jpeg" target="_blank">View large</a><a section="5193957" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst02601.jpeg?Expires=2147483647&Signature=ZCeRw33S4-NU9zVC7XrqNmUPcMnVZV7CzePXG~ij-IWHz4YczpdkmyJy0pgyZgTDSytzGawlw7G7f7Yf~9nAlafM43dkRMywlSv~DSIOuIzZQw29AHbn4K1bQ73uX0OG5MVXKVNIiIFgm9hoKWYtUwDrBi2Xlf6WID8ufk4ifITLtyKOEwMpKYYcbFfVXqjdInTON3yFfLUnUCsimgUUuYnkcQilRswiT4305lBb~Vb7nPwRpkJA0IXGaqJcOr54n49X3fu4RaGymama3XknDbtYHjlnyTwRNytEmLzHbCp0NnWtTYkErWwD7H4UkwWiiHxaqEEfN8epnMZHtP-MRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193957&ar=359997&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of 263,423 SSR loci (a) and 111,887 new developed SSR markers (b) with unique physical positions across 10 chromosomes in the B73 reference genome (www.maizesequence.org Release 4a.53). Different colors represent levels of density of SSRs.</p></div></div>                            <h3 scrollto-destination=5193958 id="5193958" class="section-title" data-legacy-id=s3c>3.3. Distribution of SSRs in different genomic regions</h3>
<p>A total of 264 658 SSRs were detected in 17 genomes, and 263 423 loci of them have specific physical position. The distributions of 263 423 SSR loci and 111 887 newly developed SSR markers refined by e-PCR programme across 17 tested genomes were shown in Fig. <a reveal-id="DST026F2" data-open="DST026F2" class="link link-reveal link-table xref-fig">2</a> a and b , respectively. SSRs were unevenly distributed on chromosome regions, and there were much more loci located in near telomeric regions than near centromeres, which was accordance with the distribution patterns of genes in maize.<sup><a reveal-id="DST026C21" data-open="DST026C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Moreover, we compared SSR distributions across five genomic regions using tested genomes of P1, B73, and <em>Z. mexicana</em> to represent tropical, temperate, and wild maize germplasm, respectively (Table <a reveal-id="DST026TB6" data-open="DST026TB6" class="link link-reveal link-table xref-fig">6</a>). SSR loci were most abundant in intergenic region and least frequent in UTR region. Polymorphism rate and GC content of SSRs in coding regions were higher than other genic regions. </p>                        <a id="5193960" scrollto-destination="5193960"></a>
<div content-id="DST026TB6" class="table-modal"><div class="table-wrap"><div id="DST026TB6" class="table-wrap-title" data-id="DST026TB6"><div class="label -label">Table 6.</div><div class="caption"><p>The distribution of SSRs in different genomic regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="4">All SSR loci<hr /> </th><th colspan="4">SSR loci with unique flanking sequences<hr /> </th><th colspan="4">SSR loci with polymorphism<hr /> </th></tr><tr><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th></tr></thead><tbody><tr><td colspan="13">B73 </td></tr><tr><td> 5′-UTR </td><td>3804 </td><td>6.91 </td><td>16.10 </td><td>32.20 </td><td>3601 </td><td>7.30 </td><td>16.19 </td><td>32.60 </td><td>1335 </td><td>19.69 </td><td>16.23 </td><td>26.55 </td></tr><tr><td> 3′-UTR </td><td>3829 </td><td>6.87 </td><td>16.13 </td><td>33.68 </td><td>3628 </td><td>7.25 </td><td>16.22 </td><td>33.94 </td><td>1341 </td><td>19.62 </td><td>15.90 </td><td>26.70 </td></tr><tr><td> CDS </td><td>2553 </td><td>17.27 </td><td>19.80 </td><td>74.82 </td><td>2391 </td><td>23.10 </td><td>19.87 </td><td>75.28 </td><td>309 </td><td>178.73 </td><td>21.28 </td><td>56.84 </td></tr><tr><td> Intron </td><td>11 857 </td><td>10.13 </td><td>16.03 </td><td>29.94 </td><td>11 038 </td><td>10.88 </td><td>16.12 </td><td>29.97 </td><td>4704 </td><td>25.54 </td><td>16.70 </td><td>29.60 </td></tr><tr><td> Promotors </td><td>10 370 </td><td>6.26 </td><td>17.35 </td><td>27.57 </td><td>9248 </td><td>7.02 </td><td>17.54 </td><td>26.71 </td><td>3344 </td><td>19.42 </td><td>17.26 </td><td>20.82 </td></tr><tr><td> Intergenic </td><td>107 658 </td><td>21.63 </td><td>16.17 </td><td>50.49 </td><td>65 098 </td><td>28.56 </td><td>17.12 </td><td>46.22 </td><td>21 650 </td><td>85.88 </td><td>17.70 </td><td>42.24 </td></tr><tr><td> Total/average </td><td>136 039 </td><td>15.15 </td><td>16.33 </td><td>47.01 </td><td>90 441 </td><td>22.79 </td><td>17.11 </td><td>42.62 </td><td>30 900 </td><td>66.70 </td><td>17.49 </td><td>37.61 </td></tr><tr><td colspan="13"><em>Z. Mexicana</em> </td></tr><tr><td> 5′-UTR </td><td>2520 </td><td>10.25 </td><td>15.55 </td><td>26.23 </td><td>2263 </td><td>11.42 </td><td>15.66 </td><td>24.10 </td><td>1007 </td><td>25.66 </td><td>14.35 </td><td>17.18 </td></tr><tr><td> 3′-UTR </td><td>3329 </td><td>7.76 </td><td>17.34 </td><td>38.34 </td><td>3092 </td><td>8.35 </td><td>17.42 </td><td>38.10 </td><td>1100 </td><td>23.48 </td><td>15.81 </td><td>23.81 </td></tr><tr><td> CDS </td><td>3636 </td><td>14.29 </td><td>17.70 </td><td>45.01 </td><td>3252 </td><td>15.98 </td><td>17.93 </td><td>43.96 </td><td>954 </td><td>54.47 </td><td>16.14 </td><td>23.08 </td></tr><tr><td> Intron </td><td>8102 </td><td>14.23 </td><td>16.24 </td><td>32.41 </td><td>7168 </td><td>16.08 </td><td>16.48 </td><td>30.43 </td><td>3035 </td><td>37.98 </td><td>15.63 </td><td>24.71 </td></tr><tr><td> Promotors </td><td>10 903 </td><td>5.86 </td><td>17.79 </td><td>34.35 </td><td>10 035 </td><td>6.37 </td><td>17.95 </td><td>34.04 </td><td>3627 </td><td>17.62 </td><td>16.32 </td><td>21.71 </td></tr><tr><td> Intergenic </td><td>107 785 </td><td>16.50 </td><td>16.18 </td><td>49.95 </td><td>57 813 </td><td>30.76 </td><td>17.51 </td><td>42.07 </td><td>22 413 </td><td>79.33 </td><td>16.64 </td><td>38.84 </td></tr><tr><td> Total/average </td><td>133 346 </td><td>15.46 </td><td>16.38 </td><td>47.13 </td><td>80 545 </td><td>25.59 </td><td>17.50 </td><td>39.82 </td><td>30 787 </td><td>66.94 </td><td>16.45 </td><td>34.45 </td></tr><tr><td colspan="13">P1 </td></tr><tr><td> 5′-UTR </td><td>2684 </td><td>9.60 </td><td>15.27 </td><td>24.75 </td><td>2419 </td><td>10.65 </td><td>15.37 </td><td>22.50 </td><td>926 </td><td>27.82 </td><td>15.44 </td><td>20.02 </td></tr><tr><td> 3’-UTR </td><td>3478 </td><td>7.41 </td><td>17.04 </td><td>37.07 </td><td>3230 </td><td>7.98 </td><td>17.11 </td><td>36.90 </td><td>1044 </td><td>24.68 </td><td>16.96 </td><td>29.53 </td></tr><tr><td> CDS </td><td>3724 </td><td>13.91 </td><td>17.44 </td><td>44.49 </td><td>3351 </td><td>15.46 </td><td>17.63 </td><td>43.48 </td><td>911 </td><td>56.86 </td><td>17.51 </td><td>28.36 </td></tr><tr><td> Intron </td><td>8622 </td><td>13.32 </td><td>15.96 </td><td>31.18 </td><td>7659 </td><td>14.99 </td><td>16.14 </td><td>29.13 </td><td>3073 </td><td>37.36 </td><td>16.28 </td><td>28.44 </td></tr><tr><td> Promotors </td><td>11 218 </td><td>5.68 </td><td>17.56 </td><td>33.82 </td><td>10 316 </td><td>6.18 </td><td>17.67 </td><td>33.50 </td><td>3486 </td><td>18.28 </td><td>17.56 </td><td>24.74 </td></tr><tr><td> Intergenic </td><td>110 043 </td><td>16.17 </td><td>16.06 </td><td>49.51 </td><td>59 167 </td><td>30.07 </td><td>17.26 </td><td>41.81 </td><td>22 475 </td><td>79.16 </td><td>17.39 </td><td>41.03 </td></tr><tr><td> Total/average </td><td>136 613 </td><td>15.09 </td><td>16.24 </td><td>46.58 </td><td>82 831 </td><td>24.88 </td><td>17.24 </td><td>39.38 </td><td>30 621 </td><td>67.31 </td><td>17.28 </td><td>37.09 </td></tr></tbody></table></div><div content-id="DST026TB6" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="4">All SSR loci<hr /> </th><th colspan="4">SSR loci with unique flanking sequences<hr /> </th><th colspan="4">SSR loci with polymorphism<hr /> </th></tr><tr><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th></tr></thead><tbody><tr><td colspan="13">B73 </td></tr><tr><td> 5′-UTR </td><td>3804 </td><td>6.91 </td><td>16.10 </td><td>32.20 </td><td>3601 </td><td>7.30 </td><td>16.19 </td><td>32.60 </td><td>1335 </td><td>19.69 </td><td>16.23 </td><td>26.55 </td></tr><tr><td> 3′-UTR </td><td>3829 </td><td>6.87 </td><td>16.13 </td><td>33.68 </td><td>3628 </td><td>7.25 </td><td>16.22 </td><td>33.94 </td><td>1341 </td><td>19.62 </td><td>15.90 </td><td>26.70 </td></tr><tr><td> CDS </td><td>2553 </td><td>17.27 </td><td>19.80 </td><td>74.82 </td><td>2391 </td><td>23.10 </td><td>19.87 </td><td>75.28 </td><td>309 </td><td>178.73 </td><td>21.28 </td><td>56.84 </td></tr><tr><td> Intron </td><td>11 857 </td><td>10.13 </td><td>16.03 </td><td>29.94 </td><td>11 038 </td><td>10.88 </td><td>16.12 </td><td>29.97 </td><td>4704 </td><td>25.54 </td><td>16.70 </td><td>29.60 </td></tr><tr><td> Promotors </td><td>10 370 </td><td>6.26 </td><td>17.35 </td><td>27.57 </td><td>9248 </td><td>7.02 </td><td>17.54 </td><td>26.71 </td><td>3344 </td><td>19.42 </td><td>17.26 </td><td>20.82 </td></tr><tr><td> Intergenic </td><td>107 658 </td><td>21.63 </td><td>16.17 </td><td>50.49 </td><td>65 098 </td><td>28.56 </td><td>17.12 </td><td>46.22 </td><td>21 650 </td><td>85.88 </td><td>17.70 </td><td>42.24 </td></tr><tr><td> Total/average </td><td>136 039 </td><td>15.15 </td><td>16.33 </td><td>47.01 </td><td>90 441 </td><td>22.79 </td><td>17.11 </td><td>42.62 </td><td>30 900 </td><td>66.70 </td><td>17.49 </td><td>37.61 </td></tr><tr><td colspan="13"><em>Z. Mexicana</em> </td></tr><tr><td> 5′-UTR </td><td>2520 </td><td>10.25 </td><td>15.55 </td><td>26.23 </td><td>2263 </td><td>11.42 </td><td>15.66 </td><td>24.10 </td><td>1007 </td><td>25.66 </td><td>14.35 </td><td>17.18 </td></tr><tr><td> 3′-UTR </td><td>3329 </td><td>7.76 </td><td>17.34 </td><td>38.34 </td><td>3092 </td><td>8.35 </td><td>17.42 </td><td>38.10 </td><td>1100 </td><td>23.48 </td><td>15.81 </td><td>23.81 </td></tr><tr><td> CDS </td><td>3636 </td><td>14.29 </td><td>17.70 </td><td>45.01 </td><td>3252 </td><td>15.98 </td><td>17.93 </td><td>43.96 </td><td>954 </td><td>54.47 </td><td>16.14 </td><td>23.08 </td></tr><tr><td> Intron </td><td>8102 </td><td>14.23 </td><td>16.24 </td><td>32.41 </td><td>7168 </td><td>16.08 </td><td>16.48 </td><td>30.43 </td><td>3035 </td><td>37.98 </td><td>15.63 </td><td>24.71 </td></tr><tr><td> Promotors </td><td>10 903 </td><td>5.86 </td><td>17.79 </td><td>34.35 </td><td>10 035 </td><td>6.37 </td><td>17.95 </td><td>34.04 </td><td>3627 </td><td>17.62 </td><td>16.32 </td><td>21.71 </td></tr><tr><td> Intergenic </td><td>107 785 </td><td>16.50 </td><td>16.18 </td><td>49.95 </td><td>57 813 </td><td>30.76 </td><td>17.51 </td><td>42.07 </td><td>22 413 </td><td>79.33 </td><td>16.64 </td><td>38.84 </td></tr><tr><td> Total/average </td><td>133 346 </td><td>15.46 </td><td>16.38 </td><td>47.13 </td><td>80 545 </td><td>25.59 </td><td>17.50 </td><td>39.82 </td><td>30 787 </td><td>66.94 </td><td>16.45 </td><td>34.45 </td></tr><tr><td colspan="13">P1 </td></tr><tr><td> 5′-UTR </td><td>2684 </td><td>9.60 </td><td>15.27 </td><td>24.75 </td><td>2419 </td><td>10.65 </td><td>15.37 </td><td>22.50 </td><td>926 </td><td>27.82 </td><td>15.44 </td><td>20.02 </td></tr><tr><td> 3’-UTR </td><td>3478 </td><td>7.41 </td><td>17.04 </td><td>37.07 </td><td>3230 </td><td>7.98 </td><td>17.11 </td><td>36.90 </td><td>1044 </td><td>24.68 </td><td>16.96 </td><td>29.53 </td></tr><tr><td> CDS </td><td>3724 </td><td>13.91 </td><td>17.44 </td><td>44.49 </td><td>3351 </td><td>15.46 </td><td>17.63 </td><td>43.48 </td><td>911 </td><td>56.86 </td><td>17.51 </td><td>28.36 </td></tr><tr><td> Intron </td><td>8622 </td><td>13.32 </td><td>15.96 </td><td>31.18 </td><td>7659 </td><td>14.99 </td><td>16.14 </td><td>29.13 </td><td>3073 </td><td>37.36 </td><td>16.28 </td><td>28.44 </td></tr><tr><td> Promotors </td><td>11 218 </td><td>5.68 </td><td>17.56 </td><td>33.82 </td><td>10 316 </td><td>6.18 </td><td>17.67 </td><td>33.50 </td><td>3486 </td><td>18.28 </td><td>17.56 </td><td>24.74 </td></tr><tr><td> Intergenic </td><td>110 043 </td><td>16.17 </td><td>16.06 </td><td>49.51 </td><td>59 167 </td><td>30.07 </td><td>17.26 </td><td>41.81 </td><td>22 475 </td><td>79.16 </td><td>17.39 </td><td>41.03 </td></tr><tr><td> Total/average </td><td>136 613 </td><td>15.09 </td><td>16.24 </td><td>46.58 </td><td>82 831 </td><td>24.88 </td><td>17.24 </td><td>39.38 </td><td>30 621 </td><td>67.31 </td><td>17.28 </td><td>37.09 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193960" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST026TB6" class="table-wrap-title" data-id="DST026TB6"><div class="label -label">Table 6.</div><div class="caption"><p>The distribution of SSRs in different genomic regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="4">All SSR loci<hr /> </th><th colspan="4">SSR loci with unique flanking sequences<hr /> </th><th colspan="4">SSR loci with polymorphism<hr /> </th></tr><tr><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th></tr></thead><tbody><tr><td colspan="13">B73 </td></tr><tr><td> 5′-UTR </td><td>3804 </td><td>6.91 </td><td>16.10 </td><td>32.20 </td><td>3601 </td><td>7.30 </td><td>16.19 </td><td>32.60 </td><td>1335 </td><td>19.69 </td><td>16.23 </td><td>26.55 </td></tr><tr><td> 3′-UTR </td><td>3829 </td><td>6.87 </td><td>16.13 </td><td>33.68 </td><td>3628 </td><td>7.25 </td><td>16.22 </td><td>33.94 </td><td>1341 </td><td>19.62 </td><td>15.90 </td><td>26.70 </td></tr><tr><td> CDS </td><td>2553 </td><td>17.27 </td><td>19.80 </td><td>74.82 </td><td>2391 </td><td>23.10 </td><td>19.87 </td><td>75.28 </td><td>309 </td><td>178.73 </td><td>21.28 </td><td>56.84 </td></tr><tr><td> Intron </td><td>11 857 </td><td>10.13 </td><td>16.03 </td><td>29.94 </td><td>11 038 </td><td>10.88 </td><td>16.12 </td><td>29.97 </td><td>4704 </td><td>25.54 </td><td>16.70 </td><td>29.60 </td></tr><tr><td> Promotors </td><td>10 370 </td><td>6.26 </td><td>17.35 </td><td>27.57 </td><td>9248 </td><td>7.02 </td><td>17.54 </td><td>26.71 </td><td>3344 </td><td>19.42 </td><td>17.26 </td><td>20.82 </td></tr><tr><td> Intergenic </td><td>107 658 </td><td>21.63 </td><td>16.17 </td><td>50.49 </td><td>65 098 </td><td>28.56 </td><td>17.12 </td><td>46.22 </td><td>21 650 </td><td>85.88 </td><td>17.70 </td><td>42.24 </td></tr><tr><td> Total/average </td><td>136 039 </td><td>15.15 </td><td>16.33 </td><td>47.01 </td><td>90 441 </td><td>22.79 </td><td>17.11 </td><td>42.62 </td><td>30 900 </td><td>66.70 </td><td>17.49 </td><td>37.61 </td></tr><tr><td colspan="13"><em>Z. Mexicana</em> </td></tr><tr><td> 5′-UTR </td><td>2520 </td><td>10.25 </td><td>15.55 </td><td>26.23 </td><td>2263 </td><td>11.42 </td><td>15.66 </td><td>24.10 </td><td>1007 </td><td>25.66 </td><td>14.35 </td><td>17.18 </td></tr><tr><td> 3′-UTR </td><td>3329 </td><td>7.76 </td><td>17.34 </td><td>38.34 </td><td>3092 </td><td>8.35 </td><td>17.42 </td><td>38.10 </td><td>1100 </td><td>23.48 </td><td>15.81 </td><td>23.81 </td></tr><tr><td> CDS </td><td>3636 </td><td>14.29 </td><td>17.70 </td><td>45.01 </td><td>3252 </td><td>15.98 </td><td>17.93 </td><td>43.96 </td><td>954 </td><td>54.47 </td><td>16.14 </td><td>23.08 </td></tr><tr><td> Intron </td><td>8102 </td><td>14.23 </td><td>16.24 </td><td>32.41 </td><td>7168 </td><td>16.08 </td><td>16.48 </td><td>30.43 </td><td>3035 </td><td>37.98 </td><td>15.63 </td><td>24.71 </td></tr><tr><td> Promotors </td><td>10 903 </td><td>5.86 </td><td>17.79 </td><td>34.35 </td><td>10 035 </td><td>6.37 </td><td>17.95 </td><td>34.04 </td><td>3627 </td><td>17.62 </td><td>16.32 </td><td>21.71 </td></tr><tr><td> Intergenic </td><td>107 785 </td><td>16.50 </td><td>16.18 </td><td>49.95 </td><td>57 813 </td><td>30.76 </td><td>17.51 </td><td>42.07 </td><td>22 413 </td><td>79.33 </td><td>16.64 </td><td>38.84 </td></tr><tr><td> Total/average </td><td>133 346 </td><td>15.46 </td><td>16.38 </td><td>47.13 </td><td>80 545 </td><td>25.59 </td><td>17.50 </td><td>39.82 </td><td>30 787 </td><td>66.94 </td><td>16.45 </td><td>34.45 </td></tr><tr><td colspan="13">P1 </td></tr><tr><td> 5′-UTR </td><td>2684 </td><td>9.60 </td><td>15.27 </td><td>24.75 </td><td>2419 </td><td>10.65 </td><td>15.37 </td><td>22.50 </td><td>926 </td><td>27.82 </td><td>15.44 </td><td>20.02 </td></tr><tr><td> 3’-UTR </td><td>3478 </td><td>7.41 </td><td>17.04 </td><td>37.07 </td><td>3230 </td><td>7.98 </td><td>17.11 </td><td>36.90 </td><td>1044 </td><td>24.68 </td><td>16.96 </td><td>29.53 </td></tr><tr><td> CDS </td><td>3724 </td><td>13.91 </td><td>17.44 </td><td>44.49 </td><td>3351 </td><td>15.46 </td><td>17.63 </td><td>43.48 </td><td>911 </td><td>56.86 </td><td>17.51 </td><td>28.36 </td></tr><tr><td> Intron </td><td>8622 </td><td>13.32 </td><td>15.96 </td><td>31.18 </td><td>7659 </td><td>14.99 </td><td>16.14 </td><td>29.13 </td><td>3073 </td><td>37.36 </td><td>16.28 </td><td>28.44 </td></tr><tr><td> Promotors </td><td>11 218 </td><td>5.68 </td><td>17.56 </td><td>33.82 </td><td>10 316 </td><td>6.18 </td><td>17.67 </td><td>33.50 </td><td>3486 </td><td>18.28 </td><td>17.56 </td><td>24.74 </td></tr><tr><td> Intergenic </td><td>110 043 </td><td>16.17 </td><td>16.06 </td><td>49.51 </td><td>59 167 </td><td>30.07 </td><td>17.26 </td><td>41.81 </td><td>22 475 </td><td>79.16 </td><td>17.39 </td><td>41.03 </td></tr><tr><td> Total/average </td><td>136 613 </td><td>15.09 </td><td>16.24 </td><td>46.58 </td><td>82 831 </td><td>24.88 </td><td>17.24 </td><td>39.38 </td><td>30 621 </td><td>67.31 </td><td>17.28 </td><td>37.09 </td></tr></tbody></table></div><div content-id="DST026TB6" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="4">All SSR loci<hr /> </th><th colspan="4">SSR loci with unique flanking sequences<hr /> </th><th colspan="4">SSR loci with polymorphism<hr /> </th></tr><tr><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th><th>Count </th><th>Interval (kb) </th><th>Length (bp) </th><th>GC% </th></tr></thead><tbody><tr><td colspan="13">B73 </td></tr><tr><td> 5′-UTR </td><td>3804 </td><td>6.91 </td><td>16.10 </td><td>32.20 </td><td>3601 </td><td>7.30 </td><td>16.19 </td><td>32.60 </td><td>1335 </td><td>19.69 </td><td>16.23 </td><td>26.55 </td></tr><tr><td> 3′-UTR </td><td>3829 </td><td>6.87 </td><td>16.13 </td><td>33.68 </td><td>3628 </td><td>7.25 </td><td>16.22 </td><td>33.94 </td><td>1341 </td><td>19.62 </td><td>15.90 </td><td>26.70 </td></tr><tr><td> CDS </td><td>2553 </td><td>17.27 </td><td>19.80 </td><td>74.82 </td><td>2391 </td><td>23.10 </td><td>19.87 </td><td>75.28 </td><td>309 </td><td>178.73 </td><td>21.28 </td><td>56.84 </td></tr><tr><td> Intron </td><td>11 857 </td><td>10.13 </td><td>16.03 </td><td>29.94 </td><td>11 038 </td><td>10.88 </td><td>16.12 </td><td>29.97 </td><td>4704 </td><td>25.54 </td><td>16.70 </td><td>29.60 </td></tr><tr><td> Promotors </td><td>10 370 </td><td>6.26 </td><td>17.35 </td><td>27.57 </td><td>9248 </td><td>7.02 </td><td>17.54 </td><td>26.71 </td><td>3344 </td><td>19.42 </td><td>17.26 </td><td>20.82 </td></tr><tr><td> Intergenic </td><td>107 658 </td><td>21.63 </td><td>16.17 </td><td>50.49 </td><td>65 098 </td><td>28.56 </td><td>17.12 </td><td>46.22 </td><td>21 650 </td><td>85.88 </td><td>17.70 </td><td>42.24 </td></tr><tr><td> Total/average </td><td>136 039 </td><td>15.15 </td><td>16.33 </td><td>47.01 </td><td>90 441 </td><td>22.79 </td><td>17.11 </td><td>42.62 </td><td>30 900 </td><td>66.70 </td><td>17.49 </td><td>37.61 </td></tr><tr><td colspan="13"><em>Z. Mexicana</em> </td></tr><tr><td> 5′-UTR </td><td>2520 </td><td>10.25 </td><td>15.55 </td><td>26.23 </td><td>2263 </td><td>11.42 </td><td>15.66 </td><td>24.10 </td><td>1007 </td><td>25.66 </td><td>14.35 </td><td>17.18 </td></tr><tr><td> 3′-UTR </td><td>3329 </td><td>7.76 </td><td>17.34 </td><td>38.34 </td><td>3092 </td><td>8.35 </td><td>17.42 </td><td>38.10 </td><td>1100 </td><td>23.48 </td><td>15.81 </td><td>23.81 </td></tr><tr><td> CDS </td><td>3636 </td><td>14.29 </td><td>17.70 </td><td>45.01 </td><td>3252 </td><td>15.98 </td><td>17.93 </td><td>43.96 </td><td>954 </td><td>54.47 </td><td>16.14 </td><td>23.08 </td></tr><tr><td> Intron </td><td>8102 </td><td>14.23 </td><td>16.24 </td><td>32.41 </td><td>7168 </td><td>16.08 </td><td>16.48 </td><td>30.43 </td><td>3035 </td><td>37.98 </td><td>15.63 </td><td>24.71 </td></tr><tr><td> Promotors </td><td>10 903 </td><td>5.86 </td><td>17.79 </td><td>34.35 </td><td>10 035 </td><td>6.37 </td><td>17.95 </td><td>34.04 </td><td>3627 </td><td>17.62 </td><td>16.32 </td><td>21.71 </td></tr><tr><td> Intergenic </td><td>107 785 </td><td>16.50 </td><td>16.18 </td><td>49.95 </td><td>57 813 </td><td>30.76 </td><td>17.51 </td><td>42.07 </td><td>22 413 </td><td>79.33 </td><td>16.64 </td><td>38.84 </td></tr><tr><td> Total/average </td><td>133 346 </td><td>15.46 </td><td>16.38 </td><td>47.13 </td><td>80 545 </td><td>25.59 </td><td>17.50 </td><td>39.82 </td><td>30 787 </td><td>66.94 </td><td>16.45 </td><td>34.45 </td></tr><tr><td colspan="13">P1 </td></tr><tr><td> 5′-UTR </td><td>2684 </td><td>9.60 </td><td>15.27 </td><td>24.75 </td><td>2419 </td><td>10.65 </td><td>15.37 </td><td>22.50 </td><td>926 </td><td>27.82 </td><td>15.44 </td><td>20.02 </td></tr><tr><td> 3’-UTR </td><td>3478 </td><td>7.41 </td><td>17.04 </td><td>37.07 </td><td>3230 </td><td>7.98 </td><td>17.11 </td><td>36.90 </td><td>1044 </td><td>24.68 </td><td>16.96 </td><td>29.53 </td></tr><tr><td> CDS </td><td>3724 </td><td>13.91 </td><td>17.44 </td><td>44.49 </td><td>3351 </td><td>15.46 </td><td>17.63 </td><td>43.48 </td><td>911 </td><td>56.86 </td><td>17.51 </td><td>28.36 </td></tr><tr><td> Intron </td><td>8622 </td><td>13.32 </td><td>15.96 </td><td>31.18 </td><td>7659 </td><td>14.99 </td><td>16.14 </td><td>29.13 </td><td>3073 </td><td>37.36 </td><td>16.28 </td><td>28.44 </td></tr><tr><td> Promotors </td><td>11 218 </td><td>5.68 </td><td>17.56 </td><td>33.82 </td><td>10 316 </td><td>6.18 </td><td>17.67 </td><td>33.50 </td><td>3486 </td><td>18.28 </td><td>17.56 </td><td>24.74 </td></tr><tr><td> Intergenic </td><td>110 043 </td><td>16.17 </td><td>16.06 </td><td>49.51 </td><td>59 167 </td><td>30.07 </td><td>17.26 </td><td>41.81 </td><td>22 475 </td><td>79.16 </td><td>17.39 </td><td>41.03 </td></tr><tr><td> Total/average </td><td>136 613 </td><td>15.09 </td><td>16.24 </td><td>46.58 </td><td>82 831 </td><td>24.88 </td><td>17.24 </td><td>39.38 </td><td>30 621 </td><td>67.31 </td><td>17.28 </td><td>37.09 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5193960" target="_blank">View Large</a></div></div>                        <a id="5193961" scrollto-destination="5193961"></a>
<div id="DST026F2" data-id="DST026F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/m_dst02602.jpeg?Expires=2147483647&Signature=y0WSfcufoZFs7AE6d-0u0GUOqi20aFsIJIZErcVn4Yw9Ptrz8aD~EpgmjWJivfOdn-qVeqJOJsOLO0EgZ3Xr-oWxNY6ueR0bVAWZhjip~2dhnwT9fwRCHueN2Gee3l8H6ch22-ovxjTzyTeaXbziKfBBj9qzHVpw8etwifiIsGKXju4X34fM2cYQXJov3oRIP5KKvRL0mSITbX9nMJ3cR7CpysOyM7bwReEG13QJJDT-O4eofwYACcolLBO05RbdqMAlBnvH4OjFhSJDfyk8w5bhJ9r~r3SNpqTw1HS0jZ7ti6Ktr~EDf9xrRYxCFZZvZAgB-brBub9juASfkjTmbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between SSR numbers and SSR lengths." path-from-xml="dst02602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193961/dst02602.jpeg" path-from-xml="dst02602.jpeg" target="_blank">View large</a><a section="5193961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst02602.jpeg?Expires=2147483647&Signature=1AuNK6hJMcWD5JxIdMkNhrrtVMw049PO2zSq0Cu6xLZcbfIDMPgk53MFSvwKP~KoQUwQyFeFSfuunWb6Op0FoAjdB1vj6y6unZgN~go9sotPTQeiEIFbNMcQCg3IDolcwET8nWnhbPCgZRELHPEPCA-qWO8gwPx5YhwEv7E1asKx11fNj58wXNvUdY0VkuC0OKqx5qMKAeSDHRlZjglXyoyelPMOu-Cz1TYuKJ-ZIyW-cTkulib3lbnacABbGiYTp3YPJzr7IrAlvpqX4O29lcdg8IVgGmaXaMrLJKUV9XA~YXAsU8SoTFStVa8~kSoyhFDViDB~YB~4l3RfRJ8VPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193961&ar=359997&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between SSR numbers and SSR lengths.</p></div></div><div content-id="DST026F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/m_dst02602.jpeg?Expires=2147483647&Signature=y0WSfcufoZFs7AE6d-0u0GUOqi20aFsIJIZErcVn4Yw9Ptrz8aD~EpgmjWJivfOdn-qVeqJOJsOLO0EgZ3Xr-oWxNY6ueR0bVAWZhjip~2dhnwT9fwRCHueN2Gee3l8H6ch22-ovxjTzyTeaXbziKfBBj9qzHVpw8etwifiIsGKXju4X34fM2cYQXJov3oRIP5KKvRL0mSITbX9nMJ3cR7CpysOyM7bwReEG13QJJDT-O4eofwYACcolLBO05RbdqMAlBnvH4OjFhSJDfyk8w5bhJ9r~r3SNpqTw1HS0jZ7ti6Ktr~EDf9xrRYxCFZZvZAgB-brBub9juASfkjTmbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between SSR numbers and SSR lengths." path-from-xml="dst02602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193961/dst02602.jpeg" path-from-xml="dst02602.jpeg" target="_blank">View large</a><a section="5193961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst02602.jpeg?Expires=2147483647&Signature=1AuNK6hJMcWD5JxIdMkNhrrtVMw049PO2zSq0Cu6xLZcbfIDMPgk53MFSvwKP~KoQUwQyFeFSfuunWb6Op0FoAjdB1vj6y6unZgN~go9sotPTQeiEIFbNMcQCg3IDolcwET8nWnhbPCgZRELHPEPCA-qWO8gwPx5YhwEv7E1asKx11fNj58wXNvUdY0VkuC0OKqx5qMKAeSDHRlZjglXyoyelPMOu-Cz1TYuKJ-ZIyW-cTkulib3lbnacABbGiYTp3YPJzr7IrAlvpqX4O29lcdg8IVgGmaXaMrLJKUV9XA~YXAsU8SoTFStVa8~kSoyhFDViDB~YB~4l3RfRJ8VPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193961&ar=359997&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between SSR numbers and SSR lengths.</p></div></div><p>The average intervals between SSRs were the longest in intergenic regions, second in CDS regions, and smallest in promotors (Table <a reveal-id="DST026TB6" data-open="DST026TB6" class="link link-reveal link-table xref-fig">6</a>). Distributions for the SSRs with unique flanking sequences and for the polymorphic SSRs across tested genomes also varied among the six genomic regions, but the trend was consistent with that for all the candidate SSR loci. This result is also in full agreement with a previous report in rice.<sup><a reveal-id="DST026C22" data-open="DST026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> In addition, SSR distribution was rather similar among the three representative genotypes.</p><p>Furthermore, the repeat types of SSRs in CDS region of B73 were investigated. SSRs with tri-nucleotide repeats were the most (1832) among the six repeat types, with proportion of 71.8% in CDS region. The tri- and hexa-nucleotide SSRs that would not bring the frame shift accounted for 84.1% (2148) of the SSRs in CDS region. Therefore, only 15.9% of the SSRs in CDS region have potential threats to the gene structure.</p>                            <h3 scrollto-destination=5193964 id="5193964" class="section-title" data-legacy-id=s3d>3.4. SSR markers validated for quality and polymorphism</h3>
<p>A total of 151 SSR markers were randomly chosen for experimental validation using 20 maize inbreds and 7 teosinte lines (Fig. <a reveal-id="DST026F3" data-open="DST026F3" class="link link-reveal link-table xref-fig">3</a> and Table <a reveal-id="DST026TB1" data-open="DST026TB1" class="link link-reveal link-table xref-fig">1</a>). Of them, 121 primer pairs (80.1%) generated specific products and distinct bands, while 30 primer pairs failed to produce stable or clear bands due to the lack of sequence specificity in the genomic DNA samples. The majority of the 121 primer pairs (112 primer pairs) revealed high levels of allelic diversity in tested 27 lines, with PIC values of 0.074–0.796 (an average of 0.478). The 112 polymorphic SSR loci contained 329 alleles in total and an average of 2.94 alleles with a range of 2–5 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). </p>                        <a id="5193966" scrollto-destination="5193966"></a>
<div id="DST026F3" data-id="DST026F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/m_dst02603.jpeg?Expires=2147483647&Signature=2uqm1T-yii1Hx9HkZxwREq3Ayv9AYn2y89ZvGVk-1pyk4f52Z3tIuZo0KWnbT8yLWHY0r17rqje3nEQ-KrgU2Hl8WqZrzRAPvWwMHLth2Vp6hWNUAT1vlL5TYJ5DJrTj5N7FjILW5IXvAMzDHfvCA-1kaz74k6TiyCcoWRvU9SJFD9SD5CcN80RQudpHriygc4PzSOL6WDmzvlXzwwYzAfHJMyPRrTAQd1ryTB7eFlQkFCAJeuOAyqfijXDzCF635sdnlv-yKrtOWW7IVGo9xRa96GV1BGb15Vi5yJnKhFQ2SgYcoyxlkmDFebCXdDrb8mdMFTcPEai45QIVOd4Lgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of six randomly selected SSR markers in 27 genotypes. Lanes 1–27 were PCR products of Zea perennis, Z. diploperennis, Z. mays ssp. parviglumis, Z. mays ssp. huehuetenangensis, Z. nicaraguensis, Z. luxurians, Z. mays ssp. mexicana, RP125, 18Red, 18White, CML206, 81565, A318, P1, Han21, CML85, CML411, Ye478, Mo17, Zheng22, 178, 48-2, B73, Lu9801, ES40, Huangzao4, and Dan598, respectively." path-from-xml="dst02603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193966/dst02603.jpeg" path-from-xml="dst02603.jpeg" target="_blank">View large</a><a section="5193966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst02603.jpeg?Expires=2147483647&Signature=YgRmE~E2dp9hsJ2moQ5Mg7NQsz2QczdSI1l-9nST1OOv-g5~z-oId4pDFP-wWsNG78nzXqpuH-CL76OqBLiuWhDz4JeOXHkGEZ~JZ~sSOvPyraCPJMj1zuGTUvKL7aWirTs2aTupn6RtXL-ao2OOQD73Ri3IOZENMi6~avjaB5eEiPb2PLxYRsFniVkYhem~luzPVUivp8nGAiZI~aPobXVNxJVu2ENjNdIFrO4JipXvnADE8HpKWHYZE6A2Uz4ZyUhXHvxFTNCa3odv7NE2NBjuiVFXTKnL1XuRaM1ZgYDKHYI1HwLhv9zzORXJ3jbyhO~xpEZeo-LEHSyou6ymeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193966&ar=359997&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of six randomly selected SSR markers in 27 genotypes. Lanes 1–27 were PCR products of <em>Zea perennis</em>, <em>Z. diploperennis</em>, <em>Z. mays</em> ssp. <em>parviglumis</em>, <em>Z. mays</em> ssp. <em>huehuetenangensis</em>, <em>Z. nicaraguensis</em>, <em>Z. luxurians</em>, <em>Z. mays</em> ssp. <em>mexicana</em>, RP125, 18Red, 18White, CML206, 81565, A318, P1, Han21, CML85, CML411, Ye478, Mo17, Zheng22, 178, 48-2, B73, Lu9801, ES40, Huangzao4, and Dan598, respectively.</p></div></div><div content-id="DST026F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/m_dst02603.jpeg?Expires=2147483647&Signature=2uqm1T-yii1Hx9HkZxwREq3Ayv9AYn2y89ZvGVk-1pyk4f52Z3tIuZo0KWnbT8yLWHY0r17rqje3nEQ-KrgU2Hl8WqZrzRAPvWwMHLth2Vp6hWNUAT1vlL5TYJ5DJrTj5N7FjILW5IXvAMzDHfvCA-1kaz74k6TiyCcoWRvU9SJFD9SD5CcN80RQudpHriygc4PzSOL6WDmzvlXzwwYzAfHJMyPRrTAQd1ryTB7eFlQkFCAJeuOAyqfijXDzCF635sdnlv-yKrtOWW7IVGo9xRa96GV1BGb15Vi5yJnKhFQ2SgYcoyxlkmDFebCXdDrb8mdMFTcPEai45QIVOd4Lgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of six randomly selected SSR markers in 27 genotypes. Lanes 1–27 were PCR products of Zea perennis, Z. diploperennis, Z. mays ssp. parviglumis, Z. mays ssp. huehuetenangensis, Z. nicaraguensis, Z. luxurians, Z. mays ssp. mexicana, RP125, 18Red, 18White, CML206, 81565, A318, P1, Han21, CML85, CML411, Ye478, Mo17, Zheng22, 178, 48-2, B73, Lu9801, ES40, Huangzao4, and Dan598, respectively." path-from-xml="dst02603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5193966/dst02603.jpeg" path-from-xml="dst02603.jpeg" target="_blank">View large</a><a section="5193966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst02603.jpeg?Expires=2147483647&Signature=YgRmE~E2dp9hsJ2moQ5Mg7NQsz2QczdSI1l-9nST1OOv-g5~z-oId4pDFP-wWsNG78nzXqpuH-CL76OqBLiuWhDz4JeOXHkGEZ~JZ~sSOvPyraCPJMj1zuGTUvKL7aWirTs2aTupn6RtXL-ao2OOQD73Ri3IOZENMi6~avjaB5eEiPb2PLxYRsFniVkYhem~luzPVUivp8nGAiZI~aPobXVNxJVu2ENjNdIFrO4JipXvnADE8HpKWHYZE6A2Uz4ZyUhXHvxFTNCa3odv7NE2NBjuiVFXTKnL1XuRaM1ZgYDKHYI1HwLhv9zzORXJ3jbyhO~xpEZeo-LEHSyou6ymeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5193966&ar=359997&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of six randomly selected SSR markers in 27 genotypes. Lanes 1–27 were PCR products of <em>Zea perennis</em>, <em>Z. diploperennis</em>, <em>Z. mays</em> ssp. <em>parviglumis</em>, <em>Z. mays</em> ssp. <em>huehuetenangensis</em>, <em>Z. nicaraguensis</em>, <em>Z. luxurians</em>, <em>Z. mays</em> ssp. <em>mexicana</em>, RP125, 18Red, 18White, CML206, 81565, A318, P1, Han21, CML85, CML411, Ye478, Mo17, Zheng22, 178, 48-2, B73, Lu9801, ES40, Huangzao4, and Dan598, respectively.</p></div></div><p>In addition, we made a detailed comparison of the allele number and PIC value <em>in silicon</em> analysis and in maker validation in 17 tested genotypes. Forty-one of the 121 primers possessed the practical alleles in accordance with the expected alleles, 38 primer pairs had more allele number, and 42 primer pairs had less allele number <em>in silicon</em> analysis than in maker validation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Additionally, comparing polymorphism <em>in silicon</em> analysis using 17 tested genomes and in maker validation using 27 genotypes, we found that 51 primer pairs showed more alleles and higher PIC values in validation experiment (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Interestingly, 26 of 151 SSR markers with no polymorphism <em>in silicon</em> analysis showed more than one alleles in validation experiment. We also found that the length of PCR products <em>in silicon</em> analysis almost consist with those in marker validation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>). The results indicate that newly developed SSR markers are informative and useful, and 70% of the SSR markers in our database are valid and polymorphic.</p>                            <h2 scrollto-destination=5193968 id="5193968" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>SSRs are co-dominant, abundant, high polymorphic, and dispersed throughout plant genomes. Based on the survey across genomes, on average one SSR was found every 1.14 kb in <em>Arabidopsis</em>,<sup><a reveal-id="DST026C23" data-open="DST026C23" class="link link-ref link-reveal xref-bibr">23</a></sup> 3.6 kb in rice,<sup><a reveal-id="DST026C22" data-open="DST026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> 4 kb in <em>Brassica oleracea</em>,<sup><a reveal-id="DST026C24" data-open="DST026C24" class="link link-ref link-reveal xref-bibr">24</a></sup> 4.5 kb in soybean,<sup><a reveal-id="DST026C10" data-open="DST026C10" class="link link-ref link-reveal xref-bibr">10</a></sup> 220 kb in sorghum,<sup><a reveal-id="DST026C25" data-open="DST026C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and 578 kb in wheat.<sup><a reveal-id="DST026C26" data-open="DST026C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In this study, average SSR density was one SSR every 15.48 kb. These may reflect real genetic differences existing among plant genomes at DNA level, and also the differences involved in sequencing methods and procedures. We used maize inbred B73 as the reference genome, some reads from maize wild relative, Z. <em>mexicana</em>, could not be mapped onto the reference, resulting in a relatively lower genome coverage and thus, less SSRs identified compared with other maize inbreds. Therefore, the number of SSR loci identified from Z. <em>Mexicana</em> may be underestimated.</p><p>In general, a small difference in SSR distribution was found for different populations or ecotypes in the same species. For instance, a very similar SSR distribution was found between indica and japonica rice, and the SSR density (interval between two SSRs) varied from one SSR every 2.0–8.1 kb, which was higher in 5′-UTR (one SSR every 2.1 and 2.0 kb, respectively) but low in CDS regions (one SSR every 8.1 and 7.7 kb, respectively).<sup><a reveal-id="DST026C22" data-open="DST026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Our study revealed a similar SSR distribution pattern across the tested temperate, tropical, and wild maize lines. However, SSRs are not evenly distributed in different genomic regions with much lower SSR density in CDS region than in UTR and intronic regions. Intriguingly, we found that majority of SSRs resided in CDS region were tri-nucleotide repeats, which was consistent with other report and implied the specific selection against frame shift mutations in coding regions.<sup><a reveal-id="DST026C27" data-open="DST026C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Comparing with rice genome, the average SSR length was approximately identical (16–17 bp), but the average GC content in maize SSR sequences was much higher (47%) than rice (27%). In the maize genome, the proportions of mono-, di-, and tri-nucleotide SSR motifs were ∼60, 20, and 10%, respectively. Tetra-, penta-, and hexa-nucleotide SSR motifs were less abundant, together accounting for 10%, which was accordance with the report in rice.<sup><a reveal-id="DST026C22" data-open="DST026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> SSR densities for different motifs were also unbalanced and the average interval varied from 26 to 950 kb. Moreover, we found that C/G, AT, and CAG/CTG repeats were the most common for mono-, di-, and tri-nucleotide SSRs, respectively, in maize, while A/T, AG, and AGG/CCT repeats are the most common in rice.<sup><a reveal-id="DST026C22" data-open="DST026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Meanwhile, AT repeats were also the most common dinucleotide motifs in sorghum.<sup><a reveal-id="DST026C25" data-open="DST026C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>Short-read data from next-generation sequencing technologies are now being generated across a range of research projects. The fidelity of this data can be affected by several factors, and mapping errors and gaps still exist to a certain extent.<sup><a reveal-id="DST026C28" data-open="DST026C28" class="link link-ref link-reveal xref-bibr">28</a></sup> However, the availability of the maize genome sequence still affords us a simple and economical way to survey and identify markers, thus enabling us to develop more convenient molecular markers for breeding applications. Several sets of maize germplasm including temperate, tropical, and their wild relatives were resequenced using next-generation sequencing technology.<sup><a reveal-id="DST026C18" data-open="DST026C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST026C29  DST026C31" data-open="DST026C29  DST026C31" class="link link-ref link-reveal xref-bibr">29–31</a></sup> There are two major advantages in using currently available data for the analysis of SSR distribution and marker development. The maize germplasms from different ecological regions and heterotic groups (PB, SPT, PA, LRC, and BSSS) are highly diverse and host rare and unique alleles, providing an opportunity of using these types of genetic variation in hybrid maize breeding. On the other hand, whole-genome sequence data provide an ideal resource and the most complete picture of genetic variation for developing high-density genetic markers.</p><p>SSR markers of highly polymorphic among diverse germplasms provide some advantages in genetics and breeding applications. In spite of considerable efforts in developing molecular markers in maize, the number of SSRs publicly available is still limited. From &gt;260 000 SSRs identified from 17 tested genomes, we detected 111 887 SSR loci with unique flanking sequence and single binding site through genome sequence blast and e-PCR analysis. These SSR loci can be developed as polymorphic markers <em>in silico</em> and public on the MaizeGDB database, which are ∼60 times more than those deposited in the MaizeGDB database so far. A total of 1556 SSR markers from the MaizeGDB database have specific location, and 16.6% of the newly developed SSR markers shared the same loci with public SSR markers. For some of the public SSR markers, the amplicon size was too large and it contained several newly developed SSR primers with different motifs. Therefore, only 0.47% (527) of newly developed SSR markers had completely compatible position with public SSR markers. Another reason for a few common SSRs shared with the two datasets maybe the traditional method for SSR marker development was based on screening of small-insert or microsatellite-enriched genomic libraries by hybridization in different materials,<sup><a reveal-id="DST026C32" data-open="DST026C32" class="link link-ref link-reveal xref-bibr">32</a></sup> which was different from our analyses based on B73 reference genome and other resequenced genomes. Furthermore, the second-generation sequencing was different from the Sanger sequencing, which also lead to the differences. The experimental validation also proved to detect more alleles than the expected <em>in silicon</em> analysis due to diverse materials used in the study, but some SSR loci with little length differences were also hard to distinguish. The average marker density for the newly developed dataset reached one SSR per 14.7 kb in the B73 reference genome, indicating that maize is a highly polymorphic species.<sup><a reveal-id="DST026C33" data-open="DST026C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The availability of abundant SSR markers allows dramatic improvement in the efficiency of marker-assisted selection and fine mapping of QTL regions.</p><p>Previous studies have mainly focused on di-, tri-, and tetra-nucleotide SSRs, whereas mono-, penta-, and hexa-nucleotide SSRs have not drawn enough attention for marker development. We found that mono-nucleotide SSRs had much higher polymorphism rates than others, and penta- and hexa-nucleotide SSRs had relatively longer repeat units. Intron and UTR SSRs were more polymorphic than CDS SSRs due to low selective pressure in non-coding regions, which were consistent with previous reports.<sup><a reveal-id="DST026C22" data-open="DST026C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST026C34  DST026C36" data-open="DST026C34  DST026C36" class="link link-ref link-reveal xref-bibr">34–36</a></sup> Experimental validation using 20 maize inbreds and 7 teosinte species showed that over 70% of the primer pairs could generate the target bands with length polymorphism, promising a great potential for the application of these SSR markers. In practice, it would be very powerful when they are used for genetic populations derived from various types of maize germplasm that were sampled for this study.</p>                            <h2 scrollto-destination=5193974 id="5193974" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5193976 id="5193976" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the National High Technology Research and Development Program of China (2012AA101104) to Y.L. and Y.X., Sichuan Youth Science and Technology Foundation of China (2012JQ0003), and the National Natural Science Foundation of China (31101162) to Y.L.</p>                            <h2 scrollto-destination=5193978 id="5193978" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST026C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fulton</div> <div class="given-names">R.S.</div></div>, et al. </div><div class="article-title">The B73 maize genome: complexity, diversity, and dynamics</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1112</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics&amp;author=P.S.%20Schnable&amp;author=D.%20Ware&amp;author=R.S.%20Fulton&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1112-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1178534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1178534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1178534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chambers</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">MacAvoy</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">Microsatellites: consensus and controversy</div>, <div class="source">Comp. Biochem. Physiol. B Biochem. Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">455</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20consensus%20and%20controversy&amp;author=G.K.%20Chambers&amp;author=E.S.%20MacAvoy&amp;publication_year=2000&amp;journal=Comp.%20Biochem.%20Physiol.%20B%20Biochem.%20Mol.%20Biol.&amp;volume=126&amp;pages=455-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0305-0491(00)00233-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0305-0491(00)00233-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0305-0491(00)00233-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11026658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Microsatellites: simple sequences with complex evolution</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">435</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20simple%20sequences%20with%20complex%20evolution&amp;author=H.%20Ellegren&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=435-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCouch</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Panaud</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Microsatellite marker development, mapping and applications in rice genetics and breeding</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">89</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20marker%20development%2C%20mapping%20and%20applications%20in%20rice%20genetics%20and%20breeding&amp;author=S.R.%20McCouch&amp;author=X.%20Chen&amp;author=O.%20Panaud&amp;publication_year=1997&amp;journal=Plant%20Mol.%20Biol.&amp;volume=35&amp;pages=89-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1005711431474" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1005711431474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1005711431474"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9291963" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zalapa</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Cuevas</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Using next-generation sequencing approaches to isolate simple sequence repeat (SSR) loci in the plant sciences</div>, <div class="source">Am. J. Botany</div> , <div class="year">2012</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">193</div>-<div class="lpage">208</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20next-generation%20sequencing%20approaches%20to%20isolate%20simple%20sequence%20repeat%20(SSR)%20loci%20in%20the%20plant%20sciences&amp;author=J.E.%20Zalapa&amp;author=H.%20Cuevas&amp;author=H.%20Zhu&amp;publication_year=2012&amp;journal=Am.%20J.%20Botany&amp;volume=99&amp;pages=193-208" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1100394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1100394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1100394"> </span></div> </div></div></div></div></div><div content-id="DST026C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kelkar</div> <div class="given-names">Y.D.</div></div>,  <div class="name"><div class="surname">Tyekucheva</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chiaromonte</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Makova</div> <div class="given-names">K.D.</div></div>. </div><div class="article-title">The genome-wide determinants of human and chimpanzee microsatellite evolution</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">30</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome-wide%20determinants%20of%20human%20and%20chimpanzee%20microsatellite%20evolution&amp;author=Y.D.%20Kelkar&amp;author=S.%20Tyekucheva&amp;author=F.%20Chiaromonte&amp;author=K.D.%20Makova&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=30-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.7113408" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.7113408" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.7113408"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18032720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genic microsatellite markers in plants: features and applications</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">48</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20plants%3A%20features%20and%20applications&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Biotechnol.&amp;volume=23&amp;pages=48-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.11.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Guimaraes</div> <div class="given-names">C.T.</div></div>, et al. </div><div class="article-title">Molecular characterization of global maize breeding germplasm based on genome-wide single nucleotide polymorphisms</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">93</div>-<div class="lpage">115</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20global%20maize%20breeding%20germplasm%20based%20on%20genome-wide%20single%20nucleotide%20polymorphisms&amp;author=Y.%20Lu&amp;author=J.%20Yan&amp;author=C.T.%20Guimaraes&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=93-115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1162-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1162-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1162-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19823800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sonah</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Deshmukh</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genome-wide distribution and organization of microsatellites in plants: an insight into marker development in <em>Brachypodium</em></div>, <div class="source">PloS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e21298</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20distribution%20and%20organization%20of%20microsatellites%20in%20plants%3A%20an%20insight%20into%20marker%20development%20in%20Brachypodium&amp;author=H.%20Sonah&amp;author=R.K.%20Deshmukh&amp;author=A.%20Sharma&amp;publication_year=2011&amp;journal=PloS%20One&amp;volume=6&amp;pages=e21298" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0021298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0021298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0021298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21713003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Jia</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Abundance of SSR motifs and development of candidate polymorphic SSR markers (BARCSOYSSR_1. 0) in soybean</div>, <div class="source">Crop Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">1950</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%20of%20SSR%20motifs%20and%20development%20of%20candidate%20polymorphic%20SSR%20markers%20(BARCSOYSSR_1.%200)%20in%20soybean&amp;author=Q.%20Song&amp;author=G.%20Jia&amp;author=Y.%20Zhu&amp;publication_year=2010&amp;journal=Crop%20Sci.&amp;volume=50&amp;pages=1950-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2009.10.0607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2009.10.0607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2009.10.0607"> </span></div> </div></div></div></div></div><div content-id="DST026C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barchi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Lanteri</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Portis</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Identification of SNP and SSR markers in eggplant using RAD tag sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">304</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20SNP%20and%20SSR%20markers%20in%20eggplant%20using%20RAD%20tag%20sequencing&amp;author=L.%20Barchi&amp;author=S.%20Lanteri&amp;author=E.%20Portis&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21663628" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Sang</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">961</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=961-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Goodman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Muse</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Doebley</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">2117</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20diversity%20among%20maize%20inbred%20lines%20as%20inferred%20from%20DNA%20microsatellites&amp;author=K.%20Liu&amp;author=M.%20Goodman&amp;author=S.%20Muse&amp;author=J.S.%20Smith&amp;author=E.%20Buckler&amp;author=J.%20Doebley&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=2117-28" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Reif</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Melchinger</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genetic diversity among CIMMYT maize inbred lines investigated with SSR markers</div>, <div class="source">Crop Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">2573</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20among%20CIMMYT%20maize%20inbred%20lines%20investigated%20with%20SSR%20markers&amp;author=X.%20Xia&amp;author=J.%20Reif&amp;author=A.%20Melchinger&amp;publication_year=2005&amp;journal=Crop%20Sci.&amp;volume=45&amp;pages=2573-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2005.0246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2005.0246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2005.0246"> </span></div> </div></div></div></div></div><div content-id="DST026C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warburton</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Reif</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Frisch</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genetic diversity in CIMMYT nontemperate maize germplasm: landraces, open pollinated varieties, and inbred lines</div>, <div class="source">Crop Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">617</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20CIMMYT%20nontemperate%20maize%20germplasm%3A%20landraces%2C%20open%20pollinated%20varieties%2C%20and%20inbred%20lines&amp;author=M.%20Warburton&amp;author=J.%20Reif&amp;author=M.%20Frisch&amp;publication_year=2008&amp;journal=Crop%20Sci.&amp;volume=48&amp;pages=617-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2007.02.0103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2007.02.0103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2007.02.0103"> </span></div> </div></div></div></div></div><div content-id="DST026C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">SOAP2: an improved ultrafast tool for short read alignment</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1966</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment&amp;author=R.%20Li&amp;author=C.%20Yu&amp;author=Y.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1966-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Transcriptome characterization and sequencing-based identification of salt-responsive genes in <em>Millettia pinnata</em>, a semi-mangrove plant</div>, <div class="source">DNA Res.</div> , <div class="year">2012</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">195</div>-<div class="lpage">207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20characterization%20and%20sequencing-based%20identification%20of%20salt-responsive%20genes%20in%20Millettia%20pinnata%2C%20a%20semi-mangrove%20plant&amp;author=J.%20Huang&amp;author=X.%20Lu&amp;author=H.%20Yan&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=195-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22351699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chia</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>, et al. </div><div class="article-title">Maize HapMap2 identifies extant variation from a genome in flux</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">803</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maize%20HapMap2%20identifies%20extant%20variation%20from%20a%20genome%20in%20flux&amp;author=J.M.%20Chia&amp;author=C.%20Song&amp;author=P.J.%20Bradbury&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=803-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chin</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Shu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">An evaluation of the utility of SSR loci as molecular markers in maize (<em>Zea mays</em> L.): comparisons with data from RFLPs and pedigree</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">163</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evaluation%20of%20the%20utility%20of%20SSR%20loci%20as%20molecular%20markers%20in%20maize%20(Zea%20mays%20L.)%3A%20comparisons%20with%20data%20from%20RFLPs%20and%20pedigree&amp;author=J.%20Smith&amp;author=E.%20Chin&amp;author=H.%20Shu&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=95&amp;pages=163-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050544"> </span></div> </div></div></div></div></div><div content-id="DST026C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">DeClerck</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lukashova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cartinhour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Computational and experimental analysis of microsatellites in rice (<em>Oryza sativa</em> L.): frequency, length variation, transposon associations, and genetic marker potential</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1441</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20analysis%20of%20microsatellites%20in%20rice%20(Oryza%20sativa%20L.)%3A%20frequency%2C%20length%20variation%2C%20transposon%20associations%2C%20and%20genetic%20marker%20potential&amp;author=S.%20Temnykh&amp;author=G.%20DeClerck&amp;author=A.%20Lukashova&amp;author=L.%20Lipovich&amp;author=S.%20Cartinhour&amp;author=S.%20McCouch&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1441-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.184001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.184001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.184001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soderlund</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Descour</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kudrna</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Sequencing, mapping, and analysis of 27,455 maize full-length cDNAs</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000740</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%2C%20mapping%2C%20and%20analysis%20of%2027%2C455%20maize%20full-length%20cDNAs&amp;author=C.%20Soderlund&amp;author=A.%20Descour&amp;author=D.%20Kudrna&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000740" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19936069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Deng</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xue</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">A genome-wide microsatellite polymorphism database for the indica and japonica rice</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">37</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20microsatellite%20polymorphism%20database%20for%20the%20indica%20and%20japonica%20rice&amp;author=Z.%20Zhang&amp;author=Y.%20Deng&amp;author=J.%20Tan&amp;author=S.%20Hu&amp;author=J.%20Yu&amp;author=Q.%20Xue&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=37-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawson</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Distinct patterns of SSR distribution in the Arabidopsis thaliana and rice genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">R14</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20patterns%20of%20SSR%20distribution%20in%20the%20Arabidopsis%20thaliana%20and%20rice%20genomes&amp;author=M.J.%20Lawson&amp;author=L.%20Zhang&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=R14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-2-r14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-2-r14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-2-r14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16507170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iniguez-Luy</div> <div class="given-names">F.L.</div></div>,  <div class="name"><div class="surname">Voort</div> <div class="given-names">A.V.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Development of a set of public SSR markers derived from genomic sequence of a rapid cycling <em>Brassica oleracea</em> L. genotype</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">977</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20set%20of%20public%20SSR%20markers%20derived%20from%20genomic%20sequence%20of%20a%20rapid%20cycling%20Brassica%20oleracea%20L.%20genotype&amp;author=F.L.%20Iniguez-Luy&amp;author=A.V.%20Voort&amp;author=T.C.%20Osborn&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=117&amp;pages=977-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0837-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0837-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0837-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18651126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yonemaru</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ando</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizubayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kasuga</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yano</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Development of genome-wide simple sequence repeat markers using whole-genome shotgun sequences of sorghum (<em>Sorghum bicolor</em> (L.) moench)</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">187</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genome-wide%20simple%20sequence%20repeat%20markers%20using%20whole-genome%20shotgun%20sequences%20of%20sorghum%20(Sorghum%20bicolor%20(L.)%20moench)&amp;author=J.%20Yonemaru&amp;author=T.%20Ando&amp;author=T.%20Mizubayashi&amp;author=S.%20Kasuga&amp;author=T.%20Matsumoto&amp;author=M.%20Yano&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=187-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19363056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hanafey</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes</div>, <div class="source">Nat. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">194</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20are%20preferentially%20associated%20with%20nonrepetitive%20DNA%20in%20plant%20genomes&amp;author=M.%20Morgante&amp;author=M.%20Hanafey&amp;author=W.%20Powell&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=30&amp;pages=194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng822" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11799393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Metzgar</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bytof</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wills</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Selection against frameshift mutations limits microsatellite expansion in coding DNA</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">72</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20against%20frameshift%20mutations%20limits%20microsatellite%20expansion%20in%20coding%20DNA&amp;author=D.%20Metzgar&amp;author=J.%20Bytof&amp;author=C.%20Wills&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=72-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10645952" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.V.</div></div>,  <div class="name"><div class="surname">Blades</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ding</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sultana</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Parmigiani</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Estimation of sequencing error rates in short reads</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">185</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20sequencing%20error%20rates%20in%20short%20reads&amp;author=X.V.%20Wang&amp;author=N.%20Blades&amp;author=J.%20Ding&amp;author=R.%20Sultana&amp;author=G.%20Parmigiani&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-13-185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-13-185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22846331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hufford</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Van Heerwaarden</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Comparative population genomics of maize domestication and improvement</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">808</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20population%20genomics%20of%20maize%20domestication%20and%20improvement&amp;author=M.B.%20Hufford&amp;author=X.%20Xu&amp;author=J.%20Van%20Heerwaarden&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=808-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Genome-wide patterns of genetic variation among elite maize inbred lines</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">1027</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20patterns%20of%20genetic%20variation%20among%20elite%20maize%20inbred%20lines&amp;author=J.%20Lai&amp;author=R.%20Li&amp;author=X.%20Xu&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=1027-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.684"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972441" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Genome-wide genetic changes during modern breeding of maize</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">812</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20changes%20during%20modern%20breeding%20of%20maize&amp;author=Y.%20Jiao&amp;author=H.%20Zhao&amp;author=L.%20Ren&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=812-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2312" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2312" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2312"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharopova</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">McMullen</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Schultz</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Development and mapping of SSR markers for maize</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">463</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20mapping%20of%20SSR%20markers%20for%20maize&amp;author=N.%20Sharopova&amp;author=M.D.%20McMullen&amp;author=L.%20Schultz&amp;publication_year=2002&amp;journal=Plant%20Mol.%20Biol.&amp;volume=48&amp;pages=463-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1014868625533" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1014868625533" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1014868625533"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12004892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">McMullen</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Molecular and functional diversity of maize</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">172</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20and%20functional%20diversity%20of%20maize&amp;author=E.S.%20Buckler&amp;author=B.S.%20Gaut&amp;author=M.D.%20McMullen&amp;publication_year=2006&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=9&amp;pages=172-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2006.01.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2006.01.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2006.01.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16459128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hao</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize</div>, <div class="source">PloS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e24861</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20SNP%20and%20haplotype%20analysis%20reveals%20a%20higher%20genetic%20diversity%20and%20rapider%20LD%20decay%20in%20tropical%20than%20temperate%20germplasm%20in%20maize&amp;author=Y.%20Lu&amp;author=T.%20Shah&amp;author=Z.%20Hao&amp;publication_year=2011&amp;journal=PloS%20One&amp;volume=6&amp;pages=e24861" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0024861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0024861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0024861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21949770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST026C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Genome-scale identification of resistance gene analogs and the development of their intron length polymorphism markers in maize</div>, <div class="source">Mol. Breeding</div> , <div class="year">2012</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">437</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20identification%20of%20resistance%20gene%20analogs%20and%20the%20development%20of%20their%20intron%20length%20polymorphism%20markers%20in%20maize&amp;author=H.%20Liu&amp;author=Y.%20Lin&amp;author=G.%20Chen&amp;author=Y.%20Shen&amp;author=J.%20Liu&amp;author=S.%20Zhang&amp;publication_year=2012&amp;journal=Mol.%20Breeding&amp;volume=29&amp;pages=437-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9560-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9560-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9560-3"> </span></div> </div></div></div></div></div><div content-id="DST026C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST026C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Strassmann</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Queller</div> <div class="given-names">D.C.</div></div>. </div><div class="article-title">Genome nucleotide composition shapes variation in simple sequence repeats</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">899</div>-<div class="lpage">909</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20nucleotide%20composition%20shapes%20variation%20in%20simple%20sequence%20repeats&amp;author=X.%20Tian&amp;author=J.E.%20Strassmann&amp;author=D.C.%20Queller&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=899-909" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq266" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq266" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq266"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20943830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359997&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst026-dst026supp_table2.xls?Expires=2147483647&amp;Signature=ya3MwSdQhRrClAvEiQv3VKG8~yRlR5iMzvKDGZGLFWqSpMajR4ccR-NcNgxwEj6hj8EZO86To1yOkkL5xtbKLqXqhDS-ocpr83Y1ubR-lwT2NC1up1gz2Ll0yqyH35FhXNg2q5S60UrU0C0GgCUmhXohJQ98xdC-Fom0NwvklT502pTMwb~Sgc-J~5movS7UzV07mn-F0DlpdKZ5AKixfKJiEW1OvFxBh4heHfURVzkztE8RpcHzI0NYg6I8YSaI08EasYGX9qgZvcK2M3FgKzd6iC1hcbnu5dVqKw8RCqpLEoLNnNna4H2SNb1ir0zaIrTsNB9JKsjD0wSu2XBQng__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst026-dst026supp_table1.xls?Expires=2147483647&amp;Signature=FjtU8d4PQvEF01oX7MNq0IMmGPzSzoUsFVk9uuUn8uN6mfE~d2YEYYyC91xUhmXDm~4~qXG-C-x3hrXEGTJ71IBzP28HlrF~C9fo-DxTGHanvSxMOTP0heSLkAyolRehRD1B1LgPDpTMxTAc9lLsfwwC9-dPQ~8Xi62zRjpyv61saAr6v6rNH4hgTd5qYM-UmyfLmnFrRExcqyNySfPB-7jLFUYEdCoIPTxV0Cs8No84sZdw43tIzWmA1LwJxs7IbCz-IDmN69ENjwPBXIH1ayn-M7pyeH4eRW5Mji7XsEOus3ZTtbn5gSlibIpxHXivI2svbkuXKamA1N5iP8FJMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst026/2/dst026-dst026supp.docx?Expires=2147483647&amp;Signature=v1V0Na3fHK5LG~H5tU4B~pf97RTu8rj6EIsvk8fQbsAU3uURVlPqvq4k02bpgBlflCXbi6TSnlc3yeJMDGTSxZR3spNYH~FMd3GvFEHz3tmuBn6~XbsSLN3dfSgxrFZtTmnBMrshU~hyt130SCPuT27BeySPCyknrv1kcf3UcbZfeOWq84kfRM8alW~9TJEp~Pkc61LEBMviQiZYIyRUdNnb6H~AoW2umul806G5wZTKsfpPdmi7RZoOYnOiy1xvOP9NfWZiStV7bencoBK-dfVy8eFhOiLQJ~sMYvg-cWh554pgA5HHs91OfOaYGReigrTiTmAf5uZMMcUbgIEKAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">488</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330167500007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">27</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst026"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359997"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330167500007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f5%2f497%2f359997">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330167500007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (27)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F5%2F497%2F359997" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359997'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Development and Characterization of Simple Sequence Repeat Markers Providing Genome-Wide Coverage and High Resolution in Maize" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst026");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f497%2f359997">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

