

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/6/585/2754539","@type":"ScholarlyArticle","name":"The Draft Genome and Transcriptome of  Amaranthus hypochondriacus : A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal","datePublished":"2014-07-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2014-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu021","keywords":["Caryophyllales","grain amaranth","<italic>Amaranthus hypochondriacus</italic>","lysine biosynthesis","C4 photosynthesis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/6/585/2754539","author":[{"name":"Sunil, Meeta","affiliation":"1  Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Hariharan, Arun K.","affiliation":"2  GANIT Labs: Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Nayak, Soumya","affiliation":"1  Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Gupta, Saurabh","affiliation":"2  GANIT Labs: Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Nambisan, Suran R.","affiliation":"1  Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Gupta, Ravi P.","affiliation":"1  Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Panda, Binay","affiliation":"2  GANIT Labs: Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Choudhary, Bibha","affiliation":"1  Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"},{"name":"Srinivasan, Subhashini","affiliation":"1  Institute of Bioinformatics and Applied Biotechnology ,  Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100 ,  India","@type":"Person"}],"description":"Grain amaranths, edible C4 dicots, produce pseudo-cereals high in lysine. Lysine being one of the most limiting essential amino acids in cereals and C4 photosyn","pageStart":"585","pageEnd":"602","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The Draft Genome and Transcriptome of  Amaranthus hypochondriacus : A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Draft Genome and Transcriptome of  Amaranthus hypochondriacus : A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal" />
    <meta property="og:description" content="Grain amaranths, edible C4 dicots, produce pseudo-cereals high in lysine. Lysine being one of the most limiting essential amino acids in cereals and C4 photosyn" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/6/585/2754539" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-07-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sunil, Meeta" /><meta name="citation_author_institution" content="Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Hariharan, Arun K." /><meta name="citation_author_institution" content="GANIT Labs: Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Nayak, Soumya" /><meta name="citation_author_institution" content="Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Gupta, Saurabh" /><meta name="citation_author_institution" content="GANIT Labs: Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Nambisan, Suran R." /><meta name="citation_author_institution" content="Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Gupta, Ravi P." /><meta name="citation_author_institution" content="Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Panda, Binay" /><meta name="citation_author_institution" content="GANIT Labs: Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Choudhary, Bibha" /><meta name="citation_author_institution" content="Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_author" content="Srinivasan, Subhashini" /><meta name="citation_author_institution" content="Institute of Bioinformatics and Applied Biotechnology, Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100, India" /><meta name="citation_title" content="The Draft Genome and Transcriptome of Amaranthus hypochondriacus: A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal" /><meta name="citation_firstpage" content="585" /><meta name="citation_lastpage" content="602" /><meta name="citation_doi" content="10.1093/dnares/dsu021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2014/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_author=Innovation, N.R.C. (U.S.) A.C; citation_publisher=National Academies, Washington, DC, USA;  citation_title=Amaranth: Modern Prospects for an Ancient Crop;  citation_year=1984; " /><meta name="citation_reference" content="citation_title=Amaranth: a comeback for the food of the Aztecs?;  citation_journal_title=Science;  citation_year=1977;  citation_volume=198;  citation_pages=40" /><meta name="citation_reference" content="citation_title=State of knowledge on amaranth grain: a comprehensive review;  citation_journal_title=J. Food Sci.;  citation_year=2012;  citation_volume=77;  citation_pages=R93-104; " /><meta name="citation_reference" content="citation_title=Are high-lysine cereal crops still a challenge?;  citation_journal_title=Braz. J. Med. Biol. Res. Rev. Bras. Pesqui. Médicas E Biológicas Soc. Bras. Biofísica Al.;  citation_year=2005;  citation_volume=38;  citation_pages=985-94; " /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=83-9; " /><meta name="citation_reference" content="citation_title=Comparative responses of model C3 and C4 plants to drought in low and elevated CO2;  citation_journal_title=Glob. Change Biol.;  citation_year=1999;  citation_volume=5;  citation_pages=857-67; " /><meta name="citation_reference" content="citation_title=Phylogeny of Amranthaceae and Chenopodiaceae and the evolution of C4 photosynthesis;  citation_journal_title=Int. J. Plant Sci.;  citation_year=2003;  citation_volume=164;  citation_pages=959-86; " /><meta name="citation_reference" content="citation_title=Complex pigment evolution in the Caryophyllales;  citation_journal_title=New Phytol.;  citation_year=2011;  citation_volume=190;  citation_pages=854-64; " /><meta name="citation_reference" content="citation_title=Anthocyanins and betalains: evolution of the mutually exclusive pathways;  citation_journal_title=Plant Sci.;  citation_year=1994;  citation_volume=101;  citation_pages=91-8; " /><meta name="citation_reference" content="citation_title=Biosynthesis of betalains: yellow and violet plant pigments;  citation_journal_title=Trends Plant Sci.;  citation_year=2013;  citation_volume=18;  citation_pages=334-43; " /><meta name="citation_reference" content="citation_title=Tyrosinase modulation by five Rwandese herbal medicines traditionally used for skin treatment;  citation_journal_title=J. Ethnopharmacol.;  citation_year=2013;  citation_volume=146;  citation_pages=824-34; " /><meta name="citation_reference" content="citation_title=The genome of the recently domesticated crop plant sugar beet (Beta vulgaris);  citation_journal_title=Nature;  citation_year=2014;  citation_volume=505;  citation_pages=546-9; " /><meta name="citation_reference" content="citation_title=Transcriptomic analysis of grain amaranth (Amaranthus hypochondriacus) using 454 pyrosequencing: comparison with A. tuberculatus, expression profiling in stems and in response to biotic and abiotic stress;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=363" /><meta name="citation_reference" content="citation_title=Characterization of de novo transcriptome for waterhemp (Amaranthus tuberculatus) using GS-FLX 454 pyrosequencing and its application for studies of herbicide target-site genes;  citation_journal_title=Pest Manag. Sci.;  citation_year=2010;  citation_volume=66;  citation_pages=1042-52; " /><meta name="citation_reference" content="citation_title=Construction of an bacterial artificial chromosome library and genomic sequencing of herbicide target genes;  citation_journal_title=Crop. Sci.;  citation_year=2008;  citation_volume=48;  citation_pages=S85" /><meta name="citation_reference" content="citation_title=Development and characterization of microsatellite markers for the grain amaranths;  citation_journal_title=Crop. Sci.;  citation_year=2008;  citation_volume=48;  citation_pages=1098" /><meta name="citation_reference" content="citation_title=Development, characterization, and linkage mapping of single nucleotide polymorphisms in the grain amaranths (Amaranthus sp.);  citation_journal_title=Plant Genome;  citation_year=2011;  citation_volume=4;  citation_pages=92-101; " /><meta name="citation_reference" content="citation_title=Grain amaranth research in Nepal;  citation_journal_title=J. Fac. Agric. Shinshu Univ.;  citation_year=1998;  citation_volume=34;  citation_pages=49-58; " /><meta name="citation_reference" content="citation_title=A simple and rapid method for RNA isolation from plant tissues with high phenolic compounds and polysaccharides;  citation_journal_title=Protoc. Exch.;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing;  citation_journal_title=Genome. Res.;  citation_year=2010;  citation_volume=20;  citation_pages=265-72; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Caloramator australicus strain RC3T, a thermoanaerobe from the Great Artesian Basin of Australia;  citation_journal_title=J. Bacteriol.;  citation_year=2011;  citation_volume=193;  citation_pages=2664-5; " /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=SyMAP: a system for discovering and viewing syntenic regions of FPC maps;  citation_journal_title=Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1159-68; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Prediction of complete gene structures in human genomic DNA.;  citation_journal_title=J. Mol. Biol.;  citation_year=1997;  citation_volume=268;  citation_pages=78-94; " /><meta name="citation_reference" content="citation_title=AUGUSTUS: a web server for gene finding in eukaryotes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=W309-12; " /><meta name="citation_reference" content="citation_title=GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W451-4; " /><meta name="citation_reference" content="citation_title=PlantGDB: a resource for comparative plant genomics;  citation_journal_title=Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D959-65; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of eukaryotic repetitive elements, Cytogenet;  citation_journal_title=Genome Res.;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_issue=Suppl 1;  citation_pages=i351-8; " /><meta name="citation_reference" content="  Smit A.  Hubley R.   RepeatModeler Open-1.0  2008   http://www.repeatmasker.org (9 October 2013, date last accessed)  " /><meta name="citation_reference" content="citation_title=Using and understanding RepeatMasker;  citation_journal_title=Methods Mol. Biol.;  citation_year=2012;  citation_volume=859;  citation_pages=29-51; " /><meta name="citation_reference" content="  Smit A.  Hubley R.  Green P.   RepeatMasker Open-3.0  1996   http://www.repeatmasker.org. (9 October 2013, date last accessed)  " /><meta name="citation_reference" content="citation_title=PGDD: a database of gene and genome duplication in plants;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012; " /><meta name="citation_reference" content=" R Development Core Team  R: A language and environment for statistical computing  2008,  R Foundation for Statistical Computing   http://www.R-project.org. (2 June 2014, date last accessed)  " /><meta name="citation_reference" content="citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity;  citation_journal_title=BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=113" /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-25; " /><meta name="citation_reference" content="citation_title=Confidence limits on phylogenies: an approach using Bootstrap;  citation_journal_title=Evolution;  citation_year=1985;  citation_volume=39;  citation_pages=783-91; " /><meta name="citation_reference" content="citation_publisher=Academic Press, ;  citation_title=Evol. Genes Proteins;  citation_year=1965;  citation_pages=97-166; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Interactive Tree Of Life (iTOL): an online tool for phylogenetic tree display and annotation;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=127-8; " /><meta name="citation_reference" content="citation_title=Interactive Tree Of Life v2: online annotation and display of phylogenetic trees made easy;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=W475-8; " /><meta name="citation_reference" content="citation_title=Measurements of genomic GC content in plant genomes with flow cytometry: a test for reliability;  citation_journal_title=New Phytol.;  citation_year=2012;  citation_volume=193;  citation_pages=513-21; " /><meta name="citation_reference" content="citation_title=De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification;  citation_journal_title=DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=53-63; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; citation_author=Consortium, T.T.G;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP detection, validation, and development of an 8K SNP array for apple;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e31745" /><meta name="citation_reference" content="citation_title=Development of genome-wide DNA polymorphism database for map-based cloning of rice genes;  citation_journal_title=Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=1198-205; " /><meta name="citation_reference" content="citation_title=SNP frequency, haplotype structure and linkage disequilibrium in elite maize inbred lines;  citation_journal_title=BMC Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=A high quality draft consensus sequence of the genome of a heterozygous grapevine variety;  citation_journal_title=PloS ONE;  citation_year=2007;  citation_volume=2;  citation_pages=e1326" /><meta name="citation_reference" content="citation_title=Characterization of SNPs in strawberry cultivars in China;  citation_journal_title=Genet. Mol. Res.;  citation_year=2013;  citation_volume=12;  citation_pages=639-45; " /><meta name="citation_reference" content="citation_title=Genome sequence and analysis of the tuber crop potato;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=189-95; " /><meta name="citation_reference" content="citation_title=Next generation sequencing-based analysis of repetitive DNA in the model Dioceous plant Silene latifolia;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e27335" /><meta name="citation_reference" content="citation_title=Parameters of proteome evolution from histograms of amino-acid sequence identities of paralogous proteins;  citation_journal_title=Biol. Direct;  citation_year=2007;  citation_volume=2;  citation_pages=32" /><meta name="citation_reference" content="citation_title=The flowering world: a tale of duplications;  citation_journal_title=Trends Plant Sci.;  citation_year=2009;  citation_volume=14;  citation_pages=680-8; " /><meta name="citation_reference" content="citation_title=Plants with double genomes might have had a better chance to survive the Cretaceous-Tertiary extinction event;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=5737-42; " /><meta name="citation_reference" content="citation_title=Evolution of C4 phosphoenolpyruvate carboxylase. Genes and proteins: a case study with the genus Flaveria;  citation_journal_title=Ann. Bot.;  citation_year=2004;  citation_volume=93;  citation_pages=13-23; " /><meta name="citation_reference" content="citation_title=Integrating phylogeny into studies of C4 variation in the grasses;  citation_journal_title=Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=82-7; " /><meta name="citation_reference" content="citation_title=New grass phylogeny resolves deep evolutionary relationships and discovers C4 origins; citation_author=Grass Phylogeny Working Group II.;  citation_journal_title=New Phytol.;  citation_year=2012;  citation_volume=193;  citation_pages=304-12; " /><meta name="citation_reference" content="citation_title=Complex evolutionary transitions and the significance of C3-C4 intermediate forms of photosynthesis in Molluginaceae;  citation_journal_title=Evol. Int. J. Org. Evol.;  citation_year=2011;  citation_volume=65;  citation_pages=643-60; " /><meta name="citation_reference" content="citation_title=The aspartate-family pathway of plants: linking production of essential amino acids with energy and stress regulation;  citation_journal_title=Plant Signal. Behav.;  citation_year=2011;  citation_volume=6;  citation_pages=192-5; " /><meta name="citation_reference" content="citation_title=Data, information, knowledge and principle: back to metabolism in KEGG;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=D199-205; " /><meta name="citation_reference" content="citation_title=KEGG: Kyoto encyclopedia of genes and genomes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=27-30; " /><meta name="citation_reference" content="citation_title=Allosteric monofunctional aspartate kinases from Arabidopsis;  citation_journal_title=FEBS J.;  citation_year=2007;  citation_volume=274;  citation_pages=164-76; " /><meta name="citation_reference" content="citation_title=Regulation of lysine and threonine synthesis;  citation_journal_title=Plant Cell.;  citation_year=1995;  citation_volume=7;  citation_pages=899-906; " /><meta name="citation_reference" content="  Madden T.   2002  USA  National Center for Biotechnology Information  NCBI Handbook [Internet]. Chapter 16The BLAST Sequence Analysis Tool http://www.ncbi.nlm.nih.gov/books/NBK21097/ (26 December 2013, date last accessed)  " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/6/585/17462755/dsu021.pdf" /><meta name="description" content="Grain amaranths, edible C4 dicots, produce pseudo-cereals high in lysine. Lysine being one of the most limiting essential amino acids in cereals and C4 photosyn" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/6/585/2754539" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/6/585/2754539" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/6/585/2754539");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu021';
    </script>
<title>
	Draft Genome and Transcriptome of Amaranthus hypochondriacus: A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754539" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcShATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3VuaWwsIE1lZXRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzIGFuZCBBcHBsaWVkIEJpb3RlY2hub2xvZ3ksIEJpb3RlY2ggUGFyaywgRWxlY3Ryb25pY3MgQ2l0eSBQaGFzZSBJLCBCYW5nYWxvcmUsIEthcm5hdGFrYSA1NjAxMDAsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFyaWhhcmFuLCBBcnVuIEsuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR0FOSVQgTGFiczogSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzIGFuZCBBcHBsaWVkIEJpb3RlY2hub2xvZ3ksIEJpb3RlY2ggUGFyaywgRWxlY3Ryb25pY3MgQ2l0eSBQaGFzZSBJLCBCYW5nYWxvcmUsIEthcm5hdGFrYSA1NjAxMDAsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmF5YWssIFNvdW15YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcyBhbmQgQXBwbGllZCBCaW90ZWNobm9sb2d5LCBCaW90ZWNoIFBhcmssIEVsZWN0cm9uaWNzIENpdHkgUGhhc2UgSSwgQmFuZ2Fsb3JlLCBLYXJuYXRha2EgNTYwMTAwLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1cHRhLCBTYXVyYWJoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR0FOSVQgTGFiczogSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzIGFuZCBBcHBsaWVkIEJpb3RlY2hub2xvZ3ksIEJpb3RlY2ggUGFyaywgRWxlY3Ryb25pY3MgQ2l0eSBQaGFzZSBJLCBCYW5nYWxvcmUsIEthcm5hdGFrYSA1NjAxMDAsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFtYmlzYW4sIFN1cmFuIFIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzIGFuZCBBcHBsaWVkIEJpb3RlY2hub2xvZ3ksIEJpb3RlY2ggUGFyaywgRWxlY3Ryb25pY3MgQ2l0eSBQaGFzZSBJLCBCYW5nYWxvcmUsIEthcm5hdGFrYSA1NjAxMDAsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3VwdGEsIFJhdmkgUC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MgYW5kIEFwcGxpZWQgQmlvdGVjaG5vbG9neSwgQmlvdGVjaCBQYXJrLCBFbGVjdHJvbmljcyBDaXR5IFBoYXNlIEksIEJhbmdhbG9yZSwgS2FybmF0YWthIDU2MDEwMCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYW5kYSwgQmluYXkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHQU5JVCBMYWJzOiBJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MgYW5kIEFwcGxpZWQgQmlvdGVjaG5vbG9neSwgQmlvdGVjaCBQYXJrLCBFbGVjdHJvbmljcyBDaXR5IFBoYXNlIEksIEJhbmdhbG9yZSwgS2FybmF0YWthIDU2MDEwMCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaG91ZGhhcnksIEJpYmhhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzIGFuZCBBcHBsaWVkIEJpb3RlY2hub2xvZ3ksIEJpb3RlY2ggUGFyaywgRWxlY3Ryb25pY3MgQ2l0eSBQaGFzZSBJLCBCYW5nYWxvcmUsIEthcm5hdGFrYSA1NjAxMDAsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3Jpbml2YXNhbiwgU3ViaGFzaGluaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcyBhbmQgQXBwbGllZCBCaW90ZWNobm9sb2d5LCBCaW90ZWNoIFBhcmssIEVsZWN0cm9uaWNzIENpdHkgUGhhc2UgSSwgQmFuZ2Fsb3JlLCBLYXJuYXRha2EgNTYwMTAwLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIERyYWZ0IEdlbm9tZSBhbmQgVHJhbnNjcmlwdG9tZSBvZiBBbWFyYW50aHVzIGh5cG9jaG9uZHJpYWN1czogQSBDNCBEaWNvdCBQcm9kdWNpbmcgSGlnaC1MeXNpbmUgRWRpYmxlIFBzZXVkby1DZXJlYWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1ODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjYwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTAyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPUlubm92YXRpb24sIE4uUi5DLiAoVS5TLikgQS5DOyBjaXRhdGlvbl9wdWJsaXNoZXI9TmF0aW9uYWwgQWNhZGVtaWVzLCBXYXNoaW5ndG9uLCBEQywgVVNBOyAgY2l0YXRpb25fdGl0bGU9QW1hcmFudGg6IE1vZGVybiBQcm9zcGVjdHMgZm9yIGFuIEFuY2llbnQgQ3JvcDsgIGNpdGF0aW9uX3llYXI9MTk4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1hcmFudGg6IGEgY29tZWJhY2sgZm9yIHRoZSBmb29kIG9mIHRoZSBBenRlY3M/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTE5ODsgIGNpdGF0aW9uX3BhZ2VzPTQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGUgb2Yga25vd2xlZGdlIG9uIGFtYXJhbnRoIGdyYWluOiBhIGNvbXByZWhlbnNpdmUgcmV2aWV3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBGb29kIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz1SOTMtMTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmUgaGlnaC1seXNpbmUgY2VyZWFsIGNyb3BzIHN0aWxsIGEgY2hhbGxlbmdlPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJhei4gSi4gTWVkLiBCaW9sLiBSZXMuIFJldi4gQnJhcy4gUGVzcXVpLiBNw6lkaWNhcyBFIEJpb2zDs2dpY2FzIFNvYy4gQnJhcy4gQmlvZsOtc2ljYSBBbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz05ODUtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcGFsYWVvcG9seXBsb2lkIHNveWJlYW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBwaWdlb25wZWEgKENhamFudXMgY2FqYW4pLCBhbiBvcnBoYW4gbGVndW1lIGNyb3Agb2YgcmVzb3VyY2UtcG9vciBmYXJtZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9ODMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcmVzcG9uc2VzIG9mIG1vZGVsIEMzIGFuZCBDNCBwbGFudHMgdG8gZHJvdWdodCBpbiBsb3cgYW5kIGVsZXZhdGVkIENPMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2xvYi4gQ2hhbmdlIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz04NTctNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VueSBvZiBBbXJhbnRoYWNlYWUgYW5kIENoZW5vcG9kaWFjZWFlIGFuZCB0aGUgZXZvbHV0aW9uIG9mIEM0IHBob3Rvc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTY0OyAgY2l0YXRpb25fcGFnZXM9OTU5LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV4IHBpZ21lbnQgZXZvbHV0aW9uIGluIHRoZSBDYXJ5b3BoeWxsYWxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTkwOyAgY2l0YXRpb25fcGFnZXM9ODU0LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRob2N5YW5pbnMgYW5kIGJldGFsYWluczogZXZvbHV0aW9uIG9mIHRoZSBtdXR1YWxseSBleGNsdXNpdmUgcGF0aHdheXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9OTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlvc3ludGhlc2lzIG9mIGJldGFsYWluczogeWVsbG93IGFuZCB2aW9sZXQgcGxhbnQgcGlnbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzM0LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UeXJvc2luYXNlIG1vZHVsYXRpb24gYnkgZml2ZSBSd2FuZGVzZSBoZXJiYWwgbWVkaWNpbmVzIHRyYWRpdGlvbmFsbHkgdXNlZCBmb3Igc2tpbiB0cmVhdG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV0aG5vcGhhcm1hY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDY7ICBjaXRhdGlvbl9wYWdlcz04MjQtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIHJlY2VudGx5IGRvbWVzdGljYXRlZCBjcm9wIHBsYW50IHN1Z2FyIGJlZXQgKEJldGEgdnVsZ2FyaXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTA1OyAgY2l0YXRpb25fcGFnZXM9NTQ2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWljIGFuYWx5c2lzIG9mIGdyYWluIGFtYXJhbnRoIChBbWFyYW50aHVzIGh5cG9jaG9uZHJpYWN1cykgdXNpbmcgNDU0IHB5cm9zZXF1ZW5jaW5nOiBjb21wYXJpc29uIHdpdGggQS4gdHViZXJjdWxhdHVzLCBleHByZXNzaW9uIHByb2ZpbGluZyBpbiBzdGVtcyBhbmQgaW4gcmVzcG9uc2UgdG8gYmlvdGljIGFuZCBhYmlvdGljIHN0cmVzczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBkZSBub3ZvIHRyYW5zY3JpcHRvbWUgZm9yIHdhdGVyaGVtcCAoQW1hcmFudGh1cyB0dWJlcmN1bGF0dXMpIHVzaW5nIEdTLUZMWCA0NTQgcHlyb3NlcXVlbmNpbmcgYW5kIGl0cyBhcHBsaWNhdGlvbiBmb3Igc3R1ZGllcyBvZiBoZXJiaWNpZGUgdGFyZ2V0LXNpdGUgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBlc3QgTWFuYWcuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz0xMDQyLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYW4gYmFjdGVyaWFsIGFydGlmaWNpYWwgY2hyb21vc29tZSBsaWJyYXJ5IGFuZCBnZW5vbWljIHNlcXVlbmNpbmcgb2YgaGVyYmljaWRlIHRhcmdldCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPVM4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIHRoZSBncmFpbiBhbWFyYW50aHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xMDk4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQsIGNoYXJhY3Rlcml6YXRpb24sIGFuZCBsaW5rYWdlIG1hcHBpbmcgb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBpbiB0aGUgZ3JhaW4gYW1hcmFudGhzIChBbWFyYW50aHVzIHNwLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9OTItMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HcmFpbiBhbWFyYW50aCByZXNlYXJjaCBpbiBOZXBhbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRmFjLiBBZ3JpYy4gU2hpbnNodSBVbml2LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTQ5LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBhbmQgcmFwaWQgbWV0aG9kIGZvciBSTkEgaXNvbGF0aW9uIGZyb20gcGxhbnQgdGlzc3VlcyB3aXRoIGhpZ2ggcGhlbm9saWMgY29tcG91bmRzIGFuZCBwb2x5c2FjY2hhcmlkZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3RvYy4gRXhjaC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgaHVtYW4gZ2Vub21lcyB3aXRoIG1hc3NpdmVseSBwYXJhbGxlbCBzaG9ydCByZWFkIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZS4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI2NS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIENhbG9yYW1hdG9yIGF1c3RyYWxpY3VzIHN0cmFpbiBSQzNULCBhIHRoZXJtb2FuYWVyb2JlIGZyb20gdGhlIEdyZWF0IEFydGVzaWFuIEJhc2luIG9mIEF1c3RyYWxpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz0yNjY0LTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bGwtbGVuZ3RoIHRyYW5zY3JpcHRvbWUgYXNzZW1ibHkgZnJvbSBSTkEtU2VxIGRhdGEgd2l0aG91dCBhIHJlZmVyZW5jZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02NDQtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5TUFQOiBhIHN5c3RlbSBmb3IgZGlzY292ZXJpbmcgYW5kIHZpZXdpbmcgc3ludGVuaWMgcmVnaW9ucyBvZiBGUEMgbWFwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTU5LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYWZhc3QgYW5kIG1lbW9yeS1lZmZpY2llbnQgYWxpZ25tZW50IG9mIHNob3J0IEROQSBzZXF1ZW5jZXMgdG8gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIEFsaWdubWVudC9NYXAgZm9ybWF0IGFuZCBTQU10b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBcmFiaWRvcHNpcyBJbmZvcm1hdGlvbiBSZXNvdXJjZSAoVEFJUik6IGltcHJvdmVkIGdlbmUgYW5ub3RhdGlvbiBhbmQgbmV3IHRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgY29tcGxldGUgZ2VuZSBzdHJ1Y3R1cmVzIGluIGh1bWFuIGdlbm9taWMgRE5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjg7ICBjaXRhdGlvbl9wYWdlcz03OC05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVVHVVNUVVM6IGEgd2ViIHNlcnZlciBmb3IgZ2VuZSBmaW5kaW5nIGluIGV1a2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPVczMDktMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVNYXJrOiB3ZWIgc29mdHdhcmUgZm9yIGdlbmUgZmluZGluZyBpbiBwcm9rYXJ5b3RlcywgZXVrYXJ5b3RlcyBhbmQgdmlydXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9VzQ1MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudEdEQjogYSByZXNvdXJjZSBmb3IgY29tcGFyYXRpdmUgcGxhbnQgZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQ5NTktNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGJhc2UgVXBkYXRlLCBhIGRhdGFiYXNlIG9mIGV1a2FyeW90aWMgcmVwZXRpdGl2ZSBlbGVtZW50cywgQ3l0b2dlbmV0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz00NjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBpZGVudGlmaWNhdGlvbiBvZiByZXBlYXQgZmFtaWxpZXMgaW4gbGFyZ2UgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbCAxOyAgY2l0YXRpb25fcGFnZXM9aTM1MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgIFNtaXQgQS4gIEh1YmxleSBSLiAgIFJlcGVhdE1vZGVsZXIgT3Blbi0xLjAgIDIwMDggICBodHRwOi8vd3d3LnJlcGVhdG1hc2tlci5vcmcgKDkgT2N0b2JlciAyMDEzLCBkYXRlIGxhc3QgYWNjZXNzZWQpICAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBhbmQgdW5kZXJzdGFuZGluZyBSZXBlYXRNYXNrZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04NTk7ICBjaXRhdGlvbl9wYWdlcz0yOS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iICBTbWl0IEEuICBIdWJsZXkgUi4gIEdyZWVuIFAuICAgUmVwZWF0TWFza2VyIE9wZW4tMy4wICAxOTk2ICAgaHR0cDovL3d3dy5yZXBlYXRtYXNrZXIub3JnLiAoOSBPY3RvYmVyIDIwMTMsIGRhdGUgbGFzdCBhY2Nlc3NlZCkgICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBHREQ6IGEgZGF0YWJhc2Ugb2YgZ2VuZSBhbmQgZ2Vub21lIGR1cGxpY2F0aW9uIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgUiBEZXZlbG9wbWVudCBDb3JlIFRlYW0gIFI6IEEgbGFuZ3VhZ2UgYW5kIGVudmlyb25tZW50IGZvciBzdGF0aXN0aWNhbCBjb21wdXRpbmcgIDIwMDgsICBSIEZvdW5kYXRpb24gZm9yIFN0YXRpc3RpY2FsIENvbXB1dGluZyAgIGh0dHA6Ly93d3cuUi1wcm9qZWN0Lm9yZy4gKDIgSnVuZSAyMDE0LCBkYXRlIGxhc3QgYWNjZXNzZWQpICAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NVVNDTEU6IGEgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IG1ldGhvZCB3aXRoIHJlZHVjZWQgdGltZSBhbmQgc3BhY2UgY29tcGxleGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmVpZ2hib3Itam9pbmluZyBtZXRob2Q6IGEgbmV3IG1ldGhvZCBmb3IgcmVjb25zdHJ1Y3RpbmcgcGh5bG9nZW5ldGljIHRyZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MDYtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbmZpZGVuY2UgbGltaXRzIG9uIHBoeWxvZ2VuaWVzOiBhbiBhcHByb2FjaCB1c2luZyBCb290c3RyYXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTc4My05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCA7ICBjaXRhdGlvbl90aXRsZT1Fdm9sLiBHZW5lcyBQcm90ZWluczsgIGNpdGF0aW9uX3llYXI9MTk2NTsgIGNpdGF0aW9uX3BhZ2VzPTk3LTE2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFjdGl2ZSBUcmVlIE9mIExpZmUgKGlUT0wpOiBhbiBvbmxpbmUgdG9vbCBmb3IgcGh5bG9nZW5ldGljIHRyZWUgZGlzcGxheSBhbmQgYW5ub3RhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMjctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJhY3RpdmUgVHJlZSBPZiBMaWZlIHYyOiBvbmxpbmUgYW5ub3RhdGlvbiBhbmQgZGlzcGxheSBvZiBwaHlsb2dlbmV0aWMgdHJlZXMgbWFkZSBlYXN5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XNDc1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lYXN1cmVtZW50cyBvZiBnZW5vbWljIEdDIGNvbnRlbnQgaW4gcGxhbnQgZ2Vub21lcyB3aXRoIGZsb3cgY3l0b21ldHJ5OiBhIHRlc3QgZm9yIHJlbGlhYmlsaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz01MTMtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgY2hpY2twZWEgdHJhbnNjcmlwdG9tZSB1c2luZyBzaG9ydCByZWFkcyBmb3IgZ2VuZSBkaXNjb3ZlcnkgYW5kIG1hcmtlciBpZGVudGlmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz01My02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgZmxvd2VyaW5nIHBsYW50IEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBjaXRhdGlvbl9hdXRob3I9QXJhYmlkb3BzaXMgR2Vub21lIEluaXRpYXRpdmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz03OTYtODE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3JhcGV2aW5lIGdlbm9tZSBzZXF1ZW5jZSBzdWdnZXN0cyBhbmNlc3RyYWwgaGV4YXBsb2lkaXphdGlvbiBpbiBtYWpvciBhbmdpb3NwZXJtIHBoeWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDQ5OyAgY2l0YXRpb25fcGFnZXM9NDYzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0b21hdG8gZ2Vub21lIHNlcXVlbmNlIHByb3ZpZGVzIGluc2lnaHRzIGludG8gZmxlc2h5IGZydWl0IGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPUNvbnNvcnRpdW0sIFQuVC5HOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg1OyAgY2l0YXRpb25fcGFnZXM9NjM1LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBTTlAgZGV0ZWN0aW9uLCB2YWxpZGF0aW9uLCBhbmQgZGV2ZWxvcG1lbnQgb2YgYW4gOEsgU05QIGFycmF5IGZvciBhcHBsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzMTc0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGdlbm9tZS13aWRlIEROQSBwb2x5bW9ycGhpc20gZGF0YWJhc2UgZm9yIG1hcC1iYXNlZCBjbG9uaW5nIG9mIHJpY2UgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTExOTgtMjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlAgZnJlcXVlbmN5LCBoYXBsb3R5cGUgc3RydWN0dXJlIGFuZCBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGluIGVsaXRlIG1haXplIGluYnJlZCBsaW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoIHF1YWxpdHkgZHJhZnQgY29uc2Vuc3VzIHNlcXVlbmNlIG9mIHRoZSBnZW5vbWUgb2YgYSBoZXRlcm96eWdvdXMgZ3JhcGV2aW5lIHZhcmlldHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMTMyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgU05QcyBpbiBzdHJhd2JlcnJ5IGN1bHRpdmFycyBpbiBDaGluYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIE1vbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTYzOS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIGFuZCBhbmFseXNpcyBvZiB0aGUgdHViZXIgY3JvcCBwb3RhdG87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NzU7ICBjaXRhdGlvbl9wYWdlcz0xODktOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nLWJhc2VkIGFuYWx5c2lzIG9mIHJlcGV0aXRpdmUgRE5BIGluIHRoZSBtb2RlbCBEaW9jZW91cyBwbGFudCBTaWxlbmUgbGF0aWZvbGlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTI3MzM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGFyYW1ldGVycyBvZiBwcm90ZW9tZSBldm9sdXRpb24gZnJvbSBoaXN0b2dyYW1zIG9mIGFtaW5vLWFjaWQgc2VxdWVuY2UgaWRlbnRpdGllcyBvZiBwYXJhbG9nb3VzIHByb3RlaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBEaXJlY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZsb3dlcmluZyB3b3JsZDogYSB0YWxlIG9mIGR1cGxpY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz02ODAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnRzIHdpdGggZG91YmxlIGdlbm9tZXMgbWlnaHQgaGF2ZSBoYWQgYSBiZXR0ZXIgY2hhbmNlIHRvIHN1cnZpdmUgdGhlIENyZXRhY2VvdXMtVGVydGlhcnkgZXh0aW5jdGlvbiBldmVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9NTczNy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIEM0IHBob3NwaG9lbm9scHlydXZhdGUgY2FyYm94eWxhc2UuIEdlbmVzIGFuZCBwcm90ZWluczogYSBjYXNlIHN0dWR5IHdpdGggdGhlIGdlbnVzIEZsYXZlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0xMy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpbmcgcGh5bG9nZW55IGludG8gc3R1ZGllcyBvZiBDNCB2YXJpYXRpb24gaW4gdGhlIGdyYXNzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTgyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBncmFzcyBwaHlsb2dlbnkgcmVzb2x2ZXMgZGVlcCBldm9sdXRpb25hcnkgcmVsYXRpb25zaGlwcyBhbmQgZGlzY292ZXJzIEM0IG9yaWdpbnM7IGNpdGF0aW9uX2F1dGhvcj1HcmFzcyBQaHlsb2dlbnkgV29ya2luZyBHcm91cCBJSS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTMwNC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxleCBldm9sdXRpb25hcnkgdHJhbnNpdGlvbnMgYW5kIHRoZSBzaWduaWZpY2FuY2Ugb2YgQzMtQzQgaW50ZXJtZWRpYXRlIGZvcm1zIG9mIHBob3Rvc3ludGhlc2lzIGluIE1vbGx1Z2luYWNlYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2wuIEludC4gSi4gT3JnLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTY0My02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFzcGFydGF0ZS1mYW1pbHkgcGF0aHdheSBvZiBwbGFudHM6IGxpbmtpbmcgcHJvZHVjdGlvbiBvZiBlc3NlbnRpYWwgYW1pbm8gYWNpZHMgd2l0aCBlbmVyZ3kgYW5kIHN0cmVzcyByZWd1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBTaWduYWwuIEJlaGF2LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTkyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGEsIGluZm9ybWF0aW9uLCBrbm93bGVkZ2UgYW5kIHByaW5jaXBsZTogYmFjayB0byBtZXRhYm9saXNtIGluIEtFR0c7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQxOTktMjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHOiBLeW90byBlbmN5Y2xvcGVkaWEgb2YgZ2VuZXMgYW5kIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbGxvc3RlcmljIG1vbm9mdW5jdGlvbmFsIGFzcGFydGF0ZSBraW5hc2VzIGZyb20gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFQlMgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mjc0OyAgY2l0YXRpb25fcGFnZXM9MTY0LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIGx5c2luZSBhbmQgdGhyZW9uaW5lIHN5bnRoZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTg5OS05MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiAgTWFkZGVuIFQuICAgMjAwMiAgVVNBICBOYXRpb25hbCBDZW50ZXIgZm9yIEJpb3RlY2hub2xvZ3kgSW5mb3JtYXRpb24gIE5DQkkgSGFuZGJvb2sgW0ludGVybmV0XS4gQ2hhcHRlciAxNlRoZSBCTEFTVCBTZXF1ZW5jZSBBbmFseXNpcyBUb29sIGh0dHA6Ly93d3cubmNiaS5ubG0ubmloLmdvdi9ib29rcy9OQksyMTA5Ny8gKDI2IERlY2VtYmVyIDIwMTMsIGRhdGUgbGFzdCBhY2Nlc3NlZCkgICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS82LzU4NS8xNzQ2Mjc1NS9kc3UwMjEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJHcmFpbiBhbWFyYW50aHMsIGVkaWJsZSBDNCBkaWNvdHMsIHByb2R1Y2UgcHNldWRvLWNlcmVhbHMgaGlnaCBpbiBseXNpbmUuIEx5c2luZSBiZWluZyBvbmUgb2YgdGhlIG1vc3QgbGltaXRpbmcgZXNzZW50aWFsIGFtaW5vIGFjaWRzIGluIGNlcmVhbHMgYW5kIEM0IHBob3Rvc3luIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNzU0NTM5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZN+qmzWPKOc3EKCEOe3dE3X3Vmvv" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f585%2f2754539" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f585%2f2754539" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=22047" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&amp;Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/6">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87196275">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196275">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196282">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196282">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196330">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196330">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196355">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196355">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196359">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196359">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196361">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196361">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196363">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196363">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/6/569/551125">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/6/603/2754543">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The Draft Genome and Transcriptome of <em>Amaranthus hypochondriacus</em>: A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Meeta Sunil</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Meeta+Sunil">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sunil M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sunil M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Meeta Sunil</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arun K. Hariharan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">GANIT Labs: Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arun+K.+Hariharan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hariharan A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hariharan A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Arun K. Hariharan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Soumya Nayak</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Soumya+Nayak">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nayak S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nayak S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Soumya Nayak</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Saurabh Gupta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">GANIT Labs: Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Saurabh+Gupta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gupta S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gupta S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Saurabh Gupta</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suran R. Nambisan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suran+R.+Nambisan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nambisan S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nambisan S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Suran R. Nambisan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ravi P. Gupta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ravi+P.+Gupta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gupta R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gupta R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ravi P. Gupta</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Binay Panda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">GANIT Labs: Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Binay+Panda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Panda B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Panda B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Binay Panda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bibha Choudhary</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bibha+Choudhary">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choudhary B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choudhary B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bibha Choudhary</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Subhashini Srinivasan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics and Applied Biotechnology</div>, <div class="addr-line">Biotech Park, Electronics City Phase I, Bangalore, Karnataka 560100</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 80-285-289-00. Fax. +91 80-285-289-04. Email: <a href="mailto:ssubha@ibab.ac.in" target="_blank">ssubha@ibab.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Subhashini+Srinivasan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srinivasan S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Srinivasan S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Subhashini Srinivasan</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label"><sup>#</sup></div><div class="footnote-content">These authors are registered Graduate students at Manipal University, Manipal, Karnataka, India.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label"><sup>†</sup></div><div class="footnote-content">These authors are investigators of Amaranth project at IBAB.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 585–602, <a href='https://doi.org/10.1093/dnares/dsu021'>https://doi.org/10.1093/dnares/dsu021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 July 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 March 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 June 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754539" href="/dnaresearch/article-pdf/21/6/585/17462755/dsu021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Meeta Sunil, Arun K. Hariharan, Soumya Nayak, Saurabh Gupta, Suran R. Nambisan, Ravi P. Gupta, Binay Panda, Bibha Choudhary, Subhashini Srinivasan;  The Draft Genome and Transcriptome of <em>Amaranthus hypochondriacus</em>: A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal, <em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 585–602, <a href='https://doi.org/10.1093/dnares/dsu021'>https://doi.org/10.1093/dnares/dsu021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754539?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754539?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754539?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754539?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754539?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754539?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754539?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20Draft%20Genome%20and%20Transcriptome%20of%20%26lt%3Bem%26gt%3BAmaranthus%20hypochondriacus%26lt%3B%2Fem%26gt%3B%3A%20A%20C4%20Dicot%20Producing%20High-Lysine%20Edible%20Pseudo-Cereal&amp;publicationDate=2014-07-28&amp;volumeNum=21&amp;issueNum=6&amp;author=Sunil%2C%20Meeta%3B%20Hariharan%2C%20Arun%20K.&amp;startPage=585&amp;endPage=602&amp;contentId=10.1093%2Fdnares%2Fdsu021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f585%2f2754539" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>Grain amaranths, edible C4 dicots, produce pseudo-cereals high in lysine. Lysine being one of the most limiting essential amino acids in cereals and C4 photosynthesis being one of the most sought-after phenotypes in protein-rich legume crops, the genome of one of the grain amaranths is likely to play a critical role in crop research. We have sequenced the genome and transcriptome of <em>Amaranthus hypochondriacus</em>, a diploid (2<em>n</em> = 32) belonging to the order Caryophyllales with an estimated genome size of 466 Mb. Of the 411 linkage single-nucleotide polymorphisms (SNPs) reported for grain amaranths, 355 SNPs (86%) are represented in the scaffolds and 74% of the 8.6 billion bases of the sequenced transcriptome map to the genomic scaffolds. The genome of <em>A. hypochondriacus</em>, codes for at least 24,829 proteins, shares the paleohexaploidy event with species under the superorders Rosids and Asterids, harbours 1 SNP in 1,000 bases, and contains 13.76% of repeat elements. Annotation of all the genes in the lysine biosynthetic pathway using comparative genomics and expression analysis offers insights into the high-lysine phenotype. As the first grain species under Caryophyllales and the first C4 dicot genome reported, the work presented here will be beneficial in improving crops and in expanding our understanding of angiosperm evolution.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Caryophyllales">Caryophyllales</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;grain amaranth&quot;">grain amaranth</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Amaranthus hypochondriacus&quot;"><em>Amaranthus hypochondriacus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;lysine biosynthesis&quot;">lysine biosynthesis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;C4 photosynthesis&quot;">C4 photosynthesis</a></div>                            <h2 scrollto-destination=87196275 id="87196275" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Grain amaranths, collectively referring to all the three grain-producing species of <em>Amaranthus</em> under the family Amaranthaceae, are among the most neglected crops in human history. Grain amaranths, <em>A. caudatus</em>, <em>A. cruentus</em>, and <em>A. hypochondriacus</em>, were domesticated for human consumption some 8,000 years ago and were the staple food for the Inca and the Aztec civilizations for thousands of years.<sup><a reveal-id="DSU021C1" data-open="DSU021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> However, cultivation of grain amaranths remains obscure during modern times following the ban on cultivation. In 1980s, the USA declared grain amaranths ‘the crop of the future’ based on its unique nutritional profile.<sup><a reveal-id="DSU021C1" data-open="DSU021C1" class="link link-ref link-reveal xref-bibr">1</a>–<a reveal-id="DSU021C3" data-open="DSU021C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The excitement over grain amaranths grew in 1972 when an Australian scientist John Downton reported that the edible cereal-like grains of amaranths display lysine content as high as in milk.<sup><a reveal-id="DSU021C1" data-open="DSU021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The importance of this finding is even more crucial considering decades of failed attempts, with existing engineering strategies, to improve the lysine content in rice and wheat because of the inherent inverse correlation between lysine content and yield.<sup><a reveal-id="DSU021C4" data-open="DSU021C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Grain amaranths are among the rare edible dicots that have evolved to use C4 photosynthesis. It is well established that the species with C4 photosynthesis are more efficient during drought and other environmental stress conditions. Considering that all species producing protein-rich edible grains, such as legumes and nuts, use C3 photosynthesis, C4 photosynthesis must be one of the most sought-after phenotypes in edible dicot crops. Now that the genomes of many C3 legumes have been sequenced,<sup><a reveal-id="DSU021C5" data-open="DSU021C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSU021C6" data-open="DSU021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> the genome of one of the grain amaranths is likely to aid in the improvement of protein-rich dicot plants using comparative genomics. Besides, it has been shown that even under carbon dioxide-rich conditions, C4 dicot species under the Amaranthaceae family perform better than respective C3 dicots in terms of dry mass,<sup><a reveal-id="DSU021C7" data-open="DSU021C7" class="link link-ref link-reveal xref-bibr">7</a></sup> making this species a contestant among other plant species as bioenergy resources.</p><p>Grain amaranths are also important contestants for expanding our narrow food base. Decorating the phylogenetic tree of angiosperms based on the Angiosperm Phylogeny Group (APG) III system of classification with edible plants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu021_Supplementary_Data.zip?Expires=2147483647&Signature=RS8wW9S16qC10ZeBwPtMKpIdFXcJ1ty71jwgljMOVgTZ745p1lwudxkFeUmRzdgeWB947GCkvQyVXnLkB15IPL5JSno7EMP-pnMvLdomtR61S2-czTjU6C8M8Tm1LS1UF9YcJTFnzIq5A8786GsCIGXT-6xIDR8-T6eBtr1mJqugzCC8xMSX9NiD0qdzQePC2y8nQk6GYv~ZqiVpo~jPDhmCTpk22kFwLcw-ldtg5f4Ef8IJiCha~Dd9A6Qhe4IMJtpUFAcnJUpBWgMhYvTWqEzKIZZWL6g0zyk1~7gk~I6heKdMSUJh8paB1wsvwYQ5P~C8BuiHSMAVGSkz01p~lw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>) reveals that all the major clades across the plant kingdom, except Caryophyllales, are loaded with edible crops. The order Caryophyllales (<a class="link link-uri" href="http://www.cs.man.ac.uk/~david/flora/flora.html">http://www.cs.man.ac.uk/~david/flora/flora.html</a>) comprises 70 genera and 800 species<sup><a reveal-id="DSU021C8" data-open="DSU021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and is relatively void of edible crops. All the cereals are classified under monocots, the legumes, fruits, and nuts under Rosids, and the staple root crops such as potato and sweet potato under Asterids. Hence, grain amaranths, classified under the order Caryophyllales, offer an attractive alternative for expanding our food base via supplementing protein-rich cereals in the plant-based diet.</p><p>The species under the order Caryophyllales, including grain amaranths, are also unique from other angiosperms with regard to the synthesis of important secondary metabolites.<sup><a reveal-id="DSU021C9" data-open="DSU021C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Unlike the use of phenylalanine-derived anthocyanins for use as colouring pigments by most angiosperms, species under Caryophyllales use betalains for the same purpose, which are derived from tyrosine.<sup><a reveal-id="DSU021C10" data-open="DSU021C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU021C11" data-open="DSU021C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Since tyrosine is also the precursor for melanin, the colouring pigment in mammals, the genome of a grain amaranth can, perhaps, also be exploited to develop cosmetics and cure for various skin disorders.<sup><a reveal-id="DSU021C12" data-open="DSU021C12" class="link link-ref link-reveal xref-bibr">12</a></sup> For example, based on the ethnopharmaceutical survey, one of the plant species used by Rwandese to cure skin discolouration is a species under Caryophyllales, <em>Chenopodium ugandae</em>.<sup><a reveal-id="DSU021C12" data-open="DSU021C12" class="link link-ref link-reveal xref-bibr">12</a></sup> To this end, more recently, the genome of the first Caryophyllales, <em>Beta vulgaris</em>, has been reported and will aid in expanding our understanding of betalains.<sup><a reveal-id="DSU021C13" data-open="DSU021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The genome of one of the grain amaranths, whose flowers are already used as food colouration, will add to the study of enzymes involved in betalain synthesis.</p><p>Despite the huge potential of grain amaranths with regard to human health and disease, the genome of grain amaranths is yet to be deciphered. A partial transcriptome of <em>A. hypochondriacus</em>, one of the grain amaranths, has been reported, which was generated to understand how species under the Amaranthaceae family respond to environmental stress.<sup><a reveal-id="DSU021C14" data-open="DSU021C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In another application, limited sequencing of the genome of <em>Amaranthus tuberculatus</em>, a weed variety, has been generated to decipher the mechanism by which this species develops resistance to triple herbicides.<sup><a reveal-id="DSU021C15" data-open="DSU021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> A more systematic genomic effort on grain amaranths includes creation of BAC libraries for all the three grain species,<sup><a reveal-id="DSU021C16" data-open="DSU021C16" class="link link-ref link-reveal xref-bibr">16</a></sup> developing microsatellite markers,<sup><a reveal-id="DSU021C17" data-open="DSU021C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and creating a single-nucleotide polymorphism (SNP)-based linkage map.<sup><a reveal-id="DSU021C18" data-open="DSU021C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Genomic studies of grain amaranths from the point of view of their unique nutritional profiles and C4 photosynthesis among edible dicots are limited. In various parts of South Asia including Nepal, grain amaranths have been cultivated as staple crops for consumption as an alternative to cereals by the locals.<sup><a reveal-id="DSU021C19" data-open="DSU021C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Here, we present the draft genome and transcriptome of a landrace that belongs to <em>A. hypochondriacus</em> (Rajgira) obtained from stable lines from the farmers in northern Karnataka, India, to aid in deciphering the genotype behind these important and unique traits from the edible variety.</p>                            <h2 scrollto-destination=87196282 id="87196282" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87196283 id="87196283" class="section-title" data-legacy-id=s2a>2.1. Sample collection for genome and transcriptome</h3>
<div class="block-child-p">We obtained the domesticated grains of <em>A. hypochondriacus</em> from farmers in northern Karnataka growing this as a crop for consumption in the name of ‘Rajgira’ or ‘Rajeera’ throughout southern India. We purchased the seeds of other grain species, <em>A. cruentus</em> and <em>A. caudatus</em>, from Park Seeds sold in the names of ‘Autumn's Touch’ and ‘Love-Lies-Bleeding’, respectively. These three types of seeds were grown in campus grounds in three separate lots for taxonomic purposes (Fig. <a reveal-id="DSU021F1" data-open="DSU021F1" class="link link-reveal link-table xref-fig">1</a>). Large numbers of seeds from the first round from each grain species were whitish or pinkish for <em>A. hypochondriacus</em>, whitish for <em>A. cruentus</em>, and reddish for <em>A. caudatus</em> (shown inset in Fig. <a reveal-id="DSU021F1" data-open="DSU021F1" class="link link-reveal link-table xref-fig">1</a>). <em>Amaranthus hypochondriacus</em> produced two types of plants—one with white and the other with red inflorescences. Separating the seeds of white and red varieties of <em>A. hypochondriacus</em> and growing for over two generations retained the inflorescence of the parent plants in the successive generations, thus suggesting purity of lines obtained from farmers. Also, repeated generations produced not only 100% white seeds for the white plants of <em>A. hypochondriacus</em>, but also the plants looked very similar in size, inflorescence, and yield. We have used the tissues from the plants of <em>A. hypochondriacus</em> with white inflorescence after two generations for sequencing both the genome and the transcriptomes reported here. <div data-id="DSU021F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02101.jpeg?Expires=2147483647&Signature=XlA62dbZgu9oz2L3ICZ2ojLbHka7XGinjR~3OQxvOJ~qEqqN5p5EjRqcvoGtCwn-MdRalJeY4yjNjWBOd18AFQSzQV0KLpWEYvwKV2EpRibB1VN859jXQvphjgkicX0V136ewwZXuAGBrc31HuDiZiuN9DrqOaawZN6FqtyGrt-tCfu92cmZMIOSI9FYYmCx7~hLaeC6hQHHdQaOJGXBDeHV9qaBmh6jTESZDOrVmdUXA7ZiGPxHIDuTlW~YseJa9MBXQ5LclzLoEazGfyNsNr6x6yUMvKzO2Oqu7wMn8JMFWYiIJicGoALXhrRw7Il9DUAo0JGv2pcbgSN-jlDusw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Photographs of the three grain species of Amaranthus grown on campus grounds. The plants from left to right are: A. hypochondriacus, A. cruentus, and A. caudatus, as identified by their taxonomic features including those of the inflorescence and the leaves. Amaranthus hypochondriacus contains both a red variety and a white variety (the sequenced one). Insets are the corresponding seeds obtained after two generations of growing these species on campus grounds. For A. hypochondriacus (left), the seeds from red plants are at the bottom left and those from white plants are shown at the top right." path-from-xml="dsu02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196284/dsu02101.jpeg" path-from-xml="dsu02101.jpeg" target="_blank">View large</a><a section="87196284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02101.jpeg?Expires=2147483647&Signature=klradhDU6NYN-amfW9muDLQkdRWzfFzdE-McEDgW5p7~seMArE4PApnxWEEgL4Z1IjXBDKzkGt0SDwnSxfPEe6u5-EAZDBwv30V1FiU1TS58jcMiUewxnOf71wnHQ52rV6brVFfPQcFEcTpGw-jLHBFeGGK9Qf8hikI8eDERwf1-bvftY1DK0BQDmfPaBBj76aninj0Vo6jLTvntW3OuiztM0lPT5fS2UnM3sCaylrx3i8AzNTGyj5hDTBun9-qit0bX1ECUF-Lv-LowWEC~24vYuXEQags7ZcB4B-nFYkDk-wR52cfnR9jL177NgR6zHn6BtXfQqmnLELpO8xJOMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196284&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Photographs of the three grain species of <em>Amaranthus</em> grown on campus grounds. The plants from left to right are: <em>A. hypochondriacus</em>, <em>A. cruentus</em>, and <em>A. caudatus</em>, as identified by their taxonomic features including those of the inflorescence and the leaves. <em>Amaranthus hypochondriacus</em> contains both a red variety and a white variety (the sequenced one). Insets are the corresponding seeds obtained after two generations of growing these species on campus grounds. For <em>A. hypochondriacus</em> (left), the seeds from red plants are at the bottom left and those from white plants are shown at the top right.</p></div></div><div content-id="DSU021F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02101.jpeg?Expires=2147483647&Signature=XlA62dbZgu9oz2L3ICZ2ojLbHka7XGinjR~3OQxvOJ~qEqqN5p5EjRqcvoGtCwn-MdRalJeY4yjNjWBOd18AFQSzQV0KLpWEYvwKV2EpRibB1VN859jXQvphjgkicX0V136ewwZXuAGBrc31HuDiZiuN9DrqOaawZN6FqtyGrt-tCfu92cmZMIOSI9FYYmCx7~hLaeC6hQHHdQaOJGXBDeHV9qaBmh6jTESZDOrVmdUXA7ZiGPxHIDuTlW~YseJa9MBXQ5LclzLoEazGfyNsNr6x6yUMvKzO2Oqu7wMn8JMFWYiIJicGoALXhrRw7Il9DUAo0JGv2pcbgSN-jlDusw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Photographs of the three grain species of Amaranthus grown on campus grounds. The plants from left to right are: A. hypochondriacus, A. cruentus, and A. caudatus, as identified by their taxonomic features including those of the inflorescence and the leaves. Amaranthus hypochondriacus contains both a red variety and a white variety (the sequenced one). Insets are the corresponding seeds obtained after two generations of growing these species on campus grounds. For A. hypochondriacus (left), the seeds from red plants are at the bottom left and those from white plants are shown at the top right." path-from-xml="dsu02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196284/dsu02101.jpeg" path-from-xml="dsu02101.jpeg" target="_blank">View large</a><a section="87196284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02101.jpeg?Expires=2147483647&Signature=klradhDU6NYN-amfW9muDLQkdRWzfFzdE-McEDgW5p7~seMArE4PApnxWEEgL4Z1IjXBDKzkGt0SDwnSxfPEe6u5-EAZDBwv30V1FiU1TS58jcMiUewxnOf71wnHQ52rV6brVFfPQcFEcTpGw-jLHBFeGGK9Qf8hikI8eDERwf1-bvftY1DK0BQDmfPaBBj76aninj0Vo6jLTvntW3OuiztM0lPT5fS2UnM3sCaylrx3i8AzNTGyj5hDTBun9-qit0bX1ECUF-Lv-LowWEC~24vYuXEQags7ZcB4B-nFYkDk-wR52cfnR9jL177NgR6zHn6BtXfQqmnLELpO8xJOMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196284&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Photographs of the three grain species of <em>Amaranthus</em> grown on campus grounds. The plants from left to right are: <em>A. hypochondriacus</em>, <em>A. cruentus</em>, and <em>A. caudatus</em>, as identified by their taxonomic features including those of the inflorescence and the leaves. <em>Amaranthus hypochondriacus</em> contains both a red variety and a white variety (the sequenced one). Insets are the corresponding seeds obtained after two generations of growing these species on campus grounds. For <em>A. hypochondriacus</em> (left), the seeds from red plants are at the bottom left and those from white plants are shown at the top right.</p></div></div></div>                            <h3 scrollto-destination=87196285 id="87196285" class="section-title" data-legacy-id=s2b>2.2. Taxonomy of Amaranthus</h3>
<p><em>Amaranthus</em> is a monoecious plant and the inflorescence is a thyrse with a racemose (catkin) main axis and cymose clusters (cymules) of one male flower and two or more female flowers. <em>Amaranthus hypochondriacus</em> is characterized by its apical erect inflorescence, which is heavily loaded with small edible seeds at maturity. The inflorescence can be bright red or whitish green depending on the presence or absence of betalain pigments in the plant variety (left, Fig. <a reveal-id="DSU021F1" data-open="DSU021F1" class="link link-reveal link-table xref-fig">1</a>). The inflorescence of <em>A. cruentus</em> is also apical but with a Christmas-tree like topology that may gradually turn orange at maturity (middle, Fig. <a reveal-id="DSU021F1" data-open="DSU021F1" class="link link-reveal link-table xref-fig">1</a>). The seeds are comparatively larger in size and the stem more robust than that of the other two species. <em>Amaranthus caudatus</em> produces apical but drooping inflorescences that are red in colour and produce small reddish seeds (right, Fig. <a reveal-id="DSU021F1" data-open="DSU021F1" class="link link-reveal link-table xref-fig">1</a>). The plant is also comparatively fragile unlike the other two grain species.</p>                            <h3 scrollto-destination=87196287 id="87196287" class="section-title" data-legacy-id=s2c>2.3. Karyotyping</h3>
<div class="block-child-p">The seeds of the plants selected for sequencing were allowed to germinate in water for 24 h. The root tips were incubated at 4°C for 4 h and then fixed using Carnoy's solution. The processed root tips were excised and treated with 1 N HCl for 1 min, followed by treatment with 45% acetic acid for 10 min, staining with 0.2% aceto-orcein along with gentle heat fixing over a spirit lamp, squashing with the thumb, and mounting using 50% glycerol on a microscopic glass slide. The image was captured at 100× magnification under a compound light microscope. As shown in Fig. <a reveal-id="DSU021F2" data-open="DSU021F2" class="link link-reveal link-table xref-fig">2</a>, the number of chromosomes in the selected diploid plant was confirmed to be 32 (<em>n</em> = 16) for sequencing purposes. <div data-id="DSU021F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02102.jpeg?Expires=2147483647&Signature=qDfvcGdBild96jNSkynb9O6uJeHmjHDmtgksEyY9d4iVfOX0PWesvyhGaOdzhR7M0qSJeqNhj3COdZWsG58ZgNvY6LtMy91KeEuaGtcrTjYd6g3yVu4R5MlL8Wl7-pvNd6OdYxSJINpVjK7kVsTRLSGW8S8Zyzd7agHN9zX8BccMuaNUmy~Q2A5AcHXpbny4oVdhObkG~mBwMLgW92Hu9cAgripc6pnXDceKI4ypVl6GlN~~fScUuv4KIP25KuR14AU3txTCqf3MHgR6y43V1vc~T9O5Xym1EubAq5EVmXWiAYkk6XT-Z-a0z25mwkjFQh1pzY8LOVu73JfBrQthEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Karyotype of A. hypochondriacus, the species selected for sequencing, showing diploidy with 32 chromosomes in mitotic root tips. The karyotype shows 32 chromosomes under 100× magnification stained with 0.2% aceto-orcein, as would be expected from the diploid root tips of A. hypochondriacus. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196288/dsu02102.jpeg" path-from-xml="dsu02102.jpeg" target="_blank">View large</a><a section="87196288" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02102.jpeg?Expires=2147483647&Signature=BqLGARCdZUtTzISSjuDzENEMolx1dJrSUnXu3sUCNEY-dfgIqBvRIbrvOgRRapEjzwAgCOaVOi6~-ZYKhF3FEvwiliDmgl-cuYH28wFT8jBPLH1JfRNdmWHMqtvOhbwVJd1BGxmJw2rcU7oW-OBv~juGIKD3vnzvszQbEhHDLOprgM7R0LXyH3xBlJ0EQTfcbLnfrIFEXOwGHlRNLjTcQ8lVe2v5l0Ml8VULrY9QPjUeHNg5Oh9NsEblu4Zxl2yfUrr2z7spGi6lWPQz-qwej0x6v5t8EF2KWjiNAkktSWvX-qsDOiTdW1vK1u42NLLdAFFRShU~wq57IKQyRCL1Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196288&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Karyotype of <em>A. hypochondriacus</em>, the species selected for sequencing, showing diploidy with 32 chromosomes in mitotic root tips. The karyotype shows 32 chromosomes under 100× magnification stained with 0.2% aceto-orcein, as would be expected from the diploid root tips of <em>A. hypochondriacus</em>. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU021F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02102.jpeg?Expires=2147483647&Signature=qDfvcGdBild96jNSkynb9O6uJeHmjHDmtgksEyY9d4iVfOX0PWesvyhGaOdzhR7M0qSJeqNhj3COdZWsG58ZgNvY6LtMy91KeEuaGtcrTjYd6g3yVu4R5MlL8Wl7-pvNd6OdYxSJINpVjK7kVsTRLSGW8S8Zyzd7agHN9zX8BccMuaNUmy~Q2A5AcHXpbny4oVdhObkG~mBwMLgW92Hu9cAgripc6pnXDceKI4ypVl6GlN~~fScUuv4KIP25KuR14AU3txTCqf3MHgR6y43V1vc~T9O5Xym1EubAq5EVmXWiAYkk6XT-Z-a0z25mwkjFQh1pzY8LOVu73JfBrQthEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Karyotype of A. hypochondriacus, the species selected for sequencing, showing diploidy with 32 chromosomes in mitotic root tips. The karyotype shows 32 chromosomes under 100× magnification stained with 0.2% aceto-orcein, as would be expected from the diploid root tips of A. hypochondriacus. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196288/dsu02102.jpeg" path-from-xml="dsu02102.jpeg" target="_blank">View large</a><a section="87196288" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02102.jpeg?Expires=2147483647&Signature=BqLGARCdZUtTzISSjuDzENEMolx1dJrSUnXu3sUCNEY-dfgIqBvRIbrvOgRRapEjzwAgCOaVOi6~-ZYKhF3FEvwiliDmgl-cuYH28wFT8jBPLH1JfRNdmWHMqtvOhbwVJd1BGxmJw2rcU7oW-OBv~juGIKD3vnzvszQbEhHDLOprgM7R0LXyH3xBlJ0EQTfcbLnfrIFEXOwGHlRNLjTcQ8lVe2v5l0Ml8VULrY9QPjUeHNg5Oh9NsEblu4Zxl2yfUrr2z7spGi6lWPQz-qwej0x6v5t8EF2KWjiNAkktSWvX-qsDOiTdW1vK1u42NLLdAFFRShU~wq57IKQyRCL1Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196288&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Karyotype of <em>A. hypochondriacus</em>, the species selected for sequencing, showing diploidy with 32 chromosomes in mitotic root tips. The karyotype shows 32 chromosomes under 100× magnification stained with 0.2% aceto-orcein, as would be expected from the diploid root tips of <em>A. hypochondriacus</em>. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div></div>                            <h3 scrollto-destination=87196289 id="87196289" class="section-title" data-legacy-id=s2d>2.4. Extraction of genomic DNA</h3>
<p>DNA for sequencing was extracted from fresh leaves of 45- to 50-day-old plants using the DNEasy Mini Plant DNA Extraction kit (Qiagen). The quality of the extracted DNA was checked using fluorometry (Qubit, Invitrogen) and by agarose gel electrophoresis.</p>                            <h3 scrollto-destination=87196291 id="87196291" class="section-title" data-legacy-id=s2e>2.5. Extraction of total RNA</h3>
<p>For RNA extraction, leaf and stem tissues were collected from a pool of more than 10 individuals, from the plants grown from the same seed collection from which the genome was sequenced, at (i) 15 days, (ii) 25 and (iii) 30 days of age, and from (iv) mature seeds. The leaf and stem tissues, on excision, were immediately cleaned with DEPC-treated water, flash-frozen in liquid nitrogen, and stored at −80°C until RNA extraction was done. The mature seeds were collected from the inflorescence which was sun-dried, threshed, cleaned, and then stored at room temperature until RNA extraction was done. Total RNA was extracted using the conventional phenol–chloroform extraction method<sup><a reveal-id="DSU021C20" data-open="DSU021C20" class="link link-ref link-reveal xref-bibr">20</a></sup> as standardized in the laboratory. Quality assessment of the total RNA was done on 1% agarose gel and the Agilent 2100 Bioanalyzer using high-sensitivity RNA chips.</p>                            <h3 scrollto-destination=87196293 id="87196293" class="section-title" data-legacy-id=s2f>2.6. Library preparation</h3>
<p>For genome sequencing, one paired-end (PE; with an insert size of 300 bp) and four mate-pair (MP; with insert sizes of 1.75, 3, 5 and 10 kb) DNA libraries were made using the TruSeq DNA Sample Preparation Kit (Illumina) by following the manufacturer's low-throughput (LT) protocol. One microgram and 10 µg of the genomic DNA were used for the preparation of the PE and the MP libraries, respectively. DNA molecules were sheared by Covaris S2 according to the manufacturer's instructions and processed as follows.</p>                            <h4 scrollto-destination=87196295 id="87196295" class="section-title" data-legacy-id=s2f1>2.6.1. PE genomic libraries</h4>
<p>The fragmented DNA molecules were made blunt by performing end repair in End-Repair Mix (containing T4 polynucleotide kinase for 5′ phosphorylation, T4 DNA polymerase to fill in the 5′ overhangs, and Klenow to remove the 3′ overhangs) for 30 min at 30°C, and purified using Agencourt AMPure XP beads (Beckman Coulter) according to the manufacturer's recommendations. Furthermore, 3′ adenylation of the blunt-end DNA was carried out by incubating with A-Tailing Mix (containing a Klenow fragment) at 37°C for 30 min. Ligation of TruSeq adapters was done using DNA Ligase Mix (containing T4 DNA ligase) by incubating at 30°C for 10 min, and the reaction was stopped by adding Stop Ligase Mix. The adenylated DNA was purified using the Agencourt AMPure XP beads as done previously. Library size selection (for an insert size of 300 bp) was done using agarose gel electrophoresis, and the adapter-ligated purified DNA was further enriched by PCR for 10 cycles using adapter complementary primers (P5 and P7) followed by a clean-up using the Agencourt AMPure XP beads. The quality and quantity of the library were estimated by spectrophotometry (NanoDrop, Thermo Scientific) and fluorometry (Qubit, Invitrogen), and the size distribution was analysed on the Agilent 2100 Bioanalyzer using high-sensitivity DNA chips.</p>                            <h4 scrollto-destination=87196297 id="87196297" class="section-title" data-legacy-id=s2f2>2.6.2. MP genomic libraries</h4>
<p>The fragmented DNA molecules were made blunt by performing end repair in End-Repair Mix for 15 min at 20°C, followed by end labelling for 15 min at 20°C using biotin-dNTP mix and purification using the Agencourt AMPure XP beads. DNA of appropriate size (1.75, 3, 5, and 10 kb) was selected using agarose gel electrophoresis and circularized using Circularization Ligase. Any remaining linear DNA in the circularization step was digested using Exonuclease, and the circular DNA was again sheared using Covaris S2 as done previously. The biotinylated DNA fragments were purified using Dynal magnetic M-280 streptavidin beads as per the manufacturer's guidelines. Again, end repair was performed by incubating the biotinylated DNA fragments (immobilized on the streptavidin beads) in End-Repair Mix for 30 min at 20°C and purified; 3′ adenylation was done by incubating the immobilized DNA molecules with A-Tailing Mix at 37°C for 30 min. Ligation of TruSeq adapters was done using DNA Ligase Mix by incubating at 20°C for 15 min and the reaction was stopped by adding Stop Ligase Mix. The biotinylated adapter-ligated immobilized DNA was further enriched by PCR for 18 cycles using adapter complementary primers followed by a clean-up using the Agencourt AMPure XP beads. The quality and quantity of the library were estimated by spectrophotometry (NanoDrop, Thermo Scientific) and fluorometry (Qubit, Invitrogen), and the size distribution was analysed on the Agilent 2100 Bioanalyzer using high-sensitivity DNA chips.</p><p>Transcriptome libraries of 155 bp were prepared using the TruSeq RNA Sample Preparation Kit (Illumina) following the manufacturer's LT protocol. Two micrograms of total RNA were subjected to mRNA selection using poly-T oligo-attached magnetic beads using two rounds of purification. The selected poly-A RNA was subjected to fragmentation to sizes between 100 and 300 nt by incubating in fragmentation mix for 8 min at 94°C. First-strand cDNA was synthesized by adding 1 µl of Superscript II reverse transcriptase (Invitrogen) to solution containing primed RNA and first-strand master mix followed by incubation at 25°C for 10 min, 42°C for 50 min, and 70°C for 15 min. The complementary second-strand cDNA was synthesized by incubating first-strand cDNA in second-strand master mix at 16°C for 60 min. End repair was performed to remove the 3′ overhangs and fill the 5′ overhangs by incubating the DNA in End-Repair Mix containing T4 polynucleotide kinase, T4 DNA polymerase, and a large (Klenow) fragment of DNA polymerase I for 30 min at 30°C, and purified using Agencourt AMPure XP beads (Beckman Coulter) according to the manufacturer's recommendations. A-tailing of DNA was performed at 37°C for 30 min with the Klenow fragment followed by ligation of TruSeq adaptors using T4 DNA ligase by incubating at 30°C for 10 min. The reaction was stopped by adding stop-ligase mix and purified by using Agencourt AMPure XP beads. Adaptor-ligated DNA was then subjected to PCR enrichment with adaptor complementary primers for 15 cycles followed by clean-up using Agencourt AMPure XP beads. The quality and quantity of the library were estimated by NanoDrop and fluorometry (Qubit, Invitrogen), and the size distribution was analysed on the Agilent Bioanalyzer using high-sensitivity DNA chips.</p>                            <h3 scrollto-destination=87196300 id="87196300" class="section-title" data-legacy-id=s2g>2.7 Quantification of prepared library and sequencing</h3>
<p>The libraries were quantified using SYBR-Green-based qPCR reagents (Kapa Biosystems). The qPCR results were compared with the predetermined concentration of the PhiX library. Eight picomolar of all the DNA libraries were used for cluster generation in cBot (Illumina). Also, 8 pM of the four RNA libraries prepared were pooled in equimolar ratio and seeded on four lanes of the flow cell for cluster generation. All the libraries were sequenced on Illumina Genome Analyzer IIx following the manufacturer's standard protocols. PE genomic reads of length 75 bases, MP genomic reads of length 36 bases, and PE transcriptomic reads of length 72 bases were generated.</p>                            <h3 scrollto-destination=87196302 id="87196302" class="section-title" data-legacy-id=s2h>2.8. Assembly</h3>
                            <h4 scrollto-destination=87196303 id="87196303" class="section-title" data-legacy-id=s2h1>2.8.1. QC of sequence reads</h4>
<p>The reads were analysed for their quality and all those with &gt;75% of bases with phred scores of ≥20 and &lt;15 Ns were considered to be of high quality.</p>                            <h4 scrollto-destination=87196305 id="87196305" class="section-title" data-legacy-id=s2h2>2.8.2. Assembly of genomic reads from Illumina GAIIx platform</h4>
<p>After testing for quality, the reads were assembled using SOAPdenovo (SOAPdenovo31mer version 1.05).<sup><a reveal-id="DSU021C21" data-open="DSU021C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Assembly was performed in two stages in a high configuration system with 96 GB RAM. First, 465,246,312 PE reads of 72-mers were assembled using default parameters and a <em>k</em>-mer size of 31. In the other set, using the same parameters, all the reads totalling 869,113,408 both from PE and MP libraries were assembled. The resulting scaffolds were subject to various analyses such as synteny, heterozygosity, Gene Ontology (GO) annotation, repeat analysis, duplication study, and proteome prediction, using available tools and scripts developed in-house.</p>                            <h4 scrollto-destination=87196307 id="87196307" class="section-title" data-legacy-id=s2h3>2.8.3. Assembly of transcriptomic reads from 454 platform</h4>
<p>Transcriptomic reads with the seed accession ID of IC 38040 (PI 480569) were downloaded from NCBI-SRA (SRX055331) and assembled into contigs using MIRA.<sup><a reveal-id="DSU021C22" data-open="DSU021C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h4 scrollto-destination=87196309 id="87196309" class="section-title" data-legacy-id=s2h4>2.8.4. Assembly of transcriptomic reads from Illumina platform</h4>
<p>The high-quality reads from various tissues were pooled together and used as a set with Trinity (Release 16 February 2013) for assembly.<sup><a reveal-id="DSU021C23" data-open="DSU021C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Trinity, a <em>de novo</em> transcriptome assembler, uses a fixed <em>k</em>-mer length of 25 and it was run on a high configuration system with 48 GB RAM to generate trinity components.</p>                            <h3 scrollto-destination=87196311 id="87196311" class="section-title" data-legacy-id=s2i>2.9. Filtering criteria</h3>
<p>The coverage of the scaffolds was measured by aligning the genomic scaffolds of <em>A. hypochondriacus</em> with various public sequencing data. All comparisons with the same species were done at the DNA level (blastn) and with other species at the amino acid levels (tblastn or tblastx).<sup><a reveal-id="DSU021C24" data-open="DSU021C24" class="link link-ref link-reveal xref-bibr">24</a></sup> A filtering criterion of &gt;99% over a contiguous stretch of &gt;100 bases is used to establish identity with sequences from the same species, and a filtering criterion of 60% match over a contiguous stretch of 35 or more amino acids is used to establish the orthology of genes with other plant species such as <em>Arabidopsis thaliana</em>, <em>Vitis vinifera</em>, and <em>Solanum lycopersicum</em>.</p>                            <h3 scrollto-destination=87196313 id="87196313" class="section-title" data-legacy-id=s2j>2.10. Synteny analyses</h3>
<p>All the synteny analyses were performed using SyMap<sup><a reveal-id="DSU021C25" data-open="DSU021C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and in-house perl scripts.</p>                            <h3 scrollto-destination=87196315 id="87196315" class="section-title" data-legacy-id=s2k>2.11. Heterozygosity analysis</h3>
<p>Frequency of mutation in the genomic scaffolds was estimated by piling up the genomic reads to the scaffolds using Bowtie<sup><a reveal-id="DSU021C26" data-open="DSU021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and SAMtools<sup><a reveal-id="DSU021C27" data-open="DSU021C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and counting the number of reads with an alternate base (minor allele) from that present in the reference. For the study, positions with depth between 10X and 315X (three times the read coverage) and minor allele frequency of 0.3 or more were considered. Also, the <em>k</em>-mer spectra of the sequenced genome and transcriptome were generated from the high-quality sequence reads using SOAPdenovo. The <em>k</em>-mer size was selected as 31 for the purpose as well as for the assembly of the genomic reads. The <em>k</em>-mer count with a very low frequency of 1 and 2 was removed.</p>                            <h3 scrollto-destination=87196317 id="87196317" class="section-title" data-legacy-id=s2l>2.12. GO annotation</h3>
<p>The GO annotation for all the <em>A. thaliana</em> genes with orthology to other species was compared using the ‘GO annotation search, functional categorization and download’ tool available at TAIR.<sup><a reveal-id="DSU021C28" data-open="DSU021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The percentage of <em>A. thaliana</em> genes within each GO category that are orthologous to scaffolds in the draft genomes of <em>V. vinifera</em>, <em>S. lycopersicum</em>, and <em>A. hypochondriacus</em> is computed as a percentage of the total number of <em>A. thaliana</em> proteins within the respective category for that species. The percentage of genes within each category reported by GO analysis for each species is normalized with respect to that of <em>A. thaliana</em> to estimate the number of genes deciphered within each GO category for the three species compared.</p>                            <h3 scrollto-destination=87196319 id="87196319" class="section-title" data-legacy-id=s2m>2.13. Proteome prediction</h3>
<p><em>Ab initio</em> gene predictions were done on the genomic scaffolds using GENSCAN,<sup><a reveal-id="DSU021C29" data-open="DSU021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> AUGUSTUS,<sup><a reveal-id="DSU021C30" data-open="DSU021C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and GeneMark,<sup><a reveal-id="DSU021C31" data-open="DSU021C31" class="link link-ref link-reveal xref-bibr">31</a></sup> with the smallest predicted sequence being 100 nucleotides long. These predicted sequences were annotated using BLAST against plant protein databases. Extrinsic prediction from genomic scaffolds and transcripts was also done using BLAST against plant proteomes downloaded from PlantGDB<sup><a reveal-id="DSU021C32" data-open="DSU021C32" class="link link-ref link-reveal xref-bibr">32</a></sup> including UniProt, mRNA, and HTC databases.</p>                            <h3 scrollto-destination=87196321 id="87196321" class="section-title" data-legacy-id=s2n>2.14. Repeat analysis</h3>
<p>We have used both extrinsic and <em>ab initio</em> methods to assess the extent of repeat elements in <em>A. hypochondriacus</em>, compared with selected plant species. For extrinsic analysis, we have used Repbase<sup><a reveal-id="DSU021C33" data-open="DSU021C33" class="link link-ref link-reveal xref-bibr">33</a></sup> using <em>A. thaliana</em> repeats as templates. For the <em>de novo</em> method, we have used RepeatScout<sup><a reveal-id="DSU021C34" data-open="DSU021C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and RepeatModeler<sup><a reveal-id="DSU021C35" data-open="DSU021C35" class="link link-ref link-reveal xref-bibr">35</a></sup> to predict novel repeats from the genomic scaffolds and masked the genome with the respective repeat element using RepeatMasker.<sup><a reveal-id="DSU021C36" data-open="DSU021C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSU021C37" data-open="DSU021C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h3 scrollto-destination=87196323 id="87196323" class="section-title" data-legacy-id=s2o>2.15. Duplication event</h3>
<p>Protein-level homology was studied between paralogous pairs in several species belonging to different plant orders including <em>A. thaliana</em>, <em>Glycine max</em>, and <em>V. vinifera</em> from Rosids, <em>S. lycopersicum</em> from Asterids, <em>A. hypochondriacus</em> and <em>B. vulgaris</em> (USDA version) from Caryophyllales, and <em>Oryza sativa</em>, <em>Sorghum bicolor</em>, and <em>Zea mays</em> from the monocots. The proteomes for <em>A. thaliana</em>, <em>G. max</em>, <em>V. vinifera</em>, <em>S. lycopersicum</em>, <em>O. sativa</em>, <em>S. bicolor</em>, and <em>Z. mays</em> were downloaded from public repositories (TAIR<sup><a reveal-id="DSU021C28" data-open="DSU021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and PGDD<sup><a reveal-id="DSU021C38" data-open="DSU021C38" class="link link-ref link-reveal xref-bibr">38</a></sup>), whereas those for <em>A. hypochondriacus</em> and <em>B. vulgaris</em> were predicted <em>ab initio</em> using Genscan from genomic scaffolds. Also, the GENSCAN-predicted proteome of <em>S. lycopersicum</em> was used as control to negate any possibility of bias in the result due to the gene prediction tool used for <em>A. hypochondriacus</em> and <em>B. vulgaris</em>. Furthermore, frequency polygons and kernel density plots of percentage identities of significant hits (those that were not self-hits or isoforms with 100 percent identity, and where the alignment covered at least 80% of the query or the subject sequence, whichever was longer) were plotted using R.<sup><a reveal-id="DSU021C39" data-open="DSU021C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                            <h3 scrollto-destination=87196325 id="87196325" class="section-title" data-legacy-id=s2p>2.16. Phylogenetic tree</h3>
<p>Amino acid sequences of aspartate kinase (AK) and phosphoenolpyruvate carboxylase (PEPC) genes available at GenBank for plants of interest were collected. Sequences for the remaining species for which genomic scaffolds were available were extracted from the results of BLAST<sup><a reveal-id="DSU021C24" data-open="DSU021C24" class="link link-ref link-reveal xref-bibr">24</a></sup> against the sequence from <em>A. thaliana</em> or from the closest taxon for which the sequence was available.</p><p>Multiple sequence alignment was carried out using MUSCLE,<sup><a reveal-id="DSU021C40" data-open="DSU021C40" class="link link-ref link-reveal xref-bibr">40</a></sup> present as an application in MEGA5, using the default parameters. The evolutionary history was inferred using the Neighbor-Joining method.<sup><a reveal-id="DSU021C41" data-open="DSU021C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The bootstrap consensus tree inferred from 1,000 replicates is taken to represent the evolutionary history of the taxa analysed.<sup><a reveal-id="DSU021C42" data-open="DSU021C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Branches corresponding to partitions reproduced in &lt;50% bootstrap replicates are collapsed. The evolutionary distances were computed using the Poisson correction method in the units of the number of amino acid substitutions per site.<sup><a reveal-id="DSU021C43" data-open="DSU021C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The analyses involved 25 AK and 75 PEPC sequences. Positions in the alignment containing gaps and missing data were eliminated to get a contiguous stretch of aligned sequences. There were a total of 229 and 278 positions in the final dataset for AK and PEPC proteins, respectively. Evolutionary analyses were conducted in MEGA5.<sup><a reveal-id="DSU021C44" data-open="DSU021C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The phylogenetic trees were visualized using iTOL.<sup><a reveal-id="DSU021C45" data-open="DSU021C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSU021C46" data-open="DSU021C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p>                            <h3 scrollto-destination=87196328 id="87196328" class="section-title" data-legacy-id=s2q>2.17. Gene expression profiling</h3>
<p>RNA-seq reads from each sample were mapped using bowtie to the full-length cDNA obtained for all the enzymes in the lysine biosynthesis pathway by virtual splicing from genomic scaffolds and transcriptome. The read counts were converted to reads per kilo base of exons per million reads mapped (RPKM) to normalize the measurement across samples.</p>                            <h2 scrollto-destination=87196330 id="87196330" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87196331 id="87196331" class="section-title" data-legacy-id=s3a>3.1. Coverage by sequencing, assembly, and mapping</h3>
<p>The species <em>A. hypochondriacus</em> (Rajgira), grown in northern Karnataka, India, was selected for sequencing. Following plant selection and genomic DNA extraction, five PE libraries of 300 bp and four MP libraries with insert sizes ranging from 1.5 to 10 kb were created. A total of 869,113,408 high-quality reads including 465,246,312 75-mers from PE and 403,867,096 36-mers from MP libraries were sequenced to produce a total of 49.4 billion bases, thus providing a coverage of 106-fold over the estimated genome size of 466 million bases.<sup><a reveal-id="DSU021C16" data-open="DSU021C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Also, we have generated 8.6 billion bases of transcriptome from three stages of shoot development and from mature seeds.</p><div class="block-child-p">Table <a reveal-id="DSU021TB1" data-open="DSU021TB1" class="link link-reveal link-table xref-fig">1</a> includes the statistics from the assembly of genomic reads into scaffolds. The total ATGC content, without Ns, within the assembled scaffolds is 273 million bases. Addition of MP reads improved the size of the largest scaffold by 20 folds to 485,353 from 24,878 bases and scaffold N50 to 35,089 from contig N50 of 1,884. The genome of <em>A. hypochondriacus</em> was found to be AT-rich with 66% AT content (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=RS8wW9S16qC10ZeBwPtMKpIdFXcJ1ty71jwgljMOVgTZ745p1lwudxkFeUmRzdgeWB947GCkvQyVXnLkB15IPL5JSno7EMP-pnMvLdomtR61S2-czTjU6C8M8Tm1LS1UF9YcJTFnzIq5A8786GsCIGXT-6xIDR8-T6eBtr1mJqugzCC8xMSX9NiD0qdzQePC2y8nQk6GYv~ZqiVpo~jPDhmCTpk22kFwLcw-ldtg5f4Ef8IJiCha~Dd9A6Qhe4IMJtpUFAcnJUpBWgMhYvTWqEzKIZZWL6g0zyk1~7gk~I6heKdMSUJh8paB1wsvwYQ5P~C8BuiHSMAVGSkz01p~lw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), which compares with the genomes of other plants.<sup><a reveal-id="DSU021C47" data-open="DSU021C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Figure <a reveal-id="DSU021F3" data-open="DSU021F3" class="link link-reveal link-table xref-fig">3</a> shows the uniform coverage of the 16 chromosomes by genomic scaffolds, which harbour 355 (86%) SNPs of the 411 linkage SNPs reported for grain amaranths.<sup><a reveal-id="DSU021C18" data-open="DSU021C18" class="link link-ref link-reveal xref-bibr">18</a></sup><div content-id="DSU021TB1" class="table-modal"><div class="table-wrap"><div id="DSU021TB1" data-id="DSU021TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of statistics of genome and transcriptome assembly</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Assembled ‘ome </th><th>Genome </th><th>Genome </th><th>Transcriptome </th></tr></thead><tbody><tr><td>Types of reads assembled </td><td>Paired end (PE) </td><td>PE and mate pair (MP) </td><td>PE </td></tr><tr><td>Read length (bases) </td><td>75 </td><td>PE: 75<br />MP: 36 </td><td>72 </td></tr><tr><td>Total number of high-quality reads sequenced </td><td>465,246,312 </td><td>PE: 465,246,312<br />MP: 403,867,096 </td><td>119,875,998 </td></tr><tr><td>Total number of bases sequenced (in high-quality reads) </td><td>34,893,473,400 </td><td>49,432,688,856 </td><td>8,631,071,856 </td></tr><tr><td>Total number of bases in the assembly (% of genome) </td><td>315,828,977 (63.2%) </td><td>645,211,960 (&gt;100%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>Number of A, T, G, and C in the assembly (% of genome) </td><td>276,891,658 (58.9%) </td><td>273,809,695 (58.3%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>G+C base content (% of the ‘ome) </td><td>91,893,918 (19.63%) </td><td>90,943,537 (19.43%) </td><td>52,037,597 (38%) </td></tr><tr><td>Total number of assembled sequences </td><td>491,569 </td><td>367,441 </td><td>57,658 </td></tr><tr><td>Longest sequence assembled (bases) </td><td>24,878 </td><td>485,353 </td><td>17,471 </td></tr><tr><td>N50/NG50 (bases) </td><td>1,884/648 </td><td>35,089/50,869 </td><td>3,279/– </td></tr><tr><td>Number of sequences above N50/NG50 </td><td>42,507/111,228 </td><td>4,897/2,826 </td><td>15,134/– </td></tr></tbody></table></div><div content-id="DSU021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Assembled ‘ome </th><th>Genome </th><th>Genome </th><th>Transcriptome </th></tr></thead><tbody><tr><td>Types of reads assembled </td><td>Paired end (PE) </td><td>PE and mate pair (MP) </td><td>PE </td></tr><tr><td>Read length (bases) </td><td>75 </td><td>PE: 75<br />MP: 36 </td><td>72 </td></tr><tr><td>Total number of high-quality reads sequenced </td><td>465,246,312 </td><td>PE: 465,246,312<br />MP: 403,867,096 </td><td>119,875,998 </td></tr><tr><td>Total number of bases sequenced (in high-quality reads) </td><td>34,893,473,400 </td><td>49,432,688,856 </td><td>8,631,071,856 </td></tr><tr><td>Total number of bases in the assembly (% of genome) </td><td>315,828,977 (63.2%) </td><td>645,211,960 (&gt;100%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>Number of A, T, G, and C in the assembly (% of genome) </td><td>276,891,658 (58.9%) </td><td>273,809,695 (58.3%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>G+C base content (% of the ‘ome) </td><td>91,893,918 (19.63%) </td><td>90,943,537 (19.43%) </td><td>52,037,597 (38%) </td></tr><tr><td>Total number of assembled sequences </td><td>491,569 </td><td>367,441 </td><td>57,658 </td></tr><tr><td>Longest sequence assembled (bases) </td><td>24,878 </td><td>485,353 </td><td>17,471 </td></tr><tr><td>N50/NG50 (bases) </td><td>1,884/648 </td><td>35,089/50,869 </td><td>3,279/– </td></tr><tr><td>Number of sequences above N50/NG50 </td><td>42,507/111,228 </td><td>4,897/2,826 </td><td>15,134/– </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="DSU021TB1" data-id="DSU021TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of statistics of genome and transcriptome assembly</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Assembled ‘ome </th><th>Genome </th><th>Genome </th><th>Transcriptome </th></tr></thead><tbody><tr><td>Types of reads assembled </td><td>Paired end (PE) </td><td>PE and mate pair (MP) </td><td>PE </td></tr><tr><td>Read length (bases) </td><td>75 </td><td>PE: 75<br />MP: 36 </td><td>72 </td></tr><tr><td>Total number of high-quality reads sequenced </td><td>465,246,312 </td><td>PE: 465,246,312<br />MP: 403,867,096 </td><td>119,875,998 </td></tr><tr><td>Total number of bases sequenced (in high-quality reads) </td><td>34,893,473,400 </td><td>49,432,688,856 </td><td>8,631,071,856 </td></tr><tr><td>Total number of bases in the assembly (% of genome) </td><td>315,828,977 (63.2%) </td><td>645,211,960 (&gt;100%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>Number of A, T, G, and C in the assembly (% of genome) </td><td>276,891,658 (58.9%) </td><td>273,809,695 (58.3%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>G+C base content (% of the ‘ome) </td><td>91,893,918 (19.63%) </td><td>90,943,537 (19.43%) </td><td>52,037,597 (38%) </td></tr><tr><td>Total number of assembled sequences </td><td>491,569 </td><td>367,441 </td><td>57,658 </td></tr><tr><td>Longest sequence assembled (bases) </td><td>24,878 </td><td>485,353 </td><td>17,471 </td></tr><tr><td>N50/NG50 (bases) </td><td>1,884/648 </td><td>35,089/50,869 </td><td>3,279/– </td></tr><tr><td>Number of sequences above N50/NG50 </td><td>42,507/111,228 </td><td>4,897/2,826 </td><td>15,134/– </td></tr></tbody></table></div><div content-id="DSU021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Assembled ‘ome </th><th>Genome </th><th>Genome </th><th>Transcriptome </th></tr></thead><tbody><tr><td>Types of reads assembled </td><td>Paired end (PE) </td><td>PE and mate pair (MP) </td><td>PE </td></tr><tr><td>Read length (bases) </td><td>75 </td><td>PE: 75<br />MP: 36 </td><td>72 </td></tr><tr><td>Total number of high-quality reads sequenced </td><td>465,246,312 </td><td>PE: 465,246,312<br />MP: 403,867,096 </td><td>119,875,998 </td></tr><tr><td>Total number of bases sequenced (in high-quality reads) </td><td>34,893,473,400 </td><td>49,432,688,856 </td><td>8,631,071,856 </td></tr><tr><td>Total number of bases in the assembly (% of genome) </td><td>315,828,977 (63.2%) </td><td>645,211,960 (&gt;100%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>Number of A, T, G, and C in the assembly (% of genome) </td><td>276,891,658 (58.9%) </td><td>273,809,695 (58.3%) </td><td>136,933,803 (29.26%) </td></tr><tr><td>G+C base content (% of the ‘ome) </td><td>91,893,918 (19.63%) </td><td>90,943,537 (19.43%) </td><td>52,037,597 (38%) </td></tr><tr><td>Total number of assembled sequences </td><td>491,569 </td><td>367,441 </td><td>57,658 </td></tr><tr><td>Longest sequence assembled (bases) </td><td>24,878 </td><td>485,353 </td><td>17,471 </td></tr><tr><td>N50/NG50 (bases) </td><td>1,884/648 </td><td>35,089/50,869 </td><td>3,279/– </td></tr><tr><td>Number of sequences above N50/NG50 </td><td>42,507/111,228 </td><td>4,897/2,826 </td><td>15,134/– </td></tr></tbody></table></div></div><div data-id="DSU021F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02103.jpeg?Expires=2147483647&amp;Signature=qLFGTqXyidigqHcaCrwwc3rMWkLSw7DwteNrDQFVypwVKYiAZ95-e9bwjJWHiGgshkJr-X3Z3r1Sc3v6Q9KIqkDkvgHnRpoybZb~fCeWI8GFK86a0q1riWmumio7zTI5VzeyGYDryuDmdzSSA7Bmf6bPzyXo~iPjrXuAVvYFi-B4F9cR7j03~mNoeEpyKfAmv0uF7ETIPbvRF142I0UGAQbK6CuS8-LkjJhlzaP2mT-vXsmFSGY4hBw8hbU5ER2FUOtHh0er-AJEyUmTLJM3jpIaXx4ZYNJ8S3t9Bsp8AGdoSurNQub5vwLxlfqK2qIiPiI7Yi8P~Qya82xcrw5W0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sixteen supercontigs of A. hypochondriacus corresponding to 16 haploid chromosomes drawn in centiMorgan scale showing the location of respective scaffolds anchored using 355 linkage SNPs out of the 411 reported. The red labels represent scaffolds containing the 355 SNPs and those in blue show the scaffolds that only map to one side of the SNPs with high confidence." path-from-xml="dsu02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196333/dsu02103.jpeg" path-from-xml="dsu02103.jpeg" target="_blank">View large</a><a section="87196333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02103.jpeg?Expires=2147483647&amp;Signature=S2GQc1HPhoqwbEzq5wWoMO6ILWG7w-TFK~GI5sOmW5~0phpIr0Tr33ygVc5hUTSe7r5Bko97ApMQNWuN2x9tG1okEx9iETfIuWxGaxM~jPiv0Vmkv17N5iwixY7~rhNkdr7T8W7j99MV5pT25~pHs9oT8FBXjSXMo04GnqUlvrOv4J3YuwKlw3-ESUenimGJFT7QKTYVO9FmkCkCbzkIaSz2Kl1n1bsGuINYUtHffQ~7dN4~u5OGGtj3HZVj~9pz6vwL2JALBtfCfKmWLpSxw2e9DcBVVh4uAjW~BiKV3yPSqBKW5r-~IfFsy~ZqR-4Yb6ETixsQkyhV~hQHcovMKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196333&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sixteen supercontigs of <em>A. hypochondriacus</em> corresponding to 16 haploid chromosomes drawn in centiMorgan scale showing the location of respective scaffolds anchored using 355 linkage SNPs out of the 411 reported. The red labels represent scaffolds containing the 355 SNPs and those in blue show the scaffolds that only map to one side of the SNPs with high confidence.</p></div></div><div content-id="DSU021F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02103.jpeg?Expires=2147483647&amp;Signature=qLFGTqXyidigqHcaCrwwc3rMWkLSw7DwteNrDQFVypwVKYiAZ95-e9bwjJWHiGgshkJr-X3Z3r1Sc3v6Q9KIqkDkvgHnRpoybZb~fCeWI8GFK86a0q1riWmumio7zTI5VzeyGYDryuDmdzSSA7Bmf6bPzyXo~iPjrXuAVvYFi-B4F9cR7j03~mNoeEpyKfAmv0uF7ETIPbvRF142I0UGAQbK6CuS8-LkjJhlzaP2mT-vXsmFSGY4hBw8hbU5ER2FUOtHh0er-AJEyUmTLJM3jpIaXx4ZYNJ8S3t9Bsp8AGdoSurNQub5vwLxlfqK2qIiPiI7Yi8P~Qya82xcrw5W0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sixteen supercontigs of A. hypochondriacus corresponding to 16 haploid chromosomes drawn in centiMorgan scale showing the location of respective scaffolds anchored using 355 linkage SNPs out of the 411 reported. The red labels represent scaffolds containing the 355 SNPs and those in blue show the scaffolds that only map to one side of the SNPs with high confidence." path-from-xml="dsu02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196333/dsu02103.jpeg" path-from-xml="dsu02103.jpeg" target="_blank">View large</a><a section="87196333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02103.jpeg?Expires=2147483647&amp;Signature=S2GQc1HPhoqwbEzq5wWoMO6ILWG7w-TFK~GI5sOmW5~0phpIr0Tr33ygVc5hUTSe7r5Bko97ApMQNWuN2x9tG1okEx9iETfIuWxGaxM~jPiv0Vmkv17N5iwixY7~rhNkdr7T8W7j99MV5pT25~pHs9oT8FBXjSXMo04GnqUlvrOv4J3YuwKlw3-ESUenimGJFT7QKTYVO9FmkCkCbzkIaSz2Kl1n1bsGuINYUtHffQ~7dN4~u5OGGtj3HZVj~9pz6vwL2JALBtfCfKmWLpSxw2e9DcBVVh4uAjW~BiKV3yPSqBKW5r-~IfFsy~ZqR-4Yb6ETixsQkyhV~hQHcovMKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196333&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sixteen supercontigs of <em>A. hypochondriacus</em> corresponding to 16 haploid chromosomes drawn in centiMorgan scale showing the location of respective scaffolds anchored using 355 linkage SNPs out of the 411 reported. The red labels represent scaffolds containing the 355 SNPs and those in blue show the scaffolds that only map to one side of the SNPs with high confidence.</p></div></div></div><p>The assembly statistics for transcriptome reads are also included in Table <a reveal-id="DSU021TB1" data-open="DSU021TB1" class="link link-reveal link-table xref-fig">1</a>. The transcriptome is also AT-rich like that of other plant transcriptomes.<sup><a reveal-id="DSU021C48" data-open="DSU021C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The total number of non-redundant bases resulting from assembly is 47 million bases without any intervening Ns, which would suggest that 10% of the <em>A. hypochondriacus</em> genome is transcribed in the tissues sequenced. Also, 74% of raw transcript reads map onto the genomic scaffolds, which is much more than what can be expected based on the ATGC content of the assembled genomic scaffolds which comprise 58% of the genome.</p><p>The genomic scaffolds reported here were also compared with a publicly available transcriptome sequence (SRX055331) with the seed accession of IC 38040/PI 480569 (India 38040) generated using 454 platform by another independent effort.<sup><a reveal-id="DSU021C14" data-open="DSU021C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Despite the fact that India 38040 has a reddish stem and drooping inflorescence (personal communication), we find significant homology between the edible white variety sequenced here and India 38040. For example, of the total 6,222,321 non-redundant bases in the assembled transcriptome from India 38040, 5,185,660 bases (83.34%) aligned to the genomic scaffolds with identities greater than 99% over a contiguous stretch greater than 100 bases.</p>                            <h3 scrollto-destination=87196336 id="87196336" class="section-title" data-legacy-id=s3b>3.2. Quality assessment and synteny of genomic scaffolds with B. vulgaris</h3>
<div class="block-child-p">The recently reported genome of <em>B. vulgaris</em> is the only reported genome of a member of the Caryophyllales plant order,<sup><a reveal-id="DSU021C13" data-open="DSU021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> which is used here to both assess the level of synteny between the two species and to assess the quality of the scaffolds reported here. All the 4,897 scaffolds above the N50 of 35,089 bases, constituting 322.6 Mb (69% including Ns in the scaffolds) of the <em>A. hypochondriacus</em> genome, find synteny for collinear blocks adding up to 85 Mb with the genome of <em>B. vulgaris</em>. As shown in Fig. <a reveal-id="DSU021F4" data-open="DSU021F4" class="link link-reveal link-table xref-fig">4</a>, among the top 100 scaffolds (greater than 161,660 bases), 60% finds unique synteny with the genome of <em>B. vulgaris</em> compared with only 4% with <em>A. thaliana</em>. Sixty-nine of the top 100 scaffolds have collinear blocks syntenic to unique loci on <em>B. vulgaris</em> chromosomes, whereas 7 scaffolds show synteny, which is split over two different chromosomes of <em>B. vulgaris</em>. Twenty-four scaffolds that find no synteny with <em>B. vulgaris</em> are not necessarily from a gene-less region, but contain predicted genes most represented in GO category under ‘other cellular processes’ and ‘other metabolic processes’. These predicted genes may be present in <em>B. vulgaris</em>, but may have diverged significantly beyond the filtering criteria used for synteny analysis. <div data-id="DSU021F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02104.jpeg?Expires=2147483647&Signature=bMRAHJXvuuQIpuMHGsLTVmLIQ6ZRGjqz7jT8AER-tTiNVJ8roTbKsnwXX0a8dlteg4M7Q~KWikPywdlcBG8xVRbW3GIyGLlsmO0Y56AVKtw0TqE2sTDSPwuHs9ssNE~0xYlxNOLS3B5A3D-148bxlCGug4KCaAtPU1acB4d-TF4saclcx5jfA7N3p2oTRJKxsYhQ~k3MH5ShcRLu~9~C0uTDVdKeav9S8e0p2YDHj74hNcfXdaxPFvDHP-gfevgCXUupdY6trXnA9Kt1zq7m71f0i7WZ-7Gal0c-iXiw8zIGz8UdlIfgJgacrA7NOBtAn04Xv-nF11khYY4yReonDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative analyses of the longest 100 scaffolds (in blocks of 20) of A. hypochondriacus with the nine chromosomes of B. vulgaris showing a high level of synteny between the two species. It is also evident that the collinear blocks in A. hypochondriacus cover individual scaffolds across their compete lengths in most cases indicating a good quality of the de novo assembly. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196337/dsu02104.jpeg" path-from-xml="dsu02104.jpeg" target="_blank">View large</a><a section="87196337" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02104.jpeg?Expires=2147483647&Signature=BJQgT~jAE-TrhsiqygBVy6pUBEN8LYesgOJNp2efVTvX4KAoNLdzvP8FpvbGyQdoFHPjfSuY1bKQaoPd1GDWSNgMqyNPbWnLdf-6ndAOA68uvbvUz2PK8wBt~Ky8gHWmh2RUdtRfKU9lZYMHghZb9Ykd8~H04-y16WbKqh~PUoFVUNMZgsdMuy7NZJbL~PI1tg2~JNjHNugcxtJo1tig5QSSOHkHiT6~v1LxI8QBd4GbSdQiGOQ0VM8K9692Xpcai8LkgHgKHOHWNJoBOlwrkGuJNy8telc4DnsGD-xzfar3eEPmZM-VIt~QVU~0vuICTZgkiY3drvOevHwcRd-khA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196337&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative analyses of the longest 100 scaffolds (in blocks of 20) of <em>A. hypochondriacus</em> with the nine chromosomes of <em>B. vulgaris</em> showing a high level of synteny between the two species. It is also evident that the collinear blocks in <em>A. hypochondriacus</em> cover individual scaffolds across their compete lengths in most cases indicating a good quality of the <em>de novo</em> assembly. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU021F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02104.jpeg?Expires=2147483647&Signature=bMRAHJXvuuQIpuMHGsLTVmLIQ6ZRGjqz7jT8AER-tTiNVJ8roTbKsnwXX0a8dlteg4M7Q~KWikPywdlcBG8xVRbW3GIyGLlsmO0Y56AVKtw0TqE2sTDSPwuHs9ssNE~0xYlxNOLS3B5A3D-148bxlCGug4KCaAtPU1acB4d-TF4saclcx5jfA7N3p2oTRJKxsYhQ~k3MH5ShcRLu~9~C0uTDVdKeav9S8e0p2YDHj74hNcfXdaxPFvDHP-gfevgCXUupdY6trXnA9Kt1zq7m71f0i7WZ-7Gal0c-iXiw8zIGz8UdlIfgJgacrA7NOBtAn04Xv-nF11khYY4yReonDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative analyses of the longest 100 scaffolds (in blocks of 20) of A. hypochondriacus with the nine chromosomes of B. vulgaris showing a high level of synteny between the two species. It is also evident that the collinear blocks in A. hypochondriacus cover individual scaffolds across their compete lengths in most cases indicating a good quality of the de novo assembly. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196337/dsu02104.jpeg" path-from-xml="dsu02104.jpeg" target="_blank">View large</a><a section="87196337" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02104.jpeg?Expires=2147483647&Signature=BJQgT~jAE-TrhsiqygBVy6pUBEN8LYesgOJNp2efVTvX4KAoNLdzvP8FpvbGyQdoFHPjfSuY1bKQaoPd1GDWSNgMqyNPbWnLdf-6ndAOA68uvbvUz2PK8wBt~Ky8gHWmh2RUdtRfKU9lZYMHghZb9Ykd8~H04-y16WbKqh~PUoFVUNMZgsdMuy7NZJbL~PI1tg2~JNjHNugcxtJo1tig5QSSOHkHiT6~v1LxI8QBd4GbSdQiGOQ0VM8K9692Xpcai8LkgHgKHOHWNJoBOlwrkGuJNy8telc4DnsGD-xzfar3eEPmZM-VIt~QVU~0vuICTZgkiY3drvOevHwcRd-khA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196337&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative analyses of the longest 100 scaffolds (in blocks of 20) of <em>A. hypochondriacus</em> with the nine chromosomes of <em>B. vulgaris</em> showing a high level of synteny between the two species. It is also evident that the collinear blocks in <em>A. hypochondriacus</em> cover individual scaffolds across their compete lengths in most cases indicating a good quality of the <em>de novo</em> assembly. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div></div>                            <h3 scrollto-destination=87196338 id="87196338" class="section-title" data-legacy-id=s3c>3.3. Comparative coverage by homology
to A. thaliana proteome</h3>
<div class="block-child-p">An amino acid level comparison of <em>A. thaliana</em> proteome<sup><a reveal-id="DSU021C49" data-open="DSU021C49" class="link link-ref link-reveal xref-bibr">49</a></sup> against genomic scaffolds of <em>A. hypochondriacus</em> shows that 60.11% of <em>A. thaliana</em> proteins find orthology within the genomic scaffolds using the filter mentioned in the Materials and Methods section. Similar efforts with the scaffolds of two other species, <em>V. vinifera</em><sup><a reveal-id="DSU021C50" data-open="DSU021C50" class="link link-ref link-reveal xref-bibr">50</a></sup> and <em>S. lycopersicum</em>,<sup><a reveal-id="DSU021C51" data-open="DSU021C51" class="link link-ref link-reveal xref-bibr">51</a></sup> finds comparable, 17,311 (63%) and 16,538 (60.32%), respectively, orthology to the <em>A. thaliana</em> proteome. Also, as shown in Fig. <a reveal-id="DSU021F5" data-open="DSU021F5" class="link link-reveal link-table xref-fig">5</a>, the orthology of 14,590 <em>A. thaliana</em> genes common to the genomic scaffolds of <em>V. vinifera</em>, <em>S. lycopersicum</em>, and <em>A. hypochondriacus</em> suggests that functions of these genes are common to all dicots. A GO annotation of the <em>A. thaliana</em> orthologs in <em>V. vinifera</em>, <em>S. lycopersicum</em>, and <em>A. hypochondriacus</em> scaffolds shows that more than 80% of the genes annotated under each GO category in <em>A. hypochondriacus</em> has been deciphered (Fig. <a reveal-id="DSU021F6" data-open="DSU021F6" class="link link-reveal link-table xref-fig">6</a>), and that the coverage of <em>A. hypochondriacus</em> genes within each category is comparable to the other two reported genomes of <em>V. vinifera</em> and <em>S. lycopersicum</em>. <div data-id="DSU021F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02105.jpeg?Expires=2147483647&amp;Signature=VWft3Z0zOPimhyRy~5up4HOyl8tjtYSpspBQ3FPev4-I7IlyarKCZdhIfyUGBqeqiiHY8dN3iaeB9puPTMe1sasIs59XvKxiH7kfHud-tMOKRSfMn5OkCpe9uHg76wecH6f1YIGp4DVtAgT7fM-Zd4PT9zfPcyjdYqaivLcD0n7On79bhMtBio-3Wb8udCGLDr~dkkOFUjsaNBmjypFGJVl~lnIcsW3xnjt~0r3ps~aVFcjS4zAyYihlVeEaH6rFsE0EATL~VJI0QF3vHhSBUUkA4m9Eyg2eCFGNzzUL00QYczBXtQAJP7MJKiAjL-kwyizMJJjWPfA10nSQ-LPkxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative proteome analysis of A. hypochondriacus, V. vinifera and S. lycopersicum. The pie chart shows the source of evidence for proteins from the assembled transcriptome and genome. 17,770 proteins find evidence in both homology and presence in the transcriptome, 3,880 are ab initio predicted with presence in the transcriptome, and 3,179 are those with only evidence from homology to known plant proteins. The Venn diagram compares the total numbers of A. thaliana genes that are orthologous to the respective genomic scaffolds of A. hypochondriacus, V. vinifera and S. lycopersicum with 14,590 proteins common to all the three species representing three major clades under dicots. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196339/dsu02105.jpeg" path-from-xml="dsu02105.jpeg" target="_blank">View large</a><a section="87196339" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02105.jpeg?Expires=2147483647&amp;Signature=AR-XzffFYNVL~wrWXp1JiSQzqiAfq8Brimcra2yJ4OxV-tsGpnLG6rJFhaSFNI~gVGeuiUFqZNMV2vP4trXxkDKvBEnIDXncqmr99oFjNGoLamwe5nghVhdkLuU7uGNyocatOtU1WzTx3cUo9bAhE7trmVBCbHbXQIvRKIjW~1sdlPC7ur8UAcKviOmDc-AEkUN5vosfnLM8GWhMVSmnsrEc8MHVyfcdTGgzwcPKLx1mEsTeRSOERY14mXSoRQDsykSYN8MljSjrYzdBlLnRsqsu701iy73GOGsecSoNGGpoc2LFyYEpI26QhRxmJSmA~CK77RltKiRIYdYKRzw3Hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196339&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative proteome analysis of <em>A. hypochondriacus</em>, <em>V. vinifera</em> and <em>S. lycopersicum</em>. The pie chart shows the source of evidence for proteins from the assembled transcriptome and genome. 17,770 proteins find evidence in both homology and presence in the transcriptome, 3,880 are <em>ab initio</em> predicted with presence in the transcriptome, and 3,179 are those with only evidence from homology to known plant proteins. The Venn diagram compares the total numbers of <em>A. thaliana</em> genes that are orthologous to the respective genomic scaffolds of <em>A. hypochondriacus</em>, <em>V. vinifera</em> and <em>S. lycopersicum</em> with 14,590 proteins common to all the three species representing three major clades under dicots. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU021F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02105.jpeg?Expires=2147483647&amp;Signature=VWft3Z0zOPimhyRy~5up4HOyl8tjtYSpspBQ3FPev4-I7IlyarKCZdhIfyUGBqeqiiHY8dN3iaeB9puPTMe1sasIs59XvKxiH7kfHud-tMOKRSfMn5OkCpe9uHg76wecH6f1YIGp4DVtAgT7fM-Zd4PT9zfPcyjdYqaivLcD0n7On79bhMtBio-3Wb8udCGLDr~dkkOFUjsaNBmjypFGJVl~lnIcsW3xnjt~0r3ps~aVFcjS4zAyYihlVeEaH6rFsE0EATL~VJI0QF3vHhSBUUkA4m9Eyg2eCFGNzzUL00QYczBXtQAJP7MJKiAjL-kwyizMJJjWPfA10nSQ-LPkxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative proteome analysis of A. hypochondriacus, V. vinifera and S. lycopersicum. The pie chart shows the source of evidence for proteins from the assembled transcriptome and genome. 17,770 proteins find evidence in both homology and presence in the transcriptome, 3,880 are ab initio predicted with presence in the transcriptome, and 3,179 are those with only evidence from homology to known plant proteins. The Venn diagram compares the total numbers of A. thaliana genes that are orthologous to the respective genomic scaffolds of A. hypochondriacus, V. vinifera and S. lycopersicum with 14,590 proteins common to all the three species representing three major clades under dicots. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196339/dsu02105.jpeg" path-from-xml="dsu02105.jpeg" target="_blank">View large</a><a section="87196339" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02105.jpeg?Expires=2147483647&amp;Signature=AR-XzffFYNVL~wrWXp1JiSQzqiAfq8Brimcra2yJ4OxV-tsGpnLG6rJFhaSFNI~gVGeuiUFqZNMV2vP4trXxkDKvBEnIDXncqmr99oFjNGoLamwe5nghVhdkLuU7uGNyocatOtU1WzTx3cUo9bAhE7trmVBCbHbXQIvRKIjW~1sdlPC7ur8UAcKviOmDc-AEkUN5vosfnLM8GWhMVSmnsrEc8MHVyfcdTGgzwcPKLx1mEsTeRSOERY14mXSoRQDsykSYN8MljSjrYzdBlLnRsqsu701iy73GOGsecSoNGGpoc2LFyYEpI26QhRxmJSmA~CK77RltKiRIYdYKRzw3Hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196339&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative proteome analysis of <em>A. hypochondriacus</em>, <em>V. vinifera</em> and <em>S. lycopersicum</em>. The pie chart shows the source of evidence for proteins from the assembled transcriptome and genome. 17,770 proteins find evidence in both homology and presence in the transcriptome, 3,880 are <em>ab initio</em> predicted with presence in the transcriptome, and 3,179 are those with only evidence from homology to known plant proteins. The Venn diagram compares the total numbers of <em>A. thaliana</em> genes that are orthologous to the respective genomic scaffolds of <em>A. hypochondriacus</em>, <em>V. vinifera</em> and <em>S. lycopersicum</em> with 14,590 proteins common to all the three species representing three major clades under dicots. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div data-id="DSU021F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02106.jpeg?Expires=2147483647&amp;Signature=31dwKf~xuKlBcg6y5Rg8JH73uCZJfqy-uOipXi3g4l1DVaXRt64WGApPlNM3FMP8RLpTFYUiRCZt2mrzR7iyM8lyD~PDdjCXGwd-8j-Yy885xImzsDU1dV8AE2~GwZOhOHh3hrA1BuBaUfv1Bl8obA~q8dYQCkC8H46uwkxZhfXtc4yuVVKZPECl2MYZocTsEaXSqWsV-nP3LxLgpcanyptVFzVTmxJVVsMKHWUuxthJxUhVCer~8vAy71F6ki3oV0A~jvHzKcr5PSUNeedVPYNKzEQQCncQROVrjf~0NOPKFtNij~irYJiuR056hLfVsChBdjE7wl9tO~s06R-Scw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative GO annotation of three species including A. hypochondriacus, V. vinifera and S. lycopersicum. The percentage of genes deciphered for A. hypochondriacus, S. lycopersicum and V. vinifera within each category of GO as annotated by comparison with A. thaliana based on orthology and further normalized for the percentage of genes in A. thaliana shows that annotation of the draft genome is comparable to that of the published genomes. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196339/dsu02106.jpeg" path-from-xml="dsu02106.jpeg" target="_blank">View large</a><a section="87196339" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02106.jpeg?Expires=2147483647&amp;Signature=F7pQs5MDGkx5Qqc9Q6v~5OW3uHY~AfEQO9kFEDDywGeprTr7LQ~jfPUJ9K0xo4zbqH5rCi2RTXIOieYWkIJTi0rj6ALJkx33sHl4k2X9P12byQmql89HsHdeTAr5S~DVdzLIIycuP3oECVboMSH-8qOaKjpLkXsaQusbl-T-NQjiL7clBjfZI2tt8BxOm8-UmQDCxB~TNWkllJx9jwz0eUZQ0DybyoH4f4Szr6PKL0qys5QU2TU1X1YmlpwAFJ03r9nUTmsiqf-3DhCePZl7zoCVCPFEUtGK2QvOJsjI~StsdDESHw2Bf3~sMPx5JOx94XTv8gHY0B3-~jfzvW4Gug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196339&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative GO annotation of three species including <em>A. hypochondriacus</em>, <em>V. vinifera</em> and <em>S. lycopersicum</em>. The percentage of genes deciphered for <em>A. hypochondriacus</em>, <em>S. lycopersicum</em> and <em>V. vinifera</em> within each category of GO as annotated by comparison with <em>A. thaliana</em> based on orthology and further normalized for the percentage of genes in <em>A. thaliana</em> shows that annotation of the draft genome is comparable to that of the published genomes. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU021F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02106.jpeg?Expires=2147483647&amp;Signature=31dwKf~xuKlBcg6y5Rg8JH73uCZJfqy-uOipXi3g4l1DVaXRt64WGApPlNM3FMP8RLpTFYUiRCZt2mrzR7iyM8lyD~PDdjCXGwd-8j-Yy885xImzsDU1dV8AE2~GwZOhOHh3hrA1BuBaUfv1Bl8obA~q8dYQCkC8H46uwkxZhfXtc4yuVVKZPECl2MYZocTsEaXSqWsV-nP3LxLgpcanyptVFzVTmxJVVsMKHWUuxthJxUhVCer~8vAy71F6ki3oV0A~jvHzKcr5PSUNeedVPYNKzEQQCncQROVrjf~0NOPKFtNij~irYJiuR056hLfVsChBdjE7wl9tO~s06R-Scw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative GO annotation of three species including A. hypochondriacus, V. vinifera and S. lycopersicum. The percentage of genes deciphered for A. hypochondriacus, S. lycopersicum and V. vinifera within each category of GO as annotated by comparison with A. thaliana based on orthology and further normalized for the percentage of genes in A. thaliana shows that annotation of the draft genome is comparable to that of the published genomes. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196339/dsu02106.jpeg" path-from-xml="dsu02106.jpeg" target="_blank">View large</a><a section="87196339" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02106.jpeg?Expires=2147483647&amp;Signature=F7pQs5MDGkx5Qqc9Q6v~5OW3uHY~AfEQO9kFEDDywGeprTr7LQ~jfPUJ9K0xo4zbqH5rCi2RTXIOieYWkIJTi0rj6ALJkx33sHl4k2X9P12byQmql89HsHdeTAr5S~DVdzLIIycuP3oECVboMSH-8qOaKjpLkXsaQusbl-T-NQjiL7clBjfZI2tt8BxOm8-UmQDCxB~TNWkllJx9jwz0eUZQ0DybyoH4f4Szr6PKL0qys5QU2TU1X1YmlpwAFJ03r9nUTmsiqf-3DhCePZl7zoCVCPFEUtGK2QvOJsjI~StsdDESHw2Bf3~sMPx5JOx94XTv8gHY0B3-~jfzvW4Gug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196339&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative GO annotation of three species including <em>A. hypochondriacus</em>, <em>V. vinifera</em> and <em>S. lycopersicum</em>. The percentage of genes deciphered for <em>A. hypochondriacus</em>, <em>S. lycopersicum</em> and <em>V. vinifera</em> within each category of GO as annotated by comparison with <em>A. thaliana</em> based on orthology and further normalized for the percentage of genes in <em>A. thaliana</em> shows that annotation of the draft genome is comparable to that of the published genomes. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div></div>                            <h3 scrollto-destination=87196340 id="87196340" class="section-title" data-legacy-id=s3d>3.4. Genomic features of A. hypochondriacus</h3>
<p>The mRNA reads from transcriptome sequencing efforts were assembled into transcripts with an N50 of 3,279 containing 15,179 transcripts above the median. Both extrinsic and <em>ab initio</em> methods were chosen for predicting the number of genes coded by <em>A. hypochondriacus</em> from both 57,658 assembled transcripts and 367,441 genomic scaffolds. For extrinsic prediction, we have used plant proteome databases including UniProt, HTC, and plant mRNA at plantGDB. For <em>ab initio</em> prediction of genes from the genomic scaffolds, we have used gene prediction tools including GENSCAN with 36,487, Augustus with 28,662, and GeneMark with 63,044 predicted genes. From the assembled transcriptome of <em>A. hypochondriacus</em>, the total number of potential proteins is estimated at 21,650 genes with either evidence of homology in the plant proteome database and/or evidence of gene prediction from the genomic scaffolds. An additional 3,179 potential proteins were added from genomic scaffolds, which were missed by gene prediction methods and not represented in the transcriptome, but show homology to the plant proteome database, thus making the total number of estimated proteins coded by <em>A. hypochondriacus</em> genome 24,829. This compares with the 27,421 proteins predicted for <em>B. vulgaris</em>, the other sequenced member of Caryophyllales.<sup><a reveal-id="DSU021C13" data-open="DSU021C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>Mapping of raw genomic reads onto the assembled scaffolds revealed ∼1 SNP per 1,000 bases based on 282,692 SNPs out of 260 Mb base positions scanned. This indicates that the genome of <em>A. hypochondriacus</em> is relatively more homozygous than other sequenced plants. For example, the frequency of SNPs in <em>Malus domestica</em>, <em>O. sativa</em>, <em>Z. mays</em>, <em>V. vinifera</em>, and <em>Fragaria vesca</em> is 1 in 288 bases, 1 in 268 bases, 1 in 124 bases, 1 in 100 bases, and 1 in 15 bases, respectively.<sup><a reveal-id="DSU021C52" data-open="DSU021C52" class="link link-ref link-reveal xref-bibr">52</a>–<a reveal-id="DSU021C56" data-open="DSU021C56" class="link link-ref link-reveal xref-bibr">56</a></sup> The <em>k</em>-mer spectrum of the genome (not shown) also gives a unimodal curve, supporting the high level of homozygosity within the assembled part of the genome.<sup><a reveal-id="DSU021C57" data-open="DSU021C57" class="link link-ref link-reveal xref-bibr">57</a></sup></p><p>Characterizing the repeat elements in the <em>A. hypochondriacus</em> genome is challenged by the fact that repeats from species under the plant order Caryophyllales is missing in major repeat databases such as RepBase. Masking the draft genomes of a number of species including an unpublished draft of <em>B. vulgaris</em> using <em>A. thaliana</em> repeats as templates reveals that the <em>A. hypochondriacus</em> genome contains a relatively low percentage of repeat elements (Table <a reveal-id="DSU021TB2" data-open="DSU021TB2" class="link link-reveal link-table xref-fig">2</a>). Also, masking the draft genomes of several species using the predicted repeats from the respective genomes using RepeatScout reveals 7.47% of repeats in the <em>A. hypochondriacus</em> genome, compared with 69.86% in <em>B. vulgaris</em> (USDA version, unpublished), 68.34% in <em>S. lycopersicum</em>, 55.02% in <em>V. vinifera</em>, 18.43% in <em>A. thaliana</em>, 68.67% in <em>S. bicolor</em>, and 47.69% in <em>O. sativa</em> (Table <a reveal-id="DSU021TB2" data-open="DSU021TB2" class="link link-reveal link-table xref-fig">2</a>). A more rigorous prediction of repeats using RepeatModeler masked 13.76% of the <em>A. hypochondriacus</em> genome, still far below the 63% for <em>B. vulgaris</em> using RepeatModeler reported recently.<sup><a reveal-id="DSU021C13" data-open="DSU021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Based on the number of proteins predicted for the genome and the success achieved in splicing full-length cDNA for a majority of genes from select biochemical pathways, we rule out the possibility that the low percentage of repeats in the <em>A. hypochondriacus</em> genome could stem from a fragmented assembly. <div content-id="DSU021TB2" class="table-modal"><div class="table-wrap"><div id="DSU021TB2" data-id="DSU021TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Comparative repeat element analysis using both <em>A. thaliana</em> repeats and predicted repeats</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Ah </th><th>Bv </th><th>Sla </th><th>Sl </th><th>Vv </th><th>At </th><th>Sb </th><th>Os </th></tr><tr><th>Genome size (Mb) </th><th>466 </th><th>760 </th><th>2,800 </th><th>950 </th><th>475 </th><th>157 </th><th>698 </th><th>430 </th></tr></thead><tbody><tr><td>LTR </td><td>0.69% </td><td>4.60% </td><td>4.41% </td><td>6.68% </td><td>4.87% </td><td>6.66% </td><td>4.49% </td><td>2.08% </td></tr><tr><td> Gypsy </td><td>0.23% </td><td>2.07% </td><td>3.50% </td><td>4.28% </td><td>3.38% </td><td>5.15% </td><td>3.36% </td><td>1.35% </td></tr><tr><td> Copia </td><td>0.45% </td><td>2.51% </td><td>0.88% </td><td>2.40% </td><td>1.49% </td><td>1.44% </td><td>1.11% </td><td>0.71% </td></tr><tr><td>LINE </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.01% </td><td>0.04% </td><td>0.03% </td></tr><tr><td> L1 </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.00% </td><td>0.04% </td><td>0.03% </td></tr><tr><td>Total genome masked using Repbase with <em>A. thaliana</em> as the template </td><td>1.70% </td><td>6.25% </td><td>7.18% </td><td>7.14% </td><td>5.30% </td><td>14.90% </td><td>4.95% </td><td>2.74% </td></tr><tr><td>Total genome masked using RepeatScout predicted repeats from respective genomes </td><td>7.47 </td><td>69.86 </td><td>NA </td><td>68.34 </td><td>55.02 </td><td>18.43 </td><td>68.67 </td><td>47.69 </td></tr></tbody></table></div><div content-id="DSU021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Ah </th><th>Bv </th><th>Sla </th><th>Sl </th><th>Vv </th><th>At </th><th>Sb </th><th>Os </th></tr><tr><th>Genome size (Mb) </th><th>466 </th><th>760 </th><th>2,800 </th><th>950 </th><th>475 </th><th>157 </th><th>698 </th><th>430 </th></tr></thead><tbody><tr><td>LTR </td><td>0.69% </td><td>4.60% </td><td>4.41% </td><td>6.68% </td><td>4.87% </td><td>6.66% </td><td>4.49% </td><td>2.08% </td></tr><tr><td> Gypsy </td><td>0.23% </td><td>2.07% </td><td>3.50% </td><td>4.28% </td><td>3.38% </td><td>5.15% </td><td>3.36% </td><td>1.35% </td></tr><tr><td> Copia </td><td>0.45% </td><td>2.51% </td><td>0.88% </td><td>2.40% </td><td>1.49% </td><td>1.44% </td><td>1.11% </td><td>0.71% </td></tr><tr><td>LINE </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.01% </td><td>0.04% </td><td>0.03% </td></tr><tr><td> L1 </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.00% </td><td>0.04% </td><td>0.03% </td></tr><tr><td>Total genome masked using Repbase with <em>A. thaliana</em> as the template </td><td>1.70% </td><td>6.25% </td><td>7.18% </td><td>7.14% </td><td>5.30% </td><td>14.90% </td><td>4.95% </td><td>2.74% </td></tr><tr><td>Total genome masked using RepeatScout predicted repeats from respective genomes </td><td>7.47 </td><td>69.86 </td><td>NA </td><td>68.34 </td><td>55.02 </td><td>18.43 </td><td>68.67 </td><td>47.69 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>The species used for comparison are—Ah: <em>A. hypochondriacus</em>; Bv: <em>B. vulgaris</em> (USDA version); Sla: <em>Silene latifolia</em>;<sup><a reveal-id="DSU021C58" data-open="DSU021C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Sl: <em>S. lycopersicum</em>; Vv: <em>V. vinifera</em>; At: <em>A. thaliana</em>; Sb: <em>S. bicolor</em>; Os: <em>O. sativa.</em></p></div></div></div></div></div><div class="table-wrap"><div id="DSU021TB2" data-id="DSU021TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Comparative repeat element analysis using both <em>A. thaliana</em> repeats and predicted repeats</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Ah </th><th>Bv </th><th>Sla </th><th>Sl </th><th>Vv </th><th>At </th><th>Sb </th><th>Os </th></tr><tr><th>Genome size (Mb) </th><th>466 </th><th>760 </th><th>2,800 </th><th>950 </th><th>475 </th><th>157 </th><th>698 </th><th>430 </th></tr></thead><tbody><tr><td>LTR </td><td>0.69% </td><td>4.60% </td><td>4.41% </td><td>6.68% </td><td>4.87% </td><td>6.66% </td><td>4.49% </td><td>2.08% </td></tr><tr><td> Gypsy </td><td>0.23% </td><td>2.07% </td><td>3.50% </td><td>4.28% </td><td>3.38% </td><td>5.15% </td><td>3.36% </td><td>1.35% </td></tr><tr><td> Copia </td><td>0.45% </td><td>2.51% </td><td>0.88% </td><td>2.40% </td><td>1.49% </td><td>1.44% </td><td>1.11% </td><td>0.71% </td></tr><tr><td>LINE </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.01% </td><td>0.04% </td><td>0.03% </td></tr><tr><td> L1 </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.00% </td><td>0.04% </td><td>0.03% </td></tr><tr><td>Total genome masked using Repbase with <em>A. thaliana</em> as the template </td><td>1.70% </td><td>6.25% </td><td>7.18% </td><td>7.14% </td><td>5.30% </td><td>14.90% </td><td>4.95% </td><td>2.74% </td></tr><tr><td>Total genome masked using RepeatScout predicted repeats from respective genomes </td><td>7.47 </td><td>69.86 </td><td>NA </td><td>68.34 </td><td>55.02 </td><td>18.43 </td><td>68.67 </td><td>47.69 </td></tr></tbody></table></div><div content-id="DSU021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Ah </th><th>Bv </th><th>Sla </th><th>Sl </th><th>Vv </th><th>At </th><th>Sb </th><th>Os </th></tr><tr><th>Genome size (Mb) </th><th>466 </th><th>760 </th><th>2,800 </th><th>950 </th><th>475 </th><th>157 </th><th>698 </th><th>430 </th></tr></thead><tbody><tr><td>LTR </td><td>0.69% </td><td>4.60% </td><td>4.41% </td><td>6.68% </td><td>4.87% </td><td>6.66% </td><td>4.49% </td><td>2.08% </td></tr><tr><td> Gypsy </td><td>0.23% </td><td>2.07% </td><td>3.50% </td><td>4.28% </td><td>3.38% </td><td>5.15% </td><td>3.36% </td><td>1.35% </td></tr><tr><td> Copia </td><td>0.45% </td><td>2.51% </td><td>0.88% </td><td>2.40% </td><td>1.49% </td><td>1.44% </td><td>1.11% </td><td>0.71% </td></tr><tr><td>LINE </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.01% </td><td>0.04% </td><td>0.03% </td></tr><tr><td> L1 </td><td>0.07% </td><td>0.50% </td><td>0.00% </td><td>0.03% </td><td>0.11% </td><td>1.00% </td><td>0.04% </td><td>0.03% </td></tr><tr><td>Total genome masked using Repbase with <em>A. thaliana</em> as the template </td><td>1.70% </td><td>6.25% </td><td>7.18% </td><td>7.14% </td><td>5.30% </td><td>14.90% </td><td>4.95% </td><td>2.74% </td></tr><tr><td>Total genome masked using RepeatScout predicted repeats from respective genomes </td><td>7.47 </td><td>69.86 </td><td>NA </td><td>68.34 </td><td>55.02 </td><td>18.43 </td><td>68.67 </td><td>47.69 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>The species used for comparison are—Ah: <em>A. hypochondriacus</em>; Bv: <em>B. vulgaris</em> (USDA version); Sla: <em>Silene latifolia</em>;<sup><a reveal-id="DSU021C58" data-open="DSU021C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Sl: <em>S. lycopersicum</em>; Vv: <em>V. vinifera</em>; At: <em>A. thaliana</em>; Sb: <em>S. bicolor</em>; Os: <em>O. sativa.</em></p></div></div></div></div></p><div class="block-child-p">Estimation of 24,829 proteins provided an opportunity to look for any sign of recent whole genome duplication using a method reported by Axelsen <em>et al.</em><sup><a reveal-id="DSU021C59" data-open="DSU021C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Figure <a reveal-id="DSU021F7" data-open="DSU021F7" class="link link-reveal link-table xref-fig">7</a> shows the comparison of proteomes of various species to find levels of divergence among paralogs. The peak near 90% for <em>Z. mays</em>, <em>A. thaliana</em>, and <em>G. max</em> confirms the recent whole genome duplication events reported for these species.<sup><a reveal-id="DSU021C60" data-open="DSU021C60" class="link link-ref link-reveal xref-bibr">60</a></sup><em>Amaranthus hypochondriacus</em> and <em>B. vulgaris</em>, both species classified under the order Caryophyllales, lack the peak near 90% similar to <em>V. vinifera</em>, suggesting no recent whole genome duplication event in <em>A. hypochondriacus</em> and <em>B. vulgaris</em>. The peak near 30 for all dicot species represent a paleohexaploidy event common to all dicots,<sup><a reveal-id="DSU021C50" data-open="DSU021C50" class="link link-ref link-reveal xref-bibr">50</a></sup> including the two species of Caryophyllales studied here. Based on this analysis, we have redrawn the place for <em>A. hypochondriacus</em> in Fig. <a reveal-id="DSU021F8" data-open="DSU021F8" class="link link-reveal link-table xref-fig">8</a>, which is reproduced with permission from the authors.<sup><a reveal-id="DSU021C61" data-open="DSU021C61" class="link link-ref link-reveal xref-bibr">61</a></sup><div data-id="DSU021F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02107.jpeg?Expires=2147483647&amp;Signature=YHfiRi623Wzk9Ud-0zWt86IyoeYqCewLsX7pE1KiHjUXjyFIl5EtmBVUfsyjTxaoIuGLKxs1uwjOqXZh6V1tdE6mUz9yTDdS0zacsK-slTTyGb6wLAFlSOT0wqDO7xwWdoYWN06er~AasbLmVbMDvJuOYqC9wD4RgY3m5tNimCEAO5IvLtu~7k~jIPcOmqjx8PqQZWYc9CobngdfMPVlJadKYol-UfYNCrr3OJoBdenuVD1VeixJzpr-tw1iIAyy8QOe57bzkpRhBEwVeSb3GH1KetBq77dNwRf-5cVmBsLdrqpdOJBkEclKZrOBvyTmdr2mJcm5Dzz0nuiPFjCBDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genome duplication study in A. thaliana, G. max, V. vinifera, S. lycopersicum, A. hypochondriacus, B. vulgaris, O. sativa, S. bicolor, and Z. mays by comparison of the distribution of percentage identities among paralogous pairs from the respective proteomes. The GENSCAN-predicted proteome of S. lycopersicum is labelled as S. lycopersicum (G) in the figure. The Kernel density plot of the density function of the number of paralogous pairs against the percentage identities for the diverse sequenced plant species shows the first peak at 20–40% identity corresponding to the paleaohexaploidy event and the sharp peak at 90–100% identity representing a recent whole genome duplication event. From the plot, it is evident that A. hypochondriacus, like B. vulgaris, did not undergo any recent whole genome duplication." path-from-xml="dsu02107.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196344/dsu02107.jpeg" path-from-xml="dsu02107.jpeg" target="_blank">View large</a><a section="87196344" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02107.jpeg?Expires=2147483647&amp;Signature=eDi8QiQi7Nv9H-lWOUQHytm7et6EJOrn1WPtygh-Sz-Q32Ed2dISxMJoyYVb7W0onGKOLll6T2ZsXL-9Q-ELSjgOaPR5sRiNQbT6g4JJ74hKGOHipUUqzgiG--h3z0QwOyNdlamiqoCMxVt-npkwxpwElqv98zoidXZgI1G4cgkfWqCcceXHvd~Lw3DKhu6VzrLY5vl0DXQCnLBUk-AjowRB9FfZSGjYHEswCoQfMn~36xkQh09Lpy50HS9ivSKUTEexddceRguY~MidAB9bpY0bfDD4jo5ItjNs598nKrRLgzySM9CcTpNJ8uxVMsAKNe2czJLEMIGEtDy9XorQjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196344&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02107.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>A genome</em> duplication study in <em>A. thaliana</em>, <em>G. max</em>, <em>V. vinifera</em>, <em>S. lycopersicum</em>, <em>A. hypochondriacus</em>, <em>B. vulgaris</em>, <em>O. sativa</em>, <em>S. bicolor</em>, and <em>Z. mays</em> by comparison of the distribution of percentage identities among paralogous pairs from the respective proteomes. The GENSCAN-predicted proteome of <em>S. lycopersicum</em> is labelled as <em>S. lycopersicum</em> (G) in the figure. The Kernel density plot of the density function of the number of paralogous pairs against the percentage identities for the diverse sequenced plant species shows the first peak at 20–40% identity corresponding to the paleaohexaploidy event and the sharp peak at 90–100% identity representing a recent whole genome duplication event. From the plot, it is evident that <em>A. hypochondriacus</em>, like <em>B. vulgaris</em>, did not undergo any recent whole genome duplication.</p></div></div><div content-id="DSU021F7" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02107.jpeg?Expires=2147483647&amp;Signature=YHfiRi623Wzk9Ud-0zWt86IyoeYqCewLsX7pE1KiHjUXjyFIl5EtmBVUfsyjTxaoIuGLKxs1uwjOqXZh6V1tdE6mUz9yTDdS0zacsK-slTTyGb6wLAFlSOT0wqDO7xwWdoYWN06er~AasbLmVbMDvJuOYqC9wD4RgY3m5tNimCEAO5IvLtu~7k~jIPcOmqjx8PqQZWYc9CobngdfMPVlJadKYol-UfYNCrr3OJoBdenuVD1VeixJzpr-tw1iIAyy8QOe57bzkpRhBEwVeSb3GH1KetBq77dNwRf-5cVmBsLdrqpdOJBkEclKZrOBvyTmdr2mJcm5Dzz0nuiPFjCBDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A genome duplication study in A. thaliana, G. max, V. vinifera, S. lycopersicum, A. hypochondriacus, B. vulgaris, O. sativa, S. bicolor, and Z. mays by comparison of the distribution of percentage identities among paralogous pairs from the respective proteomes. The GENSCAN-predicted proteome of S. lycopersicum is labelled as S. lycopersicum (G) in the figure. The Kernel density plot of the density function of the number of paralogous pairs against the percentage identities for the diverse sequenced plant species shows the first peak at 20–40% identity corresponding to the paleaohexaploidy event and the sharp peak at 90–100% identity representing a recent whole genome duplication event. From the plot, it is evident that A. hypochondriacus, like B. vulgaris, did not undergo any recent whole genome duplication." path-from-xml="dsu02107.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196344/dsu02107.jpeg" path-from-xml="dsu02107.jpeg" target="_blank">View large</a><a section="87196344" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02107.jpeg?Expires=2147483647&amp;Signature=eDi8QiQi7Nv9H-lWOUQHytm7et6EJOrn1WPtygh-Sz-Q32Ed2dISxMJoyYVb7W0onGKOLll6T2ZsXL-9Q-ELSjgOaPR5sRiNQbT6g4JJ74hKGOHipUUqzgiG--h3z0QwOyNdlamiqoCMxVt-npkwxpwElqv98zoidXZgI1G4cgkfWqCcceXHvd~Lw3DKhu6VzrLY5vl0DXQCnLBUk-AjowRB9FfZSGjYHEswCoQfMn~36xkQh09Lpy50HS9ivSKUTEexddceRguY~MidAB9bpY0bfDD4jo5ItjNs598nKrRLgzySM9CcTpNJ8uxVMsAKNe2czJLEMIGEtDy9XorQjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196344&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02107.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>A genome</em> duplication study in <em>A. thaliana</em>, <em>G. max</em>, <em>V. vinifera</em>, <em>S. lycopersicum</em>, <em>A. hypochondriacus</em>, <em>B. vulgaris</em>, <em>O. sativa</em>, <em>S. bicolor</em>, and <em>Z. mays</em> by comparison of the distribution of percentage identities among paralogous pairs from the respective proteomes. The GENSCAN-predicted proteome of <em>S. lycopersicum</em> is labelled as <em>S. lycopersicum</em> (G) in the figure. The Kernel density plot of the density function of the number of paralogous pairs against the percentage identities for the diverse sequenced plant species shows the first peak at 20–40% identity corresponding to the paleaohexaploidy event and the sharp peak at 90–100% identity representing a recent whole genome duplication event. From the plot, it is evident that <em>A. hypochondriacus</em>, like <em>B. vulgaris</em>, did not undergo any recent whole genome duplication.</p></div></div><div data-id="DSU021F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02108.jpeg?Expires=2147483647&amp;Signature=LN9a2OlbBybiEqRhfRPpWT85uGdbssczlHgQqMKvRTz9zJSS6-3AKrzx07V7KxQ3e-c~p~sr5KWhPrBWnloh4bEM3YGGA4W4RJZJZbXZZ~AcvgGh9v2Wx11vRX71TgXyknw3zDpXCbxz4EjQfvFl6MjswZ1XItaE~dLW51krho1Y0dSpsKewQ3PZMtFtoEnsslLdIrtINZJajctzauqDTh3jE3FV4mPDwfsqUdxJNoIxDQfKaNGh6~YjrorXYPpqWvKuFH~bzmeYLxk189Sp5Ny2281vKTej7UlAn25iEvGevBQE3oZPx3V2SzlQAm2j9qXtos0XrxfIZnsJ0wgKYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rooted tree comparing the relative whole genome duplication events in many plant species including A. hypochondriacus. The tree is taken from PNAS journal with permission from the authorand modified to include A. hypochondriacus, creating a branch for Caryophyllales, using the phylogenetic tree generated in-house using rbcL gene (not shown) from 75 species across all plant orders. The predicted and reported ages of the duplication events are retained from the original paper. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02108.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196344/dsu02108.jpeg" path-from-xml="dsu02108.jpeg" target="_blank">View large</a><a section="87196344" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02108.jpeg?Expires=2147483647&amp;Signature=lFHCbIfHoiPnZn0ro1k2abNLpIWXYhYEknSIXeiXjosWdAaJzFWZyIxN~8vqmi7GLE5V1TyPmQjMKY7uw9IsWzVHh0pO8c8tpCLuk9p3M0LxebAUL6jxxW-heXEWZ~AGtcMIzoziONxBOT3KEPfWgu1-525gg3Mii-V~Fpj3dM-1H0YIaL~0JVz0CFvA-cnTowWbBV96GAb7~XRt5uBR9zLU9Bnv~lM42D1d4M~NSSRu1iAXL89WpzvRmA4MRqI7OmTw0pj7IIFhxpH-rnERFx8UXOwEux7DaK1kfyWbtGCOm0~K4jG-hzsLWAVjjfrZjYX0IeRtgckKVXszPYEBCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196344&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02108.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rooted tree comparing the relative whole genome duplication events in many plant species including <em>A. hypochondriacus</em>. The tree is taken from PNAS journal with permission from the authorand modified to include <em>A. hypochondriacus</em>, creating a branch for Caryophyllales, using the phylogenetic tree generated in-house using rbcL gene (not shown) from 75 species across all plant orders. The predicted and reported ages of the duplication events are retained from the original paper. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU021F8" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02108.jpeg?Expires=2147483647&amp;Signature=LN9a2OlbBybiEqRhfRPpWT85uGdbssczlHgQqMKvRTz9zJSS6-3AKrzx07V7KxQ3e-c~p~sr5KWhPrBWnloh4bEM3YGGA4W4RJZJZbXZZ~AcvgGh9v2Wx11vRX71TgXyknw3zDpXCbxz4EjQfvFl6MjswZ1XItaE~dLW51krho1Y0dSpsKewQ3PZMtFtoEnsslLdIrtINZJajctzauqDTh3jE3FV4mPDwfsqUdxJNoIxDQfKaNGh6~YjrorXYPpqWvKuFH~bzmeYLxk189Sp5Ny2281vKTej7UlAn25iEvGevBQE3oZPx3V2SzlQAm2j9qXtos0XrxfIZnsJ0wgKYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rooted tree comparing the relative whole genome duplication events in many plant species including A. hypochondriacus. The tree is taken from PNAS journal with permission from the authorand modified to include A. hypochondriacus, creating a branch for Caryophyllales, using the phylogenetic tree generated in-house using rbcL gene (not shown) from 75 species across all plant orders. The predicted and reported ages of the duplication events are retained from the original paper. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02108.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196344/dsu02108.jpeg" path-from-xml="dsu02108.jpeg" target="_blank">View large</a><a section="87196344" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02108.jpeg?Expires=2147483647&amp;Signature=lFHCbIfHoiPnZn0ro1k2abNLpIWXYhYEknSIXeiXjosWdAaJzFWZyIxN~8vqmi7GLE5V1TyPmQjMKY7uw9IsWzVHh0pO8c8tpCLuk9p3M0LxebAUL6jxxW-heXEWZ~AGtcMIzoziONxBOT3KEPfWgu1-525gg3Mii-V~Fpj3dM-1H0YIaL~0JVz0CFvA-cnTowWbBV96GAb7~XRt5uBR9zLU9Bnv~lM42D1d4M~NSSRu1iAXL89WpzvRmA4MRqI7OmTw0pj7IIFhxpH-rnERFx8UXOwEux7DaK1kfyWbtGCOm0~K4jG-hzsLWAVjjfrZjYX0IeRtgckKVXszPYEBCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196344&amp;ar=2754539&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02108.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rooted tree comparing the relative whole genome duplication events in many plant species including <em>A. hypochondriacus</em>. The tree is taken from PNAS journal with permission from the authorand modified to include <em>A. hypochondriacus</em>, creating a branch for Caryophyllales, using the phylogenetic tree generated in-house using rbcL gene (not shown) from 75 species across all plant orders. The predicted and reported ages of the duplication events are retained from the original paper. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div></div><p>We find that there is 15% overestimation of the genome size of <em>A. hypochondriacus</em> based solely on assembly, even when only scaffolds greater than 1,000 are considered for the estimation. Similar overestimation has been reported for the <em>V. vinifera</em> genome, which is attributed to 11% hemizygosity.<sup><a reveal-id="DSU021C55" data-open="DSU021C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Since, in genomes with a significant percentage of hemizygosity, the coverage of the homozygous regions can be expected to be two times compared with the hemizygous regions, assembly tools are likely to be more biased towards the homozygous regions, thus providing the basis for high coverage of coding regions.</p>                            <h3 scrollto-destination=87196346 id="87196346" class="section-title" data-legacy-id=s3e>3.5. C4 evolution</h3>
<p>It is now well established that PEPC (EC 4.1.1.31) is the key enzyme in the C4 pathway.<sup><a reveal-id="DSU021C62" data-open="DSU021C62" class="link link-ref link-reveal xref-bibr">62</a></sup> Thus, comparative analysis of PEPC genes across plant kingdom, including C3 and C4 plants, has been attempted by many groups.<sup><a reveal-id="DSU021C63" data-open="DSU021C63" class="link link-ref link-reveal xref-bibr">63</a>,<a reveal-id="DSU021C64" data-open="DSU021C64" class="link link-ref link-reveal xref-bibr">64</a></sup> There are multiple PEPC isoenzymes in all plant species including C3 plants. Work by various groups has shown that the C4-specific PEPC gene harbours a serine in place of alanine at position 780 of maize PEPC gene with accession CAA33317 (A780S) among both dicots and monocots.<sup><a reveal-id="DSU021C65" data-open="DSU021C65" class="link link-ref link-reveal xref-bibr">65</a></sup> For example, one of the four sorghum PEPC genes harbours this mutation.</p><div class="block-child-p">We have identified full-length cDNA sequences for all the four PEPC isoenzymes from the genome of <em>A. hypochondriacus</em> with only one harbouring the A780S mutation. In order to check if all PEPC genes across various species harbouring A780S mutation result from divergent or convergent evolution, we have created a phylogenetic tree using the multiple sequence alignment of all the PEPC isoenzymes from representative species under various major plant orders including many C3 and C4 from both dicots and monocots (Fig. <a reveal-id="DSU021F9" data-open="DSU021F9" class="link link-reveal link-table xref-fig">9</a>). All PEPC isoenzymes harbouring the A780S mutation, among monocot species, cluster together, suggesting C4 evolution predating speciation in monocots. However, the PEPC gene harbouring the A780S mutation in C4 dicots (represented by <em>A. hypochondriacus</em> and <em>Mollugo cerviana</em> from Caryophyllales and <em>Flaveria trinervia</em> from Asterids), clusters in distal clades. Based on this observation, there can be two hypotheses: (i) the C4-specific mutation occurred independently under Asterid and Caryophyllales; (ii) all C3 dicot plants have selectively lost an ancestral C4-specific isoform during the course of evolution. The first hypothesis has been reported in the literature<sup><a reveal-id="DSU021C65" data-open="DSU021C65" class="link link-ref link-reveal xref-bibr">65</a></sup> and our observation supports the same. In other words, C4 switch in dicots is a convergent evolution.<sup><a reveal-id="DSU021C65" data-open="DSU021C65" class="link link-ref link-reveal xref-bibr">65</a></sup> For the second hypothesis to be true, one would need to show that at least one C3 dicot plant continues to retain a C4-specific PEPC gene. <div data-id="DSU021F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02109.jpeg?Expires=2147483647&Signature=Ueai5lAlUX0sEE15t43aKlidyZguMsuR0jOS7ZUmw748xQmyburn9TZiYYHrx5BQVKDhzs1uXeEBiLgvjQF7IMQmzhdNA6M01eR7Gy82x1p3hNWg7dPqat8nsawuYwvAKEf1Fl2osEIyzQypCkTo0IuTbE3U91a3NWeQAy64EQqPgTgXkkiTBviTFxMgMiKrJvF4BwCUaislLNzvNYbu5G-We3gqzv79yDacnO1lsT-tSPPBVMhD2PzD5ZLPdZUmB3~lFelXhmodDEmSWIoldE8dSfwnRT5fuQg-xPbLgFawo4JpmxL~Ac628EPpOyfIDdLBsA2i0gLFOKC8QkuoIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rooted phylogram of PEPC 1, 2, 3, and 4. Branches labelled A780S are C4-specific PEPC isoforms containing alanine to serine mutation at residue 780 of Z. mays C4-specific gene with accession CAA33317. All the bacterial-type PEPC isoforms including A. hypochondriacus 4 and B. vulgaris 4 cluster in one clade. All the C4-specific PEPC genes from monocots cluster together. Among dicots, C4-specific PEPC genes are split into two clades representing Asterids and Caryophyllales. There is only one C4-specific PEPC gene in A. hypochondriacus as is also found in other C4 plants. In the figure, the protein sequence of PEPC taken from GenBank (gb ADO15315.1) has been labelled as A. hypochondriacus gb and the one extracted from the genomic scaffolds is labelled as A. hypochondriacus 1. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02109.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196348/dsu02109.jpeg" path-from-xml="dsu02109.jpeg" target="_blank">View large</a><a section="87196348" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02109.jpeg?Expires=2147483647&Signature=JtNGHf2QF68lOupy30IP21hb-VYKgetxRyyxiKuqCwMgmxqKTdq5XmMH1IRqB-8sFSkis0PJlVNpB7dKanA7d09i9ao4ori56KWCoAXdzqvYYBXXeDPIUK2~cPTzNsMk7VedmLHj4E1OvnADo9WbkqAiG2xENryDh2PZtjbMzQ6olzN61jVmChd2CdD8uH7mQ~E37cuZviAgWxaVmYvMucJgpDobOP~9Px3uZKPTenMcoQ-ufmI-mb86g6XFufnU9si2q7l6todYESDfeOlJTZ4nDcpE4e0TGci5ttyCtlZvJRVYzrz6sntxxiPPLhNCeXOBAOIxOhsj0QON3~nG7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196348&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02109.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rooted phylogram of PEPC 1, 2, 3, and 4. Branches labelled A780S are C4-specific PEPC isoforms containing alanine to serine mutation at residue 780 of <em>Z. mays</em> C4-specific gene with accession CAA33317. All the bacterial-type PEPC isoforms including <em>A. hypochondriacus</em> 4 and <em>B. vulgaris</em> 4 cluster in one clade. All the C4-specific PEPC genes from monocots cluster together. Among dicots, C4-specific PEPC genes are split into two clades representing Asterids and Caryophyllales. There is only one C4-specific PEPC gene in <em>A. hypochondriacus</em> as is also found in other C4 plants. In the figure, the protein sequence of PEPC taken from GenBank (gb ADO15315.1) has been labelled as <em>A. hypochondriacus</em> gb and the one extracted from the genomic scaffolds is labelled as <em>A. hypochondriacus</em> 1. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU021F9" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02109.jpeg?Expires=2147483647&Signature=Ueai5lAlUX0sEE15t43aKlidyZguMsuR0jOS7ZUmw748xQmyburn9TZiYYHrx5BQVKDhzs1uXeEBiLgvjQF7IMQmzhdNA6M01eR7Gy82x1p3hNWg7dPqat8nsawuYwvAKEf1Fl2osEIyzQypCkTo0IuTbE3U91a3NWeQAy64EQqPgTgXkkiTBviTFxMgMiKrJvF4BwCUaislLNzvNYbu5G-We3gqzv79yDacnO1lsT-tSPPBVMhD2PzD5ZLPdZUmB3~lFelXhmodDEmSWIoldE8dSfwnRT5fuQg-xPbLgFawo4JpmxL~Ac628EPpOyfIDdLBsA2i0gLFOKC8QkuoIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rooted phylogram of PEPC 1, 2, 3, and 4. Branches labelled A780S are C4-specific PEPC isoforms containing alanine to serine mutation at residue 780 of Z. mays C4-specific gene with accession CAA33317. All the bacterial-type PEPC isoforms including A. hypochondriacus 4 and B. vulgaris 4 cluster in one clade. All the C4-specific PEPC genes from monocots cluster together. Among dicots, C4-specific PEPC genes are split into two clades representing Asterids and Caryophyllales. There is only one C4-specific PEPC gene in A. hypochondriacus as is also found in other C4 plants. In the figure, the protein sequence of PEPC taken from GenBank (gb ADO15315.1) has been labelled as A. hypochondriacus gb and the one extracted from the genomic scaffolds is labelled as A. hypochondriacus 1. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02109.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196348/dsu02109.jpeg" path-from-xml="dsu02109.jpeg" target="_blank">View large</a><a section="87196348" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02109.jpeg?Expires=2147483647&Signature=JtNGHf2QF68lOupy30IP21hb-VYKgetxRyyxiKuqCwMgmxqKTdq5XmMH1IRqB-8sFSkis0PJlVNpB7dKanA7d09i9ao4ori56KWCoAXdzqvYYBXXeDPIUK2~cPTzNsMk7VedmLHj4E1OvnADo9WbkqAiG2xENryDh2PZtjbMzQ6olzN61jVmChd2CdD8uH7mQ~E37cuZviAgWxaVmYvMucJgpDobOP~9Px3uZKPTenMcoQ-ufmI-mb86g6XFufnU9si2q7l6todYESDfeOlJTZ4nDcpE4e0TGci5ttyCtlZvJRVYzrz6sntxxiPPLhNCeXOBAOIxOhsj0QON3~nG7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196348&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02109.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rooted phylogram of PEPC 1, 2, 3, and 4. Branches labelled A780S are C4-specific PEPC isoforms containing alanine to serine mutation at residue 780 of <em>Z. mays</em> C4-specific gene with accession CAA33317. All the bacterial-type PEPC isoforms including <em>A. hypochondriacus</em> 4 and <em>B. vulgaris</em> 4 cluster in one clade. All the C4-specific PEPC genes from monocots cluster together. Among dicots, C4-specific PEPC genes are split into two clades representing Asterids and Caryophyllales. There is only one C4-specific PEPC gene in <em>A. hypochondriacus</em> as is also found in other C4 plants. In the figure, the protein sequence of PEPC taken from GenBank (gb ADO15315.1) has been labelled as <em>A. hypochondriacus</em> gb and the one extracted from the genomic scaffolds is labelled as <em>A. hypochondriacus</em> 1. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div></div>                            <h3 scrollto-destination=87196349 id="87196349" class="section-title" data-legacy-id=s3f>3.6. Functional characterization and profiling of genes
in the lysine biosynthetic pathway</h3>
<p>In plants, the aspartate family pathway is responsible for the biosynthesis of four amino acids—threonine, isoleucine, methionine, and lysine.<sup><a reveal-id="DSU021C66" data-open="DSU021C66" class="link link-ref link-reveal xref-bibr">66</a></sup> Based on the lysine biosynthetic pathway in KEGG,<sup><a reveal-id="DSU021C67" data-open="DSU021C67" class="link link-ref link-reveal xref-bibr">67</a>,<a reveal-id="DSU021C68" data-open="DSU021C68" class="link link-ref link-reveal xref-bibr">68</a></sup> here are a total of seven enzymatic steps in the synthesis of <span class="small-caps">l</span>-lysine starting from <span class="small-caps">l</span>-aspartate, of which the first two steps are shared for all four amino acids before the lysine biosynthesis pathway diverges. The number of isoenzymes encoded by <em>A. hypochondriacus</em> for all the seven enzymatic steps in the aspartate pathway of lysine biosynthesis has been identified and is listed in Table <a reveal-id="DSU021TB3" data-open="DSU021TB3" class="link link-reveal link-table xref-fig">3</a> for comparison with those in other plant species. <div content-id="DSU021TB3" class="table-modal"><div class="table-wrap"><div id="DSU021TB3" data-id="DSU021TB3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Isozyme number polymorphism in <em>A. hypochondriacus</em> compared with other species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>EC number </th><th>EC 2.7.2.4 </th><th>EC 1.2.1.11 </th><th>EC 4.3.3.7 </th><th>EC 1.17.1.8 </th><th>EC 2.6.1.83 </th><th>EC 5.1.1.7 </th><th>EC 4.1.1.20 </th></tr><tr><th>Enzyme name </th><th>AK </th><th>ASD </th><th>DHDPS </th><th>DHDPR </th><th>DAPAT </th><th>DAPE </th><th>DAPDC </th></tr></thead><tbody><tr><td><em>A. thaliana</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td><em>O. sativa</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>Short </td><td>1 </td></tr><tr><td><em>Z. mays</em> </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>S. bicolor</em> </td><td>3 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td></tr><tr><td><em>G. max</em> </td><td>4 </td><td>2 </td><td>3 </td><td>3 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>V. vinifera</em> </td><td>2 </td><td>2 </td><td>3 </td><td>2 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>Ricinus communis</em> </td><td>1 </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>1 </td></tr><tr><td><em>A. hypochondriacus</em> </td><td>1 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div content-id="DSU021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>EC number </th><th>EC 2.7.2.4 </th><th>EC 1.2.1.11 </th><th>EC 4.3.3.7 </th><th>EC 1.17.1.8 </th><th>EC 2.6.1.83 </th><th>EC 5.1.1.7 </th><th>EC 4.1.1.20 </th></tr><tr><th>Enzyme name </th><th>AK </th><th>ASD </th><th>DHDPS </th><th>DHDPR </th><th>DAPAT </th><th>DAPE </th><th>DAPDC </th></tr></thead><tbody><tr><td><em>A. thaliana</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td><em>O. sativa</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>Short </td><td>1 </td></tr><tr><td><em>Z. mays</em> </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>S. bicolor</em> </td><td>3 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td></tr><tr><td><em>G. max</em> </td><td>4 </td><td>2 </td><td>3 </td><td>3 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>V. vinifera</em> </td><td>2 </td><td>2 </td><td>3 </td><td>2 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>Ricinus communis</em> </td><td>1 </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>1 </td></tr><tr><td><em>A. hypochondriacus</em> </td><td>1 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>The enzymes listed in the table are: AK: monofunctional aspartate kinase; ASD: aspartate semialdehyde dehydrogenase; DHDPS: dihydrodipicolinate synthase; DHDPR: dihydrodipicolinate reductase; DAPAT: diaminopimelate aminotransferase; DAPE: diaminopimelate epimerase; DAPDC: diaminopimelate decarboxylase.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU021TB3" data-id="DSU021TB3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Isozyme number polymorphism in <em>A. hypochondriacus</em> compared with other species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>EC number </th><th>EC 2.7.2.4 </th><th>EC 1.2.1.11 </th><th>EC 4.3.3.7 </th><th>EC 1.17.1.8 </th><th>EC 2.6.1.83 </th><th>EC 5.1.1.7 </th><th>EC 4.1.1.20 </th></tr><tr><th>Enzyme name </th><th>AK </th><th>ASD </th><th>DHDPS </th><th>DHDPR </th><th>DAPAT </th><th>DAPE </th><th>DAPDC </th></tr></thead><tbody><tr><td><em>A. thaliana</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td><em>O. sativa</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>Short </td><td>1 </td></tr><tr><td><em>Z. mays</em> </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>S. bicolor</em> </td><td>3 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td></tr><tr><td><em>G. max</em> </td><td>4 </td><td>2 </td><td>3 </td><td>3 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>V. vinifera</em> </td><td>2 </td><td>2 </td><td>3 </td><td>2 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>Ricinus communis</em> </td><td>1 </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>1 </td></tr><tr><td><em>A. hypochondriacus</em> </td><td>1 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div content-id="DSU021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>EC number </th><th>EC 2.7.2.4 </th><th>EC 1.2.1.11 </th><th>EC 4.3.3.7 </th><th>EC 1.17.1.8 </th><th>EC 2.6.1.83 </th><th>EC 5.1.1.7 </th><th>EC 4.1.1.20 </th></tr><tr><th>Enzyme name </th><th>AK </th><th>ASD </th><th>DHDPS </th><th>DHDPR </th><th>DAPAT </th><th>DAPE </th><th>DAPDC </th></tr></thead><tbody><tr><td><em>A. thaliana</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td><em>O. sativa</em> </td><td>3 </td><td>1 </td><td>2 </td><td>4 </td><td>2 </td><td>Short </td><td>1 </td></tr><tr><td><em>Z. mays</em> </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>S. bicolor</em> </td><td>3 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td></tr><tr><td><em>G. max</em> </td><td>4 </td><td>2 </td><td>3 </td><td>3 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>V. vinifera</em> </td><td>2 </td><td>2 </td><td>3 </td><td>2 </td><td>3 </td><td>2 </td><td>2 </td></tr><tr><td><em>Ricinus communis</em> </td><td>1 </td><td>1 </td><td>1 </td><td>2 </td><td>3 </td><td>1 </td><td>1 </td></tr><tr><td><em>A. hypochondriacus</em> </td><td>1 </td><td>1 </td><td>2 </td><td>2 </td><td>2 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>The enzymes listed in the table are: AK: monofunctional aspartate kinase; ASD: aspartate semialdehyde dehydrogenase; DHDPS: dihydrodipicolinate synthase; DHDPR: dihydrodipicolinate reductase; DAPAT: diaminopimelate aminotransferase; DAPE: diaminopimelate epimerase; DAPDC: diaminopimelate decarboxylase.</p></div></div></div></div></p><p>It is known that both in prokaryotes and photosynthetic eukaryotes, lysine biosynthesis is mainly regulated by two allosteric enzymes.<sup><a reveal-id="DSU021C69" data-open="DSU021C69" class="link link-ref link-reveal xref-bibr">69</a></sup> These are (i) monofunctional AK (EC 2.7.2.4), the first and the critical enzyme in the aspartate family pathway, and (ii) dihydrodipicolinate synthase (DHDPS, EC 4.3.3.7), the first enzyme towards lysine biosynthesis within the aspartate pathway. Given this, polymorphisms of any kind within these two gene loci can be expected to have a high likelihood of correlating with the high-lysine phenotype in grain amaranth.</p><div class="block-child-p">There are varying numbers of monofunctional AK isoenzymes with varying lysine sensitivity in plants. For example, AK1, AK2, and AK3 genes of <em>A. thaliana</em> vary significantly in their affinity to lysine, with AK1 being the least sensitive.<sup><a reveal-id="DSU021C69" data-open="DSU021C69" class="link link-ref link-reveal xref-bibr">69</a></sup> In this context, it is of interest to know if the lysine sensitivity of the only monofunctional AK gene of <em>A. hypochondriacus</em> compares with AK1 or the other two enzymes of <em>A. thaliana</em>. Phylogenetic analysis (Fig. <a reveal-id="DSU021F10" data-open="DSU021F10" class="link link-reveal link-table xref-fig">10</a>) using the multiple sequence alignment of protein sequences of AK gene paralogs from diverse species, clusters the AK gene of <em>A. hypochondriacus</em> in the same clade as the AK1 gene of <em>A. thaliana</em>, suggesting that the AK gene of <em>A. hypochondriacus</em> is similar in lysine sensitivity to the AK1 gene. Loss of orthologs of the two lysine-sensitive AK2 and AK3 genes in <em>A. hypochondriacus</em> may be one of the reasons for the high-lysine phenotype in <em>A. hypochondriacus</em>, thus providing a testable hypothesis. <div data-id="DSU021F10" class="fig fig-section"><div class="label fig-label">Figure 10.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02110.jpeg?Expires=2147483647&Signature=PS3p-nwOfjTHkqqkNsPPDzX~lbLq1jTTjEfgNR72PqjeibA-dvNHF8yN8evS-X5iJXjr0gXkhyET7l8p2iVmulRq33zB3ir8uTDqgozrK9AFSk1GreOlQQnQhZxZRcGza5lV0blkLUZ7LmLJwTbWlYKH5wgkNFljW7QfsCSir00~onZ-715zlrLslXIAEzfmNwpxAE76~g1X7lrRJpafgaaZprTtcFM5VWqhQL3gP0GXBA2l8ZElTJZsrKZchx~OlwZYQ3vC7nI2uPBbVMxMYMXfzL78-hbHGTCsxW5jZIHgZU5J5nwFTxdrjM9kWHw7Hlb73zf3rjqZADso2OEjJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unrooted phylogram of monofunctional aspartate kinases, the first enzyme in the asparate family pathway. The tree shows that A. hypochondriacus AK gene is in the same clade as AK1 (A. thaliana 1) of A. thaliana, B. vulgaris, and V. vinifera. The AK2 and AK3 (A. thaliana 2 and 3) cluster with V. vinifera 3. The sequences that could not be annotated to any of AK 1/2/3 are labelled as x1, x2, and so on. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02110.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196352/dsu02110.jpeg" path-from-xml="dsu02110.jpeg" target="_blank">View large</a><a section="87196352" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02110.jpeg?Expires=2147483647&Signature=Hws86CNr5XLj-hN7p9C-EgjmFQfStRJ70CS-ggXOsMrweoy6httXkIgkJmIrB4Z5wN75U~jFvvXfa~rpvSXdAWCzClKfRlC9VRKni7tdLc3HeSF~VXQO45giqAiL495xPBiXz20Xc1HCATgUjMU0gCDvYuDca3mNLJDCTmGXtjXLnKEUMmNvNwDoePrL03Yr5QHfB8WuynsfpPlsib~1AXUGAEe4YADXqo8xkydjiK-cNFezb9Apdma1QFqIPhC1-pF5p2ReuaJd~0Z3H4XdnaFgztNgumlBwqEc7r2yW1y9iFCdtTF3XJi8c2lzd~aCTxGrw~QOdiaCkDtsD7MOuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196352&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02110.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unrooted phylogram of monofunctional aspartate kinases, the first enzyme in the asparate family pathway. The tree shows that <em>A. hypochondriacus</em> AK gene is in the same clade as AK1 (<em>A. thaliana</em> 1) of <em>A. thaliana</em>, <em>B. vulgaris</em>, and <em>V. vinifera</em>. The AK2 and AK3 (<em>A. thaliana</em> 2 and 3) cluster with <em>V. vinifera</em> 3. The sequences that could not be annotated to any of AK 1/2/3 are labelled as x1, x2, and so on. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU021F10" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 10.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/m_dsu02110.jpeg?Expires=2147483647&Signature=PS3p-nwOfjTHkqqkNsPPDzX~lbLq1jTTjEfgNR72PqjeibA-dvNHF8yN8evS-X5iJXjr0gXkhyET7l8p2iVmulRq33zB3ir8uTDqgozrK9AFSk1GreOlQQnQhZxZRcGza5lV0blkLUZ7LmLJwTbWlYKH5wgkNFljW7QfsCSir00~onZ-715zlrLslXIAEzfmNwpxAE76~g1X7lrRJpafgaaZprTtcFM5VWqhQL3gP0GXBA2l8ZElTJZsrKZchx~OlwZYQ3vC7nI2uPBbVMxMYMXfzL78-hbHGTCsxW5jZIHgZU5J5nwFTxdrjM9kWHw7Hlb73zf3rjqZADso2OEjJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unrooted phylogram of monofunctional aspartate kinases, the first enzyme in the asparate family pathway. The tree shows that A. hypochondriacus AK gene is in the same clade as AK1 (A. thaliana 1) of A. thaliana, B. vulgaris, and V. vinifera. The AK2 and AK3 (A. thaliana 2 and 3) cluster with V. vinifera 3. The sequences that could not be annotated to any of AK 1/2/3 are labelled as x1, x2, and so on. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu02110.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196352/dsu02110.jpeg" path-from-xml="dsu02110.jpeg" target="_blank">View large</a><a section="87196352" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu02110.jpeg?Expires=2147483647&Signature=Hws86CNr5XLj-hN7p9C-EgjmFQfStRJ70CS-ggXOsMrweoy6httXkIgkJmIrB4Z5wN75U~jFvvXfa~rpvSXdAWCzClKfRlC9VRKni7tdLc3HeSF~VXQO45giqAiL495xPBiXz20Xc1HCATgUjMU0gCDvYuDca3mNLJDCTmGXtjXLnKEUMmNvNwDoePrL03Yr5QHfB8WuynsfpPlsib~1AXUGAEe4YADXqo8xkydjiK-cNFezb9Apdma1QFqIPhC1-pF5p2ReuaJd~0Z3H4XdnaFgztNgumlBwqEc7r2yW1y9iFCdtTF3XJi8c2lzd~aCTxGrw~QOdiaCkDtsD7MOuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196352&ar=2754539&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02110.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unrooted phylogram of monofunctional aspartate kinases, the first enzyme in the asparate family pathway. The tree shows that <em>A. hypochondriacus</em> AK gene is in the same clade as AK1 (<em>A. thaliana</em> 1) of <em>A. thaliana</em>, <em>B. vulgaris</em>, and <em>V. vinifera</em>. The AK2 and AK3 (<em>A. thaliana</em> 2 and 3) cluster with <em>V. vinifera</em> 3. The sequences that could not be annotated to any of AK 1/2/3 are labelled as x1, x2, and so on. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div></div><p>DHDPS, the other most important set of enzymes in the lysine biosynthesis, is known to be orders of magnitude more sensitive to lysine.<sup><a reveal-id="DSU021C70" data-open="DSU021C70" class="link link-ref link-reveal xref-bibr">70</a></sup> Similar to many other plant species from various plant orders, there are two DHDPS isoenzymes in <em>A. hypochondriacus</em>. Profiling gene expression levels from transcriptome sequencing for three stages of shoot development and mature seeds, we observe that the expression levels of both DHDPS isoenzymes decrease with developmental stages of the shoot and significantly spike up in the seeds (Table <a reveal-id="DSU021TB4" data-open="DSU021TB4" class="link link-reveal link-table xref-fig">4</a>). The high levels of expression of DHDPS isoenzymes in seeds suggest yet another hypothesis for producing high-lysine grains in grain amaranths. <div content-id="DSU021TB4" class="table-modal"><div class="table-wrap"><div id="DSU021TB4" data-id="DSU021TB4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Expression level of genes implicated in the lysine biosynthetic pathway in the shoot at three stages of development and in mature seeds</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Enzyme </th><th rowspan="2">EC #<br />AT ID </th><th colspan="4">RPKM<hr /> </th></tr><tr><th>Shoot<br />15 days </th><th>Shoot<br />25 days </th><th>Shoot<br />30 days </th><th>Mature seeds </th></tr></thead><tbody><tr><td>AK </td><td>2.7.2.4<br />AT5G13280 </td><td>12.21 </td><td>11.13 </td><td>12.82 </td><td>4.17 </td></tr><tr><td>AK-homoserine dehydrogenase </td><td>2.7.2.4–1.1.1.3 </td><td>23.80 </td><td>20.43 </td><td>16.28 </td><td>6.98 </td></tr><tr><td>Aspartate semialdehyde dehydrogenase </td><td>1.2.1.11<br />AT1G14810 </td><td>8.36 </td><td>6.91 </td><td>7.73 </td><td>3.67 </td></tr><tr><td rowspan="2">Dihydrodipicolinate synthase </td><td>4.3.3.7<br />AT2G45440 </td><td>11.51 </td><td>11.88 </td><td>7.43 </td><td>30.77 </td></tr><tr><td>4.3.3.7<br />AT3G60880 </td><td>5.09 </td><td>5.00 </td><td>5.16 </td><td>18.57 </td></tr><tr><td rowspan="3">Dihydrodipicolinate reductase </td><td>1.17.1.8<br />AT3G59890 </td><td>2.37 </td><td>1.16 </td><td>1.35 </td><td>0.26 </td></tr><tr><td>1.17.1.8<br />AT2G44040 </td><td>2.77 </td><td>2.45 </td><td>2.29 </td><td>0.61 </td></tr><tr><td>1.17.1.8<br />AT5G52100 </td><td>27.58 </td><td>19.22 </td><td>8.36 </td><td>0.00 </td></tr><tr><td rowspan="2">Diaminopimelate aminotransferase </td><td>2.6.1.83<br />AT4G33680 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>2.6.1.83<br />AT2G13810 </td><td>7.06 </td><td>6.36 </td><td>6.18 </td><td>1.64 </td></tr><tr><td>Diaminopimelate epimerase </td><td>5.1.1.7<br />AT3G53580 </td><td>42.28 </td><td>37.48 </td><td>34.32 </td><td>4.91 </td></tr><tr><td rowspan="2">Diaminopimelate decarboxylase </td><td>4.1.1.20<br />AT3G14390 </td><td>10.67 </td><td>10.19 </td><td>12.64 </td><td>3.37 </td></tr><tr><td>4.1.1.20<br />AT5G11880 </td><td>25.53 </td><td>20.79 </td><td>18.06 </td><td>3.06 </td></tr></tbody></table></div><div content-id="DSU021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Enzyme </th><th rowspan="2">EC #<br />AT ID </th><th colspan="4">RPKM<hr /> </th></tr><tr><th>Shoot<br />15 days </th><th>Shoot<br />25 days </th><th>Shoot<br />30 days </th><th>Mature seeds </th></tr></thead><tbody><tr><td>AK </td><td>2.7.2.4<br />AT5G13280 </td><td>12.21 </td><td>11.13 </td><td>12.82 </td><td>4.17 </td></tr><tr><td>AK-homoserine dehydrogenase </td><td>2.7.2.4–1.1.1.3 </td><td>23.80 </td><td>20.43 </td><td>16.28 </td><td>6.98 </td></tr><tr><td>Aspartate semialdehyde dehydrogenase </td><td>1.2.1.11<br />AT1G14810 </td><td>8.36 </td><td>6.91 </td><td>7.73 </td><td>3.67 </td></tr><tr><td rowspan="2">Dihydrodipicolinate synthase </td><td>4.3.3.7<br />AT2G45440 </td><td>11.51 </td><td>11.88 </td><td>7.43 </td><td>30.77 </td></tr><tr><td>4.3.3.7<br />AT3G60880 </td><td>5.09 </td><td>5.00 </td><td>5.16 </td><td>18.57 </td></tr><tr><td rowspan="3">Dihydrodipicolinate reductase </td><td>1.17.1.8<br />AT3G59890 </td><td>2.37 </td><td>1.16 </td><td>1.35 </td><td>0.26 </td></tr><tr><td>1.17.1.8<br />AT2G44040 </td><td>2.77 </td><td>2.45 </td><td>2.29 </td><td>0.61 </td></tr><tr><td>1.17.1.8<br />AT5G52100 </td><td>27.58 </td><td>19.22 </td><td>8.36 </td><td>0.00 </td></tr><tr><td rowspan="2">Diaminopimelate aminotransferase </td><td>2.6.1.83<br />AT4G33680 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>2.6.1.83<br />AT2G13810 </td><td>7.06 </td><td>6.36 </td><td>6.18 </td><td>1.64 </td></tr><tr><td>Diaminopimelate epimerase </td><td>5.1.1.7<br />AT3G53580 </td><td>42.28 </td><td>37.48 </td><td>34.32 </td><td>4.91 </td></tr><tr><td rowspan="2">Diaminopimelate decarboxylase </td><td>4.1.1.20<br />AT3G14390 </td><td>10.67 </td><td>10.19 </td><td>12.64 </td><td>3.37 </td></tr><tr><td>4.1.1.20<br />AT5G11880 </td><td>25.53 </td><td>20.79 </td><td>18.06 </td><td>3.06 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="DSU021TB4" data-id="DSU021TB4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Expression level of genes implicated in the lysine biosynthetic pathway in the shoot at three stages of development and in mature seeds</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Enzyme </th><th rowspan="2">EC #<br />AT ID </th><th colspan="4">RPKM<hr /> </th></tr><tr><th>Shoot<br />15 days </th><th>Shoot<br />25 days </th><th>Shoot<br />30 days </th><th>Mature seeds </th></tr></thead><tbody><tr><td>AK </td><td>2.7.2.4<br />AT5G13280 </td><td>12.21 </td><td>11.13 </td><td>12.82 </td><td>4.17 </td></tr><tr><td>AK-homoserine dehydrogenase </td><td>2.7.2.4–1.1.1.3 </td><td>23.80 </td><td>20.43 </td><td>16.28 </td><td>6.98 </td></tr><tr><td>Aspartate semialdehyde dehydrogenase </td><td>1.2.1.11<br />AT1G14810 </td><td>8.36 </td><td>6.91 </td><td>7.73 </td><td>3.67 </td></tr><tr><td rowspan="2">Dihydrodipicolinate synthase </td><td>4.3.3.7<br />AT2G45440 </td><td>11.51 </td><td>11.88 </td><td>7.43 </td><td>30.77 </td></tr><tr><td>4.3.3.7<br />AT3G60880 </td><td>5.09 </td><td>5.00 </td><td>5.16 </td><td>18.57 </td></tr><tr><td rowspan="3">Dihydrodipicolinate reductase </td><td>1.17.1.8<br />AT3G59890 </td><td>2.37 </td><td>1.16 </td><td>1.35 </td><td>0.26 </td></tr><tr><td>1.17.1.8<br />AT2G44040 </td><td>2.77 </td><td>2.45 </td><td>2.29 </td><td>0.61 </td></tr><tr><td>1.17.1.8<br />AT5G52100 </td><td>27.58 </td><td>19.22 </td><td>8.36 </td><td>0.00 </td></tr><tr><td rowspan="2">Diaminopimelate aminotransferase </td><td>2.6.1.83<br />AT4G33680 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>2.6.1.83<br />AT2G13810 </td><td>7.06 </td><td>6.36 </td><td>6.18 </td><td>1.64 </td></tr><tr><td>Diaminopimelate epimerase </td><td>5.1.1.7<br />AT3G53580 </td><td>42.28 </td><td>37.48 </td><td>34.32 </td><td>4.91 </td></tr><tr><td rowspan="2">Diaminopimelate decarboxylase </td><td>4.1.1.20<br />AT3G14390 </td><td>10.67 </td><td>10.19 </td><td>12.64 </td><td>3.37 </td></tr><tr><td>4.1.1.20<br />AT5G11880 </td><td>25.53 </td><td>20.79 </td><td>18.06 </td><td>3.06 </td></tr></tbody></table></div><div content-id="DSU021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Enzyme </th><th rowspan="2">EC #<br />AT ID </th><th colspan="4">RPKM<hr /> </th></tr><tr><th>Shoot<br />15 days </th><th>Shoot<br />25 days </th><th>Shoot<br />30 days </th><th>Mature seeds </th></tr></thead><tbody><tr><td>AK </td><td>2.7.2.4<br />AT5G13280 </td><td>12.21 </td><td>11.13 </td><td>12.82 </td><td>4.17 </td></tr><tr><td>AK-homoserine dehydrogenase </td><td>2.7.2.4–1.1.1.3 </td><td>23.80 </td><td>20.43 </td><td>16.28 </td><td>6.98 </td></tr><tr><td>Aspartate semialdehyde dehydrogenase </td><td>1.2.1.11<br />AT1G14810 </td><td>8.36 </td><td>6.91 </td><td>7.73 </td><td>3.67 </td></tr><tr><td rowspan="2">Dihydrodipicolinate synthase </td><td>4.3.3.7<br />AT2G45440 </td><td>11.51 </td><td>11.88 </td><td>7.43 </td><td>30.77 </td></tr><tr><td>4.3.3.7<br />AT3G60880 </td><td>5.09 </td><td>5.00 </td><td>5.16 </td><td>18.57 </td></tr><tr><td rowspan="3">Dihydrodipicolinate reductase </td><td>1.17.1.8<br />AT3G59890 </td><td>2.37 </td><td>1.16 </td><td>1.35 </td><td>0.26 </td></tr><tr><td>1.17.1.8<br />AT2G44040 </td><td>2.77 </td><td>2.45 </td><td>2.29 </td><td>0.61 </td></tr><tr><td>1.17.1.8<br />AT5G52100 </td><td>27.58 </td><td>19.22 </td><td>8.36 </td><td>0.00 </td></tr><tr><td rowspan="2">Diaminopimelate aminotransferase </td><td>2.6.1.83<br />AT4G33680 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>2.6.1.83<br />AT2G13810 </td><td>7.06 </td><td>6.36 </td><td>6.18 </td><td>1.64 </td></tr><tr><td>Diaminopimelate epimerase </td><td>5.1.1.7<br />AT3G53580 </td><td>42.28 </td><td>37.48 </td><td>34.32 </td><td>4.91 </td></tr><tr><td rowspan="2">Diaminopimelate decarboxylase </td><td>4.1.1.20<br />AT3G14390 </td><td>10.67 </td><td>10.19 </td><td>12.64 </td><td>3.37 </td></tr><tr><td>4.1.1.20<br />AT5G11880 </td><td>25.53 </td><td>20.79 </td><td>18.06 </td><td>3.06 </td></tr></tbody></table></div></div></p><p>To correlate the observed gene number polymorphism for AK genes and eQTL of the DHDPS gene, we have generated horizontal and vertical profiles of free lysine across the different developmental stages of all three grain species using ultra-fast liquid chromatography. The study reveals that the free lysine content in seeds of the genus is relatively higher than that in the shoot (0.152 µmol/100 mg in 25-day-old shoot and 0.189 µmol/100 mg in mature seeds). The free lysine content analysis correlates with the expression levels of the DHDPS gene, suggesting eQTL for the high-lysine phenotype. However, the expression profiles of all the other enzymes including AK shows no correlation with high-lysine content in seeds compared with the shoot. Perhaps, the absence of orthologs of the two lysine-sensitive AK genes in <em>A. hypochondriacus</em> may be sufficient for high-lysine phenotype.</p>                            <h2 scrollto-destination=87196355 id="87196355" class="section-title" data-legacy-id=s4>4. Conclusion</h2>
<p>We have reported the draft genome of the first grain species under the plant order Caryophyllales and the first C4 dicot to be sequenced. Using the reported SNPs derived from homozygous regions of grain amaranths,<sup><a reveal-id="DSU021C18" data-open="DSU021C18" class="link link-ref link-reveal xref-bibr">18</a></sup> we conclude that as high as 86% of the homozygous gene-rich region has been deciphered. Similarly, annotation of the genome using various criteria, including transcriptome mapping, gene prediction, and GO annotation, vindicates the conclusion. Also, based on the high level of synteny to the genome of <em>B. vulgaris</em>, we authenticate the quality of the reported scaffolds. We also confirm that the paleohexaploidy event reported to be common to all sequenced dicot species under Rosids and Asterids is also common to Caryophyllales. We have corroborated that the C4 evolution under Caryophyllales has occurred independently of other C4 dicots, for example, <em>F. trinervia</em> under Asterid. Also, we have proposed two testable hypotheses for the high-lysine phenotype including gene number polymorphism for the AK gene and eQTL for DHDPS. We have demonstrated that the draft genome reported here can be useful in advancing our understanding of the diverse phenotypes that are unique to grain amaranths including the unique nutritional profile, aggressive growth, drought resistance and adaptability to environmental stress and characterize genes involved in the biosynthesis of betalains.</p>                            <h3 scrollto-destination=87196357 id="87196357" class="section-title" data-legacy-id=s4a>4.1. Availability</h3>
<p>The short genomic reads used in the study are deposited at NCBI-SRA under the accession ID of SRP031880. The scaffolds for the <em>A. hypochondriacus</em> genome and transcriptome are available at GenBank under the BioProject IDs of PRJNA214803 and PRJNA214804, respectively, and also at <a class="link link-uri" href="http://resource.ibab.ac.in/Plant_Genomics">http://resource.ibab.ac.in/Plant_Genomics</a> for download. An instance of NCBI's wwwBLAST<sup><a reveal-id="DSU021C71" data-open="DSU021C71" class="link link-ref link-reveal xref-bibr">71</a></sup> has also been set up at the same url where users can BLAST a given gene against both the genomic and transcriptomic scaffolds sequenced here and other plant genomes compared here. The seeds of the sequenced plant are in the germplasm currently maintained at IBAB under the accession ID of IAh0001. We are in the process of depositing to the germplasm repository maintained by the National Bureau of Plant Genetic Resources in India. Also, the multiple sequence alignment files in .meg (MEGA5) format used in figs <a reveal-id="DSU021F9" data-open="DSU021F9" class="link link-reveal link-table xref-fig">9</a> and <a reveal-id="DSU021F10" data-open="DSU021F10" class="link link-reveal link-table xref-fig">10</a>, and the rs IDs for the anchored scaffolds in fig. <a reveal-id="DSU021F3" data-open="DSU021F3" class="link link-reveal link-table xref-fig">3</a> are available from the authors on request.</p>                            <h2 scrollto-destination=87196359 id="87196359" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The sequencing work reported here is covered by grants to GANIT Labs from the Department of Information Technology, Government of India (Ref no: 18(4)/2010-E-Infra., 31-03-2010) and the Department of Information Technology, Biotechnology and Science &amp; Technology, Government of Karnataka, India (Ref no: 3451-00-090-2-22). IBAB is recognized as a ‘Centre of Excellence for Research and Training in Bioinformatics’ by the Department of Electronics and Information Technology, Government of India which also covers the computational resources utilized for the work. Department of Biotechnology (DBT), New Delhi, India has provided support to Dr S.S. and the Experimental work via the Ramalingaswamy fellowship (Ref no: BT/HRD/35/02/17/2009) and to Mr S.R.N. via BINC fellowship. Support from DST-FIST was obtained as infrastructural grant to IBAB. Funding to pay the Open Access publication charges for this article was also provided by the Department of Biotechnology to Dr S.S. (Ref no:BT/HRD/35/02/17/2009).</p>                            <h2 scrollto-destination=87196361 id="87196361" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors wish to extend special thanks to Dr S.S. Shivakumara of IBAB for detailed comments on the initial manuscript but for which the manuscript will not be in the current form. The authors would like to acknowledge Dr Imad Eujayl, NWISRL-ARS-USDA and Kimberly, Idaho for sharing information on the BvvSeq-1 assembly, the only other sequenced genome under the plant order Caryophyllales. The authors would also like to acknowledge Yves Van de Peer for allowing them to both use and modify the phylogenetic tree from his publication that depicts the age of recent whole genome duplications across several sequenced plant species.<sup><a reveal-id="DSU021C61" data-open="DSU021C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Last but not the least, the authors also wish to acknowledge IBAB for providing the facilities for field and laboratory work.</p>                            <h2 scrollto-destination=87196363 id="87196363" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU021C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="collab">Innovation, N.R.C. (U.S.) A.C</div>
          <div class="source">Amaranth: Modern Prospects for an Ancient Crop</div> 
          <div class="year">1984</div>
          <div class="publisher-loc">Washington, DC, USA</div>
          <div class="publisher-name">National Academies</div>
        </div></div></div></div><div content-id="DSU021C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marx</div>
               <div class="given-names">J.L.</div>
            </div>
          </div>
          <div class="article-title">Amaranth: a comeback for the food of the Aztecs?</div>
          <div class="source">Science</div> 
          <div class="year">1977</div>
          <div class="volume">198</div>
          <div class="fpage">40</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amaranth%3A%20a%20comeback%20for%20the%20food%20of%20the%20Aztecs%3F&amp;author=J.L.%20Marx&amp;publication_year=1977&amp;journal=Science&amp;volume=198&amp;pages=40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.198.4312.40" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.198.4312.40" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.198.4312.40"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17741884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Caselato-Sousa</div>
               <div class="given-names">V.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Amaya-Farfán</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">State of knowledge on amaranth grain: a comprehensive review</div>
          <div class="source">J. Food Sci.</div> 
          <div class="year">2012</div>
          <div class="volume">77</div>
          <div class="fpage">R93</div>
          <div class="lpage">104</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=State%20of%20knowledge%20on%20amaranth%20grain%3A%20a%20comprehensive%20review&amp;author=V.M.%20Caselato-Sousa&amp;author=J.%20Amaya-Farf%C3%A1n&amp;publication_year=2012&amp;journal=J.%20Food%20Sci.&amp;volume=77&amp;pages=R93-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1750-3841.2012.02645.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1750-3841.2012.02645.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1750-3841.2012.02645.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22515252" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ferreira</div>
               <div class="given-names">R.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Varisi</div>
               <div class="given-names">V.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Meinhardt</div>
               <div class="given-names">L.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lea</div>
               <div class="given-names">P.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Azevedo</div>
               <div class="given-names">R.A.</div>
            </div>
          </div>
          <div class="article-title">Are high-lysine cereal crops still a challenge?</div>
          <div class="source">Braz. J. Med. Biol. Res. Rev. Bras. Pesqui. Médicas E Biológicas Soc. Bras. Biofísica Al.</div> 
          <div class="year">2005</div>
          <div class="volume">38</div>
          <div class="fpage">985</div>
          <div class="lpage">94</div>
        </div></div></div></div><div content-id="DSU021C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schmutz</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cannon</div>
               <div class="given-names">S.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schlueter</div>
               <div class="given-names">J.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Genome sequence of the palaeopolyploid soybean</div>
          <div class="source">Nature</div> 
          <div class="year">2010</div>
          <div class="volume">463</div>
          <div class="fpage">178</div>
          <div class="lpage">83</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Y.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Draft genome sequence of pigeonpea (<em>Cajanus cajan</em>), an orphan legume crop of resource-poor farmers</div>
          <div class="source">Nat. Biotechnol.</div> 
          <div class="year">2012</div>
          <div class="volume">30</div>
          <div class="fpage">83</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20pigeonpea%20(Cajanus%20cajan)%2C%20an%20orphan%20legume%20crop%20of%20resource-poor%20farmers&amp;author=R.K.%20Varshney&amp;author=W.%20Chen&amp;author=Y.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=83-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2022"> </span></div> </div></div></div></div></div><div content-id="DSU021C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ward</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tissue</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Thomas</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Strain</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="article-title">Comparative responses of model C3 and C4 plants to drought in low and elevated CO<sub>2</sub></div>
          <div class="source">Glob. Change Biol.</div> 
          <div class="year">1999</div>
          <div class="volume">5</div>
          <div class="fpage">857</div>
          <div class="lpage">67</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20responses%20of%20model%20C3%20and%20C4%20plants%20to%20drought%20in%20low%20and%20elevated%20CO2&amp;author=J.%20Ward&amp;author=D.%20Tissue&amp;author=R.%20Thomas&amp;author=B.%20Strain&amp;publication_year=1999&amp;journal=Glob.%20Change%20Biol.&amp;volume=5&amp;pages=857-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2486.1999.00270.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2486.1999.00270.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2486.1999.00270.x"> </span></div> </div></div></div></div></div><div content-id="DSU021C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kadereit</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Borch</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Weising</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Freitag</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="article-title">Phylogeny of Amranthaceae and Chenopodiaceae and the evolution of C4 photosynthesis</div>
          <div class="source">Int. J. Plant Sci.</div> 
          <div class="year">2003</div>
          <div class="volume">164</div>
          <div class="fpage">959</div>
          <div class="lpage">86</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20Amranthaceae%20and%20Chenopodiaceae%20and%20the%20evolution%20of%20C4%20photosynthesis&amp;author=G.%20Kadereit&amp;author=T.%20Borch&amp;author=K.%20Weising&amp;author=H.%20Freitag&amp;publication_year=2003&amp;journal=Int.%20J.%20Plant%20Sci.&amp;volume=164&amp;pages=959-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/378649" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F378649" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F378649"> </span></div> </div></div></div></div></div><div content-id="DSU021C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brockington</div>
               <div class="given-names">S.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Walker</div>
               <div class="given-names">R.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Glover</div>
               <div class="given-names">B.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Soltis</div>
               <div class="given-names">P.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Soltis</div>
               <div class="given-names">D.E.</div>
            </div>
          </div>
          <div class="article-title">Complex pigment evolution in the Caryophyllales</div>
          <div class="source">New Phytol.</div> 
          <div class="year">2011</div>
          <div class="volume">190</div>
          <div class="fpage">854</div>
          <div class="lpage">64</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20pigment%20evolution%20in%20the%20Caryophyllales&amp;author=S.F.%20Brockington&amp;author=R.H.%20Walker&amp;author=B.J.%20Glover&amp;author=P.S.%20Soltis&amp;author=D.E.%20Soltis&amp;publication_year=2011&amp;journal=New%20Phytol.&amp;volume=190&amp;pages=854-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2011.190.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2011.190.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2011.190.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21714182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stafford</div>
               <div class="given-names">H.A.</div>
            </div>
          </div>
          <div class="article-title">Anthocyanins and betalains: evolution of the mutually exclusive pathways</div>
          <div class="source">Plant Sci.</div> 
          <div class="year">1994</div>
          <div class="volume">101</div>
          <div class="fpage">91</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anthocyanins%20and%20betalains%3A%20evolution%20of%20the%20mutually%20exclusive%20pathways&amp;author=H.A.%20Stafford&amp;publication_year=1994&amp;journal=Plant%20Sci.&amp;volume=101&amp;pages=91-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9452(94)90244-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9452(94)90244-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9452(94)90244-5"> </span></div> </div></div></div></div></div><div content-id="DSU021C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gandía-Herrero</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">García-Carmona</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">Biosynthesis of betalains: yellow and violet plant pigments</div>
          <div class="source">Trends Plant Sci.</div> 
          <div class="year">2013</div>
          <div class="volume">18</div>
          <div class="fpage">334</div>
          <div class="lpage">43</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20betalains%3A%20yellow%20and%20violet%20plant%20pigments&amp;author=F.%20Gand%C3%ADa-Herrero&amp;author=F.%20Garc%C3%ADa-Carmona&amp;publication_year=2013&amp;journal=Trends%20Plant%20Sci.&amp;volume=18&amp;pages=334-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2013.01.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2013.01.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2013.01.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23395307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kamagaju</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Morandini</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bizuru</div>
               <div class="given-names">E.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Tyrosinase modulation by five Rwandese herbal medicines traditionally used for skin treatment</div>
          <div class="source">J. Ethnopharmacol.</div> 
          <div class="year">2013</div>
          <div class="volume">146</div>
          <div class="fpage">824</div>
          <div class="lpage">34</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tyrosinase%20modulation%20by%20five%20Rwandese%20herbal%20medicines%20traditionally%20used%20for%20skin%20treatment&amp;author=L.%20Kamagaju&amp;author=R.%20Morandini&amp;author=E.%20Bizuru&amp;publication_year=2013&amp;journal=J.%20Ethnopharmacol.&amp;volume=146&amp;pages=824-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jep.2013.02.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jep.2013.02.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jep.2013.02.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23439030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dohm</div>
               <div class="given-names">J.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Minoche</div>
               <div class="given-names">A.E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Holtgräwe</div>
               <div class="given-names">D.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">The genome of the recently domesticated crop plant sugar beet (<em>Beta vulgaris</em>)</div>
          <div class="source">Nature</div> 
          <div class="year">2014</div>
          <div class="volume">505</div>
          <div class="fpage">546</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20recently%20domesticated%20crop%20plant%20sugar%20beet%20(Beta%20vulgaris)&amp;author=J.C.%20Dohm&amp;author=A.E.%20Minoche&amp;author=D.%20Holtgr%C3%A4we&amp;publication_year=2014&amp;journal=Nature&amp;volume=505&amp;pages=546-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12817" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12817" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12817"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24352233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Délano-Frier</div>
               <div class="given-names">J.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Avilés-Arnaut</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Casarrubias-Castillo</div>
               <div class="given-names">K.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Transcriptomic analysis of grain amaranth (<em>Amaranthus hypochondriacus</em>) using 454 pyrosequencing: comparison with <em>A. tuberculatus</em>, expression profiling in stems and in response to biotic and abiotic stress</div>
          <div class="source">BMC Genomics</div> 
          <div class="year">2011</div>
          <div class="volume">12</div>
          <div class="fpage">363</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20of%20grain%20amaranth%20(Amaranthus%20hypochondriacus)%20using%20454%20pyrosequencing%3A%20comparison%20with%20A.%20tuberculatus%2C%20expression%20profiling%20in%20stems%20and%20in%20response%20to%20biotic%20and%20abiotic%20stress&amp;author=J.P.%20D%C3%A9lano-Frier&amp;author=H.%20Avil%C3%A9s-Arnaut&amp;author=K.%20Casarrubias-Castillo&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=363" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21752295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Riggins</div>
               <div class="given-names">C.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peng</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Stewart</div>
               <div class="given-names">C.N.</div>
              Jr
            </div>
            <div class="name string-name">
              <div class="surname">Tranel</div>
               <div class="given-names">P.J.</div>
            </div>
          </div>
          <div class="article-title">Characterization of <em>de novo</em> transcriptome for waterhemp (<em>Amaranthus tuberculatus</em>) using GS-FLX 454 pyrosequencing and its application for studies of herbicide target-site genes</div>
          <div class="source">Pest Manag. Sci.</div> 
          <div class="year">2010</div>
          <div class="volume">66</div>
          <div class="fpage">1042</div>
          <div class="lpage">52</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20de%20novo%20transcriptome%20for%20waterhemp%20(Amaranthus%20tuberculatus)%20using%20GS-FLX%20454%20pyrosequencing%20and%20its%20application%20for%20studies%20of%20herbicide%20target-site%20genes&amp;author=C.W.%20Riggins&amp;author=Y.%20Peng&amp;author=C.N.%20Stewart&amp;author=P.J.%20Tranel&amp;publication_year=2010&amp;journal=Pest%20Manag.%20Sci.&amp;volume=66&amp;pages=1042-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ps.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fps.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fps.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20680963" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Maughan</div>
               <div class="given-names">P.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sisneros</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Luo</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kudrna</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ammiraju</div>
               <div class="given-names">J.S.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wing</div>
               <div class="given-names">R.A.</div>
            </div>
          </div>
          <div class="article-title">Construction of an bacterial artificial chromosome library and genomic sequencing of herbicide target genes</div>
          <div class="source">Crop. Sci.</div> 
          <div class="year">2008</div>
          <div class="volume">48</div>
          <div class="fpage">S85</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20bacterial%20artificial%20chromosome%20library%20and%20genomic%20sequencing%20of%20herbicide%20target%20genes&amp;author=P.J.%20Maughan&amp;author=N.%20Sisneros&amp;author=M.%20Luo&amp;author=D.%20Kudrna&amp;author=J.S.S.%20Ammiraju&amp;author=R.A.%20Wing&amp;publication_year=2008&amp;journal=Crop.%20Sci.&amp;volume=48&amp;pages=S85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2007.08.0434tpg" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2007.08.0434tpg" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2007.08.0434tpg"> </span></div> </div></div></div></div></div><div content-id="DSU021C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mallory</div>
               <div class="given-names">M.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hall</div>
               <div class="given-names">R.V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">McNabb</div>
               <div class="given-names">A.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pratt</div>
               <div class="given-names">D.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jellen</div>
               <div class="given-names">E.N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Maughan</div>
               <div class="given-names">P.J.</div>
            </div>
          </div>
          <div class="article-title">Development and characterization of microsatellite markers for the grain amaranths</div>
          <div class="source">Crop. Sci.</div> 
          <div class="year">2008</div>
          <div class="volume">48</div>
          <div class="fpage">1098</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20microsatellite%20markers%20for%20the%20grain%20amaranths&amp;author=M.A.%20Mallory&amp;author=R.V.%20Hall&amp;author=A.R.%20McNabb&amp;author=D.B.%20Pratt&amp;author=E.N.%20Jellen&amp;author=P.J.%20Maughan&amp;publication_year=2008&amp;journal=Crop.%20Sci.&amp;volume=48&amp;pages=1098" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2007.08.0457" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2007.08.0457" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2007.08.0457"> </span></div> </div></div></div></div></div><div content-id="DSU021C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Maughan</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Smith</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fairbanks</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jellen</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">Development, characterization, and linkage mapping of single nucleotide polymorphisms in the grain amaranths (<em>Amaranthus</em> sp.)</div>
          <div class="source">Plant Genome</div> 
          <div class="year">2011</div>
          <div class="volume">4</div>
          <div class="fpage">92</div>
          <div class="lpage">101</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%2C%20characterization%2C%20and%20linkage%20mapping%20of%20single%20nucleotide%20polymorphisms%20in%20the%20grain%20amaranths%20(Amaranthus%20sp.)&amp;author=P.%20Maughan&amp;author=S.%20Smith&amp;author=D.%20Fairbanks&amp;author=E.%20Jellen&amp;publication_year=2011&amp;journal=Plant%20Genome&amp;volume=4&amp;pages=92-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2010.12.0027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2010.12.0027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2010.12.0027"> </span></div> </div></div></div></div></div><div content-id="DSU021C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nemoto</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Baniya</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Minami</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ujihara</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Grain amaranth research in Nepal</div>
          <div class="source">J. Fac. Agric. Shinshu Univ.</div> 
          <div class="year">1998</div>
          <div class="volume">34</div>
          <div class="fpage">49</div>
          <div class="lpage">58</div>
        </div></div></div></div><div content-id="DSU021C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chan</div>
               <div class="given-names">K.-L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ho</div>
               <div class="given-names">C.-L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Namasivayam</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Napis</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">A simple and rapid method for RNA isolation from plant tissues with high phenolic compounds and polysaccharides</div>
          <div class="source">Protoc. Exch.</div> 
          <div class="year">2007</div>
          <div class="comment">
            <a class="link link-uri" href="http://dx.doi.org/10.1038/nprot.2007.184" target="_blank">doi:10.1038/nprot.2007.184</a>
          </div>
        </div></div></div></div><div content-id="DSU021C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhu</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ruan</div>
               <div class="given-names">J.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">
            <em>De novo</em> assembly of human genomes with massively parallel short read sequencing</div>
          <div class="source">Genome. Res.</div> 
          <div class="year">2010</div>
          <div class="volume">20</div>
          <div class="fpage">265</div>
          <div class="lpage">72</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20human%20genomes%20with%20massively%20parallel%20short%20read%20sequencing&amp;author=R.%20Li&amp;author=H.%20Zhu&amp;author=J.%20Ruan&amp;publication_year=2010&amp;journal=Genome.%20Res.&amp;volume=20&amp;pages=265-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.097261.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.097261.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.097261.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20019144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ogg</div>
               <div class="given-names">C.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Patel</div>
               <div class="given-names">B.K.C.</div>
            </div>
          </div>
          <div class="article-title">Draft genome sequence of <em>Caloramator australicus</em> strain RC3T, a thermoanaerobe from the Great Artesian Basin of Australia</div>
          <div class="source">J. Bacteriol.</div> 
          <div class="year">2011</div>
          <div class="volume">193</div>
          <div class="fpage">2664</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Caloramator%20australicus%20strain%20RC3T%2C%20a%20thermoanaerobe%20from%20the%20Great%20Artesian%20Basin%20of%20Australia&amp;author=C.D.%20Ogg&amp;author=B.K.C.%20Patel&amp;publication_year=2011&amp;journal=J.%20Bacteriol.&amp;volume=193&amp;pages=2664-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00193-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00193-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00193-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21421756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grabherr</div>
               <div class="given-names">M.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Haas</div>
               <div class="given-names">B.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yassour</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>
          <div class="source">Nat. Biotechnol.</div> 
          <div class="year">2011</div>
          <div class="volume">29</div>
          <div class="fpage">644</div>
          <div class="lpage">52</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gish</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Miller</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Myers</div>
               <div class="given-names">E.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lipman</div>
               <div class="given-names">D.J.</div>
            </div>
          </div>
          <div class="article-title">Basic local alignment search tool</div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1990</div>
          <div class="volume">215</div>
          <div class="fpage">403</div>
          <div class="lpage">10</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Soderlund</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nelson</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Shoemaker</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Paterson</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">SyMAP: a system for discovering and viewing syntenic regions of FPC maps</div>
          <div class="source">Genome Res.</div> 
          <div class="year">2006</div>
          <div class="volume">16</div>
          <div class="fpage">1159</div>
          <div class="lpage">68</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SyMAP%3A%20a%20system%20for%20discovering%20and%20viewing%20syntenic%20regions%20of%20FPC%20maps&amp;author=C.%20Soderlund&amp;author=W.%20Nelson&amp;author=A.%20Shoemaker&amp;author=A.%20Paterson&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1159-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5396706" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5396706" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5396706"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16951135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pop</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Salzberg</div>
               <div class="given-names">S.L.</div>
            </div>
          </div>
          <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>
          <div class="source">Genome Biol.</div> 
          <div class="year">2009</div>
          <div class="volume">10</div>
          <div class="fpage">R25</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Handsaker</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wysoker</div>
               <div class="given-names">A.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">The Sequence Alignment/Map format and SAMtools</div>
          <div class="source">Bioinformatics</div> 
          <div class="year">2009</div>
          <div class="volume">25</div>
          <div class="fpage">2078</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lamesch</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Berardini</div>
               <div class="given-names">T.Z.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">D.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2011</div>
          <div class="comment">
            <a class="link link-uri" href="http://dx.doi.org/10.1093/nar/gkr1090" target="_blank">doi:10.1093/nar/gkr1090</a>
          </div>
        </div></div></div></div><div content-id="DSU021C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Burge</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Prediction of complete gene structures in human genomic DNA.</div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1997</div>
          <div class="volume">268</div>
          <div class="fpage">78</div>
          <div class="lpage">94</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20complete%20gene%20structures%20in%20human%20genomic%20DNA.&amp;author=C.%20Burge&amp;author=S.%20Karlin&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol.&amp;volume=268&amp;pages=78-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.0951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.0951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.0951"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9149143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stanke</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Steinkamp</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Waack</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Morgenstern</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="article-title">AUGUSTUS: a web server for gene finding in eukaryotes</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2004</div>
          <div class="volume">32</div>
          <div class="fpage">W309</div>
          <div class="lpage">12</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20a%20web%20server%20for%20gene%20finding%20in%20eukaryotes&amp;author=M.%20Stanke&amp;author=R.%20Steinkamp&amp;author=S.%20Waack&amp;author=B.%20Morgenstern&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=W309-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15215400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Besemer</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Borodovsky</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2005</div>
          <div class="volume">33</div>
          <div class="fpage">W451</div>
          <div class="lpage">4</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark%3A%20web%20software%20for%20gene%20finding%20in%20prokaryotes%2C%20eukaryotes%20and%20viruses&amp;author=J.%20Besemer&amp;author=M.%20Borodovsky&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W451-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki487" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki487" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki487"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duvick</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fu</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Muppirala</div>
               <div class="given-names">U.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">PlantGDB: a resource for comparative plant genomics</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2008</div>
          <div class="volume">36</div>
          <div class="fpage">D959</div>
          <div class="lpage">65</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlantGDB%3A%20a%20resource%20for%20comparative%20plant%20genomics&amp;author=J.%20Duvick&amp;author=A.%20Fu&amp;author=U.%20Muppirala&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D959-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm1041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm1041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm1041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18063570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jurka</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kapitonov</div>
               <div class="given-names">V.V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pavlicek</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Klonowski</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kohany</div>
               <div class="given-names">O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Walichiewicz</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Repbase Update, a database of eukaryotic repetitive elements, Cytogenet</div>
          <div class="source">Genome Res.</div> 
          <div class="year">2005</div>
          <div class="volume">110</div>
          <div class="fpage">462</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements%2C%20Cytogenet&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;author=P.%20Klonowski&amp;author=O.%20Kohany&amp;author=J.%20Walichiewicz&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div> </div></div></div></div></div><div content-id="DSU021C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Price</div>
               <div class="given-names">A.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jones</div>
               <div class="given-names">N.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pevzner</div>
               <div class="given-names">P.A.</div>
            </div>
          </div>
          <div class="article-title">De novo identification of repeat families in large genomes</div>
          <div class="source">Bioinformatics</div> 
          <div class="year">2005</div>
          <div class="volume">21</div>
          <div class="issue">Suppl 1</div>
          <div class="fpage">i351</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smit</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hubley</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">RepeatModeler Open-1.0</div>
          <div class="year">2008</div>
          <div class="comment">
            <a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a> (9 October 2013, date last accessed)</div>
        </div></div></div></div><div content-id="DSU021C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tempel</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Using and understanding RepeatMasker</div>
          <div class="source">Methods Mol. Biol.</div> 
          <div class="year">2012</div>
          <div class="volume">859</div>
          <div class="fpage">29</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20and%20understanding%20RepeatMasker&amp;author=S.%20Tempel&amp;publication_year=2012&amp;journal=Methods%20Mol.%20Biol.&amp;volume=859&amp;pages=29-51" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smit</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hubley</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Green</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="article-title">RepeatMasker Open-3.0</div>
          <div class="year">1996</div>
          <div class="comment">
            <a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>. (9 October 2013, date last accessed)</div>
        </div></div></div></div><div content-id="DSU021C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">T.-H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tang</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Paterson</div>
               <div class="given-names">A.H.</div>
            </div>
          </div>
          <div class="article-title">PGDD: a database of gene and genome duplication in plants</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2012</div>
          <div class="comment">
            <a class="link link-uri" href="http://dx.doi.org/10.1093/nar/gks1104" target="_blank">doi:10.1093/nar/gks1104</a>
          </div>
        </div></div></div></div><div content-id="DSU021C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="collab">R Development Core Team</div>
          <div class="article-title">R: A language and environment for statistical computing</div>
          <div class="year">2008,</div>
          <div class="publisher-name">R Foundation for Statistical Computing</div>
          <div class="comment">
            <a class="link link-uri" href="http://www.R-project.org">http://www.R-project.org</a>. (2 June 2014, date last accessed)</div>
        </div></div></div></div><div content-id="DSU021C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.C.</div>
            </div>
          </div>
          <div class="article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</div>
          <div class="source">BMC Bioinformatics</div> 
          <div class="year">2004</div>
          <div class="volume">5</div>
          <div class="fpage">113</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20a%20multiple%20sequence%20alignment%20method%20with%20reduced%20time%20and%20space%20complexity&amp;author=R.C.%20Edgar&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15318951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saitou</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>
          <div class="source">Mol. Biol. Evol.</div> 
          <div class="year">1987</div>
          <div class="volume">4</div>
          <div class="fpage">406</div>
          <div class="lpage">25</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Felsenstein</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Confidence limits on phylogenies: an approach using Bootstrap</div>
          <div class="source">Evolution</div> 
          <div class="year">1985</div>
          <div class="volume">39</div>
          <div class="fpage">783</div>
          <div class="lpage">91</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20Bootstrap&amp;author=J.%20Felsenstein&amp;publication_year=1985&amp;journal=Evolution&amp;volume=39&amp;pages=783-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2408678" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2408678" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2408678"> </span></div> </div></div></div></div></div><div content-id="DSU021C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zuckerkandl</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pauling</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bryson</div>
               <div class="given-names">V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vogel</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="source">Evol. Genes Proteins</div> 
          <div class="year">1965</div>
          <div class="publisher-name">Academic Press</div>
          <div class="fpage">97</div>
          <div class="lpage">166</div>
        </div></div></div></div><div content-id="DSU021C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamura</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peterson</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peterson</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Stecher</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>
          <div class="source">Mol. Biol. Evol.</div> 
          <div class="year">2011</div>
          <div class="volume">28</div>
          <div class="fpage">2731</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Letunic</div>
               <div class="given-names">I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bork</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="article-title">Interactive Tree Of Life (iTOL): an online tool for phylogenetic tree display and annotation</div>
          <div class="source">Bioinformatics</div> 
          <div class="year">2007</div>
          <div class="volume">23</div>
          <div class="fpage">127</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interactive%20Tree%20Of%20Life%20(iTOL)%3A%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation&amp;author=I.%20Letunic&amp;author=P.%20Bork&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=127-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17050570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Letunic</div>
               <div class="given-names">I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bork</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="article-title">Interactive Tree Of Life v2: online annotation and display of phylogenetic trees made easy</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2011</div>
          <div class="volume">39</div>
          <div class="fpage">W475</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interactive%20Tree%20Of%20Life%20v2%3A%20online%20annotation%20and%20display%20of%20phylogenetic%20trees%20made%20easy&amp;author=I.%20Letunic&amp;author=P.%20Bork&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=W475-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21470960" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smarda</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bureš</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Smerda</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Horová</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="article-title">Measurements of genomic GC content in plant genomes with flow cytometry: a test for reliability</div>
          <div class="source">New Phytol.</div> 
          <div class="year">2012</div>
          <div class="volume">193</div>
          <div class="fpage">513</div>
          <div class="lpage">21</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measurements%20of%20genomic%20GC%20content%20in%20plant%20genomes%20with%20flow%20cytometry%3A%20a%20test%20for%20reliability&amp;author=P.%20Smarda&amp;author=P.%20Bure%C5%A1&amp;author=J.%20Smerda&amp;author=L.%20Horov%C3%A1&amp;publication_year=2012&amp;journal=New%20Phytol.&amp;volume=193&amp;pages=513-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2011.03942.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.03942.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2011.03942.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22050640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Garg</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Patel</div>
               <div class="given-names">R.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tyagi</div>
               <div class="given-names">A.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jain</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">
            <em>De novo</em> assembly of chickpea transcriptome using short reads for gene discovery and marker identification</div>
          <div class="source">DNA Res.</div> 
          <div class="year">2011</div>
          <div class="volume">18</div>
          <div class="fpage">53</div>
          <div class="lpage">63</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20chickpea%20transcriptome%20using%20short%20reads%20for%20gene%20discovery%20and%20marker%20identification&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=53-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="collab">Arabidopsis Genome Initiative</div>
          <div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>
          <div class="source">Nature</div> 
          <div class="year">2000</div>
          <div class="volume">408</div>
          <div class="fpage">796</div>
          <div class="lpage">815</div>
        <div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jaillon</div>
               <div class="given-names">O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Aury</div>
               <div class="given-names">J.-M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Noel</div>
               <div class="given-names">B.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>
          <div class="source">Nature</div> 
          <div class="year">2007</div>
          <div class="volume">449</div>
          <div class="fpage">463</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20genome%20sequence%20suggests%20ancestral%20hexaploidization%20in%20major%20angiosperm%20phyla&amp;author=O.%20Jaillon&amp;author=J.-M.%20Aury&amp;author=B.%20Noel&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="collab">Consortium, T.T.G</div>
          <div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>
          <div class="source">Nature</div> 
          <div class="year">2012</div>
          <div class="volume">485</div>
          <div class="fpage">635</div>
          <div class="lpage">41</div>
        <div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chagné</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Crowhurst</div>
               <div class="given-names">R.N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Troggio</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Genome-wide SNP detection, validation, and development of an 8K SNP array for apple</div>
          <div class="source">PLoS ONE</div> 
          <div class="year">2012</div>
          <div class="volume">7</div>
          <div class="fpage">e31745</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20detection%2C%20validation%2C%20and%20development%20of%20an%208K%20SNP%20array%20for%20apple&amp;author=D.%20Chagn%C3%A9&amp;author=R.N.%20Crowhurst&amp;author=M.%20Troggio&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e31745" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0031745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0031745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0031745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22363718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shen</div>
               <div class="given-names">Y.-J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jiang</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jin</div>
               <div class="given-names">J.-P.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Development of genome-wide DNA polymorphism database for map-based cloning of rice genes</div>
          <div class="source">Plant Physiol.</div> 
          <div class="year">2004</div>
          <div class="volume">135</div>
          <div class="fpage">1198</div>
          <div class="lpage">205</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genome-wide%20DNA%20polymorphism%20database%20for%20map-based%20cloning%20of%20rice%20genes&amp;author=Y.-J.%20Shen&amp;author=H.%20Jiang&amp;author=J.-P.%20Jin&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=1198-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.038463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.038463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.038463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15266053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ching</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Caldwell</div>
               <div class="given-names">K.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jung</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">SNP frequency, haplotype structure and linkage disequilibrium in elite maize inbred lines</div>
          <div class="source">BMC Genet.</div> 
          <div class="year">2002</div>
          <div class="volume">3</div>
          <div class="fpage">1</div>
          <div class="lpage">14</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20frequency%2C%20haplotype%20structure%20and%20linkage%20disequilibrium%20in%20elite%20maize%20inbred%20lines&amp;author=A.%20Ching&amp;author=K.S.%20Caldwell&amp;author=M.%20Jung&amp;publication_year=2002&amp;journal=BMC%20Genet.&amp;volume=3&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-3-19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-3-19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-3-19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11806754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Velasco</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zharkikh</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Troggio</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">A high quality draft consensus sequence of the genome of a heterozygous grapevine variety</div>
          <div class="source">PloS ONE</div> 
          <div class="year">2007</div>
          <div class="volume">2</div>
          <div class="fpage">e1326</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20quality%20draft%20consensus%20sequence%20of%20the%20genome%20of%20a%20heterozygous%20grapevine%20variety&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=M.%20Troggio&amp;publication_year=2007&amp;journal=PloS%20ONE&amp;volume=2&amp;pages=e1326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0001326" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0001326" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0001326"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18094749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ge</div>
               <div class="given-names">A.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Han</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">X.D.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Characterization of SNPs in strawberry cultivars in China</div>
          <div class="source">Genet. Mol. Res.</div> 
          <div class="year">2013</div>
          <div class="volume">12</div>
          <div class="fpage">639</div>
          <div class="lpage">45</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20SNPs%20in%20strawberry%20cultivars%20in%20China&amp;author=A.J.%20Ge&amp;author=J.%20Han&amp;author=X.D.%20Li&amp;publication_year=2013&amp;journal=Genet.%20Mol.%20Res.&amp;volume=12&amp;pages=639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4238/2013.March.7.2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4238%2F2013.March.7.2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4238%2F2013.March.7.2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23546945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pan</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">S.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Genome sequence and analysis of the tuber crop potato</div>
          <div class="source">Nature</div> 
          <div class="year">2011</div>
          <div class="volume">475</div>
          <div class="fpage">189</div>
          <div class="lpage">95</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20analysis%20of%20the%20tuber%20crop%20potato&amp;author=X.%20Xu&amp;author=S.%20Pan&amp;author=S.%20Cheng&amp;publication_year=2011&amp;journal=Nature&amp;volume=475&amp;pages=189-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Macas</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kejnovský</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Neumann</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Novák</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Koblížková</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vyskot</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="article-title">Next generation sequencing-based analysis of repetitive DNA in the model Dioceous plant <em>Silene latifolia</em></div>
          <div class="source">PLoS ONE</div> 
          <div class="year">2011</div>
          <div class="volume">6</div>
          <div class="fpage">e27335</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing-based%20analysis%20of%20repetitive%20DNA%20in%20the%20model%20Dioceous%20plant%20Silene%20latifolia&amp;author=J.%20Macas&amp;author=E.%20Kejnovsk%C3%BD&amp;author=P.%20Neumann&amp;author=P.%20Nov%C3%A1k&amp;author=A.%20Kobl%C3%AD%C5%BEkov%C3%A1&amp;author=B.%20Vyskot&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e27335" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0027335" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0027335" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0027335"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22096552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Axelsen</div>
               <div class="given-names">J.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yan</div>
               <div class="given-names">K.-K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Maslov</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Parameters of proteome evolution from histograms of amino-acid sequence identities of paralogous proteins</div>
          <div class="source">Biol. Direct</div> 
          <div class="year">2007</div>
          <div class="volume">2</div>
          <div class="fpage">32</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parameters%20of%20proteome%20evolution%20from%20histograms%20of%20amino-acid%20sequence%20identities%20of%20paralogous%20proteins&amp;author=J.B.%20Axelsen&amp;author=K.-K.%20Yan&amp;author=S.%20Maslov&amp;publication_year=2007&amp;journal=Biol.%20Direct&amp;volume=2&amp;pages=32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-2-32" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-2-32" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-2-32"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Van de Peer</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fawcett</div>
               <div class="given-names">J.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Proost</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sterck</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vandepoele</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="article-title">The flowering world: a tale of duplications</div>
          <div class="source">Trends Plant Sci.</div> 
          <div class="year">2009</div>
          <div class="volume">14</div>
          <div class="fpage">680</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20flowering%20world%3A%20a%20tale%20of%20duplications&amp;author=Y.%20Van%20de%20Peer&amp;author=J.A.%20Fawcett&amp;author=S.%20Proost&amp;author=L.%20Sterck&amp;author=K.%20Vandepoele&amp;publication_year=2009&amp;journal=Trends%20Plant%20Sci.&amp;volume=14&amp;pages=680-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2009.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2009.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2009.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19818673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fawcett</div>
               <div class="given-names">J.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Maere</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Van de Peer</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="article-title">Plants with double genomes might have had a better chance to survive the Cretaceous-Tertiary extinction event</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div> 
          <div class="year">2009</div>
          <div class="volume">106</div>
          <div class="fpage">5737</div>
          <div class="lpage">42</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plants%20with%20double%20genomes%20might%20have%20had%20a%20better%20chance%20to%20survive%20the%20Cretaceous-Tertiary%20extinction%20event&amp;author=J.A.%20Fawcett&amp;author=S.%20Maere&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=5737-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0900906106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0900906106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0900906106"> </span></div> </div></div></div></div></div><div content-id="DSU021C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Westhoff</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gowik</div>
               <div class="given-names">U.</div>
            </div>
          </div>
          <div class="article-title">Evolution of C4 phosphoenolpyruvate carboxylase. Genes and proteins: a case study with the genus <em>Flaveria</em></div>
          <div class="source">Ann. Bot.</div> 
          <div class="year">2004</div>
          <div class="volume">93</div>
          <div class="fpage">13</div>
          <div class="lpage">23</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20C4%20phosphoenolpyruvate%20carboxylase.%20Genes%20and%20proteins%3A%20a%20case%20study%20with%20the%20genus%20Flaveria&amp;author=P.%20Westhoff&amp;author=U.%20Gowik&amp;publication_year=2004&amp;journal=Ann.%20Bot.&amp;volume=93&amp;pages=13-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mch003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmch003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmch003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14644912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Christin</div>
               <div class="given-names">P.-A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Salamin</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kellogg</div>
               <div class="given-names">E.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vicentini</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Besnard</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="article-title">Integrating phylogeny into studies of C4 variation in the grasses</div>
          <div class="source">Plant Physiol.</div> 
          <div class="year">2009</div>
          <div class="volume">149</div>
          <div class="fpage">82</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrating%20phylogeny%20into%20studies%20of%20C4%20variation%20in%20the%20grasses&amp;author=P.-A.%20Christin&amp;author=N.%20Salamin&amp;author=E.A.%20Kellogg&amp;author=A.%20Vicentini&amp;author=G.%20Besnard&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=82-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.128553" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.128553" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.128553"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126698" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="collab">Grass Phylogeny Working Group II.</div>
          <div class="article-title">New grass phylogeny resolves deep evolutionary relationships and discovers C4 origins</div>
          <div class="source">New Phytol.</div> 
          <div class="year">2012</div>
          <div class="volume">193</div>
          <div class="fpage">304</div>
          <div class="lpage">12</div>
        <div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2011.03972.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.03972.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2011.03972.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22115274" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Christin</div>
               <div class="given-names">P.-A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sage</div>
               <div class="given-names">T.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Edwards</div>
               <div class="given-names">E.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ogburn</div>
               <div class="given-names">R.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Khoshravesh</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sage</div>
               <div class="given-names">R.F.</div>
            </div>
          </div>
          <div class="article-title">Complex evolutionary transitions and the significance of C3-C4 intermediate forms of photosynthesis in Molluginaceae</div>
          <div class="source">Evol. Int. J. Org. Evol.</div> 
          <div class="year">2011</div>
          <div class="volume">65</div>
          <div class="fpage">643</div>
          <div class="lpage">60</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20evolutionary%20transitions%20and%20the%20significance%20of%20C3-C4%20intermediate%20forms%20of%20photosynthesis%20in%20Molluginaceae&amp;author=P.-A.%20Christin&amp;author=T.L.%20Sage&amp;author=E.J.%20Edwards&amp;author=R.M.%20Ogburn&amp;author=R.%20Khoshravesh&amp;author=R.F.%20Sage&amp;publication_year=2011&amp;journal=Evol.%20Int.%20J.%20Org.%20Evol.&amp;volume=65&amp;pages=643-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1558-5646.2010.01168.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1558-5646.2010.01168.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1558-5646.2010.01168.x"> </span></div> </div></div></div></div></div><div content-id="DSU021C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Galili</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="article-title">The aspartate-family pathway of plants: linking production of essential amino acids with energy and stress regulation</div>
          <div class="source">Plant Signal. Behav.</div> 
          <div class="year">2011</div>
          <div class="volume">6</div>
          <div class="fpage">192</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20aspartate-family%20pathway%20of%20plants%3A%20linking%20production%20of%20essential%20amino%20acids%20with%20energy%20and%20stress%20regulation&amp;author=G.%20Galili&amp;publication_year=2011&amp;journal=Plant%20Signal.%20Behav.&amp;volume=6&amp;pages=192-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.6.2.14425" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.6.2.14425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.6.2.14425"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21512320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kanehisa</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Goto</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sato</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kawashima</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Furumichi</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tanabe</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">Data, information, knowledge and principle: back to metabolism in KEGG</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2014</div>
          <div class="volume">42</div>
          <div class="fpage">D199</div>
          <div class="lpage">205</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Data%2C%20information%2C%20knowledge%20and%20principle%3A%20back%20to%20metabolism%20in%20KEGG&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;author=Y.%20Sato&amp;author=M.%20Kawashima&amp;author=M.%20Furumichi&amp;author=M.%20Tanabe&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D199-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24214961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kanehisa</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Goto</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">KEGG: Kyoto encyclopedia of genes and genomes</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2000</div>
          <div class="volume">28</div>
          <div class="fpage">27</div>
          <div class="lpage">30</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Curien</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Laurencin</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Robert-Genthon</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dumas</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Allosteric monofunctional aspartate kinases from <em>Arabidopsis</em></div>
          <div class="source">FEBS J.</div> 
          <div class="year">2007</div>
          <div class="volume">274</div>
          <div class="fpage">164</div>
          <div class="lpage">76</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allosteric%20monofunctional%20aspartate%20kinases%20from%20Arabidopsis&amp;author=G.%20Curien&amp;author=M.%20Laurencin&amp;author=M.%20Robert-Genthon&amp;author=R.%20Dumas&amp;publication_year=2007&amp;journal=FEBS%20J.&amp;volume=274&amp;pages=164-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1742-4658.2006.05573.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2006.05573.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1742-4658.2006.05573.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17140415" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Galili</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="article-title">Regulation of lysine and threonine synthesis</div>
          <div class="source">Plant Cell.</div> 
          <div class="year">1995</div>
          <div class="volume">7</div>
          <div class="fpage">899</div>
          <div class="lpage">906</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20lysine%20and%20threonine%20synthesis&amp;author=G.%20Galili&amp;publication_year=1995&amp;journal=Plant%20Cell.&amp;volume=7&amp;pages=899-906" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.7.7.899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.7.7.899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.7.7.899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12242392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU021C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU021C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Madden</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2002</div>
          <div class="publisher-loc">USA</div>
          <div class="publisher-name">National Center for Biotechnology Information</div>
          <div class="comment">NCBI Handbook [Internet]. Chapter 16The BLAST Sequence Analysis Tool <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/books/NBK21097/">http://www.ncbi.nlm.nih.gov/books/NBK21097/</a> (26 December 2013, date last accessed)</div>
        </div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754539&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu021/2/dsu021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=RS8wW9S16qC10ZeBwPtMKpIdFXcJ1ty71jwgljMOVgTZ745p1lwudxkFeUmRzdgeWB947GCkvQyVXnLkB15IPL5JSno7EMP-pnMvLdomtR61S2-czTjU6C8M8Tm1LS1UF9YcJTFnzIq5A8786GsCIGXT-6xIDR8-T6eBtr1mJqugzCC8xMSX9NiD0qdzQePC2y8nQk6GYv~ZqiVpo~jPDhmCTpk22kFwLcw-ldtg5f4Ef8IJiCha~Dd9A6Qhe4IMJtpUFAcnJUpBWgMhYvTWqEzKIZZWL6g0zyk1~7gk~I6heKdMSUJh8paB1wsvwYQ5P~C8BuiHSMAVGSkz01p~lw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">838</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754539"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f6%2f585%2f2754539">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000347101100002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F6%2F585%2F2754539" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754539'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Draft Genome and Transcriptome of Amaranthus hypochondriacus: A C4 Dicot Producing High-Lysine Edible Pseudo-Cereal" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f585%2f2754539">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

