

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/5/299/420917","@type":"ScholarlyArticle","name":"Nucleotide sequence analysis of the enterotoxigenic  Escherichia coli  Ent plasmid","datePublished":"2009-09-18","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2009-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp015","keywords":["enterotoxin gene","pathogenicity islet","virulence plasmid"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/5/299/420917","author":[{"name":"Ochi, Sadayuki","affiliation":"1  Department of Microbiology ,  School of Medicine, Fujita Health University ,  Toyoake, Aichi 470-1192 ,  Japan","@type":"Person"},{"name":"Shimizu, Tohru","affiliation":"2  Department of Microbiology ,  Graduate School of Medical Science, Kanazawa University ,  Kanazawa, Ishikawa 920-8640 ,  Japan","@type":"Person"},{"name":"Ohtani, Kaori","affiliation":"2  Department of Microbiology ,  Graduate School of Medical Science, Kanazawa University ,  Kanazawa, Ishikawa 920-8640 ,  Japan","@type":"Person"},{"name":"Ichinose, Yoshio","affiliation":"3  Department of Eco- epidemiology ,  Kenya Research Station, Institute of Tropical Medicine, Nagasaki University ,  Sakamoto, Nagasaki 852-8523 ,  Japan","@type":"Person"},{"name":"Arimitsu, Hideyuki","affiliation":"1  Department of Microbiology ,  School of Medicine, Fujita Health University ,  Toyoake, Aichi 470-1192 ,  Japan","@type":"Person"},{"name":"Tsukamoto, Kentaro","affiliation":"1  Department of Microbiology ,  School of Medicine, Fujita Health University ,  Toyoake, Aichi 470-1192 ,  Japan","@type":"Person"},{"name":"Kato, Michio","affiliation":"1  Department of Microbiology ,  School of Medicine, Fujita Health University ,  Toyoake, Aichi 470-1192 ,  Japan","@type":"Person"},{"name":"Tsuji, Takao","affiliation":"1  Department of Microbiology ,  School of Medicine, Fujita Health University ,  Toyoake, Aichi 470-1192 ,  Japan","@type":"Person"}],"description":"Abstract.  We report here the complete nucleotide sequence of pEntH10407 (65 147 bp), an enterotoxigenic Escherichia coli enterotoxin plasmid (Ent plasmid), whi","pageStart":"299","pageEnd":"309","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Nucleotide sequence analysis of the enterotoxigenic  Escherichia coli  Ent plasmid","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Nucleotide sequence analysis of the enterotoxigenic  Escherichia coli  Ent plasmid" />
    <meta property="og:description" content="Abstract.  We report here the complete nucleotide sequence of pEntH10407 (65 147 bp), an enterotoxigenic Escherichia coli enterotoxin plasmid (Ent plasmid), whi" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/5/299/420917" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-09-18" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ochi, Sadayuki" /><meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, Fujita Health University, Toyoake, Aichi 470-1192, Japan" /><meta name="citation_author" content="Shimizu, Tohru" /><meta name="citation_author_institution" content="Department of Microbiology, Graduate School of Medical Science, Kanazawa University, Kanazawa, Ishikawa 920-8640, Japan" /><meta name="citation_author" content="Ohtani, Kaori" /><meta name="citation_author_institution" content="Department of Microbiology, Graduate School of Medical Science, Kanazawa University, Kanazawa, Ishikawa 920-8640, Japan" /><meta name="citation_author" content="Ichinose, Yoshio" /><meta name="citation_author_institution" content="Department of Eco- epidemiology, Kenya Research Station, Institute of Tropical Medicine, Nagasaki University, Sakamoto, Nagasaki 852-8523, Japan" /><meta name="citation_author" content="Arimitsu, Hideyuki" /><meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, Fujita Health University, Toyoake, Aichi 470-1192, Japan" /><meta name="citation_author" content="Tsukamoto, Kentaro" /><meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, Fujita Health University, Toyoake, Aichi 470-1192, Japan" /><meta name="citation_author" content="Kato, Michio" /><meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, Fujita Health University, Toyoake, Aichi 470-1192, Japan" /><meta name="citation_author" content="Tsuji, Takao" /><meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, Fujita Health University, Toyoake, Aichi 470-1192, Japan" /><meta name="citation_title" content="Nucleotide sequence analysis of the enterotoxigenic Escherichia coli Ent plasmid" /><meta name="citation_firstpage" content="299" /><meta name="citation_lastpage" content="309" /><meta name="citation_doi" content="10.1093/dnares/dsp015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2009/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Plasmid-controlled colonization factor associated with virulence in Escherichia coli enterotoxigenic for humans; Infect. Immun.;  citation_year=1975;  citation_volume=12;  citation_pages=656-67; " /><meta name="citation_reference" content="citation_title=Enterotoxigenic Escherichia coli in developing countries: epidemiology, microbiology, clinical features, treatment, and prevention; Clin. Microbiol. Rev.;  citation_year=2005;  citation_volume=18;  citation_pages=465-83; " /><meta name="citation_reference" content="citation_title=Enterotoxigenic Escherichia coli isolated from patients with severe cholera-like disease; J. Infect. Dis.;  citation_year=1971;  citation_volume=123;  citation_pages=378-85; " /><meta name="citation_reference" content="citation_title=Actions of cholera toxin and the prevention and treatment of cholera; Nature;  citation_year=1981;  citation_volume=292;  citation_pages=413-7; " /><meta name="citation_reference" content="citation_title=Molecular heterogeneity of heat-labile enterotoxins from human and porcine enterotoxigenic Escherichia coli; Infect. Immun.;  citation_year=1982;  citation_volume=38;  citation_pages=444-8; " /><meta name="citation_reference" content="citation_title=Plasmids coding for heat-labile enterotoxin production isolated from Escherichia coli O78: comparison of properties; J. Bacteriol.;  citation_year=1980;  citation_volume=143;  citation_pages=158-67; " /><meta name="citation_reference" content="citation_title=Plasmid association and nucleotide sequence relationships of two genes encoding heat-stable enterotoxin production in Escherichia coli H-10407; J. Bacteriol.;  citation_year=1983;  citation_volume=156;  citation_pages=441-3; " /><meta name="citation_reference" content="citation_title=The relationship between two apparently different enterotoxins produced by enteropathogenic strains of Escherichia coli of porcine origin; J. Med. Microbiol.;  citation_year=1970;  citation_volume=3;  citation_pages=387-401; " /><meta name="citation_reference" content="citation_title=Colonization factors of human enterotoxigenic Escherichia coli (ETEC); Trends Microbiol.;  citation_year=1996;  citation_volume=4;  citation_pages=444-52; " /><meta name="citation_reference" content="citation_title=Mapping of a plasmid, coding for colonization, factor antigen I and heat-stable enterotoxin production, isolated from an enterotoxigenic strain of Escherichia coli; J. Bacteriol.;  citation_year=1982;  citation_volume=149;  citation_pages=264-75; " /><meta name="citation_reference" content="citation_title=Molecular comparison of plasmids encoding heat-labile enterotoxin isolated from Escherichia coli strains of human origin; J. Bacteriol.;  citation_year=1980;  citation_volume=143;  citation_pages=168-75; " /><meta name="citation_reference" content="citation_title=Plasmids of enterotoxigenic Escherichia coli H10407: evidence for two heat-stable enterotoxin genes and a conjugal transfer system; J. Bacteriol.;  citation_year=1983;  citation_volume=153;  citation_pages=1352-60; " /><meta name="citation_reference" content="citation_title=The pCoo plasmid of enterotoxigenic Escherichia coli is a mosaic cointegrate; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=6509-16; " /><meta name="citation_reference" content="citation_title=Plasmid-mediated properties of a heat-stable enterotoxin-producing Escherichia coli associated with infantile diarrhea; Infect. Immun.;  citation_year=1976;  citation_volume=14;  citation_pages=403-7; " /><meta name="citation_reference" content="citation_title=Horizontal transfer of CS1 pilin genes of enterotoxigenic Escherichia coli; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=3230-7; " /><meta name="citation_reference" content="citation_title=Tn5 in vitro transposition; J. Biol. Chem.;  citation_year=1998;  citation_volume=273;  citation_pages=7367-74; " /><meta name="citation_reference" content="citation_title=Primary structure of heat-labile enterotoxin produced by Escherichia coli pathogenic for humans; J. Biol. Chem.;  citation_year=1984;  citation_volume=259;  citation_pages=5037-44; " /><meta name="citation_reference" content="citation_title=Evolutionary origin of pathogenic determinants in enterotoxigenic Escherichia coli and Vibrio cholerae O1; J. Bacteriol.;  citation_year=1987;  citation_volume=169;  citation_pages=1352-7; " /><meta name="citation_reference" content="citation_title=Escherichia coli heat-labile enterotoxin genes are flanked by repeated deoxyribonucleic acid sequences; J. Bacteriol.;  citation_year=1981;  citation_volume=145;  citation_pages=850-60; " /><meta name="citation_reference" content="citation_title=Pathogenicity islands: a molecular toolbox for bacterial virulence; Cell Microbiol.;  citation_year=2006;  citation_volume=8;  citation_pages=1707-19; " /><meta name="citation_reference" content="citation_title=Insertion and deletion mutations in the repA4 region of the IncFII plasmid NR1 cause unstable inheritance; J. Bacteriol.;  citation_year=1993;  citation_volume=175;  citation_pages=5350-8; " /><meta name="citation_reference" content="citation_title=Genes and sites involved in replication and incompatibility of an R100 plasmid derivative based on nucleotide sequence analysis; Mol. Gen. Genet.;  citation_year=1980;  citation_volume=179;  citation_pages=527-37; " /><meta name="citation_reference" content="citation_title=Replication and control of circular bacterial plasmids; Microbiol. Mol. Biol. Rev.;  citation_year=1998;  citation_volume=62;  citation_pages=434-64; " /><meta name="citation_reference" content="citation_title=Mechanism of postsegregational killing by the hok gene product of the parB system of plasmid R1 and its homology with the relF gene product of the E. coli relB operon; EMBO J.;  citation_year=1986;  citation_volume=5;  citation_pages=2023-9; " /><meta name="citation_reference" content="citation_title=Mechanism of post-segregational killing: Sok antisense RNA interacts with Hok mRNA via its 5′-end single-stranded leader and competes with the 3′-end of Hok mRNA for binding to the mok translational initiation region; EMBO J.;  citation_year=1994;  citation_volume=13;  citation_pages=1960-8; " /><meta name="citation_reference" content="citation_title=Mechanism of DNA segregation in prokaryotes: ParM partitioning protein of plasmid R1 co-localizes with its replicon during the cell cycle; EMBO J.;  citation_year=1999;  citation_volume=18;  citation_pages=4076-84; " /><meta name="citation_reference" content="citation_title=Mechanism of DNA segregation in prokaryotes: replicon pairing by parC of plasmid R1; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=8550-5; " /><meta name="citation_reference" content="citation_title=The complete sequence and analysis of the large virulence plasmid pSS of Shigella sonnei; Plasmid;  citation_year=2005;  citation_volume=54;  citation_pages=149-59; " /><meta name="citation_reference" content="citation_title=A family of stability determinants in pathogenic bacteria; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=6415-8; " /><meta name="citation_reference" content="citation_title=Comparison of proteins involved in pilus synthesis and mating pair stabilization from the related plasmids F and R100-1: insights into the mechanism of conjugation; J. Bacteriol.;  citation_year=1999;  citation_volume=181;  citation_pages=5149-59; " /><meta name="citation_reference" content="citation_title=Nucleotide sequence analysis of the enteropathogenic Escherichia coli adherence factor plasmid pMAR7; Infect. Immun.;  citation_year=2006;  citation_volume=74;  citation_pages=5408-13; " /><meta name="citation_reference" content="citation_title=Analysis of the sequence and gene products of the transfer region of the F sex factor; Microbiol. Rev.;  citation_year=1994;  citation_volume=58;  citation_pages=162-210; " /><meta name="citation_reference" content="citation_title=F factor conjugation is a true type IV secretion system; FEMS Microbiol. Lett.;  citation_year=2003;  citation_volume=224;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Capacity of aquatic bacteria to act as recipients of plasmid DNA; Appl. Environ. Microbiol.;  citation_year=1988;  citation_volume=54;  citation_pages=115-7; " /><meta name="citation_reference" content="citation_title=Genetic analysis of the role of the transfer gene, traN, of the F and R100-1 plasmids in mating pair stabilization during conjugation; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=4036-43; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/5/299/1166260/dsp015.pdf" /><meta name="description" content="Abstract.  We report here the complete nucleotide sequence of pEntH10407 (65 147 bp), an enterotoxigenic Escherichia coli enterotoxin plasmid (Ent plasmid), whi" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/5/299/420917" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/5/299/420917" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/5/299/420917");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp015';
    </script>
<title>
	Nucleotide sequence analysis of the enterotoxigenic Escherichia coli Ent plasmid | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./420917" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbNaPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPY2hpLCBTYWRheXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBTY2hvb2wgb2YgTWVkaWNpbmUsIEZ1aml0YSBIZWFsdGggVW5pdmVyc2l0eSwgVG95b2FrZSwgQWljaGkgNDcwLTExOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbWl6dSwgVG9ocnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljYWwgU2NpZW5jZSwgS2FuYXphd2EgVW5pdmVyc2l0eSwgS2FuYXphd2EsIElzaGlrYXdhIDkyMC04NjQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9odGFuaSwgS2FvcmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljYWwgU2NpZW5jZSwgS2FuYXphd2EgVW5pdmVyc2l0eSwgS2FuYXphd2EsIElzaGlrYXdhIDkyMC04NjQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkljaGlub3NlLCBZb3NoaW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEVjby0gZXBpZGVtaW9sb2d5LCBLZW55YSBSZXNlYXJjaCBTdGF0aW9uLCBJbnN0aXR1dGUgb2YgVHJvcGljYWwgTWVkaWNpbmUsIE5hZ2FzYWtpIFVuaXZlcnNpdHksIFNha2Ftb3RvLCBOYWdhc2FraSA4NTItODUyMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcmltaXRzdSwgSGlkZXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgU2Nob29sIG9mIE1lZGljaW5lLCBGdWppdGEgSGVhbHRoIFVuaXZlcnNpdHksIFRveW9ha2UsIEFpY2hpIDQ3MC0xMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdWthbW90bywgS2VudGFybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBTY2hvb2wgb2YgTWVkaWNpbmUsIEZ1aml0YSBIZWFsdGggVW5pdmVyc2l0eSwgVG95b2FrZSwgQWljaGkgNDcwLTExOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgTWljaGlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNaWNyb2Jpb2xvZ3ksIFNjaG9vbCBvZiBNZWRpY2luZSwgRnVqaXRhIEhlYWx0aCBVbml2ZXJzaXR5LCBUb3lvYWtlLCBBaWNoaSA0NzAtMTE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3VqaSwgVGFrYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgU2Nob29sIG9mIE1lZGljaW5lLCBGdWppdGEgSGVhbHRoIFVuaXZlcnNpdHksIFRveW9ha2UsIEFpY2hpIDQ3MC0xMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTnVjbGVvdGlkZSBzZXF1ZW5jZSBhbmFseXNpcyBvZiB0aGUgZW50ZXJvdG94aWdlbmljIEVzY2hlcmljaGlhIGNvbGkgRW50IHBsYXNtaWQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyOTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjMwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcDAxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA5LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhc21pZC1jb250cm9sbGVkIGNvbG9uaXphdGlvbiBmYWN0b3IgYXNzb2NpYXRlZCB3aXRoIHZpcnVsZW5jZSBpbiBFc2NoZXJpY2hpYSBjb2xpIGVudGVyb3RveGlnZW5pYyBmb3IgaHVtYW5zOyBJbmZlY3QuIEltbXVuLjsgIGNpdGF0aW9uX3llYXI9MTk3NTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW50ZXJvdG94aWdlbmljIEVzY2hlcmljaGlhIGNvbGkgaW4gZGV2ZWxvcGluZyBjb3VudHJpZXM6IGVwaWRlbWlvbG9neSwgbWljcm9iaW9sb2d5LCBjbGluaWNhbCBmZWF0dXJlcywgdHJlYXRtZW50LCBhbmQgcHJldmVudGlvbjsgQ2xpbi4gTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NDY1LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnRlcm90b3hpZ2VuaWMgRXNjaGVyaWNoaWEgY29saSBpc29sYXRlZCBmcm9tIHBhdGllbnRzIHdpdGggc2V2ZXJlIGNob2xlcmEtbGlrZSBkaXNlYXNlOyBKLiBJbmZlY3QuIERpcy47ICBjaXRhdGlvbl95ZWFyPTE5NzE7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9Mzc4LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3Rpb25zIG9mIGNob2xlcmEgdG94aW4gYW5kIHRoZSBwcmV2ZW50aW9uIGFuZCB0cmVhdG1lbnQgb2YgY2hvbGVyYTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTI5MjsgIGNpdGF0aW9uX3BhZ2VzPTQxMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgaGV0ZXJvZ2VuZWl0eSBvZiBoZWF0LWxhYmlsZSBlbnRlcm90b3hpbnMgZnJvbSBodW1hbiBhbmQgcG9yY2luZSBlbnRlcm90b3hpZ2VuaWMgRXNjaGVyaWNoaWEgY29saTsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz00NDQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhc21pZHMgY29kaW5nIGZvciBoZWF0LWxhYmlsZSBlbnRlcm90b3hpbiBwcm9kdWN0aW9uIGlzb2xhdGVkIGZyb20gRXNjaGVyaWNoaWEgY29saSBPNzg6IGNvbXBhcmlzb24gb2YgcHJvcGVydGllczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDM7ICBjaXRhdGlvbl9wYWdlcz0xNTgtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYXNtaWQgYXNzb2NpYXRpb24gYW5kIG51Y2xlb3RpZGUgc2VxdWVuY2UgcmVsYXRpb25zaGlwcyBvZiB0d28gZ2VuZXMgZW5jb2RpbmcgaGVhdC1zdGFibGUgZW50ZXJvdG94aW4gcHJvZHVjdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpIEgtMTA0MDc7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9NDQxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWxhdGlvbnNoaXAgYmV0d2VlbiB0d28gYXBwYXJlbnRseSBkaWZmZXJlbnQgZW50ZXJvdG94aW5zIHByb2R1Y2VkIGJ5IGVudGVyb3BhdGhvZ2VuaWMgc3RyYWlucyBvZiBFc2NoZXJpY2hpYSBjb2xpIG9mIHBvcmNpbmUgb3JpZ2luOyBKLiBNZWQuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5NzA7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny00MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbG9uaXphdGlvbiBmYWN0b3JzIG9mIGh1bWFuIGVudGVyb3RveGlnZW5pYyBFc2NoZXJpY2hpYSBjb2xpIChFVEVDKTsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQ0NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBvZiBhIHBsYXNtaWQsIGNvZGluZyBmb3IgY29sb25pemF0aW9uLCBmYWN0b3IgYW50aWdlbiBJIGFuZCBoZWF0LXN0YWJsZSBlbnRlcm90b3hpbiBwcm9kdWN0aW9uLCBpc29sYXRlZCBmcm9tIGFuIGVudGVyb3RveGlnZW5pYyBzdHJhaW4gb2YgRXNjaGVyaWNoaWEgY29saTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0yNjQtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjb21wYXJpc29uIG9mIHBsYXNtaWRzIGVuY29kaW5nIGhlYXQtbGFiaWxlIGVudGVyb3RveGluIGlzb2xhdGVkIGZyb20gRXNjaGVyaWNoaWEgY29saSBzdHJhaW5zIG9mIGh1bWFuIG9yaWdpbjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDM7ICBjaXRhdGlvbl9wYWdlcz0xNjgtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYXNtaWRzIG9mIGVudGVyb3RveGlnZW5pYyBFc2NoZXJpY2hpYSBjb2xpIEgxMDQwNzogZXZpZGVuY2UgZm9yIHR3byBoZWF0LXN0YWJsZSBlbnRlcm90b3hpbiBnZW5lcyBhbmQgYSBjb25qdWdhbCB0cmFuc2ZlciBzeXN0ZW07IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl92b2x1bWU9MTUzOyAgY2l0YXRpb25fcGFnZXM9MTM1Mi02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBDb28gcGxhc21pZCBvZiBlbnRlcm90b3hpZ2VuaWMgRXNjaGVyaWNoaWEgY29saSBpcyBhIG1vc2FpYyBjb2ludGVncmF0ZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz02NTA5LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFzbWlkLW1lZGlhdGVkIHByb3BlcnRpZXMgb2YgYSBoZWF0LXN0YWJsZSBlbnRlcm90b3hpbi1wcm9kdWNpbmcgRXNjaGVyaWNoaWEgY29saSBhc3NvY2lhdGVkIHdpdGggaW5mYW50aWxlIGRpYXJyaGVhOyBJbmZlY3QuIEltbXVuLjsgIGNpdGF0aW9uX3llYXI9MTk3NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQwMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3Jpem9udGFsIHRyYW5zZmVyIG9mIENTMSBwaWxpbiBnZW5lcyBvZiBlbnRlcm90b3hpZ2VuaWMgRXNjaGVyaWNoaWEgY29saTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xODY7ICBjaXRhdGlvbl9wYWdlcz0zMjMwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRuNSBpbiB2aXRybyB0cmFuc3Bvc2l0aW9uOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNzM7ICBjaXRhdGlvbl9wYWdlcz03MzY3LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltYXJ5IHN0cnVjdHVyZSBvZiBoZWF0LWxhYmlsZSBlbnRlcm90b3hpbiBwcm9kdWNlZCBieSBFc2NoZXJpY2hpYSBjb2xpIHBhdGhvZ2VuaWMgZm9yIGh1bWFuczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MjU5OyAgY2l0YXRpb25fcGFnZXM9NTAzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IG9yaWdpbiBvZiBwYXRob2dlbmljIGRldGVybWluYW50cyBpbiBlbnRlcm90b3hpZ2VuaWMgRXNjaGVyaWNoaWEgY29saSBhbmQgVmlicmlvIGNob2xlcmFlIE8xOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE2OTsgIGNpdGF0aW9uX3BhZ2VzPTEzNTItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNjaGVyaWNoaWEgY29saSBoZWF0LWxhYmlsZSBlbnRlcm90b3hpbiBnZW5lcyBhcmUgZmxhbmtlZCBieSByZXBlYXRlZCBkZW94eXJpYm9udWNsZWljIGFjaWQgc2VxdWVuY2VzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE0NTsgIGNpdGF0aW9uX3BhZ2VzPTg1MC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0aG9nZW5pY2l0eSBpc2xhbmRzOiBhIG1vbGVjdWxhciB0b29sYm94IGZvciBiYWN0ZXJpYWwgdmlydWxlbmNlOyBDZWxsIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE3MDctMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2VydGlvbiBhbmQgZGVsZXRpb24gbXV0YXRpb25zIGluIHRoZSByZXBBNCByZWdpb24gb2YgdGhlIEluY0ZJSSBwbGFzbWlkIE5SMSBjYXVzZSB1bnN0YWJsZSBpbmhlcml0YW5jZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzU7ICBjaXRhdGlvbl9wYWdlcz01MzUwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVzIGFuZCBzaXRlcyBpbnZvbHZlZCBpbiByZXBsaWNhdGlvbiBhbmQgaW5jb21wYXRpYmlsaXR5IG9mIGFuIFIxMDAgcGxhc21pZCBkZXJpdmF0aXZlIGJhc2VkIG9uIG51Y2xlb3RpZGUgc2VxdWVuY2UgYW5hbHlzaXM7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9NTI3LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBsaWNhdGlvbiBhbmQgY29udHJvbCBvZiBjaXJjdWxhciBiYWN0ZXJpYWwgcGxhc21pZHM7IE1pY3JvYmlvbC4gTW9sLiBCaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9NDM0LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc20gb2YgcG9zdHNlZ3JlZ2F0aW9uYWwga2lsbGluZyBieSB0aGUgaG9rIGdlbmUgcHJvZHVjdCBvZiB0aGUgcGFyQiBzeXN0ZW0gb2YgcGxhc21pZCBSMSBhbmQgaXRzIGhvbW9sb2d5IHdpdGggdGhlIHJlbEYgZ2VuZSBwcm9kdWN0IG9mIHRoZSBFLiBjb2xpIHJlbEIgb3Blcm9uOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yMDIzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiBwb3N0LXNlZ3JlZ2F0aW9uYWwga2lsbGluZzogU29rIGFudGlzZW5zZSBSTkEgaW50ZXJhY3RzIHdpdGggSG9rIG1STkEgdmlhIGl0cyA14oCyLWVuZCBzaW5nbGUtc3RyYW5kZWQgbGVhZGVyIGFuZCBjb21wZXRlcyB3aXRoIHRoZSAz4oCyLWVuZCBvZiBIb2sgbVJOQSBmb3IgYmluZGluZyB0byB0aGUgbW9rIHRyYW5zbGF0aW9uYWwgaW5pdGlhdGlvbiByZWdpb247IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xOTYwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiBETkEgc2VncmVnYXRpb24gaW4gcHJva2FyeW90ZXM6IFBhck0gcGFydGl0aW9uaW5nIHByb3RlaW4gb2YgcGxhc21pZCBSMSBjby1sb2NhbGl6ZXMgd2l0aCBpdHMgcmVwbGljb24gZHVyaW5nIHRoZSBjZWxsIGN5Y2xlOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NDA3Ni04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtIG9mIEROQSBzZWdyZWdhdGlvbiBpbiBwcm9rYXJ5b3RlczogcmVwbGljb24gcGFpcmluZyBieSBwYXJDIG9mIHBsYXNtaWQgUjE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz04NTUwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBzZXF1ZW5jZSBhbmQgYW5hbHlzaXMgb2YgdGhlIGxhcmdlIHZpcnVsZW5jZSBwbGFzbWlkIHBTUyBvZiBTaGlnZWxsYSBzb25uZWk7IFBsYXNtaWQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xNDktNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFtaWx5IG9mIHN0YWJpbGl0eSBkZXRlcm1pbmFudHMgaW4gcGF0aG9nZW5pYyBiYWN0ZXJpYTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xODA7ICBjaXRhdGlvbl9wYWdlcz02NDE1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgcHJvdGVpbnMgaW52b2x2ZWQgaW4gcGlsdXMgc3ludGhlc2lzIGFuZCBtYXRpbmcgcGFpciBzdGFiaWxpemF0aW9uIGZyb20gdGhlIHJlbGF0ZWQgcGxhc21pZHMgRiBhbmQgUjEwMC0xOiBpbnNpZ2h0cyBpbnRvIHRoZSBtZWNoYW5pc20gb2YgY29uanVnYXRpb247IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTgxOyAgY2l0YXRpb25fcGFnZXM9NTE0OS01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvdGlkZSBzZXF1ZW5jZSBhbmFseXNpcyBvZiB0aGUgZW50ZXJvcGF0aG9nZW5pYyBFc2NoZXJpY2hpYSBjb2xpIGFkaGVyZW5jZSBmYWN0b3IgcGxhc21pZCBwTUFSNzsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz01NDA4LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgc2VxdWVuY2UgYW5kIGdlbmUgcHJvZHVjdHMgb2YgdGhlIHRyYW5zZmVyIHJlZ2lvbiBvZiB0aGUgRiBzZXggZmFjdG9yOyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0xNjItMjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GIGZhY3RvciBjb25qdWdhdGlvbiBpcyBhIHRydWUgdHlwZSBJViBzZWNyZXRpb24gc3lzdGVtOyBGRU1TIE1pY3JvYmlvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjI0OyAgY2l0YXRpb25fcGFnZXM9MS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FwYWNpdHkgb2YgYXF1YXRpYyBiYWN0ZXJpYSB0byBhY3QgYXMgcmVjaXBpZW50cyBvZiBwbGFzbWlkIEROQTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTExNS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuYWx5c2lzIG9mIHRoZSByb2xlIG9mIHRoZSB0cmFuc2ZlciBnZW5lLCB0cmFOLCBvZiB0aGUgRiBhbmQgUjEwMC0xIHBsYXNtaWRzIGluIG1hdGluZyBwYWlyIHN0YWJpbGl6YXRpb24gZHVyaW5nIGNvbmp1Z2F0aW9uOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTQwMzYtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNi81LzI5OS8xMTY2MjYwL2RzcDAxNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgcmVwb3J0IGhlcmUgdGhlIGNvbXBsZXRlIG51Y2xlb3RpZGUgc2VxdWVuY2Ugb2YgcEVudEgxMDQwNyAoNjUgMTQ3IGJwKSwgYW4gZW50ZXJvdG94aWdlbmljIEVzY2hlcmljaGlhIGNvbGkgZW50ZXJvdG94aW4gcGxhc21pZCAoRW50IHBsYXNtaWQpLCB3aGkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQyMDkxNyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGS35SAsVXLSIgmMbsQCyp6kUPEmyA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f299%2f420917" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f299%2f420917" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/5/1/m_cover.gif?Expires=2147483647&amp;Signature=vHjcTqPezdHoAGnGzycKsSlFc8Gej6TP70uQwBn0qFpZszwo~DxESYey6jXrFQdL3azkQleuWxsU9Gx6KgyZrhpkI-FY8mT2FNSaEQgMVQBUD5PJfR5Xh4MSBpg01w16~XqS5ONKzDqUEjOWBe6gjnlQEKg5ezp6b7HYSOApJ92p6HDMJ7E-oFo80vPo7Cj5zQocrpOPrViJaukjgYrI1SV9FCGcupU2HBYOlUaGdNIt9AncDj7Iog4iBwhVbtB5erQ~pCAdLkoVAnXEM1p3hlOv1w~m1J04HZNiAF0MD5UIZZKnBlqALRrMuf1inukdVJZ6nbB7tWN485~NoYhI0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/5">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6237567">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237567">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237592">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237592">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237594">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237594">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237596">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237596">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/5/287/421945">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Nucleotide sequence analysis of the enterotoxigenic <em>Escherichia coli</em> Ent plasmid
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sadayuki Ochi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">School of Medicine, Fujita Health University</div>, <div class="addr-line">Toyoake, Aichi 470-1192</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 562-93-2433. Email: <a href="mailto:ochi@fujita-hu.ac.jp" target="_blank">ochi@fujita-hu.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sadayuki+Ochi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ochi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ochi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sadayuki Ochi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tohru Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Graduate School of Medical Science, Kanazawa University</div>, <div class="addr-line">Kanazawa, Ishikawa 920-8640</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tohru+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tohru Shimizu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kaori Ohtani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Graduate School of Medical Science, Kanazawa University</div>, <div class="addr-line">Kanazawa, Ishikawa 920-8640</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kaori+Ohtani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtani K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtani K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kaori Ohtani</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshio Ichinose</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Eco- epidemiology</div>, <div class="insititution">Kenya Research Station, Institute of Tropical Medicine, Nagasaki University</div>, <div class="addr-line">Sakamoto, Nagasaki 852-8523</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshio+Ichinose">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ichinose Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ichinose Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshio Ichinose</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideyuki Arimitsu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">School of Medicine, Fujita Health University</div>, <div class="addr-line">Toyoake, Aichi 470-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideyuki+Arimitsu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arimitsu H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arimitsu H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideyuki Arimitsu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kentaro Tsukamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">School of Medicine, Fujita Health University</div>, <div class="addr-line">Toyoake, Aichi 470-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kentaro+Tsukamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsukamoto K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsukamoto K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kentaro Tsukamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michio Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">School of Medicine, Fujita Health University</div>, <div class="addr-line">Toyoake, Aichi 470-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michio+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michio Kato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takao Tsuji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">School of Medicine, Fujita Health University</div>, <div class="addr-line">Toyoake, Aichi 470-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takao+Tsuji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuji T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuji T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takao Tsuji</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 299–309, <a href='https://doi.org/10.1093/dnares/dsp015'>https://doi.org/10.1093/dnares/dsp015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">18 September 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 March 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 August 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="420917" href="/dnaresearch/article-pdf/16/5/299/1166260/dsp015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sadayuki Ochi, Tohru Shimizu, Kaori Ohtani, Yoshio Ichinose, Hideyuki Arimitsu, Kentaro Tsukamoto, Michio Kato, Takao Tsuji;  Nucleotide sequence analysis of the enterotoxigenic <em>Escherichia coli</em> Ent plasmid, <em>DNA Research</em>, Volume 16, Issue 5, 1 October 2009, Pages 299–309, <a href='https://doi.org/10.1093/dnares/dsp015'>https://doi.org/10.1093/dnares/dsp015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/420917?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/420917?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/420917?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/420917?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/420917?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/420917?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/420917?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Nucleotide%20sequence%20analysis%20of%20the%20enterotoxigenic%20%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B%20Ent%20plasmid&amp;publicationDate=2009-09-18&amp;volumeNum=16&amp;issueNum=5&amp;author=Ochi%2C%20Sadayuki%3B%20Shimizu%2C%20Tohru&amp;startPage=299&amp;endPage=309&amp;contentId=10.1093%2Fdnares%2Fdsp015&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f299%2f420917" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6237567 id="6237567" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We report here the complete nucleotide sequence of pEntH10407 (65 147 bp), an enterotoxigenic <em>Escherichia coli</em> enterotoxin plasmid (Ent plasmid), which is self-transmissible at low frequency. Within the plasmid, we identified 100 open reading frames (ORFs) which could encode polypeptides. These ORFs included regions encoding heat-labile (LT) and heat-stable (STIa) enterotoxins, regions encoding tools for plasmid replication and an incomplete <em>tra</em> (conjugation) region. The LT and STIa region was located 13.5 kb apart and was surrounded by three IS<em>1</em>s and an IS<em>600</em> in opposite reading orientations, indicating that the enterotoxin genes may have been horizontally transferred into the plasmid. We identified a single RepFIIA replication region (2.0 kb) including RepA proteins similar to RepA1, RepA2, RepA3 and RepA4. The incomplete <em>tra</em> region was made up of 17 <em>tra</em> genes, which were nearly identical to the corresponding genes of R100, and showed evidence of multiple insertions of IS<em>Ec8</em> and IS<em>Ec8</em>-like elements. These data suggest that pEntH10407 has the mosaic nature characteristic of bacterial virulence plasmids, which contains information about its evolution. Although the <em>tra</em> genes might originally have rendered pEntH10407 self-transferable to the same degree as R100, multiple insertion events have occurred in the <em>tra</em> region of pEntH10407 to make it less mobile. Another self-transmissible plasmid might help pEntH10407 to transfer efficiently into H10407 strain. In this paper, we suggest another possibility: that the enterotoxigenic H10407 strain might be formed by auto-transfer of pEntH10407 at a low rate using the incomplete <em>tra</em> region.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;enterotoxin gene&quot;">enterotoxin gene</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;pathogenicity islet&quot;">pathogenicity islet</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;virulence plasmid&quot;">virulence plasmid</a></div><p>Enterotoxigenic <em>Escherichia coli</em> (ETEC) strains are important causes of acute and persistent diarrhoea in infants in developing countries.<sup><a reveal-id="DSP015C1  DSP015C3" data-open="DSP015C1  DSP015C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> ETEC strains produce two kinds of classical virulence factors. One is an enterotoxin such as heat-labile (LT) or heat-stable (ST) enterotoxin, which causes diarrhoea. LT consists of two components (A and B subunits), and is immunologically, biochemically and genetically similar to cholera toxin.<sup><a reveal-id="DSP015C4" data-open="DSP015C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSP015C5" data-open="DSP015C5" class="link link-ref link-reveal xref-bibr">5</a></sup> ST is a low-molecular-weight peptide consisting of 18 or 19 amino acids.<sup><a reveal-id="DSP015C6  DSP015C8" data-open="DSP015C6  DSP015C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> The other major virulence factor is a colonization factor antigen (CFA), which is responsible for adhesion of bacterial cells to the intestinal epithelial cells.<sup><a reveal-id="DSP015C1" data-open="DSP015C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP015C9" data-open="DSP015C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSP015C10" data-open="DSP015C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>It has been reported that these classical virulence factors are carried by plasmids such as pLT (encoding LT), pST (encoding ST), pLT-ST (encoding LT and ST), pCFA (encoding CFA), pCFA-LT (encoding CFA and LT) and pCFA-ST (encoding CFA and ST).<sup><a reveal-id="DSP015C6" data-open="DSP015C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP015C7" data-open="DSP015C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP015C11" data-open="DSP015C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSP015C12" data-open="DSP015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, the full sequences of such plasmids have not been reported, with the exception of a pCFA in ETEC O6:H16.<sup><a reveal-id="DSP015C13" data-open="DSP015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Among these plasmids, some families such as pLT,<sup><a reveal-id="DSP015C6" data-open="DSP015C6" class="link link-ref link-reveal xref-bibr">6</a></sup> pST<sup><a reveal-id="DSP015C7" data-open="DSP015C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP015C14" data-open="DSP015C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and pCFA<sup><a reveal-id="DSP015C13" data-open="DSP015C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP015C15" data-open="DSP015C15" class="link link-ref link-reveal xref-bibr">15</a></sup> have been reported to be auto-transmissible and may be evolving rapidly. Moreover, it has been reported that pLT-ST may also be self-transmissible in some serotypes of ETEC such as O78:H12, but non-self-transmissible in others such as in O78:H11.<sup><a reveal-id="DSP015C6" data-open="DSP015C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Therefore, it has not yet been determined whether pLT-ST has the genes required for transfer to other strains, or what other factors may render pLT-ST non-self-transmissible.</p><p><em>Escherichia coli</em> H10407 (serotype O78:H11), the best characterized of the ETEC strains, was isolated from a patient with diarrhoea in Bangladesh in 1971.<sup><a reveal-id="DSP015C1" data-open="DSP015C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP015C8" data-open="DSP015C8" class="link link-ref link-reveal xref-bibr">8</a></sup> It has been reported previously that the original <em>E. coli</em> H10407 strain carries five kinds of plasmid: pCFA/I-STIb [molecular mass (MM) 62 × 10<sup>6</sup> Da)] specifying CFA/I and STIb production, pEntH10407 (MM 42 × 10<sup>6</sup> Da) specifying LT and STIa production, pTRANS (MM 42 × 10<sup>6</sup> Da) specifying self-transmission and two other plasmids (MM 3.7 × 10<sup>6</sup> and 3.8 × 10<sup>6</sup> Da) manifesting no detectable phenotype.<sup><a reveal-id="DSP015C1" data-open="DSP015C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP015C6" data-open="DSP015C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP015C7" data-open="DSP015C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP015C11" data-open="DSP015C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSP015C12" data-open="DSP015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> It has further been reported that pTRANS promotes transfer of pCFA/I-STIb or pEntH10407 to other bacterial strains, suggesting that pCFA/I-STIb and pEntH10407 may be non-self-transmissible.<sup><a reveal-id="DSP015C12" data-open="DSP015C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p><p>Although the H10407 strain has been characterized in detail, there has been no report of the full sequence of the plasmids which it carries. In order to analyse the molecular evolution of pEntH10407 and to identify the factors conferring auto-transmissibility, we determined its full nucleotide sequence.</p><p>The ETEC H10407 strain maintained in our laboratory was used in these experiments. The sequenced plasmid, which will be referred to as pEntH10407K, was derived from the native plasmid pEntH10407 by an <em>in vitro</em> transposition system using hyperactive Tn<em>5</em><sup><a reveal-id="DSP015C16" data-open="DSP015C16" class="link link-ref link-reveal xref-bibr">16</a></sup> (1938 bp) to provide the selectable marker of kanamycin resistance.</p><p>The fully assembled sequence of pEntH10407K (GenBank accession no. AP010910), including the kanamycin cassette, consisted of 67 094 bp, 65 147 bp of which was specific to the pEntH10407 of ETEC H10407. The average G + C content of pEntH10407 is 51.2%. The sequence assembly of pEntH10407K was confirmed by comparing the restriction-enzyme-digestion patterns predicted from the sequence with those obtained by digestion of the plasmid, using several different restriction enzymes (data not shown). We also designed a set of PCR primer pairs to amplify segments overlapped with adjacent segments at both ends, covering the entire region of pEntH10407K. All pairs of primers yielded PCR products whose sizes matched the expected sizes (data not shown). These results show that no region was inserted or omitted in our sequence assembly of pEntH10407K.</p><p>The map shown in Fig. <a reveal-id="DSP015F1" data-open="DSP015F1" class="link link-reveal link-table xref-fig">1</a> depicts pEntH10407K, a circular plasmid which consists of 67 094 bp containing 100 open reading frames (ORFs) (Table <a reveal-id="DSP015TB1" data-open="DSP015TB1" class="link link-reveal link-table xref-fig">1</a>) and three regulatory RNA genes: <em>sok</em>, <em>copA</em> and <em>finP</em>. Putative functions could be assigned to 72 ORFs (72.0%); 24 ORFs were similar to conserved hypothetical proteins; the remaining 4 ORFs had no regions of significant similarity with proteins in the current database.</p>                        <a id="6237576" scrollto-destination="6237576"></a>
<div id="DSP015F1" data-id="DSP015F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/m_dsp01501.jpeg?Expires=2147483647&Signature=UbZcuRQxMMGKu5bKm8OeNCfAbgDoMZWmn1eHisBu0st8SXnds7178o6WO8DQ~VWaQiiU9T9OCOIP7V1x3Y1QGSpITkTgoMjzEIJsy19~NOZQYNLTy9qg2op4~KDfI80GcEsTrQOSIhvuiP4v5KLdGlCq2tCjxO9IpLI6RVnru63D-1z7YaYbY6YBYu7ZWuos5rBKIEzVh3MjJkaX2RDm0aPyhAxX14yIHG5N7Vnug56vDlO1VRESQd-dEwgZOo1z-zj2y6E4xZTwsn-TZlDJEOmvVn2TC4~EXEBu1LXCoNSerNFFh0eD2Vuhv9yW-uHeWBCdK92Rtx74eYk~-Tixqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Map of pH10407K. The sequence of pEntH10407K was determined by a whole-genome shotgun strategy. Sequence reads were assembled and gaps were closed by direct sequencing of PCR products amplified with oligonucleotide primers designed to anneal to each end of neighbouring contigs. The sequence was annotated using GenomeGambler (Xanagen Inc., Kanagawa, Japan). ORFs encoding products that were at least 50 amino acids in length were identified first; then possible ORFs were selected by combinations of database matches and by the presence of a ribosome binding site. Inner circle: ORFs, with their orientations colour-coded by functional category: red, known or putative virulence-associated proteins; pink, conjugal DNA transfer; orange, IS-related or transposase fragments; purple, intact IS or transposase; blue, plasmid replication, maintenance or other DNA metabolic functions; green, conserved hypothetical proteins; yellow, putative proteins. The outer circle shows the scale in base pairs. Nomenclature of ORFs is given in Table 1. The figure was generated using the program ‘in silico MolecularCloning GE’ (In Silico Biology, Inc., Kanagawa, Japan)." path-from-xml="dsp01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237576/dsp01501.jpeg" path-from-xml="dsp01501.jpeg" target="_blank">View large</a><a section="6237576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/dsp01501.jpeg?Expires=2147483647&Signature=hhciDd90Qa9bQ7~xASfd5EVPA1Qd4NbV6Xi~7OKC7HkKgr2mkU2Bkz3J~KIYpxiYDv0HugMRjYYQAIe59XLaL-NogDGwEk0bgdaJZ2xVf8v-ofug0j7KDPv8zteeEF-~iPq-hLX-ec-T2SsPGbqNlbPTlh4etD0bSop3t3-9FSM3sYF1f06zhgQIPzQMUzAhQxUqMj2SIeCSwWGwgVZZOtVBpMihemscm5XuDAjNfO6ojlBOeprQwjUyJodklvcH5C44d2jYahIa6ZtxvyohlN-ipugxDkYvVXv4vcogSNqc~nnrWplb5dBFvyORgF7yPC-BarzsA1Mma~CCQNKOMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237576&ar=420917&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Map of pH10407K. The sequence of pEntH10407K was determined by a whole-genome shotgun strategy. Sequence reads were assembled and gaps were closed by direct sequencing of PCR products amplified with oligonucleotide primers designed to anneal to each end of neighbouring contigs. The sequence was annotated using GenomeGambler (Xanagen Inc., Kanagawa, Japan). ORFs encoding products that were at least 50 amino acids in length were identified first; then possible ORFs were selected by combinations of database matches and by the presence of a ribosome binding site. Inner circle: ORFs, with their orientations colour-coded by functional category: red, known or putative virulence-associated proteins; pink, conjugal DNA transfer; orange, IS-related or transposase fragments; purple, intact IS or transposase; blue, plasmid replication, maintenance or other DNA metabolic functions; green, conserved hypothetical proteins; yellow, putative proteins. The outer circle shows the scale in base pairs. Nomenclature of ORFs is given in Table <a reveal-id="DSP015TB1" data-open="DSP015TB1" class="link link-reveal link-table xref-fig">1</a>. The figure was generated using the program ‘in silico MolecularCloning GE’ (In Silico Biology, Inc., Kanagawa, Japan).</p></div></div><div content-id="DSP015F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/m_dsp01501.jpeg?Expires=2147483647&Signature=UbZcuRQxMMGKu5bKm8OeNCfAbgDoMZWmn1eHisBu0st8SXnds7178o6WO8DQ~VWaQiiU9T9OCOIP7V1x3Y1QGSpITkTgoMjzEIJsy19~NOZQYNLTy9qg2op4~KDfI80GcEsTrQOSIhvuiP4v5KLdGlCq2tCjxO9IpLI6RVnru63D-1z7YaYbY6YBYu7ZWuos5rBKIEzVh3MjJkaX2RDm0aPyhAxX14yIHG5N7Vnug56vDlO1VRESQd-dEwgZOo1z-zj2y6E4xZTwsn-TZlDJEOmvVn2TC4~EXEBu1LXCoNSerNFFh0eD2Vuhv9yW-uHeWBCdK92Rtx74eYk~-Tixqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Map of pH10407K. The sequence of pEntH10407K was determined by a whole-genome shotgun strategy. Sequence reads were assembled and gaps were closed by direct sequencing of PCR products amplified with oligonucleotide primers designed to anneal to each end of neighbouring contigs. The sequence was annotated using GenomeGambler (Xanagen Inc., Kanagawa, Japan). ORFs encoding products that were at least 50 amino acids in length were identified first; then possible ORFs were selected by combinations of database matches and by the presence of a ribosome binding site. Inner circle: ORFs, with their orientations colour-coded by functional category: red, known or putative virulence-associated proteins; pink, conjugal DNA transfer; orange, IS-related or transposase fragments; purple, intact IS or transposase; blue, plasmid replication, maintenance or other DNA metabolic functions; green, conserved hypothetical proteins; yellow, putative proteins. The outer circle shows the scale in base pairs. Nomenclature of ORFs is given in Table 1. The figure was generated using the program ‘in silico MolecularCloning GE’ (In Silico Biology, Inc., Kanagawa, Japan)." path-from-xml="dsp01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237576/dsp01501.jpeg" path-from-xml="dsp01501.jpeg" target="_blank">View large</a><a section="6237576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/dsp01501.jpeg?Expires=2147483647&Signature=hhciDd90Qa9bQ7~xASfd5EVPA1Qd4NbV6Xi~7OKC7HkKgr2mkU2Bkz3J~KIYpxiYDv0HugMRjYYQAIe59XLaL-NogDGwEk0bgdaJZ2xVf8v-ofug0j7KDPv8zteeEF-~iPq-hLX-ec-T2SsPGbqNlbPTlh4etD0bSop3t3-9FSM3sYF1f06zhgQIPzQMUzAhQxUqMj2SIeCSwWGwgVZZOtVBpMihemscm5XuDAjNfO6ojlBOeprQwjUyJodklvcH5C44d2jYahIa6ZtxvyohlN-ipugxDkYvVXv4vcogSNqc~nnrWplb5dBFvyORgF7yPC-BarzsA1Mma~CCQNKOMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237576&ar=420917&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Map of pH10407K. The sequence of pEntH10407K was determined by a whole-genome shotgun strategy. Sequence reads were assembled and gaps were closed by direct sequencing of PCR products amplified with oligonucleotide primers designed to anneal to each end of neighbouring contigs. The sequence was annotated using GenomeGambler (Xanagen Inc., Kanagawa, Japan). ORFs encoding products that were at least 50 amino acids in length were identified first; then possible ORFs were selected by combinations of database matches and by the presence of a ribosome binding site. Inner circle: ORFs, with their orientations colour-coded by functional category: red, known or putative virulence-associated proteins; pink, conjugal DNA transfer; orange, IS-related or transposase fragments; purple, intact IS or transposase; blue, plasmid replication, maintenance or other DNA metabolic functions; green, conserved hypothetical proteins; yellow, putative proteins. The outer circle shows the scale in base pairs. Nomenclature of ORFs is given in Table <a reveal-id="DSP015TB1" data-open="DSP015TB1" class="link link-reveal link-table xref-fig">1</a>. The figure was generated using the program ‘in silico MolecularCloning GE’ (In Silico Biology, Inc., Kanagawa, Japan).</p></div></div>                        <a id="6237577" scrollto-destination="6237577"></a>
<div content-id="DSP015TB1" class="table-modal"><div class="table-wrap"><div id="DSP015TB1" class="table-wrap-title" data-id="DSP015TB1"><div class="label -label">Table 1</div><div class="caption"><p>ORFs of pEntH10407K</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ORF </th><th>Gene </th><th>Orientation<sup>a</sup> </th><th>Position (bp) </th><th>Size (aa) </th><th>Homologue by BLAST </th><th>Identity/similarity (%) </th><th>Accession no. </th></tr></thead><tbody><tr><td>ORF001 </td><td><em>mltE</em> </td><td>+ </td><td>39–641 </td><td>200 </td><td>Lytic transglycosylase </td><td>94/96 </td><td>ABG29580 </td></tr><tr><td>ORF002 </td><td> </td><td>− </td><td>801–667 </td><td>44 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>ABD60010 </td></tr><tr><td>ORF003 </td><td> </td><td>− </td><td>1759–938 </td><td>273 </td><td>Conserved hypothetical protein YubP </td><td>9/99 </td><td>BAA78846 </td></tr><tr><td>ORF004 </td><td> </td><td>− </td><td>2166–1870 </td><td>98 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAA78845 </td></tr><tr><td>ORF005 </td><td> </td><td>− </td><td>2318–2190 </td><td>42 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF006 </td><td><em>ppdC</em> </td><td>+ </td><td>2509–2826 </td><td>105 </td><td>Conserved hypothetical protein </td><td>91/93 </td><td>BBA97937 </td></tr><tr><td>ORF007 </td><td> </td><td>− </td><td>2971–2807 </td><td>54 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>EDX27828 </td></tr><tr><td>ORF008 </td><td><em>hok</em> </td><td>− </td><td>3223–3062 </td><td>53 </td><td>Post-segregation killing protein </td><td>72/84 </td><td>P16077 </td></tr><tr><td>ORF009 </td><td><em>mok</em> </td><td>− </td><td>3281–3066 </td><td>71 </td><td>Modulator of post-segregation killing protein </td><td>50/58 </td><td>P23587 </td></tr><tr><td>ORF010 </td><td> </td><td>− </td><td>3746–3432 </td><td>104 </td><td>Zn-dependent dehydrogenases </td><td>99/100 </td><td>CAI79556 </td></tr><tr><td>ORF011 </td><td><em>psiA</em> </td><td>− </td><td>4462–3743 </td><td>239 </td><td>PsiA </td><td>97/98 </td><td>BAA78841 </td></tr><tr><td>ORF012 </td><td><em>psiB</em> </td><td>− </td><td>4893–4459 </td><td>144 </td><td>PsiB </td><td>100/100 </td><td>ABD51587 </td></tr><tr><td>ORF013 </td><td> </td><td>− </td><td>5649–4948 </td><td>233 </td><td>Predicted transcriptional regulator </td><td>91/96 (truncated) </td><td>ABD51586 </td></tr><tr><td>ORF014 </td><td> </td><td>− </td><td>6912–5662 </td><td>416 </td><td>Predicted transcriptional regulator </td><td>96/97 (truncated) </td><td>AAW58879 </td></tr><tr><td>ORF015 </td><td> </td><td>− </td><td>7209–6976 </td><td>77 </td><td>Conserved hypothetical protein </td><td>98/99 </td><td>ABE10669 </td></tr><tr><td>ORF016 </td><td><em>ssb</em> </td><td>− </td><td>7832–7266 </td><td>188 </td><td>Ssb </td><td>94/96 </td><td>BBA78826 </td></tr><tr><td>ORF017 </td><td> </td><td>+ </td><td>7858–8094 </td><td>78 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF018 </td><td> </td><td>− </td><td>8273–8025 </td><td>82 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF019 </td><td> </td><td>+ </td><td>8997–10 664 </td><td>555 </td><td>Conserved hypothetical protein YkfC </td><td>63/77 </td><td>ABI41559 </td></tr><tr><td>ORF020 </td><td> </td><td>− </td><td>11 020–10 778 </td><td>80 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>ZP_00719262 </td></tr><tr><td>ORF021 </td><td> </td><td>− </td><td>11 583–11 020 </td><td>187 </td><td>Conserved hypothetical protein </td><td>97/99 </td><td>BAF33947 </td></tr><tr><td>ORF022 </td><td> </td><td>− </td><td>12 991–11 630 </td><td>453 </td><td>Conserved hypothetical protein </td><td>90/94 </td><td>BAF33946 </td></tr><tr><td>ORF023 </td><td> </td><td>+ </td><td>12 995–13 135 </td><td>46 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>CAP07686 </td></tr><tr><td>ORF024 </td><td> </td><td>− </td><td>13 849–13 415 </td><td>144 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>BBA78815 </td></tr><tr><td>ORF025 </td><td> </td><td>− </td><td>14 084–13 863 </td><td>73 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>ABE10652 </td></tr><tr><td>ORF026 </td><td> </td><td>− </td><td>14 768–14 085 </td><td>227 </td><td>Conserved hypothetical protein </td><td>97/98 </td><td>AAS76 410 </td></tr><tr><td>ORF027 </td><td> </td><td>– </td><td>15 251–14 844 </td><td>135 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>AAW58863 </td></tr><tr><td>ORF028 </td><td><em>stbA</em> </td><td>+ </td><td>15 284–16 246 </td><td>320 </td><td>StbA </td><td>99/99 </td><td>ABD59972 </td></tr><tr><td>ORF029 </td><td><em>stbB</em> </td><td>+ </td><td>16 246–16 599 </td><td>117 </td><td>StbB </td><td>99/99 </td><td>ABD59971 </td></tr><tr><td>ORF030 </td><td><em>insA</em> </td><td>+ </td><td>17 011–17 286 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF031 </td><td><em>insB</em> </td><td>+ </td><td>17 205–17 708 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>100/100 </td><td>AAA96694 </td></tr><tr><td>ORF032 </td><td> </td><td>− </td><td>17 982–17 719 </td><td>87 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF033 </td><td> </td><td>− </td><td>18 983–18 720 </td><td>87 </td><td>Conserved hypothetical protein </td><td>95/97 </td><td>AAS58634 </td></tr><tr><td>ORF034 </td><td><em>insB</em> </td><td>− </td><td>19 618–19 115 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF035 </td><td><em>insA</em> </td><td>− </td><td>19 812–19 537 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>99/100 </td><td>AAA58242 </td></tr><tr><td>ORF036 </td><td> </td><td>+ </td><td>19 925–20 158 </td><td>77 </td><td>ORF1 of IS<em>600</em> </td><td>80/80 (truncated) </td><td>ABB68584 </td></tr><tr><td>ORF037 </td><td> </td><td>+ </td><td>20 208–21 026 </td><td>272 </td><td>ORF2 of IS<em>600</em> </td><td>99/99 </td><td>AAN43456 </td></tr><tr><td>ORF038 </td><td><em>eltB</em> </td><td>− </td><td>21 722–21 348 </td><td>124 </td><td>LT-B </td><td>100/100 </td><td>AAC60441 </td></tr><tr><td>ORF039 </td><td><em>eltA</em> </td><td>− </td><td>22 549–21 719 </td><td>276 </td><td>LT-A </td><td>100/100 </td><td>P43530 </td></tr><tr><td>ORF040 </td><td> </td><td>+ </td><td>22 812–23 084 </td><td>90 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>AAZ91090 </td></tr><tr><td>ORF041 </td><td> </td><td>+ </td><td>23 065–23 334 </td><td>89 </td><td>Transposase of IS<em>801</em> </td><td>76/79 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF042 </td><td> </td><td>+ </td><td>23 250–23 564 </td><td>104 </td><td>Transposase of IS<em>801</em> </td><td>99/99 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF043 </td><td> </td><td>+ </td><td>23 779–25 002 </td><td>407 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>AAT35239 </td></tr><tr><td>ORF044 </td><td> </td><td>− </td><td>25 347–24 928 </td><td>139 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>CAI79504 </td></tr><tr><td>ORF045 </td><td> </td><td>+ </td><td>25 487–26 164 </td><td>225 </td><td>ORF1 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF046 </td><td> </td><td>+ </td><td>26 164–26 511 </td><td>115 </td><td>ORF2 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF047 </td><td> </td><td>+ </td><td>26 531–28 102 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF048 </td><td><em>estIa</em> </td><td>− </td><td>28 394–28 176 </td><td>72 </td><td>STIa </td><td>100/100 </td><td>P01559 </td></tr><tr><td>ORF049 </td><td><em>insB</em> </td><td>− </td><td>29 343–28 840 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF050 </td><td><em>insA</em> </td><td>− </td><td>29 537–29 262 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF051 </td><td> </td><td>+ </td><td>29 641–29 796 </td><td>51 </td><td>Putative transposase </td><td>98/100 (truncated) </td><td>CAA07835 </td></tr><tr><td>ORF052 </td><td> </td><td>− </td><td>30 316–30 008 </td><td>102 </td><td>Predicted transcriptional regulator </td><td>100/100 </td><td>ZP_00713086 </td></tr><tr><td>ORF053 </td><td> </td><td>+ </td><td>30 288–30 512 </td><td>74 </td><td>Putative transposase </td><td>92/94 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF054<sup>b</sup> </td><td> </td><td>+ </td><td>30 664–31 479 </td><td>271 </td><td>Aminoglucoside 3′-phosphotransferase </td><td>100/100 </td><td>AAA80260 </td></tr><tr><td>ORF055 </td><td> </td><td>+ </td><td>32 512–32 628 </td><td>38 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>ACD54240 </td></tr><tr><td>ORF056 </td><td><em>baeS</em> </td><td>− </td><td>34 173–32 701 </td><td>490 </td><td>BaeS </td><td>73/86 </td><td>ABE10335 </td></tr><tr><td>ORF057 </td><td><em>ompR</em> </td><td>− </td><td>34 892–34 170 </td><td>240 </td><td>BaeR </td><td>83/90 </td><td>ABE10334 </td></tr><tr><td>ORF058 </td><td><em>ydhU</em> </td><td>+ </td><td>35 034–35 633 </td><td>199 </td><td>Thiosulphate reductase cytochrome B subunit </td><td>67/82 </td><td>CAD42043 </td></tr><tr><td>ORF059 </td><td> </td><td>+ </td><td>35 644–36 414 </td><td>256 </td><td>Oxidoreductase, molybdopterin-binding subunit </td><td>82/91 </td><td>CAD42042 </td></tr><tr><td>ORF060 </td><td> </td><td>+ </td><td>36 444–36 683 </td><td>79 </td><td>Conserved hypothetical protein </td><td>59/70 (truncated) </td><td>ABE10331 </td></tr><tr><td>ORF061 </td><td> </td><td>− </td><td>39 074–37 563 </td><td>503 </td><td>Putative ATP binding protein </td><td>25/43 </td><td>CAD16966 </td></tr><tr><td>ORF062 </td><td><em>stbE</em> </td><td>− </td><td>39 721–39 434 </td><td>95 </td><td>RelE </td><td>93/96 </td><td>ABD51640 </td></tr><tr><td>ORF063 </td><td><em>stbD</em> </td><td>− </td><td>39 969–39 718 </td><td>83 </td><td>RelB </td><td>100/100 </td><td>ABD51639 </td></tr><tr><td>ORF064 </td><td> </td><td>− </td><td>40 233–40 042 </td><td>63 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>AAL72549 </td></tr><tr><td>ORF065 </td><td><em>repA4</em> </td><td>− </td><td>40 370–40 185 </td><td>61 </td><td>RepA4 </td><td>90/93 (truncated) </td><td>BAA78895 </td></tr><tr><td>ORF066 </td><td><em>repA4</em> </td><td>− </td><td>40 573–40 325 </td><td>82 </td><td>RepA4 </td><td>88/90 (truncated) </td><td>ABC42205 </td></tr><tr><td>ORF067 </td><td> </td><td>− </td><td>40 873–40 703 </td><td>56 </td><td>Conserved hypothetical protein </td><td>81/85 </td><td>ACD06080 </td></tr><tr><td>ORF068 </td><td><em>repA1</em> </td><td>− </td><td>41 793–40 936 </td><td>285 </td><td>RepA1 </td><td>99/100 </td><td>CAI79519 </td></tr><tr><td>ORF069 </td><td><em>tapA</em> </td><td>− </td><td>41 860–41 786 </td><td>24 </td><td>TapA </td><td>100/100 </td><td>BBA78893 </td></tr><tr><td>ORF070 </td><td><em>repA3</em> </td><td>− </td><td>41 937–41 806 </td><td>43 </td><td>RepA3 </td><td>87/89 (truncated) </td><td>AAA26066 </td></tr><tr><td>ORF071 </td><td><em>repA2</em> </td><td>− </td><td>42 354–42 094 </td><td>86 </td><td>RepA2 </td><td>99/100 </td><td>ABE10578 </td></tr><tr><td>ORF072 </td><td> </td><td>− </td><td>43 184–42 594 </td><td>196 </td><td>Superfamily I DNA/RNA helicase </td><td>100/100 </td><td>AAW58927 </td></tr><tr><td>ORF073 </td><td> </td><td>− </td><td>43 430–43 227 </td><td>67 </td><td>YmoA </td><td>97/97 </td><td>AAO49553 </td></tr><tr><td>ORF074 </td><td> </td><td>− </td><td>43 937–43 476 </td><td>153 </td><td>Thermonuclease family protein </td><td>98/98 </td><td>ABE10720 </td></tr><tr><td>ORF075 </td><td> </td><td>− </td><td>44 394–44 182 </td><td>70 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAF33997 </td></tr><tr><td>ORF076 </td><td><em>finO</em> </td><td>− </td><td>45 086–44 526 </td><td>186 </td><td>FinO </td><td>99/99 </td><td>AAC70069 </td></tr><tr><td>ORF077 </td><td><em>traX</em> </td><td>− </td><td>45 944–45 141 </td><td>267 </td><td>TraX </td><td>99/100 </td><td>BAF33995 </td></tr><tr><td>ORF078 </td><td><em>traI</em> </td><td>− </td><td>51 177–45 907 </td><td>1756 </td><td>TraI </td><td>98/99 </td><td>CAA39337 </td></tr><tr><td>ORF079 </td><td><em>traD</em> </td><td>− </td><td>53 465–51 177 </td><td>762 </td><td>TraD </td><td>96/96 </td><td>BAA78884 </td></tr><tr><td>ORF080 </td><td> </td><td>− </td><td>53 944–53 516 </td><td>142 </td><td>Conserved hypothetical protein YhfA </td><td>99/99 (truncated) </td><td>ABE10712 </td></tr><tr><td>ORF081 </td><td> </td><td>− </td><td>55 646–54 075 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF082 </td><td> </td><td>− </td><td>56 013–55 666 </td><td>115 </td><td>ORF2 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF083 </td><td> </td><td>− </td><td>56 690–56 013 </td><td>225 </td><td>ORF1 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF084 </td><td> </td><td>+ </td><td>56 910–57 311 </td><td>133 </td><td>L0013 of IS<em>Ec8</em> </td><td>99/100 </td><td>ABG71816 </td></tr><tr><td>ORF085 </td><td> </td><td>+ </td><td>57 308–57 655 </td><td>115 </td><td>L0014 of IS<em>Ec8</em> </td><td>100/100 </td><td>AAG54624 </td></tr><tr><td>ORF086 </td><td> </td><td>+ </td><td>57 705–59 243 </td><td>512 </td><td>L0015 of IS<em>Ec8</em> </td><td>99/99 </td><td>AAG54625 </td></tr><tr><td>ORF087 </td><td> </td><td>− </td><td>59 613–59 377 </td><td>78 </td><td>Conserved hypothetical protein YfhA </td><td>99/99 (truncated) </td><td>ABD60024 </td></tr><tr><td>ORF088 </td><td><em>traR</em> </td><td>− </td><td>59 827–59 606 </td><td>73 </td><td>TraR </td><td>99/100 </td><td>ABC42235 </td></tr><tr><td>ORF089 </td><td><em>traV</em> </td><td>− </td><td>60 477–59 962 </td><td>171 </td><td>TraV </td><td>98/99 </td><td>BAA78858 </td></tr><tr><td>ORF090 </td><td><em>trbG</em> </td><td>− </td><td>60 725–60 474 </td><td>83 </td><td>TrbG </td><td>96/98 </td><td>BAA97951 </td></tr><tr><td>ORF091 </td><td><em>trbD</em> </td><td>− </td><td>61 038–60 718 </td><td>106 </td><td>TrbD </td><td>89/94 </td><td>ABC42237 </td></tr><tr><td>ORF092 </td><td><em>traP</em> </td><td>− </td><td>61 612–61 025 </td><td>195 </td><td>TraP </td><td>96/97 </td><td>BAA78856 </td></tr><tr><td>ORF093 </td><td><em>traB</em> </td><td>− </td><td>63 032–61 581 </td><td>483 </td><td>TraB </td><td>99/99 </td><td>BAA78855 </td></tr><tr><td>ORF094 </td><td><em>traK</em> </td><td>− </td><td>63 760–63 032 </td><td>242 </td><td>TraK </td><td>99/99 </td><td>BAA78854 </td></tr><tr><td>ORF095 </td><td><em>traE</em> </td><td>− </td><td>64 313–63 747 </td><td>188 </td><td>TraE </td><td>99/100 </td><td>BAA78853 </td></tr><tr><td>ORF096 </td><td><em>traL</em> </td><td>− </td><td>64 646–64 335 </td><td>103 </td><td>TraL </td><td>99/100 </td><td>BAA97945 </td></tr><tr><td>ORF097 </td><td><em>traA</em> </td><td>− </td><td>65 026–64 661 </td><td>121 </td><td>TraA </td><td>96/98 </td><td>BAA78851 </td></tr><tr><td>ORF098 </td><td><em>traY</em> </td><td>− </td><td>65 453–65 058 </td><td>131 </td><td>TraY </td><td>100/100 </td><td>BAA97943 </td></tr><tr><td>ORF099 </td><td><em>traJ</em> </td><td>− </td><td>66 241–65 552 </td><td>229 </td><td>TraJ </td><td>98/99 </td><td>BAA97942 </td></tr><tr><td>ORF100 </td><td><em>traM</em> </td><td>− </td><td>66 811–66 428 </td><td>127 </td><td>TraM </td><td>98/100 </td><td>ABD51596 </td></tr></tbody></table></div><div content-id="DSP015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>ORF </th><th>Gene </th><th>Orientation<sup>a</sup> </th><th>Position (bp) </th><th>Size (aa) </th><th>Homologue by BLAST </th><th>Identity/similarity (%) </th><th>Accession no. </th></tr></thead><tbody><tr><td>ORF001 </td><td><em>mltE</em> </td><td>+ </td><td>39–641 </td><td>200 </td><td>Lytic transglycosylase </td><td>94/96 </td><td>ABG29580 </td></tr><tr><td>ORF002 </td><td> </td><td>− </td><td>801–667 </td><td>44 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>ABD60010 </td></tr><tr><td>ORF003 </td><td> </td><td>− </td><td>1759–938 </td><td>273 </td><td>Conserved hypothetical protein YubP </td><td>9/99 </td><td>BAA78846 </td></tr><tr><td>ORF004 </td><td> </td><td>− </td><td>2166–1870 </td><td>98 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAA78845 </td></tr><tr><td>ORF005 </td><td> </td><td>− </td><td>2318–2190 </td><td>42 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF006 </td><td><em>ppdC</em> </td><td>+ </td><td>2509–2826 </td><td>105 </td><td>Conserved hypothetical protein </td><td>91/93 </td><td>BBA97937 </td></tr><tr><td>ORF007 </td><td> </td><td>− </td><td>2971–2807 </td><td>54 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>EDX27828 </td></tr><tr><td>ORF008 </td><td><em>hok</em> </td><td>− </td><td>3223–3062 </td><td>53 </td><td>Post-segregation killing protein </td><td>72/84 </td><td>P16077 </td></tr><tr><td>ORF009 </td><td><em>mok</em> </td><td>− </td><td>3281–3066 </td><td>71 </td><td>Modulator of post-segregation killing protein </td><td>50/58 </td><td>P23587 </td></tr><tr><td>ORF010 </td><td> </td><td>− </td><td>3746–3432 </td><td>104 </td><td>Zn-dependent dehydrogenases </td><td>99/100 </td><td>CAI79556 </td></tr><tr><td>ORF011 </td><td><em>psiA</em> </td><td>− </td><td>4462–3743 </td><td>239 </td><td>PsiA </td><td>97/98 </td><td>BAA78841 </td></tr><tr><td>ORF012 </td><td><em>psiB</em> </td><td>− </td><td>4893–4459 </td><td>144 </td><td>PsiB </td><td>100/100 </td><td>ABD51587 </td></tr><tr><td>ORF013 </td><td> </td><td>− </td><td>5649–4948 </td><td>233 </td><td>Predicted transcriptional regulator </td><td>91/96 (truncated) </td><td>ABD51586 </td></tr><tr><td>ORF014 </td><td> </td><td>− </td><td>6912–5662 </td><td>416 </td><td>Predicted transcriptional regulator </td><td>96/97 (truncated) </td><td>AAW58879 </td></tr><tr><td>ORF015 </td><td> </td><td>− </td><td>7209–6976 </td><td>77 </td><td>Conserved hypothetical protein </td><td>98/99 </td><td>ABE10669 </td></tr><tr><td>ORF016 </td><td><em>ssb</em> </td><td>− </td><td>7832–7266 </td><td>188 </td><td>Ssb </td><td>94/96 </td><td>BBA78826 </td></tr><tr><td>ORF017 </td><td> </td><td>+ </td><td>7858–8094 </td><td>78 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF018 </td><td> </td><td>− </td><td>8273–8025 </td><td>82 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF019 </td><td> </td><td>+ </td><td>8997–10 664 </td><td>555 </td><td>Conserved hypothetical protein YkfC </td><td>63/77 </td><td>ABI41559 </td></tr><tr><td>ORF020 </td><td> </td><td>− </td><td>11 020–10 778 </td><td>80 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>ZP_00719262 </td></tr><tr><td>ORF021 </td><td> </td><td>− </td><td>11 583–11 020 </td><td>187 </td><td>Conserved hypothetical protein </td><td>97/99 </td><td>BAF33947 </td></tr><tr><td>ORF022 </td><td> </td><td>− </td><td>12 991–11 630 </td><td>453 </td><td>Conserved hypothetical protein </td><td>90/94 </td><td>BAF33946 </td></tr><tr><td>ORF023 </td><td> </td><td>+ </td><td>12 995–13 135 </td><td>46 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>CAP07686 </td></tr><tr><td>ORF024 </td><td> </td><td>− </td><td>13 849–13 415 </td><td>144 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>BBA78815 </td></tr><tr><td>ORF025 </td><td> </td><td>− </td><td>14 084–13 863 </td><td>73 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>ABE10652 </td></tr><tr><td>ORF026 </td><td> </td><td>− </td><td>14 768–14 085 </td><td>227 </td><td>Conserved hypothetical protein </td><td>97/98 </td><td>AAS76 410 </td></tr><tr><td>ORF027 </td><td> </td><td>– </td><td>15 251–14 844 </td><td>135 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>AAW58863 </td></tr><tr><td>ORF028 </td><td><em>stbA</em> </td><td>+ </td><td>15 284–16 246 </td><td>320 </td><td>StbA </td><td>99/99 </td><td>ABD59972 </td></tr><tr><td>ORF029 </td><td><em>stbB</em> </td><td>+ </td><td>16 246–16 599 </td><td>117 </td><td>StbB </td><td>99/99 </td><td>ABD59971 </td></tr><tr><td>ORF030 </td><td><em>insA</em> </td><td>+ </td><td>17 011–17 286 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF031 </td><td><em>insB</em> </td><td>+ </td><td>17 205–17 708 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>100/100 </td><td>AAA96694 </td></tr><tr><td>ORF032 </td><td> </td><td>− </td><td>17 982–17 719 </td><td>87 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF033 </td><td> </td><td>− </td><td>18 983–18 720 </td><td>87 </td><td>Conserved hypothetical protein </td><td>95/97 </td><td>AAS58634 </td></tr><tr><td>ORF034 </td><td><em>insB</em> </td><td>− </td><td>19 618–19 115 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF035 </td><td><em>insA</em> </td><td>− </td><td>19 812–19 537 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>99/100 </td><td>AAA58242 </td></tr><tr><td>ORF036 </td><td> </td><td>+ </td><td>19 925–20 158 </td><td>77 </td><td>ORF1 of IS<em>600</em> </td><td>80/80 (truncated) </td><td>ABB68584 </td></tr><tr><td>ORF037 </td><td> </td><td>+ </td><td>20 208–21 026 </td><td>272 </td><td>ORF2 of IS<em>600</em> </td><td>99/99 </td><td>AAN43456 </td></tr><tr><td>ORF038 </td><td><em>eltB</em> </td><td>− </td><td>21 722–21 348 </td><td>124 </td><td>LT-B </td><td>100/100 </td><td>AAC60441 </td></tr><tr><td>ORF039 </td><td><em>eltA</em> </td><td>− </td><td>22 549–21 719 </td><td>276 </td><td>LT-A </td><td>100/100 </td><td>P43530 </td></tr><tr><td>ORF040 </td><td> </td><td>+ </td><td>22 812–23 084 </td><td>90 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>AAZ91090 </td></tr><tr><td>ORF041 </td><td> </td><td>+ </td><td>23 065–23 334 </td><td>89 </td><td>Transposase of IS<em>801</em> </td><td>76/79 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF042 </td><td> </td><td>+ </td><td>23 250–23 564 </td><td>104 </td><td>Transposase of IS<em>801</em> </td><td>99/99 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF043 </td><td> </td><td>+ </td><td>23 779–25 002 </td><td>407 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>AAT35239 </td></tr><tr><td>ORF044 </td><td> </td><td>− </td><td>25 347–24 928 </td><td>139 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>CAI79504 </td></tr><tr><td>ORF045 </td><td> </td><td>+ </td><td>25 487–26 164 </td><td>225 </td><td>ORF1 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF046 </td><td> </td><td>+ </td><td>26 164–26 511 </td><td>115 </td><td>ORF2 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF047 </td><td> </td><td>+ </td><td>26 531–28 102 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF048 </td><td><em>estIa</em> </td><td>− </td><td>28 394–28 176 </td><td>72 </td><td>STIa </td><td>100/100 </td><td>P01559 </td></tr><tr><td>ORF049 </td><td><em>insB</em> </td><td>− </td><td>29 343–28 840 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF050 </td><td><em>insA</em> </td><td>− </td><td>29 537–29 262 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF051 </td><td> </td><td>+ </td><td>29 641–29 796 </td><td>51 </td><td>Putative transposase </td><td>98/100 (truncated) </td><td>CAA07835 </td></tr><tr><td>ORF052 </td><td> </td><td>− </td><td>30 316–30 008 </td><td>102 </td><td>Predicted transcriptional regulator </td><td>100/100 </td><td>ZP_00713086 </td></tr><tr><td>ORF053 </td><td> </td><td>+ </td><td>30 288–30 512 </td><td>74 </td><td>Putative transposase </td><td>92/94 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF054<sup>b</sup> </td><td> </td><td>+ </td><td>30 664–31 479 </td><td>271 </td><td>Aminoglucoside 3′-phosphotransferase </td><td>100/100 </td><td>AAA80260 </td></tr><tr><td>ORF055 </td><td> </td><td>+ </td><td>32 512–32 628 </td><td>38 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>ACD54240 </td></tr><tr><td>ORF056 </td><td><em>baeS</em> </td><td>− </td><td>34 173–32 701 </td><td>490 </td><td>BaeS </td><td>73/86 </td><td>ABE10335 </td></tr><tr><td>ORF057 </td><td><em>ompR</em> </td><td>− </td><td>34 892–34 170 </td><td>240 </td><td>BaeR </td><td>83/90 </td><td>ABE10334 </td></tr><tr><td>ORF058 </td><td><em>ydhU</em> </td><td>+ </td><td>35 034–35 633 </td><td>199 </td><td>Thiosulphate reductase cytochrome B subunit </td><td>67/82 </td><td>CAD42043 </td></tr><tr><td>ORF059 </td><td> </td><td>+ </td><td>35 644–36 414 </td><td>256 </td><td>Oxidoreductase, molybdopterin-binding subunit </td><td>82/91 </td><td>CAD42042 </td></tr><tr><td>ORF060 </td><td> </td><td>+ </td><td>36 444–36 683 </td><td>79 </td><td>Conserved hypothetical protein </td><td>59/70 (truncated) </td><td>ABE10331 </td></tr><tr><td>ORF061 </td><td> </td><td>− </td><td>39 074–37 563 </td><td>503 </td><td>Putative ATP binding protein </td><td>25/43 </td><td>CAD16966 </td></tr><tr><td>ORF062 </td><td><em>stbE</em> </td><td>− </td><td>39 721–39 434 </td><td>95 </td><td>RelE </td><td>93/96 </td><td>ABD51640 </td></tr><tr><td>ORF063 </td><td><em>stbD</em> </td><td>− </td><td>39 969–39 718 </td><td>83 </td><td>RelB </td><td>100/100 </td><td>ABD51639 </td></tr><tr><td>ORF064 </td><td> </td><td>− </td><td>40 233–40 042 </td><td>63 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>AAL72549 </td></tr><tr><td>ORF065 </td><td><em>repA4</em> </td><td>− </td><td>40 370–40 185 </td><td>61 </td><td>RepA4 </td><td>90/93 (truncated) </td><td>BAA78895 </td></tr><tr><td>ORF066 </td><td><em>repA4</em> </td><td>− </td><td>40 573–40 325 </td><td>82 </td><td>RepA4 </td><td>88/90 (truncated) </td><td>ABC42205 </td></tr><tr><td>ORF067 </td><td> </td><td>− </td><td>40 873–40 703 </td><td>56 </td><td>Conserved hypothetical protein </td><td>81/85 </td><td>ACD06080 </td></tr><tr><td>ORF068 </td><td><em>repA1</em> </td><td>− </td><td>41 793–40 936 </td><td>285 </td><td>RepA1 </td><td>99/100 </td><td>CAI79519 </td></tr><tr><td>ORF069 </td><td><em>tapA</em> </td><td>− </td><td>41 860–41 786 </td><td>24 </td><td>TapA </td><td>100/100 </td><td>BBA78893 </td></tr><tr><td>ORF070 </td><td><em>repA3</em> </td><td>− </td><td>41 937–41 806 </td><td>43 </td><td>RepA3 </td><td>87/89 (truncated) </td><td>AAA26066 </td></tr><tr><td>ORF071 </td><td><em>repA2</em> </td><td>− </td><td>42 354–42 094 </td><td>86 </td><td>RepA2 </td><td>99/100 </td><td>ABE10578 </td></tr><tr><td>ORF072 </td><td> </td><td>− </td><td>43 184–42 594 </td><td>196 </td><td>Superfamily I DNA/RNA helicase </td><td>100/100 </td><td>AAW58927 </td></tr><tr><td>ORF073 </td><td> </td><td>− </td><td>43 430–43 227 </td><td>67 </td><td>YmoA </td><td>97/97 </td><td>AAO49553 </td></tr><tr><td>ORF074 </td><td> </td><td>− </td><td>43 937–43 476 </td><td>153 </td><td>Thermonuclease family protein </td><td>98/98 </td><td>ABE10720 </td></tr><tr><td>ORF075 </td><td> </td><td>− </td><td>44 394–44 182 </td><td>70 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAF33997 </td></tr><tr><td>ORF076 </td><td><em>finO</em> </td><td>− </td><td>45 086–44 526 </td><td>186 </td><td>FinO </td><td>99/99 </td><td>AAC70069 </td></tr><tr><td>ORF077 </td><td><em>traX</em> </td><td>− </td><td>45 944–45 141 </td><td>267 </td><td>TraX </td><td>99/100 </td><td>BAF33995 </td></tr><tr><td>ORF078 </td><td><em>traI</em> </td><td>− </td><td>51 177–45 907 </td><td>1756 </td><td>TraI </td><td>98/99 </td><td>CAA39337 </td></tr><tr><td>ORF079 </td><td><em>traD</em> </td><td>− </td><td>53 465–51 177 </td><td>762 </td><td>TraD </td><td>96/96 </td><td>BAA78884 </td></tr><tr><td>ORF080 </td><td> </td><td>− </td><td>53 944–53 516 </td><td>142 </td><td>Conserved hypothetical protein YhfA </td><td>99/99 (truncated) </td><td>ABE10712 </td></tr><tr><td>ORF081 </td><td> </td><td>− </td><td>55 646–54 075 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF082 </td><td> </td><td>− </td><td>56 013–55 666 </td><td>115 </td><td>ORF2 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF083 </td><td> </td><td>− </td><td>56 690–56 013 </td><td>225 </td><td>ORF1 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF084 </td><td> </td><td>+ </td><td>56 910–57 311 </td><td>133 </td><td>L0013 of IS<em>Ec8</em> </td><td>99/100 </td><td>ABG71816 </td></tr><tr><td>ORF085 </td><td> </td><td>+ </td><td>57 308–57 655 </td><td>115 </td><td>L0014 of IS<em>Ec8</em> </td><td>100/100 </td><td>AAG54624 </td></tr><tr><td>ORF086 </td><td> </td><td>+ </td><td>57 705–59 243 </td><td>512 </td><td>L0015 of IS<em>Ec8</em> </td><td>99/99 </td><td>AAG54625 </td></tr><tr><td>ORF087 </td><td> </td><td>− </td><td>59 613–59 377 </td><td>78 </td><td>Conserved hypothetical protein YfhA </td><td>99/99 (truncated) </td><td>ABD60024 </td></tr><tr><td>ORF088 </td><td><em>traR</em> </td><td>− </td><td>59 827–59 606 </td><td>73 </td><td>TraR </td><td>99/100 </td><td>ABC42235 </td></tr><tr><td>ORF089 </td><td><em>traV</em> </td><td>− </td><td>60 477–59 962 </td><td>171 </td><td>TraV </td><td>98/99 </td><td>BAA78858 </td></tr><tr><td>ORF090 </td><td><em>trbG</em> </td><td>− </td><td>60 725–60 474 </td><td>83 </td><td>TrbG </td><td>96/98 </td><td>BAA97951 </td></tr><tr><td>ORF091 </td><td><em>trbD</em> </td><td>− </td><td>61 038–60 718 </td><td>106 </td><td>TrbD </td><td>89/94 </td><td>ABC42237 </td></tr><tr><td>ORF092 </td><td><em>traP</em> </td><td>− </td><td>61 612–61 025 </td><td>195 </td><td>TraP </td><td>96/97 </td><td>BAA78856 </td></tr><tr><td>ORF093 </td><td><em>traB</em> </td><td>− </td><td>63 032–61 581 </td><td>483 </td><td>TraB </td><td>99/99 </td><td>BAA78855 </td></tr><tr><td>ORF094 </td><td><em>traK</em> </td><td>− </td><td>63 760–63 032 </td><td>242 </td><td>TraK </td><td>99/99 </td><td>BAA78854 </td></tr><tr><td>ORF095 </td><td><em>traE</em> </td><td>− </td><td>64 313–63 747 </td><td>188 </td><td>TraE </td><td>99/100 </td><td>BAA78853 </td></tr><tr><td>ORF096 </td><td><em>traL</em> </td><td>− </td><td>64 646–64 335 </td><td>103 </td><td>TraL </td><td>99/100 </td><td>BAA97945 </td></tr><tr><td>ORF097 </td><td><em>traA</em> </td><td>− </td><td>65 026–64 661 </td><td>121 </td><td>TraA </td><td>96/98 </td><td>BAA78851 </td></tr><tr><td>ORF098 </td><td><em>traY</em> </td><td>− </td><td>65 453–65 058 </td><td>131 </td><td>TraY </td><td>100/100 </td><td>BAA97943 </td></tr><tr><td>ORF099 </td><td><em>traJ</em> </td><td>− </td><td>66 241–65 552 </td><td>229 </td><td>TraJ </td><td>98/99 </td><td>BAA97942 </td></tr><tr><td>ORF100 </td><td><em>traM</em> </td><td>− </td><td>66 811–66 428 </td><td>127 </td><td>TraM </td><td>98/100 </td><td>ABD51596 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>+, clockwise; −, counterclockwise.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>It originates from kanamycin resistance gene inserted in pEntH10407 to provide a selectable marker.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237577" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP015TB1" class="table-wrap-title" data-id="DSP015TB1"><div class="label -label">Table 1</div><div class="caption"><p>ORFs of pEntH10407K</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ORF </th><th>Gene </th><th>Orientation<sup>a</sup> </th><th>Position (bp) </th><th>Size (aa) </th><th>Homologue by BLAST </th><th>Identity/similarity (%) </th><th>Accession no. </th></tr></thead><tbody><tr><td>ORF001 </td><td><em>mltE</em> </td><td>+ </td><td>39–641 </td><td>200 </td><td>Lytic transglycosylase </td><td>94/96 </td><td>ABG29580 </td></tr><tr><td>ORF002 </td><td> </td><td>− </td><td>801–667 </td><td>44 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>ABD60010 </td></tr><tr><td>ORF003 </td><td> </td><td>− </td><td>1759–938 </td><td>273 </td><td>Conserved hypothetical protein YubP </td><td>9/99 </td><td>BAA78846 </td></tr><tr><td>ORF004 </td><td> </td><td>− </td><td>2166–1870 </td><td>98 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAA78845 </td></tr><tr><td>ORF005 </td><td> </td><td>− </td><td>2318–2190 </td><td>42 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF006 </td><td><em>ppdC</em> </td><td>+ </td><td>2509–2826 </td><td>105 </td><td>Conserved hypothetical protein </td><td>91/93 </td><td>BBA97937 </td></tr><tr><td>ORF007 </td><td> </td><td>− </td><td>2971–2807 </td><td>54 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>EDX27828 </td></tr><tr><td>ORF008 </td><td><em>hok</em> </td><td>− </td><td>3223–3062 </td><td>53 </td><td>Post-segregation killing protein </td><td>72/84 </td><td>P16077 </td></tr><tr><td>ORF009 </td><td><em>mok</em> </td><td>− </td><td>3281–3066 </td><td>71 </td><td>Modulator of post-segregation killing protein </td><td>50/58 </td><td>P23587 </td></tr><tr><td>ORF010 </td><td> </td><td>− </td><td>3746–3432 </td><td>104 </td><td>Zn-dependent dehydrogenases </td><td>99/100 </td><td>CAI79556 </td></tr><tr><td>ORF011 </td><td><em>psiA</em> </td><td>− </td><td>4462–3743 </td><td>239 </td><td>PsiA </td><td>97/98 </td><td>BAA78841 </td></tr><tr><td>ORF012 </td><td><em>psiB</em> </td><td>− </td><td>4893–4459 </td><td>144 </td><td>PsiB </td><td>100/100 </td><td>ABD51587 </td></tr><tr><td>ORF013 </td><td> </td><td>− </td><td>5649–4948 </td><td>233 </td><td>Predicted transcriptional regulator </td><td>91/96 (truncated) </td><td>ABD51586 </td></tr><tr><td>ORF014 </td><td> </td><td>− </td><td>6912–5662 </td><td>416 </td><td>Predicted transcriptional regulator </td><td>96/97 (truncated) </td><td>AAW58879 </td></tr><tr><td>ORF015 </td><td> </td><td>− </td><td>7209–6976 </td><td>77 </td><td>Conserved hypothetical protein </td><td>98/99 </td><td>ABE10669 </td></tr><tr><td>ORF016 </td><td><em>ssb</em> </td><td>− </td><td>7832–7266 </td><td>188 </td><td>Ssb </td><td>94/96 </td><td>BBA78826 </td></tr><tr><td>ORF017 </td><td> </td><td>+ </td><td>7858–8094 </td><td>78 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF018 </td><td> </td><td>− </td><td>8273–8025 </td><td>82 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF019 </td><td> </td><td>+ </td><td>8997–10 664 </td><td>555 </td><td>Conserved hypothetical protein YkfC </td><td>63/77 </td><td>ABI41559 </td></tr><tr><td>ORF020 </td><td> </td><td>− </td><td>11 020–10 778 </td><td>80 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>ZP_00719262 </td></tr><tr><td>ORF021 </td><td> </td><td>− </td><td>11 583–11 020 </td><td>187 </td><td>Conserved hypothetical protein </td><td>97/99 </td><td>BAF33947 </td></tr><tr><td>ORF022 </td><td> </td><td>− </td><td>12 991–11 630 </td><td>453 </td><td>Conserved hypothetical protein </td><td>90/94 </td><td>BAF33946 </td></tr><tr><td>ORF023 </td><td> </td><td>+ </td><td>12 995–13 135 </td><td>46 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>CAP07686 </td></tr><tr><td>ORF024 </td><td> </td><td>− </td><td>13 849–13 415 </td><td>144 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>BBA78815 </td></tr><tr><td>ORF025 </td><td> </td><td>− </td><td>14 084–13 863 </td><td>73 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>ABE10652 </td></tr><tr><td>ORF026 </td><td> </td><td>− </td><td>14 768–14 085 </td><td>227 </td><td>Conserved hypothetical protein </td><td>97/98 </td><td>AAS76 410 </td></tr><tr><td>ORF027 </td><td> </td><td>– </td><td>15 251–14 844 </td><td>135 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>AAW58863 </td></tr><tr><td>ORF028 </td><td><em>stbA</em> </td><td>+ </td><td>15 284–16 246 </td><td>320 </td><td>StbA </td><td>99/99 </td><td>ABD59972 </td></tr><tr><td>ORF029 </td><td><em>stbB</em> </td><td>+ </td><td>16 246–16 599 </td><td>117 </td><td>StbB </td><td>99/99 </td><td>ABD59971 </td></tr><tr><td>ORF030 </td><td><em>insA</em> </td><td>+ </td><td>17 011–17 286 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF031 </td><td><em>insB</em> </td><td>+ </td><td>17 205–17 708 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>100/100 </td><td>AAA96694 </td></tr><tr><td>ORF032 </td><td> </td><td>− </td><td>17 982–17 719 </td><td>87 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF033 </td><td> </td><td>− </td><td>18 983–18 720 </td><td>87 </td><td>Conserved hypothetical protein </td><td>95/97 </td><td>AAS58634 </td></tr><tr><td>ORF034 </td><td><em>insB</em> </td><td>− </td><td>19 618–19 115 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF035 </td><td><em>insA</em> </td><td>− </td><td>19 812–19 537 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>99/100 </td><td>AAA58242 </td></tr><tr><td>ORF036 </td><td> </td><td>+ </td><td>19 925–20 158 </td><td>77 </td><td>ORF1 of IS<em>600</em> </td><td>80/80 (truncated) </td><td>ABB68584 </td></tr><tr><td>ORF037 </td><td> </td><td>+ </td><td>20 208–21 026 </td><td>272 </td><td>ORF2 of IS<em>600</em> </td><td>99/99 </td><td>AAN43456 </td></tr><tr><td>ORF038 </td><td><em>eltB</em> </td><td>− </td><td>21 722–21 348 </td><td>124 </td><td>LT-B </td><td>100/100 </td><td>AAC60441 </td></tr><tr><td>ORF039 </td><td><em>eltA</em> </td><td>− </td><td>22 549–21 719 </td><td>276 </td><td>LT-A </td><td>100/100 </td><td>P43530 </td></tr><tr><td>ORF040 </td><td> </td><td>+ </td><td>22 812–23 084 </td><td>90 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>AAZ91090 </td></tr><tr><td>ORF041 </td><td> </td><td>+ </td><td>23 065–23 334 </td><td>89 </td><td>Transposase of IS<em>801</em> </td><td>76/79 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF042 </td><td> </td><td>+ </td><td>23 250–23 564 </td><td>104 </td><td>Transposase of IS<em>801</em> </td><td>99/99 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF043 </td><td> </td><td>+ </td><td>23 779–25 002 </td><td>407 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>AAT35239 </td></tr><tr><td>ORF044 </td><td> </td><td>− </td><td>25 347–24 928 </td><td>139 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>CAI79504 </td></tr><tr><td>ORF045 </td><td> </td><td>+ </td><td>25 487–26 164 </td><td>225 </td><td>ORF1 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF046 </td><td> </td><td>+ </td><td>26 164–26 511 </td><td>115 </td><td>ORF2 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF047 </td><td> </td><td>+ </td><td>26 531–28 102 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF048 </td><td><em>estIa</em> </td><td>− </td><td>28 394–28 176 </td><td>72 </td><td>STIa </td><td>100/100 </td><td>P01559 </td></tr><tr><td>ORF049 </td><td><em>insB</em> </td><td>− </td><td>29 343–28 840 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF050 </td><td><em>insA</em> </td><td>− </td><td>29 537–29 262 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF051 </td><td> </td><td>+ </td><td>29 641–29 796 </td><td>51 </td><td>Putative transposase </td><td>98/100 (truncated) </td><td>CAA07835 </td></tr><tr><td>ORF052 </td><td> </td><td>− </td><td>30 316–30 008 </td><td>102 </td><td>Predicted transcriptional regulator </td><td>100/100 </td><td>ZP_00713086 </td></tr><tr><td>ORF053 </td><td> </td><td>+ </td><td>30 288–30 512 </td><td>74 </td><td>Putative transposase </td><td>92/94 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF054<sup>b</sup> </td><td> </td><td>+ </td><td>30 664–31 479 </td><td>271 </td><td>Aminoglucoside 3′-phosphotransferase </td><td>100/100 </td><td>AAA80260 </td></tr><tr><td>ORF055 </td><td> </td><td>+ </td><td>32 512–32 628 </td><td>38 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>ACD54240 </td></tr><tr><td>ORF056 </td><td><em>baeS</em> </td><td>− </td><td>34 173–32 701 </td><td>490 </td><td>BaeS </td><td>73/86 </td><td>ABE10335 </td></tr><tr><td>ORF057 </td><td><em>ompR</em> </td><td>− </td><td>34 892–34 170 </td><td>240 </td><td>BaeR </td><td>83/90 </td><td>ABE10334 </td></tr><tr><td>ORF058 </td><td><em>ydhU</em> </td><td>+ </td><td>35 034–35 633 </td><td>199 </td><td>Thiosulphate reductase cytochrome B subunit </td><td>67/82 </td><td>CAD42043 </td></tr><tr><td>ORF059 </td><td> </td><td>+ </td><td>35 644–36 414 </td><td>256 </td><td>Oxidoreductase, molybdopterin-binding subunit </td><td>82/91 </td><td>CAD42042 </td></tr><tr><td>ORF060 </td><td> </td><td>+ </td><td>36 444–36 683 </td><td>79 </td><td>Conserved hypothetical protein </td><td>59/70 (truncated) </td><td>ABE10331 </td></tr><tr><td>ORF061 </td><td> </td><td>− </td><td>39 074–37 563 </td><td>503 </td><td>Putative ATP binding protein </td><td>25/43 </td><td>CAD16966 </td></tr><tr><td>ORF062 </td><td><em>stbE</em> </td><td>− </td><td>39 721–39 434 </td><td>95 </td><td>RelE </td><td>93/96 </td><td>ABD51640 </td></tr><tr><td>ORF063 </td><td><em>stbD</em> </td><td>− </td><td>39 969–39 718 </td><td>83 </td><td>RelB </td><td>100/100 </td><td>ABD51639 </td></tr><tr><td>ORF064 </td><td> </td><td>− </td><td>40 233–40 042 </td><td>63 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>AAL72549 </td></tr><tr><td>ORF065 </td><td><em>repA4</em> </td><td>− </td><td>40 370–40 185 </td><td>61 </td><td>RepA4 </td><td>90/93 (truncated) </td><td>BAA78895 </td></tr><tr><td>ORF066 </td><td><em>repA4</em> </td><td>− </td><td>40 573–40 325 </td><td>82 </td><td>RepA4 </td><td>88/90 (truncated) </td><td>ABC42205 </td></tr><tr><td>ORF067 </td><td> </td><td>− </td><td>40 873–40 703 </td><td>56 </td><td>Conserved hypothetical protein </td><td>81/85 </td><td>ACD06080 </td></tr><tr><td>ORF068 </td><td><em>repA1</em> </td><td>− </td><td>41 793–40 936 </td><td>285 </td><td>RepA1 </td><td>99/100 </td><td>CAI79519 </td></tr><tr><td>ORF069 </td><td><em>tapA</em> </td><td>− </td><td>41 860–41 786 </td><td>24 </td><td>TapA </td><td>100/100 </td><td>BBA78893 </td></tr><tr><td>ORF070 </td><td><em>repA3</em> </td><td>− </td><td>41 937–41 806 </td><td>43 </td><td>RepA3 </td><td>87/89 (truncated) </td><td>AAA26066 </td></tr><tr><td>ORF071 </td><td><em>repA2</em> </td><td>− </td><td>42 354–42 094 </td><td>86 </td><td>RepA2 </td><td>99/100 </td><td>ABE10578 </td></tr><tr><td>ORF072 </td><td> </td><td>− </td><td>43 184–42 594 </td><td>196 </td><td>Superfamily I DNA/RNA helicase </td><td>100/100 </td><td>AAW58927 </td></tr><tr><td>ORF073 </td><td> </td><td>− </td><td>43 430–43 227 </td><td>67 </td><td>YmoA </td><td>97/97 </td><td>AAO49553 </td></tr><tr><td>ORF074 </td><td> </td><td>− </td><td>43 937–43 476 </td><td>153 </td><td>Thermonuclease family protein </td><td>98/98 </td><td>ABE10720 </td></tr><tr><td>ORF075 </td><td> </td><td>− </td><td>44 394–44 182 </td><td>70 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAF33997 </td></tr><tr><td>ORF076 </td><td><em>finO</em> </td><td>− </td><td>45 086–44 526 </td><td>186 </td><td>FinO </td><td>99/99 </td><td>AAC70069 </td></tr><tr><td>ORF077 </td><td><em>traX</em> </td><td>− </td><td>45 944–45 141 </td><td>267 </td><td>TraX </td><td>99/100 </td><td>BAF33995 </td></tr><tr><td>ORF078 </td><td><em>traI</em> </td><td>− </td><td>51 177–45 907 </td><td>1756 </td><td>TraI </td><td>98/99 </td><td>CAA39337 </td></tr><tr><td>ORF079 </td><td><em>traD</em> </td><td>− </td><td>53 465–51 177 </td><td>762 </td><td>TraD </td><td>96/96 </td><td>BAA78884 </td></tr><tr><td>ORF080 </td><td> </td><td>− </td><td>53 944–53 516 </td><td>142 </td><td>Conserved hypothetical protein YhfA </td><td>99/99 (truncated) </td><td>ABE10712 </td></tr><tr><td>ORF081 </td><td> </td><td>− </td><td>55 646–54 075 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF082 </td><td> </td><td>− </td><td>56 013–55 666 </td><td>115 </td><td>ORF2 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF083 </td><td> </td><td>− </td><td>56 690–56 013 </td><td>225 </td><td>ORF1 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF084 </td><td> </td><td>+ </td><td>56 910–57 311 </td><td>133 </td><td>L0013 of IS<em>Ec8</em> </td><td>99/100 </td><td>ABG71816 </td></tr><tr><td>ORF085 </td><td> </td><td>+ </td><td>57 308–57 655 </td><td>115 </td><td>L0014 of IS<em>Ec8</em> </td><td>100/100 </td><td>AAG54624 </td></tr><tr><td>ORF086 </td><td> </td><td>+ </td><td>57 705–59 243 </td><td>512 </td><td>L0015 of IS<em>Ec8</em> </td><td>99/99 </td><td>AAG54625 </td></tr><tr><td>ORF087 </td><td> </td><td>− </td><td>59 613–59 377 </td><td>78 </td><td>Conserved hypothetical protein YfhA </td><td>99/99 (truncated) </td><td>ABD60024 </td></tr><tr><td>ORF088 </td><td><em>traR</em> </td><td>− </td><td>59 827–59 606 </td><td>73 </td><td>TraR </td><td>99/100 </td><td>ABC42235 </td></tr><tr><td>ORF089 </td><td><em>traV</em> </td><td>− </td><td>60 477–59 962 </td><td>171 </td><td>TraV </td><td>98/99 </td><td>BAA78858 </td></tr><tr><td>ORF090 </td><td><em>trbG</em> </td><td>− </td><td>60 725–60 474 </td><td>83 </td><td>TrbG </td><td>96/98 </td><td>BAA97951 </td></tr><tr><td>ORF091 </td><td><em>trbD</em> </td><td>− </td><td>61 038–60 718 </td><td>106 </td><td>TrbD </td><td>89/94 </td><td>ABC42237 </td></tr><tr><td>ORF092 </td><td><em>traP</em> </td><td>− </td><td>61 612–61 025 </td><td>195 </td><td>TraP </td><td>96/97 </td><td>BAA78856 </td></tr><tr><td>ORF093 </td><td><em>traB</em> </td><td>− </td><td>63 032–61 581 </td><td>483 </td><td>TraB </td><td>99/99 </td><td>BAA78855 </td></tr><tr><td>ORF094 </td><td><em>traK</em> </td><td>− </td><td>63 760–63 032 </td><td>242 </td><td>TraK </td><td>99/99 </td><td>BAA78854 </td></tr><tr><td>ORF095 </td><td><em>traE</em> </td><td>− </td><td>64 313–63 747 </td><td>188 </td><td>TraE </td><td>99/100 </td><td>BAA78853 </td></tr><tr><td>ORF096 </td><td><em>traL</em> </td><td>− </td><td>64 646–64 335 </td><td>103 </td><td>TraL </td><td>99/100 </td><td>BAA97945 </td></tr><tr><td>ORF097 </td><td><em>traA</em> </td><td>− </td><td>65 026–64 661 </td><td>121 </td><td>TraA </td><td>96/98 </td><td>BAA78851 </td></tr><tr><td>ORF098 </td><td><em>traY</em> </td><td>− </td><td>65 453–65 058 </td><td>131 </td><td>TraY </td><td>100/100 </td><td>BAA97943 </td></tr><tr><td>ORF099 </td><td><em>traJ</em> </td><td>− </td><td>66 241–65 552 </td><td>229 </td><td>TraJ </td><td>98/99 </td><td>BAA97942 </td></tr><tr><td>ORF100 </td><td><em>traM</em> </td><td>− </td><td>66 811–66 428 </td><td>127 </td><td>TraM </td><td>98/100 </td><td>ABD51596 </td></tr></tbody></table></div><div content-id="DSP015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>ORF </th><th>Gene </th><th>Orientation<sup>a</sup> </th><th>Position (bp) </th><th>Size (aa) </th><th>Homologue by BLAST </th><th>Identity/similarity (%) </th><th>Accession no. </th></tr></thead><tbody><tr><td>ORF001 </td><td><em>mltE</em> </td><td>+ </td><td>39–641 </td><td>200 </td><td>Lytic transglycosylase </td><td>94/96 </td><td>ABG29580 </td></tr><tr><td>ORF002 </td><td> </td><td>− </td><td>801–667 </td><td>44 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>ABD60010 </td></tr><tr><td>ORF003 </td><td> </td><td>− </td><td>1759–938 </td><td>273 </td><td>Conserved hypothetical protein YubP </td><td>9/99 </td><td>BAA78846 </td></tr><tr><td>ORF004 </td><td> </td><td>− </td><td>2166–1870 </td><td>98 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAA78845 </td></tr><tr><td>ORF005 </td><td> </td><td>− </td><td>2318–2190 </td><td>42 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF006 </td><td><em>ppdC</em> </td><td>+ </td><td>2509–2826 </td><td>105 </td><td>Conserved hypothetical protein </td><td>91/93 </td><td>BBA97937 </td></tr><tr><td>ORF007 </td><td> </td><td>− </td><td>2971–2807 </td><td>54 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>EDX27828 </td></tr><tr><td>ORF008 </td><td><em>hok</em> </td><td>− </td><td>3223–3062 </td><td>53 </td><td>Post-segregation killing protein </td><td>72/84 </td><td>P16077 </td></tr><tr><td>ORF009 </td><td><em>mok</em> </td><td>− </td><td>3281–3066 </td><td>71 </td><td>Modulator of post-segregation killing protein </td><td>50/58 </td><td>P23587 </td></tr><tr><td>ORF010 </td><td> </td><td>− </td><td>3746–3432 </td><td>104 </td><td>Zn-dependent dehydrogenases </td><td>99/100 </td><td>CAI79556 </td></tr><tr><td>ORF011 </td><td><em>psiA</em> </td><td>− </td><td>4462–3743 </td><td>239 </td><td>PsiA </td><td>97/98 </td><td>BAA78841 </td></tr><tr><td>ORF012 </td><td><em>psiB</em> </td><td>− </td><td>4893–4459 </td><td>144 </td><td>PsiB </td><td>100/100 </td><td>ABD51587 </td></tr><tr><td>ORF013 </td><td> </td><td>− </td><td>5649–4948 </td><td>233 </td><td>Predicted transcriptional regulator </td><td>91/96 (truncated) </td><td>ABD51586 </td></tr><tr><td>ORF014 </td><td> </td><td>− </td><td>6912–5662 </td><td>416 </td><td>Predicted transcriptional regulator </td><td>96/97 (truncated) </td><td>AAW58879 </td></tr><tr><td>ORF015 </td><td> </td><td>− </td><td>7209–6976 </td><td>77 </td><td>Conserved hypothetical protein </td><td>98/99 </td><td>ABE10669 </td></tr><tr><td>ORF016 </td><td><em>ssb</em> </td><td>− </td><td>7832–7266 </td><td>188 </td><td>Ssb </td><td>94/96 </td><td>BBA78826 </td></tr><tr><td>ORF017 </td><td> </td><td>+ </td><td>7858–8094 </td><td>78 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF018 </td><td> </td><td>− </td><td>8273–8025 </td><td>82 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF019 </td><td> </td><td>+ </td><td>8997–10 664 </td><td>555 </td><td>Conserved hypothetical protein YkfC </td><td>63/77 </td><td>ABI41559 </td></tr><tr><td>ORF020 </td><td> </td><td>− </td><td>11 020–10 778 </td><td>80 </td><td>Conserved hypothetical protein </td><td>93/93 </td><td>ZP_00719262 </td></tr><tr><td>ORF021 </td><td> </td><td>− </td><td>11 583–11 020 </td><td>187 </td><td>Conserved hypothetical protein </td><td>97/99 </td><td>BAF33947 </td></tr><tr><td>ORF022 </td><td> </td><td>− </td><td>12 991–11 630 </td><td>453 </td><td>Conserved hypothetical protein </td><td>90/94 </td><td>BAF33946 </td></tr><tr><td>ORF023 </td><td> </td><td>+ </td><td>12 995–13 135 </td><td>46 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>CAP07686 </td></tr><tr><td>ORF024 </td><td> </td><td>− </td><td>13 849–13 415 </td><td>144 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>BBA78815 </td></tr><tr><td>ORF025 </td><td> </td><td>− </td><td>14 084–13 863 </td><td>73 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>ABE10652 </td></tr><tr><td>ORF026 </td><td> </td><td>− </td><td>14 768–14 085 </td><td>227 </td><td>Conserved hypothetical protein </td><td>97/98 </td><td>AAS76 410 </td></tr><tr><td>ORF027 </td><td> </td><td>– </td><td>15 251–14 844 </td><td>135 </td><td>Conserved hypothetical protein </td><td>94/97 </td><td>AAW58863 </td></tr><tr><td>ORF028 </td><td><em>stbA</em> </td><td>+ </td><td>15 284–16 246 </td><td>320 </td><td>StbA </td><td>99/99 </td><td>ABD59972 </td></tr><tr><td>ORF029 </td><td><em>stbB</em> </td><td>+ </td><td>16 246–16 599 </td><td>117 </td><td>StbB </td><td>99/99 </td><td>ABD59971 </td></tr><tr><td>ORF030 </td><td><em>insA</em> </td><td>+ </td><td>17 011–17 286 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF031 </td><td><em>insB</em> </td><td>+ </td><td>17 205–17 708 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>100/100 </td><td>AAA96694 </td></tr><tr><td>ORF032 </td><td> </td><td>− </td><td>17 982–17 719 </td><td>87 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>ORF033 </td><td> </td><td>− </td><td>18 983–18 720 </td><td>87 </td><td>Conserved hypothetical protein </td><td>95/97 </td><td>AAS58634 </td></tr><tr><td>ORF034 </td><td><em>insB</em> </td><td>− </td><td>19 618–19 115 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF035 </td><td><em>insA</em> </td><td>− </td><td>19 812–19 537 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>99/100 </td><td>AAA58242 </td></tr><tr><td>ORF036 </td><td> </td><td>+ </td><td>19 925–20 158 </td><td>77 </td><td>ORF1 of IS<em>600</em> </td><td>80/80 (truncated) </td><td>ABB68584 </td></tr><tr><td>ORF037 </td><td> </td><td>+ </td><td>20 208–21 026 </td><td>272 </td><td>ORF2 of IS<em>600</em> </td><td>99/99 </td><td>AAN43456 </td></tr><tr><td>ORF038 </td><td><em>eltB</em> </td><td>− </td><td>21 722–21 348 </td><td>124 </td><td>LT-B </td><td>100/100 </td><td>AAC60441 </td></tr><tr><td>ORF039 </td><td><em>eltA</em> </td><td>− </td><td>22 549–21 719 </td><td>276 </td><td>LT-A </td><td>100/100 </td><td>P43530 </td></tr><tr><td>ORF040 </td><td> </td><td>+ </td><td>22 812–23 084 </td><td>90 </td><td>Conserved hypothetical protein </td><td>99/100 </td><td>AAZ91090 </td></tr><tr><td>ORF041 </td><td> </td><td>+ </td><td>23 065–23 334 </td><td>89 </td><td>Transposase of IS<em>801</em> </td><td>76/79 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF042 </td><td> </td><td>+ </td><td>23 250–23 564 </td><td>104 </td><td>Transposase of IS<em>801</em> </td><td>99/99 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF043 </td><td> </td><td>+ </td><td>23 779–25 002 </td><td>407 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>AAT35239 </td></tr><tr><td>ORF044 </td><td> </td><td>− </td><td>25 347–24 928 </td><td>139 </td><td>Putative transposase </td><td>99/99 (truncated) </td><td>CAI79504 </td></tr><tr><td>ORF045 </td><td> </td><td>+ </td><td>25 487–26 164 </td><td>225 </td><td>ORF1 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF046 </td><td> </td><td>+ </td><td>26 164–26 511 </td><td>115 </td><td>ORF2 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF047 </td><td> </td><td>+ </td><td>26 531–28 102 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF048 </td><td><em>estIa</em> </td><td>− </td><td>28 394–28 176 </td><td>72 </td><td>STIa </td><td>100/100 </td><td>P01559 </td></tr><tr><td>ORF049 </td><td><em>insB</em> </td><td>− </td><td>29 343–28 840 </td><td>167 </td><td>InsB of IS<em>1</em> </td><td>99/99 </td><td>AAA96694 </td></tr><tr><td>ORF050 </td><td><em>insA</em> </td><td>− </td><td>29 537–29 262 </td><td>91 </td><td>InsA of IS<em>1</em> </td><td>100/100 </td><td>AAA58242 </td></tr><tr><td>ORF051 </td><td> </td><td>+ </td><td>29 641–29 796 </td><td>51 </td><td>Putative transposase </td><td>98/100 (truncated) </td><td>CAA07835 </td></tr><tr><td>ORF052 </td><td> </td><td>− </td><td>30 316–30 008 </td><td>102 </td><td>Predicted transcriptional regulator </td><td>100/100 </td><td>ZP_00713086 </td></tr><tr><td>ORF053 </td><td> </td><td>+ </td><td>30 288–30 512 </td><td>74 </td><td>Putative transposase </td><td>92/94 (truncated) </td><td>AAM14707 </td></tr><tr><td>ORF054<sup>b</sup> </td><td> </td><td>+ </td><td>30 664–31 479 </td><td>271 </td><td>Aminoglucoside 3′-phosphotransferase </td><td>100/100 </td><td>AAA80260 </td></tr><tr><td>ORF055 </td><td> </td><td>+ </td><td>32 512–32 628 </td><td>38 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>ACD54240 </td></tr><tr><td>ORF056 </td><td><em>baeS</em> </td><td>− </td><td>34 173–32 701 </td><td>490 </td><td>BaeS </td><td>73/86 </td><td>ABE10335 </td></tr><tr><td>ORF057 </td><td><em>ompR</em> </td><td>− </td><td>34 892–34 170 </td><td>240 </td><td>BaeR </td><td>83/90 </td><td>ABE10334 </td></tr><tr><td>ORF058 </td><td><em>ydhU</em> </td><td>+ </td><td>35 034–35 633 </td><td>199 </td><td>Thiosulphate reductase cytochrome B subunit </td><td>67/82 </td><td>CAD42043 </td></tr><tr><td>ORF059 </td><td> </td><td>+ </td><td>35 644–36 414 </td><td>256 </td><td>Oxidoreductase, molybdopterin-binding subunit </td><td>82/91 </td><td>CAD42042 </td></tr><tr><td>ORF060 </td><td> </td><td>+ </td><td>36 444–36 683 </td><td>79 </td><td>Conserved hypothetical protein </td><td>59/70 (truncated) </td><td>ABE10331 </td></tr><tr><td>ORF061 </td><td> </td><td>− </td><td>39 074–37 563 </td><td>503 </td><td>Putative ATP binding protein </td><td>25/43 </td><td>CAD16966 </td></tr><tr><td>ORF062 </td><td><em>stbE</em> </td><td>− </td><td>39 721–39 434 </td><td>95 </td><td>RelE </td><td>93/96 </td><td>ABD51640 </td></tr><tr><td>ORF063 </td><td><em>stbD</em> </td><td>− </td><td>39 969–39 718 </td><td>83 </td><td>RelB </td><td>100/100 </td><td>ABD51639 </td></tr><tr><td>ORF064 </td><td> </td><td>− </td><td>40 233–40 042 </td><td>63 </td><td>Conserved hypothetical protein </td><td>95/95 </td><td>AAL72549 </td></tr><tr><td>ORF065 </td><td><em>repA4</em> </td><td>− </td><td>40 370–40 185 </td><td>61 </td><td>RepA4 </td><td>90/93 (truncated) </td><td>BAA78895 </td></tr><tr><td>ORF066 </td><td><em>repA4</em> </td><td>− </td><td>40 573–40 325 </td><td>82 </td><td>RepA4 </td><td>88/90 (truncated) </td><td>ABC42205 </td></tr><tr><td>ORF067 </td><td> </td><td>− </td><td>40 873–40 703 </td><td>56 </td><td>Conserved hypothetical protein </td><td>81/85 </td><td>ACD06080 </td></tr><tr><td>ORF068 </td><td><em>repA1</em> </td><td>− </td><td>41 793–40 936 </td><td>285 </td><td>RepA1 </td><td>99/100 </td><td>CAI79519 </td></tr><tr><td>ORF069 </td><td><em>tapA</em> </td><td>− </td><td>41 860–41 786 </td><td>24 </td><td>TapA </td><td>100/100 </td><td>BBA78893 </td></tr><tr><td>ORF070 </td><td><em>repA3</em> </td><td>− </td><td>41 937–41 806 </td><td>43 </td><td>RepA3 </td><td>87/89 (truncated) </td><td>AAA26066 </td></tr><tr><td>ORF071 </td><td><em>repA2</em> </td><td>− </td><td>42 354–42 094 </td><td>86 </td><td>RepA2 </td><td>99/100 </td><td>ABE10578 </td></tr><tr><td>ORF072 </td><td> </td><td>− </td><td>43 184–42 594 </td><td>196 </td><td>Superfamily I DNA/RNA helicase </td><td>100/100 </td><td>AAW58927 </td></tr><tr><td>ORF073 </td><td> </td><td>− </td><td>43 430–43 227 </td><td>67 </td><td>YmoA </td><td>97/97 </td><td>AAO49553 </td></tr><tr><td>ORF074 </td><td> </td><td>− </td><td>43 937–43 476 </td><td>153 </td><td>Thermonuclease family protein </td><td>98/98 </td><td>ABE10720 </td></tr><tr><td>ORF075 </td><td> </td><td>− </td><td>44 394–44 182 </td><td>70 </td><td>Conserved hypothetical protein </td><td>100/100 </td><td>BAF33997 </td></tr><tr><td>ORF076 </td><td><em>finO</em> </td><td>− </td><td>45 086–44 526 </td><td>186 </td><td>FinO </td><td>99/99 </td><td>AAC70069 </td></tr><tr><td>ORF077 </td><td><em>traX</em> </td><td>− </td><td>45 944–45 141 </td><td>267 </td><td>TraX </td><td>99/100 </td><td>BAF33995 </td></tr><tr><td>ORF078 </td><td><em>traI</em> </td><td>− </td><td>51 177–45 907 </td><td>1756 </td><td>TraI </td><td>98/99 </td><td>CAA39337 </td></tr><tr><td>ORF079 </td><td><em>traD</em> </td><td>− </td><td>53 465–51 177 </td><td>762 </td><td>TraD </td><td>96/96 </td><td>BAA78884 </td></tr><tr><td>ORF080 </td><td> </td><td>− </td><td>53 944–53 516 </td><td>142 </td><td>Conserved hypothetical protein YhfA </td><td>99/99 (truncated) </td><td>ABE10712 </td></tr><tr><td>ORF081 </td><td> </td><td>− </td><td>55 646–54 075 </td><td>523 </td><td>ORF3 of IS<em>Ec8</em>-like IS </td><td>100/100 </td><td>AAW51736 </td></tr><tr><td>ORF082 </td><td> </td><td>− </td><td>56 013–55 666 </td><td>115 </td><td>ORF2 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51735 </td></tr><tr><td>ORF083 </td><td> </td><td>− </td><td>56 690–56 013 </td><td>225 </td><td>ORF1 of IS<em>Ec8-</em>like IS </td><td>100/100 </td><td>AAW51734 </td></tr><tr><td>ORF084 </td><td> </td><td>+ </td><td>56 910–57 311 </td><td>133 </td><td>L0013 of IS<em>Ec8</em> </td><td>99/100 </td><td>ABG71816 </td></tr><tr><td>ORF085 </td><td> </td><td>+ </td><td>57 308–57 655 </td><td>115 </td><td>L0014 of IS<em>Ec8</em> </td><td>100/100 </td><td>AAG54624 </td></tr><tr><td>ORF086 </td><td> </td><td>+ </td><td>57 705–59 243 </td><td>512 </td><td>L0015 of IS<em>Ec8</em> </td><td>99/99 </td><td>AAG54625 </td></tr><tr><td>ORF087 </td><td> </td><td>− </td><td>59 613–59 377 </td><td>78 </td><td>Conserved hypothetical protein YfhA </td><td>99/99 (truncated) </td><td>ABD60024 </td></tr><tr><td>ORF088 </td><td><em>traR</em> </td><td>− </td><td>59 827–59 606 </td><td>73 </td><td>TraR </td><td>99/100 </td><td>ABC42235 </td></tr><tr><td>ORF089 </td><td><em>traV</em> </td><td>− </td><td>60 477–59 962 </td><td>171 </td><td>TraV </td><td>98/99 </td><td>BAA78858 </td></tr><tr><td>ORF090 </td><td><em>trbG</em> </td><td>− </td><td>60 725–60 474 </td><td>83 </td><td>TrbG </td><td>96/98 </td><td>BAA97951 </td></tr><tr><td>ORF091 </td><td><em>trbD</em> </td><td>− </td><td>61 038–60 718 </td><td>106 </td><td>TrbD </td><td>89/94 </td><td>ABC42237 </td></tr><tr><td>ORF092 </td><td><em>traP</em> </td><td>− </td><td>61 612–61 025 </td><td>195 </td><td>TraP </td><td>96/97 </td><td>BAA78856 </td></tr><tr><td>ORF093 </td><td><em>traB</em> </td><td>− </td><td>63 032–61 581 </td><td>483 </td><td>TraB </td><td>99/99 </td><td>BAA78855 </td></tr><tr><td>ORF094 </td><td><em>traK</em> </td><td>− </td><td>63 760–63 032 </td><td>242 </td><td>TraK </td><td>99/99 </td><td>BAA78854 </td></tr><tr><td>ORF095 </td><td><em>traE</em> </td><td>− </td><td>64 313–63 747 </td><td>188 </td><td>TraE </td><td>99/100 </td><td>BAA78853 </td></tr><tr><td>ORF096 </td><td><em>traL</em> </td><td>− </td><td>64 646–64 335 </td><td>103 </td><td>TraL </td><td>99/100 </td><td>BAA97945 </td></tr><tr><td>ORF097 </td><td><em>traA</em> </td><td>− </td><td>65 026–64 661 </td><td>121 </td><td>TraA </td><td>96/98 </td><td>BAA78851 </td></tr><tr><td>ORF098 </td><td><em>traY</em> </td><td>− </td><td>65 453–65 058 </td><td>131 </td><td>TraY </td><td>100/100 </td><td>BAA97943 </td></tr><tr><td>ORF099 </td><td><em>traJ</em> </td><td>− </td><td>66 241–65 552 </td><td>229 </td><td>TraJ </td><td>98/99 </td><td>BAA97942 </td></tr><tr><td>ORF100 </td><td><em>traM</em> </td><td>− </td><td>66 811–66 428 </td><td>127 </td><td>TraM </td><td>98/100 </td><td>ABD51596 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>+, clockwise; −, counterclockwise.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>It originates from kanamycin resistance gene inserted in pEntH10407 to provide a selectable marker.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237577" target="_blank">View Large</a></div></div><p>A total of 20 pEntH10407K ORFs (21.1%) have already been reported. These genes encode the major virulence factors accounting for LT and STIa (STp)<sup><a reveal-id="DSP015C7" data-open="DSP015C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP015C17" data-open="DSP015C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and Tra proteins in the plasmid. The toxin region, containing the LT and STIa genes (<em>elt</em> and <em>estIa</em>), was 13 501 bp in length. This region carries three virulence-associated genes (<em>eltAB</em> and <em>estIa</em>). <em>eltAB</em> is 37.6% G + C and <em>estIa</em> is 30.1% G + C. In addition, this region is shown to contain three IS<em>1</em>s (IS<em>1</em>A, B and C), IS<em>600</em> (Table <a reveal-id="DSP015TB1" data-open="DSP015TB1" class="link link-reveal link-table xref-fig">1</a>) and an IS<em>Ec8</em>-like element (Table <a reveal-id="DSP015TB1" data-open="DSP015TB1" class="link link-reveal link-table xref-fig">1</a>). IS<em>1</em>A, IS<em>1</em>B and IS<em>600</em> are on the same side of the toxin region and IS<em>1</em>C is on the other side (Table <a reveal-id="DSP015TB1" data-open="DSP015TB1" class="link link-reveal link-table xref-fig">1</a>). The IS<em>Ec8</em>-like element is located between <em>eltAB</em> and <em>estIa</em>. The outer end (OE) terminal inverted repeats (IR) of IS<em>1</em>A, IS<em>1</em>B and IS<em>600</em> begin 4014, 1910 and 696 bp downstream of <em>eltB</em>, respectively, and the OE terminal IR of IS<em>1</em>C ends 446 bp upstream of <em>estIa</em>. Although IS<em>1</em>A, IS<em>1</em>B and IS<em>600</em> have complete direct repeats (DR) on both sides, IS<em>1</em>C has DR sequences only on one side. Moreover, in this region, there were seven transposase genes between <em>eltA</em> and <em>estIa</em>, and two transposase genes were found upstream of <em>estIa</em>. These observations suggest that the toxin region has evolved through multiple transposition events and may have been transferred horizontally into pEntH10407 with IS<em>1</em>s, as predicted previously.<sup><a reveal-id="DSP015C18" data-open="DSP015C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSP015C19" data-open="DSP015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The organization of the toxin region in pEntH10407K is suggestive of a pathogenicity islet,<sup><a reveal-id="DSP015C20" data-open="DSP015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> an island of small pieces of DNA (1–10 kb).</p><p>Proteins involved in plasmid replication and DNA maintenance (Table <a reveal-id="DSP015TB1" data-open="DSP015TB1" class="link link-reveal link-table xref-fig">1</a>) are encoded by 16 ORFs (16.8% of pEntH10407K), constituting a putative replication region characteristic of IncFIIA plasmids. RepA4 (ORF065, ORF066), RepA1 (ORF068), TapA (ORF069), RepA3 (ORF070) and RepA2 (ORF071) of pEntH10407K exhibit 53.8–100.0% amino acid sequence identity with the corresponding proteins from plasmid R100.<sup><a reveal-id="DSP015C21  DSP015C23" data-open="DSP015C21  DSP015C23" class="link link-ref link-reveal xref-bibr">21–23</a></sup> As for other IncFII replicons, a 9 bp DnaA box (TTATCCACA) and a putative origin of replication (<em>oriR</em>) were detected downstream of <em>repA1</em> on pEntH10407K. The putative CopA antisense RNA of pEntH10407K exhibits 89.9% DNA sequence identity with the corresponding sequence of plasmid R100, which suggests that the copy number of pEntH10407 might be similar to that of R100 (NR1), i.e. one or two copies per chromosome.</p><p>Plasmid segregation systems are essential for inheritance of low-copy number plasmids in daughter cells. Sequence analysis revealed the presence of three segregation systems on pEntH10407K. The first system, designated <em>hok/sok</em> (bp 3062–3330), is similar to the <em>hok/sok</em> system of plasmid R100,<sup><a reveal-id="DSP015C24" data-open="DSP015C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSP015C25" data-open="DSP015C25" class="link link-ref link-reveal xref-bibr">25</a></sup> the second system, designated <em>stb</em> (bp 15 284–16 599), is similar to the <em>stbAB</em> (<em>parMRC</em>) system of R100,<sup><a reveal-id="DSP015C26" data-open="DSP015C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSP015C27" data-open="DSP015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and the third system, designated <em>stbDE</em> (bp 39 434–39 969), is similar to the <em>stbDE</em> system of pSS, a large virulence plasmid found in <em>Shigella sonnei</em>.<sup><a reveal-id="DSP015C28" data-open="DSP015C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The <em>hok/sok</em> system of R100 consists of two <em>hok</em> and <em>mok</em> genes, organized in an operon, and a locus denoted <em>sok</em>. Orf008 and Orf009 of pEntH10407K exhibit 70.6 and 50.0% amino acid sequence identity to Hok and Mok, respectively, of R100, and the region located upstream of <em>hok</em> on pEntH10407K exhibits 82.0% DNA sequence identity to the region (<em>sok</em>) upstream of the <em>hok</em> of R100. With regard to the second segregation system, StbA and StbB of pEntH10407K exhibit 99.1 and 99.1% amino acid sequence identity with StbA (ParM) and StbB (ParR), respectively, of R100, and a <em>cis</em>-acting site (<em>parC</em>) (90.4% DNA sequence identity with the corresponding sequence in R100) is present upstream of <em>stbA</em> on pEntH10407K. These similarities suggest that the <em>hok/sok</em> and <em>stbAB</em> systems of pEntH10407K are functional. In regard to the third segregation system, StbD and StbE of pEntH10407K exhibit 98.8 and 93.7% amino acid sequence identity with StbD and StbE, respectively, of pSS. StbDE is a novel segregational stability system that was identified on plasmid R485, which originates from <em>Morganella morganii</em>. Experimental data have shown that the StbE protein may be toxic to its host and that StbD is likely to be an antitoxin protein.<sup><a reveal-id="DSP015C29" data-open="DSP015C29" class="link link-ref link-reveal xref-bibr">29</a></sup> This region shows high similarity to the corresponding sequence of the enteropathogenic <em>E. coli</em> plasmid pB171. The ORFs encoding StbD (83 aa) and StbE (95 aa) are totally identical in both nucleic acid and protein sequence to orf44 and orf43 on pB171 (accession no. AB024946). Homologues of the <em>stbDE</em> genes were also identified on the enterohaemorrhagic <em>E. coli</em> plasmid pO86A1 (pO86A1_p141 and p140 in accession no. AB255435) and on the chromosomes of some pathogenic bacteria.</p><p>Orf011 and Orf012 were found to be very similar to PsiA (97.1% amino acid sequence identity) and PsiB (99.3% amino acid sequence identity), respectively, on R100. They function as inhibitors of SOS induction in bacteria, including <em>E. coli</em>. Orf016 shows high identity (93.7%) at the amino acid sequence level with a single-strand DNA binding protein (SSB) of R100.</p><p>The complete <em>tra</em> region responsible for conjugal transfer is composed of 40 and 37 ORFs in the R100 and F plasmids,<sup><a reveal-id="DSP015C30" data-open="DSP015C30" class="link link-ref link-reveal xref-bibr">30</a>–<a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a></sup> respectively. However, only 17 genes showing similarity to the <em>tra</em> genes of other bacteria were found in pEntH10407K. This incomplete <em>tra</em> region consisted of <em>traM</em>, <em>traJ</em>, <em>traY</em>, <em>traA</em>, <em>traL</em>, <em>traE</em>, <em>traK</em>, <em>traB</em>, <em>traP</em>, <em>trbD</em>, <em>trbG</em>, <em>traV</em>, <em>traR</em>, <em>traD</em>, <em>traI</em>, <em>traX</em> and <em>finO</em>. It contained multiple insertions of IS<em>Ec8</em> and IS<em>Ec8</em>-like elements (Fig. <a reveal-id="DSP015F2" data-open="DSP015F2" class="link link-reveal link-table xref-fig">2</a>A), and comprised about 50% of the complete <em>tra</em> operon seen in R100.<sup><a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSP015C33" data-open="DSP015C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The ORFs (ORF080 and ORF087) on both sides of the multiple insertions were highly homologous to YhfA and YfhA in R100, but no similar ORF was found in the F plasmid (Fig. <a reveal-id="DSP015F2" data-open="DSP015F2" class="link link-reveal link-table xref-fig">2</a>A, B). Moreover, <em>oriT</em>, a 463-nucleotide segment located immediately upstream of <em>traM</em><sup><a reveal-id="DSP015C31" data-open="DSP015C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a></sup> in R100, was highly conserved in pEntH10407, having perfect identity to the nicking-site region of R100 (Fig. <a reveal-id="DSP015F2" data-open="DSP015F2" class="link link-reveal link-table xref-fig">2</a>C).</p>                        <a id="6237583" scrollto-destination="6237583"></a>
<div id="DSP015F2" data-id="DSP015F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/m_dsp01502.jpeg?Expires=2147483647&Signature=DJLavdwHXD-bZkEFxQPSGY1Bp5L4GSDimygFRemJZ6rf8kV3d9PdpeHe525IOUUYXIIaruBGzjhcoryXIFQbj0sZFy1EWvGA5rXw1D8qfqo4CYz15x0dKKhIoBU3KGdj99tH0E5vl2fInL9DnZLxTlNcEINxt54-dnUS434~WQaOvUFK1upod8VyK1E4erpNASJqEBhb9EbtnwMR29YWcoTUfToI8Z1aWeA60QGEkhtleLdO0ruPmswMD5G2tFumKlTM79q3qQGX1KjeGIGjNUI664iALPxt7EWlgqXmCpwDdRKuSueLAD47Hz1g35QKvnay7XAJ6Ap1GB2kUfHgng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship of complete transfer operons of R100 and F plasmid to the incomplete operon of pEntH10407K. (A) The incomplete tra region in pEntH10407K is compared with the transfer operons of R100 and F plasmids. ORFs are orientated according to pEntH10407K. Diagonally hatched arrows indicate genes of the transfer system. tra genes are in uppercase; trb genes are in lower case. ORFs are not to scale. Although transcription proceeds from right to left in R100 and F plasmids, pEntH10407K only contains the portion of the tra region from traM to traR, which are similar to the analogous ORFs in the F plasmid, and another portion from traD to traX, which is similar to the corresponding region from traD to traX of R100. Between the two tra regions, both transposase and putative transposase genes (from ORF080 to ORF087) were recognized. (B) Per cent sequence similarity for each ORF in pEntH10407K to analogous sequences from the R100 and F plasmids. HP indicates a hypothetical protein. Values indicate identity obtained by BLASTX comparison of each pEntH10407K ORF with the corresponding ORF from R100 and F plasmids. For each pair of comparisons, the higher identity value is in the shaded box. Continuous line indicates that a similar ORF was not found. (C) Comparison of putative oriT region of conjugative plasmid aligned at nick site (arrow) according to Frost et al.32 Sequences were aligned; conserved sequence is marked with the shaded box." path-from-xml="dsp01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237583/dsp01502.jpeg" path-from-xml="dsp01502.jpeg" target="_blank">View large</a><a section="6237583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/dsp01502.jpeg?Expires=2147483647&Signature=R3QmszfGG17Fy5qWdDXcQG3XW6PV7WACzl~WAXHl2k93RTeuVjE02yyKzY-0lzHnYmIZxRHNMyC2M-v754X5KwXrqysA2DNwIlIOYUagTqXJF5g0Y7Q22rGu5FJ3UD9zX8xqQTKU2PA-46jvys7yywmjbOUFwvLi1R01GEpLHgZjhPwLcajnK8GGVB6-YKL8sqVUGAghnUiC-Bwb4KFE0gsKHknc3KzDgXzmV5sbHYfVZRn-PeNaTdCWdtUIVRuGYgxVuRzgB5iZYC6qCl7TclygyHgJZvVaS3FhvGqZ1njg3G3IUVYGeYua7BuvPikQm1f6fZlUML2PvmPT902JbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237583&ar=420917&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship of complete transfer operons of R100 and F plasmid to the incomplete operon of pEntH10407K. (A) The incomplete <em>tra</em> region in pEntH10407K is compared with the transfer operons of R100 and F plasmids. ORFs are orientated according to pEntH10407K. Diagonally hatched arrows indicate genes of the transfer system. <em>tra</em> genes are in uppercase; <em>trb</em> genes are in lower case. ORFs are not to scale. Although transcription proceeds from right to left in R100 and F plasmids, pEntH10407K only contains the portion of the <em>tra</em> region from <em>traM</em> to <em>traR</em>, which are similar to the analogous ORFs in the F plasmid, and another portion from <em>traD</em> to <em>traX</em>, which is similar to the corresponding region from <em>traD</em> to <em>traX</em> of R100. Between the two <em>tra</em> regions, both transposase and putative transposase genes (from ORF080 to ORF087) were recognized. (B) Per cent sequence similarity for each ORF in pEntH10407K to analogous sequences from the R100 and F plasmids. HP indicates a hypothetical protein. Values indicate identity obtained by BLASTX comparison of each pEntH10407K ORF with the corresponding ORF from R100 and F plasmids. For each pair of comparisons, the higher identity value is in the shaded box. Continuous line indicates that a similar ORF was not found. (C) Comparison of putative <em>oriT</em> region of conjugative plasmid aligned at nick site (arrow) according to Frost <em>et al</em>.<sup><a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Sequences were aligned; conserved sequence is marked with the shaded box.</p></div></div><div content-id="DSP015F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/m_dsp01502.jpeg?Expires=2147483647&Signature=DJLavdwHXD-bZkEFxQPSGY1Bp5L4GSDimygFRemJZ6rf8kV3d9PdpeHe525IOUUYXIIaruBGzjhcoryXIFQbj0sZFy1EWvGA5rXw1D8qfqo4CYz15x0dKKhIoBU3KGdj99tH0E5vl2fInL9DnZLxTlNcEINxt54-dnUS434~WQaOvUFK1upod8VyK1E4erpNASJqEBhb9EbtnwMR29YWcoTUfToI8Z1aWeA60QGEkhtleLdO0ruPmswMD5G2tFumKlTM79q3qQGX1KjeGIGjNUI664iALPxt7EWlgqXmCpwDdRKuSueLAD47Hz1g35QKvnay7XAJ6Ap1GB2kUfHgng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship of complete transfer operons of R100 and F plasmid to the incomplete operon of pEntH10407K. (A) The incomplete tra region in pEntH10407K is compared with the transfer operons of R100 and F plasmids. ORFs are orientated according to pEntH10407K. Diagonally hatched arrows indicate genes of the transfer system. tra genes are in uppercase; trb genes are in lower case. ORFs are not to scale. Although transcription proceeds from right to left in R100 and F plasmids, pEntH10407K only contains the portion of the tra region from traM to traR, which are similar to the analogous ORFs in the F plasmid, and another portion from traD to traX, which is similar to the corresponding region from traD to traX of R100. Between the two tra regions, both transposase and putative transposase genes (from ORF080 to ORF087) were recognized. (B) Per cent sequence similarity for each ORF in pEntH10407K to analogous sequences from the R100 and F plasmids. HP indicates a hypothetical protein. Values indicate identity obtained by BLASTX comparison of each pEntH10407K ORF with the corresponding ORF from R100 and F plasmids. For each pair of comparisons, the higher identity value is in the shaded box. Continuous line indicates that a similar ORF was not found. (C) Comparison of putative oriT region of conjugative plasmid aligned at nick site (arrow) according to Frost et al.32 Sequences were aligned; conserved sequence is marked with the shaded box." path-from-xml="dsp01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237583/dsp01502.jpeg" path-from-xml="dsp01502.jpeg" target="_blank">View large</a><a section="6237583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/dsp01502.jpeg?Expires=2147483647&Signature=R3QmszfGG17Fy5qWdDXcQG3XW6PV7WACzl~WAXHl2k93RTeuVjE02yyKzY-0lzHnYmIZxRHNMyC2M-v754X5KwXrqysA2DNwIlIOYUagTqXJF5g0Y7Q22rGu5FJ3UD9zX8xqQTKU2PA-46jvys7yywmjbOUFwvLi1R01GEpLHgZjhPwLcajnK8GGVB6-YKL8sqVUGAghnUiC-Bwb4KFE0gsKHknc3KzDgXzmV5sbHYfVZRn-PeNaTdCWdtUIVRuGYgxVuRzgB5iZYC6qCl7TclygyHgJZvVaS3FhvGqZ1njg3G3IUVYGeYua7BuvPikQm1f6fZlUML2PvmPT902JbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237583&ar=420917&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship of complete transfer operons of R100 and F plasmid to the incomplete operon of pEntH10407K. (A) The incomplete <em>tra</em> region in pEntH10407K is compared with the transfer operons of R100 and F plasmids. ORFs are orientated according to pEntH10407K. Diagonally hatched arrows indicate genes of the transfer system. <em>tra</em> genes are in uppercase; <em>trb</em> genes are in lower case. ORFs are not to scale. Although transcription proceeds from right to left in R100 and F plasmids, pEntH10407K only contains the portion of the <em>tra</em> region from <em>traM</em> to <em>traR</em>, which are similar to the analogous ORFs in the F plasmid, and another portion from <em>traD</em> to <em>traX</em>, which is similar to the corresponding region from <em>traD</em> to <em>traX</em> of R100. Between the two <em>tra</em> regions, both transposase and putative transposase genes (from ORF080 to ORF087) were recognized. (B) Per cent sequence similarity for each ORF in pEntH10407K to analogous sequences from the R100 and F plasmids. HP indicates a hypothetical protein. Values indicate identity obtained by BLASTX comparison of each pEntH10407K ORF with the corresponding ORF from R100 and F plasmids. For each pair of comparisons, the higher identity value is in the shaded box. Continuous line indicates that a similar ORF was not found. (C) Comparison of putative <em>oriT</em> region of conjugative plasmid aligned at nick site (arrow) according to Frost <em>et al</em>.<sup><a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Sequences were aligned; conserved sequence is marked with the shaded box.</p></div></div><p>In nearly all locations where the R100 and F plasmid <em>tra</em> genes differ, the pEntH10407K homologues more closely resembled R100. The <em>trbG</em> of pEntH10407K, however, had unique sequence similarity to the corresponding ORF of the F plasmid but not to that of the R100 (Fig. 2B). The other Tra proteins encoded in pEntH10407K were more similar at the amino acid sequence level to R100 proteins than F plasmid ones with the exceptions of FinO (98.4% similar to F plasmid), TrbD (82.0%), TraL (99.0%), TraY (100%), TraJ (97.8%) and TraM (97.6%) (Fig. <a reveal-id="DSP015F2" data-open="DSP015F2" class="link link-reveal link-table xref-fig">2</a>B). Moreover, the 5′ end of ORF080 and the 3′ end of ORF087 were identical at the amino acid level to YhfA and YfhA, which are found in R100 and not in F, suggesting that the backbone of pEntH10407 might be close to that of R100, and that the multiple insertion events into the R100-homologous region might affect the capabilities conferred by this region, such as self-transmissibility.</p><p>We compared the self-transmissibility of pEntH10407K with that of R100 using a derivative of the <em>E. coli</em> K-12 strain. R100 was transferred at a frequency of 2.44 × 10<sup>−6</sup> transconjugants per donor (Table <a reveal-id="DSP015TB2" data-open="DSP015TB2" class="link link-reveal link-table xref-fig">2</a>). In contrast, self-transfer of pEntH10407K occurred at a very low frequency of 2.84 × 10<sup>−9</sup> transconjugants per donor (Table <a reveal-id="DSP015TB2" data-open="DSP015TB2" class="link link-reveal link-table xref-fig">2</a>). This value is as low as the transfer frequency reported in the conjugal plasmid R68 of <em>Pseudomonas aeruginosa</em>.<sup><a reveal-id="DSP015C34" data-open="DSP015C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Parallel experiments using K-12 donor bacteria carrying pUC19 or pBluescript II SK(+) showed that these plasmids are non-self-transmissible (&lt;10<sup>−11</sup> transconjugants per donor for each plasmid) (Table <a reveal-id="DSP015TB2" data-open="DSP015TB2" class="link link-reveal link-table xref-fig">2</a>). These data suggest that pEntH10407 has a low self-transmissibility. Yamamoto and Yokota<sup><a reveal-id="DSP015C12" data-open="DSP015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> reported that pEntH10407 did not have self-transmissibility and that pTRANS induced co-transfer of pEntH10407 and pCFA/I-STIb into the H10407 strain (O78:H11 strain). Non-piliated cells do not transfer DNA, showing that the pilus is absolutely required for conjugal DNA transfer, while mutations in the <em>traN</em> or <em>traG</em> genes drastically reduce transfer efficiency by several orders of magnitude but do not completely abolish it.<sup><a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSP015C35" data-open="DSP015C35" class="link link-ref link-reveal xref-bibr">35</a></sup> TraN and TraG are thought to stabilize mating pairs through an as yet unknown mechanism.<sup><a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSP015C35" data-open="DSP015C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The genes that code for TraN and TraG are deleted in the incomplete <em>tra</em> region of pEntH10407K. As it has been reported that the products of the incomplete <em>tra</em> region take part in the synthesis and assembly of the sex pilus,<sup><a reveal-id="DSP015C32" data-open="DSP015C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSP015C33" data-open="DSP015C33" class="link link-ref link-reveal xref-bibr">33</a></sup> the remaining <em>tra</em> genes in pEntH10407 appear to form a sex pilus which is involved in the transfer of pEntH10407. The formation of an unstable mating pair due to the deficiency of TraN and TraG might be an important cause of the low transfer efficiency in pEntH10407.</p>                        <a id="6237586" scrollto-destination="6237586"></a>
<div content-id="DSP015TB2" class="table-modal"><div class="table-wrap"><div id="DSP015TB2" class="table-wrap-title" data-id="DSP015TB2"><div class="label -label">Table 2</div><div class="caption"><p>Transfer proficiency of pEntH10407K and the mutant plasmids</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Plasmid in donor </th><th>Relevant genotype in <em>tra</em> region </th><th>Transfer frequency </th></tr></thead><tbody><tr><td>pUC19 </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>pBluescript II SK(+) </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>R100 </td><td><em>tra</em><sup>+b</sup> </td><td>2.44 × 10<sup>−6</sup> </td></tr><tr><td>pEntH10407K </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup> </td><td>2.84 × 10<sup>−9</sup> </td></tr><tr><td>pEntH10407KΔ<em>traA</em> </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup>, Δ<em>traA</em> </td><td>&lt;10<sup>−11</sup> </td></tr></tbody></table></div><div content-id="DSP015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Plasmid in donor </th><th>Relevant genotype in <em>tra</em> region </th><th>Transfer frequency </th></tr></thead><tbody><tr><td>pUC19 </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>pBluescript II SK(+) </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>R100 </td><td><em>tra</em><sup>+b</sup> </td><td>2.44 × 10<sup>−6</sup> </td></tr><tr><td>pEntH10407K </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup> </td><td>2.84 × 10<sup>−9</sup> </td></tr><tr><td>pEntH10407KΔ<em>traA</em> </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup>, Δ<em>traA</em> </td><td>&lt;10<sup>−11</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>To evaluate the transmissibility of pEntH10407K, various plasmids [pUC19, pBluescript II SK(+), R100, pEntH10407K and mutated pEntH10407K] were electropolated into a derivative (a spontaneous nalidixic acid-resistant mutant) of the <em>E. coli</em> K-12 strain. The transformants were used in the experiment as donor strains. Aliquots from overnight cultures of the donor and the recipient in Luria–Bertani medium were mixed in 1:1, and incubated at 37°C for 12 h. After mating, the mixtures were diluted and spread (both diluted and undiluted) on selective agar. As controls, aliquots of the donor and recipient cultures were also spread separately on selective plates. Transfer frequencies were calculated per donor bacterium.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>No <em>tra</em> region exists.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>A complete <em>tra</em> region is present.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237586" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP015TB2" class="table-wrap-title" data-id="DSP015TB2"><div class="label -label">Table 2</div><div class="caption"><p>Transfer proficiency of pEntH10407K and the mutant plasmids</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Plasmid in donor </th><th>Relevant genotype in <em>tra</em> region </th><th>Transfer frequency </th></tr></thead><tbody><tr><td>pUC19 </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>pBluescript II SK(+) </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>R100 </td><td><em>tra</em><sup>+b</sup> </td><td>2.44 × 10<sup>−6</sup> </td></tr><tr><td>pEntH10407K </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup> </td><td>2.84 × 10<sup>−9</sup> </td></tr><tr><td>pEntH10407KΔ<em>traA</em> </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup>, Δ<em>traA</em> </td><td>&lt;10<sup>−11</sup> </td></tr></tbody></table></div><div content-id="DSP015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Plasmid in donor </th><th>Relevant genotype in <em>tra</em> region </th><th>Transfer frequency </th></tr></thead><tbody><tr><td>pUC19 </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>pBluescript II SK(+) </td><td>—<sup>a</sup> </td><td>&lt;10<sup>−11</sup> </td></tr><tr><td>R100 </td><td><em>tra</em><sup>+b</sup> </td><td>2.44 × 10<sup>−6</sup> </td></tr><tr><td>pEntH10407K </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup> </td><td>2.84 × 10<sup>−9</sup> </td></tr><tr><td>pEntH10407KΔ<em>traA</em> </td><td><em>traT</em><sup><em>−</em></sup>, <em>traS</em><sup><em>−</em></sup>, <em>traG</em><sup><em>−</em></sup>, <em>traH</em><sup><em>−</em></sup>, <em>trbF</em><sup><em>−</em></sup>, <em>trbJ</em><sup><em>−</em></sup>, <em>trbB</em><sup><em>−</em></sup>, <em>traQ</em><sup><em>−</em></sup>, <em>trbA</em><sup><em>−</em></sup>, <em>traF</em><sup><em>−</em></sup>, <em>trbE</em><sup><em>−</em></sup>, <em>traN</em><sup><em>−</em></sup>, <em>trbC</em><sup><em>−</em></sup>, <em>traU</em><sup><em>−</em></sup>, <em>traW</em><sup><em>−</em></sup>, <em>trbI</em><sup><em>−</em></sup>, <em>traC</em><sup><em>−</em></sup>, Δ<em>traA</em> </td><td>&lt;10<sup>−11</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>To evaluate the transmissibility of pEntH10407K, various plasmids [pUC19, pBluescript II SK(+), R100, pEntH10407K and mutated pEntH10407K] were electropolated into a derivative (a spontaneous nalidixic acid-resistant mutant) of the <em>E. coli</em> K-12 strain. The transformants were used in the experiment as donor strains. Aliquots from overnight cultures of the donor and the recipient in Luria–Bertani medium were mixed in 1:1, and incubated at 37°C for 12 h. After mating, the mixtures were diluted and spread (both diluted and undiluted) on selective agar. As controls, aliquots of the donor and recipient cultures were also spread separately on selective plates. Transfer frequencies were calculated per donor bacterium.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>No <em>tra</em> region exists.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>A complete <em>tra</em> region is present.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237586" target="_blank">View Large</a></div></div><p>About 20 genes encoded within the <em>tra</em> operon in R100 are thought to be necessary for conjugation to occur, since mutations in these genes abrogate a plasmid transfer. Among these genes, it is known that <em>traA</em> gene encodes pilus subunit, pilin, which is an essential factor for plasmid self-transmissibility in F plasmid and R100. Then, we constructed a mutant plasmid (pEntH10407KΔ<em>traA</em>) lacking <em>traA</em> of pEntH10407K by a homologous recombination method and examined the self-transmissibility of the pEntH10407KΔ<em>traA</em>. In the mating assay, the pEntH10407KΔ<em>traA</em> did not exhibit self-transmissibility (&lt;10<sup>−11</sup> transconjutants per donor) (Table <a reveal-id="DSP015TB2" data-open="DSP015TB2" class="link link-reveal link-table xref-fig">2</a>). The results suggest that the <em>traA</em> gene is required for the self-transmissibility of pEntH10407K.</p><p>Moreover, McConnell <em>et al</em>.<sup><a reveal-id="DSP015C6" data-open="DSP015C6" class="link link-ref link-reveal xref-bibr">6</a></sup> reported that an Ent plasmid (pEntO78:H12) from the O78:H12 strain in Bangladesh might contain both enterotoxin genes and be auto-transmissible, but that pEntH10407 (from the O78:H11 strain) might not be auto-transmissible. However, these ETEC strains were isolated also in Bangladesh and in the same period in the 1970s. In addition, the difference of the MM of pEntO78:H12 and pEntH10407 is similar to the MM of the region from <em>traC</em> to <em>traT</em> in R100. These suggest that the pLT-ST may originally have a complete set of <em>tra</em> genes matching those of the R100 or F plasmid, and that pEntH10407 might have been formed by multiple insertions into the <em>tra</em> genes to produce the sequence determined in this paper.</p><p>The dot matrix analysis was performed to elucidate sequence similarity between pEntH10407K and R100. This analysis indicated that the pEntH10407K shared the nucleotide sequence similarities in regions involved in plasmid replication/maintenance, and plasmid transfer (<em>tra</em> region) (Fig. <a reveal-id="DSP015F3" data-open="DSP015F3" class="link link-reveal link-table xref-fig">3</a>). The interruption of the similarity within the incomplete <em>tra</em> region of pEntH10407K corresponded to the swapping of <em>tra</em> genes with IS<em>Ec8</em> and IS<em>Ec8</em>-like elements. There was no sequence similarity in the toxin region except for IS<em>1</em> sequences, which clearly indicates the presence of numerous unique ORFs including <em>eltAB</em> and <em>estIa</em> in the toxin region.</p>                        <a id="6237590" scrollto-destination="6237590"></a>
<div id="DSP015F3" data-id="DSP015F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/m_dsp01503.jpeg?Expires=2147483647&Signature=s2PZ6Qcy9MNLBXWi3qS8xh9AWC4Qemhf9T0Oy2Q-3NNFyAy9VC1KDAGVZy7~L4OcrQQuqmuNLron5r~zOfTzLuRdpcuqoCZoqMCMCWGD9Zxj1mUdR2axPJwRjdGteEylr5ScYkK0s3BhEA0Hr33wWrk3foJ1G1rJvoo41ZbyV5DgYTLQNlxsQ3hCgEfLkn2bT1FxaMhI~XcdMHPdT~7xEMpUSRxGif2gbAwF7QLslZU3G638Pm6Kc-Pfa6tKV6rZEV6laKwaxlKaJYtjFC26JcQPY~knKzFTAHH9jaV8tunLUug9nk~AUYemBsX9bd-e149zs5gN2oPvJ3HCEiSwvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plot analysis for pEntH10407K versus R100. Dot matrix analysis was performed using the Harrplot 2.0 software (Software Development) with a windows setting at 15 and threshold at 10. Solid diagonal line represents similarity. The toxin region (pathogenicity islet) and the incomplete tra region of pEntH10407K are indicated by horizontally and diagonally hatched box, respectively, at the top of the dot plot. The region of the ISEc8 and ISEc8-like elements in the incomplete tra region is indicated by open box. The tra region of R100 is indicated by closed box at the right side of the dot plot." path-from-xml="dsp01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237590/dsp01503.jpeg" path-from-xml="dsp01503.jpeg" target="_blank">View large</a><a section="6237590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/dsp01503.jpeg?Expires=2147483647&Signature=jcPYCo3ljdlP9nvTBi9hnTClOmiYyfb782np~DApMT2-EacONtKXzuLTiLrrRY8jXUxvEdog35sjuQQNnIkd1BKnhGxS7~bCC1s5w7yNR1NDDRyMO2abP~m~TOx6k-SYxyQQZszqMHbRuCS~4OhAOFXt3Oaqtq8dmnUSs2342E8kOoA43nXI3yXQGjxRD8EM9wKha32pOSBt7Yh8qYDFiaYDGgnD6F4OfUpivCrRaMXKMsWWLOnptBQYOsZiKbUB2GNGHdNa3SwDxfLWEEPoJbnd7bDmsfzGvVP9YyD78z5u5KclUAzZ~5k2yaRUpGqmclyU49IhLNYt7nvFwX4sxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237590&ar=420917&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plot analysis for pEntH10407K versus R100. Dot matrix analysis was performed using the Harrplot 2.0 software (Software Development) with a windows setting at 15 and threshold at 10. Solid diagonal line represents similarity. The toxin region (pathogenicity islet) and the incomplete <em>tra</em> region of pEntH10407K are indicated by horizontally and diagonally hatched box, respectively, at the top of the dot plot. The region of the IS<em>Ec8</em> and IS<em>Ec8</em>-like elements in the incomplete <em>tra</em> region is indicated by open box. The <em>tra</em> region of R100 is indicated by closed box at the right side of the dot plot.</p></div></div><div content-id="DSP015F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/m_dsp01503.jpeg?Expires=2147483647&Signature=s2PZ6Qcy9MNLBXWi3qS8xh9AWC4Qemhf9T0Oy2Q-3NNFyAy9VC1KDAGVZy7~L4OcrQQuqmuNLron5r~zOfTzLuRdpcuqoCZoqMCMCWGD9Zxj1mUdR2axPJwRjdGteEylr5ScYkK0s3BhEA0Hr33wWrk3foJ1G1rJvoo41ZbyV5DgYTLQNlxsQ3hCgEfLkn2bT1FxaMhI~XcdMHPdT~7xEMpUSRxGif2gbAwF7QLslZU3G638Pm6Kc-Pfa6tKV6rZEV6laKwaxlKaJYtjFC26JcQPY~knKzFTAHH9jaV8tunLUug9nk~AUYemBsX9bd-e149zs5gN2oPvJ3HCEiSwvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plot analysis for pEntH10407K versus R100. Dot matrix analysis was performed using the Harrplot 2.0 software (Software Development) with a windows setting at 15 and threshold at 10. Solid diagonal line represents similarity. The toxin region (pathogenicity islet) and the incomplete tra region of pEntH10407K are indicated by horizontally and diagonally hatched box, respectively, at the top of the dot plot. The region of the ISEc8 and ISEc8-like elements in the incomplete tra region is indicated by open box. The tra region of R100 is indicated by closed box at the right side of the dot plot." path-from-xml="dsp01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237590/dsp01503.jpeg" path-from-xml="dsp01503.jpeg" target="_blank">View large</a><a section="6237590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/5/10.1093/dnares/dsp015/2/dsp01503.jpeg?Expires=2147483647&Signature=jcPYCo3ljdlP9nvTBi9hnTClOmiYyfb782np~DApMT2-EacONtKXzuLTiLrrRY8jXUxvEdog35sjuQQNnIkd1BKnhGxS7~bCC1s5w7yNR1NDDRyMO2abP~m~TOx6k-SYxyQQZszqMHbRuCS~4OhAOFXt3Oaqtq8dmnUSs2342E8kOoA43nXI3yXQGjxRD8EM9wKha32pOSBt7Yh8qYDFiaYDGgnD6F4OfUpivCrRaMXKMsWWLOnptBQYOsZiKbUB2GNGHdNa3SwDxfLWEEPoJbnd7bDmsfzGvVP9YyD78z5u5KclUAzZ~5k2yaRUpGqmclyU49IhLNYt7nvFwX4sxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237590&ar=420917&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plot analysis for pEntH10407K versus R100. Dot matrix analysis was performed using the Harrplot 2.0 software (Software Development) with a windows setting at 15 and threshold at 10. Solid diagonal line represents similarity. The toxin region (pathogenicity islet) and the incomplete <em>tra</em> region of pEntH10407K are indicated by horizontally and diagonally hatched box, respectively, at the top of the dot plot. The region of the IS<em>Ec8</em> and IS<em>Ec8</em>-like elements in the incomplete <em>tra</em> region is indicated by open box. The <em>tra</em> region of R100 is indicated by closed box at the right side of the dot plot.</p></div></div><p>In summary, we report the complete 67 094 bp sequence of pEntH10407K, an Ent plasmid from ETEC H10407. pEntH10407K contains three distinct major regions: (i) a pathogenicity islet containing enterotoxin genes, (ii) a region involved in plasmid replication and maintenance, and (iii) a region including <em>tra</em> genes that cause the self-transmissibility. Our analysis of the pEntH10407 sequence emphasizes its mosaic nature and has implications for its evolution. Self-transmissibility mediated by the incomplete <em>tra</em> region is retained in pEntH10407, despite the deletion of many <em>tra</em> genes. This finding raises the possibility that the enterotoxigenic H10407 strain might have developed due to the helper plasmid assisting in the transfer of the pEntH10407. However, our finding raises another possibility that the H10407 strain might have been born by self-transfer that occurred totally through the self-transmissibility of pEntH10 407. A comprehensive comparative analysis to clarify the evolution and diversity of Ent plasmids is currently in progress in our laboratory.</p>                            <h2 scrollto-destination=6237592 id="6237592" class="section-title" data-legacy-id=s1>Funding</h2>
<p>This work was supported by Grants-in-Aid for Scientific Research in the Priority Area of ‘Applied Genomics’ from the Ministry of Education, Culture, Sports, Science and Technology and the Japan Society for the Promotion of Science (JSPS) and for the Open Research Center's Project of Fujita Health University from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=6237594 id="6237594" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr H. Hirakawa, Department of Genetic Resources Technology, Faculty of Agriculture, Kyushu University, for bioinformatic analysis of pEntH10407K sequence and data handling.</p>                            <h2 scrollto-destination=6237596 id="6237596" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP015C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Evans</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Silver</div> <div class="given-names">R.P.</div></div>,  <div class="name"><div class="surname">Evans</div> <div class="given-names">D.J.</div>Jr</div>,  <div class="name"><div class="surname">Chase</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Gorbach</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Plasmid-controlled colonization factor associated with virulence in <em>Escherichia coli</em> enterotoxigenic for humans</div>, <div class="source">Infect. Immun.</div> , <div class="year">1975</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmid-controlled%20colonization%20factor%20associated%20with%20virulence%20in%20Escherichia%20coli%20enterotoxigenic%20for%20humans&amp;author=D.G.%20Evans&amp;author=R.P.%20Silver&amp;author=D.J.%20Evans&amp;author=D.G.%20Chase&amp;author=S.L.%20Gorbach&amp;publication_year=1975&amp;journal=Infect.%20Immun.&amp;volume=12&amp;pages=656-67" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1100526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qadri</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Svennerholm</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Faruque</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Sack</div> <div class="given-names">R.B.</div></div>. </div><div class="article-title">Enterotoxigenic <em>Escherichia coli</em> in developing countries: epidemiology, microbiology, clinical features, treatment, and prevention</div>, <div class="source">Clin. Microbiol. Rev.</div> , <div class="year">2005</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">465</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enterotoxigenic%20Escherichia%20coli%20in%20developing%20countries%3A%20epidemiology%2C%20microbiology%2C%20clinical%20features%2C%20treatment%2C%20and%20prevention&amp;author=F.%20Qadri&amp;author=A.M.%20Svennerholm&amp;author=A.S.%20Faruque&amp;author=R.B.%20Sack&amp;publication_year=2005&amp;journal=Clin.%20Microbiol.%20Rev.&amp;volume=18&amp;pages=465-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/CMR.18.3.465-483.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FCMR.18.3.465-483.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FCMR.18.3.465-483.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16020685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sack</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Gorbach</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Banwell</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Jacobs</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Chatterjee</div> <div class="given-names">B.D.</div></div>,  <div class="name"><div class="surname">Mitra</div> <div class="given-names">R.C.</div></div>. </div><div class="article-title">Enterotoxigenic <em>Escherichia coli</em> isolated from patients with severe cholera-like disease</div>, <div class="source">J. Infect. Dis.</div> , <div class="year">1971</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">378</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enterotoxigenic%20Escherichia%20coli%20isolated%20from%20patients%20with%20severe%20cholera-like%20disease&amp;author=R.B.%20Sack&amp;author=S.L.%20Gorbach&amp;author=J.G.%20Banwell&amp;author=B.%20Jacobs&amp;author=B.D.%20Chatterjee&amp;author=R.C.%20Mitra&amp;publication_year=1971&amp;journal=J.%20Infect.%20Dis.&amp;volume=123&amp;pages=378-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/infdis/123.4.378" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Finfdis%2F123.4.378" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Finfdis%2F123.4.378"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4938945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holmgren</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Actions of cholera toxin and the prevention and treatment of cholera</div>, <div class="source">Nature</div> , <div class="year">1981</div>, vol. <div class="volume">292</div> (pg. <div class="fpage">413</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Actions%20of%20cholera%20toxin%20and%20the%20prevention%20and%20treatment%20of%20cholera&amp;author=J.%20Holmgren&amp;publication_year=1981&amp;journal=Nature&amp;volume=292&amp;pages=413-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/292413a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F292413a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F292413a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7019725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsuji</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Taga</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Honda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takeda</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Miwatani</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Molecular heterogeneity of heat-labile enterotoxins from human and porcine enterotoxigenic <em>Escherichia coli</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">1982</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">444</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20heterogeneity%20of%20heat-labile%20enterotoxins%20from%20human%20and%20porcine%20enterotoxigenic%20Escherichia%20coli&amp;author=T.%20Tsuji&amp;author=S.%20Taga&amp;author=T.%20Honda&amp;author=Y.%20Takeda&amp;author=T.%20Miwatani&amp;publication_year=1982&amp;journal=Infect.%20Immun.&amp;volume=38&amp;pages=444-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7141704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McConnell</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Willshaw</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Scotland</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Rowe</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Plasmids coding for heat-labile enterotoxin production isolated from <em>Escherichia coli</em> O78: comparison of properties</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1980</div>, vol. <div class="volume">143</div> (pg. <div class="fpage">158</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmids%20coding%20for%20heat-labile%20enterotoxin%20production%20isolated%20from%20Escherichia%20coli%20O78%3A%20comparison%20of%20properties&amp;author=M.M.%20McConnell&amp;author=H.R.%20Smith&amp;author=G.A.%20Willshaw&amp;author=S.M.%20Scotland&amp;author=B.%20Rowe&amp;publication_year=1980&amp;journal=J.%20Bacteriol.&amp;volume=143&amp;pages=158-67" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6995426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moseley</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Samadpour-Motalebi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Falkow</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Plasmid association and nucleotide sequence relationships of two genes encoding heat-stable enterotoxin production in <em>Escherichia coli</em> H-10407</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1983</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">441</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmid%20association%20and%20nucleotide%20sequence%20relationships%20of%20two%20genes%20encoding%20heat-stable%20enterotoxin%20production%20in%20Escherichia%20coli%20H-10407&amp;author=S.L.%20Moseley&amp;author=M.%20Samadpour-Motalebi&amp;author=S.%20Falkow&amp;publication_year=1983&amp;journal=J.%20Bacteriol.&amp;volume=156&amp;pages=441-3" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6352687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">H.W.</div></div>,  <div class="name"><div class="surname">Gyles</div> <div class="given-names">C.L.</div></div>. </div><div class="article-title">The relationship between two apparently different enterotoxins produced by enteropathogenic strains of <em>Escherichia coli</em> of porcine origin</div>, <div class="source">J. Med. Microbiol.</div> , <div class="year">1970</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">387</div>-<div class="lpage">401</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relationship%20between%20two%20apparently%20different%20enterotoxins%20produced%20by%20enteropathogenic%20strains%20of%20Escherichia%20coli%20of%20porcine%20origin&amp;author=H.W.%20Smith&amp;author=C.L.%20Gyles&amp;publication_year=1970&amp;journal=J.%20Med.%20Microbiol.&amp;volume=3&amp;pages=387-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00222615-3-3-387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00222615-3-3-387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00222615-3-3-387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4919579" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gaastra</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Svennerholm</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Colonization factors of human enterotoxigenic <em>Escherichia coli</em> (ETEC)</div>, <div class="source">Trends Microbiol.</div> , <div class="year">1996</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">444</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Colonization%20factors%20of%20human%20enterotoxigenic%20Escherichia%20coli%20(ETEC)&amp;author=W.%20Gaastra&amp;author=A.M.%20Svennerholm&amp;publication_year=1996&amp;journal=Trends%20Microbiol.&amp;volume=4&amp;pages=444-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0966-842X(96)10068-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0966-842X(96)10068-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0966-842X(96)10068-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8950814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Willshaw</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Rowe</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Mapping of a plasmid, coding for colonization, factor antigen I and heat-stable enterotoxin production, isolated from an enterotoxigenic strain of <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1982</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">264</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20a%20plasmid%2C%20coding%20for%20colonization%2C%20factor%20antigen%20I%20and%20heat-stable%20enterotoxin%20production%2C%20isolated%20from%20an%20enterotoxigenic%20strain%20of%20Escherichia%20coli&amp;author=H.R.%20Smith&amp;author=G.A.%20Willshaw&amp;author=B.%20Rowe&amp;publication_year=1982&amp;journal=J.%20Bacteriol.&amp;volume=149&amp;pages=264-75" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6274845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Willshaw</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Barclay</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">McConnell</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Rowe</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Molecular comparison of plasmids encoding heat-labile enterotoxin isolated from <em>Escherichia coli</em> strains of human origin</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1980</div>, vol. <div class="volume">143</div> (pg. <div class="fpage">168</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20comparison%20of%20plasmids%20encoding%20heat-labile%20enterotoxin%20isolated%20from%20Escherichia%20coli%20strains%20of%20human%20origin&amp;author=G.A.%20Willshaw&amp;author=E.A.%20Barclay&amp;author=H.R.%20Smith&amp;author=M.M.%20McConnell&amp;author=B.%20Rowe&amp;publication_year=1980&amp;journal=J.%20Bacteriol.&amp;volume=143&amp;pages=168-75" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6249787" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yokota</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Plasmids of enterotoxigenic <em>Escherichia coli</em> H10407: evidence for two heat-stable enterotoxin genes and a conjugal transfer system</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1983</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">1352</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmids%20of%20enterotoxigenic%20Escherichia%20coli%20H10407%3A%20evidence%20for%20two%20heat-stable%20enterotoxin%20genes%20and%20a%20conjugal%20transfer%20system&amp;author=T.%20Yamamoto&amp;author=T.%20Yokota&amp;publication_year=1983&amp;journal=J.%20Bacteriol.&amp;volume=153&amp;pages=1352-60" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6298182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Froehlich</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sanders</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Quail</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">The pCoo plasmid of enterotoxigenic <em>Escherichia coli</em> is a mosaic cointegrate</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">6509</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pCoo%20plasmid%20of%20enterotoxigenic%20Escherichia%20coli%20is%20a%20mosaic%20cointegrate&amp;author=B.%20Froehlich&amp;author=J.%20Parkhill&amp;author=M.%20Sanders&amp;author=M.A.%20Quail&amp;author=J.R.%20Scott&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=6509-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.18.6509-6516.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.18.6509-6516.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.18.6509-6516.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16159784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wachsmuth</div> <div class="given-names">I.K.</div></div>,  <div class="name"><div class="surname">Falkow</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ryder</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Plasmid-mediated properties of a heat-stable enterotoxin-producing <em>Escherichia coli</em> associated with infantile diarrhea</div>, <div class="source">Infect. Immun.</div> , <div class="year">1976</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">403</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmid-mediated%20properties%20of%20a%20heat-stable%20enterotoxin-producing%20Escherichia%20coli%20associated%20with%20infantile%20diarrhea&amp;author=I.K.%20Wachsmuth&amp;author=S.%20Falkow&amp;author=R.W.%20Ryder&amp;publication_year=1976&amp;journal=Infect.%20Immun.&amp;volume=14&amp;pages=403-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/786884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Froehlich</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Holtzapple</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Read</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Horizontal transfer of CS1 pilin genes of enterotoxigenic <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">3230</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20transfer%20of%20CS1%20pilin%20genes%20of%20enterotoxigenic%20Escherichia%20coli&amp;author=B.%20Froehlich&amp;author=E.%20Holtzapple&amp;author=T.D.%20Read&amp;author=J.R.%20Scott&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=3230-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.10.3230-3237.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.10.3230-3237.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.10.3230-3237.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15126486" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goryshin</div> <div class="given-names">I.Y.</div></div>,  <div class="name"><div class="surname">Reznikoff</div> <div class="given-names">W.S.</div></div>. </div><div class="article-title">Tn5 in vitro transposition</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1998</div>, vol. <div class="volume">273</div> (pg. <div class="fpage">7367</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tn5%20in%20vitro%20transposition&amp;author=I.Y.%20Goryshin&amp;author=W.S.%20Reznikoff&amp;publication_year=1998&amp;journal=J.%20Biol.%20Chem.&amp;volume=273&amp;pages=7367-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.273.13.7367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.273.13.7367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.273.13.7367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9516433" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yokota</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Primary structure of heat-labile enterotoxin produced by <em>Escherichia coli</em> pathogenic for humans</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1984</div>, vol. <div class="volume">259</div> (pg. <div class="fpage">5037</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primary%20structure%20of%20heat-labile%20enterotoxin%20produced%20by%20Escherichia%20coli%20pathogenic%20for%20humans&amp;author=T.%20Yamamoto&amp;author=T.%20Tamura&amp;author=T.%20Yokota&amp;publication_year=1984&amp;journal=J.%20Biol.%20Chem.&amp;volume=259&amp;pages=5037-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6325417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yokota</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Evolutionary origin of pathogenic determinants in enterotoxigenic <em>Escherichia coli</em> and <em>Vibrio cholerae</em> O1</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1987</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">1352</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20origin%20of%20pathogenic%20determinants%20in%20enterotoxigenic%20Escherichia%20coli%20and%20Vibrio%20cholerae%20O1&amp;author=T.%20Yamamoto&amp;author=T.%20Gojobori&amp;author=T.%20Yokota&amp;publication_year=1987&amp;journal=J.%20Bacteriol.&amp;volume=169&amp;pages=1352-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3546273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yokota</div> <div class="given-names">T.</div></div>. </div><div class="article-title"><em>Escherichia coli</em> heat-labile enterotoxin genes are flanked by repeated deoxyribonucleic acid sequences</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1981</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">850</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20heat-labile%20enterotoxin%20genes%20are%20flanked%20by%20repeated%20deoxyribonucleic%20acid%20sequences&amp;author=T.%20Yamamoto&amp;author=T.%20Yokota&amp;publication_year=1981&amp;journal=J.%20Bacteriol.&amp;volume=145&amp;pages=850-60" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6257652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gal-Mor</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Finlay</div> <div class="given-names">B.B.</div></div>. </div><div class="article-title">Pathogenicity islands: a molecular toolbox for bacterial virulence</div>, <div class="source">Cell Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1707</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pathogenicity%20islands%3A%20a%20molecular%20toolbox%20for%20bacterial%20virulence&amp;author=O.%20Gal-Mor&amp;author=B.B.%20Finlay&amp;publication_year=2006&amp;journal=Cell%20Microbiol.&amp;volume=8&amp;pages=1707-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cmi.2006.8.issue-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcmi.2006.8.issue-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcmi.2006.8.issue-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16939533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Min</div> <div class="given-names">Y.N.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Womble</div> <div class="given-names">D.D.</div></div>,  <div class="name"><div class="surname">Rownd</div> <div class="given-names">R.H.</div></div>. </div><div class="article-title">Insertion and deletion mutations in the repA4 region of the IncFII plasmid NR1 cause unstable inheritance</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1993</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">5350</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertion%20and%20deletion%20mutations%20in%20the%20repA4%20region%20of%20the%20IncFII%20plasmid%20NR1%20cause%20unstable%20inheritance&amp;author=T.%20Jiang&amp;author=Y.N.%20Min&amp;author=W.%20Liu&amp;author=D.D.%20Womble&amp;author=R.H.%20Rownd&amp;publication_year=1993&amp;journal=J.%20Bacteriol.&amp;volume=175&amp;pages=5350-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8396115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ryder</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Inokuchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Genes and sites involved in replication and incompatibility of an R100 plasmid derivative based on nucleotide sequence analysis</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1980</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">527</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20and%20sites%20involved%20in%20replication%20and%20incompatibility%20of%20an%20R100%20plasmid%20derivative%20based%20on%20nucleotide%20sequence%20analysis&amp;author=J.%20Rosen&amp;author=T.%20Ryder&amp;author=H.%20Inokuchi&amp;author=H.%20Ohtsubo&amp;author=E.%20Ohtsubo&amp;publication_year=1980&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=179&amp;pages=527-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00271742" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00271742" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00271742"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7003300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">del Solar</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Giraldo</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ruiz-Echevarria</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Espinosa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Diaz-Orejas</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Replication and control of circular bacterial plasmids</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">1998</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">434</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication%20and%20control%20of%20circular%20bacterial%20plasmids&amp;author=G.%20del%20Solar&amp;author=R.%20Giraldo&amp;author=M.J.%20Ruiz-Echevarria&amp;author=M.%20Espinosa&amp;author=R.%20Diaz-Orejas&amp;publication_year=1998&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=62&amp;pages=434-64" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9618448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gerdes</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Bech</div> <div class="given-names">F.W.</div></div>,  <div class="name"><div class="surname">Jorgensen</div> <div class="given-names">S.T.</div></div>, et al. </div><div class="article-title">Mechanism of postsegregational killing by the hok gene product of the parB system of plasmid R1 and its homology with the relF gene product of the <em>E. coli</em> relB operon</div>, <div class="source">EMBO J.</div> , <div class="year">1986</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">2023</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20postsegregational%20killing%20by%20the%20hok%20gene%20product%20of%20the%20parB%20system%20of%20plasmid%20R1%20and%20its%20homology%20with%20the%20relF%20gene%20product%20of%20the%20E.%20coli%20relB%20operon&amp;author=K.%20Gerdes&amp;author=F.W.%20Bech&amp;author=S.T.%20Jorgensen&amp;publication_year=1986&amp;journal=EMBO%20J.&amp;volume=5&amp;pages=2023-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3019679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thisted</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sorensen</div> <div class="given-names">N.S.</div></div>,  <div class="name"><div class="surname">Wagner</div> <div class="given-names">E.G.</div></div>,  <div class="name"><div class="surname">Gerdes</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Mechanism of post-segregational killing: Sok antisense RNA interacts with Hok mRNA via its 5′-end single-stranded leader and competes with the 3′-end of Hok mRNA for binding to the mok translational initiation region</div>, <div class="source">EMBO J.</div> , <div class="year">1994</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1960</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20post-segregational%20killing%3A%20Sok%20antisense%20RNA%20interacts%20with%20Hok%20mRNA%20via%20its%205%E2%80%B2-end%20single-stranded%20leader%20and%20competes%20with%20the%203%E2%80%B2-end%20of%20Hok%20mRNA%20for%20binding%20to%20the%20mok%20translational%20initiation%20region&amp;author=T.%20Thisted&amp;author=N.S.%20Sorensen&amp;author=E.G.%20Wagner&amp;author=K.%20Gerdes&amp;publication_year=1994&amp;journal=EMBO%20J.&amp;volume=13&amp;pages=1960-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8168493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jensen</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Gerdes</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Mechanism of DNA segregation in prokaryotes: ParM partitioning protein of plasmid R1 co-localizes with its replicon during the cell cycle</div>, <div class="source">EMBO J.</div> , <div class="year">1999</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">4076</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20DNA%20segregation%20in%20prokaryotes%3A%20ParM%20partitioning%20protein%20of%20plasmid%20R1%20co-localizes%20with%20its%20replicon%20during%20the%20cell%20cycle&amp;author=R.B.%20Jensen&amp;author=K.%20Gerdes&amp;publication_year=1999&amp;journal=EMBO%20J.&amp;volume=18&amp;pages=4076-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/18.14.4076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F18.14.4076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F18.14.4076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10406812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jensen</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Lurz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gerdes</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Mechanism of DNA segregation in prokaryotes: replicon pairing by parC of plasmid R1</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">8550</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20DNA%20segregation%20in%20prokaryotes%3A%20replicon%20pairing%20by%20parC%20of%20plasmid%20R1&amp;author=R.B.%20Jensen&amp;author=R.%20Lurz&amp;author=K.%20Gerdes&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=8550-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.15.8550" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.15.8550" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.15.8550"> </span></div> </div></div></div></div></div><div content-id="DSP015C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">The complete sequence and analysis of the large virulence plasmid pSS of <em>Shigella sonnei</em></div>, <div class="source">Plasmid</div> , <div class="year">2005</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">149</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20sequence%20and%20analysis%20of%20the%20large%20virulence%20plasmid%20pSS%20of%20Shigella%20sonnei&amp;author=Y.%20Jiang&amp;author=F.%20Yang&amp;author=X.%20Zhang&amp;publication_year=2005&amp;journal=Plasmid&amp;volume=54&amp;pages=149-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plasmid.2005.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plasmid.2005.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plasmid.2005.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16122562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayes</div> <div class="given-names">F.</div></div>. </div><div class="article-title">A family of stability determinants in pathogenic bacteria</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">6415</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20family%20of%20stability%20determinants%20in%20pathogenic%20bacteria&amp;author=F.%20Hayes&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=6415-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9829958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anthony</div> <div class="given-names">K.G.</div></div>,  <div class="name"><div class="surname">Klimke</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Manchak</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Frost</div> <div class="given-names">L.S.</div></div>. </div><div class="article-title">Comparison of proteins involved in pilus synthesis and mating pair stabilization from the related plasmids F and R100-1: insights into the mechanism of conjugation</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1999</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">5149</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20proteins%20involved%20in%20pilus%20synthesis%20and%20mating%20pair%20stabilization%20from%20the%20related%20plasmids%20F%20and%20R100-1%3A%20insights%20into%20the%20mechanism%20of%20conjugation&amp;author=K.G.%20Anthony&amp;author=W.A.%20Klimke&amp;author=J.%20Manchak&amp;author=L.S.%20Frost&amp;publication_year=1999&amp;journal=J.%20Bacteriol.&amp;volume=181&amp;pages=5149-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10464182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brinkley</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Burland</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Nucleotide sequence analysis of the enteropathogenic <em>Escherichia coli</em> adherence factor plasmid pMAR7</div>, <div class="source">Infect. Immun.</div> , <div class="year">2006</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">5408</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20sequence%20analysis%20of%20the%20enteropathogenic%20Escherichia%20coli%20adherence%20factor%20plasmid%20pMAR7&amp;author=C.%20Brinkley&amp;author=V.%20Burland&amp;author=R.%20Keller&amp;publication_year=2006&amp;journal=Infect.%20Immun.&amp;volume=74&amp;pages=5408-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.01840-05" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.01840-05" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.01840-05"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16926437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frost</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Ippen-Ihler</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Skurray</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Analysis of the sequence and gene products of the transfer region of the F sex factor</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1994</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">162</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20sequence%20and%20gene%20products%20of%20the%20transfer%20region%20of%20the%20F%20sex%20factor&amp;author=L.S.%20Frost&amp;author=K.%20Ippen-Ihler&amp;author=R.A.%20Skurray&amp;publication_year=1994&amp;journal=Microbiol.%20Rev.&amp;volume=58&amp;pages=162-210" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7915817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawley</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Klimke</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Gubbins</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Frost</div> <div class="given-names">L.S.</div></div>. </div><div class="article-title">F factor conjugation is a true type IV secretion system</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2003</div>, vol. <div class="volume">224</div> (pg. <div class="fpage">1</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=F%20factor%20conjugation%20is%20a%20true%20type%20IV%20secretion%20system&amp;author=T.D.%20Lawley&amp;author=W.A.%20Klimke&amp;author=M.J.%20Gubbins&amp;author=L.S.%20Frost&amp;publication_year=2003&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=224&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1097(03)00430-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1097(03)00430-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1097(03)00430-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12855161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Genthner</div> <div class="given-names">F.J.</div></div>,  <div class="name"><div class="surname">Chatterjee</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Barkay</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bourquin</div> <div class="given-names">A.W.</div></div>. </div><div class="article-title">Capacity of aquatic bacteria to act as recipients of plasmid DNA</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1988</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">115</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Capacity%20of%20aquatic%20bacteria%20to%20act%20as%20recipients%20of%20plasmid%20DNA&amp;author=F.J.%20Genthner&amp;author=P.%20Chatterjee&amp;author=T.%20Barkay&amp;author=A.W.%20Bourquin&amp;publication_year=1988&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=54&amp;pages=115-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3345074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP015C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP015C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klimke</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Frost</div> <div class="given-names">L.S.</div></div>. </div><div class="article-title">Genetic analysis of the role of the transfer gene, traN, of the F and R100-1 plasmids in mating pair stabilization during conjugation</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">4036</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20the%20role%20of%20the%20transfer%20gene%2C%20traN%2C%20of%20the%20F%20and%20R100-1%20plasmids%20in%20mating%20pair%20stabilization%20during%20conjugation&amp;author=W.A.%20Klimke&amp;author=L.S.%20Frost&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=4036-43" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9696748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Short Communication">Short Communication</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=420917&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">647</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="420917"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000271610400005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f5%2f299%2f420917">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000271610400005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F5%2F299%2F420917" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/420917'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Nucleotide sequence analysis of the enterotoxigenic Escherichia coli Ent plasmid" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f5%2f299%2f420917">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

