

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1669,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/4/409/5003450","@type":"ScholarlyArticle","name":"Draft genome of a high value tropical timber tree, Teak ( Tectona grandis  L. f): insights into SSR diversity, phylogeny and conservation","datePublished":"2018-05-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2018-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy013","keywords":["teak","genome sequencing","SSRs","phylogeny","divergence"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/4/409/5003450","author":[{"name":"Yasodha, Ramasamy","affiliation":"Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India","@type":"Person"},{"name":"Vasudeva, Ramesh","affiliation":"Forest Genetics and Biotechnology Division, Kerala Forest Research Institute, Peechi, Thrissur, Kerala, India","@type":"Person"},{"name":"Balakrishnan, Swathi","affiliation":"Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India","@type":"Person"},{"name":"Sakthi, Ambothi Rathnasamy","affiliation":"Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India","@type":"Person"},{"name":"Abel, Nicodemus","affiliation":"Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India","@type":"Person"},{"name":"Binai, Nagarajan","affiliation":"Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India","@type":"Person"},{"name":"Rajashekar, Balaji","affiliation":"Genotypic Technology Private Limited, Bengaluru, Karnataka, India  Institute of Computer Science, University of Tartu, Estonia","@type":"Person"},{"name":"Bachpai, Vijay Kumar Waman","affiliation":"Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India","@type":"Person"},{"name":"Pillai, Chandrasekhara","affiliation":"Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India","@type":"Person"},{"name":"Dev, Suma Arun","affiliation":"Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India","@type":"Person"}],"description":"Abstract.  Teak (Tectona grandis L. f.) is one of the precious bench mark tropical hardwood having qualities of durability, strength and visual pleasantries. Na","pageStart":"409","pageEnd":"419","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Draft genome of a high value tropical timber tree, Teak ( Tectona grandis  L. f): insights into SSR diversity, phylogeny and conservation","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Draft genome of a high value tropical timber tree, Teak ( Tectona grandis  L. f): insights into SSR diversity, phylogeny and conservation" />
    <meta property="og:description" content="Abstract.  Teak (Tectona grandis L. f.) is one of the precious bench mark tropical hardwood having qualities of durability, strength and visual pleasantries. Na" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/4/409/5003450" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-05-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yasodha, Ramasamy" /><meta name="citation_author_institution" content="Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India" /><meta name="citation_author" content="Vasudeva, Ramesh" /><meta name="citation_author_institution" content="Forest Genetics and Biotechnology Division, Kerala Forest Research Institute, Peechi, Thrissur, Kerala, India" /><meta name="citation_author" content="Balakrishnan, Swathi" /><meta name="citation_author_institution" content="Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India" /><meta name="citation_author" content="Sakthi, Ambothi Rathnasamy" /><meta name="citation_author_institution" content="Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India" /><meta name="citation_author" content="Abel, Nicodemus" /><meta name="citation_author_institution" content="Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India" /><meta name="citation_author" content="Binai, Nagarajan" /><meta name="citation_author_institution" content="Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India" /><meta name="citation_author" content="Rajashekar, Balaji" /><meta name="citation_author_institution" content="Genotypic Technology Private Limited, Bengaluru, Karnataka, India" /><meta name="citation_author_institution" content="Institute of Computer Science, University of Tartu, Estonia" /><meta name="citation_author" content="Bachpai, Vijay Kumar Waman" /><meta name="citation_author_institution" content="Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India" /><meta name="citation_author" content="Pillai, Chandrasekhara" /><meta name="citation_author_institution" content="Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India" /><meta name="citation_author" content="Dev, Suma Arun" /><meta name="citation_author_institution" content="Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India" /><meta name="citation_title" content="Draft genome of a high value tropical timber tree, Teak (Tectona grandis L. f): insights into SSR diversity, phylogeny and conservation" /><meta name="citation_firstpage" content="409" /><meta name="citation_lastpage" content="419" /><meta name="citation_doi" content="10.1093/dnares/dsy013" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2018/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="Tewari D.N.  1992 , A Monograph on Teak ( Tectona grandis Linn. f.), Dehra Dun, India: International Book Distributors " /><meta name="citation_reference" content="citation_title=Classification and geography of the flowering plants;  citation_journal_title=Bot. Rev;  citation_year=1992;  citation_volume=58;  citation_pages=225-327; " /><meta name="citation_reference" content="citation_publisher=Springer, Berlin;  citation_title=Families and Genera of Vascular Plants. Flowering Plants. Dicotyledons ∼ Lamiales (except Acanthaceae Including Avicenniaceae;  citation_year=2004;  citation_volume=vol. 7;  citation_pages=167-275; " /><meta name="citation_reference" content="citation_title=A large-scale chloroplast phylogeny of the Lamiaceae sheds new light on its subfamilial classification;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=3434" /><meta name="citation_reference" content="Kollert W., Kleine M. , eds. 2017 , The Global Teak Study. Analysis, Evaluation and Future Potential of Teak Resources, IUFRO World Series, 36. Vienna. pp 108. " /><meta name="citation_reference" content="Kollert W., Cherubini L.  2012 , Teak resources and market assessment 2010, FAO Planted Forests and Trees Working Paper, FP/47/E, Rome. " /><meta name="citation_reference" content="citation_title=Climatic-induced shifts in the distribution of teak (Tectona grandis) in tropical Asia: implications for forest management and planning;  citation_journal_title=Environ. Manag;  citation_year=2017;  citation_volume=60;  citation_pages=422-35; " /><meta name="citation_reference" content="citation_title=Worldwide translocation of teak—origin of landraces and present genetic base;  citation_journal_title=Tree Genet. Genomes;  citation_year=2017;  citation_volume=13;  citation_pages=87" /><meta name="citation_reference" content="citation_publisher=Danida Forest Seed Centre, Humlebaek;  citation_title=Evaluation of an International Series of Teak Provenance Trials;  citation_year=1986; " /><meta name="citation_reference" content="citation_publisher=Danida Forest Seed Centre, Humlebaek;  citation_title=Second Evaluation of an International Series of Teak Provenance Trails;  citation_year=1995; " /><meta name="citation_reference" content="citation_title=Genetic variation in major phenotypic traits among diverse genetic origins of teak (Tectona grandis L.f.) planted in Taliwas, Sabah, East Malaysia;  citation_journal_title=Ann. For. Sci;  citation_year=2011;  citation_volume=68;  citation_pages=1015-26; " /><meta name="citation_reference" content="citation_title=Genetic variation of growth and tree quality traits among 42 diverse genetic origins of Tectona grandis planted under humid tropical conditions in Sabah, East Malaysia;  citation_journal_title=Tree Genet. Genomes;  citation_year=2011;  citation_volume=7;  citation_pages=1263-75; " /><meta name="citation_reference" content="citation_title=Genetic parameters and correlations between stem size, forking, and flowering in teak (Tectona grandis);  citation_journal_title=Can. J. For. Res;  citation_year=2013;  citation_volume=43;  citation_pages=1145-50; " /><meta name="citation_reference" content="citation_title=Field growth performances of teak genotypes of different ages clonally produced by rooted cuttings, in vitro microcuttings, and meristem culture;  citation_journal_title=Can. J. For. Res;  citation_year=2015;  citation_volume=45;  citation_pages=9-14; " /><meta name="citation_reference" content="citation_title=Variation in flowering phenology in a clonal seed orchard of teak;  citation_journal_title=J. Tree Sci;  citation_year=2002;  citation_volume=21;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_publisher=Yokohama: India and International Tropical Timber Organization, ;  citation_title=Quality Timber Products of Teak from Sustainable Forest Management. Kerala Forest Research Institute;  citation_year=2005;  citation_pages=321-30; " /><meta name="citation_reference" content="citation_title=Assessment of genetic diversity in Tectona grandis using amplified fragment length polymorphism markers;  citation_journal_title=Can. J. For. Res;  citation_year=2005;  citation_volume=35;  citation_pages=1017-22; " /><meta name="citation_reference" content="citation_title=Genetic diversity of nine natural Tectona grandis L.f. populations of the Western Ghats in Southern India;  citation_journal_title=Conserv. Genet;  citation_year=2012;  citation_volume=13;  citation_pages=1409-19; " /><meta name="citation_reference" content="citation_title=Diversity and genetic structure of teak (Tectona grandisL.f.) in its natural range using DNA microsatellite markers;  citation_journal_title=New Forests;  citation_year=2009;  citation_volume=37;  citation_pages=175-95; " /><meta name="citation_reference" content="citation_title=Genetic resources of teak (Tectona grandis Linn. f.)—strong genetic structure among natural populations;  citation_journal_title=Tree Genet. Genomes;  citation_year=2015;  citation_volume=11;  citation_pages=802" /><meta name="citation_reference" content="citation_title=Large-scale transcriptional profiling of lignified tissues in Tectona grandis;  citation_journal_title=BMC Plant Biol;  citation_year=2015;  citation_volume=15;  citation_pages=221" /><meta name="citation_reference" content="citation_title=Transcriptome of teak (Tectona grandis, L.f) in vegetative to generative stages development;  citation_journal_title=J. Plant Sci;  citation_year=2015;  citation_volume=10;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Isolation of plant DNA from fresh tissue;  citation_journal_title=Focus;  citation_year=1990;  citation_volume=12;  citation_pages=13-5; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=The MaSuRCA genome assembler;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=2669-77; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of large genomes using compressed data structures;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=549-56; " /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=PSR: polymorphic SSR retrieval;  citation_journal_title=BMC Res. Notes;  citation_year=2015;  citation_volume=8; " /><meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools;  citation_journal_title=Nuc. Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=4876-82; " /><meta name="citation_reference" content="citation_title=BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT;  citation_journal_title=Nucleic Acids Symp. Ser;  citation_year=1999;  citation_volume=41;  citation_pages=95-8; " /><meta name="citation_reference" content="citation_title=JModelTest 2: more models, new heuristics and parallel computing;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=772" /><meta name="citation_reference" content="citation_title=A new look at statistical model identification;  citation_journal_title=IEEE Trans. Automat. Contr;  citation_year=1974;  citation_volume=19;  citation_pages=716-23; " /><meta name="citation_reference" content="citation_title=MRBAYES 3: bayesian phylogenetic inference under mixed models;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=1572-4; " /><meta name="citation_reference" content="citation_title=Bayesian phylogenetics with BEAUti and the BEAST 1.7;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=1969-73; " /><meta name="citation_reference" content="citation_title=New insights into evolutionary relationships within the subfamily Lamioideae (Lamiaceae) based on pentatricopeptide repeat (PPR) nuclear DNA sequences;  citation_journal_title=Amer. J. Bot;  citation_year=2015;  citation_volume=102;  citation_pages=1721-35; " /><meta name="citation_reference" content="citation_title=Relaxed phylogenetics and dating with confidence;  citation_journal_title=PLoS Biol;  citation_year=2006;  citation_volume=4;  citation_pages=e88-710; " /><meta name="citation_reference" content="citation_title=Elucidation of terpenoid metabolism in Scoparia dulcis by RNA-seq analysis;  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=43311" /><meta name="citation_reference" content="citation_title=Advances in ecological genomics in forest trees and applications to genetic resources conservation and breeding;  citation_journal_title=Mol. Ecol;  citation_year=2017;  citation_volume=26;  citation_pages=706-17; " /><meta name="citation_reference" content="citation_title=Nuclear DNA amounts in some tropical hardwoods;  citation_journal_title=Caryologia;  citation_year=1986;  citation_volume=39;  citation_pages=303-7; " /><meta name="citation_reference" content="citation_title=Draft genome of the wheat A-genome progenitor Triticum urartu;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=87-90; " /><meta name="citation_reference" content="citation_title=Transposable element insertions have strongly affected human evolution, Proc;  citation_journal_title=Natl. Acad. Sci. U. S. A;  citation_year=2010;  citation_volume=107;  citation_pages=19945-8; " /><meta name="citation_reference" content="citation_title=Repetitive sequences in plant nuclear DNA: types, distribution, evolution and function;  citation_journal_title=Genomics Proteomics Bioinformatics;  citation_year=2014;  citation_volume=12;  citation_pages=164-71; " /><meta name="citation_reference" content="citation_title=Molecular drive in multigene families: how biological novelties arise, spread and are assimilated;  citation_journal_title=Trends Genet;  citation_year=1986;  citation_volume=2;  citation_pages=159-65; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the medicinal plant;  citation_journal_title=Salvia Miltiorrhiza, Mol. Plant;  citation_year=2016;  citation_volume=9;  citation_pages=949-52; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Mentha longifolia and development of resources for mint cultivar improvement;  citation_journal_title=Mol. Plant;  citation_year=2017;  citation_volume=10;  citation_pages=323-39; " /><meta name="citation_reference" content="citation_title=Unravelling the genome of Holy basil: an “incomparable” “elixir of life” of traditional Indian medicine;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=413" /><meta name="citation_reference" content="citation_title=Genome sequencing of herb Tulsi (Ocimum tenuiflorum) unravels key genes behind its strong medicinal properties;  citation_journal_title=BMC Plant Biol;  citation_year=2015;  citation_volume=15;  citation_pages=212" /><meta name="citation_reference" content="citation_publisher=Koeltz Scientific Books, Königstein, Germany;  citation_title=Next Generation Sequencing in Plant Systematic, Regnum Vegetabile 157;  citation_year=2015;  citation_pages=1-25; " /><meta name="citation_reference" content="citation_title=Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies;  citation_journal_title=Genome Biol.;  citation_year=2014;  citation_volume=15;  citation_pages=R59" /><meta name="citation_reference" content="citation_title=An advanced draft genome assembly of a desi type chickpea (Cicer arietinum L.);  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=12806" /><meta name="citation_reference" content="citation_title=Phylogeny of Lamiidae;  citation_journal_title=Am. J. Bot;  citation_year=2014;  citation_volume=101;  citation_pages=287-99; " /><meta name="citation_reference" content="citation_title=Bioactive apocarotenoids from Tectona grandis;  citation_journal_title=Phytochemistry;  citation_year=2008;  citation_volume=69;  citation_pages=2708-15; " /><meta name="citation_reference" content="citation_title=Tectonoelins, new norlignans from a bioactive extract of Tectona grandis;  citation_journal_title=Phytochem. Lett;  citation_year=2012;  citation_volume=5;  citation_pages=382-6; " /><meta name="citation_reference" content="citation_title=An extended model of heartwood secondary metabolism informed by functional genomics;  citation_journal_title=Tree Physiol;  citation_year=2018;  citation_volume=14;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Microsatellites in different eukaryotic genome: survey and analysis, Genome,;  citation_journal_title=Res;  citation_year=2000;  citation_volume=10;  citation_pages=967-981; " /><meta name="citation_reference" content="citation_title=Microsatellite markers: what they mean and why they are so useful;  citation_journal_title=Genet. Mol. Biol;  citation_year=2016;  citation_volume=39;  citation_pages=312-28; " /><meta name="citation_reference" content="citation_publisher=Regency Publications, New Delhi;  citation_title=Flora of Eastern Ghats;  citation_year=2007;  citation_volume=Vol. 3; " /><meta name="citation_reference" content="citation_publisher=Key Botanic Garden, Kew, UK;  citation_title=Advances in Labiatae Science;  citation_year=1992;  citation_pages=511-22; " /><meta name="citation_reference" content="citation_title=A comparison of phylogenetic nomenclature with the current system: a botanical case study, Syst.;  citation_journal_title=Biol;  citation_year=1997;  citation_volume=46;  citation_pages=313-31; " /><meta name="citation_reference" content="citation_title=Phylogeny of Labiatae and Verbenaceae inferred from rbcL sequences;  citation_journal_title=Syst. Bot;  citation_year=1997;  citation_volume=22;  citation_pages=165-79; " /><meta name="citation_reference" content="citation_title=Phylogeny in Labiatae s. l., inferred from cpDNA sequences;  citation_journal_title=Plant Syst. Evol;  citation_year=1998;  citation_volume=209;  citation_pages=265-74; " /><meta name="citation_reference" content="citation_title=A computer-assisted approach for chemotaxonomic studies - diterpenes in Lamiaceae;  citation_journal_title=Phytochemistry;  citation_year=2001;  citation_volume=56;  citation_pages=583-95; " /><meta name="citation_reference" content="citation_title=Troublesome tropical mints: re-examining generic limits of Vitex and relations (Lamiaceae) in South East Asia;  citation_journal_title=Taxon;  citation_year=2009;  citation_volume=58;  citation_pages=500-10; " /><meta name="citation_reference" content="citation_title=The atypical codon usage of the plant psbA gene may be the remnant of an ancestral bias;  citation_journal_title=Proc. Nat. Aca. Sci. U.S.A;  citation_year=1997;  citation_volume=94;  citation_pages=11434-8; " /><meta name="citation_reference" content="citation_title=Molecular phylogeny and taxonomy of genus Lamium L (Lamiaceae): Disentangling origins of presumed allotetraploids;  citation_journal_title=Taxon;  citation_year=2011;  citation_volume=60;  citation_pages=986-1000; " /><meta name="citation_reference" content="citation_title=Molecular phylogenetics of Chelonopsis (Lamiaceae: gomphostemmateae) as inferred from nuclear and plastid DNA and morphology;  citation_journal_title=Taxon;  citation_year=2013;  citation_volume=62;  citation_pages=375-86; " /><meta name="citation_reference" content="citation_title=Evolution and origins of the Mazatec hallucinogenic sage, Salvia divinorum (Lamiaceae): a molecular phylogenetic approach;  citation_journal_title=J. Plant Res;  citation_year=2011;  citation_volume=124;  citation_pages=593-600; " /><meta name="citation_reference" content="citation_title=Circumscription and phylogenetic relationships of Prostanthera densa and P. marifolia (Lamiaceae);  citation_journal_title=Telopea;  citation_year=2013;  citation_volume=15;  citation_pages=149-64; " /><meta name="citation_reference" content="citation_title=Phylogenetic utility of ycf1 in orchids: a plastid gene more variable than matK;  citation_journal_title=Plant Syst. Evol.;  citation_year=2009;  citation_volume=277;  citation_pages=75-84; " /><meta name="citation_reference" content="citation_title=ycf1, the most promising plastid DNA barcode of land plants;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=8348" /><meta name="citation_reference" content="citation_title=Highly variable chloroplast markers for evaluating plant phylogeny at low taxonomic levels and for DNA barcoding;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e35071" /><meta name="citation_reference" content="citation_publisher=Engelmann, Leipzig;  citation_title=Die Naturlichen Pflanzenfamilien Teil 4.;  citation_year=1897;  citation_pages=132-182; " /><meta name="citation_reference" content="citation_publisher=Borntraeger, Berlin;  citation_title=A. Engler’s Syllabus Der Pflanzenfamilien;  citation_year=1964;  citation_volume=vol. 2;  citation_pages=666" /><meta name="citation_reference" content="citation_title=Notes on new and noteworthy plants LXXVII;  citation_journal_title=Phytologia;  citation_year=1975;  citation_volume=31;  citation_pages=28" /><meta name="citation_reference" content="citation_title=Outline of the classification of flowering plants (Magnoliophyta);  citation_journal_title=Brittonia;  citation_year=1983;  citation_volume=35;  citation_pages=254-359; " /><meta name="citation_reference" content="citation_title=Two new subfamilies in Lamiaceae;  citation_journal_title=Phytotaxa;  citation_year=2017;  citation_volume=313;  citation_pages=222-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/4/409/25501624/dsy013.pdf" /><meta name="description" content="Abstract.  Teak (Tectona grandis L. f.) is one of the precious bench mark tropical hardwood having qualities of durability, strength and visual pleasantries. Na" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/4/409/5003450" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/4/409/5003450" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/4/409/5003450");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy013';
    </script>
<title>
	Draft genome of a high value tropical timber tree, Teak (Tectona grandis L. f): insights into SSR diversity, phylogeny and conservation | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5003450" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcCuATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFzb2RoYSwgUmFtYXNhbXkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBQbGFudCBCaW90ZWNobm9sb2d5LCBJbnN0aXR1dGUgb2YgRm9yZXN0IEdlbmV0aWNzIGFuZCBUcmVlIEJyZWVkaW5nLCBSLlMuIFB1cmFtLCBDb2ltYmF0b3JlLCBUYW1pbCBOYWR1LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZhc3VkZXZhLCBSYW1lc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGb3Jlc3QgR2VuZXRpY3MgYW5kIEJpb3RlY2hub2xvZ3kgRGl2aXNpb24sIEtlcmFsYSBGb3Jlc3QgUmVzZWFyY2ggSW5zdGl0dXRlLCBQZWVjaGksIFRocmlzc3VyLCBLZXJhbGEsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmFsYWtyaXNobmFuLCBTd2F0aGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEZvcmVzdCBCaW9sb2d5IGFuZCBUcmVlIEltcHJvdmVtZW50LCBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQ29sbGVnZSBvZiBGb3Jlc3RyeSwgU2lyc2ksIFV0dGFyYSBLYW5uYWRhLCBLYXJuYXRha2EsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FrdGhpLCBBbWJvdGhpIFJhdGhuYXNhbXkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBQbGFudCBCaW90ZWNobm9sb2d5LCBJbnN0aXR1dGUgb2YgRm9yZXN0IEdlbmV0aWNzIGFuZCBUcmVlIEJyZWVkaW5nLCBSLlMuIFB1cmFtLCBDb2ltYmF0b3JlLCBUYW1pbCBOYWR1LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFiZWwsIE5pY29kZW11cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIFBsYW50IEJpb3RlY2hub2xvZ3ksIEluc3RpdHV0ZSBvZiBGb3Jlc3QgR2VuZXRpY3MgYW5kIFRyZWUgQnJlZWRpbmcsIFIuUy4gUHVyYW0sIENvaW1iYXRvcmUsIFRhbWlsIE5hZHUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmluYWksIE5hZ2FyYWphbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIFBsYW50IEJpb3RlY2hub2xvZ3ksIEluc3RpdHV0ZSBvZiBGb3Jlc3QgR2VuZXRpY3MgYW5kIFRyZWUgQnJlZWRpbmcsIFIuUy4gUHVyYW0sIENvaW1iYXRvcmUsIFRhbWlsIE5hZHUsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFqYXNoZWthciwgQmFsYWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR2Vub3R5cGljIFRlY2hub2xvZ3kgUHJpdmF0ZSBMaW1pdGVkLCBCZW5nYWx1cnUsIEthcm5hdGFrYSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQ29tcHV0ZXIgU2NpZW5jZSwgVW5pdmVyc2l0eSBvZiBUYXJ0dSwgRXN0b25pYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJhY2hwYWksIFZpamF5IEt1bWFyIFdhbWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgUGxhbnQgQmlvdGVjaG5vbG9neSwgSW5zdGl0dXRlIG9mIEZvcmVzdCBHZW5ldGljcyBhbmQgVHJlZSBCcmVlZGluZywgUi5TLiBQdXJhbSwgQ29pbWJhdG9yZSwgVGFtaWwgTmFkdSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQaWxsYWksIENoYW5kcmFzZWtoYXJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBGb3Jlc3QgQmlvbG9neSBhbmQgVHJlZSBJbXByb3ZlbWVudCwgVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIENvbGxlZ2Ugb2YgRm9yZXN0cnksIFNpcnNpLCBVdHRhcmEgS2FubmFkYSwgS2FybmF0YWthLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRldiwgU3VtYSBBcnVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBGb3Jlc3QgQmlvbG9neSBhbmQgVHJlZSBJbXByb3ZlbWVudCwgVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIENvbGxlZ2Ugb2YgRm9yZXN0cnksIFNpcnNpLCBVdHRhcmEgS2FubmFkYSwgS2FybmF0YWthLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRHJhZnQgZ2Vub21lIG9mIGEgaGlnaCB2YWx1ZSB0cm9waWNhbCB0aW1iZXIgdHJlZSwgVGVhayAoVGVjdG9uYSBncmFuZGlzIEwuIGYpOiBpbnNpZ2h0cyBpbnRvIFNTUiBkaXZlcnNpdHksIHBoeWxvZ2VueSBhbmQgY29uc2VydmF0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDA5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0MTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlRld2FyaSBELk4uICAxOTkyICwgQSBNb25vZ3JhcGggb24gVGVhayAoIFRlY3RvbmEgZ3JhbmRpcyBMaW5uLiBmLiksIERlaHJhIER1biwgSW5kaWE6IEludGVybmF0aW9uYWwgQm9vayBEaXN0cmlidXRvcnMgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xhc3NpZmljYXRpb24gYW5kIGdlb2dyYXBoeSBvZiB0aGUgZmxvd2VyaW5nIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qm90LiBSZXY7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0yMjUtMzI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIEJlcmxpbjsgIGNpdGF0aW9uX3RpdGxlPUZhbWlsaWVzIGFuZCBHZW5lcmEgb2YgVmFzY3VsYXIgUGxhbnRzLiBGbG93ZXJpbmcgUGxhbnRzLiBEaWNvdHlsZWRvbnMg4oi8IExhbWlhbGVzIChleGNlcHQgQWNhbnRoYWNlYWUgSW5jbHVkaW5nIEF2aWNlbm5pYWNlYWU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9dm9sLiA3OyAgY2l0YXRpb25fcGFnZXM9MTY3LTI3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBsYXJnZS1zY2FsZSBjaGxvcm9wbGFzdCBwaHlsb2dlbnkgb2YgdGhlIExhbWlhY2VhZSBzaGVkcyBuZXcgbGlnaHQgb24gaXRzIHN1YmZhbWlsaWFsIGNsYXNzaWZpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzQzNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IktvbGxlcnQgVy4sIEtsZWluZSBNLiAsIGVkcy4gMjAxNyAsIFRoZSBHbG9iYWwgVGVhayBTdHVkeS4gQW5hbHlzaXMsIEV2YWx1YXRpb24gYW5kIEZ1dHVyZSBQb3RlbnRpYWwgb2YgVGVhayBSZXNvdXJjZXMsIElVRlJPIFdvcmxkIFNlcmllcywgMzYuIFZpZW5uYS4gcHAgMTA4LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJLb2xsZXJ0IFcuLCBDaGVydWJpbmkgTC4gIDIwMTIgLCBUZWFrIHJlc291cmNlcyBhbmQgbWFya2V0IGFzc2Vzc21lbnQgMjAxMCwgRkFPIFBsYW50ZWQgRm9yZXN0cyBhbmQgVHJlZXMgV29ya2luZyBQYXBlciwgRlAvNDcvRSwgUm9tZS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xpbWF0aWMtaW5kdWNlZCBzaGlmdHMgaW4gdGhlIGRpc3RyaWJ1dGlvbiBvZiB0ZWFrIChUZWN0b25hIGdyYW5kaXMpIGluIHRyb3BpY2FsIEFzaWE6IGltcGxpY2F0aW9ucyBmb3IgZm9yZXN0IG1hbmFnZW1lbnQgYW5kIHBsYW5uaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbnZpcm9uLiBNYW5hZzsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTQyMi0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V29ybGR3aWRlIHRyYW5zbG9jYXRpb24gb2YgdGVha+KAlG9yaWdpbiBvZiBsYW5kcmFjZXMgYW5kIHByZXNlbnQgZ2VuZXRpYyBiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVlIEdlbmV0LiBHZW5vbWVzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9ODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9RGFuaWRhIEZvcmVzdCBTZWVkIENlbnRyZSwgSHVtbGViYWVrOyAgY2l0YXRpb25fdGl0bGU9RXZhbHVhdGlvbiBvZiBhbiBJbnRlcm5hdGlvbmFsIFNlcmllcyBvZiBUZWFrIFByb3ZlbmFuY2UgVHJpYWxzOyAgY2l0YXRpb25feWVhcj0xOTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9RGFuaWRhIEZvcmVzdCBTZWVkIENlbnRyZSwgSHVtbGViYWVrOyAgY2l0YXRpb25fdGl0bGU9U2Vjb25kIEV2YWx1YXRpb24gb2YgYW4gSW50ZXJuYXRpb25hbCBTZXJpZXMgb2YgVGVhayBQcm92ZW5hbmNlIFRyYWlsczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyB2YXJpYXRpb24gaW4gbWFqb3IgcGhlbm90eXBpYyB0cmFpdHMgYW1vbmcgZGl2ZXJzZSBnZW5ldGljIG9yaWdpbnMgb2YgdGVhayAoVGVjdG9uYSBncmFuZGlzIEwuZi4pIHBsYW50ZWQgaW4gVGFsaXdhcywgU2FiYWgsIEVhc3QgTWFsYXlzaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gRm9yLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0xMDE1LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHZhcmlhdGlvbiBvZiBncm93dGggYW5kIHRyZWUgcXVhbGl0eSB0cmFpdHMgYW1vbmcgNDIgZGl2ZXJzZSBnZW5ldGljIG9yaWdpbnMgb2YgVGVjdG9uYSBncmFuZGlzIHBsYW50ZWQgdW5kZXIgaHVtaWQgdHJvcGljYWwgY29uZGl0aW9ucyBpbiBTYWJhaCwgRWFzdCBNYWxheXNpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTI2My03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBwYXJhbWV0ZXJzIGFuZCBjb3JyZWxhdGlvbnMgYmV0d2VlbiBzdGVtIHNpemUsIGZvcmtpbmcsIGFuZCBmbG93ZXJpbmcgaW4gdGVhayAoVGVjdG9uYSBncmFuZGlzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2FuLiBKLiBGb3IuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTExNDUtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpZWxkIGdyb3d0aCBwZXJmb3JtYW5jZXMgb2YgdGVhayBnZW5vdHlwZXMgb2YgZGlmZmVyZW50IGFnZXMgY2xvbmFsbHkgcHJvZHVjZWQgYnkgcm9vdGVkIGN1dHRpbmdzLCBpbiB2aXRybyBtaWNyb2N1dHRpbmdzLCBhbmQgbWVyaXN0ZW0gY3VsdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2FuLiBKLiBGb3IuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTktMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpbiBmbG93ZXJpbmcgcGhlbm9sb2d5IGluIGEgY2xvbmFsIHNlZWQgb3JjaGFyZCBvZiB0ZWFrOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBUcmVlIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Zb2tvaGFtYTogSW5kaWEgYW5kIEludGVybmF0aW9uYWwgVHJvcGljYWwgVGltYmVyIE9yZ2FuaXphdGlvbiwgOyAgY2l0YXRpb25fdGl0bGU9UXVhbGl0eSBUaW1iZXIgUHJvZHVjdHMgb2YgVGVhayBmcm9tIFN1c3RhaW5hYmxlIEZvcmVzdCBNYW5hZ2VtZW50LiBLZXJhbGEgRm9yZXN0IFJlc2VhcmNoIEluc3RpdHV0ZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3BhZ2VzPTMyMS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiBnZW5ldGljIGRpdmVyc2l0eSBpbiBUZWN0b25hIGdyYW5kaXMgdXNpbmcgYW1wbGlmaWVkIGZyYWdtZW50IGxlbmd0aCBwb2x5bW9ycGhpc20gbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2FuLiBKLiBGb3IuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IG9mIG5pbmUgbmF0dXJhbCBUZWN0b25hIGdyYW5kaXMgTC5mLiBwb3B1bGF0aW9ucyBvZiB0aGUgV2VzdGVybiBHaGF0cyBpbiBTb3V0aGVybiBJbmRpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29uc2Vydi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNDA5LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgYW5kIGdlbmV0aWMgc3RydWN0dXJlIG9mIHRlYWsgKFRlY3RvbmEgZ3JhbmRpc0wuZi4pIGluIGl0cyBuYXR1cmFsIHJhbmdlIHVzaW5nIEROQSBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgRm9yZXN0czsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTE3NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyByZXNvdXJjZXMgb2YgdGVhayAoVGVjdG9uYSBncmFuZGlzIExpbm4uIGYuKeKAlHN0cm9uZyBnZW5ldGljIHN0cnVjdHVyZSBhbW9uZyBuYXR1cmFsIHBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVlIEdlbmV0LiBHZW5vbWVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgdHJhbnNjcmlwdGlvbmFsIHByb2ZpbGluZyBvZiBsaWduaWZpZWQgdGlzc3VlcyBpbiBUZWN0b25hIGdyYW5kaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBvZiB0ZWFrIChUZWN0b25hIGdyYW5kaXMsIEwuZikgaW4gdmVnZXRhdGl2ZSB0byBnZW5lcmF0aXZlIHN0YWdlcyBkZXZlbG9wbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIHBsYW50IEROQSBmcm9tIGZyZXNoIHRpc3N1ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Rm9jdXM7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMy01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZGUgbm92byBhc3NlbWJseSBvZiBoaWdobHkgaGV0ZXJvenlnb3VzIGdlbm9tZXMgZnJvbSB3aG9sZS1nZW5vbWUgc2hvdGd1biBzaG9ydCByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBNYVN1UkNBIGdlbm9tZSBhc3NlbWJsZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MjY2OS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NhZmZvbGRpbmcgcHJlLWFzc2VtYmxlZCBjb250aWdzIHVzaW5nIFNTUEFDRTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGRlIG5vdm8gYXNzZW1ibHkgb2YgbGFyZ2UgZ2Vub21lcyB1c2luZyBjb21wcmVzc2VkIGRhdGEgc3RydWN0dXJlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbmF0aXZlIGFuZCBzeW50ZW5pY2FsbHkgbWFwcGVkIGNETkEgYWxpZ25tZW50cyB0byBpbXByb3ZlIGRlIG5vdm8gZ2VuZSBmaW5kaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTYzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UFNSOiBwb2x5bW9ycGhpYyBTU1IgcmV0cmlldmFsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUmVzLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ0xVU1RBTF9YIHdpbmRvd3MgaW50ZXJmYWNlOiBmbGV4aWJsZSBzdHJhdGVnaWVzIGZvciBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYWlkZWQgYnkgcXVhbGl0eSBhbmFseXNpcyB0b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00ODc2LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9FZGl0OiBhIHVzZXItZnJpZW5kbHkgYmlvbG9naWNhbCBzZXF1ZW5jZSBhbGlnbm1lbnQgZWRpdG9yIGFuZCBhbmFseXNpcyBwcm9ncmFtIGZvciBXaW5kb3dzIDk1Lzk4L05UOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFN5bXAuIFNlcjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTk1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUpNb2RlbFRlc3QgMjogbW9yZSBtb2RlbHMsIG5ldyBoZXVyaXN0aWNzIGFuZCBwYXJhbGxlbCBjb21wdXRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NzcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgbG9vayBhdCBzdGF0aXN0aWNhbCBtb2RlbCBpZGVudGlmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SUVFRSBUcmFucy4gQXV0b21hdC4gQ29udHI7ICBjaXRhdGlvbl95ZWFyPTE5NzQ7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz03MTYtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1SQkFZRVMgMzogYmF5ZXNpYW4gcGh5bG9nZW5ldGljIGluZmVyZW5jZSB1bmRlciBtaXhlZCBtb2RlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTU3Mi00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXllc2lhbiBwaHlsb2dlbmV0aWNzIHdpdGggQkVBVXRpIGFuZCB0aGUgQkVBU1QgMS43OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xOTY5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgaW5zaWdodHMgaW50byBldm9sdXRpb25hcnkgcmVsYXRpb25zaGlwcyB3aXRoaW4gdGhlIHN1YmZhbWlseSBMYW1pb2lkZWFlIChMYW1pYWNlYWUpIGJhc2VkIG9uIHBlbnRhdHJpY29wZXB0aWRlIHJlcGVhdCAoUFBSKSBudWNsZWFyIEROQSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtZXIuIEouIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xNzIxLTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWxheGVkIHBoeWxvZ2VuZXRpY3MgYW5kIGRhdGluZyB3aXRoIGNvbmZpZGVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTg4LTcxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWx1Y2lkYXRpb24gb2YgdGVycGVub2lkIG1ldGFib2xpc20gaW4gU2NvcGFyaWEgZHVsY2lzIGJ5IFJOQS1zZXEgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00MzMxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIGVjb2xvZ2ljYWwgZ2Vub21pY3MgaW4gZm9yZXN0IHRyZWVzIGFuZCBhcHBsaWNhdGlvbnMgdG8gZ2VuZXRpYyByZXNvdXJjZXMgY29uc2VydmF0aW9uIGFuZCBicmVlZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NzA2LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZWFyIEROQSBhbW91bnRzIGluIHNvbWUgdHJvcGljYWwgaGFyZHdvb2RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DYXJ5b2xvZ2lhOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MzAzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBvZiB0aGUgd2hlYXQgQS1nZW5vbWUgcHJvZ2VuaXRvciBUcml0aWN1bSB1cmFydHU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTY7ICBjaXRhdGlvbl9wYWdlcz04Ny05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NhYmxlIGVsZW1lbnQgaW5zZXJ0aW9ucyBoYXZlIHN0cm9uZ2x5IGFmZmVjdGVkIGh1bWFuIGV2b2x1dGlvbiwgUHJvYzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTE5OTQ1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGV0aXRpdmUgc2VxdWVuY2VzIGluIHBsYW50IG51Y2xlYXIgRE5BOiB0eXBlcywgZGlzdHJpYnV0aW9uLCBldm9sdXRpb24gYW5kIGZ1bmN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljcyBQcm90ZW9taWNzIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTY0LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZHJpdmUgaW4gbXVsdGlnZW5lIGZhbWlsaWVzOiBob3cgYmlvbG9naWNhbCBub3ZlbHRpZXMgYXJpc2UsIHNwcmVhZCBhbmQgYXJlIGFzc2ltaWxhdGVkOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbWVkaWNpbmFsIHBsYW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TYWx2aWEgTWlsdGlvcnJoaXphLCBNb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz05NDktNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBNZW50aGEgbG9uZ2lmb2xpYSBhbmQgZGV2ZWxvcG1lbnQgb2YgcmVzb3VyY2VzIGZvciBtaW50IGN1bHRpdmFyIGltcHJvdmVtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MzIzLTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbnJhdmVsbGluZyB0aGUgZ2Vub21lIG9mIEhvbHkgYmFzaWw6IGFuIOKAnGluY29tcGFyYWJsZeKAnSDigJxlbGl4aXIgb2YgbGlmZeKAnSBvZiB0cmFkaXRpb25hbCBJbmRpYW4gbWVkaWNpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIG9mIGhlcmIgVHVsc2kgKE9jaW11bSB0ZW51aWZsb3J1bSkgdW5yYXZlbHMga2V5IGdlbmVzIGJlaGluZCBpdHMgc3Ryb25nIG1lZGljaW5hbCBwcm9wZXJ0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Lb2VsdHogU2NpZW50aWZpYyBCb29rcywgS8O2bmlnc3RlaW4sIEdlcm1hbnk7ICBjaXRhdGlvbl90aXRsZT1OZXh0IEdlbmVyYXRpb24gU2VxdWVuY2luZyBpbiBQbGFudCBTeXN0ZW1hdGljLCBSZWdudW0gVmVnZXRhYmlsZSAxNTc7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl9wYWdlcz0xLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWNvZGluZyB0aGUgbWFzc2l2ZSBnZW5vbWUgb2YgbG9ibG9sbHkgcGluZSB1c2luZyBoYXBsb2lkIEROQSBhbmQgbm92ZWwgYXNzZW1ibHkgc3RyYXRlZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9UjU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYWR2YW5jZWQgZHJhZnQgZ2Vub21lIGFzc2VtYmx5IG9mIGEgZGVzaSB0eXBlIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTI4MDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbnkgb2YgTGFtaWlkYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9Mjg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9hY3RpdmUgYXBvY2Fyb3Rlbm9pZHMgZnJvbSBUZWN0b25hIGdyYW5kaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MjcwOC0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVjdG9ub2VsaW5zLCBuZXcgbm9ybGlnbmFucyBmcm9tIGEgYmlvYWN0aXZlIGV4dHJhY3Qgb2YgVGVjdG9uYSBncmFuZGlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHl0b2NoZW0uIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTM4Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBleHRlbmRlZCBtb2RlbCBvZiBoZWFydHdvb2Qgc2Vjb25kYXJ5IG1ldGFib2xpc20gaW5mb3JtZWQgYnkgZnVuY3Rpb25hbCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlZSBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXMgaW4gZGlmZmVyZW50IGV1a2FyeW90aWMgZ2Vub21lOiBzdXJ2ZXkgYW5kIGFuYWx5c2lzLCBHZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1SZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz05NjctOTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBtYXJrZXJzOiB3aGF0IHRoZXkgbWVhbiBhbmQgd2h5IHRoZXkgYXJlIHNvIHVzZWZ1bDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTMxMi0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVJlZ2VuY3kgUHVibGljYXRpb25zLCBOZXcgRGVsaGk7ICBjaXRhdGlvbl90aXRsZT1GbG9yYSBvZiBFYXN0ZXJuIEdoYXRzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPVZvbC4gMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUtleSBCb3RhbmljIEdhcmRlbiwgS2V3LCBVSzsgIGNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIExhYmlhdGFlIFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl9wYWdlcz01MTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGFyaXNvbiBvZiBwaHlsb2dlbmV0aWMgbm9tZW5jbGF0dXJlIHdpdGggdGhlIGN1cnJlbnQgc3lzdGVtOiBhIGJvdGFuaWNhbCBjYXNlIHN0dWR5LCBTeXN0LjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTMxMy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55IG9mIExhYmlhdGFlIGFuZCBWZXJiZW5hY2VhZSBpbmZlcnJlZCBmcm9tIHJiY0wgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjUtNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VueSBpbiBMYWJpYXRhZSBzLiBsLiwgaW5mZXJyZWQgZnJvbSBjcEROQSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFN5c3QuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjA5OyAgY2l0YXRpb25fcGFnZXM9MjY1LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXB1dGVyLWFzc2lzdGVkIGFwcHJvYWNoIGZvciBjaGVtb3RheG9ub21pYyBzdHVkaWVzIC0gZGl0ZXJwZW5lcyBpbiBMYW1pYWNlYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9NTgzLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ucm91Ymxlc29tZSB0cm9waWNhbCBtaW50czogcmUtZXhhbWluaW5nIGdlbmVyaWMgbGltaXRzIG9mIFZpdGV4IGFuZCByZWxhdGlvbnMgKExhbWlhY2VhZSkgaW4gU291dGggRWFzdCBBc2lhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UYXhvbjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTUwMC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGF0eXBpY2FsIGNvZG9uIHVzYWdlIG9mIHRoZSBwbGFudCBwc2JBIGdlbmUgbWF5IGJlIHRoZSByZW1uYW50IG9mIGFuIGFuY2VzdHJhbCBiaWFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXQuIEFjYS4gU2NpLiBVLlMuQTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTExNDM0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBwaHlsb2dlbnkgYW5kIHRheG9ub215IG9mIGdlbnVzIExhbWl1bSBMIChMYW1pYWNlYWUpOiBEaXNlbnRhbmdsaW5nIG9yaWdpbnMgb2YgcHJlc3VtZWQgYWxsb3RldHJhcGxvaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UYXhvbjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTk4Ni0xMDAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgcGh5bG9nZW5ldGljcyBvZiBDaGVsb25vcHNpcyAoTGFtaWFjZWFlOiBnb21waG9zdGVtbWF0ZWFlKSBhcyBpbmZlcnJlZCBmcm9tIG51Y2xlYXIgYW5kIHBsYXN0aWQgRE5BIGFuZCBtb3JwaG9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UYXhvbjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTM3NS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIGFuZCBvcmlnaW5zIG9mIHRoZSBNYXphdGVjIGhhbGx1Y2lub2dlbmljIHNhZ2UsIFNhbHZpYSBkaXZpbm9ydW0gKExhbWlhY2VhZSk6IGEgbW9sZWN1bGFyIHBoeWxvZ2VuZXRpYyBhcHByb2FjaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGxhbnQgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTU5My02MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bXNjcmlwdGlvbiBhbmQgcGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHMgb2YgUHJvc3RhbnRoZXJhIGRlbnNhIGFuZCBQLiBtYXJpZm9saWEgKExhbWlhY2VhZSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRlbG9wZWE7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNDktNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyB1dGlsaXR5IG9mIHljZjEgaW4gb3JjaGlkczogYSBwbGFzdGlkIGdlbmUgbW9yZSB2YXJpYWJsZSB0aGFuIG1hdEs7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFN5c3QuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3NzsgIGNpdGF0aW9uX3BhZ2VzPTc1LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT15Y2YxLCB0aGUgbW9zdCBwcm9taXNpbmcgcGxhc3RpZCBETkEgYmFyY29kZSBvZiBsYW5kIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTgzNDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHkgdmFyaWFibGUgY2hsb3JvcGxhc3QgbWFya2VycyBmb3IgZXZhbHVhdGluZyBwbGFudCBwaHlsb2dlbnkgYXQgbG93IHRheG9ub21pYyBsZXZlbHMgYW5kIGZvciBETkEgYmFyY29kaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTM1MDcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUVuZ2VsbWFubiwgTGVpcHppZzsgIGNpdGF0aW9uX3RpdGxlPURpZSBOYXR1cmxpY2hlbiBQZmxhbnplbmZhbWlsaWVuIFRlaWwgNC47ICBjaXRhdGlvbl95ZWFyPTE4OTc7ICBjaXRhdGlvbl9wYWdlcz0xMzItMTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Qm9ybnRyYWVnZXIsIEJlcmxpbjsgIGNpdGF0aW9uX3RpdGxlPUEuIEVuZ2xlcuKAmXMgU3lsbGFidXMgRGVyIFBmbGFuemVuZmFtaWxpZW47ICBjaXRhdGlvbl95ZWFyPTE5NjQ7ICBjaXRhdGlvbl92b2x1bWU9dm9sLiAyOyAgY2l0YXRpb25fcGFnZXM9NjY2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm90ZXMgb24gbmV3IGFuZCBub3Rld29ydGh5IHBsYW50cyBMWFhWSUk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU91dGxpbmUgb2YgdGhlIGNsYXNzaWZpY2F0aW9uIG9mIGZsb3dlcmluZyBwbGFudHMgKE1hZ25vbGlvcGh5dGEpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Ccml0dG9uaWE7ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0yNTQtMzU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gbmV3IHN1YmZhbWlsaWVzIGluIExhbWlhY2VhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG90YXhhOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTMxMzsgIGNpdGF0aW9uX3BhZ2VzPTIyMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvNC80MDkvMjU1MDE2MjQvZHN5MDEzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUZWFrIChUZWN0b25hIGdyYW5kaXMgTC4gZi4pIGlzIG9uZSBvZiB0aGUgcHJlY2lvdXMgYmVuY2ggbWFyayB0cm9waWNhbCBoYXJkd29vZCBoYXZpbmcgcXVhbGl0aWVzIG9mIGR1cmFiaWxpdHksIHN0cmVuZ3RoIGFuZCB2aXN1YWwgcGxlYXNhbnRyaWVzLiBOYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTAwMzQ1MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTHW9MijyCfQfWVAwgtJzz/Qfr0HQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f409%2f5003450" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f409%2f5003450" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109685" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&amp;Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/4">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="120121804">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121804">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121806">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121806">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121811">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121811">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121828">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121828">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121850">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121850">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121852">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121852">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121856">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121856">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/4/395/4989276">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/4/421/5020793">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Draft genome of a high value tropical timber tree, Teak (<em>Tectona grandis</em> L. f): insights into SSR diversity, phylogeny and conservation
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ramasamy Yasodha</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy013-cor1" data-open="dsy013-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +91 422 248 4114. Email: <a href="mailto:yasodha@icfre.org" target="_blank">yasodha@icfre.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ramasamy+Yasodha">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yasodha R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yasodha R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ramasamy Yasodha</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ramesh Vasudeva</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Forest Genetics and Biotechnology Division, Kerala Forest Research Institute, Peechi, Thrissur, Kerala, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ramesh+Vasudeva">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vasudeva R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vasudeva R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ramesh Vasudeva</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Swathi Balakrishnan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Swathi+Balakrishnan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Balakrishnan S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Balakrishnan S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Swathi Balakrishnan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ambothi Rathnasamy Sakthi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ambothi+Rathnasamy+Sakthi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakthi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakthi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ambothi Rathnasamy Sakthi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nicodemus Abel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nicodemus+Abel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abel N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abel N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nicodemus Abel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nagarajan Binai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nagarajan+Binai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Binai N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Binai N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nagarajan Binai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Balaji Rajashekar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Genotypic Technology Private Limited, Bengaluru, Karnataka, India</div><div class="aff">Institute of Computer Science, University of Tartu, Estonia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Balaji+Rajashekar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rajashekar B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rajashekar B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Balaji Rajashekar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vijay Kumar Waman Bachpai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Plant Biotechnology, Institute of Forest Genetics and Tree Breeding, R.S. Puram, Coimbatore, Tamil Nadu, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vijay+Kumar+Waman+Bachpai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bachpai V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bachpai V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Vijay Kumar Waman Bachpai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chandrasekhara Pillai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chandrasekhara+Pillai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pillai C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pillai C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chandrasekhara Pillai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suma Arun Dev</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Forest Biology and Tree Improvement, University of Agricultural Sciences, College of Forestry, Sirsi, Uttara Kannada, Karnataka, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suma+Arun+Dev">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dev S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dev S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Suma Arun Dev</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy013-FM2"><div class="fn"><div class="footnote-content">Ramasamy Yasodha and Suma Arun Dev authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy013-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 409–419, <a href='https://doi.org/10.1093/dnares/dsy013'>https://doi.org/10.1093/dnares/dsy013</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 May 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 December 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 April 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5003450" href="/dnaresearch/article-pdf/25/4/409/25501624/dsy013.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Ramasamy Yasodha, Ramesh Vasudeva, Swathi Balakrishnan, Ambothi Rathnasamy Sakthi, Nicodemus Abel, Nagarajan Binai, Balaji Rajashekar, Vijay Kumar Waman Bachpai, Chandrasekhara Pillai, Suma Arun Dev;  Draft genome of a high value tropical timber tree, Teak (<em>Tectona grandis</em> L. f): insights into SSR diversity, phylogeny and conservation, <em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 409–419, <a href='https://doi.org/10.1093/dnares/dsy013'>https://doi.org/10.1093/dnares/dsy013</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5003450?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5003450?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5003450?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5003450?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5003450?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5003450?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5003450?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Draft%20genome%20of%20a%20high%20value%20tropical%20timber%20tree%2C%20Teak%20(%26lt%3Bem%26gt%3BTectona%20grandis%26lt%3B%2Fem%26gt%3B%20L.%20f)%3A%20insights%20into%20SSR%20diversity%2C%20phylogeny%20and%20conservation&amp;publicationDate=2018-05-24&amp;volumeNum=25&amp;issueNum=4&amp;author=Yasodha%2C%20Ramasamy%3B%20Vasudeva%2C%20Ramesh&amp;startPage=409&amp;endPage=419&amp;contentId=10.1093%2Fdnares%2Fdsy013&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f409%2f5003450" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=120121804 id="120121804" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Teak (<em>Tectona grandis</em> L. f.) is one of the precious bench mark tropical hardwood having qualities of durability, strength and visual pleasantries. Natural teak populations harbour a variety of characteristics that determine their economic, ecological and environmental importance. Sequencing of whole nuclear genome of teak provides a platform for functional analyses and development of genomic tools in applied tree improvement. A draft genome of 317 Mb was assembled at 151× coverage and annotated 36, 172 protein-coding genes. Approximately about 11.18% of the genome was repetitive. Microsatellites or simple sequence repeats (SSRs) are undoubtedly the most informative markers in genotyping, genetics and applied breeding applications. We generated 182,712 SSRs at the whole genome level, of which, 170,574 perfect SSRs were found; 16,252 perfect SSRs showed <em>in silico</em> polymorphisms across six genotypes suggesting their promising use in genetic conservation and tree improvement programmes. Genomic SSR markers developed in this study have high potential in advancing conservation and management of teak genetic resources. Phylogenetic studies confirmed the taxonomic position of the genus <em>Tectona</em> within the family Lamiaceae. Interestingly, estimation of divergence time inferred that the Miocene origin of the <em>Tectona</em> genus to be around 21.4508 million years ago.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="teak">teak</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SSRs">SSRs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogeny">phylogeny</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="divergence">divergence</a></div>                            <h2 scrollto-destination=120121806 id="120121806" class="section-title" >1. Introduction</h2>
<p>Teak (<em>Tectona grandis</em> L. f.; 2<em>n</em> = 2<em>x</em> = 36) belonging to the mint family Lamiaceae is one of the world’s highly valued tropical timber species that occurs naturally in India, Laos, Myanmar and Thailand.<a reveal-id="dsy013-B1" data-open="dsy013-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><sup>–</sup><a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> The timber is highly valued because of its extreme durability, strength, stability as well as resistance to pests, chemicals and water. Quinones and other extractives found abundant in the teakwood are responsible for its anti-termite and anti-fungal properties conferring the longevity of its timber. Thus, the wood is used in building ships, railway carriages, sleepers, construction, furniture, veneer and carving. Owing to its admirable timber qualities and aesthetical properties (<a reveal-id="dsy013-F1" data-open="dsy013-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), teak has been successfully established as pure plantations in India and elsewhere since 1850.<a reveal-id="dsy013-B5" data-open="dsy013-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The recent log export ban imposed by Myanmar has resulted steep rise in international prices of plantation grown teak from Latin America and Africa leading to expansion of plantation area. The estimated planted area of teak is about 4.25–6.89 million ha with over 1.7 million ha in India.<a reveal-id="dsy013-B5" data-open="dsy013-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Though the share of teak is &lt;2% of tropical round wood production,<a reveal-id="dsy013-B6" data-open="dsy013-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> its high value continuously attracts new planters. At the same time, natural populations are continuously diminishing due to illegal logging, anthropogenic pressures and climate change. A recent study on the effect of climate change in teak expresses the risks of biological invasion into teak habitats and recommends conservation of crucial teak growing areas and suitable management planning.<a reveal-id="dsy013-B7" data-open="dsy013-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Population structure of teak across natural and introduced locations reveal that the landraces in introduced locations have comparatively narrow genetic diversity,<a reveal-id="dsy013-B8" data-open="dsy013-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> thus demanding exploration of genetic diversity of natural provenances and their conservation. </p>                        <a id="120121808" scrollto-destination="120121808"></a>
<div id="dsy013-F1" data-id="dsy013-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f1.png?Expires=2147483647&Signature=E34clTKzjazKmb-pRhNBQweSisdjUjiyzIYuwBrX0lXtZlvvnedarTYqHThPiC6e1MDyEyYE8o-M7-CApxSYFZay9QFqjbnsBVAI8SGuaQZpySBYgcYAFWk7RbQ7SkCN1VliB5zxuJkgJ7kLbaFD2uH4QqK57H4QTl2U3StgFICk72vRMt69Sie9qFGZ8dKfYdTZxVuA8QuekDNrBRKi1tPd6HRl6~5R7c~8Xm3vNjOQWG1k5djKk34ZTyR9cozOM06UyaBMVn4kQOVrdwZkSsGLiuwXm22D6nDpR1jCmvlT2y-H1LOGp00aNOPvAJM66diBFkkvE2wKZ0uMM6AD2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cross section of teak wood showing major features (a) pith; (b) heart wood; (c) sap wood; (d) growth ring; and (e) medullary rays." path-from-xml="dsy013f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121808/dsy013f1.tif" path-from-xml="dsy013f1.tif" target="_blank">View large</a><a section="120121808" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f1.png?Expires=2147483647&Signature=M~sFFttxvGWH6IoUuOMPI133ioNT2gGZOMNjkJRBJ2M7q40KV5q9NHTUFpcBk6gERHbtI8e3ACYL~PhJfy~coDY0E0esYBiN4kAx~4CX8DCVkh0gy4ZikbrAVZek~SOD0jgx-2pQLjAi2Nrt~8Q92l13mqRc~S~7C8LYHfj9WoU-2B1pUwEQEQSuJA1~7qghwXTTZpY6psJD0ClitN4qpMURcBWu~uUtPdcNenelMJnmu7IkzCoUb6Y9yBMDHF~JQ-q2s1bO17TEo5Rhn3~TKUkzWsckr~EvN1zlfPJhpZlntMWf3yzkhcdp8LcG51J210izaBLls0MRgQuy-LBWFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121808&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cross section of teak wood showing major features (a) pith; (b) heart wood; (c) sap wood; (d) growth ring; and (e) medullary rays.</p></div></div><div content-id="dsy013-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f1.png?Expires=2147483647&Signature=E34clTKzjazKmb-pRhNBQweSisdjUjiyzIYuwBrX0lXtZlvvnedarTYqHThPiC6e1MDyEyYE8o-M7-CApxSYFZay9QFqjbnsBVAI8SGuaQZpySBYgcYAFWk7RbQ7SkCN1VliB5zxuJkgJ7kLbaFD2uH4QqK57H4QTl2U3StgFICk72vRMt69Sie9qFGZ8dKfYdTZxVuA8QuekDNrBRKi1tPd6HRl6~5R7c~8Xm3vNjOQWG1k5djKk34ZTyR9cozOM06UyaBMVn4kQOVrdwZkSsGLiuwXm22D6nDpR1jCmvlT2y-H1LOGp00aNOPvAJM66diBFkkvE2wKZ0uMM6AD2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cross section of teak wood showing major features (a) pith; (b) heart wood; (c) sap wood; (d) growth ring; and (e) medullary rays." path-from-xml="dsy013f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121808/dsy013f1.tif" path-from-xml="dsy013f1.tif" target="_blank">View large</a><a section="120121808" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f1.png?Expires=2147483647&Signature=M~sFFttxvGWH6IoUuOMPI133ioNT2gGZOMNjkJRBJ2M7q40KV5q9NHTUFpcBk6gERHbtI8e3ACYL~PhJfy~coDY0E0esYBiN4kAx~4CX8DCVkh0gy4ZikbrAVZek~SOD0jgx-2pQLjAi2Nrt~8Q92l13mqRc~S~7C8LYHfj9WoU-2B1pUwEQEQSuJA1~7qghwXTTZpY6psJD0ClitN4qpMURcBWu~uUtPdcNenelMJnmu7IkzCoUb6Y9yBMDHF~JQ-q2s1bO17TEo5Rhn3~TKUkzWsckr~EvN1zlfPJhpZlntMWf3yzkhcdp8LcG51J210izaBLls0MRgQuy-LBWFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121808&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cross section of teak wood showing major features (a) pith; (b) heart wood; (c) sap wood; (d) growth ring; and (e) medullary rays.</p></div></div><p>Teak has several intrinsic genetic qualities that allow its genetic improvement for timber production. Wide and discontinuous natural distribution across varying edaphic and climatic conditions in India offers enormous potential for capturing adaptive genetic variation for genetic improvement. As a first step in the genetic improvement programme at a global level, a series of seventy five international provenance trials co-ordinated by Danish International Development Agency were established during 1973–76 across sixteen countries. Evaluation of the provenances indicated wide variation for survival, growth rate, stem form, flowering, fruit yield and wood characteristics.<a reveal-id="dsy013-B9" data-open="dsy013-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy013-B10" data-open="dsy013-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The results of genetic improvement in teak showed an overall positive trend, however, possible existence of non-additive genetic control for economically important traits in seed progeny generated high genetic variability even within a family.<a reveal-id="dsy013-B11" data-open="dsy013-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsy013-B12" data-open="dsy013-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Further, it was suggested that selection of teak stem size can be carried out at the age of 3 years, wherein indirect selection on flowering age will improve forking height.<a reveal-id="dsy013-B13" data-open="dsy013-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Clonal propagation through budding, rooting of cuttings and <em>in vitro</em> propagation has facilitated tree improvement and deployment of superior performers for commercial cultivation towards increasing timber yield.<a reveal-id="dsy013-B14" data-open="dsy013-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Globally, although clonal seed orchards (CSOs) were established for production of quality seed stock, reproductive fitness and success of CSO was a chronic problem largely pivoted to asynchronous flowering.<a reveal-id="dsy013-B15" data-open="dsy013-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p><p>Economic importance, concerns for conservation of natural populations and increase in plantation area, demand understanding the genetic basis of the economic traits in teak. Hence, like many other forest plantation species (e.g. <em>Pinus, Populus</em> and <em>Eucalyptus</em>), genetic and genomic resources in teak needs to be comprehended. Genetic diversity in natural and introduced populations of teak has been assessed with markers such as random amplified polymorphic DNA,<a reveal-id="dsy013-B16" data-open="dsy013-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> amplified fragment length polymorphism<a reveal-id="dsy013-B17" data-open="dsy013-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsy013-B18" data-open="dsy013-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and simple sequence repeat markers (SSRs) or Microsatellites.<a reveal-id="dsy013-B19" data-open="dsy013-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsy013-B20" data-open="dsy013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> These studies generated information on population genetic structure of natural teak populations. Indian teak is genetically very distinct from Thai and Indonesian provenances and African landraces.<a reveal-id="dsy013-B17" data-open="dsy013-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsy013-B19" data-open="dsy013-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Knowledge generated in these studies is highly useful to implement conservation programmes in teak to improve sustainable management of teak forests.<a reveal-id="dsy013-B20" data-open="dsy013-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Recently, transcriptomes of secondary wood<a reveal-id="dsy013-B21" data-open="dsy013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and vegetative to flowering transition stage<a reveal-id="dsy013-B22" data-open="dsy013-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> were developed, leading to identification of genes involved in lignification, secondary metabolite production and flower formation. Although numerous studies focused on various life history traits exist in teak, comprehensive understanding on the complete genome information remains unexplored. Next generation sequencing-based whole genome sequencing (NGS-WGS) yields more information on genomic scans of polymorphism to precisely estimate various population genetic parameters including demographic history. In this context, to enrich genomic resources in teak, the present study reports whole genome of teak through Illumina Hiseq 2000 NGS platform followed by <em>de novo</em> contig assembly, gene annotation and subsequent discovery of SSR polymorphism.</p>                            <h2 scrollto-destination=120121811 id="120121811" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=120121812 id="120121812" class="section-title" >2.1. Plant material and genomic DNA sample preparation</h3>
<p>Open-pollinated seeds were collected from six dominant trees, one each from a provenance covering the entire latitudinal range of natural distribution of teak in India (<a reveal-id="dsy013-T1" data-open="dsy013-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Seedlings were raised with family identity and one vigorously growing seedling randomly chosen from each family was used in this study. Genomic DNA was extracted from fresh and young leaves using standard CTAB method<a reveal-id="dsy013-B23" data-open="dsy013-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and was purified using DNAeasy Plant Mini kit (Qiagen, USA). The quantity and quality of the genomic DNA were assessed using Nanodrop2000 (Thermo Fisher Scientific, USA), Qubit (Thermo Fisher Scientific, USA) and agarose gel electrophoresis. <div content-id="dsy013-T1" class="table-modal"><div class="table-wrap"><div id="dsy013-T1" data-id="dsy013-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Details of plant materials used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Name of the provenance </th><th>State </th><th>Latitude </th><th>Longitude </th><th>Altitude (m) </th><th>Rainfall (mm) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>Nilambur </td><td>Kerala </td><td>11° 17' N </td><td>76° 19' E </td><td>49 </td><td>2,600 </td></tr><tr><td>2 </td><td>AU </td><td>Arienkavu </td><td>Kerala </td><td>8° 96’ N </td><td>77° 14’ E </td><td>240 </td><td>2,600 </td></tr><tr><td>3 </td><td>WR </td><td>Walayar </td><td>Kerala </td><td>10° 52' N </td><td>76° 46' E </td><td>216 </td><td>1,500 </td></tr><tr><td>4 </td><td>DI </td><td>Dandeli </td><td>Karnataka </td><td>15° 07' N </td><td>74° 35' E </td><td>510 </td><td>2,200 </td></tr><tr><td>5 </td><td>HI </td><td>Hojai </td><td>Assam </td><td>26° 39’ N </td><td>92° 36’ E </td><td>69 </td><td>1,750 </td></tr><tr><td>6 </td><td>TP </td><td>Topslip </td><td>Tamilnadu </td><td>10° 26' N </td><td>76° 50' E </td><td>640 </td><td>1,350 </td></tr></tbody></table></div><div content-id="dsy013-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Name of the provenance </th><th>State </th><th>Latitude </th><th>Longitude </th><th>Altitude (m) </th><th>Rainfall (mm) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>Nilambur </td><td>Kerala </td><td>11° 17' N </td><td>76° 19' E </td><td>49 </td><td>2,600 </td></tr><tr><td>2 </td><td>AU </td><td>Arienkavu </td><td>Kerala </td><td>8° 96’ N </td><td>77° 14’ E </td><td>240 </td><td>2,600 </td></tr><tr><td>3 </td><td>WR </td><td>Walayar </td><td>Kerala </td><td>10° 52' N </td><td>76° 46' E </td><td>216 </td><td>1,500 </td></tr><tr><td>4 </td><td>DI </td><td>Dandeli </td><td>Karnataka </td><td>15° 07' N </td><td>74° 35' E </td><td>510 </td><td>2,200 </td></tr><tr><td>5 </td><td>HI </td><td>Hojai </td><td>Assam </td><td>26° 39’ N </td><td>92° 36’ E </td><td>69 </td><td>1,750 </td></tr><tr><td>6 </td><td>TP </td><td>Topslip </td><td>Tamilnadu </td><td>10° 26' N </td><td>76° 50' E </td><td>640 </td><td>1,350 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy013-T1" data-id="dsy013-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Details of plant materials used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Name of the provenance </th><th>State </th><th>Latitude </th><th>Longitude </th><th>Altitude (m) </th><th>Rainfall (mm) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>Nilambur </td><td>Kerala </td><td>11° 17' N </td><td>76° 19' E </td><td>49 </td><td>2,600 </td></tr><tr><td>2 </td><td>AU </td><td>Arienkavu </td><td>Kerala </td><td>8° 96’ N </td><td>77° 14’ E </td><td>240 </td><td>2,600 </td></tr><tr><td>3 </td><td>WR </td><td>Walayar </td><td>Kerala </td><td>10° 52' N </td><td>76° 46' E </td><td>216 </td><td>1,500 </td></tr><tr><td>4 </td><td>DI </td><td>Dandeli </td><td>Karnataka </td><td>15° 07' N </td><td>74° 35' E </td><td>510 </td><td>2,200 </td></tr><tr><td>5 </td><td>HI </td><td>Hojai </td><td>Assam </td><td>26° 39’ N </td><td>92° 36’ E </td><td>69 </td><td>1,750 </td></tr><tr><td>6 </td><td>TP </td><td>Topslip </td><td>Tamilnadu </td><td>10° 26' N </td><td>76° 50' E </td><td>640 </td><td>1,350 </td></tr></tbody></table></div><div content-id="dsy013-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Name of the provenance </th><th>State </th><th>Latitude </th><th>Longitude </th><th>Altitude (m) </th><th>Rainfall (mm) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>Nilambur </td><td>Kerala </td><td>11° 17' N </td><td>76° 19' E </td><td>49 </td><td>2,600 </td></tr><tr><td>2 </td><td>AU </td><td>Arienkavu </td><td>Kerala </td><td>8° 96’ N </td><td>77° 14’ E </td><td>240 </td><td>2,600 </td></tr><tr><td>3 </td><td>WR </td><td>Walayar </td><td>Kerala </td><td>10° 52' N </td><td>76° 46' E </td><td>216 </td><td>1,500 </td></tr><tr><td>4 </td><td>DI </td><td>Dandeli </td><td>Karnataka </td><td>15° 07' N </td><td>74° 35' E </td><td>510 </td><td>2,200 </td></tr><tr><td>5 </td><td>HI </td><td>Hojai </td><td>Assam </td><td>26° 39’ N </td><td>92° 36’ E </td><td>69 </td><td>1,750 </td></tr><tr><td>6 </td><td>TP </td><td>Topslip </td><td>Tamilnadu </td><td>10° 26' N </td><td>76° 50' E </td><td>640 </td><td>1,350 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=120121814 id="120121814" class="section-title" >2.2. Library preparation and genome sequencing</h3>
<p>WGS was performed using Illumina HiSeq 2000 platform and Oxford Nanopore Technologies MinION device by the Genotypic Technology, Bengaluru, India in accordance to standard protocols. Accession 2 was selected for the generation of high quality reference genome assembly. Accessions 1, 3, 4, 5 and 6 were subjected to low coverage genome sequencing to identify polymorphic SSRs. In the case of accession 2, one paired end (PE) (150 bp × 2) library of size 300–700 bp, two mate pair (MP) libraries (2–4 and 4–6 kb fragments) and one nanopore library with genomic DNA (2 μg) were prepared for sequencing. In Illumina HiSeq 2000 platform, one lane of the flow cell was used for each sequencing library. Nanopore sequencing was performed using R9.4 flow cells on a MinION Mk 1B device (Oxford Nanopore) with the MinKNOW software (versions 1.0.5–1.5.12) and base calling was performed using Albacore 1.1.0 (Oxford Nanopore). Template reads were exported as FASTA using poretools version 0.6. In the case of other five accessions (1, 3, 4, 5 and 6) one PE library for each with the size of 300–700 bp was sequenced at ∼15× coverage through Illumina Hiseq 2000 platform. The sequence data is uploaded in genome database of GenBank (Project id: <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/PRJNA374940" target="_blank">PRJNA374940</a>). The assembled genome, protein sequences and its annotation, GO and pathway information are available in the web link <a class="link link-uri" href="https://biit.cs.ut.ee/supplementary/WGSteak/" target="_blank">https://biit.cs.ut.ee/supplementary/WGSteak/</a>.</p>                            <h3 scrollto-destination=120121816 id="120121816" class="section-title" >2.3. <em>De novo</em> genome assembly</h3>
<p>The Illumina PE raw reads were filtered using FastQC and the raw reads were processed by in-house (Genotypic Technology, Bangaluru, India) ABLT script for low-quality bases and adapters removal. The MP reads were processed using Platanus<a reveal-id="dsy013-B24" data-open="dsy013-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> internal trimmer for adapters and low-quality regions towards 3’-end. The processed PE reads along with MP and nanopore reads were used for contig generation using MaSuRCA v 3.2.2 <em>de novo</em> assembler.<a reveal-id="dsy013-B25" data-open="dsy013-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> To assemble the genome following command was used in MaSuRCA assembler: GRAPH_KMER_SIZE = auto, LIMIT_JUMP_COVERAGE = 300, JF_SIZE = 38000000000, DO_HOMOPOLYMER_TRIM = 1. Scaffolding of the assembled contigs was performed using SSPACE v 2.0.5<a reveal-id="dsy013-B26" data-open="dsy013-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> with processed PE and MP reads followed by gap filling using Gap Closer v 1.12.<a reveal-id="dsy013-B27" data-open="dsy013-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The genome size was estimated automatically during read computing stage which utilized both the Illuimna and Nanopore reads. Similarly, the low depth Illumina reads generated for five accessions of teak were assembled using accession 2 as reference. The sequenced data was uploaded to the Genome database of GenBank (Project id: <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/PRJNA421422" target="_blank">PRJNA421422</a>).</p>                            <h3 scrollto-destination=120121818 id="120121818" class="section-title" >2.4. Genome annotation</h3>
<p>For a functional overview of draft genome, assembled scaffolds were converted to FASTA formatted sequences, hard masked by RepeatMasker tool (RepeatMasker Open-3.0; <a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">www.repeatmasker.org</a> (10 November 2017, date last accessed)). Repeats of <em>Arabidopsis thaliana</em> were used as reference for genome masking. Gene prediction was carried out using Augustus 3.0.2<a reveal-id="dsy013-B28" data-open="dsy013-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> programme and predicted proteins were searched against the Uniprot non-redundant plant protein database (Taxonomy = <em>Viridiplante</em>) with BlastX algorithm with an <em>e</em>-value (<em>e</em>-10) for gene ontology and annotation. Pathway annotation was performed by mapping the sequences obtained from Blast2GO to the contents of the KEGG Automatic Annotation Server (<a class="link link-uri" href="http://www.genome.jp/kegg/kaas/" target="_blank">http://www.genome.jp/kegg/kaas/</a> (10 November 2017, date last accessed)). List of eudicot plants used as reference organism for pathway identification in KASS server is given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s1-s3_s7_s9-s10.doc?Expires=2147483647&Signature=eTnP40iJtj5DxIBCwFiS3ZARptTtP-uv4EyApY2QfXgyVe9eAAfNsEIQfnSidLxB6kPKdZFUo3RvZxN363eis1NiSZ3yXCiy41hYX1nSnVgJtOjuNrP4XP6vKSG4-4HG6YiJ~xnfiRGn-y6QJuX2jXisawegWAjfWWrKA80ww~mKby~8Y6QyovAheKJvTo4p9fNGgFED2sx1IU5D~sp25AE6ijNy81cSsfuNSY~05I7cflkbvQliBC2uMFpWMyn139znnYjtPUV2riu-9G5RSH7vqWZfB0j-UOYjxc8QN2JuJYNTdMkmGqZnhwRaikXyuRQHVEjTaMAuu9vNfK8z-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h3 scrollto-destination=120121820 id="120121820" class="section-title" >2.5. Identification of SSRs and detection of polymorphism</h3>
<p>FASTA formatted scaffolds of teak were analysed for frequency and density of SSRs using the Perl script MIcroSAtelitte (MISA; <a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/" target="_blank">http://pgrc.ipk-gatersleben.de/misa/</a> (20 November 2017, date last accessed)). Initially SSRs of 1–6 nucleotides motifs were identified with the minimum repeat unit defined as 10 for mononucleotides, 6 for dinucleotides, 5 for trinucleotides, 4 for tetra-nucleotides, and 3 each for penta and hexa-nucleotides. Compound SSRs were defined as ≥2 SSRs interrupted by ≤100 bases. To design primers flanking the microsatellite loci, two interface Perl script modules were used to interchange data between MISA and the primer designing software Primer 3. The SSR containing scaffolds were used to design the primers with the following parameters. Primer length 18–25 bp, with 20 bp as optimum; primer GC content = 30–0%, with the optimum value of 50%; primer Tm 57–63°C, and product size ranged 100–300 bp.</p><p>Polymorphic SSRs across five samples were analysed using accession 2 as reference. Polymorphic SSR retrieval tool (PSR)<a reveal-id="dsy013-B29" data-open="dsy013-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> comprising two modules (PSR_read_retrieval and PSR_poly_finder) were deployed to detect SSR length polymorphisms of perfect repeats from NGS data. It is to be noted that PSR tool identifies length polymorphisms in perfect microsatellites only. Also, it filters out all the reads that match twice or more on the reference sequence as well as non-overlapping paired-end reads that are aligned on the same microsatellite locus. Minimum number of supporting reads and read depth was fixed to 10 and 30, respectively. This process detects polymorphic SSRs based on the availability of left and right border unique sequences based on the complete coverage of the SSR region in the sequence data.</p><p>The polymorphic SSR data generated from PSR software was validated through gel electrophoresis. Totally 10 SSRs representing di, tri and tetra-nucleotide motifs were randomly chosen and amplified with 10 randomly selected teak trees from Topslip (Latitude: 10°29’09.5’N; Longitude: 75°50’03.8’E Altitude: 736m) provenance (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s1-s3_s7_s9-s10.doc?Expires=2147483647&Signature=eTnP40iJtj5DxIBCwFiS3ZARptTtP-uv4EyApY2QfXgyVe9eAAfNsEIQfnSidLxB6kPKdZFUo3RvZxN363eis1NiSZ3yXCiy41hYX1nSnVgJtOjuNrP4XP6vKSG4-4HG6YiJ~xnfiRGn-y6QJuX2jXisawegWAjfWWrKA80ww~mKby~8Y6QyovAheKJvTo4p9fNGgFED2sx1IU5D~sp25AE6ijNy81cSsfuNSY~05I7cflkbvQliBC2uMFpWMyn139znnYjtPUV2riu-9G5RSH7vqWZfB0j-UOYjxc8QN2JuJYNTdMkmGqZnhwRaikXyuRQHVEjTaMAuu9vNfK8z-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). PCR amplification were set to 10 µl volume containing 5 ng of template DNA, 2.5 µM MgCl<sub>2</sub>, 2.5 µl 10 × PCR buffer, 0.5 µM of primer, 0.5 U Taq DNA polymerase, and 2.5 mM of dNTPs. The PCR cycling profile was programmed to 94°C for 5 min, 35 cycles at 94°C for 45 s, 58°C (annealing temperature varied for each locus) for 45 s (<a reveal-id="dsy013-T2" data-open="dsy013-T2" class="link link-reveal link-table xref-fig">Table 2</a>), 72°C for 45 s, and a final extension at 72°C for 10 min. Banding pattern was visualised by silver staining after denaturing polyacrylamide gel (5%) electrophoresis. <div content-id="dsy013-T2" class="table-modal"><div class="table-wrap"><div id="dsy013-T2" data-id="dsy013-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Details on 10 SSR markers used for amplification of teak germplasm</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SSR code </th><th>SSR Motif </th><th>Primers (5’-3’) </th><th>Annealing temperature (<sup>o</sup>C) </th><th>Number of alleles </th><th>Product size (bp) </th></tr></thead><tbody><tr><td>IFT83 </td><td>(AG)<sub>21</sub> </td><td><span class="monospace">F5’AATTGGCATAAAGCGTGCTACTR5’CGCACGTCCTATTTTGGTTTAT</span> </td><td>54 </td><td>5 </td><td>312-393 </td></tr><tr><td>IFT821 </td><td>(AC)<sub>24</sub> </td><td><span class="monospace">F5’CCCCAATTATGTCAACCGACT R5’GGCATTATCTAAGATCGCAAGG</span> </td><td>53.9 </td><td>3 </td><td>331-350 </td></tr><tr><td>IFT63 </td><td>(ATG)<sub>12</sub> </td><td><span class="monospace">F5’CCCAAAGCGAATAATATCCTAC R5’CATGACTTGTTCGATGGGCTAAT</span> </td><td>54 </td><td>3 </td><td>250-275 </td></tr><tr><td>IFT168 </td><td>(TCT)<sub>12</sub> </td><td><span class="monospace">F5’ATCTTCAGCAGAGGAGGCTATG R5’GTGCCCTTTTCTCTCTTCTTCA</span> </td><td>55 </td><td>4 </td><td>287-306 </td></tr><tr><td>IFT479b </td><td>(GGA)<sub>11</sub> </td><td><span class="monospace">F5’GTGAAGATTCGGGTATGGAGAG R5’TACTCCCAGATTTCCCAATCAC</span> </td><td>55 </td><td>3 </td><td>331-343 </td></tr><tr><td>IFT382 </td><td>(TATG)<sub>7</sub> </td><td><span class="monospace">F5’TACTCATCACTGTCCCCAGTTG R5’GAACGGGAATCTAGAGTTGTGG</span> </td><td>56 </td><td>3 </td><td>337-350 </td></tr><tr><td>IFT 28 </td><td>(AAAG)<sub>6</sub> </td><td><span class="monospace">F5’CAGCCTCTGCATGTCAAATAAA R5’TTAGAGCTGGATATGCCATTGA</span> </td><td>53.6 </td><td>3 </td><td>381-393 </td></tr><tr><td>IFT14 </td><td>(TTCT)<sub>9</sub> </td><td><span class="monospace">F5’TGTGGTATTGGACCATCTGAAA R5’GGTAACCCACCAACAAATATGC</span> </td><td>54 </td><td>5 </td><td>265-278 </td></tr><tr><td>IFT3 </td><td>(GAAAG)<sub>5</sub> </td><td><span class="monospace">F5’TTCCACCTACTGGTTAAGGAAC R5’ATGGCTTACCAATTTACCAAACC</span> </td><td>54 </td><td>1 </td><td>330 </td></tr><tr><td>IFT777 </td><td>(TCAGG)<sub>6</sub> </td><td><span class="monospace">F5’TACTAACCGGAAGAGGGAAACC R5’TGTCGCTATGGACAGTTCATCT</span> </td><td>56 </td><td>4 </td><td>312-343 </td></tr></tbody></table></div><div content-id="dsy013-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>SSR code </th><th>SSR Motif </th><th>Primers (5’-3’) </th><th>Annealing temperature (<sup>o</sup>C) </th><th>Number of alleles </th><th>Product size (bp) </th></tr></thead><tbody><tr><td>IFT83 </td><td>(AG)<sub>21</sub> </td><td><span class="monospace">F5’AATTGGCATAAAGCGTGCTACTR5’CGCACGTCCTATTTTGGTTTAT</span> </td><td>54 </td><td>5 </td><td>312-393 </td></tr><tr><td>IFT821 </td><td>(AC)<sub>24</sub> </td><td><span class="monospace">F5’CCCCAATTATGTCAACCGACT R5’GGCATTATCTAAGATCGCAAGG</span> </td><td>53.9 </td><td>3 </td><td>331-350 </td></tr><tr><td>IFT63 </td><td>(ATG)<sub>12</sub> </td><td><span class="monospace">F5’CCCAAAGCGAATAATATCCTAC R5’CATGACTTGTTCGATGGGCTAAT</span> </td><td>54 </td><td>3 </td><td>250-275 </td></tr><tr><td>IFT168 </td><td>(TCT)<sub>12</sub> </td><td><span class="monospace">F5’ATCTTCAGCAGAGGAGGCTATG R5’GTGCCCTTTTCTCTCTTCTTCA</span> </td><td>55 </td><td>4 </td><td>287-306 </td></tr><tr><td>IFT479b </td><td>(GGA)<sub>11</sub> </td><td><span class="monospace">F5’GTGAAGATTCGGGTATGGAGAG R5’TACTCCCAGATTTCCCAATCAC</span> </td><td>55 </td><td>3 </td><td>331-343 </td></tr><tr><td>IFT382 </td><td>(TATG)<sub>7</sub> </td><td><span class="monospace">F5’TACTCATCACTGTCCCCAGTTG R5’GAACGGGAATCTAGAGTTGTGG</span> </td><td>56 </td><td>3 </td><td>337-350 </td></tr><tr><td>IFT 28 </td><td>(AAAG)<sub>6</sub> </td><td><span class="monospace">F5’CAGCCTCTGCATGTCAAATAAA R5’TTAGAGCTGGATATGCCATTGA</span> </td><td>53.6 </td><td>3 </td><td>381-393 </td></tr><tr><td>IFT14 </td><td>(TTCT)<sub>9</sub> </td><td><span class="monospace">F5’TGTGGTATTGGACCATCTGAAA R5’GGTAACCCACCAACAAATATGC</span> </td><td>54 </td><td>5 </td><td>265-278 </td></tr><tr><td>IFT3 </td><td>(GAAAG)<sub>5</sub> </td><td><span class="monospace">F5’TTCCACCTACTGGTTAAGGAAC R5’ATGGCTTACCAATTTACCAAACC</span> </td><td>54 </td><td>1 </td><td>330 </td></tr><tr><td>IFT777 </td><td>(TCAGG)<sub>6</sub> </td><td><span class="monospace">F5’TACTAACCGGAAGAGGGAAACC R5’TGTCGCTATGGACAGTTCATCT</span> </td><td>56 </td><td>4 </td><td>312-343 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy013-T2" data-id="dsy013-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Details on 10 SSR markers used for amplification of teak germplasm</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SSR code </th><th>SSR Motif </th><th>Primers (5’-3’) </th><th>Annealing temperature (<sup>o</sup>C) </th><th>Number of alleles </th><th>Product size (bp) </th></tr></thead><tbody><tr><td>IFT83 </td><td>(AG)<sub>21</sub> </td><td><span class="monospace">F5’AATTGGCATAAAGCGTGCTACTR5’CGCACGTCCTATTTTGGTTTAT</span> </td><td>54 </td><td>5 </td><td>312-393 </td></tr><tr><td>IFT821 </td><td>(AC)<sub>24</sub> </td><td><span class="monospace">F5’CCCCAATTATGTCAACCGACT R5’GGCATTATCTAAGATCGCAAGG</span> </td><td>53.9 </td><td>3 </td><td>331-350 </td></tr><tr><td>IFT63 </td><td>(ATG)<sub>12</sub> </td><td><span class="monospace">F5’CCCAAAGCGAATAATATCCTAC R5’CATGACTTGTTCGATGGGCTAAT</span> </td><td>54 </td><td>3 </td><td>250-275 </td></tr><tr><td>IFT168 </td><td>(TCT)<sub>12</sub> </td><td><span class="monospace">F5’ATCTTCAGCAGAGGAGGCTATG R5’GTGCCCTTTTCTCTCTTCTTCA</span> </td><td>55 </td><td>4 </td><td>287-306 </td></tr><tr><td>IFT479b </td><td>(GGA)<sub>11</sub> </td><td><span class="monospace">F5’GTGAAGATTCGGGTATGGAGAG R5’TACTCCCAGATTTCCCAATCAC</span> </td><td>55 </td><td>3 </td><td>331-343 </td></tr><tr><td>IFT382 </td><td>(TATG)<sub>7</sub> </td><td><span class="monospace">F5’TACTCATCACTGTCCCCAGTTG R5’GAACGGGAATCTAGAGTTGTGG</span> </td><td>56 </td><td>3 </td><td>337-350 </td></tr><tr><td>IFT 28 </td><td>(AAAG)<sub>6</sub> </td><td><span class="monospace">F5’CAGCCTCTGCATGTCAAATAAA R5’TTAGAGCTGGATATGCCATTGA</span> </td><td>53.6 </td><td>3 </td><td>381-393 </td></tr><tr><td>IFT14 </td><td>(TTCT)<sub>9</sub> </td><td><span class="monospace">F5’TGTGGTATTGGACCATCTGAAA R5’GGTAACCCACCAACAAATATGC</span> </td><td>54 </td><td>5 </td><td>265-278 </td></tr><tr><td>IFT3 </td><td>(GAAAG)<sub>5</sub> </td><td><span class="monospace">F5’TTCCACCTACTGGTTAAGGAAC R5’ATGGCTTACCAATTTACCAAACC</span> </td><td>54 </td><td>1 </td><td>330 </td></tr><tr><td>IFT777 </td><td>(TCAGG)<sub>6</sub> </td><td><span class="monospace">F5’TACTAACCGGAAGAGGGAAACC R5’TGTCGCTATGGACAGTTCATCT</span> </td><td>56 </td><td>4 </td><td>312-343 </td></tr></tbody></table></div><div content-id="dsy013-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>SSR code </th><th>SSR Motif </th><th>Primers (5’-3’) </th><th>Annealing temperature (<sup>o</sup>C) </th><th>Number of alleles </th><th>Product size (bp) </th></tr></thead><tbody><tr><td>IFT83 </td><td>(AG)<sub>21</sub> </td><td><span class="monospace">F5’AATTGGCATAAAGCGTGCTACTR5’CGCACGTCCTATTTTGGTTTAT</span> </td><td>54 </td><td>5 </td><td>312-393 </td></tr><tr><td>IFT821 </td><td>(AC)<sub>24</sub> </td><td><span class="monospace">F5’CCCCAATTATGTCAACCGACT R5’GGCATTATCTAAGATCGCAAGG</span> </td><td>53.9 </td><td>3 </td><td>331-350 </td></tr><tr><td>IFT63 </td><td>(ATG)<sub>12</sub> </td><td><span class="monospace">F5’CCCAAAGCGAATAATATCCTAC R5’CATGACTTGTTCGATGGGCTAAT</span> </td><td>54 </td><td>3 </td><td>250-275 </td></tr><tr><td>IFT168 </td><td>(TCT)<sub>12</sub> </td><td><span class="monospace">F5’ATCTTCAGCAGAGGAGGCTATG R5’GTGCCCTTTTCTCTCTTCTTCA</span> </td><td>55 </td><td>4 </td><td>287-306 </td></tr><tr><td>IFT479b </td><td>(GGA)<sub>11</sub> </td><td><span class="monospace">F5’GTGAAGATTCGGGTATGGAGAG R5’TACTCCCAGATTTCCCAATCAC</span> </td><td>55 </td><td>3 </td><td>331-343 </td></tr><tr><td>IFT382 </td><td>(TATG)<sub>7</sub> </td><td><span class="monospace">F5’TACTCATCACTGTCCCCAGTTG R5’GAACGGGAATCTAGAGTTGTGG</span> </td><td>56 </td><td>3 </td><td>337-350 </td></tr><tr><td>IFT 28 </td><td>(AAAG)<sub>6</sub> </td><td><span class="monospace">F5’CAGCCTCTGCATGTCAAATAAA R5’TTAGAGCTGGATATGCCATTGA</span> </td><td>53.6 </td><td>3 </td><td>381-393 </td></tr><tr><td>IFT14 </td><td>(TTCT)<sub>9</sub> </td><td><span class="monospace">F5’TGTGGTATTGGACCATCTGAAA R5’GGTAACCCACCAACAAATATGC</span> </td><td>54 </td><td>5 </td><td>265-278 </td></tr><tr><td>IFT3 </td><td>(GAAAG)<sub>5</sub> </td><td><span class="monospace">F5’TTCCACCTACTGGTTAAGGAAC R5’ATGGCTTACCAATTTACCAAACC</span> </td><td>54 </td><td>1 </td><td>330 </td></tr><tr><td>IFT777 </td><td>(TCAGG)<sub>6</sub> </td><td><span class="monospace">F5’TACTAACCGGAAGAGGGAAACC R5’TGTCGCTATGGACAGTTCATCT</span> </td><td>56 </td><td>4 </td><td>312-343 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=120121824 id="120121824" class="section-title" >2.6. Phylogenetic tree construction</h3>
<p>Plastid gene sequences of <em>psb</em>B, <em>rbc</em>L, <em>psa</em>A and <em>ycf</em>2 from 16 species of the family Lamiaceae were accessed from the Genbank public domain (<a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/https://www.ncbi.nlm.nih.gov/genbank" target="_blank">https://www.ncbi.nlm.nih.gov/genbank</a> (13 February 2018, date last accessed)) for constructing the phylogenetic tree with <em>Olea europaea</em> (Oleaceae) as the out group (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s1-s3_s7_s9-s10.doc?Expires=2147483647&Signature=eTnP40iJtj5DxIBCwFiS3ZARptTtP-uv4EyApY2QfXgyVe9eAAfNsEIQfnSidLxB6kPKdZFUo3RvZxN363eis1NiSZ3yXCiy41hYX1nSnVgJtOjuNrP4XP6vKSG4-4HG6YiJ~xnfiRGn-y6QJuX2jXisawegWAjfWWrKA80ww~mKby~8Y6QyovAheKJvTo4p9fNGgFED2sx1IU5D~sp25AE6ijNy81cSsfuNSY~05I7cflkbvQliBC2uMFpWMyn139znnYjtPUV2riu-9G5RSH7vqWZfB0j-UOYjxc8QN2JuJYNTdMkmGqZnhwRaikXyuRQHVEjTaMAuu9vNfK8z-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Sequences were aligned using multiple sequence alignment tool implemented in ClustalX ver. 2.0.<a reveal-id="dsy013-B30" data-open="dsy013-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The sequences were manually refined in BioEdit ver. 7.0.9.<a reveal-id="dsy013-B31" data-open="dsy013-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Phylogenetic analyses were performed for concatenated sequences of plastid gene regions. JModeltest ver. 2.1.7<a reveal-id="dsy013-B32" data-open="dsy013-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> was used to choose the appropriate model of sequence evolution according to the Akaike information criterion (AIC).<a reveal-id="dsy013-B33" data-open="dsy013-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Bayesian interference analysis was performed in MrBayes ver. 3.2.6.<a reveal-id="dsy013-B34" data-open="dsy013-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The Markov chain Monte Carlo algorithm was run for ten million generations, over four chains each, sampled every 1000 generations. The estimated sample size was checked using Tracer ver. 1.4 (<a class="link link-uri" href="http://beast.bio.ed.ac.uk/Tracer" target="_blank">http://beast.bio.ed.ac.uk/Tracer</a> (22 February 2018, date last accessed)). The first 25% of the sampled trees was discarded as burn-in. The phylogenetic tree out of MrBayes ver.3.2.6 was visualized in FigTree ver.1.4.2 (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/" target="_blank">http://tree.bio.ed.ac.uk/software/figtree/</a> (22 February 2018, date last accessed)).</p>                            <h3 scrollto-destination=120121826 id="120121826" class="section-title" >2.7. Divergence time estimation</h3>
<p>The estimation of divergence time was performed using Bayesian approach implemented in BEAST ver. 2.4.4 programme.<a reveal-id="dsy013-B35" data-open="dsy013-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Bayesian approach was deployed to estimate divergence time of the genus <em>Tectona</em> with respect to the other subfamilies and genera within the family Lamiaceae. The HKY model was used based on the result of AIC from JModeltest under an uncorrelated lognormal relaxed clock model. Yule speciation model was used as tree prior. Two calibration points (57.6 million years ago, Mya) for Nepetoideae and 23.9 Mya for Lamioideae) were used based on the previous reports to determine specific nodes prior and lognormal distributions.<a reveal-id="dsy013-B36" data-open="dsy013-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Markov Chain was run for 10<sup>9</sup> generations, while every 1,000 generations were sampled. The chronograms shown were calculated using the median clade credibility tree plus 95% confidence intervals. Results were analysed using Tracer ver. 1.4 to assess the convergence statistics of the sequences. The effective sample sizes for all parameters and the tree files from the four runs of BEAST were combined using LogCombiner ver. 2.4.4.<a reveal-id="dsy013-B35" data-open="dsy013-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Twenty percent of the trees were removed as burn-in and the resulting trees were summarized with Treeannotator ver. 2.4.4.<a reveal-id="dsy013-B37" data-open="dsy013-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Finally, the summarized single trees were visualized in FigTree ver. 1.4.2.</p>                            <h2 scrollto-destination=120121828 id="120121828" class="section-title" >3. Results and discussion</h2>
<p>Several members of the order Lamiales are well known for their secondary metabolites of medicinal value<a reveal-id="dsy013-B38" data-open="dsy013-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and 17 species of the family were sequenced for the whole genome (<a class="link link-uri" href="https://www.ncbi.nlm.nih.gov/genome/? term=Lamiales" target="_blank">https://www.ncbi.nlm.nih.gov/genome/? term=Lamiales</a> (13 February 2018, date last accessed)) due to their economic importance. However, the woody species teak, one of the world’s premier timber species cultivated across 65 countries has only 3,269 nucleotide accessions including 6 ESTs available so far in the public domain (7 November 2017, date last accessed). Owing to the commercial importance, this study was undertaken to unravel the genome structure to facilitate conservation and improvement of teak genetic resources. The only available genomic resource in teak is the <em>de novo</em> assembly with transcriptome in 12- and 60-year-old trees to generate unigenes related to lignin biosynthesis.<a reveal-id="dsy013-B21" data-open="dsy013-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> All the earlier gene assemblies were based on short-read technology. This work on draft genome assembly using long read technology like MP and MinION nanopore sequencing would provide an excellent resource to comprehend genome structure, genetic variation and conservation.</p>                            <h3 scrollto-destination=120121830 id="120121830" class="section-title" >3.1. <em>De novo</em> assembly and characterization of genomic sequences</h3>
<p>The study has generated a high quality reference genome for the accession 2 which was assembled from PE, MP and nanopore library sequences. The numbers of raw and processed reads are summarized in <a reveal-id="dsy013-T3" data-open="dsy013-T3" class="link link-reveal link-table xref-fig">Table 3</a>. High depth (109×) PE sequencing provided a global overview of teak genome with over 137.2 million PE sequences. After suitable filtration, a total of 128.2 million sequences, representing 93.43% of the raw reads were obtained. Two different MP libraries with 2–4 and 4–6 Kb generated raw reads of about 7,681 (20×) and 5,819 Mbp (15×) of which processed read length was 2,408 and 1,898 Mbp, respectively. <div content-id="dsy013-T3" class="table-modal"><div class="table-wrap"><div id="dsy013-T3" data-id="dsy013-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Raw data statistics of Illumina PE, MP and Nanopore reads of teak genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Platform </th><th>Chemistry </th><th>Number of raw reads </th><th>Total bases of raw reads (bp) </th><th>No of processed reads (bp) </th><th>Total bases after processing (bp) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>Illumina HiSeq </td><td>PE (150× 2) </td><td>137,231,716 </td><td>41,443,978,232 </td><td>128,115,515 </td><td>37,507,019,850 </td><td>109 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B2" data-open="dsy013-B2" class="link link-ref link-reveal xref-bibr">2</a>-4 Kb) </td><td>MP (150× 2) </td><td>25,436,869 </td><td>7,681,934,438 </td><td>10,776,772 </td><td>2,408,197,618 </td><td>20 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr">4</a>-6Kb) </td><td>MP (150× 2) </td><td>19,268,378 </td><td>5,819,050,156 </td><td>8,470,072 </td><td>1,898,380,817 </td><td>15 </td></tr><tr><td>Nanopore </td><td>Long read (5-1,345,484) </td><td>782,591 </td><td>2,685,280,348 </td><td>782,591 </td><td>2,685,280,348 </td><td>7.06 </td></tr><tr><td colspan="6">Total coverage </td><td>151× </td></tr></tbody></table></div><div content-id="dsy013-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Platform </th><th>Chemistry </th><th>Number of raw reads </th><th>Total bases of raw reads (bp) </th><th>No of processed reads (bp) </th><th>Total bases after processing (bp) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>Illumina HiSeq </td><td>PE (150× 2) </td><td>137,231,716 </td><td>41,443,978,232 </td><td>128,115,515 </td><td>37,507,019,850 </td><td>109 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B2" data-open="dsy013-B2" class="link link-ref link-reveal xref-bibr">2</a>-4 Kb) </td><td>MP (150× 2) </td><td>25,436,869 </td><td>7,681,934,438 </td><td>10,776,772 </td><td>2,408,197,618 </td><td>20 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr">4</a>-6Kb) </td><td>MP (150× 2) </td><td>19,268,378 </td><td>5,819,050,156 </td><td>8,470,072 </td><td>1,898,380,817 </td><td>15 </td></tr><tr><td>Nanopore </td><td>Long read (5-1,345,484) </td><td>782,591 </td><td>2,685,280,348 </td><td>782,591 </td><td>2,685,280,348 </td><td>7.06 </td></tr><tr><td colspan="6">Total coverage </td><td>151× </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy013-T3" data-id="dsy013-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Raw data statistics of Illumina PE, MP and Nanopore reads of teak genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Platform </th><th>Chemistry </th><th>Number of raw reads </th><th>Total bases of raw reads (bp) </th><th>No of processed reads (bp) </th><th>Total bases after processing (bp) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>Illumina HiSeq </td><td>PE (150× 2) </td><td>137,231,716 </td><td>41,443,978,232 </td><td>128,115,515 </td><td>37,507,019,850 </td><td>109 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B2" data-open="dsy013-B2" class="link link-ref link-reveal xref-bibr">2</a>-4 Kb) </td><td>MP (150× 2) </td><td>25,436,869 </td><td>7,681,934,438 </td><td>10,776,772 </td><td>2,408,197,618 </td><td>20 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr">4</a>-6Kb) </td><td>MP (150× 2) </td><td>19,268,378 </td><td>5,819,050,156 </td><td>8,470,072 </td><td>1,898,380,817 </td><td>15 </td></tr><tr><td>Nanopore </td><td>Long read (5-1,345,484) </td><td>782,591 </td><td>2,685,280,348 </td><td>782,591 </td><td>2,685,280,348 </td><td>7.06 </td></tr><tr><td colspan="6">Total coverage </td><td>151× </td></tr></tbody></table></div><div content-id="dsy013-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Platform </th><th>Chemistry </th><th>Number of raw reads </th><th>Total bases of raw reads (bp) </th><th>No of processed reads (bp) </th><th>Total bases after processing (bp) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>Illumina HiSeq </td><td>PE (150× 2) </td><td>137,231,716 </td><td>41,443,978,232 </td><td>128,115,515 </td><td>37,507,019,850 </td><td>109 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B2" data-open="dsy013-B2" class="link link-ref link-reveal xref-bibr">2</a>-4 Kb) </td><td>MP (150× 2) </td><td>25,436,869 </td><td>7,681,934,438 </td><td>10,776,772 </td><td>2,408,197,618 </td><td>20 </td></tr><tr><td>Illumina HiSeq (<a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr">4</a>-6Kb) </td><td>MP (150× 2) </td><td>19,268,378 </td><td>5,819,050,156 </td><td>8,470,072 </td><td>1,898,380,817 </td><td>15 </td></tr><tr><td>Nanopore </td><td>Long read (5-1,345,484) </td><td>782,591 </td><td>2,685,280,348 </td><td>782,591 </td><td>2,685,280,348 </td><td>7.06 </td></tr><tr><td colspan="6">Total coverage </td><td>151× </td></tr></tbody></table></div></div></p><p>Nanopore library generated 782,591 reads with read length of 2,685, 280,348 bp, corresponding to 7.06× coverage of the genome. Longest read was 1,345,484 bp and the average read length was 3,431 bp. All these sequences with 151× coverage were included in whole genome assembly. Application of nanopore sequencing was challenging mainly due to large size and repetitive nature of the plant genome. However, high yield of nanopore using 9.4 chemistry has resolved this issue. Genomic applications to genetic resource conservation and breeding in forest trees is expected to harness much benefits due to the long read sequencing technologies.<a reveal-id="dsy013-B39" data-open="dsy013-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The processed PE reads along with MP and nanopore reads were assembled using MaSuRCA <em>de novo</em> assembler. It used all the Illumina and Nanopore reads choosing a kmer size of 105 and estimated the teak genome size of 371,016,305 bp. Genome size of teak was estimated 465 Mbp through flow cytometry (1C = 0.48 pg).<a reveal-id="dsy013-B40" data-open="dsy013-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Typically, data obtained from the WGS approach in plants with genome size exceeding a few hundred mega bases are difficult to assemble satisfactorily due to highly repetitive DNA.<a reveal-id="dsy013-B41" data-open="dsy013-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> The final draft genome used for downstream analysis had 2,993 filtered contigs (&gt;1 Kbp) with maximum, minimum and average contig length of 1,718,606, 1,100 and 106,098 bp, respectively (<a reveal-id="dsy013-T4" data-open="dsy013-T4" class="link link-reveal link-table xref-fig">Table 4</a>). The N50 value of the assembly was 357,576 bp. Comparative WGS analysis including estimated genome size, assembly statistics and annotation details of Lamiaceae members is provided in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s4.xlsx?Expires=2147483647&Signature=fY5sIBQchRuOnFo1DQD3KzaR2Y9Kf-lbsL-pa0DJLg2l0KguC1ZanbibWUhPYunU0Qwu5WaVhQ39uT2U5~KHM8KaS7ZUfoUtlM0rT~5Dpcds70e5TrBN9IAgdci4OHMCL6YfKN2ws5il2n5NSD3MhJiJz36k128iFA~32PDWOeLx2H5qGlA3eVW8lTWLDYoZgghhH-FHR2S6Gi1Y6jtsN165uWfzOnse9c9QILG4MYA95k~2JoAbpwAhiRtOnMYJ8UpnNe0Wi3Af~NLXogoZQXc2pX2c19K6L7SBIQewEzFviG4w6VLljg~LWnPDr2S9uuQjr7oUxPa55HJWw9G3ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. <div content-id="dsy013-T4" class="table-modal"><div class="table-wrap"><div id="dsy013-T4" data-id="dsy013-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Draft genome assembly statistics of teak</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Parameters </th><th>Contig </th><th>Scaffold </th><th>Gap closer </th><th>Draft genome </th></tr></thead><tbody><tr><td>Contigs generated </td><td>3,500 </td><td>3,004 </td><td>3,004 </td><td>2,993 </td></tr><tr><td>Maximum contig length (bp) </td><td>1,718,119 </td><td>1,718,322 </td><td>1,718,606 </td><td>1,718,606 </td></tr><tr><td>Minimum contig length (bp) </td><td>332 </td><td>445 </td><td>445 </td><td>1,100 </td></tr><tr><td>Average contig length (bp) </td><td>90,394 </td><td>105,594 </td><td>105,712 </td><td>106,098 </td></tr><tr><td>Total contigs length (bp) </td><td>316,377,938 </td><td>317,203,315 </td><td>317,558,121 </td><td>317,551,182 </td></tr><tr><td>Total number of non-ATGC characters </td><td>2,084,563 </td><td>2,374,171 </td><td>808,446 </td><td>808,446 </td></tr><tr><td>Percentage of non-ATGC characters </td><td>   0.659 </td><td>   0.748 </td><td>   0.255 </td><td>   0.255 </td></tr><tr><td>Contigs ≥ 500 bp </td><td>3,484 </td><td>3,003 </td><td>3,003 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 1 Kbp </td><td>3,478 </td><td>2,993 </td><td>2,993 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 10 Kbp </td><td>2,431 </td><td>2,069 </td><td>2,069 </td><td>2,069 </td></tr><tr><td>Contigs ≥ 1 Mbp </td><td>8 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>N50 value (bp) </td><td>277,872 </td><td>357,576 </td><td>357,576 </td><td>357,576 </td></tr></tbody></table></div><div content-id="dsy013-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>Parameters </th><th>Contig </th><th>Scaffold </th><th>Gap closer </th><th>Draft genome </th></tr></thead><tbody><tr><td>Contigs generated </td><td>3,500 </td><td>3,004 </td><td>3,004 </td><td>2,993 </td></tr><tr><td>Maximum contig length (bp) </td><td>1,718,119 </td><td>1,718,322 </td><td>1,718,606 </td><td>1,718,606 </td></tr><tr><td>Minimum contig length (bp) </td><td>332 </td><td>445 </td><td>445 </td><td>1,100 </td></tr><tr><td>Average contig length (bp) </td><td>90,394 </td><td>105,594 </td><td>105,712 </td><td>106,098 </td></tr><tr><td>Total contigs length (bp) </td><td>316,377,938 </td><td>317,203,315 </td><td>317,558,121 </td><td>317,551,182 </td></tr><tr><td>Total number of non-ATGC characters </td><td>2,084,563 </td><td>2,374,171 </td><td>808,446 </td><td>808,446 </td></tr><tr><td>Percentage of non-ATGC characters </td><td>   0.659 </td><td>   0.748 </td><td>   0.255 </td><td>   0.255 </td></tr><tr><td>Contigs ≥ 500 bp </td><td>3,484 </td><td>3,003 </td><td>3,003 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 1 Kbp </td><td>3,478 </td><td>2,993 </td><td>2,993 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 10 Kbp </td><td>2,431 </td><td>2,069 </td><td>2,069 </td><td>2,069 </td></tr><tr><td>Contigs ≥ 1 Mbp </td><td>8 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>N50 value (bp) </td><td>277,872 </td><td>357,576 </td><td>357,576 </td><td>357,576 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy013-T4" data-id="dsy013-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Draft genome assembly statistics of teak</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Parameters </th><th>Contig </th><th>Scaffold </th><th>Gap closer </th><th>Draft genome </th></tr></thead><tbody><tr><td>Contigs generated </td><td>3,500 </td><td>3,004 </td><td>3,004 </td><td>2,993 </td></tr><tr><td>Maximum contig length (bp) </td><td>1,718,119 </td><td>1,718,322 </td><td>1,718,606 </td><td>1,718,606 </td></tr><tr><td>Minimum contig length (bp) </td><td>332 </td><td>445 </td><td>445 </td><td>1,100 </td></tr><tr><td>Average contig length (bp) </td><td>90,394 </td><td>105,594 </td><td>105,712 </td><td>106,098 </td></tr><tr><td>Total contigs length (bp) </td><td>316,377,938 </td><td>317,203,315 </td><td>317,558,121 </td><td>317,551,182 </td></tr><tr><td>Total number of non-ATGC characters </td><td>2,084,563 </td><td>2,374,171 </td><td>808,446 </td><td>808,446 </td></tr><tr><td>Percentage of non-ATGC characters </td><td>   0.659 </td><td>   0.748 </td><td>   0.255 </td><td>   0.255 </td></tr><tr><td>Contigs ≥ 500 bp </td><td>3,484 </td><td>3,003 </td><td>3,003 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 1 Kbp </td><td>3,478 </td><td>2,993 </td><td>2,993 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 10 Kbp </td><td>2,431 </td><td>2,069 </td><td>2,069 </td><td>2,069 </td></tr><tr><td>Contigs ≥ 1 Mbp </td><td>8 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>N50 value (bp) </td><td>277,872 </td><td>357,576 </td><td>357,576 </td><td>357,576 </td></tr></tbody></table></div><div content-id="dsy013-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>Parameters </th><th>Contig </th><th>Scaffold </th><th>Gap closer </th><th>Draft genome </th></tr></thead><tbody><tr><td>Contigs generated </td><td>3,500 </td><td>3,004 </td><td>3,004 </td><td>2,993 </td></tr><tr><td>Maximum contig length (bp) </td><td>1,718,119 </td><td>1,718,322 </td><td>1,718,606 </td><td>1,718,606 </td></tr><tr><td>Minimum contig length (bp) </td><td>332 </td><td>445 </td><td>445 </td><td>1,100 </td></tr><tr><td>Average contig length (bp) </td><td>90,394 </td><td>105,594 </td><td>105,712 </td><td>106,098 </td></tr><tr><td>Total contigs length (bp) </td><td>316,377,938 </td><td>317,203,315 </td><td>317,558,121 </td><td>317,551,182 </td></tr><tr><td>Total number of non-ATGC characters </td><td>2,084,563 </td><td>2,374,171 </td><td>808,446 </td><td>808,446 </td></tr><tr><td>Percentage of non-ATGC characters </td><td>   0.659 </td><td>   0.748 </td><td>   0.255 </td><td>   0.255 </td></tr><tr><td>Contigs ≥ 500 bp </td><td>3,484 </td><td>3,003 </td><td>3,003 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 1 Kbp </td><td>3,478 </td><td>2,993 </td><td>2,993 </td><td>2,993 </td></tr><tr><td>Contigs ≥ 10 Kbp </td><td>2,431 </td><td>2,069 </td><td>2,069 </td><td>2,069 </td></tr><tr><td>Contigs ≥ 1 Mbp </td><td>8 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>N50 value (bp) </td><td>277,872 </td><td>357,576 </td><td>357,576 </td><td>357,576 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=120121833 id="120121833" class="section-title" >3.2. Repetitive genome elements</h3>
<p>Repetitive DNAs and transposable elements are ubiquitously present in eukaryotic genomes. They provide wide variety of variations across plant species. Repetitive elements are fast evolving components of nuclear genome that play an important role in evolution of the species<a reveal-id="dsy013-B42" data-open="dsy013-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> and interspecific divergence.<a reveal-id="dsy013-B43" data-open="dsy013-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The internal sequence variability of various repeat elements depends on the ratio between mutation and homogenization/fixation rates within a species.<a reveal-id="dsy013-B44" data-open="dsy013-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Repeat masking of the reference genome of teak with <em>A. thaliana</em> showed that a total of 19,046,577 bp (6% of the genome) had repeat elements which was very low when compared to <em>Salvia</em> repeat elements.<a reveal-id="dsy013-B45" data-open="dsy013-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> The classification of repetitive elements of teak genome is provided in <a reveal-id="dsy013-T5" data-open="dsy013-T5" class="link link-reveal link-table xref-fig">Table 5</a>. The simple repeats were dominant occupying about 3.36% of the total genome that can aid in molecular marker development. A total of 6,976 (1.62%) retroelements, 6,518 (1.58%) long terminal repeats (LTRs), 4,431 (0.24%) DNA transposons, 2,687 (0.70%) Copia-type, and 2,499 (0.81%) Gypsy-type sequences were predicted in teak. Number of repeat elements in teak differed from other Lamiaceae members, <em>Mentha longifolia</em><a reveal-id="dsy013-B46" data-open="dsy013-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> and <em>Ocimum sanctum</em>.<a reveal-id="dsy013-B47" data-open="dsy013-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsy013-B48" data-open="dsy013-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> In <em>M. longifolia</em><a reveal-id="dsy013-B46" data-open="dsy013-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> LTR elements were predicted as 3,866, whereas in teak it was 6,518. In <em>Ocimum tenuiflorum</em><a reveal-id="dsy013-B48" data-open="dsy013-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a>, the percent distribution of long interspersed nuclear elements (LINEs), LTR, unclassified repeat elements and total interspersed repeats was 0.3, 11.07, 27.99, and 40.71, respectively, but in teak the distribution was 0.04, 1.58, 0.02, and 1.88 respectively. In contrast, small RNA repeats were not recorded in <em>O. tenuiflorum</em><a reveal-id="dsy013-B47" data-open="dsy013-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> but 0.03% observed in teak. Presence of higher number of simple repeats, retroelements and interspersed repeats are common in plants,<a reveal-id="dsy013-B49" data-open="dsy013-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> which was reflected in teak genome as well. Overall, the percent distribution of repeat elements in teak genome seemed to be very low compared to pine genome, where 82% of genome is repetitive in nature.<a reveal-id="dsy013-B50" data-open="dsy013-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> The sequencing methods, per cent coverage and regions covered in the genome influence the representation of repeat elements.<a reveal-id="dsy013-B51" data-open="dsy013-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> The teak genome is estimated to be 317 Mb in length, and at least 11% of its sequence is observed to be made of repeat elements. Variations in repeat elements among the members of Lamiaceae could be due to the variations in chromosome number and ploidy level among the species which may reflect on evolutionary distances between genomes.<a reveal-id="dsy013-B43" data-open="dsy013-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Further, in this study, only 6% of the genome was used for masking and WGS assembly may have missed out some regions that are rich in repeated sequences. <div content-id="dsy013-T5" class="table-modal"><div class="table-wrap"><div id="dsy013-T5" data-id="dsy013-T5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Overview of repeat elements in teak genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>Number of elements </th><th>Length occupied (bp) </th><th>Percentage in genome </th></tr></thead><tbody><tr><td>Retroelements </td><td>6,976 </td><td>5,153,927 </td><td>1.62 </td></tr><tr><td>SINEs </td><td>1 </td><td>46 </td><td>0.00 </td></tr><tr><td>LINEs </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>L1/CIN4 </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>LTR elements </td><td>6,518 </td><td>5,031,200 </td><td>1.58 </td></tr><tr><td>Ty1/Copia </td><td>2,687 </td><td>2,212,758 </td><td>0.70 </td></tr><tr><td>Gypsy/DIRS1 </td><td>2,499 </td><td>2,564,424 </td><td>0.81 </td></tr><tr><td>DNA transposons </td><td>4,431 </td><td>749,766 </td><td>0.24 </td></tr><tr><td>hobo-Activator </td><td>647 </td><td>139,917 </td><td>0.04 </td></tr><tr><td>Tc1-IS630-Pogo </td><td>1,298 </td><td>216,636 </td><td>0.07 </td></tr><tr><td>Tourist/Harbinger </td><td>285 </td><td>70,800 </td><td>0.02 </td></tr><tr><td>Unclassified </td><td>242 </td><td>63,153 </td><td>0.02 </td></tr><tr><td>Total interspersed repeats </td><td>−  </td><td>5,966,846 </td><td>1.88 </td></tr><tr><td>Small RNA </td><td>158 </td><td>104,298 </td><td>0.03 </td></tr><tr><td>Satellites </td><td>1 </td><td>54 </td><td>0.00 </td></tr><tr><td>Simple repeats </td><td>253,260 </td><td>10,655,131 </td><td>3.36 </td></tr><tr><td>Low complexity </td><td>46,994 </td><td>2,328,770 </td><td>0.73 </td></tr><tr><td>Total bases masked </td><td colspan="2"> 19,046,577 </td><td>6.00 </td></tr></tbody></table></div><div content-id="dsy013-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>Number of elements </th><th>Length occupied (bp) </th><th>Percentage in genome </th></tr></thead><tbody><tr><td>Retroelements </td><td>6,976 </td><td>5,153,927 </td><td>1.62 </td></tr><tr><td>SINEs </td><td>1 </td><td>46 </td><td>0.00 </td></tr><tr><td>LINEs </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>L1/CIN4 </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>LTR elements </td><td>6,518 </td><td>5,031,200 </td><td>1.58 </td></tr><tr><td>Ty1/Copia </td><td>2,687 </td><td>2,212,758 </td><td>0.70 </td></tr><tr><td>Gypsy/DIRS1 </td><td>2,499 </td><td>2,564,424 </td><td>0.81 </td></tr><tr><td>DNA transposons </td><td>4,431 </td><td>749,766 </td><td>0.24 </td></tr><tr><td>hobo-Activator </td><td>647 </td><td>139,917 </td><td>0.04 </td></tr><tr><td>Tc1-IS630-Pogo </td><td>1,298 </td><td>216,636 </td><td>0.07 </td></tr><tr><td>Tourist/Harbinger </td><td>285 </td><td>70,800 </td><td>0.02 </td></tr><tr><td>Unclassified </td><td>242 </td><td>63,153 </td><td>0.02 </td></tr><tr><td>Total interspersed repeats </td><td>−  </td><td>5,966,846 </td><td>1.88 </td></tr><tr><td>Small RNA </td><td>158 </td><td>104,298 </td><td>0.03 </td></tr><tr><td>Satellites </td><td>1 </td><td>54 </td><td>0.00 </td></tr><tr><td>Simple repeats </td><td>253,260 </td><td>10,655,131 </td><td>3.36 </td></tr><tr><td>Low complexity </td><td>46,994 </td><td>2,328,770 </td><td>0.73 </td></tr><tr><td>Total bases masked </td><td colspan="2"> 19,046,577 </td><td>6.00 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy013-T5" data-id="dsy013-T5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Overview of repeat elements in teak genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>Number of elements </th><th>Length occupied (bp) </th><th>Percentage in genome </th></tr></thead><tbody><tr><td>Retroelements </td><td>6,976 </td><td>5,153,927 </td><td>1.62 </td></tr><tr><td>SINEs </td><td>1 </td><td>46 </td><td>0.00 </td></tr><tr><td>LINEs </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>L1/CIN4 </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>LTR elements </td><td>6,518 </td><td>5,031,200 </td><td>1.58 </td></tr><tr><td>Ty1/Copia </td><td>2,687 </td><td>2,212,758 </td><td>0.70 </td></tr><tr><td>Gypsy/DIRS1 </td><td>2,499 </td><td>2,564,424 </td><td>0.81 </td></tr><tr><td>DNA transposons </td><td>4,431 </td><td>749,766 </td><td>0.24 </td></tr><tr><td>hobo-Activator </td><td>647 </td><td>139,917 </td><td>0.04 </td></tr><tr><td>Tc1-IS630-Pogo </td><td>1,298 </td><td>216,636 </td><td>0.07 </td></tr><tr><td>Tourist/Harbinger </td><td>285 </td><td>70,800 </td><td>0.02 </td></tr><tr><td>Unclassified </td><td>242 </td><td>63,153 </td><td>0.02 </td></tr><tr><td>Total interspersed repeats </td><td>−  </td><td>5,966,846 </td><td>1.88 </td></tr><tr><td>Small RNA </td><td>158 </td><td>104,298 </td><td>0.03 </td></tr><tr><td>Satellites </td><td>1 </td><td>54 </td><td>0.00 </td></tr><tr><td>Simple repeats </td><td>253,260 </td><td>10,655,131 </td><td>3.36 </td></tr><tr><td>Low complexity </td><td>46,994 </td><td>2,328,770 </td><td>0.73 </td></tr><tr><td>Total bases masked </td><td colspan="2"> 19,046,577 </td><td>6.00 </td></tr></tbody></table></div><div content-id="dsy013-T5" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>Number of elements </th><th>Length occupied (bp) </th><th>Percentage in genome </th></tr></thead><tbody><tr><td>Retroelements </td><td>6,976 </td><td>5,153,927 </td><td>1.62 </td></tr><tr><td>SINEs </td><td>1 </td><td>46 </td><td>0.00 </td></tr><tr><td>LINEs </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>L1/CIN4 </td><td>457 </td><td>122,681 </td><td>0.04 </td></tr><tr><td>LTR elements </td><td>6,518 </td><td>5,031,200 </td><td>1.58 </td></tr><tr><td>Ty1/Copia </td><td>2,687 </td><td>2,212,758 </td><td>0.70 </td></tr><tr><td>Gypsy/DIRS1 </td><td>2,499 </td><td>2,564,424 </td><td>0.81 </td></tr><tr><td>DNA transposons </td><td>4,431 </td><td>749,766 </td><td>0.24 </td></tr><tr><td>hobo-Activator </td><td>647 </td><td>139,917 </td><td>0.04 </td></tr><tr><td>Tc1-IS630-Pogo </td><td>1,298 </td><td>216,636 </td><td>0.07 </td></tr><tr><td>Tourist/Harbinger </td><td>285 </td><td>70,800 </td><td>0.02 </td></tr><tr><td>Unclassified </td><td>242 </td><td>63,153 </td><td>0.02 </td></tr><tr><td>Total interspersed repeats </td><td>−  </td><td>5,966,846 </td><td>1.88 </td></tr><tr><td>Small RNA </td><td>158 </td><td>104,298 </td><td>0.03 </td></tr><tr><td>Satellites </td><td>1 </td><td>54 </td><td>0.00 </td></tr><tr><td>Simple repeats </td><td>253,260 </td><td>10,655,131 </td><td>3.36 </td></tr><tr><td>Low complexity </td><td>46,994 </td><td>2,328,770 </td><td>0.73 </td></tr><tr><td>Total bases masked </td><td colspan="2"> 19,046,577 </td><td>6.00 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=120121835 id="120121835" class="section-title" >3.3. Annotation and gene prediction</h3>
<p>Functional annotation, process of identifying sequence similarity to other known genes or proteins, in teak was carried out using the hard masked draft genome for gene prediction. A total of 36,172 proteins were predicted of which 31,126 (86%) proteins were annotated against <em>Viridiplantae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s5.xls?Expires=2147483647&amp;Signature=YLgLixRIoaSsWdnwLj~xQ5p01dXjNm0zGV3UO5EVjgXTm~c9NpXQNSspuZF15Ah~PoM4VA37PYeGTsHHczXNJgkg9Dp6b5rB-G9HYUs3oNhkPJC5qB5HYZU9zLgR4iaWqAWAevN4iybzHve3UM7tDRHUpxHeTtduBaMGuuI1MYIGRoNVqVIlCnemD8VZWFY8SeoUWQchd3KKEbDik8Rt3dhZyJQRmc1gbOMmE0KM0Z923uD8j5xIF2-JrSo1pI7gYhz3NGe9a6Q3id-LViTszadrSzHN500skShRtWPly46MpjOq0h3belUUUmUxvBXs6f8p-fVDV153cgMAjnq~GA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>) and 5,046 proteins were unannotated. In this study, length of the longest and shortest annotated protein sequence was 5,648 and 32 amino acids, respectively. The number of predicted genes is in accordance with Lamiaceae members, <em>Mentha</em> and <em>Ocimum</em>, except for one cultivar of <em>O. tenuiflorum</em>, where 53,480 genes were predicted.<a reveal-id="dsy013-B46 dsy013-B47 dsy013-B48" data-open="dsy013-B46 dsy013-B47 dsy013-B48" class="link link-ref link-reveal xref-bibr"><sup>46–48</sup></a> List of species considered for pathway analysis by homology based alignment of the <em>T. grandis</em> draft genome is given in the <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s6.xls?Expires=2147483647&amp;Signature=BV4CEdgvKb-Z75jOQ3WdC8Ek0mLue3yPTXgrk4AmTpRL1ft~nyh2v6jpBuRWbYRUr-eQ363VdvG3M0glXYLzWUBZVbXZDW25InAQE0Roarp-yZSLEvZ1YOuPFDAFgscASkPOubbKbfT5SZ4qgHnobMnekzJm380w43F4MWpU-qsL3oBXxex~ldMsxLJi7W2HIT5jtvP94NtN35b4dcA8MIzPE3AMalbsXXqTtbs~1sV5-4-toelEAtwj4UwOvyTgvSkMnK9~bJD-SoAUGXDJhYvRgYUj8nSfGPxghfmHz-zPvqIUqzQj0wssy8~fibpPVsLYOXr82jNoRJpeIuWnMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>. The proteins with &gt;30% identity cut off were taken for pathway analysis. Overall, 344 taxa had similarity hits when searched against Uniprot <em>Viridiplantae</em> protein database for similarity using BLASTP programme with an <em>e</em>-value of <em>e</em>-10. Fifteen plant species showed high homology is listed out in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s1-s3_s7_s9-s10.doc?Expires=2147483647&amp;Signature=eTnP40iJtj5DxIBCwFiS3ZARptTtP-uv4EyApY2QfXgyVe9eAAfNsEIQfnSidLxB6kPKdZFUo3RvZxN363eis1NiSZ3yXCiy41hYX1nSnVgJtOjuNrP4XP6vKSG4-4HG6YiJ~xnfiRGn-y6QJuX2jXisawegWAjfWWrKA80ww~mKby~8Y6QyovAheKJvTo4p9fNGgFED2sx1IU5D~sp25AE6ijNy81cSsfuNSY~05I7cflkbvQliBC2uMFpWMyn139znnYjtPUV2riu-9G5RSH7vqWZfB0j-UOYjxc8QN2JuJYNTdMkmGqZnhwRaikXyuRQHVEjTaMAuu9vNfK8z-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. Among the 31,126 predicted protein-coding genes, 17,353 genes (55%) showed high similarity to <em>Erythranthe guttata</em> 2,478 to <em>Coffea canephora</em>, 1,922 to <em>Solanum tuberosum</em> and 1,049 to <em>Vitis vinifera</em>. Highest per cent of similarity in the predicted coding genes with <em>E. guttata</em> (17,353 genes) could reflect genetic relationship of this genus with teak as both these belonging to the class Lamioideae.<a reveal-id="dsy013-B52" data-open="dsy013-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a></p><p>Gene ontology analysis revealed that 48.08% genes related to molecular functions (<a reveal-id="dsy013-F2" data-open="dsy013-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a>), 34.35% genes related to cellular components (<a reveal-id="dsy013-F2" data-open="dsy013-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>) and 12.56% involved in biological processes (<a reveal-id="dsy013-F2" data-open="dsy013-F2" class="link link-reveal link-table xref-fig">Fig. 2c</a>). In terms of biological processes, the major categories were transcription, regulation of transcription and metabolic and defense processes. Cellular component consisted of a major portion of integral component of membrane, followed by nucleus and cytoplasm components. In terms of molecular function, the top three GO terms were ATP binding, DNA binding and metal ion binding activities. Classification of all the protein sequences grouped under five categories such as metabolism, cellular processes, environmental information processing, genetic information processing and organismal systems. Metabolism related sequences were represented in the highest number, in which genes were representing carbohydrate metabolism (1,100) ranked first, followed by amino acid metabolism (643) (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s8.xls?Expires=2147483647&amp;Signature=GnjmH6loFnEfvSYOeaemZXp7R3U9oEeCopFQSeLkgbBa1y78K-YrtCxZYJuZ6J0K8WdOk66MAj6zSh0Tk-m2H3pujh5VD2EZNca1H3V9b5OiYW-2wujbvaIL0RltnPTSAqaM0trcpDE3dV1LkGlniN460yJN9z6YYm4fkBPia6b52iPQrMML9Vtp3kiY5KPbvGMw4Bz6BktaKaHj9UnYwwlMwxrtBgZtobGSDRchZv3dn-qZn3htUcWRSdu1I3gsTTnRbFIm6tQvX792S7U~y3vN29yQuYpGsDSjmdaTQK60sns0BiL04r1XV8nTyiSP20iquWvjfZHhYtRSD1pMzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The top five pathways were involved in plant-pathogen interaction, plant hormone signal transduction, carbon metabolism, ribosomes, and protein processing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s1-s3_s7_s9-s10.doc?Expires=2147483647&amp;Signature=eTnP40iJtj5DxIBCwFiS3ZARptTtP-uv4EyApY2QfXgyVe9eAAfNsEIQfnSidLxB6kPKdZFUo3RvZxN363eis1NiSZ3yXCiy41hYX1nSnVgJtOjuNrP4XP6vKSG4-4HG6YiJ~xnfiRGn-y6QJuX2jXisawegWAjfWWrKA80ww~mKby~8Y6QyovAheKJvTo4p9fNGgFED2sx1IU5D~sp25AE6ijNy81cSsfuNSY~05I7cflkbvQliBC2uMFpWMyn139znnYjtPUV2riu-9G5RSH7vqWZfB0j-UOYjxc8QN2JuJYNTdMkmGqZnhwRaikXyuRQHVEjTaMAuu9vNfK8z-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). </p>                        <a id="120121838" scrollto-destination="120121838"></a>
<div id="dsy013-F2" data-id="dsy013-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f2.png?Expires=2147483647&Signature=JtunIpzsh3gsZ9hAA4fP3d5cgEmX13xtnRTQs6oCG-gtJK1HB5TuyKci~C~Hvv~7TotHdoB9C41mNv8BI2SifLlRi6lmml8UB3vBC4AviXlg6aqRfZapGLjXoT5kX-hI0kflaETgSn0Wmo6jdFzKpiRmRP-SUhNDdd-ktWuHhuJrAEk0wwGXarNLzIMj6NJV3uzJ0FIr0ULxfEPvMI4MyR5o2GzgBe5tSbf2ocAZgE2myPwqWSDAN0lnfQhKm8apn98WXRi6AXon9IJW3bBCi5l43JIYmb6iqeVms3WL9ZKmqrg5tyUHc0E7sBsWLKMLnzmyTkMq7fVf8TNazfrjEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of teak genome sequence by gene ontology categories: (a) Biological process; (b) Molecular function; (c) Cellular component." path-from-xml="dsy013f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121838/dsy013f2.tif" path-from-xml="dsy013f2.tif" target="_blank">View large</a><a section="120121838" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f2.png?Expires=2147483647&Signature=FmwI8WGJ2RxZbaOEfV4ZWibDXH3dG7SdJi8rn8KcrEMaSMUTO1XRga84Duu2ZzlyrU8K~YsG~DNdx7L5bQAcIhR2EIO6fQO2rrMeLUQbHh~iDeXOnSx5ZDICpt4urGSp1Mn0XgFGvDFGkHnHVAP0uqrdzVLT4mJfxcaWUAteswV5Ctr5wSwvYLZbPPh~q1RDZKvhtGknIS8sZgDPvA-h6SIb2Ho6YhFoOuOvVl7reo3ygnuqdp3XXHBA3kjkQRl-oUi3lY5NTUQI36XzmFwR5cH~ixZ4UB7qXmY3Oe781FDtqJQattNO~ByUdoffcwFvvSbBrihjI4cLuSnwcHp9-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121838&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characterization of teak genome sequence by gene ontology categories: (a) Biological process; (b) Molecular function; (c) Cellular component.</p></div></div><div content-id="dsy013-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f2.png?Expires=2147483647&Signature=JtunIpzsh3gsZ9hAA4fP3d5cgEmX13xtnRTQs6oCG-gtJK1HB5TuyKci~C~Hvv~7TotHdoB9C41mNv8BI2SifLlRi6lmml8UB3vBC4AviXlg6aqRfZapGLjXoT5kX-hI0kflaETgSn0Wmo6jdFzKpiRmRP-SUhNDdd-ktWuHhuJrAEk0wwGXarNLzIMj6NJV3uzJ0FIr0ULxfEPvMI4MyR5o2GzgBe5tSbf2ocAZgE2myPwqWSDAN0lnfQhKm8apn98WXRi6AXon9IJW3bBCi5l43JIYmb6iqeVms3WL9ZKmqrg5tyUHc0E7sBsWLKMLnzmyTkMq7fVf8TNazfrjEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of teak genome sequence by gene ontology categories: (a) Biological process; (b) Molecular function; (c) Cellular component." path-from-xml="dsy013f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121838/dsy013f2.tif" path-from-xml="dsy013f2.tif" target="_blank">View large</a><a section="120121838" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f2.png?Expires=2147483647&Signature=FmwI8WGJ2RxZbaOEfV4ZWibDXH3dG7SdJi8rn8KcrEMaSMUTO1XRga84Duu2ZzlyrU8K~YsG~DNdx7L5bQAcIhR2EIO6fQO2rrMeLUQbHh~iDeXOnSx5ZDICpt4urGSp1Mn0XgFGvDFGkHnHVAP0uqrdzVLT4mJfxcaWUAteswV5Ctr5wSwvYLZbPPh~q1RDZKvhtGknIS8sZgDPvA-h6SIb2Ho6YhFoOuOvVl7reo3ygnuqdp3XXHBA3kjkQRl-oUi3lY5NTUQI36XzmFwR5cH~ixZ4UB7qXmY3Oe781FDtqJQattNO~ByUdoffcwFvvSbBrihjI4cLuSnwcHp9-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121838&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characterization of teak genome sequence by gene ontology categories: (a) Biological process; (b) Molecular function; (c) Cellular component.</p></div></div><p>Teak is known for its natural resistance against various decaying agents and is highly durable. Many biochemical studies on teak wood indicated the role of several secondary metabolites and phenolic compounds including flavanoids, alkaloids, terpenoids, quinines and tannins that play a major role for its durability.<a reveal-id="dsy013-B53" data-open="dsy013-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a><sup>,</sup><a reveal-id="dsy013-B54" data-open="dsy013-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> This study has identified a total of 615 gene sequences that directly code for enzymes involved in the synthesis of specialized secondary metabolites (363 genes) and biosynthesis of terpenoids and polyketides (252 genes). Lipid metabolism related genes were also represented in higher number (542) (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s8.xls?Expires=2147483647&Signature=GnjmH6loFnEfvSYOeaemZXp7R3U9oEeCopFQSeLkgbBa1y78K-YrtCxZYJuZ6J0K8WdOk66MAj6zSh0Tk-m2H3pujh5VD2EZNca1H3V9b5OiYW-2wujbvaIL0RltnPTSAqaM0trcpDE3dV1LkGlniN460yJN9z6YYm4fkBPia6b52iPQrMML9Vtp3kiY5KPbvGMw4Bz6BktaKaHj9UnYwwlMwxrtBgZtobGSDRchZv3dn-qZn3htUcWRSdu1I3gsTTnRbFIm6tQvX792S7U~y3vN29yQuYpGsDSjmdaTQK60sns0BiL04r1XV8nTyiSP20iquWvjfZHhYtRSD1pMzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The colour of wood is associated with extractive content, and is a useful parameter to estimate the durability of heartwood. Identification of several genes responsible for the production of above compounds in the teak genome would pave way for understanding the basis of natural resistance of teak timber. Recently, it was shown that heartwood specific transcriptome signatures were responsible for the presence of particular secondary metabolites through functional genomics studies in <em>Santalum album.</em><a reveal-id="dsy013-B55" data-open="dsy013-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Similar approaches would provide further insights into secondary wood formation in teak.</p>                            <h3 scrollto-destination=120121840 id="120121840" class="section-title" >3.4. The frequency and distribution of SSR types in teak genome</h3>
<p>In this study, totally 2,993 scaffolds amounting to 317.5 Mbp were examined for SSRs, of which 2,938 sequences were harbouring SSRs. Further, 2,846 sequences had more than one SSR and 11,255 SSRs were in compound form. Different types of SSR recorded in the teak genome are shown in <a reveal-id="dsy013-T6" data-open="dsy013-T6" class="link link-reveal link-table xref-fig">Table 6</a>. A total of 182,712 SSR motifs were identified, where perfect SSRs were represented in maximum numbers (170,574) with an overall frequency of 537.15 loci/Mbp accounting for 93% of SSRs. Compound, complex and interrupted types constituted 7% of the total SSRs, where interrupted complex type was the least in number. Among the pure repeat motifs, mononucleotide repeats were represented in maximum counts (88,766) followed by di (81,215), tri (14,654), tetra (8,086), penta (1,967) and hexanucleotides (1,161) (<a reveal-id="dsy013-T7" data-open="dsy013-T7" class="link link-reveal link-table xref-fig">Table 7</a>). Predominant (&gt;1,000) repeat times were 12–22 for mononucleotides, 7–16 for dinucleotides, 5–8 for trinucleotides, 4–7 for tetranucleotides and 4 for pentanucleotides. The major repeat motifs with over 5000 loci were (A)n, (T)n, (C)n, (G)n, (AC)n, (AT)n, (AG)n, (GT)n and (CT)n. Nine trinucleotide, four tetranucleotide and two pentanucleotide motifs were predominant (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s1-s3_s7_s9-s10.doc?Expires=2147483647&amp;Signature=eTnP40iJtj5DxIBCwFiS3ZARptTtP-uv4EyApY2QfXgyVe9eAAfNsEIQfnSidLxB6kPKdZFUo3RvZxN363eis1NiSZ3yXCiy41hYX1nSnVgJtOjuNrP4XP6vKSG4-4HG6YiJ~xnfiRGn-y6QJuX2jXisawegWAjfWWrKA80ww~mKby~8Y6QyovAheKJvTo4p9fNGgFED2sx1IU5D~sp25AE6ijNy81cSsfuNSY~05I7cflkbvQliBC2uMFpWMyn139znnYjtPUV2riu-9G5RSH7vqWZfB0j-UOYjxc8QN2JuJYNTdMkmGqZnhwRaikXyuRQHVEjTaMAuu9vNfK8z-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). (AT)n repeat motif with frequency of 131.2 loci/Mb was the most predominant dinucleotide SSRs, accounting for over 51.3% of the total dinucleotide SSRs. Primers were designed for 86,854 SSRs which had sufficient left and right sequences (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s11.xlsx?Expires=2147483647&amp;Signature=GN~6EqJezzkJ6Z7YwYJyLmvzxRCpvgId-mxzaI~KDeKoamX6YL9as4xpoIbPyzWWUbvpcr~l3paO-OBxaOqAIxkrXBGJ2MtwdGx9mAQvN7XkD0mZXOAyziXHeizLIaTC7t84nuRKVzJHa-rc4tzNYLgC3wGoCyZgatj4unV5iJU0CfAvcIxQopUWlY81GSt9of~aJBc~RK58~bH6NHkWUj3v2Tjn0vD5fVZXfDugkZlcMovbwUBzoeaJHehte7P3dudkRVz8uHlGTuRlDIA9fv8KOITvQMOv9HPxV~X-xZrdmI75MgNtSpCenw7Yu-zik1Ro54Y5FJBUBz-4IuLOWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). Presence of large number of short repeat type SSR loci in the teak genome may be due to the higher genomic mutation rate and long evolutionary history of the genus.<a reveal-id="dsy013-B56" data-open="dsy013-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><div content-id="dsy013-T6" class="table-modal"><div class="table-wrap"><div id="dsy013-T6" data-id="dsy013-T6" class="table-wrap-title"><div class="label -label">Table 6.</div><div class="caption"><p>Characteristics of six types of SSRs in teak genome</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SSR type </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th></tr></thead><tbody><tr><td>cd </td><td>6,309 </td><td>248,207 </td><td>39.34 </td><td>19.87 </td><td>781.63 </td></tr><tr><td>cx </td><td>227 </td><td>13,835 </td><td>60.95 </td><td>0.71 </td><td>43.57 </td></tr><tr><td>icd </td><td>4,049 </td><td>170,803 </td><td>42.18 </td><td>12.75 </td><td>537.88 </td></tr><tr><td>icx </td><td>670 </td><td>43,668 </td><td>65.18 </td><td>2.11 </td><td>137.51 </td></tr><tr><td>ip </td><td>883 </td><td>34,519 </td><td>39.09 </td><td>2.78 </td><td>108.7 </td></tr><tr><td>p </td><td>170,574 </td><td>3,006,200 </td><td>17.62 </td><td>537.15 </td><td>9,466.82 </td></tr></tbody></table></div><div content-id="dsy013-T6" class="table-modal"><table><thead name="thead" align=""><tr><th>SSR type </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th></tr></thead><tbody><tr><td>cd </td><td>6,309 </td><td>248,207 </td><td>39.34 </td><td>19.87 </td><td>781.63 </td></tr><tr><td>cx </td><td>227 </td><td>13,835 </td><td>60.95 </td><td>0.71 </td><td>43.57 </td></tr><tr><td>icd </td><td>4,049 </td><td>170,803 </td><td>42.18 </td><td>12.75 </td><td>537.88 </td></tr><tr><td>icx </td><td>670 </td><td>43,668 </td><td>65.18 </td><td>2.11 </td><td>137.51 </td></tr><tr><td>ip </td><td>883 </td><td>34,519 </td><td>39.09 </td><td>2.78 </td><td>108.7 </td></tr><tr><td>p </td><td>170,574 </td><td>3,006,200 </td><td>17.62 </td><td>537.15 </td><td>9,466.82 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy013-T6" data-id="dsy013-T6" class="table-wrap-title"><div class="label -label">Table 6.</div><div class="caption"><p>Characteristics of six types of SSRs in teak genome</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SSR type </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th></tr></thead><tbody><tr><td>cd </td><td>6,309 </td><td>248,207 </td><td>39.34 </td><td>19.87 </td><td>781.63 </td></tr><tr><td>cx </td><td>227 </td><td>13,835 </td><td>60.95 </td><td>0.71 </td><td>43.57 </td></tr><tr><td>icd </td><td>4,049 </td><td>170,803 </td><td>42.18 </td><td>12.75 </td><td>537.88 </td></tr><tr><td>icx </td><td>670 </td><td>43,668 </td><td>65.18 </td><td>2.11 </td><td>137.51 </td></tr><tr><td>ip </td><td>883 </td><td>34,519 </td><td>39.09 </td><td>2.78 </td><td>108.7 </td></tr><tr><td>p </td><td>170,574 </td><td>3,006,200 </td><td>17.62 </td><td>537.15 </td><td>9,466.82 </td></tr></tbody></table></div><div content-id="dsy013-T6" class="table-modal"><table><thead name="thead" align=""><tr><th>SSR type </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th></tr></thead><tbody><tr><td>cd </td><td>6,309 </td><td>248,207 </td><td>39.34 </td><td>19.87 </td><td>781.63 </td></tr><tr><td>cx </td><td>227 </td><td>13,835 </td><td>60.95 </td><td>0.71 </td><td>43.57 </td></tr><tr><td>icd </td><td>4,049 </td><td>170,803 </td><td>42.18 </td><td>12.75 </td><td>537.88 </td></tr><tr><td>icx </td><td>670 </td><td>43,668 </td><td>65.18 </td><td>2.11 </td><td>137.51 </td></tr><tr><td>ip </td><td>883 </td><td>34,519 </td><td>39.09 </td><td>2.78 </td><td>108.7 </td></tr><tr><td>p </td><td>170,574 </td><td>3,006,200 </td><td>17.62 </td><td>537.15 </td><td>9,466.82 </td></tr></tbody></table></div></div><div content-id="dsy013-T7" class="table-modal"><div class="table-wrap"><div id="dsy013-T7" data-id="dsy013-T7" class="table-wrap-title"><div class="label -label">Table 7.</div><div class="caption"><p>The number, length, frequency and density of six different types of SSRs</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Nucleotide </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th><th>SSRs in the whole genome (%) </th></tr></thead><tbody><tr><td>Mononucleotide </td><td>88,766 </td><td>1,321,753 </td><td>14.89 </td><td>279.53 </td><td>4,162.3 </td><td>45.32 </td></tr><tr><td>Dinucleotide </td><td>81,215 </td><td>1,664,278 </td><td>20.49 </td><td>255.75 </td><td>5,241 </td><td>41.4 </td></tr><tr><td>Trinucleotide </td><td>14,654 </td><td>286,074 </td><td>19.52 </td><td>46.15 </td><td>900.88 </td><td>7.48 </td></tr><tr><td>Tetranucleotide </td><td>8,086 </td><td>146,728 </td><td>18.15 </td><td>25.46 </td><td>462.06 </td><td>4.13 </td></tr><tr><td>Pentanucleotide </td><td>1,967 </td><td>42,960 </td><td>21.84 </td><td>6.19 </td><td>135.29 </td><td>1 </td></tr><tr><td>Hexanucleotide </td><td>1,161 </td><td>29,724 </td><td>25.6 </td><td>3.66 </td><td>93.604 </td><td>0.59 </td></tr></tbody></table></div><div content-id="dsy013-T7" class="table-modal"><table><thead name="thead" align=""><tr><th>Nucleotide </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th><th>SSRs in the whole genome (%) </th></tr></thead><tbody><tr><td>Mononucleotide </td><td>88,766 </td><td>1,321,753 </td><td>14.89 </td><td>279.53 </td><td>4,162.3 </td><td>45.32 </td></tr><tr><td>Dinucleotide </td><td>81,215 </td><td>1,664,278 </td><td>20.49 </td><td>255.75 </td><td>5,241 </td><td>41.4 </td></tr><tr><td>Trinucleotide </td><td>14,654 </td><td>286,074 </td><td>19.52 </td><td>46.15 </td><td>900.88 </td><td>7.48 </td></tr><tr><td>Tetranucleotide </td><td>8,086 </td><td>146,728 </td><td>18.15 </td><td>25.46 </td><td>462.06 </td><td>4.13 </td></tr><tr><td>Pentanucleotide </td><td>1,967 </td><td>42,960 </td><td>21.84 </td><td>6.19 </td><td>135.29 </td><td>1 </td></tr><tr><td>Hexanucleotide </td><td>1,161 </td><td>29,724 </td><td>25.6 </td><td>3.66 </td><td>93.604 </td><td>0.59 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Cd, compound; cx, complex; icd, interrupted compound; icx, interrupted complex; ip, imperfect; p, perfect.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy013-T7" data-id="dsy013-T7" class="table-wrap-title"><div class="label -label">Table 7.</div><div class="caption"><p>The number, length, frequency and density of six different types of SSRs</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Nucleotide </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th><th>SSRs in the whole genome (%) </th></tr></thead><tbody><tr><td>Mononucleotide </td><td>88,766 </td><td>1,321,753 </td><td>14.89 </td><td>279.53 </td><td>4,162.3 </td><td>45.32 </td></tr><tr><td>Dinucleotide </td><td>81,215 </td><td>1,664,278 </td><td>20.49 </td><td>255.75 </td><td>5,241 </td><td>41.4 </td></tr><tr><td>Trinucleotide </td><td>14,654 </td><td>286,074 </td><td>19.52 </td><td>46.15 </td><td>900.88 </td><td>7.48 </td></tr><tr><td>Tetranucleotide </td><td>8,086 </td><td>146,728 </td><td>18.15 </td><td>25.46 </td><td>462.06 </td><td>4.13 </td></tr><tr><td>Pentanucleotide </td><td>1,967 </td><td>42,960 </td><td>21.84 </td><td>6.19 </td><td>135.29 </td><td>1 </td></tr><tr><td>Hexanucleotide </td><td>1,161 </td><td>29,724 </td><td>25.6 </td><td>3.66 </td><td>93.604 </td><td>0.59 </td></tr></tbody></table></div><div content-id="dsy013-T7" class="table-modal"><table><thead name="thead" align=""><tr><th>Nucleotide </th><th>Total counts </th><th>Total length (bp) </th><th>Average length (bp) </th><th>Frequency (loci/Mb) </th><th>Density (bp/Mb) </th><th>SSRs in the whole genome (%) </th></tr></thead><tbody><tr><td>Mononucleotide </td><td>88,766 </td><td>1,321,753 </td><td>14.89 </td><td>279.53 </td><td>4,162.3 </td><td>45.32 </td></tr><tr><td>Dinucleotide </td><td>81,215 </td><td>1,664,278 </td><td>20.49 </td><td>255.75 </td><td>5,241 </td><td>41.4 </td></tr><tr><td>Trinucleotide </td><td>14,654 </td><td>286,074 </td><td>19.52 </td><td>46.15 </td><td>900.88 </td><td>7.48 </td></tr><tr><td>Tetranucleotide </td><td>8,086 </td><td>146,728 </td><td>18.15 </td><td>25.46 </td><td>462.06 </td><td>4.13 </td></tr><tr><td>Pentanucleotide </td><td>1,967 </td><td>42,960 </td><td>21.84 </td><td>6.19 </td><td>135.29 </td><td>1 </td></tr><tr><td>Hexanucleotide </td><td>1,161 </td><td>29,724 </td><td>25.6 </td><td>3.66 </td><td>93.604 </td><td>0.59 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Cd, compound; cx, complex; icd, interrupted compound; icx, interrupted complex; ip, imperfect; p, perfect.</p></div></div></div></div></p>                            <h3 scrollto-destination=120121842 id="120121842" class="section-title" >3.5. Selection of polymorphic SSRs and validation</h3>
<p>SSRs have become powerful markers for population genetic analysis, QTL mapping and other related genetic and genomic studies.<a reveal-id="dsy013-B57" data-open="dsy013-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> The conventional methods for SSR genotyping are labour intensive, time consuming and costly, especially for tree species that lack DNA sequence information in the public databases. The recent advances in NGS methods offer rapid identification of repeat size variations by sequencing. Five teak accessions were re-sequenced with coverage level of 7.3x–11.6x (<a reveal-id="dsy013-T8" data-open="dsy013-T8" class="link link-reveal link-table xref-fig">Table 8</a>). The PSR tool was used to compare sequence variants among the five assemblies against the SSR sequences of reference genome. Among the 170,574 perfect SSRs found in teak genome, 16,252 showed polymorphisms across these genotypes and primer pairs were developed for 13,007 SSRs (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementray_table_s12_r1.xls?Expires=2147483647&Signature=KAanOGnLoTCb3bfL9OZL8hyjj4X1RWQIb1-GOYq1uUcdt~GGCB2Km6~w0z~MFZfmIuD~rp0k4jOdplFfvgCrkTy0JRFp~gr3wEyGqE1HxS8t2GPLD57ecAUmgBQALIH6qzPAm4zzuWP3sajvUifOs~qnXMA6bRIhZ2VX2k3nVThwG13e5Z-l26MblS0FlrBdIyrOlw3PoB0ZLnbCXdg80eiP5a28I0Yp~xoCz3BFzmT2gjaid0bkSgH65cotD~1qWqYxZSnmF~-QcgVQ1Hr7WQRC1v4psLQdJdtHpcupbgsBaTlIqDHIxEJAGXI1AEA5UKQ4f-LkGMZai6QpmoR~iQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). Heterozygous and homozygous conditions at each microsatellite locus across all genotypes were detected by the comparative module (PSR poly finder). <div content-id="dsy013-T8" class="table-modal"><div class="table-wrap"><div id="dsy013-T8" data-id="dsy013-T8" class="table-wrap-title"><div class="label -label">Table 8.</div><div class="caption"><p>Read Statistics of the teak samples sequenced at low depth coverage for identification of polymorphic SSRs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Total raw reads (bp) </th><th>Total processed reads (bp) </th><th>Total reference covered (%) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>22,503,220 </td><td>21,315,714 </td><td>90.1 </td><td>9.6 </td></tr><tr><td>3 </td><td>WR </td><td>28,219,627 </td><td>26,462,008 </td><td>90.6 </td><td>11.6 </td></tr><tr><td>4 </td><td>DI </td><td>19,670,649 </td><td>18,335,390 </td><td>87.6 </td><td>7.8 </td></tr><tr><td>5 </td><td>HI </td><td>17,029,396 </td><td>16,043,025 </td><td>85.8 </td><td>7.3 </td></tr><tr><td>6 </td><td>TP </td><td>18,838,068 </td><td>18,451,073 </td><td>94.3 </td><td>8.2 </td></tr><tr><td colspan="5">Average </td><td>8.9 </td></tr></tbody></table></div><div content-id="dsy013-T8" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Total raw reads (bp) </th><th>Total processed reads (bp) </th><th>Total reference covered (%) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>22,503,220 </td><td>21,315,714 </td><td>90.1 </td><td>9.6 </td></tr><tr><td>3 </td><td>WR </td><td>28,219,627 </td><td>26,462,008 </td><td>90.6 </td><td>11.6 </td></tr><tr><td>4 </td><td>DI </td><td>19,670,649 </td><td>18,335,390 </td><td>87.6 </td><td>7.8 </td></tr><tr><td>5 </td><td>HI </td><td>17,029,396 </td><td>16,043,025 </td><td>85.8 </td><td>7.3 </td></tr><tr><td>6 </td><td>TP </td><td>18,838,068 </td><td>18,451,073 </td><td>94.3 </td><td>8.2 </td></tr><tr><td colspan="5">Average </td><td>8.9 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy013-T8" data-id="dsy013-T8" class="table-wrap-title"><div class="label -label">Table 8.</div><div class="caption"><p>Read Statistics of the teak samples sequenced at low depth coverage for identification of polymorphic SSRs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Total raw reads (bp) </th><th>Total processed reads (bp) </th><th>Total reference covered (%) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>22,503,220 </td><td>21,315,714 </td><td>90.1 </td><td>9.6 </td></tr><tr><td>3 </td><td>WR </td><td>28,219,627 </td><td>26,462,008 </td><td>90.6 </td><td>11.6 </td></tr><tr><td>4 </td><td>DI </td><td>19,670,649 </td><td>18,335,390 </td><td>87.6 </td><td>7.8 </td></tr><tr><td>5 </td><td>HI </td><td>17,029,396 </td><td>16,043,025 </td><td>85.8 </td><td>7.3 </td></tr><tr><td>6 </td><td>TP </td><td>18,838,068 </td><td>18,451,073 </td><td>94.3 </td><td>8.2 </td></tr><tr><td colspan="5">Average </td><td>8.9 </td></tr></tbody></table></div><div content-id="dsy013-T8" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession ID </th><th>Sample code </th><th>Total raw reads (bp) </th><th>Total processed reads (bp) </th><th>Total reference covered (%) </th><th>Coverage (×) </th></tr></thead><tbody><tr><td>1 </td><td>NR </td><td>22,503,220 </td><td>21,315,714 </td><td>90.1 </td><td>9.6 </td></tr><tr><td>3 </td><td>WR </td><td>28,219,627 </td><td>26,462,008 </td><td>90.6 </td><td>11.6 </td></tr><tr><td>4 </td><td>DI </td><td>19,670,649 </td><td>18,335,390 </td><td>87.6 </td><td>7.8 </td></tr><tr><td>5 </td><td>HI </td><td>17,029,396 </td><td>16,043,025 </td><td>85.8 </td><td>7.3 </td></tr><tr><td>6 </td><td>TP </td><td>18,838,068 </td><td>18,451,073 </td><td>94.3 </td><td>8.2 </td></tr><tr><td colspan="5">Average </td><td>8.9 </td></tr></tbody></table></div></div></p><p>Gel electrophoresis of all the 10 primer pairs developed for polymorphic SSRs generated by the PSR software produced perfect banding pattern and no optimization of primer annealing temperature were required. All the amplified loci generated polymorphism except one locus as in the PSR results.<a reveal-id="dsy013-B29" data-open="dsy013-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Validation of more number of primer pairs in efficient allele separation systems like capillary electrophoresis would strengthen the SSR marker development. These results showed that identification of polymorphic SSRs by sequencing is highly cost efficient and rapid compared to conventional methods SSR identification.</p>                            <h3 scrollto-destination=120121845 id="120121845" class="section-title" >3.6. Phylogeny and divergence time estimation</h3>
<p>The Lamiaceae (Labiatae<em>)</em> family contains 236 genera and over 7,000 species, and is one of the largest families of seed plants.<a reveal-id="dsy013-B3" data-open="dsy013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The family Verbenaceae is closely related to Lamiaceae and differentiated mainly on the basis of terminal (Verbenceae)/gynobasis (Lamiaceae) style with difficulties in separating members of one family from the other.<a reveal-id="dsy013-B58" data-open="dsy013-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Previously, several genera from Verbenaceae were transferred to Lamiaceae including Tectona<a reveal-id="dsy013-B2" data-open="dsy013-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy013-B59" data-open="dsy013-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> but the systematic position of genus <em>Tectona</em> still lacks clarity. Out of 236 genera of Lamiaceae, 226 were placed under seven subfamilies (Ajugoideae, Lamioideae, Nepetoideae, Prostantheroideae, Scutellarioideae, Symphorematoideae and Viticoideae) and 10 genera were listed as Incertae sedis<a reveal-id="dsy013-B3" data-open="dsy013-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> (of ‘uncertain placement’) by considering morphology, secondary metabolites and molecular phylogeny.<a reveal-id="dsy013-B60 dsy013-B61 dsy013-B62 dsy013-B63 dsy013-B64" data-open="dsy013-B60 dsy013-B61 dsy013-B62 dsy013-B63 dsy013-B64" class="link link-ref link-reveal xref-bibr"><sup>60–64</sup></a> However, a recent study on chloroplast phylogeny proposed additional three subfamilies in Lamiaceae to encompass eight genera of Incertae sedis, leaving <em>Tectona</em> and <em>Callicarpa</em> unassigned.<a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> In the present study, potential phylogenetic plastid marker sequences, <em>ycf</em>2 and <em>psb</em>B were used to disentangle the taxonomic position of the genus <em>Tectona</em>. Although several different genes were used for phylogeny analysis of teak, the <em>ycf</em>2 and <em>psb</em>B genes were not reported so far. The plastid gene <em>psb</em>B which codes for the core protein of Photosystem II, has the highest level of translation among the chloroplast genes and provides an excellent opportunity to investigate an unusual evolutionary situation.<a reveal-id="dsy013-B65" data-open="dsy013-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> Phylogenetic utility of <em>psb</em>B gene sequences has been tested in many plant families under the Order Lamiales such as <em>Lamium</em>,<a reveal-id="dsy013-B66" data-open="dsy013-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a><sup>,</sup><em>Chelonopsis</em>,<a reveal-id="dsy013-B67" data-open="dsy013-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a><sup>,</sup><em>Salvia</em><a reveal-id="dsy013-B68" data-open="dsy013-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> and <em>Prosanthera</em><a reveal-id="dsy013-B69" data-open="dsy013-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> among others. Similar to <em>psb</em>B, studies on molecular systematics have undoubtedly proved that <em>ycf</em> gene is more variable than <em>mat</em>K in many taxa to resolve the phylogeny issues.<a reveal-id="dsy013-B70" data-open="dsy013-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a><sup>,</sup><a reveal-id="dsy013-B71" data-open="dsy013-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> This gene harbours highest level nucleotide genetic diversity among the angiosperm plastid genomes.<a reveal-id="dsy013-B72" data-open="dsy013-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> As the number of entries of Lamiaceae members in the public domain is very less, combination of two genes, <em>ycf</em>2 and <em>psb</em>B sequences generated phylogeny tree with 16 Lamiaceae species. Inclusion of other genes (<em>rbc</em>L and <em>psa</em>A) resulted in phylogenetic tree with lesser number (10–14) of Lamiaceae species (data not shown). The resulted phylogenetic tree retained the previously reported uncertain taxonomic status of the genus <em>Tectona</em>, which formed a separate sister clade to the clades comprising of Laminoideae, Ajugoideae, Nepetoideae, Scutellarioideae and Premonoideae (<a reveal-id="dsy013-F3" data-open="dsy013-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Monophyly of all the sub families under Lamiaceae were also reported.<a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a></p>                        <a id="120121847" scrollto-destination="120121847"></a>
<div id="dsy013-F3" data-id="dsy013-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f3.png?Expires=2147483647&Signature=3YiX7GxTaLYYRGx7cOpGdpENT2OtLjviv70wsN4U8V~aBxxGrU5-nI9g1UAvLsmbLVcKx3Zb5wPCTeNAwJUiJqdYa2JQoYbCvCImUf2W7kI~880xT1FR80zCUcoFOsiXF6VTyKJ0TEh0t4Daqe~QcPvmDH3YXcrGzE78hN7gP3Tc2mncFf6zbzFdJxS9UJGlj9UGXPDnIdrWSit--bPa5IstvaTdTKFHwdsPLHEAQIL38fQF-pOuCahPx0h9bP11-mglPUjtf~ScCO4D8BdjEcyBY9ElJ6nP-6a7K2E3Z~KH8hFSEoEwIa6d1~HF~KptMRH10tsLT~JdDOBvr3Mbgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bayesian tree generated by analysis of two plastid sequences psb and ycf2. Posterior probabilities are shown above the branches. Scale bar specifies mean branch length. Six major clades representing subfamilies of the Lamiaceae family are indicated." path-from-xml="dsy013f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121847/dsy013f3.tif" path-from-xml="dsy013f3.tif" target="_blank">View large</a><a section="120121847" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f3.png?Expires=2147483647&Signature=TyyteLW3jrn2j9nOnTrTONn6OTYJeAjCdx~ayleBKKcQ1SUv9UgP-HL-U~V3Bc6JQJhPGIBeD39hmreLukcC8xNUHeafB2Au4r7fwhCYC8dN2bj6mySfLYYXsw~kbxSfI61lsrb54TvQ4~ji37-a-7bCllV2aY9zD76rwoC4WcI7vE6ZjOv-g1jk~cOdtklKIt58m1J-VOYjPkdQyK54SQ7~rgR59Me-CfqN~8f9Wa6urVDUu9m-xUtzTmAaCB3-NjE1hyZHcOD20XfrgtYdzQ5yZ1LWZHK4kYJ2-9SBUPAy731pq1XxM9OQEaev~8FEwqYS-ud63m8sJM0rYoKzgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121847&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bayesian tree generated by analysis of two plastid sequences <em>psb</em> and <em>ycf</em>2. Posterior probabilities are shown above the branches. Scale bar specifies mean branch length. Six major clades representing subfamilies of the Lamiaceae family are indicated.</p></div></div><div content-id="dsy013-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f3.png?Expires=2147483647&Signature=3YiX7GxTaLYYRGx7cOpGdpENT2OtLjviv70wsN4U8V~aBxxGrU5-nI9g1UAvLsmbLVcKx3Zb5wPCTeNAwJUiJqdYa2JQoYbCvCImUf2W7kI~880xT1FR80zCUcoFOsiXF6VTyKJ0TEh0t4Daqe~QcPvmDH3YXcrGzE78hN7gP3Tc2mncFf6zbzFdJxS9UJGlj9UGXPDnIdrWSit--bPa5IstvaTdTKFHwdsPLHEAQIL38fQF-pOuCahPx0h9bP11-mglPUjtf~ScCO4D8BdjEcyBY9ElJ6nP-6a7K2E3Z~KH8hFSEoEwIa6d1~HF~KptMRH10tsLT~JdDOBvr3Mbgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bayesian tree generated by analysis of two plastid sequences psb and ycf2. Posterior probabilities are shown above the branches. Scale bar specifies mean branch length. Six major clades representing subfamilies of the Lamiaceae family are indicated." path-from-xml="dsy013f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121847/dsy013f3.tif" path-from-xml="dsy013f3.tif" target="_blank">View large</a><a section="120121847" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f3.png?Expires=2147483647&Signature=TyyteLW3jrn2j9nOnTrTONn6OTYJeAjCdx~ayleBKKcQ1SUv9UgP-HL-U~V3Bc6JQJhPGIBeD39hmreLukcC8xNUHeafB2Au4r7fwhCYC8dN2bj6mySfLYYXsw~kbxSfI61lsrb54TvQ4~ji37-a-7bCllV2aY9zD76rwoC4WcI7vE6ZjOv-g1jk~cOdtklKIt58m1J-VOYjPkdQyK54SQ7~rgR59Me-CfqN~8f9Wa6urVDUu9m-xUtzTmAaCB3-NjE1hyZHcOD20XfrgtYdzQ5yZ1LWZHK4kYJ2-9SBUPAy731pq1XxM9OQEaev~8FEwqYS-ud63m8sJM0rYoKzgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121847&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bayesian tree generated by analysis of two plastid sequences <em>psb</em> and <em>ycf</em>2. Posterior probabilities are shown above the branches. Scale bar specifies mean branch length. Six major clades representing subfamilies of the Lamiaceae family are indicated.</p></div></div><p>Earlier classifications on morphology considered <em>Tectona</em> under tribe Tectoneae in subfamily Viticoideae.<a reveal-id="dsy013-B73 dsy013-B74 dsy013-B75 dsy013-B76" data-open="dsy013-B73 dsy013-B74 dsy013-B75 dsy013-B76" class="link link-ref link-reveal xref-bibr"><sup>73–76</sup></a> Molecular phylogeny analysis and novel combination of morphological features delineated <em>Tectona</em> to be an earliest diverging lineage in Lamiaceae. The phylogenetic studies of Lamiaceae family deduced <em>T. grandis</em> to be an early diverging lineage.<a reveal-id="dsy013-B4" data-open="dsy013-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> The time estimated in the present study confirmed the early origin and divergence of the genera around 21.4508 Mya [95% highest posterior density (HPD): 10.11–34.52 Mya] (<a reveal-id="dsy013-F4" data-open="dsy013-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Most of the genera of the family Lamiaceae were found to have a Miocene origin. Recently, two new subfamilies have been proposed to Lamiaceae viz. Callicarpoideae and Tectonoideae with <em>Tectona</em> as a monotypic taxon.<a reveal-id="dsy013-B77" data-open="dsy013-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> Accordingly, assigning <em>Tectona</em> to a monogeneric subfamily Tectonoideae need to be considered, however it demands an extensive sampling within Lamiaceae and multigene phylogeny analysis to provide an appropriate taxonomical position. </p>                        <a id="120121849" scrollto-destination="120121849"></a>
<div id="dsy013-F4" data-id="dsy013-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f4.png?Expires=2147483647&Signature=nWW~qRrnnezoGdbzoO3h-YnC~miiWASl3oVdL7nHkpEMQHRmMy4I7QVyLMUrpGr6e56z2tMUMA0g9jbgAj9Wb8cR5IgqoGtQF2fnA8LvWLEo-qLhvJszRImtTO1IEGf~NtrUoHHFUIzQCofREl~nNjn~ig7gaPZWLOyg5aAElnpo7Zt6etY3ncXMIcHS46AbpWxcEtLX44oKtR6aJEATsZ3Ckzlu2k~lG9Ftz9xtxzkGkwEbfu~Yl~nFKVQOVK2Mt~~fGW~xRrGwRtp5lphfvXkFcr55Vw6dz6VXVZd359zGEV2-5J4nnkDOVRQv3FoPuPd-~IoXo4JnMd3CufKkUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chronogram of Lamiaceae (genus Tectona) based on two plastid sequences psb and ycf2, estimated from secondary calibration strategies as implemented in BEAST. Calibration points are indicated with black dots. Node bar indicates 95% HPD interval for node ages. Geological time scale is given in Mya. " path-from-xml="dsy013f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121849/dsy013f4.tif" path-from-xml="dsy013f4.tif" target="_blank">View large</a><a section="120121849" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f4.png?Expires=2147483647&Signature=sckIduOY0epQrgHtJkDwtGeAdrwAxokyyHaZKZOELOJLnB7TkRjN8R454W-LriaeoBZHtXWggKERceLIqZXiYErFLr1bnt8hJbUoFHS5Drr1qQBLJKkPrkQsrGgtGo0Z-XMjjNfoMHXf-qIQ2G~aDwKnhF2lxRdKYE7Y8zsGZTrifZfI1YP31GvWKzJ-kc66qcowYyuzi63R4kFvIomvUr-sMJgMbuRF6lzeIWHlXnE2x1vDW4eQsokqYQqMdMBB7RQ1HDQXeJW0djE6MP5yJwgZVPMswqcmVz7Z2Wh85Jn2GkpMn4RXDWApADFJwLOSqbwsoiY8joLVvt0MwBekuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121849&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chronogram of Lamiaceae (genus <em>Tectona</em>) based on two plastid sequences <em>psb</em> and <em>ycf</em>2, estimated from secondary calibration strategies as implemented in BEAST. Calibration points are indicated with black dots. Node bar indicates 95% HPD interval for node ages. Geological time scale is given in Mya. </p></div></div><div content-id="dsy013-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/m_dsy013f4.png?Expires=2147483647&Signature=nWW~qRrnnezoGdbzoO3h-YnC~miiWASl3oVdL7nHkpEMQHRmMy4I7QVyLMUrpGr6e56z2tMUMA0g9jbgAj9Wb8cR5IgqoGtQF2fnA8LvWLEo-qLhvJszRImtTO1IEGf~NtrUoHHFUIzQCofREl~nNjn~ig7gaPZWLOyg5aAElnpo7Zt6etY3ncXMIcHS46AbpWxcEtLX44oKtR6aJEATsZ3Ckzlu2k~lG9Ftz9xtxzkGkwEbfu~Yl~nFKVQOVK2Mt~~fGW~xRrGwRtp5lphfvXkFcr55Vw6dz6VXVZd359zGEV2-5J4nnkDOVRQv3FoPuPd-~IoXo4JnMd3CufKkUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chronogram of Lamiaceae (genus Tectona) based on two plastid sequences psb and ycf2, estimated from secondary calibration strategies as implemented in BEAST. Calibration points are indicated with black dots. Node bar indicates 95% HPD interval for node ages. Geological time scale is given in Mya. " path-from-xml="dsy013f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121849/dsy013f4.tif" path-from-xml="dsy013f4.tif" target="_blank">View large</a><a section="120121849" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013f4.png?Expires=2147483647&Signature=sckIduOY0epQrgHtJkDwtGeAdrwAxokyyHaZKZOELOJLnB7TkRjN8R454W-LriaeoBZHtXWggKERceLIqZXiYErFLr1bnt8hJbUoFHS5Drr1qQBLJKkPrkQsrGgtGo0Z-XMjjNfoMHXf-qIQ2G~aDwKnhF2lxRdKYE7Y8zsGZTrifZfI1YP31GvWKzJ-kc66qcowYyuzi63R4kFvIomvUr-sMJgMbuRF6lzeIWHlXnE2x1vDW4eQsokqYQqMdMBB7RQ1HDQXeJW0djE6MP5yJwgZVPMswqcmVz7Z2Wh85Jn2GkpMn4RXDWApADFJwLOSqbwsoiY8joLVvt0MwBekuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121849&ar=5003450&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy013f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chronogram of Lamiaceae (genus <em>Tectona</em>) based on two plastid sequences <em>psb</em> and <em>ycf</em>2, estimated from secondary calibration strategies as implemented in BEAST. Calibration points are indicated with black dots. Node bar indicates 95% HPD interval for node ages. Geological time scale is given in Mya. </p></div></div>                            <h2 scrollto-destination=120121850 id="120121850" class="section-title" >4. Conclusion</h2>
<p>Shrinkage of natural populations of teak in its native locations and worldwide increase of managed plantations demand conservation of native forests, which are critical in providing the best possible alleles to maintain genetic diversity. Captive plantations inherently have narrow genetic base, limited gene flow, and exist in non-native environments, and these characteristics often significantly alter the evolutionary trajectory leading to decrease in population fitness. Conservation and maintenance of wild progenitors are increasingly important for genetic improvement programmes. Thus, full genome sequence of teak was developed and an attempt for the functional analyses of genome components was carried out to use it as a tool for conservation and tree breeding programmes. The polymorphic DNA markers developed in this study will propel the genetic and genomic research in teak, hitherto unavailable for the highly valuable timber yielding tropical hardwood species. The draft genome of teak along with a large number of markers will benefit various explorative studies including, genetic basis of wood properties, pest tolerance, adaptive traits, germplasm movement and genetic resource conservation.</p>                            <h2 scrollto-destination=120121852 id="120121852" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are thankful to the State Forest Departments of Kerala, Tamil Nadu and Karnataka for their permission to collect teak samples. The financial support received from Department of Biotechnology, Government of India (No. BT/PR7143/PBD/16/1011/2012) is gratefully acknowledged.</p>                            <h3 scrollto-destination=120121854 id="120121854" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=120121856 id="120121856" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy013-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tewari</div> <div class="given-names">D.N.</div></div></div> <div class="year">1992</div>, A Monograph on Teak (<em>Tectona grandis</em> Linn. f.), Dehra Dun, India: International Book Distributors</div></div></div></div><div content-id="dsy013-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thorne</div> <div class="given-names">R. F.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Classification and geography of the flowering plants</div>, <div class="source">Bot. Rev</div> ., <div class="volume">58</div>, <div class="fpage">225</div>–<div class="lpage">327</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classification%20and%20geography%20of%20the%20flowering%20plants&amp;author=R.%20F.%20Thorne&amp;publication_year=1992&amp;journal=Bot.%20Rev&amp;volume=58&amp;pages=225-327" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02858611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02858611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02858611"> </span></div> </div></div></div></div></div><div content-id="dsy013-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harley</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Atkins</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Budantsey</div> <div class="given-names">A. L.</div></div></div> <div class="year">2004</div>, Labiatae. In: <div class="person-group"><div class="name string-name"><div class="surname">Kubitzki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kadereit</div> <div class="given-names">J. W.</div></div></div> (eds) <div class="source">Families and Genera of Vascular Plants. Flowering Plants. Dicotyledons ∼ Lamiales (except Acanthaceae Including Avicenniaceae</div> , <div class="volume">vol. 7</div>. 
<div class="publisher-loc">Berlin</div>: 
<div class="publisher-name">Springer</div>, pp. <div class="fpage">167</div>–<div class="lpage">275</div>.</div></div></div></div><div content-id="dsy013-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Cantino</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Olmstead</div> <div class="given-names">R. G.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">A large-scale chloroplast phylogeny of the Lamiaceae sheds new light on its subfamilial classification</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">3434</div>.</div></div></div></div><div content-id="dsy013-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kollert</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kleine</div> <div class="given-names">M.</div></div></div>, eds. <div class="year">2017</div>, The Global Teak Study. Analysis, Evaluation and Future Potential of Teak Resources, IUFRO World Series, 36. Vienna. pp 108.</div></div></div></div><div content-id="dsy013-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kollert</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Cherubini</div> <div class="given-names">L.</div></div></div> <div class="year">2012</div>, Teak resources and market assessment 2010, FAO Planted Forests and Trees Working Paper, FP/47/E, Rome.</div></div></div></div><div content-id="dsy013-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deb</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Phinn</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Butt</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Mcalpine</div> <div class="given-names">C. A.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Climatic-induced shifts in the distribution of teak (<em>Tectona grandis</em>) in tropical Asia: implications for forest management and planning</div>, <div class="source">Environ. Manag</div> ., <div class="volume">60</div>, <div class="fpage">422</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Climatic-induced%20shifts%20in%20the%20distribution%20of%20teak%20(Tectona%20grandis)%20in%20tropical%20Asia%3A%20implications%20for%20forest%20management%20and%20planning&amp;author=J.%20C.%20Deb&amp;author=S.%20Phinn&amp;author=N.%20Butt&amp;author=C.%20A.%20Mcalpine&amp;publication_year=2017&amp;journal=Environ.%20Manag&amp;volume=60&amp;pages=422-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00267-017-0884-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00267-017-0884-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00267-017-0884-6"> </span></div> </div></div></div></div></div><div content-id="dsy013-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">O. K.</div></div>, <div class="name string-name"><div class="surname">Changtragoon</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ponoy</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lopez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Richard</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kjær</div> <div class="given-names">E. D.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Worldwide translocation of teak—origin of landraces and present genetic base</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">13</div>, <div class="fpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Worldwide%20translocation%20of%20teak%E2%80%94origin%20of%20landraces%20and%20present%20genetic%20base&amp;author=O.%20K.%20Hansen&amp;author=S.%20Changtragoon&amp;author=B.%20Ponoy&amp;author=J.%20Lopez&amp;author=J.%20Richard&amp;author=E.%20D.%20Kj%C3%A6r&amp;publication_year=2017&amp;journal=Tree%20Genet.%20Genomes&amp;volume=13&amp;pages=87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-017-1170-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-017-1170-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-017-1170-8"> </span></div> </div></div></div></div></div><div content-id="dsy013-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Keiding</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wellendorf</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lauridsen</div> <div class="given-names">E. B.</div></div></div> <div class="year">1986</div>, <div class="source">Evaluation of an International Series of Teak Provenance Trials</div> . 
<div class="publisher-loc">Humlebaek</div>: 
<div class="publisher-name">Danida Forest Seed Centre</div>.</div></div></div></div><div content-id="dsy013-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kjaer</div> <div class="given-names">E. D.</div></div>, <div class="name string-name"><div class="surname">Lauridsen</div> <div class="given-names">E. B.</div></div>, <div class="name string-name"><div class="surname">Wellendorf</div> <div class="given-names">H.</div></div></div> <div class="year">1995</div>, <div class="source">Second Evaluation of an International Series of Teak Provenance Trails</div> . 
<div class="publisher-loc">Humlebaek</div>: 
<div class="publisher-name">Danida Forest Seed Centre</div>.</div></div></div></div><div content-id="dsy013-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chaix</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Monteuuis</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Genetic variation in major phenotypic traits among diverse genetic origins of teak (<em>Tectona grandis</em> L.f.) planted in Taliwas, Sabah, East Malaysia</div>, <div class="source">Ann. For. Sci</div> ., <div class="volume">68</div>, <div class="fpage">1015</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20major%20phenotypic%20traits%20among%20diverse%20genetic%20origins%20of%20teak%20(Tectona%20grandis%20L.f.)%20planted%20in%20Taliwas%2C%20Sabah%2C%20East%20Malaysia&amp;author=G.%20Chaix&amp;author=O.%20Monteuuis&amp;author=C.%20Garcia&amp;publication_year=2011&amp;journal=Ann.%20For.%20Sci&amp;volume=68&amp;pages=1015-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13595-011-0109-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13595-011-0109-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13595-011-0109-8"> </span></div> </div></div></div></div></div><div content-id="dsy013-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Monteuuis</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Goh</div> <div class="given-names">D. K. S.</div></div>, <div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Alloysius</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gidiman</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bacilieri</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chaix</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genetic variation of growth and tree quality traits among 42 diverse genetic origins of <em>Tectona grandis</em> planted under humid tropical conditions in Sabah, East Malaysia</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">7</div>, <div class="fpage">1263</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20of%20growth%20and%20tree%20quality%20traits%20among%2042%20diverse%20genetic%20origins%20of%20Tectona%20grandis%20planted%20under%20humid%20tropical%20conditions%20in%20Sabah%2C%20East%20Malaysia&amp;author=O.%20Monteuuis&amp;author=D.%20K.%20S.%20Goh&amp;author=C.%20Garcia&amp;author=D.%20Alloysius&amp;author=J.%20Gidiman&amp;author=R.%20Bacilieri&amp;author=G.%20Chaix&amp;publication_year=2011&amp;journal=Tree%20Genet.%20Genomes&amp;volume=7&amp;pages=1263-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-011-0411-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-011-0411-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-011-0411-5"> </span></div> </div></div></div></div></div><div content-id="dsy013-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Callister</div> <div class="given-names">A. N.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genetic parameters and correlations between stem size, forking, and flowering in teak (<em>Tectona grandis</em>)</div>, <div class="source">Can. J. For. Res</div> ., <div class="volume">43</div>, <div class="fpage">1145</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20parameters%20and%20correlations%20between%20stem%20size%2C%20forking%2C%20and%20flowering%20in%20teak%20(Tectona%20grandis)&amp;author=A.%20N.%20Callister&amp;publication_year=2013&amp;journal=Can.%20J.%20For.%20Res&amp;volume=43&amp;pages=1145-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/cjfr-2013-0226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fcjfr-2013-0226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fcjfr-2013-0226"> </span></div> </div></div></div></div></div><div content-id="dsy013-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Monteuuis</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Goh</div> <div class="given-names">D. K. S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Field growth performances of teak genotypes of different ages clonally produced by rooted cuttings, <em>in vitro</em> microcuttings, and meristem culture</div>, <div class="source">Can. J. For. Res</div> ., <div class="volume">45</div>, <div class="fpage">9</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Field%20growth%20performances%20of%20teak%20genotypes%20of%20different%20ages%20clonally%20produced%20by%20rooted%20cuttings%2C%20in%20vitro%20microcuttings%2C%20and%20meristem%20culture&amp;author=O.%20Monteuuis&amp;author=D.%20K.%20S.%20Goh&amp;publication_year=2015&amp;journal=Can.%20J.%20For.%20Res&amp;volume=45&amp;pages=9-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/cjfr-2014-0339" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fcjfr-2014-0339" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fcjfr-2014-0339"> </span></div> </div></div></div></div></div><div content-id="dsy013-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gunaga</div> <div class="given-names">R. P.</div></div>, <div class="name string-name"><div class="surname">Vasudeva</div> <div class="given-names">R.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Variation in flowering phenology in a clonal seed orchard of teak</div>, <div class="source">J. Tree Sci</div> ., <div class="volume">21</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.</div></div></div></div><div content-id="dsy013-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nicodemus</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nagarajan</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Narayanan</div> <div class="given-names">C.</div></div></div> <div class="year">2005</div>, RAPD variation in Indian teak populations and its implications for breeding and conservation. In: <div class="person-group"><div class="name string-name"><div class="surname">Bhat</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Nair</div> <div class="given-names">K.K.N.</div></div>, <div class="name string-name"><div class="surname">Bhat</div> <div class="given-names">K.V.</div></div>, <div class="name string-name"><div class="surname">Muralidharan</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">J.K.</div></div></div> (eds.), <div class="source">Quality Timber Products of Teak from Sustainable Forest Management. Kerala Forest Research Institute</div> , 
<div class="publisher-name">Yokohama: India and International Tropical Timber Organization</div>, pp. <div class="fpage">321</div>–<div class="lpage">30</div>.</div></div></div></div><div content-id="dsy013-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shrestha</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">Volkaert</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Straeten</div> <div class="given-names">D. V. D.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Assessment of genetic diversity in <em>Tectona grandis</em> using amplified fragment length polymorphism markers</div>, <div class="source">Can. J. For. Res</div> ., <div class="volume">35</div>, <div class="fpage">1017</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20in%20Tectona%20grandis%20using%20amplified%20fragment%20length%20polymorphism%20markers&amp;author=M.%20K.%20Shrestha&amp;author=H.%20Volkaert&amp;author=D.%20V.%20D.%20Straeten&amp;publication_year=2005&amp;journal=Can.%20J.%20For.%20Res&amp;volume=35&amp;pages=1017-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/x05-033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fx05-033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fx05-033"> </span></div> </div></div></div></div></div><div content-id="dsy013-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sreekanth</div> <div class="given-names">P. M.</div></div>, <div class="name string-name"><div class="surname">Balasundaran</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nazeem</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Suma</div> <div class="given-names">T. B.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genetic diversity of nine natural <em>Tectona grandis</em> L.f. populations of the Western Ghats in Southern India</div>, <div class="source">Conserv. Genet</div> ., <div class="volume">13</div>, <div class="fpage">1409</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20nine%20natural%20Tectona%20grandis%20L.f.%20populations%20of%20the%20Western%20Ghats%20in%20Southern%20India&amp;author=P.%20M.%20Sreekanth&amp;author=M.%20Balasundaran&amp;author=P.%20A.%20Nazeem&amp;author=T.%20B.%20Suma&amp;publication_year=2012&amp;journal=Conserv.%20Genet&amp;volume=13&amp;pages=1409-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10592-012-0383-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10592-012-0383-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10592-012-0383-5"> </span></div> </div></div></div></div></div><div content-id="dsy013-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fofana</div> <div class="given-names">I. J.</div></div>, <div class="name string-name"><div class="surname">Ofori</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Poitel</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Verhaegen</div> <div class="given-names">D.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Diversity and genetic structure of teak (<em>Tectona grandis</em>L.f.) in its natural range using DNA microsatellite markers</div>, <div class="source">New Forests</div> , <div class="volume">37</div>, <div class="fpage">175</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20and%20genetic%20structure%20of%20teak%20(Tectona%20grandisL.f.)%20in%20its%20natural%20range%20using%20DNA%20microsatellite%20markers&amp;author=I.%20J.%20Fofana&amp;author=D.%20Ofori&amp;author=M.%20Poitel&amp;author=D.%20Verhaegen&amp;publication_year=2009&amp;journal=New%20Forests&amp;volume=37&amp;pages=175-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11056-008-9116-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11056-008-9116-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11056-008-9116-5"> </span></div> </div></div></div></div></div><div content-id="dsy013-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">O. K.</div></div>, <div class="name string-name"><div class="surname">Changtragoon</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ponoy</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genetic resources of teak (<em>Tectona grandis Linn. f</em>.)—strong genetic structure among natural populations</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">11</div>, <div class="fpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20resources%20of%20teak%20(Tectona%20grandis%20Linn.%20f.)%E2%80%94strong%20genetic%20structure%20among%20natural%20populations&amp;author=O.%20K.%20Hansen&amp;author=S.%20Changtragoon&amp;author=B.%20Ponoy&amp;publication_year=2015&amp;journal=Tree%20Genet.%20Genomes&amp;volume=11&amp;pages=802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-014-0802-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-014-0802-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-014-0802-5"> </span></div> </div></div></div></div></div><div content-id="dsy013-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Galeano</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Vasconcelos</div> <div class="given-names">T. S.</div></div>, <div class="name string-name"><div class="surname">Vidal</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mejia-Guerra</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">Carrer</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Large-scale transcriptional profiling of lignified tissues in <em>Tectona grandis</em></div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">15</div>, <div class="fpage">221</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20transcriptional%20profiling%20of%20lignified%20tissues%20in%20Tectona%20grandis&amp;author=E.%20Galeano&amp;author=T.%20S.%20Vasconcelos&amp;author=M.%20Vidal&amp;author=M.%20K.%20Mejia-Guerra&amp;author=H.%20Carrer&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol&amp;volume=15&amp;pages=221" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-015-0599-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-015-0599-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-015-0599-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26369560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Diningrat</div> <div class="given-names">D. S.</div></div>, <div class="name string-name"><div class="surname">Widiyanto</div> <div class="given-names">S. M.</div></div>, <div class="name string-name"><div class="surname">Pancoro</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Transcriptome of teak (<em>Tectona grandis</em>, L.f) in vegetative to generative stages development</div>, <div class="source">J. Plant Sci</div> ., <div class="volume">10</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20of%20teak%20(Tectona%20grandis%2C%20L.f)%20in%20vegetative%20to%20generative%20stages%20development&amp;author=D.%20S.%20Diningrat&amp;author=S.%20M.%20Widiyanto&amp;author=A.%20Pancoro&amp;publication_year=2015&amp;journal=J.%20Plant%20Sci&amp;volume=10&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3923/jps.2015.1.14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3923%2Fjps.2015.1.14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3923%2Fjps.2015.1.14"> </span></div> </div></div></div></div></div><div content-id="dsy013-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J. J.</div></div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J. L.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Isolation of plant DNA from fresh tissue</div>, <div class="source">Focus</div> , <div class="volume">12</div>, <div class="fpage">13</div>–<div class="lpage">5</div>.</div></div></div></div><div content-id="dsy013-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zimin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Puiu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yorke</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The MaSuRCA genome assembler</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">2669</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MaSuRCA%20genome%20assembler&amp;author=A.%20Zimin&amp;author=G.%20Mar%C3%A7ais&amp;author=D.%20Puiu&amp;author=M.%20Roberts&amp;author=S.%20Salzberg&amp;author=J.%20Yorke&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=2669-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt476" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt476" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt476"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23990416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Henkel</div> <div class="given-names">C. V.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">H. J.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.%20V.%20Henkel&amp;author=H.%20J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Efficient de novo assembly of large genomes using compressed data structures</div>, <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">549</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20large%20genomes%20using%20compressed%20data%20structures&amp;author=J.%20T.%20Simpson&amp;author=R.%20Durbin&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=549-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.126953.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.126953.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.126953.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22156294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve <em>de novo</em> gene finding</div>, <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cantarella</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">D’Agostino</div> <div class="given-names">N.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">PSR: polymorphic SSR retrieval</div>, <div class="source">BMC Res. Notes</div> , <div class="volume">8</div>,</div></div></div></div><div content-id="dsy013-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">Gibson</div> <div class="given-names">T. J.</div></div>, <div class="name string-name"><div class="surname">Plewniak</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Jeanmougin</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">D. G.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools</div>, <div class="source">Nuc. Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">4876</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20CLUSTAL_X%20windows%20interface%3A%20flexible%20strategies%20for%20multiple%20sequence%20alignment%20aided%20by%20quality%20analysis%20tools&amp;author=J.%20D.%20Thompson&amp;author=T.%20J.%20Gibson&amp;author=F.%20Plewniak&amp;author=F.%20Jeanmougin&amp;author=D.%20G.%20Higgins&amp;publication_year=1997&amp;journal=Nuc.%20Acids%20Res&amp;volume=25&amp;pages=4876-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.24.4876" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.24.4876"> </span></div> </div></div></div></div></div><div content-id="dsy013-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hall</div> <div class="given-names">T. A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT</div>, <div class="source">Nucleic Acids Symp. Ser</div> ., <div class="volume">41</div>, <div class="fpage">95</div>–<div class="lpage">8</div>.</div></div></div></div><div content-id="dsy013-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darriba</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Taboada</div> <div class="given-names">G. L.</div></div>, <div class="name string-name"><div class="surname">Doallo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">JModelTest 2: more models, new heuristics and parallel computing</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">772</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=JModelTest%202%3A%20more%20models%2C%20new%20heuristics%20and%20parallel%20computing&amp;author=D.%20Darriba&amp;author=G.%20L.%20Taboada&amp;author=R.%20Doallo&amp;author=D.%20Posada&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=772" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22847109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akaike</div> <div class="given-names">H.</div></div></div> <div class="year">1974</div>, 
<div class="article-title">A new look at statistical model identification</div>, <div class="source">IEEE Trans. Automat. Contr</div> ., <div class="volume">19</div>, <div class="fpage">716</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20look%20at%20statistical%20model%20identification&amp;author=H.%20Akaike&amp;publication_year=1974&amp;journal=IEEE%20Trans.%20Automat.%20Contr&amp;volume=19&amp;pages=716-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TAC.1974.1100705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTAC.1974.1100705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTAC.1974.1100705"> </span></div> </div></div></div></div></div><div content-id="dsy013-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ronquist</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Huelsenbeck</div> <div class="given-names">J. P.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">MRBAYES 3: bayesian phylogenetic inference under mixed models</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">1572</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MRBAYES%203%3A%20bayesian%20phylogenetic%20inference%20under%20mixed%20models&amp;author=F.%20Ronquist&amp;author=J.%20P.%20Huelsenbeck&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=1572-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg180" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg180"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12912839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Drummond</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Suchard</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Rambaut</div> <div class="given-names">A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">1969</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20phylogenetics%20with%20BEAUti%20and%20the%20BEAST%201.7&amp;author=A.%20J.%20Drummond&amp;author=M.%20A.%20Suchard&amp;author=D.%20Xie&amp;author=A.%20Rambaut&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=1969-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roy</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Lindqvist</div> <div class="given-names">C.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">New insights into evolutionary relationships within the subfamily Lamioideae (Lamiaceae) based on pentatricopeptide repeat (PPR) nuclear DNA sequences</div>, <div class="source">Amer. J. Bot</div> ., <div class="volume">102</div>, <div class="fpage">1721</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20insights%20into%20evolutionary%20relationships%20within%20the%20subfamily%20Lamioideae%20(Lamiaceae)%20based%20on%20pentatricopeptide%20repeat%20(PPR)%20nuclear%20DNA%20sequences&amp;author=T.%20Roy&amp;author=C.%20Lindqvist&amp;publication_year=2015&amp;journal=Amer.%20J.%20Bot&amp;volume=102&amp;pages=1721-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1500233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1500233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1500233"> </span></div> </div></div></div></div></div><div content-id="dsy013-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Drummond</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Ho</div> <div class="given-names">S. Y. W.</div></div>, <div class="name string-name"><div class="surname">Phillips</div> <div class="given-names">M. J.</div></div>, <div class="name string-name"><div class="surname">Rambaut</div> <div class="given-names">A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Relaxed phylogenetics and dating with confidence</div>, <div class="source">PLoS Biol</div> ., <div class="volume">4</div>, <div class="fpage">e88</div>–<div class="lpage">710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relaxed%20phylogenetics%20and%20dating%20with%20confidence&amp;author=A.%20J.%20Drummond&amp;author=S.%20Y.%20W.%20Ho&amp;author=M.%20J.%20Phillips&amp;author=A.%20Rambaut&amp;publication_year=2006&amp;journal=PLoS%20Biol&amp;volume=4&amp;pages=e88-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kurosaki</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J. B.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Elucidation of terpenoid metabolism in <em>Scoparia dulcis</em> by RNA-seq analysis</div>, <div class="source">Sci. Rep</div> ., <div class="volume">7</div>, <div class="fpage">43311</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elucidation%20of%20terpenoid%20metabolism%20in%20Scoparia%20dulcis%20by%20RNA-seq%20analysis&amp;author=Y.%20Yamamura&amp;author=F.%20Kurosaki&amp;author=J.%20B.%20Lee&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=43311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep43311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep43311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep43311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28266568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Holliday</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Aitken</div> <div class="given-names">S. N.</div></div>, <div class="name string-name"><div class="surname">Cooke</div> <div class="given-names">J. E. K.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Advances in ecological genomics in forest trees and applications to genetic resources conservation and breeding</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">26</div>, <div class="fpage">706</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20ecological%20genomics%20in%20forest%20trees%20and%20applications%20to%20genetic%20resources%20conservation%20and%20breeding&amp;author=J.%20A.%20Holliday&amp;author=S.%20N.%20Aitken&amp;author=J.%20E.%20K.%20Cooke&amp;publication_year=2017&amp;journal=Mol.%20Ecol&amp;volume=26&amp;pages=706-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.13963" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.13963" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.13963"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27997049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohri</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Nuclear DNA amounts in some tropical hardwoods</div>, <div class="source">Caryologia</div> , <div class="volume">39</div>, <div class="fpage">303</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20amounts%20in%20some%20tropical%20hardwoods&amp;author=D.%20Ohri&amp;author=A.%20Kumar&amp;publication_year=1986&amp;journal=Caryologia&amp;volume=39&amp;pages=303-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/00087114.1986.10797792" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F00087114.1986.10797792" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F00087114.1986.10797792"> </span></div> </div></div></div></div></div><div content-id="dsy013-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ling</div> <div class="given-names">H. Q.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Draft genome of the wheat A-genome progenitor <em>Triticum urartu</em></div>, <div class="source">Nature</div> , <div class="volume">496</div>, <div class="fpage">87</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20of%20the%20wheat%20A-genome%20progenitor%20Triticum%20urartu&amp;author=H.%20Q.%20Ling&amp;author=S.%20Zhao&amp;author=D.%20Liu&amp;author=J.%20Wang&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=87-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11997"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23535596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Britten</div> <div class="given-names">R. J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Transposable element insertions have strongly affected human evolution, <em>Proc</em></div>. <div class="source">Natl. Acad. Sci. U. S. A</div> ., <div class="volume">107</div>, <div class="fpage">19945</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20element%20insertions%20have%20strongly%20affected%20human%20evolution%2C%20Proc&amp;author=R.%20J.%20Britten&amp;publication_year=2010&amp;journal=Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=107&amp;pages=19945-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1014330107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1014330107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1014330107"> </span></div> </div></div></div></div></div><div content-id="dsy013-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mehrotra</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Goyal</div> <div class="given-names">V.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Repetitive sequences in plant nuclear DNA: types, distribution, evolution and function</div>, <div class="source">Genomics Proteomics Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">164</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repetitive%20sequences%20in%20plant%20nuclear%20DNA%3A%20types%2C%20distribution%2C%20evolution%20and%20function&amp;author=S.%20Mehrotra&amp;author=V.%20Goyal&amp;publication_year=2014&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=12&amp;pages=164-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gpb.2014.07.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gpb.2014.07.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gpb.2014.07.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25132181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dover</div> <div class="given-names">G. A.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Molecular drive in multigene families: how biological novelties arise, spread and are assimilated</div>, <div class="source">Trends Genet</div> ., <div class="volume">2</div>, <div class="fpage">159</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20drive%20in%20multigene%20families%3A%20how%20biological%20novelties%20arise%2C%20spread%20and%20are%20assimilated&amp;author=G.%20A.%20Dover&amp;publication_year=1986&amp;journal=Trends%20Genet&amp;volume=2&amp;pages=159-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9525(86)90211-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9525(86)90211-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9525(86)90211-8"> </span></div> </div></div></div></div></div><div content-id="dsy013-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Analysis of the genome sequence of the medicinal plant</div>, <div class="source">Salvia Miltiorrhiza, Mol. Plant</div> , <div class="volume">9</div>, <div class="fpage">949</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20genome%20sequence%20of%20the%20medicinal%20plant&amp;author=H.%20Xu&amp;author=J.%20Song&amp;author=H.%20Luo&amp;publication_year=2016&amp;journal=Salvia%20Miltiorrhiza%2C%20Mol.%20Plant&amp;volume=9&amp;pages=949-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molp.2016.03.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2016.03.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molp.2016.03.010"> </span></div> </div></div></div></div></div><div content-id="dsy013-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vining</div> <div class="given-names">K. J.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">S. R.</div></div>, <div class="name string-name"><div class="surname">Ahkami</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Draft genome sequence of <em>Mentha longifolia</em> and development of resources for mint cultivar improvement</div>, <div class="source">Mol. Plant</div> ., <div class="volume">10</div>, <div class="fpage">323</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Mentha%20longifolia%20and%20development%20of%20resources%20for%20mint%20cultivar%20improvement&amp;author=K.%20J.%20Vining&amp;author=S.%20R.%20Johnson&amp;author=A.%20Ahkami&amp;publication_year=2017&amp;journal=Mol.%20Plant&amp;volume=10&amp;pages=323-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molp.2016.10.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2016.10.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molp.2016.10.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27867107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rastogi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kalra</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Unravelling the genome of Holy basil: an “incomparable” “elixir of life” of traditional Indian medicine</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">413</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unravelling%20the%20genome%20of%20Holy%20basil%3A%20an%20%E2%80%9Cincomparable%E2%80%9D%20%E2%80%9Celixir%20of%20life%E2%80%9D%20of%20traditional%20Indian%20medicine&amp;author=S.%20Rastogi&amp;author=A.%20Kalra&amp;author=V.%20Gupta&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=413" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1640-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1640-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1640-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26017011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Upadhyay</div> <div class="given-names">A. K.</div></div>, <div class="name string-name"><div class="surname">Chacko</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Gandhimathi</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genome sequencing of herb Tulsi (<em>Ocimum tenuiflorum</em>) unravels key genes behind its strong medicinal properties</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">15</div>, <div class="fpage">212</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20of%20herb%20Tulsi%20(Ocimum%20tenuiflorum)%20unravels%20key%20genes%20behind%20its%20strong%20medicinal%20properties&amp;author=A.%20K.%20Upadhyay&amp;author=A.%20R.%20Chacko&amp;author=A.%20Gandhimathi&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol&amp;volume=15&amp;pages=212" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-015-0562-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-015-0562-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-015-0562-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26315624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weiss-Schneeweiss</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Leitch</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Jamie</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jang</div> <div class="given-names">T. S.</div></div>, <div class="name string-name"><div class="surname">Macas</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, Employing next generation sequencing to explore the repeat landscape of the plant genome. In: <div class="person-group"><div class="name string-name"><div class="surname">Hörandl</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Appelhans</div> <div class="given-names">M.</div></div></div> (eds). <div class="source">Next Generation Sequencing in Plant Systematic, Regnum Vegetabile 157</div> . 
<div class="publisher-loc">Königstein, Germany</div>: 
<div class="publisher-name">Koeltz Scientific Books</div>, pp. <div class="fpage">1</div>–<div class="lpage">25</div>.</div></div></div></div><div content-id="dsy013-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Neale</div> <div class="given-names">D. B.</div></div>, <div class="name string-name"><div class="surname">Wegrzyn</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Stevens</div> <div class="given-names">K. A.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies</div>, <div class="source">Genome Biol.</div>  , <div class="volume">15</div>, <div class="fpage">R59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Decoding%20the%20massive%20genome%20of%20loblolly%20pine%20using%20haploid%20DNA%20and%20novel%20assembly%20strategies&amp;author=D.%20B.%20Neale&amp;author=J.%20L.%20Wegrzyn&amp;author=K.%20A.%20Stevens&amp;publication_year=2014&amp;journal=Genome%20Biol.&amp;volume=15&amp;pages=R59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-3-r59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-3-r59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-3-r59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24647006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parween</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nawaz</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Roy</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">An advanced draft genome assembly of a desi type chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">12806</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20advanced%20draft%20genome%20assembly%20of%20a%20desi%20type%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=S.%20Parween&amp;author=K.%20Nawaz&amp;author=R.%20Roy&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=12806" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep12806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep12806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep12806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26259924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Refulio-Rodriguez</div> <div class="given-names">N. F.</div></div>, <div class="name string-name"><div class="surname">Olmstead</div> <div class="given-names">R. G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Phylogeny of Lamiidae</div>, <div class="source">Am. J. Bot</div> ., <div class="volume">101</div>, <div class="fpage">287</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20Lamiidae&amp;author=N.%20F.%20Refulio-Rodriguez&amp;author=R.%20G.%20Olmstead&amp;publication_year=2014&amp;journal=Am.%20J.%20Bot&amp;volume=101&amp;pages=287-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1300394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1300394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1300394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24509797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Francisco</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Rodne</div> <div class="given-names">Y. L.</div></div>, <div class="name string-name"><div class="surname">Rosa</div> <div class="given-names">M. V.</div></div>, <div class="name string-name"><div class="surname">Clara</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Jose</div> <div class="given-names">M. G. M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Bioactive apocarotenoids from <em>Tectona grandis</em></div>, <div class="source">Phytochemistry</div> , <div class="volume">69</div>, <div class="fpage">2708</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioactive%20apocarotenoids%20from%20Tectona%20grandis&amp;author=A.%20M.%20Francisco&amp;author=Y.%20L.%20Rodne&amp;author=M.%20V.%20Rosa&amp;author=N.%20Clara&amp;author=M.%20G.%20M.%20Jose&amp;publication_year=2008&amp;journal=Phytochemistry&amp;volume=69&amp;pages=2708-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.phytochem.2007.07.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.phytochem.2007.07.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.phytochem.2007.07.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18834604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lacret</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Varela</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Molinillo</div> <div class="given-names">J. M. G.</div></div>, <div class="name string-name"><div class="surname">Nogueiras</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Macías</div> <div class="given-names">F. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Tectonoelins, new norlignans from a bioactive extract of <em>Tectona grandis</em></div>, <div class="source">Phytochem. Lett</div> ., <div class="volume">5</div>, <div class="fpage">382</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tectonoelins%2C%20new%20norlignans%20from%20a%20bioactive%20extract%20of%20Tectona%20grandis&amp;author=R.%20Lacret&amp;author=R.%20M.%20Varela&amp;author=J.%20M.%20G.%20Molinillo&amp;author=C.%20Nogueiras&amp;author=F.%20A.%20Mac%C3%ADas&amp;publication_year=2012&amp;journal=Phytochem.%20Lett&amp;volume=5&amp;pages=382-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.phytol.2012.03.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.phytol.2012.03.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.phytol.2012.03.008"> </span></div> </div></div></div></div></div><div content-id="dsy013-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Celedon</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Bohlmann</div> <div class="given-names">J.</div></div></div> <div class="year">2018</div>, 
<div class="article-title">An extended model of heartwood secondary metabolism informed by functional genomics</div>, <div class="source">Tree Physiol</div> ., <div class="volume">14</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsy013-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Toth</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Gaspari</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Microsatellites in different eukaryotic genome: survey and analysis, <em>Genome</em>,</div> <div class="source">Res</div> ., <div class="volume">10</div>, <div class="fpage">967</div>–<div class="lpage">981</div>.</div></div></div></div><div content-id="dsy013-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vieira</div> <div class="given-names">M. L. C.</div></div>, <div class="name string-name"><div class="surname">Santini</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Diniz</div> <div class="given-names">A. L.</div></div>, <div class="name string-name"><div class="surname">Munhoz</div> <div class="given-names">C. D. F.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Microsatellite markers: what they mean and why they are so useful</div>, <div class="source">Genet. Mol. Biol</div> ., <div class="volume">39</div>, <div class="fpage">312</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%3A%20what%20they%20mean%20and%20why%20they%20are%20so%20useful&amp;author=M.%20L.%20C.%20Vieira&amp;author=L.%20Santini&amp;author=A.%20L.%20Diniz&amp;author=C.%20D.%20F.%20Munhoz&amp;publication_year=2016&amp;journal=Genet.%20Mol.%20Biol&amp;volume=39&amp;pages=312-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/1678-4685-GMB-2016-0027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2F1678-4685-GMB-2016-0027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2F1678-4685-GMB-2016-0027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27561112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pullaiah</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sri Rama Murthy</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Karuppusamy</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, <div class="source">Flora of Eastern Ghats</div> , <div class="volume">Vol. 3</div>. 
<div class="publisher-loc">New Delhi</div>: 
<div class="publisher-name">Regency Publications</div>.</div></div></div></div><div content-id="dsy013-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cantino</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Harley</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Wagstaff</div> <div class="given-names">S. J.</div></div></div> <div class="year">1992</div>, Genera of Labiatae: status and classification. In: <div class="person-group"><div class="name string-name"><div class="surname">Harley</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Reynolds</div> <div class="given-names">T.</div></div></div> (eds). <div class="source">Advances in Labiatae Science</div> , 
<div class="publisher-loc">Kew, UK</div>: 
<div class="publisher-name">Key Botanic Garden</div>, pp. <div class="fpage">511</div>–<div class="lpage">22</div>.</div></div></div></div><div content-id="dsy013-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cantino</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Olmstead</div> <div class="given-names">R. G.</div></div>, <div class="name string-name"><div class="surname">Wagstaff</div> <div class="given-names">S. J.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">A comparison of phylogenetic nomenclature with the current system: a botanical case study, <em>Syst.</em></div> <div class="source">Biol</div> ., <div class="volume">46</div>, <div class="fpage">313</div>–<div class="lpage">31</div>.</div></div></div></div><div content-id="dsy013-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wagstaff</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Olmstead</div> <div class="given-names">R. G.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Phylogeny of Labiatae and Verbenaceae inferred from <em>rbcL</em> sequences</div>, <div class="source">Syst. Bot</div> ., <div class="volume">22</div>, <div class="fpage">165</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20Labiatae%20and%20Verbenaceae%20inferred%20from%20rbcL%20sequences&amp;author=S.%20J.%20Wagstaff&amp;author=R.%20G.%20Olmstead&amp;publication_year=1997&amp;journal=Syst.%20Bot&amp;volume=22&amp;pages=165-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2419684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2419684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2419684"> </span></div> </div></div></div></div></div><div content-id="dsy013-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wagstaff</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Hickerson</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Spangler</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Reeves</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Olmstead</div> <div class="given-names">R. G.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Phylogeny in Labiatae s. l., inferred from cpDNA sequences</div>, <div class="source">Plant Syst. Evol</div> ., <div class="volume">209</div>, <div class="fpage">265</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20in%20Labiatae%20s.%20l.%2C%20inferred%20from%20cpDNA%20sequences&amp;author=S.%20J.%20Wagstaff&amp;author=L.%20Hickerson&amp;author=R.%20Spangler&amp;author=P.%20A.%20Reeves&amp;author=R.%20G.%20Olmstead&amp;publication_year=1998&amp;journal=Plant%20Syst.%20Evol&amp;volume=209&amp;pages=265-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00985232" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00985232" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00985232"> </span></div> </div></div></div></div></div><div content-id="dsy013-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alvarenga</div> <div class="given-names">S. A. V.</div></div>, <div class="name string-name"><div class="surname">Gastmans</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Rodrigues</div> <div class="given-names">G. D. V.</div></div>, <div class="name string-name"><div class="surname">Moreno</div> <div class="given-names">P. R. H.</div></div>, <div class="name string-name"><div class="surname">Emerenciano</div> <div class="given-names">V. D. P.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">A computer-assisted approach for chemotaxonomic studies - diterpenes in Lamiaceae</div>, <div class="source">Phytochemistry</div> , <div class="volume">56</div>, <div class="fpage">583</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20computer-assisted%20approach%20for%20chemotaxonomic%20studies%20-%20diterpenes%20in%20Lamiaceae&amp;author=S.%20A.%20V.%20Alvarenga&amp;author=J.%20P.%20Gastmans&amp;author=G.%20D.%20V.%20Rodrigues&amp;author=P.%20R.%20H.%20Moreno&amp;author=V.%20D.%20P.%20Emerenciano&amp;publication_year=2001&amp;journal=Phytochemistry&amp;volume=56&amp;pages=583-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0031-9422(00)00424-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0031-9422(00)00424-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0031-9422(00)00424-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11281136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bramley</div> <div class="given-names">G. L. C.</div></div>, <div class="name string-name"><div class="surname">Forest</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">De Kok</div> <div class="given-names">R. P. J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Troublesome tropical mints: re-examining generic limits of <em>Vitex</em> and relations (Lamiaceae) in South East Asia</div>, <div class="source">Taxon</div> , <div class="volume">58</div>, <div class="fpage">500</div>–<div class="lpage">10</div>.</div></div></div></div><div content-id="dsy013-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morton</div> <div class="given-names">B. R.</div></div>, <div class="name string-name"><div class="surname">Levin</div> <div class="given-names">J. A.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">The atypical codon usage of the plant <em>psbA</em> gene may be the remnant of an ancestral bias</div>, <div class="source">Proc. Nat. Aca. Sci. U.S.A</div> ., <div class="volume">94</div>, <div class="fpage">11434</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20atypical%20codon%20usage%20of%20the%20plant%20psbA%20gene%20may%20be%20the%20remnant%20of%20an%20ancestral%20bias&amp;author=B.%20R.%20Morton&amp;author=J.%20A.%20Levin&amp;publication_year=1997&amp;journal=Proc.%20Nat.%20Aca.%20Sci.%20U.S.A&amp;volume=94&amp;pages=11434-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.21.11434" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.21.11434" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.21.11434"> </span></div> </div></div></div></div></div><div content-id="dsy013-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bendiksby</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Brysting</div> <div class="given-names">A. K.</div></div>, <div class="name string-name"><div class="surname">Thorbek</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gussarov</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Ryding</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Molecular phylogeny and taxonomy of genus <em>Lamium</em> L (Lamiaceae): Disentangling origins of presumed allotetraploids</div>, <div class="source">Taxon</div> , <div class="volume">60</div>, <div class="fpage">986</div>–<div class="lpage">1000</div>.,</div></div></div></div><div content-id="dsy013-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xiang</div> <div class="given-names">C.-L.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Scheen</div> <div class="given-names">A.-C.</div></div>, <div class="name string-name"><div class="surname">Cantino</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Funamoto</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Peng</div> <div class="given-names">H.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Molecular phylogenetics of <em>Chelonopsis</em> (Lamiaceae: gomphostemmateae) as inferred from nuclear and plastid DNA and morphology</div>, <div class="source">Taxon</div> , <div class="volume">62</div>, <div class="fpage">375</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogenetics%20of%20Chelonopsis%20(Lamiaceae%3A%20gomphostemmateae)%20as%20inferred%20from%20nuclear%20and%20plastid%20DNA%20and%20morphology&amp;author=C.-L.%20Xiang&amp;author=Q.%20Zhang&amp;author=A.-C.%20Scheen&amp;author=P.%20D.%20Cantino&amp;author=T.%20Funamoto&amp;author=H.%20Peng&amp;publication_year=2013&amp;journal=Taxon&amp;volume=62&amp;pages=375-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.12705/622.11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.12705%2F622.11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.12705%2F622.11"> </span></div> </div></div></div></div></div><div content-id="dsy013-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jenks</div> <div class="given-names">A. A.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">J. B.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S. C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Evolution and origins of the Mazatec hallucinogenic sage, <em>Salvia divinorum</em> (Lamiaceae): a molecular phylogenetic approach</div>, <div class="source">J. Plant Res</div> ., <div class="volume">124</div>, <div class="fpage">593</div>–<div class="lpage">600</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20origins%20of%20the%20Mazatec%20hallucinogenic%20sage%2C%20Salvia%20divinorum%20(Lamiaceae)%3A%20a%20molecular%20phylogenetic%20approach&amp;author=A.%20A.%20Jenks&amp;author=J.%20B.%20Walker&amp;author=S.%20C.%20Kim&amp;publication_year=2011&amp;journal=J.%20Plant%20Res&amp;volume=124&amp;pages=593-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10265-010-0394-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10265-010-0394-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10265-010-0394-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21125306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conn</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">T. C.</div></div>, <div class="name string-name"><div class="surname">Henwood</div> <div class="given-names">M. J.</div></div>, <div class="name string-name"><div class="surname">Proft</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Circumscription and phylogenetic relationships of <em>Prostanthera densa</em> and <em>P. marifolia</em> (Lamiaceae)</div>, <div class="source">Telopea</div> , <div class="volume">15</div>, <div class="fpage">149</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circumscription%20and%20phylogenetic%20relationships%20of%20Prostanthera%20densa%20and%20P.%20marifolia%20(Lamiaceae)&amp;author=B.%20J.%20Conn&amp;author=T.%20C.%20Wilson&amp;author=M.%20J.%20Henwood&amp;author=K.%20Proft&amp;publication_year=2013&amp;journal=Telopea&amp;volume=15&amp;pages=149-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7751/telopea2013019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7751%2Ftelopea2013019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7751%2Ftelopea2013019"> </span></div> </div></div></div></div></div><div content-id="dsy013-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Neubig</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Whitten</div> <div class="given-names">W. M.</div></div>, <div class="name string-name"><div class="surname">Carlsward</div> <div class="given-names">B. S.</div></div>, <div class="name string-name"><div class="surname">Blanco</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Endara</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">N. H.</div></div>, <div class="name string-name"><div class="surname">Moore</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Phylogenetic utility of <em>ycf1</em> in orchids: a plastid gene more variable than <em>matK</em></div>, <div class="source">Plant Syst. Evol.</div> , <div class="volume">277</div>, <div class="fpage">75</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20utility%20of%20ycf1%20in%20orchids%3A%20a%20plastid%20gene%20more%20variable%20than%20matK&amp;author=K.%20M.%20Neubig&amp;author=W.%20M.%20Whitten&amp;author=B.%20S.%20Carlsward&amp;author=M.%20A.%20Blanco&amp;author=L.%20Endara&amp;author=N.%20H.%20Williams&amp;author=M.%20Moore&amp;publication_year=2009&amp;journal=Plant%20Syst.%20Evol.&amp;volume=277&amp;pages=75-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00606-008-0105-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00606-008-0105-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00606-008-0105-0"> </span></div> </div></div></div></div></div><div content-id="dsy013-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dong</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title"><em>ycf</em>1, the most promising plastid DNA barcode of land plants</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">8348</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ycf1%2C%20the%20most%20promising%20plastid%20DNA%20barcode%20of%20land%20plants&amp;author=W.%20Dong&amp;author=C.%20Xu&amp;author=C.%20Li&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=8348" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep08348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep08348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep08348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25672218" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dong</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Highly variable chloroplast markers for evaluating plant phylogeny at low taxonomic levels and for DNA barcoding</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e35071</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20variable%20chloroplast%20markers%20for%20evaluating%20plant%20phylogeny%20at%20low%20taxonomic%20levels%20and%20for%20DNA%20barcoding&amp;author=W.%20Dong&amp;author=J.%20Liu&amp;author=J.%20Yu&amp;author=L.%20Wang&amp;author=S.%20Zhou&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e35071" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0035071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0035071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0035071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22511980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy013-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Briquet</div> <div class="given-names">J.</div></div></div> <div class="year">1897</div>, <em>Verbenaceae</em>. In: <div class="person-group"><div class="name string-name"><div class="surname">Engler</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Prantl</div> <div class="given-names">K.</div></div></div> (eds). <div class="source">Die Naturlichen Pflanzenfamilien Teil 4.</div>  
<div class="publisher-loc">Leipzig</div>: 
<div class="publisher-name">Engelmann</div>, pp. <div class="fpage">132</div>–<div class="lpage">182</div>.</div></div></div></div><div content-id="dsy013-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Melchior</div> <div class="given-names">H.</div></div></div> <div class="year">1964</div>, <div class="source">A. Engler’s Syllabus Der Pflanzenfamilien</div> , <div class="volume">vol. 2</div>. 
<div class="publisher-loc">Berlin</div>: 
<div class="publisher-name">Borntraeger</div>, pp. <div class="fpage">666</div>.</div></div></div></div><div content-id="dsy013-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moldenke</div> <div class="given-names">H. N.</div></div></div> <div class="year">1975</div>, 
<div class="article-title">Notes on new and noteworthy plants LXXVII</div>, <div class="source">Phytologia</div> , <div class="volume">31</div>, <div class="fpage">28</div>.</div></div></div></div><div content-id="dsy013-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takhtajan</div> <div class="given-names">A.</div></div></div> <div class="year">1983</div>, 
<div class="article-title">Outline of the classification of flowering plants (Magnoliophyta)</div>, <div class="source">Brittonia</div> , <div class="volume">35</div>, <div class="fpage">254</div>–<div class="lpage">359</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Outline%20of%20the%20classification%20of%20flowering%20plants%20(Magnoliophyta)&amp;author=A.%20Takhtajan&amp;publication_year=1983&amp;journal=Brittonia&amp;volume=35&amp;pages=254-359" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02831474" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02831474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02831474"> </span></div> </div></div></div></div></div><div content-id="dsy013-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy013-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Olmstead</div> <div class="given-names">R.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Two new subfamilies in Lamiaceae</div>, <div class="source">Phytotaxa</div> , <div class="volume">313</div>, <div class="fpage">222</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20new%20subfamilies%20in%20Lamiaceae&amp;author=B.%20Li&amp;author=R.%20Olmstead&amp;publication_year=2017&amp;journal=Phytotaxa&amp;volume=313&amp;pages=222-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.11646/phytotaxa.313.2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.11646%2Fphytotaxa.313.2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.11646%2Fphytotaxa.313.2"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5003450&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s1-s3_s7_s9-s10.doc?Expires=2147483647&amp;Signature=eTnP40iJtj5DxIBCwFiS3ZARptTtP-uv4EyApY2QfXgyVe9eAAfNsEIQfnSidLxB6kPKdZFUo3RvZxN363eis1NiSZ3yXCiy41hYX1nSnVgJtOjuNrP4XP6vKSG4-4HG6YiJ~xnfiRGn-y6QJuX2jXisawegWAjfWWrKA80ww~mKby~8Y6QyovAheKJvTo4p9fNGgFED2sx1IU5D~sp25AE6ijNy81cSsfuNSY~05I7cflkbvQliBC2uMFpWMyn139znnYjtPUV2riu-9G5RSH7vqWZfB0j-UOYjxc8QN2JuJYNTdMkmGqZnhwRaikXyuRQHVEjTaMAuu9vNfK8z-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1-S3 S7 S9-S10</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s4.xlsx?Expires=2147483647&amp;Signature=fY5sIBQchRuOnFo1DQD3KzaR2Y9Kf-lbsL-pa0DJLg2l0KguC1ZanbibWUhPYunU0Qwu5WaVhQ39uT2U5~KHM8KaS7ZUfoUtlM0rT~5Dpcds70e5TrBN9IAgdci4OHMCL6YfKN2ws5il2n5NSD3MhJiJz36k128iFA~32PDWOeLx2H5qGlA3eVW8lTWLDYoZgghhH-FHR2S6Gi1Y6jtsN165uWfzOnse9c9QILG4MYA95k~2JoAbpwAhiRtOnMYJ8UpnNe0Wi3Af~NLXogoZQXc2pX2c19K6L7SBIQewEzFviG4w6VLljg~LWnPDr2S9uuQjr7oUxPa55HJWw9G3ww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s5.xls?Expires=2147483647&amp;Signature=YLgLixRIoaSsWdnwLj~xQ5p01dXjNm0zGV3UO5EVjgXTm~c9NpXQNSspuZF15Ah~PoM4VA37PYeGTsHHczXNJgkg9Dp6b5rB-G9HYUs3oNhkPJC5qB5HYZU9zLgR4iaWqAWAevN4iybzHve3UM7tDRHUpxHeTtduBaMGuuI1MYIGRoNVqVIlCnemD8VZWFY8SeoUWQchd3KKEbDik8Rt3dhZyJQRmc1gbOMmE0KM0Z923uD8j5xIF2-JrSo1pI7gYhz3NGe9a6Q3id-LViTszadrSzHN500skShRtWPly46MpjOq0h3belUUUmUxvBXs6f8p-fVDV153cgMAjnq~GA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s6.xls?Expires=2147483647&amp;Signature=BV4CEdgvKb-Z75jOQ3WdC8Ek0mLue3yPTXgrk4AmTpRL1ft~nyh2v6jpBuRWbYRUr-eQ363VdvG3M0glXYLzWUBZVbXZDW25InAQE0Roarp-yZSLEvZ1YOuPFDAFgscASkPOubbKbfT5SZ4qgHnobMnekzJm380w43F4MWpU-qsL3oBXxex~ldMsxLJi7W2HIT5jtvP94NtN35b4dcA8MIzPE3AMalbsXXqTtbs~1sV5-4-toelEAtwj4UwOvyTgvSkMnK9~bJD-SoAUGXDJhYvRgYUj8nSfGPxghfmHz-zPvqIUqzQj0wssy8~fibpPVsLYOXr82jNoRJpeIuWnMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S6</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s8.xls?Expires=2147483647&amp;Signature=GnjmH6loFnEfvSYOeaemZXp7R3U9oEeCopFQSeLkgbBa1y78K-YrtCxZYJuZ6J0K8WdOk66MAj6zSh0Tk-m2H3pujh5VD2EZNca1H3V9b5OiYW-2wujbvaIL0RltnPTSAqaM0trcpDE3dV1LkGlniN460yJN9z6YYm4fkBPia6b52iPQrMML9Vtp3kiY5KPbvGMw4Bz6BktaKaHj9UnYwwlMwxrtBgZtobGSDRchZv3dn-qZn3htUcWRSdu1I3gsTTnRbFIm6tQvX792S7U~y3vN29yQuYpGsDSjmdaTQK60sns0BiL04r1XV8nTyiSP20iquWvjfZHhYtRSD1pMzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S8</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementary_table_s11.xlsx?Expires=2147483647&amp;Signature=GN~6EqJezzkJ6Z7YwYJyLmvzxRCpvgId-mxzaI~KDeKoamX6YL9as4xpoIbPyzWWUbvpcr~l3paO-OBxaOqAIxkrXBGJ2MtwdGx9mAQvN7XkD0mZXOAyziXHeizLIaTC7t84nuRKVzJHa-rc4tzNYLgC3wGoCyZgatj4unV5iJU0CfAvcIxQopUWlY81GSt9of~aJBc~RK58~bH6NHkWUj3v2Tjn0vD5fVZXfDugkZlcMovbwUBzoeaJHehte7P3dudkRVz8uHlGTuRlDIA9fv8KOITvQMOv9HPxV~X-xZrdmI75MgNtSpCenw7Yu-zik1Ro54Y5FJBUBz-4IuLOWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S11</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy013/1/dsy013_supplementray_table_s12_r1.xls?Expires=2147483647&amp;Signature=KAanOGnLoTCb3bfL9OZL8hyjj4X1RWQIb1-GOYq1uUcdt~GGCB2Km6~w0z~MFZfmIuD~rp0k4jOdplFfvgCrkTy0JRFp~gr3wEyGqE1HxS8t2GPLD57ecAUmgBQALIH6qzPAm4zzuWP3sajvUifOs~qnXMA6bRIhZ2VX2k3nVThwG13e5Z-l26MblS0FlrBdIyrOlw3PoB0ZLnbCXdg80eiP5a28I0Yp~xoCz3BFzmT2gjaid0bkSgH65cotD~1qWqYxZSnmF~-QcgVQ1Hr7WQRC1v4psLQdJdtHpcupbgsBaTlIqDHIxEJAGXI1AEA5UKQ4f-LkGMZai6QpmoR~iQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S12</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">7,007</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy013"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5003450"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f4%2f409%2f5003450">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F4%2F409%2F5003450" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5003450'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Draft genome of a high value tropical timber tree, Teak (Tectona grandis L. f): insights into SSR diversity, phylogeny and conservation" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy013");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f409%2f5003450">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

