

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/3/231/474863","@type":"ScholarlyArticle","name":"Alternative Splicing Regulates Targeting of Malate Dehydrogenase in  Yarrowia lipolytica ","datePublished":"2012-02-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2012-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss007","keywords":["yeast","TCA cycle","glyoxylate cycle","MDH2","intron"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/3/231/474863","author":[{"name":"Kabran, Philomène","affiliation":"1  INRA ,  UMR 1319 Micalis, F-78352 Jouy-en-Josas ,  France    2  Division of Biological Sciences ,  University of California, San Diego ,  9500 Gilman Drive, La Jolla, CA 92093 ,  USA","@type":"Person"},{"name":"Rossignol, Tristan","affiliation":"1  INRA ,  UMR 1319 Micalis, F-78352 Jouy-en-Josas ,  France","@type":"Person"},{"name":"Gaillardin, Claude","affiliation":"3  AgroParisTech ,  Micalis, F-78352 Jouy-en-Josas ,  France","@type":"Person"},{"name":"Nicaud, Jean-Marc","affiliation":"4  CNRS, Micalis, F-78352 Jouy-en-Josas ,  France","@type":"Person"},{"name":"Neuvéglise, Cécile","affiliation":"1  INRA ,  UMR 1319 Micalis, F-78352 Jouy-en-Josas ,  France","@type":"Person"}],"description":"Abstract.  Alternative pre-mRNA splicing is a major mechanism contributing to the proteome complexity of most eukaryotes, especially mammals. In less complex or","pageStart":"231","pageEnd":"244","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Alternative Splicing Regulates Targeting of Malate Dehydrogenase in  Yarrowia lipolytica ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Alternative Splicing Regulates Targeting of Malate Dehydrogenase in  Yarrowia lipolytica " />
    <meta property="og:description" content="Abstract.  Alternative pre-mRNA splicing is a major mechanism contributing to the proteome complexity of most eukaryotes, especially mammals. In less complex or" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/3/231/474863" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-02-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kabran, Philomène" /><meta name="citation_author_institution" content="INRA, UMR 1319 Micalis, F-78352 Jouy-en-Josas, France" /><meta name="citation_author_institution" content="Division of Biological Sciences, University of California, San Diego, 9500 Gilman Drive, La Jolla, CA 92093, USA" /><meta name="citation_author" content="Rossignol, Tristan" /><meta name="citation_author_institution" content="INRA, UMR 1319 Micalis, F-78352 Jouy-en-Josas, France" /><meta name="citation_author" content="Gaillardin, Claude" /><meta name="citation_author_institution" content="AgroParisTech, Micalis, F-78352 Jouy-en-Josas, France" /><meta name="citation_author" content="Nicaud, Jean-Marc" /><meta name="citation_author_institution" content="CNRS, Micalis, F-78352 Jouy-en-Josas, France" /><meta name="citation_author" content="Neuvéglise, Cécile" /><meta name="citation_author_institution" content="INRA, UMR 1319 Micalis, F-78352 Jouy-en-Josas, France" /><meta name="citation_title" content="Alternative Splicing Regulates Targeting of Malate Dehydrogenase in Yarrowia lipolytica" /><meta name="citation_firstpage" content="231" /><meta name="citation_lastpage" content="244" /><meta name="citation_doi" content="10.1093/dnares/dss007" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2012/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Detection and analysis of alternative splicing in Yarrowia lipolytica reveal structural constraints facilitating nonsense-mediated decay of intron-retaining transcripts; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R65" /><meta name="citation_reference" content="citation_title=The intronome of budding yeasts; C R Biol.;  citation_year=2011;  citation_volume=334;  citation_pages=662-70; " /><meta name="citation_reference" content="citation_title=High-density yeast-tiling array reveals previously undiscovered introns and extensive regulation of meiotic splicing; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=1522-7; " /><meta name="citation_reference" content="citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing; Science;  citation_year=2008;  citation_volume=320;  citation_pages=1344-9; " /><meta name="citation_reference" content="citation_title=The inner nuclear membrane protein Src1 associates with subtelomeric genes and alters their regulated gene expression; J. Cell Biol.;  citation_year=2008;  citation_volume=182;  citation_pages=897-910; " /><meta name="citation_reference" content="citation_title=Alternative splicing of PTC7 in Saccharomyces cerevisiae determines protein localization; Genetics;  citation_year=2009;  citation_volume=183;  citation_pages=185-94; " /><meta name="citation_reference" content="citation_title=Peroxisomal and mitochondrial carnitine acetyltransferases of Saccharomyces cerevisiae are encoded by a single gene; EMBO J.;  citation_year=1995;  citation_volume=14;  citation_pages=3472-9; " /><meta name="citation_reference" content="citation_title=Two differentially regulated mRNAs with different 5′ ends encode secreted with intracellular forms of yeast invertase; Cell;  citation_year=1982;  citation_volume=28;  citation_pages=145-54; " /><meta name="citation_reference" content="citation_title=mRNA quality control: an ancient machinery recognizes and degrades mRNAs with nonsense codons; FEBS Lett.;  citation_year=2007;  citation_volume=581;  citation_pages=2845-53; " /><meta name="citation_reference" content="citation_title=The meaning of nonsense; Trends Cell Biol.;  citation_year=2008;  citation_volume=18;  citation_pages=315-21; " /><meta name="citation_reference" content="citation_title=Malate dehydrogenases—structure and function; Gen. Physiol. Biophys.;  citation_year=2002;  citation_volume=21;  citation_pages=257-65; " /><meta name="citation_reference" content="citation_title=Global transcription analysis of Krebs tricarboxylic acid cycle mutants reveals an alternating pattern of gene expression and effects on hypoxic and oxidative genes; Mol. Biol. Cell;  citation_year=2003;  citation_volume=14;  citation_pages=958-72; " /><meta name="citation_reference" content="citation_title=Genetic and biochemical interactions involving tricarboxylic acid cycle (TCA) function using a collection of mutants defective in all TCA cycle genes; Genetics;  citation_year=1999;  citation_volume=152;  citation_pages=153-66; " /><meta name="citation_reference" content="citation_title=Isolation and expression of the gene encoding yeast mitochondrial malate dehydrogenase; J. Bacteriol.;  citation_year=1987;  citation_volume=169;  citation_pages=5157-66; " /><meta name="citation_reference" content="citation_title=Gene sequence and primary structure of mitochondrial malate dehydrogenase from Saccharomyces cerevisiae; Biochemistry;  citation_year=1988;  citation_volume=27;  citation_pages=8393-400; " /><meta name="citation_reference" content="citation_title=A central role for the peroxisomal membrane in glyoxylate cycle function; Biochim. Biophys. Acta (BBA)—Mol. Cell Res.;  citation_year=2006;  citation_volume=1763;  citation_pages=1441-52; " /><meta name="citation_reference" content="citation_title=Isolation, nucleotide sequence analysis, and disruption of the MDH2 gene from Saccharomyces cerevisiae: evidence for three isozymes of yeast malate dehydrogenase; Mol. Cell. Biol.;  citation_year=1991;  citation_volume=11;  citation_pages=370-80; " /><meta name="citation_reference" content="citation_title=Expression and function of a mislocalized form of peroxisomal malate dehydrogenase (MDH3) in yeast; J. Biol. Chem.;  citation_year=1995;  citation_volume=270;  citation_pages=21220-5; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of the yeast gene encoding the MDH3 isozyme of malate dehydrogenase; J. Biol. Chem.;  citation_year=1992;  citation_volume=267;  citation_pages=24708-15; " /><meta name="citation_reference" content="citation_title=A conserved tripeptide sorts proteins to peroxisomes; J. Cell Biol.;  citation_year=1989;  citation_volume=108;  citation_pages=1657-64; " /><meta name="citation_reference" content="citation_title=The carboxyl-terminal tripeptide Ala-Lys-Ile is essential for targeting Candida tropicalis trifunctional enzyme to yeast peroxisomes; J. Biol. Chem.;  citation_year=1991;  citation_volume=266;  citation_pages=23197-203; " /><meta name="citation_reference" content="citation_title=Peroxisome biogenesis in yeast; Mol. Microbiol.;  citation_year=1992;  citation_volume=6;  citation_pages=3455-60; " /><meta name="citation_reference" content="citation_title=Antibodies directed against a yeast carboxyl-terminal peroxisomal targeting signal specifically recognize peroxisomal proteins from various yeasts; Yeast;  citation_year=1992;  citation_volume=8;  citation_pages=721-34; " /><meta name="citation_reference" content="citation_title=Génolevures: protein families and synteny among complete hemiascomycetous yeast proteomes and genomes; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D550-4; " /><meta name="citation_reference" content="citation_title=The dimorphic fungus Yarrowia lipolytica; citation_publisher=Springer Verlag, Heidelberg; Genetics, Biochemistry and Molecular Biology of Non-Conventional Yeasts;  citation_year=1996;  citation_pages=313-88; " /><meta name="citation_reference" content="citation_title=Lipid accumulation, lipid body formation, and acyl coenzyme a oxidases of the yeast Yarrowia lipolytica; Appl. Environ. Microbiol.;  citation_year=2004;  citation_volume=70;  citation_pages=3918-24; " /><meta name="citation_reference" content="citation_title=grofit: fitting biological growth curves with R; J. Stat. Softw.;  citation_year=2010;  citation_volume=33;  citation_pages=1-21; " /><meta name="citation_reference" content="citation_title=R: a language for data analysis and graphics; J. Comput. Graph. Stat.;  citation_year=1996;  citation_volume=5;  citation_pages=299-314; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor; Molecular Cloning: A Laboratory Manual;  citation_year=1989; " /><meta name="citation_reference" content="citation_title=Molecular monitoring of wine fermentations conducted by active dry yeast strains; Appl. Environ. Microbiol.;  citation_year=1992;  citation_volume=58;  citation_pages=2948-53; " /><meta name="citation_reference" content="citation_title=Protein expression and secretion in the yeast Yarrowia lipolytica; FEMS Yeast Res.;  citation_year=2002;  citation_volume=2;  citation_pages=371-9; " /><meta name="citation_reference" content="citation_title=Multiple-copy integration in the yeast Yarrowia lipolytica; Curr. Genet.;  citation_year=1994;  citation_volume=26;  citation_pages=38-44; " /><meta name="citation_reference" content="citation_title=A versatile toolbox for PCR-based tagging of yeast genes: new fluorescent proteins, more markers and promoter substitution cassettes; Yeast;  citation_year=2004;  citation_volume=21;  citation_pages=947-62; " /><meta name="citation_reference" content="citation_title=Comparison of expression systems in the yeasts Saccharomyces cerevisiae, Hansenula polymorpha, Klyveromyces lactis, Schizosaccharomyces pombe and Yarrowia lipolytica. Cloning of two novel promoters from Yarrowia lipolytica; Yeast;  citation_year=1998;  citation_volume=14;  citation_pages=1267-83; " /><meta name="citation_reference" content="citation_title=SOAP: short oligonucleotide alignment program; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=713-4; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J Mol Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; " /><meta name="citation_reference" content="citation_title=Multiple sequence alignment with hierarchical clustering; Nucleic Acids Res.;  citation_year=1988;  citation_volume=16;  citation_pages=10881-90; " /><meta name="citation_reference" content="citation_title=The CLUSTAL_X Windows Interface: Flexible Strategies for Multiple Sequence Alignment Aided by Quality Analysis Tools; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=4876-82; " /><meta name="citation_reference" content="citation_title=Computational method to predict mitochondrially imported proteins and their targeting sequences; Eur. J. Biochem.;  citation_year=1996;  citation_volume=241;  citation_pages=779-86; " /><meta name="citation_reference" content="citation_title=Prediction of peroxisomal targeting signal 1 containing proteins from amino acid sequence; J. Mol. Biol.;  citation_year=2003;  citation_volume=328;  citation_pages=581-92; " /><meta name="citation_reference" content="citation_title=Genome evolution in yeasts; Nature;  citation_year=2004;  citation_volume=430;  citation_pages=35-44; " /><meta name="citation_reference" content="citation_title=Genome sequence of the recombinant protein production host Pichia pastoris; Nat. Biotech.;  citation_year=2009;  citation_volume=27;  citation_pages=561-6; " /><meta name="citation_reference" content="citation_title=Ashbya Genome Database 3.0: a cross-species genome and transcriptome browser for yeast biologists; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=9" /><meta name="citation_reference" content="citation_title=The membrane of peroxisomes in Saccharomyces cerevisiae is impermeable to NAD(H) and acetyl-CoA under in vivo conditions; EMBO J.;  citation_year=1995;  citation_volume=14;  citation_pages=3480-6; " /><meta name="citation_reference" content="citation_title=TCA cycle-independent acetate metabolism via the glyoxylate cycle in Saccharomyces cerevisiae; Yeast;  citation_year=2011;  citation_volume=28;  citation_pages=153-66; " /><meta name="citation_reference" content="citation_title=Mutants of Saccharomyces cerevisiae with defects in acetate metabolism: isolation and characterization of Acn- mutants; Genetics;  citation_year=1996;  citation_volume=144;  citation_pages=57-69; " /><meta name="citation_reference" content="citation_title=Functional characterization of the unconventional splicing of Yarrowia lipolytica HAC1 mRNA induced by unfolded protein response; Yeast;  citation_year=2010;  citation_volume=27;  citation_pages=443-52; " /><meta name="citation_reference" content="citation_title=Characterization of Ire1 in the yeast Yarrowia lipolytica reveals an important role for the Sls1 nucleotide exchange factor in unfolded protein response regulation; Curr. Genet.;  citation_year=2008;  citation_volume=53;  citation_pages=337-46; " /><meta name="citation_reference" content="citation_title=Metabolic effects of altering redundant targeting signals for yeast mitochondrial malate dehydrogenase; Arch. Biochem. Biophys.;  citation_year=1997;  citation_volume=344;  citation_pages=53-60; " /><meta name="citation_reference" content="citation_title=Targeting of malate synthase 1 to the peroxisomes of Saccharomyces cerevisiae cells depends on growth on oleic acid medium; Eur. J. Biochem.;  citation_year=2002;  citation_volume=269;  citation_pages=915-22; " /><meta name="citation_reference" content="citation_title=Imaging-based live cell yeast screen identifies novel factors involved in peroxisome assembly; J. Proteome Res;  citation_year=2009;  citation_volume=8;  citation_pages=20-7; " /><meta name="citation_reference" content="citation_title=Cloning of the isocitrate lyase gene (ICL1) from Yarrowia lipolytica and characterization of the deduced protein; Mol Gen Genet;  citation_year=1993;  citation_volume=241;  citation_pages=422-30; " /><meta name="citation_reference" content="citation_title=Arabidopsis peroxisomal malate dehydrogenase functions in beta-oxidation but not in the glyoxylate cycle; Plant J.;  citation_year=2007;  citation_volume=50;  citation_pages=381-90; " /><meta name="citation_reference" content="citation_title=Metabolite transport across the peroxisomal membrane; Biochem J.;  citation_year=2007;  citation_volume=401;  citation_pages=365-75; " /><meta name="citation_reference" content="citation_title=Engineering polyhydroxyalkanoate content and monomer composition in the oleaginous yeast Yarrowia lipolytica by modifying the ß-oxidation multifunctional protein; Appl. Microbiol. Biotechnol.;  citation_year=2011;  citation_volume=91;  citation_pages=1327-40; " /><meta name="citation_reference" content="citation_title=Alternative routes of acetyl-CoA synthesis identified by comparative genomic analysis: Involvement in lipid production of oleaginous yeast and fungi; citation_text=in press, doi:10.1099/mic.0.051946-0; Microbiology;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=SUS1 introns are required for efficient mRNA nuclear export in yeast; citation_text=doi:10.10.93/nar/gkr496; Nucleic Acids Res.;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Key features of the two-intron Saccharomyces cerevisiae gene SUS1 contribute to its alternative splicing; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=8612-27; " /><meta name="citation_reference" content="citation_title=Test of intron predictions reveals novel splice sites, alternatively spliced mRNAs and new introns in meiotically regulated genes of yeast; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=1700-6; " /><meta name="citation_reference" content="citation_title=Meiosis-specific RNA splicing in yeast; Cell;  citation_year=1991;  citation_volume=66;  citation_pages=1257-68; " /><meta name="citation_reference" content="citation_title=The Saccharomyces cerevisiae MER3 gene, encoding a novel helicase-like protein, is required for crossover control in meiosis; EMBO J.;  citation_year=1999;  citation_volume=18;  citation_pages=5714-23; " /><meta name="citation_reference" content="citation_title=An RNA structure involved in feedback regulation of splicing and of translation is critical for biological fitness; Proc. Natl Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=1596-600; " /><meta name="citation_reference" content="citation_title=Autoregulation of the mRNA export factor Yra1p requires inefficient splicing of its pre-mRNA; RNA;  citation_year=2006;  citation_volume=12;  citation_pages=994-1006; " /><meta name="citation_reference" content="citation_title=A third of the yeast mitochondrial proteome is dual localized: a question of evolution; Proteomics;  citation_year=2011;  citation_volume=11;  citation_pages=4468-76; " /><meta name="citation_reference" content="citation_title=Roles of multiple acyl-CoA oxidases in the routing of carbon flow towards β-oxidation and polyhydroxyalkanoate biosynthesis in Yarrowia lipolytica; FEMS Yeast Res.;  citation_year=2010;  citation_volume=10;  citation_pages=917-27; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/3/231/1283812/dss007.pdf" /><meta name="description" content="Abstract.  Alternative pre-mRNA splicing is a major mechanism contributing to the proteome complexity of most eukaryotes, especially mammals. In less complex or" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/3/231/474863" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/3/231/474863" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/3/231/474863");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss007';
    </script>
<title>
	Alternative Splicing Regulates Targeting of Malate Dehydrogenase in Yarrowia lipolytica | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./474863" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbqMATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FicmFuLCBQaGlsb23DqG5lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgVU1SIDEzMTkgTWljYWxpcywgRi03ODM1MiBKb3V5LWVuLUpvc2FzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIENhbGlmb3JuaWEsIFNhbiBEaWVnbywgOTUwMCBHaWxtYW4gRHJpdmUsIExhIEpvbGxhLCBDQSA5MjA5MywgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm9zc2lnbm9sLCBUcmlzdGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgVU1SIDEzMTkgTWljYWxpcywgRi03ODM1MiBKb3V5LWVuLUpvc2FzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYWlsbGFyZGluLCBDbGF1ZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZ3JvUGFyaXNUZWNoLCBNaWNhbGlzLCBGLTc4MzUyIEpvdXktZW4tSm9zYXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5pY2F1ZCwgSmVhbi1NYXJjIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ05SUywgTWljYWxpcywgRi03ODM1MiBKb3V5LWVuLUpvc2FzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOZXV2w6lnbGlzZSwgQ8OpY2lsZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIFVNUiAxMzE5IE1pY2FsaXMsIEYtNzgzNTIgSm91eS1lbi1Kb3NhcywgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBbHRlcm5hdGl2ZSBTcGxpY2luZyBSZWd1bGF0ZXMgVGFyZ2V0aW5nIG9mIE1hbGF0ZSBEZWh5ZHJvZ2VuYXNlIGluIFlhcnJvd2lhIGxpcG9seXRpY2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEyLzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIGFuZCBhbmFseXNpcyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiBZYXJyb3dpYSBsaXBvbHl0aWNhIHJldmVhbCBzdHJ1Y3R1cmFsIGNvbnN0cmFpbnRzIGZhY2lsaXRhdGluZyBub25zZW5zZS1tZWRpYXRlZCBkZWNheSBvZiBpbnRyb24tcmV0YWluaW5nIHRyYW5zY3JpcHRzOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1SNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW50cm9ub21lIG9mIGJ1ZGRpbmcgeWVhc3RzOyBDIFIgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzM0OyAgY2l0YXRpb25fcGFnZXM9NjYyLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWRlbnNpdHkgeWVhc3QtdGlsaW5nIGFycmF5IHJldmVhbHMgcHJldmlvdXNseSB1bmRpc2NvdmVyZWQgaW50cm9ucyBhbmQgZXh0ZW5zaXZlIHJlZ3VsYXRpb24gb2YgbWVpb3RpYyBzcGxpY2luZzsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0xNTIyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0cmFuc2NyaXB0aW9uYWwgbGFuZHNjYXBlIG9mIHRoZSB5ZWFzdCBnZW5vbWUgZGVmaW5lZCBieSBSTkEgc2VxdWVuY2luZzsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjA7ICBjaXRhdGlvbl9wYWdlcz0xMzQ0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbm5lciBudWNsZWFyIG1lbWJyYW5lIHByb3RlaW4gU3JjMSBhc3NvY2lhdGVzIHdpdGggc3VidGVsb21lcmljIGdlbmVzIGFuZCBhbHRlcnMgdGhlaXIgcmVndWxhdGVkIGdlbmUgZXhwcmVzc2lvbjsgSi4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODI7ICBjaXRhdGlvbl9wYWdlcz04OTctOTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBvZiBQVEM3IGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBkZXRlcm1pbmVzIHByb3RlaW4gbG9jYWxpemF0aW9uOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz0xODUtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcm94aXNvbWFsIGFuZCBtaXRvY2hvbmRyaWFsIGNhcm5pdGluZSBhY2V0eWx0cmFuc2ZlcmFzZXMgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGFyZSBlbmNvZGVkIGJ5IGEgc2luZ2xlIGdlbmU7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zNDcyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBkaWZmZXJlbnRpYWxseSByZWd1bGF0ZWQgbVJOQXMgd2l0aCBkaWZmZXJlbnQgNeKAsiBlbmRzIGVuY29kZSBzZWNyZXRlZCB3aXRoIGludHJhY2VsbHVsYXIgZm9ybXMgb2YgeWVhc3QgaW52ZXJ0YXNlOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTQ1LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1tUk5BIHF1YWxpdHkgY29udHJvbDogYW4gYW5jaWVudCBtYWNoaW5lcnkgcmVjb2duaXplcyBhbmQgZGVncmFkZXMgbVJOQXMgd2l0aCBub25zZW5zZSBjb2RvbnM7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTgxOyAgY2l0YXRpb25fcGFnZXM9Mjg0NS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1lYW5pbmcgb2Ygbm9uc2Vuc2U7IFRyZW5kcyBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzE1LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYWxhdGUgZGVoeWRyb2dlbmFzZXPigJRzdHJ1Y3R1cmUgYW5kIGZ1bmN0aW9uOyBHZW4uIFBoeXNpb2wuIEJpb3BoeXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjU3LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgdHJhbnNjcmlwdGlvbiBhbmFseXNpcyBvZiBLcmVicyB0cmljYXJib3h5bGljIGFjaWQgY3ljbGUgbXV0YW50cyByZXZlYWxzIGFuIGFsdGVybmF0aW5nIHBhdHRlcm4gb2YgZ2VuZSBleHByZXNzaW9uIGFuZCBlZmZlY3RzIG9uIGh5cG94aWMgYW5kIG94aWRhdGl2ZSBnZW5lczsgTW9sLiBCaW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9OTU4LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuZCBiaW9jaGVtaWNhbCBpbnRlcmFjdGlvbnMgaW52b2x2aW5nIHRyaWNhcmJveHlsaWMgYWNpZCBjeWNsZSAoVENBKSBmdW5jdGlvbiB1c2luZyBhIGNvbGxlY3Rpb24gb2YgbXV0YW50cyBkZWZlY3RpdmUgaW4gYWxsIFRDQSBjeWNsZSBnZW5lczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTUyOyAgY2l0YXRpb25fcGFnZXM9MTUzLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGV4cHJlc3Npb24gb2YgdGhlIGdlbmUgZW5jb2RpbmcgeWVhc3QgbWl0b2Nob25kcmlhbCBtYWxhdGUgZGVoeWRyb2dlbmFzZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz01MTU3LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHNlcXVlbmNlIGFuZCBwcmltYXJ5IHN0cnVjdHVyZSBvZiBtaXRvY2hvbmRyaWFsIG1hbGF0ZSBkZWh5ZHJvZ2VuYXNlIGZyb20gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBCaW9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz04MzkzLTQwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjZW50cmFsIHJvbGUgZm9yIHRoZSBwZXJveGlzb21hbCBtZW1icmFuZSBpbiBnbHlveHlsYXRlIGN5Y2xlIGZ1bmN0aW9uOyBCaW9jaGltLiBCaW9waHlzLiBBY3RhIChCQkEp4oCUTW9sLiBDZWxsIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTc2MzsgIGNpdGF0aW9uX3BhZ2VzPTE0NDEtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiwgbnVjbGVvdGlkZSBzZXF1ZW5jZSBhbmFseXNpcywgYW5kIGRpc3J1cHRpb24gb2YgdGhlIE1ESDIgZ2VuZSBmcm9tIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTogZXZpZGVuY2UgZm9yIHRocmVlIGlzb3p5bWVzIG9mIHllYXN0IG1hbGF0ZSBkZWh5ZHJvZ2VuYXNlOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MzcwLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIGFuZCBmdW5jdGlvbiBvZiBhIG1pc2xvY2FsaXplZCBmb3JtIG9mIHBlcm94aXNvbWFsIG1hbGF0ZSBkZWh5ZHJvZ2VuYXNlIChNREgzKSBpbiB5ZWFzdDsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjcwOyAgY2l0YXRpb25fcGFnZXM9MjEyMjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSB5ZWFzdCBnZW5lIGVuY29kaW5nIHRoZSBNREgzIGlzb3p5bWUgb2YgbWFsYXRlIGRlaHlkcm9nZW5hc2U7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI2NzsgIGNpdGF0aW9uX3BhZ2VzPTI0NzA4LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbnNlcnZlZCB0cmlwZXB0aWRlIHNvcnRzIHByb3RlaW5zIHRvIHBlcm94aXNvbWVzOyBKLiBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTE2NTctNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjYXJib3h5bC10ZXJtaW5hbCB0cmlwZXB0aWRlIEFsYS1MeXMtSWxlIGlzIGVzc2VudGlhbCBmb3IgdGFyZ2V0aW5nIENhbmRpZGEgdHJvcGljYWxpcyB0cmlmdW5jdGlvbmFsIGVuenltZSB0byB5ZWFzdCBwZXJveGlzb21lczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjY2OyAgY2l0YXRpb25fcGFnZXM9MjMxOTctMjAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJveGlzb21lIGJpb2dlbmVzaXMgaW4geWVhc3Q7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzQ1NS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aWJvZGllcyBkaXJlY3RlZCBhZ2FpbnN0IGEgeWVhc3QgY2FyYm94eWwtdGVybWluYWwgcGVyb3hpc29tYWwgdGFyZ2V0aW5nIHNpZ25hbCBzcGVjaWZpY2FsbHkgcmVjb2duaXplIHBlcm94aXNvbWFsIHByb3RlaW5zIGZyb20gdmFyaW91cyB5ZWFzdHM7IFllYXN0OyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz03MjEtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUfDqW5vbGV2dXJlczogcHJvdGVpbiBmYW1pbGllcyBhbmQgc3ludGVueSBhbW9uZyBjb21wbGV0ZSBoZW1pYXNjb215Y2V0b3VzIHllYXN0IHByb3Rlb21lcyBhbmQgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9RDU1MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGltb3JwaGljIGZ1bmd1cyBZYXJyb3dpYSBsaXBvbHl0aWNhOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIgVmVybGFnLCBIZWlkZWxiZXJnOyBHZW5ldGljcywgQmlvY2hlbWlzdHJ5IGFuZCBNb2xlY3VsYXIgQmlvbG9neSBvZiBOb24tQ29udmVudGlvbmFsIFllYXN0czsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3BhZ2VzPTMxMy04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlwaWQgYWNjdW11bGF0aW9uLCBsaXBpZCBib2R5IGZvcm1hdGlvbiwgYW5kIGFjeWwgY29lbnp5bWUgYSBveGlkYXNlcyBvZiB0aGUgeWVhc3QgWWFycm93aWEgbGlwb2x5dGljYTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT03MDsgIGNpdGF0aW9uX3BhZ2VzPTM5MTgtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWdyb2ZpdDogZml0dGluZyBiaW9sb2dpY2FsIGdyb3d0aCBjdXJ2ZXMgd2l0aCBSOyBKLiBTdGF0LiBTb2Z0dy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xLTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SOiBhIGxhbmd1YWdlIGZvciBkYXRhIGFuYWx5c2lzIGFuZCBncmFwaGljczsgSi4gQ29tcHV0LiBHcmFwaC4gU3RhdC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTI5OS0zMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Db2xkIFNwcmluZyBIYXJib3IgTGFib3JhdG9yeSBQcmVzcywgQ29sZCBTcHJpbmcgSGFyYm9yOyBNb2xlY3VsYXIgQ2xvbmluZzogQSBMYWJvcmF0b3J5IE1hbnVhbDsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1vbml0b3Jpbmcgb2Ygd2luZSBmZXJtZW50YXRpb25zIGNvbmR1Y3RlZCBieSBhY3RpdmUgZHJ5IHllYXN0IHN0cmFpbnM7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0yOTQ4LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGV4cHJlc3Npb24gYW5kIHNlY3JldGlvbiBpbiB0aGUgeWVhc3QgWWFycm93aWEgbGlwb2x5dGljYTsgRkVNUyBZZWFzdCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zNzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUtY29weSBpbnRlZ3JhdGlvbiBpbiB0aGUgeWVhc3QgWWFycm93aWEgbGlwb2x5dGljYTsgQ3Vyci4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzgtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdmVyc2F0aWxlIHRvb2xib3ggZm9yIFBDUi1iYXNlZCB0YWdnaW5nIG9mIHllYXN0IGdlbmVzOiBuZXcgZmx1b3Jlc2NlbnQgcHJvdGVpbnMsIG1vcmUgbWFya2VycyBhbmQgcHJvbW90ZXIgc3Vic3RpdHV0aW9uIGNhc3NldHRlczsgWWVhc3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz05NDctNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgZXhwcmVzc2lvbiBzeXN0ZW1zIGluIHRoZSB5ZWFzdHMgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlLCBIYW5zZW51bGEgcG9seW1vcnBoYSwgS2x5dmVyb215Y2VzIGxhY3RpcywgU2NoaXpvc2FjY2hhcm9teWNlcyBwb21iZSBhbmQgWWFycm93aWEgbGlwb2x5dGljYS4gQ2xvbmluZyBvZiB0d28gbm92ZWwgcHJvbW90ZXJzIGZyb20gWWFycm93aWEgbGlwb2x5dGljYTsgWWVhc3Q7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjY3LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQOiBzaG9ydCBvbGlnb251Y2xlb3RpZGUgYWxpZ25tZW50IHByb2dyYW07IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NzEzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgSiBNb2wgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NVVNDTEU6IG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB3aXRoIGhpZ2ggYWNjdXJhY3kgYW5kIGhpZ2ggdGhyb3VnaHB1dDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTc5Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgd2l0aCBoaWVyYXJjaGljYWwgY2x1c3RlcmluZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTA4ODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDTFVTVEFMX1ggV2luZG93cyBJbnRlcmZhY2U6IEZsZXhpYmxlIFN0cmF0ZWdpZXMgZm9yIE11bHRpcGxlIFNlcXVlbmNlIEFsaWdubWVudCBBaWRlZCBieSBRdWFsaXR5IEFuYWx5c2lzIFRvb2xzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00ODc2LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIG1ldGhvZCB0byBwcmVkaWN0IG1pdG9jaG9uZHJpYWxseSBpbXBvcnRlZCBwcm90ZWlucyBhbmQgdGhlaXIgdGFyZ2V0aW5nIHNlcXVlbmNlczsgRXVyLiBKLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDE7ICBjaXRhdGlvbl9wYWdlcz03NzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgcGVyb3hpc29tYWwgdGFyZ2V0aW5nIHNpZ25hbCAxIGNvbnRhaW5pbmcgcHJvdGVpbnMgZnJvbSBhbWlubyBhY2lkIHNlcXVlbmNlOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMyODsgIGNpdGF0aW9uX3BhZ2VzPTU4MS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGV2b2x1dGlvbiBpbiB5ZWFzdHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzA7ICBjaXRhdGlvbl9wYWdlcz0zNS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSByZWNvbWJpbmFudCBwcm90ZWluIHByb2R1Y3Rpb24gaG9zdCBQaWNoaWEgcGFzdG9yaXM7IE5hdC4gQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNoYnlhIEdlbm9tZSBEYXRhYmFzZSAzLjA6IGEgY3Jvc3Mtc3BlY2llcyBnZW5vbWUgYW5kIHRyYW5zY3JpcHRvbWUgYnJvd3NlciBmb3IgeWVhc3QgYmlvbG9naXN0czsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz05IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1lbWJyYW5lIG9mIHBlcm94aXNvbWVzIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBpcyBpbXBlcm1lYWJsZSB0byBOQUQoSCkgYW5kIGFjZXR5bC1Db0EgdW5kZXIgaW4gdml2byBjb25kaXRpb25zOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzQ4MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UQ0EgY3ljbGUtaW5kZXBlbmRlbnQgYWNldGF0ZSBtZXRhYm9saXNtIHZpYSB0aGUgZ2x5b3h5bGF0ZSBjeWNsZSBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IFllYXN0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTUzLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhbnRzIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSB3aXRoIGRlZmVjdHMgaW4gYWNldGF0ZSBtZXRhYm9saXNtOiBpc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgQWNuLSBtdXRhbnRzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDQ7ICBjaXRhdGlvbl9wYWdlcz01Ny02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSB1bmNvbnZlbnRpb25hbCBzcGxpY2luZyBvZiBZYXJyb3dpYSBsaXBvbHl0aWNhIEhBQzEgbVJOQSBpbmR1Y2VkIGJ5IHVuZm9sZGVkIHByb3RlaW4gcmVzcG9uc2U7IFllYXN0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDQzLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIElyZTEgaW4gdGhlIHllYXN0IFlhcnJvd2lhIGxpcG9seXRpY2EgcmV2ZWFscyBhbiBpbXBvcnRhbnQgcm9sZSBmb3IgdGhlIFNsczEgbnVjbGVvdGlkZSBleGNoYW5nZSBmYWN0b3IgaW4gdW5mb2xkZWQgcHJvdGVpbiByZXNwb25zZSByZWd1bGF0aW9uOyBDdXJyLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz0zMzctNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFib2xpYyBlZmZlY3RzIG9mIGFsdGVyaW5nIHJlZHVuZGFudCB0YXJnZXRpbmcgc2lnbmFscyBmb3IgeWVhc3QgbWl0b2Nob25kcmlhbCBtYWxhdGUgZGVoeWRyb2dlbmFzZTsgQXJjaC4gQmlvY2hlbS4gQmlvcGh5cy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MzQ0OyAgY2l0YXRpb25fcGFnZXM9NTMtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGluZyBvZiBtYWxhdGUgc3ludGhhc2UgMSB0byB0aGUgcGVyb3hpc29tZXMgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGNlbGxzIGRlcGVuZHMgb24gZ3Jvd3RoIG9uIG9sZWljIGFjaWQgbWVkaXVtOyBFdXIuIEouIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI2OTsgIGNpdGF0aW9uX3BhZ2VzPTkxNS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1hZ2luZy1iYXNlZCBsaXZlIGNlbGwgeWVhc3Qgc2NyZWVuIGlkZW50aWZpZXMgbm92ZWwgZmFjdG9ycyBpbnZvbHZlZCBpbiBwZXJveGlzb21lIGFzc2VtYmx5OyBKLiBQcm90ZW9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTIwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgb2YgdGhlIGlzb2NpdHJhdGUgbHlhc2UgZ2VuZSAoSUNMMSkgZnJvbSBZYXJyb3dpYSBsaXBvbHl0aWNhIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBkZWR1Y2VkIHByb3RlaW47IE1vbCBHZW4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjQxOyAgY2l0YXRpb25fcGFnZXM9NDIyLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBwZXJveGlzb21hbCBtYWxhdGUgZGVoeWRyb2dlbmFzZSBmdW5jdGlvbnMgaW4gYmV0YS1veGlkYXRpb24gYnV0IG5vdCBpbiB0aGUgZ2x5b3h5bGF0ZSBjeWNsZTsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0zODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFib2xpdGUgdHJhbnNwb3J0IGFjcm9zcyB0aGUgcGVyb3hpc29tYWwgbWVtYnJhbmU7IEJpb2NoZW0gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDAxOyAgY2l0YXRpb25fcGFnZXM9MzY1LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmdpbmVlcmluZyBwb2x5aHlkcm94eWFsa2Fub2F0ZSBjb250ZW50IGFuZCBtb25vbWVyIGNvbXBvc2l0aW9uIGluIHRoZSBvbGVhZ2lub3VzIHllYXN0IFlhcnJvd2lhIGxpcG9seXRpY2EgYnkgbW9kaWZ5aW5nIHRoZSDDny1veGlkYXRpb24gbXVsdGlmdW5jdGlvbmFsIHByb3RlaW47IEFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xMzI3LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSByb3V0ZXMgb2YgYWNldHlsLUNvQSBzeW50aGVzaXMgaWRlbnRpZmllZCBieSBjb21wYXJhdGl2ZSBnZW5vbWljIGFuYWx5c2lzOiBJbnZvbHZlbWVudCBpbiBsaXBpZCBwcm9kdWN0aW9uIG9mIG9sZWFnaW5vdXMgeWVhc3QgYW5kIGZ1bmdpOyBjaXRhdGlvbl90ZXh0PWluIHByZXNzLCBkb2k6MTAuMTA5OS9taWMuMC4wNTE5NDYtMDsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVVMxIGludHJvbnMgYXJlIHJlcXVpcmVkIGZvciBlZmZpY2llbnQgbVJOQSBudWNsZWFyIGV4cG9ydCBpbiB5ZWFzdDsgY2l0YXRpb25fdGV4dD1kb2k6MTAuMTAuOTMvbmFyL2drcjQ5NjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LZXkgZmVhdHVyZXMgb2YgdGhlIHR3by1pbnRyb24gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGdlbmUgU1VTMSBjb250cmlidXRlIHRvIGl0cyBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9ODYxMi0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVzdCBvZiBpbnRyb24gcHJlZGljdGlvbnMgcmV2ZWFscyBub3ZlbCBzcGxpY2Ugc2l0ZXMsIGFsdGVybmF0aXZlbHkgc3BsaWNlZCBtUk5BcyBhbmQgbmV3IGludHJvbnMgaW4gbWVpb3RpY2FsbHkgcmVndWxhdGVkIGdlbmVzIG9mIHllYXN0OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xNzAwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1laW9zaXMtc3BlY2lmaWMgUk5BIHNwbGljaW5nIGluIHllYXN0OyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9MTI1Ny02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBNRVIzIGdlbmUsIGVuY29kaW5nIGEgbm92ZWwgaGVsaWNhc2UtbGlrZSBwcm90ZWluLCBpcyByZXF1aXJlZCBmb3IgY3Jvc3NvdmVyIGNvbnRyb2wgaW4gbWVpb3NpczsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTU3MTQtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIFJOQSBzdHJ1Y3R1cmUgaW52b2x2ZWQgaW4gZmVlZGJhY2sgcmVndWxhdGlvbiBvZiBzcGxpY2luZyBhbmQgb2YgdHJhbnNsYXRpb24gaXMgY3JpdGljYWwgZm9yIGJpb2xvZ2ljYWwgZml0bmVzczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTE1OTYtNjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvcmVndWxhdGlvbiBvZiB0aGUgbVJOQSBleHBvcnQgZmFjdG9yIFlyYTFwIHJlcXVpcmVzIGluZWZmaWNpZW50IHNwbGljaW5nIG9mIGl0cyBwcmUtbVJOQTsgUk5BOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9OTk0LTEwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGhpcmQgb2YgdGhlIHllYXN0IG1pdG9jaG9uZHJpYWwgcHJvdGVvbWUgaXMgZHVhbCBsb2NhbGl6ZWQ6IGEgcXVlc3Rpb24gb2YgZXZvbHV0aW9uOyBQcm90ZW9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDQ2OC03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZXMgb2YgbXVsdGlwbGUgYWN5bC1Db0Egb3hpZGFzZXMgaW4gdGhlIHJvdXRpbmcgb2YgY2FyYm9uIGZsb3cgdG93YXJkcyDOsi1veGlkYXRpb24gYW5kIHBvbHloeWRyb3h5YWxrYW5vYXRlIGJpb3N5bnRoZXNpcyBpbiBZYXJyb3dpYSBsaXBvbHl0aWNhOyBGRU1TIFllYXN0IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz05MTctMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS8zLzIzMS8xMjgzODEyL2RzczAwNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWx0ZXJuYXRpdmUgcHJlLW1STkEgc3BsaWNpbmcgaXMgYSBtYWpvciBtZWNoYW5pc20gY29udHJpYnV0aW5nIHRvIHRoZSBwcm90ZW9tZSBjb21wbGV4aXR5IG9mIG1vc3QgZXVrYXJ5b3RlcywgZXNwZWNpYWxseSBtYW1tYWxzLiBJbiBsZXNzIGNvbXBsZXggb3IiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ3NDg2MyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQVQ2ZpEGTxgdUBIEN1zCOwabRF0A==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f231%2f474863" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f231%2f474863" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18620" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&amp;Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/3">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7159154">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159154">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159156">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159156">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159162">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159162">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159180">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159180">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159200">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159200">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159208">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159208">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159210">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159210">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159212">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159212">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7159214">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7159214">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/3/219/474767">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/3/245/475122">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Alternative Splicing Regulates Targeting of Malate Dehydrogenase in <em>Yarrowia lipolytica</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Philom&#232;ne Kabran</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA</div>, <div class="addr-line">UMR 1319 Micalis, F-78352 Jouy-en-Josas</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Biological Sciences</div>, <div class="insititution">University of California, San Diego</div>, <div class="addr-line">9500 Gilman Drive, La Jolla, CA 92093</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Philom%c3%a8ne+Kabran">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kabran P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kabran P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Philomène Kabran</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tristan Rossignol</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA</div>, <div class="addr-line">UMR 1319 Micalis, F-78352 Jouy-en-Josas</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tristan+Rossignol">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rossignol T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rossignol T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tristan Rossignol</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Claude Gaillardin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">AgroParisTech</div>, <div class="addr-line">Micalis, F-78352 Jouy-en-Josas</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Claude+Gaillardin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gaillardin C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gaillardin C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Claude Gaillardin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jean-Marc Nicaud</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">CNRS, Micalis, F-78352 Jouy-en-Josas</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jean-Marc+Nicaud">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nicaud J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nicaud J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jean-Marc Nicaud</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">C&#233;cile Neuv&#233;glise</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">INRA</div>, <div class="addr-line">UMR 1319 Micalis, F-78352 Jouy-en-Josas</div>, <div class="country">France</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. INRA, UMR 1319 Micalis, Biologie intégrative du métabolisme lipidique microbien, Bât. CBAI, F-78850 Thiverval-Grignon, France. Tel. +33 1-30-81-54-78. Fax. +33 1-30-81-54-57. Email: <a href="mailto:cecile.neuveglise@grignon.inra.fr" target="_blank">cecile.neuveglise@grignon.inra.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=C%c3%a9cile+Neuv%c3%a9glise">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Neuv&#233;glise C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Neuv&#233;glise C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Cécile Neuvéglise</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 231–244, <a href='https://doi.org/10.1093/dnares/dss007'>https://doi.org/10.1093/dnares/dss007</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 February 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 November 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 January 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="474863" href="/dnaresearch/article-pdf/19/3/231/1283812/dss007.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Philomène Kabran, Tristan Rossignol, Claude Gaillardin, Jean-Marc Nicaud, Cécile Neuvéglise;  Alternative Splicing Regulates Targeting of Malate Dehydrogenase in <em>Yarrowia lipolytica</em>, <em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 231–244, <a href='https://doi.org/10.1093/dnares/dss007'>https://doi.org/10.1093/dnares/dss007</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/474863?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/474863?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/474863?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/474863?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/474863?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/474863?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/474863?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Alternative%20Splicing%20Regulates%20Targeting%20of%20Malate%20Dehydrogenase%20in%20%26lt%3Bem%26gt%3BYarrowia%20lipolytica%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2012-02-24&amp;volumeNum=19&amp;issueNum=3&amp;author=Kabran%2C%20Philom%C3%A8ne%3B%20Rossignol%2C%20Tristan&amp;startPage=231&amp;endPage=244&amp;contentId=10.1093%2Fdnares%2Fdss007&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f231%2f474863" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7159154 id="7159154" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative pre-mRNA splicing is a major mechanism contributing to the proteome complexity of most eukaryotes, especially mammals. In less complex organisms, such as yeasts, the numbers of genes that contain introns are low and cases of alternative splicing (AS) with functional implications are rare<em>.</em> We report the first case of AS with functional consequences in the yeast <em>Yarrowia lipolytica</em>. The splicing pattern was found to govern the cellular localization of malate dehydrogenase, an enzyme of the central carbon metabolism. This ubiquitous enzyme is involved in the tricarboxylic acid cycle in mitochondria and in the glyoxylate cycle, which takes place in peroxisomes and the cytosol. In <em>Saccharomyces cerevisiae</em>, three genes encode three compartment-specific enzymes. In contrast, only two genes exist in <em>Y. lipolytica</em>. One gene (<em>YlMDH1</em>, YALI0D16753g) encodes a predicted mitochondrial protein, whereas the second gene (<em>YlMDH2</em>, YALI0E14190g) generates the cytosolic and peroxisomal forms through the alternative use of two 3′-splice sites in the second intron. Both splicing variants were detected in cDNA libraries obtained from cells grown under different conditions. Mutants expressing the individual YlMdh2p isoforms tagged with fluorescent proteins confirmed that they localized to either the cytosolic or the peroxisomal compartment.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="yeast">yeast</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;TCA cycle&quot;">TCA cycle</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;glyoxylate cycle&quot;">glyoxylate cycle</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="MDH2">MDH2</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="intron">intron</a></div>                            <h2 scrollto-destination=7159156 id="7159156" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The hemiascomycetous yeast <em>Yarrowia lipolytica</em> has relatively few introns compared with other Opisthokonts such as vertebrates or even compared with basidiomycota and filamentous ascomycota. However, with introns in ∼15% of its genes,<sup><a reveal-id="DSS007C1" data-open="DSS007C1" class="link link-ref link-reveal xref-bibr">1</a></sup><em>Y. lipolytica</em> has many more introns than any other hemiascomycete yeast, e.g<em>.</em> four times more than <em>Saccharomyces cerevisiae.</em><sup><a reveal-id="DSS007C2" data-open="DSS007C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Despite this paucity of introns, the <em>S. cerevisiae</em> transcriptome is proving to be more complex than previously appreciated.<sup><a reveal-id="DSS007C3" data-open="DSS007C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS007C4" data-open="DSS007C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Alternative transcription start sites have been reported and, for some intron-containing genes, transcript variants result from either alternative splicing (AS) or intron retention. Overall, alternative transcripts leading to different proteins remain uncommon in <em>S. cerevisiae</em> and only one case of AS (<em>SRC1</em><sup><a reveal-id="DSS007C5" data-open="DSS007C5" class="link link-ref link-reveal xref-bibr">5</a></sup>) and one of intron retention (<em>PTC7</em><sup><a reveal-id="DSS007C6" data-open="DSS007C6" class="link link-ref link-reveal xref-bibr">6</a></sup>) have been described, along with several cases of alternative start sites (e.g. <em>YCAT</em><sup><a reveal-id="DSS007C7" data-open="DSS007C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and <em>SUC2</em><sup><a reveal-id="DSS007C8" data-open="DSS007C8" class="link link-ref link-reveal xref-bibr">8</a></sup>). The large majority of alternative transcripts in yeasts are presumed to be untranslated, as most of them contain premature termination codons and therefore activate the nonsense-mediated mRNA decay pathway that degrades defective mRNAs prior to translation (for review, see Behm-Ansmant <em>et al</em>.<sup><a reveal-id="DSS007C9" data-open="DSS007C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and Stalder and Muhlemann<sup><a reveal-id="DSS007C10" data-open="DSS007C10" class="link link-ref link-reveal xref-bibr">10</a></sup>).</p><p>AS has been investigated at the genome scale in <em>Y. lipolytica</em> and all known modes of AS have been observed, i.e<em>.</em> exon skipping and alternative 5′- or 3′-splice sites, as well as intron retention and alternative start sites.<sup><a reveal-id="DSS007C1" data-open="DSS007C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Several of these genes are involved in central carbon metabolism. The malate dehydrogenase (MDH) gene (<em>YlMDH2</em>, YALI0E14190g) was particularly attractive, as AS does not generate a premature termination codon, but leads to putative functional variants of the protein.</p><p>The MDH isoenzymes catalyse the conversion of malate into oxaloacetate with a concomitant reduction of NAD<sup>+</sup> (see Minarik <em>et al</em>.<sup><a reveal-id="DSS007C11" data-open="DSS007C11" class="link link-ref link-reveal xref-bibr">11</a></sup> for a review). This reaction, which is reversible, represents an important step in the tricarboxylic acid (TCA) cycle, a central metabolic pathway occurring in mitochondria and critical for cellular respiration and ATP production.<sup><a reveal-id="DSS007C12" data-open="DSS007C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS007C13" data-open="DSS007C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The reaction also takes place in the glyoxylate cycle, which is a variant of the TCA cycle that shares three of its five enzymes with the TCA cycle, including MDH. In plants, nematodes and yeasts, the glyoxylate cycle is partly localized in peroxisomes, and thus contributes to the degradation of free fatty acids by providing coenzyme A and/or reoxidizing NADH for the β-oxidation cycle. MDH is also involved in gluconeogenesis, which takes place in the cytosol and plays a significant role in the malate/aspartate shuttle across the mitochondrial membrane.</p><p>In <em>S. cerevisiae</em>, three MDH genes (<em>MDH1</em>, <em>MDH2</em> and <em>MDH3</em>) have been identified and each encodes an enzyme targeted for a different subcellular compartment. Mdh1p is localized in mitochondria,<sup><a reveal-id="DSS007C14" data-open="DSS007C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSS007C15" data-open="DSS007C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Mdh2p in the cytosol<sup><a reveal-id="DSS007C16" data-open="DSS007C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSS007C17" data-open="DSS007C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and Mdh3p in peroxisomes.<sup><a reveal-id="DSS007C18" data-open="DSS007C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS007C19" data-open="DSS007C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Large sequence variations have been observed in the MDH regions that encode the putative domains involved in compartmental targeting. For example, <em>MDH1</em> encodes a 17-amino acid N-terminal extension that is absent from the other isozymes and is removed upon mitochondrial import.<sup><a reveal-id="DSS007C15" data-open="DSS007C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Mdh3p has a unique C-terminal tripeptide sequence, Ser-Lys-Leu, characteristic of peroxisomal targeting sequences (PTSs).<sup><a reveal-id="DSS007C20" data-open="DSS007C20" class="link link-ref link-reveal xref-bibr">20</a></sup> This targeting sequence, called PTS1, is conserved in <em>S. cerevisiae</em>, but other conservative variants, such as Ala-Lys-Ile, exist among yeasts.<sup><a reveal-id="DSS007C21  DSS007C23" data-open="DSS007C21  DSS007C23" class="link link-ref link-reveal xref-bibr">21–23</a></sup></p><p>Homologues of all three <em>S. cerevisiae</em> MDH genes are present in many other hemiascomycetous species.<sup><a reveal-id="DSS007C24" data-open="DSS007C24" class="link link-ref link-reveal xref-bibr">24</a></sup> In contrast, the oleagineous yeast <em>Y. lipolytica</em> contains only two MDH genes: YALI0D16753g (<em>YlMDH1</em>) and YALI0E14190g (<em>YlMDH2</em>). Whereas the first gene encodes a protein predicted to have a mitochondrial location, the second gene encodes a putatively cytosolic form. The presence of only two MDH genes in <em>Y. lipolytica</em> compared with the three genes possessed by other hemiascomycetes, and the fact that <em>YlMDH2</em> is potentially subject to AS,<sup><a reveal-id="DSS007C1" data-open="DSS007C1" class="link link-ref link-reveal xref-bibr">1</a></sup> suggest a possible role for AS in the regulation of MDH compartmentalization in this yeast. We thus focused on these <em>Y. lipolytica</em> MDH genes to decipher the roles and regulation of the different isoenzymes. In this study, we show that <em>YlMDH2</em> encodes the cytosolic and peroxisomal forms of MDH. This dual localization is due to the presence of an alternative 3′-splice site in the second intron of the gene, located at the 3′-end of the coding sequence. The use of this alternative splice site creates an mRNA that encodes a carboxyl-terminal peroxisomal targeting sequence (PTS1) and allows specific peroxisomal localization, which was revealed by colocalization studies.</p>                            <h2 scrollto-destination=7159162 id="7159162" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7159163 id="7159163" class="section-title" data-legacy-id=s2a>2.1. Strains and culture conditions</h3>
<p>The yeast and bacterial strains used in this study are listed in Table <a reveal-id="DSS007TB1" data-open="DSS007TB1" class="link link-reveal link-table xref-fig">1</a>. The bacterial strains, Mach1 T1<sup>®</sup> (Invitrogen, Cergy Pontoise, France) and DH5α (Gibco BRL, Rockville, MD, USA) used for the amplification of recombinant plasmids, were grown at 37°C in Luria-Bertani medium supplemented with 100 μg/l ampicillin or 40 μg/l kanamycin, if required. </p>                        <a id="7159165" scrollto-destination="7159165"></a>
<div content-id="DSS007TB1" class="table-modal"><div class="table-wrap"><div id="DSS007TB1" class="table-wrap-title" data-id="DSS007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Yarrowia lipolytica and bacterial strains used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th>Genotype </th><th>Reference </th><th>Comments </th></tr></thead><tbody><tr><td colspan="4"><strong>Yeasts</strong> </td></tr><tr><td> PO1d </td><td>Mat A, <em>leu2–270, ura3–302, xpr2–322</em> </td><td><a reveal-id="DSS007C25" data-open="DSS007C25" class="link link-ref link-reveal xref-bibr">25</a> </td><td> </td></tr><tr><td> JMY1699 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY1711 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY1685 </td><td>PO1d<em>, URA3-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2416 </td><td>JMY1699<em>, LEU2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2426 </td><td>JMY1711<em>, LEU2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2428 </td><td>JMY1685<em>, LEU2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2451 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2457 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2459 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2499 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2501 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2500 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2493 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2496 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2495 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td><strong>Bacteria</strong> </td><td><strong>Plasmid/genotype</strong> </td><td> </td><td> </td></tr><tr><td> JME802 </td><td>JMP62 <em>LEU2ex</em>, pPOX2 expression vector with the excisable <em>LEU2ex</em> marker </td><td><a reveal-id="DSS007C66" data-open="DSS007C66" class="link link-ref link-reveal xref-bibr">66</a> </td><td> </td></tr><tr><td> JME803 </td><td>JMP62 <em>URA3ex</em>, pPOX2 expression vector with the excisable <em>URA3ex</em> marker </td><td><a reveal-id="DSS007C56" data-open="DSS007C56" class="link link-ref link-reveal xref-bibr">56</a> </td><td> </td></tr><tr><td> JME1018 </td><td>JMP62 <em>URA3</em> pPOX eYFP N-Ter </td><td>This study </td><td> </td></tr><tr><td> JME1392 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2-SKL </td><td>This study </td><td> </td></tr><tr><td> JME1394 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2 </td><td>This study </td><td> </td></tr></tbody></table></div><div content-id="DSS007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th>Genotype </th><th>Reference </th><th>Comments </th></tr></thead><tbody><tr><td colspan="4"><strong>Yeasts</strong> </td></tr><tr><td> PO1d </td><td>Mat A, <em>leu2–270, ura3–302, xpr2–322</em> </td><td><a reveal-id="DSS007C25" data-open="DSS007C25" class="link link-ref link-reveal xref-bibr">25</a> </td><td> </td></tr><tr><td> JMY1699 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY1711 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY1685 </td><td>PO1d<em>, URA3-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2416 </td><td>JMY1699<em>, LEU2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2426 </td><td>JMY1711<em>, LEU2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2428 </td><td>JMY1685<em>, LEU2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2451 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2457 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2459 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2499 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2501 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2500 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2493 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2496 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2495 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td><strong>Bacteria</strong> </td><td><strong>Plasmid/genotype</strong> </td><td> </td><td> </td></tr><tr><td> JME802 </td><td>JMP62 <em>LEU2ex</em>, pPOX2 expression vector with the excisable <em>LEU2ex</em> marker </td><td><a reveal-id="DSS007C66" data-open="DSS007C66" class="link link-ref link-reveal xref-bibr">66</a> </td><td> </td></tr><tr><td> JME803 </td><td>JMP62 <em>URA3ex</em>, pPOX2 expression vector with the excisable <em>URA3ex</em> marker </td><td><a reveal-id="DSS007C56" data-open="DSS007C56" class="link link-ref link-reveal xref-bibr">56</a> </td><td> </td></tr><tr><td> JME1018 </td><td>JMP62 <em>URA3</em> pPOX eYFP N-Ter </td><td>This study </td><td> </td></tr><tr><td> JME1392 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2-SKL </td><td>This study </td><td> </td></tr><tr><td> JME1394 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2 </td><td>This study </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7159165" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS007TB1" class="table-wrap-title" data-id="DSS007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Yarrowia lipolytica and bacterial strains used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th>Genotype </th><th>Reference </th><th>Comments </th></tr></thead><tbody><tr><td colspan="4"><strong>Yeasts</strong> </td></tr><tr><td> PO1d </td><td>Mat A, <em>leu2–270, ura3–302, xpr2–322</em> </td><td><a reveal-id="DSS007C25" data-open="DSS007C25" class="link link-ref link-reveal xref-bibr">25</a> </td><td> </td></tr><tr><td> JMY1699 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY1711 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY1685 </td><td>PO1d<em>, URA3-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2416 </td><td>JMY1699<em>, LEU2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2426 </td><td>JMY1711<em>, LEU2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2428 </td><td>JMY1685<em>, LEU2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2451 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2457 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2459 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2499 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2501 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2500 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2493 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2496 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2495 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td><strong>Bacteria</strong> </td><td><strong>Plasmid/genotype</strong> </td><td> </td><td> </td></tr><tr><td> JME802 </td><td>JMP62 <em>LEU2ex</em>, pPOX2 expression vector with the excisable <em>LEU2ex</em> marker </td><td><a reveal-id="DSS007C66" data-open="DSS007C66" class="link link-ref link-reveal xref-bibr">66</a> </td><td> </td></tr><tr><td> JME803 </td><td>JMP62 <em>URA3ex</em>, pPOX2 expression vector with the excisable <em>URA3ex</em> marker </td><td><a reveal-id="DSS007C56" data-open="DSS007C56" class="link link-ref link-reveal xref-bibr">56</a> </td><td> </td></tr><tr><td> JME1018 </td><td>JMP62 <em>URA3</em> pPOX eYFP N-Ter </td><td>This study </td><td> </td></tr><tr><td> JME1392 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2-SKL </td><td>This study </td><td> </td></tr><tr><td> JME1394 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2 </td><td>This study </td><td> </td></tr></tbody></table></div><div content-id="DSS007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th>Genotype </th><th>Reference </th><th>Comments </th></tr></thead><tbody><tr><td colspan="4"><strong>Yeasts</strong> </td></tr><tr><td> PO1d </td><td>Mat A, <em>leu2–270, ura3–302, xpr2–322</em> </td><td><a reveal-id="DSS007C25" data-open="DSS007C25" class="link link-ref link-reveal xref-bibr">25</a> </td><td> </td></tr><tr><td> JMY1699 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY1711 </td><td>PO1d<em>, URA3-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY1685 </td><td>PO1d<em>, URA3-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2416 </td><td>JMY1699<em>, LEU2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2426 </td><td>JMY1711<em>, LEU2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2428 </td><td>JMY1685<em>, LEU2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2451 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1422</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2457 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> Δ<em>intron 1335-1426</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2459 </td><td>PO1d, <em>URA3-pPOX-eYFP-YALI0E14190g</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2499 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2501 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2500 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td> JMY2493 </td><td>JMY2451, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Cytosolic form of <em>YlMDH2</em> </td></tr><tr><td> JMY2496 </td><td>JMY2457, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Peroxisomal form of <em>YlMDH2</em> </td></tr><tr><td> JMY2495 </td><td>JMY2459, <em>LEU2-pTEF-REDSTAR2-SKL</em> </td><td>This study </td><td>Wild-type form of <em>YlMDH2</em> </td></tr><tr><td><strong>Bacteria</strong> </td><td><strong>Plasmid/genotype</strong> </td><td> </td><td> </td></tr><tr><td> JME802 </td><td>JMP62 <em>LEU2ex</em>, pPOX2 expression vector with the excisable <em>LEU2ex</em> marker </td><td><a reveal-id="DSS007C66" data-open="DSS007C66" class="link link-ref link-reveal xref-bibr">66</a> </td><td> </td></tr><tr><td> JME803 </td><td>JMP62 <em>URA3ex</em>, pPOX2 expression vector with the excisable <em>URA3ex</em> marker </td><td><a reveal-id="DSS007C56" data-open="DSS007C56" class="link link-ref link-reveal xref-bibr">56</a> </td><td> </td></tr><tr><td> JME1018 </td><td>JMP62 <em>URA3</em> pPOX eYFP N-Ter </td><td>This study </td><td> </td></tr><tr><td> JME1392 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2-SKL </td><td>This study </td><td> </td></tr><tr><td> JME1394 </td><td>JMP62 <em>LEU2ex</em> pTEF Redstar2 </td><td>This study </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7159165" target="_blank">View Large</a></div></div><p>Growth media and conditions used for <em>Y. lipolytica</em> have been previously described.<sup><a reveal-id="DSS007C25" data-open="DSS007C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The yeasts were grown on rich medium YPD and minimum medium YNB<sup><a reveal-id="DSS007C26" data-open="DSS007C26" class="link link-ref link-reveal xref-bibr">26</a></sup> or YNBE (YNB supplemented with 0.1% yeast extract), supplemented with 0.2% (w/v) casa-amino acids or 0.05 g/ml uracil, if required. Carbon sources were dextrose (D), oleic acid (OA), tributyrin (TB), triolein (TO), alkane (hexadecane), acetate (ammonium or sodium), glycerol or ethanol at a final concentration of 2%. The hydrophobic substrates were emulsified by sonication in a mixture containing 20% OA, TB or TO and 0.625% (v/v) Tween 40. For solid media, 2% agar was added. Yeast strains were grown at 28°C. Growth on 96-well plates in 100 µl was performed under constant agitation with glucose, malate, succinate or citrate at 0.5% as the carbon source. Growth was monitored by measuring the optical density at 600 nm every 10 min using a microtiter plate reader (Biotek, Colmar, France). For each strain and condition, four biological replicates were performed. Calculations of average OD, blank reduction, the lag phase (<em>µ</em>) and the maximum OD were performed using the grofit package in R.<sup><a reveal-id="DSS007C27" data-open="DSS007C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Differences between growth curves were evaluated by analysis of variance with R statistical software.<sup><a reveal-id="DSS007C28" data-open="DSS007C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h3 scrollto-destination=7159167 id="7159167" class="section-title" data-legacy-id=s2b>2.2. Molecular biology techniques</h3>
<p>Standard molecular biology techniques were used throughout this study.<sup><a reveal-id="DSS007C29" data-open="DSS007C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Restriction enzymes were obtained from Ozyme (Saint-Quentin-en-Yvelines, France). Genomic DNA from yeast was prepared as previously reported.<sup><a reveal-id="DSS007C30" data-open="DSS007C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Polymerase chain reaction (PCR) amplifications were performed using an Eppendorf 2720 thermocycler, Pyrobest DNA polymerase (Lonza, Levallois-Perret, France) and the primers are listed in Table <a reveal-id="DSS007TB2" data-open="DSS007TB2" class="link link-reveal link-table xref-fig">2</a>. PCR fragments were purified using the QIAquick PCR Puriﬁcation Kit (Qiagen, Courtaboeuf, France). DNA fragments were recovered from agarose gels using the QIAquick Gel Extraction Kit (Qiagen). </p>                        <a id="7159169" scrollto-destination="7159169"></a>
<div content-id="DSS007TB2" class="table-modal"><div class="table-wrap"><div id="DSS007TB2" class="table-wrap-title" data-id="DSS007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Oligonucleotides used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Primer </th><th>Sequence </th><th>Purpose </th></tr></thead><tbody><tr><td>E14190F1 </td><td>GCCTACATCTACCTTGAC </td><td>cDNA sequencing </td></tr><tr><td>MDH1-Start </td><td>CAG<span class="underline">ATCGAT</span><strong>TAA</strong>C<strong>TGA</strong>C<strong>TAG</strong>GTTAAAGCTGTCGTTGCCGGAGC </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-Sa </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGCTAGAATGGTTAGTGATCGTGTAGTTCAAATGG </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-04 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-09 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDHS1 </td><td>AACAGGCAACAATGGCATGC </td><td>Probe for Southern blot </td></tr><tr><td>MDHS2 </td><td>AACTGGATTCGCTTGACGAG </td><td>Probe for Southern blot </td></tr><tr><td>MDHgfp1 </td><td>CGC<span class="underline">TGATCA</span><strong>A</strong>gtgagttatcatggtgggag </td><td>Fusion cassette </td></tr><tr><td>MDHgfp2 </td><td>CGC<span class="underline">TGATCA</span><strong>ATG</strong>GTTAAAGCTGTCGTTGCC </td><td>Fusion cassette </td></tr><tr><td>RedSKL1 </td><td>CAC<span class="underline">GGATCC</span>CACAATGAGTGCTTCTTCTGAAGATG </td><td>Localization cassette </td></tr><tr><td>RedSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA<em>AAGCTTGGA</em></strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr><tr><td>RedNoSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA</strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr></tbody></table></div><div content-id="DSS007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Primer </th><th>Sequence </th><th>Purpose </th></tr></thead><tbody><tr><td>E14190F1 </td><td>GCCTACATCTACCTTGAC </td><td>cDNA sequencing </td></tr><tr><td>MDH1-Start </td><td>CAG<span class="underline">ATCGAT</span><strong>TAA</strong>C<strong>TGA</strong>C<strong>TAG</strong>GTTAAAGCTGTCGTTGCCGGAGC </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-Sa </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGCTAGAATGGTTAGTGATCGTGTAGTTCAAATGG </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-04 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-09 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDHS1 </td><td>AACAGGCAACAATGGCATGC </td><td>Probe for Southern blot </td></tr><tr><td>MDHS2 </td><td>AACTGGATTCGCTTGACGAG </td><td>Probe for Southern blot </td></tr><tr><td>MDHgfp1 </td><td>CGC<span class="underline">TGATCA</span><strong>A</strong>gtgagttatcatggtgggag </td><td>Fusion cassette </td></tr><tr><td>MDHgfp2 </td><td>CGC<span class="underline">TGATCA</span><strong>ATG</strong>GTTAAAGCTGTCGTTGCC </td><td>Fusion cassette </td></tr><tr><td>RedSKL1 </td><td>CAC<span class="underline">GGATCC</span>CACAATGAGTGCTTCTTCTGAAGATG </td><td>Localization cassette </td></tr><tr><td>RedSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA<em>AAGCTTGGA</em></strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr><tr><td>RedNoSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA</strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Restriction sites are underlined: <em>Cla</em>I ATCGAT, <em>Avr</em>II CCTAGG, <em>Bcl</em>I TGATCA, <em>Bam</em>HI GGATCC. Stop codons and methionine codons (or the first nucleotide of the codon) are in bold. Nucleotides complementary to the S-K-L codons are in bold italic. Intron nucleotide sequence is in lower case.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7159169" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS007TB2" class="table-wrap-title" data-id="DSS007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Oligonucleotides used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Primer </th><th>Sequence </th><th>Purpose </th></tr></thead><tbody><tr><td>E14190F1 </td><td>GCCTACATCTACCTTGAC </td><td>cDNA sequencing </td></tr><tr><td>MDH1-Start </td><td>CAG<span class="underline">ATCGAT</span><strong>TAA</strong>C<strong>TGA</strong>C<strong>TAG</strong>GTTAAAGCTGTCGTTGCCGGAGC </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-Sa </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGCTAGAATGGTTAGTGATCGTGTAGTTCAAATGG </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-04 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-09 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDHS1 </td><td>AACAGGCAACAATGGCATGC </td><td>Probe for Southern blot </td></tr><tr><td>MDHS2 </td><td>AACTGGATTCGCTTGACGAG </td><td>Probe for Southern blot </td></tr><tr><td>MDHgfp1 </td><td>CGC<span class="underline">TGATCA</span><strong>A</strong>gtgagttatcatggtgggag </td><td>Fusion cassette </td></tr><tr><td>MDHgfp2 </td><td>CGC<span class="underline">TGATCA</span><strong>ATG</strong>GTTAAAGCTGTCGTTGCC </td><td>Fusion cassette </td></tr><tr><td>RedSKL1 </td><td>CAC<span class="underline">GGATCC</span>CACAATGAGTGCTTCTTCTGAAGATG </td><td>Localization cassette </td></tr><tr><td>RedSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA<em>AAGCTTGGA</em></strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr><tr><td>RedNoSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA</strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr></tbody></table></div><div content-id="DSS007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Primer </th><th>Sequence </th><th>Purpose </th></tr></thead><tbody><tr><td>E14190F1 </td><td>GCCTACATCTACCTTGAC </td><td>cDNA sequencing </td></tr><tr><td>MDH1-Start </td><td>CAG<span class="underline">ATCGAT</span><strong>TAA</strong>C<strong>TGA</strong>C<strong>TAG</strong>GTTAAAGCTGTCGTTGCCGGAGC </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-Sa </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGCTAGAATGGTTAGTGATCGTGTAGTTCAAATGG </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-04 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCCTAGTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDH1-09 </td><td>AGG<span class="underline">CCTAGG</span>TTTAGATCTTGGCAGGAGGAGGGTTAACAATG. </td><td>MDH disruption cassette </td></tr><tr><td>MDHS1 </td><td>AACAGGCAACAATGGCATGC </td><td>Probe for Southern blot </td></tr><tr><td>MDHS2 </td><td>AACTGGATTCGCTTGACGAG </td><td>Probe for Southern blot </td></tr><tr><td>MDHgfp1 </td><td>CGC<span class="underline">TGATCA</span><strong>A</strong>gtgagttatcatggtgggag </td><td>Fusion cassette </td></tr><tr><td>MDHgfp2 </td><td>CGC<span class="underline">TGATCA</span><strong>ATG</strong>GTTAAAGCTGTCGTTGCC </td><td>Fusion cassette </td></tr><tr><td>RedSKL1 </td><td>CAC<span class="underline">GGATCC</span>CACAATGAGTGCTTCTTCTGAAGATG </td><td>Localization cassette </td></tr><tr><td>RedSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA<em>AAGCTTGGA</em></strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr><tr><td>RedNoSKL2 </td><td>TGGTG<span class="underline">CCTAGG</span>CTGC<strong>TTA</strong>CAAGAACAAGTGGTGTCTACC </td><td>Localization cassette </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Restriction sites are underlined: <em>Cla</em>I ATCGAT, <em>Avr</em>II CCTAGG, <em>Bcl</em>I TGATCA, <em>Bam</em>HI GGATCC. Stop codons and methionine codons (or the first nucleotide of the codon) are in bold. Nucleotides complementary to the S-K-L codons are in bold italic. Intron nucleotide sequence is in lower case.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7159169" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7159170 id="7159170" class="section-title" data-legacy-id=s2c>2.3. Genetic modifications</h3>
<p>Three constructions were made to determine the phenotypes of the strains containing deletions of either the short or long introns of <em>YlMDH2</em>. The primers used for PCR amplification are listed in Table <a reveal-id="DSS007TB2" data-open="DSS007TB2" class="link link-reveal link-table xref-fig">2</a>. The forward primer contained a <em>Cla</em>I restriction site and stop codons in the three phases. The reverse primers contained an <em>Avr</em>II site and sequences corresponding to the wild-type gene and to the two variants. The corresponding <em>Cla</em>I-<em>Avr</em>II PCR fragments were cloned into the JMP62 <em>URA3ex</em> vector<sup><a reveal-id="DSS007C31" data-open="DSS007C31" class="link link-ref link-reveal xref-bibr">31</a></sup> previously digested with <em>Cla</em>I-<em>Avr</em>II. The resulting plasmids were purified, digested with <em>Bam</em>HI (a <em>Bam</em>HI site is present in the middle of the gene), and used to transform the PO1d strain by the lithium acetate method.<sup><a reveal-id="DSS007C32" data-open="DSS007C32" class="link link-ref link-reveal xref-bibr">32</a></sup> After single crossing-over, the mutated version had integrated at the <em>YlMDH2</em> locus under its own promoter and the genomic version was invalidated by the introduction of stop codons at the 5′-end of the wild-type gene (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The resulting strains JMY1699, JMY1707 and JMY1711 (Table <a reveal-id="DSS007TB1" data-open="DSS007TB1" class="link link-reveal link-table xref-fig">1</a>) expressed the predicted cytosolic, peroxisomal and wild-type versions of the <em>YlMDH2</em> gene products and were confirmed by southern blot (data not shown). Prototrophic strains were obtained by transforming the different mutants with the <em>LEU2</em> marker, which gave rise to the strains JMY2416, JMY2426 and JMY2428, respectively (Table <a reveal-id="DSS007TB1" data-open="DSS007TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>We monitored localization of the MDH proteins in different subcellular compartments by expressing recombinant genes that encoded the peroxisomal or cytosolic mutant or the wild-type protein fused to the eYFP fluorescent protein tag at their N-termini. The three variant constructs were PCR amplified as <em>Bcl</em>I-<em>Avr</em>II fragments using specific primers (Table <a reveal-id="DSS007TB2" data-open="DSS007TB2" class="link link-reveal link-table xref-fig">2</a>) and then cloned into the JMP62 <em>URA3ex</em>-YFP-N vector previously digested by <em>Bam</em>HI and <em>Avr</em>II. This vector was derived from the JMP62 <em>URA3ex</em> vector<sup><a reveal-id="DSS007C31" data-open="DSS007C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and allows N-terminal, eYFP-tagged proteins to be produced under control of the <em>POX2</em> promoter. The three different expression cassettes were excised from their respective vectors by <em>Not</em>I digestion, gel purified, and subsequently used to transform PO1d to create the JMY2451, JMY2457 and JMY2459 strains, respectively (Table <a reveal-id="DSS007TB1" data-open="DSS007TB1" class="link link-reveal link-table xref-fig">1</a>). These strains were subsequently transformed with constructions containing the sequence of the fluorescent protein RedStar2. The RedStar2 gene was PCR amplified from the plasmid pYM43<sup><a reveal-id="DSS007C33" data-open="DSS007C33" class="link link-ref link-reveal xref-bibr">33</a></sup> using primers containing <em>Bam</em>HI and <em>Avr</em>II restriction sites (Table <a reveal-id="DSS007TB2" data-open="DSS007TB2" class="link link-reveal link-table xref-fig">2</a>) and cloned into the JMP62 <em>LEU2ex</em> pTEF vector previously digested by <em>Bam</em>HI and <em>Avr</em>II. The JMP62 <em>LEU2ex</em> pTEF plasmid corresponds to JMP62 <em>LEU2ex</em> in which the pPOX2 promoter has been exchanged with the pTEF promoter.<sup><a reveal-id="DSS007C34" data-open="DSS007C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Primers were designed to add or not add a PTS1 consensus sequence (Ser-Lys-Leu; SKL) onto the C-terminus of RedStar2 and allow expression of cytosolic or peroxisome-targeted RedStar2 protein. Each RedStar2 expression cassette was released by <em>Not</em>I digestion, gel purified, and subsequently used to transform the JMY2451, JMY2457 and JMY2459 strains. The resulting strains harboured one of the three different eYFP-tagged MDH protein expression constructs controlled by the <em>POX2</em> promoter and an addition construct for expressing either the RedStar2 or the RedStar2-SKL protein under control of pTEF. All the strains generated are listed in Table <a reveal-id="DSS007TB1" data-open="DSS007TB1" class="link link-reveal link-table xref-fig">1</a>.</p>                            <h3 scrollto-destination=7159173 id="7159173" class="section-title" data-legacy-id=s2d>2.4. Microscopy</h3>
<p>Images were acquired using a Zeiss Axio Imager.M2 microscope (Zeiss, Le Pecq, France) using a 100× objective and Zeiss filters 45 and 46 for fluorescent microscopy. Image acquisition was performed with the Axiovision 4.8 software (Zeiss). All images were post-processed similarly (background reduction) and merged using ImageJ software (<a class="link link-uri" href="http://rsbweb.nih.gov/ij/">http://rsbweb.nih.gov/ij/</a>). Strains were grown 12h on YNBE 2% OA, YNBE 2% D or YNBE with both 1% D and 2% OA.</p>                            <h3 scrollto-destination=7159175 id="7159175" class="section-title" data-legacy-id=s2e>2.5. RNA-seq analysis of YlMDH2 transcripts</h3>
<p>RNA-seq data produced in the lab were screened for the different splicing variants of <em>YlMDH2</em>. Total RNAs from cells cultivated in six different conditions (glucose, OA, TO, TB, alkane, and glycerol) were prepared using the Qiagen RNeasy kit (Qiagen). mRNAs were purified by the selection of poly(A)<sup>+</sup> transcripts, which were then sequenced by the Illumina Solexa sequencing technology with either a Genome Analyzer IIX or a HiSeq sequencing system. Thirteen to 29 million single-end reads were generated per sample with a read length of 36 nt, 50 nt or 100 nt (accession numbers E-MTAB-939 and E-MTAB-940). The reads were aligned to both forms of <em>YlMDH2</em> spliced transcripts (long or short intron) using SOAPaligner version 2.20.<sup><a reveal-id="DSS007C35" data-open="DSS007C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Only reads aligned to the exon–exon junction (34 nt, 48 nt or 98 nt from each exon, depending on the read length) of intron 2 were counted and the ratio between the two isoforms was determined. We used Fisher's exact tests to determine whether the expression levels of <em>YlMDH2</em> differed in the different growth conditions and to determine the significance of the differences observed among the AS ratios.</p>                            <h3 scrollto-destination=7159177 id="7159177" class="section-title" data-legacy-id=s2f>2.6. Bioinformatics</h3>
<p>Homologues of <em>S. cerevisiae</em> MDH were identified in fully sequenced <em>Y. lipolytica</em> genomes from Génolevures (<a class="link link-uri" href="http://www.genolevures.org/">http://www.genolevures.org/</a>) using BLASTp.<sup><a reveal-id="DSS007C36" data-open="DSS007C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Homologues were also found in <em>Pichia pastoris</em> (Pichia_GS115.pep_0509; <a class="link link-uri" href="https://bioinformatics.psb.ugent.be/gdb/pichia/">https://bioinformatics.psb.ugent.be/gdb/pichia/</a>) and in <em>Schizosaccharomyces pombe</em> (pompep_17022010; <a class="link link-uri" href="http://www.sanger.ac.uk/Projects/S_pombe/">http://www.sanger.ac.uk/Projects/S_pombe/</a>). Amino acids sequences were then aligned using MUSCLE<sup><a reveal-id="DSS007C37" data-open="DSS007C37" class="link link-ref link-reveal xref-bibr">37</a></sup> or Multalin<sup><a reveal-id="DSS007C38" data-open="DSS007C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and columns of gap-containing residues were removed manually. Phylogenetic trees were constructed with the Neighbour-Joining algorithm using ClustalX.<sup><a reveal-id="DSS007C39" data-open="DSS007C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>Predictions for mitochondrial protein targeting were performed using MITOPROT II version 1.101 (<a class="link link-uri" href="http://ihg.gsf.de/ihg/mitoprot.html">http://ihg.gsf.de/ihg/mitoprot.html</a>).<sup><a reveal-id="DSS007C40" data-open="DSS007C40" class="link link-ref link-reveal xref-bibr">40</a></sup> PTS1 targeting sequence predictions were performed using the PTS1 predictor using default cut-off (<a class="link link-uri" href="http://mendel.imp.ac.at/mendeljsp/sat/pts1/PTS1predictor.jsp">http://mendel.imp.ac.at/mendeljsp/sat/pts1/PTS1predictor.jsp</a>).<sup><a reveal-id="DSS007C41" data-open="DSS007C41" class="link link-ref link-reveal xref-bibr">41</a></sup></p>                            <h2 scrollto-destination=7159180 id="7159180" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7159181 id="7159181" class="section-title" data-legacy-id=s3a>3.1. Malate dehydrogenase protein families in hemiascomycetous yeasts</h3>
<p>In <em>S. cerevisiae</em>, the three genes that encode MDH are: YKL085W (<em>MDH1</em>), which encodes the mitochondrial enzyme;<sup><a reveal-id="DSS007C14" data-open="DSS007C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSS007C15" data-open="DSS007C15" class="link link-ref link-reveal xref-bibr">15</a></sup> YOL126C (<em>MDH2</em>), which encodes the cytosolic enzyme;<sup><a reveal-id="DSS007C16" data-open="DSS007C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSS007C17" data-open="DSS007C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and YDL378C (<em>MDH3</em>), which encodes the peroxisomal enzyme.<sup><a reveal-id="DSS007C18" data-open="DSS007C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS007C19" data-open="DSS007C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The three isozymes show amino acid identities ranging from 43 to 50%.<sup><a reveal-id="DSS007C19" data-open="DSS007C19" class="link link-ref link-reveal xref-bibr">19</a></sup> A Blast search for <em>S. cerevisiae</em> MDH homologues in the <em>Y. lipolytica</em> genome<sup><a reveal-id="DSS007C42" data-open="DSS007C42" class="link link-ref link-reveal xref-bibr">42</a></sup> identified only two genes, YALI0D16753g (<em>YlMDH1</em>) and YALI0E14190g (<em>YlMDH2</em>). Both genes exhibit the highest levels of amino acid conservation with the <em>S. cerevisiae MDH1</em> gene, with 63% identity and 79% similarity for YALI0D16753g and 47% identity and 62% similarity for <em>YlMDH2</em>. The computed proteome families of the Génolevures database<sup><a reveal-id="DSS007C24" data-open="DSS007C24" class="link link-ref link-reveal xref-bibr">24</a></sup> revealed that both <em>Y. lipolytica</em> MDH genes belong to the GL3R0092 family, which includes all of the MDH proteins encoded by the nine fully sequenced yeast genomes in the Génolevures database. All of these genomes contain three MDH genes. The only exceptions are <em>Y. lipolytica</em> (two genes) and <em>Lachancea (Kluyveromyces) thermotolerans</em>, which possesses three additional genes. We found these additional genes (KLTH0D00440g, KLTH0G19536g and KLTH0G19558g) to be closely related to each other (64–66% identity and 80–83% similarity) but very divergent from all other hemiascomycetous MDH genes. Due to their mutual similarity and their localization (one is subtelomeric and two are repeated in tandem), they likely represent species-specific gene duplications. Given their divergence from the other hemiascomycete proteins, we did not consider them in our phylogenetic analysis. As <em>Y. lipolytica</em> is relatively isolated among the hemiascomycetes from a phylogenetic point of view, we added two other yeasts, not included in the previous yeast set, more closely related to <em>Y. lipolytica</em>: <em>Arxula adeninivorans</em> (recently sequenced and annotated by the Génolevures consortium, to be published) and <em>P. pastoris.</em><sup><a reveal-id="DSS007C43" data-open="DSS007C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Like <em>Y. lipolytica</em>, these two species contain only two MDH genes. We used these data to construct a phylogenetic tree of MDHs in the hemiascomycetous yeasts with the unique MDH gene from <em>S. pombe</em> as the outgroup (Fig. <a reveal-id="DSS007F1" data-open="DSS007F1" class="link link-reveal link-table xref-fig">1</a>A). Simultaneously, we performed <em>in silico</em> sequence analysis to predict mitochondrial and peroxisomal localization for the different genes using MITOPROT and PTS1 prediction online software (Fig. <a reveal-id="DSS007F1" data-open="DSS007F1" class="link link-reveal link-table xref-fig">1</a>B). The phylogenetic tree highlights clear distinctions between the three types of MDH. The most conserved branch corresponds to the mitochondrial MDH, with high levels of amino acid conservation among the different proteins: 61–85% identity and 76–92% similarity (Fig. <a reveal-id="DSS007F1" data-open="DSS007F1" class="link link-reveal link-table xref-fig">1</a>A). The peroxisomal branch is less conserved, with minimums of 34% identity and 44% similarity for the most distantly related proteins. The cytosolic MDHs segregated into two branches, with similarity and identity values close to those obtained for the peroxisomal branch (35–62% identity and 45–77% similarity). According to the localization predictions, only one gene, DEHA2F09020g from <em>Debaryomyces hansenii</em>, which has a PTS1 peroxisome targeting sequence (C-terminal sequence SKL and PTS1 predictor score of −8.044), is mislocalized in the cytosolic branch (Fig. <a reveal-id="DSS007F1" data-open="DSS007F1" class="link link-reveal link-table xref-fig">1</a>). <em>Eremothecium</em> (<em>Ashbya</em>) <em>gossypii</em> has three MDH genes, but the localization prediction failed to identify any PTS1 sequence in these three genes. The ERGO0D15114g protein belonging to the peroxisomal branch harbours an unexpectedly long C-terminal extension of ∼150 amino acids. In a recently corrected version of the <em>Ashbya</em> Genome Database (<a class="link link-uri" href="http://agd.vital-it.ch/index.html">http://agd.vital-it.ch/index.html</a><sup><a reveal-id="DSS007C44" data-open="DSS007C44" class="link link-ref link-reveal xref-bibr">44</a></sup>; version 11 of 26 May 2011), the gene AGOS_ADR252W encodes a protein of 339 amino acids (<em>vs</em> 486 aa in ERGO0D15114p) and has a putative PTS1 with a significant score (C-terminal sequence ARL and PTS1 predictor score 3.865). </p>                        <a id="7159183" scrollto-destination="7159183"></a>
<div id="DSS007F1" data-id="DSS007F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00701.jpeg?Expires=2147483647&Signature=zcEJHq7ZCAuhJYu5FMZveLvhBcwSAjZOpqLhtha8QjLFSm0RjM0sMkIU7XBOz8ea0qxuzZMHTy~HERt79JtASycSiYvQNTlkHWIAv~JvxUquU5ApWImz9l7Qn8f2eJoLAd2gzVf-q~TiEhH3A9p1pM6uOHeoRArwCYCZQPcfE3xFx3PqwX~iXxK-vF8VUOhlxGJYCd2CJ8eVvCsU3ONFJG3F40IT4p0IJgjbehQyBbYPGy9u~EaTWlpSqGsRz1uKcXaBIfP3m1UUn6zoXkatxwxaCxnjLT1lNG35GCps5L-q29olXU703ldQUKU2Cv2lLjdq5GZ8SD4e5wZLdH-joQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Phylogenetic tree of the MDHs from 11 fully sequenced hemiascomycetous yeast species using the unique MDH gene from S. pombe (SPCC306.08) as an outgroup. The yeasts are Candida glabrata (CAGL), D. hansenii (DEHA), Kluyveromyces lactis (KLLA), Lachancea thermotolerans (KLTH), Lachancea kluyveri (SAKL), Zygosaccharomyces rouxii (ZYRO), Eremothecium gossypii (ERGO), P. pastoris (Pipas), A. adeninivorans (ARAD), Y. lipolytica (YALI) and S. cerevisiae. The blue zone indicates genes with a typical mitochondrial targeting sequence and the pink zone indicates genes with a potential PTS. The percentages of amino acid identity and similarity among the proteins of each group (peroxisomal, mitochondrial and cytosolic) are indicated next to each group. (B) In silico predictions of protein targeting. The MITOPROT score, PTS1 predictor score and C-terminal sequence are indicated for each MDH." path-from-xml="dss00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159183/dss00701.jpeg" path-from-xml="dss00701.jpeg" target="_blank">View large</a><a section="7159183" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00701.jpeg?Expires=2147483647&Signature=kT6n6zFxbawl~v09X46gpql20W~ptcWDf6lFI1dalPQTsCXLMK2M6feCK-noa2IMXBvsjgkpvMqPWLDhcavXm~rWNkamWxLRIcHz4GY0Ht3bf7dfcxnrQIREZ478893MVe9KUsntDnH~105j~qhhxHiIs0E5EAdmF8A61uQG9lldovf6ZMA7mWUTBFIf-s4-DtkpUQt6Cn34TH0ddaRGMSqJacAd5ljnsnbm7wJK4C9-ModSuSiyEt8OgdZvJLVheC1Zb3qCKhuGTyJpHOThhiVpRkpDVYapTgl7mu1EbnCxXchrSP1MUVSo4blxHUtMz8qXdgfLjNyrN74P9bq9Xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159183&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Phylogenetic tree of the MDHs from 11 fully sequenced hemiascomycetous yeast species using the unique MDH gene from <em>S. pombe</em> (SPCC306.08) as an outgroup. The yeasts are <em>Candida glabrata</em> (CAGL), <em>D. hansenii</em> (DEHA), <em>Kluyveromyces lactis</em> (KLLA), <em>Lachancea thermotolerans</em> (KLTH), <em>Lachancea kluyveri</em> (SAKL), <em>Zygosaccharomyces rouxii</em> (ZYRO), <em>Eremothecium gossypii</em> (ERGO), <em>P. pastoris</em> (Pipas), <em>A. adeninivorans</em> (ARAD), <em>Y. lipolytica</em> (YALI) and <em>S. cerevisiae</em>. The blue zone indicates genes with a typical mitochondrial targeting sequence and the pink zone indicates genes with a potential PTS. The percentages of amino acid identity and similarity among the proteins of each group (peroxisomal, mitochondrial and cytosolic) are indicated next to each group. (B) <em>In silico</em> predictions of protein targeting. The MITOPROT score, PTS1 predictor score and C-terminal sequence are indicated for each MDH.</p></div></div><div content-id="DSS007F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00701.jpeg?Expires=2147483647&Signature=zcEJHq7ZCAuhJYu5FMZveLvhBcwSAjZOpqLhtha8QjLFSm0RjM0sMkIU7XBOz8ea0qxuzZMHTy~HERt79JtASycSiYvQNTlkHWIAv~JvxUquU5ApWImz9l7Qn8f2eJoLAd2gzVf-q~TiEhH3A9p1pM6uOHeoRArwCYCZQPcfE3xFx3PqwX~iXxK-vF8VUOhlxGJYCd2CJ8eVvCsU3ONFJG3F40IT4p0IJgjbehQyBbYPGy9u~EaTWlpSqGsRz1uKcXaBIfP3m1UUn6zoXkatxwxaCxnjLT1lNG35GCps5L-q29olXU703ldQUKU2Cv2lLjdq5GZ8SD4e5wZLdH-joQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Phylogenetic tree of the MDHs from 11 fully sequenced hemiascomycetous yeast species using the unique MDH gene from S. pombe (SPCC306.08) as an outgroup. The yeasts are Candida glabrata (CAGL), D. hansenii (DEHA), Kluyveromyces lactis (KLLA), Lachancea thermotolerans (KLTH), Lachancea kluyveri (SAKL), Zygosaccharomyces rouxii (ZYRO), Eremothecium gossypii (ERGO), P. pastoris (Pipas), A. adeninivorans (ARAD), Y. lipolytica (YALI) and S. cerevisiae. The blue zone indicates genes with a typical mitochondrial targeting sequence and the pink zone indicates genes with a potential PTS. The percentages of amino acid identity and similarity among the proteins of each group (peroxisomal, mitochondrial and cytosolic) are indicated next to each group. (B) In silico predictions of protein targeting. The MITOPROT score, PTS1 predictor score and C-terminal sequence are indicated for each MDH." path-from-xml="dss00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159183/dss00701.jpeg" path-from-xml="dss00701.jpeg" target="_blank">View large</a><a section="7159183" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00701.jpeg?Expires=2147483647&Signature=kT6n6zFxbawl~v09X46gpql20W~ptcWDf6lFI1dalPQTsCXLMK2M6feCK-noa2IMXBvsjgkpvMqPWLDhcavXm~rWNkamWxLRIcHz4GY0Ht3bf7dfcxnrQIREZ478893MVe9KUsntDnH~105j~qhhxHiIs0E5EAdmF8A61uQG9lldovf6ZMA7mWUTBFIf-s4-DtkpUQt6Cn34TH0ddaRGMSqJacAd5ljnsnbm7wJK4C9-ModSuSiyEt8OgdZvJLVheC1Zb3qCKhuGTyJpHOThhiVpRkpDVYapTgl7mu1EbnCxXchrSP1MUVSo4blxHUtMz8qXdgfLjNyrN74P9bq9Xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159183&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Phylogenetic tree of the MDHs from 11 fully sequenced hemiascomycetous yeast species using the unique MDH gene from <em>S. pombe</em> (SPCC306.08) as an outgroup. The yeasts are <em>Candida glabrata</em> (CAGL), <em>D. hansenii</em> (DEHA), <em>Kluyveromyces lactis</em> (KLLA), <em>Lachancea thermotolerans</em> (KLTH), <em>Lachancea kluyveri</em> (SAKL), <em>Zygosaccharomyces rouxii</em> (ZYRO), <em>Eremothecium gossypii</em> (ERGO), <em>P. pastoris</em> (Pipas), <em>A. adeninivorans</em> (ARAD), <em>Y. lipolytica</em> (YALI) and <em>S. cerevisiae</em>. The blue zone indicates genes with a typical mitochondrial targeting sequence and the pink zone indicates genes with a potential PTS. The percentages of amino acid identity and similarity among the proteins of each group (peroxisomal, mitochondrial and cytosolic) are indicated next to each group. (B) <em>In silico</em> predictions of protein targeting. The MITOPROT score, PTS1 predictor score and C-terminal sequence are indicated for each MDH.</p></div></div><p>The product of the <em>Y. lipolytica YlMDH1</em> gene is predicted to be targeted to the mitochondria (MITOPROT probability 0.9891), whereas the product of the <em>YlMDH2</em> gene is not (MITOPROT probability 0.3528). Neither of these genes harbours any PTSs. This is in line with their positions in the phylogenetic tree, i.e. <em>YlMDH1</em> is positioned within the mitochondrial branch, and <em>YlMDH2</em> is within the cytosolic branch of the tree. Similarly, <em>A. adeninivorans</em> and <em>P. pastoris</em> both have a mitochondrial form (ARAD1B14036g and Pipas_chr2-1_0238, respectively) and a cytosolic form (ARAD1D04884g and Pipas_chr4_0815, respectively) according to phylogeny and signal sequence prediction. Consequently, for these three closely related species, sequence analysis of their complete genomes did not identify any peroxisomal forms of MDH.</p>                            <h3 scrollto-destination=7159185 id="7159185" class="section-title" data-legacy-id=s3b>3.2. The YlMDH2 gene is alternatively spliced</h3>
<p>The peroxisomal form of MDH has been reported to be mandatory for fatty acid degradation in <em>S. cerevisiae</em>.<sup><a reveal-id="DSS007C45" data-open="DSS007C45" class="link link-ref link-reveal xref-bibr">45</a></sup><em>Yarrowia lipolytica</em> is an oleagineous yeast, with an affinity for fatty acids as a carbon source. Thus, the absence of a peroxisomal form of MDH was unlikely. A deeper analysis of the sequence of <em>YlMDH2</em> revealed a putative alternative 3′-splice site for the second intron, defined by comparison with the splicing sequence pattern of the genosplicing database (<a class="link link-uri" href="http://genome.jouy.inra.fr/genosplicing2">http://genome.jouy.inra.fr/genosplicing2</a>). The gene model of <em>YlMDH2</em> is represented in Fig. <a reveal-id="DSS007F2" data-open="DSS007F2" class="link link-reveal link-table xref-fig">2</a>A, with the positions of the putative intron boundaries (5′- and 3′-splice sites) and the branch point (BP) at the exon 2–intron 2–exon 3 junctions in Fig. <a reveal-id="DSS007F2" data-open="DSS007F2" class="link link-reveal link-table xref-fig">2</a>B. The putative AS event leads to a very minor change in the length of the mRNA sequence (4 nt) but introduces a 3′-frameshift and thus generates different stop codons. If the intron is spliced at the upstream 3′-splice site (short intron), the amino acid sequence of the C-terminus encoded by the <em>YlMDH2</em> mRNA is P-A-N. If splicing occurs at the downstream 3′-splice site (long intron), the C-terminal amino acid sequence ends with A-K-I (Fig. <a reveal-id="DSS007F2" data-open="DSS007F2" class="link link-reveal link-table xref-fig">2</a>B), which is a characteristic PTS1 sequence identified in several yeasts.<sup><a reveal-id="DSS007C21" data-open="DSS007C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS007C23" data-open="DSS007C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                        <a id="7159187" scrollto-destination="7159187"></a>
<div id="DSS007F2" data-id="DSS007F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00702.jpeg?Expires=2147483647&Signature=xDGowD1uIDc75~PRbur52U5JO-gUf1FuwI2PMrxZeP0y9RS~TTEqpjgX34kcbgn~cdy80rsFa8tH7BKOepvyhL65xEQWbNcGA6yTlwmhlwMDlIMPVU2vnJw122cjR0WSEufvROwAtNjQlK8Y6RNpf1qu5a8Nb4~ciASZ5h6y70OJIrIbf6gmu-KKIF0hwFtQltdLEUogWve30KjP22Zf14F2NPJvP-tT-cNv3TMBkX6omODdpIEzxz8o00jMkoHyILsTulWHj0yKC4KtU20lFcxNnp-TsR0u~qK120jFmI6sGUmi~eDycwpsqybyhOK0u~QVaNt~JyNp7rnfyG9s9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene model for YlMDH2. (A) Schematic representation of alternative transcripts from the multi-intronic MDH gene YlMDH2. Exons are represented by grey rectangles and introns are symbolized by thin black articulated lines. Vertical bars on each of the three phases (0, +1 and +2) represent in-frame stop codons. (B) Sequence representation of the 3′ regions centred on the second intron. Coloured circles indicate the 5′-splice site, the BP and the two 3′-splice sites used to generate the two mRNA variants. Exon parts are represented by grey rectangles and the two putative C-terminal protein sequences are indicated." path-from-xml="dss00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159187/dss00702.jpeg" path-from-xml="dss00702.jpeg" target="_blank">View large</a><a section="7159187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00702.jpeg?Expires=2147483647&Signature=m1b7ytmx0vjWzXV8hxNO0nqKsDgl4ZtbyB6a~FJMbuNPuzGdcz14ovBDPSP5epAIy2jbxFg5TOGpyIFM9K5CJZercndUGAJicnlFjRtRe7YYTZjDHtoSvf0JzYwOKlVC-RErxctDjmwJbJg-O4Mbn9pIiprwDK2-~8aGO40KGRVdWsJ-iOgF6Hm~S32l899plA4-S1cUSnLAgAd7vdXS-yyf2D0m-Nj4oJSxXqha245prnO2B8100qYZYDyJBRNS~9EIM0-RZwWrQ0OMrzZWcS1NEbKQM-A~7gT~j7kAXd0Uj8pA6Io9lfRFPwK4keM06axgKGmKSWBf7jnofk0V~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159187&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene model for <em>YlMDH2</em>. (A) Schematic representation of alternative transcripts from the multi-intronic MDH gene <em>YlMDH2</em>. Exons are represented by grey rectangles and introns are symbolized by thin black articulated lines. Vertical bars on each of the three phases (0, +1 and +2) represent in-frame stop codons. (B) Sequence representation of the 3′ regions centred on the second intron. Coloured circles indicate the 5′-splice site, the BP and the two 3′-splice sites used to generate the two mRNA variants. Exon parts are represented by grey rectangles and the two putative C-terminal protein sequences are indicated.</p></div></div><div content-id="DSS007F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00702.jpeg?Expires=2147483647&Signature=xDGowD1uIDc75~PRbur52U5JO-gUf1FuwI2PMrxZeP0y9RS~TTEqpjgX34kcbgn~cdy80rsFa8tH7BKOepvyhL65xEQWbNcGA6yTlwmhlwMDlIMPVU2vnJw122cjR0WSEufvROwAtNjQlK8Y6RNpf1qu5a8Nb4~ciASZ5h6y70OJIrIbf6gmu-KKIF0hwFtQltdLEUogWve30KjP22Zf14F2NPJvP-tT-cNv3TMBkX6omODdpIEzxz8o00jMkoHyILsTulWHj0yKC4KtU20lFcxNnp-TsR0u~qK120jFmI6sGUmi~eDycwpsqybyhOK0u~QVaNt~JyNp7rnfyG9s9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene model for YlMDH2. (A) Schematic representation of alternative transcripts from the multi-intronic MDH gene YlMDH2. Exons are represented by grey rectangles and introns are symbolized by thin black articulated lines. Vertical bars on each of the three phases (0, +1 and +2) represent in-frame stop codons. (B) Sequence representation of the 3′ regions centred on the second intron. Coloured circles indicate the 5′-splice site, the BP and the two 3′-splice sites used to generate the two mRNA variants. Exon parts are represented by grey rectangles and the two putative C-terminal protein sequences are indicated." path-from-xml="dss00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159187/dss00702.jpeg" path-from-xml="dss00702.jpeg" target="_blank">View large</a><a section="7159187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00702.jpeg?Expires=2147483647&Signature=m1b7ytmx0vjWzXV8hxNO0nqKsDgl4ZtbyB6a~FJMbuNPuzGdcz14ovBDPSP5epAIy2jbxFg5TOGpyIFM9K5CJZercndUGAJicnlFjRtRe7YYTZjDHtoSvf0JzYwOKlVC-RErxctDjmwJbJg-O4Mbn9pIiprwDK2-~8aGO40KGRVdWsJ-iOgF6Hm~S32l899plA4-S1cUSnLAgAd7vdXS-yyf2D0m-Nj4oJSxXqha245prnO2B8100qYZYDyJBRNS~9EIM0-RZwWrQ0OMrzZWcS1NEbKQM-A~7gT~j7kAXd0Uj8pA6Io9lfRFPwK4keM06axgKGmKSWBf7jnofk0V~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159187&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene model for <em>YlMDH2</em>. (A) Schematic representation of alternative transcripts from the multi-intronic MDH gene <em>YlMDH2</em>. Exons are represented by grey rectangles and introns are symbolized by thin black articulated lines. Vertical bars on each of the three phases (0, +1 and +2) represent in-frame stop codons. (B) Sequence representation of the 3′ regions centred on the second intron. Coloured circles indicate the 5′-splice site, the BP and the two 3′-splice sites used to generate the two mRNA variants. Exon parts are represented by grey rectangles and the two putative C-terminal protein sequences are indicated.</p></div></div><p>We generated three cDNA libraries from <em>Y. lipolytica</em> cells cultivated in media containing either glucose (exponential and stationary phases) or oleate (exponential phase) as the carbon source<sup><a reveal-id="DSS007C1" data-open="DSS007C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and obtained 30 cDNA clones corresponding to <em>YlMDH2</em> transcripts. All of these clones derived from cells harvested from glucose medium: 10 clones from cells harvested in the exponential phase (expo library) and 20 clones from cells harvested in the stationary phase (stat library). We sequenced the 3′-ends of 11 of these <em>YlMDH2</em> cDNA clones and found that seven corresponded to mRNAs spliced at the upstream 3′-splice site (short intron 2 splice) and four corresponded to mRNAs spliced at the downstream 3′-splice site (long intron 2). As expected from our model, the two groups of cDNAs differed by only four nucleotides (Fig. <a reveal-id="DSS007F2" data-open="DSS007F2" class="link link-reveal link-table xref-fig">2</a>B) and thus validated AS for <em>YlMDH2</em>. The different numbers of cDNAs for the two spliced forms (five cDNAs with a short intron and two cDNAs with a long intron in the expo library, and only four cDNAs with a long intron, none with a short intron, in the expo library) was a first indication that the AS was regulated by the environment of the cells, i.e<em>.</em> the growth conditions. However, due to the low number of cDNAs sequenced, the statistical significance of this difference could not be confirmed. This prompted us to investigate larger data sets using RNA-seq experiments.</p><p>In a first experiment (Illumina Solexa with single reads of 36 bp; Table <a reveal-id="DSS007TB3" data-open="DSS007TB3" class="link link-reveal link-table xref-fig">3</a>) involving six different growth conditions, we retrieved a total of 1829 RNA-seq reads containing the exon–exon junction of the second intron of <em>YlMDH2</em>: 1181 reads corresponded to the short intron variant, and 648 reads corresponded to the long intron variant. The short intron/long intron usage ratio varied from 0.9 to 5.05 according to the carbon source used for growth (Table <a reveal-id="DSS007TB3" data-open="DSS007TB3" class="link link-reveal link-table xref-fig">3</a>). These results strongly suggest that environmental conditions influence the regulation of <em>YlMDH2</em> AS. For example, for <em>Y. lipolytica</em> grown on TB medium, the number of transcripts containing the upstream 3′-splice site is five times the number of transcripts containing the downstream 3′-splice site. This ratio is significantly different from those found in all other conditions (<em>P</em> &lt; 0.0001, for all comparisons to TB). In contrast, the levels of both variants were equivalent in cells grown on alkane. Additional RNA-seq experiments with two replicates on glucose (HiSeq with single reads of 50 bp; Table <a reveal-id="DSS007TB3" data-open="DSS007TB3" class="link link-reveal link-table xref-fig">3</a>) and OA (HiSeq with paired-end reads of 2 × 100 bp) media confirmed that both ratios are conserved. However, these promising results on splicing regulation are presented as preliminary results since they came from a single time point, with a single concentration of carbon source and using RNA-seq as a single method of quantification. They obviously should be confirmed by additional experiments and alternative methods to understand the kinetic and the carbon source concentration-dependent regulations. This will be further investigated and will constitute the aim of a separate study. </p>                        <a id="7159190" scrollto-destination="7159190"></a>
<div content-id="DSS007TB3" class="table-modal"><div class="table-wrap"><div id="DSS007TB3" class="table-wrap-title" data-id="DSS007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of reads specific for the short or long intron 2 of <em>YlMDH2</em> obtained by RNA-seq analysis under various growth conditions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Carbon source </th><th>Illumina Solexa sequencing technology and read length </th><th>Total reads </th><th>Reads mapping short intron 2 </th><th>Reads mapping long intron 2 </th><th>Ratio short/long </th></tr></thead><tbody><tr><td>Triolein </td><td>GAIIX single reads 36 nt </td><td>19 589 043 </td><td>94 (57%) </td><td>71 (43%) </td><td>1.33 </td></tr><tr><td>Tributyrin </td><td>GAIIX single reads 36 nt </td><td>15 996 762 </td><td>298 (83%) </td><td>59 (17%) </td><td>5.05<sup>a</sup> </td></tr><tr><td>Glycerol </td><td>GAIIX single reads 36 nt </td><td>22 935 285 </td><td>308 (62%) </td><td>186 (38%) </td><td>1.66 </td></tr><tr><td>Alkane </td><td>GAIIX single reads 36 nt </td><td>23 903 839 </td><td>99 (47%) </td><td>110 (53%) </td><td>0.9<sup>b</sup> </td></tr><tr><td>Glucose </td><td>GAIIX single reads 36 nt </td><td>13 022 908 </td><td>121 (68%) </td><td>56 (32%) </td><td>2.16 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>12 965 094 </td><td>202 (67%) </td><td>86 (33%) </td><td>2.21 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>10 592 928 </td><td>203 (63%) </td><td>116 (37%) </td><td>1.70 </td></tr><tr><td>Oleic acid </td><td>GAIIX single reads 36 nt </td><td>19 270 870 </td><td>261 (61%) </td><td>166 (39%) </td><td>1.57 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>25 974 957 </td><td>118 (60%) </td><td>78 (40%) </td><td>1.51 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>13 316 230 </td><td>260 (67%) </td><td>130 (33%) </td><td>2.00 </td></tr></tbody></table></div><div content-id="DSS007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Carbon source </th><th>Illumina Solexa sequencing technology and read length </th><th>Total reads </th><th>Reads mapping short intron 2 </th><th>Reads mapping long intron 2 </th><th>Ratio short/long </th></tr></thead><tbody><tr><td>Triolein </td><td>GAIIX single reads 36 nt </td><td>19 589 043 </td><td>94 (57%) </td><td>71 (43%) </td><td>1.33 </td></tr><tr><td>Tributyrin </td><td>GAIIX single reads 36 nt </td><td>15 996 762 </td><td>298 (83%) </td><td>59 (17%) </td><td>5.05<sup>a</sup> </td></tr><tr><td>Glycerol </td><td>GAIIX single reads 36 nt </td><td>22 935 285 </td><td>308 (62%) </td><td>186 (38%) </td><td>1.66 </td></tr><tr><td>Alkane </td><td>GAIIX single reads 36 nt </td><td>23 903 839 </td><td>99 (47%) </td><td>110 (53%) </td><td>0.9<sup>b</sup> </td></tr><tr><td>Glucose </td><td>GAIIX single reads 36 nt </td><td>13 022 908 </td><td>121 (68%) </td><td>56 (32%) </td><td>2.16 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>12 965 094 </td><td>202 (67%) </td><td>86 (33%) </td><td>2.21 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>10 592 928 </td><td>203 (63%) </td><td>116 (37%) </td><td>1.70 </td></tr><tr><td>Oleic acid </td><td>GAIIX single reads 36 nt </td><td>19 270 870 </td><td>261 (61%) </td><td>166 (39%) </td><td>1.57 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>25 974 957 </td><td>118 (60%) </td><td>78 (40%) </td><td>1.51 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>13 316 230 </td><td>260 (67%) </td><td>130 (33%) </td><td>2.00 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The ratio on tributyrin is statistically different from that on all other media (<em>P</em> &lt; 0.0001).<sup>b</sup>The ratio on alkane is statistically different from that on all other media, except on triolein (<em>P</em> = 0.0765).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7159190" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS007TB3" class="table-wrap-title" data-id="DSS007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of reads specific for the short or long intron 2 of <em>YlMDH2</em> obtained by RNA-seq analysis under various growth conditions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Carbon source </th><th>Illumina Solexa sequencing technology and read length </th><th>Total reads </th><th>Reads mapping short intron 2 </th><th>Reads mapping long intron 2 </th><th>Ratio short/long </th></tr></thead><tbody><tr><td>Triolein </td><td>GAIIX single reads 36 nt </td><td>19 589 043 </td><td>94 (57%) </td><td>71 (43%) </td><td>1.33 </td></tr><tr><td>Tributyrin </td><td>GAIIX single reads 36 nt </td><td>15 996 762 </td><td>298 (83%) </td><td>59 (17%) </td><td>5.05<sup>a</sup> </td></tr><tr><td>Glycerol </td><td>GAIIX single reads 36 nt </td><td>22 935 285 </td><td>308 (62%) </td><td>186 (38%) </td><td>1.66 </td></tr><tr><td>Alkane </td><td>GAIIX single reads 36 nt </td><td>23 903 839 </td><td>99 (47%) </td><td>110 (53%) </td><td>0.9<sup>b</sup> </td></tr><tr><td>Glucose </td><td>GAIIX single reads 36 nt </td><td>13 022 908 </td><td>121 (68%) </td><td>56 (32%) </td><td>2.16 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>12 965 094 </td><td>202 (67%) </td><td>86 (33%) </td><td>2.21 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>10 592 928 </td><td>203 (63%) </td><td>116 (37%) </td><td>1.70 </td></tr><tr><td>Oleic acid </td><td>GAIIX single reads 36 nt </td><td>19 270 870 </td><td>261 (61%) </td><td>166 (39%) </td><td>1.57 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>25 974 957 </td><td>118 (60%) </td><td>78 (40%) </td><td>1.51 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>13 316 230 </td><td>260 (67%) </td><td>130 (33%) </td><td>2.00 </td></tr></tbody></table></div><div content-id="DSS007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Carbon source </th><th>Illumina Solexa sequencing technology and read length </th><th>Total reads </th><th>Reads mapping short intron 2 </th><th>Reads mapping long intron 2 </th><th>Ratio short/long </th></tr></thead><tbody><tr><td>Triolein </td><td>GAIIX single reads 36 nt </td><td>19 589 043 </td><td>94 (57%) </td><td>71 (43%) </td><td>1.33 </td></tr><tr><td>Tributyrin </td><td>GAIIX single reads 36 nt </td><td>15 996 762 </td><td>298 (83%) </td><td>59 (17%) </td><td>5.05<sup>a</sup> </td></tr><tr><td>Glycerol </td><td>GAIIX single reads 36 nt </td><td>22 935 285 </td><td>308 (62%) </td><td>186 (38%) </td><td>1.66 </td></tr><tr><td>Alkane </td><td>GAIIX single reads 36 nt </td><td>23 903 839 </td><td>99 (47%) </td><td>110 (53%) </td><td>0.9<sup>b</sup> </td></tr><tr><td>Glucose </td><td>GAIIX single reads 36 nt </td><td>13 022 908 </td><td>121 (68%) </td><td>56 (32%) </td><td>2.16 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>12 965 094 </td><td>202 (67%) </td><td>86 (33%) </td><td>2.21 </td></tr><tr><td>Glucose </td><td>Hiseq single reads 50 nt </td><td>10 592 928 </td><td>203 (63%) </td><td>116 (37%) </td><td>1.70 </td></tr><tr><td>Oleic acid </td><td>GAIIX single reads 36 nt </td><td>19 270 870 </td><td>261 (61%) </td><td>166 (39%) </td><td>1.57 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>25 974 957 </td><td>118 (60%) </td><td>78 (40%) </td><td>1.51 </td></tr><tr><td>Oleic acid </td><td>Hiseq paired-end 100 nt </td><td>13 316 230 </td><td>260 (67%) </td><td>130 (33%) </td><td>2.00 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The ratio on tributyrin is statistically different from that on all other media (<em>P</em> &lt; 0.0001).<sup>b</sup>The ratio on alkane is statistically different from that on all other media, except on triolein (<em>P</em> = 0.0765).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7159190" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7159191 id="7159191" class="section-title" data-legacy-id=s3c>3.3. Absence of either the peroxisomal or the cytosolic form of the MDH does not affect growth rate, irrespective of the carbon source</h3>
<p>In order to evaluate the biological roles of the two MDH isoforms encoded by <em>YlMDH2</em>, we constructed strains that expressed cDNAs encoding either the cytosolic (cyto), peroxisomal (pero) or wild-type (wt) form of YlMdh2p at the <em>YlMDH2</em> locus in the <em>Y. lipolytica</em> PO1d strain (JMY2416, JMY2426 and JMY2428 strains, respectively). After complementation to restore prototrophy, strains were screened for growth on various substrates known to affect the growth of <em>S. cerevisiae</em> MDH mutants, i.e<em>.</em> glucose, acetate, ethanol and OA.<sup><a reveal-id="DSS007C17" data-open="DSS007C17" class="link link-ref link-reveal xref-bibr">17</a>–<a reveal-id="DSS007C19" data-open="DSS007C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS007C46" data-open="DSS007C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS007C47" data-open="DSS007C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Surprisingly, we did not observe any growth rate differences among the three strains in either liquid or solid culture using these substrates as the sole carbon sources (see Fig. <a reveal-id="DSS007F3" data-open="DSS007F3" class="link link-reveal link-table xref-fig">3</a> for growth on solid substrates and Fig. <a reveal-id="DSS007F4" data-open="DSS007F4" class="link link-reveal link-table xref-fig">4</a>A for glucose on liquid medium). We also screened these strains for growth on other lipid substrates, i.e<em>.</em> TB and TO, but failed to identify any specific growth phenotypes (Fig. <a reveal-id="DSS007F3" data-open="DSS007F3" class="link link-reveal link-table xref-fig">3</a>). We also performed growth rate comparisons in a 96-well plate system using various carbon sources. Under these experimental conditions, we observed significant increases in the growth rate and maximum cell growth for the cyto strain (JMY2416) for all carbon sources tested, but observed no differences in the lag phase. For example, the <em>P</em>-values calculated for four replicates grown on glucose are 1.544e<sup>−2</sup> and 1.184e<sup>−4</sup> for the growth rate and maximum cell growth, respectively (Fig. <a reveal-id="DSS007F4" data-open="DSS007F4" class="link link-reveal link-table xref-fig">4</a>B). We speculated that this result may be due to low oxygenation of the culture in a microtiter plate. In order to simulate low oxygen levels in a flask culture, we complemented the growth medium with antimycin A, a drug that impairs the respiratory chain, at a concentration that allows <em>Y. lipolytica</em> to grow but with a reduced rate (0.1 µg/ml). However, the growth rates of all strains were similarly reduced (data not shown), which indicated that the increased growth rate of the cyto strain is not directly linked to respiration efficiency. This phenomenon could be linked to oxygen access for other pathways or could reflect a different morphological state that affects optical density and may not rely on growth rate directly. </p>                        <a id="7159193" scrollto-destination="7159193"></a>
<div id="DSS007F3" data-id="DSS007F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00703.jpeg?Expires=2147483647&Signature=TxeyTubNPVBCLhhgHuNtQTYhwh6z1daT95usBVk477clW7YouwAyvu1e7euOScQMmeMPer2yfGu6l4supIX-w8P15ajxfc0xFkt8eeDap-K-z6Fzj4ZJE9a0JsMF6EGq5e4YvoxDohUoqsGpUCZCTRW56IkO9nhUHfSuVkkzO7Hn6sDVIITb9wMHFK5mZTryRW0zWJKn-tib~ubicqY6-e-iMQ5us7USHA91Kyxhn7~VIxwqR-vdzk~YvXb~NOZvhdmlaBwP4mOjw3G6GzFg5wm9-~5bqBrT3be8jY-wKFSxxUUyNk9iIffwVhUFnyO7m1WeknidodEHzpiMWkuvjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of alternatively spliced forms of YlMDH2 and growth on different substrates. Serial dilutions (serial dilution factor of five) of cultures of the wild-type (wt—JMY2428) strain, the cytosolic variant (cyto—JMY2416) and the peroxisomal variant (pero—JMY2426) were inoculated on YNB medium supplemented with different carbon sources. No growth differences between the mutants were detected; both of them were able to grow on all the media." path-from-xml="dss00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159193/dss00703.jpeg" path-from-xml="dss00703.jpeg" target="_blank">View large</a><a section="7159193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00703.jpeg?Expires=2147483647&Signature=03mtumUqsgyzGXisJ2yFpIULpG2w~UA6USBWQa~NrRrIR5I0W2uocsR4gKXVBjIzXmu6TzF7noBm1RWeOpQNaf~xNZSh8NKXo91N0BPU75Jns6JaxQgJ124XcxA3vPJT3lVwn2DbDeEqoELADaSUlxeybrl3XaUmuy27b3C64HnXRaXD20l-aicTURF8M31HeOlQzbLCvE7Yg3rcjE6H9d451EbqZyFcpYcUZGS8dtOUHg6QLnnmOMEBMczjqD0muTG8x-q9sm8sr-DG5QD4649ORiRKJY4wsLFKOZz1a2N3TNb-JSdal5kvs8LvDvkiVazoDEMkPecJ6iIhn2-RNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159193&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of alternatively spliced forms of <em>YlMDH2</em> and growth on different substrates. Serial dilutions (serial dilution factor of five) of cultures of the wild-type (wt—JMY2428) strain, the cytosolic variant (cyto—JMY2416) and the peroxisomal variant (pero—JMY2426) were inoculated on YNB medium supplemented with different carbon sources. No growth differences between the mutants were detected; both of them were able to grow on all the media.</p></div></div><div content-id="DSS007F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00703.jpeg?Expires=2147483647&Signature=TxeyTubNPVBCLhhgHuNtQTYhwh6z1daT95usBVk477clW7YouwAyvu1e7euOScQMmeMPer2yfGu6l4supIX-w8P15ajxfc0xFkt8eeDap-K-z6Fzj4ZJE9a0JsMF6EGq5e4YvoxDohUoqsGpUCZCTRW56IkO9nhUHfSuVkkzO7Hn6sDVIITb9wMHFK5mZTryRW0zWJKn-tib~ubicqY6-e-iMQ5us7USHA91Kyxhn7~VIxwqR-vdzk~YvXb~NOZvhdmlaBwP4mOjw3G6GzFg5wm9-~5bqBrT3be8jY-wKFSxxUUyNk9iIffwVhUFnyO7m1WeknidodEHzpiMWkuvjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of alternatively spliced forms of YlMDH2 and growth on different substrates. Serial dilutions (serial dilution factor of five) of cultures of the wild-type (wt—JMY2428) strain, the cytosolic variant (cyto—JMY2416) and the peroxisomal variant (pero—JMY2426) were inoculated on YNB medium supplemented with different carbon sources. No growth differences between the mutants were detected; both of them were able to grow on all the media." path-from-xml="dss00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159193/dss00703.jpeg" path-from-xml="dss00703.jpeg" target="_blank">View large</a><a section="7159193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00703.jpeg?Expires=2147483647&Signature=03mtumUqsgyzGXisJ2yFpIULpG2w~UA6USBWQa~NrRrIR5I0W2uocsR4gKXVBjIzXmu6TzF7noBm1RWeOpQNaf~xNZSh8NKXo91N0BPU75Jns6JaxQgJ124XcxA3vPJT3lVwn2DbDeEqoELADaSUlxeybrl3XaUmuy27b3C64HnXRaXD20l-aicTURF8M31HeOlQzbLCvE7Yg3rcjE6H9d451EbqZyFcpYcUZGS8dtOUHg6QLnnmOMEBMczjqD0muTG8x-q9sm8sr-DG5QD4649ORiRKJY4wsLFKOZz1a2N3TNb-JSdal5kvs8LvDvkiVazoDEMkPecJ6iIhn2-RNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159193&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of alternatively spliced forms of <em>YlMDH2</em> and growth on different substrates. Serial dilutions (serial dilution factor of five) of cultures of the wild-type (wt—JMY2428) strain, the cytosolic variant (cyto—JMY2416) and the peroxisomal variant (pero—JMY2426) were inoculated on YNB medium supplemented with different carbon sources. No growth differences between the mutants were detected; both of them were able to grow on all the media.</p></div></div>                        <a id="7159194" scrollto-destination="7159194"></a>
<div id="DSS007F4" data-id="DSS007F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00704.jpeg?Expires=2147483647&Signature=Wayhq7leUR-yUU7Q92R4-gwWqxA4Y5-oojoRKMVUSwgORD3qfCTT3r4HBUPrEVU5iB993n2RBLN~OtNzFJ3ToqXRpR4n6VUIrDRUBeB1na8EnDYyrD71rNXD9xv7gg~BmSghgHdH8eGGOcS-KSE6ZhsGTRTYL~SxOwq5DxTgFxrSn-xarDD9iEGOUzWUuicjrR-ZGxRWYtOvMa4LjLSzW4IwTrSJvpwhvajDt0BAVoaUAmzjmgOJhRRMMnfOyxWUxrIxP7k7p9dEwr9Ygbkhcbnn5QnlRoYYhUEZ-39v8DegcPxPJ~ztnD2tP5Hrt19mFyd3WaqJMagK1Z26VOEbDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative growth of YlMDH2 mutants. (A) Growth curves in flasks with agitation in YNBE medium with glucose 2%. (B) Growth curves on 96-well plates in YNBE with glucose 0.5%. Coloured curves represent the different splicing mutants of YlMDH2: the cytosolic form is in black (MDHc), the peroxisomal form is in red (MDHp) and the wild-type is in green (MDHwt). OD, optical density measured at 600 nm." path-from-xml="dss00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159194/dss00704.jpeg" path-from-xml="dss00704.jpeg" target="_blank">View large</a><a section="7159194" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00704.jpeg?Expires=2147483647&Signature=An-a1ARUyZ0od~Yec7W0~A0WsSKoQNWet5fAzKunlHzSGzcv1hS6eYhI8yh9QViCS8bn59PDTBylkLKfB3j0Bcq0gue61ITziIxx2hCG6CMjwXWaVh0xWAXKKoWwZFtyT-5q~nFyZ12o~zaeD6oEtFbeJV~FYFwKu21eLkY3zJmH9XB7fmoGqp4OFh4IlSy4JJskL2EUPf1X8lw-sMJwdEMMiGxPJeKApBgeRC4WklgmqVNqkdplX4PL~6GqhtA3wmjZ7i7djoGIdWhM1xgSnIPkjAt8rhkJhsyiG22k30jZhhZirrTWZgKVezx2Cmmp-RbPzRfQzKVdCq~emssSqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159194&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative growth of <em>YlMDH2</em> mutants. (A) Growth curves in flasks with agitation in YNBE medium with glucose 2%. (B) Growth curves on 96-well plates in YNBE with glucose 0.5%. Coloured curves represent the different splicing mutants of <em>YlMDH2</em>: the cytosolic form is in black (MDHc), the peroxisomal form is in red (MDHp) and the wild-type is in green (MDHwt). OD, optical density measured at 600 nm.</p></div></div><div content-id="DSS007F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00704.jpeg?Expires=2147483647&Signature=Wayhq7leUR-yUU7Q92R4-gwWqxA4Y5-oojoRKMVUSwgORD3qfCTT3r4HBUPrEVU5iB993n2RBLN~OtNzFJ3ToqXRpR4n6VUIrDRUBeB1na8EnDYyrD71rNXD9xv7gg~BmSghgHdH8eGGOcS-KSE6ZhsGTRTYL~SxOwq5DxTgFxrSn-xarDD9iEGOUzWUuicjrR-ZGxRWYtOvMa4LjLSzW4IwTrSJvpwhvajDt0BAVoaUAmzjmgOJhRRMMnfOyxWUxrIxP7k7p9dEwr9Ygbkhcbnn5QnlRoYYhUEZ-39v8DegcPxPJ~ztnD2tP5Hrt19mFyd3WaqJMagK1Z26VOEbDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative growth of YlMDH2 mutants. (A) Growth curves in flasks with agitation in YNBE medium with glucose 2%. (B) Growth curves on 96-well plates in YNBE with glucose 0.5%. Coloured curves represent the different splicing mutants of YlMDH2: the cytosolic form is in black (MDHc), the peroxisomal form is in red (MDHp) and the wild-type is in green (MDHwt). OD, optical density measured at 600 nm." path-from-xml="dss00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159194/dss00704.jpeg" path-from-xml="dss00704.jpeg" target="_blank">View large</a><a section="7159194" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00704.jpeg?Expires=2147483647&Signature=An-a1ARUyZ0od~Yec7W0~A0WsSKoQNWet5fAzKunlHzSGzcv1hS6eYhI8yh9QViCS8bn59PDTBylkLKfB3j0Bcq0gue61ITziIxx2hCG6CMjwXWaVh0xWAXKKoWwZFtyT-5q~nFyZ12o~zaeD6oEtFbeJV~FYFwKu21eLkY3zJmH9XB7fmoGqp4OFh4IlSy4JJskL2EUPf1X8lw-sMJwdEMMiGxPJeKApBgeRC4WklgmqVNqkdplX4PL~6GqhtA3wmjZ7i7djoGIdWhM1xgSnIPkjAt8rhkJhsyiG22k30jZhhZirrTWZgKVezx2Cmmp-RbPzRfQzKVdCq~emssSqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159194&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative growth of <em>YlMDH2</em> mutants. (A) Growth curves in flasks with agitation in YNBE medium with glucose 2%. (B) Growth curves on 96-well plates in YNBE with glucose 0.5%. Coloured curves represent the different splicing mutants of <em>YlMDH2</em>: the cytosolic form is in black (MDHc), the peroxisomal form is in red (MDHp) and the wild-type is in green (MDHwt). OD, optical density measured at 600 nm.</p></div></div>                            <h3 scrollto-destination=7159195 id="7159195" class="section-title" data-legacy-id=s3d>3.4. Malate dehydrogenase localization</h3>
<p>In order to locate the two MDH isoforms produced by AS, we constructed vectors to express the eYFP fluorescent protein fused to the N-termini of the cytosolic and peroxisomal MDH isoforms and to the N-terminus of MDH expressed from the wt <em>YlMDH2</em> gene. <em>Yarrowia lipolytica</em> strains containing these constructs were subsequently transformed with a construction encoding the RedStar2 fluorescent protein without or with a PTS1 sequence (Ser-Lys-Leu; SKL) and used for colocalization studies. The native RedStar2 protein was cytosolic, whereas the RedStar2-SKL protein was targeted to the peroxisomes, which were visible as fluorescent dots in the cells (Fig. <a reveal-id="DSS007F5" data-open="DSS007F5" class="link link-reveal link-table xref-fig">5</a>). In the strain expressing the predicted cytosolic form of eYFP-MDH, green fluorescence colocalized with red fluorescence produced by cytosolic RedStar2 (Fig. <a reveal-id="DSS007F5" data-open="DSS007F5" class="link link-reveal link-table xref-fig">5</a>; JMY2499), but did not colocalize with the peroxisomal form of RedStar2 (Fig. <a reveal-id="DSS007F5" data-open="DSS007F5" class="link link-reveal link-table xref-fig">5</a>; JMY2493). Conversely, the predicted peroxisomal form of eYFP-MDH colocalized with the peroxisomal form of RedStar2 (colocalization of fluorescent dots; Fig. <a reveal-id="DSS007F5" data-open="DSS007F5" class="link link-reveal link-table xref-fig">5</a>; JMY2496) but not with cytosolic RedStar2 (Fig. <a reveal-id="DSS007F5" data-open="DSS007F5" class="link link-reveal link-table xref-fig">5</a>; JMY2501). These colocalization experiments confirmed that the upstream splice variant mRNA encodes a cytosolic form of MDH, whereas the downstream splice variant encodes MDH with a C-terminal PTS1 and is peroxisomal. For all these constructions, we observed similar localizations and colocalizations with either glucose or OA as the carbon source. For the eYFP-tagged wild-type copy of YlMdh2p, which encoded both MDH isoforms, we observed clear cytosolic fluorescence (Fig. <a reveal-id="DSS007F5" data-open="DSS007F5" class="link link-reveal link-table xref-fig">5</a>; JMY2500) but no clear labelling of peroxisomes (Fig. <a reveal-id="DSS007F5" data-open="DSS007F5" class="link link-reveal link-table xref-fig">5</a>; JMY2495). However, overlapping fluorescence might have prevented distinguishing the peroxisomal and cytosolic localizations, especially since the expression level of the cytosolic isoform is higher than that of the peroxisomal form, as suggested by the cyto/pero RNA ratio (Table <a reveal-id="DSS007TB3" data-open="DSS007TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="7159197" scrollto-destination="7159197"></a>
<div id="DSS007F5" data-id="DSS007F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00705.jpeg?Expires=2147483647&Signature=nOPhVzau1h5M6C7AIqMDwT-yOsV8qvrCcYY02jS-m-sKbkifrjlEcr7o5jRmYvHq1b8rPpuYdWB~k9SagYihnEWm7Cj1vvR9YPqBdWCPqMUmEgZRj12-fFAYFrmwvo56r2FvEVy7Ly8W3Ikn9gBlLf6-U06sAzKm~XvJRQlSGNwO4I03ykAe3AL-fh3Vhs4zykk-4JAWLGEEaN1BOkqSMbTDSQBfT~3mo7YtT6bUrAsEfsi9tm49tQ4wlw1vtVkKAi4lMvPzPklqE5kaO3fr8SfDqlyotQPwdsDHCyEqjjg6pn-lry4iiIKBwaINXWWDrlSgugvF4ijxvH0twLnMoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Colocalization of the two YlMdh2p isoforms with cytosolic or peroxisomal forms of the RedStar2 protein. eYFP-tagged peroxisomal YlMdh2p (eYFP-MDHp), cytosolic YlMdh2p (eYFP-MDHc) or wild-type YlMdh2p (eYFP-MDHwt) were co-expressed with either the peroxisomal (redstar2p) or cytosolic (redstar2c) forms of the RedStar2 protein. For each strain, both proteins (MDH and RedStar2) were visualized simultaneously by fluorescence microscopy. Cells were imaged after 12h of growth in YNBE 2% OA using differential interference contrast (DIC) for eYFP fluorescence (eYFP, green) and for RedStar2 fluorescence (redstar2, red). eYFP and RedStar2 images were merged (right panel). Yellow colour indicates overlapping fluorescence and evidences colocalization." path-from-xml="dss00705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159197/dss00705.jpeg" path-from-xml="dss00705.jpeg" target="_blank">View large</a><a section="7159197" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00705.jpeg?Expires=2147483647&Signature=slcqL0j6IIwdiAFytgzib-jElxus4au2ma~lxlYaMQvCCDznh6qjDSII3dOKSGw13DV7T-6Mu4Uiz6b67dvDNaAEJpV3HQakDB0AcQwk2D1eQWt~KJZU17IGZBYEaJ6AxmtK8z4APbX8Nwgn73p3PJuvv-NWDNNfG3Fojj~6wKLR6sP2wxF5Eb706s3pKRO8ITp0GfemBFoGnDtOVVPFOLf91A5P7HudaXKwPCMq8yhB5A8IZqXpKfhe-W54GSLWQ0oFsqKPRcAIP~6DI0q4i3b0DsI9kR4QblO~EG0Ku4PXbbnsO2LyCFaYCGn3ayDqc8dv28fyPVbtHF57yCXxzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159197&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Colocalization of the two YlMdh2p isoforms with cytosolic or peroxisomal forms of the RedStar2 protein. eYFP-tagged peroxisomal YlMdh2p (eYFP-MDHp), cytosolic YlMdh2p (eYFP-MDHc) or wild-type YlMdh2p (eYFP-MDHwt) were co-expressed with either the peroxisomal (redstar2p) or cytosolic (redstar2c) forms of the RedStar2 protein. For each strain, both proteins (MDH and RedStar2) were visualized simultaneously by fluorescence microscopy. Cells were imaged after 12h of growth in YNBE 2% OA using differential interference contrast (DIC) for eYFP fluorescence (eYFP, green) and for RedStar2 fluorescence (redstar2, red). eYFP and RedStar2 images were merged (right panel). Yellow colour indicates overlapping fluorescence and evidences colocalization.</p></div></div><div content-id="DSS007F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/m_dss00705.jpeg?Expires=2147483647&Signature=nOPhVzau1h5M6C7AIqMDwT-yOsV8qvrCcYY02jS-m-sKbkifrjlEcr7o5jRmYvHq1b8rPpuYdWB~k9SagYihnEWm7Cj1vvR9YPqBdWCPqMUmEgZRj12-fFAYFrmwvo56r2FvEVy7Ly8W3Ikn9gBlLf6-U06sAzKm~XvJRQlSGNwO4I03ykAe3AL-fh3Vhs4zykk-4JAWLGEEaN1BOkqSMbTDSQBfT~3mo7YtT6bUrAsEfsi9tm49tQ4wlw1vtVkKAi4lMvPzPklqE5kaO3fr8SfDqlyotQPwdsDHCyEqjjg6pn-lry4iiIKBwaINXWWDrlSgugvF4ijxvH0twLnMoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Colocalization of the two YlMdh2p isoforms with cytosolic or peroxisomal forms of the RedStar2 protein. eYFP-tagged peroxisomal YlMdh2p (eYFP-MDHp), cytosolic YlMdh2p (eYFP-MDHc) or wild-type YlMdh2p (eYFP-MDHwt) were co-expressed with either the peroxisomal (redstar2p) or cytosolic (redstar2c) forms of the RedStar2 protein. For each strain, both proteins (MDH and RedStar2) were visualized simultaneously by fluorescence microscopy. Cells were imaged after 12h of growth in YNBE 2% OA using differential interference contrast (DIC) for eYFP fluorescence (eYFP, green) and for RedStar2 fluorescence (redstar2, red). eYFP and RedStar2 images were merged (right panel). Yellow colour indicates overlapping fluorescence and evidences colocalization." path-from-xml="dss00705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7159197/dss00705.jpeg" path-from-xml="dss00705.jpeg" target="_blank">View large</a><a section="7159197" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss00705.jpeg?Expires=2147483647&Signature=slcqL0j6IIwdiAFytgzib-jElxus4au2ma~lxlYaMQvCCDznh6qjDSII3dOKSGw13DV7T-6Mu4Uiz6b67dvDNaAEJpV3HQakDB0AcQwk2D1eQWt~KJZU17IGZBYEaJ6AxmtK8z4APbX8Nwgn73p3PJuvv-NWDNNfG3Fojj~6wKLR6sP2wxF5Eb706s3pKRO8ITp0GfemBFoGnDtOVVPFOLf91A5P7HudaXKwPCMq8yhB5A8IZqXpKfhe-W54GSLWQ0oFsqKPRcAIP~6DI0q4i3b0DsI9kR4QblO~EG0Ku4PXbbnsO2LyCFaYCGn3ayDqc8dv28fyPVbtHF57yCXxzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7159197&ar=474863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Colocalization of the two YlMdh2p isoforms with cytosolic or peroxisomal forms of the RedStar2 protein. eYFP-tagged peroxisomal YlMdh2p (eYFP-MDHp), cytosolic YlMdh2p (eYFP-MDHc) or wild-type YlMdh2p (eYFP-MDHwt) were co-expressed with either the peroxisomal (redstar2p) or cytosolic (redstar2c) forms of the RedStar2 protein. For each strain, both proteins (MDH and RedStar2) were visualized simultaneously by fluorescence microscopy. Cells were imaged after 12h of growth in YNBE 2% OA using differential interference contrast (DIC) for eYFP fluorescence (eYFP, green) and for RedStar2 fluorescence (redstar2, red). eYFP and RedStar2 images were merged (right panel). Yellow colour indicates overlapping fluorescence and evidences colocalization.</p></div></div>                            <h3 scrollto-destination=7159198 id="7159198" class="section-title" data-legacy-id=s3e>3.5. AS of MDH is unique to Y. lipolytica</h3>
<p>A comparative study of the MDH gene family revealed that <em>P. pastoris</em> and <em>A. adeninivorans</em> also contain only two genes coding for MDH, and in each case, a peroxisomal form is absent. Whereas the <em>P. pastoris</em> MDH genes do not contain any introns, model predictions for the gene that encodes the cytosolic form of MDH in <em>A. adeninivorans</em> revealed the presence of two introns. The second intron is located at exactly the same position as the second intron of <em>YlMDH2</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Unexpectedly, the terminal exon has undergone a large expansion, probably by fusion with a downstream CDS, which is conserved in synteny in <em>Y. lipolytica.</em> Despite the presence of the same positioned intron, no alternative 3′-splice site was detected in <em>A. adeninivorans</em>.</p>                            <h2 scrollto-destination=7159200 id="7159200" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p><em>Yarrowia lipolytica</em> genes contain more introns than those of <em>S. cerevisiae</em> (15 and 4%, respectively) or other hemiascomycetes, and <em>Y. lipolytica</em> exhibits all known forms of AS.<sup><a reveal-id="DSS007C1" data-open="DSS007C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS007C2" data-open="DSS007C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This suggests possible roles for splicing in the post-transcriptional regulation of gene expression and/or in generating additional proteome complexity in <em>Y. lipolytica,</em> and that splicing might have a much higher impact in <em>Y. lipolytica</em> than in <em>S. cerevisiae</em>. As an example of protein targeting regulation, we found clear evidence that the MDH gene <em>YlMDH2</em> is alternatively spliced and leads to the production of two proteins destined for distinct cellular compartments: cytosol and peroxisomes. The <em>HAC1</em> gene in <em>Y. lipolytica</em><sup><a reveal-id="DSS007C48" data-open="DSS007C48" class="link link-ref link-reveal xref-bibr">48</a></sup> is also subject to functional AS; however, this gene possesses a non-spliceosomal intron and the splicing mechanism is mediated under stress conditions by Ire1p, a transmembrane kinase/endonuclease.<sup><a reveal-id="DSS007C49" data-open="DSS007C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Thus, the <em>YlMDH2</em> gene constitutes the first case of functional AS involving a spliceosomal intron in <em>Y. lipolytica</em>.</p><p>Overall, few cases of AS have been described in hemiascomycete yeasts, which agrees with their low intron content. The only examples that contribute to the diversity of both the transcriptome and proteome were found in <em>S. cerevisiae</em>. In two reported cases, the protein isoforms encoded by the alternatively spliced transcripts are targeted to different subcellular compartments: <em>PTC7</em> proteins localize to the nuclear envelope and the mitochondria<sup><a reveal-id="DSS007C6" data-open="DSS007C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and <em>YCAT</em> proteins to the mitochondria and peroxisomes.<sup><a reveal-id="DSS007C7" data-open="DSS007C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In the case of <em>YlMDH2</em>, AS also leads to dual localization. We clearly demonstrated by cDNA sequencing that two transcript variants exist, which result from the use of two alternative 3′-splice sites separated by 4 nt. The predicted protein isoforms differ by only two C-terminal amino acids (AN or AKI), the latter of which constitutes a typical PTS1. Visualization of the eYFP-labelled cyto and pero isoforms showed that both proteins localized to their respective predicted compartments.</p><p>In <em>S. cerevisiae</em>, the distinct phenotypes of <em>mdh2-</em> and <em>mdh3-</em>deleted strains have been extensively studied. The <em>mdh2</em> mutant, which is equivalent to the <em>Y. lipolytica</em> pero strain (deleted for the cytosolic form), exhibits no growth defect on medium with glucose as a carbon source but is unable to grow on acetate or ethanol in minimum medium.<sup><a reveal-id="DSS007C17  DSS007C19" data-open="DSS007C17  DSS007C19" class="link link-ref link-reveal xref-bibr">17–19</a>,<a reveal-id="DSS007C46" data-open="DSS007C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS007C47" data-open="DSS007C47" class="link link-ref link-reveal xref-bibr">47</a></sup> We did not observe a similar phenotype for the <em>Y. lipolytica</em> pero strain. In <em>S. cerevisiae</em>, a functional glyoxylate cycle is mandatory for growth on C2 carbon sources and the cytosolic oxaloacetate-malate conversion step by Mdh2p is likely to be required for the glyoxylate cycle.<sup><a reveal-id="DSS007C16" data-open="DSS007C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Our finding that a <em>Y. lipolytica</em> strain deleted for cytosolic MDH is able to grow on ethanol and acetate suggests that the peroxisomal form can perform the malate–oxaloacetate conversion of the glyoxylate cycle. If we assume that the <em>Y. lipolytica</em> glyoxylate cycle follows the <em>S. cerevisiae</em> model, this result has two implications. First, the cytosolic and peroxisomal MDH proteins in <em>Y. lipolytica</em> must share the same enzymatic activities, which is entirely likely as the two isoforms differ by only two C-terminal amino acids. Moreover, in <em>S. cerevisiae</em>, cytosolic versions of Mdh3p (lacking the PTS1) or Mdh1p (lacking the mitochondrial targeting sequence) are able to complement an <em>mdh2</em> deletion for growth on ethanol, indicating that all isoforms share similar enzymatic activities despite their amino acid sequence differences.<sup><a reveal-id="DSS007C18" data-open="DSS007C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS007C50" data-open="DSS007C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Second, in cases of growth on C2 compounds, the peroxisomal form may be retained in the cytosol independent of the splicing event and the intermediates of the glyoxylate cycle, i.e<em>.</em> malate and oxaloacetate, may be able to cross the peroxisomal membrane in both directions. Examples of mislocalization of peroxisomal proteins have been reported in <em>S. cerevisiae:</em> Mls1p is sequestered in the cytosol when the yeast is cultivated on ethanol<sup><a reveal-id="DSS007C51" data-open="DSS007C51" class="link link-ref link-reveal xref-bibr">51</a></sup> and the Mdh3p–GFP fusion protein remained in the cytosol in an <em>mdh2Δ</em> mutant.<sup><a reveal-id="DSS007C52" data-open="DSS007C52" class="link link-ref link-reveal xref-bibr">52</a></sup> However, an alternative model to that of <em>S. cerevisiae</em> is that the entire glyoxylate cycle takes place in the peroxisome. Indeed, in <em>Y. lipolytica</em>, isocitrate lyase, one of the two enzymes specific to the glyoxylate cycle, is located in peroxisomes, whereas it is located into the cytosol in <em>S. cerevisiae.</em><sup><a reveal-id="DSS007C53" data-open="DSS007C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Thus, peroxisomal MDH may be involved in the malate–oxaloacetate conversion. However, the deletion of the peroxisomal form, as exemplified by our cyto strain, had no effect on growth on C2 compounds, suggesting that the <em>S. cerevisiae</em> model also reflects the situation in <em>Y. lipolytica</em>, at least in the absence of peroxisomal MDH.</p><p>van Roermund <em>et al.</em> reported that growth of the <em>S. cerevisiae mdh3</em> mutant (peroxisomal form disrupted) on OA was impaired. They showed that β-oxidation is also impaired in this mutant, which suggested an indirect role for Mdh3p in this process through reoxidation of the NADH generated during the β-oxidation of fatty acids.<sup><a reveal-id="DSS007C45" data-open="DSS007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> This relationship has also been demonstrated in <em>Arabidopsis thaliana</em>.<sup><a reveal-id="DSS007C54" data-open="DSS007C54" class="link link-ref link-reveal xref-bibr">54</a></sup> In <em>Y. lipolytica</em>, strains expressing only the cytosolic MDH (cyto strain) grow as well as the wild-type strain on various lipid substrates and we did not observe any other distinct phenotypes. Thus, the peroxisomal MDH is not essential for NADH reoxidation in the peroxisome. This suggests that reoxidation is carried out by an alternative pathway in peroxisomes and might take the form of shuttles based on either lactate dehydrogenase (LDH) or glycerol-3-phosphate dehydrogenase (G3PDH), which are present in mammalian peroxisomes that do not contain MDH.<sup><a reveal-id="DSS007C55" data-open="DSS007C55" class="link link-ref link-reveal xref-bibr">55</a></sup> This alternative activity remains to be discovered in <em>Y. lipolytica</em>.</p><p>Several less convincing alternative hypotheses can also be addressed. First, cytosolic MDH may be targeted to the peroxisomes by a PTS distinct from PTS1 or may be imported through a peroxisomal protein complex. Second, the <em>Y. lipolytica</em> peroxisomal membrane may be permeable to NADH metabolites, which would allow NADH recycling through a cytosolic reoxidation. However, this is clearly not the case in <em>S. cerevisiae</em>.<sup><a reveal-id="DSS007C45" data-open="DSS007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Third, another β-oxidation pathway allowing growth on fatty acids in the absence of peroxisomal MDH may exist in <em>Y. lipolytica</em>. In line with this concept, vestigial mitochondrial β-oxidation may exist in <em>Y. lipolytica</em>.<sup><a reveal-id="DSS007C56" data-open="DSS007C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSS007C57" data-open="DSS007C57" class="link link-ref link-reveal xref-bibr">57</a></sup></p><p>From the sequencing of our cDNA libraries and the RNA-seq data, it appears that <em>YlMDH2</em> splicing is probably regulated or affected by the carbon source and/or the growth conditions. Several examples of splicing regulation in yeast, principally in <em>S. cerevisiae</em>, are known to exist. The unspliced form of SUS1 mRNA (intron retention) accumulates after a temperature shift.<sup><a reveal-id="DSS007C58" data-open="DSS007C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSS007C59" data-open="DSS007C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Splicing of the <em>MER</em> genes in response to sporulation conditions and/or meiosis has also been well documented.<sup><a reveal-id="DSS007C3" data-open="DSS007C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS007C60  DSS007C62" data-open="DSS007C60  DSS007C62" class="link link-ref link-reveal xref-bibr">60–62</a></sup> AS autoregulation by the protein product of the gene itself, through negative or positive feedback, has been reported for <em>RPL30</em>,<sup><a reveal-id="DSS007C63" data-open="DSS007C63" class="link link-ref link-reveal xref-bibr">63</a></sup><em>YRA1</em><sup><a reveal-id="DSS007C64" data-open="DSS007C64" class="link link-ref link-reveal xref-bibr">64</a></sup> and <em>SUS1.</em><sup><a reveal-id="DSS007C58" data-open="DSS007C58" class="link link-ref link-reveal xref-bibr">58</a></sup> In <em>Y. lipolytica</em>, regulation of <em>YlMDH2</em> splicing could thus control the level of MDH in the peroxisome or the cytosol. Dual localization of a protein can be regulated at the level of transcription, usually involving alternative start sites, and at the post-transcriptional level. The differential targeting of YCAT is controlled at the transcriptional level by means of alternative transcription initiation sites, whose frequency of use depends on the carbon source available.<sup><a reveal-id="DSS007C7" data-open="DSS007C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In the case of <em>PTC7</em> (intron retention), splicing is not regulated in response to carbon source availability, but the ratio of the two protein isoforms produced by the gene depends on whether the available carbon sources are fermentable or not, indicating that PTC7 isoform production is regulated post-transcriptionally.<sup><a reveal-id="DSS007C6" data-open="DSS007C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p><p>Thus, splicing regulation, as well as dual protein localization, appears to be more prevalent in yeasts than previously believed.<sup><a reveal-id="DSS007C65" data-open="DSS007C65" class="link link-ref link-reveal xref-bibr">65</a></sup> Whether this is reminiscent of ancestral attributes or corresponds to evolutionary changes that led to selective advantages remains to be investigated. Our data illustrate distinct differences in the metabolic behaviour of <em>Y. lipolytica</em> and <em>S. cerevisiae</em> in terms of their glyoxylate and TCA pathways and their MDH enzymatic activities. This underscores a possible role in the adaptation for growth on lipid substrates and for maintaining efficient β-oxidation across evolution. The AS mechanism that regulates MDH localization in <em>Y. lipolytica</em> remains so far an exclusive innovation of this yeast. <em>YlMDH2</em> AS regulation and its impact on protein localization and metabolism are under investigation and will provide new insights into the role of splicing on the proteome complexity in <em>Y. lipolytica</em>.</p>                            <h2 scrollto-destination=7159208 id="7159208" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank François Brunel for RNA-seq library construction, Nicolas Morin for his help in RNA-seq data analysis and Stéphanie Michely for her assistance with R software utilization. We also thank all our colleagues from the Génolevures consortium for access to the genome sequence of <em>A. adeninivorans</em>, which is not yet publicly available.</p>                            <h2 scrollto-destination=7159210 id="7159210" class="backsection-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7159212 id="7159212" class="backsection-title" data-legacy-id=s6>Funding</h2>
<p>This work was funded by the GDR CNRS 2354 Génolevures-3, the ANR Genarise (ANR-2005-BLAN-0331) and the ANR YeastIntrons (ANR-2010-BLAN-1620). P.K. was supported by a grant from the president of Republic of Côte d'Ivoire and by an INRA-MICA department fellowship.</p>                            <h2 scrollto-destination=7159214 id="7159214" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS007C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mekouar</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Blanc-Lenfle</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Ozanne</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Detection and analysis of alternative splicing in <em>Yarrowia lipolytica</em> reveal structural constraints facilitating nonsense-mediated decay of intron-retaining transcripts</div>, <div class="source">Genome Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">R65</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20analysis%20of%20alternative%20splicing%20in%20Yarrowia%20lipolytica%20reveal%20structural%20constraints%20facilitating%20nonsense-mediated%20decay%20of%20intron-retaining%20transcripts&amp;author=M.%20Mekouar&amp;author=I.%20Blanc-Lenfle&amp;author=C.%20Ozanne&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-6-r65" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-6-r65" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-6-r65"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20573210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neuveglise</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Marck</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div>. </div><div class="article-title">The intronome of budding yeasts</div>, <div class="source">C R Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">334</div> (pg. <div class="fpage">662</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20intronome%20of%20budding%20yeasts&amp;author=C.%20Neuveglise&amp;author=C.%20Marck&amp;author=C.%20Gaillardin&amp;publication_year=2011&amp;journal=C%20R%20Biol.&amp;volume=334&amp;pages=662-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.crvi.2011.05.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.crvi.2011.05.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.crvi.2011.05.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21819948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Juneau</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Palm</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Miranda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">High-density yeast-tiling array reveals previously undiscovered introns and extensive regulation of meiotic splicing</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">1522</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20yeast-tiling%20array%20reveals%20previously%20undiscovered%20introns%20and%20extensive%20regulation%20of%20meiotic%20splicing&amp;author=K.%20Juneau&amp;author=C.%20Palm&amp;author=M.%20Miranda&amp;author=R.W.%20Davis&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=1522-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0610354104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0610354104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0610354104"> </span></div> </div></div></div></div></div><div content-id="DSS007C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagalakshmi</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Waern</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">320</div> (pg. <div class="fpage">1344</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20of%20the%20yeast%20genome%20defined%20by%20RNA%20sequencing&amp;author=U.%20Nagalakshmi&amp;author=Z.%20Wang&amp;author=K.%20Waern&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=1344-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1158441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1158441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1158441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18451266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grund</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Fischer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Cabal</div> <div class="given-names">G.G.</div></div>,  <div class="name"><div class="surname">Antunez</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Pérez-Ortin</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Hurt</div> <div class="given-names">E.</div></div>. </div><div class="article-title">The inner nuclear membrane protein Src1 associates with subtelomeric genes and alters their regulated gene expression</div>, <div class="source">J. Cell Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">897</div>-<div class="lpage">910</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20inner%20nuclear%20membrane%20protein%20Src1%20associates%20with%20subtelomeric%20genes%20and%20alters%20their%20regulated%20gene%20expression&amp;author=S.E.%20Grund&amp;author=T.%20Fischer&amp;author=G.G.%20Cabal&amp;author=O.%20Antunez&amp;author=J.E.%20P%C3%A9rez-Ortin&amp;author=E.%20Hurt&amp;publication_year=2008&amp;journal=J.%20Cell%20Biol.&amp;volume=182&amp;pages=897-910" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.200803098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.200803098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.200803098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18762579" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Juneau</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nislow</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Alternative splicing of PTC7 in <em>Saccharomyces cerevisiae</em> determines protein localization</div>, <div class="source">Genetics</div> , <div class="year">2009</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">185</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20PTC7%20in%20Saccharomyces%20cerevisiae%20determines%20protein%20localization&amp;author=K.%20Juneau&amp;author=C.%20Nislow&amp;author=R.W.%20Davis&amp;publication_year=2009&amp;journal=Genetics&amp;volume=183&amp;pages=185-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.105155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.105155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.105155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19564484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elgersma</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">van Roermund</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Wanders</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Tabak</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">Peroxisomal and mitochondrial carnitine acetyltransferases of Saccharomyces cerevisiae are encoded by a single gene</div>, <div class="source">EMBO J.</div> , <div class="year">1995</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">3472</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Peroxisomal%20and%20mitochondrial%20carnitine%20acetyltransferases%20of%20Saccharomyces%20cerevisiae%20are%20encoded%20by%20a%20single%20gene&amp;author=Y.%20Elgersma&amp;author=C.W.%20van%20Roermund&amp;author=R.J.%20Wanders&amp;author=H.F.%20Tabak&amp;publication_year=1995&amp;journal=EMBO%20J.&amp;volume=14&amp;pages=3472-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7628448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carlson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Two differentially regulated mRNAs with different 5′ ends encode secreted with intracellular forms of yeast invertase</div>, <div class="source">Cell</div> , <div class="year">1982</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">145</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20differentially%20regulated%20mRNAs%20with%20different%205%E2%80%B2%20ends%20encode%20secreted%20with%20intracellular%20forms%20of%20yeast%20invertase&amp;author=M.%20Carlson&amp;author=D.%20Botstein&amp;publication_year=1982&amp;journal=Cell&amp;volume=28&amp;pages=145-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(82)90384-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(82)90384-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(82)90384-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7039847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Behm-Ansmant</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kashima</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Rehwinkel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sauliere</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wittkopp</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Izaurralde</div> <div class="given-names">E.</div></div>. </div><div class="article-title">mRNA quality control: an ancient machinery recognizes and degrades mRNAs with nonsense codons</div>, <div class="source">FEBS Lett.</div> , <div class="year">2007</div>, vol. <div class="volume">581</div> (pg. <div class="fpage">2845</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mRNA%20quality%20control%3A%20an%20ancient%20machinery%20recognizes%20and%20degrades%20mRNAs%20with%20nonsense%20codons&amp;author=I.%20Behm-Ansmant&amp;author=I.%20Kashima&amp;author=J.%20Rehwinkel&amp;author=J.%20Sauliere&amp;author=N.%20Wittkopp&amp;author=E.%20Izaurralde&amp;publication_year=2007&amp;journal=FEBS%20Lett.&amp;volume=581&amp;pages=2845-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2007.05.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2007.05.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2007.05.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17531985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stalder</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Muhlemann</div> <div class="given-names">O.</div></div>. </div><div class="article-title">The meaning of nonsense</div>, <div class="source">Trends Cell Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">315</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20meaning%20of%20nonsense&amp;author=L.%20Stalder&amp;author=O.%20Muhlemann&amp;publication_year=2008&amp;journal=Trends%20Cell%20Biol.&amp;volume=18&amp;pages=315-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tcb.2008.04.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tcb.2008.04.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tcb.2008.04.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18524595" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minarik</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Tomaskova</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kollarova</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Antalik</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Malate dehydrogenases—structure and function</div>, <div class="source">Gen. Physiol. Biophys.</div> , <div class="year">2002</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">257</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Malate%20dehydrogenases%E2%80%94structure%20and%20function&amp;author=P.%20Minarik&amp;author=N.%20Tomaskova&amp;author=M.%20Kollarova&amp;author=M.%20Antalik&amp;publication_year=2002&amp;journal=Gen.%20Physiol.%20Biophys.&amp;volume=21&amp;pages=257-65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12537350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCammon</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Epstein</div> <div class="given-names">C.B.</div></div>,  <div class="name"><div class="surname">Przybyla-Zawislak</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">McAlister-Henn</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Butow</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Global transcription analysis of Krebs tricarboxylic acid cycle mutants reveals an alternating pattern of gene expression and effects on hypoxic and oxidative genes</div>, <div class="source">Mol. Biol. Cell</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">958</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transcription%20analysis%20of%20Krebs%20tricarboxylic%20acid%20cycle%20mutants%20reveals%20an%20alternating%20pattern%20of%20gene%20expression%20and%20effects%20on%20hypoxic%20and%20oxidative%20genes&amp;author=M.T.%20McCammon&amp;author=C.B.%20Epstein&amp;author=B.%20Przybyla-Zawislak&amp;author=L.%20McAlister-Henn&amp;author=R.A.%20Butow&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Cell&amp;volume=14&amp;pages=958-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.E02-07-0422" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.E02-07-0422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.E02-07-0422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12631716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Przybyla-Zawislak</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Gadde</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Ducharme</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">McCammon</div> <div class="given-names">M.T.</div></div>. </div><div class="article-title">Genetic and biochemical interactions involving tricarboxylic acid cycle (TCA) function using a collection of mutants defective in all TCA cycle genes</div>, <div class="source">Genetics</div> , <div class="year">1999</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">153</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20biochemical%20interactions%20involving%20tricarboxylic%20acid%20cycle%20(TCA)%20function%20using%20a%20collection%20of%20mutants%20defective%20in%20all%20TCA%20cycle%20genes&amp;author=B.%20Przybyla-Zawislak&amp;author=D.M.%20Gadde&amp;author=K.%20Ducharme&amp;author=M.T.%20McCammon&amp;publication_year=1999&amp;journal=Genetics&amp;volume=152&amp;pages=153-66" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10224250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McAlister-Henn</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">L.M.</div></div>. </div><div class="article-title">Isolation and expression of the gene encoding yeast mitochondrial malate dehydrogenase</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1987</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">5157</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20expression%20of%20the%20gene%20encoding%20yeast%20mitochondrial%20malate%20dehydrogenase&amp;author=L.%20McAlister-Henn&amp;author=L.M.%20Thompson&amp;publication_year=1987&amp;journal=J.%20Bacteriol.&amp;volume=169&amp;pages=5157-66" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3312168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Sutherland</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Steffan</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">McAlister-Henn</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Gene sequence and primary structure of mitochondrial malate dehydrogenase from <em>Saccharomyces cerevisiae</em></div>, <div class="source">Biochemistry</div> , <div class="year">1988</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">8393</div>-<div class="lpage">400</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20sequence%20and%20primary%20structure%20of%20mitochondrial%20malate%20dehydrogenase%20from%20Saccharomyces%20cerevisiae&amp;author=L.M.%20Thompson&amp;author=P.%20Sutherland&amp;author=J.S.%20Steffan&amp;author=L.%20McAlister-Henn&amp;publication_year=1988&amp;journal=Biochemistry&amp;volume=27&amp;pages=8393-400" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00422a015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00422a015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00422a015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3072021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunze</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pracharoenwattana</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Hartig</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A central role for the peroxisomal membrane in glyoxylate cycle function</div>, <div class="source">Biochim. Biophys. Acta (BBA)—Mol. Cell Res.</div> , <div class="year">2006</div>, vol. <div class="volume">1763</div> (pg. <div class="fpage">1441</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20central%20role%20for%20the%20peroxisomal%20membrane%20in%20glyoxylate%20cycle%20function&amp;author=M.%20Kunze&amp;author=I.%20Pracharoenwattana&amp;author=S.M.%20Smith&amp;author=A.%20Hartig&amp;publication_year=2006&amp;journal=Biochim.%20Biophys.%20Acta%20(BBA)%E2%80%94Mol.%20Cell%20Res.&amp;volume=1763&amp;pages=1441-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbamcr.2006.09.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbamcr.2006.09.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbamcr.2006.09.009"> </span></div> </div></div></div></div></div><div content-id="DSS007C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minard</div> <div class="given-names">K.I.</div></div>,  <div class="name"><div class="surname">McAlister-Henn</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Isolation, nucleotide sequence analysis, and disruption of the MDH2 gene from <em>Saccharomyces cerevisiae</em>: evidence for three isozymes of yeast malate dehydrogenase</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">370</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%2C%20nucleotide%20sequence%20analysis%2C%20and%20disruption%20of%20the%20MDH2%20gene%20from%20Saccharomyces%20cerevisiae%3A%20evidence%20for%20three%20isozymes%20of%20yeast%20malate%20dehydrogenase&amp;author=K.I.%20Minard&amp;author=L.%20McAlister-Henn&amp;publication_year=1991&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=11&amp;pages=370-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.11.1.370" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.11.1.370" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.11.1.370"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1986231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McAlister-Henn</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Steffan</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Minard</div> <div class="given-names">K.I.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Expression and function of a mislocalized form of peroxisomal malate dehydrogenase (MDH3) in yeast</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">21220</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20and%20function%20of%20a%20mislocalized%20form%20of%20peroxisomal%20malate%20dehydrogenase%20(MDH3)%20in%20yeast&amp;author=L.%20McAlister-Henn&amp;author=J.S.%20Steffan&amp;author=K.I.%20Minard&amp;author=S.L.%20Anderson&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem.&amp;volume=270&amp;pages=21220-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.270.36.21220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.36.21220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.270.36.21220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7673155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steffan</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">McAlister-Henn</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Isolation and characterization of the yeast gene encoding the MDH3 isozyme of malate dehydrogenase</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1992</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">24708</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20the%20yeast%20gene%20encoding%20the%20MDH3%20isozyme%20of%20malate%20dehydrogenase&amp;author=J.S.%20Steffan&amp;author=L.%20McAlister-Henn&amp;publication_year=1992&amp;journal=J.%20Biol.%20Chem.&amp;volume=267&amp;pages=24708-15" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1447211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gould</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Hosken</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wilkinson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Subramani</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A conserved tripeptide sorts proteins to peroxisomes</div>, <div class="source">J. Cell Biol.</div> , <div class="year">1989</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">1657</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20conserved%20tripeptide%20sorts%20proteins%20to%20peroxisomes&amp;author=S.J.%20Gould&amp;author=G.A.%20Keller&amp;author=N.%20Hosken&amp;author=J.%20Wilkinson&amp;author=S.%20Subramani&amp;publication_year=1989&amp;journal=J.%20Cell%20Biol.&amp;volume=108&amp;pages=1657-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.108.5.1657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.108.5.1657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.108.5.1657"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2654139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aitchison</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">W.W.</div></div>,  <div class="name"><div class="surname">Rachubinski</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">The carboxyl-terminal tripeptide Ala-Lys-Ile is essential for targeting <em>Candida tropicalis</em> trifunctional enzyme to yeast peroxisomes</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1991</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">23197</div>-<div class="lpage">203</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20carboxyl-terminal%20tripeptide%20Ala-Lys-Ile%20is%20essential%20for%20targeting%20Candida%20tropicalis%20trifunctional%20enzyme%20to%20yeast%20peroxisomes&amp;author=J.D.%20Aitchison&amp;author=W.W.%20Murray&amp;author=R.A.%20Rachubinski&amp;publication_year=1991&amp;journal=J.%20Biol.%20Chem.&amp;volume=266&amp;pages=23197-203" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1744117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aitchison</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Nuttley</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Szilard</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Brade</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Glover</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Rachubinski</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Peroxisome biogenesis in yeast</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1992</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">3455</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Peroxisome%20biogenesis%20in%20yeast&amp;author=J.D.%20Aitchison&amp;author=W.M.%20Nuttley&amp;author=R.K.%20Szilard&amp;author=A.M.%20Brade&amp;author=J.R.%20Glover&amp;author=R.A.%20Rachubinski&amp;publication_year=1992&amp;journal=Mol.%20Microbiol.&amp;volume=6&amp;pages=3455-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1992.6.issue-23" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1992.6.issue-23" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1992.6.issue-23"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1474890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aitchison</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Szilard</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Nuttley</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Rachubinski</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Antibodies directed against a yeast carboxyl-terminal peroxisomal targeting signal specifically recognize peroxisomal proteins from various yeasts</div>, <div class="source">Yeast</div> , <div class="year">1992</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">721</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antibodies%20directed%20against%20a%20yeast%20carboxyl-terminal%20peroxisomal%20targeting%20signal%20specifically%20recognize%20peroxisomal%20proteins%20from%20various%20yeasts&amp;author=J.D.%20Aitchison&amp;author=R.K.%20Szilard&amp;author=W.M.%20Nuttley&amp;author=R.A.%20Rachubinski&amp;publication_year=1992&amp;journal=Yeast&amp;volume=8&amp;pages=721-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1279909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sherman</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nikolski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cayla</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Souciet</div> <div class="given-names">J.-L.</div></div>,  <div class="name"><div class="surname">Durrens</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Génolevures: protein families and synteny among complete hemiascomycetous yeast proteomes and genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D550</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=G%C3%A9nolevures%3A%20protein%20families%20and%20synteny%20among%20complete%20hemiascomycetous%20yeast%20proteomes%20and%20genomes&amp;author=D.J.%20Sherman&amp;author=T.%20Martin&amp;author=M.%20Nikolski&amp;author=C.%20Cayla&amp;author=J.-L.%20Souciet&amp;author=P.%20Durrens&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D550-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barth</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Wolf</div> <div class="given-names">K.</div></div>. </div><div class="article-title">The dimorphic fungus <em>Yarrowia lipolytica</em></div>, <div class="source">Genetics, Biochemistry and Molecular Biology of Non-Conventional Yeasts</div> , <div class="year">1996</div><div class="publisher-loc">Heidelberg</div><div class="publisher-name">Springer Verlag</div>(pg. <div class="fpage">313</div>-<div class="lpage">88</div>)</div></div></div></div><div content-id="DSS007C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mlickova</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Roux</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Athenstaedt</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Lipid accumulation, lipid body formation, and acyl coenzyme a oxidases of the yeast <em>Yarrowia lipolytica</em></div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">3918</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lipid%20accumulation%2C%20lipid%20body%20formation%2C%20and%20acyl%20coenzyme%20a%20oxidases%20of%20the%20yeast%20Yarrowia%20lipolytica&amp;author=K.%20Mlickova&amp;author=E.%20Roux&amp;author=K.%20Athenstaedt&amp;publication_year=2004&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=70&amp;pages=3918-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.70.7.3918-3924.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.70.7.3918-3924.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.70.7.3918-3924.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15240264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kahm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hasenbrink</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lichtenberg-Fraté</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ludwig</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kschischo</div> <div class="given-names">M.</div></div>. </div><div class="article-title">grofit: fitting biological growth curves with R</div>, <div class="source">J. Stat. Softw.</div> , <div class="year">2010</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=grofit%3A%20fitting%20biological%20growth%20curves%20with%20R&amp;author=M.%20Kahm&amp;author=G.%20Hasenbrink&amp;author=H.%20Lichtenberg-Frat%C3%A9&amp;author=J.%20Ludwig&amp;author=M.%20Kschischo&amp;publication_year=2010&amp;journal=J.%20Stat.%20Softw.&amp;volume=33&amp;pages=1-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.18637/jss.v033.i07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.18637%2Fjss.v033.i07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.18637%2Fjss.v033.i07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20808728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ihaka</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gentleman</div> <div class="given-names">R.</div></div>. </div><div class="article-title">R: a language for data analysis and graphics</div>, <div class="source">J. Comput. Graph. Stat.</div> , <div class="year">1996</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">299</div>-<div class="lpage">314</div>)</div></div></div></div><div content-id="DSS007C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fritsch</div> <div class="given-names">E.F.</div></div>. </div>, <div class="source">Molecular Cloning: A Laboratory Manual</div> , <div class="year">1989</div><div class="publisher-loc">Cold Spring Harbor</div><div class="publisher-name">Cold Spring Harbor Laboratory Press</div></div></div></div></div><div content-id="DSS007C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Querol</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Barrio</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Huerta</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ramon</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Molecular monitoring of wine fermentations conducted by active dry yeast strains</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1992</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">2948</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20monitoring%20of%20wine%20fermentations%20conducted%20by%20active%20dry%20yeast%20strains&amp;author=A.%20Querol&amp;author=E.%20Barrio&amp;author=T.%20Huerta&amp;author=D.%20Ramon&amp;publication_year=1992&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=58&amp;pages=2948-53" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16348768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nicaud</div> <div class="given-names">J.-M.</div></div>,  <div class="name"><div class="surname">Madzak</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">van den Broek</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Protein expression and secretion in the yeast Yarrowia lipolytica</div>, <div class="source">FEMS Yeast Res.</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">371</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20expression%20and%20secretion%20in%20the%20yeast%20Yarrowia%20lipolytica&amp;author=J.-M.%20Nicaud&amp;author=C.%20Madzak&amp;author=P.%20van%20den%20Broek&amp;publication_year=2002&amp;journal=FEMS%20Yeast%20Res.&amp;volume=2&amp;pages=371-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12702287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le Dall</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Nicaud</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Multiple-copy integration in the yeast <em>Yarrowia lipolytica</em></div>, <div class="source">Curr. Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">38</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple-copy%20integration%20in%20the%20yeast%20Yarrowia%20lipolytica&amp;author=M.T.%20Le%20Dall&amp;author=J.M.%20Nicaud&amp;author=C.%20Gaillardin&amp;publication_year=1994&amp;journal=Curr.%20Genet.&amp;volume=26&amp;pages=38-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00326302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00326302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00326302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7954894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Janke</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Magiera</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Rathfelder</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">A versatile toolbox for PCR-based tagging of yeast genes: new fluorescent proteins, more markers and promoter substitution cassettes</div>, <div class="source">Yeast</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">947</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20versatile%20toolbox%20for%20PCR-based%20tagging%20of%20yeast%20genes%3A%20new%20fluorescent%20proteins%2C%20more%20markers%20and%20promoter%20substitution%20cassettes&amp;author=C.%20Janke&amp;author=M.M.%20Magiera&amp;author=N.%20Rathfelder&amp;publication_year=2004&amp;journal=Yeast&amp;volume=21&amp;pages=947-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15334558" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Müller</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sandal</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kamp-Hansen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Dalbøge</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Comparison of expression systems in the yeasts <em>Saccharomyces cerevisiae, Hansenula polymorpha, Klyveromyces lactis, Schizosaccharomyces pombe</em> and <em>Yarrowia lipolytica</em>. Cloning of two novel promoters from <em>Yarrowia lipolytica</em></div>, <div class="source">Yeast</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1267</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20expression%20systems%20in%20the%20yeasts%20Saccharomyces%20cerevisiae%2C%20Hansenula%20polymorpha%2C%20Klyveromyces%20lactis%2C%20Schizosaccharomyces%20pombe%20and%20Yarrowia%20lipolytica.%20Cloning%20of%20two%20novel%20promoters%20from%20Yarrowia%20lipolytica&amp;author=S.%20M%C3%BCller&amp;author=T.%20Sandal&amp;author=P.%20Kamp-Hansen&amp;author=H.%20Dalb%C3%B8ge&amp;publication_year=1998&amp;journal=Yeast&amp;volume=14&amp;pages=1267-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9802206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kristiansen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>. </div><div class="article-title">SOAP: short oligonucleotide alignment program</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">713</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP%3A%20short%20oligonucleotide%20alignment%20program&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=K.%20Kristiansen&amp;author=J.%20Wang&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=713-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18227114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J Mol Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J%20Mol%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">R.C.</div></div>. </div><div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">1792</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Corpet</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Multiple sequence alignment with hierarchical clustering</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1988</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">10881</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20sequence%20alignment%20with%20hierarchical%20clustering&amp;author=F.%20Corpet&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res.&amp;volume=16&amp;pages=10881-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.22.10881" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.22.10881" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.22.10881"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2849754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Plewniak</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Jeanmougin</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>. </div><div class="article-title">The CLUSTAL_X Windows Interface: Flexible Strategies for Multiple Sequence Alignment Aided by Quality Analysis Tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">4876</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20CLUSTAL_X%20Windows%20Interface%3A%20Flexible%20Strategies%20for%20Multiple%20Sequence%20Alignment%20Aided%20by%20Quality%20Analysis%20Tools&amp;author=J.D.%20Thompson&amp;author=T.J.%20Gibson&amp;author=F.%20Plewniak&amp;author=F.%20Jeanmougin&amp;author=D.G.%20Higgins&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=4876-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.24.4876" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.24.4876"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9396791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Claros</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Vincens</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Computational method to predict mitochondrially imported proteins and their targeting sequences</div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">1996</div>, vol. <div class="volume">241</div> (pg. <div class="fpage">779</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20method%20to%20predict%20mitochondrially%20imported%20proteins%20and%20their%20targeting%20sequences&amp;author=M.G.%20Claros&amp;author=P.%20Vincens&amp;publication_year=1996&amp;journal=Eur.%20J.%20Biochem.&amp;volume=241&amp;pages=779-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ejb.1996.241.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fejb.1996.241.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fejb.1996.241.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8944766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neuberger</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Maurer-Stroh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Eisenhaber</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hartig</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Eisenhaber</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Prediction of peroxisomal targeting signal 1 containing proteins from amino acid sequence</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">328</div> (pg. <div class="fpage">581</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20peroxisomal%20targeting%20signal%201%20containing%20proteins%20from%20amino%20acid%20sequence&amp;author=G.%20Neuberger&amp;author=S.%20Maurer-Stroh&amp;author=B.%20Eisenhaber&amp;author=A.%20Hartig&amp;author=F.%20Eisenhaber&amp;publication_year=2003&amp;journal=J.%20Mol.%20Biol.&amp;volume=328&amp;pages=581-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(03)00319-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(03)00319-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(03)00319-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12706718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dujon</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fischer</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Genome evolution in yeasts</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">430</div> (pg. <div class="fpage">35</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20evolution%20in%20yeasts&amp;author=B.%20Dujon&amp;author=D.%20Sherman&amp;author=G.%20Fischer&amp;publication_year=2004&amp;journal=Nature&amp;volume=430&amp;pages=35-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15229592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De Schutter</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">Y.-C.</div></div>,  <div class="name"><div class="surname">Tiels</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Genome sequence of the recombinant protein production host <em>Pichia pastoris</em></div>, <div class="source">Nat. Biotech.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">561</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20recombinant%20protein%20production%20host%20Pichia%20pastoris&amp;author=K.%20De%20Schutter&amp;author=Y.-C.%20Lin&amp;author=P.%20Tiels&amp;publication_year=2009&amp;journal=Nat.%20Biotech.&amp;volume=27&amp;pages=561-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1544"> </span></div> </div></div></div></div></div><div content-id="DSS007C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gattiker</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rischatsch</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Demougin</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Ashbya Genome Database 3.0: a cross-species genome and transcriptome browser for yeast biologists</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">9</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ashbya%20Genome%20Database%203.0%3A%20a%20cross-species%20genome%20and%20transcriptome%20browser%20for%20yeast%20biologists&amp;author=A.%20Gattiker&amp;author=R.%20Rischatsch&amp;author=P.%20Demougin&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17212814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Roermund</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Elgersma</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wanders</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Tabak</div> <div class="given-names">H.F.</div></div>. </div><div class="article-title">The membrane of peroxisomes in <em>Saccharomyces cerevisiae</em> is impermeable to NAD(H) and acetyl-CoA under <em>in vivo</em> conditions</div>, <div class="source">EMBO J.</div> , <div class="year">1995</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">3480</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20membrane%20of%20peroxisomes%20in%20Saccharomyces%20cerevisiae%20is%20impermeable%20to%20NAD(H)%20and%20acetyl-CoA%20under%20in%20vivo%20conditions&amp;author=C.W.%20van%20Roermund&amp;author=Y.%20Elgersma&amp;author=N.%20Singh&amp;author=R.J.%20Wanders&amp;author=H.F.%20Tabak&amp;publication_year=1995&amp;journal=EMBO%20J.&amp;volume=14&amp;pages=3480-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7628449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">Y.J.</div></div>,  <div class="name"><div class="surname">Jang</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Maeng</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">TCA cycle-independent acetate metabolism via the glyoxylate cycle in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Yeast</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">153</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TCA%20cycle-independent%20acetate%20metabolism%20via%20the%20glyoxylate%20cycle%20in%20Saccharomyces%20cerevisiae&amp;author=Y.J.%20Lee&amp;author=J.W.%20Jang&amp;author=K.J.%20Kim&amp;author=P.J.%20Maeng&amp;publication_year=2011&amp;journal=Yeast&amp;volume=28&amp;pages=153-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/yea.v28.2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fyea.v28.2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fyea.v28.2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21246628" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCammon</div> <div class="given-names">M.T.</div></div>. </div><div class="article-title">Mutants of <em>Saccharomyces cerevisiae</em> with defects in acetate metabolism: isolation and characterization of Acn- mutants</div>, <div class="source">Genetics</div> , <div class="year">1996</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">57</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutants%20of%20Saccharomyces%20cerevisiae%20with%20defects%20in%20acetate%20metabolism%3A%20isolation%20and%20characterization%20of%20Acn-%20mutants&amp;author=M.T.%20McCammon&amp;publication_year=1996&amp;journal=Genetics&amp;volume=144&amp;pages=57-69" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8878673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oh</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Cheon</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Kang</div> <div class="given-names">H.A.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.Y.</div></div>. </div><div class="article-title">Functional characterization of the unconventional splicing of <em>Yarrowia lipolytica</em> HAC1 mRNA induced by unfolded protein response</div>, <div class="source">Yeast</div> , <div class="year">2010</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">443</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20characterization%20of%20the%20unconventional%20splicing%20of%20Yarrowia%20lipolytica%20HAC1%20mRNA%20induced%20by%20unfolded%20protein%20response&amp;author=M.H.%20Oh&amp;author=S.A.%20Cheon&amp;author=H.A.%20Kang&amp;author=J.Y.%20Kim&amp;publication_year=2010&amp;journal=Yeast&amp;volume=27&amp;pages=443-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/yea.v27:7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fyea.v27:7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fyea.v27:7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20162530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Babour</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kabani</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Boisrame</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Beckerich</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Characterization of Ire1 in the yeast Yarrowia lipolytica reveals an important role for the Sls1 nucleotide exchange factor in unfolded protein response regulation</div>, <div class="source">Curr. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">337</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Ire1%20in%20the%20yeast%20Yarrowia%20lipolytica%20reveals%20an%20important%20role%20for%20the%20Sls1%20nucleotide%20exchange%20factor%20in%20unfolded%20protein%20response%20regulation&amp;author=A.%20Babour&amp;author=M.%20Kabani&amp;author=A.%20Boisrame&amp;author=J.M.%20Beckerich&amp;publication_year=2008&amp;journal=Curr.%20Genet.&amp;volume=53&amp;pages=337-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00294-008-0190-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00294-008-0190-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00294-008-0190-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18421459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Small</div> <div class="given-names">W.C.</div></div>,  <div class="name"><div class="surname">McAlister-Henn</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Metabolic effects of altering redundant targeting signals for yeast mitochondrial malate dehydrogenase</div>, <div class="source">Arch. Biochem. Biophys.</div> , <div class="year">1997</div>, vol. <div class="volume">344</div> (pg. <div class="fpage">53</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20effects%20of%20altering%20redundant%20targeting%20signals%20for%20yeast%20mitochondrial%20malate%20dehydrogenase&amp;author=W.C.%20Small&amp;author=L.%20McAlister-Henn&amp;publication_year=1997&amp;journal=Arch.%20Biochem.%20Biophys.&amp;volume=344&amp;pages=53-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/abbi.1997.0179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fabbi.1997.0179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fabbi.1997.0179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9244381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunze</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kragler</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Binder</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hartig</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gurvitz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Targeting of malate synthase 1 to the peroxisomes of <em>Saccharomyces cerevisiae</em> cells depends on growth on oleic acid medium</div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">2002</div>, vol. <div class="volume">269</div> (pg. <div class="fpage">915</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeting%20of%20malate%20synthase%201%20to%20the%20peroxisomes%20of%20Saccharomyces%20cerevisiae%20cells%20depends%20on%20growth%20on%20oleic%20acid%20medium&amp;author=M.%20Kunze&amp;author=F.%20Kragler&amp;author=M.%20Binder&amp;author=A.%20Hartig&amp;author=A.%20Gurvitz&amp;publication_year=2002&amp;journal=Eur.%20J.%20Biochem.&amp;volume=269&amp;pages=915-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.0014-2956.2001.02727.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.0014-2956.2001.02727.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.0014-2956.2001.02727.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolinski</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Petrovic</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Mattiazzi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Imaging-based live cell yeast screen identifies novel factors involved in peroxisome assembly</div>, <div class="source">J. Proteome Res</div> , <div class="year">2009</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">20</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Imaging-based%20live%20cell%20yeast%20screen%20identifies%20novel%20factors%20involved%20in%20peroxisome%20assembly&amp;author=H.%20Wolinski&amp;author=U.%20Petrovic&amp;author=M.%20Mattiazzi&amp;publication_year=2009&amp;journal=J.%20Proteome%20Res&amp;volume=8&amp;pages=20-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/pr800782n" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fpr800782n" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fpr800782n"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19118449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barth</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Scheuber</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Cloning of the isocitrate lyase gene (ICL1) from Yarrowia lipolytica and characterization of the deduced protein</div>, <div class="source">Mol Gen Genet</div> , <div class="year">1993</div>, vol. <div class="volume">241</div> (pg. <div class="fpage">422</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20of%20the%20isocitrate%20lyase%20gene%20(ICL1)%20from%20Yarrowia%20lipolytica%20and%20characterization%20of%20the%20deduced%20protein&amp;author=G.%20Barth&amp;author=T.%20Scheuber&amp;publication_year=1993&amp;journal=Mol%20Gen%20Genet&amp;volume=241&amp;pages=422-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8246896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pracharoenwattana</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Cornah</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">S.M.</div></div>. </div><div class="article-title">Arabidopsis peroxisomal malate dehydrogenase functions in beta-oxidation but not in the glyoxylate cycle</div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">381</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20peroxisomal%20malate%20dehydrogenase%20functions%20in%20beta-oxidation%20but%20not%20in%20the%20glyoxylate%20cycle&amp;author=I.%20Pracharoenwattana&amp;author=J.E.%20Cornah&amp;author=S.M.%20Smith&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=50&amp;pages=381-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03055.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03055.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03055.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17376163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Visser</div> <div class="given-names">W.F.</div></div>,  <div class="name"><div class="surname">van Roermund</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Ijlst</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Waterham</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Wanders</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Metabolite transport across the peroxisomal membrane</div>, <div class="source">Biochem J.</div> , <div class="year">2007</div>, vol. <div class="volume">401</div> (pg. <div class="fpage">365</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolite%20transport%20across%20the%20peroxisomal%20membrane&amp;author=W.F.%20Visser&amp;author=C.W.%20van%20Roermund&amp;author=L.%20Ijlst&amp;author=H.R.%20Waterham&amp;author=R.J.%20Wanders&amp;publication_year=2007&amp;journal=Biochem%20J.&amp;volume=401&amp;pages=365-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BJ20061352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBJ20061352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBJ20061352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17173541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haddouche</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Poirier</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Delessert</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Engineering polyhydroxyalkanoate content and monomer composition in the oleaginous yeast Yarrowia lipolytica by modifying the ß-oxidation multifunctional protein</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1327</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20polyhydroxyalkanoate%20content%20and%20monomer%20composition%20in%20the%20oleaginous%20yeast%20Yarrowia%20lipolytica%20by%20modifying%20the%20%C3%9F-oxidation%20multifunctional%20protein&amp;author=R.%20Haddouche&amp;author=Y.%20Poirier&amp;author=S.%20Delessert&amp;publication_year=2011&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=91&amp;pages=1327-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-011-3331-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-011-3331-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-011-3331-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21603933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vorapreeda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Thammarongtham</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cheevadhanarak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Laoteng</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Alternative routes of acetyl-CoA synthesis identified by comparative genomic analysis: Involvement in lipid production of oleaginous yeast and fungi</div>, <div class="source">Microbiology</div> , <div class="year">2011</div> <div class="comment">in press, doi:10.1099/mic.0.051946-0</div></div></div></div></div><div content-id="DSS007C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cuenca-Bono</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Garcia-Molinero</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Pascual-Garcia</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">SUS1 introns are required for efficient mRNA nuclear export in yeast</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div> <div class="comment">doi:10.10.93/nar/gkr496</div></div></div></div></div><div content-id="DSS007C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hossain</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Rodriguez</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">T.L.</div></div>. </div><div class="article-title">Key features of the two-intron <em>Saccharomyces cerevisiae</em> gene SUS1 contribute to its alternative splicing</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">8612</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Key%20features%20of%20the%20two-intron%20Saccharomyces%20cerevisiae%20gene%20SUS1%20contribute%20to%20its%20alternative%20splicing&amp;author=M.A.%20Hossain&amp;author=C.M.%20Rodriguez&amp;author=T.L.%20Johnson&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=8612-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21749978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davis</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Grate</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Spingola</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ares</div> <div class="given-names">M.</div>Jr</div>. </div><div class="article-title">Test of intron predictions reveals novel splice sites, alternatively spliced mRNAs and new introns in meiotically regulated genes of yeast</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1700</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Test%20of%20intron%20predictions%20reveals%20novel%20splice%20sites%2C%20alternatively%20spliced%20mRNAs%20and%20new%20introns%20in%20meiotically%20regulated%20genes%20of%20yeast&amp;author=C.A.%20Davis&amp;author=L.%20Grate&amp;author=M.%20Spingola&amp;author=M.%20Ares&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=1700-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.8.1700" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.8.1700" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.8.1700"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10734188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Engebrecht</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Voelkel-Meiman</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Roeder</div> <div class="given-names">G.S.</div></div>. </div><div class="article-title">Meiosis-specific RNA splicing in yeast</div>, <div class="source">Cell</div> , <div class="year">1991</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">1257</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meiosis-specific%20RNA%20splicing%20in%20yeast&amp;author=J.%20Engebrecht&amp;author=K.%20Voelkel-Meiman&amp;author=G.S.%20Roeder&amp;publication_year=1991&amp;journal=Cell&amp;volume=66&amp;pages=1257-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(91)90047-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(91)90047-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(91)90047-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1840507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakagawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ogawa</div> <div class="given-names">H.</div></div>. </div><div class="article-title">The <em>Saccharomyces cerevisiae</em> MER3 gene, encoding a novel helicase-like protein, is required for crossover control in meiosis</div>, <div class="source">EMBO J.</div> , <div class="year">1999</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">5714</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Saccharomyces%20cerevisiae%20MER3%20gene%2C%20encoding%20a%20novel%20helicase-like%20protein%2C%20is%20required%20for%20crossover%20control%20in%20meiosis&amp;author=T.%20Nakagawa&amp;author=H.%20Ogawa&amp;publication_year=1999&amp;journal=EMBO%20J.&amp;volume=18&amp;pages=5714-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/18.20.5714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F18.20.5714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F18.20.5714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10523314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Vilardell</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Warner</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">An RNA structure involved in feedback regulation of splicing and of translation is critical for biological fitness</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">1596</div>-<div class="lpage">600</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20RNA%20structure%20involved%20in%20feedback%20regulation%20of%20splicing%20and%20of%20translation%20is%20critical%20for%20biological%20fitness&amp;author=B.%20Li&amp;author=J.%20Vilardell&amp;author=J.R.%20Warner&amp;publication_year=1996&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=1596-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.4.1596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.4.1596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.4.1596"> </span></div> </div></div></div></div></div><div content-id="DSS007C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Preker</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Guthrie</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Autoregulation of the mRNA export factor Yra1p requires inefficient splicing of its pre-mRNA</div>, <div class="source">RNA</div> , <div class="year">2006</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">994</div>-<div class="lpage">1006</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Autoregulation%20of%20the%20mRNA%20export%20factor%20Yra1p%20requires%20inefficient%20splicing%20of%20its%20pre-mRNA&amp;author=P.J.%20Preker&amp;author=C.%20Guthrie&amp;publication_year=2006&amp;journal=RNA&amp;volume=12&amp;pages=994-1006" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.6706" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.6706" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.6706"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16618971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ben-Menachem</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Tal</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pines</div> <div class="given-names">O.</div></div>. </div><div class="article-title">A third of the yeast mitochondrial proteome is dual localized: a question of evolution</div>, <div class="source">Proteomics</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">4468</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20third%20of%20the%20yeast%20mitochondrial%20proteome%20is%20dual%20localized%3A%20a%20question%20of%20evolution&amp;author=R.B.%20Ben-Menachem&amp;author=M.%20Tal&amp;author=O.%20Pines&amp;publication_year=2011&amp;journal=Proteomics&amp;volume=11&amp;pages=4468-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/pmic.201100199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fpmic.201100199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fpmic.201100199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21910249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS007C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS007C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haddouche</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Delessert</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sabirova</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Neuvéglise</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Poirier</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nicaud</div> <div class="given-names">J.-M.</div></div>. </div><div class="article-title">Roles of multiple acyl-CoA oxidases in the routing of carbon flow towards β-oxidation and polyhydroxyalkanoate biosynthesis in <em>Yarrowia lipolytica</em></div>, <div class="source">FEMS Yeast Res.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">917</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Roles%20of%20multiple%20acyl-CoA%20oxidases%20in%20the%20routing%20of%20carbon%20flow%20towards%20%CE%B2-oxidation%20and%20polyhydroxyalkanoate%20biosynthesis%20in%20Yarrowia%20lipolytica&amp;author=R.%20Haddouche&amp;author=S.%20Delessert&amp;author=J.%20Sabirova&amp;author=C.%20Neuv%C3%A9glise&amp;author=Y.%20Poirier&amp;author=J.-M.%20Nicaud&amp;publication_year=2010&amp;journal=FEMS%20Yeast%20Res.&amp;volume=10&amp;pages=917-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2010.10.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2010.10.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2010.10.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20726896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Takashi Ito</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=474863&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss007-dss007supp_fig2.pdf?Expires=2147483647&amp;Signature=sRAY4TqLsirLIZZ8HZeK~3KufLlMTkLVSeGo9akSs24CGGUAh7J~qSAYDpTs-BAKP65zP1dHQjX3bi2-2BPD9z97GSNolx99sMJ-ZWyJT3K2J1Yks7m7gxCuBM6TShPp87TXZN87J9bTGiJHQXM55yHI8LyjSzdDumzdoYK65pXSGvLljl5vQtt9NehZ6svgjNDnF2OR~9rtPcW0ncczquqkppr~HNfpBdGtvaJII5TN8-BjWCHcIQIAbiskAQHRurObcwEUn1H6IuIrUkaqCoqIgw2mCXEe8yEa4Y2T5IqQNKXDrooX0v4TfrBRmrYGp-ZkKgkx7-Yqnb-Cr4q1-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss007/2/dss007-dss007supp_fig1.pdf?Expires=2147483647&amp;Signature=oybWQPWhqk~LDVT4mgKiWZUKDOhNaEO~ZQKlbZuhgF8LQlLY9P6E4EEOY~mWOOyfMKCbYrkh6erSPCdTeT99I4vKete7EH5DvHna-YY62pFBauhbJHaTF8kX-lmD9lQZlv6dXNgv0EbJKDLDD0RH1ryAwWV86utgyvCMj2mM2geOp9ZrkcQoauCg-Do7FbPCTf6pr04YNMVOAIhbolAiV7FqBfQUFLBHnJFuIFy-J2RfSCGQJOVRYro8Xnr6ylprK~qab0vigO1Wk22JZL4VEQYu5AJMhxr7OVYWR3yQnBeuecN2ubFvln7OM9ALVuccl0pqAy-uomVBPbnQk-cGmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">264</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">24</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss007"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="474863"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f3%2f231%2f474863">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000305723200003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (24)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F3%2F231%2F474863" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/474863'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Alternative Splicing Regulates Targeting of Malate Dehydrogenase in Yarrowia lipolytica" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss007");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f231%2f474863">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

