

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2172,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/3/271/1745617","@type":"ScholarlyArticle","name":"Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication","datePublished":"2016-04-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2016-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw014","keywords":["artificial selection","cultivar differentiation","domestication","linkage disequilibrium","tree crop"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/3/271/1745617","author":[{"name":"Akagi, Takashi","affiliation":"1  Laboratory of Pomology, Graduate School of Agriculture ,  Kyoto University ,  Kita-shirakawa, Oiwake-cho, Kyoto 606-8502 ,  Japan","@type":"Person"},{"name":"Hanada, Toshio","affiliation":"2  Apple Research Division ,  NARO Institute of Fruit Tree Science ,  Morioka 020-0123 ,  Japan","@type":"Person"},{"name":"Yaegaki, Hideaki","affiliation":"3  Breeding and Pest Management Division, NARO Institute ,  Tsukuba, Ibaragi 305-8605 ,  Japan","@type":"Person"},{"name":"Gradziel, Thomas M.","affiliation":"4  Department of Plant Sciences ,  University of California  Davis, CA 95616 ,  USA","@type":"Person"},{"name":"Tao, Ryutaro","affiliation":"1  Laboratory of Pomology, Graduate School of Agriculture ,  Kyoto University ,  Kita-shirakawa, Oiwake-cho, Kyoto 606-8502 ,  Japan","@type":"Person"}],"description":"Abstract.  Domestication and cultivar differentiation are requisite processes for establishing cultivated crops. These processes inherently involve substantial ","pageStart":"271","pageEnd":"282","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication" />
    <meta property="og:description" content="Abstract.  Domestication and cultivar differentiation are requisite processes for establishing cultivated crops. These processes inherently involve substantial " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/3/271/1745617" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-04-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Akagi, Takashi" /><meta name="citation_author_institution" content="Laboratory of Pomology, Graduate School of Agriculture, Kyoto University, Kita-shirakawa, Oiwake-cho, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Hanada, Toshio" /><meta name="citation_author_institution" content="Apple Research Division, NARO Institute of Fruit Tree Science, Morioka 020-0123, Japan" /><meta name="citation_author" content="Yaegaki, Hideaki" /><meta name="citation_author_institution" content="Breeding and Pest Management Division, NARO Institute, Tsukuba, Ibaragi 305-8605, Japan" /><meta name="citation_author" content="Gradziel, Thomas M." /><meta name="citation_author_institution" content="Department of Plant Sciences, University of CaliforniaDavis, CA 95616, USA" /><meta name="citation_author" content="Tao, Ryutaro" /><meta name="citation_author_institution" content="Laboratory of Pomology, Graduate School of Agriculture, Kyoto University, Kita-shirakawa, Oiwake-cho, Kyoto 606-8502, Japan" /><meta name="citation_title" content="Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication" /><meta name="citation_firstpage" content="271" /><meta name="citation_lastpage" content="282" /><meta name="citation_doi" content="10.1093/dnares/dsw014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2016/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Documenting domestication: the intersection of genetics and archaeology;  citation_journal_title=Trends Genet.;  citation_year=2006;  citation_volume=22;  citation_pages=139-55; " /><meta name="citation_reference" content="citation_title=The molecular genetics of crop domestication;  citation_journal_title=Cell;  citation_year=2006;  citation_volume=127;  citation_pages=1309-21; " /><meta name="citation_reference" content="citation_title=Genome-wide patterns of nucleotide polymorphism in domesticated rice;  citation_journal_title=PLoS Genet.;  citation_year=2007;  citation_volume=3;  citation_pages=e163" /><meta name="citation_reference" content="citation_title=The genetic expectations of a protracted model for the origins of domesticated crops;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=13982-86; " /><meta name="citation_reference" content="citation_title=Genetic perspectives on crop domestication;  citation_journal_title=Trends Plant Sci.;  citation_year=2010;  citation_volume=15;  citation_pages=529-37; " /><meta name="citation_reference" content="citation_title=Genomic screening for artificial selection during domestication and improvement in maize;  citation_journal_title=Ann. Bot.;  citation_year=2007;  citation_volume=100;  citation_pages=967-73; " /><meta name="citation_reference" content="citation_title=Recent and ongoing selection in the human genome;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2007;  citation_volume=8;  citation_pages=857-68; " /><meta name="citation_reference" content="citation_title=Artificial selection for a green revolution gene during japonica rice domestication;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=11034-39; " /><meta name="citation_reference" content="citation_title=Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes;  citation_journal_title=Nat. Biotech.;  citation_year=2012;  citation_volume=30;  citation_pages=105-11; " /><meta name="citation_reference" content="citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=8057-62; " /><meta name="citation_reference" content="citation_title=Pattern of diversity in the genomic region near the maize domestication gene tb1;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=700-7; " /><meta name="citation_reference" content="citation_title=Long-range patterns of diversity and linkage disequilibrium surrounding the maize Y1 gene are indicative of an asymmetric selective sweep;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=9885-90; " /><meta name="citation_reference" content="citation_title=The effects of artificial selection on the maize genome;  citation_journal_title=Science;  citation_year=2005;  citation_volume=308;  citation_pages=1310-4; " /><meta name="citation_reference" content="citation_title=A large-scale screen for artificial selection in maize identifies candidate agronomic loci for domestication and crop improvement;  citation_journal_title=Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=2859-72; " /><meta name="citation_reference" content="citation_title=Crop genomics: advances and applications;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=13;  citation_pages=85-96; " /><meta name="citation_reference" content="citation_title=Genomic selection for crop improvement;  citation_journal_title=Crop Sci.;  citation_year=2009;  citation_volume=49;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies for complex traits: consensus, uncertainty and challenges;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2008;  citation_volume=9;  citation_pages=356-9; " /><meta name="citation_reference" content="citation_title=Genetic structure and domestication history of the grape;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=3530-5; " /><meta name="citation_reference" content="citation_title=Tracing the geographic origins of major avocado cultivars;  citation_journal_title=J. Hered.;  citation_year=2009;  citation_volume=100;  citation_pages=56.65" /><meta name="citation_reference" content="citation_title=Domestication of a Mesoamerican cultivated fruit tree, Spondias purpurea;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=12801-06; " /><meta name="citation_reference" content="citation_title=From forest to field: perennial fruit crops domestication;  citation_journal_title=Am. J. Bot.;  citation_year=2011;  citation_volume=98;  citation_pages=1389-414; " /><meta name="citation_reference" content="citation_title=New insight into the history of domesticated apple: secondary contribution of the European wild apple to the genome of cultivated varieties;  citation_journal_title=PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002703" /><meta name="citation_reference" content="citation_title=Effects of colonization processes on genetic diversity: differences between annual plants and tree species;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=154;  citation_pages=1309-21; " /><meta name="citation_reference" content="citation_title=Some evolutionary consequences of being a tree;  citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;  citation_year=2006;  citation_volume=37;  citation_pages=187-214; " /><meta name="citation_reference" content="citation_title=Genomic diversity in forest trees;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2007;  citation_volume=10;  citation_pages=162-7; " /><meta name="citation_reference" content="citation_title=A comparative view of the evolution of grasses under domestication;  citation_journal_title=New Phytol.;  citation_year=2009;  citation_volume=183;  citation_pages=273-90; " /><meta name="citation_reference" content="citation_title=Self-incompatibility in sexually propagated cultivated plants;  citation_journal_title=Euphytica;  citation_year=1964;  citation_volume=13;  citation_pages=157-62; " /><meta name="citation_reference" content="citation_publisher=Wiley, New York;  citation_title=Origin and dissemination of peach, Hort. Rev.;  citation_year=1995;  citation_volume=17;  citation_pages=331-80; " /><meta name="citation_reference" content="citation_publisher=CAB International, ;  citation_title=History of cultivation and trends in China, the peach: botany production and uses;  citation_year=2008;  citation_pages=37-60; " /><meta name="citation_reference" content="citation_title=Comparative population genomics reveals the domestication history of the peach, Prunus persica, and human influences on perennial fruit crops;  citation_journal_title=Genome Biol.;  citation_year=2014;  citation_volume=15;  citation_pages=415" /><meta name="citation_reference" content="citation_title=Inbreeding and coancestry of freestone peach cultivars of the eastern United States and implications for peach germplasm improvement;  citation_journal_title=J. Amer. Soc. Hort. Sci.;  citation_year=1985;  citation_volume=110;  citation_pages=547-52; " /><meta name="citation_reference" content="citation_title=Genetic diversity in peach (Prunus persica L. Batch) revealed by randomly amplified polymorphic DNA (RAPD) markers and compared to inbreeding coefficients;  citation_journal_title=J. Am. Soc. Hortic. Sci.;  citation_year=1996;  citation_volume=121;  citation_pages=1012-9; " /><meta name="citation_reference" content="citation_title=Genetic variation, population structure and linkage disequilibrium in peach commercial varieties;  citation_journal_title=BMC Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=69" /><meta name="citation_reference" content="citation_title=Genetic characterization of different demes in Prunus persica revealed by RAPD markers;  citation_journal_title=Sci. Hortic.;  citation_year=2007;  citation_volume=111;  citation_pages=242-7; " /><meta name="citation_reference" content="citation_title=SSR fingerprinting Chinese peach cultivars and landraces (Prunus persica) and analysis of their genetic relationships;  citation_journal_title=Sci. Hortic.;  citation_year=2009;  citation_volume=120;  citation_pages=188-93; " /><meta name="citation_reference" content="citation_title=Evaluation of the genetic diversity of Asian peach accessions using a selected set of SSR markers;  citation_journal_title=Sci. Hortic.;  citation_year=2010;  citation_volume=125;  citation_pages=622-9; " /><meta name="citation_reference" content="citation_title=Relationships among peach, almond, and related species as detected by simple sequence repeat marker;  citation_journal_title=J. Am. Soc. Hortic. Sci.;  citation_year=2003;  citation_volume=128;  citation_pages=667-71; " /><meta name="citation_reference" content="citation_title=Genetic relationship of ornamental peach determined using AFLP markers;  citation_journal_title=Hort. Sci.;  citation_year=2005;  citation_volume=40;  citation_pages=1782-6; " /><meta name="citation_reference" content="citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=587-91; " /><meta name="citation_reference" content="citation_title=A nonparametric test reveals selection for rapid flowering in the Arabidopsis genome;  citation_journal_title=PLoS Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=e137" /><meta name="citation_reference" content="citation_title=Comparative population genomics of maize domestication and improvement;  citation_journal_title=Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=808-11; " /><meta name="citation_reference" content="citation_title=Population genomic and genome-wide association studies of agroclimatic traits in sorghum;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=453-8; " /><meta name="citation_reference" content="citation_title=Development and evaluation of a 9K SNP array for peach by internationally coordinated SNP detection and validation in breeding germplasm;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=10" /><meta name="citation_reference" content="citation_title=The high-quality draft genome of peach (Prunus persica) identifies unique patterns of genetic diversity, domestication and genome evolution; citation_author=International Peach Genome Initiative;  citation_journal_title=Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=487-94; " /><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2007;  citation_volume=81;  citation_pages=559-75; " /><meta name="citation_reference" content="citation_title=GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism;  citation_journal_title=J. Hered.;  citation_year=1995;  citation_volume=86;  citation_pages=248-9; " /><meta name="citation_reference" content="citation_title=Genepop'007: a complete reimplementation of the Genepop software for Windows and Linux;  citation_journal_title=Mol. Ecol. Resources;  citation_year=2008;  citation_volume=8;  citation_pages=103-6; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-59; " /><meta name="citation_reference" content="citation_title=An MCMC approach for joint inference of population structure and inbreeding rates from multi-locus genotype data;  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=1635-51; " /><meta name="citation_reference" content="citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study;  citation_journal_title=Mol. Ecol.;  citation_year=2005;  citation_volume=14;  citation_pages=2611-20; " /><meta name="citation_reference" content="citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2006;  citation_volume=78;  citation_pages=629-44; " /><meta name="citation_reference" content="citation_title=Detecting recent positive selection in the human genome from haplotype structure;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=419;  citation_pages=832-7; " /><meta name="citation_reference" content="citation_title=Genome-wide detection and characterization of positive selection in human populations;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=449;  citation_pages=913-8; " /><meta name="citation_reference" content="citation_title=A map of recent positive selection in the human genome;  citation_journal_title=PLoS Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=e72" /><meta name="citation_reference" content="citation_title=A Y-chromosome–encoded small RNA acts as a sex determinant in persimmons;  citation_journal_title=Science;  citation_year=2014;  citation_volume=346;  citation_pages=646-50; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1451-2; " /><meta name="citation_reference" content="citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=221-4; " /><meta name="citation_reference" content="citation_title=The extent of linkage disequilibrium in rice (Oryza sativa L.);  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=177;  citation_pages=2223-32; " /><meta name="citation_reference" content="citation_title=Genetic characterization and linkage disequilibrium estimation of a global maize collection using SNP markers;  citation_journal_title=PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e8451" /><meta name="citation_reference" content="citation_title=Adaptation to climate across the Arabidopsis thaliana genome;  citation_journal_title=Science;  citation_year=2011;  citation_volume=334;  citation_pages=83-6; " /><meta name="citation_reference" content="citation_title=Tracking footprints of artificial selection in the dog genome;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=1160-5; " /><meta name="citation_reference" content="citation_title=PETALLOSS gene regulates initiation and orientation of second whorl organs in the Arabidopsis flower;  citation_journal_title=Development;  citation_year=1999;  citation_volume=126;  citation_pages=5635-44; " /><meta name="citation_reference" content="citation_title=PETALLOSS, a trihelix transcription factor gene, regulates perianth architecture in the Arabidopsis flower;  citation_journal_title=Development;  citation_year=2004;  citation_volume=131;  citation_pages=4035-45; " /><meta name="citation_reference" content="citation_title=PETALLOSS is a boundary gene that inhibits growth between developing sepals in Arabidopsis thaliana;  citation_journal_title=Plant J.;  citation_year=2012;  citation_volume=71;  citation_pages=724-35; " /><meta name="citation_reference" content="citation_title=Evidence of a secondary grapevine domestication centre detected by SSR analysis;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=1315-20; " /><meta name="citation_reference" content="citation_title=QTL analysis of quality traits in an advanced backcross between Prunus persica cultivars and the wild relative species, P. davidiana;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=884-97; " /><meta name="citation_reference" content="citation_title=Structural and transcriptional analysis of the self-incompatibility locus of almond: identification of a pollen-expressed F-box gene with haplotype-specific polymorphism;  citation_journal_title=Plant Cell;  citation_year=2003;  citation_volume=15;  citation_pages=771-81; " /><meta name="citation_reference" content="citation_title=The S-RNase-based gametophytic self-incompatibility system in Prunus exhibits distinct genetic and molecular features;  citation_journal_title=Sci. Hort.;  citation_year=2010;  citation_volume=124;  citation_pages=423-33; " /><meta name="citation_reference" content="citation_title=Self-compatible peach (Prunus persica) has mutant versions of the S haplotypes found in self-incompatible Prunus species;  citation_journal_title=Plant Mol. Biol.;  citation_year=2007;  citation_volume=63;  citation_pages=109-23; " /><meta name="citation_reference" content="citation_title=Two novel self-compatible S haplotypes in peach (Prunus persica);  citation_journal_title=J. Jpn Soc. Hortic. Sci.;  citation_year=2014;  citation_volume=83;  citation_pages=201-13; " /><meta name="citation_reference" content="citation_title=Basic RNases of wild almond (Prunus webbii): cloning and characterization of six new S-RNase and one “non-S RNase” genes;  citation_journal_title=J. Plant Physiol.;  citation_year=2009;  citation_volume=166;  citation_pages=395-402; " /><meta name="citation_reference" content="citation_title=Metabolic engineering of carotenoid biosynthesis in plants;  citation_journal_title=Trends Biotech.;  citation_year=2008;  citation_volume=26;  citation_pages=139-45; " /><meta name="citation_reference" content="citation_title=An expanded genetic linkage map of Prunus based on an interspecific cross between almond and peach;  citation_journal_title=Genome;  citation_year=2002;  citation_volume=45;  citation_pages=520-9; " /><meta name="citation_reference" content="citation_title=Utility of RAPD markers in identifying genetic linkages to genes of economic interest in peach;  citation_journal_title=Theor. Appl. Genet.;  citation_year=1996;  citation_volume=93;  citation_pages=920-5; " /><meta name="citation_reference" content="citation_title=QTL analysis of fruit quality traits in two peach intraspecific populations and importance of maturity date pleiotropic effect;  citation_journal_title=Tree Genet. Genomes;  citation_year=2011;  citation_volume=7;  citation_pages=323-35; " /><meta name="citation_reference" content="citation_title=Arabidopsis thaliana AtHAL3: a flavoprotein related to salt and osmotic tolerance and plant growth;  citation_journal_title=Plant J.;  citation_year=1999;  citation_volume=20;  citation_pages=529-39; " /><meta name="citation_reference" content="citation_title=Multiple roles of Arabidopsis VRN1 in vernalization and flowering time control;  citation_journal_title=Science;  citation_year=2002;  citation_volume=297;  citation_pages=243-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/3/271/17463287/dsw014.pdf" /><meta name="description" content="Abstract.  Domestication and cultivar differentiation are requisite processes for establishing cultivated crops. These processes inherently involve substantial " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/3/271/1745617" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/3/271/1745617" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/3/271/1745617");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw014';
    </script>
<title>
	Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745617" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfWuATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWthZ2ksIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIFBvbW9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIEt5b3RvIFVuaXZlcnNpdHksIEtpdGEtc2hpcmFrYXdhLCBPaXdha2UtY2hvLCBLeW90byA2MDYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYW5hZGEsIFRvc2hpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFwcGxlIFJlc2VhcmNoIERpdmlzaW9uLCBOQVJPIEluc3RpdHV0ZSBvZiBGcnVpdCBUcmVlIFNjaWVuY2UsIE1vcmlva2EgMDIwLTAxMjMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFlZ2FraSwgSGlkZWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJyZWVkaW5nIGFuZCBQZXN0IE1hbmFnZW1lbnQgRGl2aXNpb24sIE5BUk8gSW5zdGl0dXRlLCBUc3VrdWJhLCBJYmFyYWdpIDMwNS04NjA1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdyYWR6aWVsLCBUaG9tYXMgTS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIENhbGlmb3JuaWFEYXZpcywgQ0EgOTU2MTYsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbywgUnl1dGFybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUG9tb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgS3lvdG8gVW5pdmVyc2l0eSwgS2l0YS1zaGlyYWthd2EsIE9pd2FrZS1jaG8sIEt5b3RvIDYwNi04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lLXdpZGUgdmlldyBvZiBnZW5ldGljIGRpdmVyc2l0eSByZXZlYWxzIHBhdGhzIG9mIHNlbGVjdGlvbiBhbmQgY3VsdGl2YXIgZGlmZmVyZW50aWF0aW9uIGluIHBlYWNoIGRvbWVzdGljYXRpb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI4MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG9jdW1lbnRpbmcgZG9tZXN0aWNhdGlvbjogdGhlIGludGVyc2VjdGlvbiBvZiBnZW5ldGljcyBhbmQgYXJjaGFlb2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xMzktNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtb2xlY3VsYXIgZ2VuZXRpY3Mgb2YgY3JvcCBkb21lc3RpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEyNzsgIGNpdGF0aW9uX3BhZ2VzPTEzMDktMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHBhdHRlcm5zIG9mIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGluIGRvbWVzdGljYXRlZCByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTE2MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5ldGljIGV4cGVjdGF0aW9ucyBvZiBhIHByb3RyYWN0ZWQgbW9kZWwgZm9yIHRoZSBvcmlnaW5zIG9mIGRvbWVzdGljYXRlZCBjcm9wczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0xMzk4Mi04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBwZXJzcGVjdGl2ZXMgb24gY3JvcCBkb21lc3RpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTUyOS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBzY3JlZW5pbmcgZm9yIGFydGlmaWNpYWwgc2VsZWN0aW9uIGR1cmluZyBkb21lc3RpY2F0aW9uIGFuZCBpbXByb3ZlbWVudCBpbiBtYWl6ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTk2Ny03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjZW50IGFuZCBvbmdvaW5nIHNlbGVjdGlvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz04NTctNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFydGlmaWNpYWwgc2VsZWN0aW9uIGZvciBhIGdyZWVuIHJldm9sdXRpb24gZ2VuZSBkdXJpbmcgamFwb25pY2EgcmljZSBkb21lc3RpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTExMDM0LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNlcXVlbmNpbmcgNTAgYWNjZXNzaW9ucyBvZiBjdWx0aXZhdGVkIGFuZCB3aWxkIHJpY2UgeWllbGRzIG1hcmtlcnMgZm9yIGlkZW50aWZ5aW5nIGFncm9ub21pY2FsbHkgaW1wb3J0YW50IGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2guOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBjb21wYXJhdGl2ZSBkaXZlcnNpdHkgdW5jb3ZlcnMgbXVsdGlwbGUgdGFyZ2V0cyBvZiBzZWxlY3Rpb24gZm9yIGltcHJvdmVtZW50IGluIGhleGFwbG9pZCB3aGVhdCBsYW5kcmFjZXMgYW5kIGN1bHRpdmFyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz04MDU3LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXR0ZXJuIG9mIGRpdmVyc2l0eSBpbiB0aGUgZ2Vub21pYyByZWdpb24gbmVhciB0aGUgbWFpemUgZG9tZXN0aWNhdGlvbiBnZW5lIHRiMTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz03MDAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZy1yYW5nZSBwYXR0ZXJucyBvZiBkaXZlcnNpdHkgYW5kIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gc3Vycm91bmRpbmcgdGhlIG1haXplIFkxIGdlbmUgYXJlIGluZGljYXRpdmUgb2YgYW4gYXN5bW1ldHJpYyBzZWxlY3RpdmUgc3dlZXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9OTg4NS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmVjdHMgb2YgYXJ0aWZpY2lhbCBzZWxlY3Rpb24gb24gdGhlIG1haXplIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDg7ICBjaXRhdGlvbl9wYWdlcz0xMzEwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbGFyZ2Utc2NhbGUgc2NyZWVuIGZvciBhcnRpZmljaWFsIHNlbGVjdGlvbiBpbiBtYWl6ZSBpZGVudGlmaWVzIGNhbmRpZGF0ZSBhZ3Jvbm9taWMgbG9jaSBmb3IgZG9tZXN0aWNhdGlvbiBhbmQgY3JvcCBpbXByb3ZlbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTI4NTktNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3AgZ2Vub21pY3M6IGFkdmFuY2VzIGFuZCBhcHBsaWNhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz04NS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBzZWxlY3Rpb24gZm9yIGNyb3AgaW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWRpZXMgZm9yIGNvbXBsZXggdHJhaXRzOiBjb25zZW5zdXMsIHVuY2VydGFpbnR5IGFuZCBjaGFsbGVuZ2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBzdHJ1Y3R1cmUgYW5kIGRvbWVzdGljYXRpb24gaGlzdG9yeSBvZiB0aGUgZ3JhcGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MzUzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFjaW5nIHRoZSBnZW9ncmFwaGljIG9yaWdpbnMgb2YgbWFqb3IgYXZvY2FkbyBjdWx0aXZhcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEhlcmVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz01Ni42NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvbWVzdGljYXRpb24gb2YgYSBNZXNvYW1lcmljYW4gY3VsdGl2YXRlZCBmcnVpdCB0cmVlLCBTcG9uZGlhcyBwdXJwdXJlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xMjgwMS0wNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJvbSBmb3Jlc3QgdG8gZmllbGQ6IHBlcmVubmlhbCBmcnVpdCBjcm9wcyBkb21lc3RpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTEzODktNDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgaW5zaWdodCBpbnRvIHRoZSBoaXN0b3J5IG9mIGRvbWVzdGljYXRlZCBhcHBsZTogc2Vjb25kYXJ5IGNvbnRyaWJ1dGlvbiBvZiB0aGUgRXVyb3BlYW4gd2lsZCBhcHBsZSB0byB0aGUgZ2Vub21lIG9mIGN1bHRpdmF0ZWQgdmFyaWV0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI3MDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3RzIG9mIGNvbG9uaXphdGlvbiBwcm9jZXNzZXMgb24gZ2VuZXRpYyBkaXZlcnNpdHk6IGRpZmZlcmVuY2VzIGJldHdlZW4gYW5udWFsIHBsYW50cyBhbmQgdHJlZSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTQ7ICBjaXRhdGlvbl9wYWdlcz0xMzA5LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb21lIGV2b2x1dGlvbmFyeSBjb25zZXF1ZW5jZXMgb2YgYmVpbmcgYSB0cmVlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEVjb2wuIEV2b2wuIFN5c3QuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTg3LTIxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBkaXZlcnNpdHkgaW4gZm9yZXN0IHRyZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE2Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBhcmF0aXZlIHZpZXcgb2YgdGhlIGV2b2x1dGlvbiBvZiBncmFzc2VzIHVuZGVyIGRvbWVzdGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MzsgIGNpdGF0aW9uX3BhZ2VzPTI3My05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZi1pbmNvbXBhdGliaWxpdHkgaW4gc2V4dWFsbHkgcHJvcGFnYXRlZCBjdWx0aXZhdGVkIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0xOTY0OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTU3LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXksIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9T3JpZ2luIGFuZCBkaXNzZW1pbmF0aW9uIG9mIHBlYWNoLCBIb3J0LiBSZXYuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzMxLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q0FCIEludGVybmF0aW9uYWwsIDsgIGNpdGF0aW9uX3RpdGxlPUhpc3Rvcnkgb2YgY3VsdGl2YXRpb24gYW5kIHRyZW5kcyBpbiBDaGluYSwgdGhlIHBlYWNoOiBib3RhbnkgcHJvZHVjdGlvbiBhbmQgdXNlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3BhZ2VzPTM3LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBwb3B1bGF0aW9uIGdlbm9taWNzIHJldmVhbHMgdGhlIGRvbWVzdGljYXRpb24gaGlzdG9yeSBvZiB0aGUgcGVhY2gsIFBydW51cyBwZXJzaWNhLCBhbmQgaHVtYW4gaW5mbHVlbmNlcyBvbiBwZXJlbm5pYWwgZnJ1aXQgY3JvcHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTQxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluYnJlZWRpbmcgYW5kIGNvYW5jZXN0cnkgb2YgZnJlZXN0b25lIHBlYWNoIGN1bHRpdmFycyBvZiB0aGUgZWFzdGVybiBVbml0ZWQgU3RhdGVzIGFuZCBpbXBsaWNhdGlvbnMgZm9yIHBlYWNoIGdlcm1wbGFzbSBpbXByb3ZlbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW1lci4gU29jLiBIb3J0LiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTU0Ny01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgaW4gcGVhY2ggKFBydW51cyBwZXJzaWNhIEwuIEJhdGNoKSByZXZlYWxlZCBieSByYW5kb21seSBhbXBsaWZpZWQgcG9seW1vcnBoaWMgRE5BIChSQVBEKSBtYXJrZXJzIGFuZCBjb21wYXJlZCB0byBpbmJyZWVkaW5nIGNvZWZmaWNpZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW0uIFNvYy4gSG9ydGljLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyB2YXJpYXRpb24sIHBvcHVsYXRpb24gc3RydWN0dXJlIGFuZCBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGluIHBlYWNoIGNvbW1lcmNpYWwgdmFyaWV0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGNoYXJhY3Rlcml6YXRpb24gb2YgZGlmZmVyZW50IGRlbWVzIGluIFBydW51cyBwZXJzaWNhIHJldmVhbGVkIGJ5IFJBUEQgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBIb3J0aWMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTI0Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TU1IgZmluZ2VycHJpbnRpbmcgQ2hpbmVzZSBwZWFjaCBjdWx0aXZhcnMgYW5kIGxhbmRyYWNlcyAoUHJ1bnVzIHBlcnNpY2EpIGFuZCBhbmFseXNpcyBvZiB0aGVpciBnZW5ldGljIHJlbGF0aW9uc2hpcHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gSG9ydGljLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz0xODgtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gb2YgdGhlIGdlbmV0aWMgZGl2ZXJzaXR5IG9mIEFzaWFuIHBlYWNoIGFjY2Vzc2lvbnMgdXNpbmcgYSBzZWxlY3RlZCBzZXQgb2YgU1NSIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gSG9ydGljLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz02MjItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVsYXRpb25zaGlwcyBhbW9uZyBwZWFjaCwgYWxtb25kLCBhbmQgcmVsYXRlZCBzcGVjaWVzIGFzIGRldGVjdGVkIGJ5IHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbS4gU29jLiBIb3J0aWMuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTI4OyAgY2l0YXRpb25fcGFnZXM9NjY3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHJlbGF0aW9uc2hpcCBvZiBvcm5hbWVudGFsIHBlYWNoIGRldGVybWluZWQgdXNpbmcgQUZMUCBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Ib3J0LiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTc4Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgcmVzZXF1ZW5jaW5nIHJldmVhbHMgbG9jaSB1bmRlciBzZWxlY3Rpb24gZHVyaW5nIGNoaWNrZW4gZG9tZXN0aWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NDsgIGNpdGF0aW9uX3BhZ2VzPTU4Ny05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub25wYXJhbWV0cmljIHRlc3QgcmV2ZWFscyBzZWxlY3Rpb24gZm9yIHJhcGlkIGZsb3dlcmluZyBpbiB0aGUgQXJhYmlkb3BzaXMgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcG9wdWxhdGlvbiBnZW5vbWljcyBvZiBtYWl6ZSBkb21lc3RpY2F0aW9uIGFuZCBpbXByb3ZlbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz04MDgtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gZ2Vub21pYyBhbmQgZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllcyBvZiBhZ3JvY2xpbWF0aWMgdHJhaXRzIGluIHNvcmdodW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9NDUzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBldmFsdWF0aW9uIG9mIGEgOUsgU05QIGFycmF5IGZvciBwZWFjaCBieSBpbnRlcm5hdGlvbmFsbHkgY29vcmRpbmF0ZWQgU05QIGRldGVjdGlvbiBhbmQgdmFsaWRhdGlvbiBpbiBicmVlZGluZyBnZXJtcGxhc207ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBoaWdoLXF1YWxpdHkgZHJhZnQgZ2Vub21lIG9mIHBlYWNoIChQcnVudXMgcGVyc2ljYSkgaWRlbnRpZmllcyB1bmlxdWUgcGF0dGVybnMgb2YgZ2VuZXRpYyBkaXZlcnNpdHksIGRvbWVzdGljYXRpb24gYW5kIGdlbm9tZSBldm9sdXRpb247IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIFBlYWNoIEdlbm9tZSBJbml0aWF0aXZlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ny05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UExJTks6IGEgdG9vbCBzZXQgZm9yIHdob2xlLWdlbm9tZSBhc3NvY2lhdGlvbiBhbmQgcG9wdWxhdGlvbi1iYXNlZCBsaW5rYWdlIGFuYWx5c2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz01NTktNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdFTkVQT1AgKHZlcnNpb24gMS4yKTogcG9wdWxhdGlvbiBnZW5ldGljcyBzb2Z0d2FyZSBmb3IgZXhhY3QgdGVzdHMgYW5kIGVjdW1lbmljaXNtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz0yNDgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXBvcCcwMDc6IGEgY29tcGxldGUgcmVpbXBsZW1lbnRhdGlvbiBvZiB0aGUgR2VuZXBvcCBzb2Z0d2FyZSBmb3IgV2luZG93cyBhbmQgTGludXg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gUmVzb3VyY2VzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMDMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mZXJlbmNlIG9mIHBvcHVsYXRpb24gc3RydWN0dXJlIHVzaW5nIG11bHRpbG9jdXMgZ2Vub3R5cGUgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU1OyAgY2l0YXRpb25fcGFnZXM9OTQ1LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBNQ01DIGFwcHJvYWNoIGZvciBqb2ludCBpbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgYW5kIGluYnJlZWRpbmcgcmF0ZXMgZnJvbSBtdWx0aS1sb2N1cyBnZW5vdHlwZSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY7ICBjaXRhdGlvbl9wYWdlcz0xNjM1LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3RpbmcgdGhlIG51bWJlciBvZiBjbHVzdGVycyBvZiBpbmRpdmlkdWFscyB1c2luZyB0aGUgc29mdHdhcmUgU1RSVUNUVVJFOiBhIHNpbXVsYXRpb24gc3R1ZHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNjExLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZhc3QgYW5kIGZsZXhpYmxlIHN0YXRpc3RpY2FsIG1vZGVsIGZvciBsYXJnZS1zY2FsZSBwb3B1bGF0aW9uIGdlbm90eXBlIGRhdGE6IGFwcGxpY2F0aW9ucyB0byBpbmZlcnJpbmcgbWlzc2luZyBnZW5vdHlwZXMgYW5kIGhhcGxvdHlwaWMgcGhhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTYyOS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIHJlY2VudCBwb3NpdGl2ZSBzZWxlY3Rpb24gaW4gdGhlIGh1bWFuIGdlbm9tZSBmcm9tIGhhcGxvdHlwZSBzdHJ1Y3R1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTk7ICBjaXRhdGlvbl9wYWdlcz04MzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZGV0ZWN0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHBvc2l0aXZlIHNlbGVjdGlvbiBpbiBodW1hbiBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0OTsgIGNpdGF0aW9uX3BhZ2VzPTkxMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1hcCBvZiByZWNlbnQgcG9zaXRpdmUgc2VsZWN0aW9uIGluIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU3MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgWS1jaHJvbW9zb21l4oCTZW5jb2RlZCBzbWFsbCBSTkEgYWN0cyBhcyBhIHNleCBkZXRlcm1pbmFudCBpbiBwZXJzaW1tb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0NjsgIGNpdGF0aW9uX3BhZ2VzPTY0Ni01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzZXF1ZW5jZSBhbGlnbm1lbnQvbWFwIGZvcm1hdCBhbmQgU0FNdG9vbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbmFTUCB2NTogYSBzb2Z0d2FyZSBmb3IgY29tcHJlaGVuc2l2ZSBhbmFseXNpcyBvZiBETkEgcG9seW1vcnBoaXNtIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTQ1MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWFWaWV3IHZlcnNpb24gNDogYSBtdWx0aXBsYXRmb3JtIGdyYXBoaWNhbCB1c2VyIGludGVyZmFjZSBmb3Igc2VxdWVuY2UgYWxpZ25tZW50IGFuZCBwaHlsb2dlbmV0aWMgdHJlZSBidWlsZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIyMS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXh0ZW50IG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc3OyAgY2l0YXRpb25fcGFnZXM9MjIyMy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjaGFyYWN0ZXJpemF0aW9uIGFuZCBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGVzdGltYXRpb24gb2YgYSBnbG9iYWwgbWFpemUgY29sbGVjdGlvbiB1c2luZyBTTlAgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU4NDUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRhcHRhdGlvbiB0byBjbGltYXRlIGFjcm9zcyB0aGUgQXJhYmlkb3BzaXMgdGhhbGlhbmEgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTMzNDsgIGNpdGF0aW9uX3BhZ2VzPTgzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYWNraW5nIGZvb3RwcmludHMgb2YgYXJ0aWZpY2lhbCBzZWxlY3Rpb24gaW4gdGhlIGRvZyBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MTE2MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QRVRBTExPU1MgZ2VuZSByZWd1bGF0ZXMgaW5pdGlhdGlvbiBhbmQgb3JpZW50YXRpb24gb2Ygc2Vjb25kIHdob3JsIG9yZ2FucyBpbiB0aGUgQXJhYmlkb3BzaXMgZmxvd2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXZlbG9wbWVudDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz01NjM1LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QRVRBTExPU1MsIGEgdHJpaGVsaXggdHJhbnNjcmlwdGlvbiBmYWN0b3IgZ2VuZSwgcmVndWxhdGVzIHBlcmlhbnRoIGFyY2hpdGVjdHVyZSBpbiB0aGUgQXJhYmlkb3BzaXMgZmxvd2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXZlbG9wbWVudDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz00MDM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QRVRBTExPU1MgaXMgYSBib3VuZGFyeSBnZW5lIHRoYXQgaW5oaWJpdHMgZ3Jvd3RoIGJldHdlZW4gZGV2ZWxvcGluZyBzZXBhbHMgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9NzI0LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBvZiBhIHNlY29uZGFyeSBncmFwZXZpbmUgZG9tZXN0aWNhdGlvbiBjZW50cmUgZGV0ZWN0ZWQgYnkgU1NSIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTEzMTUtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBhbmFseXNpcyBvZiBxdWFsaXR5IHRyYWl0cyBpbiBhbiBhZHZhbmNlZCBiYWNrY3Jvc3MgYmV0d2VlbiBQcnVudXMgcGVyc2ljYSBjdWx0aXZhcnMgYW5kIHRoZSB3aWxkIHJlbGF0aXZlIHNwZWNpZXMsIFAuIGRhdmlkaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz04ODQtOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgYW5kIHRyYW5zY3JpcHRpb25hbCBhbmFseXNpcyBvZiB0aGUgc2VsZi1pbmNvbXBhdGliaWxpdHkgbG9jdXMgb2YgYWxtb25kOiBpZGVudGlmaWNhdGlvbiBvZiBhIHBvbGxlbi1leHByZXNzZWQgRi1ib3ggZ2VuZSB3aXRoIGhhcGxvdHlwZS1zcGVjaWZpYyBwb2x5bW9ycGhpc207ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz03NzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTLVJOYXNlLWJhc2VkIGdhbWV0b3BoeXRpYyBzZWxmLWluY29tcGF0aWJpbGl0eSBzeXN0ZW0gaW4gUHJ1bnVzIGV4aGliaXRzIGRpc3RpbmN0IGdlbmV0aWMgYW5kIG1vbGVjdWxhciBmZWF0dXJlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBIb3J0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjQ7ICBjaXRhdGlvbl9wYWdlcz00MjMtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGYtY29tcGF0aWJsZSBwZWFjaCAoUHJ1bnVzIHBlcnNpY2EpIGhhcyBtdXRhbnQgdmVyc2lvbnMgb2YgdGhlIFMgaGFwbG90eXBlcyBmb3VuZCBpbiBzZWxmLWluY29tcGF0aWJsZSBQcnVudXMgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTEwOS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIG5vdmVsIHNlbGYtY29tcGF0aWJsZSBTIGhhcGxvdHlwZXMgaW4gcGVhY2ggKFBydW51cyBwZXJzaWNhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSnBuIFNvYy4gSG9ydGljLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9MjAxLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBSTmFzZXMgb2Ygd2lsZCBhbG1vbmQgKFBydW51cyB3ZWJiaWkpOiBjbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNpeCBuZXcgUy1STmFzZSBhbmQgb25lIOKAnG5vbi1TIFJOYXNl4oCdIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0zOTUtNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhYm9saWMgZW5naW5lZXJpbmcgb2YgY2Fyb3Rlbm9pZCBiaW9zeW50aGVzaXMgaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMzktNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV4cGFuZGVkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgUHJ1bnVzIGJhc2VkIG9uIGFuIGludGVyc3BlY2lmaWMgY3Jvc3MgYmV0d2VlbiBhbG1vbmQgYW5kIHBlYWNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01MjAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXRpbGl0eSBvZiBSQVBEIG1hcmtlcnMgaW4gaWRlbnRpZnlpbmcgZ2VuZXRpYyBsaW5rYWdlcyB0byBnZW5lcyBvZiBlY29ub21pYyBpbnRlcmVzdCBpbiBwZWFjaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTkyMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVEwgYW5hbHlzaXMgb2YgZnJ1aXQgcXVhbGl0eSB0cmFpdHMgaW4gdHdvIHBlYWNoIGludHJhc3BlY2lmaWMgcG9wdWxhdGlvbnMgYW5kIGltcG9ydGFuY2Ugb2YgbWF0dXJpdHkgZGF0ZSBwbGVpb3Ryb3BpYyBlZmZlY3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTMyMy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJhYmlkb3BzaXMgdGhhbGlhbmEgQXRIQUwzOiBhIGZsYXZvcHJvdGVpbiByZWxhdGVkIHRvIHNhbHQgYW5kIG9zbW90aWMgdG9sZXJhbmNlIGFuZCBwbGFudCBncm93dGg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NTI5LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSByb2xlcyBvZiBBcmFiaWRvcHNpcyBWUk4xIGluIHZlcm5hbGl6YXRpb24gYW5kIGZsb3dlcmluZyB0aW1lIGNvbnRyb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk3OyAgY2l0YXRpb25fcGFnZXM9MjQzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy8zLzI3MS8xNzQ2MzI4Ny9kc3cwMTQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIERvbWVzdGljYXRpb24gYW5kIGN1bHRpdmFyIGRpZmZlcmVudGlhdGlvbiBhcmUgcmVxdWlzaXRlIHByb2Nlc3NlcyBmb3IgZXN0YWJsaXNoaW5nIGN1bHRpdmF0ZWQgY3JvcHMuIFRoZXNlIHByb2Nlc3NlcyBpbmhlcmVudGx5IGludm9sdmUgc3Vic3RhbnRpYWwgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0xNzQ1NjE3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZGA8gRNBl50NnYQhzBZ8zn/mQ/W9" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f271%2f1745617" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f271%2f1745617" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69063" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&amp;Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/3">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200498">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200498">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200500">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200500">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200505">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200505">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200524">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200524">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200571">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200571">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200585">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200585">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200589">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200589">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200591">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200591">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/3/263/1745725">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/3/283/1745776">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Akagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Pomology, Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kita-shirakawa, Oiwake-cho, Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 75-753-6051. E-mail: <a href="mailto:takashia@kais.kyoto-u.ac.jp" target="_blank">takashia@kais.kyoto-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Akagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Akagi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Akagi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Akagi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshio Hanada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Apple Research Division</div>, <div class="insititution">NARO Institute of Fruit Tree Science</div>, <div class="addr-line">Morioka 020-0123</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshio+Hanada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hanada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hanada T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshio Hanada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideaki Yaegaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Breeding and Pest Management Division, NARO Institute</div>, <div class="addr-line">Tsukuba, Ibaragi 305-8605</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideaki+Yaegaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yaegaki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yaegaki H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideaki Yaegaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thomas M. Gradziel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Plant Sciences</div>, <div class="insititution">University of California</div><div class="addr-line">Davis, CA 95616</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thomas+M.+Gradziel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gradziel T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gradziel T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Thomas M. Gradziel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryutaro Tao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Pomology, Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kita-shirakawa, Oiwake-cho, Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryutaro+Tao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tao R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tao R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ryutaro Tao</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 271–282, <a href='https://doi.org/10.1093/dnares/dsw014'>https://doi.org/10.1093/dnares/dsw014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 April 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 November 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 March 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745617" href="/dnaresearch/article-pdf/23/3/271/17463287/dsw014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takashi Akagi, Toshio Hanada, Hideaki Yaegaki, Thomas M. Gradziel, Ryutaro Tao;  Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication, <em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 271–282, <a href='https://doi.org/10.1093/dnares/dsw014'>https://doi.org/10.1093/dnares/dsw014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745617?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745617?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745617?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745617?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745617?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745617?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745617?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20view%20of%20genetic%20diversity%20reveals%20paths%20of%20selection%20and%20cultivar%20differentiation%20in%20peach%20domestication&amp;publicationDate=2016-04-15&amp;volumeNum=23&amp;issueNum=3&amp;author=Akagi%2C%20Takashi%3B%20Hanada%2C%20Toshio&amp;startPage=271&amp;endPage=282&amp;contentId=10.1093%2Fdnares%2Fdsw014&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f271%2f1745617" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200498 id="87200498" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Domestication and cultivar differentiation are requisite processes for establishing cultivated crops. These processes inherently involve substantial changes in population structure, including those from artificial selection of key genes. In this study, accessions of peach (<em>Prunus persica</em>) and its wild relatives were analysed genome-wide to identify changes in genetic structures and gene selections associated with their differentiation. Analysis of genome-wide informative single-nucleotide polymorphism loci revealed distinct changes in genetic structures and delineations among domesticated peach and its wild relatives and among peach landraces and modern fruit (F) and modern ornamental (O-A) cultivars. Indications of distinct changes in linkage disequilibrium extension/decay and of strong population bottlenecks or inbreeding were identified. Site frequency spectrum- and extended haplotype homozygosity-based evaluation of genome-wide genetic diversities supported selective sweeps distinguishing the domesticated peach from its wild relatives and each F/O-A cluster from the landrace clusters. The regions with strong selective sweeps harboured promising candidates for genes subjected to selection. Further sequence-based evaluation further defined the candidates and revealed their characteristics. All results suggest opportunities for identifying critical genes associated with each differentiation by analysing genome-wide genetic diversity in currently established populations. This approach obviates the special development of genetic populations, which is particularly difficult for long-lived tree crops.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;artificial selection&quot;">artificial selection</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cultivar differentiation&quot;">cultivar differentiation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="domestication">domestication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage disequilibrium&quot;">linkage disequilibrium</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tree crop&quot;">tree crop</a></div>                            <h2 scrollto-destination=87200500 id="87200500" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Plant domestication establishes a vital co-dependence between humans and plants.<sup><a reveal-id="DSW014C1" data-open="DSW014C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Previous studies of the plant domestication process have focused mainly on seed-propagated annual (herbaceous) species, often with the goal of identifying wild progenitors, as well as changes in genetic structures associated with domestication.<sup><a reveal-id="DSW014C2  DSW014C5" data-open="DSW014C2  DSW014C5" class="link link-ref link-reveal xref-bibr">2–5</a></sup> The results have provided historical insights into the genetic adaptation required for domestication, which has relevance to crop breeding strategies. During the domestication and early breeding process, crops typically experience population bottlenecks, including extensive artificial selection for improved crop quality and local adaptation.<sup><a reveal-id="DSW014C6" data-open="DSW014C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Evidence of this selection remains in the patterns of genetic diversity within cultivated genomes,<sup><a reveal-id="DSW014C7" data-open="DSW014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> as characterized for major crops such as rice,<sup><a reveal-id="DSW014C8" data-open="DSW014C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSW014C9" data-open="DSW014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> wheat,<sup><a reveal-id="DSW014C10" data-open="DSW014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and maize.<sup><a reveal-id="DSW014C11  DSW014C14" data-open="DSW014C11  DSW014C14" class="link link-ref link-reveal xref-bibr">11–14</a></sup> These studies also identified important genomic regions and/or genes associated with artificial selection. Genes under strong selection reflect the main driving forces for each historical differentiation. Thus, their detection can provide valuable targets for further crop improvement research,<sup><a reveal-id="DSW014C15" data-open="DSW014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> as well as opportunities to improve current genomic selection (GS) models.<sup><a reveal-id="DSW014C16" data-open="DSW014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Conventional mapping approaches such as genotype–phenotype association mapping, though effective for identifying loci contributing to specific traits, are not as applicable to diversified populations with their large variety of genetic structures.<sup><a reveal-id="DSW014C15" data-open="DSW014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> For approaches using linkage mapping, perennial plants present additional challenges because of their large size and long generation time. Thus, genome-wide association study (GWAS)<sup><a reveal-id="DSW014C17" data-open="DSW014C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> used in conjunction with analysis of the effect of selection on the genomes of current populations may be a good strategy for identifying critical loci defining perennial crops.</p><p>In contrast to those for annual crops, studies depicting paths of domestication for perennial plants, including tree crops, are rare.<sup><a reveal-id="DSW014C18  DSW014C22" data-open="DSW014C18  DSW014C22" class="link link-ref link-reveal xref-bibr">18–22</a></sup> Trees have unique features affecting their manner and rate of domestication: (i) individual genotypes can be maintained by vegetative or clonal propagation, (ii) trees have a long generation time, frequently with a long juvenile phase, and (iii) wild populations are often large with high levels of gene flow.<sup><a reveal-id="DSW014C23  DSW014C25" data-open="DSW014C23  DSW014C25" class="link link-ref link-reveal xref-bibr">23–25</a></sup> Additionally, different reproductive systems and cultural practices among species may have led to substantially different effects on domestication and breeding pathways. A typical example for reproductive systems is self-incompatibility (SI), which promotes outcrossing. Outcrossing generally disturbs the fixation of selected alleles and the resulting domestication.<sup><a reveal-id="DSW014C26" data-open="DSW014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Rowlands hypothesized that important crops undergo breakdown of SI during domestication and/or breeding, owing to artificial selection for stable production in more homozygous lines.<sup><a reveal-id="DSW014C27" data-open="DSW014C27" class="link link-ref link-reveal xref-bibr">27</a></sup> It has been proposed<sup><a reveal-id="DSW014C22" data-open="DSW014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> that the domesticated apple did not experience a population bottleneck in the domestication step, presumably because of its long generation time, the nature of the SI system, and breeding practices such as selection from open-pollinated seed (leading to ‘chance seedlings’). Similarly, domesticated grapevine shows only a weak bottleneck, partially because of specific cultural practices including the maintenance of many old varieties with wide genetic diversity.<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> It has been proposed that tree crops are, in general, less affected by population bottlenecks and other selection limitations during the domestication process.<sup><a reveal-id="DSW014C21" data-open="DSW014C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSW014C22" data-open="DSW014C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>Peach [<em>Prunus persica</em> (L.) Batsch] is one of the most important tree crops worldwide. It is in the Rosaceae family, which also includes apricot, almond, plum, strawberry, rose, pear, and apple, and has been reported to have been cultivated for &gt;4,000 yrs.<sup><a reveal-id="DSW014C28" data-open="DSW014C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Archaeological findings support the domestication of peach and its artificial selection for various desirable traits in China from as early as 1000 BC,<sup><a reveal-id="DSW014C29" data-open="DSW014C29" class="link link-ref link-reveal xref-bibr">29</a></sup> although there are only a few reports on specific origins or required traits for domestication.<sup><a reveal-id="DSW014C30" data-open="DSW014C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Modern varieties of cultivated peach are conventionally categorized into two main groups: fruit varieties and flower-ornamental varieties, though this grouping does not consider genetic structure. For fruit varieties, many regional cultivars exist. However, most modern cultivars in Europe and North America appear to have originated from only a few old cultivars used in early twentieth-century North American breeding programs.<sup><a reveal-id="DSW014C28" data-open="DSW014C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSW014C31" data-open="DSW014C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Based on molecular marker data, cv. Chinese Cling has been proposed as an important founding parent in these early breeding programs.<sup><a reveal-id="DSW014C32" data-open="DSW014C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSW014C33" data-open="DSW014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In Asia, cv. Chinese Cling has also been proposed to be a founding parent of numerous modern cultivars,<sup><a reveal-id="DSW014C34  DSW014C36" data-open="DSW014C34  DSW014C36" class="link link-ref link-reveal xref-bibr">34–36</a></sup> though it should be mentioned that this designation may refer to multiple independent landraces and local selections.<sup><a reveal-id="DSW014C37" data-open="DSW014C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The exact identities of many old cultivars and classification names reported in the older literature remain confusing given that they have been cultivated in China for thousands of years, often with multiple introductions to Western countries.<sup><a reveal-id="DSW014C38" data-open="DSW014C38" class="link link-ref link-reveal xref-bibr">38</a></sup> For the flower-ornamental peach group, little is known about the genetic structure and genetic diversity, except for some limited cultivars and landraces.<sup><a reveal-id="DSW014C30" data-open="DSW014C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>Artificial selection is expected to occur not only during domestication but also in subsequent breeding for specific objectives. In domesticated chicken, genome-wide comparison of genetic diversity during recent breeding differentiation to layers or broilers revealed multiple independent targets of artificial selection.<sup><a reveal-id="DSW014C39" data-open="DSW014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The recent availability of genome-wide scans of plant species’ DNA polymorphisms allows for a more detailed study of their population genetics, including characterization of linkage disequilibrium (LD) as well as genetic diversity, including pi-values for the detection of targets of selection during plant domestication.<sup><a reveal-id="DSW014C9" data-open="DSW014C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW014C10" data-open="DSW014C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW014C30" data-open="DSW014C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSW014C40  DSW014C42" data-open="DSW014C40  DSW014C42" class="link link-ref link-reveal xref-bibr">40–42</a></sup> The objective of this study was to better characterize the genetic structure of peach and to identify important selection events during domestication and subsequent cultivar differentiation, using a genome-wide single-nucleotide polymorphism (SNP) chip-based approach.</p>                            <h2 scrollto-destination=87200505 id="87200505" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200506 id="87200506" class="section-title" data-legacy-id=s2a>2.1. Sample collection and genome-wide genotyping</h3>
<p>For genome-wide genotyping, leaves were collected from 67 accessions of domesticated peach (<em>P. persica</em>; subg. <em>Amygdalus</em>), 20 accessions of peach relatives (four <em>Prunus davidiana</em>, one <em>P. kansuensis</em>, 12 <em>P. mira</em>, one <em>P. tangutica</em>, and two <em>P. webbii</em>; subg. <em>Amygdalus</em>), and 8 accessions of outgroup species [two Japanese plum (<em>P. salicina</em>; subg. <em>Prunus</em>), two Japanese apricot (<em>P. mume</em>; subgen. <em>Prunus</em>), and four sweet cherry (<em>P. avium</em>; subg. <em>Cerasus</em>)], from the UC Davis and USDA <em>Prunus</em> germplasm collections in Winters, California, USA, the NIFTS <em>Prunus</em> germplasm collections, Tsukuba, Japan, the Research Institute for Agriculture Okayama Prefectural Technology Center for Agriculture, Forestry, and Fisheries, Akaiwa, Japan, and the experimental orchard of Kyoto University, Kyoto, Japan, as summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. Tissues from a total of 95 accessions were subjected to DNA extraction using Nucleon PhytoPure (GE Healthcare, Tokyo, Japan) and thereafter phenol/chloroform extraction.</p><p>Genotype calling was performed with an Illumina infinium peach 9K SNP chip, which was defined from a total of 1,022,354 SNPs that were identified from the resequencing data in a wide variety of 56 accessions in <em>Amygdalus</em>, mainly including peach (<em>P. persica</em>) and almond (<em>Prunus dulcis</em>), and ∼75% of genic SNPs were verified in the peach genome.<sup><a reveal-id="DSW014C43" data-open="DSW014C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSW014C44" data-open="DSW014C44" class="link link-ref link-reveal xref-bibr">44</a></sup> We assayed 7,873 SNPs in this study. Each 5,180 and 6,605 informative SNP was selected for structural analysis and evaluation of LD and selective sweeps, respectively (see below).</p>                            <h3 scrollto-destination=87200509 id="87200509" class="section-title" data-legacy-id=s2b>2.2. Allele pruning</h3>
<p>SNPs showing &gt;5% missing data or &lt;0.05 minor allele frequency (MAF) in domesticated peach cultivars were pruned with PLINK.<sup><a reveal-id="DSW014C45" data-open="DSW014C45" class="link link-ref link-reveal xref-bibr">45</a></sup> After filtering, 6,605 SNPs remained for use in estimating LD. SNP pairs showing strong LD were further pruned by defining a window of 50 SNPs, removing one of a pair of SNPs if <em>R</em><sup>2</sup> &gt; 0.5 (VIF threshold values = 2), and then shifting the window by three SNPs and repeating the procedure using PLINK. After filtering, 5,180 SNPs remained for use in the analyses of population structure.</p>                            <h3 scrollto-destination=87200511 id="87200511" class="section-title" data-legacy-id=s2c>2.3. Analysis of population structure</h3>
<p>For the topology of the evolutionary tree, we aligned 5,180 concatenated SNPs to give two aligned positions for each SNP locus to reflect the diploid allelic states. The aligned concatenated sequences were subjected to neighbour joining (NJ) using version 5.05 under the Poisson matrix with gamma distribution for the rates and with 1,000 bootstrap replications. Principal component analysis (PCA) was performed using the same 5,180 SNP set, using prcomp implemented in R version 2.15.3. Heterozygosity [expected (H<sub>E</sub>) and observed (H<sub>O</sub>)], inbreeding coefficient (F<sub>IS</sub>), and Weir &amp; Cockerham <em>F</em> statistics (F<sub>ST</sub>) were calculated with GENEPOP 4.0.<sup><a reveal-id="DSW014C46" data-open="DSW014C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSW014C47" data-open="DSW014C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Identity by descent (IBD) proportions were calculated as pi-hat values with PLINK, considering all pairings of the 67 accessions of domesticated peach (<em>P. persica</em>). We used the pruned 5,180 SNPs for the calculation. We had no calibration to infer first- or second-degree relationships from the information of actual pedigrees owing to the lack of information on reliable pedigrees and to the small number of accessions.</p><p>To evaluate delimitations in population structure, we performed individual-based Bayesian clustering with Markov chain Monte Carlo (MCMC) simulations, using STRUCTURE 2.2<sup><a reveal-id="DSW014C48" data-open="DSW014C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and InStruct<sup><a reveal-id="DSW014C49" data-open="DSW014C49" class="link link-ref link-reveal xref-bibr">49</a></sup> to infer the population ancestry of genotypes in <em>K</em> predefined clusters. <em>K</em> values ranging from 2 to 10 were evaluated for subdivision of the full (domesticated or wild) peach population (<em>n</em> = 87). We fixed <em>K</em> at 2 for characterizing proportions of ancestry from two predefined ancestral gene pools in some combinations of the varietal complexes or of the resulting clusters, to support population subdivision, and to infer genetic introgression among the complexes according to Cornille et al.<sup><a reveal-id="DSW014C22" data-open="DSW014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> We performed an independent test of each <em>K</em> using at least 200,000 MCMC iterations after 50,000 burn-in iterations. To evaluate inference of <em>K</em>, the model with the highest ln Pr(<em>K</em>)<sup><a reveal-id="DSW014C48" data-open="DSW014C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and the Δ<em>K</em> model with the greatest second-order rate of change in ln Pr(<em>K</em>)<sup><a reveal-id="DSW014C50" data-open="DSW014C50" class="link link-ref link-reveal xref-bibr">50</a></sup> were examined.</p><p>Pairwise LD among the SNPs were calculated with PLINK, using 6,605 SNP sets considering MAF (&lt;0.05) and missing reads (&gt;20%) in the full population as candidates. We exploited all <em>R</em><sup>2</sup> values in all pairs of SNPs in 10,000-kb windows. The <em>R</em><sup>2</sup> values were independently calculated for each subpopulation for each chromosome. The LD decay distances in each domesticated peach subpopulation, in the whole domesticated peach population, and in a wild related population were defined as the first points at which the average <em>R</em><sup>2</sup> values in a 100-kb bin revealed no significant difference (<em>P</em> &gt; 0.1) against the background <em>R</em><sup>2</sup> values, which were calculated from average intra-chromosomal comparisons.</p>                            <h3 scrollto-destination=87200515 id="87200515" class="section-title" data-legacy-id=s2d>2.4. Haplotype phasing</h3>
<p>We characterized haplotypes of SNPs in each chromosome with fastPHASE v.1.2 using 6,605 SNP sets considering MAF and missing rates in the entire population.<sup><a reveal-id="DSW014C49" data-open="DSW014C49" class="link link-ref link-reveal xref-bibr">49</a></sup> In advance and against the full population, including 87 wild or domesticated accessions, we estimated imputation error rates using 1,583 SNPs on chromosome 4 and the following options in fastPHASE: number of random starts of the EM algorithm = 10, EM iterations = 25, lower limit numbers of clusters = 1, upper limit numbers of clusters = 20, and interval between values for number of clusters = 1. This analysis gave an imputation error rate of 0.0822 with <em>K</em> = 8 as an optimal condition, only slightly larger than the values reported for phasing of SNPs on grapevine chromosome 8 and in humans.<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSW014C51" data-open="DSW014C51" class="link link-ref link-reveal xref-bibr">51</a></sup> We adopted <em>K</em> = 8 and constructed haplotypes for each chromosome. Construction of haplotypes was also performed using each subpopulation defined in structure analysis (see Results section), which often showed a lower imputation error rate (<em>ca.</em> 0.06–0.17) than the full population.</p>                            <h3 scrollto-destination=87200517 id="87200517" class="section-title" data-legacy-id=s2e>2.5. Detection of selective sweeps</h3>
<div class="block-child-p">The 6,605 SNPs considering MAF and missing rates were used for three approaches, site frequency spectrum (SFS)-, integrated Haplotype Score (iHS)-, and XP-extended haplotype homozygosity (EHH)-based methods. The SFS-based approach was applied using the pooled heterozygosity (<em>H</em><sub>p</sub>)<sup><a reveal-id="DSW014C39" data-open="DSW014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> of each cluster in a 400-kb sliding window with a 100-kb step. Windows contained only four or fewer positions where SNPs were removed. All SNP alleles were completely given in bi-allelic states, and thus, the pooled heterozygosity was given by the average expected heterozygosity in each window, as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mi>H</mi><mi>p</mi></msub><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mn>1</mn><mi>S</mi></mfrac></mrow><msubsup><mo movablelimits="false">∑</mo><mrow><mi>m</mi><mo>=</mo><mn>1</mn></mrow><mi>S</mi></msubsup><mrow><mfenced close=")" open="("><mrow><mstyle><mrow><mfrac><mrow><mn>2</mn><mi>N</mi><msub><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow><mrow><mrow><mi mathvariant="normal">maj</mi></mrow></mrow></msub><mi>N</mi><msub><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow><mrow><mo movablelimits="true">min</mo></mrow></msub></mrow><mrow><msup><mrow><mo fence="false" stretchy="false">{</mo><mi>N</mi><msub><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow><mrow><mrow><mi mathvariant="normal">maj</mi></mrow></mrow></msub><mo>+</mo><mi>N</mi><msub><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow><mrow><mo movablelimits="true">min</mo></mrow></msub><mo fence="false" stretchy="false">}</mo></mrow><mn>2</mn></msup></mrow></mfrac></mrow></mstyle></mrow></mfenced></mrow></mstyle></math></div></div></div></div><p>Here, <em>S</em> is the number of SNP positions in a window, <em>N</em>(<em>m</em>)<sub>maj</sub> and <em>N</em>(<em>m</em>)<sub>min</sub> are the numbers of major and minor alleles, respectively, in <em>m</em>th SNP locus in a window. Individual <em>H</em><sub>p</sub> values were then <em>Z</em>-transformed as follows: <em>ZH</em><sub>p</sub> = (<em>H</em><sub>p</sub> − μ<em>H</em><sub>p</sub>)/σ<em>H</em><sub>p</sub>, according to a report on pooled heterozygosity for selective sweep analysis in chicken.<sup><a reveal-id="DSW014C39" data-open="DSW014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The allele information was used in an unphased state.</p><p>Both iHS and XP-EHH tests were based on EHH, which is defined as the probability that two randomly chosen chromosomes carrying the core haplotype of interest are identical by descent (as assayed by homozygosity at all SNPs) for the entire interval from the core region to point <em>x</em>.<sup><a reveal-id="DSW014C52" data-open="DSW014C52" class="link link-ref link-reveal xref-bibr">52</a></sup> We detected EHH in each subpopulation using the program Sweep.<sup><a reveal-id="DSW014C53" data-open="DSW014C53" class="link link-ref link-reveal xref-bibr">53</a></sup> We approached the computation of the integral of observed EHH (iHH)<sup><a reveal-id="DSW014C54" data-open="DSW014C54" class="link link-ref link-reveal xref-bibr">54</a></sup> in 1,000 kb from each SNP core by measuring EHH in every 100-kb bin. EHH decayed to below 0.05 by 1,000 kb from SNP cores in most cases.</p><p>For detection of iHS, iHH values from two core alleles at one SNP core position in one population were defined as iHH<sub>A</sub> and iHH<sub>D</sub>, which originally corresponded to ancestral and derived alleles, respectively.<sup><a reveal-id="DSW014C54" data-open="DSW014C54" class="link link-ref link-reveal xref-bibr">54</a></sup> iHS statistic is then given as unstandardized <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">iHS</mtext><mo xmlns="">=</mo><mi xmlns="">ln</mi><mo xmlns="">⁡</mo><mo stretchy="false" xmlns="">(</mo><mtext xmlns="">iH</mtext><msub xmlns=""><mtext>H</mtext><mrow><mi mathvariant="normal">A</mi></mrow></msub><mrow xmlns=""><mo>/</mo></mrow><mtext xmlns="">iH</mtext><msub xmlns=""><mtext>H</mtext><mrow><mi mathvariant="normal">D</mi></mrow></msub><mo stretchy="false" xmlns="">)</mo></math></span> (1).<sup><a reveal-id="DSW014C54" data-open="DSW014C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Finally, we obtained the standardized iHS as a <em>Z</em>-transformed value of the unstandardized iHS. We examined the reciprocal states of two core alleles (iHH<sub>A</sub> and iHH<sub>D</sub>) because of the sparseness of information on the derivation of the core alleles among peach subpopulations. XP-EHH is also obtained by Equation (1), focusing on the same core alleles in comparison to two populations.<sup><a reveal-id="DSW014C53" data-open="DSW014C53" class="link link-ref link-reveal xref-bibr">53</a></sup> For two populations, A and B, the log values of the integral EHH, <em>I</em><sub>A</sub> and <em>I</em><sub>B</sub> (like iHH<sub>A</sub> and iHH<sub>D</sub> in the iHS test), ln(<em>I</em><sub>A</sub>/<em>I</em><sub>B</sub>) give an index of selection specific to either of the two populations. An unusually positive value of ln(<em>I</em><sub>A</sub>/<em>I</em><sub>B</sub>) suggests selection in population A, whereas a negative value suggests selection in population B. The standardized XP-EHH is given as a <em>Z</em>-transformed value of ln (<em>I</em><sub>A</sub>/<em>I</em><sub>B</sub>), the details of which are presented by Sabeti.<sup><a reveal-id="DSW014C53" data-open="DSW014C53" class="link link-ref link-reveal xref-bibr">53</a></sup> The standardized iHS and XP-EHH were transformed to <em>P</em> values using R for graphical plotting.</p>                            <h3 scrollto-destination=87200522 id="87200522" class="section-title" data-legacy-id=s2f>2.6. Sequencing and genetic diversity analysis in selected regions</h3>
<p>To exploit genes under selection during cultivar differentiation, full lengths of a total of 15 genes located on the region under putative selection were amplified by PCR using 30 peach accessions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>a) and then sequenced with an Illumina HiSeq 2000 (Illumina) as paired-end 100 (PE100). The libraries were constructed based on in-house-developed protocols described previously.<sup><a reveal-id="DSW014C55" data-open="DSW014C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Approximately 4% of a sequencing lane was dedicated to all samples to yield at least 100× coverage in any region. Over 15% of read coverage for each SNP was considered as informative. All bioinformatic and statistical analyses were performed on local servers at the UC Davis Genome Center (Davis, CA, USA). Raw reads without adapter sequences were subjected to trimming (length &gt; 35 bp, mean sliding window of 5 bp phred quality score ≥20) using custom Python scripts. Reads were then mapped to the reference sequences from the peach genome (Peach v1.0) using the Burrows–Wheeler Aligner (BWA) tools.<sup><a reveal-id="DSW014C56" data-open="DSW014C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Informative SNPs were identified using Sequence Alignment/Map (SAM) tool<sup><a reveal-id="DSW014C57" data-open="DSW014C57" class="link link-ref link-reveal xref-bibr">57</a></sup> and custom Python scripts. Informative SNP strings and statistics for selective pressure (nucleotide diversity (π) and Tajima's <em>D</em>) were evaluated with DnaSP 5.1 using sequence alignments constructed with ClustalX version 2.0 with minor revision using SeaView version 4, according to previous studies.<sup><a reveal-id="DSW014C8" data-open="DSW014C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSW014C12" data-open="DSW014C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSW014C58" data-open="DSW014C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSW014C59" data-open="DSW014C59" class="link link-ref link-reveal xref-bibr">59</a></sup></p>                            <h2 scrollto-destination=87200524 id="87200524" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87200525 id="87200525" class="section-title" data-legacy-id=s3a>3.1. Genotyping and defining the genetic structure</h3>
<p>The topology of the evolutionary tree, constructed using 5,180 concatenated SNPs, showed distinct differentiation between peach and its wild relatives and also among the cultivar complexes, based on classical classifications by morphology or use (Fig. <a reveal-id="DSW014F1" data-open="DSW014F1" class="link link-reveal link-table xref-fig">1</a>). Six major clusters were defined in the evolutionary tree: wild species (W), landraces (L), modern fruit cultivars (F), and modern ornamental cultivars (O-A and O-B for only ornamental usage and FO for fruit and ornamentals). The F (including FO) and O-A clearly diverged from landraces with strong statistical support (with bootstrap values of 70/100 and 74/100, respectively). The L clade was divided into two clades, of which one contained mainly East Asian cultivars (L-EA), and the other (L-OT) showed no specificity for geographic distribution. Some ornamental cultivars (O-B) were included in clade L-EA. Note that two wild species (<em>P. davidiana</em> 2325-21A, and <em>P. mira</em> 2228-21A) were located near or in landrace clusters, probably because of recent frequent hybridization with domesticated peach cultivars or ancestral shared polymorphisms, as supported by PCA and STRUCTURE analyses (see below). </p>                        <a id="87200527" scrollto-destination="87200527"></a>
<div id="DSW014F1" data-id="DSW014F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01401.png?Expires=2147483647&Signature=jYy4u0yZt4Bu~9Og~szYwkgZ3pkfU8T~MRIoEVkKfIWd3qAyA35IriW8s-zJ6828FSoUm0yI6wgmvJYWSpbC4BdOVSbK8BRzrarc8XDQs98BiOLG-PZY8tzi2sQgiR03dxCmV2a9EU6lXa0RpU2kGIrzITv~le8Ol9kJRqxLXL1Y0DFv4MBE2FDJ2~9Ws2-WadK6Lf-OqQKN-IPkD6n8bfGdesPIy0BNymWzxecWZxwVId40H6xWLBgyK8I79GejEVoZ3eSwOwtRDpBomNolr53NSYKFV~8MbZftHngoO1Er326lrf6P7rCZsZC4cMpz~bgXSd9IGZ9aU38njicSbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary tree of domesticated peach and its wild relatives. Evolutionary tree constructed by a maximum likelihood (ML) approach using information from 5,180 genome-wide SNPs in 87 accessions of domesticated peach (Prunus persica) and five wild species (P. davidiana, P. mira, P. kansuensis, P. tangutica, and P. webbii), with 8 accessions of outgroup species (P. salicina, P. mume, and P. avium). Operational taxonomic units (OTU) are colored in ocher, black, green, blue, and red, for major clusters: wild species (W), landraces (L) (EA and OT for accessions in East Asia and other region, respectively), modern fruit cultivars (F), and modern ornamental cultivars (O-A and O-B for only ornamental usage, and FO for fruit and ornamentals), respectively, according to the tree topology, and to classical classifications considering uses and morphologies. The main differentiation steps of the domesticated peach, the East Asian cultivars (including modern cultivars), the O-A cluster, and the F cluster, are shown as black, green, red, and blue thick branches, respectively, with statistically significant support (bootstrap &gt; 650/1,000, except for the differentiation to the East Asian cultivars)." path-from-xml="dsw01401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200527/dsw01401.tiff" path-from-xml="dsw01401.tiff" target="_blank">View large</a><a section="87200527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01401.png?Expires=2147483647&Signature=0o5E0ynlHQbdYrJleb~8kSInCwwqMHkTXcOwJafvfqVYoD8DPc7wy02t0KbTig0DAavILbnueSomtGzK0FaX0xu6XxkaFmistPaQlRUiKfjLXXN8VHcH1GigTUVg-IK9brVFIkK2kz~dhgYw3M0xwQaW-yIjOFx0db4Jz8i-u4CSKOP2tKI1Fb18o5b2VeSPZSZSlBLQwdG88twStMB~S7Olnm6XcWfeEIorOc35otAC0L1U-F1cEqCmH9lwZkCrFIfZtlEX1Z5ZnqsegoNgsHCSIJ2uNlRKdLbMu4ZvCQmsK3NQIc84a-MVTqWAcmu9gGLsqVtJbZ0jTSmjh2VBgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200527&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary tree of domesticated peach and its wild relatives. Evolutionary tree constructed by a maximum likelihood (ML) approach using information from 5,180 genome-wide SNPs in 87 accessions of domesticated peach (<em>Prunus persica</em>) and five wild species (<em>P. davidiana</em>, <em>P. mira</em>, <em>P. kansuensis</em>, <em>P. tangutica</em>, and <em>P. webbii</em>), with 8 accessions of outgroup species (<em>P. salicina</em>, <em>P. mume</em>, and <em>P. avium</em>). Operational taxonomic units (OTU) are colored in ocher, black, green, blue, and red, for major clusters: wild species (W), landraces (L) (EA and OT for accessions in East Asia and other region, respectively), modern fruit cultivars (F), and modern ornamental cultivars (O-A and O-B for only ornamental usage, and FO for fruit and ornamentals), respectively, according to the tree topology, and to classical classifications considering uses and morphologies. The main differentiation steps of the domesticated peach, the East Asian cultivars (including modern cultivars), the O-A cluster, and the F cluster, are shown as black, green, red, and blue thick branches, respectively, with statistically significant support (bootstrap &gt; 650/1,000, except for the differentiation to the East Asian cultivars).</p></div></div><div content-id="DSW014F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01401.png?Expires=2147483647&Signature=jYy4u0yZt4Bu~9Og~szYwkgZ3pkfU8T~MRIoEVkKfIWd3qAyA35IriW8s-zJ6828FSoUm0yI6wgmvJYWSpbC4BdOVSbK8BRzrarc8XDQs98BiOLG-PZY8tzi2sQgiR03dxCmV2a9EU6lXa0RpU2kGIrzITv~le8Ol9kJRqxLXL1Y0DFv4MBE2FDJ2~9Ws2-WadK6Lf-OqQKN-IPkD6n8bfGdesPIy0BNymWzxecWZxwVId40H6xWLBgyK8I79GejEVoZ3eSwOwtRDpBomNolr53NSYKFV~8MbZftHngoO1Er326lrf6P7rCZsZC4cMpz~bgXSd9IGZ9aU38njicSbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary tree of domesticated peach and its wild relatives. Evolutionary tree constructed by a maximum likelihood (ML) approach using information from 5,180 genome-wide SNPs in 87 accessions of domesticated peach (Prunus persica) and five wild species (P. davidiana, P. mira, P. kansuensis, P. tangutica, and P. webbii), with 8 accessions of outgroup species (P. salicina, P. mume, and P. avium). Operational taxonomic units (OTU) are colored in ocher, black, green, blue, and red, for major clusters: wild species (W), landraces (L) (EA and OT for accessions in East Asia and other region, respectively), modern fruit cultivars (F), and modern ornamental cultivars (O-A and O-B for only ornamental usage, and FO for fruit and ornamentals), respectively, according to the tree topology, and to classical classifications considering uses and morphologies. The main differentiation steps of the domesticated peach, the East Asian cultivars (including modern cultivars), the O-A cluster, and the F cluster, are shown as black, green, red, and blue thick branches, respectively, with statistically significant support (bootstrap &gt; 650/1,000, except for the differentiation to the East Asian cultivars)." path-from-xml="dsw01401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200527/dsw01401.tiff" path-from-xml="dsw01401.tiff" target="_blank">View large</a><a section="87200527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01401.png?Expires=2147483647&Signature=0o5E0ynlHQbdYrJleb~8kSInCwwqMHkTXcOwJafvfqVYoD8DPc7wy02t0KbTig0DAavILbnueSomtGzK0FaX0xu6XxkaFmistPaQlRUiKfjLXXN8VHcH1GigTUVg-IK9brVFIkK2kz~dhgYw3M0xwQaW-yIjOFx0db4Jz8i-u4CSKOP2tKI1Fb18o5b2VeSPZSZSlBLQwdG88twStMB~S7Olnm6XcWfeEIorOc35otAC0L1U-F1cEqCmH9lwZkCrFIfZtlEX1Z5ZnqsegoNgsHCSIJ2uNlRKdLbMu4ZvCQmsK3NQIc84a-MVTqWAcmu9gGLsqVtJbZ0jTSmjh2VBgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200527&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary tree of domesticated peach and its wild relatives. Evolutionary tree constructed by a maximum likelihood (ML) approach using information from 5,180 genome-wide SNPs in 87 accessions of domesticated peach (<em>Prunus persica</em>) and five wild species (<em>P. davidiana</em>, <em>P. mira</em>, <em>P. kansuensis</em>, <em>P. tangutica</em>, and <em>P. webbii</em>), with 8 accessions of outgroup species (<em>P. salicina</em>, <em>P. mume</em>, and <em>P. avium</em>). Operational taxonomic units (OTU) are colored in ocher, black, green, blue, and red, for major clusters: wild species (W), landraces (L) (EA and OT for accessions in East Asia and other region, respectively), modern fruit cultivars (F), and modern ornamental cultivars (O-A and O-B for only ornamental usage, and FO for fruit and ornamentals), respectively, according to the tree topology, and to classical classifications considering uses and morphologies. The main differentiation steps of the domesticated peach, the East Asian cultivars (including modern cultivars), the O-A cluster, and the F cluster, are shown as black, green, red, and blue thick branches, respectively, with statistically significant support (bootstrap &gt; 650/1,000, except for the differentiation to the East Asian cultivars).</p></div></div><p>To examine the structure in more detail, PCA, population structure, and F<sub>ST</sub> analysis were conducted for 87 <em>Amygdalus</em> subgenus accessions using the 5,180 SNP set. PCA analysis showed distinct differentiation of the W clusters from <em>P. persica</em> with two exceptions (<em>P. davidiana</em> 2325-21A and <em>P. mira</em> 2228-21A) (Fig. <a reveal-id="DSW014F2" data-open="DSW014F2" class="link link-reveal link-table xref-fig">2</a>A for the first two principal components). In addition, this analysis revealed that the F, FO, and O-A clusters had genetic structures that were distinct from those of the others, although O-A may have experienced some hybridization with the landraces in East Asia or ancestrally shared polymorphisms with them (Fig. <a reveal-id="DSW014F2" data-open="DSW014F2" class="link link-reveal link-table xref-fig">2</a>A). The results of population structure analysis by STRUCTURE 2.2 and InStruct, assuming subpopulation <em>K</em> = 2–10 distinct clusters, were consistent with each other, and supported almost the same conclusion as in the evolutionary tree and the PCA (Fig. <a reveal-id="DSW014F2" data-open="DSW014F2" class="link link-reveal link-table xref-fig">2</a>B and C, for <em>K</em> = 2–7). Model selection based on the Δ<em>K</em> and ln Pr(<em>K</em>) supported <em>K</em> = 6 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>A and B). With <em>K</em> = 5 or less, ornamental cultivars and landraces were not clearly separated, whereas with <em>K</em> = 7, they displayed different population structures, although some genetic hybridization was likely (Fig. <a reveal-id="DSW014F2" data-open="DSW014F2" class="link link-reveal link-table xref-fig">2</a>B and C, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>C for comparison of the O and L-EA clusters in predefining two ancestral gene pools). The FO cluster was inferred to have experienced clear hybridization with the F and O/L-EA clusters in <em>K</em> = 4–7, a reasonable inference, given that they originated by recent hybridization between fruit and ornamental cultivars (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>D). Two accessions of wild species, <em>P. davidiana</em> 2325-21A and <em>P. mira</em> 2228-21A, appeared to have experienced recent hybridization or ancestrally shared polymorphic alleles with landraces at <em>K</em> = 7, and for <em>K</em> = 2, their structures were almost the same as that of the domesticated <em>P. persica</em> (Fig. <a reveal-id="DSW014F2" data-open="DSW014F2" class="link link-reveal link-table xref-fig">2</a>B and C). The F<sub>ST</sub> test supported frequent hybridization in the FO cluster and the F and L-EA clusters and suggested significant differentiation in the domesticated peach (<em>P. persica</em>) from wild relatives (F<sub>ST</sub> = 0.268 and 0.275 from <em>P. mira</em> and <em>P. davidiana</em>, respectively), and in F and O-A (F<sub>ST</sub> = 0.158 and 0.135, respectively) from the landrace L-EA (Table <a reveal-id="DSW014TB1" data-open="DSW014TB1" class="link link-reveal link-table xref-fig">1</a>), relative to the F<sub>ST</sub> values for other tree crops and wild relatives.<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSW014C22" data-open="DSW014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Also, the O-A and O-B clusters, both of which are categorized as ornamental varieties, showed distinct differentiation (F<sub>ST</sub> = 0.228). </p>                        <a id="87200531" scrollto-destination="87200531"></a>
<div content-id="DSW014TB1" class="table-modal"><div class="table-wrap"><div id="DSW014TB1" class="table-wrap-title" data-id="DSW014TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Pairwise F<sub>ST</sub> in comparison to domesticated/wild species (A) and subclusters (B)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">A </th><th rowspan="2">Peach P-AL </th><th colspan="3">Wild relatives<hr /> </th><th colspan="2"> </th></tr><tr><th>W-AL </th><th>DAV </th><th>MIR </th><th colspan="2"> </th></tr></thead><tbody><tr><td colspan="7">Domesticated peach (<em>Prunus persica</em>) (P-AL) </td></tr><tr><td>All wild relatives (W-AL) </td><td>0.301 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. davidiana</em> (DAV) </td><td>0.275 </td><td>– </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. mira</em> (MIR) </td><td>0.268 </td><td>– </td><td>0.212 </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>F </td><td>FO </td><td>O-A </td><td>O-B </td><td>L-EA </td><td>L-OT </td></tr><tr><td colspan="7">Fruit cluster (F) </td></tr><tr><td colspan="7">Ornamental </td></tr><tr><td> Fruit and ornamental cluster (O-F) </td><td>0.062 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-A cluster (O-A) </td><td>0.245 </td><td>0.141 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-B cluster (O-B) </td><td>0.229 </td><td>0.222 </td><td>0.228 </td><td> </td><td> </td><td> </td></tr><tr><td colspan="7">Landraces </td></tr><tr><td> East Asia (L-EA) </td><td>0.158 </td><td>0.058 </td><td>0.135 </td><td>0.103 </td><td> </td><td> </td></tr><tr><td> Other regions (L-OT) </td><td>0.189 </td><td>0.154 </td><td>0.205 </td><td>0.211 </td><td>0.092 </td><td> </td></tr></tbody></table></div><div content-id="DSW014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">A </th><th rowspan="2">Peach P-AL </th><th colspan="3">Wild relatives<hr /> </th><th colspan="2"> </th></tr><tr><th>W-AL </th><th>DAV </th><th>MIR </th><th colspan="2"> </th></tr></thead><tbody><tr><td colspan="7">Domesticated peach (<em>Prunus persica</em>) (P-AL) </td></tr><tr><td>All wild relatives (W-AL) </td><td>0.301 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. davidiana</em> (DAV) </td><td>0.275 </td><td>– </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. mira</em> (MIR) </td><td>0.268 </td><td>– </td><td>0.212 </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>F </td><td>FO </td><td>O-A </td><td>O-B </td><td>L-EA </td><td>L-OT </td></tr><tr><td colspan="7">Fruit cluster (F) </td></tr><tr><td colspan="7">Ornamental </td></tr><tr><td> Fruit and ornamental cluster (O-F) </td><td>0.062 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-A cluster (O-A) </td><td>0.245 </td><td>0.141 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-B cluster (O-B) </td><td>0.229 </td><td>0.222 </td><td>0.228 </td><td> </td><td> </td><td> </td></tr><tr><td colspan="7">Landraces </td></tr><tr><td> East Asia (L-EA) </td><td>0.158 </td><td>0.058 </td><td>0.135 </td><td>0.103 </td><td> </td><td> </td></tr><tr><td> Other regions (L-OT) </td><td>0.189 </td><td>0.154 </td><td>0.205 </td><td>0.211 </td><td>0.092 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200531" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW014TB1" class="table-wrap-title" data-id="DSW014TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Pairwise F<sub>ST</sub> in comparison to domesticated/wild species (A) and subclusters (B)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">A </th><th rowspan="2">Peach P-AL </th><th colspan="3">Wild relatives<hr /> </th><th colspan="2"> </th></tr><tr><th>W-AL </th><th>DAV </th><th>MIR </th><th colspan="2"> </th></tr></thead><tbody><tr><td colspan="7">Domesticated peach (<em>Prunus persica</em>) (P-AL) </td></tr><tr><td>All wild relatives (W-AL) </td><td>0.301 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. davidiana</em> (DAV) </td><td>0.275 </td><td>– </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. mira</em> (MIR) </td><td>0.268 </td><td>– </td><td>0.212 </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>F </td><td>FO </td><td>O-A </td><td>O-B </td><td>L-EA </td><td>L-OT </td></tr><tr><td colspan="7">Fruit cluster (F) </td></tr><tr><td colspan="7">Ornamental </td></tr><tr><td> Fruit and ornamental cluster (O-F) </td><td>0.062 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-A cluster (O-A) </td><td>0.245 </td><td>0.141 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-B cluster (O-B) </td><td>0.229 </td><td>0.222 </td><td>0.228 </td><td> </td><td> </td><td> </td></tr><tr><td colspan="7">Landraces </td></tr><tr><td> East Asia (L-EA) </td><td>0.158 </td><td>0.058 </td><td>0.135 </td><td>0.103 </td><td> </td><td> </td></tr><tr><td> Other regions (L-OT) </td><td>0.189 </td><td>0.154 </td><td>0.205 </td><td>0.211 </td><td>0.092 </td><td> </td></tr></tbody></table></div><div content-id="DSW014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">A </th><th rowspan="2">Peach P-AL </th><th colspan="3">Wild relatives<hr /> </th><th colspan="2"> </th></tr><tr><th>W-AL </th><th>DAV </th><th>MIR </th><th colspan="2"> </th></tr></thead><tbody><tr><td colspan="7">Domesticated peach (<em>Prunus persica</em>) (P-AL) </td></tr><tr><td>All wild relatives (W-AL) </td><td>0.301 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. davidiana</em> (DAV) </td><td>0.275 </td><td>– </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> <em>P. mira</em> (MIR) </td><td>0.268 </td><td>– </td><td>0.212 </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>F </td><td>FO </td><td>O-A </td><td>O-B </td><td>L-EA </td><td>L-OT </td></tr><tr><td colspan="7">Fruit cluster (F) </td></tr><tr><td colspan="7">Ornamental </td></tr><tr><td> Fruit and ornamental cluster (O-F) </td><td>0.062 </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-A cluster (O-A) </td><td>0.245 </td><td>0.141 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> Ornamental-B cluster (O-B) </td><td>0.229 </td><td>0.222 </td><td>0.228 </td><td> </td><td> </td><td> </td></tr><tr><td colspan="7">Landraces </td></tr><tr><td> East Asia (L-EA) </td><td>0.158 </td><td>0.058 </td><td>0.135 </td><td>0.103 </td><td> </td><td> </td></tr><tr><td> Other regions (L-OT) </td><td>0.189 </td><td>0.154 </td><td>0.205 </td><td>0.211 </td><td>0.092 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200531" target="_blank">View Large</a></div></div>                        <a id="87200533" scrollto-destination="87200533"></a>
<div id="DSW014F2" data-id="DSW014F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01402.png?Expires=2147483647&Signature=pD4bpdKy5mEPikJsm3UJzNzcYOYnNOHtKzgxRCbfi14jgPoy~TmU8oYP9Mc1zaKnNPBjdIHzNiLujJh45ql8gKeFRhcNbrnbZlbYn0i8DQfXuo65dlfbWCl~STYPCXmhxNFMFl9KgkRy4LM0J7emlQfMW8td47oQvcFv4-e4Q5qUbO1xxPMTtgAViZfjiSFZBDYczWQFNfVvzT4aCOiXeWLR46O4AU1h6kJ3ivdQO~mAteIXaFh-onR07uRItXFjX4QkiQVeljG07nHT5MXlABCXi4zvoP-lNuNytj646VQIbFj0MyGGVr1~tlLjaMDkI5s5DVQuVHPtb95BWBXGiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population structure analysis in peach. (A) Principal component analysis using information from 5,180 genome-wide SNPs in 87 accessions including domesticated peach and its wild relatives (right) and in 67 accessions focusing on domesticated peach (left). The first two components in PCA (PC1 and PC2) are plotted on the axes to visualize the genetic relationships. The proportion of variance explained by each PC is given in parentheses along each axis. The wild species is shown as a cross. The landraces are shown as squares in green and yellow for accessions in East Asia and other regions, respectively. The modern cultivars are shown as circles in blue and red, corresponding to fruit and ornamental cultivars, respectively. (B and C) Structure analysis of subdivision of the population (K = 2–7), with STRUCTURE 2.2 (B) and with InStruct (C). Each individual is shown as a vertical bar. In K = 2, wild species showed a cluster distinct from domesticated peach, except for two accessions experiencing frequent hybridization with domesticated peach. In K = 5 or more, the F, L-OT, and O-A/O-B/L-EA clusters show clear separation. The O-A and O-B/L-EA showed significant separation at K = 7 or more." path-from-xml="dsw01402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200533/dsw01402.tiff" path-from-xml="dsw01402.tiff" target="_blank">View large</a><a section="87200533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01402.png?Expires=2147483647&Signature=NkyLkViHBxzHm8Cs8FAsyn81CTJVlYl1E4~vxkDKtQaF~bYkAHKaCkg1cGl0QucIA~AJj8l2Y7ncV1M~ynT7Ku17JvtRyTXqnGAMfrmBhMSxYyigIK0~S--xDBOCZHE-keQ26M7-hPfbY83-sOE2eC7vt~25~qq5MSPNXETAs6UeVNQu5nBL7vgJi0E8kylm4gbFDoOqmMvAKn663htFVd9ks5ymnrwT3cJifq8rwGlbeDj0YdBsZkK1UuXqYfPmLDnEB3JM0N~2xS-k9qDXItR9yMOS5kxgI-aeGldzsYGnvA2Qs4FOWQGETUFE~g3S7ruGEs4Rk2RUf7aW~t7IzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200533&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population structure analysis in peach. (A) Principal component analysis using information from 5,180 genome-wide SNPs in 87 accessions including domesticated peach and its wild relatives (right) and in 67 accessions focusing on domesticated peach (left). The first two components in PCA (PC1 and PC2) are plotted on the axes to visualize the genetic relationships. The proportion of variance explained by each PC is given in parentheses along each axis. The wild species is shown as a cross. The landraces are shown as squares in green and yellow for accessions in East Asia and other regions, respectively. The modern cultivars are shown as circles in blue and red, corresponding to fruit and ornamental cultivars, respectively. (B and C) Structure analysis of subdivision of the population (<em>K</em> = 2–7), with STRUCTURE 2.2 (B) and with InStruct (C). Each individual is shown as a vertical bar. In <em>K</em> = 2, wild species showed a cluster distinct from domesticated peach, except for two accessions experiencing frequent hybridization with domesticated peach. In <em>K</em> = 5 or more, the F, L-OT, and O-A/O-B/L-EA clusters show clear separation. The O-A and O-B/L-EA showed significant separation at <em>K</em> = 7 or more.</p></div></div><div content-id="DSW014F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01402.png?Expires=2147483647&Signature=pD4bpdKy5mEPikJsm3UJzNzcYOYnNOHtKzgxRCbfi14jgPoy~TmU8oYP9Mc1zaKnNPBjdIHzNiLujJh45ql8gKeFRhcNbrnbZlbYn0i8DQfXuo65dlfbWCl~STYPCXmhxNFMFl9KgkRy4LM0J7emlQfMW8td47oQvcFv4-e4Q5qUbO1xxPMTtgAViZfjiSFZBDYczWQFNfVvzT4aCOiXeWLR46O4AU1h6kJ3ivdQO~mAteIXaFh-onR07uRItXFjX4QkiQVeljG07nHT5MXlABCXi4zvoP-lNuNytj646VQIbFj0MyGGVr1~tlLjaMDkI5s5DVQuVHPtb95BWBXGiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population structure analysis in peach. (A) Principal component analysis using information from 5,180 genome-wide SNPs in 87 accessions including domesticated peach and its wild relatives (right) and in 67 accessions focusing on domesticated peach (left). The first two components in PCA (PC1 and PC2) are plotted on the axes to visualize the genetic relationships. The proportion of variance explained by each PC is given in parentheses along each axis. The wild species is shown as a cross. The landraces are shown as squares in green and yellow for accessions in East Asia and other regions, respectively. The modern cultivars are shown as circles in blue and red, corresponding to fruit and ornamental cultivars, respectively. (B and C) Structure analysis of subdivision of the population (K = 2–7), with STRUCTURE 2.2 (B) and with InStruct (C). Each individual is shown as a vertical bar. In K = 2, wild species showed a cluster distinct from domesticated peach, except for two accessions experiencing frequent hybridization with domesticated peach. In K = 5 or more, the F, L-OT, and O-A/O-B/L-EA clusters show clear separation. The O-A and O-B/L-EA showed significant separation at K = 7 or more." path-from-xml="dsw01402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200533/dsw01402.tiff" path-from-xml="dsw01402.tiff" target="_blank">View large</a><a section="87200533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01402.png?Expires=2147483647&Signature=NkyLkViHBxzHm8Cs8FAsyn81CTJVlYl1E4~vxkDKtQaF~bYkAHKaCkg1cGl0QucIA~AJj8l2Y7ncV1M~ynT7Ku17JvtRyTXqnGAMfrmBhMSxYyigIK0~S--xDBOCZHE-keQ26M7-hPfbY83-sOE2eC7vt~25~qq5MSPNXETAs6UeVNQu5nBL7vgJi0E8kylm4gbFDoOqmMvAKn663htFVd9ks5ymnrwT3cJifq8rwGlbeDj0YdBsZkK1UuXqYfPmLDnEB3JM0N~2xS-k9qDXItR9yMOS5kxgI-aeGldzsYGnvA2Qs4FOWQGETUFE~g3S7ruGEs4Rk2RUf7aW~t7IzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200533&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population structure analysis in peach. (A) Principal component analysis using information from 5,180 genome-wide SNPs in 87 accessions including domesticated peach and its wild relatives (right) and in 67 accessions focusing on domesticated peach (left). The first two components in PCA (PC1 and PC2) are plotted on the axes to visualize the genetic relationships. The proportion of variance explained by each PC is given in parentheses along each axis. The wild species is shown as a cross. The landraces are shown as squares in green and yellow for accessions in East Asia and other regions, respectively. The modern cultivars are shown as circles in blue and red, corresponding to fruit and ornamental cultivars, respectively. (B and C) Structure analysis of subdivision of the population (<em>K</em> = 2–7), with STRUCTURE 2.2 (B) and with InStruct (C). Each individual is shown as a vertical bar. In <em>K</em> = 2, wild species showed a cluster distinct from domesticated peach, except for two accessions experiencing frequent hybridization with domesticated peach. In <em>K</em> = 5 or more, the F, L-OT, and O-A/O-B/L-EA clusters show clear separation. The O-A and O-B/L-EA showed significant separation at <em>K</em> = 7 or more.</p></div></div><p>Based on these population structure results, the O-B cluster was included in the L-EA (or L) cluster and the F-O was no longer considered because at least one of them originated from the recent hybridization between F and O-A gene pools (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>D).</p>                            <h3 scrollto-destination=87200540 id="87200540" class="section-title" data-legacy-id=s3b>3.2. Detection of genetic diversity, LD, and inbreeding/bottlenecks</h3>
<p>Using the information from the 5,180 SNP set, we evaluated the possibility of inbreeding/bottleneck situations by comparing observed H<sub>O</sub> versus H<sub>E</sub> and using F<sub>IS</sub> values with a null hypothesis of random mating within each species. When considering the entire domesticated peach group (<em>P. persica</em>, <em>n</em> = 67), the H<sub>O</sub> value was found to be significantly lower than the He value (H<sub>O</sub> = 0.264, H<sub>E</sub> = 0.401, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), which suggests the occurrence of a population bottleneck or a recent shift to higher inbreeding during domestication. A similar situation was observed for the L clusters (H<sub>O</sub> = 0.220, H<sub>E</sub> = 0.375 for L-EA, H<sub>O</sub> = 0.173, H<sub>E</sub> = 0.345 for L-OT). These findings were also supported by F<sub>IS</sub> values, suggesting degrees of inbreeding (F<sub>IS</sub> = 0.33–0.52; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In contrast, the F and O clusters demonstrated no such trends in H<sub>O</sub>/H<sub>E</sub> and F<sub>IS</sub> values (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Pairwise characterization of IBD to infer pedigree relationships among 67 <em>P. persica</em> individuals, however, showed that all accessions in the F cluster had first- or second-degree relationships with cv. Chinese Cling (pi-hat &gt; 0.25 for all individuals, average pi-hat = 0.44; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), supporting previous proposals regarding the establishment of modern cultivars<sup><a reveal-id="DSW014C32" data-open="DSW014C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSW014C33" data-open="DSW014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and supporting the presence of strong genetic drift in the establishment of the F cluster. We detected significant IBD proportions between <em>P. davidiana</em> 2325-21A or <em>P. mira</em> 2228-21A and some individuals in the L cluster (pi-hat = ∼0.189), despite the fact that <em>P. mira</em> and <em>P. davidiana</em> nested to the W cluster revealed no significant IBD values (pi-hat = 0). Moreover, the FO cluster demonstrated significant IBD values against the F and O-A clusters (pi-hat = ∼0.419), although the F and O-A clusters have almost no pedigree relationships. These results support the possibility of recent hybridizations among these clusters.</p><p>The degree of population structuring is related to LD extension/decay. In domesticated peach, LD in European and US cultivars has been reported to be longer (up to 13–15cM) than in other major perennial crops such as grape (&lt;10 kb),<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> based on an analysis of 50 microsatellite markers by Aranzana et al.<sup><a reveal-id="DSW014C33" data-open="DSW014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> However, the LD in landraces and the edible peaches primarily from China calculated using genome-wide SNPs was much shorter (up to 50 kb).<sup><a reveal-id="DSW014C30" data-open="DSW014C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The LD decay in domesticated peach cultivars, in which the average pairwise LD values showed no significant difference against the background, was extended to ∼1,000–2,500 kb and would be even more delayed in the wild relatives (although the LD decay in the Wild cluster was unclear, presumably owing to a shortage of effective accession numbers and of polymorphisms in the W cluster when this SNPs chip was used) (Fig. <a reveal-id="DSW014F3" data-open="DSW014F3" class="link link-reveal link-table xref-fig">3</a>A for chromosome 4, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> for other chromosomes and LD plots). This distance of LD decay in the domesticated peach population is consistent with that in US and European peach cultivars<sup><a reveal-id="DSW014C33" data-open="DSW014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and greater than that in other major domesticated crops such as rice (up to 50–150 kb),<sup><a reveal-id="DSW014C9" data-open="DSW014C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW014C60" data-open="DSW014C60" class="link link-ref link-reveal xref-bibr">60</a></sup> sorghum (up to ca. 150 kb),<sup><a reveal-id="DSW014C42" data-open="DSW014C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and maize (up to 2 kb).<sup><a reveal-id="DSW014C61" data-open="DSW014C61" class="link link-ref link-reveal xref-bibr">61</a></sup> In comparing clusters, the F cluster showed much greater delay in decay of LD than the two subclusters of the L cluster (L-EA and L-OT), ranging over 3,000 kb on some chromosomes (Fig. <a reveal-id="DSW014F3" data-open="DSW014F3" class="link link-reveal link-table xref-fig">3</a>B, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). This finding supports a bottleneck or shift to higher inbreeding in the establishment of the F cluster from the progenitor landraces. The results of IBD test would support a recent bottleneck from cv. Chinese Cling (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The O-A cluster, in general, showed no significant differences in LD decay from the two L subclusters (Fig. <a reveal-id="DSW014F3" data-open="DSW014F3" class="link link-reveal link-table xref-fig">3</a>B), although a slightly significant increase in LD could be detected for the O-A cluster on a few chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). A significant delay in LD decay might also be expected from the distribution of EHH presented later. In the genome-wide LD analysis, we detected a clear monotonic decrease in static values of LD with the physical distance on any chromosome ranging from at least 1,000 kb in the domesticated peach cluster to even more in the F subcluster. This situation would make it preferable to identify selective sweeps based on LD index using small numbers of marker sets. In this study, we could use at least 5,000 pruned SNP sets (one SNP locus per <em>ca.</em> 45 kb on average in the peach genome) for subsequent analysis of selected genomic regions, yielding sufficient information for the identification of statistically distorted LD values from each SNP core in the genome. </p>                        <a id="87200546" scrollto-destination="87200546"></a>
<div id="DSW014F3" data-id="DSW014F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01403.png?Expires=2147483647&Signature=Ut45BdrMDolgYgFfu4OO3N9TtBAHa4oz2cnwFSP6JHrySq1G3vwHUynZwd-C~HQtNAg8Zyz-E7MIpeKJ1gd0VOxsOC5No1XlO0TS24gbUTInPxaijcEXb9W9eDomGk-PwvdOB4zS4ZFEmGVZzfCrr-R4KD~~LbEB2m~wFrlM5880eATCVAfav0b75jAaZ3Edp-nTgAajplUMq-gQ~ajA~9vh9YdlirccVwd86R~-EsivvbCu2aZ~91~u3cUgLzjpKsitbJh9VtL6F93vRgqONP5OIY~ogFLNfenaCIywYFlkovEpRrsGJW92PK2NdfUZGVeCheq9a4wd-D9SDxHXiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of LD decay in the wild species and each variety complex of domesticated peach. The average values of R2 in pairwise LD among the 1,346 SNPs on Chromosome 4, which carries the most SNPs among the eight chromosomes, in windows of up to 10,000 kb are shown. The Y-axis standards are adjusted according to the background values in each subpopulation for visualization of LD decay for comparison among subpopulations. The X-axis shows physical distance among the SNPs, and the average R2 values at 100-kb intervals are plotted. (A) Comparison of LD decay in domesticated peach and wild relatives. Black and white arrows indicate LD decay points in domesticated peach and wild relative, respectively. (B) Comparison of LD decay in the F, O-A, L-EA, and L-OT clusters. The F cluster shows only significantly expanded LD, as shown by the black arrow, in comparison to the other clusters, whose LD decays are shown with white arrows and black bars." path-from-xml="dsw01403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200546/dsw01403.tiff" path-from-xml="dsw01403.tiff" target="_blank">View large</a><a section="87200546" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01403.png?Expires=2147483647&Signature=Bq5IAE27aiAz-o9I1VSpyrOf-hDAo2KlFhDzsEL6yUb3Lv5Qt4dIxOT0bZe44XU1nVqN32MY2TEBuiXcrz2u0FhHlGvBPS7xbOEaR2Jr3Ro0mGWQVYOwm5mx3PuA4~gtATBUSj-pKx7qTN~fptRAgTHqMZTeibjCWT6TdMS0ANWxARUUW2P32EVW-9qZTLZ4lV2EYr-1h6R86xD7rqW64IniGvDhu1tl4Z4yQ7v~ttl-UV1BpjAlx3eaHiUIvxUjo~L4DLxvF3o22kqNEXnGl-aQPPBwcH--oEt5mYZv3q2SqqWm9J~lCKnfID3Ia2MIYVLxHrBdDiGKWnZcjW3WmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200546&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of LD decay in the wild species and each variety complex of domesticated peach. The average values of <em>R</em><sup>2</sup> in pairwise LD among the 1,346 SNPs on Chromosome 4, which carries the most SNPs among the eight chromosomes, in windows of up to 10,000 kb are shown. The <em>Y</em>-axis standards are adjusted according to the background values in each subpopulation for visualization of LD decay for comparison among subpopulations. The <em>X</em>-axis shows physical distance among the SNPs, and the average <em>R</em><sup>2</sup> values at 100-kb intervals are plotted. (A) Comparison of LD decay in domesticated peach and wild relatives. Black and white arrows indicate LD decay points in domesticated peach and wild relative, respectively. (B) Comparison of LD decay in the F, O-A, L-EA, and L-OT clusters. The F cluster shows only significantly expanded LD, as shown by the black arrow, in comparison to the other clusters, whose LD decays are shown with white arrows and black bars.</p></div></div><div content-id="DSW014F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01403.png?Expires=2147483647&Signature=Ut45BdrMDolgYgFfu4OO3N9TtBAHa4oz2cnwFSP6JHrySq1G3vwHUynZwd-C~HQtNAg8Zyz-E7MIpeKJ1gd0VOxsOC5No1XlO0TS24gbUTInPxaijcEXb9W9eDomGk-PwvdOB4zS4ZFEmGVZzfCrr-R4KD~~LbEB2m~wFrlM5880eATCVAfav0b75jAaZ3Edp-nTgAajplUMq-gQ~ajA~9vh9YdlirccVwd86R~-EsivvbCu2aZ~91~u3cUgLzjpKsitbJh9VtL6F93vRgqONP5OIY~ogFLNfenaCIywYFlkovEpRrsGJW92PK2NdfUZGVeCheq9a4wd-D9SDxHXiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of LD decay in the wild species and each variety complex of domesticated peach. The average values of R2 in pairwise LD among the 1,346 SNPs on Chromosome 4, which carries the most SNPs among the eight chromosomes, in windows of up to 10,000 kb are shown. The Y-axis standards are adjusted according to the background values in each subpopulation for visualization of LD decay for comparison among subpopulations. The X-axis shows physical distance among the SNPs, and the average R2 values at 100-kb intervals are plotted. (A) Comparison of LD decay in domesticated peach and wild relatives. Black and white arrows indicate LD decay points in domesticated peach and wild relative, respectively. (B) Comparison of LD decay in the F, O-A, L-EA, and L-OT clusters. The F cluster shows only significantly expanded LD, as shown by the black arrow, in comparison to the other clusters, whose LD decays are shown with white arrows and black bars." path-from-xml="dsw01403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200546/dsw01403.tiff" path-from-xml="dsw01403.tiff" target="_blank">View large</a><a section="87200546" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01403.png?Expires=2147483647&Signature=Bq5IAE27aiAz-o9I1VSpyrOf-hDAo2KlFhDzsEL6yUb3Lv5Qt4dIxOT0bZe44XU1nVqN32MY2TEBuiXcrz2u0FhHlGvBPS7xbOEaR2Jr3Ro0mGWQVYOwm5mx3PuA4~gtATBUSj-pKx7qTN~fptRAgTHqMZTeibjCWT6TdMS0ANWxARUUW2P32EVW-9qZTLZ4lV2EYr-1h6R86xD7rqW64IniGvDhu1tl4Z4yQ7v~ttl-UV1BpjAlx3eaHiUIvxUjo~L4DLxvF3o22kqNEXnGl-aQPPBwcH--oEt5mYZv3q2SqqWm9J~lCKnfID3Ia2MIYVLxHrBdDiGKWnZcjW3WmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200546&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of LD decay in the wild species and each variety complex of domesticated peach. The average values of <em>R</em><sup>2</sup> in pairwise LD among the 1,346 SNPs on Chromosome 4, which carries the most SNPs among the eight chromosomes, in windows of up to 10,000 kb are shown. The <em>Y</em>-axis standards are adjusted according to the background values in each subpopulation for visualization of LD decay for comparison among subpopulations. The <em>X</em>-axis shows physical distance among the SNPs, and the average <em>R</em><sup>2</sup> values at 100-kb intervals are plotted. (A) Comparison of LD decay in domesticated peach and wild relatives. Black and white arrows indicate LD decay points in domesticated peach and wild relative, respectively. (B) Comparison of LD decay in the F, O-A, L-EA, and L-OT clusters. The F cluster shows only significantly expanded LD, as shown by the black arrow, in comparison to the other clusters, whose LD decays are shown with white arrows and black bars.</p></div></div>                            <h3 scrollto-destination=87200550 id="87200550" class="section-title" data-legacy-id=s3c>3.3. Genome-wide detection of selection in domestication and cultivar differentiation pathways</h3>
<p>To evaluate selective sweeps in local genomic regions, indexes based on the site frequency spectrum (SFS), such as π or Tajima's D which can detect fixed sweeps in a population, have been applied in plant species.<sup><a reveal-id="DSW014C8" data-open="DSW014C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSW014C12  DSW014C14" data-open="DSW014C12  DSW014C14" class="link link-ref link-reveal xref-bibr">12–14</a>,<a reveal-id="DSW014C30" data-open="DSW014C30" class="link link-ref link-reveal xref-bibr">30</a></sup> In genome-wide analysis of selected regions, similar approaches including LD- or haploblock-based methods, which usually detect very recent and ongoing sweeps, are reported to be effective for some plant species<sup><a reveal-id="DSW014C9" data-open="DSW014C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW014C10" data-open="DSW014C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW014C62" data-open="DSW014C62" class="link link-ref link-reveal xref-bibr">62</a></sup> and for some domesticated animals such as dog<sup><a reveal-id="DSW014C63" data-open="DSW014C63" class="link link-ref link-reveal xref-bibr">63</a></sup> and chicken.<sup><a reveal-id="DSW014C39" data-open="DSW014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Here, selections for domesticated peaches were expected to be recent or ongoing, particularly for the F cluster. Peach is a species with a long generation time (at least 2–3 yrs to flowering) and one for which individual genotypes can be maintained by vegetative propagation for long time periods, suggesting that selected genes may have been maintained in heterozygous states. Thus, we adopted indexes based not only on SFS but also on LD or haploblock for exploiting selective sweep, following genome-wide analyses of selection in human genomes.<sup><a reveal-id="DSW014C7" data-open="DSW014C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSW014C52  DSW014C54" data-open="DSW014C52  DSW014C54" class="link link-ref link-reveal xref-bibr">52–54</a></sup></p><p>In SFS-based analysis, we evaluated the transition of pooled heterozygosity (<em>H</em><sub>p</sub>) of each cluster using informative SNPs in a 400-kb sliding window with 100-kb steps, following a previous report on SFS-based analysis in chicken.<sup><a reveal-id="DSW014C39" data-open="DSW014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The distributions of observed <em>H</em><sub>p</sub> values and <em>Z</em>-score values of <em>H</em><sub>p</sub> (<em>ZH</em><sub>p</sub>), which considered coalescent effects by using the entire SNP sets in the peach genome, are shown in Fig. <a reveal-id="DSW014F4" data-open="DSW014F4" class="link link-reveal link-table xref-fig">4</a>A for the domesticated peach population and two subpopulations, F and O-A. All showed some genomic regions containing candidates for selective sweeps with significant homozygous states in comparison to the whole genome (<em>P</em> &lt; 0.001). However, the possibility that they are derived from differences in the substitution ratio in the peach genome, distortion of the availability in SNP patterns, or simple drift cannot be ruled out. Comparison of the <em>H</em><sub>p</sub> values between the F- or O-A clusters and the L cluster (Fig. <a reveal-id="DSW014F4" data-open="DSW014F4" class="link link-reveal link-table xref-fig">4</a>B) revealed a significant bias in homozygous states among the clusters in specific genomic regions and suggested that at least some represent true selective sweeps that specifically occurred in the path to modern fruit or ornamental cultivar development. The middle of Chromosome 7 (<em>ca.</em> 8,000–8,800 kb) and the bottom of Chromosome 4 (<em>ca.</em> 29,200–29,600 kb) showed particularly clear tendencies to selective sweeps specific to the F and O-A subpopulations, respectively (Fig. <a reveal-id="DSW014F4" data-open="DSW014F4" class="link link-reveal link-table xref-fig">4</a>C and D). </p>                        <a id="87200556" scrollto-destination="87200556"></a>
<div id="DSW014F4" data-id="DSW014F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01404.png?Expires=2147483647&Signature=LRgcBJHFS6RbqUEbeKygeQ4Wle3IOv6Dt9HbnZaeSNz6HmuQcOMchQUKTjIdf4w~CAEOiIDRm54WH~DaWAaZzAJ4Az4uxnaTf-ldrIzZbrZMgj5uY0nskaG6sRpYxGuJmSuoR7sRHgZjs3jykkArwyKm-0yd7XUTkEDJcltyFlnTxKUjL-WSQ5NJeUDuHOuVE3WfTr-QqLRmwhDJ-NoFP5xF19ZbNEMNwuBubfCsg6-l5XUobN54J6JPFcdM47FtTnAd7eePeWr1iEjTUGlO5DtICJVycBxZ6G2ATVjFSit4ugjLUcOG238LOu0MP~vp6sWA8K6iiPcocgqvb7Fs6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide selective sweep analysis based on ZHp, in the domesticated peach and two modern varietal complexes. (A) Transitions of Z-transformed values of Hp in 400-kb bins with 100-kb steps are shown for the domesticated peach population and two subpopulations, the F and O-A clusters. Putative regions showing selective sweeps (P &lt; 0.001) are indicated by outlined triangles. (B) For the F and O-A clusters, plots of relative values of Hp in 400-kb bins with 100-kb steps against the L cluster are shown. Putative regions showing selective sweeps in the paths of differentiation from landraces are indicated by triangles. For the two regions indicated by black triangles, detailed characterization of ZHp in a comparison among four subpopulations (F, O-A, L-EA, and L-OT) is shown in (C) for the top of Chromosome 7, which corresponded to a putative selective sweep in the F cluster, and in (D) for the bottom of Chromosome 4, showing a putative selective sweep in the O-A cluster." path-from-xml="dsw01404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200556/dsw01404.tiff" path-from-xml="dsw01404.tiff" target="_blank">View large</a><a section="87200556" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01404.png?Expires=2147483647&Signature=VeOiM9Now-GeQOKsSBsLAgdFyVKK8drvpFj-Q3aQLGFDEhrE6Tmg-tMw7qnDNXunBR2L2cZZHC~YDxpAr-sQedNhbH-QDNG0XX1noXj7OeBSjQHR8POb-Cfh4FtlRQUHxTkKFDXlzBxDEBf9PkcnPhu2aPFuQlYBK4B107h2vKKA29MpdsGFCXg4TyIqo36hsi8RXBlFIvIrRvTD1kT8h9L3t0ifgiHdRkwFYs5-wYHJRVUPycsmyvyU-QG4C90ZmkyXxeQzuUCYwNslFtJ9fzELUewFY0L3MiMQoogQhUvL6fohgOo4fFUp1faQ5D6dzPFB4ys3tnVk0eVKQ8kckQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200556&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide selective sweep analysis based on <em>ZH</em><sub>p</sub>, in the domesticated peach and two modern varietal complexes. (A) Transitions of <em>Z</em>-transformed values of <em>H</em><sub>p</sub> in 400-kb bins with 100-kb steps are shown for the domesticated peach population and two subpopulations, the F and O-A clusters. Putative regions showing selective sweeps (<em>P</em> &lt; 0.001) are indicated by outlined triangles. (B) For the F and O-A clusters, plots of relative values of <em>H</em><sub>p</sub> in 400-kb bins with 100-kb steps against the L cluster are shown. Putative regions showing selective sweeps in the paths of differentiation from landraces are indicated by triangles. For the two regions indicated by black triangles, detailed characterization of <em>ZH</em><sub>p</sub> in a comparison among four subpopulations (F, O-A, L-EA, and L-OT) is shown in (C) for the top of Chromosome 7, which corresponded to a putative selective sweep in the F cluster, and in (D) for the bottom of Chromosome 4, showing a putative selective sweep in the O-A cluster.</p></div></div><div content-id="DSW014F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01404.png?Expires=2147483647&Signature=LRgcBJHFS6RbqUEbeKygeQ4Wle3IOv6Dt9HbnZaeSNz6HmuQcOMchQUKTjIdf4w~CAEOiIDRm54WH~DaWAaZzAJ4Az4uxnaTf-ldrIzZbrZMgj5uY0nskaG6sRpYxGuJmSuoR7sRHgZjs3jykkArwyKm-0yd7XUTkEDJcltyFlnTxKUjL-WSQ5NJeUDuHOuVE3WfTr-QqLRmwhDJ-NoFP5xF19ZbNEMNwuBubfCsg6-l5XUobN54J6JPFcdM47FtTnAd7eePeWr1iEjTUGlO5DtICJVycBxZ6G2ATVjFSit4ugjLUcOG238LOu0MP~vp6sWA8K6iiPcocgqvb7Fs6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide selective sweep analysis based on ZHp, in the domesticated peach and two modern varietal complexes. (A) Transitions of Z-transformed values of Hp in 400-kb bins with 100-kb steps are shown for the domesticated peach population and two subpopulations, the F and O-A clusters. Putative regions showing selective sweeps (P &lt; 0.001) are indicated by outlined triangles. (B) For the F and O-A clusters, plots of relative values of Hp in 400-kb bins with 100-kb steps against the L cluster are shown. Putative regions showing selective sweeps in the paths of differentiation from landraces are indicated by triangles. For the two regions indicated by black triangles, detailed characterization of ZHp in a comparison among four subpopulations (F, O-A, L-EA, and L-OT) is shown in (C) for the top of Chromosome 7, which corresponded to a putative selective sweep in the F cluster, and in (D) for the bottom of Chromosome 4, showing a putative selective sweep in the O-A cluster." path-from-xml="dsw01404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200556/dsw01404.tiff" path-from-xml="dsw01404.tiff" target="_blank">View large</a><a section="87200556" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01404.png?Expires=2147483647&Signature=VeOiM9Now-GeQOKsSBsLAgdFyVKK8drvpFj-Q3aQLGFDEhrE6Tmg-tMw7qnDNXunBR2L2cZZHC~YDxpAr-sQedNhbH-QDNG0XX1noXj7OeBSjQHR8POb-Cfh4FtlRQUHxTkKFDXlzBxDEBf9PkcnPhu2aPFuQlYBK4B107h2vKKA29MpdsGFCXg4TyIqo36hsi8RXBlFIvIrRvTD1kT8h9L3t0ifgiHdRkwFYs5-wYHJRVUPycsmyvyU-QG4C90ZmkyXxeQzuUCYwNslFtJ9fzELUewFY0L3MiMQoogQhUvL6fohgOo4fFUp1faQ5D6dzPFB4ys3tnVk0eVKQ8kckQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200556&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide selective sweep analysis based on <em>ZH</em><sub>p</sub>, in the domesticated peach and two modern varietal complexes. (A) Transitions of <em>Z</em>-transformed values of <em>H</em><sub>p</sub> in 400-kb bins with 100-kb steps are shown for the domesticated peach population and two subpopulations, the F and O-A clusters. Putative regions showing selective sweeps (<em>P</em> &lt; 0.001) are indicated by outlined triangles. (B) For the F and O-A clusters, plots of relative values of <em>H</em><sub>p</sub> in 400-kb bins with 100-kb steps against the L cluster are shown. Putative regions showing selective sweeps in the paths of differentiation from landraces are indicated by triangles. For the two regions indicated by black triangles, detailed characterization of <em>ZH</em><sub>p</sub> in a comparison among four subpopulations (F, O-A, L-EA, and L-OT) is shown in (C) for the top of Chromosome 7, which corresponded to a putative selective sweep in the F cluster, and in (D) for the bottom of Chromosome 4, showing a putative selective sweep in the O-A cluster.</p></div></div><p>LD- or haploblock-based analyses were performed in accordance with the concept of EHH.<sup><a reveal-id="DSW014C52" data-open="DSW014C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The program Sweep<sup><a reveal-id="DSW014C53" data-open="DSW014C53" class="link link-ref link-reveal xref-bibr">53</a></sup> was used to evaluate the iHS,<sup><a reveal-id="DSW014C54" data-open="DSW014C54" class="link link-ref link-reveal xref-bibr">54</a></sup> which quantifies the difference of EHH values around the selected locus in one population, and the Cross-Population EHH (XP-EHH),<sup><a reveal-id="DSW014C53" data-open="DSW014C53" class="link link-ref link-reveal xref-bibr">53</a></sup> which calculates EHH values from the same SNPs core between two populations using phased SNP strings. In both analyses, EHH values collected in 1,000-kb sections from each SNP locus showed significant reductions (<em>ca.</em> 0.05).</p><p>The iHS in domesticated peach, as well as the two subpopulations F and O-A, showed specific patterns of significant peaks (<em>P</em> &lt; 0.0001) corresponding to putative selective sweeps (Fig. <a reveal-id="DSW014F5" data-open="DSW014F5" class="link link-reveal link-table xref-fig">5</a>A, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S5 and S6</a></span>A). The patterns of the peaks for the selective sweep in iHS were considerably different from those in the SFS-based analysis (Fig. <a reveal-id="DSW014F4" data-open="DSW014F4" class="link link-reveal link-table xref-fig">4</a>), perhaps because the iHS detects mainly selected alleles in the heterozygous state, whereas the SFS-based analysis is applicable mainly to the detection of homozygosity. Still, the same positions for putative selection in SFS-based and iHS analyses were selected on Chromosome 7 (<em>ca.</em> 8,000–8,800 kb) in the F cluster (<em>P</em> &lt; 0.00001 in iHS). The XP-EHH analyses were performed in comparisons between domesticated peach and the W clusters, and between the F or O-A and the L clusters (Fig. <a reveal-id="DSW014F5" data-open="DSW014F5" class="link link-reveal link-table xref-fig">5</a>B and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>B) to analyse genes selected in the course of domestication and cultivar differentiation. In the XP-EHH analysis of domesticated peach and the W clusters, the peaks were at the bottoms of Chromosome 4 (<em>ca.</em> 25,000–27,000 kb for <em>P</em> &lt; 0.001), and Chromosome 6 (<em>ca.</em> 25,000–26,500 kb for <em>P</em> &lt; 0.001). In XP-EHH analyses, they were similar to those in iHS analyses of the domesticated peach (Fig. <a reveal-id="DSW014F5" data-open="DSW014F5" class="link link-reveal link-table xref-fig">5</a>, <em>P</em> &lt; 0.000001 for chr. 4, and <em>P</em> &lt; 0.0001 for chr. 6). Some of the significant peaks in XP-EHH analyses were different from those in the iHS and SFS-based approaches. Similar situations have been reported in analyses using EHH- and SFS-based methods in human evolution, which captured certain selected regions in all approaches, further supporting the value of using multiple approaches for comprehensive analysis of positive selection.<sup><a reveal-id="DSW014C52" data-open="DSW014C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSW014C53" data-open="DSW014C53" class="link link-ref link-reveal xref-bibr">53</a></sup> The XP-EHH analyses of the F or O-A cluster relative to the L cluster also showed some specific peaks, though different from the iHS and SFS-based results. Significant peaks could still be detected on Chromosome 4 (<em>ca.</em> 8,500–9,500 kb) and Chromosome 7 (<em>ca.</em> 8,000–8,800 kb) in the F cluster (<em>P</em> &lt; 0.00001). The peak on Chromosome 4 was also detected in the domesticated peach cluster by the SFS-based method (Fig. <a reveal-id="DSW014F4" data-open="DSW014F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="87200564" scrollto-destination="87200564"></a>
<div id="DSW014F5" data-id="DSW014F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01405.png?Expires=2147483647&Signature=351lQX9oRh2bGjYhV~fX5GXWiZoFYx~aXM0I2ygc3BPeOQxsYtesP~wSUcDhqhETZnI9vMNu~xk~qymBXZ5-1i5VWf0ml~M9rYgr2ExG9lxbSDuP5V3l9vgCc-q607zuzLapvSgCJ-LGE31SQDYnxs~qO6XgaGISyYVQ5irz2Uz254r-Zzesax1spJCFSjuB4FWRs9zh4f85BJ3rBxTRnqpJF34xhveHn8sFX3MVqFUqrbFHpsNuvfk9elt1hu1MIxhxL2A792Hf4PRaQpsygIvtFkgmLyCgGrV5KM2524TseR6xCs4NO7J-7R3g1Y0RVOs6GdTEiLfW8Z1Rt-RX2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide analysis for positive selection based on EHH. (A) iHS from each SNP core showing heterozygous states in one population transformed to P values with R and plotted as a logarithmic value. Underlining indicates threshold values for selection (P &lt; 0.0001). (B) XP-EHH from each SNP core showing the same nucleotide between the subject and the comparison target, also transformed to P values and plotted in logarithmic scale. For the domesticated peach, the wild relative cluster was used for comparison. For the F and O-A clusters, the landrace (L) cluster was independently used for comparison. Underlining indicates threshold values for selection (P &lt; 0.001). In both iHS and XP-EHH tests, putative selected regions are indicated by outlined triangles." path-from-xml="dsw01405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200564/dsw01405.tiff" path-from-xml="dsw01405.tiff" target="_blank">View large</a><a section="87200564" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01405.png?Expires=2147483647&Signature=v91pmr40iIqDHwlTzMpyWir6ONO77mocw9mIIMi3Uyh1jpmPQBQwJffrl28JwVRp8Z2c-5DwYDTvgFrA3zCkdwHqc1huEmGQrvf9ZUqvJ3XQLXrT~UJxMX5pAQzGvZk7tOL1Dfh9Hj9W5MwvIK6AA3u0bk4Q5QaW~qdLqd-wrbFkDQdAjzkiyoJPZzHryL6qV9jXa3fCfkEqfjV2BowzNkF3~JztpeJelOPk0KeIFcMkWjf2XADkK37c2eHXl4Xql9bXmKOUSDApMUuuXRzZW1QFZUFzvyRWw2pR7PU34SikkcohErjoqsbsKRvdtTXEaiJ9oNp0kTK8d-a3ust-WQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200564&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide analysis for positive selection based on EHH. (A) iHS from each SNP core showing heterozygous states in one population transformed to <em>P</em> values with <em>R</em> and plotted as a logarithmic value. Underlining indicates threshold values for selection (<em>P</em> &lt; 0.0001). (B) XP-EHH from each SNP core showing the same nucleotide between the subject and the comparison target, also transformed to <em>P</em> values and plotted in logarithmic scale. For the domesticated peach, the wild relative cluster was used for comparison. For the F and O-A clusters, the landrace (L) cluster was independently used for comparison. Underlining indicates threshold values for selection (<em>P</em> &lt; 0.001). In both iHS and XP-EHH tests, putative selected regions are indicated by outlined triangles.</p></div></div><div content-id="DSW014F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/m_dsw01405.png?Expires=2147483647&Signature=351lQX9oRh2bGjYhV~fX5GXWiZoFYx~aXM0I2ygc3BPeOQxsYtesP~wSUcDhqhETZnI9vMNu~xk~qymBXZ5-1i5VWf0ml~M9rYgr2ExG9lxbSDuP5V3l9vgCc-q607zuzLapvSgCJ-LGE31SQDYnxs~qO6XgaGISyYVQ5irz2Uz254r-Zzesax1spJCFSjuB4FWRs9zh4f85BJ3rBxTRnqpJF34xhveHn8sFX3MVqFUqrbFHpsNuvfk9elt1hu1MIxhxL2A792Hf4PRaQpsygIvtFkgmLyCgGrV5KM2524TseR6xCs4NO7J-7R3g1Y0RVOs6GdTEiLfW8Z1Rt-RX2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide analysis for positive selection based on EHH. (A) iHS from each SNP core showing heterozygous states in one population transformed to P values with R and plotted as a logarithmic value. Underlining indicates threshold values for selection (P &lt; 0.0001). (B) XP-EHH from each SNP core showing the same nucleotide between the subject and the comparison target, also transformed to P values and plotted in logarithmic scale. For the domesticated peach, the wild relative cluster was used for comparison. For the F and O-A clusters, the landrace (L) cluster was independently used for comparison. Underlining indicates threshold values for selection (P &lt; 0.001). In both iHS and XP-EHH tests, putative selected regions are indicated by outlined triangles." path-from-xml="dsw01405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200564/dsw01405.tiff" path-from-xml="dsw01405.tiff" target="_blank">View large</a><a section="87200564" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw01405.png?Expires=2147483647&Signature=v91pmr40iIqDHwlTzMpyWir6ONO77mocw9mIIMi3Uyh1jpmPQBQwJffrl28JwVRp8Z2c-5DwYDTvgFrA3zCkdwHqc1huEmGQrvf9ZUqvJ3XQLXrT~UJxMX5pAQzGvZk7tOL1Dfh9Hj9W5MwvIK6AA3u0bk4Q5QaW~qdLqd-wrbFkDQdAjzkiyoJPZzHryL6qV9jXa3fCfkEqfjV2BowzNkF3~JztpeJelOPk0KeIFcMkWjf2XADkK37c2eHXl4Xql9bXmKOUSDApMUuuXRzZW1QFZUFzvyRWw2pR7PU34SikkcohErjoqsbsKRvdtTXEaiJ9oNp0kTK8d-a3ust-WQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200564&ar=1745617&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide analysis for positive selection based on EHH. (A) iHS from each SNP core showing heterozygous states in one population transformed to <em>P</em> values with <em>R</em> and plotted as a logarithmic value. Underlining indicates threshold values for selection (<em>P</em> &lt; 0.0001). (B) XP-EHH from each SNP core showing the same nucleotide between the subject and the comparison target, also transformed to <em>P</em> values and plotted in logarithmic scale. For the domesticated peach, the wild relative cluster was used for comparison. For the F and O-A clusters, the landrace (L) cluster was independently used for comparison. Underlining indicates threshold values for selection (<em>P</em> &lt; 0.001). In both iHS and XP-EHH tests, putative selected regions are indicated by outlined triangles.</p></div></div>                            <h3 scrollto-destination=87200567 id="87200567" class="section-title" data-legacy-id=s3d>3.4. Candidate genes under selection</h3>
<p>For the F and O-A clusters, genetic diversity of candidate genes located on representative regions under putative strong positive selection was estimated. These included Chromosome 7 (8,000–8,800 kb) for the F cluster, Chromosomes 4 (29,200–29,400 kb) and 8 (16,900–17,550 kb) for the O-A cluster, and Chromosome 1 for both (1,200–1,500 kb), using the 30 domesticated peach accessions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Note that the focus was only on genes annotated with functions possibly conferring advantages of differentiation. Of these, ppa004528m, ppa016246m, and ppa025156m showed significant reduction in genetic diversity in the F cluster (π = 0–0.00007) against the O-A and landrace (L-EA and L-OT) clusters (average π = 0.00114 ± 0.00021, 0.00279 ± 0.00011, and 0.00629 ± 0.00091, respectively, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). However, no disruptions or substitutions clearly associated with the functional changes in the ORF sequences could be identified, although minor substitutions on protein sequences were observed in comparison to some landraces. For the O-A cluster, ppa003808m showed a significant reduction in genetic diversity (π = 0) in comparison to the other clusters (average π = 0.00150 ± 0.00017, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). ppa003808m retained a trihelix motif, showing high similarity to the <em>PETALLOSS</em> (<em>PTL</em>) gene, which is involved in petal size and morphology in <em>Arabidopsis thaliana</em>.<sup><a reveal-id="DSW014C64  DSW014C66" data-open="DSW014C64  DSW014C66" class="link link-ref link-reveal xref-bibr">64–66</a></sup></p>                            <h2 scrollto-destination=87200571 id="87200571" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=87200573 id="87200573" class="section-title" data-legacy-id=s4a>4.1. Domestication path in peach</h3>
<p>The domestication events for major annual crops, such as maize or rice, have been well characterized, owing in part to the relatively straightforward selection for consumable parts as well as the genetic fixation of key controlling genes in distinct species or cultivars. Tree crops appear to have more complicated domestication histories.<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSW014C22" data-open="DSW014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> For example, landrace accessions of <em>P. persica</em>, such as Kemomo Nagoshijou or Kutao, which show wild species-like characteristics, share a similar genetic structure with cultivated peach accessions. Thus, it may be difficult to define a single domestication path. Two scenarios are supported by the data: (i) current wild peach cultivars (feral peaches) were derived from divergent lines after a general domestication event or (ii) multiple independent domestications occurred for each trait or region. The first scenario (i) would fit well with the premise that <em>P. persica</em> was first domesticated from a wild species in China before diffusion to other regions.<sup><a reveal-id="DSW014C28" data-open="DSW014C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSW014C29" data-open="DSW014C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Such a general domestication event is supported by the tendencies for population bottlenecks or inbreeding to be detected using the full population of peach (<em>P. persica</em>) in this study. In contrast, other domesticated perennial crops such as grape and apple are reported to lack narrow domestication bottlenecks.<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSW014C22" data-open="DSW014C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSW014C67" data-open="DSW014C67" class="link link-ref link-reveal xref-bibr">67</a></sup> However, in theory, perennial woody crops are under strong bottleneck pressures during domestication, resulting from the ease of their clonal propagation as well as difficulties associated with their long generation time. The wide domestication bottleneck in grape might be explained by widespread use of vegetative propagation of a large number of landraces with a resulting maintenance of genetic diversity as well as of many local genetic clusters.<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> This situation would be more aligned with the second scenario (ii). The results of IBD analysis in peach showed that, except for the F cluster, most cultivars have no first- or second-degree relationships. This situation is distinct from that of domesticated grapevine, which shows a high ratio of first-degree relationships with a relatively small number of representative cultivars.<sup><a reveal-id="DSW014C18" data-open="DSW014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> This finding indicates that there were not many opportunities for domestication or construction of local clusters in peach, at least for the cultivars examined in this study.</p>                            <h3 scrollto-destination=87200577 id="87200577" class="section-title" data-legacy-id=s4b>4.2. Selection in peach domestication and cultivar differentiation</h3>
<p>Ancient selection in the peach genome appears likely, although some observations may be due to genetic drift. Results from QTL analysis have also been used to support selection during domestication and cultivar differentiation in rice<sup><a reveal-id="DSW014C8" data-open="DSW014C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and chicken<sup><a reveal-id="DSW014C39" data-open="DSW014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> In the present study, the strong putative selection on Chromosome 4 (<em>ca</em> 8,500–9,500 kb) detected in modern fruit cultivars (the F cluster) is consistent with a major QTL (UDP96-003) for important domestication traits, including fruit size and maturity date previously reported by Quilot et al. in a backcross between cultivated peach and its wild relative <em>P. davidiana</em>.<sup><a reveal-id="DSW014C68" data-open="DSW014C68" class="link link-ref link-reveal xref-bibr">68</a></sup> This finding suggests that favourable allele(s) in this region had been selected from wild ancestors during domestication. Furthermore, evidence for selection is observed at the bottom of Chromosome 6 (<em>ca.</em> 25,000–26,500 kb) which corresponds to the genomic regions XP-EHH and iHS, known to contain the self-incompatibility (<em>S</em>) controlling haplotype, including an F-box gene as the pollen-<em>S</em> (<em>SFB</em>) factor and an RNase gene as the pistil-<em>S</em> (<em>S</em>-RNase) factor in <em>Prunus</em> species.<sup><a reveal-id="DSW014C69" data-open="DSW014C69" class="link link-ref link-reveal xref-bibr">69</a>,<a reveal-id="DSW014C70" data-open="DSW014C70" class="link link-ref link-reveal xref-bibr">70</a></sup> Previously, it has been suggested that the whole domesticated peach (<em>P. persica</em>), including all clusters apart from the W cluster in this study, is a self-compatible (SC) species with at least four forms of pollen-<em>S</em>, <em>PpSFB</em><sub>1–4</sub>, disruptive mutations.<sup><a reveal-id="DSW014C71" data-open="DSW014C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DSW014C72" data-open="DSW014C72" class="link link-ref link-reveal xref-bibr">72</a></sup> Meanwhile, to some extent, the wild species (W cluster) in the <em>Amygdalus</em> subgenus are supposed to demonstrate SI.<sup><a reveal-id="DSW014C73" data-open="DSW014C73" class="link link-ref link-reveal xref-bibr">73</a></sup> In general, a shift from SI to SC is strongly selected during domestication to facilitate the stable production of inbred lines.<sup><a reveal-id="DSW014C27" data-open="DSW014C27" class="link link-ref link-reveal xref-bibr">27</a></sup> In contrast, a change in the mating system from SI to SC could have a strong influence on the pattern of polymorphisms, affecting genetic diversity and LD. This might result in the differences in genetic structures between wild species and domesticated peach.</p><p>At least three candidates for selection were identified in the F cluster, with many on Chromosome 7. The candidates, ppa004528m, ppa016246m, and ppa025156m, show high similarity to <em>lycopene cyclase</em>-like At3g10230 (<em>LYC</em>)/At5g57030 (<em>LUT2</em>), At3g18030 (<em>Arabidopsis thaliana Hal3-like protein A</em>; <em>AtHAL3A</em>), and At1g18990 (<em>reduced vernalization response 1</em>; <em>VRN1</em>) in the <em>Arabidopsis thaliana</em> genome. Lycopene cyclases (LYCs) play a role in the biosynthesis of lutein, which is a member of the carotenoid pathway.<sup><a reveal-id="DSW014C74" data-open="DSW014C74" class="link link-ref link-reveal xref-bibr">74</a></sup> The simplest explanation for the selection of ppa004528m may be its association with the yellow carotenoid pigmentation in peach. However, the flesh coloration of peach is determined mainly by the <em>Y</em> locus, which is reportedly located on Chromosome 1 (LG1).<sup><a reveal-id="DSW014C75" data-open="DSW014C75" class="link link-ref link-reveal xref-bibr">75</a>,<a reveal-id="DSW014C76" data-open="DSW014C76" class="link link-ref link-reveal xref-bibr">76</a></sup> Major QTLs for skin color are also different from those in most regions identified as having experienced selective sweeps, although one of them is located on Chromosome 7.<sup><a reveal-id="DSW014C77" data-open="DSW014C77" class="link link-ref link-reveal xref-bibr">77</a></sup> The other two candidates, <em>AtHal3A</em>-like and <em>VRN1</em>-like genes, may contribute salt/osmotic tolerance and flowering/bud-burst timing, respectively, based on the functions of their homologs in <em>Arabidopsis</em>.<sup><a reveal-id="DSW014C78" data-open="DSW014C78" class="link link-ref link-reveal xref-bibr">78</a>,<a reveal-id="DSW014C79" data-open="DSW014C79" class="link link-ref link-reveal xref-bibr">79</a></sup> In considering the uses and characteristics of modern fruit cultivars, we may expect that some genes directly involved in fruit traits, such as fruit size, sugar contents, or the important flesh-softening trait often called ‘melting texture’, have been under strong selection. Where selection for more ornamental traits is expected, as in the O-A cluster, ppa003808m appears to be a good candidate, given that the associated <em>PETALLOSS</em> trait is known to affect morphological or architectural changes, particularly for flowers.</p><p>Genes with annotation for fruit quality and ornamental value are limited at present, presumably because of the lack of information for these traits in model plant species. Thus, the still-uncharacterized genes located in the selected regions in the F and O-A clusters may provide opportunities for elucidating critical tree domestication factors not present in current annual plant models. Genes controlling such key domestication and cultivar differentiation factors could be identified by improved characterization of the selection pathways involved, perhaps by further exploiting the wide genetic diversity present in peach and similar perennial tree crops. This research approach could prove powerful, particularly for long-lived perennial crops where mutagenetic and map-based approaches are restricted.</p>                            <h2 scrollto-destination=87200585 id="87200585" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by a Grant-in-Aid for Research Activity Start-up from the Japan Society for the Promotion of Science (number 23880013 to T.A.). Funding to pay the Open Access publication charges for this article was provided by Grant-in-Aid for Young Scientists (A) (no. 26712005 to TA) from JSPS.</p>                            <h2 scrollto-destination=87200589 id="87200589" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr Masanori Yamasaki, Food Resources Education and Research Center, Graduate School of Agricultural Science, Kobe University (Kasai, Hyogo 675-2103, Japan) for many critical suggestions on experimental design, and to Drs Luca Comai and Isabelle M. Henry, Genome Center, University of California Davis (Davis, CA, USA) for bioinformatic support. We gratefully acknowledge gifts of plant materials from Yukio Sasabe, Research Institute for Agriculture Okayama Prefectural Technology Center for Agriculture, Forestry, and Fisheries (Akaiwa, Japan).</p>                            <h2 scrollto-destination=87200591 id="87200591" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW014C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zeder</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Emshwiller</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">B.D.</div></div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">D.G.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Documenting domestication: the intersection of genetics and archaeology</div>, <div class="source">Trends Genet.</div> , <div class="volume">22</div>, <div class="fpage">139</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Documenting%20domestication%3A%20the%20intersection%20of%20genetics%20and%20archaeology&amp;author=M.A.%20Zeder&amp;author=E.%20Emshwiller&amp;author=B.D.%20Smith&amp;author=D.G.%20Bradley&amp;publication_year=2006&amp;journal=Trends%20Genet.&amp;volume=22&amp;pages=139-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2006.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2006.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2006.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16458995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Doebley</div>
               <div class="given-names">J.F.</div>
            </div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">B.D.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">The molecular genetics of crop domestication</div>, <div class="source">Cell</div> , <div class="volume">127</div>, <div class="fpage">1309</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20genetics%20of%20crop%20domestication&amp;author=J.F.%20Doebley&amp;author=B.S.%20Gaut&amp;author=B.D.%20Smith&amp;publication_year=2006&amp;journal=Cell&amp;volume=127&amp;pages=1309-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17190597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Caicedo</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Williamson</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Hernandez</div> <div class="given-names">R.D.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Genome-wide patterns of nucleotide polymorphism in domesticated rice</div>, <div class="source">PLoS Genet.</div> , <div class="volume">3</div>, <div class="fpage">e163</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20patterns%20of%20nucleotide%20polymorphism%20in%20domesticated%20rice&amp;author=A.L.%20Caicedo&amp;author=S.H.%20Williamson&amp;author=R.D.%20Hernandez&amp;publication_year=2007&amp;journal=PLoS%20Genet.&amp;volume=3&amp;pages=e163" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030163" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030163" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030163"> </span></div> </div></div></div></div></div><div content-id="DSW014C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Allaby</div>
               <div class="given-names">R.G.</div>
            </div>, <div class="name string-name"><div class="surname">Fuller</div> <div class="given-names">D.Q.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">T.A.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">The genetic expectations of a protracted model for the origins of domesticated crops</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="comment">107</div>, <div class="volume">105</div>, <div class="fpage">13982</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetic%20expectations%20of%20a%20protracted%20model%20for%20the%20origins%20of%20domesticated%20crops&amp;author=R.G.%20Allaby&amp;author=D.Q.%20Fuller&amp;author=T.A.%20Brown&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=13982-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0803780105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0803780105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0803780105"> </span></div> </div></div></div></div></div><div content-id="DSW014C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gross</div>
               <div class="given-names">B.L.</div>
            </div>, <div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">K.M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Genetic perspectives on crop domestication</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">15</div>, <div class="fpage">529</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20perspectives%20on%20crop%20domestication&amp;author=B.L.%20Gross&amp;author=K.M.%20Olsen&amp;publication_year=2010&amp;journal=Trends%20Plant%20Sci.&amp;volume=15&amp;pages=529-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.05.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.05.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.05.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20541451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamasaki</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">S.I.</div></div>, <div class="name string-name"><div class="surname">McMullen</div> <div class="given-names">M.D.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Genomic screening for artificial selection during domestication and improvement in maize</div>, <div class="source">Ann. Bot.</div> , <div class="volume">100</div>, <div class="fpage">967</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20screening%20for%20artificial%20selection%20during%20domestication%20and%20improvement%20in%20maize&amp;author=M.%20Yamasaki&amp;author=S.I.%20Wright&amp;author=M.D.%20McMullen&amp;publication_year=2007&amp;journal=Ann.%20Bot.&amp;volume=100&amp;pages=967-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcm173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcm173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcm173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17704539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nielsen</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Hellmann</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hubisz</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bustamante</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">A.G.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Recent and ongoing selection in the human genome</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">8</div>, <div class="fpage">857</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20and%20ongoing%20selection%20in%20the%20human%20genome&amp;author=R.%20Nielsen&amp;author=I.%20Hellmann&amp;author=M.%20Hubisz&amp;author=C.%20Bustamante&amp;author=A.G.%20Clark&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=8&amp;pages=857-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17943193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Asano</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Yamasaki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Takuno</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Artificial selection for a green revolution gene during japonica rice domestication</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">11034</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Artificial%20selection%20for%20a%20green%20revolution%20gene%20during%20japonica%20rice%20domestication&amp;author=K.%20Asano&amp;author=M.%20Yamasaki&amp;author=S.%20Takuno&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=11034-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1019490108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1019490108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1019490108"> </span></div> </div></div></div></div></div><div content-id="DSW014C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes</div>, <div class="source">Nat. Biotech.</div> , <div class="volume">30</div>, <div class="fpage">105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%2050%20accessions%20of%20cultivated%20and%20wild%20rice%20yields%20markers%20for%20identifying%20agronomically%20important%20genes&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;author=S.%20Ge&amp;publication_year=2012&amp;journal=Nat.%20Biotech.&amp;volume=30&amp;pages=105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2050"> </span></div> </div></div></div></div></div><div content-id="DSW014C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cavanagh</div>
               <div class="given-names">C.R.</div>
            </div>, <div class="name string-name"><div class="surname">Chao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">8057</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20diversity%20uncovers%20multiple%20targets%20of%20selection%20for%20improvement%20in%20hexaploid%20wheat%20landraces%20and%20cultivars&amp;author=C.R.%20Cavanagh&amp;author=S.%20Chao&amp;author=S.%20Wang&amp;publication_year=2013&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=8057-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1217133110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1217133110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1217133110"> </span></div> </div></div></div></div></div><div content-id="DSW014C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clark</div>
               <div class="given-names">R.M.</div>
            </div>, <div class="name string-name"><div class="surname">Linton</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Messing</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Doebley</div> <div class="given-names">J.F.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Pattern of diversity in the genomic region near the maize domestication gene <em>tb1</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">101</div>, <div class="fpage">700</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pattern%20of%20diversity%20in%20the%20genomic%20region%20near%20the%20maize%20domestication%20gene%20tb1&amp;author=R.M.%20Clark&amp;author=E.%20Linton&amp;author=J.%20Messing&amp;author=J.F.%20Doebley&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=700-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.2237049100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.2237049100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.2237049100"> </span></div> </div></div></div></div></div><div content-id="DSW014C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Palaisa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tingey</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Long-range patterns of diversity and linkage disequilibrium surrounding the maize Y1 gene are indicative of an asymmetric selective sweep</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">101</div>, <div class="fpage">9885</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-range%20patterns%20of%20diversity%20and%20linkage%20disequilibrium%20surrounding%20the%20maize%20Y1%20gene%20are%20indicative%20of%20an%20asymmetric%20selective%20sweep&amp;author=K.%20Palaisa&amp;author=M.%20Morgante&amp;author=S.%20Tingey&amp;author=A.%20Rafalski&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=9885-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307839101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307839101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307839101"> </span></div> </div></div></div></div></div><div content-id="DSW014C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wright</div>
               <div class="given-names">S.I.</div>
            </div>, <div class="name string-name"><div class="surname">Bi</div> <div class="given-names">I.V.</div></div>, <div class="name string-name"><div class="surname">Schroeder</div> <div class="given-names">S.G.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">The effects of artificial selection on the maize genome</div>, <div class="source">Science</div> , <div class="volume">308</div>, <div class="fpage">1310</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effects%20of%20artificial%20selection%20on%20the%20maize%20genome&amp;author=S.I.%20Wright&amp;author=I.V.%20Bi&amp;author=S.G.%20Schroeder&amp;publication_year=2005&amp;journal=Science&amp;volume=308&amp;pages=1310-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1107891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1107891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1107891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15919994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamasaki</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Tenaillon</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Bi</div> <div class="given-names">I.V.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">A large-scale screen for artificial selection in maize identifies candidate agronomic loci for domestication and crop improvement</div>, <div class="source">Plant Cell</div> , <div class="volume">17</div>, <div class="fpage">2859</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large-scale%20screen%20for%20artificial%20selection%20in%20maize%20identifies%20candidate%20agronomic%20loci%20for%20domestication%20and%20crop%20improvement&amp;author=M.%20Yamasaki&amp;author=M.I.%20Tenaillon&amp;author=I.V.%20Bi&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=2859-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.037242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.037242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.037242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16227451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morrell</div>
               <div class="given-names">P.L.</div>
            </div>, <div class="name string-name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Ross-Ibarra</div> <div class="given-names">J.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Crop genomics: advances and applications</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">13</div>, <div class="fpage">85</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crop%20genomics%3A%20advances%20and%20applications&amp;author=P.L.%20Morrell&amp;author=E.S.%20Buckler&amp;author=J.%20Ross-Ibarra&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=13&amp;pages=85-96" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22207165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Heffner</div>
               <div class="given-names">E.L.</div>
            </div>, <div class="name string-name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Jannink</div> <div class="given-names">J.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Genomic selection for crop improvement</div>, <div class="source">Crop Sci.</div> , <div class="volume">49</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20selection%20for%20crop%20improvement&amp;author=E.L.%20Heffner&amp;author=M.E.%20Sorrells&amp;author=J.L.%20Jannink&amp;publication_year=2009&amp;journal=Crop%20Sci.&amp;volume=49&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2008.08.0512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2008.08.0512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2008.08.0512"> </span></div> </div></div></div></div></div><div content-id="DSW014C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McCarthy</div>
               <div class="given-names">M.I.</div>
            </div>, <div class="name string-name"><div class="surname">Abecasis</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Cardon</div> <div class="given-names">L.R.</div></div>, <div class="name string-name"><div class="surname">Goldstein</div> <div class="given-names">D.B.</div></div>, <div class="name string-name"><div class="surname">Little</div> <div class="given-names">J.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Genome-wide association studies for complex traits: consensus, uncertainty and challenges</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">9</div>, <div class="fpage">356</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20for%20complex%20traits%3A%20consensus%2C%20uncertainty%20and%20challenges&amp;author=M.I.%20McCarthy&amp;author=G.R.%20Abecasis&amp;author=L.R.%20Cardon&amp;author=D.B.%20Goldstein&amp;author=J.%20Little&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=9&amp;pages=356-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18398418" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Myles</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Boyko</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Owens</div> <div class="given-names">C.L.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Genetic structure and domestication history of the grape</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">3530</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20domestication%20history%20of%20the%20grape&amp;author=S.%20Myles&amp;author=A.R.%20Boyko&amp;author=C.L.%20Owens&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=3530-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1009363108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1009363108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1009363108"> </span></div> </div></div></div></div></div><div content-id="DSW014C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Morrell</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Ashworth</div> <div class="given-names">V.E.T.M.</div></div>, <div class="name string-name"><div class="surname">de la Cruz</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Clegg</div> <div class="given-names">M.T.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Tracing the geographic origins of major avocado cultivars</div>, <div class="source">J. Hered.</div> , <div class="volume">100</div>, <div class="fpage">56.65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracing%20the%20geographic%20origins%20of%20major%20avocado%20cultivars&amp;author=H.%20Chen&amp;author=P.L.%20Morrell&amp;author=V.E.T.M.%20Ashworth&amp;author=M.%20de%20la%20Cruz&amp;author=M.T.%20Clegg&amp;publication_year=2009&amp;journal=J.%20Hered.&amp;volume=100&amp;pages=56.65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18779226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Miller</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Schaal</div> <div class="given-names">B.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Domestication of a Mesoamerican cultivated fruit tree, <em>Spondias purpurea</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">102</div>, <div class="fpage">12801</div>–<div class="lpage">06</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Domestication%20of%20a%20Mesoamerican%20cultivated%20fruit%20tree%2C%20Spondias%20purpurea&amp;author=A.%20Miller&amp;author=B.%20Schaal&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=12801-06" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0505447102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0505447102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0505447102"> </span></div> </div></div></div></div></div><div content-id="DSW014C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Miller</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Gross</div> <div class="given-names">B.L.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">From forest to field: perennial fruit crops domestication</div>, <div class="source">Am. J. Bot.</div> , <div class="volume">98</div>, <div class="fpage">1389</div>–<div class="lpage">414</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20forest%20to%20field%3A%20perennial%20fruit%20crops%20domestication&amp;author=A.%20Miller&amp;author=B.L.%20Gross&amp;publication_year=2011&amp;journal=Am.%20J.%20Bot.&amp;volume=98&amp;pages=1389-414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1000522" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1000522" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1000522"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21865506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cornille</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Gladieux</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Smulders</div> <div class="given-names">M.J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">New insight into the history of domesticated apple: secondary contribution of the European wild apple to the genome of cultivated varieties</div>, <div class="source">PLoS Genet.</div> , <div class="volume">8</div>, <div class="fpage">e1002703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20insight%20into%20the%20history%20of%20domesticated%20apple%3A%20secondary%20contribution%20of%20the%20European%20wild%20apple%20to%20the%20genome%20of%20cultivated%20varieties&amp;author=A.%20Cornille&amp;author=P.%20Gladieux&amp;author=M.J.%20Smulders&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002703" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002703" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002703"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22589740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Austerlitz</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Mariette</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Machon</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Gouyon</div> <div class="given-names">P.H.</div></div>, <div class="name string-name"><div class="surname">Godelle</div> <div class="given-names">B.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Effects of colonization processes on genetic diversity: differences between annual plants and tree species</div>, <div class="source">Genetics</div> , <div class="volume">154</div>, <div class="fpage">1309</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20colonization%20processes%20on%20genetic%20diversity%3A%20differences%20between%20annual%20plants%20and%20tree%20species&amp;author=F.%20Austerlitz&amp;author=S.%20Mariette&amp;author=N.%20Machon&amp;author=P.H.%20Gouyon&amp;author=B.%20Godelle&amp;publication_year=2000&amp;journal=Genetics&amp;volume=154&amp;pages=1309-21" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10757772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Petit</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Hampe</div> <div class="given-names">A.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Some evolutionary consequences of being a tree</div>, <div class="source">Annu. Rev. Ecol. Evol. Syst.</div> , <div class="volume">37</div>, <div class="fpage">187</div>–<div class="lpage">214</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Some%20evolutionary%20consequences%20of%20being%20a%20tree&amp;author=R.J.%20Petit&amp;author=A.%20Hampe&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Ecol.%20Evol.%20Syst.&amp;volume=37&amp;pages=187-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ecolsys.37.091305.110215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ecolsys.37.091305.110215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ecolsys.37.091305.110215"> </span></div> </div></div></div></div></div><div content-id="DSW014C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Savolainen</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Pyhäjärvi</div> <div class="given-names">T.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Genomic diversity in forest trees</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">10</div>, <div class="fpage">162</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20diversity%20in%20forest%20trees&amp;author=O.%20Savolainen&amp;author=T.%20Pyh%C3%A4j%C3%A4rvi&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=10&amp;pages=162-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2007.01.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2007.01.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2007.01.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17292660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Glémin</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Bataillon</div> <div class="given-names">T.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">A comparative view of the evolution of grasses under domestication</div>, <div class="source">New Phytol.</div> , <div class="volume">183</div>, <div class="fpage">273</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20view%20of%20the%20evolution%20of%20grasses%20under%20domestication&amp;author=S.%20Gl%C3%A9min&amp;author=T.%20Bataillon&amp;publication_year=2009&amp;journal=New%20Phytol.&amp;volume=183&amp;pages=273-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2009.02884.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2009.02884.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2009.02884.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19515223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rowlands</div>
               <div class="given-names">D.G.</div>
            </div>
          </div>
          <div class="year">1964</div>, <div class="article-title">Self-incompatibility in sexually propagated cultivated plants</div>, <div class="source">Euphytica</div> , <div class="volume">13</div>, <div class="fpage">157</div>–<div class="lpage">62</div>.</div></div></div></div><div content-id="DSW014C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Faust</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Timon</div> <div class="given-names">B.</div></div></div>
          <div class="year">1995</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">Janick</div> <div class="given-names">J.</div></div></div> (ed) <div class="source">Origin and dissemination of peach, Hort. Rev.</div> , Vol. <div class="volume">17</div>, <div class="publisher-name">Wiley</div>: <div class="publisher-loc">New York</div>, pp. <div class="fpage">331</div>–<div class="lpage">80</div>.</div></div></div></div><div content-id="DSW014C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2008</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">Layne</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Bassi</div> <div class="given-names">D.</div></div></div> (eds) <div class="source">History of cultivation and trends in China, the peach: botany production and uses</div> , <div class="publisher-name">CAB International</div>, pp. <div class="fpage">37</div>–<div class="lpage">60</div>.</div></div></div></div><div content-id="DSW014C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cao</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Comparative population genomics reveals the domestication history of the peach, <em>Prunus persica</em>, and human influences on perennial fruit crops</div>, <div class="source">Genome Biol.</div> , <div class="volume">15</div>, <div class="fpage">415</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20population%20genomics%20reveals%20the%20domestication%20history%20of%20the%20peach%2C%20Prunus%20persica%2C%20and%20human%20influences%20on%20perennial%20fruit%20crops&amp;author=K.%20Cao&amp;author=Z.%20Zheng&amp;author=L.%20Wang&amp;publication_year=2014&amp;journal=Genome%20Biol.&amp;volume=15&amp;pages=415" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25079967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Scorza</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Mehlenbacher</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Lightner</div> <div class="given-names">G.W.</div></div></div>
          <div class="year">1985</div>, <div class="article-title">Inbreeding and coancestry of freestone peach cultivars of the eastern United States and implications for peach germplasm improvement</div>, <div class="source">J. Amer. Soc. Hort. Sci.</div> , <div class="volume">110</div>, <div class="fpage">547</div>–<div class="lpage">52</div>.</div></div></div></div><div content-id="DSW014C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Warburton</div>
               <div class="given-names">M.L.</div>
            </div>, <div class="name string-name"><div class="surname">Bliss</div> <div class="given-names">F.A.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Genetic diversity in peach (<em>Prunus persica</em> L. Batch) revealed by randomly amplified polymorphic DNA (RAPD) markers and compared to inbreeding coefficients</div>, <div class="source">J. Am. Soc. Hortic. Sci.</div> , <div class="volume">121</div>, <div class="fpage">1012</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="DSW014C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Aranzana</div>
               <div class="given-names">M.J.</div>
            </div>, <div class="name string-name"><div class="surname">Abbassi</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Howad</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Arús</div> <div class="given-names">P.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Genetic variation, population structure and linkage disequilibrium in peach commercial varieties</div>, <div class="source">BMC Genet.</div> , <div class="volume">11</div>, <div class="fpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%2C%20population%20structure%20and%20linkage%20disequilibrium%20in%20peach%20commercial%20varieties&amp;author=M.J.%20Aranzana&amp;author=E.%20Abbassi&amp;author=W.%20Howad&amp;author=P.%20Ar%C3%BAs&amp;publication_year=2010&amp;journal=BMC%20Genet.&amp;volume=11&amp;pages=69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-11-69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-11-69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-11-69"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20646280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">Z.P.</div>
            </div>
          </div>
          <div class="year">2007</div>, <div class="article-title">Genetic characterization of different demes in <em>Prunus persica</em> revealed by RAPD markers</div>, <div class="source">Sci. Hortic.</div> , <div class="volume">111</div>, <div class="fpage">242</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20characterization%20of%20different%20demes%20in%20Prunus%20persica%20revealed%20by%20RAPD%20markers&amp;author=Z.P.%20Cheng&amp;publication_year=2007&amp;journal=Sci.%20Hortic.&amp;volume=111&amp;pages=242-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2006.10.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2006.10.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2006.10.020"> </span></div> </div></div></div></div></div><div content-id="DSW014C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">Z.P.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">H.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">SSR fingerprinting Chinese peach cultivars and landraces (<em>Prunus persica</em>) and analysis of their genetic relationships</div>, <div class="source">Sci. Hortic.</div> , <div class="volume">120</div>, <div class="fpage">188</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SSR%20fingerprinting%20Chinese%20peach%20cultivars%20and%20landraces%20(Prunus%20persica)%20and%20analysis%20of%20their%20genetic%20relationships&amp;author=Z.P.%20Cheng&amp;author=H.%20Huang&amp;publication_year=2009&amp;journal=Sci.%20Hortic.&amp;volume=120&amp;pages=188-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2008.10.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2008.10.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2008.10.008"> </span></div> </div></div></div></div></div><div content-id="DSW014C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xie</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Chai</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Evaluation of the genetic diversity of Asian peach accessions using a selected set of SSR markers</div>, <div class="source">Sci. Hortic.</div> , <div class="volume">125</div>, <div class="fpage">622</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20the%20genetic%20diversity%20of%20Asian%20peach%20accessions%20using%20a%20selected%20set%20of%20SSR%20markers&amp;author=R.%20Xie&amp;author=X.%20Li&amp;author=M.%20Chai&amp;publication_year=2010&amp;journal=Sci.%20Hortic.&amp;volume=125&amp;pages=622-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2010.05.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2010.05.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2010.05.015"> </span></div> </div></div></div></div></div><div content-id="DSW014C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martinez-Gómez</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Arulsekar</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Potter</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gradziel</div> <div class="given-names">T.M.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Relationships among peach, almond, and related species as detected by simple sequence repeat marker</div>, <div class="source">J. Am. Soc. Hortic. Sci.</div> , <div class="volume">128</div>, <div class="fpage">667</div>–<div class="lpage">71</div>.</div></div></div></div><div content-id="DSW014C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hu</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Genetic relationship of ornamental peach determined using AFLP markers</div>, <div class="source">Hort. Sci.</div> , <div class="volume">40</div>, <div class="fpage">1782</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSW014C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rubin</div>
               <div class="given-names">C.J.</div>
            </div>, <div class="name string-name"><div class="surname">Zody</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Eriksson</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Whole-genome resequencing reveals loci under selection during chicken domestication</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">587</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20resequencing%20reveals%20loci%20under%20selection%20during%20chicken%20domestication&amp;author=C.J.%20Rubin&amp;author=M.C.%20Zody&amp;author=J.%20Eriksson&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=587-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08832" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08832" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08832"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20220755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Toomajian</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">T.T.</div></div>, <div class="name string-name"><div class="surname">Aranzana</div> <div class="given-names">M.J.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">A nonparametric test reveals selection for rapid flowering in the Arabidopsis genome</div>, <div class="source">PLoS Biol.</div> , <div class="volume">4</div>, <div class="fpage">e137</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20nonparametric%20test%20reveals%20selection%20for%20rapid%20flowering%20in%20the%20Arabidopsis%20genome&amp;author=C.%20Toomajian&amp;author=T.T.%20Hu&amp;author=M.J.%20Aranzana&amp;publication_year=2006&amp;journal=PLoS%20Biol.&amp;volume=4&amp;pages=e137" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16623598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hufford</div>
               <div class="given-names">M.B.</div>
            </div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">van Heerwaarden</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Comparative population genomics of maize domestication and improvement</div>, <div class="source">Nat. Genet.</div> , <div class="volume">44</div>, <div class="fpage">808</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20population%20genomics%20of%20maize%20domestication%20and%20improvement&amp;author=M.B.%20Hufford&amp;author=X.%20Xu&amp;author=J.%20van%20Heerwaarden&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=808-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morris</div>
               <div class="given-names">G.P.</div>
            </div>, <div class="name string-name"><div class="surname">Ramu</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Deshpande</div> <div class="given-names">S.P.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Population genomic and genome-wide association studies of agroclimatic traits in sorghum</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">453</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genomic%20and%20genome-wide%20association%20studies%20of%20agroclimatic%20traits%20in%20sorghum&amp;author=G.P.%20Morris&amp;author=P.%20Ramu&amp;author=S.P.%20Deshpande&amp;publication_year=2013&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=453-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1215985110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1215985110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1215985110"> </span></div> </div></div></div></div></div><div content-id="DSW014C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Verde</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Bassil</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Scalabrin</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Development and evaluation of a 9K SNP array for peach by internationally coordinated SNP detection and validation in breeding germplasm</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20evaluation%20of%20a%209K%20SNP%20array%20for%20peach%20by%20internationally%20coordinated%20SNP%20detection%20and%20validation%20in%20breeding%20germplasm&amp;author=I.%20Verde&amp;author=N.%20Bassil&amp;author=S.%20Scalabrin&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/annotation/33f1ba92-c304-4757-91aa-555de64a0768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fannotation%2F33f1ba92-c304-4757-91aa-555de64a0768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fannotation%2F33f1ba92-c304-4757-91aa-555de64a0768"> </span></div> </div></div></div></div></div><div content-id="DSW014C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="collab">International Peach Genome Initiative</div>, <div class="person-group"><div class="name string-name"><div class="surname">Verde</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Abbott</div> <div class="given-names">A.G.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">The high-quality draft genome of peach (<em>Prunus persica</em>) identifies unique patterns of genetic diversity, domestication and genome evolution</div>, <div class="source">Nat. Genet.</div> , <div class="volume">45</div>, <div class="fpage">487</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20high-quality%20draft%20genome%20of%20peach%20(Prunus%20persica)%20identifies%20unique%20patterns%20of%20genetic%20diversity%2C%20domestication%20and%20genome%20evolution&amp;author=I.%20Verde&amp;author=A.G.%20Abbott&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=487-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2586"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23525075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Purcell</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Neale</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Todd-Brown</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">81</div>, <div class="fpage">559</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd-Brown&amp;publication_year=2007&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=81&amp;pages=559-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/519795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F519795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F519795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17701901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Raymond</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Rousset</div> <div class="given-names">F.</div></div></div>
          <div class="year">1995</div>, <div class="article-title">GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism</div>, <div class="source">J. Hered.</div> , <div class="volume">86</div>, <div class="fpage">248</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GENEPOP%20(version%201.2)%3A%20population%20genetics%20software%20for%20exact%20tests%20and%20ecumenicism&amp;author=M.%20Raymond&amp;author=F.%20Rousset&amp;publication_year=1995&amp;journal=J.%20Hered.&amp;volume=86&amp;pages=248-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jhered.a111573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jhered.a111573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jhered.a111573"> </span></div> </div></div></div></div></div><div content-id="DSW014C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rousset</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">2008</div>, <div class="article-title">Genepop'007: a complete reimplementation of the Genepop software for Windows and <em>Linux</em></div>, <div class="source">Mol. Ecol. Resources</div> , <div class="volume">8</div>, <div class="fpage">103</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genepop'007%3A%20a%20complete%20reimplementation%20of%20the%20Genepop%20software%20for%20Windows%20and%20Linux&amp;author=F.%20Rousset&amp;publication_year=2008&amp;journal=Mol.%20Ecol.%20Resources&amp;volume=8&amp;pages=103-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1471-8286.2007.01931.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1471-8286.2007.01931.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1471-8286.2007.01931.x"> </span></div> </div></div></div></div></div><div content-id="DSW014C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pritchard</div>
               <div class="given-names">J.K.</div>
            </div>, <div class="name string-name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Inference of population structure using multilocus genotype data</div>, <div class="source">Genetics</div> , <div class="volume">155</div>, <div class="fpage">945</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gao</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Williamson</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bustamante</div> <div class="given-names">C.D.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">An MCMC approach for joint inference of population structure and inbreeding rates from multi-locus genotype data</div>, <div class="source">Genetics</div> , <div class="volume">176</div>, <div class="fpage">1635</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20MCMC%20approach%20for%20joint%20inference%20of%20population%20structure%20and%20inbreeding%20rates%20from%20multi-locus%20genotype%20data&amp;author=H.%20Gao&amp;author=S.%20Williamson&amp;author=C.D.%20Bustamante&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=1635-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.072371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.072371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.072371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Evanno</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Regnaut</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Goudet</div> <div class="given-names">J.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study</div>, <div class="source">Mol. Ecol.</div> , <div class="volume">14</div>, <div class="fpage">2611</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;publication_year=2005&amp;journal=Mol.%20Ecol.&amp;volume=14&amp;pages=2611-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2005.14.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2005.14.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2005.14.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Scheet</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">78</div>, <div class="fpage">629</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%20and%20flexible%20statistical%20model%20for%20large-scale%20population%20genotype%20data%3A%20applications%20to%20inferring%20missing%20genotypes%20and%20haplotypic%20phase&amp;author=P.%20Scheet&amp;author=M.%20Stephens&amp;publication_year=2006&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=78&amp;pages=629-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/502802" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F502802" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F502802"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16532393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sabeti</div>
               <div class="given-names">P.C.</div>
            </div>, <div class="name string-name"><div class="surname">Reich</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">J.M.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Detecting recent positive selection in the human genome from haplotype structure</div>, <div class="source">Nature</div> , <div class="volume">419</div>, <div class="fpage">832</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20recent%20positive%20selection%20in%20the%20human%20genome%20from%20haplotype%20structure&amp;author=P.C.%20Sabeti&amp;author=D.E.%20Reich&amp;author=J.M.%20Higgins&amp;publication_year=2002&amp;journal=Nature&amp;volume=419&amp;pages=832-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12397357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sabeti</div>
               <div class="given-names">P.C.</div>
            </div>, <div class="name string-name"><div class="surname">Varilly</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Fry</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Genome-wide detection and characterization of positive selection in human populations</div>, <div class="source">Nature</div> , <div class="volume">449</div>, <div class="fpage">913</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20detection%20and%20characterization%20of%20positive%20selection%20in%20human%20populations&amp;author=P.C.%20Sabeti&amp;author=P.%20Varilly&amp;author=B.%20Fry&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=913-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17943131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Voight</div>
               <div class="given-names">B.F.</div>
            </div>, <div class="name string-name"><div class="surname">Kudaravalli</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">A map of recent positive selection in the human genome</div>, <div class="source">PLoS Biol.</div> , <div class="volume">4</div>, <div class="fpage">e72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20map%20of%20recent%20positive%20selection%20in%20the%20human%20genome&amp;author=B.F.%20Voight&amp;author=S.%20Kudaravalli&amp;author=X.%20Wen&amp;author=J.K.%20Pritchard&amp;publication_year=2006&amp;journal=PLoS%20Biol.&amp;volume=4&amp;pages=e72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16494531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akagi</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Henry</div> <div class="given-names">I.M.</div></div>, <div class="name string-name"><div class="surname">Tao</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Comai</div> <div class="given-names">L.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">A Y-chromosome–encoded small RNA acts as a sex determinant in persimmons</div>, <div class="source">Science</div> , <div class="volume">346</div>, <div class="fpage">646</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Y-chromosome%E2%80%93encoded%20small%20RNA%20acts%20as%20a%20sex%20determinant%20in%20persimmons&amp;author=T.%20Akagi&amp;author=I.M.%20Henry&amp;author=R.%20Tao&amp;author=L.%20Comai&amp;publication_year=2014&amp;journal=Science&amp;volume=346&amp;pages=646-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1257225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1257225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1257225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25359977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Librado</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Rozas</div> <div class="given-names">J.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1451</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaSP%20v5%3A%20a%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data&amp;author=P.%20Librado&amp;author=J.%20Rozas&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1451-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19346325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gouy</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">27</div>, <div class="fpage">221</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SeaView%20version%204%3A%20a%20multiplatform%20graphical%20user%20interface%20for%20sequence%20alignment%20and%20phylogenetic%20tree%20building&amp;author=M.%20Gouy&amp;author=S.%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=27&amp;pages=221-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19854763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mather</div>
               <div class="given-names">K.A.</div>
            </div>, <div class="name string-name"><div class="surname">Caicedo</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Polato</div> <div class="given-names">N.R.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">The extent of linkage disequilibrium in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Genetics</div> , <div class="volume">177</div>, <div class="fpage">2223</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20extent%20of%20linkage%20disequilibrium%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=K.A.%20Mather&amp;author=A.L.%20Caicedo&amp;author=N.R.%20Polato&amp;publication_year=2007&amp;journal=Genetics&amp;volume=177&amp;pages=2223-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.079616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.079616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.079616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17947413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yan</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Shah</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Warburton</div> <div class="given-names">M.L.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Genetic characterization and linkage disequilibrium estimation of a global maize collection using SNP markers</div>, <div class="source">PLoS ONE</div> , <div class="volume">4</div>, <div class="fpage">e8451</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20characterization%20and%20linkage%20disequilibrium%20estimation%20of%20a%20global%20maize%20collection%20using%20SNP%20markers&amp;author=J.%20Yan&amp;author=T.%20Shah&amp;author=M.L.%20Warburton&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e8451" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0008451" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0008451" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0008451"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20041112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hancock</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Brachi</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Faure</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Adaptation to climate across the <em>Arabidopsis thaliana</em> genome</div>, <div class="source">Science</div> , <div class="volume">334</div>, <div class="fpage">83</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptation%20to%20climate%20across%20the%20Arabidopsis%20thaliana%20genome&amp;author=A.M.%20Hancock&amp;author=B.%20Brachi&amp;author=N.%20Faure&amp;publication_year=2011&amp;journal=Science&amp;volume=334&amp;pages=83-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1209244" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1209244" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1209244"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21980108" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akey</div>
               <div class="given-names">J.M.</div>
            </div>, <div class="name string-name"><div class="surname">Ruhe</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Akey</div> <div class="given-names">D.T.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Tracking footprints of artificial selection in the dog genome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">1160</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracking%20footprints%20of%20artificial%20selection%20in%20the%20dog%20genome&amp;author=J.M.%20Akey&amp;author=A.L.%20Ruhe&amp;author=D.T.%20Akey&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=1160-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909918107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909918107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909918107"> </span></div> </div></div></div></div></div><div content-id="DSW014C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Griffith</div>
               <div class="given-names">M.E.</div>
            </div>, <div class="name string-name"><div class="surname">da Silva Conceição</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Smyth</div> <div class="given-names">D.R.</div></div></div>
          <div class="year">1999</div>, <div class="article-title"><em>PETALLOSS</em> gene regulates initiation and orientation of second whorl organs in the <em>Arabidopsis</em> flower</div>, <div class="source">Development</div> , <div class="volume">126</div>, <div class="fpage">5635</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PETALLOSS%20gene%20regulates%20initiation%20and%20orientation%20of%20second%20whorl%20organs%20in%20the%20Arabidopsis%20flower&amp;author=M.E.%20Griffith&amp;author=A.%20da%20Silva%20Concei%C3%A7%C3%A3o&amp;author=D.R.%20Smyth&amp;publication_year=1999&amp;journal=Development&amp;volume=126&amp;pages=5635-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10572040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brewer</div>
               <div class="given-names">P.B.</div>
            </div>, <div class="name string-name"><div class="surname">Howles</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Dorian</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title"><em>PETALLOSS</em>, a trihelix transcription factor gene, regulates perianth architecture in the <em>Arabidopsis</em> flower</div>, <div class="source">Development</div> , <div class="volume">131</div>, <div class="fpage">4035</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PETALLOSS%2C%20a%20trihelix%20transcription%20factor%20gene%2C%20regulates%20perianth%20architecture%20in%20the%20Arabidopsis%20flower&amp;author=P.B.%20Brewer&amp;author=P.A.%20Howles&amp;author=K.%20Dorian&amp;publication_year=2004&amp;journal=Development&amp;volume=131&amp;pages=4035-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.01279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.01279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.01279"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15269176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lampugnani</div>
               <div class="given-names">E.R.</div>
            </div>, <div class="name string-name"><div class="surname">Kilinc</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Smyth</div> <div class="given-names">D.R.</div></div></div>
          <div class="year">2012</div>, <div class="article-title"><em>PETALLOSS</em> is a boundary gene that inhibits growth between developing sepals in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant J.</div> , <div class="volume">71</div>, <div class="fpage">724</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PETALLOSS%20is%20a%20boundary%20gene%20that%20inhibits%20growth%20between%20developing%20sepals%20in%20Arabidopsis%20thaliana&amp;author=E.R.%20Lampugnani&amp;author=A.%20Kilinc&amp;author=D.R.%20Smyth&amp;publication_year=2012&amp;journal=Plant%20J.&amp;volume=71&amp;pages=724-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2012.71.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2012.71.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2012.71.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22507233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grassi</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Labra</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Imazio</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Evidence of a secondary grapevine domestication centre detected by SSR analysis</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">107</div>, <div class="fpage">1315</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20of%20a%20secondary%20grapevine%20domestication%20centre%20detected%20by%20SSR%20analysis&amp;author=F.%20Grassi&amp;author=M.%20Labra&amp;author=S.%20Imazio&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=1315-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1321-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1321-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1321-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13679993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quilot</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">B.H.</div></div>, <div class="name string-name"><div class="surname">Kervella</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">QTL analysis of quality traits in an advanced backcross between <em>Prunus persica</em> cultivars and the wild relative species, <em>P. davidiana</em></div>. <div class="source">Theor. Appl. Genet.</div> , <div class="volume">109</div>, <div class="fpage">884</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20quality%20traits%20in%20an%20advanced%20backcross%20between%20Prunus%20persica%20cultivars%20and%20the%20wild%20relative%20species%2C%20P.%20davidiana&amp;author=B.%20Quilot&amp;author=B.H.%20Wu&amp;author=J.%20Kervella&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=884-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1703-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1703-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1703-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15168024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ushijima</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Sassa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Dandekar</div> <div class="given-names">A.M.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Structural and transcriptional analysis of the self-incompatibility locus of almond: identification of a pollen-expressed F-box gene with haplotype-specific polymorphism</div>, <div class="source">Plant Cell</div> , <div class="volume">15</div>, <div class="fpage">771</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20transcriptional%20analysis%20of%20the%20self-incompatibility%20locus%20of%20almond%3A%20identification%20of%20a%20pollen-expressed%20F-box%20gene%20with%20haplotype-specific%20polymorphism&amp;author=K.%20Ushijima&amp;author=H.%20Sassa&amp;author=A.M.%20Dandekar&amp;publication_year=2003&amp;journal=Plant%20Cell&amp;volume=15&amp;pages=771-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.009290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.009290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.009290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12615948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tao</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Iezzoni</div> <div class="given-names">A.F.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">The <em>S</em>-RNase-based gametophytic self-incompatibility system in <em>Prunus</em> exhibits distinct genetic and molecular features</div>, <div class="source">Sci. Hort.</div> , <div class="volume">124</div>, <div class="fpage">423</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20S-RNase-based%20gametophytic%20self-incompatibility%20system%20in%20Prunus%20exhibits%20distinct%20genetic%20and%20molecular%20features&amp;author=R.%20Tao&amp;author=A.F.%20Iezzoni&amp;publication_year=2010&amp;journal=Sci.%20Hort.&amp;volume=124&amp;pages=423-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2010.01.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2010.01.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2010.01.025"> </span></div> </div></div></div></div></div><div content-id="DSW014C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tao</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Watari</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hanada</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Self-compatible peach (<em>Prunus persica</em>) has mutant versions of the <em>S</em> haplotypes found in self-incompatible <em>Prunus</em> species</div>, <div class="source">Plant Mol. Biol.</div> , <div class="volume">63</div>, <div class="fpage">109</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Self-compatible%20peach%20(Prunus%20persica)%20has%20mutant%20versions%20of%20the%20S%20haplotypes%20found%20in%20self-incompatible%20Prunus%20species&amp;author=R.%20Tao&amp;author=A.%20Watari&amp;author=T.%20Hanada&amp;publication_year=2007&amp;journal=Plant%20Mol.%20Biol.&amp;volume=63&amp;pages=109-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-006-9076-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-006-9076-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-006-9076-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17006593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hanada</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Watari</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kibe</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Two novel self-compatible <em>S</em> haplotypes in peach (<em>Prunus persica</em>)</div>, <div class="source">J. Jpn Soc. Hortic. Sci.</div> , <div class="volume">83</div>, <div class="fpage">201</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20novel%20self-compatible%20S%20haplotypes%20in%20peach%20(Prunus%20persica)&amp;author=T.%20Hanada&amp;author=A.%20Watari&amp;author=T.%20Kibe&amp;publication_year=2014&amp;journal=J.%20Jpn%20Soc.%20Hortic.%20Sci.&amp;volume=83&amp;pages=201-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.CH-099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.CH-099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.CH-099"> </span></div> </div></div></div></div></div><div content-id="DSW014C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Banović</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Šurbanovski</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Konstantinović</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Maksimović</div> <div class="given-names">V.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Basic RNases of wild almond (<em>Prunus webbii</em>): cloning and characterization of six new <em>S</em>-RNase and one “non-<em>S</em> RNase” genes</div>, <div class="source">J. Plant Physiol.</div> , <div class="volume">166</div>, <div class="fpage">395</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20RNases%20of%20wild%20almond%20(Prunus%20webbii)%3A%20cloning%20and%20characterization%20of%20six%20new%20S-RNase%20and%20one%20%E2%80%9Cnon-S%20RNase%E2%80%9D%20genes&amp;author=B.%20Banovi%C4%87&amp;author=N.%20%C5%A0urbanovski&amp;author=M.%20Konstantinovi%C4%87&amp;author=V.%20Maksimovi%C4%87&amp;publication_year=2009&amp;journal=J.%20Plant%20Physiol.&amp;volume=166&amp;pages=395-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2008.06.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2008.06.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2008.06.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18778875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Giuliano</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Tavazza</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Diretto</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Beyer</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">M.A.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Metabolic engineering of carotenoid biosynthesis in plants</div>, <div class="source">Trends Biotech.</div> , <div class="volume">26</div>, <div class="fpage">139</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20engineering%20of%20carotenoid%20biosynthesis%20in%20plants&amp;author=G.%20Giuliano&amp;author=R.%20Tavazza&amp;author=G.%20Diretto&amp;author=P.%20Beyer&amp;author=M.A.%20Taylor&amp;publication_year=2008&amp;journal=Trends%20Biotech.&amp;volume=26&amp;pages=139-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2007.12.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2007.12.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2007.12.003"> </span></div> </div></div></div></div></div><div content-id="DSW014C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bliss</div>
               <div class="given-names">F.A.</div>
            </div>, <div class="name string-name"><div class="surname">Arulsekar</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Foolad</div> <div class="given-names">M.R.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">An expanded genetic linkage map of <em>Prunus</em> based on an interspecific cross between almond and peach</div>, <div class="source">Genome</div> , <div class="volume">45</div>, <div class="fpage">520</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20expanded%20genetic%20linkage%20map%20of%20Prunus%20based%20on%20an%20interspecific%20cross%20between%20almond%20and%20peach&amp;author=F.A.%20Bliss&amp;author=S.%20Arulsekar&amp;author=M.R.%20Foolad&amp;publication_year=2002&amp;journal=Genome&amp;volume=45&amp;pages=520-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g02-011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg02-011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg02-011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12033621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Warburton</div>
               <div class="given-names">M.L.</div>
            </div>, <div class="name string-name"><div class="surname">Becerra-Velásquez</div> <div class="given-names">V.L.</div></div>, <div class="name string-name"><div class="surname">Goffreda</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Bliss</div> <div class="given-names">F.A.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Utility of RAPD markers in identifying genetic linkages to genes of economic interest in peach</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">93</div>, <div class="fpage">920</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Utility%20of%20RAPD%20markers%20in%20identifying%20genetic%20linkages%20to%20genes%20of%20economic%20interest%20in%20peach&amp;author=M.L.%20Warburton&amp;author=V.L.%20Becerra-Vel%C3%A1squez&amp;author=J.C.%20Goffreda&amp;author=F.A.%20Bliss&amp;publication_year=1996&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=93&amp;pages=920-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00224094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00224094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00224094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24162426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eduardo</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Pacheco</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Chietera</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">QTL analysis of fruit quality traits in two peach intraspecific populations and importance of maturity date pleiotropic effect</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">7</div>, <div class="fpage">323</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20fruit%20quality%20traits%20in%20two%20peach%20intraspecific%20populations%20and%20importance%20of%20maturity%20date%20pleiotropic%20effect&amp;author=I.%20Eduardo&amp;author=I.%20Pacheco&amp;author=G.%20Chietera&amp;publication_year=2011&amp;journal=Tree%20Genet.%20Genomes&amp;volume=7&amp;pages=323-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-010-0334-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-010-0334-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-010-0334-6"> </span></div> </div></div></div></div></div><div content-id="DSW014C78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Espinosa-Ruiz</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bellés</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Serrano</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Culiáñez-Macià</div> <div class="given-names">F.A.</div></div></div>
          <div class="year">1999</div>, <div class="article-title"><em>Arabidopsis thaliana</em> AtHAL3: a flavoprotein related to salt and osmotic tolerance and plant growth</div>, <div class="source">Plant J.</div> , <div class="volume">20</div>, <div class="fpage">529</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20thaliana%20AtHAL3%3A%20a%20flavoprotein%20related%20to%20salt%20and%20osmotic%20tolerance%20and%20plant%20growth&amp;author=A.%20Espinosa-Ruiz&amp;author=J.M.%20Bell%C3%A9s&amp;author=R.%20Serrano&amp;author=F.A.%20Culi%C3%A1%C3%B1ez-Maci%C3%A0&amp;publication_year=1999&amp;journal=Plant%20J.&amp;volume=20&amp;pages=529-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1999.00626.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1999.00626.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1999.00626.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10652125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW014C79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW014C79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Levy</div>
               <div class="given-names">Y.Y.</div>
            </div>, <div class="name string-name"><div class="surname">Mesnage</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mylne</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Gendall</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">C.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Multiple roles of <em>Arabidopsis VRN1</em> in vernalization and flowering time control</div>, <div class="source">Science</div> , <div class="volume">297</div>, <div class="fpage">243</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20roles%20of%20Arabidopsis%20VRN1%20in%20vernalization%20and%20flowering%20time%20control&amp;author=Y.Y.%20Levy&amp;author=S.%20Mesnage&amp;author=J.S.%20Mylne&amp;author=A.R.%20Gendall&amp;author=C.%20Dean&amp;publication_year=2002&amp;journal=Science&amp;volume=297&amp;pages=243-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1072147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1072147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1072147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12114624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745617&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw014/1/dsw014_Supplementary_Data.zip?Expires=2147483647&amp;Signature=lOlE-ySSMqFz~MUA4Sy70hNsvWWxoMwezIk5ZZFAhFMSzpsbt01ZBjr8eQ~tRPp78xF~VN13H9Ol-FaKv1CyLm2Ty1RPiTMfROv7flDRHg3ycNR49DSIKJMribllbqiXtRoq-Pi023bxEin~4o3L5dBdlEvf31OtB2IA6dy15dqlaKJzqVmPp4TauMTUnp-uAOxypLApZc3MPrd7a81wCcFQ0WoCt~Y8IVHqsFVysDMJ~eN0aS0HsYUkChc3eYMUDRF8hqWKa6HrjweTs3tI88zvrgWbwEtoY4atepamWfHcPLV35ucJvS9rKmPTay9PvGzUi1fIiE8Sw6MI~SX~pg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">844</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745617"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f3%2f271%2f1745617">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000379741500009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F3%2F271%2F1745617" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745617'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide view of genetic diversity reveals paths of selection and cultivar differentiation in peach domestication" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f271%2f1745617">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

