

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/2/165/419961","@type":"ScholarlyArticle","name":"Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat ( Triticum aestivum )","datePublished":"2012-02-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2012-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss001","keywords":["wheat","ESTs","annotation","transcription factors","miRNA","SNP","correspondence analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/2/165/419961","author":[{"name":"Manickavelu, Alagu","affiliation":"1  Kihara Institute for Biological Research ,  Yokohama City University ,  Maioka-cho 641-12, Yokohama 244-0813 ,  Japan","@type":"Person"},{"name":"Kawaura, Kanako","affiliation":"1  Kihara Institute for Biological Research ,  Yokohama City University ,  Maioka-cho 641-12, Yokohama 244-0813 ,  Japan","@type":"Person"},{"name":"Oishi, Kazuko","affiliation":"2  Genome Biology Laboratory ,  National Institute of Genetics ,  Mishima 411-8540 ,  Japan","@type":"Person"},{"name":"Shin-I, Tadasu","affiliation":"2  Genome Biology Laboratory ,  National Institute of Genetics ,  Mishima 411-8540 ,  Japan","@type":"Person"},{"name":"Kohara, Yuji","affiliation":"2  Genome Biology Laboratory ,  National Institute of Genetics ,  Mishima 411-8540 ,  Japan","@type":"Person"},{"name":"Yahiaoui, Nabila","affiliation":"3  Institute of Plant Biology ,  University of Zurich ,  Zollikerstrasse 107, 8008 Zurich ,  Switzerland","@type":"Person"},{"name":"Keller, Beat","affiliation":"3  Institute of Plant Biology ,  University of Zurich ,  Zollikerstrasse 107, 8008 Zurich ,  Switzerland","@type":"Person"},{"name":"Abe, Reina","affiliation":"4  Bioinformatics Laboratory, Faculty of Agriculture ,  Meiji University ,  1-1-1 Higashi Mita, Kawasaki 214-8571 ,  Japan","@type":"Person"},{"name":"Suzuki, Ayako","affiliation":"4  Bioinformatics Laboratory, Faculty of Agriculture ,  Meiji University ,  1-1-1 Higashi Mita, Kawasaki 214-8571 ,  Japan","@type":"Person"},{"name":"Nagayama, Taishi","affiliation":"4  Bioinformatics Laboratory, Faculty of Agriculture ,  Meiji University ,  1-1-1 Higashi Mita, Kawasaki 214-8571 ,  Japan","@type":"Person"},{"name":"Yano, Kentaro","affiliation":"4  Bioinformatics Laboratory, Faculty of Agriculture ,  Meiji University ,  1-1-1 Higashi Mita, Kawasaki 214-8571 ,  Japan","@type":"Person"},{"name":"Ogihara, Yasunari","affiliation":"1  Kihara Institute for Biological Research ,  Yokohama City University ,  Maioka-cho 641-12, Yokohama 244-0813 ,  Japan","@type":"Person"}],"description":"Abstract.  About 1 million expressed sequence tag (EST) sequences comprising 125.3 Mb nucleotides were accreted from 51 cDNA libraries constructed from a variet","pageStart":"165","pageEnd":"177","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat ( Triticum aestivum )","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat ( Triticum aestivum )" />
    <meta property="og:description" content="Abstract.  About 1 million expressed sequence tag (EST) sequences comprising 125.3 Mb nucleotides were accreted from 51 cDNA libraries constructed from a variet" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/2/165/419961" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-02-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Manickavelu, Alagu" /><meta name="citation_author_institution" content="Kihara Institute for Biological Research, Yokohama City University, Maioka-cho 641-12, Yokohama 244-0813, Japan" /><meta name="citation_author" content="Kawaura, Kanako" /><meta name="citation_author_institution" content="Kihara Institute for Biological Research, Yokohama City University, Maioka-cho 641-12, Yokohama 244-0813, Japan" /><meta name="citation_author" content="Oishi, Kazuko" /><meta name="citation_author_institution" content="Genome Biology Laboratory, National Institute of Genetics, Mishima 411-8540, Japan" /><meta name="citation_author" content="Shin-I, Tadasu" /><meta name="citation_author_institution" content="Genome Biology Laboratory, National Institute of Genetics, Mishima 411-8540, Japan" /><meta name="citation_author" content="Kohara, Yuji" /><meta name="citation_author_institution" content="Genome Biology Laboratory, National Institute of Genetics, Mishima 411-8540, Japan" /><meta name="citation_author" content="Yahiaoui, Nabila" /><meta name="citation_author_institution" content="Institute of Plant Biology, University of Zurich, Zollikerstrasse 107, 8008 Zurich, Switzerland" /><meta name="citation_author" content="Keller, Beat" /><meta name="citation_author_institution" content="Institute of Plant Biology, University of Zurich, Zollikerstrasse 107, 8008 Zurich, Switzerland" /><meta name="citation_author" content="Abe, Reina" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Faculty of Agriculture, Meiji University, 1-1-1 Higashi Mita, Kawasaki 214-8571, Japan" /><meta name="citation_author" content="Suzuki, Ayako" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Faculty of Agriculture, Meiji University, 1-1-1 Higashi Mita, Kawasaki 214-8571, Japan" /><meta name="citation_author" content="Nagayama, Taishi" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Faculty of Agriculture, Meiji University, 1-1-1 Higashi Mita, Kawasaki 214-8571, Japan" /><meta name="citation_author" content="Yano, Kentaro" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Faculty of Agriculture, Meiji University, 1-1-1 Higashi Mita, Kawasaki 214-8571, Japan" /><meta name="citation_author" content="Ogihara, Yasunari" /><meta name="citation_author_institution" content="Kihara Institute for Biological Research, Yokohama City University, Maioka-cho 641-12, Yokohama 244-0813, Japan" /><meta name="citation_title" content="Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat (Triticum aestivum)" /><meta name="citation_firstpage" content="165" /><meta name="citation_lastpage" content="177" /><meta name="citation_doi" content="10.1093/dnares/dss001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2012/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Multi location testing as a tool to identify plant response to global climate change; citation_publisher=CABI, London, UK; Climate Change and Crop Production;  citation_year=2010; " /><meta name="citation_reference" content="citation_publisher=International Food Policy Research Institute, Washington, DC; Global Food Demand, Supply, and Price Prospects to 2010;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=A transcriptomics resource for wheat functional genomics; Plant Biotech.;  citation_year=2004;  citation_volume=2;  citation_pages=495-506; " /><meta name="citation_reference" content="citation_title=Expression profiling using cDNA microarrays; Nat. Genet.;  citation_year=1999;  citation_volume=21;  citation_issue=Suppl. 21;  citation_pages=10-4; " /><meta name="citation_reference" content="citation_title=Mining EST databases to resolve evolutionary events in major crop species; Genome;  citation_year=2004;  citation_volume=47;  citation_pages=868-76; " /><meta name="citation_reference" content="citation_title=Identification, analysis, and utilization of conserved ortholog set markers for comparative genomics in higher plants; Plant Cell;  citation_year=2002;  citation_volume=14;  citation_pages=1457-67; " /><meta name="citation_reference" content="citation_title=Large-scale statistical analyses of rice ESTs reveal correlated patterns of gene expression; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=950-9; " /><meta name="citation_reference" content="citation_title=Comparative analyses of potato expressed sequence tag libraries; Plant Physiol.;  citation_year=2003;  citation_volume=131;  citation_pages=419-29; " /><meta name="citation_reference" content="citation_title=Expression patterns of duplicate genes in the developing root in Arabidopsis thaliana; Mol. Evol.;  citation_year=2004;  citation_volume=60;  citation_pages=247-56; " /><meta name="citation_reference" content="citation_title=EST analysis in barley defines a unigene set comprising 4,000 genes; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=97-103; " /><meta name="citation_reference" content="citation_title=Arabidopsis microarray service facilities; Plant Physiol.;  citation_year=2000;  citation_volume=124;  citation_pages=1468-71; " /><meta name="citation_reference" content="citation_title=Gene expression profiles during the initial phase of salt stress in rice; Plant Cell;  citation_year=2001;  citation_volume=13;  citation_pages=889-905; " /><meta name="citation_reference" content="citation_title=ESTs, cDNA microarrays, and gene expression profiling: tools for dissecting plant physiology and development; Plant J.;  citation_year=2004;  citation_volume=39;  citation_pages=697-714; " /><meta name="citation_reference" content="citation_title=Functional genomics of cell elongation in developing cotton fibers; Plant. Mol. Biol.;  citation_year=2004;  citation_volume=54;  citation_pages=911-29; " /><meta name="citation_reference" content="citation_title=A new resource for cereal genomics: 22K barley GeneChip comes of age; Plant Physiol.;  citation_year=2004;  citation_volume=134;  citation_pages=960-8; " /><meta name="citation_reference" content="citation_title=Analysis and functional annotation of an expressed sequence tag collection for tropical crop sugarcane; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2725-35; " /><meta name="citation_reference" content="citation_title=A global assembly of cotton ESTs; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=441-50; " /><meta name="citation_reference" content="citation_title=Bioinformatic analysis of ESTs collected by Sanger and pyrosequencing methods for a keystone forest tree species: oak; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=650" /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant, Arabidopsis thaliana; citation_author=The Arabidopsis Genome Initiative; , Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=11479-84; " /><meta name="citation_reference" content="citation_title=Patterns of DNA sequence polymorphism along chromosome 1 of maize (Zea mays L.); Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=9161-6; " /><meta name="citation_reference" content="citation_title=Applications of single nucleotide polymorphisms in crop genetics; Curr. Opin. Plant Biol.;  citation_year=2002;  citation_volume=5;  citation_pages=94-100; " /><meta name="citation_reference" content="citation_title=Mining single-nucleotide polymorphisms from hexaploid wheat ESTs; Genome;  citation_year=2003;  citation_volume=49;  citation_pages=431-7; " /><meta name="citation_reference" content="citation_title=Correlated clustering and virtual display of gene expression patterns in the wheat life cycle by large-scale statistical analyses of expressed sequence tags; Plant J.;  citation_year=2003;  citation_volume=33;  citation_pages=1001-11; " /><meta name="citation_reference" content="citation_title=Construction of a full-length cDNA library from young spikelets of hexaploid wheat and its characterization by large-scale sequencing of expressed sequence tags; Genes Genet. Syst.;  citation_year=2004;  citation_volume=79;  citation_pages=227-32; " /><meta name="citation_reference" content="citation_title=Tissue expression map of a large number of expressed sequence tags and its application to in silico screening of stress response genes in common wheat; Mol. Genet. Genomics;  citation_year=2006;  citation_volume=276;  citation_pages=304-12; " /><meta name="citation_reference" content="citation_title=Assessment of adaptive evolution between wheat and rice as deduced from the full-length cDNA sequence data and the expression patterns of common wheat; BMC Genomics;  citation_year=2009;  citation_volume=18;  citation_pages=271" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of salinity stress responses in common wheat using a 22k oligo-DNA microarray; Funct. Integr. Genomics;  citation_year=2006;  citation_volume=6;  citation_pages=132-42; " /><meta name="citation_reference" content="citation_title=Comparative gene expression analysis of susceptible and resistance near-isogenic lines in common wheat infected by Puccinia triticina; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=211-22; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using PHRED. I. Accuracy assessment; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=175-85; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=The Gene Ontology project in 2008; citation_author=Gene Ontology Consortium; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D440-4; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=PlnTFDB: updated content and new features of the plant transcription factor database; Nucleic Acids Res.;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Comprehensive sequence analysis of 24,783 barley full-length cDNAs derived from 12 clone libraries; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=20-8; " /><meta name="citation_reference" content="citation_title=MiRBase: tools for microRNA genomics; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D154-8; " /><meta name="citation_reference" content="citation_title=Identification of conserved microRNAs and their targets from Solanum lycopersicum Mill; Gene;  citation_year=2008;  citation_volume=423;  citation_pages=1-7; " /><meta name="citation_reference" content="citation_title=Computational identification of novel microRNAs and targets in Brassica napus; FEBS Lett.;  citation_year=2007;  citation_volume=581;  citation_pages=1464-74; " /><meta name="citation_reference" content="citation_title=Tablet-next generation sequence assembly visualization; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=401-2; " /><meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=14863-8; " /><meta name="citation_reference" content="citation_title=OryzaExpress: an integrated database of gene expression networks and omics annotations in rice; Plant Cell Physiol.;  citation_year=2011;  citation_volume=52;  citation_pages=220-9; " /><meta name="citation_reference" content="citation_title=Megabase level sequencing reveals contrasted organization and evolution patterns of the wheat gene and transposable element spaces; Plant Cell;  citation_year=2010;  citation_volume=22;  citation_pages=1686-701; " /><meta name="citation_reference" content="citation_title=Discrimination of homoeologous gene expression in hexaploid wheat by SNP analysis of contigs grouped from a large number of expressed sequence tags; Mol. Genet. Genomics;  citation_year=2003;  citation_volume=270;  citation_pages=371-7; " /><meta name="citation_reference" content="citation_title=Construction and evaluation of cDNA libraries for large-scale expressed sequence tag sequencing in wheat (Triticum aestivum L.); Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=595-608; " /><meta name="citation_reference" content="citation_title=Differential methylation of genes and repeats in land plants; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1431-40; " /><meta name="citation_reference" content="citation_title=Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice; Science;  citation_year=2003;  citation_volume=301;  citation_pages=376" /><meta name="citation_reference" content="citation_title=Insights into corn genes derived from large-scale cDNASequencing; Plant Mol. Biol.;  citation_year=2009;  citation_volume=69;  citation_pages=179-94; " /><meta name="citation_reference" content="citation_title=One-step identification of conserved miRNAs, their argets, potential transcription factors and effector genes of complete secondary metabolism pathways after 454 pyrosequencing of calyx cDNAs from the Labiate (Salvia sclarea L.); Gene;  citation_year=2010;  citation_volume=450;  citation_pages=55-62; " /><meta name="citation_reference" content="citation_title=Transcriptional regulation in mammalian cells by sequence-specific DNA binding proteins; Science;  citation_year=1989;  citation_volume=245;  citation_pages=371-8; " /><meta name="citation_reference" content="citation_title=Eukaryotic transcriptional regulatory proteins; Annu. Rev. Biochem.;  citation_year=1989;  citation_volume=58;  citation_pages=799-839; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis for the chicken based on 19,626 finished cDNA sequences and 485,337 expressed sequence tags; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=174-83; " /><meta name="citation_reference" content="citation_title=Computational analysis of miRNA targets in plants: current status and challenges; Brief Bioinform.;  citation_year=2011;  citation_volume=21;  citation_pages=115-21; " /><meta name="citation_reference" content="citation_title=Correspondence analysis applied to microarray data; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=10781-6; " /><meta name="citation_reference" content="citation_title=A new method for gene discovery in large-scale microarray data; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=1532-9; " /><meta name="citation_reference" content="citation_title=A high-throughput data mining of single nucleotide polymorphisms in coffea species expressed sequence tags suggests differential homeologous gene expression in the allotetraploid (Coffea arabica); Plant Physiol.;  citation_year=2010;  citation_volume=154;  citation_pages=1053-66; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/2/165/1164316/dss001.pdf" /><meta name="description" content="Abstract.  About 1 million expressed sequence tag (EST) sequences comprising 125.3 Mb nucleotides were accreted from 51 cDNA libraries constructed from a variet" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/2/165/419961" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/2/165/419961" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/2/165/419961");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss001';
    </script>
<title>
	Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat (Triticum aestivum) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./419961" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZKAATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFuaWNrYXZlbHUsIEFsYWd1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2loYXJhIEluc3RpdHV0ZSBmb3IgQmlvbG9naWNhbCBSZXNlYXJjaCwgWW9rb2hhbWEgQ2l0eSBVbml2ZXJzaXR5LCBNYWlva2EtY2hvIDY0MS0xMiwgWW9rb2hhbWEgMjQ0LTA4MTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YXVyYSwgS2FuYWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2loYXJhIEluc3RpdHV0ZSBmb3IgQmlvbG9naWNhbCBSZXNlYXJjaCwgWW9rb2hhbWEgQ2l0eSBVbml2ZXJzaXR5LCBNYWlva2EtY2hvIDY0MS0xMiwgWW9rb2hhbWEgMjQ0LTA4MTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2lzaGksIEthenVrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlbm9tZSBCaW9sb2d5IExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgTWlzaGltYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGluLUksIFRhZGFzdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlbm9tZSBCaW9sb2d5IExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgTWlzaGltYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2hhcmEsIFl1amkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHZW5vbWUgQmlvbG9neSBMYWJvcmF0b3J5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIE1pc2hpbWEgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFoaWFvdWksIE5hYmlsYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBQbGFudCBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIFp1cmljaCwgWm9sbGlrZXJzdHJhc3NlIDEwNywgODAwOCBadXJpY2gsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2VsbGVyLCBCZWF0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIFBsYW50IEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgWnVyaWNoLCBab2xsaWtlcnN0cmFzc2UgMTA3LCA4MDA4IFp1cmljaCwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmUsIFJlaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgTGFib3JhdG9yeSwgRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgTWVpamkgVW5pdmVyc2l0eSwgMS0xLTEgSGlnYXNoaSBNaXRhLCBLYXdhc2FraSAyMTQtODU3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIEF5YWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgTGFib3JhdG9yeSwgRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgTWVpamkgVW5pdmVyc2l0eSwgMS0xLTEgSGlnYXNoaSBNaXRhLCBLYXdhc2FraSAyMTQtODU3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWdheWFtYSwgVGFpc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgTGFib3JhdG9yeSwgRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgTWVpamkgVW5pdmVyc2l0eSwgMS0xLTEgSGlnYXNoaSBNaXRhLCBLYXdhc2FraSAyMTQtODU3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW5vLCBLZW50YXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgTGFib3JhdG9yeSwgRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgTWVpamkgVW5pdmVyc2l0eSwgMS0xLTEgSGlnYXNoaSBNaXRhLCBLYXdhc2FraSAyMTQtODU3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ2loYXJhLCBZYXN1bmFyaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktpaGFyYSBJbnN0aXR1dGUgZm9yIEJpb2xvZ2ljYWwgUmVzZWFyY2gsIFlva29oYW1hIENpdHkgVW5pdmVyc2l0eSwgTWFpb2thLWNobyA2NDEtMTIsIFlva29oYW1hIDI0NC0wODEzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29tcHJlaGVuc2l2ZSBGdW5jdGlvbmFsIEFuYWx5c2VzIG9mIEV4cHJlc3NlZCBTZXF1ZW5jZSBUYWdzIGluIENvbW1vbiBXaGVhdCAoVHJpdGljdW0gYWVzdGl2dW0pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpIGxvY2F0aW9uIHRlc3RpbmcgYXMgYSB0b29sIHRvIGlkZW50aWZ5IHBsYW50IHJlc3BvbnNlIHRvIGdsb2JhbCBjbGltYXRlIGNoYW5nZTsgY2l0YXRpb25fcHVibGlzaGVyPUNBQkksIExvbmRvbiwgVUs7IENsaW1hdGUgQ2hhbmdlIGFuZCBDcm9wIFByb2R1Y3Rpb247ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1JbnRlcm5hdGlvbmFsIEZvb2QgUG9saWN5IFJlc2VhcmNoIEluc3RpdHV0ZSwgV2FzaGluZ3RvbiwgREM7IEdsb2JhbCBGb29kIERlbWFuZCwgU3VwcGx5LCBhbmQgUHJpY2UgUHJvc3BlY3RzIHRvIDIwMTA7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdHJhbnNjcmlwdG9taWNzIHJlc291cmNlIGZvciB3aGVhdCBmdW5jdGlvbmFsIGdlbm9taWNzOyBQbGFudCBCaW90ZWNoLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NDk1LTUwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBwcm9maWxpbmcgdXNpbmcgY0ROQSBtaWNyb2FycmF5czsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbC4gMjE7ICBjaXRhdGlvbl9wYWdlcz0xMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5pbmcgRVNUIGRhdGFiYXNlcyB0byByZXNvbHZlIGV2b2x1dGlvbmFyeSBldmVudHMgaW4gbWFqb3IgY3JvcCBzcGVjaWVzOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz04NjgtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uLCBhbmFseXNpcywgYW5kIHV0aWxpemF0aW9uIG9mIGNvbnNlcnZlZCBvcnRob2xvZyBzZXQgbWFya2VycyBmb3IgY29tcGFyYXRpdmUgZ2Vub21pY3MgaW4gaGlnaGVyIHBsYW50czsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE0NTctNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIHN0YXRpc3RpY2FsIGFuYWx5c2VzIG9mIHJpY2UgRVNUcyByZXZlYWwgY29ycmVsYXRlZCBwYXR0ZXJucyBvZiBnZW5lIGV4cHJlc3Npb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz05NTAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzZXMgb2YgcG90YXRvIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWcgbGlicmFyaWVzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz00MTktMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gcGF0dGVybnMgb2YgZHVwbGljYXRlIGdlbmVzIGluIHRoZSBkZXZlbG9waW5nIHJvb3QgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7IE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0yNDctNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVCBhbmFseXNpcyBpbiBiYXJsZXkgZGVmaW5lcyBhIHVuaWdlbmUgc2V0IGNvbXByaXNpbmcgNCwwMDAgZ2VuZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9OTctMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBtaWNyb2FycmF5IHNlcnZpY2UgZmFjaWxpdGllczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9MTQ2OC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBleHByZXNzaW9uIHByb2ZpbGVzIGR1cmluZyB0aGUgaW5pdGlhbCBwaGFzZSBvZiBzYWx0IHN0cmVzcyBpbiByaWNlOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9ODg5LTkwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RVNUcywgY0ROQSBtaWNyb2FycmF5cywgYW5kIGdlbmUgZXhwcmVzc2lvbiBwcm9maWxpbmc6IHRvb2xzIGZvciBkaXNzZWN0aW5nIHBsYW50IHBoeXNpb2xvZ3kgYW5kIGRldmVsb3BtZW50OyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTY5Ny03MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgZ2Vub21pY3Mgb2YgY2VsbCBlbG9uZ2F0aW9uIGluIGRldmVsb3BpbmcgY290dG9uIGZpYmVyczsgUGxhbnQuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz05MTEtMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IHJlc291cmNlIGZvciBjZXJlYWwgZ2Vub21pY3M6IDIySyBiYXJsZXkgR2VuZUNoaXAgY29tZXMgb2YgYWdlOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz05NjAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgYW5kIGZ1bmN0aW9uYWwgYW5ub3RhdGlvbiBvZiBhbiBleHByZXNzZWQgc2VxdWVuY2UgdGFnIGNvbGxlY3Rpb24gZm9yIHRyb3BpY2FsIGNyb3Agc3VnYXJjYW5lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI3MjUtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2xvYmFsIGFzc2VtYmx5IG9mIGNvdHRvbiBFU1RzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQ0MS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvaW5mb3JtYXRpYyBhbmFseXNpcyBvZiBFU1RzIGNvbGxlY3RlZCBieSBTYW5nZXIgYW5kIHB5cm9zZXF1ZW5jaW5nIG1ldGhvZHMgZm9yIGEga2V5c3RvbmUgZm9yZXN0IHRyZWUgc3BlY2llczogb2FrOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz02NTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBmbG93ZXJpbmcgcGxhbnQsIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBjaXRhdGlvbl9hdXRob3I9VGhlIEFyYWJpZG9wc2lzIEdlbm9tZSBJbml0aWF0aXZlOyAsIE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz03OTYtODE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgb2YgbGlua2FnZSBkaXNlcXVpbGlicml1bSBhbmQgcGhlbm90eXBpYyBhc3NvY2lhdGlvbnMgaW4gdGhlIG1haXplIGdlbm9tZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTExNDc5LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXR0ZXJucyBvZiBETkEgc2VxdWVuY2UgcG9seW1vcnBoaXNtIGFsb25nIGNocm9tb3NvbWUgMSBvZiBtYWl6ZSAoWmVhIG1heXMgTC4pOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9OTE2MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBsaWNhdGlvbnMgb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBpbiBjcm9wIGdlbmV0aWNzOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTQtMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5pbmcgc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBmcm9tIGhleGFwbG9pZCB3aGVhdCBFU1RzOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz00MzEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRlZCBjbHVzdGVyaW5nIGFuZCB2aXJ0dWFsIGRpc3BsYXkgb2YgZ2VuZSBleHByZXNzaW9uIHBhdHRlcm5zIGluIHRoZSB3aGVhdCBsaWZlIGN5Y2xlIGJ5IGxhcmdlLXNjYWxlIHN0YXRpc3RpY2FsIGFuYWx5c2VzIG9mIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTEwMDEtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGZ1bGwtbGVuZ3RoIGNETkEgbGlicmFyeSBmcm9tIHlvdW5nIHNwaWtlbGV0cyBvZiBoZXhhcGxvaWQgd2hlYXQgYW5kIGl0cyBjaGFyYWN0ZXJpemF0aW9uIGJ5IGxhcmdlLXNjYWxlIHNlcXVlbmNpbmcgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3M7IEdlbmVzIEdlbmV0LiBTeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT03OTsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGlzc3VlIGV4cHJlc3Npb24gbWFwIG9mIGEgbGFyZ2UgbnVtYmVyIG9mIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzIGFuZCBpdHMgYXBwbGljYXRpb24gdG8gaW4gc2lsaWNvIHNjcmVlbmluZyBvZiBzdHJlc3MgcmVzcG9uc2UgZ2VuZXMgaW4gY29tbW9uIHdoZWF0OyBNb2wuIEdlbmV0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzY7ICBjaXRhdGlvbl9wYWdlcz0zMDQtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc21lbnQgb2YgYWRhcHRpdmUgZXZvbHV0aW9uIGJldHdlZW4gd2hlYXQgYW5kIHJpY2UgYXMgZGVkdWNlZCBmcm9tIHRoZSBmdWxsLWxlbmd0aCBjRE5BIHNlcXVlbmNlIGRhdGEgYW5kIHRoZSBleHByZXNzaW9uIHBhdHRlcm5zIG9mIGNvbW1vbiB3aGVhdDsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MjcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmFseXNpcyBvZiBzYWxpbml0eSBzdHJlc3MgcmVzcG9uc2VzIGluIGNvbW1vbiB3aGVhdCB1c2luZyBhIDIyayBvbGlnby1ETkEgbWljcm9hcnJheTsgRnVuY3QuIEludGVnci4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTEzMi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2VuZSBleHByZXNzaW9uIGFuYWx5c2lzIG9mIHN1c2NlcHRpYmxlIGFuZCByZXNpc3RhbmNlIG5lYXItaXNvZ2VuaWMgbGluZXMgaW4gY29tbW9uIHdoZWF0IGluZmVjdGVkIGJ5IFB1Y2NpbmlhIHRyaXRpY2luYTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yMTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2UtY2FsbGluZyBvZiBhdXRvbWF0ZWQgc2VxdWVuY2VyIHRyYWNlcyB1c2luZyBQSFJFRC4gSS4gQWNjdXJhY3kgYXNzZXNzbWVudDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE3NS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FQMzogYSBETkEgc2VxdWVuY2UgYXNzZW1ibHkgcHJvZ3JhbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbmUgT250b2xvZ3kgcHJvamVjdCBpbiAyMDA4OyBjaXRhdGlvbl9hdXRob3I9R2VuZSBPbnRvbG9neSBDb25zb3J0aXVtOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1ENDQwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbjogcHJvdGVpbiBkb21haW5zIGlkZW50aWZpZXI7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPVcxMTYtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsblRGREI6IHVwZGF0ZWQgY29udGVudCBhbmQgbmV3IGZlYXR1cmVzIG9mIHRoZSBwbGFudCB0cmFuc2NyaXB0aW9uIGZhY3RvciBkYXRhYmFzZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIHNlcXVlbmNlIGFuYWx5c2lzIG9mIDI0LDc4MyBiYXJsZXkgZnVsbC1sZW5ndGggY0ROQXMgZGVyaXZlZCBmcm9tIDEyIGNsb25lIGxpYnJhcmllczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9MjAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWlSQmFzZTogdG9vbHMgZm9yIG1pY3JvUk5BIGdlbm9taWNzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1EMTU0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGNvbnNlcnZlZCBtaWNyb1JOQXMgYW5kIHRoZWlyIHRhcmdldHMgZnJvbSBTb2xhbnVtIGx5Y29wZXJzaWN1bSBNaWxsOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQyMzsgIGNpdGF0aW9uX3BhZ2VzPTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHV0YXRpb25hbCBpZGVudGlmaWNhdGlvbiBvZiBub3ZlbCBtaWNyb1JOQXMgYW5kIHRhcmdldHMgaW4gQnJhc3NpY2EgbmFwdXM7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTgxOyAgY2l0YXRpb25fcGFnZXM9MTQ2NC03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFibGV0LW5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jZSBhc3NlbWJseSB2aXN1YWxpemF0aW9uOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQwMS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyIGFuYWx5c2lzIGFuZCBkaXNwbGF5IG9mIGdlbm9tZS13aWRlIGV4cHJlc3Npb24gcGF0dGVybnM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xNDg2My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Pcnl6YUV4cHJlc3M6IGFuIGludGVncmF0ZWQgZGF0YWJhc2Ugb2YgZ2VuZSBleHByZXNzaW9uIG5ldHdvcmtzIGFuZCBvbWljcyBhbm5vdGF0aW9ucyBpbiByaWNlOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9MjIwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZ2FiYXNlIGxldmVsIHNlcXVlbmNpbmcgcmV2ZWFscyBjb250cmFzdGVkIG9yZ2FuaXphdGlvbiBhbmQgZXZvbHV0aW9uIHBhdHRlcm5zIG9mIHRoZSB3aGVhdCBnZW5lIGFuZCB0cmFuc3Bvc2FibGUgZWxlbWVudCBzcGFjZXM7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjg2LTcwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY3JpbWluYXRpb24gb2YgaG9tb2VvbG9nb3VzIGdlbmUgZXhwcmVzc2lvbiBpbiBoZXhhcGxvaWQgd2hlYXQgYnkgU05QIGFuYWx5c2lzIG9mIGNvbnRpZ3MgZ3JvdXBlZCBmcm9tIGEgbGFyZ2UgbnVtYmVyIG9mIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOyBNb2wuIEdlbmV0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzA7ICBjaXRhdGlvbl9wYWdlcz0zNzEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIGFuZCBldmFsdWF0aW9uIG9mIGNETkEgbGlicmFyaWVzIGZvciBsYXJnZS1zY2FsZSBleHByZXNzZWQgc2VxdWVuY2UgdGFnIHNlcXVlbmNpbmcgaW4gd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY4OyAgY2l0YXRpb25fcGFnZXM9NTk1LTYwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIG1ldGh5bGF0aW9uIG9mIGdlbmVzIGFuZCByZXBlYXRzIGluIGxhbmQgcGxhbnRzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE0MzEtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbGxlY3Rpb24sIG1hcHBpbmcsIGFuZCBhbm5vdGF0aW9uIG9mIG92ZXIgMjgsMDAwIGNETkEgY2xvbmVzIGZyb20gamFwb25pY2EgcmljZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDE7ICBjaXRhdGlvbl9wYWdlcz0zNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNpZ2h0cyBpbnRvIGNvcm4gZ2VuZXMgZGVyaXZlZCBmcm9tIGxhcmdlLXNjYWxlIGNETkFTZXF1ZW5jaW5nOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MTc5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbmUtc3RlcCBpZGVudGlmaWNhdGlvbiBvZiBjb25zZXJ2ZWQgbWlSTkFzLCB0aGVpciBhcmdldHMsIHBvdGVudGlhbCB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgYW5kIGVmZmVjdG9yIGdlbmVzIG9mIGNvbXBsZXRlIHNlY29uZGFyeSBtZXRhYm9saXNtIHBhdGh3YXlzIGFmdGVyIDQ1NCBweXJvc2VxdWVuY2luZyBvZiBjYWx5eCBjRE5BcyBmcm9tIHRoZSBMYWJpYXRlIChTYWx2aWEgc2NsYXJlYSBMLik7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDUwOyAgY2l0YXRpb25fcGFnZXM9NTUtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uIGluIG1hbW1hbGlhbiBjZWxscyBieSBzZXF1ZW5jZS1zcGVjaWZpYyBETkEgYmluZGluZyBwcm90ZWluczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDU7ICBjaXRhdGlvbl9wYWdlcz0zNzEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXVrYXJ5b3RpYyB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yeSBwcm90ZWluczsgQW5udS4gUmV2LiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTc5OS04Mzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgZm9yIHRoZSBjaGlja2VuIGJhc2VkIG9uIDE5LDYyNiBmaW5pc2hlZCBjRE5BIHNlcXVlbmNlcyBhbmQgNDg1LDMzNyBleHByZXNzZWQgc2VxdWVuY2UgdGFnczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNzQtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYW5hbHlzaXMgb2YgbWlSTkEgdGFyZ2V0cyBpbiBwbGFudHM6IGN1cnJlbnQgc3RhdHVzIGFuZCBjaGFsbGVuZ2VzOyBCcmllZiBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTE1LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZXNwb25kZW5jZSBhbmFseXNpcyBhcHBsaWVkIHRvIG1pY3JvYXJyYXkgZGF0YTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTEwNzgxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IG1ldGhvZCBmb3IgZ2VuZSBkaXNjb3ZlcnkgaW4gbGFyZ2Utc2NhbGUgbWljcm9hcnJheSBkYXRhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xNTMyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC10aHJvdWdocHV0IGRhdGEgbWluaW5nIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgaW4gY29mZmVhIHNwZWNpZXMgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3Mgc3VnZ2VzdHMgZGlmZmVyZW50aWFsIGhvbWVvbG9nb3VzIGdlbmUgZXhwcmVzc2lvbiBpbiB0aGUgYWxsb3RldHJhcGxvaWQgKENvZmZlYSBhcmFiaWNhKTsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9MTA1My02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE5LzIvMTY1LzExNjQzMTYvZHNzMDAxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBYm91dCAxIG1pbGxpb24gZXhwcmVzc2VkIHNlcXVlbmNlIHRhZyAoRVNUKSBzZXF1ZW5jZXMgY29tcHJpc2luZyAxMjUuMyBNYiBudWNsZW90aWRlcyB3ZXJlIGFjY3JldGVkIGZyb20gNTEgY0ROQSBsaWJyYXJpZXMgY29uc3RydWN0ZWQgZnJvbSBhIHZhcmlldCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDE5OTYxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZPtqlTdny1SaS75tEOU0u9VtZnsO" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f165%2f419961" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f165%2f419961" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16354" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&amp;Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/2">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6221280">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221280">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221284">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221284">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221294">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221294">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221316">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221316">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221343">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221343">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221356">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221356">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221358">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221358">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221360">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221360">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221363">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221363">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6221365">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6221365">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/2/153/419549">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/2/179/420473">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat (<em>Triticum aestivum</em>)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alagu Manickavelu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kihara Institute for Biological Research</div>, <div class="insititution">Yokohama City University</div>, <div class="addr-line">Maioka-cho 641-12, Yokohama 244-0813</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alagu+Manickavelu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Manickavelu A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Manickavelu A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Alagu Manickavelu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kanako Kawaura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kihara Institute for Biological Research</div>, <div class="insititution">Yokohama City University</div>, <div class="addr-line">Maioka-cho 641-12, Yokohama 244-0813</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kanako+Kawaura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawaura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawaura K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kanako Kawaura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuko Oishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Genome Biology Laboratory</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuko+Oishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oishi K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oishi K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kazuko Oishi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadasu Shin-I</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Genome Biology Laboratory</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadasu+Shin-I">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shin-I T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shin-I T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tadasu Shin-I</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Genome Biology Laboratory</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuji+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuji Kohara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nabila Yahiaoui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Institute of Plant Biology</div>, <div class="insititution">University of Zurich</div>, <div class="addr-line">Zollikerstrasse 107, 8008 Zurich</div>, <div class="country">Switzerland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nabila+Yahiaoui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yahiaoui N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yahiaoui N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Nabila Yahiaoui</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Beat Keller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Institute of Plant Biology</div>, <div class="insititution">University of Zurich</div>, <div class="addr-line">Zollikerstrasse 107, 8008 Zurich</div>, <div class="country">Switzerland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Beat+Keller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Keller B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Keller B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Beat Keller</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reina Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Bioinformatics Laboratory, Faculty of Agriculture</div>, <div class="insititution">Meiji University</div>, <div class="addr-line">1-1-1 Higashi Mita, Kawasaki 214-8571</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reina+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Reina Abe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayako Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Bioinformatics Laboratory, Faculty of Agriculture</div>, <div class="insititution">Meiji University</div>, <div class="addr-line">1-1-1 Higashi Mita, Kawasaki 214-8571</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayako+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ayako Suzuki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taishi Nagayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Bioinformatics Laboratory, Faculty of Agriculture</div>, <div class="insititution">Meiji University</div>, <div class="addr-line">1-1-1 Higashi Mita, Kawasaki 214-8571</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taishi+Nagayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagayama T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagayama T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Taishi Nagayama</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kentaro Yano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Bioinformatics Laboratory, Faculty of Agriculture</div>, <div class="insititution">Meiji University</div>, <div class="addr-line">1-1-1 Higashi Mita, Kawasaki 214-8571</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kentaro+Yano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yano K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yano K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kentaro Yano</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasunari Ogihara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kihara Institute for Biological Research</div>, <div class="insititution">Yokohama City University</div>, <div class="addr-line">Maioka-cho 641-12, Yokohama 244-0813</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 45-820-2405. Fax. +81 45-820-1901. Email: <a href="mailto:yogihara@yokohama-cu.ac.jp" target="_blank">yogihara@yokohama-cu.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasunari+Ogihara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogihara Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogihara Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yasunari Ogihara</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: ICARDA, PO Box 5466, Aleppo, Syria.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 165–177, <a href='https://doi.org/10.1093/dnares/dss001'>https://doi.org/10.1093/dnares/dss001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 February 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 May 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 December 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="419961" href="/dnaresearch/article-pdf/19/2/165/1164316/dss001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Alagu Manickavelu, Kanako Kawaura, Kazuko Oishi, Tadasu Shin-I, Yuji Kohara, Nabila Yahiaoui, Beat Keller, Reina Abe, Ayako Suzuki, Taishi Nagayama, Kentaro Yano, Yasunari Ogihara;  Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat (<em>Triticum aestivum</em>), <em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 165–177, <a href='https://doi.org/10.1093/dnares/dss001'>https://doi.org/10.1093/dnares/dss001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/419961?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/419961?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/419961?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/419961?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/419961?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/419961?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/419961?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comprehensive%20Functional%20Analyses%20of%20Expressed%20Sequence%20Tags%20in%20Common%20Wheat%20(%26lt%3Bem%26gt%3BTriticum%20aestivum%26lt%3B%2Fem%26gt%3B)&amp;publicationDate=2012-02-14&amp;volumeNum=19&amp;issueNum=2&amp;author=Manickavelu%2C%20Alagu%3B%20Kawaura%2C%20Kanako&amp;startPage=165&amp;endPage=177&amp;contentId=10.1093%2Fdnares%2Fdss001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f165%2f419961" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6221280 id="6221280" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>About 1 million expressed sequence tag (EST) sequences comprising 125.3 Mb nucleotides were accreted from 51 cDNA libraries constructed from a variety of tissues and organs under a range of conditions, including abiotic stresses and pathogen challenges in common wheat (<em>Triticum aestivum</em>). Expressed sequence tags were assembled with stringent parameters after processing with inbuild scripts, resulting in 37 138 contigs and 215 199 singlets. In the assembled sequences, 10.6% presented no matches with existing sequences in public databases. Functional characterization of wheat unigenes by gene ontology annotation, mining transcription factors, full-length cDNA, and miRNA targeting sites were carried out. A bioinformatics strategy was developed to discover single-nucleotide polymorphisms (SNPs) within our large EST resource and reported the SNPs between and within (homoeologous) cultivars. Digital gene expression was performed to find the tissue-specific gene expression, and correspondence analysis was executed to identify common and specific gene expression by selecting four biotic stress-related libraries. The assembly and associated information cater a framework for future investigation in functional genomics.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="wheat">wheat</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ESTs">ESTs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="annotation">annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factors&quot;">transcription factors</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="miRNA">miRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;correspondence analysis&quot;">correspondence analysis</a></div>                            <h2 scrollto-destination=6221284 id="6221284" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Wheat provides 21% of food calories and 20% of protein to more than 4.5 billion people worldwide.<sup><a reveal-id="DSS001C1" data-open="DSS001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Demand for wheat in the developing world is projected to increase 60% by 2050. At the same time, climate change-induced temperature increases are estimated to reduce wheat production by 29%.<sup><a reveal-id="DSS001C2" data-open="DSS001C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>The advent of new molecular genetic technology and the dramatic increase in plant gene sequence data have provided opportunities to underpin wheat breeding programmes in order to improve yield, grain quality, and disease resistance.<sup><a reveal-id="DSS001C3" data-open="DSS001C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Many of these technologies have been designed to facilitate detection and understanding of the alterations in gene expression that accompany differential development or that result from the perception of changes to the environment. Expressed sequence tag (EST) projects provide a very useful and quick means of accessing gene sequence and expression information. When combined with breakthroughs in highly parallel designs for gene expression analysis, large-scale EST projects now offer new perspectives for understanding the molecular basis of important traits in plants of agricultural relevance.<sup><a reveal-id="DSS001C4" data-open="DSS001C4" class="link link-ref link-reveal xref-bibr">4</a></sup> EST sequencing projects have been completed or are under way for many plant species. These projects have provided useful tools for intragenomic<sup><a reveal-id="DSS001C5" data-open="DSS001C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and intergenomic<sup><a reveal-id="DSS001C6" data-open="DSS001C6" class="link link-ref link-reveal xref-bibr">6</a></sup> comparisons, gene discovery,<sup><a reveal-id="DSS001C7  DSS001C9" data-open="DSS001C7  DSS001C9" class="link link-ref link-reveal xref-bibr">7–9</a></sup> molecular marker identification,<sup><a reveal-id="DSS001C10" data-open="DSS001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> microarray development,<sup><a reveal-id="DSS001C11  DSS001C15" data-open="DSS001C11  DSS001C15" class="link link-ref link-reveal xref-bibr">11–15</a></sup> and polyploid species genomic resource development.<sup><a reveal-id="DSS001C16  DSS001C18" data-open="DSS001C16  DSS001C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup> As robot throughput increases and cost-per-read drops, determination of a sequence tag for a large proportion of genes is now reasonable using this random cDNA sequencing approach. For example, the availability of the complete genome sequence of <em>Arabidopsis thaliana</em> revealed that the 105 000 ESTs available at the end of 2000 were enough to tag 60% of the 25 500 genes.<sup><a reveal-id="DSS001C19" data-open="DSS001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The complete genome sequences of several plant species are known and the rate at which whole genomes are being sequenced is increasing. Correct annotation of these genomes remains problematic despite gene prediction algorithms becoming ever more sophisticated. While wheat genome sequencing is rapidly progressing (<a class="link link-uri" href="http://www.wheatgenome.org">www.wheatgenome.org</a>), a quicker and complementary approach to identifying a large number of wheat genes is EST and full-length cDNA sequencing. These resources will prove invaluable for annotating the genomes of wheat and other monocots and as substrates for transgenic improvement of crops. As in Arabidopsis and rice, these tools will prove to be critical in speeding up the genetic improvement of wheat.</p><p>DNA markers constructed from ESTs are effective since they are contained within an exon region of genes that are actually expressed. Examination of DNA sequence databases permits a direct search for sequence polymorphisms and thus molecular markers. These polymorphisms are typically single-nucleotide polymorphisms (SNPs) or small insertions–deletions (indels). More importantly, the SNPs are identified in EST sequences, thus the polymorphisms can be used to directly map functional, expressed genes, rather than DNA sequences derived from conventional RAPD and AFLP techniques, which are typically not genes. This has led to studies on linkage disequilibrium in genes to better characterize associations between phenotype and genotype.<sup><a reveal-id="DSS001C20  DSS001C22" data-open="DSS001C20  DSS001C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup> To identify an SNP from an EST database, the database must be composed of ESTs derived from different genotypes, followed by alignment of the same EST sequences from different genotypes.<sup><a reveal-id="DSS001C23" data-open="DSS001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> SNP markers rely upon the underlying redundancy within EST collections and assume that distinct genotype of a plant genome will be represented within a collection.</p><p>Earlier, we described extensive wheat EST resources including full-length sequences,<sup><a reveal-id="DSS001C24  DSS001C27" data-open="DSS001C24  DSS001C27" class="link link-ref link-reveal xref-bibr">24–27</a></sup> and the usage of wheat transcriptome analysis by making a custom microarray with ESTs.<sup><a reveal-id="DSS001C28" data-open="DSS001C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSS001C29" data-open="DSS001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Here, we extend our efforts and describe a collection of further ESTs and the complete functional analysis of the whole EST so far developed (∼1 million ESTs). Our work is based on a set of cDNA libraries established from 51 different tissues of interest varied from growth stages and biotic and abiotic stresses in 10 different cultivars.</p>                            <h2 scrollto-destination=6221294 id="6221294" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6221295 id="6221295" class="section-title" data-legacy-id=s2a>2.1. Plant materials and cDNA library construction</h3>
<p>Eighteen libraries from various growth stages, 25 libraries from abiotic stresses (cold, drought, saline, and mineral toxic), and eight libraries from biotic stresses (leaf rust, powdery mildew, and blast) were constructed from eight different wheat lines (Table <a reveal-id="DSS001TB1" data-open="DSS001TB1" class="link link-reveal link-table xref-fig">1</a>). Out of 51 libraries, 20 libraries were newly constructed and included for this study. Double-stranded cDNAs were synthesized as previously described.<sup><a reveal-id="DSS001C24" data-open="DSS001C24" class="link link-ref link-reveal xref-bibr">24</a></sup> cDNAs were ligated with pBlueScript SK(+) digested with <em>Eco</em>RI and <em>Xho</em>I. After transformation by electroporation, transformed bacterial cells were initially cultured in the SOC medium for 1 h before culture at 37°C for 2 h in 2× LB medium. Cultured cells were stored in 20% glycerol at −80°C until use. Transformed bacteria were randomly selected and plasmid DNA was extracted.<sup><a reveal-id="DSS001C24" data-open="DSS001C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Inserted cDNAs were sequenced from both ends using dye terminator cycle sequencing (Applied Biosystems, Foster City, CA, USA). </p>                        <a id="6221298" scrollto-destination="6221298"></a>
<div content-id="DSS001TB1" class="table-modal"><div class="table-wrap"><div id="DSS001TB1" class="table-wrap-title" data-id="DSS001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List and characteristics of cDNA libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library name </th><th>Genotype </th><th>Stage </th><th>Condition </th><th>No. of EST </th><th>Accession number </th></tr></thead><tbody><tr><td>whcs </td><td>CS </td><td>Callus </td><td>GS </td><td>11 505 </td><td>CJ518205–CJ523460, CJ627048–CJ632007 </td></tr><tr><td>whr </td><td>CS </td><td>Root </td><td>GS </td><td>19 227 </td><td>BJ277129–BJ287630 </td></tr><tr><td>whs </td><td>CS </td><td>Seedling </td><td>GS </td><td>13 356 </td><td>HX000001–HX010004 </td></tr><tr><td>whdl </td><td>CS </td><td>Seedling crown </td><td>GS </td><td>12 761 </td><td>BJ221844–BJ231912 </td></tr><tr><td>whh </td><td>CS </td><td>Spike at heading </td><td>GS </td><td>20 648 </td><td>BJ255495–BJ266779 </td></tr><tr><td>whf </td><td>CS </td><td>Spike at flowering </td><td>GS </td><td>21 106 </td><td>BJ243195–BJ255494 </td></tr><tr><td>whoh </td><td>CS </td><td>Pistil at heading </td><td>GS </td><td>20 736 </td><td>BJ266780–BJ277128 </td></tr><tr><td>whpc </td><td>CS </td><td>Anther at meiosis </td><td>GS </td><td>11 016 </td><td>CJ576197–CJ580898, CJ682880–CJ687382 </td></tr><tr><td>whhg </td><td>CSMT4B<sup>a</sup> </td><td>Anther at meiosis </td><td>GS </td><td>9669 </td><td>CJ549536–CJ554132, CJ657247–CJ661657 </td></tr><tr><td>whsh </td><td>CS </td><td>Young spikelet </td><td>GS </td><td>11 302 </td><td>CJ730709–CJ736986 </td></tr><tr><td>whyd </td><td>CS </td><td>Spikelet at late flowering </td><td>GS </td><td>14 708 </td><td>BJ300204–BJ312233 </td></tr><tr><td>whok </td><td>CS </td><td>DPA5 </td><td>GS </td><td>12 159 </td><td>CJ570869–CJ576196, CJ677747–CJ682879 </td></tr><tr><td>whms </td><td>CSDT3DL<sup>b</sup> </td><td>DPA5 </td><td>GS </td><td>12 036 </td><td>CJ565425–CJ570868, CJ672462–CJ677746 </td></tr><tr><td>whe </td><td>CS </td><td>DPA10 </td><td>GS </td><td>19 200 </td><td>BJ231913–BJ243194 </td></tr><tr><td>whdp </td><td>CS </td><td>DPA20 </td><td>GS </td><td>13 455 </td><td>CJ523461–CJ529179, CJ632008–CJ637596 </td></tr><tr><td>whsl </td><td>CS </td><td>DPA30 </td><td>GS </td><td>15 522 </td><td>BJ287631–BJ300203 </td></tr><tr><td>whsp </td><td>CS </td><td>Seedling </td><td>GS </td><td>12 783 </td><td>HX247045–HX247474 </td></tr><tr><td>whca </td><td>CS(Sp5A)<sup>c</sup> </td><td>Seedling </td><td>GS </td><td>794 </td><td>HX247475–HX257200 </td></tr><tr><td>whkp </td><td>CSKmppd<sup>d</sup> </td><td>Seedling </td><td>Grown under continuous light </td><td>12 950 </td><td>CJ554133–CJ559953, CJ661658–CJ667190 </td></tr><tr><td>whkv </td><td>CS </td><td>Seedling </td><td>Grown under continuous light after cold treatment </td><td>15 360 </td><td>CJ559954–CJ565424, CJ667191–CJ672461 </td></tr><tr><td>whem </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply </td><td>11 671 </td><td>CJ539482–CJ544724, CJ647722–CJ652633 </td></tr><tr><td>whei </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply after wounded </td><td>11 743 </td><td>CJ534326–CJ539481, CJ642661–CJ647721 </td></tr><tr><td>whsc </td><td>Kitakei1354 </td><td>Shoot </td><td>Cold treatment after excision of grain part </td><td>13 079 </td><td>CJ586310–CJ591776, CJ692607–CJ697797 </td></tr><tr><td>whsd </td><td>Kitakei1354 </td><td>Shoot </td><td>Dehydration </td><td>11 897 </td><td>CJ591777–CJ596845, CJ697798–CJ702615 </td></tr><tr><td>whrd </td><td>Kitakei1354 </td><td>Root </td><td>Dehydration </td><td>12 436 </td><td>CJ580899–CJ586309, CJ687383–CJ692606 </td></tr><tr><td>whv3 </td><td>Valuevskaya </td><td>Shoot </td><td>3 days cold condition </td><td>10 069 </td><td>CJ601934–CJ606680, CJ707296–CJ711731 </td></tr><tr><td>whv </td><td>Valuevskaya </td><td>Shoot </td><td>16 days cold condition </td><td>11 087 </td><td>CJ596846–CJ601933, CJ702616–CJ707295 </td></tr><tr><td>whva </td><td>Valuevskaya </td><td>Shoot </td><td>ABA treatment </td><td>10 631 </td><td>CJ606681–CJ611586, CJ711732–CJ715867 </td></tr><tr><td>whvd </td><td>Valuevskaya </td><td>Shoot </td><td>Five days dehydration </td><td>11 767 </td><td>CJ611587–CJ616926, CJ715868–CJ720922 </td></tr><tr><td>whvh </td><td>Valuevskaya </td><td>Shoot </td><td>Heat shock treatment </td><td>10 090 </td><td>CJ616927–CJ621531, CJ720923–CJ725419 </td></tr><tr><td>whvs </td><td>Valuevskaya </td><td>Liquid cultured cells </td><td>Liquid cultured cells </td><td>12 327 </td><td>CJ621532–CJ627047, CJ725420–CJ730708 </td></tr><tr><td>whrs6<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 6 h </td><td>13 110 </td><td>HX010005–HX019847 </td></tr><tr><td>whss6<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 6 h </td><td>13 312 </td><td>HX019848–HX030180 </td></tr><tr><td>whrs24<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 24 h </td><td>12 949 </td><td>HX030181–HX040252 </td></tr><tr><td>whss24<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 24 h </td><td>12 487 </td><td>HX040253–HX050054 </td></tr><tr><td>whatl<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>No treatment </td><td>20 519 </td><td>CJ773323–CJ797201 </td></tr><tr><td>whatlal<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>28 795 </td><td>CJ822818–CJ848636 </td></tr><tr><td>whsct<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>No treatment </td><td>27 717 </td><td>CJ797202–CJ822817 </td></tr><tr><td>whsctal<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>29 824 </td><td>CJ848637–CJ872807 </td></tr><tr><td>whhb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>No treatment </td><td>22 488 </td><td>HX124648–HX143755 </td></tr><tr><td>whhbb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 522 </td><td>HX143756–HX163093 </td></tr><tr><td>whcr<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>No treatment </td><td>22 680 </td><td>HX163094–HX182808 </td></tr><tr><td>whcrb<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 616 </td><td>HX182809–HX201765 </td></tr><tr><td>whthls<sup>e</sup> </td><td>Thatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>30 307 </td><td>CJ872808–CJ896490 </td></tr><tr><td>whthkles<sup>e</sup> </td><td>NILThatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>24 701 </td><td>CJ896491–CJ919993 </td></tr><tr><td>whchan<sup>e</sup> </td><td>Chancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>29 281 </td><td>CJ919994–CJ944155 </td></tr><tr><td>whchu<sup>e</sup> </td><td>NILChancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>28 799 </td><td>CJ944156–CJ968175 </td></tr><tr><td>whnr<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>No treatment </td><td>34 415 </td><td>HX050055–HX071918 </td></tr><tr><td>whnrpr48<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr48 at 23°C for 4 days </td><td>23 987 </td><td>HX071919–HX084716 </td></tr><tr><td>whnrpr58r<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 23°C for 4 days </td><td>36 360 </td><td>HX084717–HX106894 </td></tr><tr><td>whnrpr58s<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 27°C for 4 days </td><td>30 797 </td><td>HX106895–HX124647 </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td>894 756 </td><td> </td></tr></tbody></table></div><div content-id="DSS001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library name </th><th>Genotype </th><th>Stage </th><th>Condition </th><th>No. of EST </th><th>Accession number </th></tr></thead><tbody><tr><td>whcs </td><td>CS </td><td>Callus </td><td>GS </td><td>11 505 </td><td>CJ518205–CJ523460, CJ627048–CJ632007 </td></tr><tr><td>whr </td><td>CS </td><td>Root </td><td>GS </td><td>19 227 </td><td>BJ277129–BJ287630 </td></tr><tr><td>whs </td><td>CS </td><td>Seedling </td><td>GS </td><td>13 356 </td><td>HX000001–HX010004 </td></tr><tr><td>whdl </td><td>CS </td><td>Seedling crown </td><td>GS </td><td>12 761 </td><td>BJ221844–BJ231912 </td></tr><tr><td>whh </td><td>CS </td><td>Spike at heading </td><td>GS </td><td>20 648 </td><td>BJ255495–BJ266779 </td></tr><tr><td>whf </td><td>CS </td><td>Spike at flowering </td><td>GS </td><td>21 106 </td><td>BJ243195–BJ255494 </td></tr><tr><td>whoh </td><td>CS </td><td>Pistil at heading </td><td>GS </td><td>20 736 </td><td>BJ266780–BJ277128 </td></tr><tr><td>whpc </td><td>CS </td><td>Anther at meiosis </td><td>GS </td><td>11 016 </td><td>CJ576197–CJ580898, CJ682880–CJ687382 </td></tr><tr><td>whhg </td><td>CSMT4B<sup>a</sup> </td><td>Anther at meiosis </td><td>GS </td><td>9669 </td><td>CJ549536–CJ554132, CJ657247–CJ661657 </td></tr><tr><td>whsh </td><td>CS </td><td>Young spikelet </td><td>GS </td><td>11 302 </td><td>CJ730709–CJ736986 </td></tr><tr><td>whyd </td><td>CS </td><td>Spikelet at late flowering </td><td>GS </td><td>14 708 </td><td>BJ300204–BJ312233 </td></tr><tr><td>whok </td><td>CS </td><td>DPA5 </td><td>GS </td><td>12 159 </td><td>CJ570869–CJ576196, CJ677747–CJ682879 </td></tr><tr><td>whms </td><td>CSDT3DL<sup>b</sup> </td><td>DPA5 </td><td>GS </td><td>12 036 </td><td>CJ565425–CJ570868, CJ672462–CJ677746 </td></tr><tr><td>whe </td><td>CS </td><td>DPA10 </td><td>GS </td><td>19 200 </td><td>BJ231913–BJ243194 </td></tr><tr><td>whdp </td><td>CS </td><td>DPA20 </td><td>GS </td><td>13 455 </td><td>CJ523461–CJ529179, CJ632008–CJ637596 </td></tr><tr><td>whsl </td><td>CS </td><td>DPA30 </td><td>GS </td><td>15 522 </td><td>BJ287631–BJ300203 </td></tr><tr><td>whsp </td><td>CS </td><td>Seedling </td><td>GS </td><td>12 783 </td><td>HX247045–HX247474 </td></tr><tr><td>whca </td><td>CS(Sp5A)<sup>c</sup> </td><td>Seedling </td><td>GS </td><td>794 </td><td>HX247475–HX257200 </td></tr><tr><td>whkp </td><td>CSKmppd<sup>d</sup> </td><td>Seedling </td><td>Grown under continuous light </td><td>12 950 </td><td>CJ554133–CJ559953, CJ661658–CJ667190 </td></tr><tr><td>whkv </td><td>CS </td><td>Seedling </td><td>Grown under continuous light after cold treatment </td><td>15 360 </td><td>CJ559954–CJ565424, CJ667191–CJ672461 </td></tr><tr><td>whem </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply </td><td>11 671 </td><td>CJ539482–CJ544724, CJ647722–CJ652633 </td></tr><tr><td>whei </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply after wounded </td><td>11 743 </td><td>CJ534326–CJ539481, CJ642661–CJ647721 </td></tr><tr><td>whsc </td><td>Kitakei1354 </td><td>Shoot </td><td>Cold treatment after excision of grain part </td><td>13 079 </td><td>CJ586310–CJ591776, CJ692607–CJ697797 </td></tr><tr><td>whsd </td><td>Kitakei1354 </td><td>Shoot </td><td>Dehydration </td><td>11 897 </td><td>CJ591777–CJ596845, CJ697798–CJ702615 </td></tr><tr><td>whrd </td><td>Kitakei1354 </td><td>Root </td><td>Dehydration </td><td>12 436 </td><td>CJ580899–CJ586309, CJ687383–CJ692606 </td></tr><tr><td>whv3 </td><td>Valuevskaya </td><td>Shoot </td><td>3 days cold condition </td><td>10 069 </td><td>CJ601934–CJ606680, CJ707296–CJ711731 </td></tr><tr><td>whv </td><td>Valuevskaya </td><td>Shoot </td><td>16 days cold condition </td><td>11 087 </td><td>CJ596846–CJ601933, CJ702616–CJ707295 </td></tr><tr><td>whva </td><td>Valuevskaya </td><td>Shoot </td><td>ABA treatment </td><td>10 631 </td><td>CJ606681–CJ611586, CJ711732–CJ715867 </td></tr><tr><td>whvd </td><td>Valuevskaya </td><td>Shoot </td><td>Five days dehydration </td><td>11 767 </td><td>CJ611587–CJ616926, CJ715868–CJ720922 </td></tr><tr><td>whvh </td><td>Valuevskaya </td><td>Shoot </td><td>Heat shock treatment </td><td>10 090 </td><td>CJ616927–CJ621531, CJ720923–CJ725419 </td></tr><tr><td>whvs </td><td>Valuevskaya </td><td>Liquid cultured cells </td><td>Liquid cultured cells </td><td>12 327 </td><td>CJ621532–CJ627047, CJ725420–CJ730708 </td></tr><tr><td>whrs6<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 6 h </td><td>13 110 </td><td>HX010005–HX019847 </td></tr><tr><td>whss6<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 6 h </td><td>13 312 </td><td>HX019848–HX030180 </td></tr><tr><td>whrs24<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 24 h </td><td>12 949 </td><td>HX030181–HX040252 </td></tr><tr><td>whss24<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 24 h </td><td>12 487 </td><td>HX040253–HX050054 </td></tr><tr><td>whatl<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>No treatment </td><td>20 519 </td><td>CJ773323–CJ797201 </td></tr><tr><td>whatlal<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>28 795 </td><td>CJ822818–CJ848636 </td></tr><tr><td>whsct<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>No treatment </td><td>27 717 </td><td>CJ797202–CJ822817 </td></tr><tr><td>whsctal<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>29 824 </td><td>CJ848637–CJ872807 </td></tr><tr><td>whhb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>No treatment </td><td>22 488 </td><td>HX124648–HX143755 </td></tr><tr><td>whhbb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 522 </td><td>HX143756–HX163093 </td></tr><tr><td>whcr<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>No treatment </td><td>22 680 </td><td>HX163094–HX182808 </td></tr><tr><td>whcrb<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 616 </td><td>HX182809–HX201765 </td></tr><tr><td>whthls<sup>e</sup> </td><td>Thatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>30 307 </td><td>CJ872808–CJ896490 </td></tr><tr><td>whthkles<sup>e</sup> </td><td>NILThatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>24 701 </td><td>CJ896491–CJ919993 </td></tr><tr><td>whchan<sup>e</sup> </td><td>Chancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>29 281 </td><td>CJ919994–CJ944155 </td></tr><tr><td>whchu<sup>e</sup> </td><td>NILChancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>28 799 </td><td>CJ944156–CJ968175 </td></tr><tr><td>whnr<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>No treatment </td><td>34 415 </td><td>HX050055–HX071918 </td></tr><tr><td>whnrpr48<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr48 at 23°C for 4 days </td><td>23 987 </td><td>HX071919–HX084716 </td></tr><tr><td>whnrpr58r<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 23°C for 4 days </td><td>36 360 </td><td>HX084717–HX106894 </td></tr><tr><td>whnrpr58s<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 27°C for 4 days </td><td>30 797 </td><td>HX106895–HX124647 </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td>894 756 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>CS, Chinese Spring; GS, growth stage; DPA, days to post-anthesis; NIL, near-isogenic lines.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Mono-telosomic 4BS of CS.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Ditelosomic 4BS of CS.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Spelta5A chromosome substituted in CS.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Near-isogenic line.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Newly constructed libraries.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6221298" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS001TB1" class="table-wrap-title" data-id="DSS001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List and characteristics of cDNA libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library name </th><th>Genotype </th><th>Stage </th><th>Condition </th><th>No. of EST </th><th>Accession number </th></tr></thead><tbody><tr><td>whcs </td><td>CS </td><td>Callus </td><td>GS </td><td>11 505 </td><td>CJ518205–CJ523460, CJ627048–CJ632007 </td></tr><tr><td>whr </td><td>CS </td><td>Root </td><td>GS </td><td>19 227 </td><td>BJ277129–BJ287630 </td></tr><tr><td>whs </td><td>CS </td><td>Seedling </td><td>GS </td><td>13 356 </td><td>HX000001–HX010004 </td></tr><tr><td>whdl </td><td>CS </td><td>Seedling crown </td><td>GS </td><td>12 761 </td><td>BJ221844–BJ231912 </td></tr><tr><td>whh </td><td>CS </td><td>Spike at heading </td><td>GS </td><td>20 648 </td><td>BJ255495–BJ266779 </td></tr><tr><td>whf </td><td>CS </td><td>Spike at flowering </td><td>GS </td><td>21 106 </td><td>BJ243195–BJ255494 </td></tr><tr><td>whoh </td><td>CS </td><td>Pistil at heading </td><td>GS </td><td>20 736 </td><td>BJ266780–BJ277128 </td></tr><tr><td>whpc </td><td>CS </td><td>Anther at meiosis </td><td>GS </td><td>11 016 </td><td>CJ576197–CJ580898, CJ682880–CJ687382 </td></tr><tr><td>whhg </td><td>CSMT4B<sup>a</sup> </td><td>Anther at meiosis </td><td>GS </td><td>9669 </td><td>CJ549536–CJ554132, CJ657247–CJ661657 </td></tr><tr><td>whsh </td><td>CS </td><td>Young spikelet </td><td>GS </td><td>11 302 </td><td>CJ730709–CJ736986 </td></tr><tr><td>whyd </td><td>CS </td><td>Spikelet at late flowering </td><td>GS </td><td>14 708 </td><td>BJ300204–BJ312233 </td></tr><tr><td>whok </td><td>CS </td><td>DPA5 </td><td>GS </td><td>12 159 </td><td>CJ570869–CJ576196, CJ677747–CJ682879 </td></tr><tr><td>whms </td><td>CSDT3DL<sup>b</sup> </td><td>DPA5 </td><td>GS </td><td>12 036 </td><td>CJ565425–CJ570868, CJ672462–CJ677746 </td></tr><tr><td>whe </td><td>CS </td><td>DPA10 </td><td>GS </td><td>19 200 </td><td>BJ231913–BJ243194 </td></tr><tr><td>whdp </td><td>CS </td><td>DPA20 </td><td>GS </td><td>13 455 </td><td>CJ523461–CJ529179, CJ632008–CJ637596 </td></tr><tr><td>whsl </td><td>CS </td><td>DPA30 </td><td>GS </td><td>15 522 </td><td>BJ287631–BJ300203 </td></tr><tr><td>whsp </td><td>CS </td><td>Seedling </td><td>GS </td><td>12 783 </td><td>HX247045–HX247474 </td></tr><tr><td>whca </td><td>CS(Sp5A)<sup>c</sup> </td><td>Seedling </td><td>GS </td><td>794 </td><td>HX247475–HX257200 </td></tr><tr><td>whkp </td><td>CSKmppd<sup>d</sup> </td><td>Seedling </td><td>Grown under continuous light </td><td>12 950 </td><td>CJ554133–CJ559953, CJ661658–CJ667190 </td></tr><tr><td>whkv </td><td>CS </td><td>Seedling </td><td>Grown under continuous light after cold treatment </td><td>15 360 </td><td>CJ559954–CJ565424, CJ667191–CJ672461 </td></tr><tr><td>whem </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply </td><td>11 671 </td><td>CJ539482–CJ544724, CJ647722–CJ652633 </td></tr><tr><td>whei </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply after wounded </td><td>11 743 </td><td>CJ534326–CJ539481, CJ642661–CJ647721 </td></tr><tr><td>whsc </td><td>Kitakei1354 </td><td>Shoot </td><td>Cold treatment after excision of grain part </td><td>13 079 </td><td>CJ586310–CJ591776, CJ692607–CJ697797 </td></tr><tr><td>whsd </td><td>Kitakei1354 </td><td>Shoot </td><td>Dehydration </td><td>11 897 </td><td>CJ591777–CJ596845, CJ697798–CJ702615 </td></tr><tr><td>whrd </td><td>Kitakei1354 </td><td>Root </td><td>Dehydration </td><td>12 436 </td><td>CJ580899–CJ586309, CJ687383–CJ692606 </td></tr><tr><td>whv3 </td><td>Valuevskaya </td><td>Shoot </td><td>3 days cold condition </td><td>10 069 </td><td>CJ601934–CJ606680, CJ707296–CJ711731 </td></tr><tr><td>whv </td><td>Valuevskaya </td><td>Shoot </td><td>16 days cold condition </td><td>11 087 </td><td>CJ596846–CJ601933, CJ702616–CJ707295 </td></tr><tr><td>whva </td><td>Valuevskaya </td><td>Shoot </td><td>ABA treatment </td><td>10 631 </td><td>CJ606681–CJ611586, CJ711732–CJ715867 </td></tr><tr><td>whvd </td><td>Valuevskaya </td><td>Shoot </td><td>Five days dehydration </td><td>11 767 </td><td>CJ611587–CJ616926, CJ715868–CJ720922 </td></tr><tr><td>whvh </td><td>Valuevskaya </td><td>Shoot </td><td>Heat shock treatment </td><td>10 090 </td><td>CJ616927–CJ621531, CJ720923–CJ725419 </td></tr><tr><td>whvs </td><td>Valuevskaya </td><td>Liquid cultured cells </td><td>Liquid cultured cells </td><td>12 327 </td><td>CJ621532–CJ627047, CJ725420–CJ730708 </td></tr><tr><td>whrs6<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 6 h </td><td>13 110 </td><td>HX010005–HX019847 </td></tr><tr><td>whss6<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 6 h </td><td>13 312 </td><td>HX019848–HX030180 </td></tr><tr><td>whrs24<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 24 h </td><td>12 949 </td><td>HX030181–HX040252 </td></tr><tr><td>whss24<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 24 h </td><td>12 487 </td><td>HX040253–HX050054 </td></tr><tr><td>whatl<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>No treatment </td><td>20 519 </td><td>CJ773323–CJ797201 </td></tr><tr><td>whatlal<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>28 795 </td><td>CJ822818–CJ848636 </td></tr><tr><td>whsct<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>No treatment </td><td>27 717 </td><td>CJ797202–CJ822817 </td></tr><tr><td>whsctal<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>29 824 </td><td>CJ848637–CJ872807 </td></tr><tr><td>whhb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>No treatment </td><td>22 488 </td><td>HX124648–HX143755 </td></tr><tr><td>whhbb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 522 </td><td>HX143756–HX163093 </td></tr><tr><td>whcr<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>No treatment </td><td>22 680 </td><td>HX163094–HX182808 </td></tr><tr><td>whcrb<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 616 </td><td>HX182809–HX201765 </td></tr><tr><td>whthls<sup>e</sup> </td><td>Thatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>30 307 </td><td>CJ872808–CJ896490 </td></tr><tr><td>whthkles<sup>e</sup> </td><td>NILThatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>24 701 </td><td>CJ896491–CJ919993 </td></tr><tr><td>whchan<sup>e</sup> </td><td>Chancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>29 281 </td><td>CJ919994–CJ944155 </td></tr><tr><td>whchu<sup>e</sup> </td><td>NILChancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>28 799 </td><td>CJ944156–CJ968175 </td></tr><tr><td>whnr<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>No treatment </td><td>34 415 </td><td>HX050055–HX071918 </td></tr><tr><td>whnrpr48<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr48 at 23°C for 4 days </td><td>23 987 </td><td>HX071919–HX084716 </td></tr><tr><td>whnrpr58r<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 23°C for 4 days </td><td>36 360 </td><td>HX084717–HX106894 </td></tr><tr><td>whnrpr58s<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 27°C for 4 days </td><td>30 797 </td><td>HX106895–HX124647 </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td>894 756 </td><td> </td></tr></tbody></table></div><div content-id="DSS001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library name </th><th>Genotype </th><th>Stage </th><th>Condition </th><th>No. of EST </th><th>Accession number </th></tr></thead><tbody><tr><td>whcs </td><td>CS </td><td>Callus </td><td>GS </td><td>11 505 </td><td>CJ518205–CJ523460, CJ627048–CJ632007 </td></tr><tr><td>whr </td><td>CS </td><td>Root </td><td>GS </td><td>19 227 </td><td>BJ277129–BJ287630 </td></tr><tr><td>whs </td><td>CS </td><td>Seedling </td><td>GS </td><td>13 356 </td><td>HX000001–HX010004 </td></tr><tr><td>whdl </td><td>CS </td><td>Seedling crown </td><td>GS </td><td>12 761 </td><td>BJ221844–BJ231912 </td></tr><tr><td>whh </td><td>CS </td><td>Spike at heading </td><td>GS </td><td>20 648 </td><td>BJ255495–BJ266779 </td></tr><tr><td>whf </td><td>CS </td><td>Spike at flowering </td><td>GS </td><td>21 106 </td><td>BJ243195–BJ255494 </td></tr><tr><td>whoh </td><td>CS </td><td>Pistil at heading </td><td>GS </td><td>20 736 </td><td>BJ266780–BJ277128 </td></tr><tr><td>whpc </td><td>CS </td><td>Anther at meiosis </td><td>GS </td><td>11 016 </td><td>CJ576197–CJ580898, CJ682880–CJ687382 </td></tr><tr><td>whhg </td><td>CSMT4B<sup>a</sup> </td><td>Anther at meiosis </td><td>GS </td><td>9669 </td><td>CJ549536–CJ554132, CJ657247–CJ661657 </td></tr><tr><td>whsh </td><td>CS </td><td>Young spikelet </td><td>GS </td><td>11 302 </td><td>CJ730709–CJ736986 </td></tr><tr><td>whyd </td><td>CS </td><td>Spikelet at late flowering </td><td>GS </td><td>14 708 </td><td>BJ300204–BJ312233 </td></tr><tr><td>whok </td><td>CS </td><td>DPA5 </td><td>GS </td><td>12 159 </td><td>CJ570869–CJ576196, CJ677747–CJ682879 </td></tr><tr><td>whms </td><td>CSDT3DL<sup>b</sup> </td><td>DPA5 </td><td>GS </td><td>12 036 </td><td>CJ565425–CJ570868, CJ672462–CJ677746 </td></tr><tr><td>whe </td><td>CS </td><td>DPA10 </td><td>GS </td><td>19 200 </td><td>BJ231913–BJ243194 </td></tr><tr><td>whdp </td><td>CS </td><td>DPA20 </td><td>GS </td><td>13 455 </td><td>CJ523461–CJ529179, CJ632008–CJ637596 </td></tr><tr><td>whsl </td><td>CS </td><td>DPA30 </td><td>GS </td><td>15 522 </td><td>BJ287631–BJ300203 </td></tr><tr><td>whsp </td><td>CS </td><td>Seedling </td><td>GS </td><td>12 783 </td><td>HX247045–HX247474 </td></tr><tr><td>whca </td><td>CS(Sp5A)<sup>c</sup> </td><td>Seedling </td><td>GS </td><td>794 </td><td>HX247475–HX257200 </td></tr><tr><td>whkp </td><td>CSKmppd<sup>d</sup> </td><td>Seedling </td><td>Grown under continuous light </td><td>12 950 </td><td>CJ554133–CJ559953, CJ661658–CJ667190 </td></tr><tr><td>whkv </td><td>CS </td><td>Seedling </td><td>Grown under continuous light after cold treatment </td><td>15 360 </td><td>CJ559954–CJ565424, CJ667191–CJ672461 </td></tr><tr><td>whem </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply </td><td>11 671 </td><td>CJ539482–CJ544724, CJ647722–CJ652633 </td></tr><tr><td>whei </td><td>Kitakei1354 </td><td>Dormant seed </td><td>With water supply after wounded </td><td>11 743 </td><td>CJ534326–CJ539481, CJ642661–CJ647721 </td></tr><tr><td>whsc </td><td>Kitakei1354 </td><td>Shoot </td><td>Cold treatment after excision of grain part </td><td>13 079 </td><td>CJ586310–CJ591776, CJ692607–CJ697797 </td></tr><tr><td>whsd </td><td>Kitakei1354 </td><td>Shoot </td><td>Dehydration </td><td>11 897 </td><td>CJ591777–CJ596845, CJ697798–CJ702615 </td></tr><tr><td>whrd </td><td>Kitakei1354 </td><td>Root </td><td>Dehydration </td><td>12 436 </td><td>CJ580899–CJ586309, CJ687383–CJ692606 </td></tr><tr><td>whv3 </td><td>Valuevskaya </td><td>Shoot </td><td>3 days cold condition </td><td>10 069 </td><td>CJ601934–CJ606680, CJ707296–CJ711731 </td></tr><tr><td>whv </td><td>Valuevskaya </td><td>Shoot </td><td>16 days cold condition </td><td>11 087 </td><td>CJ596846–CJ601933, CJ702616–CJ707295 </td></tr><tr><td>whva </td><td>Valuevskaya </td><td>Shoot </td><td>ABA treatment </td><td>10 631 </td><td>CJ606681–CJ611586, CJ711732–CJ715867 </td></tr><tr><td>whvd </td><td>Valuevskaya </td><td>Shoot </td><td>Five days dehydration </td><td>11 767 </td><td>CJ611587–CJ616926, CJ715868–CJ720922 </td></tr><tr><td>whvh </td><td>Valuevskaya </td><td>Shoot </td><td>Heat shock treatment </td><td>10 090 </td><td>CJ616927–CJ621531, CJ720923–CJ725419 </td></tr><tr><td>whvs </td><td>Valuevskaya </td><td>Liquid cultured cells </td><td>Liquid cultured cells </td><td>12 327 </td><td>CJ621532–CJ627047, CJ725420–CJ730708 </td></tr><tr><td>whrs6<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 6 h </td><td>13 110 </td><td>HX010005–HX019847 </td></tr><tr><td>whss6<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 6 h </td><td>13 312 </td><td>HX019848–HX030180 </td></tr><tr><td>whrs24<sup>e</sup> </td><td>CS </td><td>Root </td><td>Salt stress for 24 h </td><td>12 949 </td><td>HX030181–HX040252 </td></tr><tr><td>whss24<sup>e</sup> </td><td>CS </td><td>Leaf </td><td>Salt stress for 24 h </td><td>12 487 </td><td>HX040253–HX050054 </td></tr><tr><td>whatl<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>No treatment </td><td>20 519 </td><td>CJ773323–CJ797201 </td></tr><tr><td>whatlal<sup>e</sup> </td><td>Atlas66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>28 795 </td><td>CJ822818–CJ848636 </td></tr><tr><td>whsct<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>No treatment </td><td>27 717 </td><td>CJ797202–CJ822817 </td></tr><tr><td>whsctal<sup>e</sup> </td><td>Scout66 </td><td>Root </td><td>50 mM Al for 6 h </td><td>29 824 </td><td>CJ848637–CJ872807 </td></tr><tr><td>whhb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>No treatment </td><td>22 488 </td><td>HX124648–HX143755 </td></tr><tr><td>whhbb<sup>e</sup> </td><td>Halberd </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 522 </td><td>HX143756–HX163093 </td></tr><tr><td>whcr<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>No treatment </td><td>22 680 </td><td>HX163094–HX182808 </td></tr><tr><td>whcrb<sup>e</sup> </td><td>Cranbrook </td><td>Root </td><td>10mM boric acid for 24 h </td><td>22 616 </td><td>HX182809–HX201765 </td></tr><tr><td>whthls<sup>e</sup> </td><td>Thatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>30 307 </td><td>CJ872808–CJ896490 </td></tr><tr><td>whthkles<sup>e</sup> </td><td>NILThatcher </td><td>Seedling </td><td>Infected with leaf rust </td><td>24 701 </td><td>CJ896491–CJ919993 </td></tr><tr><td>whchan<sup>e</sup> </td><td>Chancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>29 281 </td><td>CJ919994–CJ944155 </td></tr><tr><td>whchu<sup>e</sup> </td><td>NILChancellor </td><td>Seedling </td><td>Infected with powdry mildew </td><td>28 799 </td><td>CJ944156–CJ968175 </td></tr><tr><td>whnr<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>No treatment </td><td>34 415 </td><td>HX050055–HX071918 </td></tr><tr><td>whnrpr48<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr48 at 23°C for 4 days </td><td>23 987 </td><td>HX071919–HX084716 </td></tr><tr><td>whnrpr58r<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 23°C for 4 days </td><td>36 360 </td><td>HX084717–HX106894 </td></tr><tr><td>whnrpr58s<sup>e</sup> </td><td>Norin4 </td><td>Seedling </td><td>Infected with blast strain Pr58 at 27°C for 4 days </td><td>30 797 </td><td>HX106895–HX124647 </td></tr><tr><td> </td><td> </td><td> </td><td>Total </td><td>894 756 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>CS, Chinese Spring; GS, growth stage; DPA, days to post-anthesis; NIL, near-isogenic lines.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Mono-telosomic 4BS of CS.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Ditelosomic 4BS of CS.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Spelta5A chromosome substituted in CS.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Near-isogenic line.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Newly constructed libraries.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6221298" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6221299 id="6221299" class="section-title" data-legacy-id=s2b>2.2. EST processing and assembly</h3>
<p>The chromatogram files were base called and quality trimmed using PHRED<sup><a reveal-id="DSS001C30" data-open="DSS001C30" class="link link-ref link-reveal xref-bibr">30</a></sup> with default parameters. Vector, library linker-primer, and <em>Eco</em>RI adapter sequences were removed using CROSSMATCH. Repeat, ambiguous sequences (PHRED quality values &lt;30) and poly (A) tails or poly (T) sequences (at most 10 bases) in the ESTs were trimmed. Subsequently, ESTs with sequences &lt;30 bp were omitted from the final data set. The remaining high-quality sequence was used for further study. All sequence data are available from the DNA database of Japan (Table <a reveal-id="DSS001TB1" data-open="DSS001TB1" class="link link-reveal link-table xref-fig">1</a>). The processed EST sequence files were combined and assembled into contigs using the CAP3 program<sup><a reveal-id="DSS001C31" data-open="DSS001C31" class="link link-ref link-reveal xref-bibr">31</a></sup> with a high and low stringency level (high 95% homology in a 20 bp overlap; low 80% homology in a 15 bp overlap). Default CAP3 settings include -p 90 -h 20; the custom parameter settings used were -p 85 -h 90. The CAP3 -p option specifies overlap per cent identity cut-off, while the -h option specifies the maximum alignment overhang percentage.</p>                            <h4 scrollto-destination=6221302 id="6221302" class="section-title" data-legacy-id=s2b1>2.2.1. Sequence annotation</h4>
<p>Using the BLAST program (BLASTX with a search threshold of 1e−5), the sequences of the contigs were searched against seven databases (NCBI's nr; <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genbank">http://www.ncbi.nlm.nih.gov/genbank</a>, Uniprot; <a class="link link-uri" href="http://www.uniprot.org">http://www.uniprot.org</a>, RAP-DB; <a class="link link-uri" href="http://rapdb.dna.affrc.go.jp">http://rapdb.dna.affrc.go.jp</a>, RGAP; <a class="link link-uri" href="http://rice.plantbiology.msu.edu">http://rice.plantbiology.msu.edu</a>, Tair9; <a class="link link-uri" href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>, MaizeGDB; <a class="link link-uri" href="http://www.maizegdb.org">http://www.maizegdb.org</a>, and Brachypodium database; <a class="link link-uri" href="http://db.brachypodium.org">http://db.brachypodium.org</a>). According to Ewing <em>et al</em>.,<sup><a reveal-id="DSS001C7" data-open="DSS001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> only contigs were taken for further analyses. The gene ontology (GO) terms<sup><a reveal-id="DSS001C32" data-open="DSS001C32" class="link link-ref link-reveal xref-bibr">32</a></sup> of each contig was derived by InterProScan.<sup><a reveal-id="DSS001C33" data-open="DSS001C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The GO terms were then converted into GO slim term using EBI website (<a class="link link-uri" href="http://www.ebi.ac.uk/QuickGO/">http://www.ebi.ac.uk/QuickGO/</a>) by written perl-script for this purpose. Open reading frames (ORFs) were searched by translating sequences into amino acids by six frames (three per frame in the plus and minus strands).</p>                            <h3 scrollto-destination=6221304 id="6221304" class="section-title" data-legacy-id=s2c>2.3. Transcription factor</h3>
<p>The PlnTFDB<sup><a reveal-id="DSS001C34" data-open="DSS001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> containing 29 473 sequences of plant genes involved in transcriptional control was used to mine our data by local BLAST. The default parameters mentioned in the database was used for prediction (Filter ‘on’, gapped alignment ‘on’, substitution matrix ‘blosum 62’, <em>E</em>-value ≤1e−10). We included two meta-rules in our classification scheme: (i) if a protein harbours domains characteristic of a transcription factor (TF) family and a transcriptional regulators (TR) family, we assigned it to the TF family, (ii) when the protein of interest contains domains characteristic of more than one TF family or more than one TR family, it was assigned to the family to which its characteristic domains matched with the lowest <em>E</em>-value.</p>                            <h3 scrollto-destination=6221306 id="6221306" class="section-title" data-legacy-id=s2d>2.4. Full-length cDNA</h3>
<p>The contigs were classified as full length if it aligned with our full-length cDNA data.<sup><a reveal-id="DSS001C27" data-open="DSS001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> BLASTN searches of the contigs against the full-length sequences yielded a candidate hit list (<em>E</em>-value ≤1e−100) of putative full-length sequences that either covered the start and stop codon of the subject sequence or possessed sufficient sequence up/down-stream of the match to contain putative start and stop signals. In a few instances, some contigs covered all but the start methionine, and were also included as full-length sequences. In addition, contigs were aligned (<em>E</em>-value &lt;1e−5, ≥98% query coverage, ≥98% identity) with barley full-length sequences<sup><a reveal-id="DSS001C35" data-open="DSS001C35" class="link link-ref link-reveal xref-bibr">35</a></sup> to know the similarity as well as the full-length nature.</p>                            <h3 scrollto-destination=6221309 id="6221309" class="section-title" data-legacy-id=s2e>2.5. miRNA analysis</h3>
<p>To identify conserved miRNA in wheat, contigs were annotated with the plant small RNA regulator target analysis database (<a class="link link-uri" href="http://plantgrn.noble.org/psRNATarget/">http://plantgrn.noble.org/psRNATarget/</a>) containing small RNA of 15 plant species including wheat.<sup><a reveal-id="DSS001C36" data-open="DSS001C36" class="link link-ref link-reveal xref-bibr">36</a></sup> This database contained 2192 published miRNA sequences, including 32 from <em>Triticum aestivum</em>, 148 from sorghum, 496 from <em>Oryza sativa</em>, 319 from maize, and 224 from <em>A. thaliana</em>. Potential targets were predicted according to the rules applied by<sup><a reveal-id="DSS001C37" data-open="DSS001C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSS001C38" data-open="DSS001C38" class="link link-ref link-reveal xref-bibr">38</a></sup>: (i) the number of allowed mis-matches at complementary sites between miRNA sequences and potential mRNA targets is four or fewer; and (ii) no gaps are allowed at the complementary sites.</p>                            <h3 scrollto-destination=6221311 id="6221311" class="section-title" data-legacy-id=s2f>2.6. SNP discovery</h3>
<p>Sequence variants or SNPs were mined in wheat contigs with two criteria, and perl-scripts were written for each category. In the first criterion, only contigs with ≥4 ESTs were selected, and SNPs were declared only when there was no mismatch, no gaps, or N's were admitted before and after an SNP site; in addition, the alternative base to the consensus sequence was present at least more than twice in an alignment. In the second criterion, the SNPs were mined only in the significant sequence of contigs that was worked out by counting the nucleotides of either end of the contigs containing a minimum of four EST members. To find the SNP between cultivars, in addition to the above parameters, the contigs containing the minimum of two consistent EST from the same cultivar were selected. For homoeologous SNPs, the contigs containing the minimum of four EST from the same cultivar were chosen. The visual inspection of SNP was carried out using Tablet software.<sup><a reveal-id="DSS001C39" data-open="DSS001C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                            <h3 scrollto-destination=6221313 id="6221313" class="section-title" data-legacy-id=s2g>2.7. Digital gene expression and correspondence analysis</h3>
<p>For statistical analysis of gene expression profiles, contigs harbouring five or more constituents were selected from 37 138 contigs. Similarities between contigs or libraries were estimated using Pearson's correlation coefficient.<sup><a reveal-id="DSS001C40" data-open="DSS001C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Hierarchical clustering was applied to compare EST expression profiles among the 51 wheat tissue/treatments and libraries. Expression profiles are displayed based on the number of constituents in a contig (from 0 to 4647; red intensity), along with an increasing number of constituents. Contigs specific to DREB (dehydration-responsive element binding), NAC (nitrogen assimilation control), OMT (<em>O</em>-methyl transferase), and miRNA 172 were selected to show the differential gene expression in cultivars and growth stage.</p><p>Correspondence analysis (CA) was carried out by selecting four disease-related libraries (<em>whthls</em>, <em>whthkles</em>, <em>whchan</em>, and <em>whchul</em>; Table <a reveal-id="DSS001TB1" data-open="DSS001TB1" class="link link-reveal link-table xref-fig">1</a>) as per the procedure detailed in Hamada <em>et al</em>.<sup><a reveal-id="DSS001C41" data-open="DSS001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and visualized by a custom-build viewer (available based on request).</p>                            <h2 scrollto-destination=6221316 id="6221316" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6221317 id="6221317" class="section-title" data-legacy-id=s3a>3.1. cDNA library construction and assembly</h3>
<p>Our previous studies carried out the construction and analysis of 31 libraries.<sup><a reveal-id="DSS001C26" data-open="DSS001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Here, we have reported the further addition of 20 libraries and their combined analyses for comprehensive view of wheat EST. At the maximum, we have accumulated ∼1 million ESTs (Table <a reveal-id="DSS001TB1" data-open="DSS001TB1" class="link link-reveal link-table xref-fig">1</a>). The libraries were generated from developmental stages and stresses. After trimming low-quality bases, vector sequences, and shortness (&lt;30 bp), 0.68 million ESTs were used for CAP3 assembly under stringent conditions resulting in 37 138 contigs and 215 199 singlets. When assembled using relaxed settings in CAP3, 65 426 contigs and 66 875 singlets were obtained. The high stringent condition was chosen to achieve a more complete isolation of individual paralogues, orthologues, and homoeologues compared with using a low stringent condition.</p><p>The total sequence of transcript assemblies in stringent parameter settings, containing both the singlets and contigs, developed in this study was 125.3 Mb with the GC% of 51.9%. This is the maximum transcriptome sequences developed in any plant species. The GC% value is similar to rice but less than that reported in the wheat 3B chromosome exon coding sequence.<sup><a reveal-id="DSS001C42" data-open="DSS001C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The length of the singlets varied from 31 to 884 bp with an average of 430 bp. The maximum singlets were grouped under 500–600 bp lengths (Fig. <a reveal-id="DSS001F1" data-open="DSS001F1" class="link link-reveal link-table xref-fig">1</a>). As we found a large number of singlets, we subsequently discriminated the singlet contribution among the 10 cultivars (Fig. <a reveal-id="DSS001F1" data-open="DSS001F1" class="link link-reveal link-table xref-fig">1</a>B). There was no correlation observed between the number of ESTs and the singlets. However, the stress-related libraries from four cultivars contributed to 55% of the total singlets. The contig length ranged from 46 to 3960 bp with an average of 879 bp, and ∼70% of the contigs extended from 501 to 1000 bp (Fig. <a reveal-id="DSS001F2" data-open="DSS001F2" class="link link-reveal link-table xref-fig">2</a>A). The number of ESTs grouped in each contig varied between 2 and 4647, with 78% of the contigs containing 2–10 EST members (Fig. <a reveal-id="DSS001F2" data-open="DSS001F2" class="link link-reveal link-table xref-fig">2</a>B). </p>                        <a id="6221320" scrollto-destination="6221320"></a>
<div id="DSS001F1" data-id="DSS001F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00101.jpeg?Expires=2147483647&Signature=Xriqa0zPlMp4IiJO84UKRhE~HN7h0JkTUnfbLxEGmwjbQpP-W08KgRertIDDBeUsbt90XPIRhY0zHzv8ZjRYOe2zsWjB2cOEhLhYGmgbktE2gaO9DoCxht7WzgLGglnb8CnUqQytcs4jd0bOiccCGOcQUN1od4MkZgX38OuhaO1GqaLkd4ejGKuOilPw6bgmL5Y8o8l5axRgT4fk05KvTK90UsKE6b0BdrOJn2cSvMwJf~ahS74svmvwNpInmoraA-jXKGvW2g77hy2lD8vANXv~ZBIg3wyNH62qbMIkk0YWaBpQX2nnRtx0xJ8Mlx8B7BTswlo~OHn5hf9hiNxsRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of singlet sequence length and their genotype-wise distribution. (A) Sequence length distribution of singlet. (B) Genotype-wise frequency (%) of singlet (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya)." path-from-xml="dss00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221320/dss00101.jpeg" path-from-xml="dss00101.jpeg" target="_blank">View large</a><a section="6221320" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00101.jpeg?Expires=2147483647&Signature=tHnok0AhcWHjIg8wdKSnKzpxsDZdFFBwFCZVHGYnkYGWNoKrB--AQHQtrH1N74YlfbTkBYLxn55xL-LIHvtQehJpCaHDUO6ZxKxGNnzAb-TwP0M9vkX1uHJ2tUMliin-jpso3NgWCrOD6pkOKe6wPDKaLtlLzgZo4E13wyopti3nUN33DYoN43b0jFZrnrMOTHwQUbVnk~hPr-sg9WMuvMqgaxE8b880GwSUkKi9upKpgJCzkqkayXtabtJS8p4y5NSfUUIm86F~a2MXe3YJOVq-uhAtufVDSCdWGRoRn0LiDIs9xJbjj7TnODIetNFTixITm~3nL5dRNTEiFb3FmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221320&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of singlet sequence length and their genotype-wise distribution. (A) Sequence length distribution of singlet. (B) Genotype-wise frequency (%) of singlet (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya).</p></div></div><div content-id="DSS001F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00101.jpeg?Expires=2147483647&Signature=Xriqa0zPlMp4IiJO84UKRhE~HN7h0JkTUnfbLxEGmwjbQpP-W08KgRertIDDBeUsbt90XPIRhY0zHzv8ZjRYOe2zsWjB2cOEhLhYGmgbktE2gaO9DoCxht7WzgLGglnb8CnUqQytcs4jd0bOiccCGOcQUN1od4MkZgX38OuhaO1GqaLkd4ejGKuOilPw6bgmL5Y8o8l5axRgT4fk05KvTK90UsKE6b0BdrOJn2cSvMwJf~ahS74svmvwNpInmoraA-jXKGvW2g77hy2lD8vANXv~ZBIg3wyNH62qbMIkk0YWaBpQX2nnRtx0xJ8Mlx8B7BTswlo~OHn5hf9hiNxsRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of singlet sequence length and their genotype-wise distribution. (A) Sequence length distribution of singlet. (B) Genotype-wise frequency (%) of singlet (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya)." path-from-xml="dss00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221320/dss00101.jpeg" path-from-xml="dss00101.jpeg" target="_blank">View large</a><a section="6221320" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00101.jpeg?Expires=2147483647&Signature=tHnok0AhcWHjIg8wdKSnKzpxsDZdFFBwFCZVHGYnkYGWNoKrB--AQHQtrH1N74YlfbTkBYLxn55xL-LIHvtQehJpCaHDUO6ZxKxGNnzAb-TwP0M9vkX1uHJ2tUMliin-jpso3NgWCrOD6pkOKe6wPDKaLtlLzgZo4E13wyopti3nUN33DYoN43b0jFZrnrMOTHwQUbVnk~hPr-sg9WMuvMqgaxE8b880GwSUkKi9upKpgJCzkqkayXtabtJS8p4y5NSfUUIm86F~a2MXe3YJOVq-uhAtufVDSCdWGRoRn0LiDIs9xJbjj7TnODIetNFTixITm~3nL5dRNTEiFb3FmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221320&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of singlet sequence length and their genotype-wise distribution. (A) Sequence length distribution of singlet. (B) Genotype-wise frequency (%) of singlet (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya).</p></div></div>                        <a id="6221322" scrollto-destination="6221322"></a>
<div id="DSS001F2" data-id="DSS001F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00102.jpeg?Expires=2147483647&Signature=f2~u5hQblqiHgORO98WgbUcpt1nAqzxYJKogqcNxuxjymO~Qa2Nl6-YWomupyVjGCycTDZED-0c6YQaT6oJqJgo~WmUGvOW65lblEYsupEVARCUIcu~D9yah--S~dXr6rG7QKrHAKJ~lfa1Jkc6JoC--7YKiYX7TKaAHWLcvoT3mx6FeaEyZ2olu4F6ne7CyPymV6Za8kzcIH3w7KS6r1xAMnldKyiUqipbpIOI7ZHEBTeGRNss6IgWjSFIUmuv3uHQVzw5JHzw9qUmAk5-gZRMONyK6ZO1VuWsGTg9qyAdSywQG-XlzAhQPeylmSQvylkp7P7YDcdOAregFb4gjMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of contig length and their EST member constitution. (A) Sequence length frequency of contigs. (B) Number of EST members in contigs." path-from-xml="dss00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221322/dss00102.jpeg" path-from-xml="dss00102.jpeg" target="_blank">View large</a><a section="6221322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00102.jpeg?Expires=2147483647&Signature=ZWr~avwgFRCeAmNRq9KvCNm5Ex8aRamM-9AelxljYpyoXKcYC0vd8OupItYjILZpoMmG4WMMegqos9CS-Oxv8Vwa9VP5VaTykF~32C4Oms7ZdUzgxvZeKx7UZPNDZZxbsyt0nZ1wKq2c-dxotpemKCxbunMWSDHx8SENBx3ZDnY1LlA0fiEqv9F9o~g2alJBai47ofSgnkPxTqEDYzhQVZu2SQi~3tW~fVWjzb2kSdyiX-8lN7tPdij2b26Wio41wyvnpdIvXStsL68FlO94skIE6FCvbk6G75wGeCjkR4m2H4jNbzkg0RcRjhtR6oQaiB5KJSy9e7sfmQkDaYqzeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221322&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of contig length and their EST member constitution. (A) Sequence length frequency of contigs. (B) Number of EST members in contigs.</p></div></div><div content-id="DSS001F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00102.jpeg?Expires=2147483647&Signature=f2~u5hQblqiHgORO98WgbUcpt1nAqzxYJKogqcNxuxjymO~Qa2Nl6-YWomupyVjGCycTDZED-0c6YQaT6oJqJgo~WmUGvOW65lblEYsupEVARCUIcu~D9yah--S~dXr6rG7QKrHAKJ~lfa1Jkc6JoC--7YKiYX7TKaAHWLcvoT3mx6FeaEyZ2olu4F6ne7CyPymV6Za8kzcIH3w7KS6r1xAMnldKyiUqipbpIOI7ZHEBTeGRNss6IgWjSFIUmuv3uHQVzw5JHzw9qUmAk5-gZRMONyK6ZO1VuWsGTg9qyAdSywQG-XlzAhQPeylmSQvylkp7P7YDcdOAregFb4gjMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of contig length and their EST member constitution. (A) Sequence length frequency of contigs. (B) Number of EST members in contigs." path-from-xml="dss00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221322/dss00102.jpeg" path-from-xml="dss00102.jpeg" target="_blank">View large</a><a section="6221322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00102.jpeg?Expires=2147483647&Signature=ZWr~avwgFRCeAmNRq9KvCNm5Ex8aRamM-9AelxljYpyoXKcYC0vd8OupItYjILZpoMmG4WMMegqos9CS-Oxv8Vwa9VP5VaTykF~32C4Oms7ZdUzgxvZeKx7UZPNDZZxbsyt0nZ1wKq2c-dxotpemKCxbunMWSDHx8SENBx3ZDnY1LlA0fiEqv9F9o~g2alJBai47ofSgnkPxTqEDYzhQVZu2SQi~3tW~fVWjzb2kSdyiX-8lN7tPdij2b26Wio41wyvnpdIvXStsL68FlO94skIE6FCvbk6G75wGeCjkR4m2H4jNbzkg0RcRjhtR6oQaiB5KJSy9e7sfmQkDaYqzeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221322&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of contig length and their EST member constitution. (A) Sequence length frequency of contigs. (B) Number of EST members in contigs.</p></div></div>                            <h3 scrollto-destination=6221323 id="6221323" class="section-title" data-legacy-id=s3b>3.2. Functional annotation</h3>
<p>The contig resulted from stringent parameter assembly was used for functional annotation. The function of each contig was derived after annotation with rice, Arabidopsis, maize, and Brachypodium databases, in addition to the protein sequences in the GenBank nr database (BLASTX; <em>E</em>-value &lt;1e−5). The recently sequenced Brachypodium was included for its close originated relation with wheat. On annotation, maximum similarity was observed in Brachypodium followed by rice (Table <a reveal-id="DSS001TB2" data-open="DSS001TB2" class="link link-reveal link-table xref-fig">2</a>). In rice, further annotation was carried out to find the chromosome-wise sequence similarity and identified that chromosome 1 is having much co-linearity followed by chromosome 3 (Fig. <a reveal-id="DSS001F3" data-open="DSS001F3" class="link link-reveal link-table xref-fig">3</a>). On overall annotation, ∼3500 genes were found to have no similarity, suggesting new genes in our data. To further validate these new genes, updated tentative consensus sequences from the DFCI wheat gene index (<a class="link link-uri" href="http://compbio.dfci.harvard.edu/cgi-bin/tgi/gimain.pl?gudb=wheat">http://compbio.dfci.harvard.edu/cgi-bin/tgi/gimain.pl?gudb=wheat</a>) were annotated and resulted in the same number of new genes (∼3500), confirming the importance of our new EST assembly and analysis in wheat. To estimate the total number of full-length cDNAs in our collection, we searched our contig data against our 11 902 full-length cDNA data. With the stringent criteria of &gt;95% similarity and the expected cut-off value of &lt;1e−100, we found ∼7000 contigs that were full length in nature, further validated by identification of high similarity of wheat contigs with barley full-length sequences, indicating the robustness of our data and their applications to wheat functional genomics. </p>                        <a id="6221326" scrollto-destination="6221326"></a>
<div content-id="DSS001TB2" class="table-modal"><div class="table-wrap"><div id="DSS001TB2" class="table-wrap-title" data-id="DSS001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Annotation of wheat contigs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Database and Species </th><th>URL </th><th>No. and percentage of similarity </th></tr></thead><tbody><tr><td>RAP-DB (build 5) (Rice) </td><td><a class="link link-uri" href="http://rapdb.dna.affrc.go.jp">http://rapdb.dna.affrc.go.jp</a> </td><td>32.405 (87.26%) </td></tr><tr><td>RGAP (Rice) </td><td><a class="link link-uri" href="http://rice.plantbiology.msu.edu">http://rice.plantbiology.msu.edu</a> </td><td>32.430 (87.32%) </td></tr><tr><td>TAIR9 (Arabidopsis) </td><td><a class="link link-uri" href="http://www.arabidopsis.org">http://www.arabidopsis.org</a> </td><td>30.504 (82.14%) </td></tr><tr><td>MaizeGDB </td><td><a class="link link-uri" href="http://www.maizegdb.org/">http://www.maizegdb.org/</a> </td><td>31.206 (84.03%) </td></tr><tr><td>Brachypodium Database </td><td><a class="link link-uri" href="http://db.brachypodium.org">http://db.brachypodium.org</a> </td><td>32.522 (87.57%) </td></tr><tr><td>All databases </td><td> </td><td>33.909 (91.31%) </td></tr><tr><td>Total contigs </td><td> </td><td>37.138 </td></tr></tbody></table></div><div content-id="DSS001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Database and Species </th><th>URL </th><th>No. and percentage of similarity </th></tr></thead><tbody><tr><td>RAP-DB (build 5) (Rice) </td><td><a class="link link-uri" href="http://rapdb.dna.affrc.go.jp">http://rapdb.dna.affrc.go.jp</a> </td><td>32.405 (87.26%) </td></tr><tr><td>RGAP (Rice) </td><td><a class="link link-uri" href="http://rice.plantbiology.msu.edu">http://rice.plantbiology.msu.edu</a> </td><td>32.430 (87.32%) </td></tr><tr><td>TAIR9 (Arabidopsis) </td><td><a class="link link-uri" href="http://www.arabidopsis.org">http://www.arabidopsis.org</a> </td><td>30.504 (82.14%) </td></tr><tr><td>MaizeGDB </td><td><a class="link link-uri" href="http://www.maizegdb.org/">http://www.maizegdb.org/</a> </td><td>31.206 (84.03%) </td></tr><tr><td>Brachypodium Database </td><td><a class="link link-uri" href="http://db.brachypodium.org">http://db.brachypodium.org</a> </td><td>32.522 (87.57%) </td></tr><tr><td>All databases </td><td> </td><td>33.909 (91.31%) </td></tr><tr><td>Total contigs </td><td> </td><td>37.138 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The updated (until May 2011) sequence was retrieved and similarity search was carried out.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6221326" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS001TB2" class="table-wrap-title" data-id="DSS001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Annotation of wheat contigs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Database and Species </th><th>URL </th><th>No. and percentage of similarity </th></tr></thead><tbody><tr><td>RAP-DB (build 5) (Rice) </td><td><a class="link link-uri" href="http://rapdb.dna.affrc.go.jp">http://rapdb.dna.affrc.go.jp</a> </td><td>32.405 (87.26%) </td></tr><tr><td>RGAP (Rice) </td><td><a class="link link-uri" href="http://rice.plantbiology.msu.edu">http://rice.plantbiology.msu.edu</a> </td><td>32.430 (87.32%) </td></tr><tr><td>TAIR9 (Arabidopsis) </td><td><a class="link link-uri" href="http://www.arabidopsis.org">http://www.arabidopsis.org</a> </td><td>30.504 (82.14%) </td></tr><tr><td>MaizeGDB </td><td><a class="link link-uri" href="http://www.maizegdb.org/">http://www.maizegdb.org/</a> </td><td>31.206 (84.03%) </td></tr><tr><td>Brachypodium Database </td><td><a class="link link-uri" href="http://db.brachypodium.org">http://db.brachypodium.org</a> </td><td>32.522 (87.57%) </td></tr><tr><td>All databases </td><td> </td><td>33.909 (91.31%) </td></tr><tr><td>Total contigs </td><td> </td><td>37.138 </td></tr></tbody></table></div><div content-id="DSS001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Database and Species </th><th>URL </th><th>No. and percentage of similarity </th></tr></thead><tbody><tr><td>RAP-DB (build 5) (Rice) </td><td><a class="link link-uri" href="http://rapdb.dna.affrc.go.jp">http://rapdb.dna.affrc.go.jp</a> </td><td>32.405 (87.26%) </td></tr><tr><td>RGAP (Rice) </td><td><a class="link link-uri" href="http://rice.plantbiology.msu.edu">http://rice.plantbiology.msu.edu</a> </td><td>32.430 (87.32%) </td></tr><tr><td>TAIR9 (Arabidopsis) </td><td><a class="link link-uri" href="http://www.arabidopsis.org">http://www.arabidopsis.org</a> </td><td>30.504 (82.14%) </td></tr><tr><td>MaizeGDB </td><td><a class="link link-uri" href="http://www.maizegdb.org/">http://www.maizegdb.org/</a> </td><td>31.206 (84.03%) </td></tr><tr><td>Brachypodium Database </td><td><a class="link link-uri" href="http://db.brachypodium.org">http://db.brachypodium.org</a> </td><td>32.522 (87.57%) </td></tr><tr><td>All databases </td><td> </td><td>33.909 (91.31%) </td></tr><tr><td>Total contigs </td><td> </td><td>37.138 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The updated (until May 2011) sequence was retrieved and similarity search was carried out.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6221326" target="_blank">View Large</a></div></div>                        <a id="6221327" scrollto-destination="6221327"></a>
<div id="DSS001F3" data-id="DSS001F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00103.jpeg?Expires=2147483647&Signature=xw1Ddm2sBfNJpvJd5shD4Zu9gx4k9N2W7Usa2zSybhYhA4lQXLmJ3frOem2yfiL467Mi804s8Ew38gKQRB6SHOqv8bLVxBBxdDJb9-B4ijzkhrlHNAmCZ7PGmBO8PB95yfTga2-0tH~DZGCR4sY7NlL3F3Mcpw753t18fNNVcyvpuH~JeC0u~iQUjIOO9n7zHGDGvmY115jPxEFFqou~51QfVpZLcMKYdMifGk1yeD8EJb-h9fmyTkiC~bXfvQ34MfGFsGvRq9fgmyyAcwFqPbxSQze3NOZc1h-HIPCz4rHYVnOISWETSgzTP3dKI9V7pAsz8ygZVeubHCmNHDeNfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence similarity of wheat contigs with rice genome. Based on the result, the contig was grouped in rice chromosome wise." path-from-xml="dss00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221327/dss00103.jpeg" path-from-xml="dss00103.jpeg" target="_blank">View large</a><a section="6221327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00103.jpeg?Expires=2147483647&Signature=deWN-dTrmW-sRxS3eSrJsdJd3njtzOdje4izV7vjDwTwidoFL3j3-35lF2iuPab1XoSpRBaNgs2weGHXVc-uKPzN1rXxPWvEOCz8aiXDy4nSOItdyA8~RmWgEw3Vk~nS6myvI4kb~q~xj-mfz6~aaRt2H-S6cIUpXfdLUVtsc-SORx5suywkS5taMbgCVI5umSEqVKA4rHTC29n3pqaTIx5ZRcL8ZOXJ4i~oorXmHBDMsqM1PlQN9bZRNS84fe5xHBylB7~9c59NHb7f9jsehx8JaypqQbnZ2rWwWvtqgZjsQ6DvxMf3sPgKxEn2mzDDIgoEH4-KxY7g38BEeHCoCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221327&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence similarity of wheat contigs with rice genome. Based on the result, the contig was grouped in rice chromosome wise.</p></div></div><div content-id="DSS001F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00103.jpeg?Expires=2147483647&Signature=xw1Ddm2sBfNJpvJd5shD4Zu9gx4k9N2W7Usa2zSybhYhA4lQXLmJ3frOem2yfiL467Mi804s8Ew38gKQRB6SHOqv8bLVxBBxdDJb9-B4ijzkhrlHNAmCZ7PGmBO8PB95yfTga2-0tH~DZGCR4sY7NlL3F3Mcpw753t18fNNVcyvpuH~JeC0u~iQUjIOO9n7zHGDGvmY115jPxEFFqou~51QfVpZLcMKYdMifGk1yeD8EJb-h9fmyTkiC~bXfvQ34MfGFsGvRq9fgmyyAcwFqPbxSQze3NOZc1h-HIPCz4rHYVnOISWETSgzTP3dKI9V7pAsz8ygZVeubHCmNHDeNfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence similarity of wheat contigs with rice genome. Based on the result, the contig was grouped in rice chromosome wise." path-from-xml="dss00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221327/dss00103.jpeg" path-from-xml="dss00103.jpeg" target="_blank">View large</a><a section="6221327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00103.jpeg?Expires=2147483647&Signature=deWN-dTrmW-sRxS3eSrJsdJd3njtzOdje4izV7vjDwTwidoFL3j3-35lF2iuPab1XoSpRBaNgs2weGHXVc-uKPzN1rXxPWvEOCz8aiXDy4nSOItdyA8~RmWgEw3Vk~nS6myvI4kb~q~xj-mfz6~aaRt2H-S6cIUpXfdLUVtsc-SORx5suywkS5taMbgCVI5umSEqVKA4rHTC29n3pqaTIx5ZRcL8ZOXJ4i~oorXmHBDMsqM1PlQN9bZRNS84fe5xHBylB7~9c59NHb7f9jsehx8JaypqQbnZ2rWwWvtqgZjsQ6DvxMf3sPgKxEn2mzDDIgoEH4-KxY7g38BEeHCoCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221327&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence similarity of wheat contigs with rice genome. Based on the result, the contig was grouped in rice chromosome wise.</p></div></div><p>We also analysed length and ORF distribution in the contigs for both plus and minus strands. To obtain meaningful results, only contigs with &gt;5 ESTs were selected and ORFs were identified. GO annotation of the wheat contigs was performed on the basis of ORF mining of the data. The GO terms were organized into three categories representing molecular functions, biological processes, and cellular components.<sup><a reveal-id="DSS001C32" data-open="DSS001C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The sum of the wheat contigs per category did not add up to 100% as some contigs were classified into more than one category. Of the total contig set, 21 125 (56%) were annotated into the molecular function category (describing the biochemical activity performed by the gene product), 13 354 (36%) into the biological process GO category (describing the ordered assembly of more than one molecular function), and 13 356 (36%) into the cellular component GO category (describing subcellular compartments of a cell) (Fig. <a reveal-id="DSS001F4" data-open="DSS001F4" class="link link-reveal link-table xref-fig">4</a>). Among the molecular function, the most highly represented categories were binding, catalytic activity, redox activity, and structural activity (Fig. <a reveal-id="DSS001F4" data-open="DSS001F4" class="link link-reveal link-table xref-fig">4</a>A). Among the biological processes, the largest proportion of functionally assigned contigs fell into metabolic, transport, and translation processes, while redox activity, biosynthetic process, regulation, phosphorylation, and transcription comprised 34% of the contigs (Fig. <a reveal-id="DSS001F4" data-open="DSS001F4" class="link link-reveal link-table xref-fig">4</a>B). For the cell component category, almost all contig sequences were annotated into the cell–cell subcategory, 28% into the membrane category, and 23% into the intracellular category (Fig. <a reveal-id="DSS001F4" data-open="DSS001F4" class="link link-reveal link-table xref-fig">4</a>C). Together, all three GO categories accounted for ∼82% of the assigned wheat contig set. </p>                        <a id="6221329" scrollto-destination="6221329"></a>
<div id="DSS001F4" data-id="DSS001F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00104.jpeg?Expires=2147483647&Signature=E9ZcJkTl7fkMBfHHA2es6Jt8DsCmNQnBu-Rz51ahhOsharDkGuA0kLGKatmkqfZtyvmUb9QNrEp7RmO-vfN5Izq1CW4RW8S3yRp75yOkAD9NjX8oXBtxdBl3at5lmgpl7w9QvGU8lVJjQCTI47-HD1c06WTYbiN~UE68h3Mbyc55PZH5NqyvgETCv8nBT5LzgZ9NvL42W1wkdxQEFvbVzpd4qXAvcPrtCvEZ5~z0qnjMv-UJe42MQOqlsT032uGd6FS7~9h-AWsg0Hb9UQSOsTtP5ro8sCTCC2Ld1RE7EZit0OU26jjN4qnBe~IIA9TTqrzAg1BOMDGZS0Yq-q5dVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of contig sequences based on GO categorization. Sequences were evaluated for their predicted involvement in molecular function, biological process, and cellular component." path-from-xml="dss00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221329/dss00104.jpeg" path-from-xml="dss00104.jpeg" target="_blank">View large</a><a section="6221329" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00104.jpeg?Expires=2147483647&Signature=koSKvzhf0ExyzpGuye5CUY0qAcPLZEIQaUzIb3IGZbXqqtiS5LiDvm0592pLELTjRp9hvn317R6fsJT2D~mW3~oKYOTst77KJ6b5LxVeBzDUDH~i5P7IndyZBvMuhLv8DPF87IIGGOamKq5uMsZ85qAbqqJ280SowgYzlgNTQUtoKdl2d2IostjG7k9zLB-kvr43Qt0ztvr9TzZwzMOao3Toz0Za0YZ9u8NrfuIL6krc5QWgMaiezbAcw3o7iB9BoUjomRgIOht1x-3h8OMJkjN5~CM7KE2lsFf7l7Pwxg9m29Dh6Nqd86lYrJrWBVm~hBbsds3SnNxXDrnvqlv43g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221329&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of contig sequences based on GO categorization. Sequences were evaluated for their predicted involvement in molecular function, biological process, and cellular component.</p></div></div><div content-id="DSS001F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00104.jpeg?Expires=2147483647&Signature=E9ZcJkTl7fkMBfHHA2es6Jt8DsCmNQnBu-Rz51ahhOsharDkGuA0kLGKatmkqfZtyvmUb9QNrEp7RmO-vfN5Izq1CW4RW8S3yRp75yOkAD9NjX8oXBtxdBl3at5lmgpl7w9QvGU8lVJjQCTI47-HD1c06WTYbiN~UE68h3Mbyc55PZH5NqyvgETCv8nBT5LzgZ9NvL42W1wkdxQEFvbVzpd4qXAvcPrtCvEZ5~z0qnjMv-UJe42MQOqlsT032uGd6FS7~9h-AWsg0Hb9UQSOsTtP5ro8sCTCC2Ld1RE7EZit0OU26jjN4qnBe~IIA9TTqrzAg1BOMDGZS0Yq-q5dVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of contig sequences based on GO categorization. Sequences were evaluated for their predicted involvement in molecular function, biological process, and cellular component." path-from-xml="dss00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221329/dss00104.jpeg" path-from-xml="dss00104.jpeg" target="_blank">View large</a><a section="6221329" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00104.jpeg?Expires=2147483647&Signature=koSKvzhf0ExyzpGuye5CUY0qAcPLZEIQaUzIb3IGZbXqqtiS5LiDvm0592pLELTjRp9hvn317R6fsJT2D~mW3~oKYOTst77KJ6b5LxVeBzDUDH~i5P7IndyZBvMuhLv8DPF87IIGGOamKq5uMsZ85qAbqqJ280SowgYzlgNTQUtoKdl2d2IostjG7k9zLB-kvr43Qt0ztvr9TzZwzMOao3Toz0Za0YZ9u8NrfuIL6krc5QWgMaiezbAcw3o7iB9BoUjomRgIOht1x-3h8OMJkjN5~CM7KE2lsFf7l7Pwxg9m29Dh6Nqd86lYrJrWBVm~hBbsds3SnNxXDrnvqlv43g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221329&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of contig sequences based on GO categorization. Sequences were evaluated for their predicted involvement in molecular function, biological process, and cellular component.</p></div></div><p>The role and importance of TF lead to mining of our data and resulted in 1183 contigs containing either single or multiple transcription factors. Among the TFs, the <em>CCAAT</em> family was found in as many as 69 contigs. miRNA target sequence analysis of the wheat transcriptome identified different miRNA target sequences in 5180 contigs which ranged from 19 to 24 nt long. The majority of the small RNAs are 20–24 nt long, which is a typical range for dicer-derived products; the 21-nt class is predominant. Among species-specific miRNA, rice had maximum homology followed by maize and <em>Medicago truncatula</em> (Fig. <a reveal-id="DSS001F5" data-open="DSS001F5" class="link link-reveal link-table xref-fig">5</a>). The number of hits for each species is roughly proportional to the number of sequences for that species in the database. Due to the limited number of wheat miRNA sequences in the database, there was only 200 contigs with wheat-specific miRNA target sequences. Among miRNAs, miRNA 395, 172, and 164 target sequences alone were found in 831 contigs, showing the relative abundance of these miRNA target sequences in wheat. </p>                        <a id="6221333" scrollto-destination="6221333"></a>
<div id="DSS001F5" data-id="DSS001F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00105.jpeg?Expires=2147483647&Signature=XZmZC9jFHput1C-i7-Xy~IjlORQhv5hcO~nuLAjvyF9RpTlfiJGzTBRzClGpsxmeEVr5yYNdEY0-NH9vBt8uUVIzO-PdTRa2MfdaqjUOVNMN9u-~UERnDHn0klm-GO7kOgDtCPJ9Xthc0ZOk6PMjp21b16gLnfLbps8hkz39L4s1hE0NkXYMT4zGrKxbryzv3angOpJLSJ8cbkPCUOT-kX9eJTEz5BM0gUpYhi2v-VTiOZZGASBAG4oeEKe8WsUZDMn63faYWWf9ZfgV4O3MRvTyDhyOS0cofTnMFGorFCvU9xj2xtQZFsxpuuvy6dF-Y5ZaAS2Kp3~g8LZ4nnC7~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="miRNA target sequence analysis in wheat contigs. The database bars indicate the available miRNA in the database and the hit bars indicate the number of wheat genes having miRNA target sequence. tae, Triticum; sbi, Sorghum bicolor; osa, Oryza sativa; zma, Zea mays; ath, Arabidopsis thaliana; mtr, Medicago truncatula; ghr, Gossibium hirsutum; ptc, Populus trichocorpa; bna, Brassica napus; gma, Glycine max; pta, Pinus taeda; sly, Solanum lycopersicum; bra, Brassica rapa; bol, Brassica oleraceae; cre, Chlamydomonas reinhardtii." path-from-xml="dss00105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221333/dss00105.jpeg" path-from-xml="dss00105.jpeg" target="_blank">View large</a><a section="6221333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00105.jpeg?Expires=2147483647&Signature=Nh4fdAJ5rnSVyZSKJZkJTkPgFm9pg5fOSCa5PJdpZSe~QdagTDOIF2ENS9Yb4BhdAQ1N4RbHxak9ZSITHvTIJdP5-YhUCOyHvnGv4tiWkuIL6QQ1TBKg80Xd69hAIluzeEQ2tJNJf9eo6SrXvIjn8nFS~0p6e87B-WUpLxXTcmhJLKfMeOOhNmK83e~O7nRTyykw89JKBSv6r65aLnye-wdLo7MLkRz-A98~I0s4df6w8pTSlYFmLcKPLkWwfL5OplaS9w6ijFFE39HT35BX366BMlDa5eNHrQnpiDdyPGtX3TJIP6Wpmb2-On4Pn-zoMCwyldPDH1qbKB-LABjcXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221333&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>miRNA</em> target sequence analysis in wheat contigs. The database bars indicate the available <em>miRNA</em> in the database and the hit bars indicate the number of wheat genes having <em>miRNA</em> target sequence. tae, <em>Triticum</em>; sbi, <em>Sorghum bicolor</em>; osa, <em>Oryza sativa</em>; zma, <em>Zea mays</em>; ath, <em>Arabidopsis thaliana</em>; mtr, <em>Medicago truncatula</em>; ghr, <em>Gossibium hirsutum</em>; ptc, <em>Populus trichocorpa</em>; bna, <em>Brassica napus</em>; gma, <em>Glycine max</em>; pta, <em>Pinus taeda</em>; sly, <em>Solanum lycopersicum</em>; bra, <em>Brassica rapa</em>; bol, <em>Brassica oleraceae</em>; cre, <em>Chlamydomonas reinhardtii</em>.</p></div></div><div content-id="DSS001F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00105.jpeg?Expires=2147483647&Signature=XZmZC9jFHput1C-i7-Xy~IjlORQhv5hcO~nuLAjvyF9RpTlfiJGzTBRzClGpsxmeEVr5yYNdEY0-NH9vBt8uUVIzO-PdTRa2MfdaqjUOVNMN9u-~UERnDHn0klm-GO7kOgDtCPJ9Xthc0ZOk6PMjp21b16gLnfLbps8hkz39L4s1hE0NkXYMT4zGrKxbryzv3angOpJLSJ8cbkPCUOT-kX9eJTEz5BM0gUpYhi2v-VTiOZZGASBAG4oeEKe8WsUZDMn63faYWWf9ZfgV4O3MRvTyDhyOS0cofTnMFGorFCvU9xj2xtQZFsxpuuvy6dF-Y5ZaAS2Kp3~g8LZ4nnC7~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="miRNA target sequence analysis in wheat contigs. The database bars indicate the available miRNA in the database and the hit bars indicate the number of wheat genes having miRNA target sequence. tae, Triticum; sbi, Sorghum bicolor; osa, Oryza sativa; zma, Zea mays; ath, Arabidopsis thaliana; mtr, Medicago truncatula; ghr, Gossibium hirsutum; ptc, Populus trichocorpa; bna, Brassica napus; gma, Glycine max; pta, Pinus taeda; sly, Solanum lycopersicum; bra, Brassica rapa; bol, Brassica oleraceae; cre, Chlamydomonas reinhardtii." path-from-xml="dss00105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221333/dss00105.jpeg" path-from-xml="dss00105.jpeg" target="_blank">View large</a><a section="6221333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00105.jpeg?Expires=2147483647&Signature=Nh4fdAJ5rnSVyZSKJZkJTkPgFm9pg5fOSCa5PJdpZSe~QdagTDOIF2ENS9Yb4BhdAQ1N4RbHxak9ZSITHvTIJdP5-YhUCOyHvnGv4tiWkuIL6QQ1TBKg80Xd69hAIluzeEQ2tJNJf9eo6SrXvIjn8nFS~0p6e87B-WUpLxXTcmhJLKfMeOOhNmK83e~O7nRTyykw89JKBSv6r65aLnye-wdLo7MLkRz-A98~I0s4df6w8pTSlYFmLcKPLkWwfL5OplaS9w6ijFFE39HT35BX366BMlDa5eNHrQnpiDdyPGtX3TJIP6Wpmb2-On4Pn-zoMCwyldPDH1qbKB-LABjcXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221333&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>miRNA</em> target sequence analysis in wheat contigs. The database bars indicate the available <em>miRNA</em> in the database and the hit bars indicate the number of wheat genes having <em>miRNA</em> target sequence. tae, <em>Triticum</em>; sbi, <em>Sorghum bicolor</em>; osa, <em>Oryza sativa</em>; zma, <em>Zea mays</em>; ath, <em>Arabidopsis thaliana</em>; mtr, <em>Medicago truncatula</em>; ghr, <em>Gossibium hirsutum</em>; ptc, <em>Populus trichocorpa</em>; bna, <em>Brassica napus</em>; gma, <em>Glycine max</em>; pta, <em>Pinus taeda</em>; sly, <em>Solanum lycopersicum</em>; bra, <em>Brassica rapa</em>; bol, <em>Brassica oleraceae</em>; cre, <em>Chlamydomonas reinhardtii</em>.</p></div></div>                            <h3 scrollto-destination=6221334 id="6221334" class="section-title" data-legacy-id=s3c>3.3. Sequence polymorphism/SNP mining</h3>
<p>The SNPs were mined in our large-scale wheat transcriptome data by applying both relax and stringent criteria. For both criteria, to find reliable SNPs, the common conditions of SNP should be present at a given position when there is no mismatch present 2 bp before or after the SNP site. A total of 51 067 SNPs were detected from 20 609 contigs using the first criterion, resulting in the identification of an SNP site every 96 bp. This value is considerably higher than those reported for earlier studies of wheat.<sup><a reveal-id="DSS001C23" data-open="DSS001C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSS001C43" data-open="DSS001C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Hence, the second criterion was applied with an interest to differentiate the homoeologous SNPs from intergenome SNP by calculating the significant sequence length of each contig before mining the SNPs. This approach avoided finding SNPs on either end of the contigs and resulted in the identification of only 6352 SNPs in the wheat contigs. Further classification of the SNPs present between cultivars found 3515 SNPs with a frequency of one SNP per 614 bp. As there were genome constituents of wheat (three homologous genomes) and selective gene expression among the three genomes, we examined the SNP within each cultivar and found 2837 SNPs with an SNP site every 470 bp. The overall SNP frequency based on the stringent criteria was one SNP per 483 bp. Transitions (70%) were more frequent than transversions (30%). As expected, a significant positive correlation (<em>P</em> &lt; 0.05) was observed between the number of SNPs detected in a contig and the number of reads present in that contig. The cultivars, except Chinese spring and Norin4, contain more inter-cultivar sequence variation than homoeologous SNPs. Among cultivars, Halberd, Valuevskaya, and Cranbrook contain almost the same number of SNPs in both cases. Cultivars Atlas, Kitakei1354, Scout, and Thatcher had much less homoeologous sequence variation than cultivar differences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h3 scrollto-destination=6221337 id="6221337" class="section-title" data-legacy-id=s3d>3.4. Digital gene expression</h3>
<p>EST frequencies approximate the message abundance in the mRNA population used to construct a cDNA library. We have already attempted to make tissue expression maps of a large number of ESTs from stress-related libraries for <em>in silico</em> screening of stress responsive genes in wheat.<sup><a reveal-id="DSS001C26" data-open="DSS001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Here, we aimed to determine the global gene expression of wheat from 51 cDNA libraries, including growth stages and biotic and abiotic stresses. Contigs containing &gt;5 ESTs were subjected to a correlated clustering analysis<sup><a reveal-id="DSS001C7" data-open="DSS001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> to compare expression profiles in the different libraries. When the result was displayed in the form of a dendrogram (Fig. <a reveal-id="DSS001F6" data-open="DSS001F6" class="link link-reveal link-table xref-fig">6</a>), many libraries with similar origins agglomerated together. All four libraries derived from root tissues treated with boric acid and aluminium united. In the same manner, biotic stress (leaf rust, powdery mildew, and blast)-related tissues were grouped in the same clade. The tissues collected from cv. Valucvskaya, which mainly undergo abiotic stress, were clustered separately. </p>                        <a id="6221339" scrollto-destination="6221339"></a>
<div id="DSS001F6" data-id="DSS001F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00106.jpeg?Expires=2147483647&Signature=smlIfGhvP-4VhKmygIOy5drhzLEr~GH-~9n8GRzfcK98P0xs~rconu5SHyPsaVAst-rHrzdHdwBXjnan9iIKEQFoAiX2jND7M4oLGEt9kOvHgrEZomHBnxOX8ewvcwAXV0gv7tpyxF9-nTrkdkCIxSQdLk8aui0MvtmftSzjue4D7nThoAcz8stipwRFcvoRMVxhjyLTnCRzQ-yG1NSNq7TvHg24-yjpJqW3ewOsbT0seRza7n97vd6iplAh4hQ2xZcIQOLGVKgAzbNyyGTXwf5tU3wQrXLb2dOIOob33fHK~FzSExG9gh5GZDkErAVTlAXBlOj737TPFBzAJpKG1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlated clustering of wheat cDNA libraries based on gene expression (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya)." path-from-xml="dss00106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221339/dss00106.jpeg" path-from-xml="dss00106.jpeg" target="_blank">View large</a><a section="6221339" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00106.jpeg?Expires=2147483647&Signature=ViG0bEnQYenYWvpTGQCJyVOn3zzl9U~rNMsaEwI0IRNLiI50rC1HyzxTBB-kpy9Mhrph6BWW6eAHbC93Gzjrz~JOfdmz8WzOxT5u5CDhTGhVvjDcNVSLMhNH7j2RGgGKLBgjA2rmHh3EAsFrggXMchQ46mJ3cNokLT5PC-qUqGm5Rn-cVcoV27hLSZFgBFxWXXQo1jyZkqVEysBGvTidMr4Zd1FExpe8qsCMP19daHeCfiDjIQuYzu0Y-bRe-XspAINwWZYro05cVSM~UB-PL6HWJZRy5xQ~ZkIKPc1j5L3fxw~EuYTpSGR5ymVDWpIo-FXblqgw91gKXUKs9kH58w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221339&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlated clustering of wheat cDNA libraries based on gene expression (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya).</p></div></div><div content-id="DSS001F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00106.jpeg?Expires=2147483647&Signature=smlIfGhvP-4VhKmygIOy5drhzLEr~GH-~9n8GRzfcK98P0xs~rconu5SHyPsaVAst-rHrzdHdwBXjnan9iIKEQFoAiX2jND7M4oLGEt9kOvHgrEZomHBnxOX8ewvcwAXV0gv7tpyxF9-nTrkdkCIxSQdLk8aui0MvtmftSzjue4D7nThoAcz8stipwRFcvoRMVxhjyLTnCRzQ-yG1NSNq7TvHg24-yjpJqW3ewOsbT0seRza7n97vd6iplAh4hQ2xZcIQOLGVKgAzbNyyGTXwf5tU3wQrXLb2dOIOob33fHK~FzSExG9gh5GZDkErAVTlAXBlOj737TPFBzAJpKG1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlated clustering of wheat cDNA libraries based on gene expression (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya)." path-from-xml="dss00106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221339/dss00106.jpeg" path-from-xml="dss00106.jpeg" target="_blank">View large</a><a section="6221339" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00106.jpeg?Expires=2147483647&Signature=ViG0bEnQYenYWvpTGQCJyVOn3zzl9U~rNMsaEwI0IRNLiI50rC1HyzxTBB-kpy9Mhrph6BWW6eAHbC93Gzjrz~JOfdmz8WzOxT5u5CDhTGhVvjDcNVSLMhNH7j2RGgGKLBgjA2rmHh3EAsFrggXMchQ46mJ3cNokLT5PC-qUqGm5Rn-cVcoV27hLSZFgBFxWXXQo1jyZkqVEysBGvTidMr4Zd1FExpe8qsCMP19daHeCfiDjIQuYzu0Y-bRe-XspAINwWZYro05cVSM~UB-PL6HWJZRy5xQ~ZkIKPc1j5L3fxw~EuYTpSGR5ymVDWpIo-FXblqgw91gKXUKs9kH58w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221339&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlated clustering of wheat cDNA libraries based on gene expression (AT, Atlas; SC, Scout; CC, Chancellor; TC, Thatcher; CR, Cranbrook; HB, Halberd; CS, Chinese Spring; KT, Kitakei1354; NR, Norin4; VV, Valuevskaya).</p></div></div><p>To further determine the tissue-specific gene expression of select genes, digital gene expression was carried out for DREB and NAC TFs, OMT gene, and miRNA 172 targeting site genes in wheat transcripts (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The DREB genes are expressed mainly in dehydration-related tissue libraries and the spatial expression was mostly at root tissues. In the case of NAC TF genes, as expected, the expression was only noted in salt-treated libraries and the expression level was greater in root tissue followed by shoot, spikelet, and seed. Interestingly, some biotic stress-related libraries also expressed NAC TF. Based on the preliminary result obtained from our previous study on OMT against self-defence in wheat,<sup><a reveal-id="DSS001C29" data-open="DSS001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> the OMT-related contigs were mined and its gene expression was analysed. We found ubiquitous expression of OMT genes irrespective of stress, suggesting its defence role against stress. While mining of miRNA in wheat transcriptome, we found an abundance of miRNA 172 target sites. The digital gene expression analysis showed its abundance in all tissues and under all treatments in wheat.</p><p>In addition to digital expression, a new method of gene discovery and/or gene expression based on CA was carried out to determine specific and common gene expression between libraries or treatments. To identify the common molecular plant–athogen interactions, four libraries constructed for leaf rust and powdery mildew diseases were selected and examined by CA. Contigs with more than or equal to four ESTs were selected, to identify specific genes for leaf rust and powdery mildew diseases, in addition to common disease resistance- and susceptibility-related genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The number of genes expressed for powdery mildew outplayed the leaf rust disease. However, the number of disease susceptibility-related genes was more in leaf rust, suggesting different disease reaction mechanisms among diseases in wheat. Overall around 100 new genes were identified from these four disease-related libraries, which could have immense value for future research of molecular plant–pathogen interactions.</p>                            <h2 scrollto-destination=6221343 id="6221343" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Global wheat transcriptome analysis was carried out by accumulating ∼1 million ESTs from 51 cDNA libraries. In comparison with other studies of wheat which were biased towards one stress or growth stage of a few cultivars,<sup><a reveal-id="DSS001C3" data-open="DSS001C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS001C26" data-open="DSS001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> here we used all growth stages, and biotic and abiotic stresses for 10 different cultivars. The work flow of EST assembly and further analyses were summarized in Fig. <a reveal-id="DSS001F7" data-open="DSS001F7" class="link link-reveal link-table xref-fig">7</a>. Many perl-script programs were written specifically for processing the ESTs, resulting in a 24% reduction in total ESTs. The stringent parameter in EST assembly resulted in more singlets compared with contigs which helped for further analysis, i.e. SNP mining.<sup><a reveal-id="DSS001C23" data-open="DSS001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The average length of the contigs (879 bp) is higher than other studies,<sup><a reveal-id="DSS001C3" data-open="DSS001C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS001C23" data-open="DSS001C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSS001C44" data-open="DSS001C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and ∼80% of the contigs containing 2–10 EST members had homoeologous or paralogous genes. To determine the EST member contribution to the contig, we further classified the data into stress- and growth stage-related parameters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Among the 20 stress-related libraries, biotic stress-related libraries contributed more ESTs to the contigs than libraries for abiotic stress (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>A). Among the growth stages, libraries of the spikelet at late flowering contributed more than those of other stages, suggesting differential gene expression among the cultivars with or without any stress (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>B). The high number of unigenes and GC% also suggests the possibility of more genes present in wheat than in other crops.<sup><a reveal-id="DSS001C45" data-open="DSS001C45" class="link link-ref link-reveal xref-bibr">45</a></sup> This is supported by the recent study of the megabase level sequencing in 3B, which estimated 50 000 genes per diploid genome as a result of the additional non-collinear genes interspersed within the highly conserved ancestral grass gene backbone, suggesting accelerated evolution in the Triticeae lineages.<sup><a reveal-id="DSS001C44" data-open="DSS001C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The presence of additional genes was further confirmed by identification of new genes based on functional annotation (Fig. <a reveal-id="DSS001F3" data-open="DSS001F3" class="link link-reveal link-table xref-fig">3</a>). When putative wheat gene sequences were analysed for ORF length based on their hit status, we observed significantly shorter ORFs in sequences with no hits. These results suggest that ORF length, not sequence length, is a better indicator of finding transcripts with protein coding capacity and subsequently getting a hit in a sequence database. On the other hand, more than one-third of the sequences without a hit still contained an ORF &gt;300 bp, suggesting that sequences without a hit but with a relatively long ORF may represent new genes with protein coding capacity. We confirmed this by finding more full-length cDNA sequences. Our results also showed a higher no hit percentage in singlet sequences, most likely due to the fact that singletons represent rare genes in the wheat genome that are not well described in other organisms. </p>                        <a id="6221345" scrollto-destination="6221345"></a>
<div id="DSS001F7" data-id="DSS001F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00107.jpeg?Expires=2147483647&Signature=YxZn8x4F1G0SXRakGC0SXmPwpp1noGiF~uSNG48qgO4A5Rsr2IZIdNxD7jf0AvhMyuYqustBF33PsC11E1gkzGtjeYyZo5VvliFTCVKUHYqkhen0CDVy2vpZEZ2qi~ANJo0vOciU20G-JuWEi1qgRdkh5NYSX2UIlJcsft9fLFiOKqvEbqSmj74xVmOW8h6QZ3P4enmbHs6tAxrMOGCywwnny8smv9cOPzV8llpIS2kQPMCmZR79nZDt4qJOWrfuUp3CUxvgdIpv7Mw8SsHjZPLLu3jiKx-F1PeXwKw~ADQYSZDJAomUR00KC-rt1Q8y8Fy5ajLgQkbdAkDag0d20w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic diagram explaining the comprehensive EST analysis. The software used in the respective step was mentioned in parallel." path-from-xml="dss00107.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221345/dss00107.jpeg" path-from-xml="dss00107.jpeg" target="_blank">View large</a><a section="6221345" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00107.jpeg?Expires=2147483647&Signature=r16-BhB~KV1uS~Kji8ibQ8AD56wxxTamMfZ6fqm-1FNnftyhKXb8~kSyKRZvVA81Nv7Z6lHmbPQ9Cz3m47XnkVynWHT3j6vzQ7ZdCKKpFGya9Fz82sbVtVb9GJX3us0AOByIiTRwil1RbEUSSWfT~mWLoexDXKk3125G7Y6i2pp7v3u-wn27cKOi8zP-OMY3~goVLMJK8WhBYHi5P6ms5ukuvzqHAlM2H8ftIYR~8o08lW6~85lXT97N7f~R2FGiKwBU13hp0kR-zswnyzwnQF-vcU5mxNL3olVqe9~e8UfUGaoMuFCshh1LpFc-gVhT7nD5KjtKdLhoL6ScfUqBdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221345&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00107.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram explaining the comprehensive EST analysis. The software used in the respective step was mentioned in parallel.</p></div></div><div content-id="DSS001F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/m_dss00107.jpeg?Expires=2147483647&Signature=YxZn8x4F1G0SXRakGC0SXmPwpp1noGiF~uSNG48qgO4A5Rsr2IZIdNxD7jf0AvhMyuYqustBF33PsC11E1gkzGtjeYyZo5VvliFTCVKUHYqkhen0CDVy2vpZEZ2qi~ANJo0vOciU20G-JuWEi1qgRdkh5NYSX2UIlJcsft9fLFiOKqvEbqSmj74xVmOW8h6QZ3P4enmbHs6tAxrMOGCywwnny8smv9cOPzV8llpIS2kQPMCmZR79nZDt4qJOWrfuUp3CUxvgdIpv7Mw8SsHjZPLLu3jiKx-F1PeXwKw~ADQYSZDJAomUR00KC-rt1Q8y8Fy5ajLgQkbdAkDag0d20w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic diagram explaining the comprehensive EST analysis. The software used in the respective step was mentioned in parallel." path-from-xml="dss00107.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6221345/dss00107.jpeg" path-from-xml="dss00107.jpeg" target="_blank">View large</a><a section="6221345" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss00107.jpeg?Expires=2147483647&Signature=r16-BhB~KV1uS~Kji8ibQ8AD56wxxTamMfZ6fqm-1FNnftyhKXb8~kSyKRZvVA81Nv7Z6lHmbPQ9Cz3m47XnkVynWHT3j6vzQ7ZdCKKpFGya9Fz82sbVtVb9GJX3us0AOByIiTRwil1RbEUSSWfT~mWLoexDXKk3125G7Y6i2pp7v3u-wn27cKOi8zP-OMY3~goVLMJK8WhBYHi5P6ms5ukuvzqHAlM2H8ftIYR~8o08lW6~85lXT97N7f~R2FGiKwBU13hp0kR-zswnyzwnQF-vcU5mxNL3olVqe9~e8UfUGaoMuFCshh1LpFc-gVhT7nD5KjtKdLhoL6ScfUqBdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6221345&ar=419961&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00107.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram explaining the comprehensive EST analysis. The software used in the respective step was mentioned in parallel.</p></div></div>                            <h3 scrollto-destination=6221347 id="6221347" class="section-title" data-legacy-id=s4a>4.1. Functional characterization</h3>
<p>GO analysis revealed expected categories such as molecular, biological, and cellular processes (Fig. <a reveal-id="DSS001F5" data-open="DSS001F5" class="link link-reveal link-table xref-fig">5</a>). In wheat, the major molecular processes were binding and catalytic activities, similarly found in other Poaceae species.<sup><a reveal-id="DSS001C44" data-open="DSS001C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSS001C46" data-open="DSS001C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS001C47" data-open="DSS001C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Metabolic, transport, and translation functions accounted for 50% of the biological processes. Among cellular processes, as expected, membrane, intracellular, and ribosome functions played a major role. Using data from the plant TF database,<sup><a reveal-id="DSS001C34" data-open="DSS001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> we found 1183 contigs from wheat that had a high similarity with 2197 different coding sequences of TF from seven species; the high similarity percentage was found with rice. The number of TF found in wheat contigs was higher than reported in <em>Salvia sclorea</em> calyx which was sequenced using 454 pyrosequncing.<sup><a reveal-id="DSS001C48" data-open="DSS001C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The most represented TF family in wheat was <em>CCAAT</em>. The <em>CCAAT</em> box is a common <em>cis</em>-acting element found in the promoter and enhancer regions of a large number of genes in higher eukaryotes (for review).<sup><a reveal-id="DSS001C49" data-open="DSS001C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSS001C50" data-open="DSS001C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In addition to this TF family, several other TF families known to be involved in plant development were also present in our data.</p><p>The role of miRNAs in developmental and stress regulation is not well established in wheat, and increasingly tissue-specific and developmental regulation of miRNAs is being found mostly in animal species.<sup><a reveal-id="DSS001C51" data-open="DSS001C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Through cDNA sequencing efforts, we have identified transcripts that encode 945 different miRNAs,<sup><a reveal-id="DSS001C52" data-open="DSS001C52" class="link link-ref link-reveal xref-bibr">52</a></sup> although additional wheat-specific miRNAs may still remain in the cDNA collection. Indeed, a large number of ncRNAs have the potential to form miRNA-like stem loop precursors (data not shown), but experimental validation of these potential miRNA is required. In our study, we found an abundance of miRNA 172 target sites in as many as 236 contigs, and their uniform expression irrespective of growth stage and/or stress was confirmed by <em>in silico</em> gene expression analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>D).</p><p>The digital gene expression pattern of genes related to biotic and abiotic stresses (OMT, DREB, and NAC TFs), and epigenetic gene silencing has helped us to determine their quantitative and qualitative gene expression patterns.<sup><a reveal-id="DSS001C7" data-open="DSS001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> This approach permits both the association of tissues via their common patterns of gene expression and the association of genes via their tissue-dependent expression patterns. The correlation clustering of 51 libraries formed the groups of the libraries based on respective treatments or stages which confirmed the importance of libraries as well gene expression specific to treatment.<sup><a reveal-id="DSS001C26" data-open="DSS001C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p><p>CA is an explorative computational method for the study of associations between variables. Much like principle component analysis, it displays a low dimensional projection of the data, e.g. into the plane with three-dimensional view (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>), which can be achieved for two to three variables simultaneously, thus revealing associations between them. Traditionally, CA has been used prevalently in categorical data in the social sciences, but its application has been extended also to physical quantities and to proteomics.<sup><a reveal-id="DSS001C53" data-open="DSS001C53" class="link link-ref link-reveal xref-bibr">53</a></sup> This method allows us to quickly analyse the set of EST libraries and to discover molecular pathogenicity on wheat. In four disease-related libraries, ribulose 1–5 biphosphate and <em>S</em>-adenosyl methionine genes were commonly found. In leaf rust, UDP-glucosyl transferase and chlorophyll <em>a</em>–<em>b</em> binding protein were highly expressed,<sup><a reveal-id="DSS001C29" data-open="DSS001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> while in powdery mildew treatment, ADP-ribosylation factor, lipid transfer protein, and oxalate oxidase were specifically expressed. The advantage of CA analysis helped to find the common molecular resistance mechanism in wheat.<sup><a reveal-id="DSS001C54" data-open="DSS001C54" class="link link-ref link-reveal xref-bibr">54</a></sup> While comparing susceptible and resistance reaction mechanisms, some of the genes, such as 40S, 60S ribosomal protein and Zinc finger domain-containing protein genes, have copy number variations between the two mechanisms. We have shown that the application of CA to EST data provides an informative and concise means of visualizing these data, being capable of uncovering relationships both among either gene and between genes, in particular or common stages.</p>                            <h3 scrollto-destination=6221353 id="6221353" class="section-title" data-legacy-id=s4b>4.2. SNP mining</h3>
<p>Assembly of EST sequences into contigs in a polyploidy species like hexaploid wheat results in each contig being composed of ESTs from homoeologous loci and members of gene families.<sup><a reveal-id="DSS001C23" data-open="DSS001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> SNPs are the most abundantly found co-dominant polymorphic sites in greater proportion both in intronic and in exonic regions of the genome. They occur with variable frequencies and have become very popular in plant genetics and breeding due to their amenability for high throughput genotyping. In continuation of our earlier study to discriminate homoeologous gene expression of hexaploid wheat by SNP analysis of contigs grouped from 10 cDNA libraries from Chinese Spring,<sup><a reveal-id="DSS001C43" data-open="DSS001C43" class="link link-ref link-reveal xref-bibr">43</a></sup> we have mined our large-scale data to find SNPs from 10 different cultivars with various stress treatments. With relaxed criteria, we estimated ∼50 000 SNPs in wheat with an SNP frequency of one SNP per 96 bp—a number that is higher than our previous report.<sup><a reveal-id="DSS001C43" data-open="DSS001C43" class="link link-ref link-reveal xref-bibr">43</a></sup> This high number might be due to the EST originating from 10 different cultivars with various stress conditions, although the possibilities of over-estimation from the end sequences could not be excluded. Hence, a new criterion was applied by calculating the significant sequence length to avoid the end sequence and sequencing error-based SNPs. This approach accounted for only 20% of the SNPs obtained by the initial approach, and could lead to an underestimation of nucleotide diversity, although it guarantees the elimination of false positives. The stringent parameter resulted in the SNP frequency of one SNP in every 483 bp. In comparison of SNPs among huge genome size species, coffee has one SNP every 222 bp;<sup><a reveal-id="DSS001C55" data-open="DSS001C55" class="link link-ref link-reveal xref-bibr">55</a></sup> sugarcane has one SNP every 290 bp;<sup><a reveal-id="DSS001C16" data-open="DSS001C16" class="link link-ref link-reveal xref-bibr">16</a></sup> cotton has one SNP every 500 bp;<sup><a reveal-id="DSS001C17" data-open="DSS001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> oak has a frequency similar to cotton with one SNP every 471 bp.<sup><a reveal-id="DSS001C18" data-open="DSS001C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Our result in wheat compared with other species could explain the low polymorphism found in polyploidy species. The comparison of SNPs found between and within cultivars showed higher SNP frequency between cultivars (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), confirming that the low level of polymorphism identified between homoeologous genomes compared with inter-genome differences could be useful to select parents for linkage mapping studies.</p>                            <h2 scrollto-destination=6221356 id="6221356" class="section-title" data-legacy-id=s5>5. Conclusion</h2>
<p>The global wheat EST assembly presented here provides an unprecedented look at the wheat transcriptome and contributes tools for wheat genetics and genomics effort. The development and inclusion of cDNA libraries from all growth stages, various tissues, and treatments portray the complete picture of wheat transcriptome. Functional annotation and characterization give new ideas about wheat expressed genome, at least in part. The identified SNPs are invaluable resources for functional genomics and molecular breeding application. This set of processed EST sequences provides a seed for future investigation of wheat functional genomics using both long and short oligonucleotide arrays. Our data will thus act as a backbone for wheat genome sequence assembly, which is progressing rapidly.</p>                            <h2 scrollto-destination=6221358 id="6221358" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6221360 id="6221360" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by Grants-in-Aid for Scientific Research on priority areas ‘Comparative Genomics’ and the National Bio-resource Project from the Ministry of Education, Culture, Sports, Science and Technology of Japan. This is contribution No. 1007 from the Kihara Institute for Biological Research, Yokohama City University.</p>                            <h2 scrollto-destination=6221363 id="6221363" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Prof. K. Murai, Fukui Prefectural University, Prof. N. Kawakami, Meiji University, Dr S. Nasuda, Kyoto University, Drs S. Takumi and Y. Tosa, Kobe University, Dr T. Sutton and Prof. P. Langridge, University of Adelaide, and Dr. T. Sasaki, Okayama University, for supplying extracted RNA samples.</p>                            <h2 scrollto-destination=6221365 id="6221365" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS001C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Braun</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Atlin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Payne</div> <div class="given-names">T.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Reynolds</div> <div class="given-names">C.R.P</div></div>. </div><div class="article-title">Multi location testing as a tool to identify plant response to global climate change</div>, <div class="source">Climate Change and Crop Production</div> , <div class="year">2010</div><div class="publisher-loc">London, UK</div><div class="publisher-name">CABI</div></div></div></div></div><div content-id="DSS001C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosegrant</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Agcaoili</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Global Food Demand, Supply, and Price Prospects to 2010</div> , <div class="year">2010</div><div class="publisher-loc">Washington, DC</div><div class="publisher-name">International Food Policy Research Institute</div></div></div></div></div><div content-id="DSS001C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wilson</div> <div class="given-names">I.D.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">G.L.A.</div></div>,  <div class="name"><div class="surname">Beswick</div> <div class="given-names">R.W.</div></div>, et al. </div><div class="article-title">A transcriptomics resource for wheat functional genomics</div>, <div class="source">Plant Biotech.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">495</div>-<div class="lpage">506</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20transcriptomics%20resource%20for%20wheat%20functional%20genomics&amp;author=I.D.%20Wilson&amp;author=G.L.A.%20Barker&amp;author=R.W.%20Beswick&amp;publication_year=2004&amp;journal=Plant%20Biotech.&amp;volume=2&amp;pages=495-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2004.2.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2004.2.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2004.2.issue-6"> </span></div> </div></div></div></div></div><div content-id="DSS001C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duggan</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Bittner</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Meltzer</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Trent</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Expression profiling using cDNA microarrays</div>, <div class="source">Nat. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">21</div> <div class="issue">Suppl. 21</div>(pg. <div class="fpage">10</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20profiling%20using%20cDNA%20microarrays&amp;author=D.J.%20Duggan&amp;author=M.%20Bittner&amp;author=Y.%20Chen&amp;author=P.%20Meltzer&amp;author=J.M.%20Trent&amp;publication_year=1999&amp;journal=Nat.%20Genet.&amp;volume=21&amp;pages=10-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/4434" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F4434" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F4434"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9915494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Granger</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Mining EST databases to resolve evolutionary events in major crop species</div>, <div class="source">Genome</div> , <div class="year">2004</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">868</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20EST%20databases%20to%20resolve%20evolutionary%20events%20in%20major%20crop%20species&amp;author=J.A.%20Schlueter&amp;author=P.%20Dixon&amp;author=C.%20Granger&amp;publication_year=2004&amp;journal=Genome&amp;volume=47&amp;pages=868-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g04-047" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg04-047" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg04-047"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15499401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fulton</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Hoeven</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Eannetta</div> <div class="given-names">N.T.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">Identification, analysis, and utilization of conserved ortholog set markers for comparative genomics in higher plants</div>, <div class="source">Plant Cell</div> , <div class="year">2002</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1457</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%2C%20analysis%2C%20and%20utilization%20of%20conserved%20ortholog%20set%20markers%20for%20comparative%20genomics%20in%20higher%20plants&amp;author=T.M.%20Fulton&amp;author=R.%20Hoeven&amp;author=N.T.%20Eannetta&amp;author=S.D.%20Tanksley&amp;publication_year=2002&amp;journal=Plant%20Cell&amp;volume=14&amp;pages=1457-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.010479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.010479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.010479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12119367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Kahla</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Poirot</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Lopez</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Large-scale statistical analyses of rice ESTs reveal correlated patterns of gene expression</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">950</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20statistical%20analyses%20of%20rice%20ESTs%20reveal%20correlated%20patterns%20of%20gene%20expression&amp;author=R.M.%20Ewing&amp;author=A.B.%20Kahla&amp;author=O.%20Poirot&amp;author=F.%20Lopez&amp;author=S.%20Audic&amp;author=J.M.%20Claverie&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=950-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.10.950" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.10.950" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.10.950"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10523523" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ronning</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Stegalkina</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Ascenzi</div> <div class="given-names">R.A.</div></div>, et al. </div><div class="article-title">Comparative analyses of potato expressed sequence tag libraries</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">131</div> (pg. <div class="fpage">419</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analyses%20of%20potato%20expressed%20sequence%20tag%20libraries&amp;author=C.M.%20Ronning&amp;author=S.S.%20Stegalkina&amp;author=R.A.%20Ascenzi&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=131&amp;pages=419-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.013581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.013581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.013581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12586867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hughes</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Friedman</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Expression patterns of duplicate genes in the developing root in <em>Arabidopsis thaliana</em></div>, <div class="source">Mol. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">247</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20patterns%20of%20duplicate%20genes%20in%20the%20developing%20root%20in%20Arabidopsis%20thaliana&amp;author=A.%20Hughes&amp;author=R.%20Friedman&amp;publication_year=2004&amp;journal=Mol.%20Evol.&amp;volume=60&amp;pages=247-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-004-0171-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-004-0171-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-004-0171-z"> </span></div> </div></div></div></div></div><div content-id="DSS001C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Michalek</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Weschke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Pleissner</div> <div class="given-names">K.P.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EST analysis in barley defines a unigene set comprising 4,000 genes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">97</div>-<div class="lpage">103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST%20analysis%20in%20barley%20defines%20a%20unigene%20set%20comprising%204%2C000%20genes&amp;author=W.%20Michalek&amp;author=W.%20Weschke&amp;author=K.P.%20Pleissner&amp;author=A.%20Graner&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=97-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220200011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220200011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220200011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12579433" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wisman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ohlrogge</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Arabidopsis microarray service facilities</div>, <div class="source">Plant Physiol.</div> , <div class="year">2000</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1468</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20microarray%20service%20facilities&amp;author=E.%20Wisman&amp;author=J.%20Ohlrogge&amp;publication_year=2000&amp;journal=Plant%20Physiol.&amp;volume=124&amp;pages=1468-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.124.4.1468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.124.4.1468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.124.4.1468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11115861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawasaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Borchert</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Deyholos</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Gene expression profiles during the initial phase of salt stress in rice</div>, <div class="source">Plant Cell</div> , <div class="year">2001</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">889</div>-<div class="lpage">905</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20profiles%20during%20the%20initial%20phase%20of%20salt%20stress%20in%20rice&amp;author=S.%20Kawasaki&amp;author=C.%20Borchert&amp;author=M.%20Deyholos&amp;publication_year=2001&amp;journal=Plant%20Cell&amp;volume=13&amp;pages=889-905" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.13.4.889" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.13.4.889" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.13.4.889"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11283343" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alba</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fei</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Payton</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">ESTs, cDNA microarrays, and gene expression profiling: tools for dissecting plant physiology and development</div>, <div class="source">Plant J.</div> , <div class="year">2004</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">697</div>-<div class="lpage">714</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ESTs%2C%20cDNA%20microarrays%2C%20and%20gene%20expression%20profiling%3A%20tools%20for%20dissecting%20plant%20physiology%20and%20development&amp;author=R.%20Alba&amp;author=Z.%20Fei&amp;author=P.%20Payton&amp;publication_year=2004&amp;journal=Plant%20J.&amp;volume=39&amp;pages=697-714" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.39.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.39.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.39.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15315633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arpat</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Waugh</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sullivan</div> <div class="given-names">J.P.</div></div>, et al. </div><div class="article-title">Functional genomics of cell elongation in developing cotton fibers</div>, <div class="source">Plant. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">911</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20genomics%20of%20cell%20elongation%20in%20developing%20cotton%20fibers&amp;author=A.%20Arpat&amp;author=M.%20Waugh&amp;author=J.P.%20Sullivan&amp;publication_year=2004&amp;journal=Plant.%20Mol.%20Biol.&amp;volume=54&amp;pages=911-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-004-0392-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-004-0392-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-004-0392-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15604659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Wanamaker</div> <div class="given-names">S.I.</div></div>,  <div class="name"><div class="surname">Caldo</div> <div class="given-names">R.A.</div></div>, et al. </div><div class="article-title">A new resource for cereal genomics: 22K barley GeneChip comes of age</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">960</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20resource%20for%20cereal%20genomics%3A%2022K%20barley%20GeneChip%20comes%20of%20age&amp;author=T.J.%20Close&amp;author=S.I.%20Wanamaker&amp;author=R.A.%20Caldo&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=134&amp;pages=960-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.034462" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.034462" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.034462"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15020760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vettore</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Silva</div> <div class="given-names">F.R.</div></div>,  <div class="name"><div class="surname">Kemper</div> <div class="given-names">E.L.</div></div>, et al. </div><div class="article-title">Analysis and functional annotation of an expressed sequence tag collection for tropical crop sugarcane</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2725</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20and%20functional%20annotation%20of%20an%20expressed%20sequence%20tag%20collection%20for%20tropical%20crop%20sugarcane&amp;author=A.L.%20Vettore&amp;author=F.R.%20Silva&amp;author=E.L.%20Kemper&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2725-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1532103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1532103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1532103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14613979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Udall</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Swanson</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Haller</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A global assembly of cotton ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">441</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20global%20assembly%20of%20cotton%20ESTs&amp;author=J.A.%20Udall&amp;author=J.M.%20Swanson&amp;author=K.%20Haller&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=441-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4602906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4602906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4602906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16478941" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ueno</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Provost</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Léger</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Bioinformatic analysis of ESTs collected by Sanger and pyrosequencing methods for a keystone forest tree species: oak</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">650</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatic%20analysis%20of%20ESTs%20collected%20by%20Sanger%20and%20pyrosequencing%20methods%20for%20a%20keystone%20forest%20tree%20species%3A%20oak&amp;author=S.%20Ueno&amp;author=G.%20Provost&amp;author=V.%20L%C3%A9ger&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=650" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21092232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="collab">The Arabidopsis Genome Initiative</div><div class="article-title">Analysis of the genome sequence of the flowering plant, <em>Arabidopsis thaliana</em></div>, <div class="source">, Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">796</div>-<div class="lpage">815</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Remington</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Thornsberry</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Structure of linkage disequilibrium and phenotypic associations in the maize genome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">11479</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20linkage%20disequilibrium%20and%20phenotypic%20associations%20in%20the%20maize%20genome&amp;author=D.L.%20Remington&amp;author=J.M.%20Thornsberry&amp;author=Y.%20Matsuoka&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=11479-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.201394398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.201394398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.201394398"> </span></div> </div></div></div></div></div><div content-id="DSS001C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tenaillon</div> <div class="given-names">M.I.</div></div>,  <div class="name"><div class="surname">Swakins</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Doebley</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>. </div><div class="article-title">Patterns of DNA sequence polymorphism along chromosome 1 of maize (<em>Zea mays</em> L.)</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">9161</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20DNA%20sequence%20polymorphism%20along%20chromosome%201%20of%20maize%20(Zea%20mays%20L.)&amp;author=M.I.%20Tenaillon&amp;author=M.C.%20Swakins&amp;author=A.D.%20Long&amp;author=R.L.%20Gaut&amp;author=J.F.%20Doebley&amp;author=B.S.%20Gaut&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=9161-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.151244298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.151244298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.151244298"> </span></div> </div></div></div></div></div><div content-id="DSS001C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Applications of single nucleotide polymorphisms in crop genetics</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">94</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20single%20nucleotide%20polymorphisms%20in%20crop%20genetics&amp;author=A.%20Rafalski&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=5&amp;pages=94-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(02)00240-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(02)00240-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(02)00240-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11856602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Somers</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Kirkpatrick</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Moniwa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Walsh</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Mining single-nucleotide polymorphisms from hexaploid wheat ESTs</div>, <div class="source">Genome</div> , <div class="year">2003</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">431</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20single-nucleotide%20polymorphisms%20from%20hexaploid%20wheat%20ESTs&amp;author=D.J.%20Somers&amp;author=R.%20Kirkpatrick&amp;author=M.%20Moniwa&amp;author=A.%20Walsh&amp;publication_year=2003&amp;journal=Genome&amp;volume=49&amp;pages=431-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-027"> </span></div> </div></div></div></div></div><div content-id="DSS001C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nemoto</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Correlated clustering and virtual display of gene expression patterns in the wheat life cycle by large-scale statistical analyses of expressed sequence tags</div>, <div class="source">Plant J.</div> , <div class="year">2003</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1001</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlated%20clustering%20and%20virtual%20display%20of%20gene%20expression%20patterns%20in%20the%20wheat%20life%20cycle%20by%20large-scale%20statistical%20analyses%20of%20expressed%20sequence%20tags&amp;author=Y.%20Ogihara&amp;author=K.%20Mochida&amp;author=Y.%20Nemoto&amp;publication_year=2003&amp;journal=Plant%20J.&amp;volume=33&amp;pages=1001-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01687.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01687.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01687.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12631325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kawaura</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Construction of a full-length cDNA library from young spikelets of hexaploid wheat and its characterization by large-scale sequencing of expressed sequence tags</div>, <div class="source">Genes Genet. Syst.</div> , <div class="year">2004</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">227</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20full-length%20cDNA%20library%20from%20young%20spikelets%20of%20hexaploid%20wheat%20and%20its%20characterization%20by%20large-scale%20sequencing%20of%20expressed%20sequence%20tags&amp;author=Y.%20Ogihara&amp;author=K.%20Mochida&amp;author=K.%20Kawaura&amp;publication_year=2004&amp;journal=Genes%20Genet.%20Syst.&amp;volume=79&amp;pages=227-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.79.227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.79.227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.79.227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15514442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kawaura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shimosaka</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Tissue expression map of a large number of expressed sequence tags and its application to <em>in silico</em> screening of stress response genes in common wheat</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">304</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tissue%20expression%20map%20of%20a%20large%20number%20of%20expressed%20sequence%20tags%20and%20its%20application%20to%20in%20silico%20screening%20of%20stress%20response%20genes%20in%20common%20wheat&amp;author=K.%20Mochida&amp;author=K.%20Kawaura&amp;author=E.%20Shimosaka&amp;publication_year=2006&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=276&amp;pages=304-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-006-0120-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-006-0120-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-006-0120-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16832693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Enju</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Assessment of adaptive evolution between wheat and rice as deduced from the full-length cDNA sequence data and the expression patterns of common wheat</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">18</div> pg. <div class="fpage">271</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20adaptive%20evolution%20between%20wheat%20and%20rice%20as%20deduced%20from%20the%20full-length%20cDNA%20sequence%20data%20and%20the%20expression%20patterns%20of%20common%20wheat&amp;author=K.%20Kawaura&amp;author=K.%20Mochida&amp;author=A.%20Enju&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=271" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-271"> </span></div> </div></div></div></div></div><div content-id="DSS001C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamazaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Transcriptome analysis of salinity stress responses in common wheat using a 22k oligo-DNA microarray</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">132</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20salinity%20stress%20responses%20in%20common%20wheat%20using%20a%2022k%20oligo-DNA%20microarray&amp;author=K.%20Kawaura&amp;author=K.%20Mochida&amp;author=Y.%20Yamazaki&amp;author=Y.%20Ogihara&amp;publication_year=2006&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=6&amp;pages=132-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-005-0010-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-005-0010-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-005-0010-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16328439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manickavelu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kawaura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Oishi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Comparative gene expression analysis of susceptible and resistance near-isogenic lines in common wheat infected by <em>Puccinia triticina</em></div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">211</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20gene%20expression%20analysis%20of%20susceptible%20and%20resistance%20near-isogenic%20lines%20in%20common%20wheat%20infected%20by%20Puccinia%20triticina&amp;author=A.%20Manickavelu&amp;author=K.%20Kawaura&amp;author=K.%20Oishi&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=211-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20360266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wendl</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using PHRED. I. Accuracy assessment</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">175</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20PHRED.%20I.%20Accuracy%20assessment&amp;author=B.%20Ewing&amp;author=L.%20Hillier&amp;author=M.C.%20Wendl&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=175-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="collab">Gene Ontology Consortium</div><div class="article-title">The Gene Ontology project in 2008</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D440</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17984083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W116</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perez-Rodriguez</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Riano-Pachon</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Correa</div> <div class="given-names">L.G.G.</div></div>,  <div class="name"><div class="surname">Rensing</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Kersten</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Mueller-Roeber</div> <div class="given-names">B.</div></div>. </div><div class="article-title">PlnTFDB: updated content and new features of the plant transcription factor database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div><div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkp805" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkp805</a></div></div></div></div></div><div content-id="DSS001C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsumoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Comprehensive sequence analysis of 24,783 barley full-length cDNAs derived from 12 clone libraries</div>, <div class="source">Plant Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">20</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20sequence%20analysis%20of%2024%2C783%20barley%20full-length%20cDNAs%20derived%20from%2012%20clone%20libraries&amp;author=T.%20Matsumoto&amp;author=T.%20Tanaka&amp;author=H.%20Sakai&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=20-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.171579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.171579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.171579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21415278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Griffiths-Jones</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saini</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Van Dongen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Enright</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">MiRBase: tools for microRNA genomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D154</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MiRBase%3A%20tools%20for%20microRNA%20genomics&amp;author=S.%20Griffiths-Jones&amp;author=H.K.%20Saini&amp;author=S.%20Van%20Dongen&amp;author=A.J.%20Enright&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D154-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm952" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm952" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm952"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17991681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liao</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">Identification of conserved microRNAs and their targets from <em>Solanum lycopersicum</em> Mill</div>, <div class="source">Gene</div> , <div class="year">2008</div>, vol. <div class="volume">423</div> (pg. <div class="fpage">1</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20conserved%20microRNAs%20and%20their%20targets%20from%20Solanum%20lycopersicum%20Mill&amp;author=J.%20Zhang&amp;author=R.%20Zeng&amp;author=J.%20Chen&amp;author=X.%20Liu&amp;author=Q.%20Liao&amp;publication_year=2008&amp;journal=Gene&amp;volume=423&amp;pages=1-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.05.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.05.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.05.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18602455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">F.L.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">S.Q.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Computational identification of novel microRNAs and targets in <em>Brassica napus</em></div>, <div class="source">FEBS Lett.</div> , <div class="year">2007</div>, vol. <div class="volume">581</div> (pg. <div class="fpage">1464</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20identification%20of%20novel%20microRNAs%20and%20targets%20in%20Brassica%20napus&amp;author=F.L.%20Xie&amp;author=S.Q.%20Huang&amp;author=K.%20Guo&amp;publication_year=2007&amp;journal=FEBS%20Lett.&amp;volume=581&amp;pages=1464-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2007.02.074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2007.02.074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2007.02.074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17367786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Milne</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Bayer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cardle</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Tablet-next generation sequence assembly visualization</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">401</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tablet-next%20generation%20sequence%20assembly%20visualization&amp;author=I.%20Milne&amp;author=M.%20Bayer&amp;author=L.%20Cardle&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=401-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp666" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp666" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp666"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eisen</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Spellman</div> <div class="given-names">P.T.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P.O.</div></div>,  <div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Cluster analysis and display of genome-wide expression patterns</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">14863</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns&amp;author=M.B.%20Eisen&amp;author=P.T.%20Spellman&amp;author=P.O.%20Brown&amp;author=D.%20Botstein&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=14863-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.25.14863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.25.14863"> </span></div> </div></div></div></div></div><div content-id="DSS001C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hongo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">OryzaExpress: an integrated database of gene expression networks and omics annotations in rice</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">220</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OryzaExpress%3A%20an%20integrated%20database%20of%20gene%20expression%20networks%20and%20omics%20annotations%20in%20rice&amp;author=K.%20Hamada&amp;author=K.%20Hongo&amp;author=K.%20Suwabe&amp;publication_year=2011&amp;journal=Plant%20Cell%20Physiol.&amp;volume=52&amp;pages=220-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcq195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcq195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcq195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choulet</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Wicker</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Rustenholz</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Megabase level sequencing reveals contrasted organization and evolution patterns of the wheat gene and transposable element spaces</div>, <div class="source">Plant Cell</div> , <div class="year">2010</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1686</div>-<div class="lpage">701</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Megabase%20level%20sequencing%20reveals%20contrasted%20organization%20and%20evolution%20patterns%20of%20the%20wheat%20gene%20and%20transposable%20element%20spaces&amp;author=F.%20Choulet&amp;author=T.%20Wicker&amp;author=C.%20Rustenholz&amp;publication_year=2010&amp;journal=Plant%20Cell&amp;volume=22&amp;pages=1686-701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.110.074187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.074187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.110.074187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20581307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamazaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Discrimination of homoeologous gene expression in hexaploid wheat by SNP analysis of contigs grouped from a large number of expressed sequence tags</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2003</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">371</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discrimination%20of%20homoeologous%20gene%20expression%20in%20hexaploid%20wheat%20by%20SNP%20analysis%20of%20contigs%20grouped%20from%20a%20large%20number%20of%20expressed%20sequence%20tags&amp;author=K.%20Mochida&amp;author=Y.%20Yamazaki&amp;author=Y.%20Ogihara&amp;publication_year=2003&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=270&amp;pages=371-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-003-0939-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-003-0939-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-003-0939-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14595557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Wanamaker</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Construction and evaluation of cDNA libraries for large-scale expressed sequence tag sequencing in wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">168</div> (pg. <div class="fpage">595</div>-<div class="lpage">608</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20evaluation%20of%20cDNA%20libraries%20for%20large-scale%20expressed%20sequence%20tag%20sequencing%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=D.%20Zhang&amp;author=D.W.%20Choi&amp;author=S.%20Wanamaker&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=595-608" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.034785" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.034785" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.034785"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15514038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rabinowicz</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Citek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Budiman</div> <div class="given-names">M.A.</div></div>, et al. </div><div class="article-title">Differential methylation of genes and repeats in land plants</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1431</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20methylation%20of%20genes%20and%20repeats%20in%20land%20plants&amp;author=P.D.%20Rabinowicz&amp;author=R.%20Citek&amp;author=M.A.%20Budiman&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1431-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4100405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4100405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4100405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16204196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kikuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sathoh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">301</div> pg. <div class="fpage">376</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Collection%2C%20mapping%2C%20and%20annotation%20of%20over%2028%2C000%20cDNA%20clones%20from%20japonica%20rice&amp;author=S.%20Kikuchi&amp;author=K.%20Sathoh&amp;author=T.%20Nagata&amp;publication_year=2003&amp;journal=Science&amp;volume=301&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1081288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1081288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1081288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alexandrov</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">Brover</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Freidin</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Insights into corn genes derived from large-scale cDNASequencing</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">179</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20corn%20genes%20derived%20from%20large-scale%20cDNASequencing&amp;author=N.N.%20Alexandrov&amp;author=V.V.%20Brover&amp;author=S.%20Freidin&amp;publication_year=2009&amp;journal=Plant%20Mol.%20Biol.&amp;volume=69&amp;pages=179-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-008-9415-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-008-9415-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-008-9415-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18937034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Legrand</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Valot</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nicolé</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">One-step identification of conserved miRNAs, their argets, potential transcription factors and effector genes of complete secondary metabolism pathways after 454 pyrosequencing of calyx cDNAs from the Labiate (<em>Salvia sclarea</em> L.)</div>, <div class="source">Gene</div> , <div class="year">2010</div>, vol. <div class="volume">450</div> (pg. <div class="fpage">55</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One-step%20identification%20of%20conserved%20miRNAs%2C%20their%20argets%2C%20potential%20transcription%20factors%20and%20effector%20genes%20of%20complete%20secondary%20metabolism%20pathways%20after%20454%20pyrosequencing%20of%20calyx%20cDNAs%20from%20the%20Labiate%20(Salvia%20sclarea%20L.)&amp;author=S.%20Legrand&amp;author=N.%20Valot&amp;author=F.%20Nicol%C3%A9&amp;publication_year=2010&amp;journal=Gene&amp;volume=450&amp;pages=55-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2009.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2009.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2009.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19840835" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitchell</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Tjian</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Transcriptional regulation in mammalian cells by sequence-specific DNA binding proteins</div>, <div class="source">Science</div> , <div class="year">1989</div>, vol. <div class="volume">245</div> (pg. <div class="fpage">371</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulation%20in%20mammalian%20cells%20by%20sequence-specific%20DNA%20binding%20proteins&amp;author=P.J.%20Mitchell&amp;author=R.%20Tjian&amp;publication_year=1989&amp;journal=Science&amp;volume=245&amp;pages=371-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2667136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2667136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2667136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2667136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">McKnight</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Eukaryotic transcriptional regulatory proteins</div>, <div class="source">Annu. Rev. Biochem.</div> , <div class="year">1989</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">799</div>-<div class="lpage">839</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Eukaryotic%20transcriptional%20regulatory%20proteins&amp;author=P.F.%20Johnson&amp;author=S.L.%20McKnight&amp;publication_year=1989&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=58&amp;pages=799-839" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.bi.58.070189.004055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.bi.58.070189.004055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.bi.58.070189.004055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2673023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hubbard</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Grafham</div> <div class="given-names">D.V.</div></div>,  <div class="name"><div class="surname">Beattie</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Overton</div> <div class="given-names">I.M.</div></div>,  <div class="name"><div class="surname">McLaren</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Croning</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Transcriptome analysis for the chicken based on 19,626 finished cDNA sequences and 485,337 expressed sequence tags</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">174</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20for%20the%20chicken%20based%20on%2019%2C626%20finished%20cDNA%20sequences%20and%20485%2C337%20expressed%20sequence%20tags&amp;author=S.J.%20Hubbard&amp;author=D.V.%20Grafham&amp;author=K.J.%20Beattie&amp;author=I.M.%20Overton&amp;author=S.R.%20McLaren&amp;author=M.D.%20Croning&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=174-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3011405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3011405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3011405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15590942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dai</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhuang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">P.X.</div></div>. </div><div class="article-title">Computational analysis of miRNA targets in plants: current status and challenges</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2011</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">115</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20analysis%20of%20miRNA%20targets%20in%20plants%3A%20current%20status%20and%20challenges&amp;author=X.%20Dai&amp;author=Z.%20Zhuang&amp;author=P.X.%20Zhao&amp;publication_year=2011&amp;journal=Brief%20Bioinform.&amp;volume=21&amp;pages=115-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbq065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbq065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbq065"> </span></div> </div></div></div></div></div><div content-id="DSS001C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fellenberg</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hauser</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Brors</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Neutzner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hoheisel</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Vingron</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Correspondence analysis applied to microarray data</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">10781</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correspondence%20analysis%20applied%20to%20microarray%20data&amp;author=K.%20Fellenberg&amp;author=N.C.%20Hauser&amp;author=B.%20Brors&amp;author=A.%20Neutzner&amp;author=J.D.%20Hoheisel&amp;author=M.%20Vingron&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=10781-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.181597298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.181597298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.181597298"> </span></div> </div></div></div></div></div><div content-id="DSS001C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Imai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hanashita</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A new method for gene discovery in large-scale microarray data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">1532</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20method%20for%20gene%20discovery%20in%20large-scale%20microarray%20data&amp;author=K.%20Yano&amp;author=K.%20Imai&amp;author=A.%20Shimizu&amp;author=T.%20Hanashita&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=1532-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16537840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS001C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS001C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vidal</div> <div class="given-names">R.O.</div></div>,  <div class="name"><div class="surname">Mondego</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Pot</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">A high-throughput data mining of single nucleotide polymorphisms in coffea species expressed sequence tags suggests differential homeologous gene expression in the allotetraploid (<em>Coffea arabica</em>)</div>, <div class="source">Plant Physiol.</div> , <div class="year">2010</div>, vol. <div class="volume">154</div> (pg. <div class="fpage">1053</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-throughput%20data%20mining%20of%20single%20nucleotide%20polymorphisms%20in%20coffea%20species%20expressed%20sequence%20tags%20suggests%20differential%20homeologous%20gene%20expression%20in%20the%20allotetraploid%20(Coffea%20arabica)&amp;author=R.O.%20Vidal&amp;author=J.M.%20Mondego&amp;author=D.%20Pot&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=154&amp;pages=1053-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.162438" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.162438" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.162438"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20864545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=419961&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss001/2/dss001-dss001supp.pdf?Expires=2147483647&amp;Signature=uBxOXQ7-qXcyJw90tc9CJ-CmUMWNyJAYU8zLn5mk26mlCygegxLvj8Sv0nD59mcrDOTjmWck6hUsdFvMXkEnb7~lRRIU~vVy-qTy-9FQ-0S~DzmfQD8d6ddne~1lPpgzObmxbOPB017-SizI2EWEEic~JhlA7D-BRF~MU7Q1cpxoBC4lT0tUfSkbj~8fWTygO5eXqs8VLrwzEWF3s2ZWPW98Ow3~dkJUHi1n8FFeDESVih~l2Lbu7vl2tJF8eu9iziYhZMUnKtzkkgicUF3qttOU08hmIYIfYA~qrYKTS7y7D~0U7yDumumAIxZyNiJmT9iVOHW8z-PphYKFPJKrHg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">317</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">17</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="419961"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f2%2f165%2f419961">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000303290100006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (17)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F2%2F165%2F419961" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/419961'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comprehensive Functional Analyses of Expressed Sequence Tags in Common Wheat (Triticum aestivum)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f165%2f419961">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

