

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/5/281/350803","@type":"ScholarlyArticle","name":"Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi015","keywords":["self-organizing map","environmental samples","metagenome","phylogenetic classification","SOM"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/5/281/350803","author":[{"name":"Abe, Takashi","affiliation":"1  Center for Information Biology, National Institute of Genetics, The Graduate University for Advanced Studies (Sokendai)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Sugawara, Hideaki","affiliation":"1  Center for Information Biology, National Institute of Genetics, The Graduate University for Advanced Studies (Sokendai)  Mishima, Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Kinouchi, Makoto","affiliation":"2  Department of Bio-System Engineering, Faculty of Engineering, Yamagata University  Yonezawa, Yamagata 992-8510, Japan","@type":"Person"},{"name":"Kanaya, Shigehiko","affiliation":"3  Department of Bioinformatics and Genomes, Graduate School of Information Science, Nara Institute of Science and Technology  Takayama, Ikoma, Nara 630-0101, Japan","@type":"Person"},{"name":"Ikemura, Toshimichi","affiliation":"4  The Graduate University for Advanced Studies (Sokendai), Hayama Center for Advanced Research  Hayama-cho, Kanagawa 240-0193, Japan","@type":"Person"}],"description":"Abstract.  A self-organizing map (SOM) was developed as a novel bioinformatics strategy for phylogenetic classification of sequence fragments obtained from pool","pageStart":"281","pageEnd":"290","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/5/0/m_dnaresearch12_5.cover.gif?Expires=2147483647&Signature=aCda7p9Uq5iwZjUzLjBT9gM9GcXHBhpbSpHktSZ3aBfaLgUTRrUBMfrhnbDOftVRcOxJYPlWQoOSDtYCMRCSqLqznp4erEvRKXpqo14qAWzUQzXAUf0R2hzkxTrttMY8s2309oR8tNHgB9rVuF2hpqh3QH4eyjVeWXlnJ40m9gcWoDr-jQH7WNKVcWkYKOzDqj0zz3HLWNzUSiszlQ7mPDPy4j~szH2kDsY0ptS4bBqTMPSDLuVeIksQM7hpXYsVJSgGsIGUFqHvKMFIXeS6xBjZCvV-jy2mqIn0wg5gOITNC5m-u-f0z~ur8QMlAsoeOXCWmD3HyGddqFtbkE9Www__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/5/0/m_dnaresearch12_5.cover.gif?Expires=2147483647&Signature=aCda7p9Uq5iwZjUzLjBT9gM9GcXHBhpbSpHktSZ3aBfaLgUTRrUBMfrhnbDOftVRcOxJYPlWQoOSDtYCMRCSqLqznp4erEvRKXpqo14qAWzUQzXAUf0R2hzkxTrttMY8s2309oR8tNHgB9rVuF2hpqh3QH4eyjVeWXlnJ40m9gcWoDr-jQH7WNKVcWkYKOzDqj0zz3HLWNzUSiszlQ7mPDPy4j~szH2kDsY0ptS4bBqTMPSDLuVeIksQM7hpXYsVJSgGsIGUFqHvKMFIXeS6xBjZCvV-jy2mqIn0wg5gOITNC5m-u-f0z~ur8QMlAsoeOXCWmD3HyGddqFtbkE9Www__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples" />
    <meta property="og:description" content="Abstract.  A self-organizing map (SOM) was developed as a novel bioinformatics strategy for phylogenetic classification of sequence fragments obtained from pool" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/5/281/350803" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Abe, Takashi" /><meta name="citation_author_institution" content="1Center for Information Biology, National Institute of Genetics, The Graduate University for Advanced Studies (Sokendai)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Sugawara, Hideaki" /><meta name="citation_author_institution" content="1Center for Information Biology, National Institute of Genetics, The Graduate University for Advanced Studies (Sokendai)Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Kinouchi, Makoto" /><meta name="citation_author_institution" content="2Department of Bio-System Engineering, Faculty of Engineering, Yamagata UniversityYonezawa, Yamagata 992-8510, Japan" /><meta name="citation_author" content="Kanaya, Shigehiko" /><meta name="citation_author_institution" content="3Department of Bioinformatics and Genomes, Graduate School of Information Science, Nara Institute of Science and TechnologyTakayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Ikemura, Toshimichi" /><meta name="citation_author_institution" content="4The Graduate University for Advanced Studies (Sokendai), Hayama Center for Advanced ResearchHayama-cho, Kanagawa 240-0193, Japan" /><meta name="citation_title" content="Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples" /><meta name="citation_firstpage" content="281" /><meta name="citation_lastpage" content="290" /><meta name="citation_doi" content="10.1093/dnares/dsi015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Phylogenetic identification and in situ detection of individual microbial cells without cultivation; Microbiol. Rev.;  citation_year=1995;  citation_volume=59;  citation_pages=143-169; " /><meta name="citation_reference" content="citation_title=Identifying microbial diversity in the natural environment: a molecular phylogenetic approach; Trends Biotechnol.;  citation_year=1996;  citation_volume=14;  citation_pages=190-197; " /><meta name="citation_reference" content="citation_title=Cloning the soil metagenome: a strategy for accessing the genetic and functional diversity of uncultured microorganisms; Appl. Environ. Microbiol.;  citation_year=2000;  citation_volume=66;  citation_pages=2541-2547; " /><meta name="citation_reference" content="citation_title=Screening for novel enzymes for biocatalytic processes: accessing the metagenome as a resource of novel functional sequence space; Curr. Opin. Biotechnol.;  citation_year=2002;  citation_volume=13;  citation_pages=572-577; " /><meta name="citation_reference" content="citation_title=Microbial population genomics and ecology; Curr. Opin. Microbiol.;  citation_year=2002;  citation_volume=5;  citation_pages=520-524; " /><meta name="citation_reference" content="citation_title=Biotechnological prospects from metagenomics; Curr. Opin. Biotechnol.;  citation_year=2003;  citation_volume=14;  citation_pages=303-310; " /><meta name="citation_reference" content="citation_title=The self-organizing map; Proc. IEEE;  citation_year=1990;  citation_volume=78;  citation_pages=1464-1480; " /><meta name="citation_reference" content="citation_title=Engineering applications of the self-organizing map; Proc. IEEE;  citation_year=1996;  citation_volume=84;  citation_pages=1358-1384; " /><meta name="citation_reference" content="citation_title=Informatics for unveiling hidden genome signatures; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=693-702; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the E. coli O157 genome; Gene;  citation_year=2001;  citation_volume=276;  citation_pages=89-99; " /><meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea; Science;  citation_year=2004;  citation_volume=304;  citation_pages=66-74; " /><meta name="citation_reference" content="citation_title=Comparative DNA analysis across diverse genomes; Annu. Rev. Genet.;  citation_year=1998;  citation_volume=32;  citation_pages=185-225; " /><meta name="citation_reference" content="citation_title=Doublet frequencies in evolutionary distinct groups; Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=1749-1763; " /><meta name="citation_reference" content="citation_title=Genome-scale compositional comparisons in eukaryotes; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=540-546; " /><meta name="citation_reference" content="citation_title=Application of tetranucleotide frequencies for the assignment of genomic fragments; Environ. Microbiol.;  citation_year=2004;  citation_volume=6;  citation_pages=938-947; " /><meta name="citation_reference" content="citation_title=Community structure and metabolism through reconstruction of microbial genomes from the environment; Nature;  citation_year=2004;  citation_volume=428;  citation_pages=37-43; " /><meta name="citation_reference" content="citation_title=Substrate-induced gene-expression screening of environmental metagenome libraries for isolation of catabolic genes; Nat. Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=88-93; " /><meta name="citation_reference" content="citation_title=Direct cloning of genes encoding novel xylanases from human gut; Can. J. Microbiol.;  citation_year=2005;  citation_volume=51;  citation_pages=251-259; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis; Nature;  citation_year=1997;  citation_volume=390;  citation_pages=249-256; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/5/281/17462214/dsi015.pdf" /><meta name="description" content="Abstract.  A self-organizing map (SOM) was developed as a novel bioinformatics strategy for phylogenetic classification of sequence fragments obtained from pool" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/5/281/350803" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/5/281/350803" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/5/281/350803");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi015';
    </script>
<title>
	Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./350803" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcwzPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmUsIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxQ2VudGVyIGZvciBJbmZvcm1hdGlvbiBCaW9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFRoZSBHcmFkdWF0ZSBVbml2ZXJzaXR5IGZvciBBZHZhbmNlZCBTdHVkaWVzIChTb2tlbmRhaSlNaXNoaW1hLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdWdhd2FyYSwgSGlkZWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgVGhlIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMgKFNva2VuZGFpKU1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktpbm91Y2hpLCBNYWtvdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIyRGVwYXJ0bWVudCBvZiBCaW8tU3lzdGVtIEVuZ2luZWVyaW5nLCBGYWN1bHR5IG9mIEVuZ2luZWVyaW5nLCBZYW1hZ2F0YSBVbml2ZXJzaXR5WW9uZXphd2EsIFlhbWFnYXRhIDk5Mi04NTEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthbmF5YSwgU2hpZ2VoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iM0RlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MgYW5kIEdlbm9tZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5VGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklrZW11cmEsIFRvc2hpbWljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSI0VGhlIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMgKFNva2VuZGFpKSwgSGF5YW1hIENlbnRlciBmb3IgQWR2YW5jZWQgUmVzZWFyY2hIYXlhbWEtY2hvLCBLYW5hZ2F3YSAyNDAtMDE5MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik5vdmVsIFBoeWxvZ2VuZXRpYyBTdHVkaWVzIG9mIEdlbm9taWMgU2VxdWVuY2UgRnJhZ21lbnRzIERlcml2ZWQgZnJvbSBVbmN1bHR1cmVkIE1pY3JvYmUgTWl4dHVyZXMgaW4gRW52aXJvbm1lbnRhbCBhbmQgQ2xpbmljYWwgU2FtcGxlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI4MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNpMDE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDUvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgaWRlbnRpZmljYXRpb24gYW5kIGluIHNpdHUgZGV0ZWN0aW9uIG9mIGluZGl2aWR1YWwgbWljcm9iaWFsIGNlbGxzIHdpdGhvdXQgY3VsdGl2YXRpb247IE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTE0My0xNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZ5aW5nIG1pY3JvYmlhbCBkaXZlcnNpdHkgaW4gdGhlIG5hdHVyYWwgZW52aXJvbm1lbnQ6IGEgbW9sZWN1bGFyIHBoeWxvZ2VuZXRpYyBhcHByb2FjaDsgVHJlbmRzIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTkwLTE5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyB0aGUgc29pbCBtZXRhZ2Vub21lOiBhIHN0cmF0ZWd5IGZvciBhY2Nlc3NpbmcgdGhlIGdlbmV0aWMgYW5kIGZ1bmN0aW9uYWwgZGl2ZXJzaXR5IG9mIHVuY3VsdHVyZWQgbWljcm9vcmdhbmlzbXM7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz0yNTQxLTI1NDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjcmVlbmluZyBmb3Igbm92ZWwgZW56eW1lcyBmb3IgYmlvY2F0YWx5dGljIHByb2Nlc3NlczogYWNjZXNzaW5nIHRoZSBtZXRhZ2Vub21lIGFzIGEgcmVzb3VyY2Ugb2Ygbm92ZWwgZnVuY3Rpb25hbCBzZXF1ZW5jZSBzcGFjZTsgQ3Vyci4gT3Bpbi4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz01NzItNTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2JpYWwgcG9wdWxhdGlvbiBnZW5vbWljcyBhbmQgZWNvbG9neTsgQ3Vyci4gT3Bpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NTIwLTUyNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvdGVjaG5vbG9naWNhbCBwcm9zcGVjdHMgZnJvbSBtZXRhZ2Vub21pY3M7IEN1cnIuIE9waW4uIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzAzLTMxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlbGYtb3JnYW5pemluZyBtYXA7IFByb2MuIElFRUU7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz0xNDY0LTE0ODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZ2luZWVyaW5nIGFwcGxpY2F0aW9ucyBvZiB0aGUgc2VsZi1vcmdhbml6aW5nIG1hcDsgUHJvYy4gSUVFRTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTEzNTgtMTM4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mb3JtYXRpY3MgZm9yIHVudmVpbGluZyBoaWRkZW4gZ2Vub21lIHNpZ25hdHVyZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjkzLTcwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgZGl2ZXJzaXR5IG9mIGJhY3RlcmlhbCBnZW5lcyB3aXRoIGEgc2VsZi1vcmdhbml6aW5nIG1hcCAoU09NKTogY2hhcmFjdGVyaXphdGlvbiBvZiBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgd2l0aCBlbXBoYXNpcyBvbiB0aGUgRS4gY29saSBPMTU3IGdlbm9tZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzY7ICBjaXRhdGlvbl9wYWdlcz04OS05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW52aXJvbm1lbnRhbCBnZW5vbWUgc2hvdGd1biBzZXF1ZW5jaW5nIG9mIHRoZSBTYXJnYXNzbyBTZWE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA0OyAgY2l0YXRpb25fcGFnZXM9NjYtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIEROQSBhbmFseXNpcyBhY3Jvc3MgZGl2ZXJzZSBnZW5vbWVzOyBBbm51LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTE4NS0yMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvdWJsZXQgZnJlcXVlbmNpZXMgaW4gZXZvbHV0aW9uYXJ5IGRpc3RpbmN0IGdyb3VwczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTc0OS0xNzYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtc2NhbGUgY29tcG9zaXRpb25hbCBjb21wYXJpc29ucyBpbiBldWthcnlvdGVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTU0MC01NDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIHRldHJhbnVjbGVvdGlkZSBmcmVxdWVuY2llcyBmb3IgdGhlIGFzc2lnbm1lbnQgb2YgZ2Vub21pYyBmcmFnbWVudHM7IEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTkzOC05NDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbW11bml0eSBzdHJ1Y3R1cmUgYW5kIG1ldGFib2xpc20gdGhyb3VnaCByZWNvbnN0cnVjdGlvbiBvZiBtaWNyb2JpYWwgZ2Vub21lcyBmcm9tIHRoZSBlbnZpcm9ubWVudDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQyODsgIGNpdGF0aW9uX3BhZ2VzPTM3LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWJzdHJhdGUtaW5kdWNlZCBnZW5lLWV4cHJlc3Npb24gc2NyZWVuaW5nIG9mIGVudmlyb25tZW50YWwgbWV0YWdlbm9tZSBsaWJyYXJpZXMgZm9yIGlzb2xhdGlvbiBvZiBjYXRhYm9saWMgZ2VuZXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz04OC05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IGNsb25pbmcgb2YgZ2VuZXMgZW5jb2Rpbmcgbm92ZWwgeHlsYW5hc2VzIGZyb20gaHVtYW4gZ3V0OyBDYW4uIEouIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0yNTEtMjU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBncmFtLXBvc2l0aXZlIGJhY3Rlcml1bSBCYWNpbGx1cyBzdWJ0aWxpczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM5MDsgIGNpdGF0aW9uX3BhZ2VzPTI0OS0yNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xMi81LzI4MS8xNzQ2MjIxNC9kc2kwMTUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEEgc2VsZi1vcmdhbml6aW5nIG1hcCAoU09NKSB3YXMgZGV2ZWxvcGVkIGFzIGEgbm92ZWwgYmlvaW5mb3JtYXRpY3Mgc3RyYXRlZ3kgZm9yIHBoeWxvZ2VuZXRpYyBjbGFzc2lmaWNhdGlvbiBvZiBzZXF1ZW5jZSBmcmFnbWVudHMgb2J0YWluZWQgZnJvbSBwb29sIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTA4MDMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkEyAjalQlyhUdgVweSh93IHnd7o8=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f281%2f350803" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f281%2f350803" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/5/0/m_dnaresearch12_5.cover.gif?Expires=2147483647&amp;Signature=aCda7p9Uq5iwZjUzLjBT9gM9GcXHBhpbSpHktSZ3aBfaLgUTRrUBMfrhnbDOftVRcOxJYPlWQoOSDtYCMRCSqLqznp4erEvRKXpqo14qAWzUQzXAUf0R2hzkxTrttMY8s2309oR8tNHgB9rVuF2hpqh3QH4eyjVeWXlnJ40m9gcWoDr-jQH7WNKVcWkYKOzDqj0zz3HLWNzUSiszlQ7mPDPy4j~szH2kDsY0ptS4bBqTMPSDLuVeIksQM7hpXYsVJSgGsIGUFqHvKMFIXeS6xBjZCvV-jy2mqIn0wg5gOITNC5m-u-f0z~ur8QMlAsoeOXCWmD3HyGddqFtbkE9Www__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/5">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87193351">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193351">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193355">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193355">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193359">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193359">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193412">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193412">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193429">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193429">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193435">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193435">Supplementary material</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193439">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193439">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/12/5/269/350906">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/12/5/291/350187">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, National Institute of Genetics, The Graduate University for Advanced Studies (Sokendai)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-55-981-6835, Fax. +81-55-981-6896, Email: <a href="mailto:takaabe@genes.nig.ac.jp" target="_blank">takaabe@genes.nig.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Abe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideaki Sugawara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Center for Information Biology, National Institute of Genetics, The Graduate University for Advanced Studies (Sokendai)</div><div class="addr-line">Mishima, Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideaki+Sugawara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugawara H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugawara H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideaki Sugawara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Kinouchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Department of Bio-System Engineering, Faculty of Engineering, Yamagata University</div><div class="addr-line">Yonezawa, Yamagata 992-8510, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Kinouchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kinouchi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kinouchi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Makoto Kinouchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigehiko Kanaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Department of Bioinformatics and Genomes, Graduate School of Information Science, Nara Institute of Science and Technology</div><div class="addr-line">Takayama, Ikoma, Nara 630-0101, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigehiko+Kanaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanaya S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanaya S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigehiko Kanaya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshimichi Ikemura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">The Graduate University for Advanced Studies (Sokendai), Hayama Center for Advanced Research</div><div class="addr-line">Hayama-cho, Kanagawa 240-0193, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshimichi+Ikemura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikemura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikemura T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshimichi Ikemura</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 5, 1 January 2005, Pages 281290, <a href='https://doi.org/10.1093/dnares/dsi015'>https://doi.org/10.1093/dnares/dsi015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 July 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">02 September 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="350803" href="/dnaresearch/article-pdf/12/5/281/17462214/dsi015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takashi Abe, Hideaki Sugawara, Makoto Kinouchi, Shigehiko Kanaya, Toshimichi Ikemura;  Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples, <em>DNA Research</em>, Volume 12, Issue 5, 1 January 2005, Pages 281290, <a href='https://doi.org/10.1093/dnares/dsi015'>https://doi.org/10.1093/dnares/dsi015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/350803?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/350803?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/350803?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/350803?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/350803?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/350803?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/350803?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Novel%20Phylogenetic%20Studies%20of%20Genomic%20Sequence%20Fragments%20Derived%20from%20Uncultured%20Microbe%20Mixtures%20in%20Environmental%20and%20Clinical%20Samples&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=5&amp;author=Abe%2C%20Takashi%3B%20Sugawara%2C%20Hideaki&amp;startPage=281&amp;endPage=290&amp;contentId=10.1093%2Fdnares%2Fdsi015&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f281%2f350803" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87193351 id="87193351" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>A self-organizing map (SOM) was developed as a novel bioinformatics strategy for phylogenetic classification of sequence fragments obtained from pooled genome samples of uncultured microbes in environmental and clinical samples. This phylogenetic classification was possible without either orthologous sequence sets or sequence alignments. We first constructed SOMs for tetranucleotide frequencies in 210000 5 kb sequence fragments obtained from 1502 prokaryotes for which at least 10 kb of genomic sequence has been deposited in public DNA databases. The sequences could be classified primarily according to phylogenetic groups without information regarding the species. We used the SOM method to classify sequence fragments derived from environmental samples of the Sargasso Sea and of an acidophilic biofilm growing in acid mine drainage. Phylogenetic diversity of the environmental sequences was effectively visualized on a single map. Sequences that were derived from a single genome but cloned independently could be reassociated <em>in silico</em>. G + C% has been used for a long period as a fundamental parameter for phylogenetic classification of microbes, but the G + C% is apparently too simple a parameter to differentiate a wide variety of known species. Oligonucleotide frequency can be used to distinguish the species because oligonucleotide frequencies vary significantly among their genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;self-organizing map&quot;">self-organizing map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;environmental samples&quot;">environmental samples</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenome">metagenome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic classification&quot;">phylogenetic classification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SOM">SOM</a></div>                            <h2 scrollto-destination=87193355 id="87193355" class="section-title" >1. Introduction</h2>
<p>Most environmental microorganisms cannot be cultured easily under laboratory conditions. Genomes of uncultured organisms have remained mostly uncharacterized and are thought to contain a wide range of novel genes of scientific and industrial interest.<a reveal-id="b1  b6" data-open="b1  b6" class="link link-ref link-reveal xref-bibr">16</a> Metagenomic approaches, which are analyses of mixed populations of uncultured microbes, have been developed to identify novel and industrially useful genes and to study microbial diversity in a wide variety of environments. With the metagenomic approach, genomic DNAs are extracted directly from an environmental sample containing multiple organisms, and the DNA fragments are cloned and sequenced. This is a powerful strategy for comprehensive analysis of biodiversity in an ecosystem.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="b4  b6" data-open="b4  b6" class="link link-ref link-reveal xref-bibr">46</a> However, with a simple collection of many sequence fragments, it is difficult to predict from what phylotypes individual sequences are derived or the phylogenetic novelty of the individual sequences. This is because the conventional phylogenetic classification of sequences is based on sequence homology searches, which require orthologous sequence sets; therefore, this strategy cannot be applied to poorly characterized or novel sequences. In the present study, a new phylogenetic classification method that does not require the orthologs was developed on the basis of an unsupervised neural network algorithm, Kohonen's self-organizing map (SOM).<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> The conventional SOM was previously modified for genome informatics to make the learning process and resulting map independent of the order of data input,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> and the modified SOM method was developed for phylogenetic classification of genomic sequences from environmental and clinical samples. Sequence fragments derived from environmental samples of the Sargasso Sea near Bermuda<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> could be classified into phylotypes.</p>                            <h2 scrollto-destination=87193359 id="87193359" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=87193361 id="87193361" class="section-title" >2.1. SOM methods</h3>
<p>We previously modified the conventional Kohonen's SOM<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> for genome informatics to make the learning process and resulting map independent of the order of data input.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> When we constructed the SOMs for di-, tri- and tetranucleotide frequencies in 10 kb genomic sequences from a wide range of prokaryotes and eukaryotes, the sequences were clustered primarily according to species without information regarding the species, and increasing the lengths of the oligonucleotides from di- to tetranucleotides increased the clustering power.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> In the present study, therefore, tri- and tetranucleotide SOMs were used according to the procedures as described previously.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> The initial weight vectors were defined by principal component analysis (PCA) instead of random values. Weight vectors (<em>w<sub>ij</sub></em>) were arranged in the 2D lattice denoted by <em>i</em> (= 0, 1, , <em>I</em>  1) and <em>j</em> (= 0, 1, , <em>J</em>  1). <em>I</em> was set as 350 and 550 in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Figs 1</a> and 3, respectively, and <em>J</em> was defined by the nearest integer greater than (<sub>2</sub>/<sub>1</sub>)  <em>I</em>. <sub>1</sub> and <sub>2</sub> were the standard deviations of the first and second principal components, respectively. Weight vectors (<em>w<sub>ij</sub></em>) were set and updated as described previously.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> The batch-learning SOM program could be obtained from Xanagen, Inc. (<a class="link link-uri" href="http://www.xanagen.com" target="_blank"></a>; Email: <a href="mailto:info@xanagen.com" target="_blank">info@xanagen.com</a>) and G-inforBIO (<a class="link link-uri" href="http://wdcm.nig.ac.jp/inforbio/" target="_blank"></a>).</p>                        <a id="87193365" scrollto-destination="87193365"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f1.jpeg?Expires=2147483647&amp;Signature=kpag-mXPY~Dg0i2yGDYbZ1MbEuupI9MKgEA1J5ZI6aabtGX34ghCE1loAAphOokz-5CD8oMmRN8ajBPOLe8bVDYjR0os1w9-13M7doM8Ho6XAQGbKwt2ZOJkky1VLo4qpbDH~8VbxcZW5v4Uu1wLw9scIvLhcW1IUT4RfFB62XyzIYRxbp6q560FbqrxOs~O4RDA4l4QngD-dk8hKzVzStkKM4PWY-ceNY9aANev376EnoisCJ4jR~FSavE~lHDjbNSD~ROiLORvBzHp5AAPLNNU0JkM8SsuFZe~iU6XGf~3vvN0-oaBpGrX1vB35M689HgFS2QRLn~fIw-9WCU7UA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SOMs for non-overlapping 1 kb and overlapping 5 kb sequences of 81 prokaryotic genomes. (A) Tri- and tetranucleotide SOMs. Lattice points that include sequences from more than one species are indicated in black, and those containing sequences from a single species are indicated in color as follows: Aquifex aeolicus (), Archaeoglobus fulgidus (), Aeropyrum pernix (), Agrobacterium tumefaciens (), Borrelia burgdorferi (), Bacillus halodurans (), Brucella melitensis (), Bacillus subtilis (), Buchnera sp. (), Clostridium acetobutylicum (), Caulobacter crescentus (), Campylobacter jejuni (), Chlamydia muridarum (), Clostridium perfringens (), Chlamydophila pneumoniae (), Chlamydia trachomatis (), Deinococcus radiodurans (), Escherichia coli (), Fusobacterium nucleatum (), Halobacterium sp. (), Haemophilus influenzae (), Helicobacter pylori (), Lactococcus lactis (), Listeria monocytogenes and innocua (), Methanosarcina acetivorans (), Mycoplasma genitalium (), Methanococcus jannaschii (), Methanopyrus kandleri (), Mycobacterium leprae (), Mesorhizobium loti (), Mycoplasma pneumoniae (), Mycoplasma pulmonis (), Methanobacterium thermoautotrophicum (), Mycobacterium tuberculosis (), Neisseria meningitidis (), Pyrococcus abyssi (), Pseudomonas aeruginosa (), Pyrobaculum aerophilum (), Pyrococcus furiosus (), Pyrococcus horikoshii (), Pasteurella multocida (), Rickettsia conorii (), Rickettsia prowazekii (), Ralstonia solanacearum (), Streptococcus agalactiae (), Staphylococcus aureus (), Streptomyces coelicolor (), Sinorhizobium meliloti (), Streptococcus pneumoniae (), Sulfolobus solfataricus (), Sulfolobus tokodaii (), Salmonella typhimurium (), Synechocystis sp. (), Thermoplasma acidophilum (), Thermotoga maritima (), Teponema pallidum (), Thermoanaerobacter tengcongensis (), Thermoplasma volcanium (), Ureaplasma urealyticum (), Vibrio cholerae (), Xanthomonas campestris and axonopodis (), Xylella fastidiosa (), and Yersinia pestis (). For details of species-specific territories, see Supplementary Figure, on which numbers indicating species names are added. (B) Phylogenetic classification into 13 prokaryotic groups on DegeTetra-SOMs. Lattice points that include sequences from more than one phylotype are indicated in black, and those containing sequences from one phylotype are indicated in color as follows: -proteobacteria (), -proteobacteria (), -proteobacteria (), -proteobacteria (), Actinobacteria (), Archaea (), Chlamydia (), Cyanobacteria (), Deinococcus-Thermus (), Firmicutes (), Fusobacteria (), Spirochaetales () and Thermotogales ()." path-from-xml="dsi015f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193365/dsi015f1.jpeg" path-from-xml="dsi015f1.jpeg" target="_blank">View large</a><a section="87193365" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f1.jpeg?Expires=2147483647&amp;Signature=JHQSTrwzm1Nof~MtbtDolP~MOt3~mApuwn9q1ME~nt5gbGkSLhKxoEPPP5VLGKC~gmTE0JeyntqtHE4pxXqJKWRYu06txoSukInD2Fzx7lApdq9IW2sQIXhrIvjogUiPiePyXHsZiwVBIwxrLNx2qPDIq5baRbNwHBibR0zbolF1H1T6Fk66J5S3luDxQBDla39OUKOseHTgIlsTiR6AGoNtrlCoZuKxuFKulgDTraf6GiTHJSqiHrql-JaQItkw7WMcstfbK2cNnnVJN8ATgHnFEiiAsq-RbigjLeZse-VVtXtwiqYH8HgmkX9DwWnAY6OI0tE85YRoBQUlE2L56Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193365&amp;ar=350803&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi015f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SOMs for non-overlapping 1 kb and overlapping 5 kb sequences of 81 prokaryotic genomes. (<strong>A</strong>) Tri- and tetranucleotide SOMs. Lattice points that include sequences from more than one species are indicated in black, and those containing sequences from a single species are indicated in color as follows: <em>Aquifex aeolicus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i4.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i4.jpeg?Expires=2147483647&amp;Signature=xOH-UwkA3CzBeW5AOqGuKlMPsf0NI~6aEaXzIuTDq5f9KbADQdVdm5me5HTYLOXF-tIK~eFwHg6154M0X2qcLz6ojIQqILGPg3BYdHMap6I5EzYxOeK2jJFG4izosyM4Y4~aoJpGaglB8RQHoxTdVbKZus0vm7rhycSH-FJpXi5DzgVD-DIqR1rMnJJsvOX4eYJGex~Z2himd5577jXQjDDSlA56rnmXkGkJkcWFeQtQE73SJZjMgF9GFXz37k7G2jqLw7hRB1muzZCxOdZt3Y5ZZ0VfAvTIQ-ki22WSsyi0hbKSEE8MLe68tOv7dheI7PR7F8hnYXyRGwCgA17BJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i4.jpeg" /></a></div>), <em>Archaeoglobus fulgidus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i6.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i6.jpeg?Expires=2147483647&amp;Signature=c2yw9rqM0fNCOsavaKhCuZ6f3C2mHrLo2j1yMms80LQBlpgtGldtLfyYdTo424aFGZxtgx7AHKow2Szb3JbbaIVijq4XD7eRAzL0zR-ye4dogLGAE88eTln1CB8QsdnSk0Y446uQ74ihr~doL7j18Y2fXCLqd7R3wOT8k14hg7rAXoWveFFRbrY8hu9ZgXirMD2~YUfcbcv7tmriOOw8Tc75hWLmvbQzvR0zywy1ZwX7HqsbGEPjJOuK3cayCgwwxobT7GNFnGQJNkVxTPpLE9fLLSSeygBeFdXR4vtk6c1dSugQMkcKu316VG5biBZOd~r63htMG7aZkkt3-axt-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i6.jpeg" /></a></div>), <em>Aeropyrum pernix</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i7.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i7.jpeg?Expires=2147483647&amp;Signature=0ZRNxwAhm~3BzUibCVLnqvTFjGVbmnVBJ0g3x6mGaYKRliT0NS-Zwh~ohok0cdZDwLArP7ItNavfQD8IQ8R3v6dBtP40p8UnzV1iwYjkGbZuBTah1czcGs7USP7kE6PgcfrVWAloIDXVDLMicWA26vCY4FoidTqzZaMMTkUqKfyuwt19ZPUl5zxZFOHWCzVa2qZdg-m5aMOy2GLrytyaXfXW7L1zr7BQIdTKnYPbb6oRmmnK6fXoY8uw2fiRo~4SUP8fU-tIA7CKqPwImgHWr1ssAH-Epr8y~T-QhxI9paDDexsr2JxfBboNFMaOj3jfqpgrO2MNZ-ievRUTdSWing__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i7.jpeg" /></a></div>), <em>Agrobacterium tumefaciens</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i8.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i8.jpeg?Expires=2147483647&amp;Signature=lY6T8S22V0W79mvDr1Tnta2hWlZxb4x047QcbgQiEPc5TJbK2dewRT947nXyuDc9EPQCVlaicn79vd2hDoIuGxgKiaJ5X~-3dCJvWSpx5J5LknwyyC~GNfqEAd4l4gz3aK-Nbo9zGy-gA3BcQW3pNND2qXOqHnNQsJs3ff-vkAVlH0Rb6v1JPKiFq5rnmTXK1T-PIUe~4O7sY0G5laoMzPiM4Ny1LCQ0zgITLD8pgLFDtRQqSo2efFbCdalzwTpLg3zwGC6vyhlim3SoBvrajNVLitDuy3n0Ht2foC522uQ1zSNSOXk0NOVg~fpON4T6q2hkrvVqw0SnfcVQ4ztdmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i8.jpeg" /></a></div>), <em>Borrelia burgdorferi</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i9.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i9.jpeg?Expires=2147483647&amp;Signature=G2m5KrkMhjQTizqIqhrYoyWxBLZoq223wDED4IqF9~Ozdk4GV-Dst-4AYbpKyY1ecS5ot8blloXTUsnsm5dDjAa22rIQ6mrm8USgfSFiE03tFf~wv96XjPHaSnwZSCifsIoytg5cdut781vfNsARR4Z1ebpDq7TAhatWDU2qAyVBPM0OENibmeuL3-GrvWenHXc9aK1IFzM~JXT2KQ6cZkGpBAQFud-ksehVGQ-blRLcjmHgU8Iu~-9wXG3yrN-cZdgVzBKdR76uqR6G415lur-HaSWHXxGjKnOGdCLw3zgOxgXiMjIZNJuJrpJGdt5FmHzLf7pgp5yP1eHQ05yveQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i9.jpeg" /></a></div>), <em>Bacillus halodurans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i10.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i10.jpeg?Expires=2147483647&amp;Signature=V1ruz3jNOUAR4GnKF4n-PsqGgaEVONybH4wU6dqegJTRkO1drEZqfbOsWGN5I6oWk6A8oo2z~gOZWXXyxsiNqA-ZOUELsEam-v5dGvHh-uHIeX8--U5sRIzWB0RKMMr6T~2q2i4Wr61sP~JSItpMngQC~wt1psnbfYsO8naDVD~xJlRxAmCm6eoVj0j~6KgGnH824fFDT8w7ocMvVbmGF9FaHd7hyJTLqb3nipsbHkzeRhPLsuCr6fEbmtS4la34L3lfVkcBa5xuv9pHBfbdboKMxDbbWqe88zL4I-befzEhAL-QTIrhCQKE2QAHphA8OZnQrwRwIFfOxmE5NcVNyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i10.jpeg" /></a></div>), <em>Brucella melitensis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i11.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i11.jpeg?Expires=2147483647&amp;Signature=WODqdQ-VLqMFL15Wf-2b6wurLM~dqor72nnhKqtC7S7UIExlhuTGpqAulQLuNwNUHtfF6AbqKr-mlPrtZuk3DosiHWYLd0oHM3Yr3F0DxnzC8X5CPs~FNkIIIX6UYGYL6XD6X1KegSb3fdMzySGAEhD04B2APezML8e8OdXzFYbDfg416CVnZZAByPoWkQqJGnfutw5mMVGFMoenh~lPuA7-sBRnIWmiy1L1G2vgihPBAyAxrTUEum09Skm2Agz7mchIQtlr9iROGkh9vHrFNDFqRjMpQPpY2FTpowhEm00MTijBSeClmQ1-Jmt5WjsEhxrEvFas4uNmsZulLh4pMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i11.jpeg" /></a></div>), <em>Bacillus subtilis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i12.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i12.jpeg?Expires=2147483647&amp;Signature=S75jRXXUVYI~rPZLjG-yaNWEjk4QVHAepGbJvmq0wqEKYMNfC4mlMNNyO9WKfN3Yx~2WooyUxKTu9dEC6cDCVznXNksiSavr4xpQJEGc~79uoE37ySiUY84G1EDhqLeMPUnlMmhOejtMZZmepD2SgjB462ypypX3~4cfF4-6Lk~o4fzsOcYaDEpmOnIsaopHGzVpDj2bR~ydorzvKDu3RRQeSvQr9BE-CdeB5GuZkmiZlwH2mD4eAL4Ze1T6JH3wWmel9bOd5jeSDDmcwnbFDNq6YMvoowmB0gPdcyY~YBG8ismcpG9pv9Uckdf82yvlEpRJwjRfeZDYPq9xaNNp1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i12.jpeg" /></a></div>), <em>Buchnera</em> sp. (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i13.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i13.jpeg?Expires=2147483647&amp;Signature=rfibgUoEkEBOmM7Mr9JH0rlIozjRwi863UBrxm4IrdGirfFDDqCXnCxeepVC9LCvhww2CWK1lRVibHbriX0dK1Dzl-VPid8kLs68fLIVU6k1CZ4tIg4~RFSbmGwvfQecvdTsssQTMPNUP~RiYPWdS5aj5~Gn8DWTcVVEQk~nkE3FDYQe2bi1ZNqcUTInN0Ib6BCpEVNGeTCAPIMbjZCn7pz-pr0bK62nDMu9EzNw0-p7iUUhJfgv~~N9JsMOJAZnlZDdNoiKif7PbZhpsabJFKJMM6VO9cbC2YBF49BlrbbzrjKDmcAOuwN1i67xx3g9Ld6~kvyI~CCVPFm3YevLBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i13.jpeg" /></a></div>), <em>Clostridium acetobutylicum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i14.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i14.jpeg?Expires=2147483647&amp;Signature=NyDYBy1r2VGMnJdWQRxPj5vpwIFCYG4SEtHhx3q5hsNQuIScgugd2J5PfZCK3Nwroa6FIgEAsAKlP42XUGioa8~VVtc4aIftZUYeVXDoPUPHuegf0fMt2IUdueP4EqFEHytmDayJc0udBCdTiLvax8NZwnTlqqwzBTBeB-eab~lRGhNBD8Vu1HnQ~L3Hn8mcZ~q2g7Y0LMkqtT08Clq4O-WKoT9YAg0wIKJuU--h7U9gNu01E~X4yvgW6SwyZLqdSS6BGXUGGF14UNMVUo4WKHb-c3iTQGDrzte~mi1nv9l5TvWPd1gB2iSPn4lAGflswJVenMCyXUP2w1n40jadNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i14.jpeg" /></a></div>), <em>Caulobacter crescentus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i15.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i15.jpeg?Expires=2147483647&amp;Signature=l06MYwZ68zk7rFq1Vug4GyccFEgCj44OTbs3dF0--INTe5OnyL2vC0XS1pUe3YvaOnnKyEMkHthx9QOQ5IpUOIXk4OUI8SbH8QYgrZKlfTjp9jMPvYniwIKysyMYjnYqYnPrP9ZKs5UOPrj7mH99~6v4u0Waz9-TfxPAyHEhJiV2fTgjHETTWADkLtbPasXjjboDf7EBpfgoRrr4KGeP40GPbzmb0IY4I5EtLJ5~jo5XZMgRmEuoE81OZA-jJc1-Nx8tyKA~VvxDqZLezCcxaR3WVNyolxK6paZoHWVYD720V2ikCDPvhRRkEGKNsLEHqfmNziZMN3MpPXM93TkMSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i15.jpeg" /></a></div>), <em>Campylobacter jejuni</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i16.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i16.jpeg?Expires=2147483647&amp;Signature=EZX5fHrwfkrk6VQfnGvewyggm4O8WBXgWj33VmS7lpPrn6hcgSpoCi~vrsSdO-fLk48pXspuJcf7L-r4SmW7dSCghyyd6oxzqRLIKHc9Sg2K2NOQQWSpPcKu~MRkm2KNIo8TrDds2XLAslrmvn6vXAYXoamYQbBckgOfEgdav~jhiyHXtN7w7H7~J3VAoUvsU7dgP12Oo5LNhSPjYqdivWoDrPJiBsdjPz8cH5wOCpf9oa63xio053qO~wUGjC~gsJBYRGmt0bjghhZ8tle4CZN5RkFvUSp1JPA9LgUS7MVOSX-3~SMaz2Oa9cQAq33-K-jQpaYZLOs7U-uS3E4Zpg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i16.jpeg" /></a></div>), <em>Chlamydia muridarum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i17.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i17.jpeg?Expires=2147483647&amp;Signature=vou1Fa8iGX9QAg2hqkvcKdJk4t3I2A0etQDBa0-GaH41pPhxkS1FB2s9t703bGyXrL3hUc~Q8gkwTHWxJXxWLLuv1GR7XkG6uFabNQO4wEQYE0QA6eJ2W4Y-7q~QlR-N5UoIeIh1Xg4Ccf0zItIoSPEhBL61fDzkNpBtKRw15DHQnVHmoN9LjEgemw3eIQnH0aS-AUX5aQDFdaWlwiPJ76Oez3s~Rr63txcv-kwql3EfklWW-OK-HbNaFKKFOtHuzXVInsNOnu5a3VVHqiAfbRuZs0YuvGyBzLi7N2JPxb0kbX9j0lmGWKr7FRmYf1cGrTHHmO6NObitkIyWXEZj6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i17.jpeg" /></a></div>), <em>Clostridium perfringens</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i18.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i18.jpeg?Expires=2147483647&amp;Signature=AtexqjDu-ILh1yHG1NYaxISKZeD6FfyuY84~S~Pmt5oKH4JXZbffY5DSYfytwuoRuVJxuhUwFjJXjURsUVwZmUQEA~qKunqNFCKwkkPfHa2OvrJGIXWJB-QuefffLiIOq22MESkK-9pk9~RVbDjh3YmT93I2oH~oX2CvhDjhoCuQIzn-gl6emoF7QGZuksTi1yfMCe~EQSfKy0G9FiCTklV0RE5~~7p2ZrCOYIzbVn0-X2KDqZ5gEcVbD7DlDZnA3ChSOJ~KXYo1WsnbZ4MX8Oe~wWwiiciNsjWKkWQfFYcEfsqj~RhnX6~JYgxkoJCZLOlB4l64J5HRzl01qFKZpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i18.jpeg" /></a></div>), <em>Chlamydophila pneumoniae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i19.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i19.jpeg?Expires=2147483647&amp;Signature=yB1ljGdmZDe9qh1oU5KwKmFshcB1vy7vmHfqLOihzzipNPfk0UWhzw8fwNUofaS5b7k2k6buu7A50HrbWhD3ws63MFZ8ZvxX7wbd5c0s1lJjwZQ0rgz3m1FAWTgYVnyD3V5ZJEcknlWbiRMxfiP6l0HVZ9fQOKKFg4jQ9izSjR42REUHoQN3p1~Pr0q76wHfvGpSzIM2ZsysYXXAQEWPgF0XFsz6TlrDauO-MnSt-PbShGSQCZbwFSBVZedcuyKsBj-0GFf9iOjEBYJ8qLvl6DVAuxM9OAhg59hMePhR3AXxBO8SBY2Ej0~qneS5jGCX5Mlh0SGljYm0HAbCH0SSzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i19.jpeg" /></a></div>), <em>Chlamydia trachomatis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i20.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i20.jpeg?Expires=2147483647&amp;Signature=oLgA4gC5jvowGoUOL6j1o1UreE4BFQ78dVq~zAu6-svAjNA9qJMSNOegbn4zLnpNhPdN6pcL9-hCXh4DCvfIKvcjRTMtMhqpCYs8pHDladlDJJiCbNtffapxF9GgkJMZACR9XUt-yNC7DpNP9zKL0L3WxJ-mJ2~AsnNYVBpUd~gMAUjl7ckOIEYWCe4Jm2FLdPEzvrZ-dL0ToP15xI-uSfXyseuae~UcSahrfNsQxGnXfNqmb2Z05oL0pXEdpD3LshClcuJtmmNRHG4~Q1ncBwm1p70aLftTzzpa3sg0xp3hX-KKlUowH25bdlUQWPMfp-Ip2q4ngLuMVA7jk-DiOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i20.jpeg" /></a></div>), <em>Deinococcus radiodurans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i21.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i21.jpeg?Expires=2147483647&amp;Signature=LOuHz2kUQJi5ACaYT1P5ZzNLm5PcMkdstCbJPvMSL7nZZ9a0bLsPvtKVZOaIG0QmKgL3QeGQYRWy3GdiWBmq4-u4avCDfvIzrB6YVK4KCDdPJp-HGENXJ~SWQbqtsKiKFyr-9b4IpcIjZfQaSarHoUSMGCyzQN-xiUIQwE5kia-ARIafdtuGUNAqxNgkNJqRAbdZ7IqLm-LBvzdlgPJcrcnyoSImjVbbPObEgwTW2Am5wLAhCYPTzJyXxrlcjUrop2qbytgxa1bMwOmfL0IYdSaqkl90zQj1k4LLB~ZCKrPThAjAd6MeQ3rULyfkALXYScuGM6aimN7-L3-tOBRhuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i21.jpeg" /></a></div>), <em>Escherichia coli</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i22.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i22.jpeg?Expires=2147483647&amp;Signature=vSOOkxF4GG-6Z9Yd11m~cq3yJxnagVO6McDGIsVrbLj8J8HO33FCnk~C50hBTFj2zhiq~UfiHlKxOCbZgv564U7wdU~7Zi5wLz1GqGrUFtxj7QV70YHrukiVpOXZe2-UcoBvBiKP-8NIVuPkaD2Miz~Y2wiAIff4~qofJdzh3OEGjyJQvEE4d09MQVq4nAqF~-vX71E7OTTd0xnhdPFC69skL~tG5KvGQ4WA~CfuamxWKNCtEAOAN8dgMrEvPdw6wBy2IuCOvaBWlWu663k3l3kTbXXVQxCAILQcFbDvTwYxJiOANufRTkf2MENVxLPHGznjQNRRdT8iqV~wuAJwSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i22.jpeg" /></a></div>), <em>Fusobacterium nucleatum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i23.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i23.jpeg?Expires=2147483647&amp;Signature=elVMkNwaHAuprcOBSSf7TLnNBLv91nUX6ZWmxkOmzds87ZyHHPLvgnX-Ci0vrMhyGsh9hGzcarqLKUpl-lts3JyW6YJsZ733UWbO9j5-SE6CRhY~B2S9Rpplc51f2NkUlQDxxhXdlwWJ4lWbBPa-IOg9pKVtIRiCw8r1aF8y9-vH9K5PmTu4WXsqmaQoXB1L8vvA-uNXqiLnFxZZqlh5OIah~I~xP5L46vhiF-K9fWJNvxF8PluYPED7lKU~PY8V0Q-fwh3wnGSBw8f930qu76k4KsyaVYIdoxYr5cE-3RsZcXE0vDYwD4zjfwTZv7-ylh2n2SL1YEYZZ0TQ0Qmp8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i23.jpeg" /></a></div>), <em>Halobacterium</em> sp. (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i24.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i24.jpeg?Expires=2147483647&amp;Signature=3KysfMZXhbXiyBl-Z4ltsS7TeAT-xYLnDA~rAFPkLFs8Yr3xqjHoPlAk72AqMu1jzaGP5ZF3ejmzAIXc59NLSn5OwKU4E8G1ocAlfCANrrzCgt7qVdhBQeEOvn4iKEwUfpqN2ztiSNkdiNCdm4klsZD-~jqGb9fxRIjnIFDxSHcm3AlvEL6ybF4ja9eF84DA1F~-230JYsks1wfErrAvVwRKuo4pTVVlvYjTaBc8Q1J-byq2dqZPiMYxBqyUo0XXWii9npNvI9tvX1GZQkzvD3iOEnJ1JqTULs5Qo8l9oFzU-YUr4gWe~6ncy5AS5YQw4j-vGrLWbmsazD6J84fUug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i24.jpeg" /></a></div>), <em>Haemophilus influenzae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i25.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i25.jpeg?Expires=2147483647&amp;Signature=SvMlIerWZsYLcry5PYEb~GpT8nxLDvTyA19WH-dCOykXSoj7o6dJG5GTwepdbdyLbULhsBUxxPhKw-8-mIeyIMtjtJBI0r784AuVV9WG-le~Gt2dLxiZrgzYIfF~xf0nOXs8Llt7hy~lKt9GiKnot4PCOd6XE~44YXPTm0YhryI-cGzyxrorpFoXD8kpEDz0GryMeLW6phz94tAdb4XFfUuJ0POThF-X6cfz2KModVJajb6hYfnEeNMs0vq5SZwb0-RuImLBO5n9CTAqoGwnsdx5N5RGiC9rYaC3WBRmpGG0zTq1AUMnQyOI23CCQMaW39VP2UKjaEbqYIqsuhXiYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i25.jpeg" /></a></div>), <em>Helicobacter pylori</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i26.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i26.jpeg?Expires=2147483647&amp;Signature=wUy3w49N~hrGyHDPU7siaCIM30HYfbCBJsDCsIR7NMHSr0R9fwDxfub5X5HU8Gulsuf1Pa1cV3kLz8NF5EmZz~lMHA~RgaQLBVNLC7HPPLrG1pxtSJ-UmCNw4ilwFO~kffO2sdwzjSmgYjFBaflDcuvvP-JOWfjxpcK90njddjpB5zbqj0yeH03sa6nOP6wecdzTcs7~eyY7GdO7eJzposM360DJPUaSOKttMKcshbrgq06AP8roJofWiJx~lXDgOVwkQQyzVynrlHKKduaSqUkJeC53BnbZXZgqXc0kDs05rm7ByqN7CqFo1vpdsRzdPwB6GVcBFqJZo45eqWHYrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i26.jpeg" /></a></div>), <em>Lactococcus lactis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i27.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i27.jpeg?Expires=2147483647&amp;Signature=bZeoJa9Dyyjumw2ekFNi86qS4Qqdee5oBpATewk~TL0BSPnTrFthFYSbKbP4OMtpm2jsCLJFDdlZ8d-wSlZ8ZRRSAanX1e3RhagsYsQLhMBr7Lw9bN147EywP2JU9nsSP64Uq7RbD90jykZjyVclUzShPBg~YFgnicT-zaJmyPzECCVIho--Se9~IPuYLKIa8GHUPCKQ7dryfJvI3OXhR2dlB2rhj8qVwXwzhfCRUF1RL6RbZQN8cqXV5OQCdkYfzx0V~WphcoiHtQYov-0xtf6sB-3k9uabLQb62tM7u36OPnOD35Mz4MW-32Vquhr84XirQQgi~wIVXrRJ5DOJIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i27.jpeg" /></a></div>), <em>Listeria monocytogenes</em> and <em>innocua</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i28.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i28.jpeg?Expires=2147483647&amp;Signature=pKHojGpDSWaCYO2WKenY77caCzrEEkGMiQoZoEXF~ty2ztXs8OsnCAPt-gRDspvtccgNHbMgi3f1X0FmOlsdthsUG-~P72RWYQw52llOTtfofLJoHvSr4r-jX93AQB7qU2ErLfzsCweeXtHiZJLPKy5ubrvPaEM85wOQfWGjCj1UXRT-Iiw8P-mgcLFVzza3w33cTfjCiqOMVZHboD72uLy3TmQu3XxT7odGOUIW~c5AzhU3xDpHa4nyXC70LDy7jt3lvbTrsZ6epRrCmqmxANoEBzeeeSa75vGRnvu6qEbWpXbINoD6Hh6tlnS8oF6a2KCiMiF1umEaL2O6Qdft1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i28.jpeg" /></a></div>), <em>Methanosarcina acetivorans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i29.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i29.jpeg?Expires=2147483647&amp;Signature=gFjD5jV~yyTPzNK41mPLR6G1zuBikV6lh-ddX7jzlJB63i0jiAEtMs4YIS23tHa8FEHWWlALud--I-d7IiABwyTOes-03nSwuKaBnFTf3YB7EnxJ-pvbXLjzrMQV65IsmI8uAgaZvdZcstU3q7OtrvG8EpuTckO65pYLtCWc9ptgdg7-alI8DWqXZJRjxjAoRq8J9U~I~cfJNdxv7HO4OQq-J1dBogwdmAV7ztY1uj0b32pYDfxxDTwAL1Y0S8iHoXgS~yTZCqejs089pA1qtLoA-rnOg3QJpFUrwHAnLp~jz8t3k6e2-8PZKdH8iE38JQ-96XojTgBbKj5McLxWVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i29.jpeg" /></a></div>), <em>Mycoplasma genitalium</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i30.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i30.jpeg?Expires=2147483647&amp;Signature=lJjFu0Vt53kgjRA1fJTaGVdx5ab9dNBf~ObrTB3I3Z9PTYUMzCwwsf6pzTG37oTYrxFgKV4tBGSbeX01kRYZsu7pgtoCnmnZUMAQSdn-JS5JHGQl4pIU2cTk0-2qEUvZGifPN2rNAEvpQIzAHeKrPLe1e77tY~t4DlS04dG-uHajumYZFiy2XG60kSsQZF7XXHb1eoE5aWP-wH7-jahIRL1EHGVtzgRFctOSWJbDJX1ZMru8iIwsL~Cw8mVYW4wGwVQx3-iMTqsozIIl9CLvtqJgevz4LT8t9nCIU176z4BvVGZIaYsSxB2Uspdxo-mo7fT~KNelf4B1DZgjynoi3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i30.jpeg" /></a></div>), <em>Methanococcus jannaschii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i31.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i31.jpeg?Expires=2147483647&amp;Signature=M0~P4wI3vDF9xd9KhoVJ2hZSAGOAw4KsuQjOMU2JS7BvUE-zZxSSEQKK4w-VvaoFZks7NNpiqcAYDAUnprUfSCv3u1RDFopBlafn0PjNYEOdRjyHNJpR2drQrXIJ~KIBSkWIPIwRcwwkHpx-I6DWOEc3MJUK5bQWmbz8xz6W8kbO5dmgOAcEGOezyaX0jkRW4N4fvndAHC~H62ArkCtLRdP8AI0glItLLvpO7tc4zAzwveIMC5Aek2CD8Fig8vg76BV00XHDsKMAImOuBfXx6sU9ODj0Sc4HLL8o41bso09oBZCqQqTWx1OC0AVwXlm~5DNXtgij0HIeQzjywsnHdA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i31.jpeg" /></a></div>), <em>Methanopyrus kandleri</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i32.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i32.jpeg?Expires=2147483647&amp;Signature=Kz8j4YaC7vXUpouVL3Iysocoonsx3m9R31Bl~VxgLmXFMjvarKvmZf~Vsa8imlnZU86XQGr6RVeDYlYE6FE5gHRK-8iTzcC1KtijlgYHvJnQgh46usC9iXQADIjOmyYCIKqBXtO-v06y9G1MMfF9i0Ml00QOdf0BPgKICXuEaBD8KkIjAiFZTmS7lj-eLcpoXyZNva8mmNEFSxPPbAMWqEscqTFQB6fLAgZhEXLbS79kkndYdHCPqovg1lVRonVzv7yfnJXnpjs4lAVZx3RWuedBMYeGLhyL-9Y0j5VQ3SE46hfrTuy9BejN4kH8ozNuE9Uyyq-k97nf6vjHyvKpcA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i32.jpeg" /></a></div>), <em>Mycobacterium leprae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i33.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i33.jpeg?Expires=2147483647&amp;Signature=B-wU4hfcIJBIMkkgsKXr8z11tDODQzTfiZCr3qemUxoFybjAGSKuIXlg72FvI6l~39CDnsbE6KYmGP~8jW8kZlICLUpqzsHUkx2X0uZoXXd8Mzg6kbFsSknzO2kmQa3jKRi7OcDEBl51BjNV23aaDcUVaPHxxQMxGrAFRbqormjttaYOH2ODZxVkXCPqaT9Ftg5UacCUwFUBftFv~JSGC-RicfU8ir0lckIOkFRBfamjvjkMY4NeBomElRiRTdEg3PwgmOZlqeZXKz3evvy4rFVD56NeZq2PuN3-5je-ek50sTUV7Ulq-hLjZN44sjG5Dd~9ifW104OYMamc1XK8NQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i33.jpeg" /></a></div>), <em>Mesorhizobium loti</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i34.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i34.jpeg?Expires=2147483647&amp;Signature=ZOSiwsTGOv9KbXAeR7TcQWjZOOOw0xiDYjiLhkqkR71DRq6yo-nSKyKtNkqQL7PmKCPJcrrhZsrS8012hrIFCyZj6KXKTBTZSQJ1ietVkYW0925IAHZ7uTXA~4bJpimi~jNcbqE7uaOpPExil~Fvz7-gPTE1DG9OVvEwqHtPFEqgumrlVTCKeguJisgdATkkrsjeeiy66tJEMhxXMfTO~H6s~QxmwFMeB2oBd-Fx8Q3PGb1p3ZD-z~oCFmSVTxqOymHm9abb8pFazlHtdUWNTSPee77BFu8LN1TCKpdr5yuv17~bUNFg8Vx0UXsGs9RSnQq0nR8Q0ieMTe5fRfO25A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i34.jpeg" /></a></div>), <em>Mycoplasma pneumoniae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i35.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i35.jpeg?Expires=2147483647&amp;Signature=xPAN6asuQByIYsO8WEnsYRT9ZpLOecOXFhWF~2RPKKaO7Ja3RbQCzvQ345T62Vrt4XMxC5nQ6vVnuIkIt3d~DkF7sJfuYiEnChIF5becDk6Z1n-zo1l8T0vzx-xgPU3ym4t8YL~RES1vEVM9XXBDhr5Kn4lPnOL2GHyW~7-MxqOLHCZV1sz82XVCZ6H9dHJW73EKVmsOGc7~sexKwDpF7OU0JoEsOVOzKdVfVEE8Aac94VTY~BqwGl-Q6My3U6ASS81YuPFcGwyipfzcv3eeiddSqUwL9ww4RoMExsHhGSOH9jVZwvOBk4nqEVLMUHOX2G9~hrNJayMikkluANNb8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i35.jpeg" /></a></div>), <em>Mycoplasma pulmonis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i36.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i36.jpeg?Expires=2147483647&amp;Signature=ZEHZ~Ln4qwvFE6yBdgg-3HrQLMOvWkg4BC69OchogqENlNsLXEllZJBGdXk-ZMp5On7uLg7Ty2JwXK3h9ODbq5NN5Km4im8gFM50c2OWu5XNP39QNM4wCHqVwqIvZWTJn908OU6EJORCZfYMZB20SB5WhXOBnrCpgkn7ieuT58RK~giPLoTb3YK1cWp3jb4Ely3q0ZGEUXfwLRlbZI3jLVuCTR56XfYnyv4yMETCB87cWWj7ou6CBd8HKaKP5yy1VCMNPC8Q21zdVH8Xm5tHI8~Ofvn48pvWaIufW4QMkHaU9Nau7gbx6QkOCtgedJYT~g-MSB1dlDfNRh51mxy6Gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i36.jpeg" /></a></div>), <em>Methanobacterium thermoautotrophicum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i37.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i37.jpeg?Expires=2147483647&amp;Signature=iSS~SrCK0MQ9ULCtXp-Mn6tZhEgR2B3v7DWPfZ2GubZvFIk5vWj8qlb9X9lbTnU7U6zAi6LvNWmJIYJfRM5UHD~kemVASiE8i5hr7kU5sthr0d8XbkFA5slE155wdvN5xnPten2Jy53MnRbQhQ0Bsk~lcFZPi4z1fFSkxRSTXkbZML~vNwtQq7UsuKFF5jRCcoLGOf~h~Meyb00aZnkC7FZbS0yy40WSW2Xp-PJFgFi1AuE2gwqIDGb~mS-zJcY3NI9f51Ys-e9tO8kkKnVeta29P9yRzcWgmOy~e5GBLixZArt91DBGESHH7wNU4urD2bkHylEw-tco65Lw38PQTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i37.jpeg" /></a></div>), <em>Mycobacterium tuberculosis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i38.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i38.jpeg?Expires=2147483647&amp;Signature=OEII-K57UgCZBdJzNn6ZBqCjx4S5m6B5qj-jmkDexZI2Mnu4xtHTNf8VEwNy5J-q0uTqBIdvl8omQMlEhzM3bMcfKHFuyntRC338frwo0zfEChOGNIi-fl00JagnIqzVcoe2a-rIUiKcn1newuou1dAATw7lVYpszIGgvDpq54MI6Ms2pcsuGkisB~N49701Myo0qiuPIZWyuxV93W8~XEwb093VXUSk2zuUoxR2IjCacPrg0iHDsp3TOmru8cecRq9owrDKMJRGMQTCZ8XyomccPnXxqI7heFrpjk6ewtCHduIrbqDG3T2zm-A02kBiilmYo4K21SmwAnVvlr7XJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i38.jpeg" /></a></div>), <em>Neisseria meningitidis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i39.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i39.jpeg?Expires=2147483647&amp;Signature=fEBRlAUFnvDTerPVwPqxR85d5nKPZXcgDpF~CVb1KQFAWgrfNW~NMTB1iqCfK3Fyb-9H~bv7EFzy~28J4YDcXvcpDkXRy0eEnRuCehd7xcI0AU4YCyUNlOm-OCQpM2YSN3ytVSvkaJkQpPotUrq3tIgMBM-dGy0HSDFUNh8nazFOxqPFaRtE3m3l1RKnGO~G38-DFzf~Y82ZUPPXEXSPl85kcecwGSeU5TI-LwTpPF69F0AFd0BtgXI0u~G60a8zLQMB7FJdfmMKtbdpAZYMUdJRRMLIc6KsLdlJIa2h1hqvdS236mow3SWBtV9q18P5gNJO6IeySRS6ApvTd4VqaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i39.jpeg" /></a></div>), <em>Pyrococcus abyssi</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i40.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i40.jpeg?Expires=2147483647&amp;Signature=LW9RmjSRzOjsBTqXttlejChLqFPRB~O~cfbwagCER3pvzLqqGvSqEuATfwzf8amqlZLdcAnisqfxLdlFUkodVGRuOH8iuawUhs4-kVHMx16NZ-cfYBcXbUDsBKQcQFE3yIuO-E3CN0gD5sIWsjbQ2-Y2N6uhkSwxLT-r5DETyhBkFmXgy06MYCX0-l0pD7FfeIjWN~0LRvAHdpjOkAg6AFYyRZDOVg6xF7VqH2soxu~rRMYosVDTW1IYHcmbKi2zNBaKqywU1d8VFTzs2XVrxfO8~PZcjoNpkf2zOh5nnz4a~yQxivr5cmATZmbWYR4sFAzy5d0~vqU3ekW80waLCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i40.jpeg" /></a></div>), <em>Pseudomonas aeruginosa</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i41.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i41.jpeg?Expires=2147483647&amp;Signature=scRUpBFgt7VfXQsnqi8fYionCBhDLFuF7EwycJkXMTRKZANXbf08rKoor9bROZ1cs6iWc0HA4oDl9vTsmjZu1tBdj8Dm2DuApt9s~RsZ1tbUxeEUL7A9-TS-XUdads8v4Zdf5LdM3zBR9LFi8McCwod3OB--F3OGTH7dzkszbA8F3eZ2GvPYhA5x~qiJpZ0Pti4NPQg6Sv41eS5XdBbtV765nFZAadnDvZxt2zTYBOsv7Bb0GUyEc6A0YhSU4M2w~xqVGsUuD4eYsxGOKUmwKDi7rvt1CEUi-EGg1ds5t2hUQmpicE8ZNh4lKP28CkysNIZ2igr9K3IFHuv-Fvp1~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i41.jpeg" /></a></div>), <em>Pyrobaculum aerophilum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i42.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i42.jpeg?Expires=2147483647&amp;Signature=f4RWTuZo6SUXVBa5Vo-CT9xP3IXng9naxxsNPSzybI9nJNfkIfXjphHz9xYAQHZaX1qboNZixBbdQNa6XkHm274s2eBlkj7zMgwQ1tfm9sSvk9VFLR9rcFXwBVT2eQoP-0SPE5Gr-Ob3fuNUO-ozuD~ufeFN5hoZzvsMbrvmawNS4fUDpu6qjH2nkKhwhqgghdroB9tekuXHrpXL4pYmp3hGK1B3~eloZlwvUxZK2NLubQHyg-X01qo2Y5FRo2IPxN3IkOUk3uQHZ97eN0YM~Bxg5mK9RWBFZWddXk5chX-r4ovRL4SiXljk8p-RvkY0wQTiOX2SOPtCMQzg6RnOnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i42.jpeg" /></a></div>), <em>Pyrococcus furiosus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i43.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i43.jpeg?Expires=2147483647&amp;Signature=z4yeKobCH6Bs5g6ft7cpkql1iM1lYWFtci5Nmys97ncDdED-7UvMlYhd9ewwsV91J2QBogFaevjUBQNrSP5UBOp0y8aMQIB1jCPIVBgB7TMd00AkxyHvbgv3UiCfxnaSxif3T3DybxEyAT-yanZz5NH0dBa~Qu6ioJhOgMlbYQNlQ-Lo6cukqYwcUxcvND7YW-Yoxg033JIfDCUea5EW-ZpwkPbDtVmxNaZk91fuU-hQx5hKbdjlDdYlN9ijOcZQb1WC~cuzcgmUI-6Btx0jLEsKF5OqPFGD5d9YQH8hAdtetK0zRPRlzIv3bUUxLNSTUAkjdQdT7H8vrnT6x5mh5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i43.jpeg" /></a></div>), <em>Pyrococcus horikoshii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i44.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i44.jpeg?Expires=2147483647&amp;Signature=WZlIhl6laO0am4eGWIR0wQyS3p988ri~cFfcQmNrY6Ro2-vu-zIvIDL7WO8StIiPsanDL08LFavc6v~vQxuaC0CEPtdwz165j2YnO2XWVoGI~GytMaOhCBJHv-vMcmvsB23zXoahEMLCWMT9i9yqkForjdCs-rH4iNeZkDyNYCKlZybOiez4D2eGarDnWt-TPhZMPRF-PXllgyzE4kfIEHfgftNe6KbkzT-VwywqgqxTS0YHfHCy9NIOr63x8TDW13vjmh9sDzclxJhWiNowbW50RQr3tuP5k397FDUBQyLTi9HLhnpSzlUuigyJMV31jf8SD2A7Y25bXs0v4rAbKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i44.jpeg" /></a></div>), <em>Pasteurella multocida</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i45.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i45.jpeg?Expires=2147483647&amp;Signature=jaMp8yLjdVrwwJQIS7ibPIXq8tE1sQiOyJ6Ci17cnAJINMiS~Ra4nZOEWJLWJ0SR3Urw-6gMQllV2LSwvh6kqzRQmvt82ofTrlfVaHXPT51ZKXbxcrqeLmV7HlNuER~KUPryxfdUwSY9wfKdlCHIKoCcDsSV1PJ~kP8oBbeEPlNdU9XAD1OfmfMtwSBq~iYdc6IapK~eRiaX7Msm6P2EFGhvlWRXp1YZlxpeQ-r2fZFi32Nbw7ijWr8GdQPDfCz4Z6RhrZJ9Eg9eiKa5w04K0-f79EEEwfPxJclDUFVfxhxkBZFT8PalFP-Mg2Cx-9uAFyIZe77PuwtqKkMrFenr3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i45.jpeg" /></a></div>), <em>Rickettsia conorii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i46.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i46.jpeg?Expires=2147483647&amp;Signature=LF2yVISWGQ7PxemkxcbZElXssQvPlWa-RDbPlooppb6D7RCCAWokKmm~r9Pwu6gwv330ONCAS6QtC9U~8JZBYY4xVgiu8CY2UTJLusAO~okvx4RuO8vPYuDDreHsZjshi~DYsQ5ePm6-71QCcPx0MGSI3DToahjm-2TXNRZlcIEeS9JINs9UWw0S5Fm9~ud8Ws3b18rRauCpHMqKVy8-kdOso2bGfxo0aPAv3Yd3Rxz8VFvf1NAGAjCoWQNRtAW5mOQc4UgQgUP4N1f3X3ygL0lfHH3YsB0EDcrn6ozLEBUQhCkP6Yi1RCpK8Gz8zNuIY4aZl1CRRuGhGVJtm8X1Qw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i46.jpeg" /></a></div>), <em>Rickettsia prowazekii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i47.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i47.jpeg?Expires=2147483647&amp;Signature=ZQFKUVJGQfo26521~ygP-zcTLLfsVawv6jgt5n9W8rCh9EX8oHX8yJzYgp~DFTeVwx-rG4YZvFBWoqZzRcYexdhJKDAbksY0xC7zAx7NRILi7qNWWMjY3f7rYBJG6dc4md-MeDTFBMdHY1MdhumESpAZX70~EFbk6nzbXQ1aIbnkWAhB8w-1FwgHRWC5B5NW6ooewheiyFy2rz9u4ZugMLjA2lcZ36ZOZAxFE-5xDSTO3V980SaNpgt20tvmmg~GfiHP6Q8TQATbzsD6dLf4QTUt26QiY1~qzMMd2VcnoaS6qaDqIZb5aQVd07hDrAuIOVLF-dMFxrQ17lhrW5-laA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i47.jpeg" /></a></div>), <em>Ralstonia solanacearum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i48.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i48.jpeg?Expires=2147483647&amp;Signature=erUqPDJwiwByjFbUo8ImW5j3qp2hELL5K9IhjnkKJgTLaBMXdee18nq6Treo7FV8Rzno3viEgon2Fc2k3bNQUqhjvWjsInZo-r0w3Bq~mYN7A0Mn~LiOEdTfF57X3Zab0t24tyVqHzBal9oAVQNyVXaezMsq1kRRNnzRNnxpFlMPaOdPaPfVFhqoFct~jzaDhwkrivHQWt-7Q6UaKo0Yh-LJnfFuNPm~is8PrDyYVJEByjL6vRCXuY89RbQ6Q-riNi8ZLid1Q-vaZMBFge2VsW2nM5vAZWx7rny85TII5ymMV5mvKA7h74w0mQyrhm9XApiYPdJZxbsCe0PJQD8Yxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i48.jpeg" /></a></div>), <em>Streptococcus agalactiae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i49.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i49.jpeg?Expires=2147483647&amp;Signature=a~BnxfFzjgf-PPVZlm2h5ofR-KPJ3OvICDV~F2LqK9JDO~0e4~x0r9LE-D7rLeW446CcD9CYY6MtJrDGtE5WTvVT2ZmjtYILN6L9QtZXBQ84B-PHyqmP6rKvpCDiRl6kiU59D6IAuWFcASHDKcLKQwPaFm701WfkxvJ9FaDr7~ndGjBuqFEZE1ceyGXY4A9uLp99dMZ~lNrnXxT5lVVkh4CWVV4Vt8hhlz8SvLtaNydkwRMI8wC0waPFG8RsEiVX2Rmff0m4G8tDM0J4lom-LRv2C0oWlGG-A41GpuhI2hYizhwHbSLa~2n8bVSo~aI4B6lqaNbX0LpkkN1IbRWTSg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i49.jpeg" /></a></div>), <em>Staphylococcus aureus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i50.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i50.jpeg?Expires=2147483647&amp;Signature=wkt96cdqeaxoe8K5p36h9eDB7MpGUxVBQJb1KB02ys0zmqWm31PuzAie8U4o1eCdU5Bf8Ci9mvHa9wksxnc6BcafFpRhIXefco~jUBf364i0GfogOc7iBVIJLBN~SqOmteeArc6SYvMlTKk4LqZF347YGYjXDYz2IqPGxj56vRKBszQAxQIXy1tTopLLnK~QfDI~d0ZwyWOLnRnZv5kd9N3KYRNnWI2oBl4~MxbbKmZ~s-u1HWBnG~E5Nt-GbuNA9QDOAiN3hgTF446a0mCgTsrRht7AnMD4akS05I52d1~MNGqh0b5KDFptFvQQu3POLvIC0MpZflayAt8MjGQyoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i50.jpeg" /></a></div>), <em>Streptomyces coelicolor</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i51.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i51.jpeg?Expires=2147483647&amp;Signature=z1OjCzfHC8ePBNK3BSLtsF5UFFA4zNvgPy2MPFSfaJaRCJEb03bQBLM0qNJWj8XEcpV7lGapu5EKNfWPPlHoG85qm8ufIgE0IzhM-JKFIqrpxwJke0QE-AChKOREySZzWIbWWbD4JnVL4FpYN0zwkL0pt~IpkOSp73s1Q8k57Zw6L~nH3xKs0UwiUk0ScpQ46WVd7M8BxElGFYpp43AI8jom2YuekGhLTt~JCv04cKvorH-u~ojm1~d4n7Fld24cHkqBkqEWogY0dAWebLH-~ZasVtuapxQHsMXiU7u8n7ig1ZkZuAPi84~GbTZzGZPfdxPmWh-StCljc9cZBKIMQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i51.jpeg" /></a></div>), <em>Sinorhizobium meliloti</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i52.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i52.jpeg?Expires=2147483647&amp;Signature=H7EpRs0fJhe8vNLuABFWP8rdj4XYDfdNsvvknqgsbK1eJfebOcylLpHHwJRsN4B~p6LnIlvOlkzzZXsdZXuFpII6gkNg6cTfkjWz~UD8fu46avvUH7ZoNz5Q1Zs8HNWlepaBq~7fQ1G7igyDJuH23QCp9qoNmDImGEY9l8-SzX8bWhIhWooH8nL4YwZJdJxmmBPY4TOqIlhRoIAwLbngN-NiZL1aSq0dh3-DnuHnX~3IZQl3YDIWbe91grJJUKXGPilloZ-llHC3Q~VtPfGn9p7xwCwdM0jHDsX4F0gGX0l~VyKSiYP5CNU006qjAtt5je0bburgzXUdu8yjFhA3bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i52.jpeg" /></a></div>), <em>Streptococcus pneumoniae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i53.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i53.jpeg?Expires=2147483647&amp;Signature=jV4GjPR17hFSBnYPCVDuxkBYRL9itNefk7Pi0uOk72zeqJ5E8P8T0tjat-HLmvnucT3wiLklsRjwNzca4QMQsy0yMhwW2UYJ9q-cn2p~SJwmHz2UVEMyOfqQbig4~9pQw7DcHv6UZ~pTOCs1~hqJuRPJqm0Vz45YSbfz6I-s0CV5p7P2y-~MU-ypqQ4oepBngH15ox0rDktFgDuw5tnOG6T~auoATHfwcwx6ERs40Bb-YPXxTZoKXn71s90~JDwXLPfGmb8p6VxiRx3vOHLBVfxYPWcDjkAndOk2mJ4Fed3Dikk8St~pDLNNGPf6E1sj7t-IsFJ3FLr1EYG63AwgNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i53.jpeg" /></a></div>), <em>Sulfolobus solfataricus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i54.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i54.jpeg?Expires=2147483647&amp;Signature=cx-NoAdgi9AAs0mT~ODNm-hPcQxfkOfFqDfEuYyUxjyl95FKRDOrC6uqr2Iyo10iZmD9L3wIQDXnqrBCXjizH9FdArohNYtNTQfwxNd1bBOfjf89Wojri1y87IU5j1s4H6zaxgBKv7U4Zr5fsnIqorgiqDIo5B1Hl0sQCzTm437hm48yBGSTlVNnEg7bD1vNVLC3yI~336tukXbvGY6HZR8F9Jzy8QGd8hw3ZmXD~4pmMS1o1WXP0Y9680TiTLQkGZ4~uQYKfL9a7PDJg4qYL3HJ4s0sLdde9QPyls0wAuKX-7ixSCfpFviQaXafXRDtqr3eXCfNW9ToSmfEMwmoGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i54.jpeg" /></a></div>), <em>Sulfolobus tokodaii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i55.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i55.jpeg?Expires=2147483647&amp;Signature=V9lYsRvsfqXBGiETg~-mhfznfkivNWSQPF8-pm2C5RBwwXWB3-tuqHLCUDJLXik9-B~Vy-Za93fPsWvLz-QMuv1UATN24Cryr4IqPFgF1pp6SR2y8s~rEu-WUOBoRGv9JKTzejxX4rCijiRR9VjXXCSiOgTrfoiWdn1LYHBdbCAqJjzlJ0y5coJs-P29V0PEWIp9Qbd0Ssdw-aqchIcDnzKfuENIs83BXp7N~59xdtX7dRixtdsbDEvzFUCDLZFbMfDlnD4Zy4J0zhNjysuCf5CYhjQNmG0O9E79SfeTcudOkEmBzNssBTrN7aoepaRVxVozMyjDzqRShUkjzn3J0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i55.jpeg" /></a></div>), <em>Salmonella typhimurium</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i56.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i56.jpeg?Expires=2147483647&amp;Signature=GwTXMeVPpsLP50DsSTxzN~65FQn0X~jf~Nff5pdgqkWsfr2NiAYSNlPoap7bksNCGqPpgnQzycI8DW6F5GKYzFNdX1Z6UKF4O774AxveUuGu-tnneOwX8C8y5lfWFv-1RQXQo3-TEDaLDPEkIn9o42xgBj4W0SFNH8veLWAUMavikmni-DHOJSibaMyyGzgsbEv54lxfeeZzTCKGqZiSJwzYoCJTxnz8Z7VBPi2NQiBOfLawAozgzdoMcGo4Lv9Hj51UJasDKxQLzt5eYWXOTP7ap8xKTGoo~k~6qzni7AFwv438gBR9ADcW2exSQPfmtT50YjMukhYCy1JK~i48yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i56.jpeg" /></a></div>), <em>Synechocystis</em> sp. (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i57.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i57.jpeg?Expires=2147483647&amp;Signature=k06rYZptl1jLPHGn8yWnHxTPo~SQh93r56GESOL2CPghK7dRWXRxWmje7j~AERFnCK6z46tz41HEXIhjeGJcCITnTr6VRW7pb3td1OHxLsYemrve-CQFuVrS2vjCNYEJsjqCTKJfl-izFDKP3LNYOERSp4xfQ6BecHzER0tgPehQizHRvgNrDQ6kS5h6GCCDjIbjq10ntob9pc7i6ofKNA74CnRvtT5N22GVlOy7JWAt9hTW19FjVVwxFlOClqV0s27fgIk0KkEqY1YltoahIzxTqfzdhWvgd0rVJ4qdHna8lx54ExXrc4pkE~eJJsvXQMVFVkBNXFKZg~h4DQshDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i57.jpeg" /></a></div>), <em>Thermoplasma acidophilum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i58.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i58.jpeg?Expires=2147483647&amp;Signature=o20gkg6zKtO3x3KezZ7ClDZFnUm-t~QpNUR19-cM0ClkjpUDRaEYRwm1xak2em0KWro9lWKPoFsJ1BxgUCIX2icMWPt4KdYWtpmWMzPGjQsNHS10mJOWN1ba~oE6tO-eyyzGwkc5q8PCUmDMoRnxUPrJ6OCUzBTdlTeBqCuxPLn9C6c~sdMNQgh-5XA5y6XLdLGHBTQ4TssMU--WUEO4gvIYsr1EQL~qEVZvWKmyP0eLAV73Fx2g6arGRS-Nb7tTOtx6l8jJfZvbRt8CQn0HSbcEpDXmAka-M6tVCf~WHqeQ7GTZMER9zPye0OwQDv73VISwDKDgEbJbXCZKm-HBaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i58.jpeg" /></a></div>), <em>Thermotoga maritima</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i59.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i59.jpeg?Expires=2147483647&amp;Signature=YDT4ZPHhKrxyvNtDN5282Toqkxx1gFBr-UIQkKB7BalfXjYsn3opfmmetr~~7TMXKln6xeB~axUPSl3iiVa46UZ1bwpXB~ROafzsgn4gCePPGTO5XKUnTQqnV4YzUIUP4J9Sqcy9kpSjo7~jVEQMiDRJsy6PIOSkNp4WBmSOIvDKUR9LhktkXW9bds2peSXdn~6dd4mJMGwQ77tnDhs0RJAXTVp59T2iiajGA7mFq75lgA1m~nClMNXS1QWrMr72Cpa-9QjQugKyBvGme1Q~bIQiEj98S-CM0ZNm-ap~6NSpmd1U-QhmT7dW2h19ubFXx6PGOU4tySCLB-1BCK~dkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i59.jpeg" /></a></div>), <em>Teponema pallidum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i60.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i60.jpeg?Expires=2147483647&amp;Signature=kNTKPRpf3PVsICUqaKCXwnF0-2O3WEk77rlgHuoJoCv1JqQhU5B1DZoTxrIsjEzVw~gpHDAgfjfiqQlS4yV03KAr2IUXOfTX4v9hndV8njlmkSriwUKeUBMh70XWHj0UIcMnPFLTUeJwiKs94FfI-8rxZJNN89k7Bc60fijcDICP~85u0HsW3ax4sZopo~seQLYj67iEC4JMdEa4UicGneM9QPxRDMPgEXTXR4ndjDS40~4-z22oD9punBcKf4HSmKc0OddLyLm3yvZlwxWasOujsavAU1L-1ROqdoJqvReOQMi5kjYhQv5Gkmp-nkrzTXISTow2WNBu5lPRD~W5fA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i60.jpeg" /></a></div>), <em>Thermoanaerobacter tengcongensis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i61.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i61.jpeg?Expires=2147483647&amp;Signature=B5v5-6pKsWA0XOw9UiG0I3gMzkfqCngycIxtbG3b1iy5OASSalOUzQE0H1pTOzRB~NOauRkJb5kylasjVNIabIjWILJ~DQnQdvOB7pCK6XuqGJxL4gs3qJG7CUumnJtCyfCxGTmkaIUxf-UM9TMDsICKA8F8y4Vz1r-9EwPnYRweJ4CsEzyf5s72DkvdBYuDRbJoHpuyWQDnsle1IxxcrTwbPTHXMKZ1FrAAml~O0bnKzZlYRU4EJMMOAXTvDhEGAAD-Mpzc2J0-75T31NIdQgu-yUYGARE03CKIyOtGNGLdLJHVF2Y1lncfqewFQ-4DiAAY0NO6jo546IrOXrgu1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i61.jpeg" /></a></div>), <em>Thermoplasma volcanium</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i62.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i62.jpeg?Expires=2147483647&amp;Signature=wT763tQZ4fOlSZrN6oIGdi-fJQ1fBWQCJuH2oaHGb-sbchp8qjzTPbAkQEjgLxNrmolG6Ybo7WQY2Ixy1vvjkV-WCwAImPd4wacDwXOREvL3PYEznk8fecYS-SxCdVe5oK83ef8jwRZDk6R0cnEWMXq30DVnkSQf28VcxgiHdXe4mXoXR-MS2kxh1ON8M6iWWZngVMLvoalIEBFCpQGEJ44a62mo22C~SDXFVJ1ulZAQ25TVGYViaRxuCmXolHJK8n7~8tD3XzjsoWz6BO6tJtjlSmPfzYjItvitYrq5sPUiuD5W~6xPK7mD-sMjykNma2NyC1IrVT42yw~UX5bgTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i62.jpeg" /></a></div>), <em>Ureaplasma urealyticum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i63.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i63.jpeg?Expires=2147483647&amp;Signature=wb34flugPjwZiZfW6XPbbAkOE7Og~wnJklaDOW85Cgb32zpP~Be8Am-oKyRsDROLkqchkRUo3i270ZecaamkbIu8u9Ub0HVZ-bXSRRUOozCayd5QD7eNj-EDoIm16ddx6hmUY-mWEl2Q3AKpm8WdaV7Lq7bBot8QWbNT~kbURU6EEVJ5R0fNnMh1C5jFDZp44lB52F-IdulYvMpvJISUMxBBrpsuaOs3svo-~u6IaN-kTMq6p5MtjO6fzaqBejpSlCRhr99Rmp5DIMQbRHi5T3nzlzwfXPk5KRFmQ3JYYQPwEgotCYKkJt0ZILQ318EOU2a651M3Wby9MJwq9am3Fg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i63.jpeg" /></a></div>), <em>Vibrio cholerae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i64.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i64.jpeg?Expires=2147483647&amp;Signature=b4eb-aLgkrI7BOrYxHAJZKAfgup5f9FALe-fwPfIl02059d~ITAIkidmukm~BZe50TNIA8UWYG~m0YKn9UgZWC3dEabtOA9Di2nabXYtLrA5qJDfA94lbyk6coXysT6xA8qJuOpcL3gqfwP3wnEyirN88Ghh-LCmtUk52roV3frMyzuVkHT2msLNNkC9SKVyDgM6-V7StSaIpss~pAR4O0oICkiPy1dKMyZLsYoKUTjHnZrojlNcAyEoarn~oSZuC4V~K5fgQJqzApWgJg6Gg8qIHKp~9k5-u-19qEEBKVlWpf1CPLaYCTvpwLGuSBdAzQUQXJ0ItIzjkNHaOoYX2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i64.jpeg" /></a></div>), <em>Xanthomonas campestris</em> and <em>axonopodis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i65.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i65.jpeg?Expires=2147483647&amp;Signature=Q2MCe7wE4YkYy9Z97MNlnLA-rvpqNNsLmqcPEpfrhNwv2LLy0BgI9bQCgm77YCgfRKCg7XSmIvoY02p8ZSMxYcTi-WfoPs6Mc~PlhQC5MAV225bJO4WthbzkAeHNi2XNXWZlGKGmyoo6W2kU98aqLWoML2-yTc2nQoImGb-YWO2g9K1hqwYixlc2JrmaRLgKOZKSdwFq4OsenkT6Huj4sNwkyxCPWGVBOUd3ILawtDZ~Q8cxurNsqqqkkdvlLv19Br2MG984MGMJMTl4ix8nrnjTYPgCUncbPT1ReWrPAOy97makzNhhd~r-XLDqFBc-JWUrgGa57solfpHZVb5CAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i65.jpeg" /></a></div>), <em>Xylella fastidiosa</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i66.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i66.jpeg?Expires=2147483647&amp;Signature=q1yeB8w917bIXX0ybfgblayEY2JV4m3eWPRW4cZqR-vTG1B~PiQKWevzB5BF7baX4yyaGHaAzbGlB0XoTwjUKEe5570VolPZTSn1Pekt570UpRmTe-qQyPP6gD49YxLBDT2HgnOCkYJHl3fqNYSmO4svuf74S3xi0ALHGnLoQG0kzDihKuIviVQfjaELxW3QvPn32xkyBQloHhzeelp7ikE4YVmmNqYl1i62Ns--8FPbbsoGotsMMpHwyEzkIhVUYeVClJqFWZezJ0rEkrfrlkNBVaoW8umkokVWRCz59~T4Fo8uv2~XGWw8iMCVKEQFL5gxLBCGiHTjK0AjbKHPGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i66.jpeg" /></a></div>), and <em>Yersinia pestis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i67.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i67.jpeg?Expires=2147483647&amp;Signature=EKyIJqbZ2QTU0ESvYYL~4uWy5~iRRvFk8hMduDA2lg2YzgCRwNZ4VA68EhHJoEKOqPD~KPJXwydEeRnsKldzAn1GqZbPTyQxhEBvATGqd0k~PWLU4ey7GE5XD4SxHWvSgwwR4exvfb8pYYB8GWRkIPaF7y9ptFSYNKKjCSeULYBnIHZ7zpVHzRDYHW5UcymvvtcGMsf0GIzQ~xYi-91H5cUA5F0CdBSRKaPveOZOgyDmqkNFkBi0lVgCWdmlKb-Yq-jYsow70D9lP4Az5vOVpYnxjN04RQ0Jv8ewvS-VUSZbAr2wseH1oTNqDGm6IlwAX~2z~n30xc8mO1KnzMmUMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i67.jpeg" /></a></div>). For details of species-specific territories, see Supplementary Figure, on which numbers indicating species names are added. (<strong>B</strong>) Phylogenetic classification into 13 prokaryotic groups on DegeTetra-SOMs. Lattice points that include sequences from more than one phylotype are indicated in black, and those containing sequences from one phylotype are indicated in color as follows: -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i68.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i68.jpeg?Expires=2147483647&amp;Signature=SI9bTx5rRo3XPKq8ZcRIZcbASrKRf~EmR9MQJlFoBlnuPKD0mcPYmDyMmuxLKGpjItn-Rg5uqo-XNu-zuDCjUr2Kq~kdOGIGgeAm-~fgoSvCkxXdJrbSGVQI3L3eKJtqJzmWvx2Pertsb4KWOJGRiEz~ln8KevV52IwCrmGEat7ldfYQep8GrViGEMobe52lgVjgFKHLQMePc-Wt6jiy5uznQy2dewc96bO6ab2l4t6ISi-L3paLnJkJJcwfZe6oNioJ95XohMQW~NuKbV5SfCps8rihfFcJ5pwAo~BO6dC230p-0e~JHAUGj~B4vc7JbioqyrAoqO-FMLWPqCNGdQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i68.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i69.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i69.jpeg?Expires=2147483647&amp;Signature=ttDPqgH8HBCTkkbM89FvLalnRqmSE7eYYgRwMpN~L0ZFVQoVGF-WHA9muGYaa5Fo4THcoWZvamK7v1OXnzsUzse27iH6r7ivQUKAOvST9pRNWVjAf7P0KBU1kyXVOwTbe83zvdjFaE0~5zKJD3Wis8MfBRAKBZXfCmifyFiPnuDznxd1j~AMnq8DZMq49MqRBkk7mlUUZXT3Bvb7SyUb7cciJXg3RoTGc1OvSN5tcKfzaYJJXUO~0dyKIOWvq6CBNkUNEUDN4QJosmCIrZjlAq0y9n6a439WZLVBsrw35vOd68HOcPsT4WH7pDfPzrPObm~IYDgVTiSOOWQbaVax2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i69.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i70.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i70.jpeg?Expires=2147483647&amp;Signature=vgAvIB5YEpMlYf0jCB8~V0ipl1uKYAeI36zDyDPU7bhyzwAH0pfzLxmbQAAnI1bG3FX27UYKbaxOCNJxuBALSz2fWqFRtYO3ydTDw540rat7XNSiA8gQICiA6AcswARn7FeUhCx4MRg91r3zWQQE-h5k3wxHrqzQNBXFUV494yBa9gf9d78uepvxB~jzqlbQ5UZVXailieo-npdXYWlSZrJBaI6kNW7ZioDoW7rcM4uOKtBHzb0hy9ebT8GOp8vTkYhhPDRIUU0cnW7pKdOzOchvj~A2vEKnuJyd28E0rtDCit~bszXoYryfOM6eVia4J5Htr6puYfCc52RMRDgs6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i70.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i71.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i71.jpeg?Expires=2147483647&amp;Signature=AHcmpkYgrwsDzWYacRS732lNI~j9zMRDUfrWuWm25VIQqm7sDwNOTGzSWnhZeuMoErCyX7I~ropeHnM~-MBvzZ7RQxwOE9JGfTsUW565Mxy1K334OEvQu59HkFMTmMkcMSv~LUno2ke2LnpEAa2wOSAVcMs2J4BGUcW9xYzDRoWUOGZYGcczQHN97L6pq4JBDDdA6JGwlgcxjKFEGFThRCiBanBZGXUpcqD53TuJ-hDzZJC9-dE7vISaOaW64wGC2ybgEXT4KxK0CXDA6s6phtht45lCk4une25F0pnKMF4uJ4rV05xv35WFT587~8s56OzzNvIwjOQwGYuIh8ybEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i71.jpeg" /></a></div>), Actinobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i72.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i72.jpeg?Expires=2147483647&amp;Signature=VdVY6OFhMArIVdbkZW0vq9AaIC42aXgr6iZoovSCJuGUhIFEYVmqIzHW5KHDXraewljgnP9NxJjX8y-s9xKck2bdP4Y4Zdt-dkZbcRuSLXGkTYTK~CJeyxNbc2WdO7zDSFCAtj90zyG6lGx3Py6Ox-8LGGdAk3CKylroR0p~a9qD70WJAQKbcZp3P-FM77bLGDF83d0tsHNW1Q5n86NjXiSR2s6uhLYADPE85BrvQkL0qSY7iGLbp0sCezB6tcgsJ3eYoCNTEJW~rhf4wGBnYUjiyDokAW5ZZBIK4KFbyCeX54aEWbk3z0matu9YmwwKjv2yDF2mO7IcQiejVrjKJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i72.jpeg" /></a></div>), Archaea (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i73.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i73.jpeg?Expires=2147483647&amp;Signature=A4yLsxkM6FD3ag0l6WbyX9iLmk1UgcmqsEZtzhUVwt10KeaHCGSDhQJ6W2Hkk7gruG4PT6jKu0b0KO2nvcn-QtTQTiySwhMdR47efAheRnFIBrFDDH3Pb2nHsrCO21V0hI6ZQeA8-CtEDAGq0on~2JCqrvJIAU~BMz3mrBdxwcCDqe0kIGp~~eoU1UkUwKBCU6W1HLMMJG-AOj-cUUwFf26H4dwUsG5I5DIkF4aXNTbFjdJ1sAny2s8StQVeiFCWwssVi8qfvHDK3BYo1dPhnlYmCmsZTgJQ016XP9yh0v9iDcZMskm-uYIY7-EzEIbAu3HFbzBidOSKaqPT2JMtMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i73.jpeg" /></a></div>), Chlamydia (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i74.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i74.jpeg?Expires=2147483647&amp;Signature=a32A~ptTQio6k1C2p698eoxCSbJM08h7YW6VqBcy0Bn7ISMJR7kGIu6Jg1vXBQZ-giBmsQSXhDKmx98QhY9I-QaLwMJqbF39UXCLmJEAV7prK27wRcv9jcLwomoylaDHfoMZ~N05RxN3MSFGFjxE8gXlUrNts9elosULomB6KI1OJveOLqdCy6q8r2SWWyJRD0kEHV~apGgeF43ujB1sUs6eAARrGS9IAflrb6sJE04TcD9KNX8NQYqIXAU8TooNNAiHfCplqCpA-RqN7Wi~Idm-FLxoGOhvjXMLe~cLYBSMkBK6eC7JZmTbzxTRdKcc-Qu-n~OfJrHGJW33H-Y85Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i74.jpeg" /></a></div>), Cyanobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i75.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i75.jpeg?Expires=2147483647&amp;Signature=KwqGHgKi-Njmc7SDW9Vp99r-X44Yi6VMCo1BvtqAMtu1Ts4bHqZ5uIKyk-~Oy2OuEXMbxOraK82PF4oHidoSg6Y~IoFBq~~h~hj66LZBvJiGEQ0onusEpx6hQBGOwjCW9u12MYIguBp3jmPRI70oXuBgi4aWuHKbeEhR~1qf-pJro0FWk2RhX9rvBZXq5DM8ENxL~PmXQtgYRdi2mLyIUas11uFprUXVWAgMePPI5NXsYAp2owXKpYyjfSQWHFMBSdOUGmlBIb8qYot7SkXdzQGAV40qtSpAfw0HwceD5em84JIKQq8~Bj010cJuf2IZPwgfO46~WF5aFuPA4GyqgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i75.jpeg" /></a></div>), Deinococcus-Thermus (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i76.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i76.jpeg?Expires=2147483647&amp;Signature=x0Zvpwlmr-EP54ykcF17yeqmoxEeywZso1sV3LNx2TT3K~rOqIJUafdWEqO3uR~pii82xO9A-7j3FeW7C1V5DR2xsjUcRhckmYEYvcpsjt1aY9ocTo912aC0BddJVRQKltJsazgaG~EawjvqHJnhxWhy93g-iOusfAcY3RBDWzxLQmc3nlsqkcstDXlXS84z0koFLf6JWKSnB7n7Sa9rGOPcKBhuzPMTFSOdO4mx7fFHARbwkoAnXU3S9wbofZJZPBQUoKqhjtLhS9OD24T89lmBpX9ESelfdSY3lwaN9dPHDJBkXgJfaWeww2EgEeXIO3VDQ9HQUWnWSC7ex-1bJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i76.jpeg" /></a></div>), Firmicutes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i77.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i77.jpeg?Expires=2147483647&amp;Signature=K5BJPg6V5cMeBndd5oigruvRxkDdFQsApJTce2KXz6ZweSRql7OFjcjiaGZd546NDOWhZR1-G-nEf4A8qjBLwcBVcoCjJdpz97BwfOTn673Yvsgf0xYmsNyLc22R4od8UKIG~V4mtY5jqJgAr90AVCfHxq8f33EOzH3PRHYlUCMnl0fUzhzVLPHFrUD6d2wyWUP~eAkGIv0TtPfGfnEuCmggmSn9UV5nHdgkNBCR~rJHpc9mS-n3B8cFOdn1S~GX5yzg57rW7giZdq1wcQahVgCRv7dVLgvAHj8ULQak56CDvNDx~PdHQG1FbWsr5QeM5s9pAWcND-FZyrp9s3qXfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i77.jpeg" /></a></div>), Fusobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i78.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i78.jpeg?Expires=2147483647&amp;Signature=ZrPbX95vuEw-LUs-45rwSUbdNjk39UwsQpup4CoPpWAmHFuC-oBDhDkKfsAzHxkP1DhB3HTbiVfmbkEarqnK3NAlAxfApeyRZI3GpCbHO~N-fSieuA0bS4rcXaWNNa0ZMAWs77U6L0kdNte7A0lyV5UyXWpi7LNvY1Yz6wnKp-UHaQL4B6faApEcv5VZw-n0JQkv6hXSV~fwfWFhIZh5iY7ule7wfJWAnc2y6eAzOOzD~abvLG1J3hzdKSZmCO02o7MbylFaW~28djUSDvggxLoSirGQ8smmiotMi-2r8FXEHM07vWdDf6d5R5kHOeZg5QEiU0Cycx-rK9LIeYcfqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i78.jpeg" /></a></div>), Spirochaetales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i79.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i79.jpeg?Expires=2147483647&amp;Signature=G5mUGqRjd8bpeLxp2Utwn~Z42gWke0XK7H67Ib-uVL3iWyGxSJogaWQwyDM~kCO3s7-1rLySAvXBppa7KwDkQbfB5fJjIrgRd0Tpqf~~--4WpRugAGdO254uRFoRA-AGt5pP~O3X6orxna-EQMja0kce87zA7IjorZeAS0EhxQm6wXxdVwFqaZ15iEdIjdSfT0wqWdkB~nobr1heLOLLI4rKxLa6sDIhy37iEIVxsW-W2cic4Qfbkkl3MOqHzGLSxSw2zAcDhQvYcEGWy8YMrBJLdulz0AUJtvtVycmBFI-WwLfKDuTubSI3emdj9R6eka0oOKGZ4pGMxVaH80aYPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i79.jpeg" /></a></div>) and Thermotogales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i80.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i80.jpeg?Expires=2147483647&amp;Signature=a1zRMrEZ6sBx0PVyikjPEl2CXGsRO9V7HI3baPCn4Y9O-okgY-jXWT-PPhW~-YkGk0FY3~P0qMXKJns6Xes90puIKI--qccsglRXfK10HxaWkMyfxvfxOS~myLGJq-x~R6tF5zjy4aL6I3l8IbGxIMYC7SeSc5-F0o2dOGWXq~hkYqJ3ZwP6LCyxol-XdqxFDCrCEx1VW03jzc0K1h-1U0vbgJPDNk-FhtJP5c7MOrc~tuNEGtNbYko60q8Z29FB093ZAdIhX0QpaJ8diz3ExqDvGrE4fhXtPmioaHvleMqfbdcQpaD8lOx5DGzf3ZynFnFEeRduhagExU4maTFd9g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i80.jpeg" /></a></div>).</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f1.jpeg?Expires=2147483647&amp;Signature=kpag-mXPY~Dg0i2yGDYbZ1MbEuupI9MKgEA1J5ZI6aabtGX34ghCE1loAAphOokz-5CD8oMmRN8ajBPOLe8bVDYjR0os1w9-13M7doM8Ho6XAQGbKwt2ZOJkky1VLo4qpbDH~8VbxcZW5v4Uu1wLw9scIvLhcW1IUT4RfFB62XyzIYRxbp6q560FbqrxOs~O4RDA4l4QngD-dk8hKzVzStkKM4PWY-ceNY9aANev376EnoisCJ4jR~FSavE~lHDjbNSD~ROiLORvBzHp5AAPLNNU0JkM8SsuFZe~iU6XGf~3vvN0-oaBpGrX1vB35M689HgFS2QRLn~fIw-9WCU7UA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SOMs for non-overlapping 1 kb and overlapping 5 kb sequences of 81 prokaryotic genomes. (A) Tri- and tetranucleotide SOMs. Lattice points that include sequences from more than one species are indicated in black, and those containing sequences from a single species are indicated in color as follows: Aquifex aeolicus (), Archaeoglobus fulgidus (), Aeropyrum pernix (), Agrobacterium tumefaciens (), Borrelia burgdorferi (), Bacillus halodurans (), Brucella melitensis (), Bacillus subtilis (), Buchnera sp. (), Clostridium acetobutylicum (), Caulobacter crescentus (), Campylobacter jejuni (), Chlamydia muridarum (), Clostridium perfringens (), Chlamydophila pneumoniae (), Chlamydia trachomatis (), Deinococcus radiodurans (), Escherichia coli (), Fusobacterium nucleatum (), Halobacterium sp. (), Haemophilus influenzae (), Helicobacter pylori (), Lactococcus lactis (), Listeria monocytogenes and innocua (), Methanosarcina acetivorans (), Mycoplasma genitalium (), Methanococcus jannaschii (), Methanopyrus kandleri (), Mycobacterium leprae (), Mesorhizobium loti (), Mycoplasma pneumoniae (), Mycoplasma pulmonis (), Methanobacterium thermoautotrophicum (), Mycobacterium tuberculosis (), Neisseria meningitidis (), Pyrococcus abyssi (), Pseudomonas aeruginosa (), Pyrobaculum aerophilum (), Pyrococcus furiosus (), Pyrococcus horikoshii (), Pasteurella multocida (), Rickettsia conorii (), Rickettsia prowazekii (), Ralstonia solanacearum (), Streptococcus agalactiae (), Staphylococcus aureus (), Streptomyces coelicolor (), Sinorhizobium meliloti (), Streptococcus pneumoniae (), Sulfolobus solfataricus (), Sulfolobus tokodaii (), Salmonella typhimurium (), Synechocystis sp. (), Thermoplasma acidophilum (), Thermotoga maritima (), Teponema pallidum (), Thermoanaerobacter tengcongensis (), Thermoplasma volcanium (), Ureaplasma urealyticum (), Vibrio cholerae (), Xanthomonas campestris and axonopodis (), Xylella fastidiosa (), and Yersinia pestis (). For details of species-specific territories, see Supplementary Figure, on which numbers indicating species names are added. (B) Phylogenetic classification into 13 prokaryotic groups on DegeTetra-SOMs. Lattice points that include sequences from more than one phylotype are indicated in black, and those containing sequences from one phylotype are indicated in color as follows: -proteobacteria (), -proteobacteria (), -proteobacteria (), -proteobacteria (), Actinobacteria (), Archaea (), Chlamydia (), Cyanobacteria (), Deinococcus-Thermus (), Firmicutes (), Fusobacteria (), Spirochaetales () and Thermotogales ()." path-from-xml="dsi015f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193365/dsi015f1.jpeg" path-from-xml="dsi015f1.jpeg" target="_blank">View large</a><a section="87193365" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f1.jpeg?Expires=2147483647&amp;Signature=JHQSTrwzm1Nof~MtbtDolP~MOt3~mApuwn9q1ME~nt5gbGkSLhKxoEPPP5VLGKC~gmTE0JeyntqtHE4pxXqJKWRYu06txoSukInD2Fzx7lApdq9IW2sQIXhrIvjogUiPiePyXHsZiwVBIwxrLNx2qPDIq5baRbNwHBibR0zbolF1H1T6Fk66J5S3luDxQBDla39OUKOseHTgIlsTiR6AGoNtrlCoZuKxuFKulgDTraf6GiTHJSqiHrql-JaQItkw7WMcstfbK2cNnnVJN8ATgHnFEiiAsq-RbigjLeZse-VVtXtwiqYH8HgmkX9DwWnAY6OI0tE85YRoBQUlE2L56Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193365&amp;ar=350803&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi015f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SOMs for non-overlapping 1 kb and overlapping 5 kb sequences of 81 prokaryotic genomes. (<strong>A</strong>) Tri- and tetranucleotide SOMs. Lattice points that include sequences from more than one species are indicated in black, and those containing sequences from a single species are indicated in color as follows: <em>Aquifex aeolicus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i4.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i4.jpeg?Expires=2147483647&amp;Signature=xOH-UwkA3CzBeW5AOqGuKlMPsf0NI~6aEaXzIuTDq5f9KbADQdVdm5me5HTYLOXF-tIK~eFwHg6154M0X2qcLz6ojIQqILGPg3BYdHMap6I5EzYxOeK2jJFG4izosyM4Y4~aoJpGaglB8RQHoxTdVbKZus0vm7rhycSH-FJpXi5DzgVD-DIqR1rMnJJsvOX4eYJGex~Z2himd5577jXQjDDSlA56rnmXkGkJkcWFeQtQE73SJZjMgF9GFXz37k7G2jqLw7hRB1muzZCxOdZt3Y5ZZ0VfAvTIQ-ki22WSsyi0hbKSEE8MLe68tOv7dheI7PR7F8hnYXyRGwCgA17BJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i4.jpeg" /></a></div>), <em>Archaeoglobus fulgidus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i6.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i6.jpeg?Expires=2147483647&amp;Signature=c2yw9rqM0fNCOsavaKhCuZ6f3C2mHrLo2j1yMms80LQBlpgtGldtLfyYdTo424aFGZxtgx7AHKow2Szb3JbbaIVijq4XD7eRAzL0zR-ye4dogLGAE88eTln1CB8QsdnSk0Y446uQ74ihr~doL7j18Y2fXCLqd7R3wOT8k14hg7rAXoWveFFRbrY8hu9ZgXirMD2~YUfcbcv7tmriOOw8Tc75hWLmvbQzvR0zywy1ZwX7HqsbGEPjJOuK3cayCgwwxobT7GNFnGQJNkVxTPpLE9fLLSSeygBeFdXR4vtk6c1dSugQMkcKu316VG5biBZOd~r63htMG7aZkkt3-axt-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i6.jpeg" /></a></div>), <em>Aeropyrum pernix</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i7.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i7.jpeg?Expires=2147483647&amp;Signature=0ZRNxwAhm~3BzUibCVLnqvTFjGVbmnVBJ0g3x6mGaYKRliT0NS-Zwh~ohok0cdZDwLArP7ItNavfQD8IQ8R3v6dBtP40p8UnzV1iwYjkGbZuBTah1czcGs7USP7kE6PgcfrVWAloIDXVDLMicWA26vCY4FoidTqzZaMMTkUqKfyuwt19ZPUl5zxZFOHWCzVa2qZdg-m5aMOy2GLrytyaXfXW7L1zr7BQIdTKnYPbb6oRmmnK6fXoY8uw2fiRo~4SUP8fU-tIA7CKqPwImgHWr1ssAH-Epr8y~T-QhxI9paDDexsr2JxfBboNFMaOj3jfqpgrO2MNZ-ievRUTdSWing__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i7.jpeg" /></a></div>), <em>Agrobacterium tumefaciens</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i8.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i8.jpeg?Expires=2147483647&amp;Signature=lY6T8S22V0W79mvDr1Tnta2hWlZxb4x047QcbgQiEPc5TJbK2dewRT947nXyuDc9EPQCVlaicn79vd2hDoIuGxgKiaJ5X~-3dCJvWSpx5J5LknwyyC~GNfqEAd4l4gz3aK-Nbo9zGy-gA3BcQW3pNND2qXOqHnNQsJs3ff-vkAVlH0Rb6v1JPKiFq5rnmTXK1T-PIUe~4O7sY0G5laoMzPiM4Ny1LCQ0zgITLD8pgLFDtRQqSo2efFbCdalzwTpLg3zwGC6vyhlim3SoBvrajNVLitDuy3n0Ht2foC522uQ1zSNSOXk0NOVg~fpON4T6q2hkrvVqw0SnfcVQ4ztdmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i8.jpeg" /></a></div>), <em>Borrelia burgdorferi</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i9.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i9.jpeg?Expires=2147483647&amp;Signature=G2m5KrkMhjQTizqIqhrYoyWxBLZoq223wDED4IqF9~Ozdk4GV-Dst-4AYbpKyY1ecS5ot8blloXTUsnsm5dDjAa22rIQ6mrm8USgfSFiE03tFf~wv96XjPHaSnwZSCifsIoytg5cdut781vfNsARR4Z1ebpDq7TAhatWDU2qAyVBPM0OENibmeuL3-GrvWenHXc9aK1IFzM~JXT2KQ6cZkGpBAQFud-ksehVGQ-blRLcjmHgU8Iu~-9wXG3yrN-cZdgVzBKdR76uqR6G415lur-HaSWHXxGjKnOGdCLw3zgOxgXiMjIZNJuJrpJGdt5FmHzLf7pgp5yP1eHQ05yveQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i9.jpeg" /></a></div>), <em>Bacillus halodurans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i10.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i10.jpeg?Expires=2147483647&amp;Signature=V1ruz3jNOUAR4GnKF4n-PsqGgaEVONybH4wU6dqegJTRkO1drEZqfbOsWGN5I6oWk6A8oo2z~gOZWXXyxsiNqA-ZOUELsEam-v5dGvHh-uHIeX8--U5sRIzWB0RKMMr6T~2q2i4Wr61sP~JSItpMngQC~wt1psnbfYsO8naDVD~xJlRxAmCm6eoVj0j~6KgGnH824fFDT8w7ocMvVbmGF9FaHd7hyJTLqb3nipsbHkzeRhPLsuCr6fEbmtS4la34L3lfVkcBa5xuv9pHBfbdboKMxDbbWqe88zL4I-befzEhAL-QTIrhCQKE2QAHphA8OZnQrwRwIFfOxmE5NcVNyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i10.jpeg" /></a></div>), <em>Brucella melitensis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i11.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i11.jpeg?Expires=2147483647&amp;Signature=WODqdQ-VLqMFL15Wf-2b6wurLM~dqor72nnhKqtC7S7UIExlhuTGpqAulQLuNwNUHtfF6AbqKr-mlPrtZuk3DosiHWYLd0oHM3Yr3F0DxnzC8X5CPs~FNkIIIX6UYGYL6XD6X1KegSb3fdMzySGAEhD04B2APezML8e8OdXzFYbDfg416CVnZZAByPoWkQqJGnfutw5mMVGFMoenh~lPuA7-sBRnIWmiy1L1G2vgihPBAyAxrTUEum09Skm2Agz7mchIQtlr9iROGkh9vHrFNDFqRjMpQPpY2FTpowhEm00MTijBSeClmQ1-Jmt5WjsEhxrEvFas4uNmsZulLh4pMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i11.jpeg" /></a></div>), <em>Bacillus subtilis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i12.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i12.jpeg?Expires=2147483647&amp;Signature=S75jRXXUVYI~rPZLjG-yaNWEjk4QVHAepGbJvmq0wqEKYMNfC4mlMNNyO9WKfN3Yx~2WooyUxKTu9dEC6cDCVznXNksiSavr4xpQJEGc~79uoE37ySiUY84G1EDhqLeMPUnlMmhOejtMZZmepD2SgjB462ypypX3~4cfF4-6Lk~o4fzsOcYaDEpmOnIsaopHGzVpDj2bR~ydorzvKDu3RRQeSvQr9BE-CdeB5GuZkmiZlwH2mD4eAL4Ze1T6JH3wWmel9bOd5jeSDDmcwnbFDNq6YMvoowmB0gPdcyY~YBG8ismcpG9pv9Uckdf82yvlEpRJwjRfeZDYPq9xaNNp1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i12.jpeg" /></a></div>), <em>Buchnera</em> sp. (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i13.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i13.jpeg?Expires=2147483647&amp;Signature=rfibgUoEkEBOmM7Mr9JH0rlIozjRwi863UBrxm4IrdGirfFDDqCXnCxeepVC9LCvhww2CWK1lRVibHbriX0dK1Dzl-VPid8kLs68fLIVU6k1CZ4tIg4~RFSbmGwvfQecvdTsssQTMPNUP~RiYPWdS5aj5~Gn8DWTcVVEQk~nkE3FDYQe2bi1ZNqcUTInN0Ib6BCpEVNGeTCAPIMbjZCn7pz-pr0bK62nDMu9EzNw0-p7iUUhJfgv~~N9JsMOJAZnlZDdNoiKif7PbZhpsabJFKJMM6VO9cbC2YBF49BlrbbzrjKDmcAOuwN1i67xx3g9Ld6~kvyI~CCVPFm3YevLBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i13.jpeg" /></a></div>), <em>Clostridium acetobutylicum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i14.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i14.jpeg?Expires=2147483647&amp;Signature=NyDYBy1r2VGMnJdWQRxPj5vpwIFCYG4SEtHhx3q5hsNQuIScgugd2J5PfZCK3Nwroa6FIgEAsAKlP42XUGioa8~VVtc4aIftZUYeVXDoPUPHuegf0fMt2IUdueP4EqFEHytmDayJc0udBCdTiLvax8NZwnTlqqwzBTBeB-eab~lRGhNBD8Vu1HnQ~L3Hn8mcZ~q2g7Y0LMkqtT08Clq4O-WKoT9YAg0wIKJuU--h7U9gNu01E~X4yvgW6SwyZLqdSS6BGXUGGF14UNMVUo4WKHb-c3iTQGDrzte~mi1nv9l5TvWPd1gB2iSPn4lAGflswJVenMCyXUP2w1n40jadNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i14.jpeg" /></a></div>), <em>Caulobacter crescentus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i15.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i15.jpeg?Expires=2147483647&amp;Signature=l06MYwZ68zk7rFq1Vug4GyccFEgCj44OTbs3dF0--INTe5OnyL2vC0XS1pUe3YvaOnnKyEMkHthx9QOQ5IpUOIXk4OUI8SbH8QYgrZKlfTjp9jMPvYniwIKysyMYjnYqYnPrP9ZKs5UOPrj7mH99~6v4u0Waz9-TfxPAyHEhJiV2fTgjHETTWADkLtbPasXjjboDf7EBpfgoRrr4KGeP40GPbzmb0IY4I5EtLJ5~jo5XZMgRmEuoE81OZA-jJc1-Nx8tyKA~VvxDqZLezCcxaR3WVNyolxK6paZoHWVYD720V2ikCDPvhRRkEGKNsLEHqfmNziZMN3MpPXM93TkMSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i15.jpeg" /></a></div>), <em>Campylobacter jejuni</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i16.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i16.jpeg?Expires=2147483647&amp;Signature=EZX5fHrwfkrk6VQfnGvewyggm4O8WBXgWj33VmS7lpPrn6hcgSpoCi~vrsSdO-fLk48pXspuJcf7L-r4SmW7dSCghyyd6oxzqRLIKHc9Sg2K2NOQQWSpPcKu~MRkm2KNIo8TrDds2XLAslrmvn6vXAYXoamYQbBckgOfEgdav~jhiyHXtN7w7H7~J3VAoUvsU7dgP12Oo5LNhSPjYqdivWoDrPJiBsdjPz8cH5wOCpf9oa63xio053qO~wUGjC~gsJBYRGmt0bjghhZ8tle4CZN5RkFvUSp1JPA9LgUS7MVOSX-3~SMaz2Oa9cQAq33-K-jQpaYZLOs7U-uS3E4Zpg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i16.jpeg" /></a></div>), <em>Chlamydia muridarum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i17.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i17.jpeg?Expires=2147483647&amp;Signature=vou1Fa8iGX9QAg2hqkvcKdJk4t3I2A0etQDBa0-GaH41pPhxkS1FB2s9t703bGyXrL3hUc~Q8gkwTHWxJXxWLLuv1GR7XkG6uFabNQO4wEQYE0QA6eJ2W4Y-7q~QlR-N5UoIeIh1Xg4Ccf0zItIoSPEhBL61fDzkNpBtKRw15DHQnVHmoN9LjEgemw3eIQnH0aS-AUX5aQDFdaWlwiPJ76Oez3s~Rr63txcv-kwql3EfklWW-OK-HbNaFKKFOtHuzXVInsNOnu5a3VVHqiAfbRuZs0YuvGyBzLi7N2JPxb0kbX9j0lmGWKr7FRmYf1cGrTHHmO6NObitkIyWXEZj6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i17.jpeg" /></a></div>), <em>Clostridium perfringens</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i18.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i18.jpeg?Expires=2147483647&amp;Signature=AtexqjDu-ILh1yHG1NYaxISKZeD6FfyuY84~S~Pmt5oKH4JXZbffY5DSYfytwuoRuVJxuhUwFjJXjURsUVwZmUQEA~qKunqNFCKwkkPfHa2OvrJGIXWJB-QuefffLiIOq22MESkK-9pk9~RVbDjh3YmT93I2oH~oX2CvhDjhoCuQIzn-gl6emoF7QGZuksTi1yfMCe~EQSfKy0G9FiCTklV0RE5~~7p2ZrCOYIzbVn0-X2KDqZ5gEcVbD7DlDZnA3ChSOJ~KXYo1WsnbZ4MX8Oe~wWwiiciNsjWKkWQfFYcEfsqj~RhnX6~JYgxkoJCZLOlB4l64J5HRzl01qFKZpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i18.jpeg" /></a></div>), <em>Chlamydophila pneumoniae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i19.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i19.jpeg?Expires=2147483647&amp;Signature=yB1ljGdmZDe9qh1oU5KwKmFshcB1vy7vmHfqLOihzzipNPfk0UWhzw8fwNUofaS5b7k2k6buu7A50HrbWhD3ws63MFZ8ZvxX7wbd5c0s1lJjwZQ0rgz3m1FAWTgYVnyD3V5ZJEcknlWbiRMxfiP6l0HVZ9fQOKKFg4jQ9izSjR42REUHoQN3p1~Pr0q76wHfvGpSzIM2ZsysYXXAQEWPgF0XFsz6TlrDauO-MnSt-PbShGSQCZbwFSBVZedcuyKsBj-0GFf9iOjEBYJ8qLvl6DVAuxM9OAhg59hMePhR3AXxBO8SBY2Ej0~qneS5jGCX5Mlh0SGljYm0HAbCH0SSzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i19.jpeg" /></a></div>), <em>Chlamydia trachomatis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i20.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i20.jpeg?Expires=2147483647&amp;Signature=oLgA4gC5jvowGoUOL6j1o1UreE4BFQ78dVq~zAu6-svAjNA9qJMSNOegbn4zLnpNhPdN6pcL9-hCXh4DCvfIKvcjRTMtMhqpCYs8pHDladlDJJiCbNtffapxF9GgkJMZACR9XUt-yNC7DpNP9zKL0L3WxJ-mJ2~AsnNYVBpUd~gMAUjl7ckOIEYWCe4Jm2FLdPEzvrZ-dL0ToP15xI-uSfXyseuae~UcSahrfNsQxGnXfNqmb2Z05oL0pXEdpD3LshClcuJtmmNRHG4~Q1ncBwm1p70aLftTzzpa3sg0xp3hX-KKlUowH25bdlUQWPMfp-Ip2q4ngLuMVA7jk-DiOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i20.jpeg" /></a></div>), <em>Deinococcus radiodurans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i21.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i21.jpeg?Expires=2147483647&amp;Signature=LOuHz2kUQJi5ACaYT1P5ZzNLm5PcMkdstCbJPvMSL7nZZ9a0bLsPvtKVZOaIG0QmKgL3QeGQYRWy3GdiWBmq4-u4avCDfvIzrB6YVK4KCDdPJp-HGENXJ~SWQbqtsKiKFyr-9b4IpcIjZfQaSarHoUSMGCyzQN-xiUIQwE5kia-ARIafdtuGUNAqxNgkNJqRAbdZ7IqLm-LBvzdlgPJcrcnyoSImjVbbPObEgwTW2Am5wLAhCYPTzJyXxrlcjUrop2qbytgxa1bMwOmfL0IYdSaqkl90zQj1k4LLB~ZCKrPThAjAd6MeQ3rULyfkALXYScuGM6aimN7-L3-tOBRhuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i21.jpeg" /></a></div>), <em>Escherichia coli</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i22.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i22.jpeg?Expires=2147483647&amp;Signature=vSOOkxF4GG-6Z9Yd11m~cq3yJxnagVO6McDGIsVrbLj8J8HO33FCnk~C50hBTFj2zhiq~UfiHlKxOCbZgv564U7wdU~7Zi5wLz1GqGrUFtxj7QV70YHrukiVpOXZe2-UcoBvBiKP-8NIVuPkaD2Miz~Y2wiAIff4~qofJdzh3OEGjyJQvEE4d09MQVq4nAqF~-vX71E7OTTd0xnhdPFC69skL~tG5KvGQ4WA~CfuamxWKNCtEAOAN8dgMrEvPdw6wBy2IuCOvaBWlWu663k3l3kTbXXVQxCAILQcFbDvTwYxJiOANufRTkf2MENVxLPHGznjQNRRdT8iqV~wuAJwSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i22.jpeg" /></a></div>), <em>Fusobacterium nucleatum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i23.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i23.jpeg?Expires=2147483647&amp;Signature=elVMkNwaHAuprcOBSSf7TLnNBLv91nUX6ZWmxkOmzds87ZyHHPLvgnX-Ci0vrMhyGsh9hGzcarqLKUpl-lts3JyW6YJsZ733UWbO9j5-SE6CRhY~B2S9Rpplc51f2NkUlQDxxhXdlwWJ4lWbBPa-IOg9pKVtIRiCw8r1aF8y9-vH9K5PmTu4WXsqmaQoXB1L8vvA-uNXqiLnFxZZqlh5OIah~I~xP5L46vhiF-K9fWJNvxF8PluYPED7lKU~PY8V0Q-fwh3wnGSBw8f930qu76k4KsyaVYIdoxYr5cE-3RsZcXE0vDYwD4zjfwTZv7-ylh2n2SL1YEYZZ0TQ0Qmp8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i23.jpeg" /></a></div>), <em>Halobacterium</em> sp. (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i24.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i24.jpeg?Expires=2147483647&amp;Signature=3KysfMZXhbXiyBl-Z4ltsS7TeAT-xYLnDA~rAFPkLFs8Yr3xqjHoPlAk72AqMu1jzaGP5ZF3ejmzAIXc59NLSn5OwKU4E8G1ocAlfCANrrzCgt7qVdhBQeEOvn4iKEwUfpqN2ztiSNkdiNCdm4klsZD-~jqGb9fxRIjnIFDxSHcm3AlvEL6ybF4ja9eF84DA1F~-230JYsks1wfErrAvVwRKuo4pTVVlvYjTaBc8Q1J-byq2dqZPiMYxBqyUo0XXWii9npNvI9tvX1GZQkzvD3iOEnJ1JqTULs5Qo8l9oFzU-YUr4gWe~6ncy5AS5YQw4j-vGrLWbmsazD6J84fUug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i24.jpeg" /></a></div>), <em>Haemophilus influenzae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i25.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i25.jpeg?Expires=2147483647&amp;Signature=SvMlIerWZsYLcry5PYEb~GpT8nxLDvTyA19WH-dCOykXSoj7o6dJG5GTwepdbdyLbULhsBUxxPhKw-8-mIeyIMtjtJBI0r784AuVV9WG-le~Gt2dLxiZrgzYIfF~xf0nOXs8Llt7hy~lKt9GiKnot4PCOd6XE~44YXPTm0YhryI-cGzyxrorpFoXD8kpEDz0GryMeLW6phz94tAdb4XFfUuJ0POThF-X6cfz2KModVJajb6hYfnEeNMs0vq5SZwb0-RuImLBO5n9CTAqoGwnsdx5N5RGiC9rYaC3WBRmpGG0zTq1AUMnQyOI23CCQMaW39VP2UKjaEbqYIqsuhXiYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i25.jpeg" /></a></div>), <em>Helicobacter pylori</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i26.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i26.jpeg?Expires=2147483647&amp;Signature=wUy3w49N~hrGyHDPU7siaCIM30HYfbCBJsDCsIR7NMHSr0R9fwDxfub5X5HU8Gulsuf1Pa1cV3kLz8NF5EmZz~lMHA~RgaQLBVNLC7HPPLrG1pxtSJ-UmCNw4ilwFO~kffO2sdwzjSmgYjFBaflDcuvvP-JOWfjxpcK90njddjpB5zbqj0yeH03sa6nOP6wecdzTcs7~eyY7GdO7eJzposM360DJPUaSOKttMKcshbrgq06AP8roJofWiJx~lXDgOVwkQQyzVynrlHKKduaSqUkJeC53BnbZXZgqXc0kDs05rm7ByqN7CqFo1vpdsRzdPwB6GVcBFqJZo45eqWHYrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i26.jpeg" /></a></div>), <em>Lactococcus lactis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i27.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i27.jpeg?Expires=2147483647&amp;Signature=bZeoJa9Dyyjumw2ekFNi86qS4Qqdee5oBpATewk~TL0BSPnTrFthFYSbKbP4OMtpm2jsCLJFDdlZ8d-wSlZ8ZRRSAanX1e3RhagsYsQLhMBr7Lw9bN147EywP2JU9nsSP64Uq7RbD90jykZjyVclUzShPBg~YFgnicT-zaJmyPzECCVIho--Se9~IPuYLKIa8GHUPCKQ7dryfJvI3OXhR2dlB2rhj8qVwXwzhfCRUF1RL6RbZQN8cqXV5OQCdkYfzx0V~WphcoiHtQYov-0xtf6sB-3k9uabLQb62tM7u36OPnOD35Mz4MW-32Vquhr84XirQQgi~wIVXrRJ5DOJIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i27.jpeg" /></a></div>), <em>Listeria monocytogenes</em> and <em>innocua</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i28.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i28.jpeg?Expires=2147483647&amp;Signature=pKHojGpDSWaCYO2WKenY77caCzrEEkGMiQoZoEXF~ty2ztXs8OsnCAPt-gRDspvtccgNHbMgi3f1X0FmOlsdthsUG-~P72RWYQw52llOTtfofLJoHvSr4r-jX93AQB7qU2ErLfzsCweeXtHiZJLPKy5ubrvPaEM85wOQfWGjCj1UXRT-Iiw8P-mgcLFVzza3w33cTfjCiqOMVZHboD72uLy3TmQu3XxT7odGOUIW~c5AzhU3xDpHa4nyXC70LDy7jt3lvbTrsZ6epRrCmqmxANoEBzeeeSa75vGRnvu6qEbWpXbINoD6Hh6tlnS8oF6a2KCiMiF1umEaL2O6Qdft1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i28.jpeg" /></a></div>), <em>Methanosarcina acetivorans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i29.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i29.jpeg?Expires=2147483647&amp;Signature=gFjD5jV~yyTPzNK41mPLR6G1zuBikV6lh-ddX7jzlJB63i0jiAEtMs4YIS23tHa8FEHWWlALud--I-d7IiABwyTOes-03nSwuKaBnFTf3YB7EnxJ-pvbXLjzrMQV65IsmI8uAgaZvdZcstU3q7OtrvG8EpuTckO65pYLtCWc9ptgdg7-alI8DWqXZJRjxjAoRq8J9U~I~cfJNdxv7HO4OQq-J1dBogwdmAV7ztY1uj0b32pYDfxxDTwAL1Y0S8iHoXgS~yTZCqejs089pA1qtLoA-rnOg3QJpFUrwHAnLp~jz8t3k6e2-8PZKdH8iE38JQ-96XojTgBbKj5McLxWVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i29.jpeg" /></a></div>), <em>Mycoplasma genitalium</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i30.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i30.jpeg?Expires=2147483647&amp;Signature=lJjFu0Vt53kgjRA1fJTaGVdx5ab9dNBf~ObrTB3I3Z9PTYUMzCwwsf6pzTG37oTYrxFgKV4tBGSbeX01kRYZsu7pgtoCnmnZUMAQSdn-JS5JHGQl4pIU2cTk0-2qEUvZGifPN2rNAEvpQIzAHeKrPLe1e77tY~t4DlS04dG-uHajumYZFiy2XG60kSsQZF7XXHb1eoE5aWP-wH7-jahIRL1EHGVtzgRFctOSWJbDJX1ZMru8iIwsL~Cw8mVYW4wGwVQx3-iMTqsozIIl9CLvtqJgevz4LT8t9nCIU176z4BvVGZIaYsSxB2Uspdxo-mo7fT~KNelf4B1DZgjynoi3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i30.jpeg" /></a></div>), <em>Methanococcus jannaschii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i31.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i31.jpeg?Expires=2147483647&amp;Signature=M0~P4wI3vDF9xd9KhoVJ2hZSAGOAw4KsuQjOMU2JS7BvUE-zZxSSEQKK4w-VvaoFZks7NNpiqcAYDAUnprUfSCv3u1RDFopBlafn0PjNYEOdRjyHNJpR2drQrXIJ~KIBSkWIPIwRcwwkHpx-I6DWOEc3MJUK5bQWmbz8xz6W8kbO5dmgOAcEGOezyaX0jkRW4N4fvndAHC~H62ArkCtLRdP8AI0glItLLvpO7tc4zAzwveIMC5Aek2CD8Fig8vg76BV00XHDsKMAImOuBfXx6sU9ODj0Sc4HLL8o41bso09oBZCqQqTWx1OC0AVwXlm~5DNXtgij0HIeQzjywsnHdA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i31.jpeg" /></a></div>), <em>Methanopyrus kandleri</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i32.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i32.jpeg?Expires=2147483647&amp;Signature=Kz8j4YaC7vXUpouVL3Iysocoonsx3m9R31Bl~VxgLmXFMjvarKvmZf~Vsa8imlnZU86XQGr6RVeDYlYE6FE5gHRK-8iTzcC1KtijlgYHvJnQgh46usC9iXQADIjOmyYCIKqBXtO-v06y9G1MMfF9i0Ml00QOdf0BPgKICXuEaBD8KkIjAiFZTmS7lj-eLcpoXyZNva8mmNEFSxPPbAMWqEscqTFQB6fLAgZhEXLbS79kkndYdHCPqovg1lVRonVzv7yfnJXnpjs4lAVZx3RWuedBMYeGLhyL-9Y0j5VQ3SE46hfrTuy9BejN4kH8ozNuE9Uyyq-k97nf6vjHyvKpcA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i32.jpeg" /></a></div>), <em>Mycobacterium leprae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i33.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i33.jpeg?Expires=2147483647&amp;Signature=B-wU4hfcIJBIMkkgsKXr8z11tDODQzTfiZCr3qemUxoFybjAGSKuIXlg72FvI6l~39CDnsbE6KYmGP~8jW8kZlICLUpqzsHUkx2X0uZoXXd8Mzg6kbFsSknzO2kmQa3jKRi7OcDEBl51BjNV23aaDcUVaPHxxQMxGrAFRbqormjttaYOH2ODZxVkXCPqaT9Ftg5UacCUwFUBftFv~JSGC-RicfU8ir0lckIOkFRBfamjvjkMY4NeBomElRiRTdEg3PwgmOZlqeZXKz3evvy4rFVD56NeZq2PuN3-5je-ek50sTUV7Ulq-hLjZN44sjG5Dd~9ifW104OYMamc1XK8NQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i33.jpeg" /></a></div>), <em>Mesorhizobium loti</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i34.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i34.jpeg?Expires=2147483647&amp;Signature=ZOSiwsTGOv9KbXAeR7TcQWjZOOOw0xiDYjiLhkqkR71DRq6yo-nSKyKtNkqQL7PmKCPJcrrhZsrS8012hrIFCyZj6KXKTBTZSQJ1ietVkYW0925IAHZ7uTXA~4bJpimi~jNcbqE7uaOpPExil~Fvz7-gPTE1DG9OVvEwqHtPFEqgumrlVTCKeguJisgdATkkrsjeeiy66tJEMhxXMfTO~H6s~QxmwFMeB2oBd-Fx8Q3PGb1p3ZD-z~oCFmSVTxqOymHm9abb8pFazlHtdUWNTSPee77BFu8LN1TCKpdr5yuv17~bUNFg8Vx0UXsGs9RSnQq0nR8Q0ieMTe5fRfO25A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i34.jpeg" /></a></div>), <em>Mycoplasma pneumoniae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i35.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i35.jpeg?Expires=2147483647&amp;Signature=xPAN6asuQByIYsO8WEnsYRT9ZpLOecOXFhWF~2RPKKaO7Ja3RbQCzvQ345T62Vrt4XMxC5nQ6vVnuIkIt3d~DkF7sJfuYiEnChIF5becDk6Z1n-zo1l8T0vzx-xgPU3ym4t8YL~RES1vEVM9XXBDhr5Kn4lPnOL2GHyW~7-MxqOLHCZV1sz82XVCZ6H9dHJW73EKVmsOGc7~sexKwDpF7OU0JoEsOVOzKdVfVEE8Aac94VTY~BqwGl-Q6My3U6ASS81YuPFcGwyipfzcv3eeiddSqUwL9ww4RoMExsHhGSOH9jVZwvOBk4nqEVLMUHOX2G9~hrNJayMikkluANNb8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i35.jpeg" /></a></div>), <em>Mycoplasma pulmonis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i36.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i36.jpeg?Expires=2147483647&amp;Signature=ZEHZ~Ln4qwvFE6yBdgg-3HrQLMOvWkg4BC69OchogqENlNsLXEllZJBGdXk-ZMp5On7uLg7Ty2JwXK3h9ODbq5NN5Km4im8gFM50c2OWu5XNP39QNM4wCHqVwqIvZWTJn908OU6EJORCZfYMZB20SB5WhXOBnrCpgkn7ieuT58RK~giPLoTb3YK1cWp3jb4Ely3q0ZGEUXfwLRlbZI3jLVuCTR56XfYnyv4yMETCB87cWWj7ou6CBd8HKaKP5yy1VCMNPC8Q21zdVH8Xm5tHI8~Ofvn48pvWaIufW4QMkHaU9Nau7gbx6QkOCtgedJYT~g-MSB1dlDfNRh51mxy6Gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i36.jpeg" /></a></div>), <em>Methanobacterium thermoautotrophicum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i37.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i37.jpeg?Expires=2147483647&amp;Signature=iSS~SrCK0MQ9ULCtXp-Mn6tZhEgR2B3v7DWPfZ2GubZvFIk5vWj8qlb9X9lbTnU7U6zAi6LvNWmJIYJfRM5UHD~kemVASiE8i5hr7kU5sthr0d8XbkFA5slE155wdvN5xnPten2Jy53MnRbQhQ0Bsk~lcFZPi4z1fFSkxRSTXkbZML~vNwtQq7UsuKFF5jRCcoLGOf~h~Meyb00aZnkC7FZbS0yy40WSW2Xp-PJFgFi1AuE2gwqIDGb~mS-zJcY3NI9f51Ys-e9tO8kkKnVeta29P9yRzcWgmOy~e5GBLixZArt91DBGESHH7wNU4urD2bkHylEw-tco65Lw38PQTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i37.jpeg" /></a></div>), <em>Mycobacterium tuberculosis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i38.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i38.jpeg?Expires=2147483647&amp;Signature=OEII-K57UgCZBdJzNn6ZBqCjx4S5m6B5qj-jmkDexZI2Mnu4xtHTNf8VEwNy5J-q0uTqBIdvl8omQMlEhzM3bMcfKHFuyntRC338frwo0zfEChOGNIi-fl00JagnIqzVcoe2a-rIUiKcn1newuou1dAATw7lVYpszIGgvDpq54MI6Ms2pcsuGkisB~N49701Myo0qiuPIZWyuxV93W8~XEwb093VXUSk2zuUoxR2IjCacPrg0iHDsp3TOmru8cecRq9owrDKMJRGMQTCZ8XyomccPnXxqI7heFrpjk6ewtCHduIrbqDG3T2zm-A02kBiilmYo4K21SmwAnVvlr7XJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i38.jpeg" /></a></div>), <em>Neisseria meningitidis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i39.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i39.jpeg?Expires=2147483647&amp;Signature=fEBRlAUFnvDTerPVwPqxR85d5nKPZXcgDpF~CVb1KQFAWgrfNW~NMTB1iqCfK3Fyb-9H~bv7EFzy~28J4YDcXvcpDkXRy0eEnRuCehd7xcI0AU4YCyUNlOm-OCQpM2YSN3ytVSvkaJkQpPotUrq3tIgMBM-dGy0HSDFUNh8nazFOxqPFaRtE3m3l1RKnGO~G38-DFzf~Y82ZUPPXEXSPl85kcecwGSeU5TI-LwTpPF69F0AFd0BtgXI0u~G60a8zLQMB7FJdfmMKtbdpAZYMUdJRRMLIc6KsLdlJIa2h1hqvdS236mow3SWBtV9q18P5gNJO6IeySRS6ApvTd4VqaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i39.jpeg" /></a></div>), <em>Pyrococcus abyssi</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i40.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i40.jpeg?Expires=2147483647&amp;Signature=LW9RmjSRzOjsBTqXttlejChLqFPRB~O~cfbwagCER3pvzLqqGvSqEuATfwzf8amqlZLdcAnisqfxLdlFUkodVGRuOH8iuawUhs4-kVHMx16NZ-cfYBcXbUDsBKQcQFE3yIuO-E3CN0gD5sIWsjbQ2-Y2N6uhkSwxLT-r5DETyhBkFmXgy06MYCX0-l0pD7FfeIjWN~0LRvAHdpjOkAg6AFYyRZDOVg6xF7VqH2soxu~rRMYosVDTW1IYHcmbKi2zNBaKqywU1d8VFTzs2XVrxfO8~PZcjoNpkf2zOh5nnz4a~yQxivr5cmATZmbWYR4sFAzy5d0~vqU3ekW80waLCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i40.jpeg" /></a></div>), <em>Pseudomonas aeruginosa</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i41.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i41.jpeg?Expires=2147483647&amp;Signature=scRUpBFgt7VfXQsnqi8fYionCBhDLFuF7EwycJkXMTRKZANXbf08rKoor9bROZ1cs6iWc0HA4oDl9vTsmjZu1tBdj8Dm2DuApt9s~RsZ1tbUxeEUL7A9-TS-XUdads8v4Zdf5LdM3zBR9LFi8McCwod3OB--F3OGTH7dzkszbA8F3eZ2GvPYhA5x~qiJpZ0Pti4NPQg6Sv41eS5XdBbtV765nFZAadnDvZxt2zTYBOsv7Bb0GUyEc6A0YhSU4M2w~xqVGsUuD4eYsxGOKUmwKDi7rvt1CEUi-EGg1ds5t2hUQmpicE8ZNh4lKP28CkysNIZ2igr9K3IFHuv-Fvp1~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i41.jpeg" /></a></div>), <em>Pyrobaculum aerophilum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i42.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i42.jpeg?Expires=2147483647&amp;Signature=f4RWTuZo6SUXVBa5Vo-CT9xP3IXng9naxxsNPSzybI9nJNfkIfXjphHz9xYAQHZaX1qboNZixBbdQNa6XkHm274s2eBlkj7zMgwQ1tfm9sSvk9VFLR9rcFXwBVT2eQoP-0SPE5Gr-Ob3fuNUO-ozuD~ufeFN5hoZzvsMbrvmawNS4fUDpu6qjH2nkKhwhqgghdroB9tekuXHrpXL4pYmp3hGK1B3~eloZlwvUxZK2NLubQHyg-X01qo2Y5FRo2IPxN3IkOUk3uQHZ97eN0YM~Bxg5mK9RWBFZWddXk5chX-r4ovRL4SiXljk8p-RvkY0wQTiOX2SOPtCMQzg6RnOnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i42.jpeg" /></a></div>), <em>Pyrococcus furiosus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i43.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i43.jpeg?Expires=2147483647&amp;Signature=z4yeKobCH6Bs5g6ft7cpkql1iM1lYWFtci5Nmys97ncDdED-7UvMlYhd9ewwsV91J2QBogFaevjUBQNrSP5UBOp0y8aMQIB1jCPIVBgB7TMd00AkxyHvbgv3UiCfxnaSxif3T3DybxEyAT-yanZz5NH0dBa~Qu6ioJhOgMlbYQNlQ-Lo6cukqYwcUxcvND7YW-Yoxg033JIfDCUea5EW-ZpwkPbDtVmxNaZk91fuU-hQx5hKbdjlDdYlN9ijOcZQb1WC~cuzcgmUI-6Btx0jLEsKF5OqPFGD5d9YQH8hAdtetK0zRPRlzIv3bUUxLNSTUAkjdQdT7H8vrnT6x5mh5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i43.jpeg" /></a></div>), <em>Pyrococcus horikoshii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i44.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i44.jpeg?Expires=2147483647&amp;Signature=WZlIhl6laO0am4eGWIR0wQyS3p988ri~cFfcQmNrY6Ro2-vu-zIvIDL7WO8StIiPsanDL08LFavc6v~vQxuaC0CEPtdwz165j2YnO2XWVoGI~GytMaOhCBJHv-vMcmvsB23zXoahEMLCWMT9i9yqkForjdCs-rH4iNeZkDyNYCKlZybOiez4D2eGarDnWt-TPhZMPRF-PXllgyzE4kfIEHfgftNe6KbkzT-VwywqgqxTS0YHfHCy9NIOr63x8TDW13vjmh9sDzclxJhWiNowbW50RQr3tuP5k397FDUBQyLTi9HLhnpSzlUuigyJMV31jf8SD2A7Y25bXs0v4rAbKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i44.jpeg" /></a></div>), <em>Pasteurella multocida</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i45.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i45.jpeg?Expires=2147483647&amp;Signature=jaMp8yLjdVrwwJQIS7ibPIXq8tE1sQiOyJ6Ci17cnAJINMiS~Ra4nZOEWJLWJ0SR3Urw-6gMQllV2LSwvh6kqzRQmvt82ofTrlfVaHXPT51ZKXbxcrqeLmV7HlNuER~KUPryxfdUwSY9wfKdlCHIKoCcDsSV1PJ~kP8oBbeEPlNdU9XAD1OfmfMtwSBq~iYdc6IapK~eRiaX7Msm6P2EFGhvlWRXp1YZlxpeQ-r2fZFi32Nbw7ijWr8GdQPDfCz4Z6RhrZJ9Eg9eiKa5w04K0-f79EEEwfPxJclDUFVfxhxkBZFT8PalFP-Mg2Cx-9uAFyIZe77PuwtqKkMrFenr3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i45.jpeg" /></a></div>), <em>Rickettsia conorii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i46.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i46.jpeg?Expires=2147483647&amp;Signature=LF2yVISWGQ7PxemkxcbZElXssQvPlWa-RDbPlooppb6D7RCCAWokKmm~r9Pwu6gwv330ONCAS6QtC9U~8JZBYY4xVgiu8CY2UTJLusAO~okvx4RuO8vPYuDDreHsZjshi~DYsQ5ePm6-71QCcPx0MGSI3DToahjm-2TXNRZlcIEeS9JINs9UWw0S5Fm9~ud8Ws3b18rRauCpHMqKVy8-kdOso2bGfxo0aPAv3Yd3Rxz8VFvf1NAGAjCoWQNRtAW5mOQc4UgQgUP4N1f3X3ygL0lfHH3YsB0EDcrn6ozLEBUQhCkP6Yi1RCpK8Gz8zNuIY4aZl1CRRuGhGVJtm8X1Qw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i46.jpeg" /></a></div>), <em>Rickettsia prowazekii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i47.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i47.jpeg?Expires=2147483647&amp;Signature=ZQFKUVJGQfo26521~ygP-zcTLLfsVawv6jgt5n9W8rCh9EX8oHX8yJzYgp~DFTeVwx-rG4YZvFBWoqZzRcYexdhJKDAbksY0xC7zAx7NRILi7qNWWMjY3f7rYBJG6dc4md-MeDTFBMdHY1MdhumESpAZX70~EFbk6nzbXQ1aIbnkWAhB8w-1FwgHRWC5B5NW6ooewheiyFy2rz9u4ZugMLjA2lcZ36ZOZAxFE-5xDSTO3V980SaNpgt20tvmmg~GfiHP6Q8TQATbzsD6dLf4QTUt26QiY1~qzMMd2VcnoaS6qaDqIZb5aQVd07hDrAuIOVLF-dMFxrQ17lhrW5-laA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i47.jpeg" /></a></div>), <em>Ralstonia solanacearum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i48.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i48.jpeg?Expires=2147483647&amp;Signature=erUqPDJwiwByjFbUo8ImW5j3qp2hELL5K9IhjnkKJgTLaBMXdee18nq6Treo7FV8Rzno3viEgon2Fc2k3bNQUqhjvWjsInZo-r0w3Bq~mYN7A0Mn~LiOEdTfF57X3Zab0t24tyVqHzBal9oAVQNyVXaezMsq1kRRNnzRNnxpFlMPaOdPaPfVFhqoFct~jzaDhwkrivHQWt-7Q6UaKo0Yh-LJnfFuNPm~is8PrDyYVJEByjL6vRCXuY89RbQ6Q-riNi8ZLid1Q-vaZMBFge2VsW2nM5vAZWx7rny85TII5ymMV5mvKA7h74w0mQyrhm9XApiYPdJZxbsCe0PJQD8Yxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i48.jpeg" /></a></div>), <em>Streptococcus agalactiae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i49.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i49.jpeg?Expires=2147483647&amp;Signature=a~BnxfFzjgf-PPVZlm2h5ofR-KPJ3OvICDV~F2LqK9JDO~0e4~x0r9LE-D7rLeW446CcD9CYY6MtJrDGtE5WTvVT2ZmjtYILN6L9QtZXBQ84B-PHyqmP6rKvpCDiRl6kiU59D6IAuWFcASHDKcLKQwPaFm701WfkxvJ9FaDr7~ndGjBuqFEZE1ceyGXY4A9uLp99dMZ~lNrnXxT5lVVkh4CWVV4Vt8hhlz8SvLtaNydkwRMI8wC0waPFG8RsEiVX2Rmff0m4G8tDM0J4lom-LRv2C0oWlGG-A41GpuhI2hYizhwHbSLa~2n8bVSo~aI4B6lqaNbX0LpkkN1IbRWTSg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i49.jpeg" /></a></div>), <em>Staphylococcus aureus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i50.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i50.jpeg?Expires=2147483647&amp;Signature=wkt96cdqeaxoe8K5p36h9eDB7MpGUxVBQJb1KB02ys0zmqWm31PuzAie8U4o1eCdU5Bf8Ci9mvHa9wksxnc6BcafFpRhIXefco~jUBf364i0GfogOc7iBVIJLBN~SqOmteeArc6SYvMlTKk4LqZF347YGYjXDYz2IqPGxj56vRKBszQAxQIXy1tTopLLnK~QfDI~d0ZwyWOLnRnZv5kd9N3KYRNnWI2oBl4~MxbbKmZ~s-u1HWBnG~E5Nt-GbuNA9QDOAiN3hgTF446a0mCgTsrRht7AnMD4akS05I52d1~MNGqh0b5KDFptFvQQu3POLvIC0MpZflayAt8MjGQyoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i50.jpeg" /></a></div>), <em>Streptomyces coelicolor</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i51.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i51.jpeg?Expires=2147483647&amp;Signature=z1OjCzfHC8ePBNK3BSLtsF5UFFA4zNvgPy2MPFSfaJaRCJEb03bQBLM0qNJWj8XEcpV7lGapu5EKNfWPPlHoG85qm8ufIgE0IzhM-JKFIqrpxwJke0QE-AChKOREySZzWIbWWbD4JnVL4FpYN0zwkL0pt~IpkOSp73s1Q8k57Zw6L~nH3xKs0UwiUk0ScpQ46WVd7M8BxElGFYpp43AI8jom2YuekGhLTt~JCv04cKvorH-u~ojm1~d4n7Fld24cHkqBkqEWogY0dAWebLH-~ZasVtuapxQHsMXiU7u8n7ig1ZkZuAPi84~GbTZzGZPfdxPmWh-StCljc9cZBKIMQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i51.jpeg" /></a></div>), <em>Sinorhizobium meliloti</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i52.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i52.jpeg?Expires=2147483647&amp;Signature=H7EpRs0fJhe8vNLuABFWP8rdj4XYDfdNsvvknqgsbK1eJfebOcylLpHHwJRsN4B~p6LnIlvOlkzzZXsdZXuFpII6gkNg6cTfkjWz~UD8fu46avvUH7ZoNz5Q1Zs8HNWlepaBq~7fQ1G7igyDJuH23QCp9qoNmDImGEY9l8-SzX8bWhIhWooH8nL4YwZJdJxmmBPY4TOqIlhRoIAwLbngN-NiZL1aSq0dh3-DnuHnX~3IZQl3YDIWbe91grJJUKXGPilloZ-llHC3Q~VtPfGn9p7xwCwdM0jHDsX4F0gGX0l~VyKSiYP5CNU006qjAtt5je0bburgzXUdu8yjFhA3bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i52.jpeg" /></a></div>), <em>Streptococcus pneumoniae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i53.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i53.jpeg?Expires=2147483647&amp;Signature=jV4GjPR17hFSBnYPCVDuxkBYRL9itNefk7Pi0uOk72zeqJ5E8P8T0tjat-HLmvnucT3wiLklsRjwNzca4QMQsy0yMhwW2UYJ9q-cn2p~SJwmHz2UVEMyOfqQbig4~9pQw7DcHv6UZ~pTOCs1~hqJuRPJqm0Vz45YSbfz6I-s0CV5p7P2y-~MU-ypqQ4oepBngH15ox0rDktFgDuw5tnOG6T~auoATHfwcwx6ERs40Bb-YPXxTZoKXn71s90~JDwXLPfGmb8p6VxiRx3vOHLBVfxYPWcDjkAndOk2mJ4Fed3Dikk8St~pDLNNGPf6E1sj7t-IsFJ3FLr1EYG63AwgNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i53.jpeg" /></a></div>), <em>Sulfolobus solfataricus</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i54.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i54.jpeg?Expires=2147483647&amp;Signature=cx-NoAdgi9AAs0mT~ODNm-hPcQxfkOfFqDfEuYyUxjyl95FKRDOrC6uqr2Iyo10iZmD9L3wIQDXnqrBCXjizH9FdArohNYtNTQfwxNd1bBOfjf89Wojri1y87IU5j1s4H6zaxgBKv7U4Zr5fsnIqorgiqDIo5B1Hl0sQCzTm437hm48yBGSTlVNnEg7bD1vNVLC3yI~336tukXbvGY6HZR8F9Jzy8QGd8hw3ZmXD~4pmMS1o1WXP0Y9680TiTLQkGZ4~uQYKfL9a7PDJg4qYL3HJ4s0sLdde9QPyls0wAuKX-7ixSCfpFviQaXafXRDtqr3eXCfNW9ToSmfEMwmoGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i54.jpeg" /></a></div>), <em>Sulfolobus tokodaii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i55.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i55.jpeg?Expires=2147483647&amp;Signature=V9lYsRvsfqXBGiETg~-mhfznfkivNWSQPF8-pm2C5RBwwXWB3-tuqHLCUDJLXik9-B~Vy-Za93fPsWvLz-QMuv1UATN24Cryr4IqPFgF1pp6SR2y8s~rEu-WUOBoRGv9JKTzejxX4rCijiRR9VjXXCSiOgTrfoiWdn1LYHBdbCAqJjzlJ0y5coJs-P29V0PEWIp9Qbd0Ssdw-aqchIcDnzKfuENIs83BXp7N~59xdtX7dRixtdsbDEvzFUCDLZFbMfDlnD4Zy4J0zhNjysuCf5CYhjQNmG0O9E79SfeTcudOkEmBzNssBTrN7aoepaRVxVozMyjDzqRShUkjzn3J0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i55.jpeg" /></a></div>), <em>Salmonella typhimurium</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i56.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i56.jpeg?Expires=2147483647&amp;Signature=GwTXMeVPpsLP50DsSTxzN~65FQn0X~jf~Nff5pdgqkWsfr2NiAYSNlPoap7bksNCGqPpgnQzycI8DW6F5GKYzFNdX1Z6UKF4O774AxveUuGu-tnneOwX8C8y5lfWFv-1RQXQo3-TEDaLDPEkIn9o42xgBj4W0SFNH8veLWAUMavikmni-DHOJSibaMyyGzgsbEv54lxfeeZzTCKGqZiSJwzYoCJTxnz8Z7VBPi2NQiBOfLawAozgzdoMcGo4Lv9Hj51UJasDKxQLzt5eYWXOTP7ap8xKTGoo~k~6qzni7AFwv438gBR9ADcW2exSQPfmtT50YjMukhYCy1JK~i48yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i56.jpeg" /></a></div>), <em>Synechocystis</em> sp. (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i57.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i57.jpeg?Expires=2147483647&amp;Signature=k06rYZptl1jLPHGn8yWnHxTPo~SQh93r56GESOL2CPghK7dRWXRxWmje7j~AERFnCK6z46tz41HEXIhjeGJcCITnTr6VRW7pb3td1OHxLsYemrve-CQFuVrS2vjCNYEJsjqCTKJfl-izFDKP3LNYOERSp4xfQ6BecHzER0tgPehQizHRvgNrDQ6kS5h6GCCDjIbjq10ntob9pc7i6ofKNA74CnRvtT5N22GVlOy7JWAt9hTW19FjVVwxFlOClqV0s27fgIk0KkEqY1YltoahIzxTqfzdhWvgd0rVJ4qdHna8lx54ExXrc4pkE~eJJsvXQMVFVkBNXFKZg~h4DQshDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i57.jpeg" /></a></div>), <em>Thermoplasma acidophilum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i58.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i58.jpeg?Expires=2147483647&amp;Signature=o20gkg6zKtO3x3KezZ7ClDZFnUm-t~QpNUR19-cM0ClkjpUDRaEYRwm1xak2em0KWro9lWKPoFsJ1BxgUCIX2icMWPt4KdYWtpmWMzPGjQsNHS10mJOWN1ba~oE6tO-eyyzGwkc5q8PCUmDMoRnxUPrJ6OCUzBTdlTeBqCuxPLn9C6c~sdMNQgh-5XA5y6XLdLGHBTQ4TssMU--WUEO4gvIYsr1EQL~qEVZvWKmyP0eLAV73Fx2g6arGRS-Nb7tTOtx6l8jJfZvbRt8CQn0HSbcEpDXmAka-M6tVCf~WHqeQ7GTZMER9zPye0OwQDv73VISwDKDgEbJbXCZKm-HBaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i58.jpeg" /></a></div>), <em>Thermotoga maritima</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i59.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i59.jpeg?Expires=2147483647&amp;Signature=YDT4ZPHhKrxyvNtDN5282Toqkxx1gFBr-UIQkKB7BalfXjYsn3opfmmetr~~7TMXKln6xeB~axUPSl3iiVa46UZ1bwpXB~ROafzsgn4gCePPGTO5XKUnTQqnV4YzUIUP4J9Sqcy9kpSjo7~jVEQMiDRJsy6PIOSkNp4WBmSOIvDKUR9LhktkXW9bds2peSXdn~6dd4mJMGwQ77tnDhs0RJAXTVp59T2iiajGA7mFq75lgA1m~nClMNXS1QWrMr72Cpa-9QjQugKyBvGme1Q~bIQiEj98S-CM0ZNm-ap~6NSpmd1U-QhmT7dW2h19ubFXx6PGOU4tySCLB-1BCK~dkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i59.jpeg" /></a></div>), <em>Teponema pallidum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i60.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i60.jpeg?Expires=2147483647&amp;Signature=kNTKPRpf3PVsICUqaKCXwnF0-2O3WEk77rlgHuoJoCv1JqQhU5B1DZoTxrIsjEzVw~gpHDAgfjfiqQlS4yV03KAr2IUXOfTX4v9hndV8njlmkSriwUKeUBMh70XWHj0UIcMnPFLTUeJwiKs94FfI-8rxZJNN89k7Bc60fijcDICP~85u0HsW3ax4sZopo~seQLYj67iEC4JMdEa4UicGneM9QPxRDMPgEXTXR4ndjDS40~4-z22oD9punBcKf4HSmKc0OddLyLm3yvZlwxWasOujsavAU1L-1ROqdoJqvReOQMi5kjYhQv5Gkmp-nkrzTXISTow2WNBu5lPRD~W5fA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i60.jpeg" /></a></div>), <em>Thermoanaerobacter tengcongensis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i61.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i61.jpeg?Expires=2147483647&amp;Signature=B5v5-6pKsWA0XOw9UiG0I3gMzkfqCngycIxtbG3b1iy5OASSalOUzQE0H1pTOzRB~NOauRkJb5kylasjVNIabIjWILJ~DQnQdvOB7pCK6XuqGJxL4gs3qJG7CUumnJtCyfCxGTmkaIUxf-UM9TMDsICKA8F8y4Vz1r-9EwPnYRweJ4CsEzyf5s72DkvdBYuDRbJoHpuyWQDnsle1IxxcrTwbPTHXMKZ1FrAAml~O0bnKzZlYRU4EJMMOAXTvDhEGAAD-Mpzc2J0-75T31NIdQgu-yUYGARE03CKIyOtGNGLdLJHVF2Y1lncfqewFQ-4DiAAY0NO6jo546IrOXrgu1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i61.jpeg" /></a></div>), <em>Thermoplasma volcanium</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i62.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i62.jpeg?Expires=2147483647&amp;Signature=wT763tQZ4fOlSZrN6oIGdi-fJQ1fBWQCJuH2oaHGb-sbchp8qjzTPbAkQEjgLxNrmolG6Ybo7WQY2Ixy1vvjkV-WCwAImPd4wacDwXOREvL3PYEznk8fecYS-SxCdVe5oK83ef8jwRZDk6R0cnEWMXq30DVnkSQf28VcxgiHdXe4mXoXR-MS2kxh1ON8M6iWWZngVMLvoalIEBFCpQGEJ44a62mo22C~SDXFVJ1ulZAQ25TVGYViaRxuCmXolHJK8n7~8tD3XzjsoWz6BO6tJtjlSmPfzYjItvitYrq5sPUiuD5W~6xPK7mD-sMjykNma2NyC1IrVT42yw~UX5bgTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i62.jpeg" /></a></div>), <em>Ureaplasma urealyticum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i63.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i63.jpeg?Expires=2147483647&amp;Signature=wb34flugPjwZiZfW6XPbbAkOE7Og~wnJklaDOW85Cgb32zpP~Be8Am-oKyRsDROLkqchkRUo3i270ZecaamkbIu8u9Ub0HVZ-bXSRRUOozCayd5QD7eNj-EDoIm16ddx6hmUY-mWEl2Q3AKpm8WdaV7Lq7bBot8QWbNT~kbURU6EEVJ5R0fNnMh1C5jFDZp44lB52F-IdulYvMpvJISUMxBBrpsuaOs3svo-~u6IaN-kTMq6p5MtjO6fzaqBejpSlCRhr99Rmp5DIMQbRHi5T3nzlzwfXPk5KRFmQ3JYYQPwEgotCYKkJt0ZILQ318EOU2a651M3Wby9MJwq9am3Fg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i63.jpeg" /></a></div>), <em>Vibrio cholerae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i64.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i64.jpeg?Expires=2147483647&amp;Signature=b4eb-aLgkrI7BOrYxHAJZKAfgup5f9FALe-fwPfIl02059d~ITAIkidmukm~BZe50TNIA8UWYG~m0YKn9UgZWC3dEabtOA9Di2nabXYtLrA5qJDfA94lbyk6coXysT6xA8qJuOpcL3gqfwP3wnEyirN88Ghh-LCmtUk52roV3frMyzuVkHT2msLNNkC9SKVyDgM6-V7StSaIpss~pAR4O0oICkiPy1dKMyZLsYoKUTjHnZrojlNcAyEoarn~oSZuC4V~K5fgQJqzApWgJg6Gg8qIHKp~9k5-u-19qEEBKVlWpf1CPLaYCTvpwLGuSBdAzQUQXJ0ItIzjkNHaOoYX2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i64.jpeg" /></a></div>), <em>Xanthomonas campestris</em> and <em>axonopodis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i65.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i65.jpeg?Expires=2147483647&amp;Signature=Q2MCe7wE4YkYy9Z97MNlnLA-rvpqNNsLmqcPEpfrhNwv2LLy0BgI9bQCgm77YCgfRKCg7XSmIvoY02p8ZSMxYcTi-WfoPs6Mc~PlhQC5MAV225bJO4WthbzkAeHNi2XNXWZlGKGmyoo6W2kU98aqLWoML2-yTc2nQoImGb-YWO2g9K1hqwYixlc2JrmaRLgKOZKSdwFq4OsenkT6Huj4sNwkyxCPWGVBOUd3ILawtDZ~Q8cxurNsqqqkkdvlLv19Br2MG984MGMJMTl4ix8nrnjTYPgCUncbPT1ReWrPAOy97makzNhhd~r-XLDqFBc-JWUrgGa57solfpHZVb5CAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i65.jpeg" /></a></div>), <em>Xylella fastidiosa</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i66.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i66.jpeg?Expires=2147483647&amp;Signature=q1yeB8w917bIXX0ybfgblayEY2JV4m3eWPRW4cZqR-vTG1B~PiQKWevzB5BF7baX4yyaGHaAzbGlB0XoTwjUKEe5570VolPZTSn1Pekt570UpRmTe-qQyPP6gD49YxLBDT2HgnOCkYJHl3fqNYSmO4svuf74S3xi0ALHGnLoQG0kzDihKuIviVQfjaELxW3QvPn32xkyBQloHhzeelp7ikE4YVmmNqYl1i62Ns--8FPbbsoGotsMMpHwyEzkIhVUYeVClJqFWZezJ0rEkrfrlkNBVaoW8umkokVWRCz59~T4Fo8uv2~XGWw8iMCVKEQFL5gxLBCGiHTjK0AjbKHPGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i66.jpeg" /></a></div>), and <em>Yersinia pestis</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i67.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i67.jpeg?Expires=2147483647&amp;Signature=EKyIJqbZ2QTU0ESvYYL~4uWy5~iRRvFk8hMduDA2lg2YzgCRwNZ4VA68EhHJoEKOqPD~KPJXwydEeRnsKldzAn1GqZbPTyQxhEBvATGqd0k~PWLU4ey7GE5XD4SxHWvSgwwR4exvfb8pYYB8GWRkIPaF7y9ptFSYNKKjCSeULYBnIHZ7zpVHzRDYHW5UcymvvtcGMsf0GIzQ~xYi-91H5cUA5F0CdBSRKaPveOZOgyDmqkNFkBi0lVgCWdmlKb-Yq-jYsow70D9lP4Az5vOVpYnxjN04RQ0Jv8ewvS-VUSZbAr2wseH1oTNqDGm6IlwAX~2z~n30xc8mO1KnzMmUMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i67.jpeg" /></a></div>). For details of species-specific territories, see Supplementary Figure, on which numbers indicating species names are added. (<strong>B</strong>) Phylogenetic classification into 13 prokaryotic groups on DegeTetra-SOMs. Lattice points that include sequences from more than one phylotype are indicated in black, and those containing sequences from one phylotype are indicated in color as follows: -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i68.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i68.jpeg?Expires=2147483647&amp;Signature=SI9bTx5rRo3XPKq8ZcRIZcbASrKRf~EmR9MQJlFoBlnuPKD0mcPYmDyMmuxLKGpjItn-Rg5uqo-XNu-zuDCjUr2Kq~kdOGIGgeAm-~fgoSvCkxXdJrbSGVQI3L3eKJtqJzmWvx2Pertsb4KWOJGRiEz~ln8KevV52IwCrmGEat7ldfYQep8GrViGEMobe52lgVjgFKHLQMePc-Wt6jiy5uznQy2dewc96bO6ab2l4t6ISi-L3paLnJkJJcwfZe6oNioJ95XohMQW~NuKbV5SfCps8rihfFcJ5pwAo~BO6dC230p-0e~JHAUGj~B4vc7JbioqyrAoqO-FMLWPqCNGdQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i68.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i69.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i69.jpeg?Expires=2147483647&amp;Signature=ttDPqgH8HBCTkkbM89FvLalnRqmSE7eYYgRwMpN~L0ZFVQoVGF-WHA9muGYaa5Fo4THcoWZvamK7v1OXnzsUzse27iH6r7ivQUKAOvST9pRNWVjAf7P0KBU1kyXVOwTbe83zvdjFaE0~5zKJD3Wis8MfBRAKBZXfCmifyFiPnuDznxd1j~AMnq8DZMq49MqRBkk7mlUUZXT3Bvb7SyUb7cciJXg3RoTGc1OvSN5tcKfzaYJJXUO~0dyKIOWvq6CBNkUNEUDN4QJosmCIrZjlAq0y9n6a439WZLVBsrw35vOd68HOcPsT4WH7pDfPzrPObm~IYDgVTiSOOWQbaVax2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i69.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i70.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i70.jpeg?Expires=2147483647&amp;Signature=vgAvIB5YEpMlYf0jCB8~V0ipl1uKYAeI36zDyDPU7bhyzwAH0pfzLxmbQAAnI1bG3FX27UYKbaxOCNJxuBALSz2fWqFRtYO3ydTDw540rat7XNSiA8gQICiA6AcswARn7FeUhCx4MRg91r3zWQQE-h5k3wxHrqzQNBXFUV494yBa9gf9d78uepvxB~jzqlbQ5UZVXailieo-npdXYWlSZrJBaI6kNW7ZioDoW7rcM4uOKtBHzb0hy9ebT8GOp8vTkYhhPDRIUU0cnW7pKdOzOchvj~A2vEKnuJyd28E0rtDCit~bszXoYryfOM6eVia4J5Htr6puYfCc52RMRDgs6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i70.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i71.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i71.jpeg?Expires=2147483647&amp;Signature=AHcmpkYgrwsDzWYacRS732lNI~j9zMRDUfrWuWm25VIQqm7sDwNOTGzSWnhZeuMoErCyX7I~ropeHnM~-MBvzZ7RQxwOE9JGfTsUW565Mxy1K334OEvQu59HkFMTmMkcMSv~LUno2ke2LnpEAa2wOSAVcMs2J4BGUcW9xYzDRoWUOGZYGcczQHN97L6pq4JBDDdA6JGwlgcxjKFEGFThRCiBanBZGXUpcqD53TuJ-hDzZJC9-dE7vISaOaW64wGC2ybgEXT4KxK0CXDA6s6phtht45lCk4une25F0pnKMF4uJ4rV05xv35WFT587~8s56OzzNvIwjOQwGYuIh8ybEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i71.jpeg" /></a></div>), Actinobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i72.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i72.jpeg?Expires=2147483647&amp;Signature=VdVY6OFhMArIVdbkZW0vq9AaIC42aXgr6iZoovSCJuGUhIFEYVmqIzHW5KHDXraewljgnP9NxJjX8y-s9xKck2bdP4Y4Zdt-dkZbcRuSLXGkTYTK~CJeyxNbc2WdO7zDSFCAtj90zyG6lGx3Py6Ox-8LGGdAk3CKylroR0p~a9qD70WJAQKbcZp3P-FM77bLGDF83d0tsHNW1Q5n86NjXiSR2s6uhLYADPE85BrvQkL0qSY7iGLbp0sCezB6tcgsJ3eYoCNTEJW~rhf4wGBnYUjiyDokAW5ZZBIK4KFbyCeX54aEWbk3z0matu9YmwwKjv2yDF2mO7IcQiejVrjKJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i72.jpeg" /></a></div>), Archaea (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i73.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i73.jpeg?Expires=2147483647&amp;Signature=A4yLsxkM6FD3ag0l6WbyX9iLmk1UgcmqsEZtzhUVwt10KeaHCGSDhQJ6W2Hkk7gruG4PT6jKu0b0KO2nvcn-QtTQTiySwhMdR47efAheRnFIBrFDDH3Pb2nHsrCO21V0hI6ZQeA8-CtEDAGq0on~2JCqrvJIAU~BMz3mrBdxwcCDqe0kIGp~~eoU1UkUwKBCU6W1HLMMJG-AOj-cUUwFf26H4dwUsG5I5DIkF4aXNTbFjdJ1sAny2s8StQVeiFCWwssVi8qfvHDK3BYo1dPhnlYmCmsZTgJQ016XP9yh0v9iDcZMskm-uYIY7-EzEIbAu3HFbzBidOSKaqPT2JMtMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i73.jpeg" /></a></div>), Chlamydia (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i74.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i74.jpeg?Expires=2147483647&amp;Signature=a32A~ptTQio6k1C2p698eoxCSbJM08h7YW6VqBcy0Bn7ISMJR7kGIu6Jg1vXBQZ-giBmsQSXhDKmx98QhY9I-QaLwMJqbF39UXCLmJEAV7prK27wRcv9jcLwomoylaDHfoMZ~N05RxN3MSFGFjxE8gXlUrNts9elosULomB6KI1OJveOLqdCy6q8r2SWWyJRD0kEHV~apGgeF43ujB1sUs6eAARrGS9IAflrb6sJE04TcD9KNX8NQYqIXAU8TooNNAiHfCplqCpA-RqN7Wi~Idm-FLxoGOhvjXMLe~cLYBSMkBK6eC7JZmTbzxTRdKcc-Qu-n~OfJrHGJW33H-Y85Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i74.jpeg" /></a></div>), Cyanobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i75.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i75.jpeg?Expires=2147483647&amp;Signature=KwqGHgKi-Njmc7SDW9Vp99r-X44Yi6VMCo1BvtqAMtu1Ts4bHqZ5uIKyk-~Oy2OuEXMbxOraK82PF4oHidoSg6Y~IoFBq~~h~hj66LZBvJiGEQ0onusEpx6hQBGOwjCW9u12MYIguBp3jmPRI70oXuBgi4aWuHKbeEhR~1qf-pJro0FWk2RhX9rvBZXq5DM8ENxL~PmXQtgYRdi2mLyIUas11uFprUXVWAgMePPI5NXsYAp2owXKpYyjfSQWHFMBSdOUGmlBIb8qYot7SkXdzQGAV40qtSpAfw0HwceD5em84JIKQq8~Bj010cJuf2IZPwgfO46~WF5aFuPA4GyqgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i75.jpeg" /></a></div>), Deinococcus-Thermus (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i76.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i76.jpeg?Expires=2147483647&amp;Signature=x0Zvpwlmr-EP54ykcF17yeqmoxEeywZso1sV3LNx2TT3K~rOqIJUafdWEqO3uR~pii82xO9A-7j3FeW7C1V5DR2xsjUcRhckmYEYvcpsjt1aY9ocTo912aC0BddJVRQKltJsazgaG~EawjvqHJnhxWhy93g-iOusfAcY3RBDWzxLQmc3nlsqkcstDXlXS84z0koFLf6JWKSnB7n7Sa9rGOPcKBhuzPMTFSOdO4mx7fFHARbwkoAnXU3S9wbofZJZPBQUoKqhjtLhS9OD24T89lmBpX9ESelfdSY3lwaN9dPHDJBkXgJfaWeww2EgEeXIO3VDQ9HQUWnWSC7ex-1bJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i76.jpeg" /></a></div>), Firmicutes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i77.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i77.jpeg?Expires=2147483647&amp;Signature=K5BJPg6V5cMeBndd5oigruvRxkDdFQsApJTce2KXz6ZweSRql7OFjcjiaGZd546NDOWhZR1-G-nEf4A8qjBLwcBVcoCjJdpz97BwfOTn673Yvsgf0xYmsNyLc22R4od8UKIG~V4mtY5jqJgAr90AVCfHxq8f33EOzH3PRHYlUCMnl0fUzhzVLPHFrUD6d2wyWUP~eAkGIv0TtPfGfnEuCmggmSn9UV5nHdgkNBCR~rJHpc9mS-n3B8cFOdn1S~GX5yzg57rW7giZdq1wcQahVgCRv7dVLgvAHj8ULQak56CDvNDx~PdHQG1FbWsr5QeM5s9pAWcND-FZyrp9s3qXfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i77.jpeg" /></a></div>), Fusobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i78.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i78.jpeg?Expires=2147483647&amp;Signature=ZrPbX95vuEw-LUs-45rwSUbdNjk39UwsQpup4CoPpWAmHFuC-oBDhDkKfsAzHxkP1DhB3HTbiVfmbkEarqnK3NAlAxfApeyRZI3GpCbHO~N-fSieuA0bS4rcXaWNNa0ZMAWs77U6L0kdNte7A0lyV5UyXWpi7LNvY1Yz6wnKp-UHaQL4B6faApEcv5VZw-n0JQkv6hXSV~fwfWFhIZh5iY7ule7wfJWAnc2y6eAzOOzD~abvLG1J3hzdKSZmCO02o7MbylFaW~28djUSDvggxLoSirGQ8smmiotMi-2r8FXEHM07vWdDf6d5R5kHOeZg5QEiU0Cycx-rK9LIeYcfqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i78.jpeg" /></a></div>), Spirochaetales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i79.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i79.jpeg?Expires=2147483647&amp;Signature=G5mUGqRjd8bpeLxp2Utwn~Z42gWke0XK7H67Ib-uVL3iWyGxSJogaWQwyDM~kCO3s7-1rLySAvXBppa7KwDkQbfB5fJjIrgRd0Tpqf~~--4WpRugAGdO254uRFoRA-AGt5pP~O3X6orxna-EQMja0kce87zA7IjorZeAS0EhxQm6wXxdVwFqaZ15iEdIjdSfT0wqWdkB~nobr1heLOLLI4rKxLa6sDIhy37iEIVxsW-W2cic4Qfbkkl3MOqHzGLSxSw2zAcDhQvYcEGWy8YMrBJLdulz0AUJtvtVycmBFI-WwLfKDuTubSI3emdj9R6eka0oOKGZ4pGMxVaH80aYPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i79.jpeg" /></a></div>) and Thermotogales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i80.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i80.jpeg?Expires=2147483647&amp;Signature=a1zRMrEZ6sBx0PVyikjPEl2CXGsRO9V7HI3baPCn4Y9O-okgY-jXWT-PPhW~-YkGk0FY3~P0qMXKJns6Xes90puIKI--qccsglRXfK10HxaWkMyfxvfxOS~myLGJq-x~R6tF5zjy4aL6I3l8IbGxIMYC7SeSc5-F0o2dOGWXq~hkYqJ3ZwP6LCyxol-XdqxFDCrCEx1VW03jzc0K1h-1U0vbgJPDNk-FhtJP5c7MOrc~tuNEGtNbYko60q8Z29FB093ZAdIhX0QpaJ8diz3ExqDvGrE4fhXtPmioaHvleMqfbdcQpaD8lOx5DGzf3ZynFnFEeRduhagExU4maTFd9g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i80.jpeg" /></a></div>).</p></div></div>                            <h3 scrollto-destination=87193410 id="87193410" class="section-title" >2.2. Nucleotide sequences</h3>
<p>Nucleotide sequences were obtained from <a class="link link-uri" href="http://www.ddbj.nig.ac.jp/anoftp-e.html" target="_blank"></a>. When the number of undetermined nucleotides (Ns) in a sequence exceeded 10% of the window size, the sequence was omitted from the analysis. When the number of Ns was &lt;10%, the oligonucleotide frequencies were normalized to the length without Ns and included in the analysis.</p>                            <h2 scrollto-destination=87193412 id="87193412" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=87193413 id="87193413" class="section-title" >3.1. SOMs for species-known prokaryotes</h3>
<p>To investigate the clustering power of SOM for a wide range of prokaryotic sequences, we first analyzed 81 prokaryotic genomes for which complete sequence is available (a total of 226 Mb). To avoid overrepresentation of specific genomes, one genome among different strains of one species or of closely related species was used. SOMs were constructed with tri- and tetranucleotide frequencies for 226000 non-overlapping 1 kb and overlapping 5 kb sequences with a 1 kb sliding step. These short sequence fragments were tested because such short fragments can be cloned even from a small quantity of DNA samples difficult to be obtained, e.g. those from extreme environments. To define the initial weight vectors, tri- and tetranucleotide frequencies in the 226000 sequences were analyzed by PCA as described previously.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> After 100 learning cycles, the tri- and tetranucleotide frequencies in the sequences were represented by the weight vectors on the tri- and tetranucleotide SOMs (Tri- and Tetra-SOMs in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, see also Supplementary Figure). Then, lattice points that contained sequences from a single species are indicated in color, and those that included sequences from more than one species are indicated in black. The clustering power of the 5 kb SOM was much higher than that of the 1 kb SOM for both the Tri- and Tetra-SOMs, and the clustering power of the Tetra-SOM was higher than that of the Tri-SOM for the same sequence length (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                        <a id="87193415" scrollto-destination="87193415"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>The proportion (%) of colored lattice points and sequences belonging to the colored points.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th></th><th colspan="4">Tri-SOM</th><th colspan="4">DegeTri-SOM</th><th colspan="4">Tetra-SOM</th><th colspan="4">DegeTetra-SOM</th></tr><tr><th></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th></tr><tr><th></th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th></tr><tr><th></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th></tr><tr><th></th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th></tr></thead><tbody><tr><td>1 kb SOM</td><td>43.9</td><td char=".">54.9</td><td>66.6</td><td char=".">72.5</td><td>45.6</td><td char=".">64.9</td><td>67.7</td><td char=".">78.1</td><td>50.6</td><td char=".">37.0</td><td>70.8</td><td char=".">61.2</td><td>55.4</td><td char=".">40.6</td><td>73.8</td><td char=".">69.2</td></tr><tr><td>5 kb SOM</td><td>87.4</td><td char=".">65.5</td><td>96.5</td><td char=".">79.1</td><td>92.3</td><td char=".">72.3</td><td>97.4</td><td char=".">81.5</td><td>80.6</td><td char=".">67.8</td><td>98.4</td><td char=".">84.9</td><td>95.3</td><td char=".">74.6</td><td>98.9</td><td char=".">88.0</td></tr><tr><td>1 kb on 5 kb SOM</td><td></td><td char=".">64.4</td><td></td><td char=".">78.1</td><td></td><td char=".">74.7</td><td></td><td char=".">83.3</td><td></td><td char=".">68.1</td><td></td><td char=".">80.2</td><td></td><td char=".">76.4</td><td></td><td char=".">84.6</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th></th><th colspan="4">Tri-SOM</th><th colspan="4">DegeTri-SOM</th><th colspan="4">Tetra-SOM</th><th colspan="4">DegeTetra-SOM</th></tr><tr><th></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th></tr><tr><th></th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th></tr><tr><th></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th></tr><tr><th></th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th></tr></thead><tbody><tr><td>1 kb SOM</td><td>43.9</td><td char=".">54.9</td><td>66.6</td><td char=".">72.5</td><td>45.6</td><td char=".">64.9</td><td>67.7</td><td char=".">78.1</td><td>50.6</td><td char=".">37.0</td><td>70.8</td><td char=".">61.2</td><td>55.4</td><td char=".">40.6</td><td>73.8</td><td char=".">69.2</td></tr><tr><td>5 kb SOM</td><td>87.4</td><td char=".">65.5</td><td>96.5</td><td char=".">79.1</td><td>92.3</td><td char=".">72.3</td><td>97.4</td><td char=".">81.5</td><td>80.6</td><td char=".">67.8</td><td>98.4</td><td char=".">84.9</td><td>95.3</td><td char=".">74.6</td><td>98.9</td><td char=".">88.0</td></tr><tr><td>1 kb on 5 kb SOM</td><td></td><td char=".">64.4</td><td></td><td char=".">78.1</td><td></td><td char=".">74.7</td><td></td><td char=".">83.3</td><td></td><td char=".">68.1</td><td></td><td char=".">80.2</td><td></td><td char=".">76.4</td><td></td><td char=".">84.6</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>CL, the proportion (%) of colored lattice points, which contained sequences from a single species or group; SQ, The proportion (%) of sequences belonging to the colored lattice points.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193415" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>The proportion (%) of colored lattice points and sequences belonging to the colored points.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th></th><th colspan="4">Tri-SOM</th><th colspan="4">DegeTri-SOM</th><th colspan="4">Tetra-SOM</th><th colspan="4">DegeTetra-SOM</th></tr><tr><th></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th></tr><tr><th></th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th></tr><tr><th></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th></tr><tr><th></th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th></tr></thead><tbody><tr><td>1 kb SOM</td><td>43.9</td><td char=".">54.9</td><td>66.6</td><td char=".">72.5</td><td>45.6</td><td char=".">64.9</td><td>67.7</td><td char=".">78.1</td><td>50.6</td><td char=".">37.0</td><td>70.8</td><td char=".">61.2</td><td>55.4</td><td char=".">40.6</td><td>73.8</td><td char=".">69.2</td></tr><tr><td>5 kb SOM</td><td>87.4</td><td char=".">65.5</td><td>96.5</td><td char=".">79.1</td><td>92.3</td><td char=".">72.3</td><td>97.4</td><td char=".">81.5</td><td>80.6</td><td char=".">67.8</td><td>98.4</td><td char=".">84.9</td><td>95.3</td><td char=".">74.6</td><td>98.9</td><td char=".">88.0</td></tr><tr><td>1 kb on 5 kb SOM</td><td></td><td char=".">64.4</td><td></td><td char=".">78.1</td><td></td><td char=".">74.7</td><td></td><td char=".">83.3</td><td></td><td char=".">68.1</td><td></td><td char=".">80.2</td><td></td><td char=".">76.4</td><td></td><td char=".">84.6</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th></th><th colspan="4">Tri-SOM</th><th colspan="4">DegeTri-SOM</th><th colspan="4">Tetra-SOM</th><th colspan="4">DegeTetra-SOM</th></tr><tr><th></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th><th colspan="4"><hr /></th></tr><tr><th></th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th><th colspan="2">Species</th><th colspan="2">Group</th></tr><tr><th></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th><th colspan="2"><hr /></th></tr><tr><th></th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th><th>CL</th><th>SQ</th></tr></thead><tbody><tr><td>1 kb SOM</td><td>43.9</td><td char=".">54.9</td><td>66.6</td><td char=".">72.5</td><td>45.6</td><td char=".">64.9</td><td>67.7</td><td char=".">78.1</td><td>50.6</td><td char=".">37.0</td><td>70.8</td><td char=".">61.2</td><td>55.4</td><td char=".">40.6</td><td>73.8</td><td char=".">69.2</td></tr><tr><td>5 kb SOM</td><td>87.4</td><td char=".">65.5</td><td>96.5</td><td char=".">79.1</td><td>92.3</td><td char=".">72.3</td><td>97.4</td><td char=".">81.5</td><td>80.6</td><td char=".">67.8</td><td>98.4</td><td char=".">84.9</td><td>95.3</td><td char=".">74.6</td><td>98.9</td><td char=".">88.0</td></tr><tr><td>1 kb on 5 kb SOM</td><td></td><td char=".">64.4</td><td></td><td char=".">78.1</td><td></td><td char=".">74.7</td><td></td><td char=".">83.3</td><td></td><td char=".">68.1</td><td></td><td char=".">80.2</td><td></td><td char=".">76.4</td><td></td><td char=".">84.6</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>CL, the proportion (%) of colored lattice points, which contained sequences from a single species or group; SQ, The proportion (%) of sequences belonging to the colored lattice points.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193415" target="_blank">View Large</a></div></div><p>SOM recognized in sequence fragments the key combinations of short oligonucleotide frequencies that are the signature feature of each genome<a reveal-id="b12  b15" data-open="b12  b15" class="link link-ref link-reveal xref-bibr">1215</a> and separated the sequences into species-specific territories without information regarding the species. In DNA databases, only one strand of a pair of complementary sequences is registered. Our previous analyses revealed that sequence fragments from a single prokaryotic genome are often split into two territories that reflect the transcriptional polarities of the genes present in the fragment.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> For phylogenetic classification of sequences from uncultured microbes, it is not necessary to know the transcriptional polarity of the sequence, and the split into two territories complicates assignment to species. Therefore, we tested a new type of SOM in which frequencies of a pair of complementary oligonucleotides (e.g. AACC and GGTT) were summed, and the SOMs for the degenerate sets of tri- and tetranucleotides were designated DegeTri- and DegeTetra-SOM, respectively. This nearly halved the computation time without loss of clustering power. The highest clustering power was observed for the 5 kb DegeTetra-SOM (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>), and therefore, this SOM is used for the later analysis of phylogenetic classification of novel sequences from environmental samples.</p>                            <h3 scrollto-destination=87193417 id="87193417" class="section-title" >3.2. Diagnostic oligonucleotides for species separations</h3>
<p>Analysis of the weight vectors of individual lattice points in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> revealed that vectors with strongly biased frequencies were located on the edge of the map. The G + C% calculated from each lattice vector was reflected in the horizontal axis and increased from left to right in the 5 kb DegeTetra-SOM (G + C% in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>). In other words, sequences with high G + C% (red), which were derived primarily from GC-rich genomes, were clustered on the right side of the map. To visualize the species-specific patterns of oligonucleotide frequencies recognized by the SOM, the frequency of each pair of complementary tetranucleotides at each lattice point was calculated, divided into five level categories with an equal number of lattice points, and visualized with the level of red or blue; seven examples of the diagnostic patterns of the species separations are presented in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>. Transitions between red (overrepresentation) and blue (underrepresentation) for various tetranucleotides often coincided with borders for species separations. To clarify the biological implications of diagnostic tetranucleotides for the species separations, we examined association between the levels of palindromic tetranucleotides and restriction enzyme systems. We could identify 24 four-base cutter enzymes for the 81 prokaryotes analyzed, as referred to the REBASE restriction enzyme database (<a class="link link-uri" href="http://rebase.neb.com/rebase/rebase.html" target="_blank"></a>). The restriction site tetranucleotides were underrepresented for 23 of 24 enzymes (refer to blue zones noted with light blue arrows in palindromic tetranucleotide panels in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Underrepresentation of the restriction site tetranucleotides in the genome with the enzyme gene may be related to the potential danger for self-damage even in the presence of methylation systems. This result showed that the SOM effectively classified sequences according to biological categories, and a complex combination of many oligonucleotides should contribute to species separation. For example, wide varieties of oligonucleotide sequences are known to function as genetic signals (e.g. regulatory signals for gene expression), and some of the important signal sequences may be biased significantly toward non-random occurrence and presumably be diagnostic for species separation.</p>                        <a id="87193419" scrollto-destination="87193419"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f2.jpeg?Expires=2147483647&Signature=XTSkqXj3ad27~g67CCN9iyMXiGNcMmJnXayeGlGBMC1vH5dSOfLulrkfVpmidPMSB6i8KwIfz0lAlTjPd1vdBpR18ePss1nrfZGe3yCsKYyRLZSQztlPFGJRc~Gn~f5xrU-EvbueYeIRb6OFJ~owz~O6kYTDzPE~nGxlgNy8ag4PV7HPUYG8pzv-8Oph5Q2ZoygCPI7ozOnR8GTHthMFV7dgCfU7KFROqw-BSp3U87udOnAaVuShaFqoM~qQQ31hBiKiKogYIQxYfMUAHcePGO~PPIaqWt62tWx1iqcuJ6WNrC9m-tvmTy6cVVb1a6ZIwyOnxjcx9q62AEYKAiwbpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotides related with species separation. The 5 kb DegeTetra-SOM presented in Fig. 1B. G + C%: G + C% calculated from the weight vector of individual lattice points in the 5 kb DegeTetra-SOM were divided into five level categories with an equal number of lattice points; and lattice points belonging to the categories of the highest, second-highest, middle, second-lowest and lowest G + C% are shown in dark red, light red, white, light blue and dark blue, respectively. Seven examples of diagnostic tetranucleotides for species separation are presented; five represent palindromic cases. Levels of each pair of complementary tetranucleotides calculated from individual lattice vectors were divided into five categories containing an equal number of lattice points, and the categories are shown as described above. Zones for species that have genes encoding a restriction enzyme that recognizes the respective tetranucleotide are noted by light blue arrows with the following numbers to show the species name: 1, H. pylori; 2, M. jannaschii; 3, F. nucleatum; 4, S. aureus; 5, L. lactis; 6, S. pneumoniae; 7, N. meningitidis." path-from-xml="dsi015f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193419/dsi015f2.jpeg" path-from-xml="dsi015f2.jpeg" target="_blank">View large</a><a section="87193419" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f2.jpeg?Expires=2147483647&Signature=KNa9IU8wY76On4IaIeIC5osLeOvwtsiLQcBNCnmAoVCbunEqJFfbAZGtDRLdsbkLyOl4QMUuSzQ80LFIxCVwTBoljjuzi9dVmax9dKu9QR~zEpbQg-mHI2lZb4eKWy~OGqZ90ptvaSP5ffMGPjzBqJTGSqRlldgztGnquFpV-WczejdFvQ9Wt9vqR0dh4Bce6h-jyJ8ouOqRPgQsKzGvyz6m-udXZVORJJJTd5Z1mYxt7Flw3Fw97mqwC66cQ5JU206Q-GLK0G8fdiNlUMbPvpwNs9BB9fvxny2OObExHZXMkN2ol7m-B-pz9nqti~XkxgGdwp4eb8xuKbJBNfBCEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193419&ar=350803&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi015f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotides related with species separation. The 5 kb DegeTetra-SOM presented in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a>. G + C%: G + C% calculated from the weight vector of individual lattice points in the 5 kb DegeTetra-SOM were divided into five level categories with an equal number of lattice points; and lattice points belonging to the categories of the highest, second-highest, middle, second-lowest and lowest G + C% are shown in dark red, light red, white, light blue and dark blue, respectively. Seven examples of diagnostic tetranucleotides for species separation are presented; five represent palindromic cases. Levels of each pair of complementary tetranucleotides calculated from individual lattice vectors were divided into five categories containing an equal number of lattice points, and the categories are shown as described above. Zones for species that have genes encoding a restriction enzyme that recognizes the respective tetranucleotide are noted by light blue arrows with the following numbers to show the species name: 1, <em>H. pylori</em>; 2, <em>M. jannaschii</em>; 3, <em>F. nucleatum</em>; 4, <em>S. aureus</em>; 5, <em>L. lactis</em>; 6, <em>S. pneumoniae</em>; 7, <em>N. meningitidis</em>.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f2.jpeg?Expires=2147483647&Signature=XTSkqXj3ad27~g67CCN9iyMXiGNcMmJnXayeGlGBMC1vH5dSOfLulrkfVpmidPMSB6i8KwIfz0lAlTjPd1vdBpR18ePss1nrfZGe3yCsKYyRLZSQztlPFGJRc~Gn~f5xrU-EvbueYeIRb6OFJ~owz~O6kYTDzPE~nGxlgNy8ag4PV7HPUYG8pzv-8Oph5Q2ZoygCPI7ozOnR8GTHthMFV7dgCfU7KFROqw-BSp3U87udOnAaVuShaFqoM~qQQ31hBiKiKogYIQxYfMUAHcePGO~PPIaqWt62tWx1iqcuJ6WNrC9m-tvmTy6cVVb1a6ZIwyOnxjcx9q62AEYKAiwbpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotides related with species separation. The 5 kb DegeTetra-SOM presented in Fig. 1B. G + C%: G + C% calculated from the weight vector of individual lattice points in the 5 kb DegeTetra-SOM were divided into five level categories with an equal number of lattice points; and lattice points belonging to the categories of the highest, second-highest, middle, second-lowest and lowest G + C% are shown in dark red, light red, white, light blue and dark blue, respectively. Seven examples of diagnostic tetranucleotides for species separation are presented; five represent palindromic cases. Levels of each pair of complementary tetranucleotides calculated from individual lattice vectors were divided into five categories containing an equal number of lattice points, and the categories are shown as described above. Zones for species that have genes encoding a restriction enzyme that recognizes the respective tetranucleotide are noted by light blue arrows with the following numbers to show the species name: 1, H. pylori; 2, M. jannaschii; 3, F. nucleatum; 4, S. aureus; 5, L. lactis; 6, S. pneumoniae; 7, N. meningitidis." path-from-xml="dsi015f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193419/dsi015f2.jpeg" path-from-xml="dsi015f2.jpeg" target="_blank">View large</a><a section="87193419" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f2.jpeg?Expires=2147483647&Signature=KNa9IU8wY76On4IaIeIC5osLeOvwtsiLQcBNCnmAoVCbunEqJFfbAZGtDRLdsbkLyOl4QMUuSzQ80LFIxCVwTBoljjuzi9dVmax9dKu9QR~zEpbQg-mHI2lZb4eKWy~OGqZ90ptvaSP5ffMGPjzBqJTGSqRlldgztGnquFpV-WczejdFvQ9Wt9vqR0dh4Bce6h-jyJ8ouOqRPgQsKzGvyz6m-udXZVORJJJTd5Z1mYxt7Flw3Fw97mqwC66cQ5JU206Q-GLK0G8fdiNlUMbPvpwNs9BB9fvxny2OObExHZXMkN2ol7m-B-pz9nqti~XkxgGdwp4eb8xuKbJBNfBCEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193419&ar=350803&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi015f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotides related with species separation. The 5 kb DegeTetra-SOM presented in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a>. G + C%: G + C% calculated from the weight vector of individual lattice points in the 5 kb DegeTetra-SOM were divided into five level categories with an equal number of lattice points; and lattice points belonging to the categories of the highest, second-highest, middle, second-lowest and lowest G + C% are shown in dark red, light red, white, light blue and dark blue, respectively. Seven examples of diagnostic tetranucleotides for species separation are presented; five represent palindromic cases. Levels of each pair of complementary tetranucleotides calculated from individual lattice vectors were divided into five categories containing an equal number of lattice points, and the categories are shown as described above. Zones for species that have genes encoding a restriction enzyme that recognizes the respective tetranucleotide are noted by light blue arrows with the following numbers to show the species name: 1, <em>H. pylori</em>; 2, <em>M. jannaschii</em>; 3, <em>F. nucleatum</em>; 4, <em>S. aureus</em>; 5, <em>L. lactis</em>; 6, <em>S. pneumoniae</em>; 7, <em>N. meningitidis</em>.</p></div></div>                            <h3 scrollto-destination=87193420 id="87193420" class="section-title" >3.3. Phylogenetic classification of species-known sequences</h3>
<p>In the phylogenetic classification of unculturable and poorly characterized species, classification into phylotypes rather than into individual species is important. Therefore, focusing on the DegeTetra-SOMs in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>, we examined the classification of sequences from the 81 prokaryotes into 13 major phylotypes by referring to the DNA Data Bank of Japan (DDBJ) Taxonomy Database (<a class="link link-uri" href="http://sakura.ddbj.nig.ac.jp/uniTax.html" target="_blank"></a>). Lattice points that contained sequences from one phylotype are indicated in color, and those that included sequences from more than one phylotype are shown in black (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). The number of black points was lower than that of the species classification. On the 5 kb DegeTetra-SOM, 88% of the sequences were classified into the correct phylotype territory (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>), which was defined as a continuous territory represented by a single color for the phylotype. Species that showed close phylogenetic relations in one phylotype tended to have neighboring zones and produced a continuous territory in the phylotype classification. It should also be noted that different species belonging to one phylotype were often separated from each other. One cause of this separation was related to differences in genome G + C% between species of a single phylotype. For example, we observed a separation between GC- and AT-rich -proteobacteria (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i1.jpeg?Expires=2147483647&Signature=iLzwIhDSHbn8uhrYZFek5ADCytzPOdRGOqXFmVeVuSU85-fKtOFdmBd~oicNeiZtOo2iwKTeyolkErSh1r~3iTnEdFIUoxnyIpQSslvujrEBIVtv~DMTy6xRAKmWWqItWtchv77cbQByQymf3gQLi9hMB3DPX7u06wKI-IaUlke3E2t2jfAH-1xQqJGIeRhkJ6u5KYvCR9uLliMSiKi0ES9TYg53PMTshiOhPC6aqY646tXU9CF5-9e15Zk-iGAXUqbUaEFGt7tGSGea7qN~ZCKEpyy1xJgLJKAUEbTa6QSWCXhorOzXLQvSNj6QnqunycSdIy8ZqxkUYrD9xOJ2~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i1.jpeg" />).</p><p>It is worth noting that paired-end reads in DNA sequencing, such as two 500 nt sequences from one cloned fragment, can be used as a single 1 kb sequence in the calculation of oligonucleotide frequencies. When oligonucleotide frequencies were calculated and normalized for sequence length, 1 kb sequences could be mapped on the 5 kb SOM (1 kb on 5 kb SOM in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). The proportions of 1 kb sequences classified into the correct phylotype were 69 and 85% on the 1 and 5 kb DegeTetra-SOMs, respectively, indicating that for phylogenetic classification of species-unknown sequences, even short sequences (e.g. 1 kb) must be mapped on an SOM constructed with longer species-known sequences (e.g. 5 kb). The increased hit level in mapping on the 5 kb SOM is presumably because the SOM can extract species-specific characteristics of oligonucleotide frequencies more accurately and because the statistical fluctuation of oligonucleotide frequencies decreases as the analyzed sequences become longer.</p><p>For phylogenetic classification of novel sequences from environmental samples, SOMs should be constructed in advance with all available sequences of known species rather than only those from completely sequenced genomes. We thus constructed a DegeTetra-SOM with 210000 non-overlapping 5 kb sequences (a total of 1.05 Gb) from 1502 species-known prokaryotes for which at least 10 kb of sequence has been deposited in DDBJ/EMBL/GenBank. These 1502 prokaryotes were classified into 25 phylotypes with reference to the NCBI Taxonomy Database. The classification according to phylotype was apparent (Species-known Seq. in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>).</p>                        <a id="87193424" scrollto-destination="87193424"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f3.jpeg?Expires=2147483647&amp;Signature=o6XCSboAATOSh-BXUxAWwX6PtshWWxULGxy4T~eL5qqFbL-CQtSPBRIRzMfGcerIap7LokEG7eQnKUuv80C2LkKf2vFK2kMKz8Jmzj9cHEvXiM1hLWAfte2XCvwUi5Y4-Knrrvq0Jqt7k3w5ZPXbFUuXbhY1OOV5pPuUZsiF629Fi2uTal6ro3SuxqCvI3R4R4DLJtv7s9eqFPEK1esAFyk3BcTYkL8rzlNTKvjQ99DZX2HUugZ085hp9t0W5z0Zb~ea0zA4qC~6TVr0nKF~NN6DKSrM9ln165rjddObpSV7FefAf~sFLxoXAGPhiEAAZRXoLmUmF2A3MUJ7Ac-7lA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic classification of environmental sequences. (A) DegeTetra-SOM of non-overlapping 5 kb sequences of species-known prokaryotes. The genomic sequences from 1502 prokaryotes were used. Species-known Seq.: prokaryotic sequences were classified into 25 phylotypes. Lattice points that include sequences from more than one phylotype are indicated in black, and those that contain sequences from a single phylotype are indicated in color as follows: -proteobacteria (), -proteobacteria (), -proteobacteria (), -proteobacteria (), -proteobacteria (), Actinobacteria (), Aquificae (), Bacteroidetes (), Chlamydiae (), Chlorobi (), Chloroflexi (), Crenarchaeota (), Cyanobacteria (), Deinococcus-Thermus (), Dictyoglomi (), Euryarchaeota (), Fibrobacteres (), Firmicutes (), Fusobacteria (), Nitrospirae (), Planctomycetes (), Spirochaetales (), Thermodesulfobacteriales (), Thermotogales () and Verrucomicrobiae (). Sargasso Seq. &gt;5 kb and 1 kb: 1 kb sequence fragments derived from contigs longer than 5 and 1 kb were mapped on the 5 kb DegeTetra-SOM. The residual sequence segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show phylotype borders. The dominant genomes noted by Venter et al.11 are indicated by arrows with the following numbers to show the genus name: 1, Synechococcus; 2, Prochlorococcus; 3, 4, Shewanella; 5, 6, Burkholderia. All Sargasso Seq. 3D: all 811000 Sargasso sequences were mapped after normalization of the sequence length, and square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar. (B) DegeTetra-SOM of environmental and species-known sequences. SOM was constructed with 211000 5 kb prokaryote sequences plus 218000 Sargasso and 5000 biofilm 1 kb sequences after normalization of the sequence length. Sargasso and biofilm sequence entries longer than 1 kb were selected and divided into 1 kb fragments, and the residual segment &lt;1 kb was omitted from the analysis. Species-known plus environmental Seq.: lattice points that contain sequences from a single phylotype are indicated in color as described in Fig. 3A, those that contain only Sargasso or biofilm sequences are indicated in color () or (), and those that include environmental and species-known sequences or those from more than one known phylotype are indicated in black. Biofilm Seq.: square root of the number of biofilm sequences classified into each lattice point is indicated by the height of the bar distinctively colored to show the dominant species reported by Tyson et al.16: Ferroplasma (), Leptospirillum () and Thermoplasmatales (). Sargasso Seq. unclassified: square root of the number of Sargasso sequences classified into each lattice point containing no species-known sequences is indicated by the height of the bar indicated in color (). Sargasso Seq. classified: square root of the number of Sargasso sequences that were classified into lattice points containing species-known sequences from a single phylotype is indicated by the height of the bar distinctively colored to show the phylotype." path-from-xml="dsi015f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193424/dsi015f3.jpeg" path-from-xml="dsi015f3.jpeg" target="_blank">View large</a><a section="87193424" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f3.jpeg?Expires=2147483647&amp;Signature=VVS4lTJ-44wCICU-hxnZjXQM9O3A~1TsEOKGyJRoavWKzMbZc2vOpVfe2lMp1f6rVENxIkpR9Re9B8eIfTIq7we3QfKdu8asJOqQ3rTZljQvm~0EBGQgJ0h0hNLmucNPDaYxvm34-SgQOulDnGAX~0MdJ~CstK1OSTPD-GEyPOCFpkYo8yCemsEh1vMuLZAMhodPVHh3601W1Gct29wz1ta5MXB~Qz8BR3E22RB1AUIjv5-Htt7lAopHL2sDyj33cB8U3GyUo7hv6ISpT9T1KNQ96mdseBTj-DnAOIGkfPe5LlgWawU2e5euTDtbtOdLICLZ44YIJ6iWbjsXVt9uew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193424&amp;ar=350803&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi015f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic classification of environmental sequences. (<strong>A</strong>) DegeTetra-SOM of non-overlapping 5 kb sequences of species-known prokaryotes. The genomic sequences from 1502 prokaryotes were used. Species-known Seq.: prokaryotic sequences were classified into 25 phylotypes. Lattice points that include sequences from more than one phylotype are indicated in black, and those that contain sequences from a single phylotype are indicated in color as follows: -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i81.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i81.jpeg?Expires=2147483647&amp;Signature=fTD3R6nYY-RHvTCo1~RtqQgavtB6pAUoY1ycfGFf7-P2BHmk38CW5IPV8ycyDfOsiN4YOeZ9W1w22ZPRcIaqb058yEXQfs7asBB8ARPr0YGvN47NMzdfdQlA580y2~yrEZszoO2FrbiXUHjrbWOlbbpQK7YYXSc7T49BwjHUFKhSIuThWnqHL3enr8En3p8bsb8LR2jvfyO77Lp9OuYext2mNewn9Ut6mqy5HXAbGAGxNLE2ClyeC-EJ1xxjrOmRCnDoF70D8QvhJnnHwAUIKfLsoaLROGSQimACedvdwf-orvzBnP1GuYV8aJU~1IQa63g0fAjYAk5YYH0wxqPp1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i81.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i82.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i82.jpeg?Expires=2147483647&amp;Signature=qN5EFW4eqNb3vFNfxq587VH3KoHOh11CCvdW2MQT18QdbkXt5CGIasdodeuvt9lAZE9pjCGdHH5KBn2AxDpQY~exMwayKB9BgKhb3sBQzwBMweHhuyDVzZ0K8KYG7dA-Gn0RmdQgawmT8ltvK9IiBX8Un-~-gVde4J~to9dIlRPJcOTES0Huh9CSigvdNOShwHVoAo9SkzW~U0bzXMY1jCbIrPALuZb9fHpx6J6xkgA0J8LFtf~ubAGWNSN1ACQlhWYwbqRYQ9z4RjtiEq2pfyK35pVc~t-YKMNo3iGgsyhUQPAgnVLpbXtml53ANxvda6oAm2aQ4uSfTGDYXwb9kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i82.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i83.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i83.jpeg?Expires=2147483647&amp;Signature=A08qbgw5a0fEuFhfapU2GU2WpyHyDcLLiMgyN6grSRYrfDMmEfMID6U9JLWjGc4o8vryXaOSyu8wyZPBzYzQJFcpNlQZI7QDzOTrCT7lK3TrR-pDSB7igjBpfGWYfwWkTXgNOlG5Cg10Y8Xx9UpEZvFCysv6~A3Ej4pSFJKEBaSoLklA37tn~u9lOW5LSGvCL3FDPoGZpxMvf4s7-~v4sYc2dknK3zst3B5NHNgmlZjL6nq3aVLRUw5u4Zcrv9SPV6ZQmBsJ4cdW8IZvlQhj3aqD8LV0EPiTCX01EuYcNQndtVNP5zw637uPcLd026vFmPg~g1~cN~3QJ~9d744-PQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i83.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i84.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i84.jpeg?Expires=2147483647&amp;Signature=uUsJz99at37oP6~SNzee702E69ojM5kV95i5OsOXBOZBLn8jxrQCw4Pxsw3GyxCc6ytjAu8b9rlVqWSoUtRR0IGCE1MBR7KRUMFTZHCNIBHZR2wpIFpkcCSsH2kBRoO~vsIKyDmgJw3PlPvX3jIqUUpBgFV7G8jyjm7AlKaf26Dz8YTOVWuTe3FKnf6if2ykjxRYVHb~ncW~cs0BojO2ze8AOtRjBscYfiWednJtbwz1dZrBl06JKyBW6hVJNyOSvLyJBD9DQD902uv7YCf-WI5d~e7L~5hdbiEAfHcPOeUaYYdJGygIgTo4r8-xk5ly2W8J27XOdIaRPv5y7VBDTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i84.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i85.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i85.jpeg?Expires=2147483647&amp;Signature=EdromkVHDrhN9SxY7UjX6dfecJnjdsqrGNEs6eEI9Es-pOUbfmgGLaXdG4-aEcK2S-aqmsGWFPUIXJM~XiL9Zi9je3lI2ubBDznvzcSMG4UjswJBB~vFgi6QWqh6MVtHK6J20XBvl9Gpze5pPbmcskaxImpYzkbw6fxiMZALnz-JjjOkLE2Q0mYnJWjTOn-qLcj0uokt1fkZM6EPZHZD9hlNF7A6ummpDw1ZpjUF2s8LLPIb9q8fLGxWasS7lJoWNyUr6b-owpEpR~i27t8aZqc1ed5cWIUefgDSH8Mx7J7YDRrRcG8XwO9wAg~rMD6SRFeeMPcEJ-r8F1kSHcZ78Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i85.jpeg" /></a></div>), Actinobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i86.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i86.jpeg?Expires=2147483647&amp;Signature=5G4meJe-fOSuh1dmNMx7ofNS8y-ExkHmRl12WribUJOH35ITYIhSBsMcObMEDeTEKvPv4BXPzfLpWARD1BRysuKIued95KJIYBhJWl5YvnxiUAyOENPLdm1B7~eT~6CakU4OsXlslP8t6yZQB1vDVKU-NawiLosdhy9KWO1mcyJNqyzBSrMDROuwbivYlpSU8nOYV4emoSK6goYZyVJdXapSFHFy6nSQJVX6xTNH~w23c1glHx2lxMBoIv9eC6hjrezGR5uhxIkj~v8J2nvlu1zW26olGezBuoOHn0KjPU4YToadnCPqeJ7AFuMcECXBDicbFa2F5k6tOFIxaIrPbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i86.jpeg" /></a></div>), Aquificae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i87.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i87.jpeg?Expires=2147483647&amp;Signature=0Pia8Pq5xOLdQLfb8B7GvZV~9zv1Q41D7JjDYGM5c3eEsvatXGc-GoHt-daAA9cAoF4MTZb6NRiwYFztq2XHk2fktQ3HWwCtFpBtb-AnHo7tK~3N4~n9kiTyil8M54Z28yYEHPwI-lVpSBMmY71uJayvVIG0joemlzx56gaWQgO0nMwqXipz1iuKU52vB3iN5MS2odAwEr5avZQaoxKYIG5KyO-B4Vgqu9Lg7KXJXFzh9m4AVfipFHhWxWBBLWlwTr-uU-LccYsib5EGyMVTNX-55TWse1CNtQRnnSVzJYZL8CE37qDHMj8X5qIKZRF7SYjm4kTnIbP~KI-TUEqp6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i87.jpeg" /></a></div>), Bacteroidetes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i88.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i88.jpeg?Expires=2147483647&amp;Signature=5GQx0uX5kB-o4AvfGKb5KsA5iHurIgJzxzxnYwG4YJmipIDWi5XAFnrKS4jBdFwKnGXUNSD66s~d2mY4viZpABCKwabmODtKyu-jFSUTgrbY6cekmDHNtYuDa1JW9qy~LqM7qGygxCLyMXiBqOMGOJ85kHkfjL3XxcEmBQ8B3AZ3o-fFCVo2Q2E89EfLvVmQKReb7GVjpZpajP8TLbxUYSJ6DCen8pTwhpZuqFS0ZywUs0E5IEQ715qNgT5rboXHFRyo-f1KjkH6GK2L7w87z7kkK5Vdxp6oIVq1KjovYneJpVVUKPVFSrTfWPJpILzvxc7Oato7Owv-gFyuThmNMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i88.jpeg" /></a></div>), Chlamydiae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i89.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i89.jpeg?Expires=2147483647&amp;Signature=lZTdIlcnbV4YFXaqWwX~zzJ8e-X-xs8p8PN-4bM6ddHyum09TEjmjfvAMzajmPm9JNCNR-xnkeTC27VHvcCVr1x9F42GKZoV~DcogH7yIWwaxXFfm2CMpLzoVt5-OM3JUrV45XEBLjqjlVxGCCsjSO1RePCI~DePJM3YWx4L0FfdQNgyGhwxBgL7UrdBVk65I4IQA4FAZ7BInzHXa9Fj3lfqcZhQxtS5PWv06j1zzAy7HQzyH1uiJoN38mCcAg4UYpqlsGYIUhy8b2S4ZZ17SNPqJKmtRyl0PTWLYTwqa38TqouFC4bnEQeP5iV4bqXKZt3kYgSThwwat-fuVRVNxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i89.jpeg" /></a></div>), Chlorobi (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i90.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i90.jpeg?Expires=2147483647&amp;Signature=al3T18d8VpsUz62~YsETBf8qS7Sosc5gxlVdtE70IOodAjsJVB6AoyCF9Bpx6VABmiF9xA4f~g-myRwr5kNq9gSXKfhEq0tEYGfC0qPFZQKxrV~x~URisINlTx8lbbULtmdl9MvXwYidNkJbY0EZky2Od-978o-P-KeFdJAzwDKVci6tGH~cOYl09c9PpCtykKlD6EFJ3FzNUYarCQ~HK5ITZ6vqA2dMPzVX4r9~QYqXgUtymhdeSduamfkUzZjEeaiHJMf35EDGJJXUt06dIhjoaWe6HuVMm294UC5u8pVufMoFqL6Fh7D269owmWpD7pAZ4zuHkLcFAFc9UV9SNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i90.jpeg" /></a></div>), Chloroflexi (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i91.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i91.jpeg?Expires=2147483647&amp;Signature=C0DFHpUFr8rrCNFT0LPjjKsgoTL7n6qxmTwVmHL5KVdAmmTUZTDhVm5UkeiN0nsfrTSQDYnb5LkYdRfDqD7-roQfkZ7cBOOKUrEfr1RkjG5~ifrIKihCQKmvWMutM62Dw8ZMdTwddd1T9~7Ez9JDCWQXEMWhAUpi4i5DiSu0F-5EW431pK3e~PaHZHhTAG4DGcPqimKgHTJr4NxebEUGv7GfqaCl9JJjsAQx8~Ycl6GbUl052Fus3DFYU8gHYdIIQ5e0qBWVKgOSFZNr5FYJEzt4mxACUQ7Nqp9olB9mtNijW0go3zbriHmjBOfe74EKrgC70OPdrBO1YPW6yjQilA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i91.jpeg" /></a></div>), Crenarchaeota (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i92.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i92.jpeg?Expires=2147483647&amp;Signature=DHwBclSbc~iS8uLNiAcdDhDEkTzz9Lvbx-l8GoGDALy~9Rj29og5MZ8cNrPz3aaSEeGkjqOPvPeIgoaL0P0dZhDQVW-Sv606RIPMxI0Olc2xYdtCAwV0y-1IGRjrck1iqpZbKp8s2Q50MfBH-1rWKD8djYIyCbpfW~scwbJHyCZrgRbJ9aGHnq~LrmGCfACivWmIe5PKYajGtZqYm-y2rMo1GASj6t9cSi3S-GykIvJD~VUl2ecXLxtCaLKw5r4bVR-FoxLWDM3QllWiLgBsvV2sjQMy4TevOUweJIg5-E6NKwuMWD38RohC4BZMf6KKFy78GWp27bG98drC5XBSmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i92.jpeg" /></a></div>), Cyanobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i93.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i93.jpeg?Expires=2147483647&amp;Signature=KAhO3dP0A~ajuefCc~1A~DdTJhr8VTqdKQuWPkc0hkwoCrOcOf~mO29BZZV0uQpCz8pfWtrO34DW23C7931Nc0DkED9tBLJJ8gHKoJFRgezCmHB~RqH71PRIjWs~Tbrz19DWf0EcgPzMYIsLoBkJbvZXYVUEZ6O5iIlCvkYQzUr5sxNEXlQbEtxlVjG7tfBVI6qe~Pb1Q49o7-6WLBfNATgrKVzwRM9iej1XSDLzJgxvw2ayxtrX-uNKZury~cRCoAFXXNmCaLILZMvlXzYgOJJoX2-e6uoZ2bfb61kFsS2sXZLR2jEKSg2oH60QZGcuNmU4s6XJPIJlmFpNfytWfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i93.jpeg" /></a></div>), Deinococcus-Thermus (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i94.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i94.jpeg?Expires=2147483647&amp;Signature=X~3TE4g2Lt5WWq3vCAzwzw3EDt11ryJWm~~to66ZgJQ2OPqIZ0msSUTfot7kYXxEmfEq77RajASuhNCkEKcMWjGfJv25S3gh9FNvDQlwxKUD8kGTlH2uMu14XqYaIKgXoQz9WrEB6e98KMMdhC7ZzeSZPWgobi1X7alAloKXyueyOqG-dtGgy48-6vpaRwHiCUE1V04xE-3ih5c5CrHyN8iEoU7Y75c7KzoTYeZtpx2yuLMR4RLz9kt7Z-jC-yTp6Chrtb~BFxq553WZ1rAJmBEyjdkzl2NVCL1Af8-vkCsYigC8mpNECmTf120sRwNkAC2FeSJ8lLRYD60m04X5Ow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i94.jpeg" /></a></div>), Dictyoglomi (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i95.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i95.jpeg?Expires=2147483647&amp;Signature=278h0Ct0CQxjHtL5IfjBgZ6oVnxtJ38PJqbOICSEK0dCzAugMTo8LuPtEL0FMXsdSatY4U5f2HxZbnApfVz-vjlvCTzaGU7Vsu9GuTU5zMH3-H5G7HvUqcl-0xlz~e3NFinX8Ij~ln1Tz4l2yc83TVVml5vaPJ3Dj8yALIQaG48JP-GN~FOFFF3nPljoeZ5HNbOXvxy41Ttpawaf4JYe86xvT5oxnIouVjGk9DTvntvoNv8wWCumpCQ7QcpqcrkvwtcZBegScFhlR3-p6ww5m~Rgxz5Zc~fwwJmWIkl1MNPZJ3n2dLwKNqx3njyPXNCorY0RlGGYmjtz3VsDfZEVpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i95.jpeg" /></a></div>), Euryarchaeota (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i96.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i96.jpeg?Expires=2147483647&amp;Signature=JUkVj-DUChCbNDMsqqqFVqt~4qBXPk-G6j3Leu8Tnl4j5QDr2I8dc49kLGqr3729LeTl7EPSwUGRHNnvOnzsuW1fk3TvlFWP1mAezoF7HBZn0vVTHRG4QxY2s0Fe2BSqGzPusbShwy0yekGCaUB3~x~-DkGt-FYW~JwDOrximSbyuz2rvN1xKG8CddQdIKr1rGY0z-bOLmaNIP0i81S-8Kp-lM-NPiaUrSfzgsSO1zmflBAW0IZ9UblXxLASZOdAlAlBh2gmAVvbOXar7QkA7OLBl1BxECm7h0JKk~uqWCWb02HX6~gyRSaMrHaiCuLQTq30RPyluE5lUDxSogOwKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i96.jpeg" /></a></div>), Fibrobacteres (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i97.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i97.jpeg?Expires=2147483647&amp;Signature=n8e72YuHeB9FaED14Bo8Rxwc0XV8FQtXLUtYYU7XOlQw9byD7IXKcreKfLpqpLH00M1llnhWiM48740tYBmJY2tqFzzlwVNtZNaVut7IWD8pAX6RRYil-ylp1TcsyIJxzLAwIHAfFmO80Fg-w6z2dXgTnW-2Z1uGLWm2J8EsLTEFnYjebPYYai9hNYyRxjWYQ7eY-0N~NyxWTGBEPWrwaqoXK1FKySQ0iauL8Oih~hMoxX68yLBrHx7wPHq9LQnaQGQmA66aajeS50Af7czJvCL4uxUK3j6LdbjRGefav12aWg5RrkhhwbEgpYVFgoNmipEkyIs5C7TOxpfWJ29kTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i97.jpeg" /></a></div>), Firmicutes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i98.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i98.jpeg?Expires=2147483647&amp;Signature=FFf1kwridAGU-1nX7xtLNx2iFPx~4Iq9mj~iBXb5cnOTmtO4g-CBi7n0hD3t87-hgO~Qk6lv~8WYVgHvuWBuLzb8n2cCq9MZYOnXB3bX1jsEfczGdeiBTYKil3eX1wzIHKgh7dHgvGjxwjVqY8Bve-PmT8izfxw0~8W-A90RvpO6niSQ18YJrepgRKasXeIfWC93qniXofOMn2~PzdnTBNz7MJ~M9V29fiMO3wWj-a5GLpvR3IqVUWfY6x4H-DLVZDxANr8-S3F4fKLXBlnBkXCHKpGHsFB0oK6DGUD~Juy8TMP79iExp3gErlAEa01ankXS~mDPg8vsHgo1cXm4Gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i98.jpeg" /></a></div>), Fusobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i99.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i99.jpeg?Expires=2147483647&amp;Signature=2PqjAztjUgZzK-NawslXfqVdemFAlAaZ~0QAlcvMlwnR-~wZXmrLnySut-K3A2InJ45jsvodrpQr2EmMdJ90rQ9qLKiKe3t1Ls7r8AZM6vIIZp-f3--K6nFpcPreowi6MSuVDd7D0P-Q0SH9~QzZ-5JgUPxlyKCA~ffC4-8pv7TyBYL~uhsVvxM6tQNqFt2~b5-P9MmVnDJxnxKu9~3~~zp8l0bsVVOmZJSc0EPL1c2GeYu5YLhLXeTXmq17TbulfP13NTlU~F5gjw38Cv9N7j6QcIxYfS5eHBgb4rXMCwo98CaftyUiotoEugWpjpLLpybyd3Guikg6ch86tOT-1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i99.jpeg" /></a></div>), Nitrospirae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i100.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i100.jpeg?Expires=2147483647&amp;Signature=I0twKWxK9CWQNh7cae1e1s9Vjlz0Br4cCwmgVONT3Y3nS~LxpOnpw3XPSF9c3cLVr7vNrKxGd-0DfjMwR970osCQf9YJXJjEkWfIb9ln5dz0GmX1mLoDi2C2WdJA0WWWL5ot-O7Wk~OXbw0qXDE80edWUEnJ25HVrjUYJaSsN2POBg7xh28atRWHy-g4ji7M5FqlxT0q10NJtuJw4Vic6S56GnWun230jsZTP7LFK3wWRqFldXg~iLzizDC5-qYP7ER0giRPkXPt7Tkq7LDZdTVlNw3vIxjP~iRTYfYREHpDxsvQaXwp65ol~FGCimUmVfCd0mWmgRZWV-f12A2TWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i100.jpeg" /></a></div>), Planctomycetes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i101.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i101.jpeg?Expires=2147483647&amp;Signature=B31jtVZkSShtMPXR1Dx~6XXLzpYtUuu8kySjOECU7hIDou0YAS0hz0FhPTPyyRwjc2g00ZjTlVu5Trc2-0xvTHgjdZW7DDNPkpAVhvb4S0IBIh6XuVcPxZ81joU4Q4Z3Od3-ucIhmmAfU~hRak9JlAmvn~cmA-UiyVcO8C82qsjnFpUbFY4MvVLcmZnfRSxM94Qw3xFlt~jEzvQoufyCDJEscUfYDTtxrM4svtR1bFcLwhKgoWc5vQtXlxWLvkyQslbl4aiDQJtwLsfPbl7C5c9x~d55qpwMHGwQ~1ucJNMPCQOdPRaKTwC18VEIjyYYB35G8ujq02Z~kcIBPErKnw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i101.jpeg" /></a></div>), Spirochaetales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i102.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i102.jpeg?Expires=2147483647&amp;Signature=1KgyN75BAj~zT0coMvKewsT7ZYZW~jzKItl0302WcNr0RhaoMOU501XK50G4GhH2C-sAWCLoNoK7h5vm~9JpJGyXZwNze6JHSmbg0AU-5TmFyiF5sQbjl1U8aLT302rQHQoC1FHeiB6jcTdRCxvixPgtETjxcd0fUld4jjgxODpbgbNuAmN1Hfl2EiptyzyqOgeM8Nm0CZ32ZRy7x1Dy2u2Iq8Iw~4HrLdyVLptb3J~mt6iTquuP3qHSOED9tW9Wk0Mm8XdHFFtZFSk9GCJwcUKNoXbEYheQaSD7QJ37Fd9HJBOle-JOBQGH82x-A3bYF4V0A1urOBwwvMkenZGF9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i102.jpeg" /></a></div>), Thermodesulfobacteriales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i103.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i103.jpeg?Expires=2147483647&amp;Signature=P4WVmLELeKHiNwbELs56N5JRAko4FmK3hoin0rBduQ5i4HRa5Q-EcX26hQG4FJTHLaAh44ad66RKYAdnxlwUyzglNXjenCoU8kcolLvF5ccgIgWZW4jQeY5jDkyUZBwrow39ryojnMPSlT9YLC6slFegWLhjlc7lbleAOKHkxtKcBLffbXE56P69QSYVnxji9oVgE2jUXf-nawZtgnXQ7T5DZgqzcrnyiLtfkuQRLS8DxAl88jsiMadhS6DPFe8fNKrAU2Dz3N2WcH70GtG4Qq0BM00g1bLhMW~-8WQnVsLFM9JNggZYUOKFQNuBEOdmlef066T5X2wDcEY~Z8dPBQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i103.jpeg" /></a></div>), Thermotogales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i104.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i104.jpeg?Expires=2147483647&amp;Signature=K5Bql6hoWdq7DsFsEJdhOoLJoX85bDNyRIjlahgMO2EH8OcwQVmtOts70FaGhq0W5twzIL88LLgAgbXZDt9Z64pZBCEijajgzE2HsMsqnkFi2K9X6nLWkEnHfCmiwq~DpbmL-dcVsd46QgEUBwz21KDtP9~OQVy1Vb0vIN6mGzFvpmePdK~EJdqF0Xr5b5G8irrAgrqv1M8zQlqLZBdu-T0UrT22yBHdbV21whh7H9nE7T673x7RV2xIMANOu3VtVK5P9NkZVEumvM47dno4Gj~FVWEqYhAw3LcPvIgHYz1rpeB3zOv9~FfHxzuvRBdSJCsGTgIRLFrnALXbW3jQFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i104.jpeg" /></a></div>) and Verrucomicrobiae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i105.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i105.jpeg?Expires=2147483647&amp;Signature=uRfHCL0t72mvGxHrErmeEQ5qSwy682ka2cpfaEv96VW1VTlK~3uAs7ICZXoPK-I0WIUrXKGbFiZRfgHCS7~OjsefTPoQWDCPOem4zx2BMncJVXTpQ2U5sm6zKXbNxbrmllZgiLv8OtKygyMxHUVZAsG9ZnpOV-3eH-WejWhBgzkJ-n-i16Zgc3byvDlvozUoUGzboNBqS6aA7ZBSizHk76~1pU5IiM7GphYn-Ebe6nrEVEwL6ma~EqO~-F5QQ6kejuvrRpqknVqDq623pQtEicmocXhgEEuLP8CHl85-BtcN6XvwUF2D8WG0nMLAiCb7EdKdJtgJ371FFQjg4eiV7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i105.jpeg" /></a></div>). Sargasso Seq. &gt;5 kb and 1 kb: 1 kb sequence fragments derived from contigs longer than 5 and 1 kb were mapped on the 5 kb DegeTetra-SOM. The residual sequence segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show phylotype borders. The dominant genomes noted by Venter et al.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> are indicated by arrows with the following numbers to show the genus name: 1, <em>Synechococcus</em>; 2, <em>Prochlorococcus</em>; 3, 4, <em>Shewanella</em>; 5, 6, <em>Burkholderia</em>. All Sargasso Seq. 3D: all 811000 Sargasso sequences were mapped after normalization of the sequence length, and square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar. (<strong>B</strong>) DegeTetra-SOM of environmental and species-known sequences. SOM was constructed with 211000 5 kb prokaryote sequences plus 218000 Sargasso and 5000 biofilm 1 kb sequences after normalization of the sequence length. Sargasso and biofilm sequence entries longer than 1 kb were selected and divided into 1 kb fragments, and the residual segment &lt;1 kb was omitted from the analysis. Species-known plus environmental Seq.: lattice points that contain sequences from a single phylotype are indicated in color as described in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, those that contain only Sargasso or biofilm sequences are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i106.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i106.jpeg?Expires=2147483647&amp;Signature=GICa94IBX3oLeTGmL0Yk9CKmxhFwGQ8~9RCzPMDf9R6fM3g54ipgERaeFhJIo1vS75b3U5arDO9xpdyMTalDXG~ia2sj1brxz3CeoBpNZneFF9ZU74F2ZZrJQWh3fway9fNwpytaZji7oE-lfy0tvUsqFMTmZrmos7f34Q2eYn9lSQD7vS564u4FIckbVOx79PDOHWz93yWVt5gOdXQNnxENeU8HWFHqB8URHJG0TgRQZDWwkowyOuQK0a6sBTd21u1VemT-fGsSxYG4pvo8GuxxKxbB4P6~WhDwgmPqyCWmp3B8OVqqlBm4Q9uTu2uMIaULLTLC-IOcdywBy~SZYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i106.jpeg" /></a></div>) or (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i107.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i107.jpeg?Expires=2147483647&amp;Signature=QKX~fq1zhe0X9mkzzhWL8p~L9IkWM4aNCLfWeX1moUx0XyBaE17SiApZbb0gxX1HOmtrJFdnlwDWEYE8Qy~5sR69KlQ9hYD7r-RfNAiYnFB9umZ4Bn52WJ5IyMTjGekiwHe5WXM5A2Ce~sraZvPUTEpstdlYBr2JnnTnKdygMw0lQmJ8AXgsJOfNW0cKQhM6vOsrvtuGYUqZ7LbcQMz9-FiEKpR~9fJYPxA-SOfLRM47DaBGl5t43FbkT59Bh0szKwpUTqRiJE5WjrhWdeTn9gfYwvwrrBKf-bSDaFInc58Totbbkpj7QET3vZCKc09mqZwUfJSZKC993hUJRKkr3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i107.jpeg" /></a></div>), and those that include environmental and species-known sequences or those from more than one known phylotype are indicated in black. Biofilm Seq.: square root of the number of biofilm sequences classified into each lattice point is indicated by the height of the bar distinctively colored to show the dominant species reported by Tyson et al.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>: <em>Ferroplasma</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i108.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i108.jpeg?Expires=2147483647&amp;Signature=q-PMmj1zFTqnOEy6ND4O7hBow9Zp~CGLTHzXrFQ8Qq5dIxL63LsheYGUkV2DC85FIBLZlu32t1mP7FeKbXkqphezZF5zPV1hKhMKIeHWg5JAS9gzRcoPG-SDOJvI~c8bvMnQQhyz53WzwChyfjRcDVumy5umchfYHoGYViUX-CSXV~0lsqmYE0q3b3App0iTEX7xWCuYNvPIYTLosjhTrbwWobpjncK6Z2wZ7xD9AwYXuyPbLiBJEjNwo5fcB9IKFtpZEHipaxXvc00TH2dM-RpxesVYiL1yLulB~5K-CB5n7rCNEfLfRU8469mlKBIxHx49Esuc~IXnjxV8a9Wkbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i108.jpeg" /></a></div>), <em>Leptospirillum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i109.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i109.jpeg?Expires=2147483647&amp;Signature=U9WisvFGAoYN0LI8SJnNeh2aoJS0l2NrH1NzxowZ--AlBgu-pEXfbG5PgDg2VAuqGIDZmW7FEPaz2P4HW0I1P8W-86kg6oZQ9hF1RF3s5CEy1wSEv3o8-a-zb0M5imASIrkr6C1FAsCOV3liVtNULAi0ezvbo-jLObFuycmUiXHoughMn4qmbJF28VdsMfmHoxVMq9Lgn~w4HsksGnEt7U4a7Np2KEZOO-usH6fhociyuy2FkIeYu123qaLnXt1v-48nHJeW2BTNOSTgv1MUAYuYUhmbT6OJEPRjpJ1RIY6xg56E~jOo08Q2g5JMwrlnKoTKGmZOCB-ME1iXeyjHFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i109.jpeg" /></a></div>) and <em>Thermoplasmatales</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i110.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i110.jpeg?Expires=2147483647&amp;Signature=HmrYcStfufp9Vk4lXi3DGgEG0HQcuY2uu-hINEK8SP~bygScCSb12R9QmpZV~dPQOlLNNfcRNPDcEX6m5~JTMk5lupkCjBY~7QOcMNu2xPqphk7K68TPe-9lqxs1R47qFPeKIDdzilt47UR4sXzDlfmmsS9ANKII6LDdp7eAPWrzvNY6b0qUWCfOa340ha9LWQUFkECUKQXjSCxdIKdcM1jbq~vUa-O9~CIEZ5pWGK690LjFKRloJAsv-Ktc0dbAcV1LuxWuL9q2VPcIPhW6FWSyiEB6X6xhoDb5FQ~VFT45aUsu6Mgx2giAji2AV8TldEuWOa4y~1RRP5VGj2yafQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i110.jpeg" /></a></div>). Sargasso Seq. unclassified: square root of the number of Sargasso sequences classified into each lattice point containing no species-known sequences is indicated by the height of the bar indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i111.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i111.jpeg?Expires=2147483647&amp;Signature=cgEIYKVuJgvMO~xIRQ0xr2jkNLes0Xja1i-Svk4KI-ClLujp3tYYwufsAKiGOZcidHegR0ybvo9a53qoyMMq8j~Am2QC2JNGgx8o0q8rDJc8DtNtABNBZsCmKnH0sfhrfF0casw2AUOGLZgpGyYcPZQ8Jt1mAL6Uq0gfmTWV6YwYMb6osm94jEwS7YxVJGeCkFOC0tjou2UtdbZ8-1Xz5zFw~pUrDKlFwCYk6cOsPn1R7F4LOFS6CwW5wuVSJFcMjUBj3UyuAQkV7hBFexeP3~Bh-RXSCCttomaSDlcKG9y7YXSFpJM1uDvyQK1bbFYicdavnMJMhpHRj~P553xSwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i111.jpeg" /></a></div>). Sargasso Seq. classified: square root of the number of Sargasso sequences that were classified into lattice points containing species-known sequences from a single phylotype is indicated by the height of the bar distinctively colored to show the phylotype.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f3.jpeg?Expires=2147483647&amp;Signature=o6XCSboAATOSh-BXUxAWwX6PtshWWxULGxy4T~eL5qqFbL-CQtSPBRIRzMfGcerIap7LokEG7eQnKUuv80C2LkKf2vFK2kMKz8Jmzj9cHEvXiM1hLWAfte2XCvwUi5Y4-Knrrvq0Jqt7k3w5ZPXbFUuXbhY1OOV5pPuUZsiF629Fi2uTal6ro3SuxqCvI3R4R4DLJtv7s9eqFPEK1esAFyk3BcTYkL8rzlNTKvjQ99DZX2HUugZ085hp9t0W5z0Zb~ea0zA4qC~6TVr0nKF~NN6DKSrM9ln165rjddObpSV7FefAf~sFLxoXAGPhiEAAZRXoLmUmF2A3MUJ7Ac-7lA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic classification of environmental sequences. (A) DegeTetra-SOM of non-overlapping 5 kb sequences of species-known prokaryotes. The genomic sequences from 1502 prokaryotes were used. Species-known Seq.: prokaryotic sequences were classified into 25 phylotypes. Lattice points that include sequences from more than one phylotype are indicated in black, and those that contain sequences from a single phylotype are indicated in color as follows: -proteobacteria (), -proteobacteria (), -proteobacteria (), -proteobacteria (), -proteobacteria (), Actinobacteria (), Aquificae (), Bacteroidetes (), Chlamydiae (), Chlorobi (), Chloroflexi (), Crenarchaeota (), Cyanobacteria (), Deinococcus-Thermus (), Dictyoglomi (), Euryarchaeota (), Fibrobacteres (), Firmicutes (), Fusobacteria (), Nitrospirae (), Planctomycetes (), Spirochaetales (), Thermodesulfobacteriales (), Thermotogales () and Verrucomicrobiae (). Sargasso Seq. &gt;5 kb and 1 kb: 1 kb sequence fragments derived from contigs longer than 5 and 1 kb were mapped on the 5 kb DegeTetra-SOM. The residual sequence segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show phylotype borders. The dominant genomes noted by Venter et al.11 are indicated by arrows with the following numbers to show the genus name: 1, Synechococcus; 2, Prochlorococcus; 3, 4, Shewanella; 5, 6, Burkholderia. All Sargasso Seq. 3D: all 811000 Sargasso sequences were mapped after normalization of the sequence length, and square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar. (B) DegeTetra-SOM of environmental and species-known sequences. SOM was constructed with 211000 5 kb prokaryote sequences plus 218000 Sargasso and 5000 biofilm 1 kb sequences after normalization of the sequence length. Sargasso and biofilm sequence entries longer than 1 kb were selected and divided into 1 kb fragments, and the residual segment &lt;1 kb was omitted from the analysis. Species-known plus environmental Seq.: lattice points that contain sequences from a single phylotype are indicated in color as described in Fig. 3A, those that contain only Sargasso or biofilm sequences are indicated in color () or (), and those that include environmental and species-known sequences or those from more than one known phylotype are indicated in black. Biofilm Seq.: square root of the number of biofilm sequences classified into each lattice point is indicated by the height of the bar distinctively colored to show the dominant species reported by Tyson et al.16: Ferroplasma (), Leptospirillum () and Thermoplasmatales (). Sargasso Seq. unclassified: square root of the number of Sargasso sequences classified into each lattice point containing no species-known sequences is indicated by the height of the bar indicated in color (). Sargasso Seq. classified: square root of the number of Sargasso sequences that were classified into lattice points containing species-known sequences from a single phylotype is indicated by the height of the bar distinctively colored to show the phylotype." path-from-xml="dsi015f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193424/dsi015f3.jpeg" path-from-xml="dsi015f3.jpeg" target="_blank">View large</a><a section="87193424" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f3.jpeg?Expires=2147483647&amp;Signature=VVS4lTJ-44wCICU-hxnZjXQM9O3A~1TsEOKGyJRoavWKzMbZc2vOpVfe2lMp1f6rVENxIkpR9Re9B8eIfTIq7we3QfKdu8asJOqQ3rTZljQvm~0EBGQgJ0h0hNLmucNPDaYxvm34-SgQOulDnGAX~0MdJ~CstK1OSTPD-GEyPOCFpkYo8yCemsEh1vMuLZAMhodPVHh3601W1Gct29wz1ta5MXB~Qz8BR3E22RB1AUIjv5-Htt7lAopHL2sDyj33cB8U3GyUo7hv6ISpT9T1KNQ96mdseBTj-DnAOIGkfPe5LlgWawU2e5euTDtbtOdLICLZ44YIJ6iWbjsXVt9uew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193424&amp;ar=350803&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi015f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic classification of environmental sequences. (<strong>A</strong>) DegeTetra-SOM of non-overlapping 5 kb sequences of species-known prokaryotes. The genomic sequences from 1502 prokaryotes were used. Species-known Seq.: prokaryotic sequences were classified into 25 phylotypes. Lattice points that include sequences from more than one phylotype are indicated in black, and those that contain sequences from a single phylotype are indicated in color as follows: -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i81.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i81.jpeg?Expires=2147483647&amp;Signature=fTD3R6nYY-RHvTCo1~RtqQgavtB6pAUoY1ycfGFf7-P2BHmk38CW5IPV8ycyDfOsiN4YOeZ9W1w22ZPRcIaqb058yEXQfs7asBB8ARPr0YGvN47NMzdfdQlA580y2~yrEZszoO2FrbiXUHjrbWOlbbpQK7YYXSc7T49BwjHUFKhSIuThWnqHL3enr8En3p8bsb8LR2jvfyO77Lp9OuYext2mNewn9Ut6mqy5HXAbGAGxNLE2ClyeC-EJ1xxjrOmRCnDoF70D8QvhJnnHwAUIKfLsoaLROGSQimACedvdwf-orvzBnP1GuYV8aJU~1IQa63g0fAjYAk5YYH0wxqPp1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i81.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i82.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i82.jpeg?Expires=2147483647&amp;Signature=qN5EFW4eqNb3vFNfxq587VH3KoHOh11CCvdW2MQT18QdbkXt5CGIasdodeuvt9lAZE9pjCGdHH5KBn2AxDpQY~exMwayKB9BgKhb3sBQzwBMweHhuyDVzZ0K8KYG7dA-Gn0RmdQgawmT8ltvK9IiBX8Un-~-gVde4J~to9dIlRPJcOTES0Huh9CSigvdNOShwHVoAo9SkzW~U0bzXMY1jCbIrPALuZb9fHpx6J6xkgA0J8LFtf~ubAGWNSN1ACQlhWYwbqRYQ9z4RjtiEq2pfyK35pVc~t-YKMNo3iGgsyhUQPAgnVLpbXtml53ANxvda6oAm2aQ4uSfTGDYXwb9kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i82.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i83.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i83.jpeg?Expires=2147483647&amp;Signature=A08qbgw5a0fEuFhfapU2GU2WpyHyDcLLiMgyN6grSRYrfDMmEfMID6U9JLWjGc4o8vryXaOSyu8wyZPBzYzQJFcpNlQZI7QDzOTrCT7lK3TrR-pDSB7igjBpfGWYfwWkTXgNOlG5Cg10Y8Xx9UpEZvFCysv6~A3Ej4pSFJKEBaSoLklA37tn~u9lOW5LSGvCL3FDPoGZpxMvf4s7-~v4sYc2dknK3zst3B5NHNgmlZjL6nq3aVLRUw5u4Zcrv9SPV6ZQmBsJ4cdW8IZvlQhj3aqD8LV0EPiTCX01EuYcNQndtVNP5zw637uPcLd026vFmPg~g1~cN~3QJ~9d744-PQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i83.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i84.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i84.jpeg?Expires=2147483647&amp;Signature=uUsJz99at37oP6~SNzee702E69ojM5kV95i5OsOXBOZBLn8jxrQCw4Pxsw3GyxCc6ytjAu8b9rlVqWSoUtRR0IGCE1MBR7KRUMFTZHCNIBHZR2wpIFpkcCSsH2kBRoO~vsIKyDmgJw3PlPvX3jIqUUpBgFV7G8jyjm7AlKaf26Dz8YTOVWuTe3FKnf6if2ykjxRYVHb~ncW~cs0BojO2ze8AOtRjBscYfiWednJtbwz1dZrBl06JKyBW6hVJNyOSvLyJBD9DQD902uv7YCf-WI5d~e7L~5hdbiEAfHcPOeUaYYdJGygIgTo4r8-xk5ly2W8J27XOdIaRPv5y7VBDTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i84.jpeg" /></a></div>), -proteobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i85.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i85.jpeg?Expires=2147483647&amp;Signature=EdromkVHDrhN9SxY7UjX6dfecJnjdsqrGNEs6eEI9Es-pOUbfmgGLaXdG4-aEcK2S-aqmsGWFPUIXJM~XiL9Zi9je3lI2ubBDznvzcSMG4UjswJBB~vFgi6QWqh6MVtHK6J20XBvl9Gpze5pPbmcskaxImpYzkbw6fxiMZALnz-JjjOkLE2Q0mYnJWjTOn-qLcj0uokt1fkZM6EPZHZD9hlNF7A6ummpDw1ZpjUF2s8LLPIb9q8fLGxWasS7lJoWNyUr6b-owpEpR~i27t8aZqc1ed5cWIUefgDSH8Mx7J7YDRrRcG8XwO9wAg~rMD6SRFeeMPcEJ-r8F1kSHcZ78Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i85.jpeg" /></a></div>), Actinobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i86.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i86.jpeg?Expires=2147483647&amp;Signature=5G4meJe-fOSuh1dmNMx7ofNS8y-ExkHmRl12WribUJOH35ITYIhSBsMcObMEDeTEKvPv4BXPzfLpWARD1BRysuKIued95KJIYBhJWl5YvnxiUAyOENPLdm1B7~eT~6CakU4OsXlslP8t6yZQB1vDVKU-NawiLosdhy9KWO1mcyJNqyzBSrMDROuwbivYlpSU8nOYV4emoSK6goYZyVJdXapSFHFy6nSQJVX6xTNH~w23c1glHx2lxMBoIv9eC6hjrezGR5uhxIkj~v8J2nvlu1zW26olGezBuoOHn0KjPU4YToadnCPqeJ7AFuMcECXBDicbFa2F5k6tOFIxaIrPbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i86.jpeg" /></a></div>), Aquificae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i87.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i87.jpeg?Expires=2147483647&amp;Signature=0Pia8Pq5xOLdQLfb8B7GvZV~9zv1Q41D7JjDYGM5c3eEsvatXGc-GoHt-daAA9cAoF4MTZb6NRiwYFztq2XHk2fktQ3HWwCtFpBtb-AnHo7tK~3N4~n9kiTyil8M54Z28yYEHPwI-lVpSBMmY71uJayvVIG0joemlzx56gaWQgO0nMwqXipz1iuKU52vB3iN5MS2odAwEr5avZQaoxKYIG5KyO-B4Vgqu9Lg7KXJXFzh9m4AVfipFHhWxWBBLWlwTr-uU-LccYsib5EGyMVTNX-55TWse1CNtQRnnSVzJYZL8CE37qDHMj8X5qIKZRF7SYjm4kTnIbP~KI-TUEqp6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i87.jpeg" /></a></div>), Bacteroidetes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i88.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i88.jpeg?Expires=2147483647&amp;Signature=5GQx0uX5kB-o4AvfGKb5KsA5iHurIgJzxzxnYwG4YJmipIDWi5XAFnrKS4jBdFwKnGXUNSD66s~d2mY4viZpABCKwabmODtKyu-jFSUTgrbY6cekmDHNtYuDa1JW9qy~LqM7qGygxCLyMXiBqOMGOJ85kHkfjL3XxcEmBQ8B3AZ3o-fFCVo2Q2E89EfLvVmQKReb7GVjpZpajP8TLbxUYSJ6DCen8pTwhpZuqFS0ZywUs0E5IEQ715qNgT5rboXHFRyo-f1KjkH6GK2L7w87z7kkK5Vdxp6oIVq1KjovYneJpVVUKPVFSrTfWPJpILzvxc7Oato7Owv-gFyuThmNMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i88.jpeg" /></a></div>), Chlamydiae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i89.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i89.jpeg?Expires=2147483647&amp;Signature=lZTdIlcnbV4YFXaqWwX~zzJ8e-X-xs8p8PN-4bM6ddHyum09TEjmjfvAMzajmPm9JNCNR-xnkeTC27VHvcCVr1x9F42GKZoV~DcogH7yIWwaxXFfm2CMpLzoVt5-OM3JUrV45XEBLjqjlVxGCCsjSO1RePCI~DePJM3YWx4L0FfdQNgyGhwxBgL7UrdBVk65I4IQA4FAZ7BInzHXa9Fj3lfqcZhQxtS5PWv06j1zzAy7HQzyH1uiJoN38mCcAg4UYpqlsGYIUhy8b2S4ZZ17SNPqJKmtRyl0PTWLYTwqa38TqouFC4bnEQeP5iV4bqXKZt3kYgSThwwat-fuVRVNxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i89.jpeg" /></a></div>), Chlorobi (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i90.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i90.jpeg?Expires=2147483647&amp;Signature=al3T18d8VpsUz62~YsETBf8qS7Sosc5gxlVdtE70IOodAjsJVB6AoyCF9Bpx6VABmiF9xA4f~g-myRwr5kNq9gSXKfhEq0tEYGfC0qPFZQKxrV~x~URisINlTx8lbbULtmdl9MvXwYidNkJbY0EZky2Od-978o-P-KeFdJAzwDKVci6tGH~cOYl09c9PpCtykKlD6EFJ3FzNUYarCQ~HK5ITZ6vqA2dMPzVX4r9~QYqXgUtymhdeSduamfkUzZjEeaiHJMf35EDGJJXUt06dIhjoaWe6HuVMm294UC5u8pVufMoFqL6Fh7D269owmWpD7pAZ4zuHkLcFAFc9UV9SNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i90.jpeg" /></a></div>), Chloroflexi (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i91.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i91.jpeg?Expires=2147483647&amp;Signature=C0DFHpUFr8rrCNFT0LPjjKsgoTL7n6qxmTwVmHL5KVdAmmTUZTDhVm5UkeiN0nsfrTSQDYnb5LkYdRfDqD7-roQfkZ7cBOOKUrEfr1RkjG5~ifrIKihCQKmvWMutM62Dw8ZMdTwddd1T9~7Ez9JDCWQXEMWhAUpi4i5DiSu0F-5EW431pK3e~PaHZHhTAG4DGcPqimKgHTJr4NxebEUGv7GfqaCl9JJjsAQx8~Ycl6GbUl052Fus3DFYU8gHYdIIQ5e0qBWVKgOSFZNr5FYJEzt4mxACUQ7Nqp9olB9mtNijW0go3zbriHmjBOfe74EKrgC70OPdrBO1YPW6yjQilA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i91.jpeg" /></a></div>), Crenarchaeota (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i92.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i92.jpeg?Expires=2147483647&amp;Signature=DHwBclSbc~iS8uLNiAcdDhDEkTzz9Lvbx-l8GoGDALy~9Rj29og5MZ8cNrPz3aaSEeGkjqOPvPeIgoaL0P0dZhDQVW-Sv606RIPMxI0Olc2xYdtCAwV0y-1IGRjrck1iqpZbKp8s2Q50MfBH-1rWKD8djYIyCbpfW~scwbJHyCZrgRbJ9aGHnq~LrmGCfACivWmIe5PKYajGtZqYm-y2rMo1GASj6t9cSi3S-GykIvJD~VUl2ecXLxtCaLKw5r4bVR-FoxLWDM3QllWiLgBsvV2sjQMy4TevOUweJIg5-E6NKwuMWD38RohC4BZMf6KKFy78GWp27bG98drC5XBSmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i92.jpeg" /></a></div>), Cyanobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i93.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i93.jpeg?Expires=2147483647&amp;Signature=KAhO3dP0A~ajuefCc~1A~DdTJhr8VTqdKQuWPkc0hkwoCrOcOf~mO29BZZV0uQpCz8pfWtrO34DW23C7931Nc0DkED9tBLJJ8gHKoJFRgezCmHB~RqH71PRIjWs~Tbrz19DWf0EcgPzMYIsLoBkJbvZXYVUEZ6O5iIlCvkYQzUr5sxNEXlQbEtxlVjG7tfBVI6qe~Pb1Q49o7-6WLBfNATgrKVzwRM9iej1XSDLzJgxvw2ayxtrX-uNKZury~cRCoAFXXNmCaLILZMvlXzYgOJJoX2-e6uoZ2bfb61kFsS2sXZLR2jEKSg2oH60QZGcuNmU4s6XJPIJlmFpNfytWfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i93.jpeg" /></a></div>), Deinococcus-Thermus (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i94.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i94.jpeg?Expires=2147483647&amp;Signature=X~3TE4g2Lt5WWq3vCAzwzw3EDt11ryJWm~~to66ZgJQ2OPqIZ0msSUTfot7kYXxEmfEq77RajASuhNCkEKcMWjGfJv25S3gh9FNvDQlwxKUD8kGTlH2uMu14XqYaIKgXoQz9WrEB6e98KMMdhC7ZzeSZPWgobi1X7alAloKXyueyOqG-dtGgy48-6vpaRwHiCUE1V04xE-3ih5c5CrHyN8iEoU7Y75c7KzoTYeZtpx2yuLMR4RLz9kt7Z-jC-yTp6Chrtb~BFxq553WZ1rAJmBEyjdkzl2NVCL1Af8-vkCsYigC8mpNECmTf120sRwNkAC2FeSJ8lLRYD60m04X5Ow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i94.jpeg" /></a></div>), Dictyoglomi (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i95.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i95.jpeg?Expires=2147483647&amp;Signature=278h0Ct0CQxjHtL5IfjBgZ6oVnxtJ38PJqbOICSEK0dCzAugMTo8LuPtEL0FMXsdSatY4U5f2HxZbnApfVz-vjlvCTzaGU7Vsu9GuTU5zMH3-H5G7HvUqcl-0xlz~e3NFinX8Ij~ln1Tz4l2yc83TVVml5vaPJ3Dj8yALIQaG48JP-GN~FOFFF3nPljoeZ5HNbOXvxy41Ttpawaf4JYe86xvT5oxnIouVjGk9DTvntvoNv8wWCumpCQ7QcpqcrkvwtcZBegScFhlR3-p6ww5m~Rgxz5Zc~fwwJmWIkl1MNPZJ3n2dLwKNqx3njyPXNCorY0RlGGYmjtz3VsDfZEVpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i95.jpeg" /></a></div>), Euryarchaeota (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i96.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i96.jpeg?Expires=2147483647&amp;Signature=JUkVj-DUChCbNDMsqqqFVqt~4qBXPk-G6j3Leu8Tnl4j5QDr2I8dc49kLGqr3729LeTl7EPSwUGRHNnvOnzsuW1fk3TvlFWP1mAezoF7HBZn0vVTHRG4QxY2s0Fe2BSqGzPusbShwy0yekGCaUB3~x~-DkGt-FYW~JwDOrximSbyuz2rvN1xKG8CddQdIKr1rGY0z-bOLmaNIP0i81S-8Kp-lM-NPiaUrSfzgsSO1zmflBAW0IZ9UblXxLASZOdAlAlBh2gmAVvbOXar7QkA7OLBl1BxECm7h0JKk~uqWCWb02HX6~gyRSaMrHaiCuLQTq30RPyluE5lUDxSogOwKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i96.jpeg" /></a></div>), Fibrobacteres (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i97.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i97.jpeg?Expires=2147483647&amp;Signature=n8e72YuHeB9FaED14Bo8Rxwc0XV8FQtXLUtYYU7XOlQw9byD7IXKcreKfLpqpLH00M1llnhWiM48740tYBmJY2tqFzzlwVNtZNaVut7IWD8pAX6RRYil-ylp1TcsyIJxzLAwIHAfFmO80Fg-w6z2dXgTnW-2Z1uGLWm2J8EsLTEFnYjebPYYai9hNYyRxjWYQ7eY-0N~NyxWTGBEPWrwaqoXK1FKySQ0iauL8Oih~hMoxX68yLBrHx7wPHq9LQnaQGQmA66aajeS50Af7czJvCL4uxUK3j6LdbjRGefav12aWg5RrkhhwbEgpYVFgoNmipEkyIs5C7TOxpfWJ29kTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i97.jpeg" /></a></div>), Firmicutes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i98.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i98.jpeg?Expires=2147483647&amp;Signature=FFf1kwridAGU-1nX7xtLNx2iFPx~4Iq9mj~iBXb5cnOTmtO4g-CBi7n0hD3t87-hgO~Qk6lv~8WYVgHvuWBuLzb8n2cCq9MZYOnXB3bX1jsEfczGdeiBTYKil3eX1wzIHKgh7dHgvGjxwjVqY8Bve-PmT8izfxw0~8W-A90RvpO6niSQ18YJrepgRKasXeIfWC93qniXofOMn2~PzdnTBNz7MJ~M9V29fiMO3wWj-a5GLpvR3IqVUWfY6x4H-DLVZDxANr8-S3F4fKLXBlnBkXCHKpGHsFB0oK6DGUD~Juy8TMP79iExp3gErlAEa01ankXS~mDPg8vsHgo1cXm4Gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i98.jpeg" /></a></div>), Fusobacteria (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i99.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i99.jpeg?Expires=2147483647&amp;Signature=2PqjAztjUgZzK-NawslXfqVdemFAlAaZ~0QAlcvMlwnR-~wZXmrLnySut-K3A2InJ45jsvodrpQr2EmMdJ90rQ9qLKiKe3t1Ls7r8AZM6vIIZp-f3--K6nFpcPreowi6MSuVDd7D0P-Q0SH9~QzZ-5JgUPxlyKCA~ffC4-8pv7TyBYL~uhsVvxM6tQNqFt2~b5-P9MmVnDJxnxKu9~3~~zp8l0bsVVOmZJSc0EPL1c2GeYu5YLhLXeTXmq17TbulfP13NTlU~F5gjw38Cv9N7j6QcIxYfS5eHBgb4rXMCwo98CaftyUiotoEugWpjpLLpybyd3Guikg6ch86tOT-1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i99.jpeg" /></a></div>), Nitrospirae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i100.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i100.jpeg?Expires=2147483647&amp;Signature=I0twKWxK9CWQNh7cae1e1s9Vjlz0Br4cCwmgVONT3Y3nS~LxpOnpw3XPSF9c3cLVr7vNrKxGd-0DfjMwR970osCQf9YJXJjEkWfIb9ln5dz0GmX1mLoDi2C2WdJA0WWWL5ot-O7Wk~OXbw0qXDE80edWUEnJ25HVrjUYJaSsN2POBg7xh28atRWHy-g4ji7M5FqlxT0q10NJtuJw4Vic6S56GnWun230jsZTP7LFK3wWRqFldXg~iLzizDC5-qYP7ER0giRPkXPt7Tkq7LDZdTVlNw3vIxjP~iRTYfYREHpDxsvQaXwp65ol~FGCimUmVfCd0mWmgRZWV-f12A2TWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i100.jpeg" /></a></div>), Planctomycetes (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i101.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i101.jpeg?Expires=2147483647&amp;Signature=B31jtVZkSShtMPXR1Dx~6XXLzpYtUuu8kySjOECU7hIDou0YAS0hz0FhPTPyyRwjc2g00ZjTlVu5Trc2-0xvTHgjdZW7DDNPkpAVhvb4S0IBIh6XuVcPxZ81joU4Q4Z3Od3-ucIhmmAfU~hRak9JlAmvn~cmA-UiyVcO8C82qsjnFpUbFY4MvVLcmZnfRSxM94Qw3xFlt~jEzvQoufyCDJEscUfYDTtxrM4svtR1bFcLwhKgoWc5vQtXlxWLvkyQslbl4aiDQJtwLsfPbl7C5c9x~d55qpwMHGwQ~1ucJNMPCQOdPRaKTwC18VEIjyYYB35G8ujq02Z~kcIBPErKnw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i101.jpeg" /></a></div>), Spirochaetales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i102.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i102.jpeg?Expires=2147483647&amp;Signature=1KgyN75BAj~zT0coMvKewsT7ZYZW~jzKItl0302WcNr0RhaoMOU501XK50G4GhH2C-sAWCLoNoK7h5vm~9JpJGyXZwNze6JHSmbg0AU-5TmFyiF5sQbjl1U8aLT302rQHQoC1FHeiB6jcTdRCxvixPgtETjxcd0fUld4jjgxODpbgbNuAmN1Hfl2EiptyzyqOgeM8Nm0CZ32ZRy7x1Dy2u2Iq8Iw~4HrLdyVLptb3J~mt6iTquuP3qHSOED9tW9Wk0Mm8XdHFFtZFSk9GCJwcUKNoXbEYheQaSD7QJ37Fd9HJBOle-JOBQGH82x-A3bYF4V0A1urOBwwvMkenZGF9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i102.jpeg" /></a></div>), Thermodesulfobacteriales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i103.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i103.jpeg?Expires=2147483647&amp;Signature=P4WVmLELeKHiNwbELs56N5JRAko4FmK3hoin0rBduQ5i4HRa5Q-EcX26hQG4FJTHLaAh44ad66RKYAdnxlwUyzglNXjenCoU8kcolLvF5ccgIgWZW4jQeY5jDkyUZBwrow39ryojnMPSlT9YLC6slFegWLhjlc7lbleAOKHkxtKcBLffbXE56P69QSYVnxji9oVgE2jUXf-nawZtgnXQ7T5DZgqzcrnyiLtfkuQRLS8DxAl88jsiMadhS6DPFe8fNKrAU2Dz3N2WcH70GtG4Qq0BM00g1bLhMW~-8WQnVsLFM9JNggZYUOKFQNuBEOdmlef066T5X2wDcEY~Z8dPBQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i103.jpeg" /></a></div>), Thermotogales (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i104.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i104.jpeg?Expires=2147483647&amp;Signature=K5Bql6hoWdq7DsFsEJdhOoLJoX85bDNyRIjlahgMO2EH8OcwQVmtOts70FaGhq0W5twzIL88LLgAgbXZDt9Z64pZBCEijajgzE2HsMsqnkFi2K9X6nLWkEnHfCmiwq~DpbmL-dcVsd46QgEUBwz21KDtP9~OQVy1Vb0vIN6mGzFvpmePdK~EJdqF0Xr5b5G8irrAgrqv1M8zQlqLZBdu-T0UrT22yBHdbV21whh7H9nE7T673x7RV2xIMANOu3VtVK5P9NkZVEumvM47dno4Gj~FVWEqYhAw3LcPvIgHYz1rpeB3zOv9~FfHxzuvRBdSJCsGTgIRLFrnALXbW3jQFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i104.jpeg" /></a></div>) and Verrucomicrobiae (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i105.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i105.jpeg?Expires=2147483647&amp;Signature=uRfHCL0t72mvGxHrErmeEQ5qSwy682ka2cpfaEv96VW1VTlK~3uAs7ICZXoPK-I0WIUrXKGbFiZRfgHCS7~OjsefTPoQWDCPOem4zx2BMncJVXTpQ2U5sm6zKXbNxbrmllZgiLv8OtKygyMxHUVZAsG9ZnpOV-3eH-WejWhBgzkJ-n-i16Zgc3byvDlvozUoUGzboNBqS6aA7ZBSizHk76~1pU5IiM7GphYn-Ebe6nrEVEwL6ma~EqO~-F5QQ6kejuvrRpqknVqDq623pQtEicmocXhgEEuLP8CHl85-BtcN6XvwUF2D8WG0nMLAiCb7EdKdJtgJ371FFQjg4eiV7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i105.jpeg" /></a></div>). Sargasso Seq. &gt;5 kb and 1 kb: 1 kb sequence fragments derived from contigs longer than 5 and 1 kb were mapped on the 5 kb DegeTetra-SOM. The residual sequence segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show phylotype borders. The dominant genomes noted by Venter et al.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> are indicated by arrows with the following numbers to show the genus name: 1, <em>Synechococcus</em>; 2, <em>Prochlorococcus</em>; 3, 4, <em>Shewanella</em>; 5, 6, <em>Burkholderia</em>. All Sargasso Seq. 3D: all 811000 Sargasso sequences were mapped after normalization of the sequence length, and square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar. (<strong>B</strong>) DegeTetra-SOM of environmental and species-known sequences. SOM was constructed with 211000 5 kb prokaryote sequences plus 218000 Sargasso and 5000 biofilm 1 kb sequences after normalization of the sequence length. Sargasso and biofilm sequence entries longer than 1 kb were selected and divided into 1 kb fragments, and the residual segment &lt;1 kb was omitted from the analysis. Species-known plus environmental Seq.: lattice points that contain sequences from a single phylotype are indicated in color as described in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, those that contain only Sargasso or biofilm sequences are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i106.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i106.jpeg?Expires=2147483647&amp;Signature=GICa94IBX3oLeTGmL0Yk9CKmxhFwGQ8~9RCzPMDf9R6fM3g54ipgERaeFhJIo1vS75b3U5arDO9xpdyMTalDXG~ia2sj1brxz3CeoBpNZneFF9ZU74F2ZZrJQWh3fway9fNwpytaZji7oE-lfy0tvUsqFMTmZrmos7f34Q2eYn9lSQD7vS564u4FIckbVOx79PDOHWz93yWVt5gOdXQNnxENeU8HWFHqB8URHJG0TgRQZDWwkowyOuQK0a6sBTd21u1VemT-fGsSxYG4pvo8GuxxKxbB4P6~WhDwgmPqyCWmp3B8OVqqlBm4Q9uTu2uMIaULLTLC-IOcdywBy~SZYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i106.jpeg" /></a></div>) or (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i107.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i107.jpeg?Expires=2147483647&amp;Signature=QKX~fq1zhe0X9mkzzhWL8p~L9IkWM4aNCLfWeX1moUx0XyBaE17SiApZbb0gxX1HOmtrJFdnlwDWEYE8Qy~5sR69KlQ9hYD7r-RfNAiYnFB9umZ4Bn52WJ5IyMTjGekiwHe5WXM5A2Ce~sraZvPUTEpstdlYBr2JnnTnKdygMw0lQmJ8AXgsJOfNW0cKQhM6vOsrvtuGYUqZ7LbcQMz9-FiEKpR~9fJYPxA-SOfLRM47DaBGl5t43FbkT59Bh0szKwpUTqRiJE5WjrhWdeTn9gfYwvwrrBKf-bSDaFInc58Totbbkpj7QET3vZCKc09mqZwUfJSZKC993hUJRKkr3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i107.jpeg" /></a></div>), and those that include environmental and species-known sequences or those from more than one known phylotype are indicated in black. Biofilm Seq.: square root of the number of biofilm sequences classified into each lattice point is indicated by the height of the bar distinctively colored to show the dominant species reported by Tyson et al.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>: <em>Ferroplasma</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i108.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i108.jpeg?Expires=2147483647&amp;Signature=q-PMmj1zFTqnOEy6ND4O7hBow9Zp~CGLTHzXrFQ8Qq5dIxL63LsheYGUkV2DC85FIBLZlu32t1mP7FeKbXkqphezZF5zPV1hKhMKIeHWg5JAS9gzRcoPG-SDOJvI~c8bvMnQQhyz53WzwChyfjRcDVumy5umchfYHoGYViUX-CSXV~0lsqmYE0q3b3App0iTEX7xWCuYNvPIYTLosjhTrbwWobpjncK6Z2wZ7xD9AwYXuyPbLiBJEjNwo5fcB9IKFtpZEHipaxXvc00TH2dM-RpxesVYiL1yLulB~5K-CB5n7rCNEfLfRU8469mlKBIxHx49Esuc~IXnjxV8a9Wkbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i108.jpeg" /></a></div>), <em>Leptospirillum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i109.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i109.jpeg?Expires=2147483647&amp;Signature=U9WisvFGAoYN0LI8SJnNeh2aoJS0l2NrH1NzxowZ--AlBgu-pEXfbG5PgDg2VAuqGIDZmW7FEPaz2P4HW0I1P8W-86kg6oZQ9hF1RF3s5CEy1wSEv3o8-a-zb0M5imASIrkr6C1FAsCOV3liVtNULAi0ezvbo-jLObFuycmUiXHoughMn4qmbJF28VdsMfmHoxVMq9Lgn~w4HsksGnEt7U4a7Np2KEZOO-usH6fhociyuy2FkIeYu123qaLnXt1v-48nHJeW2BTNOSTgv1MUAYuYUhmbT6OJEPRjpJ1RIY6xg56E~jOo08Q2g5JMwrlnKoTKGmZOCB-ME1iXeyjHFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i109.jpeg" /></a></div>) and <em>Thermoplasmatales</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i110.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i110.jpeg?Expires=2147483647&amp;Signature=HmrYcStfufp9Vk4lXi3DGgEG0HQcuY2uu-hINEK8SP~bygScCSb12R9QmpZV~dPQOlLNNfcRNPDcEX6m5~JTMk5lupkCjBY~7QOcMNu2xPqphk7K68TPe-9lqxs1R47qFPeKIDdzilt47UR4sXzDlfmmsS9ANKII6LDdp7eAPWrzvNY6b0qUWCfOa340ha9LWQUFkECUKQXjSCxdIKdcM1jbq~vUa-O9~CIEZ5pWGK690LjFKRloJAsv-Ktc0dbAcV1LuxWuL9q2VPcIPhW6FWSyiEB6X6xhoDb5FQ~VFT45aUsu6Mgx2giAji2AV8TldEuWOa4y~1RRP5VGj2yafQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i110.jpeg" /></a></div>). Sargasso Seq. unclassified: square root of the number of Sargasso sequences classified into each lattice point containing no species-known sequences is indicated by the height of the bar indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i111.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i111.jpeg?Expires=2147483647&amp;Signature=cgEIYKVuJgvMO~xIRQ0xr2jkNLes0Xja1i-Svk4KI-ClLujp3tYYwufsAKiGOZcidHegR0ybvo9a53qoyMMq8j~Am2QC2JNGgx8o0q8rDJc8DtNtABNBZsCmKnH0sfhrfF0casw2AUOGLZgpGyYcPZQ8Jt1mAL6Uq0gfmTWV6YwYMb6osm94jEwS7YxVJGeCkFOC0tjou2UtdbZ8-1Xz5zFw~pUrDKlFwCYk6cOsPn1R7F4LOFS6CwW5wuVSJFcMjUBj3UyuAQkV7hBFexeP3~Bh-RXSCCttomaSDlcKG9y7YXSFpJM1uDvyQK1bbFYicdavnMJMhpHRj~P553xSwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i111.jpeg" /></a></div>). Sargasso Seq. classified: square root of the number of Sargasso sequences that were classified into lattice points containing species-known sequences from a single phylotype is indicated by the height of the bar distinctively colored to show the phylotype.</p></div></div>                            <h3 scrollto-destination=87193425 id="87193425" class="section-title" >3.4. Phylogenetic classification of environmental sequences</h3>
<p>One goal of metagenomic studies is to characterize and hopefully reconstruct multiple genomes, at least for dominant species in an environment, by sequencing a large number of DNA fragments obtained from an environmental sample. Such an approach should allow extensive surveys of sequences useful in scientific and industrial applications and assist in developing accurate views of the ecology of uncultured microbes. The traditional homology-based phylogenetic classification methods, however, have inevitably focused on well-characterized genes such as rDNA, for which orthologs from a wide range of phylotypes are available, but most well-characterized genes, including rDNA, are not industrially attractive. It would be best if microbial diversity could be assessed during the process of screening for novel genes with industrial and scientific significance. We developed such a method.</p><p>Venter et al.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> applied shotgun sequencing to mixed genomes collected from the Sargasso Sea near Bermuda and deposited 811000 sequence fragments (a total of 1 Gb) in DDBJ/EMBL/GenBank. These environmental sequences were analyzed with the newly developed SOM method. A majority of Sargasso sequence entries in the database corresponded to one-path read sequences; however, a limited number represented contigs assembled from multiple sequences derived from dominant genomes in this environment. We first selected 4258 entries of the contigs longer than 5 kb and mapped 34000 fragments of 1 kb derived from these contigs on the aforementioned SOM constructed for the 1502 known prokaryotes (Sargasso Seq. &gt;5 kb in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Distinct clusters of Sargasso sequences were observed, and all dominant species reported previously<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> were included in these clusters. We then mapped 218400 fragments of 1 kb derived from 134600 entries longer than 1 kb and all 811000 sequences (Sargasso Seq. &gt;1 kb and All Sargasso Seq. 3D, respectively). Although the sequences were not clustered tightly, skewed distributions were observed. This is more easily visualized with a 3D representation in which the number of Sargasso sequences classified into each lattice point is indicated by the height of a bar. Zones with abundant Sargasso sequences were located at the bottom left of the SOM, which is an area that contains AT-rich sequences. In the respective zone, the sizes of individual territories of known phylotypes were relatively small, and the territories were complex (Species-known Seq. in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), indicating that sequences from various poorly characterized species were crowded there. The A + T-rich Sargasso sequences derived from novel species might be mapped there because of the high heterogeneity for the species-known sequences in the respective zones.</p><p>To investigate this possibility, we next constructed a DegeTetra-SOM with the species-known sequences plus Sargasso sequences longer than 1 kb. In addition, to confirm <em>in silico</em> reassociation of sequences derived from a single genome but cloned independently in a metagenomic library, we included sequence fragments derived from microbe mixtures in an acidophilic biofilm growing in acid mine drainage<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> which is a worldwide environmental problem. Because of low-complexity of the mixed genomes, Tyson et al.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> focused on this biofilm in order to reconstruct dominant genomes with shotgun sequencing of the metagenomic library. In the Species-known and environmental Seq. panel in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, lattice points that contained only Sargasso or biofilm sequences are indicated by (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i2.jpeg?Expires=2147483647&amp;Signature=hofJobe4MQA75cWztDSKDUvjsIWwBoP3U~Sth1K2wbPGEh5OZnPGedpsArLBouDaAT1g9yK8fDhSMFD1Ds3qlMO90T-LsOXVzw~ose2nmUlw68fEAl8Y~PiOUcFRct0IlrQPl6I~o56rDbJj1OBrVQhgMB1XdlHAkQZ-aRuoBLonyqzebDwcXAUMTzfKCuFpLkWVcRq5PsvHUOGqMH8fylk~NwFXd6azXMVRSR6p6TUOlkFXtM4AFuG71HJtPUqKnkp1aNjT4zHwzWVaJGI7IPmedJ8liFcIsxOv5jUTh69NLZOOb3JQhxtTQKY5TqtGU8mCesIP7ED~MkOLSl1qXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i2.jpeg" />) or (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i3.jpeg?Expires=2147483647&amp;Signature=lsvV44RL9DJw8cfLLOUj3ev6A-3otS9JKL9U7JexvzmkJ~blqkU1xBrQqwn2j6EaKAIZwGzKKw-MMzRkyjS4rEGIMOI3xpUfB-pNoM4IL3HrATOaJJq~LK7yXIQrky9YEVpY-HZYNls330-U3V5D2En-MywarSO~rHrDQgyDu4MOCOfc4M6Ekxmk9eBAgXty2tVuygPfrZipXsHqTogeTPRhARXH~BCk3PUgR-1ilg8gKMZc4megltd7dVDXl9xsGvKVPwl9gI87MhdEuRh6g8ndduo~P8802sqWJEyDv6zrcEHeATpgV72JX2-d9GdWmj8NN3pJBXyjmW69-NbKYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i3.jpeg" />), respectively; those that contained sequences from a single known phylotype are colored as described in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>; and those that included sequences from more than one phylotype or both environmental and species-known sequences are shown in black. Most of the biofilm sequences derived from the low-complexity metagenome library were located in a few distinct territories (Biofilm Seq. in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>), confirming that most sequence fragments derived from a single genome but cloned independently can be reassociated <em>in silico</em>. This can provide the rationale for phylogenetic classification of sequences derived from a high-complexity library such as the Sargasso sequences. The territories of species-known sequences in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a> shrank appreciably when compared with those in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, indicating that a large portion of Sargasso sequences had oligonucleotide frequencies distinct from those of species-known prokaryotes. The number of Sargasso sequences classified into lattice points containing no species-known sequences is indicated by the height of the (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i111.jpeg?Expires=2147483647&amp;Signature=cgEIYKVuJgvMO~xIRQ0xr2jkNLes0Xja1i-Svk4KI-ClLujp3tYYwufsAKiGOZcidHegR0ybvo9a53qoyMMq8j~Am2QC2JNGgx8o0q8rDJc8DtNtABNBZsCmKnH0sfhrfF0casw2AUOGLZgpGyYcPZQ8Jt1mAL6Uq0gfmTWV6YwYMb6osm94jEwS7YxVJGeCkFOC0tjou2UtdbZ8-1Xz5zFw~pUrDKlFwCYk6cOsPn1R7F4LOFS6CwW5wuVSJFcMjUBj3UyuAQkV7hBFexeP3~Bh-RXSCCttomaSDlcKG9y7YXSFpJM1uDvyQK1bbFYicdavnMJMhpHRj~P553xSwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i111.jpeg" />) bar (Sargasso Seq. unclassified in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>); 79% of the Sargasso sequences belonged to this phylotype-unclassified category. These Sargasso sequences should correspond to sequences derived from genomes that are poorly characterized at the present moment. The remaining 21% of Sargasso sequences were associated with species-known sequences (black lattices in Species-known and environmental Seq. in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The number of Sargasso sequences classified into lattice points containing sequences only from a single phylotype regarding species-known sequences is indicated by the height of the bar colored to indicate the phylotype (Sargasso Seq. classified). On our Web page (<a class="link link-uri" href="http://lavender.genes.nig.ac.jp/takaabe/DNAres/SPT1.htm" target="_blank"></a>), 92 genera, whose sequences were associated with the Sargasso sequences, are listed together with numbers of the associated Sargasso sequences. It should be noted that most of the Sargasso sequences have not been characterized phylogenetically because there were no orthologs for these novel sequences. Detailed phylotype predictions for individual Sargasso sequences with the SOM method are available on our Web page (<a class="link link-uri" href="http://lavender.genes.nig.ac.jp/takaabe/DNAres/SPT2.htm" target="_blank"></a>).</p>                            <h2 scrollto-destination=87193429 id="87193429" class="section-title" >4. Discussion</h2>
<p>G + C% has been used for a long period as a fundamental parameter for phylogenetic classification of microbes, but the G + C% is apparently too simple a parameter to differentiate a wide variety of known species. Oligonucleotide frequency can be used to distinguish genomes because oligonucleotide frequencies vary significantly among genomes.<a reveal-id="b12  b15" data-open="b12  b15" class="link link-ref link-reveal xref-bibr">1215</a> The present phylogenetic classification is designed as an extension of one parameter G + C% to the multiple parameters oligonucleotide frequencies utilizing SOM. In the case of the homology-based phylogenetic classification, a set of orthologous sequences from a wide range of species is an absolute requirement, and therefore, the conventional method is difficult to apply to the classification of novel sequences from poorly characterized species. Because the SOM does not require the orthologs, the present strategy can enhance novel metagenomic studies of uncultured microbes.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> For example, the method can be applied to analyses of microbial and viral sequences derived from clinical specimens (feces and phlegm), including sterilized ones. If SOMs are constructed in advance with a large number of sequences from a wide variety of clinical samples systematically collected from various sources plus sequences from all known species, this technique may be used to screen unidentified pathogenic microorganisms responsible for poorly characterized infectious diseases.</p><p>When we consider phylogenetic classification of sequences from uncultured microbes in a very complex ecosystem, it will become important to construct SOMs in advance with all prokaryotic and eukaryotic sequences available. For example, a certain portion of Sargasso sequences may be derived from eukaryotic genomes such as those of fungi, protozoa and fishes. Furthermore, when microorganisms that have symbiotic/parasitic relation with a macroorganism are studied, sequences from the macroorganism may be included in the sequence collection. Specimens in medical studies may also contain various eukaryotic DNAs. To test the SOM separation of prokaryotic and eukaryotic sequences with special reference to Sargasso sequences, we constructed a DegeTetra-SOM with 5 kb sequences from the 1502 prokaryotes plus those from 11 unicellular eukaryotes (6 fungi and 5 protozoa) and zebrafish (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). The power of SOM to separate prokaryotic from eukaryotic sequences was very high; only 0.1% prokaryotic sequences were classified into eukaryotic territories. Furthermore, separation among eukaryotic species was apparent (Eukaryote Seq. separately colored). Next we mapped Sargasso sequences longer than 1 kb on this SOM (Sargasso Seq. mapped). Although a major portion of the Sargasso sequences were classified into specific prokaryote territories, 9.9% were classified into eukaryotic territories that corresponded mainly to territories of unicellular eukaryotes; 13 times more densely in unicellular eukaryote territories than in the zebrafish territory.</p>                        <a id="87193432" scrollto-destination="87193432"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f4.jpeg?Expires=2147483647&amp;Signature=rVDW4lRrBBiDZS1mtLfUzg6Xuj91tLDiw9be-kBs4uHXzTzGwK0SVP9xiIp7rPem5q5REG2pvGH-mfrgi5IzOeHEAsAuuMUOY2h5--GCUX~Ap2ZuUhc0xLJLrHAaclhr5GrOi7-YTB9DgpFfcX2uY45ywrBrDuI58HYdMsODlZy3cVxiwqAud0D1nizovaJZq9SGp5Hb0OXc1NcBR8y5YCw~bJjrLG~IDNn6Q2uQe8CT800fSdtKEPKix-Ko7qLtdlK2-soMScsUt0J5jjlz-JjMLI-nlS46UPQuDmOwwYgewXVk5PxiBaJE~CdHzf9anQBVPV~GNudBicaUoxTosw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SOMs for prokaryotic and eukaryotic sequences and for 16S rDNAs. (A) SOM was constructed with 211000 and 210600 5 kb sequences of 1502 prokaryotes and 12 eukaryotes, respectively. Prokaryote Seq. separately colored: lattice points that contain sequences from a single prokaryotic phylotype are indicated in colors as described in Fig. 3A; those that contain only eukaryotic sequences are indicated in color (); those that include both prokaryotic and eukaryotic sequences or sequences from more than one prokaryotic phylotype are indicated in black. Eukaryote Seq. separately colored: lattice points that contain only prokaryotic sequences are indicated in color (); those that include both prokaryotic and eukaryotic sequences or sequences from more than one eukaryotic species are indicated in black; those that contain sequences only from a single eukaryote are indicated in color as follows: Aspergillus nidulans (), Encephalitozoon cuniculi (), Eremothecium gossypii (), Saccharomyces cerevisiae (), Schizosaccharomyces pombe (), Neurospora crassa (), Cryptosporidium parvum (), Dictyostelium discoideum (), Plasmodium yoelii (), Theileria annulata (), Trypanosoma brucei () and zebrafish Danio rerio (). Sargasso Seq. mapped: 1 kb sequence fragments derived from contigs longer than 1 kb were mapped on the 5 kb DegeTetra-SOM after normalization of the sequence length and are indicated in color (). The residual segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show borders of eukaryotic species. Sargasso Seq. mapped, 3D: square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar in color (). (B) Species-known 16S rDNA: Tetra-SOMs were constructed for 38660 16S rDNA sequences longer than 1 kb from 19196 known prokaryotes after normalization for the sequence length. Lattice points are indicated by black or in colors as described in Fig. 3A. Species-unknown 16S rDNA: we selected 6855 sequences for which classification into 25 phylotypes is annotated in GenBank and mapped these sequences on the species-known 16S rDNA Tetra-SOM. Lattice points that contained sequences of a single phylotype annotated are indicated in color representing the phylotype, and those that included sequences from more than one phylotype are indicated in black." path-from-xml="dsi015f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193432/dsi015f4.jpeg" path-from-xml="dsi015f4.jpeg" target="_blank">View large</a><a section="87193432" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f4.jpeg?Expires=2147483647&amp;Signature=bBFSTv8cJYHXP4uVBi5WvrHEG~5QbU8cABi6rETPfnLGPtnN4lL9Zbw7NhIdXvwS8BXLPtbFTgdd0F4M4KA-cpW8GzIwZrKf6sJcFN4aIAI7xk7LiBUOrG8EV5Yxtuyxxuqrh4AnHn-wPGARkd472oUr1Dhx8wnxJzQcz29n9~gKS1fAHqV9HSwNQA6C25FpVs63RRiHAXFyDqBI1isL9WFzBMudk4UNuvsGq3z4Yt5XMegzYtn90fwcWODbF355x4eO30EvFcHscsSnUvu896UTkUWUPRQkKjdyJO5dPDuCpZ4fp33hVT8Q3UaGMfikRPbCmwhpG2LBkPyeyBZy0A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193432&amp;ar=350803&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi015f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SOMs for prokaryotic and eukaryotic sequences and for 16S rDNAs. (<strong>A</strong>) SOM was constructed with 211000 and 210600 5 kb sequences of 1502 prokaryotes and 12 eukaryotes, respectively. Prokaryote Seq. separately colored: lattice points that contain sequences from a single prokaryotic phylotype are indicated in colors as described in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>; those that contain only eukaryotic sequences are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i112.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i112.jpeg?Expires=2147483647&amp;Signature=PTAJLAO0XqL7Cr7sJ8wNwzHtStEW2S-~8zYLvh0uHK7n~4GP21xjUNJaCZeKCVgrKOmHgcY62zoFsUPm-PGOv1ZJvWwtQtpYzzmJ~SSV-~nGhsHg537b4kLGnT6PL1ZjDxjk5-DY-GrR7E80FqIXS7pDdk3NuHLlD0E6uIlMpiQEqVaaLu9lCmIjR4wHoEhOAgSuaqPZn2GlPxWY~8ILiStQYUGwk06Lrb9ThNd3bxL4SJADnsC2yRoVVCLNgT4U3XuWlS9ZGOBV4wSCq3D4XBXIKDaPnH2LpYRKDd7Hen6PXAmRMWqXkJI5Oy0P5lWjpccs4Outv3xraXqquZH-Zg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i112.jpeg" /></a></div>); those that include both prokaryotic and eukaryotic sequences or sequences from more than one prokaryotic phylotype are indicated in black. Eukaryote Seq. separately colored: lattice points that contain only prokaryotic sequences are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i113.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i113.jpeg?Expires=2147483647&amp;Signature=aqDhn6FiDknThFlL9bqEtYHgz4n0I9Gp63R8cHIld65x73bXs6CdKkx4vEjZMs8EmcgbXIKTOe2CJQXKM7vVCb86NpOIE~Q0eTFUWliiLxCHPTtD5A0M10n~w0NrXHa3~WmXR0sCtWPgwZNUWkJXPTs3oKDpEC1BGJRaM5T71Shv~tfAewwZ0qQh3OktbNIpvcwCEDOmXJSGMBiu0OCQEZCddEySXFOmijjGEWBf4FoY6QKuRrTC5hJzfzJ17HsKLISr5nt~eU33eD7mAWbeRxZGnQlx-Ktw7Rmex8sGsTR9aNWITh0sauCJb-~PcEpwBdymVz69AH25fh-5ci24Aw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i113.jpeg" /></a></div>); those that include both prokaryotic and eukaryotic sequences or sequences from more than one eukaryotic species are indicated in black; those that contain sequences only from a single eukaryote are indicated in color as follows: <em>Aspergillus nidulans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i114.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i114.jpeg?Expires=2147483647&amp;Signature=OPznyfSlv3jpuFUmkhFwNVaHxAlYN15cHArUc1thEVIeMqoHrLVEMrYHIfFGJAVMqPAaiDelvocDuz7vMuD9J6MoDWX1xB~0h7GKJkUDEabc9Obdo460TT4v3334~wmN7Ds6U~r6qmyqssHaDjjie-okO2RIMZkG~Yd~XUQA9cmpn86-yeu3E4K~I2rc6GvSnnh7GaSVB7ymec7eHZ~KOmfo-h1wtnk0mcD9rJaw9rQHufkqrfVx~af6WC~QDY6ROoWfaz96r4rWOrQdqS7DHXcdIxrEvdWLOlRZwK08Vde-KuoHbEGY~h~VKKYC5Z0~tCXLNg41Bk42a9dTFWvPTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i114.jpeg" /></a></div>), <em>Encephalitozoon cuniculi</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i115.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i115.jpeg?Expires=2147483647&amp;Signature=B7gOSzdA60HxB~Mkjw5rMqaD~tqLi7wJ-CS85FbtnBpWOKQmeewpK2~cGPkTMYyQtQoB5pVYoZ1fuJU7NvjoLt5BZL89cseidqTdIYTTxrpR8zyn~M-i9k~LO5QwNkzQoyyo5QS~NjAg8JXVcAHie7JrwLpSPE4wMaWO7gCDMbJVd1F3xxfO5h8vtNQX3PH2hQJXrS3t0La23YbV0xLE04K8Wzu71UIJ95o~fSHWU8gcxqxtqKRkgHKQfkSAOFSbUfy7SMNhZTNAZem-zmOADxbCOQSJxUxOYPps0vnQx0KCuwlGx~0AHPNd~pqIwjGAsZn3nG0YsLbVJ-itIqqsCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i115.jpeg" /></a></div>), <em>Eremothecium gossypii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i116.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i116.jpeg?Expires=2147483647&amp;Signature=BTHMALNsQ1znhhVjX5gc0ZreX1SVGhwgRezT6z-H927E7S3HSo0Yuzyl6gihAGQ~gYTcbeZ7gvqCJyEcqwIj1LQa6ozK-3g3Pbb7GnmA3HXy774oPtSlNOKX6stk5geHBzkXFqcfLupPUo~W4FqUJ9ATu3-1P8eu~65Wav52cpR-SRSHFSBDvx77ButVTb5nH29NoXqBAJi9x9UzXZPEFLrGKgBStFvnnFBF7aUc2RzBc5871y1Bb-3VYrcQRG2cChpsGej7scqmCecdfXnQFXRJ7HBe94HaSO9g2MRUOXUfijUYgitcgm0hPx67hTXxrYiU9IWP~qD1SOpsGDAw5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i116.jpeg" /></a></div>), <em>Saccharomyces cerevisiae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i117.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i117.jpeg?Expires=2147483647&amp;Signature=PWyDZ7qoYMVB2Vn8ICryph-GjwkrWukocCCtWHoL9JU3Ofonzy8IpjvBeWq4EF53AbpV1l9TWr2zwj8H21Njp3JXp9NLfSlJ-Vh6YwOmXheq7bTlouGHr2FHDts8ZQwCWy4~f~BlvrKruDXO0eoGx6Ho9ZO7wQX9TGOEAq2XBeiJL44mmXremrqee1ZK~yZ9Fl0STP1WfwN-VNgxY2vwUzryw5kd4GkjA7SsBMpT31SWedUTwHYx7g5jy2DETXN~y8gj3l-Gk1ZL6SWYZTcSRaycpBGMRWQxoAcathTCHgrtwFMnZWYJ9OrjghTp-EUu1vkS5Grqy3q2Bt7dQ9IfxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i117.jpeg" /></a></div>), <em>Schizosaccharomyces pombe</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i118.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i118.jpeg?Expires=2147483647&amp;Signature=a2QATYdAdSsUqj89WhMJ7ul-Cq6oAxj5fy5gQYwxYd4ht0zwSessGDu-fhWYpXyUeOV98kN8sw2wITwaSI-wrCUpUDQEOZm4Xiwyg5-CfeCxuzhSclkic5vGlPpA7V567IBW6gPa0EG9RvnPLErWmsXGQqmG460o-qZNuQAUh6s1p7DV0inOTIoyO9lfS9qeMmXvQPg3OI6H6MucAdjDDus6IsIijcR8WUnye1OSXkIcU0VoScSKEkvJC6ju6Q1UnfnZ-lkUR-R0yRtAjFr4r73W8U~kZlnfMNLgBxXeRJ2dbrIvj4mHJXn5Np2heKcR~GxZMYTbtuvc5gj7HfOr5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i118.jpeg" /></a></div>), <em>Neurospora crassa</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i119.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i119.jpeg?Expires=2147483647&amp;Signature=WCnNSUhBDskZ31DeECXNXIOBQiuhdJsCaL4B0-Y4EpfztDzfaWEEqSCxg3~gg1CWRA~c6jXIyzTqcppMqDw-Pxz2Mw5~7mP5wgna3HlOkzT0AY7r~rZlTizOemgRouklL~o~NSmiJHZxQbuBAaYzpKHiBUONeWeipLsVDpa8tFKF731IRXb~DeMkso84ua5Wbb3TcHAakttpm-G24qG3PAQEfJzmWxWjaij-FwQzq-c0AD3yZ~hdguckcwegf67Seu5kN0--2mZvsXaqfkFPd83vpQL3jElUgMt7~wjVs8zq94BajpWCje5942uBhhVPIGm5X3sLXGIWv46ONCgaCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i119.jpeg" /></a></div>), <em>Cryptosporidium parvum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i120.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i120.jpeg?Expires=2147483647&amp;Signature=bFTIs4BI-5-uE0Eg3vzm80i1TSRjzfTGAfjqgjgrICie5AtMOp-b3WDn~KKauZ0Uu-ufS75LUl4O0ybVMa8quTckIgK1~pL5mlWnyIoMOH1micPaDTRHtdIgyXBFK93~7okcUfU5OfHfaEoqY6z37GNbM8fZPxUURxj9AR5Io9DHTERvjjFWRgl1LSYCdjEnA~7KDS4YJV7Kc3ckx9lN0Grg5zZx6BBdzU3jML~Z0Qu8gs7YrnotrB-NbXKlf4AI61D5UZqx3EfJ3WlKS64DmwXIyIcDhRoLbdA7LlKXq3hjIzZ1Hj0TWBZBs6l-kIPlKa6bMPr5jqzoCEfS3UylpA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i120.jpeg" /></a></div>), <em>Dictyostelium discoideum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i121.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i121.jpeg?Expires=2147483647&amp;Signature=li30k1U18VhZyriDaBKWRDPYG-6onEfpEK7t2Trrz3SRg1kd5qo7-8eWZVEemYxlR88qXjqYBadjA~oO7NOx~X6edv5pBfdRMPH~Y3MNqT-5UT2l3SmQ7lwvhmvXQf5Kjae7z0fl7ccZvwJFz5oN~2cGcrF8oaDVGvh5tqb3pZNPSynFg33wYx317a3krYhnwckKZLdT8roVyln8R1-fu4ee-k1aFL2t2NC2Vxza8tBZGX-LMu~~U1S2yJ47XEEiNARjXeoAE7Dt2ffkBpkRNFJg-F7BZ2XB1fj0SDYsWn6Bz1GpAmyIcDZ5I1~LwuAjsx1l79xLRJTUhTSMwNIA8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i121.jpeg" /></a></div>), <em>Plasmodium yoelii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i122.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i122.jpeg?Expires=2147483647&amp;Signature=i2z1rSQXaOSooErhvpNre~-BOqyxeQJ8xoFkAErvCXrQXKyAqJGEtMi5IOKHRF2enIHtg5il7tvYnAZtRpVcRH6ZJevIuEg3x38iTNjniDySGiZaXOM08MALKtKuCfpb1CqDTyrVJlG4UfND8vf9kKVoy8hBXjm7xGTAz~UqFhNry-AjRMUGqr1zGScnokcv4deH4aJlXtKFw8y6Do4OEiacOAPBbVN2SK~-syAZs5nzsQmC6Zxv2qLDsrkXVn5NG-gZ9YaE86IH5hoIdy7oB94SjsrajDWTPs9IsoqBjGd8doxbQ61iTOVln7opJ7IdAlBTGQRGA9JXFk86kT56sA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i122.jpeg" /></a></div>), <em>Theileria annulata</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i123.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i123.jpeg?Expires=2147483647&amp;Signature=MDJPzHiNWJzpNdYeancNwvp~yzxh7v5T1miSGkDkMTBrU9CSY5qrSDKfqxFW2TAXeUJY5GwJFusUiu5byy8t1OO3JF7gme4UNVdjd7m1AakrmIGTIdCC7tWgOGoDyhEQYkElHXCqz~siaFjXJdPM9PIcbl9vh~eFOPxfHnFvuBFoTx6RTuzwJVRxSJxIkdAUVlWDTCqTz3tbdfYiXvCD~tVtjIDUvIFW0qnx7nZyJYMisCZHyMvWFDqUtX~CfWzOPMCgzAsvqGmG1fUTMnpJQ43i7kCOMlV64tKjSrTIoeW3rptAxzHRlhEfwIrIphQjD6fxadmKA66IwRyrWlzKow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i123.jpeg" /></a></div>), <em>Trypanosoma brucei</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i124.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i124.jpeg?Expires=2147483647&amp;Signature=fKgXQNfmhJ~Pp1EyLb3KLkDt-YhrA0INrxYQwIGHweJur-S41ZwAfmsjIeZlu19IZR-bLjsmr4gcj0EeTvhjksz3q3Bow9Q1W3wFgyfHUSmT7CYiDZTWfAdP7SLHum5xqd~869vvlgX75-jhwmAQ~ztOX~OcTYhg5Xo~yfB89fb01nfODmambh5S0S3WPzqT9YTb8aAmdd5LEqMfH6jVfThxtMxo-w4dhNBzhi36GXT19ohzfsKppNR7bxylzZhefoCfpKGF2GCmHvrr748~r5fH6xoL6C2NFm9hqxPC9sHoxhNxDkdk7RPBrBK-7tK2RKU9dBE9Yx1AIO2P9D-VNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i124.jpeg" /></a></div>) and zebrafish <em>Danio rerio</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i125.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i125.jpeg?Expires=2147483647&amp;Signature=nU43CF9Rg4e4KkN5bO2AeRuV3jmDeaCnJkdGsRIFgbfuM-5E7A4VvqkcYDBljD-b-bBI2dV1JcSQoLTJFN01xgDuj5ZjgzmNeLl3gkNF7vo1z1Sm6OYlQt9Vt88uO7rmJFOkXX~aVHdnPn6NvpDQVwJ1ehuOYJhGM9EuEv9rvThO9rGb9rwT5z9WIizUJZV6xuIVYAOay-XQdjoM103Tx2suXXy~shy6j804kP8N8zB8j89C91V1BcQ4Tdl9A3IkxTPGcoSJe7NZ5eUtZ2OQJGHB4Oy8iWM7hk-XrUCNf7o7YRyl0egXZUR1oP191NTtNqQZLEiDafSmpvecCbuB5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i125.jpeg" /></a></div>). Sargasso Seq. mapped: 1 kb sequence fragments derived from contigs longer than 1 kb were mapped on the 5 kb DegeTetra-SOM after normalization of the sequence length and are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i126.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i126.jpeg?Expires=2147483647&amp;Signature=hRA6WIfIfYSf0fykM0Lp7P0PZR~JHTdDE8bSST4gWABHFeeu5rRTgX0HUuJnnEcP~u3g3T56TZfIi34rg2caT0z73MU3EVxlDN1LWtkfzMtPn910mRLy8zV1TpsGHUqd9pClSs8LFROqu9UQD640O2lMr2nulO2P0H28QsDQiMOX1QM4Oy73l30eDccA7kayioG57~A3a3RhaphMNAp9UqWg5hEqJdoR2WWGlgSejsaboiq4BWaSFbjqW6hs94NZ559lAU8BkOxWw8YEKf5rPODq4BPOd7if5-B0Vw8VBrJn1gP9ZgnPG3~SK3dEiTDgnGXV-rz7HWXSlg~gUmKXAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i126.jpeg" /></a></div>). The residual segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show borders of eukaryotic species. Sargasso Seq. mapped, 3D: square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i126.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i126.jpeg?Expires=2147483647&amp;Signature=hRA6WIfIfYSf0fykM0Lp7P0PZR~JHTdDE8bSST4gWABHFeeu5rRTgX0HUuJnnEcP~u3g3T56TZfIi34rg2caT0z73MU3EVxlDN1LWtkfzMtPn910mRLy8zV1TpsGHUqd9pClSs8LFROqu9UQD640O2lMr2nulO2P0H28QsDQiMOX1QM4Oy73l30eDccA7kayioG57~A3a3RhaphMNAp9UqWg5hEqJdoR2WWGlgSejsaboiq4BWaSFbjqW6hs94NZ559lAU8BkOxWw8YEKf5rPODq4BPOd7if5-B0Vw8VBrJn1gP9ZgnPG3~SK3dEiTDgnGXV-rz7HWXSlg~gUmKXAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i126.jpeg" /></a></div>). (<strong>B</strong>) Species-known 16S rDNA: Tetra-SOMs were constructed for 38660 16S rDNA sequences longer than 1 kb from 19196 known prokaryotes after normalization for the sequence length. Lattice points are indicated by black or in colors as described in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. Species-unknown 16S rDNA: we selected 6855 sequences for which classification into 25 phylotypes is annotated in GenBank and mapped these sequences on the species-known 16S rDNA Tetra-SOM. Lattice points that contained sequences of a single phylotype annotated are indicated in color representing the phylotype, and those that included sequences from more than one phylotype are indicated in black.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015f4.jpeg?Expires=2147483647&amp;Signature=rVDW4lRrBBiDZS1mtLfUzg6Xuj91tLDiw9be-kBs4uHXzTzGwK0SVP9xiIp7rPem5q5REG2pvGH-mfrgi5IzOeHEAsAuuMUOY2h5--GCUX~Ap2ZuUhc0xLJLrHAaclhr5GrOi7-YTB9DgpFfcX2uY45ywrBrDuI58HYdMsODlZy3cVxiwqAud0D1nizovaJZq9SGp5Hb0OXc1NcBR8y5YCw~bJjrLG~IDNn6Q2uQe8CT800fSdtKEPKix-Ko7qLtdlK2-soMScsUt0J5jjlz-JjMLI-nlS46UPQuDmOwwYgewXVk5PxiBaJE~CdHzf9anQBVPV~GNudBicaUoxTosw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SOMs for prokaryotic and eukaryotic sequences and for 16S rDNAs. (A) SOM was constructed with 211000 and 210600 5 kb sequences of 1502 prokaryotes and 12 eukaryotes, respectively. Prokaryote Seq. separately colored: lattice points that contain sequences from a single prokaryotic phylotype are indicated in colors as described in Fig. 3A; those that contain only eukaryotic sequences are indicated in color (); those that include both prokaryotic and eukaryotic sequences or sequences from more than one prokaryotic phylotype are indicated in black. Eukaryote Seq. separately colored: lattice points that contain only prokaryotic sequences are indicated in color (); those that include both prokaryotic and eukaryotic sequences or sequences from more than one eukaryotic species are indicated in black; those that contain sequences only from a single eukaryote are indicated in color as follows: Aspergillus nidulans (), Encephalitozoon cuniculi (), Eremothecium gossypii (), Saccharomyces cerevisiae (), Schizosaccharomyces pombe (), Neurospora crassa (), Cryptosporidium parvum (), Dictyostelium discoideum (), Plasmodium yoelii (), Theileria annulata (), Trypanosoma brucei () and zebrafish Danio rerio (). Sargasso Seq. mapped: 1 kb sequence fragments derived from contigs longer than 1 kb were mapped on the 5 kb DegeTetra-SOM after normalization of the sequence length and are indicated in color (). The residual segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show borders of eukaryotic species. Sargasso Seq. mapped, 3D: square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar in color (). (B) Species-known 16S rDNA: Tetra-SOMs were constructed for 38660 16S rDNA sequences longer than 1 kb from 19196 known prokaryotes after normalization for the sequence length. Lattice points are indicated by black or in colors as described in Fig. 3A. Species-unknown 16S rDNA: we selected 6855 sequences for which classification into 25 phylotypes is annotated in GenBank and mapped these sequences on the species-known 16S rDNA Tetra-SOM. Lattice points that contained sequences of a single phylotype annotated are indicated in color representing the phylotype, and those that included sequences from more than one phylotype are indicated in black." path-from-xml="dsi015f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193432/dsi015f4.jpeg" path-from-xml="dsi015f4.jpeg" target="_blank">View large</a><a section="87193432" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015f4.jpeg?Expires=2147483647&amp;Signature=bBFSTv8cJYHXP4uVBi5WvrHEG~5QbU8cABi6rETPfnLGPtnN4lL9Zbw7NhIdXvwS8BXLPtbFTgdd0F4M4KA-cpW8GzIwZrKf6sJcFN4aIAI7xk7LiBUOrG8EV5Yxtuyxxuqrh4AnHn-wPGARkd472oUr1Dhx8wnxJzQcz29n9~gKS1fAHqV9HSwNQA6C25FpVs63RRiHAXFyDqBI1isL9WFzBMudk4UNuvsGq3z4Yt5XMegzYtn90fwcWODbF355x4eO30EvFcHscsSnUvu896UTkUWUPRQkKjdyJO5dPDuCpZ4fp33hVT8Q3UaGMfikRPbCmwhpG2LBkPyeyBZy0A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87193432&amp;ar=350803&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsi015f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SOMs for prokaryotic and eukaryotic sequences and for 16S rDNAs. (<strong>A</strong>) SOM was constructed with 211000 and 210600 5 kb sequences of 1502 prokaryotes and 12 eukaryotes, respectively. Prokaryote Seq. separately colored: lattice points that contain sequences from a single prokaryotic phylotype are indicated in colors as described in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>; those that contain only eukaryotic sequences are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i112.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i112.jpeg?Expires=2147483647&amp;Signature=PTAJLAO0XqL7Cr7sJ8wNwzHtStEW2S-~8zYLvh0uHK7n~4GP21xjUNJaCZeKCVgrKOmHgcY62zoFsUPm-PGOv1ZJvWwtQtpYzzmJ~SSV-~nGhsHg537b4kLGnT6PL1ZjDxjk5-DY-GrR7E80FqIXS7pDdk3NuHLlD0E6uIlMpiQEqVaaLu9lCmIjR4wHoEhOAgSuaqPZn2GlPxWY~8ILiStQYUGwk06Lrb9ThNd3bxL4SJADnsC2yRoVVCLNgT4U3XuWlS9ZGOBV4wSCq3D4XBXIKDaPnH2LpYRKDd7Hen6PXAmRMWqXkJI5Oy0P5lWjpccs4Outv3xraXqquZH-Zg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i112.jpeg" /></a></div>); those that include both prokaryotic and eukaryotic sequences or sequences from more than one prokaryotic phylotype are indicated in black. Eukaryote Seq. separately colored: lattice points that contain only prokaryotic sequences are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i113.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i113.jpeg?Expires=2147483647&amp;Signature=aqDhn6FiDknThFlL9bqEtYHgz4n0I9Gp63R8cHIld65x73bXs6CdKkx4vEjZMs8EmcgbXIKTOe2CJQXKM7vVCb86NpOIE~Q0eTFUWliiLxCHPTtD5A0M10n~w0NrXHa3~WmXR0sCtWPgwZNUWkJXPTs3oKDpEC1BGJRaM5T71Shv~tfAewwZ0qQh3OktbNIpvcwCEDOmXJSGMBiu0OCQEZCddEySXFOmijjGEWBf4FoY6QKuRrTC5hJzfzJ17HsKLISr5nt~eU33eD7mAWbeRxZGnQlx-Ktw7Rmex8sGsTR9aNWITh0sauCJb-~PcEpwBdymVz69AH25fh-5ci24Aw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i113.jpeg" /></a></div>); those that include both prokaryotic and eukaryotic sequences or sequences from more than one eukaryotic species are indicated in black; those that contain sequences only from a single eukaryote are indicated in color as follows: <em>Aspergillus nidulans</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i114.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i114.jpeg?Expires=2147483647&amp;Signature=OPznyfSlv3jpuFUmkhFwNVaHxAlYN15cHArUc1thEVIeMqoHrLVEMrYHIfFGJAVMqPAaiDelvocDuz7vMuD9J6MoDWX1xB~0h7GKJkUDEabc9Obdo460TT4v3334~wmN7Ds6U~r6qmyqssHaDjjie-okO2RIMZkG~Yd~XUQA9cmpn86-yeu3E4K~I2rc6GvSnnh7GaSVB7ymec7eHZ~KOmfo-h1wtnk0mcD9rJaw9rQHufkqrfVx~af6WC~QDY6ROoWfaz96r4rWOrQdqS7DHXcdIxrEvdWLOlRZwK08Vde-KuoHbEGY~h~VKKYC5Z0~tCXLNg41Bk42a9dTFWvPTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i114.jpeg" /></a></div>), <em>Encephalitozoon cuniculi</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i115.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i115.jpeg?Expires=2147483647&amp;Signature=B7gOSzdA60HxB~Mkjw5rMqaD~tqLi7wJ-CS85FbtnBpWOKQmeewpK2~cGPkTMYyQtQoB5pVYoZ1fuJU7NvjoLt5BZL89cseidqTdIYTTxrpR8zyn~M-i9k~LO5QwNkzQoyyo5QS~NjAg8JXVcAHie7JrwLpSPE4wMaWO7gCDMbJVd1F3xxfO5h8vtNQX3PH2hQJXrS3t0La23YbV0xLE04K8Wzu71UIJ95o~fSHWU8gcxqxtqKRkgHKQfkSAOFSbUfy7SMNhZTNAZem-zmOADxbCOQSJxUxOYPps0vnQx0KCuwlGx~0AHPNd~pqIwjGAsZn3nG0YsLbVJ-itIqqsCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i115.jpeg" /></a></div>), <em>Eremothecium gossypii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i116.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i116.jpeg?Expires=2147483647&amp;Signature=BTHMALNsQ1znhhVjX5gc0ZreX1SVGhwgRezT6z-H927E7S3HSo0Yuzyl6gihAGQ~gYTcbeZ7gvqCJyEcqwIj1LQa6ozK-3g3Pbb7GnmA3HXy774oPtSlNOKX6stk5geHBzkXFqcfLupPUo~W4FqUJ9ATu3-1P8eu~65Wav52cpR-SRSHFSBDvx77ButVTb5nH29NoXqBAJi9x9UzXZPEFLrGKgBStFvnnFBF7aUc2RzBc5871y1Bb-3VYrcQRG2cChpsGej7scqmCecdfXnQFXRJ7HBe94HaSO9g2MRUOXUfijUYgitcgm0hPx67hTXxrYiU9IWP~qD1SOpsGDAw5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i116.jpeg" /></a></div>), <em>Saccharomyces cerevisiae</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i117.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i117.jpeg?Expires=2147483647&amp;Signature=PWyDZ7qoYMVB2Vn8ICryph-GjwkrWukocCCtWHoL9JU3Ofonzy8IpjvBeWq4EF53AbpV1l9TWr2zwj8H21Njp3JXp9NLfSlJ-Vh6YwOmXheq7bTlouGHr2FHDts8ZQwCWy4~f~BlvrKruDXO0eoGx6Ho9ZO7wQX9TGOEAq2XBeiJL44mmXremrqee1ZK~yZ9Fl0STP1WfwN-VNgxY2vwUzryw5kd4GkjA7SsBMpT31SWedUTwHYx7g5jy2DETXN~y8gj3l-Gk1ZL6SWYZTcSRaycpBGMRWQxoAcathTCHgrtwFMnZWYJ9OrjghTp-EUu1vkS5Grqy3q2Bt7dQ9IfxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i117.jpeg" /></a></div>), <em>Schizosaccharomyces pombe</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i118.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i118.jpeg?Expires=2147483647&amp;Signature=a2QATYdAdSsUqj89WhMJ7ul-Cq6oAxj5fy5gQYwxYd4ht0zwSessGDu-fhWYpXyUeOV98kN8sw2wITwaSI-wrCUpUDQEOZm4Xiwyg5-CfeCxuzhSclkic5vGlPpA7V567IBW6gPa0EG9RvnPLErWmsXGQqmG460o-qZNuQAUh6s1p7DV0inOTIoyO9lfS9qeMmXvQPg3OI6H6MucAdjDDus6IsIijcR8WUnye1OSXkIcU0VoScSKEkvJC6ju6Q1UnfnZ-lkUR-R0yRtAjFr4r73W8U~kZlnfMNLgBxXeRJ2dbrIvj4mHJXn5Np2heKcR~GxZMYTbtuvc5gj7HfOr5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i118.jpeg" /></a></div>), <em>Neurospora crassa</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i119.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i119.jpeg?Expires=2147483647&amp;Signature=WCnNSUhBDskZ31DeECXNXIOBQiuhdJsCaL4B0-Y4EpfztDzfaWEEqSCxg3~gg1CWRA~c6jXIyzTqcppMqDw-Pxz2Mw5~7mP5wgna3HlOkzT0AY7r~rZlTizOemgRouklL~o~NSmiJHZxQbuBAaYzpKHiBUONeWeipLsVDpa8tFKF731IRXb~DeMkso84ua5Wbb3TcHAakttpm-G24qG3PAQEfJzmWxWjaij-FwQzq-c0AD3yZ~hdguckcwegf67Seu5kN0--2mZvsXaqfkFPd83vpQL3jElUgMt7~wjVs8zq94BajpWCje5942uBhhVPIGm5X3sLXGIWv46ONCgaCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i119.jpeg" /></a></div>), <em>Cryptosporidium parvum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i120.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i120.jpeg?Expires=2147483647&amp;Signature=bFTIs4BI-5-uE0Eg3vzm80i1TSRjzfTGAfjqgjgrICie5AtMOp-b3WDn~KKauZ0Uu-ufS75LUl4O0ybVMa8quTckIgK1~pL5mlWnyIoMOH1micPaDTRHtdIgyXBFK93~7okcUfU5OfHfaEoqY6z37GNbM8fZPxUURxj9AR5Io9DHTERvjjFWRgl1LSYCdjEnA~7KDS4YJV7Kc3ckx9lN0Grg5zZx6BBdzU3jML~Z0Qu8gs7YrnotrB-NbXKlf4AI61D5UZqx3EfJ3WlKS64DmwXIyIcDhRoLbdA7LlKXq3hjIzZ1Hj0TWBZBs6l-kIPlKa6bMPr5jqzoCEfS3UylpA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i120.jpeg" /></a></div>), <em>Dictyostelium discoideum</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i121.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i121.jpeg?Expires=2147483647&amp;Signature=li30k1U18VhZyriDaBKWRDPYG-6onEfpEK7t2Trrz3SRg1kd5qo7-8eWZVEemYxlR88qXjqYBadjA~oO7NOx~X6edv5pBfdRMPH~Y3MNqT-5UT2l3SmQ7lwvhmvXQf5Kjae7z0fl7ccZvwJFz5oN~2cGcrF8oaDVGvh5tqb3pZNPSynFg33wYx317a3krYhnwckKZLdT8roVyln8R1-fu4ee-k1aFL2t2NC2Vxza8tBZGX-LMu~~U1S2yJ47XEEiNARjXeoAE7Dt2ffkBpkRNFJg-F7BZ2XB1fj0SDYsWn6Bz1GpAmyIcDZ5I1~LwuAjsx1l79xLRJTUhTSMwNIA8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i121.jpeg" /></a></div>), <em>Plasmodium yoelii</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i122.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i122.jpeg?Expires=2147483647&amp;Signature=i2z1rSQXaOSooErhvpNre~-BOqyxeQJ8xoFkAErvCXrQXKyAqJGEtMi5IOKHRF2enIHtg5il7tvYnAZtRpVcRH6ZJevIuEg3x38iTNjniDySGiZaXOM08MALKtKuCfpb1CqDTyrVJlG4UfND8vf9kKVoy8hBXjm7xGTAz~UqFhNry-AjRMUGqr1zGScnokcv4deH4aJlXtKFw8y6Do4OEiacOAPBbVN2SK~-syAZs5nzsQmC6Zxv2qLDsrkXVn5NG-gZ9YaE86IH5hoIdy7oB94SjsrajDWTPs9IsoqBjGd8doxbQ61iTOVln7opJ7IdAlBTGQRGA9JXFk86kT56sA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i122.jpeg" /></a></div>), <em>Theileria annulata</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i123.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i123.jpeg?Expires=2147483647&amp;Signature=MDJPzHiNWJzpNdYeancNwvp~yzxh7v5T1miSGkDkMTBrU9CSY5qrSDKfqxFW2TAXeUJY5GwJFusUiu5byy8t1OO3JF7gme4UNVdjd7m1AakrmIGTIdCC7tWgOGoDyhEQYkElHXCqz~siaFjXJdPM9PIcbl9vh~eFOPxfHnFvuBFoTx6RTuzwJVRxSJxIkdAUVlWDTCqTz3tbdfYiXvCD~tVtjIDUvIFW0qnx7nZyJYMisCZHyMvWFDqUtX~CfWzOPMCgzAsvqGmG1fUTMnpJQ43i7kCOMlV64tKjSrTIoeW3rptAxzHRlhEfwIrIphQjD6fxadmKA66IwRyrWlzKow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i123.jpeg" /></a></div>), <em>Trypanosoma brucei</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i124.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i124.jpeg?Expires=2147483647&amp;Signature=fKgXQNfmhJ~Pp1EyLb3KLkDt-YhrA0INrxYQwIGHweJur-S41ZwAfmsjIeZlu19IZR-bLjsmr4gcj0EeTvhjksz3q3Bow9Q1W3wFgyfHUSmT7CYiDZTWfAdP7SLHum5xqd~869vvlgX75-jhwmAQ~ztOX~OcTYhg5Xo~yfB89fb01nfODmambh5S0S3WPzqT9YTb8aAmdd5LEqMfH6jVfThxtMxo-w4dhNBzhi36GXT19ohzfsKppNR7bxylzZhefoCfpKGF2GCmHvrr748~r5fH6xoL6C2NFm9hqxPC9sHoxhNxDkdk7RPBrBK-7tK2RKU9dBE9Yx1AIO2P9D-VNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i124.jpeg" /></a></div>) and zebrafish <em>Danio rerio</em> (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i125.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i125.jpeg?Expires=2147483647&amp;Signature=nU43CF9Rg4e4KkN5bO2AeRuV3jmDeaCnJkdGsRIFgbfuM-5E7A4VvqkcYDBljD-b-bBI2dV1JcSQoLTJFN01xgDuj5ZjgzmNeLl3gkNF7vo1z1Sm6OYlQt9Vt88uO7rmJFOkXX~aVHdnPn6NvpDQVwJ1ehuOYJhGM9EuEv9rvThO9rGb9rwT5z9WIizUJZV6xuIVYAOay-XQdjoM103Tx2suXXy~shy6j804kP8N8zB8j89C91V1BcQ4Tdl9A3IkxTPGcoSJe7NZ5eUtZ2OQJGHB4Oy8iWM7hk-XrUCNf7o7YRyl0egXZUR1oP191NTtNqQZLEiDafSmpvecCbuB5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i125.jpeg" /></a></div>). Sargasso Seq. mapped: 1 kb sequence fragments derived from contigs longer than 1 kb were mapped on the 5 kb DegeTetra-SOM after normalization of the sequence length and are indicated in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i126.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i126.jpeg?Expires=2147483647&amp;Signature=hRA6WIfIfYSf0fykM0Lp7P0PZR~JHTdDE8bSST4gWABHFeeu5rRTgX0HUuJnnEcP~u3g3T56TZfIi34rg2caT0z73MU3EVxlDN1LWtkfzMtPn910mRLy8zV1TpsGHUqd9pClSs8LFROqu9UQD640O2lMr2nulO2P0H28QsDQiMOX1QM4Oy73l30eDccA7kayioG57~A3a3RhaphMNAp9UqWg5hEqJdoR2WWGlgSejsaboiq4BWaSFbjqW6hs94NZ559lAU8BkOxWw8YEKf5rPODq4BPOd7if5-B0Vw8VBrJn1gP9ZgnPG3~SK3dEiTDgnGXV-rz7HWXSlg~gUmKXAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i126.jpeg" /></a></div>). The residual segment &lt;1 kb derived from each contig was omitted from the analysis. Lines were drawn to show borders of eukaryotic species. Sargasso Seq. mapped, 3D: square root of the number of Sargasso sequences mapped on each lattice point is indicated by the height of the bar in color (<div class="fig-graphic"><a class="jumplink-placeholder"  id="dsi015i126.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/m_dsi015i126.jpeg?Expires=2147483647&amp;Signature=hRA6WIfIfYSf0fykM0Lp7P0PZR~JHTdDE8bSST4gWABHFeeu5rRTgX0HUuJnnEcP~u3g3T56TZfIi34rg2caT0z73MU3EVxlDN1LWtkfzMtPn910mRLy8zV1TpsGHUqd9pClSs8LFROqu9UQD640O2lMr2nulO2P0H28QsDQiMOX1QM4Oy73l30eDccA7kayioG57~A3a3RhaphMNAp9UqWg5hEqJdoR2WWGlgSejsaboiq4BWaSFbjqW6hs94NZ559lAU8BkOxWw8YEKf5rPODq4BPOd7if5-B0Vw8VBrJn1gP9ZgnPG3~SK3dEiTDgnGXV-rz7HWXSlg~gUmKXAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsi015i126.jpeg" /></a></div>). (<strong>B</strong>) Species-known 16S rDNA: Tetra-SOMs were constructed for 38660 16S rDNA sequences longer than 1 kb from 19196 known prokaryotes after normalization for the sequence length. Lattice points are indicated by black or in colors as described in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. Species-unknown 16S rDNA: we selected 6855 sequences for which classification into 25 phylotypes is annotated in GenBank and mapped these sequences on the species-known 16S rDNA Tetra-SOM. Lattice points that contained sequences of a single phylotype annotated are indicated in color representing the phylotype, and those that included sequences from more than one phylotype are indicated in black.</p></div></div><p>Genome segments introduced through horizontal gene transfer from a phylogenetically distant genome tend to retain the sequence characteristics of the donor genome and can be distinguished from those of the host genome. Even in the 5 kb DegeTetra-SOM in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a>, there are lattice points marked in black which should contain sequences with tetranucleotide frequencies distinct from the major portion of the host genome. Such sequences may correspond, at least in part, to segments transferred horizontally from a distant phylotype. When we investigated <em>Bacillus subtilis</em> sequences located outside the Firmicutes territory, we found that many were derived from pathogenic islands where prophage- and foreign-type sequences are clustered.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> The SOM, as well as the sequence homology-based method, may classify such horizontally transferred sequences into the donor genome. Although the information regarding horizontal transfer is interesting, it creates problems in phylogenetic classification of environmental sequences. Therefore, it is desirable to develop a strategy to address this problem. When we constructed Tetra- and DegeTetra-SOMs with 38660 16S ribosomal RNA gene (rDNA) sequences longer than 1 kb from 19196 known prokaryotes available in GenBank, clear clustering according to phylotype was found with reference to the NCBI Taxonomy Database; 97 and 95% of sequences were classified into the correct phylotype territory on the Tetra- and DegeTetra-SOMs, respectively, and the result of the Tetra-SOM is presented (Species-known 16S rDNA in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). The reason why the hit level of Tetra-SOM is slightly higher than that of the DegeTetra-SOM may be attributable to the fact that rDNA sequences with the same polarity are registered in GenBank for all species. This is in contrast with the situation for usual genomic sequences. The SOM may detect details of sequence characteristics of 16S rDNAs that can be identified only in sequences with the same polarity.</p><p>When we focused on 16S rDNA sequences from species-unidentified prokaryotes in GenBank, we found 16652 sequences longer than 1 kb. Because the main purpose of 16S rDNA sequencing of the unidentified prokaryotes in environments is phylogenetic assignment, phylotypes assigned with the homology-based method are often annotated in GenBank. We selected 6855 sequences for which classification into major phylotypes is annotated and mapped these sequences on the species-known 16S rDNA Tetra-SOM (Species-unknown 16S rDNA in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). Lattice points that contained sequences of a single phylotype annotated are indicated in color representing the phylotype, and those that included sequences from more than one phylotype are indicated in black. The major zones were marked in color, and the color pattern was almost identical to that of known species. Although the SOM does not require sequence alignments, assignments were practically identical to those obtained by the homology-based method for this case where a large number of orthologous sequences are available. When extensive shotgun sequencing of an environmental sample is done, substantial amounts of both non-rDNA and rDNA sequences become available. Combined SOM analyses of rDNA and non-rDNA sequences may provide detailed information regarding microbial diversity on the basis of detailed knowledge of rDNA phylogeny. This may also solve, at least in part, complications caused by horizontal transfer of sequences. For example, when no rDNA sequences are found in a certain phylotype territory in the rDNA SOM, but a statistically significant number of non-rDNA sequences is found in the respective phylotype territory in the SOM constructed with usual genomic sequences, these non-rDNA sequences may be candidates for horizontally transferred sequences. When a statistically significant number of rDNA sequences map to a certain phylotype territory in the rDNA SOM and is assigned to be derived from one species with the homology-based method on the basis of the detailed rDNA phylogeny, a major portion of the non-rDNA sequences mapped to this phylotype territory on the standard SOM can be predicted to be derived from the respective species. Collectively, the combination of SOMs for rDNA and non-rDNA sequences will reconstruct <em>in silico</em> the rDNA and non-rDNA sequences derived from a single genome but cloned independently.</p>                            <h2 scrollto-destination=87193435 id="87193435" class="section-title" >Supplementary material</h2>
<p>Supplementary material is available online at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p><p>This work was supported by grants from ACT-Japan Science and Technology Corporation and the Advanced and Innovational Research Program in Life Sciences and a Grant-in-Aid for Scientific Research on Priority Areas Applied Genome from the Ministry of Education, Culture, Sports, Science and Technology of Japan. A part of the calculation was done with the Earth Simulator of Japan Agency of Marine-Earth Science and Technology.</p>                            <h2 scrollto-destination=87193439 id="87193439" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amann</div> <div class="given-names">R. I.</div></div>, <div class="name"><div class="surname">Ludwig</div> <div class="given-names">W.</div></div>, <div class="name"><div class="surname">Schleifer</div> <div class="given-names">K. H.</div></div>.</div><div class="article-title">Phylogenetic identification and <em>in situ</em> detection of individual microbial cells without cultivation</div>,<div class="source">Microbiol. Rev.</div>,<div class="year">1995</div>,vol.<div class="volume">59</div>(pg.<div class="fpage">143</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20identification%20and%20in%20situ%20detection%20of%20individual%20microbial%20cells%20without%20cultivation&amp;author=R.%20I.%20Amann&amp;author=W.%20Ludwig&amp;author=K.%20H.%20Schleifer&amp;publication_year=1995&amp;journal=Microbiol.%20Rev.&amp;volume=59&amp;pages=143-169" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7535888" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>, <div class="name"><div class="surname">Pace</div> <div class="given-names">N. R.</div></div>.</div><div class="article-title">Identifying microbial diversity in the natural environment: a molecular phylogenetic approach</div>,<div class="source">Trends Biotechnol.</div>,<div class="year">1996</div>,vol.<div class="volume">14</div>(pg.<div class="fpage">190</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20microbial%20diversity%20in%20the%20natural%20environment%3A%20a%20molecular%20phylogenetic%20approach&amp;author=P.%20Hugenholtz&amp;author=N.%20R.%20Pace&amp;publication_year=1996&amp;journal=Trends%20Biotechnol.&amp;volume=14&amp;pages=190-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0167-7799(96)10025-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0167-7799(96)10025-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0167-7799(96)10025-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8663938" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rondon</div> <div class="given-names">M. R.</div></div>, <div class="name"><div class="surname">August</div> <div class="given-names">P. R.</div></div>, <div class="name"><div class="surname">Bettermann</div> <div class="given-names">A. D.</div></div>,et al.</div><div class="article-title">Cloning the soil metagenome: a strategy for accessing the genetic and functional diversity of uncultured microorganisms</div>,<div class="source">Appl. Environ. Microbiol.</div>,<div class="year">2000</div>,vol.<div class="volume">66</div>(pg.<div class="fpage">2541</div>-<div class="lpage">2547</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20the%20soil%20metagenome%3A%20a%20strategy%20for%20accessing%20the%20genetic%20and%20functional%20diversity%20of%20uncultured%20microorganisms&amp;author=M.%20R.%20Rondon&amp;author=P.%20R.%20August&amp;author=A.%20D.%20Bettermann&amp;publication_year=2000&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=66&amp;pages=2541-2547" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.66.6.2541-2547.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.66.6.2541-2547.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.66.6.2541-2547.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10831436" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lorenz</div> <div class="given-names">P.</div></div>, <div class="name"><div class="surname">Liebeton</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Niehaus</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Eck</div> <div class="given-names">J.</div></div>.</div><div class="article-title">Screening for novel enzymes for biocatalytic processes: accessing the metagenome as a resource of novel functional sequence space</div>,<div class="source">Curr. Opin. Biotechnol.</div>,<div class="year">2002</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">572</div>-<div class="lpage">577</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Screening%20for%20novel%20enzymes%20for%20biocatalytic%20processes%3A%20accessing%20the%20metagenome%20as%20a%20resource%20of%20novel%20functional%20sequence%20space&amp;author=P.%20Lorenz&amp;author=K.%20Liebeton&amp;author=F.%20Niehaus&amp;author=J.%20Eck&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=13&amp;pages=572-577" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(02)00345-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(02)00345-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(02)00345-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12482516" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DeLong</div> <div class="given-names">E. F.</div></div>.</div><div class="article-title">Microbial population genomics and ecology</div>,<div class="source">Curr. Opin. Microbiol.</div>,<div class="year">2002</div>,vol.<div class="volume">5</div>(pg.<div class="fpage">520</div>-<div class="lpage">524</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20population%20genomics%20and%20ecology&amp;author=E.%20F.%20DeLong&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=5&amp;pages=520-524" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(02)00353-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(02)00353-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(02)00353-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12354561" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schloss</div> <div class="given-names">P. D.</div></div>, <div class="name"><div class="surname">Handelsman</div> <div class="given-names">J.</div></div>.</div><div class="article-title">Biotechnological prospects from metagenomics</div>,<div class="source">Curr. Opin. Biotechnol.</div>,<div class="year">2003</div>,vol.<div class="volume">14</div>(pg.<div class="fpage">303</div>-<div class="lpage">310</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biotechnological%20prospects%20from%20metagenomics&amp;author=P.%20D.%20Schloss&amp;author=J.%20Handelsman&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=14&amp;pages=303-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(03)00067-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(03)00067-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(03)00067-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12849784" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>.</div><div class="article-title">The self-organizing map</div>,<div class="source">Proc. IEEE</div>,<div class="year">1990</div>,vol.<div class="volume">78</div>(pg.<div class="fpage">1464</div>-<div class="lpage">1480</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20self-organizing%20map&amp;author=T.%20Kohonen&amp;publication_year=1990&amp;journal=Proc.%20IEEE&amp;volume=78&amp;pages=1464-1480" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/5.58325" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2F5.58325" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2F5.58325"> </span></div></div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Oja</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Simula</div> <div class="given-names">O.</div></div>, <div class="name"><div class="surname">Visa</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Kangas</div> <div class="given-names">J.</div></div>.</div><div class="article-title">Engineering applications of the self-organizing map</div>,<div class="source">Proc. IEEE</div>,<div class="year">1996</div>,vol.<div class="volume">84</div>(pg.<div class="fpage">1358</div>-<div class="lpage">1384</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20applications%20of%20the%20self-organizing%20map&amp;author=T.%20Kohonen&amp;author=E.%20Oja&amp;author=O.%20Simula&amp;author=A.%20Visa&amp;author=J.%20Kangas&amp;publication_year=1996&amp;journal=Proc.%20IEEE&amp;volume=84&amp;pages=1358-1384" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/5.537105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2F5.537105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2F5.537105"> </span></div></div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Ichiba</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Kozuki</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Informatics for unveiling hidden genome signatures</div>,<div class="source">Genome Res.</div>,<div class="year">2003</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">693</div>-<div class="lpage">702</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Informatics%20for%20unveiling%20hidden%20genome%20signatures&amp;author=T.%20Abe&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;author=Y.%20Ichiba&amp;author=T.%20Kozuki&amp;author=T.%20Ikemura&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=693-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.634603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.634603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.634603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671005" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,et al.</div><div class="article-title">Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the E. coli O157 genome</div>,<div class="source">Gene</div>,<div class="year">2001</div>,vol.<div class="volume">276</div>(pg.<div class="fpage">89</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20codon%20usage%20diversity%20of%20bacterial%20genes%20with%20a%20self-organizing%20map%20(SOM)%3A%20characterization%20of%20horizontally%20transferred%20genes%20with%20emphasis%20on%20the%20E.%20coli%20O157%20genome&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;author=T.%20Abe&amp;publication_year=2001&amp;journal=Gene&amp;volume=276&amp;pages=89-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(01)00673-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(01)00673-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(01)00673-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591475" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venter</div> <div class="given-names">J. C.</div></div>, <div class="name"><div class="surname">Remington</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J. F.</div></div>,et al.</div><div class="article-title">Environmental genome shotgun sequencing of the Sargasso Sea</div>,<div class="source">Science</div>,<div class="year">2004</div>,vol.<div class="volume">304</div>(pg.<div class="fpage">66</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20genome%20shotgun%20sequencing%20of%20the%20Sargasso%20Sea&amp;author=J.%20C.%20Venter&amp;author=K.%20Remington&amp;author=J.%20F.%20Heidelberg&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=66-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1093857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1093857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1093857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15001713" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Campbell</div> <div class="given-names">A. M.</div></div>, <div class="name"><div class="surname">Mrazek</div> <div class="given-names">J.</div></div>.</div><div class="article-title">Comparative DNA analysis across diverse genomes</div>,<div class="source">Annu. Rev. Genet.</div>,<div class="year">1998</div>,vol.<div class="volume">32</div>(pg.<div class="fpage">185</div>-<div class="lpage">225</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20DNA%20analysis%20across%20diverse%20genomes&amp;author=S.%20Karlin&amp;author=A.%20M.%20Campbell&amp;author=J.%20Mrazek&amp;publication_year=1998&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=32&amp;pages=185-225" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.32.1.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.32.1.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.32.1.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9928479" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nussinov</div> <div class="given-names">R.</div></div>.</div><div class="article-title">Doublet frequencies in evolutionary distinct groups</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">1984</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">1749</div>-<div class="lpage">1763</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Doublet%20frequencies%20in%20evolutionary%20distinct%20groups&amp;author=R.%20Nussinov&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=1749-1763" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.3.1749" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.3.1749" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.3.1749"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6583663" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gentles</div> <div class="given-names">A. J.</div></div>, <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>.</div><div class="article-title">Genome-scale compositional comparisons in eukaryotes</div>,<div class="source">Genome Res.</div>,<div class="year">2001</div>,vol.<div class="volume">11</div>(pg.<div class="fpage">540</div>-<div class="lpage">546</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20compositional%20comparisons%20in%20eukaryotes&amp;author=A.%20J.%20Gentles&amp;author=S.%20Karlin&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=540-546" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.163101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.163101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.163101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11282969" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Teeling</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Meyerdierks</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Bauer</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Amann</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Glockner</div> <div class="given-names">F. O.</div></div>.</div><div class="article-title">Application of tetranucleotide frequencies for the assignment of genomic fragments</div>,<div class="source">Environ. Microbiol.</div>,<div class="year">2004</div>,vol.<div class="volume">6</div>(pg.<div class="fpage">938</div>-<div class="lpage">947</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20tetranucleotide%20frequencies%20for%20the%20assignment%20of%20genomic%20fragments&amp;author=H.%20Teeling&amp;author=A.%20Meyerdierks&amp;author=M.%20Bauer&amp;author=R.%20Amann&amp;author=F.%20O.%20Glockner&amp;publication_year=2004&amp;journal=Environ.%20Microbiol.&amp;volume=6&amp;pages=938-947" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2004.6.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2004.6.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2004.6.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15305919" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tyson</div> <div class="given-names">G. W.</div></div>, <div class="name"><div class="surname">Chapman</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>,et al.</div><div class="article-title">Community structure and metabolism through reconstruction of microbial genomes from the environment</div>,<div class="source">Nature</div>,<div class="year">2004</div>,vol.<div class="volume">428</div>(pg.<div class="fpage">37</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Community%20structure%20and%20metabolism%20through%20reconstruction%20of%20microbial%20genomes%20from%20the%20environment&amp;author=G.%20W.%20Tyson&amp;author=J.%20Chapman&amp;author=P.%20Hugenholtz&amp;publication_year=2004&amp;journal=Nature&amp;volume=428&amp;pages=37-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14961025" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiyama</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Watanabe</div> <div class="given-names">K.</div></div>.</div><div class="article-title">Substrate-induced gene-expression screening of environmental metagenome libraries for isolation of catabolic genes</div>,<div class="source">Nat. Biotechnol.</div>,<div class="year">2005</div>,vol.<div class="volume">23</div>(pg.<div class="fpage">88</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substrate-induced%20gene-expression%20screening%20of%20environmental%20metagenome%20libraries%20for%20isolation%20of%20catabolic%20genes&amp;author=T.%20Uchiyama&amp;author=T.%20Abe&amp;author=T.%20Ikemura&amp;author=K.%20Watanabe&amp;publication_year=2005&amp;journal=Nat.%20Biotechnol.&amp;volume=23&amp;pages=88-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608629" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Sakamoto</div> <div class="given-names">M.</div></div>,et al.</div><div class="article-title">Direct cloning of genes encoding novel xylanases from human gut</div>,<div class="source">Can. J. Microbiol.</div>,<div class="year">2005</div>,vol.<div class="volume">51</div>(pg.<div class="fpage">251</div>-<div class="lpage">259</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20cloning%20of%20genes%20encoding%20novel%20xylanases%20from%20human%20gut&amp;author=H.%20Hayashi&amp;author=T.%20Abe&amp;author=M.%20Sakamoto&amp;publication_year=2005&amp;journal=Can.%20J.%20Microbiol.&amp;volume=51&amp;pages=251-259" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/w04-136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fw04-136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fw04-136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15920623" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunst</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Moszer</div> <div class="given-names">I.</div></div>,et al.</div><div class="article-title">The complete genome sequence of the gram-positive bacterium Bacillus subtilis</div>,<div class="source">Nature</div>,<div class="year">1997</div>,vol.<div class="volume">390</div>(pg.<div class="fpage">249</div>-<div class="lpage">256</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20gram-positive%20bacterium%20Bacillus%20subtilis&amp;author=F.%20Kunst&amp;author=N.%20Ogasawara&amp;author=I.%20Moszer&amp;publication_year=1997&amp;journal=Nature&amp;volume=390&amp;pages=249-256" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/36786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F36786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F36786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9384377" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=350803&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/5/10.1093_dnares_dsi015/1/dsi015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=c35M3h-Iv16wJoMhCE2vFBkhsnh8CaA9x1OPYmWEXxNn8E9SMJt595-4kJVSMXC7GNwD~L0r8AW9IRQ28~p0INsCS4LkxPqtjeiaEFyolSiiPEVSP90waHCSG3ns5grldWpIg~xHy7LT3-hImtfgJtglZX4cIZ3Z3ZRfMTCj9Mwtc33tTUOOSSCmIV7mUz0DXX7NAfUedfgAGHG48D3TJ~Uk24rsMY~SZkwlAdiUXC~rVg4wwrlG~Aw32mIFaJsQIpLiCuD7rkvWUnFME2ulfDH0sO9GrGYhoeCYu8aBO~55KGXjdt0-fnxx2i749~SvS-7kV7ZpiQTRimDKPXxUJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">249</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119800002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">76</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="350803"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119800002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f5%2f281%2f350803">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119800002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (76)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F5%2F281%2F350803" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/350803'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Novel Phylogenetic Studies of Genomic Sequence Fragments Derived from Uncultured Microbe Mixtures in Environmental and Clinical Samples" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f5%2f281%2f350803">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

