

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/1/1/484514","@type":"ScholarlyArticle","name":"Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in  Ziziphus Celata  (Rhamnaceae), a Highly Endangered, Florida Endemic Plant","datePublished":"2011-10-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2012-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr037","keywords":["transcriptome","conservation genetics","microsatellites","S-locus","<italic>Ziziphus</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/1/1/484514","author":[{"name":"Edwards, Christine E.","affiliation":"1  USACE ERDC, Environmental Laboratory, 3909 Halls Ferry Road, Vicksburg, MS 39180 ,  USA","@type":"Person"},{"name":"Parchman, Thomas L.","affiliation":"2  Department of Botany ,  University of Wyoming ,  Laramie, WY 82071 ,  USA","@type":"Person"},{"name":"Weekley, Carl W.","affiliation":"3  Archbold Biological Station ,  PO Box 2057, Lake Placid, FL 33862 ,  USA","@type":"Person"}],"description":"Abstract.  Large-scale DNA sequence data may enable development of genetic resources in endangered species, thereby facilitating conservation efforts. Ziziphus ","pageStart":"1","pageEnd":"9","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in  Ziziphus Celata  (Rhamnaceae), a Highly Endangered, Florida Endemic Plant","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in  Ziziphus Celata  (Rhamnaceae), a Highly Endangered, Florida Endemic Plant" />
    <meta property="og:description" content="Abstract.  Large-scale DNA sequence data may enable development of genetic resources in endangered species, thereby facilitating conservation efforts. Ziziphus " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/1/1/484514" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-10-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Edwards, Christine E." /><meta name="citation_author_institution" content="USACE ERDC, Environmental Laboratory, 3909 Halls Ferry Road, Vicksburg, MS 39180, USA" /><meta name="citation_author" content="Parchman, Thomas L." /><meta name="citation_author_institution" content="Department of Botany, University of Wyoming, Laramie, WY 82071, USA" /><meta name="citation_author" content="Weekley, Carl W." /><meta name="citation_author_institution" content="Archbold Biological Station, PO Box 2057, Lake Placid, FL 33862, USA" /><meta name="citation_title" content="Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in Ziziphus Celata (Rhamnaceae), a Highly Endangered, Florida Endemic Plant" /><meta name="citation_firstpage" content="1" /><meta name="citation_lastpage" content="9" /><meta name="citation_doi" content="10.1093/dnares/dsr037" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2012/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The expansion of conservation genetics; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=702-12; " /><meta name="citation_reference" content="citation_title=Rapid genetic decline in a translocated population of the endangered plant Grevillea scapigera; Conserv. Biol.;  citation_year=2002;  citation_volume=16;  citation_pages=986-94; " /><meta name="citation_reference" content="citation_title=Parentage and relatedness in captive and natural populations of the roseate spoonbill (Aves: Ciconiiformes) based on microsatellite Data; J. Exp. Zool. Part A;  citation_year=2009;  citation_volume=311A;  citation_pages=453-64; " /><meta name="citation_reference" content="citation_title=Genetic diversity following demographic recovery in the insular endemic plant Galium catalinense subspecies acrispum; Conserv. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=2015-25; " /><meta name="citation_reference" content="citation_title=Evidence for recent population bottlenecks in northern spotted owls (Strix occidentalis caurina); Conserv. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=1013-21; " /><meta name="citation_reference" content="citation_title=Rampant drift in artificially fragmented populations of the endangered tidewater goby (Eucyclogobius newberryi); Mol. Ecol.;  citation_year=2010;  citation_volume=19;  citation_pages=3315-27; " /><meta name="citation_reference" content="citation_title=Genomics and the future of conservation genetics; Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=697-709; " /><meta name="citation_reference" content="citation_title=The molecular ecologists guide to expressed sequence tags; Mol. Ecol.;  citation_year=2007;  citation_volume=16;  citation_pages=907-24; " /><meta name="citation_reference" content="citation_title=The impact of next-generation sequencing technology on genetics; Trends Genet.;  citation_year=2008;  citation_volume=24;  citation_pages=133-41; " /><meta name="citation_reference" content="citation_title=Sequencing breakthroughs for genomic ecology and evolutionary biology; Mol. Ecol. Resour.;  citation_year=2008;  citation_volume=8;  citation_pages=3-17; " /><meta name="citation_reference" content="citation_title=Rapidly developing functional genomics in ecological model systems via 454 transcriptome sequencing; Genetica;  citation_year=2010;  citation_volume=138;  citation_pages=433-51; " /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=Rapid transcriptome characterization for a nonmodel organism using 454 pyrosequencing; Mol. Ecol.;  citation_year=2008;  citation_volume=17;  citation_pages=1636-47; " /><meta name="citation_reference" content="citation_title=Sequencing goes 454 and takes large-scale genomics into the wild; Mol. Ecol.;  citation_year=2008;  citation_volume=17;  citation_pages=1629-31; " /><meta name="citation_reference" content="citation_title=The new paradigm of flow cell sequencing; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=839-46; " /><meta name="citation_reference" content="citation_title=Sequencing and de novo analysis of a coral larval transcriptome using 454 GS-FLX; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=219" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=180" /><meta name="citation_reference" content="citation_title=Development of genic SSR markers by deep transcriptome sequencing in pigeonpea Cajanus cajan (L.) Millspaugh; BMC Plant Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=17" /><meta name="citation_reference" content="citation_title=De novo transcriptome characterization and development of genomic tools for Scabiosa columbaria L. using next-generation sequencing techniques; Mol. Ecol. Resour.;  citation_year=2011;  citation_volume=11;  citation_pages=662-74; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of lentil based on second-generation technology permits large-scale unigene assembly and SSR marker discovery; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=265" /><meta name="citation_reference" content="citation_title=De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=53-63; " /><meta name="citation_reference" content="citation_title=Transcriptome characterization and polymorphism detection between subspecies of big sagebrush (Artemisia tridentata); BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=370" /><meta name="citation_reference" content="citation_title=Next-generation pyrosequencing of gonad transcriptomes in the polyploid lake sturgeon (Acipenser fulvescens): the relative merits of normalization and rarefaction in gene discovery; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=203" /><meta name="citation_reference" content="citation_title=A population genetic analysis of Ziziphus celata, an endangered Florida shrub; J. Hered.;  citation_year=1997;  citation_volume=88;  citation_pages=531-3; " /><meta name="citation_reference" content="citation_title=Genetic impoverishment and cross-incompatibility in remnant genotypes of Ziziphus celata (Rhamnaceae), a rare shrub endemic to the Lake Wales Ridge, Florida; Biodivers. Conserv.;  citation_year=2002;  citation_volume=11;  citation_pages=2027-46; " /><meta name="citation_reference" content="citation_title=Microsatellite evidence for high clonality and limited genetic diversity in Ziziphus celata (Rhamnaceae), an endangered, self-incompatible Lake Wales Ridge, Florida, USA endemic; citation_text=in review; Conserv. Genet; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Inc, Sunderland; Plant Systematics: A Phylogenetic Approach;  citation_year=2008; " /><meta name="citation_reference" content="citation_title=Florida Ziziphus; citation_author=USFWS; citation_publisher=U.S. Fish and Wildlife Service, Atlanta, GA; Multi-species Recovery Plan for the Threatened and Endangered Species of South Florida;  citation_year=1999;  citation_pages=1986-99; " /><meta name="citation_reference" content="citation_author=USFWS; citation_publisher=U.S. Fish and Wildlife Service, Vero Beach, FL; Florida Ziziphus (Ziziphus celata 5-year Status Review: Summary and Evaluation;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Reproductive biology of Colletia spinosissima (Rhamnaceae) in Argentina; Plant Syst. Evol.;  citation_year=2001;  citation_volume=229;  citation_pages=79-89; " /><meta name="citation_reference" content="citation_title=Protandry, pollination, and self-incompatibility in Discaria toumatou; N. Z. J. Bot.;  citation_year=1985;  citation_volume=23;  citation_pages=331-5; " /><meta name="citation_reference" content="citation_title=Nonrandom mating in Discaria americana (Rhamnaceae); Plant Syst. Evol.;  citation_year=1996;  citation_volume=203;  citation_pages=179-80; " /><meta name="citation_reference" content="citation_title=Reproductive biology of Frangula alnus (Rhamnaceae) in southern Spain; Plant Syst. Evol.;  citation_year=1994;  citation_volume=193;  citation_pages=173-86; " /><meta name="citation_reference" content="citation_title=Reproductive biology of the andromonoecious shrub Trevoa quinquenervia (Rhamnaceae); Bot. J. Linn. Soc.;  citation_year=1998;  citation_volume=12;  citation_pages=191-206; " /><meta name="citation_reference" content="citation_title=The breeding system of Ziziphus celata Judd and DW Hall (Rhamnaceae), a rare endemic plant of the Lake Wales Ridge, Florida, USA: implications for recovery; Biol. Conserv.;  citation_year=2001;  citation_volume=100;  citation_pages=207-13; " /><meta name="citation_reference" content="citation_title=Flowering of Ziziphus mucronata subsp. mucronata (Rhamnaceae)—anthesis, pollination and protein synthesis; Bot. Bull. Acad. Sinica;  citation_year=1992;  citation_volume=33;  citation_pages=33-42; " /><meta name="citation_reference" content="citation_title=Summary of 2005–2007 observations of pollen grains and pollen tube growth in Florida ziziphus and observations of open-pollinations and aborted fruits; citation_publisher=Final Report to Plant Conservation Program, Florida Division of Forestry, Tallahassee, FL; Continuation of Research on the Federally-listed Lake Wales Ridge Endemic Florida Ziziphus (Ziziphus celata);  citation_year=2007; " /><meta name="citation_reference" content="citation_title=An S-RNase-based gametophytic self-incompatibility system evolved only once in eudicots; J. Mol. Evol.;  citation_year=2008;  citation_volume=67;  citation_pages=179-90; " /><meta name="citation_reference" content="citation_title=Evolutionary relationships among self-incompatibility RNases; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=13167-71; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=UniRef: comprehensive and non-redundant UniProt reference clusters; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1282-8; " /><meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): gene structure and function annotation; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D1009-14; " /><meta name="citation_reference" content="citation_text=2.0 a11; Se-Al: A Manual Sequence Alignment Editor;  citation_year=2003; " /><meta name="citation_reference" content="citation_title=BatchPrimer3: a high throughput web application for PCR and sequencing primer design; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=253" /><meta name="citation_reference" content="citation_title=De novo characterization of the gametophyte transcriptome in bracken fern, Pteridium aquilinum; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=99" /><meta name="citation_reference" content="citation_title=Gene discovery and annotation using LCM-454 transcriptome sequencing; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=69-73; " /><meta name="citation_reference" content="citation_title=High-throughput gene and SNP discovery in Eucalyptus grandis, an uncharacterized genome; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=312" /><meta name="citation_reference" content="citation_title=EST-SSRs as a resource for population genetic analyses; Heredity;  citation_year=2007;  citation_volume=99;  citation_pages=125-32; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/1/1/1309780/dsr037.pdf" /><meta name="description" content="Abstract.  Large-scale DNA sequence data may enable development of genetic resources in endangered species, thereby facilitating conservation efforts. Ziziphus " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/1/1/484514" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/1/1/484514" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/1/1/484514");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr037';
    </script>
<title>
	Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in Ziziphus Celata (Rhamnaceae), a Highly Endangered, Florida Endemic Plant | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484514" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdplPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJFZHdhcmRzLCBDaHJpc3RpbmUgRS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVU0FDRSBFUkRDLCBFbnZpcm9ubWVudGFsIExhYm9yYXRvcnksIDM5MDkgSGFsbHMgRmVycnkgUm9hZCwgVmlja3NidXJnLCBNUyAzOTE4MCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyY2htYW4sIFRob21hcyBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQm90YW55LCBVbml2ZXJzaXR5IG9mIFd5b21pbmcsIExhcmFtaWUsIFdZIDgyMDcxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXZWVrbGV5LCBDYXJsIFcuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQXJjaGJvbGQgQmlvbG9naWNhbCBTdGF0aW9uLCBQTyBCb3ggMjA1NywgTGFrZSBQbGFjaWQsIEZMIDMzODYyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkFzc2VtYmx5LCBHZW5lIEFubm90YXRpb24gYW5kIE1hcmtlciBEZXZlbG9wbWVudCBVc2luZyA0NTQgRmxvcmFsIFRyYW5zY3JpcHRvbWUgU2VxdWVuY2VzIGluIFppemlwaHVzIENlbGF0YSAoUmhhbW5hY2VhZSksIGEgSGlnaGx5IEVuZGFuZ2VyZWQsIEZsb3JpZGEgRW5kZW1pYyBQbGFudCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBleHBhbnNpb24gb2YgY29uc2VydmF0aW9uIGdlbmV0aWNzOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz03MDItMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGdlbmV0aWMgZGVjbGluZSBpbiBhIHRyYW5zbG9jYXRlZCBwb3B1bGF0aW9uIG9mIHRoZSBlbmRhbmdlcmVkIHBsYW50IEdyZXZpbGxlYSBzY2FwaWdlcmE7IENvbnNlcnYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9OTg2LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJlbnRhZ2UgYW5kIHJlbGF0ZWRuZXNzIGluIGNhcHRpdmUgYW5kIG5hdHVyYWwgcG9wdWxhdGlvbnMgb2YgdGhlIHJvc2VhdGUgc3Bvb25iaWxsIChBdmVzOiBDaWNvbmlpZm9ybWVzKSBiYXNlZCBvbiBtaWNyb3NhdGVsbGl0ZSBEYXRhOyBKLiBFeHAuIFpvb2wuIFBhcnQgQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTFBOyAgY2l0YXRpb25fcGFnZXM9NDUzLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBmb2xsb3dpbmcgZGVtb2dyYXBoaWMgcmVjb3ZlcnkgaW4gdGhlIGluc3VsYXIgZW5kZW1pYyBwbGFudCBHYWxpdW0gY2F0YWxpbmVuc2Ugc3Vic3BlY2llcyBhY3Jpc3B1bTsgQ29uc2Vydi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjAxNS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgZm9yIHJlY2VudCBwb3B1bGF0aW9uIGJvdHRsZW5lY2tzIGluIG5vcnRoZXJuIHNwb3R0ZWQgb3dscyAoU3RyaXggb2NjaWRlbnRhbGlzIGNhdXJpbmEpOyBDb25zZXJ2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDEzLTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYW1wYW50IGRyaWZ0IGluIGFydGlmaWNpYWxseSBmcmFnbWVudGVkIHBvcHVsYXRpb25zIG9mIHRoZSBlbmRhbmdlcmVkIHRpZGV3YXRlciBnb2J5IChFdWN5Y2xvZ29iaXVzIG5ld2JlcnJ5aSk7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zMzE1LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBhbmQgdGhlIGZ1dHVyZSBvZiBjb25zZXJ2YXRpb24gZ2VuZXRpY3M7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz02OTctNzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIGVjb2xvZ2lzdHMgZ3VpZGUgdG8gZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3M7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz05MDctMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbXBhY3Qgb2YgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5vbG9neSBvbiBnZW5ldGljczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzMy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBicmVha3Rocm91Z2hzIGZvciBnZW5vbWljIGVjb2xvZ3kgYW5kIGV2b2x1dGlvbmFyeSBiaW9sb2d5OyBNb2wuIEVjb2wuIFJlc291ci47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTMtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkbHkgZGV2ZWxvcGluZyBmdW5jdGlvbmFsIGdlbm9taWNzIGluIGVjb2xvZ2ljYWwgbW9kZWwgc3lzdGVtcyB2aWEgNDU0IHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgR2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTM4OyAgY2l0YXRpb25fcGFnZXM9NDMzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2luZyBpbiBtaWNyb2ZhYnJpY2F0ZWQgaGlnaC1kZW5zaXR5IHBpY29saXRyZSByZWFjdG9yczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzNzsgIGNpdGF0aW9uX3BhZ2VzPTM3Ni04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgdHJhbnNjcmlwdG9tZSBjaGFyYWN0ZXJpemF0aW9uIGZvciBhIG5vbm1vZGVsIG9yZ2FuaXNtIHVzaW5nIDQ1NCBweXJvc2VxdWVuY2luZzsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE2MzYtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgZ29lcyA0NTQgYW5kIHRha2VzIGxhcmdlLXNjYWxlIGdlbm9taWNzIGludG8gdGhlIHdpbGQ7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xNjI5LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmV3IHBhcmFkaWdtIG9mIGZsb3cgY2VsbCBzZXF1ZW5jaW5nOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTgzOS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBhbmQgZGUgbm92byBhbmFseXNpcyBvZiBhIGNvcmFsIGxhcnZhbCB0cmFuc2NyaXB0b21lIHVzaW5nIDQ1NCBHUy1GTFg7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBpbiBhbiBlY29sb2dpY2FsbHkgaW1wb3J0YW50IHRyZWUgc3BlY2llczogYXNzZW1ibHksIGFubm90YXRpb24sIGFuZCBtYXJrZXIgZGlzY292ZXJ5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBnZW5pYyBTU1IgbWFya2VycyBieSBkZWVwIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBpbiBwaWdlb25wZWEgQ2FqYW51cyBjYWphbiAoTC4pIE1pbGxzcGF1Z2g7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byB0cmFuc2NyaXB0b21lIGNoYXJhY3Rlcml6YXRpb24gYW5kIGRldmVsb3BtZW50IG9mIGdlbm9taWMgdG9vbHMgZm9yIFNjYWJpb3NhIGNvbHVtYmFyaWEgTC4gdXNpbmcgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5pcXVlczsgTW9sLiBFY29sLiBSZXNvdXIuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjYyLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgb2YgbGVudGlsIGJhc2VkIG9uIHNlY29uZC1nZW5lcmF0aW9uIHRlY2hub2xvZ3kgcGVybWl0cyBsYXJnZS1zY2FsZSB1bmlnZW5lIGFzc2VtYmx5IGFuZCBTU1IgbWFya2VyIGRpc2NvdmVyeTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBhc3NlbWJseSBvZiBjaGlja3BlYSB0cmFuc2NyaXB0b21lIHVzaW5nIHNob3J0IHJlYWRzIGZvciBnZW5lIGRpc2NvdmVyeSBhbmQgbWFya2VyIGlkZW50aWZpY2F0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGNoYXJhY3Rlcml6YXRpb24gYW5kIHBvbHltb3JwaGlzbSBkZXRlY3Rpb24gYmV0d2VlbiBzdWJzcGVjaWVzIG9mIGJpZyBzYWdlYnJ1c2ggKEFydGVtaXNpYSB0cmlkZW50YXRhKTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzcwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIHB5cm9zZXF1ZW5jaW5nIG9mIGdvbmFkIHRyYW5zY3JpcHRvbWVzIGluIHRoZSBwb2x5cGxvaWQgbGFrZSBzdHVyZ2VvbiAoQWNpcGVuc2VyIGZ1bHZlc2NlbnMpOiB0aGUgcmVsYXRpdmUgbWVyaXRzIG9mIG5vcm1hbGl6YXRpb24gYW5kIHJhcmVmYWN0aW9uIGluIGdlbmUgZGlzY292ZXJ5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yMDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBvcHVsYXRpb24gZ2VuZXRpYyBhbmFseXNpcyBvZiBaaXppcGh1cyBjZWxhdGEsIGFuIGVuZGFuZ2VyZWQgRmxvcmlkYSBzaHJ1YjsgSi4gSGVyZWQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9NTMxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgaW1wb3ZlcmlzaG1lbnQgYW5kIGNyb3NzLWluY29tcGF0aWJpbGl0eSBpbiByZW1uYW50IGdlbm90eXBlcyBvZiBaaXppcGh1cyBjZWxhdGEgKFJoYW1uYWNlYWUpLCBhIHJhcmUgc2hydWIgZW5kZW1pYyB0byB0aGUgTGFrZSBXYWxlcyBSaWRnZSwgRmxvcmlkYTsgQmlvZGl2ZXJzLiBDb25zZXJ2LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTIwMjctNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIGV2aWRlbmNlIGZvciBoaWdoIGNsb25hbGl0eSBhbmQgbGltaXRlZCBnZW5ldGljIGRpdmVyc2l0eSBpbiBaaXppcGh1cyBjZWxhdGEgKFJoYW1uYWNlYWUpLCBhbiBlbmRhbmdlcmVkLCBzZWxmLWluY29tcGF0aWJsZSBMYWtlIFdhbGVzIFJpZGdlLCBGbG9yaWRhLCBVU0EgZW5kZW1pYzsgY2l0YXRpb25fdGV4dD1pbiByZXZpZXc7IENvbnNlcnYuIEdlbmV0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U2luYXVlciBBc3NvY2lhdGVzLCBJbmMsIFN1bmRlcmxhbmQ7IFBsYW50IFN5c3RlbWF0aWNzOiBBIFBoeWxvZ2VuZXRpYyBBcHByb2FjaDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmxvcmlkYSBaaXppcGh1czsgY2l0YXRpb25fYXV0aG9yPVVTRldTOyBjaXRhdGlvbl9wdWJsaXNoZXI9VS5TLiBGaXNoIGFuZCBXaWxkbGlmZSBTZXJ2aWNlLCBBdGxhbnRhLCBHQTsgTXVsdGktc3BlY2llcyBSZWNvdmVyeSBQbGFuIGZvciB0aGUgVGhyZWF0ZW5lZCBhbmQgRW5kYW5nZXJlZCBTcGVjaWVzIG9mIFNvdXRoIEZsb3JpZGE7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl9wYWdlcz0xOTg2LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9VVNGV1M7IGNpdGF0aW9uX3B1Ymxpc2hlcj1VLlMuIEZpc2ggYW5kIFdpbGRsaWZlIFNlcnZpY2UsIFZlcm8gQmVhY2gsIEZMOyBGbG9yaWRhIFppemlwaHVzIChaaXppcGh1cyBjZWxhdGEgNS15ZWFyIFN0YXR1cyBSZXZpZXc6IFN1bW1hcnkgYW5kIEV2YWx1YXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcHJvZHVjdGl2ZSBiaW9sb2d5IG9mIENvbGxldGlhIHNwaW5vc2lzc2ltYSAoUmhhbW5hY2VhZSkgaW4gQXJnZW50aW5hOyBQbGFudCBTeXN0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjk7ICBjaXRhdGlvbl9wYWdlcz03OS04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGFuZHJ5LCBwb2xsaW5hdGlvbiwgYW5kIHNlbGYtaW5jb21wYXRpYmlsaXR5IGluIERpc2NhcmlhIHRvdW1hdG91OyBOLiBaLiBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MzMxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbnJhbmRvbSBtYXRpbmcgaW4gRGlzY2FyaWEgYW1lcmljYW5hIChSaGFtbmFjZWFlKTsgUGxhbnQgU3lzdC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjAzOyAgY2l0YXRpb25fcGFnZXM9MTc5LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXByb2R1Y3RpdmUgYmlvbG9neSBvZiBGcmFuZ3VsYSBhbG51cyAoUmhhbW5hY2VhZSkgaW4gc291dGhlcm4gU3BhaW47IFBsYW50IFN5c3QuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTE3My04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwcm9kdWN0aXZlIGJpb2xvZ3kgb2YgdGhlIGFuZHJvbW9ub2VjaW91cyBzaHJ1YiBUcmV2b2EgcXVpbnF1ZW5lcnZpYSAoUmhhbW5hY2VhZSk7IEJvdC4gSi4gTGlubi4gU29jLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE5MS0yMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBicmVlZGluZyBzeXN0ZW0gb2YgWml6aXBodXMgY2VsYXRhIEp1ZGQgYW5kIERXIEhhbGwgKFJoYW1uYWNlYWUpLCBhIHJhcmUgZW5kZW1pYyBwbGFudCBvZiB0aGUgTGFrZSBXYWxlcyBSaWRnZSwgRmxvcmlkYSwgVVNBOiBpbXBsaWNhdGlvbnMgZm9yIHJlY292ZXJ5OyBCaW9sLiBDb25zZXJ2LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0yMDctMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZsb3dlcmluZyBvZiBaaXppcGh1cyBtdWNyb25hdGEgc3Vic3AuIG11Y3JvbmF0YSAoUmhhbW5hY2VhZSnigJRhbnRoZXNpcywgcG9sbGluYXRpb24gYW5kIHByb3RlaW4gc3ludGhlc2lzOyBCb3QuIEJ1bGwuIEFjYWQuIFNpbmljYTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTMzLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdW1tYXJ5IG9mIDIwMDXigJMyMDA3IG9ic2VydmF0aW9ucyBvZiBwb2xsZW4gZ3JhaW5zIGFuZCBwb2xsZW4gdHViZSBncm93dGggaW4gRmxvcmlkYSB6aXppcGh1cyBhbmQgb2JzZXJ2YXRpb25zIG9mIG9wZW4tcG9sbGluYXRpb25zIGFuZCBhYm9ydGVkIGZydWl0czsgY2l0YXRpb25fcHVibGlzaGVyPUZpbmFsIFJlcG9ydCB0byBQbGFudCBDb25zZXJ2YXRpb24gUHJvZ3JhbSwgRmxvcmlkYSBEaXZpc2lvbiBvZiBGb3Jlc3RyeSwgVGFsbGFoYXNzZWUsIEZMOyBDb250aW51YXRpb24gb2YgUmVzZWFyY2ggb24gdGhlIEZlZGVyYWxseS1saXN0ZWQgTGFrZSBXYWxlcyBSaWRnZSBFbmRlbWljIEZsb3JpZGEgWml6aXBodXMgKFppemlwaHVzIGNlbGF0YSk7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIFMtUk5hc2UtYmFzZWQgZ2FtZXRvcGh5dGljIHNlbGYtaW5jb21wYXRpYmlsaXR5IHN5c3RlbSBldm9sdmVkIG9ubHkgb25jZSBpbiBldWRpY290czsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTE3OS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHJlbGF0aW9uc2hpcHMgYW1vbmcgc2VsZi1pbmNvbXBhdGliaWxpdHkgUk5hc2VzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTMxNjctNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pUmVmOiBjb21wcmVoZW5zaXZlIGFuZCBub24tcmVkdW5kYW50IFVuaVByb3QgcmVmZXJlbmNlIGNsdXN0ZXJzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEyODItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEFyYWJpZG9wc2lzIEluZm9ybWF0aW9uIFJlc291cmNlIChUQUlSKTogZ2VuZSBzdHJ1Y3R1cmUgYW5kIGZ1bmN0aW9uIGFubm90YXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQxMDA5LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90ZXh0PTIuMCBhMTE7IFNlLUFsOiBBIE1hbnVhbCBTZXF1ZW5jZSBBbGlnbm1lbnQgRWRpdG9yOyAgY2l0YXRpb25feWVhcj0yMDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXRjaFByaW1lcjM6IGEgaGlnaCB0aHJvdWdocHV0IHdlYiBhcHBsaWNhdGlvbiBmb3IgUENSIGFuZCBzZXF1ZW5jaW5nIHByaW1lciBkZXNpZ247IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBnYW1ldG9waHl0ZSB0cmFuc2NyaXB0b21lIGluIGJyYWNrZW4gZmVybiwgUHRlcmlkaXVtIGFxdWlsaW51bTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9OTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGRpc2NvdmVyeSBhbmQgYW5ub3RhdGlvbiB1c2luZyBMQ00tNDU0IHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz02OS03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IGdlbmUgYW5kIFNOUCBkaXNjb3ZlcnkgaW4gRXVjYWx5cHR1cyBncmFuZGlzLCBhbiB1bmNoYXJhY3Rlcml6ZWQgZ2Vub21lOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTMxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVC1TU1JzIGFzIGEgcmVzb3VyY2UgZm9yIHBvcHVsYXRpb24gZ2VuZXRpYyBhbmFseXNlczsgSGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xMjUtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS8xLzEvMTMwOTc4MC9kc3IwMzcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIExhcmdlLXNjYWxlIEROQSBzZXF1ZW5jZSBkYXRhIG1heSBlbmFibGUgZGV2ZWxvcG1lbnQgb2YgZ2VuZXRpYyByZXNvdXJjZXMgaW4gZW5kYW5nZXJlZCBzcGVjaWVzLCB0aGVyZWJ5IGZhY2lsaXRhdGluZyBjb25zZXJ2YXRpb24gZWZmb3J0cy4gWml6aXBodXMgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODQ1MTQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkI6FAGKxXSG50BHeccGcvowK2gWg=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f1%2f484514" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f1%2f484514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19070" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&amp;Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/1">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7360671">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360671">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360673">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360673">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360681">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360681">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360692">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360692">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360706">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360706">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360715">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360715">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360717">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360717">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360719">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360719">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7360721">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7360721">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/19/1/11/484721">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in <em>Ziziphus Celata</em> (Rhamnaceae), a Highly Endangered, Florida Endemic Plant
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christine E. Edwards</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USACE ERDC, Environmental Laboratory, 3909 Halls Ferry Road, Vicksburg, MS 39180</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Email <a href="mailto:christine.e.edwards@usace.army.mil" target="_blank">christine.e.edwards@usace.army.mil</a> (C.E.E.); <a href="mailto:tparchma@uwyo.edu" target="_blank">tparchma@uwyo.edu</a> (T.L.P.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christine+E.+Edwards">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Edwards C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Edwards C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christine E. Edwards</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thomas L. Parchman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Botany</div>, <div class="insititution">University of Wyoming</div>, <div class="addr-line">Laramie, WY 82071</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thomas+L.+Parchman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parchman T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parchman T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Thomas L. Parchman</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Carl W. Weekley</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Archbold Biological Station</div>, <div class="addr-line">PO Box 2057, Lake Placid, FL 33862</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Carl+W.+Weekley">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Weekley C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Weekley C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Carl W. Weekley</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 1–9, <a href='https://doi.org/10.1093/dnares/dsr037'>https://doi.org/10.1093/dnares/dsr037</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 October 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 July 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 September 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484514" href="/dnaresearch/article-pdf/19/1/1/1309780/dsr037.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Christine E. Edwards, Thomas L. Parchman, Carl W. Weekley;  Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in <em>Ziziphus Celata</em> (Rhamnaceae), a Highly Endangered, Florida Endemic Plant, <em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 1–9, <a href='https://doi.org/10.1093/dnares/dsr037'>https://doi.org/10.1093/dnares/dsr037</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484514?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484514?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484514?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484514?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484514?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484514?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484514?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Assembly%2C%20Gene%20Annotation%20and%20Marker%20Development%20Using%20454%20Floral%20Transcriptome%20Sequences%20in%20%26lt%3Bem%26gt%3BZiziphus%20Celata%26lt%3B%2Fem%26gt%3B%20(Rhamnaceae)%2C%20a%20Highly%20Endangered%2C%20Florida%20Endemic%20Plant&amp;publicationDate=2011-10-27&amp;volumeNum=19&amp;issueNum=1&amp;author=Edwards%2C%20Christine%20E.%3B%20Parchman%2C%20Thomas%20L.&amp;startPage=1&amp;endPage=9&amp;contentId=10.1093%2Fdnares%2Fdsr037&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f1%2f484514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7360671 id="7360671" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Large-scale DNA sequence data may enable development of genetic resources in endangered species, thereby facilitating conservation efforts. <em>Ziziphus celata</em>, a federally endangered, self-incompatible plant species occurring in Florida, USA, is one species for which genetic resources are necessary to facilitate new introductions and augmentations essential for recovery of the species. We used 454 pyrosequencing of a <em>Z. celata</em> normalized floral cDNA library to create a genomic resource for gene and marker discovery. A half-plate GS-FLX Titanium run yielded 655 337 reads averaging 250 bp. A total of 474 025 reads were assembled <em>de novo</em> into 84 645 contigs averaging 408 bp, while 181 312 reads remained unassembled. Forty-seven and 43% of contig consensus sequences had BLAST matches to known proteins in the Uniref50 and TAIR9 annotated protein databases, respectively; many contigs fully represented orthologous proteins in TAIR9. A total of 22 707 unique genes were sequenced, indicating substantial coverage of the <em>Z. celata</em> transcriptome. We detected single-nucleotide polymorphisms and simple sequence repeats (SSRs) and developed thousands of SSR primers for use in future genetic studies. As a first step towards understanding self-incompatibility in <em>Z. celata</em>, we identified sequences belonging to the gene family encoding self-incompatibility. This study demonstrates the efficacy of 454 transcriptome sequencing for rapid gene and marker discovery in an endangered plant.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;conservation genetics&quot;">conservation genetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="microsatellites">microsatellites</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="S-locus">S-locus</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Ziziphus"><em>Ziziphus</em></a></div>                            <h2 scrollto-destination=7360673 id="7360673" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>While genetic information is just one of many tools that may be used to conserve endangered species, the use of such information is often valuable to facilitate recovery efforts and the management of surviving populations.<sup><a reveal-id="DSR037C1" data-open="DSR037C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Genetic data from endangered species have been used for multiple conservation applications, e.g. to carry out parentage analysis in <em>ex situ</em> breeding and translocation programmes,<sup><a reveal-id="DSR037C2" data-open="DSR037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSR037C3" data-open="DSR037C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and to quantify levels of genetic diversity and identify factors such as founder effects, genetic drift, genetic bottlenecks, and inbreeding that may threaten endangered species.<sup><a reveal-id="DSR037C4  DSR037C6" data-open="DSR037C4  DSR037C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup> However, because of the difficulty of isolating genetic markers and the considerable cost and time associated with extensive genotyping of individuals in populations, many conservation-genetic studies have employed relatively small numbers of genetic loci, such as RFLPs, AFLPs, and SSRs (simple sequence repeats, or microsatellites). Increasing the number of sampled loci may increase the precision and accuracy of estimates of genetic structure and population genetic parameters such as levels of genetic diversity and inbreeding coefficients.<sup><a reveal-id="DSR037C7" data-open="DSR037C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Furthermore, large numbers of genetic loci are necessary to unequivocally assign parentage, which may be very useful for establishing genetically diverse populations of endangered species through the augmentation of existing genetically depauperate populations or the creation of new populations.</p><p>Recently, however, the number of genomic-scale sequence collections has increased, enabling rapid gene and marker discovery for an increased number of taxa.<sup><a reveal-id="DSR037C8" data-open="DSR037C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Previously, the development of such genomic resources was largely limited to model organisms; however, recent advances in DNA sequencing technology have reduced the cost and time required for the development of genomic resources, resulting in a rapid growth in the availability of such data, particularly in non-model organisms.<sup><a reveal-id="DSR037C9  DSR037C11" data-open="DSR037C9  DSR037C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> Transcriptome sequencing, which is DNA sequencing of the mRNA pool of a given tissue, has allowed sequencing efforts to focus on the protein-coding portion of the genome, and has become a valuable approach for developing genomic-level resources of the transcribed portion of the genome.<sup><a reveal-id="DSR037C8" data-open="DSR037C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Next-generation sequencing technologies, such as 454 pyrosequencing, remove many time-consuming steps involved in Sanger sequencing and now facilitate transcriptome sequencing at a minute fraction of the previously required time and cost.<sup><a reveal-id="DSR037C10  DSR037C15" data-open="DSR037C10  DSR037C15" class="link link-ref link-reveal xref-bibr">10–15</a></sup> A single-plate run on the 454 GS-FLX titanium pyrosequencing platform typically produces around a million reads averaging 400 bp in length, often resulting in near complete transcriptome coverage.<sup><a reveal-id="DSR037C11" data-open="DSR037C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSR037C16" data-open="DSR037C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR037C17" data-open="DSR037C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The generation of such large-scale sequence data is enabling gene discovery, molecular marker development, and comparative analyses in ecologically important, non-model plant taxa,<sup><a reveal-id="DSR037C11" data-open="DSR037C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSR037C18  DSR037C22" data-open="DSR037C18  DSR037C22" class="link link-ref link-reveal xref-bibr">18–22</a></sup> including those of conservation concern.<sup><a reveal-id="DSR037C23" data-open="DSR037C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p><p>One species for which conservation efforts would be greatly assisted by the establishment of a genome-level resource is <em>Ziziphus celata</em> (Rhamnaceae), a federally endangered, diploid shrub endemic to central Florida, which is currently known from only 14 populations along the Lake Wales Ridge in Polk and Highlands Counties. The species is highly clonal and previous genetic analyses using allozymes,<sup><a reveal-id="DSR037C24" data-open="DSR037C24" class="link link-ref link-reveal xref-bibr">24</a></sup> RAPDs,<sup><a reveal-id="DSR037C25" data-open="DSR037C25" class="link link-ref link-reveal xref-bibr">25</a></sup> AFLPs, and SSRs<sup><a reveal-id="DSR037C26" data-open="DSR037C26" class="link link-ref link-reveal xref-bibr">26</a></sup> found that 9 of the 14 extant wild populations are uniclonal (i.e. each population comprises a single genetic individual). Depending on the method of determining the number of genotypes, the remaining five populations comprise between 22 and 32 genotypes;<sup><a reveal-id="DSR037C26" data-open="DSR037C26" class="link link-ref link-reveal xref-bibr">26</a></sup> however, unambiguous quantification of the number of genotypes in these populations will require the employment of a larger number of genetic markers. Very little genetic data are presently available for <em>Z. celata</em>, complicating the development of genetic markers, e.g. as of April 2011, only six DNA sequences had been deposited at NCBI for <em>Z. celata</em>, and none of these represented expressed genes. Furthermore, the entire Rhamnaceae family, which contains ∼900 species,<sup><a reveal-id="DSR037C27" data-open="DSR037C27" class="link link-ref link-reveal xref-bibr">27</a></sup> was represented by fewer than 2000 sequences at NCBI. The development of a genome-level resource for <em>Z. celata</em> stands to improve conservation-genetic approaches in <em>Z. celata</em> and to provide a future basis for comparative genomic studies in the Rhamnaceae, which contains economically important crop and ornamental tree and shrub species.</p><p>Given the small numbers of populations and genotypes of <em>Z. celata</em>, its recovery relies on the augmentation of uniclonal populations and the introduction of multiple genotypes to publicly protected sites containing appropriate habitat;<sup><a reveal-id="DSR037C28" data-open="DSR037C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR037C29" data-open="DSR037C29" class="link link-ref link-reveal xref-bibr">29</a></sup> however, the ability to establish self-sustaining, sexually reproducing, translocated populations has been complicated by the reproductive failure of most experimental crosses. Such reproductive failure has been attributed to self-incompatibility in <em>Z. celata</em>. Self-incompatibility has been documented or proposed in at least five genera in the Rhamnaceae, including <em>Colletia</em>,<sup><a reveal-id="DSR037C30" data-open="DSR037C30" class="link link-ref link-reveal xref-bibr">30</a></sup><em>Discaria</em>,<sup><a reveal-id="DSR037C31" data-open="DSR037C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSR037C32" data-open="DSR037C32" class="link link-ref link-reveal xref-bibr">32</a></sup><em>Frangula</em>,<sup><a reveal-id="DSR037C33" data-open="DSR037C33" class="link link-ref link-reveal xref-bibr">33</a></sup><em>Trevoa</em>,<sup><a reveal-id="DSR037C34" data-open="DSR037C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and <em>Ziziphus</em>.<sup><a reveal-id="DSR037C25" data-open="DSR037C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSR037C35" data-open="DSR037C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSR037C36" data-open="DSR037C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p><p>Gametophytic self-incompatibility (GSI) has been proposed as the system of self-incompatibility in <em>Z. celata</em> based on results from hundreds of experimental hand pollinations, analysis of RAPD data, and studies of pollen tube inhibition.<sup><a reveal-id="DSR037C25" data-open="DSR037C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSR037C35" data-open="DSR037C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSR037C37" data-open="DSR037C37" class="link link-ref link-reveal xref-bibr">37</a></sup> GSI is a system whereby pollen is rejected when its <em>S</em>-haplotype is the same as either of the <em>S</em>-haplotypes present in the pistil.<sup><a reveal-id="DSR037C38" data-open="DSR037C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSR037C39" data-open="DSR037C39" class="link link-ref link-reveal xref-bibr">39</a></sup> In four of the five plant families with well-characterized GSI systems, GSI is mediated by the <em>S-RNase</em> gene.<sup><a reveal-id="DSR037C38" data-open="DSR037C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSR037C39" data-open="DSR037C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Previous research suggests that a <em>S-RNase</em>-based system of GSI likely evolved only once in Angiosperms,<sup><a reveal-id="DSR037C38" data-open="DSR037C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSR037C39" data-open="DSR037C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and the presence of <em>S-RNase</em>-based GSI in the Rosaceae, a plant family which is closely related to the Rhamnaceae, suggests that this is the most likely mechanism causing GSI in Rhamnaceae.</p><p>An understanding of the genetic basis of self-incompatibility and the ability to assess mating types in <em>Z. celata</em> would greatly facilitate recovery efforts. Previous research has shown that as few as two mating types may be present in <em>Z. celata</em> wild populations,<sup><a reveal-id="DSR037C25" data-open="DSR037C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSR037C35" data-open="DSR037C35" class="link link-ref link-reveal xref-bibr">35</a></sup> the minimum necessary to maintain sexual reproduction in a GSI species; however, more mating types may exist, as the compatibility of most genotypes has not yet been tested. The most efficient approach to determine cross-compatibility is to directly sequence the genes that encode the GSI reaction. Another efficient approach is parentage analysis of progeny to provide indirect evidence of compatible mating types; however, more genetic data and molecular markers are required to carry out either of these approaches in <em>Z. celata</em>.</p><p>In this study, we conducted 454 pyrosequencing of a normalized cDNA library isolated from flowers of three genotypes of <em>Z. celata</em>. Our goals were to (i) characterize the floral transcriptome of <em>Z. celata</em>, (ii) identify and characterize a large number of gene-based markers, including single-nucleotide polymorphisms (SNPs) and SSRs, for future genetic analyses, and (iii) determine whether analysis of the floral transciptome includes genes belonging to gene family that encodes the GSI reaction. The increased genomic information produced in this study will aid in creating self-sustaining, sexually reproducing populations of this endangered species and bolster comparative genomic studies in a plant family for which very little genetic data are currently available.</p>                            <h2 scrollto-destination=7360681 id="7360681" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7360682 id="7360682" class="section-title" data-legacy-id=s2a>2.1. Tissue sampling, cDNA library creation, and 454 sequencing</h3>
<p>During January–February 2010, we collected several hundred newly opened flowers from each of three <em>Z. celata</em> genets, including the two S-locus mating types confirmed in the wild and one genet of unknown mating type. We collected flowers in sterile RNase-free tubes, which were placed immediately into liquid nitrogen. Flowers from each of the three mating genotypes were then shipped on dry ice to GATC Inc., in Konstanz, Germany. Flowers from the three genotypes were pooled in equal amounts before RNA extraction. RNA extraction, cDNA synthesis, cDNA library normalization, and 454 sequencing were performed by GATC staff following previously outlined methods.<sup><a reveal-id="DSR037C17" data-open="DSR037C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The <em>Z. celata</em> floral cDNA library was sequenced in a half-plate run on a 454 GS XLR70 Titanium genomic sequencer (Roche, Inc.).</p>                            <h3 scrollto-destination=7360684 id="7360684" class="section-title" data-legacy-id=s2b>2.2. Assembly and annotation</h3>
<p>We trimmed 454 primer sequences from all reads prior to assembly and removed reads with average quality scores lower than 18. We used Seqman Ngen v2.0 (DNAstar, Inc.) to assemble reads into contigs, as this program has been successful in assembling 454 sequences from transcriptomes.<sup><a reveal-id="DSR037C13" data-open="DSR037C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR037C17" data-open="DSR037C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Because no reference genome exists for <em>Ziziphus</em>, reads were assembled <em>de novo</em>. The assembly was run with a minimum match size of 19 nucleotides, match percentage of 95%, mismatch penalty of 18, and gap penalty of 30 (further information on assembly is available from the authors by request). The resulting contig consensus sequences and remaining singletons were then combined into a single set for the following analyses, except where noted.</p><p>We annotated the 454 sequences by using local BLASTx<sup><a reveal-id="DSR037C40" data-open="DSR037C40" class="link link-ref link-reveal xref-bibr">40</a></sup> to align the consensus sequences from the assembled contigs and the singleton sequences to the Uniref50 15.4<sup><a reveal-id="DSR037C41" data-open="DSR037C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and the TAIR9 <em>Arabidopsis thaliana</em><sup><a reveal-id="DSR037C42" data-open="DSR037C42" class="link link-ref link-reveal xref-bibr">42</a></sup> annotated protein databases using an <em>E-</em>value threshold of 10<sup>−11</sup>. BLAST results were passed through a custom Perl pipeline that produced tab-delimited tables containing accession numbers, gene name, taxonomic ID, query length, orthologue sequence length, sequence alignment, <em>E-</em>value, and bit score for each protein matching to the <em>Z. celata</em> 454 sequences. To determine the number of unique genes represented, we filtered these files for redundancy in protein accessions. To assess the taxonomic distribution of BLAST hits, we used a custom perl script that employs a BioPerl package to retrieve hierarchical taxonomic identities for each protein accession ID. Assignment of gene ontology (GO) terms to sequences with BLAST matches to known proteins was then performed by importing the accession numbers for the BLAST hits to unique proteins into Blast2go (version 2.3.6; <a class="link link-uri" href="http://www.blast2go.org">www.blast2go.org</a>). We based these analyses on the contig consensus sequences with positive BLAST matches to 13 401 unique accessions in the TAIR annotated protein database.</p>                            <h3 scrollto-destination=7360687 id="7360687" class="section-title" data-legacy-id=s2c>2.3. Search for the S-locus</h3>
<p>In most other taxa, the female component of GSI is encoded by an <em>S-RNase</em>, a member of the <em>T2 RNase</em> family.<sup><a reveal-id="DSR037C38" data-open="DSR037C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSR037C39" data-open="DSR037C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Thus, as a first step towards characterizing the genes involved in GSI in <em>Z. celata</em>, we searched annotation results and carried out BLAST searches to find <em>Z. celata</em> 454 sequences belonging to the <em>T2 RNase</em> gene family. We searched the <em>Z. celata</em> annotation results for the terms ‘<em>T2 RNase</em>’, ‘<em>RNase</em>’, ‘<em>ribonuclease T2</em>’, and ‘<em>S-RNase</em>’. We also created a blastable database of characterized <em>S-RNase</em> sequences deposited at NCBI's dbEST, including <em>Nicotiana alata</em> S2, <em>Antirrhinum hispanicum</em> S2, <em>Prunus avium</em> S1, and <em>Pyrus pyrifolia</em> S4.<sup><a reveal-id="DSR037C38" data-open="DSR037C38" class="link link-ref link-reveal xref-bibr">38</a></sup> We used BLASTx to align all <em>Z. celata</em> contig consensus sequences and singletons to these sequences with an <em>E</em>-value threshold of 10<sup>−10</sup>. Once we identified sequences that putatively belonged to the <em>T2 RNase</em> gene family, we translated them to proteins, manually aligned them with amino acid sequences from verified members of the <em>T2 RNase</em> gene family<sup><a reveal-id="DSR037C38" data-open="DSR037C38" class="link link-ref link-reveal xref-bibr">38</a></sup> using Se-Al,<sup><a reveal-id="DSR037C43" data-open="DSR037C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and inspected their protein-coding motifs for homology with verified members of the gene family.</p>                            <h3 scrollto-destination=7360689 id="7360689" class="section-title" data-legacy-id=s2d>2.4. Molecular marker characterization</h3>
<p>We used custom perl scripts to locate di-, tri-, and tetra-nucleotide SSRs in the 454 sequences with a minimum length of 12 bp and six contiguous repeating units for di-nucleotide motifs, four contiguous repeats for tri-nucleotide motifs, and three repeating units for tetra-nucleotide motifs. We determined which SSRs occurred in coding regions of genes by extracting the aligned portions of contig consensus sequences that had BLAST matches to annotated protein-coding orthologues in Uniref50, and then used the same algorithm as above to detect SSRs in both the aligned and remaining portions of these contigs. To construct polymerase chain reaction (PCR) primers in the flanking regions of SSRs, we used the program BatchPrimer3.<sup><a reveal-id="DSR037C44" data-open="DSR037C44" class="link link-ref link-reveal xref-bibr">44</a></sup> We created primers with a minimum GC content of 30%, a GC clamp (the last two nucleotides were G or C), a melting temperature between 52 and 55, and we positioned primers to obtain PCR products between 100 and 450 bp.</p><p>We isolated SNPs in contigs with high coverage depths using the SNP reporter feature in Seqman Pro (DNAstar, Inc.). We identified SNPs with coverage depth of at least 10 and with an alternate allele in a minimum of 20% in all contigs containing &gt;25 reads, in order to obtain an estimate of the transcriptome-wide occurrence of high-quality SNPs. We then enumerated SNPs at sites where coverage depth was at least 8 and where alternate alleles were present at a minimum frequency of 20%.</p>                            <h2 scrollto-destination=7360692 id="7360692" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7360693 id="7360693" class="section-title" data-legacy-id=s3a>3.1. Assembly and annotation</h3>
<p>A half-plate run on the 454 GS-FLX Titanium platform produced 655 337 sequences that, after adaptor and vector trimming, averaged 254 bases in length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Files containing the 454 DNA sequences and quality scores have been deposited at NCBIs Short Read Archive (accession SRA045662). A total of 474 025 reads (72% of total) were assembled into 84 645 contigs, with 181 312 reads remaining as singletons. The average contig length was 408 bases (min = 19, max = 2438), with an average of 5.6 (min = 2, max = 1337) reads assembled per contig (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>) and a GC content of 0.39. The average read length of the singleton sequences was 255 bp, similar to the average length of the reads assembled into contigs (253 bp), and these sequences had an average GC content of 0.37. The mean coverage depth per nucleotide position in the assembled contigs was 3.1 (min = 1, max = 868), indicating reasonably good coverage depth for a half-plate 454 GS-FLX Titanium run. As expected, contig length increased as a function of coverage depth and the number of reads assembled into each contig (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p><p>BLAST annotation of contig consensus and singleton sequences to a large number of unique genes indicated extensive coverage of the <em>Z. celata</em> floral transcriptome. Results of BLAST searches against the Uniref50 and TAIR9 databases, including the species name, accession number, identity percentage, and <em>E</em>-value, are presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>, respectively. Forty-seven per cent of the contig consensus sequences had BLAST hits to annotated proteins in Uniref50, and 43% had BLAST hits to annotated proteins in TAIR9 (Table <a reveal-id="DSR037TB1" data-open="DSR037TB1" class="link link-reveal link-table xref-fig">1</a>). Smaller percentages of the singleton sequences also had BLAST matches to the annotated protein databases (25% UniRef; 23% TAIR). Nonetheless, the large number of singleton sequences with BLAST matches (Table <a reveal-id="DSR037TB1" data-open="DSR037TB1" class="link link-reveal link-table xref-fig">1</a>) indicates that unassembled sequences still provide an abundance of valuable sequence information and improve overall transcriptome coverage breadth. In many cases, multiple sequences representing both contigs and singletons had BLAST matches to the same protein. After correcting for redundancy, the combined set of contig consensus sequences and singleton sequences had matches to 22 707 unique proteins in Uniref50, and 16 089 unique proteins in TAIR, indicating that many <em>Z. celata</em> genes are represented (Table <a reveal-id="DSR037TB1" data-open="DSR037TB1" class="link link-reveal link-table xref-fig">1</a>). Of the 13 514 unique BLAST matches of contig sequences to protein accessions in the TAIR database, 10 808 were annotated to GO terms. Of the assigned GO terms, 6083 were to biological processes, 8036 to molecular function, and 6565 to cellular components, indicating a large functional diversity of genes in the trancriptomic data. The even distribution of assignments of proteins to more specialized GO terms further indicates that the <em>Z. celata</em> 454 sequences represent proteins from a diverse range of functional classes (Fig. <a reveal-id="DSR037F1" data-open="DSR037F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="7360696" scrollto-destination="7360696"></a>
<div content-id="DSR037TB1" class="table-modal"><div class="table-wrap"><div id="DSR037TB1" class="table-wrap-title" data-id="DSR037TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The number and percentage of the total <em>Ziziphus celata</em> 454 contigs and singletons that matched to annotated protein databases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Contigs<hr /> </th><th colspan="2">Singletons<hr /> </th><th colspan="2">Combined set<hr /> </th></tr><tr><th>Database </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique hits </th></tr></thead><tbody><tr><td>Uniref </td><td>39 815 (47%) </td><td>17 420 </td><td>46 124 (25%) </td><td>16 332 </td><td>85 939 (32%) </td><td>22 707 </td></tr><tr><td>TAIR </td><td>36 801 (43%) </td><td>13 514 </td><td>42 707 (23%) </td><td>12 479 </td><td>79 508 (30%) </td><td>16 089 </td></tr></tbody></table></div><div content-id="DSR037TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Contigs<hr /> </th><th colspan="2">Singletons<hr /> </th><th colspan="2">Combined set<hr /> </th></tr><tr><th>Database </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique hits </th></tr></thead><tbody><tr><td>Uniref </td><td>39 815 (47%) </td><td>17 420 </td><td>46 124 (25%) </td><td>16 332 </td><td>85 939 (32%) </td><td>22 707 </td></tr><tr><td>TAIR </td><td>36 801 (43%) </td><td>13 514 </td><td>42 707 (23%) </td><td>12 479 </td><td>79 508 (30%) </td><td>16 089 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7360696" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR037TB1" class="table-wrap-title" data-id="DSR037TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The number and percentage of the total <em>Ziziphus celata</em> 454 contigs and singletons that matched to annotated protein databases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Contigs<hr /> </th><th colspan="2">Singletons<hr /> </th><th colspan="2">Combined set<hr /> </th></tr><tr><th>Database </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique hits </th></tr></thead><tbody><tr><td>Uniref </td><td>39 815 (47%) </td><td>17 420 </td><td>46 124 (25%) </td><td>16 332 </td><td>85 939 (32%) </td><td>22 707 </td></tr><tr><td>TAIR </td><td>36 801 (43%) </td><td>13 514 </td><td>42 707 (23%) </td><td>12 479 </td><td>79 508 (30%) </td><td>16 089 </td></tr></tbody></table></div><div content-id="DSR037TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Contigs<hr /> </th><th colspan="2">Singletons<hr /> </th><th colspan="2">Combined set<hr /> </th></tr><tr><th>Database </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique </th><th>Total hits </th><th>Unique hits </th></tr></thead><tbody><tr><td>Uniref </td><td>39 815 (47%) </td><td>17 420 </td><td>46 124 (25%) </td><td>16 332 </td><td>85 939 (32%) </td><td>22 707 </td></tr><tr><td>TAIR </td><td>36 801 (43%) </td><td>13 514 </td><td>42 707 (23%) </td><td>12 479 </td><td>79 508 (30%) </td><td>16 089 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7360696" target="_blank">View Large</a></div></div>                        <a id="7360697" scrollto-destination="7360697"></a>
<div id="DSR037F1" data-id="DSR037F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/m_dsr03701.jpeg?Expires=2147483647&Signature=NBM0wfeknHuJ~wUJ8cbqiQtKIikOuRsb8ZzMhKSr92aomBUSViHOZ0lyLzLb5R03lNALzbvuMfPS7EznVE~uRwzzdehIUqY5m7gU4O39BpuP~whYZyUaWGm8RXkXA0c6I8mZgqI-QlUKXhitfg~nIkUION1Q2j-AjohsNNqYpGsiF6vR0vIe46jGJPTgGjHWMCcLBDj5HxXmX6XM4zLEkVY5~J-WzsaMwGlJuS6nDKQ6Nb4xkwQvbQkJZ2VeGsf9tHxl5UA7~fKpXQp5az~cnNETebjvO8wOk2TzGQJ2GWh5UTbW85wU9ma0vNGG1r1BaE4Y4Yp6Yjgnqtegm1mJnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional gene diversity in the Z. celata transcriptome data. Bars represent the number of assignments of Z. celata proteins with BLAST matches in the TAIR9 database to each GO term." path-from-xml="dsr03701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7360697/dsr03701.jpeg" path-from-xml="dsr03701.jpeg" target="_blank">View large</a><a section="7360697" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr03701.jpeg?Expires=2147483647&Signature=VZgVQqInKP4M~5x7E6ESRqs0PYxwFzGUHHkFmjXNYBMZJos0TvB8LoI10JdMwWllj7a8yK35~wLBOfKwfqjgDIo2Sr~an3i9qulspNX8zySzLJMhKp4qxBpdaryOUnNtYn7JZ6CUOhVootuoob0J9mBWbQVw19IiJW8l-fwEjqr8c-bMbZq1NVDEdlKBH5A6eqD5BI9BmH~kc5OHSGcFuyI8LJDp7GWN8R1XRCJ-g30N8HYn71HoTpJpcYC5vG7JKHb2QW0N5mWi7VgiJbgCh32jhLI6OI6dMDNZGNAOc5u1z6QtIvOYMCQCXj2gx9UhUsgspRJzfJwZMHxZIVrhpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7360697&ar=484514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional gene diversity in the <em>Z. celata</em> transcriptome data. Bars represent the number of assignments of <em>Z. celata</em> proteins with BLAST matches in the TAIR9 database to each GO term.</p></div></div><div content-id="DSR037F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/m_dsr03701.jpeg?Expires=2147483647&Signature=NBM0wfeknHuJ~wUJ8cbqiQtKIikOuRsb8ZzMhKSr92aomBUSViHOZ0lyLzLb5R03lNALzbvuMfPS7EznVE~uRwzzdehIUqY5m7gU4O39BpuP~whYZyUaWGm8RXkXA0c6I8mZgqI-QlUKXhitfg~nIkUION1Q2j-AjohsNNqYpGsiF6vR0vIe46jGJPTgGjHWMCcLBDj5HxXmX6XM4zLEkVY5~J-WzsaMwGlJuS6nDKQ6Nb4xkwQvbQkJZ2VeGsf9tHxl5UA7~fKpXQp5az~cnNETebjvO8wOk2TzGQJ2GWh5UTbW85wU9ma0vNGG1r1BaE4Y4Yp6Yjgnqtegm1mJnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional gene diversity in the Z. celata transcriptome data. Bars represent the number of assignments of Z. celata proteins with BLAST matches in the TAIR9 database to each GO term." path-from-xml="dsr03701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7360697/dsr03701.jpeg" path-from-xml="dsr03701.jpeg" target="_blank">View large</a><a section="7360697" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr03701.jpeg?Expires=2147483647&Signature=VZgVQqInKP4M~5x7E6ESRqs0PYxwFzGUHHkFmjXNYBMZJos0TvB8LoI10JdMwWllj7a8yK35~wLBOfKwfqjgDIo2Sr~an3i9qulspNX8zySzLJMhKp4qxBpdaryOUnNtYn7JZ6CUOhVootuoob0J9mBWbQVw19IiJW8l-fwEjqr8c-bMbZq1NVDEdlKBH5A6eqD5BI9BmH~kc5OHSGcFuyI8LJDp7GWN8R1XRCJ-g30N8HYn71HoTpJpcYC5vG7JKHb2QW0N5mWi7VgiJbgCh32jhLI6OI6dMDNZGNAOc5u1z6QtIvOYMCQCXj2gx9UhUsgspRJzfJwZMHxZIVrhpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7360697&ar=484514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional gene diversity in the <em>Z. celata</em> transcriptome data. Bars represent the number of assignments of <em>Z. celata</em> proteins with BLAST matches in the TAIR9 database to each GO term.</p></div></div><p>Many contig consensus sequences were sufficiently long to cover full or nearly full transcripts (Fig. <a reveal-id="DSR037F2" data-open="DSR037F2" class="link link-reveal link-table xref-fig">2</a>). Examination of the 36 801 contigs with BLAST matches to protein-coding sequences in the TAIR9 protein database indicates that many contigs were as long as their corresponding full-length orthologous <em>Arabidopsis</em> transcripts (Fig. <a reveal-id="DSR037F2" data-open="DSR037F2" class="link link-reveal link-table xref-fig">2</a>). As coverage depth increased, the ratio of <em>Z. celata</em> contig length/<em>Arabidopsis</em> orthologue length increased, with a large number of <em>Z. celata</em> contig sequences likely covering full-length transcripts (Fig. <a reveal-id="DSR037F2" data-open="DSR037F2" class="link link-reveal link-table xref-fig">2</a>). This indicates that the assembly was of high quality and that we obtained extensive coverage breadth and depth for much of the floral transcriptome. </p>                        <a id="7360699" scrollto-destination="7360699"></a>
<div id="DSR037F2" data-id="DSR037F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/m_dsr03702.jpeg?Expires=2147483647&Signature=0de5y4u8UWBxO4m39YUfbIMz0Fzvxbw0tXHjtG-Kzv4gwgmCZ9bKLKT~XZ33TWisI3IGtQE~IKonEP2pInh28kfp3XkJMOorD4ZS4opRPNIJ8xHyylwTOKXEPehkzT7u~h0WawFqcSFaGrQUDbrk2TYSE4UDjQMs999eHr7k5tyM5ryG9DTsnsSu-OMgOYgZdQedTX8~AMv9SoA32oCCmlIrD2vtaRED545R0g5SdtekgwKSFR2o1Y~gqnxZUV5uoqX2zpxhyGqIedOkPWQbnLBiAcJB5qvwra-QjWdAMQNimXFdmGY6MdvaUELpNd3lhUmXmHv4D95E7eh3jGliKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ratio of Z. celata contig length to A. thaliana orthologue length as a function of contig coverage depth. The dotted line corresponds to a ratio of 1, where 454 contigs are as long or longer than the BLAST matched A. thaliana orthologues." path-from-xml="dsr03702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7360699/dsr03702.jpeg" path-from-xml="dsr03702.jpeg" target="_blank">View large</a><a section="7360699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr03702.jpeg?Expires=2147483647&Signature=CKEQwQ7KG2fTXmuBwNTUtgnnxG~wiRl4OxnFRLJJWGkxOWqudqSgt92NujpDRCkvXK1uanciYN0rbJiJY9AmNWUnQSVhBeFP3YEMI2EpK2ZDa8WJQOaUZLHjUOrWaeK33SLNdES7plzK59fKB1ddygBSvhKokkI4MKXEimXEFgjLiTWRxlLiO6it1NAbk3J2GDdgTUGz2A99nJPG~tDuyZUEA1u~qX4s2-YebzwCoyW8qeG3TQ1bjXwiGoIYY2OEkWj9ZKANVMta3vsP5RphkAJFKgOArAWtDcS1RJtdfut6S0Rzwp1RXwPaKkae1Rboi~39mufRaF8rC5L3ihu7WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7360699&ar=484514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The ratio of <em>Z. celata</em> contig length to <em>A. thaliana</em> orthologue length as a function of contig coverage depth. The dotted line corresponds to a ratio of 1, where 454 contigs are as long or longer than the BLAST matched <em>A. thaliana</em> orthologues.</p></div></div><div content-id="DSR037F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/m_dsr03702.jpeg?Expires=2147483647&Signature=0de5y4u8UWBxO4m39YUfbIMz0Fzvxbw0tXHjtG-Kzv4gwgmCZ9bKLKT~XZ33TWisI3IGtQE~IKonEP2pInh28kfp3XkJMOorD4ZS4opRPNIJ8xHyylwTOKXEPehkzT7u~h0WawFqcSFaGrQUDbrk2TYSE4UDjQMs999eHr7k5tyM5ryG9DTsnsSu-OMgOYgZdQedTX8~AMv9SoA32oCCmlIrD2vtaRED545R0g5SdtekgwKSFR2o1Y~gqnxZUV5uoqX2zpxhyGqIedOkPWQbnLBiAcJB5qvwra-QjWdAMQNimXFdmGY6MdvaUELpNd3lhUmXmHv4D95E7eh3jGliKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ratio of Z. celata contig length to A. thaliana orthologue length as a function of contig coverage depth. The dotted line corresponds to a ratio of 1, where 454 contigs are as long or longer than the BLAST matched A. thaliana orthologues." path-from-xml="dsr03702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7360699/dsr03702.jpeg" path-from-xml="dsr03702.jpeg" target="_blank">View large</a><a section="7360699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr03702.jpeg?Expires=2147483647&Signature=CKEQwQ7KG2fTXmuBwNTUtgnnxG~wiRl4OxnFRLJJWGkxOWqudqSgt92NujpDRCkvXK1uanciYN0rbJiJY9AmNWUnQSVhBeFP3YEMI2EpK2ZDa8WJQOaUZLHjUOrWaeK33SLNdES7plzK59fKB1ddygBSvhKokkI4MKXEimXEFgjLiTWRxlLiO6it1NAbk3J2GDdgTUGz2A99nJPG~tDuyZUEA1u~qX4s2-YebzwCoyW8qeG3TQ1bjXwiGoIYY2OEkWj9ZKANVMta3vsP5RphkAJFKgOArAWtDcS1RJtdfut6S0Rzwp1RXwPaKkae1Rboi~39mufRaF8rC5L3ihu7WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7360699&ar=484514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The ratio of <em>Z. celata</em> contig length to <em>A. thaliana</em> orthologue length as a function of contig coverage depth. The dotted line corresponds to a ratio of 1, where 454 contigs are as long or longer than the BLAST matched <em>A. thaliana</em> orthologues.</p></div></div><p>As expected, the vast majority of BLAST hits matched to known plant proteins (Table <a reveal-id="DSR037TB2" data-open="DSR037TB2" class="link link-reveal link-table xref-fig">2</a>). The abundance of BLAST matches to plant-specific proteins was further indicated by sequences matching nearly as many unique proteins in searches of TAIR as those with Uniref50. There were a small number of BLAST hits to non-plant proteins (Table <a reveal-id="DSR037TB2" data-open="DSR037TB2" class="link link-reveal link-table xref-fig">2</a>), which may be due to representation of these proteins in Uniref50 by non-plant taxa, short or poor quality consensus sequences in 454 <em>Z. celata</em> contigs, or because the sampled tissues contained other organisms such as pathogens. </p>                        <a id="7360701" scrollto-destination="7360701"></a>
<div content-id="DSR037TB2" class="table-modal"><div class="table-wrap"><div id="DSR037TB2" class="table-wrap-title" data-id="DSR037TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of unique BLAST matches to annotated proteins in different taxonomic groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Taxonomic group </th><th>Unique BLAST hits </th></tr></thead><tbody><tr><td>Plants </td><td>19 790 </td></tr><tr><td>Algae </td><td>100 </td></tr><tr><td>Other eukaryote </td><td>496 </td></tr><tr><td>Fungi </td><td>138 </td></tr><tr><td>Bacteria </td><td>131 </td></tr><tr><td>Virus </td><td>553 </td></tr><tr><td>Other </td><td>1499 </td></tr></tbody></table></div><div content-id="DSR037TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Taxonomic group </th><th>Unique BLAST hits </th></tr></thead><tbody><tr><td>Plants </td><td>19 790 </td></tr><tr><td>Algae </td><td>100 </td></tr><tr><td>Other eukaryote </td><td>496 </td></tr><tr><td>Fungi </td><td>138 </td></tr><tr><td>Bacteria </td><td>131 </td></tr><tr><td>Virus </td><td>553 </td></tr><tr><td>Other </td><td>1499 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7360701" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR037TB2" class="table-wrap-title" data-id="DSR037TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of unique BLAST matches to annotated proteins in different taxonomic groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Taxonomic group </th><th>Unique BLAST hits </th></tr></thead><tbody><tr><td>Plants </td><td>19 790 </td></tr><tr><td>Algae </td><td>100 </td></tr><tr><td>Other eukaryote </td><td>496 </td></tr><tr><td>Fungi </td><td>138 </td></tr><tr><td>Bacteria </td><td>131 </td></tr><tr><td>Virus </td><td>553 </td></tr><tr><td>Other </td><td>1499 </td></tr></tbody></table></div><div content-id="DSR037TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Taxonomic group </th><th>Unique BLAST hits </th></tr></thead><tbody><tr><td>Plants </td><td>19 790 </td></tr><tr><td>Algae </td><td>100 </td></tr><tr><td>Other eukaryote </td><td>496 </td></tr><tr><td>Fungi </td><td>138 </td></tr><tr><td>Bacteria </td><td>131 </td></tr><tr><td>Virus </td><td>553 </td></tr><tr><td>Other </td><td>1499 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7360701" target="_blank">View Large</a></div></div><p>Annotation and BLAST searches identified six <em>Z. celata</em> sequences that putatively belonged to the <em>T2 RNase</em> gene family, five of which were isolated from contig consensus sequences and one of which was isolated from a singleton sequence. We translated these <em>Z. celata</em> sequences into amino acid sequences and aligned them with previously characterized, complete <em>T2-RNase</em> amino acid sequences from 12 plant families. Although the 5′ and 3′ ends of the amino acid sequences were highly variable and difficult to align across taxa, we were able to easily align ∼110 highly conserved amino acids, suggesting that the six <em>Z. celata</em> sequences identified in this study are likely members of the <em>T2 RNase</em> gene family. The <em>Z. celata T2 RNases</em> were 115–206 residues in length (347–647 bp) and were shorter than all 12 of the complete <em>T2 RNase</em> sequences (which ranged from 222 to 275 residues). Alignments revealed that the <em>Z. celata T2 RNase</em> sequences did not span the full <em>T2 RNase</em> protein-coding region, as all were truncated on either the 5′ or 3′ end. We detected significant diversity among the six <em>Z. celata T2 RNases</em>, but additional analyses of these alignments will be necessary to determine whether any of these <em>T2 RNase</em> sequences are orthologues of <em>S-RNase</em> genes that encode the female component of GSI in other taxa.</p>                            <h3 scrollto-destination=7360703 id="7360703" class="section-title" data-legacy-id=s3b>3.2. Molecular marker characterization</h3>
<p>SSRs were highly abundant in the 454 DNA sequences, occurring in 17% of sequences from the combined set of contig consensus sequences and singletons. We identified 10 954 di-, tri-, and tetra-nucleotide repeats in the 84 465 contig consensus sequences, and an additional 33 587 SSRs in the 181 312 singleton sequences (Table <a reveal-id="DSR037TB3" data-open="DSR037TB3" class="link link-reveal link-table xref-fig">3</a>). Although influenced by the criteria used to identify these SSRs, tri-nucleotide repeats were the most common, followed by di- and tetra- nucleotide repeats (Table <a reveal-id="DSR037TB3" data-open="DSR037TB3" class="link link-reveal link-table xref-fig">3</a>). A total of 11 926 SSRs occurred in contigs with BLAST matches to Uniref50 annotated proteins, of which 4246 occurred in protein-coding regions of these sequences. The density of SSRs was much higher in non-coding regions that followed coding regions than in coding regions (0.0016 SSRs per basepair in coding regions vs. 0.0075 in non-coding regions). Of the three repeat motifs, di-nucleotide repeats were the most abundant in coding regions (2551), followed by tri- (1684) and tetra-nucleotide (11) repeats. Primer design was successful for a much larger percentage of contig consensus sequences (76%) than singleton sequences (23%), presumably because the shorter length of the singleton sequences limited the availability of suitable priming sites. <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a> provide primer sequences, GC content, melting temperatures, expected product length, SSR motif, SSR sequence, SSR length, and sequence identification for the SSRs developed from the <em>Z. celata</em> 454 sequences. SNPs were also highly abundant in the <em>Z. celata</em> 454 contigs. Across 1.4 million bases represented by a minimum of 8× coverage, we identified 11 056 SNPs that were present at a minimum frequency of 25%, resulting in a SNP occurrence rate of 0.008 per base, similar to that reported in other 454 transcriptome studies.<sup><a reveal-id="DSR037C13" data-open="DSR037C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR037C16" data-open="DSR037C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR037C17" data-open="DSR037C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                        <a id="7360705" scrollto-destination="7360705"></a>
<div content-id="DSR037TB3" class="table-modal"><div class="table-wrap"><div id="DSR037TB3" class="table-wrap-title" data-id="DSR037TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of each type of SSRs detected in the <em>Z. celata</em> transcriptome. Values in parentheses indicate the number of sequences for which PCR primers were successfully designed.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat motif </th><th>Contigs </th><th>Singletons </th><th>Total </th></tr></thead><tbody><tr><td>Di </td><td>3493 (2145) </td><td>6620 (2075) </td><td>10 113 (4220) </td></tr><tr><td>Tri </td><td>4195 (3836) </td><td>11 638 (3188) </td><td>15 833 (7024) </td></tr><tr><td>Tetra </td><td>3266 (2313) </td><td>15 329 (2379) </td><td>18 595 (4692) </td></tr><tr><td>Total </td><td>10 954 (8294) </td><td>33 587 (7643) </td><td>44 541 (15 936) </td></tr></tbody></table></div><div content-id="DSR037TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat motif </th><th>Contigs </th><th>Singletons </th><th>Total </th></tr></thead><tbody><tr><td>Di </td><td>3493 (2145) </td><td>6620 (2075) </td><td>10 113 (4220) </td></tr><tr><td>Tri </td><td>4195 (3836) </td><td>11 638 (3188) </td><td>15 833 (7024) </td></tr><tr><td>Tetra </td><td>3266 (2313) </td><td>15 329 (2379) </td><td>18 595 (4692) </td></tr><tr><td>Total </td><td>10 954 (8294) </td><td>33 587 (7643) </td><td>44 541 (15 936) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7360705" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR037TB3" class="table-wrap-title" data-id="DSR037TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of each type of SSRs detected in the <em>Z. celata</em> transcriptome. Values in parentheses indicate the number of sequences for which PCR primers were successfully designed.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat motif </th><th>Contigs </th><th>Singletons </th><th>Total </th></tr></thead><tbody><tr><td>Di </td><td>3493 (2145) </td><td>6620 (2075) </td><td>10 113 (4220) </td></tr><tr><td>Tri </td><td>4195 (3836) </td><td>11 638 (3188) </td><td>15 833 (7024) </td></tr><tr><td>Tetra </td><td>3266 (2313) </td><td>15 329 (2379) </td><td>18 595 (4692) </td></tr><tr><td>Total </td><td>10 954 (8294) </td><td>33 587 (7643) </td><td>44 541 (15 936) </td></tr></tbody></table></div><div content-id="DSR037TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat motif </th><th>Contigs </th><th>Singletons </th><th>Total </th></tr></thead><tbody><tr><td>Di </td><td>3493 (2145) </td><td>6620 (2075) </td><td>10 113 (4220) </td></tr><tr><td>Tri </td><td>4195 (3836) </td><td>11 638 (3188) </td><td>15 833 (7024) </td></tr><tr><td>Tetra </td><td>3266 (2313) </td><td>15 329 (2379) </td><td>18 595 (4692) </td></tr><tr><td>Total </td><td>10 954 (8294) </td><td>33 587 (7643) </td><td>44 541 (15 936) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7360705" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=7360706 id="7360706" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=7360707 id="7360707" class="section-title" data-legacy-id=s4a>4.1. Transcriptome assembly, coverage breadth and depth, and gene annotation</h3>
<p>454 pyrosequencing has arisen as a powerful tool for the sequencing of transcriptomes, and many studies have used <em>de novo</em> assembly of such data to produce and characterize genome-level resources for non-model organisms.<sup><a reveal-id="DSR037C11" data-open="DSR037C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSR037C13" data-open="DSR037C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR037C16" data-open="DSR037C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR037C23" data-open="DSR037C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSR037C45" data-open="DSR037C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Similarly, a half-plate run on the 454 GS-FLX Titanium platform provided substantial coverage of the <em>Z. celata</em> floral transcriptome, and <em>de novo</em> assembly placed a large fraction (72%) of the 454 sequences into contigs, a large number of which were of considerable length and coverage (Fig <a reveal-id="DSR037F2" data-open="DSR037F2" class="link link-reveal link-table xref-fig">2</a>). BLAST hits to more than 22 000 unique proteins indicate that a large portion of the floral transcriptome was likely sequenced. The large number of diverse GO assignments of these transcripts also highlights the diversity of genes likely represented by these data. Furthermore, many contigs were as long as, or longer than, the corresponding <em>Arabidopsis</em> orthologues, indicating thorough transcript coverage for many genes. These sequences provide a significant genomic-level resource for an endangered plant species and plant family for which very little DNA sequence data previously existed.</p><p>Nearly all of the unique BLAST matches were to proteins characterized in green plants (Table <a reveal-id="DSR037TB3" data-open="DSR037TB3" class="link link-reveal link-table xref-fig">3</a>). A small subset, however, were to proteins characterized in other organisms, and could represent contaminant RNA in the floral tissues we sampled. In particular, a reasonably large number of matches were to plant viral genes (Table <a reveal-id="DSR037TB3" data-open="DSR037TB3" class="link link-reveal link-table xref-fig">3</a>). However, low frequencies of BLAST hits to distantly related taxonomic groups are common in such transcriptome sequencing studies and are often attributable to factors other than contaminant RNA,<sup><a reveal-id="DSR037C13" data-open="DSR037C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR037C17" data-open="DSR037C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR037C45" data-open="DSR037C45" class="link link-ref link-reveal xref-bibr">45</a></sup> e.g. these sequences could be short contigs containing little information, genes that have not been well characterized in plants, or novel <em>Z. celata</em> genes.</p><p>Transcriptome sequencing studies have proven highly valuable for gene discovery, characterization, and variant analysis in both model and non-model organisms.<sup><a reveal-id="DSR037C11" data-open="DSR037C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSR037C16" data-open="DSR037C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR037C17" data-open="DSR037C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR037C46" data-open="DSR037C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSR037C47" data-open="DSR037C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Here, we detected genes expressed in the floral tissues of <em>Z. celata</em> that are putative members of the <em>T2 RNase</em> gene family, demonstrating that 454 transcriptome sequencing is a useful approach for identifying sequences belonging to specific gene families in non-model organisms. We detected significant variation among amino acid sequences within the gene family, indicating that we may have isolated multiple paralogues. Further phylogenetic analyses and analyses of protein-coding motifs for these sequences will be necessary to determine whether these sequences are orthologues or paralogues of <em>S-RNases</em>. If any of the sequences identified in this study encode the <em>S-RNase</em> gene, future research will focus on genotyping this gene in all individuals of <em>Z. celata</em>, which will allow us to identify rare mating types and determine the compatibility of extant genotypes in this self-incompatible species. An understanding of S-locus diversity in <em>Z. celata</em> will help to establish genetically diverse populations capable of reproducing sexually, thereby increasing levels of genetic diversity in this highly endangered species, with the ultimate goal of forming self-sustaining and demographically viable populations.</p>                            <h3 scrollto-destination=7360711 id="7360711" class="section-title" data-legacy-id=s4b>4.2. Molecular marker characterization</h3>
<p>The 454 pyrosequencing and other next-generation sequencing technologies have increased the opportunity for molecular marker development in non-model study organisms at an unprecedented scale.<sup><a reveal-id="DSR037C10" data-open="DSR037C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSR037C16" data-open="DSR037C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR037C17" data-open="DSR037C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In addition, the gene-based markers developed from transcriptome sequencing projects have the advantages of higher cross-species transferability.<sup><a reveal-id="DSR037C8" data-open="DSR037C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR037C48" data-open="DSR037C48" class="link link-ref link-reveal xref-bibr">48</a></sup> We detected and characterized an enormous number of SSR and SNP loci that will likely facilitate future evolutionary, ecological, and conservation-genetic-oriented studies in <em>Z. celata</em>. The primer sets developed for candidate SSRs will allow higher density genotyping that will facilitate accurate assessment of the number of extant genotypes in this species. These markers will also be used for parentage analysis of seedlings to identify the parents with the highest fitness and to ensure that translocated populations contain the highest possible levels of genetic diversity. Furthermore, if we are unable to identify the genes encoding GSI, we will employ these SSR markers to carry out parentage analysis to provide indirect estimates of compatible mating types. The thousands of high-quality SNPs contained in deeply covered contigs offer an enormous number of informative sites that will facilitate genome-wide analyses of molecular variation in future studies.</p>                            <h3 scrollto-destination=7360713 id="7360713" class="section-title" data-legacy-id=s4c>4.3. Conclusions</h3>
<p>In this study, we characterized the floral transcriptome, isolated thousands of molecular markers, and identified transcripts that may encode the S-locus in <em>Z. celata</em>, a highly endangered shrub species. This work has taken an important initial step in characterizing the protein-coding portion of the genome of <em>Z. celata</em>, providing a substantial genomic-level resource for an endangered plant species and plant family for which little previous DNA sequence data existed. Indeed, given that very few endangered plant species have been the focus of genomic or next-generation DNA sequencing efforts, <em>Z. celata</em> may now have one of the best-characterized genomes of any endangered plant species. This work also highlights the utility of using next-generation sequencing for marker and gene discovery; we isolated thousands of genetic markers to be used for parentage analysis, and identified members of a specific gene family that may encode GSI in <em>Z. celata</em>. These resources will be employed to facilitate recovery efforts for <em>Z. celata</em>, highlighting the utility of the new technology in rapidly expanding the resources available to conservation biologists to aid in the recovery of highly endangered species.</p>                            <h2 scrollto-destination=7360715 id="7360715" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7360717 id="7360717" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the US Fish and Wildlife Service (Grant #40181AG003 to C.W.W.), the Florida Division of Forestry Plant Conservation Grants Program, and Archbold Biological Station.</p>                            <h2 scrollto-destination=7360719 id="7360719" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Alex Buerkle and Johan Grahnen for assistance with data analysis, Martin Heine and staff at GATC Inc. for technical work; Stacy Smith and Megan Larson for field assistance in collecting Florida <em>Ziziphus</em> flower samples under difficult field conditions; David Bender for acquisition and administration of the USFWS grant; Dennis Hardin and Mike Jenkins of the Florida Division of Forestry for DOF funding, and Eric Menges, Satoshi Tabata, and two anonymous reviewers for comments on previous versions of this manuscript.</p>                            <h2 scrollto-destination=7360721 id="7360721" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR037C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DeSalle</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Amato</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The expansion of conservation genetics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">702</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20expansion%20of%20conservation%20genetics&amp;author=R.%20DeSalle&amp;author=G.%20Amato&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=702-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1425" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1425"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15372093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krauss</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">K.W.</div></div>. </div><div class="article-title">Rapid genetic decline in a translocated population of the endangered plant <em>Grevillea scapigera</em></div>, <div class="source">Conserv. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">986</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20genetic%20decline%20in%20a%20translocated%20population%20of%20the%20endangered%20plant%20Grevillea%20scapigera&amp;author=S.L.%20Krauss&amp;author=B.%20Dixon&amp;author=K.W.%20Dixon&amp;publication_year=2002&amp;journal=Conserv.%20Biol.&amp;volume=16&amp;pages=986-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1523-1739.2002.01105.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1523-1739.2002.01105.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1523-1739.2002.01105.x"> </span></div> </div></div></div></div></div><div content-id="DSR037C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mino</div> <div class="given-names">C.I.</div></div>,  <div class="name"><div class="surname">Sawyer</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Benjamin</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Del Lama</div> <div class="given-names">S.N.</div></div>. </div><div class="article-title">Parentage and relatedness in captive and natural populations of the roseate spoonbill (Aves: Ciconiiformes) based on microsatellite Data</div>, <div class="source">J. Exp. Zool. Part A</div> , <div class="year">2009</div>, vol. <div class="volume">311A</div> (pg. <div class="fpage">453</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parentage%20and%20relatedness%20in%20captive%20and%20natural%20populations%20of%20the%20roseate%20spoonbill%20(Aves%3A%20Ciconiiformes)%20based%20on%20microsatellite%20Data&amp;author=C.I.%20Mino&amp;author=G.M.%20Sawyer&amp;author=R.C.%20Benjamin&amp;author=S.N.%20Del%20Lama&amp;publication_year=2009&amp;journal=J.%20Exp.%20Zool.%20Part%20A&amp;volume=311A&amp;pages=453-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jez.v311a:6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjez.v311a:6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjez.v311a:6"> </span></div> </div></div></div></div></div><div content-id="DSR037C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Riley</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">McGlaughlin</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Helenurm</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genetic diversity following demographic recovery in the insular endemic plant <em>Galium catalinense</em> subspecies acrispum</div>, <div class="source">Conserv. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">2015</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20following%20demographic%20recovery%20in%20the%20insular%20endemic%20plant%20Galium%20catalinense%20subspecies%20acrispum&amp;author=L.%20Riley&amp;author=M.E.%20McGlaughlin&amp;author=K.%20Helenurm&amp;publication_year=2010&amp;journal=Conserv.%20Genet.&amp;volume=11&amp;pages=2015-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10592-010-0094-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10592-010-0094-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10592-010-0094-8"> </span></div> </div></div></div></div></div><div content-id="DSR037C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Funk</div> <div class="given-names">W.C.</div></div>,  <div class="name"><div class="surname">Forsman</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mullins</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Haig</div> <div class="given-names">S.M.</div></div>. </div><div class="article-title">Evidence for recent population bottlenecks in northern spotted owls (<em>Strix occidentalis caurina</em>)</div>, <div class="source">Conserv. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1013</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20recent%20population%20bottlenecks%20in%20northern%20spotted%20owls%20(Strix%20occidentalis%20caurina)&amp;author=W.C.%20Funk&amp;author=E.D.%20Forsman&amp;author=M.%20Johnson&amp;author=T.D.%20Mullins&amp;author=S.M.%20Haig&amp;publication_year=2010&amp;journal=Conserv.%20Genet.&amp;volume=11&amp;pages=1013-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10592-009-9946-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10592-009-9946-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10592-009-9946-5"> </span></div> </div></div></div></div></div><div content-id="DSR037C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCraney</div> <div class="given-names">W.T.</div></div>,  <div class="name"><div class="surname">Goldsmith</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Jacobs</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Kinziger</div> <div class="given-names">A.P.</div></div>. </div><div class="article-title">Rampant drift in artificially fragmented populations of the endangered tidewater goby (<em>Eucyclogobius newberryi</em>)</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">3315</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rampant%20drift%20in%20artificially%20fragmented%20populations%20of%20the%20endangered%20tidewater%20goby%20(Eucyclogobius%20newberryi)&amp;author=W.T.%20McCraney&amp;author=G.%20Goldsmith&amp;author=D.K.%20Jacobs&amp;author=A.P.%20Kinziger&amp;publication_year=2010&amp;journal=Mol.%20Ecol.&amp;volume=19&amp;pages=3315-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-294X.2010.04755.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2010.04755.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-294X.2010.04755.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20666998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allendorf</div> <div class="given-names">F.W.</div></div>,  <div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Luikart</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Genomics and the future of conservation genetics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">697</div>-<div class="lpage">709</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20and%20the%20future%20of%20conservation%20genetics&amp;author=F.W.%20Allendorf&amp;author=P.A.%20Hohenlohe&amp;author=G.%20Luikart&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=697-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2844" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2844" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2844"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20847747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bouck</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vision</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The molecular ecologists guide to expressed sequence tags</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2007</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">907</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20ecologists%20guide%20to%20expressed%20sequence%20tags&amp;author=A.%20Bouck&amp;author=T.%20Vision&amp;publication_year=2007&amp;journal=Mol.%20Ecol.&amp;volume=16&amp;pages=907-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-294X.2006.03195.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2006.03195.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-294X.2006.03195.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17305850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mardis</div> <div class="given-names">E.R.</div></div>. </div><div class="article-title">The impact of next-generation sequencing technology on genetics</div>, <div class="source">Trends Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">133</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20next-generation%20sequencing%20technology%20on%20genetics&amp;author=E.R.%20Mardis&amp;publication_year=2008&amp;journal=Trends%20Genet.&amp;volume=24&amp;pages=133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.12.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.12.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.12.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18262675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Sequencing breakthroughs for genomic ecology and evolutionary biology</div>, <div class="source">Mol. Ecol. Resour.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">3</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20breakthroughs%20for%20genomic%20ecology%20and%20evolutionary%20biology&amp;author=M.E.%20Hudson&amp;publication_year=2008&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=8&amp;pages=3-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1471-8286.2007.02019.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1471-8286.2007.02019.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1471-8286.2007.02019.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21585713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheat</div> <div class="given-names">C.W.</div></div>. </div><div class="article-title">Rapidly developing functional genomics in ecological model systems via 454 transcriptome sequencing</div>, <div class="source">Genetica</div> , <div class="year">2010</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">433</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapidly%20developing%20functional%20genomics%20in%20ecological%20model%20systems%20via%20454%20transcriptome%20sequencing&amp;author=C.W.%20Wheat&amp;publication_year=2010&amp;journal=Genetica&amp;volume=138&amp;pages=433-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-008-9326-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-008-9326-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-008-9326-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W.E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vera</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Wheat</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Fescemyer</div> <div class="given-names">H.W.</div></div>, et al. </div><div class="article-title">Rapid transcriptome characterization for a nonmodel organism using 454 pyrosequencing</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1636</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20transcriptome%20characterization%20for%20a%20nonmodel%20organism%20using%20454%20pyrosequencing&amp;author=J.C.%20Vera&amp;author=C.W.%20Wheat&amp;author=H.W.%20Fescemyer&amp;publication_year=2008&amp;journal=Mol.%20Ecol.&amp;volume=17&amp;pages=1636-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2008.17.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2008.17.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2008.17.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Sequencing goes 454 and takes large-scale genomics into the wild</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1629</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20goes%20454%20and%20takes%20large-scale%20genomics%20into%20the%20wild&amp;author=H.%20Ellegren&amp;publication_year=2008&amp;journal=Mol.%20Ecol.&amp;volume=17&amp;pages=1629-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2008.17.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2008.17.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2008.17.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18284566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holt</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.J.M.</div></div>. </div><div class="article-title">The new paradigm of flow cell sequencing</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">839</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20new%20paradigm%20of%20flow%20cell%20sequencing&amp;author=R.A.%20Holt&amp;author=S.J.M.%20Jones&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=839-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.073262.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.073262.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.073262.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18519653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meyer</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Aglyamova</div> <div class="given-names">G.V.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Sequencing and <em>de novo</em> analysis of a coral larval transcriptome using 454 GS-FLX</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">219</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20de%20novo%20analysis%20of%20a%20coral%20larval%20transcriptome%20using%20454%20GS-FLX&amp;author=E.%20Meyer&amp;author=G.V.%20Aglyamova&amp;author=S.%20Wang&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19435504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parchman</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Geist</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Grahnen</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Benkman</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Buerkle</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">180</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20in%20an%20ecologically%20important%20tree%20species%3A%20assembly%2C%20annotation%2C%20and%20marker%20discovery&amp;author=T.L.%20Parchman&amp;author=K.S.%20Geist&amp;author=J.A.%20Grahnen&amp;author=C.W.%20Benkman&amp;author=C.A.%20Buerkle&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-180" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-180" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-180"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20233449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dutta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kumawat</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">B.P.</div></div>, et al. </div><div class="article-title">Development of genic SSR markers by deep transcriptome sequencing in pigeonpea <em>Cajanus cajan</em> (L.) Millspaugh</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">17</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genic%20SSR%20markers%20by%20deep%20transcriptome%20sequencing%20in%20pigeonpea%20Cajanus%20cajan%20(L.)%20Millspaugh&amp;author=S.%20Dutta&amp;author=G.%20Kumawat&amp;author=B.P.%20Singh&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol.&amp;volume=11&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21251263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Angeloni</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Wagemaker</div> <div class="given-names">C.A.M.</div></div>,  <div class="name"><div class="surname">Jetten</div> <div class="given-names">M.S.M.</div></div>, et al. </div><div class="article-title">De novo transcriptome characterization and development of genomic tools for <em>Scabiosa columbaria</em> L. using next-generation sequencing techniques</div>, <div class="source">Mol. Ecol. Resour.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">662</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20characterization%20and%20development%20of%20genomic%20tools%20for%20Scabiosa%20columbaria%20L.%20using%20next-generation%20sequencing%20techniques&amp;author=F.%20Angeloni&amp;author=C.A.M.%20Wagemaker&amp;author=M.S.M.%20Jetten&amp;publication_year=2011&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=11&amp;pages=662-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2011.11.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2011.11.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2011.11.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21676196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaur</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cogan</div> <div class="given-names">N.O.I.</div></div>,  <div class="name"><div class="surname">Pembleton</div> <div class="given-names">L.W.</div></div>, et al. </div><div class="article-title">Transcriptome sequencing of lentil based on second-generation technology permits large-scale unigene assembly and SSR marker discovery</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">265</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20lentil%20based%20on%20second-generation%20technology%20permits%20large-scale%20unigene%20assembly%20and%20SSR%20marker%20discovery&amp;author=S.%20Kaur&amp;author=N.O.I.%20Cogan&amp;author=L.W.%20Pembleton&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=265" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21609489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>De novo</em> assembly of chickpea transcriptome using short reads for gene discovery and marker identification</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">53</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20chickpea%20transcriptome%20using%20short%20reads%20for%20gene%20discovery%20and%20marker%20identification&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=53-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bajgain</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Richardson</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Price</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Cronn</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Udall</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Transcriptome characterization and polymorphism detection between subspecies of big sagebrush (<em>Artemisia tridentata</em>)</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">370</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20characterization%20and%20polymorphism%20detection%20between%20subspecies%20of%20big%20sagebrush%20(Artemisia%20tridentata)&amp;author=P.%20Bajgain&amp;author=B.A.%20Richardson&amp;author=J.C.%20Price&amp;author=R.C.%20Cronn&amp;author=J.A.%20Udall&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=370" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-370" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-370" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-370"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21767398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hale</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">McCormick</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">DeWoody</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Next-generation pyrosequencing of gonad transcriptomes in the polyploid lake sturgeon (<em>Acipenser fulvescens</em>): the relative merits of normalization and rarefaction in gene discovery</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">203</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20pyrosequencing%20of%20gonad%20transcriptomes%20in%20the%20polyploid%20lake%20sturgeon%20(Acipenser%20fulvescens)%3A%20the%20relative%20merits%20of%20normalization%20and%20rarefaction%20in%20gene%20discovery&amp;author=M.C.%20Hale&amp;author=C.R.%20McCormick&amp;author=J.R.%20Jackson&amp;author=J.A.%20DeWoody&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19402907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Godt</div> <div class="given-names">M.J.W.</div></div>,  <div class="name"><div class="surname">Race</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hamrick</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">A population genetic analysis of Ziziphus celata, an endangered Florida shrub</div>, <div class="source">J. Hered.</div> , <div class="year">1997</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">531</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20population%20genetic%20analysis%20of%20Ziziphus%20celata%2C%20an%20endangered%20Florida%20shrub&amp;author=M.J.W.%20Godt&amp;author=T.%20Race&amp;author=J.L.%20Hamrick&amp;publication_year=1997&amp;journal=J.%20Hered.&amp;volume=88&amp;pages=531-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jhered.a023151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jhered.a023151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jhered.a023151"> </span></div> </div></div></div></div></div><div content-id="DSR037C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weekley</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Kubisiak</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Race</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Genetic impoverishment and cross-incompatibility in remnant genotypes of <em>Ziziphus celata</em> (Rhamnaceae), a rare shrub endemic to the Lake Wales Ridge, Florida</div>, <div class="source">Biodivers. Conserv.</div> , <div class="year">2002</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">2027</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20impoverishment%20and%20cross-incompatibility%20in%20remnant%20genotypes%20of%20Ziziphus%20celata%20(Rhamnaceae)%2C%20a%20rare%20shrub%20endemic%20to%20the%20Lake%20Wales%20Ridge%2C%20Florida&amp;author=C.W.%20Weekley&amp;author=T.L.%20Kubisiak&amp;author=T.M.%20Race&amp;publication_year=2002&amp;journal=Biodivers.%20Conserv.&amp;volume=11&amp;pages=2027-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1020810800820" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1020810800820" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1020810800820"> </span></div> </div></div></div></div></div><div content-id="DSR037C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gitzendanner</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Weekley</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Germain-Aubrey</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Soltis</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Soltis</div> <div class="given-names">P.S.</div></div>. </div><div class="article-title">Microsatellite evidence for high clonality and limited genetic diversity in <em>Ziziphus celata</em> (Rhamnaceae), an endangered, self-incompatible Lake Wales Ridge, Florida, USA endemic</div>, <div class="source">Conserv. Genet</div>  <div class="comment">in review</div></div></div></div></div><div content-id="DSR037C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Judd</div> <div class="given-names">W.S.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Kellogg</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Stevens</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Donoghue</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Plant Systematics: A Phylogenetic Approach</div> , <div class="year">2008</div><div class="publisher-loc">Sunderland</div><div class="publisher-name">Sinauer Associates, Inc</div></div></div></div></div><div content-id="DSR037C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="collab">USFWS</div><div class="article-title">Florida Ziziphus</div>, <div class="source">Multi-species Recovery Plan for the Threatened and Endangered Species of South Florida</div> , <div class="year">1999</div><div class="publisher-loc">Atlanta, GA</div><div class="publisher-name">U.S. Fish and Wildlife Service</div>(pg. <div class="fpage">1986</div>-<div class="lpage">99</div>)</div></div></div></div><div content-id="DSR037C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="collab">USFWS</div>, <div class="source">Florida Ziziphus (<em>Ziziphus celata</em> 5-year Status Review: Summary and Evaluation</div> , <div class="year">2009</div><div class="publisher-loc">Vero Beach, FL</div><div class="publisher-name">U.S. Fish and Wildlife Service</div></div></div></div></div><div content-id="DSR037C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medan</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Basilio</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Reproductive biology of <em>Colletia spinosissima</em> (Rhamnaceae) in Argentina</div>, <div class="source">Plant Syst. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">229</div> (pg. <div class="fpage">79</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reproductive%20biology%20of%20Colletia%20spinosissima%20(Rhamnaceae)%20in%20Argentina&amp;author=D.%20Medan&amp;author=A.M.%20Basilio&amp;publication_year=2001&amp;journal=Plant%20Syst.%20Evol.&amp;volume=229&amp;pages=79-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s006060170019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs006060170019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs006060170019"> </span></div> </div></div></div></div></div><div content-id="DSR037C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Webb</div> <div class="given-names">C.J.</div></div>. </div><div class="article-title">Protandry, pollination, and self-incompatibility in <em>Discaria toumatou</em></div>, <div class="source">N. Z. J. Bot.</div> , <div class="year">1985</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">331</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protandry%2C%20pollination%2C%20and%20self-incompatibility%20in%20Discaria%20toumatou&amp;author=C.J.%20Webb&amp;publication_year=1985&amp;journal=N.%20Z.%20J.%20Bot.&amp;volume=23&amp;pages=331-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/0028825X.1985.10425335" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F0028825X.1985.10425335" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F0028825X.1985.10425335"> </span></div> </div></div></div></div></div><div content-id="DSR037C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medan</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Vasellati</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Nonrandom mating in <em>Discaria americana</em> (Rhamnaceae)</div>, <div class="source">Plant Syst. Evol.</div> , <div class="year">1996</div>, vol. <div class="volume">203</div> (pg. <div class="fpage">179</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonrandom%20mating%20in%20Discaria%20americana%20(Rhamnaceae)&amp;author=D.%20Medan&amp;author=V.%20Vasellati&amp;publication_year=1996&amp;journal=Plant%20Syst.%20Evol.&amp;volume=203&amp;pages=179-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00985243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00985243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00985243"> </span></div> </div></div></div></div></div><div content-id="DSR037C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medan</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Reproductive biology of <em>Frangula alnus</em> (Rhamnaceae) in southern Spain</div>, <div class="source">Plant Syst. Evol.</div> , <div class="year">1994</div>, vol. <div class="volume">193</div> (pg. <div class="fpage">173</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reproductive%20biology%20of%20Frangula%20alnus%20(Rhamnaceae)%20in%20southern%20Spain&amp;author=D.%20Medan&amp;publication_year=1994&amp;journal=Plant%20Syst.%20Evol.&amp;volume=193&amp;pages=173-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00983549" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00983549" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00983549"> </span></div> </div></div></div></div></div><div content-id="DSR037C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medan</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">D'Ambrogio</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Reproductive biology of the andromonoecious shrub <em>Trevoa quinquenervia</em> (Rhamnaceae)</div>, <div class="source">Bot. J. Linn. Soc.</div> , <div class="year">1998</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">191</div>-<div class="lpage">206</div>)</div></div></div></div><div content-id="DSR037C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weekley</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Race</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The breeding system of <em>Ziziphus celata</em> Judd and DW Hall (Rhamnaceae), a rare endemic plant of the Lake Wales Ridge, Florida, USA: implications for recovery</div>, <div class="source">Biol. Conserv.</div> , <div class="year">2001</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">207</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20breeding%20system%20of%20Ziziphus%20celata%20Judd%20and%20DW%20Hall%20(Rhamnaceae)%2C%20a%20rare%20endemic%20plant%20of%20the%20Lake%20Wales%20Ridge%2C%20Florida%2C%20USA%3A%20implications%20for%20recovery&amp;author=C.W.%20Weekley&amp;author=T.%20Race&amp;publication_year=2001&amp;journal=Biol.%20Conserv.&amp;volume=100&amp;pages=207-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0006-3207(01)00024-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0006-3207(01)00024-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0006-3207(01)00024-6"> </span></div> </div></div></div></div></div><div content-id="DSR037C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zietsman</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Botha</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">Flowering of <em>Ziziphus mucronata</em> subsp. <em>mucronata</em> (Rhamnaceae)—anthesis, pollination and protein synthesis</div>, <div class="source">Bot. Bull. Acad. Sinica</div> , <div class="year">1992</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">33</div>-<div class="lpage">42</div>)</div></div></div></div><div content-id="DSR037C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Faivre</div> <div class="given-names">A.E.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Weekley</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Menges</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">Summary of 2005–2007 observations of pollen grains and pollen tube growth in Florida ziziphus and observations of open-pollinations and aborted fruits</div>, <div class="source">Continuation of Research on the Federally-listed Lake Wales Ridge Endemic Florida Ziziphus (<em>Ziziphus celata</em>)</div> , <div class="year">2007</div><div class="publisher-loc">Tallahassee, FL</div><div class="publisher-name">Final Report to Plant Conservation Program, Florida Division of Forestry</div></div></div></div></div><div content-id="DSR037C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vieira</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fonseca</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Vieira</div> <div class="given-names">C.P.</div></div>. </div><div class="article-title">An <em>S-RNase</em>-based gametophytic self-incompatibility system evolved only once in eudicots</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">179</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20S-RNase-based%20gametophytic%20self-incompatibility%20system%20evolved%20only%20once%20in%20eudicots&amp;author=J.%20Vieira&amp;author=N.A.%20Fonseca&amp;author=C.P.%20Vieira&amp;publication_year=2008&amp;journal=J.%20Mol.%20Evol.&amp;volume=67&amp;pages=179-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-008-9137-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-008-9137-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-008-9137-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18626680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Igic</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Kohn</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Evolutionary relationships among self-incompatibility <em>RNases</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">13167</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20relationships%20among%20self-incompatibility%20RNases&amp;author=B.%20Igic&amp;author=J.R.%20Kohn&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=13167-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.231386798" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.231386798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.231386798"> </span></div> </div></div></div></div></div><div content-id="DSR037C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzek</div> <div class="given-names">B.E.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">H.Z.</div></div>,  <div class="name"><div class="surname">McGarvey</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Mazumder</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">C.H.</div></div>. </div><div class="article-title">UniRef: comprehensive and non-redundant UniProt reference clusters</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1282</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniRef%3A%20comprehensive%20and%20non-redundant%20UniProt%20reference%20clusters&amp;author=B.E.%20Suzek&amp;author=H.Z.%20Huang&amp;author=P.%20McGarvey&amp;author=R.%20Mazumder&amp;author=C.H.%20Wu&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1282-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17379688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swarbreck</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wilks</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The Arabidopsis Information Resource (TAIR): gene structure and function annotation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D1009</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20gene%20structure%20and%20function%20annotation&amp;author=D.%20Swarbreck&amp;author=C.%20Wilks&amp;author=P.%20Lamesch&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D1009-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm965" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm965" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm965"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17986450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rambaut</div> <div class="given-names">A.</div></div>. </div>, <div class="source">Se-Al: A Manual Sequence Alignment Editor</div> , <div class="year">2003</div> <div class="comment">2.0 a11</div></div></div></div></div><div content-id="DSR037C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">You</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Huo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">Y.Q.</div></div>, et al. </div><div class="article-title">BatchPrimer3: a high throughput web application for PCR and sequencing primer design</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">253</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BatchPrimer3%3A%20a%20high%20throughput%20web%20application%20for%20PCR%20and%20sequencing%20primer%20design&amp;author=F.M.%20You&amp;author=N.%20Huo&amp;author=Y.Q.%20Gu&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18510760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Der</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Wickett</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">dePamphilis</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">P.G.</div></div>. </div><div class="article-title"><em>De novo</em> characterization of the gametophyte transcriptome in bracken fern, <em>Pteridium aquilinum</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">99</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20characterization%20of%20the%20gametophyte%20transcriptome%20in%20bracken%20fern%2C%20Pteridium%20aquilinum&amp;author=J.P.%20Der&amp;author=M.S.%20Barker&amp;author=N.J.%20Wickett&amp;author=C.W.%20dePamphilis&amp;author=P.G.%20Wolf&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-99" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-99" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-99"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21303537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Emrich</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Barbazuk</div> <div class="given-names">W.B.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>. </div><div class="article-title">Gene discovery and annotation using LCM-454 transcriptome sequencing</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">69</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20discovery%20and%20annotation%20using%20LCM-454%20transcriptome%20sequencing&amp;author=S.J.%20Emrich&amp;author=W.B.%20Barbazuk&amp;author=L.%20Li&amp;author=P.S.%20Schnable&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=69-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5145806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5145806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5145806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17095711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Novaes</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Drost</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Farmerie</div> <div class="given-names">W.G.</div></div>, et al. </div><div class="article-title">High-throughput gene and SNP discovery in <em>Eucalyptus grandis</em>, an uncharacterized genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">312</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20gene%20and%20SNP%20discovery%20in%20Eucalyptus%20grandis%2C%20an%20uncharacterized%20genome&amp;author=E.%20Novaes&amp;author=D.R.%20Drost&amp;author=W.G.%20Farmerie&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-312" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-312" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-312"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18590545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR037C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR037C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellis</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Burke</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">EST-SSRs as a resource for population genetic analyses</div>, <div class="source">Heredity</div> , <div class="year">2007</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">125</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST-SSRs%20as%20a%20resource%20for%20population%20genetic%20analyses&amp;author=J.R.%20Ellis&amp;author=J.M.%20Burke&amp;publication_year=2007&amp;journal=Heredity&amp;volume=99&amp;pages=125-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6801001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6801001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6801001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17519965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484514&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr037-dsr037supp_table4.xls?Expires=2147483647&amp;Signature=P8O0fZlHQoa7hiMhxvt3p51sseRjx7RCVOqF97cFC~vh4-ae5WmzOZHcjX~3O~5uQG347urMO9Yxf1fBIWc5PJBhJ8SI59~4G9ail1Ooz9vyWP8rLRfMAymnN98k0WVecVmcfLqIWaMDDhCZZXXSafybSPKH2suIAV-o9KR6HogwfuLAOixnRD6~~-m0s1Le-sWIfsKkFl5qzTh36D0qER6nQHKVn3wWNPkhemJqGya3ZQmU9Z~EKJEbQ8Jo5-8FV8~nQVZr02z8I00sakVqcyGf3B91tI2TG1ufAGMOtU5~Hi2AXELFq914FqNROC8e3kRtE8h6n3xn6GuTGprHVA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr037-dsr037supp_table3.xls?Expires=2147483647&amp;Signature=iKvIqnWPlRcdY1cmwvEmejsbTC9ZROvjtLspoYYcVHPsguhNTu2yLrW-4yw03gdPzmrz56BdFP0uBiskpgrQAf7Ba3THkKAjWTvzLh2aohoVUaNIpr0Z6KrH1y5B09qo7ABflVDv5~RRj8v2RZQ7zE3HClOuZdllBJTeD~lV5eP8NZWSGCc~1UuK7eRhDnypJT5nyd4-cu3cs6vqExm~By3QQPFY3JOe5HVPwFKfqZpqOf9cUiAgqjW6V7R0vcBXYiz5QUeYPFvoBD7fZcoddPjZR8k5N-t2GZlwR2SufymXZLelECWVJz0NTJquvHIugR3bDlFuFDNsfP10elAQww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr037-dsr037supp_table2.xlsx?Expires=2147483647&amp;Signature=eKooqKcxvvy0f6qtBqf8abQe29~JVwNJEhyJBHTklS3FM20icY-K5cSA3BmquFTo3Z8haBJrbmWCzC7rZxT5rI-QNk95Xwus8uU7hElRCWrC4Ea~y1R7afUBI1qTHMEKSutMpeBLASOVurfQ-Ou3UpvN5NOOrfkFsZ1qwjwu92v7VS2btl~fuC6rrna1ABzeX3-iOoPwBy2Aff8E2MIXbEQc9xC9dRQrvmuSlztRu7ebsLZ~arEYfEXG8XWHemZVcekvvvnbyXHx8GYLTL4Yro~O1hY0O3XbUYM5r~rvqMv~vO3isdQmwFEIJzdsU70WlwDcsbpTM6vFkHy8NUVqog__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr037-dsr037supp_table1.xlsx?Expires=2147483647&amp;Signature=DyaUvEJRGKXTiLm1VprhjFXuNqWZ-7v-kuJXv3yZ2R62y3aCiyjjEQr4bxxy4BfPcBuHRWDKXoxf8ftlqsafKYMr9ks9cLZuSnG-OEQal63r8Cnm4siysMH1hGnMoc7xXVUXt8Qj2MA5i7SJoH9mt1hab7KkEZj7blBcwbspp6IKdqwY1heL00b5lratUg1J017-VyKIOnDoOsdUrb0OopYRNphlJ4YkLaCU1QdvhN9HLXjqW9x2SqM6Kcek5mFy9PEktuG~MTlZs6szGG3IASdboM-Bpcn19PDzzBS-LqWkF6zXbYYwh15LJuRKFFjuWHctPKVkmXCWqZW7gCwZIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr037-dsr037supp_fig2.pdf?Expires=2147483647&amp;Signature=OZ77Frtp1Fc4OucFNKFmzGNalsUDx81Tp4rkugn41QuVACReDRomhmeUqnoYdP~xXge83FRLSj6XQV2PahkAvohuYzZKzr0A4W-wkw4bg4QIJDmGetWjxFazEUn0EK37l~rSr0bz0eyWitfS3TUVt71uPV04cehl~S4kevmHrbccqvuxUAJMrSNNs8MYGiZjkMpRNMLbLteq0eNfOiOamGVJpb4F0fkZLFaAniLLCItq5D1jn7LCdx3Pvtd4DNpNJlPNznV-m42~D2jXYDs3ghb~MXYbzbBEQn4k84-b2OgUuwAYTQ4VXiXf~GaHe1zsiR862S4zSxOju8TALdY9Gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr037-dsr037supp_fig1.pdf?Expires=2147483647&amp;Signature=E7lT289g9Nvs5Kqw94uXfb3t93ED79iZDYnvka48HovbKgTC-uPt9J9OIoXhMXcJF~Dv6l00xkmWN28OUcYN6WAqM2mwk0GUx9u-WU~FRX5C4ca3d7NgYuaLbB3l7wQi54h5-EtGKn-g5H3DTSgHM-DyH5hweXjmiMa-crEvk4So6GIG4oVvRoYtxY0r4UxJ2h1qpxfS-o7rZk90F9goa26TaGVmNlBxy1XXZ-TP1KNRlUHIQ-NTUSX5MGBucOTCD~tsnPc0fcWiQvy~e6OKlMHh3fxqtURC2UbtguGkKtDyFb1sxiDapYdxOjhaoRq6AI4Np5cN-UiAR1zmW1jvdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr037/2/dsr037-dsr037supp.doc?Expires=2147483647&amp;Signature=p0uJVP~6vC9dG5KAVDJ8IzD7vkKwhR88tWdUaAq7lZ3T53~fxM3anDvQPoKp~TAXaf6~XU0AxE2BhiCDboaoL1poJPf4bDimqB-xHOTHkfmuZnAvpIFsoBLTPeqw26V6xovXGSHWYjIvOwCSIxIzGSdQVa6nM0KbKBV0KBRX3KR8URuy4YDwHbBhqZqzQ8yH7FzHyj6jKjkD5s-Hh9UgWD89PiYkdAy8zv7m347lw~ijXZw-1ZpQpKdVcUpPPeqSallr~thdHuQtVM79qOe41GoeOTh66GA9JmNf6i4iDzFCPkiAFRvh32pb71qWvxZeRprY0gx6SPdl6d5VqdL1Gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">545</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">28</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr037"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484514"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f1%2f1%2f484514">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000300760000001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (28)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F1%2F1%2F484514" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484514'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Assembly, Gene Annotation and Marker Development Using 454 Floral Transcriptome Sequences in Ziziphus Celata (Rhamnaceae), a Highly Endangered, Florida Endemic Plant" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr037");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f1%2f484514">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

